### Accession
PXD002032

### Title
Proteomic analysis of HMGA1-silencing

### Description
HMGA1 is a well-established oncogene and is a master regulator in breast cancer cells controlling the shift from a non-tumorigenic epithelial-like phenotype towards a highly aggressive mesenchymal-like one. In this work we compared HMGA1-silenced versus control MDA-MB-231 by means of a label free shotgun proteomics approach and crossed these data with DNA microarray expression profile obtained on the same cells.  Resulting data were then filtered for genes linked to poor prognosis in breast cancer gene expression meta-datasets. This workflow allows us to establish a small molecular signature composed by 21 members with a prognostic value as regards overall-, recurrence free-, and distant metastasis free-survival in breast cancer.

### Sample Protocol
Each biological replicate was analyzed in technical duplicates using MED-FASP approach [Wisniewski and Mann, 2012]. Peptides released by LysC and tryptic digestions were analyzed in separate LC-MS/MS runs. The LC separation was performed on a C18 reverse phase (12 cm i.d × 75 µm) column packed with 5 µm resin and the analysis was conducted using Orbitrap instrument as described previously [Wisniewski and Mann, 2012].

### Data Protocol
Analysis of the raw data was performed using the MaxQuant software (version 1.2.6.20) [Cox and Mann, 2008]. Proteins were identified by searching MS and MS/MS data of peptides against a decoy version of the UniProtKB/Swiss-Prot. Carbamidomethylation of cysteines was set as fixed modification. The maximum false peptide discovery rate was specified as 0.01. Label-free quantitation of the data was based on LFQ intensities. Only proteins identified at least in 4 samples were subject of quantitation. Missing values were imputed (width 0.3, downshift 1.8) and sample data were normalized by the corresponding median values. T-test was applied for testing differences in protein intensities. Significance of the outliers was calculated by multiple hypotheses testing with the threshold value of 0.05 [Benjamini and Hochberg, 1995].

### Publication Abstract
Cancer is a very heterogeneous disease, and biological variability adds a further level of complexity, thus limiting the ability to identify new genes involved in cancer development. Oncogenes whose expression levels control cell aggressiveness are very useful for developing cellular models that permit differential expression screenings in isogenic contexts. HMGA1 protein has this unique property because it is a master regulator in breast cancer cells that control the transition from a nontumorigenic epithelial-like phenotype toward a highly aggressive mesenchymal-like one. The proteins extracted from HMGA1-silenced and control MDA-MB-231 cells were analyzed using label-free shotgun mass spectrometry. The differentially expressed proteins were cross-referenced with DNA microarray data obtained using the same cellular model and the overlapping genes were filtered for factors linked to poor prognosis in breast cancer gene expression meta-data sets, resulting in an HMGA1 protein signature composed of 21 members (HRS, HMGA1 reduced signature). This signature had a prognostic value (overall survival, relapse-free survival, and distant metastasis-free survival) in breast cancer. qRT-PCR, Western blot, and immunohistochemistry analyses validated the link of three members of this signature (KIFC1, LRRC59, and TRIP13) with HMGA1 expression levels both in vitro and in vivo and wound healing assays demonstrated that these three proteins are involved in modulating tumor cell motility. Combining proteomic and genomic data with the aid of bioinformatic tools, our results highlight the potential involvement in neoplastic transformation of a restricted list of factors with an as-yet-unexplored role in cancer. These factors are druggable targets that could be exploited for the development of new, targeted therapeutic approaches in triple-negative breast cancer.

### Keywords
High mobility protein hmga1, Filter aided sample preparation, Label free proteomics, Breast cancer

### Affiliations
Proteomics
Max-Planck Intitute of Biochemistry, Dept. Proteomics and Signal Transduction

### Submitter
Mario Oroshi

### Lab Head
Dr Jacek R Wisniewski
Max-Planck Intitute of Biochemistry, Dept. Proteomics and Signal Transduction


