### Accession
PXD008057

### Title
Fine-scale adaptive divergence and plasticity revealed by genetic and phenotypic variation in a marine invertebrate

### Description
The interplay between phenotypic plasticity and adaptive evolution has long been an important topic of evolutionary biology. This process is critical to our understanding of a species evolutionary potential in light of rapid climate changes. Despite recent theoretical work, empirical studies of natural populations, especially in marine invertebrates, are scarce. In this study, we investigated the relationship between adaptive divergence and plasticity by integrating genetic and phenotypic variation in Pacific oysters from its natural range in China. Genome resequencing of 371 oysters revealed unexpected fine-scale genetic structure that is largely consistent with phenotypic divergence in growth, physiology, thermal tolerance and gene expression across environmental gradient. These findings suggest that selection and local adaptation are pervasive and together with limited gene flow shape adaptive divergence. Plasticity in gene expression is positively correlated with evolved divergence, indicating that plasticity is adaptive and likely favored by selection in organisms facing dynamic environments such as oysters. Divergence in heat response and tolerance implies that the evolutionary potential to a warming climate differs among oyster populations. We suggest that trade-offs in energy allocation are important to adaptive divergence with acetylation playing a role in energy depression under thermal stress.

### Sample Protocol
Mantle samples were the same as in the transcriptomic assessment. Equal amounts of gills from 10 oysters at each time points of 0h, 6h and 24h for each population (JZ and QD) were mixed and sonicated for protein extraction (a total of six mixed samples). Approximately 100 Î¼g of protein for each sample was reduced, alkylated and trypsin digested. The resulting peptides were labeled with TMT isobaric labels, mixed and fractionated by high pH reverse-phase HPLC. For the acetylome, affinity enrichment for acetylated peptides was required using pan-acetyl lysine antibody agarose beads. Enriched (acetylome) and nonenriched (proteome) fractions were analyzed by a reverse phase liquid chromatography coupled to tandem mass spectrometry (MS/MS).

### Data Protocol
The resulting MS/MS data was analyzed using MaxQuant with the integrated Andromeda search engine (v.1.5.2.8). Trypsin/P was specified as cleavage enzyme allowing up to 4 missing cleavages, 5 modifications per peptide and 5 charges. Mass error was set to 10 ppm for precursor ions and 0.02 Da for fragment ions. Tandem mass spectra were searched against the uniprot Crassostrea gigas database. False discovery rate thresholds for protein, peptide and modification site were specified at 1%. Minimum peptide length was set at 7. Relative acetylation was calibrated to corresponding protein level29. In total, 6,047 protein groups were identified, among which 4,778 proteins were quantified. Also, 1,054 Lysine acetylation sites in 664 protein groups were identified, among which 718 sites in 461 proteins were quantified. The fold-change cutoff was set so that proteins with quantitative ratios above 1.5 or below 1/1.5 were deemed significant.

### Publication Abstract
None

### Keywords
Genetic and phenotypic variation, Marine invertebrate, Acetylome, Proteome

### Affiliations
PTM Biolabs Inc.
Key Laboratory of Experimental Marine Biology, Institute of Oceanology, Chinese Academy of Sciences, Qingdao, China

### Submitter
Rachel  Green

### Lab Head
Dr Guofan Zhang
Key Laboratory of Experimental Marine Biology, Institute of Oceanology, Chinese Academy of Sciences, Qingdao, China


