### Accession
PXD006650

### Title
Human, melanoma, p62/SQSTM1, quantitative proteomics

### Description
p62/SQSTM1 was identified as a modulator of metastatic genes selectively enriched in melanoma in autophagy independent manner. iTRAQ quantitative proteomic approach was performed in melanoma cell lines (SK-Mel-103 and UACC-62) deficient for p62 to identify downstream effectors of p62. Similar studies were performed for ATG5, a core component of autophagy, as a reference for autophagy-associated changes in protein abundance. Additionally, melanoma cells were subjected to affinity purification (AP-MS) to identify the interactors of p62. Overall, these studies underscore a novel unexpected role of p62 regulating the stability of prometastatic factors via the interaction with RNA Binding Proteins, thus leading to the inhibition of protein translation.

### Sample Protocol
In the iTRAQ experiments (SK-Mel-103, UACC-62 and ATG5), cell pellets were extracted and proteins were digested using the FASP protocol with some modifications. Peptides were labeled with iTRAQ reagents and samples were pooled. Then, the complex mixture was subjected to IEF fractionation. The resulting fractions were separated by on-line nano-LC and analyzed by electrospray MS/MS using a LTQ Orbitrap Velos mass spectrometer (Thermo Scientific).  In the AP-MS experiment, the proteins were eluated from the beads with 8M urea in 0.1M Tris-HCl. The eluates were doubly digested using the FASP procedure with some modifications. The resulting peptides were analyzed by LC-MS/MS using a LTQ Orbitrap Velos mass spectrometer (Thermo Scientific).

### Data Protocol
Data analysis in the iTRAQ experiments: The raw files were processed using the Proteome Discoverer 1.3.0.339 software suite (Thermo Scientific). The fragmentation spectra were searched against the concatenated SwissProt Human database (release date: March 21, 2012; 20329 entries) using MASCOT (v2.2) ( for UACC62 and SKMel103 datasets) or SequestHT (ATG5 dataset) as the search engine with the precursor and fragment mass tolerances set to 10 ppm and 0.075 Da, respectively, and with up to two missed cleavages. Lysine and peptide N-termini labeling with iTRAQ-4plex reagent as well as carbamidomethylation of cysteine were considered as fixed modifications, while oxidation of methionine was chosen as variable modification for database searching. Minimal peptide length was set to 6 amino acids and Mascot ion score > 20. Peptide identification false discovery rate (FDR) was less than 1%. Sequest HT, in conjunction with Percolator provided the peptide list filtered at 1% FDR. In case that identified peptides were shared by two or more proteins (homologs or isoforms), they were reported by Proteome Discoverer as one protein group. The results were then exported into Excel for manual data interpretation. Although relative quantification and some statistical data were provided by the Proteome Discoverer software, an additional 1.3-fold change cutoff for all iTRAQ ratios (ratio <0.77 or >1.3) was selected to classify proteins as up- or down-regulated. Proteins with iTRAQ ratios below the low range (0.77) were considered to be under-expressed, while those above the high range (1.3) were considered overexpressed.   Data analysis in the AP-MS experiment: Raw files were searched against a Uniprot Homo sapiens database (release date: March 21, 2012; 20329 entries) using Andromeda as the search engine through the MaxQuant software (v1.4.1.2). Carbamidomethylation of cysteines was considered as a fixed modification whereas oxidation of methionines was set as variable modification. Other parameters were set as default. Peptide and protein identifications were filtered by Percolator at 1% FDR using the target-decoy strategy.

### Publication Abstract
Modulators of mRNA stability are not well understood in melanoma, an aggressive tumor with complex changes in the transcriptome. Here we report the ability of p62/SQSTM1 to extend mRNA half-life of a spectrum of pro-metastatic factors. These include FERMT2 and other transcripts with no previous links to melanoma. Transcriptomic, proteomic, and interactomic analyses, combined with validation in clinical biopsies and mouse models, identified a selected set of RNA-binding proteins (RBPs) recruited by p62, with IGF2BP1 as a key partner. This p62-RBP interaction distinguishes melanoma from other tumors where p62 controls autophagy or oxidative stress. The relevance of these data is emphasized by follow-up analyses of patient prognosis revealing p62 and FERMT2 as adverse determinants of disease-free survival.

### Keywords
Human, Ap-ms, Quantitative proteomics, P62/sqstm1, Lc-ms/ms, Melanoma cell lines

### Affiliations
Proteomics Unit Spanish National Cancer Research Centre (CNIO) Madrid,SPAIN
Proteomics Group

### Submitter
Pilar Ximenez-Embun

### Lab Head
Dr Javier Mu√±oz Peralta
Proteomics Unit Spanish National Cancer Research Centre (CNIO) Madrid,SPAIN


