### Accession
PXD014317

### Title
Proteomic analysis of mitochondrial biogenesis in cardiomyocytes differentiated from human induced pluripotent stem cell

### Description
Mitochondria play a crucial role in the differentiation and maturation of human cardiomyocytes (CMs). To identify mitochondrial pathways and regulators that are involved in cardiac differentiation and maturation, we examined human induced pluripotent stem cell-derived cardiomyocytes (hiPSC-CMs). Proteomic analysis was performed on enriched mitochondrial protein extracts isolated from hiPSC-CMs differentiated from dermal fibroblasts (dFCM) and cardiac fibroblasts (cFCM), at different days of differentiation (between 12 and 115 days), and also from adult and neonatal mouse hearts for comparison. Mitochondrial proteins with a â‰¥2-fold change between differentiation time points in dFCMs and cFCMs, and between adult versus neonatal mouse hearts, were subjected to Ingenuity Pathway Analysis (IPA), and some upregulated proteins were validated by immunoblotting. The highest significant upregulation was in metabolic pathways for fatty acid oxidation (FAO), the tricarboxylic acid (TCA) cycle, oxidative phosphorylation (OXPHOS) and branched chain amino acid (BCAA) catabolism. The top upstream regulators predicted by IPA were- peroxisome proliferator-activated receptor gamma coactivator 1 alpha (PGC1-a), the insulin receptor and the retinoblastoma protein (Rb) transcriptional repressor. In addition, IPA and immunoblotting showed substantial upregulation of the mitochondrial LonP1 protease, which regulates mitochondrial proteostasis, energetics and metabolism. Using this proteomics approach, we have identified key metabolic and intracellular signaling pathways that are up- and down- regulated during the biogenesis of mitochondria in differentiating and maturing cardiac myocytes.

### Sample Protocol
Isolated mitochondria were analyzed from- 1) cFCMs at days 12, 40 and 115, and 2) dFCMs at days 12, 39 and 105 and 3) hearts from neonatal (1 day) and adult (90 days) mouse. Proteins were extracted using lysis buffer containing 100 mM Tris/HCl pH 7.5, 4% (w/v) SDS, and 1% protease inhibitor cocktail. Protein concentration was measured by the bicinchoninic acid (BCA) assay. Fifteen micrograms of proteins from each sample were subjected to SDS-PAGE, and the entire lane of the gel for each sample was excised and washed, followed by in-gel trypsin digestion. The resultant peptides were desalted with a C18 spin column and then analyzed by LC-MS/MS on a Q Exactive instrument.

### Data Protocol
The MS/MS spectra were searched against a SwissProt human database using a local MASCOT (V.2.4) search engine on the Proteome Discoverer (V1.4) platform. The protein false discovery rate was less than 1%. The relative quantification of proteins on day 40 versus day 12, day 115 versus day 12, and day 115 versus day 40 was calculated based on the spectral counting method. LC-MS/MS identified between 2041-2690 proteins from the different samples analyzed, 926 of these were identified as mitochondrial proteins based on the gene ontology annotations in Scaffold software (v 4.6, Proteome Software, Portland, Oregon).

### Publication Abstract
Mitochondria play key roles in the differentiation and maturation of human cardiomyocytes (CMs). As human induced pluripotent stem cell-derived cardiomyocytes (hiPSC-CMs) hold potential in the treatment of heart diseases, we sought to identify key mitochondrial pathways and regulators, which may provide targets for improving cardiac differentiation and maturation. Proteomic analysis was performed on enriched mitochondrial protein extracts isolated from hiPSC-CMs differentiated from dermal fibroblasts (dFCM) and cardiac fibroblasts (cFCM) at time points between 12 and 115 days of differentiation, and from adult and neonatal mouse hearts. Mitochondrial proteins with a twofold change at time points up to 120 days relative to 12 days were subjected to ingenuity pathway analysis (IPA). The highest upregulation was in metabolic pathways for fatty acid oxidation (FAO), the tricarboxylic acid (TCA) cycle, oxidative phosphorylation (OXPHOS), and branched chain amino acid (BCAA) degradation. The top upstream regulators predicted to be activated were peroxisome proliferator-activated receptor &#x3b3; coactivator 1 &#x3b1; (PGC1-&#x3b1;), the insulin receptor (IR), and the retinoblastoma protein (Rb1) transcriptional repressor. IPA and immunoblotting showed upregulation of the mitochondrial LonP1 protease-a regulator of mitochondrial proteostasis, energetics, and metabolism. LonP1 knockdown increased FAO in neonatal rat ventricular cardiomyocytes (nRVMs). Our results support the notion that LonP1 upregulation negatively regulates FAO in cardiomyocytes to calibrate the flux between glucose and fatty acid oxidation. We discuss potential mechanisms by which IR, Rb1, and LonP1 regulate the metabolic shift from glycolysis to OXPHOS and FAO. These newly identified factors and pathways may help in optimizing the maturation of iPSC-CMs.

### Keywords
Upstream regulator., Mitochondria, Proteomics, Cardiac differentiation, Human induced pluripotent stem cells, Canonical pathways

### Affiliations
Rutgers-NJMS
Dept. Cell Biology and Molecular Medicince, Rutgers-New Jersey Mediccal School, Newark NJ

### Submitter
Erdene Baljinnyam

### Lab Head
Dr Junichi Sadoshima
Dept. Cell Biology and Molecular Medicince, Rutgers-New Jersey Mediccal School, Newark NJ


