### Accession
PXD008642

### Title
Human breast cancer cell line and tissue, ISE-affinity proteogenomics

### Description
Introducing a clinical-practical, alternative splicing activity-based proteogenomic method that identifies, in their oncogenically active states, biomarker genes bearing patient-specific GE or copy-number alterations of prognostic significance. This integrated multi-omics method uses intronic splicing enhancers (ISEs) probes to sort in situ ISE-interacting trans-acting protein factors (trans-interactome) directly from a heterogeneous tumor for subsequent mass spectrometry (MS) characterization.

### Sample Protocol
ISE pull-downs. 1 mg protein extracted from either cell lines, or clinical tissues was incubated with 10 μl biotinylated non-ISE RNA or ISE probes (0.75 μM) at 4OC for 2 h or overnight. Fifty μl neutravidin-agarose (Thermo-Fisher) was then added for an additional 2 h at 4OC, and washed 3 times with 1 ml lysis buffer to remove non-specific proteins.  For a gel-based approach, the RNA/ISE-bound proteins were eluted, heated for 5 min at 95 °C in 40 μl 2 X SDS loading buffer, separated by SDS PAGE for in-gel trypsin digestion, and desalted with a C18 stage tip. The peptide elutes were subjected to nanoLC-MS/MS. For on-beads sampling and processing, in addition to 3 washings with lysis buffer, a washing buffer (50 mM Tris-HCl, pH 8.0, 150 mM NaCl) was used 5 times to remove residual detergents. On-beads tryptic digestion was performed with 125 μl buffer containing 2 M urea, 50 mM Tris-HCl, pH 8.0, 1 mM DTT, 600 ng trypsin (Promega) for 30 min at room temperature on a mixer (Eppendorf). The tryptic digests were eluted twice with a 100 μl buffer containing 2 M urea, 50 mM Tris-HCl, pH 8.0, 5 mM iodoacetamide. Combined elutions were acidified with 5 μl trifluoroacetic acid (TFA, MS grade, Thermo-Fisher) and desalted by C18 stage tip. LC-MS/MS analysis.  Desalted peptide mixtures were dissolved in 30 μl 0.1% formic acid (Thermo-Fisher), of which 4 μl containing the peptides from 60-100 μg total protein was injected and analyzed by a ultra2D nanoLC system (Eksigent) coupled to a Velos LTQ Orbitrap mass spectrometer  (Thermo Fisher Scientific, San Jose, CA) or an Easy nanoLC 1000 coupled to a Q-Exactive Orbitrap mass spectrometer (Thermo Fisher Scientific, San Jose, CA). In the nanoLC-Velos setup, peptides were first loaded on to a 2 mm × 0.5 mm reverse-phase (RP) C18 trap column (Eksigent) at a flow rate of 1 μl/min, then eluted and fractionated on a 25 cm C18 RP column (360 μm × 75 μm × 3 μm) with a gradient of 5-40% buffer B (ACN and 0.1% formic acid) at a constant flow rate of 250 nl/min over 180 min. In the Easy nanoLC- Q Exactive setup, peptides were loaded on to a 15 cm C18 RP column (280 μm × 75 μm × 2 μm, Acclaim Pepmap RSLC, Thermo-Fisher) and eluted with a gradient of 1-30% buffer B  at a constant flow rate of 300 nl/min for 110 min. The Velos LTQ Orbitrap was operated in the positive-ion mode with a data-dependent automatic switch between survey Full-MS scan (m/z 300-1800) (externally calibrated to a mass accuracy of <5 ppm and a resolution of 60,000 at m/z 400) and CID MS/MS acquisition of the top 15 most intense ions. The Q-Exactive was also operated in the positive-ion mode but using a data-dependent top 20 method. Survey scans were acquired at a resolution of 70,000 at m/z 400. Up to the top 20 most abundant isotope patterns with charge ≥ 2 from the survey scan were selected with an isolation window of 3 m/z and fragmented by HCD with normalized collision energies of 28. The maximum ion injection times for the survey scan and the MS/MS scans were 250 ms and 120 ms, respectively, and the ion target values were set to 1e6 and 2e5, respectively. Selected sequenced ions were dynamically excluded for 20 seconds.

### Data Protocol
Mass spec data analysis. Mass spectral processing and peptide identification were performed on the Andromeda search engine in MaxQuant software (Version 1.5.2.8) against a human UniProt database. All searches were conducted with a defined modification of cysteine carbamidomethylation, with methionine oxidation and protein amino-terminal acetylation as dynamic modifications. Peptides were confidently identified using a target-decoy approach with a peptide false discovery rate (FDR) of 1% and a protein FDR of 5%. A minimum peptide length of 7 amino acids was required, maximally two missed cleavages were allowed, initial mass deviation for precursor ion was up to 7 ppm, and the maximum allowed mass deviation for fragment ions was 0.5 Da. For AACT/SILAC quantification, the multiplicity was set at 2 or 3 (2-plex K0R0, K8R10, 3-plex K0R0, K6R6, K8R10). For LFQ analysis, a match between runs option was enabled and time window at 0.7 minutes. Data processing and statistical analysis were performed on Perseus (Version 1.5.1.6)(Cox and Mann, 2011). Protein quantitation was performed on biological replicate runs, and a two-sample t-test statistics was used with a p-value of 5% to report statistically significant expression fold-changes. The AACT ratios of the proteins were derived from the comparison of the extracted ion chromatogram peak areas of all matched light (L) peptides with those of the medium (M) or heavy (H) peptides. The ratios of M/L or H/L were also verified by visual inspection of the raw mass spectra.

### Publication Abstract
To discriminate the patient subpopulations with different clinical outcomes within each breast cancer (BC) subtype, we introduce a robust, clinical-practical, activity-based proteogenomic method that identifies, in their oncogenically active states, candidate biomarker genes bearing patient-specific transcriptomic/genomic alterations of prognostic value. First, we used the intronic splicing enhancer (ISE) probes to sort ISE-interacting trans-acting protein factors (trans-interactome) directly from a tumor tissue for subsequent mass spectrometry characterization. In the retrospective, proteogenomic analysis of patient datasets, we identified those ISE trans-factor-encoding genes showing interaction-correlated expression patterns (iCEPs) as new BC-subtypic genes. Further, patient-specific co-alterations in mRNA expression of select iCEP genes distinguished high-risk patient subsets/subpopulations from other patients within a single BC subtype. Function analysis further validated a tumor-phenotypic trans-interactome contained the drivers of oncogenic splicing switches, representing the predominant tumor cells in a tissue, from which novel personalized biomarkers were clinically characterized/validated for precise prognostic prediction and subsequent individualized alignment of optimal therapy.

### Keywords
Ise trans-interactomes, Ise-affinity proteomics, Precision marker

### Affiliations
Postdoctoral Research Associate
Department of Biochemistry and Biophysics, University of North Carolina, Chapel Hill,NC 27599,USA

### Submitter
Li Wang

### Lab Head
Dr Xian Chen
Department of Biochemistry and Biophysics, University of North Carolina, Chapel Hill,NC 27599,USA


