### Accession
PXD025389

### Title
Activation of CD44 / AKT signaling promotes resistance to FGFR1 inhibition in squamous cell lung cancer

### Description
Lung cancer is the leading cause of cancer related deaths, worldwide. Fibroblast growth factor receptor 1 (FGFR1) gene amplification is one of the most prominent and potentially targetable genetic alterations in squamous cell lung cancer (SQCLC). Highly selective tyrosine kinase inhibitors have been developed to target FGFR1, however, resistance mechanisms originally existing in patients or acquired throughout treatment have limited treatment efficiency in clinical trials, so far. In this study, we performed a wide-scale phosphoproteomic mass spectrometry analysis to explore signaling pathways that lead to FGFR1 inhibition resistance in lung cancer cells with intrinsic, induced and mutational resistance. We identified a CD44/AKT signaling axis as a new and common mechanism of resistance to FGFR1 inhibition in lung cancer. Co-inhibition of AKT or CD44 synergistically sensitized intrinsic and induced resistant cells to FGFR1 inhibition. Furthermore, strong CD44 expression was significantly correlated to AKT activation in squamous cell lung cancer patients. Collectively, our phosphoproteomic analysis of FGFR1 inhibitor resistant lung cancer cells promotes a large data library of resistance associated phosphorylation patterns and proposes a common resistance pathway consisting of CD44 and AKT activation. Examination of CD44/AKT activation could help to predict response to FGFR1 inhibition and combination with AKT inhibitors might path the way for an effective therapy of FGFR1 dependent lung cancer patients in case of treatment resistance.

### Sample Protocol
Cells were lysed with global phosphoproteome (GPome) or tyrosine enrichment (pYome) lysis buffers. The buffer for GPome analyses was composed of 50 mM Tris-HCL (pH 5.5), 1 mM EDTA, 1% NP-40, protease inhibitor cocktail (Roche), 5 mM NaF, 10 mM NEM and LC/MS grade water. The buffer for pYome analyses was prepared according to manufacturer’s protocol (P-Tyr-1000 kit, Cell Signaling Technology). Cells were scraped in ice-cold PBS and incubated with the appropriate lysis buffer for 20 minutes on ice (GPome) or at RT (pYome).  For GPome analyses the cell suspensions were sonicated for 10 s in 3 cycles and centrifuged for 15 min at 4 °C and 20,000 x g. After equal mixing of 1 mg protein per SILAC condition the supernatants were subjected to acetone precipitation at -20 °C over night and the dried proteins were suspended in urea buffer (8 M urea, 20 mM HEPES, pH 8.0, phosphatase inhibitors), followed by reduction with DTT and alkylation with iodoacetamide. For protein digestion, the samples were incubated with Lys-C (Wako) in an enzyme-to-substrate-ratio of 1:100 (w/w) at 37 °C for 2 h and after dilution to 2 M urea with trypsin (Promega) at 37°C over night in an enzyme-to-substrate ratio of 1:100 (w/w). The peptide samples were purified using Sep-Pak C18 classic cartridges (360 mg, Waters) and dried by vacuum centrifugation. For pYome analyses sample preparation was carried out according to the manufacturer of the deployed enrichment kit (P-Tyr-1000 kit, Cell Signaling Technology). Briefly, the cell suspensions were sonicated and supernatants from SILAC conditions were mixed in equal protein amounts, followed by reduction with DTT and alkylation with iodoacetamide. After dilution to 2 M urea with 20 mM HEPES, pH 8.0 the protein samples were digested with trypsin (Promega) in an enzyme-to-substrate-ratio of 1:100 (w/w) at 37 °C over night. The peptide samples were purified using Sep-Pak C18 classic cartridges and lyophilized. For GPome analyses phosphopeptides were enriched from the dried peptide samples using titanium dioxide affinity chromatography in a spin tip format (Pierce High-Select TiO2 phosphopeptide enrichment kit, Thermo Fisher Scientific) following the instructions of the manufacturer. Briefly, the peptides were suspended in 150 µl of the kit buffer and subjected to phosphopeptide enrichment and purification. After eluting the phospho-peptides from the spin columns two times, the samples were immediately dried by vacuum centrifugation and subjected to high-pH RPLC pre-fractionation (Pierce High pH RPLC fractionation kit, Thermo Fisher Scientific). For pYome analyses, the lyophilized peptide samples were suspended in 1.4 ml of IAP buffer and tyrosine-phosphorylated peptides were immuno-precipitated with the P-Tyr-1000 antibody kit according to the manufacturer’s instructions (Cell Signaling Technology). Enriched phopsho-peptides were eluted under acidic conditions, purified using C18 RPLC microtips and dried by vacuum centrifugation. Phospho-peptide samples were analyzed on a Q Exactive HF orbitrap mass spectrometer (Thermo Fisher Scientific) coupled to an Ultimate 3000 RSLCnano HPLC system (Dionex / Thermo Fisher Scientific) via a nano-ESI interface (Nanospray flex, Thermo Fisher Scientific). First, the peptides were trapped on a precolumn (ReproSil-Pur 120 C18-AQ, 5 µm; Dr. Maisch GmbH; 100 µm x 5 cm) and then separated using an analytical column (ReproSil-Pur 120 C18-AQ, 1.9 μm; Dr. Maisch GmbH; 30.5 × 0.075 mm) at a flow rate of 300 nl/min with a 90-min (GPome) or 120-min (pYome) linear gradient of 2–40% solvent B [80% (vol/vol) ACN, 0.1% FA] over solvent A (0.1% FA). Peptides eluting from the column were ionized and analyzed by data-dependent acquisition using a TopN MS/MS method with a survey scan resolution setting of 120,000 FWHM and an MS/MS resolution setting of 30,000 FWHM at 200 m/z, respectively (settings for pYome measurements indicated in brackets). The 15 (20) most abundant ions within the m/z range 350-1600 and having charge states of 2-6 were selected for HCD with an NCE setting of 28% and an isolation width of 1.6 m/z. AGC target values and maximum ion injection times for MS and MS/MS were set to 1 x 10^6 in 50 ms (40 ms) and 1 x 10^5 in 110 ms (128 ms), respectively. Selected precursor mass-to-charge ratio values were dynamically excluded from fragmentation for 30 s (20 s).

### Data Protocol
Raw data files were analyzed with the MaxQuant software (version 1.6.5.0, MPI for Biochemistry). Mass spectra were searched using the integrated Andromeda search engine against the UniProtKB human reference protein database (date: February 2019) supplemented with 245 common contaminants. Trypsin was set as enzyme for protein digestion and the precursor and fragment ion mass tolerances were set to 4.5 and 20 ppm, respectively. Oxidation of methionine, protein N-terminal acetylation and phosphorylation of serine, threonine and tyrosine were allowed as variable modifications. Carbamidomethylation of cysteine was set as a fixed modification. Minimal peptide length was set to seven amino acids, allowing two missed cleavages. On both the peptide and protein level the maximum false discovery rate (FDR) was set to 1% based on a decoy database search. SILAC multiplicity was set to triple labeling (Lys+0/Arg+0, Lys+4/Arg+6, Lys+8/Arg+10) and at least two ratio counts were required for peptide quantitation. Furthermore, the “re-quantify” option was enabled. 27  For downstream data analysis the Perseus software (1.6.10.43) was used. Identified phosphosites were filtered for potential contaminants, hits to the decoy database and localization probability (>0.75). Samples were grouped by average of technical replicates and statistical significance of biological replicates. Data was compensated for missing values and z-scored to generate heatmaps according to ANOVA+ significant phosphosites.

### Publication Abstract
Lung cancer is the leading cause of cancer-related deaths worldwide. Fibroblast growth factor receptor 1 (FGFR1) gene amplification is one of the most prominent and potentially targetable genetic alterations in squamous-cell lung cancer (SQCLC). Highly selective tyrosine kinase inhibitors have been developed to target FGFR1; however, resistance mechanisms originally existing in patients or acquired during treatment have so far led to limited treatment efficiency in clinical trials. In this study we performed a wide-scale phosphoproteomic mass-spectrometry analysis to explore signaling pathways that lead to resistance toward FGFR1 inhibition in lung cancer cells that display (i) intrinsic, (ii) pharmacologically induced and (iii) mutationally induced resistance. Additionally, we correlated AKT activation to CD44 expression in 175 lung cancer patient samples. We identified a CD44/PAK1/AKT signaling axis as a commonly occurring resistance mechanism to FGFR1 inhibition in lung cancer. Co-inhibition of AKT/FGFR1, CD44/FGFR1 or PAK1/FGFR1 sensitized 'intrinsically resistant' and 'induced-resistant' lung-cancer cells synergetically to FGFR1 inhibition. Furthermore, strong CD44 expression was significantly correlated with AKT activation in SQCLC patients. Collectively, our phosphoproteomic analysis of lung-cancer cells resistant to FGFR1 inhibitor provides a large data library of resistance-associated phosphorylation patterns and leads to the proposal of a common resistance pathway comprising CD44, PAK1 and AKT activation. Examination of CD44/PAK1/AKT activation could help to predict response to FGFR1 inhibition. Moreover, combination between AKT and FGFR1 inhibitors may pave the way for an effective therapy of patients with treatment-resistant FGFR1-dependent lung cancer.

### Keywords
Phosphoproteomics, Lung cancer, Akt, Mass spectrometry, Cd44, Fgfr1

### Affiliations
University medical center Goettingen, Institute of Pathology
Institute of Pathology University Medical Center Robert-Koch-Str. 40 37075 Goettingen Germany

### Submitter
Hanibal Bohnenberger

### Lab Head
Dr Hanibal Bohnenberger
Institute of Pathology University Medical Center Robert-Koch-Str. 40 37075 Goettingen Germany


