### Accession
PXD023357

### Title
Metaproteogenomic profiling of chemosynthetic microbial biofilms reveals metabolic flexibility during colonization of a shallow-water gas vent

### Description
Tor Caldara is a shallow-water gas vent located in the Mediterranean Sea, with active venting of CO 2 , H 2 S. At Tor Caldara, filamentous microbial biofilms, mainly composed of Epsilon- and Gammaproteobacteria, grow on substrates exposed to the gas venting. In this study, we took a metaproteogenomic approach to identify the metabolic potential and in situ expression of central metabolic pathways at two stages of biofilm maturation. Our findings indicate that inorganic reduced sulfur species are the main electron donors and CO 2 the main carbon source for the filamentous biofilms, which conserve energy by oxygen and nitrate respiration, fix dinitrogen gas and detoxify heavy metals. Three metagenome-assembled genomes (MAGs), representative of key members in the biofilm community, were also recovered. Metaproteomic data show that metabolically active chemoautotrophic sulfide-oxidizing members of the Epsilonproteobacteria dominated the young microbial biofilms, while Gammaproteobacteria become prevalent in the established community. The co-expression of different pathways for sulfide oxidation by these two classes of bacteria suggests exposure to different sulfide concentrations within the biofilms, as well as fine-tuned adaptations of the enzymatic complexes. Taken together, our findings demonstrate a shift in the taxonomic composition and associated metabolic activity of these biofilms in the course of the colonization process.

### Sample Protocol
Total proteins were extracted from the YF and EF biofilms by addition of Laemmli buffer, followed by sonication and incubation at 95 Â°C. Proteins were then precipitated by addition of urea, purified and digested with trypsin on a SDS-PAGE gel, and then label-free samples were run on the LC-MSMS.

### Data Protocol
The resulting tandem mass spectra were searched against the predicted peptide sequences encoded by the combined metagenomes of the two filament samples, using the open-source software X!Tandem (Craig and Beavis, 2004). A total of 138,387 and 97,607 peptides were obtained in the metaproteomes of EF and YF, respectively. The proteins identified via X!Tandem were functionally and taxonomically annotated using the corresponding annotated metagenomes. Protein counts were normalized to the total proteins obtained from each sample. Normalized counts for proteins of interest were manually compared between the two samples. These normalized counts were expressed as a percent of total proteins observed in each sample. Raw data was uploaded to the ProteomeXchange database.

### Publication Abstract
None

### Keywords
Tor caldara, Microbial filaments, Metabolic profile, Biofilms, Shallow-water gas vent, Gammaproteobacteria, Epsilonproteobacteria, Metaproteome, Mediterranean sea, Metagenome

### Affiliations
CNR italy
Department of Biochemistry and Microbiology, Rutgers University, New Brunswick, NJ, USA

### Submitter
francesco smedile

### Lab Head
Dr Costantino Vetriani
Department of Biochemistry and Microbiology, Rutgers University, New Brunswick, NJ, USA


