### Accession
PXD011250

### Title
TFIIIC dinamically binds Alu elements to control gene expression through chromatin looping

### Description
Folding of the mammalian genome is governed by architectural proteins, such asCTCF. TFIIIC, a RNA polymerase III transcription factor, has been identified as aninsulator but its role in genome topology is totally unknown. Here, we show that TFIIICestablishes long-range genomic interactions that affect gene expression. Upon serumstarvation (SS), TFIIIC occupancy increases at Alu elements (AEs) near promoters ofcell cycle-related genes. Bound AEs become H3K18 hyper-acetylated and fold tocontact distal pre-loaded CTCF sites near other cell cycle genes. The promoters ofthese genes also become hyper-acetylated ensuring their basal transcription during SSand their increased expression during serum re-exposure. Ablation of TFIIIC ordeletion of the TFIIIC-bound AE that loops to the G2/M cycling F (CCNF) locus affectsits expression and nuclear positioning. These results illustrate a novel function ofhuman TFIIIC in changing 3D genome topology through the epigenetic state of AEs.

### Sample Protocol
Cells were cross-linked with paraformaldehyde for 8 min (Vicent et al., 2014), and stopped by adding a final 200 mM glycine and incubating for 5 min at room temperature. Plates were kept on ice and washed twice with cold phosphate-buffered saline (PBS). Cells were scrapped in ice-cold PBS with protease inhibitors and collected in a 15 ml tube suitable for sonication (BD Polyestyrene, 352095). Cells were centrifuged at 4,000 rpm for 5 min and washed twice with cold PBS. All buffers contain freshly added inhibitors in the following concentration: cOmplete EDTA-free as recommended by manufacturer (1 tablet for 50 ml), 10 µM phenylmethylsulfonyl fluoride and 10 µM Na3VO4l. Cell pellets were resuspended in 10 ml of lysis buffer 1 (50 mM Hepes pH 7.5, 140 mM NaCl, 1 mM EDTA, 10% glycerol, 0.5% NP-40, 0.25% Triton X-100), and incubated on ice for 10 min. After centrifugation, the pellet was resuspended in 10 ml of lysis buffer 2 (10 mM Tris pH 8.0, 200 mM NaCl, 1 mM EDTA, 0.5mM EGTA) and incubated on rotation for 5 min at 4ºC and centrifuged again. The pellet was finally resuspended in 400 µl of lysis buffer 3 (10 mM Tris pH 8.0, 100 mM NaCl, 1 mM EDTA, 0.5 mM EGTA, 0.1% Na-deoxycholate, 0.5% N-lauroylsarcosine) by carefully pipetting up and down for ten times. The extracts were sonicated in a Bioruptor (Diagnode) at 4ºC, for 9 cycles of 30 s on/30 s off, at high output. After sonication, sample was transferred to a 1.7 ml siliconized tube, and 10% Triton X-100 was added. Lysates were centrifuged and supernatant was added to the antibody-conjugated beads. Antibody binding to the beads was done typically, for 107 cells, with 100 µl of Protein A magnetic beads washed once in PBS, resuspended in 500 µl of LB3, with the appropriate amount of antibody or IgG (6 µl of anti-GTF3C2, at 0.2 µg/µl, or 12 ul of rabbit IgG), incubated for 3 h at 4ºC and washed with LB3 twice, 500 µl each. After over-night incubation, the beads were washed 10 times with RIPA buffer (50 mM Tris pH 7.4, 150 mM NaCl, 0.5% Na-deoxycholate, 1% NP-40, 0.1% SDS) and 2 times with 100 mM ammonium hydrogen carbonate (AMBIC) solution. For the second wash, the beads were transferred to new 1.7 ml tubes. The samples were reduced by adding 10 µl of 10 mM DTT in 100 mM ammonium bicarbonate (ABC) buffer (1 h, 37ºC) and alkylated by adding 10 µl of 20 mM iodocetamide in 100 mM ABC (30 min, room temperature, in the dark). The digestion was done in two steps: first, with 1 µg of endopeptidase LysC, incubated over night at 37º, 700 rpm. Second, 1 µg of sequencing grade Trypsin was added and incubated for 8 hr at 37ºC, 700rpm. The digestion reaction was stopped with formic acid (5% final concentration). The supernatant was taken and tryptic peptides were desalted with C18 columns, dried in a Speed-vac and re-suspended in 10 µL of H2O + 0.1% formic acid. The proteomics analyses were performed at the CRG/UPF Proteomics Unit which is part of the of Proteored, PRB3 and is supported by grant PT17/0019, of the PE I+D+i 2013-2016, funded by ISCIII and ERDF. From the resuspended sample, 4.5 µl of each peptide mixture was analyzed using a LTQ-Orbitrap Velos Pro mass spectrometer (Thermo Fisher Scientific, San Jose, USA) coupled to a nano-LC (Proxeon, Odense, Denmark) equipped with a reversed-phase chromatography 2-cm C18 pre-column (Acclaim PepMap-100, Thermo; 100 µm i.d., 5 µm), and a 25-cm C18 analytical column (Nikkyo Technos, 75 μm i.d., 3 μm). Chromatographic gradients started at 3% buffer B with a flow rate of 300 nL/min and gradually increased to 7% buffer B in 1 min and to 35% buffer B in 60 min. After each analysis, the column was washed for 10 min with 90% buffer B (Buffer A: 0.1% formic acid in water; Buffer B: 0.1% formic acid in acetonitrile). The mass spectrometer was operated in positive ionization mode with nanospray voltage set at 2.5 kV and source temperature at 200 °C. Ultramark 1621 was used for external calibration of the FT mass analyzer prior the analyses. The background polysiloxane ion signal at m/z 445.1200 was used as lock mass. The instrument was operated in data-dependent acquisition mode, and full MS scans with 1 microscan at resolution of 60,000 were used over a mass range of m/z 350−1,500 with detection in the Orbitrap. Auto gain control (AGC) was set to 106, dynamic exclusion was set at 60 s, and the charge-state filter disqualifying singly charged peptides for fragmentation was activated. Following each survey scan, the 10 most intense ions with multiple charged ions above a threshold ion count of 5000 were selected for fragmentation at normalized collision energy of 35%. Fragment ion spectra produced via collision-induced dissociation were acquired in the linear ion trap, AGC was set to 3·104 and isolation window of 2.0 m/z, activation time of 30 ms, and maximum injection time of 250 ms were used. All data were acquired with Xcalibur software v2.2.

### Data Protocol
Acquired data were analyzed using the Proteome Discoverer software suite (v1.4, Thermo Fisher Scientific), and the Mascot search engine (v2.5, Matrix Science) was used for peptide identification. Data were searched against the human protein database derived from the SwissProt database plus common contaminants (April 2016; 20,200 sequences). A precursor ion mass tolerance of 7 ppm was used, and up to three missed cleavages were allowed. The fragment ion mass tolerance was set to 0.5 Da, and oxidation (M), and acetylation (Protein N-term) were defined as variable modifications, whereas carbamidomethylation (C) was set as fixed modification. The identified peptides were filtered by FDR < 0.01 (1%). Proteome Discoverer gives an approximate estimation of protein amount with the parameter “Area Under the Curve” (AUC) which is the average peak area of the 3 top peptides for a given protein. For the assessment of protein-protein interactors, we used the SAINT software (Choi et al., 2011), using rabbit IgG as negative controls, and label-free quantitative experiments with samples in triplicate (see Table S9 for results). Protein-protein interactors (ppi) were assessed with the SAINT software (Choi et al., 2011) which calculates the probability of genuine ppi. As an output, SAINT provides a Bayesian Fold Discovery Rate (hereafter FDR), an IgG fold change (IgG FC) and a SAINT score. FDR is the probability to a given identified protein to be a false positive. IgG FC is the enrichment of a given protein compared to the same identified protein at the IgG control. SAINT score is a probability for a given interacting protein to be true, in an inversely relation with FDR. After obtaining a given FDR and IgG FC values for every protein at each condition, Fisher's combined probability test (Fisher 1925) was used for computing 1 single FDR for every identified protein, fusing data from replicates.

### Publication Abstract
How repetitive elements, epigenetic modifications, and architectural proteins interact ensuring proper genome expression remains poorly understood. Here, we report regulatory mechanisms unveiling a central role of Alu elements (AEs) and RNA polymerase III transcription factor C (TFIIIC) in structurally and functionally modulating the genome via chromatin looping and histone acetylation. Upon serum deprivation, a subset of AEs pre-marked by the activity-dependent neuroprotector homeobox Protein (ADNP) and located near cell-cycle genes recruits TFIIIC, which alters their chromatin accessibility by direct acetylation of histone H3 lysine-18 (H3K18). This facilitates the contacts of AEs with distant CTCF sites near promoter of other cell-cycle genes, which also become hyperacetylated at H3K18. These changes ensure basal transcription of cell-cycle genes and are critical for their re-activation upon serum re-exposure. Our study reveals how direct manipulation of the epigenetic state of AEs by a general transcription factor regulates 3D genome folding and expression.

### Keywords
Human, Tfiiic, Breast cancer, Alu

### Affiliations
Center for Genomic Regulation
Gene Regulation, Stem Cells and Cancer Program Centre de Regulació Genòmica (CRG) The Barcelona Institute of Science and Technology PRBB Dr. Aiguader 88 E-08003 Barcelona  Spain  Tlf   +34 93 316 0119 Fax +34 93 316 0099 e-mail: miguel.beato@crg.eu  Secretary: Gemma Perez Tlf  +34 93 316 0165 e-mail: gemma.perez@crg.eu  www.crg.eu/4dgenome

### Submitter
Roberto Ferrari

### Lab Head
Dr Miguel Beato
Gene Regulation, Stem Cells and Cancer Program Centre de Regulació Genòmica (CRG) The Barcelona Institute of Science and Technology PRBB Dr. Aiguader 88 E-08003 Barcelona  Spain  Tlf   +34 93 316 0119 Fax +34 93 316 0099 e-mail: miguel.beato@crg.eu  Secretary: Gemma Perez Tlf  +34 93 316 0165 e-mail: gemma.perez@crg.eu  www.crg.eu/4dgenome


