### Accession
PXD012849

### Title
Characterization of the Zein Protein Composition in Maize Flour Extracts

### Description
Highly homogenous zein protein was isolated from maize kernels in an environment-friendly process using 95 % ethanol as solvent. High purity of the zein protein product was determined by SDS PAGE analysis and by 2 D gel electrophoresis followed by MALDI-ToF-MS peptide mass fingerprinting after in-gel chymotrypsin digestion. Being a natural product that is encoded by multiple gene copies, the polymorphic zein protein product revealed two rows of protein spots, one at 25 kDa and one at 20 kDa apparent molecular mass. MALDI-ToF-MS peptide mapping of the proteins from all spots indicated the presence of only alpha zein proteins. The most prominent ion signals in the MALDI mass spectra after in-gel digestion were recorded at m/z 1083.5 and m/z 1691.8. These ion signals have been found typical for zein proteins and may serve as marker ion signals which upon chymotryptic digestion reliably indicate the presence of zein protein in both hybrid corn products. Due to the given polyploidy and genetic polymorphism of the plant source the application of high resolution separation methods in conjunction with precise analytical methods, such as MALDI-ToF-MS, is required to accurately estimate homogeneity of products that contain natural zein protein.

### Sample Protocol
Preparation of zein protein extracts started with milling corn seeds. The resulting flour was defatted. Defatted flour samples were resuspended in 65% and 95% aqueous ethanol solution, respectively, and sonicated. Zein proteins were precipitated by addition of acetone. The pellets contained the purified zein proteins. Zein proteins were subjected to SDS PAGE and 2DE, respectively. Protein bands and protein spots were excised and treated by in-gel tryptic digest. Peptides were eluted and subjected to MALDI-MS peptide mass fingerprinting.

### Data Protocol
261 entries for maize protein sequences from the UniProtKB database (http://www.uniprot.org/) (UniProt release 2015_02) were manually collected. From these, 112 were assigned as zein protein sequences. These 112 protein sequence entries were manually checked and nine of them were found to be fragments from other proteins and two sequences, although carrying two different accession numbers, were identical. Eliminating the redundant sequence entries and the fragment sequences left 102 non-redundant zein protein amino acid sequence data base entries. All the 102 zein protein sequence data base entries with unique data base accession numbers from the UniProtKB data base were assembled in a home-built data base, named UKB_zein_maize data base. The peptide masses from all ion signals of the recorded MALDI mass spectra were used for matching them to amino acid sequences which were derived from an in-silico digest of all sequence data base entries of the UKB_zein_maize data base. The Mascot software was used for ion signal matching and protein identification, i.e. sequence data base entry identification, using the following search parameters: apply known chymotrypsin cleavage sites, set carbamidomethylation of cysteine residues as fixed modification and oxidation of methionine residues as variable modification, allow 2 missed cleavage sites and set peptide mass tolerance to 80 ppm. An alpha zein protein was considered as positively identified to be present in the bands of the SDS gels when at least 13 peptide ion masses matched to an alpha zein proteinâ€™s calculated peptide masses as retrieved from the respective database sequence entries. The number of 7 matching masses was to be reached by alpha zein proteins of the spots of the 2D gels for alpha zein protein identification.

### Publication Abstract
Highly homogenous &#x3b1; zein protein was isolated from maize kernels in an environment-friendly process using 95% ethanol as solvent. Due to the polyploidy and genetic polymorphism of the plant source, the application of high resolution separation methods in conjunction with precise analytical methods, such as MALDI-TOF-MS, is required to accurately estimate homogeneity of products that contain natural zein protein. The &#x3b1; zein protein product revealed two main bands in SDS-PAGE analysis, one at 25&#xa0;kDa and other at 20&#xa0;kDa apparent molecular mass. Yet, high resolution 2DE revealed approximately five protein spot groups in each row, the first at ca. 25&#xa0;kDa and the second at ca. 20&#xa0;kDa. Peptide mass fingerprinting data of the proteins in the two dominant SDS-PAGE bands matched to 30 amino acid sequence entries out of 102 non-redundant data base entries. MALDI-TOF-MS peptide mapping of the proteins from all spots indicated the presence of only &#x3b1; zein proteins. The most prominent ion signals in the MALDI mass spectra of the protein mixture of the 25&#xa0;kDa SDS gel band after in-gel digestion were found at m/z 1272.6 and m/z 2009.1, and the most prominent ion signals of the protein mixture of the 20&#xa0;kDa band after in-gel digestion were recorded at m/z 1083.5 and m/z 1691.8. These ion signals have been found typical for &#x3b1; zein proteins and may serve as marker ion signals which upon chymotryptic digestion reliably indicate the presence of &#x3b1; zein protein in two hybrid corn products.

### Keywords
Peptide mass fingerprinting, 2d gel electrophoresis, Data base search, Maldi-ms, Zein proteins, Maize, Custom-built sequence data base, Flour

### Affiliations
University Medical Center Rostock
Proteome Center Rostock Department for Proteome Research Institute of Immunology Medical Faculty and Natural Science Faculty University of Rostock Schillingallee 69 18057 Rostock Germany

### Submitter
Michael Kreutzer

### Lab Head
Dr Prof. Dr. Michael O. Glocker
Proteome Center Rostock Department for Proteome Research Institute of Immunology Medical Faculty and Natural Science Faculty University of Rostock Schillingallee 69 18057 Rostock Germany


