### Accession
PXD013775

### Title
Chemical phosphoproteomic approaches reveal substrate specificity of phosphoprotein phosphatases-1 and -2

### Description
The phosphatases PP1 and PP2A are responsible for the majority of dephosphorylation reactions on phosphoserine (pSer) and â€“threonine (pThr), and are involved in basically all cellular processes and many related diseases. They are thought to have no appreciable intrinsic substrate specificity, but to gain specificity only in their holoenzyme forms. Through the development of a peptide library approach and application of a complementary phosphoproteomics assay, we uncover that PP1 and PP2A show intrinsic specificity towards pThr over pSer, as well as toward the sequence context surrounding the phospho-site. Our data reveal that PP1 is a key phosphatase of the 14-3-3 protein binding motif. This result enabled us to establish a previously unknown role for PP1 as regulator of the GRB-associated-binding-protein 2 (Gab2)/14-3-3 complex, exemplifying predictive potential of the data. Thus, our work should serve as a rich resource for (de)phosphorylation studies covering multiple cellular processes and phosphoproteins.

### Sample Protocol
Peptide library synthesis Peptides were synthesized following fluorenylmethyloxycarbonyl- (Fmoc) solid-phase peptide synthesis (SPPS). Wang resin or pre-coupled H-Lys(Boc)-2-ClTrt resin was used for all peptides and libraries. Resins were swollen for 20 min in N,N-dimethylformamide (DMF) prior to synthesis and then transferred to the peptide synthesizer.  One round of synthesis was performed for each amino acid in the peptide sequence.  A synthetic round consisted of double coupling to a Fmoc and side chain-protected amino acid residue, followed by a capping step with acetic anhydride (Ac2O), and then removal of the Fmoc group with piperidine in preparation for the next round of coupling.  Coupling reactions were performed by adding Fmoc-protected amino acids (4 eq.), O-benzotriazole-N,N,N',N'-tetramethyl-uronium hexafluorophosphate (HBTU) (4 eq.), N-hydroxybenzotriazole hydrate (HOBt) (4 eq), and N-methylmorpholine (NMM) to the resin in 1 mL DMF and reacting for 30 - 45 min.  Capping was done by reaction with a 1 mL solution of 5% acetic anhydride Ac2O and 5% 2,6-Lutidine in DMF for 5 min.  Fmoc-deprotection was achieved by addition of a 20% piperidine in DMF solution to the resin, first for 3 min, then again for 8 min.  Resins were washed with DMF between each step. After Fmoc removal of the final amino acid residue to reveal the free N-terminus, peptides and libraries were removed from the resin and fully deprotected in one step by shaking for 3 - 4 h in cleavage cocktail (92.5% trifluoroacetic acid (TFA), 5% triisopropylsilane (TIPS), and 2.5% H2O).  Peptide libraries were isolated by co-evaporation of the cleavage cocktail with toluene in vacuo and directly used for dephosphorylation assays and subsequent LC-MS/MS analysis. Individual peptides were precipitated in cold ether (-20 Â°C) and collected by centrifugation. Peptides were then validated and purified by MALDI, HPLC-MS, and Prep HPLC. For both sets of libraries, the following isokinetic ratio was used for the amino acid position including the phosphate group: phosphoserine and phosphothreonine (amino acid, mol %): Fmoc-Ser(PO(OBzl)OH)-OH, 50; Fmoc-Thr(PO(OBzl)OH)-OH, 50.  Directed peptide synthesis PDP-Nal, PDPm-Nal and peptides for confirmation of MS results were synthesized as previously reported (21) and used from a 10mM stock in 5% DMSO in water.  Isokinetic amino acid mixture for library synthesis For the syntheses of the peptide libraries, variable positions were added to the peptide sequence.  At these variable positions, the resin was reacted with an isokinetic mixture of amino acids, to ensure an equal distribution of amino acids at that position.  The following optimized isokinetic ratios were used to create the amino acid mixtures (amino acid, mol %). x1 Libraries: Fmoc-Ala-OH, 4.6; Fmoc-Arg(Pbf)-OH, 6.8; Fmoc-Glu(OtBu)-OH, 7.4; Fmoc-Gln(Trt)-OH, 11.0; Fmoc-Gly-OH, 9.9; Fmoc-His(Trt)-OH, 4.8; Fmoc-Leu-OH, 3.4; Fmoc-Lys(Boc)-OH, 8.6; Fmoc-Phe-OH, 3.4; Fmoc-Pro-OH, 5.9; Fmoc-Ser(tBu)-OH, 7.6; Fmoc-Thr(tBu)-OH, 13.2; Fmoc-Tyr(tBu)-OH, 5.6; Fmoc-Val-OH, 7.8. x2 Libraries: Fmoc-Ala-OH, 4.3; Fmoc-Arg(Pbf)-OH, 6.3; Fmoc-Asn(Trt)-OH, 10.2; Fmoc-Asp(OtBu)-OH, 6.8; Fmoc-Glu(OtBu)-OH, 6.8; Fmoc-Gln(Trt)-OH, 10.2; Fmoc-Gly-OH, 9.2; Fmoc-Leu-OH, 3.1; Fmoc-Lys(Boc)-OH, 8.0; Fmoc-Phe-OH, 3.1; Fmoc-Pro-OH, 5.5; Fmoc-Ser(tBu)-OH, 7.0; Fmoc-Thr(tBu)-OH, 12.3; Fmoc-Val-OH, 7.2.  These ratios were based on a previously reported mixture (14) and optimized for our syntheses.  Peptide/library dephosphorylation assay The libraries were dissolved in 25% DMSO and 75% H2O at 10 mM. The EnzChekÂ® Phosphate Assay Kit (Thermo Fisher Scientific) was used to assess the kinetics/dynamics of the library or peptide dephosphorylation by PP1c and PP2Ac. The assay was carried out in a volume of 100â€‰ÂµL reaction buffer (20 mM Tris, 100 mM NaCl, 1 mM DTT, 0.15 Units PNP, 0.2 mM MESG,) containing 100 ÂµM phosphopeptide library and 25 nM phosphatase.  The change of absorbance was detected at 360 nm at 25 Â°C on a Synergy H1 microplate reader. For the dephosphorylation assay with the library we aimed for a dephosphorylation rate between 30% â€“ 40% of total dephosphorylation in order to have enough dephosphorylated peptides for significant analysis and to see substrate preference of the respective phosphatase. For analysis of enzyme kinetics, raw data was further analyzed in GraphPad Prism v6.

### Data Protocol
Data analysis using R After IsobarQuant and Mascot filtering the resulting data files were processed using the R programming language (www.r-project.org; R version 3.5.1 (2018-07-02)) with the additional packages ggplot2, graphics, tidyverse, Biostrings, dplyr, stringi, reshape2, readr and splitstackshape for advanced data analysis.  Data was first assessed for empirically wrong peptides (length unequal to ten amino acids, false sequence, no phosphorylation, wrong phosphorylation), and empirically correct peptides (EmpCorr). We then calculated the false discovery rate (FDR) as the ratio of false positives to the sum of true and false positives for each (sub-) library. For further analysis, we excluded all known empirically wrong peptides and applied the determined cut-off at an FDR of 0.05 to true positive hits (EmpCorr) only. The grand average of hydropathy scores (GRAVY scores) were calculated using the free gravy calculator (http://www.gravy-calculator.de/). For each phosphatase-condition, we calculated the counts of a given amino acid in a given position in the dephosphorylated subset compared to the total of all peptides. The dataset was therefore normalized for variation in amino acid incorporation shown in Figure S2A. This percentage was divided by the global dephosphorylation rate of a given treatment resulting in fold-over-/under-representations. After filtering we obtained, ranging between conditions, 9072-9513 unique sequences in a total of at least 111897 peptides for Nterm-libraries and 1570-2069 unique sequences in at set of at least 12473 peptides for Cterm-libraries.

### Publication Abstract
The phosphatases PP1 and PP2A are responsible for the majority of dephosphorylation reactions on phosphoserine (pSer) and phosphothreonine (pThr), and are involved in virtually all cellular processes and numerous diseases. The catalytic subunits exist in cells in form of holoenzymes, which impart substrate specificity. The contribution of the catalytic subunits to the recognition of substrates is unclear. By developing a phosphopeptide library approach and a phosphoproteomic assay, we demonstrate that the specificity of PP1 and PP2A holoenzymes towards pThr and of PP1 for basic motifs adjacent to the phosphorylation site are due to intrinsic properties of the catalytic subunits. Thus, we dissect this amino acid specificity of the catalytic subunits from the contribution of regulatory proteins. Furthermore, our approach enables discovering a role for PP1 as regulator of the GRB-associated-binding protein 2 (GAB2)/14-3-3 complex. Beyond this, we expect that this approach is broadly applicable to detect enzyme-substrate recognition preferences.

### Keywords
Pp1, Phospho peptide library, Pp2a

### Affiliations
EMBL, Heidelberg
EMBL PCF

### Submitter
Dominic Helm

### Lab Head
Dr Mikhail Savitski
EMBL, Heidelberg


