### Accession
PXD006010

### Title
Citrus Proteomics: Lemon CLas graft inoculation Week 10

### Description
Lisbon lemon trees were grafted with budwood infected with the citrus greening bacterium, 'Candidatus Liberibacter asiaticus', or control budwood, and leaf samples were collected every two weeks post graft for LC/MS analysis.  This project has shotgun proteomics data for leaf samples from 5 control and 5 CLas grafted trees at the 10 week post graft timepoint.

### Sample Protocol
Proteins were extracted from one leaf sample per tree for each timepoint.  Leaf samples were flash frozen in liquid nitrogen when collected, and were ground in liquid nitrogen by mortar and pestle.  Protein precipitation solvent (10% trichloroacetic acid in acetone with 2% beta-mercaptoethanol) was made fresh and kept on ice until use.  Precipitation solvent (10 mL/mg wet leaf weight) was added to ground leaf samples in conical screw cap tubes.  Samples were vortexed and stored overnight at -20°C.  Precipitated protein pellets were washed three times with 10 mL ice-cold acetone.  During the third acetone resuspension, protein precipitate solution was transferred into 1.5 mL microcentrifuge tubes and centrifuged.  After decanting the supernatant from the final wash the pellet was dried to completion.  Pellets were resuspended in 500 μL protein reconstitution solvent [8 M urea, 50 mM triethylammonium bicarbonate (TEAB) in water], and samples were incubated in an agitator (TOMY microtube mixer MT-360) with a small stir bar at room temperature overnight.  Samples were centrifuged at full speed to pellet insoluble material, and the supernatant was collected for protein analysis.  Sample protein concentration was measured using the Quick Start Bradford Protein Assay (Bio-Rad). For each sample, a 100 μL solution containing 40 μg protein was prepared in 10 mM dithiotreitol, 50 mM TEAB.  Samples were vortexed, briefly centrifuged, and incubated for 1 h at 30°C.  Protein samples were reduced with tris-carboxy-ethyl phosphine (TCEP): 5 μL of 200 mM TCEP was added to 40 μg of protein in 100 μL TEAB and incubated at 55°C for 1 hour.  Samples were briefly centrifuged and were cooled to room temperature.  Cysteine alkylation was performed by adding 5 μL of 375 mM iodoacetamide to each sample.  Samples were vortexed, centrifuged, and incubated for 1 h at room temperature in the dark. If necessary, TEAB was added to samples prior to trypsin digestion to reduce urea concentration to <1 M.  Sequencing grade, modified trypsin (Promega, Madison, WI) was added to each sample (trypsin:protein ratio 1:40, by weight), and samples were vortexed, centrifuged, and incubated overnight at 30°C.  Trypsin-digested samples were dried and resuspended in 500ul 0.1% Formic Acid in water. Samples are acidified to a pH ≤ 3 by adding 5μl full strength Formic Acid.  Waters Sep-Pak C18 1cc vacuum cartridges (cat#: WAT054955) are used with a Phenomenex vacuum manifold, and pressure is kept between 4-5 inHg.  Columns were conditioned with 3ml 100% Acetonitrile, followed by 3ml 0.1% Formic Acid. Columns were briefly dried, then samples were added and run through. Columns were then washed with 3ml 0.1% Formic Acid, and briefly dried. Samples were eluted off the column into a collection tube by adding 500ul 80% Acetonitrile, 20% (0.1% Formic Acid). Column was dried to ensure all eluent is collected. Cleaned samples were dried down before mass spec analysis. Reversed phase chromatographic separation of peptide samples was performed using an EASY-nLC 1000 system (Thermo Scientific) equipped with a 3 cm x 100 um trapping column and a 60 cm x 75 um analytical column both packed with 5 um Reprosil C8 particles with 120 angstrom pores (Dr. Maisch GmbH).  Peptides were loaded onto the trapping column using 20 uL of solvent A (H20 containing 0.1 % formic acid) at a flow rate 2 uL/min.  Reversed phase separation over the analytical column was performed by applying a linear gradient from 98 % solvent A and 2 % solvent B (acetonitrile containing 0.1 % formic acid) to 60 % solvent A and 40 % solvent B over 120 minutes at a flow rate of 300 nL/min.  Eluting peptides were ionized by electrospray ionization by applying a voltage of 2.2 kV to a laser pulled tip at the tip of the analytical column.  Mass spectrometry was performed using Q-Exactive Plus mass spectrometer (Thermo Scientific) operated using a data dependent analysis method with the following settings: a high resolution MS1 scan from 400-2000 m/z at resolving power of 70,000 @ 200 m/z, automatic gain control (AGC) target of 106 ions, maximum ion time of 100 ms. Up to the 20 most abundant ions in the MS1 were selected for MS2 with an isolation window of 1.6 m/z and a normalized collision energy of 35.  MS2 scans covered the range for 200-2000 m/z at a resolving power of 35,000 with an AGC target of 50,000 ions and a maximum ion time of 50 ms.  Ions with one, six or greater or undetermined charge states were excluded from MS2. Ions selected for MS2 were dynamically excluded for 30 s.

### Data Protocol
Thermo *.raw files were converted into Mascot Generic Format (.mgf) using MSConvert in Proteowizard.  A non-redundant protein database containing 166,858 proteins (including forward and reverse sequences) was created using predicted proteins from the sequenced genomes of C. clementina, C. sinensis, and CLas, proteins from other citrus species from Genbank, and predicted lemon proteins from the cufflinks assembly of the RNAseq reads.  Mascot Daemon 2.3.2 (Matrix Science, Boston, MA) was used to search .mgf files against the citrus protein database.  MS/MS search parameters included fixed modifications (cysteine: carbamidomethyl), variable modifications (asparagine, glutamine: deamidated; methionine: oxidation), and maximum one missed cleavage.  Files with the *.dat extension were exported from Mascot and loaded into Scaffold Q+ 4.6.1 (version 4.6.1, Proteome Software, Portland, OR) and used to calculate normalized spectral counts for each protein from each sample.  Scaffold protein and peptide thresholds were set at 95%, with a minimum peptide number of two per protein – this resulted in a protein false discovery rate (FDR) of 0.3% and a peptide FDR of 0.0%.

### Publication Abstract
Presymptomatic detection of citrus trees infected with <i>Candidatus Liberibacter asiaticus</i> (<i>C</i>Las), the bacterial pathogen associated with Huanglongbing (HLB; citrus greening disease), is critical to controlling the spread of the disease. To test whether infected citrus trees produce systemic signals that may be used for indirect disease detection, lemon (<i>Citrus limon</i>) plants were graft-inoculated with either <i>C</i>Las-infected or control (<i>C</i>Las-) budwood, and leaf samples were longitudinally collected over 46 weeks and analyzed for plant changes associated with <i>C</i>Las infection. RNA, protein, and metabolite samples extracted from leaves were analyzed using RNA-Seq, mass spectrometry, and <sup>1</sup>H NMR spectroscopy, respectively. Significant differences in specific transcripts, proteins, and metabolites were observed between <i>C</i>Las-infected and control plants as early as 2 weeks post graft (wpg). The most dramatic differences between the transcriptome and proteome of <i>C</i>Las-infected and control plants were observed at 10 wpg, including coordinated increases in transcripts and proteins of citrus orthologs of known plant defense genes. This integrated approach to quantifying plant molecular changes in leaves of <i>C</i>Las-infected plants supports the development of diagnostic technology for presymptomatic or early disease detection as part of efforts to control the spread of HLB into uninfected citrus groves.

### Keywords
Citrus, Citrus greening, Huanglongbing, 'candidatus liberibacter asiaticus'

### Affiliations
USDA-ARS
USDA ARS; Boyce Thompson Institute; Cornell University

### Submitter
John Ramsey

### Lab Head
Dr Michelle Cilia
USDA ARS; Boyce Thompson Institute; Cornell University


