### Accession
PXD001023

### Title
Global protein oxidation profiling suggests efficient mitochondrial proteome homeostasis during ageing

### Description
The free radical theory of ageing is based on the idea that reactive oxygen species (ROS) may lead to the accumulation of age-related protein oxidation. Since the majority of cellular ROS is generated at the respiratory electron transport chain, this study focuses on the mitochondrial proteome of the ageing model Podospora anserina as target for ROS-induced damage. To ensure the detection of even low abundant modified peptides, separation by long gradient nLC-ESI-MS/MS and an appropriate statistical workflow for iTRAQ quantification was developed. Artificial protein oxidation was minimized by establishing gel-free sample preparation in the presence of reducing and iron-chelating agents. This first large scale, oxidative modification-centric study for P. anserina allowed the comprehensive quantification of 22 different oxidative amino acid modifications, and notably the quantitative comparison of oxidised and non-oxidised protein species. In total 2341 proteins were quantified. For 746 both protein species (unmodified and oxidatively modified) were detected and the modification sites determined. The data revealed that methionine residues are preferably oxidised. Further prominent identified modifications in decreasing order of occurrence were carbonylation as well as formation of N-formylkynurenine and pyrrolidinone. Interestingly, for the majority of proteins a positive correlation of changes in protein amount and oxidative damage were noticed, and a general decrease in protein amounts at late age. However, it was discovered that few proteins changed in oxidative damage in accordance with former reports. Our data suggest that P. anserina is efficiently capable to counteract ROS-induced protein damage during ageing as long as protein de-novo synthesis is functioning, ultimately leading to an overall constant relationship between damaged und undamaged protein species. These findings contradict a massive increase in protein oxidation during ageing and rather suggest a protein damage homeostasis mechanism even at late age.

### Sample Protocol
The mitochondrial fractions were processed according to the in-filter protein digestion (FASP II) procedure described by (Wiśniewski et al. 2009) with minor modifications. Since we focus on the mitochondrial proteome a solubilisation step is inevitable for high protein coverage. 50 µg of crude mitochondria were denatured and solubilized in SDT-lysis buffer by sonication for 3 min and subsequent shaking at RT for 30 min. Thereupon samples were mixed with 8 M urea and prepared using the FASP II protocol. All solutions for FASP II included additionally 1 mM DTT and 1 mM EDTA to avoid artificial oxidation during sample processing. In addition, triethylammonium bicarbonate (TEAB) was used as a tertiary amine buffer instead of ammonium bicarbonate. After centrifugation the resulting peptides were acidified with 50 % (v/v) ACN in 0.5 % (v/v) TFA and dried using a SpeedVac. Digested and carbamidomethylated samples were labeled with iTRAQ 4plex tags according to the manufacturer’s protocol. Instead of the manufacturer’s protocol, peptide samples were labeled with half the amount of iTRAQ reagent after tryptic digestion for multiplexing all age stages from one biological replicate. Briefly, each iTRAQ reagent was dissolved in 70 µL ethanol and the aliquots of iTRAQ 114, 115, 116, and 117 were combined with the peptide mixtures of mitochondria from four different age stages (6, 9, 13 and 16 days), respectively. However, for each experiment 35 µl of reagent solution was used for 50 µg of sample, which was resuspended in 25 µl iTRAQ dissolution buffer (0.5 M TEAB). After 1.5 h of incubation at RT, labeling reaction was stopped with 10 mM glycine. The differently-labeled samples of all 4 age stages from one individual were pooled and dried again in a SpeedVac. Prior to nLC-MS analysis all samples were desalted by solid-phase extraction using Spec C18AR pipette tips according to the manufacturer’s protocol. All desalted samples were resuspended in 2 % ACN in 0.1 % FA by sonication for 10 min prior to one -dimensional nLC-ESI-MS/MS analysis. Measurements were performed on a LTQ-Orbitrap Velos mass spectrometer (Thermo Fisher Scientific Inc., Waltham, MA, USA) coupled to a nanoACQUITY gradient UPLC pump system combined with an autosampler (all Waters, Milford, MA, USA). The nanoACQUITY UPLC system was equipped with a reversed phase UPLC HSS T3 column (1.8 µm, 75 μm x 250 mm, Waters, Milford, MA, USA) and a PicoTip Emitter (Silica TipTM, 10 µm i.d ,New Objective, Woburn, MA, USA.) form) as a nanospray source. 4 μl of the sample were loaded directly onto the analytical column using the nanoACQUITY autosampler with 99% buffer A (0.1% FA) and 1 % buffer B (0.1 % FA in ACN) for 60 min. Peptides were eluted by a multiple step gradient of buffer A and B at a flow rate of 300 nL/min in 270 min. Consequently the concentration of buffer B was increased to 3 % within 20 min, followed by a linear gradient to 30 % buffer B in 225 min (60-80 min: 3 % buffer B; 80-200 min: 13 % buffer B; 200-260 min: 20 % buffer B; 260-305 min: 30 % buffer B). To elute all peptides from the column, the concentration of buffer B was raised to 99 % in 20 min and kept constant for 5 min before the column was re-equilibrated at 1 % buffer B within 40 min. The column oven was set to 45 °C and the heated desolvation capillary was set to 275 °C. Data-dependent acquisition on the LTQ Orbitrap Velos was operated via instrument method files of Xcalibur (Rev. 2.1.0) in positive ion mode at a spray voltage of 1.3 - 1.8 kV. The full MS scan was performed in the Orbitrap in a range of 400-1400 m/z at a resolution of 60.000 using polysiloxane (m/z 445.120024) for internal lock mass calibration (Olsen et al. 2005). The ten most intense precursors per cycle were isolated and fragmented consecutively with CID and HCD. For peptide identification, CID was performed with relative collision energy of 35 %, an isolation width of 1.5 and an activation time of 10 ms before analysis at normal scan rate and mass range in the linear ion trap. For reporter ion quantification, HCD fragmentation spectra were acquired with normalized collision energy of 65 %, an isolation width of 1.2 and an activation time of 0.1 ms at a resolution of 7500. Dynamic exclusion was enabled with a repeat count of four and a 90 s exclusion duration window. Unassigned charge states, singly and more than triply charged ions were rejected from MS/MS. For each biological replicate three technical replicates were performed with following MS-settings in Xcalibur: 1. Top10, 2. Bottom10, and 3. Top10 with exclusion list which contained both the retention times and the m/z of previously identified peptides to be excluded for MS/MS. Furthermore, Top10 method meant that each of the ten most intense peaks in a full scan were fragmented in the order of highest to lowest intensity, whilst fragmentation order was from lowest to highest in the Bottom 10 method.

### Data Protocol
MS data search was performed against a P. anserina protein database the SEQUEST algorithm embedded in ProteomeDiscoverer 1.3.0.339 (Thermo Electron © 2008-2011). All acquired raw-files from three technical replicates of one biological replicate were combined in one data search, for which following search parameters were applied: (i) Trypsin as enzyme, (ii) a maximum of two missed cleavages, (iii) precursor ion mass tolerance of 5 ppm, (iv) fragment ions mass tolerance of 1 Da, (v) Carbamidomethylation of Cys and (vi) iTRAQ4plex(N-term) as fixed as well as (vii) iTRAQ4plex(K,Y) as variable modifications. The S/N threshold of Peak Filters in the Orbitrap was set to 3. Of note, database searches for suspected protein modifications are limited in the ProteomeDiscoverer 1.3.0.339 to four. Therefore database searches had to be sequentially conducted considering different oxidative amino acid modifications to allow detection of all possible PTMs. Protein quantification is performed with the Reporter Ions Quantifier tool embedded in the ProteomeDiscoverer software. Settings were kept at the default values: (i) integration window tolerance of 20 ppm, (ii) integration method of most confident centroid, (iii) mass analyzer is the Orbitrap with (iv) MS2 order and (v) HCD as activation type. Reporter based quantification is normalized by the protein ratio median. The factor normalizes all peptide ratios by the median protein ratio. Additionally, to minimize unwanted quantification of co-isolated peptides the allowed relative isolation interference was set to < 20 % of precursor signal intensity. For the determination of the false discovery rate (FDR) a decoy database search was performed with the percolator validation in Proteome Discoverer. The q-value is the minimal false discovery rate at which the identification is considered correct and was set to 1 %. For data analysis the mass spec format-(msf)-files were filtered with peptide confidence “high” and one peptide per protein. To compare all mitochondrial proteomes from different ages to each other we calculated the following iTRAQ protein ratios for each biological replicate: (i) 114/115 (6 /9 days), (ii) 114/116 (6/13 days), (iii) 114/117 (6/16 days). The iTRAQ protein ratios were calculated as the median over all distinct peptide ratios belonging to a protein. Consequently we obtained three different iTRAQ ratios per protein (i, ii, iii) for each biological replicate, resulting in (3 x 6) iTRAQ protein ratios in total, also here referred to as replicate ratios. Next, to calculate a final protein ratio for each of the three age stages, all individual biological replicate ratios were combined into one iTRAQ protein ratio per age ratio (i, ii, iii) using again the median. We computed a quality score q for each protein ratio, enabling us to evaluate the reliability of the computed final protein ratio. The quality score is a value within the interval [0;1] where 1 indicates a highly reliable experimental measurement and 0 an unreliable protein ratio due to strong experimental variation. The quality score bases upon three assumptions: (1) Number of detected replicates, (2) Quotient of up- to down-regulated replicates, (3) The variance of the replicates ratios. Finally we combined all values to one quality score using the geometric mean. We assume a reliable protein ratio if q ≥ 0.6 holds. To decide whether a protein (modified- or unmodified) was up- or down-regulated on a certain day in respect to the reference day 6 we had to choose a reliable threshold. For this purpose we calculated the quartiles for all logarithmic ratios (to the base of 2) and defined the 25% and the 75% quartile as our lower- and upper-threshold, respectively. I.e. 50% of all ratios lay within this interquartile range and all others beyond these borders (see supplementary table 27) and are stronger down- or up-regulated than 50% of all other ratios. To test whether abundance changes of oxidized and un-oxidized protein species are similar, we compared these profiles and decided whether they are equal by means of the two-sided two-sample Kolmogorow-Smirnoff-Test, which tests for differences between two one-dimensional distributions. The n replicate-ratios for profile 1 and the m replicate-ratios for profile 2 were used to compute the empirical distribution functions F1,n and F2,m. Subsequently, based on the D-statistic a p-value has been computed and a significance threshold of 0.05 was applied, meaning that if the p-value was less than or equal to 0.05 the null-hypothesis was rejected and we assumed that two profiles were significantly different.

### Publication Abstract
The free radical theory of aging is based on the idea that reactive oxygen species (ROS) may lead to the accumulation of age-related protein oxidation. Because themajority of cellular ROS is generated at the respiratory electron transport chain, this study focuses on the mitochondrial proteome of the aging model Podospora anserina as target for ROS-induced damage. To ensure the detection of even low abundant modified peptides, separation by long gradient nLC-ESI-MS/MS and an appropriate statistical workflow for iTRAQ quantification was developed. Artificial protein oxidation was minimized by establishing gel-free sample preparation in the presence of reducing and iron-chelating agents. This first large scale, oxidative modification-centric study for P. anserina allowed the comprehensive quantification of 22 different oxidative amino acid modifications, and notably the quantitative comparison of oxidized and nonoxidized protein species. In total 2341 proteins were quantified. For 746 both protein species (unmodified and oxidatively modified) were detected and the modification sites determined. The data revealed that methionine residues are preferably oxidized. Further prominent identified modifications in decreasing order of occurrence were carbonylation as well as formation of N-formylkynurenine and pyrrolidinone. Interestingly, for the majority of proteins a positive correlation of changes in protein amount and oxidative damage were noticed, and a general decrease in protein amounts at late age. However, it was discovered that few proteins changed in oxidative damage in accordance with former reports. Our data suggest that P. anserina is efficiently capable to counteract ROS-induced protein damage during aging as long as protein de novo synthesis is functioning, ultimately leading to an overall constant relationship between damaged and undamaged protein species. These findings contradict a massive increase in protein oxidation during aging and rather suggest a protein damage homeostasis mechanism even at late age.

### Keywords
Podospora anserina, Oxidative ptms, Quantitative mass spectrometry, Itraq, Mitochondrial ageing

### Affiliations
Plant Biochemistry, PG Poetsch, Ruhr-University Bochum, Germany
Ruhr UniversitÃ¤t Bochum

### Submitter
Ansgar Poetsch

### Lab Head
Dr Carina Ramallo Guevara
Plant Biochemistry, PG Poetsch, Ruhr-University Bochum, Germany


