### Accession
PXD012423

### Title
Proteome-wide solubility and thermal stability profiling reveals distinct regulatory roles for ATP - TPP datasets

### Description
Nucleotide triphosphates (NTPs) regulate numerous biochemical processes in cells as (co-)substrates, allosteric modulators, biosynthetic precursors, and signaling molecules1-3. Apart from its roles as energy source fueling cellular biochemistry, adenosine triphosphate (ATP), the most abundant NTP in cells, has been reported to affect macromolecular assemblies, such as protein complexes4,5 and membrane-less organelles6-8. Moreover, both ATP and guanosine triphosphate (GTP) have recently been shown to dissolve protein aggregates9. However, system-wide studies to characterize NTP- interactions under conditions approximating the native cellular environment are lacking, which limits our perspective of the diverse physiological roles of NTPs. Here, we have mapped and quantified proteome-wide NTP-interactions by assessing thermal stability and solubility of proteins using mechanically disrupted cells. Our results reveal diverse biological roles of ATP depending on its concentration. We found that ATP specifically interacts with proteins that utilize it as substrate or allosteric modulator at doses lower than 500 Î¼M, while it affects protein-protein interactions of protein complexes at mildly higher concentrations (between 1-2 mM). At high concentrations (> 2 mM), ATP modulates the solubility state of a quarter of the insoluble proteome, consisting of positively charged, intrinsically disordered, nucleic acid binding proteins, which are part of membrane-less organelles. The extent of solubilization depends on the localization of proteins to different membrane-less organelles. Furthermore, we uncover that ATP regulates protein-DNA interactions of the Barrier to autointegration factor (BANF1). Our data provides the first quantitative proteome-wide map of ATP affecting protein structure and protein complex stability and solubility, providing unique clues on its role in protein phase transitions.

### Sample Protocol
Samples were processed with a modified SP3 protocol, digested using trypsin and labeled with TMT. Shotgun proteomics using a Q-Exactive and Fusion Lumos Orbitrap was then performed.

### Data Protocol
TMT quantification was used, by using the isobarquant python package (Franken et al., Nat Protoc (2015)), and Mascot 2.4 was used for protein identification. Search parameters: trypsin, missed cleavages 3, peptide tolerance 10ppm, 0.02Da for MS/MS tolerance. Fixed modifications were carbamidomethyl on cysteines and TMT10plex on lysine. Variable modifications included acetylation on protein N-terminus, oxidation of methionine and TMT10plex on peptide N-termini.

### Publication Abstract
Adenosine triphosphate (ATP) plays fundamental roles in cellular biochemistry and was recently discovered to function as a biological hydrotrope. Here, we use mass spectrometry to interrogate ATP-mediated regulation of protein thermal stability and protein solubility on a proteome-wide scale. Thermal proteome profiling reveals high affinity interactions of ATP as a substrate and as an allosteric modulator that has widespread influence on protein complexes and their stability. Further, we develop a strategy for proteome-wide solubility profiling, and discover ATP-dependent solubilization of at least 25% of the insoluble proteome. ATP increases the solubility of positively charged, intrinsically disordered proteins, and their susceptibility for solubilization varies depending on their localization to different membrane-less organelles. Moreover, a few proteins, exhibit an ATP-dependent decrease in solubility, likely reflecting polymer formation. Our data provides a proteome-wide, quantitative insight into how ATP influences protein structure and solubility across the spectrum of physiologically relevant concentrations.

### Keywords
Tpp, Jurkat, Atp

### Affiliations
EMBL Heidelberg
European Molecular Biology Laboratory (EMBL), Genome Biology Unit

### Submitter
Nils Kurzawa

### Lab Head
Dr Mikhail M Savitski
European Molecular Biology Laboratory (EMBL), Genome Biology Unit


