### Accession
PXD005857

### Title
pSILAC mass spectrometry reveals ZFP91 as novel IMiD dependent substrate of the CRL4CRBN ligase - PART 2

### Description
Thalidomide and its derivatives lenalidomide and pomalidomide (IMiDs) are effective treatments of hematologic malignancies. It was shown that IMiDs impart gain of function properties to the CUL4-RBX1-DDB1-CRBN (CRL4CRBN) ubiquitin ligase that enable binding, ubiquitination and degradation of key therapeutic targets such as IKFZ1, IKZF3 and CSNK1A1. While these substrates have been implicated as efficacy targets in multiple myeloma (MM) and 5q deletion associated myelodysplastic syndrome (del(5q)-MDS), other targets likely exist. Using a pulse-chase SILAC mass spectrometry-based proteomics approach, here we demonstrate that lenalidomide induces the ubiquitination and degradation of ZFP91. We establish that ZFP91 is a bona fide IMiD dependent CRL4CRBN substrate and further show that ZFP91 harbors a zinc finger (ZnF) motif, related to the IKZF1/3 ZnF, critical for IMiD dependent CRBN binding. These findings demonstrate that single time point pulse-chase SILAC mass spectrometry-based proteomics (pSILAC-MS) is a sensitive approach for target identification of small molecules inducing selective protein

### Sample Protocol
HEK293T cells were treated with either 30 uM lenalidomide or DMSO control and harvested after 6, 10 (the .raw files was accidently labeled as T8 but represents a 10 hour time point), or 16 hours. Whole cell lysates were prepared and proteins fractionated using SDS-PAGE, reduced, alkylated and digested following standard procedures. Derived peptides were analyzed on a Orbitrap VELOS mass spectrometer.

### Data Protocol
MaxQuant (version 1.5.0.30) was used for RAW file processing and controlling peptide and protein level false-discovery rates, assembling proteins from peptides, and protein quantification from peptides. Fragment ion spectra were searched against a human Uniprot database (downloaded on January 29 2015) and common contaminant proteins (included in the MaxQuant software). Carbamidomethylation (Cys) was set as a fixed modification, deamidation (Gln, Asn) and oxidation (Met) were selected as variable modifications. SILAC amino acids used for quantification were Lys8 and Arg10. The MaxQuant ProteinGroups.txt output file was imported into the R framework for all subsequent analyses.

### Publication Abstract
Thalidomide and its derivatives lenalidomide and pomalidomide (IMiDs) are effective treatments of haematologic malignancies. It was shown that IMiDs impart gain-of-function properties to the CUL4-RBX1-DDB1-CRBN (CRL4<sup>CRBN</sup>) ubiquitin ligase that enable binding, ubiquitination and degradation of key therapeutic targets such as IKZF1, IKZF3 and CSNK1A1. While these substrates have been implicated as efficacy targets in multiple myeloma (MM) and 5q deletion associated myelodysplastic syndrome (del(5q)-MDS), other targets likely exist. Using a pulse-chase SILAC mass spectrometry-based proteomics approach, we demonstrate that lenalidomide induces the ubiquitination and degradation of ZFP91. We establish ZFP91 as a bona fide IMiD-dependent CRL4<sup>CRBN</sup> substrate and further show that ZFP91 harbours a zinc finger (ZnF) motif, related to the IKZF1/3 ZnF, critical for IMiD-dependent CRBN binding. These findings demonstrate that single time point pulse-chase SILAC mass spectrometry-based proteomics (pSILAC MS) is a sensitive approach for target identification of small molecules inducing selective protein degradation.

### Keywords
Imids, Lenalidomide, Zfp91, Silac

### Affiliations
Dana-Farber Cancer Institute
BCMP, Harvard Medical School, Boston, USA  Cancer Biology, Dana-Farber Cancer Institute, Boston, USA

### Submitter
Eric Fischer

### Lab Head
Dr Eric Sebastian Fischer
BCMP, Harvard Medical School, Boston, USA  Cancer Biology, Dana-Farber Cancer Institute, Boston, USA


