### Accession
PXD025589

### Title
Integrative proteogenomics for differential expression and splicing variation in a DM1 mouse model

### Description
Here, we performed a large-scale coordinated transcriptomic and proteomic analysis to characterize a DM1 mouse model (HSALR) in comparison to wild-type. Our integrative proteogenomics approach comprised gene- and splicing-level assessments for mRNA and protein. It recapitulated many known instances of aberrant mRNA splicing in DM1 and identified new ones. It enabled the design and targeting of splicing-specific peptides and confirmed the translation of known instances of aberrantly spliced disease-related genes (e.g. Atp2a1, Bin1, Ryr1), complemented by novel findings (e.g. Ywhae, Flnc, Svil). Comparative analysis of large-scale mRNA and protein expression data showed remarkable agreement of differential patterns between disease and wild-type on both the gene and especially the splicing level.

### Sample Protocol
Aliquots of 10 mg frozen muscle tissue powder were resuspended in 250 Î¼L of 50 mM Tris buffer pH 8.5 containing 1% of sodium dodecyl sulfate. Tissue lysis was performed by tip sonication for 15 sec at 25% amplitude on ice for 3-5 times, with 1 min breaks to allow sample cooling. Protein were then precipitated using chloroform-methanol extraction. After precipitation, protein pellets were dried in a stream of N2 and dissolved in 200 Î¼l 50 mM Tris buffer 8.5 pH with 8M Urea using a sonication bath. Protein extracts were reduced in 5 mM dithiothreitol at 56Â°C for 40 min and alkylated in 15 mM iodoacetamide at room temperature for 30 min in the dark. Finally, to quench IAA excess and prevent overalkylation, DTT was added to the solution to 15mM concentration. For digestion, samples were diluted 3 times with 50mM Tris buffer and digested overnight at 37Â°C using a Trypsin/Lys-C mixture (Promega) at a ratio 1:100 w/w. An additional digestion step was performed for 3 hours with the protease mixture at the ratio 1:200 w/w. After the reaction was stopped, the sample was centrifuged at 10 000Ã—g for 5 min, followed by supernatant cleanup on a C18 Sep-Pak cartridge. Finally, the samples were dried. For TMT-labeling, 100 Î¼g of peptides from each sample were processed according to manufacturer instructions (TMT 10plex). After labelling, all samples were mixed and desalted on C18 Sep-Pak cartridges, followed by dissolving samples in 10 mM ammonium formate buffer for high-pH reverse phase fractionation. An Agilent1200 Series HPLC system with a Waters XBridge column (C18 3.5 Âµm, 150 x 2.1 mm) was used to fractionate samples into 72 fractions during a one-hour linear gradient from 0% to 50% solvent B (ACN with 20mM ammonium formate pH 10). The 72 fractions were concatenated into 24 fractions, dried, and dissolved to 0.3 Î¼g/Î¼L concentration (2% ACN with 0.1% FA) for further analysis.â€¯  LC-MS/MS/MS analysis was performed using an Orbitrap Fusion Lumos mass spectrometer coupled with a Thermo Easy-nLC system. From each high pH fraction, 5 Î¼L of peptides were injected and separated on a heated (55 Â°C) Aurora nanoZero C18 column (1.6 Î¼m,75 Î¼m i.d. x 250 mm, 120 Ã…). Mobile phases were as follows: (A) 0.1% FA in water; (B) 90% ACN, 0.1% FA in water. Peptides were eluted using a linear gradient from 2% B to 6% B for 50 min, followed by a linear gradient to 15% B for 50 min and to 30% B for 68 min at a flow rate of 400 nL/min. The column was washed at 95% B for 20 minutes and equilibrated to the start concentration of mobile phase B. Mass spectrometry measurements were performed using data-dependent acquisition (DDA) mode (Top Speed, 3s/cycle), using an MS3 method for accurate TMT quantification. MS1 scans were measured in the Orbitrap mass-analyzer with following settings: mass range from 350 m/z to 1,500 m/z, resolving power of 120K, maximum injection time (IT) set to 50 ms, automatic gain control (AGC) for MS1 was 2.0e5, dynamic exclusion set to 40 s. Precursor ions were isolated using a 0.7 Th window, followed by fragmentation using CID at NCE of 35%. Fragment ions were measured in the ion trap with maximum IT of 50 ms and AGC value of 1.0e4. After MS/MS, 5 notches were isolated with 2 Th window for HCD at NCE 55%. The MS3 scans were measured in the Orbitrap mass-analyzer in the mass range from 100 m/z to 500 m/z with resolving power of 50K, maximum IT 50 ms and AGC value 1.0e5. â€¯For PRM analysis, samples were dissolved in 2% ACN with 0.1% FA, spiked with heavy labelled peptides. 1.5 Î¼g of peptide mixture was injected for analysis. The amount of spiked heavy labelled peptides was adjusted to obtain similar ion current (the final amount varied in the range of 10 â€“ 100 fmol per sample). The analysis was performed using an Orbitrap QExactive HF-X mass spectrometer coupled with a Thermo Easy-nLC system. For each sample, the analysis was conducted twice using two different chromatographic conditions. Under the first condition, a 90 minute gradient (from 3 to 40 % solvent B) and a two-column set up with a C18 PepMap100 trap-column (5Î¼m, 300Î¼m i.d.x 5 mm, 100Ã…) and a 25 cm analytical column (EASY-Spray PepMap C18, 2 Î¼m, 75Î¼m i.d, 100Ã…, Thermo Scientific) were used. Under the second conditions, a 150 min gradient with a longer 50 cm analytical column (EASY-Spray PepMap C18, 2 Î¼m, 75Î¼m i.d, 100Ã…) was employed as a second condition. Mass spectrometry measurements were performed using targeted PRM mode with the following settings: mass range from 340 m/z to 1,200 m/z, resolving power of 60K, maximum injection time (IT) set to 50 ms, automatic gain control (AGC) for MS1 was 1.0e6, dynamic exclusion set to 50 s. Precursor ions were isolated using a 0.7 Th window followed by their fragmentation using high collision dissociation (HCD) at normalized collision energy (NCE) of 27%. Fragment ions were measured with maximum IT of 100 ms, AGC value of 2.0e5, and resolution 60K.

### Data Protocol
For TMT-based experiments, database searching was performed using Thermo Proteome Discoverer (version 2.1.0.81) with Sequest HT search engine and Percolator post-processing against a RefSeq mouse database (RefSeq release 91, 77337 entries, 58517 non-redundant protein sequences). Search parameters were as follows: 10 ppm for precursor mass tolerance; 0.6 Da for fragment mass tolerance; maximum 2 missed cleavage sites; fixed carbamidomethylated cysteine, TMT labelled N-terminus and lysine; potential methionine oxidation. The identified peptides were filtered to 1.0% false discovery rate. The threshold for precursor contamination was set to 50%, and the minimal average reporter S/N to 10; all missing values and shared peptides were excluded from quantitation analysis. The TMT-intensities were normalized inside each sample to the total intensity as well as to the average value across the wild type group. The differential abundance analysis was done using moderated t-statistics as implemented in the R package applying limma test with Benjamini-Hochberg correction for multiple testing.â€¯â€¯The search against Canonical Uniprot database (25021 entries, 23445 non-redundant protein sequence) was done with the same parameters.    The Skyline software was used for PRM analyses. The library was built based on Mascot 2.4 identifications of LC-MS/MS runs with the excess heavy labelled peptides spiked into a mixture of samples. The fragments from the second to the penultimate ion were used for quantification to increase specificity. The identifications with â€œrdotpâ€� and â€œdotpâ€� indexes lower than 0.7 and 0.9, respectively, were excluded. All matches with an â€œrdotpâ€� index lower than 0.9 were manually checked for fragment co-elution and chromatographic peak shape. Only fragments represented in all samples were considered, others were excluded manually.

### Publication Abstract
Dysregulated mRNA splicing is involved in the pathogenesis of many diseases including cancer, neurodegenerative diseases, and muscular dystrophies such as myotonic dystrophy type 1 (DM1). Comprehensive assessment of dysregulated splicing on the transcriptome and proteome level has been methodologically challenging, and thus investigations have often been targeting only few genes. Here, we performed a large-scale coordinated transcriptomic and proteomic analysis to characterize a DM1 mouse model (HSA<sup>LR</sup>) in comparison to wild type. Our integrative proteogenomics approach comprised gene- and splicing-level assessments for mRNAs and proteins. It recapitulated many known instances of aberrant mRNA splicing in DM1 and identified new ones. It enabled the design and targeting of splicing-specific peptides and confirmed the translation of known instances of aberrantly spliced disease-related genes (e.g., Atp2a1, Bin1, Ryr1), complemented by novel findings (Flnc and Ywhae). Comparative analysis of large-scale mRNA and protein expression data showed quantitative agreement of differentially expressed genes and splicing patterns between disease and wild type. We hence propose this work as a suitable blueprint for a robust and scalable integrative proteogenomic strategy geared toward advancing our understanding of splicing-based disorders. With such a strategy, splicing-based biomarker candidates emerge as an attractive and accessible option, as they can be efficiently asserted on the mRNA and protein level in coordinated fashion.

### Keywords
Alternative splicing, Proteogenomics, Transcriptomics, Proteomics, Mouse model, Myotonic dystrophy type 1 (dm1)

### Affiliations
Novartis Institute for BioMedical Research, Scientific Data Analysis group
NIBR

### Submitter
Elizaveta Solovyeva

### Lab Head
Dr Sebastian Hoersch
Novartis Institute for BioMedical Research, Scientific Data Analysis group


