### Accession
PXD000801

### Title
Effects of Campylobacter jejuni CJIE1 prophage on protein expression experiment 4

### Description
Temperate bacteriophages (prophages) have recently been demonstrated in Campylobacter jejuni. However, what they do there is largely unknown. In the series of studies that are the subject of these submissions we have investigated the relative expression levels of proteins in C. jejuni isolates that differ in the presence or absence of the CJIE1 prophage. At the time of the initial investigations whole genome sequence data were not available for the isolates used, though DNA microarray data indicated that the isolates were very closely related. The overall project was carried out through four separate experiments.  Previous work in the scientific literature indicated that growth on medium lacking blood but containing sodium deoxycholate induced the expression of at least some proteins associated with virulence and provided data thought to be of relevance to the virulence of the bacterium. The second set of experiments (experiment 2)  compared protein expression in 4-plex iTRAQ experiments using two isolates grown on medium with and without 0.1% sodium deoxycholate to further explore the effects of the CJIE1 prophage, if any, on protein expression. Results from that experiment indicated that there was an unexpectedly large bile response, and that there were differences in expression from the first experiment that may be attributable to the presence of blood in the culture medium in the first experiment, compared with the use of medium without blood in the second experiment. Experiment 3 was therefore designed to compare protein expression of the CJIE1-carrying isolate 00-2425 after growth on medium containing blood, without blood, and without blood but with bile salts. In experiment 4 we grew isolate 00-2426, which lacks prophage CJIE1, on the same four media. The goals were to determine if the differences in protein expression (detection) were similar to those in isolate 00-2425 and if there were any differences that could be attributed to the presence or absence of the CJIE1 prophage.

### Sample Protocol
Isolates were grown on Mueller-Hinton agar containing 10% blood, bacteria were suspended in water, boiled, and total crude proteins were prepared by bead-beating and vortexing bacterial suspensions. Standardized aliquots of 100 micrograms of protein were solubilized in solubilization buffer (4% SDS, 50 mM HEPES buffer pH 8.3, 100 mM dithiothreitol) by boiling for 5 minutes. All subsequent steps were done in Nanosep 10K cartridges (spin columns). The solubilization buffer was replaced with Urea Exchange buffer (UEB buffer, 8 M urea in 50 mM HEPES buffer pH 8.3) . The volume of the retentate was reduced to a minimum and the proteins were alkylated by adding 50 mM iodoacetamide in UEB, mixing, and incubating at room temperature for 20 min. The iodoacetamide was diluted out by repeated exchanges with UEB followed by 50 mM HEPES buffer pH 8.3 without urea. After reducing the volume of the retentate to a minimum, DNA was removed by adding Benzonase (20 U/ul Benzonase in 42 mM HEPES, pH 8.3 containing 2 mM magnesium chloride), mixing, and incubating 30 min at room temperature. After washing with 50 mM HEPES, pH 8.3 the peptides were digested in the spin column with Trypsin Gold (mass spectrometry grade) 50 mM HEPES, pH 8.3 overnight at 37 C. The peptides were then collected, dried, and labelled with iTRAQ reagents. The relative amounts of iTRAQ labelled peptides in each labelling reaction was checked by nano LC-MSMS, and the volumes of peptides added to the 4-plex mixture were normalized so that there were equal amounts of total protein carrying each iTRAQ label. iTRAQ labels were associated with peptides from samples as follows: growth on Mueller-Hinton agar (MH) containing 10% sheep blood, 114; MH only, 115; MH + 0.1% sodium deoxycholate, 116; MH + 2.5% Oxgall, 117.  Each 4-plex protein mixture containing iTRAQ-labelled tryptic peptide fragments was fractionated by high pH C18-reversed phase liquid chromatography on a a micro-flow Agilent 1100/1200 series system using a Waters Xbridge C18 guard column (10 mm long, 2.1 mm internal diameter, 3.5 um particles) and a Waters X-bridge C18 analytical column (10 cm long, 2.1 mm internal diameter, 3.5 um particles). A gradient of buffer A (20 mM ammonium formate, pH 10) and buffer B (20 mM ammonium formate, pH 10, and 90% acetonitrile) was used for the separation, beginning with 3% buffer B for 10 minutes followed by: 8 - 11% buffer B from 10 - 17 min, 11 - 60% buffer B from 17 - 75 min, 95% buffer B from 75 - 80 min. Fractions were collected across the peptide elution profile (17 - 75 min), pooleed to reduce the number of fractions by approximately half, dried, and suspended in buffer A for nano LC/MS/MS. Each fraction thus obtained was analysed separately using a nano-flow Easy nLC II instrument connected in-line to an LTQ Orbitrap Velos mass spectrometer with a nanoelectrospray ion source. Each peptide fraction (5 ul) was loaded onto a C18 reversed-phase trap column (2 cm long, 100 um inner diameter, 5 um particles) with 100% buffer A (2% acetonitrile, 0.1% formic acid) at 4 ul/min for a total volume of 30 ul, then separated on a C18 reversed-phase column (10 cm long, 75 um inner diameter, 3 um particles). Both columns were packed in-house with ReproSil-Pur C18-AQ resin. Peptides were eluted using a linear gradient of 0-30% buffer B (98% acetonitrile, 0.1% formic acid) over 120 min at a constant flow rate of 300 nl/min. The total LC/MS/MS run time was 160 min, including the loading, linear gradient, column wash at 95% buffer B, and the equilibration.

### Data Protocol
Data were acquired using a data-dependent method, dynamically choosing the top 10 abundant precursor ions from each survey scan for isolation in the LTQ and fragmentation by HCD at 45% normalized collision energy. The survey scans were acquired in the Orbitrap over m/z 300-1700 with a target resolution of 60,000 at m/z 400, and the subsequent fragment ion scans were acquired in the Orbitrap over a dynamic m/z range with a target resolution of 7500 at m/z 400. The lower threshold for selecting a precursor ion for fragmentation was 1000 counts. Dynamic exclusion was enabled using a list size of 500 features, a m/z tolerance of 15 ppm, a repeat count of 1, a repeat duration of 30 s, and an exclusion duration of 15 s, with early expiration disabled. All spectra were processed using Mascot Distiller v2.3.2 (Matrix Science), and database searching was done with Mascot v2.3 (Matrix Science). Searches were performed against an in-house built, non-redundant database consisting of NCBIâ€™s Genome database of bacteria [ftp://ftp.ncbi.nlm.nih.gov/genomes/Bacteria/] and prophage sequences generated in-house]. The decoy database option was selected and the following parameters were used: carbamidomethylation (C) and iTRAQ (K and N-terminus) as fixed modifications, oxidations (M) as a variable modification, fragment ion mass tolerance of 0.5 Da, parent ion tolerance of 10 ppm, and trypsin enzyme with up to 1 missed cleavage. Mascot search results were imported into Scaffold Q+ v3.4 (Proteome Software) and filtered using 80% confidence for peptides, 99% confidences for proteins, and at least 2 peptides per protein.  iTRAQ labels were associated with peptides from samples as follows: growth on Mueller-Hinton agar (MH) containing 10% sheep blood, 114; MH only, 115; MH + 0.1% sodium deoxycholate, 116; MH + 2.5% Oxgall, 117.

### Publication Abstract
None

### Keywords
; cjie1 prophage, Lc-msms, Campylobacter jejun, Itraq

### Affiliations
National Microbiology Laboratory, Public Health Agency of Canada Proteomics and Mass Spectrometry Core Facility
National Microbiology Laboratory, Public Health Agency of Canada

### Submitter
Clifford Clark

### Lab Head
Dr Garrett R. Westmacott
National Microbiology Laboratory, Public Health Agency of Canada Proteomics and Mass Spectrometry Core Facility


