### Accession
PXD024111

### Title
Characterization of a novel Lbr Mutation

### Description
We have characterised the product of a novel N terminal deletion of Lbr gene.

### Sample Protocol
Please enter the sample processing protocol here. Please enter the sample processing protocol here. Please enter the sample processing protocol here. Bands were cut from the gel and subjected to in-gel digestion with trypsin (PMID: 25278616). Peptides were extracted from the gel pieces by sonication for 15 minutes, followed by centrifugation and supernatant collection. A solution of 50:50 water: acetonitrile, 1 % formic acid (2 x the volume of the gel pieces) was added for a second extraction and the samples were again sonicated for 15 minutes, centrifuged and the supernatant pooled with the first extract.  The pooled supernatants processed using speed vacuum centrifugation.  The samples were dissolved in 10 Ãƒâ€šÃ‚ÂµL of reconstitution buffer (96:4 water: acetonitrile, 1% formic acid and analyzed by LC-MS/MS.   Peptides were separated using the nanoAcquity UPLC system (Waters) fitted with a trapping devise (nanoAcquity Symmetry C18, 5Ãƒâ€šÃ‚Âµm, 180 Ãƒâ€šÃ‚Âµm x 20 mm) and an analytical column (nanoAcquity BEH C18, 1.7Ãƒâ€šÃ‚Âµm, 75Ãƒâ€šÃ‚Âµm x 200mm).  The outlet of the analytical column was coupled directly to an LTQ Orbitrap Velos (Thermo Fisher Scientific) using the Proxeon nanospray source. Solvent A was water, 0.1 % formic acid and solvent B was acetonitrile, 0.1 % formic acid.  The samples were loaded with a constant flow of solvent A at 5 Ãƒâ€šÃ‚ÂµL/min onto the trapping column.  Trapping time was 6 minutes.  Peptides were eluted via the analytical column a constant flow of 0.3 Ãƒâ€šÃ‚ÂµL/min. During the elution step, the percentage of solvent B increased in a linear fashion. The peptides were introduced into the mass spectrometer (Orbitrap Velos, Thermo) via a Pico-Tip Emitter 360 ÃƒÅ½Ã‚Â¼m OD x 20 ÃƒÅ½Ã‚Â¼m ID; 10 ÃƒÅ½Ã‚Â¼m tip (New Objective) and a spray voltage of 2.2 kV was applied. The capillary temperature was set at 300 Ãƒâ€šÃ‚Â°C. Full scan MS spectra with mass range 300-1700 m/z were acquired in profile mode in the FT with a resolution of 30000. The filling time was set at maximum of 500 ms with limitation of 1.0 x 106 ions. The most intense ions (up to 15) from the full scan MS were selected for sequencing in the LTQ. Normalized collision energy of 40% was used, and the fragmentation was performed after accumulation of 3.0 x104 ions or after filling time of 100 ms for each precursor ion (whichever occurred first). MS/MS data was acquired in centroid mode. Only multiply charged (2+, 3+, 4+) precursor ions were selected for MS/MS. The dynamic exclusion list was restricted to 500 entries with maximum retention period of 30 s and relative mass window of 10 ppm. In order to improve the mass accuracy, a lock mass correction using the ion (m/z 445.12003) was applied.

### Data Protocol
The raw mass spectrometry data was processed with MaxQuant (v1.6.3.4)46 and searched against the Uniprot mus musculus (UP000000589; 55462entries; latest update: 20200321). The data was searched with the following modifications: Carbamidomethyl (C) (fixed modification), Acetyl (N-term) and Oxidation (M) (variable modifications).  The mass error tolerance for the full scan MS spectra was set to 20 ppm and for the MS/MS spectra to 0.5 Da. A maximum of 2 missed cleavages for trypsin/P was allowed. For protein identification a minimum of 2 unique peptides with a peptide length of at least seven amino acids and a false discovery rate below 0.01 were required on the peptide and protein level. Quantification was performed using iBAQ values47 calculated as the sum of the intensities of the identified peptides and divided by the number of observable peptides of a protein.

### Publication Abstract
Mutations in the gene encoding Lamin B receptor (LBR), a nuclear-membrane protein with sterol reductase activity, have been linked to rare human disorders. Phenotypes range from a benign blood disorder, such as Pelger-Huet anomaly (PHA), affecting the morphology and chromatin organization&#xa0;of white blood cells, to embryonic lethality as for Greenberg dysplasia (GRBGD). Existing PHA mouse models do not fully recapitulate the human phenotypes, hindering efforts to understand the molecular etiology of this disorder. Here we show, using CRISPR/Cas-9 gene editing technology, that a 236bp N-terminal deletion in the mouse Lbr gene, generating a protein missing&#xa0;the N-terminal domains of LBR, presents a superior model of human PHA. Further, we address recent reports of a link between Lbr and defects in X chromosome inactivation (XCI) and show that our mouse mutant displays minor X chromosome inactivation defects that do not lead to any overt phenotypes in vivo. We suggest that our N-terminal deletion model provides a valuable pre-clinical tool to the research community and will aid in further understanding the etiology of PHA and the diverse functions of LBR.

### Keywords
Lamin b receptor

### Affiliations
EMBL
Blizard Institute, Centre for Genomics and Child Health, Barts and The London School of Medicine and Dentistry, Queen Mary University of London, London, E1 2AT, UK

### Submitter
Frank Stein

### Lab Head
Dr Andrea Cerase
Blizard Institute, Centre for Genomics and Child Health, Barts and The London School of Medicine and Dentistry, Queen Mary University of London, London, E1 2AT, UK


