### Accession
PXD001165

### Title
Hendra virus infection of bat (Pteropus alecto) and human cells

### Description
SILAC labeled human kidney cells (293 cells) or bat kidney cells (PakiT03cells)were infected with Hendra virus for 8 or 24 hours and compared to uninfected control cells. Protein identification and quantitation relied on a combination of Uniprot lists of proteins and Proteomics Informed by Transcriptomics (PIT) analysis whereby RNA extracted from the same samples was deep sequenced and the sequencing data was used to construct mRNA from which possible ORFS were inferred and used as a search space by MaxQuant.

### Sample Protocol
Cells were lysed and boiled in SDS-PAGE loading buffer and run on a gel, the gel was sliced into 20 slices and each on subjected to in gel digestion with trypsin before LC-MS/MS using an Orbitrap Velos

### Data Protocol
Maxquant was used and the search space was a mixture of uniprot lists for the species concerned (and the uniprot Hendra virus list) alongside our PIT analysis as outlined in Nature Methods (Evans et al. 2012, Nature Methods 9 1207-1211). In short the RNAseq reads are de novo assembled back into mRNA from which we use an all frames translation to derive a possible search space. In addition we map reads to the target host genome and use cufflinks to derive a mRNA list from the mapping data.

### Publication Abstract
None

### Keywords
Human pteropus alecto hendra virus orbitrap velos

### Affiliations
University of Bristol
School of Cellular and Molecular Medicine, University Walk, Bristol  BS8 1TD

### Submitter
David Matthews

### Lab Head
Dr David A Matthews
School of Cellular and Molecular Medicine, University Walk, Bristol  BS8 1TD


