### Accession
PXD014739

### Title
Proteomic profiling of histone acetylation pattern in Arabidopsis

### Description
The levels of acetylated and non-acetylated histone peptide forms were quantified in A. thaliana plants with loss of function of HDA6, and plants exposed to HDAs inhibitors (trichostatin and sodium butyrate)during germination .

### Sample Protocol
Plant material - Arabidopsis thaliana ecotype Columbia 0 â€“ wild type, hda6-6 mutant line, hda6-7 mutant line.  Histone extraction from plant material into sulfuric acid. Chemical derivatization of histone proteins. Protein digestion in the filter unit. Peptide propionylation in solution. LC-MS/MS analysis - Orbitrap Elite hybrid spectrometer.

### Data Protocol
The RAW mass spectrometric data files were analyzed using Proteome Discoverer software (Thermo Fisher Scientific; version 1.4) with in-house Mascot search engine (Matrixscience, version 2.6) to compare acquired spectra with entries in the UniProtKB Arabidopsis thaliana protein database (version 2017_11; 27567 protein sequences; downloaded  from ftp://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/reference_proteomes/Eukaryota/UP000006548_3702.fasta.gz), cRAP contaminant database (downloaded from http://www.thegpm.org/crap/) and in-house histone database (version 2017_02; 71 protein sequences). Mass tolerances for peptides and MS/MS fragments were 7 ppm and 0.025 Da, respectively. Semi-Arg-C for enzyme specificity allowing up to two missed cleavages was set. For searches against cRAP and UniProtKB Arabidopsis thaliana databases the variable modification settings were oxidation (M), deamidation (N, Q), acetylation (K, protein N-term) and propionylation (K, N-term), while for histone database searches they were acetylation (K, protein N-term), methylation (K, R), dimethylation (K), trimethylation (K), phosphorylation (S, T), and propionylation (K, N-term, S, T, Y). Only peptides with statistically significant peptide score (p < 0.01) were considered. The peak area corresponding to each precursor ion was calculated from the extracted ion chromatograms (XICs) using the Precursor Ions Area Detector node. Selected histone peptide identifications were manually verified and quantified from the peak areas derived from the XICs using Skyline 3.6 software, including identification alignment across the raw files based on retention time and m/z.

### Publication Abstract
A high degree of developmental plasticity enables plants to adapt to continuous, often unfavorable and unpredictable changes in their environment. At the molecular level, adaptive advantages for plants are primarily provided by epigenetic machinery including DNA methylation, histone modifications, and the activity of noncoding RNA molecules. Using a mass spectrometry-based proteomic approach, we examined the levels of acetylated histone peptide forms in Arabidopsis plants with a loss of function of histone deacetylase 6 (HDA6), and in plants germinated in the presence of HDA inhibitors trichostatin A (TSA) and sodium butyrate (NaB). Our analyses revealed particular lysine sites at histone sequences targeted by the HDA6 enzyme, and by TSA- and NaB-sensitive HDAs. Compared with plants exposed to drugs, more dramatic changes in the overall profiles of histone post-translational modifications were identified in <i>hda6</i> mutants. However, loss of HDA6 was not sufficient by itself to induce hyperacetylation to the maximum degree, implying complementary activities of other HDAs. In contrast to <i>hda6</i> mutants that did not exhibit any obvious phenotypic defects, the phenotypes of seedlings exposed to HDA inhibitors were markedly affected, showing that the effect of these drugs on early plant development is not limited to the modulation of histone acetylation levels.

### Keywords
Arabidopsis thaliana, Histone deacetylase inhibitor, Post-translational modifications, Epigenetics, Histone, Mass spectrometry

### Affiliations
CEITEC-MU, Masaryk University, Brno
Mendel Centre for Plant Genomics and Proteomics, Central European Institute of Technology, Masaryk University, Brno, Czech Republic

### Submitter
Gabriela Lochmanova

### Lab Head
Dr Zbynek Zdrahal
Mendel Centre for Plant Genomics and Proteomics, Central European Institute of Technology, Masaryk University, Brno, Czech Republic


