### Accession
PXD011618

### Title
ER chemical proteomics with ER-localizable reactive molecules

### Description
The endoplasmic reticulum (ER) is an organelle that performs a variety of essential cellular functions via interactions with other organelles. Despite its important role, chemical tools for profiling the composition and dynamics of ER proteins remain very limited because of the labile nature of these proteins. Here, we developed ER-localizable reactive molecules (called ERMs) as tools for ER-focused chemical proteomics. ERMs can spontaneously localize in the ER of living cells and selectively label ER-associated proteins with a combined affinity and imaging tag, enabling tag-mediated ER protein enrichment and identification with liquid chromatography tandem-mass spectrometry (LC-MS/MS). The ERM probes could be used simultaneously with the nucleus- and mitochondria-localizable reactive molecules previously developed by our group, which enabled orthogonal organellar chemoproteomics in a single biological sample. Moreover, quantitative analysis of the dynamic changes in ER-associated proteins in response to tunicamycin-induced ER stress was performed by combining ER-specific labeling with SILAC (stable isotope labeling by amino acids in cell culture)-based quantitative MS technology. Our results demonstrated that ERM-based chemical proteomics provides a powerful tool for labeling and profiling ER-related proteins in living cells.

### Sample Protocol
Sample processing HeLa cells (5.0 × 106 cells) were cultured on 10 cm dishes under a humidified atmosphere of 5% CO2 in air at 37 °C for 72 h in DMEM. The cells were washed twice with serum-free DMEM, and incubated in serum free DMEM containing ERM (0.1 µM) for 1 h. The cells were washed twice with chilled PBS and lysed by sonication in 500 µL of 4% SDS lysis buffer (25 mM Tris, 150 mM NaCl, 1% Nonidet P-40, 4% Sodium dodecyl sulfate, 1% sodium deoxycholate, pH 7.6) containing 1% protease inhibitor cocktail set III (Novagen) at 4 °C. Proteins were precipitated with chilled acetone, and the pellets were solubilized by sonication in the 4% SDS lysis buffer. The solubilized protein solution was diluted to adjust a SDS concentration of 1% with wash buffer (25 mM Tris, 150 mM NaCl, 1% Nonidet P-40, 0.1% Sodium dodecyl sulfate, 1% sodium deoxycholate, pH 7.6). Concentration was measured with BCA assay kit (Thermo) and 2.5 mg of proteins were used for immunoprecipitation. 25 µL of nProtein A sepharose 4 Fast Flow (GE Healthcare) was incubated with 5.0 µL of anti-rhodol antiserum for 2 h at 4 °C with continuous rotation. The beads were washed 5 times with the 1% SDS lysis buffer followed by addition of 2.5 mg of proteins and further rotation at 4 °C for overnight. After incubation, the beads were washed 5 times with the 1% SDS lysis buffer (1 mL) and 5 times with PBS (-) (1 mL), and then boiled in 2 × SDS–PAGE loading buffer (125 mM Tris, 10% glycerol, 0.01% bromophenol blue, 4% SDS, pH 6.8) containing 200 mM DTT for 5 min.  Preparation of nanoLC-MS/MS samples The samples were loaded to SDS-PAGE gel, and resolved 1.0 cm by the running gel and fixed with 45% methanol/water containing 5% acetic acid for 20 min. The fixed gel was washed with 50% methanol/water and pure water, and manually cut into 3 gel slices without gel staining. The excised gels were subjected to in-gel digestion using Trypsin/Lys-C Mix (Promega, 200 ng/gel slice) according to manufacture’s instruction. The eluted peptide solution from the gel was freeze dried and redissolved in 20 µL of 2% acetonitrile/water containing 0.1% TFA. The solutions (15 µL) were subjected to nanoLC-MS/MS analyses.   Identification of proteins with nanoLC-MS/MS NanoLC–MS/MS analyses were performed on a Q-Exactive mass spectrometer (Thermo Fisher Scientific) and an Ultimate 3000 nanoLC pump (AMR) as described previously (Nat. Commun. 2018, 9, 1.). Samples were automatically injected using PAL system (CTC analytics, Zwingen, Switzerland) into a peptide L-trap column OSD (5 µm) attached to an injector valve for desalinating and concentrating peptides. After washing the trap with MS-grade water containing 0.1% TFA and 2% acetonitrile, the peptides were loaded into a nano HPLC capillary column (C18 packed with the gel particle size of 3 µm, 0.1 x 125 mm, Nikkyo Technos, Tokyo Japan) by switching the valve. The injection volume was 5 or 10 μL and the flow rate was 500 nL/min. The mobile phases consisted of (A) 0.5% acetic acid and (B) 0.5% acetic acid and 80% acetonitrile. A two-step linear gradient of 5–45% B in 60 min, 45–95% B in 1 min, 95% B for 20 min was employed. Spray voltages of 2,000 V were applied. The mass scan ranges were m/z 350–1,800, and top ten precursor ions were selected in each MS scan for subsequent MS/MS scans. The normalized collision energy was set to be 30.

### Data Protocol
Data processing protocol The raw MS data files were analyzed by Proteome Discoverer 2.2 (Thermo Fisher Scientific) to create peak lists based on the recorded fragmentation spectra. Peptides and proteins were identified by means of automated database searching using Sequest HT (Thermo Fisher Scientific) against UniprotKB/Swiss-Prot release 2018-04-14 with a precursor mass tolerance of 10 p.p.m., a fragment ion mass tolerance of 0.02 Da, and trypsin specificity that allows for up to three or two missed cleavages. Cysteine carbamidomethylation was set as fixed modifications. Methionine oxidation was allowed as a variable modification. A reversed decoy database search was conducted to set false discovery rates (FDRs) of less than 1% both at peptide and protein levels. Three independent experiments were performed, in which proteins detected at least twice were included in the datasets as identified proteins.  Sample information ・100 nM ERM-ph-TE labeled HeLa cell (2017-59_e1, e2, e3) ・100 nM ERM-ph-BC labeled HeLa cell (2017-59_e4, e5, e6) ・100 nM ERM-ph-SF labeled HeLa cell (2017-59_e7, e8, 2017-59_(1)) ・DMSO contorol (2017-59_e9, e10, e11)

### Publication Abstract
None

### Keywords
Er, Hela cell, Chemical proteomics

### Affiliations
Department of Synthetic Chemistry & Biological Chemistry, Graduate School of Engineering, Kyoto Univ. Katsura, Kyoto 615-8510, JAPAN
Kyoto Univ.

### Submitter
Alma Fujisawa

### Lab Head
Dr Itaru Hamachi
Department of Synthetic Chemistry & Biological Chemistry, Graduate School of Engineering, Kyoto Univ. Katsura, Kyoto 615-8510, JAPAN


