### Accession
PXD017610

### Title
A multi-omics approach to liver diseases: integration of Single Nuclei Transcriptomics with Proteomics and Hi-Cap Bulk Data in human liver

### Description
Abstract The liver is the largest solid organ and a primary metabolic hub. In recent years, intact cell nuclei were used to perform single-nuclei RNA-seq (snRNA-seq) for tissues difficult to dissociate and for flash-frozen archived tissue samples to discover unknown and rare cell sub-populations. In this study, we performed snRNA-seq of a liver sample to identify sub-populations of cells based on nuclear transcriptomics. In 4,282 single nuclei we detected on average 1,377 active genes and we identified seven major cell types. We integrated data from 94,286 distal interactions (p<0.05) for 7,682 promoters from a targeted chromosome conformation capture technique (HiCap) and mass spectrometry (MS) proteomics for the same liver sample. We observed a reasonable correlation between proteomics and in silico bulk snRNA-seq (r=0.47) using tissue-independent gene-specific protein abundancy estimation factors. We specifically looked at genes of medical importance. The DPYD gene is involved in the pharmacogenetics of fluoropyrimidines toxicity and some of its variants are analyzed for clinical purposes. We identified a new putative polymorphic regulatory element, which may contribute to variation in toxicity. Hepatocellular carcinoma (HCC) is the most common type of primary liver cancer and we investigated all known risk genes. We identified a complex regulatory landscape for the SLC2A2 gene with 16 candidate enhancers. Three of them harbor somatic motif breaking and other mutations in HCC in the Pan Cancer Analysis of Whole Genomes dataset and are candidates to contribute to malignancy. Our results highlight the potential of a multi-omics approach in the study of human diseases.

### Sample Protocol
sample was boiled in lysis buffer containing 10% Trifluoroethannol, 50 mM Tris pH 8 and 5 mM DTT and 20 mM Chloroacetoamide. Sample was sonicated for 15 min on Bioruptor. Proteins were digested with LysC and Trypsin for overnight at 37 degrees. The peptides were purified on SDP-RPS material. The purified peptides were measured using LC-MS instrumentation consisting of an EASY-nLC 1200 system coupled to a nano-electrospray ion source and a Q Exactive HF Orbitrap (all Thermo Fischer Scientific). Purified peptides were separated on 50 cm HPLC columns (in house packed into the tip with Reprosil-Pur C18-AQ 1.9 Âµm resin (Dr.Maisch GmbH). Purified peptides were loaded in buffer A (0.1% formic acid) and eluted with a linear 100-min gradient of 3-30% of buffer B (0.1% formic acid, 80% (v/v) acetonitrile). The column temperature was kept at 60 degrees by a peltier element containing an in-house-developed oven. MS data was acquired with Top15 data-dependent MS/MS scan method (topN method). The target value for full scan MS spectra was set to 3e6 in the 300-1,650 m/z range with a maximum injection time of 25 ms and a resolution of 60,000 at 200 m/z. Fragmentation of precursor ions was performed by high-energy C-trap dissociation (HCD) with a normalized collision energy of 27 eV. MS/MS scans were performed at a resolution of 15,000 at m/z 200 with target ion values of 1e5 and maximum injection time of 25 ms.

### Data Protocol
MS raw file was analyzed using the MaxQuant software (Cox and Mann, 2008) and peptide list were searched against the human UniProt FASTA database with the Andromeda search engine (Cox et al., 2011). For the search, a contamination database was included, cysteine modification was set as a fixed modification and N-terminal acetylation and methionine oxidation was set as variable modification. False discovery rate (FDR) was 0.01 for both the protein and peptide level with a minimum length of 7 amino acids and the FDR was determined by searching reverse database. Enzyme specificity was set as C-terminal to arginine and lysine using trypsin protease, and maximum two missed cleavage were allowed for search. The peptides were identified with an initial precursor mass deviation of up to 7 ppm and fragment mass deviation of 20 ppm. In case of identified peptides that were shared between two or more proteins, these were combined and reported in protein groups.

### Publication Abstract
The liver is the largest solid organ and a primary metabolic hub. In recent years, intact cell nuclei were used to perform single-nuclei RNA-seq (snRNA-seq) for tissues difficult to dissociate and for flash-frozen archived tissue samples to discover unknown and rare cell subpopulations. In this study, we performed snRNA-seq of a liver sample to identify subpopulations of cells based on nuclear transcriptomics. In 4282 single nuclei, we detected, on average, 1377 active genes and we identified seven major cell types. We integrated data from 94,286 distal interactions (<i>p</i>&#x2009;&lt;&#x2009;0.05) for 7682 promoters from a targeted chromosome conformation capture technique (HiCap) and mass spectrometry proteomics for the same liver sample. We observed a reasonable correlation between proteomics and <i>in silico</i> bulk snRNA-seq (<i>r</i>&#x2009;=&#x2009;0.47) using tissue-independent gene-specific protein abundancy estimation factors. We specifically looked at genes of medical importance. The <i>DPYD</i> gene is involved in the pharmacogenetics of fluoropyrimidine toxicity and some of its variants are analyzed for clinical purposes. We identified a new putative polymorphic regulatory element, which may contribute to variation in toxicity. Hepatocellular carcinoma (HCC) is the most common type of primary liver cancer and we investigated all known risk genes. We identified a complex regulatory landscape for the <i>SLC2A2</i> gene with 16 candidate enhancers. Three of them harbor somatic motif breaking and other mutations in HCC in the Pan Cancer Analysis of Whole Genomes dataset and are candidates to contribute to malignancy. Our results highlight the potential of a multi-omics approach in the study of human diseases.

### Keywords
Human liver single nuclei lcms

### Affiliations
Science for Life Laboratory, Department of Immunology, Genetics and Pathology, Uppsala University, 751 23 Uppsala, Sweden
Copenhagen University

### Submitter
atul shahaji deshmukh

### Lab Head
Dr Claes Wadelius
Science for Life Laboratory, Department of Immunology, Genetics and Pathology, Uppsala University, 751 23 Uppsala, Sweden


