### Accession
PXD032052

### Title
Identification of substrates of palmitoyl protein thioesterase 1 highlights roles of depalmitoylation in disulfide bond formation and synaptic function

### Description
Loss-of-function mutations in the depalmitoylating enzyme palmitoyl protein thioesterase 1 (PPT1) cause Neuronal Ceroid Lipofuscinosis (NCL), a devastating neurodegenerative disease. The substrates of PPT1 are largely undescribed, posing a limitation on molecular dissection of disease mechanisms and therapeutic development. Here, we provide a resource identifying >100 novel PPT1 substrates. We utilized Acyl Resin-Assisted Capture and mass spectrometry to identify proteins with increased in vivo palmitoylation in PPT1 knockout mouse brains. We then validated putative substrates through direct depalmitoylation with recombinant PPT1. This stringent screen elucidated diverse PPT1 substrates at the synapse, including channels and transporters, G-protein-associated molecules, endo/exocytic components, synaptic adhesion molecules, and mitochondrial proteins. Cysteine depalmitoylation sites in transmembrane PPT1 substrates frequently participate in disulfide bonds in the mature protein. We confirmed that depalmitoylation plays a role in disulfide bond formation in a tertiary screen analyzing post-translational modifications. Collectively, these data highlight the role of PPT1 in mediating synapse functions, implicate molecular pathways in the etiology of NCL and other neurodegenerative diseases, and advance our basic understanding of the purpose of depalmitoylation.

### Sample Protocol
Sample Processing: Global whole brain, synaptosomal, and Acyl Resin-Assisted Capture palmitoyloeme were analyzed by mass spectrometric analysis. Synaptosomes of the denoted genotypes were prepared as described previously (1).  1. Westphal CH & Chandra SS (2013) Monomeric synucleins generate membrane curvature. J Biol Chem 288(3):1829-1840.

### Data Protocol
Data processing: LFQ of protein samples was performed on a mass spectrometer (Thermo Scientific Orbitrap Fusion or Thermo Scientific Q-Exactive Plus) connected to a UPLC system (Waters nanoACQUITY) equipped with a Waters Symmetry® C18 180 μm × 20 mm trap column and a 1.7-μm, 75 μm × 250 mm nano ACQUITY UPLC column (35°C). Additional details on UPLC and mass spectrometer conditions can be found in Charkoftaki, et.al., 2019. The LC-MS/MS data was processed using Progenesis QI Proteomics software (Nonlinear Dynamics, version 4.0), and protein identification was carried out using the Mascot search algorithm (Matrix Science, Boston MA). The MS data was normalized by comparing the abundance of the spike in Pierce Retention Time Calibration mixture among all the samples as well as the sum of squares of spectral count for each replicate. Relative protein-level fold changes were calculated from the sum of all unique, normalized peptide ion abundances for each protein on each run. Only proteins identified by at least 2 unique peptides were considered. Mass spectrometry data P-values were calculated using ANOVA. We did not use FDR correction for multiple comparisons in our screen to maximize our hits. P-values <0.05 will be considered significant. In addition, we used a 1.5-fold change as a cut-off for the mass spectrometric data. All protein names are listed using UniProt nomenclature. Additional analyses were performed using Qlucore Omics Explorer. Pathway analyses were conducted with three distinct software – Ingenuity Pathway Analysis (IPA) available from Qiagen, Gene Ontology Resource, and STRING analysis.  2. Henderson MX, et al. (2016) Neuronal ceroid lipofuscinosis with DNAJC5/CSPα mutation has PPT1 pathology and exhibit aberrant protein palmitoylation. Acta Neuropathologica 131:621-637.

### Publication Abstract
None

### Keywords
Synaptic adhesion molecules, Synapse, Synaptic vesicle endocytosis, Neurodegeneration, Palmitoylation, Disulfide bond, Palmitoyl protein thioesterase 1, Na+/k+ transporting atpase, Zdhhc, Neuronal ceroid lipofuscinosis

### Affiliations
Yale University
Yale University (Keck MS & Proteomics Resource)

### Submitter
TuKiet Lam

### Lab Head
Dr TuKiet Lam
Yale University (Keck MS & Proteomics Resource)


