### Accession
PXD044445

### Title
Remodeling of the human skeletal muscle proteome found after long-term endurance training but not after strength training

### Description
Proteomic characterization of skeletal muscle biopsies collected from the vastus lateralis of 44 male and female human subjects. Research subjects were divided in three different groups based on their individual exercise backgrounds and physical performance testing: 1) endurance trained (males; ME, n = 9 and females; FE, n = 9, with at least 15 years’ of regular training experience), 2) strength-trained males (MS, n = 9, with at least 15 years’ of regular training experience), and 3) age-matched healthy untrained controls (males, MC, n = 9 and females FC, n = 8, with a self-reported history of <2 exercise bouts per week over the past 15 years).

### Sample Protocol
Samples (please add details about your samples) were dissolved in 500 µl Lysis buffer (4% SDS, 50 mM HEPES pH 7,6, 1 mM DTT), heated to 95°C and homogenized using a Retsch homogenizer with metallic spheres. The samples were then sonicated and centrifuged. The total protein amount was estimated (Bio-Rad DC). Samples were then prepared for mass spectrometry analysis using  a modified version of the SP3 protein clean-up and a digestion protocol (Moggridge et al., 2018) (Hughes et al., 2014), where proteins were digested by LycC and trypsin (sequencing grade modified, Pierce). In brief, 200 µg protein from each sample was alkylated with 4 mM Chloroacetamide. Sera‐Mag SP3 bead mix (20 µl) was transferred into the protein sample together with 100% Acetonitrile to a final concentration of 70 %. The mix was incubated under rotation at room temperature for 18 min. The mix was placed on the magnetic rack and the supernatant was discarded, followed by two washes with 70 % ethanol and one with 100 % acetonitrile. The beads-protein mixture was reconstituted in 100 µl LysC buffer (0.5 M Urea, 50 mM HEPES pH: 7.6 and 1:50 enzyme (LysC) to protein ratio) and incubated overnight. Finally, trypsin was added in 1:50 enzyme to protein ratio in 100 µl 50 mM HEPES pH 7.6 and incubated overnight. The peptides were eluted from the mixture after placing the mixture on a magnetic rack, followed by peptide concentration measurement (Bio-Rad DC Assay). The samples were then pH adjusted using TEAB pH 8.5 (100 mM final conc.), 100 µg of peptides from each sample were labelled with isobaric TMT-tags (TMT10plex reagent) according to the manufacturer’s protocol (Thermo Scientific), and separated by immobilized pH gradient - isoelectric focusing (IPG-IEF) on 3–10 strips as described previously (Branca et al., 2014).  Of note, the labelling efficiency was determined by LC-MS/MS before pooling of the samples. For the sample clean-up step, a solid phase extraction (SPE strata-X-C, Phenomenex) was performed and purified samples were dried in a SpeedVac. An aliquot of approximately 10 µg was suspended in LC mobile phase A and 1 µg was injected on the LC-MS/MS system. Online LC-MS was performed as previously described (Branca et al 2014) using a Dionex UltiMate™ 3000 RSLCnano System coupled to a Q-Exactive-HF mass spectrometer (Thermo Scientific). Each of the 72 plate wells was dissolved in 20ul solvent A and 10ul were injected. Samples were trapped on a C18 guard-desalting column (Acclaim PepMap 100, 75μm x 2 cm, nanoViper, C18, 5 µm, 100Å), and separated on a 50 cm long C18 column (Easy spray PepMap RSLC, C18, 2 μm, 100Å, 75 μm x 50 cm). The nano capillary solvent A was 95% water, 5%DMSO, 0.1% formic acid; and solvent B was 5% water, 5% DMSO, 95% acetonitrile, 0.1% formic acid. At a constant flow of 0.25 μl min−1, the curved gradient went from 6-8% B up to 40% B in each fraction in a dynamic range of gradient length, followed by a steep increase to 100% B in 5 min. FTMS master scans with 60,000 resolution (and mass range 300-1500 m/z) were followed by data-dependent MS/MS (30 000 resolution) on the top 5 ions using higher energy collision dissociation (HCD) at 30% normalized collision energy. Precursors were isolated with a 2 m/z window. Automatic gain control (AGC) targets were 1e6 for MS1 and 1e5 for MS2. Maximum injection times were 100 ms for MS1 and 100 ms for MS2. The entire duty cycle lasted ~2.5 s. Dynamic exclusion was used with 30 s duration. Precursors with unassigned charge state or charge state 1 were excluded. An underfill ratio of 1% was used.

### Data Protocol
Orbitrap raw MS/MS files were converted to mzML format using msConvert from the ProteoWizard tool suite (Holman et al., 2014). Spectra were then searched using MSGF+ (v10072) (Kim and Pevzner, 2014) and Percolator (v2.08) (Granholm et al., 2014), where search results from 8 subsequent fraction were grouped for Percolator target/decoy analysis. All searches were done against the human protein subset of ENSEMBL_101 homo sapiens protein database in the Galaxy platform (Boekel et al., 2015). MSGF+ settings included precursor mass tolerance of 10 ppm, fully-tryptic peptides, maximum peptide length of 50 amino acids and a maximum charge of 6. Fixed modifications were TMT-10plex on lysines and peptide N-termini, and carbamidomethylation on cysteine residues, a variable modification was used for oxidation on methionine residues. Quantification of TMT-10plex reporter ions was done using OpenMS project's IsobaricAnalyzer (v2.0) (Sturm et al., 2008). PSMs found at 1% FDR (false discovery rate) were used to infer gene identities.  Protein quantification by TMT10plex reporter ions was calculated using TMT PSM ratios to the entire sample set (all 10 TMT-channels) and normalized to the sample median. The median PSM TMT reporter ratio from peptides unique to a gene symbol was used for quantification. Protein false discovery rates were calculated using the picked-FDR method using gene symbols as protein groups and limited to 1% FDR (Savitski et al., 2015).

### Publication Abstract
Exercise training has tremendous systemic tissue-specific health benefits, but the molecular adaptations to long-term exercise training are not completely understood. We investigated the skeletal muscle proteome of highly endurance-trained, strength-trained, and untrained individuals and performed exercise- and sex-specific analyses. Of the 6,000+ proteins identified, &gt;650 were differentially expressed in endurance-trained individuals compared with controls. Strikingly, 92% of the shared proteins with higher expression in both the male and female endurance groups were known mitochondrial. In contrast to the findings in endurance-trained individuals, minimal differences were found in strength-trained individuals and between females and males. Lastly, a co-expression network and comparative literature analysis revealed key proteins and pathways related to the health benefits of exercise, which were primarily related to differences in mitochondrial proteins. This network is available as an interactive database resource where investigators can correlate clinical data with global gene and protein expression data for hypothesis generation.

### Keywords
Human subjects, Exercise physiology, Skeletal muscle proteomics, Resistance training, Endurance training, Skeletal muscle metabolism, Skeletal muscle

### Affiliations
Affiliation Dept. Oncology Pathology, Karolinska Institutet, and Scilifelab, Stockholm, Sweden
Karolinska Institutet

### Submitter
Georgios Mermelekas

### Lab Head
Dr Janne Lehtiö
Affiliation Dept. Oncology Pathology, Karolinska Institutet, and Scilifelab, Stockholm, Sweden


