### Accession
PXD006544

### Title
Interactome of Ca2+-bound secretagogin in INS-1E cells

### Description
By using His6-tagged recombinant human secretagogin in the presence of Ca2+ (100 µM), we precipitated putative interacting partners from INS-1E cells and identified their amino acid sequences by mass spectrometry. We detected canonical interacting proteins participating in vesicle-mediated transport, exocytosis and cytoskeletal organization. Besides, we captured proteins controlling protein folding and (de-)ubiquitination. Our data support that secretagogin modulates protein folding and degradation through protein-protein interactions and interacts with USP9X in β cells to control cell survival.

### Sample Protocol
Eluted proteins (17.5 µg) of His6- secretagogin (n = 3) and control pull-downs (n = 3) were digested with trypsin via filter-aided sample preparation (Wisniewski et al., 2009) with minor modifications (Aradska et al., 2015). Tryptic digests were desalted and concentrated on customized reversed-phase C18 stage tips (Rappsilber et al., 2007). Purified peptides were reconstituted in 5% formic acid (FA) and 2x 1 µg of each sample (technical replicates) were analyzed by liquid chromatography-tandem mass spectrometry (LCMS/MS). Peptides were separated on an Ultimate 3000 nanoRSLC system (ThermoFisher) equipped with a PM100-C18 precolumn (3 μm, 75 μm × 20 mm) and a PepMap-C18 analytical column (2 μm, 75 μm x 500 mm). Ten μl of each sample was loaded onto the trap column for 5 min using 0.1% FA in water at a flow rate of 5 μl/min. Afterwards, peptides were forward-flushed to and separated on the analytical column at a flow rate of 300 nl/min using a 55-min gradient ranging from 5 to 37.5% solvent B, followed by a 5-min gradient from 37.5 to 50% solvent B and finally, to 90% solvent B for 5 min before re-equilibration to 5% solvent B (solvent A: 0.1% FA in water; solvent B: 0.1% FA in 80% acetonitrile). Eluted peptides were in-line analyzed on a Thermo Q-Exactive Plus mass spectrometer (ThermoFisher) in positive ion mode. MS scans were performed in a range of m/z 380-1800 at a resolution of 70,000. In a data-dependent acquisition mode, the 20 most intense precursor ions were selected and fragmented via higher-energy collisional dissociation at 27% normalized collision energy with a fixed first mass of 100 m/z. Fragment ions were detected at a resolution of 17,500. Dynamic exclusion of selected peptides was enabled for 60 s and maximal accumulation times were set to 100 and 50 ms in MS and MSn modes, respectively. References Aradska J, Bulat T, Sialana FJ, Birner-Gruenberger R, Erich B, and Lubec G (2015) Gel-free mass spectrometry analysis of Drosophila melanogaster heads. Proteomics, 15, 3356-3360. Rappsilber J, Mann M, and Ishihama Y (2007) Protocol for micro-purification, enrichment, prefractionation and storage of peptides for proteomics using StageTips. Nat Protoc, 2, 1896-1906. Wisniewski JR, Zougman A, Nagaraj N, and Mann M (2009) Universal sample preparation method for proteome analysis. Nat Methods, 6, 359-362.

### Data Protocol
To describe the secretagogin-specific interactome, all MS raw data files were analyzed by the open source software MaxQuant 1.5.3.30 (Cox and Mann, 2008) as previously described (Smidak et al., 2016). Proteins were identified against the Swiss-Prot rat (Rattus norvegicus) reference proteome database (as of November 2015; 31,457 entries) permitting a mass tolerance of 5 ppm and 20 ppm for MS and MS2 spectra, respectively. Maximum two missed cleavages were allowed and minimum two peptide identifications per protein were required. Carbamidomethylation of cysteines was set as fixed modification while methionine oxidation and Nterminal protein acetylation were chosen as variable modifications. For both peptide and protein identification, the false discovery rate (FDR) was set to 0.01. MaxQuant results were further processed using the Perseus statistical package (version 1.5.4.1) to identify interactors from label-free data (LFQ intensities). Proteins were filtered for reversed sequences, contaminants, and if they were only identified by site. Intensity values were log2-transformed and proteins were further filtered for having LFQ values in minimum 3 samples at least in one group (either in the target or in the control). Zerointensities were input and replaced by normal distribution and secretagogin pull-downs were compared to control samples via two-sided Student’s t-test with p < 0.05 (applying permutation-based FDR for 18 truncation). Proteins with a minimum 2-fold increase of their LFQ intensities in the target pull-down were considered as specific for secretagogin if they were identified by at least two unique peptides. References Cox J and Mann M (2008) MaxQuant enables high peptide identification rates, individualized p.p.b.- range mass accuracies and proteome-wide protein quantification. Nat Biotechnol, 26, 1367-1372. Smidak R, Mayer RL, Bileck A, Gerner C, Mechtcheriakova D, Stork O, Lubec G, and Li L (2016) Quantitative proteomics reveals protein kinases and phosphatases in the individual phases of contextual fear conditioning in the C57BL/6J mouse. Behav Brain Res, 303, 208-217.

### Publication Abstract
None

### Keywords
His6-tagged recombinant secretagogin, Ins-1e cell line, Pull down, Protein-protein interactions

### Affiliations
Dept. of Molecular Neurosciences, Center for Brain Research, Medical University of Vienna, Vienna, Austria
Department of Molecular Neurosciences, Center for Brain Research, Medical University of Vienna, Vienna, Austria

### Submitter
Edit Szodorai

### Lab Head
Dr Univ. Prof. Dr. Tibor Harkany
Department of Molecular Neurosciences, Center for Brain Research, Medical University of Vienna, Vienna, Austria


