### Accession
PXD017673

### Title
MealTime-MS: A Machine Learning-Guided Real-Time Mass Spectrometry Analysis for Proteomics

### Description
Mass spectrometry-based proteomics technologies are the prime methods for the high-throughput identification of proteins expressed in complex biological samples. Nevertheless, mass spectrometry’s technical limitations still hinder its ability to identify low abundance proteins in complex samples. Characterizing such proteins is essential to provide a comprehensive understanding of the biological processes taking place in a sample. Still today, a large part of the mass spectrometry-based proteomics performed use a data-dependent approach that favors the acquisition of mass spectra and detection of proteins of higher abundance. Combined to the fact that the computational identification of proteins from mass spectrometry data is typically performed after mass spectrometry data acquisition, large numbers of mass spectra are redundantly collected from the same abundant proteins and little to no mass spectra are acquired for proteins of lower abundance. To address this problem, we propose a novel supervised learning algorithm that identifies proteins in real-time as mass spectrometry data is acquired and prevents the further data acquisition related to confidently identified proteins to improve the identification sensitivity of low abundance proteins. We show in real-time simulations of a previously performed mass spectrometry analysis of a HEK293 cell lysates that our approach can identify 92.1% of the proteins using 66.2% of the MS2 spectra acquired in the experiment. We also demonstrate that our approach is fast enough for real-time mass spectrometry analysis, is flexible and that it outperforms previously proposed methods. Our method efficient usage of mass spectrometry resources will provide a more comprehensive characterization of proteomes in complex samples.

### Sample Protocol
Cell culture and protein extraction HEK293 cells were grown to 80% confluence in 15cm dishes and harvested in RIPA buffer (25 mM Tris-HCl (pH 7.6), 150 mM NaCl, 1% NP-40, 1% sodium deoxycholate, 0.5% SDS) after two washes with PBS. Cells were sonicated for 1 min with 20 seconds pulse using 30% power on Sonic Dismembrator 500 w/ Branson 1020 Sonicator (Fisher Scientific) to increase protein recovery. Proteins were precipitated to remove detergent by adding 5X volume of cold acetone overnight followed by two washes with cold acetone as well.   Sample digestion 8M Urea in 50mM ABC was used to reconstitute proteins. Reduction and alkylation were done by adding DTT to a final concentration of 10 mM at 56°C for 30min followed by 20mM IAA at room temperature. The solution was then diluted 5 times by adding 50mM ABC. Trypsin was added to achieve a protein-enzyme ratio of 50:1. Digestion was performed at 37°C overnight, with continuous shaking. Digested peptides were then desalted on Sep-Pak column (Waters) and dried down in SpeedVac (ThermoFisher Scientific, San Jose, CA). Dried peptides were reconstituted in 0.5% (v/v) FA, 2% ACN in water.  LC-MS/MS Analysis Eksigent 2D+ nanoLC system (Dublin, CA) was hooked up with a Q-Exactive mass spectrometer (Thermo Electron, Waltham, MA), equipped with a nano-electrospray interface operated in positive ion mode. The solvent system consists of buffer A of 0.1% FA in water, and buffer B of 0.1%FA in 80% acetonitrile. Reconstituted peptides were loaded on 200 μm I.D. × 20 mm fused silica pre-column packed in-house with 5μm ReproSil-Pur C18 beads (100 Å; Dr. Maisch GmbH, Ammerbuch, Germany)  at a flow rate of 4uL/min for 10min, and analyzed on 75 μm I.D. × 150 mm fused silica analytical column packed in-house with 1.9μm ReproSil-Pur C18 beads (100 Å; Dr. Maisch GmbH, Ammerbuch, Germany)  at a flow rate of 200 nL/min for either 120min or 240min to create to different dataset of the HEK293 cell lysate. Gradient elution was set from 5 to 35% buffer B(80% ACN). The spray voltage was set to 2.0 kV and the temperature of the heated capillary was 300 °C. The instrument method consisted of one full MS scan from 300 to 1800 m/z followed by the acquisition of data-dependent MS/MS scan of the 12 most intense ions, a dynamic exclusion repeat count of 1 in 30s, and an exclusion duration of 30s. The full mass was scanned in Orbitrap analyzer with R = 70,000 (defined at m/z 400) for MS1 and 17,500 for MS2. To improve the mass accuracy, all the measurements in the Orbitrap mass analyzer were performed with a real-time internal calibration by the lock mass of background ion 445.120025. The charge state rejection function was enabled, and charge states with unknown and single charge state were excluded for subsequent MS/MS analysis. All data were recorded with Xcalibur software (ThermoFisher Scientific, San Jose, CA).

### Data Protocol
Raw output from a previously completed mass spectrometry experiment was converted to mzML file format using msconvert from the Trans-Proteomics Pipeline. LC-MS/MS mzml files were searched with the Comet protein sequence database search algorithm. Comet search was performed against the UniProt Swiss-Prot human protein sequence database (downloaded on 2017-01-11). Database search was performed with a precursor ion mass tolerance of 20 ppm, considering only fully tryptic peptides with a maximum of 2 miscleavages allowed. Carbamidomethylation of cysteine was considered as a fixed modification.

### Publication Abstract
Mass spectrometry-based proteomics technologies are prime methods for the high-throughput identification of proteins in complex biological samples. Nevertheless, there are still technical limitations that hinder the ability of mass spectrometry to identify low abundance proteins in complex samples. Characterizing such proteins is essential to provide a comprehensive understanding of the biological processes taking place in cells and tissues. Still today, most mass spectrometry-based proteomics approaches use a data-dependent acquisition strategy, which favors the collection of mass spectra from proteins of higher abundance. Since the computational identification of proteins from proteomics data is typically performed after mass spectrometry analysis, large numbers of mass spectra are typically redundantly acquired from the same abundant proteins, and little to no mass spectra are acquired for proteins of lower abundance. We therefore propose a novel supervised learning algorithm, MealTime-MS, that identifies proteins in real-time as mass spectrometry data are acquired and prevents further data collection from confidently identified proteins to ultimately free mass spectrometry resources to improve the identification sensitivity of low abundance proteins. We use real-time simulations of a previously performed mass spectrometry analysis of a HEK293 cell lysate to show that our approach can identify 92.1% of the proteins detected in the experiment using 66.2% of the MS2 spectra. We also demonstrate that our approach outperforms a previously proposed method, is sufficiently fast for real-time mass spectrometry analysis, and is flexible. Finally, MealTime-MS' efficient usage of mass spectrometry resources will provide a more comprehensive characterization of proteomes in complex samples.

### Keywords
Human, Cell line, Hek293, Lc-ms/ms

### Affiliations
Department Biochemistry, Microbiology and Immunology, Ottawa Institute of Systems Biology, University of Ottawa
University of Ottawa

### Submitter
Mathieu Lavallée-Adam

### Lab Head
Dr Mathieu Lavallée-Adam
Department Biochemistry, Microbiology and Immunology, Ottawa Institute of Systems Biology, University of Ottawa


