### Accession
PXD011225

### Title
MaxQuant.Live enables global targeting of more than 25,000 peptides

### Description
Mass spectrometry (MS)-based proteomics is generally performed in a shotgun format, in which as many peptide precursors as possible are selected from full or MS1 scans so that their fragment spectra can be recorded in MS2 scans. While achieving great proteome depths, shotgun proteomics cannot guarantee that each precursor will be measured in each run. In contrast, targeted proteomics aims to reproducibly and sensitively fragment a restricted number of precursors in each run, based on pre-scheduled mass-to-charge and retention time windows. Here we set out to merge these two concepts by a global targeting approach in which an arbitrary number of previously measured precursors is detected in real-time, followed by standard fragmentation or advanced peptide-specific analyses. We made use of a fast application programming interface to a quadrupole Orbitrap instrument and recalibration in mass, retention time and intensity dimensions to predict peptide identity. MaxQuant.Live is freely available and has a graphical user interface to specify many pre-defined data acquisition strategies. Controlling the acquisition with MaxQuant.Live rather than the vendor software, we observed no decline in acquisition speed. The power of our approach is demonstrated with the acquisition of breakdown curves for thousands of precursors of interest. It is also possible to uncover precursors that are not even visible in MS1 scans, using elution time prediction based on co-eluting isotope standards or the auto-adjusted, predicted retention time alone. Finally, we demonstrate that more than 25,000 precursors can be successfully recognized and targeted in single LC-MS runs. We conclude that global targeting combines the advantages of two classical approaches in MS-based proteomics, while expanding the analytical toolbox with many new possibilities.

### Sample Protocol
Cell culture and sample preparation - We cultured the human HeLa cancer cell line (S3, ATCC) in Dulbeccoâ€™s modified Eagleâ€™s medium (DMEM) with 10% fetal bovine serum, 20 mM glutamine and 1% penicillin-streptomycin added (PAA Laboratories). Metabolic stable isotope labeling was performed in arginine- and lysine-free DMEM, fortified with arginine and lysine with natural isotope abundances (light channel) or stable-isotope labeled arginine-10 and lysine-8 (Cambridge Isotope Laboratories) as previously described. Cells were collected by centrifugation, washed twice with cold phosphate-buffered saline, pelleted and stored at -80 Â°C.  We lysed the cells, and reduced and alkylated the proteins in a single reaction vial with sodium deoxycholate (SDC) buffer containing chloroacetamide (PreOmics GmbH) similar to our previously published protocol. Cells were suspended in the SDC buffer and boiled for 10 min at 95 Â°C. To disrupt remaining cellular structures and shear nucleic acids, we sonicated the suspension for 15 min at full power (Bioruptor, Diagenode). The crude protein extracts were enzymatically digested with LysC and trypsin (1:100, enzyme wt : protein wt) overnight at 37 Â°C before stopping the reaction with 5 volumes of isopropanol/1% trifluoroacetic acid (TFA). Peptide micro-purification and de-salting was performed on styrenedivinylbenzene-reversed phase sulfonate StageTips. Following sequential washing steps with isopropanol/1% TFA and water with 0.1% TFA, peptides were eluted with 80% acetonitrile (ACN) containing 1% ammonia. The vacuum dried eluates were reconstituted in water with 2% ACN and 0.1% TFA for further analysis.

### Data Protocol
Shotgun proteomics raw data acquired with either the standard user interface or MaxQuant.Live were processed with MaxQuant version 1.6.1.13 using the default settings if not stated otherwise. The built-in Andromeda search engine21 scored MS2 spectra against fragment masses of tryptic peptides derived from a human reference proteome containing 95,057 entries including isoforms (UniProt, release 2018/06) and a list of 245 potential contaminants. We required a minimum peptide length of 7 amino acids and limited the search space to a maximum peptide mass of 4,600 Da and two missed cleavage sites. Carbamidomethylation of cysteine was specified as a fixed modification, and methionine oxidation and acetylation at protein N-termini as variable modifications. MaxQuant uses individual mass tolerances for each peptide, while the initial maximum precursor mass tolerances were set to 20 ppm in the first search and 4.5 ppm in the main search, and the maximum fragment mass tolerance was set to 20 ppm. The false discovery rate was controlled with a target-decoy approach at less than 1% for peptide spectrum matches and less than 1% for protein group identifications.

### Publication Abstract
Mass spectrometry (MS)-based proteomics is often performed in a shotgun format, in which as many peptide precursors as possible are selected from full or MS1 scans so that their fragment spectra can be recorded in MS2 scans. Although achieving great proteome depths, shotgun proteomics cannot guarantee that each precursor will be fragmented in each run. In contrast, targeted proteomics aims to reproducibly and sensitively record a restricted number of precursor/fragment combinations in each run, based on prescheduled mass-to-charge and retention time windows. Here we set out to unify these two concepts by a global targeting approach in which an arbitrary number of precursors of interest are detected in real-time, followed by standard fragmentation or advanced peptide-specific analyses. We made use of a fast application programming interface to a quadrupole Orbitrap instrument and real-time recalibration in mass, retention time and intensity dimensions to predict precursor identity. MaxQuant.Live is freely available (www.maxquant.live) and has a graphical user interface to specify many predefined data acquisition strategies. Acquisition speed is as fast as with the vendor software and the power of our approach is demonstrated with the acquisition of breakdown curves for hundreds of precursors of interest. We also uncover precursors that are not even visible in MS1 scans, using elution time prediction based on the auto-adjusted retention time alone. Finally, we successfully recognized and targeted more than 25,000 peptides in single LC-MS runs. Global targeting combines the advantages of two classical approaches in MS-based proteomics, whereas greatly expanding the analytical toolbox.

### Keywords
Proteomics, Mass spectrometry

### Affiliations
Department of Proteomics and Signal Transduction Max Planck Institute of Biochemistry
Proteomics

### Submitter
Mario Oroshi

### Lab Head
Dr Matthias Mann
Department of Proteomics and Signal Transduction Max Planck Institute of Biochemistry


