### Accession
PXD003468

### Title
Incorporation of nonproteinogenic amino acids into bacterial proteome analyzed by unbiased quantitative MS of protein modifications

### Description
To study incorporation of nonproteinogenic amino acids in bacterial proteome we performed a global, unbiased protein modification analysis of the E. coli K12 strain with defective editing mechanism of the leucyl tRNA synthetase (LeuRS), which in addition to leucine incorporates a nonproteinogenic amino acid norvaline. We measured widespread loss of methyl groups at leucine positions in the LeuRS mutant, indicative of norvaline incorporation. We performed a Super-SILAC experiment and established that under microaerobic conditions norvaline misincorporation reaches a maximum approximately 10 hours after entry into stationary phase, when up to 10% of all measured leucines are substituted with norvaline.

### Sample Protocol
A total of 100 μg of crude protein extract in denaturation buffer were digested prior to peptide separation via isoelectric focusing, while 20 μg (10 μg SSS and 10 μg Lys-0) were digested for the super-SILAC experiment or for the “heavy”-label incorporation checks. Proteins were first reduced using 1 mM dithiothreitol and alkylated using 5.5 mM iodoacetamide. Protein digestion was performed using endoproteinase Lys-C (1:100 w/w) for 3 h at room temperature, followed by dilution with 4 volumes of 20 mM ammonium bicarbonate and addition of endoproteinase Lys-C (1:100 w/w) for overnight digestion (approximately 16 h, 750 rpm at room temperature). Protein digestion was stopped by acidification of the solution using trifluoroacetic acid (TFA) to a final concentration of 0.1% (v/v) and desalted using desalted using C18 StageTips (Rappsilber J. et al 2007, Nat. Protoc. 2, 1896–1906). Samples that were further fractionated using isoelectric focusing were not acidified. Peptides derived from the in-solution digestion were separated according to their isoelectric point into 12 fractions using the 3100 OFFGEL Fractionator (Agilent Technologies) following the manufacturer´s instructions. Separation was performed on a 13-cm Immobiline DryStrips with a pH 3-10 gradient (GE Healthcare) at a maximum current of 50 µA until 20 kVh were reached. The peptide fractions were acidified with acidic solution (30% v/v acetonitrile, 5% v/v acetic acid and 10% v/v TFA) and desalted using Stage-Tips. Desalted peptide samples were separated by an EASY-nLC II system (Proxeon Biosystems) coupled on-line to an Orbitrap Elite mass spectrometer (Thermo Fisher Scientific) through a nanoelectrospray ion source (Proxeon Biosystems). Chromatographic separation was performed on a 15 cm fused silica emitter with an inner diameter of 75 µm and a tip diameter of 8 µm, packed in-house with reversed-phase ReproSil-Pur C18-AQ 3 µm resin (Dr. Maisch GmbH). Peptides were injected onto the column with solvent A at 700 nl/min using a maximum pressure of 280 bar. The peptides were eluted using 79 min or 219 min (for isoleucine incorporation experiment) segmented gradient of 5-50% solvent B at a constant flow rate of 200 nl/min. The mass spectrometer was operated in a data-dependent mode, switching automatically between one full-scan and subsequent MS/MS scans of the 15 most abundant peaks (Top15 method) selected with an isolation window of 4 Th. Full scan MS spectra were acquired in a mass range from m/z 300-2000 at a target value of 1 x 106 charges with the maximum injection time of 100 ms and a resolution of 120,000 (defined at m/z 400). The higher energy collisional dissociation (HCD) MS/MS spectra were recorded at a target value of 4 x 104 with the maximum injection time of 150 ms at a resolution of 15,000 (defined at m/z 400) with a normalized collision energy of 35%. The masses of sequenced precursor ions were dynamically excluded from MS/MS fragmentation for 60 s. Ions with single and unassigned charge were excluded from fragmentation selection.

### Data Protocol
Acquired raw data was processed using MaxQuant software (version 1.2.2.9) and the derived peak list was searched using Andromeda search engine integrated in the MaxQuant against a reference E. coli K12 proteome (taxonomy ID 83333) obtained from UniProt (4,311 protein entries; release, February 2014) and a file containing 248 common laboratory contaminants. The first search was carried out with a mass tolerance of 20 ppm, while during the main search, the mass tolerance of precursor and the fragment ions were set to 6 and 20 ppm, respectively. Multiplicity was set to two, matching the number of SILAC labels used in both “norvaline” and “isoleucine” experiments: Lys-4 or Leu3 were specified as heavy labels, respectively. The minimum required peptide length was set to seven amino acids with the maximum of two missed cleavages allowed for endoproteinase Lys-C that was set to specifically cleave at lysine C-terminus. Methionine oxidation and protein N-terminal acetylation were defined as variable modifications and carbamidomethylation of cysteines was set as a fixed modification. Additionally, in the “norvaline” experiments, loss of a methyl group (-14.01565 Da) from leucine was defined as a variable modification (“norvaline”) specifying leucine substitution by norvaline. During a separate unbiased search without “norvaline” being specified as a variable modification, dependent peptide option was enabled with mass bin size of 0.0055 Da. Peptide, protein and modification site identifications were filtered using a target-decoy approach at a false discovery rate (FDR) set to 0.01. For protein quantification a minimum of two unmodified peptide ratio counts were required, while norvaline sites were quantified based on at least one modified peptide with a minimal ratio count of one. The “match between runs” option was enabled with a match time window set to 2 min, allowing the matching of identifications across measurements. Proteins identified by the same set of peptides were combined to a single protein group. Norvaline containing peptides were filtered for a posterior error probability (PEP) score of ≤0.01 calculated for at least one experiment. Only norvaline substitution events with a localization probability of ≥0.75 positions were considered as localized on the leucine positions. For spectral counting, MS/MS spectra were filtered for PEP score of ≤0.01 and labeling state of either 0 for Lys-0- or 1 for Lys-4-labeled peptides.

### Publication Abstract
The genetic code is virtually universal in biology and was likely established before the advent of cellular life. The extent to which mistranslation occurs is poorly understood and presents a fundamental question in basic research and production of recombinant proteins. Here we used shotgun proteomics combined with unbiased protein modification analysis to quantitatively analyze in vivo mistranslation in an E. coli strain with a defect in the editing mechanism of leucyl-tRNA synthetase. We detected the misincorporation of a non-proteinogenic amino acid norvaline on 10% of all measured leucine residues under microaerobic conditions and revealed preferential deployment of a tRNA(Leu)(CAG) isoacceptor during norvaline misincorporation. The strain with the norvalylated proteome demonstrated a substantial reduction in cell fitness under both prolonged aerobic and microaerobic cultivation. Unlike norvaline, isoleucine did not substitute for leucine even under harsh error-prone conditions. Our study introduces shotgun proteomics as a powerful tool in quantitative analysis of mistranslation.

### Keywords
Mistranslation, Norvaline, Trna synthetase, Leucine, Nonproteinogenic amino acids, Orbitrap

### Affiliations
Proteome Center Tuebingen, Interfaculty Institute for Cell Biology, University of Tuebingen, Tuebingen, Germany
Proteome Center Tuebingen, Tuebingen, Germany

### Submitter
Maja Semanjski

### Lab Head
Dr Boris Macek
Proteome Center Tuebingen, Interfaculty Institute for Cell Biology, University of Tuebingen, Tuebingen, Germany


