### Accession
PXD042540

### Title
Characterization of the biochemical activity and tumor-promoting role of the dual protein methyltransferase METL-13/METTL13 in Caenorhabditis elegans

### Description
The methyltransferase-like protein 13 (METTL13) methylates the eukaryotic elongation factor 1 alpha (eEF1A) on two locations: the N-terminal amino group and lysine 55. The absence of this methylation leads to reduced protein synthesis and cell proliferation in human cancer cells. Previous studies showed that METTL13 is dispensable in non-transformed cells, making it potentially interesting for cancer therapy. However, METTL13 has not been examined yet in whole animals. Here, we used the nematode Caenorhabditis elegans as a simple model to assess the functions of METTL13. Using methyltransferase assays and mass spectrometry, we show that the C. elegans METTL13 (METL-13) methylates eEF1A (EEF-1A) in the same way as the human protein. Crucially, the cancer-promoting role of METL-13 is also conserved and depends on the methylation of EEF-1A, like in human cells. At the same time, METL-13 appears dispensable for animal growth, development, and stress responses. This makes C. elegans a convenient whole-animal model for studying METL13-dependent carcinogenesis without the complications of interfering with essential wild-type functions.

### Sample Protocol
Protein samples enriched for EEF-1A were resolved by SDS-PAGE and stained with Coomassie Blue. A band corresponding to the mass of EEF-1A (~ 50 kDa) was excised from the gel and subjected to in-gel digestion using chymotrypsin or Glu-C, after which the resulting proteolytic fragments were desalted using 10 Âµl OMIX C18 micro-SPE pipette tips (Agilent, Santa Clara, CA, USA). The samples were analyzed by nanoLC-MS using either an Ultimate 3000 RSLCnano-UHPLC system connected to a Q Exactive mass spectrometer (Thermo Fisher Scientific, Bremen, Germany) or a NanoElute-UHPLC coupled to a TimsTOF Pro mass spectrometer (Bruker Daltonics, Bremen, Germany).

### Data Protocol
The LC-MS data were analyzed using Peaks Studio version 10.6 (Bioinformatics Solution Inc.), searching against the C. elegans protein sequence database from Swiss-Prot (4155 entries). The maximum number of variable modifications detected on a peptide was restricted to three, allowing the following variable modifications: methionine oxidation, cysteine propionamidation, methylation of lysine (mono, di, and tri), and methylation of the N-terminus (mono, di, and tri). MS/MS spectra of peptides containing the N-terminal G2 or K55 of EEF-1A were inspected for presence of methylated residues, followed by quantitation of G2 and K55 methylation status using PEAKS Studio.

### Publication Abstract
The methyltransferase-like protein 13 (METTL13) methylates the eukaryotic elongation factor 1 alpha (eEF1A) on two locations: the N-terminal amino group and lysine 55. The absence of this methylation leads to reduced protein synthesis and cell proliferation in human cancer cells. Previous studies showed that METTL13 is dispensable in non-transformed cells, making it potentially interesting for cancer therapy. However, METTL13 has not been examined yet in whole animals. Here, we used the nematode Caenorhabditis elegans as a simple model to assess the functions of METTL13. Using methyltransferase assays and mass spectrometry, we show that the C. elegans METTL13 (METL-13) methylates eEF1A (EEF-1A) in the same way as the human protein. Crucially, the cancer-promoting role of METL-13 is also conserved and depends on the methylation of EEF-1A, like in human cells. At the same time, METL-13 appears dispensable for animal growth, development, and stress responses. This makes C. elegans a convenient whole-animal model for studying METL13-dependent carcinogenesis without the complications of interfering with essential wild-type functions.

### Keywords
C. elegans, Mettl13, Protein synthesis, Methylation, Eef1a

### Affiliations
University of Oslo
Department of Biosciences University of Oslo

### Submitter
Bernd Thiede

### Lab Head
Dr Jedrzej M. Malecki
Department of Biosciences University of Oslo


