### Accession
PXD009622

### Title
Quantitative Proteomics Reveals Antibiotics Resistance Function of Outer Membrane Proteins in Aeromonas hydrophila

### Description
Each colony of A. hydrophila and the OXY-R strain were incubated in 5 mL LB medium overnight, and then diluted in 100 mL fresh LB medium at a ratio of 1:100 and subsequently cultured until the optical density at 600 nm (OD600) reached 1.0. The cultures were harvested via centrifuging for 20 min at 10,000 x g, 4°C, and then the bacterial cells were washed with cold phosphate buffered saline (PBS, pH 7.4) for three times. The cell pellets were resuspending in the 10 mL ultrasonic buffer (50 mM Tris-HCl, pH 7.4, 1 mM PMSF) and disrupted with intermittent sonic oscillation for a total of 30 minutes at 9 seconds intervals on ice. Subsequently, the cell debris and unbroken cells were separated by centrifugation at 8,000 x g for 20 min at 4°C. Then the supernatant was centrifuged at 100,000 x g for 1 h at 4°C in the Optima LE-80 K Ultracentrifuge (Beckman, Palo Alto, CA, USA). After the pellet was dissolved with 2% sodium lauroyl sarcosine (in 50 mM Tris-HCl, pH 7.5) for 40 min at room temperature (RT), the pellets were ultracentrifuged again at 100,000 x g for 1 h at 4°C. Finally, the precipitate was dissolved in an appropriate volume of SDT buffer (4 % SDS, 0.1 M DTT [dithiothreitol], and 0.5 M triethylammonium bicarbonate buffer [TEAB, pH 8.5]). The protein concentration was determined using the Bradford method and then stored at -20°C until subsequent use.The proteins were digested with trypsin after being reduced with DTT and alkylated with Iodoacetamide using a filter-aided sample preparation method (Tanca et al., 2013; Li et al., 2016b). After washing three times with 0.5 M TEAB followed by fractionation using the 10 kDa ultrafiltration system (Millipore, Billerica, MA, USA), about 100 μg digested peptide from each group, including two biological replicates, was taken out for further labeled using TMT isobaric and isotopic mass-tagging kits (Thermo Fisher Scientific, MA, USA), which was performed according to instructions from the kit.

### Sample Protocol
Labeled peptides were re-suspended in 0.1 % formic acid and submitted to analysis with the AB/Sciex TripleTOF 5600 Plus Mass Spectrometer (AB SCIEX, Concord, ON, Canada) combined with the NanoAcquity ultraperformance liquid chromatography (UPLC) system (Waters Inc., Milford, MA, USA),all of labeled peptides were injected twice as technological replicates.The raw data were analyzed by Mascot Server 2.4 (Matrix Science, Framingham MA, USA) combined with Scaffold 4.3.4 (Proteome Software, Portland, OR, USA) against the A. hydrophila ATCC 7966 database in Uniprot. The .mgf files were procuced by the .wiff and .scan files utilizing Ms_data_converter_V1.3 (AB SCIEX), and the data of two technological replicates were mixed together and subsequently further identified and analyzed by Mascot software. In the current work, every experiment was performed twice as biological replicates.

### Data Protocol
We used the online Venn tool to display the overlap of identified proteins from two biological replicates (Khan and Mathelier, 2017). With respect to prediction of protein subcellular location, the accession numbers of differential proteins in overlap were firstly generated the protein sequences in Fasta format utilizing the Retrieve/ID mapping online software in Uniprot database (http://www.uniprot.org/), and the Fasta format was uploaded into the online Gneg-multi software to predicate the subcellular locations of proteins (Chou and Shen, 2006). Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes analyses of pathways corresponding to the altered proteins were performed using OmicsBean online software.(Li et al., 2017) We also combined STRING v.10 and Cytoscape v.3.3 software to predict and visualize the protein-protein interactions (PPIs) of differential proteins, and a confidence score ≥ 0.4 was used to select in the STRING (Lopes et al., 2010; Szklarczyk et al., 2015).

### Publication Abstract
Outer membrane proteins (OMPs) play essential roles in antibiotic resistance, particularly in Gram-negative bacteria; however, they still have many unidentified functions regarding their behavior in response to antibiotic stress. In the current work, quantitative tandem mass tag labeling-based mass spectrometry was used to compare the outer membrane related proteins between an oxytetracycline-resistant (OXY-R) and its original control stain (OXY-O) in <i>Aeromonas hydrophila</i>. Consequently, a total of 261 commonly altered proteins in two biological repeats were identified including 29 proteins that increased and 28 that decreased. Gene ontology analysis showed that the expression of transport proteins was significantly reduced, and translation-related proteins were downregulated in the OXY-R strain. After using western blotting to validate selected altered proteins, eight OMP-related genes were knocked out and their roles in antibiotic resistance were further evaluated. The survival assays showed that some mutants such as &#x394;<i>AHA_4281</i>, &#x394;<i>AHA_2766</i>, &#x394;<i>AHA_2282</i>, &#x394;<i>AHA_1181</i>, and &#x394;<i>AHA_1280</i> affected the susceptibility of <i>A. hydrophila</i> to antimicrobials. Moreover, the minimum inhibitory concentration assay showed that these candidate mutants also respond differently to other types of antibiotics. Our results reveal several novel outer membrane related proteins of <i>A. hydrophila</i> that play important roles in antibiotic resistance, and as such, may be helpful for screening studies to identify novel drug targets.

### Keywords
Aeromonas hydrophila， outer membrane proteins， antibiotics resistance, Lc-ms/ms

### Affiliations
Fujian Agricultural and Forestry University
College of Life Sciences, Fujian Agriculture and Forestry University

### Submitter
Xiangmin Lin

### Lab Head
Dr Xiangmin Lin
College of Life Sciences, Fujian Agriculture and Forestry University


