### Accession
PXD011741

### Title
Mouse HiRIEF LCMS Proteomics  - Sedentary VS Exercise, WT VS PolgA mutant

### Description
Mitochondrial dysfunction is implicated in aging and aging-related disorders, such as neurodegenerative diseases and stroke. To study the effects of progressive mitochondrial dysfunction, a homozygous knock-in mouse expressing a proof-reading deficient version of the nucleus-encoded catalytic subunit of mitochondrial DNA (mtDNA) polymerase (PolgA) has been developed. In the mtDNA mutator mouse the proofreading activity of PolgA has been abolished by a single amino acid change. PolgA is the catalytic subunit of the polymerase gamma, which is involved in replicating and proofreading the mitochondrial DNA. As a result, mtDNA mutator mice develop high levels of point mutations and linear deletions, which lead to several human-like phenotypes associated with aging, including reduced lifespan (42-44 weeks), weight loss, alopecia, anemia, kyphosis, osteoporosis, sarcopenia, loss of subcutaneous fat, and reduced fertility. We investigate the molecular mechanism through which exercise may improve the phenotype of the mtDNA mutator mouse, which is a model of premature aging induced by mitochondrial dysfunction. Remarkably, forced endurance exercise has been shown to rescue the progeroid aging phenotypes of the mtDNA mutator mice, and to induce systemic mitochondrial rejuvenation. Here, using voluntary, rather than forced exercise, we investigate the molecular mechanisms underlying such a dramatic improvement, and also assess the effect of exercise on brain tissues, such as cortex and striatum in our model. The complete proteome of key tissues (muscle, brain cortex, brain striatum) from exercising and sedentary mtDNA mutator mice as well as exercising and sedentary wild type mice is quantified using peptide high-resolution isoelectric focusing (HiRIEF) coupled with liquid chromatography tandem mass spectrometry (LC-MS/MS) with an isobaric tag (TMT10plex) strategy.

### Sample Protocol
Four mice were used for each of the four conditions (WT_sedentary, WT_exercised, Mutant_sedentary, Mutant_Exercised). From each animal muscle, brain cortex and brain striatum tissues were collected. The tissue samples were lysed with SDS buffer (4%SDS, 1mM Dithiothreitol, 25mM HEPES pH 7.6). Protein extracts were then digested to peptides using the FASP (Filter Aided Sample Prep) protocol. Each peptide mixture sample was labeled with the respective TMT10plex reagent prior to pooling into TMT sets. For each tissue the sample arrangement in two TMT sets was the same and as follows: set1_126 - WT_sed; set1_127N - WT_sed; set1_127C - WT_ex; set1_128N - WT_ex; set1_128C - Mut_Sed; set1_129N - Mut_Sed; set1_129C - Mut_Ex; set1_130N - Mut_Ex; set1_130C - internal pooled standard; set1_131 - internal pooled standard; set2_126 - WT_sed; set2_127N - WT_sed; set2_127C - WT_ex; set2_128N - WT_ex; set2_128C - Mut_Sed; set2_129N - Mut_Sed; set2_129C - Mut_Ex; set2_130N - Mut_Ex; set2_130C - internal pooled standard; set2_131 - internal pooled standard. The pooled TMT sets were cleaned by strong cation exchange-solid phase extraction and separated by High Resolution Isoelectric Focusing (HiRIEF) prior to data-dependent LC-MS/MS analysis on a Q-Exactive Mass Spectrometer.

### Data Protocol
All MS/MS spectra were searched by MSGF+/Percolator under the Galaxy platform (https://usegalaxy.org) using a target-decoy strategy. The reference database was the Uniprot reference Mus Musculus proteome (canonical and isoform, 51529 protein entries, downloaded from uniprot.org on 2014-08-01). We used a precursor mass tolerance of 10 ppm and high resolution setting on MS2 level. Only peptides with fully tryptic termini were allowed. We considered carbamidomethylation on cysteine and TMT-10plex on lysine and N-terminus as fixed modifications; and oxidation of methionine as variable modification. Quantitation of TMT-10plex reporter ions was done using an integration window tolerance of 10 ppm.

### Publication Abstract
The accumulation of mitochondrial DNA (mtDNA) mutations is a suspected driver of aging and age-related diseases, but forestalling these changes has been a major challenge. One of the best-studied models is the prematurely aging mtDNA mutator mouse, which carries a homozygous knock-in of a proofreading deficient version of the catalytic subunit of mtDNA polymerase-&#x3b3; (PolgA). We investigated how voluntary exercise affects the progression of aging phenotypes in this mouse, focusing on mitochondrial and protein homeostasis in both brain and peripheral tissues. Voluntary exercise significantly ameliorated several aspects of the premature aging phenotype, including decreased locomotor activity, alopecia, and kyphosis, but did not have major effects on the decreased lifespan of mtDNA mutator mice. Exercise also decreased the mtDNA mutation load. In-depth tissue proteomics revealed that exercise normalized the levels of about half the proteins, with the majority involved in mitochondrial function and nuclear-mitochondrial crosstalk. There was also a specific increase in the nuclear-encoded proteins needed for the tricarboxylic acid cycle and complex II, but not in mitochondrial-encoded oxidative phosphorylation proteins, as well as normalization of enzymes involved in coenzyme Q biosynthesis. Furthermore, we found tissue-specific alterations, with brain coping better as compared to muscle and with motor cortex being better protected than striatum, in response to mitochondrial dysfunction. We conclude that voluntary exercise counteracts aging in mtDNA mutator mice by counteracting protein dysregulation in muscle and brain, decreasing the mtDNA mutation burden in muscle, and delaying overt aging phenotypes.

### Keywords
Hirief, Striatum, Brain, Exercise, Mouse, Cortex, Lcms, Muscle, Polga, Sedentary, Wt

### Affiliations
Dept. Oncology-Pathology, Science for Life Laboratory, Karolinska Institutet, Sweden
Clinical Proteomics Unit, Dep. of Oncology-Pathology

### Submitter
Rui Branca

### Lab Head
Dr Janne Lehti√∂
Dept. Oncology-Pathology, Science for Life Laboratory, Karolinska Institutet, Sweden


