### Accession
PXD033628

### Title
Temporal changes in the microglial proteome of male and female mice after a diffuse brain injury using label-free quantitative proteomics

### Description
Traumatic brain injury (TBI) triggers neuroinflammatory cascades mediated by microglia, which promotes tissue repair in the short-term. These cascades may exacerbate TBI-induced tissue damage and symptoms in the months to years post-injury. However, the progression of the microglial function across time post-injury and whether this differs between biological sexes is not well understood. In this study, we examined the microglial proteome in the days (3- and 7-days) to 1 month (28 days) after a midline fluid percussion injury (mFPI) in male and female mice using label-free quantitative proteomics. We identified a reduction in microglial proteins involved with clearance of neuronal debris via phagocytosis at 3- and 7-days post-injury. At 28 days post-injury pro-inflammatory proteins were decreased and anti-inflammatory proteins were increased in microglia. These results indicate a reduction in microglial clearance of neuronal debris in the days post-injury with a shift to anti-inflammatory function by 1 month. The changes in the microglial proteome that occurred across time post-injury did not differ between biological sexes. However, we did identify an increase in microglial proteins related to pro-inflammation as well as insulin and estrogen signalling in males compared with female mice that occurred with or without a brain injury. Although microglial response was similar between males and females up to 1 month following TBI, biological sex differences in the basal microglial proteome has implications for the efficacy of treatment strategies targeting the microglial response post-injury.

### Sample Protocol
Microglia were sonicated three times, with 15 seconds in the sonicator and 5 seconds on ice, then kept at 4¬∞C for 2 hours. Protein concentration was determined by performing a Bicinchoninic Acid (BCA) assay as per manufactures protocol (PierceTM BCA protein assay kit, ThermoScientific) and adjusted to 50 ùúág/mL. Samples containing ~5 ùúág of protein were sequentially reduced then alkylated using standard methods (Hughes et al. 2019), then digested at a 1:25 enzyme:protein ratio with proteomics grade trypsin/rLysC (V5071; Promega, Madison, WI, USA) overnight at 37¬∞C according to the SP3 method (Hughes et al. 2019). Following digestion, peptides were acidified with 2 ùúál of 1% TFA then desalted using C18 ZipTipsÔÉí (Millipore, cat no. Z720070-96EA).

### Data Protocol
DIA-MS raw files were processed using Spectronaut software (v14.8; Biognosys AB). A project-specific library was generated using the Pulsar search engine to search the DIA MS2 spectra against the Mus musculus UniProt reference proteome (comprising 44,456 entries, April 2017). Biognosys factory settings were used for both spectral library generation and DIA data extraction, with the exception that single-hit proteins were excluded. For library generation, search parameters allowed for N-terminal acetylation and methionine oxidation as variable modifications and cysteine carbamidomethylation as a fixed modification and up to two missed cleavages were permitted. Peptide, protein and PSM thresholds set to 0.01. Mass tolerances were based on first pass calibration and extensive calibration for the calibration and main searches, respectively, with correction factors set to 1 at the MS1 and MS2 levels. Targeted searching of the library based on XIC extraction deployed dynamic retention time alignment with a correction factor of 1. Protein identification deployed a 1% q-value cut-off at precursor and protein levels, automatic generation of mutated peptide decoys based on 10% of the library and dynamic decoy limitation for protein identification. Protein label-free quantitation used the Quant 2.0 setting, based on MS2-level data using the intensity values for the Top3 peptides (stripped sequences) and cross-run normalization was based on median peptide intensity.

### Publication Abstract
Traumatic brain injury (TBI) triggers neuroinflammatory cascades mediated by microglia, which promotes tissue repair in the short-term. These cascades may exacerbate TBI-induced tissue damage and symptoms in the months to years post-injury. However, the progression of the microglial function across time post-injury and whether this differs between biological sexes is not well understood. In this study, we examined the microglial proteome at 3-, 7-, or 28-days after a midline fluid percussion injury (mFPI) in male and female mice using label-free quantitative proteomics. Data are available via ProteomeXchange with identifier PXD033628. We identified a reduction in microglial proteins involved with clearance of neuronal debris via phagocytosis at 3- and 7-days post-injury. At 28&#x2009;days post-injury, pro-inflammatory proteins were decreased and anti-inflammatory proteins were increased in microglia. These results indicate a reduction in microglial clearance of neuronal debris in the days post-injury with a shift to anti-inflammatory function by 28&#x2009;days following TBI. The changes in the microglial proteome that occurred across time post-injury did not differ between biological sexes. However, we did identify an increase in microglial proteins related to pro-inflammation and phagocytosis as well as insulin and estrogen signaling in males compared with female mice that occurred with or without a brain injury. Although the microglial response was similar between males and females up to 28&#x2009;days following TBI, biological sex differences in the microglial proteome, regardless of TBI, has implications for the efficacy of treatment strategies targeting the microglial response post-injury.

### Keywords
Traumatic brain injury, Proteomics, Tbi, Inflammation, Biological sex, Microglia

### Affiliations
University of Tasmania
Proteomics facility, Central Science Laboratory, University of Tasmania

### Submitter
Richard Wilson

### Lab Head
Dr Dr Richard Wilson
Proteomics facility, Central Science Laboratory, University of Tasmania


