### Accession
PXD001845

### Title
Benchmarking multiple fragmentation methods on an Orbitrap Fusion for top-down phospho-proteoform characterization

### Description
Top-down analysis of intact proteins by mass spectrometry provides an ideal platform for comprehensive proteoform characterization, in particular, for the identification and localization of post-translational modifications (PTM) co-occurring on a protein. One of the main bottlenecks in top-down proteomics is insufficient protein sequence coverage caused by incomplete protein fragmentation. Based on previous work on peptides, increasing sequence coverage and PTM localization by combining sequential ETD and HCD fragmentation in a single fragmentation event, we hypothesized that protein sequence coverage and phospho-proteoform characterization could be equally improved by this new dual fragmentation method termed EThcD, recently been made available on the Orbitrap Fusion. Here, we systematically benchmark the performance of several (hybrid) fragmentation methods for intact protein analysis on an Orbitrap Fusion, using as a model system a 17.5 kDa N-terminal fragment of the mitotic regulator Bora. During cell division Bora becomes multiply phosphorylated by a variety of cell cycle kinases, including Aurora A and Plk1, albeit at distinctive sites. Here, we monitor the phosphorylation of Bora by Aurora A and Plk1, analyzing the generated distinctive phospho-proteoforms by top-down fragmentation. We show that EThcD and ETciD on a Fusion are feasible and capable of providing richer fragmentation spectra compared to HCD or ETD alone, increasing protein sequence coverage, and thereby facilitating phosphosite localization and the determination of kinase specific phosphorylation sites in these phospho-proteoforms.

### Sample Protocol
Kinase Phosphorylation assay The N-terminal fragment of Bora (156 amino acids, 17.5 kDa), the kinase domain of AurA and full-length Plk1 were expressed in and purified from E.coli and prepared as described previously 15. Bora and AurA were incubated for 20 min and 2 h, Bora and Plk1 for 2 h, at a 2:1 molar ratio in 20 mM HEPES pH 7.5, 100 mM sodium chloride at room temperature in the presence of 40 mM magnesium chloride and 40 mM ATP. The kinase reaction was quenched by addition of 80 mM EDTA and placed on ice. Prior to top-down MS analysis, the buffer was exchanged to 0.1% FA. Mass Spectrometry All data was acquired on a Thermo Scientific Orbitrap Fusion mass spectrometer (Thermo Fisher Scientific, San Jose, CA). For direct infusion, Bora samples were diluted to a final concentration of 2 Î¼M in 40% MeOH, 0.1% FA. A 100 Î¼L Hamilton syringe was used for direct infusion, and the flow rate was set to 2 Î¼L/min. Instrument control software Tune 1.1.982 with EThcD and ETciD as built-in methods was used. All data were acquired in the Orbitrap mass analyzer at a resolution of 120,000 (full width at half maximum, FWHM) in intact protein mode (3mTorr ion-routing multipole (IRM) pressure). Ten scans, each consisting of 10 microscans were averaged.  The 18, 19, 22 and 24+ charge states of intact, unphosphorylated Bora and the most abundant precursor charge state (19+) of mono-, di- and tri-phosphorylated Bora were subjected to fragmentation. Spectra for all fragmentation methods were acquired using a mass range of 150â€“2000 m/z. Precursor ion isolation was performed with mass selecting quadrupole and the isolation window was set to 2.0 m/z. The precursor automatic gain control (AGC) target value was 5e5, maximum injection time 200 ms. For HCD, normalized collision energy (NCE) was set to 15, 20 and 25%. ETD ion/ion reaction times (anion AGC 1e6, 200 ms maximum injection time), supplemental collisional activation (SA) in ETciD, and NCE in EThcD varied as indicated.

### Data Protocol
Intact protein spectra were deconvoluted with XTRACT with a signal-to-noise ratio (S/N) threshold of 3, a fit factor of 80% and a remainder threshold of 15% using Protein Deconvolution 3.0 (Thermo Scientific). Phosphosite localization was determined using ProSight lite (Proteomics Center of Excellence, Evanston, IL) in combination with an adapted version of the in-house developed previously described program SlinkS11a for fragment ion matching, higher throughput and statistical data analysis. SlinkS can take into account b, y, c, z, b-H2O, y-1, y-H2O, and z+1 ions. Single protein searches were performed with 10 ppm fragment mass tolerance for all fragmentation methods. Manual interpretation was performed by fragment comparison of unprocessed raw spectra to theoretical masses generated by ProteinProspector v5.10.3 (http://prospector.ucsf.edu). Percentage sequence coverage was calculated as the total number of non-redundant inter-residue cleavages divided by the maximum number possible. Phosphosite localization and positional phospho-isomers were further validated by manual inspection of the spectra.

### Publication Abstract
Top-down analysis of intact proteins by mass spectrometry provides an ideal platform for comprehensive proteoform characterization, in particular, for the identification and localization of post-translational modifications (PTM) co-occurring on a protein. One of the main bottlenecks in top-down proteomics is insufficient protein sequence coverage caused by incomplete protein fragmentation. Based on previous work on peptides, increasing sequence coverage and PTM localization by combining sequential ETD and HCD fragmentation in a single fragmentation event, we hypothesized that protein sequence coverage and phospho-proteoform characterization could be equally improved by this new dual fragmentation method termed EThcD, recently been made available on the Orbitrap Fusion. Here, we systematically benchmark the performance of several (hybrid) fragmentation methods for intact protein analysis on an Orbitrap Fusion, using as a model system a 17.5 kDa N-terminal fragment of the mitotic regulator Bora. During cell division Bora becomes multiply phosphorylated by a variety of cell cycle kinases, including Aurora A and Plk1, albeit at distinctive sites. Here, we monitor the phosphorylation of Bora by Aurora A and Plk1, analyzing the generated distinctive phospho-proteoforms by top-down fragmentation. We show that EThcD and ETciD on a Fusion are feasible and capable of providing richer fragmentation spectra compared to HCD or ETD alone, increasing protein sequence coverage, and thereby facilitating phosphosite localization and the determination of kinase specific phosphorylation sites in these phospho-proteoforms. Data are available via ProteomeXchange with identifier PXD001845.

### Keywords
Top down proteomics, Phosphorylation, Proteoform, Bora, Ethcd, Fragmentation, Orbitrap fusion

### Affiliations
Biomolecular Mass Spectrometry and Proteomics Bijvoet Center for Biomolecular Research and Utrecht Institute for Pharmaceutical Sciences Utrecht University Padualaan 8 3584 CH Utrecht The Netherlands
Utrecht University

### Submitter
Andrea Mizzi Brunner

### Lab Head
Dr Albert J.R. Heck
Biomolecular Mass Spectrometry and Proteomics Bijvoet Center for Biomolecular Research and Utrecht Institute for Pharmaceutical Sciences Utrecht University Padualaan 8 3584 CH Utrecht The Netherlands


