### Accession
PXD013160

### Title
Integrative phosphoproteome and interactome analysis of the role of Ubash3b in BCR-ABL signaling

### Description
Ubash3b, also known as suppressor of T-cell receptor signaling or Sts-1, is an ill-studied atypical tyrosine phosphatase with ubiquitin binding ability. In our previous study, we hypothesized that Ubash3b plays an inhibitory role in BCR-ABL signaling through binding and dephosphorylating BCR-ABL and its interactors. The Hantschel lab recently solved the crystal structures of the p210 PH and DH domains, which are absent in the p190 variant, and demonstrated that loss-of-function mutations in the PH domain altered BCR-ABL localization, thereby reducing the interaction between Ubash3b and p2104. Taken together, this suggests differential subcellular localization of Ubash3b as a mechanism by which it interacts more strongly with p201 as compared to p190. To better understand the global impact Ubash3b has on p210, its direct kinase substrates and proteins in its phosphotyrosine signaling network, we have taken an integrative approach by combining global phosphotyrosine profiling, proximity-dependent biotinylation (BioID) and total protein analysis to investigate p210 signaling upon Ubash3b knockdown (KD). The BioID system was used to characterize Ubash3b function in p210 signaling by examining its interactome. Importantly, in all of our BioID experiments, we employed a newly technique that we have recently developed, Biotinylation Site Identification Technology (BioSITe), which directly identifies biotinylated peptides thereby increasing the reliability of the identified interactors.  Here, we additionally used short hairpin RNA (shRNA) interference and generated Ubash3b knock-down (KD) and non-targeting control shRNA lines in Ba/F3 BirA*-p210 cells. Ubash3b expression was reduced >90 % in the KD cells and had a substantial effect on global tyrosine phosphorylation and on the interactome of p210. Of the 1,421 unique tyrosine phosphorylation sites identified from 830 proteins, 379 sites (from 286 proteins) exhibited a substantial increase (ÃƒÂ¢Ã¢â‚¬Â°Ã‚Â¥2-fold) in tyrosine phosphorylation upon Ubash3b KD cells compared to control cells. To date, the interactome of Ubash3b has not been extensively investigated, however, some examination of Ubash3b in the context p210 signaling has been undertaken. We designed constructs of C-terminal BirA* tagged full length Ubash3b and a deletion mutant lacking the UBA and SH3 domains leaving only the phosphatase domain tethered to BirA*. A comparative analysis of the core interactors of p210 from previous studies and Ubash3b interactome from the current study revealed 36 proteins that interact with both p210 and Ubash3b.

### Sample Protocol
Peptide enrichment preparation  Protein extraction was carried out by sonication (three rounds, duty cycle 30%, 20 s pulses) in 50 mM TEABC and 8 M urea. The protein concentration of samples was measured by BCA assay. For SILAC experiments heavy and light samples were mixed 1:1 and a total of 10 mg of lysate per replicate was then reduced and alkylated were by serial incubation of 10 mM DTT for 30 min and by 20 mM IAA for 30 min in the dark. Lysate was diluted to 2 M urea by adding three cell lysate volumes of 50 mM TEABC. The proteins were digested with trypsin (1:20 of trypsin to protein) at 37 Ãƒâ€šÃ‚Â°C overnight. The resulting tryptic peptides were desalted using a Sep-PAK C18 column and subsequently lyophilized.  Immunoaffinity Purification of Phosphotyrosine Peptides and total protein analysis  Peptides were resuspended in IAP buffer (CST #9993) and pH was brought to neutral with 1M tris base. Phosphopeptides were enriched using the PTMScanÃƒâ€šÃ‚Â® Phospho-Tyrosine Mouse mAb (P-Tyr-100) Kit (CST #5636) and anti-phosphotyrosine immunoprecipitation was carried out following the manufacturerÃƒÂ¢Ã¢â€šÂ¬Ã¢â€žÂ¢s instructions.  For total protein analysis a 300 ÃƒÅ½Ã‚Â¼g aliquot of SILAC mix was also subjected to a separate in-solution tryptic digestion and was fractionated by basic-pH reversed-phase liquid chromatography into 24 fractions which were dried, desalted, and concentrated using in-house prepared C18 reversed-phase column. BioID with BioSITe Protein G agarose beads (Millipore Sigma, #16-266) were washed twice with PBS and 100 ÃƒÅ½Ã‚Â¼g of anti-biotin antibody (Bethyl Laboratories, Inc. A150-109A were coupled to 120 ÃƒÅ½Ã‚Â¼L of protein G bead slurry, pre-replicate, overnight at 4 Ãƒâ€šÃ‚Â°C. Antibody-coupled beads were further washed with PBS once and BioSITe capture buffer (50 mM Tris, 150 mM NaCl, 0.5% Triton X-100) twice. Lyophilized peptides were dissolved in 1 mL of BioSITe capture buffer and pH solution was adjusted to neutral (7.0 to 7.5). Peptides were subsequently incubated with anti-biotin antibody-bound protein G beads for 1 h at 4 Ãƒâ€šÃ‚Â°C. The bead slurry was sequentially washed three times with BioSITe capture buffer, three times with 50 mL of Tris, and two times with ultrapure water. Biotinylated peptides were eluted with four rounds of 200 ÃƒÅ½Ã‚Â¼L elution buffer (80% acetonitrile and 0.2% trifluoroacetic acid in water). The eluents were dried, desalted, and concentrated using homemade C18 reversed-phase column. Liquid-Chromatography/Mass Spectrometry Analysis: Phosphotyrosine-enriched peptide samples were reconstituted in 10ul of Solvent A (0.1% Formic Acid in water) and 8 ul of each sample was analyzed by nano-LC-MS/MS using Orbitrap Fusion Lumos (Thermo Scientific, San Jose) interfaced with Ultimate 3000 RSLCnano UPLC system (Thermo Scientific). Peptides were loaded on a pre-column (100 ÃƒÅ½Ã‚Â¼m id ÃƒÆ’Ã¢â‚¬â€� 5 mm, C18 PepMap100, 5 ÃƒÅ½Ã‚Â¼m, Thermo Scientific) at 8 ÃƒÅ½Ã‚Â¼L/min for 6 minutes using 0.1% Formic acid in water. After 6 min, the peptides were separated on an EasySpray column (75 ÃƒÅ½Ã‚Â¼m id ÃƒÆ’Ã¢â‚¬â€� 50 cm, C18 PepMap, 2ÃƒÅ½Ã‚Â¼m, Thermo Scientific) at 300 nL/min using solvent B (0.1% formic Acid in 95% Acetonitrile). The gradient consisted of initial step of 8ÃƒÂ¢Ã¢â€šÂ¬Ã¢â‚¬Å“32% B over 100 min followed by 32ÃƒÂ¢Ã¢â€šÂ¬Ã¢â‚¬Å“50% over 7 min, 32ÃƒÂ¢Ã¢â€šÂ¬Ã¢â‚¬Å“90%B over 3 min, held at 90%B for 5 min and then equilibrated for 20 min at 5% B, where mobile phase A consisted on water containing 0.1% formic acid and mobile phase B consisted of 95/5 acetonitrile/water containing 0.1% formic acid. Separation was performed at 45Ãƒâ€šÃ‚Â°C and the total acquisition time was 140 min. The mass spectrometer was fitted with an EasySpray source (Thermo Scientific) and operated in DDA manner. Each DDA cycle consisted of one OT MS survey scan acquired at 120,000 resolution at m/z 200 and precursors ions meeting user defined criteria such as charge state, monoisotopic precursor selection, intensity, and dynamic exclusion were selected for MS2 based on ÃƒÂ¢Ã¢â€šÂ¬Ã…â€œcycle timeÃƒÂ¢Ã¢â€šÂ¬Ã¯Â¿Â½ of 3 sec. Precursor ions were isolated using the quadrupole (1.4 Th isolation width with isolation offset of 0.4 Th) and fragmentation was carried out using higher-energy dissociation (HCD) method using normalized collision energy (NCE) of 32. The MS2 scans were detected in OT at 30,000 resolution at m/z 200.

### Data Protocol
Proteome Discoverer (v 2.2; Thermo Scientific) suite was used for quantitation and identification of peptides from LCÃƒÂ¢Ã¢â€šÂ¬Ã¢â‚¬Å“MS/MS runs. Spectrum selector was used to import spectrum from raw file. During MS/MS preprocessing, the top 10 peaks in each window of 100 m/z were selected for database search. The tandem mass spectrometry data were then searched using SEQUEST algorithm against protein databases (mouse NCBI RefSeq 73 (58039 entries) with the addition of fasta file entries; BirA*-p210-BCR-ABL (for p210 BioID with BioSITe/pTyr/total protein experiment), Ubash3b-BirA* and p210-BCR-ABL (For Ubash3b BioID with BioSITe experiment) and common contaminant proteins. The search parameters for identification of biotinylated peptides were as follows: (a) trypsin as a proteolytic enzyme (with up to three missed cleavages); (b) peptide mass error tolerance of 10 ppm; (c) fragment mass error tolerance of 0.02 Da; and (d) carbamido-methylation of cysteine (+57.02146 Da) as a fixed modification and oxidation of methionine (+15.99492 Da) and biotinylation of lysine (+226.07759 Da) as variable modifications. Peptides and proteins were filtered at a 1% false-discovery rate (FDR) at the PSM level using percolator node and at the protein level using protein FDR validator node, respectively. For SILAC experiments relative quantification of identified peptides was carried out using PyQuant by deriving peak areas of light and heavy isotopic peptides from MS1 level data in raw files. Then peak areas of identical peptides were merged to derive overall abundance estimates of samples being compared. Site level information of biotinylated peptides was then processed to derive abundance and degree of biotinylation at protein level.  For Ubash3b BioID experiment MS1 level quantification was carried out with the Minora Feature Detector, using the programÃƒÂ¢Ã¢â€šÂ¬Ã¢â€žÂ¢s standard parameters and all of the raw files from the three replicates were quantified together. Unique and razor peptides both were used for peptide quantification, while protein groups were considered for peptide uniqueness. Identified protein and peptide spectral match (PSM) level data were exported as tabular files from Proteome Discoverer 2.2. We used an in-house Python script to compile the peptide level site information mapped to RefSeq databases. We eliminated all non-biotinylated peptides from our analysis. The summary count on the number of supported peptides, PSMs, number of biotinylation sites and quantification was then calculated at the protein level.

### Publication Abstract
None

### Keywords
Bioid, Mouse, Biosite, Leukemia, Bcr-abl, Phosphoproteome

### Affiliations
Department of Laboratory Medicine and Pathology, Mayo Clinic, Rochester, MN 55905
Department of Laboratory Medicine and Pathology, Mayo Clinic, Rochester, MN 55905, USA

### Submitter
Akhilesh Pandey

### Lab Head
Dr Akhilesh Pandey
Department of Laboratory Medicine and Pathology, Mayo Clinic, Rochester, MN 55905, USA


