### Accession
PXD022378

### Title
Subcellular proteomics of murine ESC-derived neurons

### Description
We studied the effects of different stimuli on on the subcellular proteome of neurons differentiated from murine embryonic stem cells (ESC). We focused on the changes in the chromatin-bound fraction to elucidate specific differences in the gene regulatory machinery activated upon stimulation with Brain-derived neurotrophic factor (BDNF) or a general membrane-depolarization stimulus, potassium chloride (KCl).

### Sample Protocol
Subcellular proteome analysis by mass spectrometry Stimulated mESC-derived neurons were harvested after stimulation using a scraper and pelleted at 500 g for 5 minutes. Each stimulation condition was assayed in duplicates, and 10 million cells were used for each replicate. Cell pellets were washed twice with ice-cold PBS and subjected to subcellular protein extractions using a Subcellular Protein Fractionation for Cultured Cells kit (Thermo Fisher Scientific #78840) following manufacturerâ€™s instructions. Each subcellular protein fraction was denatured with 1% SDS at 95Â°C for 5 minutes, after which nucleic acids were degraded with benzonase (EMD Millipore, #71206-25KUN; final concentration 1â€“2 U/Âµl) for 45 minutes at 37Â°C and 300 rpm. Protein concentration was measured using PierceTM BCA protein assay kit (Thermo Fisher Scientific , #23225) and 10 Î¼g protein were subjected to sample preparation for MS using a modified SP3 protocol (https://doi.org/10.15252/msb.20145625). Briefly, protein samples were precipitated onto Sera-Mag SpeedBeads (GE Healthcare, #45152105050250 and #65152105050250) in the presence of 50% EtOH and 2.5% formic acid (FA) for 15 min at RT, followed by four washes with 70% ethanol on 0.22 Âµm filter plates (Millipore #MSGVN22). Proteins were digested on beads with trypsin and Lys-C (5 ng/Âµl final concentration each) in 90 mM HEPES (pH 8.5), 5 mM chloroacetic acid and 1.25 mM TCEP overnight at room temperature. Peptides were eluted by centrifugation using 2% DMSO and vacuum dried. Dry peptides were reconstituted in 10 Âµl water and labelled by adding 4 Âµl TMT label (20 Âµg/Âµl in acetonitrile (ACN)) (TMT10plex and TMT11, Thermo Fisher Scientific #90110, #A37724) and incubating 1 h at room temperature. Labeling was quenched with hydroxylamine (1.1% final concentration). For each replicate, labelled peptides from the following fractions were pooled to form a TMT11 set: unstimulated, BDNF-stimulated and KCL-stimulated, cytosolic (CEF), nuclear (NE), and chromatin-bound (CHR) fractions (channels 1-9), as well as a membrane fraction (channel 10) and a cytoskeletal fraction (channel 11) pooled from all three different conditions to increase coverage. Pooled peptides were desalted by loading them on an OASIS HLB ÂµElution plate (Waters 186001828BA), washing twice with 0.05% FA, eluting with 80% ACN, 0.05% FA, and vacuum dried. Dried peptides were taken up in 20 mM ammonium formate (pH 10) and prefractionated into 12 fractions on an Ultimate 3000 (Dionex) HPLC using high-pH reversed-phase chromatography (running buffer A: 20 mM ammonium formate pH 10; elution buffer B: ACN) on an X-bridge column (2.1 x 10 mm, C18, 3.5 Âµm, Waters). Prefractionated peptides were vacuum dried. For LC-MS/MS analysis, peptides were reconstituted in 0.1% FA, 4% ACN and analyzed by nanoLC-MS/MS on an Ultimate 3000 RSLC (Thermo Fisher Scientific) connected to a Q Exactive Plus (Thermo Fisher Scientific) mass spectrometer, using an Acclaim C18 PepMap 100 trapping cartridge (5Âµm, 300 Âµm i.d. x 5 mm, 100 Ã…) (Thermo Fisher Scientific) and a nanoEase M/Z HSS C18 T3 (100Ã…, 1.8 Âµm, 75 Âµm x 250 mm) analytical column (Waters). Solvent A: aqueous 0.1% formic acid; Solvent B: 0.1% FA in ACN (all solvents LC-MS grade from Fisher Scientific). Peptides were loaded on the trapping cartridge using solvent A for 3 min with a flow of 30 Âµl/min. Peptides were separated on the analytical column with a constant flow of 0.3 Âµl/min applying a 120 min gradient of 2â€“40% of solvent B in A. Peptides were directly analyzed in positive ion mode. Full scan MS spectra with a mass range of 375â€“1200 m/z were acquired in profile mode using a resolution of 70,000 (maximum fill time of 250 ms or a maximum of 3e6 ions (AGC). Precursors were isolated using a Top10 method with an isolation window of 0.7 m/z, fragmented using 30 NCE (normalized collision energy), and MS/MS spectra were acquired in profile mode with a resolution of 35,000, and an AGC target of 2e5 with a dynamic exclusion window of 30 s.

### Data Protocol
Mass spectrometry raw files were processed using IsobarQuant (https://pubmed.ncbi.nlm.nih.gov/26379230/) and peptide and protein identification was obtained with Mascot 2.5.1 (Matrix Science) using a reference mouse proteome (uniprot Proteome ID: UP000000589, downloaded 14.5.2016) modified to include known common contaminants and reversed protein sequences. Mascot search parameters were: trypsin; max. 2 missed cleavages; peptide tolerance 10 ppm; MS/MS tolerance 0.02 Da; fixed modifications: Carbamidomethyl (C), TMT16plex (K); variable modifications: Acetyl (Protein N-term), Oxidation (M), TMT16plex (N-term). IsobarQuant output data was analyzed on a protein level in R (https://www.R-project.org) using an in-house data analysis pipeline. In brief, protein data was filtered to remove contaminants, proteins with less than 2 unique quantified peptide matches, as well as proteins, which were only detected in a single replicate. Subsequently, protein reporter signal sums were normalized within each subcellular fraction across the two TMT sets (replicates) and across the three conditions using the vsn package (https://pubmed.ncbi.nlm.nih.gov/12169536/). Significantly changing proteins between BDNF and KCL stimulation conditions in the CHR subcellular fraction were identified by applying a limma analysis (https://pubmed.ncbi.nlm.nih.gov/25605792/) on the vsn-corrected signal sums. Replicate information was included as a covariate to adjust for batch effects caused by the separate TMT labelings and MS runs. T-statistics and p-values were obtained using the eBayes function from the limma package, and resulting p-values were corrected for multiple testing using the Benjamini-Hochberg method with the topTable function in the limma package.

### Publication Abstract
Neuronal stimulation induced by the brain-derived neurotrophic factor (BDNF) triggers gene expression, which is crucial for neuronal survival, differentiation, synaptic plasticity, memory formation, and neurocognitive health. However, its role in chromatin regulation is unclear. Here, using temporal profiling of chromatin accessibility and transcription in mouse primary cortical neurons upon either BDNF stimulation or depolarization (KCl), we identify features that define BDNF-specific chromatin-to-gene expression programs. Enhancer activation is an early event in the regulatory control of BDNF-treated neurons, where the bZIP motif-binding Fos protein pioneered chromatin opening and cooperated with co-regulatory transcription factors (Homeobox, EGRs, and CTCF) to induce transcription. Deleting cis-regulatory sequences affect BDNF-mediated Arc expression, a regulator of synaptic plasticity. BDNF-induced accessible regions are linked to preferential exon usage by neurodevelopmental disorder-related genes and the&#x2009;heritability of neuronal complex traits, which were validated in human iPSC-derived neurons. Thus, we provide a comprehensive&#x2009;view of BDNF-mediated genome regulatory features using comparative genomic approaches to dissect mammalian neuronal stimulation.

### Keywords
Thermo q exactive plus, Subcellular fractionation, Neurons, Bdnf, Kcl, Stimulation, Tmt

### Affiliations
Genome Biology Unit and Structural and Computational Biology Unit, European Molecular Biology Laboratory, 69117 Heidelberg, Germany
Genome Biology Unit, European Molecular Biology Laboratory, 69117 Heidelberg, Germany

### Submitter
Henrik Hammarén

### Lab Head
Dr Mikhail M Savitski
Genome Biology Unit, European Molecular Biology Laboratory, 69117 Heidelberg, Germany


