### Accession
PXD005241

### Title
A phosphoproteomic landscape of rice (Oryza sativa L.) tissues

### Description
Here, we first reported the construction of a phosphoproteomic landscape of 6 tissues, including callus, leaves, roots, shoot meristem (SM), young panicles (YP) and mature panicles (MP), from Nipponbare (Oryza sativa ssp. japonica). By employing a non-gel, quantitative phosphoproteomic approach, a total of 4792 phosphopeptides from 2657 phosphoproteins were identified, which were found to be differentially phosphorylated among tissues.

### Sample Protocol
The protein digestion and phosphopeptide enrichment were performed as described previously (Qiu et al. 2016a).Firstly, proteins were treated for a sequential of 5 mM DTT reduction, 20 mM IAA alkylation and 5 mM DTT reduction in the dark , and digested with trypsin at pH8.0 (enzyme:protein =1:50) on the 30kDa filter unit (Millipore, Darmstadt, Germany) over night. After the filter-aided sample preparation (FASP), C18 Sep-Pak (Waters) was applied to desalted the peptides, and binding buffer (80% ACN, 5% TFA, 1M lac acid) was used to resolved the peptides. Subsequently, TiO2 beads (GL sciences, Tokyo, Japan) were employed to incubate the peptides with a ratio of TiO2:peptide:=4:1 for 30 minutes three times. Then the peptides were washed with binding buffer, and eluted with the elution buffer (40% ACN, 15% NH3-H2O) four times. The eluates were subsequently vacuum dried using  a SpeedVac and reconstituted with 5% MeOH in 1% TFA solution for LC-MS/MS analysis.

### Data Protocol
The LC-MS/MS and data analysis were performed in Beijing Proteome Research Center as previously reported (Qiu et al. 2016a). The detailed procedures were described as follows. Firstly, a linear 5-30% acetonitrile gradient on an Easy-nLC1000 liquid chromatography system (Thermo, Waltham, U.S.A.) was used to separated the enriched phosphopeptides. Then a Q Exactive Plus spectrometer (Thermo, Waltham, U.S.A.) (mass range: 300-1400 m/z; resolution: 70000) was employed to perform full-scan mass spectra identification. Secondly, Mascot search engine against the rice genome annotation project database: (ftp://ftp.plantbiology.msu.edu/pub/data/Eukaryotic_Projects/o_sativa/annotation_dbs/pseudomolecules/version_7.0/all.dir/) was applied to identify phosphopeptide and quantify phosphosite through the raw spectral data. Notably, in the Mascot searches for phosphopeptides, Oxidation (M), Acetyl (Protein-N term) and Phospho(S/T/Y) were set as variable modifications and one missed cleavage on trypsin was allowed. And the Percolator tool of the Protein Discoverer package was used to identify peptide with FDR (False discovery rates) set to <1%. After that, the phosphoRS software within the Protein Discoverer software suite was performed to automatically re-analyze all phosphopeptide hits. Only the PhosphoRS probability higher than 90% was considered as a phosphorylation site and those peptides which were phosphorylated at least two of the three biological replicates were determined as truly phosphopeptides. The phosphorylated proteins having over two fold changes in the average intensity with Studentâ€™s t-test (P<0.05) were defined as differentially phosphorylated proteins.

### Publication Abstract
Protein phosphorylation is an important posttranslational modification that regulates various plant developmental processes. Here, we report a comprehensive, quantitative phosphoproteomic profile of six rice tissues, including callus, leaf, root, shoot meristem, young panicle and mature panicle from Nipponbare by employing a mass spectrometry (MS)-based, label-free approach. A total of 7171 unique phosphorylation sites in 4792 phosphopeptides from 2657 phosphoproteins were identified, of which 4613 peptides were differentially phosphorylated (DP) among the tissues. Motif-X analysis revealed eight significantly enriched motifs, such as [sP], [Rxxs] and [tP] from the rice phosphosites. Hierarchical clustering analysis divided the DP peptides into 63 subgroups, which showed divergent spatial-phosphorylation patterns among tissues. These clustered proteins are functionally related to nutrition uptake in roots, photosynthesis in leaves and tissue differentiation in panicles. Phosphorylations were specific in the tissues where the target proteins execute their functions, suggesting that phosphorylation might be a key mechanism to regulate the protein activity in different tissues. This study greatly expands the rice phosphoproteomic dataset, and also offers insight into the regulatory roles of phosphorylation in tissue development and functions.

### Keywords
Phosphoproteomic rice  tissues

### Affiliations
China National Rice Research Institute
China national rice research institute

### Submitter
Jian Zhang

### Lab Head
Dr Jian Zhang
China National Rice Research Institute


