### Accession
PXD027855

### Title
Comparison of Cell Wall Proteomics of Drug Resistant M. smegmatis with and without Sub-inhibitory Rifampicin Exposure

### Description
We previously showed sub-lethal exposure to the front-line anti-TB drug rifampicin resulted in substantial adaptive remodelling of the proteome of the model organism M. smegmatis in the drug sensitive mc2155 strain (WT). Here we investigate whether these responses are conserved in an engineered, isogenic M. smegmatis mc2155 mutant harbouring the clinically relevant S531L rifampicin resistance-conferring mutation (SL) and distinguish responses that are specific to rifampicinâ€™s anti-bacterial activity from those that are dependent on rifampicin's presence alone. We used a cell wall enrichment strategy to focus attention on the cell wall proteome and observed 253 proteins to be dysregulated in SL bacteria as compared with 716 proteins in WT. We observed that down regulation of ABC transporters and up regulation of ribosomal machinery were conserved in the SL strain, while up regulation of transcriptional machinery, and down regulation of numerous two-component systems, were not.

### Sample Protocol
Drug resistant mutant strain was generated from M. smegmatis mc2155 by engineered mutation of codon 531 in the RRDR from serine to leucine (S531L) and validated by sequencing.   Bacteria were grown in 7H9 broth supplemented with 0.1% Tween-20, 0.2% glycerol and 10% OADC with four flasks per treatment group, yielding 8 total samples. On reaching an optical density (A600Â nm) of 1.2 the cultures were treated with either 2.5 Âµg/mL rifampicin in DMSO (treated) or with DMSO only (control). The bacterial cultures were harvested at 255 minutes post-treatment by transferal of each culture to two 50 mL centrifuge tubes and centrifugation at 3000 RCF at 8°C for 15 minutes. Pellets were washed twice by re-suspension in PBS, snap frozen via liquid nitrogen and stored at -80°C before lysis and cell wall enrichment.  Washed cell pellets were resuspended in lysis buffer (PBS pH 7.4, protease inhibitors) and lysed by sonication for six rounds of 30 seconds each with 1-minute cooling in an ice/water slurry in between. Cell debris were pelleted by centrifugation at 8,000 RCF for 10 minutes and the lysate filtered through a 0.2 µm filter before pelleting the cell wall fraction by centrifugation at 22,000 RCF for 30 minutes at 4Â°C. The supernatant containing the cytosolic fraction was precipitated with chloroform-methanol and the protein pellet solubilised in UA buffer (8 M urea, 0.1 M Tris, pH 8.5). The cell wall pellet was washed twice with PBS and resuspended in 0.1% Triton X100, 0.05% Tween-20 and 0.2% CHAPS. Insoluble aggregates were removed by centrifugation at 22,000 RCF for 10 minutes.  The residual cell debris pellets were washed once with PBS and resuspended in 0.1 M Tris, 2% SDS and 100 mM DTT, pH 8.5, and boiled at 95°C for one hour. The samples were cooled to room temperature on ice and mixed with urea to a final concentration of 6M by shaking for 30 minutes at room temperature.  Proteome fractions were precipitated via chloroform/methanol and solubilised in UA buffer prior to FASP digestion. Centrifugation steps were performed at 14,000 RCF for 15 minutes. 50 µg of sample were added into separate Ultracel 30,000 MWCO centrifugal units and washed three times with UA buffer. Proteins were reduced by addition of 100 mM DTT for 30 minutes at room temperature and excess DTT was removed by centrifugation. Reduced cysteines were alkylated by addition of 0.05 M iodoacetamide for 20 minutes in the dark at room temperature. Alkylating reagent was removed by three rounds of buffer exchange with UA buffer followed by three rounds of buffer exchange into ABC buffer (0.05 M ammonium bicarbonate pH 8.0, 20 mM CaCl2).  Sequence-grade Trypsin (NEB) was added at an enzyme to protein ratio of 1:50 and digestion proceeded for 18 hours at 37°C in a humidified chamber. Tryptic peptides were eluted by three rounds of buffer exchange with ABC buffer and desalted using home-made STAGE tips containing Empore Octadecyl C18 solid-phase extraction disks (Supelco). C18 disks were activated with three washes solvent A (80% acetonitrile, 0.1% formic acid) and equilibrated with three washes with solvent B (2% acetonitrile, 0.1% formic acid). 10 µg of tryptic peptides was loaded on the C18 disc, centrifuged at 4,000 x g for 1 minute and washed three times with solvent B. Peptides were then eluted three times with solvent C (60% acetonitrile, 0.1% formic acid) into glass capillary tubes. Eluted peptides were dried in a vacuum and resuspended with solvent A (2% acetonitrile, 0.1% formic acid) at a concentration of 200 ng/ÂµL.  Chromatographic separation was achieved using a 100 µm ID, 20 mm pre-column connected to a 75 µm, 300 mm analytical column packed with C18 Luna beads connected to an Ultimate 3500 RS nano UPLC system (Dionex). ~600 ng of desalted peptides per sample were loaded onto the column with a starting mobile phase of 2% ACN, 0.1% formic acid and separated at a constant flow rate of 400 nL/min over a 70 minute gradient as follows: 10 minutes at 2% ACN, increase to 6% ACN over 2 minutes, to 40% ACN over 70 mins, to 80% ACN over 5 minutes and held at 80% for 15 minutes as a column wash. Mass spectra were collected on a Q Exactive mass spectrometer (Thermo) operating in positive mode, with data-dependent acquisition and a top-10 method. Intensity threshold for MS2 ion selection was 1.3e4 with charge exclusion of z = 1 and z > 5. Peptides were ionised by ESI and MS spectra acquired at a resolution of 70,000 MS1 and 17,500 MS2. AGC target was set to 3e6 with maximum integration time of 250 ms (MS1) or 1e5 with a maximum integration time of 80 ms (MS2). MS1 Scan range was 300 to 1750 Dalton and peptide fragmentation was performed with HCD energy set to 28 NCE and product ion scan range was 200 to 2000 Dalton.

### Data Protocol
RAW data was processed with Maxquant version 1.5.3.12 for protein and peptide identification using the Andromeda search engine and the Uniprot proteome for M. smegmatis. Default parameter settings were used for the MS/MS database search, with carbamidomethylation of cysteine residues and acetylation of the protein N-terminus selected as fixed modifications. Trypsin/P was selected as protease, iBAQ quantitation and the match between runs feature was enabled. Protein identifications were required to pass an FDR of 1% with one or more unique peptides for inclusion in further analysis.

### Publication Abstract
None

### Keywords
Rifampicin, Proteomics, Mycobacterium smegmatis, Phenotypic adaption, Abc transporters, Mass spectrometry

### Affiliations
University of Cape Town
Department of Chemical and Systems Biology, Institute of Infectious Disease and Molecular Medicine, Faculty of Health Sciences, University of Cape Town, South Africa

### Submitter
Alexander Giddey

### Lab Head
Dr Jonathan M Blackburn
Department of Chemical and Systems Biology, Institute of Infectious Disease and Molecular Medicine, Faculty of Health Sciences, University of Cape Town, South Africa


