### Accession
PXD033148

### Title
human lung adenocarcinoma cell lines TMT based proteomics

### Description
In our study, we found that profilin 1 (PFN1) promoted non-small cell lung cancer (NSCLC) metastasis.For further investigate the mechanisms of PFN1's roles in NSCLC metastasis, we constructed PFN1 overxpression H1299 cell lines(H1299 PFN1 OE cells). And we sent samples of H1299 PFN1 OE cells and EV cells for TMT based quantative proteome profiling.

### Sample Protocol
H1299 EV and PFN1 OE cells were harvested at logarithmic growth period and each cell type was collected in biological triplicate. Total cellular protein extracted by RIPA lysis buffer and concentration of proteins were determined by bicinchoninic acid (BCA) protein assay kit (Beyotime) according to manufacturer instructions. A total of 100ug proteins of each sample were then precipitated by acetone at -20‚ÑÉ overnight. Dissolve protein precipitation by water bath ultrasound for 3 min after adding 100 ùúáL protein resolve buffer. Dithiothreitol (DTT) (Sigma-Aldrich) was used to reduced disulfide bond. Then reduced disulfide bonds were Alkylated by iodoacetamide (IAA)(Sigma-Aldrich). Thoroughly  mixed trypsin with the samples in the ratio of trypsin: protein = 1:50 and incubate overnight at 37 ‚ÑÉ 1000 rpm to digest proteins. Peptides were labeled by TMT Isobaric Label Reagent Set (Thermo Fisher Scientific, Waltham, MA,USA) according to manufacturer instructions. After SDS cleanup, peptide desalting and  gigh-pH pre-fractionation, peptides were freeze at - 80 ‚ÑÉ after vacuum drying and ready for later nanoLC-MS/MS analysis. For each sample, 2ug of total peptides were separated and analyzed with a nano-UPLC (EASY-nLC1200) coupled to a Q Exactive HFX Orbitrap instrument (Thermo Fisher Scientific) with a nano-electrospray ion source. Separation was performed by using a reversed-phase column (100 ùúá ID √ó15 cm, Reprosil-Pur 120 C18-AQ, 1.9 ùúá, Dr. Maisch). Mobile phases were H2O with 0.1% FA, 2% ACN (phase A) and 80% ACN, 0.1% FA (phase B). Separation of sample was executed with a 90 min gradient at 300 nL/min flow rate. Gradient B: 2-5% for 2 min, 5-22% for 68 min, 22-45% for 16 min, 45-95% for 2 min, 95% for 2 min. Data dependent acquisition (DDA) was performed in profile and positive mode with Orbitrap analyzer at a resolution of 120,000 (@200 m/z) and m/z range of 350-1600 for MS1. For MS2, the resolution was set to 15k with a fixed first mass of 110 m/z. The automatic gain control (AGC) target for MS1 was set to 3E6 with

### Data Protocol
After pretreatment, 5438 detected proteins were retained. Data were processed with Proteome Discoverer software (Thermo Fisher Scientific, version 2.4.0.305). Principal component analysis (PCA) was performed using the R package. (version3.6.3, https://www.r-project.org/) (Zhang et al., 2018) or SIMCA (V16.0.2, Sartorius Stedim Data Analytics AB, Umea, Sweden). Differentially expressed proteins (DEPs) were defined as student‚Äôs t-test p-value < 0.05 and fold changeÔºàfold change <0.83 or > 1.2Ôºâ. DEPs were visualized in the form of volcano plot. Hierarchical clustering for representing the DEPs was conducted by R Package pheatmap. The eukaryotic clusters of orthologous groups (KOG) database (http://www.ncbi.nlm.nih.gov/COG/) of protein database was carried out for functional classification of DEPs. Gene ontology (GO) database (http://geneontology.org/) and a Kyoto Encyclopedia of Genes and Genomes (KEGG) database(https://www.kegg.jp/) were used to classify and annotate the functions and pathways of differentially expressed proteins. All enrichment analyses were based on the Fisher's exact test with Benjamini‚àíHochberg correction (p < 0.05)

### Publication Abstract
None

### Keywords
Human, Lung adenocarcinoma cell line, Tmt

### Affiliations
Department of Respiratory Medicine, Xiangya Hospital, Central South University
Xiangya Hospital, Central South University

### Submitter
Ya Wang

### Lab Head
Dr Chengping Hu
Department of Respiratory Medicine, Xiangya Hospital, Central South University


