### Accession
PXD037586

### Title
nitric oxide-reducing microbial community metaproteomics

### Description
Here, we successfully used NO as the direct electron acceptor for the enrichment of a microbial community in a continuous bioreactor. The enrichment culture, mainly comprised of two new organisms from the Sterolibacteriaceae family, grew on NO reduction to N2 and formate oxidation, with virtually no accumulation of N2O. The microbial growth kinetics of the enrichment culture as well as its affinity for different N-oxides were determined. In parallel, using metagenomics, metatranscriptomics, and metaproteomics, the biochemical reactions underlying the growth of these microorganisms on NO were investigated. This study demonstrates that microorganisms thrive and can be enriched on NO, and presents new opportunities to study microbial growth on this highly energetic and climate-active molecule that may have been pivotal in the evolution of aerobic respiration.

### Sample Protocol
On day 1548, 15 ml of cell culture were collected, pelleted by centrifugation at 7942 x g and 4 Â°C for 20 min, and stored at âˆ’80 Â°C. Cell pellets were resuspended in 100 Âµl milliQ water and heated at 95 Â°C for 5 min with agitation prior to sonication for 10 min in a sonication bath (Branson 5800). Samples were centrifuged at 14,000 x g for 20 min at 4 Â°C and supernatant was transferred to a new Eppendorf tube as â€œsolubleâ€� extracts and RapiGest SF (Waters) was added to a final concentration of 0.1% (v:v). Pellets were washed with 100 Âµl milliQ water prior to addition of 50 Âµl 2% (v:v) RapiGest SF solution and heated at 95 Â°C for 5 min. Samples were reduced with bond-breaker TCEP solution (Thermo fisher scientific) for 20 min prior to alkylation with 50 mM chloroacetamide for 20 min at room temperature in the dark. Enzymatic digestion was performed by incubating protein extracts overnight at 37 Â°C with 0.4 Âµg LysC (Wako chemicals) and 0.4 Âµg Trypsin (Promega). RapiGest SF was removed by incubating tryptic digests in 0.5% trifluoroacetic acid at 37 Â°C for 40 min and subsequent centrifugation at 14,000 x g for 20 min at 4 Â°C. Samples were analyzed both before and after salt-mediated organic solvent precipitation(Baghalabadi et al., 2021).

### Data Protocol
Tryptic digests were measured in duplo by nanoflow liquid chromatography (LC; nano-Elute, Bruker Daltonics) with online ion mobility spectrometry-tandem mass spectrometry (IMS-MS/MS; timsTOF Pro 2, Bruker Daltonics) and real-time protein identification technology using the parallel database search engine in real-time (PaSER v2023, Bruker Daltonics). Peptides were loaded directly onto the C18 reversed phase analytical column (Bruker FIFTEEN 0.075 x 150 mm, 1.9 Âµm particles, 120 Ã… pore size, C18 ReproSil AQ; Bruker Daltonics) at a constant pressure of 800 bar and separated using a 60 minutes-long linear gradient of 3â€“45% acetonitrile in 0.1% formic acid with 0.02% trifluoroacetic acid at 500 nl minâˆ’1 at 45 Â°C. Peptides eluting from the column were analyzed by parallel accumulation serial fragmentation in data dependent acquisition mode (dda-PASEF)(Meier et al., 2018) using the default 1.1 seconds duty cycle method (mass range: 100â€“1700 m/z, mobility range: 0.6-1.6 1/K0, accumulation time: 100 ms, ramp time: 100 ms, PASEF cycles: 10, dynamic exclusion 0.4 min). Acquired IMS-MS/MS spectra were streamed directly to the PaSER box for real-time database searching using the ProLuCID search engine(Xu et al., 2015). Fragmentation spectra were searched against a custom protein sequence database with the following settings: 20 ppm precursor mass tolerance, 30 ppm fragment ion mass tolerance, strict tryptic cleavage, maximum of 4 missed cleavages, carbamidomethyl (C) as fixed modification, deamidation (NQ) and oxidation (M) and formylation (K) as variable modifications. Individual search results were combined and validated using DTA select with TIMScore enabled to achieve â‰¤1% false discovery rate at protein level.

### Publication Abstract
None

### Keywords
Metaproteomics

### Affiliations
Translational Metabolic Laboratory, Radboudumc, Nijmegen, The Netherlands
Radboudumc

### Submitter
Hans Wessels

### Lab Head
Dr Hans Wessels
Translational Metabolic Laboratory, Radboudumc, Nijmegen, The Netherlands


