### Accession
PXD041944

### Title
Multiomics analysis of naturally efficacious LNP coronas reveals high-density lipoprotein is necessary for LNP function. VLDLHDL corona

### Description
The MS data set contains the result from HDL VLDL corona used in the publication, titled Multiomics analysis of naturally efficacious LNP coronas reveals high-density lipoprotein is necessary for LNP function.

### Sample Protocol
Protein digestion was performed on recovered LNPs from individual plasmas containing an equal amount of mRNA. Briefly, sample denaturation and reduction were performed using a 30 min one-step 8M urea (#U1250, Merck) and TCEP bond-breaker solution (#77720, Thermo Fisher Scientific), followed by a 30 min alkylation step using a 2-chloroacetamide reagent (#22790, Merck). Protein digestion was done overnight in trypsin (#EMS0004, Merck) and ceased by the addition of formic acid. Digestions were measured using a Q-Exactive HF mass spectrometer (Thermo Fisher Scientific) coupled with an Evosep One (Evosep) automatic sample loader equipped with Evotip disposable C18 trap columns (Evosep) for in-line peptide desalting and purification immediately prior to analytical column separation with a preset, 30 samples per day (30-SPD), loading sequence. Briefly, purified peptides were separated on an 8 cm analytical reverse-phase column (Evosep) with gradient off-set focusing to achieve a 3%-40% acetonitrile within a 44 min loop at a 0.5 ÂµL/min flow rate.

### Data Protocol
MS raw files were analyzed by MaxQuant software (v1.6.6.0). Proteins were identified using the Uniprot FASTA database (Rattus norvegicus, June 2019) with N-terminal acetylation and methionine oxidations as variable modifications, and cysteine carbamidomethylation as a fixed modification. The false discovery rate (FDR) was set to 1% by reverse database search for both proteins and peptides with a minimum length of 7 amino acids. Enzyme specificity was set as trypsin (cleavage at C-terminal to arginine and lysine). A maximum of 2 missed cleavages was allowed in the database search. Peptide identification was performed with an initial precursor mass deviation tolerance up to 6 ppm and a main mass deviation tolerance to 20 ppm. Matching between runs was performed among samples. Proteins matching to the reversed database were filtered out. For protein quantification, MaxQuant computes raw protein intensities as the sum of all identified peptide intensities. Label-free quantification (LFQ) and intensity-based absolute quantification (iBAQ) were calibrated from raw protein intensities with a minimum peptide ratio count of 1. To compute protein relative abundance (PRA), intensity-based absolute quantification (iBAQ) obtained from raw protein intensities were divided by the number of theoretically observable peptides calculated using in silico protein digestion. Then, iBAQ intensities of identified protein were used to calculate PRA. To compare each identified protein between samples, statistical analyses were first performed with the Perseus software (v1.6.2.3) using LFQ intensities. A valid value threshold was first applied to identify significantly altered corona proteins among all identified proteins. Only proteins identified within all conditions and replicates, followed by a multi-sample ANOVA test, are considered valid hits (significance cutoff: permutation-based FDR <5%). Further hierarchical clustering analysis was performed with JMP (v15.0.0). OPLS analysis was performed with SIMCA (v16.0.1), using z-score normalized LFQ values. The Pearson correlation between single corona component and cellular eGFP was carried out with Prism GraphPad (v9.0.0). The correlation coefficient r was controlled by nonparametric (Spearman) correlation P value.

### Publication Abstract
None

### Keywords
Vldl hdl corona

### Affiliations
AstraZeneca

### Submitter
Kai Liu

### Lab Head
Dr Kai Liu
AstraZeneca


