### Accession
PXD020461

### Title
Multi-level proteomics reveals host-perturbation strategies of SARS-CoV-2 and SARS-CoV

### Description
The  sudden  global  emergence  of  SARS-CoV-2  urgently  requires  an  in-depth  understanding  of  39 molecular  functions  of  viral  proteins  and  their  interactions  with  the  host  proteome.  Several  omics studies  have  extended  our  knowledge  of  COVID-19  pathophysiology,  including  some  focused  on  proteomic aspects1–3. To understand how SARS-CoV-2 and related coronaviruses manipulate the host we here characterized interactome, proteome and signaling processes in a systems-wide manner. This identified  connections  between  the  corresponding  cellular  events,  revealed  functional  effects  of  the  individual  viral  proteins  and  put  these  findings  into  the  context  of  host  signaling  pathways.  We  investigated  the  closely  related  SARS-CoV-2  and  SARS-CoV  viruses  as  well  as  the  influence  of  SARS-CoV-2  on  transcriptome,  proteome,  ubiquitinome  and  phosphoproteome  of  a  lung-derived  human  cell  line.  Projecting  these  data  onto  the  global  network  of  cellular  interactions  revealed  48 relationships  between  the  perturbations  taking  place  upon  SARS-CoV-2  infection  at  different  layers  and  identified  unique  and  common  molecular  mechanisms  of  SARS  coronaviruses.  The  results  highlight  the  functionality  of  individual  proteins  as  well  as  vulnerability  hotspots  of  SARS-CoV-2,  which  we  targeted  with  clinically  approved  drugs.  We  exemplify  this  by  identification  of  kinase  inhibitors as well as MMPase inhibitors with significant antiviral effects against SARS-CoV-2

### Sample Protocol
Samples  were  loaded  onto  a  15  cm  reversed  phase  column  (150  μm  inner  diameter,  packed  in  house  628 with  ReproSil-Pur  C18-AQ  1.9  μm  resin  [Dr.  Maisch  GmbH]),  which  was  kept  in  a  homemade  column  oven  at  60°C.  Peptides  were  separated  by  the  Evosep  One  LC  system  using  the  pre-programmed   44   minutes   gradient   for   proteome   samples   and   the   21   minutes   gradient   for   phosphoproteome  samples.  The  same  gradients  were  used  for  the  acquisition  of  proteome  and  phosphoproteome  library  fractions.  The  Evosep  One  system  was  coupled  to  a  Q  Exactive  HF-X  Orbitrap (Thermo Fisher Scientific) via a nano-electrospray source. The  proteome  and  phosphoproteome  fractions  we  used  to  build  the  libraries  were  measured  in  DDA  mode.  To  acquire  proteome  fractions  the  mass  spectrometer  was  operated  in  Top15  data-dependent mode  with  a  full  scan  range  of  300-1650  m/z  at  60,000  resolution,  an  automatic  gain  control  (AGC) target of 3e6 and a maximum fill time of 20ms. For the generation of the phosphoproteome library the 638 mass spectrometer was operated in Top12 data-dependent mode (DDA) with a full scan range of 300-1650 m/z at 60,000 resolution, an automatic gain control (AGC) target of 3e6 and a maximum fill time of  25ms.  For  both  libraries  precursor  ions  were  isolated  with  a  width  of  1.4  m/z  and  fragmented  by  higher-energy  collisional  dissociation  (HCD)  (NCE  27%).  Fragment  scans  were  performed  at  a  resolution  of  15,000,  an  AGC  of  1e5  and  a  maximum  fill  time  of  28  ms.  Dynamic  exclusion  was  enabled and set to 30 s for the proteome library and 20 s for the phosphoproteome library. For proteome and phosphoproteome DIA measurements, full MS resolution was set to 60,000 with a full scan range of 300-1650 m/z, a maximum fill time of 60 ms and an automatic gain control (AGC) target of 3e6. One full scan was followed by 32 windows with a resolution of 30,000 and a maximum fill  time  of  54  ms  for  proteome  measurements  and  40  windows  for  phosphoproteome  measurements with  a  resolution  of  15,000  and  maximum  fill  time  of  28  ms  in  profile  mode.  Precursor  ions  were  fragmented by higher-energy collisional dissociation (HCD) (NCE 27%).

### Data Protocol
For  the  experiments  conducted  in  DIA  mode,  Spectronaut  version  13  (Biognosys)  was  used  to  generate  the  proteome  and  phosphoproteome  libraries  from  DDA  runs  by  combining  files  of  respective fractionations using the human fasta file (Uniprot, 2019, 42,431 entries). For the generation of  the  proteome  library  default  settings  were  left  unchanged.  For  the  phosphoproteome  library  generation 2 x 24 files received by both fractionation strategies were combined and phosphorylation at Serine/Threonine/Tyrosine  was  added  as  variable  modification  to  default  settings.  Maximum  number  of fragment ions per peptide was increased from 6 to 25. Proteome DIA files were analyzed using the proteome  library  with  default  settings  and  disabled  cross  run  normalization.  Phospho  DIA  files  were  analyzed  using  the  phosphoproteome  library  using  default  settings  with  disabled  PTM  localization  filter and cross run normalization. To search for viral proteins, we also generated the “hybrid” spectral library by merging DDA proteome library with a direct-DIA library generated from the DIA analysis of  DIA  proteome  samples.  For  this  search,  the  sequences  of  viral  proteins  were  added  to  the  human  fasta file.

### Publication Abstract
None

### Keywords
Sars-cov-2, Covid-19, Proteomics, Dia, Mass spectrometry

### Affiliations
Max Planck Institute of Biochemistry
Proteomics

### Submitter
Mario Oroshi

### Lab Head
Dr Matthias Mann
Max Planck Institute of Biochemistry


