### Accession
PXD023584

### Title
Exome reanalysis and proteomic profiling identified TRIP4 as a novel cause of cerebellar hypoplasia and spinal muscular atrophy (PCH1)

### Description
TRIP4 is one of the subunits of the transcriptional coregulator ASC-1, a ribonucleoprotein complex that participates in transcriptional coactivation and RNA processing events.  Recessive variants in the TRIP4 gene have been associated with spinal muscular atrophy with bone fractures as well as a severe form of congenital muscular dystrophy. Here we present the diagnostic journey of a patient with cerebellar hypoplasia and spinal muscular atrophy (PCH1) and congenital bone fractures. Initial exome sequencing analysis revealed no candidate variants. Reanalysis of the exome data by inclusion in the Solve-RD project resulted in the identification of a homozygous stop-gain variant in the TRIP4 gene, previously reported as disease-causing.  This highlights the importance of analysis reiteration and improved and updated bioinformatic pipelines. Proteomic profile of the patient’s fibroblasts showed altered RNA-processing and impaired exosome activity supporting the pathogenicity of the detected variant. In addition, we identified a novel genetic form of PCH1, further strengthening the link of this characteristic phenotype with altered RNA metabolism.

### Sample Protocol
Snap-frozen fibroblasts were lysed in a buffer containing 1% SDS, 50 mM Tris, 150 mM NaCl, pH 7.8 and cOmplete™ ULTRA protease inhibitor using the Bioruptor® (Diagenode) for 10 minutes (30 seconds on, 30 seconds off, 10 cycles) at 4 °C. Afterwards, 20 µl of each sample was taken and diluted 1:4 with 10 mM ammoniumbicarbonate buffer, pH 7.8 (ABC) to conduct a BCA-based determination of protein concentration according to the manufacturer’s instructions (Pierce BCA protein assay kit). Reduction and carbamidomethylation of the remaining samples were carried out using 10 mM Tris-(2-carboxyethyl)-phosphin (TCEP) for 30 min at 37  C followed by application of 15 mM Iodacetamide (IAA) for further 30 min at room temperature (RT). Samples were further processed using the S-Trap™ (Protifi) sample preparation procedure: after acidifying the samples by adding 12% aqueous phosphoric acid, they were diluted with S-Trap binding buffer (90% methanol (MeOH), triethylammonium bicarbonate (TEAB) 100 mM, pH 7.1). Protein-loading to S-Trap columns including centrifugation steps was performed according to the manufacturer’s instructions. Filter-based tryptic digestion was carried out for 2 h at 47 °C using a trypsin to protein ratio of 1:20. Afterwards, peptides were eluted by applying multiple eluting steps starting with 10 mM ABC followed by elution with 0.1% formic acid (FA) and last with 80% acetonitrile (ACN). Drying of the eluted peptides was performed in a vacuum concentrator followed by dissolving of peptides in 0.1% TFA for subsequent LC-MS/MS analysis.

### Data Protocol
For the analysis of the samples acquired with nano-LC-MS/MS in DIA mode, the data was introduced to the Spectronaut software and analyzed with a library-based search. As library, an in house created spectral library was used. Search and extraction settings were kept as standard (BGS Factory settings). As background proteome, same data were used as selected for the establishment of the spectral library.

### Publication Abstract
TRIP4 is one of the subunits of the transcriptional coregulator ASC-1, a ribonucleoprotein complex that participates in transcriptional coactivation and RNA processing events. Recessive variants in the TRIP4 gene have been associated with spinal muscular atrophy with bone fractures as well as a severe form of congenital muscular dystrophy. Here we present the diagnostic journey of a patient with cerebellar hypoplasia and spinal muscular atrophy (PCH1) and congenital bone fractures. Initial exome sequencing analysis revealed no candidate variants. Reanalysis of the exome data by inclusion in the Solve-RD project resulted in the identification of a homozygous stop-gain variant in the TRIP4 gene, previously reported as disease-causing. This highlights the importance of analysis reiteration and improved and updated bioinformatic pipelines. Proteomic profile of the patient's fibroblasts showed altered RNA-processing and impaired exosome activity supporting the pathogenicity of the detected variant. In addition, we identified a novel genetic form of PCH1, further strengthening the link of this characteristic phenotype with altered RNA metabolism.

### Keywords
Proteogenomics, Exome reanalysis, Pch1, Trip4, Solve-rd

### Affiliations
Leibniz Institut für Analytische Wissenschaften
Department of Bioanalytics, Leibniz-Institut für Analytische Wissenschaften ISAS e.V., Dortmund, Germany.

### Submitter
Andreas Hentschel

### Lab Head
Dr Andreas Hentschel
Department of Bioanalytics, Leibniz-Institut für Analytische Wissenschaften ISAS e.V., Dortmund, Germany.


