### Accession
PXD037741

### Title
Mitochondrial dysfunction rapidly modulates the abundance and thermal stability of cellular proteins

### Description
Cellular functionality relies on a well-balanced, but highly dynamic proteome. Dysfunctionof mitochondrial protein import leads to the cytosolic accumulation of mitochondrial precursor proteins which compromise cellular proteostasis and trigger the mitoprotein-induced stress response. To dissect the effects of mitochondrial dysfunction on the cellular proteome as a whole, we developed pre-post thermal proteome profiling (ppTPP). This multiplexed time-resolved proteome-wide thermal stability profiling approach with isobaric peptide tags in combination with a pulse SILAC labeling elucidated dynamic proteostasis changes in several dimensions: In addition to adaptations in protein abundance, we observed rapid modulations of the thermal stability of individual cellular proteins. Strikingly, different functional groups of proteins showed characteristic response patterns and reacted with group-specific kinetics, allowing the identification of the functional modules that are relevant for mitoprotein-induced stress. Thus, our new ppTPP approach uncovered a complex response network that orchestrates proteome homeostasis in eukaryotic cells by time-controlled adaptations of protein abundance and protein stability.

### Sample Protocol
Yeast culture of an arginine and lysine auxotroph yeast strain YPH499 ÃŽâ€�arg4 with clogger or cytosolic DHFR plasmid was grown in synthetic minimal medium containing 2% lactate as carbon source and Ã¢â‚¬ËœlightÃ¢â‚¬Ëœ isotopes of arginine (Arg0, 12C6/14N4) and lysine (Lys0, 12C6/14N2).In mid-logarithmic growth phase, the medium was removed by centrifugation, cells were washed once in medium without arginine or lysine and resuspended in medium containing Ã¢â‚¬ËœheavyÃ¢â‚¬â„¢ arginine (Arg10, 13C6/15N4) and lysine (Lys8, 13C6/15N2) plus 0.5% Gal to induce the clogger or the cytosolic DHFR control. Samples were collected by centrifugation before induction (0 h) or after induction for 30, 90 and 270 min. For the TPP experiment harvested cells from dynamic SILAC shift were used directly. Cells were washed with PBS, resuspended in the same buffer and aliquoted to eleven wells of a PCR plate. After brief centrifugation, the plate was subjected to a temperature gradient (37Ã‚Â°C - 66.3Ã‚Â°C) for 3 min in a PCR machine, followed by 3 min at room temperature. Cells were lysed with NP40 and zymolyase, non-soluble proteins were removed by filtration. In-solution digests were performed as previously described (Hollmann, 2020). Soluble fractions were labeled with isobaric mass tags (TMT) for protein quantification. Here, different conditions of one temperature point were combined in a single TMT-experiment. Peptides were mixed, subjected to a reverse phase clean-up step and subjected to an off-line fractionation under high pH condition.   The resulting 12 fractions were then analyzed by LC-MS/MS on an Orbitrap Fusion Lumos massspectrometer (Thermo Scentific) as previously described (Sridharan, 2019). To this end, peptides were separated using an Ultimate 3000 nano RSLC system (Dionex) equipped with a trapping cartridge (Precolumn C18 PepMap100, 5 mm, 300 ÃŽÂ¼m i.d., 5 ÃŽÂ¼m, 100 Ãƒâ€¦) and an analytical column (Acclaim PepMap 100. 75 Ãƒâ€” 50 cm C18, 3 mm, 100 Ãƒâ€¦) connected to a nanospray-Flex ion source. The peptides were loaded onto the trap column at 30 Ã‚Âµl per min using solvent A (0.1% formic acid) and eluted using a gradient from 2 to 40% Solvent B (0.1% formic acid in acetonitrile) over 2h at 0.3 Ã‚Âµl per min (all solvents were of LC-MS grade). The Orbitrap Fusion Lumos was operated in positive ion mode with a spray voltage of 2.4 kV and capillary temperature of 275 Ã‚Â°C. Full scan MS spectra with a mass range of 375Ã¢â‚¬â€œ1500 m/z were acquired in profile mode using a resolution of 120,000 (maximum fill time of 50 ms or a maximum of 4e5 ions (AGC) and a RF lens setting of 30%. Fragmentation was triggered for 3 s cycle time for peptide like features with charge states of 2Ã¢â‚¬â€œ7 on the MS scan (data-dependent acquisition). Precursors were isolated using the quadrupole with a window of 0.7 m/z and fragmented with a normalized collision energy of 38. Fragment mass spectra were acquired in profile mode and a resolution of 30,000 in profile mode. Maximum fill time was set to 64 ms or an AGC target of 1e5 ions). The dynamic exclusion was set to 45 s.

### Data Protocol
Acquired data were analyzed using IsobarQuant (Franken, 2015) and Mascot V2.4 (Matrix Science) using a reverse UniProt FASTA Saccharomyces_cerrevisiae_database (UP000002311) including common contaminants. In order to distinguish between newly synthesized SILAC-labelled (heavy) and non-labelled (light) proteins, two separate Mascot searches were conducted. (1) The following modifications were taken into account for the identification of mature (light, pre induction) proteins: Carbamidomethyl (C, fixed), TMT10plex (K, fixed), Acetyl (N-term, variable), Oxidation (M, variable) and TMT10plex (N-term, variable). (2) For the analysis of peptides derived from newly synthesized isotope-labelled proteins (heavy), the following modifications were considered as previously described (MÃƒÂ¤ÃƒÂ¤ttÃƒÂ¤, 2020): Carbamidomethyl (C, fixed), Label: 13C(6)15N(4) (R, fixed), TMT10plexSILAC (K, fixed; composition: 13C(10)15N(3)C(2)H(20)N(-1)O(2)), Acetyl (Protein N-term, variable), Oxidation (M, variable), TMT10plex (N-term, variable). The mass error tolerance for full scan MS spectra was set to 10 ppm and for MS/MS spectra to 0.02 Da. A maximum of 2 missed cleavages were allowed. A minimum of 2 unique peptides with a peptide length of at least seven amino acids and a false discovery rate below 0.01 were required on the peptide and protein level.

### Publication Abstract
None

### Keywords
Mitochondria / proteasome / protein folding / proteostasis / thermal proteome profiling / stress response

### Affiliations
EMBL
Univeristy of Kaiserslautern Cell Biology

### Submitter
Frank Stein

### Lab Head
Dr Johannes M. Herrmann
Univeristy of Kaiserslautern Cell Biology


