### Accession
PXD034511

### Title
A complex epigenome-splicing crosstalk governs epithelial to mesenchymal transition in metastasis and brain development

### Description
Epithelial to Mesenchymal Transition (EMT) renders epithelial cells to acquire migratory characteristics during development and cancer metastasis.  While epigenetic and splicing changes have been implicated in EMT, the mechanisms governing their crosstalk remain poorly understood. Here, we identify C2H2 zinc finger protein, ZNF827, a novel factor, is strongly induced during  important EMT mediated processes including in brain development and breast cancer metastasis and is required for the molecular and phenotypic changes underlying EMT in these processes. Mechanistically, ZNF827 mediated these responses by orchestrating a large-scale remodeling of the splicing landscape by recruiting HDAC1 for epigenetic modulation of distinct genomic loci, thereby slowing RNA Pol II progression and altering the splicing of transcripts encoding key EMT regulators in cis. These findings reveal an unprecedented complexity between epigenetic landscape and splicing and identifies ZNF827 as a master regulator coupling these processes during EMT in brain development and breast cancer metastasis.

### Sample Protocol
We labelled MDA MB 231 cells expressing Flag- HA tagged ZNF827 with two distinct, alternate combinations of Lysine and Arginine Isotopes (heavy and light) and performed IP assay for IgG and Flag in both conditions. At the final step, we mixed the heavy IgG with light Flag and vice versa. Subsequently, we processed both set of IP samples for mass-spectrometry and revealed the significantly enriched interaction partners reproducible under both conditions. Peptide fractions were analyzed using a quadrupole Orbitrap mass spectrometer (Q Exactive Plus, Thermo Scientific) equipped with a UHPLC system (EASY-nLC 1000, Thermo Scientific),as described. Peptide samples were loaded onto C18 reversed-phase columns and eluted for 2 hours with a linear gradient of acetonitrile from 8 to 40% containing 0.1% formic acid. The mass spectrometer was operated in data-dependent mode with automatic switching between MS and MS2 acquisition. Survey full scan MS spectra (m/z 300 â€“ 1650) were acquired in the Orbitrap. The 10 most intense ions were sequentially isolated and fragmented by higher-energy C-trap dissociation (HCD) . Peptides with unassigned charge states or charge states less than +2 were excluded from fragmentation. The fragment spectra were acquired in the Orbitrap mass analyzer.

### Data Protocol
Peptide identification: raw data files were analyzed using MaxQuant (development version 1.5.2.8) . Parent ion and MS2 spectra were searched against a database containing 88,473 human protein sequences obtained from the UniProtKB released in December 2013 using the Andromeda search engine . The spectra were searched with a mass tolerance of 6 ppm in MS mode, 20 ppm in HCD MS2 mode, strict trypsin specificity and allowing up to 3 miscleavages. Cysteine carbamidomethylation was searched as a fixed modification, whereas protein N-terminal acetylation and methionine oxidation were searched as variable modifications. The dataset was filtered based on posterior error probability (PEP) to arrive at a false discovery rate below 1%, estimated using a target-decoy approach.

### Publication Abstract
None

### Keywords
Emt

### Affiliations
Queen's University Belfast
Reader,Wellcome-Wolfson Institute for Experimental Medicine, School of Medicine, Dentistry & Biomedical Science, Queens University Belfast

### Submitter
Mohammed Inayatullah

### Lab Head
Dr Dr. Vijay Tiwari
Reader,Wellcome-Wolfson Institute for Experimental Medicine, School of Medicine, Dentistry & Biomedical Science, Queens University Belfast


