### Accession
PXD017347

### Title
TMT spectrometry unveils key M6a interactorsTMT spectrometry unveils key M6a interactors

### Description
The membrane glycoprotein M6a -together with proteolipid protein (PLP), DM20 and M6b- belongs to the tetraspan PLP family. M6a is a neuronal surface protein that promotes neuronal stem cell differentiation, migration, neurite and axonal outgrowth, filopodia/spine induction and synapse formation in primary neuronal cultures and non-neuronal cell lines. M6a has two extracellular loops (EC1 and EC2), four transmembrane domains, one intracellular loop and the N and C terminus facing the cell cytoplasm. However, the complete mechanism of action or proteins associated with it through its extracellular loops remained unknown. In previous work we provided strong evidence that M6aÃƒâ€šÃ‚Â´s extracellular loops widely contribute to its function. To asses this question, we designed and purified a chimera protein (M6a-loops as a bait protein) which was subjected to co-immunoprecipitation (Co-Ip) with rat hippocampi homogenates followed by TMT-MS. The TMT-MS and data analysis was done by the Proteomics Core Facility from the European Molecular Biology Laboratory (EMBL, Heidelberg, Germany).

### Sample Protocol
Bait (recombinant protein): Minor extracellular loop (EC1) of GPM6A (UniProtKB - Q812E9) + poli (G)4 linker +major extracellular loop (EC2) of GPM6A + CH3 of the immunoglobulin IgG + V5 (tag virus epitope SV5) + BAP (Biotin Aceptor Peptide). The completely chimera protein has 36 kDa approximately and 3 sites of N-glycosylation on position 84 (NTTL), 128 (NMTF) and 197 (NGSY) were predicted. The complete aminoacid sequence is:  AGAGHEALSGTVNILQTYFELARTAGDTLDVFTMIDIFKYVGGGGSSLPVYMYFNVWTICRNTTLVEGANLCLDLRQFGIVTIGEEKKICTVSENFLRMCESTELNMTFHLGGSGRPKAPQVYTIPPPKEQMAKDKVSLTCMITDFFPEDITVEWQWNGQPAENYKNTQPIMNTNGSYFVYSKLNVQKSNWEAGNTFTCSVLHEGLHNHHTEKSLSHSPGKGTSGASGKPIPNPLLGLDGSAGGSGGLNDIFEAQKIEWHEGS.   9 Sprague Dawley rats of postnatal day 14, 21 and 30 (3 each) were sacrificed and rat hippocampi were removed, pooled in 3 groups and were sonicated in PBS-Triton X100 1% and with protease inhibitor cocktail.  Three independents Co-Ip were done between 35 - 70 Ãƒâ€šÃ‚Âµg of M6a-loops and 0.5 ÃƒÂ¢Ã¢â€šÂ¬Ã¢â‚¬Å“ 1 mg of protein extracts from rat hippocampi homogenates. Briefly, we used 50 Ãƒâ€šÃ‚Âµl of protein A magnetic SureBeads (Bio-Rad) to immobilized by crosslinking 5 Ãƒâ€šÃ‚Âµg of mouse antibody against SV5. Afterwards, we incubated for 1 h at room temperature with M6a-loops (condition IP and CE) followed by 3 washes. We have also a Mock condition in which primary antibodies against SV5 weren't captured in protein A magnetic beads. Then, each condition was incubated with clarified hippocampi homogenates overnight at 4 C. The day after, proteins were eluted with 50 Ãƒâ€šÃ‚Âµl of SDS buffer with DTT (10 mM) at 70 Ãƒâ€šÃ‚ÂºC. Then 0.50 Ãƒâ€šÃ‚Âµl of protease inhibitor cocktail was added. We performed 3 independent experiments with their replicates (n=9). Thus, for each Co-ip there were 3 samples: 1 control (Mock) and two replicates (IP and CE)  Reduction of disulphide bridges in cysteine containing proteins was done with 10 mM DTT for 30 min at 56 Ãƒâ€šÃ‚ÂºC. Reduced cysteins were alkylated with 20 mM 2-chloroacetamide for 30 min at room temperature. After, proteins were digested overnight at 37 Ãƒâ€šÃ‚ÂºC with trypsin (sequencing grade) in an enzyme to protein ratio 1:50. Digested samples were labeled with 0.8 mg of TMT10plex isobaric label reagent (ThermoFisher) dissolved in 42 Ãƒâ€šÃ‚Âµl acetonitrile for 1 hour. Peptides were separated using an UltiMate 3000 RSLC nano LC system (Dionex), eluted and coupled directly to the QExactive plus (ThermoFisher) mass spectrometer. Full mass scan was acquired with mass range 375-1200 m/z in profile mode with resolution of 70000. Data dependent acquisition (DDA) was performed with the resolution of the Orbitrap set to 35000, with a fill time of 120 ms and a limitation of 2x105 ions.

### Data Protocol
processing protocol:  IsobarQuant and Mascot (v2.2.07) were used to process the acquired data, which was searched against a Uniprot Rattus norvegicus proteome database containing common contaminants and reversed sequences (ID: 000002494, downloading date: 2016-05-14, number of entries: 31673). The following modifications were included into the search parameters: Carbamidomethyl (C) and TMT10 (K) (fixed modification), Acetyl (Protein N-term), Oxidation (M) and TMT10 (N-term) (variable modifications). For the full scan (MS1), a mass error tolerance of 10 ppm and for MS/MS (MS2) spectra of 0.02 Da was set. Further parameters were set: trypsin as protease (with an allowance of maximum two missed cleavages), seven amino acids as minimum peptide length and at least two unique peptides were required for the same protein for identification. The false discovery rate on peptide and protein level was set to 0.01. The final processed dataset files were analyzed as follows: IP 1, 2 and 3 as IP1 replicate 1 (IP1R1), IP2R1 and IP3R1 respectively. CE 1, 2 and 3 as IP1 replicate 2 (IP1R2), IP2R2 and IP3R2 respectively and Mock condition 1, 2 and 3.

### Publication Abstract
None

### Keywords
Co-immunoprecipitation, Tmt-ms, Rattus norvegicus, Glycoprotein m6a, Hippocampus, Protein-protein interactions

### Affiliations
Group Leader. Laboratorio de Neurobiologia Molecular y Celular. Institutto de Investigaciones Biotecnologicas (IIBio) Consejo de Investigaciones Cientificas y Tecnicas (CONICET) Universidad Nacional de San Martin (UNSAM)
Instituto de Investigaciones Biotecnológicas Dr. Rodolgo Ugalde - Universidad Nacional de San Martín

### Submitter
Camila Scorticati

### Lab Head
Dr Camila Scorticati
Group Leader. Laboratorio de Neurobiologia Molecular y Celular. Institutto de Investigaciones Biotecnologicas (IIBio) Consejo de Investigaciones Cientificas y Tecnicas (CONICET) Universidad Nacional de San Martin (UNSAM)


