### Accession
PXD014290

### Title
Two distinct functional axes of positive feedback-enforced PRC2 recruitment in mouse embryonic stem cells

### Description
Polycomb Repressive Complex 2 (PRC2) plays an essential role in development by catalysing trimethylation of histone H3 lysine 27 (H3K27me3), resulting in gene repression. PRC2 consists of two sub-complexes, PRC2.1 and PRC2.2, in which the PRC2 core associates with distinct ancillary subunits such as MTF2 and JARID2, respectively. Both MTF2, present in PRC2.1, and JARID2, present in PRC2.2, play a role in core PRC2 recruitment to target genes in mouse embryonic stem cells (mESCs). However, it remains unclear how these distinct sub-complexes cooperate to establish H3K27me3 domains. Here, we combine a range of Polycomb mutant mESCs with chemical inhibition of PRC2 catalytic activity, to systematically dissect their relative contributions to PRC2 binding to target loci. We find that PRC2.1 and PRC2.2 mediate two distinct paths for recruitment, with mutually reinforced binding. Part of the cross-talk between PRC2.1 and PRC2.2 occurs via their catalytic product H3K27me3, which is bound by the PRC2 core-subunit EED, thereby mediating a positive feedback. Strikingly, removal of either JARID2 or H3K27me3 only has a minor effect on PRC2 recruitment, whereas their combined ablation largely attenuates PRC2 recruitment. This strongly suggests an unexpected redundancy between JARID2 and EED-H3K27me3-mediated recruitment of PRC2. Furthermore, we demonstrate that all core PRC2 recruitment occurs through the combined action of MTF2-mediated recruitment of PRC2.1 to DNA and PRC1-mediated recruitment of JARID2-containing PRC2.2. Both axes of binding are supported by EED-H3K27me3 positive feedback, but to a different degree. Finally, we provide evidence that PRC1 and PRC2 mutually reinforce reciprocal binding. Together, these data disentangle the interdependent and cooperative interactions between Polycomb complexes that are important to establish Polycomb repression at target sites.

### Sample Protocol
Cell pellets were dissolved in RIPA buffer at a density of 104 cells per µl and briefly sonicated to ensure proper cell lysis. Protein extracts (10 µg) were processed using Filter Aided Sample Preparation (FASP) and digested overnight with Trypsin. Peptide mixtures were desalted prior to LC-MS analysis. For ChEP, cells were cross-linked on plates and homogenized in 1 mL cold lysis buffer (25 mM TRIS pH 7.4, 0.1% Triton X-100, 85 mM KCl, 1X Roche protease inhibitor). Suspensions were centrifuged at 2,300g for 5 min at 4 °C, supernatant was aspirated and pellets were lysed in 500 µL of lysis buffer at 37 °C for 15 min. Suspensions were centrifuged at 2,300g for 10 min at 4 °C, supernatant was aspirated, pellets were resuspended in 500 µL of SDS buffer (10 mM TRIS pH 7.4, 10 mM EDTA, 4% SDS, 1X Roche protease inhibitor) and incubated at room temperature for 10 min. 1.5 mL of urea buffer (10 mM TRIS pH 7.4, 1 mM EDTA, 8 M urea) was mixed with samples and they were centrifuged at 16,100g for 30 min at room temperature. This step was repeated before the pellet was washed in 2 ml SDS buffer. Pellets were carefully resuspended in storage buffer (10 mM TRIS pH 7.4, 1 mM EDTA, 25 mM NaCl, 10% glycerol, 1X Roche protease inhibitor) and sonicated for 6 min at high intensity (30s on/off alternation) on a NGS bioruptor (Diagenode). Protein concentration was determined using a Qubit assay (Invitrogen). 30 µg protein extract per sample was processed using Filter Aided Sample Preparation (FASP) and digested overnight with Trypsin. Peptide mixtures were desalted prior to LC-MS analysis.

### Data Protocol
Thermo RAW files were analysed using MaxQuant 1.5.1.0 with default settings and LFQ, IBAQ and match between runs enabled. In Perseus, contaminant and reverse hits were filtered out. For whole cell proteomes, WT, Mtf2 knockout and Jarid2 knockout ESCs were grouped. For ChEP, wildtype, Mtf2 knockout, Jarid2 knockout, Eed knockout and EED226 treated samples were grouped. Only proteins that had an LFQ value in at least one of the conditions were maintained. Missing values were imputed using default settings in Perseus.

### Publication Abstract
None

### Keywords
Chromatin, Prc2, Polycomb, Embryonic stem cells

### Affiliations
Department of Molecular Developmental Biology, Radboud Institute for Molecular Life Sciences (RIMLS), Nijmegen, the Netherlands
Radboud University

### Submitter
Guido van Mierlo

### Lab Head
Dr Gert Jan Veenstra
Department of Molecular Developmental Biology, Radboud Institute for Molecular Life Sciences (RIMLS), Nijmegen, the Netherlands


