### Accession
PXD032031

### Title
Primary Myeloma Proteomics using Micro IST method

### Description
The myeloma cell surface proteome (â€œsurfaceomeâ€�) not only determines tumor interaction with the microenvironment but serves as an emerging arena for therapeutic development. Here, we use glycoprotein capture proteomics to first define surface markers most-enriched on myeloma when compared to B-cell malignancy models, revealing unexpected biological signatures unique to malignant plasma cells. We next integrate our proteomic dataset with existing transcriptome databases, nominating CCR10 and TXNDC11 as possible monotherapeutic targets and CD48 as a promising co-target for increasing avidity of BCMA-directed cellular therapies. We further identify potential biomarkers of resistance to proteasome inhibitors and lenalidomide including changes in CD53, EVI2B, CD10, and CD33. Comparison of short-term treatment with chronic resistance delineates large differences in surface proteome profile under each type of drug exposure. Finally, we develop a miniaturized version of the surface proteomics protocol and present the first surface proteomic profile of a primary MM patient plasma cell sample. Our dataset provides a unique resource to advance the biological, therapeutic, and diagnostic understanding of myeloma. Note that this is a data supplement to PXD022553.

### Sample Protocol
Labeled cell pellets were lysed in 500 Î¼L buffer containing 2X RIPA (Millipore 20-188), 1X HALT protease inhibitor (Thermo 78430), and 2 mM EDTA. Lysates were sonicated at 1-Hz pulses for 30 seconds with a probe sonicator and incubated on ice for 10 minutes. To remove precipitates, samples were spun at 17,200g for 10 minutes at 4Â°C. To prepare for enrichment of biotinylated proteins, 100, 80, 60 of High Capacity NeutrAvidin slurry (Thermo PI29204) were washed three times with 1 mL 2X RIPA/1mM EDTA buffer in 2-mL chromatography columns (Bio-Rad 7326008) attached to a vacuum manifold (Promega A7231). Clarified lysates were then transferred to the chromatography column (3e7, 1e7, 1e6 cell inputs into 100, 80, 60 slurry, respectively) and incubated at 4Â°C for 60 minutes on a rotisserie. For the macro protocol, protein-bound beads were transferred to a 10-mL chromatography column (Bio-Rad 7311550) and washed with the following buffers: 50 mL 1x RIPA/1mM EDTA, 50 mL 1X PBS/1M NaCl, and 50 mL 2M urea/50mM ammonium bicarbonate (ABC). For the micro protocol, beads were transferred to a 2-mL chromatography column and washed with 5 mL of each buffer. For the macro protocol, rinsed beads were dried fully and transferred to a 1.5-mL tube using 100 Î¼L of digestion buffer containing 1.5M urea (VWR 97063-798), 50mM ABC, 10 mM 2-chloroacetamide (VWR 97064-926), and 5mM TCEP (GoldBio TCEP10). For the micro protocol, P200-StageTips were packed with four C18 disks (3M 14-386-2) and activated with 60 Î¼L methanol, 60 Î¼L 80% acetonitrile (ACN)/0.1% formic acid (FA), and twice with 60 Î¼L 0.1% trifluoroacetic acid (TFA) prior to transferring the beads to the tip using 100 Î¼L of the digestion buffer. For protein digestion, trypsin/LysC (Promega PRV5073) was reconstituted to 1 Î¼g/Î¼L with ultrapure water, and added to each sample (1500, 500, 50 ng trypsin for 3e7, 1e7, 1e6 cell inputs, respectively). The macro samples were mixed at 1000 RPM on an orbital shaker while the micro samples were wrapped in parafilm and placed on an end-to-end rotisserie at RT for the overnight digestion. For the macro protocol, the soluble fraction containing the tryptic peptides was separated from the beads by spinning at 1000g for one minute. Samples were then transferred to fresh tube and acidified with 10% TFA to reach a final concentration of 1% TFA. For desalting, C18 SOLA columns (Thermo 03150391) were activated with 500 Î¼L ACN and equilibrated twice with 500 Î¼L 0.1% TFA on a vacuum manifold. Acidified samples were passed through the column twice followed by two washes with 1000 Î¼L 0.1% TFA and one wash with 500 Î¼L 2% ACN/0.1% FA. Finally, peptides were eluted once with 150 Î¼L 80% ACN/0.1% FA and again with 200 Î¼L 80% ACN/0.1% FA. Samples were fully dried by SpeedVac and stored in -80Â°C. For the micro protocol, 10% TFA was added to each sample to reach a final concentration of 1% TFA. An initial peptide binding step was performed by spinning the acidified samples at 1000g prior to desalting three times with 100 Î¼L 0.1% TFA. Peptides were then eluted twice with 50 Î¼L 80% ACN/0.1% FA. Samples were fully dried by SpeedVac and stored in -80Â°C. CD138+ MM cells were isolated from patient peripheral blood using a magnetic bead-based approach (EasySep Human CD138 Positive Selection Kit, StemCell Technologies 17877). The CD138+ -enriched sample was titrated into 2.5e7, 1.7, 5e6, 1e6, 5e5, 3e5, 1e5 cells for cell-surface labeling and processed to obtain surface peptides using the micro protocol described above. 1 ug of peptides were injected into a Dionex Ultimate 3000 NanoRSLC with 15-cm Acclaim PEPMAP C18 (Thermo) reverse phase column coupled to a Thermo Q Exactive Plus. A linear gradient from 2.4% Acetonitrile to 40% Acetonitrile in 0.1% Formic Acid over 195 minutes at flow rate of 0.2uL/min, with increase to 80% Acetonitrile for column wash prior to re-equilibration. For MS1 acquisition, spectra were collected in data dependent top 15 method with full MS scan resolution of 70,000, AGC target was set to 3e6, and maximum IT set to 100ms. For MS2 acquisition, resolution was set to 17,500, AGC set to 5e4, and maximum IT to 180ms.

### Data Protocol
Mass spectrometry data was processed in Maxquant (12) (version 1.6.2.1) with settings as follows: enzyme specificity as trypsin with up to two missed cleavages, PSM/Protein FDR 0.01, cysteine carbidomethylation as fixed modification, methionine oxidation and N-terminal acetylation as variable modifications, minimum peptide length = 7, matching time window 0.7 min, alignment time 20 min, with match between runs, along with other default settings. Data was searched against the Uniprot Swiss-Prot human proteome (obtained Sept. 3, 2018). Proteingroups files were exported from Maxquant, filtered to remove contaminants, and filtered for proteins with at least two unique peptides for analysis. Data analysis was performed in Perseus (64) and R. Identified proteins were filtered against curated lists of Uniprot-annotated membrane proteins or those identified in the analysis of Bausch-Fluck et al. (10), as described.

### Publication Abstract
None

### Keywords
Myeloma

### Affiliations
Stanford University
Dept. of Laboratory Medicine, University of California, San Francisco, CA

### Submitter
Ian Ferguson

### Lab Head
Dr Arun Wiita
Dept. of Laboratory Medicine, University of California, San Francisco, CA


