### Accession
PXD000524

### Title
Modified cytosine binding proteins

### Description
DNA methylation (5mC) plays important roles in epigenetic regulation of genome function, and recently the TET1-3 hydroxylases have been found to oxidize 5mC to hydroxymethylcytosine (5hmC), formylcytosine (5fC), and carboxylcytosine (5caC) in DNA. These derivatives have a role in demethylation of DNA but in addition may have epigenetic signaling functions in their own right. A recent study identified proteins with preferential binding to 5-methylcytosine (5mC) and its oxidized forms where readers for 5mC and 5hmC (5-hydroxymethylcytosine) showed little overlap while further oxidation forms enriched for repair proteins and transcription regulators. We extend this study by using promoter sequences as baits and compare protein binding patterns to unmodified or modified cytosine containing DNA using mouse embryonic stem cell (mESCs) extracts. The dataset contains 3 biological replicates each of mouse ES cell nuclear proteins binding to Pax6 and FGF15 promoter sequences containing different modified forms of cytosine. Data analysis: Mass spectrometric data were processed using Proteome Discoverer v1.3 and searched against a mammalian entries in Uniprot 2011.09 using Mascot v2.3 with the following parameters: Enzyme - trypsin; max 1 missed cleavage; Precursor Mass Tolerance - 10 ppm; Fragment Mass Tolerance - 0.6 Da; Dynamic Modification - Oxidation (M); Static Modification - Carbamidomethyl at C.

### Sample Protocol
Not available

### Data Protocol
Not available

### Publication Abstract
None

### Keywords
Hydroxymethylcytosine, Methylcytosine, Fgf15, Es cells, Formylcytosine, Pax6

### Affiliations
Babraham Institute

### Submitter
David Oxley

### Lab Head


