### Accession
PXD024169

### Title
Comparision of Nano and Micro-flow LC-MS/MS Based Single Shot Proteomics

### Description
In this study, we systematically optimized the micro-flow LC-MS/MS system based ssDDA proteomics. Our results show that combination of 2 – 3 h gradient with prosit rescore method identified the highest number of proteins and peptides. There is no benefit to increase column length. A single 3 h run identified about 9,200 proteins from 50 µg of Arabidopsis protein digest. The micro-flow LC-MS/MS system based ssDDA proteomics can be widely used for routine proteomics analysis of large cohort samples. We compared our micro data with the state of art nano LC data.

### Sample Protocol
For the cell line sample, cells were harvested at ∼80% confluence by washing twice with PBS buffer and subsequently adding ice-cold lysis buffer containing  2% (w/v) sodium dodecyl sulfate (SDS), and 40 mM Tris-HCl (pH 7.6). The cell lysate was heated at 95 °C for 5 min, followed by the addition of trifluoroacetic acid (TFA) to a final concentration of 1% (v/v). Subsequently, N-Methylmorpholine was added to a final concentration of 4% (v/v) to adjust the pH value to ~7.5, following by the SP3 digestion protocol. For the tissue sample, the tissue were extracted with 8 M urea buffer, and diluted to 1 M before in solution trypsin digestion.   All the samples were analyzed on a micro-flow LC-MS/MS system using a modified Vanquish pump (Thermo Fisher Scientific) coupled to a Q Exactive Orbitrap HF-X (Thermo Fisher Scientific) mass spectrometer17. Chromatographic separation was performed via direct injection on a 15 cm Acclaim PepMap 100 C18 column (2 µm, 1 mm ID, Thermo Fisher Scientific) at a flow rate of 50 µL/min. The solvent A is 0.1% FA in 3% DMSO, and solvent B is 0.1% FA, 3% DMSO in ACN. For the sample loading test, a 60 min linear gradient of 3% to 28% B was used. For gradient test ranging from 1 h to 6 h, the solvent B was increased from 1% to 24% in 0.87 * x min, followed by 24% to 35% in 0.13 * x min, x represents the gradient time. The HF-X was operated at positive polarity, with spray voltage at 3.5 kV, funnel RF lens value at 40, capillary temperature at 320 °C, auxillary gas heater temperature at 300 °C. The flow rates for sheath gas, aux gas and sweep gas were set to 35, 5, and 0, respectively. The full MS resolution was set to 120,000 at m/z 200 with a mass range was set to 360–1300. The full MS AGC target was set to 3E6 with a maximum injection time (IT) of 50 ms. For MS2 spectra, the AGC target value was set to 1E5, and the minimum AGC target was kept at 2E3. The isolation width was set to 1.3 m/z, and the first mass was fixed at 100 m/z. The normalized collision energy was set to 28%. Peptide match was set to preferred, and isotope exclusion was on. MS1 and MS2 spectra were acquired in profile and centroid mode, respectively. Dynamic exclusion values were set to 20 s, 40 s, 50 s, 60 s, 90s and 90 s for 1 h, 2 h, 3 h, 4 h, 5 h and 6 h gradients, respectively. The MS2 spectra resolutions of the 28 Hz and 41 Hz methods were set to 15,000 and 7500, the maximum IT values of precursors of the 28 Hz and 41 Hz methods were set to 22 ms and 11 ms. In the 1 h gradient loading test experiment, the top N values were set to 20 and 40 in the 28 Hz and 41 Hz methods. For all the other methods, 28 Hz method was used with maximum 50 MS2 spectra per MS1 scan. For the cell samples, the protein concentration was determined by the BCA method. Protein digestion was performed with single-pot solid-phase-enhanced sample preparation (SP3) protocol16 []. The cell lysate containing 100 µg of protein was mixed with the 20 µl of SP3 magnetic beads, which were prepared by mixing equal volume of Sera-Mag Speed Beads A and B (Fisher Scientific, Germany). Then, the mixture was uncubated in a ThermoMixer () at 24oC for 10 min at 800 rpm, following by immobilization of beads on a magnetic rack for 2 min. The supernatant was discarded, and the beads were washed 3 times with 200 μL of 80% (v/v) ethanol/water and one time with 200 μL of ACN. The bound proteins were reduced by adding 200 μL of 10 mM dithiothreitol (DTT) in 100 mM Tris-HCl (pH 8.5), the reduction was performed in a ThermoMixer at 37 °C and 1000 rpm for 1 h. Subsequently, proteins were alkylated with 55 mM chloroacetamide (CAA) for 0.5 h at 37 °C in the dark. Finally, trypsin (Roche) was added to the digestion buffer with an enzyme:protein ratio of 1:50 (w/w), and the digestion was performed overnight at 37 °C in a ThermoMixer at 1000 rpm. After protein digestion, samples were acidified with FA at final concentration of 1% (v/v), desalted using the Sep-Pak tC18 96-well plate (Waters, 25 mg). Peptides were eluted with 0.1% TFA in 50% acetonitrile (ACN) and vacuum-dried in a Thermo Savant SPD SpeedVac (Thermo Fisher Scientific), and stored at -80 °C until further usage.

### Data Protocol
The raw data files were processed with MaxQuant v1.6.2.3 using the integrated Andromeda Search engine and against the Arabidopsis or Uniprot (human and mouse) reference databases. The Arabidopsis database contains 48,359 entries (Araport11_genes.201606.pep.fasta; downloaded 06/2016). The human and mouse protein database contains 20,230 (downloaded 06/2017) and 17,027 (downloaded 02/2020) canonical entries, separately. Default MaxQuant parameters were used. Trypsin was specified as the enzyme, cleaving after all lysine and arginine residues and allowing up to two missed cleavages. Carbamidomethylation of cysteine was specified as fixed modification and protein N-terminal acetylation and oxidation of methionine were considered as variable modifications. The false discovery rate (FDR) was set to 1% on the site, peptide-spectrum match (PSM) and protein levels. For the Prosit rescore analysis, the MaxQuant search was performed with 100% false discovery rate (FDR) on the site, peptide-spectrum match (PSM) and protein levels. All the other parameters are the same as the normal 1% FDR MaxQuant search. The msms.txt output and raw data files were uploaded to Prosit website, peptides with q value less than 0.01 were kept. The proteins were assigned based on the picked group FDR method. Data analysis downstream of MaxQuant output results was performed in R.

### Publication Abstract
A current trend in proteomics is to acquire data in a "single-shot" by LC-MS/MS because it simplifies workflows and promises better throughput and quantitative accuracy than schemes that involve extensive sample fractionation. However, single-shot approaches can suffer from limited proteome coverage when performed by data dependent acquisition (ssDDA) on nanoflow LC systems. For applications where sample quantities are not scarce, this study shows that high proteome coverage can be obtained using a microflow LC-MS/MS system operating a 1 mm i.d.&#x2009;&#xd7; 150&#x2009;mm column, at a flow-rate of 50&#x2009;&#x3bc;L/min and coupled to an Orbitrap HF-X mass spectrometer. The results demonstrate the identification of &#x223c;9&#x202f;000 proteins from 50 &#x3bc;g of protein digest from <i>Arabidopsis</i> roots, 7&#x202f;500 from mouse thymus, and 7&#x202f;300 from human breast cancer cells in 3 h of analysis time in a single run. The dynamic range of protein quantification measured by the iBAQ approach spanned 5 orders of magnitude and replicate analysis showed that the median coefficient of variation was below 20%. Together, this study shows that ssDDA by &#x3bc;LC-MS/MS is a robust method for comprehensive and large-scale proteome analysis and which may be further extended to more rapid chromatography and data independent acquisition approaches in the future.&#x300;.

### Keywords
Nano-flow lc, Micro-flow lc, Single shot proteomics

### Affiliations
Chair of Proteomics and Bioanalytics, Technical University of Munich
Chair of Proteomics and Bioanalytics, Technische Universitaet Muenchen

### Submitter
Yangyang  Bian

### Lab Head
Dr Bernhard Kuster
Chair of Proteomics and Bioanalytics, Technische Universitaet Muenchen


