### Accession
PXD028271

### Title
Chromosomal instability adaptation dependencies confer unique targetable vulnerabilities in therapy refractory lethal prostate cancer

### Description
Chromosomal instability (CIN), an ongoing rate of chromosome missegregation during mitosis, is a defining feature of cancer. However, high chromosomal aberrations are detrimental for cell fitness. Here we investigated mechanisms allowing lethal prostate cancer (PCa) to tolerate and survive increasing CIN. Transcriptomic and proteomic analysis of patient datasets and experimental models showed a concomitant increase of CIN and cell division fidelity kinases in lethal PCa. Functional studies identified MASTL as a key kinase to which therapy-resistant PCa cells become addicted to restrain lethal CIN and ensure survival. Combined analysis of transcription factors increased in high CIN PCa patient datasets with detailed promoter analysis identified that MASTL expression is regulated by the Androgen Receptor variant 7 (AR-V7) and E2F7. Finally, targeting MASTL addiction vulnerability in vivo using the small molecule inhibitor GKI-1, improves survival of pre-clinical models. These findings provide proof-of-concept for exploiting CIN levels as a therapeutic approach in cancer.

### Sample Protocol
Quantitative Proteomics (SILAC) Methods Amino Acid Labeling in Cell Culture. 22Rv1 and 22Rv1-DR cells were adapted to grow in SILAC RPMI-1640 media deficient in lysine and arginine (Thermo 88365) supplemented with antibiotics (Pen/Strep), 10% dialyzed fetal bovine serum (Gemini) and normal amino acids “Light Amino Acid Media” (175 μM L-arginine (Arg0) plus 250 μM L-lysine (Lys0)), or with the same concentrations of stable isotope-labeled amino acids to make “Heavy Amino Acid Media” (L-arginine-13C6, 15N4 hydrochloride (Arg10) and L-lysine-13C6, 15N2 hydrochloride(Lys8), Cambridge Isotope Labs) as described before 46. After six cell doublings and confirmation via mass spectrometry that heavy amino acids incorporation was equal or above 98% we proceeded with the experiment. Cells growing in 15 cm dishes at 70% confluence were treated with 200ng/ml nocodazole for 14 hours, cells arrested in prometaphase collected by shake-off as previously described 115 and snap-frozen until sample preparation for mass spectrometry. Labeling scheme included three forward biological replicates (DR cells heavy/parental cells light) and three reverse replicates (DR cells light label/parental cells heavy). Sample Preparation and Mass Spectrometry. Samples were prepared for lysis in denaturing buffer (8 M urea, 50 mM Tris-HCl pH 8.2, 75 mM NaCl plus protease and phosphatase inhibitors (Roche) and processed as previously described 116. Briefly, pellets were lysed in followed by incubation 15min on ice, sonicated and spun down for 35min at 10,000g 4C. Cell extracts were quantified and light and heavy samples mixed 1:1. After pooling the SILAC-labeled cell lysates, samples were processed for digestion and phosphopeptide enrichment and analyzed by mass spectrometry following standard protocols 116.  The purified peptides were diluted to 0.1% formic acid and fraction/section was analyzed separately by microcapillary liquid chromatography with tandem mass spectrometry using the NanoAcquity (Waters) with a 100-μm-inner-diameter × 10-cm-length C18 column (1.7 um BEH130, Waters) configured with a 180-µm × 2-cm trap column coupled to a Q-Exactive Plus mass spectrometer (Thermo Fisher Scientific). Key parameters for the mass spectrometer were: AGC 3 E6, resolution 70,000, 380-1800 m/z, top 10 method.

### Data Protocol
All MS/MS data was processed with the MaxQuant software (Max Planck Institute of Biochemistry, Martinsried, Germany; version 1.5.3.30) 117. The default was used for first search tolerance and main search tolerance: 20 and 6 ppm, respectively. Labels were set to Arg10 and Lys8. MaxQuant was set up to search the reference human proteome database downloaded from Uniprot on Sept 6, 2016. Maxquant performed the search assuming trypsin digestion with up to two missed cleavages. Peptide, site, and protein FDR were all set to 1% with a minimum of 1 peptide needed for identification but two peptides needed to calculate a protein level ratio. The following modifications were used as variable modifications for identifications and included for protein quantification: oxidation of methionine, acetylation of the protein N-terminus, phosphorylation of serine, threonine and tyrosine residues, and carbamidomethyl on cysteine. Specific phosphosites were assigned by the MaxQuant PTM score algorithm 118,119. Phosphopeptides measured in all biological replicates were log2-transformed, plotted and statistically analyzed via ANOVA. Fold changes (Log2FC) ≥ 1.5 and Benjamini-Horchberg FDR < 0.05 in 22Rv1-DR cells vs. parental cells were rated as significant. Phophoproteins increased in 22Rv1-DR cells in at least 3 replicates were query for functional enrichment analysis using gProfiler 47 to determine significantly enriched Gene Ontology Biological Processes. P-value was computed by Fisher’s test and corrected with Benjamini-Hochberg FDR.

### Publication Abstract
None

### Keywords
Mastl expression, Chromosomal instability

### Affiliations
MSKCC
Cancer Biology Department, Sidney Kimmel Cancer Center, Thomas Jefferson University

### Submitter
Zhuoning Li

### Lab Head
Dr Veronica Rodriguez-Bravo
Cancer Biology Department, Sidney Kimmel Cancer Center, Thomas Jefferson University


