### Accession
PXD008885

### Title
Pneumococcal surface tryptic digestion

### Description
Surface digestion of living pneumococcal cells grown in different media that attempt to simulate different infection models (for example, in culture media containing the soluble secreted fraction orignated by the growth of macrophages and epithelial cells)

### Sample Protocol
Pneumococcal cells were grown on different culture media and, when the mid exponential phase was reached, they were collected by centrifugation. Once cells were washed, a tryptic digestion protocol was carried out to cut surface domain proteins.

### Data Protocol
Protein identification by database searching. Tandem mass spectra were extracted using Thermo Proteome- Discoverer 1.0 (Thermo Fisher Scientific). All MS/MS samples were analyzed using Sequest (Thermo Fisher Scientific, version v.27), applying the following search parameters: peptide tolerance, 10 ppm; tolerance for fragment ions, 0.8 Da; b- and y-ion series; oxidation of methionine and deamidation of asparagine and glutamine were considered as variable modifications; and maximum trypsin missed cleavage sites, 3. The raw data were searched against an in-house joint database containing the protein sequences from all the sequenced and annotated S. pneumoniae strains available at the Uniprot site. Peptide identifications were accepted if they exceeded the filter parameter Xcorr score vs charge state with SequestNode Probability Score (+1 = 1.5, +2 = 2.0, +3 = 2.25, +4 = 2.5). With these search and filter parameters, no false-positive hits were obtained. For proteins identified from only one peptide, fragmentations were checked manually. Strain R6 was used as reference for providing the accession numbers of the identified proteins; whenever a protein belonging to another strain was found, homology with a corresponding protein of strain R6 was given by using protein-BLAST. If homology with R6 or TIGR4 or Hungary19A-6 proteins was not observed, then the protein accession numbers of the other strains were used.

### Publication Abstract
Infections caused by the Gram-positive bacterium <i>Streptococcus pneumoniae</i> have become a major health problem worldwide because of their high morbidity and mortality rates, especially in developing countries. This microorganism colonizes the human upper respiratory tract and becomes pathogenic under certain circumstances, which are not well known. In the interaction with the host, bacterial surface structures and proteins play major roles. To gain knowledge into gradual changes and adaptive mechanisms that this pathogen undergoes from when it enters the host, we mimicked several in vivo situations representing interaction with epithelial and macrophage cells, as well as a condition of presence in blood. Then, we analyzed, in four pneumococcal strains, two major surface structures, the capsule and extracellular vesicles produced by the pneumococci, as well as surface proteins by proteomics, using the "shaving" approach, followed by LC-MS/MS. We found important differences in both surface ultrastructures and proteins among the culture conditions and strains used. Thus, this work provides insights into physiological adaptations of the pneumococcus when it interacts with the host, which may be useful for the design of strategies to combat infections caused by this pathogen.

### Keywords
Trypsin, Surface proteins, Surfomic, Vaccine, Streptococcus pneumoniae

### Affiliations
University of Cordoba
Department of Biochemistry and Molecular Biology. University of Córdoba

### Submitter
Alfonso Olaya Abril

### Lab Head
Dr Manuel José Rodríguez Ortega
Department of Biochemistry and Molecular Biology. University of Córdoba


