### Accession
PXD015978

### Title
 Global proteomics of HEK293T cells treated with Val-boroPro for 48h

### Description
Global proteomics of HEK293T cells treated with Val-boroPro for 48h

### Sample Protocol
Sample preparation: HEK293T cells were treated with DMSO or Val-boroPro (10 uM) for 48h. Samples were harvested and lysed. Soluble proteins were reduced, alkylated and trypsinized. The tryptic peptides were labeled with TMTsixplex according to manufacturer specifications (126, 127, 128 = DMSO; 129, 130, 131 = VbP). The labeled peptides were pooled and desalted with high pH fractionation for tandem mass spectrometry.  Tandem LC-MS/MS/MS: Mass spectrometry data was collected on an Orbitrap Fusion Lumos mass spectrometer coupled to an Easy-nLC 1200 (Thermo Fisher Scientific). Peptides were separated over a 220 min gradient of 0 to 50% acetonitrile in water with 0.1% formic acid at a flow rate of 300 nL/min on a 50 cm long PepMap RSLC C18 column (2 um, 100Ã…, 75u, x 50 cm). The full MS spectra were acquired in the Orbitrap at a resolution of 120,000. The 10 most intense MS1 ions were selected for MS2 analysis. The isolation width was set at 0.7 m/z and isolated precursors were fragmented by CID (35% CE). Following acquisition of each MS2 spectrum, a synchronous precursor selection (SPS) MS3 scan was collected on the top 10 most intense ions in the MS2 spectrum. The isolation width was set at 1.2 m/z and isolated precursors were fragmented 13 using HCD.

### Data Protocol
Proteomic Analysis. MS raw files were converted into MGF and processed using Proteome Discoverer version 2.2.0.388 (Thermo Scientific) by searching against the Uniprot human Database supplemented with GFP and common contaminant protein sequences and quantifying according to SPS MS3 reporter ions. Identification was based on CID spectra using SequestHT. Search criteria included: 20 ppm mass tolerance for MS spectra, 0.5 Da fragment mass tolerance for MS/MS spectra, a maximum of two allowed missed cleavages, static carbamidomethylation (+57.021 Da) of cysteine and TMTsixplex (+229.163 Da) of peptide N-terminus, dynamic modifications of methionine oxidation (+15.995 Da), N-terminal protein acetylation (+42.011 Da), asparagine or glutamine deamidation (+0.984 Da), and serine, tyrosine or tryptophan phosphorylation (+79.966 Da) and a false discovery rate of <0.01.

### Publication Abstract
Several cytosolic pattern-recognition receptors (PRRs) form multiprotein complexes called canonical inflammasomes in response to intracellular danger signals. Canonical inflammasomes recruit and activate caspase-1 (CASP1), which in turn cleaves and activates inflammatory cytokines and gasdermin D (GSDMD), inducing pyroptotic cell death. Inhibitors of the dipeptidyl peptidases DPP8 and DPP9 (DPP8/9) activate both the human NLRP1 and CARD8 inflammasomes. NLRP1 and CARD8 have different N-terminal regions but have similar C-terminal regions that undergo autoproteolysis to generate two non-covalently associated fragments. Here, we show that DPP8/9 inhibition activates a proteasomal degradation pathway that targets disordered and misfolded proteins for destruction. CARD8's N&#xa0;terminus contains a disordered region of &#x223c;160 amino acids that is recognized and destroyed by this degradation pathway, thereby freeing its C-terminal fragment to activate CASP1 and induce pyroptosis. Thus, CARD8 serves as an alarm to signal the activation of a degradation pathway for disordered and misfolded proteins.

### Keywords
Val-boropro, Tmtsixplex

### Affiliations
Chemical Biology Program, Memorial Sloan Kettering Cancer Center, New York, New York 10065, USA
Weill Cornell/Rockefeller/Sloan Kettering Tri-Institutional MD-PhD Program, New York, New York, USA.

### Submitter
Andrew Griswold

### Lab Head
Dr Daniel A. Bachovchin
Chemical Biology Program, Memorial Sloan Kettering Cancer Center, New York, New York 10065, USA


