### Accession
PXD006905

### Title
Single-cell Proteomics of Developing Cells in Live Vertebrate Embryos by Microprobe High-Resolution Mass Spectrometry

### Description
In this work, we developed a microprobe capillary electrophoresis high-resolution mass spectrometry approach to identify proteins in single identified cells in live embryos of the South African clawed frog (Xenopus laevis) and also zebrafish. We used a pulled borosilicate capillary to aspirate a calibrated portion of the cell of interest. Proteins in the collected aspirate were detected and quantified via a bottom-up proteomic workflow that was downscaled to the single cells. The microprobe mass spectrometry platform is scalable to smaller cells in live, morphologically complex, developing embryos. As an example, we used this approach to analyze proteomic changes in cells that form a clone in the early developing Xenopus laevis embryo.

### Sample Protocol
Proteins from the single-cell aspirates were released into a digestion solution containing 50 mM ammonium bicarbonate with 10% acetonitrile, 50 mM ammonium bicarbonate with 0.05% RapiGest (Waters), or 50 mM ammonium bicarbonate. Reduction or alkylation steps were eliminated. Proteins were enzymatically cleaved using trypsin at 37 Â°C for 5 h. The resulting peptides were separated by capillary electrophoresis and ionized in a nanoelectrospray ionization interface. The generated peptide ions were sequenced using a quadrupole Orbitrap high-resolution mass spectrometer (Q-Exactive Plus, Thermo Fisher Scientific).

### Data Protocol
The raw mass spectrometry data were processed in MaxQuant ver 1.5.5.7 running the Andromeda search engine against Xenopus or zebrafish-specific proteome databases. The frog database was custom-built by concatenating the Xenopus proteome from SwissProt and PHROG 1.0, an mRNA-derived database. Similarly, we used the UniProt protein database (concatenated & isoforms) for zebrafish. Peptide and protein identifications are reported with <1% false discovery rate.

### Publication Abstract
Label-free single-cell proteomics by mass spectrometry (MS) is currently incompatible with complex tissues without requiring cell culturing, single-cell dissection, or tissue dissociation. We here report the first example of label-free single-cell MS-based proteomics directly in single cells in live vertebrate embryos. Our approach integrates optically guided in situ subcellular capillary microsampling, one-pot extraction-digestion of the collected proteins, peptide separation by capillary electrophoresis, ionization by an ultrasensitive electrokinetically pumped nanoelectrospray, and detection by high-resolution MS (Orbitrap). With a 700 zmol (420&#x202f;000 copies) lower limit of detection, this trace-sensitive technology confidently identified and quantified &#x223c;750-800 protein groups (&lt;1% false-discovery rate) by analyzing just &#x223c;5 ng of protein digest, viz. &lt;0.05% of the total protein content from individual cells in a 16-cell Xenopus laevis (frog) embryo. After validating the approach by recovering animal-vegetal-pole proteomic asymmetry in the frog zygote, the technology was applied to uncover proteomic reorganization as the animal-dorsal (D11) cell of the 16-cell embryo gave rise to its neural-tissue-fated clone in the embryo developing to the 32-, 64-, and 128-cell stages. In addition to enabling proteomics on smaller cells in X. laevis, we also demonstrated this technology to be scalable to single cells in live zebrafish embryos. Microsampling single-cell MS-based proteomics raises exciting opportunities to study cell and developmental processes directly in complex tissues and whole organisms at the level of the building block of life: the cell.

### Keywords
Xenopus, Embryo, Label free quantification, Capillary electrophoresis, Lfq, Electrospray ionization, Proteomics, Q-exactive plus, Orbitrap, Blastomere, Mass spectrometry, Single cell analysis

### Affiliations
University of Maryland, College Park

### Submitter
Peter Nemes

### Lab Head
Dr Peter Nemes
University of Maryland, College Park


