### Accession
PXD000553

### Title
Polysome profiling reveals translational control of gene expression in the human malaria parasite Plasmodium falciparum

### Description
MudPIT analysis of Plasmodium polysomes. TCA precipitated pellet from Plasmodium falciparum polysomes (~50ug) was solubilized in TRIS-HCl pH8.5 and Urea; reduced and alkylated.  The protein suspension was digested overnight using Endoproteinase Lys-C followed by a second overnight digestion at 37Â°C using Trypsin.  Formic acid (5% final) was added to stop the reactions.  Sample was loaded on split-triple-phase fused-silica micro-capillary column and placed in-line with linear Velos pro ion-trap mass spectrometer (Thermo Scientific), coupled with quaternary Agilent 1260 series HPLC.  Fully automated 10-step chromatography run (for a total of 20 hours) was carried out on the sample.  Each full MS scan (400-1600 m/z) was followed by ten data-dependent MS/MS scans. The number of the micro scans was set to 1 both for MS and MS/MS. The dynamic exclusion settings used were as follows: repeat count 2; repeat duration 30s; exclusion list size 500 and exclusion duration 90 sec, the minimum signal threshold was set to 500. The MS/MS dataset was searched using SEQUEST (v.27; rev. 9) against a database of 72358 sequences, consisting of 5487 P. falciparum non-redundant proteins (downloaded from PlasmoDB on 2012-07-12), 30536 H. sapiens non-redundant proteins (downloaded from NCBI on 2012-08-27), 177 usual contaminants (such as human keratins, IgGs, and proteolytic enzymes), and, to estimate false discovery rates, 36179 randomized amino acid sequences derived from each non-redundant protein entry.  To account for alkylation by CAM, 57 Da were added statically to cysteine residues and to account for the oxidation of methionine residues to methionine sulfoxide (that can occur as an artifact during sample processing) 16Da were added as a differential modification to methionine residue.  Peptide/spectrum matches were sorted, selected using DTASelect/CONTRAST (v 1.9).  Proteins had to be detected by 1 peptide with 2 independent spectra, leading to FDRs at the protein and spectral levels of 2.89% and 0.26%, respectively.

### Sample Protocol
Not available

### Data Protocol
Not available

### Publication Abstract
None

### Keywords
Translation, Transcription, Malaria, Cell cycle

### Affiliations
Proteomics Center

### Submitter
Anita Saraf

### Lab Head


