### Accession
PXD005871

### Title
CARM1 substrate identification

### Description
The goal of the project was global identification of CARM1 substrates. Arginine methylation landscapes were profiled and compared in wild type and CARM1 knockout cells to determine in vivo substrates of CARM1.

### Sample Protocol
We compared ADMA profiles of wild type (WT) and complete CARM1 knockout (KO) MCF7 and MDA-MB-231 cells (Fig. 1). Three biological replicas of each cell line were digested with trypsin and labeled with isobaric tags (TMT 10 plex, Pierce). Samples were pooled together and enriched in a single step using ADMA antibodies2 (Cell Signaling). Peptides were analyzed on Orbitrap Fusion LumosÂ® mass spectrometer.

### Data Protocol
Raw data were processed with COMPASS software suite. P values were calculated using student T test and corrected for multiple hypothesis testing3. MotifX online software4 was used to extract enriched motifs.

### Publication Abstract
Protein arginine methyltransferases (PRMTs) introduce arginine methylation, a post-translational modification with the increasingly eminent role in normal physiology and disease. PRMT4 or coactivator-associated arginine methyltransferase 1 (CARM1) is a propitious target for cancer therapy; however, few CARM1 substrates are known, and its mechanism of substrate recognition is poorly understood. Here we employed a quantitative mass spectrometry approach to globally profile CARM1 substrates in breast cancer cell lines. We identified &gt;130 CARM1 protein substrates and validated in vitro &gt;90% of sites they encompass. Bioinformatics analyses reveal enrichment of proline-containing motifs, in which both methylation sites and their proximal sequences are frequently targeted by somatic mutations in cancer. Finally, we demonstrate that the N-terminus of CARM1 is involved in substrate recognition and nearly indispensable for substrate methylation. We propose that development of CARM1-specific inhibitors should focus on its N-terminus and predict that other PRMTs may employ similar mechanism for substrate recognition.

### Keywords
Mcf7, Bottom-up, Orbitrap fusion lumos, Mda-mb-231, Arginine methylation

### Affiliations
University of Wisconsin - Madison
Dept of Biomolecular Chemistry Univ of Wisconsin - Madison Madison WI USA

### Submitter
Evgenia Shishkova

### Lab Head
Dr Joshua J Coon
Dept of Biomolecular Chemistry Univ of Wisconsin - Madison Madison WI USA


