### Accession
PXD024663

### Title
Liver-dependent lung remodeling in the wake of systemic inflammation shapes responses to secondary infection

### Description
:Systemic duress, like that elicited by sepsis, burns or trauma, predispose patients to nosocomial pneumonia, demanding better understanding of host pathways influencing this deleterious connection. These pre-existing circumstances are capable of triggering the hepatic acute phase response (APR), which we previously demonstrated is essential for limiting susceptibility to secondary lung infections. To identify potential mechanisms underlying protection afforded by the lung-liver axis, our studies aimed to evaluate liver-dependent lung reprogramming when a systemic inflammatory challenge precedes pneumonia. WT mice and APR-deficient littermates lacking hepatocyte STAT3 (hepSTAT3-/-), a transcription factor necessary for full APR initiation, were challenged intraperitoneally with LPS to induce endotoxemia. After 18h, pneumonia was induced by intratracheal E. coli instillation. Endotoxemia elicited significant transcriptional alterations in the lungs of WT and hepSTAT3-/- mice, with nearly 2,000 differentially expressed genes between genotypes. The gene signatures revealed exaggerated immune activity in the lungs of hepSTAT3-/- mice, which were compromised in their capacity to launch additional cytokine responses to secondary infection.Â  Proteomics revealed substantial liver-dependent modifications in the airspaces of pneumonic mice, implicating a network of dispatched liver-derived mediators influencing lung homeostasis. These results indicate that following systemic inflammation, liver acute phase changes dramatically remodel the lungs, resulting in a modified landscape for any stimuli encountered thereafter. Based on the established vulnerability of hepSTAT3-/- mice to secondary lung infections, we believe that intact liver function is critical for maintaining the immunological responsiveness of the lungs. Further studies are needed to confirm whether and how such lung changes directly influence pneumonia susceptibility.

### Sample Protocol
BALF samples were sent to the Yale MS & Proteomics Resource (New Haven, CT), where samples they were processed and analyzed via a Label Free Quantification workflow as described in Charkoftaki et al (2019) . The sample preparation was slightly modified from Charkoftaki et al (2019) protocol since these were BALF fluids; briefly, samples were filtered through a 3kDa Amicon Ultra filter and retentate were speedvac dried and utilized for downstream proteomics preparation.  Dried protein pellet were reduced with DTT, alkylated with iodoacetamide, enzymatically digested with LysC and Trypsin, and desalted utilizing C18 RP microspin column.  High resolution LC MS/MS data were collected on an Orbitrap Fusion mass spectrometer coupled to a NanoACQUITY UPLC.

### Data Protocol
Data were analyzed utilizing Progenesis QI (Waters, Inc, Milford, MA) and Mascot search engine.  Quantitative data were normalized based on equal total amount of peptides/proteins injected on column, and positive protein identification and quantitation were based on hits with two or more unique peptides per protein, experimental groups were compared and considered statistically significant when the FDR q value was <0.05, unless otherwise stated. Ingenuity Pathway Analysis (Qiagen Bioinformatics) was used for further analyses.

### Publication Abstract
None

### Keywords
Pneumonia, Lung, Liver, Sepsis, Acute phase reactants, Rodent, Systemic inflammation, Label free quantitation

### Affiliations
Yale University
Yale University, Department of MBB, Yale MS & Proteomics Resource

### Submitter
TuKiet Lam

### Lab Head
Dr TuKiet Lam
Yale University, Department of MBB, Yale MS & Proteomics Resource


