### Accession
PXD016944

### Title
Quick steps to get glioma extracellular vesicles surface membrane proteins mapping based on tryspin/Lysin C digestion followed by Amicon and shot gun proteomic: identification of ghost protein

### Description
Extracellular vesicles (EVs) are known to mediate intercellular communication and regulate a diverse range of crucial biological processes. Novel therapeutic strategies have emerged based on these EVs as biological nanoparticles. EVs is as a generic term for all secreted vesicles released including at least ectosomes, microvesicles, apoptotic bodies and exosomes. In order to determine if it is possible to get access to the protein present at the surface of the EVs, we developed first a quickstep strategy. We performed a Trypsin/Lys C digestion treatment for EVs pellet, followed by an Amicon filtration. This procedure sep-arates the EVs from peptides of protein aggregates and external part of EVs membrane proteins. After separation step, all the fractions have been subjected of proteomic analyses. Comparison between T-0h and T-6h, Trypsin/Lysin C digestion revealed an evolution of the surface EVs proteins. Some surface proteins have been demasked after 6h enzymatic digestion like CD81, CD82 ,Ust, Vcan, Lamp 1,rab43, Cspg4, Annexin A2, Synthenin 1,VSP37b. These results established an evolu-tion in time course of the nature of the surface proteins in glioma tumor.

### Sample Protocol
Samples preparation for mass spectrometry analysis. EVs treated with Trypsin/Lys C for 6h (T-6h) or not (T-0h), were subjected of a filtration using Amicon Ultra-0.5 mL Centrifugal Filters 10 KDa for 20min on 14000xg at 4������C. In the Amicon ���������Filtrate��������� tube the peptides of protein aggre-gates and external part of EVs membrane proteins were recovered. The vesicles were recovered from Amicon filter by centrifugation for 5min on 1000xg at 4������C. EVs proteins were extracted with RIPA buffer. Samples were prepared for mass spectrometry analysis by Filter-Aided Sample Preparation (FASP) as previously described [22]. The pro-teins of all fractions were denatured with 2 M urea in 10 mM HEPES, pH 8.0 by sonication. Then they were reduced with 10 mM DTT for 40 min followed by alkylation with 55 mM iodoacetamide for 40 min in the dark. The iodoa-cetamide was quenched with 100 mM thiourea. The pro-teins were digested with 1 ������g LysC/Trypsin mixture over-night at 37������C. The digestion was stopped with 0.5% TFA. The peptides were desalted with a Millipore ZipTip device in a final volume of 20 ������l of 80% ACN elution solution. The solution was then dried using the SpeedVac. Dried samples were solubilized in water/0.1% formic acid before LC MS/MS analysis.  LC MS/MS analysis. Samples were separated by online reversed-phase chromatography using a Thermo Scientific Proxeon Easy-nLC system equipped with a Proxeon trap column (100 ������m ID x 2 cm, Thermo Scientific) and a C18 packed-tip column (75 ������m ID x 50 cm, Thermo Scientific). Peptides were separated using an increasing amount of acetonitrile (5���������35% for 100 min) at a flow rate of 300 nL/min. The LC eluent was electrosprayed directly from the analytical column, and a voltage of 1.7 kV was applied via the liquid junction of the nanospray source. The chro-matography system was coupled with a Thermo Scientific Q Exactive mass spectrometer programmed to acquire a data dependent Top 10 method. Survey scans were ac-quired at a resolution of 70 000 at m/z 400.

### Data Protocol
Data analyses. All the MS data were processed with MaxQuant (version 1.5.1.2) [23] using the Andromeda search engine. Proteins were identified by searching MS and MS/MS data against Decoy version of the complete proteome for Rattus norvegicus of the UniProt database [24] (Release June 2014, 33,675 entries) combined with 262 commonly detected contaminants. The protein data-base was uploaded from Openprot (https://openprot.org/) and included RefProt, novel isoforms and AltProts predict-ed from both Ensembl and RefSeq annotations (GRCh38.83, GRCh38.p7).Trypsin specificity was used for the digestion mode with N-terminal acetylation and methi-onine oxidation selected as the variable. Carbamydometh-ylation of cysteines was set as a fixed modification, and we allowed up to two missed cleavages. For MS spectra, an initial mass accuracy of 6 ppm was selected, and the MS/MS tolerance was set to 20 ppm for HCD data. For identification, the FDR at the peptide spectrum matches (PSMs) and protein level was set to 0.01. Relative, label-free quantification of proteins was performed using the MaxLFQ algorithm[25] integrated into MaxQuant with the default parameters. Analysis of the proteins identified was performed using Perseus software (http://www.perseus-framework.org/) (version 1.5.0.31). The file containing the information from identification was used with hits to the reverse database, and proteins only identified with modi-fied peptides and potential contaminants were removed. Then, the LFQ intensity was logarithmized (log2[x]). Cate-gorical annotation of rows was used to define different groups depending on the following: 1) the cell type, 2) the treatment. Multiple-samples tests were performed using an ANOVA test with a FDR of 5% and preserved grouping in randomization. To determine enrichment of categorical annotations (Gene Ontology terms and KEGG pathway), a Fisher���������s exact test was used, taking in account the results of the ANOVA test for each group. Normalization was achieved using a Z-score with matrix access by rows. Only proteins presenting as significant by the ANOVA tests were used for statistical analysis. A hierarchical clustering was first performed using the Euclidean parameter for distance calculation and an average option for linkage in row and column trees using a maximum of 300 clusters. To quanti-fy fold changes of proteins across samples, we used MaxLFQ. To visualize these fold changes in the context of individual protein abundances in the proteome, we project-ed them onto the summed peptide intensities normalized by the number of theoretically observable peptides.

### Publication Abstract
Extracellular vesicles (EVs) mediate intercellular communication and regulate a broad range of biological processes. Novel therapeutic strategies have emerged based on the use of EVs as biological nanoparticles. To separate isolated EVs from protein aggregates and the external part of EVs membrane proteins, we performed a Trypsin/Lys C digestion treatment of EVs pellets, followed by Amicon filtration. After these steps, all the fractions have been subjected to proteomic analyses. Comparison between 6 h Trypsin/Lys C treatment or non-treated EVs revealed a quantitative variation of the surface proteins. Some surface proteins have been demasked after 6 h enzymatic digestion like CD81, CD82, Ust, Vcan, Lamp 1, Rab43, Annexin A2, Synthenin, and VSP37b. Moreover, six ghost proteins have also been identified and one corresponds to a long noncoding RNA. We thus demonstrate the presence of ghost proteins in EVs produced by glioma cells that can contribute to tumorigenesis.

### Keywords
Vesicle, Ghost-protein, Glioma, Lc-msms

### Affiliations
Laboratoire PRISM INSERM U-1192 Universite de Lille France
PRISM

### Submitter
Soulaimane Aboulouard

### Lab Head
Dr Michel Salzet
Laboratoire PRISM INSERM U-1192 Universite de Lille France


