### Accession
PXD004166

### Title
EpiProfile 2.0: a proteomics tool to quantify histone modifications by mass spectrometry

### Description
The histone LC-MS/MS data analysis is challenging due to the large number and variety of isobaric histone peptides, and the high dynamic range of histone peptide abundances. We introduce EpiProfile 2.0 to quantify histone post-translational modifications from mass spectrometry data, in which fragment ions are used to determine the retention time of peptides and discriminate isobaric peptides. EpiProfile can automatically recognize and efficiently process both data-dependent and data-independent acquisition runs (DDA and DIA), and high-resolution and low-resolution instrument runs. The proteomics tool was tested on human cells and validated by synthetic histone peptides. EpiProfile is the first automatic program to determine the retention time and discriminate isobaric peptides for histone modifications with DIA MS.

### Sample Protocol
Endogenous histones from HeLa cells HeLa Cell Culture - As previously described (1), HeLa cells were cultured in suspension with Minimum Essential Medium Eagle (MEM) Joklik modification for Suspension Cultures with 10% Newborn Calf Serum (Thermo Scientific SH30118.03), 1% Glutamax (Gibco, 35050-079), and 1% Penicillin/Streptomycin (Gibco, 15140-122). The cell density was maintained within the range of 1-10Ã—105 cells/mL. Histones were acid extracted and subjected to chemical derivatization using propionic anhydride and trypsin digestion as previously described (2). Samples were then desalted and loaded by auto-sampler onto an online nanoLC-MS/MS as described in (2). Nanoflow Liquid Chromatography Tandem Mass Spectrometry (nanoLC-MS/MS) - The samples were loaded onto an Orbitrap Velos Pro mass spectrometer at 300 nL/min. Histone peptides were resolved on a two-step gradient from 2% ACN to 30% ACN in 0.1% formic acid over 40 minutes, then from 30% ACN to 95% ACN in 0.1% formic acid over 20 minutes. Orbitrap Velos Pro was operated in the data dependent acquisition (DDA) mode with dynamic exclusion enabled (repeat count: 1, exclusion duration: 0.5 min). MS instrument methods were set up as previously reported (3). Every cycle one full MS scan (m/z 290 to 1600) was collected and followed by 12 MS2 scans using collision induced dissociation (CID). The isolation window was set at 2.0 m/z. Ions with a charge state of one and a rejection list of common contaminant ions (including keratin, trypsin and BSA) (exclusion width=10 ppm) were excluded from MS2 spectra. 1. Zee, B. M., Britton, L. M., Wolle, D., Haberman, D. M., and Garcia, B. A. (2012) Origins and formation of histone methylation across the human cell cycle. Mol Cell Biol 32, 2503-2514. 2. Lin, S., and Garcia, B. A. (2012) Examining histone posttranslational modification patterns by high-resolution mass spectrometry. Methods Enzymol 512, 3-28. 3. Zee, B. M., Levin, R. S., Dimaggio, P. A., and Garcia, B. A. (2010) Global turnover of histone post-translational modifications and variants in human cells. Epigenetics Chromatin 3, 22.

### Data Protocol
When the spectra are ready, they can be used for data analysis. To determine the histone layouts for new organisms, DDA and DIA can be both used, while DDA is for identification and DIA is for validation. DDA data can be searched against the histone database, with lots of variable modifications, e.g. acetylation (ac), mono-, di-, tri-methylation (me1, me2, and me3), phosphorylation (ph), crotonylation (cr), ubiquitination (ub), etc. A lot of search engines can be used. Two of them are recommended for histone identification, i.e. pFind and Mascot. pFind 3.0 is especially easy for histone identification, because both the MS2 and MS1 can be manually checked. In the DDA mode, some low-abundance peptides do not have the MS2 spectra. Even if the chromatographic peaks can be seen, we still do not have the MS/MS spectra. In contrast, DIA mode is a kind of validation. In the DIA mode, both the fragment ions and the chromatographic peaks can be seen. The retention time relationship of PTMs on the same peptide sequence (layout) is important. For example, ac (42.01 Da) and me3 (42.04 Da) are close in mass, but they are different in retention time (ac is later than me3). Isobaric peptides have the same mass, and in some case they co-elute. In this case the unique fragment ions in the MS2 spectra can be used to discriminate them. EpiProfile is developed to generate the layouts of PTMs on the same peptide sequence, discriminate the isobaric peptides, and then quantify the area under curve for each histone peptide.

### Publication Abstract
Epigenetics has become a fundamental scientific discipline with various implications for biology and medicine. Epigenetic marks, mostly DNA methylation and histone post-translational modifications (PTMs), play important roles in chromatin structure and function. Accurate quantification of these marks is an ongoing challenge due to the variety of modifications and their wide dynamic range of abundance. Here we present EpiProfile 2.0, an extended version of our 2015 software (v1.0), for accurate quantification of histone peptides based on liquid chromatography-tandem mass spectrometry (LC-MS/MS) analysis. EpiProfile 2.0 is now optimized for data-independent acquisition through the use of precursor and fragment extracted ion chromatography to accurately determine the chromatographic profile and to discriminate isobaric forms of peptides. The software uses an intelligent retention time prediction trained on the analyzed samples to enable accurate peak detection. EpiProfile 2.0 supports label-free and isotopic labeling, different organisms, known sequence mutations in diseases, different derivatization strategies, and unusual PTMs (such as acyl-derived modifications). In summary, EpiProfile 2.0 is a universal and accurate platform for the quantification of histone marks via LC-MS/MS. Being the first software of its kind, we anticipate that EpiProfile 2.0 will play a fundamental role in epigenetic studies relevant to biology and translational medicine. EpiProfile is freely available at https://github.com/zfyuan/EpiProfile2.0_Family .

### Keywords
Post-translational modification, Quantification, Proteomics, Histone, Mass spectrometry

### Affiliations
UPenn
Epigenetics Program, Department of Biochemistry and Biophysics, Perelman School of Medicine, University of Pennsylvania

### Submitter
Zuo-Fei Yuan

### Lab Head
Dr Benjamin A. Garcia
Epigenetics Program, Department of Biochemistry and Biophysics, Perelman School of Medicine, University of Pennsylvania


