### Accession
PXD000404

### Title
Comparative Proteome Profiling of Bovine and Human Staphylococcus epidermidis Strains to Search for Cell-surface Bound Factors with Plausible Roles in Virulence or Adaptation

### Description
Here, we analyse two SE strains, an infectious bovine strain PM221 isolated from persistent subclinical bovine IMI, and ATCC12228 representing a low-infectious human skin strain, on total proteome level.

### Sample Protocol
Duplicate ATCC12228 and PM221 samples acquired from stationary phase cultures were disrupted, and proteins for SDS-PAGE were extracted. Equal amount of protein from each cell-free extract was separated with SDS-PAGE and in-gel digested with trypsin. The tryptic peptides were analyzed with nano-liquid chromatography-tandem mass spectrometry using Ultimate 3000 nano-liquid chromatograph (Dionex) and QSTAR Elite hybrid quadrupole time-of-flight mass spectrometer (Applied Biosystems/MDS Sciex) with nano-ESI ionization. The samples were first concentrated and desalted on a C18 trap column (10 mm × 150 μm, 3 μm, 120 Å; PROTECOLTM, SGE Analytical Science, Griesheim, Germany) followed by peptide separation on a PepMap100 C18 analytical column (15 cm × 75 μm, 5 μm, 100 Å; LC Packings, Sunnyvale, CA) at 200 nl/min. The peptides were eluted in a linear gradient of acetonitrile from 0%  to 40% in 120 min (in 0.1% formic acid) at a flow rate of 200 nL• min-1. MS data were acquired automatically with Analyst QS 2.0 Software (Applied Biosystems, United States) using SMART IDA. The information-dependent acquisition method consisted of a 0.5-s TOF-MS survey scan of m/z 400–1400. From every survey scan, the two most abundant ions with charge states 2+ to 4+ were selected for product ion scans. Once an ion was selected for MS/MS fragmentation, it was put on an exclusion list for 60 s.

### Data Protocol
Databases of predicted protein sequences encoded by PM221 and ATCC12228 genomes and plasmids were compiled for protein identifications. The MS/MS data were searched against the in-house protein databases using Mascot and ParagonTM search algorithms through the ProteinPilotTM. The search criteria for Mascot searches were: trypsin digestion with one allowed miss-cleavage; carbamidomethyl modification of cysteine as a fixed modification; oxidation of methionine as a variable modification; peptide mass tolerance of 50 ppm; MS/MS fragment tolerance of 0.2 Da; and peptide charges of 1+, 2+, and 3+. The parameters for Paragon searches using the Rapid search mode included the previously mentioned methionine and cysteine modifications. The Compid tool was used to parse significant hits from the compiled Mascot and Paragon output files into tab delimited data files. Protein identifications that had probability-based Mascot Mowse scores ≥ 50 and p < 0.05 and/or Paragon Unused ProtScores ≥ 1.3 and p < 0.05 were considered reliable high-confidence identifications. To estimate the false discovery rates (FDRs), all Mascot and Paragon searches were repeated using identical search parameters and validation criteria against PM221 and ATCC12228 decoy databases containing all protein sequences in both forward and reverse orientations. Sequences were reversed using the Compid tool. The FDR percentages were calculated using the formula 2xnreverse/(nreverse + nforward).

### Publication Abstract
The present study reports a comparative proteome cataloging of a bovine mastitis and a human-associated Staphylococcus epidermidis strain with a specific focus on surfome (cell-wall bound and extracellular) proteins. Protein identification by 1DE coupled with LC-MS/MS analyses resulted in 1400 and 1287 proteins from the bovine (PM221) and human (ATCC12228) strains, respectively, covering over 50% of all predicted and more than 30% of all predicted surfome proteins in both strains. Comparison of the identification results suggests elevated levels of proteins involved in adherence, biofilm formation, signal transduction, house-keeping functions, and immune evasion in PM221, whereas ATCC12228 was more effective in expressing host defense evasion proteases, skin adaptation lipases, hemagglutination, and heavy-metal resistance proteins. Phenotypic analyses showed that only PM221 displays protein- and DNA-mediated adherent growth, and that PM221 was more efficient in cleaving tributyrin, a natural compound of milk fat under low CO2 conditions. These findings are in line with the identification data and suggest that distinct expression of lipases and adhesive surfome proteins could lead to the observed phenotypes. This study is the first extensive survey of S. epidermidis proteomes to date, providing several protein candidates to be examined for their roles in adaptation and virulence in vivo. All MS data have been deposited in the ProteomeXchange with identifier PXD000404 (http://proteomecentral.proteomexchange.org/dataset/PXD000404).

### Keywords
Staphylococcus epidermidis / surfome / gelc-ms/ms / adaptation / host-specificity

### Affiliations
Institute of Biotechnology
Institute of Biotechnology, Proteomic Unit, University of Helsinki, Finland

### Submitter
Pia SiljamÃ¤ki

### Lab Head
Dr Tuula A Nyman
Institute of Biotechnology, Proteomic Unit, University of Helsinki, Finland


