### Accession
PXD040888

### Title
RNA polymerase II acts to promote the organization of chromatin following DNA replication

### Description
In mammals, following the passage of the replisome, nucleosomes lose their defined positioning and transcription contributes to their re-organisation. However, whether transcription plays a greater role in the organization of chromatin following DNA replication remains unclear. 
The data uploaded are associated to the iPOND-TMT data uploaded under PXD046514 where protein re-association to newly replicated DNA was monitored upon transcription inhibition. To see whether changes observed in the iPOND-TMT time course experiments (PXD046514) are due to changes in overall protein abundance, whole extract samples were taken for each time point (Nascent, 1h and 2h after EdU labelling) and analysed by TMT-Mass Spec.

### Sample Protocol
To inhibit transcription initiation or elongation 2 hours prior EdU labelling, media was changed for media containing 1μM triptolide or 50 μM 5,6-dichlorobenzimida-zole 1-b-D-ribofuranoside, respectively. Control cells were treated with DMSO. Inhibitors and DMSO were included for the duration of EdU labelling and chromatin maturation in all experiments. The iPOND was performed as described by Sirbu et al., 2012, with a few modifications. In brief, 1 × 10e8 asynchronised TIG-3 cells (per time point) were treated 2 hours with inhibitor and then labelled with 20 μM EdU for 11 min. The thymidine chased sample were washed twice and subsequently further incubated in medium containing 20 μM thymidine (-/+ inhibitor) for the indicated times. All samples were crosslinked with 1% formaldehyde for 15 min at room temperature followed by 5 min incubation with 0.125 M glycine to quench the formaldehyde. Afterwards, cells were scraped from the plate (on ice), permeabilized in 0.25% Triton X-100 in PBS for 30 min at room temperature and washed with 0.5% BSA in PBS. To conjugate biotin to EdU-labeled DNA, click chemistry reactions were performed for 1.5h in the dark. Cells were lysed (1% SDS in 50 mM Tris, pH 8.0, protease inhibitor), sonicated (Bioruptor, Diagenode), the lysate diluted 1:1 (v/v) with cold PBS containing protease inhibitor, and streptavidin beads (Thermo Fisher, 65002) used to capture the biotin-conjugated DNA-protein complexes. Before adding the Streptavidin beads, a sample of the whole cell lysate was collected for each time point to be analysed together with the iPOND pulldown.

### Data Protocol
MaxQaunt software was used for the database searching and TMT quantification of proteins. The three replicates SB45, SB46 and SB47 were searched together, with the fasta and parameter files uploaded.

### Publication Abstract
Understanding how chromatin organisation is duplicated on the two daughter strands is a central question in epigenetics. In mammals, following the passage of the replisome, nucleosomes lose their defined positioning and transcription contributes to their re-organisation. However, whether transcription plays a greater role in the organization of chromatin following DNA replication remains unclear. Here we analysed protein re-association with newly replicated DNA upon inhibition of transcription using iPOND coupled to quantitative mass spectrometry. We show that nucleosome assembly and the re-establishment of most histone modifications are uncoupled from transcription. However, RNAPII acts to promote the re-association of hundreds of proteins with newly replicated chromatin via pathways that are not observed in steady-state chromatin. These include ATP-dependent remodellers, transcription factors and histone methyltransferases. We also identify a set of DNA repair factors that may handle transcription-replication conflicts during normal transcription in human non-transformed cells. Our study reveals that transcription plays a greater role in the organization of chromatin post-replication than previously anticipated.

### Keywords
Dna replications, Proteomics, Tmt

### Affiliations
Centre for Gene Regulation and Expression, School of Life Science, University of Dundee
University of Dundee

### Submitter
Susanne Bandau

### Lab Head
Dr Constance Alabert
Centre for Gene Regulation and Expression, School of Life Science, University of Dundee


