### Accession
PXD024086

### Title
Quantitative Proteomics Identifies Protein Tyrosine Phosphatase 1B as Modulator of B Cell Antigen Receptor Signaling

### Description
B cell antigen receptor (BCR) signaling is initiated by protein kinases and limited by counteracting phosphatases that currently are less well studied in their regulation of BCR signaling. We here used the B cell line Ramos to identify and quantify human B cell signaling components. Specifically, a protein tyrosine phosphatase profiling revealed a high expression of the protein tyrosine phosphatase 1B (PTP1B) in Ramos and human naive B cells. The loss of PTP1B leads to increased B cell activation. Through substrate trapping in combination with quantitative mass spectrometry, we identified 22 putative substrates or interactors of PTP1B. We validated Igalpha, CD22, PLCgamma1/2, CBL, BCAP and APLP2 as specific substrates of PTP1B in Ramos B cells. The tyrosine kinase BTK and the two adaptor proteins GRB2 and VAV1 were identified as direct binding partners and potential substrates of PTP1B. We showed that PTP1B dephosphorylates the inhibitory receptor protein CD22 at phosphotyrosine 807. We conclude that PTP1B negatively modulates BCR signaling by dephosphorylating distinct phosphotyrosines in B cell specific receptor proteins and various downstream signaling components.

### Sample Protocol
Co-Immunoprecipitation 2x10^8 cells were lysed for 25 min in 5 ml ice-cold lysis buffer (50 mM Tris-HCl, pH 8.5, 137.5 mM NaCl, 1 mM EDTA, 1% Triton X-100, 1 mM sodium fluoride, EDTA-free protease inhibitor cocktail [Roche]). 1 mM orthovanadate was added to the lysis buffer to prevent binding of substrates to the PTP1B trapping mutant. Lysates were cleared by centrifugation for 10 min at 4,969xg. A small aliquot was taken for Western blot analysis of the lysate, the remaining sample was mixed with 40 µl FLAG M2 magnetic beads (Sigma). The samples were incubated for 2-4 h on a rotating wheel (10 rpm) at 8°C, washed 3x with 1 ml lysis buffer and eluted with 40 µl 1x sample buffer (10% glycerol, 1% ??mercaptoethanol, 2% SDS, 50 mM Tris-HCl, pH 6.8, 0.004% bromophenol blue).  For MS analysis, the proteins were eluted with 40 µl 3x FLAG peptide (Sigma-Aldrich) and collected by centrifugation for 30 min at 100xg.  In-Gel Protein Digestion Co-IP samples were loaded and separated on a 4-20% Tris-Glycine gel (Invitrogen). Proteins were stained with colloidal Coomassie Brilliant Blue G-250. In gel-digestion of proteins was performed essentially as described before (Schwarz et al. 2015) with the exception that protein thiols were reduced in 5 mM tris (2-chloroethyl)phosphine (TCEP) for 10 min at 60°C and alkylated in 100 mM 2-chloroacetamide (CAM) for 15 min at 37°C. High-Performance Liquid Chromatography and Mass Spectrometry Reversed-phase HPLC was performed on an UltimateTM 3000 RSLCnano system (Thermo Fisher Scientific, Dreieich, Germany) equipped with two PepMapTM C18 ?-precolumn (0.3 mm × 5 mm, 5 µm, 300 Å Thermo Fisher Scientific) and an AcclaimTM PepMapTM analytical column (75 ?m × 250 mm, 3 ?m, 100 Å, Thermo Fisher Scientific).  For LC-MS analysis of affinity-purified samples, the RSLC system was coupled to a Velos Orbitrap Elite instrument (Thermo Fisher Scientific, Bremen, Germany). A binary solvent system was used with solvent A consisting of 0.1% formic acid (FA) and 4% DMSO and solvent B consisting of 48% methanol, 30% ACN, 0.1% FA and 4% DMSO. Peptides from SILAC Co-IPs were washed and concentrated for 5 min with 0.1% TFA on the pre-column. The gradient applied for peptide separation on the analytical column (flow rate of 0.250 µl/min) was as follows: 1% to 65% solvent B in 50 min, increased to 95% B in 5 min, 5 min at 95% B, followed by a decrease to 1% B in 1 min. Subsequently, the column was re-equilibrated for 14 min with 1% B. The Velos Orbitrap Elite instrument was operated with the following parameter settings: spray voltage, 1.5 kV; capillary voltage, 200 V; automatic gain control (AGC), 1x106 ions; max. ion time (IT), 200 ms. Full scans were acquired in the orbitrap with a mass range of m/z 370 to 1,700 and a resolution (R) of 60,000 at m/z 400. A TOP15 method was applied for collision-induced dissociation (CID) of multiply charged peptides performed in the linear ion trap with a normalized collision energy (NCE) of 35% and an activation q of 0.25. The AGC for CID was set to 5,000, the max. IT was 150 ms and the dynamic exclusion time (DE) for previously selected precursor ions was 45 s.

### Data Protocol
Raw data were searched against the Uniprot Human Proteome set (release 2017_06, 93,591 protein entries) using MaxQuant 1.5.5.1 with its integrated search engine Andromeda (Cox and Mann 2008; Cox et al. 2011). The species was restricted to homo sapiens as Ramos B cells are of human origin. MaxQuant default settings were used unless stated otherwise. Database searches were conducted with trypsin, a maximum number of three missed cleavages, and mass tolerance of 4.5 ppm for precursor and 0.5 Da (CID data) or 20 ppm (HCD data) for fragment ions. Carbamidomethylation of cysteine residues was set as fixed modification, and oxidation of methionine, N-terminal acetylation and serine/threonine/tyrosine phosphorylation were applied as variable modifications. The minimum number of unique peptides was set to 1. A false discovery rate (FDR) of 1% was applied to both peptide and protein lists. For SILAC Co-IPs, the multiplicity was set to 3 and “Requantify” was enabled. The maximum number of labelled amino acids was set to 4. For protein quantification, “razer and unique peptides” were used with a minimum ratio count of 2. For the determination of significantly enriched PTP1B substrate candidates, non-normalized SILAC ratios were log10-transformed and a mean log10 ratio over at least 2 out of 3 replicates was calculated. P-values were determined with a right-sided t-test in Perseus 1.5.5.3 (Tyanova et al. 2016).

### Publication Abstract
B cell antigen receptor (BCR) signaling is initiated by protein kinases and limited by counteracting phosphatases that currently are less well studied in their regulation of BCR signaling. Here, we used the B cell line Ramos to identify and quantify human B cell signaling components. Specifically, a protein tyrosine phosphatase profiling revealed a high expression of the protein tyrosine phosphatase 1B (PTP1B) in Ramos and human na&#xef;ve B cells. The loss of PTP1B leads to increased B cell activation. Through substrate trapping in combination with quantitative mass spectrometry, we identified 22 putative substrates or interactors of PTP1B. We validated Ig&#x3b1;, CD22, PLC&#x3b3;1/2, CBL, BCAP, and APLP2 as specific substrates of PTP1B in Ramos B cells. The tyrosine kinase BTK and the two adaptor proteins GRB2 and VAV1 were identified as direct binding partners and potential substrates of PTP1B. We showed that PTP1B dephosphorylates the inhibitory receptor protein CD22 at phosphotyrosine 807. We conclude that PTP1B negatively modulates BCR signaling by dephosphorylating distinct phosphotyrosines in B cell-specific receptor proteins and various downstream signaling components.

### Keywords
Ptp1b, Bcr signaling, Substrate trapping, Ramos b cells, Silac, Co-ip

### Affiliations
University of Freiburg
Biochemistry and Functional Proteomics, Institute of Biology II, Faculty of Biology, University of Freiburg, 79104 Freiburg, Germany

### Submitter
Jennifer Schwarz

### Lab Head
Dr Bettina Warscheid
Biochemistry and Functional Proteomics, Institute of Biology II, Faculty of Biology, University of Freiburg, 79104 Freiburg, Germany


