### Accession
PXD030881

### Title
TMT 10-plex based quantitative proteomic analysis of drug resistant breast cancer cell lines

### Description
Breast cancer accounts for roughly 30% of all cancers in women worldwide, has a 15% death rate, and incidence rates are increasing at a rate of about 0.5% per year. Breast cancer comprises a heterogeneous group of tumor subtypes, whether defined by the histopathology of the primary tumor, the expression pattern of hormone receptors (estrogen and/or progesterone receptors; ER/PR) and epidermal growth factor receptor 2 (HER2), genetic alterations of transcriptomic traits. These patient-to-patient differences (as known as �쁦ntertumoral heterogeneity��, largely affect patient prognosis and treatment options. Alongside intertumoral heterogeneity, many studies reported that breast cancers heterogeneous consisting of many different cells or subclones of which different gene expression profiles within a patient�셲 primary tumor and individual metastases. These differences within the tumor are referred to as intratumor heterogeneity, which is caused by a combination of extrinsic factors from the tumor microenvironment and intrinsic parameters including genetic, epigenetic and transcriptomic traits, ability of proliferation, migration and invasion, cell plasticity, and the extent of stemness. These heterogeneities endow tumors with multiple capabilities and biological characteristics, making them more prone to metastasis, recurrence, and drug resistance.  To overcome these facing challenges, understanding the proteome mechanisms behind transcriptome profiling from the aspect of treatment can help to improve resistance to cancer therapy. Recent proteomics technologies based on mass spectrometry enable an unbiased investigation of drug-induced changes in protein abundance and post-translational modifications. Several studies on resistance to chemotherapy have recently published data on mass spectrometry-based chemotherapeutic proteome profiling, which has the potential to discover molecular subtypes and related pathway features that may have been missed in prior transcriptome analyses. Nevertheless, few proteomics studies to date explore three types of drug-specific resistance of breast cancer signatures. In this study, we employed tandem mass tag (TMT) based proteomics technology to process the acquired mass spectrometry data to test the hypothesis that the chemotherapy in breast cancer cells may have distinct protein profiles that may result in their drug properties and new clinical implications. By unraveling the protein signatures across tamoxifen, doxorubicin, and paclitaxel and their relationship between drug-resistant cell lines and normal breast cancer cells, our study advances the understanding of drug-specific resistance and provides potential diagnostic and prognostic markers, as well as testable targets of therapy specific to breast cancer resistant cells.

### Sample Protocol
Cell samples were prepared with the FASP procedure described previously. Before TMT labeling, the protein concentration of each sample was measured with the tryptophan assay. TMT labeling was then performed with several modifications to the manufacturer�셲 protocol. Briefly, the TMT reagent (0.8 mg) was dissolved in acetonitrile, and 10 ul of the reagent was added to 30 ug of peptides along with ACN to give a final concentration of 30% v/v. The peptides were then pooled after incubating at room temperature for 2 hours. The pooled sample was dried almost completely and desalted with an in-house C18 stage-tip.

### Data Protocol
After the MS analysis, all raw files were processed in Proteome Discoverer 2.4 software based on the SEQUEST-HT search engine against the Human Uniprot FASTA database. Parameters were as follows: enzyme, full-trypsin; precursor ion tolerance, 10 ppm; MS/MS ion tolerance, 20 ppm; fixed modifications, TMT tags (N-term and Lys, +229.163 Da) and carbamidomethylation (Cys, +57.021 Da); and variable modifications, oxidation (Met, +15.995 Da). The false discovery rate (FDR) for all peptide-spectrum match (PSM) were adjusted to a 1%, and protein assignments were performed by the Percolator software package. Only the PSMs that contained all six reporter ions. Protein quantitation values were exported for further analysis in Microsoft Excel. Each reporter ion channel was summed across all quantified proteins and normalized assuming equal protein loading of all ten samples

### Publication Abstract
Chemoresistance is a daunting obstacle to the effective treatment of breast cancer patients receiving chemotherapy. Although the mechanism of chemotherapy drug resistance has been explored broadly, the precise mechanism at the proteome level remains unclear. Especially, comparative studies between widely used anticancer drugs in breast cancer are very limited. In this study, we employed proteomics and bioinformatics approaches on chemoresistant breast cancer cell lines to understand the underlying resistance mechanisms that resulted from doxorubicin (DR), paclitaxel (PR), and tamoxifen (TAR). In total, 10,385 proteins were identified and quantified from three TMT 6-plex and one TMT 10-plex experiments. Bioinformatics analysis showed that Notch signaling, immune response, and protein re-localization processes were uniquely associated with DR, PR, and TAR resistance, respectively. In addition, proteomic signatures related to drug resistance were identified as potential targets of many FDA-approved drugs. Furthermore, we identified potential prognostic proteins with significant effects on overall survival. Representatively, PLXNB2 expression was associated with a highly significant increase in risk, and downregulation of ACOX3 was correlated with a worse overall survival rate. Consequently, our study provides new insights into the proteomic aspects of the distinct mechanisms underlying chemoresistance in breast cancer.

### Keywords
Human, Tmt, Breast cancer, Drug resistance

### Affiliations
Proteomics Core Facility, Biomedical Research Institute, Seoul National University Hospital

### Submitter
Dohyun Han

### Lab Head
Dr Dohyun Han
Proteomics Core Facility, Biomedical Research Institute, Seoul National University Hospital


