### Accession
PXD016883

### Title
RNA binding proteins modification by drought stress

### Description
RNA-binding proteins are increasingly recognized as regulatory component of post-transcriptional gene expression. RBPs interact with mRNAs via RNA-binding domains and these interactions affect RNA availability for translation, RNA stability and turn-over thus affecting both RNA and protein expression essential for developmental and stimulus specific responses. Here we investigate the effect of severe drought stress on the RNA-binding proteome to gain insights into the mechanisms that govern drought stress responses at the systems level

### Sample Protocol
Arabidopsis thaliana root cells were treated with 40% polyethylene glycol to induce drought stress, and after 1hr and 4hr treatment, samples were UV crosslinked at 254 nm wavelenghth and the mRNA interactome was captured by oligo(dT) capture. Proteins associated with mRNA were analysed using label-free quantitative mass spectrometry.

### Data Protocol
Protein identification was performed using mascot search engine and re-scored using the Percolator algorithm in Proteome Discoverer v2.1. Classical and non-classical RNA-binding domains were detected using pfam (http://pfam.xfam.org). Gene ontology enrichments were performed using AGRIGO (http://bioinfo.cau.edu.cn/agriGO/) and pathway analysis was done with the KEGG mapper (http://www.kegg.jp/kegg/tool/annotate_sequence.html), which annotates sequences by BlastKOALA. Amino acid composition enrichment was determined using the web-based composition profiler program (http://www.cprofiler.org/). Length and sequences of amino acid were retrieved from TAIR (https://www.arabidopsis.org/tools/bulk/sequences/index.jsp), pI were obtained from TAIR (https://www.arabidopsis.org/tools/bulk/protein/index.jsp) and hydrophobicity values were calculated using the GRAVY calculator (http://www.gravy-calculator.de). Evolutionary conservation of drought stress responsive proteins were analysed using InParanoid version 8 (http://inparanoid.sbc.su.se/cgi-bin/index.cgi).

### Publication Abstract
The spliceosome processes RNAs from a pre-RNA state to a mature mRNA thereby influencing RNA availability for translation, localization, and turnover. It consists of complex structures containing RNA-binding proteins (RBPs) essential for post-transcriptional gene expression control. Here we investigate the dynamic modifications of spliceosomal RBPs under stress and in particular drought stress. We do so by mRNA interactome capture in <i>Arabidopsis thaliana</i> using label free quantitation. This approach identified 44 proteins associated with the spliceosome and further 32 proteins associated with stress granules. We noted a high enrichment in the motifs RDRR and RSRSRS that are characteristic of RNA interacting proteins. Identification of splicing factors reflect direct and/or indirect stress induced splicing events that have a direct effect on transcriptome and proteome changes under stress. Furthermore, detection of stress granule components is consistent with transcriptional arrest. Identification of drought induced stress granule components is critical in determining common abiotic stress-induced foci that can have biotechnological applications. This study may therefore open ways to modify plant stress responses at a systems level through the modification of key spliceosome components.

### Keywords
Spliceosome, Abiotic stress, Rna binding proteins, Drought stress, Mass spectrometry

### Affiliations
King Abdullah University of Science and Technology
KAUST

### Submitter
claudius Marondedze

### Lab Head
Dr Christoph Gehring
King Abdullah University of Science and Technology


