### Accession
PXD031533

### Title
Comparative proteomics of CTS effect on human lung cancer H1975

### Description
Cryptotanshinone (CTS) is a lipophilic constituent of Salvia miltiorrhiza, with a broad-spectrum anticancer activity. We have observed that CTS enhances the efficacy of gefitinib in human lung cancer H1975 cells, yet little is known about its molecular mechanism. To explore how CTS enhances H1975 cell sensitivity to gefitinib, we figured out differential proteins of H1975 cells treated by gefitinib alone or in combination with CTS using label-free liquid chromatography-mass spectrometry (LC-MS). Gene Ontology (GO), Kyoto Encyclopedia of Genes and Genomes (KEGG), and protein-protein interaction (PPI) bioinformatic analyses of the differential proteins were performed. CTS enhanced H1975 cell sensitivity to gefitinib in vitro and in vivo, with 115 and 128 differential proteins identified respectively. GO enrichment, KEGG analysis, and PPI network comprehensively demonstrated that CTS mainly impacted the redox process and fatty acid metabolism in H1975 cells. Moreover, three differential proteins were validated by RT-qPCR and Western blot. In conclusion, we used a proteomic method to study the mechanism of CTS enhancing gefitinib sensitivity in H1975 cells. Our finding reveals the potential protein targets of CTS in overcoming gefitinib resistance which may be therapeutical targets in lung cancer.

### Sample Protocol
Protein extraction  Treated cells were washed with PBS (Gibco, USA) and lysed with RIPA containing 1% PMSF (both from Beyotime, China) on ice for 30 min. Cell lysate was scraped, collected and sonicated with 35% amplitude on ice for 10-15 s. After centrifuged at 12000 g, 4 ℃ for 20 min in centrifuge (Eppendorf, Germany), supernatant was obtained as cellular protein solution. For tissue sample, 100 mg tissue in 1 mL RIPA containing 1% PMSF was homogenated four times under 5000 rmp for 8 s with a 10 s interval by the homogenator (Birtin, France). After centrifuged at 12000 g, 4 ℃ for 20 min, supernatant was obtained as tissue protein solution. Protein concentrations were measured with BCA Protein Assay Kit (Thermo, USA). Acetone precipitation and enzymatic hydrolysis  The protein solution (300 μg protein diluted with 50mM NH4HCO3 to 100 μL) was added with 400 μL cold acetone, mixed and precipitated overnight at 4°C. After centrifuged at 15000g, 4°C for 30 min, supernatant was discarded and the precipitate was washed with cold acetone, 70% ethanol, and acetone in sequence, 500 μL for each. The sample was centrifuged at 15000 g, 4°C for 30 min after each wash and supernatant was discarded. After evaporated to dry in a fume hood, the final precipitate was resuspended with 50 μL UA buffer, shaken at room temperature for 2 h until the precipitate was completely dissolved. 2 μL diluted DTT solution was added (DTT final concentration = 2 mM), mixed and the sample was placed in 30°C water bath for 1.5 h. Then 13 μL diluted IAM solution was added (IAM final concentration=10 mM), mixed and the sample was placed in darkness for 40 min. Then it was diluted with 50mM NH4HCO3 to 600 μL (Urea final concentration = 0.7 mol/L), added with 20 μL Trypsin (Promega, USA), mixed and hydrolyzed overnight at 37℃. Then 10% TFA was used to stop the reaction by acidifying the solution to a final concentration of 0.4%. C18 Stage Tip peptide desalting To activate C18 Stage Tip, 200 μL methanol was added, centrifuged at 1200 g for 10 min, and the effluent was discarded, repeated three times. To equilibrate Tip, 200 μL 80% ACN/0.1% FA was added, centrifuged at 4000 g for 4 min, and the effluent was discarded, repeated three times. Then 200 μL 0.1% TFA was added to Tip, centrifuged at 6000 g for 4 min, the effluent was discarded, repeated three times. 5 μg digested protein sample was dissolved in 200 μL 0.1% TFA, and the sample was loaded twice, centrifuged at 2000 g for 12 min, and the effluent was discarded. For salt elution 200 μL 0.1% TFA was added to Tip, centrifuged at 6000 g for 4 min, and the effluent was discarded. For TFA elution, 200 μL 0.1% FA was added to Tip, centrifuged at 6000 g for 4 min, and the effluent was discarded, repeated twice. Then 180 μL 80% ACN/0.1% FA was added to Tip, centrifuged at 2000 g for 4 min, repeated twice. The effluent was collected and vacuum-dried overnight. The dried sample was redissolved with 10 μL 0.1% FA, centrifuged at 16000 g for 10 min. The supernatant was transferred to the sample bottle.  LC-MS/MS Analysis The C18 column (75 μm×150 mm, NanoViper C18, 2 μm, 100 μ; Thermo, PA, USA) was used to separate peptide samples in 0.1% formic acid in the Nano LC-Q Exactive Plus mass spectrometer (Thermo, USA). The eluent A was 0.1% formic acid aqueous solution and eluent B was 0.1% formic acid 80% acetonitrile solution. The elution gradient is set as follows: 0~95 min, 3%→32% (B); 95~105 min, 32%→100% (B); 105~120 min, 100% (B), the total running time is 120 min at a flow rate of 300 nL/min. All MS and MS2 spectra were collected, where 10 strongest ions are fragmented by collision-induced dissociation. A full mass scan in the range of 355~1700 m/z was obtained, with a mass resolution of 70,000.

### Data Protocol
The LC-MS/MS data was analyzed by searching the UniProt Human database (https://www.uniprot.org/proteomes/UP000005640) downloaded on December 7, 2017 using Thermo Proteome Discoverer 2.2. Parameters were set and some modifications were made as described: Fixed modification: ureido modification; Dynamic modification: oxidation modification; Enzyme: trypsin; Precursor mass tolerance: 20 ppm, maximum; Missing cut sites: 2; Fragment quality tolerance: 0.02 Da; Target false discovery rate (FDR) (strict): 0.01; Target FDR (loose): 0.05. Verification was based on q value. Result was exported from Thermo Proteome Discoverer 2.2 to Excel for analysis. Proteins meeting all the criteria (p<0.05, high protein FDR confidence, and abundance ratio>1.5 or <0.67) were recognized as differential proteins.

### Publication Abstract
None

### Keywords
Human lung cancer h1975, Lc-ms/ms

### Affiliations
Sun Yat-sen University

### Submitter
Gaofan Sheng

### Lab Head
Dr Jing Jin
Sun Yat-sen University


