### Accession
PXD001026

### Title
Systematic evaluation of quantotypic peptides for targeted analysis of the human kinome

### Description
In targeted proteomics it is critical that peptides are not only proteotypic, but also accurately represent the level of the protein (quantotypic). Numerous approaches are used to identify proteotypic peptides, but quantotypic properties are rarely assessed. Here, we show that measuring ratios of proteotypic peptides across biological samples can be used to empirically identify peptides with good quantotypic properties, and use this to identify quantotypic peptides for 21% of the human kinome.

### Sample Protocol
Cells were washed in PBS, lysed in PLC buffer, enriched for nucleotide binding proteins using ActivX desthiobiotin-ATP or ADP, separated by SDS-PAGE and gel bands cut. Proteins were digested in-gel and eluted in 50% ACN, 5% TFA, lyophilized and resuspended in 0.1% FA. All samples for DDA were analysed on a LTQ Orbitrap Velos mass spectrometer (Thermo Scientific) coupled to a NanoLC-Ultra 2D (Eksigent). All samples for SRM were analysed on a TSQ Vantage Triple Quadrupole (Thermo Scientific) coupled to a NanoLC-Ultra 1D (Eksigent).

### Data Protocol
For DDA analysis, raw data files produced in Xcalibur software (Thermo Scientific) were processed in Proteome Discoverer v1.4 (Thermo Scientific) and searched against the Uniprot human database using Macsot (v2.2). Searches were performed with a precursor mass tolerance set to 10ppm, fragment mass tolerance set to 0.5 Da and a maximum number of missed cleavages set to 2. Static modifications was limited to carbamidomethylation of cysteine, and variable modifications used were oxidation of methionine and deamidation of asparagine / glutamine residues. Peptides were further filtered using a mascot significance threshold <0.05, a peptide ion score >20 and a FDR <0.01 (evaluated by Percolator). For SRM analysis, raw data files produced in Xcalibur software (Thermo Scientific) were analysed using Skyline (Maccoss lab) and Pinpoint (Thermo Scientific). We used the extracted ion chromatograms for the 2 most intense transitions to determine the peptide abundance. These were summed together to get an area per peptide, and these areas for were summed with all peptides per protein to acquire final protein areas.

### Publication Abstract
In targeted proteomics it is critical that peptides are not only proteotypic but also accurately represent the level of the protein (quantotypic). Numerous approaches are used to identify proteotypic peptides, but quantotypic properties are rarely assessed. We show that measuring ratios of proteotypic peptides across biological samples can be used to empirically identify peptides with good quantotypic properties. We applied this technique to identify quantotypic peptides for 21% of the human kinome.

### Keywords
Quantotypic, Bioinformatics, Srm

### Affiliations
CRUK Manchester Institute
CRUK MI, ICR

### Submitter
Jonathan Worboys

### Lab Head
Dr Claus JÃ¸rgensen
CRUK Manchester Institute


