### Accession
PXD040814

### Title
Osteosarcoma cells-derived secretome induce pre-metastatic changes in the lung microenvironment that support lung metastasis formation

### Description
To investigate the molecular and cellular alterations occurring in the lung tissue during the pre-metastatic niche formation in osteosarcoma, we used two two experimental mouse models and a multi-omics approach.

### Sample Protocol
Proteins were reduced and alkylated with 100 mM Tris pH 8.5, 1% sodium deoxycholate, 10 mM tris(2-carboxyethyl)phosphine (TCEP), and 40 mM chloroacetamide for 10 minutes at 95ÂºC at 1000 rpm (Thermomixer, Eppendorf). 100 micrograms of protein were processed for proteomics analysis following the solid-phase-enhanced sample-preparation (SP3) protocol as described by Hughes et al. Enzymatic digestion was performed with Trypsin/LysC (2 micrograms) overnight at 37ÂºC at 1000 rpm. The resulting peptides were cleaned-up and desalted with C18 micro columns and further quantified. Protein identification was performed by nanoLC-MS/MS. This equipment is composed of an Ultimate 3000 liquid chromatography system coupled to a Q-Exactive Hybrid Quadrupole-Orbitrap mass spectrometer (Thermo Scientific, Bremen, Germany). Peptides were loaded onto a trapping cartridge (Acclaim PepMap C18 100Ã…, 5 mm x 300 Âµm i.d., 160454, Thermo Scientific) in a mobile phase of 2% ACN, 0.1% FA at 10 ÂµL/min. After 3 min loading, the trap column was switched in-line to a 50 cm by 75Î¼m inner diameter EASY-Spray column (ES803, PepMap RSLC, C18, 2 Î¼m, Thermo Scientific, Bremen, Germany) at 300 nL/min. Separation was generated by mixing A: 0.1% FA, and B: 80% ACN, with the following gradient:  5 min (2.5% B to 10% B), 120 min (10% B to 30% B), 20 min (30% B to 50% B), 5 min (50% B to 99% B) and 10 min (hold 99% B). Subsequently, the column was equilibrated with 2.5% B for 17 min. Data acquisition was controlled by Xcalibur 4.0 and Tune 2.9 software (Thermo Scientific, Bremen, Germany). The mass spectrometer was operated in data-dependent (dd) positive acquisition mode alternating between a full scan (m/z 380-1580) and subsequent HCD MS/MS of the 10 most intense peaks from the full scan (normalized collision energy of 27%). ESI spray voltage was 1.9 kV.  Global settings: use lock masses best (m/z445.12003), lock mass injection Full MS, chrom. peak width (FWHM) 15s. Full scan settings: 70k resolution (m/z 200), AGC target 3e6, maximum injection time 120 ms. dd settings: minimum AGC target 8e3, intensity threshold 7.3e4, charge exclusion: unassigned, 1, 8, >8, peptide match preferred, exclude isotopes on, dynamic exclusion 45s. MS2 settings: microscans 1, resolution 35k (m/z 200), AGC target 2e5, maximum injection time 110 ms, isolation window 2.0 m/z, isolation offset 0.0 m/z, spectrum data type profile.   Proteins were reduced and alkylated with 100 mM Tris pH 8.5, 1% sodium deoxycholate, 10 mM tris(2-carboxyethyl)phosphine (TCEP), and 40 mM chloroacetamide for 10 minutes at 95ÂºC at 1000 rpm (Thermomixer, Eppendorf). 100 micrograms of protein were processed for proteomics analysis following the solid-phase-enhanced sample-preparation (SP3) protocol as described by Hughes et al. Enzymatic digestion was performed with Trypsin/LysC (2 micrograms) overnight at 37ÂºC at 1000 rpm. The resulting peptides were cleaned-up and desalted with C18 micro columns and further quantified. Protein identification was performed by nanoLC-MS/MS. This equipment is composed of an Ultimate 3000 liquid chromatography system coupled to a Q-Exactive Hybrid Quadrupole-Orbitrap mass spectrometer (Thermo Scientific, Bremen, Germany). Peptides were loaded onto a trapping cartridge (Acclaim PepMap C18 100Ã…, 5 mm x 300 Âµm i.d., 160454, Thermo Scientific) in a mobile phase of 2% ACN, 0.1% FA at 10 ÂµL/min. After 3 min loading, the trap column was switched in-line to a 50 cm by 75Î¼m inner diameter EASY-Spray column (ES803, PepMap RSLC, C18, 2 Î¼m, Thermo Scientific, Bremen, Germany) at 300 nL/min. Separation was generated by mixing A: 0.1% FA, and B: 80% ACN, with the following gradient:  5 min (2.5% B to 10% B), 120 min (10% B to 30% B), 20 min (30% B to 50% B), 5 min (50% B to 99% B) and 10 min (hold 99% B). Subsequently, the column was equilibrated with 2.5% B for 17 min. Data acquisition was controlled by Xcalibur 4.0 and Tune 2.9 software (Thermo Scientific, Bremen, Germany). The mass spectrometer was operated in data-dependent (dd) positive acquisition mode alternating between a full scan (m/z 380-1580) and subsequent HCD MS/MS of the 10 most intense peaks from the full scan (normalized collision energy of 27%). ESI spray voltage was 1.9 kV.  Global settings: use lock masses best (m/z445.12003), lock mass injection Full MS, chrom. peak width (FWHM) 15s. Full scan settings: 70k resolution (m/z 200), AGC target 3e6, maximum injection time 120 ms. dd settings: minimum AGC target 8e3, intensity threshold 7.3e4, charge exclusion: unassigned, 1, 8, >8, peptide match preferred, exclude isotopes on, dynamic exclusion 45s. MS2 settings: microscans 1, resolution 35k (m/z 200), AGC target 2e5, maximum injection time 110 ms, isolation window 2.0 m/z, isolation offset 0.0 m/z, spectrum data type profile.

### Data Protocol
The raw data was processed using the Proteome Discoverer software (Thermo Scientific) and searched against the UniProt database for Homo sapiens Proteome. The Sequest HT search engine was used to identify tryptic peptides. The ion mass tolerance was 10 ppm for precursor ions and 0.02 Da for fragment ions.  The maximum allowed for missing cleavage sites was set to 2. Cysteine carbamidomethylation was defined as constant modification. Methionine oxidation and protein N-terminus acetylation were defined as variable modifications. Peptide confidence was set to high.  The processing node Percolator was enabled with the following settings: maximum delta Cn 0.05; decoy database search target FDR 1%, validation based on q-value. Imputation of missing values was performed only when a peptide was detected in at least one of the replicates analyzed. Quantitative evaluation was performed by pairwise comparisons of all detected peptides and the median ratio was used for protein level comparison. Significance assessment was performed using the background-based ANOVA method implemented in Proteome Discoverer 2.2 and multiple comparison adjustment of the p values was performed.  The protein functional enrichment analyses for Gene Ontology (GO) terms and Reactome pathways were performed using the Database for Annotation, Visualization and Integrated Discovery (DAVID) 6.8 bioinformatics tool ( https://david.ncifcrf.gov/summary.jsp). The data analysis and visualization of differently expressed proteins were conducted in the R programming language (version 4.1.1) in the RStudio integrated development environment. The GO â€“ biological process plots were achieved using the ggplot2 (version 3.3.5) and parckcircles package available at https://r-graph-gallery.com/305-basic-circle-packing-with-one-level.html. The Venn diagrams were generated with the webtool available at bioinformatics.psb.ugent.be/webtools/Venn/.

### Publication Abstract
None

### Keywords
Lung metastasis, Transcriptomics, Osteosarcoma, Premetastatic niche, Proteomics, Secretome

### Affiliations
i3S - Instituto de investigação e inovação em Saúde
Laboratory of Pharmacology and Experimental Therapeutics, Institute for Biomedical Imaging and Life Sciences (IBILI), Faculty of Medicine and CNC.IBILI Research Consortium, University of Coimbra, Coimbra, Portugal

### Submitter
Ana Sofia Ribeiro

### Lab Head
Dr Célia Gomes
Laboratory of Pharmacology and Experimental Therapeutics, Institute for Biomedical Imaging and Life Sciences (IBILI), Faculty of Medicine and CNC.IBILI Research Consortium, University of Coimbra, Coimbra, Portugal


