### Accession
PXD016967

### Title
NuRD facilitates lineage decisions in human pluripotent cells

### Description
Differentiation of mammalian pluripotent cells involves large-scale changes in transcription and, among the molecules that orchestrate these changes, chromatin remodellers are essential to initiate, establish and maintain a new gene regulatory network. The NuRD complex is a highly conserved chromatin remodeller which fine-tunes gene expression in embryonic stem cells. While the function of NuRD in mouse pluripotent cells has been well defined, no study yet has defined NuRD function in human pluripotent cells. We investigated the structure and function of NuRD in human induced pluripotent stem cells (hiPSCs). Using immunoprecipitation followed by mass-spectrometry in hiPSCs and in naive or primed mouse pluripotent stem cells, we find that NuRD structure and biochemical interactors are generally conserved. Using RNA sequencing, we find that, whereas in mouse primed stem cells and in mouse naive ES cells, NuRD is required for an appropriate level of transcriptional response to differentiation signals, hiPSCs require NuRD to initiate these responses. This difference indicates that mouse and human cells interpret and respond to induction of differentiation differently.

### Sample Protocol
Label-free Flag pulldowns were performed in triplicate as previously described (https://www.ncbi.nlm.nih.gov/pubmed/30389936).  2 mg of nuclear extract was incubated with 15 ������l Flag-sepharose beads (Sigma) and 50 ������g/mL ethidium bromide in Buffer C (300 mM NaCl, 20 mM HEPES/KOH, pH 7.9, 20% v/v glycerol, 2 mM MgCl2, 0.2 mM EDTA) with 0.1% NP-40, protease inhibitors, and 0.5 mM DTT in a total volume of 400 ������l. After incubation, 6 washes were performed: 2 with Buffer C and 0.5% NP-40, 2 with PBS and 0.5% NP-40, and 2 with PBS. Affinity purified proteins were subject to on-bead trypsin digestion as previously described (https://www.ncbi.nlm.nih.gov/pubmed/23066101). In short, beads were resuspended in 50 ������l elution buffer (2 M urea, 50 mM Tris pH 7.5, 10 mM DTT) and incubated for 20 min in a thermoshaker at 1400 rpm at room temperature. After addition of 50 mM iodoacetamide (IAA), beads were incubated for 10 min at 1400 rpm at room temperature in the dark. Proteins were then on-bead digested into tryptic peptides by addition of 0.25 ������g trypsin (Promega) and subsequent incubation for 2 h at 1400 rpm at room temperature. The supernatant was transferred to new tubes and further digested overnight at room temperature with an additional 0.1 ������g of trypsin. Tryptic peptides were acidified and desalted using StageTips (https://www.ncbi.nlm.nih.gov/pubmed/17703201) prior to mass spectrometry analyses.   Tryptic peptides were separated with an Easy-nLC 1000 (Thermo Scientific). Buffer A was 0.1% formic acid and Buffer B was 80% acetonitrile and 0.1% formic acid.  Mouse epiblast stem cell LFQ samples were separated using a 120-min gradient from 7% until 32% Buffer B followed by step-wise increases up to 95% Buffer B. Mass spectra were recorded on a LTQ-Orbitrap Velos mass spectrometer (Thermo Fisher Scientific), selecting the 10-15 most intense precursor ions of every full scan for fragmentation. The tryptic peptides from human iPS cells were measured by developing a gradient from 9-32% Buffer B for 114 minutes before washes at 50% then 95% Buffer B, for 140 minutes of total data collection time. Mass spectra were recorded on an LTQ-Orbitrap Fusion Tribrid mass spectrometer (Thermo Fisher Scientific). Scans were collected in data-dependent top speed mode with dynamic exclusion set at 60 seconds.

### Data Protocol
Thermo RAW files from human iPS cell LFQ AP-MS/MS were analyzed with MaxQuant version 1.4.0.3 using default settings and searching against the UniProt curated human proteome, release 2013_06. Additional options for Match between runs, LFQ, and iBAQ were selected. Thermo RAW files from mouse epiblast stem cell LFQ AP-MS/MS were analyzed with MaxQuant version 1.3.0.5 using default settings and searching against the UniProt curated mouse proteome, release 2013_06. Additional options for Match between runs, LFQ, and iBAQ were selected.  Stoichiometry calculations and volcano plots were produced essentially as described (https://www.ncbi.nlm.nih.gov/pubmed/20479470, https://www.ncbi.nlm.nih.gov/pubmed/23066101) using Perseus version 1.4.0.8 (https://www.ncbi.nlm.nih.gov/pubmed/27348712) and in-house R scripts. For interactor identification, t-test-based statistics are applied on label-free quantified (LFQ) mass spec data. First, the logarithm (log2) of the LFQ values is taken, resulting in a Gaussian distribution of the data. This allows imputation of missing values by normal distribution, assuming these proteins are close to the detection limit. Statistical outliers for the Flag pull-downs in both tagged and control cell lines are then determined using a two-tailed t-test. Multiple testing correction is applied by using a permutation-based false discovery rate (FDR) method in Perseus.

### Publication Abstract
None

### Keywords
Human, Mouse, Ipsc, Epiblast

### Affiliations
Radboud University
Department of Molecular Biology, Radboud Institute for Molecular Life Sciences, Radboud University, Nijmegen, The Netherlands

### Submitter
Susan Kloet

### Lab Head
Dr Michiel Vermeulen
Department of Molecular Biology, Radboud Institute for Molecular Life Sciences, Radboud University, Nijmegen, The Netherlands


