### Accession
PXD030130

### Title
Age-related differences in structure and function of nasal epithelial cultures from healthy children and elderly people

### Description
The nasal epithelium represents the first line of defense against inhaled pathogens, allergens, and irritants and plays a key role in the pathogenesis of a spectrum of acute and chronic airways diseases. Despite age-dependent clinical phenotypes triggered by these noxious stimuli, little is known about how ageing affects the structure and function of the airway epithelium that is crucial for lung homeostasis and host defense. The aim of this study was therefore to determine age-related differences in structural and functional properties of primary nasal epithelial cultures from healthy children and non-smoking elderly people. To achieve this goal, highly differentiated nasal epithelial cultures were established from nasal brushes at air-liquid-interface and used to study epithelial cell type composition, mucin (MUC5AC and MUC5B) expression and ion transport properties. Further, we determined age-dependent molecular signatures using global proteomic analysis. We found lower numeric densities of ciliated cells and higher levels of MUC5AC expression in cultures from children vs. elderly people. Bioelectric studies showed no differences in basal ion transport properties, ENaC-mediated sodium absorption or CFTR-mediated chloride transport, but detected decreased calcium-activated TMEM16A-mediated chloride secretory responses in cultures from children vs. elderly people. Proteome analysis identified distinct age-dependent molecular signatures associated with ciliation and mucin biosynthesis, as well as other pathways implicated in ageing. Our data identified intrinsic, age-related differences in structure and function of the nasal epithelium and provide a basis for further studies on the role of these findings in age-dependent airways disease phenotypes observed with a spectrum of respiratory infections and other noxious stimuli.

### Sample Protocol
Preparation of cell lysates  Filters were washed with PBS and 80 ¬µl RIPA buffer (Thermo Fisher Scientific, Waltham, MA, USA) containing cOmplete protease inhibitor (Merck, Darmstadt, Germany) was added, the cells were scraped 159 and vortexed briefly. 3 filters/individual were pooled. After 30 min incubation on ice, the lysates were cleared by centrifugation. The protein concentration of the supernatant was determined using Pierce‚Ñ¢ BCA Protein Assay Kit (Thermo Fisher Scientific, Waltham, MA, USA), according to the manufacturer‚Äôs instructions.   Sample preparation for proteomic analysis  100 ¬µg of protein was transferred to AFA tubes (PN 520292, 500639) and filled to 60 ¬µl with RIPA buffer. Proteins were extracted and DNA sheared DNA (Covaris LE220Rsc: PIP 350 W, DF 25%, CPB 200, 2 repeats, 300 s pulse, 20 C). 25 ¬µg of protein was used for SP3 protein preparation on a Biomek i7 workstation with single-step reduction and alkylation (M√ºller et al., 2020). Briefly, 16.6 Œºl reduction and alkylation buffer (40 mM TCEP, 160 mM CAA, 200mM ABC) were added, samples incubated for 5 min at 95 ¬∞C and cooled to room temperature. Proteins were bound to 2.5 Œºg paramagnetic beads (1:1 ratio of hydrophilic/hydrophobic beads) by adding acetonitrile (ACN) to 50 %. Samples were washed twice with 80% ethanol and once with 100% ACN, before reconstitution in 35 Œºl 100 mM ABC. Digestion was completed overnight at 37 ¬∞C using a trypsin/ LysC enzyme mix (Promega, Madison, WI, USA) at a ratio of protein:enzyme of 50:1 (w/w) and stopped with formic acid (0.1%). The peptides were stored at -80 ¬∞C until analysis by LC-MS/MS without further conditioning or clean-up.    Liquid chromatography mass spectrometry  The tryptic digests were injected on the 25 cm Aurora Series with emitter column (CSI, 25 cm x 75¬µm ID, 1.6 ¬µm C18, IonOpticks), installed in the nano-electrospray source (CaptiveSpray source, Bruker Daltonics, Germany) at 50 ¬∞C using UltiMate 3000 (Thermo Scientific Dionex) coupled with TIMS quadrupole time of-flight instrument (timsTOF Pro2, Bruker Daltonics, Germany) and measured in diaPASEF mode (Meier et al., 2020). The mobile phases water/0.1% FA and ACN/0.1% FA (A and B respectively) were applied in linear gradients starting from 2% B and increasing to 17% in 87 min, followed by an increase to 25% B in  93 min, 37% B in 98 min, 80% B in 99 min to 104 min, the column was equilibrated in 2% B by next 15 min. For calibration of ion mobility dimension, three ions of Agilent ESI-Low Tuning Mix ions were selected (m/z [Th], 1/ùêæùêæ0 [Th]: 622.0289, 0.9848; 922.0097, 1.1895; 1221.9906, 1.3820). The diaPASEF windows scheme was ranging in dimension m/z from 396 to 1103 Th and in dimension 1/ùêæùêæ0 0.7 ‚Äì 1.3 vs cm - 2, with 59 x 12 Th windows). All measurements were done in low sample amount mode with ramp time 166 ms.

### Data Protocol
The raw data was processed using DIA-NN 1.8 (Demichev et al., 2020) with the ion mobility module for diaPASEF (Demichev et al., 2021). MS2 and MS1 mass accuracies were both set to 10 ppm, and scan window size to 10. DIA-NN was run in library-free mode with standard settings (fasta digest and deep learning-based spectra, RT, and IMs prediction) using the UniProt human reference proteome annotations  (UniProt Consortium, 2019) (downloaded on 2019.12.20.) and the match-between-runs (MBR) option.   Peptide normalized intensities were subjected to quality control with all 27 samples passing acceptance criteria. Peptides with excessive missing values (> 35 % per group) were excluded from the analysis. The missing values of the remaining peptides were imputed group-based using the PCA method (Josse and  Husson, 2016). Normalization was performed with LIMMA (Ritchie et al., 2015) and implementation of  cyclic loess method (Bolstad et al., 2003) with option ‚Äúfast‚Äù (Ballman et al., 2004). To obtain a quantitative protein data matrix, the log2-intensities of peptides were filtered, only peptides belonging to one protein group were kept and then summarized into protein log intensity by ‚ÄúmaxLFQ‚Äù method (Cox et al., 2014), implemented in R package iq (Pham et al., 2020). Sample protein distributions were median centered.  Statistical analysis of proteomics data was carried out using internally developed R scripts based on publicly available packages. Linear modeling was based on the R package LIMMA (Ritchie et al., 2015).  Following model was applied to each tissue data set (log(p) is log2 transformed expression of a protein):  log(p) ~ 0 + Class. The categorical factor Class had two levels: old, young; reference level: young. For finding regulated features following criteria were applied: Significance level alpha was set to guarantee the false discovery rate below ~ 5%. We found that alpha = 0.005 was delivering required level of  Benjamini‚ÄìHochberg FDR (Benjamini and Hochberg, 1995). The log fold-change criterion was applied to guarantee that the measured signal is above the average noise level. As such we have taken median residual standard deviation of linear model: log2 (T) = median residual SD of linear modeling (= log2(1.38)).  Functional analysis was carried out using R package clusterProfiler (Yu et al., 2012) for GSEA. For selecting the most (de)regulated GO terms we applied filter: 5 ‚â§ term size ‚â§ 350. Unless specified separately,  analyses were carried out with Benjamini‚ÄìHochberg FDR threshold 5%. To identify epithelial cell subtypes from our proteome data, we extracted cell type markers from the Single-cell atlas of the airway epithelium scRNAseq dataset (grch38 genes annotation) and associated the log2 fold-change proteome values  (Deprez et al., 2020).

### Publication Abstract
None

### Keywords
Human, Airway ephitelium

### Affiliations
Core  Facility -High-Throughput  Mass  Spectrometry,  Charit√© -Universit√§tsmedizin Berlin,  corporate  member  of  Freie  Universit√§t  Berlin,  Humboldt-Universit√§t  zu  Berlin and Berlin Institute of Health (BIH), Berlin, Germany
Department of Pediatric Respiratory Medicine, Immunology and Critical Care Medicine, Charite ‚Äê Universitatsmedizin, Germany; German Center for Lung Research (DZL), Associated Partner Site, Germany; Berlin Institute of Health (BIH) at Charit√© - Universit√§tsmedizin Berlin, Germa

### Submitter
Michael M√ºlleder

### Lab Head
Dr Marcus Mall
Department of Pediatric Respiratory Medicine, Immunology and Critical Care Medicine, Charite ‚Äê Universitatsmedizin, Germany; German Center for Lung Research (DZL), Associated Partner Site, Germany; Berlin Institute of Health (BIH) at Charit√© - Universit√§tsmedizin Berlin, Germa


