### Accession
PXD011713

### Title
Proteomic analysis of frontal cortex area 8 in frontotemporal lobar degeneration with c9orf72 mutation

### Description
To probe novel molecular disturbances in frontal cortex (area 8) from c9FTLD cases respect to neurologically intact controls, a Label-free MS-based approach was used.

### Sample Protocol
Sample preparation for proteomic analysis Frozen samples of frontal cortex were  homogenized in lysis buffer containing 7M urea, 2M thiourea, 4% (w/v) CHAPS, 50mM DTT. The homogenates were spinned down at 100,000×g for 1h at 15°C. Protein concentration was measured in the supernatants with the Bradford assay kit (Biorad, Hercules, CA, USA).  Label free LC-MS/MS The protein extract for each sample was diluted in Laemmli sample buffer and loaded into a 0.75 mm thick polyacrylamide gel with a 4% stacking gel casted over a 12.5% resolving gel. The run was stopped as soon as the front entered 3mm into the resolving gel so that the whole proteome became concentrated in the stacking/resolving gel interface. Bands were stained with Coomassie Brilliant Blue and excised from the gel. Protein enzymatic cleavage (10ug20ug) was carried out with trypsin (1:20, w/w, Promega, Madison, WI, USA) at 37°C for 16h. Purification and concentration of peptides was performed using C18 Zip Tip Solid Phase Extraction (Millipore, Burlington, MA, USA). Peptides mixtures were separated by reverse phase chromatography using an Eksigent nanoLC ultra 2D pump fitted with a 75μm ID column (Eksigent 0.075×250). Samples were first loaded for desalting and concentration into a 0.5cm length 100μm ID pre-column packed with the same chemistry as the separating column. Mobile phases were 100% water 0.1% formic acid (FA) (buffer A) and 100% acetonitrile 0.1% FA (buffer B). Column gradient was developed in a two-step gradient from 5% B to 25% B during 210min and 25%B to 40% B during 30min. Column was equilibrated in 95% B for 9min and 5% B for 14min. During all process, pre-column was in line with column, and the flow maintained all along the gradient at 300nl/min. Eluting peptides from the column were analyzed using an AB Sciex 5600 Triple-TOF system (AB Sciex, Nieuwerkerk, Holland). Information data was acquired upon a survey scan performed in a mass range from 350m/z up to 1,250m/z in a scan time of 250ms. Top 35 peaks were selected for fragmentation. Minimum accumulation time for MS/MS was set to 100 ms giving a total cycle time of 3.8 s. Product ions were scanned in a mass range from 230m/z up to 1,500m/z and excluded for further fragmentation during 15s.

### Data Protocol
MS/MS data acquisition was performed using Analyst 1.7.1 (AB Sciex, Nieuwerkerk, Holland) and spectra files were processed through Protein Pilot Software (v.5.0-ABSciex) using Paragon™ algorithm (v.4.0.0.0) for database search, Progroup™ for data grouping, and searched against the concatenated target-decoy UniProt proteome reference Human database (Proteome ID: UP000005640, 70902 proteins, December 2015). False discovery rate was performed using a non-lineal fitting method (Tang et al., 2008) and displayed results were those reporting a 1% Global false discovery rate or better. The peptide quantification was performed using the Progenesis LC−MS software (version 2.0.5556.29015, Nonlinear Dynamics, Newcastle, UK). Using the accurate mass measurements from full survey scans in the TOF detector and the observed retention times, runs were aligned to compensate for between-run variations in our nanoLC separation system. To this end, all runs were aligned to a reference run automatically chosen by the software, and a master list of features considering m/z values and retention times was generated. The quality of these alignments was manually supervised with the help of quality scores provided by the software. The peptide identifications were exported from Protein Pilot software and imported in Progenesis LC-MS software where they were matched to the respective features. Output data files were managed using R scripts  Perseus Software (Tyanova S. Nature Methods 2016) for subsequent statistical analyses and representation. Proteins identified by site (identification based only on a modification), reverse proteins (identified by decoy database) and potential contaminants were filtered out. Proteins quantified with at least two unique peptides, an ANOVA  a P-value lower than 0.05, and an absolute fold change of <0.77 (down-regulation) or >1.3 (up-regulation) in linear scale were considered to be significantly differentially expressed.

### Publication Abstract
None

### Keywords
Proteomics, Mass spectrometry, Dementia, C9orf72

### Affiliations
Proteomics unit
Proteomics Unit Navarrabiomed-UPNA-IdiSNA Proteored-ISCIII

### Submitter
Enrique SantamarÃ­a

### Lab Head
Dr Enrique Santamaria
Proteomics Unit Navarrabiomed-UPNA-IdiSNA Proteored-ISCIII


