### Accession
PXD001497

### Title
Proteome Analysis of Actinoplanes sp. SE50/110

### Description
The acarviose metabolite acarbose is an a glucosidase inhibitor produced by Actinoplanes sp. SE50/110. It is medically important because it is used in the treatment of type 2 diabetes. In this work a comprehensive proteome analysis of Actinoplanes sp. SE50/110 was carried out. The associated txt and RAW files were used for two different analyses and publications. While one study focused on a comparative analysis of Actinoplanes sp. SE50/110 to elucidate differences in the proteome cultures that were grown with either maltose or glucose, the other study applied spectral counting and analyzed only the maltose-grown cultures to determine the major proteins and their location in the cell. The txt files for the comparative data are labeled as "heavy_light" and of the spectral counting data as "light". Both datasets were derived from the same RAW files.

### Sample Protocol
Cultivation of Actinoplanes sp. SE50/110 in Maltose Minimal Medium Actinoplanes sp. SE50/110 (ATCC 31044; CBS 674.73) was cultivated following a modified protocol used by Wendler et al.[1,2]. This protocol included three stages with two shake flask pre cultures and a bioreactor main culture. Both pre-cultures were grown in baffled polycarbonate Erlenmeyer flasks (Corning, Tewksbury, MA, USA) with Silicosen C 40 plugs (Hirschmann Laborgeräte, Eberstadt, Germany) at 140 rpm and 28 °C in a GFL shaking incubator 3032 (GFL, Burgwedel, Germany). For the first pre culture glucose complex medium33 was inoculated with 1.75 ml glycerol cryo cultures in 250 ml flasks with 50 ml medium, grown for 48 h, harvested by centrifugation (1900 rcf, 2 min, 4 °C) and washed twice with 150 mM NaCl solution. Subsequently, cells were resuspended in 22.5 ml NaCl solution of which 1.5 ml was used for inoculation of the second pre culture (250 ml flasks, 50 ml medium). In the second pre culture cells were adapted to the minimal main culture medium. The medium was composed of solutions S1 S6 described in Table 1. After 48 h of cultivation cells were harvested by centrifugation (1900 rcf, 2 min, 4 °C), washed twice with 150 mM NaCl solution and resuspended in 26 ml NaCl solution (150 mM). Of the resuspended cells 25 ml were used for the inoculation of the main culture that was carried out in 1 l Biostat Qplus bioreactors (Sartorius AG, Göttingen, Germany) filled with 775 ml minimal medium (Table 1). For the bioreactor minimal medium the different solutions were combined after being prepared and sterilized separately: S1 and S3 were prepared and sterile filtered separately; S2 was autoclaved in the bioreactor; S4, CaCO3 solution and antifoam were autoclaved separately. The setpoints for the cultivation parameters were pH 6.5 and 30 °C. The pH was automatically controlled through the addition of 2 M NaOH and 10 % (w/v) H3PO4. The regulation of the dissolved oxygen level at 50 % was ensured through a cascade. If needed the gas flow with air was increased (minimum 0.075 to 0.75 NL min-1) followed by an increase of the stirrer speed (minimum 600 to 1200 rpm). Preparation of Proteomic Samples For all analyzed fractions proteomic samples were taken after 46 h of cultivation. For this, 90 ml of the culture were harvested by centrifugation (1900 rcf, 2 min, 4 °C) and washed twice with 150 mM NaCl. During sample treatment equal protein amounts of 14N unlabelled samples and 15N labelled pools were mixed to allow relative quantification of proteins. The present study only relies on information obtained from the 14N-unlabelled mass trace of the proteome samples. Preparation of the different subcellular proteome fractions was carried out as described by Otto et al. [3]. For the cytosolic and extracellular fraction an additional preceding step was the phenol extraction of proteins from a part of harvested cells or supernatant as described by Wendler et al. [2]. In the workflow proteins of the cytosolic, the enriched membrane and the extracellular fraction were fractionated using 1D SDS gels and were in gel digested with trypsin. In the membrane shaving protocol, membranes were spun down by ultracentrifugation, soluble loops were digested by Proteinase K in urea (shaving), and remaining transmembrane domains were digested with chymotrypsin in a buffer containing the MS compatible detergent RapiGest (Waters Corporation, Milford, MA, USA). LC MS/MS measurement Sample preparation and LC-MS/MS measurements were carried out according to Otto et al. [3]. Therefore peptides of all four fractions were subjected to reversed phase C18 column chromatography operated on a Proxeon EASYnLC (Thermo Fisher Scientific, Waltham, MA, USA). MS and MS/MS data were acquired with a LTQ-Orbitrap mass spectrometer (Thermo Fisher Scientific, Waltham, MA, USA) online coupled to the LC system.

### Data Protocol
Relative Quantification of Proteins by Spectral Counting Normalized spectral abundance factors (NSAF) were derived to estimate the relative abundance of each protein within different fractions according to Zybailov [4]. NSAF values (SpC/(L•?SpC)) are the number of MS/MS spectra (SpC) assigned to a protein divided by protein length (L) and the total number of spectra assigned to proteins (?SpC) within one sample. NSAFs are indicated as mean values of three replicates. Bioinformatics Tools used for Analysis of Proteins The LocateP v.2.0 pipeline47 was used to predict subcellular locations of proteins. Subcellular locations were assigned as follows: (1) intracellular {intracellular}; (2) integral membrane {multitransmembrane, multitransmembrane (lipid-modified N termini)}; (3) membrane-associated {lipid anchored, N terminally anchored (no cleavage site), N terminally anchored (with cleavage site), C terminally anchored (with cleavage site), intracellular / TMH start after 60}; (4) secreted {secretory (released) (with cleavage site)}. In the manual analysis of signal peptides cleavage sites were determined with SignalP66. The hydrophobic H domains were determined with the ExPASy ProtScale Tool according to Kyte & Doolittle67. Twin arginine motifs (RRxFLk) were predicted with TatP68. Transmembrane helices were determined by the TMHMM69 algorithm.

### Publication Abstract
None

### Keywords
Actinoplanes sp. se50/110

### Affiliations
Andreas Otto, Institute for Microbiology, Ernst-Moritz-Arndt-University of Greifswald, F.-L. Jahnstrasse 15, 17489 Greifswald, Germany
Institute for Microbiology

### Submitter
Andreas Otto

### Lab Head
Dr Andreas Otto
Andreas Otto, Institute for Microbiology, Ernst-Moritz-Arndt-University of Greifswald, F.-L. Jahnstrasse 15, 17489 Greifswald, Germany


