### Accession
PXD009651

### Title
HiRIEF TMT Proteomics of Arabidosis thaliana, wildtype versus oligopeptidases' KO mutants

### Description
Oligopeptidases Prep  and OOP are enzymes located in mitochondria and chloroplasts responsible for the cleavage of long peptides (8 to 65 aas) down to short peptides (3 to 7 aas), which in turn are degraded to free amino acids by aminopeptidases such as M1 and M17. In this study, Arabidopsis thaliana knockout mutants of the genes for these enzymes were studied by proteomics using a TMT (tandem mass tags) isobaric tags quantification strategy combined with HiRIEF LC-MS on a Q-Exactive Mass Spectrometer.

### Sample Protocol
Samples of the young leaves of Arabidopsis thaliana were collected from: i) wildtype (Col-0); ii) Prep1 and Prep2 double knockout mutant; iii) Prep1, Prep2 and OOP triple knockout mutant 1; iv) Prep1, Prep2 and OOP triple knockout mutant 2; and v) M1 and M17 aminopeptidases double knockout mutant. The leaves were ground in HEPES SDS lysis buffer, and the extracts were heated and sonicated. After centrifugation at 14000 g, the supernatant was collected and processed through the Filter Aided Sample Prep (FASP) protocol (Wisniewski et al, Nature Methods, 2009). The resulting peptides were then labeled with TMT using biological duplicates of each sample type, thus: 126 - WT; 127N - WT; 127C - Prep double KO; 128N - Prep double KO; 128C - triple KO 1; 129N - triple KO 1; 129C - triple KO 2; 130N - triple KO 2; 130C - M1-M17 double KO; 131 - M1-M17 double KO. Samples were then pooled into one TMT10plex set, cleaned by strong cation exchange-solid phase extraction and separated by High Resolution Isoelectric Focusing (HiRIEF, Branca et al, Nature Methods, 2014) prior to data-dependent LC-MS/MS analysis on a Q-Exactive Mass Spectrometer.

### Data Protocol
All MS/MS spectra were searched by MSGF+/Percolator under the Galaxy platform (https://usegalaxy.org) using a target-decoy strategy. The reference database used was Araport11 (40782 protein entries, 2016-06). We used a precursor mass tolerance of 10 ppm and high resolution setting on MS2 level. Only peptides with fully tryptic termini were allowed. We considered carbamidomethylation on cysteine and TMT-10plex on lysine and N-terminus as fixed modifications; and oxidation of methionine as variable modification. Quantitation of TMT-10plex reporter ions was done using an integration window tolerance of 10 ppm.

### Publication Abstract
The stepwise degradation of peptides to amino acids in plant mitochondria and chloroplasts is catalyzed by a network of oligopeptidases (presequence protease PreP, organellar oligopeptidase OOP) and aminopeptidases. In the present report, we show that the lack of oligopeptidase activity in Arabidopsis thaliana results in the accumulation of endogenous free peptides, mostly of chloroplastic origin (targeting peptides and degradation products). Using mRNA sequencing and deep coverage proteomics, allowing for the identification of 17&#xa0;000 transcripts and 11&#xa0;000 proteins, respectively, we uncover a peptide-stress response occurring in plants lacking PreP and OOP oligopeptidase activity. The peptide-stress response results in the activation of the classical plant defense pathways in the absence of pathogenic challenge. The constitutive activation of the pathogen-defense pathways imposes a strong growth penalty and a reduction of the plants reproductive fitness. Our results indicate that the absence of organellar oligopeptidases PreP1/2 and OOP results in the accumulation of peptides that are perceived as pathogenic effectors and activate the signaling pathways of plant-defense response.

### Keywords
Arabidopsis thaliana, Hirief, Oligopeptidase, Oop, Proteomics, Lcms, Tmt, Prep

### Affiliations
Dept. Oncology-Pathology, Science for Life Laboratory, Karolinska Institutet, Sweden
Clinical Proteomics Unit, Dep. of Oncology-Pathology

### Submitter
Rui Branca

### Lab Head
Dr Janne Lehti√∂
Dept. Oncology-Pathology, Science for Life Laboratory, Karolinska Institutet, Sweden


