### Accession
PXD026468

### Title
Large scale toxicoepigenetics on histones: a mass spectrometry-based screening assay applied to developmental toxicity

### Description
Toxicoepigenetics is an emerging field that studies the toxicological impact of compounds on protein expression through heritable, non-genetic mechanisms, such as histone post-translational modifications (hPTMs). Due to substantial progress in the large-scale study of hPTMs, integration into the field of toxicology is a promising addition that offers the opportunity to gain novel insights into toxicological phenomena. Moreover, there is a growing demand for high-throughput human-based in vitro assays for toxicity testing and especially for developmental toxicity. Consequently, we developed a mass spectrometry based proof-of-concept to develop an assay to screen the histone code and hence detecting multiple hPTMs changes simultaneously in human embryonic stem cells. To prove the applicability and performance, we first validated the untargeted workflow with valproic acid (VPA), a histone deacetylase inhibitor. These results demonstrate that our workflow is capable of mapping the hPTM-dynamics, with a general increase in acetylations as an internal control. To illustrate the scalability, the workflow was applied on a proof-of-concept dose-response library of a total of ten compounds with either i) a known effect on the hPTMs (BIX-01294, 3-Deazaneplanocin A, Trichostatin A, and VPA), ii) a presumed developmental toxicity, including compounds of abuse (Caffeine, Ethanol, Nicotine, Methotrexate, and All-trans retinoic acid), or iii) no proven embryotoxicity (Penicillin G). In conclusion, we show that toxicoepigenetic screening on histones is feasible and yields very rich data that holds great potential, not only for applications in the pharmaceutical industry, but also for environmental toxicity and food safety.

### Sample Protocol
Cell culture and harvest Oct4-eGFP Knock-In hESCs were cultured in Essential 8 (E8) medium on a precoated Vitronectin XF™  plate in 5% O2, 5% CO2 and 37°C. Cells were routinely passaged with 0.5 mM Ethylenediaminetetraacetic acid (EDTA) in Dulbecco's phosphate-buffered saline (DPBS) according to the manufacturer’s protocol of culturing hESCs in E8 medium. After every passage the cells were replated in E8 medium; on day 4, the medium containing the test compound was added. After an incubation of 24h, the cells were harvested. Briefly, cells were washed in PBS and incubated for 5 min at 37°C in 2.5mL Trypsin-EDTA (0.05%). Subsequently, 2.5mL trypsin-inhibitor was added and the cells were dissociated by gentle pipetting.  Flow cytometry Before the analysis, cells were resuspended in PBS + 1% bovine serum albumin (BSA) solution. Propidium iodide (PI) (Sigma-Aldrich) was added to measure cell viability. Flow count beads (Analis, Suarlée, Belgium) were added to acquire absolute cell counts. The samples were analyzed using Beckman Coulter Cytomics FC500 and CXP analysis software.  RT-qPCR 500.000 cells were immediately resuspended in TRIzol and stored at −80°C. For RNA isolation, chloroform was added to the thawed samples, with subsequent phase separation and purification using an RNeasy Mini kit. After DNase treatment and a washing step, RNA was eluted. RNA quality was assessed using the RNA 6000 Nano Kit and quantified using a RiboGreen assay. Complementary DNA (cDNA) was synthesized using The High-Capacity RNA-to-cDNA™ Kit. RT-qPCR was performed using the LightCycler 480 (Roche, Basel, Switzerland). For each reaction 1 μl of cDNA (2 ng/μl) was mixed with 10 μl of the LightCycler® 480 SYBR Green I Master (Roche) in a 384-well plate.  Histone extraction, propionylation and digestion The cell pellet was resuspended in 0.4 N hydrogen chloride (HCl) and incubated for 4h on a rotator at 4 °C. The histones were precipitated with 33% trichloroacetic acid (TCA) on ice for 30 min. The amount of extract which corresponds to 400.000 cells was used for histone quantification by gel-electrophoresis on a 18% TGX gel. The remaining purified histones were dissolved in 20 µL 1M triethylammonium bicarbonate (TEAB) buffer, pH 8.5. Next, 20 μL of propionylation reagent (propionic anhydride: 2-propanol 1:80 (v/v)) was added, for an incubation of 30 min at RT. This was followed by adding 20 µl milliQ water for 30 min at 37°C. Histones were then digested overnight at 37°C using trypsin (at an enzyme/histone ratio of 1:20 (m/m)) in 500 mM TEAB, supplemented with calcium chloride (CaCl2) and acetonitrile (ACN) to a final concentration of 1.0 mM and 5% respectively. Subsequently, the derivatization reaction was carried out again to cap peptide N-termini. Aspecific overpropionylation at serine (S), threonine (T) and tyrosine (Y) was reversed by adding hydroxylamine.  Liquid chromatography and mass spectrometry analysis The propionylated peptides were resuspended in 0.1% formic acid (FA) to ensure that a 5μl injection resulted in 2 μg of histones and 50 fmol of Beta-Galactosidase (ß-gal) internal standard on-column. Peptides were trapped on a Triart C18 column (5 x 0.5 mm, YMC) and separation was performed using a Triart C18 column (150 x 0.3 mm, YMC) on a NanoLC 425 system operating in capillary flow mode (5 μl/min). The mobile phase consisted of 0,1% FA in water supplemented with 3% dimethyl sulfoxide (DMSO) (Buffer A) and 0,1% FA in ACN (Buffer B). A low pH reversed phase (RP) 60 min gradient going from 3%–45% Buffer B was used, with a total run time of 75 min per sample. Data-dependent acquisition (DDA) was executed on a TripleTOF 5600 (AB Sciex) operating in positive mode, acquiring full scan MS1  (m/z 400–1250) and MS2 spectra (m/z 65–2000, high sensitivity mode) with a scan time of 250 and 200 ms respectively. For the MS2 spectra, a rolling collision energy with a spread of 15 V was applied and a maximum of 10 precursors (charge state +2 to +5) exceeding 300 cps were isolated for fragmentation followed by an exclusion for 10 s.

### Data Protocol
For every compound, raw data from all runs were imported in a single experiment and all runs were aligned against bovine histone samples in Progenesis QIP 3.0 (Nonlinear Dynamics, Waters). Next, feature detection was performed on the samples excluding the bovine histone samples to eliminate features that are only present in the bovine histones and not in the hESCs samples. The twenty MS/MS spectra closest to the elution apex were selected and merged into a single ∗.mgf file. On this file, two types of searches in Mascot (Matrix Science) were performed: 1) a standard search for the identification of non-propionylated standards (ß-gal), and 2) an error tolerant search without biological modifications against a complete Human Swissprot database (downloaded from Uniprot and supplemented with contaminants from the cRAP database (https: //www.thegpm.org/crap/)) to identify the proteins present in the sample. Based on the error tolerant search, a FASTA-database was generated, and the best PTM-set was determined for further analysis (i.e. based on the highest ranked PTMs in the error tolerant searches for each compound, together with the biologically most interesting PTMs (acetylations and methylations), a fixed PTM set was determined for all 10 compounds). Next, a second ∗.mgf file containing the three MS/MS spectra closest to the elution apex per feature was exported to perform a Mascot-search with the following parameters: 1) mass error tolerances for the precursor ions and its fragment ions were set at 10 ppm and 50 ppm respectively; 2) enzyme specificity was set to Arg-C, allowing for up to one missed cleavage site; 3) variable modifications included acetylation, butyrylation, crotonylation, trimethylation and formylation on K, methylation on R, dimethylation on both K and R, deamidation on asparagine (N), glutamine (Q) and R and oxidation of methionine (M); and 4) N-terminal propionylation and propionylation on K were set as fixed modifications. Database searching was performed against the above mentioned custom-made FASTA-database. The Mascot result files (∗.xml-format) were again imported into Progenesis QIP 3.0 for annotation. Features that were annotated as peptidoforms derived from histones were manually validated and curated by an expert to resolve isobaric near-coelution. Normalization of the samples (e.g. to correct for different sample loading) was performed against all histone peptides. Outlier detection and removal was based on normalization and this on two levels: 1) Before identification, when normalization is still done against all proteins, a normalization factor greater than 10 was used to filter out under-loaded samples (this was only the case for replicate 001C and 01B2 of DZNep), and 2) After identification, when normalization is done against all histone peptides, an estimated standard deviation (~STD) greater than 0,4 was used to filter out samples with too much internal variation. Progenesis QIP 3.0 uses ratiometric data in log space, along with a median and mean absolute deviation outlier filtering approach to calculate the estimated standard deviation (~STD) and normalization factor. Finally, the deconvoluted peptide ion data for every experiment (i.e. for each component separately) was exported from Progenesis QIP 3.0 for further analysis. Heatmaps were generated using Qlucore Omics Explorer for a predefined set of target peptides. For every compound, averages of the normalized abundances were calculated per concentration and the log fold changes were determined for every concentration towards the negative control. For VPA, relative abundances (RAb) were calculated as previously described (De Clerck et al. 2019b) by dividing the area under the curve (AUC) for each peptidoform containing the considered hPTM by the sum of the AUCs for all observed forms of that peptide.

### Publication Abstract
Evidence of the involvement of epigenetics in pathologies such as cancer, diabetes, and neurodegeneration has increased global interest in epigenetic modifications. For nearly thirty years, it has been known that cancer cells exhibit abnormal DNA methylation patterns. In contrast, the large-scale analysis of histone post-translational modifications (hPTMs) has lagged behind because classically, histone modification analysis has relied on site specific antibody-based techniques. Mass spectrometry (MS) is a technique that holds the promise to picture the histone code comprehensively in a single experiment. Therefore, we developed an MS-based method that is capable of tracking all possible hPTMs in an untargeted approach. In this way, trends in single and combinatorial hPTMs can be reported and enable prediction of the epigenetic toxicity of compounds. Moreover, this method is based on the use of human cells to provide preliminary data, thereby omitting the need to sacrifice laboratory animals. Improving the workflow and the user-friendliness in order to become a high throughput, easily applicable, toxicological screening assay is an ongoing effort. Still, this novel toxicoepigenetic assay and the data it generates holds great potential for, among others, pharmaceutical industry, food science, clinical diagnostics and, environmental toxicity screening. &#x2022;There is a growing interest in epigenetic modifications, and more specifically in histone post-translational modifications (hPTMs).&#x2022;We describe an MS-based workflow that is capable of tracking all possible hPTMs in an untargeted approach that makes use of human cells.&#x2022;Improving the workflow and the user-friendliness in order to become a high throughput, easily applicable, toxicological screening assay is an ongoing effort.

### Keywords
Histone post-translational modifications, Developmental toxicity, Toxicoepigenetics, Lc-ms/ms, Drug screening

### Affiliations
ProGenTomics, Laboratory of Pharmaceutical Biotechnology, Ghent University, B-9000 Ghent, Belgium
Faculity of Pharmaceutical Biotechnology

### Submitter
Maarten Dhaenens

### Lab Head
Dr Maarten Dhaenens
ProGenTomics, Laboratory of Pharmaceutical Biotechnology, Ghent University, B-9000 Ghent, Belgium


