### Accession
PXD034454

### Title
bladder cancer patient urine exosomes

### Description
Extracellular vesicles (EVs), particularly nano-sized small EV exosomes, are emerging biomarker sources. However, due to heterogeneous populations secreted from diverse cell types, mapping exosome multi-omic molecular information specifically to their pathogenesis origin for cancer biomarker identification is still extraordinarily challenging. Herein, we introduced a novel 3D-structured nanographene immunomagnetic particles (NanoPoms) with unique flower pom-poms morphology and photo-click chemistry for specific marker-defined capture and release of intact exosome. This specific exosome isolation approach leads to the expanded identification of targetable cancer biomarkers with enhanced specificity and sensitivity, as demonstrated by multi-omic exosome analysis of bladder cancer patient tissue fluids using the next generation sequencing of somatic DNA mutations, miRNAs, and the global proteome. The NanoPoms prepared exosomes also exhibit distinctive in vivo biodistribution patterns, highlighting the highly viable and integral quality. The developed method is simple and straightforward, which is applicable to nearly all types of biological fluids and amenable for enrichment, scale up, and high-throughput exosome isolation.

### Sample Protocol
Urinary EV pellets resultant from ~2 mL of urine from both bladder cancer patients and healthy individuals were reconstituted in 400 µL of M-PER Mammalian Protein Extraction Buffer supplemented with 1× Halt Protease Inhibitors and sonicated in an ultrasonic water bath for 15 min. NanoPom exosome samples were processed by in-gel digestion for LC-MS analysis. Approximately 20 µg of protein from each sample was resolved on a 4-20% gradient gel (Bio-Rad), then visualized by SimplyBlue SafeStain (ThermoFisher). Each lane was cut into 8 slices of approximately the same size, then reduced, alkylated, and digested with 400 ng of trypsin overnight at 37°C. Digestion was quenched with 1% formic acid (FA) in 50 mM ammonium bicarbonate buffer/50% acetonitrile (ACN). Peptides were dried using a speed-vac and stored at -20°C. For LC-MS analysis, peptides were reconstituted in 3% ACN/0.1% FA. Peptides were injected onto an Acclaim PepMap 100 C18 trap column (75 µm x 2 cm, ThermoFisher) using an Agilent 1260 Infinity capillary pump and auto sampler (Agilent Technologies). The autosampler was maintained at 4°C, the capillary pump flow rate was set to 1.5 µL/min, and an isocratic solvent system consisting of 3% ACN/0.1% FA. After 10 minutes, the trap column was valve was switched to be in-line with an Acclaim PepMap RSLC C18 analytical column (50 µm x 25 cm, ThermoFisher), using an Agilent 1290 Infinity II column compartment, kept at 42°C. Peptides were resolved on the analytical column using an Agilent 1290 Infinity II UHPLC with nanoflow passive split flow adapter, maintaining 200 µL/min flow pre-split, resulting in ~300 nL/min flow on the analytical column at the beginning of the run. A two solvent system consisting of (A) water/0.1% FA and (B) ACN/0.1% FA was used, with a gradient as follows: 3% B at 0 min, ramping to 8% B at 8 minutes, ramping to 26% B at 90 min, ramping to 35% B at 105 min, ramping to 40% B at 120 min, then ramping to 70% B at 122, held at 70% B until 127 min, before returning to 3% at 130 min and holding until the end of the run at 150 min, with a post run equilibration of 12 min. Eluted peptides were analyzed by an Agilent 6550 QToF mass spectrometer equipped with a G1992A nanoESI source (Agilent Technologies). The source parameters were as follows: drying gas temperature was set to 200°C, flow of 11 L/min, a capillary voltage of 1200 V, and fragmentor voltage of 360 V was used. Data was acquired in positive ion mode using data dependent acquisition, with an MS scan range of 290 to 1700 m/z at 8 spectra/s, MS/MS scan range of 50-1700 m/z at 3 spectra/s, and an isolation width set to narrow (~1.3 m/z). Maximum precursors per cycle was set to 10, with dynamic exclusion enabled after 2 spectra, and released time set to 0.5 min. Peptides were fragmented by collision induced dissociation (CID) using N2 gas and a variable collision energy depending on the precursor charge and m/z. Reference mass correction in real time was enabled, with lock masses of 299 and 1221 m/z used.

### Data Protocol
Data acquired for each sample was converted to Mascot Generic Format (.MGF) using the Agilent Data Reprocessor (Agilent Technologies). Database searching of .MGF files was done using Mascot Daemon v2.2.2 (Matrix Science). Data was searched against a concatenated decoy FASTA file containing Homo sapiens proteins downloaded from Uniprot. Search results from all 3 engines was combined and analyzed using Scaffold4 v4.8.1 (Proteome Software Inc.). Thresholds of 1% FDR protein, 1% FDR peptide, and 2 peptides minimum were set for protein identification.

### Publication Abstract
Extracellular vesicles (EVs), particularly nano-sized small EV exosomes, are emerging biomarker sources. However, due to heterogeneous populations secreted from diverse cell types, mapping exosome multi-omic molecular information specifically to their pathogenesis origin for cancer biomarker identification is still extraordinarily challenging. Herein, we introduced a novel 3D-structured nanographene immunomagnetic particles (NanoPoms) with unique flower pom-poms morphology and photo-click chemistry for specific marker-defined capture and release of intact exosome. This specific exosome isolation approach leads to the expanded identification of targetable cancer biomarkers with enhanced specificity and sensitivity, as demonstrated by multi-omic exosome analysis of bladder cancer patient tissue fluids using the next generation sequencing of somatic DNA mutations, miRNAs, and the global proteome (Data are available via ProteomeXchange with identifier PXD034454). The NanoPoms prepared exosomes also exhibit distinctive in vivo biodistribution patterns, highlighting the highly viable and integral quality. The developed method is simple and straightforward, which is applicable to nearly all types of biological fluids and amenable for enrichment, scale up, and high-throughput exosome isolation.

### Keywords
Urine, Human, Exosomes, Bladder cancer

### Affiliations
University of Florida
University of FLorida

### Submitter
MEI HE

### Lab Head
Dr Mei He
University of FLorida


