### Accession
PXD011160

### Title
Grazing-resistance, antibiotic production and internal carbon cycling are characteristics for bacteria that prevail in long-term enclosure experiments

### Description
Enclosure experiments are frequently used to investigate the impact of changing environmental conditions on microbial assemblages. Yet, the question how individual members of bacterial communities respond to challenges posed by the incubation itself remained unanswered. We used metaproteomic profiling, 16S rRNA gene analysis and high nucleic acid content analysis to monitor bacterial communities during long-term incubations (55 days) under marine (M1), mesohaline (M2) and oligohaline (M3) conditions with and without the addition of terrestrial dissolved organic matter. Our results showed that early in the experiment (after one week, T2), bacterial communities were highly diverse and their composition differed significantly between marine, mesohaline and oligohaline conditions. Controls (BS) and tDOM-treated samples (FKB) showed notable differences at this stage. In contrast, in the late phase of the experiment (after 55 days, T6), bacterial communities in both, manipulated and untreated marine and mesohaline enclosures were quite similar to each other and were dominated by gammaproteobacterial Spongiibacter. In the oligohaline enclosure, the actinobacterial hgc-I clade was very abundant in this phase. Our findings suggest that individual capacities, e.g. grazing-resistance, antibiotics production, and the ability to access alternative carbon sources may enable Spongiibacter and hgc-I clade members to successfully prevail during long-term incubations. Bacterial community composition in enclosure experiments thus seems to be strongly influenced by the individual inherent bacterial strategies to cope with the incubation as such. Researchers intending to investigate the effects of manipulation on complex microbial communities may therefore want to use short incubation periods or sophisticated systems that avoid these unspecific effects of long-term experiments.

### Sample Protocol
Protein extraction from filtered samples was performed as described in detail by Teeling et al. (Teeling et al., 2012a), with minor modifications: One half of each frozen filter was used for the analyses. Filter halves were cut into small pieces and transferred to 1.8 mL tubes containing lysis buffer (50 mM Tris, 2% SDS, 10% Glycerol, 0.1 M DTT, pH 6.8) and 180 µm glass beads. Cells on the filter pieces were disrupted by bead beating in a bead mill (3 x 30 s, 6.5 m/s, 5 min pause at 4°C between runs). After pelleting of beads and filter pieces by centrifugation (4 min, 15,500 x g, 4°C), supernatants (i.e., protein raw extracts) were collected and subjected to ultracentrifugation (1 h, 100,000 g, 4°C). Supernatants (i.e., enriched soluble proteins devoid of cell membranes and debris) were precipitated in ice-cold acetone at -20°C for 16 h. Proteins were pelleted (10 min, 15,500 g, 4°C), washed in 96% ethanol and dried in a vacuum centrifuge. For protein separation by sodium dodecyl sulfate (SDS) polyacrylamide gel electrophoresis (PAGE), protein pellets were gently resuspended in sample buffer (0.1 M Tris-HCl pH 6.8, 10% SDS, 20% glycerol, 5% mercaptoethanol, bromophenol blue), denatured (10 min, 90°C) and loaded onto precast mini gels (BioRad TGX, 4-20%). After electrophoresis (50 min, 150 V) and staining with Coomassie Brilliant Blue for 16 h (G250, Sigma Aldrich), protein lanes were excised from the gels and divided into 10 equal-sized pieces (subsamples) per sample. After destaining of the gel pieces (200 mM NH4HCO3, 30% acetonitrile), proteins were in-gel digested at 37°C for 16 h using trypsin solution (Sequencing grade modified trypsin, Promega, 1 µg/mL)(Eymann et al., 2004), and peptides were purified using ZipTip pipette tips (Millipore) according to the manufacturer’s instructions. Electrospray ionization tandem mass spectrometry (ESI-MS/MS) measurements and data filtering and analysis were performed as recently described in detail by Ponnudurai et al. (2017). Briefly, peptide mixes were separated in a nano-high-performance liquid chromatography (nano-HPLC) system (Easy-nLCII HPLC, Thermo Fisher Scientific) followed by MS/MS analysis in an LTQ Orbitrap Velos mass spectrometer (Thermo Fisher Scientific).

### Data Protocol
All MS/MS spectra were searched against the respective sampling site-specific target-decoy metaproteome database, i.e., samples taken at the marine site (M1) were searched against the M1 database, mesohaline samples were searched against the mesohaline enclosure (M2) database, etc., using the SORCERERTM system (SageN, Milpitas, CA, USA) with the integrated SEQUEST algorithm (Thermo Fisher Scientific, San Jose, CA, USA; version 27, revision 11). Spectra from biological replicates (n = 2) were combined and searched together to increase the identification rate of the individual samples. The resulting protein identifications were filtered and analyzed with Scaffold (version 4.0.6.1, Proteome Software Inc., Portland, OR).

### Publication Abstract
Enclosure experiments are frequently used to investigate the impact of changing environmental conditions on microbial assemblages. Yet, how the incubation itself challenges complex bacterial communities is thus far unknown. In this study, metaproteomic profiling, 16S rRNA gene analyses, and cell counts were combined to evaluate bacterial communities derived from marine, mesohaline, and oligohaline conditions after long-term batch incubations. Early in the experiment, the three bacterial communities were highly diverse and differed significantly in their compositions. Manipulation of the enclosures with terrigenous dissolved organic carbon resulted in notable differences compared to the control enclosures at this early phase of the experiment. However, after 55&#x2009;days, bacterial communities in the manipulated and the control enclosures under marine and mesohaline conditions were all dominated by gammaproteobacterium <i>Spongiibacter</i> In the oligohaline enclosures, actinobacterial cluster I of the hgc group (hgc-I) remained abundant in the late phase of the incubation. Metaproteome analyses suggested that the ability to use outer membrane-based internal energy stores, in addition to the previously described grazing resistance, may enable the gammaproteobacterium <i>Spongiibacter</i> to prevail in long-time incubations. Under oligohaline conditions, the utilization of external recalcitrant carbon appeared to be more important (hgc-I). Enclosure experiments with complex natural microbial communities are important tools to investigate the effects of manipulations. However, species-specific properties, such as individual carbon storage strategies, can cause manipulation-independent effects and need to be considered when interpreting results from enclosures.<b>IMPORTANCE</b> In microbial ecology, enclosure studies are often used to investigate the effect of single environmental factors on complex bacterial communities. However, in addition to the manipulation, unintended effects ("bottle effect") may occur due to the enclosure itself. In this study, we analyzed the bacterial communities that originated from three different salinities of the Baltic Sea, comparing their compositions and physiological activities both at the early stage and after 55&#x2009;days of incubation. Our results suggested that internal carbon storage strategies impact the success of certain bacterial species, independent of the experimental manipulation. Thus, while enclosure experiments remain valid tools in environmental research, microbial community composition shifts must be critically followed. This investigation of the metaproteome during long-term batch enclosures expanded our current understanding of the so-called "bottle effect," which is well known to occur during enclosure experiments.

### Keywords
Lc-msms, Metaproteome, Enclosure experiments, Baltic sea

### Affiliations
Universität Greifswald Institut für Pharmazie, AG Pharmazeutische Biotechnologie Greifswald, Germany
University of Greifswald

### Submitter
Frank Unfried

### Lab Head
Dr Stephanie Markert
Universität Greifswald Institut für Pharmazie, AG Pharmazeutische Biotechnologie Greifswald, Germany


