### Accession
PXD001020

### Title
Redox-state of pentraxin 3 as a novel biomarker for  resolution of inflammation and survival in sepsis

### Description
In an endotoxaemic mouse model of sepsis, a tissue-based proteomics approach for biomarker discovery identified long pentraxin 3 (PTX3) as the lead candidate for inflamed myocardium. PTX3 accumulated as an octamer due to disulphide-bond formation in heart, kidney and lung ?common organ dysfunctions seen in patients with sepsis. Oligomeric moieties of PTX3 were also detectable in the circulation. The redox-state of PTX3 was quantified over the first 11 days in critically ill adult patients with sepsis. On admission day, there was no difference in the redox-state of PTX3 between survivors and non-survivors. From day 2 onwards, the conversion of octameric to monomeric PTX3 was consistently associated with a greater survival after 28 days of follow-up. For example, by day 2 post admission, octameric PTX3 was undetectable in survivors, but still constituted more than half of total PTX3 in non-survivors (P<0.001). Monomeric PTX3 was inversely associated with cardiac damage markers NT-proBNP, high sensitivity troponin I and T. In comparison to the conventional measurements of total PTX3 or NT-proBNP, the redox-sensitive oligomerization of PTX3 was more dynamic and a superior predictor of disease outcome.

### Sample Protocol
The Triton-insoluble fractions from control (n=4) and septic group (n=4) were denatured in Laemmli buffer at 97ç™ˆ for 5min. The proteins were separated by 5-20% gradient Tris-glycine gel at 5W for 45min then 30W for 4.5 hours. After electrophoresis, gels were stained using the PlusOne Silver staining Kit (GE Healthcare). Silver staining was used for band staining to avoid cross-contamination with fainter gel bands. All gel bands (n=48 per sample) were excised in identical parallel positions across lanes and no empty gel pieces were left behind. Subsequently, all gel bands were subjected to in-gel tryptic digestion using an Investigator ProGest (Digilab) robotic digestion system. Tryptic peptides were separated on a nanoflow LC system (Dionex Ultimate 3000, Thermo Fisher Scientific) using reversed phase columns (PepMap C18, 25cm x 75um) and eluted with a 40min gradient (10-25% B in 35 min, 25-40% B in 5min, 90% B in 10min, and 2% B in 20min, where A is 2% ACN, 0.1% formic acid in HPLC H2O and B is 90% ACN, 0.1% formic acid in HPLC H2O). Sequentially eluted peptides were directly analysed by tandem mass spectrometry (LTQ Orbitrap XL, Thermo Fisher Scientific) using full ion scan mode over the mass-to-charge (m/z) range 450-1600. Tandem MS (MS/MS) was performed on the top 6 ions using data-dependent mode with dynamic exclusion. Peak lists were generated by extract_msn_com.exe (version 5.0, Thermo Fisher Scientific) and searched against UniProt/SwissProt mouse database (version 2014_01, 16656 protein entries) using Mascot (version 2.3.01, Matrix Science). The mass tolerance was set at 30ppm for the precursor ions and at 0.8 Da for fragment ions. Carboxyamidomethylation of cysteine was used as a fixed modification and oxidation of methionine as a variable modification. Two missed cleavages were allowed.

### Data Protocol
Scaffold (version 4.3.0, Proteome Software Inc., Portland, OR) was used to calculate the normalized spectral counts and to validate peptide and protein identifications. According to the default values in the Scaffold software, peptide identifications were accepted if they could be established at greater than 95% probability as specified by the Peptide Prophet algorithm. Only tryptic peptides were included in the analysis. Protein identifications were accepted if they could be established at greater than 99% probability with at least two independent peptides. Proteomic differential expression was assessed using the normalized spectral abundance factor - power law global error model (NSAF-PLGEM). The Normalized Spectral Abundance Factor was calculated for each protein detected. Spectral count values of 0 were replaced by an empirically derived fractional value. The value was calculated to be the smallest value between 0 and 1, which provided the best fit to a normal distribution, as dertermined by a Shapiro-Wilks test. The NASF values were fit to a power-law global error model; differentially expressed proteins were then identified through a signal-to-noise (STN) test statistic. Proteins, which did not have a sufficient number of samples with spectra (less than 2 samples in each group) were removed prior to analysis.

### Publication Abstract
In an endotoxaemic mouse model of sepsis, a tissue-based proteomics approach for biomarker discovery identified long pentraxin 3 (PTX3) as the lead candidate for inflamed myocardium. When the redox-sensitive oligomerization state of PTX3 was further investigated, PTX3 accumulated as an octamer as a result of disulfide-bond formation in heart, kidney, and lung-common organ dysfunctions seen in patients with sepsis. Oligomeric moieties of PTX3 were also detectable in circulation. The oligomerization state of PTX3 was quantified over the first 11 days in critically ill adult patients with sepsis. On admission day, there was no difference in the oligomerization state of PTX3 between survivors and non-survivors. From day 2 onward, the conversion of octameric to monomeric PTX3 was consistently associated with a greater survival after 28 days of follow-up. For example, by day 2 post-admission, octameric PTX3 was barely detectable in survivors, but it still constituted more than half of the total PTX3 in non-survivors (p &lt; 0.001). Monomeric PTX3 was inversely associated with cardiac damage markers NT-proBNP and high-sensitivity troponin I and T. Relative to the conventional measurements of total PTX3 or NT-proBNP, the oligomerization of PTX3 was a superior predictor of disease outcome.

### Keywords
Biomarkers; host-pathogen interaction; inflammatory response; prognostic markers; protein modification; serum/plasma

### Affiliations
King's British Heart Foundation Centre, King's College London, UK
Cardiovascular Division, King's College London

### Submitter
Xiaoke Yin

### Lab Head
Dr Manuel Mayr
King's British Heart Foundation Centre, King's College London, UK


