### Accession
PXD009117

### Title
C2 skeletal myotube phosphoproteome

### Description
Skeletal muscle is known to adapt dynamically to changes in workload by regulatory processes of the phosphatidylinositide 3-kinase (PI3K)/Akt pathway. We performed a global quantitative phosphoproteomics analysis of contracting mouse C2 myotubes treated with insulin growth factor 1 (IGF-1) or LY294002 to activate or inhibit PI3K/Akt signaling, respectively.

### Sample Protocol
For stable isotope labeling by amino acids in cell culture (SILAC) (Ong et al., 2002), C2 myoblasts were cultured in high glucose SILAC DMEM medium (PAA, GE Healthcare Life Sciences, Freiburg, Germany) supplemented with 15% dialyzed FCS (PAA), 1% non-essential amino acids, 1% penicillin/streptomycin, 1% sodium pyruvate, 1% proline (all Life Technologies), 84 mg/l arginine and 146 mg/l lysine (Cambridge Isotope Laboratories Inc., Tewksbury, USA) for at least nine cell doublings. Light, medium and heavy labeling of cells was performed with 13C6 L-arginine/12C6 L-lysine, 12C6 L-arginine/D4 L-lysine and 13C615N4 L-arginine/13C615N2 L-lysine, respectively. Stable isotope-labeled myoblasts were seeded into 6-well plates (Corning Incorporated, New York, USA) and grown to 90% confluency. Differentiation was induced by reduction of the dialyzed FCS content to 2% (Ong et al., 2002) in the absence of sodium pyruvate. Differentiation medium was changed every 48 h until day 4. Subsequently, cells were serum-starved for 16 h overnight and sarcomere formation was improved by electrical pulse stimulation (EPS, 0.05 Hz, 4 ms, 10 V) for 4 h using a C-Pace EP Culture Pacer (IonOptix, Milton, USA). One hour prior to cell lysis, cells were either treated with 10 ng/ml IGF-1, 10 M LY294002 or DMSO. SILAC experiments were generally performed in three independent biological replicates with a label switch. Phosphopeptides were enriched using TiO2 spherical beads as described before (Reimann et al, 2017; Wiese et al, 2014).  Reversed-phase liquid chromatography-mass spectrometry was performed using the UltiMateTM 3000 RSLCnano system (Dionex LC Packings/Thermo Fisher Scientific, Dreieich, Germany) coupled online to a Velos Orbitrap Elite (Thermo Fisher Scientific, Bremen, Germany) instrument. The UHPLC system was equipped with two C18 -precolumns (Ø 0.3 mm x 5 mm; PepMapTM, Thermo Fisher Scientific) and an Acclaim® PepMapTM analytical column (ID: 75 m x 500 mm, 2 m, 100 Å, Dionex LC Packings/Thermo Fisher Scientific). The MS instrument was externally calibrated using standard compounds and equipped with a nanoelectrospray ion source and a stainless steel emitter (Thermo Fischer Scientific). MS/MS analyses were generally performed on multiply charged peptide ions applying normalized collision energy (NCE) of 35% with an activation q of 0.25 and an activation time of 30 ms unless otherwise stated. For global quantitative phosphoproteomics, peptide mixtures from SCX fractions of three independent biological replicates were analyzed by LC-MS/MS applying a 2 h LC gradient and CID for peptide fragmentation. SCX fractions enriched for phosphopeptides by TiO2 were analyzed in two technical replicates applying the same 2 h LC gradient and multistage activation (MSA) with neutral loss (NL) masses of 32.7, 49 and 98 Da for technical replicate #1 and higher-energy collisional dissociation (HCD) with a NCE of 28% for technical replicate #2.

### Data Protocol
For quantitative phosphoproteomics data, Andromeda integrated in MaxQuant 1.5.1.0 (Cox et al., 2011) was used to search peak lists against the UniProt ProteomeSet mouse database (release 12.2016, 58,430 protein entries). The precursor mass tolerance was set to 20 ppm for the first search and to 4.5 ppm for the main search. For MSA/CID data, the fragment mass tolerance was set to 0.5 Da. Trypsin was set as proteolytic enzyme allowing up to two missed cleavages. Oxidation of methionine and phosphorylation of serine, threonine and tyrosine were set as variable modifications and cysteine carbamidomethylation as fixed modification. A false discovery rate (FDR) of 1% was applied on both peptide (on modified peptides separately) and protein lists. Numbers of unique phosphopeptides were counted based on the MaxQuant peptide ID in the Phospho(STY) sites table. Phosphosites scored with a MaxQuant localization probability of ≥ 0.75 were deemed “localized”, while sites with a localization probability of < 0.75 were counted as putative sites given that the amino acid (aa) sequence in combination with the number of phosphate groups was not identified with localized sites elsewhere in the dataset.

### Publication Abstract
The PI3K/Akt pathway promotes skeletal muscle growth and myogenic differentiation. Although its importance in skeletal muscle biology is well documented, many of its substrates remain to be identified. We here studied PI3K/Akt signaling in contracting skeletal muscle cells by quantitative phosphoproteomics. We identified the extended basophilic phosphosite motif RxRxxp[S/T]xxp[S/T] in various proteins including filamin-C (FLNc). Importantly, this extended motif, located in a unique insert in Ig-like domain 20 of FLNc, is doubly phosphorylated. The protein kinases responsible for this dual-site phosphorylation are Akt and PKC&#x3b1;. Proximity proteomics and interaction analysis identified filamin A-interacting protein 1 (FILIP1) as direct FLNc binding partner. FILIP1 binding induces filamin degradation, thereby negatively regulating its function. Here, dual-site phosphorylation of FLNc not only reduces FILIP1 binding, providing a mechanism to shield FLNc from FILIP1-mediated degradation, but also enables fast dynamics of FLNc necessary for its function as signaling adaptor in cross-striated muscle cells.

### Keywords
Pi3k/akt, Phosphoproteomics, C2 myotubes

### Affiliations
AG Warscheid
Biologie II
Albert-Ludwigs-Universität Freiburg
Schänzlestr. 1
79104 Freiburg
Germany

Department of Biochemistry and Functional Proteomics, Institute of Biology II, Faculty of Biology and BIOSS Centre for Biological Signalling Studies, University of Freiburg, 79104 Freiburg im Breisgau, Germany

### Submitter
Friedel Drepper

### Lab Head
Dr Bettina Warscheid
Department of Biochemistry and Functional Proteomics, Institute of Biology II, Faculty of Biology and BIOSS Centre for Biological Signalling Studies, University of Freiburg, 79104 Freiburg im Breisgau, Germany


