### Accession
PXD044157

### Title
ProteoMixture: A Cell Deconvolution Tool for Bulk Tissue Proteomics Data

### Description
Numerous multi-omic investigations of cancer tissue have documented varying and poor pairwise transcript:protein quantitative correlations and most deconvolution tools aiming to predict cell type proportions (cell admixture) have been developed and credentialed using transcript-level data alone. To estimate cell admixture using protein abundance data, we analyzed proteome (and transcriptome data) generated from contrived admixtures of tumor, stroma, and immune cell models or those selectively harvested from the tissue microenvironment by laser microdissection from high grade serous ovarian cancer (HGSOC) tumors. Co-quantified transcripts and proteins performed similarly to estimate stroma and immune cell admixture in two commonly used deconvolution algorithms ESTIMATE and ConsensusTME (r ≥ 0.63). Here we have developed and optimized protein-based signatures to estimate cell admixture proportions and benchmarked these using bulk tumor proteomics data from over 150 HGSOC patients. The optimized protein signatures supporting cell type proportion estimates from bulk tissue proteomic data are available at https://lmdomics.org/ProteoMixture/.

### Sample Protocol
Human cell lines OVCAR-3 and Jurkat T human cell lines and human ovarian serous cancer associated fibroblasts were grown according to manufacturer’s instructions. Fresh-frozen (FF) tumor specimens (stage  IIC) were obtained from primary (adnexal mass) or metastasis (omentum) from five patients with a primary HGSOC disease site originating at the ovary or fallopian tube. Regions of tissue annotated using the HALO image analysis software (Indica Labs) were exported and collected using laser microdissection (LMD7, Leica Microsystems). Cell and tissue samples underwent pressure-assisted trypsin digestion employing a barocycler (2320EXT, Pressure BioSciences) and a heat-stable form of trypsin (SMART Trypsin, Thermo Fisher Scientific). Pooled admixtures (n=13) were generated by combining peptide digests (10 μg total) from OVCAR-3, tumor associated fibroblasts, and Jurkat T cells at pre-defined compositional ratios. Similarly, pooled admixtures (n=17; 10 μg total) were generated by combining peptide digests from LMD enriched populations of tumor, stroma, and lymphocytes. Cell and tissue admixture samples were labeled with tandem mass tag (TMT) reagents according to the manufacturer’s instructions (TMTpro, Thermo Fisher Scientific) and multiplexes were reversed-phase fractionated on a 1260 Infinity II offline liquid chromatography system (Agilent Technologies) into 96 fractions followed by concatenation into 36 pooled fractions. Liquid chromatography-tandem mass spectrometry (LC-MS/MS) analyses of TMTpro16 and TMTpro18 multiplexes were performed on a nanoflow high-performance LC system (EASY-nLC 1200, Thermo Fisher Scientific, Inc.) coupled online with an Orbitrap mass spectrometer (Q Exactive HF-X, Thermo Fisher Scientific, Inc.). Samples were loaded on a reversed-phase trap column (AcclaimTM PepMapTM 100 Å, C-18, 20 mm length, nanoViper Trap column, Thermo Fisher Scientific, Inc.) and eluted on a heated (50 °C) reversed-phase analytical column (AcclaimTM PepMapTM RSLC C-18, 2 μm, 100 Å, 75 μm × 500 mm, nanoViper, Thermo Fisher Scientific, Inc.) by developing a linear gradient from 2% mobile phase A (2% acetonitrile, 0.1% formic acid) to 32% mobile phase B (95% acetonitrile, 0.1% formic acid) over 120 min at a constant flow rate of 250 nL/min. Full scan mass spectra (MS) were acquired using a mass range of m/z 400-1600, followed by selection of the top 12 most intense molecular ions in each MS scan for high-energy collisional dissociation (HCD).

### Data Protocol
Global protein-level abundances were generated from peptide spectral matches identified by searching .raw data files against a publicly available, non-redundant human proteome database (http://www.uniprot.org/, SwissProt, Homo sapiens, downloaded 12-01-2017 for cell line and 10-29-2021 for tissue) using Mascot (Matrix Science, v2.6.0), Proteome Discoverer (v2.2.0.388, Thermo Fisher Scientific, Inc.), and in-house tools. Quan correction was applied to all reagent ion abundances using TMTpro16 reagent lot UL296296 or TMTpro18 reagent lots WK334339 and WJ338613. Global proteome data was visualized by principal component analyses (PCA) using the top 100 most variable proteins by mean absolute deviation (MAD) using ggplot2 (version 3.4.1) in R (version 4.2.2). Gene names in the proteomic data were harmonized with any gene synonyms used in the ConsensusTME and ESTIMATE tools prior to their utilization. Pearson correlation analysis was performed between ESTIMATE or ConsensusTME and percent cell type (quartile cell percentages). Spearman correlation was performed on proteomic data from tissue admixture models and heatmap of quartile percentage tissue models were generated using ComplexHeatmap (version 2.14.0). Proteomic data from tissue admixture models were assessed by Decomprolute. Differential analysis was performed using limma to identify proteins uniquely elevated within cell populations of interest enriched from tissue using LMD. Protein features of interest passed an adjusted p value < 0.05 comparing tumor with stroma/immune collections, immune with tumor/stroma collections and stroma protein features passed a p value < 0.05 and as well as a foldchange (FC) cutoff ± 1.5. Assignment of admixed samples with prognostic HGSOC molecular subtypes was performed using consensusOV (ver 1.18.0). Correlation plots were generated with seaborn version 0.11.2. Mann-WhitneyWilcoxon Test was used to assess the statistical significance between groups. Protein signatures were validated using proteomic abundance data from enriched and bulk HGSOC (n = 9) and bulk HGSOC tissues (n = 169) by performing ssGSEA.

### Publication Abstract
None

### Keywords
High grade serous ovarian cancer, Laser microdissection, Proteomics, Cellular admixture, Tumor signature, Stroma signature, Immune signature

### Affiliations
Inova Health System
Women’s Service Line Ionva Health System

### Submitter
Thomas Conrads

### Lab Head
Dr Thomas P. Conrads
Women’s Service Line Ionva Health System


