### Accession
PXD026370

### Title
Proteome profiling of brain tissue from multiple system atrophy patients and from controls

### Description
Human brain tissue was collected post mortem from patients diagnosed with multiple system atrophy (n=45) and from controls (n=30) in order to perform a comparative quantitative proteome profiling of tissue from the prefrontal cortex (Broadman area 9).

### Sample Protocol
For each sample, approx 100 ug brain tissue was mixed with 500 uL Tissue Extraction Reagent II (Invitrogen) supplied with a protease inhibitor cocktail (Sigma-Aldrich) in MagNA Lyser Green Beads tubes before two incubations (6k RPM, 25 sec) on a MagNA Lyser Instrument (Roche) with cooling on ice between runs. Protein concentration were estimated using a Bradford assay. Aliquots containing 100 ug protein were adjusted to 66 uL volume and precipitated by following the first steps of the protein precipitation protocol from a Bio-Rad 2-D clean-up kit. Precipitated proteins were resolubilized in 50 uL 8 M urea supplemented with 5 mM TCEP and 10 mM CAA (final concentrations) for reduction and alkylation, respectively. Proteins were digested in a two-step procedure by addition of 1 ug endo-Lys C (Wako Pure Chemical Corp.) and digested for 3 hrs before dilution with 50 mM ammonium bicarbonate to 1 M urea (final concentration) and continued digestion over night by addition of 2 ug sequencing grade trypsin (Promega). The digestion was quenched by addition of 10% TFA (1% TFA final concentration). Generated peptides were desalted using SepPak C18 columns (Waters) and acetonitrile were removed using a speedvac before A280 measurement using a NanoDrop spectrophotometer for peptide concentration estimates. For each sample, 1 ug peptide were loaded onto a 50 cm C18 column (1.9 um beads, Dr. Maisch GmbH) using an Easy nLC 1200 system. Loaded peptides were eluted using a 180 min gradient into a Q-Exactive HF-X mass spectrometer (ThermoFisher Scientific). MS data were acquired recording MS1 spectra (300-1500 Da) with 120,000 resolution at 400 m/z and MS2 spectra with 30,000 resolution in a data-dependent mode fragmenting the 12 most abundant ions (Charge state 2 or higher) by higher collision dissociation at 28 % normalized collision energy. MS2 spectra were recorded using dynamic exclusion (30s) to minimize repeated fragmentation of the same peptides.

### Data Protocol
The recorded raw files were analyzed using MaxQuant v 1.6.0.17 for peptide quantification by MS1 intensity and for protein identification using the build-in Andromeda search engine searching against a modified UniProt database downloaded February 2018, where predicted signal peptides had been removed from the protein sequences. The MaxQuant analysis were performed with the following settings: Enzyme: Trypsin, maximum 2 missed cleavage sites. Precursor mass tolerance: 4.5 ppm; fragment mass tolerance: 20 ppm. Variable modifications: Oxidation (M), acetyl (protein N-terminal). Fixed modifications: Carbamidomethyl (C). Peptide FDR: 1%, Protein FDR: 1 %. Match between run: On, 0.7 min alignment window. All other settings were left at the default settings.

### Publication Abstract
None

### Keywords
Human, Multiple system atrophy, Proteome, Prefrontal cortex, Q-exactive hf-x, Mass spectrometry, Brain tissue

### Affiliations
Novo Nordisk Foundation Center for Protein Research, Faculty of Health and Medical Sciences
University of Copenhagen, Blegdamsvej 3B,  DK-2200 Copenhagen N, Denmark

University of Copenhagen, Faculty of Health and Medical Sciences, Novo Nordisk Foundation Center for Protein Research, Blegdamsvej 3B, DK-2200 Copenhagen N, Denmark

### Submitter
Ole Ã˜stergaard

### Lab Head
Dr Jesper Velgaard Olsen
University of Copenhagen, Faculty of Health and Medical Sciences, Novo Nordisk Foundation Center for Protein Research, Blegdamsvej 3B, DK-2200 Copenhagen N, Denmark


