### Accession
PXD016772

### Title
Npl3 stabilizes R-loops at telomeres to prevent accelerated replicative senescence

### Description
Telomere shortening rates must be regulated to prevent premature replicative senescence. TERRA R-loops become stabilized at critically short telomeres to promote their elongation through homology-directed repair (HDR), thereby counteracting senescence onset. Using a non-bias proteomic approach to identify telomere binding factors, we identified Npl3, an RNA-binding protein previously implicated in multiple RNA biogenesis processes. Using Chromatin- and RNA immunoprecipitation, we demonstrate that Npl3 interacts with TERRA and telomeres. Furthermore, we show that Npl3 associates to telomeres in an R-loop dependent manner, as changes in R-loop levels, e.g. at short telomeres, modulate the recruitment of Npl3 to chromosome ends. Through a series of genetic and biochemical approaches we demonstrate that Npl3 binds to TERRA and stabilizes R-loops at short telomeres, which in turn promotes HDR and prevents premature replicative senescence onset. This may have implications for diseases associated with excessive telomere shortening.

### Sample Protocol
Eluted proteins from telomere pull-down were separated on a 4-12% NuPAGE Bis –Tris precasted PAGE gel (Thermo). Eluates were run at 180V for 10 min and processed by in-gel digestion (Shevchenko et al 2006). Briefly, samples were reduced in reduction buffer (10 mM DTT in 50 mM ammonium bicarbonate (ABC) buffer) for 1h at 56° C and alkylated in alkylation buffer (50 mM iodoacetamide (IAA) in 50 mM ABC buffer) for 45 min in the dark. Proteins were digested with 2 µg Protease LysC (Wako Chemicals) overnight at 37° C in 50 mM ABC buffer. Digested peptides were desalted on a C18 StageTip as described (Rappsilber et al. 2007),  and analyzed by nanoflow liquid chromatography on an EASY-nLC 1000 system (Thermo) coupled to a Q Exactive Plus mass spectrometer (Thermo). The peptides were separarated on a self-packed reverse phase capillary (75 µm diameter, 25 cm length packed with C18 beads of 1.9 µm (Dr Maisch GmbH). The capillary was clamped on an electrospray ion source (Nanospray Flex™, Thermo). A 90 min gradient starting from 2%-60% gradient acetonitrile in 0.1% formic acid was used at a flow of 225 nl/min. Data was collected in data-dependent acquisition mode with one MS full scan followed by up to 10 MS/MS scan with HCD fragmentation.

### Data Protocol
MS raw files were processed using the MaxQuant software (version 1.5.2.8) and the ENSEMBL S.cerevisiae protein database (Saccharomyces_cerevisiae.R64-1-1.24). LFQ quantitation and match between run options were activated. MaxQuant output files were analyzed using an in-house R script. Briefly, known contaminants, reverse hits and protein groups only identified by site were removed. Identified protein groups (minimum 2 peptides, 1 of them unique) were further filtered to a minimum of 2 quantification events per experiment. Missing values were imputed using a downshifted and compressed beta distribution within the 0.001 and 0.015 percentile of the measured values for each replicate individually. The LFQ intensities were log2 transformed and a two sample Welch t-test was performed. Volcano plots were generated by plotting -log10(p-values) and fold changes. The threshold line for enriched proteins is defined empirically with p-value = 0.05, s0=1 and c=0.5. Gene ontology analysis for telomere associated candidates was performed with thePantherDB.org overrepresentation Test (Release 20190701) with the annotation database released on 20190202. Fisher’s exact test followed by Bonferroni correction was applied. Heatmaps for enriched proteins were generated using the “pheatmap” package in R with clustering the complete data based on the Euclidean distance. Biogrid protein interaction were clustered using the complete data based on binary distance.

### Publication Abstract
None

### Keywords
Lc-ms/ms; telomere; pulldown

### Affiliations
Institute of Molecular Biology (IMB), Ackermannweg 4, 55128 Mainz
Quantitative Proteomics
Institute of Molecular Biology (IMB)

### Submitter
F Butter

### Lab Head
Dr Falk Butter
Institute of Molecular Biology (IMB), Ackermannweg 4, 55128 Mainz


