### Accession
PXD017056

### Title
VESICULATION OF RED BLOOD CELLS IN THE BLOOD BANK: A MULTI-OMICS APPROACH TOWARDS IDENTIFICATION OF CAUSES AND CONSEQUENCES

### Description
Microvesicle generation is an integral part of the aging process of red blood cells in vivo and in vitro. Extensive vesiculation impairs function and survival of red blood cells after transfusion, and microvesicles contribute to transfusion reactions. The triggers and mechanism of microvesicle generation are largely unknown. In this study, we combined morphological, immunochemical, proteomic, lipidomic and metabolomic analyses in order to obtain an integrated understanding of the mechanisms underlying microvesicle generation during storage of red blood cell concentrates. Our data indicate that changes in membrane organization, triggered by altered protein conformation, constitute the main mechanism of vesiculation, and precede changes in lipid organization. The resulting selective accumulation of membrane components in microvesicles is accompanied by the recruitment of plasma proteins involved in inflammation and coagulation. Our data may serve as a basis for further dissection of the fundamental mechanisms of red blood cell aging and vesiculation, for identifying the cause-effect relationship between blood bank storage and transfusion complications, and for assessing the role of microvesicles in pathologies affecting red blood cells.

### Sample Protocol
Microvesicles (MVs) were obtained from five standard RBC concentrates from the regional blood bank Sanquin Bank South East Region, Nijmegen, The Netherlands, following the guidelines of the local medical ethical committee and in accordance with the declaration of Helsinki. At each storage time (one, three and five weeks), the MVs were isolated from 10 ml samples as described before. All analyses were performed on at least three biological replicates. MVs were analyzed by flow cytometry using mixtures of FITC-labeled CD235a (1:100, clone 11E4B-7-6, Beckman Coulter, Fullerton, CA, USA) and Annexin V-PE (1:25, BD Pharmingen, Hoeven, the Netherlands), and with rabbit anti-human band 3 (1:1000, clone K2N6B/PM) combined with secondary antibody goat anti-rabbit Alexa 633 (IgG, A 21070, Rockford, USA). CD59 content was determined using PE-labeled CD59 (clone MEM43, 1:400, IQ products, Groningen, the Netherlands) in combination with FITC-labeled CD235a. Sulfate latex microspheres (0.9 ��m; Invitrogen, Carlsbad CA, USA) and washed Flow-Count calibration beads (Beckman Coulter, Brea CA, USA) were used for quantification. Cryo-transmission electron microscopy (Cryo-TEM) was performed using a JEM 1400 Flash electron microscope (120KV, Jeol, Tokyo, Japan; Nieuw-Vennep, The Netherlands) as described before. In order to obtain more detailed strucural information, some samples were imaged using bright field phase contrast transmission electron microscopy at 200kV on a Talos Arctica microscope with Falcon II direct electron detector and Volta phase plate with a total dose of about 100e-/��2 in linear mode. Volta phase plate imaging was performed in focus. Samples were plunge-frozen into liquid ethane cooled by liquid nitrogen to just above its freezing point, with some solid ethane kept in equilibrium with the liquid. The plunging was performed on a Thermo Fisher Vitrobot Mark IV using 2.5s blot time, 4oC, 100% relative humidity, and the blot force adjusted to have the blot pads just touch without blot paper. Quantifoil R1.2/1.3 holey carbon grids were used, and 2.5 ���l of the sample was applied just before blotting. For proteomic analysis, MV samples were processed using a filter-aided sample preparation (FASP) procedure as described previously [28], and enzyme digests were acidified to a final concentration of 0.1% TFA and purified by STAGE tips [29]. LC-MS/MS experiments were 4 carried out by the LC-MS/MS platform composed of an Ultimate 3000 UPLC (Thermo Fisher, Germany) connected to the Orbitrap Velos Pro mass spectrometer (Thermo Fisher, Germany). Peptides were loaded on a 2 cm Acclaim PepMap100 Nano-Trap Column (Thermo Fisher, Germany) and separated by a 25 cm Acclaim PepMap100 Nano LC column (Thermo Fisher, Germany) packed with 3 ��m C18 beads. The flow-rate was set at 300nl/min in a 120 min gradient of 95% buffer A/5% buffer B to 65% buffer A /35 % buffer B (buffer A: 0.5% acetic acid. Buffer B: 0.5% acetic acid in 100% acetonitrile). Peptides eluting from the column were electrosprayed into the mass spectrometer at 2.3 kV spray voltage. MS data acquisition was set in a data-dependent mode, automatically switching between MS and MS2 where full-scan spectra of intact peptides (m/z 350-1500) were acquired with an automated gain control accumulation value of 1,000,000 ions. The ten most abundant ions were sequentially isolated and fragmented in the C trap, where dissociation was induced by HCD mode, using an accumulation target value of 10,000, a normalized collision energy of 45% and a capillary temperature of 275��C. Dynamic exclusion of ions sequenced within the 45 previous seconds was applied. Unassigned charge states and singly charged ions were excluded from sequencing. For MS2 selection, a minimum of 10,000 counts was required.

### Data Protocol
The MaxQuant - Andromeda search engine integrated in the MaxQuant software (Version 1.3.0.5) was used for identifying proteins. Peak lists were generated for the top 12 most intense MS peaks in 100 Da windows by MaxQuant prior to the human UNIPROT database (release 2017/04) search. The protein database was supplemented with frequently observed contaminants from MaxQuant. Andromeda search parameters for protein identification were set to tolerance of 6 ppm for the parental peptide and 0.5 Da for fragmentation spectra and trypsin specificity allowing up to 2 miscleaved sites. Deamination of glutamine, oxidation of methionine, and protein N-terminal acetylation were set as variable modifications, carboxyamidomethylation of cysteines was specified as a fixed modification. Minimal required peptide length was specified at six amino acids. The ���match between run option��� for an elution time window of two minutes was enabled. Peptides and proteins detected by at least two peptides in one of the samples with a false discovery rate (FDR) of 1% were accepted. Excluded from validation were proteins identified by site only, external contaminants and reversed proteins. Proteins were quantified by normalized summed peptide intensities, computed in MaxQuant with the label free quantification (LFQ) option switched on.

### Publication Abstract
Microvesicle generation is an integral part of the aging process of red blood cells in vivo and in vitro. Extensive vesiculation impairs function and survival of red blood cells after transfusion, and microvesicles contribute to transfusion reactions. The triggers and mechanisms of microvesicle generation are largely unknown. In this study, we combined morphological, immunochemical, proteomic, lipidomic, and metabolomic analyses to obtain an integrated understanding of the mechanisms underlying microvesicle generation during the storage of red blood cell concentrates. Our data indicate that changes in membrane organization, triggered by altered protein conformation, constitute the main mechanism of vesiculation, and precede changes in lipid organization. The resulting selective accumulation of membrane components in microvesicles is accompanied by the recruitment of plasma proteins involved in inflammation and coagulation. Our data may serve as a basis for further dissection of the fundamental mechanisms of red blood cell aging and vesiculation, for identifying the cause-effect relationship between blood bank storage and transfusion complications, and for assessing the role of microvesicles in pathologies affecting red blood cells.

### Keywords
Metabolomics, Rbc, Proteomics, Lipidomics, Blood

### Affiliations
Department of Biochemistry (286) Radboud University Medical Centre Radboud Institute for Molecular Life Sciences  P.O. Box 9101 NL-6500 HB Nijmegen The Netherlands
Plymouth University Peninsula Schools of Medicine and Dentistry

### Submitter
Vikram Sharma

### Lab Head
Dr Giel Bosman
Department of Biochemistry (286) Radboud University Medical Centre Radboud Institute for Molecular Life Sciences  P.O. Box 9101 NL-6500 HB Nijmegen The Netherlands


