### Accession
PXD011074

### Title
The proteome of Medicago truncatula in response to ammonium and urea nutrition

### Description
comprehensive and quantitative proteomic study of the roots of the NH4+-tolerant legume Medicago truncatula grown with nitrate, NH4+ or urea as sole N source using the iTRAQ method.

### Sample Protocol
Total proteins of M. truncatula roots were extracted from 300 mg frozen tissues. Roots were ground in liquid nitrogen using mortar and pestle with the addition of a proteases inhibitor. Powdered  samples  were  homogenized  with  500µl  of  extraction  buffer  (7  M  urea,  2  M thiourea,  4%  (w/v)  CHAPS,  2%  (v/v)  TritonX-100  and  50  mM  DTT)  to  maximize  the extraction  of  water-soluble  and  membrane  proteins.  The  homogenate  was  centrifuged at 10000g and 4 ºC for 10 min. Consecutively, supernatants containing the root proteins (150µg) were precipitated with methanol/chloroform, and the obtained pellets were suspended in a mixture of 7  M  urea,  2  M  thiourea and  4% (v/v) CHAPS.  Protein  content  of each extract (three  biological  replicates  for  nitrate  and  ammonium  nutrition,  and  two  for  urea)  was quantified with the Bradford assay kit (Bio-Rad) prior to carry out the comparative proteomic analysis of proteins using the isobaric Tags for Relative and Absolute Quantitation (iTRAQ) approach  (Unwin  et  al.,  2010).  The  samples  were  iTRAQ-labelled  according  to  the manufacturer’s protocol (Sciex) as follows: 100 µg of protein from each sample were reduced with 50 mM tris (2-carboxyethyl) phosphine (TCEP) at 60 °C for 1 h. Then, cysteine residues were alkylated with 200 mM methylmethanethiosulfonate (MMTS) at room temperature for 15 min. Trypsin (Promega; 1:20, w/v) was used to cleavage the protein mix for 16 h at 37 ºC, and every tryptic digest was labelled with an isobaric amine-reactive tag by incubation during 1 h (iTRAQ 8-plex experiment). Finally, every set of labelled samples were independently pooled and evaporated in a vacuum centrifuge until a volume lower than 40 µl was reached.  Peptide fractionation by HPLC and mass spectrometry analysis In  order  to  increase  proteome  coverage,  peptide  fractionation  was  performed  before characterization of the proteins. The peptide pool was injected to an Ettan LC system with a X-Terra RP18 pre-column (2.1 x 20mm) and a high pH stable X-Terra RP18 column (C18; 2.1mm x 150mm; 3.5µm) (Waters) at a flow rate of 40 µl/min. Peptides were eluted with a mobile phase B of 5–65% linear gradient over 35 min (A, 5 mM ammonium bicarbonate in water  at  pH  9.8;  B,  5  mM  ammonium  bicarbonate  in  acetonitrile  at  pH  9.8).  Then, 11 fractions  were  collected,  evaporated  under  vacuum  and  reconstituted  into  20  μl  of  2% acetonitrile, 0.1% formic acid, 98% milli-Q H2O prior to mass spectrometric analysis.  Peptides mixtures contained in each fraction were separated by reverse-phase chromatography using an Eksigent nanoLC ultra 2D pump fitted with a 75 μm ID column (Eksigent 0.075 x 25cm). Initially, samples were loaded for desalting and concentration into a 0.5 cm length and 300  μm  ID pre-column  packed  with  the same chemistry as  the separating column. Mobile phases were 0.1% formic acid in 100% water as buffer A and 0.1% formic acid in 100% acetonitrile as buffer B. The column gradient was developed in a two-step gradient from 5% B to 25% B in 180 min and 25% B to 40% B in 30 min. Column was equilibrated in 95% B for 10 min and 5% B for 15 min. During the processes, the pre-column was in line with the column and flow was maintained all along the gradient at 300 nl/min. Eluting peptides from the column were analyzed using an Sciex 5600 TripleTOF™ system. Data were acquired upon a survey  scan performed in a mass range from 350 m/z up to 1250 m/z in a scan time of 250  ms.  Top  35 peaks were  selected  for  fragmentation.  Minimum accumulation  time  for MS/MS was set to 100 ms giving a total cycle time of 3.8 s. Product ions were scanned in a mass range from 100 m/z up to 1700 m/z and excluded for further fragmentation during 15 s. 160

### Data Protocol
Raw data (wiff, Sciex) were analyzed with MaxQuant software (Cox and Mann, 2008). For peak list  generation, default Sciex  Q-TOF  instrument parameters were used except the main search  peptide  tolerance  and  MS/MS  match  tolerance,  which  were  set  to  0.01  Da  and increased up to 50 ppm respectively. A contaminant database (.fasta) was used for filtering out contaminants. Peak lists were searched against JCVI database1 of Medicago truncatula protein  sequences,  and  Andromeda  was  used  as  a  search  engine  (Cox  et  al.,  2011). Methionine  oxidation  was  set  as  variable  modification,  and  the  carbamidomethylation of cysteine residues was set as fixed modification. Reporter ion intensities were bias corrected for the overlapping isotope contributions from the iTRAQ tags according to the certificate of analysis  provided  by the  reagent  manufacturer  (Sciex).  The  peptide  and  protein selection criteria for relative quantitation were performed as follows. Maximum false discovery rates (FDR) were set to 0.01 at protein and peptide levels. Analyses were limited to peptides of six or more amino  acids  in length,  and  considering  a maximum of two  missed  cleavages. The output  data  files  with  the  relative  protein  abundance  were  managed  using  R  scripts  for subsequent statistical analyses and representation. Proteins identified by site (identification based only on a modification), reverse proteins (identified by decoy database) and potential contaminants were filtered out. Only proteins with more than one identified peptide were used for quantification. For the possible rescue of quantification data, up to one missing value for each group was rescued replacing it by the mean of the rest in-group samples. Data were normalized  and  transformed  for  later  comparison  using  quantiles  normalization  and log2 transformation, respectively. The Limma Bioconductor software package in R was used for ANOVA analyses. Significant and differential data were selected by a p-value lower tan 0.01, fold changes of <0.77 (down-accumulation) and >1.3 (up-accumulation) in linear scale.

### Publication Abstract
None

### Keywords
Proteomics, Medicago

### Affiliations
Proteomics Unit, Navarrabiomed Research Center (Pamplona-Navarra-Spain) Proteored-ISCIII (Spain)
Proteomics unit

### Submitter
Enrique SantamarÃ­a

### Lab Head
Dr Enrique Santamaria
Proteomics Unit, Navarrabiomed Research Center (Pamplona-Navarra-Spain) Proteored-ISCIII (Spain)


