### Accession
PXD014575

### Title
Protein-level evidence of novel β-type hemocyanin and heterogeneous subunit usage in the Pacific whiteleg shrimp, Litopenaeus vannamei

### Description
The functional diversity of crustacean hemocyanins is broad, encompassing O2 delivery, innate immune response, metabolite storage, and osmolyte balance, all in a heterogeneous protein structure. As such, the sequence diversity of this class of proteins and its subunit composition are the focus of many studies on crustacean adaptation to environmental challenges. Recent transcriptomic and genomic sequencing on the Pacific whiteleg shrimp Litopenaeus vannamei has identified unique isoforms of hemocyanin including an ancestral β-type subunit thought to be lost in penaeid shrimp. However, it is unknown the degree to which these isoforms are translated as proteins, and whether they differ in function. The present study uses proteomic approaches to characterize the protein-level abundance and organization of these hemocyanin isoforms within their native oligomeric structures. Fractions of each hemocyanin oligomeric form were purified by size-exclusion high performance liquid chromatography for identification of subunit isoforms using tandem mass spectrometry at <1% protein false discovery rate. Relative abundances of hemocyanin oligomers and monomeric subunits from hemolymph and fractions were also quantified by polyacrylamide gel electrophoresis with and without denaturation for comparison of relative abundance.  Ten hemocyanin isoforms were identified by tandem mass spectrometry in both hemocyanin oligomer fractions including a single small subunit, eight large subunits, and the first protein-level evidence of a β-type subunit in penaeid shrimp.  Hemocyanin subunits were organized primarily as hexamers (95-99% relative abundance) as opposed to dodecamers. Hexamers utilized a significantly higher ratio (2.05:1) of small subunit to large subunit compared to dodecamers (1.04:1), and the relative abundances of the large subunit isoforms was dominated by HcL1 in both fractions. The ability to distinguish and quantify hemocyanin isoforms within oligomeric structures will aid future studies linking hemocyanin genes to function and physiology as well as offer insight into the evolutionary history of crustaceans.

### Sample Protocol
Preparation of Tryptic Peptides for LC-MS/MS Hemocyanin subunit isoforms and other proteins were identified by LC-MS/MS in dodecamer and hexamer fractions separated by SE-HPLC in five of the same eight shrimps. The three remaining shrimps were omitted due to discrepancies in TIC between fractions. Fractions collected by SE-HPLC underwent five total centricon washes and five buffer replacements using mass-spectrometry grade water. Following re-concentration of fractions, 5 µg of each sample were brought to 50 µL in100 mM ammonium bicarbonate buffer (pH 8.0). The 50 µL samples were then reduced by adding 100 mM dithiothreitol (DTT) to a final concentration of 5 mM DTT incubated at 60 °C for 30 min. After cooling, the reduced samples were alkylated by adding 500 mM iodoacetic acid (IAA) to a final concentration of 15 mM IAA and incubated at room temperature for 30 min in the dark. Trypsin Gold, MS grade (Promega) was added to each sample at an enzyme to protein ratio of 1:25 (w:w). All samples were brought to a total volume of 150 µL with 100 mM ammonium bicarbonate, pH >7.5, then incubated at 37 °C to facilitate trypsin activity. The trypsin digest was stopped after 18 h by bringing samples to 1 mL volume with 0.1% (v/v) formic acid (FA). Peptides were loaded onto solid phase extraction columns (30 mg Strata-X 33 µ polymeric reverse phase solid phase extraction columns, Phenomenex, Torrance, CA, USA) first conditioned with 1 mL 100% acetonitrile (ACN) followed by 1 mL 0.1% formic acid (v/v). The columns were then washed three times with 1 mL of 0.1% FA, and samples were eluted with 500 µL of 60% ACN (v/v), 0.1% FA into a Protein Lo-bind tube (Eppendorf) and dried by SpeedVac. Dodecamer fraction samples were re-suspended in 30 µL 0.1% FA, and hexamer fraction samples in 80 µL 0.1% FA. Protein concentrations were then estimated using NanoDrop detection (OD280). LC-MS/MS The performance of the LC-MS/MS was calibrated by evaluating precursor retention times and fragment ion areas of a standard tryptic digest of β-galactosidase prior to loading samples. Based on NanoDrop estimates of protein concentration, a dilution series of sample peptides was used to determine appropriate loading amounts for each fraction matching TIC between runs. As a result, dodecamer fraction samples were diluted to 0.1 µg µL−1 and hexamer fraction samples were diluted to 0.05 µg µL−1 using 0.1% FA. Samples were then centrifuged at maximum speed for 10 min and 10 µL of dodecamer fraction supernatant and 3 µL of hexamer fraction supernatant was injected onto a 100 µm × 2 cm C18 (100 Å with 5-µm particles) trap column (AcclaimTM PepMap 100; Thermo Fisher Scientific) with mobile phase A [2% ACN (v/v), 0.1% FA], and separated on a 75 µm × 15 cm C18 (300 Å with 3-µm particles) analytical column (AcclaimTM PepMap 100; Thermo Fisher Scientific). Reverse phase separation at 350 nL min−1 was performed with a gradient of 0-50% mobile phase B [95% ACN (v/v), 0.1% FA (v/v)] over 36 min on a 2D+ NanoLC system (Eksigent, Dublin, CA, USA). The LC was interfaced to a TripleTOF 5600 Mass Spectrometer (AB Sciex, Foster City, CA, USA) with a nanospray source. Source temperature was 120 °C, source voltage was 2500 V, declustering potential was 100 V, and collision energy was 10 eV. The instrument was run in positive ion instrument dependent acquisition mode with precursor ion scans accumulating for 250 ms with up to 20 candidate ions if precursors were 350-1250 m/z, exceeded 500 counts per second (cps), and had 2+ to 5+ charge. The mass tolerance was 50 mDa and isotopes within 4 Da were excluded. The product ion scans were performed with an accumulation time of 200 ms for TOF masses 100-1500 Da. Collision energy was set to rolling collision energy.

### Data Protocol
Data were visualized using PeakView (AB Sciex) software and files were converted from .wiff to .mgf format. Mascot (v2.6.2) searches were performed accounting for fixed Carbamidomethyl (C) modifications, variable modifications (oxidation (M), deamidation (NQ), carbamidomethyl (N-term), a decoy database, 2 maximum missed cleavages, a peptide tolerance of ± 20 ppm, MS/MS tolerance of 0.25 Da, and charges of +2, +3, and +4. Peptides were searched against databases including the representative L. vannamei genome assembly proteins (ASM378908v1; Zhang et al., 2019), a translated L. vannamei transcriptome assembly including 5 full-length hemocyanins and multiple contig variants (Johnson et al., 2015), and contaminant protein databases. Search results were then imported into Scaffold (4.8.8) software and scored by peptide and protein prophet with legacy grouping. All peptides and proteins were identified at <1% protein FDR with 2 minimum peptides at <1% local peptide FDR. Normalized spectral abundance factors (NSAF) were determined using functions embedded in Scaffold. Exponentially modified protein abundance index (emPAI) values were also calculated within Scaffold according to Ishihama et al. (2005). Relative abundances of proteins were calculated as the proportion NSAF or emPAI within each fraction sample and averaged between 5 shrimp.

### Publication Abstract
None

### Keywords
Hemocyanin, Crustacean, Proteomics, Subunit heterogeneity, Mass spectrometry

### Affiliations
University of Southern California
Biology Department, College of Charleston, SC, USA

### Submitter
Jason Wang

### Lab Head
Dr Karen Gray Burnett
Biology Department, College of Charleston, SC, USA


