### Accession
PXD033149

### Title
the effects of the nanobody on SW480 cells using label-free proteomics approach

### Description
determine the effect of cell-penetrable nanobody (R9VH36) on cell viability and proteomic profile in EGFR-positive human colorectal cancer cell lines.

### Sample Protocol
The cells were placed in a T25 flask (Nest, U.S.A.) (~107 cells/flask) in triplicates. The day after, the cells were then replaced with a fresh medium containing 10 nM of R9VH36 or gefitinib for 1 h.  The treated cell lysates were lysed in lysis buffer (10 mM HEPES/NaOH, pH 7.4, 0.25 M sucrose, 10 mM NaCl, 3 mM MgCl2, 0.5% Triton X-100) supplemented with a protease inhibitor cocktail. The supernatant was collected by centrifugation at 12000g. and subsequent to ice-cold acetone precipitation. After precipitation, all samples were reconstituted in sample buffer (6 M Urea, 2 M Thiourea, 0.05% SDS, 10mM NaCl). The protein solution was diluted in 10mM ammonium bicarbonate at 1:20 ratio (v/v). The total protein (25 µg) was subjected to gel-free based digestion. Next, sulfhydryl bond reduction was performed using 5 mM DTT (Sigma Aldrich Co.) in 10 mM ammonium bicarbonate at 25°C for 3 hour and sulfhydryl alkylation using IAA (Sigma Aldrich Co.) at room temperature for 30 minutes in the dark. All samples were enzymatically digested for 16 hours. The tryptic peptides were cleaned-up using C18 Zip-tip and reconstituted in 0.1% formic acid before being subjected to LC-MS/MS.

### Data Protocol
The tryptic peptides were analyzed using tandem mass spectrometers, Orbitrap HF hybrid mass spectrometer combined with an UltiMate 3000 LC system. The tryptic peptides were first desalted on the line of a reverse-phase C18 PepMap 100 trapping column, before being resolved onto a C18 PepMapTM 100 capillary column with a 135-min gradient of CH3CN, 0.1% formic acid, at a flow rate of 300 nL/min. Peptides were analyzed by applying a data-dependent top10 method consisting of a scan cycle initiated by a full scan of peptide ions, followed by high-energy collisional dissociation and MS/MS scans on the 10 most abundant precursor ions. Full scan mass spectra were acquired from m/z 400 to 1600 with an AGC target set at 3 × 106 ions and a resolution of 70,000. MS/MS scan was initiated when the ACG target reached 105 ions. Ion selection was performed applying a dynamic exclusion window of 15 seconds.   Raw files were analyzed by the Proteome Discoverer software version. 2.4 (Thermo Scientific) using the SEQUEST, Percolator, and Minora algorithms. LC-MS spectrum was matched against the UniProtKB reviewed database (11/05/2021). For protein identification and quantification, the setting parameters were as follows: a maximum of two trypsin missed cleavages were allowed with a precursor mass tolerance of 20 ppm and fragment mass tolerance of 0.01 Da. Carbamidomethylation +57.021 Da (Cysteine) was selected as static modifications and oxidation +15.995 Da (Methionine) was selected as dynamic modifications. The Fase discovery rate (FDR) of peptide and protein identification were both set to 0.05. The normalization of relative protein abundances ratio was performed by total peptide amount for each LC-runs (across all runs; n=18) by normalization algorithm of Proteome discoverer software. To assembly differential expressed protein list, multiple consensus workflows were used within the Proteome Discoverer software to assemble the PSMs into peptide groups, protein database matches, and finally, non-redundant proteins groups using the principle of strict parsimony as defined by the vendor software defaults.

### Publication Abstract
None

### Keywords
Sw480, Nanobody, Peptides, Gefitinib, Egfr, Her2

### Affiliations
Mahidol University
Center of neuroscience, Faculty of science, Mahidol University, Thailand

### Submitter
Sucheewin Krobthong

### Lab Head
Dr Sucheewin Krobthong
Center of neuroscience, Faculty of science, Mahidol University, Thailand


