### Accession
PXD026484

### Title
Acute depletion of CTCF rewires genome-wide chromatin accessibility

### Description
The transcription factor CTCF appears indispensable in defining topologically associated domain boundaries and maintaining chromatin loop structures within these domains, supported by numerous functional studies. However, acute depletion of CTCF globally reduces chromatin interactions but does not significantly alter transcription. Here we systematically integrated multi-omics data including ATAC-seq, RNA-seq, WGBS, Hi-C, Cut&Run, CRISPR-Cas9 survival dropout screening, time-solved deep proteomic and phosphoproteomic analyses in cells carrying auxin-induced degron at endogenous CTCF locus. Acute CTCF protein degradation markedly rewired genome-wide chromatin accessibility. Increased accessible chromatin regions were largely located adjacent to CTCF-binding sites at promoter regions and insulator sites and were associated with enhanced transcription of nearby genes. In addition, we used CTCF-associated multi-omics data to establish a combinatorial data analysis pipeline to discover CTCF co-regulatory partners in regulating downstream gene expression. We successfully identified 40 candidates, including multiple established partners (i.e., MYC) supported by all layers of evidence. Interestingly, many CTCF co-regulators (e.g., YY1, ZBTB7A) that have evident alterations of respective downstream gene expression do not show changes at their expression levels across the multi-omics measurements upon acute CTCF loss, highlighting the strength of our system to discover hidden co-regulatory partners associated with CTCF-mediated transcription. This study highlights CTCF loss rewires genome-wide chromatin accessibility, which plays a critical role in transcriptional regulation

### Sample Protocol
Two million CTCFAID SEM cells from 4 treatment groups: no IAA, + IAA 12 hours, + IAA 24 hours, and + IAA 48 hours, each with triplicates, were applied for deep proteomic and phosphoproteomic analyses using a well-established protocol(Tan, Wu et al. 2015, Wang, Yang et al. 2015, Niu, Cho et al. 2017). In brief, cells were lysed in fresh lysis buffer (50 mM HEPES, pH 8.5, 8 M urea, 1x PhosStop Phosphatase inhibitor, 0.5% sodium deoxycholate). Proteins were quantified by the BCA protein assay (Thermo Fisher Scientific). About 100 µg proteins from each sample were digested with Lys-C (Wako, 1:100 w/w) for 2 hours, followed by trypsin digestion (Promega, 1:50 w/w) overnight at room temperature after 4x dilution with 50mM HEPES buffer. The resulting peptides from each sample were desalted, labeled with TMTpro reagents, and equally pooled. TiO2 based phosphoproteomic enrichment was then performed on the pooled sample, flow-through was further desalted and applied for off-line basic-PH fractionation. Peptides were separated into 80 fractions via a 2h gradient, and every other fraction was dried and reconstituted in 5% formic acid for MS analysis.  Peptides were analyzed on an Orbitrap HF mass spectrometer (Thermo Fisher Scientific) after separated on a 20cm X 75µm id column packed with 1.9 µm C18 resin (Dr. Maisch GmbH, Germany) and heated at 55 oC. Peptides separation was achieved through a 2h ~15-40% buffer B (0.2% formic acid, 65% CAN, 3% DMSO) gradient. The mass spectrometer was set in DDA mode with 60 000 resolution, 1 X 106 AGC target, and 50ms maximal ion time for MS1. Top 10, 1 X 105 AGC target, 105ms maximal ion time, 1 m/z isolation window and 0.2 m/z offset, 38 NCE, and 15s dynamic exclusion for MS2

### Data Protocol
Proteomic data were processed by the hybrid JUMP software suites for improving sensitivity and specificity(Wang, Li et al. 2014, Li, Wang et al. 2016). Briefly, raw files were searched against the Uniprot human database, and the same search and filtering parameters were applied to achieve 1% protein or phosphopeptide FDR using the target-decoy approach(Peng, Elias et al. 2003).

### Publication Abstract
None

### Keywords
Ctcf, Multiomics, Proteome, Phosphoproteome

### Affiliations
St Jude Children's Research Hospital
St.Jude children's Research hospi

### Submitter
Hong Wang

### Lab Head
Dr Hong Wang
St Jude Children's Research Hospital


