### Accession
PXD027937

### Title
Leptin-deficient Mouse Serum TMT-label Discovery Proteomics Profiling

### Description
The aim of this project was to profile the proteome of mouse-derived serum in order to identify those that showed significant quantitative different proteins among leptin-deficient (db/db) mouse, the genetic non-alcoholic fatty liver disease (NAFLD) mouse model and their lean bks mouse group.

### Sample Protocol
1. Protein extraction Samples were taken out from -80 oC refrigerator and centrifuged at 12000 g for 10 min at 4 oC. Cell debris was removed. The supernatant was transferred to a new centrifuge tube. Protein concentration was determined using BCA kit. 2.Trypsin enzymatic hydrolysis The protein of each sample was hydrolyzed in the same amount, and a proper amount of standard protein was added. The volume was adjusted to equal the lysate, and then dithionitol (DTT) was added to make the final concentration of 5 mM, and then reduced for 30 min at 56 oC. Iodoacetamide (IAA) was added to the final concentration of 11 mM and incubated at room temperature for 15 min in the dark. The alkylated samples were transferred to ultrafiltration tubes, centrifuged at 12000 g for 20 min at room temperature, replaced with 8 M urea for 3 times, and then replaced with replacement buffer for 3 times. Trypsin was added at the ratio of 1:50 (protease: protein, M/M), and enzymatic hydrolysis was performed overnight. The peptide was recovered by centrifugation at 12000 g for 10min at room temperature. The peptide was recovered by ultra pure water once and the peptide solution was combined twice. 3. TMT mark The trypsin-hydrolyzed peptides were desalted by Strata X C18 (Phenomenex) and vacuum freeze-dried. The peptide was dissolved with 0.5 M TEAB and labeled according to the instructions of the TMT kit. The simple operation was as follows. After thawing, the labeling reagent was dissolved in acetonitrile, mixed with the peptide and incubated at room temperature for 2 h. After mixing, the labeled peptide was desalted and freeze-dried in vacuum. 5. HPLC classification The peptides were classified by high pH reversed phase HPLC on an Agilent 300 Extend C18 column (5 μm diameter, 4.6 mm diameter, 250 mm length). The operation was as follows. The peptide was graded with a gradient of 8%-32% acetonitrile at pH 9 for 60 min to separate the 60 components, and then the peptide was merged into 14 components. The combined components were freeze-dried in vacuum for subsequent operations. 6. Liquid chromatography-mass spectrometry analysis The peptides were dissolved in liquid chromatography mobile phase A and separated using easy-NLC 1000 ultra performance liquid system. Mobile phase A consisted of an aqueous solution containing 0.1% formic acid and 2% acetonitrile. Mobile phase B consisted of an aqueous solution containing 0.1% formic acid and 90% acetonitrile. Liquid gradient setting: 0-38 min, 7%~20%B; 38-52 min, 20%-32% B; 52-56 min, 32%-80% B; 56-60 min, 80%B, flow rate maintained at 500 nL/min. The peptides were separated by an ultra performance liquid system and injected into an NSI ion source for ionization and analysis by Q Exactive™ Plus mass spectrometry. The ion source voltage was set at 2.1 kV, and the parent ion and its secondary fragments were detected and analyzed using high resolution Orbitrap. The scanning range of the first-level mass spectrometry was set to 400-1500 m/z, and the scanning resolution was set to 70000. The scanning range of the secondary mass spectrum is fixed at 100 m/z, and the resolution of the secondary mass spectrum is set to 35,000. In the data acquisition mode, the data dependent scanning (DDA) program was used, that is, after the primary scanning, the first 20 peptide parent ions with the highest signal strength were selected and successively entered the HCD collision pool for fragmentation at 28,32% of the fragmentation energy, and the secondary mass spectrometry was also performed successively. In order to improve the effective utilization of mass spectrometry, the automatic gain control (AGC) was set to 3E6, the signal threshold was set to 3.8E4 ions/s, the maximum injection time was set to 50 ms, and the dynamic exclusion time of tandem mass spectrometry was set to 30 s to avoid repeated scanning of parent ions.

### Data Protocol
1. Annotation Methods GO Annotation: Gene Ontology (GO) annotation proteome was derived from the UniProt-GOA database ( http://www.ebi.ac.uk/GOA/).  Domain Annotation: Identified proteins domain functional description were annotated by InterProScan based on protein sequence alignment method, and the InterPro (http://www.ebi.ac.uk/interpro/) domain database was used. KEGG Pathway Annotation: Firstly, using KEGG online service tools KAAS to annotated protein’s KEGG database description. Then mapping the annotation result on the KEGG pathway database using KEGG online service tools KEGG mapper. Subcellular Localization: We used wolfpsort a subcellular localization predication soft to predict subcellular localization. Special for protokaryon species, Subcellular localization prediction soft CELLO was used. 2. Functional Enrichment Enrichment of GO analysis: Proteins were classified by GO annotation into three categories: biological process, cellular compartment and molecular function. For each category, a two-tailed Fisher’s exact test was employed to test the enrichment of the differentially expressed protein against all identified proteins. The GO with a corrected p-value < 0.05 is considered significant. Enrichment of pathway analysis: Encyclopedia of Genes and Genomes (KEGG) database was used to identify enriched pathways by a two-tailed Fisher’s exact test to test the enrichment of the differentially expressed protein against all identified proteins. The pathway with a corrected p-value < 0.05 was considered significant. These pathways were classified into hierarchical categories according to the KEGG website. Enrichment of protein domain analysis: For each category proteins, InterPro database was researched and a two-tailed Fisher’s exact test was employed to test the enrichment of the differentially expressed protein against all identified proteins. Protein domains with a corrected p-value < 0.05 were considered significant. 3. Enrichment-based Clustering For further hierarchical clustering based on differentially expressed protein functional classification (such as: GO, Domain, Pathway, Complex). We first collated all the categories obtained after enrichment along with their P values, and then filtered for those categories which were at least enriched in one of the clusters with P value <0.05. This filtered P value matrix was transformed by the function x = −log10 (P value). Finally these x values were z-transformed for each functional category. These z scores were then clustered by one-way hierarchical clustering (Euclidean distance, average linkage clustering) in Genesis. Cluster membership were visualized by a heat map using the “heatmap.2” function from the “gplots” R-package. 4. Protein-protein Interaction Network All differentially expressed protein database accession or sequence were searched against the STRING database version 11.0 for protein-protein interactions. Only interactions between the proteins belonging to the searched data set were selected, thereby excluding external candidates. STRING defines a metric called “confidence score” to define interaction confidence; we fetched all interactions that had a confidence score ≥ 0.7 (high confidence). Interaction network form STRING was visualized in R package “networkD3”. 5.Database retrieval We used Maxquant (V1.6.15.0) to retrieve the secondary mass spectrometry data. Retrieval parameter Settings: the database was Mus_musculus_10090_SP_20201214. Fasta (17063 sequences), and the inverse database was added to calculate the false positive rate (FDR) caused by random matching, and the common contamination database was added to eliminate the influence of contaminated proteins in the identification results. The digestion method was set as Trypsin/P; The number of missed bits is set to 2; The minimum length of the peptide was set to 7 amino acid residues. The maximum modification number of peptide was set to 5. The mass error tolerance of the first-order parent ion in the First search and the Main search is set as 20 PPM and 4.5 PPM respectively, and the mass error tolerance of the second-order fragment ion is 20 PPM. Cysteine alkylation Carbamidomethyl (C) was set to fixed modification, variable modification to methionine oxidation, protein n-terminal acetylation, deamidation (NQ). The quantitative method was set to TMT-10plex, and the FDR for both protein identification was set to 1%. The accuracy of FDR was set as 1% at the three levels of spectrum, peptide and protein. The identified protein must contain at least one specific (unique) peptide.

### Publication Abstract
None

### Keywords
Tmt-lable, Lc-ms/ms, Serum

### Affiliations
University of Science and Technology of China (USTC)
Department of Endocrinology, The First Affiliated Hospital of USTC, Division of Life Sciences and Medicine, University of Science and Technology of China, Hefei, China.

### Submitter
Yu Zhang

### Lab Head
Dr Suowen Xu
Department of Endocrinology, The First Affiliated Hospital of USTC, Division of Life Sciences and Medicine, University of Science and Technology of China, Hefei, China.


