### Accession
PXD046585

### Title
An oscillatory transcription factor complex regulates adaptive changes in macropinocytosis during the growth-to-development transition

### Description
Macropinocytosis is an evolutionarily conserved endocytic pathway that mediates non-selective bulk uptake of extracellular fluid. It is the major route by which axenic Dictyostelium cells obtain nutrients and has emerged as a nutrient-scavenging pathway for mammalian cells. How cells adjust macropinocytic activity in various physiological or developmental contexts is an important question that remains largely unanswered. We discovered that, in Dictyostelium cells, the transcription factors Hbx5 and MybG form a functional complex in the nucleus to maintain macropinocytic activity during the growth stage. In contrast, during starvation-induced multicellular development, the transcription factor complex undergoes nucleocytoplasmic shuttling in response to oscillatory cyclic adenosine 3',5'-monophosphate (cAMP) signals, which leads to increased cytoplasmic retention of the complex and progressive downregulation of macropinocytosis. Therefore, by coupling macropinocytosis-related gene expression to the cAMP oscillation system that facilitates long-range cell-cell communication, the dynamic translocation of the Hbx5-MybG complex orchestrates a population-level adjustment of macropinocytic activity to adapt to changing environmental conditions.

### Sample Protocol
Equal amounts of proteins were separated by SDS-PAGE and stained using the Colloidal Blue Staining Kit (Thermo Fisher Scientific). Each gel lane was cut into four slices and digested with trypsin. Liquid chromatographyâ€“tandem MS analysis of the resulting peptides was performed with a Thermo Orbitrap Fusion Lumos (Thermo Fisher Scientific) using a nano-electrospray ion source with an electrospray voltage of 2.2kV. The MS full scan was performed using an Orbitrap Mass Analyzer (300-1500 m/z) with a resolution of 120,000@m/z 200, AGC target 500,000, and a maximum injection time of 50ms, followed by MS/MS scans generated by HCD fragmentation at a resolution of 30,000@m/z 200.AGC target 50,000 and maximum injection time mode was set dynamic. Isolation width was set at 1.6m/z and the HCD collision energy was set at 30%. MS scan functions and nLC solvent gradients were controlled by the Xcalibur data system (Thermo Fisher Scientific).

### Data Protocol
Full MS and tandem mass spectra were extracted from raw files, and the tandem mass spectra were searched against the Uniprot Dictyostelium_discoideum protein database using Proteome Discoverer 2.2 software (Thermo Fisher Scientific). The processing workflow includes Sequest HT, Percolator, PhosphoRS and Precursor Ions Quantifier (Thermo Proteome Discoverer User Guide). Enzyme specificity was set as trypsin and the maximum of missed trypsin cleavage sites as 2. The precursor mass tolerance was set to 10 ppm, and the fragment ion mass tolerance was set to 0.02 Da. Carbamidomethylation (C) was considered as a fixed modification. Oxidation (M) and acetylation (protein N-term) were variable modifications. Serine, threonine and tyrosine were treated as dynamically modified by +79.9663 Da for phosphorylation. The false discovery rate (FDR) applied at the peptide and protein levels was 1%. The threshold of PTM site probability using PhosphoRS was set as 99%, Precursor abundance of the indicated site was determined by area at the MS1 level using Precursor Ions Quantifier. The abundance ratio at a given site was calculated as the ratio of the abundance of phosphorylated residues to the total residue abundance.

### Publication Abstract
None

### Keywords
Dictyostelium discoideum, Lc-ms, Hbx5, Mybg, Transcription factor

### Affiliations
Institute of Biophysics, Chinese Academy of Sciences
National Laboratory of Biomacromolecules, Institute of Biophysics, Chinese Academy of Sciences, Beijing 100101, China 

### Submitter
Yazhou Hao

### Lab Head
Dr Huaqing Cai
Institute of Biophysics, Chinese Academy of Sciences


