### Accession
PXD034045

### Title
Shotgun LC-MS analysis of human serum from COVID-19 patients

### Description
Coronavirus disease 2019 (COVID-19) is caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), whose outbreak in 2019 led to an ongoing pandemic with devastating consequences for the global economy and human health. According to World Health Organization, COVID-19 has affected more than 481 million people worldwide, with 6 million confirmed deaths. The joint efforts of the scientific community have undoubtedly increased the pace of production of COVID-19 vaccines, but there is still so much uncharted ground to cover regarding the mechanisms of SARS-CoV-2 infection, replication and host response. This issues can be approached by proteomics with unprecedented capacity paving the way for the development of more efficient strategies for patient care. In this study, we present a deep proteome analysis that has been performed on a series of COVID-19 patients aiming to identify proteins assessing the dynamics of the disease at different age ranges

### Sample Protocol
Serum aliquots (5 µl) of each individual (corresponding to about 300µg of protein), were mixed in a 96-well multiplate with 5µl of a buffer containing 50mM Tris, 2 % SDS and 100mM DTT, and boiled for 5 minutes for protein denaturation. Then, proteins were subjected to  filter-aided digestion using 96-well plates (AcroPrep™ 96-well Filter Plates, PALL Nanosep Centrifugal Devices with Omega Membrane-10K, PALL) coupled to a vacuum manifold (PALL) according to manufacter’s instructions. Briefly, 200µl of urea were added to each sample to dilute SDS and cysteine residues were blocked with 50mM iodoacetamide for 1h at 37ºC in darkness. After two washes with 100 µl urea followed by two washes with 100 µl of 100mM ammonium bicarbonate pH 8.8 (AB), proteins were digested with trypsin (1:30-trypsin:protein, Promega) O/N at 37 ˚C. After protein digestion, peptides were eluted from the filter in two different steps using 40 µl of AB and 50 µl of 500mM NaCl, respectively. Peptides were acidified with 25% trifluoroacetic acid (TFA) to a final concentration of 1%, desalted with Oasis cartridges (Waters) following manufacture’s instructions, dried with a Speed-Vac and stored at -20 ˚C. For stable isobaric labeling, peptides were dissolved in 70 µl of 100 mM Triethylammonium bicarbonate (TEAB) buffer, and peptide concentration was measured using DirectDetect Infrared Spectrometer (Merck). 50 µg of peptides from each sample were subjected to multiplexed isobaric labeling with 10-plex TMT reagents (Thermo Fisher Scientific) following manufacter’s instructions with minor changes. Each TMT experiment contained 9 individuals and one-channel was reserved for reference internal standard samples created by pooling plasma samples. Peptides were labeled with 0.8 mg of TMT reagents, previously reconstituted in 42 µl of acetonitrile, at room temperature for 1 hour. Reaction was stopped with 0.5% TFA after 10 min incubation. After labeling, peptides from all the samples in each TMT experiment were mixed in the same tube, concentrated in a Speed-Vac, acidified with 25%TFA to a final concentration of 1% and desalted with Oasis cartridges (Waters). Aliquots of 1/10 (in volume) were reserved for direct MS analysis (without peptide fractionation) and the remaining peptides were saved for further peptide fractionation before MS analysis. Labeled peptide aliquots were dried-down and stored at -20 ˚C. Peptides were fractionated using the high pH reversed-phase peptide fractionation kit (Thermo Fisher Scientific) according to manufacter’s instructions.

### Data Protocol
Labeled peptide samples were analyzed using an Ultimate 3000 HPLC system (Thermo Fisher Scientific) coupled via a nanoelectrospray ion source (Thermo Fisher Scientific, Bremen, Germany) to a Q Exactive HF mass spectrometer (Thermo Fisher Scientific). C18-based reverse phase separation was performed using a PepMap 100 C18 5 μm  0.3 × 5 mm as trapping column (Thermo Fisher Scientific) and a PepMap RSLC C18 EASY-Spray column 50 cm × 75 μm ID as analytical column (Thermo Fisher Scientific). Peptides were loaded in buffer A (0.1% of formic acid in water (v/v)) and eluted with a 338 min linear gradient of buffer B (100% ACN, 0.1% formic acid (v/v)) at 200nl/min. Mass spectra were acquired in data-dependent manner. MS spectra were acquired in the Orbitrap analyser using full ion-scan mode with a mass range of 400-1500 mass-to-charge (m/z) and 70,000 FT resolution. The automatic gain control target was set at 2x 105 with 50 ms maximun injection time. MS/MS was performed using the top-speed adquisition mode with 3s cycle time. HCD fragmentation was performed at 30% of normalized collision energy and MS/MS spectra were analysed at a 60,000 resolution in the Orbitrap.  For peptide identification MS/MS spectra were searched with the SEQUEST HT algorithm implemented in Proteome Discoverer 2.1 (Thermo Scientific) against a Uniprot database comprised human protein sequences (July 2014, 157430 entries), using trypsin digestion with a maximum of 2 missed cleavages, using as fixed modifications Cys carbamidomethylation (57.021464 Da) and TMT labeling at N-terminal end and Lys (229.162932 Da), and Met oxidation (15.994915) as dynamic modification. Precursor mass tolerance was set at 800 ppm and fragment mass tolerance at 0.03 Da; precursor charge range was set to 2-4.

### Publication Abstract
Coronavirus disease 2019 (COVID-19) is caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), whose outbreak in 2019 led to an ongoing pandemic with devastating consequences for the global economy and human health. According to the World Health Organization, COVID-19 has affected more than 481 million people worldwide, with 6 million confirmed deaths. The joint efforts of the scientific community have undoubtedly increased the pace of production of COVID-19 vaccines, but there is still so much uncharted ground to cover regarding the mechanisms of SARS-CoV-2 infection, replication and host response. These issues can be approached by proteomics with unprecedented capacity paving the way for the development of more efficient strategies for patient care. In this study, we present a deep proteome analysis that has been performed on a cohort of 72 COVID-19 patients aiming to identify serum proteins assessing the dynamics of the disease at different age ranges. A panel of 53 proteins that participate in several functions such as acute-phase response and inflammation, blood coagulation, cell adhesion, complement cascade, endocytosis, immune response, oxidative stress and tissue injury, have been correlated with patient severity, suggesting a molecular basis for their clinical stratification. Eighteen protein candidates were further validated by targeted proteomics in an independent cohort of 84 patients including a group of individuals that had satisfactorily resolved SARS-CoV-2 infection. Remarkably, all protein alterations were normalized 100 days after leaving the hospital, which further supports the reliability of the selected proteins as hallmarks of COVID-19 progression and grading. The optimized protein panel may prove its value for optimal severity assessment as well as in the follow up of COVID-19 patients.

### Keywords
Human, Lc-ms, Covid-19, Serum

### Affiliations
Laboratorio de Proteómica Cardiovascular, Centro Nacional de Investigaciones Cardiovasculares (CNIC), Madrid, Spain
Centro Nacional de Investigaciones Cardiovasculares

### Submitter
Estefanía Núñez

### Lab Head
Dr Jesús Vázquez
Laboratorio de Proteómica Cardiovascular, Centro Nacional de Investigaciones Cardiovasculares (CNIC), Madrid, Spain


