### Accession
PXD007813

### Title
Time-dependent olfactory  proteomics in Tg2576 mice

### Description
Olfaction is often deregulated in Alzheimer´s disease (AD) patients, being also impaired in transgenic Tg2576 AD mouse model, which overexpress the Swedish mutated form of human amyloid precursor protein (APP). However, little is known about the molecular mechanisms that accompany the neurodegeneration of olfactory structures in Tg2576 mice. For that, we have applied proteome-wide approaches to probe molecular disturbances in the olfactory bulb (OB) dissected from Tg2576 mice respect to age matched wild-type littermates (2 and 6 months of age).

### Sample Protocol
Sample preparation for proteomic analysis – Murine OB specimens were homogenized in lysis buffer containing 7 M urea, 2 M thiourea, 50 mM DTT. The homogenates were spinned down at 100.000 x g for 1 h at 15ºC. Protein concentration was measured in the supernatants with the Bradford assay kit (Bio-rad). Protein Digestion and Peptide iTRAQ Labeling. A shotgun comparative proteomic analysis of OB proteomes using iTRAQ (isobaric Tags for Relative and Absolute Quantitation) was performed. Protein extracts were precipitated with Clean-up kit (Bio-Rad), and pellets dissolved in 7M urea, 2 M thiourea. Protein quantitation was performed with the Bradford assay kit (Bio-Rad). iTRAQ labeling of each sample was performed according to the manufacturer's protocol (Sciex). Briefly, equal amounts of OB proteins (50 μg) from each sample were reduced with 50 mM tris (2-carboxyethyl) phosphine (TCEP) at 60 °C for 1 h. Cysteine residues were alkylated with 200 mM methylmethanethiosulfonate (MMTS) at room temperature for 15 min. Protein enzymatic cleavage was carried out with trypsin (Promega; 1:20, w/w) at 37 °C for 16 h. Each tryptic digest was labelled according to the manufacturer's instructions with one isobaric amine-reactive tags as follows: SET1 (2 months): Tag113, WT-1; Tag114, WT-2; Tag115, WT-3; Tag116, Tg2576-1; Tag117, Tg2576-2; Tag118, tg2576-3; SET2 (6 months of age): Tag113, WT-1; Tag114, WT-2; Tag115, WT-3; Tag116, Tg2576-1; Tag117, Tg2576-2; Tag118, tg2576-3. After 2h incubation, the set of labelled samples were pooled and evaporated in a vacuum centrifuge.  Liquid Chromatography tandem mass Spectrometry and data Analysis. To increase the proteome coverage, the peptide pool was submitted to cation exchange chromatography using spin Columns (Pierce). 12 fractions were collected (from 10mM to 150 mM KCl), evaporated under vacuum and reconstituted into10 μl of 2% acetonitrile, 0.1% formic acid, 98% MilliQ-H20 prior to mass spectrometric analysis. Peptide mixtures were separated by reverse phase chromatography using an Eksigent nanoLC ultra 2D pump fitted with a 75 μm ID column (Eksigent 0.075 x 250). Samples were first loaded for desalting and concentration into a 0.5 cm length 100 μm ID precolumn packed with the same chemistry as the separating column. Mobile phases were 100% water 0.1% formic acid (FA) (buffer A) and 100% Acetonitrile 0.1% FA (buffer B). Column gradient was developed in a 146 min two step gradient from 5% B to 25% B in 120 min and 25%B to 70% B in 15 min. Column was equilibrated in 95% B for 9 min and 5% B for 14 min. During all process, precolumn was in line with column and flow maintained all along the gradient at 300 nl/min. Eluting peptides from the column were analyzed using an Sciex 5600 Triple-TOF system. Information data acquisition was acquired upon a survey scan performed in a mass range from 350 m/z up to 1250 m/z in a scan time of 250 ms. Top 25 peaks were selected for fragmentation. Minimum accumulation time for MS/MS was set to 100 ms giving a total cycle time of 2.75 s. Product ions were scanned in a mass range from 100 m/z up to 1500 m/z and excluded for further fragmentation during 15 s.

### Data Protocol
Data analysis. The raw MS/MS spectra search were processed using the freely available MaxQuant software (v.1.5.8.3) [20]. Initial maximum precursor and fragment mass deviations were set to 25 ppm and 0.5 Da, respectively. Variable modification (methionine oxidation and N-terminal acetylation) and fixed modification (MMTS) were set for the search and trypsin with a maximum of 1 missed cleavages was chosen for searching. The minimum peptide length was set to 7 amino acids and the false discovery rate (FDR) for PSM and protein identification was set to 1%. Matching against the protein sequence database was carried out after improving the precursor ion mass accuracy using the mass-dependent recalibration option of the software. The false-discovery rate was controlled at various levels by using a target-decoy search strategy, which integrates multiple peptide parameters such as length, charge, number of modifications and the identification score into a single quality that acts as the statistical evidence on the quality of each single peptide spectrum match. The frequently observed laboratory contaminants were removed and the protein identification was considered valid only when at least one unique or “razor” peptide was present. Following protein identification, the intensity for each identified protein was calculated using ion reporter intensities. The protein quantification was calculated using at least 2 razor + unique peptides, and calculation of statistical significance was carried out using two-way Student-t test (p < 0.05). A 1.3-fold change cutoff for all iTRAQ ratios (ratio ≤0.77 or ≥1.3) was selected to classify proteins as up- or down-regulated. Proteins with iTRAQ ratios below the low range (0.77) were considered to be underexpressed, whereas those above the high range (1.3) were considered to be overexpressed. The freely available software Perseus (version 1.5.6.0) [21] was used for statistical analysis and data visualization.

### Publication Abstract
The olfactory bulb (OB) is the first processing station in the olfactory pathway. Despite smell impairment, which is considered an early event in Alzheimer's disease (AD), little is known about the initial molecular disturbances that accompany the AD development at olfactory level. We have interrogated the time-dependent OB molecular landscape in Tg2576 AD mice prior to the appearance of neuropathological amyloid plaques (2-, and 6-month-old), using combinatorial omics analysis. The metabolic modulation induced by overproduction of human mutated amyloid precursor protein (APP) clearly differs between both time points. Besides the progressive perturbation of the APP interactome, functional network analysis unveiled an inverse regulation of downstream extracellular signal-regulated kinase (ERK1/2), and p38 mitogen-activated protein kinase (MAPK) routes in 2-month-old Tg2576 mice with respect to wild-type (WT) mice. In contrast, Akt and MAPK kinase 4 (SEK1)/ stress-activated protein kinase (SAPK) axis were parallel activated in the OB of 6-months-old-Tg2576 mice. Furthermore, a survival kinome profiling performed during the aging process (2-, 6-, and 18-month-old) revealed that olfactory APP overexpression leads to changes in the activation dynamics of protein kinase A (PKA), and SEK1/MKK4-SAPK/JNK between 6 and 18 months of age, when memory deficits appear and AD pathology is well established in transgenic mice. Interestingly, both olfactory pathways were differentially activated in a stage-dependent manner in human sporadic AD subjects with different neuropathological grading. Taken together, our data reflect the early impact of mutated APP on the OB molecular homeostasis, highlighting the progressive modulation of specific signaling pathways during the olfactory amyloidogenic pathology.

### Keywords
Proteomics, Olfactory bulb, Alzheimer

### Affiliations
Proteomics Unit, Navarrabiomed-Proteored (Pamplona-Spain)
Proteomics unit

### Submitter
Enrique SantamarÃ­a

### Lab Head
Dr Enrique Santamaria
Proteomics Unit, Navarrabiomed-Proteored (Pamplona-Spain)


