### Accession
PXD043092

### Title
Protein atlas of four larval stages of Maruca vitrata

### Description
The present work is the pioneering report depicting the total proteomic atlas of four major developmental stages (third, fourth, fifth instar and pupa) of M. vitrata by LC-MS/MS analyses.

### Sample Protocol
Samples were processed using a modified version of filter-aided sample preparation (FASP; WiÅ›niewski et al. 2009) as per the detailed procedure described earlier (Rathinam et al., 2020).  Hydrophilic interaction chromatography (HILIC) was performed on an Agilent 1200 HPLC system (Agilent AG) using a YMC-Pack Polyamine II column (S-5Âµm; 12nm; 250mm x 3mm; YMC Europe GmbH). Peptides were dissolved in solvent A (75% ACN; 8mM KH2PO4; pHÂ 4.5) and loaded on the column with a flow rate of 0.5 ml/min. Peptides were eluted with the following gradient of solvent B (5% ACN; 100mM KH2PO4; pHÂ 4.5; 7.5 min isocratic 0% B; 30 min to 50% B; 5 min to 100% B; 5 min isocratic 100% B; 12.5 min isocratic 0% B). Twenty-seven fractions were collected over a 60 min run time. After vacuum concentration, individual fractions were combined to eleven samples for SPE cleanup and LC-MS/MS analysis. Dissolved samples were injected using a trap/elute mode into Waters M-class UPLC equipment (Waters AG, GA, USA). A trap column made of Symmetry C18 (5 um, 180 um, and 20 mm, Waters AG) and a separation column made of HSS T3 C18 reverse phase (1.8 um, 75 um, and 250 mm, Waters AG) were employed. The columns were equilibrated with 99% solvent A (0.1% formic acid (FA) in water) and 1% solvent B (0.1% FA in ACN). Trapping of peptides was performed at 15Âµl/min for 30sec and afterwards the peptides were eluted using the following gradient: 1-30% B in 120min; 30-98% B in 5min. The flow rate was constant 0.3Âµl/min and the temperature was controlled at 50 degrees Celsius. High accuracy mass spectra were cquired with an Q-Exactive HF mass spectrometer (Thermo Scientific) that was operated in data dependent acquisition mode. A survey scan was followed by up to 12 MS2 scans. The survey scan was recorded using quadrupole transmission in the mass range of 350-1500m/z with an AGC target of 3E6, a resolution of 120â€™000 at 200m/z and a maximum injection time of 50ms. All fragment mass spectra were recorded with a resolution of 30â€™000 at 200m/z, an AGC target value of 1E5 and a maximum injection time of 50ms. The normalized collision energy was set to 28%. Dynamic exclusion was activated and set to 30sec.

### Data Protocol
All the generated data was stored within the B-Fabric platform developed and maintained at FGCZ (TÃ¼rker, C., Akal, F., & Schlapbach, R. (2011) (PMID: 21772064). After data collection, peak lists were generated using FCC (Barkow-Oesterreicher, et al. 2013) and Proteome Discoverer 2.1 (Thermo Scientific). They were searched against a protein database from RNAseq data of all the three stages (75708 entries) concatenated with its decoyed version and an in-house built contaminant database (262 entries) using the Mascot search algorithm (Mascot 2.5; Matrix Science Inc., UK). The following search parameters were used: precursor ion mass tolerance 10 ppm; fragment ion mass tolerance 0.03 Da; trypsin digestion (one missed cleavage allowed); fixed modifications of carbamidomethyl of cysteine; variable modification oxidation of methionine as well as acetylation on protein N-term. After the Mascot search, the data were further evaluated using Scaffold 4.4 (Proteome Software Inc., OR, USA), thresholds for protein identification were set to 1% false discovery rate (FDR) on protein level, 0.1% FDR on peptide level and 2 peptides per protein.

### Publication Abstract
The spotted pod borer, Maruca vitrata (Lepidoptera: Crambidae) is a destructive insect pest that inflicts significant productivity losses on important leguminous crops. Unravelling insect proteomes is vital to comprehend their fundamental molecular mechanisms. This research delved into the proteome profiles of four distinct stages -three larval and pupa of M. vitrata, utilizing LC-MS/MS label-free quantification-based methods. Employing comprehensive proteome analysis with fractionated datasets, we mapped 75&#xa0;% of 3459 Drosophila protein orthologues out of which 2695 were identified across all developmental stages while, 137 and 94 were exclusive to larval and pupal stages respectively. Cluster analysis of 2248 protein orthologues derived from MaxQuant quantitative dataset depicted six clusters based on expression pattern similarity across stages. Consequently, gene ontology and protein-protein interaction network analyses using STRING database identified cluster 1 (58 proteins) and cluster 6 (25 proteins) associated with insect immune system and lipid metabolism. Furthermore, qRT-PCR-based expression analyses of ten selected proteins-coding genes authenticated the proteome data. Subsequently, functional validation of these chosen genes through gene silencing reduced their transcript abundance accompanied by a marked increase in mortality among dsRNA-injected larvae. Overall, this is a pioneering study to effectively develop a proteome atlas of M. vitrata as a potential resource for crop protection programs.

### Keywords
Hilic, Lc-ms/ms, Larval stages

### Affiliations
ICAR-NRCPB, LBS Centre, Pusa Campus, IN-110012 New Delhi
University of Zurich

### Submitter
Bernd Roschitzki

### Lab Head
Dr Rohini Sreevathsa
ICAR-NRCPB, LBS Centre, Pusa Campus, IN-110012 New Delhi


