### Accession
PXD015873

### Title
The mitochondrial HSP90 paralog TRAP1 forms an OXPHOS-regulated tetramer and is involved in maintaining mitochondrial metabolic homeostasis

### Description
The molecular chaperone TRAP1, the mitochondrial isoform of cytosolic HSP90, remains poorly understood with respect to its pivotal role in the regulation of mitochondrial metabolism. Most studies have found it to be an inhibitor of mitochondrial oxidative phosphorylation (OXPHOS) and an inducer of the Warburg phenotype of cancer cells. However, others have reported the opposite and there is no consensus on the relevant TRAP1 interactors. This calls for a more comprehensive analysis of the TRAP1 interactome and of how TRAP1 and mitochondrial metabolism mutually affect each other. We show that the disruption of the gene for TRAP1 in a panel of cell lines dysregulates OXPHOS by a metabolic rewiring that induces the anaplerotic utilization of glutamine metabolism to replenish TCA cycle intermediates. Restoration of wild-type levels of OXPHOS requires full-length TRAP1. Whereas the TRAP1 ATPase activity is dispensable for this function, it modulates the interactions of TRAP1 with various mitochondrial proteins. Quantitatively by far the major interactors of TRAP1 are the mitochondrial chaperones mtHSP70 and HSP60. However, we find that the most stable stoichiometric TRAP1 complex is a TRAP1 tetramer, whose levels change in response to both a decline or an increase in OXPHOS. Our work provides a roadmap for further investigations of how TRAP1 and its interactors such as the ATP synthase regulate cellular energy metabolism. Our results highlight that TRAP1 function in metabolism and cancer cannot be understood without a focus on TRAP1 tetramers as potentially the most relevant functional entity.

### Sample Protocol
UMUC3 (bladder cancer) cells, either WT or with TRAP1 knock-out, were cultured in light or heavy (Lys8/Arg10) SILAC media for 5 doublings and the label incorporation verified. Cells were grown in replicate, with one label swap performed. Cells were lysed in 4% SDS, 100 mM TEAB pH 8.5 and equal amounts of each WT/KO labeling pair (100 ug) combined. Proteins were digested with trypsin following the FASP protocol. After desalting, samples were fractionated by high pH reverse phase separation. Peptes were analyzed on an Orbitrap Fusion following separation on an UltiMate 3000 RSLCnano on a 75 umx15cm 2um Acclaim PepMap column. Parent spectra were collected in the orbitrap mass analyzer, with HCD-generated fragment spectra collected in the ion trap.

### Data Protocol
Data were searched using MaxQuant 1.6.0.16 against the human reference proteome from Uniprot. SILAC labels were included for quantitation. Carbamidomethylation of cysteine was included as a static modification, oxidation of methionine and protein N-terminal acetylation as a variable modifications. Data were processed using R (https://www.r-project.org) and figures were produced using the package ggplot2 (https://ggplot2.tidyverse.org). The initial list of proteins identified in the SILAC analysis was first filtered for proteins represented in all three biological replicates for KO and WT cells. Next, this list of proteins was filtered for only mitochondrial proteins (200 proteins). The volcano plot was generated based on a minimum KO/WT fold-change of 1.5 with a p-value of â‰¤ 0.05.

### Publication Abstract
None

### Keywords
Lc-ms, Trap1, Silac, Hsp90 paralog

### Affiliations
National Cancer Institute, NIH
Laboratory of Cell Biology, National Cancer Institute, National Institutes of Health

### Submitter
Lisa Jenkins

### Lab Head
Dr Lisa Jenkins
Laboratory of Cell Biology, National Cancer Institute, National Institutes of Health


