### Accession
PXD029545

### Title
Quantitative phosphoproteomic analysis of Eimeria tenella six stages (unsporulated oocysts (USO), partial sporulated 7 h oocysts (SO7h), sporulated oocysts (SO), sporozoites (S), second generation merozoite (M2) and gametocyte (G))

### Description
we performed a comprehensive, quantitative phosphoproteomic profile analysis of six stages of the E. tenella life cycle: unsporulated oocysts (USO), partial sporulated 7 h oocysts (SO7h), sporulated oocysts (SO), sporozoites (S), second generation merozoite (M2) and gametocyte (G). The results revealed significant changes in the abundance of phosphoproteins during E. tenella development. The findings shed light on the key roles of protein phosphorylation and dephosphorylation in the E. tenella life cycle.

### Sample Protocol
Samples were first grinded by liquid nitrogen. Then each sample (0.8mL) was sonicated on ice using a high intensity ultrasonic processor (Scientz) in the 4 volumes of phenol extraction buffer (containing 10 mM dithiothreitol (DTT), 1% protease inhibitor, 1% phosphatase inhibitor). An equal volume of Tris-saturated phenol was added, and the mixture was centrifuged with 5500 × g at 4℃ for 10 min. The upper phenol phase was transferred to a new centrifuge tube. Proteins were precipitated by adding at least 5 volumes of 0.1M ammonium acetate/methanol and incubated overnight. Centrifugation at 4 °C for 10 min, the remaining precipitate was washed with 3mL methanol once, followed by 3mL acetone for two times. The protein was redissolved in 1.5mL 8 M urea and the protein concentration was determined with BCA kit. The final concentration of 20% TCA was slowly added into the proteins, precipitation at 4°C for 2h. Centrifugation (4 °C, 5 min, 4500 × g), the remaining precipitate was washed with 3mL pre-cooled acetone 2-3 times. The protein sample was then diluted by adding 3mL 200 mM TEAB. The protein solution was reduced with dithiothreitol (DTT) (final concentration 5 mM) for 30 min at 56℃ and alkylated with iodoacetamide (IAA) (final concentration 11 mM) for 15 min at room temperature in darkness. Trypsin was added in a ratio of 1:50 (protease: protein, m/m) for digestion overnight. The peptide mixtures were dissolved in the loading buffer solution (50% acetonitrile/6% trifluoroacetic acid), and ensure the concentration of peptides was 3mg/ mL. The supernatant was transferred to the pre-washed IMAC material (Thermo, Catalog Number A32992). After incubation, the IMAC resin were washed with 50% acetonitrile/6% trifluoroacetic acid and 30% acetonitrile/0.1% trifluoroacetic acid, sequentially. Elution buffer containing 10% NH4OH was added and the eluate was collected and lyophilized. The resulting peptides were desalted with C18 ZipTips (Millipore) instructions. LC-MS/MS analysis was performed with TIMS-TOF Pro mass spectrometry (Bruker) combined with nanoElute UHPLC (Bruker). Enriched phosphopeptides samples were dissolved in mobile phase A (0.1% (v/v) formic acid) and loaded onto a home-made reversed-phase analytical column. The mobile phase B was an acetonitrile (ACN) solution containing 0.1% formic acid (FA). The gradient of LC was set as: 2% to 22% buffer B (0-50 min), 22% to 35% B (50-52 min), 35% to 90% B (52-55 min), and 90% B (55-60 min). The flow rate was set to 0.45 µL/min. After the separation, the peptides were injected into Capillary ion source for ionization and analyzed by TIMS-TOF Pro mass spectrometry. The ion source voltage was set at 1.6kV, and the peptide parent ions and their secondary fragments were detected and analyzed using TOF. The scanning range of secondary mass spectrometry was set to 100-1700 m/z. The data acquisition mode is parallel cumulative serial fragmentation (PASEF) mode. After a first-order mass spectrometry collection, the second-order spectrogram with the charge number of parent ions in the range of 0-5 was collected in PASEF mode 10 times. The dynamic exclusion time of tandem mass spectrometry scanning was set to 30 seconds to avoid repeated scanning of parent ions.

### Data Protocol
The resulting MS/MS data were processed using Maxquant search engine (v.1.6.6.0). Tandem mass spectra were searched against Eimeria_tenella_5802_UP_20200108 database (9323 sequences) concatenated with reverse decoy database. Trypsin/P was specified as a cleavage enzyme allowing up to 2 missing cleavages. A common contamination database was added to eliminate the influence of contaminated proteins in the identification results. The minimum peptide length was set to 7 amino acid residues and the maximum number of peptide modifications was set to 5. The mass tolerance for precursor ions was set as 20 ppm in First search and 20 ppm in Main search, and the mass tolerance for fragment ions was set as 0.02 Da. Carbamidomethyl on Cys was specified as fixed modification, acetylation modification, oxidation on Met and phosphorylation modification was specified as variable modifications. FDR was adjusted to < 1%.

### Publication Abstract
Protein phosphorylation is an important post-translational modification (PTM) involved in diverse cellular functions. It is the most prevalent PTM in both <i>Toxoplasma gondii</i> and <i>Plasmodium falciparum</i>, but its status in <i>Eimeria tenella</i> has not been reported. Herein, we performed a comprehensive, quantitative phosphoproteomic profile analysis of four stages of the <i>E. tenella</i> life cycle: unsporulated oocysts (USO), partially sporulated (7 h) oocysts (SO7h), sporulated oocysts (SO), and sporozoites (S). A total of 15,247 phosphorylation sites on 9514 phosphopeptides corresponding to 2897 phosphoproteins were identified across the four stages. In addition, 456, 479, and 198 differentially expressed phosphoproteins (DEPPs) were identified in the comparisons SO7h vs. USO, SO vs. SO7h, and S vs. SO, respectively. Gene Ontology (GO) term and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway enrichment analyses of DEPPs suggested that they were involved in diverse functions. For SO7h vs. USO, DEPPs were mainly involved in cell division, actin cytoskeleton organization, positive regulation of transport, and pyruvate metabolism. For SO vs. SO7h, they were related to the peptide metabolic process, translation, and RNA transport. DEPPs in the S vs. SO comparison were associated with the tricarboxylic acid metabolic process, positive regulation of ATPase activity, and calcium ion binding. Time course sequencing data analysis (TCseq) identified six clusters with similar expression change characteristics related to carbohydrate metabolism, cytoskeleton organization, and calcium ion transport, demonstrating different regulatory profiles across the life cycle of <i>E. tenella</i>. The results revealed significant changes in the abundance of phosphoproteins during <i>E. tenella</i> development. The findings shed light on the key roles of protein phosphorylation and dephosphorylation in the <i>E. tenella</i> life cycle.

### Keywords
Eimeria tenella, Lc-ms/ms

### Affiliations
lanzhou veterinary research institute
Lanzhou VeterinaLanzhou Veterinary Research Institute, Chinese Academy of Agricultural Sciences (LVRI, CAASLanzhou Veterinary Research Institute, Chinese Academy of Agricultural Sciences (LVRI, CAAS)

### Submitter
baohong Liu

### Lab Head
Dr Jianping Cai
Lanzhou VeterinaLanzhou Veterinary Research Institute, Chinese Academy of Agricultural Sciences (LVRI, CAASLanzhou Veterinary Research Institute, Chinese Academy of Agricultural Sciences (LVRI, CAAS)


