### Accession
PXD023286

### Title
ChAdOx infection of human cell lines A549 MRC-5

### Description
Total proteome and phospho proteome of two human cell lines infected with the ChAdOx1 nCoV-19 virus over 72 hours

### Sample Protocol
Protein lysates were prepared from MRC5 or A549 cells only following mock infection, or infection with ChAdOx1 nCoV-19 for 24h, 48h or 72h as previously described23.  Aliquots of 100µg of each sample were digested with trypsin (2.5µg trypsin per 100µg protein; 37°C, overnight), labelled with Tandem Mass Tag (TMT) ten plex reagents according to the manufacturer’s protocol (Thermo Fisher Scientific, Loughborough, LE11 5RG, UK) and the labelled samples pooled according to cell line. For the Total proteome analysis, an aliquot of 50µg of the pooled sample was desalted using a SepPak cartridge according to the manufacturer’s instructions (Waters, Milford, Massachusetts, USA). Eluate from the SepPak cartridge was evaporated to dryness and resuspended in buffer A (20 mM ammonium hydroxide, pH 10) prior to fractionation by high pH reversed-phase chromatography using an Ultimate 3000 liquid chromatography system (Thermo Fisher Scientific).  In brief, the sample was loaded onto an XBridge BEH C18 Column (130Å, 3.5 µm, 2.1 mm X 150 mm, Waters, UK) in buffer A and peptides eluted with an increasing gradient of buffer B (20 mM ammonium hydroxide in acetonitrile, pH 10) from 0-95% over 60 minutes.  The resulting fractions (15 in total) were evaporated to dryness and resuspended in 1% formic acid prior to analysis by nano-LC MSMS using an Orbitrap Fusion Lumos mass spectrometer (Thermo Scientific). For the Phospho proteome analysis, the remainder of the TMT-labelled pooled sample was also desalted using a SepPak cartridge (Waters, Milford, Massachusetts, USA). Eluate from the SepPak cartridge was evaporated to dryness and subjected to TiO2-based phosphopeptide enrichment according to the manufacturer’s instructions (Pierce).  The flow-through and washes from the TiO2-based enrichment were then subjected to FeNTA-based phosphopeptide enrichment according to the manufacturer’s instructions (Pierce). The phospho-enriched samples were again evaporated to dryness and then resuspended in 1% formic acid prior to analysis by nano-LC MSMS using an Orbitrap Fusion Lumos mass spectrometer (Thermo Scientific).    High pH RP fractions (Total proteome analysis) or the phospho-enriched fractions (Phospho-proteome analysis) were further fractionated using an Ultimate 3000 nano-LC system in line with an Orbitrap Fusion Lumos mass spectrometer (Thermo Scientific).  In brief, peptides in 1% (vol/vol) formic acid were injected onto an Acclaim PepMap C18 nano-trap column (Thermo Scientific). After washing with 0.5% (vol/vol) acetonitrile 0.1% (vol/vol) formic acid peptides were resolved on a 250 mm × 75 μm Acclaim PepMap C18 reverse phase analytical column (Thermo Scientific) over a 150 min organic gradient, using 7  gradient segments (1-6% solvent B over 1min., 6-15% B over 58min., 15-32%B over 58min., 32-40%B over 5min., 40-90%B over 1min., held at 90%B for 6min and then reduced to 1%B over 1min.) with a flow rate of 300 nl min−1.  Solvent A was 0.1% formic acid and Solvent B was aqueous 80% acetonitrile in 0.1% formic acid.  Peptides were ionized by nano-electrospray ionization at 2.0kV using a stainless-steel emitter with an internal diameter of 30 μm (Thermo Scientific) and a capillary temperature of 300°C.  All spectra were acquired using an Orbitrap Fusion Lumos mass spectrometer controlled by Xcalibur 3.0 software (Thermo Scientific) and operated in data-dependent acquisition mode using an SPS-MS3 workflow.  FTMS1 spectra were collected at a resolution of 120 000, with an automatic gain control (AGC) target of 400 000 and a max injection time of 100ms. Precursors were filtered with an intensity threshold of 5000, according to charge state (to include charge states 2-7) and with monoisotopic peak determination set to Peptide. Previously interrogated precursors were excluded using a dynamic window (60s +/-10ppm). The MS2 precursors were isolated with a quadrupole isolation window of 0.7m/z. ITMS2 spectra were collected with an AGC target of 10 000, max injection time of 70ms and CID collision energy of 35%. For FTMS3 analysis, the Orbitrap was operated at 30 000 resolution with an AGC target of 50 000 and a max injection time of 105ms.  Precursors were fragmented by high energy collision dissociation (HCD) at a normalised collision energy of 60% to ensure maximal TMT reporter ion yield.  Synchronous Precursor Selection (SPS) was enabled to include up to 5 MS2 fragment ions in the FTMS3 scan.

### Data Protocol
The raw data files were processed using Proteome Discoverer software v2.1 (Thermo Scientific) and searched against a bespoke human database with the fasta headers amended to contain ENSG, ENST and ENSP notation of target human protein concatenated with a custom ChAdOx1 nCoV-19 protein database consisting of a list of “known proteins” predicted to exist by homology with known adenovirus proteins alongside the “proteins not known” list predicted by the transcriptome analysis pipeline described above (Additional file 3). Searches against this database and against an in-house ‘common contaminants’ database were performed using the SEQUEST HT algorithm.  Peptide precursor mass tolerance was set at 10 ppm, and MS/MS tolerance was set at 0.6 Da.  Search criteria included oxidation of methionine (+15.995 Da), acetylation of the protein N-terminus (+42.011 Da) and methionine loss plus acetylation of the protein N-terminus (-89.03 Da) as variable modifications and carbamidomethylation of cysteine (+57.021Da) and the addition of the TMT mass tag (+229.163) to peptide N-termini and lysine as fixed modifications. For the phospho-proteome analysis, phosphorylation of serine, threonine and tyrosine (+79.966 Da) was also included as a variable modification. Searches were performed with full tryptic digestion and a maximum of 2 missed cleavages were allowed.  The reverse database search option was enabled and all data was filtered to satisfy a false discovery rate (FDR) of 1%. Quantitative changes in phosphopeptide abundance were normalised with respect to total protein abundance.

### Publication Abstract
None

### Keywords
Human, Adenovirus, Chadox ncov-19

### Affiliations
University of Bristol

### Submitter
David Matthews

### Lab Head
Dr David Matthews
University of Bristol


