### Accession
PXD000136

### Title
Proteomic analysis of subchloroplastic compartments

### Description
Arabidopsis thaliana chlorosplasts extraction followed by subchloroplastic compartments fractionation (envelope, stroma, thylakoids). Afer SDS-PAGE separation and trypsin digestion, analysis of all gel bands twice by LC-MS/MS on a LTQ-FT

### Sample Protocol
Arabidopsis plants, Wassilewskija background (Ws), were grown in culture chambers at 23 °C (12-h light cycle) with a light intensity of 150 μmol·m−2·s−1 in standard conditions. Percoll-purified chloroplasts were obtained from 100–200 g of A. thaliana leaves. Purified intact chloroplasts were then lysed in hypotonic medium, and envelope, thylakoid, and stroma subfractions were purified on a sucrose gradient. Samples were separated by SDS-PAGE in the conventional separating mode. After Coomassie Blue staining to reveal proteins, the gel was cut into discrete bands. In-gel digestion with trypsin (sequencing grade; Promega, Madison, WI) was carried out. Approximately 500 ng of digested sample was injected for each LC-MS/MS analysis performed on a 7-tesla hybrid linear ion trap Fourier transform mass spectrometer (LTQ-FT, Thermo, Bremen, Germany). The experimental sequence consisted of one high resolution MS acquisition in the ICR cell and three MS/MS scans in the linear ion trap in parallel with the MS acquisition. Dynamic exclusion was activated for ions within 5 ppm of a selected peak and eluting in a 3-min window, and one repeat scan was allowed within 30 s. An “Ultimate 3000” nano-HPLC system (LC Packings, Amsterdam, The Netherlands) equipped with a dynamic flow control and a PepMap (LC Packings) column (15 cm, 75-μm diameter, 3-μm C18 particles, 100-Å pore size) was coupled to the FT-ICR instrument.

### Data Protocol
The LTQ-FT raw data were converted into mgf files using Mascot Distiller and Mascot Daemon (Matrix Science) for further database searching. Database searching was carried out using the Mascot 2.1 program (Matrix Science). Two databases were compiled: a home-made list of well known contaminants and an updated compilation of the A. thaliana protein database provided by TAIR (v6.0; July 9, 2006; 30,899 entries). The variable modifications allowed were acetyl (protein), methionine oxidation, methionine sulfone, and cysteic acid. One missed trypsin cleavage was allowed, and trypsin/P was used as the enzyme. The mass tolerances were 10 ppm for precursor ions and 0.8 Da for fragment ions.

### Publication Abstract
Advances in high-throughput proteomics have led to a rapid increase in the number, size, and complexity of the associated data sets. Managing and extracting reliable information from such large series of data sets require the use of dedicated software organized in a consistent pipeline to reduce, validate, exploit, and ultimately export data. The compilation of multiple mass-spectrometry-based identification and quantification results obtained in the context of a large-scale project represents a real challenge for developers of bioinformatics solutions. In response to this challenge, we developed a dedicated software suite called hEIDI to manage and combine both identifications and semiquantitative data related to multiple LC-MS/MS analyses. This paper describes how, through a user-friendly interface, hEIDI can be used to compile analyses and retrieve lists of nonredundant protein groups. Moreover, hEIDI allows direct comparison of series of analyses, on the basis of protein groups, while ensuring consistent protein inference and also computing spectral counts. hEIDI ensures that validated results are compliant with MIAPE guidelines as all information related to samples and results is stored in appropriate databases. Thanks to the database structure, validated results generated within hEIDI can be easily exported in the PRIDE XML format for subsequent publication. hEIDI can be downloaded from http://biodev.extra.cea.fr/docs/heidi .

### Keywords
Ltq-ft, Lc-ms/ms, Sub compartments fractionation, Chloroplast

### Affiliations
Laboratoire d'Etude de la Dynamique des ProtÃ©omes
CEA Grenoble, Commissariat à l'Energie Atomique et aux Energies Alternatives

### Submitter
Hesse Anne-Marie

### Lab Head
Dr Christophe Bruley
CEA Grenoble, Commissariat à l'Energie Atomique et aux Energies Alternatives


