### Accession
PXD026699

### Title
Synaptotagmin 13 drives pancreatic endocrine cell egression and islet morphogenesis: BioID

### Description
Identification of protein intreactions for the synaptotagmin 13 protein from MDCK cells by BioID-based proximity labelling and LFQ mass spectrometry.  Epithelial cell egression is important for organ development and cell differentiation, but also drives cancer metastasis. The tightly connected pancreatic epithelial differentiation and morphogenesis generate islets of Langerhans. However, the morphogenetic drivers and molecular mechanisms are largely unresolved. Here we identify the uncharacterized Synaptotagmin 13 (Syt13) as a major regulator of endocrine cell egression and islet morphogenesis and differentiation. We detected upregulation of Syt13 in endocrine precursors that associates with increased expression of several unique cytoskeletal components. High-resolution imaging reveals a previously unidentified apical-basal to front-rear repolarization during endocrine cell egression. Strikingly, Syt13 directly interacts with acetylated tubulin and phosphoinositide phospholipids to be recruited to the leading edge of egressing cells. Knockout of Syt13 discloses the impairment in endocrine cell egression and skews the α-to-β-cell ratio. At mechanistic levels, Syt13 regulates protein endocytosis to remodel the basement membrane and modulate cell-matrix adhesion at the leading edge of egressing endocrine cells. Altogether, these findings implicate that Ca2+-independent atypical Syt13 vesicular protein functions in regulating cell polarity to orchestrate endocrine cell egression and tissue morphogenesis.

### Sample Protocol
1 x 10^6 MDCK cells stably expressing Syt13 or Syt13-BirA recombinant proteins were cultured in 14 cm2 dishes in grocontrolh medium for 3 days at 37 °C. The medium was replaced with a fresh grocontrolh medium containing 50 µM biotin and after 16 h each dish was lysed with 1 ml lysis buffer (50 mM Tris buffer pH 7.5, 0.5% Sodiumdesoxycolate, 150 mM NaCl, 1% NP40, 0.1% SDS) and kept on ice. In parallel, 240 µL Pierce™ High Capacity NeutrAvidin™ Agarose beads (ThermoFisher Scientific, Cat: 29202) were washed 3x with the lysis buffer and resuspended in 1 mL lysis buffer. Next, the cell lysates were centrifuged for 30 min at 14000 rpm at 4 °C and 3 mL supernatant for each condition was collected. 90 µL of the samples were stored as the input samples and the rest were mixed with 300 µL beads-lysis buffer in the 15 mL falcon tubes rotating for 3 h at RT. Next, samples were centrifuged for 5 min at 500 rpm at 4 °C. Supernatants were discarded and the beads were used for western blotting or mass spectrometry as follows.   Affinity purified eluates were precipitated with chloroform and methanol followed by trypsin digestion as described before (Gloeckner et al., 2009). LC-MS/MS analysis was performed on Ultimate3000 nanoRSLC systems (Thermo Scientific) coupled to an Orbitrap Fusion Tribrid mass spectrometer (Thermo Scientific) by a nanospray ion source. Tryptic peptide mixtures were injected automatically and loaded at a flow rate of 30 μl/min in 0.1% trifluoroacetic acid in HPLC-grade water onto a nano trap column (300 μm i.d. × 5 mm Pre column, packed with Acclaim PepMap100 C18, 5 μm, 100 Å; Thermo Scientific). After 3 minutes, peptides were eluted and separated on the analytical column (75 μm i.d. × 25 cm, Acclaim PepMap RSLC C18, 2 μm, 100 Å; Thermo Scientific) by a linear gradient from 2% to 30% of buffer B (80% acetonitrile and 0.08% formic acid in HPLC-grade water) in buffer A (2% acetonitrile and 0.1% formic acid in HPLC-grade water) at a flow rate of 300 nl/min over 117 minutes. Remaining peptides were eluted by a short gradient from 30% to 95% buffer B in 5 minutes. Analysis of the eluted peptides was done on an LTQ Fusion mass spectrometer. From the high resolution MS pre-scan with a mass range of 335 to 1500, the most intense peptide ions were selected for fragment analysis in the orbitrap depending by using a high speed method if they were at least doubly charged. The normalized collision energy for HCD was set to a value of 27 and the resulting fragments were detected with a resolution of 120,000. The lock mass option was activated; the background signal with a mass of 445.12003 was used as lock mass (Olsen et al., 2005). Every ion selected for fragmentation was excluded for 20 seconds by dynamic exclusion.

### Data Protocol
MS/MS data were analyzed using the MaxQuant software (version 1.6.1.0) (Cox and Mann, 2008; Cox et al., 2009). As a digesting enzyme, Trypsin/P was selected with maximal 2 missed cleavages. Cysteine carbamidomethylation was set for fixed modifications, and oxidation of methionine and N-terminal acetylation were specified as variable modifications. The data were analyzed by label-free quantification with the minimum ratio count of 3. The first search peptide tolerance was set to 20, the main search peptide tolerance to 4.5 ppm and the re-quantify option was selected. For peptide and protein identification the human subset of the SwissProt database (release 2014_04) was used and contaminants were detected using the MaxQuant contaminant search. A minimum peptide number of 2 and a minimum length of 7 amino acids was tolerated. Unique and razor peptides were used for quantification. The match between run option was enabled with a match time window of 0.7 min and an alignment time window of 20 min. The statistical analysis including ratio, t-test and significance A calculation was done using the Perseus software (version 1.6.2.3) (Tyanova et al., 2016).

### Publication Abstract
None

### Keywords
Bioid, Proximity labelling, Synaptotagmin 13, Protein interactions

### Affiliations
Medical Bioanalytics Institute for Ophthalmic Research Eberhard-Karls University of Tuebingen
Medical Bioanalytics

### Submitter
Karsten Boldt

### Lab Head
Dr Dr. Karsten Boldt
Medical Bioanalytics Institute for Ophthalmic Research Eberhard-Karls University of Tuebingen


