### Accession
PXD040281

### Title
A standardized and reproducible workflow for membrane glass slides in routine histology and spatial proteomics

### Description
Defining the molecular phenotype of single cells in-situ is essential for understanding tissue heterogeneity in health and disease. Powerful imaging technologies have recently been joined by spatial omics technologies, promising unparalleled insights into the molecular landscape of biological samples. One approach involves laser microdissection in combination with membrane glass slides for the isolation of single cells from specific anatomical regions for further analysis by spatial omics. However, so far this is not fully compatible with automated staining platforms and routine histology procedures such as heat-induced epitope retrieval, limiting reproducibility, throughput and integration of advanced staining procedures. This study describes a robust workflow for routine use of glass membrane slides, allowing precise extraction of tissue in combination with automated and multicolor immunofluorescence staining. The key advance is the addition of glycerol to standard heat-induced epitope retrieval protocol, preventing membrane distortion while preserving antigen retrieval properties. Importantly, we show that glycerol is fully compatible with mass-spectrometry based proteomics and does not affect proteome depth or quality. Further, we enable single focal plane imaging by removing remaining trapped air pockets with an incision. We demonstrate our workflow using the recently introduced Deep Visual Proteomics technology on the single-cell type analysis of adjacent suprabasal and basal keratinocytes of human skin. Our protocol extends the utility of membrane glass slides and enables much more robust integration with routine histology procedures, high-throughput multiplexed imaging and sophisticated downstream spatial omics technologies.

### Sample Protocol
Formalin fixed and paraffin embedded (FFPE) human tissue samples were collected according to standard operating procedures. Tissue was mounted on 2 ï�­m PEN membrane slides (MicroDissect GmbH, MDG3P40AK) for further processing.  For bulk proteomics, antigen retrieval was performed using pH6 citrate buffer (Tri-sodium citrate dihydrate), Target Retrieval Solution (TRS) pH9 (Dako, S2367) and TRS pH6 (Dako, S2369) with or without 10% Glycerol (Sigma Aldrich/Merck, Darmstadt, Germany, G7757) and heated for 60 minutes at 90Â°C. Subsequently, tissue was stained on an Omnis instrument (Dako, Glostrup, Denmark) for anti-CK5 (1:200, Leica Biosystems, clone XM26) by immunohistochemistry. Tonsil tissue was collected using laser microdissection and resulting samples were lysed in 300 mM Tris/HCl pH 8.0, 12.5% acetonitrile including 5 mM TCEP and 20 mM CAA, followed by focused ultrasonication (Covaris, AFAÂ®) and repeated heating. Sample were further processed as described previously (Coscia and Doll et al., Journal of Pathology, 2020). MS data was acquired by an EASY nanoLC 1200 (Thermo Fisher Scientific) coupled to a timsTOF Pro2 mass spectrometer (Bruker Daltonics) with a nano-electrospray ion source (CaptiveSpray, Bruker Daltonics). 200 ng of peptides were loaded on a 50 cm in-house packed HPLC column (75 Î¼m inner diameter packed with 1.9 Î¼m ReproSil-Pur C18-AQ silica beads, Dr. Maisch GmbH, Germany). Sample analytes were separated using a linear 120 min gradient from 3 to 30% buffer B in 95 min, followed by an increase to 60% for 5 min and to 95% buffer B for 5min, as well as a 5 min wash at 95% buffer B and re-equilibration for 5 min at 5% buffer B (buffer A: 0.1% FA/ddH2O; buffer B: 0.1% FA, 80% ACN, and 19.9% ddH2O). The flow rate was kept constant at 300 nl/min. The mass spectrometer was operated in dda-PASEF mode as previously described (Brunner et al., Mol Sys Biol, 2022). In summary, one MS1 scan was followed by ten PASEF MS/MS scans per acquisition cycle. The ion accumulation and ramp time in the dual TIMS analyzer was 100 ms and ion mobility range was set from 1/K0â€‰=â€‰1.6â€‰Vsâ€‰cmâˆ’2 to 0.6â€‰Vsâ€‰cmâˆ’2. Single charged precursor ions were excluded with a polygon filter (timsControl, version 3.0.20.0, Bruker Daltonics) and precursors for MS/MS were picked at an intensity threshold of 2,500 arbitrary units (a.u.) and re-sequenced until reaching a target value of 20,000 a.u. considering a dynamic exclusion of 40 s elution. Deep Visual Proteomics was performed as described recently (Coscia and Mund et al., Nature Biotechnology, 2022). In brief, FFPE tissue of healthy skin tissue was deparaffinized and Glycerol-assisted antigen-retrieval was performed using preheated 1x TRS pH9 HIER buffer (DAKO, S2367) / 10% glycerol (v/v) (Sigma, G7757). Subsequent to sample blocking, primary antibodies targeting pan-CK (rabbit, DAKO Z0622, dilution 1:100) and KRT10 (mouse, abcam ab76318, dilution 1:800) were applied. An incubation with secondary antibodies against mouse-IgG (A647, Invitrogen A32728, 1:400) and rabbit-IgG (A555, Invitrogen A32732; 1:400) was followed by SYTOX green Nucleic Acid Stain (Invitrogen S7020; 1:700 in ddH20). Slides were washed twice in ddH2O, before puncturing the membrane at the proximal end of the slide using a needle (30G). Fluorescence images were acquired on a Zeiss Axioscan Z7 followed by Artificial intelligence (AI)-guided cell recognition and classification using the Biology Image Analysis Software (BIAS, Cell Signaling). 700 contours of each group were excised in quadruplicates on a LMD7 (Leica Microsystems) and prepared for MS-based proteomics in 60mM TEAB, 12% (v/v) acetonitrile and an overnight digest in 4 ng LysC and 6 ng trypsin. For the high-sensitivity, deep-visual proteomic samples, samples were measured with the Evosep One LC system (EvoSep) coupled to a timsTOF SCP mass spectrometer (Bruker) employing a nano-electrospray ion source (Bruker Daltonics). The Whisper 20 SPD (samples per day) method was used with the Aurora Elite CSI third generation column with 15 cm and 75 Î¼m ID (AUR3-15075C18-CSI, IonOpticks) at 50 Â°C. The mobile phases comprised 0.1% FA in LC-MS grade water as buffer A and 99.9% ACN/0.1% FA as buffer B. The timsTOF was operated in dia-PASEF mode with variable window widths. Optimal dia-PASEF methods cover the precursor cloud highly efficient in the m/z â€“ ion mobility (IM) plane while providing deep proteome coverage. For method generation with py_diAID, the precursor density distribution in m/z and IM was estimated based on a tryptic 48 high-pH fraction library (Skowronek, Mol Cell Protoemics, 2022). We calculated the optimal cycle time based on the chromatographic peak width of 5 ng HeLa single runs. The optimal dia-PASEF method consisted of one MS1 scan followed by twelve dia-PASEF scans with two IM ramps per dia-PASEF scan, covering a m/z range from 300 to 1200 and IM of 0.7 to 1.3 Vs cm-2.

### Data Protocol
Data acquired in dda-PASEF mode were analyzed with MaxQuant (2.0.1.0) using standard settings in reference to UniProt human databases (UP000005640_9606.fasta and UP000005640_9606_additional.fasta). Oxidation (M) and Acetyl (Protein-N-term) was set as variable and Carbamidomethyl (C) as fixed modification. The maximum number of modifications per peptide was set to 3. Trypsin/P was selected as protease and the maximum number of missed cleavages was set to 2. MS data acquired in dia-PASEF mode were processed in the DIA-NN software (version 1.8.1, [13]) using the same fasta files and standard settings. Search parameters deviated from the standard settings as follows: Peptide length range 7-55, Precursor m/z range 100-1700, Fragment ion m/z range 100-1700, Quantification strategy â€˜any LC (high accuracy)â€™, cross-run normalization â€˜globalâ€™, and MBR (â€˜match between runsâ€™) enabled. The report.pg_matrix output file of DIA-NN was used for further data analysis.

### Publication Abstract
Defining the molecular phenotype of single cells in situ is key for understanding tissue architecture in health and disease. Advanced imaging platforms have recently been joined by spatial omics technologies, promising unparalleled insights into the molecular landscape of biological samples. Furthermore, high-precision laser microdissection (LMD) of tissue on membrane glass slides is a powerful method for spatial omics technologies and single-cell type spatial proteomics in particular. However, current histology protocols have not been compatible with glass membrane slides and LMD for automated staining platforms and routine histology procedures. This has prevented the combination of advanced staining procedures with LMD. In this study, we describe a novel method for handling glass membrane slides that enables automated eight-color multiplexed immunofluorescence staining and high-quality imaging followed by precise laser-guided extraction of single cells. The key advance is the glycerol-based modification of heat-induced epitope retrieval protocols, termed "G-HIER." We find that this altered antigen-retrieval solution prevents membrane distortion. Importantly, G-HIER is fully compatible with current antigen retrieval workflows and mass spectrometry-based proteomics and does not affect proteome depth or quality. To demonstrate the versatility of G-HIER for spatial proteomics, we apply the recently introduced deep visual proteomics technology to perform single-cell type analysis of adjacent suprabasal and basal keratinocytes of human skin. G-HIER overcomes previous incompatibility of standard and advanced staining protocols with membrane glass slides and enables robust integration with routine histology procedures, high-throughput multiplexed imaging, and sophisticated downstream spatial omics technologies.

### Keywords
Heat-induced epitope retrieval, Membrane slides, Antigen retrieval, Spatial proteomics, Laser microdissection, Proteomics, Glycerol, Deep visual proteomics, Histology

### Affiliations
Proteomics
Proteomics and Signal Transduction, Max Planck Institute of Biochemistry, Martinsried, Germany

### Submitter
Mario Oroshi

### Lab Head
Dr Matthias Mann
Proteomics and Signal Transduction, Max Planck Institute of Biochemistry, Martinsried, Germany


