### Accession
PXD009374

### Title
Quantitative phosphoproteomic for CDKL5 substrates

### Description
Global phosphoproteomic screen to identify the first cellular substrates of CDKL5. CDKL5 knock-out U2OS cells and CDKL5 wt U2OS cells were generated for the TMT-based phosphoproteomic. Thi leads to the identification and further validation of several phosphopetides of MAP1S, CEP131 and CDKL5 itself. The phosphoproteomic analysis allowed the identification of the first cellular substrates for CDKL5 kinase.

### Sample Protocol
Three biological replicates for both CDKL5 knockout U2OS cells (clone 13) and the same cells in which CDKL5 was reintroduced were lysed in 8M Urea, 50mM triethylammonium bicarbonate (TEAB) (pH 8.5), sonicated and protein quantification was determined using the BCA Protein Assay Kit (Pierce Protein). Approximately 5 mg of protein extract for each sample was reduced by adding 5 mM Tris(2- carboxyethyl)phosphine (TCEP) for 30 min at room temperature and cysteines were alkylated by adding 20 mM iodoacetamide for 30 min in the dark. Samples were then diluted to 1M Urea and digested overnight at 37 °C by adding porcine trypsin (1:50, w/w) (Pierce). Peptides were desalted and concentrated via C18 SPE on Sep-Pak cartridges (Waters). Phosphopeptide enrichment was similar to previously described (Lai, Kondapalli et al., 2015, Larsen, Thingholm et al., 2005, Trost, English et al., 2009, Trost, Sauvageau et al., 2012). Tryptic peptides (5 mg per TMT channel) were resuspended in 1 mL of 2M Lactic acid/50% (v/v) acetonitrile and centrifuged 15,000 x g for 20 min. Supernatants were placed in an Eppendorf Lobind Tube containing 24 mg of titanium dioxide beads (GL Sciences) andvortexed for 1h at room temperature. Beads were washed twice with 2M lactic acid/50% (v/v)acetonitrile and once with 0.1% (v/v) TFA in 50% (v/v) acetonitrile. Phosphopeptides were eluted twice with 150 μL of 60 mMaAmmonium hydroxide (pH > 10.5), then acidified with 40 μL of 20% (v/v) formic acid and desalted via C18 Macro SpinColumns (Harvard Apparatus). Isobaric labelling of phosphorylated peptides was performed using the 6-plex tandem mass tag (TMT) reagents (Thermo Fisher). TMT reagents (0.8 mg) were dissolved in 41 μL of acetonitrile and 20 μL was added to the corresponding fractions, previously dissolved in 50 μL of 50 mM TEAB, pH 8.5. The reaction was quenched by addition of 4 μL of 5% hydroxylamine after 1 h incubation at room temperature. Labelled peptides were combined, acidified with 200 μL of 1% TFA (pH ~ 2) and concentrated using C18 SPE on Sep-Pak cartridges (Waters). Labelling efficiency was tested for each TMT labelled sample and was >95%. TMT labelled phosphorylated peptides were combined at equal amounts for each channel and subjected to HPRP. Labelled peptides were solubilized in 20 mM ammonium formate (pH 8.0) and separated on a Gemini C18 column (250 × 3 mm, 3μm C18 110 Å pore size) (Phenomenex). Using a DGP-3600BM pump system equipped with a SRD-3600 degasser (Thermo Fisher) a 40 min gradient length from 1% to 90% acetonitrile (flow rate of 0.25 mL/min), separated the peptide mixtures into a total of 40 fractions. The 40 fractions were merged into 12 samples, acidified with 1% (v/v) TFA (pH ~ 2), desalted via C18 Macro SpinColumns (Harvard Apparatus), dried under vacuum centrifugation and re-suspended in 2% (v/v) ACN / 0.1% (v/v) TFA for LC-MS/MS analysis using HCD MS2 fragmention on the Orbitrap Fusion (HCD-IT MS2).

### Data Protocol
Protein identification and TMT quantification were performed using MaxQuant Version 1.5.1.7 (Cox & Mann, 2008). Trypsin/P was set as protease; stable modification: carbamidomethyl (C); variable modifications: oxidation (M), acetyl (protein N-term), phospho (STY); maximum 8 modifications per peptide and 2 missed cleavages. Searches were conducted using a combined Uniprot-Trembl homo sapiens database with isoforms downloaded July 15, 2015 plus common contaminants (42,095 sequences). Identifications were filtered at a 1% FDR at the peptide level, accepting a minimum peptide length of 5. TMT intensities were extracted, normalized for each condition, and were used for downstream analyses in Perseus 1.5.3.1 (Cox & Mann, 2012). A student’s t-test (two-tailed, homoscedastic) was performed on the normalized TMT intensities and phosphopeptides with p<0.05 and a fold-change >1.5 were considered significantly altered in abundance between the samples

### Publication Abstract
Mutations in the gene encoding the protein kinase CDKL5 cause a debilitating neurodevelopmental disease termed CDKL5 disorder. The impact of these mutations on CDKL5 function is poorly understood because the substrates and cellular processes controlled by CDKL5 are unclear. Here, we describe a quantitative phosphoproteomic screening which identified MAP1S, CEP131 and DLG5-regulators of microtubule and centrosome function-as cellular substrates of CDKL5. Antibodies against MAP1S phospho-Ser<sup>900</sup> and CEP131 phospho-Ser<sup>35</sup> confirmed CDKL5-dependent phosphorylation of these targets in human cells. The phospho-acceptor serine residues in MAP1S, CEP131 and DLG5 lie in the motif RPX<u>S</u>A, although CDKL5 can tolerate residues other than Ala immediately C-terminal to the phospho-acceptor serine. We provide insight into the control of CDKL5 activity and show that pathogenic mutations in CDKL5 cause a major reduction in CDKL5 activity <i>in&#xa0;vitro</i> and in cells. These data reveal the first cellular substrates of CDKL5, which may represent important biomarkers in the diagnosis and treatment of CDKL5 disorder, and illuminate the functions of this poorly characterized kinase.

### Keywords
Tmt, Cdkl5 substrates, Phosphoproteomic, Ms2

### Affiliations
Newcastle university, ICAMB Newcastle Upon tyne, United Kingdom
Newcastle University

### Submitter
Matthias Trost

### Lab Head
Dr Matthias Trost
Newcastle university, ICAMB Newcastle Upon tyne, United Kingdom


