### Accession
PXD016278

### Title
Proteome Profiling in Cerebrospinal Fluid Reveals Novel Biomarkers of Alzheimerâ€™s Disease

### Description
Neurodegenerative diseases are a growing burden and there is an urgent need for better biomarkers for diagnosis, prognosis and treatment efficacy. Structural and functional brain alterations are reflected in the protein composition of cerebrospinal fluid (CSF). Alzheimerâ€™s disease (AD) patients have higher CSF levels of tau, but we lack knowledge of systems-wide changes of CSF protein levels that accompany AD. Here we present a highly reproducible mass spectrometry (MS)-based proteomics workflow for the in-depth analysis of CSF from minimal sample amounts. From three independent studies (>200 individuals), we characterize differences in proteins by AD status (>1,000 proteins, CV<20%). Proteins with previous links to neurodegeneration such as tau, SOD1 and PARK7 differed most strongly by AD status, providing strong positive controls for our approach. CSF proteome changes in Alzheimerâ€™s disease prove to be widespread and often correlated with tau concentrations. Our unbiased screen also reveals a consistent glycolytic signature across our and a recent report (bioRxiv.org doi.org/10.1101/806752). Successful reclassification of individual patients and a machine learning model suggests clinical utility of our proteomic signature.

### Sample Protocol
CSF was aliquoted in a 96-well plate and processed with an automated set-up on an Agilent Bravo liquid handling platform. In total 40 Âµl of CSF were mixed with 40 Âµl PreOmics lysis buffer (PreOmics GmbH) for reduction of disulfide bridges, cysteine alkylation and protein denaturation at 95Â°C for 10 min. After a 10 min cooling step, 0.2Âµg trypsin and 0.2Âµg LysC were added to each sample and digestion was performed at 37Â°C for 4 hours. Peptides were purified on two 14-gauge StageTip plugs according to the PreOmics iST protocol (www.preomics.com). The StageTips were centrifuged using an in-house 3D-printed StageTip tray at 1500g for washing and elution. The eluate was completely dried using a SpeedVac centrifuge at 45Â°C (Eppendorf, Concentrator plus), resuspended in 10Âµl buffer A* (2% acetonitrile (v/v), 0.1% trifluoroacetic acid (v/v) and stored at -20Â°C.  Samples were measured using an EASY-nLC 1200 (Thermo Fisher Scientific) coupled to a Q Exactive HF-X Orbitrap mass spectrometer (Thermo Fisher Scientific) via a nano-electrospray ion source (Thermo Fisher Scientific). Purified peptides were separated on 50cm UHPLC columns with an inner diameter of 75Âµm packed in-house with ReproSil-Pur C18-AQ 1.9Âµm resin (Dr.Maisch GmbH). In total 500ng of purified peptide solution in buffer A* were loaded onto the column in buffer A (0.1% v/v formic acid) and eluted at a flow rate of 300 nl/min and a temperature of 60Â°C by a linear 80 min gradient from 5% to 30% buffer B (0.1% v/v formic acid, 80% v/v acetonitrile), followed by a 4 min increase to 60% B, a further 4min increase to 95% B, a 4 min plateau phase at 95% B, a 4 min decrease to 5% B and a 4 min wash phase of 5% B. To acquire MS data, the data-independent acquisition (DIA) scan mode was used for single-shot patient samples, whereas the fractionated samples of the AD pool and non-AD pool were acquired with a top12 data-dependent acquisition (DDA) scan mode. Both acquisition schemes were combined with the same liquid chromatography gradient. The mass spectrometer was operated by the XCalibur software (Thermo Fisher). DDA scan settings on full MS level included an ion target value of 3 x 106 charges in the 300 â€“ 1650 m/z range with a maximum injection time of 20 ms and a resolution of 60,000 at m/z 200. At the MS/MS level the target value was 105 charges with a maximum injection time of 60 ms and a resolution of 15,000 at m/z 200. For MS/MS events only, precursor ions with 2-5 charges that were not on the 20 s dynamic exclusion list were isolated in a 1.4 m/z window. Fragmentation was performed by higher-energy C-trap dissociation (HCD) with a normalized collision energy of 27eV. DIA was performed with one full MS event followed by 33 MS/MS windows in one cycle resulting in a cycle time of 2.7 seconds. The full MS settings included an ion target value of 3 x 106 charges in the 300 â€“ 1650 m/z range with a maximum injection time of 60 ms and a resolution of 120,000 at m/z 200. DIA precursor windows ranged from 300.5 m/z (lower boundary of first window) to 1649.5 m/z (upper boundary of 33rd window). MS/MS settings included an ion target value of 3 x 106 charges for the precursor window with an Xcalibur-automated maximum injection time and a resolution of 30,000 at m/z 200.

### Data Protocol
The MS data of the fractionated pools (DDA MS data, 24 AD fractions, 24 non-AD fractions ) and the single shot subject samples (DIA MS data, all samples from all three cohorts) were used to generate a DDA-library and direct-DIA-library, respectively, which were computationally merged into a hybrid library in the Spectromine software, version 1.0.21621.8.15296 (Biognosys AG, Schlieren, Switzerland). The hybrid spectral library was subsequently used to search the MS data of the single shot patient samples in the Spectronaut software, version 12.0.20491.9.26669 (Biognosys AG), for final protein identification and quantitation. All searches were performed against the human UnipProt reference proteome of canonical and isoform sequences with 93,786 entries downloaded in March 2018. Searches used carbamidomethylation as fixed modification and acetylation of the protein N-terminus, oxidation of methionines and deamidation of asparagine or glutamine as variable modifications. Default settings were used for other parameters. In brief, a trypsin/P proteolytic cleavage rule was used, permitting a maximum of two miscleavages and a peptide length of 7-52 amino acids. Spectral library generation stipulated a minimum of three fragments per peptide and maximally the six best fragments were included. A protein and precursor FDR of 1% were used and protein quantities were reported in samples only if the protein passed the filter (q-value mode data filtering).

### Publication Abstract
Neurodegenerative diseases are a growing burden, and there is an urgent need for better biomarkers for diagnosis, prognosis, and treatment efficacy. Structural and functional brain alterations are reflected in the protein composition of cerebrospinal fluid (CSF). Alzheimer's disease (AD) patients have higher CSF levels of tau, but we lack knowledge of systems-wide changes of CSF protein levels that accompany AD. Here, we present a highly reproducible mass spectrometry (MS)-based proteomics workflow for the in-depth analysis of CSF from minimal sample amounts. From three independent studies (197 individuals), we characterize differences in proteins by AD status (&gt;&#xa0;1,000 proteins, CV&#xa0;&lt;&#xa0;20%). Proteins with previous links to neurodegeneration such as tau, SOD1, and PARK7 differed most strongly by AD status, providing strong positive controls for our approach. CSF proteome changes in Alzheimer's disease prove to be widespread and often correlated with tau concentrations. Our unbiased screen also reveals a consistent glycolytic signature across our cohorts and a recent study. Machine learning suggests clinical utility of this proteomic signature.

### Keywords
Alzheimerâ€™s disease/ cerebrospinal fluid/ mass spectrometry/ neurodegeneration/ proteomics

### Affiliations
Proteomics
Department of Proteomics and Signal Transduction, Max Planck Institute of Biochemistry, Martinsried, Germany  NNF Center for Protein Research, Faculty of Health Sciences, University of Copenhagen, Copenhagen, Denmark

### Submitter
Mario Oroshi

### Lab Head
Dr Matthias Mann
Department of Proteomics and Signal Transduction, Max Planck Institute of Biochemistry, Martinsried, Germany  NNF Center for Protein Research, Faculty of Health Sciences, University of Copenhagen, Copenhagen, Denmark


