### Accession
PXD006118

### Title
Quantification of mock microbial communities with metagenomes, 16S rRNA gene amplicons and metaproteomics

### Description
In this study we developed metaproteomics based methods for quantifying taxonomic composition of microbiomes (microbial communities). We also compared metaproteomics based quantification to other quantification methods, namely metagenomics and 16S rRNA gene amplicon sequencing. The metagenomic and 16S rRNA data can be found in the European Nucleotide Archive (Study number: PRJEB19901). For the method development and comparison of the methods we analyzed three types of mock communities with all three methods. The communities contain between 28 to 32 species and strains of bacteria, archaea, eukaryotes and bacteriophage. For each community type 4 biological replicate communities were generated. All four replicates were analyzed by 16S rRNA sequencing and metaproteomics. Three replicates of each community type were analyzed with metagenomics. The "C" type communities have same cell/phage particle number for all community members (C1 to C4). The "P" type communities have the same protein content for all community members (P1 to P4). The "U" (UNEVEN) type communities cover a large range of protein amounts and cell numbers (U1 to U4). We also generated proteomic data for four pure cultures to test the specificity of the protein inference method. This data is also included in this submission.

### Sample Protocol
Samples were lysed in SDT-lysis buffer (4% (w/v) SDS, 100 mM Tris-HCl pH 7.6, 0.1 M DTT). SDT lysis buffer was added in a 1:10 sample/buffer ratio to the sample pellets. Cells were disrupted in lysing matrix tubes A (MP Biomedicals, Cat. No. 116910050) for 45 seconds at 6 m/s using a OMNI bead ruptor bead mill (Omni International, https://www.omni-inc.com/) and subsequently incubated at 95° C for 10 minutes followed by pelleting of debris for 5 min at 21,000 x g. We prepared tryptic digests following the filter-aided sample preparation (FASP) protocol described by Wisniewski et al.  (2009, Nat. Meth. 6(5)). In brief, 30 µl of the cleared lysate were mixed with 200 µl of UA solution (8 M urea in 0.1 M Tris/HCl pH 8.5) in a 10 kDa MWCO 500 µl centrifugal filter unit (VWR International) and centrifuged at 14,000 x g for 40 min. 200 µl of UA solution were added again and centrifugal filter spun at 14,000 x g for 40 min. 100 µl of IAA solution (0.05 M iodoacetamide in UA solution) were added to the filter and incubated at 22° C for 20 min. The IAA solution was removed by centrifugation and the filter was washed three times by adding 100 µl of UA solution and then centrifuging. The buffer on the filter was then changed to ABC (50 mM Ammonium Bicarbonate), by washing the filter three times with 100 µl of ABC. 1 to 2 µg of MS grade trypsin (Thermo Scientific Pierce, Rockford, IL, USA) in 40 µl of ABC were added to the filter and filters incubated over night in a wet chamber at 37° C. The next day, peptides were eluted by centrifugation at 14,000 x g for 20 min, followed by addition of 50 µl of 0.5 M NaCl and again centrifugation. Peptides were desalted using Sep-Pak C18 Plus Light Cartridges (Waters, Milford, MA, USA) or C18 spin columns (Thermo Scientific Pierce, Rockford, IL, USA) according to the manufacturer’s instructions. Approximate peptide concentrations were determined using the Pierce Micro BCA assay (Thermo Scientific Pierce, Rockford, IL, USA) following the manufacturer’s instructions. Samples were analyzed by 1D-LC-MS/MS or 2D-LC-MS/MS. Two wash runs and one blank run were done between samples to reduce carry over. For the 1D-LC-MS/MS runs, 2 µg of peptide were loaded onto a 5 mm, 300 µm ID C18 Acclaim® PepMap100 pre-column (Thermo Fisher Scientific) using an UltiMateTM 3000 RSLCnano Liquid Chromatograph (Thermo Fisher Scientific). After loading, the pre-column was switched in line with a 50 cm x 75 µm analytical EASY-Spray column packed with PepMap RSLC C18, 2µm material (Thermo Fisher Scientific), which was heated to 45° C. The analytical column was connected via an Easy-Spray source to a Q Exactive Plus hybrid quadrupole-Orbitrap mass spectrometer (Thermo Fisher Scientific). Peptides were separated on the analytical column and mass spectra acquired in the Orbitrap using 260 or 460 minute gradients as described by Petersen et al. (2016, Nat. Microb. 2(16195)). For the 2D-LC-MS/MS runs, 11 µg of peptide were loaded onto a 10 cm, 300 µm ID Poros 10 S SCX column (Thermo Fisher Scientific) using the UltiMateTM 3000 RSLCnano LC. Peptides were eluted from the SCX column onto the C18 pre-column using 20 µl injection of salt plugs from the autosampler with increasing concentrations (12 salt plugs, 0 to 2000 mM NaCl). After each salt plug injection the pre-column was switched in line with the 50 cm x 75 µm analytical EASY-Spray column and peptides separated using a 120 minute gradient. Data acquisition in the Q Exactive Plus was done as described by Petersen et al. (2016, Nat. Microb. 2(16195)).

### Data Protocol
Since this is a method development study, the data processing protocols used are quite extensive. For a full account of data processing, please see the manuscript. For protein identification a database was created using all protein sequences from the reference genomes of the organisms used in the mock communities. The cRAP protein sequence database (http://www.thegpm.org/crap/) containing protein sequences of common laboratory contaminants was appended to the database. The final database contained 123,100 protein sequences. For protein identification MS/MS spectra were searched against the database using the Sequest HT node in Proteome Discoverer version 2.0.0.802 (Thermo Fisher Scientific) as described previously (Petersen et al. 2016, Nat. Microb. 2(16195)).  To test the influence of database incompleteness on quantification results we used two additional sequence databases of varying incompleteness to quantify the species in the UNEVEN community. In the first incomplete database (INCOMPLETE1) the protein sequences for Pseudomonas denitrificans, Pseudomonas fluorescens and Rhizobium leguminosarum  bv. viciae strain 3841 were removed leaving the sequences of the closely related species/strains Pseudomonas pseudoalcaligenes and Rhizobium leguminosarum  bv. viciae strain VF39 in the database. In the second incomplete database (INCOMPLETE2) the remaining Pseudomonas and Rhizobium sequences as well as the Salmonella enterica typhimurium LT2 sequences were removed.

### Publication Abstract
Measurements of stable carbon isotope ratios (&#x3b4;<sup>13</sup>C) are widely used in biology to address questions regarding food sources and metabolic pathways used by organisms. The analysis of these so-called stable isotope fingerprints (SIFs) for microbes involved in biogeochemical cycling and microbiota of plants and animals has led to major discoveries in environmental microbiology. Currently, obtaining SIFs for microbial communities is challenging as the available methods either only provide low taxonomic resolution, such as the use of lipid biomarkers, or are limited in throughput, such as nanoscale secondary ion MS imaging of single cells. Here we present "direct protein-SIF" and the Calis-p software package (https://sourceforge.net/projects/calis-p/), which enable high-throughput measurements of accurate &#x3b4;<sup>13</sup>C values for individual species within a microbial community. We benchmark the method using 20 pure culture microorganisms and show that the method reproducibly provides SIF values consistent with gold-standard bulk measurements performed with an isotope ratio mass spectrometer. Using mock community samples, we demonstrate that SIF values can also be obtained for individual species within a microbial community. Finally, a case study of an obligate bacteria-animal symbiosis shows that direct protein-SIF confirms previous physiological hypotheses and can provide unexpected insights into the symbionts' metabolism. This confirms the usefulness of this approach to accurately determine &#x3b4;<sup>13</sup>C values for different species in microbial community samples.

### Keywords
1d-lc, Metagenomics, Mock community, Metaproteomics, Biomass assessment, Metaproteome, 2d-lc, Microbiome

### Affiliations
North Carolina State University
North Carolina State University, NC, USA

### Submitter
Manuel Kleiner

### Lab Head
Dr Manuel Kleiner
North Carolina State University, NC, USA


