### Accession
PXD016951

### Title
Trichoderma applications on strawberry plants: a proteomic analysis of treated vs control plants

### Description
Many Trichoderma spp. are successful plant beneficial microbial inoculants due to their ability to act as biocontrol agents with direct antagonistic activities to phytopathogens, and as biostimulants capable of promoting plant growth. This work investigates the effects of treatments with three selected Trichoderma (strains T22, TH1 and GV41) to strawberry plants on the productivity and proteome of the formed fruit. Proteomic analysis of fruits,harvested from the plants previously treated with Trichoderma and control plants was performed by using a TMT-based protein quantification strategy. Bioinformatic analysis of the differential proteins accumulation in fruits, harvested from the treated plants, revealed a central network of interacting molecular species, that demonstrated the modulation of different plant physiological processes following the microbial inoculation.

### Sample Protocol
Lyophilized samples from pooled frozen fruits of Trichoderma-treated and control strawberry plants (9 fruits from 3 plants for each condition) were in parallel quickly extracted for proteins through a slightly modified phenol-extraction method followed by ammonium acetate���������methanol precipitation  Thus, 1 g of lyophilized powder of each sample was mixed in a mortar with 1% w/w polyvinylpyrrolidone, and resuspended in 10 ml of 0.7 M sucrose, 0.1 M KCl, 0.5 M Tris���������HCl, 50 mM EDTA, 40 mM DTT, pH 8.5, containing a protease inhibitors cocktail for plant tissues (Sigma-Aldrich, USA). Homogenization was performed in an Ultra-Turrax tissue processor. Tris-buffered phenol, pH 8.0 (Sigma-Aldrich) was added to the suspension (1:1, v/v) and the sample was mixed thoroughly and centrifuged at 10,000 x g, for 15 min, at 4 ������C. The extraction was repeated twice and phenol phases were collected and precipitated with 5 vol of cold 0.1M ammonium acetate in methanol, at -20 ������C, overnight. Protein pellets were washed twice with ice-cold methanol and finally with cold acetone containing 20 mM DTT, and then air-dried. Protein pellets (5 mg) were solubilized in 250 ���������l of 7 M urea, 2 M tiourea, 50 mM triethylammonium bicarbonate (TEAB), 2% SDS, 10 mM DTT, pH 8.5, and added with plant specific proteases inhibitors. Samples were vortexed and incubated 1 h, at 30 ������C, under shaking. Samples were centrifuged at 12,000 x g for 5 min, at 5 ������C, and the taken of the supernatant containing the corresponding protein extract. Protein concentration was determined using the Bio-Rad Protein Assay. Relative quantification of individual proteins was obtained by a tandem mass tagging (TMT)-labelling experiment using a TMT10plex Isobaric Label Reagent Kit (Thermo-Fisher Scientific, USA); protein samples were prepared according to manufacturer���������s instructions. Peptides from each protein sample were labelled with the TMT10plex Label Reagent Set (Thermo-Fisher Scientific, USA), at 25 ������C, following manufacturer���������s instructions, according to the labelling scheme: Ctrl-TMT10-126, T22-2-TMT10-127, TH1-TMT10-128 and GV41-TMT10-129. After 1���������h of reaction, 8���������������l of 5% w/v hydroxylamine was added in each tube and mixed for 15���������min, in order to quench the derivatization reaction. For a set of comparative experiments, tagged peptide mixtures were mixed in equal-molar ratios (1:1:1:1) and vacuum-dried under rotation. Then, pooled TMT-labelled peptide mixtures were suspended in 0.1% trifluoroacetic acid, and fractionated by using the Pierce��������� High pH Reversed-Phase Peptide fractionation kit (Thermo-Fisher Scientific) to remove unbound TMT reagents and to reduce sample complexity, according to manufacturer���������s instructions. After fractionation, eight fractions of TMT-labelled peptides were collected, vacuum-dried and finally reconstituted in 0.1% formic acid for subsequent mass spectrometric analysis.  TMT-labelled peptide fractions were analyzed with a nanoLC-ESI-Q-Orbitrap-MS/MS platform consisting of an UltiMate 3000 HPLC RSLC nano system (Dionex, USA) coupled to a Q-ExactivePlus mass spectrometer through a Nanoflex ion source (Thermo-Fisher Scientific). Peptides were loaded on an Acclaim PepMapTM RSLC C18 column (150 mm ������ 75 ������m ID, 2 ������m particles, 100 ������ pore size) (Thermo-Fisher Scientific), and eluted with a gradient of solvent B (19.92/80/0.08 v/v/v water/acetonitrile/FA) in solvent A (99.9/0.1 v/v water/FA), at a flow rate of 300 nl/min. The gradient of solvent B started at 5%, increased to 60% over 125 min, raised to 95% over 1 min, remained at 95% for 8 min, and finally returned to 5% in 1 min, with a column equilibrating step of 20 min before the subsequent chromatographic run. The mass spectrometer operated in data-dependent mode, using a full scan (m/z range 375-1500, nominal resolution of 70,000), followed by MS/MS scans of the 10 most abundant ions. MS/MS spectra were acquired in a scan m/z range 110-2000, using a normalized collision energy of 32%, an automatic gain control target of 100,000, a maximum ion target of 120 ms, and a resolution of 17,500. A dynamic exclusion value of 30 s was also used.

### Data Protocol
Raw data files were analyzed for protein identification and relative protein quantification with Proteome Discoverer vs 2.1 software (Thermo Scientific, USA), enabling the database search by Mascot algorithm v. 2.6 (Matrix Science, UK) using the following criteria: NCBI protein database (Viridiplantae, 6216064 protein sequences, 12/2018) including the most common protein contaminants; carbamidomethylation of Cys and TMT10plex modification of lysine and peptide N-terminal as fixed modifications; oxidation of Met, deamidation of Asn and Gln, pyroglutamate formation of Gln as variable modifications. Peptide mass tolerance was set to ������ 10 ppm and fragment mass tolerance to ������ 0.02 Da. Proteolytic enzyme and maximum number of missed cleavages were set to trypsin and 2, respectively. Protein candidates assigned on the basis of at least 2 sequenced peptides and an individual Mascot Score ��������� 30 were considered confidently identified. For quantification, ratios of TMT reporter ion intensities in the MS/MS spectra from raw datasets were used to calculate fold changes between samples. Definitive peptide assignment was always associated with manual spectra visualization and verification. Results were filtered to 1% false discovery rate

### Publication Abstract
Many <i>Trichoderma</i> spp. are successful plant beneficial microbial inoculants due to their ability to act as biocontrol agents with direct antagonistic activities to phytopathogens, and as biostimulants capable of promoting plant growth. This work investigated the effects of treatments with three selected <i>Trichoderma</i> strains (T22, TH1, and GV41) to strawberry plants on the productivity, metabolites and proteome of the formed fruits. <i>Trichoderma</i> applications stimulated plant growth, increased strawberry fruit yield, and favored selective accumulation of anthocyanins and other antioxidants in red ripened fruits. Proteomic analysis of fruits harvested from the plants previously treated with <i>Trichoderma</i> demonstrated that the microbial inoculants highly affected the representation of proteins associated with responses to stress/external stimuli, nutrient uptake, protein metabolism, carbon/energy metabolism and secondary metabolism, also providing a possible explanation to the presence of specific metabolites in fruits. Bioinformatic analysis of these differential proteins revealed a central network of interacting molecular species, providing a rationale to the concomitant modulation of different plant physiological processes following the microbial inoculation. These findings indicated that the application of <i>Trichoderma</i>-based products exerts a positive impact on strawberry, integrating well with previous observations on the molecular mechanisms activated in roots and leaves of other tested plant species, demonstrating that the efficacy of using a biological approach with beneficial microbes on the maturing plant is also able to transfer advantages to the developing fruits.

### Keywords
Trichoderma, Strawberry, Tmt, Lc/ms/ms

### Affiliations
CNR ISPAAM
stituto per il Sistema Produzione Animale in Ambiente Mediterraneo (ISPAAM) ��������� Consiglio Nazionale delle Ricerche (CNR)

### Submitter
anna maria salzano

### Lab Head
Dr Andrea Scaloni
stituto per il Sistema Produzione Animale in Ambiente Mediterraneo (ISPAAM) ��������� Consiglio Nazionale delle Ricerche (CNR)


