### Accession
PXD013531

### Title
PD-1 interacting proteins in SILAC labelled Jurkat cells stimulated with Raji cells - Part4

### Description
Identifying PD-1 interacting proteins in Jurkat cells expressing PD-1 after stimulation with Raji cells and the superantigen SEE.

### Sample Protocol
Affinity purification coupled to proteomic detection was performed to detect the interacting proteins of Jurkat cells expressing PD-1 molecules tagged with a Twin-Strep-tag (OST) (PD-1-OST) at similar levels as wild type Jurkat cells. Cells before stimulation or after 2 min of stimulation with wild type or PD-L1 expressing heavy-labelled Raji B-cells and superantigen SEE were measured in three different independent replicates.

### Data Protocol
The data was searched using MaxQuant version 1.5.2.8 against a FASTA file containing 16985 mouse protein sequences (downloaded 13. August 2018 from www.uniprot.org) and iRT peptide and enzyme sequences. Carbamidomethyl was defined as a fixed modification, and Oxidation (M) and Phosphorylation (STY) as variable modifications, and using heavy labels (Arg10,Lys8). Standard MaxQuant settings for Orbitrap were used (e.g. peptide tolerance 20 ppm for first search and 4.5 for main search). A search was performed searching 12 injections of unfractionated peptides and resulted in the identification of 13â€™842 peptides respectively. The search results were imported into Spectronaut Pulsar (Biognosys, Schlieren) to build a spectral library with the following settings: PSM FDR Cut off of 0.01, fragment m/z between 200 and 1800, peptides with at least 3 amino acids, fragment ions with a relative intensity of at least 5, pecursors with at least 5 fragments, excluding b-ions, and using the llabel workflow. Moreover, an iRT calibration was performed with a minimum root mean square error of 0.8 and segmented regression. The spectral library contained coordinates for 1840 protein groups and 11'346 proteotypic peptides. Quantitative data were extracted from the acquired SWATH-MS maps using Spectronaut Pulsar (Biognosys, Schlieren) and standard settings (e.g. dynamic MS1, and MS2 mass tolerance strategy was used, a dynamic XIC RT Extraction Window with a non-linear iRT calibration strategy, identification was performed using a precursor and protein Q value cutoff of 0.01). The quantified intensities for each fragment were extracted from 9 SWATH-MS injections and exported for further statistical analysis to R.

### Publication Abstract
None

### Keywords
Thermo fusion lumos, T cells, Pd-1

### Affiliations
1. Department of Biology, Institute of Molecular Systems Biology, ETH Zurich, 8093 Zurich, Switzerland 2. Faculty of Science, University of Zurich, 8006 Zurich, Switzerland
Idorsia Pharmaceuticals Ltd

### Submitter
Peter Blattmann

### Lab Head
Dr Ruedi Aebersold
1. Department of Biology, Institute of Molecular Systems Biology, ETH Zurich, 8093 Zurich, Switzerland 2. Faculty of Science, University of Zurich, 8006 Zurich, Switzerland


