### Accession
PXD001197

### Title
SIL1-depleted HEK293 cells -  Cellular Signature of SIL1 Depletion: Disease Pathogenesis due to Alterations in Protein Composition Beyond the ER Machinery

### Description
Marinesco-Sjögren syndrome (MSS) is a neurodegenerative disorder caused by autosomal recessive SIL1 mutations. SIL1 acts as a nucleotide exchange factor for the endoplasmic reticulum (ER) resident chaperone BiP. As BiP controls many ER-related processes, it is likely to contribute to MSS pathology. Owing to the absence of appropriate in vitro models, the precise pathophysiological mechanisms leading to neurodegeneration in MSS are still elusive. Here, we demonstrate for the first time that SIL1-depleted HEK293 cells can be used to reveal these mechanisms using ultra-structural, cell biological, and biochemical approaches.

### Sample Protocol
Cell lysis and carbamidomethylation In total six samples, i.e. two stably clonal SIL1-depleted HEK293 cell lines (∆SIL1_1 and ∆SIL1_2) and one appropriate clonal scrambled control line (Scr), each in biological duplicates, were processed independently (Fig.1).  Approximately 1 mg of cells were lysed in 0.5 mL of 50 mM Tris-HCl (pH 7.8) buffer containing 150 mM NaCl, 1% SDS, and Complete Mini. Subsequently, 10 µL of benzonase (25 U/µL) and 2 mM MgCl2 were added to the lysates and incubated at 37°C for 30 min. Samples were centrifuged at 4°C and 18,000 g for 15 min. Protein concentration of the supernatant was determined by BCA assay according to the manufacturer’s protocol. Cysteines were reduced by addition of 10 mM DTT at 56°C for 30 min, followed by alkylation of free thiol groups with 30 mM IAA at room temperature (RT) in the dark for 30 min.   Sample preparation and trypsin digestion Sample preparation and proteolysis were performed using filter-aided sample preparation (FASP) [4, 5] with minor changes. Briefly, cell lysate corresponding to 100 µg of protein was diluted 10-fold with freshly prepared 8 M urea/100 mM Tris-HCl (pH 8.5) buffer [6] and placed on a Microcon centrifugal device (30 KDa cutoff). The device was centrifuged at 13,500 g at RT for 20 min. All the following centrifugation steps were performed under the same conditions. To eliminate residual SDS, three washing steps were carried out with 100 µL of 8 M urea/100 mM Tris-HCl (pH 8.5). For buffer exchange, the device was washed thrice with 100 µL of 50 mM NH4HCO3 (pH 7.8). To the concentrated proteins, 100 µL of proteolysis buffer comprising of trypsin (Promega) (1:25 w/w, protease to substrate), 0.2 M GuHCl and 2 mM CaCl2 in 50 mM NH4HCO3 (pH 7.8), was added and incubated at 37°C for 14 h. The generated tryptic peptides were recovered by centrifugation with 50 µL of 50 mM NH4HCO3 followed by 50 µL of ultra-pure water. Finally, peptides were acidified by addition of 10% TFA (v/v) and digests were quality controlled as described previously [7].        LC-MS/MS analysis  Each condition was measured in triplicate (18 samples, 1 µg each) using an Ultimate 3000 nano RSLC system coupled to an Orbitrap Elite mass spectrometer (both Thermo Scientific). Samples were analyzed in randomized order to minimize systematic errors. Briefly, peptides were preconcentrated on a 100 µm x 2 cm C18 trapping column for 10 min using 0.1% TFA (v/v) at a flow rate of 20 µL/min followed by separation on a 75 µm x 50 cm C18 main column (both Pepmap, Thermo Scientific) with a 187 min LC gradient ranging from 3-42% of 84% ACN, 0.1% FA (v/v) at a flow rate of 230 nL/min. MS survey scans were acquired in the Orbitrap from m/z 300 to 1500 at a resolution of 60,000 using the polysiloxane ion at m/z 371.101236 as lock mass [8]. The fifteen most intense signals were subjected to collision induced dissociation (CID) in the ion trap, taking into account a dynamic exclusion of 30 s. CID spectra were acquired with a normalized collision energy of 35% and an activation time of 10 ms. AGC target values were set to 1e6 for Orbitrap MS and 1e4 for ion trap MSn scans, and maximum injection times were set to 100 ms for both full MS and MSn scans.

### Data Protocol
Label free data analysis Data analysis of the acquired label free quantitative MS data was performed using the Progenesis LC-MS software from Nonlinear Dynamics (Newcastle upon Tyne, U.K.). For both sample sets, triplicate measurements of ∆SIL1_1 and ∆SIL1_2, respectively, were compared to the corresponding control triplicates separately. Alignment of MS raw data was conducted by Progenesis which automatically selected one of the LC-MS files as reference. After peak picking, only features within retention time and m/z windows from 0-200 min and 300-1500 m/z, with charge states +2, +3, and +4 were considered for peptide statistics, analysis of variance (ANOVA) and principal component analysis (PCA). MS/MS spectra were exported as peak lists. To maximize the number of identified peptides and proteins at a given quality we used our PeptideShaker software 0.28.0 (http://code.google.com/p/peptide-shaker/) for interpretation of peptide and protein identifications. Therefore, peak lists were searched against a concatenated target/decoy version of the human Uniprot database, (downloaded on 11th of December 2013, containing 20,273 target sequences) using Mascot 2.4 (Matrix Science), OMSSA 2.1.9, and X!Tandem Jackhammer (2013.06.15) with the help of searchGUI 1.14.4 [9]. Trypsin with a maximum of two missed cleavages was selected as enzyme. Carbamidomethylation of Cys was set as fixed and oxidation of Met was selected as variable modification. MS and MS/MS tolerances were set to 10 ppm and 0.5 Da, respectively. Combined search results were filtered at a false discovery rate (FDR) of 1% on the protein level and exported using the advanced PeptideShaker features that allow direct re-import of the quality-controlled data into Progenesis. Peptide sequences containing oxidized Met and pyro-Glu (derived from X!Tandem 2nd pass search) were excluded for further analysis in order to reduce unanticipated bias in quantification. Only proteins that were quantified with unique peptides were exported. Then, for each protein, the average of the normalized abundances (obtained from Progenesis) from the triplicate analyses was calculated to determine the ratios between the scrambled controls and SIL-1 depleted samples. Only proteins which were (i) commonly quantified in all the replicates with (ii) at least two unique peptides, (iii) an ANOVA p-value of <0.05 (Progenesis) and (iv) an average ratio < 0.667 or > 1.6 (corresponding to log2 ratios of +/- 0.65) were considered as regulated.

### Publication Abstract
SIL1 acts as nucleotide exchange factor for the endoplasmic reticulum chaperone BiP. Mutations of SIL1 cause Marinesco-Sj&#xf6;gren syndrome (MSS), a neurodegenerative disorder. Moreover, a particular function of SIL1 for etiopathology of amyotrophic lateral sclerosis (ALS) was highlighted, thus declaring the functional SIL1-BiP complex as a modifier for neurodegenerative disorders. Thereby, depletion of SIL1 was associated with an earlier manifestation and in strengthened disease progression in ALS. Owing to the absence of appropriate in vitro models, the precise cellular pathophysiological mechanisms leading to neurodegeneration in MSS and triggering the same in further disorders like ALS are still elusive. We found that SIL1 depletion in human embryonic kidney 293 (HEK293) cells led to structural changes of the endoplasmic reticulum (ER) including the nuclear envelope and mitochondrial degeneration that closely mimic pathological alterations in MSS and ALS. Functional studies revealed disturbed protein transport, cytotoxicity with reduced proliferation and viability, accompanied by activation of cellular defense mechanisms including the unfolded protein response, ER-associated degradation pathway, proteolysis, and expression of apoptotic and survival factors. Our data moreover indicated that proteins involved in cytoskeletal organization, vesicular transport, mitochondrial function, and neurological processes contribute to SIL1 pathophysiology. Altered protein expression upon SIL1 depletion in vitro could be confirmed in Sil1-deficient motoneurones for paradigmatic proteins belonging to different functional classes. Our results demonstrate that SIL1-depleted HEK293 cells are an appropriate model to identify proteins modulated by SIL1 expression level and contributing to neurodegeneration in MSS and further disorders like ALS. Thereby, our combined results point out that proteins beyond such involved ER-related protein processing are affected by SIL1 depletion.

### Keywords
Hek293, Proteome, Sil1

### Affiliations
Leibniz – Institut für Analytische Wissenschaften - ISAS - e.V.
Segal Cancer Proteomics Centre

### Submitter
René Zahedi

### Lab Head
Dr René Peiman Zahedi
Leibniz – Institut für Analytische Wissenschaften - ISAS - e.V.


