### Accession
PXD003019

### Title
Identification of AMPK phosphorylation sites reveals a network of proteins involved  in cell invasion and facilitates large-scale substrate prediction

### Description
AMP-activated protein kinase (AMPK) is a central energy gauge that regulates metabolism and has been increasingly involved in non-metabolic processes and diseases. However, AMPKâ€™s direct substrates in non-metabolic contexts are largely unknown. To better understand the AMPK network, we use a chemical genetics screen coupled to a peptide capture approach in whole cells, resulting in identification of direct AMPK phosphorylation sites. Interestingly, the high-confidence AMPK substrates contain many proteins involved in cell motility, adhesion, and invasion. AMPK phosphorylation of the RHOA guanine nucleotide exchange factor NET1A inhibits extracellular matrix degradation, an early step in cell invasion. The identification of direct AMPK phosphorylation sites also facilitates large-scale prediction of AMPK substrates. We provide an AMPK motif matrix and a pipeline to predict additional AMPK substrates from quantitative phosphoproteomics datasets. As AMPK is emerging as a critical node in aging and pathological processes, our study identifies potential targets for therapeutic strategies.

### Sample Protocol
Please see the paper titled "Identification of AMPK phosphorylation sites reveals a network of proteins involved in cell invasion and facilitates large-scale substrate prediction" from Schaffer et. al. in Cell Metabolism, 2015 for details. Roughly the Analog specific Kinase with covalent capture of substrate approach (developed by Kevan Shokat) was used to enrich for substrate phosphorylations of AMPK. Samples were analyzed by shotgun HCD and/or ETD on a ThermoFisher Orbitrap Velos.

### Data Protocol
Please see the paper titled "Identification of AMPK phosphorylation sites reveals a network of proteins involved in cell invasion and facilitates large-scale substrate prediction" from Schaffer et. al. in Cell Metabolism, 2015 for details. Raw data was processed using Protein Prospector. Data was further analyzed by in-house scripts (available on GitHub).

### Publication Abstract
AMP-activated protein kinase (AMPK) is a central energy gauge that regulates metabolism and has been increasingly involved in non-metabolic processes and diseases. However, AMPK's direct substrates in non-metabolic contexts are largely unknown. To better understand the AMPK network, we use a chemical&#xa0;genetics screen coupled to a peptide capture approach in whole cells, resulting in identification of direct AMPK phosphorylation sites. Interestingly, the high-confidence AMPK substrates contain many proteins involved in cell motility, adhesion, and invasion. AMPK phosphorylation of the RHOA guanine nucleotide exchange factor NET1A inhibits extracellular matrix degradation, an early step in cell invasion. The identification of direct AMPK phosphorylation sites also facilitates large-scale prediction of AMPK substrates. We provide an AMPK motif matrix and a&#xa0;pipeline to predict additional AMPK substrates from quantitative phosphoproteomics datasets. As AMPK is emerging as a critical node in aging and pathological processes, our study identifies potential targets for therapeutic strategies.

### Keywords
Substrate identification, Phosphorylation, Substrate prediction, Ampk

### Affiliations
University of California, San Francisco
University of California-San Francisco

### Submitter
Rebecca Levin

### Lab Head
Dr Kevan M. Shokat
University of California, San Francisco


