### Accession
PXD000333

### Title
Human Primary T-cells Phospoproteomic WorkFlow

### Description
LymPHOS workflow: a collection of open source tools for mass spectrometric data mining in phosphoproteomic studies.        We present several bioinformatics applications designed as a part of a workflow for the identification and quantification of phosphoproteome components by mass spectrometry. This workflow includes a front-end graphical user interface which combines several RAW to MGF extractors (EasierMgf), two graphical user interfaces for the well-known search engines Omssa and Sequest (OmssaGui and SequestGui) and three applications, one for FASTA database management (FastaTools), other for the integration of search results from up to three search engines (Integrator), and another one for the visualization of mass spectra and their corresponding database search results (JsonVisor). These applications were developed to solve some of the common problems found in the analysis of proteomic and phosphoproteomic data and integrated in the workflow for the processing and feeding of data into our LymPHOS database . Applications are designed modularly and can be used standalone. These tools are written in Perl and Python programming languages and are supported on Windows platforms. They are all Open Source Software under a GNU license, and can be freely downloaded from our software repository hosted at GoogleCode.

### Sample Protocol
Not available

### Data Protocol
Not available

### Publication Abstract
We present several bioinformatics applications for the identification and quantification of phosphoproteome components by MS. These applications include a front-end graphical user interface that combines several Thermo RAW formats to MASCOT&#x2122; Generic Format extractors (EasierMgf), two graphical user interfaces for search engines OMSSA and SEQUEST (OmssaGui and SequestGui), and three applications, one for the management of databases in FASTA format (FastaTools), another for the integration of search results from up to three search engines (Integrator), and another one for the visualization of mass spectra and their corresponding database search results (JsonVisor). These applications were developed to solve some of the common problems found in proteomic and phosphoproteomic data analysis and were integrated in the workflow for data processing and feeding on our LymPHOS database. Applications were designed modularly and can be used standalone. These tools are written in Perl and Python programming languages and are supported on Windows platforms. They are all released under an Open Source Software license and can be freely downloaded from our software repository hosted at GoogleCode.

### Keywords
Human, T-cells, Lc-msms, Lymphos, Phosphoproteomics, Demo files

### Affiliations
LP-CSIC/UAB

### Submitter
Ãscar Gallardo RomÃ¡n

### Lab Head


