### Accession
PXD040002

### Title
Impact of TrmB on the Acinetobacter baumannii proteomic

### Description
With this work the Impact of TrmB on the Acinetobacter baumannii proteomic was examined (=/-) H2O2

### Sample Protocol
Precipitated protein samples were solubilized in 4% SDS, 100mM Tris pH 8.5 by boiling them for 10 min at 95 Â°C. The protein concentrations were then assessed by bicinchoninic acid protein assays (Thermo Fisher Scientific) and 200Î¼g of each biological replicate prepared for digestion using Micro S-traps (Protifi, USA) according to the manufacturerâ€™s instructions. Briefly, samples were reduced with 10mM DTT for 10 mins at 95Â°C and then alkylated with 40mM IAA in the dark for 1 hour. Samples were acidified to 1.2% phosphoric acid and diluted with seven volumes of S-trap wash buffer (90% methanol, 100mM Tetraethylammonium bromide pH 7.1) before being loaded onto S-traps and washed 3 times with S-trap wash buffer. Samples were then digested with 2.5Î¼g of Trypsin overnight at 37Â°C before being collected by centrifugation with washes of 100mM Tetraethylammonium bromide, followed by 0.2% formic acid followed by 0.2% formic acid / 50% acetonitrile. Samples were dried down and further cleaned up using C18 Stage tips to ensure the removal of any particulate matter.

### Data Protocol
C18 enriched proteome samples were re-suspended in Buffer A* (2% acetonitrile, 0.01% trifluoroacetic acid) and separated using a two-column chromatography setup composed of a PepMap100 C18 20-mm by 75-ï�­m trap (Thermo Fisher Scientific) and a PepMap C18 500-mm by 75-ï�­m analytical column (Thermo Fisher Scientific) using a Dionex Ultimate 3000 UPLC (Thermo Fisher Scientific). Samples were concentrated onto the trap column at 5 ï�­l/min for 6 min with Buffer A (0.1% formic acid, 2% DMSO) and then infused into an Orbitrap 480â„¢ (Thermo Fisher Scientific) at 300 nl/minute via the analytical columns. Peptides were separated by altering the buffer composition from 3% Buffer B (0.1% formic acid, 77.9% acetonitrile, 2% DMSO) to 23% B over 89 minutes, then from 23% B to 40% B over 10 minutes and then from 40% B to 80% B over 5 minutes. The composition was held at 80% B for 5 minutes before being returned to 3% B for 10 minutes. The Orbitrap 480â„¢ Mass Spectrometer was operated in a data-dependent mode automatically switching between the acquisition of a single Orbitrap MS scan (300-2000 m/z, maximal injection time of 25 ms, an Automated Gain Control (AGC) set to a maximum of 300% and a resolution of 120k) and 3 seconds of Orbitrap MS/MS HCD scans of precursors (Stepped NCE of 25;30;45%, a maximal injection time of 80 ms, a AGC of 500% and a resolution of 30k).Identification and LFQ analysis were accomplished using MaxQuant (v1.6.17.0)3 using the ARC6851 Proteome (NCBI GCA_025677625.1 / ASM2567762v1) with Carbamidomethyl (C) allowed as a fixed modification and Acetyl (Protein N-term) as well as Oxidation (M) allowed as variable modifications with the LFQ and â€œMatch Between Runâ€� options enabled. The resulting data files were processed using Perseus (version 1.6.0.7)with missing values imputed based on the total observed protein intensities with a range of 0.3 Ïƒ and a downshift of 1.8 Ïƒ. Statistical analysis was undertaken in Perseus using two-tailed unpaired T-tests and ANOVAs. Matching of protein homologs between the strain UPAB1 (NCBI GCF_006843645.1 / ASM684364v1) and ARC6851 (NCBI GCA_025677625.1 / ASM2567762v1) was undertaken using the proteome comparison tool within PATRIC, the bacterial bioinformatics database and analysis resource.

### Publication Abstract
None

### Keywords
Acinetobacter baumannii, Proteomics

### Affiliations
University of Melbourne
Department of Molecular Microbiology, Washington University School of Medicine, St. Louis, MO 63110, USA.

### Submitter
Nichollas Scott

### Lab Head
Dr Mario Feldman
Department of Molecular Microbiology, Washington University School of Medicine, St. Louis, MO 63110, USA.


