### Accession
PXD028959

### Title
Human HEK293 Gemin5 clinical variants LC-MSMS

### Description
Identification of cellular proteins interacting with Gemin5 variants found in patients with neurological disorders. P85 C-terminal fragment of Gemin5 has been used to purify the protein complexes associated by Tandem Affinity Purification. The study has been performed using two biological replicates of each bait protein: p85-WT, the dimerization mutant p85-A951E, and the clinical variants p85-R1016C, p85-D1019E and p85-L1367P.

### Sample Protocol
HEK293 cells transfected with the plasmids expressing p85-TAP proteins (WT, A951E, R1016C, D1019E and L1367P) were harvested 24 hpt. The protein complexes associated to these fusion proteins were isolated using Tandem Affinity Purification. Briefly, the extract from the TEV protease digestion of the first IgG Sepharose purification was subsequently subjected to a second Calmodulin purification step. Purified proteins were precipitated with 10% trichloroacetic acid at 4ºC overnight, pelleted at 14000 g for 15 min at 4ºC, washed three times with 1 ml of acetone and dissolved in SDS-loading buffer. An aliquot (20 %) was analyzed on silver stained SDS-PAGE gels to visualize the purification of proteins associated to p85-TAP polypeptides.  Two independent biological replicates of TAP samples obtained were applied onto a 10% SDS-PAGE gel. The protein bands concentrated in the stacking/resolving gel interface were visualized by Coomassie staining. The gel pieces were destained in acetonitrile:water (ACN:H2O, 1:1), were reduced and alkylated, and then digested in situ with sequencing grade trypsin. The gel pieces were dried and re-swollen in 50 mM ammonium bicarbonate pH 8.8 with 60 ng/µl trypsin at 5:1 protein:trypsin (w/w) ratio. The tubes were kept in ice for 2 h and incubated at 37°C for 12 h. Digestion was stopped by the addition of 1% TFA. The desalted protein digest was dried, resuspended in 10 µl of 0.1% formic acid and analyzed by RP-LC-MS/MS in an Easy-nLC II system coupled to an ion trap LTQ-Orbitrap-Velos-Pro hybrid mass spectrometer. The peptides were concentrated (on-line) by reverse phase chromatography using a 0.1 mm × 20 mm C18 RP precolumn, and then separated using a 0.075mm x 250 mm C18 RP column operating at 0.3 µl/min. Peptides were eluted using a 120 min dual gradient from 5 to 25% solvent B in 90 min followed by gradient from 25 to 40% solvent B over 120 min (Solvent A: 0.1% formic acid in water, solvent B: 0.1% formic acid, 80% acetonitrile in water). ESI ionization was done using a Nano-bore emitters Stainless Steel ID 30 µm (Proxeon) interface. The Orbitrap resolution was set at 30.000. Peptides were detected in survey scans from 400 to 1600 amu (1 µscan), followed by twenty data dependent MS/MS scans (Top 20), using an isolation width of 2 u (in mass-to-charge ratio units), normalized collision energy of 35%, and dynamic exclusion applied during 30 seconds periods.

### Data Protocol
Peptide identification from raw data was carried out using PEAKS Studio X search engine (Bioinformatics Solutions Inc). Database search was performed against UniProt-Homo sapiens FASTA (decoy-fusion database). The following constraints were used for the searches: tryptic cleavage after Arg and Lys, up to two missed cleavage sites, and tolerances of 20 ppm for precursor ions and 0.6 Da for MS/MS fragment ions and the searches were performed allowing optional Met oxidation and Cys carbamidomethylation. False discovery rates (FDR) for peptide spectrum matches (PSM) were limited to 0.01. Only those proteins with at least two distinct peptides being discovered from LC/MS/MS analyses were considered reliably identified.

### Publication Abstract
None

### Keywords
Human, Hek293, Lc-msms, Gemin5, Clinical variants

### Affiliations
Genome Dynamics and Function Program Centro de Biologia Molecular Severo Ochoa Spain
CBMSO

### Submitter
Rosario Francisco-Velilla

### Lab Head
Dr ENCARNACION MARTINEZ-SALAS
Genome Dynamics and Function Program Centro de Biologia Molecular Severo Ochoa Spain


