### Accession
PXD030771

### Title
Loss of H3K9 tri-methylation alters mitotic chromosome compaction and transcription factor retention during mitosis

### Description
Recent studies have shown that repressive chromatin machinery, including DNA methyltransferases (DNMTs) and Polycomb Repressor Complexes (PRCs), bind to chromosomes throughout mitosis and their depletion results in increased chromosome size. Enzymes that catalyse H3K9 methylation, such as Suv39h1, Suv39h2, G9a, GLP are also retained by mitotic chromosomes. Surprisingly however, mutants lacking H3K9me3 have unusually small and compact mitotic chromosomes that are associated with increased H3S10ph and H3K27me3 levels. Chromosome size and centromere compaction in these mutants was rescued by providing exogenous Suv39h1, or inhibiting Ezh2 activity. Quantitative proteomic comparisons of native mitotic chromosomes isolated from wildtype versus Suv39h1,2 double null ESCs revealed that H3K9me3 was essential for the efficient retention of bookmarking factors such as Esrrb. These results highlight an unexpected role for repressive heterochromatin domains in preserving transcription factor binding through mitosis, and underscore the importance of H3K9me3 for sustaining chromosome architecture and epigenetic memory during cell division.

### Sample Protocol
Label-free quantitative proteomics analysis was performed on mitotic lysate pellet and sorted chromosomes (equivalent number, 10 million) isolated from W26, DN57, DN57 treated with GSK343 or Hesperadin, DN57-Suv39h1-EGFP, W8, D5 using a method previously described (Djeghloul et al., 2020). Both flow-sorted mitotic chromosomes and mitotic lysates were pelleted by centrifugation (14,000 g, 15 min, 4°C). Supernatants were removed and the pellets were digested with trypsin by an in-Stage Tip digestion protocol (Kulak et al., 2014) using commercially available iST Kits (Preomics, Martinsried, Germany) according to the manufacturer’s recommendations. Protein digests were analysed by liquid chromatography-tandem mass spectrometry (LC-MS/MS) using a data-dependent acquisition method with a 50cm EasySpray column at a flow rate of 250 nl/min coupled to a Q-Exactive HF-X mass spectrometer, as described previously (Djeghloul et al., 2020). Experiments were carried out in biological triplicate and digests were analysed by LC-MS/MS as technical duplicates.   References Djeghloul, D., Patel, B., Kramer, H., Dimond, A., Whilding, C., Brown, K., Kohler, A.C., Feytout, A., Veland, N., Elliott, J., et al. (2020). Identifying proteins bound to native mitotic ESC chromosomes reveals chromatin repressors are important for compaction. Nat Commun 11, 4118. Kulak, N.A., Pichler, G., Paron, I., Nagaraj, N., and Mann, M. (2014). Minimal, encapsulated proteomic-sample processing applied to copy-number estimation in eukaryotic cells. Nat Methods 11, 319-324.

### Data Protocol
The LC-MS/MS raw data were analysed using MaxQuant v1.6.10.43 (Cox and Mann, 2008; Tyanova et al., 2016a) using the MaxLFQ Label-Free Quantification algorithm (Cox et al., 2014). Further statistical analysis as well as data visualisation were performed using the Perseus software v1.6.7.0 (Tyanova et al., 2016b).  References Cox, J., Hein, M.Y., Luber, C.A., Paron, I., Nagaraj, N., and Mann, M. (2014). Accurate proteome-wide label-free quantification by delayed normalization and maximal peptide ratio extraction, termed MaxLFQ. Mol Cell Proteomics 13, 2513-2526. Cox, J., and Mann, M. (2008). MaxQuant enables high peptide identification rates, individualized p.p.b.-range mass accuracies and proteome-wide protein quantification. Nat Biotechnol 26, 1367-1372 Tyanova, S., Temu, T., and Cox, J. (2016a). The MaxQuant computational platform for mass spectrometry-based shotgun proteomics. Nat Protoc 11, 2301-2319. Tyanova, S., Temu, T., Sinitcyn, P., Carlson, A., Hein, M.Y., Geiger, T., Mann, M., and Cox, J. (2016b). The Perseus computational platform for comprehensive analysis of (prote)omics data. Nat Methods 13, 731-740.

### Publication Abstract
None

### Keywords
Chromatin; mitotic chromosomes; h3k9me3; suv39h1; suv39h2; ezh2; esrrb

### Affiliations
MRC London Institute of Medical Sciences
Proteomics & Metabolomics Facility MRC London Institute of Medical Sciences  Hammersmith Hospital Campus Du Cane Road London, W12 0NN United Kingdom

### Submitter
Holger Kramer

### Lab Head
Dr Holger Kramer
Proteomics & Metabolomics Facility MRC London Institute of Medical Sciences  Hammersmith Hospital Campus Du Cane Road London, W12 0NN United Kingdom


