### Accession
PXD023124

### Title
Nucleotide sugar biosynthesis occurs in the glycosomes of procyclic and bloodstream form Trypanosoma brucei

### Description
In Trypanosoma brucei, there are fourteen enzymatic biotransformations that collectively convert glucose into five essential nucleotide sugars: UDP-Glc, UDP-Gal, UDP-GlcNAc, GDP-Man and GDP-Fuc. These biotransformations are catalyzed by thirteen discrete enzymes, five of which possess putative peroxisome targeting sequences. Published experimental analyses using immunofluorescence microscopy and/or digitonin latency and/or subcellular fractionation and/or organelle proteomics have localized eight and six of these enzymes to the glycosomes of bloodstream form and procyclic form T. brucei, respectively. Here we increase these glycosome localizations to eleven in both lifecycle stages while noting that one, phospho-N-acetylglucosamine mutase, also localizes to the cytoplasm. In the course of these studies, the heterogeneity of glycosome contents was also noted. These data suggest that, unlike other eukaryotes, all of nucleotide sugar biosynthesis in T. brucei is compartmentalized to the glycosomes in both lifecycle stages.

### Sample Protocol
Samples were reduced with dithiothreitol and alkylated with iodoacetamide and subsequently digested with trypsin and processed using FASP II (filter-aided sample processing), as described previously (54). Samples were desalted using a custom made 1.5cm Porous R2 column packed with 1.5mg Porous R2 reverse phase beads (ABI). Column was pre-conditioned by washing with 90% CH3CN containing 0.1% formic acid followed by equilibration in 0.1%formic acid in water. Sample was applied, washed in 0.1% formic acid and subsequently eluted three times with 90% CH3CN containing 0.1% formic acid. Elutions were combined, Speedvac dryed, resuspended in 35% CH3CN containing 0.1% formic acid and fractionated using strong cation exchange (SCX) chromatography. The combined elutions above were applied into a custom made 2cm SCX column by adding porous 50 HS SCX beads (ABI) to a SCX Ziptip (Millipore) and eluted sequencially with  20, 100, 150, 200 mM NaCl in 35% CH3CN containing 0.1% formic acid followed by final elution in 0.5% NH4OH in 50% Isopropanol. A final wash with 50% isopropanol (IPA sample) was also processed. Samples (named as RPE3, 20mM, 100mM, 150mM, 200mM and IPA) were dried and re-suspended in 50 µL 1% formic acid and then subjected to liquid chromatography on Ultimate 3000 RSLCnano system (Thermo Scientific) fitted with a 3 Acclaim PepMap 100 (C18, 100 µM x 2 cm) and then separated on an Easy-Spray PepMap RSLC C18 column (75 µM x 50 cm) (Thermo Scientific). 15µL Samples were loaded in 0.1% formic acid (buffer A) and separated using a binary gradient consisting of buffer A and buffer B (80% acetonitrile, 0.1% formic acid). Peptides were eluted with a linear gradient from 2 to 40% buffer B over 70 min. The HPLC systems were coupled to a Orbitrap Velos Mass Spectrometer (Thermo Scientific) equipped with an Easy-Spray source with a source voltage of 1.2 kV.

### Data Protocol
MS/MS data were analysed for protein identifications using MaxQuant 1.6.14 [1] with the in-built Andromeda search engine [2]. The raw files were searched against the [3], using the T. brucei brucei 927 annotated protein sequences from TriTrypDB release 46 [4] supplemented with the T. brucei brucei 427 VSG221 (Tb427.BES40.22) protein sequence. The mass tolerance was set to 4.5 ppm for precursor ions and trypsin was set as proteolytic enzyme with two missed cleavage allowed. Carbamidomethyl on cysteine was set as fixed modifications. SILAC labelling [5] of heavy arginine (Arg-6) and Lysine (Lys-6) were specified. Oxidation of methionine and Acetylation of Protein N-term were set as variable modification. The false-discovery rate for protein and peptide level identifications was set at 1%, using a target-decoy based strategy. The minimum peptide length was set to seven amino acids and protein quantification was performed on unique plus razor peptides [6]. “Reverse Hits”, “Only identified by site” and “Potential contaminant” identifications were filtered out. Only protein groups with at least two unique peptide sequences and Andromeda protein score greater than 5 were selected for further analysis.

### Publication Abstract
None

### Keywords
T. brucei, Glycosome localization, Nucleotide sugar biosynthesis

### Affiliations
Biological Chemistry and Drug Discovery Wellcome Centre for Anti-Infectives Research (WCAIR)
Dundee University

### Submitter
Michele Tinti

### Lab Head
Dr Mike Ferguson
Biological Chemistry and Drug Discovery Wellcome Centre for Anti-Infectives Research (WCAIR)


