### Accession
PXD024942

### Title
Proteomic Analysis of Integrin a5 Associated Complexes Dependent upon Integrin Activation State in Zebrafish

### Description
Integrins are a major class of heterodimeric adhesion receptors composed of an a and b subunit that link the extracellular matrix (ECM) and the cytoskeleton across the cell membrane. When activated by either the intracellular (inside-out signaling) or extracellular (outside-in signaling) environment, integrins undergo a conformational change that increases the ligand binding affinity. As the primary receptor for ECM protein fibronectin, Integrin a5 plays a critical role in zebrafish somitogenesis. To better understand integrin activation in this context, we performed co-immunoprecipitation and Mass Spectrometry (MS) based proteomics using FLAG-tagged Integrin a5 alleles that alter it activation state: constitutive active mutant a5GAAKR and inactive ligand binding deficient mutant a5FYLDD, expressed in maternal zygotic a5 mutant (MZa5-/-) embryos. Our data provide an overview of Integrin associated proteins according to the activation state of the Integrin.

### Sample Protocol
For each replicate, 120 embryos were injected with mRNA encoding GFP-FLAG-tagged integrins at the one cell stage, raised to 10-13 somite stage, and then dechorionated using pronase (Sigma). Embryos were then incubated in modified Ringer’s solution (116mM NaCl, 3mM KCl, 4mM CaCl2, 1mM MgCl2, 5mM HEPES pH 7.8) containing 5mM DTBP (Dimethyl-3,3'-Dithiobispropionimidate, Thermo Scientific) at 28.6 °C for 5 hr. After quenching in modified Ringer’s solution containing 50mM Tris-HCl pH 7.6 on ice for 20 min, samples were lysed in lysis buffer (50mM Tris pH7.6, 150mM NaCl, 1mM EDTA, 10% glycerol, one tablet cOmplete protease inhibitor cocktail, 5% Triton X-100, 0.1% IGEPAL) for 30 min on ice. Samples were immunoprecipitated with anti-FLAG M2 affinity gel (A2220, Sigma). Sample elution was load on SDS-PAGE for protein separation and gels were visualized by Coomassie staining. Gel slices were cut into small pieces and washed for 10 min with water, followed by washing for 30 min with 50% acetonitrile (ACN)/100 mM NH4HCO3 (ammonium bicarbonate, ABC). The samples were reduced by the addition of 4.5 mM dithiothreitol (DTT) in 100 mM ABC with incubation at 37°C for 30 minutes. The DTT solution was removed and the samples were cooled to room temperature. The samples were alkylated by the addition of 10mM iodoacetamide (IAN) in 100mM ABC with incubation at room temperature in the dark for 30 minutes. The IAN solution was removed and the gels were washed for 15 minutes with 50% ACN/100 mM ABC, then washed for 15 minutes with 50% ACN/25 mM ABC. The gels were briefly dried by SpeedVac, then resuspended in 80 L of 25mM ABC containing 400 ng of digestion grade trypsin (Promega, V5111) and incubated at 37°C for 16 hours. The supernatant containing tryptic peptides was transferred to a new Eppendorf tube, and the gel band was extracted with 350 L of 80% acetonitrile/0.1% trifluoroacetic acid (TFA) for 15 minutes. Supernatants were combined and dried by speed vacuum. Peptides were dissolved in 25 L MS loading buffer (2% ACN, 0.2% TFA), with 5 L injected for LC-MS/MS analysis. LC-MS/MS analysis was performed on a Thermo Scientific Q Exactive Plus equipped with a Waters nanoAcquity UPLC system utilizing a binary solvent system (A: 100% water, 0.1% formic acid; B: 100% acetonitrile, 0.1% formic acid). Trapping was performed at 5L/min, 99.5% Buffer A for 3 min using a Waters ACQUITY UPLC M-Class Symmetry C18 Trap Column (100Å, 5 m, 180 m x 20 mm, 2G, V/M). Peptides were separated at 37°C using a Waters ACQUITY UPLC M-Class Peptide BEH C18 Column (130Å, 1.7 m, 75 m x 250 mm) and eluted at 300 nL/min with the following gradient: 3% buffer B at initial conditions; 5% B at 2 minutes; 25% B at 140 minutes; 40% B at 165 minutes; 90% B at 170 minutes; 90% B at 180 min; return to initial conditions at 182 minutes. MS was acquired in profile mode over the 300-1,700 m/z range using 1 microscan, 70,000 resolution, AGC target of 3E6, and a maximum injection time of 45 ms. Data dependent MS/MS were acquired in centroid mode on the top 20 precursors per MS scan using 1 microscan, 17,500 resolution, AGC target of 1E5, maximum injection time of 100 ms, and an isolation window of 1.7 m/z. Precursors were fragmented by HCD activation with a collision energy of 28%. MS/MS were collected on species with an intensity threshold of 1E4, charge states 2-6, and peptide match preferred. Dynamic exclusion was set to 20 seconds.

### Data Protocol
Tandem mass spectra were extracted by Proteome Discoverer software (version 2.2.0.388, Thermo Scientific) and searched in-house using the Mascot algorithm (version 2.6.1, Matrix Science). The data were searched against a Uniprot reference proteome for Danio rerio (46,927 sequences). Search parameters included trypsin digestion with up to 2 missed cleavages, peptide mass tolerance of 10 ppm, and MS/MS fragment tolerance of 0.02 Da. Cysteine carbamidomethylation and methionine oxidation were configured as variable modifications. Normal and decoy database searches were run, with the confidence level was set to 95% (p<0.05). Scaffold (version Scaffold_4.9.0, Proteome Software Inc., Portland, OR) was used to validate MS/MS based peptide and protein identifications. Peptide identifications were accepted if they could be established at greater than 95.0% probability by the Scaffold Local FDR algorithm. Protein identifications were accepted if they could be established at greater than 99.0% probability and contained at least 2 identified peptides. Protein probabilities were assigned by the Protein Prophet algorithm (Nesvizhskii et al., 2003). Proteins that contained similar peptides and could not be differentiated based on MS/MS analysis alone were grouped to satisfy the principles of parsimony. Proteins sharing significant peptide evidence were grouped into clusters. The cluster representative was used for further quantification. Label-free quantification of relative protein abundance was performed using intensity-based absolute quantification (iBAQ) (Hogl et al., 2013; Schwanhäusser et al., 2011) in Scaffold.

### Publication Abstract
None

### Keywords
Lc-msms, 12 somite stage, Cell adhesion, Integrin activation, Zebrafish, Extracellular matrix, Integrin a5

### Affiliations
Yale University
Department of Molecular, Cellular and Developmental Biology. Yale University, 260 Whitney Ave, New Haven, CT. 06520, USA

### Submitter
Scott Holley

### Lab Head
Dr Scott A. Holley
Department of Molecular, Cellular and Developmental Biology. Yale University, 260 Whitney Ave, New Haven, CT. 06520, USA


