### Accession
PXD016914

### Title
Design and experimental evaluation of a minimal, innocuous watermarking strategy to distinguish near-identical DNA and RNA sequences

### Description
F. Boonekamp et al., (2019) describe an innocuous watermarking strategy for coding regions that enables the discrimination of DNA and mRNA by sequencing through a k-mer approach and facilitates selective editing of watermarked and non-watermarked sequences. Shot-gun proteomics (LFQ) was performed on the parent and the engineered yeast strain.  *Corresponding author: Pascale Daran-Lapujade (p.a.s.daran-lapujade@tudelft.nl). Department of Biotechnology, Delft University of Technology, van der Maasweg 9, 2629HZ Delft, The Netherlands

### Sample Protocol
Yeast cell pellets were resupsended in lysis buffer composed of 100 mM TEAB containing 1% SDS and phosphatase/protease inhibitors. Yeast cells were lysed by glass bead milling. Proteins were reduced using DTT and alkylated by addition of acrylamide. Protein precipitation was performed by addition of four volumes of ice-cold acetone. The proteins were solubilized using 100 mM ammonium bicarbonate. Proteolytic digestion was performed by Trypsin (Promega, Madison, WI), and incubated at 37C over-night. Solid phase extraction was performed with an Oasis HLB 96-well elution plate to purify the peptides before injection to the LC-MS analysis system. Eluates were dried using a SpeedVac vacuum concentrator and peptides were resuspended in 3% ACN/0.01% TFA prior to MS-analysis to give an approximate concentration of 250 ng per µl. An aliquot corresponding to approx. 250 ng protein digest was analysed using an one dimensional shot-gun proteomics approach. Briefly, the samples were analysed using a nano-liquid-chromatography system consisting of an ESAY nano LC 1200, equipped with an Acclaim PepMap RSLC RP C18 separation column (50 µm x 150 mm, 2µm), and an QE plus Orbitrap mass spectrometer (Thermo). Solvent A was H2O containing 0.1% formic acid, and solvent B consisted of 80% acetonitrile in H2O and 0.1% formic acid. The Orbitrap was operated in data depended acquisition mode acquiring peptide signals from 385-1250 m/z at 70K resolution. The top 10 signals were isolated at a window of 2.0 m/z and fragmented using a NCE of 28. Fragments were acquired at 17K resolution.

### Data Protocol
Data were analysed against the proteome database from Saccharomyces cerevisiae (Uniprot, strain ATCC 204508 / S288C, Tax ID: 559292, July 2018) using PEAKS Studio X (Bioinformatics Solutions Inc) allowing for 20 ppm parent ion and 0.02 m/z fragment ion mass error, 2 missed cleavages, acrylamide as fixed and methionine oxidation and N/Q deamidation as variable modifications. Peptide spectrum matches were filtered against 1% false discovery rates (FDR) and identifications with ��������� 2 unique peptides. Changes in protein abundances between both strains, parent and engineered using the label free quantification (LFQ) option provided by the PEAKS Q software tool (Bioinformatics Solutions Inc). Protein areas were normalised to the total ion count (TIC) of the respective analysis run before performing pairwise comparison between the above mentioned strains. LFQ was performed using protein identifications containing at least 2 unique peptides, which peptide identifications were filtered against 1% FDR. The significance method for evaluating the observed abundance changes was set to ANOVA. The abundances of the glycolytic enzymes were further visualised as bar graphs using Matlab2018b. The area of the biological triplicates were averaged and standard deviations were represented as error bars.

### Publication Abstract
None

### Keywords
Yeast, Lfq, Proteomics

### Affiliations
TUD
Department of Biotechnology, Delft University of Technology, The Netherlands

### Submitter
Martin Pabst

### Lab Head
Dr Martin Pabst
Department of Biotechnology, Delft University of Technology, The Netherlands


