### Accession
PXD003441

### Title
Mouse retina proteomics -  Integrative subcellular proteomic analysis allows accurate prediction of human disease causing genes

### Description
Proteomic experiments were performed on the outer segment (OS) and the remaining retina (RR). For the OS proteomics, three biological replicates were performed for mouse at the age of P15, which included 24, 26 and 26 retinas respectively. We also performed the OS proteomic experiment on 40 retinas from mice at age P13. For the RR proteomic experiments, two biological replicates were performed independently using 30 and 30 retinas respectively.

### Sample Protocol
Fresh mouse retinas were dissected and collected in sample buffer containing 24% (w/v) sucrose, 10mM MOPS pH7.4, 30mM NaCl, 60mM KCl, and 2mM MgCl2. The OS was isolated using mechanical shearing to break the CC and separated from remaining retinal components via centrifugation at 4000rpm for 1min. Crude OS-containing solution was subjected to sucrose gradient ultracentrifugation and the fraction containing purified OSs was collected. After the OS isolation, the remaining fractions were gathered as the RR and treated in the same way for downstream MS analysis. 30Âµg of total protein was extracted from both the OS fraction and the RR.  Extracted protein was resolved by SDS-PAGE and  in-gel trypsin digestion overnight. Peptides were extracted twice with 100% acetonitrile and dried completely in a vacuum concentrator. MS experiments were performed with Q Exactive Hybrid Quadrupole-Orbitrap Mass Spectrometer.

### Data Protocol
The database used for protein identification was the mouse reference sequence protein database (Refseq) from NCBI, and duplicate entries were combined before database searching. The maximum missed cleavage site was set to one and limited to trypsin cleavage sites. The precursor ion mass error tolerance was set to 20ppm and the fragment ion mass tolerance used was 0.5Da. Both static and dynamic modifications were considered. The maximum variable PTMs per peptide was set to three.

### Publication Abstract
Proteomic profiling on subcellular fractions provides invaluable information regarding both protein abundance and subcellular localization. When integrated with other data sets, it can greatly enhance our ability to predict gene function genome-wide. In this study, we performed a comprehensive proteomic analysis on the light-sensing compartment of photoreceptors called the outer segment (OS). By comparing with the protein profile obtained from the retina tissue depleted of OS, an enrichment score for each protein is calculated to quantify protein subcellular localization, and 84% accuracy is achieved compared with experimental data. By integrating the protein OS enrichment score, the protein abundance, and the retina transcriptome, the probability of a gene playing an essential function in photoreceptor cells is derived with high specificity and sensitivity. As a result, a list of genes that will likely result in human retinal disease when mutated was identified and validated by previous literature and/or animal model studies. Therefore, this new methodology demonstrates the synergy of combining subcellular fractionation proteomics with other omics data sets and is generally applicable to other tissues and diseases.

### Keywords
Outer segment, Mouse retina, Photoreceptor

### Affiliations
Baylor College of Medicine
HGSC, Baylor College of Medicine

### Submitter
Li Zhao

### Lab Head
Dr Rui Chen
HGSC, Baylor College of Medicine


