### Accession
PXD019514

### Title
Distinct and stage-specific contributions of TET1 and TET2 to stepwise cytosine oxidation in the transition from naive to primed pluripotency

### Description
Cytosine DNA bases can be methylated by DNA methyltransferases and subsequently oxidized by TET proteins. The resulting 5-hydroxymethylcytosine (5hmC), 5-formylcytosine (5fC), and 5-carboxylcytosine (5caC) are considered demethylation intermediates as well as stable epigenetic marks. To dissect the contribution of these cytosine modifying enzymes, we generated combinations of Tet knockout (KO) embryonic stem cells (ESCs) and systematically measured protein and DNA modification levels at the transition from naive to primed pluripotency. Whereas the increase of genomic 5-methylcytosine (5mC) levels during exit from pluripotency correlated with an upregulation of the de novo DNA methyltransferases DNMT3A and DNMT3B, the subsequent oxidation steps turned out to be far more complex. The strong increase of oxidized cytosine bases (5hmC, 5fC, and 5caC) was accompanied by a drop in TET2 levels, yet the analysis of KO cells suggested that TET2 is responsible for most 5fC formation. The comparison of modified cytosine and enzyme levels in Tet KO cells revealed distinct and differentiation-dependent contributions of TET1 and TET2 to 5hmC and 5fC formation arguing against a processive mechanism of 5mC oxidation. The apparent independent steps of 5hmC and 5fC formation suggest yet to be identified mechanisms regulating TET activity and may constitute another layer of epigenetic regulation.

### Sample Protocol
For full proteome measurements flash-frozen cells were lysed in 200 µL of the lysis buffer (6 M Guanidinium Chloride, 100 mM Tris-HCl pH 8.5 and freshly added 2 mM DTT). By thoroughly pipetting, samples were homogenized and subsequently boiled at 99°C for 10 min in a thermal shaker at 1700 rpm. To get rid of bubbles and to collect the evaporated liquid, samples were quickly spun down. After sonication for 15 min (30 s on/off interval, Bioruptor Plus by Diagenode) protein concentrations were estimated by a BCA assay in a TECAN reader. Chloroacetamide was added to the samples (40 mM final concentration) and samples were incubated at room temperature for 20 min. For the protein digestion, 30 µg of the lysate was diluted in 30 µL of the lysis buffer already including 2 mM DTT and 40 mM CAA. Then, samples were diluted 1:10 in the digestion buffer (25mM Tris-HCl pH 8.5 and 10% acetonitrile) containing trypsin and LysC in a 1:50 protease to protein ratio. Digestion was performed overnight at 37°C and 100 rpm. After acidifying samples with 1% trifluoroacetic acid (TFA), peptides were cleaned up on three layers of SDB-RPS matrix (Rappsilber et al. 2007). Eluted and speedvac dried peptides were resuspended in 20 µL of A* buffer (0.1% TFA and 2% acetonitrile) and peptide concentrations were estimated by nanodrop at 280 nm.  Full proteome measurements based on data-independent acquisition method Mass spectrometric analysis of peptides was performed on a quadrupole Orbitrap mass spectrometer (Q Exactive HF-X, ThermoFisher Scientific, Bremen, Germany) after prior nanoflow liquid chromatography on an Easy-nLC 1200 (ThermoFisher Scientific). The injection was mediated under high-pressure conditions via a nano-electrospray ion source. For this purpose, in-house packed 50 cm columns of ReproSil-Pur C18-AQ 1.9-µm resin (Dr. Maisch GmbH) were used to elute approximately 400 ng peptides of each sample in an acetonitrile gradient for 120 min. The flow rate was kept constantly at around 300 nL/min and the column oven temperature at 60°C.  The peptides were analyzed following a data-independent acquisition (DIA) method (MS1 scan: resolution 60,000, 300 to 1650 m/z, maximum injection time 60 ms and AGC target 3E6, MS2 scan: resolution 30,000, 32 segments at varying isolation windows ranging from 14,4 m/z to 562,8 m/z, maximum injection time 54 ms and AGC target 3E6). For MS2 scans the default charge state was set to 2. The stepped normalized collision energy was set to 25, 27.5 and 30.

### Data Protocol
The DIA raw files were analyzed with the Spectronaut Pulsar X software package (Biognosys, version 13.15.200430.43655) applying the default Biognosys factory settings for DIA analysis. To get a deeper proteome a hybrid spectral library strategy (Muntel et al. 2019) was followed using the DIA measurements as a project-specific library harboring 55,697 precursors (4,108 protein groups) and a ESC/EpiLC-specific Data-dependent acquisition (DDA) library with in total 230,581 precursors and 9,158 protein groups.

### Publication Abstract
Cytosine DNA bases can be methylated by DNA methyltransferases and subsequently oxidized by TET proteins. The resulting 5-hydroxymethylcytosine (5hmC), 5-formylcytosine (5fC), and 5-carboxylcytosine (5caC) are considered demethylation intermediates as well as stable epigenetic marks. To dissect the contributions of these cytosine modifying enzymes, we generated combinations of Tet knockout (KO) embryonic stem cells (ESCs) and systematically measured protein and DNA modification levels at the transition from naive to primed pluripotency. Whereas the increase of genomic 5-methylcytosine (5mC) levels during exit from pluripotency correlated with an upregulation of the de novo DNA methyltransferases DNMT3A and DNMT3B, the subsequent oxidation steps turned out to be far more complex. The strong increase of oxidized cytosine bases (5hmC, 5fC, and 5caC) was accompanied by a drop in TET2 levels, yet the analysis of KO cells suggested that TET2 is responsible for most 5fC formation. The comparison of modified cytosine and enzyme levels in Tet KO cells revealed distinct and differentiation-dependent contributions of TET1 and TET2 to 5hmC and 5fC formation arguing against a processive mechanism of 5mC oxidation. The apparent independent steps of 5hmC and 5fC formation suggest yet to be identified mechanisms regulating TET activity that may constitute another layer of epigenetic regulation.

### Keywords
Dna methylation, Epiblast-like cells, Epigenetics, Mouse embryonic stem cells

### Affiliations
Faculty of Biology and Center for Molecular Biosystems (BioSysM), Human Biology and BioImaging, LMU Munich & Department of Proteomics and Signal Transduction, Max-Planck Institute of Biochemistry
Department of Biology II and Center for Integrated Protein Science Munich (CIPSM), Ludwig-Maximilians-Universität München, Planegg-Martinsried, Germany

### Submitter
Enes Ugur

### Lab Head
Dr Sebastian Bultmann
Department of Biology II and Center for Integrated Protein Science Munich (CIPSM), Ludwig-Maximilians-Universität München, Planegg-Martinsried, Germany


