### Accession
PXD005111

### Title
Proteome analysis of Jurkat cells for spectral library

### Description
In order to increase the spectral library to study the ocurrence of distinct peptide fragmetation patterns, we analyzed the whole cell proteome of Jurkat cells.

### Sample Protocol
Jurkat cells were lysed and proteins were isolated according to the FASP protocol, published by Wisniewski et al. Following tryptic digestion, peptide mixtures were desalted prior to LC-MS/MS analysis. Peptides were separated on a 50 cm nano-reversed phase C18 column (Thermo-Fisher, Acclaim PepMapC18, 500x0.075 mmm x 2.4 um 100A) using an Ultimate 3000 nano-chromatography system. For peptide separation, a linear gradient from 4-32% acetonitrile in 109 min was applied in direct injection mode. Peptides eluting from the column were directly infused into the QTOF mass spectrometer 6600 TripleTOF (Sciex) via a nano-ESI emitter tip (New Objective). Peptides ions were analyzed employing a information-dependent acquisition method in positive ionization mode consiting of one survey scan and up to 30 MS/MS scans. Precusors for MS/MS analysis with CID fragmentation had to have a signal intensity of 120 cps and a charge state from 2+ to 5+. To reduce repettive analysis of strong precusro ions, dynamic exclusion of previously selected ions was enabled for 50 sec and with a mass accuracy of 20 ppm around the precursor and up to 4 signals of the isotopic pattern.

### Data Protocol
Proteomics raw data were searched against a combined forward/reversed database of human protein sequences (Uniprot 05/2015) using the Andromeda algorithm within MaxQuant (vs. 1.5.3.12). As post-translational modificatons carbamidomethylation of cysteine (fixed), methionine oxidation and protein N-terminal acetylation as variable modfications were enabled for the database search with a maximum of 3 modifications per peptide. Peptide spectrum matches and protein groups were both filtered for 1% false discovery rate with a minimal peptide length of 7 amino acids. peptide hits were further filtered for a minimal score of 10 in case of unmodified peptides and 30 in case of modified peptides.

### Publication Abstract
Spectral libraries play a central role in the analysis of data-independent-acquisition (DIA) proteomics experiments. A main assumption in current spectral library tools is that a single characteristic intensity pattern (CIP) suffices to describe the fragmentation of a peptide in a particular charge state (peptide charge pair). However, we find that this is often not the case. We carry out a systematic evaluation of spectral variability over public repositories and in-house data sets. We show that spectral variability is widespread and partly occurs under fixed experimental conditions. Using clustering of preprocessed spectra, we derive a limited number of multiple characteristic intensity patterns (MCIPs) for each peptide charge pair, which allow almost complete coverage of our heterogeneous data set without affecting the false discovery rate. We show that a MCIP library derived from public repositories performs in most cases similar to a "custom-made" spectral library, which has been acquired under identical experimental conditions as the query spectra. We apply the MCIP approach to a DIA data set and observe a significant increase in peptide recognition. We propose the MCIP approach as an easy-to-implement addition to current spectral library search engines and as a new way to utilize the data stored in spectral repositories.

### Keywords
Whole cell lysate, Spectral library, Fragmentation pattern

### Affiliations
CECAD, University of Cologne, AG Kr√ºger
Protein Analysis Unit ZfP LMU Munich BioMedical Center Grosshaderner Strasse 9 82152 Planegg-Martinsried Tel. +49 89 2180 71804

### Submitter
Andreas Schmidt

### Lab Head
Dr Axel Imhof
Protein Analysis Unit ZfP LMU Munich BioMedical Center Grosshaderner Strasse 9 82152 Planegg-Martinsried Tel. +49 89 2180 71804


