### Accession
PXD043345

### Title
Western diet triggers cardiac dysfunction in heterozygous Mybpc3-targeted knock-in hypertrophic cardiomyopathy mice

### Description
The study investigates the effect of a heterozygous MYBPC3 mutation combined with Western diet feeding on protein expression in a HCM mouse model. Hypertrophic cardiomyopathy (HCM) is the most common inherited genetic heart disease, characterized by asymmetric left ventricular hypertrophy and diastolic dysfunction. It is generally accepted that HCM is caused by mutations in genes encoding sarcomere proteins (e.g. MYBPC3, MYH7). However, not all mutation carriers will develop HCM and, moreover, phenotypical variation in terms of clinical presentation is large. The phenotypical expression of HCM thus requires additional disease modifiers on top of a sarcomere gene mutation. Clinical reports identify obesity and related comorbidities (diabetes, hyperlipidemia, hypertension) as a major factor contributing to disease expression and severity. We aimed to mimic this situation in a HCM mouse model by feeding a Western diet (8 weeks) to mice carrying a heterozygous mutation in MYBPC3. These heterozygous mice are phenotype negative under normal conditions, but developed cardiac dysfunction and hypertrophy upon Western diet feeding. The main goal of this proteomic screening is to identify clusters of proteins and pathways that are down/upregulated specifically in the left ventricle of mice suffering from a mutation and Western diet feeding. Also, we are interested in protein expression differences caused solely by mutation or Western diet. Group 1 are Wild type mice that received normal chow; group 2 are heterozygous mice that received normal chow; group 3 are Wild type mice that received a Western diet; group 4 are heterozygous mice that received a Western diet.

### Sample Protocol
Frozen LV tissue was pulverized and homogenized using a glass tissue grinder in 40 ÂµL reducing sample buffer (106 mM Tris-HCl, 141 mM Tris-base, 0.51 mM ethylenediaminetetraacetic acid, 0.22 mM Coomassie Brilliant Blue G250, 0.18 mM phenol red, 100 mM dithiothreitol, 2% (wt/vol) lithium dodecyl sulfate, 10% (vol/vol) glycerol) per mg of tissue. Proteins were denatured at 99 Â°C for 5 min, sonicated using a sonifier (Branson; 3x 10s pulses, 70% intensity) and heated again at 99 Â°C for 5 min. Samples were centrifuged at maximum speed for 10 min in a microcentrifuge. Supernatants were collected and stored at -80 Â°C. To separate proteins, 30 ÂµL of homogenate per sample (circa 30 Âµg protein) was loaded on 4-12% precast NuPAGE Bis-Tris mini gels (Invitrogen) followed by electrophoresis at 200 V in NuPAGE MES SDS running buffer until the dye reached the bottom of the gel. Gels were fixed in fixing solution (50% ethanol, 3% phosphoric acid). Proteins were stained using 0.1% Coomassie Brilliant Blue G-250 solution (1 g/L Coomassie Brilliant Blue G-250 in 34% methanol, 3% phosphoric acid, 15% ammonium sulfate).  Each gel lane was separated into 5 slices, each roughly containing an equal amount of protein. Samples were measured by liquid chromatography-mass spectrometry (LC-MS) on individual gel slice extracts. Gel slice extracts were measured group-wise, starting with the top gel slice extracts with highest molecular weight range, and ending with the bottom gel slice extracts with lowest molecular weight range. Peptides were separated using an Ultimate 3000 Nano LC-MS/MS system (Dionex LC-Packings) equipped with a 40 cm Ã— 75 Î¼m ID fused-silica column custom packed with 1.9 Î¼m,120 Ã… ReproSil Pur C18 aqua (Dr Maisch GMBH). Eluting peptides were ionized at a potential of +2 kV into a Q Exactive mass spectrometer (Thermo Fisher). MS/MS spectra were acquired at resolution 17500 (at m/z 200) in the orbitrap using an AGC target value of 1Ã—106 charges, a maxIT of 60 ms, and an underfill ratio of 0.1%. Dynamic exclusion was applied using a repeat count of 1 and an exclusion time of 30s.

### Data Protocol
MS/MS spectra were searched against a Uniprot mouse reference proteome FASTA file (2018_02, canonical and isoforms, 25131 entries) supplemented with FASTA data for retention time standards (Biognosis iRT kit) using MaxQuant version 1.5.4.1. Both peptide and protein identifications were filtered at a false discover rate (FDR) 0.01 using the target-decoy strategy (default in MaxQuant). Quantification of proteins was done using spectral counting.

### Publication Abstract
None

### Keywords
Cardiac metabolism, Pathophysiology, Sarcomere mutation, Metabolic health, Hypertrophic cardiomyopathy

### Affiliations
Amsterdam UMC, Vrije Universiteit Amsterdam, Medical Oncology, Cancer Center Amsterdam, OncoProteomics Laboratory, Amsterdam, Netherlands
Amsterdam UMC

### Submitter
Thang Pham

### Lab Head
Dr Connie Jimenez
Amsterdam UMC, Vrije Universiteit Amsterdam, Medical Oncology, Cancer Center Amsterdam, OncoProteomics Laboratory, Amsterdam, Netherlands


