### Accession
PXD037391

### Title
Oxidized thioredoxin-1 restrains the NLRP1 inflammasome

### Description
To find cellular protein binding partners of the NLRP1 inflammasome, FLAG-tagged full length NLRP1 (or a GFP-FLAG protein control) was ectopically expressed in HEK 293T cells and affinity purified from cell lysates (Note: the NLRP1-FLAG-containing lysate samples were affinity purified in the presence or absence of excess proline) before proteins were reduced, alkylated, and trypsinized before TMT labelling and pooling of the differentially tested lysates to give three conditions (1. NLRP1-FLAG; 2. GFP-FLAG; 3. NLRP1-FLAG+proline) in duplicate. These data represent the results following sample and data processing.

### Sample Protocol
3x106 ﻿HEK 293T cells were seeded in a 10 cm tissue culture dish and transfected the following day with an expression plasmid for the given FLAG-tagged protein construct. The cells were harvested 48 h post transfection and lysed in 1 mL of 0.5% NP-40 in PBS on ice for 30 min. Lysate samples were placed in a tabletop centrifuge for rotation at 2000 g for 10 min at 4˚C. The soluble fraction was separated, and protein content normalized (DCA protein assay, BIO-RAD). Protein samples were globally reduced by application of 10 mM TCEP (tris(2-carboxyethyl) phosphine) for 1h at 56˚C with shaking, followed by alkylation of free thiols with 20 mM iodoacetamide for 30 min at ambient temperature, protected from light. Protein samples were then precipitated from solution by adding -20˚C acetone to six sample volumes and incubated overnight at -20˚C. Precipitated protein was isolated by sample rotation at 8000 g for 10 min at 4˚C and decanting the liquid phase. The pellet samples were resuspended in in 100 µL 50 mM triethylammonium bicarbonate (TEAB) followed by total protein quantification by DCA assay prior to digestion with 2.5 mg trypsin per 100 mg sample at 37˚C overnight.    TMTsixplex sample reagents (ThermoFisher Scientific) were equilibrated to ambient temperature and 0.8 mg of label (126, 127 = GFP-FLAG; 128, 129 = NLRP1-FLAG; 130, 131 = unused sample) diluted into 41 µL pure acetonitrile before use. 41 µL of each TMT label reagent were added to each protein sample and incubated at ambient temperature for 1 h. 8 µL of a 5% hydroxylamine solution was added to quench each sample and incubated at ambient temperature for 15 min. Samples were then equally combined and purified by a High pH Reversed-Phase Peptide Fractionation Kit (Pierce) and dried with a Genevac EZ-2 evaporator to yield peptide pellets submitted for tandem LC-MS/MS/MS analysis. Mass spectrometry data was collected on an Orbitrap Fusion Lumos mass spectrometer coupled to an Easy-nLC 1200 (ThermoFisher Scientific). Peptides were separated over a 200 min gradient of 0-50% acetonitrile in water with 0.1 % formic acid at a flow of 300 nL/min on a 50 cm PepMap RSLC C18 column (2 mm, 100 Å, 75 µm x 50 cm). The full MS spectrum was acquired in the Orbitrap at a resolution of 120,000. The 10 most intense MS1 ions were selected for MS2 analysis. The isolation width was set at 0.7 m/z and isolated precursors were fragmented by CID (35% CE). Following acquisition of each MS2 spectrum, a synchronous precursor selection (SPS) MS3 scan was collected on the top 10 most intense ions in the MS2 spectrum. The isolation width was set at 1.2 m/z and isolated precursors were garmented using HCD.

### Data Protocol
MS raw files were converted to MGF and processed using Proteome Discoverer version 2.2.0.388 (Thermo Scientific) by searching against the Uniprot human database supplemented with common contaminant protein sequences and quantified according to SPS MS3 reporter ions. Identification was based on CID spectra using SequestHT. Search criteria included: 20 ppm mass tolerance for MS spectra, 0.5 Da fragment mass tolerance for MS/MS spectra, a maximum of two allowed missed cleavages, static carbamidomethylation (+57.021 Da) of cysteine and TMTsixplex (+229.163 Da) of lysine and of peptide N-terminus, dynamic modifications of methionine oxidation (+15.995 Da), N-terminal protein acetylation (+42.011 Da), asparagine or glutamine deamidation (+0.984 Da), and serine, tyrosine, or tryptophan phosphorylation (+79.966 Da) and a false discovery rate of < 0.01.

### Publication Abstract
The danger signals that activate the NLRP1 inflammasome have not been established. Here, we report that the oxidized, but not the reduced, form of thioredoxin-1 (TRX1) binds to NLRP1. We found that oxidized TRX1 associates with the NACHT-LRR region of NLRP1 in an ATP-dependent process, forming a stable complex that restrains inflammasome activation. Consistent with these findings, patient-derived and ATPase-inactivating mutations in the NACHT-LRR region that cause hyperactive inflammasome formation interfere with TRX1 binding. Overall, this work strongly suggests that reductive stress, the cellular perturbation that will eliminate oxidized TRX1 and abrogate the TRX1-NLRP1 interaction, is a danger signal that activates the NLRP1 inflammasome.

### Keywords
Chemical biology, Innate immunity, Programmed cell death, Inflammasome, Pattern recognition receptors

### Affiliations
Department of Chemical Biology Memorial Sloan Kettering Cancer Center New York, NY
Memorial Sloan Kettering Cancer Center

### Submitter
Daniel Ball

### Lab Head
Dr Daniel A. Bachovchin
Department of Chemical Biology Memorial Sloan Kettering Cancer Center New York, NY


