### Accession
PXD005931

### Title
Large scale intact glycopeptide identification by Mascot database search

### Description
We present the adaptability of Mascot search engine for automated identification of intact glycopeptide mass spectra. The steps involved in adopting Mascot for intact glycopeptide analysis include: i) assigning unique one letter codes for monosaccharides, ii) linearizing glycan sequences and iii) preparing custom glycoprotein databases. Stepped normalized collision energy (NCE) for HCD mostly provided both the peptide and glycan information in a single MS2 spectrum. Using standard glycoproteins, we showed that Mascot can be adopted for automated annotation of both N- and O-linked glycopeptides. In a large scale validation study, a total of 257 glycoproteins containing 970 unique glycosylation sites and 3447 non-redundant N-linked glycopeptide variants were identified in serum samples. This represent a single tool that collectively allows the i) elucidation of N- and O-linked glycopeptide spectra, ii) matching glycopeptides to known protein sequences, and iii) high-throughput, batch wise analysis of large scale glycoproteomics data sets.

### Sample Protocol
Serum samples from healthy individuals and prostate cancer patients were digested with trypsin, desalted using ZIC-HILIC SPE catridges. The sialylated glycopeptides were then enriched with TiO2 beads. The intact glycopeptides were then analyzed by LC-MS on a QExactive mass spectrometer.The mass spectrometer was operated in the data-dependent mode to automatically switch between MS and MS/MS acquisition. Survey full scan MS spectra (from m/z 400 to 2000) were acquired with the resolution R = 70,000 at m/z 200, after accumulation to a target of 1e6. The maximum allowed ion accumulation times were 100 ms. The sequential isolation of up to the seven most intense ions, depending on signal intensity (intensity threshold 5.6e3) were considered for fragmentation using higher-energy collision induced dissociation (HCD) at a target value of 100,000 charges and a resolution R = 17,500 with stepped NCE 15, 25 and 35. Target ions already selected for MS/MS were dynamically excluded for 30 sec. The isolation window was m/z = 2 without offset. The maximum allowed ion accumulation for the MS/MS spectrum was 180 ms. For accurate mass measurements, the lock mass option was enabled in MS mode for internal recalibration during the analysis.

### Data Protocol
The raw LC-MS data sets from serum samples were loaded to Mascot Distiller (Matrix Science, London, UK, version: 2.6.1.0) and corresponding peak lists in Mascot generic format (mgf) were generated using the peak processing and peak picking algorithms incorporated in Mascot Distiller. The peak lists were then searched against the custom N-linked glycoprotein database (406 glycoprotein entries) using Mascot search engine (Matrix Science, London, UK, version: 2.4).  The following search parameters were used; enzyme: trypsin, maximum missed cleavage sites: 0, precursor ion mass tolerance: 10 ppm, fragment ion tolerance: 0.05 Da, fixed modification: carbamidomethylation on cysteine, variable modifications: fucose addition on HexNAc residues, oxidation on methionine residues. After the database search, using Mascot’s standard protein grouping algorithm, glycopeptide identifications with a Mascot ion score of 25, a top scoring match to a particular spectrum and a significance threshold p-value ˂ 0.001 were only considered as positive hits and further relatively quantifed with Mascot Distiller.

### Publication Abstract
Workflows capable of determining glycopeptides in large-scale are missing in the field of glycoproteomics. We present an approach for automated annotation of intact glycopeptide mass spectra. The steps in adopting the Mascot search engine for intact glycopeptide analysis included: (i) assigning one letter codes for monosaccharides, (ii) linearizing glycan sequences and (iii) preparing custom glycoprotein databases. Automated annotation of both N- and O-linked glycopeptides was proven using standard glycoproteins. In a large-scale study, a total of 257 glycoproteins containing 970 unique glycosylation sites and 3447 non-redundant N-linked glycopeptide variants were identified in 24 serum samples. Thus, a single tool was developed that collectively allows the (i) elucidation of N- and O-linked glycopeptide spectra, (ii) matching glycopeptides to known protein sequences, and (iii) high-throughput, batch-wise analysis of large-scale glycoproteomics data sets.

### Keywords
Glycoproteomics, Mascot, Serum, Prostate cancer, Higher-energy collisional induced dissociation (hcd)

### Affiliations
University of Oslo, Department of Biosciences
University of Oslo

### Submitter
Ravi Chand Bollineni

### Lab Head
Dr Bernd Thiede
University of Oslo, Department of Biosciences


