### Accession
PXD019285

### Title
Interferon-induced degradation of the hepatitis B virus persistence form, cccDNA, depends on ISG20

### Description
Hepatitis B virus (HBV) persists by depositing a covalently closed circular DNA (cccDNA) in the nucleus of infected cells that cannot be targeted by available antivirals. Cytokine treatments can diminish HBV cccDNA via APOBEC3-mediated deamination. Here we show that overexpression of APOBEC3A alone, however, was not sufficient to reduce cccDNA in HBV-infected cells. This required addition of interferon indicating that cccDNA degradation requires an additional, interferon-stimulated gene (ISG). Transcriptome analyses identified ISG20 as the only type I and II interferon-induced, nucleus-resident protein with annotated nuclease activity. ISG20 expression was detected in human livers in acute, self-limiting but not in chronic hepatitis B. ISG20 depletion abolished the interferon-induced loss of cccDNA, and co-expression of ISG20 and APOBEC3A was sufficient to diminish cccDNA. In conclusion, non-cytolytic HBV cccDNA decline requires induction of a deaminase and nuclease. Our findings highlight that ISGs cooperate for their antiviral function and this cooperativity may be explored for therapeutic targeting.

### Sample Protocol
HBV-replicating HepG2H1.3 cells were differentiated for 10 days (DMEM medium, 1 % FCS, 100 U/ml penicillin/streptomycin, 2 mM glutamin, 1 x non-essential amino acids, 1 mM sodium pyruvate, 1.74% DMSO) and treated for 1 day with 1000 U/ml IFN. PBS-washed cells were harvested with a cell scraper and pellets were snap frozen in liquid nitrogen. DNA oligomers were synthesized by Microsynth AG: (-)ssDNA (033): Biotin-GGTCGTCCGCAGGATTCAGCGCCG; dU-containing (-)ssDNA (034): Biotin-GGTCGTUUGCAGGATTUAGUGCCG; (+)ssDNA (036): CGGCGCTGAATCCTGCGGACGACC. Oligomers were denatured (99°C; 10 min) and chilled on ice (single-stranded versions) or mixed in a 1:1-ratio, denatured and slowly cooled down at room temperature (double-stranded versions). StrepTactin sepharose slurry (IBA) was washed three times and reconstituted in TAP buffer (50 mM Tris/HCl pH7.5, 5% glycerol, 0.2% NP-40, 1.5 mM MgCl2, 100 mM NaCl) in 10x volume. Per sample, 1 µg ssDNA or 2 µg dsDNA were incubated  with 100 µl reconstituted StrepTactin beads at 4°C for 1 hour. Cell pellets were lysed in TAP buffer including protease inhibitors on ice for 20 min and centrifuged thereafter (12000 rpm, 15 min, 4°C). Supernatants were then cleared by incubation with washed and reconstituted StrepTactin beads (3.3x volume TAP buffer) in a 10:1-ratio (15 min, 4°C, rotary wheel) and centrifugation. Protein concentration was determined by absorbance at 280 nm and Pierce 660 nm Protein Assay (Thermo Scientific). ssDNA- or dsDNA-bound beads were washed three times with TAP buffer, resuspended in TAP buffer including protease inhibitors and incubated with 2 mg protein lysate per sample (1 hour, 4°C). After washing in TAP buffer and TAP buffer without NP-40, beads were harvested (2000 rcf, 1 min). Enriched proteins on agarose beads were denatured, reduced and alkylated by sequential addition of 40 µl U/T buffer (6 M urea, 2 M thiourea, 10 mM HEPES, pH 8.0), 10 mM DTT (30 min) and 55 mM IAA (20 min, in the dark). For protein digestion, 160 µl digestion buffer (50 mM ammonium bicarbonate, pH 8.0) containing 0.5 µg LysC (FUJIFILM Wako Pure Chemical Corporation) and 0.5 µg Trypsin (Progema) was added to the beads and incubated at 25°C overnight. Peptides were desalted and concentrated using three layers of C18 StageTips (3M) as described previously. Purified peptides were loaded onto a 50 cm reverse-phase analytical column (75 µm diameter; ReproSil-Pur C18-AQ 1.9 µm resin; Dr. Maisch GmbH) and separated using an EASY-nLC 1200 system (Thermo Fisher Scientific) with a binary buffer system of buffer a (0.1% formic acid), buffer b (80% ACN, 0.1% formic acid) and a 120 min gradient (5% to 30% buffer b for 90 min, 30% to 95% for 20 min, wash-out at 95% for 5 min, readjustment to 5% in 5 min) at a flow rate of 300 nl per min. Eluting peptides were directly analyzed on a Q-Exactive HF mass spectrometer (Thermo Fisher Scientific). Data-dependent acquisition included repeating cycles of one MS1 full scan (300–1,600 m/z, resolution of 60,000) at an ion target of 3×106, followed by 15 MS2 scans of the highest abundant isolated and higher-energy collisional dissociation (HCD) fragmented peptide precursors (resolution of 15,000). For MS2 scans, collection of isolated peptide precursors was limited by an ion target of 1×105 and a maximum injection time of 120 ms. Isolation and fragmentation of the same peptide precursor was eliminated by dynamic exclusion for 20 s. Raw files were processed with MaxQuant (version 1.6.10.43; Cox Lab, Max-Planck-Institute of Biochemistry) using the standard settings, label-free quantification (iBAQ; LFQ, LFQ min ratio count 1, normalization type none, stabilize large LFQ ratios disabled) and match between run options enabled. Spectra were searched against forward and reverse sequences of the reviewed human proteome including isoforms (UniprotKB, release 10.2019) and HBV proteins by the built-in Andromeda search engine.

### Data Protocol
The output of MaxQuant was analysed with Perseus (version 1.6.10.43, [https://www.ncbi.nlm.nih.gov/pubmed/?term=27348712]), R (version 3.6.0) and RStudio (version 1.2.1335). Detected protein groups identified as known contaminants, reverse sequence matches or only identified by site as well as replicate samples with poor quality were excluded from the analysis. Following log2 transformation, iBAQ intensities were normalized to correct for technical variation by using the intensity of unspecifically bound proteins that were quantified in all samples. Following normalization, proteins without quantification in at least three replicates of one condition were removed and missing values were imputed for each replicate individually by sampling values from a normal distribution calculated from the original data distribution (width = 0.3×s.d., downshift = -1.8×s.d.). Differentially expressed protein groups were identified via two-sided Welch’s t-tests (S0 = 1) corrected for multiple hypothesis testing applying a permutation-based FDR (FDR < 0.01, 250 randomizations).

### Publication Abstract
None

### Keywords
Isg20, Interferon-gamma, Cccdna, Chronic hepatitis b, Interferon-alpha, Apobec3a, Hbv

### Affiliations
Institute of Virology, Technical University Munich

### Submitter
Christian Urban

### Lab Head
Dr Andreas Pichlmair
Institute of Virology, Technical University Munich


