### Accession
PXD037563

### Title
The proteome of B cell maturation identifies PDCD4 as limiting the response of marginal zone B cells – marginal zone B cells WT and PDCD4 KO

### Description
During B cell maturation, transitional and mature B cells acquire cell-intrinsic features that determine their ability to exit quiescence and mount effective immune responses. We used high-resolution mass spectrometry to quantify the proteome of B cell subsets from the mouse spleen and map the differential expression of environmental sensing, transcription- and translation initiation-factors that define cellular identity and function. By comparing transcriptome and proteome, we identified mRNAs linked to B cell activation and differentiation that are expressed without detectable protein. These "poised" mRNAs might enable rapid protein production through increased translation or protein stability. In addition, we found that the translational repressor PDCD4 restrains the response of marginal zone B cells to a T-independent antigen. Our molecular characterization of B cell maturation is a valuable resource to further explore the mechanisms underpinning the specialised functions of B cell subsets.

### Sample Protocol
Sample preparation and mass spectrometry Samples were generated from 1 x10E6 FACS-sorted WT and PDCD4 KO MZ B cells from the spleen and derived from competitive bone marrow (BM) chimeras. MZ B cells were identified as CD19+ CD93- CD21+ CD23- B cells. For generation of competitive BM chimeras, male B6.SJL-PtprcaPepcb/BoyJ (B6.SJL-CD45.1) mice were lethally irradiated (2x 5.0 Gy) and reconstituted with 3x10E6 BM cells, which consisted in a mixture of 40% B6.SJL-CD45.1 and 60% C57BL/6 (WT) or B6.129S6-Pdcd4tm1Yhcn/J (PDCD4 KO) CD45.2 cells.  FACS-sorted B cells were washed twice with ice-cold PBS and cell pellets were snap frozen in liquid nitrogen. Cells were lysed in 5% sodium dodecyl sulphate, 50mM TEAB pH 8.5, 10mM TCEP under agitation. Lysates were boiled for 5min at 95°C, sonicated with a BioRuptor (15 cycles of 30sec each) and treated with 1l benzonase for 15 minutes at 37°C. Protein yield was determined using the EZQ protein quantitation it (ThermoFisher Scientific) according to manufacturer’s instructions. Lysates were then alkylated with 20mM iodoacetamide for 1h at RT in the dark and loaded onto S-Trap mini columns (ProtiFi). Proteins were subsequently digested with 15g Trypsin Gold (Promega) in 50mM ammonium bicarbonate (Sigma-Aldrich) for 1.5h at 47°C before peptides were eluted from columns. Eluted peptides were dried by SpeedVac and resuspended in 5% formic acid for peptide fractionation by high pH reverse-phase chromatography.  Peptides were fractionated by HPLC using a Dionex Ultimate3000 system (ThermoFisher Scientific), which consists in a 25min-multitep gradient of buffer A (10mM ammonium formate at pH 9 in 2% acetonitrile) and buffer B (10mM ammonium formate at pH 9 in 80% acetonitrile), at a flow rate of 0.3ml/min. Peptides were separated in 16 fractions, which were then consolidated in 8 fractions. The fractions were subsequently dried by SpeedVac and dissolved in 5% formic acid. 1g was analysed for each fraction using a nanoscale C18 reverse-phase chromatography system (UltiMate 3000 RSLC nano, Thermo Scientific) coupled to an Orbitrap Q Exactive Plus mass spectrometer (Thermo Scientific), as described previously (Marchingo et al., Elife 2020).

### Data Protocol
Data analysis The raw mass spectrometry data were processed with the MaxQuant software package (version 2.0.1.0). Proteins and peptides were identified using a hybrid database generated from the UniProt mouse database (June 2020). This hybrid protein database consisted of manually annotated mouse SwissProt entries, along with mouse TrEMBL entries with a manually annotated homologue within the human SwissProt database. The following variable modifications were set within MaxQuant: methionine oxidation, acetylation (N-termini), glutamine to pyroglutamate and glutamine and asparagine deamidation. Carbamidomethylation of cysteine was set as a fixed modification. Maximum missed cleavages was set at 2 while protein and PSM false discovery rate was set at 1%. Match between runs was disabled. The dataset was then filtered to remove proteins categorized as “contaminant”, “reverse” and “only identified by site” using Persues (1.6.6.0). Copy numbers were calculated using the proteomic ruler plugin within Perseus as previously described (Wisniewski et al., MCP, 2014).  SAMPLE KEY: MZ-1, MZ-3, MZ-4 = WT MZ-2, MZ-5, MZ-6 = PDCD4 KO

### Publication Abstract
None

### Keywords
Immunology, Pdcd4, B cells

### Affiliations
University of Dundee
Babraham Institute, Cambridge, CB22 3AT, UK

### Submitter
Andrew Howden

### Lab Head
Dr Martin Turner
Babraham Institute, Cambridge, CB22 3AT, UK


