### Accession
PXD031108

### Title
Identification of neoantigens in oesophageal adenocarcinoma

### Description
We present data from tissues of seven oesophageal adenocarcinomas of CD4 and CD8 T cell epitopes eluted from the cell surface using mass spectrometry (immunopeptidomics) of presented HLA bound peptides.

This dataset forms part of the publication:

Nicholas, B., Bailey, A., McCann, K.J., Wood, O., Walker, R.C., Parker, R., Ternette, N., Elliott, T., Underwood, T.J., Johnson, P. and Skipp, P. (2022), Identification of neoantigens in esophageal adenocarcinoma. Immunology. Accepted Author Manuscript. https://doi.org/10.1111/imm.13578

### Sample Protocol
Snap frozen tissue samples were briefly thawed and weighed prior to 30 S of mechanical homogenization (Fisher, using disposable probes) in 4 ml lysis buffer (0.02M Tris, 0.5% (w/v) IGEPAL, 0.25% (w/v) sodium deoxycholate, 0.15mM NaCl, 1mM EDTA, 0.2mM iodoacetamide supplemented with EDTA-free protease inhibitor mix). Homogenates were clarified for 10 min @2,000g, 4¬∞C and then for a further 60 min @13,500g, 4¬∞C. 2 mg of anti-MHC-I mouse monoclonal antibodies (W6/32) covalently conjugated to Protein A sepharose (Repligen) were added to the clarified supernatants and incubated with constant agitation for 2 h at 4¬∞C. The captured MHC-I/ùõΩ2microglobulin/immunopeptide complex on the beads was washed sequentially with 10 column volumes of low (isotonic, 0.15M NaCl) and high (hypertonic, 0.4M NaCl) TBS washes prior to elution in 10% acetic acid and dried under vacuum. The MHC-I-depleted lysate was then incubated with anti-MHC-II mouse monoclonal antibodies (IVA12) and MHC-II bound peptides were captured and eluted in the same conditions.Immunopeptides were separated from MHC-I/ùõΩ2M or MHC-II heavy chain using offline HPLC on a c18 reverse phase column. Briefly, dried immunoprecipitates were reconstituted in buffer (1% acetonitrile,0.1% TFA) and applied to a 10cm RP-18e chromolith column using an Ultimate 3000 HPLC equipped with UV monitor. Immunopeptides were then eluted using a 15 min 0-40% linear acetonitrile gradient at a flow rate of 1 ml/min.  HLA peptides were separated by an Ultimate 3000 RSLC nano system (Thermo Scientific) using a PepMap C18 EASY-Spray LC column, 2 ¬µm particle size, 75 ¬µm x 75 cm column (Thermo Scientific) in buffer A (0.1% Formic acid) and coupled on-line to an Orbitrap Fusion Tribrid Mass Spectrometer (Thermo Fisher Scientific,UK) with a nano-electrospray ion source. Peptides were eluted with a linear gradient of 3%-30% buffer B (Acetonitrile and 0.1% Formic acid) at a flow rate of 300 nL/min over 110 minutes. Full scans were acquired in the Orbitrap analyser using the Top Speed data dependent mode, preforming a MS scan every 3 second cycle, followed by higher energy collision-induced dissociation (HCD) MS/MS scans. MS spectra were acquired at resolution of 120,000 at 300 m/z, RF lens 60% and an automatic gain control (AGC) ion target value of 4.0e5 for a maximum of 100 ms. MS/MS resolution was 30,000 at 100 m/z. Higher‚Äêenergy collisional dissociation (HCD) fragmentation was induced at an energy setting of 28 for peptides with a charge state of 2‚Äì4, while singly charged peptides were fragmented at an energy setting of 32 at lower priority. Fragments were analysed in the Orbitrap at 30,000 resolution. Fragmented m/z values were dynamically excluded for 30 seconds.

### Data Protocol
Raw spectrum files were analysed using Peaks Studio 10.0 build 20190129, and the data processed to generate reduced charge state and deisotoped precursorand associated product ion peak lists which were searched against the Uniprot database (20,350 entries, 2020-04-07) plus the corresponding mutanome for each sample (~1,000-5,000 sequences) and contaminants list in unspecific digest mode. Parent mass error tolerance was set a 5ppm and fragment mass error tolerance at 0.03 Da. Variable modifications were set for N-term acetylation (42.01 Da), methionine oxidation (15.99 Da), carboxyamidomethylation (57.02 Da) of cysteine. A maximum of three variable modifications per peptide was set. The false discovery rate (FDR) was estimated with decoy-fusion database searches (Zhang et al., 2012) and were filtered to 1% FDR. Downstream analysis and data visualizations of the Peaks Studio identifications was performed in R using associated packages (R Core Team, 2018; Wickham et al., 2019).

### Publication Abstract
Oesophageal adenocarcinoma (OAC) has a relatively poor long-term survival and limited treatment options. Promising targets for immunotherapy are short peptide neoantigens containing tumour mutations, presented to cytotoxic T-cells by human leucocyte antigen (HLA) molecules. Despite an association between putative neoantigen abundance and therapeutic response across cancers, immunogenic neoantigens are challenging to identify. Here we characterized the mutational and immunopeptidomic landscapes of tumours from a cohort of seven patients with OAC. We directly identified one HLA-I presented neoantigen from one patient, and report functional T-cell responses from a predicted HLA-II neoantigen in a second patient. The predicted class II neoantigen contains both HLA I and II binding motifs. Our exploratory observations are consistent with previous neoantigen studies in finding that neoantigens are rarely directly observed, and an identification success rate following prediction in the order of 10%. However, our identified putative neoantigen is capable of eliciting strong T-cell responses, emphasizing the need for improved strategies for neoantigen identification.

### Keywords
Human, Immunopeptidome, Oesophageal adenocarcinoma, Hla

### Affiliations
University of Southampton
Centre for Proteomic Research B85,  Life Sciences Building, University of Southampton, University Road , Highfield Southampton,  Hants.  SO17 1BJ  United Kingdom

### Submitter
Alistair Bailey

### Lab Head
Dr Paul J Skipp
Centre for Proteomic Research B85,  Life Sciences Building, University of Southampton, University Road , Highfield Southampton,  Hants.  SO17 1BJ  United Kingdom


