### Accession
PXD029410

### Title
Parkinson's disease protein PARK7 prevents metabolite and protein damage caused by a glycolytic metabolite

### Description
Early-onset Parkinson's disease can be caused by mutations in the gene PARK7. Many functions have been proposed for PARK7, but in vivo evidence is largely lacking or conflicting with enzymological data. Here, we provide unequivocal evidence that PARK7 and its orthologues can prevent damage of proteins and metabolites inflicted by the glycolytic intermediate 1,3-bisphosphoglycerate, without changing the abundance of this metabolite. In the absence of PARK7, several glycerate-modified metabolites, as well as numerous glycerate- or phosphoglycerate-modified proteins accumulate in human cell lines, mouse brain and flies. This gives an explanation for the pleiotropic effects of PARK7, indicates that spontaneous metabolic damage contributes to the development of hereditary Parkinson's disease, and opens new roads for future therapeutic approaches. Submitted here are the proteomics analyses, where we identify and quantify glyceroyl- and phosphoglyceroyl-modifications of lysine residues in 5 different experimental models.  1. The human colorectal cancer cell line HCT116 was analyzed in four different conditions: a. Wild type cells transduced with an empty recombinant empty lentivirus b. PARK7 knockout cells transduced with an empty recombinant empty lentivirus c. PARK7 knockout cells transduced with a recombinant lentivirus driving expression of PARK7.  2. Brain samples from wild type and PARK7 knockout mice.  3. Red blood cell lysates from wild type and PARK7 knockout mice  4. Whole body extracts from wild type and DJ1beta knockout Drosophila melanogaster  5. In vitro reactions where 1,3-bisphosphoglycerate was produced either by GAPDH or PGK, in the presence and absence of PARK7. Samples were a. Enzymes in the absence of any cofactors (i.e. no 1,3-BPG production) b. Production via GAPDH (using glyceraldehyde 3-phosphate, phosphate and NAD+ as substrates) in the presence or absence of PARK7 c. Production via PGK (using 3-phosphoglycerate and ATP as substrates) in the presence or absence of PARK7.

### Sample Protocol
To detect and quantify phosphoglycerate-modified proteins in cell lines, mouse organs or flies we enriched phosphopeptides using the Easyphos protocol with minor modifications. After an overnight digestion with Trypsin and lysC at 37C shaking at 1500 rpm, an aliquot of 10 µL was withdrawn as an ‘input’. Detergent was removed from this aliquot using detergent removal spin columns (Pierce) and peptides were recovered in a solution containing 0.3 % trifluoroacetic acid (TFA) and 2 % acetonitrile (both Biosolve). These samples represent the non-enriched peptide samples. The remainder was thoroughly mixed with 400 µL of isopropanol for 30 s at 1500 rpm. Subsequently, enrichment buffer [48% (vol/vol) TFA and 8 mM KH2PO4] was added, samples were mixed again for 30 s at 1500 rpm and potential precipitates were removed by centrifugation for 15 min at 2000 g and room temperature. Titanium dioxide (TiO2) beads (GL Sciences) were washed in 6% (vol/vol) TFA/ 80% (vol/vol) acetonitrile, resuspended at 1 mg/µL and 14.4 µL of the suspension were added to each sample. After incubation for 5 min at 40 C while shaking at 2000 rpm, beads were collected by centrifugation for 2 min at 2000 g and room temperature. Beads were then washed five times with 1 mL of wash buffer [5% (vol/vol) TFA/60% (vol/vol) isopropanol] by agitation for 30 s at 2000 rpm and collection by centrifugation for 1 min at 2000 g. After the final wash, beads were transferred using 2x 75 µL wash buffer onto C8 stage tips (Affinisep, Petit-Couronne, France). Tips were centrifuged to dryness for 10 min at 2000 g using a stage-tip adaptor (GL Sciences, Tokyo, Japan). Phosphopeptides were eluted twice by addition of 30 µL freshly prepared elution buffer consisting of a 1 to 4 mixture of ammonia solution and 40 % (vol/vol) acetonitrile, followed by centrifugation for 4 min at 1500 g into low-binding 1.5 mL tubes. The combined eluate was dried using a SpeedVac vacuum concentrator at 45 °C until 15 µL remained, and desalted using SDB-RPS StageTips (Affinisep). To this end, after addition of 100 µL of SDB-RPS loading buffer consisting of 1 % TFA in isopropanol, samples were applied to StageTips and centrifuged to dryness for 10 min at 1500 g. After one wash with 100 µL of SDB-RPS loading buffer and one wash with SDB-RPS wash buffer consisting of 0.2 % TFA in 5 % acetonitrile, peptides were eluted with 60 µL of freshly prepared elution buffer produced by the addition of 20 µL of ammonium hydroxide solution to 4 mL of 60 % acetonitrile. After evaporation in a SpeedVac vacuum concentrator system at 45C, peptides were resuspended in 7.5 µL of 0.3 % TFA in 2 % acetonitrile (both from Biosolve).  Red blood cell lysates and purified proteins were digested in the presence of 1M urea and desalted on C18 Spin columns (Pierce, Thermo™ catalog no. 89870) according to the manufacturer’s protocol. After drying the samples with a vacuum concentrator at 45°C, they were resuspended in MS loading buffer (3 % ACN, 0.2 % TFA). The concentration of desalted peptides was measured using a peptide quantification assay (Pierce™, Thermo cat. 23275). 6 µL of phosphopeptide-enriched samples or up to 700 ng of the non-enriched samples were loaded onto a reversed-phase pre-column (Acclaim PepMap 100, Thermo Scientific). Peptide separation was performed using a reversed-phase analytical column (Acclaim PepMap RSLC, 0.075 × 250 mm, Thermo Scientific) on an Ultimate 3000 RSLC (Thermo Scientific) system at a flow rate of 300 nl/min (see paper for gradient). Peptides were analyzed by an Orbitrap Fusion Lumos tribrid mass spectrometer (Thermo Fisher Scientific) by tandem mass spectrometry using a nano ESI source coupled online to the UPLC. Detailed settings are presented in Table S2. Briefly, electrospray voltage was set to 2.4 kV and capillary temperature to 275°C. MS1 spectra were obtained in the Orbitrap at a resolution of 120,000 in a range of 350 to 1800 m/z with an automatic gain control (AGC) target of 40000 ions and a maximum injection time of 50 ms. A data-dependent procedure of MS/MS scans was used. Top precursor ions above threshold (see Table S2) were isolated in the quadrupole with a window of 1.2 mass units and subjected to high energy C-trap dissociation (HCD) with a normalized collision energy of 30 %. One MS scan was alternated by several MS/MS scans applied for the top precursor ions  with 30 s dynamic exclusion after 1 fragmentation using a maximum cycle time of 3 seconds. MS2 spectra were acquired using 2 distinct methods. For the analysis of non-enriched samples (‘input’) we used the ion trap to maximize sensitivity and the time available for the acquisition of MS2 spectra. In contrast, we obtained Orbitrap MS2 spectra with a resolution of 30 000 for the analysis of  phosphopeptide-enriched samples.

### Data Protocol
MS/MS data was processed with the Sequest HT search engine within Proteome Discoverer 2.4 against a human or mouse protein database obtained from Uniprot (67) or against rabbit GAPDH and yeast PGK. Trypsin was specified as cleavage enzyme allowing up to 2 missing cleavages, 5 modifications per peptide and up to 7 charges. The mass error was set to 20 ppm for precursor ions, 0.6 Da for fragment ions acquired in the ion trap (i.e. non-enriched samples, hemoglobin and purified proteins) or 0.1 Da for fragment ions acquired in the Orbitrap (i.e. phosphopeptide enriched samples). N-terminal acetylation, N-terminal methionine loss, oxidation of methionine, N-terminal glyceroylation +/- methionine loss (-43.0244 / 88.0160 Da), N-terminal phosphoglyceroylation +/- methionine loss (36.9419 / 167.9824 Da), glyceroylation of lysine (88.0160 Da), phosphoglyceroylation of lysine (167.9824 Da) as well as phosphorylation of serine, threonine or tyrosine residues were considered variable modifications, whereas carbamidomethylation of cysteine was considered as a fixed modification. Modification with glycerate or phosphoglycerate always resulted in a missed cleavage after the affected lysine residue. In Proteome Discoverer, the false discovery rate (FDR) was assessed using Percolator and thresholds for the identification of proteins, peptides and modification sites were specified at 1% (68).  Label-free quantification of peptides was performed with the Proteome Discoverer 2.4 software package (Thermo) and is based on the precursor ion intensity. Signals were normalized to the sum of all signals within each individual sample. Protein abundances were calculated as the sum of the abundances of unmodified peptides.

### Publication Abstract
Cells are continuously exposed to potentially dangerous compounds. Progressive accumulation of damage is suspected to contribute to neurodegenerative diseases and aging, but the molecular identity of the damage remains largely unknown. Here we report that PARK7, an enzyme mutated in hereditary Parkinson's disease, prevents damage of proteins and metabolites caused by a metabolite of glycolysis. We found that the glycolytic metabolite 1,3-bisphosphoglycerate (1,3-BPG) spontaneously forms a novel reactive intermediate that avidly reacts with amino groups. PARK7 acts by destroying this intermediate, thereby preventing the formation of proteins and metabolites with glycerate and phosphoglycerate modifications on amino groups. As a consequence, inactivation of PARK7 (or its orthologs) in human cell lines, mouse brain, and <i>Drosophila melanogaster</i> leads to the accumulation of these damaged compounds, most of which have not been described before. Our work demonstrates that PARK7 function represents a highly conserved strategy to prevent damage in cells that metabolize carbohydrates. This represents a fundamental link between metabolism and a type of cellular damage that might contribute to the development of Parkinson's disease.

### Keywords
Posttranslational modification, Parkinson's disease, Glycerate, Glyceroyl, Glycolysis, Park7, Dj1, Phosphoglyceroyl, Phosphoglycerate

### Affiliations
Université Catholique de Louvain
Institut de Duve
Groupe de recherche Metabolique (GRM), BCHM, Institut de Duve, UniversitÃ© Catholique de Louvain, Belgium

### Submitter
Isaac Heremans

### Lab Head
Dr Guido Thomas Bommer
Groupe de recherche Metabolique (GRM), BCHM, Institut de Duve, UniversitÃ© Catholique de Louvain, Belgium


