### Accession
PXD005748

### Title
A quantitative proteomic profile revealed the crucial roles of lysosome in the microbial infection

### Description
Lysosome played vital roles for both innate and adaptive immunity. It has been widely accepted that lysosome functioned more than a digestive organelle. However, it was still largely unknown for the proteome changes of lysosome on various microbes, and the understanding of lysosomal proteome from the purified lysosome is another obstacle that needs to be resolved. Here, we performed a proteomic study on the enriched lysosomes from THP1 cells after infected by Listeria monocytogenes (L. m), Herpes Simplex Virus 1 (HSV-1) and Vesicular Stomatitis Virus (VSV). Combining with the GO analysis, we identified 284 lysosomal related proteins from the totally identified 4560 proteins. We also constructed the protein-protein interaction networks for the changed proteins and revealed the core lysosomal proteins in responding for diverse microbial infections. From this study, we not only revealed that lysosome responded in a different manner for the bacterial or virus infection, but also proposed a new way for the proteome research for the enriched organelles.

### Sample Protocol
Cell Culture and Infection THP1 cells were cultured in RPMI1640 supplemented with 10% fetal bovine serum (FBS). Listeria monocytogenes (L. m) was amplified in Brain Heart Infusion (BHI). Herpes Simplex Virus 1 (HSV-1) and Vesicular Stomatitis Virus (VSV) were expanded in VERO cells as described elsewhere [18], respectively. THP1 cells were treated with L. m, HSV-1 or VSV at a multiplicity of infection (MOI) of 10, respectively and were harvested 9 h later after infection.  Lysosome Purification The Lysosome Isolation Kit (Cat No. LYSISO1, Sigma, MO, USA) was used for the lysosome purification according to the manufacturer’s instructions with slight modifications. Cells were harvest and washed twice with ice cold PBS and centrifuged at 500 ×g for 10 min. The pellet was then resuspended in extraction buffer and homogenized with a 15 ml Dounce homogenizer (30 cycles). The cell disruption was evaluated under an optical microscope. After ultracentrifugation (150,000 ×g for 4 h), samples were separated into 9 fractions according to the rims. The location of the lysosomal fractions was verified using an acid phosphatase (AP) assay kit and total protein concentration of each fraction was measured with a BCA protein assay kit according to the manufacturer’s instruction respectively.  Western Blot Analysis and Antibodies Lysates from each fraction prepared from gradient centrifugation were subjected to western blot analysis. Different antibodies of organelle biomarkers were used for the verification of lysosome enriched fraction as indicated:  Lamp1 (ab25245), Ezrin () and ERp72 (), RCAS1 () and Cox7b (ab140629).  Tandem Mass Tag Labeling Lysosomes were destroyed in 8 M urea lysis buffer (dissolved in 0.01 M PBS, pH 7.2) containing proteinase inhibitors (Cocktail, Roche, IN, USA) followed by protein concentration measurement using a Nano Drop 2000. Equal amounts of protein (100 µg) from each group were treated with 10 mM DTT at 37 °C for 40 min followed by 25 mM IAA at 37 °C for 1 h under dark condition. Subsequently, Trypsin/Lys-C mixture (50：1) was added to the reaction solution for protein digestion for 1 h and diluted with PBS to make the urea reached a finial concentration of 1 M, then digested for overnight under 37 °C. The digestion was quenched by heating on 65 °C and acidized with 0.1% TFA. Then, the samples were desalted by the reversed-phase column chromatography (Oasis HLB; Waters, MC, USA) and TMT isobaric labeling was performed according to the manufacture’s instruction with a slightly modification. Equal amounts of peptides derived from each group were labeled with different TMT labels: [control, TMT-126; L. m, TMT-127; HSV-1, TMT-128; and VSV TMT-130]. The labeled peptides were mixed and desalted and then separated into 50 fractions with HPLC as previously described elsewhere [19]. Briefly, the peptides were loaded on the C18 peptide separation column (Xbridge BEH300, C18, 2.5 µm, 4.6×250 mm, Waters) and were separated into 50 parts by the gradate elution buffer constituted by water/acetonitrile at pH 10.0.  LC-MS/MS and Database Searching The fractions derived from HPLC were dried by a vacuum concentrator, and then were combined and dissolved into 20 parts in 0.1% TFA. Thermo Orbitrap Fusion mass spectrometer was used for detection of each fraction after separated by an EASY-nLC 1000 system using 120 min gradate elution at a flow rate of 0.30 µl/min. The homemade fused silica capillary column (75-μm inner diameter, 150-mm length; Upchurch, Oak Harbor, WA, USA) packed with C-18 resin (300 Å, 5 μm; Varian, Lexington, MA, USA) were used as analytical column. Mobile phase A and mobile phase B were consisted by ddH2O with 0.1% formic acid and acetonitrile with 0.1% formic acid, respectively. Xcalibur 3.0 software was used for the visualization of mass spectrometer’s operation. The Orbitrap (350-1550 m/z, 120,000 resolution) was used for a single full-scan mass spectrum and 3-s data-dependent MS/MS scans at 38% normalized collision energy (HCD).

### Data Protocol
The MS/MS raw data were searched against the UniProt/SwissProt mouse database (released on November 1st 2016) using the Proteome Discovery (version 1.4). A minimum number of peaks at 200 and intensity of 20000 were set as Mass spectra filter for the SEQUEST HT analysis. Tripsin/Lys-C and a maximum of two missed cleavage sites for a range of 6 to 144 peptides were applied as tryptic parameters. The precursor tolerance was set as 20 ppm and fragment tolerance was 0.02 Da. Cysteine carbamidomethylation (+57.021) and lysine and N-terminal TMT 6-plex modification (+229.163) was set as static modification. Proteins with at least one unique peptide (with a false discovery rate (FDR) at 1%, p<0.05) was considered to be valid. All peptides ratios were normalized to the median protein ratio and protein ratio was determined by all the identified unique peptides.

### Publication Abstract
Lysosomes play vital roles in both innate and adaptive immunity. It is widely accepted that lysosomes do not function exclusively as a digestive organelle. It is also involved in the process of immune cells against pathogens. However, the changes in the lysosomal proteome caused by infection with various microbes are still largely unknown, and our understanding of the proteome of the purified lysosome is another obstacle that needs to be resolved. Here, we performed a proteomic study on lysosomes enriched from THP1 cells after infection with Listeria monocytogenes (L.m), Herpes Simplex Virus 1 (HSV-1) and Vesicular Stomatitis Virus (VSV). In combination with the gene ontology (GO) analysis, we identified 284 lysosomal-related proteins from a total of 4560 proteins. We also constructed the protein-protein interaction networks for the differentially expressed proteins and revealed the core lysosomal proteins, including SRC in the L. m treated group, SRC, GLB1, HEXA and HEXB in the HSV-1 treated group and GLB1, CTSA, CTSB, HEXA and HEXB in the VSV treated group, which are involved in responding to diverse microbial infections. This study not only reveals variable lysosome responses depending on the bacterial or virus infection, but also provides the evidence based on which we propose a novel approach to proteome research for investigation of the function of the enriched organelles.

### Keywords
Monocyte, Proteomics, Lysosome, Hypoxia

### Affiliations
State Key Laboratory of Medical Molecular Biology & Department of Immunology, Institute of Basic Medical Sciences, Chinese Academy of Medical Sciences
National Key Laboratory of Medical Molecular Biology & Department of Immunology, Institute of Basic Medical Sciences, Chinese Academy of Medical Sciences 

### Submitter
Wei Ge

### Lab Head
Dr Wei Ge
State Key Laboratory of Medical Molecular Biology & Department of Immunology, Institute of Basic Medical Sciences, Chinese Academy of Medical Sciences


