### Accession
PXD030480

### Title
Comparative analyses of vertebrate CPEB-family of proteins define two sub-families with distinct, but coordinated, functions in post-transcriptional regulation of gene expressionComparative analyses of vertebrate CPEB-family of proteins define two sub-families with distinct, but coordinated, functions in post-transcriptional regulation of gene expression

### Description
Background: The vertebrate CPEB-family of RNA-binding proteins promote translational repression or activation of their target mRNAs, which harbor cytoplasmic polyadenylation elements (CPEs) in their 3‚Äô UTRs, through cytoplasmic changes in their poly(A) tail lengths. However, their regulation(s) and mechanism(s) of action have not been systematically addressed as a family. Results: Here we perform a comparative analysis of the four vertebrate CPEBs in their regulation by phosphorylation, supramolecular assemblies‚Äô composition and properties and in their mRNA targets. Conclusions: We show that although all four CPEBs are able to recruit CCR4-NOT deadenylation complex when not phosphorylated, their mechanisms of action determine two subfamilies with distinct, but coordinated, regulation by phosphorylation and target specificity. Thus, CPEB1 forms ribonucleoprotein complexes that are remodeled upon a single phosphorylation event, by AurkA, and are associated with mRNAs containing canonical CPEs. On the other hand, CPEB2-4 are regulated by multiple proline-directed phosphorylations that control their liquid-liquid phase-separation. CPEB2-4 mRNA-targets include CPEB1-bound transcripts, with canonical CPEs, but also a specific subset of mRNAs with non-canonical CPEs. In turn, CPEB2-4 coacervates display compositional, morphological and dynamic differences. Altogether, these results show how, globally, the CPEB family of proteins is able to integrate cellular cues to generate a finetuned adaptive response in gene expression regulation.

### Sample Protocol
Protein expression Selected fully-grown oocytes were microinjected with 46 nL of 50 ng/ÔÅ≠L in vitro transcribed polyadenylated RNA. 16-hours post microinjection, when indicated, maturation was induced with 10 ÔÅ≠M progesterone (Sigma Aldrich, P0130). Maturation was scored by the appearance of a white maturation spot. 100 live oocytes were collected per condition. Oocytes were lysed by pipetting in 9 ŒºL/oocyte cold H1K 0.4% NP40 (80 mM sodium Œ≤-glycerophosphate pH 7.4, 20 mM EGTA, 15 mM MgCl2, 0.5 mM Na3VaO4, 0.4% NP40, 1x Complete EDTA-free Protease Inhibitors (Roche)) and clarified by centrifugation for 10 minutes at 12000 g 4¬∫C. 1 ŒºL of 10x IP lysis buffer (20 mM Tris-HCl pH 8, 100 mM NaCl, 0.4% NP40, 1 mM EDTA, 1 mM MgCl2,) was added per 9 ŒºL of recovered lysate. Protein purification HA-CPEB3 was immunoprecipitated with HA-conjugated magnetic beads (Pierce Anti-HA Magnetic Beads, 88837, Thermo Fisher Scientific). These were washed twice in PBS and twice in lysis buffer prior to use and a proportion of 150 ŒºL bead-slurry per 1 mL clarified lysate was employed. The lysate was incubated with the beads on a rotating wheel for 1:30 hour, at 4¬∫C. Then, the beads were washed thrice in one volume of lysis buffer and the purified proteins were eluted from the beads by incubating with Laemmli sample buffer without DTT for 20‚Äô at 60¬∫C. The supernatants were recovered using a magnetic rack (DynaMag-2, Thermo Fisher Scientific). DTT was added to a final concentration of 180 mM. The eluates were further denaturalized by boiling 5 minutes 95¬∫C. The purified protein was handled in Eppendorf LoBind microcentrifuge tubes (Thermo Fisher Scientific). The purified protein was run on precast 7.5% gels (Mini-Protean TGX, Bio-Rad). The gels were silver-stained using the Pierce Silver Stain for MS kit (24600, Thermo Fisher Scientific) following manufacturer‚Äôs instructions. The bands of interest were cut and placed on microcentrifuge tubes. Peptide purification Gel bands were washed with 50 mM NH4HCO3 (ammonium bicarbonate) and acetonitrile. Disulfide bond reduction was performed with 10 mM DTT and alkylation of thiol groups with 50 mM iodoacetamide. Some samples were digested with trypsin (V511, Sequencing Grade Modified Trypsin, Promega) and others with chymotrypsin (V1062, Chymotrypsin Sequencing Grade, Promega). The digestion was stopped by addition of 5% HCOOH (formic acid).Resulting peptides were extracted from gel. The samples were evaporated and reconstituted in 3% acetonitrile and 1% formic acid. For the nano-LC-MS/MS, 50% of the sample volume was used.  Nano-LC-MS/MS Samples were loaded to a C18 precolumn (Acclaim PepMap100, 5 ÔÅ≠m, 100 √Ö, 300 ÔÅ≠m x 5 mm, Thermo Fisher Scientific) at a flow rate of 15 Œºl/minute using a Dionex UltiMate 3000 chromatographic system (Thermo Fisher Scientific). Peptides were separated using a C18 analytical column (NanoEase MZ HSS T3 1.8 Œºm, 100 √Ö, 75 Œºm x 250 cm, Waters) with a 60 minutes run from 1 to 35% (where B was 0.1% formic acid in acetonitrile and the other eluate was 0.1% formic acid in water). The column outlet was directly connected to an Advion TriVersa NanoMate (Advion) fitted on an Orbitrap Fusion Lumos Tribrid (Thermo Fisher Scientific).  The mass spectrometer was operated in a data-dependent acquisition (DDA) mode. Survey MS scans were acquired in the Orbitrap with the resolution (defined at 200 m/z) set to 120,000. The lock mass was user-defined at 445.12 m/z in each Orbitrap scan. The top speed (most intense) ions per scan were fragmented and detected in the linear ion trap. The ion count target value was 400,000 for the survey scan and 50,000 for the MS/MS scan. Target ions already selected for MS/MS were dynamically excluded for 30 s. Spray voltage in the NanoMate source was set to 1.60 kV. RF Lenses were tuned to 30%. Minimal signal required to trigger MS to MS/MS switch was set to 25000. The spectrometer was working in positive polarity mode and singly charge state precursors were rejected for fragmentation.

### Data Protocol
MS/MS spectra were searched against Uniprot Xenopus laevis database, release 201810, contaminants and user-defined proteins using both both MaxQuant software v1.6.2.6a with Andromeda search engine and Proteome Discoverer v2.1 (Thermo Fisher Scientific) with Sequest HT and Amanda search engines. Searches were run against targeted and decoy database to determine the false discovery rate (FDR). Search parameters included trypsin enzyme specificity, allowing for two missed cleavage sites, oxidation in methionine, phosphorylation in serine/threonine/tyrosine and acetylation in protein N-terminus as dynamic modifications and carbamidomethyl in cysteine as static modification. Peptide mass tolerance was 10 ppm, the MS/MS tolerance was 0.02 Da and minimal peptide length was 7 amino acids. Peptides with FDR < 1% were considered as positive identifications with a high confidence level. Analysis A phosphorylation ratio (r) for each phosphorylation site (p-site) within CPEB3 protein was computed considering three search nodes: Andromeda, Amanda and Sequest. For each p-site, the number of position-specific phosphorylated peptide spectrum matches (PSMs) (NPhos) and the number of non-phosphorylated PSMs (NNonPhos) were counted; from which r was then computed as follows:  r = ùëÅPhos / (ùëÅPhos + ùëÅNonPhos ) In this calculation, only those p-sites with localization probability greater than 75% were considered.

### Publication Abstract
None

### Keywords
Bioid, Deadenylation, Phosphorylation, Phase-separation, Ccr4-not complex, Cpeb, 3‚Äôutr, Mrna translation

### Affiliations
Institute for Research in Biomedicine
Institute for Research in Biomedicine (IRB Barcelona), The Barcelona Institute of Science and Technology, 08028 Barcelona, Spain Instituci√≥ Catalana de Recerca i Estudis Avan√ßats (ICREA), 08010 Barcelona, Spain

### Submitter
Marina Gay

### Lab Head
Dr Raul Medez
Institute for Research in Biomedicine (IRB Barcelona), The Barcelona Institute of Science and Technology, 08028 Barcelona, Spain Instituci√≥ Catalana de Recerca i Estudis Avan√ßats (ICREA), 08010 Barcelona, Spain


