### Accession
PXD023826

### Title
Proteomic analysis in transgenic hybrid aspen (Populus tremula x tremuloides) trees overexpression of vesicle-associated membrane protein PttVAP27-17

### Description
In this study we report on transgenic hybrid aspen (Populus tremula x P. tremuloides) lines that showed potential to increase biomass production both in the greenhouse and after five years of growth in the field. The transgenic lines carried an overexpression construct for Populus tremula x tremuloides vesicle-associated membrane protein (VAMP)-associated protein PttVAP27-17 (Potri.019G116400) that was selected from a gene-mining program for novel regulators of wood formation.  A proteomic analysis was performed to characterize the overall effect of the overexpression of PttVAP27-17 on plant metabolic pathways. 20 mg of xylem sample for the selected wild type (WT) and three transgenic lines (lines 1,2 and 3) was collected in the following manner from two-months-old, greenhouse grown trees: The frozen bottom-part of the stem (10-17 cm portion from the base of the stem) was peeled, and the surface of the secondary xylem consisting of living vessels and fibers (into the depth of approximately one mm from the surface) was scraped. The xylem scrapings were ground to fine powder in liquid nitrogen and stored at -80oC. The analyses included 3-5 biological replicates for each of the transgenic lines, and seven replicates for the wild type.

### Sample Protocol
Total proteins were extracted from 20 mg of frozen stem tissue powder as described in Masuda et al. (2008). The samples were run on a SynaptTM G2 HDMS mass spectrometer (Waters, UK) equipped with a nanoflow electrospray ionization interface according to Obudulu et al. (2016). Masuda T, Tomita M, Ishihama Y. Phase Transfer Surfactant-Aided Trypsin Digestion for Membrane Proteome Analysis. J. Proteome Res. 2008;7:731Ã¢â‚¬â€œ40. Obudulu O, Bygdell J, Sundberg B, Moritz T, Hvidsten TR, Trygg J, et al. Quantitative proteomics reveals protein profiles underlying major transitions in aspen wood development. BMC Genomics 2016;17:119.

### Data Protocol
The data were processed with Protein Lynx Global Server v.3.0 (Waters) and the resulting spectra were searched against Populus trichocarpa v3.0 sequences compiled in the JGI Comparative Plant Genomics Portal database (http://phytozome.jgi.doe.gov) along with sequences for human keratin and rabbit glycogen phosphorylase. The database search settings were: enzyme-specific cleavage with one miss-cleavage allowed; oxidized methionine and protein N-terminal acetylation as variable modifications, peptide and fragment tolerance 10 and 25 ppm, respectively; and a false positive rate of 3 %. Detected proteins were quantified from the sum of the top three matched peptide intensities, as the average intensity of the three most intense peptides strongly correlates with absolute amounts of their source proteins (Silva et al., 2005; Distler et al., 2014). All samples were weighed prior to extraction and protein peak area data were normalized relative to the weight. Only unique peptides were used for calculation of the protein abundancies. Obudulu O, Bygdell J, Sundberg B, Moritz T, Hvidsten TR, Trygg J, et al. Quantitative proteomics reveals protein profiles underlying major transitions in aspen wood development. BMC Genomics 2016;17:119. Silva JC, Denny R, Dorschel CA, Gorenstein M, Kass IJ, Li GZ, McKenna T, Nold MJ, Richardson K, Young P, Geromanos S. Quantitative proteomic analysis by accurate mass retention time pairs. Anal Chem. 2005;77:2187Ã¢â‚¬â€œ2200. Distler U, Kuharev J, Navarro P, Levin Y, Schild H, Tenzer S. Drift time-specific collision energies enable deep-coverage data-independent acquisition proteomics. Nat Methods. 2014;11:167Ã¢â‚¬â€œ70.

### Publication Abstract
None

### Keywords
Wood, Hdmse, Populus

### Affiliations
Umea Plant Science Centre, Department of Forest Genetics & Plant Physiology, Swedish University of Agricultural Sciences, SE-901 83 Umea, Sweden
Computational Life Science Cluster (CLiC),
Deptartment of Chemistry, Umeå University

### Submitter
Joakim Bygdell

### Lab Head
Dr Hannele Tuominen
Umea Plant Science Centre, Department of Forest Genetics & Plant Physiology, Swedish University of Agricultural Sciences, SE-901 83 Umea, Sweden


