### Accession
PXD009630

### Title
Malignant Melanoma -  Improved survival prognostication of node-positive malignant melanoma patients utilizing shotgun proteomics guided by histopathological characterization and genomic data

### Description
In depth histopathology analysis of melanoma lymph node metastases combined with deep-mining protein expression analysis, clinical data and genomic profile information,  in the search for  novel protein markers to predict progression and survival of melanoma

### Sample Protocol
Frozen tissue slices were lysed with 6 M urea in 50 mM ammonium bicarbonate buffer for protein extraction. Urea was removed from the samples using Amicon Ultra centrifugal filters. Reduced and alkylated proteins were digested with trypsin in a ratio 1:100 w/w (trypsin:protein) overnight at 37Â°C. LC-MS/MS was performed with a Thermo Easy nLC 1000 system coupled online to a Q-Exactive Plus mass spectrometer.

### Data Protocol
Raw files were analysed with Proteome Discoverer v 2.1. Peptides were identified using SEQUEST HT against UniProtKB human database integrated into Proteome Discoverer (release May 2016). The search was performed with the following parameters applied: carbamidomethylation as static modification, oxidation of methionine as dynamic modification, 20 ppm precursor tolerance and 0.02 Da fragment tolerance. Up to two missed cleavages for tryptic peptides was allowed. Filters: high confidence at peptides and protein levels were applied (FDR 0.01). We have used unsupervised and supervised approaches to linking proteomic data to survival. The unsupervised method was performed using consensus clustering in R using ConsensusClusterPlus library (version 1.42.0). The supervised approach is based on PLS-Cox regression similar to that of [1].

### Publication Abstract
Metastatic melanoma is one of the most common deadly cancers, and robust biomarkers are still needed, e.g. to predict survival and treatment efficiency. Here, protein expression analysis of one hundred eleven melanoma lymph node metastases using high resolution mass spectrometry is coupled with in-depth histopathology analysis, clinical data and genomics profiles. This broad view of protein expression allowed to identify novel candidate protein markers that improved prediction of survival in melanoma patients. Some of the prognostic proteins have not been reported in the context of melanoma before, and few of them exhibit unexpected relationship to survival, which likely reflects the limitations of current knowledge on melanoma and shows the potential of proteomics in clinical cancer research.

### Keywords
Clinical data, Histopathology, Proteomics, Malignant melanoma, Tumor

### Affiliations
Div. Clinical Protein Science & Imaging  Clinical Sciences & Biomedical Engineering Lund University Sweden
Lund University

### Submitter
Lazaro Hiram Betancourt Nunez

### Lab Head
Dr Gyorgy Marko-Varga
Div. Clinical Protein Science & Imaging  Clinical Sciences & Biomedical Engineering Lund University Sweden


### SDRF
- organism: Homo sapiens
- organism part: trunk, head/neck, upper extremity, lower extremity, other, not applicable
- cell type: not applicable
- disease: superficial spreading melanoma, nodular melanoma, acral lentiginous melanoma, other, unknown, not available, lentigo maligna melanoma, mucosal
- Comment[instrument: Q Exactive, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus, Q Exactive Plus
- Comment[label: label free sample
- Comment[modification parameters: Carbamidomethyl
- Comment[modification parameters: oxidation

