### Accession
PXD033234

### Title
CryoEM structure of mitochondrial complex I from the thermophilic eukaryote Chaetomium thermophilum

### Description
Mitochondrial complex I is a redox-driven proton pump that generates most of the proton-motive force powering oxidative phosphorylation and ATP synthesis in eukaryotes. We report the structure of complex I from the thermophilic eukaryote Chaetomium thermophilum, determined by electron cryo-microscopy to 2.4 A resolution in the open and closed conformation. Complex I has two arms, the peripheral and membrane arm, forming an L-shape. The two conformations differ in the relative position of the two arms. The open-to-closed transition is accompanied by substantial conformational changes in the Q-binding cavity and the E-channel, and by the formation of an aqueous connection between the E-channel and an extensive aqueous passage inside the membrane arm. The observed similarities provide strong support for a conserved, common mechanism that applies across all species from fungi to mammals. Furthermore, the complex is inhibited by the detergent DDM, which binds reversibly to two sites in the Q-binding cavity.

### Sample Protocol
Purified complex I samples were reduced with TCEP and cysteines alkylated with IAA. Subsequent proteolytic digests were performed using S-TRAPs (Protifi) according to the manufacturerâ€™s instructions. Peptides were further desalted and purified on C18 SPE cartridges and dried in an Eppendorf concentrator. After solubilisation in 0.1% formic acid (FA) in acetonitrile/water (95/5 [v/v]), samples were subjected to LC-MS/MS analysis on an Ultimate 3000 nanoRSLC (Thermo Fisher) system, equipped with an Acclaim Pepmap C18 trap column (2 cm * 75 um, particle size: 3 um; Thermo Fisher) and a C18 analytical column (50 cm * 75 um, particle size: 1.7 um; CoAnn Technologies) with an integrated liquid-junction and fused silica emitter coupled to an Orbitrap Fusion Lumos mass spectrometer (Thermo Fisher). Trapping was performed for 6 min with a flow rate of 6 ul/min using loading buffer (98/2 [v/v] water/acetonitrile with 0.05% Trifluoroacetic acid) and peptides were separated on the analytical column at a flow rate of 250 nl/min with the following gradient: 4 to 48% B in 45 min, 48 to 90% B in 1 min and constant 90 % B for 8 min followed by 20 min column re-equilibration at 4% B with buffer A (0.1% FA in water) and buffer B (0.1% FA in 80/20 [v/v] acetonitrile/water). Peptides eluting from the column were ionised online using a Nano Flex ESI-source and analysed in data-dependent-mode. Survey scans were acquired over the mass range from 350 - 1400 m/z in the Orbitrap (maximum injection time: 50 ms, AGC (automatic gain control) fixed at 2x10E5 with 120K mass resolution) and sequence information was acquired by a top speed method with a fixed cycle time of 2 s for the survey and following MS/MS-scans. MS/MS-scans were acquired for the most abundant precursors with a charge state from 2 to 5 and an intensity minimum of 5x10E3. Picked precursors were isolated in the quadrupole with a 1.4 m/z isolation window and fragmented using HCD (NCE (normalized collision energy) = 30%). For MS/MS spectra, an AGC of 10E4 and a maximum injection time of 54 ms were used and detection was carried out in the Orbitrap using 30K mass resolution. The dynamic exclusion was set to 30 s with a mass tolerance of 10 ppm.

### Data Protocol
Data analysis was performed in Proteome Discoverer (version 2.5) using Sequest HT as database search algorithm for peptide identification. Raw files were recalibrated and searched against the protein database obtained from structure modelling as well as the Uniprot proteome for Chaetomium thermophilum (UP000008066; obtained 2020-10-09) and common contaminants. The search space was restricted to tryptic peptides with a length of 7-30 amino acids allowing for up to two missed cleavages and with a minimum of one unique peptide per protein group as well as precursor and fragment mass tolerances of 10 ppm and 0.02 Da respectively. Carbamidomethylation of cysteine was set as a fixed modification and oxidation of methionine was set as variable modification. Inferys rescoring and Percolator nodes were used to estimate the number of false positive identifications and results filtered for a strict target false discovery rate (FDR) < 0.01.

### Publication Abstract
Mitochondrial complex I is a redox-driven proton pump that generates proton-motive force across the inner mitochondrial membrane, powering oxidative phosphorylation and ATP synthesis in eukaryotes. We report the structure of complex I from the thermophilic fungus <i>Chaetomium thermophilum</i>, determined by cryoEM up to 2.4-&#xc5; resolution. We show that the complex undergoes a transition between two conformations, which we refer to as state 1 and state 2. The conformational switch is manifest in a twisting movement of the peripheral arm relative to the membrane arm, but most notably in substantial rearrangements of the Q-binding cavity and the E-channel, resulting in a continuous aqueous passage from the E-channel to subunit ND5 at the far end of the membrane arm. The conformational changes in the complex interior resemble those reported for mammalian complex I, suggesting a highly conserved, universal mechanism of coupling electron transport to proton pumping.

### Keywords
Chaetomium thermophilum, Mitochondrial complex i

### Affiliations
Max Planck Institute of Biophysics, Max Planck Institute for Brain Research
MPIs for Biophysics and Brain Research

### Submitter
Julian Langer

### Lab Head
Dr Julian Langer
Max Planck Institute of Biophysics, Max Planck Institute for Brain Research


