### Accession
PXD006910

### Title
Molecular effects of Vitamin C treatment in MSC80 cells, a suitable Schwann cell model

### Description
Schwann cells (SCs) are not only decisive to produce the axon-wrapping myelin sheath thus ensuring a proper nerve conduction but also exhibit with trophic function and can direct repair mechanisms of the peripheral nervous system. Consequently, suitable and well-characterized SC in vitro models are needed to perform appropriate pre-clinical studies including the investigation of the complex biochemical adaptations occurring in the peripheral nervous system (PNS) under different (patho)physiological conditions. MSC80 cells represent a SC line derived from mouse used in laboratories as a suitable in vitro system for neuropathological studies. As a comprehensive MSC80 protein catalogue remained elusive, here we introduce the most abundant 9532 proteins identified via mass spectrometry based protein analytics and thus provide the most comprehensive SC protein catalogue published thus far. Hereby, not only proteins causative for inherited neuropathies were covered but it has also been demonstrated that apart from cytoplasmic and nuclear proteins, mitochondrial proteins and such belonging to the protein processing machinery are very well covered. Moreover, we addressed the suitability of MSC80 to examine molecular effect of a drug-treatment by analyzing the proteomic signature of Vitamin C-treated MSC80 cells. Proteomic findings along with further immunological and functional experiments support the concept of a beneficial role influence of Vitamin C on oxidative stress and identified TMX1 as an oxidative stress protective factor in SC which might represent a promising target for therapeutic intervention of PNS disorders with marked oxidative stress burden such as diabetic neuropathy.

### Sample Protocol
Sample preparation for shotgun proteomics Cell pellets were resuspended in 0.5 mL of 50 mM Tris-HCl (pH 7.8) buffer containing 150 mM NaCl, 1 % sodium dodecyl sulfate (SDS), supplemented with complete mini EDTA-free and phosphoStop (both from Roche). Following brief sonication (Vibra-Cell Ultrasonic Processor, 4 second pulses for 20 seconds) and a further incubation on ice for 30 min, the homogenate was centrifuged at 16,000 rcf and 4 °C for 30 min. Supernatant was collected and protein concentration was determined via BCA assay at 570 nm wavelength according to the manufacture’s instruction (Pierce BCA Protein Assay Kit, Thermo Scientific). Afterwards, protein cysteine residues were reduced with dithiothreitol (F.c 10 mM DTT) at 56 °C for 30 min, and subsequently, the new generated free thiols were carbamidomethylated with iodoacetamide (F.c. 30 mM IAA) for 30 minutes in the dark. Next filter-aided sample preparation (FASP) was carried out. More precisely, 100 µg of proteins were subjected onto 30 kDa weight cut-off (MWCO) membrane filer (Pall Corporation, Nanosep) together with 100 µl of 8 M Urea in Tris.HCl buffer (pH 8.5), centrifugation at 13.500 x g at RT was applied to wash the proteins, the washing step was performed 3 times with Urea buffer. Afterwards, proteins were washed and concentrated with 50 mM aminobicarbonate (NH4HCO3) buffer (pH 7.8) for 3 times in the same manner as with urea buffer. Lastly, proteins were subjected into digestion with 100 µl of digestion mixture comprising of 4 µg of trypsin (Promega, Trypsin Gold, Mass Spectrometry Grade) in 2 mM CaCl2, 0.2 GuHCl and 50 mM ABC (pH 7.8) for 14 hours at 37 °C.  Next day, peptide solution was collected by centrifugation for 13.500 x g for 15 min, followed by 2 other centrifugation steps with 50 µl of 50 mM ABC buffer (pH 7.8) and 50 µl of ultra-pure H2O. The peptide mixture was finally acidified with 10 % TFA (v/v) to completely inactivate trypsin by adjusting the pH to less than 3. Peptides were subsequently stored at 4 °C or -20 °C for short or long-term usage. Prior to subjection to MS analysis, the digestion efficiency was controlled by reversed-phased Ultimate 3000 RSLC 200 µm x 5 cm.  Peptide fractionation with pH8 and LC-MS/MS analysis 30 µg peptides were desalted with the OASIS HLB Cartridge (Waters) and then dried under the SpeedVac Concentrator System (Thermo Scientific). Afterwards, the dried peptide pellet was resuspended into 15 µl of pH8 buffer (10 mM ammonium acetate, pH 8.0). The peptide solution was eventually fractionated for a final of 20 fractions on an Ultimate 3000 LC system (Thermo Scientific) using reversed-phase chromatography at pH 8.0 with a 1 mm x 150 mm C18 (Thermo Scientific) column. The LC gradient lasted for 75 min LC with the range from 3-50 % of 84 % acetonitrile in 10 mM ammonium acetat, pH 8.0. 12.5 µl/min was acquired as flowrate. 20 fractions were collected at 1 min intervals in concatenated manner. Each fraction was dried under the SpedVac and resuspended in 15 µl of 0.1 % TFA (v/v) for nano-LC-MS/MS analysis (Figure 1A). The peptide fractions took turns to go through an Ultimate 3000 nano RSLC system with Acclaim PepMap 100 µm x 2 cm, C18, 5 µm, 100 Å traping column for 10 min using 0.1 % TFA (v/v), flow rate of 20 µl/min, followed by a separation on Acclaim PepMap RSLC 75 µm x 50 cm, C18, 2 µm, 100 Å main column coupled to a Lumos mass spectrometry (all from Thermo Scientific) with a 110 min LC gradient ranging from 3 to 35 % B (84 % ACN in 0.1 % FA) at a flow rate of 250 nL/min. MS survey scans were acquired in the Orbitrap from 300 to 1500 m/z at a resolution of 120,000 using the ambient air ion at 445.12002 m/z as lock mass. The 15 most intense ions were subjected to collision induced dissociation (CID) in the ion trap, considering a dynamic exclusion of 15 s. CID spectra were acquired with a normalized collision energy of 31 % and an activation time of 10 ms. AGC target values were set to 2x105 for Orbitrap MS and 2x104 for ion trap MSn scans, and maximum injection times were set to 300 ms.

### Data Protocol
Bioinformatics analysis using Proteome discoverer MS raw data were analysed on Proteome Discoverer software 2.1.0.81. In a concrete manner, the raw data were searched against a concatenated target decoy version of mouse Uniprot proteome database (download on July 2015 containing 16,716 target sequences) using Mascot 2.4.1 (Matrix science). Trypsinated peptides with maximum of two missed cleaveages were selected. Carbamidomethylation of Cysteine together with oxidation of Methionine were chosen as fixed and variable modifications, respectively. MS and MS/MS tolerances were set to 10 ppm and 0.5 Da, respectively in accordance. A false discovery rate (FDR) of 1% on the PSM, peptide, and protein level was applied. The quality-controlled data was exported, and only proteins that were identified with ≥1 validated peptide were considered for data analysis. NSAF (normalized spectral abundance factor) value for each protein was calculated based on the number of PSMs (peptide spectrum matches) and the protein index’s length. Nano LC-MS/MS analysis and label-free protein quantitation LC-MS/MS analysis of digested lysate was performed on an Ultimate U3000 RSLC system coupled to a LTQ-Orbitrap Velos mass spectrometer (both from Thermo). First, tryptic peptides were pre-concentrated on an Acclaim PepMap 100 µm x 2 cm, C18, 5 µm, 100 Å trap column using 0.1 % trifluoroacetic acid for 5 min, at a flow rate of 20 µl/min. Next, peptide separation was carried out on an Acclaim PepMap RSLC 75 µm x 50 cm main column packed with C18 resin using a linear gradient from 97 % solvent A (0.1 % formic acid) and 3 % solvent B (84 % acetonitrile in 0.1 % formic acid) to 45 % solvent B over 100 min at a flow rate of 250 nL/min. The data acquisition mode was set to obtain one high-resolution MS scan in the FT part of the mass spectrometer at a resolution of 60,000 full width at half-maximum (at m/z 300) followed by MS/MS scans in the linear ion trap of the 10 most intense ions. Collision induced dissociation (CID) was triggered when the precursor exceeded 100 ion counts. The dynamic exclusion duration was set to 60 s. The ion accumulation time was set to 300 ms (MS) and 50 ms (MS/MS). The automatic gain control (AGC) was set to 1 × 106 for MS survey scans and 1 × 105 for MS/MS scans. Data analysis of the acquired label free quantitation started with importing the MS raw file of each sample into the Progenesis Qi LC-MS software from Nonlinear Dynamics (Newcastle upon Tyne, U.K.) in which triplicate measurements of Vitamin C-treated MSC80 cells were compared to the corresponding triplicates of control MSC80 cells (Figure 1B). Progenesis Qi automatically aligned the MS raw data by selecting one of the LC-MS files as reference. Following the peak picking step, only features within 0-110 min retention time and 300-1500 m/z, with charge states of +2, +3, and +4 were considered for peptide statistics, analysis of variance (ANOVA) and principal component analysis (PCA). MS/MS spectra exported directly from Progenesis Qi in an mgf format were subsequently searched against a decoy database of mouse proteome. The mouse proteome database was download from Uniprot in July 2015, containing 16,716 target sequences. X!Tandem Jackhammer (2015.12.15.2), MS-GF+ Beta (v10282) (12/19/2014) algorithms under the support of SearchGUI 2.7.1 and Mascot 2.4.0 (Matrix Science) search engines were applied. The research criteria were set as followed: trypsin with a maximum of two missed cleavages was selected as enzyme, carbamidomethylation of Cysteine and oxidation of Methionine were set as fixed and variable modification, respectively. Moreover, MS and MS/MS tolerances were set to 10 ppm and 0.5 Da, respectively. PeptideShaker [22] software 1.8.1 for interpretation of peptide and protein identifications was used in which combined search results were filtered at a false discovery rate (FDR) of 1 % on the protein level. The results from the database search were imported back into Progenesis Qi. Peptide sequences containing oxidized Met were excluded for further analysis. Only proteins that were quantified with unique peptides were exported. Then, for each protein, the average of the normalized abundances (obtained from Progenesis) from the triplicate analyses was calculated to determine the ratios between the treated and control samples.

### Publication Abstract
Schwann cells (SCs) are essential in the production of the axon-wrapping myelin sheath and provide trophic function and repair mechanisms in the peripheral nerves. Consequently, well-characterized SC in vitro models are needed to perform preclinical studies including the investigation of the complex biochemical adaptations occurring in the peripheral nervous system (PNS) under different (patho)physiological conditions. MSC80 cells represent a murine SC line used as an in vitro system for neuropathological studies. Here, we introduce the most abundant 9532 proteins identified via mass spectrometry-based protein analytics, and thus provide the most comprehensive SC protein catalogue published thus far. We cover proteins causative for inherited neuropathies and demonstrate that in addition to cytoplasmic, nuclear and mitochondrial proteins and others belonging to the protein processing machinery are very well covered. Moreover, we address the suitability of MSC80 to examine the molecular effect of a drug-treatment by analyzing the proteomic signature of Vitamin C-treated cells. Proteomic findings, immunocytochemistry, immunoblotting and functional experiments support the concept of a beneficial role of Vitamin C on oxidative stress and identified TMX1 as an oxidative stress protective factor, which might represent a promising avenue for therapeutic intervention of PNS-disorders with oxidative stress burden such as diabetic neuropathy.

### Keywords
Tmx1, Msc80, Schwann cells, Protein cataloguing

### Affiliations
Dr. rer. nat. Andreas Roos Project Leader "Tissue Proteomics" Leibniz-Institut für Analytische Wissenschaften ISAS e.V.
ISAS

### Submitter
Vietxuan Phan

### Lab Head
Dr Andreas Roos
Dr. rer. nat. Andreas Roos Project Leader "Tissue Proteomics" Leibniz-Institut für Analytische Wissenschaften ISAS e.V.


