### Accession
PXD041186

### Title
SAA1-dependent reprogramming of adipocytes by tumor cells is associated with triple negative breast cancer aggressiveness

### Description
Cancer development and progression depend on tumor cell intrinsic factors, the tumor microenvironment and host characteristics. Despite the identification of the plasticity of adipocytes, the primary breast stromal cells, both in physiology and cancer, we lack a complete understanding of mechanisms that regulate adipocyte-tumor cell crosstalk. Here we dissected the breast cancer crosstalk with adipocytes and studied relevant molecules. We identified that the ability of breast cancer cells to dedifferentiate adipocytes is intrinsic subtype-dependent, with all breast cancer subtypes, except for HER2+ER+ subtype, capable of inducing this phenomenon. Crosstalk between breast cancer cells and adipocytes in vitro increased cancer stem-like features and recruitment of pro-tumorigenic immune cells, through chemokine production. Serum amyloid A1 (SAA1) was in vitro identified as a regulator of the adipocyte dedifferentiation program in triple-negative breast cancer (TNBC) through CD36 and P2XR7 signaling. In human TNBCs, SAA1 expression was associated with CAA infiltration, inflammation, stimulated lipolysis, stem-like properties and distinct tumor immune microenvironment. Our findings provide evidence that interaction between tumor cells and adipocytes through SAA1 release is relevant to the aggressiveness of TNBC, potentially supporting its targeting.

### Sample Protocol
Collection samples Tumor-derived CM, without FBS, was collected and concentrated using Amicon ultra-centrifugal filters with a 3-kDa cut-off (Millipore, UFC900324), to remove excess salt, and were then subjected to vacuum drying. Dry proteins (40 g) were resuspended in TNTG lysis buffer and subjected to SDS-PAGE (NuPAGE 4-12% Bis-Tris gel, Invitrogen, NP0335BOX). Proteins were fixed (40% methanol and 7% acetic acid), stained with Brilliant Blue G Colloidal Concentrate, and de-stained in buffer (25% methanol, 10% acetic acid, 75% water). MS The protein band of interest was removed, destained, derivatized with iodoacetamide, and in situ digested by trypsin (protein:protease ratio of 10:1), before MS/MS analysis. Peptide separation was achieved on a Thermo Easy-nLC 1000, with a linear gradient, from 95% solvent A (2% ACN, 0.1% formic acid) to 30% solvent B (80% acetonitrile, 0.1% formic acid), over 60 min, from 30% to 60% solvent B over 5 min, and from 60% to 100% solvent B over 2 min, at a constant flow rate of 0.25 µL/min, with a single run time of 75 min. MS data were acquired on a Thermo QExactive–HF, with a data-dependent top 15 method. The survey full scan MS spectra (300–1650 m/z) were acquired in the Orbitrap, with 60,000 resolution, AGC target 3e6, IT 20 ms. For HCD, spectra resolution was set to 15,000, AGC target 1e5, IT 80 ms; normalized collision energy 28% and isolation width of 1.2 m/z. Raw, label-free MS/MS files from Thermo Xcalibur software (version 4.1) were analyzed using Proteome Discoverer software (version 2.2, Thermo Fisher Scientific) and searched with the Sequest algorithm against the proteome of UNIPROT- Homo Sapiens (released 6 March 2019).

### Data Protocol
The minimum required peptide length was set to 6 amino acids, minimum XCorr ≥ 1.5, carbamidomethylation as a fixed modification, with Met oxidation and Asn/Gln deamidation and Nterminal acetylation as variable modifications.

### Publication Abstract
None

### Keywords
Proteomics; nanolc-ms/ms

### Affiliations
University of Milan
DIVAS University of Milan, Italy

### Submitter
Simona Nonnis

### Lab Head
Dr Elisa Maffioli
DIVAS University of Milan, Italy


