### Accession
PXD045688

### Title
Mitochondrial gene expression is required for platelet function and blood clotting

### Description
Platelets are anucleate blood cells that contain mitochondria and regulate blood clotting in response to injury. Mitochondria contain their own gene expression machinery which relies on nuclear encoded factors for the biogenesis of the oxidative phosphorylation (OXPHOS) system to produce energy, ions and metabolites required for thrombosis. The autonomy of the mitochondrial gene expression machinery from the nucleus is not known and platelets provide a valuable model to understand the importance of mitochondrial gene expression in anucleate cells. We generated three different platelet-specific conditional knockout mouse models, each lacking a gene that is essential for mitochondrial gene expression at the level of RNA processing, stability and translation (Elac2Pf4?/Pf4?, Ptcd1Pf4?/Pf4 and Mtif3Pf4?/Pf4?). Loss of ELAC2, PTCD1 or MTIF3, leads to increased megakaryocyte ploidy, elevated circulating levels of reticulated platelets, thrombocytopenia and consequent extended bleeding time. Furthermore, impaired RNA metabolism also reduced agonist induced platelet activation. Transcriptomic and proteomic analyses showed that mitochondrial gene expression and protein synthesis facilitate fibrinolysis, haemostasis and blood coagulation in response to injury.

### Sample Protocol
Blood was collected in 100 µl of Astr Jandl citrate-based anticoagulant (85 mM sodium citrate dihydrate, 69 mM citric acid, 20 mg/ml glucose, pH 4.6) and platelets were isolated by centrifugation at 125 g for 8 minutes at room temperature (brake and acceleration off). The platelet rich plasma was transferred to a new tube with 8 µl of 50 µM prostaglandin (if platelets were not subsequently treated with agonists) and 300 µl platelet wash buffer (140 mM NaCl, 5 mM KCl, 12 mM sodium citrate, 10 mM glucose, 12.5 mM sucrose, pH 6.0) and centrifuged at 125 g for 8 minutes at room temperature. The supernatant was transferred to another tube and 300 µl platelet wash buffer added and centrifuged at 860 g for 5 minutes at room temperature. The platelet pellet was snap frozen for proteomic analyses resuspended in 100 µl of 250 µM AYPGKF and incubated at 37°C for 15 min then centrifuged at 860 g for 5 min at room temperature to separate pellet and activated supernatant. Both pellet and supernatant were snap frozen separately at -80°C for proteomics analyses. 15-20x106 platelets were isolated and resuspended in lysis buffer [6 M guanidinium chloride, 2.5 mM tris(2-carboxyethyl)phosphine hydrochloride, 10 mM chloroacetamide, and 100 mM tris-HCl]. After lysis, samples were diluted 1:10 in 20 mM tris-HCl (pH 8.0), and 100 µg of protein was mixed with 1 µg of Trypsin Gold (Promega) and incubated overnight at 37°C to achieve complete digestion. Peptides were cleaned with homemade STAGEtips (52) (Empore Octadecyl C18; 3M, Germany) and eluted in 60% acetonitrile/0.1% formic acid buffer. Samples were dried in a SpeedVac apparatus (Eppendorf Concentrator plus 5305) at 45°C, the peptides were suspended with 0.1% formic acid, and 1.5 µg of peptides was analyzed by liquid chromatographytandem mass spectrometry (LC-MS/MS). For mass spectrometric analysis, peptides were separated on a 50-cm-long, 75µminternal diameter EASY-spray PepMap C18 column (Thermo Fisher Scientific) using a Dionex UltiMate 3000 Nano-UHPLC system (Thermo Fisher Scientific). The column was maintained at 50°C. Buffers A and B were 0.1% formic acid in water and 0.1% formic acid in acetonitrile, respectively. Peptides were separated on a segmented gradient from 3 to 10% buffer B for 8 min, from 10 to 25% buffer B for 44 min, from 25 to 40% buffer B for 10 min, and from 40 to 95% buffer B for 12 min, at 300 nl/min. Eluting peptides were analyzed on an Orbitrap Fusion mass spectrometer (Thermo Fisher Scientific). The instrument was operated in a data-dependent Top Speed mode, with cycle times of 2 s. The Universal Method template was used with some modifications. Peptide precursor mass/charge ratio (m/z) measurements (MS1) were carried out at 60,000 resolution in the m/z range of 300 to 1500. The MS1 AGC target was set to 1 × 106, and the maximum injection time was set to 300 ms. Precursor priority was set to Most intense, and precursors with charge states 2 to 7 only were selected for higher-energy collisional dissociation fragmentation. Fragmentation was carried out using 27% collision energy. The m/z values of the peptide fragments were measured in the Orbitrap using an AGC target of 5 × 104 and 40-ms maximum injection time. The option Inject ions for All Available Parallelizable Time was enabled. This option varies the maximum injection time and ion targets on the fly based on the available parallelizable time.

### Data Protocol
The Proteome Discoverer (PD; v.2.2.0388; Thermo Fisher Scientific) proteomics software was used to process raw (Xcalibur) MS/MS data generated using the Orbitrap Fusion mass spectrometer. Label-free quantification analysis was carried out in PD using the standard processing and consensus workflow templates, respectively, without applying a normalization procedure according to the software instructions. The processing workflow template contained the Minora Feature Detector node and Sequest HT engine search nodes, and the consensus workflow contained the Feature Mapper and Precursor Ions Quantifier nodes. Peptide fragmentation spectra were searched against the mouse reference proteome from the UniProt database (UP00000589, release-2018_11) (53). The resulting output file contained protein abundance values that were not normalized, and in this file, we identified housekeeping proteins, excluding mitochondrial related proteins, that were used for our normalization using an in-house script. Next, we adjusted specific parameters in the consensus workflow of PD to carry out the normalization. In the Precursor Ions Quantifier node, the Normalization and Scaling tab was changed as follows: (i) the Normalization Mode was changed to the Specific Protein Amount, (ii) a FASTA file of selected housekeeping genes was selected in the Proteins For Normalization, and (iii) the Scaling Mode was left to None. Additionally, the Ratio calculation was set to the Summed Abundance Based, and the Hypothesis Test was set to the ANOVA (Individual Proteins) in the Quan Rollup Hypothesis Testing tab. The consensus workflow analyses of the data with these parameters generated a file that contained normalized protein abundance values.

### Publication Abstract
Platelets are anucleate blood cells that contain mitochondria and regulate blood clotting in response to injury. Mitochondria contain their own gene expression machinery that relies on nuclear-encoded factors for the biogenesis of the oxidative phosphorylation system to produce energy required for thrombosis. The autonomy of the mitochondrial gene expression machinery from the nucleus is unclear, and platelets provide a valuable model to understand its importance in anucleate cells. Here, we conditionally delete Elac2, Ptcd1, or Mtif3 in platelets, which are essential for mitochondrial gene expression at the level of RNA processing, stability, or translation, respectively. Loss of ELAC2, PTCD1, or MTIF3 leads to increased megakaryocyte ploidy, elevated circulating levels of reticulated platelets, thrombocytopenia, and consequent extended bleeding time. Impaired mitochondrial gene expression reduces agonist-induced platelet activation. Transcriptomic and proteomic analyses show that mitochondrial gene expression is required for fibrinolysis, hemostasis, and blood coagulation in response to injury.

### Keywords
Mitochondria, Megakaryocytes, Platelets

### Affiliations
University of Western Australia
Harry Perkins Institute of Medical Research and Centre of Excellence in Synthetic Biology, QEII Medical Centre, Nedlands, Western Australia 6009, Australia

### Submitter
Stefan Siira

### Lab Head
Professor Aleksandra Filipovska
Harry Perkins Institute of Medical Research and Centre of Excellence in Synthetic Biology, QEII Medical Centre, Nedlands, Western Australia 6009, Australia


