### Accession
PXD016912

### Title
Integrative membrane proteome analysis of High and Low-metastatic Lung Cancers

### Description
Membrane proteins play critical roles between the tumor cells and the extra-cellular matrix (ECM) during metastasis. In this study, we performed quantitative proteomic analysis of membrane proteins from two human giant-cell lung carcinoma cell strains, low- (95C) and high-(95D) metastatistic cell lines, and combining with microRNA analysis, we identified a multi-omics regulation module.

### Sample Protocol
Human Non-small cell lung cancer cell lines of low- (95C) or high- (95D) metastatic potential were purchased from the Institute of Biochemistry and Cell Biology of the Chinese Academy of Sciences (Shanghai, China). Cells were seeded in T75 cm2 flasks of 90-95% confluent cells well overnight. Cells were cultured in DMEM and RPMI 1640 medium for 95D and 95C cells separately, and supplemented with 10% fetal bovine serum at 37��������� in atmosphere and 5% CO2, 100 U/mL penicillin, and 100 U/mL. For the isolation and collection of membrane proteins, we used the Pierce������ Cell Surface Protein Isolation Kit (Thermo Scientific), and followed the protocol described by de Wit et al.(Wit et al., 2012) all reagents were cooled to 4������C before protein biotinylation, . The cells were washed twice with ice-cold phosphate buffered saline (PBS) followed by incubation with 0.25 mg/mL Sulfo-NHS-SS-Biotin in 48 mL ice-cold PBS per flask on orbital shaker for 30 minutes at 4������C. Then add 500������L of Quenching Solution to each flask to quench the reaction. After washed with ice-cold PBS, and harvested by gentle scraping and pelleted by centrifugation, the cells were then lysed using the Lysis Buffer (Pierce) which was added with protease inhibitors for 30 minutes on ice with vortexing every 5 minutes for 5 seconds. The cell lysates were centrifuged at 10,000 x g for 2 minutes at 4������C to rid cell remnants. Before clarified supernatant being used to purify biotinylated proteins on NeutrAvidin Agarose, 500 ������L of NeutrAvidin Agarose slurry was added and centrifuge 1 minute at 1000 x g and discard flow-through followed by washing with Pierce Wash Buffer in a provided column (Pierce) trice. The clarified supernatant was added and incubated for 2h at 4 ������C using an end-over-end tumbler to mix vigorously and allow the biotinylated proteins to bind to the NeutrAvidin Agarose slurry. Unbound proteins were removed by washing with 1% Nonidet-P40 and 0.1% SDS in 500 ������L PBS thrice and then washing with 0.1% Nonidet-P40 and 0.5 M NaCl in 500 ������L PBS trice. Finally, the captured proteins were eluted from the biotin-NeutrAvidin Agarose and were collected by column centrifugation at 1,000 x g for 2 minutes. Three biological replicates were obtained for both two cell lines. All protein concentrations were quantified using the BCA protein Assay Kit (Pierce) and the lysates were stored at -20������C for further analysis.

### Data Protocol
Equal amount of proteins from six samples of 2 cell lines was prepared for LC-MS/MS analysis. In brief, 10 mg of proteins were taken from each sample for reduction and alkylation by Dithiothreitol and Iodacetamide, respectively. After trypsin digestion at 37 ������C overnight and desalting, 1 mg of protein digests from each sample were injected for the LC-MS/MS analysis. Protein identification was done by a LC system (Nano Pump, Ultimate 3000, Dionex, Thermofisher) together with an ESI-Q-TOF mass spectrometer (Maxis Impact UHR Q-TOF, Impact, Bruker Daltonik, Germany). We re-dissolved each protein digested sample in 2% acetonitrile with 0.1% formic acid, and then transfered them onto peptide trap columns (100 ������m x 2 cm, 5 ������m,  hermofisher). Then we eluted the trapped peptides to a C18 capillary column (75 ������m x 15 cm, 3 mm, Thermofisher) for 90 min. Finally, we performed full MS scan in a positive mode and then select the five most dominant icons from the initial MS scan for collision-induced dissociation.

### Publication Abstract
Lung cancer is one of the most common human cancers both in incidence and mortality, with prognosis particularly poor in metastatic cases. Metastasis in lung cancer is a multifarious process driven by a complex regulatory landscape involving many mechanisms, genes, and proteins. Membrane proteins play a crucial role in the metastatic journey both inside tumor cells and the extra-cellular matrix and are a viable area of research focus with the potential to uncover biomarkers and drug targets. In this work we performed membrane proteome analysis of highly and poorly metastatic lung cells which integrated genomic, proteomic, and transcriptional data. A total of 1,762 membrane proteins were identified, and within this set, there were 163 proteins with significant changes between the two cell lines. We applied the Tied Diffusion through Interacting Events method to integrate the differentially expressed disease-related microRNAs and functionally dys-regulated membrane protein information to further explore the role of key membrane proteins and microRNAs in multi-omics context. <i>Has-miR-137</i> was revealed as a key gene involved in the activity of membrane proteins by targeting MET and PXN, affecting membrane proteins through protein-protein interaction mechanism. Furthermore, we found that the membrane proteins CDH2, EGFR, ITGA3, ITGA5, ITGB1, and CALR may have significant effect on cancer prognosis and outcomes, which were further validated <i>in vitro</i>. Our study provides multi-omics-based network method of integrating microRNAs and membrane proteome information, and uncovers a differential molecular signatures of highly and poorly metastatic lung cancer cells; these molecules may serve as potential targets for giant-cell lung metastasis treatment and prognosis.

### Keywords
Membrane protein, Lung cancer, High-metastasis cell line2, Multi-omics

### Affiliations
SJTU-Yale Joint Center for Biostatistics, School of Life Sciences and Biotechnology Shanghai Jiao Tong University
Shanghai Jiao Tong University

### Submitter
Yan Kong

### Lab Head
Dr Hui Lu
SJTU-Yale Joint Center for Biostatistics, School of Life Sciences and Biotechnology Shanghai Jiao Tong University


