### Accession
PXD007228

### Title
QUANTITATIVE PHOSPHOPROTEOMIC PROFILING OF PRIMARY RETINOBLASTOMA TUMORS

### Description
Retinoblastoma is a malignant tumor of the retina which most often occurs in children below 5 years of age with an incident rate of about 1 in 15,000 to 18,000 live births. Retinoblastoma is the first ever cancer that was reported to have a genetic basis. It occurs widely due to inactivating mutations in RB1 gene. Gene expression studies, copy number variation analysis, epigenetic profiling including miRNA and methylation of retinoblastoma has been carried to understand the disease mechanism and key players in the disease. Our group has earlier performed differential proteomics of retinoblastoma to identify proteins of therapeutic importance. However, there are no studies to understand the signalling mechanisms associated with retinoblastoma. Hence, global phosphoproteomics of retinoblastoma was carried out to identify signalling events associated with this cancer. Our study identified stress response proteins to be hyper phosphorylated which included H2AFX and sirtuin 1. In particular, Ser140 of H2AFX also known as gamma-H2AX was found to be hyperphosphorylated in retinoblastoma that indicated activation of DNA damage response pathways. We also observed activation of anti-apoptotic proteins in retinoblastoma compared to control. These observations showed activation of survival pathways and signalling networks activated in tumors.

### Sample Protocol
The present study was approved by the institutional ethics board and conducted at Medical research and Vision research foundation, Sankara Nethralaya, India. Control retinas from cadaveric people were collected from C.U. SHAH eye bank, Sankara Nethralaya in the age group of 18-28 years. Tumor tissues were collected with informed consent.  The tumour charecteristics are given in Supplementary table1. In the enucleated eye globe, a part of the tumor was collected for proteomic study and the other part was used for histopathological studies. The collected tumors were snap frozen in liquid nitrogen and transferred to -80°C until used for proteomic analyses.  A schematic representation of the method for phosphoproteome analysis is given in Figure 1. The tissues were pulverized by adding liquid nitrogen and homogenized in homogenizer. The homogenized tissue was lysed by lysis buffers to the final concentration of 2% of SDS in 50mM TEABC, phosphatase inhibitors 1mM NAF, 2.5mM Sodium Pyrophosphate, 1mM Sodium Orthovanadate,1mM β-Glycerophosphate.  Protein estimation was done by BCA protein assay kit (Pierce, Waltham, MA) according to manufacturer’s instructions. Equal amounts of protein (1.5mg) from normal retina and Rb tumors were taken and sulfhydryl bonds in cysteine were reduced by DTT to the final concentration of 5mM, incubated for 20 mins at 60°C. Buffer exchange was performed thrice to reduce the concentration of the SDS from 2% to <0.05% by 8M urea buffer in a 30 kDa MWCO filter (Millipore, USA). Alkylation was carried out with IAA to a final concentration of 20mM for 10 mins at room temperature in dark. The urea in the sample was further removed by buffer exchange with 50mM TEABC. Digestion of the proteins was performed with sequence grade trypsin at 1:20 ratio of trypsin to protein incubated overnight at 37°C. Peptides were vacuum dried until further use. For TMT labelling 0.8 mg of protein was used from each sample. The TMT labels were thawed on ice and reconstituted in 41µL of anhydrous acetonitrile (ACN). The digested peptide sample was reconstituted in 100µL of TEABC (pH 8.0). TMT labels 126, 127C, 129C, 130C were used for labelling RB tumor samples and TMT labels129N, 130N were used for labelling control retina samples and the reaction was incubated for 1 hour at room temperature. After incubation, the reaction was quenched with 8µl of 5% Hydroxylamine. The labelled peptides were lyophilized and stored until use.  Fractionation of the peptides was carried out by bRPLC using XBridge C18, 5µm, 250 x 4.6mm column (Waters Corp, Milford, MA). The lyophilized peptides were reconstituted in 2mL of solvent A (7mM TEABC pH 9.0). Fractionation was performed by increasing gradient of solvent B (7mM TEABC in 90% ACN) on an Agilent 1100 LC with a flow rate of 1mL/min. The fractions were collected in a 96 well plate with 1% formic acid added in the wells. The collected fractions were dried and to each of the fractions 400µL of 80% ACN with 0.1% TFA was used to reconstitute the dried peptides. The fractions were concatenated into 12 fractions and vacuum dried. TiO2 based phosphopeptide enrichment was carried out by reconstituting the peptides in 5% DHB solution and mixing with TiO2 beads. The peptides and TiO2 beads were mixed in a  a proportion of 1:1 and incubated for 1 hour at room temperature. The peptide-bead mixture was centrifuged and saved for further analysis and washed twice with wash solution (3% TFA in 80% ACN). The beads were resuspended in wash solution and transferred onto stage tip with C18 plug. The beads were washed once with wash solution. The phospho enriched peptides were eluted twice with 4% ammonium hydroxide into collection tube containing 4% TFA. The enriched peptides were vacuum dried and stored for downstream analysis. The enriched phosphopeptides were analyzed on LTQ Orbitrap Elite mass spectrometer interfaced with Easy nLC II nanoflow LC system. The peptides were reconstituted in 0.1% formic acid and loaded onto the trap column (75 µm x 2 cm) packed in-house with magic C18 AQ. The peptides were resolved using an analytical column (75 µm x 20 cm) and flow rate was adjusted to 350 nL/min with a linear gradient of 10-35% solvent B (0.1% formic acid and 95% ACN) over 80 mins. The sample loading and reconditioning of the column was done for 40 mins taking the total run time of 120 mins. Data dependent acquisition with full scans at 350-1700 m/z range was carried out using an Orbitrap mass analyzer at a mass resolution of 120,000 at 400 m/z. Fifteen most intense precursor ions were selected for MS/MS fragmentation using high energy collision- induced dissociation fragmentation (HCD) with 32% normalized collision energy and detected at a mass resolution of 30,000 at 400 m/z. Dynamic exclusion was set for 30s with a 10 ppm mass window. Internal calibration was carried out using lock mass option (m/z 445.1200025) from ambient air.

### Data Protocol
The Data analysis was performed on Proteome discoverer 1.4 (1.4.0.288) using Sequest and Mascot search engines. The data was searched against RefSeq protein database. The search parameters included were carbamidomethylation of cysteine, TMT 6 plex labels at the N-terminal of peptide (229.16 Da) and lysine side chain (229.16 Da) as the fixed modification; N-terminal acetylation, oxidation of methionine, phosphorylation at serine, threonine and tyrosine (+79.966 Da) were included in the algorithm as variable modification. MS/MS were searched with precursor mass tolerance of 10 ppm and fragment mass tolerance was set to 0.05 Da. The protease used was specified as trypsin and maximum of two missed cleavages were allowed. The data was searched against target decoy database and the false discovery rate was set to 1% at the peptide level. TMT ratio for each peptide-spectrum match was calculated by the quantitation node and the probability of phosphorylation for each Ser/Thr/Tyr site was calculated by the phosphoRS3.1 node in the Proteome Discoverer. Only the phosphopeptides with >75% site localization was considered for the analysis. The peptides with ratios of ≥1.5 in at least 3 samples were considered as significant differentials and used for further data analysis.

### Publication Abstract
Retinoblastoma is a malignant tumour of the retina which most often occurs in children. Earlier studies on retinoblastoma have concentrated on the identification of key players in the disease and have not provided information on activated/inhibited signalling pathways. The dysregulation of protein phosphorylation in cancer provides clues about the affected signalling cascades in cancer. Phosphoproteomics is an ideal tool for the study of phosphorylation changes in proteins. Hence, global phosphoproteomics of retinoblastoma (RB) was carried out to identify signalling events associated with this cancer. Over 350 proteins showed differential phosphorylation in RB compared to control retina. Our study identified stress response proteins to be hyperphosphorylated in RB which included H2A histone family member X (H2AFX) and sirtuin 1. In particular, Ser140 of H2AFX also known as gamma-H2AX was found to be hyperphosphorylated in retinoblastoma, which indicated the activation of DNA damage response pathways. We also observed the activation of anti-apoptosis in retinoblastoma compared to control. These observations showed the activation of survival pathways in retinoblastoma. The identification of hyperphosphorylated protein kinases including Bromodomain containing 4 (BRD4), Lysine deficient protein kinase 1 (WNK1), and Cyclin-dependent kinase 1 (CDK1) in RB opens new avenues for the treatment of RB. These kinases can be considered as probable therapeutic targets for RB, as small-molecule inhibitors for some of these kinases are already in clinical trials for the treatment other cancers.

### Keywords
Lc-ms, Phosphoproteomics, Retinoblastoma

### Affiliations
Dr. Sailaja V Elchuri,Department of Nanotechnology, VisionResearch Foundation, Sankara Nethralaya, 18-college Road, Nungambakkam, Chennai 600006 India
Sankara Nethralaya

### Submitter
Sailaja Elchuri

### Lab Head
Dr Sailaja Elchuri
Dr. Sailaja V Elchuri,Department of Nanotechnology, VisionResearch Foundation, Sankara Nethralaya, 18-college Road, Nungambakkam, Chennai 600006 India


