### Accession
PXD006716

### Title
A quantitative proteomics screen identifies a substrate repertoire of rhomboid protease RHBDL2 in human cells and implicates it in epithelial homeostasis.

### Description
Rhomboids are intramembrane serine proteases conserved in all kingdoms of life. They regulate epidermal growth factor receptor signalling in Drosophila by releasing signalling ligands from their transmembrane tethers. Their functions in mammals are poorly understood, in part because of the lack of endogenous substrates identified thus far. We used a quantitative proteomics approach to identify the substrate repertoire of the mammalian rhomboid protease RHBDL2 in human cells. We reveal a range of novel substrates that are specifically cleaved by RHBDL2, including the interleukin-6 receptor (IL6R), cell surface protease inhibitor Spint-1, the collagen receptor tyrosine kinase DDR1, NCadherin, CLCP1/DCBLD2, KIRREL, BCAM and others. We further demonstrate that these substrates can be shed by endogenously expressed RHBDL2 and that a subset of them is resistant to shedding by cell surface metalloproteases. The expression profiles and identity of the substrates implicate RHBDL2 in physiological or pathological processes affecting epithelial homeostasis.

### Sample Protocol
HEK293ET cells were stably transduced with lentiviruses expressing mouse EGF (Adrain et al., 2011), then split in two subcultures that were stably transduced with lentivirus encoding either wild type RHBDL2 or its catalytically dead mutant (RHBDL2.S186A). For SILAC labelling, cells expressing wild type RHBDL2 were grown in the presence of light L-Arg and L-Lys while cells expressing RHBDL2.S186A were grown in the presence of heavy L-Arg and L-Lys. Culture supernatants were harvested and mixed 1:1 by volume and concentrated using MWCO filters. Glycoproteins were enriched by wheat-germ agglutinin (WGA) and concanavalin A (ConA) Glycoprotein Isolation Kits (Pierce). The glycoprotein-enriched mixed light and heavy media were precipitated and prepared for analysis using a filter-aided sample preparation method (Wisniewski et al., 2009). Peptides were then reconstituted in a buffer containing pH 3-10 ampholytes and resolved into 12 fractions using an off-gel fractionator. The fractions generated were desalted and lyophilized to dryness. Dissolved peptides were analyzed by nano-scale capillary LC-MS/MS using an UltiMate U3000 HPLC (Thermo Fisher Scientific) and separated by a 110 minute gradient of acetonitrile (5â€“40%(v/v)) on a C-18 column. The peptides were ionized in a nano-flow electrospray ionization source, and analyzed with a hybrid linear quadrupole ion trap mass spectrometer (Orbitrap LTQ XL; Thermo Fisher Scientific). Data-dependent analysis was performed using a resolution of 60,000 for the full MS spectrum, followed by 10 MS/MS spectra in the linear ion trap. MS spectra were collected over an m/z range of 350 to 2,000. MS/MS scans were collected using threshold energy of 35 for collision-induced dissociation.

### Data Protocol
The raw mass spectrometric data files for each fraction were processed and analysed using MaxQuant (v1.5.2.8). The search was performed using a Uniprot/Swissprot human database containing protein isoforms (downloaded 17/03/03) with common contaminants included. Enzyme specificity was set to trypsin, with variable modifications, methionine oxidation and protein N-acetylation. Cysteine carbamidomethylation was considered a fixed modification. Heavy labels were set to R10K8. An MS/MS tolerance of 0.5 Da, a minimal peptide length 6, and 2 missed cleavages were allowed. Proteins were considered identified if they had at least one unique peptide, and quantified if they had at least one quantifiable SILAC pair. The quantified peptides were graphically mapped to the corresponding protein sequences using the Proteator tool http://proteator.appspot.com/ (Nilsea et al., 2016) with I/L nondifferentiation turned on.

### Publication Abstract
Rhomboids are intramembrane serine proteases conserved in all kingdoms of life. They regulate epidermal growth factor receptor signalling in Drosophila by releasing signalling ligands from their transmembrane tethers. Their functions in mammals are poorly understood, in part because of the lack of endogenous substrates identified thus far. We used a quantitative proteomics approach to investigate the substrate repertoire of rhomboid protease RHBDL2 in human cells. We reveal a range of novel substrates that are specifically cleaved by RHBDL2, including the interleukin-6 receptor (IL6R), cell surface protease inhibitor Spint-1, the collagen receptor tyrosine kinase DDR1, N-Cadherin, CLCP1/DCBLD2, KIRREL, BCAM and others. We further demonstrate that these substrates can be shed by endogenously expressed RHBDL2 and that a subset of them is resistant to shedding by cell surface metalloproteases. The expression profiles and identity of the substrates implicate RHBDL2 in physiological or pathological processes affecting epithelial homeostasis.

### Keywords
Substrate, Intramembrane protease, Rhomboid protease, Proteomics, Epithelia, Ectodomain shedding

### Affiliations
Institute of Organic Chemistry and Biochemistry, Czech Academy of Science, Flemingovo n. 2,
Prague, 166 10, Czech Republic
Institute of Organic Chemistry and Biochemistry, Czech Academy of Science, Flemingovo n. 2, Prague, 166 10, Czech Republic

### Submitter
Jana Brezinova

### Lab Head
Dr Kvido Strisovsky
Institute of Organic Chemistry and Biochemistry, Czech Academy of Science, Flemingovo n. 2, Prague, 166 10, Czech Republic


