### Accession
PXD023200

### Title
Competition for electrons favors N2O reduction in denitrifying Bradyrhizobium isolates

### Description
Bradyrhizobia are common members of soil microbiomes and known as N2-fixing symbionts of economically important legumes. Many are also denitrifiers, which can act as sinks or sources for N2O. Inoculation with compatible rhizobia is often needed for optimal N2-fixation, but the choice of inoculant may have consequences for N2O emission. Here, we determined the phylogeny and denitrification capacity of Bradyrhizobium strains, most of them isolated from peanut-nodules. Analyses of genomes and denitrification end-points showed that all were denitrifiers, but only ~1/3 could reduce N2O. The N2O-reducing isolates had strong preference for N2O- over NO3--reduction. Such preference was also observed in a study of other bradyrhizobia and tentatively ascribed to competition between the electron pathways to Nap (periplasmic NO3- reductase) and Nos (N2O reductase). Another possible explanation is lower abundance of Nap than Nos. Here, proteomics revealed that Nap was instead more abundant than Nos, supporting the hypothesis that the electron pathway to Nos outcompetes that to Nap. In contrast, Paracoccus denitrificans, which has membrane-bond NO3- reductase (Nar), reduced N2O and NO3- simultaneously. We propose that the control at the metabolic level, favoring N2O reduction over NO3- reduction, applies also to other denitrifiers carrying Nos and Nap but lacking Nar.

### Sample Protocol
Relative quantification of the denitrification reductases was done for the strain Bradyrhizobium sp. HAMBI 2125, chosen as a representative for the complete denitrifiers (Fig. 1), which all showed the same preference for N2O over NO3- reduction (Figs 2A, S3, S4, S5). Cells were first raised from frozen stock under strictly aerobic conditions as explained above. As the aerobic preculture reached OD600 = 0.05 (2.9*107 cells ml-1), it was used to inoculate 18 flasks (1 ml inoculum flask-1) containing YMB with 1 mM NO3- and with 1 ml N2O (approximately 80 µmol N2O-N flask-1) and 1 % O2 in headspace. Samples for protein analysis were taken i) from the aerobic inoculum; ii) just before transition to anaerobic respiration (22 hpi, when O2 reached ~4.6 µM); iii) during the period of rapid reduction of exogenous N2O (27 and 32 hpi); and iv) during the period of NO3- reduction to N2 (36 hpi). Three replicate flasks were harvested at each time point. Gas kinetics, NO2- concentrations and sampling times for proteomics are shown in Fig. S8. After measuring the OD600, the samples were centrifuged (10 000 g, 4 °C, 10 min) and the cell pellets were stored at -20 °C for protein extraction. In addition, we harvested and froze cells from the aerobic preculture.  To extract the proteins, the cell pellets were thawed, resuspended in lysis buffer (20 mM Tris-HCl pH8, 0.1 % v/v Triton X-100, 200 mM NaCl, 1 mM DTT) and treated with 3x45 s bead beating with glass beads (particle size≤106 μm Sigma) at maximum power and cooling on ice between the cycles (MP Biomedicals™ FastPrep- 24TM, Thermo Fischer Scientific Inc). Cell debris was removed by centrifugation (10 000 g; 5 min) and the supernatant, containing water soluble proteins, was used for proteomic analysis based on Orbitrap-MS.

### Data Protocol
Mass spectral data were acquired using Xcalibur (v.2.2 SP1). MS raw files were processed with the MaxQuant software suite (version 1.6.2.6) for label-free quantification using 'match between runs'. Data was filtered to 1% FDR. To quantify the denitrification reductases we calculated the fraction of the individual enzymes as percentages of the sum of all LFQ (label-free quantitation of proteins) abundances for each time point.

### Publication Abstract
Bradyrhizobia are common members of soil microbiomes and known as N<sub>2</sub> -fixing symbionts of economically important legumes. Many are also denitrifiers, which can act as sinks or sources for N<sub>2</sub> O. Inoculation with compatible rhizobia is often needed for optimal N<sub>2</sub> -fixation, but the choice of inoculant may have consequences for N<sub>2</sub> O emission. Here, we determined the phylogeny and denitrification capacity of Bradyrhizobium strains, most of them isolated from peanut-nodules. Analyses of genomes and denitrification end-points showed that all were denitrifiers, but only ~1/3 could reduce N<sub>2</sub> O. The N<sub>2</sub> O-reducing isolates had strong preference for N<sub>2</sub> O- over NO<sub>3</sub> <sup>-</sup> -reduction. Such preference was also observed in a study of other bradyrhizobia and tentatively ascribed to competition between the electron pathways to Nap (periplasmic NO<sub>3</sub> <sup>-</sup> reductase) and Nos (N<sub>2</sub> O reductase). Another possible explanation is lower abundance of Nap than Nos. Here, proteomics revealed that Nap was instead more abundant than Nos, supporting the hypothesis that the electron pathway to Nos outcompetes that to Nap. In contrast, Paracoccus denitrificans, which has membrane-bond NO<sub>3</sub> <sup>-</sup> reductase (Nar), reduced N<sub>2</sub> O and NO<sub>3</sub> <sup>-</sup> simultaneously. We propose that the control at the metabolic level, favouring N<sub>2</sub> O reduction over NO<sub>3</sub> <sup>-</sup> reduction, applies also to other denitrifiers carrying Nos and Nap but lacking Nar.

### Keywords
Denitrification, Bradyrhizobium

### Affiliations
Norwegian University of Life Sciences
Faculty of Chemistry, Biotechnology and Food Science Norwegian University of Life Sciences Post Office Box 5003 1432, Ås Norway

### Submitter
Magnus Arntzen

### Lab Head
Dr Åsa Helena Frostegård
Faculty of Chemistry, Biotechnology and Food Science Norwegian University of Life Sciences Post Office Box 5003 1432, Ås Norway


