### Accession
PXD017867

### Title
Proteomic analysis of renal biomarkers of kidney allograft fibrosis – a study in renal transplant patients

### Description
Renal transplantation is the preferred treatment of end stage renal disease, but allograft survival is limited by development of interstitial fibrosis and tubular atrophy in response to various stimuli. Much effort has been put into identifying new protein markers of fibrosis to support the diagnosis. In present work, we performed an in-depth quantitative proteomics analysis of allograft biopsies from 31 prevalent renal transplant patients and identified correlated the quantified proteins with the volume fraction of fibrosis as determined by a morphometric method. Linear regression analysis identified four proteins that were highly associated with the degree of interstitial fibrosis, namely Coagulation Factor XIII A chain (estimate 18.7, adjusted p<0.03), Uridine Phosphorylase 1 (estimate 19.4, adjusted p<0.001), Actin-related protein 2/3 subunit 2 (estimate 34.2, adjusted p<0.05) and Cytochrome C Oxidase Assembly Factor 6 homolog (estimate -44.9, adjusted p<0.002) even after multiple testing. Proteins that were negatively associated with fibrosis (p < 0.005) were primarily related to normal metabolic processes and respiration, whereas proteins that were positively associated with fibrosis (p < 0.005) were involved in catabolic processes, cytoskeleton organization and immune response. The identified proteins may be candidates for further validation with regards to renal fibrosis. The results support the notion that cytoskeleton organization and immune responses are prevalent processes in renal allograft fibrosis.

### Sample Protocol
Sample preparation: Sections of formalin fixed, paraffin-embedded kidney biopsies were washed three times in chloroform. Proteins were extracted by dissolving the deparaffinized tissue sections in extraction buffer(1M dithiothretiol (DTT), 0.2M tetraethylammonium bicarbonate (TEAB) and 10% sodium dodecyl sulphate (SDS)) followed by two rounds of ultrasonification (15 minutes of ultrasonification/15 minutes of cooling on ice) and incubated at 99°C for 20 min then at 80°C for 120 min. Protein alkylation was done by adding a 200 mM iodoacetamide (IAA) solution to a final DTT:IAA concentration ratio of 1:3. Acetone precipitated proteins were re-dissolved in 5 µl 8 M urea with 1 µg LysC and incubated at 30°C for 4 hours followed by a further dilution to 1 M urea, addition of 2 µg trypsin and overnight incubation at 30 °C. The resulting tryptic peptides were isotopically labelled using the 10-plex tandem mass tag (TMT, ThermoFisher Scientific, Waltham, Massachusetts, USA). Peptide samples were randomly labelled with the 127N, 127C, 128N, 128C, 129N, 129C and 130N mass tags, whereas a pool of all samples were labelled with mass tag 126 that served as internal standard. Tagged peptides were mixed into 10 mixed peptide samples that were fractionated using hydrophilic interaction chromatography (HILIC), as described below.HILIC fractionation: Briefly, the labelled peptide mixtures were re-dissolved in 90% ACN/0.1% TFA and 15 µl aliquots corresponding to approximately 25 µg peptides were injected onto an in-house-packed TSK gel Amide-80 HILIC 300 µm×300mm capillary high-performance liquid chromatography (HPLC) column and fractionated into 42 fractions by a Dionex UltiMate 3000 nanoHPLC using a linear 59 min gradient (85.5%B to 54%B; solvent A: 0.1%TFA; solvent B: 90% ACN/0,1% TFA) at a flowrate of 6 µl per minute. Fractions were automatically collected in micro-well plates at 1-minute intervals after UV detection at 210 nm. The fractions were dried by vacuum centrifugation and re-dissolved in 10 µl 0.1% TFA and analysed by nano-LC–MS/MS, as described below. NanoLC-MS/MS: A Q-Exactive mass spectrometer (Thermo Fisher Scientific, Bremen, Germany) equipped with a nano-HPLC interface (Dionex UltiMate 3000 nano HPLC) was applied for nano-LC-MSMS analyses. The samples (5µL) were loaded onto a custom-made, fused capillary pre-column (2 cm length, 360 µm OD, 75 µm ID) and separated on a custom-made fused capillary column (20 cm length, 360 µm OD,100 µm ID-both columns were packed with ReporSil Pur C13 3 µm resin) using a linear gradient from 95% solution A (0.1% formic acid) to 30% B (100% acetonitrile in 0.1% formic acid) over 51 minutes followed by 5 minutes at 90% B and 5 minutes at 98% A at a flow rate of 300 nL per minute. Acquisition of mass spectra was done in positive ion mode using an automatic data-dependent switch between an Orbitrap survey MS scan in the mass range from 400 to 1200 m/z; and high-energy collisional dissociation fragmentation (HCD) and Orbitrap detection of the 15 most intense ions observed in the MS scan. The Orbitrap target value for MS and MSMS scans were 1,000,000 and 50,000 ions, at resolutions of 70,000 and 35,000 at m/z 200. Fragmentation in the HCD cell was performed at normalized collision energy of peptides and 31 eV for TMT-labelled peptides. The ion selection threshold was set to 17,000 counts. Selected sequenced ions were dynamically excluded for 60 s.

### Data Protocol
The Sequest search engine and Mascot search Engine (v. 2.2.3) integrated with the Proteome Discoverer version 2.1 software (Thermo Scientific) was used to search raw data files with the following criteria; Protein database: Uniprot/Swissprot (downloaded 7th November 2012, 452,768 entries) and restricted to humans. Trypsin, one missed cleavage allowed, carbamidomethylation at cysteines and 10-plex TMT labelling at lysine and N-terminal amines were set as fixed parameters, while methionine oxidation and deamidation were set as dynamic. Precursor mass tolerance was set to 8 PPM and MSMS tolerance was set to 0.05 Da. Peptide data were extracted using a Mascot significance threshold of 0.05 and a minimum peptide length of 6. False discovery rate (FDR) was calculated using a decoy database search, and only high-confidence peptide identifications (false discovery rate < 1%) were included. Search files were further processed using Proteome Discoverer software. The interaction plots were performed using String 10.5 software (http://string-db.org/) (29).  Statistical analysis Analyses of baseline values were performed using Stata15 (StataCorp, Texas, USA). Normality of data was evaluated by histograms and QQ-plots. Depending on normality, data are described by mean ±SD or median (range). Due to the non-normal distribution of fibrosis data, a non-parametric correlation test was used (Spearman’s rank correlation). The association of specific proteins with fibrosis was evaluated using R software. For each protein, we applied a multiple linear regression model for the volume fraction of fibrosis with sex, age and donor type (deceased/living related) as covariates. The resulting p-values were corrected for multiple testing by the Benjamini-Hochberg procedure. To find a subset of proteins predicting the degree of fibrosis we used LASSO with stability selection (30). This method identifies the proteins with the highest predictive value with regards to fibrosis. The LASSO was applied to a linear model where all proteins with no missing values (1973 proteins), sex, age, donor type, and time since transplantation were used as covariates. This was repeated 1000 times on a randomly selected subsample consisting of 16 samples. The probability of selecting each protein was computed.

### Publication Abstract
Renal transplantation is the preferred treatment of end stage renal disease, but allograft survival is limited by the development of interstitial fibrosis and tubular atrophy in response to various stimuli. Much effort has been put into identifying new protein markers of fibrosis to support the diagnosis. In the present work, we performed an in-depth quantitative proteomics analysis of allograft biopsies from 31 prevalent renal transplant patients and correlated the quantified proteins with the volume fraction of fibrosis as determined by a morphometric method. Linear regression analysis identified four proteins that were highly associated with the degree of interstitial fibrosis, namely Coagulation Factor XIII A chain (estimate 18.7, adjusted <i>p</i> &lt; 0.03), Uridine Phosphorylase 1 (estimate 19.4, adjusted <i>p</i> &lt; 0.001), Actin-related protein 2/3 subunit 2 (estimate 34.2, adjusted <i>p</i> &lt; 0.05) and Cytochrome C Oxidase Assembly Factor 6 homolog (estimate -44.9, adjusted <i>p</i> &lt; 0.002), even after multiple testing. Proteins that were negatively associated with fibrosis (<i>p</i> &lt; 0.005) were primarily related to normal metabolic processes and respiration, whereas proteins that were positively associated with fibrosis (<i>p</i> &lt; 0.005) were involved in catabolic processes, cytoskeleton organization and the immune response. The identified proteins may be candidates for further validation with regards to renal fibrosis. The results support the notion that cytoskeleton organization and immune responses are prevalent processes in renal allograft fibrosis.

### Keywords
Kidney fibrosis, Nano-lc-msms, Transplant

### Affiliations
Odense University Hospital, Odense, Denmark
Proteomics Lab, Dept of Clinical Biochemistry and Pharmacology, Odense University Hospital, Odense, Denmark

### Submitter
Hans Christian Beck

### Lab Head
Dr Hans Christian Beck
Proteomics Lab, Dept of Clinical Biochemistry and Pharmacology, Odense University Hospital, Odense, Denmark


