### Accession
PXD023171

### Title
Trait tacking for predictive evolution of metabolic pheotypes

### Description
Adaptive laboratory evolution is highly effective for improving desired traits through natural selection. However, its applicability is inherently constrained to growth-correlated traits precluding traits of interest that incur a fitness cost, such as metabolite secretion. Here, we introduce the concept of tacking trait enabling naturalÂ selection of fitness-costly metabolic traits. The concept is inspired from the tacking maneuver used in sailing for traversing upwind.Â We use first-principle metabolic models to design an evolution niche wherein the tacking trait and fitness become correlated. Adaptive evolution in this niche, when followed by the reversal to the original niche, manifests in the improvement of the desired trait due to biochemical coupling between the tacking and the desired trait. We experimentally demonstrate this strategy, termed EvolveX, by evolving wine yeasts for increased aroma production. Our results pave the way for precision laboratory evolution for biotechnological and ecological applications.

### Sample Protocol
Biological triplicates of the parental and the evolved bacterial strains were inoculated in CDM46R- with an initial OD600 of 0.01. For the extraction of total proteome 10 mL of each culture were transferred into ice-cold 15 mL FalconÂ® tubes which were centrifuged immediately at 3000 rpm for 3 min at 4oC (Eppendorf centrifuge). The supernatant from the centrifugation was discarded and the cell pellets were washed once with 1 mL of cold PBS buffer. The washed pellets were snapped frozen with liquid nitrogen and stored at âˆ’80 oC. For the extraction, the cell pellets were lysed with 0.1% RapiGest (Waters) in 100 mM ammonium bicarbonate, followed by mechanical disruption with 3 rounds of sonication (1 cycle: 10 sec sonication and 10 sec rest on ice per round). Sonication was followed by 2 cycles of bead beating (200 Î¼L glass beads, 150-212 nm acid washed, Sigma), each cycle lasting 20 sec at 4 Mz/sec with 1 min cooling intervals between the cycles. Reduction of disulphide bridges in cysteine containing proteins was performed with dithiothreitol (56Â°C, 30 min, 10 mM in 50 mM HEPES, pH 8.5). Reduced cysteines were alkylated with 2-chloroacetamide (RT, in the dark, 30 min, 20 mM in 50 mM HEPES, pH 8.5). Samples were prepared using the SP3 protocol18 and trypsin (sequencing grade, Promega) was added in an enzyme to protein ratio 1:50 for overnight digestion at 37Â°C. Peptides were labelled TMT6plex19 Isobaric Label Reagent (ThermoFisher) according the manufacturerâ€™s instructions. For further sample clean up an OASISÂ® HLB ÂµElution Plate (Waters) was used. Offline high pH reverse phase fractionation was carried out on an Agilent 1200 Infinity high-performance liquid chromatography system, equipped with a Gemini C18 column (3 Î¼m, 110 Ã…, 100 x 1.0 mm, Phenomenex)20 resulting in 12 fractions. After fragmentation, the peptides were separated using an UltiMate 3000 RSLC nano LC system (Dionex) fitted with a trapping cartridge (Âµ-Precolumn C18 PepMap 100, 5Âµm, 300 Âµm i.d. x 5 mm, 100 Ã…) and an analytical column (nanoEaseâ„¢ M/Z HSS T3 column 75 Âµm x 250 mm C18, 1.8 Âµm, 100 Ã…, Waters). Trapping was carried out with a constant flow of trapping solution (0.05% trifluoroacetic acid in water) at 30 ÂµL/min onto the trapping column for 6 min. Subsequently, peptides were eluted via the analytical column running solvent A (0.1% [v/v] formic acid in water) with a constant flow of 0.3 ÂµL/min, with increasing percentage of solvent B (0.1% [v/v] formic acid in acetonitrile) from 2% to 4% in 4 min, from 4% to 8% in 2 min, then 8% to 28% for a further 37 min, in another 9 min from 28%-40%, and finally 40%-80% for 3 min followed by re-equilibration back to 2% B in 5 min. The outlet of the analytical column was coupled directly to an Orbitrap QExactiveâ„¢ plus Mass Spectrometer (Thermo) using the Nanospray Flexâ„¢ ion source in positive ion mode. The peptides were introduced into the QExactive plus via a Pico-Tip Emitter 360 Âµm OD x 20 Âµm ID; 10 Âµm tip (New Objective) and an applied spray voltage of 2.2 kV. The capillary temperature was set at 275Â°C. Full mass scan was acquired with mass range 375-1200 m/z in profile mode with resolution of 70000. The filling time was set at maximum of 100 ms with a limitation of 3x106 ions. Data dependent acquisition (DDA) was performed with the resolution of the Orbitrap set to 17500, with a fill time of 50 ms and a limitation of 2x105 ions. A normalized collision energy of 32 was applied. Dynamic exclusion time of 20 s was used. The peptide match algorithm was set to â€˜preferredâ€™ and charge exclusion â€˜unassignedâ€™, charge states 1, 5 - 8 were excluded. MS data was acquired in profile mode.

### Data Protocol
The acquired data were processed using IsobarQuant21 and Mascot (v2.2.07). A Uniprot L. plantarum proteome database (UP000000432) containing common contaminants and reversed sequences was used, with the addition of the sequences of the absent riboflavin biosynthetic proteins (RibD and RibF). The search parameters were the following: Carbamidomethyl (C) and TMT10 (K) (fixed modification), Acetyl (N-term), Oxidation (M) and TMT10 (N-term) (variable modifications). A mass error tolerance of 10 ppm was set for the full scan (MS1) and for MS/MS (MS2) spectra of 0.02 Da. Trypsin was selected as protease with an allowance of maximum two missed cleavages. A minimum peptide length of seven amino acids and at least two unique peptides were required for a protein identification. The false discovery rate on peptide and protein level was set to 0.01.

### Publication Abstract
None

### Keywords
Ale s. cerevisiae wine aroma

### Affiliations
EMBL
EMBL Heidelberg

### Submitter
Frank Stein

### Lab Head
Dr Kiran Raosaheb Patil
EMBL Heidelberg


