### Accession
PXD000938

### Title
TARA Oceans Viral Proteins -  Illuminating structural proteins in viral "dark matter" with metaproteomics

### Description
A grand challenge in biology is the lack of annotation for much of global sequence space. In particular, while microbes and their viruses are critical to Earth System functioning, surveys of their communities in nature routinely return genes of unknown function, particularly for viruses. This hampers understanding and predictive capability, and reflects a number of database limitations. Here we apply high-resolution environmental metaproteomics to 4 purified viral concentrates from the Tara Oceans Expedition to identify in situ proteins extracted from uncultivated viral particles. Using matched metagenomic databases, we identified 1875 proteins belonging to 549 non-redundant protein clusters that are predominantly of unknown function. These data help identify function for structure-associated proteins in known abundant tailed viruses (e.g., pelagiphages and cyanophages) and validate newly-identified uncultured viral sequences linked to abundant novel hosts. One protein cluster alone comprised 22% of the identified structure-associated proteins, with ~46% of the observed spectra. Structural modeling of this highly abundant viral protein cluster indicates its likely role as a novel, previously unknown capsid. Together these analyses provide much-needed, culture-independent structure-associated protein annotations critical for identifying viral signals across diverse datasets to better elucidate viral roles in nature.

### Sample Protocol
Protein sample preparation: Viral protein extracts were processed via a filter-aided sample preparation method (FASP, Protein Discovery, Knoxville TN) according to the manufacture‚Äôs instructions. Modifications were made to the FASP method and yielded two methods, FASP1 and FASP2. Data were acquired from both sample preparation methods and presented in this paper.  FASP1: Briefly, viral particle samples (150 ÔÅ≠l) were denatured by adding 300 ÔÅ≠l of 8 M Urea solution (Urea, Tris-HCl, and 10mM DTT) and nutated for 30 minutes at room temperature. Samples were then transferred to a provided spin filter, centrifuged for 14,000 x g, and the flow-through discarded. Fresh urea sample solution (no SDS or DTT) was added, centrifuged, and the flow-through discarded. 10X iodoacetamide solution and urea sample solution was added to the spin filter and incubated for 20 minutes in the dark. Samples were centrifuged and flow-through discarded. Urea sample solution was added to the spin filter, centrifuged, and flow-through discarded for a total of 3 times. Provided 50 mM ammonium bicarbonate solution was added to the spin filter, centrifuged, flow-through discarded, and the spin filter was transferred to a new collection tube. Digestion solution (20 Œºg trypsin, Promega, Madison, WI) was added and incubated at 37 ÔÇ∞C overnight. 50 mM ammonium bicarbonate solution was added to solution, centrifuged, and repeated once. Provided 0.5 M NaCl solution was added and centrifuged. The final volume of the digested protein filtrate was adjusted to 0.3 mL with 0.1% formic acid and divided into two aliquots. Aliquots were run on two instruments: LTQ Orbitrap and LTQ Orbitrap Velos Pro.  FASP2: Briefly, viral particle samples (100 ÔÅ≠l) were suspended in 100 ÔÅ≠l of 1% SDS Tris-HCl buffer and incubated at 70 ÔÇ∞C for 5 minutes. Then, 200 ÔÅ≠l of 1% SDS/10 mM DTT/8 M Urea Tris-HCl solution was added and nutated for 45 minutes at room temperature. Samples were then transferred to a provided spin filter, centrifuged for 14,000 x g, and the flow-through discarded. Fresh urea sample solution (no SDS or DTT) was added, centrifuged, and the flow-through discarded. 10X iodoacetamide solution and urea sample solution was added to the spin filter and incubated for 20 minutes in the dark. Samples were centrifuged and flow-through discarded. Urea sample solution was added to the spin filter, centrifuged, and flow-through discarded for a total of 3 times. Provided 50 mM ammonium bicarbonate solution was added to the spin filter, centrifuged, flow-through discarded, and the spin filter was transferred to a new collection tube. Digestion solution (10 Œºg trypsin, New England BioLabs, Ipswich, MA) was added and incubated at 37 ÔÇ∞C overnight. 50 mM ammonium bicarbonate solution was added to solution, centrifuged, and repeated once. Provided 0.5 M NaCl solution was added and centrifuged. The final volume of the digested protein filtrate was adjusted to 0.3 mL with 0.1% formic acid and divided into three aliquots. Aliquots were run on two instruments: LTQ Orbitrap Velos Pro and Q Exactive.  LC-MS/MS analysis: After sample preparation, individual aliquots of the complex peptide mixture were loaded overnight onto a split phase 2D RP-SCX back column. The strong cation exchange (SCX) phase was 150 ÔÅ≠m x ~3-5 cm (Luna SCX, 5 ÔÅ≠m particle size, 100 √Ö pore size, Phenomonex, Torrance, CA) and the reverse phase was 150 ÔÅ≠m x ~3-5 cm (Jupiter C18, 3 ÔÅ≠m particle size, 300 √Ö pore size, Phenomonex). After loading, the RP-SCX column was connected to the HPLC and washed with 100% aqueous solvent for 5 minutes and then ramped up to 100% organic solvent (70% acetonitrile, 0.1% formic acid) over 10 minutes. This migrates peptides from the RP phase onto the SCX phase and effectively desalts the samples. The back column was then connected to a 100 ÔÅ≠m x 15 cm RP front column (Jupiter C18, 3 ÔÅ≠m particle size, 300 √Ö pore size, Phenomonex). Samples were  analyzed via a  MudPIT strategy8. A two-dimensional separation was performed with a quaternary HPLC pump (U3000, Dionex Thermo Fisher).    During each of 4 cycles for 8 hour runs (LTQ Orbitrap Velos Pro), 6 cycles for 12 hour runs (LTQ Orbitrap), and 2 cycles for 4 hour runs (Q Exactive), an initial salt step gradient eluted peptides from the SCX column onto the RP column, and a subsequent reverse-phase gradient eluted peptides from the RP column.  Eluting peptides were ionized via a nanospray source (Proxeon Biosystems, Odense, Denmark) and introduced into 3 mass spectrometer instrument systems: 1.) LTQ Orbitrap, 2.) LTQ Orbitrap Velos Pro, and 3.) Q Exactive (Thermo Fisher Scientific, San Jose, CA). For a detailed list of which samples, sample preparation methods were run on the three different mass spectrometers

### Data Protocol
The resultant MS/MS spectra were analyzed by 3 automated informatic pipelines. MS/MS spectra were searched against a matched database consisting of metagenomic contigs from the same sample consisting of i.) 249,381 (22SUR), ii.) 209,504 (39SUR), iii.) 189,606 (67SUR), or iv.) 155,234 (39DCM) proteins predicted using Prodigal 2.504 and common contaminant proteins (trypsin and human keratins). 1.) SEQUEST, version .2710 and DTASelect, version 1.911.  Raw spectra were extracted via the readW program (ISB version 4.3.1). Searches using the metagenomic databases were performed with the following parameters: parent mass tolerance, 3.0; fragment ion tolerance, 0.5; up to 4 missed cleavages allowed, variable modification of carboxymethyl cysteine (+58.00 Da) and fully tryptic peptides only. SEQUEST outputs were sorted and filtered via DTASelect with the following parameters: delCN, >0.08; Xcorr, >1.8 (+1), >2.5 (+2), >3.5 (+3); +1 minimum charge state, +3 maximum charge state, 2 peptides/protein; and a strict FDR of 0.01.   2.) Proteome Discoverer version 1.4 using the Percolator validation algorithm (citation) The following workflow was used in the analyses of MS/MS spectra for each sample: Spectrum selector ÔÉ† Scan event filter ÔÉ† SEQUEST HT ÔÉ† Percolator. Within the spectrum selector and scan even filter processing mode default settings were set. The precursor charge state (high/low), retention time, minimum peak count, and total intensity threshold were all set to default settings. The max and min precursor mass settings were 5000 and 500 Da, respectively. The following settings were selected within the Sequest HT Search processing node: fully tryptic peptides; Max missed cleavage sites = 4; Min and Max peptide length, 6 and 144, respectively; Max delta Cn = 0.05; Precursor mass tolerance = 10 ppm; Fragment Mass Tolerance = 0.5 Da; and modification of carboxymethyl cysteine (+58.00 Da). The criteria used for acceptance of peptide assignments are as follows: 2 peptides/protein, and high confidence Xcorr value for each charged state ranging from 1 to 4 was 1.8, 2.5, 3.5, and 2.6 respectively. Target value for a decoy database search was applied: strict FDR of 0.01.  3.) Trans proteomic pipeline (TPP) version 4.6 rev 312 using X! Tandem version Cyclone (2013.2.01)13, a public domain program (www.thegpm.org/tandem). Raw files were converted into mzXML format using msconvert version 2 via ProteoWizard14. The following parameters were set: fragment monoisotopic mass error = 0.5, Max parent charge = 4, Min parent m+h = 500, Precursor mass tolerance = 10 ppm, Max missed cleavages = 4, Max valid e-value = 0.1, and Min ion count = 4. X! Tandem output was then converted to  pepxml format via TPP. Data were then analyzed by the built-in software component PeptideProphet using default settings. Output data from PeptideProphet were then processed through ProteinProphet, a statistical probability component in TPP. Acceptable proteins possessed a 99% probability or higher.     For robust spectral counting quantitation analyses, only one subset of data (FASP2: 8 hr OVP) was individually analyzed across all 3 analytical search pipelines. Shared spectra were distributed according to the ratio of unique spectra for each protein, (per Mondav & Woodcroft et al, 2014), by the following equation:    where   is the number of unique spectra for the ùëñ-th protein,   is number of shared spectra for the i-th protein, and   is the total number of spectra recruited to the ùëñ-th protein. In cases where no unique spectra ( ) were present for any peptides in the group, spectra were not distributed and no quantitation was possible for the associated proteins. We note these proteins‚Äô possible presence in the list

### Publication Abstract
Viruses are ecologically important, yet environmental virology is limited by dominance of unannotated genomic sequences representing taxonomic and functional "viral dark matter." Although recent analytical advances are rapidly improving taxonomic annotations, identifying functional dark matter remains problematic. Here, we apply paired metaproteomics and dsDNA-targeted metagenomics to identify 1,875 virion-associated proteins from the ocean. Over one-half of these proteins were newly functionally annotated and represent abundant and widespread viral metagenome-derived protein clusters (PCs). One primarily unannotated PC dominated the dataset, but structural modeling and genomic context identified this PC as a previously unidentified capsid protein from multiple uncultivated tailed virus families. Furthermore, four of the five most abundant PCs in the metaproteome represent capsid proteins containing the HK97-like protein fold previously found in many viruses that infect all three domains of life. The dominance of these proteins within our dataset, as well as their global distribution throughout the world's oceans and seas, supports prior hypotheses that this HK97-like protein fold is the most abundant biological structure on Earth. Together, these culture-independent analyses improve virion-associated protein annotations, facilitate the investigation of proteins within natural viral communities, and offer a high-throughput means of illuminating functional viral dark matter.

### Keywords
Virus, Proteins, Viral, Proteomics, Viral proteomics, Tara

### Affiliations
University of Arizona, Department of Ecology and Evolutionary Biology, Tucson Marine Phage Lab.
University of Arizona

### Submitter
Robert Jones

### Lab Head
Dr Matthew B. Sullivan
University of Arizona, Department of Ecology and Evolutionary Biology, Tucson Marine Phage Lab.


