### Accession
PXD032266

### Title
DIA analysis of paired primary and recurrent breast cancers

### Description
Molecular profiling of breast cancer has achieved great depth in defining the mutational landscapes and molecular profiles of primary tumors, though little is still known regarding cancer evolution into a recurrence. Proteogenomic workflows are particularly useful in defining the multi-layered biology of complex diseases by combining genomic, transcriptomic, and proteomic technologies so to inform not only on mutational processes, but also on their repercussion at the effector level, proteins. We employed our recently developed proteogenomic workflow to analyze a cohort of 27 primary breast cancers and their matched loco-regional recurrences by whole genome sequencing, RNA sequencing, and mass spectrometry.

### Sample Protocol
All breast cancer (PTs and LRs) tissues were processed using the AllPrep DNA/RNA/Protein (Qiagen) protocol. Tissue lysis was performed by re-suspending ~30 mg of sliced frozen tissue in a solution containing 1% β-mercaptoethanol in RLT buffer (supplemented with antifoam agent; ID 19088, Qiagen). Next, steel beads (ID 79656, Qiagen) were added and samples were incubated in a Tissue Lyser LT (Qiagen) for 4 min at 50Hz. Steel beads were then removed 400µL of 1% β-mercaptoethanol in RLT buffer was added to samples, which were then centrifuged at 14,000xg for 5 min. Supernatants were transferred to new tubes frozen. DNA, RNA, and protein extraction was performed according to manufacturer instructions (AllPrep DNA/RNA/Protein minikit; Qiagen). Each spin column flowthrough (DNA, RNA, protein) was stored at -80C until analysis (sequencing or mass spectrometry; MS). Protein flow-throughs from the AllPrep protocol were precipitated in ice-cold methanol, as previously described27. Briefly, protein pellets were then suspended in 100 mM Tris (pH 8.0) buffer containing 100 mM dithiotrheitol and 4% w/V sodium-dodecyl-sulphate, and incubated at 95ºC for 30 min under mild agitation. Samples were then cooled to room temperature, diluted in 8 M urea in 100 mM Tris (pH 8.0) buffer, and loaded on 30 KDa molecular filters (Millipore) and centrifuged at 14,000xg for 20 min. Filters were washed with urea buffer and centrifuged at 14,000xg for 10 min. Proteins were alkylated with iodoacetamide in urea buffer (30 min in the dark), washed with urea buffer and tri-ethyl-ammonium bicarbonate buffer (pH 8.0), and trypsin was added (enzyme-protein ratio 1:50; incubation at 37ºC for 16 h, 600 RPM). Filters were then centrifuged at 14,000xg for 20 min to retrieve tryptic peptides, loaded on C18 (3 stacked layers; 66883-U, Sigma) stage tips (pretreated with methanol, 0.1% formic acid (FA) in 80% acetonitrile solution, and 0.1 % FA in ultrapure water), washed with 0.1 % FA in ultrapure water solution, and eluted with 0.1% FA in 80% acetonitrile. Eluates were then dried and subjected to SP3 peptide purification, as previously described28. Briefly, 2 µL of SP3 beads (1:1 ratio of Sera Mag A and Sera Mag B re-suspended in ultrapure water; Sigma) was added to dried peptides and incubated for 2 min under gentle agitation. A volume of 200 µL of acetonitrile was then added and samples were incubated for 10 min under agitation. Sample vials were then placed on a magnetic rack and washed again with acetonitrile for 10 min. Elution was performed by adding 200 µL of 2% dimethyl sulfoxide in water to the beads-peptides mixture and incubating them for 5 min under agitation. Supernatants were then collected, dried, and stored at -80ºC until MS analysis.

### Data Protocol
Tryptic peptide mixtures were subjected to data-independent acquisition (DIA) MS analysis. Samples were eluted in a 120 min gradient (flow: 300 nl/min; mobile phase A: 0.1% FA in ultrapure water; mobile phase B: 80% acetonitrile and 0.1% FA) on a Q-Exactive HFX (Thermo-Fisher) instrument coupled online to an EASY-nLC 1200 system (Thermo-Fisher). Digested peptides were separated by RP-HPLC (ID 75 µm × 50 cm C18 2 µm 100 Å resin; Thermo-Fisher). Gradient was run as follows: 10-30% B in 90 min; 30-45% B in 20 min; 45-90% B in 30 s, and 90% B for 9 min. One high resolution MS scan (resolution: 60,000 at 200 m/z) was performed and followed by a set of 32 DIA MS cycles with variable isolation windows (resolution: 30,000 at 200 m/z; isolation windows: 13, 14, 15, 16, 17, 18, 20, 22, 23, 25, 29, 37, 45, 51, 66, 132 m/z; overlap between windows: 0.5 m/z). Ions within each window were fragmented by HCD (collision energy: 30). Automatic gain control (AGC) target for MS scans was set to 1e6 for MS and MS/MS scans, with ion accumulation time set to 100 ms and 120 ms for MS and MS/MS, respectively. Protein intensities were derived by employing our previously established computational workflow17

### Publication Abstract
Ipsilateral breast tumor recurrence (IBTR) is a clinically important event, where an isolated in-breast recurrence is a potentially curable event but associated with an increased risk of distant metastasis and breast cancer death. It remains unclear if IBTRs are associated with molecular changes that can be explored as a resource for precision medicine strategies. Here, we employed proteogenomics to analyze a cohort of 27 primary breast cancers and their matched IBTRs to define proteogenomic determinants of molecular tumor evolution. Our analyses revealed a relationship between hormonal receptors status and proliferation levels resulting in the gain of somatic mutations and copy number. This in turn re-programmed the transcriptome and proteome towards a highly replicating and genomically unstable IBTRs, possibly enhanced by APOBEC3B. In order to investigate the origins of IBTRs, a second analysis that included primaries with no recurrence pinpointed proliferation and immune infiltration as predictive of IBTR. In conclusion, our study shows that breast tumors evolve into different IBTRs depending on hormonal status and proliferation and that immune cell infiltration and Ki-67 are significantly elevated in primary tumors that develop IBTR. These results can serve as a starting point to explore markers to predict IBTR formation and stratify patients for adjuvant therapy.

### Keywords
Recurrence, Primary, Breast cancer, Dia

### Affiliations
Division of Surgery, Oncology, and Pathology, Department of Clinical Sciences, Lund University, Solvegatan 19, SE-22362, Lund, Sweden. Department of Surgery, Skåne University Hospital, Lund, Sweden.
Lund University

### Submitter
Tommaso De Marchi

### Lab Head
Dr Emma Nimeus
Division of Surgery, Oncology, and Pathology, Department of Clinical Sciences, Lund University, Solvegatan 19, SE-22362, Lund, Sweden. Department of Surgery, Skåne University Hospital, Lund, Sweden.


