### Accession
PXD037803

### Title
Multienzyme deep learning models improve peptide de novo sequencing by mass spectrometry proteomics

### Description
Generating and analyzing overlapping peptides through multienzymatic digestion is an efficient procedure for de novo protein using from bottom-up mass spectrometry (MS). Despite improved instrumentation and software, de novo MS data analysis remains challenging. In recent years, deep learning models have represented a performance breakthrough. Incorporating that technology into de novo protein sequencing workflows require machine-learning models capable of handling highly diverse MS data. In this study, we analyzed the requirements for assembling such generalizable deep learning models by systematically varying the composition and size of the training set. We assessed the generated models' performances using two test sets composed of peptides originating from the multienzyme digestion of samples from various species. The peptide recall values on the test sets showed that the deep learning models generated from a collection of highly N- and C-termini diverse peptides generalized 76% more over the termini-restricted ones. Moreover, expanding the training set's size by adding peptides from the multienzymatic digestion with five proteases of several species samples led to a 2-3 fold generalizability gain. Furthermore, we tested the applicability of these multienzyme deep learning (MEM) models by fully de novo sequencing the heavy and light monomeric chains of five commercial antibodies (mAbs). MEMs extracted over 10000 matching and overlapped peptides across six different proteases mAb samples, achieving a 100% sequence coverage for 8 of the ten polypeptide chains. We foretell that the MEMs' proven improvements to de novo analysis will positively impact several applications, such as analyzing samples of high complexity, unknown nature, or the peptidomics field.

### Sample Protocol
Detroit 562 cell line and commercial antibodies multienzymatic digestion. The peptides of the digested commercial antibodies were analyzed on Q Exactive HF-X mass spectrometer (Thermo Scientific) connected to an EASY-nLC 1200 ultra-high-performance liquid chromatography system (Thermo Scientific). The peptides were loaded onto an Acclaim PepMap 100 (75µm x 2 cm) C18 (3 µm, 100 Å) pre-column and separated on an EASY-Spray column (Thermo Scientific; ID 75µm x 50 cm, column temperature 45°C) operated at a constant pressure of 800 bar. A linear gradient from 3 to 38% of 80% acetonitrile in aqueous 0.1% formic acid was run for 120 min at a flow rate of 350 nl min-1. One full MS scan (resolution 120 000 @ 200 m/z; mass range 350  1650 m/z) was followed by MS/MS scans (resolution 15000 @ 200 m/z) of the 15 most abundant ion signals. The isolation width window for the precursor ions was 1.3 m/z, they were fragmented using higher-energy collisional-induced dissociation (HCD) at a normalized collision energy of 28. Charge state screening was enabled, and precursor ions with unknown charge states and a charge state of 1, and over 6 were rejected. Data was additionally collected for non-tryptic digestions as above, but including peptides with a charge state of 1. The dynamic exclusion window was 10 s. The automatic gain control was set to 3e6 and 1e5 for MS and MS/MS with ion accumulation times of 45 ms and 30 ms, respectively.

### Data Protocol
Ursgal package was used as an interface for searching the spectra against data's Uniprot reference proteome using five search engines, namely MSGFPlus(version 2019.07.03), MS Amanda(version 2.0.0.17442), Comet(version 2019.01.rev5), X! Tandem(version alanine), and OMSSA(version 2.1.9). Optional Met Oxidation (UniMod: 35), along with the fixed Cys carbamidomethylation (UniMod: 4) modifications, were considered in this study. Individual engine results were validated by percolator(version 3.4.0), while the Combine FDR algorithm was implemented for combining results from all search engines. Moreover, a threshold of 1% peptide FDR was set for decisive candidate inclusion.

### Publication Abstract
None

### Keywords
De novo sequencing, Deepnovo, Monoclonal antibody, Mass spectrometry

### Affiliations
Division of Infection Medicine, Department of Clinical Sciences Lund, Faculty of Medicine, Lund University,
Division of Infection Medicine, Department of Clinical Sciences Lund, Faculty of Medicine, Lund University, Klinikgatan 32, SE-22184 Lund, Sweden

### Submitter
Carlos Gueto-Tettay

### Lab Head
Dr Lars Malmström
Division of Infection Medicine, Department of Clinical Sciences Lund, Faculty of Medicine, Lund University, Klinikgatan 32, SE-22184 Lund, Sweden


