### Accession
PXD028308

### Title
AP-MS of replisomes from S. cerevisiae in S phase and after unscheduled G1 replication

### Description
Interfering with once-per-cell-cycle regulation of DNA replication initiation generates genome instability through over-replication and has been linked to early stages of cancer development. Here, we engineered different systems in budding yeast to induce over-replication in G1 and investigated its characteristics and consequences within the same cell cycle. To study replisome composition, we immunopurified replisomes via a GFP-tag on the Psf2 subunit of the GINS complex from S phase in the presence and absence of hydroxyurea and after unscheduled replication initiation in G1.

### Sample Protocol
Washed beads were incubated for 30 min with elution buffer 1 (2 M Urea, 50 mM Tris-HCl pH 7.5, 2 mM DTT, 20 Âµg/ml trypsin) followed by a second elution with elution buffer 2 (2 M Urea, 50 mM Tris-HCl pH 7.5, 10 mM Chloroacetamide) for 5 min. Both eluates were combined and further incubated at room temperature overnight. Tryptic peptide mixtures were acidified to 1% TFA and desalted with Stage Tips containing C18 reverse-phase material and analyzed by mass spectrometry. Peptides were separated on 50 cm columns packed with ReproSilâ€�Pur C18â€�AQ 1.9 Î¼m resin (Dr. Maisch GmbH). Liquid chromatography was performed on an EASYâ€�nLC 1200 ultraâ€�highâ€�pressure system coupled through a nano-electrospray source to a Q-Exactive HF-X Mass Spectrometer (Thermo Fisher Scientific). Peptides were loaded in buffer A (0.1 % formic acid) and separated with a non-linear gradient of 5 â€“ 60 % buffer B (0.1 % formic acid, 80 % acetonitrile) at a flow rate of 300 nl/min over 50 min. The column temperature was kept at 60Â° C by an in-house designed oven with a Peltier element. Data acquisition switched between a full scan (60 K resolution, 20 ms max. injection time, AGC target 3e6) and 10 dataâ€�dependent MS/MS scans (15K resolution, 60 ms max. injection time, AGC target 1e5). The isolation window was set to 1.4 and normalized collision energy to 27. Multiple sequencing of peptides was minimized by excluding the selected peptide candidates for 30 s.

### Data Protocol
Raw mass spectrometry data were analyzed with MaxQuant (v1.5.3.54). Peak lists were searched against the yeast Uniprot FASTA database combined with 262 common contaminants by the integrated Andromeda search engine. The false discovery rate was set to 1 % for both peptides (minimum length of 7 amino acids) and proteins. â€œMatch between runsâ€� (MBR) with a maximum matching time window of 0.5 min and an alignment time window of 20 min was enabled. Relative protein amounts were calculated with the MaxLFQ algorithm with a minimum ratio count of two.

### Publication Abstract
None

### Keywords
Re-replication, Replication stress, Bakerâ€™s yeast, Replication initiation, Replisome, Cell division cycle

### Affiliations
Department of Proteomics and Signal Transduction, Max-Planck-Institute of Biochemistry, Munich, Germany
Department of Proteomics and Signal Transduction, Max-Planck Institute of Biochemistry, Martinsried, Germany

### Submitter
Michael Wierer

### Lab Head
Dr Matthias Mann
Department of Proteomics and Signal Transduction, Max-Planck-Institute of Biochemistry, Munich, Germany


