### Accession
PXD002024

### Title
Phosphoproteomic analysis of methanoarchaeon Methanohalophilus portucalensis FDF1T reveals diverse functions in methanogenesis and osmoadaptation

### Description
Methane is a well-known factor in green house effect and also could be an alternative energy, which produced via methanogenesis by methanogenic archaea. Methanohalophilus portucalensis FDF1T is a halophilic methylotrophic methanogen and has been used as a model to study osmoadaptation in fluctuated abiotic environmental conditions. Here we report a genome-wide phosphoproteomic analysis of M. portucalensis FDF1T to uncover the possible phosphorylation-mediated regulations in methanogenesis and osmoadaptation. According to the biological function analysis revealed 20.6% of total 149 phosphoproteins were involved in the regulation of the single carbon metabolism for methanogenesis and osmoadaptation. Together with genome sequencing and phosphoproteomic analysis, we provided the first evidence to indicate protein phosphorylation networks in regulating the pyrrolysine (Pyl) mediated translation of methylotrophic methyltransferase. Furthermore, the functional assays of glycine sarcosine N-methyltransferase revealed that the enzymatic activity was modulated by threonine phosphorylation. Collectively, our results not only provide new insight into phosphorylation-mediated regulations in single carbon energy metabolism but also clearly demonstrate that phosphorylation plays a critical role in osmolyte betaine synthesis through a phospho-switch mechanism.

### Sample Protocol
The total protein were prepared in fresh lysis buffer (25 mM ammonium bicarbonate, PhosSTOP phosphatase inhibitor mixture tablets (Roche), 6 M urea, and 2 M thiourea) and disrupted by sonication on ice. Following the gel-free and gel-based trypsin digestion, the phosphopeptides were both enriched via custom-made HAMMOC tips, which were prepared using 0.5 mg TiO2 beads (GL Sciences, Tokyo, Japan) packed into 10-ÂµL C8-StageTips and then desalted using SDB-XC StageTip. The resulting phosphopeptides were concentrated in a vacuum centrifuge for subsequent nanoLC-MS/MS analysis.

### Data Protocol
All MS and MS/MS raw data were analyzed using MaxQuant software (version 1.4.1.2) (http://www.maxquant.org/) with the built-in search engine Andromeda for phosphopeptide identification and phosphorylation site analysis. The protein-encoding genes from M. portucalensis strain FDF1T genome sequence were predicted previously by Glimmer 2.13, GeneMark 2.4, and GeneMark.hmm 2.1 and annotated with the RefSeq Microbial Genomes database using BLASTP in standard settings (E-value < 10-5, identity > 40%, and matched length >30%). The search criteria used for phosphopeptide and phosphosite analysis were as follows: trypsin digestion; cysteine carboxyamidomethylation (+ 57.0214 Da) as the fixed modification; methionine oxidation (+ 15.9949 Da), phosphorylation of serines, threonines, tyrosines, histidines, and aspartates, and protein N-terminal acetylation as variable modifications; up to two missed cleavage allowed; the minimum seven amino acids per peptide; and mass accuracy of 10 ppm for the parent ion and 0.6 Da for the fragment ions. False discovery rates (FDR) estimated from the target-decoy strategy to distinguish correct and incorrect identification. For the identification, FDR was set to 0.01 for sites, peptides and proteins.

### Publication Abstract
Methanogens have gained much attention for their metabolic product, methane, which could be an energy substitute but also contributes to the greenhouse effect. One factor that controls methane emission, reversible protein phosphorylation, is a crucial signaling switch, and phosphoproteomics has become a powerful tool for large-scale surveying. Here, we conducted the first phosphorylation-mediated regulation study in halophilic Methanohalophilus portucalensis FDF1(T), a model strain for studying stress response mechanisms in osmoadaptation. A shotgun approach and MS-based analysis identified 149 unique phosphoproteins. Among them, 26% participated in methanogenesis and osmolytes biosynthesis pathways. Of note, we uncovered that protein phosphorylation might be a crucial factor to modulate the pyrrolysine (Pyl) incorporation and Pyl-mediated methylotrophic methanogenesis. Furthermore, heterologous expression of glycine sarcosine N-methyltransferase (GSMT) mutant derivatives in the osmosensitive Escherichia coli MKH13 revealed that the nonphosphorylated T68A mutant resulted in increased salt tolerance. In contrast, mimic phosphorylated mutant T68D proved defective in both enzymatic activity and salinity tolerance for growth. Our study provides new insights into phosphorylation modification as a crucial role of both methanogenesis and osmoadaptation in methanoarchaea, promoting biogas production or reducing future methane emission in response to global warming and climate change.

### Keywords
Methanogenesis, Phosphorylation modificatioin, Methanohalophilus portucalensis

### Affiliations
Institute of Biological Chemistry, Taipei, Taiwan
China Medical University

### Submitter
Shu-Jung Lai

### Lab Head
Dr Shu-Jung Lai
Institute of Biological Chemistry, Taipei, Taiwan


