### Accession
PXD017187

### Title
OsNSUN2-mediated 5-methylcytidine mRNA modification enhances rice adaptation to high-temperature

### Description
Heat stress is one of the extreme weather disasters potently decreasing cereal plant production. Recent studies have demonstrated that protein degradation and rRNA homeostasis as well as transcription factors are involved in thermoresponse in plants. However, how 5-methylcytosine (m5C) RNA modification contribute to heat stress response in plant remains largely unknown. Herein, we identified OsNSUN2 as the mRNA m5C methyltransferase in rice. The osnsun2 mutants specifically displayed severe phenotypes of temperature- and light-dependent lesion mimic and heat stress hypersensitivity. Heat stress enhanced mRNAs m5C methylation involving photosynthesis and detoxification systems, such as β-OsLCY, OsHO2, OsPAL1, and OsGLYI4, in an OsNSUN2-dependent manner, which then increased the protein synthesis. Consistently, the photosystem of the osnsun2 mutants was vulnerable to high ambient temperature and failed to be repaired under tolerable heat stress. Furthermore, OsNSUN2 mutation reduced photosynthesis efficiency and accumulated excessive reactive oxygen species upon heat treatment. Our findings demonstrate an important mechanism of mRNA m5C-dependent heat acclimation in rice.

### Sample Protocol
The seedling shoots of 36°C treated (0 h, 24 h, and 48 h) NIP and osnsun2-1 were subjected to protein extraction following the method described by previous report (Lv et al., 2014). Protein concentration was determined by 2-D Quant Kit (80648356, Amersham Biosciences) following the manuscript instruction. Protein digestion and tandem mass tag (TMT) labeling was performed using TMT10plex isobaric label reagent set (90110, Thermo Fisher) following the manuscript instruction.  The labeled samples were equally mixed and desalted using desalting column (89852, Thermo Fisher). The peptides were separated into 60 fractions by high pH reverse-phase HPLC with a Thermo Fisher Betasil C18 column (5 μm particle size, 10 mm ID, 250 mm length) using a gradient of 8-32% acetonitrile (pH 9.0) over 60 min. Then, the fractions were combined into 6 final fractions and dried by vacuum centrifuging. The peptide mixtures were dissolved in solvent A (0.1% formic acid), and loaded onto a reversed-phase analytical column (75 μm ID, 15 cm length) with a gradient of 6-23% solvent B (0.1% formic acid in 98% acetonitrile) over 26 min, 23-35% in 8 min and up to 80% in 3 min, then holding at 80% for 3 min at a flow rate of 400 nl/min carried out with an EASY-nLC 1000 UPLC system.  The peptides were subjected to NSI source followed by tandem mass spectrometry (MS/MS) in Q ExactiveTM Plus (Thermo Fisher) coupled online to the UPLC. The applied electrospray voltage was 2.0 kV. The m/z scan range was 350 to 1800, and intact peptides were detected in the Orbitrap with a resolution of 70,000. Peptides were then selected for MS/MS using NCE setting as 28 and detected in the Orbitrap at a resolution of 17,500. A data-dependent procedure that alternated between one MS scan followed by 20 MS/MS scans with 15.0 s dynamic exclusion. Fixed first mass was 100 m/z. Automatic gain control (AGC) was 5E4.

### Data Protocol
The resulting MS/MS data were processed using Maxquant search engine (v.1.5.2.8). Tandem mass spectra were searched against human uniprot database concatenated with reverse decoy database.  The raw data acquisition and processing were analyzed using MaxQuant (v.1.5.2.8) and searched against TIGR rice 66,399 protein sequences. Trypsin/P was set as cleavage enzyme allowing up to 4 missing cleavages. The mass tolerance for precursor ions in first and main search was set as 20 ppm and 5 ppm respectively. The mass tolerance for fragment ions was set as 0.02 Da. Carbamidomethyl on Cys was set as fixed modification, while acetylation modification and oxidation on Met were set as variable modifications. FDR was set as < 1% and minimum score for modified peptides was adjusted to > 40. Median ratio normalization and student’s t-test was used to compare between different samples.

### Publication Abstract
Extreme weather events can cause heat stress that decreases crop production. Recent studies have demonstrated that protein degradation and rRNA homeostasis as well as transcription factors are involved in the thermoresponse in plants. However, how RNA modifications contribute to temperature stress response in plant remains largely unknown. Herein, we identified OsNSUN2 as an RNA 5-methylcytosine (m<sup>5</sup>C) methyltransferase in rice. osnsun2 mutant displayed severe temperature- and light-dependent lesion-mimic phenotypes and heat-stress hypersensitivity. Heat stress enhanced the OsNSUN2-dependent m<sup>5</sup>C modification of mRNAs involved in photosynthesis and detoxification systems, such as &#x3b2;-OsLCY, OsHO2, OsPAL1, and OsGLYI4, which increased protein synthesis. Furthermore, the photosystem of osnsun2 mutant was vulnerable to high ambient temperature and failed to undergo repair under tolerable heat stress. Thus, OsNSUN2 mutation reduced photosynthesis efficiency and accumulated excessive reactive oxygen species upon heat treatment. Our findings demonstrate an important mechanism of mRNA m<sup>5</sup>C-dependent heat acclimation in rice.

### Keywords
Oryza sativa, Ros, Methyltransferase, M5c methylation, Rna modification, Nsun, Heat stress, Photosynthesis, Rice

### Affiliations
Key Laboratory of Plant Molecular Physiology, Institute of Botany, Chinese Academy of Sciences, Beijing 100093, China
Beijing Institute of Genomics

### Submitter
Chunchun GAO

### Lab Head
Dr Kang Chong
Key Laboratory of Plant Molecular Physiology, Institute of Botany, Chinese Academy of Sciences, Beijing 100093, China


