### Accession
PXD040928

### Title
Frequent ZNF217 mutations lead to transcriptional deregulation of interferon signal transduction via altered chromatin accessibility in B cell lymphoma (Phospho-Proteome)

### Description
Recent exome-wide studies discovered frequent somatic mutations in the epigenetic modifier ZNF217 in primary mediastinal B cell lymphoma (PMBCL) and related disorders. As functional consequences of ZNF217 alterations remain unknown, we comprehensively evaluated their impact in PMBCL. Targeted sequencing identified genetic lesions affecting ZNF217 in 33% of 157 PMBCL patients. Subsequent gene expression profiling (n=120) revealed changes in cytokine and interferon signal transduction in ZNF217-aberrant PMBCL cases. 
In vitro, knockout of ZNF217 led to changes in chromatin accessibility interfering with binding motifs for crucial lymphoma-associated transcription factors. This led to disturbed expression of interferon-responsive and inflammation-associated genes, altered cell behavior, and an activated B cell phenotype. Mass spectrometry demonstrates that ZNF217 acts within a histone modifier complex containing LSD1, CoREST and HDAC and interferes with H3K4 methylation and H3K27 acetylation. Concluding, our data suggest non-catalytic activity of ZNF217, which directs histone modifier complex function and controls B cell differentiation-associated patterns of chromatin structure.


### Sample Protocol
Cell pellets were lysed in SDC lysis buffer (1% Sodium deoxycholate, 150mM NaCl, 50mM Tris-HCl pH 8, 1mM EDTA, 10mM DTT (dithiothreitol, Sigma), 40mM CAA  (2-chloroacetamide, Sigma), Phosphatase inhibitor cocktail II and III (Sigma)). Samples were incubated at 95C for 10 min, cooled down and treated with Benzonase (Merck, 50 units) for 30 min at 37C. Protein concentrations were determined using the Pierce BCA assay. Extracted proteins were digested with endopeptidase LysC (Wako) and sequence-grade trypsin (Promega) overnight at 37C with an enzyme-to-protein ratio of 1:100. The resulting peptides were desalted on C18 SepPak columns (Waters, 100mg/1cc) and dried down using a SpeedVac Concentrator (Savant SC210A). Peptides were dissolved in 50 mM HEPES and 200ug of each sample were labeled with 11-plex tandem mass tag (Fisher Scientific) reagents as described (PMID: 30967486). The samples were randomly distributed an internal reference (composed of equal peptide amounts from all samples) was included for data analyses. TMT labeling was quenched with 1M Tris-HCl pH 8, and samples were combined, desalted on C18 SepPak columns (Waters, 200mg/1cc) and fractionated by high-pH reversed phase off-line chromatography (1290 Infinity, Agilent) into 30 fractions. Of each fraction, 10 % of material was dried down directly and used for proteome analyses. Phosphopeptide enrichment was performed with the remaining material, which was further pooled into 15 fractions, on an Agilent Bravo automated liquid handling platform using Fe(III)-NTA cartridges. For LC-MS/MS measurements, peptides were reconstituted in 3% acetonitrile with 0.1% formic acid and separated on a reversed-phase column (20 cm fritless silica microcolumns with an inner diameter of 75 µm, packed with ReproSil-Pur C18-AQ 1.9 µm resin (Dr. Maisch GmbH)) using a 98 min gradient with a 250 nl/min flow rate of increasing Buffer B (90% ACN, 0.1% FA) concentration (from 2% to 60%) on a High Performance Liquid Chromatography (HPLC) system (Thermo Fisher Scientific) and analyzed on a Q-Exactive Plus instrument (Thermo Fisher Scientific). The mass spectrometer was operated in data-dependent acquisition mode using the following settings: full-scan automatic gain control (AGC) target 3 x 106 at 70K resolution; scan range 350-1500m/z; Orbitrap full-scan maximum injection time 10ms; MS2 scan AGC target 5 x 104 at 35K resolution; maximum injection time 50ms (for non-enriched peptides) or 100 ms (for phosphopeptides); normalized collision energy 32; dynamic exclusion time 30s; precursor charge state 2-7, 10 MS2 scans per full scan.

### Data Protocol
RAW data were analyzed using MaxQuant software package (v 1.6.3.4)58. The internal Andromeda search engine was used to search MS2 spectra against a decoy human UniProt database (release 2019-07) containing forward and reverse sequences, as well as ZNF217 mutant sequences. The search included variable modifications of methionine oxidation and N-terminal acetylation, deamidation (N and Q), phosphorylation (STY) and fixed modification of carbamidomethyl cysteine. Reporter ion MS2 for TMT11 was selected (internal and N-terminal) and TMT batch specific corrections factors were specified. Minimal peptide length was set to seven amino acids and a maximum of 3 missed cleavages was allowed. The FDR (false discovery rate) was set to 1% for peptide and protein identifications. Unique and razor peptides were included for quantification.

### Publication Abstract
Recent exome-wide studies discovered frequent somatic mutations in the epigenetic modifier ZNF217 in primary mediastinal B cell lymphoma (PMBCL) and related disorders. As functional consequences of ZNF217 alterations remain unknown, we comprehensively evaluated their impact in PMBCL. Targeted sequencing identified genetic lesions affecting ZNF217 in 33% of 157 PMBCL patients. Subsequent gene expression profiling (n&#x2009;=&#x2009;120) revealed changes in cytokine and interferon signal transduction in ZNF217-aberrant PMBCL cases. In vitro, knockout of ZNF217 led to changes in chromatin accessibility interfering with binding motifs for crucial lymphoma-associated transcription factors. This led to disturbed expression of interferon-responsive and inflammation-associated genes, altered cell behavior, and aberrant differentiation. Mass spectrometry demonstrates that ZNF217 acts within a histone modifier complex containing LSD1, CoREST and HDAC and interferes with H3K4 methylation and H3K27 acetylation. Concluding, our data suggest non-catalytic activity of ZNF217, which directs histone modifier complex function and controls B cell differentiation-associated patterns of chromatin structure.

### Keywords
Pmbcl, Non-hodgkin lymphoma, Znf217, Genetics, Phosphoproteome, Mutation

### Affiliations
Proteomics Platform, BIH@Charite
Core Unit Proteomics, Berlin Institute of Health at Charite - Universitaetsmedizin Berlin and Max-Delbrueck-Center for Molecular Medicine, Berlin, Germany

### Submitter
Marieluise Kirchner

### Lab Head
Dr Philipp Mertins
Core Unit Proteomics, Berlin Institute of Health at Charite - Universitaetsmedizin Berlin and Max-Delbrueck-Center for Molecular Medicine, Berlin, Germany


