### Accession
PXD034244

### Title
Transcriptome and proteome profiling of activated cardiac fibroblasts supports target prioritization in cardiac fibrosis

### Description
Activated cardiac fibroblasts (CF) play a central role in cardiac fibrosis, a condition associated with most cardiovascular diseases. Conversion of quiescent into activated CF sustains heart integrity upon injury. However, permanence of CF in active state inflicts deleterious heart function effects. Mechanisms underlying this cell state conversion are still not fully disclosed, contributing to a limited target space and lack of effective anti-fibrotic therapies. To prioritize targets for drug development, we studied CF remodeling upon activation at transcriptomic and proteomic levels, using three different cell sources: primary adult CF (aHCF), primary fetal CF (fHCF), and induced pluripotent stem cells derived CF (hiPSC-CF). All cell sources showed a convergent response upon activation, with clear morphological and molecular remodeling associated with cell-cell and cell-matrix interactions. Quantitative proteomic analysis identified known cardiac fibrosis markers, such as FN1, CCN2 and Serpine1, but also revealed targets not previously associated with this condition, including MRC2, IGFBP7 and NT5DC2. Exploring such targets to modulate CF phenotype represents a valuable opportunity for development of anti-fibrotic therapies. Also, we demonstrate that hiPSC-CF is a suitable cell source for preclinical research, displaying significantly lower basal activation level relative to primary cells, while being able to elicit a convergent response upon stimuli.

### Sample Protocol
Cardiac fibroblasts (human adult cardiac fibroblasts - aHCF; human fetal cardiac fibroblasts - fHCF; hiPSC-derived cardiac fibroblasts - hiPSC-CF) were seeded at 5800 cell/cm2 on 0.1% gelatin-coated surfaces and cultured with FibroGro LS complete media supplemented daily with 2 µM SB431542. Media was exchanged every two days and on day 4, was replaced with FibroGro LS complete media supplemented with 10 ng/ml TGF-β1 (R&D systems) for activation or 2 µM SB431542 for control samples. Cells were further cultured for 48h and 3 independent experiments (n=3) were performed. For proteomics, cells were harvested using Versene (Gibco) and gentle mechanical dislodgment using a cell scraper. Cells were then sedimented by centrifugation at 300g, 5min. Supernatant was discarded, the resulting cell pellet washed with PBS, followed by centrifugation at 300g, 5min. Supernatant was discarded, pellets were snap freezed and kept at -80°C. Cell pellets were lysed in Triton X-100 lysis buffer (50 mM Tris, 5 mM EDTA, 150 mM NaCl, 1% Triton X-100 (all from Sigma-Aldrich), and 1x complete protease inhibitors cocktail (Roche)), for 45 min at 4°C. Protein quantification was performed using Micro BCA™ Protein Assay Kit (Thermo Fisher Scientific) following the manufacturer’s instructions. Proteins were precipitated in four-fold excess of methanol, centrifuged at 9000g for 10 seconds, and followed by the addition of 2 parts of chloroform with subsequent centrifugation. For phase separation, 3 parts of deionized water were added to the samples, homogenized by vigorous vortex, and centrifuged at 9000g for 1 min. The upper phase was discarded, and 3 parts of methanol were added. Samples were mixed and centrifuged at 9000g for 2min to pellet precipitated protein. Supernatant was removed, and precipitates were dried by heating at 60°C with lids slightly ajar. For surfactant-assisted in-solution protein digestion, precipitated proteins were solubilized in 0.1% of RapiGest SF Surfactant (Waters). Before digestion, samples were reduced with 5 mM of dithiothreitol (DTT) for 30mins at 60°C, alkylated with 15 mM iodoacetamide (IAA) for 30 min in dark, and boiled at 100°C for 5 mins. After cooling to room temperature, protein digestion was performed by overnight incubation with trypsin (Promega; 1.2μg per 100μg protein) at 37°C. Trypsin was inactivated with trifluoroacetic acid (TFA) at 0.5 % and incubation at 37°C for 45min. Samples were centrifuged at 16 000g for 10 min, supernatants were collected into new tubes and dried using the SpeedVac Vacufuge Plus (Eppendorf). For peptide cleanup samples were resuspended in 5 % Formic acid (Optima LC/MS grade, Fisher Scientific) using C18 microcolumns (OMIX C18 pipette tips, Agilent), and then dried. A total of 5µg from each sample (total 18 samples) were used for Nano-liquid chromatography-tandem mass spectrometry (nanoLC-MS/MS) analysis on an ekspert™ NanoLC 425 cHiPLC system coupled with a TripleTOF 6600 with a NanoSpray III source (Sciex Framingham, US). Samples from the same condition (same cell source and same treatment) were pooled (n=3). Peptides were sprayed into the MS through an uncoated fused-silica PicoTip™ emitter (360 µm O.D., 20 µm I.D., 10 ± 1.0 µm tip I.D., New Objective, Oullins, France). The source parameters were set as follows: 15 GS1, 0 GS2, 30 CUR, 2.5 keV ISVF, and 100 °C IHT. A reversed-phase nanoLC-MS/MS with a trap and elution configuration, using a Nano cHiPLC Trap column (Eksigent 350 µm x 0.5 mm, ChromXP C18-CL, 3 µm, 120 Å) and NanoLC column (Eksigent 75 µm x 15 cm, ChromXP 3C18-CL-120, 3 µm, 120 Å) was performed. Water with 0.1% (v/v) formic acid (solvent A) and 0.1% formic acid in acetonitrile (solvent B) were used. Trapping was performed at 2 µL/min for 10 min using 100% (v/v) solvent A. The separation was performed at 300 nL/min applying a gradient (v/v) of solvent B as follows: 0-1 min, 5%; 1-91 min, 5-30%; 91-93 min, 30-80%; 93-108 min, 80%; 108-110 min, 80-5%; 110-127 min, 5%. Each sample pool was subjected to two IDA runs. The mass spectrometer was set for IDA scanning full spectra (400–2000 m/z) for 250 ms (accumulation time). The top 50 most intense precursors were selected for subsequent MS/MS scans of 150–1800 m/z, in high sensitivity mode, for 40 ms, using a total cycle time of 2.3 s. The selection criteria for parent ions included a charge state between +2 and +5 and counts above a minimum threshold of 125 counts per second Ions were excluded from further MS/MS analysis for 12 s. Fragmentation was performed using rolling collision energy with a collision energy spread of 5. For quantitative analysis, 5 μg of each sample were analyzed in triplicate by sequential window acquisition of all theoretical fragment ion spectra (SWATH)-MS, using the instrument setup described for the IDA runs. The mass spectrometer was operated in a cyclic data independent acquisition (DIA).

### Data Protocol
The spectral library was created by combining all IDA raw files using ProteinPilot™ software (v5.0 ABSciex) with the Paragon algorithm and with the following search parameters: Homo sapiens from Uniprot/SwissProt database (20,394 entries, accessed on 05/01/2021); trypsin digestion; iodoacetamide cysteine alkylation; TripleTOF 6600 equipment; and biological modifications as ID focus.  After a false discovery rate (FDR) analysis, only FDR<1% were considered (4659 proteins). The output of these searches was used as the reference spectral library. SWATH-MS data were acquired with SWATH acquisition method, using a set of 64 overlapping variable SWATH windows covering the precursor mass range of 400–2,000 m/z. The variable SWATH windows were calculated using the SWATH Variable Window Calculator V1.0 (Sciex, Framingham, US) based on a reference sample. A 50 ms survey scan (400-1,800 m/z) was acquired at the beginning of each cycle, and the subsequent SWATH windows were collected from 400 to 1,600 m/z for 50 ms, resulting in a cycle time of 3.30 s. Rolling collision energy with a collision energy spread of 5 was used. Spectral alignment and targeted data extraction of DIA samples were performed using PeakView v.2.2 (Sciex, Framingham, MA USA), with the spectral library as reference. For data extraction the following parameters were used: 6 peptides/protein, 6 transitions/peptide, peptide confidence level of >96%, FDR threshold of 1%, excluding shared peptides, and extracted ion chromatogram (XIC) window of 6 min and width set at 20 ppm. Data were directly exported to Markerview 1.3.1 (Sciex, Framingham, MA USA) and normalized using total area sums to obtain the final quantification values. A total of 4459 proteins were quantified under these conditions.

### Publication Abstract
None

### Keywords
Cardiac fibroblasts; cardiac fibrosis; myofibroblasts; transcriptomics; quantitative proteomics

### Affiliations
iBET, Instituto de Biologia Experimental e Biológica, Oeiras, Portugal; Instituto de Tecnologia Química e Biológica António Xavier, Universidade Nova de Lisboa, Oeiras, Portugal
Animal Cell Technology Unit, iBET - Instituto de Biologia Experimental e Tecnológica

### Submitter
Daniel Simão

### Lab Head
Dr Daniel Simão
Animal Cell Technology Unit, iBET - Instituto de Biologia Experimental e Tecnológica


