### Accession
PXD022835

### Title
Diagnosis of hepatocellular adenomas and malignancy determination by proteomic patterns matching (1)

### Description
Biopsies are underrated and underused and our goal was to demonstrate that their inherent expression proteomic pattern could give them an added value for diagnosis. As proof of concept, we used as model hepatocellular adenomas (HCA), well characterized benign liver tumors. From a collection of 260 cases, we selected 55 typical cases to build the first HCA proteomic database. Biopsies proteomic patterns allowed HCA classification, even for complex cases. In addition, these data gave access to a malignancy pattern identifying the HCA transformation. This pioneering work proposes a proteomic based machine learning tool, operational on fixed biopsies, to improve HCA diagnosis and therefore patientâ€™s management.

### Sample Protocol
Samples were loaded on a 10% acrylamide SDS-PAGE gel. Migration was stopped when samples were entered the resolving gel and proteins were visualized by Colloidal Blue staining. Each SDS-PAGE band was cut and protein were reduced, alkylated and digested by trypsin. Online nanoLC-MS/MS analysis were performed using an Ultimate 3000 RSLC Nano-UPHLC system (Thermo Scientific, USA) coupled to a nanospray Q-Exactive hybrid quadruplole-Orbitrap mass spectrometer (Thermo Scientific, USA) or coupled to a nanospray Orbitrap Fusionâ„¢ Lumosâ„¢ Tribridâ„¢ Mass Spectrometer (Thermo Fisher Scientific, California, USA).  For Q-Exactive analysis, each peptide extracts were loaded on a 300 Âµm ID x 5 mm PepMap C18 precolumn (Thermo Scientific, USA) at a flow rate of 10 ÂµL/min. After 3 min desalting, peptides were online separated on a 75 Âµm ID x 25 cm C18 Acclaim PepMapÂ® RSLC column (Thermo Scientific, USA) with a 4-40% linear gradient of solvent B (0.1% formic acid in 80% ACN) in 108 min. The separation flow rate was set at 300 nL/min. The mass spectrometer operated in positive ion mode at a 1.8 kV needle voltage. Data were acquired using Xcalibur 3.1 software in a data-dependent mode. MS scans (m/z 350-1600) were recorded at a resolution of R = 70000 (@ m/z 200) and an AGC target of 3 x 106 ions collected within 100 ms. Dynamic exclusion was set to 30 s and top 12 ions were selected from fragmentation in HCD mode. MS/MS scans with a target value of 1 x 105 ions were collected with a maximum fill time of 100 ms and a resolution of R = 17500. Additionally, only +2 and +3 charged ions were selected for fragmentation. Others settings were as follows: no sheath and no auxiliary gas flow, heated capillary temperature, 200Â°C; normalized HCD collision energy of 27% and an isolation width of 2 m/z.  For Orbitrap Fusion Lumos analysis, each peptide extracts were loaded on a 300 Âµm ID x 5 mm PepMap C18 precolumn (Thermo Scientific, USA) at a flow rate of 10 ÂµL/min. After a 3 min desalting step, peptides were separated on a 50 cm EasySpray column (75 Âµm ID, 2 Âµm C18 beads, 100 Ã… pore size, ES803, Thermo Fisher Scientific) with a 4-40% linear gradient of solvent B (0.1% formic acid in 80% ACN) in 55 min. The separation flow rate was set at 300 nL/min. The mass spectrometer operated in positive ion mode at a 2.0 kV needle voltage. Data was acquired using Xcalibur 4.1 software in a data-dependent mode. MS scans (m/z 375-1500) were recorded at a resolution of R=120000 (@ m/z 200) and an AGC target of 4x105 ions collected within 50 ms, followed by a top speed duty cycle of up to 3 seconds for MS/MS acquisition. Precursor ions (2 to 7 charge states) were isolated in the quadrupole with a mass window of 1.6 Th and fragmented with HCD@30% normalized collision energy. MS/MS data was acquired in the Orbitrap cell with a resolution of R=30000 (@m/z 200), AGC target of 5x104 ions and a maximum injection time of 100 ms. Selected precursors were excluded for 60 seconds.

### Data Protocol
For protein identification, Mascot 2.5 algorithm through Proteome Discoverer 1.4 Software (Thermo Fisher Scientific Inc.) was used in batch mode by searching against the UniProt Homo sapiens database (73 950 entries, Reference Proteome Set, release date: November 17, 2018) from http://www.uniprot.org/ website. Two missed enzyme cleavages were allowed. Mass tolerances in MS and MS/MS were set to 10 ppm and 0.02 Da. Oxidation of methionine, acetylation of lysine and deamidation of asparagine and glutamine were searched as dynamic modifications. Carbamidomethylation on cysteine was searched as static modification. Raw LC-MS/MS data were imported in Proline Studio20 for feature detection, alignment, and quantification. Proteins identification was accepted only with at least 2 specific peptides with a pretty rank=1 and with a protein FDR value less than 1.0% calculated using the â€œdecoyâ€� option in Mascot. Label-free quantification of MS1 level by extracted ion chromatograms (XIC) was carried out with parameters indicated previously : Henriet E et al. Hepatology. 2017. The normalization was carried out on median of ratios. The inference of missing values was applied with 5% of the background noise.

### Publication Abstract
None

### Keywords
Machine learning, Biopsies, Malignancy, Diagnosis, Hepatocellular adenomas, Patterns matching, Clinical proteomics

### Affiliations
Plateforme Oncoprot, TBM-Core US 005, F-33000 Bordeaux, France
Plateforme Proteome Bordeaux

### Submitter
Dupuy Jean-William

### Lab Head
Dr Raymond Anne-Aurelie
Plateforme Oncoprot, TBM-Core US 005, F-33000 Bordeaux, France


