### Accession
PXD002232

### Title
Zebrafish embryo_phosphorylation_18O labeling

### Description
The zebrafish (Danio rerio) is a popular animal model in studies of vertebrate development and organogenesis. Recent research has shown a similarity of approximately 70% between the human and zebrafish genomes and of 84% in human disease-causing genes, specifically. Zebrafish embryos have a number of desirable features, including transparency, a large size, and rapid embryogenesis. Protein phosphorylation is a well-known post-translational modification (PTM), which performs various biological functions. Recent mass spectrometry (MS) developments have enabled the study of global phosphorylation patterns by using MS-based proteomics coupled with TiO2 phosphopeptide enrichment. In the present study, we identified 3,500 non-redundant phosphorylation sites on 2,166 phosphoproteins and 1,564 quantified phosphoproteins in zebrafish embryos.

### Sample Protocol
* Zebrafish cultures Wild type zebrafish (Danio rerio) were obtained from the Korean Zebrafish Organogenesis Mutant Bank at Kyungpook National University (Daegu, Korea). Animals were maintained under a 14 h light/10 h dark cycle at 28.5°C [5]. Developmental stages are provided as hpf based on standard developmental stages [25]. Zebrafish were fed with baby brine shrimp (Advanced Hatchery Technology Inc.) twice daily.   * Preparation of protein sample  Embryos were grown until 12, 24, 48, 72, and 120 hpf in embryo media (0.3 mg/mL of sea salt and 1 µg/mL of methylene blue) and collected. Embryos were manually dechorionated and then deyolked with deyolking buffer (55 mM NaCl, 1.8 mM KCl, and 1.25 mM NaHCO3) and deyolking washing buffer (110 mM NaCl, 3.5 mM KCl, 2.7 mM CaCl2, 10 mM Tris, pH 8.5) by pipetting. Embryos were deyolked as previously described [26], and then sonicated in lysis buffer (included complete RIPA buffer, protease inhibitor cocktail, and phosphatase inhibitor cocktail) on ice. After incubation for 10 min at 4°C, sonicated extracts were centrifuged at 14,000 ×g for 10 min at 4°C. Supernatants were transferred to new tubes and quantified using BCA assay kits. For protein precipitation, embryonic proteins (12, 24, 48, 72, and 120 h) were precipitated in 10% trichloroacetic acid overnight at 4°C and then centrifuged at 14,000 ×g for 10 min at 4°C. Pellets were washed with ice-cold acetone twice, and then dissolved in 50 mM ammonium bicarbonate. Resuspended proteins were re-quantified using BCA assay kits.  * In-solution trypsin digestion and enzymatic 18O-labeling Five hundred micrograms proteins from 12, 24, 48, and 72 hpf zebrafish embryos was resuspended in centrifuge tubes. For 120 hpf proteins, 4 tubes, each containing 500 µg protein, were prepared. Proteins were reduced using 15 mM dithiothreitol in 25 mM ammonium bicarbonate at 56°C for 30 min, and then alkylated using 60 mM iodoacetamide in 25 mM ammonium bicarbonate for 30 min at RT in the dark. Protein samples were then added to 75 mM cysteine in 25 mM ammonium bicarbonate at RT for 30 min in the dark. After adding trypsin to protein (ratio 1:50 w/w), peptide samples were digested at 37°C overnight. Digested peptides were dried using a centrifugal vacuum concentrator, and then additional trypsin was added at 1:50 (w/w). For 18O-labeling [16], 120 hpf-digested peptides were re-dissolved in 100 µL 18O water including 20% acetonitrile and incubated for 24 h at 37°C. Furthermore, 12, 24, 48, and 72 hpf peptides were re-suspended in 100 µL of purified water including 20% acetonitrile and incubated for 24 h at 37°C. In order to stop trypsin, reactions were treated with formic acid. For comparison, 12, 24, 48, and 72 hpf peptides were combined with 18O-labeled 120 hpf peptides. Finally, the mixed peptides were dried using a centrifugal vacuum concentrator.   * Phosphopeptide enrichment and desalting  Phosphopeptides were enriched using phosphopeptide enrichment TiO2 kits. Prior to loading samples, the TiO2 tip (a kit component) was activated in 250 µL of 100% acetonitrile and then equilibrated in 250 µL of buffer B (300 mg/mL glycolic acid in buffer A). Respective mixed peptides were re-dissolved in 500 µL buffer B and in 1 mL of water including 0.1% trifluoroacetic acid using a sonicator. After loading on the TiO2 tip, and washing it with 250 µL of buffer B twice followed by 250 µL of buffer A (80% acetonitrile in 2% trifluoroacetic acid) three times, phosphopeptides were eluted with 500 µL of 5% ammonium hydroxide solution followed by 500 µL of 30% acetonitrile. The desalting process, using SDB, GC, and C18 tips, was conducted according to the manufacturer's instructions. Finally, SDB-, GC-, and C18-eluted samples were combined and dried using a centrifugal vacuum concentrator.

### Data Protocol
Peptides and proteins were identified by Proteome Discoverer (version 1.3; Thermo Scientific) in Mascot (version 2.3; Matrix Science). The search database used the zebrafish proteome set (including 55,158 protein sequences), and was downloaded from UniProtKB (http://www.uniprot.org) in Mar. 2013. The search parameters used were a precursor mass error of 50 ppm and a fragment ion mass error of 0.8 Da; monoisotopic mass was selected. Trypsin was selected as the enzyme, with two potential missed cleavages. Carbamidomethylation (Cys) was searched as a fixed modification, whereas oxidation (Met), phosphorylation (Ser/Thr/Tyr), and carboxyl-terminal single and double 18O were searched as variable modifications. Mascot search results were filtered using a Mascot ion score of more than 20 and decoy database results to provide a less than 1% FDR. When the identified peptide sequence of one protein was contained in the peptide sequence of another protein, the identified protein was not considered an independent phosphoprotein hit [28]. In addition, for phosphorylation site localization, we filtered from the non-redundant sites to identify confidently localized sites based on a phosphoRS probability of more than 75 %.  To quantify zebrafish embryo proteins in 18O-labeled samples, the event detector and precursor ion quantifier of Proteome Discoverer were used with a 4 ppm mass variability and a 1 min retention time tolerance on precursor ion pairs. Protein ratios were calculated by dividing heavy protein (120 hpf) precursor ions by those of light proteins (individual 12, 24, 48, and 72 hpf) precursor ions.

### Publication Abstract
The zebrafish (Danio rerio) is a popular animal model used for studies on vertebrate development and organogenesis. Recent research has shown a similarity of approximately 70% between the human and zebrafish genomes and about 84% of human disease-causing genes have common ancestry with that of the zebrafish genes. Zebrafish embryos have a number of desirable features, including transparency, a large size, and rapid embryogenesis. Protein phosphorylation is a well-known PTM, which can carry out various biological functions. Recent MS developments have enabled the study of global phosphorylation patterns by using MS-based proteomics coupled with titanium dioxide phosphopeptide enrichment. In the present study, we identified 3500 nonredundant phosphorylation sites on 2166 phosphoproteins and quantified 1564 phosphoproteins in developing embryos of zebrafish. The distribution of Ser/Thr/Tyr phosphorylation sites in zebrafish embryos was found to be 88.9, 10.2, and 0.9%, respectively. A potential kinase motif was predicted using Motif-X analysis, for 80% of the identified phosphorylation sites, with the proline-directed motif appearing most frequently, and 35 phosphorylation sites having the pSF motif. In addition, we created six phosphoprotein clusters based on their dynamic pattern during the development of zebrafish embryos. Here, we report the largest dataset of phosphoproteins in zebrafish embryos and our results can be used for further studies on phosphorylation sites or phosphoprotein dynamics in zebrafish embryos.

### Keywords
18o labeling, : zebrafish embryo, Global phosphoproteome

### Affiliations
Kyungpook National University
College of Pharmacy, Research Institute of Pharmaceutical Sciences, Kyungpook National University

### Submitter
Oh Kwang Kwon

### Lab Head
Dr Sangkyu Lee
College of Pharmacy, Research Institute of Pharmaceutical Sciences, Kyungpook National University


