### Accession
PXD046587

### Title
Post-transcriptional mechanisms modulate the consequences of adaptive copy number variation

### Description
Copy-number variants (CNVs) are large-scale amplifications or deletions of DNA that can drive rapid adaptive evolution and result in large-scale changes in gene expression. Whereas alterations in the copy number of one or more genes within a CNV can confer a selective advantage, other genes within a CNV can decrease fitness when their dosage is changed. Dosage compensation - in which the gene expression output from multiple gene copies is less than expected - is one means by which an organism can mitigate the fitness costs of deleterious gene amplification. Previous research has shown evidence for dosage compensation at both the transcriptional level and at the level of protein expression; however, the extent of compensation differs substantially between genes, strains, and studies. Here, we investigated sources of dosage compensation at multiple levels of gene expression regulation by defining the transcriptome, translatome and proteome of experimentally evolved yeast (Saccharomyces cerevisiae) strains containing adaptive CNVs.

### Sample Protocol
We used 16Plex TMT labeling to enable pooling of peptides from 16 samples (Ch1-16) for offline fractionation and LC-MS/MS detection and quantification. Five replicates were generated for each strain and the total run was performed in two batches (“A”,“B”). Ch1 was used as the common reference standard, an equimolar mix of all the samples, labeled with TMT Ch1. All samples were lysed using a TFA approach [10.1074/mcp.TIR119.001616]. Both batches were fractionated by offline HPLC chromatography using reverse phase C18 stationary phase at high pH. Peptides from collected fractions were separated by online HPLC on C18 at low pH coupled to the MS instrument.

### Data Protocol
Spectra were analyzed using MaxQuant (v2.1.0.0).  Include contaminants True PSM FDR 0.01 PSM FDR Crosslink 0.01 Protein FDR 0.01 Site FDR 0.01 Use Normalized Ratios For Occupancy True Min. peptide Length 7 Min. score for unmodified peptides 0 Min. score for modified peptides 40 Min. delta score for unmodified peptides 0 Min. delta score for modified peptides 6 Min. unique peptides 0 Min. razor peptides 1 Min. peptides 1 Use only unmodified peptides and True Modifications included in protein quantification Oxidation (M);Acetyl (Protein N-term) --- Internal label Terminal label Correction factor -2 [%] Correction factor -1 [%] Correction factor +1 [%] Correction factor +2 [%] TMT like TMTpro16plex-Lys126C TMTpro16plex-Nter126C 0 0 9.09 0.32 TRUE TMTpro16plex-Lys127N TMTpro16plex-Nter127N 0 0.78 9.41 0.33 TRUE TMTpro16plex-Lys127C TMTpro16plex-Nter127C 0 0.93 8.63 0.27 TRUE TMTpro16plex-Lys128N TMTpro16plex-Nter128N 0 0.82 8.13 0.26 TRUE TMTpro16plex-Lys128C TMTpro16plex-Nter128C 0 1.47 6.91 0.15 TRUE TMTpro16plex-Lys129N TMTpro16plex-Nter129N 0 1.46 6.86 0.15 TRUE TMTpro16plex-Lys129C TMTpro16plex-Nter129C 0.51 2.74 6.15 0.11 TRUE TMTpro16plex-Lys130N TMTpro16plex-Nter130N 0.13 2.41 5.58 0.10 TRUE TMTpro16plex-Lys130C TMTpro16plex-Nter130C 0.04 3.1 4.82 0.06 TRUE TMTpro16plex-Lys131N TMTpro16plex-Nter131N 0.03 2.78 4.57 0.12 TRUE TMTpro16plex-Lys131C TMTpro16plex-Nter131C 0.08 3.90 3.57 0.04 TRUE TMTpro16plex-Lys132N TMTpro16plex-Nter132N 0.15 3.58 1.80 0 TRUE TMTpro16plex-Lys132C TMTpro16plex-Nter132C 0.11 4.55 1.86 0 TRUE TMTpro16plex-Lys133N TMTpro16plex-Nter133N 0.07 3.14 3.40 0.03 TRUE TMTpro16plex-Lys133C TMTpro16plex-Nter133C 0.22 4.96 1.03 0 TRUE TMTpro16plex-Lys134N TMTpro16plex-Nter134N 0.30 5.49 1.14 0 TRUE

### Publication Abstract
None

### Keywords
Copy-number variant (cnv), Long-term experimental evolution (ltee), Orbitrap, Saccharomyces cerevisiae, Glutamine-limited growth media

### Affiliations
New York University
Professor of Biology, Center for Genomics and Systems Biology Vice Dean, Graduate School of Arts and Sciences New York University, 12 Waverly Pl, New York, Ny 10003

### Submitter
Pieter Spealman

### Lab Head
Dr David Gresham
Professor of Biology, Center for Genomics and Systems Biology Vice Dean, Graduate School of Arts and Sciences New York University, 12 Waverly Pl, New York, Ny 10003


