### Accession
PXD039876

### Title
Preparation of tirapazamine-resistant hepatoma cell line

### Description
The aim of the project was to obtain murine hepatoma MH22a [1] cell subline, resistant to anticancer agent 3-amino-1,2,4-benzotriazine-1,4-dioxide (tirapazamine, TPZ), and examine the changes of the protein expression as compared with parental cells, with the main emphasis on the changes of redox proteins.      MH22a cells obtained from the Institute of Cytology of the Russian Academy of Sciences (St. Petersburg, Russia), were grown at 37Â°C in DMEM medium, supplemented with 10% fetal bovine serum and antibiotics in T25 flasks until reaching 70-80% confluence. The medium is then changed to an analogous medium with 5.0 Î¼M TPZ, and, depending on cell viability, 50% of the medium with TPZ is changed 1-2 times a week, repeating that procedure for 3.5 months to form a stable monolayer. The resulting monolayer is inoculated 1:2 and further grown by changing 50% of the culture medium twice a week until re-inoculation. 20 passages were made over 2.5 months in total. We were unable to obtain a cell subline in a stepwise fashion by growing the cells in increasing concentrations of TPZ, because its concentration increase up to 10 Î¼M caused cell death after 1 month. For analysis, one flask per each sample, 3 samples of parental cell line and 3 samples of TPZ-resistant cell line (between 15th and 20th passages), were selected. Cells were rinsed three times with PBS, detached with TrypLE Express (ThermoFisher Scientific, Waltham, MA, USA) and cell concentration and viability was determined with Trypan blue and Countess 2 (ThermoFisher Scientific, Waltham, MA, USA) automated cell counter. Cells were rinsed by centrifuging 500xg for 5 min and suspending in PBS three times. Finally, cells were flash frozen in liquid nitrogen and transferred to -86Â°C freezer for storage.       [1] European Collection of Authenticated Cell Cultures (ECACC), Catalogue No. 96121721.

### Sample Protocol
Cells were lysed in SDS lysis buffer at 95 Â°C for 5 min. After cooling down, benzonase (Merck, Rahway, NJ, USA) was added, and the solution was incubated for 60 min at 37 Â°C. The protein concentration was determined using the BCA assay. 10 Âµg of each sample were subjected to in-solution tryptic digest according to a modified version of the Single-Pot Solid-Phase-enhanced Sample Preparation (SP3) protocol [1,2]. Here the lysates were added to Sera-Mag Beads (Merck, Rahway, NJ, USA) in 10 Î¼l 15% formic acid and 30 Î¼l of ethanol. Binding of proteins was achieved by shaking for 15 min at room temperature. SDS was removed by 4 subsequent washes with 200 Î¼l 70% ethanol. Proteins were digested overnight at room temperature with 0.4 Î¼g of sequencing grade modified trypsin in 40 Î¼l HEPES/NaOH, pH 8.4 in the presence of 1.25 mM TCEP and 5 mM chloroacetamide. Beads were separated, washed with 10 Î¼l of an aqueous solution of 2% DMSO and the combined eluates were dried down. Peptides were reconstituted in 10 Î¼l of H2O and reacted for 1 h at room temperature with 80 Î¼g of TMT6plex  label reagent dissolved in 4 Î¼l acetonitrile [3]. Excess TMT reagent was quenched by the addition of 4 Î¼l of an aqueous 5% hydroxylamine solution. Peptides were reconstituted in 0.1% formic acid, mixed to achieve a 1:1 ratio across all TMT-channels and purified by a reverse phase clean-up step (OASIS HLB 96-well Î¼Elution Plate, Waters, Etten-Leur, The Netherlands). Peptides were subjected to an offline fractionation under high pH conditions [1]. The resulting 12 fractions were then analyzed by LC-MS/MS using a 90 min gradient on an Orbitrap Fusion Lumos mass spectrometer (ThermoFisher Scientific, Waltham, MA, USA). To this end, peptides were separated using an Ultimate 3000 nano RSLC system (Dionex, ThermoFisher Scientific, Waltham, MA, USA) equipped with a trapping cartridge (Precolumn C18 PepMap100, 5 mm, 300 Î¼m i.d., 5 Î¼m, 100 Ã…) and an analytical column (Acclaim PepMap 100. 75 Ã— 50 cm C18, 3 mm, 100 Ã…) connected to a nanospray-Flex ion source. Peptides were separated on the analytical column with a constant flow of 0.3â€‰Âµl/min applying a 90 min gradient of 2â€“28% of solvent B (0.1% formic acid in acetonitrile) in A (0.1% formic acid in water) followed by an increase to 40% B. The Orbitrap Fusion Lumos was operated in positive ion mode with a spray voltage of 2.4 kV and capillary temperature of 275 Â°C. Full scan MS spectra with a mass range of 375â€“1500 m/z were acquired in profile mode using a resolution of 60,000 (maximum fill time of 50 ms; AGC Target was set to Standard) and a RF lens setting of 30%. Fragmentation was triggered for 3 s cycle time for peptide like features with charge states of 2â€“7 on the MS scan (data-dependent acquisition). Precursors were isolated using the quadrupole with a window of 0.7 m/z and fragmented with a normalized collision energy of 36%. Fragment mass spectra were acquired in profile mode and a orbitrap resolution of 15,000. Maximum fill time was set to 54 ms. AGC target was set to 200%. The dynamic exclusion was set to 60 s. [1] Hughes, C.S.; Foehr, S.; Garfield, D.A.; Furlong, E.E.; Steinmetz, L.M.; Krijgsveld, J. Ultrasensitive proteome analysis using paramagnetic bead technology. Mol. Syst. Biol. 2014, 10, 757. [2] Moggridge, S.; Sorensen, P.H.; Morin, G.B.; Hughes, C.S. Extending the compatibility of the SP36 paramagnetic bead processing approach for proteomics. J. Proteome Res. 2018, 17, 1730-1740.  [3] Werner, T.; Sweetman, G.; Savitski, M.F.; Mathieson, T.; Bantscheff, M.; Savitski, M.M. Ion coalescence of neutron encoded TMT10-plex reporter ions. Anal. Chem. 2014, 86, 3594-3601.

### Data Protocol
Acquired data were analyzed using IsobarQuant [1] and Mascot V2.4 (Matrix Science) using a reverse UniProt FASTA Mus musculus database (UP000000589, downloaded May 15th, 2016 with 59550 entries) including common contaminants. The following modifications were taken into account: Carbamidomethyl (C, fixed), TMT10plex (K, fixed), Acetyl (N-term, variable), Oxidation (M, variable) and TMT10plex (N-term, variable). The mass error tolerance for full scan MS spectra was set to 10 ppm and for MS/MS spectra to 0.02 Da. A maximum of 2 missed cleavages were allowed. A minimum of 2 unique peptides with a peptide length of at least seven amino acids and a false discovery rate below 0.01 were required on the peptide and protein level [2]. The raw output files of IsobarQuant (protein.txt â€“ files) were processed using the R programming language (R Foundation, Vienna, Austria, http://www.R-project.org). Only proteins that were quantified with at least two unique peptides were considered for the analysis. 5818 proteins passed the quality control filter. Raw TMT reporter ion intensities (signal_sum columns) were first cleaned for batch effects using the removeBatchEffect function of the limma package [3] and further normalized using vsn (variance stabilization normalization â€“ [4]). Proteins were tested for differential expression using the limma package. The replicate information was added as a factor in the design matrix given as an argument to the â€˜lmFitâ€™ function of limma. For quantitative bioinformatic analysis the change of protein level of more than 1.5-fold with a false discovery rate (fdr) value <0.05 was considered as significant.  [1] Franken, H.; Mathieson, T.; Childs, D.; Sweetman, G.M.A.; Werner, T.; TÃ¶gel, I.; Doce, C.; Gade, S.; Bantscheff, M.; Drewes, G.; Reihard, F.B.M.; Huber, W.; Savitski, M.M. Thermal proteome profiling for unbiased identification of direct and indirect drug targets using multiplexed quantitative mass spectrometry. Nat. Protoc. 2015, 10, 1567-1593.  [2] Savitski, M.M.; Wilhelm, M.; Hahne, H.; Kuster, B.; Benstcheff, M. A scalable approach for protein false discovery rate estimation in large proteomic data sets. Mol. Cell. Proteomics 2015, 14, 2394-2404.  [3] Ritchie, M.E.; Phipson, B.; Wu, D.; Hu, Y.; Law, C.W.; Shi, W.; Smyth, G.K. limma powers differential expression analyses for RNA-sequencing and microarray studies. Nucleic Acids Res. 2015, 43, e47.  [4] Huber, W.; von Heydebreck, A.; SÃ¼ltmann, H.; Poustka, A.; Vingron, M. Variance stabilization applied to microarray data calibration and to the quantitation of differential expression. Bioinformatics 2002, 18 Suppl. 1, S96-S104.

### Publication Abstract
3-Amino-1,2,4-benzotriazine-1,4-dioxide (tirapazamine, TPZ) and other heteroaromatic <i>N</i>-oxides (ArN&#x2192;O) exhibit tumoricidal, antibacterial, and antiprotozoal activities. Their action is attributed to the enzymatic single-electron reduction to free radicals that initiate the prooxidant processes. In order to clarify the mechanisms of aerobic mammalian cytotoxicity of ArN&#x2192;O, we derived a TPZ-resistant subline of murine hepatoma MH22a cells (resistance index, 5.64). The quantitative proteomic of wild-type and TPZ-resistant cells revealed 5818 proteins, of which 237 were up- and 184 down-regulated. The expression of the antioxidant enzymes aldehyde- and alcohol dehydrogenases, carbonyl reductases, catalase, and glutathione reductase was increased 1.6-5.2 times, whereas the changes in the expression of glutathione peroxidase, superoxide dismutase, thioredoxin reductase, and peroxiredoxins were less pronounced. The expression of xenobiotics conjugating glutathione-S-transferases was increased by 1.6-2.6 times. On the other hand, the expression of NADPH:cytochrome P450 reductase was responsible for the single-electron reduction in TPZ and for the 2.1-fold decrease. These data support the fact that the main mechanism of action of TPZ under aerobic conditions is oxidative stress. The unchanged expression of intranuclear antioxidant proteins peroxiredoxin, glutaredoxin, and glutathione peroxidase, and a modest increase in the expression of DNA damage repair proteins, tend to support non-site-specific but not intranuclear oxidative stress as a main factor of TPZ aerobic cytotoxicity.

### Keywords
Tirapazamine; hepatoma; resistance

### Affiliations
EMBL
Institute of Biochemistry of Vilnius University, Vilnius, Lithuania

### Submitter
Frank Stein

### Lab Head
Dr Narimantas Cenas
Institute of Biochemistry of Vilnius University, Vilnius, Lithuania


