### Accession
PXD004984

### Title
Quantitative proteomic analysis of Murine Norovirus infected BV-2 cells (v2)

### Description
SILAC-labelling analysis of murine BV-2 cells infected at high multiplicity of infection (10 TCID50/cell) and harvested at 4h or 9h post-infection. Represents the first quantitative proteomics analysis of norovirus-infected cells. A paired AP-MS dataset investigates the effects of MNV infection on eukaryotic initiation factor (eIF) complex formation with this dataset providing data on the overall abundance of eIF components in infected cells. This is a reanalysis of an earlier dataset (PXD004015) performed in Maxquant (v1.5.5.1).

### Sample Protocol
Cells were labelled in SILAC DMEM containing R0K0 (0h/Mock), R6K4 (4h post-infection) or R10K8 (9h post-infection). For one repeat the R6K4 and R10K8 labelled cells were swapped. Cells were harvested in RIPA buffer (50mM Tris-HCl pH 8.0, 150mM NaCl, 1mM EDTA, 1% Triton X-100 and 0.1% SDS supplemented with protease inhibitors) at the indicated timepoint and subject to SDS-PAGE electrophoresis on precast gels. Gels were cut into 10 slices which were subjected to in-gel tryptic digestion using a ProGest automated digestion unit. The resulting peptides were fractionated using a Dionex Ultimate 3000 nanoHPLC system in line with an LTQ-Orbitrap Velos mass spectrometer (Thermo Scientific). In brief, peptides in 1% (vol/vol) formic acid were injected onto an Acclaim PepMap C18 nano-trap column (Dionex). After washing with 0.5% (vol/vol) acetonitrile 0.1% (vol/vol) formic acid peptides were resolved on a 250 mm × 75 μm Acclaim PepMap C18 reverse phase analytical column (Dionex) over a 150 min organic gradient, using 7 gradient segments (1-6% solvent B over 1min., 6-15% B over 58min., 15-32%B over 58min., 32-40%B over 3min., 40-90%B over 1min., held at 90%B for 6min and then reduced to 1%B over 1min.) with a flow rate of 300 nl min−1. Solvent A was 0.1% formic acid and Solvent B was aqueous 80% acetonitrile in 0.1% formic acid. Peptides were ionized by nano-electrospray ionization at 2.1 kV using a stainless steel emitter with an internal diameter of 30 μm (Thermo Scientific) and a capillary temperature of 250°C. Tandem mass spectra were acquired using an LTQ- Orbitrap Velos mass spectrometer controlled by Xcalibur 2.1 software (Thermo Scientific) and operated in data-dependent acquisition mode. The Orbitrap was set to analyze the survey scans at 60,000 resolution (at m/z 400) in the mass range m/z 300 to 2000 and the top six multiply charged ions in each duty cycle selected for MS/MS in the LTQ linear ion trap. Charge state filtering, where unassigned precursor ions were not selected for fragmentation, and dynamic exclusion (repeat count, 1; repeat duration, 30s; exclusion list size, 500) were used. Fragmentation conditions in the LTQ were as follows: normalized collision energy, 40%; activation q, 0.25; activation time 10ms; and minimum ion selection intensity, 500 counts.

### Data Protocol
The raw data files were processed and quantified using Maxquant v1.5.5.1  and searched against the Uniprot Mouse database (51,418 entries, dated May 14th, 2016) plus a custom fasta file generated in-house containing the MNV-1 (accession DQ285629) protein sequences using the built-in Andromeda search engine.  Peptide precursor mass tolerance was set at 4.5ppm, and MS/MS tolerance was set at 0.5Da.Search criteria included carbaminomethylation of cysteine as a fixed modification. Oxidation of methionine and N-terminal acetylation were selected as variable modifications. Quantification was based on Light (Arg 0, Lys 0) Medium (Arg 6, Lys 4) and Heavy (Arg 10, Lys 8) SILAC labels. Searches were performed with full tryptic digestion, a minimum peptide length of 7 amino acids, and a maximum of 2 missed cleavages were allowed. The reverse database search option was enabled and the maximum false discovery rate for both peptide and protein identifications was set to 0.01. Quantitation were performed using a mass precision of 2ppm and the requantify option in Maxquant was enabled. The presented protein ratios represent the median of the raw measured peptide ratios for each protein.

### Publication Abstract
Noroviruses produce viral RNAs lacking a 5' cap structure and instead use a virus-encoded viral protein genome-linked (VPg) protein covalently linked to viral RNA to interact with translation initiation factors and drive viral protein synthesis. Norovirus infection results in the induction of the innate response leading to interferon stimulated gene (ISG) transcription. However, the translation of the induced ISG mRNAs is suppressed. A SILAC-based mass spectrometry approach was employed to analyze changes to protein abundance in both whole cell and m7GTP-enriched samples to demonstrate that diminished host mRNA translation correlates with changes to the composition of the eukaryotic initiation factor complex. The suppression of host ISG translation correlates with the activity of the viral protease (NS6) and the activation of cellular caspases leading to the establishment of an apoptotic environment. These results indicate that noroviruses exploit the differences between viral VPg-dependent and cellular cap-dependent translation in order to diminish the host response to infection.

### Keywords
Virus, Translation, Eif, Norovirus, Mouse, Mnv, Silac, Initiation

### Affiliations
University of Cambridge, DIvision of Virology, Department of Pathology.
Northeastern University

### Submitter
Edward Emmott

### Lab Head
Dr Ian Goodfellow
University of Cambridge, DIvision of Virology, Department of Pathology.


