### Accession
PXD020049

### Title
Pb stress and ectomycorrhizas: stronger and more effective proteomic responses in poplar roots inoculated with Paxillus involutus isolate characterised by a low root colonization ratio

### Description
The commonly observed increased heavy metal tolerance of ectomycorrhized plants is usually linked with the protective role of the fungal hyphae covering colonized plant root tips. However, the molecular effects of ECM-triggered increases in plant mass and the ‘dilution effect’ are unknown. Here, we examined Populus × canescens microcuttings inoculated with the Paxillus involutus isolate, which triggered an increase in poplar growth despite successful colonization of only 1.9% ± 0.8 of root tips. Analyzed plants grown for 6 weeks in agar medium enriched with 0.75 mM Pb(NO3)2 lacked a protective fungal biofilter. In minimally colonized ‘bare’ roots, the proteome response to Pb was similar to that in noninoculated plants (e.g., higher abundances of PM- and V-type H+ 31 ATPases and lower abundance of ribosomal proteins). However, the more intensive activation of molecular processes leading to Pb sequestration or redirection of the root metabolic flux into amino acid and Pb chelate (phenolic and citrate) biosynthesis coexisted with lower Pb uptake compared to that in controls. The molecular Pb response of inoculated roots was more intense and effective than that of noninoculated roots in poplars.

### Sample Protocol
Root proteins were isolated from 100 mg of pooled root sample according to the modified phenol extraction method described in Szuba et al. (2019). A minimum of five replicates were performed per treatment. The proteins were resuspended in buffer (4 M urea and 50 mM ammonium carbonate), and their concentrations were estimated using a 2-D Quant Kit according to the manufacturer’s procedure (GE Healthcare LS). Root protein extracts (10 μg of protein) were digested in solution using a standard method, and the obtained peptide solutions were used for each run (directly injected onto the LC column). Peptide solutions were desalted and concentrated (using an RP C18 1 cm precolumn; Thermo Fisher Scientific, Waltham, MA, USA) prior to separation using a Dionex UltiMate 3000 RSLCnano System equipped with a 75 μm i.d. 25 cm RP C18 Acclaim PepMap column with a particle size of 2 μm and a pore size of 100 Å (Thermo Fisher Scientific). The Pierce LTQ ESI Positive Ion Calibration Solution (Thermo Fisher Scientific.) was used for system calibration. The following LC buffers were used: buffer A (0.1% (v/v) formic acid in Milli-Q water) and buffer B (0.1% formic acid in 90% acetonitrile). The peptides were eluted from the column with a constant flow rate of 300 nL min-1 and a linear gradient of buffer B from 5% to 65% over 185 min. The eluted peptides were analyzed using the Q-Exactive Orbitrap mass spectrometer (Thermo Fisher Scientific) operated in the data-dependent MS/MS mode. The instrument was operated with the following settings. The resolution was set to 70,000 for MS scans and 17,500 for MS/MS scans to increase the acquisition rate. The mass spectra were acquired in the range from 300 to 2000 m/z. The MS AGC target was set to 1ˆ106 counts, whereas the MS/MS AGC target was set to 5ˆ104. Dynamic exclusion was set with a duration of 20 s. The isolation window was set to 2 m/z.

### Data Protocol
Protein identification and label-free data normalization were performed using MaxQuant 1.5.3.30 (Max Planck, Munich, Germany) prior to quantitation. The mass spectra were compared against the SwissProt database using the Viridiplantae taxonomy filter. The data were then evaluated, and the statistics were calculated using Perseus software (version 1.4.1.3, Max Planck Institute of Biochemistry, Martinsried, Germany). The normalized data were filtered for reverse identifications (decoy analysis), contaminants, and proteins identified only by sites. The LFQ intensities were transformed to log values and filtered for blanks in samples. Then, missing values were replaced with values calculated from a normal distribution (imputation). The obtained matrix was then used for statistical calculation (see below).

### Publication Abstract
The commonly observed increased heavy metal tolerance of ectomycorrhized plants is usually linked with the protective role of the fungal hyphae covering colonized plant root tips. However, the molecular tolerance mechanisms in heavy metal stressed low-colonized ectormyocrrhizal plants characterized by an ectomycorrhiza-triggered increases in growth are unknown. Here, we examined <i>Populus &#xd7; canescens</i> microcuttings inoculated with the <i>Paxillus involutus</i> isolate, which triggered an increase in poplar growth despite successful colonization of only 1.9% &#xb1; 0.8 of root tips. The analyzed plants, lacking a mantle-a protective fungal biofilter-were grown for 6 weeks in agar medium enriched with 0.75 mM Pb(NO<sub>3</sub>)<sub>2</sub>. In minimally colonized 'bare' roots, the proteome response to Pb was similar to that in noninoculated plants (e.g., higher abundances of PM- and V-type H<sup>+</sup> ATPases and lower abundance of ribosomal proteins). However, the more intensive activation of molecular processes leading to Pb sequestration or redirection of the root metabolic flux into amino acid and Pb chelate (phenolics and citrate) biosynthesis coexisted with lower Pb uptake compared to that in controls. The molecular Pb response of inoculated roots was more intense and effective than that of noninoculated roots in poplars.

### Keywords
Lc-ms/ms, Ectomycorrhiza, Comparative proteomics, Populus canescens

### Affiliations
Institute of Bioorganic Chemistry Polish Academy of Sciences
Laboratory of Mass Spectrometry, Institute of Bioorganic Chemistry Polish Academy of Sciences

### Submitter
Lukasz Marczak

### Lab Head
Dr Lukasz Marczak
Laboratory of Mass Spectrometry, Institute of Bioorganic Chemistry Polish Academy of Sciences


