### Accession
PXD005232

### Title
N-GlycositeAtlas, a database resource for mass spectrometry-based human N-linked glycoproteins and glycosylation sites

### Description
N-Linked glycoprotein is a highly interesting class of proteins for clinical and biological research. The large-scale characterization of N-linked glycoproteins accomplished by mass spectrometry-based glycoproteomics has provided valuable insights into the interdependence of glycoprotein structure and protein function. However, these studies focus mainly on the analysis of specific sample type, and lack integration of each glycoproteomic data from different tissues, body fluids or cell types. In this study, we established N-GlycositeAtlas, a comprehensive database resource of human N-linked glycoproteins and their glycosylation sites identified by mass spectrometry. N-GlycositeAtlas contains more than 30,000 glycosite-containing peptides (representing >14,000 N-glycosylation sites) from more than 7,000 N-glycoproteins from different biological sources including human-derived tissues, body fluids and cell lines from over 100 studies. The entire human N-glycoproteome database as well as 22 sub-databases associated with individual tissues or body fluids can be downloaded from the N-GlycositeAtlas website at nglycositeatlas.biomarkercenter.org.

### Sample Protocol
The mass spectrometry identified glycosite-containing peptides from human sources (including tissues, body fluids, and cell lines) were initially collected from 33 datasets including 15 published and 18 unpublished datasets generated in our laboratory. To expand the database, we also collected human glycosite-containing peptides from all papers regarding to human glycosite-containing peptide analysis published since 2003 (collected on November, 2015). These publications were collected based on their citation of one of the following glycoproteomics technology papers: 1) hydrazide chemistry; 2) lectin enrichment; 3) hydrophilic affinity; 4) size extraction chromatography; and 5) FASP-based lectin enrichment. Using the strategy, we eventually collected 22,618 human glycosite-containing peptides from 70 papers published by other laboratories. The detailed sample information about the unpublished datasets can be found in the "all dataset information" from the uploaded files.

### Data Protocol
Among these unpublished datasets generated in our laboratory, eleven of them were generated before 2008. These samples were enriched by the SPEG method and analyzed by an LTQ ion trap (Thermo Fisher, San Jose, CA) or Q-TOF (Waters, Beverly, MA) mass spectrometers followed by being searched with the SEQUEST algorithm99 against a human International Protein Index database (IPI) 100. The peptide mass tolerance was 2.0 Da. Carbamidomethylation (C, +57.0215Da) was set as a static modification; oxidation (M, +15.9949Da) and deamination (N, +0.98Da) were set as dynamic modifications. The output files were further evaluated by INTERACT and ProteinProphet 101, 102. The identified peptides were filtered by a PeptideProphet probability score â‰¥0.9 and the deamidation of asparagine (N) in the N-X-S/T motif. These glycosite information are accessible via the Unipep website (http://www.unipep.org) 7 and/or PeptideAtlas website (http://www.peptideatlas.org)103. The other seven big datasets were generated using Orbitrap Velos and/or Q-Exactive mass spectrometers (Thermo Fisher Scientific, Bremen, Germany) after former glycopeptide enrichment using SPEG method and searched against an NCBI Reference Sequence (RefSeq) human protein database 104 using SEQUEST99 in Proteome Discoverer v1.4 (Thermo Fisher Scientific). The database searching parameters for glycosite-containing peptide identification were set as follows: two missed cleavages were allowed for trypsin digestion with 10ppm precursor mass tolerance and 0.06Da fragment mass tolerance. Carbamidomethylation (C) was set as a static modification, while oxidation (M) and deamination (N) were set as dynamic modifications. For iTRAQ-labeled samples, iTRAQ-4plex (peptide N-terminal) and iTRAQ-4plex (K) were added as dynamic modifications. The glycosite-containing peptide identifications were further filtered by 1% FDR and deamination in the N-X-S/T motif of the peptides. The detailed data processing information about the unpublished datasets can be found in the "all dataset information" from the uploaded files.

### Publication Abstract
None

### Keywords
Human, Proteomics, Glycoprotein, Database, Mass spectrometry, Glycosylation site

### Affiliations
Johns Hopkins University
Department of Pathology, Johns Hopkins University, Baltimore, Maryland 21287

### Submitter
Shisheng Sun

### Lab Head
Dr Shisheng Sun
Department of Pathology, Johns Hopkins University, Baltimore, Maryland 21287


