### Accession
PXD021939

### Title
LONP1 and mtHSP70 cooperate to promote mitochondrial protein folding

### Description
Because 99% of the mitochondrial proteome is encoded by the nucleus, most mitochondrial precursor polypeptides are imported from the cytosol into the mitochondrion, where they must efficiently fold into their functional state.  Mitochondrial precursors are imported as unfolded polypeptides due to the limited pore size of the import machinery.  For proteins of the mitochondrial matrix and inner membrane, two separate and highly conserved chaperone systems, HSP60 and mitochondrial HSP70 (mtHSP70), facilitate protein folding.  Here we show that LONP1, a AAA+ protease of the mitochondrial matrix, works with the mtHSP70 chaperone system to promote mitochondrial protein folding.  Inhibition of LONP1 results in aggregation of a protein subset similar to that caused by knockdown of DNAJA3, a co-chaperone of mtHSP70.  LONP1 is required for DNAJA3 and mtHSP70 solubility, and its ATPase, but not its protease activity, is required for this function.  In vitro, LONP1 collaborates with mtHSP70 and its co-factors to stabilize a folding intermediate of OXA1L.  In the absence of LONP1, the interaction of mtHSP70 with OXA1L is futile and results in substrate-induced coaggregation.  Our results identify mitochondrial LONP1 as a critical factor in the mtHSP70 folding pathway and directly demonstrate its long suspected chaperone activity.

### Sample Protocol
Mitochondria were isolated from control, LONP1- or DNAJA3-knockdown 143B cells and lysed using Triton X-100 extraction. Insoluble mitochondrial protein pellets were separated by centrifugation at 20,817 g for 10 min at 4Â°C and solubilized in urea buffer (8 M urea, 100 mM TrisHCl, pH 8.5). The solubilized samples were reduced, alkylated, and digested by trypsin overnight at room temperature and desalted by HPLC with a Microm Bioresources C8 peptide Microtrap.

### Data Protocol
Raw data were searched in Proteome Discoverer 2.4 (Thermo Scientific) using the Byonic search algorithm (Protein Metrics) and Uniprot human database. PD-Byonic search parameters were as follows: fully Tryptic peptides with no more than 2 missed cleavages, precursor mass tolerance of 10 ppm and fragment mass tolerance of 20 ppm, and a maximum of 2 common modifications and 2 rare modifications. Cysteine carbamidomethylation was set as a static modification, while methionine oxidation was a common dynamic modification (up to 2 per peptide). Methionine loss on protein N-termini, methionine loss + acetylation on protein N-termini, protein N-terminal acetylation, lysine acetylation, and phosphorylation of serine, threonine, and tyrosine were rare dynamic modifications (only 1 each). Byonic protein-level FDR was set at 0.01, while Percolator FDRs were set at 0.01 (strict) and 0.05 (relaxed). In the consensus step, peptide and PSM FDRs were set at 0.001 (strict) and 0.01 (relaxed), with peptide confidence at least medium, lower confidence peptides excluded, minimum peptide length set at 6, and apply strict parsimony set to false. LFQ was then performed with the Minora feature detector, feature mapper, and precursor ions quantifier nodes. Retention time alignment was performed with maximum RT shift of 5 min and a minimum S/N threshold of 10. Quantified peptides included unique + razor, protein groups were considered for peptide uniqueness, shared Quan results were not used, Quan results with missing values were not rejected, and precursor abundance was based on intensity. Abundances were normalized using the total intensities of all non-mitochondrial proteins identified in a pre-search of the raw data files from the WT, Lon, and DNAJA3 samples with identical search parameters. Imputation was then performed using the low abundance resampling method.

### Publication Abstract
Most mitochondrial precursor polypeptides are imported from the cytosol into the mitochondrion, where they must efficiently undergo folding. Mitochondrial precursors are imported as unfolded polypeptides. For proteins of the mitochondrial matrix and inner membrane, two separate chaperone systems, HSP60 and mitochondrial HSP70 (mtHSP70), facilitate protein folding. We show that LONP1, an AAA+ protease of the mitochondrial matrix, works with the mtHSP70 chaperone system to promote mitochondrial protein folding. Inhibition of LONP1 results in aggregation of a protein subset similar to that caused by knockdown of DNAJA3, a co-chaperone of mtHSP70. LONP1 is required for DNAJA3 and mtHSP70 solubility, and its ATPase, but not its protease activity, is required for this function. In vitro, LONP1 shows an intrinsic chaperone-like activity and collaborates with mtHSP70 to stabilize a folding intermediate of OXA1L. Our results identify LONP1 as a critical factor in the mtHSP70 folding pathway and demonstrate its proposed chaperone activity.

### Keywords
Human, Mitochondria, Aggregates, Protein homeostasis

### Affiliations
Caltech Proteome Exploration Lab
Proteome Exploration Laboratory Beckman Institute California Institute of Technology

### Submitter
Brett Lomenick

### Lab Head
Dr Spiros D. Garbis
Proteome Exploration Laboratory Beckman Institute California Institute of Technology


