### Accession
PXD009035

### Title
Structural analysis of human ARS2, a platform for co-transcriptional RNA sorting

### Description
ARS2 is a highly conserved metazoan protein involved in numerous aspects of nuclear RNA metabolism. As a direct partner of the nuclear cap-binding complex (CBC) it mediates interactions with diverse RNA processing and transport machineries in a transcript-dependent manner. Here we present the human ARS2 crystal structure, which exhibits similarities and metazoan-specific differences to the plant homologue SERRATE, most notably an additional RRM domain. We present biochemical, biophysical and cellular interactome data comparing wild type and mutant ARS2 that identify regions critical for interactions with FLASH (involved in histone mRNA biogenesis), NCBP3 (a putative cap-binding protein involved in mRNA export) and single-stranded RNA. We show that FLASH and NCBP3 have overlapping binding sites on ARS2 and that CBC-ARS2-NCBP3 form a ternary complex that is mutually exclusive with CBC-ARS-PHAX (involved in snRNA export). Our results support that mutually exclusive higher order CBC-ARS2 complexes are critical in determining Pol II transcript fate.

### Sample Protocol
For TMT labelling all reagents were prepared in 50 mM HEPES pH 8.5. Cysteines were reduced using dithiothreitol (56Â°C, 30 min, 10 mM). After cooling the samples to 24Â°C and they were alkylated with iodacetamide (room temperature, in the dark, 30 min, 10 mM). Subsequently, the samples were prepared for LC-MS/MS using the SP3 protocol35, digested with sequencing grade modified trypsin (enzyme to protein ratio, 1:50) (Promega) at 37Â°C overnight. TMT10plexâ„¢ Isobaric Label Reagent (ThermoFisher) was added to the samples according the manufacturerâ€™s instructions. Labelled peptides were cleaned up using OASISÂ® HLB Î¼Elution Plate (Waters). Offline high pH reverse phase fractionation was performed using an Agilent 1200 Infinity high-performance liquid chromatography system, equipped with a Gemini C18 column (3 Î¼m, 110 Ã…, 100 x 1.0 mm, Phenomenex). The solvent system consisted of 20 mM ammonium formate pH 10.0 as mobile phase and 100% (v/v) acetonitrile as mobile phase. Peptides were separated using the UltiMate 3000 RSLC nano LC system (Dionex) fitted with a trapping cartridge (Î¼-Precolumn C18 PepMap 100, 5Î¼m, 300 Î¼m i.d. x 5 mm, 100 Ã…) and an analytical column (Acclaim PepMap 100 75 Î¼m x 50 cm C18, 3 Î¼m, 100 Ã…). The outlet of the analytical column was coupled directly to a QExactive plus (Thermo) using the proxeon nanoflow source in positive ion mode. Solvent A was water with 0.1% formic acid and solvent B was acetonitrile with 0.1% formic acid. Trapping time was 6 min with a constant flow of solvent A at 30 Î¼L min-1 onto the trapping column. Peptides were eluted via the analytical column a constant flow of 0.3 Î¼L min-1. During the elution step, the percentage of solvent B increased in a linear fashion from 2% to 4% in 4 min, from 4% to 8% in 2 min, then 8% to 28% for a further 96 min, and finally from 28% to 40% in another 10 min. Column cleaning at 80% B followed, lasting 3 min, before returning to initial conditions for the re-equilibration, lasting 10 min. The peptides were introduced into the mass spectrometer (QExactive plus, ThermoFisher) via a Pico-Tip Emitter 360 Î¼m OD x 20 Î¼m ID; 10 Î¼m tip (New Objective) and a spray voltage of 2.3 kV was applied. The capillary temperature was set at 320Â°C. Full scan MS spectra with mass range 350-1400 m/z were acquired in profile mode in the FT with resolution of 70,000. The filling time was set at maximum of 100 ms with a limitation of 3x106 ions. DDA was performed with the resolution of the Orbitrap set to 35000, with a fill time of 120 ms and a limitation of 2x105 ions. Normalised collision energy of 32 was used. A loop count of 10 with count 1 was used and a minimum AGC trigger of 2e2 was set. Dynamic exclusion time of 30 s was applied. The peptide match algorithm was set to â€˜preferredâ€™ and charge exclusion â€˜unassignedâ€™, charge states 1, 5 - 8 were excluded. Isolation window was set to 1.0 m/z and 100 m/z set as the fixed first mass. MS/MS data was acquired in profile mode.

### Data Protocol
Acquired data was processed by IsobarQuant36 and Mascot (v2.2.07) and searched against a Uniprot Homo sapiens proteome database (UP000005640) containing common contaminants, reversed sequences and construct sequences. The data was searched with the following modifications: Carbamidomethyl (C) and TMT10 (K) (fixed modification), Acetyl (N-term), Oxidation (M) and TMT10 (N-term) (variable modifications). The mass error tolerance for the full scan MS spectra was set to 10 ppm and for the MS/MS spectra to 0.02 Da. A maximum of two missed cleavages was allowed. For protein identification, a minimum of two unique peptides with a peptide length of at least seven amino acids and a false discovery rate below 0.01 were required on the peptide and protein level.

### Publication Abstract
ARS2 is a highly conserved metazoan protein involved in numerous aspects of nuclear RNA metabolism. As a direct partner of the nuclear cap-binding complex (CBC), it mediates interactions with diverse RNA processing and transport machineries in a transcript-dependent manner. Here, we present the human ARS2 crystal structure, which exhibits similarities and metazoan-specific differences to the plant homologue SERRATE, most notably an additional RRM domain. We present biochemical, biophysical and cellular interactome data comparing wild type and mutant ARS2 that identify regions critical for interactions with FLASH (involved in histone mRNA biogenesis), NCBP3 (a putative cap-binding protein involved in mRNA export) and single-stranded RNA. We show that FLASH and NCBP3 have overlapping binding sites on ARS2 and that CBC-ARS2-NCBP3 form a ternary complex that is mutually exclusive with CBC-ARS-PHAX (involved in snRNA export). Our results support that mutually exclusive higher-order CBC-ARS2 complexes are critical in determining Pol II transcript fate.

### Keywords
Protein interaction, Ars2

### Affiliations
EMBL
European Molecular Biology Laboratory, Grenoble Cedex 9, France. Tel:  +(33)476207238 Email: cusack@embl.fr

### Submitter
Frank Stein

### Lab Head
Dr Stephen Cusack
European Molecular Biology Laboratory, Grenoble Cedex 9, France. Tel:  +(33)476207238 Email: cusack@embl.fr


