### Accession
PXD012117

### Title
Deuterium-free, Three-plexed Peptide Di-Ethylation for Highly Accurate Quantitative Proteomics

### Description
The deuterium, frequently used stable isotope in isotopic labeling for quantitative proteomics, could deteriorate the accuracy and precision of proteome quantification owing to the retention time shift of deuterated peptides from the hydrogenated counterpart. Herein, we introduce a novel three-plexed peptide ‘di-ethylation’ using only 13C-isotopologues of acetaldehyde and demonstrate that the accuracy and precision of our method in proteome quantification are significantly superior to the conventional deuterium-based di-methylation labeling in both a single shot and multidimensional LC-MS/MS analysis of HeLa proteome. Furthermore, in time-resolved profiling of Xenopus laevis early embryogenesis, our 3-plexed di-ethylation outperformed isobaric labeling approaches in terms of quantification accuracy or number of protein identification, generating >2 times more differentially expressed proteins. Our cost-effective and highly accurate 3-plexed di-ethylation method could contribute to various types of quantitative proteomics applications, in which three of multiplexity would be sufficient.

### Sample Protocol
Di-ethylation of peptides and other chemical labeling. The lyophilized peptides were resuspended with 100 mM sodium acetate buffer (pH 5.5) for reductive alkylation35. First for di-ethylation, one of three acetaldehyde 13C-isotopologues (CH3CHO (Sigma), 13CH3CHO (Cambridge Isotope Laboratories), and 13CH313CHO (Sigma) for light, medium, and heavy labeling, respectively) was added to the peptides sample to be 500 mM of concentration and followed by the addition of NaBH3CN (Sigma) to be 250 mM (Figure 1a). For di-methylation, a formaldehyde isotopologue (CH2O (Sigma), CD2O (CDN Isotopes), and 13CD2O (Sigma) for light, medium, and heavy labeling, respectively) was combined with the adequate reducing reagent (NaBH3CN for light and medium labeling, or NaBD3CN (CDN Isotopes) for heavy labeling) (Figure S1a). All di-methylating reagents were added at same concentration as the case of di-ethylation. Samples were briefly vortexed and incubated for 1 hour at RT in the Thermomixer (Eppendorf) with 500 rpm. The reductive alkylation process can be repeated once more, if necessary. The reductive alkylating reagents were quenched by addition of 1 M ABC solution to be 500 mM. TMT-labeling was carried out by following the manufacturer’s recommendation. 20 μg of digests from a single embryo for each stage were divided into two identical aliquots and subjected to duplicate TMT-labeling as follows: TMT-126 & 129 for stage 1, TMT-127 & 130 for stage 8, TMT-128 & 131 for stage 13.  Multidimensional mid-pH RPLC fractionation. Microscale (20 μg (HeLa) or 30 μg (X. laevis)) mid-pH RPLC off-line fractionation was carried using an in-house packed 200 μm i.d. C18 capillary column operated by nanoACQUITY UPLC (Waters) at 3 μL/min of flow rate with a linear gradient of solvent A (10 mM ABC in water) and solvent B (10 mM ABC in 90 % ACN) from 2 % to 40 % of solvent B for 90 min. The eluent was automatically collected into 96 well plate per each minute using Triversa NanoMate (Advion) and then concatenated to 6 (HeLa) or 24 (X. laevis) fractions.  LC-MS/MS or LC-MS3 analysis. LC-MS/MS analysis of enolase and HeLa samples for the comparion of DE- and DM-3plex were carried by Q-Exactive Orbitrap MS (Thermo Fisher Scientific) coupled with a nanoACQUITY UPLC equipped with an in-house packed trap (150 μm i.d. x 3 cm) and analytical column (75 μm i.d. x 100 cm) using 3 μm of Jupiter C18 particle (Phenomenex). A linear gradient of solvent A (water with 0.1 % formic acid) and solvent B (ACN with 0.1 % formic acid) in ACN) was applied at a flow rate 300 nL/min: 5 to 10 % solvent B for initial 5 min and 10 to 40 % solvent B for next 90 min.  X. laevis embryogenesis was profiled using Orbitrap Fusion Lumos Tribrid MS (Thermo Fisher Scientific) coupled with aforementioned LC system. For the MS2 only method for the analyses of DE-3plex and TMT-MS2 samples, full MS scans (m/z 300 – 1,500) were acquired at a resolution of 60k (at m/z 200). HCD fragmentation was performed under 30 % of NCE via precursor isolation within 1.6 Th of window. The MS2 scans were acquired at a resolution of 15k (ITmax 30 ms and AGC 2E4). For the MS3 analysis, the most intense 10 ions were first isolated at 0.5 Th of precursor isolation width under the identical full MS scan settings for CID MS2 in ion trap (ITmax 150 ms and AGC 4E3). The 10 most intense MS2 fragment ions were synchronously isolated for HCD MS3 (AGC 1.5E5, ITmax 250 ms, and NCE 55 %) at 2 m/z of isolation width

### Data Protocol
Database searching and quantitative analysis. MaxQuant suite36 (ver. 1.5.2.8) was used for the quantitative analysis of all the di-ethylated and di-methylated samples. UniProt human database (March, 2016) or PHROG reference set37 was used and the false discovery rate (FDR) of protein group and PSM were set < 1%. For 3-plexed modifications on N-terminus and lysine, 28.0313, 32.0564, and 36.0756 Da were filled in as the light, medium, and heavy labels, respectively, for di-methylation, while 56.0626, 58.0693, and 60.0670 Da were used for di-ethylation. More than two non-redundant peptides for a specific protein group were used for protein quantification.  Proteome Discoverer (PD) ver. 2.1 (Thermo Fisher Scientific), which is the most popular software for TMT-labeling, was used for the quantitative analysis of TMT-labeled X. laevis datasets. Resulting peptide hits were filtered at maximum 1 % FDR by Percolator algorithm. Carbamidomethylation of cysteine and TMT labeling of lysine and peptide n-termini were set as fixed modification and oxidation of methionine was chosen as a variable modification. Full lysine specificity was required with up to two missed cleavage sites allowed; mass tolerances for precursor ions and fragment ions were set to 10 ppm and 0.8 Da, respectively. The reporter ion ratios reported from PD were adjusted by applying the isotopic correction factors of TMT kit provided by the manufacturer. Only all reporter ions containing spectra were designated as ‘quantifiable spectra’.  For the clustering analysis, log2 fold-changes between embryonic stages were calculated. The k-means clustering of log2 fold-changes was done using SciPy package, open source scientific tools for Python (http://www.scipy.org). Data analysis and visualization following clustering analysis was carried out by in-house software.

### Publication Abstract
The deuterium, a frequently used stable isotope in isotopic labeling for quantitative proteomics, could deteriorate the accuracy and precision of proteome quantification owing to the retention time shift of deuterated peptides from the hydrogenated counterpart. We introduce a novel three-plexed peptide "diethylation" using only <sup>13</sup>C isotopologues of acetaldehyde and demonstrate that the accuracy and precision of our method in proteome quantification are significantly superior to the conventional deuterium-based dimethylation labeling in both a single-shot and multidimensional LC-MS/MS analysis of the HeLa proteome. Furthermore, in time-resolved profiling of Xenopus laevis&#xa0;early embryogenesis, our 3-plexed diethylation outperformed isobaric labeling approaches in terms of the quantification accuracy or the number of protein identifications, generating more than two times more differentially expressed proteins. Our cost-effective and highly accurate 3-plexed diethylation method could contribute to various types of quantitative proteomics applications in which three of multiplexity would be sufficient.

### Keywords
Deuterium effect, Quantitative accuracy, Isobaric labeling, Di-methylation, Isotopic labeling, Di-ethylation

### Affiliations
Kyung Hee University
KHU

### Submitter
Jae Hun Jung

### Lab Head
Dr Jaehun Jung
Kyung Hee University


