### Accession
PXD039780

### Title
Identification of proteins crosslinked to lysosomal sphingosine and cholesterol probes

### Description
HeLa cells were labelled with lysosomally localized photocrosslinkable and clickable probes for sphingosine and cholesterol. Upon photocrosslinking and cell lysis, the protein-lipid complexes were clicked to biotin azide and enriched via Streptavidin-immunoprecipitation. Enriched protein-lipid complexes were then analyzed by LC-MS/MS to identify proteins interacting with sphingosine and cholesterol, respectively.

### Sample Protocol
Enriched protein-lipid complexes were subjected to an in-solution tryptic digest using a modified version of the Single-Pot Solid-Phase-enhanced Sample Preparation (SP3) protocol. In total three biological replicates were prepared. Eluates were added to Sera-Mag Beads (Thermo Scientific) in 10 ÃƒÅ½Ã‚Â¼l 15% formic acid and 30 ÃƒÅ½Ã‚Â¼l of ethanol. Binding of proteins was achieved by shaking for 15 min at room temperature. SDS was removed by 4 subsequent washes with 200 ÃƒÅ½Ã‚Â¼l of 70% ethanol. Proteins were digested overnight at room temperature with 0.4 ÃƒÅ½Ã‚Â¼g of sequencing grade modified trypsin in 40 ÃƒÅ½Ã‚Â¼l Hepes/NaOH, pH 8.4 in the presence of 1.25 mM TCEP and 5 mM chloroacetamide. Beads were separated, washed with 10 ÃƒÅ½Ã‚Â¼l of an aqueous solution of 2% DMSO and the combined eluates were dried down. Peptides were reconstituted in 10 ÃƒÅ½Ã‚Â¼l of H2O and reacted for 1 h at room temperature with 80 ÃƒÅ½Ã‚Â¼g of TMT10plex label reagent dissolved in 4 ÃƒÅ½Ã‚Â¼l of acetonitrile. Excess TMT reagent was quenched by the addition of 4 ÃƒÅ½Ã‚Â¼l of an aqueous 5% hydroxylamine solution. TMT-labelled peptides were mixed and purified by a reverse phase clean-up step (OASIS HLB 96-well ÃƒÅ½Ã‚Â¼Elution Plate. Peptides were analyzed by LC-MS/MS on an Orbitrap Fusion Lumos mass spectrometer. To this end, peptides were separated using an Ultimate 3000 nano RSLC system (Dionex) equipped with a trapping cartridge (Precolumn C18 PepMap100, 5 mm, 300 ÃƒÅ½Ã‚Â¼m i.d., 5 ÃƒÅ½Ã‚Â¼m, 100 ÃƒÆ’Ã¢â‚¬Â¦) and an analytical column (Acclaim PepMap 100. 75 ÃƒÆ’Ã¢â‚¬â€� 50 cm C18, 3 mm, 100 ÃƒÆ’Ã¢â‚¬Â¦) connected to a nanospray-Flex ion source. The peptides were loaded onto the trap column at 30 ÃƒÅ½Ã‚Â¼l per min using solvent A (0.1% formic acid) and eluted using a gradient from 2 to 40% Solvent B (0.1% formic acid in acetonitrile) over 2h at 0.3 ÃƒÅ½Ã‚Â¼l per min (all solvents were of LC-MS grade). The Orbitrap Fusion Lumos was operated in positive ion mode with a spray voltage of 2.4 kV and capillary temperature of 275 Ãƒâ€šÃ‚Â°C. Full scan MS spectra with a mass range of 375ÃƒÂ¢Ã¢â€šÂ¬Ã¢â‚¬Å“1500 m/z were acquired in profile mode using a resolution of 120,000 (maximum fill time of 50 ms or a maximum of 4e5 ions (AGC) and a RF lens setting of 30%. Fragmentation was triggered for 3 s cycle time for peptide like features with charge states of 2ÃƒÂ¢Ã¢â€šÂ¬Ã¢â‚¬Å“7 on the MS scan (data-dependent acquisition). Precursors were isolated using the quadrupole with a window of 0.7 m/z and fragmented with a normalized collision energy of 38. Fragment mass spectra were acquired in profile mode and a resolution of 30,000 in profile mode. Maximum fill time was set to 64ms or an AGC target of 1e5 ions). The dynamic exclusion was set to 45 s.

### Data Protocol
Acquired data were analyzed using IsobarQuant (8) and Mascot V2.4 (Matrix Science) using a reverse UniProt FASTA Homo sapiens database (UP000005640) including common contaminants. The following modifications were taken into account: Carbamidomethyl (C, fixed), TMT10plex (K, fixed), Acetyl (N-term, variable), Oxidation (M, variable) and TMT10plex (N-term, variable). The mass error tolerance for full scan MS spectra was set to 10 ppm and for MS/MS spectra to 0.02 Da. A maximum of 2 missed cleavages were allowed. A minimum of 2 unique peptides with a peptide length of at least seven amino acids and a false discovery rate below 0.01 were required on the peptide and protein level.

### Publication Abstract
Lysosomes are catabolic organelles involved in macromolecular digestion, and their dysfunction is associated with pathologies ranging from lysosomal storage disorders to common neurodegenerative diseases, many of which have lipid accumulation phenotypes. The mechanism of lipid efflux from lysosomes is well understood for cholesterol, while the export of other lipids, particularly sphingosine, is less well studied. To overcome this knowledge gap, we have developed functionalized sphingosine and cholesterol probes that allow us to follow their metabolism, protein interactions, and their subcellular localization. These probes feature a modified cage group for lysosomal targeting and controlled release of the active lipids with high temporal precision. An additional photocrosslinkable group allowed for the discovery of lysosomal interactors for both sphingosine and cholesterol. In this way, we found that two lysosomal cholesterol transporters, NPC1 and to a lesser extent LIMP-2/SCARB2, bind to sphingosine and showed that their absence leads to lysosomal sphingosine accumulation which hints at a sphingosine transport role of both proteins. Furthermore, artificial elevation of lysosomal sphingosine levels impaired cholesterol efflux, consistent with sphingosine and cholesterol sharing a common export mechanism.

### Keywords
Photoaffinity labelling, Click chemistry, Lysosomal lipids

### Affiliations
EMBL
Heidelberg University Biohemistry Center

### Submitter
Frank Stein

### Lab Head
Dr Dr. Doris Hoeglinger
Heidelberg University Biohemistry Center


