### Accession
PXD015958

### Title
Plasma proteomic profiling of normal and over-conditioned dairy cows during the transition from late pregnancy to early lactation by LC-MS/MS

### Description
The objective of this study was to apply a quantitative proteomic approach to investigate changes in the plasma proteome of normal and over-conditioned dairy cows during the transition period. We used an animal model with a targeted difference in body condition score (BCS) at calving. For this, multiparous Holstein cows were allocated to two groups 15 weeks before anticipated calving and were fed differently to reach either high (HBCS: 7.2 NEL MJ/kg dry matter (DM) or normal BCS (NBCS: 6.8 NEL MJ/kg DM) until dry-off. Thereafter, both groups were fed identical diets. The targeted BCS difference was sustained during the dry period (7 weeks before calving) and the subsequent 12 weeks of lactation. Plasma samples (n = 5) from d -49, +7, and +21 relative to parturition were used from both groups (each n = 5) for proteome profiling in aby applying quantitative Tandem Mass Tags (TMT) approach.

### Sample Protocol
35 μg of total plasma protein from samples and internal standard (a pool of equal protein amounts from all samples used as a reference for normalization) were diluted to a volume of 50 μL using 0.1 M triethylammonium bicarbonate (TEAB, Thermo Scientific). Samples were reduced (20 mM DTT, Sigma Aldrich, St. Louis, MO, USA) for 1 h at 55 °C and alkylated (30 mM IAA, Sigma Aldrich) for 30 min at room temperature in the dark. Then, 300 µL of pre-chilled (-20 °C) acetone were added, and samples were allowed to precipitate overnight at -20 °C. Protein pellets were collected subsequently by centrifugation (9000 × g, 4 °C), dissolved in 50 μL of 0.1 M TEAB and digested on the filters at 37 °C overnight by using 1 μL of trypsin [1 mg/mL, (Promega, Madison WI); trypsin-to-protein ratio 1:35]. Peptides were labelled using the TMT 10plexTM Label Reagent Set prepared according to the manufacturer's procedure and 19 μL of the appropriate TMT label were added to each sample used for the labeling reaction (60 min, room temperature) which was quenched by 5% hydroxylamine (Sigma-Aldrich). Nine TMT-modified peptide samples were combined with the internal standard (labeled with TMT m/z 126) into the new tube, aliquoted, dried and stored at −20 °C for further analysis.  High-resolution liquid chromatography-tandem mass spectrometry (LC-MS/MS) analysis of TMT-labelled peptides was carried out using an Ultimate 3000 RSLCnano system (Dionex, Germering, Germany) coupled to a Q Exactive Plus mass spectrometer (Thermo Fisher Scientific, Bremen, Germany). Peptides were dissolved in loading solvent (1% ACN, 0.1% formic acid) and loaded onto the trap column (C18 PepMap100, 5 μm, 100 A, 300 μm × 5 mm), desalted for 12 min at a flow rate of 15 μL/min and separated on the analytical column (PepMap™ RSLC C18, 50 cm × 75 μm) using a linear gradient of 5–45% mobile phase B (0.1% formic acid in 80% ACN) over 120 min, 45% to 90% for 2 min, held at 80% for 2 min and re-equilibrated at 5% B for 20 min at the flow rate of 300 nL/min. Mobile phase A consisted of 0.1% formic acid in water. Ionization was achieved using nanospray Flex ion source (Thermo Fisher Scientific) containing a 10 μm-inner diameter SilicaTip emitter (New Objective, Woburn, MA). The MS operated in positive ion mode using the DDA Top8 method. Full scan MS spectra were acquired in the range from m/z 350.0 to m/z 1800.0 with a resolution of 70,000, 110 ms injection time, AGC target 1 × 10E6, a ± 2.0 Da isolation window and the dynamic exclusion 30 s. HCD fragmentation was performed at step collision energy (29% and 35% NCE) with a resolution of 17,500 and AGC target of 2 × 10E5. Precursor ions with unassigned charge state, as well as charge states of +1 and more than +7 were excluded from fragmentation.

### Data Protocol
Acquired MS/MS spectra were analyzed for protein identification and quantification using the SEQUEST algorithm implemented into Proteome Discoverer (version 2.0., ThermoFisher Scientific). Database search against Bos taurus FASTA files (downloaded from SwissProt database on 11/5/2016, 6365 sequences) was performed according to the following parameters: two trypsin missed cleavage sites, precursor and fragment mass tolerances of 10 ppm and 0.02 Da, respectively; carbamidomethyl (C) fixed peptide modification, oxidation (M), deamidation (N, Q) and TMT 10 plex (K, peptide N-terminus) dynamic modifications. The false discovery rate (FDR) for peptide identification was calculated using the Percolator algorithm in the Proteome Discoverer workflow based on the search results against a decoy database and was set at 1% FDR.

### Publication Abstract
This study applied a quantitative proteomics approach along with bioinformatics analyses to investigate changes in the plasma proteome of normal and overconditioned dairy cows during the transition period. Fifteen weeks before their anticipated calving date, 38 multiparous Holstein cows were selected based on their current and previous body condition scores (BCS) and allocated to either a high or a normal BCS group (19 cows each). They received different diets until dry-off to reach targeted differences in BCS and back fat thickness (BFT) until dry-off. At dry-off, normal BCS cows had a BCS &lt;3.5 (minimum, 2.75) and BFT &lt;1.2 cm (minimum, 0.58), and the high BCS cows had a BCS &gt;3.75 (maximum, 4.50) and BFT &gt;1.4 cm (maximum, 2.90). The proteomics study used a subset of 5 animals from each group. These cows were selected based on their circulating concentrations of fatty acids (FA) on d 14 postpartum and &#x3b2;-hydroxybutyrate (BHB) on d 21 postpartum, representing the greater or the lower extreme values within their BCS group, respectively. The high BCS subset (HE-HBCS) had 4.50 &lt; BCS &gt; 3.75, FA = 1.17 &#xb1; 0.46 mmol/L, and BHB = 2.15 &#xb1; 0.42 mmol/L (means &#xb1; SD), and the low BCS subset (LE-NBCS) had 3.50 &lt; BCS &gt; 2.75, FA = 0.51 &#xb1; 0.28 mmol/L, and BHB = 0.84 &#xb1; 0.17 mmol/L. Plasma samples from d -49, +7, and +21 relative to parturition were used for proteome profiling by applying the quantitative tandem mass tags (TMT) approach. Nondepleted plasma samples were subjected to reduction and digestion and then labeled with TMT 10plex reagents. High-resolution liquid chromatography-tandem mass spectrometry analysis of TMT-labeled peptides was carried out, and the acquired spectra were analyzed for protein identification and quantification. In total, 254 quantifiable proteins (criteria: 2 unique peptides and 5% false discovery rate) were identified in the plasma samples. From these, 24 differentially abundant proteins (14 more abundant, 10 less abundant) were observed in the LE-NBCS cows compared with the HE-HBCS cows during the transition period. Plasma &#x3b1;-2-macroglobulins were more abundant in HE-HBCS versus LE-NBCS cows at d +7 and +21. Gene Ontology enrichment analyses of differentially abundant proteins revealed that the acute inflammatory response, regulation of complement activation, protein activation cascade, and regulation of humoral immune response were the most enriched terms in the LE-NBCS group compared with the HE-HBCS group. In addition, we identified 24 differentially abundant proteins (16 in the LE-NBCS group, and 8 in the HE-HBCS group) during the transition period. The complement components C1q and C5 were less abundant, while C3 and C3d were more abundant in LE-NBCS compared with HE-HBCS cows. Overall, overconditioning around calving was associated with alterations in protein pathways related to acute inflammatory response and regulation of complement and coagulation cascades in transition cows.

### Keywords
Transition cows, Body condition score, Tmt proteomics

### Affiliations
VetMedZg Laboratory,Faculty of Veterinary Medicine, University of Zagreb, Zagreb, Croatia    Institute of Biodiversity, Animal Health & Comparative Medicine, College of Medicine, Veterinary Medicine and Life Sciences, University of Glasgow, Glasgow, UK
University of Zagreb, Faculty of Veterinary Medicine

### Submitter
Anita Horvatic

### Lab Head
Dr Peter David Eckersall
VetMedZg Laboratory,Faculty of Veterinary Medicine, University of Zagreb, Zagreb, Croatia    Institute of Biodiversity, Animal Health & Comparative Medicine, College of Medicine, Veterinary Medicine and Life Sciences, University of Glasgow, Glasgow, UK


