### Accession
PXD026619

### Title
Dynamic character displacement among a pair of bacterial phyllosphere commensals in situ

### Description
Differences between species promote stable coexistence in a resource-limited environment. These differences can result from interspecies competition leading to character shifts, a process referred to as character displacement. While character displacement is often interpreted as a consequence of genetically fixed trait differences between species, it can also be mediated by phenotypic plasticity in response to the presence of another species. Here, we test whether phenotypic plasticity leads to a shift in proteome allocation during co-occurrence of two bacterial species from the abundant, leaf-colonizing families Sphingomonadaceae and Rhizobiaceae in their natural habitat. Upon mono-colonizing of the phyllosphere, both species exhibit specific and shared protein functions indicating a niche overlap. During co-colonization, quantitative differences in the protein repertoire of both bacterial populations occur as a result of bacterial coexistence in planta. Specifically, the Sphingomonas strain produces enzymes for the metabolization of xylan, while the Rhizobium strain reprograms its metabolism to beta-oxidation of fatty acids fueled via the glyoxylate cycle and adapts its biotin acquisition. We demonstrate the conditional relevance of cross-species facilitation by mutagenesis leading to loss of fitness in competition in planta. Our results show that dynamic character displacement and niche facilitation mediated by phenotypic plasticity can contribute to species coexistence.

### Sample Protocol
In total, four biological independent replicates were generated for MS-based proteomics analysis. As a reference to the in planta conditions, both Rhizobium Leaf68 and Sphingomonas Leaf257 were each grown on R2A+M-agar (see above) in the plant growth chamber at the settings described above. Bacterial cell pellets were dissolved and lysed using a lysis buffer containing 100 mM ammonium bicarbonate, 8 M urea and 1x cOmplete EDTA-free protease inhibitor cocktail (Sigma-Aldrich, Buchs, Switzerland) and indirect sonication (3 x 1 min, 100% amplitude, 0.8 cycle time) in a VialTweeter (HIFU, Hielscher, Teltow, Germany). Insoluble parts were removed by centrifugation at 13,000 g for 15 min at 4°C. Protein concentration of supernatant was determined using the Pierce BCA assay kit (Thermo Fischer Scientific, Reinach, Switzerland) according to the manufacturer's instructions. Protein disulfide bonds were reduced and cysteine residues were alkylated using 5 mM tris(2-carboxylethyl)phosphine (TCEP, Sigma-Aldrich, Buchs, Switzerland) and 10 mM iodoacetamide respectively (IAA, Sigma-Aldrich, Buchs, Switzerland). Prior to protein digestion, samples were diluted 1 to 5 with freshly prepared 50 mM ammonium bicarbonate buffer to reduce the urea concentration below 2 M. Sequencing grade modified trypsin (Promega AG, Dübendorf, Switzerland) was added at an trypsin to protein ratio of 1:50 and protein digestion was carried out overnight at 37°C with shaking at 300 rpm. After incubation, trypsin was inactivated using heat incubation in a tabletop shaker at 95°C for 5 min followed by acidification through addition of formic acid to an approximate final concentration of 1%. Insoluble parts were removed by centrifugation at 20,000 g for 10 min and the supernatant was desalted using Sep-Pak Vac C18 reversed phase columns (Waters Corporation, Baden-Dättswil, Switzerland) as described previously110and dried under vacuum. The samples were re-solubilized in 3% acetonitrile (ACN) and 0.1% formic acid (FA) to a final concentration of 0.1-1.0 mg mL-1.  Mass spectrometry analyses were performed on an Orbitrap Fusion Tribrid mass spectrometer (Thermo Fischer Scientific) equipped with a digital PicoView source (New Objective, Littleton, USA) and coupled to an M-Class ultraperformance liquid chromatography (UPLC) system using an ACN/water solvent system containing two channels with 0.1% FA (v/v) in water for channel A and 0.1% FA (v/v), 99.9% ACN (v/v) for channel B. For chromatographic separation 2 µL peptide sample (at a concentration of 0.5 µg µL-1) was loaded on a nanoEase M/Z Symmetry C18 trap column (100 A, 5 µm; 180 µm x 20 mm, Waters) followed by a nanoEase M/Z HSS T3 column (100 A, 1.8 µm; 75 µm x 250 mm, Waters). Peptide samples were separated at a flow rate of 300 nL min-1 using the following gradient: 2% to 5% B in 2 min, 25% B in 93 min, 35% B in 10 min, and 95% B in 10 min. The MS was operated in data-dependent acquisition and top-speed mode with a maximum cycle time set to 3 s. Full-scan MS spectra were acquired in the Orbitrap analyzer with a mass range of 300-1500 m/z and a resolution of 120k with an automated gain control (AGC) target value of 5x105. Precursor ions were isolated with a window of 1.6 m/z and fragmented using higher energy collisional dissociation (HCD) with a normalized collision energy of 30%. For fragmentation only precursor ions with charge states from +2 to +7 and a signal intensity of at least 5x103. Fragment ion spectra (MS/MS) were acquired in the Ion trap operated in rapid scan mode with an AGC value of 8000 using a maximum injection time of 300 ms. The dynamic exclusion was set to 25 s. Sample measurements were acquired using internal lock mass calibration on m/z 371.10124 and 445.12003.

### Data Protocol
Label-free precursor (MS1) intensity-based quantification was performed using Progenesis QI for proteomics (Nonlinear Dynamics, v.4.0). Briefly, for all samples the automatic alignment was reviewed and manually adjusted before normalization. From each Progenesis peptide ion (default sensitivity in peak picking) a maximum of the top five tandem mass spectra were exported as mascot generic file (*.mgf) using the charge deconvolution and deisotoping option and a maximum number of 200 peaks per MS/MS. The mascot generic files were searched using the Mascot Server (Matrix Science, v.2.6.2) against a decoyed and reversed protein sequence database containing the 4341 annotated proteins of Rhizobium Leaf68 (NCBI Taxon ID: 1736231 [https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=1736231]) and the 4024 annotated proteins of Sphingomonas Leaf257 (NCBI Taxon ID: 1736309 [https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=1736309]) concatenated with the Arabidopsis Information Resource (TAIR) database (release TAIR10) and 260 known mass spectrometry contaminants. Parameters for precursor ion tolerance and fragment ion tolerance were set to ± 10 ppm and ± 0.5 Da, respectively. The search parameters were as followed: trypsin digestion (two missed cleavages allowed), as fixed modification the carbamidomethylation of cysteine and as variable modification the oxidation of methionine, carbamylation of the N-terminus and lysine. The mascot search was imported into Scaffold (Proteome Software, v.4.8.9) using 5% peptide and 10% protein false discovery rate (FDR) as thresholds. The scaffold spectrum reports were imported into Progenesis QI. Normalization was performed on all precursor ions of the corresponding strain. Notably, in the mixture and artificial mixture conditions normalization was performed on all precursor ions from Leaf68 or Leaf257 respectively, i.e. to quantify the Leaf68 proteome in the mixture we only considered precursor ions specific for proteins from Leaf68 for normalization and vice versa. For protein quantification the three most abundant peptide ions (Hi-3 approach) were used for protein quantification. Only proteins with two or more unique peptides detected were considered for quantification. For statistical testing, one-way ANOVA was applied and the resulting P-values were corrected using the Benjamini-Hochberg procedure directly in Progenesis QI resulting in q-values. If not indicated otherwise the general cutoffs for significantly regulated proteins were q-value < 0.05 and Log2 fold-change (FC) >1.5 or <-1.5.

### Publication Abstract
Differences between species promote stable coexistence in a resource-limited environment. These differences can result from interspecies competition leading to character shifts, a process referred to as character displacement. While character displacement is often interpreted as a consequence of genetically fixed trait differences between species, it can also be mediated by phenotypic plasticity in response to the presence of another species. Here, we test whether phenotypic plasticity leads to a shift in proteome allocation during co-occurrence of two bacterial species from the abundant, leaf-colonizing families Sphingomonadaceae and Rhizobiaceae in their natural habitat. Upon mono-colonizing of the phyllosphere, both species exhibit specific and shared protein functions indicating a niche overlap. During co-colonization, quantitative differences in the protein repertoire of both bacterial populations occur as a result of bacterial coexistence in planta. Specifically, the Sphingomonas strain produces enzymes for the metabolization of xylan, while the Rhizobium strain reprograms its metabolism to beta-oxidation of fatty acids fueled via the glyoxylate cycle and adapts its biotin acquisition. We demonstrate the conditional relevance of cross-species facilitation by mutagenesis leading to loss of fitness in competition in planta. Our results show that dynamic character displacement and niche facilitation mediated by phenotypic plasticity can contribute to species coexistence.

### Keywords
Dynamic character displacement, Phyllosphere, Proteomics, Sphingomonas, Rhizobium

### Affiliations
ETH Zürich
Department of Biology, Institute of Microbiology, ETH Zürich, Switzerland

### Submitter
Lucas Hemmerle

### Lab Head
Dr Julia A. Vorholt
Department of Biology, Institute of Microbiology, ETH Zürich, Switzerland


