### Accession
PXD001572

### Title
Interactome analysis of PYHIN immune regulators

### Description
The human PYHIN proteins, AIM2, IFI16, IFIX, and MNDA, are regulators of immune response, transcription, apoptosis, and cell cycle. However, their protein interactions and underlying mechanisms remain largely uncharacterized. Here, we provide the interaction network for all PYHIN proteins. By designing a cell-based inducible system and integrating microscopy, immunoaffinity capture, quantitative mass spectrometry, and bioinformatics, we identified over 300 PYHIN interactions reflective of diverse functions.

### Sample Protocol
Immunoisolated protein complexes were reduced with 100 mM DTT, then digested in-solution with trypsin by a filter aided sample preparation (FASP) method. Proteins were digested with 5 ng/ml trypsin in 50 mM ABC at 37ËšC overnight. Peptides were collected and desalted with in-house packed C18 stage tips, vacuum concentrated, and acidified with 1% formic acid. Peptides were analyzed on an Ultimate 3000 nanoRSLC system coupled online with an ESI-LTQ-Orbitrap Velos ETD mass spectrometer.

### Data Protocol
RAW MS/MS data were searched by the target-decoy strategy using SEQUEST (ver. 1.20) in Proteome Discoverer (ver. 1.3) against a database that included human protein sequences (SwissProt, release 2010_11) appended with common contaminants. Result files were analyzed by Scaffold (ver 3.6) using the Peptide/ProteinProphet algorithms. Additional variable modifications of lysine acetylations, asparagine and glutamine deamidation were included in a refinement search using the X!Tandem algorithm. Protein identification thresholds were selected using probability filters to achieve a peptide and protein FDR of  less than 1%. The unweighted, spectral counts for all filtered proteins were exported for each individual PYHIN bait dataset including GFP controls. The interaction specificity was assessed using SAINT (ver. 2.3).

### Publication Abstract
The human PYHIN proteins, AIM2, IFI16, IFIX, and MNDA, are critical regulators of immune response, transcription, apoptosis, and cell cycle. However, their protein interactions and underlying mechanisms remain largely uncharacterized. Here, we provide the interaction network for all PYHIN proteins and define a function in sensing of viral DNA for the previously uncharacterized IFIX protein. By designing a cell-based inducible system and integrating microscopy, immunoaffinity capture, quantitative mass spectrometry, and bioinformatics, we identify over 300 PYHIN interactions reflective of diverse functions, including DNA damage response, transcription regulation, intracellular signaling, and antiviral response. In view of the IFIX interaction with antiviral factors, including nuclear PML bodies, we further characterize IFIX and demonstrate its function in restricting herpesvirus replication. We discover that IFIX detects viral DNA in both the nucleus and cytoplasm, binding foreign DNA via its HIN domain in a sequence-non-specific manner. Furthermore, IFIX contributes to the induction of interferon response. Our results highlight the value of integrative proteomics in deducing protein function and establish IFIX as an antiviral DNA sensor important for mounting immune responses.

### Keywords
Dna sensing, Protein interactions, Ifix

### Affiliations
Molecular Biology, Princeton University
Department of Molecular Biology, Princeton University, USA

### Submitter
Todd Greco

### Lab Head
Dr Ileana M Cristea
Department of Molecular Biology, Princeton University, USA


