### Accession
PXD018027

### Title
Epitope shaving and fungal immune evasion – cell wall proteomes

### Description
The cell wall provides a major physical interface between fungal pathogens and their mammalian host. This extracellular armour is critical for fungal cell homeostasis and survival. Yet essential cell wall moieties, such as β-1,3-glucan, are recognised as pathogen-associated molecular patterns (PAMPs) that activate immune-mediated clearance mechanisms. We have reported that the opportunistic human fungal pathogen, Candida albicans, masks β-1,3-glucan following exposure to lactate, hypoxia or iron depletion. However, the precise mechanism(s) by which C. albicans masks β-1,3-glucan have remained obscure. Here, we performed proteomic analysis of cell walls from C. albicans cells grown in hypoxia or lactate compared to glucose-grown controls to identify mechanisms driving β-1,3-glucan masking.

### Sample Protocol
Cell wall protein extracts (lyophilised, 1 mg) were resuspended in 100 µl ammonium bicarbonate (50 mM) in 1.5 ml low-binding tubes. Reduction was performed by the addition of 2 µl DTT (200 mM) with incubation for 30 min at 60 °C, then alkylation by the addition of 4 µl iodoacetamide (200 mM) with incubation in the dark for 30 min at 25 °C. Proteins were digested by the addition of 10 µl sequencing grade trypsin (20 µg/ml, Promega) with incubation overnight at 37 °C. Digested cell wall samples were centrifuged at 17,000 x g and the supernatants transferred to clean tubes. The pellets were resuspended in 50 µl 0.1% TFA (v/v), 70% acetonitrile (v/v) in ultrapure water, shaken for 10 min, centrifuged again, and the supernatants added to those already taken. All digests were frozen at -70 °C and dried by vacuum centrifugation (SpeedVac SC110A, Savant). Peptides were dissolved in 40 µl 0.1% trifluoroacetic acid (TFA), desalted on µ-C18 ZipTips (Merck Millipore, Watford, UK) and dried by vacuum centrifugation. LC-MSMS: Desalted peptide samples were dissolved in 10 µl loading solvent (0.1% formic acid (v/v), 2% acetonitrile (v/v) in ultrapure water). Liquid chromatography – tandem mass spectrometry was performed using a Q Exactive Plus/Ultimate 3000 RSLCnano system (Thermo Scientific, Hemel Hempstead, UK). Sample (5 µl) was loaded onto a trapping column (C18 PepMap 100; 300 µm i.d. x 5 mm) for 5 min at 10 µl/min and reverse-flushed to the nano column (PepMap RSLC C18; 75 µm i.d. x 25 cm) at 300 µl/min using a gradient of varying proportions of solvent A (0.1% formic acid in ultrapure water) and solvent B (0.1% formic acid, 80% acetonitrile in ultrapure water). The proportion of solvent B was increased from 3-10% during 5-10 min, from 10-40% during 10-40 min, from 40-80% during 40-45 min, held at 80% during 45-53 min, then returned to 3%. Mass spectra were acquired during 5-65 min in Full MS/data-dependent MS2 mode using a ‘Top 10’ method. MS1 scans were performed from 375 – 1750 m/z with a resolution of 70,000, an automatic gain control (AGC) target of 3e6, and maximum injection time (max IT) of 50 ms. The 10 most abundant precursor ions with charge states of +2 to +5 were selected from each MS1 scan for sequential trapping and fragmentation by higher-energy collisional dissociation (normalised collision energy 26%). MS2 scans were performed with a resolution of 17,500, an AGC target of 5e4, and max IT of 100 ms. Previously selected ions were dynamically excluded for 40 s, and peptide ions were preferred.

### Data Protocol
Raw data files from the Q Exactive Plus were processed using Proteome Discoverer v1.4 (Thermo Scientific) with Mascot v2.5 (Matrix Science, London, UK) as the search engine. The database comprised 12,421 protein sequences in the file <C_albicans_SC5314_A22_current_orf_trans_all.fasta> downloaded from the Candida Genome Database (www.candidagenome.org, datestamp: 2016-02-19). Search parameters were: Enzyme = Trypsin; Maximum missed cleavage sites = 2; Instrument = ESI-FTICR; Precursor mass tolerance = 10 ppm; Fragment mass tolerance = 20 mmu; Dynamic modifications = Oxidation (M); Static modifications = Carbamidomethyl (C). Results were filtered for peptides with a Mascot significance value of 0.05, and protein area values were based on the extracted ion chromatograms of the three most abundant peptides. Peptide-spectrum matches were validated using a decoy database search with target false discovery rates of 0.01 (strict) and 0.05 (relaxed).

### Publication Abstract
The cell wall provides a major physical interface between fungal pathogens and their mammalian host. This extracellular armor is critical for fungal cell homeostasis and survival. Fungus-specific cell wall moieties, such as &#x3b2;-1,3-glucan, are recognized as pathogen-associated molecular patterns (PAMPs) that activate immune-mediated clearance mechanisms. We have reported that the opportunistic human fungal pathogen <i>Candida albicans</i> masks &#x3b2;-1,3-glucan following exposure to lactate, hypoxia, or iron depletion. However, the precise mechanism(s) by which <i>C. albicans</i> masks &#x3b2;-1,3-glucan has remained obscure. Here, we identify a secreted exoglucanase, Xog1, that is induced in response to lactate or hypoxia. Xog1 functions downstream of the lactate-induced &#x3b2;-glucan "masking" pathway to promote &#x3b2;-1,3-glucan "shaving." Inactivation of <i>XOG1</i> blocks most but not all &#x3b2;-1,3-glucan masking in response to lactate, suggesting that other activities contribute to this phenomenon. Nevertheless, <i>XOG1</i> deletion attenuates the lactate-induced reductions in phagocytosis and cytokine stimulation normally observed for wild-type cells. We also demonstrate that the pharmacological inhibition of exoglucanases undermines &#x3b2;-glucan shaving, enhances the immune visibility of the fungus, and attenuates its virulence. Our study establishes a new mechanism underlying environmentally induced PAMP remodeling that can be manipulated pharmacologically to influence immune recognition and infection outcomes.<b>IMPORTANCE</b> The immune system plays a critical role in protecting us against potentially fatal fungal infections. However, some fungal pathogens have evolved evasion strategies that reduce the efficacy of our immune defenses. Previously, we reported that the fungal pathogen <i>Candida albicans</i> exploits specific host-derived signals (such as lactate and hypoxia) to trigger an immune evasion strategy that involves reducing the exposure of &#x3b2;-glucan at its cell surface. Here, we show that this phenomenon is mediated by the induction of a major secreted exoglucanase (Xog1) by the fungus in response to these host signals. Inactivating <i>XOG1</i>-mediated "shaving" of cell surface-exposed &#x3b2;-glucan enhances immune responses against the fungus. Furthermore, inhibiting exoglucanase activity pharmacologically attenuates <i>C. albicans</i> virulence. In addition to revealing the mechanism underlying a key immune evasion strategy in a major fungal pathogen of humans, our work highlights the potential therapeutic value of drugs that block fungal immune evasion.

### Keywords
Candida albicans, Lc-msms, Cell wall

### Affiliations
University of Aberdeen
MRC Centre for Medical Mycology  University of Exeter, Geoffrey Pope Building,  Stocker Road, Exeter EX4 4QD, UK

### Submitter
David Stead

### Lab Head
Dr Professor Alistair J P Brown
MRC Centre for Medical Mycology  University of Exeter, Geoffrey Pope Building,  Stocker Road, Exeter EX4 4QD, UK


