### Accession
PXD016805

### Title
Comparison of the Human Neuronal Proteome upon Japanese Encephalitis or West Nile Virus Infection and potential role of mosquito saliva in neuropathogenesis

### Description
Neurotropic flavivirus Japanese encephalitis virus (JEV) and West Nile virus (WNV) are amongst the leading causes of encephalitis. Using label-free quantitative proteomics, we identified proteins differentially expressed upon JEV (gp-3, RP9) or WNV (IS98) infection of human neuroblastoma cells. Both viruses were associated with the up-regulation of immune response (IFIT1/3/5, ISG15, OAS, STAT1, IRF9) and the down-regulation of SSBP2, involved in gene expression, as well as PAM, involved in neuropeptide amidation. Proteins associated to membranes, involved in extracellular matrix organization and collagen metabolism represented major clusters down-regulated by JEV and WNV. Moreover, transcription regulation and mRNA processing clusters were also heavily regulated by both neurotropic flaviviruses. If the proteome of neuroblastoma cells infected by JEV or WNV was significantly modulated in the presence of mosquito saliva, both viruses showed distinct patterns. Mosquito saliva favored the modulation of proteins associated with gene regulation in JEV infected neuroblastoma cells while it was the modulation of proteins associated with protein maturation, signal transduction and ion transporters in the case of WNV infected neuroblastoma cells.

### Sample Protocol
Human neuroblastoma SK-N-SH cells were maintained at 37°C in DMEM with 10% FBS. Cells were seeded in 6-well plates with 5.105 cells/well (3 wells/condition) or 24-well plates with 2.105 cells/well in the presence of glass slides for 24h and infected with JEV genotype 3 strain RP-9 {deWispelaere:2017hb} or WNV strain IS98 {Alsaleh:2016en} at a MOI of 1 in 1 mL of DMEM with 2% FBS, in presence or not of SGE (2 µL, equivalent of 2 SG). After 48h, the 24-well plates were fixed with 3% paraformaldehyde for 20 min and stored at 4°C until use. In parallel, the supernatant was removed from the 6-well plates and cells were washed and scrapped in 1 mL of PBS. After centrifugation at 5000g, the cells were resuspended in a label free buffer (urea 8M, Tris 50 mM pH7.5). Whole cell protein extract was obtained by sonication as described for the SGE. The supernatant was transferred to clean tubes, the protein concentration was evaluated with a Nanodrop (1.63 mg/mL in SK-N-SH cells and 2 mg/ml in SK-N-SH cells+SGE) and the samples were stored at -80°C. Each experiment was performed in triplicate. Protein in solution were solubilized in denaturation buffer (Tris 100mM pH8.0, urea 8M). Reduction of disulfide bridges was performed in 5 mM dithiothreitol (DTT) for 30 min at room temperature; alkylation was performed in 20 mM iodocacetamide in the dark for 30 min at room temperature. Each sample was first digested by 500 ng of LysC (Promega, Madison, WI, USA) at 30°C for 3 hours. Solution was diluted in 100 mM ammonium bicarbonate (BA), until urea concentration was below 1.5 M. Each sample was then digested with 500 ng of trypsin (Promega, Madison, WI, USA) at 37°C overnight.  Digestion was stop by adding 1% formic acid (FA). Resulting peptides were purified using SPE C18 strategy and concentrated to almost dryness in 50% ACN 0.1% FA with a speed vac. Briefly, C18 phase (Sep-Pak, Waters) was activated in methanol, rinsed once in 80% acetonitrile (ACN) 0.1% FA, washed thrice in 0.1% FA.. Resin was washed three times in 0.1% FA, once in 2% ACN 0.1% FA. Peptides were eluted in 50% ACN 0.1% FA. Peptides in elution buffer were concentrated to almost dryness. Digested peptides were analyzed by nano LC-MS/MS using an EASY-nLC 1000 (Thermo Fisher Scientific) coupled to a Q Exactive Orbitrap mass spectrometer. About 1 µg of each sample (dissolved in 0.1% FA) were loaded and separated at 250 nl.min-1 on a home-made C18 50 cm capillary column picotip silica emitter tip (75 μm diameter filled with 1.9 μm Reprosil-Pur Basic C18-HD resin, (Dr. Maisch GmbH, Ammerbuch-Entringen, Germany)) equilibrated in solvent A (0.1 % FA). The peptides were eluted using a two slopes gradient of solvent B (0.1 % FA in ACN) from 2% to 27% in 100 min and to 27% to 60% in 50 min at 250 nL/min flow rate (total length of the chromatographic run was 180 min). The Q Exactive (Thermo Fisher Scientific, Bremen) was operated in data-dependent acquisition mode with the XCalibur software 2.2 (Thermo Fisher Scientific, Bremen). Survey scan MS were acquired in the Orbitrap on the 300 – 1700 m/z range with the resolution set to a value of 70 000 at m/z = 400 in profile mode (AGC target at 1E6). The 5 most intense ions per survey scan were selected for HCD fragmentation (NCE 27), and the resulting fragments were analyzed in the Orbitrap at 17500 of resolution (m/z 400). Isolation of parent ion was fixed at 1.6 m/z and underfill ratio at 1%. Dynamic exclusion was employed within 45 sec

### Data Protocol
Data were searched with the Andromeda search engine using MaxQuant (1.4.1.2 version) against the Human database from SwissProt and TrEMBL (2014.01.14, 88500 entries including 39715 from SwissProt), structural polyproteins of JEV and WNV, and the Culex pipiens database (2014.12.12, 130 entries, from Uniprot).  The following search parameters were applied: Carbamidomethylation of cysteines was set as a fixed modification. Oxidation of methionine and protein N-terminal acetylation were set as variable modifications. The mass tolerances in MS and MS/MS were set to 5 ppm for each, respectively. Maximum peptide charge was set to 7 and 5 amino acids were required as minimum peptide length. Two peptides were required for protein identification and quantitation. Peptides and proteins identified with an FDR lower than 0.1% were considered as valid identification.  Label free analysis was done by using the 'match between run' feature of MaxQuant (3 min time window). LFQ data were used to performed statistical analysis between conditions of infection.   Statistical analysis of Label Free MS data Statistical analysis of label free data was performed using the MS-Stat package on R environment Peptides dataset from MaxQuant analysis were used to perform the statistical analysis.  Proteins were declared significant with fold change higher than 1.5, and adjusted p-value below 5% of error

### Publication Abstract
Neurotropic flavivirus Japanese encephalitis virus (JEV) and West Nile virus (WNV) are amongst the leading causes of encephalitis. Using label-free quantitative proteomics, we identified proteins differentially expressed upon JEV (gp-3, RP9) or WNV (IS98) infection of human neuroblastoma cells. Data are available via ProteomeXchange with identifier PXD016805. Both viruses were associated with the up-regulation of immune response (IFIT1/3/5, ISG15, OAS, STAT1, IRF9) and the down-regulation of SSBP2 and PAM, involved in gene expression and in neuropeptide amidation respectively. Proteins associated to membranes, involved in extracellular matrix organization and collagen metabolism represented major clusters down-regulated by JEV and WNV. Moreover, transcription regulation and mRNA processing clusters were also heavily regulated by both viruses. The proteome of neuroblastoma cells infected by JEV or WNV was significantly modulated in the presence of mosquito saliva, but distinct patterns were associated to each virus. Mosquito saliva favored modulation of proteins associated with gene regulation in JEV infected neuroblastoma cells while modulation of proteins associated with protein maturation, signal transduction and ion transporters was found in WNV infected neuroblastoma cells.

### Keywords
Mosquito, West nile, Label free, Japanese encephalitis

### Affiliations
Pasteur
Institut Pasteur

### Submitter
MARIETTE MATONDO 

### Lab Head
Dr Benoit Besson
Institut Pasteur


