### Accession
PXD016840

### Title
Bioplatforms Australia: Antibiotic Resistant Sepsis Pathogens Framework Initiative -   Escherichia coli

### Description
The Antibiotic Resistant Sepsis Pathogens Framework Initiative aims to develop a framework dataset of 5 sepsis pathogens (5 strains each) using an integrated application of genomic, transcriptomic, metabolomic and proteomic technologies. The pathogens included in this initiative are: Escherichia coli, Klebsiella pneumoniae complex, Staphylococcus aureus, Streptococcus pyogenes, and Streptococcus pneumoniae.  This submission pertains to Escherichia coli strains B36, MS14384, MS14385, MS14386 and MS14387.

### Sample Protocol
Complete information about the bacterial cultures is available from the Bioplatforms Australia Antibiotic Resistant Sepsis Pathogens Initiative website. https://data.bioplatforms.com/organization/pages/bpa-sepsis/methods Part 1: Workflow for preparing bacterial cultures for omics analyses  (Andre Mu) Dilution used 1:1; Day 1: Streak isolates from glycerol stock to agar plates; Incubate at 37C overnight. Day 2: Set up and label consumables; Prepare overnight culture; 1:1 dilution; Grow shaking (200 rpm) at 37C overnight; Day 3: Thaw pooled human sera; 1:1 dilution; Transfer culture into falcon tubes for the following protocols at their respective volume: Proteomics – 10ml; Follow protocol Proteomic cell pellet wash protocol;  Proteomic cell pellet wash protocol 1. Transfer 10ml culture into Falcon tube 2. Centrifuge at 50 000 g at 0oC for 5 mins 3. Remove supernatant and resuspend pellet in 1ml 1x PBS 4. Transfer to 1.5ml Eppendorf tube 5. Centrifuge at 21 000 g for 8 mins 6. Remove Supernatant and resuspend 1ml 1x PBS 7. Repeats steps 5 and 6 8. Repeat step 5, then remove all the supernatant 9. Flash freeze in liquid nitrogen for 1 minute before storing at -80oC Part 2: Proteomics sample preparation Gram negative protocol (Escherichia coli and Klebsiella pneumoniae) 1. Freeze-dried bacteria were resuspended in 600µL of bead-beating buffer (50mM Tris-HCl, 145mM NaCl) containing 400µL of 0.1mm acid-washed glass beads. 2. These were then subjected to bead beating (4 cycles of 1 min, 5m/s) with 1 minute rest on ice between cycles. 3. Cellular debris was removed by centrifugation at 20,000 x g for 30 m at 4oC. 4. 250 μL of protein sample was mixed with ice cold water : methanol : chloroform in a ratio of 3 : 4 : 1 to precipitate proteins. 5. Precipitate was resuspended in 6 M urea / 2 M thiourea, and protein quantitation was performed using a QubitTM kit (Invitrogen, Carlsbad CA). 6. The solutions were reduced for 1 h with 10 mM dithiothreitol (DTT) followed by alkylation using 20 mM iodoacetamide for 1 h in the dark. 7. Samples were diluted to 1mL in 100mM TEAB, pH adjusted to 7.5 and digested with 1/25 (w/w) of porcine sequencing grade trypsin (Promega, Madison WI) for 24 h at 37oC. 8. Formic acid was then added to a final concentration of 2% (v/v) to precipitate lipids, and centrifuged at 14, 000 RPM at 25oC for 15 m to pellet lipids. 9. The supernatant was removed and subjected to purification using Oasis® HLB columns (Waters Corporation, Milford MA). 10. Cartridges were prepared by washing in succession: 1 mL 100% methanol, 1 mL 100% acetonitrile (MeCN), 1 mL 70% (v/v) MeCN / 0.1% (v/v) trifluoroaceitc acid (TFA). 11. Samples were washed through cartridges 5 times and finally washed with 5 mL 0.1% TFA. 12. Peptides were eluted with 50% MeCN / 0.1% TFA, and lypophilised by vacuum centrifugation. 13. The peptides were split into two aliquots. One aliquot was shipped to Monash Biomedical Proteomics Facility (MBPF) for MS1 quantification and spectral library generation. The second aliquot was shipped to Australian Proteome Analysis Facility (APAF) for MS2 quantification (SWATH). Part 3: Sample preparation (SWATH)  1. Take all 60 samples out from freezer. 2. Added required amount of loading buffer to each sample so that the concentration is 1 µg/µL. 3. Spun at 10,000 ×g for 5 min.  4. Transfer 2 µL of each sample for each group (same strain and same medium condition, six samples/group) + 48 µL of loading buffer to HPLC vials for 1D-IDA analysis.  5. Transfer 2 µL/sample + 8 µL of loading buffer to HPLC vials for SWATH analysis.  6. Keep all the tubes back in freezer.

### Data Protocol
1D IDA MS/MS data were acquired before SWATH data acquisitions for each strain/medium samples. 10 µL was injected onto a peptide trap for pre-concentration and desalted with 0.1% formic acid, 2% ACN, at 5 µL/min for 3 minutes. The peptide trap was then switched into line with the analytical column. Peptides were eluted from the column using linear solvent gradients, with steps, from mobile phase A: mobile phase B (98:2) to mobile phase A: mobile phase B (90:10) for 10 min, then to (65:35) for further 78 minutes at 600 nL/min. After peptide elution, the column was cleaned with 95% buffer B for 10 minutes and then equilibrated with 98% buffer A for 20 minutes before next sample injection. The reverse phase nanoLC eluent was subject to positive ion nanoflow electrospray analysis in an information dependant acquisition mode (IDA). In the IDA mode a TOFMS survey scan was acquired (m/z 350 - 1500, 0.25 second), with the 20 most intense multiply charged ions (2+ - 5+; counts >200) in the survey scan sequentially subjected to MS/MS analysis. MS/MS spectra were accumulated for 100 milliseconds in the mass range m/z 100 – 1500 with rolling collision energy. 10 µL was injected onto a peptide trap for pre-concentration and desalted with 0.1% formic acid, 2% ACN, at 5µL/min for 3 minutes. The peptide trap was then switched into line with the analytical column. Peptides were eluted from the column using linear solvent gradients, with steps, from mobile phase A: mobile phase B (98:2) to mobile phase A: mobile phase B (90:10) for 10 min, then to (65:35) for further 78 minutes at 600 nL/min. After peptide elution, the column was cleaned with 95% buffer B for 10 minutes and then equilibrated with 98% buffer A for 20 minutes before next sample injection. The reverse phase nanoLC eluent was subject to positive ion nanoflow electrospray analysis in a data independent acquisition mode (SWATH). In SWATH mode, first a TOFMS survey scan was acquired (m/z 350-1500, 0.05 sec) then the 100 predefined m/z ranges were sequentially subjected to MS/MS analysis. MS/MS spectra were accumulated for 30 milliseconds in the mass range m/z 350-1500 with rolling collision energy optimised for lowed m/z in m/z window +10%. All the raw data files for each strain generated by 1D-IDA-MS analysis of the sample were searched with ProteinPilot (v5.0) (Sciex) using the Paragon algorithm in thorough mode. The genomic sequences were submitted to ENA database as study PRJEB29930, and the protein databases are in NCBI GenBank under the accessions listed below. Strain B36: NZ_LR130545.1; NZ_LR130546.1; NZ_LR130547.1 Strain MS14384: NZ_LR130562.1; NZ_LR130563.1 Strain MS14385: NZ_LR130555.1; NZ_LR130556.1; NZ_LR130557.1; NZ_LR130558.1; NZ_LR1305559.1; NZ_LR130560.1; NZ_LR130561.1 MS14386: NZ_LR130552.1; NZ_LR130553.1; NZ_LR130554.1 Strain MS14387: NZ_LR130564.1; NZ_LR130565.1; NZ_LR130566.1 The search generated one group file for each strain and each medium. Each group file was imported into PeakView (v2.1) (AB Sciex) and exported as a SWATH library. For each strain, the two medium libraries were merged into one strain-specific local seed library using SwathXtend(v2.4). Peptides with retention time greater than 96 minutes were removed.   APAF strain-specific local libraries were merged with 2D libraries generated by MBPF to generate extended strain-specific SWATH libraries.  Strain and medium specific MBPF 2D libraries were downloaded in Spectronaut format. Each of these 2D libraries was preprocessed and converted into PeakView library format before merging with APAF local libraries. Modifications were consolidated with PeakView format; fragment ions with ExcludeFromAssay==TRUE were removed; inversed value of IsProteotypic was set as the shared column value; confidence was set as 1; mods values were extracted from ModifiedPeptide. The 10 preprocessed MBPF libraries were saved as PeakView library format.  Match quality between local seed library and the two medium 2D MBPF libraries were checked for each strain. One extended library was created for each strain by merging the APAF local seed library and the two preprocessed MBFP 2D libraries using SwathXtend (v2.4). Human and contamination proteins were removed from each extended library to form the final extended strain-specific SWATH library and subject to be used for SWATH quantification. SWATH data for each strain two media were extracted using PeakView (v2.1) with the following parameters: Top 6 most intense fragments of each peptide were extracted from the SWATH data sets (75 ppm mass tolerance, 10 min retention time window). Shared and modified peptides were excluded. After data processing, peptides (max 100 peptides per protein) with confidence 99% and FDR 1% (based on chromatographic feature after fragment extraction) were used for quantitation.

### Publication Abstract
None

### Keywords
Escherichia coli, Sera, Proteomics, Antibiotic resistance, Swath, Sepsis, Rpmi

### Affiliations
Bioplatforms Australia
Australian Proteome Analysis Facility

### Submitter
Dana Pascovici

### Lab Head
Dr Mabel Lum, Bioplatforms Australia (project manager)
Bioplatforms Australia


