### Accession
PXD020079

### Title
CIITA-transduced glioblastoma cells uncover a rich repertoire of clinically relevant tumor-associated HLA-II antigens

### Description
CD4+ T cell responses are crucial for inducing and maintaining effective anti-cancer immunity, and the identification of HLA-II cancer-specific epitopes is key to the development of potent cancer immunotherapies. In many tumor types, and especially in glioblastoma (GBM), HLA-II complexes are hardly ever naturally expressed. Hence, little is known about immunogenic HLA-II epitopes in GBM. With stable expression of CIITA coupled to a detailed and sensitive mass spectrometry based immunopeptidomics analysis, we here uncovered a remarkable breadth of the HLA-ligandome in HROG02, HROG17 and RA GBM cell lines. The effect of CIITA expression on the induction of the HLA-II presentation machinery was striking in each of the three cell lines, and it was significantly higher compared to IFNɣ treatment. In total, we identified 16,123 unique HLA-I peptides and 32,690 unique HLA-II peptides. In order to genuinely define the identified peptides as true HLA ligands, we carefully characterized their association with the different HLA allotypes. In addition, we identified 138 and 279 HLA-I and HLA-II ligands, respectively, most of which are novel in GBM, derived from known GBM-associated tumor-antigens that have been used as source proteins for a variety of GBM vaccines. Our data further indicate that CIITA-expressing GBM cells acquired an antigen presenting cell-like phenotype as we found that they directly present external proteins as HLA-II ligands. Not only that CIITA-expressing GBM cells are attractive models for antigen discovery endeavors, but such engineered cells have great therapeutic potential through massive presentation of a diverse antigenic repertoire.

### Sample Protocol
Sample preparation for shotgun proteomics Cell pellets of approximately one million cells were resuspended in 8 M Urea (and 50 mM ammonium bicarbonate buffer at pH 8. The lysates were sonicated in the Bioruptor instrument for 15 cycles of 30 s at maximum mA at 4 °C. After centrifugation at 20,000 g at 4 °C for 30 min, the soluble proteins were collected and the protein concentration was determined by a Bradford protein assay. Proteins were reduced with a final concentration of 5 mM DTT at 37 °C for 60 min, and alkylated for 60 min in the dark at RT with a final concentration of 15 mM iodoacetamide. The digestion was carried out with a mixture of endoproteinase Lys-C and Trypsin. After digestion, the samples were acidified with formic acid (FA) and desalted on C18 spin columns.   Immunoaffinity purification of HLA class I and HLA class II complexes and peptide extraction For HLA-I and -II immunoaffinity purification we used a previously described protocol from our group (39, 40). We used the 96-well single-use micro-plate with glass fiber and 10 µm polypropylene membranes. Cell lysis was performed at 4°C for 1h with PBS buffer solution containing 0.25% sodium deoxycholate 0.2 mM iodoacetamide, 1 mM EDTA, 1:200 Protease Inhibitors Mixture, 1 mM Phenylmethylsulfonylfluoride, 1% octyl-beta-D glucopyranoside. Lysates were cleared by centrifugation at 4°C at 14,200 rpm for 30min with a table-top centrifuge. Anti-pan HLA-I (HB95) and HLA-II (HB245) antibodies cross-linked to protein-A sepharose 4B beads were loaded on their respective plates at a final bead volume of 100 µl. The lysates were loaded first through the HLA-I affinity plate and then through the HLA-II affinity plate by gravity at 4°C. Using the Waters Positive Pressure-96 Processor, each plate was washed four times with 2 ml of 150 mM sodium chloride (NaCl) (Carlo-Erba, Val de Reuil, France) in 20 mM Tris-HCl pH 8, four times with 2 ml of 400 mM NaCl in 20 mM Tris-HCl pH 8, four times with 2 ml of 150 mM NaCl in 20 mM Tris-HCl pH 8, and finally, twice with 2 ml of 20 mM Tris-HCl pH 8. Two Sep-Pak tC18 100 mg Sorbent 96-well plates were required for the purification and concentration of HLA-I and HLA-II peptides. Each Sep-Pak tC18 plate was handled separately and equilibrated with 80% acetonitrile (ACN) in 0.1% TFA and washed with 0.1% TFA. Each affinity plate was stacked on top of a Sep-Pak tC18 plate and the HLA complexes were eluted with 500 µl of 1% TFA. The Sep-Pak tC18 wells were washed with 2 ml of 0.1% TFA. HLA peptides were eluted with 500 µl of 32% ACN in 0.1% TFA. Recovered HLA-I and -II peptide samples were transferred separately into Eppendorf tubes, dried using vacuum centrifugation and stored at 20 °C.   MS analyses  Samples were acquired using the UHPLC Easy-nLC 1200 coupled online to a Q Exactive HF or a HF-X Orbitrap mass spectrometers with a nanoelectrospray ion source. We packed the uncoated PicoTip 8 µm tip opening with 75 µm i.d. and 450 mm long analytical columns with ReproSil-Pur C18 that were kept at 50 °C using a column oven. HLA-I and HLA-II peptide samples were re-suspended in 12 ul of 0.1% FA/ 2% ACN and 3 ul of each sample were placed in the UHPLC autosampler. Peptides were then separated using a linear gradient of solvent A (0.1%FA) and B (0.1% FA/85% B). HLA-I peptides were separated from 2 -35% B during a total elution of 120 min. HLA-II peptides were separated from 2 – 45% B during a total elution of 90 min. Data was acquired with data-dependent “top10” method, which isolates within a 1.2 m/z window the ten most abundant precursor ions and fragments them by higher-energy collision dissociation (HCD) at normalized collision energy of 27%. For MS1 scan, the mass spectrometer was operated to a scan range from 300-1650 m/z with a resolution of 60,000 (200 m/z), a maximum injection time of 20 ms (80 ms for HF-X) and  AGC target value of 3e6 ions. For MS/MS, fragment ions were acquired at a resultion of 15,000 (200 m/z) with a maximum injection of 120 ms and AGC target value of 1e5 (2e5 for HF-X). The peptide match option was disabled. The dynamic exclusion of precursor ions from further selection was set for 20 s. For shotgun proteomics, tryptic peptides were separated through a 270 min gradient and data-dependent “top15” method was used which isolates within a 1.4 m/z window the 15 most abundant precursor ions and fragments them by higher-energy collision dissociation (HCD) at normalized collision energy of 27%. MS1 scan, the mass spectrometer was operated to a scan range from 300-1650 m/z with a resolution of 60,000 (200 m/z), a maximum injection time of 20 ms (80 ms for HF-X) and  AGC target value of 3e6 ions.For MS/MS, AGC target values of 1e5 were used with a maximum injection time 25 ms (80ms for HF-X) at resolution of 15,000 (200 m/z). The dynamic exclusion of precursor ions from further selection was set for 30.

### Data Protocol
Identification of peptides and proteins  We employed the MaxQuant computational platform version 1.6.10.43 (41) to search the peak lists against the UniProt databases (Human 20,362 entries, May 2020) and a file containing 246 frequently observed contaminants. For proteomics, the default settings were used, were Methionine oxidation and Protein N-term acetylation were set as a variable modifications and a fixed modification of cysteine carbamidomethylation was set. The second peptide identification option in Andromeda was enabled. A peptide spectrum match (PSM) false discovery rate (FDR) of 0.01 and protein FDR of 0.01 were set. The initial allowed mass deviation of the precursor ion was set to 6 ppm and the maximum fragment mass deviation was set to 20 ppm. ‘Match between runs’ and ‘label-free quantification’ (LFQ) modules were enabled with the default parameters (42). For immunopeptidomics, the default settings were used except the following parameters: enzyme specificity was set to ‘unspecific’ and no fixed modification was set, PSM FDR was set to 0.01 with no protein FDR, and ‘match between runs’ was enabled separately for each cell line, and for HLA-I and HLA-II peptide samples separately.  For neoantigen identification in proteomics and immunopeptidomics data we used MaxQuant version 1.5.9.4 with similar settings as above, except that variants were searched from the personalized fasta file (mentioned above).  Data visualization and statistical analyses were performed using the Perseus computational platform version 1.6.6.0 (43). For proteomics datasets, we used LFQ values from ‘ProteinGroups’ MaxQuant output table of the search of tryptic pepitdes. Reverse, contaminants and proteins only identified by site were filtered, Intensities were log2 transformed. The table was filtered to include proteins with three valid LFQ intensity values in at least one group per cell line and condition (Supplementary Table 2). Missing values were imputed with standard parameters and values were z-scored for visualization purposes. Volcano plots summarizing two-sided Student’s t test results were generated, and points outside the lines indicate significantly upregulated or downregulated proteins (permutation-based FDR=1%. S0=1). Annotations were imported from KEGG, and 2D annotation enrichment module in Perseus was used on Student’s t-test difference with Benjamin-Hochberg FDR of 0.02 FDR. For immunopeptidomics datasets, we used the ‘Peptides’ output table. Peptides matching to reverse and contaminants sequences were removed and intensities were log2 transformed. HLA peptides of 8-12 a.a and of 12-22 a.a were retained for HLA-I and HLA-II immunopeptidomic datasets, respectively (Supplementary Tables 3,4).

### Publication Abstract
None

### Keywords
Immunopeptidomics, Antigen presentation, Ciita, Glioblastoma

### Affiliations
UNIL/CHUV
Center of experimental therapeutics Department of oncology UNIL CHUV Ludwig Institute for Cancer Research Lausanne Centre de recherche Agora Rue du Bugnon 25A CH-1005 Lausanne

### Submitter
Michal Bassani-Sternberg

### Lab Head
Dr Michal Bassani-Sternberg
Center of experimental therapeutics Department of oncology UNIL CHUV Ludwig Institute for Cancer Research Lausanne Centre de recherche Agora Rue du Bugnon 25A CH-1005 Lausanne


