### Accession
PXD008022

### Title
Glioma proteome organization under oncolytic adenoviral infection

### Description
Glioblastoma multiforme (GBM) is the most common and aggressive type of malignant glioma. Oncolytic adenoviruses are being modified to exploit the aberrant expression of proteins in tumor cells to enhance tumor tropism and glioma-selective replication. E1A mutant adenovirus Delta-24-RGD has shown favorable toxicity profile and remarkable efficacy in a first-in-human phase I clinical trial. However, the comprehensive modulation of glioma metabolism in response to Delta-24-RGD infection is poorly understood. Integrating mass spectrometry based-quantitative proteomics, physical and functional interaction data, and biochemical approaches, we conducted a cell-wide study of cytosolic, nuclear, and secreted glioma proteomes throughout the early time course of Delta-24-RGD infection.

### Sample Protocol
Cytosolic proteome analysis. A shotgun comparative proteomic analysis of cytosolic fractions using iTRAQ (isobaric Tags for Relative and Absolute Quantitation) was performed. Gobal experiments were carried out with two/three biological replicates in each experimental condition. Cytosolic extracts (300 μg) were precipitated with methanol/choloroform, and pellets dissolved in 20µl 0.5M TEAB, 6M urea. Protein quantitation was performed with the Bradford assay kit (Bio-Rad). iTRAQ labeling of each sample was performed according to the manufacturer's protocol (Sciex). Briefly, a total of 80 μg of protein from each cellular condition was reduced with 50 mM tris (2-carboxyethyl) phosphine (TCEP) at room temperature for 1 h, and cysteine residues were alkylated with 200 mM methylmethanethiosulfonate (MMTS) at room temperature for 10min. Protein enzymatic cleavage was carried out with trypsin (Promega; 1:50, w/w) at 37 °C for 16 h. Each tryptic digest was labelled according to the manufacturer's instructions with one isobaric amine-reactive tags as follows: Tag113, Mock-infected U87 cells-1; Tag114, Mock-infected U87 cells-2; Tag115, U87-infected cells (6hpi)-1; Tag116, U87-infected cells (6hpi)-2; Tag117, U87-infected cells (6hpi)-3;Tag118, U87-infected cells (10hpi)-1; Tag119, U87-infected cells (10hpi)-2; Tag121, U87-infected cells (10hpi)-3. After 2h incubation, the set of labelled samples were pooled and evaporated in a vacuum centrifuge.  To increase the proteome coverage, the peptide pool was fractionated by SCX chromatography. Briefly the sample was first dissolved in 400µl 10mM KH2PO4, 20% ACN, pH:3, sonicated and centrifuged for 3 minutes at 18000xg at RT. The pellet was discarded and the pH ˂ 3 was adjusted with ACN. Peptides were eluted with KCl at an increasing gradient from 1mM to 500mM. 12 different fractions were collected. Purification and concentration of peptides was performed using C18 Zip Tip Solid Phase Extraction (Millipore). Then the sample was evaporated under vacuum and reconstituted into 10μl of 2% acetonitrile, 0.1% formic acid, 98% MilliQ-H20 prior to mass spectrometric analysis.

### Data Protocol
Data analysis for cytosolic proteomes. Relative quantification and protein identification were performed with the ProteinPilot™ software (version 5.0; Sciex) using the Paragon™ algorithm as the search engine. The search parameters allowed for cysteine modification by MMTS and biological modifications programm in the algorithm (i.e. phosphorylations, amidations, semitryptic fragments, etc.). Reporter ion intensities were bias corrected for the overlapping isotope contributions from the iTRAQ tags according to the certificate of analysis provided by the reagent manufacturer (Sciex). The peptide and protein selection criteria for relative quantitation were performed as follows. Only peptides unique for a given protein were considered for relative quantitation, excluding those common to other isoforms or proteins of the same family. Proteins were identified on the basis of having at least one peptide with an ion score above 99% confidence. Among the identified peptides, some of them were excluded from the quantitative analysis for one of the following reasons: (i) The peaks corresponding to the iTRAQ labels were not detected; (ii) the peptides were identified with low identification confidence (<1.0%); (iii) the sum of the signal-to-noise ratio for all of the peak pairs was <6 for the peptide ratios. The protein sequence coverage (95% conf.) was estimated for specific proteins by the percentage of matching amino acids from the identified peptides having confidence greater than or equal to 95% divided by the total number of amino acids in the sequence. Several quantitative estimates provided for each protein by ProteinPilot were utilized: the fold change ratios of differential expression between labelled protein extracts; the p-value, representing the probability that the observed ratio is different than 1 by chance. A decoy database search strategy was also used to estimate the false discovery rate (FDR), defined as the percentage of decoy proteins identified against the total protein identification. The FDR was calculated by searching the spectra against the decoy database generated from the target database. The results were then exported into Excel for manual data interpretation. Although relative quantification and statistical analysis were provided by the ProteinPilot software, an additional 1.3-fold change cutoff for all iTRAQ ratios (ratio <0.77 or >1.3) and a p-value lower tan 0.05 were selected to classify proteins as up- or down-regulated (at least in two of three biological replicates). Proteins with iTRAQ ratios below the low range (0.77) were considered to be underexpressed, whereas those above the high range (1.3) were considered to be overexpressed.

### Publication Abstract
Glioblastoma multiforme (GBM) is the most common and aggressive type of malignant glioma. Oncolytic adenoviruses are being modified to exploit the aberrant expression of proteins in tumor cells to increase the antiglioma efficacy. E1A mutant adenovirus Delta-24-RGD (DNX-2401) has shown a favorable toxicity profile and remarkable efficacy in a first-in-human phase I clinical trial. However, the comprehensive modulation of glioma metabolism in response to Delta-24-RGD infection is poorly understood. Integrating mass spectrometry based-quantitative proteomics, physical and functional interaction data, and biochemical approaches, we conducted a cell-wide study of cytosolic, nuclear, and secreted glioma proteomes throughout the early time course of Delta-24-RGD infection. In addition to the severe proteostasis impairment detected during the first hours post-infection (hpi), Delta-24-RGD induces a transient inhibition of signal transducer and activator of transcription 3 (STAT3), and transcription factor AP-1 (c-JUN) between 3 and 10hpi, increasing the nuclear factor kappa B (NF-&#x3ba;B) activity at 6hpi. Furthermore, Delta-24-RGD specifically modulates the activation dynamics of protein kinase C (PKC), extracellular signal-regulated kinase 1/2 (ERK1/2), and p38 mitogen-activated protein kinase (p38 MAPK) pathways early in infection. At extracellular level, Delta-24-RGD triggers a time -dependent dynamic production of multitasking cytokines, and chemotactic factors, suggesting potential pleiotropic effects on the immune system reactivation. Taken together, these data help us to understand the mechanisms used by Delta-24-RGD to exploit glioma proteome organization. Further mining of this proteomic resource may enable design and engineering complementary adenoviral based-vectors to increase the specificity and potency against glioma.

### Keywords
Glioma, Adenovirus, Proteomics

### Affiliations
Proteomics unit
neuroproteomics Lab Navarrabiomed-Proteored

### Submitter
Enrique SantamarÃ­a

### Lab Head
Dr enrique santamaria
neuroproteomics Lab Navarrabiomed-Proteored


