### Accession
PXD008623

### Title
Comparative proteomic analysis revealed complex responses to classical/novel duck reovirus infections in Cairna moschata

### Description
Duck reovirus (DRV) is an typical aquatic bird pathogen belonging to the Orthoreovirus genus of the Reoviridae family. Reovirus causes a fatal disease resulting to huge economic losses to the duck industry. Although DRV has been identified and isolated long ago, the responses of Cairna moschata to classical/novel duck reovirus (CDRV/NDRV) infections are largely unknown. Using an integrated approach involving TMT labeling and LC-MS/MS analysis, proteomes of C. moschata liver cells under C/NDRV infections were analyzed. In total, 5571 proteins were identified, among which 5015 proteins were quantified. The differential expressed proteins (DEPs) between the control and infected liver cells displayed various biological functions and diverse subcellular localizations. Among the DEPs, most of the metabolism-related proteins were down-regulated, suggesting a decrease in the basal metabolisms under C/NDRV infections. Several important factors in the complement, coagulation and fibrinolytic systems were significantly up-regulated by C/NDRV infections, indicating that the serine protease-mediated innate immune system may play roles in the responses to C/NDRV infections. Moreover, a number of molecular chaperones were identified, and no significantly changes in their abundances were observed in the liver cells. Our data may give a comprehensive resource for investigating the regulation mechanism involved in the responses to C/NDRV infections in C. moschata.

### Sample Protocol
Experimental birds and virus   The experimental infection was conducted in 1-day-old healthy Muscovy ducklings, and the blood samples of each duckling were checked by RT-PCR to ensure their healthy (no DRV has been detected). Twenty seven 1-day-old Muscovy ducklings were randomly divided into three groups and were housed in the isolation facility. For CDRV infection, one group (9 ducklings) was inoculated intramuscular with 0.5 mL of infected cell culture medium containing 104 infective dose of CDRV ZJ2000M. For NDRV infection, another group (9 ducklings) was inoculated intramuscular with 0.5 mL of infected cell culture medium containing 105 infective dose of NDRV HN10. Additionally, nine un-infected ducklings were with sterile DMEM in the same manner. Ducklings of each groups were euthanatized on 72 hours post infection (hpi), the liver was collected from each bird, and was snap-frozen in liquid nitrogen. Subsequently, frozen tissues were kept at − 80 °C until further processing.  Protein extraction and trypsin digestion   For each treatment group, nine ducklings were divided into three independent subgroups (three ducklings as a repeat). Samples were first grinded by liquid N2 and then were transferred into a 5 mL tube. The cell powder was sonicated using a high intensity ultrasonic processor in pre-cooled lysis buffer containing 8 M urea, 2 mM ethylenediaminetetraacetic acid, 10 mM dithiothreitol and 1% Protease Inhibitor Cocktail VI. The remaining debris was discarded by centrifugation at 20,000 × g for 10 min at 4°C. At least, the samples were precipitated with pre-cooled 15% TCA buffer for 2 h, and the supernatant was removed by centrifugation at 20,000 × g for 10 min at 4°C. Again, the remaining precipitate was washed for three times with pre-cooled acetone and was redissolved in buffer containing 8 M urea, 100 mM TEAB (pH 8.0). The resulting solution was reduced with 10 mM dithiothreitol at 37 °C and was alkylated with 20 mM iodoacetamide at 25 °C in darkness for 45 min. For trypsin digestion, the samples were digested with Trypsin Gold (Promega, Madison, WI, USA) to produce the digested peptides. The trypsin was added at a mass ratio of 1:50 (trypsin:protein) for a overnight digestion and at a mass ratio of 1:100 (trypsin: protein) for a 4 h digestion. Approximately 100 μg of each sample was digested for the further experiments.   TMT labeling   The digested peptide samples were desalted though a Strata X-C18-SPE column (Phenomenex, Torrance, CA, USA) and vacuum-dried. For TMT labeling, peptides were reconstituted in 0.5 M triethylammonium bicarbonate buffer and processed using a 6-plex TMT kit according to the its protocol (Thermo-Scientific, Rockford, USA). Briefly, one unit of TMT reagent, which was added to label 100 μg of protein, was thawed and reconstituted in 24 μL acetonitrile. The peptide-reagent mixture was incubated at room temperature for 2 h, pooled, desalted, and dried by vacuum centrifugation.

### Data Protocol
Database search   The resulting MS/MS data were processed using Thermo Proteome Discoverer (version 2.1.0.81) with mascot search engine against unip_Anas_8839 database. Trypsin/P was treated as cleavage enzyme allowing up to 2 missing cleavages, 5 modifications, and 5 charges per peptide. Mass error was set to 10 ppm for precursor ions and to 0.02 Da for fragment ions. Carbamido-methylation on cysteine, TMT-6-plex on lysine and N-term was treated as fixed modification, and oxidation on methionine was treated as variable modifications. The threshold for false discovery rates (FDRs) of protein, peptide and modification site was set at 0.01. Minimum peptide length was specified at 7. All the other parameters in Thermo Proteome Discoverer were set to default values.     Bioinformatic analysis    Gene Ontology (GO) annotation of all identified proteins was derived from the UniProt-GOA database (http://www.ebi.ac.uk/GOA/). The IDs of all identified proteins were converted to the UniProt IDs, which could be mapped to the GO maps. The unannotated proteins were annotated by the InterProScan software by the protein sequence alignment method.    Kyoto Encyclopedia of Genes and Genomes (KEGG) database was used to annotate metabolic and biological processes (http://www.genome.jp/kegg/). KEGG description of each identified protein was annotated by a KEGG online service tool, KAAS. Then, all annotated proteins were mapped onto the KEGG pathway using KEGG mapper, another KEGG online service tool.    An updated version of PSORT/PSORT II was used to predict the subcellular localizations of all identified proteins (http://www.genscript.com/wolf-psort.html).     Protein domain was analyzed by a sequence analysis software, InterProScan, basing on protein sequence alignment method (http://www.ebi.ac.uk/interpro/).

### Publication Abstract
Bordetella bronchiseptica (B. bronchiseptica) causes a respiratory disease in rabbits. To determine the proteins of B. bronchiseptica in rabbits related to the disease, differentially accumulated proteins in B. bronchiseptica-infected cells are identified by comparative proteomic analysis. Comparative proteomic analysis detects 5814 proteins and quantifies 4854 of these. Fifty eight upregulated and 38 downregulated proteins are identified in spleen tissue after B. bronchiseptica infection of rabbits (both p &lt; 0.05). The significantly enriched Kyoto Encyclopedia of Genes and Genomes (KEGG) pathways are ribosome, biosynthesis of amino acids, biosynthesis of amino acids, protein export, and carbon metabolism etc. (all p &lt; 0.01). Significantly enriched KEGG pathways include 'ocu03010 ribosome' (a); 'ocu00260 glycine, serine threonine metabolism'. Analyses of control and infected spleen cells detect responses to B. bronchiseptica infection. Many differentially affected proteins are evident, and reflect different biological changes and diverse subcellular localizations between control and infected spleen cells. Infection markedly alters the expressions of proteins linked to the serine protease system, with the 'phagosome,' 'biosynthesis of amino acids,' 'glycine, serine threonine metabolism,' 'intestinal immune network for IgA production', and 'amino sugar and nucleotide sugar metabolism' associated with B. bronchiseptica infection. The result will inform studies of responses to B. bronchiseptica infections in rabbits.

### Keywords
Cairna moschata; differential expressed protein; duck reovirus; proteome; serine protease

### Affiliations
Institute of Animal Husbandry and Veterinary Sciences, Zhejiang Academy of Agricultural Sciences
PTM Biolabs Inc.

### Submitter
Rachel  Green

### Lab Head
Dr Cun Zhang
Institute of Animal Husbandry and Veterinary Sciences, Zhejiang Academy of Agricultural Sciences


