### Accession
PXD002732

### Title
Proteomics of thyroid tumours -  Proteomics of thyroid tumours provides new insights into their molecular composition and changes associated with malignancy

### Description
Although most thyroid tumours are benign, thyroid cancer represents the most common malignancy of the endocrine system, comprising mainly follicular and papillary thyroid carcinomas (FTC and PTC, respectively). Previous studies have shed some light on the molecular pathogenesis of thyroid cancer but there have not been any comprehensive mass spectrometry-based proteomic studies to reveal protein expression differences between thyroid tumours and the molecular alterations associated with tumour malignancy.   We applied a label-free quantitative mass spectrometry analysis to compare normal thyroid tissue with the three most common tumours of the thyroid gland: follicular adenoma, follicular carcinoma and papillary carcinoma.

### Sample Protocol
resh frozen thyroid tissues were lysed on ice with sonication in 20 mM HEPES, 1% sodium deoxycholate (SDOC), 0.1% SDS, 150 mM NaCl, and 1 mM EDTA, pH 8, with complete, EDTA-free protease inhibitor cocktail (Roche) and phosphatase inhibitor tablet (Thermo Scientific Pierce). Lysates were centrifuged (12,000 g, 10 min, 4°C) and the supernatants retained. After protein quantitation (BCA method), 10 ug of protein were reduced in 10 mM dithiothreitol (DTT; 30 min, 60°C) , alkylated with 50 mM iodoacetamide (30 min in the dark), diluted to 50 uL with 1% SDOC and incubated at 37 C overnight with 0.2 ug of sequencing grade modified trypsin (Promega). After protein digestion, samples were acidified with 1 uL of formic acid and the precipitate was removed by centrifugation (14000 g, 10 min). StageTip-based strong anion-exchange (SAX) fractionation was  applied on eight peptide samples, two from each thyroid tissue type, previously purified with C18 StageTips. Sample were loaded onto a ChromXP C18 trap (200 um i.d. x 0.5 mm, 3 um particle size) with an Ekspert nanoLC 400 cHiPLC system. Peptides were then separated on a ChromXP C18 column (200 um i.d. x 15 cm, 3 um particle size) with a 120 min gradient using 0.1% FA/2% ACN (v/v) in water and 0.1% FA (v/v) in ACN as weak and strong elution solvents, respectively. The LC system was coupled to a TripleTOF 6600 mass spectrometer (AB Sciex), where we employed SWATH-MS. To this end, we first built a reference spectral library by information-dependent analysis (IDA) of six different samples and the eight samples prepared by SAX fractionation. Mass spectrometry conditions during IDA analysis were as follows: spray voltage-2.5 kV, interface heater temperature-150 °C, survey scan time-250 ms over the m/z range 350-1500, MS/MS with rolling collision energy of the 20 most intense precursor ions with charge states 2+ to 4+ and above 200 c.p.s., MS/MS accumulation time-100 ms in high sensitivity mode and dynamic exclusion of 30 s within 50 ppm. For SWATH-MS, 100 windows of variable m/z range and 50 ms accumulation time were employed, covering the precursor mass range of 350-1500 m/z. MS/MS scans with accumulation time of 30 ms were acquired in high sensitivity mode, resulting in a total cycle time of ~3.1 s.

### Data Protocol
IDA MS files from the groups of SAX-fractionated and unfractionated samples were processed using ProteinPilot 5.0 software with the Paragon database search algorithm. MS/MS spectra were searched against the UniProtKB/Swiss-Prot human protein database 2014_04 with 20,265 entries. Up to two missed cleavages were allowed and identification was performed considering cysteine alkylation by iodoacetamide, biological modifications and detected protein threshold > 1.3 with FDR analysis enabled. The Paragon group files were imported into PeakView 2.1 software with SWATH acquisition MicroApp 2.0, shared peptides were excluded and the ion libraries from unfractionated and fractionated samples were exported to create an extended ion library using the list of ions from unfractionated samples as a seed spectral library. Only the following peptide modifications were kept: cysteine alkylation and protein N-terminal acetylation. The precursor retention time of peptides passing these criteria was aligned to the seed library by supervised learning-based linear regression modelling on common peptides.  The extended library was then used for peptide identification in the SWATH files considering up to 100 peptides per protein, 6 transitions per peptide, 99% peptide confidence, 1% FDR, fragment ion extraction window of 10 min and mass tolerance of 75 ppm.

### Publication Abstract
Around 5% of the general population have palpable thyroid nodules. Although most thyroid tumours are benign, thyroid cancer represents the most common malignancy of the endocrine system, comprising mainly follicular and papillary thyroid carcinomas. Previous studies have shed some light on the molecular pathogenesis of thyroid cancer but there have not been any comprehensive mass spectrometry-based proteomic studies of large scale to reveal protein expression differences between thyroid tumours and the molecular alterations associated with tumour malignancy. We applied data-independent acquisition mass spectrometry which enabled quantitative expression analysis of over 1,600 proteins from 32 specimens to compare normal thyroid tissue with the three most common tumours of the thyroid gland: follicular adenoma, follicular carcinoma and papillary carcinoma. In follicular tumours, we found marked reduction of the tumour suppressor and therapeutic target extracellular protein decorin. We made the novel observation that TGF&#x3b2;-induced protein ig-h3 (TGFBI) was found frequently overexpressed in follicular carcinoma compared with follicular adenoma. Proteomic pathway analysis showed changes in papillary carcinoma were associated with disruption of cell contacts (loss of E-cadherin), actin cytoskeleton dynamics and loss of differentiation markers, all hallmarks of an invasive phenotype.

### Keywords
Swath, Thyroid cancer proteomics

### Affiliations
MQ University
Australian Proteome Analysis Facility, Macquarie University, NSW 2109, Australia

### Submitter
Juan Martinez-Aguilar

### Lab Head
Dr Mark P. Molloy
Australian Proteome Analysis Facility, Macquarie University, NSW 2109, Australia


