### Accession
PXD005443

### Title
Salinity-induced Palmella Formation Mechanism in Dunaliella salina Revealed by Quantitative Proteomics

### Description
The halotolerant algae Dunaliella salina is a good single-cell model for studying plant adaptation to high salinity. We found 151 salinity-responsive proteins were involved in multiple signaling and metabolic pathways upon palmella formation. The patterns of protein accumulation exhibited changes, including dynamics of cytoskeleton and cell membrane curvature, accumulation and transport of exopolycsccharides, photosynthesis and energy supplying (i.e. photosystem II stability and activity, cyclic electron transport, photorespiration, and C4 pathway), nuclear/chloroplastic gene expression regulation and protein processing, reactive oxygen species homeostasis, and salt signaling transduction.

### Sample Protocol
Dunaliella salina cells were cultivated in a modified medium containing 1 M NaCl under a 16/8 h light/dark cycle (light intensity 100 μmol photonsm-2s-1) at 26 °C with shaking at 96 rpm. Cells in the stationary phase were transferred to fresh medium containing 1 M NaCl (control) and 3 M NaCl (salinity shock), respectively. After treatment, algae cells were used freshly or stored at -80 °C for experiments. The proteins for differential stable isotope labeling experiments were extracted in the modified extraction buffer (900 mM sucrose, 100 mM Tris-HCl (pH 8.0), 65 mM DTT, 1 mM MgCl2, 1% Triton X-100 (v/v), 1 mM phenylmethanesulfonyl fluoride, 1% protease cocktail (v/v), 1 mM EDTA, 1 mM EDGA, 1 mM NaF, and 1 mM Na3VO4, 1 mM Na4O7P2, 1 mM C3H7Na2O6P) according to the phenol-methanol methods. Protein concentration was determined using a Quant-kit according to manufacture instructions (GE Healthcare, USA). The protein was resuspended in the denaturing buffer containing 8 M urea and 100 mM triethylammonium bicarbonate (pH 8.0), and then was reduced by DTT and alkylated by iodoacetamide. After that, the solution was diluted with 100 mM triethylammonium bicarbonate and trypsin was added with an enzyme-to-protein ratio of 1/25 (w/w) and incubated at 37 °C overnight. For the light and heavy dimethyl labeling, 500 μl of CH2O (4%, v/v) and CD2O (4%, v/v) was added to the tryptic digests from D. salina cells cultured in 1 M NaCl or 3 M NaCl medium, respectively, then 500 μl of freshly prepared NaBH3CN (0.6 M) was subsequently added.  For the reversed-phase (RP) liquid chromatography (LC)-MS/MS analysis, a 12 cm × 75 μm i.d. capillary column packed with C18 AQ particles (3 μm, 120 Å) was used as the separation column, and 0.1% formic acid in water and in acetonitrile was used as mobile phases A and B, respectively. For the quantitative proteomic analysis, 20 μg tryptic digests were loaded onto the strong cation exchange (SCX) monolithic column and then were eluted by 11 salt stepwise elutions to perform the 2-D RPLC-MS/MS analysis. A series of stepwise elutions with salt concentrations of 50, 100, 150, 200, 250, 300, 350, 400, 450, 500, and 1000 mM NH4AC were used to elute peptides from the SCX column to the second dimensional C18 separation column. Each RPLC-MS/MS elution step was performed in a 72 min gradient from 5 to 35% acetonitrile. The LTQ-Orbitrap Velos mass spectrometer (Thermo Fisher Scientific) was operated in data-dependent MS/MS acquisition mode. Full mass spectra were acquired in the Orbitrap at a resolution of 60,000 (m/z 400). The 20 most intense precursors were selected to fragmentation via collision induced dissociation in the LTQ. The dynamic exclusion function was set as follows: repeat count 2, repeat duration 30 s, and an exclusion duration of 60 s.

### Data Protocol
Protein identification and quantification are based on the MaxQuant software (version 1.1.1.36) according to the standard workflow. The database search was performed on the Andromeda search engine against the Chlorophyta protein database (from NCBI, containing 156,988 entries) concatenated with reversed sequences for evaluating of the false discovery rate (FDR). Carbamidomethylation on cysteine was set as a fixed modification, whereas oxidation on methionine was set as a variable modification.

### Publication Abstract
Palmella stage is critical for some unicellular algae to survive in extreme environments. The halotolerant algae <i>Dunaliella salina</i> is a good single-cell model for studying plant adaptation to high salinity. To investigate the molecular adaptation mechanism in salinity shock-induced palmella formation, we performed a comprehensive physiological, proteomics and phosphoproteomics study upon palmella formation of <i>D. salina</i> using dimethyl labeling and Ti<sup>4+</sup>-immobilized metal ion affinity chromatography (IMAC) proteomic approaches. We found that 151 salinity-responsive proteins and 35 salinity-responsive phosphoproteins were involved in multiple signaling and metabolic pathways upon palmella formation. Taken together with photosynthetic parameters and enzyme activity analyses, the patterns of protein accumulation and phosphorylation level exhibited the mechanisms upon palmella formation, including dynamics of cytoskeleton and cell membrane curvature, accumulation and transport of exopolysaccharides, photosynthesis and energy supplying (i.e., photosystem II stability and activity, cyclic electron transport, and C4 pathway), nuclear/chloroplastic gene expression regulation and protein processing, reactive oxygen species homeostasis, and salt signaling transduction. The salinity-responsive protein-protein interaction (PPI) networks implied that signaling and protein synthesis and fate are crucial for modulation of these processes. Importantly, the 3D structure of phosphoprotein clearly indicated that the phosphorylation sites of eight proteins were localized in the region of function domain.

### Keywords
Dunaliella salina, Quantitative proteomics, Salinity stress, Palmella formation

### Affiliations
NEFU
Alkali Soil Natural Environmental Science Center, Northeast Forestry University, Harbin 150040, China

### Submitter
Shaojun Dai

### Lab Head
Dr Shaojun Dai
Alkali Soil Natural Environmental Science Center, Northeast Forestry University, Harbin 150040, China


