### Accession
PXD010644

### Title
Plasmodium falciparum bromo-domain containing epigenetic protein complexes recruited to modified histone tails as identified by histone peptide pulldown (HPP)-MS/MS and further characterized by GFP-IP-MS/MS and HA-IP-MS/MS

### Description
In order to identify epigenetic protein complexes recruited to histone PTMs in the deadly human malaria parasite Plasmodium falciparum, we set out to perform histone peptide pulldowns with various histone peptides. Biotinylated peptides corresponding to P. falciparum histone variant H2A.Z, H2B.Z and H3.3,  canonical P. falciparum histone H4 or human histone H4 tail sequences (differing on position 21 from the P. falciparum peptide) bearing multiple histone post-translational modifications (acetylations and/or methylations) as well as the unmodified control peptides were coupled to beads and incubated with native nuclear extract obtained from mixed-stage asexual cultures. Proteins preferentially binding to the modified histone peptide were identified by quantitative proteomics using di-methyl labelling. This yielded 6 out of 7 P. falciparum bromo-domain proteins preferentially binding to the acetylated histone peptides, as well as many putative bromo-domain protein interactors. In addition, a PHD-domain containing protein was found to be recruiting a PfSAGA-like complex to H3K4 di- and tri-methylated peptides. 7 reader-domain containing proteins (BDP1, BDP2, BDP4, TAF1, GCN5, PHD1, PHD2) and 5 putative interactors (PF3D7_0306100, PF3D7_1124300, PF3D7_1128000, PF3D7_1225200, PF3D7_1451200) were endogenously tagged by GFP or HA and used in quantitative GFP-/HA-IP-MS/MS experiments to further characterize epigenetic reader-complex composition. As negative control, GFP- and HA-IPs were performed on native nuclear extract not encoding tagged protein. Most HPP experiment employed a 3-label setup, while few only relied on “light” and “heavy”-label sample pools (labels and conditions listed below). GFP- and HA-IP experiments were using a 2-label “light” and “heavy” di-methyl setup only. For all experiments technical replicates were performed using a label-swap approach (called F (forward) and R (reverse)) and independent biological replicates were performed for each. For GFP- and HA-IP forward reactions are set-up as: GFP-/HA-beads = “heavy” di-methyl label, control-beads = “light” di-methyl label. Reverse reactions are set-up as: GFP-/HA-beads = “light” di-methyl label, control-beads = “heavy” di-methyl label. For GFP-/HA-IP raw files the bait and tag used for fishing are included in the file name.

### Sample Protocol
Parasite nuclei were isolated from mixed stage P. falciparum 3D7 asexual wild-type or transgenic parasite line cultures and nuclear extract was generated under native conditions in the absence (HPP) or presence (GFP-/HA-IP) of DNaseI. For histone peptide pulldowns, epigenetic readers and interacting proteins were captured by biotinylated histone peptides containing multiple PTMs or unmodified control peptides coupled to beads. For GFP-/HA-IP GFP- or HA-tagged baits and their interactors were captured on GFP-TRAP or HA-beads, while control beads only bound background proteins. For both HPP and GFP-/HA-IP on-bead digestion was performed on the isolated proteins in TEAB buffer using TCEP for reduction, MMTS for alkylation and Trypsin/LysC for digestion. Trypsinized samples were labelled using “light”, “medium” and “heavy” di-methyl labels under label-swap conditions, after which H/(M)/L-sample pools were cleaned over C18 stage-tips. Part of the sample was separated over a 30cm C18-reverse phase column, eluted over a 94min gradient (5.6% acetonitrile/0.1% formic acid - 25.6% acetonitrile/0.1% formic acid) and directly injected into a QExactive mass spectrometer were data was acquired in TOP10 data-dependent acquisition mode with dynamic exclusion enabled for 20sec. Resolution for MS was set at 70.000 at m/z = 400 and for MS/MS at 17.5000.   Description HPP labels: - 20140403_HPP_TEST2_04NP40_350NaCl_F (L= unmod; M= H4K5acK8acK12ac; H= H4K5acK8acK12ac+Kac-competitor) (names: H4_3ac9; H4_3ac11) - 20140403_HPP_TEST2_04NP40_350NaCl_R (L= H4K5acK8acK12ac+Kac-competitor; M= unmod; H= H4K5acK8acK12ac) (names: H4_3ac10; H4_3ac12) - 20140627_HPP_TEST4_350mM-250ug_F (L= unmod; H= H4K5acK8acK12ac) (names: H4_3ac13) - 20140627_HPP_TEST4_350mM-250ug_R (L= H4K5acK8acK12ac; H= unmod) (names: H4_3ac14) - 20140627_HPP_TEST4_350mM-750ug_F (L= unmod; H= H4K5acK8acK12ac) (names: H4_3ac15) - 20140627_HPP_TEST4_350mM-750ug_R (L= H4K5acK8acK12ac; H= unmod) (names: H4_3ac16) - 20140710_HPP_D_H2BZ_F (L= unmod; M= H2BzK13acK14acK18ac; H= H2BzK3acK8acK13acK14acK18ac) (names: H2B.Z_3ac1; H2B.Z_5ac1) - 20140710_HPP_D_H2BZ_R (L= H2BzK13acK14acK18ac; M= H2BzK3acK8acK13acK14acK18ac; H= unmod) (names: H2B.Z_3ac2; H2B.Z_5ac2) - 20140710_HPP_D_H3-1_F (L= unmod; M= H3K9acK14ac; H= H3K4me3K9acK14ac) (names: H3K9acK14ac1; H3K4me3_1) - 20140710_HPP_D_H3-1_R (L= H3K9acK14ac; M= H3K4me3K9acK14ac; H= unmod) (names: H3K9acK14ac2; H3K4me3_2) - 20140710_HPP_D_H3-2_F (L= unmod; M= H3K4me3; H= H3K9me3) (names: H3K4me3_3; H3K9me3_1) - 20140710_HPP_D_H3-2_R (L= H3K4me3; M= H3K9me3; H= unmod) (names: H3K4me3_4; H3K9me3_2) - 20140909_HPP_E_H2AZ_F (L= unmod; M= H2AzK25acK28acK30acK35ac; H= H2AzK11acK15acK19acK25acK28acK30acK35ac) (names: H2A.Z_4ac1; H2A.Z_7ac1) - 20140909_HPP_E_H2AZ_R (L= H2AzK25acK28acK30acK35ac; M= H2AzK11acK15acK19acK25acK28acK30acK35ac; H= unmod) (names: H2A.Z_4ac2; H2A.Z_7ac2) - 20140909_HPP_E_H2BZ1_F (L= unmod; M= H2BzK13acK14acK18ac; H= H2BzK3acK8acK13acK14acK18ac) (names: H2B.Z_3ac3; H2B.Z_5ac3) - 20140909_HPP_E_H2BZ1_R (L= H2BzK13acK14acK18ac; M= H2BzK3acK8acK13acK14acK18ac; H= unmod) (names: H2B.Z_3ac4; H2B.Z_5ac4) - 20140909_HPP_E_H2BZ2_F (L= unmod; M= H2BzK3acK8ac; H= H2BzK3acK8acK13acK14acK18ac) (names: H2B.Z_2ac1; H2B.Z_5ac5) - 20140909_HPP_E_H2BZ2_R (L= H2BzK3acK8ac; M= H2BzK3acK8acK13acK14acK18ac; H= unmod) (names: H2B.Z_2ac2; H2B.Z_5ac6) - 20140909_HPP_E_H3_F (L= unmod; M= H3K4me3; H= H3K4me3K9acK14ac) (names: H3K4me3_5; H3K4me3_7) - 20140909_HPP_E_H3_R (L= H3K4me3; M= H3K4me3K9acK14ac; H= unmod) (names: H3K4me3_6; H3K4me3_8) - 20141107_HPP_G_H2aZ_hc_F (L= unmod; M= H2AzK25acK28acK30acK35ac; H= H2AzK11acK15acK19acK25acK28acK30acK35ac) (names: H2A.Z_4ac3; H2A.Z_7ac3) - 20141107_HPP_G_H2aZ_hc_R (L= H2AzK25acK28acK30acK35ac; M= H2AzK11acK15acK19acK25acK28acK30acK35ac; H= unmod) (names: H2A.Z_4ac4; H2A.Z_7ac4) - 20141107_HPP_G_H2aZ_lc_F (L= unmod; M= H2AzK25acK28acK30acK35ac; H= H2AzK11acK15acK19acK25acK28acK30acK35ac) (names: H2A.Z_4ac5; H2A.Z_7ac5) - 20141107_HPP_G_H2aZ_lc_R (L= H2AzK25acK28acK30acK35ac; M= H2AzK11acK15acK19acK25acK28acK30acK35ac; H= unmod) (names: H2A.Z_4ac6; H2A.Z_7ac6) - 20141107_HPP_G_H3K4_lc_F (L= unmod; M= H3K4me2; H= H3K4me3) (names: H3K4me2_1; H3K4me3_9) - 20141107_HPP_G_H3K4_lc_R (L= H3K4me2; M= H3K4me3; H= unmod) (names: H3K4me2_2; H3K4me3_10) - 20141107_HPP_G_H4k20_lc_F (L= unmod; H= H4K20me3) (names: H4K20me3_1) - 20141107_HPP_G_H4k20_lc_R (L= H4K20me3; H= unmod) (names: H4K20me3_2) - 20150205_HPP_I_H3K4_F (L= unmod; M= H3K4me2; H= H3K4me3) (names: H3K4me2_3; H3K4me3_11) - 20150205_HPP_I_H3K4_R (L= H3K4me2; M= H3K4me3; H= unmod) (names: H3K4me2_4; H3K4me3_12) - 20150205_HPP_I_H3K4me1_F (L= unmod; H= H3K4me1) (names: H3K4me1_1) - 20150205_HPP_I_H3K4me1_R (L= H3K4me1; H= unmod) (names: H3K4me1_2) … continued under “Data processing protocol”

### Data Protocol
Raw mass spectra were processes using MaxQuant (version 1.5.3.30) set to default parameters unless indicated. Multiplicity was set at 2 (GFP-/HA-IPs or HPP with 2 labels), with an added mass of 28.03Da (“light”-) or 36.08Da (“heavy”-label) to all lysine residues and peptide N-termini. Multiplicity was set at 3 (majority HPPs), with an added mass of 28.03Da (“light”-), 32.06Da (“medium”-) or 36.08Da (“heavy”-label) to all lysine residues and peptide N-termini. Trypsin/P was set as the specific digestion mode with maximum 2 missed cleavages and a mass of 45.99Da (MMTS) was set as fixed modification of cysteine residues. Match-between-runs and re-quantify options were enabled with default parameters and iBAQ values were calculated. Mass spectra were compared to peptide masses from the Plasmodium falciparum 3D7 annotated proteome (PlasmoDB release 33) with the entire human proteome included in the contaminants list using the integrated Andromeda search engine. For GFP-/HA-IPs the GFP-tag sequence was added to the P. falciparum proteome. Default search settings (mass tolerance at 4.5 ppm for precursor ions and 20 ppm for fragment ions) were enabled, and peptides and proteins were accepted with an 0.01 FDR cut-off. Protein quantification required minimally one “unique + razor” peptide-ratios.  Description HPP labels - continued: - 20150205_HPP_I_H3me3_F (L= unmod; M= H3K4me3; H= H3K9me3) (names: H3K4me3_13; H3K9me3_3) - 20150205_HPP_I_H3me3_R (L= H3K4me3; M= H3K9me3; H= unmod) (names: H3K4me3_14; H3K9me3_4) - 20150205_HPP_I_H4_F (L= unmod; H= H4K5acK8acK12ac) (names: H4_3ac5) - 20150205_HPP_I_H4_R (L= H4K5acK8acK12ac; H= unmod) (names: H4_3ac6) - 20150218_HPP_H_H2BZ_F (L= unmod; M= H2BzK3acK8acK13acK14acK18ac+bromosporine; H= H2BzK3acK8acK13acK14acK18ac) (names: H2B.Z_5ac7; H2B.Z_5ac9) - 20150218_HPP_H_H2BZ_R (L= H2BzK3acK8acK13acK14acK18ac+bromosporine; M= H2BzK3acK8acK13acK14acK18ac; H= unmod) (names: H2B.Z_5ac8; H2B.Z_5ac10) - 20150218_HPP_H_H4_F (L= unmod; M= H4K5acK8acK12ac+bromosporine; H= H4K5acK8acK12ac) (names: H4_3ac1; H4_3ac3) - 20150218_HPP_H_H4_R (L= H4K5acK8acK12ac+bromosporine; M= H4K5acK8acK12ac; H= unmod) (names: H4_3ac2; H4_3ac4) - 20150402_HPP_J_H2A2_F (L= unmod; M= H2AzK11acK15acK19acK25acK28acK30acK35ac+bromosporine; H= H2AzK11acK15acK19acK25acK28acK30acK35ac) (names: H2A.Z_7ac7; H2A.Z_7ac9) - 20150402_HPP_J_H2A2_R (L= H2AzK11acK15acK19acK25acK28acK30acK35ac+bromosporine; M= H2AzK11acK15acK19acK25acK28acK30acK35ac; H= unmod) (names: H2A.Z_7ac8; H2A.Z_7ac10) - 20150402_HPP_J_H2B2_F (L= unmod; M= H2BzK3acK8acK13acK14acK18ac+bromosporine; H= H2BzK3acK8acK13acK14acK18ac) (names: H2B.Z_5ac11; H2B.Z_5ac13) - 20150402_HPP_J_H2B2_R (L= H2BzK3acK8acK13acK14acK18ac+bromosporine; M= H2BzK3acK8acK13acK14acK18ac; H= unmod) (names: H2B.Z_5ac12; H2B.Z_5ac14) - 20150609_HPP_Students_1_F (L= unmod; M= H2BzK3acK8ac; H= H2BzK3acK8acK13acK14acK18ac) (names: H2B.Z_2ac3; H2B.Z_5ac15) - 20150609_HPP_Students_1_R (L= H2BzK3acK8ac; M= H2BzK3acK8acK13acK14acK18ac; H= unmod) (names: H2B.Z_2ac4; H2B.Z_5ac16) - 20150609_HPP_Students_2_F (L= unmod; M= H3K4me1; H= H3K4me2) (names: H3K4me1_3; H3K4me2_5) - 20150609_HPP_Students_2_R (L= H3K4me1; M= H3K4me2; H= unmod) (names: H3K4me1_4; H3K4me2_6) - 20150609_HPP_Students_3_F (L= unmod; H= H4K5acK8acK12ac) (names: H4_3ac7) - 20150609_HPP_Students_3_R (L= H4K5acK8acK12ac; H= unmod) (names: H4_3ac8) - 20150609_HPP_Students_4_F (L= unmod; H= H2BzK3acK8acK13acK14acK18ac) (names: H2B.Z_5ac17) - 20150609_HPP_Students_4_R (L= H2BzK3acK8acK13acK14acK18ac; H= unmod) (names: H2B.Z_5ac18) - 20150805_HPP_K_4_F (L= unmod; H= H4K20me3) (names: H4K20me3_3) - 20150805_HPP_K_4_R (L= H4K20me3; H= unmod) (names: H4K20me3_4) - 20170609_iHPP2_F (L= unmod; M= H4K5acK8acK12ac; H= H4K5acK8acK12ac+poolC) (names: H4_3ac17) - 20170609_iHPP2_R (L= H4K5acK8acK12ac; M= H4K5acK8acK12ac+poolC; H= unmod) (names: H4_3ac18)

### Publication Abstract
None

### Keywords
Gcn5, Histone methylation, Hpp, Histone acetylation, Epigenetic complex, Bromo-domain protein, Histone peptide pulldown, Plasmodium falciparum, Reader complex, Ha-ip, Gfp-ip, Interactor, Taf1

### Affiliations
Dr. Richard Bartfai ; Department of Molecular Biology ; Faculty of Science ; Radboud University ; Geert Grooteplein 28 ; 6525GA Nijmegen, The Netherlands; Tel.: +31 (0)24-3610561 ; r.bartfai@science.ru.nl
Department of Molecular Biology, Faculty of Science, Radboud University, Nijmegen
The Netherlands

### Submitter
Wieteke Hoeijmakers

### Lab Head
Dr Dr. Richard Bartfai
Dr. Richard Bartfai ; Department of Molecular Biology ; Faculty of Science ; Radboud University ; Geert Grooteplein 28 ; 6525GA Nijmegen, The Netherlands; Tel.: +31 (0)24-3610561 ; r.bartfai@science.ru.nl


