### Accession
PXD012906

### Title
Identification of PKC-alpha dependent phosphoproteins in mouse retina

### Description
Adjusting to a wide range of light intensities is an essential feature of retinal rod bipolar cell (RBC) function, and persuasive evidence suggests this modulation involves phosphorylation by protein kinase C-alpha (PKC-alpha). PKC-alpha is a serine/threonine kinase that is strongly expressed in RBCs, but the targets of PKC-alpha phosphorylation in the retina have not been identified. PKC-alpha activity and phosphorylation in RBCs was examined by immunofluorescence confocal microscopy using a conformation-specific PKC-alpha antibody and antibodies to phosphorylated PKC motifs. PKC-alpha activity was dependent on light and expression of TRPM1, and RBC dendrites were the primary sites of light-dependent phosphorylation.  PKC-alpha-dependent retinal phosphoproteins were identified using a multiplexed tandem mass tag-based approach to compare total protein and phosphopeptide abundance between phorbol ester-treated wild type and PKC-alpha knockout (PKC-alpha-KO) mouse retinas. Of the 23 proteins showing significant differential abundance between the two groups, most have roles in cytoskeleton/transport, transcriptional regulation, or metabolism/homeostasis. Phosphopeptide mass spectrometry identified over 1100 phosphopeptides in mouse retina, with 12 displaying significantly greater phosphorylation in WT compared to PKC-alpha-KO samples. The differentially phosphorylated proteins fall into the following functional groups: cytoskeleton/transport (4 proteins), ECM/adhesion (2 proteins), signaling (2 proteins), transcriptional regulation (3 proteins), and homeostasis/metabolism (1 protein). Two strongly differentially expressed phosphoproteins, BORG4 and TPBG, were localized to the synaptic layers of the retina, and may play a role in PKC-alpha-dependent modulation of RBC physiology.

### Sample Protocol
MICE RETINAS Retinas from wild-type mice (Jackson Labs #000664, n=4) and protein kinase C-alpha knock-out mice (Jackson Labs #009068, n=5) were freshly dissected and incubated at 37 C in 1 micro-M phorbol 12-myristate 13-acetate (PMA; Sigma-Aldrich; Cat# P8139) diluted in bicarbonate buffered Ames medium (US Biological Life Sciences; Cat# A1372-25) for 60 min. Following treatment, retinas were washed three times with Ames medium before being placed in chilled lysis buffer (50 mM HEPES, pH 8.5, 8 M urea, 1 mM NaF, 1 mM sodium orthovanadate, 10 mM sodium pyrophosphate, 1 mM beta-glycerophosphate), and lysed by probe sonication (Sonic Dismembrator 60; Thermo Scientific) 3 x 15 seconds at a setting of 4 with cooling on ice for 30 seconds between treatments.   PROTEIN DIGESTION Approximately 500ï€ micrograms of protein [Pierce BCA Protein Assay Kit (Thermo Scientific; Cat# 223227)] in 250 microliters of lysis buffer was used for further processing. Protein disulfides were reduced and alkylated. Sequence grade modified trypsin (Promega; Cat# V5111) was added at a 25:1 protein:trypsin ratio and samples were incubated overnight at 37C before being acidified with trifluoroacetic acid (TFA) to a final concentration of 1%. Remaining particulates were removed by centrifugation at 16000 x g for 10 min. Peptides were purified using 1 cc (50 mg) Waters Sep-Pak Vac tC18 cartridges (Waters Corporation; Cat# WAT054960). Peptide concentrations were determined using the Pierce Quantitative Colorimetric Peptide Assay (Thermo Scientific; Cat# 23275). Approximately 300 micrograms of peptide were recovered from each digest. Fifteen micrograms of each sample were reserved for TMT analysis of whole protein abundance, and the remainder was used for the phosphopeptide enrichment experiment.  PHOSPHOPEPTIDE ENRICHMENT Phosphopeptides were enriched as follows: titanosphere TiO2 5-micron beads (GL Biosciences; Cat# 5020-75000) were washed three times in 2 M lactic acid/50% ACN, then resuspended in the same solution at 24 mg/mL. Approximately 285 micrograms of dried peptide for each sample was resuspended in 950 microliters of 2 M lactic acid/50% ACN and 100 microliters (2.4 mg) of the bead suspension was added to each sample, ensuring an 8:1 ratio of beads:peptide. The bead:peptide mixture was rotated at room temperature for 1 hour, then washed twice with 500 microliter 2M lactic acid/50% ACN, 0.1% TFA/50% ACN, then 0.1% TFA/25% ACN. The enriched phosphopeptides were eluted from the beads by vortexing in 100 microliters of 50 mM K2HPO4 at pH 10 for 5 min. The elution step was repeated, and the eluate was dried by vacuum centrifugation. The enriched phosphopeptides were then purified by solid phase extraction using UltraMicro Spin columns (The Nest Group Inc.). The dried phosphopeptides were resuspended in 60 microliter 1% TFA. The samples were loaded onto the conditioned columns and passed through the columns three times, washed three times with 25 microliters of 0.1% TFA, eluted three times with 50 microliters 80% ACN/0.1% formic acid, and dried before TMT labeling.  TMT LABELING The samples were dissolved in 25 microliters of 100 mM triethylammonium bicarbonate buffer, and TMT 10-plex reagents (Thermo Scientific; Cat# 90110) were dissolved in anhydrous ACN. Each sample was labeled by adding 12 microliters (180 micrograms) of an individual TMT reagent, followed by shaking at room temperature for 1 hr. Two microliters each of the nine labeled samples were pooled, and peptides were analyzed by a single 2-hour LC-MS/MS method to normalize the total reporter ion intensities and to check labeling efficiency. The remaining unmixed samples were quenched by the addition of 2 microliters 5% hydroxylamine, then combined in adjusted volumes to yield equal summed reporter ion intensities during the subsequent two-dimensional LC/MS. The combined samples were dried by vacuum centrifugation, and TMT-labeled samples were reconstituted in 20 microliters water and separated by two-dimensional nano reverse-phase liquid chromatography/mass spectrometry using a Dionex NCS-3500 UltiMate RSLCnano UPLC and Orbitrap Fusion Tribrid mass spectrometer (Thermo Scientific) and enriched phosphopeptides were eluted by sequential 20 microliter injections of 4, 6, 8, 10, 12, 18, 20, 22, 25, 30, and 60% ACN in 10 mM ammonium formate, pH 9.  MASS SPECTROMETRY Data was collected using an Orbitrap Fusion Tribrid instrument configured with an EasySpray NanoSource (Thermo Scientific). Survey scans used the Orbitrap mass analyzer (resolution = 120000), and data-dependent MS2 scans used the linear ion trap using collision-induced dissociation (normalized collision energy = 35). Reporter ion detection was performed in the Orbitrap mass analyzer (resolution = 60000) using MS3 scans following synchronous precursor isolation of the top 10 ions, and higher-energy collisional dissociation (normalized collision energy = 65).

### Data Protocol
TMT DATA ANALYSIS RAW instrument files were processed using Proteome Discoverer version 1.4.1.14 (Thermo Scientific). Raw files from the fractions were merged and searched with the SEQUEST HT search engine with a canonical Mus musculus Swiss-Prot protein database downloaded July 2017 (16794 entries and 179 common contaminants). Searches were configured with static modifications for the TMT reagents (+229.163 Da) on lysines and N-termini, carbamidomethyl (+57.021 Da) on cysteines, dynamic modifications for oxidation of methionine residues (+15.9949 Da) and phosphorylation (+79.9799 Da on serine, threonine, or tyrosine residues), parent ion tolerance of 1.25 Da, fragment mass tolerance of 1.0005 Da, monoisotopic masses, and trypsin cleavage (max 2 missed cleavages). The phosphoRS node was used to provide site localization information. Searches used a reversed sequence decoy strategy to control peptide false discovery and identifications were validated by Percolator software. Only peptides with q scores < 0.01 were accepted.    Search results and TMT reporter ion intensities were exported as text files and processed with in-house scripts. A trimmed reporter ion intensity average cutoff of 500 was used to remove low quality data. Limited aggregation of the peptide-centric data was used to improve data quality. All PSMs for any peptides having identical peptide sequences and the same total number of modifications were combined (reporter ions summed). After combining PSMs, any missing reporter ion values were replaced with 150. Differential phosphopeptide abundance was determined by comparing the combined reporter ion intensities between groups using the Bioconductor package edgeR. Additional data normalizations, multiple testing corrections, and calculation of false discovery rates were performed within edgeR.

### Publication Abstract
Adjusting to a wide range of light intensities is an essential feature of retinal rod bipolar cell (RBC) function. While persuasive evidence suggests this modulation involves phosphorylation by protein kinase C-alpha (PKC&#x3b1;), the targets of PKC&#x3b1; phosphorylation in the retina have not been identified. PKC&#x3b1; activity and phosphorylation in RBCs was examined by immunofluorescence confocal microscopy using a conformation-specific PKC&#x3b1; antibody and antibodies to phosphorylated PKC motifs. PKC&#x3b1; activity was dependent on light and expression of TRPM1, and RBC dendrites were the primary sites of light-dependent phosphorylation. PKC&#x3b1;-dependent retinal phosphoproteins were identified using a phosphoproteomics approach to compare total protein and phosphopeptide abundance between phorbol ester-treated wild type and PKC&#x3b1; knockout (PKC&#x3b1;-KO) mouse retinas. Phosphopeptide mass spectrometry identified over 1100 phosphopeptides in mouse retina, with 12 displaying significantly greater phosphorylation in WT compared to PKC&#x3b1;-KO samples. The differentially phosphorylated proteins fall into the following functional groups: cytoskeleton/trafficking (4 proteins), ECM/adhesion (2 proteins), signaling (2 proteins), transcriptional regulation (3 proteins), and homeostasis/metabolism (1 protein). Two strongly differentially expressed phosphoproteins, BORG4 and TPBG, were localized to the synaptic layers of the retina, and may play a role in PKC&#x3b1;-dependent modulation of RBC physiology. Data are available via ProteomeXchange with identifier PXD012906. SIGNIFICANCE: Retinal rod bipolar cells (RBCs), the second-order neurons of the mammalian rod visual pathway, are able to modulate their sensitivity to remain functional across a wide range of light intensities, from starlight to daylight. Evidence suggests that this modulation requires the serine/threonine kinase, PKC&#x3b1;, though the specific mechanism by which PKC&#x3b1; modulates RBC physiology is unknown. This study examined PKC&#x3b1; phosophorylation patterns in mouse rod bipolar cells and then used a phosphoproteomics approach to identify PKC&#x3b1;-dependent phosphoproteins in the mouse retina. A small number of retinal proteins showed significant PKC&#x3b1;-dependent phosphorylation, including BORG4 and TPBG, suggesting a potential contribution to PKC&#x3b1;-dependent modulation of RBC physiology.

### Keywords
Signaling, Quantitative proteomics, Mouse, Retina, Isobaric labeling, Protein kinase c-alpha

### Affiliations
Department of Physiology & Pharmacology Oregon Health & Science University 3181 SW Sam Jackson Park Road Portland, Oregon 97239, USA
OHSU

### Submitter
Phillip Wilmarth

### Lab Head
Dr Dr. Catherine Morgans
Department of Physiology & Pharmacology Oregon Health & Science University 3181 SW Sam Jackson Park Road Portland, Oregon 97239, USA


