### Accession
PXD031364

### Title
MCL-1 Drives a Metabolic Dependency on Fatty Acid Oxidation in Cancer Independent of its Anti-Apoptotic Function

### Description
MCL-1 is an anti-apoptotic BCL-2 family protein essential to the survival of diverse cell types and is a major driver of cancer and chemoresistance. The unique oncogenic supremacy of MCL-1, as compared to its anti-apoptotic homologs, suggests that it has additional functions to complement apoptotic suppression. Here, we find that MCL-1-dependent hematologic cancer cells selectively rely on fatty acid oxidation (FAO) as a fuel source due to metabolic wiring enforced by MCL-1 itself. Importantly, this metabolic function is independent of anti-apoptotic activity, as demonstrated by metabolomic, proteomic, and genomic profiling of MCL-1-dependent leukemia cells lacking pro-apoptotic BAX and BAK. Genetic deletion of Mcl-1 results in selective downregulation of proteins within the FAO pathway, accompanied by cell death upon glucose deprivation despite apoptotic blockade. Our data reveal that MCL-1 drives a programmatic dependency on FAO in hematologic cancer cells, which can be effectively targeted by FAO inhibitors.

### Sample Protocol
Murine Bax-/-Bak-/- (DKO) and Bax-/-Bak-/-Mcl-1-/- (TKO) B-ALL cells were plated at 500,000 cells/mL in 6-well plates in regular or low glucose media. After 24 hours, cells were harvested and lysed in CHAPS buffer. The resulting lysates were quantified by the BCA Protein Assay Kit (Thermo Fisher Scientific, catalog #23225) and subsequently reduced, alkylated, and subjected to chloroform/methanol precipitation (Navarrete-Perea et al., 2018). Protein precipitates were then resuspended in 100 L of 200 mM EPPS buffer, pH 8.5 before proteolytic digestion by LysC (overnight) and trypsin (6 hours). Tryptic peptides were subsequently processed in highly parallel fashion using isobaric tagging (TMTpro16-plex) (Li et al., 2020), followed by 1:1 mixing across all channels. Samples were then desalted using a 100 mg Sep-Pak solid-phase extraction column. We fractionated the pooled, labeled peptide sample using BPRP HPLC (Wang et al., 2011). We used an Agilent 1200 pump equipped with a degasser and a detector set at 220 and 280 nm wavelength. Peptides were subjected to a 50-min linear gradient from 5% to 35% acetonitrile in 10 mM ammonium bicarbonate pH 8 at a flow rate of 0.6 mL/min over an Agilent 300Extend C18 column (3.5 m particles, 4.6 mm ID and 220 mm in length). The peptide mixture was fractionated into a total of 96 fractions, which were consolidated into 24 super-fractions (Paulo et al., 2016). Samples were subsequently acidified with 1% formic acid and vacuum centrifuged to near dryness. Each consolidated fraction was desalted via StageTip, dried again via vacuum centrifugation, and reconstituted in 5% acetonitrile, 5% formic acid for LC-MS/MS processing.

### Data Protocol
Mass spectra were processed using a Comet-based software pipeline (Eng et al., 2015; Eng et al., 2013). Database searching included all entries from the mouse UniProt database (March 20, 2020). This database was concatenated with one composed of all protein sequences in the reversed order. Searches were performed using a 50-ppm precursor ion tolerance and the product ion tolerance was set to 0.9 Da for SPS-MS3. Enzyme specificity was assigned as trypsin. TMTpro labels on lysine residues and peptide N termini (+304.207) and carbamidomethylation of cysteine residues (+57.021 Da) were set as static modifications, while oxidation of methionine residues (+15.995 Da) was set as a variable modification. Peptide-spectrum matches (PSMs) were adjusted to a 1% false discovery rate (FDR) (Elias and Gygi, 2007, 2010). PSM filtering was performed using a linear discriminant analysis, as described previously (Huttlin et al., 2010), while considering the following parameters: XCorr, peptide length, ΔCn, charge state, missed cleavages, and mass accuracy of the precursor. For TMT-based reporter ion quantitation, we extracted the signal-to-noise (S:N) ratio for each TMT channel and found the closest matching centroid to the expected mass of the TMT reporter ion. PSMs were identified, quantified, and collapsed to a 1% peptide FDR and then collapsed further to a final protein-level FDR of 1%. Peptide intensities were quantified by summing reporter ion counts across all matching PSMs so as to give greater weight to more intense ions (McAlister et al., 2012; McAlister et al., 2014).

### Publication Abstract
None

### Keywords
Fatty acid oxidation, Mcl-1, Mitochondria, Metabolism, Cancer, Bcl-2 family, Apoptosis

### Affiliations
Harvard Medical School
Department of Pediatric Oncology and Linde Program in Cancer Chemical Biology Dana-Farber Cancer Institute Boston, MA, USA

### Submitter
Joao Paulo

### Lab Head
Dr Loren Walensky
Department of Pediatric Oncology and Linde Program in Cancer Chemical Biology Dana-Farber Cancer Institute Boston, MA, USA


