### Accession
PXD002375

### Title
Mollivirus sibericum particle characterization

### Description
We describe a fourth type of giant virus infecting Acanthamoeba, Mollivirus sibericum, isolated from the same permafrost sample than Pithovirus sibericum.

### Sample Protocol
For virion proteome analysis, the protein content of 3 independent biological replicates was in-gel digested and the resulting peptides submitted to nanoLC-MS/MS (2 hours analyses using LTQ-Orbitrap Velos pro for one replicate and Q-Exactive Plus for the 2 others). For surfome analysis, full virions were incubated 30 min with or without trypsin. Resulting peptides were analysed by nanoLC-MS/MS (2 hours analyses using Q-Exactive Plus).

### Data Protocol
Raw data were processed automatically using Mascot Daemon software (version 2.5.1, Matrix Science). Concomitant searches against Mollivirus protein sequence databank (523 entries), in house built A. castellanii protein sequence databank (16,206 entries) classical contaminants database (67 126 sequences, homemade) and the corresponding reversed databases were performed using Mascot (version 2.5). ESI-TRAP was chosen as the instrument, trypsin/P as the enzyme and 2 missed cleavage allowed. Precursor and fragment mass error tolerances were set respectively at 10 ppm and 0.6 Da for data acquired on the LTQ-Orbitrap Velos and 10 ppm and 25 mmu for data acquired on the Q-Exactive Plus. Peptide modifications allowed during the search were: carbamidomethyl (C, fixes) acetyl (N-ter, variable) and oxidation (M, variable) and deamidation (NQ, variable). The IRMa software (Dupierris V et al., 2009, Bioinformatics) (version 1.31.1) was used to filter the results: conservation of rank 1 peptides, peptide identification FDR < 1% (as calculated on peptide scores by employing the reverse database strategy), and minimum of 1 specific peptide per identified protein group.

### Publication Abstract
Acanthamoeba species are infected by the largest known DNA viruses. These include icosahedral Mimiviruses, amphora-shaped Pandoraviruses, and Pithovirus sibericum, the latter one isolated from 30,000-y-old permafrost. Mollivirus sibericum, a fourth type of giant virus, was isolated from the same permafrost sample. Its approximately spherical virion (0.6-&#xb5;m diameter) encloses a 651-kb GC-rich genome encoding 523 proteins of which 64% are ORFans; 16% have their closest homolog in Pandoraviruses and 10% in Acanthamoeba castellanii probably through horizontal gene transfer. The Mollivirus nucleocytoplasmic replication cycle was analyzed using a combination of "omic" approaches that revealed how the virus highjacks its host machinery to actively replicate. Surprisingly, the host's ribosomal proteins are packaged in the virion. Metagenomic analysis of the permafrost sample uncovered the presence of both viruses, yet in very low amount. The fact that two different viruses retain their infectivity in prehistorical permafrost layers should be of concern in a context of global warming. Giant viruses' diversity remains to be fully explored.

### Keywords
Nanolc-ms/ms, Giant virus

### Affiliations
EDyP

### Submitter
Yohann CoutÃ©

### Lab Head
Dr Bruley  
EDyP


