### Accession
PXD024486

### Title
Effect of starvation conditions on cultured human H4 neuroglioma cells stably expressing shLuc and shATG5

### Description
An attempt to analyse a large proportion of the human proteome of cultured Human H4 neuroglioma cells during conditions simulating starvation. Comparisons were made under conditions of ATG5 knock-down and control (Luc) knock-down. Whole proteome crude extracts were prepared and comparisons made using SILAC in both 'forward' and 'reverse' label arrangements.

### Sample Protocol
Human H4 neuroglioma cells stably expressing shLuc and shAtg5 were grown in Dulbecco's modified Eagle's medium lacking all amino acids except L-lysine and L-arginine, which were replaced with either unlabelled amino-acids or stable isotopic forms; 13C6 15N2-lysine (Lys8) and 13C6 15N4-arginine (Arg10) (Cambridge Isotope Laboratories). The medium was supplemented with 10% dialyzed FBS. Two separate cultures of each ShLuc and ShATG5 cells were grown in light amino acids (Lys0 + Arg0) or heavy amino acids (Lys8 + Arg10) over 9 replication cycles to achieve over 96% incorporation and similar cell counts per culture. Cultures were then both halved and cells were grown under control conditions in DMEM + 10% FBS or starvation in EBSS media for 4 h such that both cell types grown in both labels were treated with both conditions (8 cultures in total). Crude cell extracts to approximately 1 mg.mL-1 protein concentration were prepared for all 8 cultures by lysis of washed cell pellets in 1.2 x LDS sample buffer with reducing agent (ThermoFisher Scientific). To allow all experimental conditions to be compared with one a light reference mix was obtained by combining all four light amino acid conditions in a 1:1:1:1 ratio by volume. This was then mixed 1:1 ratio (v:v) with each heavy amino acid extract. The same comparisons were made in reverse by combining all heavy amino acid samples into a reference mix and combining this 1:1 (v:v) with each of the four individual light amino acid conditions. See accompanying experimental design figure. Approximately 20ðg of peptide from each of the 8 mixes were fractionated by SDS-PAGE and stained gels cut into 3 slices per lane before tryptic peptides were extracted.   The resultant 24 samples of dried down peptides were resuspended in 35 ðl 0.1% TFA 0.5% acetic acid. Peptide samples were analyzed by LC-MS/MS twice; the first using 9 ðl peptide sample run over a 90 min peptide fractionation gradient, and the second using 18 ðl peptide sample run over a 240 min fractionation gradient. Peptides were analyzed using a Q exactive mass spectrometer (Thermo Scientific) couples to an EASY-nLC 1000 liquid chromatography system (Thermo Scientific), using an EASY-Spray ion source (Thermo Scientific), running a 75 ðm x 500 mm EASY-Spray column at 45ÂºC. A top 10 data-dependent method was applied. Full scan spectra (m/z 300-1800) were acquired with resolution R= 70,000 at m/z 200 (after accumulation to a target value of 1,000,000 with maximum injection time of 20ms. The most intense ions were fragmented by HCD and measured with a resolution of R= 17,500 at m/z 200 (target value of 500,000 maximum injection time of 60 ms) and intensity threshold of 2.1x104. Peptide match was set to â€œpreferredâ€, a 40 second dynamic exclusion list was applied and ions were ignored if they had an assigned charge state of 1, 8 or >8.

### Data Protocol
All 48 data files were analyzed simultaneously in MaxQuant (v1.5.2.8) using default parameters excepting the selection of SILAC labels, activation of â€˜requantifyâ€™ and â€˜match between runsâ€™, using a uniport â€˜HUMANâ€™ proteome database (downloaded 24/02/2015- 73920 entries) as search space. Raw files derived from the same mix were grouped under the â€˜Experimentâ€™ heading as Mix01-Mix08. Raw files were given MaxQuant experimental design â€˜fractionâ€™ numbers such that spectra derived from the same HPLC gradient and from the same gel slices across different lanes would be matched, as well as one sliced either side.

### Publication Abstract
Macroautophagy and the ubiquitin proteasome system work as an interconnected network in the maintenance of cellular homeostasis. Indeed, efficient activation of macroautophagy upon nutritional deprivation is sustained by degradation of preexisting proteins by the proteasome. However, the specific substrates that are degraded by the proteasome in order to activate macroautophagy are currently unknown. By quantitative proteomic analysis we identified several proteins downregulated in response to starvation independently of ATG5 expression. Among them, the most significant was HERPUD1, an ER membrane protein with low expression and known to be degraded by the proteasome under normal conditions. Contrary, under ER stress, levels of HERPUD1 increased rapidly due to a blockage in its proteasomal degradation. Thus, we explored whether HERPUD1 stability could work as a negative regulator of autophagy. In this work, we expressed a version of HERPUD1 with its ubiquitin-like domain (UBL) deleted, which is known to be crucial for its proteasome degradation. In comparison to HERPUD1-WT, we found the UBL-deleted version caused a negative role on basal and induced macroautophagy. Unexpectedly, we found stabilized HERPUD1 promotes ER remodeling independent of unfolded protein response activation observing an increase in stacked-tubular structures resembling previously described tubular ER rearrangements. Importantly, a phosphomimetic S59D mutation within the UBL mimics the phenotype observed with the UBL-deleted version including an increase in HERPUD1 stability and ER remodeling together with a negative role on autophagy. Moreover, we found UBL-deleted version and HERPUD1-S59D trigger an increase in cellular size, whereas HERPUD1-S59D also causes an increased in nuclear size. Interestingly, ER remodeling by the deletion of the UBL and the phosphomimetic S59D version led to an increase in the number and function of lysosomes. In addition, the UBL-deleted version and phosphomimetic S59D version established a tight ER-lysosomal network with the presence of extended patches of ER-lysosomal membrane-contact sites condition that reveals an increase of cell survival under stress conditions. Altogether, we propose stabilized HERPUD1 downregulates macroautophagy favoring instead a closed interplay between the ER and lysosomes with consequences in drug-cell stress survival.

### Keywords
Starvation, Atg5, Cultured human cells

### Affiliations
University of Dundee
Centre for Gene Regulation and Expression, Life Sciences Research Complex, University of Dundee, Dundee, UK

### Submitter
Mike Tatham

### Lab Head
Dr Ronald T. Hay
Centre for Gene Regulation and Expression, Life Sciences Research Complex, University of Dundee, Dundee, UK


