### Accession
PXD005697

### Title
Proteomics of tissue matrix scaffold derived from decellularized mice mammary tissues, LC-MS/MS

### Description
This project is intended to identify the compositions of the tissue matrix scaffold (TMS) generated from the extracellular matrix (ECM) of decellularized NON-SCID mice breast tissues (DBT). Since ECM is the physicochemical support for the cells living in the tissue microenvironment, it is important to know the protein contents within the ECM that are critical for extra- and intracellular signaling, cell growth, migration, cell-cell/ECM interactions. The LC-MS/MS data will serve as a foundation for the downstream applications of the TMS system, and be compared with those of the Matrigel mass spectrometry.

### Sample Protocol
Breast/mammary tissue from 8 – 12 weeks old NON-SCID mice was collected and decellularized using the SDS detergent (0.5%) for 48 hrs replacing the SDS in every 12 hrs and further treated with the isopropyl alcohol for 48 hrs replacing it in every 12 hrs. It was washed completely using the 1X PBS and lyophilized at -50 °C for 24 – 48 hrs. The lyophilized DBT was solubilized in 8 M Urea containing 0.07% ProteaseMAX (Promega), 50 mM NH4HCO3 (pH 8.5) and 10 mM TrisHCl (pH 7.0) to the concentration of ~10 mg/ml. Samples were sonicated in a sonicator bath three times for 1 minute each, then stored overnight at 4°C to facilitate further solubilization. 20 µl (~200 µg) of homogenously reconstituted samples was taken for the downstream 400 µl digestion, where sample was diluted to 240 μl for reduction step with 10 μl of 25 mM DTT, 115 μl of 25 mM NH4HCO3 (pH 8.5), 20 μl of MeOH, 40 μl of 8 M Urea and 35 μl of 0.2% ProteaseMAX. Incubated at 52 °C for 15 minutes, cooled on ice to room temperature. Then, 12 μl of 55 mM IAA was added for alkylation and incubated in dark at room temperature for 15 minutes. Reaction was quenched by adding 32 μl of 25 mM DTT. Subsequently the sample was split into two portions. One portion was digested without further handling whereas the second portion was treated with 6 µl of PNGase F enzyme (Promega) at 37 °C for 2 hrs. For protease digestion, 20 μl of Trypsin/LysC mix solution [50 ng/μl Trypsin from Promega and 50 ng/μl LysC from WAKO in 25 mM NH4HCO3] and 40 μl of 25 mM NH4HCO3 (pH 8.5) were added to a 200 µl of final volume. Digestion was conducted at 42 °C for 2 hrs, followed by addition of 10 µl of trypsin/LysC solution, and further digested at 37 °C overnight. Reaction was terminated by acidification with 2.5% TFA [Trifluoroacetic Acid] (0.3% final concentration). 50 µg of the final digest was cleaned up using OMIX C18 SPE cartridges (Agilent, Palo Alto, CA) per manufacturer protocol, eluted in 20 µl of 60/40/0.1% ACN/H2O/TFA, completely dried in speed-vac, and reconstituted in 25 µl of 0.1% formic acid.  Peptides were analyzed by nanoLC-MS/MS (Biotechnology Center, University of Wisconsin-Madison) using the Agilent 1100 nanoflow system (Agilent) connected to a new generation hybrid linear ion trap-orbitrap mass spectrometer (LTQ-Orbitrap Elite™, Thermo Fisher Scientific) equipped with an EASY-Spray™ electrospray source.  Chromatography of peptides prior to mass spectral analysis was accomplished using capillary emitter column (PepMap® C18, 3 µM, 100 Å, 150x0.075 mm, Thermo Fisher Scientific) onto which 2 µl of extracted peptides was automatically loaded. NanoHPLC system delivered solvents A: 0.1% (v/v) formic acid and B: 99.9% (v/v) acetonitrile with 0.1% (v/v) formic acid at 0.50 µL/min to load the peptides (over a period of 30 minutes), and 0.3 µl/min to elute peptides directly into the nano-electrospray with gradual gradient from 3% (v/v) B to 20% (v/v) B over 154 minutes, and concluded  with 12 minutes fast gradient from 20% (v/v) B to 50% (v/v) B at which time a 5 minute flash-out from 50-95% (v/v) B took place. As peptides eluted from the HPLC-column/electrospray source, MS scans were acquired in the Orbitrap with a resolution of 120,000 followed by MS2 fragmentation of 20 most intense peptides detected in the MS1 scan from 380 to 1800 m/z; redundancy was limited by dynamic exclusion.

### Data Protocol
Raw MS/MS data were converted to mgf file format using MSConvert (ProteoWizard: Open Source Software for Rapid Proteomics Tools Development). The resulting mgf files were used to search against Mus musculus amino acid sequence database with a decoy reverse entries and a list of common contaminants (87,154 total entries with 43,539 mouse proteins from UniProt database downloaded 09_18_2014) using in-house Mascot search engine 2.2.07 (Matrix Science) with variable Methionine and Proline oxidation, and with Asparagine and Glutamine deamidation. Peptide mass tolerance was set at 15 ppm and fragment mass at 0.6 Da. Protein annotations, significance of identification and spectral based quantification was done with the help of Scaffold software (version 4.3.2, Proteome Software Inc., Portland, OR). Protein identifications were accepted if they could be established at greater than 80.0% probability within 1% False Discovery Rate and contained at least 2 identified peptides.  Protein probabilities were assigned by the Protein Prophet algorithm (Nesvizhskii, AI Anal Chem. 2003 Sep 1;75(17):4646-58). Proteins that contained similar peptides and could not be differentiated based on MS/MS analysis alone were grouped to satisfy the principles of parsimony.

### Publication Abstract
Most of the anticancer drug candidates entering preclinical trials fail to be approved for clinical applications. The following are among the main causes of these failures: studying molecular mechanisms of cancer development, identifying therapeutic targets, and testing drug candidates using inappropriate tissue culture models, which do not recapitulate the native microenvironment where the cancer cells originate. It has become clear that three-dimensional (3D) cell cultures are more biologically and clinically relevant than 2D models. The spatial and mechanical conditions of 3D cultures enable the cancer cells to display heterogeneous growth, assume diverse phenotypes, express distinct gene and protein products, and attain metastatic potential and resistance to drugs that are reminiscent of tumors in humans. However, the current 3D culture systems using synthetic polymers or selected components of the extracellular matrix (ECM) are defective (particularly the biophysical and biochemical properties of the native ECM) and remain distant to optimally support the signaling cue-oriented cell survival and growth. We introduce a reconstitutable tissue matrix scaffold (TMS) system fabricated using native tissue ECM, with tissue-like architecture and resilience. The structural and compositional properties of TMS favor robust cell survival, proliferation, migration, and invasion in culture and vascularized tumor formation in animals. The combination of porous and hydrogel TMS allows compartmental culture of cancerous and stromal cells, which are distinguishable by biomarkers. The response of the cancer cells grown on TMS to drugs well reflects animal and clinical observations. TMS enables more biologically relevant studies and is suitable for preclinical drug screening.

### Keywords
Decellularization, Lc-ms/ms, Breast/mammary tissue, Tissue matrix scaffold, Tms, Ecm

### Affiliations
Washington State University Spokane
Elson S. Floyd College of Medicine, Washington State University Spokane

### Submitter
Weimin Li

### Lab Head
Dr Weimin Li
Elson S. Floyd College of Medicine, Washington State University Spokane


