### Accession
PXD021281

### Title
Uncovering the secretome of mesenchymal stromal cells exposed to healthy, traumatic and 1 degenerative intervertebral discs: A proteomic analysis

### Description
Mesenchymal stromal cells (MSCs) have been introduced as promising cell source for regenerative medicine. Besides their multilineage differentiation capacity, MSCs release a wide spectrum of bioactive factors. This secretome holds immunomodulatory and regenerative capacities. In intervertebral disc (IVD) cells, application of MSC secretome has been shown to decrease the apoptosis rate, induce proliferation and promote production of extracellular matrix (ECM). For clinical translation of secretome-based treatment, characterization of the secretome composition is needed to better understand the induced biological processes and identify potentially effective secretomes. Methods: This study aimed to investigate the proteome released by bone marrow derived MSCs following exposure to a healthy, traumatic, or degenerative human IVD environment by mass spectroscopy and quantitative immunoassay analyses. Exposure of MSCs to the proinflammatory stimulus interleukin 1Î² (IL-1b) was used as control.

### Sample Protocol
MSC secretomes from all 12 donors for all treatment conditions (healthy, degenerative, traumatic, baseline and IL-1b) were analyzed. For each sample, the protein concentration was measured using the  Qubit Protein Assay Kit (Life Technologies, Switzerland). The samples were prepared by using the commercial iST digestion protocol together with the Phoenix peptide clean-up kit (PreOmics, Germany). (PreOmics, Germany). Mass spectrometry analysis was performed on a Q-Exactive  HF-X mass spectrometer (Thermo Scientific) equipped with a Digital PicoView source (New Objective) and coupled to an M-Class UPLC (Waters). Solvent composition at the two channels was 0.1% formic  acid for channel A and 0.1% formic acid, 99.9% acetonitrile for channel B. For each sample, 4 uL of peptides were loaded on a commercial MZ Symmetry C18 Trap Column (100 Angstrom, 5 um, 180 um x 20 mm,  Waters) followed by nanoEase MZ C18 HSS T3 Column (100 Angstrom, 1.8 um, 75 um x 250 mm, Waters). The peptides were eluted at a flow rate of 300 nL/min by a gradient from 8 to 27% B in 82 min, 35% B in 5  min and 80% B in 1 min. Samples were acquired in a randomized order. Only precursors with intensity above 110,000 were selected for MS/MS.

### Data Protocol
The acquired raw MS data were processed by MaxQuant (version 1.6.2.3), followed by protein identification using the integrated Andromeda search engine. Spectra were searched against a Uniprot human reference proteome (taxonomy 9906, canonical version from 2016-12-09), concatenated to its reversed decoyed fasta database and common protein contaminants.

### Publication Abstract
None

### Keywords
Human, Lc-ms, Ms1 quantification, Mesenchymal stromal cells

### Affiliations
Functional Genomics Center Zurch
Focus Area Leader Disc and Cartilage Biology, PhD, PD AO Research Institute Davos (ARI)

### Submitter
Witold Wolski

### Lab Head
Dr Sibylle Grad
Focus Area Leader Disc and Cartilage Biology, PhD, PD AO Research Institute Davos (ARI)


