### Accession
PXD015800

### Title
Mycoplasma pneuminiae and Mycoplasma agalactiae Proteomics

### Description
During evolution, each bacterial strain shapes its metabolism in order to colonise a diversity of niches. Unraveling the biochemical reactions underlying bacteria metabolism is important for biotechnological purposes and for understanding relationships within a complex microbiome as well as the microbiome’s connection with its host. Here we propose a new approach to identifying active metabolic pathways, by integrating essentiality analysis and protein abundance. As an example, we used two bacterial species (Mycoplasma pneumoniae and Mycoplasma agalactiae) that share a high gene similarity yet show significant metabolic differences. After integrating all available metabolic knowledge about their enzymes, metabolites and reactions, we built detailed metabolic maps of their carbon metabolism. We determined the carbon sources that allow growth in M. agalactiae (as known for M. pneumoniae) and introduced glucose-dependent growth in M. agalactiae. By analyzing gene essentiality and performing quantitative proteomics, we could predict the active metabolic pathways and directionalities for the sugar, phospholipids, DNA/RNA precursors, glycoproteins, and glycolipids metabolism of these two bacteria. Comparison between predicted and experimentally determined active pathways shows an excellent agreement. Thus, protein essentiality profiling using transposon sequencing analysis combined with quantitative proteomics and metabolic maps could be used to determine and engineer metabolic fluxes.

### Sample Protocol
Preparing samples for mass spectrometry analysis. For M. pneumoniae, 50 µg of cells were inoculated in two T25 flask with 5 ml Hayflick medium and grown for 48 h. Cells were then washed twice with PBS and scraped for final collection in a 1.5 ml collection tube. For M. agalactiae, 5 ml cultures inoculated in Falcon tubes were grown for 48 h in standard Hayflick medium. Cultures were then centrifuged and washed twice in PBS. For both species, pelleted cells were then resuspended ten times with a 25-gauge needle to separate individual cells and resuspended in 150 μl of urea lysis buffer (6 M urea, 0.2 M NH4CO3) or SDS lysis buffer (4% SDS, 100 mM HEPES). For samples resuspended in urea buffer, two biological replicates were performed for each species. The protein concentration of each lysate was measured using the Pierce BCA Protein kit following manufacturer’s protocol. For urea samples, 10 µg samples were reduced with 30 nmol dithiothreitol (37 ºC, 60 min) and alkylated in the dark with 60 nmol iodoacetamide (25ºC, 30 min). The resulting protein extract was first diluted to 2 M urea with 200 mM ammonium bicarbonate for digestion with endoproteinase LysC (1:10 w:w, overnight at 37ºC; Wako, cat # 129-02541) and then diluted 2-fold with 200 mM ammonium bicarbonate for trypsin digestion (1:10 w:w, 8 h at 37ºC; Promega, cat # V5113). Samples with SDS lysis buffer were reduced with 90 nmol dithiothreitol (30 min at 56°C) and alkylated in the dark with 180 nmol iodoacetamide (30 min at 25°C). Following the filter-aided sample preparation (FASP) method described in (Wiśniewski 2016), samples were then digested with 3 μg LysC (Wako, cat # 129‐02541) overnight at 37°C, and then with 3 μg of trypsin (Promega, cat # V5113) for eight hours at 37°C.  After digestion, peptide mixes were acidified with formic acid and desalted with a MicroSpin C18 column (The Nest Group, Inc) prior to LC-MS/MS analysis.  Chromatographic and mass spectrometric analyses. Samples were analyzed using a LTQ-Orbitrap Velos Pro-mass spectrometer (Thermo Fisher Scientific, San Jose, CA, USA) coupled to an EASY-nLC 1000 (Thermo Fisher Scientific (Proxeon), Odense, Denmark). Peptides were loaded onto a 2-cm Nano Trap column with an inner diameter of 100 μm packed with 18C particles of 5 μm particle size (Thermo Fisher Scientific) and separated by reversed-phase chromatography using a 25-cm column with an inner diameter of 75 μm, packed with 1.9 μm 18C particles (Nikkyo Technos Co., Ltd. Japan). Chromatographic gradients started at 93% buffer A, 7% buffer B with a flow rate of 250 nl/min for 5 min, and gradually increased to 65% buffer A, 35% buffer B over 120 min. After each analysis, the column was washed for 15 min with 10% buffer A, 90% buffer B. Buffer A was 0.1% formic acid in water, and buffer B, 0.1% formic acid in acetonitrile. The mass spectrometer was operated in positive ionization mode with nano spray voltage set at 2.1 kV and source temperature at 300°C. Ultramark 1621 was used for external calibration of the FT mass analyzer prior to analyses, and an internal calibration was performed using the background polysiloxane ion signal at m/z 445.1200. Acquisition was performed in data-dependent acquisition (DDA) mode, and full MS scans with 1-micro scans at a 60,000 resolution were used over a mass range of m/z 350–2000 with detection in the Orbitrap. Auto gain control (AGC) was set to 1 × 106, dynamic exclusion (60 seconds) and charge state filtering disqualifying singly charged peptides was activated. In each cycle of DDA analysis, following each survey scan, the top twenty most intense ions with multiple charged ions above a threshold ion count of 5000 were selected for fragmentation. Fragment ion spectra were produced via collision-induced dissociation (CID) at normalized collision energy of 35% and they were acquired in the ion trap mass analyzer. AGC was set to 1 × 104, and an isolation window of 2.0 m/z, an activation time of 10 ms, and a maximum injection time of 100 ms were used. All data were acquired with Xcalibur software v2.2. Digested bovine serum albumin (New England Biolabs cat # P8108S) was analyzed between each sample run to avoid sample carryover and to assure stability of the instrument, and QCloud (Chiva et al, 2018) was used to control instrument longitudinal performance during the project.

### Data Protocol
Acquired spectra were analyzed using the Proteome Discoverer software suite (v2.0, Thermo Fisher Scientific) and the Mascot search engine (v2.6, Matrix Science (Perkins et al, 1999)). Data were searched against a M. agalactiae strain 7784 database (76,678 entries) or M. pneumoniae M129 database (87,070 entries) plus a list of common contaminants and all the corresponding decoy entries (Perkins et al, 1999; Beer et al, 2017). For peptide identification, a precursor ion mass tolerance of 7 ppm was used for the MS1 level, trypsin was used as an enzyme, and up to three missed cleavage sites were allowed. The fragment ion mass tolerance was set to 0.5 Da for MS2 spectra. Oxidation of methionine and N-terminal protein acetylation were used as variable modifications, with carbamidomethylation on cysteines set as a fixed modification. False discovery rate (FDR) in peptide identification was set to a maximum of 5%. Peptide quantification data were retrieved from the “Precursor ion area detector” node from Proteome Discoverer (v2.0) using 2 ppm mass tolerance for the peptide extracted ion current (XIC). The obtained values were used to calculate protein top 3 area with the unique peptide for ungrouped proteins.

### Publication Abstract
Here, we propose an approach to identify active metabolic pathways by integrating gene essentiality analysis and protein abundance. We use two bacterial species (Mycoplasma pneumoniae and Mycoplasma agalactiae) that share a high gene content similarity yet show significant metabolic differences. First, we build detailed metabolic maps of their carbon metabolism, the most striking difference being the absence of two key enzymes for glucose metabolism in M.&#xa0;agalactiae. We then determine carbon sources that allow growth in M.&#xa0;agalactiae, and we introduce glucose-dependent growth to show the functionality of its remaining glycolytic enzymes. By analyzing gene essentiality and performing quantitative proteomics, we can predict the active metabolic pathways connected to carbon metabolism and show significant differences in use and direction of key pathways despite sharing the large majority of genes. Gene essentiality combined with quantitative proteomics and metabolic maps can be used to determine activity and directionality of metabolic pathways.

### Keywords
Mycoplasma pneumoniae, Mycoplamsa agalactiae

### Affiliations
Centre for Genomic Regulation
Design of Biological Systems Lab, Centre for Genomic Regulation

### Submitter
Samuel Miravet Verde

### Lab Head
Dr Luis Serrano
Design of Biological Systems Lab, Centre for Genomic Regulation


