### Accession
PXD032800

### Title
Proteomics analysis of tumor antigens in allogenic tumor cell lysate and pancreatic tumor tissue.

### Description
Deep-proteome profiling of allogenic cell lysates and pancreatic ductal adenocarcinoma (PDAC) tissue sample with the aim to detect relevant tumor antigens (as described in Cheever et al. 2009, and directly linked to Lau et al., 2022).

### Sample Protocol
**Sample digest and labelling**  Volumes corresponding to 100 µg protein amount were dissolved in 2% SDC, 100 mM triethylammonium bicarbonate (TEAB), and 10 mM dithiothreitol DTT and filled up with HPLC grade-water to a total of 229 µL. Samples were heated for 2 minutes at 95 °C, intensively sonified for 2 minutes at 70% intensity, and incubated for further 30 minutes at 56 °C under mild shaking conditions. Then, samples were loaded in an Amicon Ultra-0.5 Centrifugal Filter Unit (Ultracel-30) and washed three times with 0.5% SDC, 5% acetonitrile (ACN), and 100 mM TEAB. Next, 15 mM Iodoacetamid was added, samples were incubated at darkness for 30 minutes, then first washed twice with (0.5% SDC, 5% ACN and 100 mM TEAB), and additionally washed twice with 200 mM TEAB. For sample digestion, 4 µg trying (Trying Gold, Pr omega) was added and samples were incubated at 37 °C overnight. Finally, the digest solution was spun down and a small aliquot of 1.8 µL was acidified and diluted (10x), centrifuged and used for nano-LC testruns, whereby UV absorbance was detected to determine total peptide abundance. For peptide labeling with Tandem Mass Tags (TMT), a normalized volume correspondig to 30 µg digest was taken and filled up with 200 mM TEAB to a total volume of 75 µL. Then, TMTsixplex reagents were diluted in 100 µL ACN and 31 µL label reagent was added to each sample. Samples were incubated for 1 hour at 20 °C under mild shaking conditions, and thereafter reaction was quenched by the addition of 2.7 µL 5% Hydroxylamine (Thermo Fisher Scientific). Finally, labelled samples were pooled into three pools, acidified, dried (speedvac), resuspended in 2% ACN/0.5% TFA and transfered to LC vials for the subsequent preparative fractionation. If not noted otherwise, all reagents were purchased from Sigma-Aldrich/Merck. High pH reversed phase fractionation Preparative chromatography was conducted on an Ultimate 3000 LC system equipped with C18 reversed phase column (Kinetex EVO, PN 00F-4725-AN, Phenomenex) operated at an oven temperature of 40 °C. Peptides were separated by a binary gradient from 4% to 38% solvent B in 8 minutes at a flow rate of 450 uL/min, whereby solvent A was composed of 10 mM ammonium formate buffer pH 10 and solvent B was 80% ACN and 10 mM ammonium formate pH 10. Twenty-four fractions of 200 µL were collected, dried, resuspended in 2% ACN/0.1% TFA, split in two aliquots of 25 µL and transferred to a heat-sealed 384 wellplate, where it was stored at 4 °C until subsequent LC-MS analysis.  **LC-MS measurements**  LC-MS measurements were performed on a nano-LC system (Ultimate 3000 RSLC) coupled to an Orbitrap Eclipse Tribrid Mass Spectrometer equipped with a High Field Asymmetric Waveform Ion Mobility Spectrometry (FAIMS) interface (Thermo Fisher Scientific). Twenty-four µL peptide fraction were injected and transferred onto a trap column (C18 PepMap, 300 µm ID x 5 mm; Thermo Fisher Scientific) using 0.1% trifluoroacetic acid at a flow rate of 20 µl/min, and further eluted and separated on a 50 cm analytical nano-LC column (PepMap C18, 75 µm ID x 500 mm, 2 µm, 100 Å; Thermo Fisher Scientific) using a binary 3 hours gradient from 4% to 24% solvent B in 120 minutes and further increasing to 45% B in 60 minutes, whereby aqueous solvent A was 0.1% formic acid, solvent B 80% acetonitrile and 0.08% formic acid, flow rate 300 nL/min and column temperature 40°C. Additionally, the early fractions (fraction 1 - 5, and 8) and the last fraction (fraction 24) that showed in general comparable low peptide amounts, were measured by a shorter 60 minutes gradient, with otherwise identical LC parameters. For electrospray ionization we used coated silica nano electro-spray emitters (New Objective, Woburn, MA, USA) at a spray voltage of 2.2 kV. FAIMS was setup to collect ion mobility fractions at compensation voltages (CV) of -40 , -60 and -80 V.

### Data Protocol
Acquired RAW data was processed with MSaccess and MSconvert (proteowizard version 3.0.19263, Chambers et al., 2012) to extract scan meta information and fragment ion peak lists as MGF files, respectively. For peptide and protein identification we conducted a Mascot fragment ion database search (v.2.301; Matrix Science) and subsequently determined false-discovery rates, protein grouping and exporting of the generated mass spectrometry results (spectra report) with the software package Scaffold (version 5.10; Proteome Software), whereby following settings were applied: combined human subsets of Swiss-Prot and TrEMBL (download: January 18th 2021, 194,237 entries), decoy search, carbamidomethylation (+57.021 u) of cysteine and TMT6plex labelling (+229) of lysin and the peptide n-terminus as fixed modification, oxidation (+ 15.995 u) of methionine, 10 ppm precursor ion tolerance and 0.5 Da fragment ion tolerance, and 0.1% peptide and 1% protein false discovery rate threshold.

### Publication Abstract
The data described was acquired as part of a clinical study with the aim to investigate the potential of tumor-reactive T-cell response as response to vaccination of pancreatic cancer patients with an allogenic tumor cell lysate vaccine (Lau et&#xa0;al., 2022). Proteomics analysis was carried out to identify tumor antigens that are shared between the allogeneic tumor cell lysate used for the vaccine and pancreatic ductal adenocarcinoma (PDAC) tissue samples. To this objective, cell lysates of the vaccine and of nine tissue samples were enzymatically digested and isotopically labeled with tandem mass tags (TMT) in a so-called six-plex manner (Thermo Fisher Scientific). Three pools were prepared by mixing the samples according to their TMT-labels. Subsequently, the three sample pools were fractionated into 24 fractions with high-pH reversed phase chromatography. These fractions were first analyzed on a nano-liquid chromatography (LC) system online coupled to a high-resolution Eclipse Orbitrap mass spectrometer (MS) equipped with a high-field asymmetric-waveform ion-mobility spectrometry (FAIMS) source using a data-dependent MS2 shotgun method. Overall, 126,618 unique peptide sequences, on basis of 768,638 peptide spectra matches and corresponding to 7,597 protein groups, were identified in the total sample set including 61 tumor antigens (Supplement Table S2 in Lau et&#xa0;al. 2022) that were prioritized by Cheever and co-workers as vaccine target antigens on basis of a series of objective criteria (Cheever et&#xa0;al., 2009). In the second phase of the experiment, this set of tumor antigens was targeted using a serial precursor selection (SPS) MS3 method. From this data, ion trap MS2 and Orbitrap MS3 fragment spectra were extracted for peptide identification (protein sequence database-dependent search) and relative quantification using the TMT labels, respectively. The dataset ultimately allowed the identification and quantification of 51 proteins and 163 related peptide precursors with the TMT labels (see Fig. 2B and Supplemental Fig. 8, Lau et&#xa0;al. 2022).

### Keywords
Human, Tumor antigens, Mudpit

### Affiliations
Head of Laboratory of Neuro-Oncology, Neurology, Clinical & Cancer Proteomics
Erasmus MC Rotterdam
Department of Neurology

### Submitter
Christoph Stingl

### Lab Head
Dr Theo M. Luider
Head of Laboratory of Neuro-Oncology, Neurology, Clinical & Cancer Proteomics


