### Accession
PXD021630

### Title
Olfactory proteomics in amyotrophic lateral sclerosis

### Description
Despite smell impairment has been considered a potential non-motor finding in ALS, the pathobiochemistry at olfactory level remains unknown. Here, we applied an olfactory quantitative proteotyping approach to analyze the magnitude of the olfactory bulb (OB) proteostatic imbalance in ALS subjects (n=12) respect to controls (n=8).

### Sample Protocol
Sample preparation for proteomic analysis  Whole OB specimens (70-80 mg) derived from controls and ALS cases were homogenized using ‘’mini potters’’ in lysis buffer containing 7 M urea, 2 M thiourea, 50 mM DTT. The homogenates were spun down at 100.000 x g for 1 hour at 15 ºC. Before proteomic analysis, protein extracts were precipitated and pellets were dissolved in 6M Urea and Tris 100 mM pH 7.8. Protein quantitation was performed with the Bradford assay kit (Bio-Rad). The protein extract for each sample was diluted in Laemmli sample buffer and loaded into a 0.75 mm thick polyacrylamide gel with a 4% stacking gel casted over a 12.5% resolving gel. The run was stopped as soon as the front entered 3 mm into the resolving gel so that the whole proteome became concentrated in the stacking/resolving gel interface. Bands were stained with Coomassie Brilliant Blue and excised from the gel. Protein enzymatic cleavage (10 ug) was carried out with trypsin (Promega; 1:20, w/w) at 37 ºC for 16 hours as previously described (Shevchenko et al., 2006). Peptides were purified and concentrated using C18 Zip Tip Solid Phase Extraction (Millipore, Burlington, MA, USA). Label free LC-MS/MS Mass spectrometric analyses were performed on an EASY-nLC 1200 liquid chromatography system interfaced with a Q Exactive HF-X mass spectrometer (Thermo Scientific) via a nanospray flex ion source. 1 ug of tryptic peptides were loaded onto an Acclaim PepMap100 precolumn (75 um x 2 cm, Thermo Scientific) connected to an Acclaim PepMap RSLC (75 um x 25 cm, Thermo Scientific) analytical column. Peptides were eluted from the column at a flow rate of 250 nL min-1 using the following percentage of buffer B in the gradient (buffer A: 0.1% formic acid, buffer B: 80% acetonitrile, 0.1% formic acid): 3 min 5%, 7 min 5-7%, 105 min 7-22%, 25 min 22-27%, 15 min 27-31%, 28 min 31-53%, 1 min 53-95%. The mass spectrometer was operated in positive ion mode and a data-dependent acquisition mode was used. The spray voltage was set at 1.9 keV, the capillary temperature at 300°C and the S-Lens RF level at 50. Full MS scans were acquired from m/z 375 to 1800 with a resolution of 60,000 at m/z 200. The 15 most intense ions were fragmented by higher energy C-trap dissociation with normalized collision energy of 28 and MS/MS spectra were recorded with a resolution of 15,000 at m/z 200. The maximum ion injection time was 45 ms for survey and for MS/MS scans, whereas AGC target values of 3 x 106 and 5 x 105 were used for survey and MS/MS scans, respectively. A dynamic exclusion time of 40 s was applied and singly charged ions, ions with 6 or more charges, and ions with unassigned charge state were excluded from MS/MS. Data were acquired using Xcalibur software (Thermo Scientific).

### Data Protocol
Protein identification and quantification Mass spectrometry raw data were processed using the MaxQuant software (v.1.6.3.3) (Cox J et al. Nature Biotechnology 2008) following the next parameters. Data were searched against the Homo Sapiens UniProtKB database (February 2019) also containing frequent contaminants and the reversed version of all sequences. In the group-specific parameters, main peptide search and first search tolerance were set as 4.5ppm and 20ppm, respectively. In addition, trypsin was selected as enzyme with a maximum of two missed cleavages, and methionine oxidation and N-terminal acetylation were selected as variable modifications whereas carbamidomethylation was selected as fixed. In the global parameters, the minimum peptide length was set in 7 amino acids, fragment mass deviation in 40ppm and false discovery rate (FDR) for peptide spectrum match (PSM), peptide and protein identification were set in 1%. The output file ‘’proteingroups.txt’’ generated by MaxQuant was then analysed with the Perseus software (v.1.6.2.3) (Tyanova S et al. Nat Methods 2016). Potential contaminants and proteins identified as reversed were removed. Protein identification was considered valid with at least two unique or razor peptides whereas protein quantification was calculated using at least two unique peptides. Statistical significance was calculated by a two-way Student T-test (p<0.05) and a 1.3-fold change cut-off was used. Thus, proteins with ratios below the low range of 0.77 were considered downregulated whereas those above the high range 1.33 were considered up-regulated. Data visualization was also performed with Perseus.

### Publication Abstract
Amyotrophic lateral sclerosis (ALS) is a fatal disease characterized by progressive muscle paralysis due to the degeneration of upper and lower motor neurons. Recent studies point out an involvement of the non-motor axis during disease progression. Despite smell impairment being considered a potential non-motor finding in ALS, the pathobiochemistry at the olfactory level remains unknown. Here, we applied an olfactory quantitative proteotyping approach to analyze the magnitude of the olfactory bulb (OB) proteostatic imbalance in ALS subjects (<i>n</i> = 12) with respect to controls (<i>n</i> = 8). Around 3% of the quantified OB proteome was differentially expressed, pinpointing aberrant protein expression involved in vesicle-mediated transport, macroautophagy, axon development and gliogenesis in ALS subjects. The overproduction of olfactory marker protein (<i>OMP</i>) points out an imbalance in the olfactory signal transduction in ALS. Accompanying the specific overexpression of glial fibrillary acidic protein (<i>GFAP</i>) and Bcl-xL in the olfactory tract (OT), a tangled disruption of signaling routes was evidenced across the OB-OT axis in ALS. In particular, the OB survival signaling dynamics clearly differ between ALS and frontotemporal lobar degeneration (FTLD), two faces of TDP-43 proteinopathy. To the best of our knowledge, this is the first report on high-throughput molecular characterization of the olfactory proteostasis in ALS.

### Keywords
Amyotrophic lateral slcerosis, Proteomics, Olfactory marker protein, Olfactory bulb

### Affiliations
Proteomics unit, Proteored-ISCIII Navarrabiomed, Pamplona, Spain
Proteomics unit

### Submitter
Enrique SantamarÃ­a

### Lab Head
Dr Enrique Santamaria
Proteomics unit, Proteored-ISCIII Navarrabiomed, Pamplona, Spain


