### Accession
PXD028429

### Title
Proteomics of serum samples from COVID-19 ARDS and bacterial ARDS patients

### Description
SARS-CoV-2 is a novel coronavirus that causes acute respiratory distress syndrome (ARDS), death and long-term sequelae. Innate immune cells are critical for host defense but are also the primary drivers of ARDS. The relationships between innate cellular responses in ARDS resulting from COVID-19 compared to other causes of ARDS, such as bacterial sepsis is unclear. Moreover, the beneficial effects of dexamethasone therapy during severe COVID-19 remain speculative, but understanding the mechanistic effects could improve evidence-based therapeutic interventions. To interrogate these relationships, we developed an scRNAseq atlas that is freely accessible (biernaskielab.ca/COVID_neutrophil). We discovered that compared to bacterial ARDS, COVID-19 was associated with distinct neutrophil polarization characterized by either interferon (IFN) or prostaglandin (PG) active states. Neutrophils from bacterial ARDS had higher expression of antibacterial molecules such as PLAC8 and CD83. Dexamethasone therapy in COVID patients rapidly altered the IFNactive state, downregulated interferon responsive genes, and activated IL1R2+ve neutrophils. Dexamethasone also induced the emergence of immature neutrophils expressing immunosuppressive molecules ARG1 and ANXA1, which were not present in healthy controls. Moreover, dexamethasone remodeled global cellular interactions by changing neutrophils from information receivers into information providers. Importantly, male patients had higher proportions of IFNactive neutrophils and a greater degree of steroid-induced immature neutrophil expansion. Indeed, the highest proportion of IFNactive neutrophils was associated with mortality. These results define neutrophil states unique to COVID-19 when contextualized to other life-threatening infections, thereby enhancing the relevance of our findings at the bedside. Furthermore, the molecular benefits of dexamethasone therapy are also defined. The identified molecular pathways can now be targeted to develop improved therapeutics.

### Sample Protocol
The serum of COVID-19 patients (severe = 5, dexamethasone-treated = 4) and septic controls (N = 6) were collected and subjected to quantitative proteomics. The total protein concentrations were determined by Pierce‚Ñ¢ BCA Protein Assay Kit (23225, ThermoFisher). A trichloroacetic acid (TCA)/acetone protocol was used to pellet 100¬µg of proteins per sample. Samples were subjected to a quantitative proteomics workflow as per supplier (Thermo Fisher) recommendations. Samples were reduced in 200mM tris(2-carboxyethyl)phosphine (TCEP), for 1h at 55¬∞C, reduced cysteines were alkylated by incubation with iodoacetamide solution (50mM) for 20min at room temperature. Samples were precipitated by acetone/methanol, and 600ÔÅ≠L ice-cold acetone was added followed by incubation at -20¬∞C overnight. A protein pellet was obtained by centrifugation (8,000ùëî, 10min, 4¬∞C) followed by acetone drying (2min). Precipitated pellet was resuspended in100 ÔÅ≠L of 50mM triethylammonium bicarbonate (TEAB) buffer followed by tryptase digestion (5ÔÅ≠g trypsin per 100ÔÅ≠g of protein) overnight at 37¬∞C. TMT-6plex‚Ñ¢ Isobaric Labeling Reagents (90061, Thermo Fisher) were resuspended in anhydrous acetonitrile and added to each sample (41ÔÅ≠L TMT-6plex‚Ñ¢ per 100ÔÅ≠L sample) and incubated at room temperature for 1h. The TMT labeling reaction was quenched by 2.5% hydroxylamine for 15min at room temperature. TMT labeled samples were combined and acidified in 100% trifluoroacetic acid to pH < 3.0 and subjected to C18 chromatography (Sep-Pak) according to manufacturer recommendations. Samples were stored at -80¬∞C before lyophilization, followed by resuspension in 1% formic acid before liquid chromatography and tandem mass spectrometry analysis. Tryptic peptides were analyzed on an Orbitrap Fusion Lumos Tribrid mass spectrometer (Thermo Scientific) operated with Xcalibur (version 4.0.21.10) and coupled to a Thermo Scientific Easy-nLC (nanoflow liquid chromatography) 1200 System. Tryptic peptides (2ÔÅ≠g) were loaded onto a C18 trap (75ÔÅ≠m x 2cm; Acclaim PepMap 100, P/N 164946; ThermoFisher) at a flow rate of 2ÔÅ≠L/min of solvent A (0.1% formic acid in LC-MS grade H2O). Peptides were eluted using a 120min gradient from 5 to 40% (5% to 28% in 105min followed by an increase to 40% B in 15min) of solvent B (0.1% formic acid in 80% LC-MS grade acetonitrile) at a flow rate of 0.3ÔÅ≠L/min and separated on a C18 analytical column (75ÔÅ≠m x 50cm; PepMap RSLC C18; P/N ES803A; ThermoScientific). Peptides were then electrosprayed using 2.1kV voltage into the ion transfer tube (300¬∞C) of the Orbitrap Lumos operating in positive mode. For LC-MS/MS measurements with the FAIMS Pro (Thermo Fisher Scientific), multiple compensation voltages (CV) were applied, -40V, -60V, and -80V with a cycle time of 1 second. FAIMS was used to generate technical replicates from plex 1 to 6. The Orbitrap first performed a full MS scan at a resolution of 120,000 FWHM to detect the precursor ion having a m/z between 375 and 1,575 and a +2 to +4 charge. The Orbitrap AGC (Auto Gain Control) and the maximum injection time were set at 4 x 105 and 50ms, respectively. The Orbitrap was operated using the top speed mode with a 3 second cycle time for precursor selection. The most intense precursor ions presenting a peptidic isotopic profile and having an intensity threshold of at least 2 x 104 were isolated using the quadrupole (Isolation window (m/z) of 0.7) and fragmented using HCD (38% collision energy) in the ion routing multipole. The fragment ions (MS2) were analyzed in the Orbitrap at a resolution of 15,000. The AGC and the maximum injection time were set at 1 x 105 and 105ms, respectively. The first mass for the MS2 was set at 100 to acquire the TMT reporter ions. Dynamic exclusion was enabled for 45 seconds to avoid of the acquisition of same precursor ion having a similar m/z (plus or minus 10ppm).

### Data Protocol
Spectral data acquired from the mass spectrometer were matched to peptide sequences using MaxQuant software (v.1.6.14). Due to a lack of direct compatibility with Maxquant, spectra generated using the FAIMS pro was first converted to MzXML using the FAIMS MzXML Generator from the Coon‚Äôs lab. Next, peptide sequences from the human proteome and Sars-CoV-2 proteins were obtained from the UniProt database (May 2021) and matched using the Andromeda algorithm at a peptide-spectrum match false discovery rate (FDR) of 0.05. Search parameters included a mass tolerance of 20 p.p.m. for the parent ion, 0.5 Da for the fragment ion, carbamidomethylation of cysteine residues (+57.021464 Da), variable N-terminal modification by acetylation (+42.010565 Da), and variable methionine oxidation (+15.994915 Da). Relative quantification was set as TMT 6-plex labels 126 to 131. The cleavage site specificity was set to Trypsin/P, with up to two missed cleavages allowed. Next, the evidence.txt and proteinGroups.txt were loaded into the R software (v4.0.2) for the statistical analysis. The normalization and identification of differentially expressed proteins was performed using the MSstatsTMT package. Multiple comparisons were corrected using the Benjamini-Hochberg approach.

### Publication Abstract
None

### Keywords
Human, Lc-msms, Maxquant, Covid-19, Bacterial ards, Serum, Neutrophil, Orbitrap

### Affiliations
University of Calgary
Department of Comparative Biology and Experimental Medicine, Faculty of Veterinary Medicine, University of Calgary; Hotchkiss Brain Institute, University of Calgary; Alberta Children‚Äôs Hospital Research Institute, University of Calgary

### Submitter
Luiz Gustavo de Almeida

### Lab Head
Dr Jeff Biernaskie
Department of Comparative Biology and Experimental Medicine, Faculty of Veterinary Medicine, University of Calgary; Hotchkiss Brain Institute, University of Calgary; Alberta Children‚Äôs Hospital Research Institute, University of Calgary


