### Accession
PXD007145

### Title
Benchmarking LFQ, SILAC and MS2/MS3-based TMT quantification strategies for large-scale phosphoproteomics

### Description
Comprehensive mass spectrometry (MS)-based proteomics is now feasible, but reproducible and multiplexed quantification remains challenging especially for analysis of post-translational modifications (PTMs), such as phosphorylation. Here we compared the most popular quantification techniques for phosphoproteomics in context of cell-signaling studies: label-free quantification (LFQ), stable isotope labeling by amino acids in cell culture (SILAC) and MS2- and MS3-measured tandem mass tags (TMT). In a mixed species comparison with fixed phosphopeptide-ratios, we found LFQ and SILAC to be the most accurate techniques. MS2-based TMT suffered from substantial ratio compression, which MS3-based TMT could partly rescue. However, when analyzing phosphoproteome changes in the DNA damage response (DDR), we found that MS3-based TMT was outperformed by MS2-based TMT as it identified most significantly regulated phosphopeptides due to its higher precision and higher number of identifications. Finally, we show that the high accuracy of MS3-based TMT is crucial for determination of phosphorylation site stoichiometry using a novel multiplexing-dependent algorithm.

### Sample Protocol
Human epithelial cervix carcinoma HeLa cells (female) and human epithelial osteosarcoma U2OS cells (female) were treated with DMSO, 5uM doxorubicin, 4-nitroquinoline 1-oxide or nothing for 2h. BY4742 wild-type yeast cells were frozen into droplets using liquid N2 and grinded in a ball mill. Yeast pellet powder was lysed with 1% Triton X-100 and proteins were precipitated in -80C acetone. Precipitated yeast proteins and human cells were resolved/lysed in 6M guanidine hydrochloride, with TCEP, CAA, protease and phosphatase inhibitors added. After 10min 95C, DNA was sheared by ultrasonication. For SILAC experiments, cells were labeled with light, medium and heavy lysine combined with light arginine only. Proteins were digested with LysC only for yeast experiments, or LysC and trypsin. SILAC peptides for DDR analysis were fractionated on an Ultimate 3000 HPLC into ten fractions. Phosphopeptides were enriched using TiO2 beads pre-incubated in DHB. TMT phosphopeptides for DDR analysis were fractionated on an Ultimate 3000 HPLC into 24 fractions. All samples were analyzed on Easy-nLC 1000 coupled to a Q-Exactive HF or Orbitrap Fusion Lumos, both equipped with nanoelectrospray source. Peptides were separated on a 15cm or 50cm analytical column. The MS instruments were run with different gradients and instrument methods detailed in the publication.

### Data Protocol
All raw LC-MS/MS data were processed with MaxQuant v1.5.5.4i using the Andromeda search engine and searched against the complete human UniProt database including all Swiss-Prot entries, and in case of mixed human/yeast samples additionally with the complete yeast UniProt database including all Swiss-Prot entries. In addition, the default contaminant protein database was included. The â€œmatch between runsâ€� (MBR) and SILAC requantify (REQ) features were activated where indicated. Data sets for LFQ, SILAC, MS2- and MS3-based TMT, as well as LFQ-MBR and SILAC-MBR REQ were kept in individual MaxQuant analysis groups. Carbamidomethylation of cysteine was specified as fixed modification for all groups. Variable modifications considered were oxidation of methionine, protein N-terminal acetylation, and phosphorylation of serine, threonine and tyrosine residues. The false discovery rate (FDR) was set to 1% on PSM, PTM site and Protein level.

### Publication Abstract
Comprehensive mass spectrometry (MS)-based proteomics is now feasible, but reproducible quantification remains challenging, especially for post-translational modifications such as phosphorylation. Here, we compare the most popular quantification techniques for global phosphoproteomics: label-free quantification (LFQ), stable isotope labeling by amino acids in cell culture (SILAC) and MS<sup>2</sup>- and MS<sup>3</sup>-measured tandem mass tags (TMT). In a mixed species comparison with fixed phosphopeptide ratios, we find LFQ and SILAC to be the most accurate techniques. MS<sup>2</sup>-based TMT yields the highest precision but lowest accuracy due to ratio compression, which MS<sup>3</sup>-based TMT can partly rescue. However, MS<sup>2</sup>-based TMT outperforms MS<sup>3</sup>-based TMT when analyzing phosphoproteome changes in the DNA damage response, since its higher precision and larger identification numbers allow detection of a greater number of significantly regulated phosphopeptides. Finally, we utilize the TMT multiplexing capabilities to develop an algorithm for determining phosphorylation site stoichiometry, showing that such applications benefit from the high accuracy of MS<sup>3</sup>-based TMT.

### Keywords
Phosphoproteomics, Quantification, Lfq, Technical, Benchmark, Tmt, Silac, Ms3, Ms2

### Affiliations
Department of Genome Sciences
University of Washington, Seattle, USA

Novo Nordisk Foundation Center for Protein Research, Proteomics Program, University of Copenhagen, Denmark

### Submitter
Alexander Hogrebe

### Lab Head
Dr Jesper V. Olsen
Novo Nordisk Foundation Center for Protein Research, Proteomics Program, University of Copenhagen, Denmark


