### Accession
PXD033411

### Title
Bacillus licheniformis meets ulvan: a marine polysaccharide as potential industrial feedstock

### Description
Proteomes of Bacillus licheniformis DSM13 cultivated with glucose, rhamnose as well as ulvan or two ulvan hydrolysates (UHA and UHB). Hydrolysates were generated using selected Formosa agariphila KM3901 ulvan-PUL encoded enzymes.

### Sample Protocol
For proteome analyses, late logarithmic phase cells from triplicates of ulvan, UHA, UHB, rhamnose and glucose cultures were separated from supernatants by centrifugation (20 min, 4,000 x g, 4°C). Intracellular soluble proteins were extracted by suspending cells pellets in lysis buffer (4% SDS, 1% NaDCA, 50mM TEAB) adapted to Hinzke 2017. Samples were incubated for 5 min at 600 rpm and 95 °C, then cooled on ice shortly and sonicated for 5 min. Cell debris was removed from protein extract (intracellular soluble proteome) by centrifugation (10 min, 14,000 x g, room temperature). Protein concentration was determined using the Pierce™ BCA Protein Assay Kit (Thermo Fisher Scientific). Secreted and detached proteins (extracellular proteome) were extracted from cultivation supernatants using StrataClean beads (Agilent) (Bonn 2014). In brief, 20 µl of bead solution were used to extract approximately 20 – 30 µg of protein. Beads were primed in 180 µl 37% hydrochloric acid (100°C, 6 h) and then washed in TE buffer (50 mM Tris, 10 mM EDTA, pH 8.0) twice (5 min, 3,500 x g, room temperature). 0.2 µm-filtered supernatants were incubated with prepared beads overnight in a 360° rotating shaker at 8 rpm and 4°C. The protein-loaded beads were pelleted by centrifugation (45 min, 10,000 x g, 4°C) and washed in TE buffer. In a last step, they were resuspended in 1 ml of ultrapure water and dried by vacuum centrifugation. 25 µg of protein from intracellular soluble protein extracts as well as protein-loaded beads were separated by 1D SDS PAGE (12 % SDS gels) at 120 V. Proteins were in-gel digested using trypsin (Bonn 2014). Peptides were separated by reversed phase chromatography and analyzed in an LTQ-Orbitrap Classic mass spectrometer equipped with a nanoelectrospray ion source (Otto 2010).

### Data Protocol
MS/MS spectra were searched against a target decoy database using MaxQuant v. 1.6.10.43 (Cox 2008, Tyanova 2016). The database covered all protein sequences predicted from the Bacillus licheniformis DSM13 genome, selected Formosa agariphila KM3901T ulvan-encoded enzymes and common laboratory contaminants as well as corresponding reversed sequences (decoys). The MaxQuant computed iBAQ values (intensity-based absolute quantification, Schwanhäusser 2011) were used to manually calculate %riBAQ values, giving the relative protein abundance in % per sample. Quantified F. agariphila KM3901T proteins were excluded from %riBAQ calculations. Proteins quantified in at least two out of the three replicates were considered for further calculations and for statistical tests. Welch’s two-sided t-test (permutation-based FDR 0.05) identified statistical significance to protein abundance differences between samples within the intracellular soluble proteome samples and within the extracellular samples using Perseus v. 1.6.0.7 (Tyanova 2016). Only samples with a similar number of quantified proteins were compared.

### Publication Abstract
None

### Keywords
Bacillus licheniformis, Lc-ms/ms, Ulvan

### Affiliations
University Greifswald
Institute of Pharmacy, Greifswald, Germany

### Submitter
Marie-Katherin Zühlke

### Lab Head
Dr Thomas Schweder
Institute of Pharmacy, Greifswald, Germany


