### Accession
PXD005858

### Title
Exploiting induced pluripotent stem cell-derived macrophages to unravel host factors influencing Chlamydia trachomatis pathogenesis

### Description
Chlamydia trachomatis remains a leading cause of bacterial sexually transmitted infections and preventable blindness worldwide. There are, however, limited in vitro models to study the role of host genetics in the response of macrophages to this obligate human pathogen. Here, we describe an approach using macrophages derived from human induced pluripotent stem cells (iPSdMs) to study macrophage-Chlamydia interactions in vitro. We show that iPSdMs support the full infectious life cycle of C. trachomatis in a manner that mimicks the infection of human blood-derived macrophages. Transcriptomic and proteomic profiling of the macrophage response to chlamydial infection highlights the role of the type I interferon and interleukin 10-mediated responses. Using CRISPR/Cas9 technology, we generate biallelic knockout mutations in the host genes encoding IRF5 and IL-10RA in iPSCs, confirming their roles in limiting chlamydial infection in macrophages. This model can potentially be extended to other pathogens and tissue systems to advance our understanding of host-pathogen interactions and the role of human genetics in influencing the outcome of infections.

### Sample Protocol
For proteomics , protein pellets were dissolved in 0.1M triethylammonium bicarbonate (TEAB), 0.1% SDS. Total proteins were reduced with tris-2-carboxymethyl phosphine (TCEP) and cysteine residues were blocked with Iodacetamide (IAA). Samples were digested with Trypsin (Pierce, MS grade) and the resultant peptides were labelled with TMT6plex (Thermo Scientific), and pooled. Offline peptide fractionation based on high pH Reverse Phase (RP) chromatography was performed using the Waters, XBridge C18 column on a Dionex Ultimate 3000 HPLC system equipped with autosampler. Signal was recorded at 280 nm and fractions were collected in a time dependent manner. LC-MS analysis was performed on the Dionex Ultimate 3000 UHPLC system coupled with the Orbitrap Fusion Tribrid Mass Spectrometer (Thermo Scientific). Each peptide fraction was reconstituted in 0.1% formic acid and loaded to the Acclaim PepMap 100 and subjected to a multi-step gradient elution on the Acclaim PepMap RSLC C18 capillary column (Dionex) retrofitted to an electrospray emitter (New Objective, FS360-20-10-D-20). Precursors between 400-1500 m/z were selected with mass resolution of 120k, AGC 3Ã—105 and IT 100 ms were isolated for CID fragmentation with quadrupole isolation width 0.7 Th. MS3 quantification spectra were acquired with further HCD fragmentation of the top 10 most abundant CID fragments isolated with Synchronous Precursor Selection (SPS) excluding neutral losses of maximum m/z 30. The HCD MS3 spectra were acquired with resolution of 15k. Targeted precursors were dynamically excluded for further isolation and activation for 45 seconds with 7 ppm mass tolerance.

### Data Protocol
The acquired mass spectra were submitted to SequestHT search engine implemented on the Proteome Discoverer 1.4 software for protein identification and quantification. The precursor mass tolerance was set at 20 ppm and the fragment ion mass tolerance was set at 0.5 Da. Spectra were searched for fully tryptic peptides with a maximum of 2 mis-cleavages and a minimum length of 6 amino acids. TMT6plex at N-termimus, K and Carbamidomethyl at C were defined as static modifications. Dynamic modifications included oxidation of M and deamidation of N, Q. A maximum of two different dynamic modifications were allowed for each peptide with maximum two repetitions each. Peptide confidence was estimated with the Percolator node. Peptide FDR was set at 0.01 and validation was based on q-value and decoy database search. All spectra were searched against a UniProt fasta file containing 20k human reviewed entries. The Reporter Ion Quantifier node included a custom TMT 6plex Quantification Method with integration window tolerance 20 ppm and integration method the Most Confident Centroid at the MS3 level.

### Publication Abstract
Chlamydia trachomatis remains a leading cause of bacterial sexually transmitted infections and preventable blindness worldwide. There are, however, limited in vitro models to study the role of host genetics in the response of macrophages to this obligate human pathogen. Here, we describe an approach using macrophages derived from human induced pluripotent stem cells (iPSdMs) to study macrophage-Chlamydia interactions in vitro. We show that iPSdMs support the full infectious life cycle of C. trachomatis in a manner that mimics the infection of human blood-derived macrophages. Transcriptomic and proteomic profiling of the macrophage response to chlamydial infection highlighted the role of the type I interferon and interleukin 10-mediated responses. Using CRISPR/Cas9 technology, we generated biallelic knockout mutations in host genes encoding IRF5 and IL-10RA in iPSCs, and confirmed their roles in limiting chlamydial infection in macrophages. This model can potentially be extended to other pathogens and tissue systems to advance our understanding of host-pathogen interactions and the role of human genetics in influencing the outcome of infections.

### Keywords
Chlamydia, Lc-msms, Tmt, Ips cells

### Affiliations
Wellcome Trust Sanger Institute
Functional Proteomics, Institute Cancer Research
&
Proteomic Mass Spectrometry, Wellcome Trust Sanger Institute

### Submitter
James Wright

### Lab Head
Dr Jyoti Choudhary
Wellcome Trust Sanger Institute


