### Accession
PXD009142

### Title
Generation and Proteome Profiling of PBMC-originated, iPSCs-derived Corneal Endothelial Cells

### Description
Corneal endothelial cells (CECs) are critical to maintaining clarity of the cornea. This study was initiated to develop peripheral blood mononuclear cells (PBMC)-originated induced pluripotent stem cells (iPSCs)-derived CECs. We isolated PBMC and programmed the mononuclear cells to generate iPSCs. Subsequently, the PBMC-originated iPSCs were differentiated to CECs. The morphology of differentiating iPSCs was examined at regular intervals by phase contrast microscopy. In parallel, the expression of pluripotent, and CECs-associated markers was investigated by quantitative real-time PCR (qRT-PCR). The molecular architecture of the iPSCs-derived CECs and human corneal endothelium (CE) were examined by mass spectrometry-based proteome sequencing. The PBMC-originated iPSCs expressed pluripotent-specific markers at levels similar to expression in H9 human embryonic stem cells (hESCs). Phase contrast microscopy illustrated that iPSCs-derived CECs are tightly adherent, exhibiting a hexagonal-like shape, one of the cardinal characteristics of CECs. The CECs-associated markers were expressed at many orders of magnitude higher in iPSCs-derived CECs at days 13, 20, and 30 compared to their respective levels in iPSCs. Importantly, only residual expression levels of pluripotency markers were detected in iPSCs-derived CECs. Mass spectrometry-based proteome profiling identified 10,575 proteins in iPSCs-derived CECs. In parallel, we completed proteome profiling of the human CE identifying 6345 proteins. Of these, 5763 proteins were identified in the iPSCs-derived CECs suggesting a 90.82% overlap between the iPSCs-derived CECs and human CE proteomes. Importantly, cryopreservation of iPSCs-derived CECs did not affect the tight adherence of CECs, and their hexagonal-like shape while expressing high levels of CECs-associated markers. We have successfully developed a personalized approach to generate CECs that closely mimic the molecular architecture of the human CE. To the best of our knowledge, this is the first report describing the development of PBMC-originated, iPSCs-derived CECs.

### Sample Protocol
Briefly, 10ml of human peripheral blood was used to isolate peripheral blood mononuclear cells (PBMCs). Subsequently, the cryopreserved PBMC were reprogramed into iPSCs using Sendai-virus delivery system Cytotune 2.0 kit (Life Technologies, Carlsbad, CA). Finally, the iPSCs were further differentiated into CECs. The differentiated CECs were harvested on day 20 (D20) for mass spectrometry-based proteome profiling. Protein extraction, and digestion: Human iPSCs-derived CECs at D20 from two 35-mm culture plates were harvested, pooled together and subjected to a mass spectrometry-based label-free quantitative proteomics. Briefly, sample processing, including protein isolation, digestion, and analysis on liquid chromatography-mass spectrometry (LC-MS/MS) was performed with little modifications as described previously. Briefly, the sample was lysed by sonicating in 8 M urea and 50 mM triethylammonium bicarbonate (TEAB). Protein lysates were centrifuged at 16,000g at 4 °C to exclude cell debris (pelleting at the bottom), and protein concentration was estimated using a bicinchoninic acid (BCA) assay. Total protein sample (2,600 µg) was reduced with 10 mM dithiothreitol at room temperature for one hour and alkylated with 30 mM iodoacetamide for 20 minutes in the dark. The protein sample was digested with LysC (1:100) by incubating at room temperature for three hours followed by overnight digestion at 37 °C using sequencing-grade trypsin (1:50) and finally desalted with C18 Sep-Pak (Waters Corporation, Milford, MA). The peptides were fractionated by basic pH reversed-phase liquid chromatography into 96 fractions, followed by concatenation into 24 fractions by combining every 24th fraction. Mass spectrometry analysis: The fractionated peptides were analyzed on an Orbitrap Fusion Lumos Tribrid Mass Spectrometer coupled with the Easy-nLC 1200 nano-flow liquid chromatography system (Thermo Fisher Scientific). The peptides from each fraction were reconstituted in 15 μl 0.1% formic acid and all the 15 μl were loaded on an Acclaim PepMap100 Nano-Trap Column (100 μm × 2 cm, Thermo Fisher Scientific) packed with 5 μm C18 particles at a flow rate of 5 μl per minute. Peptides were resolved at 250-nl/min flow rate using a linear gradient of 10% to 35% solvent B (0.1% formic acid in 95% acetonitrile) over 95 minutes on an EASY-Spray column (50 cm x 75 µm ID, Thermo Fisher Scientific) packed with 2 µm C18 particles, which was fitted with an EASY-Spray ion source that was operated at a voltage of 2.0 kV.  Mass spectrometry analysis was completed in a data-dependent manner with a full scan in the mass-to-charge ratio (m/z) range of 300 to 1800 in the “Top Speed” setting, three seconds per cycle. MS1 and MS2 were acquired for the precursor ions and peptide fragmentation ions and measured at a resolution of 120,000 at an m/z of 200. MS2 scan was acquired by fragmenting precursor ions using higher-energy collisional dissociation (HCD) method and detected at a mass resolution of 30,000, at an m/z of 200. Automatic gain control for MS1 was set to one million ions and for MS2 was set to 0.05 million ions. A maximum ion injection time was set to 50 milliseconds for MS1 and 100 milliseconds for MS2 (HCD was set to 32 with stepped collision energy of 5% for MS2). Precursor isolation window was set to 1.6 m/z with 0.4 m/z offset. Dynamic exclusion was set to 30 seconds, and singly-charged ions were rejected. Internal calibration was carried out using the lock mass option (m/z 445.1200025) from ambient air.

### Data Protocol
MaxQuant (v1.5.3.8.) software was used for quantitation and identification. During MS/MS preprocessing, the top 12 peaks in each window of 100 m/z were selected for database search. The tandem mass spectrometry data were then searched using Andromeda search algorithms against a UniProt (Swiss-Prot only) database (released on Apr 10, 2017) with common contaminant proteins. Search parameters included, a) trypsin as a proteolytic enzyme with up to two missed cleavages; b) first search peptide mass error tolerance of 20 ppm and the main search peptide mass error tolerance of 4 ppm; c) fragment mass error tolerance of 20 ppm;  d) carbamidomethylation of cysteine (+57.02146 Da) as a fixed modification: e) oxidation of methionine (+15.99492 Da) and protein acetyl (+ 42.01056 Da) on N-terminus as a dynamic modifications. The minimum peptide length was set to seven amino acids and the minimum number of peptide sequence per protein was set to two peptides. Peptides and proteins were filtered at 1 % false-discovery rate while the label-free quantification, the iBAQ, and the second peptides parameters were enabled. A minimum ratio count was set to two and both unique and razor peptides were used for quantification. After protein identification and quantification was completed, the protein table was imported into Perseus 1.5.2.6 software for the copy number calculation using Proteomic Ruler.

### Publication Abstract
None

### Keywords
Proteome, Mass-spectrometry, Pbmc, Ipscs-derived corneal endothelial cells

### Affiliations
The Wilmer Eye Institute, Johns Hopkins University School of Medicine, Baltimore, Maryland 21287 USA
Johns Hopkins Wilmer Eye Institute

### Submitter
OM Genetics

### Lab Head
Dr S. Amer Riazuddin
The Wilmer Eye Institute, Johns Hopkins University School of Medicine, Baltimore, Maryland 21287 USA


