### Accession
PXD033107

### Title
A photo-crosslink approach to identify class II SUMO binders

### Description
The small ubiquitin-like modifier (SUMO) is involved in various cellular processes and mediates known non-covalent protein-protein interactions by three distinct binding surfaces, whose interactions are termed class I to class III. While interactors for the class I interaction, which involves binding of a SUMO-interacting motif (SIM) to a hydrophobic groove in SUMO-1 and SUMO-2/3, are widely abundant, only a couple of examples have been reported for the other two types of interactions. Class II binding is conveyed by the E67 loop region on SUMO-1. Many previous studies to identify SUMO binders using pull-down or microarray approaches did not strategize on the SUMO binding mode. Identification of SUMO binding partners is further complicated due to the typically transient and low affinity interactions with the modifier. Here we aimed to identify SUMO binders selectively enriched for class II binding. Using a genetically encoded photo-crosslinker approach, we have designed SUMO-1 probes to covalently capture class II SUMO interactors by strategically positioning the photo crosslinking moiety on the SUMO-1 surface. The probes were validated using known class II and class I binding partners. We utilized the probe with p-benzoyl-phenylalanine (BzF, also termed BpF or Bpa) at the position of Gln69 to identify binding proteins from mammalian cell extracts using mass spectrometry. By comparison with results obtained with a similarly designed SUMO-1 probe to target SIM-mediated binders of the class I type, we identified 192 and 96 proteins specifically enriched by either probe, respectively. The implicated preferential class I or class II binding modes of these proteins will further contribute to unveiling the complex interplay of SUMO-mediated interactions.

### Sample Protocol
Biotinylated SUMO probe (15 ¬µM) was mixed with HEK293T whole cell extract of cells previously transfected with HA-DPP9 or HeLa S3 whole cell extract (8 mg/mL, supplemented with HALT protease inhibitor cocktail, Thermo Scientific) and incubated for 15 min at 4 ¬∞C. Half of the sample was UV-irradiated for 1 h (ùúÜ=365 nm, Herolab UV-16L, 8W, 6 mm distance) at room temperature (termed UV(+)). The other half was incubated without UV irradiation for 1h at RT (termed UV(-)). Similarly, ‚Äòbead control‚Äô samples were prepared without a SUMO probe. After incubation with streptavidin Sepharose beads (GE Healthcare, 17-5113-01) for 2 h at 4 ¬∞C, a stringent washing protocol was applied using spin columns (Pierce Micro-Spin Columns,10510824) and the following buffers: buffer A (8 M urea, 2% SDS, 100 mM Tris, 200 mM NaCl, pH 8), buffer B (8 M urea, 0.2% SDS, 10% ethanol, 10% isopropanol, 100 mM Tris, 1.2 M NaCl, pH 8), buffer C and D (8 M Urea, 0.2% SDS, 10% ethanol, 10% isopropanol, 100 mM Tris, 200 mM NaCl, pH5 and pH 9), buffer E (100 mM Tris, 200 mM NaCl, pH 8), and buffer F (50 mM NH4HCO3). An on-bead tryptic digest was performed after reduction (5 mM DTT, 30 min, 56 ¬∞C), alkylation (25 mM 2-iodoacetamide [IAA], 20 min, RT) and quenching (15 mM DTT) through the addition of 200 ng trypsin (dissolved in 50 mM NH4HCO3) and ProteaseMax (Promega Corp.). After overnight incubation at 37 ¬∞C in a rotation wheel, the supernatant was vacuum-dried, resuspended in acetonitrile/water (2% v/v)and acidified with formic acid (0.1% v/v).LC-MS/MS analysis was performed using an UltiMate‚Ñ¢ 3000 RS LC nano system (Thermo Fisher Scientific GmbH, Dreieich, Germany) connected to a maXis II UHR-qTOF mass spectrometer with a nano-ESI source (CaptiveSpray with nanoBooster, Bruker Daltonik GmbH, Bremen, Germany). The sample (1 ¬µL) was loaded and desalted for 20 min on a C18 trapping column (Thermo Acclaim PepMap100, C18, 5 Œºm, 0.1 mm x 20 mm, Thermo Fisher Scientific GmbH, Dreieich, Germany) at a flow rate of 5 ŒºL/min in 2% eluent B (eluent A: 0.1% formic acid in water; eluent B: 0.1% formic acid in acetonitrile). During this 20 min period, a sodium formate solution (0.5 mM in 50% 2 propanol) was injected for subsequent internal calibration of the resulting data files. The sample was separated on a C18 nano column applying a 150 min gradient (2% to 42% B, flow rate 300 nL/min). MS settings: capillary voltage 1.400 V, mass range: m/z 150-2200. The UV(+) proteomic samples were measured with four technical replicates. The bead control and UV(-) proteomic samples were measured with two to four technical replicates. MS survey scans were performed with a cycle time of 5 s. After each survey scan, the 20 to 30 most abundant precursor ions with z > 1 were selected for fragmentation using collision-induced dissociation. The MS/MS spectra rate was adjusted depending on the precursor intensity. The precursor isolation window and the collision energy were depending on the precursor m/z and charge.

### Data Protocol
Raw files were analysed using MaxQuant software v1.6.17 and Uniprot complete human proteome (UP000005640). Oxidation and N-terminal acetylation were allowed as variable modification and Carbamidomethylation was set as fixed modification with a maximum number of 5 modifications per peptide. Trypsin/P digestion was selected with a maximum of 2 missed cleavages. Match between runs was enabled with a match time window of 0.7 min and an alignment time window of 20 min. For the analysis of UV(+) samples, label-free quantification (LFQ) with an LFQ minimum ratio count of 2 was performed. Fast LFQ was disabled. Unique and razor peptides were considered for quantification.

### Publication Abstract
The small ubiquitin-like modifier (SUMO) is involved in various cellular processes and mediates known non-covalent protein-protein interactions by three distinct binding surfaces, whose interactions are termed class I to class III. While interactors for the class I interaction, which involves binding of a SUMO-interacting motif (SIM) to a hydrophobic groove in SUMO-1 and SUMO-2/3, are widely abundant, only a couple of examples have been reported for the other two types of interactions. Class II binding is conveyed by the E67 loop region on SUMO-1. Many previous studies to identify SUMO binders using pull-down or microarray approaches did not strategize on the SUMO binding mode. Identification of SUMO binding partners is further complicated due to the typically transient and low affinity interactions with the modifier. Here we aimed to identify SUMO-1 binders selectively enriched for class II binding. Using a genetically encoded photo-crosslinker approach, we have designed SUMO-1 probes to covalently capture class II SUMO-1 interactors by strategically positioning the photo-crosslinking moiety on the SUMO-1 surface. The probes were validated using known class II and class I binding partners. We utilized the probe with p-benzoyl-phenylalanine (BzF, also termed BpF or Bpa) at the position of Gln69 to identify binding proteins from mammalian cell extracts using mass spectrometry. By comparison with results obtained with a similarly designed SUMO-1 probe to target SIM-mediated binders of the class I type, we identified 192 and 96 proteins specifically enriched by either probe, respectively. The implicated preferential class I or class II binding modes of these proteins will further contribute to unveiling the complex interplay of SUMO-1-mediated interactions.

### Keywords
P-benzoyl-phenylalanine, Genetic code expansion, Sumo, Chemical crosslinking, Ubiquitin-like protein, Mass spectrometry

### Affiliations
Institute of Biochemistry, University of Muenster, Corrensstra√üe 36, 48149 Muenster
Institute of Biochemistry, University of Muenster, Corrensstra√üe 36, 48149 Muenster 

### Submitter
Kira Br√ºninghoff

### Lab Head
Dr Henning D. Mootz
Institute of Biochemistry, University of Muenster, Corrensstra√üe 36, 48149 Muenster


