### Accession
PXD038771

### Title
Grad-MS PHIKZ phage infected Pseudomonas aeruginosa PAO1 cells

### Description
Protein complexes in phage infected cells probed by MS.

### Sample Protocol
Cellular lysates of Phage infected cells were analysed on a 10-40% glycerol gradient and fractionated. Fractions were spiked with serum albumin and denatured in SDS-PAGE loading dye. 30-90 OD Cells were resuspended in 800 µl lysis buffer (20 mM Tris/HCl pH 7.4, 150 mM KCl, 2.5 mM MgCl2, 1 mM dithiothreitol (DTT), 0.1% (v/v) Triton X100, 1 mM phenylmethylsulfonyl fluoride (PMSF)). Cells were lysed in 2 ml FastPrep tubes with lysing matrix E (MP Biomedicals) for 2× 40 s FastPrep-24 instrument (MP Biomedicals) at 6 m/s at 4 °C. The lysate was cleared at 13,000 × g for 10 min and layered onto a 10-40% glycerol gradient in lysis buffer without PMSF. The gradient was centrifuged for 17 h at 100,000 × g and 4 °C. 20 590 µl fractions were collected and the pellet was resuspended extensively in the last one. Samples were mixed with ¼ volume 4X Bolt™ SDS sample buffer (Invitrogen) and were boiled for 5 min at 95 °C. 2 pmol human albumin was spiked-in per 50 µl fraction for subsequent normalisation. MS sample preparation and measurement was conducted at the Proteomics Core Facility EMBL Heidelberg by Jennifer Schwarz. Samples were reduced and alkylated with 10 mM dithiothreitol (DTT) at 56 °C for 30 min, and 2-chloroacetamide at room temperature in the dark for 30 min, respectively. Samples were cleaned-up using the SP3 protocol (Hughes et al., 2019). 300 ng sequencing grade Trypsin (Promega) in 50 mM ammonium bicarbonate was added for overnight digestion at 37 °C. Peptides were recovered by collecting the supernatant on a magnet followed by a second elution with ultrapure water. Samples were dried under vacuum centrifugation and reconstituted in 10 μl 1% formic acid, 4% acetonitrile and then stored at -80 °C until LC-MS analysis.

### Data Protocol
An UltiMate 3000 RSLC nano LC system (Dionex) fitted with a trapping cartridge (µ-Precolumn C18 PepMap 100, 5 µm, 300 µm i.d. × 5 mm, 100 Å) and an analytical column (nanoEase™ M/Z HSS T3 column 75 µm x 250 mm C18, 1.8 µm, 100 Å, Waters) was coupled directly to a QExactive Plus (Thermo) mass spectrometer using the Nanospray Flex™ ion source in positive ion mode. Trapping was carried out with a constant flow of 0.05% trifluoroacetic acid at 30 µl/min onto the trapping column for 4 min. Subsequently, peptides were eluted via the analytical column with a constant flow of 0.3 µl/min with increasing percentage of solvent B (0.1% formic acid in acetonitrile) from 2% to 4% in 4 min, from 4% to 8% in 2 min, then 8% to 25% for a further 89 min, and finally from 25% to 40% in another 17 min and from 40% to 80% in 3 min. The peptides were introduced into the QExactive plus via a Pico-Tip Emitter 360 µm OD × 20 µm ID; 10 µm tip (MSWIL) and an applied spray voltage of 2.2 kV. The capillary temperature was set at 275 °C. Full mass scans were acquired with mass range 350-1400 m/z in profile mode with resolution of 70,000. The filling time was set at maximum of 20 ms with a limitation of 3x106 ions. Data dependent acquisition (DDA) was performed with the resolution of the Orbitrap set to 17,500, with a fill time of 50 ms and a limitation of 1x105 ions. A normalized collision energy of 26 was applied. Loop count 20. Isolation window 1.7 m/z. Dynamic exclusion time of 30 s was used. The peptide match algorithm was set to ‘preferred’ and charge exclusion ‘unassigned’, charge states 1, 5 - 8 were excluded. MS2 data was acquired in centroid mode. The raw mass spectrometry data was processed with MaxQuant (v1.6.17.0) (Cox and Mann) and searched against the database UP000002438, UP000002098 for Pseudomonas aeruginosa and ΦKZ phage, respectively. As an internal standard the entry P02768 (albumin of homo sapiens) was used in each experiment. Common contaminants were included in each search. Decoy mode was set to revert. Carbamidomethyl (C) was set as fixed modification, acetylation of N-termini and oxidation of methionine were set as variable modifications. The mass error tolerance for the full scan MS spectra was set to 20 ppm and for the MS/MS spectra to 0.5 Da. A maximum of two missed cleavages was allowed. For protein identification, a minimum of one unique peptide with a peptide length of at least seven amino acids and a false discovery rate (FDR) below 0.01 were required on the peptide and protein level. Match between runs was enabled with standard settings. Quantification was performed using intensities and iBAQ values (Schwanhäusser et al., 2011) calculated as the sum of the intensities of the identified peptides and divided by the number of observable peptides of a protein.

### Publication Abstract
None

### Keywords
Phikz, Pseudomonas, Phage

### Affiliations
Institute for Molecular Infection Biology, Julius Maximilian University Wuerzburg Helmholtz Institute for RNA-based Infection Research, Helmholtz Centre for Infection Research, Wuerzburg
Institute for Molecular Infection Biology, University of Würzburg, Würzburg, Germany

### Submitter
Milan Gerovac

### Lab Head
Dr Jörg Vogel
Institute for Molecular Infection Biology, Julius Maximilian University Wuerzburg Helmholtz Institute for RNA-based Infection Research, Helmholtz Centre for Infection Research, Wuerzburg


