### Accession
PXD021901

### Title
An NSD3-targeting PROTAC suppresses the NSD3 and cMyc oncogenic nodes in cancer cells

### Description
Nuclear receptor binding SET domain protein 3 (NSD3), a gene located within the 8p11-p12 amplicon frequently detected in cancers, encodes a chromatin modulator and an attractive onco-target. However, agent that can effectively suppress the NSD3-mediated oncogenic actions is currently lacking. We report an NSD3-targeting proteolysis targeting chimera (PROTAC), termed MS9715, which achieves effective and specific depletion of NSD3 and interacting partners (including cMyc) in tumor cells. MS9715-induced NSD3 degradation relies on BI-9321, an antagonist module binding the PWWP1 domain of NSD3, and VHL, which is chemically conjugated to BI-9321 via a linker and VHL ligand module. Importantly, compared to BI-9321, a recently disclosed NSD3 antagonist, MS9715 is more potent in suppressing growth of the NSD3-dependent hematological cancer including models of MLL-rearranged acute myeloid leukemia (AML) and B-cell acute lymphoblastic leukemia (B-ALL) and multiple myeloma (MM), and uniquely mediate simultaneous depletion of cellular NSD3 and cMyc. Transcriptome profiling further demonstrates effective actions of MS9715 but not BI-9321 in suppressing both NSD3- and cMyc-associated gene-expression programs, a phenomenon reminiscent of the CRISPR/cas9-mediated knockout (KO) of NSD3. Together, this study reports a first-in-class NSD3 PROTAC/degrader suitable for co-suppressing NSD3- and cMyc-related oncogenic nodes in cancer cells, suggesting a novel therapeutic strategy.

### Sample Protocol
Proteins were reduced, alkylated, and purified by chloroform/methanol extraction prior to digestion with sequencing grade modified porcine trypsin (Promega). Tryptic peptides were labeled using tandem mass tag isobaric labeling reagents (Thermo) following the manufacturerâ€™s instructions and combined into one 10-plex sample group. The labeled peptide multiplex was separated into 36 fractions on a 100 x 1.0 mm Acquity BEH C18 column (Waters) using an UltiMate 3000 UHPLC system (Thermo) with a 40 min gradient from 99:1 to 60:40 buffer A:B ratio under basic pH conditions, and then consolidated into 18 super-fractions. Each super-fraction was then further separated by reverse phase XSelect CSH C18 2.5 um resin (Waters) on an in-line 150 x 0.075 mm column using an UltiMate 3000 RSLCnano system (Thermo). Peptides were eluted using a 60 min gradient from 97:3 to 60:40 buffer A:B ratio.  Eluted peptides were ionized by electrospray (2.2 kV) followed by mass spectrometric analysis on an Orbitrap Eclipse Tribrid mass spectrometer (Thermo) using multi-notch MS3 parameters. MS data were acquired using the FTMS analyzer in top-speed profile mode at a resolution of 120,000 over a range of 375 to 1500 m/z. Following CID activation with normalized collision energy of 35.0, MS/MS data were acquired using the ion trap analyzer in centroid mode and normal mass range. Using synchronous precursor selection, up to 10 MS/MS precursors were selected for HCD activation with normalized collision energy of 65.0, followed by acquisition of MS3 reporter ion data using the FTMS analyzer in profile mode at a resolution of 50,000 over a range of 100-500 m/z.  Buffer A = 0.1% formic acid, 0.5% acetonitrile Buffer B = 0.1% formic acid, 99.9% acetonitrile Both buffers adjusted to pH 10 with ammonium hydroxide for offline separation

### Data Protocol
Protein TMT MS3 reporter ion intensity values are assessed for quality using our in-house ProteiNorm app, a user-friendly tool for a systematic evaluation of normalization methods, imputation of missing values and comparisons of different differential abundance methods (ref). Popular normalization methods were evaluated including log2 normalization (Log2), median normalization (Median), mean normalization (Mean), variance stabilizing normalization (VSN) (ref), quantile normalization (Quantile) (ref), cyclic loess normalization (Cyclic Loess) (ref), global robust linear regression normalization (RLR) (ref), and global intensity normalization (Global Intensity) (Chawade et al). The individual performance of each method was evaluated by comparing of the following metrices: total intensity, Pooled intragroup Coefficient of Variation (PCV), Pooled intragroup Median Absolute Deviation (PMAD), Pooled intragroup estimate of variance (PEV), intragroup correlation, sample correlation heatmap (Pearson), and log2-ratio distributions. The cyclic loess normalization performed the best on the data and was used to perform statistical analysis using Linear Models for Microarray Data (limma) with empirical Bayes (eBayes) smoothing to the standard errors (ref). Proteins with an FDR adjusted p-value < 0.05 and a fold change > 2 were considered to be significant.

### Publication Abstract
None

### Keywords
Degrader, Nsd3, Chromatin, Cmyc, Protac, Epigenetics, Histone

### Affiliations
1Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill School of Medicine, Chapel Hill, NC 27599, USA 2Department of Biochemistry and Biophysics, University of North Carolina at Chapel Hill School of Medicine, Chapel Hill, NC 27599, USA
UAMS

### Submitter
Stephanie Byrum

### Lab Head
Dr Gang Greg Wang
1Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill School of Medicine, Chapel Hill, NC 27599, USA 2Department of Biochemistry and Biophysics, University of North Carolina at Chapel Hill School of Medicine, Chapel Hill, NC 27599, USA


