### Accession
PXD003271

### Title
Proteomic analysis of clear cell renal cell carcinoma tissue versus matched normal kidney tissue

### Description
We analyzed 84 tumor/normal pairs (177 total) using standard shotgun proteomic techniques in order to characterize the molecular landscape of clear cell renal cell carcinoma (ccRCC) and interrogated changes in protein abundance and biological pathways with ccRCC grade. These tissues were distributed across stage 1 (n = 34), 2 (n = 40), 3 (n = 42), and 4 (n = 52), with 9 pairs also including samples from metastasized tumor.  These data can be (and were) combined with a previously published transcriptomic data set from the same sample cohort (NCBI GEO: GSE53757).

### Sample Protocol
Tissue (10 to 15 mg) was processed by trifluoroethanol (TFE) solubilization and sonication as adapted from Wang et al., 2005 (PMID:16335993). Briefly, tissue in 50% TFE was disrupted sequentially by repeated probe sonication and heat/vortexing (1 hr, 60Â°C), with at least two repeats. At this step, 10 Âµg of an internal protein standard, maltose-binding periplasmic protein from Escherichia coli, was added to the lysate. Reducing agent [tris(2-carboxyethyl)phosphine] and alkylating agent (iodoacetamide) were added, followed by overnight trypsin digestion (1:50 based on protein concentration) at 37Â°C. The resulting peptides were desalted with a C-18 column (1cm x 1cm), dried down by SpeedVac and reconstituted with Mobile Phase A [5% ACN, 0.1% formic acid (FA), 0.005% heptafluorobutyric acid (HFBA)].  Tumor-normal pairs were always analyzed by tandem mass spectrometry temporally close together (i.e., pairs were analyzed at the same time). Digests were resuspended in 20 ÂµL Mobile Phase A. A 15 Î¼L aliquot of this peptide solution was separated on a 12 cm x 0.075 mm fused silica capillary column packed with 5 Âµm diameter C-18 beads (The Nest Group, Southborough, MA) across a 90 min linear gradient from 5% ACN, 0.1% FA, 0.005% HFBA to 95% ACN, 0.1% FA, 0.005% HFBA at 300 nL/min. The LC was interfaced by electrospray ionization with an LTQ (ThermoFinnigan, San Jose, CA). Data-dependent analysis was used to perform MS/MS on the five most intense ions between m/z = 400 and 2000 in each MS spectra with a minimum signal of 1000 cps. Dynamic exclusion was used with a repeat count of two and an exclusion duration of 180 s.

### Data Protocol
Raw files from individual sample tandem mass spectrometry runs were converted to peak list (mgf format) using MSConvert (ProteoWizard 3.0.4243; Jan 3, 2013). The default parameters were used, including â€˜Prefer Vendor for Peak Pickingâ€™. These mgfs were searched with Mascot (v2.4.1; Matrix Sciences) using the following parameters: trypsin as the enzyme with a maximum of two mis-cleavages; 1+, 2+, and 3+ charged peptides; carbamidomethyl (C) as a fixed modification, and protein N-term acetylation, deamidation (NQ) and oxidation (M) as variable modifications; instrument type was ESI-TRAP; a precursor tolerance of 2 Da and fragment ion tolerance 0.5 Da. These thresholds were chosen based on a test analysis which resulted in a 0% local FDR (above identity threshold) while maintaining the highest number of protein hits. The database used was a Homo sapiens database (taxon ID: 9606) retrieved from the 2013_04 release of the UniProt SwissProt database along with the SwissProt varsplic database, a cRAP database (common Repository of Adventitious Proteins, v 2012.01.01; The Global Proteome Machine) and the entry for Maltose-binding periplasmic protein from Escherichia coli (P0AEX9; malE) resulting in 38,480 sequences.

### Publication Abstract
Renal cell carcinoma comprises 2 to 3% of malignancies in adults with the most prevalent subtype being clear-cell RCC (ccRCC). This type of cancer is well characterized at the genomic and transcriptomic level and is associated with a loss of VHL that results in stabilization of HIF1. The current study focused on evaluating ccRCC stage dependent changes at the proteome level to provide insight into the molecular pathogenesis of ccRCC progression. To accomplish this, label-free proteomics was used to characterize matched tumor and normal-adjacent tissues from 84 patients with stage I to IV ccRCC. Using pooled samples 1551 proteins were identified, of which 290 were differentially abundant, while 783 proteins were identified using individual samples, with 344 being differentially abundant. These 344 differentially abundant proteins were enriched in metabolic pathways and further examination revealed metabolic dysfunction consistent with the Warburg effect. Additionally, the protein data indicated activation of ESRRA and ESRRG, and HIF1A, as well as inhibition of FOXA1, MAPK1 and WISP2. A subset analysis of complementary gene expression array data on 47 pairs of these same tissues indicated similar upstream changes, such as increased HIF1A activation with stage, though ESRRA and ESRRG activation and FOXA1 inhibition were not predicted from the transcriptomic data. The activation of ESRRA and ESRRG implied that HIF2A may also be activated during later stages of ccRCC, which was confirmed in the transcriptional analysis. This combined analysis highlights the importance of HIF1A and HIF2A in developing the ccRCC molecular phenotype as well as the potential involvement of ESRRA and ESRRG in driving these changes. In addition, cofilin-1, profilin-1, nicotinamide N-methyltransferase, and fructose-bisphosphate aldolase A were identified as candidate markers of late stage ccRCC. Utilization of data collected from heterogeneous biological domains strengthened the findings from each domain, demonstrating the complementary nature of such an analysis. Together these results highlight the importance of the VHL/HIF1A/HIF2A axis and provide a foundation and therapeutic targets for future studies. (Data are available via ProteomeXchange with identifier PXD003271 and MassIVE with identifier MSV000079511.).

### Keywords
Human, Tissue, Renal cell carcinoma, Lc-ms/ms, Ccrcc, Kidney

### Affiliations
National Institute of Standards and Technology
Department of Cell and Molecular Pharmacology and Experimental Therapeutics, Medical University of South Carolina, Charleston, SC, USA

### Submitter
Benjamin Neely

### Lab Head
Dr Richard R. Drake
Department of Cell and Molecular Pharmacology and Experimental Therapeutics, Medical University of South Carolina, Charleston, SC, USA


