### Accession
PXD023902

### Title
Increased levels of the mitochondrial import factor Mia40 can prevent the aggregation of polyQ proteins in the cytosol

### Description
The formation of protein aggregates is a hallmark of neurodegenerative diseases. Observations on patient material and model systems demonstrated links between aggregate formation and declining mitochondrial functionality, but the causalities remained unclear. We used yeast as model system to analyze the relevance of mitochondrial processes for the behavior of an aggregation-prone polyQ protein derived from human huntingtin. Induction of Q97-GFP rapidly leads to insoluble cytosolic aggregates and cell death. Although these aggregates impaired mitochondrial respiration only slightly, they interfered with efficient import of mitochondrial precursor proteins. Mutants in the import component Mia40 were hypersensitive to Q97-GFP. Even more surprising, Mia40 overexpression strongly suppressed the formation of toxic Q97-GFP aggregates both in yeast and in human cells. Based on our observations, we propose that the posttranslational import into mitochondria competes with aggregation-prone cytosolic proteins for chaperones and proteasome capacity. Owing to its rate-limiting role for mitochondrial protein import, Mia40 acts as regulatory component in this competition with direct relevance for cytosolic proteostatic stability. Thus, targeting Mia40 might offer exciting therapeutic potential in the future. In this dataset we analyzed the soluble proteome of wild type and Mia40 overexpression yeast cells in response to the expression of polyQ proteins.

### Sample Protocol
To compare the soluble proteomes of wild type and GAL-Mia40 yeast strains which both have been expressing Q97-GFP for 4.5 h, as well as the changes in the proteomes of WT strains expressing either Q97-GFP or Q25-GFP, the expression of the polyQ constructs was induced for 0 h and 4.5 h. 50 ml of mid-log phase cell suspension were harvested by centrifugation at RT, 4,000 x g for 5 min and washed twice with PBS before freezing the pellet in liquid nitrogen and storing it at -80Â°C. The cells were resuspended in PBS with a volume in ml equal to 0.4 x OD600 (e.g., if OD600=0.5, add 200 Âµl; calculation: 0.5 x 0.4 = 0.2 ml). 40 Âµl of the cell suspension were transferred to a new tube to which 60 Âµl of cold 1.7x lysis buffer (0.8% NP40 in PBS, 1 mM MgCl2, 0.25 U/Âµl benzonase, complete Protease Inhibitor Cocktail, Phosphatase Inhibitor, PBS) were added. Cell lysates were prepared using a FastPrep-24 5G homogenizer (MP Biomedicals, Heidelberg, Germany) with 3 cycles of 30 s, speed 8.0 m/s, 120 s breaks, glass beads. To remove cell debris, the lysates were centrifuged at 4Â°C for 10 min at 16,100 x g. 40 Âµl of the supernatant were transferred onto a pre-wet 0.45 Âµm filter plate on a 96-well plate and centrifuged for 5 min at 500 x g, 4Â°C to collect the lysate free of Q97-GFP aggregates. The sample plate was sealed with a plate seal plastic foil, dipped in liquid nitrogen and frozen at -80Â°C until use. For LC-MS/MS analysis of lysates, protein concentration of lysates were determined by BCA protein determination. 10 Âµg of each lysate were subjected to an in-solution tryptic digest using a modified version of the Single-Pot Solid-Phase-enhanced Sample Preparation (SP3) protocol (Hughes, Foehr et al., 2014, Moggridge, Sorensen et al., 2018). Lysates were added to Sera-Mag Beads (Thermo Scientific) in 10 Âµl 15% formic acid and 30 Âµl of ethanol. Binding of proteins was achieved by shaking for 15 min at room temperature. SDS was removed by 4 subsequent washes with 200 Âµl of 70% ethanol. Proteins were digested overnight at room temperature with 0.4 Âµg of sequencing grade modified trypsin (Promega) in 40 Âµl Hepes/NaOH, pH 8.4 in the presence of 1.25 mM TCEP and 5 mM chloroacetamide (Sigma-Aldrich). Beads were separated, washed with 10 Âµl of an aqueous solution of 2% DMSO and the combined eluates were dried down. Peptides were reconstituted in 10 Âµl of H2O and reacted for 1 h at room temperature with 80 Âµg of TMT10plex (Thermo Scientific)(Werner, Sweetman et al., 2014) label reagent dissolved in 4 Âµl of acetonitrile. Excess TMT reagent was quenched by the addition of 4 Âµl of an aqueous 5% hydroxylamine solution. Peptides were reconstituted in 0.1 % formic acid, mixed to achieve a 1:1 ratio across all TMT-channels and purified by a reverse phase clean-up step (OASIS HLB 96-well ÂµElution Plate, Waters). Peptides were subjected to an off-line fractionation under high pH conditions (Hughes et al., 2014). The resulting 12 fractions were then analyzed by LC-MS/MS on an Orbitrap Fusion Lumos mass spectrometer (Thermo Scentific) as previously described (Sridharan et al., 2019). To this end, peptides were separated using an Ultimate 3000 nano RSLC system (Dionex) equipped with a trapping cartridge (Precolumn C18 PepMap100, 5 mm, 300 Î¼m i.d., 5 Î¼m, 100 Ã…) and an analytical column (Acclaim PepMap 100. 75 Ã— 50 cm C18, 3 mm, 100 Ã…) connected to a nanospray-Flex ion source. The peptides were loaded onto the trap column at 30 Âµl per min using solvent A (0.1% formic acid) and eluted using a gradient from 2 to 40% Solvent B (0.1% formic acid in acetonitrile) over 2 h at 0.3 Âµl per min (all solvents were of LC-MS grade). The Orbitrap Fusion Lumos was operated in positive ion mode with a spray voltage of 2.4 kV and capillary temperature of 275 Â°C. Full scan MS spectra with a mass range of 375â€“1500 m/z were acquired in profile mode using a resolution of 120,000 (maximum fill time of 50 ms or a maximum of 4e5 ions (AGC) and a RF lens setting of 30%. Fragmentation was triggered for 3 s cycle time for peptide like features with charge states of 2â€“7 on the MS scan (data-dependent acquisition). Precursors were isolated using the quadrupole with a window of 0.7 m/z and fragmented with a normalized collision energy of 38. Fragment mass spectra were acquired in profile mode and a resolution of 30,000 in profile mode. Maximum fill time was set to 64 ms or an AGC target of 1e5 ions). The dynamic exclusion was set to 45 s.

### Data Protocol
Acquired data were analyzed using IsobarQuant (Franken, Mathieson et al., 2015) and Mascot V2.4 (Matrix Science) using a reverse UniProt FASTA Saccharomyces cerevisiae database (UP000002311) including common contaminants and the two protein sequences of Q25-GFP and Q97-GFP. The following modifications were taken into account: Carbamidomethyl (C, fixed), TMT10plex (K, fixed), Acetyl (N-term, variable), Oxidation (M, variable) and TMT10plex (N-term, variable). The mass error tolerance for full scan MS spectra was set to 10 ppm and for MS/MS spectra to 0.02 Da. A maximum of 2 missed cleavages were allowed. A minimum of 2 unique peptides with a peptide length of at least seven amino acids and a false discovery rate below 0.01 were required on the peptide and protein level (Savitski, Wilhelm et al., 2015). The raw output files of IsobarQuant (protein.txt â€“ files) were processed using the R programming language (ISBN 3-900051-07-0). Only proteins that were quantified with at least two unique peptides were considered for the analysis. Moreover, only proteins which were identified in both mass spec runs were kept. 2915 proteins passed these quality control filters. Raw TMT reporter ion signals (signal_sum columns) were first cleaned for batch effects using the â€˜removeBatchEffectâ€™ function of the limma package (Ritchie, Phipson et al., 2015) and further normalized using vsn (variance stabilization normalization) (Huber, von Heydebreck et al., 2002). Missing values were imputed with â€˜knnâ€™ method using the Msnbase package (Gatto & Lilley, 2012). Proteins were tested for differential expression using the limma package. T-value outputs of the limma package were used as an input for the â€˜fdrtool â€˜function of the fdrtool package (Strimmer, 2008) in order to estimate p-values and false discovery rates (qvalues were used). Proteins were classified as â€˜hitâ€™ with an fdr smaller 5% and a fold-change of at least 50% and classified as â€˜candidateâ€™ with an fdr smaller 20% and a fold-change of at least 40% using either the fdr estimated by limma directly or by the fdrtool package, depending on which method led to a faster approach to zero of the fdr with increasing effect size (t value).

### Publication Abstract
The formation of protein aggregates is a hallmark of neurodegenerative diseases. Observations on patient samples and model systems demonstrated links between aggregate formation and declining mitochondrial functionality, but causalities remain unclear. We used Saccharomyces cerevisiae to analyze how mitochondrial processes regulate the behavior of aggregation-prone polyQ protein derived from human huntingtin. Expression of Q97-GFP rapidly led to insoluble cytosolic aggregates and cell death. Although aggregation impaired mitochondrial respiration only slightly, it considerably interfered with the import of mitochondrial precursor proteins. Mutants in the import component Mia40 were hypersensitive to Q97-GFP, whereas Mia40 overexpression strongly suppressed the formation of toxic Q97-GFP aggregates both in yeast and in human cells. Based on these observations, we propose that the post-translational import of mitochondrial precursor proteins into mitochondria competes with aggregation-prone cytosolic proteins for chaperones and proteasome capacity. Mia40 regulates this competition as it has a rate-limiting role in mitochondrial protein import. Therefore, Mia40 is a dynamic regulator in mitochondrial biogenesis that can be exploited to stabilize cytosolic proteostasis.

### Keywords
Huntingtin / mia40 / mitochondria / protein aggregation / protein translocation

### Affiliations
EMBL
University of Kaiserslautern, Cell Biology, Germany

### Submitter
Frank Stein

### Lab Head
Dr Johannes M. Herrmann
University of Kaiserslautern, Cell Biology, Germany


