### Accession
PXD004444

### Title
A novel treatment for glioblastoma based on Taxol activation of PC1/3-depleted macrophages: evidence from a proteomic and functional study.

### Description
Tumor-associated macrophages contribute to tumor pathogenesis and represent an attractive therapeutic target. We report that the proprotein convertase PC1/3 inhibits the TLR4 Myd88-pathway induced in macrophages by the anti-cancer agent Taxol. Thus, PC1/3 knock-down in these cells exacerbates the TLR4 MyD88-dependent pathway triggered by Taxol. In PC1/3 knock-down macrophages, Taxol drives the secretion of pro-inflammatory cytokines, inhibits STAT3 signaling and counteracts tumor-supportive activities, thus inhibiting viability, growth and invasion of glioblastoma cells. Proteomic analyses indicate that their secretomes are characterized by a unique protein profile supporting a specific paracrine anti-tumoral effect. These findings unravel the potential value of a new therapeutic strategy combining PC1/3 inhibition and activation of the TLR4 MyD88-dependent pathway to switch intra-tumoral macrophages toward an anti-tumoral immunophenotype.

### Sample Protocol
Filter-aided Sample Preparation (FASP). Total protein extract (0.1 mg) was used for FASP analysis as described previously. We performed FASP using Microcon devices YM-30 (Millipore) before adding trypsin (Promega) for protein digestion (40 µg/ml in 0.05 MNH4HCO3). The samples were incubated overnight at 37°C. The digests were collected by centrifugation, and the filter device was rinsed with 50 µl of NaCl 0.5 M. Next, 5%TFA was added to the digests, and the peptides were desalted with a Millipore ZipTip device before LC-MS/MS analysis.  Proteomics analysis of secretomes. Secretomes obtained from spheroids were centrifuged at 500 g and passed through a 0.22-µm filter to remove cells and debris. The experiments were performed in biological triplicates. Four hundred microliters of the secretome was collected for each condition. The volume was reduced to 100 μl in a SpeedVac. Secretome digestion was performed as previously described. In brief, the cell supernatants were denatured with 2 M urea in 10 mM HEPES, pH 8.0 by sonication on ice. The proteins were reduced with 10 mM DTT for 40 min followed by alkylation with 55 mM iodoacetamide for 40 min in the dark. The iodoacetamide was quenched with 100 mM thiourea. The proteins were digested with 1 μg LysC/Trypsin mixture (Promega) overnight at 37°C. The digestion was stopped with 0.5% TFA. The peptides were desalted with a Millipore ZipTip device in a final volume of 20 μl of 80% ACN elution solution. The solution was then dried using the SpeedVac. Dried samples were solubilized in water/0.1% formic acid before LC MS/MS analysis.  LC MS/MS analysis. Samples were separated by online reversed-phase chromatography using a Thermo Scientific Proxeon Easy-nLC system equipped with a Proxeon trap column (100 µm ID x 2 cm, Thermo Scientific) and a C18 packed-tip column (75 µm ID x 10 cm, Thermo Scientific). Peptides were separated using an increasing amount of acetonitrile (5–35% for 100 min) at a flow rate of 300 nL/min. The LC eluent was electrosprayed directly from the analytical column, and a voltage of 1.7 kV was applied via the liquid junction of the nanospray source. The chromatography system was coupled with a Thermo Scientific Q Exactive mass spectrometer programmed to acquire a data dependent Top 10 method. Survey scans were acquired at a resolution of 70 000 at m/z 400.

### Data Protocol
All the MS data were processed with MaxQuant using the Andromeda search engine. Proteins were identified by searching MS and MS/MS data against Decoy version of the complete proteome for Rattus norvegicus of the UniProt database (Release June 2014, 33,675 entries) combined with 262 commonly detected contaminants. Trypsin specificity was used for the digestion mode with N-terminal acetylation and methionine oxidation selected as the variable. Carbarmidomethylation of cysteines was set as a fixed modification, and we allowed up to two missed cleavages. For MS spectra, an initial mass accuracy of 6 ppm was selected, and the MS/MS tolerance was set to 20 ppm for HCD data. For identification, the FDR at the peptide spectrum matches (PSMs) and protein level was set to 0.01. Relative, label-free quantification of proteins was performed using the MaxLFQ algorithm integrated into MaxQuant with the default parameters. The file containing the information from identification was used with hits to the reverse database, and proteins only identified with modified peptides and potential contaminants were removed. Then, the LFQ intensity was logarithmized (log2[x]). Categorical annotation of rows was used to define different groups depending on the following: 1) the cell line (NT or KD), 2) the treatment (Control/Taxol). Multiple-samples tests were performed using an ANOVA test with a FDR of 5% and preserved grouping in randomization. To determine enrichment of categorical annotations (Gene Ontology terms and KEGG pathway), a Fisher’s exact test was used, taking in account the results of the ANOVA test for each group. Normalization was achieved using a Z-score with matrix access by rows. Only proteins presenting as significant by the ANOVA tests were used for statistical analysis. A hierarchical clustering was first performed using the Euclidean parameter for distance calculation and an average option for linkage in row and column trees using a maximum of 300 clusters. To quantify fold changes of proteins across samples, we used MaxLFQ. To visualize these fold changes in the context of individual protein abundances in the proteome, we projected them onto the summed peptide intensities normalized by the number of theoretically observable peptides.

### Publication Abstract
High grade gliomas are the most common brain tumors in adult. These tumors are characterized by a high infiltration in microglial cells and macrophages. The immunosuppressive tumor environment is known to orient immune cells toward a pro-tumoral and anti-inflammatory phenotype. Therefore, the current challenge for cancer therapy is to find a way to reorient macrophages toward an antitumoral phenotype. Previously, we demonstrated that macrophages secreted antitumoral factors when they were invalidated for the proprotein converstase 1/3 (PC1/3) and treated with LPS. However, achieving an activation of macrophages via LPS/TLR4/Myd88-dependent pathway appears yet unfeasible in cancer patients. On the contrary, the antitumor drug Paclitaxel is also known to activate the TLR4 MyD88-dependent signaling pathway and mimics LPS action. Therefore, we evaluated if PC1/3 knock-down (KD) macrophages could be activated by Paclitaxel and efficient against glioma. We report here that such a treatment of PC1/3 KD macrophages drove to the overexpression of proteins mainly involved in cytoskeleton rearrangement. In support of this finding, we found that these cells exhibited a Ca<sup>2+</sup> increase after Paclitaxel treatment. This is indicative of a possible depolymerization of microtubules and may therefore reflect an activation of inflammatory pathways in macrophages. In such a way, we found that PC1/3 KD macrophages displayed a repression of the anti-inflammatory pathway STAT3 and secreted more pro-inflammatory cytokines. Extracellular vesicles isolated from these PC1/3 KD cells inhibited glioma growth. Finally, the supernatant collected from the coculture between glioma cells and PC1/3 KD macrophages contained more antitumoral factors. These findings unravel the potential value of a new therapeutic strategy combining Paclitaxel and PC1/3 inhibition to switch macrophages toward an antitumoral immunophenotype.

### Keywords
Tlr4 signaling, Calcium, Taxol, Intracellular trafficking, Stat3, Proprotein convertase pc1/3, Shotgun proteomic, Macrophage, Tumors

### Affiliations
Univ. Lille, Inserm, U-1192 - Laboratoire Protéomique, Réponse Inflammatoire et Spectrométrie de Masse-PRISM, F-59000 Lille, France
University of Lille

### Submitter
Maxence Wisztorski

### Lab Head
Dr Michel Salzet
Univ. Lille, Inserm, U-1192 - Laboratoire Protéomique, Réponse Inflammatoire et Spectrométrie de Masse-PRISM, F-59000 Lille, France


