### Accession
PXD020156

### Title
Analysis of Tim-3 interactome in primary mouse CD4+ T cells

### Description
We analyzed the interactome of the Tim-3 (HAVCR2) protein, using an affinity purification method based on the the endogenous expression of a One-Strep-tagged (OST) version of Tim-3  in an engineered mouse model, and affinity purification of the protein from expanded primary CD4+ T cells. Affinity purification of the OST tagged protein was performed using Streptactin beads, from T cells either non stimulated, or stimulated 30s, 120s, 300s or 900s with pervanadate. Each AP-MS purification is associated with a corresponding control (purification from CD4+ T cells isolated from WT mice, cultured and stimulated in the same conditions). The number of replicate biological experiments was n=3 for all conditions (time-points), and each sample was analyzed twice by nanoLC-MS, resulting in 60 raw files.

### Sample Protocol
CD4+ T cells were purified from pooled lymph nodes and spleens. Expanded CD4+ T cells (100 x 106) from Tim-3 OST mice and wild-type mice were kept at 37 ℃ for 5 min and either left unstimulated or stimulated with pervanadate for the specified times. Stimulation was stopped by the addition of a twice-concentrated lysis buffer (100 mM Tris, pH 7.5, 270 mM NaCl, 1 mM EDTA, 20% glycerol, 0.4% n-dodecyl-β-maltoside) supplemented with protease and phosphatase inhibitors. After 10 min of incubation on ice, cell lysates were centrifuged at 21,000 g for 5 min at 4 ℃. Equal amount of post-nuclear lysates were incubated with Strep-Tactin Sepharose beads (IBA GmbH) for 1.5 h at 4°C on a rotary wheel. Beads were then washed five times with 1 ml of lysis buffer in the absence of detergent and of protease and phosphatase inhibitors. Proteins were eluted from the Strep-Tactin Sepharose beads with 2.5 mM D-biotin. Following affinity purification, samples were partially airdried, reconstituted in Laemmli buffer containing DTT (25 mM), and heated at 95°C for 5 min. Cysteines were alkylated for 30 min at room temperature by the addition of iodoacetamide (90 mM). Protein samples were shortly migrated on a SDS–PAGE gel, and in-gel-digested using 0.6 µg of modified sequencing-grade trypsin (Promega) in 50 mM ammonium bicarbonate overnight at 37°C. The resulting peptides were extracted from the gel and purified on a C18 ZipTip (Millipore). Tryptic peptides were resuspended in 20 µl of 2% acetonitrile and 0.05% trifluoroacetic acid and 5µL of each sample were analyzed using an UltiMate 3000 system (NCS-3500RS Nano/Cap System; Dionex) coupled to an Orbitrap Velos Pro mass spectrometer (Thermo Fisher Scientific). Peptides were loaded on a C18 precolumn (300 µm inner diameter × 5 mm, Dionex), and separated on an analytical C18 column (75 µm inner diameter × 50 cm, in-house packed with Reprosil C18) equilibrated in 95% solvent A (5% acetonitrile, 0.2% formic acid) and 5% solvent B (80% acetonitrile, 0.2% formic acid), using a 5–50% gradient of solvent B over 105 min at a flow rate of 300 nl/min. The LTQ Orbitrap Velos was operated in data-dependent acquisition mode; Survey scan MS was acquired in the Orbitrap on the 350–1,800 m/z range, with the resolution set to a value of 60,000. The 20 most intense ions survey scans were selected for fragmentation by collision-induced dissociation, and the resulting fragments were analyzed in the linear trap. Dynamic exclusion was used within 60 s to prevent repetitive selection of the same peptide. Duplicate LC-MS measurements were performed for each sample.

### Data Protocol
Raw MS files were processed with MaxQuant software (version 1.5.2.8) for database search with the Andromeda search engine and quantitative analysis. Data were searched against Mus musculus entries of the UniProtKB protein database (Swiss-Prot+TrEMBL including isoforms), plus the One-Strep-tag peptide sequence, and the set of common contaminants provided by MaxQuant. Carbamidomethylation of cysteines was set as a fixed modification, whereas oxidation of methionine, protein N-terminal acetylation, and phosphorylation of serine, threonine, and tyrosine were set as variable modifications. Specificity of trypsin digestion was set for cleavage after K or R, and two missed trypsin cleavage sites were allowed. The precursor mass tolerance was set to 20 ppm for the first search and 4.5 ppm for the main Andromeda database search. The mass tolerance in tandem MS mode was set to 0.5 Da. Minimum peptide length was set to 7 amino acids, and minimum number of unique or razor peptides was set to 1 for validation. The I=L option of MaxQuant was enabled to avoid erroneous assignation of undistinguishable peptides belonging to very homologous proteins. Andromeda results were validated by the target decoy approach using a reverse database, with a false discovery rate set at 1% at both PSM (peptide sequence match) and protein level. For label-free relative quantification of the samples, the match between runs option of MaxQuant was enabled with a match time window of 1 min, to allow cross-assignment of MS features detected in the different runs, after alignment of the runs with a time window of 20min. Protein quantification was based on unique and razor peptides. The minimum ratio count was set to 1 for LFQ calculation.

### Publication Abstract
None

### Keywords
Tcr, Signalosome, Ap-ms, Primary cd4+ t cell, Interactome

### Affiliations
Institute of Pharmacology and Structural Biology, CNRS, Toulouse
CNRS/IPBS

### Submitter
Anne Gonzalez de Peredo

### Lab Head
Dr Odile Schiltz
Institute of Pharmacology and Structural Biology, CNRS, Toulouse


