### Accession
PXD023570

### Title
Heart extracellular vesicle proteome

### Description
Cardiac function is dynamically regulated by intercellular signalling. As an important intercellular signalling mediator, this study focused on isolating and defining membrane bound signalling packages termed extracellular vesicles (EVs) directly from heart. Here, we utilised a gentle, enzymatic perfusion to extract EVs from whole cardiac tissues (cEVs) for biophysical and proteomic characterisation; cEVs (100-300 nm) are Cd63+, Tsg101+, Pdcd6ip/Alix+ (EV markers) and contain proteins associated with cardiomyocyte (Actn1, Myh6), cardiac progenitor (Cd34, Abcg2), smooth muscle (Cald1, Tagln), endothelial (Vwf), and fibroblast (Ckap4) origins. Proteomic profiling of cEVs revealed 1721 proteins, which bioinformatics revealed hold an enrichment in metabolic functions of glycolysis, fatty acid oxidation, and antioxidant action, presumably to meet the high energetic demands and balance resulting oxidative stress in cardiac tissue.

### Sample Protocol
Protein samples (heart-derived EVs and in-depth heart proteome) were digested with trypsin, and the peptide mixture analyzed on a nanoflow UPLC instrument (Ultimate 3000 RSLCnano, ThermoFisher Scientific) coupled to an Q-Exactive HF Orbitrap mass spectrometer (ThermoFisher Scientific). Label-free MS-based validation study was performed on a separate cohort using single-pot, solid-phase-enhanced sample separation (SP3) and independently analyzed on an Q-Exactive HF-X Orbitrap mass spectrometer.

### Data Protocol
Peptide identification and quantification were performed using MaxQuant (v1.6.14.0) with its built-in search engine Andromeda. Perseus was used to quantify proteins whose expression was identified in at least 60% in at least one group; normalized intensities were log2 transformed, with statistical analyses performed using Studentâ€™s T-test (P < 0.05 considered significant). Gene Ontologies (biological processes) were obtained using DAVID. EnrichmentMap analysis was performed using Cytoscape.

### Publication Abstract
Cardiac intercellular communication is critical for heart function and often dysregulated in cardiovascular diseases. While cardiac extracellular vesicles (cEVs) are emerging mediators of signalling, their isolation remains a technical challenge hindering our understanding of cEV protein composition. Here, we utilised Langendorff-collagenase-based enzymatic perfusion and differential centrifugation to isolate cEVs from mouse heart (yield 3-6 &#x3bc;g/heart). cEVs are &#x223c;200 nm, express classical EV markers (Cd63/81/9<sup>+</sup> , Tsg101<sup>+</sup> , Pdcd6ip/Alix<sup>+</sup> ), and are depleted of blood (Alb/Fga/Hba) and cardiac damage markers (Mb, Tnnt2, Ldhb). Comparison with mechanically-derived EVs revealed greater detection of EV markers and decreased cardiac damage contaminants. Mass spectrometry-based proteomic profiling revealed 1721 proteins in cEVs, implicated in proteasomal and autophagic proteostasis, glycolysis, and fatty acid metabolism; essential functions often disrupted in cardiac pathologies. There was striking enrichment of 942 proteins in cEVs compared to mouse heart tissue - implicated in EV biogenesis, antioxidant activity, and lipid transport, suggesting active cargo selection and specialised function. Interestingly, cEVs contain marker proteins for cardiomyocytes, cardiac progenitors, B-cells, T-cells, macrophages, smooth muscle cells, endothelial cells, and cardiac fibroblasts, suggesting diverse cellular origin. We present a method of cEV isolation and provide insight into potential functions, enabling future studies into EV roles in cardiac physiology and disease.

### Keywords
Heart, Signaling, Proteomics, Extracellular vesicles

### Affiliations
Baker Heart and Diabetes Institute
Baker Heart and Diabetes Inst.

### Submitter
david greening

### Lab Head
Dr David Greening
Baker Heart and Diabetes Institute


