### Accession
PXD042267

### Title
Label-free proteomics quantification of pancreatic ductal adenocarcinoma patient-derived xenografts with treatment combining Gemcitabine and nab-Paclitaxel

### Description
Tumor-stroma interactions are critical in pancreatic ductal adenocarcinoma (PDAC) progression and therapeutics. Patient-derived xenograft (PDX) models faithfully recapitulate tumor-stroma interactions in PDAC, but conventional antibody-based immunoassay is largely inadequate to resolve or quantify tumor and stromal proteins. A species-deconvolved proteomics approach embedded in the ultra-high-resolution (UHR)-IonStar workflow can unambiguously quantify the proteins from tumor (human-derived) and stroma (mouse-derived) in PDX samples, enabling unbiased investigation of their proteomes with excellent quantitative reproducibility. With this strategy, 3 PDAC PDXs were analyzed. They were showed differential responses to treatment with Gemcitabine combined with nab-Paclitaxel (GEM+PTX), which is a first-line treatment regimen for PDAC. For each PDAC PDX, samples were collected after 24 hour and 192 hour with/without treatment, and each condition contained four biological replicates.

### Sample Protocol
Preparation of xenograft tumors for proteomic analysis: PDX tumors #18254, #14312, and #18269 were obtained from Roswell Park Comprehensive Cancer Center (Buffalo, NY). Tumors were harvested rapidly from donor mice after euthanasia, cut into 2×2×2 mm blocks under ice-cold RPMI 1640 medium, and surgically implanted subcutaneously in the abdomen of SCID mice under anesthesia. Gemcitabine (100mg/kg) plus Abraxane (equivalent to 30mg/kg nab-paclitaxel) or saline (control) were given to the mice by intravenous injection once their tumors reached 100-500 mm3. The tumors from treatment and control groups were collected 24 and 192 hours later and snap-frozen in liquid nitrogen for proteomics analysis. According to the tumor size measurement, we found #18254 is sensitive to the combined treatment, whereas # 14312 and #18269 PDXs are resistant to the combined treatment. Tissue protein extraction, clean-up, and digestion: PDX tissue sample preparation employed a surfactant cocktail-aided extraction/precipitation/on-pellet digestion (SEPOD) procedure, which achieves highly efficient and reproducible recovery of proteins and peptides from tissues. Briefly, the snap-frozen tissue was ground into powder and resuspended in a detergent-cocktail buffer (50 mM Tris-formic acid, 150 mM NaCl, 0.5% sodium deoxycholate, 2% SDS, 2% NP-40, pH8.0) with cOmpleteTM protease inhibitor cocktail tablet (Roche Applied Science, Indianapolis, IN) and a PhosSTOPTM phosphatase inhibitor cocktail tablet (Roche Applied Science, Indianapolis, IN). The lysate was incubated on ice for 30 min with vortexing every 10 min, and then homogenized ~10 times with a Polytron homogenizer (Kinematica AG, Switzerland) for 5-10 s at 15,000 rpm with 20s cooling cycles. The mixture was then sonicated with a probe sonicator (3 cycles of 20 sec each) to achieve extensive lysis. The samples were centrifuged at 20,000g for 30 min at 4 °C. The supernatant was collected, protein concentration was quantified by BCA protein assay (Pierce Biotechnology, Inc., Rockford, IL) to determine the protein yield, and then stored at -80 °C until analysis. After reduction/alkylation, the detergents, lipids, fatty acids, salts, and other matrix elements were removed efficiently with organic solvent-mediated precipitation. The protein pellets were then digested robustly and reproducibly, thereby ensuring reliable quantification of the 48 samples obtained from the experiment.

### Data Protocol
Nano LC and ultra-high-resolution Orbitrap MS: The nano-RPLC (reverse-phase liquid chromatography) system consisted of a Dionex Ultimate 3000 nano-LC system and an Ultimate 3000 gradient micro-LC system with a WPS-3000 autosampler. Peptide separation employed a long nano-LC column (75-μm ID × 50 cm) with Pepmap 3.5 μm C18 particles. A large-ID trap (300 μm ID × 5 mm) was packed with Zorbax 5 μm C18 materials to allow large-capacity loading and removal of hydrophobic and hydrophilic matrix components. Mobile phase A was 0.1% formic acid in 2% acetonitrile, and mobile phase B was 0.1% formic acid in 88% acetonitrile. A 4 μg peptide sample was loaded onto the trap with 1% B at 10 μL/min. After the trap was washed for 3 min, a 250 nL/min flow rate was used to back-flush the samples onto the nano-LC column for further separation. The column was enclosed in a heated sheath filled with heat-conductive silicone and maintained homogeneously at 52°C, which helps improve the chromatographic resolution and reproducibility. Chromatography was performed with: 4% B for 3 min; 4 - 11 % B for 5 min; 11 - 28 % B for 117 min; 28 - 50 % B for 10 min; 50 - 97 % B for 1 min; 97% B for 17 min. The trap was turned offline at 45 min to flush hydrophobic components. An Orbitrap Fusion Lumos Mass Spectrometer (Thermo Fisher Scientific, San Jose, CA) was employed for peptide identification and quantification. Data collection was operated in a 3-second cycle using the data-dependent top-speed mode. The MS1 survey scan (m/z 400-1500) was at a resolution of 240,000, with an automated gain control (AGC) target of 400,000 and a maximum injection time of 50 ms. Precursors were fragmented in HCD activation mode at a normalized collision energy of 35% and the dynamic exclusion was set with 45s. Precursors were filtered by quadrupole using an isolation window of 1 Th. The MS2 spectra were collected in the Orbitrap at a resolution of 15,000, with an AGC target of 50,000 and a maximum injection time of 50 ms. Protein identification: LC-MS raw files were matched to a combined Uniprot-Swissprot protein database (Homo sapiens, 26,468 entries, Mus musculus, 17,019 entries) using the SEQUEST-HT search engine in Proteome Discoverer (v1.4, Thermo Scientific). Detailed search parameters were described previously31. The search parameters were set to 20 ppm tolerance for precursor ion mass and 0.02 Da for fragment ion mass. A maximum of two missed cleavages was permitted for fully tryptic peptides. Carbamidomethylation of cysteine was set as a static modification, and a dynamic modification was defined as oxidation at methionine and acetylation at the N-terminus. Scaffold (v4.3.2) was used for quality control of identification information. The false discovery rates (FDR) of identification were estimated using a target–decoy search strategy that was dependent on a concatenated database of forward and reversed sequences. At least two distinct peptides were required for each identified protein, and the false discovery rates (FDR) for peptide and protein identification were set to 0.1% and 1% at the entire dataset level, respectively. Quantification of species-specific proteins: Protein quantification was based on the UHR-IonStar quantitative pipeline. Briefly, chromatographic alignment using the ChromAlign algorithm, and extraction and procurement of peptide peak areas were performed in SIEVE (v2.2, Thermo Scientific). To selectively extract the ion currents from the ultra-high-resolution MS1 measurement, the m/z-RT extraction window was defined as ±5 ppm m/z and ±1 min RT. Then, an in-house R Shiny-based web application named UHR-IonStar (available at https://github.com/JunQu-Lab/UHRIonStarApp) that provides an intuitive interface, was used for integrating identification information with quantitative features, as well as post-quantification data processing, data visualization, and biomarker discovery. The input data include the SIEVE output (MS1 quantitative features) and the PSM list. First, the frames are produced after matching MS1 quantitative features with the PSM list by MS2 scan number, and then the redundant features (i.e., frames assigned with multiple peptide IDs) are removed. Next, UHR-IonStar aggregates the frame-level data to the peptide-level data by the peptide IDs, and then performs total ion-current data normalization and quality control by applying a Principal Component-based mean-and-variance modulation to eliminate low-quality quantitative values on the frame data. Species-deconvolved quantification was performed by removing peptides that were shared between species, and only the species-unique peptides were employed for quantification.

### Publication Abstract
Tumor-stroma interactions are critical in pancreatic ductal adenocarcinoma (PDAC) progression and therapeutics. Patient-derived xenograft (PDX) models recapitulate tumor-stroma interactions, but the conventional antibody-based immunoassay is inadequate to discriminate tumor and stromal proteins. Here, we describe a species-deconvolved proteomics approach embedded in IonStar that can unambiguously quantify the tumor (human-derived) and stromal (mouse-derived) proteins in PDX samples, enabling unbiased investigation of tumor and stromal proteomes with excellent quantitative reproducibility. With this strategy, we studied tumor-stroma interactions in PDAC PDXs that responded differently to Gemcitabine combined with nab-Paclitaxel (GEM+PTX) treatment. By analyzing 48 PDX animals 24 h/192 h after treatment with/without GEM+PTX, we quantified 7262 species-specific proteins under stringent cutoff criteria, with high reproducibility. For the PDX sensitive to GEM+PTX, the drug-dysregulated proteins in tumor cells were involved in suppressed oxidative phosphorylation and the TCA cycle, and in the stroma, inhibition of glycolytic activity was predominant, suggesting a relieved reverse Warburg effect by the treatment. In GEM+PTX-resistant PDXs, protein changes suggested extracellular matrix deposition and activation of tumor cell proliferation. Key findings were validated by immunohistochemistry (IHC). Overall, this approach provides a species-deconvolved proteomic platform that could advance cancer therapeutic studies by enabling unbiased exploration of tumor-stroma interactions in the large number of PDX samples required for such investigations.

### Keywords
Lc-ms/ms patient-derived xenografts species-deconvolved proteomics

### Affiliations
Department of Pharmaceutical Sciences, School of Pharmacy, University at Buffalo, Buffalo, New York, United States
Department of Cell Stress Biology, Roswell Park Comprehensive Cancer Center

### Submitter
Shuo Qian

### Lab Head
Dr Jun Qu
Department of Pharmaceutical Sciences, School of Pharmacy, University at Buffalo, Buffalo, New York, United States


