### Accession
PXD016916

### Title
XPO1 amplification abrogates the effects of E571K XPO1 mutation in B-cell lymphoma

### Description
XPO1 gene encodes exportin 1 (XPO1) that controls the nuclear export of cargo proteins and RNAs. Almost 25% of primary mediastinal B-cell lymphomas (PMBL) and classical Hodgkin lymphomas (cHL) cases harboured a recurrent XPO1 point mutation (NM_003400, chr2:g61718472C>T) resulting in the E571K modification within the hydrophobic groove of the protein, the site of cargo proteins binding. We investigated the functional impact of XPO1E571K mutation using cell lines having various XPO1 status. We first confirmed that the mutation was present both within the XPO1 mRNA and the corresponding protein. In the U2940 cell line having a wild-type (wt) XPO1 gene, we introduced the E571K mutation using several CRISPR-Cas9 strategies. Using a barcoding method to trace the mutation, we observed that the mutation gave no advantage in vitro. Indeed, the effects of XPO1E571K mutation were hidden by the presence of the XPO1 wt allele. Strikingly, XPO1E571K mutation never occurred as homozygous or hemizygous in our patients��������� cohort. We further showed that XPO1E571K mutation modified selinexor binding to XPO1 and in turn, its sensitivity to this drug. We concluded that the balance between the wild-type and the mutated allele of XPO1 is a key element in defining XPO1 functions and oncogenic properties.

### Sample Protocol
Cells were lysed in a buffer containing 25 mM Tris-HCl pH7.5, 150 mM NaCl, 5 mM EDTA, 10% glycerol, 0.5 % NP40, and a cocktail of protease and phosphatase inhibitors (250 ������l for 107 cells). Immunoprecipitation (IP) of XPO1 was performed with 15 ������l of Dynabeads������������������ Protein A and 0.15 ������g of anti-XPO1 antibody (#PLA0109, Sigma-Aldrich, Saint-Louis, MO) per mg of total protein extract. XPO1 and its partners were eluted in a buffer containing 50 mM Tris (pH8.5), 2% SDS, 20 mM TCEP (Tris(2-carboxyethyl)phosphine), 50 mM chloroacetamide, heated for 5 min at 95������C and trypsin-digested overnight using S-trap micro spin columns according to manufacturer (www.protifi.com). Eluted peptides were dried in a vacuum centrifuge (SpeedVac, Eppendorf, Montesson, France) and solubilised with 10 ������l of a 10% acetonitrile (ACN), 0.1% trifluoroacetic acid (TFA) in milliQ-H2O solution.  Nano-liquid chromatography and mass spectrometry analyses were performed on an U3000 RSLC nanoflow-HPLC system eluting into the nanoelectrospray source of a Q-Exactive plus Orbitrap mass spectrometer (both from Thermo Fisher Scientific, Waltham, MA) as follows: peptides were loaded, concentrated and washed on a C18 reverse phase precolumn (3 ������m particle size, 100 ������ pore size, 75 ������m inner diameter, 2 cm length), then separated using a C18 reverse phase analytical column (2 ������m particle size, 100 ������ pore size, 75 ������m inner diameter, 25 cm length) with a 120 min gradient starting from 99% of solvent A (0.1% formic acid in milliQ-H2O) to 40% of solvent B (80% ACN and 0.085% formic acid in milliQ-H2O). The mass spectrometer acquired data throughout the elution process and operated in a data-dependent scheme with 350-1500 m/z MS scans acquired, followed by up to 10 successive MS/MS HCD-fragmentations on the most abundant ions detected. Settings were - for full MS: Automated Gain Control (AGC) target 1.106 ions within 60 ms maximum ion injection time (MIIT) and a target resolution of 70 000; -for MS/MS: precursor selection window was set at 2 m/z. HCD normalized collision energy (NCE) was set at 27% and scan resolution at 17,500 with AGC target 1.105 within 60 ms MIIT. Dynamic exclusion time was set to 30 s and spectra were recorded in profile mode.

### Data Protocol
The mass spectrometry experimental data were extracted from raw files using Proteome Discoverer 1.4 (Thermo) to mgf format peaklists which were compared with theoretical data using Mascot v2.5.1 (Matrix science) on Homo sapiens (23,076 sequences) from the SwissProt protein database containing 558,590 sequences; 200,544,181 residues (October 2018) and an in-house database containing the E571K sequence variant of XPO1. The enzyme specificity type was trypsin���������s (C-term side of K or R), and up to 1 missed cleavage was permitted. The precursor mass tolerance was set to 4 ppm and the fragment mass tolerance to 20 mmu. Carbamidomethylation of cysteins was set as constant modification and oxidation of methionines was set as variable modification. Peptides covering the XPO1 sequences were scrutinized around residue 571 for presence of K or E.

### Publication Abstract
The <i>XPO1</i> gene encodes exportin 1 (XPO1) that controls the nuclear export of cargo proteins and RNAs. Almost 25% of primary mediastinal B-cell lymphoma (PMBL) and classical Hodgkin lymphoma (cHL) cases harboured a recurrent <i>XPO1</i> point mutation (NM_003400, chr2:g61718472C&gt;T) resulting in the E571K substitution within the hydrophobic groove of the protein, the site of cargo binding. We investigated the impact of the <i>XPO1</i><sup>E571K</sup> mutation using PMBL/cHL cells having various <i>XPO1</i> statuses and CRISPR-Cas9-edited cells in which the E571K mutation was either introduced or knocked-out. We first confirmed that the mutation was present in both XPO1 mRNA and protein. We observed that the mutation did not modify the export capacity but rather the subcellular localisation of XPO1 itself. In particular, mutant XPO1 bound to importin &#x3b2;1 modified the nuclear export/import dynamics of relevant cargoes.

### Keywords
Ap-ms, Xpo1, Medb1, Exportin 1, Q-exactive plus, K1106 primary mediastinal b-cell lymphomal lines, U2940, Human cells

### Affiliations
Normandie Univ, INSERM U1245, UNICAEN, Caen, France
ICO Angers

### Submitter
François GUILLONNEAU

### Lab Head
Dr Brigitte Sola
Normandie Univ, INSERM U1245, UNICAEN, Caen, France


