### Accession
PXD014347

### Title
Mouse Mammary Gland Morphogenic Differences with BAD 3SA Mutant

### Description
The identification of genes driving organ development is central to understanding which signaling pathways drive the pathogenesis of various diseases including cancer. This dataset depicts the proteomic changes observed in C57BL/6J mice expressing wild-type or 3SA-phospho mutant versions of the Bcl-2-associated death promoter, BAD. This data shows that BAD regulates postnatal mammary gland morphogenesis in puberty. Three conserved serine residues on BAD are co-ordinately phosphorylated to regulate its activity. Non-phosphorylated BAD mutant delayed pubertal ductal elongation. This defect was specific to the epithelial compartment as transplant and ex vivo organoid assays of mutant epithelium recapitulated decreased tubule migration. Proteomic signature between BAD+/+ and phosphomutant BAD-3SA mammary glands identified differences in actin-binding and focal adhesion components. Mechanistically, non-phosphorylated BAD impedes protein translation, specifically in protrusions, through aberrant hypophosphorylated 4E-BP1. These findings reveal a critical enhancement of localized translation for efficient pubertal-mammary-gland morphogenesis and identifies BAD as a novel regulator of this process.

### Sample Protocol
Using five independent C57BL/6J mice for each screen and each group, mammary glands lysates from 5-weeks Bad+/+ and Bad3SA animals along with 4-weeks Bad+/+ were made (3 groups, 2 screens, total mice = 30). Mammary glands #4 were harvested and immediately snap frozen in liquid nitrogen. Samples were then lysed with a hand-held tissue homogenizer (VWR Cat#47747-370). Lysis buffer for mass spectrometry (MS) samples: 1% NP 40, 150mM NaCl, 50mM Tris-HCL pH 7.6, freshly added protease and phosphatase inhibitors (Roche Cat#5056489001 and Cat#4906837001, respectively). Resultant homogenates totalled 15, 5 per sample group.  Homogenates (20ug of protein per sample) were run on 10% sterile-filtered SDS polyacrylamide gels. Protein was visualized using coomassie, and lanes were excised with a scalpel. Lanes were cut into 20-22 bands each, with each band being cut into 1mm cubes, and 2 bands worth of cubes places into a well of a 96-well conical bottom plate. Gel bits were destained using 50% acetonitrile in 50mM ammonium bicarbonate prior to dehydration with acetonitrile. Samples were then reduced using 10mM 2-ME in 100mM ammonium bicarbonate before alkylating with 55mM iodoacetamide in 100mM ammonium bicarbonate. Samples were then subject to in-gel trypsinization (18 hours at 37 degrees Centigrade). Digestion was quenched using 1% formic acid in 2% acetonitrile in water. Two subsequent extractions were performed (1% formic acid in 50% acetonitrile in water, 1% formic acid in 75% acetonitrile in water), and collected extractions were concentrated using a vacuum concentrator. Each well was resuspended in 60uL of 0.2% formic acid in HPLC grade water. Peptides were analyzed by LC-MSMS using a Thermo Easy nLC-1000 in tandem with a Q-Exactive benchtop orbitrap mass spectrometer. 5uL of sample from each well was subject to a 75-minute gradient (0-45% buffer B; buffer B = 0.2% formic acid in acetonitrile) on a 2cm Acclaim 100 PepMap Nanoviper C18 trappling column with a New Objective PicoChip reverse-phase analytical LC column, with spray voltage set to 2700V.

### Data Protocol
Raw data files comprising an entire gel lane (including intermittent blanks run between samples of an individual gel lane) were grouped and searched using Proteome Discoverer 1.4.1.14's SEQUEST search algorithm using the reviewed, non-redundant Mus musculus complete proteome retrieved from Uniprot KB on October 16, 2015. Search parameters were as follows: event detector mass precision= 2ppm; spectrum selector minimum precursor mass = 350Da, maximum precursor mass = 5000Da; maximum collision energy = 1000; input data digestion enzyme = trypsin (full) with maximum missed cleavage sites = 2; precursor mass tolerance =10ppm with fragment mass tolerance =0.01Da; dynamic modifications to peptides = oxidation of methionine(+15.995Da), deamidation of asparagine and glutamine (+0.984Da); static modifications to peptides = carbamidomethylation of cysteine (+57.021Da). Precursor Ion Areas (extracted ion chromatograms) were included for each protein identified from the raw data. Search results were filtered using a minimum of 2 medium confidence peptides/protein. Magellan storage files from all n=15 lanes were opened in unison using the Proteome Discoverer 1.4.1.14 software package. Protein lists were exported to Microsoft Excel. Protein abundance was determined by looking at each protein's 'Area'. 'Area's for an entire lane were totaled to give a relative TIC, and each protein's individual 'Area' was divided by the TIC to give a 'proportion-of-total' value per sample. Pairwise comparisons of proteins observed in each experimental condition was performed, with analyses limited to proteins with observed abundance in at least 3 samples. To determine the significance of proteins that changed in abundance between experimental conditions, a two-tailed, heteroscedastic T-test was applied to the protein data arrays. P-values were uploaded to qvalue.princeton.edu to generate false-discovery rates (q-values) for the changes in abundance observed for each protein. Fold-changes in abundance were generated using log2 ratios of the average treated and untreated protein abundance. Using q-values and the log2 ratios, proteins deemed to be more abundant in a single experimental condition with high-confidence (p-value cut-off of 0.05 plus a minimum log2 ratio of +/-2) were subject to GO analysis using the Enrichr web application.

### Publication Abstract
None

### Keywords
Mouse, Bad, Breast, Lc-ms/ms, Mammary morphogenesis, Breast cancer, Mammary gland

### Affiliations
Professor, Department of Biochemistry, Faculty of Medicine and Dentistry, University of Alberta, Edmonton, Alberta, Canada
University of Alberta Department of Biochemistry

### Submitter
David Kramer

### Lab Head
Dr Richard P Fahlman
Professor, Department of Biochemistry, Faculty of Medicine and Dentistry, University of Alberta, Edmonton, Alberta, Canada


