### Accession
PXD016777

### Title
Differential responses to kinase inhibition in FGFR2-addicted triple negative breast cancer cells: a quantitative phosphoproteomics study

### Description
Fibroblast Growth Factor (FGF) dependent signalling is frequently activated in cancer by a variety of different mechanisms. However, the downstream signal transduction pathways involved are poorly characterised. Here a quantitative differential phosphoproteomics approach, SILAC, is applied to identify FGF-regulated phosphorylation events in two triple- negative breast tumour cell lines, MFM223 and SUM52PE, that exhibit amplified expression of FGF receptor 2 (FGFR2) and are dependent on continued FGFR2 signalling for cell viability. Comparative Gene Ontology proteome analysis revealed that SUM52PE cells were enriched in proteins associated with cell metabolism and MFM223 cells enriched in proteins associated with cell adhesion and migration. FGFR2 inhibition by SU5402 impacts a significant fraction of the observed phosphoproteome of these cells. This study expands the known landscape of FGF signalling and identifies many new targets for functional investigation. FGF signalling pathways are found to be flexible in architecture as both shared, and divergent, responses to inhibition of FGFR2 kinase activity in the canonical RAF/MAPK/ERK/RSK and PI3K/AKT/PDK/mTOR/S6K pathways are identified. Inhibition of phosphorylation-dependent negative-feedback pathways is observed, defining mechanisms of intrinsic resistance to FGFR2 inhibition. These findings have implications for the therapeutic application of FGFR inhibitors as they identify both common and divergent responses in cells harbouring the same genetic lesion and pathways of drug resistance.

### Sample Protocol
Proteins were reduced with 8 mM DTT, alkylated with 20 mM iodoacetamide in 50 mM ammonium bicarbonate and digested with Trypsin Gold (Promega; 1:100 enzyme:protein ratio) at 37 ˚C overnight. Digested samples were acidified by addition of 0.5 % TFA. Peptides were desalted using Sep-Pak C18 Cartridges (Waters, Milford, MA) according to manufacturer’s instructions. Desalted and dried peptides were resuspended in 100 μL mobile phase A (10 mM KH3PO4, 20 % acetonitrile, pH 3) and loaded onto a 100 × 4.6 mm polysulfoethyl A column (5 μm particle size, 200 nm pore size, PolyLC). Separation used a gradient elution profile that started with 100 % mobile phase A, increased from 0 to 50 % mobile phase B (10 mM KH3PO4, 20 % acetonitrile, 500 mM KCl, pH 3) over 30 min, increased to 100 % B over 5 min, and then returned to 100 % A. The peptides eluted from the column were split into 20 fractions. Each fraction was desalted using a C8 macrotrap cartridge (Michrom BioResources) according to manufacturer’s instructions. Phosphopeptides were enriched using TiO2 tips (TitansphereTM Phos-TiO kit, GL Sciences)

### Data Protocol
Mass spectra were processed using the MaxQuant software (version 1.5.3.8). Data were searched, using the Andromeda search engine within MaxQuant  (42) against the human Swiss-Prot database (downloaded 16.06.17). The human database contained 20,205 reviewed protein entries. The search parameters were: minimum peptide length 7, peptide tolerance 20 ppm (first search) and 4.5 ppm (main search), mass tolerance 0.5 Da, cleavage enzyme trypsin/P, and 2 missed cleavages were allowed. Carbamidomethyl (C) was set as a fixed modification. Oxidation (M), acetylation (Protein N-term), and phospho (STY) were set as variable modifications. The appropriate SILAC labels were selected and the maximum labelled amino acids was set to 3. All experiments were filtered to have a peptide and protein false-discovery rate (FDR) below 1 % and the match between runs featured was enabled.

### Publication Abstract
Fibroblast Growth Factor (FGF) dependent signalling is frequently activated in cancer by a variety of different mechanisms. However, the downstream signal transduction pathways involved are poorly characterised. Here a quantitative differential phosphoproteomics approach, SILAC, is applied to identify FGF-regulated phosphorylation events in two triple- negative breast tumour cell lines, MFM223 and SUM52, that exhibit amplified expression of FGF receptor 2 (FGFR2) and are dependent on continued FGFR2 signalling for cell viability. Comparative Gene Ontology proteome analysis revealed that SUM52 cells were enriched in proteins associated with cell metabolism and MFM223 cells enriched in proteins associated with cell adhesion and migration. FGFR2 inhibition by SU5402 impacts a significant fraction of the observed phosphoproteome of these cells. This study expands the known landscape of FGF signalling and identifies many new targets for functional investigation. FGF signalling pathways are found to be flexible in architecture as both shared, and divergent, responses to inhibition of FGFR2 kinase activity in the canonical RAF/MAPK/ERK/RSK and PI3K/AKT/PDK/mTOR/S6K pathways are identified. Inhibition of phosphorylation-dependent negative-feedback pathways is observed, defining mechanisms of intrinsic resistance to FGFR2 inhibition. These findings have implications for the therapeutic application of FGFR inhibitors as they identify both common and divergent responses in cells harbouring the same genetic lesion and pathways of drug resistance.

### Keywords
Phosphorylation, Fgf, Su5402, Silac, Mapk, Mtor

### Affiliations
University of Birmingham
SChool of Biosciences University of Birminghsam Birminghsm B15 2TT UK

### Submitter
Debbie Cunningham

### Lab Head
Dr Debbie Cunningham
SChool of Biosciences University of Birminghsam Birminghsm B15 2TT UK


