### Accession
PXD000483

### Title
Mycobacterium tuberculosis under starvation

### Description
Mycobacterium tuberculosis was grown in Middlebrook 7H9 medium supplemented with 0.4% glycerol, 0.085% NaCL, 0.5% BSA and 0.05% Tyloxapol in roller bottle culture (2 rpm at 37Â°C). To induce starvation, exponentially growing bacteria were washed and resuspended in PBS supplemented with 0.025% Tyloxapol. Cultures were harvested at 0h, 12h and 24h after starvation induction. After harvesting and lysis, proteins were extracted, reduced, alkylated, and digested using trypsin. The peptides were analysed in DDA mode on a Thermo LTQ Orbitrap XL. Data analysis: Thermo raw files were converted into mzXML format using ProteoWizard. The acquired MS2 spectra were searched with OMSSA (2.1.9), X!Tandem (CYCLONE, 2010.12.01.1), and MyriMatch (2.1.114) against an Mtb H37Rv protein database (TubercuList v2.3, April 2011) additionally containing reversed sequences of all proteins in the database. Search parameters were as follows: semi-tryptic peptides (proteolytic cleavage after lysine and arginine unless followed by proline) and up to two missed cleavages were allowed, mass tolerance of the precursor ions was set to 20 ppm. Carbamidomethylation at cysteines was set as a fixed modification and oxidation at methionines as a variable modification. The output of the search engine was processed using PeptideProphet and iProphet. Only peptides at a false discovery rate of less than 1% were taken into consideration for further analysis. For MS1 based label-free quantification the openMS v1.8 framework was used as described by (Weisser et al., 2013). Signals were normalised on peptide feature level such that the median signal in each sample is the same. Abundances of the three most intense peptides were averaged to get a protein abundance value. The same peptides were used for protein quantification across all samples and proteins with less than three peptides were included. Associated RNA-seq data have been deposited to EBI ArrayExpress with accession number E-MTAB-1616.

### Sample Protocol
Not available

### Data Protocol
Not available

### Publication Abstract
Deciphering physiological changes that mediate transition of Mycobacterium tuberculosis between replicating and nonreplicating states is essential to understanding how the pathogen can persist in an individual host for decades. We have combined RNA sequencing (RNA-seq) of 5' triphosphate-enriched libraries with regular RNA-seq to characterize the architecture and expression of M. tuberculosis promoters. We identified over 4,000 transcriptional start sites (TSSs). Strikingly, for 26% of the genes with a primary TSS, the site of transcriptional initiation overlapped with the annotated start codon, generating leaderless transcripts lacking a 5' UTR and, hence, the Shine-Dalgarno sequence commonly used to initiate ribosomal engagement in eubacteria. Genes encoding proteins with active growth functions were markedly depleted from the leaderless transcriptome, and there was a significant increase in the overall representation of leaderless mRNAs in a starvation model of growth arrest. The high percentage of leaderless genes may have particular importance in the physiology of nonreplicating M. tuberculosis.

### Keywords
Dda, Starvation, Mycobacterium tuberculosis

### Affiliations
ETH Zurich

### Submitter
Olga T. Schubert

### Lab Head


