### Accession
PXD005877

### Title
Quantitative proteomic characterization of temporal lobe during ageing from APP/PS1 mice brain

### Description
Ageing is the most important non-genetic risk factor of late-onset AD. However, the relationship between AD and aging process is still unclear. Here, by comparative analysis, we demonstrated the low correlation between AD and aging process during ageing.

### Sample Protocol
Brain tissues were then homogenised with ice-cold homogenisation buffer (8 M urea in PBS, pH 8.0; with 1 × protease inhibitor cocktail, 1 mM phenylmethanesulfonyl fluoride). The homogenate was centrifuged at 12,000 rpm at 4°C for 15 min to remove cell debris, and the protein concentration of the clarified supernatant was measured using a Nanodrop 2000 (Thermo Scientific, MA, USA). Brain extracts were categorised according to age group (20, 50 and 120 weeks for mouse brain tissue). For each group, four samples (25 µg each) were pooled together to obtain 100 µg protein extract.Extracts were treated with 10 mM dithiothreitol at 55°C for 1 h, then with 25 mM iodoacetamide in the dark at room temperature for 30 min. They were then digested with 1:100 w/w endopeptidase Lys-C at 37°C overnight. The digested extracts were diluted with PBS (pH 8.0) to a final urea concentration of 1.0 M, and further digested with 1:50 w/w trypsin at 37°C overnight. Digested extracts were acidified with 1% formic acid, and then desalted with a reverse-phase column (Oasis HLB; Waters, MA, USA). The extracts were dried with a vacuum concentrator and finally dissolved in 200 mM triethylammonium bicarbonate buffer for labelling with TMT reagents. Mouse TL samples were labelled with TMTs. We used: TMT-126 for 20 weeks APP/PS1; TMT-127 for 20 week old WT; TMT-128 for 50 weeks APP/PS1; TMT-129 for 50 week old WT; TMT-130 for 120 weeks APP/PS1 and TMT-131 for 120 week old WT. The prepared TMT reagents (0.8 mg TMT dissolved in 40 μl of 99.9% acetonitrile) were added to the prepared extracts, and the labelling reaction was allowed to progress at room temperature. The reactions were stopped after 1 h by the addition of 5 μl of 5% hydroxylamine for 5 min. All of the labelled extracts were mixed, desalted, and dried as previously described and dissolved in 20 μl of 0.1% formic acid.The TMT labelled peptides (100 μl in 0.1% formic acid) were first transferred to MS tubes for HPLC analysis (UltiMate 3000 UHPLC; Thermo Scientific), using an Xbridge BEH300 C18 column (4.6 × 250 mm, 2.5 µm; Waters, MA, USA) maintained at 45°C with a flow rate of 1.0 ml/min for further fractionation. Detection of UV absorbance was set to 214 nm. The gradient elution buffer consisted of methanol (phase A; pH 10.0) and acetonitrile (phase B; pH 10.0). Fractions were collected every 1.5 min into 50 tubes, then dried and combined into 20 tubes. Finally, they were dissolved with 20 μl of 0.1% formic acid for liquid chromatography (LC)-MS/MS analysis.LC-MS/MS analysis was conducted on an Orbitrap Fusion Lumos Tribrid Mass spectrometer (Thermo Fisher, MA, USA). The fractions were separated with a 120 min gradient elution (Phase A: ddH2O with 0.1% fluoroacetic acid; Phase B: 20% ddH2O, 80% acetonitrile and 0.1% fluoroacetic acid) at a flow rate of 0.30 μl/min. The UltiMate 3000 RSLCnano System (Thermo Scientific, MA, USA) was interfaced with the mass spectrometer. The column used for analysis was a silica capillary column (75 µm ID, 150 mm length; Upchurch Scientific, Oak Harbor, WA) packed with C18 resin (300 Å, 5 µm; Varian, Lexington, MA). To set the mass spectrometer in data-dependent acquisition mode, we used the Xcalibur 2.1.2 software. Using an Orbitrap (400-1,800 m/z; 60,000 FWHM resolution), a single full-scan mass spectrum was conducted before collecting ten data-dependent MS/MS scans at 27% normalised collision energy (higher energy C-trap dissociation, HCD).

### Data Protocol
The Proteome Discoverer 2.1 software package (Thermo Scientific, MA, USA) was used to analyse MS/MS spectra. Spectra were searched against the mouse proteome in the UniProt Swiss-Prot database (released August 22nd, 2016). The protein sequence of MoHuAPPswe was manually added into the mouse proteome when searching for Aβ and other APP peptides. The criteria for searching was set according to the software recommendations, with the following modifications: full tryptic specificity with no more than two missed cleavages permitted; carbamidomethylation (Cysteine, C) and TMT plex (lysine, K and any N-terminal) as static modifications; and oxidation (methionine, M) and deamination (asparagine, N and glutamine, Q) as dynamic modifications. The fragment ion mass tolerance was corrected to 20 mmu (all MS2 spectra) and the precursor ion mass tolerances were set at 20 ppm (all MS in an Orbitrap mass analyser). The FDR of peptide and protein identification were both set to 0.01 (1%). Based on the reporter ion intensities per peptide, relative protein intensities were quantified. Normalised protein abundance was calculated by summing the intensities of associated peptides, followed by normalisation to the total peptide amount in each sample.

### Publication Abstract
None

### Keywords
Brain, Temporal lobe, Mouse, Alzheimer's disease

### Affiliations
National Key Laboratory of Medical Molecular Biology & Department of Immunology, Institute of Basic Medical Sciences, Chinese Academy of Medical Sciences
National Key Laboratory of Medical Molecular Biology & Department of Immunology, Institute of Basic Medical Sciences, Chinese Academy of Medical Sciences 

### Submitter
Wei Ge

### Lab Head
Dr Wei Ge
National Key Laboratory of Medical Molecular Biology & Department of Immunology, Institute of Basic Medical Sciences, Chinese Academy of Medical Sciences


