### Accession
PXD026191

### Title
Revealing the Structural Plasticity of SARS-CoV-2 nsp7 and nsp8 Using Structural Proteomics (XL-MS)

### Description
SARS-CoV-2 nsp7 and nsp8 are important cofactors of the RTC, as they interact and regulate the activity of RNA-dependent RNA polymerase and other nspsHere we used solution-based structural proteomic techniques, hydrogen-deuterium exchange mass spectrometry (HDX-MS) and crosslinking mass spectrometry (XL-MS), illuminate the dynamics of SARS-CoV-2 full-length nsp7, nsp8, and nsp7:nsp8 proteins and protein complex.

### Sample Protocol
For DSSO (disuccinimidyl sulfoxide) (ThermoFisher) crosslinking reactions, individual protein and protein-protein complexes were diluted to 10 µM in crosslinking buffer (50 mM HEPES pH 8.0, 500 mM NaCl, 1 mM TCEP) and incubated for 30 min at room temperature prior to initiating the crosslinking reaction. DSSO crosslinker was freshly dissolved in crosslinking buffer to a final concentration of 75 mM before being added to the protein solution at a final concentration of 1.5 mM. The reaction was incubated at 25°C for 45 min, then quenched by adding 1 µL of 1.0 M Tris pH 8.0 and incubating an additional 10 min at 25°C. Control reactions were performed in parallel without adding the DSSO crosslinker. All crosslinking reactions were carried out in three replicates. The presence of crosslinked proteins was confirmed by comparing to the no crosslink negative control samples using SDS-PAGE and Coomassie staining. The remaining crosslinked and non-crosslinked samples were separately pooled and then precipitated using methanol and chloroform. Dried protein pellets were resuspended in 12.5 µL of resuspension buffer (50 mM ammonium bicarbonate, 8M urea, pH 8.0). ProteaseMAX (Promega - V5111) was added to 0.02% and the solutions were mixed on an orbital shaker operating at 400 RPM for 5 min. After resuspension, 87.5 µL of digestion buffer (50 mM ammonium bicarbonate, pH 8.0) was added. Protein samples were reduced by adding 1 µL of 500 mM DTT followed by incubation of the protein solutions in an orbital shaker operating at 400 RPM at 56°C for 20 minutes. After reduction, 2.7 µL of 550 mM iodoacetamide was added and the solutions were incubated at room temperature in the dark for 15 min. Reduced and alkylated protein solutions were digested overnight using trypsin at a ratio of 1:150 (w/w trypsin:protein) at 37°C. Peptides were acidified to 1% trifluoroacetic acid (TFA) and then desalted using C18 ZipTip® (Millipore cat # ZTC18 5096). Dried peptides were resuspended in 10 µL of 0.1% TFA in water. Samples were then frozen and stored at -20°C until LC-MS analysis. Liquid Chromatography and Mass Spectrometry: 500 ng of sample was injected (triplicate injections for crosslinked samples and duplicate injections for control samples) onto an UltiMate 3000 UHP liquid chromatography system (Dionex, ThermoFisher). Peptides were trapped using a μPAC C18 trapping column (PharmaFluidics) using a load pump operating at 20 µL/min. Peptides were separated on a 200 cm μPAC C18 column (PharmaFluidics) using a linear gradient (1% Solvent B for 4 min, 1-30% Solvent B from 4-70 min, 30-55% Solvent B from 70-90 min, 55-97% Solvent B from 90-112 min, and isocratic at 97% Solvent B from 112-120 min) at a flow rate of 800 nL/min. Gradient Solvent A contained 0.1% formic acid and Solvent B contained 80% acetonitrile and 0.1% formic acid. Liquid chromatography eluate was interfaced to an Orbitrap Fusion Lumos Tribrid mass spectrometer (ThermoFisher) with a Nanospray Flex ion source (ThermoFisher). The source voltage was set to 2.5 kV, and the S-Lens RF level was set to 30%. Crosslinks were identified using a previously described MS2-MS3 method with slight modifications. Full scans were recorded from m/z 150 to 1500 at a resolution of 60,000 in the Orbitrap mass analyzer. The AGC target value was set to 4e5, and the maximum injection time was set to 50 ms in the Orbitrap. MS2 scans were recorded at a resolution of 30,000 in the Orbitrap mass analyzer. Only precursors with charge state between 4-8 were selected for MS2 scans. The AGC target was set to 5e4, a maximum injection time of 150 ms, and an isolation width of 1.6 m/z. CID fragmentation energy was set to 25%. The two most abundant reporter doublets from the MS2 scans with a charge state of 2-6, a 31.9721 Da mass difference (Kao et al 2011), and a mass tolerance of ±10 ppm were selected for MS3. The MS3 scans were recorded in the ion trap in rapid mode using HCD fragmentation with 35% collision energy. The AGC target was set to 2e4, the maximum injection time was set for 200 ms, and the isolation width set to 2.0 m/z.

### Data Protocol
To identify crosslinked peptides, Thermo.Raw files were imported into Proteome Discoverer 2.5 (ThermoFisher) and analyzed via XlinkX algorithm using the MS2_MS3 workflow with the following parameters: MS1 mass tolerance—10 ppm; MS2 mass tolerance—20 ppm; MS3 mass tolerance—0.5 Da; digestion—trypsin with four missed cleavages allowed; minimum peptide length of 4 amino acids, fixed modification—carbamidomethylation (C); variable modification—oxidation (M); and DSSO (K, S, T, Y). The XlinkX/PD Validator node was used for crosslinked peptide validation with a 1% false discovery rate (FDR). Identified crosslinks were further validated and quantified using Skyline (version 19.1) using a previously described protocol. Crosslink spectral matches found in Proteome Discoverer were exported and converted to sequence spectrum list format using Excel (Microsoft). Crosslink peak areas were assessed using the MS1 full-scan filtering protocol for peaks within 8 min of the crosslink spectral match identification. Peak areas were assigned to the specified crosslinked peptide identification if the mass error was within 10 ppm of the theoretical mass, the isotope dot product was greater than 0.95, and if the peak was not found in the non-crosslinked negative control samples. The isotope dot product compares the distribution of the measured MS1 signals against the theoretical isotope abundance distribution calculated based on the peptide sequence. Its value ranges between 0 and 1, where 1 indicates a perfect match. Pair-wise comparisons were made using ‘MSstats’ package implemented in Skyline to calculate relative fold changes and significance. Significant change thresholds were defined as a log2 fold change less than -2 or greater than 2 and -log10 p-value greater than 1.3 (p-value less than 0.05). Visualization of proteins and crosslinks was generated using xiNET.

### Publication Abstract
Coronavirus (CoV) non-structural proteins (nsps) assemble to form the replication-transcription complex (RTC) responsible for viral RNA synthesis. nsp7 and nsp8 are important cofactors of the RTC, as they interact and regulate the activity of RNA-dependent RNA polymerase (RdRp) and other nsps. To date, no structure of full-length SARS-CoV-2 nsp7:nsp8 complex has been published. Current understanding of this complex is based on structures from truncated constructs or with missing electron densities and complexes from related CoV species with which SARS-CoV-2 nsp7 and nsp8 share upwards of 90% sequence identity. Despite available structures being solved using crystallography and cryo-EM representing detailed snapshots of the nsp7:nsp8 complex, it is evident that the complex has a high degree of structural plasticity. However, relatively little is known about the conformational dynamics of the complex and how it assembles to interact with other nsps. Here, the solution-based structural proteomic techniques, hydrogen-deuterium exchange mass spectrometry (HDX-MS) and crosslinking mass spectrometry (XL-MS), illuminate the structural dynamics of the SARS-CoV-2 full-length nsp7:nsp8 complex. The results presented from the two techniques are complementary and validate the interaction surfaces identified from the published three-dimensional heterotetrameric crystal structure of SARS-CoV-2 truncated nsp7:nsp8 complex. Furthermore, mapping of XL-MS data onto higher order complexes suggests that SARS-CoV-2 nsp7 and nsp8 do not assemble into a hexadecameric structure as implied by the SARS-CoV full-length nsp7:nsp8 crystal structure. Instead our results suggest that the nsp7:nsp8 heterotetramer can dissociate into a stable dimeric unit that might bind to nsp12 in the RTC without altering nsp7-nsp8 interactions.

### Keywords
Sars-cov-2, Xl-ms, Nsp8, Nsp7

### Affiliations
Scripps Research
Department of Molecular Medicine The Scripps Research Institute

### Submitter
Valentine Courouble

### Lab Head
Dr Patrick R. Griffin
Department of Molecular Medicine The Scripps Research Institute


