### Accession
PXD006484

### Title
Immune repertoire after immunization as seen by next generation sequencing and proteomics

### Description
We study the repertoire of immunoglobulin sequences after immunization in a cohort of rats. Animals were immunized with dinitrophenol modified KLH or with recombinant human HuD. In the immune repertoire, and also in the matching proteomics data, we observe that the  animals produce a repertoire that has many shared motifs for those immunized with the same antigen. Cluster analysis allows the samples to be segregated according to the immunogen used.

### Sample Protocol
Total IgG was isolated from immune sera with ProteinG affinity columns. In addition, immunoglobulins were affinity enriched from serum with either HuD or Dinitrophenol modified ovalbumin immobilized on AminoLink particles. On the immunoglobulin samples an in-solution digestion with trypsin was performed after reduction and alkylation with iodoacetamide. The digested samples were separated on a Pepmap C18 column (250mmx75Âµm) in a 90 min gradient (4-40% acetonitrile in 0.1% formic acid/water) and analyzed with an LTQ Orbitrap XL.

### Data Protocol
Thermo RAW files were imported into PEAKS Studio 6. No filtering or correction was applied in the data refine module. Subsequently, the de novo sequencing  routine was applied (10 ppm precursor tolerance, 0.5 Da fragment tolerance, carbamidomethyl C as fixed and oxidized methionine as variable modification). Subsequently, the PeaksDB database dependent search was performed with a database derived from RNA-seq data on the immune repertoire based on splenocytes harvested simultaneously with the immune sera of these animals.

### Publication Abstract
The immune system produces a diverse repertoire of immunoglobulins in response to foreign antigens. During B-cell development, VDJ recombination and somatic mutations generate diversity, whereas selection processes remove it. Using both proteomic and NGS approaches, we characterized the immune repertoires in groups of rats after immunization with purified antigens. Proteomics and NGS data on the repertoire are in qualitative agreement, but did show quantitative differences that may relate to differences between the biological niches that were sampled for these approaches. Both methods contributed complementary information in the characterization of the immune repertoire. It was found that the immune repertoires resulting from each antigen had many similarities that allowed samples to cluster together, and that mutated immunoglobulin peptides were shared among animals with a response to the same antigen significantly more than for different antigens. However, the number of shared sequences decreased in a log-linear fashion relative to the number of animals that share them, which may affect future applications. A phylogenetic analysis on the NGS reads showed that reads from different individuals immunized with the same antigen populated distinct branches of the phylogram, an indication that the repertoire had converged. Also, similar mutation patterns were found in branches of the phylogenetic tree that were associated with antigen-specific immunoglobulins through proteomics data. Thus, data from different analysis methods and different experimental platforms show that the immunoglobulin repertoires of immunized animals have overlapping and converging features. With additional research, this may enable interesting applications in biotechnology and clinical diagnostics.

### Keywords
Immunization, Immunoglobulins, Proteogenomics, Ngs, Mass spectrometry, Immune repertoire

### Affiliations
Erasmus MC
Erasmus MC Department of Neurology Clinical and Cancer Proteomics

### Submitter
Martijn van Duijn

### Lab Head
Dr Theo Luider
Erasmus MC Department of Neurology Clinical and Cancer Proteomics


