### Accession
PXD036099

### Title
Integrative transcriptome and proteome revealed high-yielding mechanisms of epsilon-poly-L-lysine by Streptomyces albulus

### Description
ε-poly-L-lysine (ε-PL) is a high value, widely used natural antimicrobial peptide additive for foods and cosmetic products that is mainly produced by S. albulus. In previous work, we developed the high-yield industrial strain S. albulus WG-608 through successive rounds of engineering. Here, we use integrated physiological, transcriptomic, and proteomics association analysis to resolve the complex mechanisms underlying high ε-PL production by comparing WG-608 with the progenitor strain M-Z18. Our results show that key genes in the glycolysis, glyoxylate, and L-lysine biosynthesis pathways are differentially upregulated in WG-608, while genes in the biosynthetic pathways for fatty acids, various branched amino acids, and secondary metabolite by-products are downregulated. This regulatory pattern results in the introduction of more carbon atoms into L-lysine biosynthesis and ε-PL production. Furthermore, transcriptional and translational upregulation of genes involved in the tricarboxylic acid cycle, oxidative phosphorylation, and pentose phosphate pathway also increase the pools of available NADH, ATP, and NADPH. In addition, significant changes in the regulation of DNA replication, transcription, and translation, two component systems, and quorum sensing may facilitate the adaptability to environmental pressure, thus further regulating the ε-PL biosynthesis. This study enables comprehensive understanding of the biosynthetic mechanisms of ε-PL in S. albulus WG-608, while providing a theoretical foundation development of advanced Streptomycetaceae microbial cell factories.

### Sample Protocol
Fresh fermentation broth was taken from the fermentor for 96h at 12000rpm and centrifuged at 4℃ for 5 minutes. The supernatant was discarded and the organisms were cold extracted with liquid nitrogen and stored at -80°C .

### Data Protocol
1.1 Data output statistics 1.2 Protein identification results 1.3 Protein quantitative results 1.4 Protein Go Analysis 1.5 protein KOG/COG/Eggnog analysis 1.6 Protein Pathway metabolic pathway analysis 1.7 GO enrichment analysis of different protein 1.8 Pathway enrichment analysis of differential proteins 1.9 Different protein KOG/COG/EGGNOG enrichment analysis 1.10 Repeated analysis (only for designing repeated experiments) 1.11 Duke Sample expression mode clustering (three or three samples can be provided) 1.12 Different expression of protein interaction analysis 1.13 Difference

### Publication Abstract
None

### Keywords
Ε-pl, Proteomics, Transcriptomic, Physiological

### Affiliations
JIANGNAN UNIVERSITY
The Key Laboratory of Industrial Biotechnology, Ministry of Education, School of Biotechnology, Jiangnan University, 1800 Lihu Avenue, Wuxi 214122, Jiangsu, China

### Submitter
hao yang

### Lab Head
Dr Wang Liang
The Key Laboratory of Industrial Biotechnology, Ministry of Education, School of Biotechnology, Jiangnan University, 1800 Lihu Avenue, Wuxi 214122, Jiangsu, China


