### Accession
PXD009643

### Title
Organisms with alternative genetic codes resolve unassigned codons via mistranslation and ribosomal rescue

### Description
Organisms possessing genetic codes with unassigned codons raise the question of how cellular machinery resolves such codons and how this could impact horizontal gene transfer. Here, we use a genomically recoded Escherichia coli to examine how organisms address translation at unassigned UAG codons, which obstruct propagation of UAG-containing viruses and plasmids. Using mass spectrometry, we show that recoded organisms resolve translation at unassigned UAG codons via near-cognate suppression, dramatic frameshifting from at least -3 to +19 nucleotides, and rescue by ssrA-encoded tmRNA, ArfA, and ArfB. We then demonstrate that deleting tmRNA restores expression of UAG-ending proteins and propagation of UAG-containing viruses and plasmids in the recoded strain, indicating that tmRNA rescue and nascent peptide degradation is the cause of impaired virus and plasmid propagation. The ubiquity of tmRNA homologs suggests that genomic recoding is a promising path to impair horizontal gene transfer and confer genetic isolation in diverse organisms.

### Sample Protocol
His-tagged GFP was purified by nickel affinity column. Trypsin digest, sample preparation for mass spectrometry, and liquid chromatography elution gradients were performed as described previously (Aerni et al., 2015). Desalted peptides were injected onto a 75 μm ID PicoFrit column (New Objective) packed to 50 cm in length with 1.9 μm ReproSil-Pur 120 Å C18-AQ (Dr. Maisch). Samples were eluted over a 90-minute gradient using an EASY-nLC 1000 UPLC (Thermo) paired with a Q Exactive Plus (Thermo), using the following parameters: (MS1) 70,000 resolution, 3x10e6 AGC target, 300-1700 m/z scan range; (MS2) 17,500 resolution, 1x10e6 AGC target, top 10 mode, 1.6 m/z isolation window, 27 normalized collision energy, 90 s dynamic exclusion, unassigned and +1 charge exclusion.

### Data Protocol
Peptide identification from collected spectra was performed using MaxQuant v1.5.1.2 (Cox et al, 2008). Samples were searched using custom databases representing potential translational outcomes in response to the UAG codon within the GFP reporter construct (db1 and db2, uploaded files), as well as the E. coli proteome (EcoCyc K-12 MG1655 v17). The searches considered carbamidomethyl (Cys) as a fixed modification and the following variable modifications: acetyl (N-terminal), oxidation (Met), deamidation (Asn, Gln), and phosphorylation (Ser/Thr/Tyr). Discovered peptides had a minimum length of 5 amino acids and could contain up to three trypsin miscleavage events.

### Publication Abstract
Organisms possessing genetic codes with unassigned codons raise the question of how cellular machinery resolves such codons and how this could impact horizontal gene transfer. Here, we use a genomically recoded <i>Escherichia coli</i> to examine how organisms address translation at unassigned UAG codons, which obstruct propagation of UAG-containing viruses and plasmids. Using mass spectrometry, we show that recoded organisms resolve translation at unassigned UAG codons via near-cognate suppression, dramatic frameshifting from at least -3 to +19 nucleotides, and rescue by <i>ssrA</i>-encoded tmRNA, ArfA, and ArfB. We then demonstrate that deleting tmRNA restores expression of UAG-ending proteins and propagation of UAG-containing viruses and plasmids in the recoded strain, indicating that tmRNA rescue and nascent peptide degradation is the cause of impaired virus and plasmid propagation. The ubiquity of tmRNA homologs suggests that genomic recoding is a promising path for impairing horizontal gene transfer and conferring genetic isolation in diverse organisms.

### Keywords
E coli, Lc-ms/ms

### Affiliations
Yale University
Department of Cellular & Molecular Physiology, Yale University School of Medicine, New Haven, CT 06520, USA Systems Biology Institute, Yale University, West Haven, CT 06516, USA

### Submitter
Karl Barber

### Lab Head
Dr Jesse Rinehart
Department of Cellular & Molecular Physiology, Yale University School of Medicine, New Haven, CT 06520, USA Systems Biology Institute, Yale University, West Haven, CT 06516, USA


