### Accession
PXD035413

### Title
Mechanistic insights into RNA surveillance by the canonical poly(A) polymerase Pla1 of the MTREC complex

### Description
The S. pombe orthologue of the human PAXT complex, Mtl1-Red1 Core (MTREC), is an eleven-subunit complex which targets cryptic unstable transcripts (CUTs) to the nuclear RNA exosome for degradation. It encompasses the canonical poly(A) polymerase Pla1, responsible for polyadenylation of nascent RNA transcripts as part of the cleavage and polyadenylation factor (CPF/CPSF). In this study we identified and characterised the interaction between Pla1 and the MTREC complex core component Red1 and analysed the functional relevance of this interaction in vivo. Our crystal structure of the Pla1-Red1 complex showed that a 58-residue fragment in Red1 binds to the RNA recognition motif domain of Pla1 and tethers it to the MTREC complex. Structure-based Pla1-Red1 interaction mutations showed that Pla1, as part of MTREC complex, hyper-adenylates CUTs for their efficient degradation. Interestingly, the Red1-Pla1 interaction was also required for the efficient assembly of the fission yeast facultative heterochromatic islands. Together, our data suggest a complex interplay between the RNA surveillance and 3â€™-end processing machineries.

### Sample Protocol
Tandem affinity purification followed by interaction analysis/ MS Flag-TEV-protein A (FTP)-tagged bait proteins were harvested from 2 L YEA cultures of yeast strains grown to OD600 1.8â€“2.2. Cell pellets were snap-frozen in liquid nitrogen and ground into powder using the Cryo-mill MM-400 (Retsch machine). Cells were resuspended in purification buffer (50 mM HEPES, pH 7.0; 100 mM NaCl; 1.5 mM MgCl2; 0.15% NP-40), supplemented with 1 mM dithiothreitol (Sigma), 1 mM phenylmethylsulphonyl fluoride (Sigma Aldrich, 78830) and protease inhibitor mix (SERVA, 39104). Cell extracts were centrifuged at 3500 g for 10 min at 4 Â°C, then the supernatants were further centrifuged at 27000 g for 45 min at 4 Â°C. Clarified supernatants were then incubated with 150 mL slurry of IgG beads (GE Healthcare, 17-0969-02) for 2h at 4 Â°C on a turning wheel. After binding, the beads were washed with 2 x 15 mL purification buffer and TEV cleavage was performed in purification buffer containing 20 units AcTEV protease (Invitrogen 12575015), 0.5 mM dithiothreitol and 50 units Benzonase (Sigma, E1014) for the removal of nucleic acids, for 2 h at 16 Â°C. The eluate was collected and incubated with 100 mL slurry of anti-Flag beads (Sigma Aldrich, A2220) for 1h at 4 Â°C. The protein- bound anti-Flag beads were washed with 2 x 10 mL purification buffer and the proteins were subsequently eluted from the beads by competition with 200 mL Flag peptide (Assay Matrix, A6001). The resulting eluate was collected and used for total protein isolation using Trichloroacetic acid- TCA (Sigma, T0699). The eluted proteins were then analysed by Coomassie staining using Brilliant Blue G colloidal concentrate (Sigma Aldrich, B2025) on 4â€“12% NuPAGE Bis-Tris gels (Life Technologies).  For mass spectrometry analysis, the precipitated protein samples were subjected to an in-solution tryptic digest using a modified version of the Single-Pot Solid-Phase-enhanced Sample Preparation (SP3) protocol92,93. In total three biological replicates were prepared including control, wild-type and mutant derived lysates (n=3). Lysates were added to Sera-Mag Beads (Thermo Scientific, #4515-2105-050250, 6515-2105-050250) in 10 Âµl 15% formic acid and 30 Âµl of ethanol. Binding of proteins was achieved by shaking for 15 min at room temperature (RT). SDS was removed by 4 subsequent washes with 200 Âµl of 70% ethanol. Proteins were digested overnight at room temperature with 0.4 Âµg of sequencing grade modified trypsin (Promega, #V5111) in 40 Âµl Hepes/NaOH, pH 8.4 in the presence of 1.25 mM TCEP and 5 mM chloroacetamide (Sigma-Aldrich, #C0267). Beads were separated, washed with 10 Âµl of an aqueous solution of 2% DMSO and the combined eluates were dried down. Peptides were reconstituted in 10 Âµl of H2O and reacted for 1 h at room temperature with 80 Âµg of TMT10plex (Thermo Scientific, #90111)94 label reagent dissolved in 4 Âµl of acetonitrile. Excess TMT reagent was quenched by the addition of 4 Âµl of an aqueous 5% hydroxylamine solution (Sigma, 438227). Peptides were reconstituted in 0.1 % formic acid, mixed to achieve a 1:1 ratio across all TMT-channels and purified by a reverse phase clean-up step (OASIS HLB 96-well ÂµElution Plate, Waters #186001828BA). Peptides were subjected to an off-line fractionation under high pH conditions93. The resulting 12 fractions were then analyzed by LC-MS/MS on an Orbitrap Fusion Lumos mass spectrometer (Thermo Scentific) as previously described95. To this end, peptides were separated using an Ultimate 3000 nano RSLC system (Dionex) equipped with a trapping cartridge (Precolumn C18 PepMap100, 5 mm, 300 Î¼m i.d., 5 Î¼m, 100 Ã…) and an analytical column (Acclaim PepMap 100. 75 Ã— 50 cm C18, 3 mm, 100 Ã…) connected to a nanospray-Flex ion source. The peptides were loaded onto the trap column at 30 Âµl per min using solvent A (0.1% formic acid) and eluted using a gradient from 2 to 40% Solvent B (0.1% formic acid in acetonitrile) over 2 h at 0.3 Âµl per min (all solvents were of LC-MS grade). The Orbitrap Fusion Lumos was operated in positive ion mode with a spray voltage of 2.4 kV and capillary temperature of 275 Â°C. Full scan MS spectra with a mass range of 375â€“1500 m/z were acquired in profile mode using a resolution of 120,000 (maximum fill time of 50 ms or a maximum of 4e5 ions (AGC) and a RF lens setting of 30%. Fragmentation was triggered for 3 s cycle time for peptide like features with charge states of 2â€“7 on the MS scan (data-dependent acquisition). Precursors were isolated using the quadrupole with a window of 0.7 m/z and fragmented with a normalized collision energy of 38. Fragment mass spectra were acquired in profile mode and a resolution of 30,000 in profile mode. Maximum fill time was set to 64 ms or an AGC target of 1e5 ions). The dynamic exclusion was set to 45 s.

### Data Protocol
Acquired data were analyzed using IsobarQuant96 and Mascot V2.4 (Matrix Science) using a reverse UniProt FASTA Schizosaccharomyces pombe database (UP000002485) including common contaminants. The following modifications were taken into account: Carbamidomethyl (C, fixed), TMT10plex (K, fixed), Acetyl (N-term, variable), Oxidation (M, variable) and TMT10plex (N-term, variable). The mass error tolerance for full scan MS spectra was set to 10 ppm and for MS/MS spectra to 0.02 Da. A maximum of 2 missed cleavages were allowed. A minimum of 2 unique peptides with a peptide length of at least seven amino acids and a false discovery rate below 0.01 were required on the peptide and protein level97, resulting in 164 proteins. Raw TMT reporter ion intensities (signal_sum columns) were first cleaned for batch effects using limma98 and further normalized using vsn (variance stabilization normalization99. Proteins were tested for differential expression using the limma package. The replicate information was not added as a factor in the design matrix since some condition were measured with a single replicate only. A protein was annotated as a hit with a false discovery rate (fdr) smaller 5 % and a fold-change of at least 100 % and as a candidate with a fdr below 20 % and a fold-change of at least 50 %.

### Publication Abstract
None

### Keywords
Mtrec complex; s. pombe; paxt complex; rna surveilance

### Affiliations
EMBL
The John Curtin School of Medical Research The Australian National University

### Submitter
Frank Stein

### Lab Head
Dr Tamas Fischer
The John Curtin School of Medical Research The Australian National University


