### Accession
PXD031083

### Title
Gel-like inclusions of C-terminal fragments of TDP-43 sequester stalled proteasomes in neurons

### Description
Aggregation of the multifunctional RNA-binding protein TDP-43 defines large subgroups of amyotrophic lateral sclerosis and frontotemporal dementia and correlates with neurodegeneration in both diseases. In disease, characteristic C-terminal fragments of ~25 kDa ("TDP-25") accumulate in cytoplasmic inclusions. Here, we analyzed gain-of-function mechanisms of TDP-25 combining cryo-electron tomography, proteomics and functional assays. In neurons, cytoplasmic TDP-25 inclusions are amorphous and photobleaching experiments revealed gel-like biophysical properties far less dynamic than nuclear TDP-43. Compared to full length TDP-43, the TDP-25 interactome was depleted of lowcomplexity domain proteins. TDP-25 inclusions are enriched in 26S proteasomes adopting exclusively substrate-processing conformations, suggesting that inclusions sequester proteasomes, which are largely stalled and no longer undergo the cyclic conformational changes required for proteolytic activity. Reporter assays confirmed that TDP-25 impaired proteostasis, which is further enhanced by ALS-causing mutations. These findings bolster the importance of proteasome dysfunction in ALS/FTD. Key words ALS / neurodegeneration / phase separation / proteasome / TDP-43 Introduction

### Sample Protocol
Immunoprecipitation Transduced neurons were lysed in 2% Triton X-100, 750 mM NaCl, 1 mM KH2PO4, freshly supplemented with 67 U/ml Benzonase Nuclease, 1x protease inhibitor cocktail and 1x phosphatase inhibitor cocktail under constant rotation for 45min at 4Â°C and then centrifuged at low speed (1,000 x g, 5min, 4Â°C) to remove cellular debris but retain inclusion proteins (Hartmann et al, 2018). Supernatants were incubated with anti-GFP-antibody (Cat# N86/38, clone N86/38, UC Davis/NIH NeuroMab) pre-bound to Protein G Dynabeads (Cat# 10004D, Life Technologies) on a shaking rotator for 3h at 4Â°C. Beads were subsequently washed three times (150 mM NaCl, 50 mM Tris-HCl, pH 7.5, and 5% Glycerol) and analyzed by mass spectrometry.  Sample preparation for LC-MS/MS Beads were resuspended in 155 Î¼l denaturing buffer (8 M urea, 50 mM Tris-HCl pH 7.5, 1 mM dithiothreitol). Proteins were digested off the beads with LysC (220 ng/sample) for 1 h at room temperature while shaking at 1200 rpm. The suspension was diluted 4-fold to lower urea concentration permitting trypsin digestion. Iodoacetamide (final concentration 5 mM) to alkylate cysteines and trypsin (220 ng/sample) were added to digest for another hour before the supernatant was transferred to another tube and digested overnight. The digest was stopped by addition of trifluoroacetic acid to 1% v/v and half of the peptide solution was further processed by desalting chromatography on three disks of C18 material using the STAGE-tip format(Kulak et al, 2014). Briefly, STAGE-tips were washed with 100 Î¼l buffer B (50% v/v acetonitrile, 0.5% v/v acetic acid), conditioned with 100 Î¼l methanol, washed twice with 100 Î¼l buffer A (2% v/v acetonitrile, 0.5% v/v acetic acid), loaded with sample peptides, washed twice with 100 Î¼l buffer A, and subjected to peptide elution by 60 Î¼l of buffer B. The eluate was evaporated to dryness in a vacuum concentrator. Peptides were re-suspended in 10 Î¼l 2% v/v acetonitrile, 0.5% v/v acetic acid, 0.1% v/v trifluoroacetic acid and stored at -20Â°C. Peptide concentration was measured spectrophotometrically at 280 nm and 500 ng of peptide/sample were subjected to LC-MS/MS analysis.

### Data Protocol
MS raw data processing To process MS raw files, we employed the MaxQuant software version 1.6.17.0(Cox & Mann, 2008), searching against the UniProtKB rat FASTA database using canonical and isoform protein sequences downloaded in March 2018. Default search parameters were utilized unless stated differently. In brief, tryptic peptides with a minimum length of 7 amino acids, a maximum mass of 4600 Da, and two miscleavages at maximum were searched. Carbamidomethlyation was set as a fixed modification and methionine oxidation and protein N-terminal acetylation as variable modifications. A maximum of five modifications per peptide was permitted. A false discovery rate (FDR) cutoff of 1% was applied at the peptide and protein level. The search feature â€œMatch between runs,â€� which allows the transfer of peptide identifications in the absence of MS/MS-based identification after nonlinear retention time alignment was enabled with a maximum retention time window of 0.7 min. Protein abundances were normalized with the MaxLFQ label-free normalization algorithm built into MaxQuant(Cox et al, 2014).

### Publication Abstract
None

### Keywords
Als / neurodegeneration / phase separation / proteasome / tdp-43

### Affiliations
Max Planck Institute of Biochemistry, Martinsried, Germany
Proteomics

### Submitter
Mario Oroshi

### Lab Head
Dr Matthias Mann
Max Planck Institute of Biochemistry, Martinsried, Germany


