### Accession
PXD008965

### Title
KMT9 writes the histone mark H4K12me1 and controls metabolism and proliferation of castration-resistant prostate cancer cells

### Description
Posttranslational modifications of histones such as methylation regulate chromatin structure and gene expression. Methylation of histone lysine residues is generally performed by SET domain methyltransferases. Here, we identify the heterodimeric C21orf127/TRMT112 complex as a specific histone methyltransferase. Assembly of the seven-b-strand protein C21orf127 (also named Hemk2, N6amt1 or PrmC) with TRMT112 is essential to form an active enzyme, hereafter named KMT9 that writes the histone mark H4K12me1 in vitro and in vivo. The H4K12me1 mark is enriched at promoters of KMT9 target genes and co-localises with the active histone mark H4K12ac. By controlling expression of genes involved in energy metabolism, KMT9 regulates oxidative phosphorylation in androgen receptor-dependent and -independent prostate tumour cells. Importantly, KMT9 depletion severely affects proliferation of castration and enzalutamide-resistant prostate cancer cells and xenograft tumours. Together, our data link the writing of the H4K12me1 histone mark by KMT9 with KMT9-dependent gene expression, which in consequence regulates energy metabolism and proliferation. KMT9 executes these functions independently of androgen receptor and androgen signalling thus, providing a promising paradigm for the treatment of castration resistant prostate cancer.

### Sample Protocol
In vitro methylation assays Samples containing purified Histone H4, purified Methyltransferase and isotopically coded S-adenosylmethionine were digested with ArgC according to the manufacturers instructions. The resulting peptide mixture were desalted using stage tips with C18 material (Empore C18, 3M), dried in a speedvac and redissolved in 0.1% TFA for LC-MS analysis. LC separations were conducted with a linear gradient from 3 to 40% acetonitrile in 50 min and the eluting peptides were directly infused into the mass spectrometer via a nanoESI source. The Orbitrap HF was run in data dependent acquisition mode with one survey scan at 60.000 resolution and up to 10 MS/MS scans at 15.000 resolution. FT spectra were recalibrated using up to three lockmasses for silicones from the ambient air. MS/MS precursor ions required a charge state between 2+ and 6+. For MS/Ms acquisition, the underfill ratio was set to 5%, the normalized collision energy to 27 and the isolation width to 1.5 m/z. Previously seleced precursors were exclude for 20 sec from repeated ananalysis.

### Data Protocol
MS raw data were searched for elution profiles of methylated histones peptides with light (12C1H2, +14 Da) or heavy (13C2H2, +18 Da) label and MS/MS spectra were used to localize the modification within the peptide sequence in comparison of the MS/MS data of the unmodified peptide. Proteomic data were searched against a human database to confirm the protein presence and elution time of histone peptides.

### Publication Abstract
None

### Keywords
Lysine methylation, Chromatin, H4k12me1, Kmt9, Trmt112, Methyl-silac, Methyltransferase, Prostate cancer

### Affiliations
CECAD, University of Cologne, AG Kr√ºger
Protein Analysis Unit ZfP LMU Munich BioMedical Center Grosshaderner Strasse 9 82152 Planegg-Martinsried Tel. +49 89 2180 71804

### Submitter
Andreas Schmidt

### Lab Head
Dr Axel Imhof
Protein Analysis Unit ZfP LMU Munich BioMedical Center Grosshaderner Strasse 9 82152 Planegg-Martinsried Tel. +49 89 2180 71804


