### Accession
PXD025784

### Title
Differential protein expression in berry skin from red grapes with varying hybrid character

### Description
Protein expression from berry skin of four different red grape biotypes was compared at a proteome-wide level by bottom-up shotgun proteomics, label free quantification and MaxQuant-assisted computational analysis. Red grapes were from a purebred Vitis vinifera (Aglianico cv.), a V. vinifera (local Sciascinoso cv.) grafted onto an American rootstock, an interspecific hybrid (V. vinifera × V. labrusca, Isabel) and an uncharacterized red grape with some hybrid lineage, as demonstrated by the presence of relatively high amounts of anthocyanidin 3,5-O-diglucosides. The aim was assessing the differences among red grape biotypes at a protein expression levels, also addressing the possible effect of the grafting on the phenotypic expression of some key metabolic enzymes in grape berries.

### Sample Protocol
Red grape samples were collected in different vineyards located in Irpinia, within the Campania, one of the regions of Southern Italy, during grape harvest 2016. Grape samples were from: 1) a purebred Vitis vinifera (Aglianico cv.) obtained from ancient grapevines grown in the wine district of Taurasi; 2) a Vitis vinifera scion from a cultivar locally known as Sciascinoso grafted onto an American Vitis berlandieri x Vitis riparia Kober 5BB rootstock; 3) a Vitis vinifera x Vitis labrusca interspecific hybrid (Isabel cv.); 4) an uncharacterized grape varieties obtained from ungrafted old grapevines locally named “Tenta”, erroneously believed a cognate of the red-fleshed Teinturier (V. vinifera) variety. Proteins were extracted after powdering grape skins (5 g) with -20°C cold acetone. The acetone powder was resuspended in 20 mL of extraction buffer containing 0.7M sucrose, 0.5M Tris-HCl pH 8, 10 mM disodium EDTA salt, 0.4% (v/v) β-mercaptoethanol and a HaltTM protease inhibitor cocktail (Pierce/Thermo Scientific, Rockford, IL, USA) under magnetic stirring in ice-cold bath for 30 min and then centrifuged at 13000 g for 30 min. The supernatant was combined with an equal volume of Tris-buffered phenol (pH 8) and stirred for 3 h in ice-cold bath and finally centrifuged at 5000 g for 20 min at 4°C to allow phase separation. The upper phenol phase was collected, while the bottom aqueous layer was back-extracted with phenol and the organic phases were pooled. Proteins were precipitated with five volumes of ice-cold 0.1M ammonium acetate in methanol at -20°C overnight. After centrifugation (10000 g, 30 min, 4°C) the protein precipitate was washed three times with cold acetone and finally air dried. Proteins were reduced and Cys-alkylated with iodoacetamide

### Data Protocol
Nanoflow-high performance liquid chromatography-electrospray tandem mass spectrometry  (nHPLC−MS/MS) analyses were performed using an Ultimate 3000 nanoflow ultrahigh performance liquid chromatography (Dionex/Thermo Scientific, San Jose, CA) coupled to a Q Exactive Orbitrap mass spectrometer (Thermo Scientific). Eluent A was 0.1% formic acid (v/v) in LC-MS grade water; eluent B was 0.1% formic acid (v/v) in 80% aqueous acetonitrile. Peptides were separated using an EASY-Spray™ PepMap C18 column (15 cm × 75 μm i.d.) with 2 μm particles and 100-Å pore size (Thermo Scientific). Peptides were separated applying a 2–50% gradient of B over 120 min after 10 min of isocratic elution at 2% B, at a constant flow rate of 300 nL/min. MS1 precursor spectra were acquired in the positive ionization mode scanning the 1600–350 m/z range with resolving power of 70,000 full width at half maximum (FWHM), an automatic gain control (AGC) target of 1×106 ions, and maximum ion injection time of 100 ms. The spectrometer operated in full scan MS1 and data-dependent acquisition mode, selecting up to the 10 most intense ions for MS/MS fragmentation and applying an 8-s dynamic exclusion. Raw files of the nHPLC−MS/MS runs were used as the output for the protein identification using the Andromeda search engine of the MaxQuant bioinformatic suite against the Vitis spp. Uniprot database downloaded in July 2020. For all the searches, parameters were the following: mass tolerance value 10 ppm for the precursor and 0.02 for the fragment ions, trypsin as the proteolytic enzyme, Cys carbamidomethylation as a static modification; Met oxidation, pyroglutamic acid at N-terminus Gln, Ser/Thr phosphorylation as variable modifications. Peptide Spectrum Matches (PSMs) were filtered using the target decoy database approach with an e value of 0.01 peptide-level false discovery rate (FDR), corresponding to a 99% confidence score. Technical replicates (n=3) were merged and considered as a unique experiment set.

### Publication Abstract
Protein expression from the berry skin of four red grape biotypes with varying hybrid character was compared at a proteome-wide level to identify the metabolic pathways underlying divergent patterns of secondary metabolites. A bottom-up shotgun proteomics approach with label-free quantification and MaxQuant-assisted computational analysis was applied. Red grapes were from (i) purebred <i>Vitis vinifera</i> (Aglianico <i>cv</i>.); (ii) <i>V. vinifera</i> (local Sciascinoso <i>cv</i>.) grafted onto an American rootstock; (iii) interspecific hybrid (<i>V. vinifera</i> &#xd7; <i>V. labrusca</i>, Isabel), and (iv) uncharacterized grape genotype with hybrid lineage, producing relatively abundant anthocyanidin 3,5-<i>O</i>-diglucosides. Proteomics supported the differences between hybrids and purebred <i>V. vinifera</i> grapes, consistently with distinct phenotypic metabolite assets. Methanol <i>O</i>-anthraniloyltransferase, which catalyses the synthesis of methyl anthranilate, primarily responsible for the "foxy" odour, was exclusive of the Isabel hybrid grape. Most of the proteins with different expression profiles converged into coordinated biosynthetic networks of primary metabolism, while many possible enzymes of secondary metabolism pathways, including 5-glucosyltransferases expected for hybrid grapes, remained unassigned due to incomplete protein annotation for the <i>Vitis</i> genus. Minor differences of protein expression distinguished <i>V. vinifera</i> scion grafted onto American rootstocks from purebred <i>V. vinifera</i> skin grapes, supporting a slight influence of the rootstock on the grape metabolism.

### Keywords
Red grape; berry skin; hybrid grapes; proteomics; glucosyltransferases; grafting

### Affiliations
Institute of Food Sciences - National Research Council (CNR), Avellino, Italy
Institute of Food Sciences - National Research Council (CNR)

### Submitter
Gianluca Picariello

### Lab Head
Dr Gianluca Picariello
Institute of Food Sciences - National Research Council (CNR), Avellino, Italy


