### Accession
PXD028288

### Title
LC-MSMS with isobaric tagging of Pgrmc1 KO mouse liver membranes expressing Flag-Pgrmc1 or Y113F Flag-Pgrmc1

### Description
Progesterone receptor membrane component 1 (PGRMC1) is a heme binding protein implicated in a wide range of cellular functions. Our previous studies showed that PGRMC1 binds to cytochromes P450 in yeast and mammalian cells and promotes activity of these enzymes. Recently, the paralog PGRMC2 was shown to function as an intracellular heme chaperone. Here, we examined the function of the Pgrmc1 Y113F mutant that is defective for heme iron coordination by measuring levels of membrane proteins in livers from Pgrmc1 knockout mice infected with AAV8 expressing either GFP, Flag-Pgrmc1, or Y113F Flag-Pgrmc1. These experiments (1) confirmed findings in uninfected wild-type and Pgrmc1 knockout mice and (2) revealed that Pgrmc1 Y113F functions to stabilize cytochrome P450 enzymes similar to wild-type Pgrmc1.

### Sample Protocol
All animal procedures were reviewed and approved by the Johns Hopkins Institutional Animal Care and Use Committee (IACUC). Male mice aged 8-12 weeks were used for this study. Mice were sacrificed 8 days after tail vein infection with 5x1011 particles of AAV8 GFP, Flag-Pgrmc1, or Y113F Flag-Pgrmc1; livers were harvested and snap frozen. Liver membrane fractions were prepared as described previously (Raucy & Lasker, 1991, Methods Enzymol.) with some minor modifications. Frozen tissue (~1 g) was thawed on ice in 5 tissue volumes of Homogenization Buffer (100 mM Tris-HCl, pH 7.4, 100 mM KCl, 100 mM EDTA) and homogenized with an immersion blender (Bamix, Switzerland) until homogenization was completed based on visual inspection of the sample. The homogenate was cleared by spinning at 10,000 x g for 30 min at 4ºC. The supernatant was spun at 100,000 x g for 90 min at 4ºC. The pellet was washed with Resuspension Buffer (100 mM sodium pyrophosphate, pH 7.4, 1 mM EDTA) and spun at 100,000 x g for 60 min at 4ºC. The pellet was defined as the enriched membrane fraction. It was resuspended in Storage Buffer (50 mM potassium phosphate, pH 7.4, 0.1 mM EDTA, 20% glycerol) to 10-20 mg protein/mL. All buffers contained 1X protease inhibitors (Protease Complete, Roche).   One milligram of liver membrane protein in 150 µL microsome Storage Buffer (6.67 mg/ml) was diluted to a final concentration of 2 mg/ml in 1 mM MgCl2. The sample was treated with 10 units of benzonase (EMD Millipore) on ice for 30 min. Next, an equal volume of 2X TAP lysis buffer [12 mM Na2HPO4, 8 mM NaH2PO4, pH 7.5, 150 mM NaCl, 4 mM EDTA, 2% (w/v) n-Dodecyl-β-D-maltoside (DDM, ACROS Organics)] was added to dilute the sample to 1 mg/ml, then samples were sonicated at 4°C.   Three biological replicates per condition were subjected to quantitative proteomics. The samples were digested with trypsin and labeled with unique Tandem Mass Tag (TMT) 10-plex reagent (Thermo Fisher) according to the manufacturer’s protocol. The combined labeled sample was cleaned up from excess TMT tag with a detergent removal spin column (Pierce). The labeled samples were resuspended in an aqueous 5% (v/v) methanol and 0.5% (v/v) formic acid solution and then desalted via Oasis HLB cartridges (Waters). Briefly, cartridges were conditioned with 100% (v/v) methanol and equilibrated with 100% (v/v) water. Samples were loaded and then washed twice with 5% (v/v) methanol in water. Peptides were then eluted with 100% (v/v) methanol and dried via vacufuge.  Peptides were then fractionated via Agilent 3100 OFFGEL isoelectric focusing gel in a twelve well setup according to the manufacturer’s protocol. Upon completion, fractions were separately desalted with Pierce C18 spin columns (Thermo Fisher) following the provided protocol. Column resin was activated with 50% (v/v) acetonitrile and equilibrated with 5% (v/v) acetonitrile + 0.5% (v/v) trifluoroacetic acid solutions. Fractions were loaded onto columns and washed with 5% (v/v) acetonitrile + 0.5% (v/v) trifluoroacetic acid solution. Samples were eluted via two cycles of addition of 70% (v/v) acetonitrile solution to the columns. All sample fractions were dried separately via vacufuge.   Each fraction was then analyzed by liquid chromatography tandem mass spectrometry (LC-MS/MS) using an nLC-1200 nano-flow liquid chromatography system (Thermo Fisher) interfaced with a Q-Exactive mass spectrometer (Thermo Fisher). Precursor and fragment ions were analyzed via full scan (resolution of 70,000, automatic gain control target of 3e6, and 40 ms maximum injection time) and a data-dependent MS2 top 10 scan (resolution of 17,500, automatic gain control target of 5e4, 150 ms maximum injection time, 0.8 m/z isolation window, 10 second dynamic exclusion period, and normalized collision energy of 27), respectively.

### Data Protocol
Data was searched against the Mus musculus 10090 Uniprot reference proteome (download date: August 26, 2015) using the Sequest HT search engine running through Proteome Discoverer v.2.1 (Thermo Fisher) with one or two missed cleavages allowed and a precursor mass tolerance of 8 ppm and fragment mass tolerance of 0.02 Da. The trypsin protease cleavage sites permitted were only on the C-terminal sides of lysine and arginine. Dynamic modifications of methionine oxidation and deamidation of asparagine and glutamine, along with static modification of carbamidomethylation of cysteine were permitted. Peptide assignments were validated using the Target Decoy Peptide Spectral Match Validator node with a relaxed false discovery rate (FDR) of <0.05 and a strict FDR of <0.01. Peptides where the “Spectrum File” prefix was “04” were further analyzed. Included peptides had 1) a Master Accession number, 2) were marked “Unique,” 3) had an isolation interference that was ≤ 30%, and 4) the Peptide Quan Info was used.   For quantitative analysis of the membrane proteome of AAV8 infected Pgrmc1 KO liver, all reporter ion intensities that belong to the same protein were used as the measure (sum) of that protein abundance in the sample. Relative protein abundances were quantified by a robust median sweep algorithm (Kammers et al., 2015, EuPA Open Proteom.) except the measure of a protein’s abundance in a sample was the sum of the peptides. For statistical analysis, conditions were analyzed in R by linear modeling with limma and p-values adjusted by the Benjamini and Hochberg FDR method (Ritchie, et al., 2015, Nucleic Acids Res.).

### Publication Abstract
Progesterone receptor membrane component 1 (PGRMC1) is a heme-binding protein implicated in a wide range of cellular functions. We previously showed that PGRMC1 binds to cytochromes P450 in yeast and mammalian cells and supports their activity. Recently, the paralog PGRMC2 was shown to function as a heme chaperone. The extent of PGRMC1 function in cytochrome P450 biology and whether PGRMC1 is also a heme chaperone are unknown. Here, we examined the function of Pgrmc1 in mouse liver using a knockout model and found that Pgrmc1 binds and stabilizes a broad range of cytochromes P450 in a heme-independent manner. Proteomic and transcriptomic studies demonstrated that Pgrmc1 binds more than 13 cytochromes P450 and supports maintenance of cytochrome P450 protein levels posttranscriptionally. In&#xa0;vitro assays confirmed that Pgrmc1 KO livers exhibit reduced cytochrome P450 activity consistent with reduced enzyme levels. Mechanistic studies in cultured cells demonstrated that PGRMC1 stabilizes cytochromes P450 and that binding and stabilization do not require PGRMC1 binding to heme. Importantly, Pgrmc1-dependent stabilization of cytochromes P450 is physiologically relevant, as Pgrmc1 deletion protected mice from acetaminophen-induced liver injury. Finally, evaluation of Y113F mutant Pgrmc1, which lacks the axial heme iron-coordinating hydroxyl group, revealed that proper iron coordination is not required for heme binding, but is required for binding to ferrochelatase, the final enzyme in heme biosynthesis. PGRMC1 was recently identified as the causative mutation in X-linked isolated pediatric cataract formation. Together, these results demonstrate a heme-independent function for PGRMC1 in cytochrome P450 stability that may underlie clinical phenotypes.

### Keywords
Flag-pgrmc1, Lc-msms, Mouse, Pgrmc1, Isobaric tag, Y113f flag-pgrmc1, Liver, Membrane, Tmt

### Affiliations
Department of Cell Biology School of Medicine Johns Hopkins University Baltmore, MD USA
Johns Hopkins University

### Submitter
Peter Espenshade

### Lab Head
Dr Peter J. Espenshade
Department of Cell Biology School of Medicine Johns Hopkins University Baltmore, MD USA


