### Accession
PXD019058

### Title
Ribosome protein composition in wt and 23S+5S rRNA ribosomes

### Description
The 120-nt long 5S rRNA, is an indispensable component of cytoplasmic ribosomes in all living organisms. The functions of 5S rRNA and the reasons for its evolutionary preservation as an independent molecule remain unclear. Here we used ribosome engineering to investigate whether maintaining 5S rRNA as an independent molecule is critical for ribosome function and cell survival. By fusing circularly permutated 5S rRNA (cp5S) with 23S rRNA and deleting all wild type 5S rRNA genes, we generated an Escherichia coli strain completely devoid of free 5S rRNA. Viability of the engineered cells demonstrates that autonomous 5S rRNA is not required for cell growth at 37Â°C and is unlikely to have essential functions outside the ribosome. The fully-assembled ribosomes carrying 23S-cp5S hybrid rRNA and lacking free 5S rRNA are highly active in translation. However, the engineered cells accumulate aberrant 50S subunits that are unable to form stable 70S ribosomes. Cryo-EM analysis revealed a dramatically malformed peptidyl transferase center in the misassembled 50S subunits. The results of our experiments argue that the key evolutionary force preserving the autonomous nature of the smallest rRNA is its role in ribosome biogenesis.

### Sample Protocol
Protein composition of wt and mutant tight-couple 70S ribosomes and 50S ribosomal subunits, prepared as described above, was determined using quantitative mass-spectrometry as described previously (56). Ribosomal preparations were mixed in a 1:1 molar ratio with SILAC-labeled wt ribosomes containing mass-labeled arginine (Arg6: [13C]6HN4O2 and lysine (Lys4: C6H10[2H]4N2O2) (Silantes GmbH, Germany). Ribosomal proteins were digested with trypsin (Sigma) or Lys-C protease (Wako, USA). Resulting peptides were fractionated and analyzed via LC-MS/MS using LTQ-Orbitrap XL (Thermo Scientific). (56)-Lilleorg, S. et al. Bacterial ribosome heterogeneity: Changes in ribosomal protein composition during transition into stationary growth phase. Biochimie 156, 169-180 (2019).

### Data Protocol
Protein identification and quantification analysis was performed using Maxquant v1.5.6.0 (57) and Perseus v1.6.2.3 (58). Maxquant search was done using E. coli MG1655 protein sequence database from UniProtKB (24.04.2019). Protein abundance was normalized for large and small subunit proteins separately by shifting mean L/M ratio to 1. (57)-Cox, J. & Mann, M. MaxQuant enables high peptide identification rates, individualized p.p.b.-range mass accuracies and proteome-wide protein quantification. Nat. Biotechnol. 26, 1367-1372 (2008). (58)-Tyanova, S. et al. The Perseus computational platform for comprehensive analysis of (prote)omics data. Nat. Methods 13, 731-740 (2016).

### Publication Abstract
5S rRNA is an indispensable component of cytoplasmic ribosomes in all species. The functions of 5S rRNA and the reasons for its evolutionary preservation as an independent molecule remain unclear. Here we used ribosome engineering to investigate whether 5S rRNA autonomy is critical for ribosome function and cell survival. By linking circularly permutated 5S rRNA with 23S rRNA we generated a bacterial strain devoid of free 5S rRNA. Viability of the engineered cells demonstrates that autonomous 5S rRNA is dispensable for cell growth under standard conditions and is unlikely to have essential functions outside the ribosome. The fully assembled ribosomes carrying 23S-5S rRNA are highly active in translation. However, the engineered cells accumulate aberrant 50S subunits unable to form stable 70S ribosomes. Cryo-EM analysis revealed a malformed peptidyl transferase center in the misassembled 50S subunits. Our results argue that the autonomy of 5S rRNA is preserved due to its role in ribosome biogenesis.

### Keywords
Proteins, Ribosome, Mass-spectrometry

### Affiliations
Institute of molecular and cellular biology, University of Tartu, Estonia
University of Tartu

### Submitter
Kaspar Reier

### Lab Head
Dr Tiina Tamm
Institute of molecular and cellular biology, University of Tartu, Estonia


