### Accession
PXD008899

### Title
Re-analysis of Hela phosphoproteome (PXD000612) with TagGraph

### Description
Thousands of protein post-translational modifications (PTMs) dynamically impact nearly all cellular functions. Although mass spectrometry is suited to PTM identification, it has historically been biased towards a few with established enrichment procedures. To measure all possible PTMs across diverse proteomes, software must overcome two fundamental challenges: intractably large search spaces and difficulty distinguishing correct from incorrect identifications. Here, we describe TagGraph, software that overcomes both challenges with a string-based search method that is orders of magnitude faster than current approaches, and a probabilistic validation model optimized for PTM assignments. When applied to a human proteome map, TagGraph triples confident identifications while revealing thousands of modification types spanning nearly one million sites across the proteome. We show new contexts for highly abundant yet understudied PTMs such as proline hydroxylation. TagGraph expands our ability to survey the full proteomic landscape of PTMs, shedding new light on their tissue-specific functions.

### Sample Protocol
See PXD000612 for data acquisition details for these 24 raw data files spannign three treatment conditions of Hela cells

### Data Protocol
"TagGraph was used to analyze proteome data sets described PXD000612. All available MS/MS were first de novo sequenced using PEAKS ver 7. The resulting peptide sequences and raw mass spectra (mzXML-formatted) were given as input to TagGraph (https://sourceforge.net/projects/taggraph/; manuscript pending).  In parallel, data were searched with SEQUEST in standard (50 ppm) and OpenSearch (500 Da) modes; with PEAKS-PTM, and MS Fragger  PEAKS de novo parameters:  fragment ion mass tolerance of 0.01 Da.  Precursor mass tolerance =  10 ppm. Cysteine carbamidomethylation as a static modification, and methionine oxidation as a differential modification.   De novo sequencing results were searched with TagGraph (ver 1.4) against the human proteome (Uniprot, downloaded 12/9/2014) plus common contaminants without any additional decoy sequences.   TagGraph empirically optimizes precursor mass tolerances as part of its hierarchical Bayes scoring model. Fragment ion tolerances were initially set to 20 ppm of the precursor mass.  We also define the quantity ""modification mass tolerance"", which describes whether a particular modification agrees with one represented in the Unimod resource, or a novel modification specific to the present dataset.  This tolerance was set to 0.1 Da. Enzyme specificity was set to Trypsin for the human proteome data set. Although enzyme specificity was considered as a scoring attribute in the hierarchical Bayes model, TagGraph is able to return high-confidence semi specific and nonspecific peptide-spectrum matches regardless of the input enzyme specificity.  We applied a probability threshold of 0.99 (log10 (1-P) â‰¥ 2). SEQUEST parameters: Trypsin protease specificity, 50 ppm precursor ion mass tolerance, and 0.5 Da fragment ion mass tolerance. Cysteine carbamidomethylation (+57.021464 Da) was set as a static modification, oxidized methionines (+15.994915 Da) and phosphorylation (+79.99633) was set as a differential modification on serine, threonine and tyrosines. OpenSearch (SEQUEST) Parameters: Identical to SEQUEST, except 500 Da mass tolerance and no modifications specified. OpenSearch (MSFragger) parameters: 500 Da precursor mass tolerance; 10 ppm precursor true tolerance; 20 ppm fragment mass tolerance; no isotope error correcton; semispecific Trypsin digestion; one missed cleavage; clipped N-terminal Methionine; cysteine carbamidomethylation (static modification); variable oxidized methionine; multiple modifications on one residue; three variable modifications per modification type; up to 1000 variable modifications per peptide; at least 15 fragment ion peaks per MS/MS spectrum. PEAKS-PTM: same parameters as PEAKS (de novo) plus LysC enzyme specificity allowing nonspecific cleavage at both peptide ends; conider 485 unimod modfications

### Publication Abstract
Although mass spectrometry is well suited to identifying thousands of potential protein post-translational modifications (PTMs), it has historically been biased towards just a few. To measure the entire set of PTMs across diverse proteomes, software must overcome the dual challenges of covering enormous search spaces and distinguishing correct from incorrect spectrum interpretations. Here, we describe TagGraph, a computational tool that overcomes both challenges with an unrestricted string-based search method that is as much as 350-fold faster than existing approaches, and a probabilistic validation model that we optimized for PTM assignments. We applied TagGraph to a published human proteomic dataset of 25&#x2009;million mass spectra and tripled confident spectrum identifications compared to its original analysis. We identified thousands of modification types on almost 1&#x2009;million sites in the proteome. We show alternative contexts for highly abundant yet understudied PTMs such as proline hydroxylation, and its unexpected association with cancer mutations. By enabling broad characterization of PTMs, TagGraph informs as to how their functions and regulation intersect.

### Keywords
Hela, Taggraph, Ptm, Posphorylation

### Affiliations
Chan Zuckerberg Biohub
Dept of Chemical & Systems Biology, Stanford University, Stanford CA  94305

### Submitter
Joshua Elias

### Lab Head
Dr Joshua E Elias
Dept of Chemical & Systems Biology, Stanford University, Stanford CA  94305


