### Accession
PXD031197

### Title
TurboID-based proximity labeling to identify LONP1-targeting proteins

### Description
It is documented that intracellular succinate levels are maintained and regulated by mitochondrial complexes II. In order to define the LONP1-dependent regulators of succinate content, unbiased TurboID-based proximity labeling was employed to exploit LONP1-targeting proteins. Our results showed that 63.5% of the proteins identified by LONP1-N-TurboID-based proximity labeling are mitochondrial proteins. Functional protein association networks analysis of the identified proteins indicated that these biotinylated proteins are related to carboxylic acid metabolic process, mitochondrial organization and mitochondrial gene expression.

### Sample Protocol
293T cells used in proteomics experiments were stably expressing the indicated LONP1-N-Turbo constructs. TurboID samples were labeled with 500 M biotin for 30 min. Labeling was stopped by placing cells on ice and washing gently with cold DPBS twice. The cells were collected and pelleted by centrifugation at 1,300 rpm for 3 min at 4°C. The supernatant was removed, and the pellet was lysed by resuspension in RIPA lysis buffer (50 mM Tris pH 8.0, 150 mM NaCl, 0.1% SDS, 1% Triton X-100, 1x protease inhibitor cocktail, and 100 mM PMSF) and incubation for 30 min at 4°C. Lysates were clarified by centrifugation at 12,000 rpm for 10 min at 4°C. For enrichment of biotinylated proteins, 200 L Pierce NeutrAvidin Agarose beads (ThermoFisher #29200) were washed twice with RIPA lysis buffer and incubated with clarified lysates with rotation at 4°C overnight. The beads were then washed twice with 1 mL of PBS, and twice with 1 mL of RIPA lysis buffer. The biotinylated proteins were eluted by boiling the beads at 95°C for 15 min in 30 L 3× protein loading buffer supplemented with 20 mM DTT and 2 mM biotin. The supernatant was collected by centrifugation at 12,000 rpm for 10 min, and transferred to a new Eppendorf tube, frozen on ice, and shipped for further processing and preparation for LC-MS/MS analysis and label-free quantification. The proteins separated on SDS-PAGE followed by the MS-compatible Coomassie brilliant blue staining and in gel digestion. Briefly, the protein bands were excised from the gel and cut into approximately 1-mm3 slices. The gel slices were washed with 50 mM ammonium bicarbonate:acetonitrile (1:1) three times, SpeedVac-dried and digested by trypsin at 1:50 trypsin-to-protein mass ratio for overnight at 37℃. The resultant peptides were extracted three times from the gel by vortexing in 10% formic acid:acetonitrile (1:1) for 15 min, and then dried by SpeedVac. The peptides were resuspended with 15uL of 3% acetonitrile 2% formic acid before being subjected to nanoLC-MS/MS analysis.

### Data Protocol
The original MS data were submitted to ProteinPilot Software (version 4.5, AB SCIEX) with Paragon Algorithm (4.5.0.0 1654) for data analysis. MS/MS data were searched against Homo Sapiens in UniProt database (April 9, 2016, containing 160,566 sequences, http://www.uniprot.org/proteomes/UP000005640). Trypsin/P was specified as cleavage enzyme allowing up to three missing cleavages, four modifications per peptide and two to five charges. Mass error was set to 20 ppm for first search, 5 ppm for main search, and 0.02 Da for fragmented ions. False discovery rate thresholds for protein, peptide, and modification site were specified at 1%. Minimum peptide length was set at seven. All the other parameters in ProteinPilot were set to default values. In addition, ProteinProspector (version 5.20.0) was used for search compare and protein quantification.

### Publication Abstract
None

### Keywords
Human, Turboid, Lonp1

### Affiliations
Medical School of Nanjing University, China
Nanjing University Medical School

### Submitter
Lei Fang

### Lab Head
Dr Lei Fang
Medical School of Nanjing University, China


