### Accession
PXD028396

### Title
Klebsiella pneumoniae x546,ATCC15380, TMT

### Description
Klebsiella pneumoniae x546,ATCC15380, TMT,The samples were ground into a powder in liquid nitrogen. Then the powder was suspended in lysis buffer [1% sodium deoxycholate (SDS), 8M urea]. The mixture was allowed to settle at 4 鎺矯 for 30 min during which the sample were vortexed at every 5 min, and treated by ultrasound at 40 kHz and 40 W for 2 min. After centrifugation at 16000 rpm at 4鎺矯 for 30 min, the concentration of protein supernatant was determined by Bicinchoninic acid (BCA) method by BCA Protein Assay Kit (Pierce, Thermo, USA). Protein quantification was performed according to the kit protocol(Chen et al., 2021)

### Sample Protocol
Protein digestion was performed according to the standard procedure and the resulting peptide mixture was labeled using the 10-plex TMT reagent (Thermo fisher, Art.No.90111). Briefly, total protein (100 娓璯) taken from each sample was mixed with 100 娓璍 of the lysate. TCEP (10 mM) was added and then the mixture was stored at 37 閳╋拷 for 60 min. Then iodoacetamide (40 mM) was added to the mixture and it was stored in dark at room temperature for 40 mins. Six folds volumes of cold acetone were added to precipitate the protein at -20 鎺矯 for 4 h. After centrifugation at 10000 rpm at 4鎺矯 for 20 min, the pellet was re-suspended with 100 纰宭 50 mM triethylammonium bicarbonate (TEAB) buffer. Trypsin was added at a 1:50 trypsin-to-protein mass ratio and incubated at 37 鎺矯 overnight. Then, one unit of TMT reagent was thawed and reconstituted in 50 纰孡 acetonitrile. After tagging for 2 h at room temperature, hydroxylamine was added to react with mixture for 15 min at room temperature(Dai et al., 2020).  In this work, the strains grown under control (Fermentation of strain domesticated at 40 g/L glycerol concentration without betaine) and optimal conditions (Fermentation of strain domesticated at 120-20 g/L glycerol concentration with betaine) at 10 h (each sample with two biological replicates) were collected by centrifugation (12000 rpm, 10 min at 4 閳╋拷) and frozen in liquid nitrogen respectively. The samples were labeled as A1, A2, B1 and B2. Finally, all samples were pooled, desalted and vacuum-dried for next use.

### Data Protocol
The RAW data files were analyzed by Proteome Discoverer (Thermo Scientific, Version 2.2) against the K. pneumoniae database (https://www.uniprot.org/taxonomy/?query= ATCC15380 &sort=score). The MS/MS search criteria were as follows: Mass tolerance of 20 ppm for MS and 0.02 Da for MS/MS Tolerance, trypsin as the enzyme with 2 missed cleavage allowed, carbamido methylation of cysteine and the TMT of N-terminus and lysine side chains of peptides as fixed modification, and methionine oxidation as dynamic modifications, respectively. The false discovery rate (FDR) of peptide identification was set as FDR 閳拷 0.01. A minimum of one unique peptide identification was used to support protein identification(Wang et al., 2020c).   A total of 3284 proteins expressed were identified as belonging to the proteome of K. pneumoniae in this study. The thresholds of fold change (FC) (>1.2 or <0.83) and P-value <0.05 were used to identify differentially expressed proteins (DEPs). Then the 170 up-regulated and 291 down-regulated proteins were found. Annotation of all identified proteins was performed by GO (http://geneontology.org/) and KEGG pathway (http://www.genome.jp/kegg/). DEPs were further used to for GO and KEGG enrichment analysis. Protein-protein interaction analysis was performed using the String v10.5.

### Publication Abstract
None

### Keywords
Atcc15380, Tmt, Klebsiella pneumoniae x546

### Affiliations
Tian Jin university
Tianjin university

### Submitter
xin wang

### Lab Head
Dr Wang Xin
Tianjin university


