### Accession
PXD019204

### Title
An azidoribose probe to track ketoamine adducts in histone ribose glycation (293T nuclei - LFQ)

### Description
Reactive cellular metabolites can modify macromolecules and form adducts known as non-enzymatic covalent modifications (NECMs). Dissecting the mechanisms, regulation and consequences of NECMs, such as glycation, has been challenging due to the complex and often ambiguous nature of the adducts formed. Directly tracking the formation of modifications on key targets to uncover their underlying physiological importance requires specific chemical tools. Here we present the novel chemoenzymatic syntheses of an active azido-modified ribose analog, 5-azidoribose (5-AR), as well as an inactive control derivative, 1-azidoribose (1-AR) and their application towards understanding protein ribose-glycation in vitro and in cellulo. With these new probes we found that, similar to MGO-glycation, ribose glycation specifically accumulates on histones. In addition to fluorescent labeling, we demonstrate the utility of the probe in enriching modified targets, which were identified by label-free quantitative proteomics and highresolution MS/MS workflows. Finally, we establish that the known oncoprotein and hexose deglycase, fructosamine 3-kinase (FN3K), recognizes and facilitates the removal of 5-AR glycation adducts in live cells, supporting the dynamic regulation of ribose glycation as well as validating the probe as a new chemical tool to monitor FN3K activity. Altogether, we demonstrate this probe’s utilities to uncover ribose-glycation and deglycation events as well as tracking FN3K activity towards establishing its potential as a new cancer vulnerability.

### Sample Protocol
Cy5-handle-facilitated Enrichment of Glycated Nuclear Extracts: Nuclei (5 million cells per sample) were isolated, glycated, labeled with DBCO Cy5. After removing the excess click reagent and washing with PBS, the pellets were resuspended in 100 µL of RIPA buffer (PBS pH 7.5, 1% NP-40. 0.5% weight/volume sodium deoxycholate, 0.1% SDS) and lysed by sonication. Next, nuclear lysates were incubated with Dynabeads Protein G (Thermo Fisher Scientific) reversibly bound to anti-Cy5 (20 µL of beads with 2 µL of 1.5 mg/mL antiCy5 (Sigma C1117) per sample) for 1 hour on ice. The beads were then washed three times with RIPA buffer and twice with PBS before enrichment evaluation or proteomics analyses (see below). On-bead enrichment evaluation samples were boiled in sample buffer, then, inputs, flow-through, and denatured beads were separated by SDS-PAGE, imaged by in gel fluorescence and subjected to western blot analysis to validate the enrichment.  LFQ Sample Preparation and Analysis: Three biological replicates of nuclear lysates were incubated with 1-AR or 5-AR, labeled with DBCO Cy5, and enriched as described above. The proteins were subjected to on-bead digestion using sequencing grade modified trypsin (Promega) for 3.5 hours. The supernatant was withdrawn and reduced/alkylated with DTT/IAA. A second round of tryptic digestion was carried out overnight. Peptides were analyzed using a Dionex 3000 Ultimate HPLC equipped with a NCS3500RS nano- and microflow pump (Dionex). Peptides were loaded onto a 100 µm x 20 mm Acclaim PepMap C18 trap column (Thermo Fisher Scientific). A 70-minute linear gradient was applied across a 75 µm x 120 mm pulled-emitter nanocolumn (Nikkyo Technos). Peptides were analyzed using a Q-Exactive HF mass spectrometer (Thermo Fisher Scientific). Data was recorded in positive mode with Top 20 DDA acquisition. Data analysis was performed with the MaxQuant software.7 Oxidation of methionine and acetylation of protein N-termini were set as dynamic modifications, carbamidomethylation of cysteine was set as a static modification, and standard settings were applied.

### Data Protocol
The search results from MaxQuant were analyzed by Perseus. Briefly, the control 1-AR and 5-AR-labeled sample replicates were grouped correspondingly. Protein abundances were measured using iBAQ intensities and were log2 transformed. Missing values were imputed with random numbers from a normal distribution, whose mean and standard deviation were chosen to best simulate low abundance values below the noise level (replaced missing values by normal distribution width = 0.3 and shift = 2.2). An FDR student’s t-test (FDR 0.05) was applied to test for significance.

### Publication Abstract
Reactive cellular metabolites can modify macromolecules and form adducts known as nonenzymatic covalent modifications (NECMs). The dissection of the mechanisms, regulation, and consequences of NECMs, such as glycation, has been challenging due to the complex and often ambiguous nature of the adducts formed. Specific chemical tools are required to directly track the formation of these modifications on key targets in order to uncover their underlying physiological importance. Here, we present the novel chemoenzymatic synthesis of an active azido-modified ribose analog, 5-azidoribose (<b>5-AR</b>), as well as the synthesis of an inactive control derivative, 1-azidoribose (<b>1-AR</b>), and their application toward understanding protein ribose-glycation <i>in vitro</i> and <i>in cellulo</i>. With these new probes we found that, similar to methylglyoxal (MGO) glycation, ribose glycation specifically accumulates on histones. In addition to fluorescent labeling, we demonstrate the utility of the probe in enriching modified targets, which were identified by label-free quantitative proteomics and high-resolution MS/MS workflows. Finally, we establish that the known oncoprotein and hexose deglycase, fructosamine 3-kinase (FN3K), recognizes and facilitates the removal of <b>5-AR</b> glycation adducts in live cells, supporting the dynamic regulation of ribose glycation as well as validating the probe as a new platform to monitor FN3K activity. Altogether, we demonstrate this probe's utilities to uncover ribose-glycation and deglycation events as well as track FN3K activity toward establishing its potential as a new cancer vulnerability.

### Keywords
Ribose, Proteomics, Epigenetics, Histones, Click chemistry, Glycation

### Affiliations
Tri-Institutional Program, New York, NY, United States; Chemical Biology Program, Memorial Sloan Kettering Cancer Center, New York, NY, United States; Department of Pharmacology, Weill Cornell Medical College, New York, NY, United States; Department of Physiology, Biophysics and Systems Biology, Weill Cornell Medical College, New York, NY, United States
Tri-Institutional PhD Program in Chemical Biology

### Submitter
Igor Maksimovic

### Lab Head
Dr Yael David
Tri-Institutional Program, New York, NY, United States; Chemical Biology Program, Memorial Sloan Kettering Cancer Center, New York, NY, United States; Department of Pharmacology, Weill Cornell Medical College, New York, NY, United States; Department of Physiology, Biophysics and Systems Biology, Weill Cornell Medical College, New York, NY, United States


