### Accession
PXD001610

### Title
Proteomic Analysis of the Response to Cell Cycle Arrests in Human Myeloid Leukemia Cells

### Description
Previously (Ly et al. 2014), we analyzed protein abundance changes across a â€˜minimally perturbedâ€™ cell cycle in human cells (NB4), using centrifugal elutriation to differentially enrich distinct cell cycle phases. Here, we compare data from elutriated cells with NB4 cells arrested at comparable phases using serum starvation, hydroxyurea, or RO-3306. While elutriated and arrested cells have similar patterns of DNA content and cyclin expression, a large fraction of the proteome changes detected in arrested cells are found to reflect arrest-specific responses (i.e., starvation, DNA damage, CDK1 inhibition), rather than physiological cell cycle regulation. For example, we show most cells arrested in G2 by CDK1 inhibition express abnormally high levels of replication and origin licensing factors, and are likely poised for genome re-replication. The protein data are available in the Encyclopedia of Proteome Dynamics (http://www.peptracker.com/epd/), an online, searchable resource.

### Sample Protocol
For protein extraction, NB4 cells were pelleted, washed twice with cold PBS and then lysed in 0.3 â€“ 1.0 ml urea lysis buffer (8 M urea, 100 mM Tris pH 7.4, Roche PhosStop). Lysates were vigorously mixed for 30 min at room temperature and homogenized using a Branson Digital Sonifier (30% power, 30 s). Proteins were reduced with TCEP (25 mM in denaturing urea buffer), for 15 min at room temperature and alkylated with iodoacetamide (55 mM in denaturing urea buffer), in the dark for 45 min at room temperature. Lysates were diluted with digest buffer (100 mM Tris pH 8.0 + 1 mM CaCl2) to reach 4 M urea, and then digested with 1:50 Lys-C (Wako Chemicals) overnight at 37 Â°C. The lysates were then further diluted with digest buffer to reach 0.8 M urea and digested with trypsin (1:50, Roche) for four hours at 37 Â°C. Digest efficiencies were checked by  SDS-PAGE analysis and Coomassie protein staining. The digests were then desalted using SepPak-C18 SPE cartridges, dried, and resuspended in 50 mM borate, pH 9.3. Peptides were separated onto a Dionex Ultimate 3000 HPLC system equipped with an AS24 strong anion exchange column, using a similar protocol to the hSAX method described previously (Ritorto et al. 2013). Peptides were chromatographed using a borate buffer system, namely 10 mM sodium borate, pH 9.3 (Buffer A) and 10 mM sodium borate, pH 9.3 + 0.5 M sodium chloride (Buffer B) and eluted using an exponential elution gradient into 12 x 750 ml fractions. The peptide fractions were pooled into four (F1, F2+F3, F4+F5+F6, F7+F8+F9+F10+F11+F12), desalted using SepPak-C18 SPE plates, and resuspended in 5% formic acid for LC-MS/MS analysis. Peptides were analyzed using a Dionex RSLCnano HPLC-coupled Q-Exactive Orbitrap mass spectrometer (Thermo Fisher Scientific). Peptides were first loaded onto a 2 cm PepMap trap column in 2% acetonitrile + 0.1% formic acid. Trapped peptides were then separated on an analytical column (75 mm x 50 cm PepMap-C18 column) using the following mobile phases: 2% acetonitrile + 0.1% formic acid (Solvent A) and 80% acetonitrile + 0.1% formic acid (Solvent B). The linear gradient began with 5% B to 35% B over 220 min with a constant flow rate of 200 nl/min. The peptide eluent flowed into a nanoelectrospray emitter at the front end of a Q-Exactive (quadrupole Orbitrap) mass spectrometer. A typical â€˜Top10â€™ acquisition method was used. Briefly, the primary mass spectrometry scan (MS1) was performed in the Oribtrap at 70,000 resolution. Then, the top 10 most abundant m/z signals were chosen from the primary scan for collision-induced dissociation in the HCD cell and MS2 analysis in the Orbitrap at 17,500 resolution. Precursor ion charge state screening was enabled and all unassigned charge states, as well as singly charged species, were rejected.

### Data Protocol
The RAW data files produced by the mass spectrometer were analysed using the quantitative proteomics software MaxQuant, version 1.5.0.0 (Cox & Mann 2008). This version of MaxQuant includes an integrated search engine, Andromeda (Scheltema et al. 2011). The database supplied to the search engine for peptide identifications was a UniProt human protein database (â€˜Human Reference Proteomeâ€™ retrieved on February 17, 2014) combined with a commonly observed contaminants list. The initial mass tolerance was set to 7 p.p.m. and MS/MS mass tolerance was 20 ppm. The digestion enzyme was set to trypsin/P with up to 2 missed cleavages. Deamidation, oxidation of methionine and Gln->pyro-Glu were searched as variable modifications. Identification was set to a false discovery rate of 1%. To achieve reliable identifications, all proteins were accepted based on the criteria that the number of forward hits in the database was at least 100-fold higher than the number of reverse database hits, thus resulting in a false discovery rate of less than 1%. Protein isoforms and proteins that cannot be distinguished based on the peptides identified are grouped by MaxQuant and displayed on a single line with multiple UniProt identifiers. The label free quantitation (LFQ) algorithm in MaxQuant was used for protein quantitation. The algorithm has been previously described (Cox 2014). Protein quantitation was performed on unmodified peptides and peptides that have modifications that are known to occur during sample processing (pyro-  Glu, deamidation). All resulting MS data were integrated and managed using PepTracker Data Manager, a laboratory information management system (LIMS) that is part of the PepTracker software platform (http://www.PepTracker.com).  Quantitative protein and peptide output from MaxQuant was analyzed using R (version 3.0.3) as implemented in the RStudio interactive development environment (version 0.98.932). In addition to the FDR thresholds implemented by default in MaxQuant, a further cleaning step was performed to improve the quality and value of the dataset. This step involved removing proteins with less than 2 peptide identifications, and those labeled as either contaminants, or reverse hits.  We employed two data mining strategies to identify enriched pathways and networks. The first method uses p-value and fold change thresholds to identify a set of proteins that reproducibly change in abundance. In this study, a one-way ANOVA was used to calculate the p-value. The protein set that meets the threshold criteria is then compared against the proteome to identify enriched annotations, such as gene ontology terms. The critical thresholds used were a p-value of less than 0.05 and a fold change (between any treatment) greater than or equal to two. Scaled protein abundances were then clustered into 4 clusters using the Ward algorithm (Ward 1963). Enriched gene ontology annotations were identified using the DAVID web resource. The second method uses threshold-free algorithms to detect enriched gene sets. The Gene Set Enrichment Analysis (GSEA) tool (Subramanian et al. 2005) was used to find enriched KEGG (Kanehisa & Goto 2000) and Reactome (Croft et al. 2011) pathways. These pathways were then visualized using Cytoscape Desktop (Shannon et al. 2003) and the Cytoscape plug-in, EnrichmentMap (Isserlin et al. 2014).

### Publication Abstract
Previously, we analyzed protein abundance changes across a 'minimally perturbed' cell cycle by using centrifugal elutriation to differentially enrich distinct cell cycle phases in human NB4 cells (Ly et al., 2014). In this study, we compare data from elutriated cells with NB4 cells arrested at comparable phases using serum starvation, hydroxyurea, or RO-3306. While elutriated and arrested cells have similar patterns of DNA content and cyclin expression, a large fraction of the proteome changes detected in arrested cells are found to reflect arrest-specific responses (i.e., starvation, DNA damage, CDK1 inhibition), rather than physiological cell cycle regulation. For example, we show most cells arrested in G2 by CDK1 inhibition express abnormally high levels of replication and origin licensing factors and are likely poised for genome re-replication. The protein data are available in the Encyclopedia of Proteome Dynamics (

### Keywords
Human, Lc-msms

### Affiliations
Principle Investigator Proteomics
School of Life Sciences

### Submitter
Vackar Afzal

### Lab Head
Dr Angus Lamond
Principle Investigator Proteomics


