### Accession
PXD016150

### Title
Proteomic responses to drought vary widely among eight diverse genotypes of rice (Oryza sativa)

### Description
Rice is a critically important food source but yields worldwide are vulnerable to periods of drought. We exposed eight genotypes of upland and lowland rice (Oryza sativa L. ssp. japonica and indica) to drought stress at the late vegetative stage and harvested leaves for protein extraction and subsequent label-free shotgun proteomics. Gene ontology analysis revealed some differentially expressed proteins were induced by drought in all eight genotypes; we speculate that these play a universal role in drought tolerance. However, some highly genotype-specific patterns of response to drought suggest that some mechanisms of metabolic reprogramming are not universal. Such proteins had largely uncharacterized functions, making them biomarker candidates for drought tolerance screens.

### Sample Protocol
Plants from 8 different rice genotypes were grown in a temperature controlled greenhouse; Nipponbare, Doongara, IAC1131, Mahsuri, Reiziq, N22, IR2006-P12 and IDSA77. Plants were grown to late vegetative stage and then exposed to drought stress, with leaf samples collected at the point of severe stress. Proteins were extracted from leaf tissue using trichloroacetic acid – acetone extraction and precipitation. In-solution digested peptides were separated and identified using nanoflow reversed-phase liquid chromatography – tandem orbitrap mass spectrometry on a Thermo Q-exactive. Peptides and proteins were identified using Global Proteome Machine software.

### Data Protocol
Raw MS data files were converted to mzXML format and processed through the global proteome machine (GPM) software version 2.2.1 of the X!Tandem algorithm, which is available from http://www.thegpm.org. Oryza sativa protein sequence databases containing 48932 sequences were downloaded from Uniprot as of March 2018. MS/MS spectra for 8 genotypes belonging to Oryza sativa were searched against Oryza sativa protein database. Common human and trypsin peptide contaminants were also included, and a reversed sequence database was analysed to evaluate the false discovery rate. Peptide to spectrum matching search parameters included a peptide mass tolerance of 4 Da, a fragment mass tolerance of 20 ppmtolerance of up to one missed tryptic cleavages and K/R-P cleavages. Fixed modifications were set for carbamidomethylation of cysteine and variable modifications were set for oxidation of methionine.

### Publication Abstract
Rice is a critically important food source but yields worldwide are vulnerable to periods of drought. We exposed eight genotypes of upland and lowland rice (<i>Oryza sativa</i> L. ssp. <i>japonica</i> and <i>indica</i>) to drought stress at the late vegetative stage, and harvested leaves for label-free shotgun proteomics. Gene ontology analysis was used to identify common drought-responsive proteins in vegetative tissues, and leaf proteins that are unique to individual genotypes, suggesting diversity in the metabolic responses to drought. Eight proteins were found to be induced in response to drought stress in all eight genotypes. A total of 213 proteins were identified in a single genotype, 83 of which were increased in abundance in response to drought stress. In total, 10 of these 83 proteins were of a largely uncharacterized function, making them candidates for functional analysis and potential biomarkers for drought tolerance.

### Keywords
Drought stress, Shotgun proteomics, Label-free quantitation, Mass spectrometry, Rice

### Affiliations
Macquarie University
Department of Molecular Sciences, Macquarie University, North Ryde, NSW 2109, Australia

### Submitter
Sara Hamzelou

### Lab Head
Dr Paul A. Haynes
Department of Molecular Sciences, Macquarie University, North Ryde, NSW 2109, Australia


