### Accession
PXD004236

### Title
In vivo protein interaction network analysis reveals porin-localized toxin inactivation in antibiotic resistant bacteria

### Description
The nosocomial pathogen Acinetobacter baumannii is a frequent cause of hospital acquired infections worldwide, and a challenge for treatment due to its evolved resistance to antibiotics, including carbapenems.  To gain insight on A. baumannii antibiotic resistance mechanisms, we analyzed the protein interaction network of a multidrug-resistant A. baumannii clinical strain Ab5075.  Using in vivo chemical cross-linking and mass spectrometry, we identified 2,068 non-redundant cross-linked peptide pairs containing 245 intra- and 398 inter- molecular interactions.  Outer membrane proteins OmpA and YiaD, and carbapenemase Oxa-23 are hubs of the identified interaction network.  Eighteen novel interactors of Oxa-23 were identified.  Interactions of Oxa-23 with outer membrane porins OmpA and CarO were verified with co-immunoprecipitation analysis.  Furthermore, transposon mutagenesis of oxa-23 or interactors of Oxa-23 demonstrated changes in meropenem or imipenem sensitivity in Ab5075.  These results provide the first view of a porin-localized toxin inactivation model and increase understanding of bacterial antibiotic resistance mechanisms.

### Sample Protocol
A. baumannii cells were grown with LB media to late log phase (OD600=1.0).  Cells were pelleted and washed with PBS three times to remove free amines which may interfere with cross-linking reactions.  Cells were cross-linked in the buffer of 500 mM Na2HPO4 (pH 7.4) and 150 mM NaCl, with three doses of BDP-NHP cross-linkers with an  incubation time of 3 H and with shaking at 1,300 rpm with a thermomixer at room temperature.  This cross-linking condition was confirmed not to lyse the Ab5075 cells (Fig. 2).  Reactions were quenched by adding 20 mM Tris-HCl (pH 8.0) to reaction mixtures.  Cell pellets were collected with centrifugation at 3,000 g at 4‚Å∞C for 15 min for proteomics analysis.  Three biological replicates were analyzed, which means that cells were grown up three times, and independently cross-linked and analyzed with LC-MS.

### Data Protocol
The raw data of MS2 precursor mass and MS3 fragmentation spectra were converted to mzXML files with ReAdW (version 4.2.1). The mzXML data were searched against a database containing A. baumannii forward and reverse protein sequences (total of 3482 entries), using Comet (version 2015.01 rev.02).  The 1,741 proteins included in the subset database were identified as putative BDP-NHP reactive proteins as described in supplemental experimental procedures.  Search settings included precursor mass tolerance 20 ppm, 13C isotope offsets (-1/0/1/2/3) enabled, peptide termini fully tryptic, maximum internal cleavage 3, fragment ion tolerance 1.0005 Da (with 0.4 Da fragment offset), fixed modification cysteine carbamidomethylation, variable modifications lysine stump (197.032422 Da), protein N-termini stump (197.032422 Da), and methionine oxidation.  The false discovery rate for peptide identification (FDR) was determined by the ratio of (the number of reverse hits) to (the number of target hits).  Peptide identification threshold (FDR < 5 %) was used to obtain peptide candidates that were included to assign cross-linked peptide pairs.  Furthermore, peptide candidates that were mapped to cross-link pairs were required to 1) have an internal lysine with stump modification; 2) C-terminal cleaved lysine residues cannot be a stump modified lysine; 3) the peptide can be paired to another peptide that is also identified < 5% FDR in the same ReACT cycle; 4) in cases that two MS3 events for the same precursors yielded different peptide IDs, the ID with the better expectation value was selected.   Using this procedure, a total of 22,647 cross-linked peptide pairs could be assigned.  In addition, decoy peptides that passed FDR < 5 % threshold were also mapped to PIR relationship to construct decoy cross-link pairs (containing one or both peptide arms with decoy peptides), and 440 decoy pairs were observed.  The false discovery rate for the cross-linked peptide pairs (FDRlink) was estimated with the ratio of (the number of decoy pairs) to (the number of target pairs), which was 440/22,647 = 1.9 %.

### Publication Abstract
The nosocomial pathogen Acinetobacter baumannii is a frequent cause of hospital-acquired infections worldwide and is a challenge for treatment due to its evolved resistance to antibiotics, including carbapenems. Here, to gain insight on A. baumannii antibiotic resistance mechanisms, we analyse the protein interaction network of a multidrug-resistant A. baumannii clinical strain (AB5075). Using in vivo chemical cross-linking and mass spectrometry, we identify 2,068 non-redundant cross-linked peptide pairs containing 245 intra- and 398 inter-molecular interactions. Outer membrane proteins OmpA and YiaD, and carbapenemase Oxa-23 are hubs of the identified interaction network. Eighteen novel interactors of Oxa-23 are identified. Interactions of Oxa-23 with outer membrane porins OmpA and CarO are verified with co-immunoprecipitation analysis. Furthermore, transposon mutagenesis of oxa-23 or interactors of Oxa-23 demonstrates changes in meropenem or imipenem sensitivity in strain AB5075. These results provide a view of porin-localized antibiotic inactivation and increase understanding of bacterial antibiotic resistance mechanisms.

### Keywords
Chemical cross-linking, Carbapenem, Acinetobacter baumannii, Antibiotic resistance, Protein-protein interactions

### Affiliations
University of Washington
Department of Genome Sciences, University of Washington, Seattle, USA

### Submitter
Xia Wu

### Lab Head
Dr James E. Bruce
Department of Genome Sciences, University of Washington, Seattle, USA


