### Accession
PXD019652

### Title
Comprehensive Histone Epigenetics (CHEpi): A mass spectrometry based screening assay to measure epigenetic toxicity

### Description
Epigenetics have taken center stage in the study of diseases such as cancer, diabetes, and neurodegeneration. For nearly thirty years, it has been known that cancer cells exhibit abnormal DNA methylation patterns. In contrast, the large scale analysis of histone posttranslational modifications (hPTMs) has lagged behind because classically, histone modification analysis has relied on site specific antibody-based techniques. Mass spectrometry (MS) is a technique that holds the promise to picture the histone code comprehensively in a single experiment. Therefore, we developed an MS-based method that is capable of tracking all possible hPTMs in an untargeted approach. In this way, trends in single and combinatorial hPTMs can be reported and enable prediction of the epigenetic toxicity of compounds. Moreover, this method is based on the use of human embryonic stem cells (hESCs), thereby omitting the need to sacrifice laboratory animals. Improving the workflow and the user-friendliness of the assay is an ongoing effort. Still, this novel toxicoepigenetic assay and the data it generates holds great potential for, among others, pharmaceutical industry, food science, clinical diagnostics and, environmental toxicity screening.

### Sample Protocol
After harvesting hESCs, histones were extracted as described by Govaert et al. First, the nuclei were isolated from frozen cell pellets by resuspension in hypotonic lysis buffer (HLB). A number of 106 cells was resuspended in 200 µL of HLB (10 mM Tris-HCl pH 8.0, 1 mM KCl, 1.5 mM MgCl2) supplemented with 1 mM dithiothreitol (DTT), Halt protease, 1 V/V% EDTA-free phosphatase inhibitor cocktail (788441, Thermo Fisher), and 1 V/V%  phosphatase inhibitor cocktails II and III (P5726 and P0044, Sigma-Aldrich). Subsequently, this mixture was incubated for 30 minutes on a rotator at 4 °C after which the nuclei were pelleted, and the supernatant was discarded. The pellet was resuspended in 125 µL of 0.4 N HCl and incubated for 30 minutes on a rotator at 4 °C. Next, the histones were precipitated with 33% trichloroacetic acid on ice for 30 minutes. A histone-load corresponding to 400,000 cells was quantified by gel-electrophoresis on an 8-16 % TGX gel (Biorad). Firstly, histone proteins consist of high levels of lysines (K) and arginines (R) compared to other proteins. This has consequences for the protein digestion, the process by which proteins are broken down into smaller peptides. In proteomics, the commonly used enzyme is trypsin, which specifically cleaves behind K and R. However, the use of trypsin for digestion of histone proteins results in relatively short tryptic peptides and would vary based on the modification status of the substrate histones. Therefore, propionlylation is performed as this modifies all free primary amines, resulting in an Arg-C specificity instead of a tryptic specificity. Secondly, short tryptic peptides will show less retention on a C18 column during reversed-phase (RP) high-performance liquid chromatography (HPLC). The presence of a propionyl-group increases the hydrophobicity, thereby enhancing the retention and separation of the peptides during HPLC. The propionylation reaction was performed as previously described. Briefly, the remaining purified histones of each sample were first vacuum dried and then dissolved in 20 µL 1 M triethylammonium bicarbonate (TEAB) buffer, pH 8.5. Next, 20 μL of propionylation reagent (propionic anhydride: 2-propanol 1:80 (v/v)) was added for incubation of 30 minutes at room (RT). This was followed by adding 20 µL MilliQ water (Merck Millipore) for 30 minutes at 37 °C. Next, histones were digested overnight at 37 °C using trypsin (enzyme/histone ratio of 1:20 (m/m)) in 500 mM TEAB, supplemented with calciumchloride (CaCl2) and acetronitrile (ACN) to a final concentration of 1.0 mM and 5%, respectively. Subsequently, the derivatization reaction was repeatedly implemented to cap the newly formed peptide N-termini. Non-specific overpropionylation at serine (S), threonine (T) and tyrosine (Y) was reversed by resuspending the vacuum dried sample in 50 μL 0.5 M NH2OH and 15 μL NH4OH at pH 12 for 20 minutes at RT. Finally, 30 μl of 100% formic acid (FA) was added, and the propionylated histones were vacuum dried for further analysis.   Among the wider scientific community, data-dependent acquisition (DDA) is the preferred untargeted acquisition strategy, mainly due to the straightforward data analysis. Because data analysis of histone samples is notoriously difficult, DDA was adopted as the acquisition strategy for this workflow. First, the propionylated histone peptides were resuspended in 0.1% FA. Next, a beta-galactosidase (Sciex) solution in 0.1% FA (25 fmol bèta-galactosidase/µL) was added to ensure that an injection volume of 5 μL results in 2 μg histones and 50 fmol beta-galactosidase standard on column. Prior to MS-analysis, peptides were separated with HPLC using a low pH RP gradient on the NanoLC 425 system operating in microflow mode (5µl/min). A Triart C18 150 × 0.3mm column (YMC) was used at 5 μl/min flow rate (0.1% FA with 3% DMSO) with a 60 min gradient from 3%–45% ACN in 0.1% FA for a total run time of 75 minutes per sample. The sample list was randomized and interspersed with injections of propionylated bovine histone standards (Roche). For the MS-analysis, a TripleTOF 5600 (Sciex) was used. Each cycle in one full MS1 scan (m/z 400-1250) consisted of 250 ms followed by MS2 data-dependent trigger events (m/z 65-2000, high sensitivity mode). A maximum of 10 candidate ions (charge state +2 to +5) exceeding 300 cps was monitored per cycle and excluded for 10 s, with an accumulation time of 200 ms and using a rolling collision energy with a spread of 15V. The cycle time was 2.3 s, in order to obtain 10 to 12 data points per LC peak. The combination of HPLC with MS/MS results in a multitude of MS1 and MS2 spectra i.e. graphs in which the intensity is plotted as a function of the mass-to-charge ratio (m/z).

### Data Protocol
Raw data from all runs were imported and all runs in a single experiment were aligned in Progenesis QIP 3.0 (Nonlinear Dynamics, Waters) following feature detection. Features are here defined as precursor m/z values in MS1 scans eluting within a limited time frame and displaying an isotopic envelope resembling that of peptides. Many of these can be expected to be peptidoforms derived from histones. To identify these peptidoforms, a predefined number of MS/MS spectra closest to the elution apex were selected per feature and exported for searches using Mascot (Matrix Science). It should be noted that Mascot is a probabilistic algorithm able to run without a more conventional decoy database. Currently, by lack of a decoy strategy that can cope with complexly modified peptides (e.g. histones), such algorithms are strongly encouraged. To ensure untargeted screening, searches are best performed with a set of 9 PTMs. To determine the best PTM set the twenty MS/MS spectra closest to the elution apex were selected and merged into a single ∗.mgf file. A standard search without biological modifications was performed using a complete Human Swissprot database (downloaded from Uniprot and supplemented with contaminants from the cRAP database (https://www.thegpm.org/crap/)) to identify the proteins present in the sample (assuming at least a single non-modified peptide can be detected for each protein). From these proteins, a FASTA database was generated. Next, based on a smaller exported *.mgf file consisting of only the three MS/MS spectra closest to the elution apex per feature, a Mascot-search was performed. In this search, the following parameters were included: 1) mass error tolerances for the precursor ions and its fragment ions were set at 10 ppm and 50 ppm respectively; 2) enzyme specificity was set to Arg-C, allowing for up to one missed cleavage site; 3) variable modifications included acetylation, butyrylation, crotonylation, trimethylation and formylation on K, methylation on R, dimethylation on both K and R, deamidation on asparagine (N), glutamine (Q) and R and oxidation of methionine (M); and 4) N-terminal propionylation and propionylation on K were set as fixed modifications. Database searching was performed against the above mentioned custom-made FASTA database.  Next, the result files containing all the identified peptidoforms were exported in *.xml-format, after which they were imported back into Progenesis QIP 3.0 to annotate the features from which they were derived. Features that were annotated as peptidoforms derived from histones were also manually validated to resolve isobaric near-coelution. Based on the area under the curve (AUC) of these features, percentages of individual hPTMs, or relative abundances (RA), were calculated by dividing the AUC for each peptidoform containing the considered hPTM by the sum of the AUCs for all observed forms of that peptide. In this way, we are able to report trends in single hPTMs. Reporting these trends is useful when, for example, hESCs are exposed to different concentrations of a compound or when a time-lapse experimental design is used. Moreover, these trends enables prediction of the epigenetic toxicity of compounds. As a result, this workflow could be an important advance in the discovery of epidrugs and in the field of personalized medicines.

### Publication Abstract
Evidence of the involvement of epigenetics in pathologies such as cancer, diabetes, and neurodegeneration has increased global interest in epigenetic modifications. For nearly thirty years, it has been known that cancer cells exhibit abnormal DNA methylation patterns. In contrast, the large-scale analysis of histone post-translational modifications (hPTMs) has lagged behind because classically, histone modification analysis has relied on site specific antibody-based techniques. Mass spectrometry (MS) is a technique that holds the promise to picture the histone code comprehensively in a single experiment. Therefore, we developed an MS-based method that is capable of tracking all possible hPTMs in an untargeted approach. In this way, trends in single and combinatorial hPTMs can be reported and enable prediction of the epigenetic toxicity of compounds. Moreover, this method is based on the use of human cells to provide preliminary data, thereby omitting the need to sacrifice laboratory animals. Improving the workflow and the user-friendliness in order to become a high throughput, easily applicable, toxicological screening assay is an ongoing effort. Still, this novel toxicoepigenetic assay and the data it generates holds great potential for, among others, pharmaceutical industry, food science, clinical diagnostics and, environmental toxicity screening. &#x2022;There is a growing interest in epigenetic modifications, and more specifically in histone post-translational modifications (hPTMs).&#x2022;We describe an MS-based workflow that is capable of tracking all possible hPTMs in an untargeted approach that makes use of human cells.&#x2022;Improving the workflow and the user-friendliness in order to become a high throughput, easily applicable, toxicological screening assay is an ongoing effort.

### Keywords
Histone post-translational modifications, Screening, Toxicoepigenetics, Proteomics, Lc-ms/ms, Drug safety, Pharmacoepigenetics

### Affiliations
ProGenTomics, Laboratory of Pharmaceutical Biotechnology, Ghent University, B-9000 Ghent, Belgium
Faculity of Pharmaceutical Biotechnology

### Submitter
Maarten Dhaenens

### Lab Head
Dr Maarten Dhaenens
ProGenTomics, Laboratory of Pharmaceutical Biotechnology, Ghent University, B-9000 Ghent, Belgium


