### Accession
PXD000133

### Title
Equine Cerebrospinal Fluid - data analysis

### Description
Cerebrospinal fluid is a diagnostic biofluid that is reflective of the overall health of the patient.  Proteomic profiling of human CSF has been performed on a variety of disease states, but a comprehensive proteomic profile of equine CSF has not been completed until now.  A total of 320 proteins were confidently identified in a pooled sample representative of 6 healthy horses.  Gene Ontology terms were mapped in Uniprot, and normalized spectral abundance factors were calculated as a measure of relative abundance. Briefly, CSF was collected from healthy horses via subarachnoid catheter or manual draw, protease inhibitors were added (Pierce, Rockford, IL), and samples were frozen at -80oC (Figure 1).  Protein concentrations were determined via Bradford Assay  (Thermo Scientific, Rockford, IL) and 30 ug of each sample underwent in-solution digestion using ProteaseMAX (Promega, Madison, WI) and urea.  Samples were solubilized in 8 M urea, 0.2% protease max, then reduced, alkylated, and digested with 1% protease max and trypsin at 37oC for 3 hours.  Samples were dried in a Speed VacÂ® vacuum centrifuge, desalted using Pierce PepClean C18 spin columns (Pierce, Rockford, IL), dried and resuspended in 30 ul 3% ACN, 0.1% formic acid.    All solvents, water, and acid were LC-MS/MS grade from Sigma (St. Louis, MO).  Online 2-dimensional LC-MS/MS with SCX (strong cation exchange) and subsequent reverse phase chromatography was performed as follows.  10 ug of digested peptides from a sample were loaded onto a Zorbax BIO-SCX II 3.5 umm, 50 x 0.8 mm column (Agilent Technologies, Santa Clara, CA).  Peptides were eluted off of the SCX column step-wise using increasing concentrations of NaCl in 0.3% ACN, 0.1% FA (20 ul NaCl salt injections:  15, 30, 45, 60, 75, 90, 120, 150, 300, 500 mM).  Peptides from each individual salt injection were then purified and concentrated using an on-line enrichment column (Agilent Zorbax C18, 5 um, 5 x 0.3mm).  Subsequent chromatographic separation was performed on a reverse phase nanospray column (Agilent 1100 nanoHPLC, Zorbax C18, 5um, 75 um ID x 150mm column) using a 60 minute linear gradient from 25%-55% buffer B (90% ACN, 0.1% formic acid) at a flow rate of 300 nanoliters/min.   Peptides were eluted directly into the mass spectrometer (Thermo Scientific LTQ linear ion trap) and spectra were collected over a range of 200-2000 m/z using a dynamic exclusion limit of 2 MS/MS spectra of a given peptide mass for 30 s (exclusion duration of 90 s).  Compound lists of the resulting spectra were generated using Bioworks 3.0 software (Thermo Scientific) with an intensity threshold of 5,000 and 1 scan/group.  This workflow generates 10 raw data files per sample. MS/MS spectra were searched against the NCBInr equine database concatenated to a reverse database (14,473 entries) using the Mascot database search engine (Matrix Science, version 2.3.02) and SEQUEST (version v.27, rev. 11, Sorcerer, Sage-N Research).  The following search parameters were used: average mass, peptide mass tolerance of 2.5 Da, fragment ion mass tolerance of 1.0 Da, complete tryptic digestion allowing one missed cleavage, variable modification of methionine oxidation, and a fixed modification of cysteine carbamidomethylation. Peptide identifications from both of the search engines were combined using protein identification algorithms in Scaffold 3 (Version 3.6.4, Proteome Software, Portland, OR).  All data files were then combined using the MudPIT option in Scaffold 3 generating a composite listing for all proteins identified across all runs.  Thresholds were set to 99% and 95% protein and peptide probability respectively, and a 2 unique peptide minimum was required.  The peptide false discovery rate (FDR) was less than 0.2% after manual validation of all proteins identified by 2 unique peptides.  Criteria for manual validation included the following: 1) a minimum of at least 3 theoretical y or b ions in consecutive order that are peaks greater than 5% of the maximum intensity; 2) an absence of prominent unassigned peaks greater than 5% of the maximum intensity; and 3) indicative residue specific fragmentation, such as intense ions N-terminal to proline and immediately C-terminal to aspartate and glutamate.

### Sample Protocol
Not available

### Data Protocol
Not available

### Publication Abstract
None

### Keywords
Proteomics, Csf, Equine

### Affiliations
Proteomics and Metabolomics Facility

### Submitter
Carolyn Broccardo

### Lab Head


