### Accession
PXD003281

### Title
A protein phosphatase network controls the temporal and spatial dynamics of differentiation commitment in human epidermis

### Description
Epidermal homeostasis depends on a balance between stem cell renewal and terminal differentiation. The transition between the two cell states,termed commitment, is poorly understood. Here we characterise commitment by integrating transcriptomic and proteomic data from disaggregated primary human keratinocytes held in suspension to induce differentiation. Cell detachment induces several protein phosphatases, five of which - DUSP6, PPTC7, PTPN1, PTPN13 and PPP3CA – promote differentiation by negatively regulating ERK MAPK and positively regulating AP1 transcription factors. Conversely, DUSP10 expression antagonises commitment. The phosphatases form a dynamic network of transient positive and negative interactions that change over time, with DUSP6 predominating at commitment. Boolean network modelling identifies a mandatory switch between two stable states (stem and differentiated) via an unstable (committed) state. Phosphatase expression is also spatially regulated in vivo and in vitro. We conclude that an auto-regulatory phosphatase network maintains epidermal homeostasis by controlling the onset and duration of commitment. 

### Sample Protocol
A pre-confluent keratinocyte culture was split into two separate cultures with FAD-lysine-arginine medium (Sigma) differentially supplemented either with K0R0 or with K8R10 (stable isotopes of amino acids Lysine and Arginine (Cambridge Isotope Laboratories)). Serum used for the SILAC medium was also depleted for Lysine and Arginine (Sigma). Cells were grown in the SILAC medium for 5-6 days to reach 70 â€“ 80% confluence and later harvested for downstream assays. For measurement of protein abundance changes by SILAC, lysates containing ~80 Î¼g protein were prepared in LDS sample buffer containing reducing agent (TCEP). Proteins were separated by electrophoresis on a 4-12% Bis-Tris NuPAGE gel, which was Coomassie-stained and cut into eight equally sized gel slices. Gel-embedded proteins were reduced with TCEP, alkylated with iodoacetamide, and trypsin-digested to release peptides. Peptides were extracted from gels using 50% acetonitrile (ACN) containing 5% formic acid (FA), dried and resuspended in 5% FA. Peptides were then analysed by LC-MS/MS on a Dionex RSLCnano coupled to a Q-Exactive Orbitrap classic instrument. Specifically, peptides were loaded onto a PepMap100 75Î¼m x 2cm trap column, which was then brought in-line with a 75Î¼m x 50cm PepMap-C18 column and eluted using a linear gradient over 220 min at a constant flow rate of 200 nl/min. The gradient composition was 5% to 35% B, where solvent A = 2% ACN + 0.1% FA and solvent B = 80% ACN + 0.1%FA. Peptides were eluted with a  linear elution gradient (5% B to 35% B) over 220 min with a constant flow rate of 200 nl/min. An initial MS scan of 70,000 resolution was acquired, followed by data-dependent MS/MS by HCD on the top 10 most intense ions of â‰¥2+ charge state at 17,500 resolution.  For phosphoenrichment analysis, lysates containing ~2 mg of protein were chloroform:methanol precipitated. Pellets were resuspended in 8M urea in digest buffer (0.1 M Tris pH 8 + 1 mM CaCl2), diluted to 4 M urea with additional digest buffer, and digested with Lys-C for 4h at 37Â°C. The digests were diluted to 1M urea and trypsin-digested overnight at 37Â°C. Digests were then acidified and desalted using SepPak-C18 vacuum cartridges. Desalted peptides were resuspended in mobile phase A for HILIC (80% ACN + 0.1% TFA). HILIC chromatography was performed using a Dionex Ultimate 3000 with a TSK Biosciences Amide-80 column (250 x 4.6 mm) (33-35). Peptides were eluted using an exponential gradient (80% B to 60% B) composed of A (above) and B (0.1% TFA) at 0.4 ml/min over 60 min. 16 fractions were collected from 25 min to 60 min. These fractions are enriched for hydrophilic peptides, including phosphopeptides. The fractions were dried before further phosphoenrichment by titanium dioxide (TiO2). For TiO2 enrichment, HILIC fractions were resuspended in loading buffer (70% ACN + 0.3% lactic acid + 3% TFA). 1.25 mg of TiO2 (GL Sciences) was added to each fraction and incubated for 10 min to bind phosphopeptides. Beads were washed with loading buffer, and two wash buffers, composed of (1) 70% ACN + 3% TFA and (2) 20% ACN + 0.5% TFA. Phosphopeptides were eluted in two steps, first with 4% of ammonium hydroxide solution (28% w/w NH3) in water for 1h and again with 2.6% of ammonium hydroxide solution in 50% ACN overnight. Elutions were collected, dried, resuspended in 5% FA and analysed by LC-MS/MS. The LC-MS analysis was performed similarly to above with the following modifications: peptides were chromatographed on an EasySpray PepMap 75 Î¼m x 50 cm column, and a â€˜Top30â€™ method was used where the top 30 most intense ions were chosen for MS/MS fragmentation.

### Data Protocol
A pre-confluent keratinocyte culture was split into two separate cultures with FAD-lysine-arginine medium (Sigma) differentially supplemented either with K0R0 or with K8R10 (stable isotopes of amino acids Lysine and Arginine (Cambridge Isotope Laboratories)). Serum used for the SILAC medium was also depleted for Lysine and Arginine (Sigma). Cells were grown in the SILAC medium for 5-6 days to reach 70 â€“ 80% confluence and later harvested for downstream assays. For measurement of protein abundance changes by SILAC, lysates containing ~80 Î¼g protein were prepared in LDS sample buffer containing reducing agent (TCEP). Proteins were separated by electrophoresis on a 4-12% Bis-Tris NuPAGE gel, which was Coomassie-stained and cut into eight equally sized gel slices. Gel-embedded proteins were reduced with TCEP, alkylated with iodoacetamide, and trypsin-digested to release peptides. Peptides were extracted from gels using 50% acetonitrile (ACN) containing 5% formic acid (FA), dried and resuspended in 5% FA. Peptides were then analysed by LC-MS/MS on a Dionex RSLCnano coupled to a Q-Exactive Orbitrap classic instrument. Specifically, peptides were loaded onto a PepMap100 75Î¼m x 2cm trap column, which was then brought in-line with a 75Î¼m x 50cm PepMap-C18 column and eluted using a linear gradient over 220 min at a constant flow rate of 200 nl/min. The gradient composition was 5% to 35% B, where solvent A = 2% ACN + 0.1% FA and solvent B = 80% ACN + 0.1%FA. Peptides were eluted with a  linear elution gradient (5% B to 35% B) over 220 min with a constant flow rate of 200 nl/min. An initial MS scan of 70,000 resolution was acquired, followed by data-dependent MS/MS by HCD on the top 10 most intense ions of â‰¥2+ charge state at 17,500 resolution.  For phosphoenrichment analysis, lysates containing ~2 mg of protein were chloroform:methanol precipitated. Pellets were resuspended in 8M urea in digest buffer (0.1 M Tris pH 8 + 1 mM CaCl2), diluted to 4 M urea with additional digest buffer, and digested with Lys-C for 4h at 37Â°C. The digests were diluted to 1M urea and trypsin-digested overnight at 37Â°C. Digests were then acidified and desalted using SepPak-C18 vacuum cartridges. Desalted peptides were resuspended in mobile phase A for HILIC (80% ACN + 0.1% TFA). HILIC chromatography was performed using a Dionex Ultimate 3000 with a TSK Biosciences Amide-80 column (250 x 4.6 mm) (33-35). Peptides were eluted using an exponential gradient (80% B to 60% B) composed of A (above) and B (0.1% TFA) at 0.4 ml/min over 60 min. 16 fractions were collected from 25 min to 60 min. These fractions are enriched for hydrophilic peptides, including phosphopeptides. The fractions were dried before further phosphoenrichment by titanium dioxide (TiO2). For TiO2 enrichment, HILIC fractions were resuspended in loading buffer (70% ACN + 0.3% lactic acid + 3% TFA). 1.25 mg of TiO2 (GL Sciences) was added to each fraction and incubated for 10 min to bind phosphopeptides. Beads were washed with loading buffer, and two wash buffers, composed of (1) 70% ACN + 3% TFA and (2) 20% ACN + 0.5% TFA. Phosphopeptides were eluted in two steps, first with 4% of ammonium hydroxide solution (28% w/w NH3) in water for 1h and again with 2.6% of ammonium hydroxide solution in 50% ACN overnight. Elutions were collected, dried, resuspended in 5% FA and analysed by LC-MS/MS. The LC-MS analysis was performed similarly to above with the following modifications: peptides were chromatographed on an EasySpray PepMap 75 Î¼m x 50 cm column, and a â€˜Top30â€™ method was used where the top 30 most intense ions were chosen for MS/MS fragmentation.  Description of the Data Analysis Methods:  Raw data was then processed using MaxQuant, which implements the Andromeda search engine, for peptide and protein identification and SILAC quantitation. As the data were collected over several years, several MaxQuant versions were used as follows (raw file labels in parentheses): 1.5.0.0 (PT4246), 1.3.0.5 (PT4026), 1.2.2.5 (PT954, PT961, PT981, PT1101, PT1142).  To identify differentially expressed proteins between time 0 and 4, 8 and 12h we collected per time point the proteins whose absolute ratios were >0.5. For each time point, the proteins were then split whether the ratio was positive or negative. 6 lists were annotated for GO terminology using GeneSpring. The extensive list of GO-terms was submitted to Revigo (36) to reduce complexity and the resulting GO-categories depicted in fig. S4. In order to evaluate the correlation between transcriptomics and proteomics we matched the 1155 significant proteins to their counterpart transcripts. Only fold  hanges to 0h (respectively for transcripts or proteins) were considered into the analysis. All data was merged into a matrix and analysed using Hopach library in R

### Publication Abstract
Epidermal homeostasis depends on a balance between stem cell renewal and terminal differentiation. The transition between the two cell states, termed commitment, is poorly understood. Here, we characterise commitment by integrating transcriptomic and proteomic data from disaggregated primary human keratinocytes held in suspension to induce differentiation. Cell detachment induces several protein phosphatases, five of which - DUSP6, PPTC7, PTPN1, PTPN13 and PPP3CA - promote differentiation by negatively regulating ERK MAPK and positively regulating AP1 transcription factors. Conversely, DUSP10 expression antagonises commitment. The phosphatases form a dynamic network of transient positive and negative interactions that change over time, with DUSP6 predominating at commitment. Boolean network modelling identifies a mandatory switch between two stable states (stem and differentiated) via an unstable (committed) state. Phosphatase expression is also spatially regulated in vivo and in vitro. We conclude that an auto-regulatory phosphatase network maintains epidermal homeostasis by controlling the onset and duration of commitment.

### Keywords
Human, U2os, Keratinocytes, Pkc, Timecourse

### Affiliations
Principle Investigator
School of Life Sciences

### Submitter
Vackar Afzal

### Lab Head
Dr Angus Lamond
Principle Investigator


