### Accession
PXD021214

### Title
Post-translational modification profiling of Saccharomyces cerevisiae histone methylation enzymes uncovers regulation by upstream signalling pathways

### Description
Histone methylation is central to the regulation of eukaryotic transcription. In Saccharomyces cerevisiae, it is controlled by a system of four methyltransferases (Set1p, Set2p, Set5p, and Dot1p) and four demethylases (Jhd1p, Jhd2p, Rph1p, and Gis1p). While the histone targets for these enzymes are well characterised, the connection of the enzymes with the intracellular signalling network and thus their regulation is poorly understood, in yeast and in all other eukaryotes. Here we report the detailed characterisation of the eight S. cerevisiae enzymes, and show that they carry a total of 75 phosphorylation sites, 93 acetylation sites, and two ubiquitination sites. All enzymes are subject to phosphorylation, although demethylases Jhd1p and Jhd2p contained one and five sites respectively whereas other enzymes carried 14 to 36 sites. Phosphorylation was absent or under-represented on catalytic and other domains but strongly enriched for regions of disorder on methyltransferases, suggesting a role in the modulation of protein-protein interactions. We show that a phosphorylation cluster within an acidic and intrinsically disordered N-terminal region of methyltransferase Set2p regulates H3K36 methylation levels in vivo, thus supporting the functional relevance of disordered phosphosites. While most kinases upstream of the yeast histone methylation enzymes remain unknown, we model the possible connections between the signalling network and the histone-based gene regulatory system and propose an integrated regulatory structure. Our results provide a foundation for future, detailed exploration of the role of specific kinases and phosphosites in the regulation of histone methylation.

### Sample Protocol
Protein extraction and affinity purification of hexahistidine-tagged enzymes was performed as per (Winter et al., 2018), with the exception that yeast cell pellets were resuspended in 40 mL of histidine purification lysis buffer, and that pan-phosphatase inhibitor (Roche, Basel, Switzerland) was added to purification buffers. Eluted proteins were concentrated and buffer-exchanged into phosphate-buffered saline (46.6 mM Na2HPO4, 3.4 mM NaH2PO4, 200 mM NaCl, pH 7.4) using Amicon Ultra 4 10K centrifugal filters (Merck Millipore, Burlington, MA) to a volume of 80 μL.Lysates (10 μL) and eluates (4 x 20 μL aliquots) were separated by polyacrylamide gel electrophoresis and prepared for liquid chromatography mass spectrometry according to (Hamey et al., 2016), with the addition of reduction and alkylation steps. This involved reducing gel bands with 10 mM DTT at 37°C for 1 h and their subsequent alkylation with iodoacetamide in the dark at ambient temperature for 1 h immediately before in-gel proteolytic digestion. Dried and rehydrated gel bands were incubated with either trypsin (Promega, Fitchburg, WI, sequencing-grade, 50 ng in 20 mM NH4HCO3), LysargiNase (Proteolysis Lab, IBMB-CSIC, 100 ng in 200 mM NH4HCO3, 10 mM CaCl2), Asp-N (Promega, sequencing-grade, 100 ng in 10 mM NH4HCO3) or chymotrypsin (Promega, sequencing-grade, 500 ng in 100 mM Tris-HCl (pH 8.0), 10 mM CaCl2) for 16 h in 100 μL reactions. Digests with trypsin, LysargiNase and Asp-N were at 37°C, while chymotryptic digestion was at 25°C. Resultant peptides were extracted from gel bands and dried down in a SpeedVac (Thermo Fisher Scientific, Waltham, MA) before resuspension in 0.1% (v/v) formic acid as described (Hamey et al., 2016). Peptide samples were separated and ionised by nano-electrospray liquid chromatography using established methods (Hart-Smith et al., 2011), and then analysed by high-resolution tandem mass spectrometry on an Orbitrap Fusion Lumos Tribrid Mass Spectrometer (Thermo Fisher Scientific) with data-dependent acquisition. Precursor scans were acquired in the orbitrap, and subsequently precursors were selected and isolated for fragmentation as described (Hart-Smith et al., 2011). Fragmentation using either higher-energy collisional dissociation (HCD) or electron transfer/higher-energy collisional dissociation (EThcD) was performed using standard instrument methods outlined in (Smith et al., 2020). HCD MS/MS used a normalised collision energy of 30%, while EThcD-MS/MS used a supplemental activation collision energy of 28% for singly to triply charged fragment ions and no supplemental activation for higher charge state fragments. Fragment ions were detected in the Orbitrap (resolution = 30,000, scan range = 110-2000 m/z, maximum injection time = 125 ms for HCD, 250 ms for EThcD).

### Data Protocol
To identify putative modification sites on histone MTases and DMases, mass spectrometry data were analysed on Proteome Discoverer 2.2 (Thermo Fisher Scientific) using basic PTM analysis processing and consensus workflows with primarily default settings (Taus et al., 2011). Data were searched against the SWISS-PROT (2019_06, 560,459 sequences) and contaminants (245 sequences) databases using the Mascot search engine (version 2.4, Matrix Sciences, London, United Kingdom). Search parameters were as follows: precursor and fragment ion mass tolerances were set to 5 ppm and 20 mmu, respectively; instrument was set to either ESI_HCD or EThcD depending on the fragmentation type; enzyme specificity was either Trypsin, LysargiNase, Asp-N¬_ambic or TrypChymo for respective preparations, with up to three missed cleavages permitted; dynamic modifications for all searches were carbamidomethyl (C), oxidation (M) and acetyl (peptide N-term), in addition to modifications exclusive to specific searches: phospho (STY) for phosphorylation; acetyl (K, protein N-term) for acetylation; methyl (K, DE), dimethyl (K) and trimethyl (K) for lysine methylation; methyl (R, DE) and dimethyl (R) for arginine methylation; GG (K) and RGG (K) for trypsin and LysargiNase ubiquitination searches, respectively; crotonyl (K) for crotonylation; ADP-ribosyl (C, D, E, K, N, R) for ADP-ribosylation; EQIGG (K) and REQIGG (K) for trypsin and LysargiNase SUMOylation searches, respectively. The ptmRS node was incorporated into processing workflows after database searching to provide localisation probabilities for putative modification sites in the search output (ptmRS Best Site Probabilities column). False discovery rate (FDR) for peptide-spectrum matches (PSMs) was determined by a decoy database search and PSMs for modified peptides (FDR < 0.01) were filtered by ion score (≥ 30), expectation value (p < 0.05) and site localisation probability (≥ 95%) and tabulated in Table S2. Fragmentation spectra for quality-filtered PSMs were manually inspected to verify unambiguous site localisation and to monitor the abundance of diagnostic neutral loss fragments that may assist in the confident identification of certain PTMs. High-confidence modification sites which satisfied our stringent thresholding and spectral requirements are reported in this study.

### Publication Abstract
None

### Keywords
Post-translational modification, Phosphorylation, Yeast, Histone methylation, Kinase, Epigenetics

### Affiliations
School of Biotechnology and Biomolecular Sciences, University of New South Wales, Sydney, New South Wales 2052, Australia
University of New South Wales

### Submitter
Ryan Separovich

### Lab Head
Dr Marc Ronald Wilkins
School of Biotechnology and Biomolecular Sciences, University of New South Wales, Sydney, New South Wales 2052, Australia


