### Accession
PXD012097

### Title
LC-MS/MS analysis of human cellular tRNA

### Description
The field of epitranscriptomics is growing in importance, with chemical modification of RNA being associated with a wide variety of biological phenomena. Mass spectrometry (MS) enables the identification of modified RNA residues within their sequence contexts, by using analogous approaches to shotgun proteomics. We have developed a free and open-source database search engine for RNA MS data, called NucleicAcidSearchEngine (NASE), as part of the OpenMS software framework. NASE allows the reliable identification of (modified) RNA sequences from LC-MS/MS data in a high-throughput fashion. For this validation dataset, we generated a sample of human total tRNA from a cellular extract - a complex mixture of highly modified RNAs. This sample was RNase-treated prior to nanoflow LC-MS/MS analysis.

### Sample Protocol
Crude human cellular tRNA was isolated from HAP1 tissue culture using an RNeasy kit (Qiagen) as according to the manufacturer’s instructions. Briefly, RNAs can be fractionated by length by differential elution, with RNAs less than 200 nucleotides mostly made up of tRNA, and the larger fraction being mostly rRNA. The “shorter” RNA fraction was digested with RNase T1, and the resultant oligonucleotides were characterised by nanoflow LC-MS/MS as follows: The RNA sample was separated by reversed-phase ion-pair liquid chromatography (using 200 mM HFIP + 8.5 mM TEA in H2O as eluent A, and 100 mM HFIP + 4.25 mM TEA in methanol as eluent B) and characterised by negative ion MS/MS in a hybrid quadrupole-orbitrap mass spectrometer (Q Exactive HF, Thermo Fisher). A gradient of 2.5 to 25% eluent B eluted oligonucleotides over 50 min from nanoflow Acclaim PepMap C18 solid phase (Thermo Fisher) at 200 nL/min. Precursor ion spectra were collected at a scan range of 600 to 3500 m/z at 120k resolution in data-dependent mode, with the top five MS1 species selected for fragmentation (NCE: 20) and MS2 at 60k resolution.

### Data Protocol
The proprietary raw file was converted to mzML format without compression and with vendor peak-picking using MSConvert (https://github.com/ProteoWizard). In the analysis with NASE, the full list of fragment ion types (a-B, a, b, c, d, w, x, y, z) was considered for peak matching. Precursor and fragment mass tolerance were both set to 3 ppm. For precursor mass correction, the monoisotopic up to the fifth (+4 neutrons) isotopologue peak were considered. 26 variable modifications (based on previous findings in yeast and human tRNA) were specified, at a maximum of three modifications per oligonucleotide: m1A, m6Am, t6A, i6A, io6A, m5C, m5Cm, f5C, ac4C, m2G, m2,2G, m5U, m5Um, acp3U, ncm5s2U, mnm5U, D, m5D, I, m1I, m1Im, Ar(p), yW, o2yW, Q, galQ. Na+ was specified as a potential adduct. The FASTA file contained 420 human tRNA sequences collected from the tRNA sequence database tRNAdb (http://trna.bioinf.uni-leipzig.de) plus the same number of reversed decoy sequences. The digestion parameters were set to RNase T1 with up to two missed cleavages. NASE produced two output files containing the search results: An mzTab-like text file suitable for further analysis, and an idXML file suitable for visualisation in OpenMS’ TOPPView.

### Publication Abstract
None

### Keywords
Epitranscriptomics, Rna, Trna

### Affiliations
STORM Therapeutics Limited
STORM Therapeutics Limited Cambridge United Kingdom

### Submitter
Hendrik Weisser

### Lab Head
Dr Byron Andrews
STORM Therapeutics Limited Cambridge United Kingdom


