### Accession
PXD023140

### Title
Tissue adaptation is the dominant driver of the proteomic landscape of intestinal intraepithelial lymphocytes

### Description
Intestinal intraepithelial T lymphocytes (T-IEL) patrol the single layer of epithelial cells lining the gut, and consist of both induced T-IEL, derived from systemic antigen-experienced lymphocytes, and natural IEL, that are developmentally targeted to the intestine. To gain functional insights into these enigmatic cells, we used high-resolution quantitative mass spectrometry to investigate the proteomic landscape of the main T-IEL populations in the gut. Comparing the proteomes of induced T-IEL, tissue-resident memory TCRαβ+ CD8αβ+ cells and natural TCRγδ+ CD8αα+ and TCRαβ+ CD8αα+ T-IEL, with naive CD8+ T cells from lymph nodes reveals striking similarities between T-IEL subsets and the dominant effect of the gut environment on T-IEL phenotypes. Analysis of copy numbers/cell of >7000 proteins provides new understanding of the differences in composition of T cell antigen receptor signal transduction pathways in T-IEL versus conventional T cells and reveals skewing of the metabolic machinery towards an exhausted T cell phenotype adapted to the intestinal environment. This study provides a resource for exploring and understanding how multiple inputs are integrated into T-IEL function.

### Sample Protocol
Peptide fractionation   The TMT samples were fractionated using off-line high-pH reverse-phase chromatography: samples were loaded onto a 4.6 mm × 250 mm XbridgeTM BEH130 C18 column with 3.5 μm particles (Waters). Using a Dionex BioRS system, the samples were separated using a 25-min multistep gradient of solvents A (10 mM formate at pH 9 in 2% acetonitrile) and B (10 mM ammonium formate at pH 9 in 80% acetonitrile), at a flow rate of 1 ml min−1. Peptides were separated into 48 fractions, which were consolidated into 24 fractions. The fractions were subsequently dried, and the peptides were dissolved in 5% formic acid and analyzed by liquid chromatography–mass spectrometry.   Liquid chromatography electrospray–tandem mass spectrometry analysis   For each fraction, 1 μg was analysed using an Orbitrap Fusion Tribrid mass spectrometer (Thermo Fisher Scientific) equipped with a Dionex ultra-high-pressure liquid chromatography system (RSLCnano). Reversed-phase liquid chromatography was performed using a Dionex RSLCnano high-performance liquid chromatography system (Thermo Fisher Scientific). Peptides were injected onto a 75 μm × 2 cm PepMap-C18 pre-column and resolved on a 75 μm × 50 cm RP C18 EASY-Spray temperature-controlled integrated column-emitter (Thermo Fisher Scientific) using a 4-h multistep gradient from 5% B to 35% B with a constant flow of 200 nl min−1. The mobile phases were: 2% acetonitrile incorporating 0.1% formic acid (solvent A) and 80% acetonitrile incorporating 0.1% formic acid (solvent B). The spray was initiated by applying 2.5 kV to the EASY-Spray emitter, and the data were acquired under the control of Xcalibur software in a data-dependent mode using the top speed and 4 s duration per cycle. The survey scan was acquired in the Orbitrap covering the m/z range from 400–1,400 Thomson units (Th), with a mass resolution of 120,000 and an automatic gain control (AGC) target of 2.0 × 105 ions. The most intense ions were selected for fragmentation using collision-induced dissociation in the ion trap with 30% collision-induced dissociation energy and an isolation window of 1.6 Th. The AGC target was set to 1.0 × 104, with a maximum injection time of 70 ms and a dynamic exclusion of 80 s. During the MS3 analysis for more accurate TMT quantifications, ten fragment ions were co-isolated using synchronous precursor selection, a window of 2 Th and further fragmented using a higher-energy collisional dissociation energy of 55%. The fragments were then analyzed in the Orbitrap with a resolution of 60,000. The AGC target was set to 1.0 × 105 and the maximum injection time was set to 300 ms.

### Data Protocol
MaxQuant processing   The raw proteomics data were analysed with Maxquant (Cox and Mann, 2008; Tyanova et al., 2016) v. 1.6.3.3 and searched against a hybrid database. The database contained all murine SwissProt entries, along with TrEMBL entries with a human paralog annotated within human SwissProt and with protein level evidence. The data was searched with the following modifications: carbamidomethylation of cysteine, as well as TMT modification on peptide amino termini and lysine side chains as fixed modifications; methionine oxidation and acetylation of amino termini of proteins were variable modifications. The false discovery rate was set to 1% for positive identification at the protein and PSM level.

### Publication Abstract
Tissue-resident intestinal intraepithelial T lymphocytes (T-IEL) patrol the gut and have important roles in regulating intestinal homeostasis. T-IEL include both induced T-IEL, derived from systemic antigen-experienced lymphocytes, and natural T-IEL, which are developmentally targeted to the intestine. While the processes driving T-IEL development have been elucidated, the precise roles of the different subsets and the processes driving activation and regulation of these cells remain unclear. To gain functional insights into these enigmatic cells, we used high-resolution, quantitative mass spectrometry to compare the proteomes of induced T-IEL and natural T-IEL subsets, with naive CD8<sup>+</sup> T cells from lymph nodes. This data exposes the dominant effect of the gut environment over ontogeny on T-IEL phenotypes. Analyses of protein copy numbers of &gt;7000 proteins in T-IEL reveal skewing of the cell surface repertoire towards epithelial interactions and checkpoint receptors; strong suppression of the metabolic machinery indicating a high energy barrier to functional activation; upregulated cholesterol and lipid metabolic pathways, leading to high cholesterol levels in T-IEL; suppression of T cell antigen receptor signalling and expression of the transcription factor TOX, reminiscent of chronically activated T cells. These novel findings illustrate how T-IEL integrate multiple tissue-specific signals to maintain their homeostasis and potentially function.

### Keywords
Iel, Lymphocytes, Tmt, T cells, Lymph nodes

### Affiliations
Centre for Gene Regulation and Expression, School of Life Sciences, University of Dundee
University of Dundee

### Submitter
Alejandro Brenes

### Lab Head
Dr Angus I. Lamond
Centre for Gene Regulation and Expression, School of Life Sciences, University of Dundee


