### Accession
PXD019774

### Title
Proteogenomic analysis unveils the expression of cancer associated and specific antigen-derived HLA Class ǀ immunopeptides in melanoma and EGFR mutant lung adenocarcinoma

### Description
Immunotherapy has shown great therapeutic potential for cancers with high tumor mutational burden (TMB), but much less promise for cancers with low TMB. One primary approach for adoptive lymphocyte transfer-based immunotherapy is to target the somatic mutated peptide neoantigens and cancer testis (CT) antigens recognized by cytotoxic T cells. Here, we employed mass spectrometry (MS)-based proteogenomic large-scale profiling to identify potential immunogenic human leukocyte antigen (HLA) Class ǀ- associated peptides in both melanoma, a “hot tumor”, and EGFR mutant lung adenocarcinoma, a “cold tumor”. We uncovered 19 common driver oncogene-derived peptides and more than 1000 post-translationally modified peptides (PTM) representing 58 different PTMs. We constructed a CT antigen database with 286 antigens by compiling reputed CT antigen resources and “in-house” genomic data and used this to identify 45 CT antigen-derived peptides from the identified HLA peptidome. Using integrated next generation sequencing data, we discovered 12 neopeptides in EGFR mutant lung cancer cell lines. Finally, we report a novel approach for non-canonical peptide discovery, whereby we leveraged a deep learning-based de novo search and a high confidence annotated long noncoding RNA (LncRNA) database to identify 44 lncRNA-derived peptides. Findings of this study, for the first time, provide evidence for a large pool of actionable cancer antigen-derived peptides for use in mutant EGFR lung cancer immunotherapy.

### Sample Protocol
For melanoma and lung adenocarcinoma cell line HLA peptide enrichment, 2.0 × 108 cells were harvested in 4ml ice-cold lysis buffer (20mM Tris-HCl pH=8.5, 100mM NaCl, 1 mM EDTA. 1% triton X-100 supplemented with Halt 1:100 protease Inhibitor cocktail Cat. No 78430, Thermo Scientific), after washing three times with ice cold PBS. After 30min on ice, lysates were subjected to needle sonication for 30s. Approximately 30mg snap-frozen lung tumor tissue was homogenized in 4ml ico-cold lysis buffer for 30s at 4°C using the Qiagen TissueLyser II. Cell/tissue lysates were centrifuged for 2 hours at 4°C at 20,000g, and the supernatant used in subsequent experiments. HLA-peptides complexes were isolated by interacting with 0.5mg HLA Class ǀ pan antibody clone W6/32 (BioXcell, West Lebanon, NH) pre-coupled to 200μl slurry of protein A/G PLUS agarose resin (Santa Cruz Biotechnology) overnight at 4°C with constant rotation. Agarose beads were then washed three times with ice-cold lysis buffer (without triton and protease inhibitors), followed by two washes in ice-cold 20mM Tris-HCl (pH=8.5), then one wash in ice-cold HPLC grade water. Complexes were eluted four times with 0.15% trifluoroacetic acid (TFA) in water at room temperature and combined. To purify immunopeptides, HLA-peptide complexes were loaded on preconditioned 50mg C18 desalting columns (Sigma Millipore), then followed three 0.1% TFA in water washes. HLA peptides were then eluted with 40% acetonitrile (ACN) in 0.1% TFA. Purified peptides were lyophilized at -80°C for 2hours, then, reconstituted in 0.1%TFA, 2%ACN loading buffer for MS analysis.

### Data Protocol
Patient and cell line specific protein sequence databases were first generated. The aligned whole exome sequencing BAM files from patient blood (germline) or tumor tissue/cell lines were used to retrieve the variants call format (VCF) using HaplotypeCaller [63], and the intermediate VCF files were further annotated by SnpEff [64]which filtered out only nonsynonymous on exome regions including SNVs and Indels. Similarly, the RNA-sequencing BAM files were used for the fusion variant calling by Star-Fusion [65]. The final VCF files were in silico translated to sample specific protein sequence libraries (FASTA files) using QUILTS [66], which were merged with refseq hg38 converted human proteome database.  The database search of MS raw files was carried out by PEKAS studio [39] (Bioinformatics Solutions) using the patient/cell line specific databases described above. In the PEAKS searching engine, no enzyme digestion was selected because HLA peptides are natural peptides without artificial digestion. Importantly, the unique PEAKS built-in functions, pan-PTMs including 650 different variable modifications and de novo searching were used. The precursor mass tolerance was set to 15ppm and fragment ion tolerance was set to 0.5Da. For the mutant neoepitope screening, the false discovery rate (FDR), estimated by decoy-fusion database, was chosen at 0.05. For the wt peptides screening (e.g., CT antigen, LncRNA and PTMs), FDR was set to 0.01. For the de novo search, we applied very stringent criteria, a) the average local confidence (ALC) score (substituted for FDR estimation) of each peptides must be >50%; b) the lowest %rank, predicted by NetMHCpan4.0, of each peptides against their corresponding HLA molecules in the same sample must be <2.0. The detected raw peptide intensity was log2 transformed for further statistical analysis.

### Publication Abstract
Immune checkpoint inhibitors and adoptive lymphocyte transfer-based therapies have shown great therapeutic potential in cancers with high tumor mutational burden (TMB), such as melanoma, but not in cancers with low TMB, such as mutant epidermal growth factor receptor (EGFR)-driven lung adenocarcinoma. Precision immunotherapy is an unmet need for most cancers, particularly for cancers that respond inadequately to immune checkpoint inhibitors. Here, we employed large-scale MS-based proteogenomic profiling to identify potential immunogenic human leukocyte antigen (HLA) class I-presented peptides in melanoma and EGFR-mutant lung adenocarcinoma. Similar numbers of peptides were identified from both tumor types. Cell line and patient-specific databases (DBs) were constructed using variants identified from whole-exome sequencing. A de novo search algorithm was used to interrogate the HLA class I immunopeptidome MS data. We identified 12 variant peptides and several classes of tumor-associated antigen-derived peptides. We constructed a cancer germ line (CG) antigen DB with 285 antigens. This allowed us to identify 40 class I-presented CG antigen-derived peptides. The class I immunopeptidome comprised more than 1000 post-translationally modified (PTM) peptides representing 58 different PTMs, underscoring the critical role PTMs may play in HLA binding. Finally, leveraging de novo search algorithm and an annotated long noncoding RNA (lncRNA) DB, we developed a novel lncRNA-encoded peptide discovery pipeline to identify 44 lncRNA-derived peptides that are presented by class I. We validated tandem MS spectra of select variant, CG antigen, and lncRNA-derived peptides using synthetic peptides and performed HLA class I-binding assays to demonstrate binding to class I proteins. In summary, we provide direct evidence of HLA class I presentation of a large number of variant and tumor-associated peptides in both low and high TMB cancer. These results can potentially be useful for precision immunotherapies, such as vaccine or adoptive cell therapies in melanoma and EGFR-mutant lung cancers.

### Keywords
Immunotherapy, Lung cancer, Melanoma, Hla immunopeptidome, Neoantigen

### Affiliations
National Cancer Institute/NIH
Thoracic and GI Malignancies Branch, Center for Cancer Research, NCI, NIH, Bethesda, MD 20892

### Submitter
Yue Andy Qi

### Lab Head
Dr Udayan Guha
Thoracic and GI Malignancies Branch, Center for Cancer Research, NCI, NIH, Bethesda, MD 20892


