### Accession
PXD011489

### Title
Quantitative Phosphoproteomic and System-Level Analysis of TOR Inhibition Unravel Distinct Organellar Acclimation in Chlamydomonas reinhardtii

### Description
Rapamycin is an inhibitor of the evolutionary conserved Target of Rapamycin (TOR) kinase which promotes and coordinates translation with cell growth and division. In heterotrophic organisms, TOR regulation is based on intra- and extracellular stimuli such as amino acids level and insulin perception. However, how plant TOR pathways have evolved to integrate plastid endosymbiosis is a remaining question. Despite the close association of the TOR signaling with the coordination between protein turn-over and growth, proteome and phosphoproteome acclimation to a rapamycin treatment have not yet been thoroughly investigated in Chlamydomonas reinhardtii. In this study, we have used in vivo label-free phospho-proteomic analysis to profile both protein and phosphorylation changes at 0, 24, and 48 h in Chlamydomonas cells treated with rapamycin. Using multivariate statistics we highlight the impact of TOR inhibition on both the proteome and the phosphoproteome. Two-way ANOVA distinguished differential levels of proteins and phosphoproteins in response either to culture duration and rapamycin treatment or combined effects. Finally, proteinâ€“ protein interaction networks and functional enrichment analysis underlined the relation between plastid and mitochondrial metabolism. Prominent changes of proteins involved in sulfur, cysteine, and methionine as well as nucleotide metabolism on the one hand, and changes in the TCA cycle on the other highlight the interplay of chloroplast and mitochondria metabolism. Furthermore, TOR inhibition revealed changes in the endomembrane trafficking system. Phosphoproteomics data, on the other hand, highlighted specific differentially regulated phosphorylation sites for calcium-regulated protein kinases as well as ATG7, S6K, and PP2C. To conclude we provide a first combined Chlamydomonas proteomics and phosphoproteomics dataset in response to TOR inhibition, which will support further investigations.

### Sample Protocol
Protein Extraction and Phosphopeptides Enrichments Total proteins from Chlamydomonas cell pellets were extracted a phenol-phase extraction protocol as previously described (Roustan et al., 2017). Protein precipitation was performed by mixing the phenol fraction with 2.5 volumes of 0.1M ammonium acetate in methanol. After a 16 h incubation period at ô€€€20C, the samples were centrifuged for 5 min at 5000  g. The protein pellets were washed twice with 0.1M ammonium acetate, one time with acetone and air dried at room temperature. The protein pellets were dissolved in 8M urea/100 mM ammonium bicarbonate (AmBic) supplemented with protease and phosphatase inhibitor cocktails as indicated by the supplier (Roche, Cat. No. 05 892 791 001 and Cat. No. 04 906 837 001). Protein concentration was determined using the Bio-Rad Bradford Assay with BSA as a standard. 200 mg (proteomic)/500 mg (phosphoproteomic) of total protein per sample was first reduced with dithiothreitol (DTT) at a concentration of 5 mM at 37C for 45 min. Cysteine residues were alkylated with 10 mM iodoacetamide (IAA) in darkness at room temperature (RT) for 60 min. Alkylation was stopped by increasing DTT concentration to 10 mM and incubating the samples in the dark at RT for 15 min. Then the urea concentration was diluted to 2 M with 50 mM AmBic/10 % acetonitrile (ACN). CaCl2 was added to a final concentration of 2 mM. Trypsin digestion (Poroszyme immobilized trypsin; 5:100 v:w) was performed at 37C overnight. Protein digests were desalted with C18 solid phase extraction (SPE) (Agilent Technologies, Santa Clara, CA, United States) and carbon graphite SPE as described by Furuhashi et al. (2014) and both fractions were subsequently pooled and dried before LC-MS measurement in the case of proteomics or before phosphopeptides enrichment for phosphoproteomics approach. 5 mg of TiO2 (Glygen Corp.) was used to enrich phosphopeptide was used as described previously (Bodenmiller et al., 2007; Chen et al., 2010) and dried in a vacuum concentrator. LC-MS for Proteomics and Phosphoproteomics For Proteomics Samples were dissolved in 2% ACN and 0.1% FA to a final concentration of 0.2 mg equivalent total protein per mL. 1 mg equivalent total protein was loaded into a one-dimensional (1D) nanoflow LC-MS/MS system equipped with a precolumn (Eksigent, Germany). Peptides were eluted using an Ascentis column (Ascentis Express, peptide ES-C18 HPLC column (SUPELCO Analytical, Bellefonte, PA, United States), dimension 15 cm  100 mm, pore size 2.7 mm) during a 120 min gradient from 5% to 50% (v/v) acetonitrile, 0.1% (v/v) formic acid. MS analysis was performed on an Orbitrap LTQ XL mass spectrometer (Thermo, Germany) with a controlled flow rate of 500 nL per minute. Specific tune settings for the MS were as follows: spray voltage was set to 1.8 kV; temperature of the heated transfer capillary was set to 180C, full scan range 350â€“1800 m/z, FTMS resolution 120000. Each full MS scan was followed by ten MS/MS scans, in which the ten most abundant peptide molecular ions were dynamically selected, with a dynamic exclusion window set to 60 s. Ions with a C 1 or unidentified charge state in the full MS were omitted from MS/MS analysis. For injection control automatic gain control (AGC) for full scan acquisition in the Orbitrap was set to 5  105 ion population, the maximum injection time (max IT) was set to 500 ms. Orbitrap online calibration using internal lock mass calibration on m/z 371.10123 from polydimethylcyclosiloxane was used. Multistage activation was enabled with neural losses of 24.49, 32.66, 48.999, 97.97, 195.94, and 293.91 Da for the 10 most intense precursor ions. Prediction of ion injection time was enabled and the trap was set to gather 3  104 ions for up to 50 ms. For Phosphoproteomics Samples were prepared and analyzed as described in Roustan et al., 2017

### Data Protocol
Data Analysis and Statistics MaxQuant 1.52 and the Andromeda search algorithm were used against the Chlamydomonas UniProt and JGI_236 version databases to perform peptide identification, phosphorylation site mapping and protein and phosphopeptide quantification (Cox and Mann, 2008; Cox et al., 2011). Two missed cleavages were allowed. Methionine oxidation and protein N-terminal acetylation were endorsed as dynamic modifications. Mass tolerance was set to 5 p.p.m. for parental ions and 0.8 Da for the MS/MS fragment. For both peptide and protein levels, false discovery rate was set to 1%. Label-free quantification of protein was done with a peptide ratio count  2. Phosphoproteome analysis was performed with similar settings to those described for the proteomics analysis. Additionally, phosphorylation of serine, threonine and tyrosine residues was permitted to occur as dynamic modifications. Further, three missed cleavages were enabled. Quantification was done at the peptide level. Perseus 1.5 software, was used for further filtering and data processing (Tyanova et al., 2016). Proteomics data set was obtained by filtering the data matrix so that proteins that were identified in at least four biological replicates in at least one class of samples were included. Before PCA, values were log2 transformed and missing values were replaced by random numbers drawn from the normal distribution that represents low-abundance measurements of each sample. Phosphopeptide data were filtered as the total proteomics data on the base of three biological samples per sample class. Additionally, only phosphopeptides that passed the class I criteria (phosphosite probability > 75% and score difference > 5) were included in the final dataset (Olsen et al., 2006). Moreover, phosphopeptide abundance was normalized to the median of each sample, log2 transformed and missing values replaced using the same method as in total proteomics data. Further, PCA, HCA based on Spearman rank correlation and an average linkage of z-transformed phosphopeptide abundances, Analysis of variance (two-way ANOVA) was performed with Perseus software (Tyanova et al., 2016). Algal Functional Annotation tool (AFAT) was used to identify enriched functions from the protein cluster resulting from the HCA analysis or from the significantly changed proteins (Lopez et al., 2011). The motif-x analysis was used to identify phosphorylation motifs that were over-represented in our dataset3 (Schwartz and Gygi, 2005). The analysis was conducted on peptides which significantly changed in treatment factor from the two-way ANOVA analysis. Two searches were performed with a serine residue or a threonine residue as central position. Background proteome was Chlamydomonas UniProt database centered on S, T and Y amino acids. Protein-protein interaction networks were created using the STRING database for Known and Predicted Protein-Protein Interactions with the standard setting4 (von Mering et al., 2005).

### Publication Abstract
Rapamycin is an inhibitor of the evolutionary conserved Target of Rapamycin (TOR) kinase which promotes and coordinates translation with cell growth and division. In heterotrophic organisms, TOR regulation is based on intra- and extracellular stimuli such as amino acids level and insulin perception. However, how plant TOR pathways have evolved to integrate plastid endosymbiosis is a remaining question. Despite the close association of the TOR signaling with the coordination between protein turn-over and growth, proteome and phosphoproteome acclimation to a rapamycin treatment have not yet been thoroughly investigated in <i>Chlamydomonas reinhardtii</i>. In this study, we have used <i>in vivo</i> label-free phospho-proteomic analysis to profile both protein and phosphorylation changes at 0, 24, and 48 h in <i>Chlamydomonas</i> cells treated with rapamycin. Using multivariate statistics we highlight the impact of TOR inhibition on both the proteome and the phosphoproteome. Two-way ANOVA distinguished differential levels of proteins and phosphoproteins in response either to culture duration and rapamycin treatment or combined effects. Finally, protein-protein interaction networks and functional enrichment analysis underlined the relation between plastid and mitochondrial metabolism. Prominent changes of proteins involved in sulfur, cysteine, and methionine as well as nucleotide metabolism on the one hand, and changes in the TCA cycle on the other highlight the interplay of chloroplast and mitochondria metabolism. Furthermore, TOR inhibition revealed changes in the endomembrane trafficking system. Phosphoproteomics data, on the other hand, highlighted specific differentially regulated phosphorylation sites for calcium-regulated protein kinases as well as ATG7, S6K, and PP2C. To conclude we provide a first combined <i>Chlamydomonas</i> proteomics and phosphoproteomics dataset in response to TOR inhibition, which will support further investigations.

### Keywords
Biofuelsx, Phosphoproteomics, Tor, Proteomics, Energy signaling, Rapamycin, Plant systems biology

### Affiliations
University of Vienna
Department of Ecogenomics and Systems Biology, University of Vienna, Vienna, Austria.

### Submitter
Valentin Roustan

### Lab Head
Dr Prof. Dr. Wolfram Weckwerth
Department of Ecogenomics and Systems Biology, University of Vienna, Vienna, Austria.


