### Accession
PXD030559

### Title
Swine Acute Diarrhea Syndrome Coronavirus Induces Autophagy to Promote Its Replication via the Akt/mTOR Pathway

### Description
Swine acute diarrhea syndrome coronavirus (SADS-CoV) is an enveloped, single‐stranded, positive‐sense RNA viruse belonging to Coronaviridae family. An increasing number of studies have demonstrated that viruses could utilize autophagy to promote their own replication. However, the relationship between SADS-CoV and autophagy machinery remained unknown. Here, we reported that SADS-CoV infection induced autophagy in cultured cells and pharmacologically increased autophagy was conducive to viral proliferation. Conversely, suppression of autophagy by pharmacological inhibitor or knockdown of autophagy-related protein impeded viral replication. Furthermore, we demonstrated the underlying mechanisms by which SADS-CoV triggered autophagy through inactivation of the Akt/mTOR pathway. Importantly, we identified integrin α3 (ITGA3) as a potential antiviral target upstream of Akt/mTOR and autophagy pathways by analyzing the proteomic profiles. Knockdown of ITGA3 using RNA inference activated autophagy and as a consequence, increased the replication of SADS-CoV. Collectively, our studies revealed a novel mechanism that SADS-CoV induced autophagy to facilitate its proliferation via Akt/mTOR pathway and found that ITGA3 is an effective antiviral factor for suppressing viral infection.

### Sample Protocol
1 Protein extraction (1) Weigh appropriate amount of sample into a 1.5mL centrifuge tube; (2) Add a 5mm steel bead and appropriate amount of Lysis Buffer 3, add PMSF with a final concentration of 1mM, EDTA with a final concentration of 2mM, vortex and let stand for 5 minutes, add DTT with a final concentration of 10mM; (3) Oscillate with a tissue grinder for 2 minutes (frequency is 50HZ); (4) 25,000g*4°C centrifugation for 20 minutes, take the supernatant; (5) Add a final concentration of 10mM DTT, water bath at 56°C for 1 hour; (6) After returning to room temperature, a final concentration of 55mM IAM was added and incubated in a dark room for 45 minutes; (7) Add cold acetone by 4 times volume of the sample, and stand at -20°C for 2 hours; (8) 25,000g*4°C centrifugation for 20 minutes, discard the supernatant; (9) Add an appropriate amount of Lysis Buffer 3 to precipitate, followed by ultrasonication to dissolve the precipitated proteins; (10) After centrifugation at 25,000g*4°C for 20 minutes, the supernatant was taken for quantification.  2 Protein extraction quality control 2.1 Bradford quantification Standard proteins (0.2μg/μL BSA) 0, 2, 4, 6, 8, 10, 12, 14, 16, 18μL were sequentially added to the 96-well microtiter plates A1 to A10, followed by the addition of pure water 20, 18, 16, 14, 12, 10, 8, 6, 4,2μL, and then 180μL of Coomassie Brilliant Blue G-250 Quantitative Working Solution was added to each well. The OD595 was measured with a microplate reader, and a linear standard curve was drawn based on the OD595 and protein concentration. The sample protein concentration was calculated from the standard curve and sample OD595. Then 10μg of protein solution was used for SDS-PAGE. 2.2 Protein enzymatic hydrolysis (1) Take 100μg of protein solution per sample and dilute with 50mM NH4HCO3 by 4 times volumes; (2) Add 2.5μg of Trypsin enzyme in the ratio of protein: enzyme = 40:1, and digest for 4 hours at 37°C; (3) Enzymatic peptides were desalted using a Strata X column and vacuumed to dryness.  2.3 High pH RP separation Equal amount of peptides were extracted from all samples to mix, and the mixture was diluted with mobile phase A (5% ACN pH 9.8) and injected. The Shimadzu LC-20AB HPLC system coupled with a Gemini high pH C18 column (5μm, 4.6 x 250mm) was used. The sample was subjected to the column and then eluted at a flow rate of 1mL/min by gradient: 5% mobile phase B (95% ACN, pH 9.8) for 10 minutes, 5% to 35% mobile phase B for 40 minutes, 35% to 95% mobile phase B for 1 minute, flow Phase B lasted 3 minutes and 5% mobile phase B equilibrated for 10 minutes. The elution peak was monitored at a wavelength of 214nm and component was collected every minute. Components were combined into a total of 10 fractions, which were then freeze-dried.  2.4 DDA and DIA analysis by nano-LC-MS/MS The dried peptide samples were reconstituted with mobile phase A (2% ACN, 0.1% FA), centrifuge at 20,000g for 10 minutes, and the supernatant was taken for injection. Separation was carried out by a Thermo UltiMate 3000 UHPLC liquid chromatograph. The sample was first enriched in the trap column and desalted, and then entered a tandem self-packed C18 column (150μm internal diameter, 1.8μm column size, 35cm column length), and separated at a flow rate of 500nL/min by the following effective gradient: 0~5 minutes, 5% mobile phase B (98% ACN, 0.1% FA); 5~120 minutes, mobile phase B linearly increased from 5% to 25%; 120~160 minutes, mobile phase B rose from 25% to 35%; 160~170 minutes, mobile phase B rose from 35% to 80%; 170~175 minutes, 80% mobile phase B; 175~180 minutes, 5% mobile phase B. The nanoliter liquid phase separation end was directly connected to the mass spectrometer as the following settings.  3. For DDA analysis, LC separated peptides were ionized by nanoESI and injected to tandem mass spectrometer Q-Exactive HF X (Thermo Fisher Scientific, San Jose, CA) with DDA (data-dependent acquisiton) detection mode. The main settings were: ion source voltage 1.9kV; MS scan range 350~1,500m/z; MS resolution 120,000, maximal injection time (MIT) 100ms; MS/MS collision type HCD, collision energy NCE 28; MS/MS resolution 30,000, MIT 100ms, dynamic exclusion duration 30 seconds. The start m/z for MS/MS was fixed to 100. Precursor for MS/MS scan satisfied: charge range 2+ to 6+, top 20 precursors with intensity over 5E4. AGC was: MS 3E6, MS/MS 1E5.  4. For DIA analysis, LC separated peptides were ionized by nanoESI and injected to tandem mass spectrometer Q-Exactive HF X (Thermo Fisher Scientific, San Jose, CA) with DIA (data-independent acquisiton) detection mode. The main settings were: ion source voltage 1.9kV; MS scan range 400~1,250m/z; MS resolution 120,000, MIT 50ms; 400~1,250m/z was eqaully divided to 45 continuous windows MS/MS scan. MS/MS collision type HCD, MIT was auto mode. Fragment ions were scanned in Orbitrap, MS/MS resolution 30,000, collision energy was distributed mode: 22.5, 25, 27.5, AGC was 1E6.

### Data Protocol
Experiment type This process is based on the sample data generated from a high-resolution mass spectrometer. DDA data was identified by Andromeda search engine within MaxQuant, and identification results wereused for spectral library construction. For large-scale DIA data, mProphet algorithm was used to complete analytical quality control, thus obtaining a large number of reliable quantitative results. This pipeline also performed GO, COG, Pathway functional annotation analysis and time series analysis. Based on the quantitative results, the differential proteins between comparison groups were found, and finally function enrichment analysis, protein-protein interaction (PPI) and subcellular localization analysis of the differential proteins were performed. The selection of database is an important step in MS based protein identification, and the final identified protein sequences are from the selected database. Currently databases in use is UniProt protein database. UniProt is the most informative and resourceful protein database. It consists of data from three major databases, i.e. Swiss-Prot, TrEMBL and PIR-PSD. It is a data set verified by experts and consists of two parts: UniProtKB/Swiss-Prot (with reviewed, manually annotated entries) and UniProtKB/TrEMBL (with unreviewed, automatically annotated entries). In general, it is recommended to give priority using the subset of UniProtKB/Swiss-Prot for protein identification. When it aims to find novel sequences (such as alternative splicing, new transcripts) or to identify allied species, UniProtKB/TrEMBL datatbase can be considered .  DDA data analysis MaxQuant (http://www.maxquant.org) is a free protein identification and quantification software developed by Max Planck Institutes for high-resolution MS data. This project was executed using this software for identification of DDA data, served as a spectrum library for subsequent DIA analysis. The analysis used raw data as input files, and set corresponding parameters and databases, then performed identification and quantitative analysis. The identified peptides satisfies FDR <=1% will be used to construct the final spectral library. During the identification of this project, the parameters were configurated as follows: Item and Value MaxQuant :1.5.3.30 Enzyme: Trypsin Minimal peptide length :7 PSM-level FDR: 0.01 Protein FDR: 0.01 Fixed modifications: Carbamidomethyl (C) Variable modifications:Oxidation (M); Acetyl (Protein N-term); Gln->pyro-Glu (N-term Q); Deamidated (NQ) Database: monkey_bac_irt.fasta (20660 sequences)  3.3 DIA data analysis The DIA data was analyzed using the iRT peptides for retention time calibration. Then, based on the target-decoy model applicable to SWATH-MS, false positive control was performed with FDR 1%, therefore obtaining significant quantitative results.  3.4 MSstats differential analysis MSstats is an R package from the Bioconductor repository. It can be used for statistical evaluation of significant differences in proteins or peptides from different samples, and is widely used in targeted proteomics MRM, label free quantitation, and SWATH quantitative experiments. The core algorithm is linear mixed effect model. The process preprocessed the data according to the predefined comparison group, and then performed the significance test based on the model.

### Publication Abstract
Swine acute diarrhea syndrome coronavirus (SADS-CoV) is an enveloped, single-stranded, positive-sense RNA virus belonging to the Coronaviridae family. Increasingly studies have demonstrated that viruses could utilize autophagy to promote their own replication. However, the relationship between SADS-CoV and autophagy remains unknown. Here, we reported that SADS-CoV infection-induced autophagy and pharmacologically increased autophagy were conducive to viral proliferation. Conversely, suppression of autophagy by pharmacological inhibitors or knockdown of autophagy-related protein impeded viral replication. Furthermore, we demonstrated the underlying mechanism by which SADS-CoV triggered autophagy through the inactivation of the Akt/mTOR pathway. Importantly, we identified integrin &#x3b1;3 (ITGA3) as a potential antiviral target upstream of Akt/mTOR and autophagy pathways. Knockdown of ITGA3 enhanced autophagy and consequently increased the replication of SADS-CoV. Collectively, our studies revealed a novel mechanism that SADS-CoV-induced autophagy to facilitate its proliferation via Akt/mTOR pathway and found that ITGA3 was an effective antiviral factor for suppressing viral infection.

### Keywords
Sars-cov；autophagy；itga3

### Affiliations
Sun Yat-sen University
School of Life Sience, Sun Yat-sen University

### Submitter
YU XIA

### Lab Head
Dr Yongchang Cao
School of Life Sience, Sun Yat-sen University


