### Accession
PXD044722

### Title
SHIFTR enables the unbiased identification of proteins bound to specific RNA regions in transcriptomes of virus and host

### Description
RNA-protein interactions determine the cellular fate of RNA and are central to regulating gene expression outcomes in health and disease. To date, no method exists to identify the proteins bound to specific regions in endogenous RNAs in an unbiased fashion. Here, we develop SHIFTR (Selective RNase-H-mediated interactome framing for target RNA regions), an efficient and scalable approach to identify proteins bound to selected RNA regions in live cells. Compared to state-of-the-art RNA antisense purification techniques, SHIFTR is superior in accuracy, captures close to zero background interactions and requires orders of magnitude lower input material. We establish SHIFTR workflows for targeting RNA biotypes of different length and abundance, including short and long non-coding RNAs, as well as mRNAs and demonstrate that SHIFTR is compatible with multiplexed RNA interactome release in a single experiment. Using SHIFTR, we comprehensively identify interactions of cis-regulatory elements located at the 5ÃŠÂ¹ and 3ÃŠÂ¹-terminal regions of the authentic SARS-CoV-2 RNA genome in infected cells and accurately recover known and novel interactions linked to the function of these viral RNA elements. SHIFTR enables the systematic mapping of region-resolved RNA interactomes for any RNA in any cell type, which has the potential to revolutionize our understanding of the transcriptomes of pathogens and their hosts.

### Sample Protocol
Protein pellets were reconstituted in1x LDS sample buffer (Thermo Fisher Scientific, B0007) prior to mass spectrometry sample preparation. Disulfide bonds were reduced with 10 mM DTT at 56Ã‚Â°C for 20 min in 50 mM HEPES pH 8.5 and alkylated with 20 mM 2-chloroacetamide at 24Ã‚Â°C for 20 min in 50 mM HEPES pH 8.5. Samples were cleaned up according to the SP3 protocol (47, 48). Sequencing grade Trypsin (Promega) was added in an enzyme to protein ratio of 1:50 and incubated overnight at 37Ã‚Â°C. Peptides were labelled with TMT10-plex or TMT6-plex Isobaric Label Reagent (ThermoFisher) according to the manufacturerÃ¢â‚¬â„¢s instructions. In short, 0.8 mg reagent was dissolved in 42 ÃŽÂ¼l 100% acetonitrile and 8 ÃŽÂ¼l of stock was added and incubated for 1 h room temperature. The reaction was quenched with 5% hydroxylamine for 15 min Samples were combined and desalted on an OASIS HLB ÃŽÂ¼Elution Plate (Waters). Offline high pH reverse phase fractionation was carried out on an Agilent 1200 Infinity high-performance liquid chromatography system, equipped with a Gemini C18 column (3 ÃŽÂ¼m, 110 Ãƒâ€¦, 100 x 1.0 mm, Phenomenex). Thirty-two fractions were collected and pooled into 6 fractions , dried under vacuum centrifugation, reconstituted in 10 ÃŽÂ¼l 1% formic acid, 4% acetonitrile for LC-MS analysis.  An UltiMate 3000 RSLC nano LC system (Dionex) equipped with a trapping cartridge (C18 PepMap 100, 5ÃŽÂ¼m, 300 ÃŽÂ¼m i.d. x 5 mm, 100 Ãƒâ€¦) and an analytical column (nanoEase M/Z HSS T3 column 75 ÃŽÂ¼m x 250 mm C18, 1.8 ÃŽÂ¼m, 100 Ãƒâ€¦, Waters) was coupled directly to an Orbitrap Fusion Lumos Tribrid Mass Spectrometer (Thermo) using the Nanospray Flex ion source in positive ion mode. The peptides are concentrated on the pre-column with a constant flow of 0.05% trifluoroacetic acid in water at 30 ÃŽÂ¼l/min for 6 minutes. The analytical column is operated with a constant flow of 0.3 ÃŽÂ¼l/min and a binary solvent system (solvent A: 0.1% formic acid in water, 3% DMSO; solvent B: 0.1% formic acid in acetonitrile, 3% DMSO) is used for elution of the peptides over a gradient from 2% to 8% in 6 min, from 8% to 26% in 42 min, from 26% to 38% in 4 min, followed by an increase of B from 38 to 80% in 4 min and a re-equilibration back to 2% B for 4 min The peptides were introduced into the Fusion Lumos via a fused silica emitter 360 ÃŽÂ¼m OD x 20 ÃŽÂ¼m ID; 10 ÃŽÂ¼m tip (CoAnn Technologies) and an applied spray voltage of 2.4 kV. The capillary temperature was set at 275Ã‚Â°C. Full mass scan (MS1) was acquired with mass range 375-1500 m/z in profile mode in the orbitrap with resolution of 60000. The filling time was set at maximum of 50 ms. Data dependent acquisition (DDA) was performed with the resolution of the Orbitrap set to 15000, with a fill time of 54 ms and a limitation of 1x105 ions. A normalized collision energy of 36 was applied. Fixed first mass m/z 110. MS2 data was acquired in profile mode.

### Data Protocol
IsobarQuant and Mascot (v2.2.07) were used to process the acquired data, which was searched against a combined Uniprot proteome database of Homo sapiens proteome database (UP000005640) and SARS-CoV-2 (UP000464024) containing common contaminants and reversed sequences. Carbamidomethyl on cystein and TMT6/10 on lysine were set as fixed modifications. Acetyl (Protein N-term), Oxidation (M) and TMT6/10 on N-termini were set as variable modifications. For the full scan (MS1) a mass error tolerance of 10 ppm and for MS/MS (MS2) spectra of 0.02 Da was allowed. As protease Trypsin was chosen with maximum of two missed cleavages. Further parameters were: a minimum peptide length of seven amino acids: false discovery rate on peptide and protein level: 0.01.  The raw output files of IsobarQuant (protein.txt Ã¢â‚¬â€œ files) were processed using the R programming language. Only proteins that were quantified with at least two unique peptides were considered for the analysis. Raw TMT reporter ion signals (signal_sum columns) were first cleaned for batch effects using limma (50) and further normalized using vsn (variance stabilization normalization (51). Proteins were tested for differential expression using the empirical Bayes statistics (eBayes) from the limma package while removing batch effects by incorporating replicate information. For correcting against multiple testing, the local false discovery rate (FDR) was calculated by fdrtool (52). Comparisons showing a local FDR value below 0.05, were considered as statistically significant. As pointed out by Strimmer et al. (52), the local FDR determines an upper bound for tail-based FDR values provided by fdrtool. To facilitate the identification of additional candidates in experiments for MALAT1, GAPDH, and ACTB, the tail-based FDR value (qvalue) was considered for differentiating between statistically significant and non-significant comparisons.

### Publication Abstract
RNA-protein interactions determine the cellular fate of RNA and are central to regulating gene expression outcomes in health and disease. To date, no method exists that is able to identify proteins that interact with specific regions within endogenous RNAs in live cells. Here, we develop SHIFTR (Selective RNase H-mediated interactome framing for target RNA regions), an efficient and scalable approach to identify proteins bound to selected regions within endogenous RNAs using mass spectrometry. Compared to state-of-the-art techniques, SHIFTR is superior in accuracy, captures minimal background interactions and requires orders of magnitude lower input material. We establish SHIFTR workflows for targeting RNA classes of different length and abundance, including short and long non-coding RNAs, as well as mRNAs and demonstrate that SHIFTR is compatible with sequentially mapping interactomes for multiple target RNAs in a single experiment. Using SHIFTR, we comprehensively identify interactions of cis-regulatory elements located at the 5' and 3'-terminal regions of authentic SARS-CoV-2 RNAs in infected cells and accurately recover known and novel interactions linked to the function of these viral RNA elements. SHIFTR enables the systematic mapping of region-resolved RNA interactomes for any RNA in any cell type and has the potential to revolutionize our understanding of transcriptomes and their regulation.

### Keywords
Sars-cov-2, Rna-binding proteins, Rna-protein interactions, Interactomes, Virus-host interactions

### Affiliations
EMBL
Helmholtz Center for Infection Research (HZI), Helmholtz Institute for RNA-based Infection Research (HIRI) Uiniversity of WÃƒÂ¼rzburg

### Submitter
Frank Stein

### Lab Head
Dr Mathias Munschauer
Helmholtz Center for Infection Research (HZI), Helmholtz Institute for RNA-based Infection Research (HIRI) Uiniversity of WÃƒÂ¼rzburg


