### Accession
PXD002331

### Title
Cx43-interaction network in the heart

### Description
The main objective of the present report was to unravel the Cx43-interaction network in the heart, and to establish the impact of heart ischemia and I/R upon these interactions. In order to characterize the cardiac Cx43 interactome under ischemia and I/R, a quantitative proteomic analysis was performed, through the combination of immunopurification of endogenous Cx43 (Cx43 IP) and identification of its binding partners with the SWATH-MS approach, using rat hearts maintained in a Langendorff apparatus.  In the present approach 444 proteins were identified, including proteins identified based on a single peptide (supplementary table I). From these 444 proteins, 299 (approximately 67 % of the entire dataset) were quantified and compared between the various experimental conditions (including non-specific binding to control IP samples). These 299 proteins were further evaluated by a series of complementary analysis to deciphering the truly interactors of Cx43. According with this evaluation, 236 out of the 299 quantified proteins were considered as putative Cx43 interacting partners in the cardiac context presented The results obtained in this study demonstrate that Cx43 mainly interacts with proteins related with metabolism, signaling and trafficking, and that this interactome can be differentially modulated in diseased hearts. Our results shed new light upon the understanding of Cx43 functions in the heart, both in health and disease, which ultimately may lead to the establishment of new therapeutic targets to modulate cardiac homeostasis.

### Sample Protocol
10-week-old Wistar rats were used. Hearts were perfused on a Langendorff apparatus [perfusion pressure of 70 mmHg, constant flow rate of 15 ml/min], with modified Krebs–Henseleit buffer, equilibrated with 95%O2/5%CO2 at 37ºC. Perfusion was stabilized for 10 min, followed by either 20 min-perfusion (control) or subjected to no-flow ischemia. Reperfusion was induced by reestablishment of the initial flow rate for additional 60 min Lysates from whole hearts were prepared by homogeneization in lysis buffer (100 mM NaCl, 50 mM Tris-HCl, 5 mM EDTA, 1% Triton X-100, pH 7.4). 25 mg of the supernatants were used for immunoprecipitation (n=3). Briefly, supernatants were incubated with 30 µg goat polyclonal antibodies directed against Cx43 (AB0016, Sicgen). Goat non-specific polyclonal antibodies (anti-GFP; AB0020, Sicgen) were used for control IP. Incubations proceeded overnight, at 4 ºC, followed by incubation with 120 µg protein G-sepharose (GE Healhcare) for 1.5 h at 4 ºC. Protein G-sepharose sediments were washed in lysis buffer, and immunopurified proteins wereeluted in Laemmli buffer aided by denaturation at 95ºC for 5 min. After denaturation, samples were alkylated with acrylamide and subjected to in gel digestion by using the short-GeLC approach. Sample was loaded into two wells of a “4–20% TGX Stain-Free Gel” (Bio-Rad) and subjected to a partially electrophoretic separation. The entire lanes were sliced into 7 bands of equal size. Each band was sliced in small pieces and were subsequently processed independently. Twenty-five µL of trypsin (0.01 µg/µL solution in 10 mM ammonium bicarbonate) were added to the dried gel bands and left on ice, to rehydrate, after this period 10 µL of 10 mM ammonium bicarbonate were added and in-gel digestion was performed overnight at room temperature in the dark. Peptides were extracted from the gel pieces by sequential addition of three solutions of acetonitrile (ACN) in 1% formic acid (FA) (30%, 50%, and 98% of ACN, respectively). At this stage, peptides extracted from different bands were pooled together into 2 peptides mixtures per sample for posterior analysis. Peptides mixtures were-salted using OMIX tips with C18 stationary phase (Agilent Technologies). Before protein digestion, samples were spiked with recombinant green fluorescent protein (GFP) to monitor samples loss during sample preparation. And the peptides mixtures were ressuspended in mobile phase (2% ACN and 0.1% FA) spiked with iRTs peptides (Biognosys AG) for retention time adjustment. Samples were analyzed on a Triple TOFTM 5600 System (ABSciex®) in two phases: information-dependent acquisition (IDA) followed by SWATH acquisition on the same sample. Peptides were resolved by liquid chromatography (nanoLC Ultra 2D, Eksigent®) on a ChromXPTM C18AR reverse phase column (300 μm ID × 15cm length, 3 μm particles, 120 Å pore size, Eksigent®) at 5μL/min. Peptides were eluted into the mass spectrometer with an acetonitrile gradient in 0.1% FA (2% to 35% ACN, in a linear gradient for 45 min), using an electrospray ionization source (DuoSprayTM Source, ABSciex®). One sixth of the two peptides mixtures of each biological replicate was pooled to form representative samples of each experimental condition. The pooled mixtures were analyzed in IDA mode, to generate peptide fragmentation spectra for further protein identification/library creation. For IDA experiments, the mass spectrometer was set to scanning full spectra (350-1250m/z) for 250ms, followed by up to 30 MS/MS scans (100–1500 m/z). Candidate ions with a charge state between +2 and +5 and counts above a minimum threshold of 70 counts per second were isolated for fragmentation, and one MS/MS spectra was collected for 100 ms before adding those ions to the exclusion list for 15 seconds (Analyst® TF 1.6, ABSciex®). Rolling collision was used with a collision energy spread of 5. To improve sample coverage, an additional IDA experiment was done for each pool using an exclusion list of the peptides previously identified. For the quantitative analysis, the remaining peptides mixtures of each biological replicates were combined into a single sample that was analyzed by a data independent acquisition (DIA). The SWATH setup was essentially as described in Anjo et al, with the same chromatographic conditions used as in the IDA run. For SWATH-MS based experiments, the mass spectrometer was operated in a looped product ion mode. The instrument was specifically tuned to allow a quadrupole resolution of 25 m/z mass selection. A set of 30 overlapping windows was constructed covering the precursor mass range of 350–1100 m/z. A 50 ms survey scan (350-1500 m/z) was acquired at the beginning of each cycle, and SWATH MS/MS spectra were collected from 100–1500 m/z for 100 ms resulting in a cycle time of 3.1 s. The collision energy for each window was determined according to the calculation for a charge +2 ion centered upon the window with a collision energy spread of 15.

### Data Protocol
Protein identification/Library generation: Peptide identification was performed with Protein Pilot software (v4.5, ABSciex®). Search parameters used were the following: search against a database composed by Rattus Norvegicus from SwissProt database (release at February 2014), GFP and iRT peptides sequences, and using acrylamide alkylated cysteines as fixed modification. An independent False Discovery Rate (FDR) analysis using the target-decoy approach provided with Protein Pilot software was used to assess the quality of the identifications and positive identifications were considered when identified proteins and peptides reached a 5% local FD. A specific library of precursor masses and fragment ions was created by combining all files (except from IP control) from the IDA experiments, and used for subsequent SWATH processing. Libraries were obtained using Protein PilotTM software (v4.5, ABSciex®) with the same parameters as described above.  SWATH data processing: Data processing was performed using SWATHTM processing plug-in for PeakViewTM (v2.0.01, ABSciex®), briefly peptides were selected automatically from the library using the following criteria: (i) the unique peptides for a specific targeted protein were ranked by the intensity of the precursor ion from the IDA analysis as estimated by the ProteinPilotTM software, and (ii) Peptides that contained biological modifications and/or were shared between different protein entries/isoforms were excluded from selection. Up to 15 peptides were chosen per protein, and SWATHTM quantitation was attempted for all proteins in library file that were identified below 5% local FDR from ProteinPilotTM searches. In SWATH™ Acquisition data, peptides are confirmed by finding and scoring peak groups, which are a set of fragment ions for the peptide. Target fragment ions, up to 5, were automatically selected and peak groups were scored following the criteria described in Lambert et al (2013). Peak group confidence threshold was determined based on a FDR analysis using the target-decoy approach and 1% extraction FDR threshold was used for all the analyses. Peptide that met the 1% FDR threshold in at least two of the three biological replicates were retained, and the peak areas of the target fragment ions of those peptides were extracted across the experiments using an extracted-ion chromatogram (XIC) window of 3 minutes and a 20 mDa XIC width. The retention time was adjusted to each samples by the use of the iRT peptides. Proteins levels were estimated by summing all the transitions from all the peptides for a given protein and normalized to the internal standard.  Clustering of Profiles and Comparative Analysis: Clustering analysis and complementary heatmaps were done using GPRroX (version 1.1.15) to identify groups of proteins with similar profiles. Clustering was done using the unsupervised clustering fuzzy c-means algorithm implemented in the Mfuzz package  which is a soft clustering algorithm being noise robust and indicating how well protein profiles are represented by respective clusters. This analysis was performed at two distinct levels: 1) it was done as complementary method to identify the Cx43 interactors from the non-specific ones, using the median normalized protein levels of all the proteins quantified in the study; 2) clustering was also done to trace the different profiles of Cx43 interactions under the various experimental conditions, using for this purpose the interaction levels (protein levels normalized for the Cx43 levels) of the previously selected Cx43 interactors.     Statistical analysis was performed in MarkerView™ Software (version 1.2.1.1, ABSciex®). Statistical significance was considered relevant for p-value < 0.1 using a multiple t-test approach for comparison between experimental groups. Data normality was accessed by a combinatory analysis of histograms and Q–Q plots  obtained in in InfernoRDN (version 1.1.5581.33355). Data is presented as median ± median absolute deviation (MAD) of a set of three independent experiments.

### Publication Abstract
The coordinated and synchronized cardiac muscle contraction relies on an efficient gap junction-mediated intercellular communication (GJIC) between cardiomyocytes, which involves the rapid anisotropic impulse propagation through connexin (Cx)-containing channels, namely of Cx43, the most abundant Cx in the heart. Expectedly, disturbing mechanisms that affect channel activity, localization and turnover of Cx43 have been implicated in several cardiomyopathies, such as myocardial ischemia. Besides gap junction-mediated intercellular communication, Cx43 has been associated with channel-independent functions, including modulation of cell adhesion, differentiation, proliferation and gene transcription. It has been suggested that the role played by Cx43 is dictated by the nature of the proteins that interact with Cx43. Therefore, the characterization of the Cx43-interacting network and its dynamics is vital to understand not only the molecular mechanisms underlying pathological malfunction of gap junction-mediated intercellular communication, but also to unveil novel and unanticipated biological functions of Cx43. In the present report, we applied a quantitative SWATH-MS approach to characterize the Cx43 interactome in rat hearts subjected to ischemia and ischemia-reperfusion. Our results demonstrate that, in the heart, Cx43 interacts with proteins related with various biological processes such as metabolism, signaling and trafficking. The interaction of Cx43 with proteins involved in gene transcription strengthens the emerging concept that Cx43 has a role in gene expression regulation. Importantly, our data shows that the interactome of Cx43 (Connexome) is differentially modulated in diseased hearts. Overall, the characterization of Cx43-interacting network may contribute to the establishment of new therapeutic targets to modulate cardiac function in physiological and pathological conditions. Data are available via ProteomeXchange with identifier PXD002331.

### Keywords
Connexin43; interactome; cardiomyocytes; ap-swath

### Affiliations
​Center for Neuroscience and Cell Biology – University of Coimbra
Center for Neuroscience and Cell Biology – University of Coimbra

### Submitter
Sandra Anjo

### Lab Head
Dr Bruno Manadas
Center for Neuroscience and Cell Biology – University of Coimbra


