### Accession
PXD004785

### Title
Proteogenomics reanalysis of human testis data

### Description
Reanalysis of the human testis tissue dataset acquired for the Chromosome-centric  Human Proteome Project (C-HPP; PRIDE project PXD002179), with a focus on  identifying novel peptides for genome annotation.

### Sample Protocol
From PXD002179: Extracted proteins were resolved by a regular SDS-PAGE (10%) and Tricine-SDS-PAGE (12%), respectively. The gel lanes were cut into multiple bands and digested with trypsin before LC-MS/MS analysis. Peptide mixture were separated and analyzed by UPLC (nano AcquityUltra Performance LC, USA)-MS/MS (LTQ OrbitrapVelos, USA) platform. Survey scans were performed in the Orbitrap analyzer at a resolution of 30,000 and target values of 1,000,000 ions over a mass range between 300-1,600 m/z.

### Data Protocol
Raw data from PXD002179 was converted to mzML format using "msconvert" from ProteoWizard (version 3.0.8789), extracting only MS2 scans. The file CHPP_SDS_3003.raw turned out to be corrupt, so the corresponding MGF file was converted to mzML instead.  The mzML files were searched against a comprehensive target/decoy sequence database using Mascot (version 2.5.1) and MS-GF+ (v10089), via MascotAdapterOnline and MSGFPlusAdapter from OpenMS (version 2.0.1). The database file ("Gencode20isoV2_TD_Jul14.fasta") and a description of its contents are available in PRIDE project PXD002967. The following parameters were used for both search engines: 10 ppm precursor mass tolerance, 0.5 Da fragment mass tolerance; trypsin cleavage with full specificity, 2 missed cleavages allowed; fixed modification: carbamidomethylation of cysteine; variable modifications: oxidation of methionine, deamidation of asparagine and glutamine, N-terminal acetylation, conversion to pyroglutamic acid of N-terminal glutamine and glutamic acid. The search results from Mascot and MS-GF+ were post-processed using MascotPercolator (version 2.16) and MS-GF+Percolator (revision "273ff55" from the Percolator GitHub repository), respectively. The PSM-level Percolator results were then converted to mzIdentML format using IDFileConverter from OpenMS (pre-2.1 development version).

### Publication Abstract
Proteogenomics leverages information derived from proteomic data to improve genome annotations. Of particular interest are "novel" peptides that provide direct evidence of protein expression for genomic regions not previously annotated as protein-coding. We present a modular, automated data analysis pipeline aimed at detecting such "novel" peptides in proteomic data sets. This pipeline implements criteria developed by proteomics and genome annotation experts for high-stringency peptide identification and filtering. Our pipeline is based on the OpenMS computational framework; it incorporates multiple database search engines for peptide identification and applies a machine-learning approach (Percolator) to post-process search results. We describe several new and improved software tools that we developed to facilitate proteogenomic analyses that enhance the wealth of tools provided by OpenMS. We demonstrate the application of our pipeline to a human testis tissue data set previously acquired for the Chromosome-Centric Human Proteome Project, which led to the addition of five new gene annotations on the human reference genome.

### Keywords
Human, Testis, Proteogenomics

### Affiliations
Wellcome Trust Sanger Institute
Proteomic Mass Spectrometry, Wellcome Trust Sanger Institute, United Kingdom

### Submitter
Hendrik Weisser

### Lab Head
Dr Jyoti Sharma Choudhary
Proteomic Mass Spectrometry, Wellcome Trust Sanger Institute, United Kingdom


