### Accession
PXD000249

### Title
DNA repair in Mycoplasma gallisepticum

### Description
In the current study, we attempted to characterize the DNA repair system of Mycopalsma gallisepticum. We identified members of different DNA repair pathways based on genomic data and publications. We quantified mRNA of respective proteins per cell in a set of adverse conditions and identified most of them on the protein level. We showed induction of SOS-response in adverse conditions on a transcriptional level in the absence of a known regulator.

### Sample Protocol
In order to assess the presence of repair systems members at the protein level, we used the methods of liquid chromatography-mass spectrometry for the comprehensive identification of all the proteins in the cells of M. gallisepticum. We have identified 561 proteins in total (data not shown) by using the following selection criteria: the number of unique peptides for protein - at least two - and a global FDR of 1% (based on the analysis in PSPEP, with threshold unused score for the protein equaled 0.4). Thus 17,221 unique peptides have been identified (global FDR 1% PSPEP). It should be noted that due to the fact that the algorithm Paragon, used for searching, works on the basis of probability factors, the total number of peptides includes not only non-modified tryptic peptides but semi-tryptic and non-tryptic peptides as well, and peptides containing all possible amino acid modifications considered in the default search algorithm of Paragon. Analysis of the presence of the DNA repair proteins showed the presence of most of them in the mycoplasmaâ€™s proteome. Bioinformatics: Raw data was analyzed with ProteinPilot 4.5 revision 1656 (ABSciex) using search algorithm Paragon 4.5.0.0, revision 1654 (ABSciex), and standard search settings to search against a database of all proteins of *M. gallsiepticum S6* (genbank id: AFFR01000000). The following parameters were used for this search: alkylation of cysteine by iodoacetamide, trypsin digestion, TripleTOF 5600 equipment, and a deep search with additional statistical analysis of results reliability. Specters were grouped with default settings by a ProGroup algorithm build-in to ProteinPilot. The statistical analysis of results reliability (and identification of threshold value of unused score) was carried out by a ProteomicS Performance Evaluation Pipeline Software (PSPEP) algorithm build-in to ProteinPilot.

### Data Protocol
Bioinformatics: Raw data was analyzed with ProteinPilot 4.5 revision 1656 (ABSciex) using search algorithm Paragon 4.5.0.0, revision 1654 (ABSciex), and standard search settings to search against a database of all proteins of *M. gallsiepticum S6* (genbank id: AFFR01000000). The following parameters were used for this search: alkylation of cysteine by iodoacetamide, trypsin digestion, TripleTOF 5600 equipment, and a deep search with additional statistical analysis of results reliability. Specters were grouped with default settings by a ProGroup algorithm build-in to ProteinPilot. The statistical analysis of results reliability (and identification of threshold value of unused score) was carried out by a ProteomicS Performance Evaluation Pipeline Software (PSPEP) algorithm build-in to ProteinPilot.

### Publication Abstract
None

### Keywords
Dna repair, Tof, Mycoplasma

### Affiliations
IPCM
Proteome dept.

### Submitter
Dmitry Alexeev

### Lab Head
Dr Dmitry Ischenko
IPCM


