### Accession
PXD003952

### Title
Proteogenomics insights into the hemi-biotrophic pathogen, Phytophthora capsici

### Description
This study was designed to evaluate the proteome profiles of Phytophthora capsici.  We have utilized a proteogenomics pipeline for identification of proteins from P. capsici.

### Sample Protocol
Phytophthora capsici grown on potato dextrose agar were used for the study. Protein were isolated from mycelial mat and P. capsici infected leaves of black pepper and the oomycete proteins were harvested in planta at 6 and 12 hrs after infection.

### Data Protocol
The tryptic peptides were separated using a nanoACQUITY UPLC chromatographic system were the peptides were separated by reversed-phase chromatography technology. Instrument control and data processing was done with MassLynx4.1 SCN781 software. Mass spectral analysis of eluting peptides from the nanoACQUITY UPLC was carried out on a SYNAPT G2 High Definition MS System (Waters). The acquired ion mobility enhanced MSE spectra was analysed using ProteinLynx Global SERVERâ„¢ v2.5.3 (PLGS, Waters) for protein identification The protein identification was carriedout by searching against a custom Phytophthora capsici LT1534 genome database were the false positive rate was set to 4 %. The parameters for protein identification was made in such a way that a peptide was required to have at least 1 fragment ion match, a protein was required to have at least 3 fragment ion matches and a protein was required to have at least 1 peptide match for identification.

### Publication Abstract
Black pepper (Piper nigrum L.), a tropical spice crop of global acclaim, is susceptible to Phytophthora capsici, an oomycete pathogen which causes the highly destructive foot rot disease. A systematic understanding of this phytopathosystem has not been possible owing to lack of genome or proteome information. In this study, we explain an integrated transcriptome-assisted label-free quantitative proteomics pipeline to study the basal immune components of black pepper when challenged with P. capsici. We report a global identification of 532 novel leaf proteins from black pepper, of which 518 proteins were functionally annotated using BLAST2GO tool. A label-free quantitation of the protein datasets revealed 194 proteins common to diseased and control protein datasets of which 22 proteins showed significant up-regulation and 134 showed significant down-regulation. Ninety-three proteins were identified exclusively on P. capsici infected leaf tissues and 245 were expressed only in mock (control) infected samples. In-depth analysis of our data gives novel insights into the regulatory pathways of black pepper which are compromised during the infection. Differential down-regulation was observed in a number of critical pathways like carbon fixation in photosynthetic organism, cyano-amino acid metabolism, fructose, and mannose metabolism, glutathione metabolism, and phenylpropanoid biosynthesis. The proteomics results were validated with real-time qRT-PCR analysis. We were also able to identify the complete coding sequences for all the proteins of which few selected genes were cloned and sequence characterized for further confirmation. Our study is the first report of a quantitative proteomics dataset in black pepper which provides convincing evidence on the effectiveness of a transcriptome-based label-free proteomics approach for elucidating the host response to biotic stress in a non-model spice crop like P. nigrum, for which genome information is unavailable. Our dataset will serve as a useful resource for future studies in this plant. Data are available via ProteomeXchange with identifier PXD003887.

### Keywords
Lc-mse, Proteinlynx global server, Piper nigrum, Nanoacquity uplc

### Affiliations
Rajiv Gandhi Center for Biotechnology
University of Oxford

### Submitter
Chidambareswaren Mahadevan

### Lab Head
Dr Manjula Sakuntala
Rajiv Gandhi Center for Biotechnology


