### Accession
PXD017004

### Title
Phosphorylation regulates secretion of Mycobacterium tuberculosisvirulence mediators ESAT6 and CFP10

### Description
Virulence effectors secreted by Mycobacterium tuberculosis help subvert host immune mechanisms and therefore are critical for establishment of infection and pathogenesis. However, knowledge in terms of signaling mechanisms thatmodulate the secretion of virulence factors is sparse. Here using high-throughput mass-spectrometric analysis of mycobacterial secretome,phospho-proteome and phospho-secretome combined with empirical validations, we show an unprecedented regulation of mycobacterial secretion via protein phosphorylation. The intricate relationship between phosphorylation and secretion was revealed by system-level analysis of PPI network superimposed with secretome and phospho-proteome profiles. This study establishes phosphorylation as a keydeterminant of secretion of virulence factors and unveils as yet unexplored targets for therapeutic interventions.

### Sample Protocol
Ms and Mtbstrains used in this study were grown in Middlebrook 7H9 media supplemented with 10% ADC at 100 rpm at 37������C. For phosphoproteomics and culture filtrate preparation, the strains were first revived in 7H9 media and then grown in Sauton���������s media in presence of 0.005% Tween80 until the OD600 reached 0.8-1.0. WCLs were prepared from cell pellet using lysis buffer (8 M urea in 25 mM NH4HCO3) supplemented with appropriate protease and phosphatase inhibitors. The protein content was estimated using BCA kit. Trypsinization and Strong Cation Exchange chromatography (SCX) of 7 mg WCL was performed. Each SCX fraction was enriched for phosphorylated peptides using IMAC beads, loaded onto prepared C18 Stage-tips and finally eluted in 50% acetonitrile containing 0.1% formic acid. Fractions were resuspended in 20 ul 5% acetonitrile containing 0.1% formic acid. 3 ul of each fraction was injected into the mass spectrometer (MS) (Thermo LTQ OrbitrapVelos). For culture filtrate preparation, cultures were grown in Sauton���������s media as described above and  the culture supernatant was filtered through 0.2 u syringe filter to remove remaining bacteria. Culture Filtrate Proteins (CFP) were concentrated using 3 kDa cutoff centricon. Protein content was estimated using BCA kit. Samples were stored at -80������C until further analysis.For proteomic analysis of CFP, 50 ������g protein was incubated with urea at a final concentration of 2 M at 37������C for 2 h. Proteins were reduced and alkylated and the urea concentration was reduced to 1 M by adding 25 mM NH4HCO3. Trypsin was added at 1:50 ratio (trypsin:protein) and digestions wereperformed at 37������C for 16 h. Samples were desalted using C18 Stage tips (Nest Group) and dried by speed vac,and reconstituted in 50 ul 5% acetonitrile containing 0.1% formic acid. 1ul sample was injected for MS analysis.  For phosphoproteomic analysis of CFP proteins, 2 mg protein was reduced, alkylated and digested as described above. Peptides were desalted using 100 mg Sep-Pak cartridge (Waters) and eluted in 1.2 ml 80% acetonitrile containing 0.1% trifluoroacetic acid and dried using speed-vac. For enrichment of phosphopeptides, peptides were resuspended in 1 ml 50% acetonitrile containing 0.1% trifluoroacetic acid and were incubated with 20 ul of IMAC beads for 30 min on a nutator. The peptides were then desalted. Dried samples were reconstituted in 20 ul of 5% acetonitrile containing 0.1% formic acid and 3 ul was injected into the MS.   For TMT labelling, 250 ������g of CFP from H37Rvand ���������ce were reduced using tris (2-carboxyethyl) phosphine at a final concentration of 10 mM at 55������C for 1 h and alkylated using 10 mM iodoacetamide for 30 min at room temperature. Six-volumes of pre-chilled acetone were added followed by overnight incubation at -20������C  to precipitate the proteins. Precipitated proteins were resuspended in 100 ul of 100 mM triethylammonium bicarbonate and trypsinzed (trypsin:proteinratio of 1:40) overnight. TMT labelling (Thermo Fisher Scientific) was performed as per manufacturer���������s instructions. Peptides from H37Rv CFP was labelled with TMT-126 and the peptides from ������ce were labelled with TMT-127 reporter ions. Labelled samples were pooled, dried, fractionated using SCX, desalted and resolved by MS.

### Data Protocol
All experiments were performed using EASY-nLC system (Thermo Fisher Scientific) coupled to LTQ Orbitrap-Velos mass spectrometer (Thermo Fisher Scientific) equipped with nanoelectrospray ion source. A 10 cm PicoFrit Self-Pack microcapillary column (New Objective) was used to resolve the peptide mixture except in phosphoproteomics experiment where 15 cm column was used. Peptides were eluted as described previousl [85]. The LTQ Orbitrap-velos was operated using the Top20 CID (High/High)data-dependent acquisition mode with a full scan in the Orbitrap and a MS/MS scan in the CID. The target values for the fullscan MS spectra were set at 0.5 x 106 charges with maximum injection time of 300ms and a resolution of 60,000 at m/z 400. Spectra were queried against Mtb H37Rv database. The precursor and fragment mass tolerances were set at 10 ppm and 0.8 Da, respectively. The protease used to generate peptides i.e. enzyme specificity was set for Trypsin along with maximum missed cleavage value of two. Proteome Discoverer 1.3 was used as the search algorithm with oxidation of methionine and carbamidomethylation of cysteine as static modification and phosphorylation of serine, threonine, and tyrosine was used as dynamic modifications. For labeling experiment, TMT 6-plex modification of peptide N-termini and lysine residues were set as fixed modification. All PSMs were identified at 1% false discovery rate (FDR). For peptide identification a peptide posterior error probability threshold of 0.01 was specified. Default settings were applied for all other parameters.The probability of phosphorylation at each residue was calculated using PhosphoRS node in Proteome Discoverer. Phosphopeptides with ���������80% localization probability and pRSscore���������50 were considered forfurther analysis.The raw MS files mapped the peptide sequence with the Uniprot ID of the respective Mtb proteins. A comprehensive and non-redundant list of all Mtb proteins along with their Uniprot ID and functional classification was generated using Tuberculist database. The Uniprot ID of identified proteins were mapped to their respective gene or Rv number using in house scripts for further analysis.

### Publication Abstract
None

### Keywords
Mycobacterium, Phosphorylation, Secretion, Esat6, Cfp10

### Affiliations
National Institute of Immunology
Signal Transduction Lab 1, National Institute of Immunology, India

### Submitter
basanti malakar

### Lab Head
Dr Vinay Kumar Nandicoori
Signal Transduction Lab 1, National Institute of Immunology, India


