### Accession
PXD018117

### Title
A SARS-CoV-2-Human Protein-Protein Interaction Map Reveals Drug Targets and Potential Drug-Purposing

### Description
An outbreak of the novel coronavirus SARS-CoV-2, the causative agent of COVID-19 respiratory disease, has infected over 170,000 people since the end of 2019, killed over 7,400, and caused worldwide social and economic disruption. SARS-CoV-2 infection has a mortality rate of 3.4% among confirmed cases, and there are currently no effective antiviral molecules or vaccines for its treatment or prevention. The search for effective antiviral treatments has recently highlighted host-directed strategies, however besides data describing viral interactions with cell surface receptors and activating proteases, the scientific community has little knowledge of the molecular details of SARS-CoV-2 infection. To shed light on the mechanisms used by SARS-CoV-2 to infect human cells, we have utilized affinity-purification mass spectrometry to globally profile physical host protein interactions for 26 viral proteins encoded in the SARS-CoV-2 genome, identifying 332 high confidence interactions. Among the human proteins, we identify many druggable human proteins targeted by existing FDA approved drugs that we are currently evaluating for efficacy in live SARS-CoV-2 infection assays. The identification of host-dependency factors mediating virus infection may provide key insights into effective molecular targets for developing broadly acting antiviral targets against SARS-CoV-2 and other deadly coronavirus strains.

### Sample Protocol
Cell culture.  HEK293T cells were cultured in Dulbecco’s Modified Eagle’s Medium (Corning) supplemented with 10% Fetal Bovine Serum (Gibco, Life Technologies) and 1% Penicillin-Streptomycin (Corning) and maintained at 37°C in a humidified atmosphere of 5% CO2.  Transfection.  For each affinity purification, ten million HEK293T cells were plated per 15-cm dish and transfected with up to 15 μg of individual Strep-tagged expression constructs after 20-24 hours.  Total plasmid was normalized to 15 μg and complexed with PolyJet Transfection Reagent (SignaGen Laboratories) at a 1:3 mg:ml ratio of plasmid to transfection reagent based on manufacturer’s recommendations.  After more than 38 hours, cells were dissociated at room temperature using 10 ml Dulbecco’s Phosphate Buffered Saline without calcium and magnesium (D-PBS) supplemented with 10 mM EDTA for at least 5 minutes and subsequently washed with 10 ml D-PBS.  After each step, cells were collected by centrifugation at 200 xg, 4°C for 5 minutes.  Cell pellets were frozen on dry ice and stored at - 80°C.  At least three biological replicates were independently prepared for purification. Affinity purification.  Frozen cell pellets were thawed on ice for 15-20 minutes and suspended in 1 ml Lysis Buffer [IP Buffer (50 mM Tris-HCl, pH 7.4 at 4°C, 150 mM NaCl, 1 mM EDTA) supplemented with 0.5% Nonidet P 40 Substitute (NP40; Fluka Analytical) and cOmplete mini EDTA-free protease and PhosSTOP phosphatase inhibitor cocktails (Roche)].  Samples were then frozen on dry ice for 10-20 minutes and partially thawed at 37°C before incubation on a tube rotator for 30 minutes at 4°C and centrifugation at 13,000 xg, 4°C for 15 minutes to pellet debris.  After reserving 50 μl lysate, up to 48 samples were arrayed into a 96-well Deepwell plate for affinity purification on the KingFisher Flex instrument (Thermo Scientific) as follows: MagStrep “type3” beads (30 μl; IBA Lifesciences) were equilibrated twice with 1 ml Wash Buffer (IP Buffer supplemented with 0.05% NP40) and incubated with 0.95 ml lysate for 2 hours.  Beads were washed three times with 1 ml Wash Buffer and then once with 1 ml IP Buffer.  In order to directly digest bead-bound proteins as well as elute proteins with biotin, suspended beads were divided in half and subsequently transferred to 50 ml Denaturation-Reduction Buffer (2 M urea, 50 mM Tris-HCl pH 8.0, 1 mM DTT) and 50 ml 1x Buffer BXT (IBA Lifesciences) dispensed into a single 96-well KF microtiter plate, respectively.  Purified proteins were first eluted at room temperature for 30 minutes with constant shaking at 1,100 rpm on a Thermomixer C incubator.  After removing eluates, on-bead digestion proceeded (below).  Strep-tagged protein expression in lysates and enrichment in eluates were assessed by western blot and silver stain, respectively. The KingFisher Flex instrument was equilibrated to 4°C overnight before use.  All protocol steps were performed using the slow mix speed and the following mix times: 30 seconds for all equilibration/wash steps, 2 hours for binding, and 1 minute for final bead release.  Three 10 second bead collection times were used between all steps. On-bead digestion.  MagStrep beads were denatured and reduced at 37°C for 30 minutes and returned to room temperature before alkylating in the dark with 3 mM iodoacetamide for 45 minutes and quenching with 3 mM DTT for 10 minutes.  To offset evaporation during digestion, 15 μl 50 mM Tris-HCl, pH 8.0 were added to samples.  MagStrep beads were then incubated at 37°C, initially for 4 hours with 1.5 μl trypsin (0.5 μg/μl; Promega) and then another 1-2 hours with 0.5 μl additional trypsin.  All steps were performed with constant shaking at 1,100 rpm on a Thermomixer C incubator.  Peptides were removed from beads, combined with 50 μl 50 mM Tris-HCl, pH 8.0 used to rinse beads and acidified with trifluoroacetic acid (final 0.5%, pH < 2.0).  Acidified peptides were desalted for MS analysis using a BioPureSPE Mini 96-Well Plate (20mg PROTO 300 C18; The Nest Group, Inc.) according to standard protocols. Mass spectrometry data acquisition and analysis. Samples were re-suspended in 4% formic acid, 2% acetonitrile solution, and separated by a reversed-phase gradient over a nanoflow C18 column (Dr. Maisch). Each sample was analyzed on two different mass spectrometers.  First, a 75 min acquisition, in which peptides were directly injected via a Easy-nLC 1200 (Thermo) into a Q-Exactive Plus mass spectrometer (Thermo), with all MS1 and MS2 spectra collected in the orbitrap.

### Data Protocol
QCloud was used to control instrument longitudinal performance during the project49. All proteomic data was searched against the human proteome (uniprot reviewed sequences downloaded February 28th, 2020), EGFP sequence, and the SARS-CoV-2 protein sequences using the default settings for MaxQuant50,51. Detected peptides and proteins were filtered to 1% false discovery rate in MaxQuant, and identified proteins were then subjected to protein-protein interaction scoring with both SAINTexpress15 and MiST16,43.

### Publication Abstract
An outbreak of the novel coronavirus SARS-CoV-2, the causative agent of COVID-19 respiratory disease, has infected over 290,000 people since the end of 2019, killed over 12,000, and caused worldwide social and economic disruption <sup>1,2</sup> . There are currently no antiviral drugs with proven efficacy nor are there vaccines for its prevention. Unfortunately, the scientific community has little knowledge of the molecular details of SARS-CoV-2 infection. To illuminate this, we cloned, tagged and expressed 26 of the 29 viral proteins in human cells and identified the human proteins physically associated with each using affinity-purification mass spectrometry (AP-MS), which identified 332 high confidence SARS-CoV-2-human protein-protein interactions (PPIs). Among these, we identify 66 druggable human proteins or host factors targeted by 69 existing FDA-approved drugs, drugs in clinical trials and/or preclinical compounds, that we are currently evaluating for efficacy in live SARS-CoV-2 infection assays. The identification of host dependency factors mediating virus infection may provide key insights into effective molecular targets for developing broadly acting antiviral therapeutics against SARS-CoV-2 and other deadly coronavirus strains.

### Keywords
['Sars-cov-2', 'Coronavirus', 'Covid-19', 'Ppi']

### Affiliations
['UCSF']

### Submitter
['Danielle Swaney']

### Lab Head
[{'title': 'Dr', 'affiliation': 'UCSF', 'email': 'daniswan@gmail.com', 'country': '', 'orcid': '', 'name': 'Nevan Krogan', 'id': '186000100'}]

### SDRF
characteristics[organism]: ['homo sapiens']
characteristics[organism part]: ['kidney']
characteristics[cell type]: ['not available']
characteristics[disease]: ['not available']
comment[label]: ['AC=MS:1002038;NT=label free sample']
comment[instrument]: ['NT=Q Exactive Plus;AC=MS:1002634']
comment[modification parameters]: ['NT=Carbamidomethyl;TA=C;AC=UNIMOD:4;MT=fixed']
comment[modification parameters].1: ['NT=Oxidation;TA=M;AC=UNIMOD:35;MT=variable']
comment[modification parameters].2: ['NT=Acetyl;AC=UNIMOD:1;PP=Protein N-term;MT=variable']

