### Accession
PXD029594

### Title
PRMT5 links the E2F pathway with a new source of tumour associated antigens

### Description
Immunopeptidomics analyses of CT26 cells +/- PRMT5 inhibition

### Sample Protocol
0.5 ml/sample Protein G beads (Expedion) were incubated with 5 mg/sample of antibody clone 34.1.2s (ATCC HB79, recognising H-2 Kd, Dd,  Ld for CT26), for 30 minutes at room temperature. The resin was washed with 10 cv (column bed volumes) of borate buffer (50 mM borate, 50 mM KCl, pH 8.0) and antibodies were cross-linked by adding 10 cv of 40 mM dimethyl pimelimidate in borate buffer (pH8.3) for 30 minutes at room temperature. The reaction was stopped with 10 cv of ice-cold 0.2 M Tris, pH 8.0, followed by a washing step of 10 cv of 0.1 M citrate, pH 3.0, to remove any unbound antibody, and finally equilibrated with 10 cv of 50 mM Tris, pH 8.0. Cell pellets were lysed in 3 ml lysis buffer (1% IGEPAL 630; 100 mM Tris, pH8.0; 300 mM NaCl; supplemented with complete Protease Inhibitor Cocktail, EDTA-free, Roche) by mild agitation. Samples were incubated for 45 minutes on ice. Lysates were then cleared by sequential centrifugation steps at 500 g for 10 minutes then 20,000 g for 1h at 4°C. Peptide-HLA class I complexes were captured by overnight incubation with the antibody-coated beads at 4°C under mild agitation. The lysate was then removed by gravity flow, and the column was washed consecutively with 10 ml wash buffer 1 (0.005% IGEPAL, 50 mM Tris pH8.0, 150 mM NaCl, 5 mM EDTA), 10 ml wash buffer 2 (50 mM Tris pH8.0, 150 mM NaCl), 10 ml wash buffer 3 (50 mM Tris pH8.0, 450 mM NaCl) and 10 ml wash buffer 4 (50 mM Tris pH8.0). Peptide-HLA complexes were eluted by addition of 5 cv of 10% acetic acid. Samples were loaded onto a Ultimate 3000 HPLC system (ThermoFisher Scientific) and peptides were separated from larger complex components using a monolithic column (4.6 x 50 mm ProSwift RP-1S, ThermoFisher Scientific) by applying a 10 minutes gradient from 2 to 35%  buffer B (0.1% TFA in acetonitrile) with a flow rate at 1000 µl/min. Each sample was fractionated in 15 fractions and alternate fractions containing the HLA peptides but not ß2-microglobulin, were pooled in two final fractions. Samples were dried, re-suspended in 20 µl of loading buffer (0.1% TFA, 1% ACN) and stored at −80°C prior to MS analysis.  CT26 HLA samples were measured on a Q Exactive HF-X coupled with an Ultimate 3000 RSLCnano System supplemented with a PepMap C18 column, 2 µm particle size, 75 µm × 50 cm (Thermo Scientific). Peptides were eluted using a 60 minute linear gradient of 3% to 25% acetonitrile in 5% DMSO, 0.1% formic acid in water at flow rate of 250 nl/min and 40°C, and introduced into the mass spectrometer using a nano EASY-Spray source at 2000 V (Thermo Scientific). The ion transfer tube was set to 305°C. Full MS mass range was 320-1600m/z, resolution was set at 120,000, and an AGC target of 300,000 was defined. Peptide ions were isolated at 1.6 amu isolation width. MS2 resolution was set to 60,000 at an AGC target of 50,000 and the collision energy was set at an energy of 28 for peptides with a charge state of 2–4 fragmentation of precursor ions and 25 for those with a charge state of 1-4.

### Data Protocol
MS data were analysed with Peaks v8.5 (Bioinformatics Solutions) for identification of peptide sequences matching to databases generated by integration of all reviewed human SwissProt protein entries (current at 1/10/2021), or all reviewed mouse SwissProt protein entries (current at 1/10/2021), combined with the respective 3 frame translations of the open reading frames obtained from the in-house RNA sequencing assemblies generated for the HCT116 data and CT26 cell lines, respectively. Searches were performed with the following parameters: no enzyme specificity, no peptide modifications, peptide tolerance: ± 5 ppm and fragment tolerance: ± 0.03 Da. The results were filtered using a false discovery rate of 1% established through parallel decoy database searches. For quantitative analysis, the data was analysed by Progenesis QI for proteomics (Waters). A one-way anova analysis was applied to assess significant regulation of peptides between conditions. GraphPad Prism 8 (GraphPad Software Inc), and Bio Venn (http://www.biovenn.nl/) were used for visualisation of the data.

### Publication Abstract
None

### Keywords
Prmt5, Ct26, Mhc, Hla

### Affiliations
University of Oxford

### Submitter
Nicola Ternette

### Lab Head
Dr Nicola Ternette
University of Oxford


