### Accession
PXD006450

### Title
Quantitative N-terminome of N-end rule mutant in Arabidopsis

### Description
According to the /N-end rule pathway, proteins with basic N-termini are targeted for degradation by the Arabidopsis thaliana E3 ligase, PROTEOLYSIS6 (PRT6). Here, we undertook a quantitative proteomics study of N-end rule mutant prt6, to investigate the impact of this pathway on the etiolated seedling. Isolation of N-terminal peptides using terminal amine isotope labelling of samples (TAILS) combined with Tandem mass tag (TMT) identified over 3000 unique N-termini. Trypsin and GluC have advantage of identification of Acetylated or neo-peptides respectively. Seed storage proteins and Cysteine proteins actives are differentially regulated in abundance in the prt6 mutants, which are represent downstream targets of transcription factors known to be N-end rule substrates.

### Sample Protocol
Plant growth and seedling treatments  Seeds were surface-sterilised and plated on nylon mesh (Sefar NITEX, 03-110/47) on 0.5 x MS medium containing 0.5 % (w/v) sucrose. After 2-3 days’ dark chilling at 4 °C, plates were exposed to light for 6 h to induce germination, then wrapped in foil and incubated in a vertical position at 22 °C for 4 d. Etiolated seedlings were harvested under green light.  Tandem Mass TagTM (TMT) labelling and enrichment of N-termini by TAILS TMT-TAILS was performed according to the Overall Lab protocols (Klein et al, 2015 & Prudova et al. 2016), with modifications. Briefly, Arabidopsis proteins were extracted in 6 M Guanidine hydrochloride, 100 mM HEPES (pH 7.5), containing Complete Mini protease inhibitor Cocktail, PhosSTOP Phosphatase Inhibitor Cocktail (Roche) and 100 µmol/L MG-132 (Sigma).  Protein aliquots (200 µg) were methanol/chloroform precipitated, re-dissolved, reduced and alkylated followed by whole protein isobaric labelling using TMT™6-plex reagents (Thermo Scientific, Waltham, MA). Multiple labels were used as biological replicates (n = 3). Equal amounts of proteins were mixed and methanol/chloroform precipitated, re-dissolved and divided into two.  After Trypsin or GluC digestion, the TMT labelled and N-terminally blocked peptides were negatively enriched by selective removal of the internal peptides, via an aldehyde functionalized dendritic polymer. The enriched N-termini were desalted using Sep-Pack light C18 (Waters). Separation of peptides was performed by reverse-phase chromatography using a Waters reverse-phase nano column as described in Zhang et al. (2015). All LC-MS/MS experiments were performed using a Dionex 3000 RSL cnano system (Thermo Scientific) and a Q Exactive™ Quadrupole-Orbitrap mass spectrometer (Thermo Scientific).

### Data Protocol
Raw data were searched against the TAIR10 database using Mascot 2.4 (Matrix Science) and Proteome Discoverer™ version 1.4.1.14 as described in Zhang et al. (2015) employing Top 10 peaks filter node and percolator nodes and reporter ions quantifier with semi-ArgC or semi-GluC enzyme specificity with a maximum of one missed cleavage. The carbamidomethylation (+57.021 Da) of cysteine and TMT isobaric labelling (+229.162 Da) of lysine were set as static modifications while TMT (+229.162 Da) labelling of the peptide N-termini, the acetylation of the peptide (+42.011) N-termini, methionine oxidation (+15.996) were considered dynamic. Mass tolerances were set to 10 ppm for MS and 0.06 Da for MS/MS. For quantification, integration window tolerance was set to 0.0075 Da. Each reporting ion was divided by the sum of total ion. The ratios were normalized by the medians of pre-TAILS samples searched with ArgC or GluC enzyme respectively. Statistical significance of quantification was assessed with an unpaired two-sample t-test on 4 degrees of freedom. Statistically significant results (p > 0.05) were further restricted to those with > 2 fold-change. No correction for multiplicity was applied. The statistical software package R was used for all analyses.

### Publication Abstract
The N-end rule pathway of targeted protein degradation is an important regulator of diverse processes in plants but detailed knowledge regarding its influence on the proteome is lacking. To investigate the impact of the Arg/N-end rule pathway on the proteome of etiolated seedlings, we used terminal amine isotopic labelling of substrates with tandem mass tags (TMT-TAILS) for relative quantification of N-terminal peptides in prt6, an Arabidopsis thaliana N-end rule mutant lacking the E3 ligase PROTEOLYSIS6 (PRT6). TMT-TAILS identified over 4000 unique N-terminal peptides representing c. 2000 protein groups. Forty-five protein groups exhibited significantly increased N-terminal peptide abundance in prt6 seedlings, including cruciferins, major seed storage proteins, which were regulated by Group VII Ethylene Response Factor (ERFVII) transcription factors, known substrates of PRT6. Mobilisation of endosperm &#x3b1;-cruciferin was delayed in prt6 seedlings. N-termini of several proteases were downregulated in prt6, including RD21A. RD21A transcript, protein and activity levels were downregulated in a largely ERFVII-dependent manner. By contrast, cathepsin B3 protein and activity were upregulated by ERFVIIs independent of transcript. We propose that the PRT6 branch of the pathway regulates protease activities in a complex manner and optimises storage reserve mobilisation in the transition from seed to seedling via control of ERFVII action.

### Keywords
N-end rule mutant etiolated seedlings tmt- labelling tails, N-terminome, Arabidopsis

### Affiliations
Department of Plant Sciences, Rothamsted Research, West  Common,  Harpenden, Herts, AL5 2JQ, UK
Plant Sciences and the Bioeconomy, Rothamsted Research, Harpenden, AL5 2JQ, UK

### Submitter
Hongtao Zhang

### Lab Head
Dr Hongtao Zhang
Department of Plant Sciences, Rothamsted Research, West  Common,  Harpenden, Herts, AL5 2JQ, UK


