### Accession
PXD013183

### Title
Proximity-dependent biotinylation by TurboID to identify protein-protein interaction networks in yeast

### Description
The use of proximity-dependent biotinylation assays coupled to mass spectrometry (PDB-MS) has changed the field of protein-protein interactions (PPI) studies. Yet, despite the recurrent and successful use of BioID-based PPI screening in mammalian cells, the implementation of PDB-MS in yeast has not been effective. Here we report a simple and rapid approach in yeast to effectively screen for proximal and interacting proteins in their natural cellular environment by using TurboID, a recently described version of the BirA biotin ligase. Using the protein arginine methyltransferase Rmt3 and the RNA exosome subunits, Rrp6 and Dis3, the application of PDB-MS in yeast by using TurboID was able to recover protein-protein interactions previously identified using other biochemical approaches and provided complementary information for a given protein bait. The development of a rapid and effective PDB assay that can systematically analyze PPIs in living yeast cells opens the way for large-scale proteomics studies in this powerful model organism.

### Sample Protocol
Trypsin digested samples were analyzed by liquid-chromatography (LC)-MS/MS, as described (Grenier St-Sauveur et al., 2013, Telekawa et al., 2018). Briefly, following trypsin digestion, peptides were sorted using a Dionex Ultimate 3000 nanoHPLC system. Approximately 2µg (10µL) of peptides in 1% (vol/vol) formic acid were injected with a flow of 4 µL/min on an Acclaim PepMap100 C18 column (0.3mmid x 5mm, Dionex Corporation). Peptides were eluted in a PepMap C18 nanocolumn (75 µm x 50 cm, Dionex Corporation) over 240 minutes with a flow of 200 nL/min using a gradient of 5–35% solvent B (90% acetonitrile with 0.1% formic acid). Through an EasySpray source, the HPLC system was combined to an OrbiTrap QExactive mass spectrometer (Thermo Fisher Scientific). The spray voltage was set to 2.0kV and the column temperature was set to 40°C. With a resolution of 70000 after the accumulation of 1000000 ions, full scan MS overall spectra (m/z 350–1600) in profile mode were acquired in the Orbitrap. After 50000 ions accumulated, fragmentation by collision induced dissociation (resolution of 17500, normalised energy 35%) of the 10 strongest peptide ions from the preview scan in the Orbitrap occurred. Top filling times were 250ms for the whole scans and 60ms for the MS/MS scans. We enabled precursor ion charge state screening and rejected all unassigned charge states as well as singly, seven and eight charged species. We limited to a maximum of 500 entries the dynamic exclusion list with a maximum retention length of 40s and a relative mass window of 10 ppm. To improve mass accuracy, the lock mass option was enabled. The Xcalibur software was used to acquire data (Mathieu et al., 2016).

### Data Protocol
The MaxQuant software package version 1.5.1.2 was used to process, search, and quantify the data collected, employing the S. pombe Uniprot proteome with 5142 protein annotations (Proteome ID: UP000002485), as recently described (Telekawa et al., 2018). The settings used for the MaxQuant analysis were: 2 miscleavages were allowed; fixed modification was carbamidomethylation on cysteine; enzyme was trypsin; variable modifications included in the analysis were methionine oxidation and protein N-terminal acetylation. For precursor ions, 7 ppm was used as mass tolerance and for fragment ions, 20 ppm was used as tolerance threshold. To obtain candid identifications with a false discovery rate (FDR) of less than 1%, every protein was considered based on the criterion that the amount of forward hits in the database was minimally 100-fold higher than the amount of reverse database hits. Each protein had a minimum of two peptides quantified. Isoforms and protein indistinguishable based on their identified peptides were grouped and organised in a single line with various accession numbers.

### Publication Abstract
The use of proximity-dependent biotinylation assays coupled to mass spectrometry (PDB-MS) has changed the field of protein-protein interaction studies. However, despite the recurrent and successful use of BioID-based protein-protein interactions screening in mammalian cells, the implementation of PDB-MS in yeast has not been effective. Here, we report a simple and rapid approach in yeast to effectively screen for proximal and interacting proteins in their natural cellular environment by using TurboID, a recently described version of the BirA biotin ligase. Using the protein arginine methyltransferase Rmt3 and the RNA exosome subunits, Rrp6 and Dis3, the application of PDB-MS in yeast by using TurboID was able to recover protein-protein interactions previously identified using other biochemical approaches and provided new complementary information for a given protein bait. The development of a rapid and effective PDB assay that can systematically analyze protein-protein interactions in living yeast cells opens the way for large-scale proteomics studies in this powerful model organism.

### Keywords
Proximity-dependent biotinylation; turboid; yeast; s. pombe; rmt3, Rna exosome

### Affiliations
University of Sherbrooke
RNA Group, Department of Biochemistry, Universit� de Sherbrooke, Sherbrooke, Qc, Canada

### Submitter
Danny Bergeron

### Lab Head
Dr Francois Bachand
RNA Group, Department of Biochemistry, Universit� de Sherbrooke, Sherbrooke, Qc, Canada


