### Accession
PXD032945

### Title
PhosphoProtein-Peptide Turnover Profiling, validation

### Description
We combined pulsed stable isotope labeling of amino acids (pSILAC) with enrichment of phosphorylated peptides in a method we term PhosphoProtein-Peptide Turnover Profiling (PhosphoPPToP). We applied the method to HeLa cells to find peptides whose clearance rates differ from the protein median. Using theoretical modelling, we show that readouts from PPToP are primarily defined by the wiring of the PTM-modification network and not by effects of the PTM on a protein's proteolytic stability. Thus PPToP can be used to identify PTMs occurring, e.g., preferentially early or late in a protein's lifetime. This enables identification of, e.g., protein maturation or protein complex assembly intermediates. This dataset covers experimental testing of identified phosphosites with PPToP. Here, proteins of interest are expressed as GFP-fusion constructs in HeLa cells and subjected to a SILAC pulse. Thereafter, proteins are pulled-down with anti-GFP beads to isolate the exogenously expressed constructs. We multiplex SILAC timepoints from wild-type and mutant constructs for every given protein of interest using TMT16.

### Sample Protocol
Sample preparation and LC-MS/MS. 23 proteins identified as carrying phosphorylation sites of interest from the proteome-wide experiment were cloned as either N or C terminal fusions to monomeric EGFP (carrying the A206K mutation) in a vector under a Ubc promoter (kind gift from Daniel Heid and Judith Zaugg). Phosphorylation sites of interest were mutated to either alanine and/or aspartate (see Table SN for full list of constructs). Cloning and mutagenesis was performed by GenScript (GenScript Biotech Netherlands B.V.). Constructs were overexpressed in HeLa cells as follows. Cells were seeded onto 12-well plates at 90E3 cells per well in Light SILAC medium the day prior to transfection using a single well per construct, per time point, per replicate. Each well was transfected with 360 ng of plasmid using Lipofectamine 3000 (Invitrogen) according to manufacturerÃ¢â‚¬â„¢s instructions, and let to transfect for 24 h, after which medium was exchanged to Heavy medium for all wells simultaneously. Cells were collected after the designated time in Heavy medium, placed on ice, washed with PBS and lysed in lysis buffer (50 mM Hepes pH7.5, 150 mM NaCl, 0.5 mM EDTA, 0.1% SDS, 1% Triton X-100, 1% deoxycholate). After lysis, cells were collected by scraping, diluted 1:1 with dilution buffer (50 mM Hepes pH7.5, 150 mM NaCl, 5 mM MgCl2, 0.625 U/ul Benzonase (Merck, 70746)), and left on ice for >30 min. Lysates were cleared by spinning 5 min at 2000 g at 4 Ã‚Â°C, and filtering through a pre-wetted 0.22 Ã‚Âµm filter plate (Merck millipore). For the GFP pull-down, cleared lysate was incubated with 2 Ã‚Âµl washed GFP-Trap magnetic agarose beads (Chromotek) per well for >4 h at 4 Ã‚Â°C, washed twice with wash buffer with NP40 (50 mM Hepes pH7.5, 150 mM NaCl, 5 mM MgCl2, 0.05% NP40), and thrice with detergent-free wash buffer (50 mM Hepes pH7.5, 150 mM NaCl, 5 mM MgCl2). Proteins were digested on beads with trypsin and Lys-C (5 ng/Ã‚Âµl final concentration each) in 90 mM HEPES (pH 8.5), 5 mM chloroacetic acid and 1.25 mM TCEP overnight at room temperature shaking at 500 rpm. Peptides were eluted using 2% DMSO and dried in a speedvac. Dry peptides were reconstituted in 5 Ã‚Âµl water and labeled by adding 2 Ã‚Âµl TMT label (20 Ã‚Âµg/Ã‚Âµl in acetonitrile (ACN)) (TMTpro 16 plex, Thermo Fisher Scientific) and incubating 1 h at room temperature. Labeling was quenched with hydroxylamine (1.1% final concentration), and samples pooled to make full TMT16 sets as shown in Figure 5B. Pooled sets were desalted on an OASIS HLB Ã‚ÂµElution plate (Waters 186001828BA); washing thrice with 0.05% FA, eluting with 80% ACN, 0.05% FA, and drying in a speedvac. TMT16-labeled peptides from pull-down experiments were reconstituted in 0.1% FA, 4% ACN and analyzed by nanoLC-MS/MS on the same hardware described above. LC-MS/MS analysis parameters were as follows: Peptides were loaded on the trapping cartridge using solvent A for 3 min with a flow of 30 Ã‚Âµl/min. Peptides were separated on the analytical column at 40 Ã‚Â°C with a constant flow of 0.3 Ã‚Âµl/min applying a 104 min gradient of 6Ã¢â‚¬â€œ28% of solvent B in A, followed by a 4 min gradient (28Ã¢â‚¬â€œ40%), and a 4 min washing step at 80% solvent B. Peptides were directly analyzed in positive ion mode with a spray voltage of 2.2 kV and a ion transfer tube temperature of 275 Ã‚Â°C. Full scan MS spectra with a mass range of 375Ã¢â‚¬â€œ1500 m/z were acquired on the orbitrap using a resolution of 120 000 with a maximum injection time of 50 ms. Data-dependent acquisition was used in top 10 mode. Precursors were isolated on the quadrupole with an intensity threshold of 2e5, charge state filter of 2Ã¢â‚¬â€œ7, and an isolation window of 0.7 m/z. Precursors were fragmented using HCD at 34% collision energy. MS/MS spectra were acquired on the orbitrap, at 30 000 resolution, a maximum injection time of 100 ms, scan range in first mass mode (at 110 m/z), and a dynamic exclusion window of 20 s.

### Data Protocol
Analysis of MS raw files. MS raw files were processed using IsobarQuant (Franken et al., 2015) and peptide and protein identification was obtained with Mascot 2.5.1 (Matrix Science) using a reference human proteome (uniprot Proteome ID: UP000005640, downloaded 9.6.2020) modified to include the overexpressed protein constructs in question, known common contaminants and reversed protein sequences. Mascot searches were done only for old (light SILAC label) proteins. Parameters were: Trypsin/P; max. 3 missed cleavages; peptide tolerance 10 ppm; MS/MS tolerance 0.02 Da; fixed modifications: Carbamidomethyl (C), TMT16plex (K); variable modifications: Acetyl (Protein N-term), Oxidation (M), Phospho (ST), Phospho (Y), TMT16plex (N-term).

### Publication Abstract
Post-translational modifications (PTMs) regulate various aspects of protein function, including degradation. Mass spectrometric methods relying on pulsed metabolic labeling are popular to quantify turnover rates on a proteome-wide scale. Such data have traditionally been interpreted in the context of protein proteolytic stability. Here, we combine theoretical kinetic modeling with experimental pulsed stable isotope labeling of amino acids in cell culture (pSILAC) for the study of protein phosphorylation. We demonstrate that metabolic labeling combined with PTM-specific enrichment does not measure effects of PTMs on protein stability. Rather, it reveals the relative order of PTM addition and removal along a protein's lifetime-a fundamentally different metric. This is due to interconversion of the measured proteoform species. Using this framework, we identify temporal phosphorylation sites on cell cycle-specific factors and protein complex assembly intermediates. Our results thus allow tying PTMs to the age of the modified proteins.

### Keywords
Post-translational modification, Phosphorylation, Tmt16, Ap-ms, Gfp-fusion, Silac, Pull-down, Mass spectrometry, Protein turnover, Protein maturation

### Affiliations
Genome Biology Unit and Structural and Computational Biology Unit, European Molecular Biology Laboratory, 69117 Heidelberg, Germany
Genome Biology Unit and Proteomics Core Facility, European Molecular Biology Laboratory, 69117 Heidelberg, Germany

### Submitter
Henrik Hammarén

### Lab Head
Dr Mikhail Savitski
Genome Biology Unit and Proteomics Core Facility, European Molecular Biology Laboratory, 69117 Heidelberg, Germany


