### Accession
PXD038377

### Title
Spike-in samples DDA/DIA for generalized peakgroup scoring models

### Description
The statistical validation of peptide and protein identifications in mass spectrometry proteomics is a critical step in the analytical workflow. This is particularly important in discovery experiments to ensure only confident identifications are accumulated for downstream analysis and biomarker consideration. However, the inherent nature of discovery proteomics experiments leads to scenarios where the search space will inflate substantially due to the increased number of potential proteins that are being queried in each sample. In these cases, issues will begin to arise when the machine learning algorithms that are trained on an experiment specific basis cannot accurately distinguish between correct and incorrect identifications and will struggle to accurately control the false discovery rate. Here, we propose an alternative validation algorithm trained on a curated external data set of 2.8 million extracted peakgroups that leverages advanced machine learning techniques to create a generalizable peakgroup scoring (GPS) method for data independent acquisition (DIA) mass spectrometry. By breaking the reliance on the experimental data at hand and instead training on a curated external dataset, GPS can confidently control the false discovery rate while increasing the number of identifications and providing more accurate quantification in different search space scenarios. To first test the performance of GPS in a standard experimental environment and to provide a benchmark against other methods, a novel spike-in data set with known varying concentrations was analyzed. When compared to existing methods GPS increased the nunmber of identifications by 5-18\% and was able to provide more accurate quantification by increasing the number of ratio validated identifications by 24-74\%. To evaluate GPS in a larger search space, a novel data set of 141 blood plasma samples from patients developing acute kidney injury after sepsis was searched with a human tissue spectral library (10000+ proteins). Using GPS, we were able to provide a 207-377\% increase in the number of candidate differentially abundant proteins compared to the existing methods while maintaining competitive numbers of global identifications. Finally, using an optimized human tissue library and workflow we were able to identify 1205 proteins from the 141 plasma samples and increase the number of candidate differentially abundant proteins by 70.87\%. With the addition of machine learning aided differential expression, we were able to identify potential new biomarkers for stratifying subphenotypes of acute kidney injury in sepsis. These findings suggest that by using a generalized model such as GPS in tandem with a massive scale spectral library it is possible to expand the boundaries of discovery experiments in DIA proteomics. GPS is open source and freely available on github at (\url{https://github.com/InfectionMedicineProteomics/gps})

### Sample Protocol
We generated a spike-in dataset of known concentrations of Yeast tryptic digest peptides (Promega) spiked in to a constant mouse kidney background. The mouse kidney material was obtained from the a previous project. All animal use and procedures were approved by the local Malmö/Lund Institutional Animal Care and Use Committee, ethical permit number 03681-2019. C57BL/6 mice (Janvier, Le Genest-Saint-Isle, France) were sacrificed, and kidneys were isolated into a tube containing DPBS and silica beads (1 mm diameter, Techtum). The kidneys were then homogenized using MagNAlyser (Roche) and stored at -80°C.  Homogenates were then thawed and centrifuged at 10,000g for 10’ at 4°C. The supernatant containing the soluble proteins was collected and protein content was estimated using BCA (Pierce). 25ug of protein was taken for reduction, alkylation, digestion and C18 clean up, as described below. A serial dilution series of yeast peptides (1X, 2X, 4X, 8X, 16X, 32X) was performed, and 10 technical replicates of each concentration were sampled for a total of 60 samples. Internal retention time peptides were also added in to each sample. Each of the 60 samples were analyzed on the an Orbitrap HF-X using both DDA and DIA. All protein samples were denatured with 8M urea and reduced with 5 mM Tris(2-carboxyethyl)phosphine hydrochloride, pH 7.0 for 45 min at 37 °C, and alkylated with 25 mM iodoacetamide (Sigma) for 30 min followed by dilution with 100 mM ammonium bicarbonate to a final urea concentration below 1.5 M. Proteins were digested by incubation with trypsin (1/100, w/w, Sequencing Grade Modified Trypsin, Porcine; Promega) for at least 9 h at 37 °C. Digestion was stopped using 5\% trifluoracetic acid (Sigma) to pH 2 to 3. The peptides were cleaned up by C18 reversed-phase spin columns as per the manufacturer's instructions (Silica C18 300 Å Columns; Harvard Apparatus). Solvents were removed using a vacuum concentrator (Genevac, miVac) and were resuspended in 50 microliters HPLC-water (Fisher Chemical) with 2\% acetonitrile and 0.2\% formic acid (Sigma). Peptide analyses were performed on a Q Exactive HF-X mass spectrometer (Thermo Fisher Scientific) connected to an EASY-nLC 1200 ultra-HPLC system (Thermo Fisher Scientific). Peptides were trapped on precolumn (PepMap100 C18 3 microliter; 75 microliter × 2 cm; Thermo Fisher Scientific) and separated on an EASY-Spray column (ES903, column temperature 45 °C; Thermo Fisher Scientific). Equilibrations of columns and sample loading were performed per manufacturer's guidelines. Mobile phases of solvent A (0.1\% formic acid), and solvent B (0.1\% formic acid, 80\% acetonitrile) was used to run a linear gradient from 5\% to 38\% over various gradient length times at a flow rate of 350 nl/min. The 44 variable windows DIA acquisition method is described by Bruderer et al 2017.

### Data Protocol
An experiment specific library for the spike-in data was built by analyzing the samples acquired using DDA using FragPipe (v18.0). First the samples were searched using MSFragger (v3.5) with default parameters using a fasta file of Swiss-prot reviewed saccharomyces cerevisiae and mus musculus proteomes concatenated with reverse sequence decoy proteins. Peptide spectrum matches (PSMs) were validated using Percolator. The Philosopher toolkit (v4.4.0) was used to perform protein level FDR control with ProteinProphet, generate downstream reports, and filter the resulting identifications. The python package easypqp was then used to convert and format the library for use by OpenSwath and the OpenMS (2.6.0) tool chain for formatting PQP files was used to optimize the assays in the library. 10 spiked-in retention time peptides (iRT) were used for alignment and retention time correction for each sample. OpenSwath (2.6.0) and GPS/PyProphet/Percolator were used for downstream analysis.

### Publication Abstract
Data independent acquisition mass spectrometry (DIA-MS) has recently emerged as an important method for the identification of blood-based biomarkers. However, the large search space required to identify novel biomarkers from the plasma proteome can introduce a high rate of false positives that compromise the accuracy of false discovery rates (FDR) using existing validation methods. We developed a generalized precursor scoring (GPS) method trained on 2.75 million precursors that can confidently control FDR while increasing the number of identified proteins in DIA-MS independent of the search space. We demonstrate how GPS can generalize to new data, increase protein identification rates, and increase the overall quantitative accuracy. Finally, we apply GPS to the identification of blood-based biomarkers and identify a panel of proteins that are highly accurate in discriminating between subphenotypes of septic acute kidney injury from undepleted plasma to showcase the utility of GPS in discovery DIA-MS proteomics.

### Keywords
Yeast, Mouse, Spike-in, Machine learning, Proteome, Software, Dia

### Affiliations
Lund University, Faculty of Medicine, Department of Clinical Sciences Lund, Division of Infection Medicine, Lund, Sweden
Faculty of Medicine, Lund University

### Submitter
Aaron Scott

### Lab Head
Dr Lars Malmström
Lund University, Faculty of Medicine, Department of Clinical Sciences Lund, Division of Infection Medicine, Lund, Sweden


