### Accession
PXD037320

### Title
Comparative analysis of proteomes and secretomes of human mesenchymal stem cells isolated from six sources during osteogenic differentiation

### Description
Ability to perform osteogenic differentiation is one of the minimal criteria of mesenchymal stem cells (MSCs). Still, it is generally unknown whether osteogenic differentiation is universal cell fate or various phenotypically similar cell states. Besides this, MSCs and their secretomes are actively using for cell/cell-free therapy development, but systemic inter-source variation in MSCs secretomes, proteomes and differentiation mechanisms are still poorly understood. Therefore, here we compared proteomic and secretomic profiles of human mesenchymal cells from six sources: osteoblasts (bone), WJ-MSCs (Warton’s jelly), AD-MSCs (adipose), PDLSCs (tooth: Periodontal Ligament Stem Cells), DPSCs (tooth: Dental Pulp Stem Cells) and GFs (tooth: Gingival Fibroblasts). For experiments we used cells in early passages (3-5) isolated from 3-6 individuals. All cells were compared in standard cultivation and in the 10th day after induction of osteogenic differentiation.

### Sample Protocol
All cells were cultured in the same conditions (37 °C in 5% CO2) in DMEM (41966-052, Gibco) supplemented with 15% FBS, 2 mM L-glutamine and penicillin/streptomycin until 70–80% confluence. The medium was changed twice a week. Cell cultures were routinely checked for mycoplasma contamination each week. Cells of 3-4 passages were used. Proteomes and secretomes were analyzed in control (standard cultivation) and on the 10th day of osteogenic differetiation. To induce osteogenic differentiation, we used a classic osteogenic medium, usually used for in vitro osteogenic differentiation: DMEM supplemented with 10% FBS, 2 mM L-glutamine, penicillin/streptomycin, 50 mg/ml ascorbic acid, 0.1 mM dexamethasone, and 10 mM b–glycerophosphate. Osteogenic differentiation was verified by Alizarin red stain (stain for calcium deposition) on the 21st day of osteogenic differentiation. After cultivation in standard or osteogenic conditions (10th day from the start of differentiation) the cells were gently washed six times with PBS (37 °C) and cultured for 48 hours with in FBS depleted medium (DMEM, 2 mM L-glutamine, penicillin/streptomycin). We compared cell morphology prior and after secretome harvesting and found no evidence of extensive cell death. After 48h conditioned medium was collected, centrifuged for 500g (20 min, +4 °C) to remove the floating cells, transferred to the new tube and centrifuged for 1500g (20 min, +4 °C) to remove the large debris. Then the medium was transferred to the new tube and centrifuged for 20000g (20 min, +4 °C) to remove all the debris. Secretome proteins were precipitated from the conditioned medium by standard acetone precipitation (EM grade acetone; EMS). The protein pellet was resuspended in 8M Urea/50mM ammonium bicarbonate (Sigma Aldrich) and centrifuged (12 000 g, 10 min, 4°C). The protein concentration was measured by a Qubit fluorometer using QuDye Protein Quantification Kit according to the manufacturer's recommendations. The cells were washed twice with PBS and lysed in a Petri dish with RIPA buffer supplemented with a cOmplete protease inhibitor cocktail. Then, the dishes were transferred to the ice for 30 min. Cell lysates were stored at −80°C before use. The samples were sonicated for 10 min in an ultrasonic bath with ice and centrifuged (12 000 g, 20 min, 4°C). Proteins were cleaned by standard acetone precipitation. The protein pellet was resuspended in 8M Urea/50mM ammonium bicarbonate and centrifuged (12 000 g, 10 min, 4°C). The protein concentration was measured by a Qubit fluorometer using QuDye Protein Quantification Kit according to the manufacturer's recommendations. 20 μg of each sample was digested by trypsin. Sample volumes were equalized by the addition of 8M Urea/50mM ammonium bicarbonate. Disulfide bonds were reduced and alkylated by incubation for 1 h at 37 °C in 5 mM DTT with subsequent incubation in 15 mM iodoacetamide for 30 min in the dark at room temperature. Then, the samples were diluted with seven volumes of 50 mM ammonium bicarbonate and incubated for 16 h at 37 °C with 400 ng of trypsin (ratio 1:50; “Trypsin Gold”, Promega).  After digestion, the samples were desalted by solid phase extraction in self-made “Stage tips”. Desalted peptides were evaporated in centrifugal concentrator Eppendorf Concentrator plus and dissolved in water/0.1% formic acid for further LC-MS/MS analysis. Approximate 500 ng of tryptic peptides were used for shotgun proteomics analysis by nanoHPLC-MS/MS with trapped ion mobility mass spectrometry in TimsToF Pro (Bruker). For secretome analysis HPLC was performed in two-column separation mode with Acclaim™ PepMap™ 5 mm Trap Cartridge and Bruker Fifteen (C18 ReproSil AQ, 150 × 0.75 мм, 1.9 мкм, 120A) in gradient mode with 400 nL/min flow rate and 40 °C. Phase A was water/0.1% formic acid, phase B was acetonitrile/0.1% formic acid. The gradient was from 2% to 38% phase B for 28 min with subsequent wash with 95% phase B for 6 min. Each secretome sample was analyzed in two analytical replicates. For the analysis of proteomes HPLC was performed in two-column separation mode with Acclaim™ PepMap™ 5 mm Trap Cartridge and Bruker Ten (C18 ReproSil AQ, 100 × 0.75 мм, 1.9 мкм, 120A) in gradient mode with 400 nL/min flow rate and 45 °C. Phase A was water/0.1% formic acid, phase B was acetonitrile/0.1% formic acid. The gradient was from 2% to 35% phase B for 35 min with subsequent wash with 95% phase B for 10 min.  MS/MS acquisition for both proteomes and secretomes was performed in automatic DDA PASEF mode with 0.5 s cycle in positive polarity with the fragmentation of ions with at least two charges in m/z range from 100 to 1700 and ion mobility range from 0.85 to 1.30 1/K0. CaptiveSpray ion source was used for electrospray ionization with 1600 V of capillary voltage, 3 l/min N2 flow, and 180 °C source temperature.

### Data Protocol
Protein identification in both cases was performed using the same Homo sapiens reference proteome (UP000005640, downloaded 27.06.2022) with contaminations included in the FragPipe software (v. 18.0) according to default LFQ-MBR workflow. The search parameters were: parent and fragment mass error tolerance 10 ppm, protein and peptide FDR less than 1%, protease rule –trypsin (cleave after KR, but not before P), two possible missed cleavage sites. Cysteine carbamidomethylation was set as fixed modification. Methionine oxidation and acetylation of protein N-term were set as variable modifications.

### Publication Abstract
Mesenchymal stem cells (MSCs) are widely used in therapy, but the differences between MSCs of various origins and their ability to undergo osteogenic differentiation and produce extracellular matrix are not fully understood. To address this, we conducted a comparative analysis of mesenchymal cell primary cultures from 6 human sources, including osteoblast-like cells from the adult femur, adipose-derived stem cells, Wharton's jelly-derived mesenchymal cells, gingival fibroblasts, dental pulp stem cells, and periodontal ligament stem cells. We analyzed these cells' secretome, proteome, and transcriptome under standard and osteogenic cultivation conditions. Despite the overall similarity in osteogenic differentiation, the cells maintain their embryonic specificity after isolation and differentiation in vitro. Furthermore, we propose classifying mesenchymal cells into 3 groups: dental stem cells of neural crest origin, mesenchymal stem cells, and fetal stem cells. Specifically, fetal stem cells have the most promising secretome for various applications, while mesenchymal stem cells have a specialized secretome optimal for extracellular matrix production. Nevertheless, mesenchymal cells from all sources secreted core bone extracellular matrix-associated proteins. In conclusion, our study illuminates the distinctive characteristics of mesenchymal stem cells from various sources, providing insights into their potential applications in regenerative medicine and enhancing our understanding of the inherent diversity of mesenchymal cells in vivo.

### Keywords
Timstof pro, Secretome, Msc, Osteogenic differeniation

### Affiliations
head of Laboratory of Regenerative Biomedicine, Institute of Cytology, Russian Academy of Sciences
Emmanuel Levy Lab, Dept. of Chemical and Structural Biology, Weizmann Institute of Science

### Submitter
Arseniy Lobov

### Lab Head
Dr Anna Malashicheva
head of Laboratory of Regenerative Biomedicine, Institute of Cytology, Russian Academy of Sciences


