### Accession
PXD039773

### Title
Elexacaftor/VX-445-mediated CFTR interactome remodeling reveals differential correction driven by mutation-specific translational dynamics

### Description
Cystic fibrosis (CF) is one of the most prevalent lethal genetic diseases with over 2000 identified genetic variants in the cystic fibrosis transmembrane conductance regulator (CFTR) gene. Pharmacological chaperones such as Lumacaftor (VX-809), Tezacaftor (VX-661) and Elexacaftor (VX-445) treat mutation-induced defects by stabilizing CFTR and are called correctors. These correctors improve proper folding and thus facilitate processing and trafficking to increase the amount of functional CFTR on the cell surface. Yet, CFTR variants display differential responses to each corrector. Here, we report variants P67L and L206W respond similarly to VX-809 but divergently to VX-445 with P67L exhibiting little rescue when treated with VX-445. We investigate the underlying cellular mechanisms of how CFTR biogenesis is altered by correctors in these variants. Affinity purification-mass spectrometry (AP-MS) multiplexed with isobaric Tandem Mass Tags (TMT) was used to quantify CFTR protein-protein interaction changes between variants P67L and L206W. VX-445 facilitates unique proteostasis factor interactions especially in translation, folding, and degradation pathways in a CFTR variant-dependent manner. A number of these interacting proteins knocked down by siRNA, such as ribosomal subunit proteins, moderately rescued fully glycosylated P67L. Importantly, these knock-downs sensitize P67L to VX-445 and further enhance the correction of this variant. Our results provide a better understanding of VX-445 biological mechanism of action and reveal cellular targets that may sensitize unresponsive CFTR variants to known and available correctors.

### Sample Protocol
Co-Immunoprecipitation and multiplexed LC-MS/MS   Co-immunoprecipitation (co-IP) of CFTR bound with interactors was carried out as described previously60. Briefly, cell lysates were pre-cleared with 4B Sepharose beads (Sigma) at 4°C for 1 h while rocking. Precleared lysates were then immunoprecipitated with Protein G beads complexed to 24-1 antibody (6 mg antibody/mL of beads) overnight at 4°C while rocking. Beads were washed three times with TNI buffer, twice with TN buffer, and frozen at –80°C for at least 1 h. Proteins were then eluted with shaking at 37°C for 1 h with elution buffer (0.2 M glycine, 0.5% IGEPAL CA-630, pH 2.3). Elution steps were repeated once and combined to immediately neutralize with ammonium bicarbonate solution.   MS sample preparation of co-IP samples were performed as described previously29. Briefly, samples were precipitated in methanol/chloroform. Precipitated pellet was rinsed and dried to reconstitute in 1% Rapigest SF (Waters). Samples were reduced with 5 mM TCEP (Sigma), alkylated with 10 mM iodoacetamide (Sigma), and digested with 0.5 μg of trypsin (Sequencing Grade, Promega, or Pierce) overnight in 50mM HEPES (pH 8.0) at 37°C with shaking. Digested peptides were labeled with TMT 11-plex reagents (Thermo Fisher). TMT-labeled samples were pooled and acidified with MS-grade formic acid (Sigma) to remove cleaved Rapigest SF via centrifugation. Samples were concentrated using a SpeedVac (Thermo Fisher) and resuspended in buffer A (95% water, 4.9% acetonitrile, and 0.1% formic acid). Sample was then loaded onto a triphasic MudPIT61 column using a high-pressure chamber.   LC-MS/MS analysis was performed on an Exploris 480 (Thermo Fisher) mass spectrometer equipped with an UltiMate3000 RSLCnano System (Thermo Fisher) as described previously29. Briefly, 10 μl sequential injections of 0, 10, 30, 60, and 100% buffer C (500 mM ammonium acetate in buffer A) were performed followed by a final injection of 90% buffer C with 10% buffer B (99.9% acetonitrile, 0.1% formic acid). Each step consisted of a 90-min gradient from 4 to 40% B with a flow rate of either 300 or 500 nL/min, followed by a 15-min gradient from 40 to 80% B with a flow rate of 500 nL/min on a 20-cm fused silica microcapillary column (ID 100 μM) ending with a laser-pulled tip filled with Aqua C18, 3 μm, 100 Å resin (Phenomenex). Electrospray ionization was performed directly from the analytical column by applying a voltage of 2.0 or 2.2 kV with an inlet capillary temperature of 275°C. Data-dependent acquisition of MS/MS spectra was performed by scanning from 300 to 1800 m/z with a resolution of 60,000 to 120,000. Peptides with an intensity above 1.0E4 with charge state 2–6 from each full scan were fragmented by HCD using normalized collision energy of 35 to 38 with a 0.4 m/z isolation window, 120 ms maximum injection time at a resolution of 45,000, scanned from 100 to 1800 m/z or defined a first mass at 110 m/z and dynamic exclusion set to 45 or 60s and a mass tolerance of 10 ppm.   Peptide identification and TMT-based protein quantification was carried out using Proteome Discoverer 2.4 as described previously29. MS/MS spectra were extracted from Thermo XCaliber .raw file format and searched using SEQUEST against a UniProt human proteome database (released 03/25/2014) containing 20,337 protein entries. The database was curated to remove redundant protein and splice-isoforms and supplemented with common biological MS contaminants. Searches were carried out using a decoy database of reversed peptide sequences and the following parameters: 10 ppm peptide precursor tolerance, 0.02 Da fragment mass tolerance, minimum peptide length of six amino acids, trypsin cleavage with a maximum of two missed cleavages, static cysteine modification of 57.0215 Da (carbamidomethylation), and static N-terminal and lysine modifications of 229.162932 Da (TMT 11-plex). SEQUEST search results were filtered using Percolator to minimize the peptide false discovery rate to 1% and a minimum of two peptides per protein identification. TMT reporter ion intensities were quantified using the Reporter Ion Quantification processing node in Proteome Discoverer 2.4 and summed for peptides belonging to the same protein.

### Data Protocol
Interactor filtering and statistical analysis   A total of 6 TMT-11plex sets of samples were analyzed via LC-MS/MS over six separate mass spectrometry runs. Interactors co-immunoprecipitated with CFTR were filtered by comparing against a mock transfection control (tdTomato). The log2 fold change of total peptide normalized abundances of each protein over mock were computed within each run. The 6 separate log2 fold change representing individual MS runs were then averaged to yield the consensus log2 fold change over mock values for each protein and a paired two-tailed t test was used to calculate the p-value (Supplemental Table 1 and 2).   To filter for true interactors of CFTR, a curved filter combining log2 fold change and p-value was used as described previously62. Briefly, the histogram of log2 fold changes over mock were fitted to a Gaussian curve using a nonlinear least- square fit to determine the standard deviation σ. Fold change cutoff for interactors was set to 1 σ. A reciprocal curve with the equation y > c/(x − x0), where y = p-value, x = log2 fold change, x0 = fold change cutoff (1 σ), and c = the curvature (c = 0.8) was used to filter interactors in each condition. These interactors were pooled to generate a master list of true CFTR interactors. The abundance values of these interactors in each condition were then normalized to WT CFTR. Briefly, each abundance value of an individual run was log2 transformed and averaged to yield the consensus log2 grouped abundance. These values for each condition were subtracted by WT log2 grouped abundance for each protein and subsequently subtracted by WT CFTR log2 grouped abundance to yield the log2 fold change of protein abundances against the WT standardized to CFTR levels across conditions.   For aggregate pathways statistics in violin plots, we used a one-way ANOVA with Geisser-Greenhouse correction and post-hoc Tukey multiple comparisons testing to evaluate the statistically significant difference between conditions for a given pathway.

### Publication Abstract
Cystic fibrosis (CF) is one of the most prevalent lethal genetic diseases with over 2000 identified mutations in the cystic fibrosis transmembrane conductance regulator (CFTR) gene. Pharmacological chaperones such as lumacaftor (VX-809), tezacaftor (VX-661), and elexacaftor (VX-445) treat mutation-induced defects by stabilizing CFTR and are called correctors. These correctors improve proper folding and thus facilitate processing and trafficking to increase the amount of functional CFTR on the cell surface. Yet, CFTR variants display differential responses to each corrector. Here, we report that variants P67L and L206W respond similarly to VX-809 but divergently to VX-445 with P67L exhibiting little rescue when treated with VX-445. We investigate the underlying cellular mechanisms of how CFTR biogenesis is altered by correctors in these variants. Affinity purification-mass spectrometry multiplexed with isobaric tandem mass tags was used to quantify CFTR protein-protein interaction changes between variants P67L and L206W. VX-445 facilitates unique proteostasis factor interactions especially in translation, folding, and degradation pathways in a CFTR variant-dependent manner. A number of these interacting proteins knocked down by siRNA, such as ribosomal subunit proteins, moderately rescued fully glycosylated P67L. Importantly, these knockdowns sensitize P67L to VX-445 and further enhance the trafficking correction of this variant. Partial inhibition of protein translation also mildly sensitizes P67L CFTR to VX-445 correction, supporting a role for translational dynamics in the rescue mechanism of VX-445. Our results provide a better understanding of VX-445 biological mechanism of action and reveal cellular targets that may sensitize nonresponsive CFTR variants to known and available correctors.

### Keywords
Cftr, Ap-ms, Cf, Trikafta, Interactomics, Elexacaftor

### Affiliations
Departments of Chemistry, Biological Sciences, Pathology, Microbiology and Immunology, Vanderbilt University
Vanderbilt University

### Submitter
Jonathan Davies

### Lab Head
Dr Lars Plate
Departments of Chemistry, Biological Sciences, Pathology, Microbiology and Immunology, Vanderbilt University


