### Accession
PXD009766

### Title
Clinical Proteomics of Breast Cancer Unravels a Novel Layer of Breast Cancer Classification

### Description
Breast cancer classification has been in the focus of numerous large worldwide efforts, analyzing the molecular basis of breast cancer subtypes, aiming to associate them with clinical outcome and improve the current diagnostic routine. Genomic and transcriptomic profiles of breast cancer have been well established, however the proteomic contribution to these profiles is yet to be elucidated. In this work, we examined inter-tumor heterogeneity by performing a mass-spectrometry (MS)-based proteomic analysis of more than 130 clinical breast samples originating from three breast cancer subtypes and healthy tissue. Unsupervised analysis identified four proteomic clusters, among them one that represents a novel luminal subtype characterized by increased cancer signaling. This subtype was further validated using an independent protein-based dataset, but not in two independent transcriptome cohorts. Altogether, our results demonstrate the importance of deep proteomic analysis, which may affect cancer treatment decision making.

### Sample Protocol
We used the super-SILAC standard to serve as a reference sample for relative quantification and for normalization between the three datasets (11). The standard was prepared as previously described (11). Briefly, four breast cancer cell lines of different origin, stage and receptor status (HCC1599, MCF7, HCC1937 and HCC2218) and normal mammary epithelial cells (HMEC) were SILAC labeled with heavy lysine and arginine and their lysates were combined at equal ratios (protein amount). All tissues and cells were solubilized in buffer containing 4% SDS, 0.1 M dithiothreitol in 0.1 M Tris-HCl pH 7.6. FFPE samples were incubated for 1h at 95Â°C followed by sonication. Prior to protein digestion, proteins from each clinical sample were combined with equal protein amounts of the super-SILAC mix. The joint lysates were then trypsin-digested following the FASP protocol, on top of 30 kDa cutoff Amicon filters (37). To increase analytical depth, the resulting peptides were fractionated using pH-based strong anion exchange chromatography (SAX) in a StageTip format (37). Each sample was separated into six fractions using buffers of different pH values (11, 8, 6, 5, 4, 3) followed by desalting and concentration on C18 StageTips (38). In 25 cases, when the protein amount was insufficient, we performed in-solution trypsin digestion and/or did not fractionate the samples. Prior to MS analysis, peptides were eluted from the StageTips with 80% acetonitrile, vacuum-concentrated and diluted in MS loading buffer (2% acetonitrile, 0.1% formic acid). Peptides were separated by reverse-phase chromatography using the nano- ultra high performance liquid chromatography system (UHPLC) (Easy-nLC1000, Thermo Fisher Scientific), coupled to the Q-Exactive or Q-Exactive Plus mass spectrometers (Thermo Fisher Scientific). Peptides were separated with 220-240 min linear gradients of water/acetonitrile. MS resolution was 70,000 with an automatic gain control (AGC) target of 3e6, and MS/MS resolution was 17,500 with AGC target of 1e5. MS analysis was performed using a top10 method in which every high resolution MS scan is followed by fragmentation of the 10 most abundant peaks by higher-energy collisional dissociation (HCD).

### Data Protocol
MS raw files of all three datasets (Dataset-I (12), Dataset-II (14), and Dataset-III) were jointly analyzed by MaxQuant (39,40) version 1.5.3.16 and the integrated Andromeda search engine (41). MS/MS spectra were referenced to the Uniprot human proteome (http://www.uniprot.org/) released September 2015. False discovery rate (FDR) of 0.01 was used on both the peptide and protein levels using the forward-decoy approach (42). The search included methionine oxidation and protein N-terminal acetylation as variable modifications, and carbamidomethyl cysteine as a fixed modification. SILAC was employed for protein quantification, defined as two MS1 labels reflecting the SILAC heavy labels (Lysine8 and Arginine10). Maximum number of allowed missed peptide cleavages was set to 2, and the minimum peptide length was set to 7 amino-acids. An initial search followed by mass recalibration was performed with a mass tolerance of 20 ppm.

### Publication Abstract
Breast cancer classification has been the focus of numerous worldwide efforts, analyzing the molecular basis of breast cancer subtypes and aiming to associate them with clinical outcome and to improve the current diagnostic routine. Genomic and transcriptomic profiles of breast cancer have been well established, however the proteomic contribution to these profiles has yet to be elucidated. In this work, we utilized mass spectrometry-based proteomic analysis on more than 130 clinical breast samples to demonstrate intertumor heterogeneity across three breast cancer subtypes and healthy tissue. Unsupervised analysis identified four proteomic clusters, among them, one that represents a novel luminal subtype characterized by increased PI3K signaling. This subtype was further validated using an independent protein-based dataset, but not in two independent transcriptome cohorts. These results demonstrate the importance of deep proteomic analysis, which may affect cancer treatment decision making.<b>Significance:</b> These findings utilize extensive proteomics to identify a novel luminal breast cancer subtype, highlighting the added value of clinical proteomics in breast cancer to identify unique features not observable by genomic approaches. <i>Cancer Res; 78(20); 6001-10. &#xa9;2018 AACR</i>.

### Keywords
Lc-ms, Breast cancer classification, Breast cancer subtypes, Proteomics

### Affiliations
Weizmann Institute of Science
Department of Human Molecular Genetics and Biochemistry, Sackler Faculty of Medicine, Tel Aviv University

### Submitter
Tamar Geiger

### Lab Head
Dr Tamar Geiger
Department of Human Molecular Genetics and Biochemistry, Sackler Faculty of Medicine, Tel Aviv University


