### Accession
PXD000504

### Title
Campylobacter concisus strain BAA-1547 GeLC-MSMS

### Description
Bacteria were washed three times in NaCl (150 mM).  Cells were resuspended in 1 ml TSU buffer (50 mM Tris pH 8.0, 0.1% SDS, 2.5 M urea) and lysed by two freeze-thaw cycles in liquid nitrogen. Lysate proteins (40 ug) were separated and digested.  Digested peptides were separated by nano-LC using an Ultimate 3000 HPLC and autosampler system (Dionex; Amsterdam, Netherlands).  Samples (1 ul) were concentrated and desalted onto a micro C18 pre-column (500 um×2 mm, Michrom Bioresources; Auburn, CA, USA) with H2O:CH3CN (98:2, 0.05% trifluoroacetic acid) at 15 ul min−1.  After a 4 min wash the pre-column was switched (Valco 10 port valve; Dionex) into line with a fritless nano column (75 u×~10 cm) containing C18 media (5 u, 200 A Magic; Michrom) manufactured according to Gatlin.  Peptides were eluted using a linear gradient of H2O:CH3CN (98:2, 0.1% formic acid) to H2O:CH3CN (64:36, 0.1% formic acid) at 250 nl min−1 over 30 min.  High voltage (2000 V) was applied to low volume tee (Upchurch Scientific) and the column tip positioned ~0.5 cm from the heated capillary (T = 280 degrees C) of an Orbitrap Velos (Thermo Electron; Bremen, Germany) mass spectrometer.  Positive ions were generated by electrospray and the Orbitrap operated in data dependent acquisition mode (DDA).  A survey scan m/z 350–1750 was acquired in the Orbitrap (Resolution = 30,000 at m/z 400, with an accumulation target value of 1,000,000 ions) with lockmass enabled. Up to the 10 most abundant ions (>5,000 counts) with charge states >+2 were sequentially isolated and fragmented within the linear ion trap using collisionally induced dissociation with an activation q = 0.25 and activation time of 30 ms at a target value of 30,000 ions.  M/z ratios selected for MS/MS were dynamically excluded for 30 s.  Peak lists were generated using Mascot Daemon/extract_msn (Matrix Science, Thermo; London, England) using the default parameters, and submitted to the database search program Mascot (version 2.1, Matrix Science).  Search parameters were:   Precursor tolerance 4 ppm and product ion tolerances +/-0.4 Da;   Oxidation (M) and Carbamidomethyl (C) specified as variable modifications,   Enzyme specificity was trypsin,   1 missed cleavage was possible

### Sample Protocol
Not available

### Data Protocol
Not available

### Publication Abstract
Direct links between proteomic and genomic/transcriptomic data are not frequently made, partly because of lack of appropriate bioinformatics tools. To help address this, we have developed the PG Nexus pipeline. The PG Nexus allows users to covisualize peptides in the context of genomes or genomic contigs, along with RNA-seq reads. This is done in the Integrated Genome Viewer (IGV). A Results Analyzer reports the precise base position where LC-MS/MS-derived peptides cover genes or gene isoforms, on the chromosomes or contigs where this occurs. In prokaryotes, the PG Nexus pipeline facilitates the validation of genes, where annotation or gene prediction is available, or the discovery of genes using a "virtual protein"-based unbiased approach. We illustrate this with a comprehensive proteogenomics analysis of two strains of Campylobacter concisus . For higher eukaryotes, the PG Nexus facilitates gene validation and supports the identification of mRNA splice junction boundaries and splice variants that are protein-coding. This is illustrated with an analysis of splice junctions covered by human phosphopeptides, and other examples of relevance to the Chromosome-Centric Human Proteome Project. The PG Nexus is open-source and available from https://github.com/IntersectAustralia/ap11_Samifier. It has been integrated into Galaxy and made available in the Galaxy tool shed.

### Keywords
Concisus, Gelc-msms, Campylobacter

### Affiliations
School of Biotechnology and Biomolecular Sciences

### Submitter
A Tay

### Lab Head


