### Accession
PXD005060

### Title
Protein expression difference in Drosophila speciation

### Description
We aim to determine the difference of protein expression in the closely related Drosophila species D.melanogaster and D.simulans to gain further insight into protein expression profiles of both species. Analysis of the protein levels might hint to pathways that have evolved into different direction and explain the separation of th species from their common ancestors. Further, we try to characterize novel gene isoform by combining RNA sequencing and proteomics.

### Sample Protocol
Proteins were extracted from D.melanogaster or D.simulans heads, precipitated with TCA to remove the biological background and digested with trypsin. After desalting of the peptide mixtures, LC-MS/MS analysis was conducted on a Sciex 5600 TripleTOF mass spectrometer coupled to an Ultimate 3000 UHPLC system for reversed phase-C18 prefractionation of the peptide mixtures. Two different approaches were used first the complete proteomes were analyzed and in the second approach a prefractionation into 5 fractions was performed. The proteomes were separated employing a linear gradient from 4-36 % Acetonitril in 145 min. Each fraction of the second sample was separated using a linear gradient from 4-36% ACN in 120 min. Over the whole program time, mass spectra were acquired in data-dependent mode with positive ionization. For the complete proteomes, a duty cycle of 1 survey scan and up to 30 MS/MS scans was programmed, enabling peptide fragmentation when the precursor has a charge state between 2+ and 5+ and signal intensity of more than 100 cps. The surevy scan mass range was from 300 to 1250 m/z, MS/MS spectra were acquired from 150-1800 m/z. Optimized rolling collision energies were employed for peptide fractionation. Previously selected precursor ions were exclude from reanalysis for 15 seconds in a mass window of 50 ppm. For proteomic ananlysis of the 5 peptide fractions, the selection parameters for peptide precursors were modified to max. charge state of 4+ and minimal intesnity of 150 cps. All other parameters were identical.

### Data Protocol
All raw data files were searched with Andromeda within the MaxQuant 1.5.1.2 software suite against a combined, forward/reversed drosophila database downloaded from flybase (vs. 5.24 for D.melanogaster and D. simulans as well as predicted sequences from RNA-Seq results). Mass difference for precursors were 30 ppm for the first dearch and 5 ppm for the second search step. MS/MS mass deviation was set to 40 ppm. Putative post-translational modifications were carbamidomethylation of cysteine (fixed), oxidation of methionine and acetylation of the protein N-terminus (both variable). A minimal Andromeda score of 40 was set for modified peptides. Peptide spectrum matches and proteins were filtered for 1% false positives.

### Publication Abstract
Spectral libraries play a central role in the analysis of data-independent-acquisition (DIA) proteomics experiments. A main assumption in current spectral library tools is that a single characteristic intensity pattern (CIP) suffices to describe the fragmentation of a peptide in a particular charge state (peptide charge pair). However, we find that this is often not the case. We carry out a systematic evaluation of spectral variability over public repositories and in-house data sets. We show that spectral variability is widespread and partly occurs under fixed experimental conditions. Using clustering of preprocessed spectra, we derive a limited number of multiple characteristic intensity patterns (MCIPs) for each peptide charge pair, which allow almost complete coverage of our heterogeneous data set without affecting the false discovery rate. We show that a MCIP library derived from public repositories performs in most cases similar to a "custom-made" spectral library, which has been acquired under identical experimental conditions as the query spectra. We apply the MCIP approach to a DIA data set and observe a significant increase in peptide recognition. We propose the MCIP approach as an easy-to-implement addition to current spectral library search engines and as a new way to utilize the data stored in spectral repositories.

### Keywords
Quantitative proteomics, Shotgun proteomics, Novel slice variants, Drosophila speciation

### Affiliations
CECAD, University of Cologne, AG Kr√ºger
Protein Analysis Unit, ZfP LMU Munich BioMedical Center Grosshaderner Strasse 9 82152 Planegg-Martinsried Tel. +49 89 2180 71804

### Submitter
Andreas Schmidt

### Lab Head
Dr Axel Imhof
Protein Analysis Unit, ZfP LMU Munich BioMedical Center Grosshaderner Strasse 9 82152 Planegg-Martinsried Tel. +49 89 2180 71804


