### Accession
PXD031071

### Title
Parallelized multidimensional analytic framework applied to mammalian cells uncovers novel regulatory principles in EMT

### Description
We combine state-of-the-art data acquisition platforms and bioinformatics tools to devise PAMAF, a workflow that simultaneously examines twelve omics modalities, i.e., protein abundance from whole-cells, nucleus, exosomes, secretome and membrane; N-glycosylation, phosphorylation; metabolites; mRNA, miRNA; and, in parallel, single-cell transcriptomes. Here we apply PAMAF in an established in vitro model of TGFβ-induced epithelial to mesenchymal transition (EMT) to quantify >61,000 molecules from 12 omics and 10 timepoints over 12 days. Bioinformatics analysis of this EMT-ExMap resource allowed us to identify; –unexpected topological coupling between omics, –four distinct cell states during EMT (E, E/M-1, E/M-2, M), –omics-specific kinetic paths, –stage-specific multi-omics characteristics, –distinct regulatory classes of genes, –ligand–receptor mediated intercellular crosstalk using an innovative pipeline integrating scRNAseq and subcellular proteomics, and –novel combinatorial drug targets (e.g., Hedgehog signaling and CAMK-II) to inhibit EMT, which we validate using a 3D mammary duct-on-a-chip platform. Overall, while this study provides an unprecedented resource on TGFβ signaling and EMT, PAMAF-like workflows can be applied to generate comprehensive molecular landscapes of other multifaceted biological processes.

### Sample Protocol
Sample pellets were solubilized in 3× volume of GuHCl lysis buffer (6 M Guanidine-HCl, 100 mM Tris-Cl pH 8.5, 10 mM TCEP, 40 mM CAA, 6 mM CaCl2) and heated for 5 minutes at 75°C. Lysates were cooled on ice for 10 minutes, sonicated (Branson probe sonifier, 10% power, 3× 20 secs On, 10 secs Off cycles), and heated again at 75°C for 5 minutes, followed by centrifugation for 30 minutes at 3,500×g and 4°C. Cleared lysates were transferred to fresh tubes, diluted 8× with 100 mM Tris-Cl pH 8.5 and protein concentration was determined using BCA assay (Thermo Scientific). Equal amounts of proteins were digested overnight with MS-grade Trypsin (Thermo Scientific) with 1:50 (protein: enzyme) ratio at 37°C with agitation (900 rpm on an Eppendorf thermomixer).  Digested peptides were acidified with 10% TFA (final 1% v/v), vacufuged to remove TFE and desalted using Sep-Pak (Waters Corp).

### Data Protocol
Raw files were processed using MaxQuant using default/standard parameters.

### Publication Abstract
None

### Keywords
Mcf10a, Proteomics, Multiomics, Tgfbeta, Emt, Mass spectrometry

### Affiliations
Boston University
Director, Center for Network Systems Biology Departments of Biochemistry & Biology Boston University,  Boston, MA, USA

### Submitter
Indranil Paul

### Lab Head
Dr Andrew Emili
Director, Center for Network Systems Biology Departments of Biochemistry & Biology Boston University,  Boston, MA, USA


