### Accession
PXD030370

### Title
Dissecting post-translational modifications in Leptospira proteome to elucidate novel insights into pathogen-host interaction using bioinformatics approach

### Description
Leptospirosis is a zoonotic disease caused by Leptospira spp. that cause serious health burden. Humans are accidental host gets infected from water contaminated with Leptospira from urine of reservoir hosts like rodents and other mammals. Currently there is no effective vaccine available for leptospirosis. Investigations on post-translational modifications (PTMs) provide insights into the bacterial pathophysiology and functional consequences during bacterial infection and that can provide significant contribution in the search for better vaccine candidates. PTMs are decisive factors in the structure, function, and localization of proteins in both prokaryotic and eukaryotic organisms.  In our previous investigations, we have identified PXD016204 (Thoduvayil et al., 2020) and PXD026044 (Sarma et al., 2021) from Leptospira interrogans serovar Copenhageni (strain Fiocruz L1-130). In this project, a systematic reanalysis of unassigned spectra were carried out from our previous total proteome data (PXD016204 and PXD026044) for multi-PTM analysis. We have identified a total of 3,693 unique high-confidence PTM sites corresponding to 1,266 proteins (PTM-Pro probability cut-off value is â‰¥75%).  There were two objectives for the study. The first objective was the functional characterization of PTM proteins, through the identification of protein-protein interactions (PPI). This will enable us to understand the type and functions of PTMs. This analysis was supported by additional characteristics like gene ontology (GO), cluster of autologous groups (COG), other protein features and functions. The second objective was to find the relationship of PTMs with the subcellular localization of leptospiral proteins. We have analyzed the PTM containing proteins identified in Triton X-114 fractions and used the iBAQ values and number of PTM containing peptides that correlate with the sub-cellular localization of leptospiral proteins.

### Sample Protocol
Triton X-114 protein fractionation: The protocol was same used earlier for PXD016204 (Thoduvayil et al., 2020) and PXD026044 (Sarma et al., 2021), briefly, the leptospires were pelleted and the supernatant media was used to enrich proteins to use as secretory proteins. The pellets were washed using phosphate-buffered saline containing 5 mm MgCl2 and the proteins found in the wash were considered washable surface proteins. The washed Leptospira pellet was used for Triton X-114-based fractionation separate leptospiral proteins to aqueous,detergent, and pellet fractions that represent cytoplasmic, outer membrane (OM), and inner membrane proteins, respectively. All five fractions were used for  LC-MS/MS.

### Data Protocol
The MS/MS raw files retrieved from PXD016204 and PXD026044 datasets were searched against the Leptospira interrogans serogroup Icterohaemorrhagiae serovar Copenhageni (strain Fiocruz L1-130) protein database downloaded from NCBI RefSeq. The commonly encountered contaminants were added to this database and the searches were performed using SEQUEST-HT and MASCOT through Proteome Discoverer (Version 2.2) software suite (Thermo Scientific, Bremen, Germany). Search parameters included trypsin as an enzyme with one missed cleavage. Oxidation (+15.995 Da) of methionine (M) and acetylation (+42.011 Da) of protein N-terminal were set as dynamic modifications. Carbamidomethylation (+57.021 Da) of cysteine (C) was set as a static modification for total proteome analysis. Unassigned MS/MS spectra obtained from total proteome analysis were subjected to further search against the reference database along with specific PTMs (Rex et al., 2021; Deolankar et al., 2019; Pathan et al., 2017). The PTMs such as citrullination (+0.984 Da) of arginine (R); phosphorylation (+79.966 Da) of serine (S), threonine (T), and tyrosine (Y); palmitoylation (+238.230 Da) of cysteine (C); acetylation (+42.011 Da of lysine (K), gluratylation (+114.032 Da) of lysine (K) were set as dynamic modifications. Carbamidomethylation (+57.021 Da) of cysteine (C) was set as a static modification. The mass error for precursor ion and fragment ions was set as 10 ppm and 0.05 Da, respectively. ptmRS node was used to determine the probability of site modification (Taus et al., 2011). The PSMs and peptides that qualified 1% FDR were considered for further analysis. A parsimonious protein list was generated from the peptide list by grouping proteins in Proteome Discoverer (Version 2.2).

### Publication Abstract
Posttranslational modifications (PTMs) are decisive factors in the structure, function, and localization of proteins in prokaryotic and eukaryotic organisms. However, prokaryotic organisms lack subcellular organelles, and protein localization based on subcellular locations like cytoplasm, inner membrane, periplasm, and outer membrane can be accounted for functional characterization. We have identified 131 acetylated, 1182 citrullinated, 72 glutarylated, 5 palmitoylated, and 139 phosphorylated proteins from Triton X-114 fractionated proteins of <i>Leptospira</i>, the pathogen of re-emerging zoonotic disease leptospirosis. In total, 74.7% of proteins were found exclusively in different Triton X-114 fractions. Additionally, 21.9% of proteins in multiple fractions had one or more PTM specific to different Triton X-114 fractions. Altogether, 96.6% of proteins showed exclusiveness to different Triton X-114 fractions either due to the presence of the entire protein or with a specific PTM type or position. Further, the PTM distribution within Triton X-114 fractions showed higher acetylation in aqueous, glutarylation in detergent, phosphorylation in pellet, and citrullination in wash fractions representing cytoplasmic, outer membrane, inner membrane, and extracellular locations, respectively. Identification of PTMs in proteins with respect to the subcellular localization will help to characterize candidate proteins before developing novel drugs and vaccines rationally to combat leptospirosis.

### Keywords
Leptospira; protein modification; drug target; diagnosis; mass spectrometry; proteome discoverer

### Affiliations
Dr Madathiparambil G. Madanan,  Scientist E,  Indian Council of Medical Research, Regional Medical Research Centre,  Dollygunj, Port Blair - 744 103,  Andaman & Nicobar Islands,INDIA.  Phone: Office: +91-3192-251158  Cell: +91-9434284576 (lab head)
Indian Council of Medical Research, Regional Medical Research Centre

### Submitter
Madathiparambil G.  Madanan

### Lab Head
Dr Madathiparambil G. Madanan
Dr Madathiparambil G. Madanan,  Scientist E,  Indian Council of Medical Research, Regional Medical Research Centre,  Dollygunj, Port Blair - 744 103,  Andaman & Nicobar Islands,INDIA.  Phone: Office: +91-3192-251158  Cell: +91-9434284576 (lab head)


