### Accession
PXD016925

### Title
Sarcoptes scabiei ES proteomics

### Description
Background: The parasitic mite Sarcoptes scabiei is an economically highly significant parasite of the skin of humans and animals worldwide. This mite causes a neglected tropical disease (NTD), called scabies - one of the commonest dermatological problems globally, resulting in major morbidity, disability, stigma and poverty. In hyperendemic situations, scabies is often associated with secondary opportunistic bacterial infections/diseases - a major concern in children. Although some stages of this mite can be treated with drugs, resistance against some drugs is emerging, and there is no vaccine available against scabies. Here, we report molecular resources (including a high-quality genome as well as transcriptomic and proteomic data sets) for S. scabiei to aid basic and applied research of this and related mites.

### Sample Protocol
Production and procurement of S. scabiei. Animal ethics approval was granted by the Queensland Institute of Medical Research ��������� Berghofer (permit nos. P630 and P2159) and the Ethics Committee of the Queensland Animal Science Precinct (permit SA 2015/03/504). S. scabiei was produced on pigs (3 months of age) maintained under (otherwise) parasite-free conditions using an established protocol (Mounsey et al. 2010). Egg and mixed motile stages (~ 1,000 of each stage per sample) were isolated, snap frozen and stored dry at -70 ������C. Faecal samples (n = 5) were collected from five populations of adult female S. scabiei (1,000 per sample) from distinct pigs; crude protein extracts were prepared (Wang et al. 2019b), freeze-dried and resuspended in 200 ������l 8M urea in 100 mM triethylammonium bicarbonate (pH 8.5) with protease inhibitor cocktail set I (Merck, Denmark).  Liquid chromatography/tandem mass spectrometry (LC-MS/MS) analysis. The proteome of faecal matter from S. scabiei was investigated using an established in-solution digestion protocol (Ang et al. 2011). In brief, 5 replicate samples (50 ������g of protein each) were reduced, alkylated and double-digested with Lys-C/trypsin mix (Promega, USA) at 37 ������C for 16 h. The tryptic samples were then acidified with 1.0% (v/v) formic acid and purified using Oasis HLB cartridges (Waters, USA). Using an established technique (Wang et al. 2019a), tryptic peptides were analysed using a Q Exactive Plus Orbitrap mass spectrometer (Thermo Fisher, USA).

### Data Protocol
ES proteins and allergens. Excretory proteins were inferred from LC-MS/MS data against the proteome predicted for S. scabiei. First, raw LC-MS/MS data were processed using the program MaxQuant (Tyanova et al. 2016) with parameters 20 ppm for a precursor tolerance, 0.05 Da for MS/MS tolerance, and fixed modifications of carbamidomethylation of cysteine (+57 Da) and methionine oxidation (+16 Da). Then, identified peptides were collected using false discovery rate (FDR; < 0.01) at both the peptide and protein levels. Identified proteins were then quantified using the label-free quantification (LFQ) value from MaxQuant employing default settings and, finally, the proteins identified with ��������� 2 peptides were accepted.  Secreted proteins were predicted using the programs SignalP 4.0 (Petersen et al. 2011) and MultiLoc2 (Blum et al. 2009). To classify a secreted protein, a predicted signal peptide and predicted extracellular location were required. Allergens were identified using BLASTp v2.2.30+ searches (E-value 1e-8) against NCBI protein nr, the allergens identified for S. scabiei var. canis (Rider et al. 2015), and the known allergens of Dermatophagoides farina (Pom������s et al. 2018)  and D. pteronyssinus (Pom������s et al. 2018); and gene models of identified allergens were manually curated using available transcriptomic data.

### Publication Abstract
None

### Keywords
I es proteomics, Lc-ms/ms, Es

### Affiliations
The University of Melbourne
Faculty of Veterinary and Agricultural Sciences, The University of Melbourne, Parkville, Victoria, Australia

### Submitter
Tao Wang

### Lab Head
Dr Robin Beat Gasser
Faculty of Veterinary and Agricultural Sciences, The University of Melbourne, Parkville, Victoria, Australia


