### Accession
PXD023217

### Title
Critical Assessment of Metaproteome Investigation (CAMPI): a Multi-Lab Comparison of Established Workflows

### Description
Metaproteomics, the study of the collective proteome within a microbial ecosystem, has substantially grown over the past few years. This growth comes from the increased awareness that it can powerfully supplement metagenomics and metatranscriptomics analyses. Although metaproteomics is more challenging than single-species proteomics, its added value has already been demonstrated in various biosystems, such as gut microbiomes or biogas plants.  Because of the many challenges, a variety of metaproteomics workflows have been developed, yet it remains unclear what the impact of the choice of workflow is on the obtained results. Therefore, we set out to compare several well-established workflows in the first community-driven, multi-lab comparison in metaproteomics: the critical assessment of metaproteome investigation (CAMPI) study. In this benchmarking study, we evaluated the influence of different workflows on sample preparation, mass spectrometry acquisition, and bioinformatic analysis on two samples: a simplified, lab-assembled human intestinal sample and a complex human fecal sample. We find that the same overall biological meaning can be inferred from the metaproteome data, regardless of the chosen workflow. Indeed, taxonomic and functional annotations were very similar across all sample-specific data sets. Moreover, this outcome was consistent regardless of whether protein groups or peptides, or differences at the spectrum or peptide level were used to infer these annotations. Where differences were observed, those originated primarily from different wet-lab methods rather than from different bioinformatic pipelines. The CAMPI study thus provides a solid foundation for benchmarking metaproteomics workflows, and will therefore be a key reference for future method improvement.

### Sample Protocol
Each laboratory used their own LC-MS/MS protocol with the largest differences and similarities highlighted in the following and details provided in Supplementary_Table_1. For LC, all laboratories separated peptides using reversed-phase chromatography with a linear gradient length ranging from 60 min to 460 min. Furthermore, one group performed an additional separation using a multidimensional protein identification technology (MudPIT) combining cation exchange and reversed-phase separation in a single column prepared in-house (Sch√§pe et al., 2019; Wolters et al., 2001). Six groups used an Orbitrap mass spectrometer (4x Q Exactive HF, 1x Q Exactive Plus, 1x Fusion Lumos, Thermo Fisher Scientific), while two groups employed a timsTOF mass spectrometer (Bruker Daltonik). All participants used data-dependent acquisition (DDA) with exclusion duration times ranging from 10s to 60s

### Data Protocol
All submitted MS/MS raw files were first analyzed with one commonly used database search method to assess both the quality of the extraction and the MS/MS acquisition, as well as the effect of the search database composition (reference proteomes vs. multi-omics). The database search engine X!Tandem (Craig & Beavis, 2004) was used with a defined set of parameters as follows: specific trypsin digest with a maximum of 2 missed cleavages; 10.0 ppm for MS1 and 0.02 Da for MS2 tolerance, respectively; fixed modifications: Carbamidomethylation of C (+57.021464 Da); variable modifications: Oxidation of M (+15.994915 Da); fixed modifications during refinement procedure: Carbamidomethylation of C (+57.021464 Da). Peptides were filtered on length (between 6 and 50 amino acids), and charge state (+2, +3 and +4) and a maximum valid expected value (e-value) of 0.1(Gupta et al., 2011).

### Publication Abstract
Metaproteomics has matured into a powerful tool to assess functional interactions in microbial communities. While many metaproteomic workflows are available, the impact of method choice on results remains unclear. Here, we carry out a community-driven, multi-laboratory comparison in metaproteomics: the critical assessment of metaproteome investigation study (CAMPI). Based on well-established workflows, we evaluate the effect of sample preparation, mass spectrometry, and bioinformatic analysis using two samples: a simplified, laboratory-assembled human intestinal model and a human fecal sample. We observe that variability at the peptide level is predominantly due to sample processing workflows, with a smaller contribution of bioinformatic pipelines. These peptide-level differences largely disappear at the protein group level. While differences are observed for predicted community composition, similar functional profiles are obtained across workflows. CAMPI demonstrates the robustness of present-day metaproteomics research, serves as a template for multi-laboratory studies in metaproteomics, and provides publicly available data sets for benchmarking future developments.

### Keywords
Metaproteomics, Microbial communities, Multi-omics

### Affiliations
Otto-von-Guericke University Magdeburg
Section eScience (S.3), Federal Institute for Materials Research and Testing, Berlin, Germany Bioinformatics Unit (MF1), Department for Methods Development and Research Infrastructure, Robert Koch Institute, Berlin, Germany

### Submitter
Kay Schallert

### Lab Head
Dr Thilo Muth
Section eScience (S.3), Federal Institute for Materials Research and Testing, Berlin, Germany Bioinformatics Unit (MF1), Department for Methods Development and Research Infrastructure, Robert Koch Institute, Berlin, Germany


