### Accession
PXD039306

### Title
Evolutionary and cellular analysis of the'dark'pseudokinase PSKH2

### Description
Pseudokinases, so named because they lack one or more conserved canonical amino acids that define their catalytically-active relatives, have evolved a variety of biological functions in both prokaryotic and eukaryotic organisms. Human PSKH2 is closely related to the canonical kinase PSKH1, which maps to the CAMK family of protein kinases. Primates encode PSKH2 in the form of a pseudokinase, which is predicted to be catalytically inactive due to loss of the invariant catalytic Asp residue. Although the biological role(s) of vertebrate PSKH2 proteins remains unclear, we previously identified species-level adaptions in PSKH2 that have led to the appearance of kinase or pseudokinase variants in vertebrate genomes alongside a canonical PSKH1 paralog. In this paper we confirm that, as predicted, PSKH2 lacks detectable protein phosphotransferase activity, and exploit structural informatics, biochemistry and cellular proteomics to begin to characterise vertebrate PSKH2 orthologues. AlphaFold 2-based structural analysis predicts functional roles for both the PSKH2 N- and Cregions that flank the pseudokinase domain core, and cellular truncation analysis confirms that the N-terminal domain, which contains a conserved myristoylation site, is required for both stable human PSKH2 expression and localisation to a membrane-rich subcellular fraction containing mitochondrial proteins. Using mass spectrometry-based proteomics, we confirm that human PSKH2 is part of a cellular mitochondrial protein network, and that its expression is regulated through client-status within the HSP90/Cdc37 molecular chaperone system. HSP90 interactions are mediated through binding to the PSKH2 C-terminal tail, leading us to predict that this region might act as both a cis and trans regulatory element, driving outputs linked to the PSKH2 pseudokinase domain that are important for functional signalling.

### Sample Protocol
HEK-293T cells were cultured in Dulbecco’s modified Eagle medium (Lonza) supplemented with 10% fetal bovine serum (HyClone), penicillin (50 U/ml), and streptomycin (0.25 μg/ml) (Lonza) and maintained at 37°C in 5% CO 2 humidified atmosphere. All exogenously expressed proteins were cloned into a pcDNA3 vector and expressed in frame with a 3C protease cleavable, N- or C-terminal Myc, FLAG or tandem STREP tag. HEK-293T cells were transfected using a 3:1 polyethylenimine (PEI [branched average Mw ~25,000 Da; Sigma-Aldrich]) to DNA ratio (30:10 μg, for a single 10-cm culture dish). For immunoprecipitation experiments, proteins were harvested 48 h post transfection in a lysis buffer containing 50 mM Tris-HCl (pH 7.4), 150 mM NaCl, 0.1% (v/v) Triton X-100, 1 mM DTT, 1% (w/v) dodecyl- -D-Maltoside (DDM), 0.1 mM ethylenediaminetetraacetic acid (EDTA), 0.1 mM ethylene glycol-bis(β-aminoethyl ether)- N , N , N′, N′-tetraacetic acid (EGTA) and 5% (v/v) glycerol and supplemented with a protease inhibitor cocktail tablet and a phosphatase inhibitor tablet (Roche). Lysates were briefly sonicated on ice and clarified by centrifuged at 20,817 x g for 20 min at 4°C, and the resulting supernatants were incubated with Strep-TactinXT 4Flow (IBA) for 3 hours with gentle agitation at 4°C. Affinity beads containing bound protein were collected and washed three times in 50 mM Tris-HCl (pH 7.4) and 150 mM NaCl and then equilibrated in storage buffer (50 mM Tris-HCl [pH 7.4], 100 mM NaCl, 1 mM DTT, 1% (w/v) DDM and 5% (v/v) glycerol). The purified proteins were then eluted from the suspended beads over a 1-hour period with 3C protease (0.5 μg) at 4°C, with gentle agitation. Affinity-capture Mass spectrometry Affinity captured PSKH2-STREP samples were diluted 10-fold in 25 mM ammonium bicarbonate (pH 8.0), reduced with dithiothreitol and alkylated with iodoacetamide, 0.2 μg trypsin gold (Promega) was added and incubating at 37°C with gentle agitation for 18 h. Digests were then subjected to strong cation exchange using in-house packed stage tips. Dried peptides were solubilized in 20 μl of 3% (v/v) acetonitrile and 0.1% (v/v) TFA in water, sonicated for 10 minutes, and centrifuged at 13,000 x g for 15 min at 4 °C prior to reversed-phase HPLC separation using an Ultimate3000 nano system (Dionex) over a 60-minute gradient. All data acquisition was performed using a Thermo Orbitrap Fusion Lumos Tribrid mass spectrometer (Thermo Scientific), with higher-energy C-trap dissociation (HCD) fragmentation set at 32% normalized collision energy for 2+ to 5+ charge states. MS1 spectra were acquired in the Orbitrap (60K resolution at 200 m/z) over a range of 350 to 1400 m/z; AGC target = standard, maximum injection time = auto, with an intensity threshold for fragmentation of 2e 4 . MS2 spectra were acquired in the Iontrap set to rapid mode (15K resolution at 200 m/z), maximum injection time = 50 ms with a 1 min dynamic exclusion window applied with a 0.5 Da tolerance.

### Data Protocol
For binding partner analysis, data was searched twice (search settings were identical, with the addition of low abundance resampling imputation of missing values in the 2nd search) using Proteome Discoverer 2.4; searching the UniProt Human Reviewed database (updated weekly) with fixed modification = carbamidomethylation (C), variable modifications = oxidation (M), instrument type = electrospray ionization–Fourier-transform ion cyclotron resonance (ESI-FTICR), MS1 mass tolerance = 10 ppm, MS2 mass tolerance = 0.5 Da. Percolator and precursor ion quantifier nodes (Hi3 Label free Quantification (LFQ)) were both enabled. All data was filtered to a 1% False discovery rate. Data from the 1st search was put into a custom R script that extracted all protein accessions with LFQ data in at least 2 of 3 replicates of a condition which was used to filter the imputation containing dataset. Accessions were used to obtain the gene name using the UniProt ID retrieval tool. Fold changes and T-tests were calculated and log2 transformed, before importing into a custom R script for plotting. For PSKH2 tag orientation plotting, LFQ data was normalized to the level of PSKH2 for that replicate prior to calculations. Data was additionally analyzed through PEAKS Studio (version XPro) using the same database, mass tolerances and modifications. PEAKS specific search settings: instrument = Orbi-Trap, Fragmentation = HCD, acquisition = DDA, De Novo details = standard and a maximum of 5 variable PTMs possible. PEAKS PTM mode was enabled and filtering parameters of De Novo score >15, - log10P(value) >30.0, Ascore >30.0 and seen in at least 2 of 3 replicates were applied for a PTM to be maintained.

### Publication Abstract
Pseudokinases, so named because they lack one or more conserved canonical amino acids that define their catalytically active relatives, have evolved a variety of biological functions in both prokaryotic and eukaryotic organisms. Human PSKH2 is closely related to the canonical kinase PSKH1, which maps to the CAMK family of protein kinases. Primates encode PSKH2 in the form of a pseudokinase, which is predicted to be catalytically inactive due to loss of the invariant catalytic Asp residue. Although the biological role(s) of vertebrate PSKH2 proteins remains unclear, we previously identified species-level adaptions in PSKH2 that have led to the appearance of kinase or pseudokinase variants in vertebrate genomes alongside a canonical PSKH1 paralog. In this paper we confirm that, as predicted, PSKH2 lacks detectable protein phosphotransferase activity, and exploit structural informatics, biochemistry and cellular proteomics to begin to characterise vertebrate PSKH2 orthologues. AlphaFold 2-based structural analysis predicts functional roles for both the PSKH2 N- and C-regions that flank the pseudokinase domain core, and cellular truncation analysis confirms that the N-terminal domain, which contains a conserved myristoylation site, is required for both stable human PSKH2 expression and localisation to a membrane-rich subcellular fraction containing mitochondrial proteins. Using mass spectrometry-based proteomics, we confirm that human PSKH2 is part of a cellular mitochondrial protein network, and that its expression is regulated through client-status within the HSP90/Cdc37 molecular chaperone system. HSP90 interactions are mediated through binding to the PSKH2 C-terminal tail, leading us to predict that this region might act as both a cis and trans regulatory element, driving outputs linked to the PSKH2 pseudokinase domain that are important for functional signalling.

### Keywords
Pseudokinase, Pskh2

### Affiliations
University of Liverpool
Centre for Proteome Research, Biochemistry and Systems Biology Department, University of Liverpool

### Submitter
leonard daly

### Lab Head
Dr Claire E Eyers
Centre for Proteome Research, Biochemistry and Systems Biology Department, University of Liverpool


