### Accession
PXD032294

### Title
Profiling yeast ubiquitylome with OtUBD pulldowns

### Description
This project includes two separate experiments where the ubiquitylated proteome of the budding yeast Saccharomyces cerevisiae was enriched from whole cell lysate by affinity purification using a novel OtUBD affinity resin. In the first experiment, two distinct experimental conditions (native, denaturing) were used to compare the proteins enriched by OtUBD affinity resin and the negative control resin. In the second experiment, the ubiquitylation profile of proteins were compared among wildtype yeast cells and yeasts lacking certain E3 ligase enzymes.

### Sample Protocol
Yeast grown to log phase was lysed by grinding in liquid nitrogen. Cleared yeast lysate, supplemented with protease inhibitors and the DUB inhibitor NEM, was subjected to no treatment or denaturation with 8M urea. The native or denatured lysates were diluted to 2 â€“ 4 mg/mL and incubated with OtUBD resin or the control resin for 2.5 hours. The resins were then extensively washed and eluted with pH 2.5 glycine buffer. Eluted proteins were immediately neutralized with tris buffer (pH9) and lyophilized. The lyophilized pellets were cleaned up with methanol/chloroform extraction. Protein pellets were redissolved, reduced with dithiothreitol and alkylated with methanethiosulfonate (MMTS) followed by digestion with trypsin. Digested peptides were desalted. For more details, please refer to the preprint (DOI: 10.1101/2021.12.02.470885) or related article.

### Data Protocol
Data was analyzed using Proteome Discoverer software v2.2 (Thermo Scientific). Data searching is performed using the Mascot algorithm (version 2.6.1) (Matrix Science) against a custom database containing protein sequences for OtUBD as well as the SwissProt database with taxonomy restricted to Saccharomyces cerevisiae (7,907 sequences). The search parameters included tryptic digestion with up to 2 missed cleavages, 10 ppm precursor mass tolerance and 0.02 Da fragment mass tolerance, and variable (dynamic) modifications of methionine oxidation; N-ethylmaleimide, N-ethylmaleimide+water, carbamidomethyl, or methylthio on cysteine; and GG adduct on lysine, protein N-terminus, serine, threonine or cysteine. Normal and decoy database searches were run, with the confidence level set to 95% (p<0.05).  Scaffold (version Scaffold_5.0, Proteome Software Inc., Portland, OR) was used to validate MS/MS based peptide and protein identifications. Peptide identifications were accepted if they could be established at greater than 95.0% probability by the Scaffold Local FDR algorithm. Protein identifications were accepted if they could be established at greater than 99.0% probability and contained at least 2 identified peptides.

### Publication Abstract
None

### Keywords
Lc-ms, Ubiquitin, Ubd, Saccharomyces cerevisiae

### Affiliations
Yale University

### Submitter
Mengwen Zhang

### Lab Head
Dr Mark Hochstrasser
Yale University


