### Accession
PXD026716

### Title
Defining the RBPome of T helper cells to study higher order post-transcriptional gene 1 regulation

### Description
Post-transcriptional gene regulation in T cells is dynamic and complex as targeted transcripts respond to various factors. This becomes evident for the Icos mRNA encoding an essential costimulatory receptorthatexhibits coordinate regulation by several RNA-binding proteins (RBPs), including Roquin-1 and Roquin-2. Utilizing global RNA interactome capture (RNA-IC) and orthogonal organic phase separation (OOPS) we identify a core RBPome of 798 mouse and 801 human T cell proteins. The RBPome also contained Stat1, Stat4 and Vav1 proteins suggesting unexpected functions for these transcription factors and signal transducers. Based on proximity to Roquin-1, we selected ~50 RBPs for testing coregulation of Roquin-1/2 targets by induced expression in wild-type or Roquin-1/2-deficient T cells. Besides Roquin-independent contributions from Rbms1 and Cpeb4 we also unraveled Roquin-1/2-dependent and target-specific coregulation of Icos byCelf1 and Igf2bp3. These findings define the cellular RBPome as post-transcriptional context that predetermines the extent of target regulation by individual trans-acting factors.

### Sample Protocol
The proximity-dependent biotin identification assay was performed according to Roux (Roux et al., 2012) with modifications. Proteins were eluted from streptavidin beads with 50 μl of biotin-saturated 1x  sample  buffer  (50  mM  Tris-HCL  pH  6,8,  12%  sucrose,  2%  SDS,  20  mM  DTT,  0,004%  Bromphenol  blue,  3  mM  Biotin)  by  incubation  for  7  min  at  98  °C.  For  identification  and  quantification  of  proteins,  samples  were  proteolysed  by  a  modified  filter  aided  sample  preparation  and  eluted  peptides  were  analysed  by  LC-MSMS  on  a  QExactive  HF  mass  spectrometer  (ThermoFisher  Scientific)  coupled  directly  to  a  UItimate  3000  RSLC  nano- HPLC (Dionex).

### Data Protocol
Label-free quantification was based on peptide intensities from extracted ion chromatograms  and  performed  with  the  Progenesis  QI  software  (Nonlinear  Dynamics, Waters). Raw files were imported and after alignment, filtering and normalization, all MSMS spectra  were  exported  and  searched  against  the  Swissprot  mouse  database  (16772  sequences,  Release  2016_02)  using  the  Mascot  search  engine  with  10  ppm  peptide  mass  tolerance  and  0.02  Da  fragment  mass  tolerance,  one  missed  cleavage  allowed,  and  carbamidomethylation  set  as  fixed  modification,  methionine  oxidation  and  asparagine  or  glutamine   deamidation   allowed   as   variable   modifications.   A   Mascot-integrated   decoy   database  search  calculated  an  average  false  discovery  of  <  1%  when  searches  were performed  with  a  mascot  percolator  score  cut-off  of  13  and  significance  threshold  of  0.05.  Peptide  assignments  were  re-imported  into  the  Progenesis  QI  software.  For  quantification,  only  unique  peptides  of  an  identified  protein  were  included,  and  the  total  cumulative normalized abundance was calculated by summing the abundances of all peptides allocated to the respective protein. A t-test implemented in the Progenesis QI software comparing the normalized  abundances  of  the  individual  proteins  between  groups  was  calculated  and  corrected  for  multiple  testing  resulting  in  q  values  (FDR  adjusted  p  values).  Only  proteins  identified  by  two  or  more  peptides  were  included  in  the list of Roquin-1 proximal proteins.

### Publication Abstract
None

### Keywords
Rbpome, Roquin, Bio-id

### Affiliations
Research Unit Protein Science, Helmholtz Center Munich
Research Unit Molecular Immune Regulation, Helmholtz Center Munich, Munich, Germany; Institute  for  Immunology,  Biomedical  Center,  Ludwig-Maximilians-Universität  in  Munich,  Planegg-Martinsried, Germany

### Submitter
Juliane Merl-Pham

### Lab Head
Dr Vigo Heissmeyer
Research Unit Molecular Immune Regulation, Helmholtz Center Munich, Munich, Germany; Institute  for  Immunology,  Biomedical  Center,  Ludwig-Maximilians-Universität  in  Munich,  Planegg-Martinsried, Germany


