### Accession
PXD009603

### Title
Proteomic Evidence of Dietary Sources in Ancient Dental Calculus

### Description
Archaeological dental calculus has emerged as a rich source of ancient biomolecules, including proteins. Previous analyses of proteins extracted from ancient dental calculus revealed the presence of the dietary milk protein Î²-lactoglobulin, providing direct evidence of dairy consumption in the archaeological record. However, the potential for calculus to preserve other food-related proteins has not yet been systematically explored. Here we analyse shotgun metaproteomic data from 100 archaeological dental calculus samples ranging from the Iron Age to the post-medieval period (8thC BC - 19thC AD) in Britain, as well as dental calculus from contemporary dental patients and recently deceased individuals, to characterise the range and extent of dietary proteins preserved in dental calculus. In addition to milk proteins, we detected proteomic evidence of foodstuffs such as cereals and plant products, as well as the digestive enzyme salivary amylase. We discuss the importance of optimized protein extraction methods, data analysis approaches, and authentication strategies in the identification of dietary proteins from archaeological dental calculus. Our ability to detect dietary proteins, although limited, demonstrates the potential of these methods to robustly identify foodstuffs in the archaeological record that are under-represented due to their poor preservation.

### Sample Protocol
A total of 100 archaeological samples of dental calculus were analyzed in this study, along with an additional 14 dental calculus deposits from living (n=10) or recently deceased (n=4) individuals. 38 of the 100 samples included existing datasets previously published in Warinner et al. (2014) focussing on populations from the UK. Raw data were downloaded from ProteomeXchange (Accession ID: PXD001362) and converted to mgf files using Proteowizard MSConvert (version 3.0.4743). For newly generated datasets, samples were extracted using one of two extraction methods: a Filter-Aided Sample Preparation Protocol (FASP) previously published in Warinner et al. (2014a, 2014b), or a Gel-Aided Sample Preparation Protocol (GASP) based on Fischer and Kessler (2015) modified for ancient mineralized samples. Protein extractions were performed at three locations: the BioArCh ancient protein laboratory at the University of York, UK; the ancient DNA laboratory of the Palaeogenomics and Bio-Archaeology Research Network (PalaeoBARN) at the University of Oxford, UK; and at the Laboratories of Molecular Anthropology and Microbiome Research (LMAMR) at the University of Oklahoma, USA. MS/MS analysis was performed at two locations: the mass spectrometry laboratories at the Target Discovery Institute, University of Oxford, UK); and at the Functional Genomics Center, University of Zurich, Switzerland.

### Data Protocol
Raw spectral data was converted to Mascot generic format (mgf) using Proteowizard MSConvert (version 3.0.4743) using the 200 most intense peaks in each MS/MS spectrum. MS/MS ion database searching was performed on Mascot (Matrix ScienceTM, version 2.4.01), against UniProt and the Human Oral Microbiome databases. Searches were performed against a decoy database to generate protein false discovery rates, which were adjusted to less than 5%. For samples extracted using GASP, propionamide (C) was set as a fixed modification and acetylation (protein N-terminus), deamidation (NQ), methionine oxidation, propionamide (K) and propionamide (N-terminus) were set as variable modifications. For samples extracted using FASP post-translational modifications were set as carbamidomethylation (fixed modification), and acetylation (protein N-term), deamidation (NQ), glutamine to pyroglutamate, and methionine oxidation as variable modifications. Peptide tolerance was 10 ppm, and MS/MS ion tolerance was 0.07 Da. Tryptic peptides were searched with up to 1 missed cleavage; semi-tryptic peptides were also searched with up to 1 missed cleavage for a subset of the samples. Protein results were filtered to a protein false discovery rate of less than 5% and an ion score of >25, and containing a minimum of two distinct peptides matching to different regions of the protein.

### Publication Abstract
Archaeological dental calculus has emerged as a rich source of ancient biomolecules, including proteins. Previous analyses of proteins extracted from ancient dental calculus revealed the presence of the dietary milk protein &#x3b2;-lactoglobulin, providing direct evidence of dairy consumption in the archaeological record. However, the potential for calculus to preserve other food-related proteins has not yet been systematically explored. Here we analyse shotgun metaproteomic data from 100 archaeological dental calculus samples ranging from the Iron Age to the post-medieval period (eighth century BC to nineteenth century AD) in England, as well as 14 dental calculus samples from contemporary dental patients and recently deceased individuals, to characterize the range and extent of dietary proteins preserved in dental calculus. In addition to milk proteins, we detect proteomic evidence of foodstuffs such as cereals and plant products, as well as the digestive enzyme salivary amylase. We discuss the importance of optimized protein extraction methods, data analysis approaches and authentication strategies in the identification of dietary proteins from archaeological dental calculus. This study demonstrates that proteomic approaches can robustly identify foodstuffs in the archaeological record that are typically under-represented due to their poor macroscopic preservation.

### Keywords
Human, Dental calculus, Dental plaque, Lc-ms/ms

### Affiliations
Max Planck Institute for the Science of Human History 
Max Planck Institute for the Science of Human History

### Submitter
Jessica Hendy

### Lab Head
Dr Jessica Hendy
Max Planck Institute for the Science of Human History


