### Accession
PXD004644

### Title
In-depth analysis of the phosphoprotein landscape of cytotoxic T lymphocytes and its regulation by cytokines

### Description
Here we present high-resolution mass spectrometry analysis of changes to the phosphoproteome of cytotoxic T lymphocytes (CTL) induced by the cytokine, Interleukin 12 (IL-12). We treated CTL with a short and long IL-12 stimulation and mapped the changes to the phosphoproteome using quantitative SILAC-based phosphoproteomics.

### Sample Protocol
P14 cytotoxic T cells were generated from splenocytes. T cells were activated for 2 days with the P14-TCR cognate ligand (peptide gp33-41 from Lymphocytic Choriomeningitis Virus, LCMV). Cells were then differentiated in SILAC medium (Life Technologies), supplemented with 200 mg/l L-proline, 84 mg/l L-arginine, 300 mg/l L-glutamate, 10% dialysed FBS with a 10 kDa cutoff (Life Technologies), 50 units/ml penicillin-G, 50 ug/ml streptomycin and 50uM b-mercaptoethanol and 20 ng/ml IL-2. The ""Light"" SILAC media contained L-[12C6, 14N4]arginine (R0) and L-[12C6, 14N2]lysine (K0). The ""Heavy"" media contained L-[13C6, 15N4]arginine (R10) and L-[13C6, 15N2]lysine (K8). Experiment 1: CTL were maintained in 20 ng/ml IL-2 for 5 days and then treated with 20 ng/ml IL-12 for 15 mins in the presence of IL-2. Experiment 2: CTL were maintained in 20 ng/ml IL-2 for 4 days and treated with 20 ng/ml IL-12 for 18 hrs in the presence of IL-2. For each phosphoproteome analysis, 3 independent biological replicate treatments were performed. For one replicate of Experiment 1, a label switch was performed, but for the other replicates, the heavy-labeled cells were subjected to the IL-12 treatment. Following treatments, control and treated cells were mixed at 1:1 ratio (total 2x10^8 cells). Cells were pelleted, washed once in cold PBS. Samples were lysed (in 8 M urea, 50 mM Tris-HCl pH 8, 1 mM TCEP) and sonicated. The proteins were precipitated with trichloroacetic acid (10%, v/v) for 15 mins at room temperature. The resulting protein pellets were washed before being resuspended in 8M urea, 50 mM Tris-HCl pH 8. The proteins were then alkylated with iodoacetamide (Sigma-Aldrich) prior to digestion with trypsin (Roche). Digestion products were desalted using C18 Sep-Pak cartridges (Waters). Desalted tryptic peptides were fractionated using Ultimate 3000 HPLC (Thermo Scientific) equipped with a 4.6 x 250- mm TSKgel Amide-80 5-um particle column (Tosoh Biosciences). The buffers used were 0.1% TFA (HILIC A) and 99.9% acetonitrile, 0.1% TFA (HILIC B). The peptide samples were resuspended in 80% HILIC B and injected onto the HILIC column. The elution gradient was: 80% B held for 20 mins followed by 80% B to 60% B in 40 mins and finally 0% B for 10 mins at a flow rate of 0.4 ml/min. 16 phosphopeptide fractions were collected for each replicate. Phosphopeptides were then enriched with titanium dioxide (Titansphere, GL Science) and then dried using a speedvac (Genevac). Phosphopeptide samples were resuspended in 1% formic acid and separated by nanoscale C18 reverse-phase liquid chromatography (Ultimate 3000 RSLC nano system, Thermo Scientific). The chromatography buffers were: HPLC A (2% acetonitrile, 0.1% formic acid), HPLC B (80% acetonitrile, 0.08% formic acid) and HPLC C (0.1% formic acid). Samples were washed onto the column with HPLC C and eluted with the following buffer gradient: 2% B (0-3 mins), 2-40% B (3-128 mins), 40-98% B (128-130 mins), 98% B (130-150 mins), 98-2% B (150-151 mins), and equilibrated in 2% B (151-180 mins) at a flow rate of 0.3 ul/min. The eluting peptide solution was automatically electrosprayed into the coupled Linear Trap Quadrupole (LTQ)-Orbitrap mass spectrometer (LTQ-Orbitrap Velos Pro; Thermo Scientific) using an Easy-Spray nanoelectrospray ion source (Thermo Scientific). The mass spectrometers were operated in positive ion mode and were used in data-dependent acquisition modes. A full scan (FT-MS) was acquired at a target value of 1,000,000 ions with resolution R = 60,000 over a mass range of 335-1800 atomic mass unit (amu). The fifteen most intense ions were selected for fragmentation in the LTQ Orbitrap Velos. Fragmentation was induced by collision-induced dissociation (CID) with a target value of 10,000 ions. For accurate mass measurement, the ""lock mass"" function (lock mass = 445.120024 Da) was enabled for MS scan modes. To improve the fragmentation of phosphopeptides, the multistage activation algorithm in the Xcalibur software was enabled for each MS/MS spectrum using the neutral loss values of 97.98, 48.99, 32.66 and 24.49 m/z units. Former target ions selected for MS/MS were dynamically excluded for 45 seconds. General mass spectrometric conditions were as follows: spray voltage, 1.8-2.5 kV; no sheath and auxiliary gas flow; ion transfer tube temperature, 250degC; normalised collision energy (35%) using wide band activation mode for MS2. The isolation width was set to 2 amu for IT-MS/MS. Ion selection thresholds were 5000 counts for MS2. An activation of q = 0.25 and activation time of 10 ms were applied in MS2 acquisitions. The fill time for FTMS was set to 500 ms and for ITMS to 100 ms. In total 112 HPLC-MS/MS runs were performed in this study.

### Data Protocol
For data analysis, we analysed the raw data obtained for three biological replicates of the 15 min IL-12 treatment (ST) with the three biological replicates of the 18 hr IL-12 treatment (LT). Data was processed using MaxQuant version 1.5.0.0 which incorporates the Andromeda search engine. Proteins were mapped to the reviewed Uniprot mouse protein database (""Mouse Complete Proteome"") using the following search parameters: a MS tolerance of 20 ppm, MS/MS tolerance of 0.5 Da and full trypsin specificity. Up to two missed cleavages were permitted. Protein N-terminal acetylation, oxidation of methionine, glutamine to pyroglutamate conversion, deamidation (NQ) and phosphorylation of serine, threonine and tyrosine were set as variable modifications, while carbamidomethylation of cysteine was set as fixed modification. The minimum peptide length for identification was set to at least 6 amino acids in length. The match between run function was enabled. False discovery rates (FDRs) of 0.01 were based on hits against a reversed sequence database and calculated at the level of peptides, proteins and modification sites.

### Publication Abstract
None

### Keywords
Phosphoproteomics, Lymphocytes, Silac, Il-12

### Affiliations
Division of Cell Signalling and Immunology, School of Life Sciences, University of Dundee, Dundee, UK
University of Dundee

### Submitter
Sarah Ross

### Lab Head
Dr Doreen Ann Cantrell
Division of Cell Signalling and Immunology, School of Life Sciences, University of Dundee, Dundee, UK


