### Accession
PXD039994

### Title
R7 Soybean seed organelle enrichment proteomics

### Description
This data is part of a study examining the results of increased malic enzyme activity in soybean seeds. The study characterizes changes to oil content and free amino acids when malic enzyme is expressed in different subcellular locations. Malic enzyme was expressed using seed-specific promoters to target activity to the seeds and was also designed to localize into and outside of the chloroplast.

### Sample Protocol
Organelle proteomic analysis was performed using a segmented Percoll gradient strategy designed to enrich for mitochondria. Fractions collected at three different points along the purification process (total lysate, centrifugal fraction, and Percoll fraction) were analyzed by proteomics resulting in the identification of unique proteins. Extracted proteins from collected fractions were digested with trypsin and cleaned up with C18 tips. LC-MS/MS was performed on an Orbitrap Fusion Lumos mass spectrometer coupled with a U3000 RSLCnano HPLC. The peptide separation utilized a 75 Âµm x 50 cm PepMap C18 column. Data-dependent acquisition was performed in which each MS1 scan was followed by Higher-energy Collisional Dissociation (HCD) of as many precursor ions in a two second cycle (Top Speed method). The mass range for the MS1 using the FTMS was 365 to 1800 m/z with resolving power set to 60,000 @ 400 m/z and the automatic gain control (AGC) target set to one million ions with a maximum fill time of 100 ms. Selected precursors were fragmented in the ion trap with an isolation window of 1.5 m/z, an AGC target value of 10,000 ions, a maximum fill time of 100 ms, a normalized collision energy of 35 and an activation time of 30 ms. Dynamic exclusion was performed with a repeat count of one, exclusion duration of 30 s, and a minimum MS ion count for triggering MS/MS set to 5000 counts.

### Data Protocol
Sequence mapping and label-free quantification were performed using Proteome Discoverer (version PD 2.4). Database searches used the Soybean proteome (Uniprot.org) appended with the two transgenic proteins from Arabidopsis with the SEQUEST search engine launched by PD. Trypsin was used to digest protein. Carbamidomethylation of cysteine was a fixed modification. Oxidation of methionine and acetylation of N-terminal of protein were specified as variable modifications. Protein quantification was achieved by using total intensities of all peptides and protein abundances were normalized against the sample with the highest total abundance. Proteins that were not identified in all three fractions or that had low false discovery rate confidence were excluded from final analysis. Remaining proteins were classified as cytosolic or mitochondrial using DeepLoc2.0 prediction algorithms. Within this group of proteins, known mitochondrial and cytosolic proteins were specifically identified from established literature. Mitochondrial purity was assessed by examining heatmap patterns of enrichment for mitochondrial proteins.

### Publication Abstract
Central metabolism produces amino and fatty acids for protein and lipids that establish seed value. Biosynthesis of storage reserves occurs in multiple organelles that exchange central intermediates including two essential metabolites, malate, and pyruvate that are linked by malic enzyme. Malic enzyme can be active in multiple subcellular compartments, partitioning carbon and reducing equivalents for anabolic and catabolic requirements. Prior studies based on isotopic labeling and steady-state metabolic flux analyses indicated malic enzyme provides carbon for fatty acid biosynthesis in plants, though genetic evidence confirming this role is lacking. We hypothesized that increasing malic enzyme flux would alter carbon partitioning and result in increased lipid levels in soybeans. Homozygous transgenic soybean plants expressing Arabidopsis malic enzyme alleles, targeting the translational products to plastid or outside the plastid during seed development, were verified by transcript and enzyme activity analyses, organelle proteomics, and transient expression assays. Protein, oil, central metabolites, cofactors, and acyl-acyl carrier protein (ACPs) levels were quantified overdevelopment. Amino and fatty acid levels were altered resulting in an increase in lipids by 0.5-2% of seed biomass (i.e. 2-9% change in oil). Subcellular targeting of a single gene product in central metabolism impacts carbon and reducing equivalent partitioning for seed storage reserves in soybeans.

### Keywords
Organelle enrichment, Glycine max, R7 seed

### Affiliations
USDA-ARS, Donald Danforth Plant Science Center, USA
USDA-ARS

### Submitter
Stewart Morley

### Lab Head
Dr Doug Allen
USDA-ARS, Donald Danforth Plant Science Center, USA


