### Accession
PXD004426

### Title
O-fucosylated glycoproteins of Toxoplasma gondii

### Description
The fucose-binding Aleuria aurantia lectin (AAL) labels punctiform structures in proximity to the nuclei of Toxoplasma gondii tachyzoites. We have utilized AAL-lectin affinity purification in conjunction with LC MS/MS to identify a set of proteins modified with fucose on serines and threonines

### Sample Protocol
Samples were prepared for LC-MS/MS analysis, by harvesting approximately 1.5x10^9 extracellular tachyzoites grown in human foreskin fibroblasts. To separate host cells from tachyzoites, the cell culture was first filtered through a 3 μm polycarbonate membrane. The tachyzoite filtrate was then concentrated via centrifugation, and subsequently washed four times with phosphate buffered saline. Tachyzoites were lysed in 2% SDS, 0.15 M NaCl, 0.1 mM DTT in 40 mM TrisHCl pH 7.4, heated for 20 min. at 50 °C and cooled to RT. The lysis solution was diluted to 0.03% SDS, 0.8% (w/v) N-octyl-glucopyranoside, 0.15 M NaCl, 40 mM DTT, 20 mM TrisHCl pH 7.4 (PD Buffer) with EDTA-free complete protease inhibitor cockatail (Roche). The diluted lysate was then incubated with 30 μg biotinylated-AAL for 2 hours rotating at 4 °C. Next, the solution was incubated with Dynabeads® MyOne™ Streptavidin T1 for 30 minutes. The beads were magnetically concentrated, then washed 5x with PD Buffer. The enriched proteins were eluted with [0.2 M] α-Methyl-Fucose in PD Buffer for 16 hours while shaking at 4 °C.  Biological replicates 1-3 (In Gel), the AAL-enriched proteins, eluted from the previous step, were loaded onto a 4-20% TGX gel (BioRad) and run until the front was about 2 cm from the top of the well. The gel was stained with colloidal Coomassie, the smear excised, reduced with DTT, alkylated with IAA, and then an in-gel trypsin digestion performed. The trypsin released peptides were extracted from the gel and the separated online using a Polaris-HR-Chip-3C18, composed of a trapping column, and a 75 μm x 150 mm analytical column. Peptides were trapped at a flow rate of 2 µl/min for 4 min at 2% mobile phase B (A: 1% acetonitrile, ACN, 0.1% formic acid, FA; B: 99% ACN, 0.1% FA). A 25 min linear gradient from 2-40% B, followed by 40–60% B 10 min gradient was used to separate peptides. The column was then washed with 95% B for 5 min and re-equilibrated at 2% B for 15 min. The chip was coupled to an Agilent 6550 QTOF mass spectrometer to acquire MS and CID MS/MS spectra. The top 20 MS peaks in the range between m/z 295-1700 were selected for MS/MS fragmentation and a custom ramped collisional energy (CE) table used, with nitrogen as the collision gas. Biological replicates 4 and 5 (In Solution), the AAL-lectin affinity eluates were precipitated in cold methanol (MeOH) with 0.1 M ammonium acetate for at least 18 h at -20 °C. The precipitate was washed in MeOH/ 0.1 M ammonium acetate three times, and collected by centrifugation. Remaining solvent was removed by speed vacuum (Speed Vac® Plus, Savant). The dried samples were re-suspended in 50 mM ammonium bicarbonate pH 8.0, then reduced with DTT, alkylated with IAA and digested with trypsin. The resulting digest was dried by speed vacuum and desalted on C18 ZipTip® concentrators (EMD Millipore). Samples were reconstituted in 2% ACN, 0.1% FA and separated on a nanoAcquity ultra-performance liquid chromatography (UPLC) capillary system (Waters), fitted with a of a  5 μm Symmetry C18 180 μm x 20 mm trap and a 1.7 μm BEH130C18 150 μm x 10 cm analytical column. The TriVersa NanoMate ion source (Advion) was coupled to either an LTQ-Orbitrap-XL-ETD or QE Plus. Samples were loaded on the trapping column for 4 min at 4 µl/ min, then separated at 0.5 µl/ min using the following conditions: 2-40% B linear gradient (43 min), hold at 40% B for 9 min, wash to 98% B and hold at 98% B (5 min), and re-equilibrate to 2% B (18 min). Alternatively, samples were re-constituted in 1% B and injected on the trapping column at 4 µl/min for 3.75 min. Analytical separation was performed as follows: 0.5 µl/ min flow, 1-60% B linear gradient for 90 min, followed by holding at 60% B for 7 min, ramp to 95% B (3 min), hold for 5 min, and re-equilibration at 1% B for 15 min. On the QE Plus, MS spectra were obtained by scanning over the range m/z 350-2000 with 1 microscan and a maximum injection time of 50 ms. MS/MS parameters: top 20 MS/MS isolation window, 1.4 m/z, 1 microscan, maximum injection time 45 ms, scan range m/z 100-2000. HCD fragmentation was performed at either 27 or 45 eV. ETD experiments were performed on an LTQ-Orbitrap-XL-ETD. MS scans: 1 microscan, range m/z 300-2000. MS/MS scans: isolation width 3 m/z, max injection time 25 ms/microscan, fragmentation using ETD with supplemental activation with a 180 ms reaction time, 4 microscans averaged per MS/MS with detection in the linear quadrupole ion trap. For HCD MS and MS/MS, spectra were acquired on the Orbitrap as follows. MS: range m/z 300-2000, 1 microscan, max. 500 ms injection time. MS/MS: 2 microscans, max. 500 ms injection time, 2 m/z isolation window, and 35 eV HCD fragmentation.

### Data Protocol
ETD spectra were searched using Mascot against the T. gondii GT1 predicted proteome (trypsin as the enzyme with max. 3 missed cleavages, 10 ppm precursor and 0.8 Da fragmentation mass tolerances). Carbamidomethyl cysteine was specified as fixed modification and dHex (on Ser/Thr) or methionine oxidation as variable modifications.  HCD MS/MS spectra were first analyzed with PEAKS. Each biological sample and any corresponding technical replicates were analyzed as individual files. The de novo search section had the following parameters: parent and fragment mass error tolerances 8 ppm and 0.05 Da, respectively. Carbamidomethyl cysteine was set as a fixed modification, and dHex Ser/Thr and oxidation of Met as variable modifications. A maximum of 8 variable PTMs/ peptide was set. Trypsin was specified as the protease with a maximum of 3 missed cleavages and one non-specific cleavage as a possibility. For the PEAKS DB search, the same settings as above were used with ToxoDB-24_TgondiiGT1_RH as database (combined predicted proteins for GT1 and RH, with redundant protein sequences removed) and False Discovery Rate (FDR) estimation enabled. For the PEAKS PTM search all parameters were identical to the previous sections except that the maximum variable PTMs/ peptide was reduced to 5, with all built in Unimod PTMs considered. Data acquired on the Agilent 6550 was analyzed as above, however,  the monoisotopic parent ion tolerance was increased to 12 ppm and fragmentation tolerance increased to 0.1 Da. To facilitate the examination of the multiple datasets, the PEAKS PTM results were exported to Scaffold 7.5 (Proteome Software). Each biological replicate and their respective technical replicates were imported as unique biosamples. Each biosample was grouped into one of two categories, “In Gel” or “in Solution”, depending upon how the samples were initially processed.

### Publication Abstract
Toxoplasma gondii is an intracellular parasite that causes disseminated infections in fetuses and immunocompromised individuals. Although gene regulation is important for parasite differentiation and pathogenesis, little is known about protein organization in the nucleus. Here we show that the fucose-binding Aleuria aurantia lectin (AAL) binds to numerous punctate structures in the nuclei of tachyzoites, bradyzoites, and sporozoites but not oocysts. AAL also binds to Hammondia and Neospora nuclei but not to more distantly related apicomplexans. Analyses of the AAL-enriched fraction indicate that AAL binds O-linked fucose added to Ser/Thr residues present in or adjacent to Ser-rich domains (SRDs). Sixty-nine Ser-rich proteins were reproducibly enriched with AAL, including nucleoporins, mRNA-processing enzymes, and cell-signaling proteins. Two endogenous SRDs-containing proteins and an SRD-YFP fusion localize with AAL to the nuclear membrane. Superresolution microscopy showed that the majority of the AAL signal localizes in proximity to nuclear pore complexes. Host cells modify secreted proteins with O-fucose; here we describe the O-fucosylation pathway in the nucleocytosol of a eukaryote. Furthermore, these results suggest O-fucosylation is a mechanism by which proteins involved in gene expression accumulate near the NPC.

### Keywords
Nucleus, Lectin, Toxoplasma gondii, Lc ms/ms, Aal, Affinity purification, Fucose

### Affiliations
Boston University
Boston University School of Medicine, Department of Biochemistry, Center for Biomedical Mass Spectrometry

### Submitter
John Haserick

### Lab Head
Dr Catherine E. Costello
Boston University School of Medicine, Department of Biochemistry, Center for Biomedical Mass Spectrometry


