### Accession
PXD005100

### Title
DREX proteome analysis for spectral library

### Description
In order to prepare a spectral library for data-indepentend acquisition mass spectrometry studies, we try to cover as many proteins from Drosophila melanogaster embryonic protein extracts.

### Sample Protocol
Drosophila embryos were harvested 0-90 min after egglaying and lysed in ice-cold extraction buffer. Protein extracts were prepared as described in Voelker-Albert M. et al. 2015, and after dechorionation, proteins were subjected to trypsin digestion in solution. Peptide mixtures were desalted on C18 stage tips and submitted to nano-reversed phase C18 chromatography with online mass spectrometry detection. The chromatographic separation was performed on an Ultimate 3000 nano-chromatography system (Thermo-Fisher) couple to an QTOF mass spectrometer 6600 TripleTOF (Sciex). Peptides were first desalted on an trapping column (Thermo-Fisher, AcclaimPepMap C18, 5x0.3 mm, 5 um) and subsequently separated on a nano-RP/C18 column (Thermo-Fisher, AcclaimPepMap C18, 250x0.075 mm, 2.4 um, 100A) employing a linear gradient from 3.2-32% ACN in 0.1% FA in 150 min. The exit of the separation column was directly coupled to the nano-electrospray ionization source via a emitter tip (New Objective). MS analysis was performed in positive ionization mode with a data dependent acquisition protocol for up to 40 MS/MS spectra of precursor ions with a charge state between 2+ and 5+ and minimal signal abundance of 150 cps. Precursors which were subjected for CID-fragmentation and their isotopic signals were excluded for 30 seconds from repeated MS/MS analysis with a mass accuracy of 20 ppm. Optimal collision energies were calculated based on m/z ratio, charge state and the instrument parameters. For sample DREX 2 a minimal signal intensity of 1500 cps was selected.

### Data Protocol
MS-Data were searched with the Andromeda-algorithm within MaxQuant 1.5.1.2 against a combined forward/reversed protein database from Flybase (version 5.24) comprising protein sequences of the Drosophila melanogaster genome. Peptides matches were filtered for 1% false discoveries, proteins for 5% revesed sequences. Protein Nterminal acetylation, lysing ubiquitination (GG) and methione oxidation were enabled as variable modifications; carbamidomethylation of cysteine was set as fixed modfification. The mass tolerance on fragment ions was 50 ppm. For the first search a tolerance of 30 ppm and for the main search 7 ppm was allowed. Only peptides with specific trypsin cleavage pattern and maximally 2 missed cleavages were reported. Peptie hits were further filtered for a minimal length of 7 amino acids and a minimal score of 20 for unmodified peptides and 40 with post-translational modifications matching.

### Publication Abstract
Spectral libraries play a central role in the analysis of data-independent-acquisition (DIA) proteomics experiments. A main assumption in current spectral library tools is that a single characteristic intensity pattern (CIP) suffices to describe the fragmentation of a peptide in a particular charge state (peptide charge pair). However, we find that this is often not the case. We carry out a systematic evaluation of spectral variability over public repositories and in-house data sets. We show that spectral variability is widespread and partly occurs under fixed experimental conditions. Using clustering of preprocessed spectra, we derive a limited number of multiple characteristic intensity patterns (MCIPs) for each peptide charge pair, which allow almost complete coverage of our heterogeneous data set without affecting the false discovery rate. We show that a MCIP library derived from public repositories performs in most cases similar to a "custom-made" spectral library, which has been acquired under identical experimental conditions as the query spectra. We apply the MCIP approach to a DIA data set and observe a significant increase in peptide recognition. We propose the MCIP approach as an easy-to-implement addition to current spectral library search engines and as a new way to utilize the data stored in spectral repositories.

### Keywords
Drosophila embryo proteome, Spectral library, Fragmentation pattern

### Affiliations
CECAD, University of Cologne, AG Kr√ºger
Protein Analysis Unit ZfP LMU Munich  BioMedical Center Grosshaderner Strasse 9 82152 Planegg-Martinsried  Tel. +49 89 2180 75420

### Submitter
Andreas Schmidt

### Lab Head
Dr Axel Imhof
Protein Analysis Unit ZfP LMU Munich  BioMedical Center Grosshaderner Strasse 9 82152 Planegg-Martinsried  Tel. +49 89 2180 75420


