### Accession
PXD015485

### Title
Protein-Protein Interactions Maps of the KRAS Pathway in HEK293 and A549 cells

### Description
Activating mutations in RAS GTPases drive one fifth of cancers, but poor understandings of many RAS effectors and regulators, and of the roles of their different paralogs, continue to impede drug development. We developed a multi-stage discovery and screening process to understand RAS function and identify RAS-related susceptibilities in lung adenocarcinoma. Using affinity purification mass spectrometry (AP/MS), we generated a protein-protein interaction map of the RAS pathway containing thousands of interactions. From this network we constructed a CRISPR dual knockout library targeting 119 RAS-related genes that we screened for genetic interactions (GIs). We found important new effectors of RAS-driven cellular functions, RADIL and the GEF RIN1, and over 250 synthetic lethal GIs, including a potent KRAS-dependent interaction between RAP1GDS1 and RHOA. Many GIs link specific paralogs within and between gene families. These findings illustrate the power of the multiomic approach to identify synthetic lethal combinations for hitherto undruggable cancers.

### Sample Protocol
Affinity purification / mass spectrometry was carried out as described in (Kanie et al., 2017), with the notable exception that HEK293 or A549 cells carrying the indicated genes were cultured instead. These cells were cultured as previously described, harvested, and lysed. Bait and prey proteins were isolated from the lysate by tandem-affinity purification by a tandem-affinity purification process. The final eluate was then fractionated by SDS-PAGE, and proteins were extracted from gel slices. These proteins were then prepared for LC-MS/MS and analyzed on an Orbitrap Fusion mass spectrometer. MS/MS data was compared to an NCBI Genbank FASTA database containing all human proteomic isoforms with the exception of the tandem affinity bait construct sequence and common contaminant proteins. Spectral counts were assumed to have undergone fully specific proteolysis and allowing up to two missed cleavages per peptide. All data was filtered and presented at a 1% false discovery rate.

### Data Protocol
Quantification was described as in (Ding et al., 2016). Briefly, for individual genes whose products were identified in each AP/MS sample, we assigned a normalized spectral abundance factor (NSAF) to each gene (Zybailov et al., 2006). Using a panel of 116 other AP/MS experiments, we inferred expected lognormal background distributions for NSAFs of each gene using the median and scaled median absolute deviation for location and scale parameters, respectively. Each experiment was screened for correlation against each previous experiment, and highly correlated experiments (Pearsonâ€™s r>0.7) were removed from consideration prior to the inference of each gene-wise null distribution. For each gene-wise signal in an experimental dataset, we compare the signal of products of that gene to its background distribution, and it is from this comparison that z-scores and p-values are reported. We also conducted an experiment using renilla luciferase as a bait protein, since most negative control experiments were not conducted in A549 cells. This score allowed us to identify proteins more highly expressed in A549 cells relative to orthogonal controls.

### Publication Abstract
Activating mutations in RAS GTPases drive many cancers, but limited understanding of less-studied RAS interactors, and of the specific roles of different RAS interactor paralogs, continues to limit target discovery. We developed a multistage discovery and screening process to systematically identify genes conferring RAS-related susceptibilities in lung adenocarcinoma. Using affinity purification mass spectrometry, we generated a protein-protein interaction map of RAS interactors and pathway components containing hundreds of interactions. From this network, we constructed a CRISPR dual knockout library targeting 119 RAS-related genes that we screened for <i>KRAS</i>-dependent genetic interactions (GI). This approach identified new RAS effectors, including the adhesion controller RADIL and the endocytosis regulator RIN1, and &gt;250 synthetic lethal GIs, including a potent <i>KRAS</i>-dependent interaction between <i>RAP1GDS1</i> and <i>RHOA.</i> Many GIs link specific paralogs within and between gene families. These findings illustrate the power of multiomic approaches to uncover synthetic lethal combinations specific for hitherto untreatable cancer genotypes. SIGNIFICANCE: We establish a deep network of protein-protein and genetic interactions in the RAS pathway. Many interactions validated here demonstrate important specificities and redundancies among paralogous RAS regulators and effectors. By comparing synthetic lethal interactions across <i>KRAS</i>-dependent and <i>KRAS</i>-independent cell lines, we identify several new combination therapy targets for RAS-driven cancers.<i>This article is highlighted in the In This Issue feature, p. 1775</i>.

### Keywords
Ap/ms, Protein-protein interaction, Kras, Lung adenocarcinoma, Cancer, Gtpase

### Affiliations
Department of Microbiology & Immunology Stanford University School of Medicine
Stanford University

### Submitter
Janos Demeter

### Lab Head
Dr Peter Kent Jackson
Department of Microbiology & Immunology Stanford University School of Medicine


