### Accession
PXD032930

### Title
Single-copy locus proteomics of early- and late-firing DNA replication origins identifies a role of Ask1/DASH complex in replication timing control

### Description
The chromatin at origins of replication is thought to influence DNA replication initiation in eukaryotic genomes. However, it remains unclear how the chromatin composition controls the firing of early-efficient (EE) or late-inefficient (LI) origins. Here, we used site-specific recombination and single-locus chromatin isolation to purify EE and LI replication origins in Saccharomyces cerevisiae . Using mass spectrometry, we define the histone modification landscape and identify the protein composition of native chromatin regions surrounding the EE and LI replication start sites. In addition to the known origin interactors, we find novel origin-associated factors, such as the kinetochore-associated Ask1/DASH complex. Strikingly, we show that Ask1 regulates the replication timing control of specific origins in yeast. Thus, our unbiased approach identifies functionally-relevant proteomes at single-copy loci and would be widely applicable to provide an in-depth quantitative characterization of histone modification and protein interaction networks of chromatin at any genomic locus of interest.

### Sample Protocol
Filter aided sample preparation (FASP) digest of protein samples for mass spectrometry The eluates from the purified samples were dried using a speed vac vacuum concentrator and resolved in 300Âµl 50mM ammonium bicarbonate (ABC) and digested using a modified FASP procedure (Grosche et al., 2016; WiÅ›niewski et al., 2009). After protein reduction and alkylation using DTT and IAA, the proteins were centrifuged on a 30kDa cutoff filter device (Sartorius), washed thrice with UA buffer (8M urea in 0.1M Tris/HCl pH 8.5) and twice with 50mM ABC. The proteins were digested for 2h at room temperature using 0.5Âµg Lys-C (Wako Chemicals, Neuss, Germany) and for 16h at 37Â°C using 1Âµg trypsin (Promega, Mannheim, Germany). After centrifugation (10min at 14.000 g) the eluted peptides were acidified with 0.5% TFA and stored at -20Â°C. LC-MS/MS measurements LC-MS/MS analysis was performed on a Q-Exactive HF and HF-X mass spectrometer (Thermo Fisher Scientific) online coupled to an Ultimate 3000 nano-RSLC (Thermo Fisher Scientific). Tryptic peptides were automatically loaded on a C18 trap column (300 Âµm inner diameter (ID) Ã— 5 mm, Acclaim PepMap100 C18, 5 Âµm, 100 Ã…, LC Packings) at 30Âµl/min flow rate prior to C18 reversed phase chromatography on the analytical column (nanoEase MZ HSS T3 Column, 100Ã…, 1.8Âµm, 75Âµm x 250mm, Waters) at 250nl/min flow rate in a 95 minutes non-linear acetonitrile gradient from 3 to 40% in 0.1% formic acid. Profile precursor spectra from 300 to 1500 m/z were recorded at 60000 resolution with an automatic gain control (AGC) target of 3e6 and a maximum injection time of 30ms and 50ms. TOP10 and TOP15 fragment spectra of charges 2 to 7 were recorded at 15000 resolution with an AGC target of 1e5, a maximum injection time of 50ms, an isolation window of 1.6 m/z, a normalized collision energy of 28 and a dynamic exclusion of 30 seconds. The mass spectrometry proteomics data have been deposited to the ProteomeXchange Consortium (http://www.proteomexchange.org/) via the PRIDE partner repository with the dataset identiï¬�er PDX032930.

### Data Protocol
Protein Identification and label-free quantification Proteome Discoverer 2.5 software (Thermo Fisher Scientific; version 2.5.0.400) was used for peptide and protein identification via a database search (Sequest HT search engine) against Swissprot yeast data base (Release 2017_04, 6721 sequences), considering full tryptic specificity, allowing for up to two missed tryptic cleavage sites, precursor mass tolerance 10 ppm, fragment mass tolerance 0.02Da. Carbamidomethylation of Cys was set as a static modification. Dynamic modifications included deamidation of Asn, Gln and Arg, oxidation of Pro and Met; and a combination of Met loss with acetylation on protein N-terminus. Percolator was used for validating peptide spectrum matches and peptides, accepting only the top-scoring hit for each spectrum, and satisfying the cutoff values for FDR <5%, and posterior error probability <0.01. The final list of proteins complied with the strict parsimony principle. The quantification of proteins, was based on abundance values for unique peptides. Abundance values were normalized on total peptide amount to account for sample loading errors. The protein abundances were calculated summing up the abundance values for admissible peptides. The final protein ratio was calculated using median abundance values of three replicate analyses each. The statistical significance of the ratio change was ascertained employing the T-test approach described in Navarro et al., 2014 which is based on the presumption that we look for expression changes for proteins that are just a few in comparison to the number of total proteins being quantified. The quantification variability of the non-changing "background" proteins can be used to infer which proteins change their expression in a statistically significant manner.

### Publication Abstract
The chromatin environment at origins of replication is thought to influence DNA replication initiation in eukaryotic genomes. However, it remains unclear how and which chromatin features control the firing of early-efficient (EE) or late-inefficient (LI) origins. Here, we use site-specific recombination and single-locus chromatin isolation to purify EE and LI replication origins in Saccharomyces cerevisiae. Using mass spectrometry, we define the protein composition of native chromatin regions surrounding the EE and LI replication start sites. In addition to known origin interactors, we find the microtubule-binding Ask1/DASH complex as an origin-regulating factor. Strikingly, tethering of Ask1 to individual origin sites advances replication timing (RT) of the targeted chromosomal domain. Targeted degradation of Ask1 globally changes RT of a subset of origins, which can be reproduced by inhibiting microtubule dynamics. Thus, our findings mechanistically connect RT and chromosomal organization via Ask1/DASH with the microtubule cytoskeleton.

### Keywords
Locus-specific chromatin isolation; site-specific recombination; affinity purification; dna replication origin; replication timing; replication efficiency; origin chromatin structure; dash complex; ask1 protein; microtubule cytoskeleton

### Affiliations
HMGU Core Facility Proteomics
Institute of Epigenetics and Stem Cells, Helmholtz ZentrumMünchen, Feodor-Lynen- Strasse 21, 81377 München, Germany

### Submitter
Ann-Christine König

### Lab Head
Dr Dr Stephan Hamperl
Institute of Epigenetics and Stem Cells, Helmholtz ZentrumMünchen, Feodor-Lynen- Strasse 21, 81377 München, Germany


