### Accession
PXD018791

### Title
Targeted proteomics analysis of cortical collecting ducts with Pendrin knockout

### Description
We analyzed microdissected cortical collecting ducts of pendrin knockouts using a sensitive targeted proteomics approach proteomics approach.

### Sample Protocol
Pendrin KO mice and their respective litter mates were killed by cervical dislocation. Their kidneys were subsequently removed and cut in coronal slices. 5 CCDs per mouse were manually dissected from the medullary rays. Tubules were washed in incubation solution (in mM: 140 NaCl, 0.4 KH2PO4, 1.6 K2HPO4, 1 MgSO4, 10 Na-acetate, 1 α-ketoglutarate, 1.3 Ca-gluconate, 5 glycine, containing 48 mg/l trypsin inhibitor and 25 mg/l DNase I at pH 7.4) and transferred to 5% SDS. Proteins were denaturized by boiling samples at 95°C for 5 min. DTT (5 mM, reduction of proteins) and Benzonase (25 U/µl for removal of RNA/DNA) were added and incubated for 30 min at 37°C. 2-Iodoacetamide (10 mM) was used to alkylate proteins for 45 min at room temperature. Samples were stored at on dry ice and –80°C until further processing. The samples were thawed on ice and directly submitted to a modified, previously described SP3 protocol 1,2 in which tryptic digestion was performed. Tubule peptides were analyzed on a nLC-MS/MS system with an easy nano-LC (Thermo scientific) coupled to a tandem mass spectrometer, a quandrupole orbitrap hybride QExactive Plus (Thermo Scientific). A 1h LC gradient on a self-packed C18 column and nanoflow (200nl/min) was used as previously described 3 .  Targeted mass spectrometry proteomics data was obtained using the inclusion list option. For a list of included masses as well as their retention times, please see table 1 and table 2.  The spectral database generated from the data dependent spectral library of the initial CCD profiling experiments, and the data was analyzed and quantified on MS2 level using skyline 4 as previously described 1 . In brief these filters required 1) a coelution of MS2 2) a mass error smaller than 10ppm and 3) elution within a predefined acquisition window.   1. Höhne, M. et al. Single-nephron proteomes connect morphology and function in proteinuric kidney disease. Kidney Int. 93, 1308–1319 (2018). 2. Rinschen, M. M. Chapter 1 - Single glomerular proteomics: A novel tool for translational glomerular cell biology. in Methods in Cell Biology (ed. Weimbs, T.) vol. 154 1–14 (Academic Press, 2019). 3. Bartram, M. P. et al. Three-layered proteomic characterization of a novel ACTN4 mutation unravels its pathogenic potential in FSGS. Hum. Mol. Genet. 25, 1152–1164 (2016). 4. MacLean, B. et al. Skyline: an open source document editor for creating and analyzing targeted proteomics experiments. Bioinformatics 26, 966–968 (2010). 5. Cox, J. & Mann, M. MaxQuant enables high peptide identification rates, individualized p.p.b.-range mass accuracies and proteome-wide protein quantification. Nat. Biotechnol. 26, 1367–1372 (2008). 6. Cox, J. et al. Accurate proteome-wide label-free quantification by delayed normalization and maximal peptide ratio extraction, termed MaxLFQ. Mol. Cell. Proteomics MCP 13, 2513–2526 (2014). 7. Schwanhäusser, B. et al. Global quantification of mammalian gene expression control. Nature 473, 337–342 (2011). 8. Plain, A. et al. Corticomedullary difference in the effects of dietary Ca2+ on tight junction properties in thick ascending limbs of Henle’s loop. Pflugers Arch. 468, 293–303 (2016). 9. Greger, R. & Schlatter, E. Properties of the lumen membrane of the cortical thick ascending limb of Henle’s loop of rabbit kidney. Pflugers Arch. 396, 315–324 (1983).

### Data Protocol
Targeted mass spectrometry proteomics data was obtained using the inclusion list option. For a list of included masses as well as their retention times, please see table 1 and table 2.  The spectral database generated from the data dependent spectral library of the initial CCD profiling experiments, and the data was analyzed and quantified on MS2 level using skyline 4 as previously described 1 . In brief these filters required 1) a coelution of MS2 2) a mass error smaller than 10ppm and 3) elution within a predefined acquisition window.

### Publication Abstract
The revolution of the omics technologies has enabled profiling of the molecules of any sample. However, the heterogeneity of the kidney with highly specialized nephron segments like the cortical collecting duct (CCD) poses a challenge regarding integration of omics data and functional analysis. We examined function and proteome from the same single CCDs of C57Bl6 mice by investigating them in a double-barreled perfusion system before targeted mass spectrometry. Transepithelial voltage (V<sub>te</sub>), transepithelial resistance, as well as amiloride-sensitive voltage (&#x394;V<sub>te</sub>amil) were recorded. CCDs were of 400-600 &#xb5;m of length, showed lumen negative V<sub>te</sub> between -8.5 and -32.5 mV and an equivalent short circuit current I'<sub>sc</sub> between 54 and 192 &#xb5;A/cm<sup>2</sup>. On a single-tubule proteome level, intercalated cell (IC) markers strongly correlated with other intercalated cell markers and negatively with principal cell markers. Integration of proteome data with phenotype data revealed that tubular length correlated with actin and Na<sup>+</sup>-K<sup>+</sup>-ATPase expression. &#x394;V<sub>te</sub>(amil) reflected the expression level of the &#x3b2;-subunit of the epithelial sodium channel. Intriguingly, &#x394;V<sub>te</sub>(amil) correlated inversely with the water channel AQP2 and the negative regulator protein NEDD4L (NEDD4-2). In pendrin knockout (KO) mice, the CCD proteome was accompanied by strong downregulation of other IC markers like CLCNKB, BSND (Barttin), and VAA (vH<sup>+</sup>-ATPase), a configuration that may contribute to the salt-losing phenotype of Pendred syndrome. Proteins normally coexpressed with pendrin were decreased in pendrin KO CCDs. In conclusion, we show that functional proteomics on a single nephron segment scale allows function-proteome correlations, and may potentially help predicting function from omics data.

### Keywords
Pendrin, Ccd, Kidney

### Affiliations
Kidney research center cologne
University Hospital Cologne

### Submitter
Markus Rinschen

### Lab Head
Dr Markus Rinschen
University Hospital Cologne


