### Accession
PXD002846

### Title
Molecular signatures of membrane protein complexes underlying muscular dystrophy

### Description
Mutations in genes encoding components of the sarcolemmal dystrophin-glycoprotein complex (DGC) are responsible for a large number of muscular dystrophies. As such, molecular dissection of the DGC is expected to both reveal pathological mechanisms, and provides a biological framework for validating new DGC components. Establishment of the molecular composition of plasma-membrane protein complexes has been hampered by a lack of suitable biochemical approaches. Here we present an analytical workflow based upon the principles of protein correlation profiling that has enabled us to model the molecular composition of plasma-membrane protein complexes in mouse skeletal muscle. We also report our analysis of protein complexes in mice harboring mutations in DGC components, which implicates cell-adhesion pathways directly targeted by NFκB in the pathophysiology of DGC-related muscular dystrophies. Our study suggests that inflammatory and compensatory mechanisms are activated in these diseases. Additionally, it provides a molecular framework that will facilitate refinement of our understanding of the DGC, identification of protein biomarkers of neuromuscular disease, and pharmacological interrogation of the DGC in adult skeletal muscle.

### Sample Protocol
Co-immunoprecipitation biochemistry and immunoblotting Skeletal muscle was isolated from rabbit skeletal muscle (Pelfreeze, 41225-2), and microsomes were prepared as described elsewhere (4). Total microsomes were solubilized overnight at 4°C while rotating in a solution of 1% digitonin, 50 mM Tris-HCl pH 7.4, 150 mM NaCl, and proteinase inhibitors. Non-solubilized proteins were pelleted at 100,000 x g. The supernatant was incubated overnight with wheat-germ agglutinin (WGA)-agarose beads (Vector Laboratories) at 4°C. The beads were washed 3x in washing buffer (0.1% digitonin, 50 mM Tris-HCl pH 7.4, 150 mM NaCl, proteinase inhibitors), and proteins were eluted by 1-hour incubation with 0.3 M N-acetyl-D-glucosamine in washing buffer, at 4°C. Buffer exchange (0.1% digitonin, 50 mM Tris-HCl pH 7.4, 150 mM NaCl, and proteinase inhibitors) was performed to remove N-acetyl-D-glucosamine. Antibody beads were generated by coupling ~3.5 mg of anti-dystroglycan antibody (IIH6), anti-ryanodine receptor antibody (XA7), or bovine serum albumin (BSA) to Sepharose beads using activated CNBr according to manufacturer’s protocol (GE Healthcare Life Sciences). WGA-enriched proteins were incubated with antibody beads, followed by washing 3x (0.1% digitonin, 50 mM Tris-HCl pH 7.4, 150 mM NaCl, proteinase inhibitors), and eluted in 0.1 M triethylamine, pH 11.5. Western blots were incubated with antibodies against α-DG and β-DG (AF6868), DMD (Abcam), SGCA (20A6), SGCB (G26), SGCG (21B5), SGCD (R214), ADBN (Transduction Laboratories), and NNOS (R200) as described elsewhere (36).  Processing of MS samples Using large-scale filter assisted sample preparation, samples were buffer exchanged four times with 8 M Urea, 50 mM Tris-HCl, pH 8.5, and 100 mM beta-mercapto-ethanol. The first three rounds of buffer exchange were performed for 2 hours, and the fourth overnight. Next, samples were concentrated (Amicon Ultra, 10 kD cut-off MW, Millipore) and separated by SDS-PAGE on 4–12% gels (Invitrogen) for analysis by silver staining. Samples were reduced with 10 mM dithiothreitol (DTT) at 37°C in 8 M Urea, 50 mM Tris-HCl, pH 8.5 for 30 minutes, and subsequently alkylated with 55 mM iodoacetamide for 30 min in the dark. Samples were diluted to a final concentration of 0.5 M urea with 50 mM Tris-HCl pH 8.5. Each sample was treated with 1 µg of sequence-grade trypsin (Promega) and incubated overnight at 37°C. Samples were then spiked with a tryptic digest of bovine serum albumin (BSA) containing iodoacetic-acid alkylated cysteine residues (Michrom Bioresources), at a 1:75 molar ratio. Samples were acidified and desalted on Vydac C18 spin-columns (The Nest Group), and then subjected to SCX fractionation on polysulfoethyl-A packed spin-columns (The Nest Group) according to the manufacturer’s protocol. Briefly, desalted samples were dissolved in SCX buffer B (5 mM KHPO4, 25% acetonitrile (ACN)) and loaded onto SCX spin-columns, and the tryptic digests were then released from the SCX spin-columns using a three-step KCl elution gradient developed from a mixture of buffer B and buffer C (5 mM KHPO4, 25% ACN, 350 mM KCl). Salt-eluted fractions were desalted, dried down, and dissolved in MS loading buffer (1% acetic acid, 1% ACN).   Mass spectrometry The samples were subjected to LC-MS/MS on the 6520 Agilent Accurate-Mass Quadrupole Time-of-Flight (Q-TOF) mass spectrometer, interfaced with the HPLC Chip Cube. The samples were loaded onto the large-capacity C18 Chip II (160 nL enrichment column, 9 mm analytical column) and subjected to LC-MS/MS analysis for 90 minutes in a gradient of 1.5% to 35% buffer B (100% ACN, 0.8% AA). Full MS (MS1) data were acquired with a mass range of 400–1250 m/z and an acquisition rate of 1 spectra/second. From these data, an ion preference list was generated using Agilent MassHunter Qualitative Software. dMS was performed using the following settings: a maximum of 10 ions per cycle, a narrow isolation width (~1.3 atomic mass units), precursor masses dynamically excluded for 30 seconds after 8 MS/MS in a 30-second time window, and use of the preferred ion list. The capillary voltage and capillary temperature settings for MS were set to 1800 V and 330 °C, respectively. The infused reference mass of 1221.9906 was used to correct precursor m/z masses in each LC-MS/MS experiment.

### Data Protocol
Protein identification  The raw.d files were searched against the UniProt mouse database (downloaded April 18th, 2012; with 55190 entries) using SpectrumMill Software version B.04.00.127 and the following settings: precursor mass tolerance of 25 parts per million (ppm), product mass tolerance of 200 ppm, and a maximum of two trypsin miss cleavages. Searches for post-translational modifications included a static carbamidomethylation on cysteine residues (C = 57.02146 AMU), differential modifications for oxidized methionine (M = 15.9949 AMU), phosphorylated serine, threonine, tyrosine (STY = 79.9663 AMU), and ubiquitinated lysine (K = 114.0429 AMU). For normalization between the samples, the raw.d files were searched against the UniProt bovine database with same search settings as above except the files were searched with a static carboxymethylatin on cysteine residues (C = 58.005 AMU). A total of 14 cysteine-containing BSA peptides were quantified and the total intensities of these peptides were collated for each sample and the differences between samples were normalized (Supplemental Table 1). The peptide hits were autovalidated with the “Fixed thresholds” setting using the following values: Score Threshold of 3, % Spectral Intensity (SPI) Threshold of 30, and Rank 1-Rank 2 Threshold of 2, with delta Fwd-Rev not checked such that the initial False-Discovery Rate (FDR) was ≤ 7%. To reduce the FDR well below ~7% the proteomic results were further filtered by applying a delta Fwd-Rev score of 1.2 to provide the most stringent filter in the removal of false positive protein identifications.

### Publication Abstract
Mutations in genes encoding components of the sarcolemmal dystrophin-glycoprotein complex (DGC) are responsible for a large number of muscular dystrophies. As such, molecular dissection of the DGC is expected to both reveal pathological mechanisms, and provides a biological framework for validating new DGC components. Establishment of the molecular composition of plasma-membrane protein complexes has been hampered by a lack of suitable biochemical approaches. Here we present an analytical workflow based upon the principles of protein correlation profiling that has enabled us to model the molecular composition of the DGC in mouse skeletal muscle. We also report our analysis of protein complexes in mice harboring mutations in DGC components. Bioinformatic analyses suggested that cell-adhesion pathways were under the transcriptional control of NF&#x3ba;B in DGC mutant mice, which is a finding that is supported by previous studies that showed NF&#x3ba;B-regulated pathways underlie the pathophysiology of DGC-related muscular dystrophies. Moreover, the bioinformatic analyses suggested that inflammatory and compensatory mechanisms were activated in skeletal muscle of DGC mutant mice. Additionally, this proteomic study provides a molecular framework to refine our understanding of the DGC, identification of protein biomarkers of neuromuscular disease, and pharmacological interrogation of the DGC in adult skeletal muscle https://www.mda.org/disease/congenital-muscular-dystrophy/research.

### Keywords
Directed ms (dms), Mouse, Membrane protein complexes, Skeletal muscle

### Affiliations
University of Iowa
Molecular Physiology & Biophysics, Wright Lab, University of Iowa, USA

### Submitter
Jordy Hsiao

### Lab Head
Dr Michael Eugene Wright
Molecular Physiology & Biophysics, Wright Lab, University of Iowa, USA


