### Accession
PXD018932

### Title
Whole proteome and Secretome analysis of  Staphylococcus aureus ClpXP mutant strains

### Description
Staphylococcus aureus represents an opportunistic pathogen which utilizes elaborate quorum sensing mechanisms to precisely control the expression and secretion of virulence factors. Previous studies indicated a role of the ClpXP proteolytic system in controlling pathogenesis. While detailed transcriptome data for ClpP and ClpX is available, corresponding studies on the proteome, secretome and metabolome level are largely lacking. In order to decipher the functional roles of ClpP and ClpX more globally we utilized S. aureus deletion mutants of the corresponding genes for in-depth proteomic LC-MS/MS analysis. These studies were complemented by a ClpP active site mutant strain to monitor changes solely depending on the presence and not the activity of the protein. A comparison of these strains with wild type revealed e.g. downregulation of virulence, purine/pyrimidine biosynthesis, iron uptake and stress response. Correspondingly, a reduction of a subset of purine and pyrimidine metabolite levels independently confirmed these results. Interestingly, comparison between the ClpP knockout and ClpP active site mutant strains revealed characteristic differences in UMP biosynthesis and the staphyloferrin B pathway, suggesting that the presence of the protein, although inactive, is important for maintaining these processes. These results are not only of fundamental importance to understand the cellular role of ClpXP but also have implications for the development of novel virulence inhibitor classes.

### Sample Protocol
Overnight cultures of S. aureus NCTC 8325 wt, knockout clpP, knockout clpX and clpP S98A were diluted in B-medium to an OD600 = 0.03 and incubated (37°C, 200 rpm) for 13.5 h (wt), 16.5 h (knockout clpP and clpX) or 17 h (clpP S98A). Varying incubation times ensured similar post-exponential growth phases for all utilized strains. Optical density of all cultures was determined after incubation time and 15 mL OD600 = 5 harvested for whole proteome analysis by centrifugation (6000 g, 10 min, 4°C). 2 mL of the resulting supernatant were sterile-filtered (0.22 µm) per sample and snap-frozen in liquid N2. Cells pellets were washed twice with cold aq. 0.9% NaCl solution and resulting bacterial cell pellet snap-frozen in liquid N2 and stored at -80°C. The procedure was carried out on multiple days resulting in 7 biological replicates per strain for whole proteome and secretome analysis. Whole proteome: Cell pellets were thawed on ice, resuspended in lysis buffer I (470 µL, 100 mM Tris-HCl, 0.4% (w/v) SDS in ddH2O, pH 7) and cells lysed by sonication using a SONOPULS sonication probe (5x 20 sec, 80% Int., Bandelin) with cooling breaks in between. Subsequently, lysis buffer II (188 µL, 100 mM Tris-HCl, 9.6% (w/v) SDS, 1.25% (w/v) sodium deoxycholate in in ddH2O, pH 7) was added to all samples, followed by an additional sonication step (1x 10 sec, 10% Int.). Samples were incubated (90°C, 10 min, 400 rpm) and subsequently centrifuged (13,000 g, 10 min, 4°C). Resulting supernatant was transferred into new Eppendorf tubes and protein concentration determined using the Roti®-Quant universal kit (Carl Roth) for BCA assay. Equal amounts of protein (300 µg) were transferred into Eppendorf LoBind tubes and proteins precipitated by addition of 4-fold volume excess of pre-chilled Acetone (-80°C) and incubation overnight at -20°C. Precipitated proteins were pelleted (21,000 g, 15 min, 4°C), supernatant discarded and resulting protein pellets resuspended in pre-chilled MeOH (500 µL, LC-MS grade, VWR) and centrifuged again. Subsequently, samples were resuspended in X-buffer (300 µL, 7 M urea, 2 M thiourea in 20 mM HEPES buffer pH 7.5) and proteins reduced by the addition of DTT (1 mM, 1 M stock in ddH2O) and incubated under gentle mixing (25Â°C, 45 min, 450 rpm). Alkylation of proteins was achieved by the addition of IAA (5.5 mM, 550 mM stock in ddH2O) and incubated under gentle mixing (25°C, 30 min, 450 rpm). Alkylation reaction was stopped by the addition of DTT (4 mM, 1 M stock in ddH2O) and incubated under gentle mixing (25Â°C, 30 min, 450 rpm). Proteins were predigested with LysC (3.33 Âµg/mL, 0.5 mg/mL stock in ddH20; Wacko) under gentle mixing (25°C, 2 h, 450 rpm). TEAB buffer (900 µL, 50mM, Sigma-Aldrich) was added, and samples further digested with trypsin (4 µL, 0.5 mg/mL stock, sequencing grade, modified; Promega) overnight under mixing (37°C, 800 rpm). Digest was stopped the following day by the addition of formic acid (0.1% final concentration; Sigma-Aldrich) and peptides desalted using Sep-Pak C18 1 cc Vac cartridges (Waters) and the following procedure: Resin was washed with MeCN (1 mL) and elution buffer (80% MeCN, 0.5% FA) prior to equilibration with 0.1% TFA (3 mL). Peptide solutions were loaded to the cartridges and bound peptides washed with 0.1% TFA (3 mL) and 0.5% FA (0.5 mL) and finally eluted with elution buffer (0.75 mL). Peptides were dried using a centrifugal vacuum concentrator (Eppendorf) and stored at -80°C. Prior to LC-MS/MS analysis dried peptide samples were reconstituted in 1% FA in ddH2O (120 µL), sonicated for 10 min and filtered using 0.22 µm Ultrafree-MC® centrifugal filters (Merck, UFC30GVNB), preequilibrated with 1% FA in ddH2O (125 ÂµL) and transferred into LC-MS vials. Secretome:Frozen sterile-filtered supernatants were freeze-dried in vacuo, and resulting pellet redissolved in H2O (150 µL, LC-MS grade, VWR). Debris was removed by centrifugation (21,000 g, 15 min, rt) and resulting supernatant transferred into new Eppendorf LoBind tubes. MeOH (600 µL, LC-MS grade, VWR) was added, samples briefly vortexed and centrifuged (13,000 rpm, 10 sec, rt). Next, Chloroform (225 µL, HPLC grade, Sigma Aldrich) was added and samples again vortexed and centrifuged. Subsequently H2O (450 µL) was added, samples briefly vortexed and sonicated for 7 min in a sonication bath. Subsequently, samples were centrifuged (13,000 rpm, 10 sec, rt) and upper liquid layer discarded. MeOH (450 µL) were added and samples again briefly vortexed and centrifuged (13,000 rpm, 20 min, rt). Resulting supernatant was discarded and protein pellet dried. Further processing of protein pellets was carried out as described for whole proteome analysis.

### Data Protocol
Samples were separated and analyzed with an Ultimate 3000 Nano HPLC system (Thermo Scientific) equipped with an Acclaim C18 PepMap100 75 µm ID x 2 cm trap column and an Acclaim PepMap C18 RSLC 75 µm ID x 50 cm separation column in an EASY-SprayTM setting, coupled to an LTQ Orbitrap Fusion (Thermo Scientific).  For peptide separation, samples were loaded on the trap column and washed for 10 min with 0.1% TFA in H2O at a flow rate of 5 µL/min. Subsequently, peptides were transferred to the analytical column for peptide separation and separated using the following 132 min gradient (Buffer A: H2O + 0.1% FA; B: MeCN + 0.1% FA) with a flowrate of 300 nL/min.: 7 min at 5%B, in 105 min from 5% to 22%, in 10 min from 22 to 35% and in another 10 min to 90% B. Separation gradient was followed by a column washing step using 90% B for 10 min and subsequent column re-equilibration with 5% B for 5 min. Peptides were ionized using an EASY-SprayTM source with variable voltage and a capillary temperature of 275°C. Orbitrap Fusion was operated in top speed mode with 3 s cycle time. Internal calibration was performed by using the ion signal of fluoranthene cations (EASY-ETD/IC source). MS full scans (300-1500 m/z) were performed in the orbitrap at a resolution of R = 120,000 (at 200 m/z) with an automatic gain control (AGC) ion target of 2e5 and a maximum injection time of 50 ms and RF Lens amplitude set to 60%. Monoisotopic precursor selection was enabeled and dynamic exclusion set to 60 s. Most intense peptides with charge states of 2-7 and an intensity higher than 5e3 were isolated in the quadrupole using a window of 1.6 m/z and subjected to higher-energy collisional dissociation (HCD collision energy set to 30%). MS2 ions were collected in the ion trap to a target of 1e4 for maximum injection time of 35 ms with inject ions for all available parallelizable time. Whole Proteome: MS raw data was analyzed using MaxQuant software (version 1.6.5.) with Andromeda search engine (searches were carried out against Uniprot database for Staphylococcus aureus NCTC 8325 (taxon identifier: 9606, downloaded on 07.01.2019, canonical). Carbamidomethlyation of cysteines (C) was set as fixed modifications and oxidation of methionines (M) as variable modifications. Trypsin (without N-terminal cleavage to proline) was set as proteolytic enzyme with a maximum of two allowed missed cleavages. For main search, precursor mass tolerance was set to 4.5 ppm and fragment mass tolerance to 0.5 Da. LFQ mode was activated with a LFQ minimum ratio count of 1. Second peptide identification was enabled, and FDR determination carried out by applying a decoy database and thresholds were set to 1% FDR at peptide-spectrum match and at protein levels and match between runs (0.7 min match and 20 min alignment time windows) option was enabled. Normalized LFQ intensities extracted from the MaxQuant result table proteinGroups.txt were further analyzed with Perseus software (version 1.6.3.2). Prior to analysis, putative contaminants, reverse hits and only identified by site hits were removed, and normalized LFQ intensities were log2 transformed. Proteins with a minimum of five valid values in at least one group were used for missing value imputation from normal distribution (width 0.3, downshift 1.8, total matrix). Two-sample Students t-test including Benjamini-Hochberg multiple testing correction (FDR = 0.05) was performed. Proteins with a ratio of 2 [log2(x) = 1] and -log10 t-test p-value of 1.3 were considered as significantly dysregulated proteins.  Secretome: MS raw data was analyzed as described for whole proteome data. Contaminants list implemented in MaxQuant was modified by the addition of the Saccharomyces cerevisiae fasta file (taxon identifier: 4932, downloaded on 31.03.2019, canonical), as medium used for bacterial growth contained pepton from yeast. Normalized LFQ intensities extracted from the MaxQuant result table proteinGroups.txt were further analyzed with Perseus (version 1.6.3.2). Prior to analysis, putative contaminants, reverse hits and only identified by site hits were removed, and normalized LFQ intensities were log2 transformed. Proteins with a minimum of five valid values in at least one group were used for missing value imputation from normal distribution (width 0.3, downshift 1.8, total matrix). For the comparison of expression levels of two bacterial strains, data was filtered for a minimum of five valid values in at least one group, prior to missing value imputation from normal distribution (width 0.3, downshift 1.8, total matrix). Two-sample Studentsâ€™s t-test including Benjamini-Hochberg multiple testing correction (FDR = 0.05) was performed. Proteins with a ratio of 2 [log2(x) = 1] and -log10 t-test p-value of 1.3 were considered as significantly dysregulated proteins.

### Publication Abstract
<i>Staphylococcus aureus</i> represents an opportunistic pathogen, which utilizes elaborate quorum sensing mechanisms to precisely control the expression and secretion of virulence factors. Previous studies indicated a role of the ClpXP proteolytic system in controlling pathogenesis. While detailed transcriptome data for <i>S. aureus</i> ClpP and ClpX knockout mutants is available, corresponding studies on the proteome and secretome level are largely lacking. To globally decipher the functional roles of ClpP and ClpX, we utilized <i>S. aureus</i> genomic deletion mutants of the corresponding genes for in-depth proteomic liquid chromatography-mass spectrometry (LC-MS)/MS analysis. These studies were complemented by an inactive ClpP active-site mutant strain to monitor changes solely depending on the activity and not the presence of the protein. A comparison of these strains with the wildtype revealed, e.g., downregulation of virulence, purine/pyrimidine biosynthesis, iron uptake, and stress response. Correspondingly, the integration of metabolomics data showed a reduction in the subset of purine and pyrimidine metabolite levels. Interestingly, a comparison between the ClpP knockout and ClpP S98A active-site mutant strains revealed characteristic differences. These results are not only of fundamental importance to understand the cellular role of ClpXP but also have implications for the development of novel virulence inhibitor classes.

### Keywords
Proteomics, Lc-ms/ms, Staphylococcus aureus, Whole proteome analysis

### Affiliations
Technical University of Munich 
Stephan A. Sieber, Center for Integrated Protein Science at the Department of Chemistry, Technische Universität München, Garching b. München 85747, Germany; Chair of Organic Chemistry II, Technische Universität München, Garching b. München 85747, Germany

### Submitter
Volker Kirsch

### Lab Head
Dr Stephan A. Sieber
Stephan A. Sieber, Center for Integrated Protein Science at the Department of Chemistry, Technische Universität München, Garching b. München 85747, Germany; Chair of Organic Chemistry II, Technische Universität München, Garching b. München 85747, Germany


