### Accession
PXD007174

### Title
Metaproteomics of viral concentrates at deep chlorophyll maximum of Scouth China Sea

### Description
Marine viral concentrates (VCs) contains a substantial amount of non-cellular biological particles, e.g. viruses, gene transfer agents (GTAs) and membrane vesicles that are ecological significant. Metagenomic sequencing of VCs has been extensively applied to study the diversity and function potential of natural virions whereas information of nonn-viral components are often excluded for investigation. Here we apply a shotgun  proteomic approach to characterize the origin and function of proteins in the VCs collected from the deep chlorophyll maximum (DCM) of the South China Sea. Using a custom database, we identified 636 non-redundant proteins represented by a total of 7220 spectra from the two VC samples. Cyanophages, pelagiphages, Phycodnaviridae and a group of uncultured viruses (previouly collected from DCM of Mediterranean Sea) contributed the most in the viral proteome. Seldom proteins related to RNA viruses and known GTAs were found despites of the presence of their sequences in the protein-searching database, suggested that these particles might be low abundant in the samples. Over 60% of identified spectra could not be assigned to viruses. The non-viral spectra were dominated by microbial groups of SAR324, SAR11, Actinobacteria and picoeukaryotic algae such as prasinophytes.Interestingly, we found that periplasmic proteins such as diverse ABC and TRAP transporters, and 56 kDa selenium-binding proteins, were enriched in this fraction.Together with other detected non-viral proteins,we could identify significant microbial functions, such as the utilization of glycine betaine, 3-dimethylsulphoniopropionate,and taurine by SAR11,and urea by prochlorococcus, nitrous oxide production by ammonia-oxidizing archaea and peroxide detoxification by unkonwn gammaproteobacteria. Our study of marine VCs demonstrates the potential application of metaproteomics to link the nano-size materials to the diversity of virions and interesting microbial functions in the ocean.

### Sample Protocol
Tangential flow filtration method was applied to collect viral concentrates. Crude proteins in the concentrated and desalted materials were precipitated with trichloroacetic acetone and solubilized in a buffer containing 7 M urea, 2M thiourea and 4% CHAPS. Purified proteins were digested via a filter-aided sample preparation method after reduction and alkylation. After loading the protein solution on a 10 kDa Microcon filtration device, the solution buffer was replaced with 100 uL of 0.5 M triethylammonium bicarbonate (TEAB) solution and rinsed three times. Centrifuged and discarded the flow-through. Digestion solution containing 1 ug of sequencing grade trypsin (Promega) in 100 uL 0.5 M TEAB solution were addedtwice at 37 â„ƒ overnight.The peptides were collected after centrifugation and dried for reconstitution with 2 mL buffer A (25 mM KH2PO4 in 25% acetonitrile, pH 3.0) and loaded onto a 4.6 x 250 mm Ultremex SCX column containing 5 um particles.Using a Shimadzu LC-20AB HPLC system, the peptides were eluted at a 1 mL/min flow rate with buffer A for 10 min, and then ramped from 5% to 35% buffer B (25 mM KH2PO4, 1M KCl in 25% acetonitrile for 30 min, and continued from 35% to 80% buffer B for 1 min. The system was then maintained in 80% buffer B for 3 min before equilibrating with buffer A for 10 min prior to the next injection. Fractions were collected every 1 min and the eluted peptides were finally pooled into 10 fractions, desaulted with a Strata X C18 column (phenomenex) and vacumm-dried.Buffer C (5% acetonitrile, 0.1% formic acid) was used to suspend peptides. After loading on a Shimadzu LC-20AD nano-HPLC, the peptides were eluted onto an in-house packed C18 column (inner diameter 75 um, 18 cm) and injected into a online coupling Q Exactive mass spectrometer (ThermFisher Scientific) with buffer D (95% acetonitrile, 0.1% formic acid) in a linear gradient from 2% to 80% for nanoelectrospray ionizationn.

### Data Protocol
The instument data file for each fraction was merged and transformated to mgf file using Proteome Discoverer (Ver. 1.3.0.339, ThermoFisher Scientific). Mascot serch engine (Ver. 2.3.0, Matrix Science) was used to identify peptide and protein via searching against a custome database including a local DCM metagenomics (SEATS-DCM-Bac, 219878 sequences), a Pacific Ocean Viromic dataset(Hurwitz and Sullivan, 2013), a metaviromic dataset from DCM of Mediterranean Sea (Mizuno et al., 2013), and protein sequences of ds-, ss-RNA viruses and retro-transcribing viruses downloaded from NCBI on  July 27th, 2014. The following parameters were used to perform the Mascot searching: parent mass tolerance, 20 ppm; fragment ion tolerance, 0.6 Da;fully tryptic peptide only; one missed cleavage allowed; fixed modification of carboxymethyl cysteine (+58.00 Da) and variable modification of methionion oxidation (+16.00 Da); peptide charge states with +1, +2 and +3; Criteria for acceptance of peptide assignmentts: 95% probability or higher; a strict FDR of 0.01 applied in a decoy database search; at least 2 unique peptides per protein. The same parameters were used for a second database searching, of which contained the SEAT-DCM-Bac metagenomic dataset, a coastal RNA viral metagenomic dataset (Steward et al., 2013) and 9597 refernce protein sequences of GTA from NCBI. This database were only used for evaluating the contribution of environmental RNA viruses and known GTAs.

### Publication Abstract
Viral concentrates (VCs), containing bioinformative DNA and proteins, have been used to study viral diversity, viral metagenomics and virus-host interactions in natural ecosystems. Besides viruses, VCs also contain many noncellular biological components including diverse functional proteins. Here, we used a shotgun proteomic approach to characterize the proteins of VCs collected from the oligotrophic deep chlorophyll maximum (DCM) of the South China Sea. Proteins of viruses infecting picophytoplankton, that is, cyanobacteria and prasinophytes, and heterotrophic bacterioplankton, such as SAR11 and SAR116, dominated the viral proteome. Almost no proteins from RNA viruses or known gene transfer agents were detected, suggesting that they were not abundant at the sampling site. Remarkably, nonviral proteins made up about two thirds of VC proteins, including overwhelmingly abundant periplasmic transporters for nutrient acquisition and proteins for diverse cellular processes, that is, translation, energy metabolism and one carbon metabolism. Interestingly, three 56 kDa selenium-binding proteins putatively involved in peroxide reduction from gammaproteobacteria were abundant in the VCs, suggesting active removal of peroxide compounds at DCM. Our study demonstrated that metaproteomics provides a valuable avenue to explore the diversity and structure of the viral community and also the pivotal biological functions affiliated with microbes in the natural environment.

### Keywords
56 kda selenium-binding protein, Metaproteomics, Virua, Viral concentrates

### Affiliations
State Key Laboratory of Marine Environmental Science / College of the Environment and Ecology, Xiamen University, China
State Key Laboratory of Marine Environmental Science/College of the Environment and Ecology, Xiamen University, China

### Submitter
Zhangxian Xie

### Lab Head
Dr Da-Zhi Wang
State Key Laboratory of Marine Environmental Science / College of the Environment and Ecology, Xiamen University, China


