### Accession
PXD032908

### Title
Validating pyAscore, A Python Package for the Localization of Protein Modifications

### Description
The submitted dataset contains results of validation experiments for pyAscore, an efficient and versatile implementation of the Ascore algorithm in Python for scoring the localization of user defined PTMs in data dependent mass spectrometry. In order to test the versatility of the package, three samples from three pride accessions were re-analyzed: label-free phosphoprotome samples were downloaded from PXD007740, TMT-labeled phosphoproteome samples were downloaded from PXD007145, and label free acetylome samples were downloaded from MSV000079068. We also wanted to evaluate the performance of pyAscore's PTM localizations on data where the modification site of peptides was known. Thus, we downloaded and re-analyzed synthetic phosphopeptide samples which were analyzed with 3 different fragmentation approaches: HCD and ETD synthetic phosphopeptide data were downloaded from PXD000138 and CID data were downloaded from PXD000759.

### Sample Protocol
Information about sample processing can be found in each sample's original publication.

### Data Protocol
Mass spectrometry data for individual experiments was downloaded directly from MassIVE or PRIDE, converted to mzML format with ThermoRawFileParser (v. 1.3.4) and then searched with the Comet database search software (v. 2021010). Recommended parameters for high resolution and low resolution searches were taken from the Comet documentation. Human samples were searched with the uniprot Homo sapiens reference proteome (downloaded Feb 8, 2022), and all synthetic peptide data was searched with the FASTA from PXD000138. All files were searched with carbamidomethylation on cysteines as static modification and oxidized methionine as variable modification. Since dataset PXD007145 reports TMT-labeled peptides, a TMT 10-plex modification was added as a static modification on lysines and peptide N-termini. For the synthetic peptide datasets, PXD000138 and PXD000759, and the other phosphoproteomic datasets, PXD007740 and PXD007145, the modification of interest was phosphorylation on serine, threonine, and tyrosine, and this was included in the variable modification list. For dataset MSV000079068, acetylation of the protein N-terminus and internal lysines was included as a variable modification, but peptide C-terminal lysines were not allowed to be acetylated. Finally, HCD and CID data were searched with the b and y ion series, whereas ETD data was searched with the c and z+H ion series. All searches were subsequently grouped by dataset and fragmentation method and analyzed with Mokapot (v. 0.7.4) using default parameters. For pyAscore localization, analyses marked as narrow were run with mz_error set to 0.05 Da, and analyses marked as wide were run with mz_error set to 0.5 Da. In both cases, the same ion series as the searches was used.

### Publication Abstract
Determining the correct localization of post-translational modifications (PTMs) on peptides aids in interpreting their effect on protein function. While most algorithms for this task are available as standalone applications or incorporated into software suites, improving their versatility through access from popular scripting languages facilitates experimentation and incorporation into novel workflows. Here we describe pyAscore, an efficient and versatile implementation of the Ascore algorithm in Python for scoring the localization of user defined PTMs in data dependent mass spectrometry. pyAscore can be used from the command line or imported into Python scripts and accepts standard file formats from popular software tools used in bottom-up proteomics. Access to internal objects for scoring and working with modified peptides adds to the toolbox for working with PTMs in Python. pyAscore is available as an open source package for Python 3.6+ on all major operating systems and can be found at pyascore.readthedocs.io.

### Keywords
Human, Synthetic, Phosphorylation, Ptm, Acetylation

### Affiliations
Department of Genome Sciences, University of Washington, Seattle, WA 98195, USA

### Submitter
Anthony Barente

### Lab Head
Dr Judit Vill√©n
Department of Genome Sciences, University of Washington, Seattle, WA 98195, USA


