### Accession
PXD010027

### Title
Mutations in disordered regions cause disease by creating dileucine motifs - peptide-protein interaction screen

### Description
Many disease-causing missense mutations affect intrinsically disordered regions (IDRs) of proteins. Since these mutations do not affect protein structure, the molecular mechanism of their pathogenicity is enigmatic. Here, we employ a peptide-based proteomic screen to investigate the impact of mutations in IDRs on protein-protein interactions. We find that mutations in disordered cytosolic regions of three transmembrane proteins (GLUT1, ITPR1 and CACNA1H) lead to an increased binding of clathrin. In all three cases, the mutation creates a dileucine motif known to mediate clathrin-dependent trafficking. Follow-up experiments on full length GLUT1 (SLC2A1), the glucose transporter causative of GLUT1 deficiency syndrome, revealed that the mutated protein mislocalizes to intracellular compartments in a model cell line and in patient-derived induced pluripotent stem cells. Mutant GLUT1 interacts with adaptor proteins (APs) in vitro, and knocking-down AP-2 reverts the cellular mislocalization. A systematic analysis of other known disease-causing variants revealed a significant and specific overrepresentation of gained dileucine motifs in structurally disordered cytosolic domains of transmembrane proteins. Thus, several mutations in disordered regions appear to cause “dileucineopathies”.

### Sample Protocol
Peptides of 15 AAs, in total 128 wild-type peptide and 128 related peptides containing the disease causing mutation (256 peptides) plus one control peptide pair were synthesized in situ on cellulose membrane using PepTrack™ techniques (JPT Peptide Technologies, Berlin, Germany). Two of those peptide filters were moistened in cell lysis buffer [50 mM HEPES pH 7.6 at 4 °C, 150 mM  NaCl, 1 mM EGTA, 1 mM MgCl2, 10% Glycerol, 0.5% Nonidet P-40, 0.05% SDS and 0.25% sodium deoxycholate, supplemented with protease inhibitor (Roche) and benzonase (Merck)]. In order to reduce nonspecific binding the membrane was incubated with 1 mg/ml yeast t-RNA (Invitrogen) for 10 min and then washed twice with cell lysis buffer. The entire peptide libraries were incubated with 15 ml of light or heavy SILAC labeled cell lysate (5 mg/ml) from SH-SY5Y cells for 2 h. Membranes were washed three times and air dried. Sample preparation for mass spectrometric analysis Single spots were punched out from cellulose membrane with a 2 mm diameter ear punch (Carl Roth) and SILAC pairs were placed together in a 96-well plate (Thermo Scientific) prepared with 30 µl of denaturation buffer [6 M urea (Sigma-Aldrich), 2 M thiourea (Sigma-Aldrich), 10 mM HEPES, pH 8]. Samples were reduced by incubating with 10 µl of 3.3 mM DTT (Sigma-Aldrich) for 30 min at RT, followed by an alkylation step using 10 µl of 18.3 mM iodoacetamide (IAA) (Sigma-Aldrich) for 60 min at RT. The samples were first digested using 1 µg endopeptidase LysC (Wako, Osaka, Japan) for 4 h. The samples were diluted by adding 100 µl of 50 mM ammonium bicarbonate (pH = 8.5), and finally digested with 1 µg trypsin (Promega) for 16 h. The digestion was stopped by acidifying each sample to pH < 2.5 by adding 10% trifluoroacetic acid solution. The peptide extracts were purified and stored on stage tips according to (Rappsilber et al., 2003).  LC-MS/MS analysis Peptides were eluted using Buffer B (80% Acetonitrile and 0.1% formic acid) and organic solvent was evaporated using a speedvac (Eppendorf). Samples were diluted in Buffer A (5% acetonitrile and 0.1% formic acid). Peptides were separated on a reversed-phase column with 45 min gradient with a 250 nl/min flow rate of increasing Buffer B concentration on a High Performance Liquid Chromatography (HPLC) system (ThermoScientific). Peptides were ionized using an electrospray ionization (ESI) source (ThermoScientific) and analyzed on a Q-exactive plus Orbitrap instrument (ThermoScientific). Dynamic exclusion for selected precursor ions was 30 s. The mass spectrometer was run in data dependent mode selecting the top 10 most intense ions in the MS full scans, selecting ions from 300 to 1700 m/z (Orbitrap resolution: 70,000; target value: 1,000,000 ions; maximum injection time of 120 ms). The resulting MS/MS spectra from the Orbitrap had a resolution of 17,500 after a maximum ion collection time of 60 ms with a target of reaching 100,000 ions.

### Data Protocol
Raw files were analyzed using MaxQuant software version 1.5.2.8 (Cox and Mann, 2008). Default settings were kept except that ‘match between runs’ and ‘re-quantify’ was turned on. Lys8 and Arg10 were set as labels and oxidation of methionines and n-term acetylation (variable modifications), carbamidomethyl of cysteines(fixed modification). The in silico digests of the human Uniprot database (2015-12), a fasta file containing all peptides used for pull-down and a database containing common contaminants were done with Trypsin/P. The false discovery rate was set to 1% at both the peptide and protein level and was assessed by in parallel searching a database containing the reversed sequences from the Uniprot database. Experimental setup: Fw: Light- Wt, Heavy-Mut Rev: Light-Mut, Heavy-Wt  Fw Rev ID 1 2 SOS1_cntrl 3 4 Q9NRG9_Q15K 5 6 P55265_G1007R 7 8 P55265_K999N 9 10 P13637_D220N 11 12 O00555_R2135C 13 14 O00555_P897R 15 16 O95180_A748V 17 18 O95180_P648L 19 20 O95180_G499S 21 22 O14936_P396S 23 24 P41180_A988V 25 26 P41180_R898Q 27 28 Q9P2K1_P1122S 29 30 Q9P2K1_T1114M 31 32 O76039_V718M 33 34 O76039_N399T 35 36 O15078_W7C 37 38 Q8WYQ3_P34S 39 40 Q9UQN3_T104N 41 42 Q9UQN3_Q206H 43 44 Q9UQN3_D148Y 45 46 Q9NWW5_R6T 47 48 Q9UBY8_E269V 49 50 P14868_R494G 51 52 Q6PI48_Y629C 53 54 O75140_S1073R 55 56 O75140_S1162G 57 58 O75140_A452V 59 60 O60832_G402R 61 62 O60832_P409L 63 64 Q5JPH6_E96K 65 66 Q5JPH6_R108W 67 68 Q5JPH6_R168G 69 70 P11161_I268N 71 72 Q13144_S447L 73 74 Q04637_R1205H 75 76 Q92889_G513R 77 78 Q03468_P1042L 79 80 Q96B26_S272T 81 82 P35637_R521C 83 84 P35637_G507D 85 86 P35637_R216C 87 88 ARHGAP36_cntrl_1 89 90 Q99259_S12C 91 92 P54803_Y490N 93 94 P04062_R87W 95 96 Q92947_M191T 97 98 P30793_P23L 99 100 P30793_R249S 101 102 Q6Y7W6_T112A 103 104 Q6Y7W6_V1242I 105 106 Q6Y7W6_D606E 107 108 P08034_R264C 109 110 P08034_C280G 111 112 P08034_R230C 113 114 Q12879_D1251N 115 116 Q14CZ8_R288C 117 118 P00492_V8G 119 120 P04792_T164A 121 122 P04792_T151I 123 124 Q96RY7_E664K 125 126 P38935_D974E 127 128 Q9Y6K9_Q183H 129 130 Q14643_P1059L 131 132 P23352_H672R 133 134 Q2M1P5_P632L 135 136 P32004_S1194L 137 138 Q99732_T49M 139 140 P10636_L583V 141 142 P10636_V654M 143 144 P10636_R5L 145 146 P43243_T622A 147 148 P51608_P225R 149 150 P51608_G161V 151 152 P51608_E10Q 153 154 Q71SY5_A335V 155 156 P25189_A221T 157 158 Q9BZ23_E134G 159 160 O60260_A82E 161 162 O60260_A92V 163 164 Q9BXM7_P196L 165 166 O60733_R632W 167 168 Q8IY17_R1350W 169 170 P54098_S511N 171 172 P54098_T251I 173 174 P54098_G517V 175 176 P04156_P102L 177 178 Q06124_Y279C 179 180 Q06124_E139D 181 182 Q06124_L564F 183 184 P04049_R256S 185 186 P51812_R729Q 187 188 P35498_L1514S 189 190 P35498_R27T 191 192 P35498_F63L 193 194 Q9Y6X0_G870S 195 196 Q9Y6X0_I871T 197 198 ARHGAP36_cntrl_2 199 200 Q9Y6X0_D868N 201 202 Q7Z333_Q653K 203 204 Q7Z333_R1294C 205 206 P11166_P485L 207 208 P17405_A196P 209 210 P00441_D77Y 211 212 P00441_I152T 213 214 P00441_G86R 215 216 Q9UBP0_P293L 217 218 Q69YQ0_T397P 219 220 O95630_R14P 221 222 Q9UNE7_E28K 223 224 P51687_S427Y 225 226 Q13148_G295S 227 228 Q13148_G348C 229 230 Q13148_G290A 231 232 P15884_R565W 233 234 P15884_D535G 235 236 P15884_G358V 237 238 Q92734_P285L 239 240 Q9BSI4_K280E 241 242 Q5SV17_P170L 243 244 Q9HBA0_R315W 245 246 P19971_R44Q 247 248 P22314_S547G 249 250 Q9UHD9_A283T 251 252 Q9UHD9_P525S 253 254 Q5ST30_A349T 255 256 O76024_A58V 257 258 Q9NZC7_P47T 259 260 Q9NQZ6_R213W 261 262 O60315_Q1119R 263 264 WIP_cntrl  B blank

### Publication Abstract
Many disease-causing missense mutations affect intrinsically disordered regions (IDRs) of proteins, but the molecular mechanism of their pathogenicity is enigmatic. Here, we employ a peptide-based proteomic screen to investigate the impact of mutations in IDRs on protein-protein interactions. We find that&#xa0;mutations in disordered cytosolic regions of three transmembrane proteins (GLUT1, ITPR1, and CACNA1H) lead to an increased clathrin binding. All three mutations create dileucine motifs known to mediate clathrin-dependent trafficking. Follow-up experiments on GLUT1 (SLC2A1), the glucose transporter causative of GLUT1 deficiency syndrome, revealed that the mutated protein mislocalizes to intracellular compartments. Mutant GLUT1 interacts with adaptor proteins (APs) in&#xa0;vitro, and knocking down AP-2 reverts the cellular mislocalization and restores glucose transport. A systematic analysis of other known disease-causing variants revealed a significant and specific overrepresentation of gained dileucine motifs in structurally disordered cytosolic domains of transmembrane proteins. Thus, several mutations in disordered regions appear to cause "dileucineopathies."

### Keywords
Disordered regions, Protein-protein interaction, Ppi, Peptide array

### Affiliations
Proteome Dynamics, Max Delbrück Center for Molecular Medicine, Robert-Rössle-Str. 10, D-13092 Berlin, Germany  Charité-Universitätsmedizin Berlin, 10117 Berlin, Germany
MDC

### Submitter
Katrina Meyer

### Lab Head
Dr Matthias Selbach
Proteome Dynamics, Max Delbrück Center for Molecular Medicine, Robert-Rössle-Str. 10, D-13092 Berlin, Germany  Charité-Universitätsmedizin Berlin, 10117 Berlin, Germany


