### Accession
PXD020760

### Title
A vascular phosphoproteome atlas unveils a functional Tie‑Wnt signalling axis in the liver

### Description
Single cell transcriptomic analyses (scRNAseq) of hepatocytes and liver endothelial cells (L-EC) have revolutionized the understanding of the spatial architecture of liver structure and function. The spatial alignment of L-EC and hepatocytes is pivotal for liver function in health and disease given that L-EC act as instructive gatekeeper of nearby hepatocytes including the maintenance of liver metabolic zonation in a Wnt-dependent manner. Advancing liver biology beyond the ’transcript-centric’ view of scRNAseq analyses is presently restricted by the limited resolution of proteomics and genome-wide techniques to analyse post-translational modifications. Here, by combining spatial cell sorting methodology with transcriptomic and quantitative proteomic/phospho-proteomic analyses, we established the first functionally and spatially-resolved proteome landscape of the liver endothelium, yielding deep mechanistic insight into zonated vascular signalling mechanisms. Phosphorylation of receptor tyrosine kinases (RTK) was detected preferentially in the central vein area resulting in an atypical enrichment of tyrosine phosphorylation. Prototypic biological validation of the identified strong phosphorylation gradient of the vascular RTK Tie1 by blockade resulted in the rapid peri-central dysregulation of the L-EC transcriptome. Notably, the expression of Wnt9b in L-EC was discovered as Tie receptor controlled with reciprocal regulation by FoxO1 and STAT3 transcription factors. Genetic inactivation of Tie1 in L-EC or antibody blockade resulted in reduced liver regeneration following partial hepatectomy with reduced Wnt ligand and Wnt target gene expression, including Axin2, Sox9, Tbx3 and Lgr5. Taken together, the study has yielded unparalleled insight into the spatial organization of L EC signalling and discovered a vascular Tie-Wnt signalling axis as regulator of liver function. The employed spatial sorting technique followed by phospho-proteomic analysis may be employed as a universally adaptable strategy for the spatial phosphoproteomic analysis of scRNAseq data-defined relevant cellular (sub)-populations.

### Sample Protocol
Protein was extracted with RIPA lysis and extraction buffer (Thermo Fisher Scientific) supplemented with phosphatase inhibitor and protease inhibitor mix then processed via tryptic in-gel digestion. Nanoflow LC-MS/MS was performed by coupling a Dionex 3000 (Thermo Fisher Scientific) to a QExactive Orbitrap HF-X (Thermo Fisher Scientific).

### Data Protocol
Data analysis was carried out by MaxQuant (version 1.6.3.3). Carbamidomethylation of cysteines was set as fixed modification. Phosphorylation of serine, threonine or tyrosine as well as oxidation of methionines and N-terminal acetylation were set as variable modifications. Identification FDR cutoffs were 0.01 on peptide level and 0.01 on protein level. The LFQ based full proteome analysis was carried out with organism specific UniProt database UP000000589 (Mus musculus; Dec, 2017; 60715 sequences) and enabled ‘match-between-runs’ function. The LFQ option was enabled and left at default parameters.

### Publication Abstract
None

### Keywords
Spatial sorting, Zonation, Phospho proteome, Liver, Endothelial, Multi-omics, Vasculature

### Affiliations
Division of Vascular Oncology and Metastasis Research,German Cancer Research Center (DKFZ), Heidelberg, Germany.  European Center for Angioscience (ECAS), Medical Faculty Mannheim, Heidelberg University, Mannheim, Germany.
German Cancer Research Center

### Submitter
Jingjing Shi

### Lab Head
Dr Hellmut G. Augustin
Division of Vascular Oncology and Metastasis Research,German Cancer Research Center (DKFZ), Heidelberg, Germany.  European Center for Angioscience (ECAS), Medical Faculty Mannheim, Heidelberg University, Mannheim, Germany.


