### Accession
PXD018148

### Title
¬¬Quantitative proteomic comparison of salt stress in Chlamydomonas reinhardtii and snow alga Chlamydomonas nivalis reveals mechanisms for salt-triggered lipid accumulation via reallocation of carbon resources

### Description
We characterised physiological and proteomic changes between C. reinhardtii and the snow alga Chlamydomonas nivalis, to reveal insights into their contrasting responses to salinity stress

### Sample Protocol
This study aimed to first quantify any lipid accumulation in response to salt stress in both Chlamydomonas species, and then quantify the associated proteomic shifts to provide insights into the lipid trigger. 8-plex iTRAQ (isobaric tags for relative and absolute quantification) was selected for the ability to simultaneously identify and relatively quantify proteins from up to 8 samples, thus enabling inclusion of biological replicates for 4 sample conditions. Labelled peptides were then fractrionated via Hypercab column befored being submitted to aThermo Q Exactive HF coupled with U300 RSL nano LC. The mass spectrometer was programmed for data dependent acquisition with 10 product ion scans (resolution 15000, automatic gain control 5e4, maximum injection time 20 ms, isolation window 1.2 Th, normalised collision energy 32, intensity threshold 2.5e5) per full MS scan (resolution 60000, automatic gain control 3e6, maximum injection time 100 ms).

### Data Protocol
Six fraction files obtained from each iTRAQ experiment were processed in data analysis software PEAKS7® to enable de novo sequencing, variable post translational modification (PTM) searching and homology searching. The data for Chlamydomonas reinhardtii and C. nivalis  were each searched against the Chlamydomonas reinhardtii Uniprot proteome database (taxa id: 3055, downloaded 6 May 2016, 15,172 entries), using the following settings: Digestion type: trypsin; Variable modifications: Oxidation (M); fixed modifications: MMTS; MS scan type: MS2; peptide-to-spectrum matches (PSM) FDR 0.01; Protein FDR 0.01; Site FDR 0.01; labelling: iTRAQ 8-plex; MS tolerance 0.2 Da; MS/MS tolerance 0.2 Da; label mass tolerance 0.01 Da; database: Uniprot proteome database for Chlamydomonas reinhardtii (taxa id: 3055, downloaded 6 May 2016, 15,172 entries); min peptide length 6; max peptide length 4600; max mis-cleavages 2; maximum charge state 7; min number of unique peptides 1. Data was run through in-house programs to integrate peptide quantification information to the protein level and to perform analysis of variance (ANOVA) based statistical analysis as previously described. A false discovery rate (FDR) of 1% was applied with a requirement of identifying at least two unique peptides per protein identification and p<0.05. The accession numbers of proteins increased and decreased in abundance were searched against the Uniprot database (www.uniprot.org) to identify the function of the proteins based on Gene Ontology annotation. KEGG analysis was carried out using the "Search&Color Pathway" tool (data shown in Supplementary Materials).

### Publication Abstract
None

### Keywords
Lipid production, Quantitative proteomics, Salt stress, Chlamydomonas nivalis, Chlamydomonas reinhardtii, Biofuel, Itraq

### Affiliations
Department of Chemical and Biological Engineering, University of Sheffield, Mappin Street, Sheffield S1 3JD, United Kingdom
Department of Chemical and Biological Engineering

### Submitter
Trong Khoa Pham

### Lab Head
Dr Trong Khoa Pham
Department of Chemical and Biological Engineering, University of Sheffield, Mappin Street, Sheffield S1 3JD, United Kingdom


