### Accession
PXD014449

### Title
Qualitative proteomic comparison of metabolic and CM-like protein fractions in old and modern wheat Italian genotypes by a shotgun approach

### Description
The close relationship between diet and health is generally recognized and the growing wellness and consciousness, especially in developed countries, have led to increasing interest for old wheat genotypes, based on perceived health benefits. Although nutritional comparison between old and modern wheat varieties is still controversial, it is generally accepted that old wheat genotypes remained unchanged over the last hundred years. By contrast, modern wheat genotypes are derived by modification of old wheats during the so-called “Green-Revolution” in the second half of the 20th century focusing on obtaining properties in terms of higher grain yield. The present work reports the first comprehensive proteomic profiling and qualitative comparison at the molecular level of metabolic and Chloroform-Methanol (CM)-like protein fractions extracted from mature kernels of two old Sicilian durum wheat landraces, Russello and Timilia Reste Bianche, and Simeto, an improved durum wheat variety widespread in Italy and other Mediterranean countries and chosen as representative of the most widely commercial cultivars. The results obtained reveal that metabolic and CM-like protein fractions of old and modern genotypes present remarkably high similarity with only minor differences. This leads to the conclusion that from a food and nutritional perspective there is a substantial equivalence of the protein composition of the old and modern cultivars.

### Sample Protocol
• The metabolic and CM-like protein fractions were extracted from three biological replicates of Russello, Timilia RB and Simeto. Flours (200 mg) were suspended in 2 mL cold (4°C) extraction solution (50 mM Tris-HCl, 100 mM KCl, 5 mM EDTA, Protease Inhibitor Cocktail, pH 7.8) in order to obtain the metabolic and CM-like proteins. The solution was incubated on ice (5 min) with intermittent mixing and centrifuged (13523 g, 15 min, 4°C). The supernatant was collected and added five volumes of 0.1 M ammonium acetate in methanol. Following incubation overnight at -20°C, the solution was centrifuged (30 g, 15 min, R.T.). The pellet (containing the metabolic proteins) was collected and rinsed in 3 mL 0.1 M ammonium bicarbonate, pH 8.2. Proteins in the supernatant (CM-like proteins) were precipitated by addition of four volumes of cold acetone, kept overnight at -20°C and subsequently centrifuged (30 g, 15 min, R.T.). Finally, the pellet containing CM-like proteins was rinsed by 1.5 mL 0.1 M ammonium bicarbonate, pH 8.2. The protein concentration for each extract was determined by a fluorimetric assay using the Qubit Protein Assay kit with the Qubit 1.0 Fluorometer (ThermoFisher Scientific, Milan, Italy). Finally, 40 µL (corresponding to about 60 µg) of each extract were reduced by adding 40 µg of DTT dissolved in the same buffer (3 h, 20°C), alkylated with 96 µg of IAA (1 h, in the dark at 20°C) and digested by porcine trypsin (Sequencing Grade Modified Trypsin, Porcine, lyophilized, Promega) at an enzyme-substrate ratio of 1:50 (overnight, 37°C). Mass spectrometry data were acquired on a Thermo Fisher Scientific Orbitrap Fusion Tribrid® (Q-OT-qIT) mass spectrometer (Thermo Fisher Scientific, Bremen, Germany). Liquid chromatography was carried out using a Thermo Scientific Dionex UltiMate 3000 RSLCnano system (Sunnyvale, CA). One microliter of peptide mixture was loaded onto an Acclaim ®Nano Trap C18 Column (100 µm i.d. x 2 cm, 5 µm particle size, 100 Å). After washing the trapping column with solvent A (H2O + 0.1% FA) for 3 min at a flow rate of 7 μL/min, the peptides were eluted from the trapping column onto a PepMap® RSLC C18 EASY-Spray column (75 µm i. d. x 50 cm, 2 µm particle size, 100 Å) and separated by elution at a flow rate of 0.25 µL/min at 40°C by a linear gradient of solvent B (ACN + 0.1% FA) in A, 5% for 3 min, followed by 5% to 20% in 32 min, 20% to 40% in 30 min, 40% to 60% in 20 min and 60% to 98% in 15 min, finishing by holding 98% B 5 min, 98% to 5% in 1 min. and re-equilibrating at 5% B for 20 min. The eluting peptide cations were converted to gas-phase ions by electrospray ionization using a source voltage of 1.75 kV and introduced into the mass spectrometer through a heated ion transfer tube (275 °C). Survey scans of peptide precursors from 200 to 1600 m/z were performed at 120 K resolution (@ 200 m/z). Tandem MS was performed by isolation at 1.6 Th with the quadrupole, HCD fragmentation with a normalized collision energy of 35, and rapid scan MS analysis in the linear ion trap (low resolution MS/MS analysis). Only those precursors with charge state 2÷4 and intensity above the threshold of 5∙103 were sampled for MS2. The dynamic exclusion duration was set to 60 s with a 10 ppm tolerance around the selected precursor and its isotopes. Monoisotopic precursor selection was turned on. The instrument was run in top speed mode with 3 s cycles, meaning it would continuously perform MS2 events until the list of non-excluded precursors diminished to zero or 3 s, whichever is shorter. MS/MS spectral quality was enhanced enabling the parallelizable time option (i.e. by using all parallelizable time during full scan detection for MS/MS precursor injection and detection). Mass spectrometer calibration was performed by using the Pierce® LTQ Velos ESI Positive Ion Calibration Solution (Thermo Fisher Scientific). MS data acquisition was carried out by utilizing the Xcalibur v. 3.0.63 software (Thermo Fisher Scientific).

### Data Protocol
• MS data were processed using PEAKS de novo sequencing software (v. 8.5, Bioinformatics Solutions Inc., Waterloo, ON Canada). Data were searched against a dedicated protein database (7612 protein sequences), including only the reviewed entries of Triticum, Oryza, Hordeum, Avena, Secale, Maize and Brachypodium species downloaded from the UniProt database (release July 2018). The common Repository of Adventitious Proteins (c-RAP) contaminant database was included in the database search. Database search was carried out using the following parameters: i) full tryptic peptides with a maximum of 3 missed cleavage sites; ii) cysteine carbamidomethylation as a fixed modification; iii) oxidation of methionine, the transformation of N-terminal glutamine and N-terminal glutamic acid residue to pyroglutamic acid form as variable modifications. The precursor mass tolerance threshold was set to 10 ppm and the maximum fragment mass error was set to 0.6 Da. Peptide spectral matches (PSM) were validated using a Target Decoy PSM Validator node based on q-values at a 0.1% False Discovery Rate (FDR). PEAKS score thresholds for Peptide spectral matches (PSMs) were set in order to achieve for each database search FDR values for PSMs, Peptide sequences and Proteins identified below the 0.1% value. This resulted in a range for PEAKS score thresholds from 39 to 47.  A protein was considered identified if a minimum of two peptides were matched. Proteins containing the same peptides and that could not be differentiated based on MS/MS analysis alone were grouped to satisfy the principles of parsimony (groups of parsimony). In these cases, proteins from Triticum, when identified, were always chosen as the group’s reference protein. When a group of parsimony did not contain a component from Triticum, the reference protein was selected from the species closest related to Triticum and represented in the group.

### Publication Abstract
The close relationship between diet and health is generally recognized and the growing wellness and consciousness, especially in developed countries, have led to increasing interest for old wheat genotypes, based on perceived health benefits. Although nutritional comparison between old and modern wheat varieties is still controversial, it is generally accepted that old wheat genotypes remained unchanged over the last hundred years. By contrast, modern wheat genotypes are derived by modification of old wheats during the so-called "Green-Revolution" in the second half of the 20th century focusing on obtaining properties in terms of higher grain yield. The present work reports the first comprehensive proteomic profiling and qualitative comparison at the molecular level of metabolic and Chloroform-Methanol (CM)-like protein fractions extracted from mature kernels of two old Sicilian durum wheat landraces, Russello and Timilia Reste Bianche, and Simeto, an improved durum wheat variety widespread in Italy and other Mediterranean countries and chosen as representative of the most widely commercial cultivars. The results obtained reveal that metabolic and CM-like protein fractions of old and modern genotypes present remarkably high similarity with only minor differences. This leads to the conclusion that from a food and nutritional perspective there is a substantial equivalence of the protein composition of the old and modern cultivars. Data are available via ProteomeXchange with identifier PXD014449. BIOLOGICAL SIGNIFICANCE: In recent years consumers have shown growing interest in the old wheat genotypes, which are generally perceived more "natural" and healthier than modern ones. However, comparison of nutritional value for modern and old wheat varieties is still controversial suggesting further studies. In particular proteome analysis of old and modern wheat genotypes is currently ongoing with particular focus on gluten proteins, whereas the metabolic protein fraction has not yet been investigated. In the present study, we conducted a comprehensive proteomic profile and qualitative comparison at the molecular level of metabolic and Chloroform-Methanol (CM)-like protein fractions of the old Sicilian landraces Russello and Timilia Reste Bianche and the modern cultivar Simeto by applying a shotgun approach. The results reveal that the metabolic and CM-like protein fractions of old and modern genotypes are remarkably similar with only minor differences, leading to the conclusion that from a food and nutritional perspective there is a substantial equivalence of these cultivars. These results may contribute to improved understanding of the relationship between protein profiles of old wheat genotypes and their potential benefits for human consumption.

### Keywords
Old and modern wheat genotypes; high resolution mass spectrometry; proteome analysis; proteins; allergies; diet and health

### Affiliations
Laboratory of Organic Mass Spectrometry, Department of Chemical Sciences, University of Catania, Viale A. Doria 6, 95125, Catania, Italy
University of Catania

### Submitter
Antonella Di Francesco

### Lab Head
Dr Salvatore Foti
Laboratory of Organic Mass Spectrometry, Department of Chemical Sciences, University of Catania, Viale A. Doria 6, 95125, Catania, Italy


