### Accession
PXD028554

### Title
Mechanisms associated with systemic effects of cancer

### Description
Cancer is considered as a disease of a specific organ, but its effects are felt throughout the body. The systemic effects of cancer can lead to weakness in muscles and heart, which hastens cancer-associated death. miR-486 is a myogenic microRNA and its reduced expression in skeletal muscle is observed in muscular dystrophy. Muscle-specific transgenic expression of miR-486 using muscle creatine kinase promoter (MCK-miR-486) partially rescues skeletal muscle defects in muscular dystrophy animal models. We had previously demonstrated reduced circulating and skeletal muscle levels of miR-486 in several cancer types and lower miR-486 levels correlated with skeletal muscle defects and functional limitations in mammary tumor models. Therefore, skeletal muscle defects induced by cancer could resemble defects observed in various dystrophies, which could be reversed through skeletal muscle expression of miR-486. We performed functional limitations studies and biochemical analysis of skeletal muscles of MMTV-Neu transgenic mice that mimic HER2+ breast cancer and MMTV-PyMT transgenic mice that mimic luminal subtype B breast cancer and these mice crossed to MCK-miR-486 transgenic mice. miR-486 significantly prevented tumor-induced reduction in muscle contraction force, grip strength, and rotarod performance in MMTV-Neu, but not in MMTV-PyMT mice. In MMTV-Neu model, miR-486 reversed several of the cancer-induced changes in skeletal muscle including loss of p53, phospho-AKT, and phospho-laminin alpha 2 (LAMA2) and gain of phosphorylation of the pre-mRNA processing factor hnRNPA0 and the splicing factor SRSF10. LAMA2 is a part of the dystrophin-associated glycoprotein complex, and its loss-of-function mutation is associated with congenital muscular dystrophy. Thus, similar to muscular dystrophy, miR-486 has the potential to reverse skeletal muscle defects and cancer burden in select cancer types.

### Sample Protocol
Approximately one gram of fresh skeletal muscle tissues was dissected from the hind limb of each mouse and digested with mouse muscle dissociation kit (MASC#130-098-305) for 30 minutes at 37oC in water bath, then homogenized with C tube (MASC# 130-093-237) using m_muscle_01_01 program set in GentleMACS Dissociator (#130-093-235). After another 30 minutes incubation in 37oC water bath with shaking every 5 minutes, digested tissue was diluted with DMEM medium and centrifuged at 200 rpm for 30 seconds. Top supernatant was transferred to a Petri dish and large fiber debris were removed under light microscope. Next, supernatant underwent a series of steps, including washing and blood cell lysis as described in the mouse muscle dissociation kit (MASC#130-098-305). Following final centrifuge, the top supernatant was discarded, the myofiber pellets were saved for the analysis of global proteins and post translational modifications.Pellets were lysed in 8 M urea (CHEBI: 16199), 100 mM Tris-HCl, pH 8.5 (CHEBI: 975446756, Sigma-Aldrich Cat No:  10812846001) by sonication in 1.5 mL Micro Tubes (TPX Plastic for Sonication from Diagende Inc.) using a Bioruptor® sonication system (Diagenode Inc. USA, North America cat number B01020001) with 30 sec/30 sec on/off cycles for 15 minutes in a water bath at 4 ºC. After subsequent centrifugation at 14,000 rcf for 20 min, protein concentrations were determined by Bradford protein assay (BioRad Cat No:  5000006). The global proteomics workflow used 25 µg equivalent of protein from each sample and the phosphoproteomics workflow utilized 3000 µg equivalent of protein from each sample.  In both cases, proteins were reduced with 5 mM tris(2-carboxyethyl)phosphine hydrochloride (TCEP, Sigma-Aldrich Cat No:  C4706) for 30 minutes at room temperature and the resulting free cysteine thiols were alkylated with 10 mM chloroacetamide (CAA, Sigma Aldrich Cat No:  C0267) for 30 min at room temperature in the dark.  Samples were diluted with 50 mM Tris.HCl, pH 8.5 to a final urea concentration of 2 M for Trypsin/Lys-C based overnight protein digestion at 37 ºC (1:100 protease:substrate ratio, Mass Spectrometry grade, Promega Corporation, Cat No: V5072.)  Peptide Purification and Labeling Digestions were acidified with trifluoracetic acid (TFA, 0.5% v/v) and desalted on Sep-Pak® Vac cartridges (WatersTM Cat No:  WAT054955) with a wash of 1 mL 0.1% TFA followed by elution in 70% acetonitrile 0.1% formic acid (FA). Peptides for phosphopeptide enrichment were dried and then enriched as described below. Peptides for the global experiment were dried by speed vacuum and resuspended in 24 µL of 50 mM triethylammonium bicarbonate. Peptide concentrations were checked by Pierce Quantitative colorimetric assay (Cat No: 23275).  The same amount of peptide from each sample was then labeled for two hours at room temperature, with 0.5 mg of Tandem Mass Tag Pro (TMTpro) reagent (16-plex kit, manufactures instructions Thermo Fisher Scientific, TMTpro™ Isobaric Label Reagent Set; Cat No:  44520, lot no. VJ316536 see Table X below) [2]. Labelling reactions were quenched by adding 2 uL of 5% hydroxylamine to the reaction mixtures at room temperature for 15 minutes. Labeled peptides were then mixed, and dried by speed vacuum.    Phosphopeptide Enrichment  Phosphopeptides were enriched using High-Select™ TiO2 Phosphopeptide Enrichment Kits (Thermo Fisher Scientific, Cat No: A32993). After preparing spin tips as per manufacturer’s instructions, each sample was applied to an individual enrichment tip, washed and eluted as per manufacturer’s instructions.  Each phosphopeptide elution was immediately dried and then labeled with TMTpro reagent as described above. High pH Basic Fractionation For high pH basic fractionation, peptides were reconstituted in 0.1% trifluoroacetic acid and fractionated on Sep-Pak® Vac cartridges using methodology and reagents from Pierce™ High pH reversed-phase peptide fractionation kit (8 fractions for phosphopeptides and 9 for global including a 70% acetonitrile final fraction; Thermo Fisher Cat No: A32993).

### Data Protocol
Nano-LC-MS/MS Analysis Nano-LC-MS/MS analyses were performed on an EASY-nLC HPLC system (SCR: 014993, Thermo Fisher Scientific) coupled to Orbitrap Eclipse™ mass spectrometer (Thermo Fisher Scientific) with a FAIMS pro interface. 1/8 of each global peptide fraction and ¼ of each phosphopeptide fraction was loaded onto a reversed phase EasySpray™ C18 column (2 μm, 100 Å, 75 μm x 25 cm, Thermo Scientific Cat No:  ES902A) at 400 nL/min. Peptides were eluted from 4-30% with mobile phase B (Mobile phases A: 0.1% FA, water; B: 0.1% FA, 80% Acetonitrile (Thermo Fisher Scientific Cat No:  LS122500)) over 160 minutes, 30-80% B over 10 mins; and dropping from 80-10% B over the final 10 min. The mass spectrometer was operated in positive ion mode with a 4 sec cycle time data-dependent acquisition method with advanced peak determination and Easy-IC (internal calibrant).  FAIMS CV was maintained at -50 V. Precursor scans (m/z 400-1750) were done with an orbitrap resolution of 120000, RF lens% 30, maximum inject time 50 ms, standard AGC target, MS2 intensity threshold of 2.5e4, including charges of 2 to 7 for fragmentation with 30 sec dynamic exclusion. MS2 scans were performed with a quadrupole isolation window of 0.7 m/z, 34% HCD CE, 50000 resolution, 200% normalized AGC target, dynamic maximum IT fixed first mass of 100 m/z. Global peptide samples were run in technical duplicate with an identical method in which 3 FAIMS CVs were used (-40, -55, and -70 V for 1.3 sec cycle time each with common dynamic exclusion lists).  The data were recorded using Thermo Fisher Scientific Xcalibur (4.3) software (Thermo Fisher Scientific Inc.). Proteome and Phosphoproteome Analysis Resulting RAW files were analyzed in Proteome Discover™ 2.5 (Thermo Fisher Scientific, RRID: SCR_014477) with a Mus musculus UniProt FASTA (both reviewed and unreviewed sequences) plus common contaminants. Quantification methods utilized isotopic impurity levels available from Thermo Fisher Scientific. All global peptide runs (18 raw files) were analyzed as one set of fractions. SEQUEST HT searches were conducted with a maximum number of 3 missed cleavages; precursor mass tolerance of 10 ppm; and a fragment mass tolerance of 0.02 Da. Static modifications used for the search were, 1) carbamidomethylation on cysteine (C) residues; 2) TMTpro label on lysine (K) residues. Dynamic modifications used for the search were oxidation of methionines, TMTpro label on the N-termini of peptides, deamidation of N, acetylation, Met-loss or Met-loss plus acetylation of protein of N-termini. For phosphopeptide samples, phosphorylation of S, T, Y residues was included as a variable modification. Percolator False Discovery Rate was set to a strict setting of 0.01 and a relaxed setting of 0.05. IMP-ptm-RS node was used for all modification site localization scores. In the consensus workflows, peptides were normalized by total peptide amount with no scaling. Co-isolation thresholds of 55% and average reporter ion S/N cutoffs of 3 were used for quantification of global peptides and 60% and 5 respectively for phosphopeptides.  Resulting normalized abundance values for each sample type, abundance ratio and log2(abundance ratio) values; and respective p-values (ANOVA) from Proteome Discover™ were exported to Microsoft Excel.

### Publication Abstract
miR-486 is a myogenic microRNA, and its reduced skeletal muscle expression is observed in muscular dystrophy. Transgenic overexpression of miR-486 using muscle creatine kinase promoter (MCK-miR-486) partially rescues muscular dystrophy phenotype. We had previously demonstrated reduced circulating and skeletal muscle miR-486 levels with accompanying skeletal muscle defects in mammary tumor models. To determine whether skeletal muscle miR-486 is functionally similar in dystrophies and cancer, we performed functional limitations and biochemical studies of skeletal muscles of MMTV-Neu mice that mimic HER2+ breast cancer and MMTV-PyMT mice that mimic luminal subtype B breast cancer and these mice crossed to MCK-miR-486 mice. miR-486 significantly prevented tumor-induced reduction in muscle contraction force, grip strength, and rotarod performance in MMTV-Neu mice. In this model, miR-486 reversed cancer-induced skeletal muscle changes, including loss of p53, phospho-AKT, and phospho-laminin alpha 2 (LAMA2) and gain of hnRNPA0 and SRSF10 phosphorylation. LAMA2 is a part of the dystrophin-associated glycoprotein complex, and its loss of function causes congenital muscular dystrophy. Complementing these beneficial effects on muscle, miR-486 indirectly reduced tumor growth and improved survival, which is likely due to systemic effects of miR-486 on production of pro-inflammatory cytokines such as IL-6. Thus, similar to dystrophy, miR-486 has the potential to reverse skeletal muscle defects and cancer burden.

### Keywords
Functional limitations, Mir-486, Lc-msms, Mouse, Breast cancer, Skeletal muscle

### Affiliations
Indiana University School of Medicine
Professor of Surgery Marian J. Morrison Chair in Breast Cancer Research Associate Director of Education, IU Simon Cancer Center Co-Director, IU Simon Cancer Center Breast Cancer Program  Department of Surgery, Department of Biochemistry and Molecular Biology, Indiana University School of Medicine, Indianapolis, IN 46202, USA Richard L Roudebush VA Medical Center, Indianapolis, IN 46202, USA  C218C, 980 West Walnut St. Indianapolis, IN 46202, USA 317 278 2238

### Submitter
Emma Doud

### Lab Head
Dr Harikrishna Nakshatri, BVSc., PhD
Professor of Surgery Marian J. Morrison Chair in Breast Cancer Research Associate Director of Education, IU Simon Cancer Center Co-Director, IU Simon Cancer Center Breast Cancer Program  Department of Surgery, Department of Biochemistry and Molecular Biology, Indiana University School of Medicine, Indianapolis, IN 46202, USA Richard L Roudebush VA Medical Center, Indianapolis, IN 46202, USA  C218C, 980 West Walnut St. Indianapolis, IN 46202, USA 317 278 2238


