### Accession
PXD009614

### Title
Quantitative analysis of the exoproteome from a Staphylococcus aureus LAC sarA mutant

### Description
The staphylococcal accessory regulator A (sarA) impacts the extracellular accumulation of Staphylococcus aureus virulence factors at the level of intracellular production and extracellular protease-mediated degradation. To assess the relative impact of these two functions, we previously used a proteomics approach that measures protein abundance as a function of all proteoforms to demonstrate that mutation of sarA results in increased levels of extracellular proteases and assess the impact of this on the accumulation of S. aureus exoproteins1. While this approach confirmed that protease-mediated degradation has a significant impact on the S. aureus exoproteome, it was potentially limited in that it did not take into account the possibility that large, stable proteolytic products from a given protein could result in false negatives when quantified by total proteoforms. Here, we present an expanded proteomics approach that utilizes a dual quantitative method for measuring abundance at both the total proteoform and full-length exoprotein levels. Specifically, proteins present in conditioned medium from overnight, stationary phase cultures of the USA300 strain LAC, an isogenic sarA mutant, and a sarA mutant unable to produce any of the known extracellular proteases (sarA/protease) were resolved using one-dimensional gel electrophoresis. Using methods that focus on total proteoforms vs. methods that focus specifically on full-length proteins, quantitative proteomic comparisons of sarA vs sarA/protease mutants identified proteins that were degraded in a protease dependent manner owing to mutation of sarA, while comparisons of a sarA/protease mutant vs the LAC parent strain identified proteins in which abundance was altered in a sarA mutant in a protease-independent manner. Furthermore, the proteins uniquely identified by the full-length data analysis approach eliminated false negatives observed in the total proteoform analysis. This approach provided for a more comprehensive and robust analysis of the impact of mutating sarA and protease-mediated degradation on the S. aureus exoproteome.

### Sample Protocol
Bacterial strains and growth conditions. The bacterial strains used in this study were fully described in our previous report 1 . Briefly, they consisted of the USA300, methicillin-resistant (MRSA) strain LAC (referred to here as wild-type), its isogenic sarA mutant (sarA), and an isogenic sarA mutant unable to produce any of the 10 recognized extracellular proteases (sarA/protease). These proteases include the metalloprotease aureolysin, the cysteine proteases ScpA and SspB (also known as staphopain A and B, respectively), and the serine proteases SspA and SplA-F.  With the exception of the sarA mutation, which was generated by insertion of a kanamycin-resistance cassette into the sarA open-reading frame 9 , all mutations are true null mutants in which the protease genes were eliminated using either pJB38, which is a modified version of the plasmid pKOR1 13 , or by replacement with an erythromycin-resistance cassette 8, 14-16 . All strains were maintained on tryptic soy agar (TSA) with antibiotic selection as appropriate, but were grown without selection when preparing conditioned medium (CM). For exoproteome analysis, stationary phase (16 hr) cultures grown in tryptic soy broth (TSB) were standardized relative to each other based on optical density and the CM prepared by centrifugation followed by filter sterilization of the clarified supernatant using 0.45 Âµm filters 1 . Proteomics studies and statistical analysis were based on three biological replicates of each strain.  Mass spectrometric analysis of proteins. An equal volume of CM from each sample was resolved by one dimensional SDS-PAGE and visualized by Coomassie-staining (Figure 1) 17-20. Each gel lane was sliced into 24 equivalent bands of 2 mm each. Gel bands were destained in 50% methanol (Fisher) containing 100 mM ammonium bicarbonate (Sigma-Aldrich). This was followed by reduction in 10 mM Tris [2-carboxyethyl] phosphine (Pierce) and alkylation in 50 mM iodoacetamide (Sigma-Aldrich). Gel slices were then dehydrated in acetonitrile (Fisher), followed by addition of 100 ng porcine sequencing grade modified trypsin (Promega) in 100 mM ammonium bicarbonate (Sigma-Aldrich) and digestion at 37oC for 12-16 hours.  Peptide products were then acidified in 0.1% formic acid (Pierce).  Tryptic peptides were separated by reverse phase Jupiter Proteo resin (Phenomenex) on a 200 x 0.075 mm column using a nanoAcquity UPLC system (Waters).  Peptides were eluted using a 30 min gradient from 97:3 to 65:35 buffer A:B ratio  [Buffer A = 0.1% formic acid, 0.5% acetonitrile; buffer B = 0.1% formic acid, 99.9% acetonitrile].  Eluted peptides were ionized by electrospray (2.15 kV) followed by MS/MS analysis using higher-energy collisional dissociation (HCD) on an Orbitrap Fusion Tribrid mass spectrometer (Thermo) in top-speed data-dependent mode.  MS data were acquired using the FTMS analyzer in profile mode at a resolution of 240,000 over a range of 375 to 1500 m/z.  Following HCD activation, MS/MS data were acquired using the ion trap analyzer in centroid mode and normal mass range with precursor mass-dependent normalized collision energy between 28.0 and 31.0.

### Data Protocol
Proteins were identified by database search using Mascot (Matrix Science, version 2.5.1) against the UniProtKB USA300 S. aureus database (2607 entries) with a parent ion tolerance of 3 ppm and a fragment ion tolerance of 0.5 Da.  Scaffold (Proteome Software) was used to verify MS/MS based peptide and protein identifications.  Peptide identifications were accepted if they could be established with less than 1.0% false discovery by the Scaffold Local FDR algorithm.  Protein identifications were accepted if they could be established with less than 1.0% false discovery and contained at least 2 identified peptides.  Protein probabilities were assigned by the Protein Prophet algorithm 21. Total spectral counts for each replicate were exported from Scaffold into Microsoft Excel for further analysis.  A dual method was taken for data analysis. First, the amount of the total proteoform was identified using the total spectral counts for a given protein in the entire gel lane (i.e., in any of the 24 gel bands from a given gel lane), with spectral counts defined as assigned tandem mass spectra for a given protein 17-20, 22. The spectral counts for a given protein detected in the entire gel lane were averaged across triplicate samples for each strain; this averaged spectral count value was defined as the total proteoform level of the protein in each strain. This method was the same total proteoform method employed in our earlier studies 1 . Second, the amount of full-length exoprotein was measured using a 3-gel band window approach also based on spectral counting; however, this method is based on using three consecutive gel bands centered on the gel band representing the full-length protein. Specifically, spectral count data collected from the wild-type was used to locate the gel band with the maximum spectral count for a given protein, which for the purposes of this study was considered the most full-length version of the protein under steady-state conditions. Spectral counts for the full-length version of that protein were determined by adding the spectral counts in that gel band with those in the gel bands immediately above and below, thus accounting for minor variations in full-length protein migration through the gel. The spectral counts observed in this 3-band window were then averaged across triplicate samples of wild-type; this averaged spectral count value was defined as the level of the steady-state, full-length version of the protein in wild-type. This information was then used to calculate spectral counts for the full-length protein in the sarA and sarA/protease mutants.  For example, if the 3-band window for a specific protein was gel bands 8-10 in wild-type, then the 3-band window used for calculating the spectral counts for the full-length protein in sarA and sarA/protease mutants was also gel bands 8-10. The spectral counts observed in the 3-band window were then averaged across triplicate samples of sarA and sarA/protease strains; this averaged spectral count value was defined as the level of the steady-state, full-length version of the protein in the sarA and sarA/protease strains.

### Publication Abstract
The staphylococcal accessory regulator A ( sarA) impacts the extracellular accumulation of Staphylococcus aureus virulence factors at the level of intracellular production and extracellular protease-mediated degradation. We previously used a proteomics approach that measures protein abundance of all proteoforms to demonstrate that mutation of sarA results in increased levels of extracellular proteases and assesses the impact of this on the accumulation of S. aureus exoproteins. Our previous approach was limited as it did not take into account that large, stable proteolytic products from a given protein could result in false negatives when quantified by total proteoforms. Here, our goal was to use an expanded proteomics approach utilizing a dual quantitative method for measuring abundance at both the total proteoform and full-length exoprotein levels to alleviate these false negatives and thereby provide for characterization of protease-dependent and -independent effects of sarA mutation on the S. aureus exoproteome. Proteins present in conditioned medium from overnight, stationary phase cultures of the USA300 strain LAC, an isogenic sarA mutant, and a sarA mutant unable to produce any of the known extracellular proteases ( sarA/protease) were resolved using one-dimensional gel electrophoresis. Quantitative proteomic comparisons of sarA versus sarA/protease mutants identified proteins that were cleaved in a protease-dependent manner owing to mutation of sarA, and comparisons of sarA/protease mutant versus the LAC parent strain identified proteins in which abundance was altered in a sarA mutant in a protease-independent manner. Furthermore, the proteins uniquely identified by the full-length data analysis approach eliminated false negatives observed in the total proteoform analysis. This expanded approach provided for a more comprehensive analysis of the impact of mutating sarA on the S. aureus exoproteome.

### Keywords
Gelcms, Protease, Proteolysis, Sara, Staphylococcus aureus

### Affiliations
UAMS
Department of Biochemistry and Molecular Biology, University of Arkansas for Medical Sciences, 4301 West Markham Street, Little Rock, AR 72205, USA Arkansas Children's Research Institute, 13 Children's Way, Little Rock, AR 72205, USA

### Submitter
Stephanie Byrum

### Lab Head
Dr Alan J. Tackett
Department of Biochemistry and Molecular Biology, University of Arkansas for Medical Sciences, 4301 West Markham Street, Little Rock, AR 72205, USA Arkansas Children's Research Institute, 13 Children's Way, Little Rock, AR 72205, USA


