### Accession
PXD035779

### Title
The PqsE Protein Interaction Network in Pseudomonas aeruginosa

### Description
The PqsE enzyme plays a vital role in quorum sensing and virulence in Pseudomonas aeruginosa, yet its enzymatic function is unknown. Here, we identify the protein interaction network of PqsE as well as that of a catalytically dead variant, PqsE(D73A) in P. aeruginosa PA14. Our analyses identify proteins that interact with PqsE that are independent of and that depend on PqsE catalytic function. One such catalysis-independent interaction is with the quorum-sensing regulator, RhlR, consistent with our previous work. We also characterize the PqsE interaction network in a delta rhlR P. aeruginosa PA14 strain and identify additional proteins as PqsE-interactors.

### Sample Protocol
Frozen cell powders from cryogenic grinding were resuspended in pre-chilled lysis buffer (20 mM HEPES pH 7.4, 100 mM potassium acetate, 2 mM MgCl2, 0.1% Tween-20 (v/v), 1 Ã‚ÂµM ZnCl2, 1 Ã‚ÂµM CaCl2, 1% Triton-X100 (v/v), 200 mM NaCl, 0.5 mM phenylmethylsulfonyl fluoride (PMSF), 1:2,500 Pierce Universal Nuclease, and a protease inhibitor cocktail (Roche, one tablet/10 mL buffer)). Resuspension was carried out by inversion, gentle vortex, and rotation, followed Polytron homogenization. 500 Ã‚Âµg of protein from each sample was incubated with 35 Ã‚ÂµL pre-washed GFP-Trap magnetic beads (Chromotek, Inc.) for 1 h at 4 Ã‹Å¡C with rotation (1 mg/mL final lysate concentration). The beads were washed three times with 500 Ã‚ÂµL wash buffer (lysis buffer lacking PMSF, nuclease, and protease inhibitors). A final wash with cold PBS was performed, and the proteins were eluted in 50 Ã‚ÂµL 1x TES buffer (2% SDS, 0.5 mM EDTA, 53 mM Tris HCl, 70 mM Tris Base) by incubation at 70 Ã‹Å¡C for 10 min and vortex for 20 s. IP eluates were concentrated two-fold in a speedvac. Proteins were reduced and alkylated with 25 mM Tris(2-carboxyethyl)phosphine (TCEP) (Thermo Fisher) and 50 mM chloroacetamide (CAM) (Fisher Scientific) by heating for 20 min at 70 Ã‚Â°C. The resulting samples were digested using an S-Trap micro column (Protifi LLC). Briefly, samples were acidified to 1.2% phosphoric acid, diluted into S-Trap binding buffer (100 mM TEAB, pH 7.1 in 90% methanol), and bound to the S-Trap column by centrifugation at 4,000 x g for 30 s. S-Trap-bound sample was washed using the same centrifugation procedure as follows: two washes with S-Trap binding buffer, five washes with methanol/chloroform (4:1 v/v), and three washes with S-Trap binding buffer. Samples were next digested on the S-Trap column with 2.5 Ã‚Âµg trypsin diluted in digestion buffer (25 mM TEAB) for 1 h at 47 Ã‚Â°C. Trypsinized peptides were eluted through a three-part elution (40 Ã‚ÂµL 25 mM TEAB, 40 Ã‚ÂµL 0.2% formic acid, 70 Ã‚ÂµL 50% acetonitrile/0.2% formic acid) by sequentially adding the elution buffers to the column followed by centrifugation as described above after addition of each buffer. The eluted peptides were pooled in an LC-MS autosampler vial (Fisher Scientific) and dried via speedvac. Peptides were resuspended in 6 Ã‚ÂµL 1% formic acid (FA)/1% acetonitrile (ACN). IP samples were analyzed on a Q-Exactive HF mass spectrometer (ThermoFisher Scientific) equipped with a Nanospray Flex Ion Source (ThermoFisher Scientific). Peptides were separated on a 50 cm column (360 ÃŽÂ¼m od, 75 ÃŽÂ¼m id, Fisher Scientific) packed in house with ReproSil-Pur C18 (120 Ãƒâ€¦ pore size, 1.9 ÃŽÂ¼m particle size, ESI Source Solutions). Peptides were separated over a 150 min gradient of 3% B to 35% B (solvent A: 0.1% FA, solvent B: 0.1% FA, 97% ACN) at a flow rate of 0.25 nL/min. MS1 scans were collected with the following parameters: 120,000 resolution, 30 ms MIT, 3e6 automatic gain control (AGC), scan range 350 to 1800 m/z, and data collected in profile. MS2 scans were collected with the following parameters: 30,000 resolution, 150 ms MIT, 1e5 AGC, 1.6 m/z isolation window, loop count of 10, NCEof 28, 100.0 m/z fixed first mass, peptide match set to preferred, and data collected in centroid and at a dynamic exclusion of 45 s.

### Data Protocol
MS/MS spectra were analyzed in Proteome Discoverer v2.4 (Thermo Fisher Scientific). Sequest HT was used to search spectra against a Uniprot database containing P. aeruginosa protein sequences (downloaded October 2020) and common contaminants. Offline mass recalibration was performed via the Spectrum Files RC node, and the Minora Feature Detector node was used for label-free MS1 quantification. Fully tryptic peptides with a maximum of two missed cleavages, a 4 ppm precursor mass tolerance, and a 0.02 Da fragment mass tolerance were used in the search. Posttranslational modifications (PTMs) that were allowed included the static modification carbamidomethylation of cysteine and the following dynamic modifications: oxidation of methionine, deamidation of asparagine, loss of methionine plus acetylation of the N-terminus of the protein, acetylation of lysine, and phosphorylation of serine, threonine, and tyrosine. Peptide spectrum match (PSM) validation was accomplished using the Percolator node and PTM sites were assigned in the ptmRS node. PSMs were assembled into peptide and protein identifications with a false discovery rate of less than 1% at both the peptide and protein levels with at least two unique peptides identified per protein. Precursor quantitation required identification in at least two out of the three replicates. Samples were normalized in a retention time dependent manner, imputation was performed using low abundance resampling, protein abundances were calculated using summed abundances, and protein ratio calculations were performed using pairwise ratios.  Data were exported to Microsoft Excel for further processing. In order for a protein to be considered a putative PqsE interactor, the following requirements had to be met: (1) there must be at least a two-fold enrichment of the protein in the sample of interest compared to the eGFP control, (2) at least two peptides had to be quantified in the sample of interest (PqsE(WT) or PqsE(D73A)) in all replicates, and (3) the Grouped Coefficient of Variation (CV%) had to be less than 75%. For proteome analyses, quantified proteins were reported that were measured by two unique peptides in both replicates of at least one sample group and had < 75% coefficient of variance for all sample groups. Raw and adjusted p-values were calculated by Proteome Discoverer using the t-test (Background) method, which contrasts individual protein ratios to the background of all quantified proteins. Differential proteins were assigned if |log2 protein ratio| ≥ 1.0 and adjusted p-value < 0.05.

### Publication Abstract
Pseudomonas aeruginosa is a human pathogen that relies on quorum sensing to establish infections. The PqsE quorum-sensing protein is required for P. aeruginosa virulence factor production and infection. PqsE has a reported enzymatic function in the biosynthesis of the quorum-sensing autoinducer called PQS. However, this activity is redundant because, in the absence of PqsE, this role is fulfilled by alternative thioesterases. Rather, PqsE drives P. aeruginosa pathogenic traits via a protein-protein interaction with the quorum-sensing receptor/transcription factor RhlR, an interaction that enhances the affinity of RhlR for target DNA sequences. PqsE catalytic activity is dispensable for interaction with RhlR. Thus, the virulence function of PqsE can be decoupled from its catalytic function. Here, we present an immunoprecipitation-mass spectrometry method employing enhanced green fluorescent protein-PqsE fusions to define the protein interactomes of wild-type PqsE and the catalytically inactive PqsE(D73A) variant in P. aeruginosa and their dependence on RhlR. Several proteins were identified to have specific interactions with wild-type PqsE while not forming associations with PqsE(D73A). In the &#x394;<i>rhlR</i> strain, an increased number of specific PqsE interactors were identified, including the partner autoinducer synthase for RhlR, called RhlI. Collectively, these results suggest that specific protein-protein interactions depend on PqsE catalytic activity and that RhlR may prevent proteins from interacting with PqsE, possibly due to competition between RhlR and other proteins for PqsE binding. Our results provide a foundation for the identification of the <i>in vivo</i> PqsE catalytic function and, potentially, new proteins involved in P. aeruginosa quorum sensing. <b>IMPORTANCE</b> Pseudomonas aeruginosa causes hospital-borne infections in vulnerable patients, including immunocompromised individuals, burn victims, and cancer patients undergoing chemotherapy. There are no effective treatments for P. aeruginosa infections, which are usually broadly resistant to antibiotics. Animal models show that, to establish infection and to cause illness, P. aeruginosa relies on an interaction between two proteins, namely, PqsE and RhlR. There could be additional protein-protein interactions involving PqsE, which, if defined, could be exploited for the design of new therapeutic strategies to combat P. aeruginosa. Here, we reveal previously unknown protein interactions in which PqsE participates, which will be investigated for potential roles in pathogenesis.

### Keywords
Ip-ms, Bacteria, Quorum sensing, Q-exactive hf

### Affiliations
Molecular Biology, Princeton University, USA
Molecular Biology, Princeton University

### Submitter
Todd Greco

### Lab Head
Dr Ileana M Cristea
Molecular Biology, Princeton University, USA


