### Accession
PXD000145

### Title
Zebrafish embryo gene and protein expression comparison

### Description
Sensitivity and throughput of transcriptomics and proteomomics technologies have advanced tremendously in recent years. With the use of deep sequencing of RNA samples (RNAseq) and mass spectrometry technology for protein identification and quantitation, it is now feasible to compare gene and protein expression on a massive scale and on any organism for which genomic data is available. Although these technologies are currently applied to many research questions in various model systems ranging from cell cultures to the entire organism level, there are few comparative studies of these technologies in the same system, let alone on the same samples. Here we present a comparison between gene and protein expression in zebrafish embryos, which is an upcoming model in disease studies. We compared Agilent custom made expression microarrays with Illumina deep sequencing for RNA analysis, showing as expected a high degree of correlation of expression of a common set of 15,927 genes. Gene expression was also found to correlate with the abundance of 1,017 quantified proteins, with several categories of genes as exceptions. These exceptions include ribosomal proteins, histones and vitellogenins, for which biological and technical explanations are discussed. By comparing state of the art transcriptomic and proteomic technologies on samples derived from the same group of organisms we have for the first time benchmarked the differences in these technologies with regard to sensitivity and bias towards detection of particular gene categories. Our datasets submitted to public repositories are a good starting point for researchers interested in disease progression in zebrafish at a stage of development highly suited for high throughput screening technologies. The transcriptomics data is deposited in NCBI GEO. Proteomics informatics: Spectra were identified by Mascot and validated by PeptideProphet  and ProteinProphet in the Trans-Proteomic Pipeline (TPP). The main search parameters are: full tryptic specificity, precursor mass searched within -0.5 to +2.5 Da, product ions within [-0.5,0.5] Da and allowing for one missed cleavage. Carbamidomethylation was assumed as the only and fixed PTM.

### Sample Protocol
Zebrafish larvae were ground with a pestle in liquid nitrogen in 1.5 mL Eppendorf tubes and vortexed for 30 seconds in a lysis buffer consisting of 9 parts 20 mM Tris–HCl, pH 8.5, 20 mM NaCl and 1 part protease inhibitor cocktail (P 8340, Sigma-Aldrich). The samples were placed on a shaking table for 20 minutes at room temperature before spinning down cellular debris at 16,100 × g for 10 minutes at 4°C. The supernatant was transferred to a fresh tube and the supernatants as well as the remaining pellets frozen in −35°C. Five μL NuPAGE® LSD Sample Buffer (Invitrogen) was added to 20 μL, 20 μg protein, as measured by the bicinchoninic acid (BCA) protein assay (ThermoScientific), and loaded in each lane on a 10-lane 4-12% NuPAGE® Bis-Tris gel. The assembled Invitrogen XCell SureLock™ Mini-Cell was filled with SDS-MOPS Running Buffer and the separation run for 1 hour at 200 V. The gel was released from its casing and immersed in NuPAGE® SureStain Colloidal Blue (55 mL deionized water, 20 mL methanol, 20 mL “Stain A” and 5 mL “Stain B”) and left on a shaker overnight at room temperature. The staining solution was decanted and the gels were washed with deionized water before being scanned in an OptiGo UV imager (Isogen Life Science, De Meern, the Netherlands). The gel lanes were cut into 48 identical slices fractions using a custom-made OneTouch Mount and Lane Picker (The Gel Company, San Francisco, CA). The gel slices were then removed with a Acu-Min® stainless steel Precision Probe with a 35° Single Bend Tip (#6) (Moody Tools Inc., Warwick, RI) and each slice placed into one well in a 96-well polypropylene PCR plate (Greiner Bio-One, Frickenhausen Germany). Using 50 and 250 μL Rainin 8-channel multi-pipettors, each gel slice was washed for 5 minutes at room temperature with 100 μL 25 mM ammonium bicarbonate (ABC), followed by a wash with 100 μL 30% acetonitrile in 25 mM ABC for 10 minutes and a final wash of 100 μL 80% acetonitrile in 25 mM ABC for 10 minutes, discarding the solution between each wash. Cystines were reduced by addition of 75 μL 10 mM DTT in 25 mM ABC and the plate incubated at 56°C for 20 minutes. After discarding the DTT solution, cysteines were alkylated by addition of 75 μL 55 mM iodoacetamide in 25 mM ABC with incubation at room temperature in darkness for 20 minutes. The gel pieces were washed again as above and the supernatant discarded. The proteins were digested in-gel by addition of 15 μL 5 ng/μL porcine trypsin (Promega, Madison, WI) and incubation for 6 hours at 37°C, after which the digestion was quenched with the addition of 1 μL 5% trifluoroacetic acid (TFA). The solution was removed, followed by a second extraction with 20 μL 0.1% TFA for 1 h and the solution pooled with the first extraction. The plate was stored at −35°C until analysis by LC-MS/MS.

### Data Protocol
All SDS-PAGE fractions were analyzed by LC-MS/MS using a splitless NanoLC-Ultra 2D plus (Eksigent, Dublin, CA) for parallel ultra-high pressure liquid chromatography (UHPLC) with an additional loading pump for fast sam-ple loading and desalting. The UHPLC system was configured with 300 μm-i.d. 5-mm PepMap C18 trap columns (Thermo Scientific, Sunnyvale, CA), 15-cm 300 μm-i.d. ChromXP C18 columns supplied by Eksigent and running 90-minute linear gradients from 4 to 33% acetonitrile in 0.05% formic acid. The UHPLC system was coupled on-line to an HCTultra PTM Discovery System ion trap (Bruker Daltonics, Bremen, Germany). After each MS scan, up to ten abundant multiply charged species in m/z 300–1300 were automatically selected for MS/MS but excluded for one minute after being selected twice. The UHPLC system was controlled using the HyStar3.4 with a plug-in from Eksigent and the HCTultra ion trap by esquireControl 6.2, all from Bruker. Peptides and proteins were identified using Mascot 2.4 running on an 8-CPU in-house server and the Danio rerio UniProt database (20130814) and validated using PeptideProphet and ProteinProphet in the Trans-Proteomic Pipeline. In the Mascot search, a peptide mass measurement tolerance of [−0.5, 2.5] Da was used with MS/MS mass measurement tolerance [−0.5, 0.5] Da and allowing one missed cleavage. Carbamidylation of cysteine was included as the only fixed modication and oxidation of methionine as the only variable modification. Protein names from UniProt were matched against the microarray transcript (gene) names, expanding each protein group to individual proteins to match as many genes as possible, bearing in mind that not all of these proteins or protein isoforms are independently quantified.

### Publication Abstract
None

### Keywords
Zebrafish, Embryos

### Affiliations
Leiden University Medical Center
Center for Proteomics and Metabolomics

### Submitter
Magnus Palmblad

### Lab Head
Dr Magnus Palmblad
Leiden University Medical Center


