### Accession
PXD029968

### Title
Induced Degradation of Lineage-specific Oncoproteins Underlies the Selective PARP1 Inhibitor Toxicity in Small Cell Lung Cancer (batch2-6)

### Description
A subset of small cell lung cancer (SCLC) shows a clinical response to PARP inhibitors (PARPi) despite being proficient in homologous repair pathways. However, the underlying mechanism(s) of PARPi sensitivity is poorly understood. We performed quantitative proteomic analyses and identified proteomic changes that signify PARPi responses in a large panel of molecularly annotated patient-derived SCLC lines. We found that the toxicity of PARPi in SCLC is explained by the PARPi-induced degradation of key lineage-specific oncoproteins including ASCL1, NEUROD1, POU2F3, KDM4A, and KDM5B. Biochemical experiments showed that PARPi-induced activation of E3 ligases (e.g., HUWE1 and RNF8) mediated the ubiquitin-proteasome system (UPS)-dependent degradation of these oncoproteins. Interestingly, although PARPi resulted in a general DNA damage response, this signal is sensed by different SCLC cell lines to generate a cell-specific response. The dissection of the cell-specific oncoprotein degradation response led to the identification of potentially predictive biomarkers for PARPi in SCLC. The combination of PARPi and agents targeting these pathways led to dramatically improved cytotoxicity in SCLC. PARPi-induced degradation of lineage-specific oncoproteins therefore represents a novel mechanism to explain the efficacy of PARPi in tumors without homologous recombination deficiency.

### Sample Protocol
All SCLC cells were treated with either Talazoparib or a panel of small-molecule compounds for 48 hrs. Cells were lysed with 1% SDS lysis buffer containing 10 mM HEPES, pH 7.0, 2 mM MgCl2, 20 U/mL universal nucleases. Protein concentrations were determined with the BCA assay (Thermo Fisher Scientific). Samples were reduced with 3 mM dithiothreitol (DTT) for 20 min and alkylated with 25 mM iodoacetamide (IDA) for 30 min at room temperature (RT) in dark. The detergents were removed by methanol/chloroform precipitation. The proteins were re-solubilized in 8 M urea and digested by Lys-C at a 1:100 (w/w) enzyme/protein ratio for 2 h, followed by trypsin digestion at 1:100 (w/w) enzyme/protein ratio overnight at RT in 2 M urea. The peptides were desalted using Oasis HLB solid-phase extraction (SPE) cartridges (Waters) and approximately 100 Î¼g of peptides for each sample were resuspended in 200 mM HEPES, pH 8.5. The peptides were then labeled with either the amine-based TMT ten- or sixteen-plex reagents (Thermo Fisher Scientific) for 1 hr at RT. Hydroxylamine solution was added to quench the reaction and the labeled peptide samples were combined. Next, the TMT samples were lyophilized and were reconstituted in buffer A (10 mM Ammonium formate, pH 10.0). It was then centrifuged at 10,000 Ã— g for 3 min using CorningÂ® CostarÂ® Spin-XÂ® Plastic Centrifuge Tube Filters (MilliporeSigma) prior to loading onto a ZORBAX 300 Extend-C18 HPLC column (Agilent, Narrow Bore RR 2.1 mm x 100 mm, 3.5 Î¼m particle size, 300 Çº pore size). Peptides were fractioned by bRPLC (basic pH reversed-phase HPLC) at a 0.2 mL/min flow rate using a gradient from 0% to 70% buffer B (1% Ammonium formate, pH 10.0 and 90% Acetonitrile). The collected seventeen fractions were lyophilized, desalted, and analyzed by LC-MS/MS as described previously 88. Briefly, peptides were separated on a PicoFritÂ® microcapillary column (75 Î¼m Ã— 15 cm, New Objective). A 180 min linear gradient was developed ranging from 7% to 32% acetonitrile in 0.1% formic acid at 300 nL/min to elute the peptides (Thermo EASY-nLC system).

### Data Protocol
The TMT sample was analyzed by LC-MS/MS on an Orbitrap Eclipseâ„¢ Tribridâ„¢ Mass Spectrometer (Thermo Fisher Scientific) using a multi-notch (SPS)-MS3 approach. MS spectra were searched against a composite database of human protein sequences (Uniprot) and their reversed complement using the Sequest algorithm (Ver28) embedded in an in-house-developed software suite89 or Proteome Discovererâ„¢ Software (Thermo Fisher Scientific). MS1 mass tolerance was set to be 50 ppm. Search parameters allowed for full tryptic peptides with a static modification of 57.02146 Da on cystine (Carbamidomethyl), a variable modiï¬cation of 15.994915 Da on methionine (oxidation), and a static modification of TMT labels (229.16293 Da, TMT 10-plex; 295.1896 Da, TMT 16-plex) on peptide N-terminus and lysine. Search results were filtered to includeâ€‰<â€‰1% matches (both peptide and protein level filtering) to the reverse database by the linear discriminator function using parameters including Xcorr, dCN, missed cleavage, charge state (exclude 1+â€‰peptides), mass accuracy, peptide length, and fraction of ions matched to MS/MS spectra. Peptide quantification was performed by using the CoreQuant algorithm implemented in an in-house-developed software suite90 or Proteome Discovererâ„¢ Software (Thermo Fisher Scientific). For TMT quantification, a 0.03 Th window was scanned around the theoretical m/z of each reporter ion to detect the presence of these ions. The maximum intensity of each ion was extracted, and the signal-to-noise (SN) value of each protein was calculated by summing the reporter ion counts across all identified peptides. Because the same amount of peptides was used for each TMT channel, the total reporter ion intensity of each channel was summed across all quantified proteins, and was then normalized and reported. Data were exported to Excel for further analysis.

### Publication Abstract
Although <i>BRCA1/2</i> mutations are not commonly found in small cell lung cancer (SCLC), a substantial fraction of SCLC shows clinically relevant response to PARP inhibitors (PARPis). However, the underlying mechanism(s) of PARPi sensitivity in SCLC is poorly understood. We performed quantitative proteomic analyses and identified proteomic changes that signify PARPi responses in SCLC cells. We found that the vulnerability of SCLC to PARPi could be explained by the degradation of lineage-specific oncoproteins (e.g., ASCL1). PARPi-induced activation of the E3 ligase HUWE1 mediated the ubiquitin-proteasome system (UPS)-dependent ASCL1 degradation. Although PARPi induced a general DNA damage response in SCLC cells, this signal generated a cell-specific response in ASCL1 degradation, leading to the identification of HUWE1 expression as a predictive biomarker for PARPi. Combining PARPi with agents targeting these pathways markedly improved therapeutic response in SCLC. The degradation of lineage-specific oncoproteins therefore represents a previously unidentified mechanism for PARPi efficacy in SCLC.

### Keywords
Parp1 inhibitor, Small cell lung cancer, Quantitative proteomics, A histone demethylase, Lineage-specific oncoproteins, Brca1/2, Predictive biomarkers, Homologous recombination, Pips, Synergistic effect, Ascl1

### Affiliations
UT Southwestern Medical Center
Department of Biochemistry, University of Texas Southwestern Medical Center, Dallas, 75390, TX, USA

### Submitter
Xu-Dong Wang

### Lab Head
Dr Yonghao Yu
Department of Biochemistry, University of Texas Southwestern Medical Center, Dallas, 75390, TX, USA


