### Accession
PXD026101

### Title
The Prostate Cancer mTOR Chromatin-Bound Interactome

### Description
Growing studies support a direct role for nuclear mTOR in gene regulation and chromatin structure. Still, the scarcity of known chromatin-bound mTOR partners limits our understanding of how nuclear mTOR controls transcription. Herein, we comprehensively mapped the mTOR chromatin-bound interactome in four cellular models of prostate cancer (PCa) identifying a conserved 67-protein interaction network enriched for epigenetic and transcription factors as well as SUMOylation machinery in both androgen-dependent and -independent cells. Notably, SUMO2/3 and nuclear pore protein NUP210 are among the strongest interactors while the androgen receptor (AR) is the dominant androgen-inducible mTOR partner. Further investigation showed that NUP210 facilitates mTOR nuclear trafficking, that mTOR, AR and NuRD act as a functional transcriptional complex, and that androgens dictate mTOR-SUMO2/3 promoter-enhancer specificity. This work identifies a vast network of mTOR-associated nuclear complexes advocating novel molecular strategies to modulate mTOR-dependent gene regulation with evident implications for PCa and other diseases.

### Sample Protocol
On-bead digestion and LC-MS/MS for RIME On-bead trypsic digestion was performed overnight at 37ºC with a ratio enzyme:protein of 1:20 in 2M Urea/50 mM AMBIC. The samples were then reduced with 13 mM dithiothreitol at 37ºC and, after cooling for 10 minutes, alkylated with 23 mM iodoacetamide at room temperature for 20 minutes in the dark. The supernatants were acidified with trifluoroacetic acid and cleaned from residual detergents and reagents with MCX cartridges (Waters Oasis MCX 96-well Elution Plate) following the manufacturer’s instructions. Samples were dried with a Speed-vac and stored at -20ºC for a few days. Samples were reconstituted under agitation for 15 min in 12 µL of 2%ACN-1%FA and loaded into a 75 um i.d. × 150 mm Self-Pack C18 column installed in the Easy-nLC II system (Proxeon Biosystems). Peptides were eluted with a three slope gradient at a flowrate of 250 nL/min. Solvent B first increased from 2 to 29% in 70 min, then from 29 to 44% in 50 min and finally from 44 to 95% B in 6 min. The HPLC system was coupled to LTQ Orbitrap Velos spectrometer (Thermo Scientific) through a Nanospray Flex Ion Source. LC-MS/MS data acquisition was accomplished using a seventeen-scan event cycle comprised of a full scan MS for scan event 1 acquired in the Orbitrap. The mass resolution for MS was set to 60,000 (at m/z 400) and used to trigger the sixteen additional MS/MS events acquired in parallel in the linear ion trap for the top sixteen most intense ions. Mass over charge ratio range was from 360 to 1700 for MS scanning with a target value of 1,000,000 charges and from ~1/3 of parent m/z ratio to 2000 for MS/MS scanning with a target value of 10,000 charges. The data dependent scan events used a maximum ion fill time of 100 ms and 1 microscan. Target ions already selected for MS/MS were dynamically excluded for 31 s after 2 counts. Nanospray and S-lens voltages were set to 1.3–1.7 kV and 50 V, respectively. Capillary temperature was set to 250ºC. MS/MS conditions were the following: normalized collision energy, 35 V; activation q, 0.25; activation time, 10 ms.

### Data Protocol
RIME protein identification The peak list files were generated with Proteome Discoverer (version 2.3) using the following parameters: minimum mass set to 500 Da, maximum mass set to 6000 Da, no grouping of MS/MS spectra, precursor charge set to auto, and minimum number of fragment ions set to 5. Protein database searching was performed with Mascot 2.6 (Matrix Science) against the human Uniprot complete protein database (downloaded April 15th, 2015). The mass tolerances for precursor and fragment ions were set to 10 ppm and 0.6 Da, respectively. Trypsin was used as the enzyme allowing for up to 2 missed cleavages. Cysteine carbamidomethylation was specified as a fixed modification, and methionine oxidation and phospho of serine, threonine, and tyrosine were variable modifications. Data interpretation was performed using Scaffold software (version 4.10.0).  RIME data processing and interpretation Peak ion-intensity-based protein quantification, integration and normalization of the samples were performed using the proteome software tool Scaffold v4.10.0. A 1% protein and 1% peptide FDR threshold were applied with a minimum number of detected peptides set to 2. Two independent analyses with normalizations based on total precursor ion (TPI) intensity values were performed: 1) on biological triplicates per condition for all conditions from the 4 PCa cell lines LNCaP, 22Rv1, DU145, PC3; 2) on biological triplicates per condition for all conditions from the normal prostate RWPE-1 cell line. Further computational analyses were performed outside of Scaffold. Briefly, quantitative values were log2-transformed, missing values were designated as “NaN” and protein clusters and DECOYs were removed. For each condition, proteins were only retained if peptides were detected in at least 2 of 3 replicates and for which we could validate the presence of 2 unique peptides in at least 1 replicate. For the identification of mTOR-interacting proteins in a specific condition, only proteins with no peptides found in matching IgG controls or having a mean TPI fold-change greater to or equal to 5 over IgG were retained. For comparison of treatment groups (R1881, Torin 1 or R1881 + Torin 1) versus vehicle-treated (DMSO) cells, a gain or loss in mTOR associations were denoted as significant if one of the following criteria were met: i) proteins found in at least 2 of 3 replicates in both conditions and display a fold-change greater to or equal to 5 or less than or equal to -5 with a significant associated p-value < 0.05 for comparison of mean TPIs; ii) proteins found in at least 2 of 3 replicates in one condition and 1 replicate in the second group and display a fold-change greater to or equal to 5 or less than or equal to -5 between the two groups (no p-value can be calculated); or iii) detected in at least 2 of 3 replicates from one group and completely absent in the other (no statistical values can be calculated).

### Publication Abstract
A growing number of studies support a direct role for nuclear mTOR in gene regulation and chromatin structure. Still, the scarcity of known chromatin-bound mTOR partners limits our understanding of how nuclear mTOR controls transcription. Herein, comprehensive mapping of the mTOR chromatin-bound interactome in both androgen-dependent and -independent cellular models of prostate cancer (PCa) identifies a conserved 67-protein interaction network enriched for chromatin modifiers, transcription factors, and SUMOylation machinery. SUMO2/3 and nuclear pore protein NUP210 are among the strongest interactors, while the androgen receptor (AR) is the dominant androgen-inducible mTOR partner. Further investigation reveals that NUP210 facilitates mTOR nuclear trafficking, that mTOR and AR form a functional transcriptional module with the nucleosome remodeling and deacetylase (NuRD) complex, and that androgens specify mTOR-SUMO2/3 promoter-enhancer association. This work identifies a vast network of mTOR-associated nuclear complexes advocating innovative molecular strategies to modulate mTOR-dependent gene regulation with conceivable implications for PCa and other diseases.

### Keywords
Human, Rwpe-1, Pc3, Lncap, Chip-ms, 22rv1, Du145, Rime, Normal prostate, Prostate cancer, Mtor

### Affiliations
McGill University - Rosalind & Morris Goodman Cancer Research Centre
McGill University

### Submitter
Vincent Giguere

### Lab Head
Dr Vincent Giguere
McGill University - Rosalind & Morris Goodman Cancer Research Centre


