### Accession
PXD026035

### Title
Proteomics analysis of cellular nucleic acid binding proteins

### Description
Organic solvents and Ti4+-IMAC capture of formaldehyde (FA) cross-linked cells followed by mass spectrometry analysis to identify nucleic acid binding proteins in human cells.

### Sample Protocol
Cells were washed two times with PBS and cross-linked by formaldehyde, then lysate by TRIZOL regent. Organic solvent was used to precipitate NAs-protein complex crude, and Ti4+-IMAC was used to capture NAs-protein complex. Proteins were de-cross-linked from NAs-protein complexes by heat at 65 Â°C for 6 h. Then digested by using Filter-aided sample preparation (FASP) approach. Proteins obtained from formaldehyde cross-linking and Organic solvent extraction optimized process was analyzed by Easy-nLC 1000 coupled with an LTQ Orbitrap XL mass spectrometer (Thermo Fisher Scientific, USA).  Proteins obtained from method comparation (293T total cell lysate, organic solvent extraction, Ti4+-IMAC capture) and method application (95C, 95D, total cell lysate and Ti4+-IMAC capture) were analyzed by Easy-nanoLC1000 coupled with an Obitrap Q-Exactive Plus mass spectrometer (Thermo Fisher Scientific, USA).

### Data Protocol
The raw data were analyzed with MaxQuant software by label-free quantification (LFQ) using raw data files of MS/MS spectra searched against the UniProtKB human database (release 2017_11_28, 20244 entries). The search followed an enzymatic cleavage rule of Trypsin/P and allowed maximal two missed cleavage sites and a mass tolerance of 20 ppm for fragment ions. Carbamidomethylation of cysteines was defined as fixed modification, while protein N-terminal acetylation and methionine oxidation were defined as variable modifications for database searching. Maximum FDR, calculated by employing a reverse database strategy, was set at 1% for peptides and proteins. Label-free quantification was performed by Intensity-based absolute quantification.

### Publication Abstract
Profiling the nucleic acid-binding proteins (NABPs) during aging process is critical to elucidate its roles in biological systems as well as transcriptional and translational regulation. Here, we developed a comprehensive strategy to survey the NABPs of mouse immune organs by using single cell preparation and selective capture technology-based proteomics. Our approach provided a global view of tissue NABPs from different organs under normal physiological conditions with extraction specificity of 70 to 90%. Through quantitative proteomics analysis of mouse spleen and thymus at 1, 4, 12, 24, 48, and 72&#xa0;weeks, we investigated the molecular features of aging-related NABPs. A total of 2674 proteins were quantified in all six stages, demonstrating distinct and time-specific expression pattern of NABPs. Thymus and spleen exhibited unique aging signatures, and differential proteins and pathways were enriched across the mouse lifespan. Three core modules and 16 hub proteins associated with aging were revealed through weighted gene correlation network analysis. Significant candidates were screened for immunoassay verification, and six hub proteins were confirmed. The integrated strategy pertains the capability to decipher the dynamic functions of NABPs in aging physiology and benefit further mechanism research.

### Keywords
Nucleic acid binding protemoics, Human lung cancer, Lc/ms-ms

### Affiliations
State Key Laboratory of Microbial Metabolism,School of Life Sciences and Biotechnology, Shanghai Jiao Tong University, Shanghai, China
18272583259

### Submitter
Huiyu Wang

### Lab Head
Dr Hua Xiao
State Key Laboratory of Microbial Metabolism,School of Life Sciences and Biotechnology, Shanghai Jiao Tong University, Shanghai, China


