### Accession
PXD039885

### Title
A mouse model of Fragile X Syndrome analyzed by quantitative proteomics: a comparison of methods

### Description
We have compared the performance of seven different strategies in the analysis of a mouse model of Fragile X Syndrome, involving the knockout of the fmr1 gene that is the leading cause of autism spectrum disorder. Focusing on the cerebellum, we show that Data-Independent Acquisition (DIA) and the TMT-based Real-time Search method (RTS) generated the most informative profiles, generating 334 and 329 significantly altered proteins respectively, although the latter still suffered from ratio compression. Label-free methods such as BoxCar and a conventional Data-Dependent Acquisition were too noisy to generate a reliable profile, while TMT methods that do not invoke RTS showed a suppressed dynamic range. The TMT method based on complementary ions (ProC) overcomes ratio compression, but current limitations in ion detection reduces sensitivity. Overall, both DIA and RTS uncovered known regulators of the syndrome and detected alterations in calcium signalling pathways that are consistent with calcium deregulation recently observed in imaging studies.

### Sample Protocol
Cerebella of three mice were combined for a given biological replicate and four biological replicates were prepared for each condition (WT and Fmr1 KO). Each sample was cryopulverized and lysed by sonication. Samples were reduced with DTT and alkylated with CAA. An SP3 protocol was used for cleanup and on-beads digestion with Trypsin/LysC.  For TMT methods, samples were labeled with TMTpro reagents according to manufacturers instructions and fractionated on a high pH reverse phase column. For LC-MS, peptides were loaded onto an EASY-nLC 1200 pump and separated on an EASY-Spray Column (50cm, 2 Î¼m) with a gradient of 80% ACN in 0.1% FA consisting of three linear steps: 5-10% B over 5 min, 10-35% over 50 min and 35-95% over 10 min. All experiments were performed on an Orbitrap Fusion Eclipse instrument. TMT-MS2. The MS was operated in Cycle Time mode where MS1 OT scan (120K resolution, 375-1600 m/z range, 100% AGC target, 50 msec maximum injection time) was followed by as many OT MS2 scans as possible in a 3 s, for intensities between 25000+ (precursor fit error maximum 50%). Ions with a charge state 2-6 were selected for fragmentation (35% HCD fragmentation energy, 0.7 Th isolation window, 50K resolution, 200% AGC target, 200 ms maximum injection time, first mass scan range mode starting at 110 m/z). Dynamic exclusion was set to 60 s. TMT-SPS. The MS was operated as above with the following changes.  A 50% normalized AGC target was used, with a 30 msec maximum injection time. The parameters used for IT-MS2 scan were: 32% HCD fragmentation energy, 0.7 Th isolation window, turbo scan rate, 100% AGC target, 20 msec maximum injection time, scan range 200-1400. Precursors were selected with intensity 5000+. For MS3, the SPS mode was used to select the 10 most intense fragments from the MS2 using isobaric tag exclusion filtering with TMTpro (precursor exclusion with 25 ppm tolerance). Ions were fragmented simultaneously and analyzed as follows: MS3 isolation window 2 Th, HCD fragmentation energy 65%, resolution 50K, scan range 100-500, AGC target 200% and maximum injection time 105 msec. TMT-RTS. The parameters for TMT-RTS were the same as for TMT-MS3 with the exception being inclusion of the RTS filter which was set to only allow identifiable PSMs for fragmentation by SPS-MS3. The SP mouse proteome database with all the applied modification and maximum 2 misscleavages was used. Maximum mass error was set to 10 ppm. Searches exceeding 35 msec were aborted. TMT-proC. The MS was operated as for TMT-MS2, with the following changes: only 2+ ions (mass range 500-1074, intensity 190000 â€“ 1E+20) and 3+ ions (mass range 375-1381, intensity 470000+) were selected. For OT-MS2, the resolution was set to 30K, isolation window was 0.4 Th, and AGC target of 250% with a maximum injection time 100 ms and stepped HCD with a combination of 20, 25 and 30% collision energies was used. LFQ-DDA. Peptides were separated over 180 min gradient:  5-30% B over 150 min, 30-45% over 25 min and 45-100% over 5 min. The MS was operated in OT/IT mode with Cycle Time settings (120K resolution, 375-1500 m/z scan range, 100% normalized AGC target, 50 ms maximum injection time), where as many ion trap MS2 scans were collected in 3 s as possible (35% CID collision energy, 1.6 Th isolation window, Rapid ion trap scan rate, 100% normalized AGC target, 35 ms maximum injection time) for ions with a charge state of 2-6, intensity 5000+, precursor fit error maximum of 70%. Dynamic exclusion was set for 60s. BoxCar. Peptides were separated as follows:  2-5% B over 2 min, 5-35% over 93 min and 35-95% over 5 min. For library creation DDA method was used with Cycle Time settings, where each MS1 scan (60K resolution, 375-1500 scan range, 100% normalized AGC target, 50 msec maximum injection time) was followed by as many OT-MS2 scans as possible in a 3 s (precursor charge state 2-5, precursor intensity 25000+, maximum precursor fit error 70%, isolation window 1.6 Th, Stepped HCD fragmentation energies 25, 30 and 35%, orbitrap resolution 15K, normalized AGC target 100%, maximum injection time 25 msec). For samples, each MS1 scan (OT resolution 120K, 375-1650 mass range, 200% normalized AGC target, 20 ms maximum injection time, repeated every 3s) was followed by three series of a BoxCar scans (OT resolution 120K, normalized AGC target 100%, maximum injection time 246 ms). DIA. The peptides were separated as in BoxCar. The data were collected in approximately 3s cycles where each MS1 orbitrap scan (120K resolution, 380-1400 scan range, normalized AGC target 100%, maximum injection time 50 ms) was followed by a series of DIA scans where m/z transmission windows of variable width were set, according to the signal distribution of tryptic peptides typical for this experiment. Fragmentation used the following parameters: HCD fragmentation energy 30%, resolution 30K, scan range 145-1450, normalized AGC target 1000%, maximum injection time 54 ms.

### Data Protocol
The TMT-MS2 reporter ion quantification data were processed using Proteome Discoverer 2.5. Results were searched with the Sequest algorithm against the SwissProt mouse proteome database combined with the common contaminants included in MaxQuant 2.1.0.0. The combined database was concatenated with its reversed sequences for FDR calculation and search allowed for 2 missed cleavages and peptide length 6-144. Methionine oxidation, N-terminal acetylation, and N-terminal methionine loss were set as dynamic modifications. Carbamidomethylation of cysteines and TMTpro on any N-terminus or lysine were set as static modifications. The maximum allowed mass error was 10 ppm for precursors and 0.05 Da for fragments. Identified peptides and proteins were filtered for maximum 1% FDR. Proteins were quantified by summing reporter ion abundances across all matching PSMs belonging to unique peptides. PSMs were filtered for isolation specificity (50% or higher) and average reporter S/N (5 or greater). Proteins with any quantification value and possessing 2 or more unique peptides were considered quantified. TMT-MS3. The data were processed using Proteome Discoverer 2.5 with the same parameters as the TMT-MS2 method except the maximum allowed mass error for fragments was set to 0.6 Da to accommodate for the lower precision of the ion trap MS2 scan, and the allowed isolation interference was relaxed to 75%. TMT-RTS. The data were processed using Proteome Discoverer 2.5 as with the TMT-MS3 method except the isolation interference filter was not used. TMTproC. Data were processed using MaxQuant 2.1.0.0 and the complementary ion clusters were deconvoluted and quantified using software published by the WÃ¼hr group. Results were searched against the SwissProt mouse proteome database combined with the common contaminants database included in MaxQuant 2.1.0.0 (for the mouse cerebellum samples) or against a combination of human and yeast (strain ATCC 204508 / S288c) SwissProt proteome databases with the common contaminants for method optimization. Two missed cleavages and a minimum peptide length of 6 were allowed in the searches. Methionine oxidation was set as variable modification. Carbamidomethylation of cysteines and TMTpro on any N-terminus or lysine were set as fixed modifications. The rest of the parameters in MaxQuant were kept at the pre-set values for orbitrap instruments. The results were filtered for 1% protein and peptide level FDR and requiring at least 2 unique peptides per protein. All unique peptide PSMs were then used for quantification in the TMTproC software package, run with the pre-set parameters. The recommended total S/N cutoff of 40 was used to filter out all low intensity data. A protein quantification value was determined as a median quantification value of all PSMs belonging to the protein. LFQ-DDA. The data were processed using MaxQuant 2.1.0.0 using the default values for orbitrap instruments and â€œStandard LFQâ€� quantification. The results were searched against the SwissProt mouse proteome database combined with the common contaminants database. Two missed cleavages and a minimum peptide length 6 were allowed in the search. Methionine oxidation was set as variable modification. Carbamidomethylation of cysteines were set as fixed modification. Only unique peptides were used for quantification. The results were filtered for 1% protein and peptide level FDR and at least 2 unique peptides per protein. BoxCar. The data for both library and BoxCar runs were analyzed with MaxQuant 2.1.0.0 using the BoxCar mode for label-free quantification, with match-between-runs applied. For the library, the database search and filtering of the results used the same parameters as described above for the DDA method. DIA.The data was analyzed by DIA-NN software version 1.8.127, using both library-based and library-free modes, and match-between-runs was enabled. The SwissProt mouse proteome database was searched, combined with the common contaminants database included in MaxQuant 2.1.0.0. The maximum number of trypsin missed cleavages was set to 2, allowed peptide lengths was set to 7-30 and precursor charge states to 2-4. Methionine oxidation was set as variable modification. Carbamidomethylation of cysteines were set as fixed modification. One variable modification per peptide was allowed. The â€œhigh accuracyâ€� mode was used for quantification. The rest of the parameters were kept on the default values. The results were filtered at 1% FDR on protein and peptide level. Only protein unique peptides were used for quantification and only proteins with at least two unique peptides were considered identified.

### Publication Abstract
Multiple methods for quantitative proteomics are available for proteome profiling. It is unclear which methods are most useful in situations involving deep proteome profiling and the detection of subtle distortions in the proteome. Here, we compared the performance of seven different strategies in the analysis of a mouse model of Fragile X Syndrome, involving the knockout of the <i>fmr1</i> gene that is the leading cause of autism spectrum disorder. Focusing on the cerebellum, we show that data-independent acquisition (DIA) and the tandem mass tag (TMT)-based real-time search method (RTS) generated the most informative profiles, generating 334 and 329 significantly altered proteins, respectively, although the latter still suffered from ratio compression. Label-free methods such as BoxCar and a conventional data-dependent acquisition were too noisy to generate a reliable profile, while TMT methods that do not invoke RTS showed a suppressed dynamic range. The TMT method using the TMTpro reagents together with complementary ion quantification (ProC) overcomes ratio compression, but current limitations in ion detection reduce sensitivity. Overall, both DIA and RTS uncovered known regulators of the syndrome and detected alterations in calcium signaling pathways that are consistent with calcium deregulation recently observed in imaging studies. Data are available via ProteomeXchange with the identifier PXD039885.

### Keywords
Quantitative proteomics, Complementary ion quantification, Tmt labeling, Method comparison, Fragile x syndrome, Label-free, Mouse brain proteomics, Boxcar, Dia

### Affiliations
University of Calgary
Department of Biochemistry and Molecular Biology, Cumming School of Medicine, University of Calgary, Calgary, AB, Canada

### Submitter
Ruzena Filandrova

### Lab Head
Dr David Christopher Schriemer
Department of Biochemistry and Molecular Biology, Cumming School of Medicine, University of Calgary, Calgary, AB, Canada


