### Accession
PXD015753

### Title
Protein phosphorylation networks in spargana of Spirometra erinaceieuropaei revealed by phosphoproteomic analysis

### Description
Background: Sparganosis caused by Spirometra erinaceieuropaei spargana is a zoonotic parasitic infection which has been reported in many countries, such as China, Japan, Thailand and Korea, as well as the Europe and United States. This parasite biological and clinical significances, genome, and transcriptome analysis have previously been investigated, but its phosphoproteomes have not been reported. Here, we investigated global and site-specific phosphoproteome profiling of the spargana. Results: 3228 phosphopeptides and 3461 phosphorylation sites were identified from 1758 spargana proteins. The annotated phosphoproteins were involved in the various biological mechanisms, which included the cellular, metabolic and single-organism processes. Additionally, functional enrichment of phosphopepetides in Gene Ontology analysis suggested that most spargana phosphoproteins were related to the cytoskeleton cellular compartment, the signaling molecular function, and a variety of biological processes, including a molecular function regulator, guanyl-nucleotide exchange factor activity, protein kinase activities, and calcium ion binding. The highly enriched pathway of phosphorylation proteins included the phosphatidylinositol signaling system, phagosome, endocytosis, inositol phosphate metabolism, terpenoid backbone biosynthesis, and peroxisome pathways. Domain analysis of phosphopeptides identified EF-hand domain and pleckstrin homology domain among the important domains. Conclusions: This study provides the first global phosphoproteomic analysis in the spargana. The reported dataset will shed light on future understanding of zoonotic parasite.

### Sample Protocol
Protein digestion Tryptic digestion of proteins was performed as described by Ren et al. [18] with minor modifications. The protein solution was reduced with 5 mM dithiothreitol for 30 min at 56°C. It then was alkylated with 11 mM iodoacetamide for 15 min at room temperature in darkness. The protein sample was diluted by to urea concentration (100 mM NH4HCO3 added). Finally, trypsin was added for first digestion (1:50 trypsin-to-protein mass ratio) overnight and second digestion (1:100 trypsin-to-protein mass ratio) for 4 h.  HPLC fractionation The tryptic peptides were divided into fractions by high pH reversed-phase HPLC using the Thermo Betasil C18 column (5 μm particles, 10 mm ID, and 250 mm length, Thermo Fisher Scientific, Waltham, MA). Initially, peptides were separated into 60 fractions with an 8% to 32% gradient of acetonitrile (pH 9.0) over 60 min. The resulting peptides were combined into 7 fractions and dried by vacuum centrifuging. Phosphopeptide enrichment Peptides enrichment modified by phosphorylation, and all the tryptic peptides were dissolved in NETN buffer (100 mM NaCl, 1 mM EDTA, 50 mM Tris-HCl, 0.5% NP-40, pH 8.0) were incubated with pre-washed IMAC antibody beads at 4°C overnight with gentle shaking. Next, the beads were washed four times with NETN buffer followed by two washes with H2O. The bound peptides were eluted from the beads with 0.1% trifluoroacetic acid. Finally, we combined and vacuum-dried the eluted fractions. For liquid chromatography-tandem mass spectrometry (LC-MS/MS) analysis, the resulting peptides were desalted with C18 ZipTips (Millipore). All the experiment are according to the manufacturer’s instructions and then analyzed by two-dimensional LC-MS/MS [19].

### Data Protocol
NanoLC-MS/MS analysis The tryptic peptides were dissolved in 0.1% formic acid (solvent A) and directly loaded on a homemade reversed-phase analytical column (15 cm length, 75 μm i.d.). The solvent B (0.1% formic acid in 98% acetonitrile) gradients were initially increased from 6% to 23% over 26 min, from 23% to 35% over 8 min, and then elevated to 80% over 3 min before holding at 80% for the final 3 min, all at a constant flow rate of 400 nL/min on an EASY-nLC 1000 ultra-performance liquid chromatography (UPLC) system. The peptides were subjected to the nano-spray-ionization (NSI) source followed by tandem mass spectrometry (MS/MS) in Q ExactiveTM Plus (Thermo Fisher Scientific, Germany) coupled online to the UPLC system. The m/z scan range was 350 to 1800 for the full scan and electrospray voltage applied was 2.0 kV. Intact peptides were detected at a resolution of 70,000. Peptides were then selected for MS/MS using 28% nominal collision energy (NCE).The fragments were detected in the Orbitrap at a resolution of 17,500. A data-dependent procedure that alternated between one MS scan followed by 20 MS/MS scans was initiated for the top 20 precursor ions above a 5.0E3 threshold ion count in the MS survey scan with 15.0 s dynamic exclusion. Automatic gain control (AGC) was set at 5E4. Phosphopeptides identification  The mass spectrometry data were retrieved by Maxquant (v1.5.2.8). Parameter settings for retrieval were set by comparing the transcriptome database generated in this study with a reverse library search to identify the false discovery rate (FDR) caused by random matching. The common contamination library was also paralleled to the database to eliminate the contaminated protein influence on the identification results. The enzyme digestion mode was specified as Trypsin/P, and the number of missing cut-points was set at 2. The minimum length of the peptide segment was set as 7 amino acid residues with the maximum modification number of peptide segments set to 5. The first and the main search were customized to 20 ppm and 5 ppm, respectively, and the mass error tolerance of secondary debris ions was 0.02 Da. Cysteine alkylation was set as a fixed modification reference, while methionine oxidation, n-terminal acetylation of proteins, and phosphorylation of serine, threonine, and tyrosine were incorporated as alterable modification references. FDR for both protein identification and peptide-spectrum match (PSM) identification was 1%.

### Publication Abstract
None

### Keywords
Spirometra erinaceieuropaei, Spargana, Imac, Phosphoproteome, Mass spectrometry

### Affiliations
College of Veterinary Medicine, South China Agricultural University, Guangzhou, Guangdong Province 510642, PR China;
LYS

### Submitter
Yisong Liu

### Lab Head
Dr Wei Liu
College of Veterinary Medicine, South China Agricultural University, Guangzhou, Guangdong Province 510642, PR China;


