### Accession
PXD018375

### Title
Global mapping of Salmonella enterica-host protein-protein interactions during infection

### Description
Intracellular bacterial pathogens inject a cocktail of effector proteins into host cells to hijack diverse cellular processes and promote their survival and proliferation. Salmonella enterica serovar Typhimurium (STm) uses two Type 3 secretion system (T3SS) to deliver a suite of effector proteins into host cells, however, knowledge of their host-targets remains sparse. To systematically map STm effector-host protein-protein interactions (PPIs) during active infection, we generated a library of 32 STm strains expressing affinity-tagged effector proteins from their endogenous chromosomal loci. Tagged-effector strains were then used to infect macrophages or epithelial cells, and PPIs were detected by Affinity-Purification Quantitative Mass-Spectrometry (AP-QMS) under both native and cross-linking conditions. We recovered 25 previously described effector-host PPIs, along with 421 novel interactions across the two cell lines tested, as well as several effector-effector PPIs. While effectors converged on specific host cellular processes, which varied across cell types, most had multiple host targets. Using reciprocal co-immunoprecipitations, we could validated 13 out of 22 new PPIs, demonstrating an accuracy of at least 59% for the AP-QMS approach. To illustrate the utility of this resource for discovering novel infection biology, we first showed that SseJ and SseL cooperate to control cholesterol transport at the Salmonella Containing Vacuole (SCV) via the Niemann-Pick C1 protein (NPC1). Second, we uncovered that PipB directly binds and recruits the organelle contact site protein PDZD8 to the SCV. Third, we elucidated a novel mechanism whereby the effector kinase SteC promotes host actin bundling, by directly interacting and phosphorylating formin-like proteins. Overall, we provide a systems-wide host-bacterial pathogen physical interactome resource, to our knowledge the first in an infection context, with broad implications for effector function and cooperation.

### Sample Protocol
Within each TMT-10plex, untagged control (wildtype), as well as 9 STF-effector strains were assessed in parallel (RAW264.7 run 1: WT, PipB, PipB2, SifA, SseJ, SseL, SspH1, SteC, SlrP, run 2: WT, AvrA, GogB, SipB, SpvC, SseI, SseK1, SspH2, SteA, SteE; HeLa run 1: WT, PipB, PipB2, SifA, SseJ, SseL, SspH1, SspH2, SteC, SlrP, run 2: WT, AvrA, GogB, SifB, SpvC, SseF, SseI, SseK1, SseK2, SteA). For each run, all STF-tagged effector strains were seeded and infected at the same time. Prior to Mass Spectrometry, 1ÂµL of the elution fractions were used in Western Blot to validate the presence of the effector bait. Total protein concentration was determined using the Pierce Micro BCA kit, according to the manufacturerâ€™s protocol. All samples were adjusted to containing 10 Âµg protein in 50 ÂµL volume and were subsequently submitted to the EMBL Proteomics Core Facility. After reduction of disulfide bridges using 10 mM dithiothreitol at 56Â°C for 30 min in HEPES buffer (50 mM HEPES, pH 8.5), alkylation was performed using 20 mM 2-chloroacetamide at room temperature in HEPES buffer for 30 min under exclusion of light. Samples were prepared according to the SP3 protocol (Hughes et al., 2019) and trypsinized (sequencing grade, Promega) at an enzyme to protein ratio 1:50 for overnight digestion at 37Â°C. Subsequently, peptides were recovered in HEPES buffer by collecting supernatant on magnet and combining it with a second elution wash of the magnetic beads with HEPES buffer. Peptides were labelled with TMT10plex (Werner et al., 2014) Isobaric Label Reagent (ThermoFisher) according the manufacturerâ€™s instructions. In short, 0.8mg reagent was dissolved in 42 ÂµL acetonitrile (100%) and 4 ÂµL of this stock were added to the samples and incubated for 1 h at room temperature. The reaction was then quenched with 5% hydroxylamine for 15 min. Samples were pooled for the TMT-10plex and then further cleaned using OASISÂ® HLB ÂµElution Plate (Waters). Subsequently, offline high pH reverse phase fractionation was performed on an Agilent 1200 Infinity high-performance liquid chromatography system, using a Gemini C18 column (3 Î¼m, 110 Ã…, 100 x 1.0 mm, Phenomenex) with 20 mM ammonium formate (pH 10.0) and 100% acetonitrile as mobile phase (Reichel et al., 2016). The first and two last fractions were discarded prior to LC-MS analysis.  Samples were analyzed on an UltiMate 3000 RSLC nano LC system (Dionex) using a Âµ-Precolumn C18 PepMap 100 trapping cartridge (5Âµm, 300 Âµm i.d. x 5 mm, 100 Ã…) and a nanoEaseâ„¢ M/Z HSS T3 column 75 Âµm x 250 mm C18 as analytical column (1.8 Âµm, 100 Ã…, Waters). After trapping with a constant flow of 0.1% formic acid in water at 30 ÂµL/min onto the trapping column for 6 min, elution was carried out via the analytical column at a constant flow of 0.3 ÂµL/min with increasing percentage of solvent (0.1% formic acid in acetonitrile): from 2% to 4% in 4 min, from 4% to 8% in 2 min, then 8% to 28% for a further 96 min, and finally from 28% to 40% in another 10 min. The analytical column was coupled to QExactive plus (Thermo) mass spectrometer and mass-spec was performed according to previously described parameters (Perez-Perri et al., 2018).

### Data Protocol
IsobarQuant (Franken et al., 2015) and Mascot (v2.2.07) were used to process the acquired data. Peptide search was performed against a Uniprot Homo sapiens proteome database (UP000005640, for HeLa cell samples) or a Uniprot Mus musculus database (UP000000589, for RAW264.7 cell samples), combined with the Salmonella typhimurium (strain 14028s / SGSC 2262) (UP000002695) database containing common contaminants and reversed sequences. The following modifications were included in the search parameters: Carbamidomethyl (C) and TMT10 (K) as fixed modifications, acetyl (protein N-terminus), oxidation (M) and TMT10 (N-terminal) as variable modifications. Mass error tolerance was set as follows: 10ppm for the full scan (MS1) and 0.02Da for MS/MS (MS2) spectra. In addition, a maximum of two missed cleavages were allowed for trypsin, a minimum peptide length of seven amino acids was required and the false discovery rate (fdr) on peptide and protein level was set to 0.01. The output files of IsobarQuant (Franken et al., 2015) were analyzed using the R programming language (ISBN 3-900051-07-0). Only proteins that were quantified with at least two unique peptides and identified in at least two out of three biological replicates were kept for further analysis. The â€˜signal_sumâ€™ columns were first annotated to their biological conditions and then a median across all conditions per replicate was computed for each protein. First, potential batch-effects were removed using the â€˜removeBatchEffectâ€™ function of the limma package (Ritchie et al., 2015). Second, data were normalized with a variance stabilization normalization (vsn â€“ (Huber et al., 2002)). Finally, missing values were imputed using the impute function (method = â€œknnâ€�) of the Msnbase package (Gatto and Lilley, 2012). Limma was employed again to test for differential expression. Fold changes with respect to the median of the respective run were calculated for each protein in each pulldown. T-values from the limma output were pasted also into fdrtool (Strimmer, 2008) in order to compute alternative fdrs. In case the standard deviation of the t-values deviated from 1 to a degree that no convergence of statistically significant hits was observed, the q-values from the fdrtool output were used as alternative fdrs. A protein was annotated as a â€˜hitâ€™ with an fdr smaller than 1% and a fold increase of at least 20%; this was done for all four datasets (RAW264.7 native, RAW264.7 crosslinked, HeLa native and HeLa crosslinked) independently. This initial hitlist was then further refined in multiple steps: 1) PPIs were combined into two datasets, one for each cell line; 2) if a PPI passed the FC criterion in both conditions (native and crosslinked), the fdr requirement was loosened to fdr â‰¤ 0.05; 3) the resulting PPIs were ranked according to fdr and according to FC for each effector and each condition (native and crosslinked); 4) only PPIs that were in the top 20 for either FC or fdr were called â€œhitâ€�; 5) in addition, all PPIs that passed the FC requirement, as well as the loosened fdr requirement in both conditions were called a â€œhitâ€�.

### Publication Abstract
None

### Keywords
Salmonella, Ap/ms, Effectors, Bacterial pathogen

### Affiliations
EMBL
European Molecular Biology Laboratory (EMBL), Genome Biology Unit, Meyerhofstrasse 1, 69117 Heidelberg, Germany

### Submitter
Frank Stein

### Lab Head
Dr Dr. Athanasios Typas
European Molecular Biology Laboratory (EMBL), Genome Biology Unit, Meyerhofstrasse 1, 69117 Heidelberg, Germany


