### Accession
PXD008676

### Title
Pseudouridylation of tRNA-derived fragments steers translational control in stem cells

### Description
Pseudouridylation (Ψ) is the most abundant and widespread type of RNA epigenetic modification in living organisms; however, the biological role of Ψ remains poorly understood. Here, we show that a Ψ-driven posttranscriptional program steers translation control to impact stem cell commitment during early embryogenesis. Mechanistically, the Ψ ‘writer’ PUS7 modifies and activates a network of tRNA-derived fragments (tRFs) targeting the translation initiation complex. PUS7 inactivation in embryonic stem cells impairs tRF-mediated translational regulation leading to increased protein biosynthesis and abnormal germ layer specification. Remarkably, dysregulation of PUS7 and tRFs in myeloid malignancies associates with altered translation rates, suggesting a role of Ψ in leukemogenesis. Our findings unveil a critical function of Ψ in directing translational control in stem cells with important implications for human disease.

### Sample Protocol
One 15 cm plate of HEK 293T cells or one 10 cm plate of H9 hESC transfected with 20 nM of either biotinylated scramble oligo or mTOG-8, using RNAiMAX reagent (ThermoFisher) were washed with ice-cold PBS, UV-crosslinked at 254 nm with 200 mJ/cm2 and harvested. Cell pellets were lysed in Lysis Buffer (50 mM Tris/HCl pH 7.4, 100 mM NaCl, 0.5% Triton X-100, 0.5% sodium deoxycholate, 0.1% SDS, 5 mM EDTA, 0.1 U/uL RNase inhibitors (ThermoFisher) and protease inhibitors (Sigma-Aldrich)). Lysates were sonicated 3x10 sec at 10 W and cleared by centrifugation. Cleared extracts were incubated with pre-washed streptadivin-C1 dynabeads (ThermoFisher) for 1 h at 4°C, rotating. After incubation beads were washed twice in Lysis buffer, twice in High-salt buffer (50 mM Tris/HCl, 1000 mM NaCl, 0.5% Triton X-100, 0.25% sodium deoxycholate, 1 M Urea, 5 mM EDTA, 1 mM DTT) and twice in PNK/Tween buffer (50 mM Tris/HCl, 10 mM MgCl2, 0.2% Tween-20). To elute, beads were resuspended in 1.5X Laemli buffer with 100 mM DTT and incubated at 75°C for 10 min shaking at 1100 rpm. Proteins were precipitated on ice for 1 h in 20% trichloroacetic acid (TCA) and washed twice in 100% ice-cold acetone. Protein pellets were dissolved in 0.1% RapiGest (Waters) in 50 mM ammonium bicarbonate. Cysteine disulfide bonds were reduced with 5 mM DTT for 30 minutes at 56 ºC, alkylated with 10 mM iodoacetamide for 30 minutes at room temperature in the dark, and proteins were digested with sequencing grade modified trypsin (enzyme:protein ratio 1:50) at 37 °C overnight. The reaction was stopped by adding trifluoroacetic acid to a final concentration of 0.5% (v/v), and RapiGest was precipitated by further incubation at 37 °C for 45 min. Protein digests were dimethyl labeled on column as previously described with slight modifications. Briefly, SepPak C18 cartridges (Waters) were washed with acetonitrile and conditioned with 0.1% (v/v) formic acid. Acidified samples were loaded and washed with 0.1% formic acid. Samples were labeled by flushing the columns with labeling reagent (light, intermediate or heavy using CH2O (Sigma-Aldrich) + NaBH3CN (Sigma-Aldrich), CD2O (Sigma-Aldrich) + NaBH3CN (Sigma-Aldrich), or 13CD2O (Sigma-Aldrich) + NaBD3CN (Sigma-Aldrich), respectively). Labeling reagents were swapped between replicates N1, N2, N3 (light: 293T CTR N1, 293T mTOG-8 N3, H9 CTR N1, H9 mTOG-8 N2; intermediate: 293T CTR N3, 293T mTOG-8 N2, H9 CTR N2; heavy: 293T CTR N2, 293T mTOG-8 N1, H9 mTOG-8 N1. After washing with 0.1% formic acid, labeled peptides were eluted with 80% (v/v) acetonitrile/0.1% (v/v) formic acid. Samples were dried by vacuum centrifugation and reconstituted in 4% acetonitrile/0.1% formic acid prior to MS analysis. MS analyses were carried out on an Orbitrap Fusion Tribrid MS system (Thermo Scientific) equipped with a Proxeon Easy-nLC 1000 (Thermo Fisher). Peptides (1 µg) were separated on the analytical column (Acclaim PepMap RSLC C18 100 Å column, 2 µm, 75 µm ID x 250 mm length) at a constant flow of 300 nl/min. The mobile phases for LC separation were 0.1% (v/v) formic acid in LC-MS grade water (solvent A) and 0.1% (v/v) formic acid in acetonitrile (solvent B). During the elution step, the percentage of solvent B increased in a linear fashion from 5% to 10% in 2 minutes, then increased to 25% in 50 minutes and finally to 60% in a further 15 minutes. One full scan spectrum from m/z 375 to 1500 at resolution 60,000 FWHM was followed by MS/MS scans (resolution 15,000 FWHM) of the most intense ions (up to 10) from the full scan MS. The precursor ions were isolated with 1.6 m/z isolation width and fragmented using higher-energy collisional-induced dissociation at a normalized collision energy of 30%. The dynamic exclusion time was set to 30 s. The automatic gain control was set to 4e5 and 5e4 for MS and MS/MS, respectively, and ion accumulation times of 50 ms (MS) and 60 ms (MS/MS). The intensity threshold for precursor ion selection was 5e4.

### Data Protocol
MS raw data files were processed with MaxQuant (version 1.6.0.16). Enzyme specificity was set to trypsin/P and a maximum of two missed cleavages were allowed. Cysteine carbamidomethylation was selected as fixed modification; methionine oxidation and acetylation of protein amino-terminus were selected as variable modifications. The derived peak list was searched using the in-built Andromeda search engine in MaxQuant against the Uniprot human database together with 265 frequently observed contaminants (Andromeda configured database) and reversed sequences of all entries. Initial maximal allowed mass tolerance was set to 20 ppm for peptide masses, followed by 4.5 ppm in the main search, and 20 ppm for fragment ion masses. The minimum peptide length was set to seven amino acid residues. A 1% false discovery rate (FDR) was required at both the protein level and the peptide level. In addition to the FDR threshold, proteins were considered identified if they had at least one unique peptide. The protein identification was reported as an indistinguishable “protein group” if no unique peptide sequence to a single database entry was identified. The ‘match between runs’ was enabled for consecutive peptide fractions with a 1.5 minutes’ time window. Ratio count 1 was used. Contaminants, reverse hits and “hits-only-identified-by-site” were excluded. The iBAQ algorithm was used for estimation of the abundance of different proteins within a single sample (proteome).

### Publication Abstract
Pseudouridylation (&#x3a8;) is the most abundant and widespread type of RNA epigenetic modification in&#xa0;living organisms; however, the biological role of&#xa0;&#x3a8; remains poorly understood. Here, we show that a &#x3a8;-driven posttranscriptional program steers translation control to impact stem cell commitment during early embryogenesis. Mechanistically, the &#x3a8;&#xa0;"writer" PUS7 modifies and activates a novel network of tRNA-derived small fragments (tRFs) targeting the translation initiation complex. PUS7 inactivation in embryonic stem cells impairs tRF-mediated translation regulation, leading to increased protein biosynthesis and defective germ layer specification. Remarkably, dysregulation of this posttranscriptional regulatory circuitry impairs hematopoietic stem cell commitment and is common to aggressive subtypes of human myelodysplastic syndromes. Our findings unveil a critical function of &#x3a8; in directing translation control in stem cells with important implications for development and disease.

### Keywords
Trna, Pseudouridylation, Dimethyl labelling

### Affiliations
Proteomic Hematology Division of Molecular Hematology Lund Stem Cell Center, Lund University BMC B12, 221 84 Lund, Sweden
Lund University

### Submitter
Kristyna Pimkova

### Lab Head
Dr Jenny Pyl Hansson
Proteomic Hematology Division of Molecular Hematology Lund Stem Cell Center, Lund University BMC B12, 221 84 Lund, Sweden


