### Accession
PXD024459

### Title
Comparative proteome analysis of the dish pathogen Renibacterium salmoninarum grown under rich- and iron‐limited conditions

### Description
Proteomic analysis of iron restriction of the marine pathogen Renibacterium salmoninarum that causes the disease (BKD), in this study two isolated Chilean H-2 strains and the type strain ATCC33209 have been used. These results show that changes are generated at the level of protein expression in pathways involved in iron metabolism and uptake.

### Sample Protocol
Total protein extracts from all R. salmoninarum strains were prepared from frozen cellular pelleted and incubated in lysis buffer (50 mM Tris–HCl, pH 7.5; 150 mM NaCl; 1% NP-40; 0.5% sodium deoxicolate; and 1% SDS) for 1 h at 4ºC. Subsequently, the solution was sonicated in ice bath for 10 min at 20 kHz frequency and later lyophilized and stored at −20ºC until use. All samples were analyzed by SDS-PAGE 12% to evaluate protein sample integrity (Laemmli 1970).  The proteins were subjected to precipitation using 5: 1 v / v cold acetone 100% v/v and incubated overnight at -20 ° C, then they were centrifuged at 15,000 x g for 10 min, the supernatant was discarded, and the pellet was washed 3 times with acetone at 90% v / v, later the proteins were dried in a rotary concentrator at 4 ° C, and finally they were resuspended in 6 M guanidine hydrochloride (CH6ClN3) and 25 mM ammonium bicarbonate (NH4HCO3), pH 8. Subsequently, proteins were reduced at room temperature for 30 min with 2 mM dithiothreitol and alkylated in the dark at room temperature for 30 min with 10 mM iodoacetamide. The reaction was diluted 6 times with 25 mM NH4HCO3, pH 8; modified trypsin (Promega, Madison, WI, USA) was added in 1:50 ratio and the reaction incubated at 37 °C for 16 h. Finally 5% formic acid (CH2O2), pH 3.0 was added to stop the reaction. For protein identification by tandem mass spectrometry all samples were concentrated on a CentriVap Concentrator (Labconco, Kansas City, MO, USA) to a final volume of 2 µL (7 µg tryptic peptides) and then loaded on a 350 µm ID fused silica peptide trap column 3 cm of a reverse-phase desalting C18 (Thermo Scientific) . Following a wash with 0.1% formic acid for 30 min at 0.5 µL/min, the efflux of the peptide trap column was directed to a 10 cm resolving reversed-phase column (100 Å, 5 µm Magic C18 particles, Michrom Bioresources), mounted on the electrospray stage of a VELOSPRO mass spectrometer (LTQ, Thermo Finnigan LLC). A 0–35% acetonitrile gradient for 260 min at a flow rate of 600 nL/min was used (Dionex Ultimate 3000 nano series). An electrospray voltage of 2.2 kV was used, with the ion transfer temperature set to 270 °C. The mass spectrometer was controlled by the Xcalibur software, which continuously performed mass-scan analysis of the six most intense ions during MS/MS scans of the ion traps. For this, one repeats scan of the same ion was dynamically excluded, using a 30 s repeat duration and 90 s exclusion duration. Normalized collision energy for the MS/MS was set to 35%.

### Data Protocol
Data Analysis Using Database Search Algorithm All tandem mass spectra MS/MS samples were analyzed using SEQUEST (v1.4.0.288; Thermo Fisher Scientific, San Jose, CA, USA) and X! Tandem (vCYCLONE 2010.12.01.1; The GPM, thegpm.org). SEQUEST searched the National Center for Biotechnology Information (NCBI) Renibacterium_salmoninarum.fasta database (NC_010168.1) assuming digestion of the enzyme trypsin. X! Tandem searched a subset of the NCBI Renibacterium salmoninarum database, also assuming trypsin digestion. SEQUEST and X! Tandem were searched with a fragment ion mass tolerance of 0.80 Da and a parent ion tolerance of 2.5 Da. Carbamidomethyl-cysteine was a fixed modification in SEQUEST and X! Tandem. In SEQUEST, asparagine and glutamine deamidation and methionine oxidation were variable modifications. In X! Tandem, Glu->pyro-Glu of the Nterminus, ammonia-loss of the n-terminus, Gln->pyro-Glu of the N-terminus, asparagine and glutamine deamidation, and methionine oxidation were variable modifications.  Criteria for protein identification All tandem mass spectra MS/MS samples were analyzed using SEQUEST (v1.4.0.288; Thermo Fisher Scientific, San Jose, CA, USA) and X! Tandem (vCYCLONE 2010.12.01.1; The GPM, thegpm.org). SEQUEST searched the National Center for Biotechnology Information (NCBI) R. salmoninarum 05-04-2018 fasta database (6267 entries) assuming digestion of the enzyme trypsin. X! Tandem searched a subset of the R. salmoninarum NCBI 11-03-2020 database, also assuming trypsin digestion. SEQUEST and X! Tandem were searched with a fragment ion mass tolerance of 0.80 Da and a parent ion tolerance of 2.5 Da. Carbamidomethyl-cysteine was a fixed modification in SEQUEST and X! Tandem. In SEQUEST, asparagine and glutamine deamidation and methionine oxidation were variable modifications.  In  X!  Tandem,  Glu->pyro-Glu  of  the  N-terminus, ammonia-loss of the n-terminus, gln->pyro-Glu of the N-terminus, asparagine and glutamine deamidation, and methionine oxidation were variable modifications.     Criteria for protein identification and quantification   Using Scaffold v.4.5.0 (Proteome Software Inc., Portland, OR) we identified 207 high-confidence proteins (FDR <5% and 3 unique peptides). To determine which proteins were differentially and significantly expressed in the treatments applied to the two strains a Fisher Exact Test with a Bejamini-Horshberg correction was applied. Any protein whose associated p-value is <0.014 is considered significant.

### Publication Abstract
None

### Keywords
Fish disease, Lfq, Bkd, Iron

### Affiliations
Proteogenomics laboratory MELISA institute
Proteogenomics laboratory, MELISA Institute

### Submitter
Mauricio Hernandez

### Lab Head
Dr Mauricio Hernandez
Proteogenomics laboratory MELISA institute


