### Accession
PXD002451

### Title
Teleogryllus oceanicus (field cricket) wing bud proteomics: iTRAQ data

### Description
In Hawaii, a rapidly-evolving mutation in the field cricket Teleogryllus oceanicus silences males by interfering with the development of sound-producing structures on their forewings. The mutation is called flatwing (fw), and it persists because of natural selection imposed by an acoustically-orienting parasitoid. We examined gene expression differences between wild-type and mutant crickets, focusing on juvenile wing buds. We profiled mRNA expression levels using RNA-seq, and characterized the wing bud proteome using quantitative mass spectrometry.

### Sample Protocol
iTRAQ labelled peptides were fractionated by cation exchange chromatography, pooled into 7 fractions and desalted before analysison a 5600+ TripleTof. 3 biological replicates of wild type and mutant male crickets were compared, using iTRAQ labels 113-118

### Data Protocol
Data was processed using the Mascot search algorithm 2.5 against a wing bud transcriptome database

### Publication Abstract
A major advance in modern evolutionary biology is the ability to start linking phenotypic evolution in the wild with genomic changes that underlie that evolution. We capitalized on a rapidly evolving Hawaiian population of crickets (Teleogryllus oceanicus) to test hypotheses about the genomic consequences of a recent Mendelian mutation of large effect which disrupts the development of sound-producing structures on male forewings. The resulting silent phenotype, flatwing, persists because of natural selection imposed by an acoustically orienting parasitoid, but it interferes with mate attraction. We examined gene expression differences in developing wing buds of wild-type and flatwing male crickets using RNA-seq and quantitative proteomics. Most differentially expressed (DE) transcripts were down-regulated in flatwing males (625 up vs. 1716 down), whereas up- and down-regulated proteins were equally represented (30 up and 34 down). Differences between morphs were clearly not restricted to a single pathway, and we recovered annotations associated with a broad array of functions that would not be predicted a priori. Using a candidate gene detection test based on homology, we identified 30% of putative Drosophila wing development genes in the cricket transcriptome, but only 10% were DE. In addition to wing-related annotations, endocrine pathways and several biological processes such as reproduction, immunity and locomotion were DE in the mutant crickets at both biological levels. Our results illuminate the breadth of genetic pathways that are potentially affected in the early stages of adaptation.

### Keywords
Teleogryllus oceanicus; itraq

### Affiliations
Biomedical Science Research Complex
Biomedical Sciences Research Complex, University of St Andrews

### Submitter
Catherine Botting

### Lab Head
Dr Catherine Helen Botting
Biomedical Sciences Research Complex, University of St Andrews


