### Accession
PXD025204

### Title
TMT-Based Quantitative Proteomic Analysis Reveals the Crucial Biological Pathways Involved in Brain Injury Induced by Chest Exposure to Shock Waves

### Description
Long term chest blast exposure can lead to mental disorders, brain inflammation and oxidative stress in soldiers. However, the underlying mechanism of brain injury caused indirectly by chest blast remains unclear. It is urgent to find more potential biomarkers to reveal the deeper pathogenesis of this phenomenon. We used iTRAQ labeling combined with LC-MS/MS to screen potential differentially expressed proteins in rat brain after chest blast at different time points. Meanwhile, we also used Go, KEGG, David and Cytoscape to analyze the proteomic profile and explore its pathogenesis. Moreover, Western blotting was used to verify the target proteins. Our data showed that, a total of 6,931 proteins were identified. A total of 255 differentially expressed proteins were identified, of which 43, 84, 52, 97 and 49 proteins were identified from brain tissues at 12h, 24h, 48h, 72h, and 1w after chest blast exposure. Bioinformatics analysis, including GO, COG, KEGG and STRING, further proved that brain damage caused by chest blast exposure was involved in many important biological processes and signal pathways, such as inflammation, cell adhesion, phagocytosis, neuronal and synaptic damage, oxidative stress and apoptosis. Moreover, Western blotting further confirmed that these differentially expressed proteins and signaling pathways were associated with brain damage caused by chest blast exposure. For the first time, we screened and verified the potential protein biomarkers of brain damage caused indirectly by chest blast, and provided a new target for the treatment of this disease.

### Sample Protocol
Protein Extraction Brain sample was grinded by liquid nitrogen into cell powder and then transferred to a 5-mL centrifuge tube. After that, four volumes of lysis buffer (8 M urea, 1% Protease Inhibitor Cocktail) were added to the cell powder, followed by sonication three times on ice using a high intensity ultrasonic processor (Scientz). The remaining debris was removed by centrifugation at 12,000g at 4°C for 10 min. Finally, the supernatant was collected and the protein concentration was determined with the BCA kit according to the manufacturer’s instructions. Trypsin Digestion For digestion, the protein solution was reduced with 5 mM dithiothreitol for 30 min at 56°C and alkylated with 11 mM iodoacetamide for 15 min at room temperature in darkness. The protein sample was then diluted by adding 100 mM TEAB to urea concentration less than 2M. Finally, trypsin was added at 1:50 trypsin-to-protein mass ratio for the first digestion overnight and 1:100 trypsin-to-protein mass ratio for a second 4 h-digestion. TMT/iTRAQ Labeling After trypsin digestion, peptide was desalted by Strata X C18 SPE column (Phenomenex) and vacuum-dried. Peptide was reconstituted in 0.5 M TEAB and processed according to the manufacturer’s protocol for TMT kit/iTRAQ kit. Briefly, one unit of TMT/iTRAQ reagent was thawed and reconstituted in acetonitrile. Peptide mixtures were then incubated for 2 h at room temperature and pooled, desalted and dried by vacuum centrifugation.  HPLC Fractionation The tryptic peptides were fractionated into fractions by high pH reverse-phase HPLC using Agilent 300Extend C18 column (5 μm particles, 4.6mm ID, 250mm length). Briefly, peptides were first separated with a gradient of 8% to 32% acetonitrile (pH 9.0) over 60 min into 60 fractions. Then, peptides were combined into 18 fractions and dried by vacuum centrifuging. LC-MS/MS Analysis The tryptic peptides were dissolved in 0.1% formic acid (solvent A), directly loaded onto a home-made reversed-phase analytical column (15-cm length, 75 μm i.d.). The gradient was comprised of an increase from 6% to 23% solvent B (0.1% formic acid in 98% acetonitrile) over 26 min, 23% to 35% in 8 min and climbing to 80% in 3 min then holding at 80% for the last 3 min, all at a constant flow rate of 400 nL/min on an EASY-nLC 1000 UPLC system. The peptides were subjected to NSI source followed by tandem mass spectrometry (MS/MS) in Q ExactiveTM Plus (Thermo) coupled online to the UPLC. The electrospray voltage applied was 2.0 kV. The m/z scan range was 350 to 1800 for full scan, and intact peptides were detected in the Orbitrap at a resolution of 70,000. Peptides were then selected for MS/MS using NCE setting as 28 and the fragments were detected in the Orbitrap at a resolution of 17,500. A data-dependent procedure that alternated between one MS scan followed by 20 MS/MS scans with 15.0s dynamic exclusion. Automatic gain control (AGC) was set at 5E4. Fixed first mass was set as 100 m/z. Database Search All MS raw files from the same batch were processed together with MaxQant (ver. 1.5.8) against the SwissProt Mus musculus protein database (version 2018.08, 16,992 entries), concatenated with the reverse decoy database. Trypsin/P was specified as a cleavage enzyme allowing up to 2 missing cleavages, 5 modifications per peptide. The mass tolerance for precursor ions was set as 20 ppm in first search and 5 ppm in main search, and the mass tolerance for fragment ions was set as 0.02 Da. The mass error was set to 20 ppm and 0.02Da for precursor ions fragment ions, respectively.  Carbamidomethylation on Cys was specified as fixed modification and oxidation on Met, and acetylation on protein N-terminal were specified as variable modifications. The minimal peptide length was set as 7 residues. The false discovery rate (FDR) of peptide and protein were all set as 1%. Quantification of Global Proteome Data The quantification analysis was performed at the protein level by MaxQuant software. TMT reporter ion intensities of each peptide was normalized by average in all samples. Protein quantitation calculated from the median ratio of protein corresponding unique peptides when there were at least two unique peptides in a protein. Protein quantitation values were normalized by column-median to correct for equal loading across samples, and then log2-transformed. All normalization steps were performed in RStudio. Differentially Expressed Protein Analysis Student’s t test was used to examine whether proteins were differentially expressed between any two different group samples. Upregulated or downregulated proteins were defined as differentially expressed protein (DEP) in test compared control (ratio> 1.2 or ratio < 1/1.2, Student’s t test nominal p < 0.05). Volcano plot of differentially expressed proteins were plot by visualization R package “ggplot2”. All calculation and visualization steps were performed in RStudio.

### Data Protocol
GO Classification Gene Ontology (GO) annotation proteome was derived from the UniProt-GOA database (www. http://www.ebi.ac.uk/GOA/). Firstly, DEPs were mapped to GO IDs by protein accession. If some DEPs were not annotated by UniProt-GOA database, the InterProScan soft would be used to annotated protein’s GO functional based on protein sequence alignment method. Then DEPs were classified by Gene Ontology annotation based on three categories: biological process, cellular component and molecular function. A bar plot graph was used to present GO terms by visualization R package “ggplot2” in RStudio. KEGG Pathway Enrichment KEGG database was used to annotate protein pathway. Firstly, using KEGG online service tools KAAS to annotated protein’s KEGG database description. Then mapping the annotation result on the KEGG pathway database using KEGG online service tools KEGG mapper. DEPs enriched pathways were identified by a two-tailed Fisher’s exact test. The pathway with p value < 0.05 was considered significant. A bubble plot graph was used to present enriched pathway by visualization R package “ggplot2”. All calculation and visualization steps were performed in RStudio. Protein-Protein Interaction Network All DEPs accessions were searched against the STRING database version 11.0 for protein-protein interactions. Only interactions between the proteins belonging to the searched data set were selected, thereby excluding external candidates. STRING defines a metric called “confidence score” to define interaction confidence; we fetched all interactions that had a confidence score >0.7 (high confidence). Interaction network form STRING was visualized in CytoScape software.

### Publication Abstract
Recurrent chest blast exposure can lead to brain inflammation, oxidative stress, and mental disorders in soldiers. However, the mechanism that underlies brain injury caused indirectly by chest blasts remains unclear. It is urgent to find additional reliable biomarkers to reveal the intimate details of the pathogenesis of this phenomenon. We used the term tandem mass tag (TMT) labeling combined with liquid chromatography-tandem mass spectrometry (LC-MS/MS) to screen for differentially expressed proteins in rat brain at different time points after a chest blast. Data are available <i>via</i> ProteomeXchange with the identifier PXD025204. Gene Ontology (GO), the Kyoto Encyclopedia of Genes and Genomes (KEGG), the Database for Annotation, Visualization and Integrated Discovery (DAVID), and Cytoscape analyses were used to analyze the proteomic profiles of blast-exposed rats. In addition, we performed Western blotting to verify protein levels. We identified 6,931 proteins, of which 255 were differentially expressed and 43, 84, 52, 97, and 49 were identified in brain tissues at 12, 24, 48, and 72 h and 1 week after chest blast exposure, respectively. In this study, the GO, KEGG, Clusters of Orthologous Groups of proteins, and Search Tool for the Retrieval of Interacting Genes/Proteins (STRING) analyses indicated that brain damage caused by chest blast exposure involved many important biological processes and signaling pathways, such as inflammation, cell adhesion, phagocytosis, neuronal and synaptic damage, oxidative stress, and apoptosis. Furthermore, Western blotting confirmed that these differentially expressed proteins and affected signaling pathways were associated with brain damage caused by chest blast exposure. This study identifies potential protein biomarkers of brain damage caused indirectly by chest blast and new targets for the treatment of this condition.

### Keywords
Chest blast; brain injury; tandem mass spectrometry quantitative proteomics; inflammation; signaling pathways

### Affiliations
Shenyang Medical College; The Second Affiliated Hospital of Shenyang Medical College. The Veterans General Hospital of Liaoning Province
Shenyang Medical College

### Submitter
Changci Tong

### Lab Head
Dr Changci Tong
Shenyang Medical College; The Second Affiliated Hospital of Shenyang Medical College. The Veterans General Hospital of Liaoning Province


