### Accession
PXD040218

### Title
Proteomic analysis of lysine 2-hydroxyisobutyryl in SLE reveals protein modification alteration in Complement and coagulation cascades and Platelet activation pathways

### Description
Post-translational modifications (PTMs) are considered to be an important factor in the pathogenesis of SLE. Lysine 2-hydroxyisobutyryl (Khib), as an emerging post-translational modification of proteins, is involved in some important biological metabolic activities. We compared the Khib levels of SLE patients and healthy controls based on liquid chromatography-tandem mass spectrometry, and then performed proteomic analysis. The results showed that Khib in SLE patients was up-regulated at 865 sites of 416 proteins and down-regulated at 630 sites of 349 proteins. The site abundance, distribution and function of Khib protein were further analyzed. Bioinformatics analysis showed that complement, coagulation cascade and platelet activation in immune-related pathways were significantly enriched, indicating that the differential modification proteins between them might affect SLE.

### Sample Protocol
Add lysis buffer solution (8 M urea and 1% protease inhibitor cocktail) to the peripheral blood sample, obtain the extracted protein through ultrasonic lysis, centrifugation, removal of cell fragments, etc., and then measure the protein concentration through BCA kit. Trypsin was added to the prepared protein solution twice and split, and the peptide obtained after splitting was labeled with TMT reagent. The whole proteolytic peptide was separated by high performance liquid chromatography (HPLC). In order to enrich L-lysine-2-hydroxyisobutyryl-modified peptides, peptide fragments were added to NETN buffe (100 mM NaCl, 1 mM EDTA, 50 mM Tris-HCl, and 0.5% NP-40 at pH 8.0), and pretreated antibody beads were added. After gently shaking and culturing at 4 Â° C overnight, wash with NETN buffer and H 2O respectively. Then elute the combined peptide from the magnetic beads with 0.1% trifluoroacetic acid. Peptides were analyzed by LC-MS/MS, and specific protein information corresponding to mass spectrometry data was retrieved in Maxquant (v1.5.2.8), and then bioinformatics analysis was performed.

### Data Protocol
1)We used Wolfpsort(v.0.2), subcellular localization predication software, to predict the subcellular localization. 2)Motif analysis using MoMo (v5.0.2) and heat map using R Package (v.2.0.3). 3)Gene Ontology (GO) analysis using the DAVID website (http://david.ncifcrf.gov), and GO bubble maps were drawn on the Oebiotech website (https://cloud.oebiotech.cn/task/). 4)KEGG analysis was performed by Cytoscape (v3.8.2) software. The original KEGG pathway map was downloaded from the Kyoto Encyclopedia of Genes and Genomes (KEGG) (https://www.genome.jp/kegg/). Based on the original pathway maps, we used material from SERVER MEDICAL ART (https://smart.servier.com) to adjust and process the KEGG pathway maps. 5)STRING website (https://cn.string-db.org) was used to draw the PPI network diagram using Cytoscape (v3.8.2) software. Use the MCODE plug-in in Cytoscape to extract sub-networks and perform KEGG pathway enrichment analysis on the DMPs in the sub-networks.

### Publication Abstract
None

### Keywords
Human, Lysine 2-hydroxyisobutyryl, Pbmc, Sle

### Affiliations
Institute of Nephrology and Blood Purification, The First Affiliated Hospital of Jinan University, Jinan University, Guangzhou 510632, China

### Submitter
Chaoying Kuang

### Lab Head
Dr Chaoying Kuang
Institute of Nephrology and Blood Purification, The First Affiliated Hospital of Jinan University, Jinan University, Guangzhou 510632, China


