### Accession
PXD025505

### Title
Synergistic effect of short- and long-read sequencing on functional meta-omics

### Description
Evaluation of short-read-only, long-read-only, and hybrid assembly approaches on metagenomic samples demonstrating how they affect gene and protein prediction which is relevant for downstream functional analyses. For a human gut microbiome sample, we use complementary metatranscriptomic, and metaproteomic data to evaluate the metagenomic-based protein predictions.

### Sample Protocol
DNA, RNA and proteins were co-extracted using a methodological framework for the sequential extraction and purification of all biomolecular fractions from single unique sample.The snap-frozen fecal sample was homogenized in a liquid nitrogen bath and aliquoted into 0.7g aliquots for subsequent multi-omics extraction based on the Qiagen Allprep kit. After the DNA and RNA extraction, the proteins were precipitated by adding the APP Buffer. The pellet was then dissolved in ALO buffer and was used for metaproteomic analysis. 1Î¼g of extracted proteins was denatured and briefly loaded on a SDS gel to produce one gel band. The reduction, alkylation and tryptic digestion of the proteins into peptides were performed in-gel. The tryptic peptides were extracted from the gel and desalted prior to mass spectrometry analysis. Peptides were analyzed using a nanoLC-MS/MS system (120 minutes gradient) connected to a Q-Exactive HF orbitrap mass spectrometer equipped with a nano-electrospray ion source. The Q-Exactive mass spectrometer was operated in data-dependent mode and the 10 most intense peptide precursors ions were selected for fragmentation and MS/MS acquisition.

### Data Protocol
For each assembly separately and for all assemblies together, the FASTA file of predicted proteins was concatenated with a cRAP database of contaminants and with the human UniProtKB Reference Proteome prior metaproteomic search. In addition, reversed sequences of all protein entries were concatenated to the databases for the estimation of false discovery rates. The search was performed using SearchGUI-3.3.20 with the X!Tandem, MS-GF+ and Comet search engines and the following parameters: Trypsin was used as the digestion enzyme, and a maximum of two missed cleavages was allowed. The tolerance levels for matching to the database was 10 ppm for MS1 and 0.2 Da for MS2. Carbamidomethylation of cysteine residues was set as a fixed modification and protein N-terminal acetylation and oxidation of methionines was allowed as variable modification. Peptides with length between 7 and 60 amino acids, and with a charge state composed between +2 and +4 were considered for identification. The results from SearchGUI were merged using PeptideShaker-1.16.45 and all identifications were filtered in order to achieve a protein false discovery rate (FDR) of 1%.

### Publication Abstract
Real-world evaluations of metagenomic reconstructions are challenged by distinguishing reconstruction artifacts from genes and proteins present in situ. Here, we evaluate short-read-only, long-read-only and hybrid assembly approaches on four different metagenomic samples of varying complexity. We demonstrate how different assembly approaches affect gene and protein inference, which is particularly relevant for downstream functional analyses. For a human gut microbiome sample, we use complementary metatranscriptomic and metaproteomic data to assess the metagenomic data-based protein predictions. Our findings pave the way for critical assessments of metagenomic reconstructions. We propose a reference-independent solution, which exploits the synergistic effects of multi-omic data integration for the in situ study of microbiomes using long-read sequencing data.

### Keywords
Long-reads, Metaproteomics, Functional omics, Gdb_metap

### Affiliations
LCSB
Luxembourg Centre for Systems Biomedicine, University of Luxembourg, Luxembourg

### Submitter
Benoit Kunath

### Lab Head
Dr Paul Wilmes
Luxembourg Centre for Systems Biomedicine, University of Luxembourg, Luxembourg


