### Accession
PXD045313

### Title
Caffeine causes cell cycle arrest at G0/G1 and increases of ubiquitinated proteins, ATP and mitochondrial membrane potential in renal cells

### Description
Caffeine is a well-known purine alkaloid commonly found in coffee. Several lines of previous and recent evidence have shown that habitual coffee drinking is associated with lower risks for chronic kidney disease (CKD) and nephrolithiasis. However, cellular and molecular mechanisms underlying its renoprotective effects remain largely unknown due to a lack of knowledge on cellular adaptive response to caffeine. This study investigated cellular adaptive response of renal tubular cells to caffeine at the protein level. Cellular proteome of MDCK cells treated with caffeine at a physiologic concentration (100 Î¼M) for 24 h was analyzed comparing with that of untreated cells by label-free quantitative proteomics. From a total of 936 proteins identified, comparative analysis revealed significant changes in levels of 148 proteins induced by caffeine. These significantly altered proteins were involved mainly in proteasome, ribosome, tricarboxylic acid (TCA) (or Krebs) cycle, DNA replication, spliceosome, biosynthesis of amino acid, carbon metabolism, nucleocytoplasmic transport, cell cycle, cytoplasmic translation, translation initiation, and mRNA metabolic process. Functional validation by various assays confirmed that caffeine decreased cell population at G2/M, increased cell population at G0/G1, increased level of ubiquitinated proteins, increased intracellular ATP and enhanced mitochondrial membrane potential in MDCK cells. These data may help unravelling molecular mechanisms underlying the biological effects of caffeine on renal tubular cells at cellular and protein levels.

### Sample Protocol
Preparation of cellular proteins for proteomic analysis  MDCK renal tubular cells (ATCC; Manassas, VA) were grown in an MEM medium (Gibco; Grand Island, NY) supplemented with 10% heat-inactivated fetal bovine serum (Gibco), 60 U/ml penicillin G (Sigma-Aldrich; St. Louis, MO) and 60 Î¼g/ml streptomycin (Sigma-Aldrich). The culture was done at 37 Â°C in a humidified incubator with 5% CO2. The cells were seeded into a 6-well plate (approximately 5 Ã—105 cells/well). After 24-h incubation, the cells were treated with caffeine (Sigma-Aldrich) at 100 Î¼M for 24 h. The untreated cells served as the controls. Cellular proteins were then extracted with SDT lysis buffer (4% SDS, 100 mM DTT, and 100 mM Tris-HCl; pH 7.6), and protein concentrations were measured using Bio-Rad protein assay (Bio-Rad; Milano, Italy) based on Bradfordâ€™s method.   In-solution tryptic digestion and analyses of proteins by nanoflow liquid chromatography coupled to tandem mass spectrometry (nanoLC-ESI-LTQ-Orbitrap MS/MS)  An equal amount (30 Âµg) of total proteins from each sample was subjected to in-solution tryptic digestion as described previously (1, 2). The digested peptides were then analyzed by nanoLC-ESI-LTQ-Orbitrap MS/MS as previously reported (3, 4). References: 1. Chaiyarit S, Thongboonkerd V. Oxidized forms of uromodulin promote calcium oxalate crystallization and growth, but not aggregation. Int J Biol Macromol. 2022;214:542-53. 2. Chaiyarit S, Thongboonkerd V. Oxidative Modifications Switch Modulatory Activities of Urinary Proteins From Inhibiting to Promoting Calcium Oxalate Crystallization, Growth, and Aggregation. Mol Cell Proteomics. 2021;20:100151. 3. Yoodee S, Noonin C, Sueksakit K, Kanlaya R, Chaiyarit S, Peerapen P, et al. Effects of secretome derived from macrophages exposed to calcium oxalate crystals on renal fibroblast activation. Commun Biol. 2021;4(1):959. 4. Wuttimongkolchai N, Kanlaya R, Nanthawuttiphan S, Subkod C, Thongboonkerd V. Chlorogenic acid enhances endothelial barrier function and promotes endothelial tube formation: A proteomics approach and functional validation. Biomed Pharmacother. 2022;153:113471.

### Data Protocol
MS/MS data processing and label-free quantitative analysis  The raw MS/MS files were processed using MaxQuant (version 2.1.4.0) equipped with the Andromeda search engine. Proteins were identified from the UniProtKB/Swiss-Prot mammalian database using the following parameters: carbamidomethylation at cysteine (C) as fixed modification; oxidation at methionine (M) as variable modification; trypsin as the digesting enzyme; only one missed cleavage was allowed; precursor mass tolerance was 4.5 ppm; fragment mass tolerance was 0.5 Da; and charge state ions = +2, +3. The false discovery rate (FDR) cut-off was 1% at both peptide-spectrum match (PSM) and protein levels.   Label-free quantification (LFQ) of proteins was performed by using the MaxQuant LFQ (MaxLFQ) algorithm with match-between-runs. The other MaxQuant settings were allowed as default values as previously reported (5). The proteins identified as contaminants and reverse hits (decoy) and those identified only by site modifications were excluded. The LFQ intensity, generated according to the MaxLFQ algorithm, was used for statistical comparison by unpaired Studentâ€™s t-test. The proteins with â‰¥ 1.5-fold-change and p-value < 0.05 were considered as significantly altered proteins. Reference: 5.  Tyanova S, Temu T, Cox J. The MaxQuant computational platform for mass spectrometry-based shotgun proteomics. Nat Protoc. 2016;11(12):2301-19.  Statistical analysis  All quantitative data are reported as mean Â± SEM obtained from three independent biological replicates. Comparative analyses between the two groups were performed using an unpaired Studentâ€™s t-test. P-values <0.05 were considered statistically significant.

### Publication Abstract
Caffeine is a well-known purine alkaloid commonly found in coffee. Several lines of previous and recent evidence have shown that habitual coffee drinking is associated with lower risks for chronic kidney disease (CKD) and nephrolithiasis. However, cellular and molecular mechanisms underlying its renoprotective effects remain largely unknown due to a lack of knowledge on cellular adaptive response to caffeine. This study investigated cellular adaptive response of renal tubular cells to caffeine at the protein level. Cellular proteome of MDCK cells treated with caffeine at a physiologic concentration (100&#xa0;&#x3bc;M) for 24&#xa0;h was analyzed comparing with that of untreated cells by label-free quantitative proteomics. From a total of 936 proteins identified, comparative analysis revealed significant changes in levels of 148 proteins induced by caffeine. These significantly altered proteins were involved mainly in proteasome, ribosome, tricarboxylic acid (TCA) (or Krebs) cycle, DNA replication, spliceosome, biosynthesis of amino acid, carbon metabolism, nucleocytoplasmic transport, cell cycle, cytoplasmic translation, translation initiation, and mRNA metabolic process. Functional validation by various assays confirmed that caffeine decreased cell population at G2/M, increased cell population at G0/G1, increased level of ubiquitinated proteins, increased intracellular ATP and enhanced mitochondrial membrane potential in MDCK cells. These data may help unravelling molecular mechanisms underlying the biological effects of caffeine on renal tubular cells at cellular and protein levels.

### Keywords
Ckd; coffee; energy; kidney; metabolism; nephrolithiasis; proteomics; renoprotection

### Affiliations
Mahidol University
Medical Proteomics Unit, Research Department, Faculty of Medicine Siriraj Hospital, Mahidol University, Bangkok, Thailand

### Submitter
Visith Thongboonkerd

### Lab Head
Dr Prof. Visith Thongboonkerd
Medical Proteomics Unit, Research Department, Faculty of Medicine Siriraj Hospital, Mahidol University, Bangkok, Thailand


