### Accession
PXD000064

### Title
The conserved set of host proteins incorporated into HIV-1 virions suggests a common egress pathway in multiple cell types

### Description
HIV-1 incorporates a large array of host proteins into virions. Determining the host protein composition in HIV virions has technical difficulties, including co-purification of microvesicles. We developed an alternative purification technique using cholesterol that differentially modulates the density of virions and microvesicles (density modification, DM) allowing for high-yield virion purification that is essential for tandem mass spectrometric and quantitative proteomic (iTRAQ) analysis. DM purified virions were analyzed using iTRAQ and validated against Optiprep (60% iodixanol) purified virions. We were able to characterize host protein incorporation in DM-purified HIV particles derived from CD4+ T cell lines; we compared this dataset to a reprocessed dataset of monocyte-derived macrophages (MDM) derived HIV-1 searched using PrpArML which uses multiple search algorithms (OMSSA, X!Tandem with native, k-score and s-score scoring, MASCOT, MyriMatch, and InSpecT ). The database used for search was the UniProt-SwissProt database (version 2010.11.02; 522,019 sequences). Peptides were combined on PepARML using a random forest approach (Weka) and the results were then parsed into MASPECTRAS 2 with minimum two peptide for a protein and a spectrum false discovery rate of 5%. Peptides assigned Â to keratin were excluded, and since the analysis was focused on host proteins, viral peptide assignments were excluded. Protein redundancy was removed by MS based evidence clustering. Link to the paper related to the reprocessed MDM  dataset: PMID: 16940516, the dataset itself is not publicly available.

### Sample Protocol
Not available

### Data Protocol
Not available

### Publication Abstract
HIV-1 incorporates a large array of host proteins into virions. Determining the host protein composition in HIV virions has technical difficulties, including copurification of microvesicles. We developed an alternative purification technique using cholesterol that differentially modulates the density of virions and microvesicles (density modification, DM) allowing for high-yield virion purification that is essential for tandem mass spectrometric and quantitative proteomic (iTRAQ) analysis. DM purified virions were analyzed using iTRAQ and validated against Optiprep (60% iodixanol) purified virions. We were able to characterize host protein incorporation in DM-purified HIV particles derived from CD4+ T-cell lines; we compared this data set to a reprocessed data set of monocyte-derived macrophages (MDM) derived HIV-1 using the same bioinformatics pipeline. Seventy-nine clustered proteins were shared between the MDM derived and T-cell derived data set. These clusters included an extensive collection of actin isoforms, HLA proteins, chaperones, and a handful of other proteins, many of which have previously been documented to interact with viral proteins. Other proteins of note were ERM proteins, the dynamin domain containing protein EH4, a phosphodiesterase, and cyclophilin A. As these proteins are incorporated in virions produced in both cell types, we hypothesize that these proteins may have direct interactions with viral proteins or may be important in the viral life cycle. Additionally, identified common set proteins are predicted to interact with &gt;1000 related human proteins. Many of these secondary interacting proteins are reported to be incorporated into virions, including ERM proteins and adhesion molecules. Thus, only a few direct interactions between host and viral proteins may dictate the host protein composition in virions. Ultimately, interaction and expression differences in host proteins between cell types may drive virion phenotypic diversity, despite conserved viral protein-host protein interactions between cell types.

### Keywords
Hiv-1, Virions, Mdm, Lc-ms/ms, Dm

### Affiliations
Johns Hopkins School of Medicine

### Submitter
Ceereena Ubaida Mohien

### Lab Head


