### Accession
PXD028335

### Title
Oncogene-Mediated Metabolic Gene Signature Predicts Breast Cancer Outcome

### Description
We performed shotgun protoemic analysis of tumor samples from HCC1806 shCtrl and shTrop2 xenograft mice.

### Sample Protocol
Frozen tissues were homogenized and sonicted. Lysate samples were then centrifuged. Pproteins were quantified by BCA. Disulfide bonds were reduced and cysteine residues were modified by iodoacetic acid.  Proteins were acetone precipiated, pelleted, and digested with trypsin.  Peptides were dried and reconstituted.  Peptides were separated by reversed phase chromatography. Three technical replicates injections for each sample were performed. The eluding peptides were ionized and the top 10 most abundant ions detected by the mass spectrometer were selected for fragmentation.

### Data Protocol
Data was searched using Byonic 2.11.0 (Protein Metrics) twice against the corresponding taxonomy reference Swiss-prot database. First, containing the human reference proteome (2017; 20,484 entries), and again using the mouse reference proteome (2017; 17,191 entries). Database searches were performed including trypsin digestion with a maximum of two missed cleavages, and mass tolerance of precursor and fragment ions were set to 0.5 Da and 10 ppm, respectively. Fixed cysteine carbamidomethylation, variable methionine oxidation, and asparagine deamination, were also specified. Peptide identifications were filtered for a false discovery rate of 1%. Finally, peptides that overlapped in both human and mouse searches were removed to perform a conservative analysis of non-homologous only human identified proteins using an in-house R script, for each of the three technical replicates per experimental condition (HCC1806 shCTL, and HCC1806 shTrop2#2). Quantitative values were extracted from MS1 spectra from all resulting peptides using an in-house R script based on MSnbase package 54 after chromatogram alignment. Using the AUC extracted, a pairwise relative quantification of each sample against the average of the corresponding controls was performed, and analyzed using Generic Integration Algorithm, applying the principles of the WSPP model 55 using SanXoT package 56. Final statistical analysis was performed using Studentâ€™s t-test, considering only proteins having more than 5 peptide counts, having a P value < 0.01, and a fold-change greater than 2 as significant.

### Publication Abstract
Breast cancer remains the second most lethal cancer among women in the United States and triple-negative breast cancer is the most aggressive subtype with limited treatment options. Trop2, a cell membrane glycoprotein, is overexpressed in almost all epithelial cancers. In this study, we demonstrate that Trop2 is overexpressed in triple-negative breast cancer (TNBC), and downregulation of Trop2 delays TNBC cell and tumor growth supporting the oncogenic role of Trop2 in breast cancer. Through proteomic profiling, we discovered a metabolic signature comprised of TALDO1, GPI, LDHA, SHMT2, and ADK proteins that were downregulated in Trop2-depleted breast cancer tumors. The identified oncogene-mediated metabolic gene signature is significantly upregulated in TNBC patients across multiple RNA-expression clinical datasets. Our study further reveals that the metabolic gene signature reliably predicts poor survival of breast cancer patients with early stages of the disease. Taken together, our study identified a new five-gene metabolic signature as an accurate predictor of breast cancer outcome.

### Keywords
Trop2, Proteomics, Breast cancer

### Affiliations
Canary Center at Stanford for Cancer Early Detection Department of Radiology Stanford University
Stanford University

### Submitter
Sharon Pitteri

### Lab Head
Dr Sharon Pitteri
Canary Center at Stanford for Cancer Early Detection Department of Radiology Stanford University


