### Accession
PXD039956

### Title
A combinatorial approach to uncover a novel Integrator subunit

### Description
RNA polymerase II (RNAPII) controls the expression of all protein coding genes and most noncoding loci in higher eukaryotes. The ultimate rate of transcription is regulated at the level of multiple checkpoints during the transcription cycle (initiation, pausing, termination). Calibrating the activity of RNAPII is fundamental to all cellular processes and requires an assortment of polymerase-associated factors that are recruited at sites of active transcription. The Integrator complex is one of the most elusive transcriptional regulators in metazoans, deemed to be recruited after initiation to help establish and modulate paused RNAPII. Recent structural, biochemical, and functional evidence suggest that Integrator is composed of 14 subunits that assemble and operate in a modular fashion. We employed proteomics and machine-learning structure prediction (AlphaFold2) approaches to identify a novel Integrator subunit, INTS15. We report that INTS15 assembles primarily with the INTS13/INTS14/INTS10 module and interfaces with the core of the Int-PP2A complex. Functional genomics analysis further reveals a role for INTS15 in modulating RNAPII pausing at a subset of genes in human cell lines. Our study shows that omics approaches combined with AlphaFold2-based predictions provide novel insights into the molecular architecture of large and flexible multiprotein complexes.

### Sample Protocol
HEK-293T cells were seeded into 6-well plates 24h before transfection at a density of 500,000 cells per well in 1.5 mL D-MEM medium with 10% FBS. 1Î¼g of DNA in 50Î¼l of opti-MEM was mixed with 3 Î¼g of PEI 40K max in 50 Î¼l of opti-MEM and incubated at room temperature for 10 min. Transfection solutions were added drop-by-drop to each well. The cells were harvested by centrifugation 48h after transfection, dissolved in 400 Î¼L of lysis buffer (150 mM KCl, 20 mM K-HEPES pH 7.8 and 0.1% Triton X-100) and sonicated for 10 s at 30% amplitude. Lysates were cleared by centrifugation in a table top centrifuge at 20 000 g at 4Â°C for 30 min. The supernatant was incubated for 2h with 5% (v/v) of affinity resin (HA- or FLAG-agarose) to capture the bait protein. Affinity resin was washed 3 times with 10 resin volumes of Buffer3 (150 mM KCl and 20 mM K-HEPES pH 7.8) and subsequently resuspended in SDS sample buffer and heated up to 95Â°C for 5 min to release bound proteins. Input and elution fractions were analyzed by western blotting. For the differential quantitative proteomics experiments of INTS10 variants, cells were grown in 10 cm dishes in triplicates and proteins were purified as above. subjected to an in-solution tryptic digest using a modified version of the Single-Pot Solid-Phase-enhanced Sample Preparation (SP3) protocol (Hughes et al., 2014; Moggridge et al., 2018). Samples were added to Sera-Mag Beads (Thermo Scientific, #4515-2105-050250, 6515-2105-050250) in 20 Î¼l 15% formic acid and 60 Î¼l of ethanol. Binding of proteins was achieved by shaking for 15 min at room temperature. SDS was removed by 4 subsequent washes with 200 Î¼l of 70% ethanol. Proteins were digested with 0.4 Î¼g of sequencing grade modified trypsin (Promega, #V5111) in 40 Î¼l Na-HEPES, pH 8.4 in the presence of 1.25 mM TCEP and 5 mM chloroacetamide (Sigma-Aldrich, #C0267) overnight at room temperature. Beads were separated, washed with 10 Î¼l of an aqueous solution of 2% DMSO and the combined eluates were dried down. Peptides were reconstituted in 10 Î¼l of H2O and reacted with 80 Î¼g of TMT10plex (Werner et al., 2014) (Thermo Scientific, #90111) label reagent dissolved in 4 Î¼l of acetonitrile for 1 h at room temperature. Excess TMT reagent was quenched by the addition of 4 Î¼l of an aqueous solution of 5% hydroxylamine (Sigma, 438227). Peptides were mixed to achieve a 1:1 ratio across all TMT-channels. Mixed peptides were subjected to a reverse phase clean-up step (OASIS HLB 96-well Î¼Elution Plate, Waters #186001828BA) and analyzed by LC-MS/MS on a Q Exactive Plus (Thermo Scentific) as previously described (Becher et al., 2018).  Briefly, peptides were separated using an UltiMate 3000 RSLC (Thermo Scientific) equipped with a trapping cartridge (Precolumn; C18 PepMap 100, 5 lm, 300 lm i.d. Ã— 5 mm, 100Ã…) and an analytical column (Waters nanoEase HSS C18 T3, 75 lm Ã— 25 cm, 1.8 lm, 100Ã…). Solvent A: aqueous 0.1% formic acid; Solvent B: 0.1% formic acid in acetonitrile (all solvents were of LC-MS grade). Peptides were loaded on the trapping cartridge using solvent A for 3 min with a flow of 30 Î¼l/min. Peptides were separated on the analytical column with a constant flow of 0.3 Î¼l/min applying a 2 h gradient of 2 â€“ 28% of solvent B in A, followed by an increase to 40% B. Peptides were directly analyzed in positive ion mode applying with a spray voltage of 2.3 kV and a capillary temperature of 320Â°C using a Nanospray-Flex ion source and a Pico-Tip Emitter 360 lm OD Ã— 20 lm ID; 10 lm tip (New Objective). MS spectra with a mass range of 375â€“1.200 m/z were acquired in profile mode using a resolution of 70.000 [maximum fill time of 250 ms or a maximum of 3e6 ions (automatic gain control, AGC)]. Fragmentation was triggered for the top 10 peaks with charge 2â€“4 on the MS scan (data-dependent acquisition) with a 30 s dynamic exclusion window (normalized collision energy was 32). Precursors were isolated with a 0.7 m/z window and MS/MS spectra were acquired in profile mode with a resolution of 35,000 (maximum fill time of 120 ms or an AGC target of 2e5 ions).

### Data Protocol
The raw output files of IsobarQuant were processed using the R programming language (ISBN 3-900051-07-0). Only proteins that were quantified with at least two unique peptides were considered for the analysis. Raw signal-sums (signal_sum columns) were normalized using variance stabilization normalization (Huber et al., 2002). Ratios were computed using these normalized TMT reporter ion signals. The top3 value is the average log10 MS1 intensity of the three most abundant peptides for each protein and serves as an estimator for the average abundance of a protein in the multiplexed mass spec run.  The mass spectrometry experiment and data analysis was conducted by the EMBL Proteomic Core Facility in Heidelberg.

### Publication Abstract
RNA polymerase II (RNAPII) controls expression of all protein-coding genes and most noncoding loci in higher eukaryotes. Calibrating RNAPII activity requires an assortment of polymerase-associated factors that are recruited at sites of active transcription. The Integrator complex is one of the most elusive transcriptional regulators in metazoans, deemed to be recruited after initiation to help establish and modulate paused RNAPII. Integrator is known to be composed of 14 subunits that assemble and operate in a modular fashion. We employed proteomics and machine-learning structure prediction (AlphaFold2) to identify an additional Integrator subunit, INTS15. We report that INTS15 assembles primarily with the INTS13/14/10 module and interfaces with the Int-PP2A module. Functional genomics analysis further reveals a role for INTS15 in modulating RNAPII pausing at a subset of genes. Our study shows that omics approaches combined with AlphaFold2-based predictions provide additional insights into the molecular architecture of large and dynamic multiprotein complexes.

### Keywords
Integrator complex, Transcription, Alphafold2, Proteomics, Functional genomics

### Affiliations
EMBL
European Molecular Biology Laboratory 71 Av. des Martyrs, 38000 Grenoble France

### Submitter
Frank Stein

### Lab Head
Dr Wojciech Galej
European Molecular Biology Laboratory 71 Av. des Martyrs, 38000 Grenoble France


