### Accession
PXD042875

### Title
Bottom-up proteomics of EZH2 mutants in isogenic cells

### Description
The Enhancer of Zeste 2 Polycomb Repressive Complex 2 Subunit (EZH2) is an essential epigenetic modifier able to methylate lysine 27 on histone H3 (H3K27) to induce chromatin compaction, protein complex recruitment and ultimately transcriptional repression. Hematologic malignancies, including Diffuse Large B cell lymphoma (DLBCL) and Acute myeloid leukemia (AML) have shown a high EZH2-mutation frequency (>20%) associated with poor clinical outcomes. Particularly, two distinct oncogenic mutations, so-called gain-of-function (Y641F and A677G) and loss-of-function (H689A and F667I) are found in the catalytic domain of EZH2. In this study, a comprehensive multi-omics approach was employed to characterize downstream effects of H3K27me3 deposition driven by EZH2 mutations. Human embryonic kidney cells (HEK293T) were transfected to generate three stable EZH2 mutants: EZH2(Y641F), EZH2(A677G), and EZH2(H689A/F667I), which were validated via immunoblotting and DIA-MS-based histone profiling assay. Subsequently, constructs were analyzed under a comprehensive multi-omics approach including label-free whole-cell proteomics, acquired with a Bruker timsTOF Pro HPLC-MS/MS with Ion Mobility. Important protein interactors at nuclear level were dysregulated, such as SMYD3 (SET and MYND domain containing 3), NSD2 (nuclear receptor binding SET domain protein 2) and CHD7 (chromodomain helicase DNA binding protein 7), suggesting a cooperative network of chromatin remodelers for gene expression reprogramming.

### Sample Protocol
Approximately 1x 107 cells were washed three times with ice-cold PBS (supplemented with 1X protease inhibitor and 1X phosphatase inhibitor) and harvested by scrapping. Cell pellets were resuspended in 2 mL of RIPA Lysis and Extraction Buffer (Thermo Fisher Scientific Cat No 89901) supplemented with inhibitors (1X protease inhibitor and 1X phosphatase inhibitor)  and sonicated for 10 min at 4 ºC (30 s on/30 off interval, Bioruptor by Diagenode). Cellular debris was pelleted by centrifugation at 15.000 rpm for 10 min at 4 ºC. The resultant supernatant was transferred and assayed for protein concentration using the Pierce™ BCA Protein Assay Kit (Thermo Fisher Scientific Cat No 23227).   Subsequently, cell lysate was reduced with dithiothreitol (DTT) for 45 minutes at 50 °C and alkylated with iodoacetamide (IAA) for 30 minutes in the dark at room temperature. Phosphoric acid was then added for acidification, followed by S-Trap binding buffer addition (90% MeOH, 100 mM final TEAB, pH 7.1) and loading into a S-trap spin column. After column centrifugation, mass spectrometry grade trypsin (1:20, enzyme:substrate) was added and incubated overnight at 37 °C. Finally, peptides were eluted using TEAB and aqueous formic acid. The flow-through was dried in a SpeedVac (Eppendorf) and resuspended in 50 mM glacial acetic acid. Peptide concentration was assessed by Nanodrop at 280nm.  2 µg of peptides were injected in a C18 column (25cm × 75μm, 1.6 μm, IonOpticks, Australia). Mobile phases consisted of water with 0.1 % formic acid (phase A), and acetonitrile with 0.1 % formic acid (phase B) at a flow rate of 0.4 mL/min and 50°C. Peptides were eluted with the following set up: a linear gradient from 2 to 25% mobile phase B during 90 min, 25 to 37% in 10 min, and 37 to 80% over 10 min; then kept for 10 min. The column was equilibrated in 2% of mobile phase B for 15 min before next sample injection. MS/MS data was acquired on a Bruker timsTOF Pro HPLC-MS/MS with Ion Mobility equipped with a CaptiveSpray source (Bruker Scientific, Billerica, MA) operated in positive ion mode. Data was acquired on the Parallel Accumulation Serial Fragmentation (PASEF) acquisition mode, with a mass range of 100 - 1700 m/z, a capillary voltage of 1.6 kV, dry gas 3 L/min, and dry temp of 180 °C. PASEF settings included 10 MS/MS scans at 1.18-s total cycle time, scheduling target intensity of 20,000, active exclusion release after 0.4 min, and CID collision energy of 42 eV.

### Data Protocol
Raw data was converted to mzML format using ProteoWizard (v3.0.2) and subsequently, OpenMS (2.6.0) was employed for peptide identification with MSGF+ search engine against a reviewed UniProt human proteome FASTA file with the following parameters: full trypsin digest; enzyme, trypsin; maximum missed cleavage sites, 2; precursor mass tolerance, 10 ppm; fragment mass tolerance, 0.05 Da; carbamidomethyl of Cys, fixed modification; oxidation of Met, variable modification. PSM rescoring was completed with Percolator and protein inference was performed with Epiphany across all samples, setting peptide and protein FDR to 0.05.

### Publication Abstract
Somatic heterozygous mutations in the active site of the enhancer of zeste homolog 2 (EZH2) are prevalent in diffuse large B-cell lymphoma (DLBCL) and acute myeloid leukemia (AML). The methyltransferase activity of EZH2 towards lysine 27 on histone H3 (H3K27) and non-histone proteins is dysregulated by the presence of gain-of-function (GOF) and loss-of-function (LOF) mutations altering chromatin compaction, protein complex recruitment, and transcriptional regulation. In this study, a comprehensive multi-omics approach was carried out to characterize the effects of differential H3K27me3 deposition driven by EZH2 mutations. Three stable isogenic mutants (EZH2<sup>Y641F</sup>, EZH2<sup>A677G</sup>, and EZH2H<sup>689A/F667I</sup>) were examined using EpiProfile, H3K27me3 CUT&amp;Tag, ATAC-Seq, transcriptomics, label-free proteomics, and untargeted metabolomics. A discrete set of genes and downstream targets were identified for the EZH2 GOF and LOF mutants that impacted pathways involved in cellular proliferation, differentiation, and migration. Disruption of protein networks and metabolic signatures able to sustain aberrant cell behavior was observed in response to EZH2 mutations. This systems biology-based analysis sheds light on EZH2-mediated cell transformative processes, from the epigenetic to the phenotypic level. These studies provide novel insights into aberrant EZH2 function along with targets that can be explored for improved diagnostics/treatment in hematologic malignancies with mutated EZH2.

### Keywords
Mutations, Leukemia, Epigenetics, Lymphoma, Ezh2

### Affiliations
The Ohio State University
Cancer Biology and Genetics, Wexner Medical Center, The Ohio State University, Columbus, OH, USA

### Submitter
Julian Aldana

### Lab Head
Dr Michael A. Freitas
Cancer Biology and Genetics, Wexner Medical Center, The Ohio State University, Columbus, OH, USA


