### Accession
PXD003896

### Title
HEK293 searched with UniProt Human and ComPIL

### Description
HEK293 proteome searched with UniProt Human and ComPIL

### Sample Protocol
Human proteome isolation HEK293 cells were grown under 5% CO2 at 37 °C in EMEM minimum essential medium (Quality Biological Catalog # 25030-081) supplemented with 2 mM L-glutamine, 1X MEM non-essential amino acids, 1X penicillin-streptomycin, 1 mM sodium pyruvate, and 10% (v/v) Heat Inactivated Fetal Bovine Serum, all from Gibco/Life Technologies. After cultures reached approximately 80% confluence, cells were gently scraped into DPBS and pelleted by centrifugation. Each cell pellet was rinsed twice with cold DPBS and frozen at -80 °C after removing all buffer. Cells pellets were later thawed on ice and resuspended in a solution of DPBS and Roche cOmplete protease inhibitor cocktail (Roche Catalog # 04693159001). The resuspended cell pellet was subjected to sonication in a Qsonica Q700 sonicator with Cup Horn attachment (water bath held at 4 °C) using a program of 1 sec on / 1 sec off at 50% amplitude for 3 x 1-min cycles.  Proteome sample preparation for mass spectrometry Protein concentrations of all cell lysates were measured against a BSA standard curve using the Pierce BCA Protein Assay Kit. 25µg of each protein sample was diluted to a final volume of 100 µL in DPBS.  30 µL of 6.1N trichloroacetic acid (Sigma-Aldrich Catalog # T0699) was added to each 100 µL sample, samples were mixed thoroughly, and placed at 4 °C overnight. Precipitate was collected the next morning by centrifugation at 21,000g for 20 minutes at 4 °C. Protein pellets were washed twice with cold acetone (JT Baker Catalog # 9002-02) and allowed to air dry at room temperature for several hours. Dry protein pellets were solubilized in 60 µL of a solution of 8 M Urea in 100 mM Tris-HCl (pH 8.5). Cystine disulfide bonds were reduced by addition of tris(2-carboxyethyl)phosphine (TCEP) to each sample to a final concentration of 5 mM and incubation at room temperature for 20 min. Cysteine residues were subsequently alkylated by addition of 2-chloroacetamide to a final concentration of 50 mM for 15 min at room temperature. Samples were diluted to 240 µL volume with 100 mM Tris-HCl (pH 8.5), supplemented with 1 mM CaCl2 and 1 µg trypsin (Promega Catalog # V5111), and samples were incubated at 37 °C overnight. Formic acid (Fisher Catalog # A117-50) was added to each tube (5%, v/v) the following morning, and samples were centrifuged for 15 minutes at 21,000g. The supernatant was collected and frozen at -20 °C.  LC column preparation and sample loading MudPIT columns were prepared in-house as previously described7. Briefly, strong cation exchange (SCX) and reverse-phase C18 dry resins were sequentially pressure-loaded into a 250-µm (internal diameter) fused silica capillary (Agilent Catalog # 160-2250-10) such that 2 centimeters of each resin were adjacent to each other. 12 centimeters of C18 resin were pressure-loaded into a 100-µm (internal diameter) fused silica capillary that had been pulled to a fine point using a pipette puller instrument (Sutter Instrument Company, Model P-2000). Acidified peptide samples were pressure-loaded on to the biphasic column, and the biphasic column was connected to the reverse-phase tip using a specialized union connector (IDEX Catalog # P-720-01).

### Data Protocol
Mass spectrometry data collection Tandem mass spectrometry data for each sample was collected on a Thermo Scientific Orbitrap Velos Pro coupled to an Agilent 1200 series HPLC quaternary pump. Peptides were eluted off the biphasic MudPIT column and separated over 10 chromatography steps, each of which consisted of a 3-minute “salt bump” of between 10% and 100% solvent C, followed by a 100-minute linear gradient of 0% to 35% solvent B (solvent A: 95% H2O, 5% acetonitrile, 0.1% formic acid; solvent B: 20% H2O, 80% acetonitrile, 0.1% formic acid; solvent C: 500 mM ammonium acetate, 95% H2O, 5% acetonitrile, 0.1% formic acid). An eleventh and final chromatography step consisted of a 3-minute 90% solvent C/10% solvent B “salt bump” followed by a 100-minute linear gradient of 0% to 35% solvent B. Precursor ions were recorded by scanning in the range of m/z 400.00-2000.00 with the FTMS analyzer and a resolution of 60,000. The top 10 peaks were selected for fragmentation using HCD with normalized collision energy set to 35.0. Fragmentation data were also collected in the FTMS analyzer, but with a resolution of 7500. Dynamic exclusion was enabled with exclusion duration set to 20.0 seconds.  Mass spectrometry proteomic searches Precursor ion and fragmentation data were extracted from Xcalibur raw files using RawXtract 1.9.9.2 or RawConverter 1.0.0.x with monoisotopic peak prediction enabled (for the microbiome searches) (http://fields.scripps.edu/downloads.php). The resulting MS2 files were searched using Blazmass integrating ComPIL/MongoDB connectivity and traditional cross-correlation scoring. MS/MS scans were searched using a precursor mass tolerance of 10 ppm for microbiome searches or 30 ppm and a fragment ion mass tolerance of 50 ppm. A static modification was specified on cysteine (+57.02146). MS2 files were each split into 25-50 chunks in order to parallelize database searches using the Linux cluster located in the TSRI High Performance Computing (HPC) core. Chunks were recombined after searches were complete. Blazmass source code and installation instructions are available at https://github.com/sandipchatterjee/blazmass_compil

### Publication Abstract
None

### Keywords
Hek293, Metaproteomics, Compil

### Affiliations
TSRI
Assistant Professor Department of Molecular and Experimental Medicine  TSRI - California Campus USA

### Submitter
Greg Stupp

### Lab Head
Dr Dennis Wolan
Assistant Professor Department of Molecular and Experimental Medicine  TSRI - California Campus USA


