### Accession
PXD031552

### Title
Global proteome of Atg5flox:CamKIIα-Cre:tdTomato FACS sorted WT and KO neurons

### Description
The experiment was performed to identify autophagy targets in wildtype and autophagy-deficient forebrain excitatory neurons. Therefore, neurons were isolated from the cortex, hippocampus and striatum of 2-3 weeks old Atg5flox/flox:CamKIIα-Cretg/wt:tdTomato+ (KO) and Atg5wt/wt:CamKIIα-Cretg/wt:tdTomato+ (WT) mice. Neurons in suspension were FACS sorted and excitatory forebrain neurons expressing tdTomato were forwarded to global proteome analysis assessed by LC-MS/MS.

### Sample Protocol
Sample preparation. Neurons for FACS sorting were isolated from 2-3 weeks old Atg5flox/flox:CamkIIα-Cretg/wt:tdTomato+ and Atg5wt/wt:CamKIIα-Cretg/wt:tdTomato+ reporter mice to receive a pure fraction of autophagy-deficient and wildtype in-vivo excitatory forebrain neurons. Therefore, the mice were transcardially perfused with ACSF to remove blood cells. Isolated brains were dissected in HibernateA (HA, Gibco) and cortex, hippocampus and striatum isolated and transferred into a fresh dish containing HABG (Hibernate-A, 2% B27, 1% GlutaMax). The tissue was cut into pieces and digested in activated Papain (Worthington) for 40 min at 37°C. Afterwards, tissue pieces were transferred back into fresh HABG. They were homogenized with a fire-polished Pasteur pipette (triturate approximately ten times in 1 min). The cell suspension was applied on an OptiPrep®Density Gradient Medium (Sigma) centrifuged at 800 xg for 15 min at 22°C to force cell type separation. Neuron enriched fractions were maintained for further processing. Gradient material was diluted with 10 mL HABG and cells were pelleted down at 3000 xg for 3 min at 22°C to clean cell suspension from debris contaminations. This step was repeated once before the cells were resuspended in NBA supplemented with 2% B27.  To obtain purified lysates from autophagy-deficient neurons, cells in suspension were stained with DAPI and DRAQ5. Cell sorting was performed using BD FACSAria Fusion IIu and BD FACSAria IIIu with FACSDiva 8.0.1 software. Cells/Neurons were sorted at 4°C using a 100 µm nozzle and sheath pressure was set at 20 psi. 0.9% NaCl was used as sheath fluid. The tdTomato highly positive/ DAPI negative/ DRAQ5 positive cell population was selected. Cells were collected in chilled 1.5 mL Eppendorf tubes containing DPBS. After sorting, the cells were centrifuged at 3000 xg for 3 min and lysed in buffers depending on subsequent analysis. Sorted cells were pelleted and resuspended in SP3 lysis buffer (5% SDS in PBS). Chromatin was degraded using a Biorupter (10 min, cycle 30/30). The samples were reduced (5 mM DTT) and alkylated (40 mM CAA) and protein content was measured with the Pierce BCS protein assay. 20 µg were used for protein digestion according to the single-pot solid-phase-enhanced sample preparation.  LCMS data acquisition. The LCMS analysis approach was adjusted to handle low input volume. Samples were analysed on a Q Exactive Exploris 480 (Thermo Scientific) mass spectrometer that was coupled to an Evosep ONE (Evosep) in the recommended “whisper” setup. Samples were loaded onto EvoTips following the manual instructions (Evosep). Peptides were chromatographically separated by the predefined “whisper 20 SPD” setup on a 58 min gradient on a PepSep 15 cm column with a 75 µm inner diameter filled with 1.9 µm Dr. Maisch resin. MS1 scans were acquired from 380 m/z to 900 m/z at 45k resolution. Maximum injection time was set to 60 ms and the AGC target to 100%. MS2 scans ranged from 400 m/z to 880 m/z and were acquired at 45 k resolution with a maximum injection time of 84 msec and an AGC target of 1000%. DIA scans covering the precursor range from 400 - 880 m/z were acquired in 15 x 30 m/z staggered windows resulting in 30 nominal 15 m/z windows after demultiplexing. All scans were stored as a centroid.

### Data Protocol
Thermo raw files were demultiplexed and transformed to mzML files using the msconvert module in Proteowizard. MzML files were converted to dia file format in DIA-NN 1.7.15. A Mouse canonical Swissprot fasta file was converted to a Prosit upload file with the convert tool in encyclopedia 0.9.0 (3) using default settings: Trypsin, up to 1 missed cleavage, range 396 m/z – 1004 m/z, charge states 2+ and 3+, default charge state 3 and NCE 33. The csv file was uploaded to the Prosit webserver and converted to a spectrum library in generic text format. The resulting library (16998 protein isoforms, 21694 protein groups and 1404872 precursors was used in DIA-NN 1.7.15 (5) to search acquired data in double-pass mode. The applied settings were: Output will be filtered at 0.01 FDR, N-terminal methionine excision enabled, Maximum number of missed cleavages set to 1, Min peptide length set to 7, Max peptide length set to 30, Min precursor m/z set to 400, Max precursor m/z set to 1000, Cysteine carbamidomethylation enabled as a fixed modification.

### Publication Abstract
None

### Keywords
In-vivo proteome, Neurons, Atg5 ko, Autophagy

### Affiliations
CECAD Research Center
CECAD Excellence Center, University of Cologne, Germany, D-50931 Center for Physiology and Pathophysiology, Faculty of Medicine and University Hospital Cologne, University of Cologne, Germany, D-50931

### Submitter
Prerana Wagle

### Lab Head
Dr N.L. Kononenko
CECAD Excellence Center, University of Cologne, Germany, D-50931 Center for Physiology and Pathophysiology, Faculty of Medicine and University Hospital Cologne, University of Cologne, Germany, D-50931


