### Accession
PXD026426

### Title
Enhanced Access to the Human Phosphoproteome with Genetically Encoded Phosphothreonine

### Description
Protein phosphorylation is a ubiquitous post-translational modification that governs signaling cascades and protein-protein interactions. Orthogonal translation systems repurpose evolutionarily divergent aminoacyl-tRNA synthetase and tRNA pairs for the co-translational insertion of a modified amino acid. Subsequent advancements over the last decade have enabled the insertion of phospho-amino acids, bypassing a priori knowledge requirements of upstream kinases for the study of phosphoproteins. Here we optimized a pThrOTS and corresponding E. coli strain for pThr protein production.  We then produced a peptidome library containing ~57,000 known threonine/phosphothreonine phosphosites using oligonucleotide library synthesis. We were able to identify approximately ~20% of the peptides encoded by the pThr library, and ~44% of the peptides encoded by the Thr library with mass spectrometry. Robust, genetically encoded phosphothreonine revealed a new activation and inhibition mechanism for the kinase CHK2. Proteome-wide surveys of interactions between active CHK2 and our peptidome library identified novel substrates and motif elements. Finally, we developed a novel technique, Hi-P+, for directly linking kinase substrate discovery to phospho-binding domain recognition, unveiling multi-level interaction networks with phosphosite resolution. This new methodology enables kinase-specific, proteome-wide surveys of multiple phosphorylation-dependent protein-protein interactions. This data set is for the Thr (non-phosphorylated) peptide library fractions.

### Sample Protocol
1.5L cultures of LB 50 μg/mL Kanamycin, 100 μg/mL Ampicillin were inoculated to OD 0.1 with overnight culture containing C321 cells expressing pThrOTSZeus and the expression plasmid containing an MBP-6xHis fusion construct. Strain, expression plasmid, and OTS combinations used for the MBP-fusion proteins used in this study can be found in.  Cultures were set to shake at 230 RPM, 37°C. At OD 0.4, 1.5L flasks were induced with IPTG and arabinose, and set back at 230 RPM, 37°C for 4 hours. An additional round of expression was similarly inoculated but grown to an OD of 0.8, induced with IPTG and arabinose, and set at 230 RPM, 20°C for 20 hours. Cultures were pelleted at 5,000 RPM for 5 minutes, transferred to separate 50 mL falcon tubes and frozen at -80°C.
Thawed cell pellets were resuspended in 30 mL of lysis buffer (50 mM Tris/HCl pH 7.4, 300 mM NaCl, 1 mM DTT, 1 mg/mL lysozyme, 50 mM NaF, 1 mM NaVO4, Roche protease inhibitor tablet, 1 μL benzonase), and incubated at room temperature for 20 min, followed by sonication. The lysates were centrifuged at 45,000g, 45 min, 4°C, and the clarified lysate was transferred to a 15 mL falcon tube and centrifuged at 45,000g, 25 min, 4°C to remove all remaining insoluble material. The clarified lysate was then purified via a gravity column. Briefly, 6 mLs (3 mL bed volume) of NiNTA agarose beads (Qiagen #30210) was pre-equilibrated with lysis buffer before applying cell lysate. Resin was then washed with 100 mL of NiNTA wash buffer (50 mM Tris/HCl pH 7.4, 500 mM NaCl, 500 μM DTT, 50 mM NaF, 1 mM NaVO4, 5% glycerol, 10 mM imidazole) before eluting with 20 mL NiNTA elution buffer (50 mM Tris/HCl pH 7.4, 500 mM NaCl, 1 mM DTT, 50 mM NaF, 1 mM NaVO4, 250 mM imidazole). Eluted protein was then diluted with 20 mL dilution buffer (20 mM Tris/HCl pH 7.4). 6 mLs (3 mL bed volume) of amylose resin (New England Biolabs #E8021L) was applied to a new column and equilibrated with 50 mL amylose wash buffer (20 mM Tris/HCl pH 7.4, 1 mM EDTA, 150 mM NaCl 50 mM NaF, 1 mM NaVO4, 5% glycerol). Diluted protein elution was then applied to amylose column and washed with 100 mL amylose wash buffer before eluting with 15 mL amylose elution buffer (20 mM Tris/HCl pH 7.4, 1 mM EDTA, 150 mM NaCl, 10 mM maltose, 50 mM NaF, 1 mM NaVO4, 5% glycerol). Samples were concentrated down using Amicon ultra-15 10kDa cut-off columns (Millipore Sigma #UFC901008). pThr phosphosite libraries were then frozen at -80°C. CHK2 kinases were dialyzed in SnakeSkin™ (ThermoFisher #68035) overnight at 4°C in dialysis buffer #1 (50 mM Tris/HCl pH 7.4, 150 mM NaCl, 5 mM BME, 10% glycerol), then dialyzed for 4 hours at 4°C in dialysis buffer #2 (50 mM Tris/HCl pH 7.4, 150 mM NaCl, 5 mM BME, 50% glycerol) before being frozen at -80°C.
5 mg of purified pThr phosphosite library was digested using S-TrapTM midi columns (Protifi) per manufacturer's instructions. Phosphoproteins were then enriched using Thermo Scientific High-Select™ Fe-NTA Phosphopeptide Enrichment Kit (#A32992) followed by Thermo Scientific High-Select™ TiO2 Phosphopeptide Enrichment Kit (#A32993) following manufacturer's instructions. Samples were split into separation fractions before being dried in a vacuum centrifuge at room temperature. Pooled Fe-NTA and TiO2 flow-through was cleaned up and desalted using C18 MicroSpin (The Nest Group #SEM SS18V) columns as described earlier. One set of phospho-enriched samples was reconstituted in 2 μl 30 % ACN 0.1 % FA and vortexed for 30s. An additional 10 μL of 3:8 70 % formic acid: 0.1% TFA was added to each sample and vortexed for 30s before being spun down. 5 μl was injected for LC-MS/MS analysis. 
The remaining dried down phospho-enriched samples were resuspended in 50 μL 1x Protein MetalloPhosphatases (PMP) buffer (NEB) with 10 mM MnCl2. 2,000 units of lambda phosphatase (NEB #P0753S) and 5 units calf intestinal phosphatase (NEB #M0525S) were added to each reaction and incubated at 30°C, 600 rpm, for 1 hour in a thermo mixer. Samples were cleaned up using 200 μL StageTips similar to what was previously described [54]. Tips consisted of 2 1.06 mM diameter punches of Empore C18 (3M #2215) fitted into 200 μL pipette tips. Columns were conditioned with 40 μl methanol and 30 μl 0.1% TFA. The sample was then acidified with 3 μl 70% FA and vortexed. The sample was applied to the column and centrifuged at 2800 rpm for 5 minutes. Columns were washed with 50 μl 0.1% TFA and centrifuged for 3 minutes at 900g. Centrifugation was repeated with 20 μl 0.1% TFA. StageTips were then transferred to 2 ml protein low bind tubes (Eppendorf #0030108450) and eluted with 30 μl of 80% ACN 0.1% TFA, centrifuged at 900g for 1 minute. Elution was repeated with 20 μL of 80% ACN 0.1% TFA, centrifuged at 900g for 2 minutes. Samples were dried down in a vacuum centrifuge at room temperature. Samples were then reconstituted in in 2 μl 30 % ACN 0.1 % FA and 10 μL of 3:8 70 % formic acid: 0.1% TFA as described earlier.

### Data Protocol
LC-MS/MS was performed using an ACQUITY UPLC M-Class (Waters) and Thermo Q Exactive Plus mass spectrometer. The analytical column employed was a 65-cm-long, 75-μm-internal-diameter PicoFrit column (New Objective) packed in-house to a length of 50 cm with 1.9 μm ReproSil-Pur 120 Å C18-AQ (Dr. Maisch) using methanol as the packing solvent. Peptide separation was achieved using mixtures of 0.1% formic acid in water (solvent A) and 0.1% formic acid in acetonitrile (solvent B) with a 90-min gradient 0/1, 2/7, 60/24, 65/48, 70/80, 75/80, 80/1, 90/1; (min/%B, linear ramping between steps). The gradient was performed with a flow rate of 250 nL/min. A single blank injection (5 μl 2% B) was performed between samples to eliminate peptide carryover on the analytical column. 100 fmol of trypsin-digested BSA or 100 ng trypsin-digested wildtype K-12 MG1655 E. coli proteins were run periodically between samples as quality control standards. The mass spectrometer was operated with the following parameters: (MS1) 70,000 resolution, 3e6 AGC target, 300–1,700 m/z scan range; (data-dependent-MS2) 17,500 resolution, 1e6 AGC target, top 10 mode, 1.6 m/z isolation window, 27 normalized collision energy, 90s dynamic exclusion, unassigned and +1 charge exclusion. Data were searched using Maxquant version 1.6.10.43 with Deamidation (NQ), Oxidation (M), and Phospho (STY) as variable modifications and Carbamidomethyl (C) or Dithiomethane (C) as a fixed modification with up to 3 missed cleavages, 6 AA minimum length, and 1% FDR against targeted libraries. Phosphosite libraries were run against a modified FASTA file of the phosphosite library containing a partially digested MBP tag and 6xHis tag (supplemental data files 3&4) in MaxQuant with a 1% FDR.

### Publication Abstract
None

### Keywords
14-3-3, Phosphoprotein interactions, Kinase, Proteomics, Checkpoint kinase 2, Genetic code expansion phosphothreonine, Synthetic biology, Phosphoproteome, Orthogonal translation system

### Affiliations
Yale University
Department of Cellular & Molecular Physiology, Yale School of Medicine, New Haven, CT 06520, USA.  Systems Biology Institute, Yale University, New Haven, CT 06516, USA

### Submitter
Jack Moen

### Lab Head
Dr Jesse Rinehart
Department of Cellular & Molecular Physiology, Yale School of Medicine, New Haven, CT 06520, USA.  Systems Biology Institute, Yale University, New Haven, CT 06516, USA


