### Accession
PXD045780

### Title
Affinity purification of ADP-ribosylated proteins from A. thaliana plants expressing the Pseudomonas syringae type III effectors AvrRpm1 or HopF2

### Description
ADP-ribosylated proteins were enriched using the engineered Af1521 Macro domain (Nowak et al., https://doi.org/10.1038/s41467-020-18981-w) from transgenic A. thaliana plants expressing the Pseudomonas syringae type III effectors AvrRpm1 or HopF2. Enriched proteins were analyzed by LC-MS/MS. Non-transgenic Col-0 plants served as control.

### Sample Protocol
Protein extracts were prepared by grinding plant leaf material in liquid N2 to a fine powder followed by resuspension in extraction buffer [50 mM Tris-HCl, 150 mM NaCl, 10% (v/v) glycerol, 1 mM EDTA, 5 mM DTT, 1× protease inhibitor cocktail (Sigma-Aldrich #P9599), 0.2% NP-40, pH 7.4] at a ratio of 2 mL buffer per 1 g leaf material. Crude protein extracts were centrifuged at 20000 x g / 4 °C / 20 min and the supernatant was subjected to ADPr-enrichment with 2.5 ug of the GST-Af1521 Macro (K35E, Y145R) domain. Per affinity purification 15 uL Glutathion-Sepharose 4B resin (Merck) was washed in 50 mM Tris-HCl pH 7.5, 50 mM NaCl, 5 mM DTT and pelleted by centrifugation (500 x g, RT, 1 min). 2.5 ug of GST-Af1521 were added in a total volume of 1 mL wash buffer and the tubes were rotated for 1 h at 4 °C. The resin was pelleted as above and washed twice with 0.8 mL plant extraction buffer (see plant protein extraction section above) supplemented with 10 uM MG132 (Merck). Cold plant extract was added directly to the resin and the samples were rotated for 2 h at 4 °C and washed three times with 1 mL cold plant extraction buffer. The proteins were eluted from the resin by adding SDS sample buffer and incubation at 75 °C for 5 min. Protein samples were separated by 12% SDS-PAGE until the running front had migrated 1 cm into the separating gel. The gels were stained with Coomassie and the protein area was cut from the gel. Proteins were in-gel digested with trypsin and desalted as described by (Majovsky et al., https://doi.org/10.1021/pr500164j). Dried peptides were dissolved in 5% acetonitrile, 0.1% trifluoric acid and injected into an EASY-nLC 1200 liquid chromatography system (Thermo Fisher Scientific). Peptides were separated using liquid chromatography C18 reverse phase chemistry employing a 120 min gradient increasing from 1 to 24 % acetonitrile in 0.1% FA and a flow rate of 250 nL/min. Eluted peptides were electrosprayed on-line into the Fusion Lumos Tribrid with a spray voltage of 2.0 kV and capillary temperature of 305°C. Peptides (MS1) were detected in the Orbitrap with the following settings: resolving power: 120,000; scan range m/z 300–1500; S-Lens RF 30%; AGC target Standard, max injection time (IT) on Auto mode, microscans: 1. Dynamic exclusion duration was set for 60 s. MS/MS peptide sequencing was performed using a Top15 DDA scan strategy with HCD at 30% NCE, to be detected in the Orbitrap with a resolution of 30000.

### Data Protocol
Peptides and proteins were identified using the Mascot software v2.7.0 (Matrix Science) linked to Proteome Discoverer v2.4 (Thermo Fisher Scientific). The enzyme specificity was set to trypsin and two missed cleavages were tolerated. Carbamidomethylation of cysteine (C) was set as fixed modification, and oxidation of methionine (M) as well as ADP-ribosylation of aspartic acid (D) or glutamic acid or phosphorylation of serine (S) and threonine (T) as variable modifications. A precursor ion mass error of 10 ppm and a fragment ion mass error of 0.02 Da were tolerated in searches of the TAIR10 database amended with modified sequences of GST-Af1521 and common contaminants. A decoy database search was performed to determine the peptide spectral match (PSM) and peptide identification false discovery rates (FDR). Phosphorylated peptides with a score surpassing the false discovery rate threshold of 0.01 (q-value<0.01) were considered positive identifications. The phosphoRS module was used to specifically map ADP-ribosylation or phosphorylation to amino acid residues within the primary structure of peptides.

### Publication Abstract
None

### Keywords
Arabidopsis thaliana, Adp-ribosylation, Plant pathogen, Type iii effector, Pseudomonas syringae

### Affiliations
Department Biochemistry of Plant Interactions, Leibniz Institute of Plant Biochemistry, 06120 Halle (Saale), Germany
Leibniz Institute of Plant Biochemistry
Dept. Biochemistry of Plant Interactions
Weinberg 3
06120 Halle (Saale)

### Submitter
Lennart Wirthmueller

### Lab Head
Dr Lennart Wirthmueller
Department Biochemistry of Plant Interactions, Leibniz Institute of Plant Biochemistry, 06120 Halle (Saale), Germany


