### Accession
PXD016156

### Title
Proteomic Genotyping: Using Mass Spectrometry to Infer SNP Genotypes in Pigmented and Non-Pigmented Hair

### Description
Proteomic genotyping is the use of genetically variant peptides (GVPs), detected in a forensic protein sample, to infer the genotype of corresponding non-synonymous SNP alleles in the donorâ€™s genome. This process does not depend on the presence of accessible or useable DNA in a sample. This makes proteomic genotyping an attractive alternative for analysis of problematic forensic samples, such as hair shafts, degraded bones or teeth, fingermarks, or sexual assault evidence. To demonstrate the concept in hair shafts, we developed an optimized sample processing protocol that could be used with high effectiveness on single hairs. This allows us to determine if the detected profiles of genetically variant peptides are robust and result in a consistent profile of inferred SNP alleles regardless of the chemical or biological history of the sample.  Several real world scenarios have been evaluated. Here we include a study of four European subjects that had both pigmented and non-pigmented (or gray and non-gray) hair shafts. We tested whether (a) protein profiles change as a result of the loss of pigmentation and (b) these changes were reflected in the inferred genotype derived from detection of genetically variant peptides. Using this information, we can determine whether the resulting GVP profiles are more dependent on the biological context of pigmentation status or the underlying genotype.

### Sample Protocol
Pigmented and non-pigmented cranial hair shafts from four unrelated self-identified European-Americans were collected (UC Davis IRB# 832726). Samples were manually separated into pigmented and non-pigmented 4 mg fractions. Each sample was processed using an optimized hair sample processing protocol [2]. Samp-le digests were analyzed using a ThermoScientific Q Exactive Plus Orbitrap mass spectrometer with inbuilt Proxeon nanospray and Proxeon Easy-nLC II HPLC, under previously outlined conditions as described by Milan et al. Forensic Science International Genetics (2019) 41:19-23

### Data Protocol
The resulting datasets (.RAW format) were converted to a standard format (.mzML) and processed using the X!Tandem peptide spectra matching algorithm (thegpm.org). The protein amounts were quantified by label free quantification using the iBAQ function in MaxQuant and Scaffold (version 4.8.7) that focused on the three most abundant peptides that were specific for each gene product.   The differences between the proteomic profiles of the two groups were also evaluated using the Q-Module function of PEAKS Studio 10.0 (Bioinformatics Solutions Inc., Waterloo, ON, Canada). The raw data files were searched against a validated UNIPROT human reference proteome (uniprot-proteome_UP000005640_Human) using default settings of the algorithm except setting the precursor mass error range and fragment ion to 10 ppm and 0.04 Da, respectively. The search included post-translational modofications (PTMs) of cysteine carbamidomethylation as a fixed modification, with oxidation and dioxidation of methionine, deamidation of glutamines and asparagines, pyroglutamation at glutamines and glutamates, and formylation and acetylation of N-termini and lysines as variable modifications. The resulting datasets, filtered with a 1% FDR, were analyzed by Q-module and a heat map was generated by label free quantitation for proteins identified by two unique peptides and with at least 2-fold difference between the groups. A significance level, defined as the likelihood of probability of differential abundance as a quotient of a probability of no change, was set at 20.

### Publication Abstract
Proteomic genotyping uses genetically variant peptides that contain single amino acid polymorphisms to infer the genotype of corresponding non-synonymous SNP alleles. We have focused on hair proteins as a source of protein-based genetic information in a forensic context. An optimized sample processing protocol for hair shafts has been developed for use on a single hair that allows us to conduct validation protocols on real world samples. This includes whether the inferred SNP genotypes are robust and not systematically affected by biological or chemical variation in hair proteomes that might be obtained from a crime scene. To this end we analyzed the hair of 4 mature individuals with a mixture of pigmented and non-pigmented hair. We demonstrate significant changes in the proteomes of grey versus pigmented hair. Vesicle specific proteins and lipid catabolism proteins were enriched in pigmented hair, and housekeeping proteins and lipid anabolic enzymes were enriched in grey, non-pigmented hair. The resulting profiles of genetically variant peptides, however, were more correlated with profiles from the same individuals regardless of pigmentation status. Together with other published evidence, this finding indicates that profiles of genetically variant peptides are robust and more correlated with other genetically variant peptide profiles from the same individual irrespective of changes occurring in the hair protein profile. Based on this small sample, investigators using profiles of genetically variant peptides to infer random match probabilities should not expect to observe differences based on the pigmentation of the hair shaft.

### Keywords
Hair, Hair pigmentation, Proteomic genotyping, Melanosomes, Genetically variant peptides

### Affiliations
University of California Davis
UC Davis

### Submitter
Glendon Parker

### Lab Head
Dr Glendon John Parker
University of California Davis


