### Accession
PXD027137

### Title
Quantitative RNA interactome capture (qRIC) identifies phosphorylation sites with regulatory potential

### Description
RNA binding proteins (RBPs) are major regulators of gene expression at the post-transcriptional level. While many posttranslational modification sites in RBPs have been identified, less is known about how these modifications regulate RBP function. Here, we develop quantitative RNA-interactome capture (qRIC) to quantify the fraction of cellular RBPs pulled down with polyadenylated mRNAs. Applying qRIC to HEK293T cells quantified pulldown efficiencies of over 300 mRBPs. Combining qRIC with phosphoproteomics allowed us to systematically compare pulldown efficiencies of phosphorylated and non-phosphorylated forms of mRBPs. Over hundred phosphorylation sites show increased or decreased pull-down efficiency compared to their host RBPs and thus have regulatory potential. Our data  captures known regulatory phosphorylation sites in ELAVL1, SF3B1 and UPF1 and identifies new potentially regulatory sites. Follow-up experiments on the cardiac splicing regulator RBM20 revealed that multiple phosphorylation sites in the C-terminal disordered region affect nucleo-cytoplasmic shuttling, association with cytosolic RNA granules and alternative splicing. Together, we show that qRIC is a scalable method to identify the function of posttranslational modifications in RBPs.

### Sample Protocol
Fully labeled SILAC Human Embryonic Kidney 293T (HEK293T) cells were used in the qRIC protocol. Label swap experiments are indicated in the MS raw file name as "Forward" or "Reverse". Proteins were reduced with 10 mM DTT and alkylated in the dark with 55 mM of iodoacetamide (experiment 1) or chloroacetamide(experiment 2 and 3). Peptides were obtained by digestion proteins with LysC and Trypsin. Peptides were desalted with C18 Stage Tips and a large fraction of the peptide sample (90 %) was used for enrichment of phosphopeptides. Remaining peptides were eluted with 50% acetonitrile 0.1% formic acid, dried and resuspended in 3% acetonitrile, 0.1% formic acid for LC-MSMS measurement. For the experiment 1 that included non-crosslinked cells in the medium-heavy channel, peptides were fractionated by cation-exchange chromatography prior to LC-MS/MS analysis to improve coverage (MS raw files are tagged as "Frac1", "Frac2" or "Frac3"). MS raw files from Oct 2018 are from the experiment 1. MS raw files from Dec 2018 and Jan 2019 are from experiment 2 (tagged as "III") and 3 (tagged as "IV").  For phosphoproteomics (MS raw files are tagged with "Phospho"), phosphorylated peptides were enriched using a microcolumn tip packed with TiO2 (Rappsilber et al., 2007). Bound phosphopeptides were eluted using successive elution with 5 % NH3.H2O solution (fraction 1) followed by 5 % piperidine solution (fraction 2) (MS raw files are tagged for elution fraction "1" or "2"). Each fraction was collected into a fresh tube separately containing 30 µL of 20 % formic acid and further acidified with formic acid until pH smaller than 2 was obtained. The phosphopeptides were desalted with C18 Stage Tips prior to LC-MS/MS analysis. For LC-MS/MS analysis, desalted peptides were eluted from Stage Tips with 50 % acetonitrile 0.1 % formic acid solution, dried and resuspended in 3 % acetonitrile 0.1% formic acid. Peptide concentration was determined based on 280 nm UV light absorbance. Reversed-phase liquid chromatography was performed employing an EASY nLC II 1200 (Thermo Fisher Scientific) using self-made 15 cm long C18 microcolumns (75 μm ID, packed with ReproSil-Pur C18-AQ 1.9-μm resin, Dr. Maisch) connected on-line to the electrospray ion source (Proxeon) of an HF-X Orbitrap mass spectrometer (Thermo Fisher Scientific). The mobile phases consisted of 0.1 % formic acid 5 % acetonitrile solution (Buffer A) and 0.1 % formic acid 80 % acetonitrile solution (Buffer B). Peptides were eluted at a flow rate of 250 nL/min over 44 to 214 min of increasing Buffer B concentration. Settings for data dependent mass spectrometry analysis were as follow: positive polarity, one full scan (resolution, 60000; m/z range, 350-1800; AGC target, 3e6; max injection time, 10 ms) followed by top 20 MS/MS scans using higher-energy collisional dissociation (resolution, 15000; m/z range, 200-2000; AGC target, 1e5; max injection time, 22 ms; isolation width, 1.3 m/z; normalized collision energy, 26). Ions with an unassigned charge state, singly charged ions, and ions with charge state higher than six were rejected. Former target ions selected for MS/MS were dynamically excluded within 20 s.

### Data Protocol
All raw files from the same experiment were analyzed together with MaxQuant software (v1.6.0.1) (Cox and Mann, 2008) using default parameters. For increasing transparency and reproducibility of data analysis the “mqpar.xml” file generated by MaxQuant was deposited together with the raw data. Briefly, search parameters used for identification and quantification included two missed cleavage sites, cysteine carbamidomethyl as fixed modification, and the following variable modifications: methionine oxidation, protein N-terminal acetylation, and asparagine or glutamine deamidation. Up to three variable modifications per peptide were allowed. Lys0 and Arg0, Lys4 and Arg6, or Lys8 and Arg10 were set as multiplicity labels. Peptide mass tolerance was 20 and 4.5 ppm for first and main search, respectively. Database search was performed with Andromeda embedded in MaxQuant against the UniProt/Swiss-Prot Human proteome (downloaded in January 2019) with common contaminant sequences provided by MaxQuant. False discovery rate was set to 1% at peptide spectrum match and protein levels. Minimum peptide count required for protein quantification was set to two. The “Requantify” option was turned on. An identical MaxQuant search but with the “Requantify” option off was performed by partial reprocessing of search post peptide searches (starting from step “Re-quantification”). The second run (with Requantify off) was used for identification and exclusion of unscrupulous ratios (defined as ratios between two requantified values). Results from both searches with and without Requantify are provided.

### Publication Abstract
Cellular mRNA-binding proteins (mRBPs) are major posttranscriptional regulators of gene expression. Although many posttranslational modification sites in mRBPs have been identified, little is known about how these modifications regulate mRBP function. Here, we developed quantitative RNA-interactome capture (qRIC) to quantify the fraction of mRBPs pulled down with polyadenylated mRNAs. Combining qRIC with phosphoproteomics allowed us to systematically compare pull-down efficiencies of phosphorylated and nonphosphorylated forms of mRBPs. Almost 200 phosphorylation events affected pull-down efficiency compared with the unmodified mRBPs and thus have regulatory potential. Our data capture known regulatory phosphorylation sites in ELAVL1, SF3B1, and UPF1 and identify potential regulatory sites. Follow-up experiments on the splicing regulator RBM20 revealed multiple phosphorylation sites in the C-terminal disordered region affecting nucleocytoplasmic localization, association with cytoplasmic ribonucleoprotein granules, and alternative splicing. Together, we show that qRIC in conjunction with phosphoproteomics is a scalable method to identify functional posttranslational modification sites in mRBPs.

### Keywords
Hek293t, Lc-msms, Silac, Phosphoproteome, Qric

### Affiliations
Max Delbruck Center for Molecular Medicine, Robert-Rössle-Str. 10, D-13092 Berlin, Germany
Max-Delbrück-Centrum für Molekulare Medizin in der Helmholtz-Gemeinschaft

### Submitter
Carlos Henrique Vieira e Vieira

### Lab Head
Dr Matthias Selbach
Max Delbruck Center for Molecular Medicine, Robert-Rössle-Str. 10, D-13092 Berlin, Germany


