### Accession
PXD007922

### Title
Moss endogenous peptides, LC-MS/MS

### Description
Short (<100 codons) open reading frames (sORFs) are  common features of all genomes. Proof of translation of sORFs is an important step towards finding of functional sORF-encoded peptides (SEPs). We verified translation of our predicted sORFs using mass-spectrometry (MS) analysis which is often considered as the gold standard in detection of proteins or peptides in cell. Taking into account the shortage of proteomic methods in identifying small proteins or peptides, in our study we used “peptidomic” dataset - endogenous peptides, extracted from three types of moss cells – gametophores, protonemata and protoplasts

### Sample Protocol
Endogenous peptide extraction was conducted as described previously (Fesenko et al, 2015, ). For a Q Exactive HF mass spectrometer (Thermo Fisher Scientific, USA), peptide samples were separated by high-performance liquid chromatography (HPLC, Ultimate 3000 Nano LC System, Thermo Scientific, USA) in a 15-cm long C18 column with a diameter of 75 μm (Acclaim® PepMap™ RSLC, Thermo Fisher Scientific, USA). The peptides were eluted with a gradient from 5-35 % buffer B (80 % acetonitrile, 0.1 % formic acid) over 45 min at a flow rate of 0.3 μl/min. Total run time including 10 min to reach 99 % buffer B, flushing 5 min with 99 % buffer B and 10 min re-equilibration to buffer A (0.1 % formic acid) amounted 70 mins.  Mass spectra were acquired at a resolution of 60 000 (MS) and 15 000 (MS/MS) in a range of 400-1 500 m/z (MS) and 200-2 000 m/z (MS/MS). An isolation threshold of 67 000 was determined for precursors selection and up to top 10 precursors were chosen for fragmentation with high-energy collisional dissociation (HCD) at 25 NCE and 100 ms activation time. Precursors with a charged state of +1 were rejected and all measured precursors were excluded from measurement for 20 s.

### Data Protocol
Dataset was searched with MaxQuant v1.5.8.3 against a custom database containing the protein sequences from Phytozome v12.0 merged with chloroplast and mitochondrial proteins, database of common contaminant proteins and the set of predicted sORFs. MaxQuant’s protein FDR filter was disabled, while 1% FDR was used to select high-confidence PSMs (peptide spectrum matches) and ambiguous peptides were filtered out. Parameter “Digestion Mode” was set as ‘unspecific’ and modifications were not allowed. All other parameters were left at default values.

### Publication Abstract
Genomes contain millions of short (&lt;100 codons) open reading frames (sORFs), which are usually dismissed during gene annotation. Nevertheless, peptides encoded by such sORFs can play important biological roles, and their impact on cellular processes has long been underestimated. Here, we analyzed approximately 70,000 transcribed sORFs in the model plant <i>Physcomitrella patens</i> (moss). Several distinct classes of sORFs that differ in terms of their position on transcripts and the level of evolutionary conservation are present in the moss genome. Over 5000 sORFs were conserved in at least one of 10 plant species examined. Mass spectrometry analysis of proteomic and peptidomic data sets suggested that tens of sORFs located on distinct parts of mRNAs and long noncoding RNAs (lncRNAs) are translated, including conserved sORFs. Translational analysis of the sORFs and main ORFs at a single locus suggested the existence of genes that code for multiple proteins and peptides with tissue-specific expression. Functional analysis of four lncRNA-encoded peptides showed that sORFs-encoded peptides are involved in regulation of growth and differentiation in moss. Knocking out lncRNA-encoded peptides resulted in a decrease of moss growth. In contrast, the overexpression of these peptides resulted in a diverse range of phenotypic effects. Our results thus open new avenues for discovering novel, biologically active peptides in the plant kingdom.

### Keywords
Moss physcomitrella patens tissues

### Affiliations
IBCH RAS

### Submitter
Igor Fesenko

### Lab Head
Dr Vadim M Govorun
IBCH RAS


