### Accession
PXD039578

### Title
Benchmarking plexDIA analysis with SILAC-labelled Hela cell samples

### Description
SILAC labelled Hela cell lysates were mixed in defined ratios and analysed using both DDA and DIA methods, to serve as a benchmark for the novel plexDIA features of DIA-NN.

### Sample Protocol
Hela cells were grown in DMEM high glucose medium containing light, intermediate- and heavy Lysine and Arginine for 10 days. Lysates were prepared and mixed in two different compositions. Samples were generated from the combined lysates via the SP3 protocol (https://www.embopress.org/doi/full/10.15252/msb.20145625), using 100 µg protein input. The samples were measured on an QExactive-HF mass spectrometer in data-independent mode, coupled to an EASY-nLC 1200 liquid chromatography system.

### Data Protocol
Raw files from DIA measurements were analysed using DIA-NN version 1.8.1. A predicted spectral library was generated from the attached reviewed human fasta file (SwissProt database, version from February 2021 with 20934 entries). Additionally, a fasta file containing common protein contaminants was added for the spectral library prediction (https://pubs.acs.org/doi/10.1021/acs.jproteome.2c00145). Default settings were used for the spectral library prediction, with the addition of Methionine oxidation as variable modification. For the processing of the raw files, the default settings of DIA-NN were used with additional functions from the plexDIA module enabled. Three SILAC channels with mass shifts corresponding to Lys, Lys4 (+4.025107 Da), Lys8 (+8.014199 Da), Arg, Arg6 (+6.020129 Da), Arg10 (+10.008269 Da) and an additional decoy channel with Lysine (+12.0033 Da) and Arginine (+13.9964 Da) were registered. Translation of retention times between peptides within the same elution group was enabled. The first 13C-isotopic peak and monoisotopic peak was included for the quantification and the MS1 deconvolution level was set to 2. Matrix output tables were filtered for FDR < 0.01, channel q-value < 0.01 and translated q-value < 0.01. The MBR function was enabled to increase data completeness. The output tables of DIA-NN specifically the “report.pr_matrix_channels_translated.tsv” was processed in the R software environment (version 4.0.3) using a custom script. Identified contaminants were removed and protein abundance was calculated using the MaxLFQ algorithm, applied to the individual SILAC channels, using the iq R-package function “process_long_format()” (https://academic.oup.com/bioinformatics/article/36/8/2611/5697917).

### Publication Abstract
None

### Keywords
Hela, Plexdia, Silac, Dia-nn, Dia

### Affiliations
German Cancer Research Center (DKFZ) Heidelberg University Clinic
German Cancer Research Center (DKFZ)

### Submitter
Toman Bortecen

### Lab Head
Dr Jeroen Krijgsveld
German Cancer Research Center (DKFZ) Heidelberg University Clinic


