### Accession
PXD013231

### Title
Analysis of 1,508 plasma samples of the DiOGenes study - Robust, single shot capillary flow data-independent acquisition to decipher proteomic profiles of weight loss and maintenance

### Description
Based on our own experience and the evaluation of recent large-scale plasma mass spectrometry (MS) based proteomic studies (1, 2), we identified two outstanding challenges: slow and delicate nano-flow liquid chromatography (LC) and irreproducibility of identification of data-dependent acquisition (DDA). We determined a optimal solution reducing these limitations with   robust capillary-flow data-independent acquisition (DIA) MS. This platform is capable of measuring 31 plasma proteomes per day. Using this setup, we acquired a large-scale plasma study of the diet, obesity and genes dietary (DiOGenes) comprising 1,508 samples.

### Sample Protocol
The plasma samples were prepared with the Sample Preparation Kit Pro (Biognosys).In brief, volumes of 10 ?l plasma were mixed with 90 ?l of denaturation buffer, reduced at 37�C for 30 min followed by alkylation in the dark for 30 min. Then, 15 ?l of denatured plasma was mixed with ammonium bicarbonate buffer and 2 ?g of trypsin. Digestion was performed at 37�C for 3 h and stopped with the digestion stop solution. Desalting was performed using MacroSpin C18 96-well plate (The Nest Group, Southborough, MA, USA) following manufacturer?s instructions. For the DIA analysis, dried samples were resuspended in 50 �l LC of solution A containing iRT and PlasmaDeepDive kit. Finally, the plates were centrifuged at 14,000 ? g at 4 �C for 30 min, prior to acquisition. The samples were then transferred to a 96-well ACQUITY UPLC 700 �l sample plate (Waters, Milford, MA, USA).

### Data Protocol
To perform Tier 3 analyses (42), DIA spectra were analyzed with Spectronaut Pulsar X 12.0.20491.6 (37). The default settings were used. In brief, retention time prediction type was set to dynamic iRT (adapted variable iRT extraction width for varying iRT precision during the gradient) and correction factor for window 1. Mass calibration was set to local mass calibration. Interference correction on MS1 and MS2 level was enabled. The algorithm is based on correlation of extracted ion currents to a consensus elution profile (37). The false discovery rate (FDR) was estimated with the mProphet approach (43) and set to 1% at peptide precursor level and 1% at protein level (44). The DDA spectra were analyzed with the MaxQuant (Version 1.6.0.1) software (45) with default settings using the Andromeda search engine (46). Digestion enzyme specificity was set to Trypsin/P. Search criteria included carbamidomethylation of cysteine as a fixed modification, oxidation of methionine and acetyl (protein N-terminus) as variable modifications. Allowing up to 2 missed cleavages. The initial mass tolerance for the precursor was 4.5 ppm and for the fragment ions was 20 ppm. The DDA files were searched against the human UniProt fasta database (state 1st July 2017, 42,223 entries) and the Biognosys? iRT peptides fasta database (uploaded to the public repository). For a global post translational modification analysis, we searched the DDA data with MetaMorpheus (V.0.0.297) (47). The default settings were applied and Hex modifications were added.  For targeted glycation DIA analysis, Maxquant searches with the variable modifications of K,R and protein N-terminal glycation by C6H10O5 (162.0528 Da) was executed. Peptides with localization probability <0.75 were removed. The libraries were generated using the library generation functionality of Spectronaut with default settings.

### Publication Abstract
Comprehensive, high throughput analysis of the plasma proteome has the potential to enable holistic analysis of the health state of an individual. Based on our own experience and the evaluation of recent large-scale plasma mass spectrometry (MS) based proteomic studies, we identified two outstanding challenges: slow and delicate nano-flow liquid chromatography (LC) and irreproducibility of identification of data-dependent acquisition (DDA). We determined an optimal solution reducing these limitations with robust capillary-flow data-independent acquisition (DIA) MS. This platform can measure 31 plasma proteomes per day. Using this setup, we acquired a large-scale plasma study of the diet, obesity and genes dietary (DiOGenes) comprising 1508 samples. Proving the robustness, the complete acquisition was achieved on a single analytical column. Totally, 565 proteins (459 identified with two or more peptide sequences) were profiled with 74% data set completeness. On average 408 proteins (5246 peptides) were identified per acquisition (319 proteins in 90% of all acquisitions). The workflow reproducibility was assessed using 34 quality control pools acquired at regular intervals, resulting in 92% data set completeness with CVs for protein measurements of 10.9%.The profiles of 20 apolipoproteins could be profiled revealing distinct changes. The weight loss and weight maintenance resulted in sustained effects on low-grade inflammation, as well as steroid hormone and lipid metabolism, indicating beneficial effects. Comparison to other large-scale plasma weight loss studies demonstrated high robustness and quality of biomarker candidates identified. Tracking of nonenzymatic glycation indicated a delayed, slight reduction of glycation in the weight maintenance phase. Using stable-isotope-references, we could directly and absolutely quantify 60 proteins in the DIA.In conclusion, we present herein the first large-scale plasma DIA study and one of the largest clinical research proteomic studies to date. Application of this fast and robust workflow has great potential to advance biomarker discovery in plasma.

### Keywords
High throughput, Plasma proteomics, Swath, Stable isotope-based quantification, Library, Single shot, Clinical proteomics, Label-free quantification, Data-independent acquisition

### Affiliations
Biognosys AG
Biognosys AG, Schlieren, Schweiz

### Submitter
Roland Bruderer

### Lab Head
Dr Lukas Reiter
Biognosys AG, Schlieren, Schweiz


