### Accession
PXD015038

### Title
Dynamic study of the cell wall integrity signaling pathway in Aspergillus nidulans

### Description
A phosphoproteomic evaluation of the cellular response to micafungin (cell wall perturbant) in the filamentous fungi, A. nidulans, reveals putative, currently unknown cell wall integrity signaling (CWIS) associated proteins. This study used thirteen timepoints were taken from 0 to 10 minutes after cell wall exposure with two biological and two technical replicates. All data was processed using label free quantification with MaxQuant software.

### Sample Protocol
Aspergillus nidulans A1405 (Fungal Genetics Stock Center; FGSC) was used as the control strain. Frozen stocks were spread on MAGV plates (2% malt extract, 1.5% agar, 2% glucose, 2% peptone, and 1mL/L Hutner’s trace elements and vitamin solution) and incubated for 2 days at 28⁰C (Chelius et al., 2019). 1E7 spores were harvested and inoculated into 50mL of YGV (pH 3.3) (0.5% yeast extract, 1% glucose, and 1mL/L Hutner’s trace elements and vitamin solution). Culture was grown in a 250mL baffled flask at 250rpm and 28⁰C. After 12 hours growth, this flask was used to seed 1.2L YGV in a 2.8L Fernbach flask. Fungi were grown until mid-exponential phase (about 20 hours) and 20 ng micafungin/mL culture per 1g/kg DCW was added to the culture. About 25mL of culture was removed from the flask while the shaker was still shaking (ensuring a homogeneous sample) at 0, 30s, 1, 1.5, 2 ,2.5, 3, 3.5, 4, 4.5, 5, 7.5, and 10 minutes after micafungin exposure. Immediately after removing sample, the fungal mass separated from broth, frozen in liquid nitrogen, and stored at -80C. Samples harvested from shake flask culture were prepared for phosphoproteomic analysis following methods of Chelius et al., 2019. Briefly, frozen biomass was crushed by mortar and pestle with liquid nitrogen and TNE buffer (50mM Tris, 140mM NaCl, and 5mM EDTA) was added. The BCA assay (Pierce, Rockford, IL) was used to determine protein concentration followed by the The Filter aided sample preparation (FASP) for protein digestion (Chelius et al., 2019). Following protein alkylation and desalting, 500mg protein was incubated with Trypsin Gold (Promega, Madison, WI) (50μg protein: 1μg trypsin) overnight in a 37⁰C water bath. The samples were lyophilized to dryness and phosphopeptides were collected using the Pierce High-Select TiO2 Phosphopeptide Enrichment Kit (Pierce, Rockford, IL) protocol. Samples were dried to completeness and stored at -80⁰C until mass spectrometry analysis.  Stored samples were reconstituted in 20 µL of acetonitrile/water/formic acid 5/95/0.5 (v/v/v) and run in duplicate on a Dionex Ultimate 3000 nanoLC system coupled to an Orbitrap Fusion Tribrid mass spectrometer (Thermo Scientific, San Jose, CA). Separation of peptides was performed on EASY spray C18 75 µm x 50cm column for a 190 min gradient at flow rate of 200 nL/min using mobile phase A of 0.1% formic acid in water and mobile phase B of acetonitrile/water/formic acid 80/20/0.1 (v/v/v). Mass spectrometry data were collected in positive ionization mode using a data dependent acquisition method with a full MS scan for m/z range 350–1500 in orbitrap at 120 K resolution. Consecutive MS/MS scans were performed in the ion trap by top-speed decision selection with a dynamic exclusion of 20 s. Precursor ions selected from the first MS scan were isolated with an isolation width of 1.6m/z for collision induced dissociation (CID) energy and normalized collision energy (NCE) set to 30. Chelius, C. L., et al., 2019. Phosphoproteomic and transcriptomic analyses reveal multiple functions for Aspergillus nidulans MpkA independent of cell wall stress. Fungal Genetics and Biology. 125, 1-12.

### Data Protocol
Samples from both biological replicates and technical replicates for all timepoints (52 samples) were processed simultaneously with MaxQuant software version 1.5.3.17 against a UniprotKB database containing 10,555 A. nidulans protein sequences. MaxQuant analysis was performed following software parameters: a fixed modification of carbamidomethyl (C), dynamic modifications of acetylation (N-terminus), oxidation (M), phosphorylation (STY), and maximum missed cleavages of 2. Peptide spectrum matches (PSM) were filtered at a false discovery rate of 0.01 and protein identification was determined at FDR of 0.01.Sample time points are associated with the following MS file names: 0 (A), 0.5 (B), 1 (C), 1.5 (D), 2 (E), 2.5 (F), 3 (G), 3.5 (H), 4 (I), 4.5 (J), 5 (K), 7.5 (L), 10 (M) minutes.

### Publication Abstract
The fungal cell-wall integrity signaling (CWIS) pathway regulates cellular response to environmental stress to enable wall repair and resumption of normal growth. This complex, interconnected, pathway has been only partially characterized in filamentous fungi. To better understand the dynamic cellular response to wall perturbation, a &#x3b2;-glucan synthase inhibitor (micafungin) was added to a growing <i>A. nidulans</i> shake-flask culture. From this flask, transcriptomic and phosphoproteomic data were acquired over 10 and 120 min, respectively. To differentiate statistically-significant dynamic behavior from noise, a multivariate adaptive regression splines (MARS) model was applied to both data sets. Over 1800 genes were dynamically expressed and over 700 phosphorylation sites had changing phosphorylation levels upon micafungin exposure. Twelve kinases had altered phosphorylation and phenotypic profiling of all non-essential kinase deletion mutants revealed putative connections between PrkA, Hk-8-4, and Stk19 and the CWIS pathway. Our collective data implicate actin regulation, endocytosis, and septum formation as critical cellular processes responding to activation of the CWIS pathway, and connections between CWIS and calcium, HOG, and SIN signaling pathways.

### Keywords
Dynamic, Cell wall integrity signaling, Phosphoproteomics, Aspergillus nidulans, Lfq, Micafungin

### Affiliations
Marten Lab Chemical, Biochemical, and Environmental Engineering University of Maryland Baltimore County Baltimore, MD 21250
University of Maryland Baltimore County

### Submitter
Cynthia Chelius

### Lab Head
Dr Mark R. Marten
Marten Lab Chemical, Biochemical, and Environmental Engineering University of Maryland Baltimore County Baltimore, MD 21250


