### Accession
PXD018115

### Title
Protein interaction partners of drosophila Unr identified by pull down of wild type and an interdomain mutant protein

### Description
To identify an RNP interactome of Unr, V5 tagged wildtype protein and a mutant containing several interdomain contacts, that shows reduced protein expression were tansfected in SL2 cells after a siRNA knoch-down of the endogenous protein levels. After pulling down the transfected protein with an antibody against the V5-tag differences interacting proteins were identified using mass spectrometry. One set was done with an RNase treatment and one without, to identify interaction partners that rely on RNA binding. The lower expression level of the mutant protein allowed us to take this as a specific background control.

### Sample Protocol
For Unr immunoprecipitation, endogenous Unr was knocked down to liberate rate-limiting targets and the SL2 cells were transfected with 2 ug pAC-Unr wild type and mutants using the effectene transfection reagent according to the manufacture's protocol. UV crosslinking was done three days after the transfection in a thin layer of ice-cold PBS at 300 mJ/cm3 to stabilize transient and weak RNA protein interactions. For cell lysis, cells were sonicated in 20 mM Hepes/NaOH pH 7.5, 100mM NaCl, 1mM MgCl2, 0.05% NP-40 and 40U/ml RNaseOUT for 3 cycles of 30 sec at a low energy level using a Bioruptor. Afterwards the cleared lysate of the wild type, mutant and empty vector transfected cells was incubated with 1ug of V5 antibody per 3 mg of total protein and incubated at 4Â°C for 2h. The total protein concentration was determined earlier by a BCA assay. After incubation, 4ul of magnetic protein A/G beads per 1ug of used antibody was added and incubated for another 10 minutes. To get rid of unspecific binding the samples were washed with 1 ml 20 mM Hepes/NaOH pH 7.5, 150 mM NaCl, 1 mM EDTA, 0.5% NP-40, 0.5 mM DTT, 40U/ml RNaseOUT and 1% Triton-X for three times, before the beads were resuspended in the final buffer and volume. In case of the RNase treated samples, 250U of benzonase were added during washing steps. The samples were resuspended in 0.030 ml 20 mM Hepes/NaOH pH 7.5, 150 mM NaCl and 10% SDS. The immunoprecipitated samples were incubated for 5 min at 95Â°C and subjected to an in-solution tryptic digest using a modified version of the Single-Pot Solid-Phase-enhanced Sample Preparation (SP3) protocol. In total three biological replicates were prepared including control, wild type and mutant derived lysates (n=3). The lysates were added to Sera-Mag Beads in 10 Âµl 15% formic acid and 30 Âµl of ethanol. Binding of proteins was achieved by shaking for 15 min at room temperature. SDS was removed by 4 subsequent washes with 200 Âµl of 70% ethanol. Proteins were digested overnight at room temperature with 0.4 Âµg of sequencing grade modified trypsin in 40 Âµl Hepes/NaOH, pH 8.4 in the presence of 1.25 mM TCEP and 5 mM chloroacetamide. Beads were separated, washed with 10 Âµl of an aqueous solution of 2% DMSO and the combined eluates were dried down. Peptides were reconstituted in 10 Âµl of H2O and reacted for 1 h at room temperature with 80 Âµg of label reagent dissolved in 4 Âµl of acetonitrile. Excess TMT reagent was quenched by the addition of 4 Âµl of an aqueous 5% hydroxylamine solution. Peptides were reconstituted in 0.1 % formic acid, mixed to achieve a 1:1 ratio across all TMT-channels and purified by a reverse phase clean-up step. Peptides were subjected to an off-line fractionation under high pH conditions. The resulting 12 fractions were then analyzed by LC-MS/MS using a 2h gradient on an Orbitrap Fusion Lumos mass spectrometer as previously described. To this end, peptides were separated using an Ultimate 3000 nano RSLC system equipped with a trapping cartridge (Precolumn C18 PepMap100, 5 mm, 300 Î¼m i.d., 5 Î¼m, 100 Ã…) and an analytical column (Acclaim PepMap 100. 75 Ã— 50 cm C18, 3 mm, 100 Ã…) connected to a nanospray-Flex ion source. The peptides were loaded onto the trap column at 30 Âµl per min using solvent A (0.1% formic acid) and eluted using a gradient from 2 to 40% Solvent B (0.1% formic acid in acetonitrile) over 2 h at 0.3 Âµl per min. The Orbitrap Fusion Lumos was operated in positive ion mode with a spray voltage of 2.4 kV and capillary temperature of 275 Â°C. Full scan MS spectra with a mass range of 375â€“1500 m/z were acquired in profile mode using a resolution of 120,000 time of 50 ms or a maximum of 4e5 ions and a RF lens setting of 30%. Fragmentation was triggered for 3 s cycle time for peptide like features with charge states of 2â€“7 on the MS scan (data-dependent acquisition). Precursors were isolated using the quadrupole with a window of 0.7 m/z and fragmented with a normalized collision energy of 38. Fragment mass spectra were acquired in profile mode and a resolution of 30,000 in profile mode. Maximum fill time was set to 64 ms or an AGC target of 1e5 ions). The dynamic exclusion was set to 45s.  Acquired data were analyzed using IsobarQuant and Mascot V2.4 using a reverse UniProt FASTA Drosophila melanogaster database (UP000000803) including common contaminants. The following modifications were taken into account: Carbamidomethyl (C, fixed), TMT10plex (K, fixed), Acetyl (N-term, variable), Oxidation (M, variable) and TMT10plex (N-term, variable). The mass error tolerance for full scan MS spectra was set to 10 ppm and for MS/MS spectra to 0.02 Da. A maximum of 2 missed cleavages were allowed. A minimum of 2 unique peptides with a peptide length of at least seven amino acids and a false discovery rate below 0.01 were required on the peptide and protein level.

### Data Protocol
The protein.txt output files of IsobarQuant (Franken et al., 2015) were processed with the R programming language (ISBN 3-900051-07-0). To ensure a good data quality, only proteins that were quantified with at least 2 unique peptides (qupm column >= 2) were used for the following analysis. Furthermore, only proteins that have been identified in two out of three replicates were kept. The â€˜signal_sumâ€™ columns were cleaned for batch effects using the removeBatchEffect function of the limma package (Ritchie et al., 2015). Then, data were normalized with the vsn package (Huber et al., 2002). A separate normalization was applied for control conditions, normal pull-down conditions and RNAse treated pull-down conditions. Potential missing values were imputed with the impute function of the Msnbase package (Gatto and Lilley, 2012). Limma was used to test for differential abundance. Within this analysis, imputed values were given a weight of 5 %. When testing for differential abundance between conditions of different normalization groups (e.g. normal pull-down vs control condition), adjusted p-values from limma output were used as the false discovery rate (fdr). For all other tests, t-values from the limma output were used as an input to the fdrtool function of fdrtool (Strimmer, 2008) in order to calculate the fdr (qvalues were used). Proteins were classified as â€˜hitâ€™ with an fdr smaller 5 % and a fold-change of at least 100 % and as â€˜candidateâ€™ with an fdr smaller 20 % and a fold-change of at least 50 %. Hit and candidate proteins (tests: mutant vs WT, mutant_RNAse vs WT_RNAse, mutant vs WT / mutant_RNAse vs WT_RNAse) were clustered based on the Euclidean distance between normalized tmt reporter io signals (signal_sums) normalized by the WT or WT_RNAse condition using the kmeans algorithm.

### Publication Abstract
None

### Keywords
Rbp, Rnp, Interactome, Unr, Pull-down

### Affiliations
EMBL
Structural and Computational Biology Unit, EMBL Heidelberg, MeyerhofstraÃŸe 1, 69117, Heidelberg, Germany

### Submitter
Frank Stein

### Lab Head
Dr Janosch Hennig
Structural and Computational Biology Unit, EMBL Heidelberg, MeyerhofstraÃŸe 1, 69117, Heidelberg, Germany


