### Accession
PXD005248

### Title
Determination of pancreatic cyst fluids peptidome and proteome

### Description
Determination of pancreatic cyst fluids (PCF) peptidome and proteome in 5 distinct patient groups: malignant (CAs), intraductal papillary mucinous neoplasms (IPMNs), mucinous cystic neoplasms (MCNs), serous cystic neoplasms (SCNs), and pseudocysts (PCs). The PCF <5kDa fraction, referred as the degradome, and proteome profiles were determined using a LTQ-Orbitrap Elite mass spectrometer coupled with a nanoAcquity LC system. Qualitative LC-MS/MS analyses were ran on pooled samples from all patients.

### Sample Protocol
Cyst fluid degradome. A pancreatic cyst fluid (PCF) degradome, which referred to the low molecular weight (LMW) fractions of the PCF proteome, was analyzed as described previously for the plasma/serum peptidome. Briefly, a mix of 100 Âµl PCF + 300 Âµl 25% acetonitrile in 25 mM NH4HCO3 was centrifuged at 5000 x g, at 15oC, for 90 min. through the 30 kDa cutoff filtration membrane (Millipore Ultrafree-MC, pore size 30000, Sigma-Aldrich), washed twice with 25% acetonitrile prior to use. Next, 100 Âµl 25% acetonitrile was applied to the filter and again centrifuged for 60 min. Finally, the combined filtrates were centrifuged at 5000 x g, at 15C, for 120 min. through the 5 kDa cutoff filtration membrane (Millipore Ultrafree-MC, pore size 5000), and the <5 kDa fraction was lyophilized (Speed Vac) in siliconized microtubes (Clear-view Snap-Cap microtubes, size 1.5 mL, low retention, Sigma-Aldrich) and stored at â€“80C for further use. Prior to LC-MS analysis, samples were dissolved in 0.1% trifluoroacetic acid. For the acquisition of MS/MS data, pooled PCF samples were prepared by combining 10 Âµl of cyst fluids from the samples enrolled in the study. Cyst fluid proteome. Proteins were precipitated from equal amounts (by protein content) of PCF samples using the ProteoExtract Protein Precipitation Kit (Calbiochem Cat. No. 539790) according to the manufacturerâ€™s protocol. Protein pellets were resuspended in 100 ÎĽl 100mM NH4HCO3, the protein concentrations were measured by the BCA method (Pierceâ„˘ BCA Protein Assay Kit) and aliquots equal to 20 g proteins were reduced in 5mM Tris-(2-carboxyethyl) phosphine (Sigma-Aldrich), alkylated with 8mM S-Methyl methanethiosulfonate (Sigma-Aldrich), trypsin-digested (Sequencing Grade Modified Trypsin, Promega) using standard protocols and, finally, acidified with 5%TFA to pH=4. LC-MS analysis of peptides was performed on a LTQ-Orbitrap Elite mass spectrometer (Thermo Scientific) coupled with a nanoAcquity (Waters Corporation) LC system. Spectrometer parameters were as follows: polarity mode, positive; capillary voltage, 2 kV. A sample was first applied to the nanoACQUITY UPLC Trapping Column (Waters) using water containing 0.1% formic acid as the mobile phase. Next, the peptide mixture was transferred to the nanoACQUITY UPLC BEH C18 Column (Waters, 75 ÎĽm inner diameter; 250 mm long) and an ACN gradient (5â€“30% over 120 min) was applied in the presence of 0.1% formic acid with a flow rate of 250 nl/min and eluted directly to the ion source of the mass spectrometer. Each LC run was preceded by a blank run to avoid sample carry-over between the analyses. Qualitative LC-MS/MS analyses were performed on pooled samples in data-dependent acquisition mode and high-energy collision dissociation (HCD) was used for peptide fragmentation. To increase the number of peptide identifications, three LC-MS/MS runs were performed per pooled sample, each covering one of three ranges of m/z values: 300â€“600, 600â€“800 and 800â€“2000.

### Data Protocol
The acquired MS/MS raw data files were preprocessed with Mascot Distiller (version 2.5.1 , Matrix Science), and the resulting peak lists were submitted to the Mascot engine (version 2.4.1, Matrix Science) and searched against a database composed of SwissProt Homo sapiens entries (release 2016.01) concatenated with and contaminant proteins sequences retrieved from the common Repository of Adventitious Proteins (cRAP, http://www.thegpm.org/crap/index.html). In total, the database included 20,233 target sequences, and the same number of reversed decoy records.  The search parameters were as follows: enzyme specificity: trypsin for proteome samples, and none for degradome; maximum number of missed cleavages (for proteome only): 1; protein mass: unrestricted; parent ions mass error tolerance: 5 ppm; fragment ions mass error tolerance: 0.01 Da; fixed modifications (for proteome only): Methylthio (C); variable modifications: Oxidation (M).  The statistical significance of identified peptides was determined using a target/decoy database search approach and a previously described procedure that provided q-value estimates for each peptide spectrum match (PSM) in the data set. Only PSMs with q-values â‰¤ 0.01 were regarded as confidently identified. Furthermore, all the peptide sequences matched to database entries representing contaminant proteins were rejected. For proteome samples proteins represented by less than two peptides were excluded from further analysis. Proteins identified by a subset of peptides from other proteins were filtered out from the results, and those matching the same set of peptides were grouped together into clusters (metaproteins). All the steps involved in Mascot results processing were performed using MScan, a proprietary Java application available at http://proteom.ibb.waw.pl/mscan.

### Publication Abstract
Pancreatic cyst fluids (PCFs) enriched in tumour-derived proteins are considered a potential source of new biomarkers. This study aimed to determine compositional and quantitative differences between the degradome and proteome of PCFs aspirated from different types of pancreatic cyst lesions (PCLs). 91 patients who underwent endoscopic ultrasound-fine needle aspiration under routine clinical diagnosis of PCLs were enrolled. Four cysts were malignant (CAs), and 87 were nonmalignant and consisted of 18 intraductal papillary mucinous neoplasms (IPMNs), 14 mucinous cystic neoplasms (MCNs), nine serous cystic neoplasms (SCNs), 29 pseudocysts (PCs), and 17 unclassified. Profiles of the &lt;5 kDa fraction, the degradome, and the trypsin-digested proteome were analysed using an LTQ-Orbitrap Elite mass spectrometer coupled with a nanoACQUITY LC system. Qualitative analyses identified 796 and 366 proteins in degradome and proteome, respectively, and 689 (77%) and 285 (78%) of them were present in the Plasma Proteome Database. Gene Ontology analysis showed a significant overrepresentation of peptidases and peptidases inhibitors in both datasets. In the degradome fraction, quantitative values were obtained for 6996 peptides originating from 657 proteins. Of these, 2287 peptides were unique to a single type, and 515 peptides, derived from 126 proteins, were shared across cyst types. 32 peptides originating from 12 proteins had differential (adjusted <i>p</i>-value &#x2264;0.05, FC &#x2265;1.5) abundance in at least one of the five cysts types. In proteome, relative expression was measured for 330 proteins. Of them, 33 proteins had significantly (adjusted <i>p</i>-value &#x2264;0.05, FC &#x2265;1.5) altered abundance in at least one of the studied groups and 19 proteins appeared to be unique to a given cyst type. PCFs are dominated by blood proteins and proteolytic enzymes. Although differences in PCF peptide composition and abundance could aid classification of PCLs, the unpredictable inherent PCF proteolytic activity may limit the practical applications of PCF protein profiling.

### Keywords
Human, Cyst fluids, Lc-ms/ms, Proteome, Pancreas, Degradome

### Affiliations
Institute of Radioelectronics and Multimedia Technology
Warsaw University of Technology
Department of Gastroenterology, Hepatology and Clinical Oncology, Medical Center for Postgraduate Education; Warsaw, Poland Department of Genetics, Maria Sklodowska-Curie Memorial Cancer Center and Institute of Oncology; Warsaw, Poland

### Submitter
Tymon Rubel

### Lab Head
Dr Jerzy Ostrowski
Department of Gastroenterology, Hepatology and Clinical Oncology, Medical Center for Postgraduate Education; Warsaw, Poland Department of Genetics, Maria Sklodowska-Curie Memorial Cancer Center and Institute of Oncology; Warsaw, Poland


