### Accession
PXD001532

### Title
Quantitative Analysis of the erythrocyte membrane proteins in polycythemia vera patients treated with hydroxycarbamide

### Description
Polycythemia vera (PV) is a myeloproliferative disorder arising in pluripotent stem cells that causes an abnormal erythrocyte mass. More than 90% of PV patients have a mutation in JAK2 protein that is closely associated with the erythrocyte membrane. We report findings on quantitative analysis of the erythrocyte membrane proteins differentially regulated in PV patients treated with hydroxycarbamide.</br>Kottahachchi et al., EuPA Open Proteomics 7, 43-53</br>Quantitative analysis of the erythrocyte membrane proteins in polycythemia vera patients treated with hydroxycarbamide</br>DOI:10.1016/j.euprot.2015.04.001</br><a href="http://www.sciencedirect.com/science/article/pii/S2212968515000100">http://www.sciencedirect.com/science/article/pii/S2212968515000100</a> 

### Sample Protocol
Blood samples were obtained from five PV patients with the JAK2 V617F mutation  (P1-P5) and five healthy volunteers (N1-N5). All 5 patients were treated with  hydroxycarbamide and low dose aspirin for over 6 months and had also been  subjected to multiple therapeutic venesections. Erythrocyte membrane fractions  were isolated by osmotic lysis of erythrocytes, followed by high speed  centrifugation. The isolated membranes were solubilized by Triton X-100. The  erythrocyte membrane proteins were separated by SDS-PAGE and silver stained,  followed by in-gel digestion with trypsin. The in-gel digests were analyzed by  LC-MS/MS using a QSTAR Elite mass spectrometer (Applied Biosystems/MDS Sciex).  Also, the erythrocyte membrane proteins were in-solution digested, followed by  detergent removal and desalting. The in-solution digests were analyzed by LC- MS/MS using a Q Exactive mass spectrometer (Thermo Fisher Scientific).

### Data Protocol
Database searches of QSTAR data were performed by Mascot (version 2.2) (Matrix Science)  search engine against Swiss-Prot database (2011_08) using a Homo sapiens taxonomy filter  (20,245 sequences). Carbamidomethylation of cysteines was set as a fixed modification and  oxidation of methionines was set as a variable modification. One missed trypsin cleavage  site was allowed. Mass tolerance for MS and MS/MS was set to 0.3 Da, and only proteins  identified with minimum two unique peptides were selected. In case of Q Exactive data,  Mascot search was preformed via Proteome Discoverer (version 1.3, Thermo Fisher Scientific)  with the same parameters but mass tolerances for MS and MS/MS were set to 10 ppm and 0.02  Da, respectively. Scaffold software (version 3.3.2, Proteome software) was used to provide  Gene Ontology annotations to proteins, which were further checked based on Uniprot data.

### Publication Abstract
None

### Keywords
Lc-ms/ms, Polycythemia vera, Erythrocyte membrane associated proteins, Jak2 v617f

### Affiliations
Turku Centre for Biotechnology, University of Turku and Åbo Akademi University, Turku, Finland
Meijo University

### Submitter
Susumu Imanishi

### Lab Head
Dr Garry L. Corthals
Turku Centre for Biotechnology, University of Turku and Åbo Akademi University, Turku, Finland


