### Accession
PXD002471

### Title
Quantitative histone mass spectrometry identifies elevated histone H3 lysine 27 trimethylation in melanoma

### Description
Normal cell growth is characterized by a regulated epigenetic program that drives cellular activities such as gene transcription, DNA replication and DNA damage repair. Perturbation of this epigenetic program can lead to events such as mis-regulation of gene transcription and diseases such as cancer. To begin to understand the epigenetic program correlated to the development of melanoma, we performed a quantitative mass spectrometric analysis of histone posttranslational modifications mis-regulated in melanoma cell culture. Aggressive melanoma cells were found to have elevated histone H3 lysine 27 trimethylation (H3K27me3) as well as over-expressed methyltransferase EZH2 that adds the specific modification. The altered epigenetic program that led to elevated H3K27me3 in melanoma cell culture was found to directly silence transcription of the tumor suppressor gene RUNX3. The elevated level of H3K27me3 and silencing of RUNX3 transcription was also validated in advanced stage human melanoma tissues. The study presented underscores the utility of using high resolution mass spectrometry to identify mis-regulated epigenetic programs in diseases such as cancer, which could ultimately lead to the identification of biological markers for diagnostic and prognostic applications.

### Sample Protocol
The human melanoma cell lines WM266-4 and WM115 were purchased from ATCC. Cells were cultured with DMEM medium supplemented with 10% fetal bovine serum (FBS) and 10% penicillin-streptomycin-L-glutamine and incubated at 37Â°C in a humidified incubator with 5% CO2. For all drug treatments, DMSO was used as negative control. Histones  were acid-extracted as reported previously (25) and resolved (20 Âµg histones per lane) by SDS-PAGE using 4-20% Novex Tris-Glycine gradient gels (Life Technologies) and stained with Thermo Scientific Pierce GelCode Blue stain reagent (Fisher).  The region of each gel lane containing the core histones was excised as one piece, diced into small pieces, de-stained, treated with d6-acetic anhydride to chemically block unmodified lysines and monomethylated lysines with an isotopically heavy acetyl, and digested in-gel with trypsin. Tryptic peptides were separated by reverse phase Jupiter Proteo resin (Phenomenex) on a 100 x 0.075 mm column using a nanoAcquity UPLC system (Waters).  Peptides were eluted using a 40 min gradient from 97:3 to 35:65 buffer A:B ratio.  [Buffer A = 0.1% formic acid, 0.5% acetonitrile; buffer B = 0.1% formic acid, 75% acetonitrile.]  Eluted peptides were ionized by electrospray (1.9 kV) followed by MS/MS analysis using collision induced dissociation on an LTQ Orbitrap Velos mass spectrometer (Thermo Scientific).  MS data were acquired using the FTMS analyzer in profile mode at a resolution of 60,000 over a range of 375 to 1500 m/z.  MS/MS data were acquired for the top 15 peaks from each MS scan using the ion trap analyzer in centroid mode and normal mass range with a normalized collision energy of 35.0.

### Data Protocol
Proteins were identified by searching the UniProtKB database (2015_06 release; restricted to Homo sapiens; 20,207 entries) using an in-house Mascot Sever (v 2.4; Matrix Science).  Peak lists were generated from raw data files by MSFileReader (v2.2; Thermo Scientific) and ExtractMSn (January 2011 release; Thermo Scientific).  Charge state deconvolution and deisotoping were not performed.  No contaminants were excluded from the data set.  Mascot search parameters were specified as follows: trypsin digestion with up to two missed cleavages; fixed carbamidomethyl modification of cysteine; variable methyl, dimethyl, trimethyl, acetyl, acetyl:2H3 and methyl+acetyl:2H3 modification of lysine; variable phosphorylation of serine and threonine; 2.0 ppm precursor ion tolerance; 0.50 Da fragment ion tolerance.  A reverse-sequence decoy search was also performed.  Peptide and protein identifications were validated using Scaffold (v4.4; Proteome Software).  Peptide identifications were accepted at >50.0% probability as determined by the Scaffold Local FDR algorithm.  Protein identifications were accepted at >95.0% probability and a minimum of two identified peptides.  Protein probabilities were assigned by the Protein Prophet algorithm.  Proteins with similar peptides that could not be differentiated based on MS/MS analysis alone were grouped to satisfy the principles of parsimony.  Histone H2A, H2B, H3 and H4 peptide precursor intensity values were manually extracted from the raw files using Qual Browser (Thermo Xcalibur 2.2 SP1.48). The observed mass for each histone peptide was extracted from Scaffold and used to locate the appropriate MS spectra and export the precursor intensity value for each peptide. The intensity values for all histone peptides were log2 normalized, centered based on the median value,  scaled based on the standard deviation, and the median of each technical triplicate value was used to calculate the Student t-test for the biological replicate samples (SAS version 9.4). The transformed data met the assumption of normality and equality of variances according to the Kolomogorov-Smirnov (p-value = 0.489) and the Levene test (p-value =.1161), respectfully (Unscrambler X, version 10.3, CAMO software).

### Publication Abstract
Normal cell growth is characterized by a regulated epigenetic program that drives cellular activities such as gene transcription, DNA replication, and DNA damage repair. Perturbation of this epigenetic program can lead to events such as mis-regulation of gene transcription and diseases such as cancer. To begin to understand the epigenetic program correlated to the development of melanoma, we performed a novel quantitative mass spectrometric analysis of histone post-translational modifications mis-regulated in melanoma cell culture as well as patient tumors. Aggressive melanoma cell lines as well as metastatic melanoma were found to have elevated histone H3 Lys(27) trimethylation (H3K27me3) accompanied by overexpressed methyltransferase EZH2 that adds the specific modification. The altered epigenetic program that led to elevated H3K27me3 in melanoma cell culture was found to directly silence transcription of the tumor suppressor genes RUNX3 and E-cadherin. The EZH2-mediated silencing of RUNX3 and E-cadherin transcription was also validated in advanced stage human melanoma tissues. This is the first study focusing on the detailed epigenetic mechanisms leading to EZH2-mediated silencing of RUNX3 and E-cadherin tumor suppressors in melanoma. This study underscores the utility of using high resolution mass spectrometry to identify mis-regulated epigenetic programs in diseases such as cancer, which could ultimately lead to the identification of biological markers for diagnostic and prognostic applications.

### Keywords
Human, Histone h3 lysine 27 trimethylation, Melanoma, Runx3, Ezh2

### Affiliations
UAMS
Department of Biochemistry and Molecular Biology University of Arkansas for Medical Sciences USA

### Submitter
Stephanie Byrum

### Lab Head
Dr Stephanie Byrum
Department of Biochemistry and Molecular Biology University of Arkansas for Medical Sciences USA


