### Accession
PXD024121

### Title
Extracellular vesicles as prognostic biomarkers for Multiple Myeloma - a clinical population-based study

### Description
Multiple myeloma (MM) is an incurable hematological malignancy of mature B lymphocytes. In this study patients with MGUS, SMM and MM followed in two hematology departments were enrolled. Extracellular vesicles were isolated from bone marrow (BM) and peripheral blood (PB) followed by mass spectrometry based bottom up proteomics.

### Sample Protocol
PB and/or BM samples were centrifuged at 500 g for 10 min. Then, the collected supernatant was centrifuged at 3000 g for 20 minutes, followed by another centrifugation at 12.000 g for 20 min. After this, sequential ultracentrifugation’s with gradient density protocol (UC-I) was performed as previously described (1).   MS sample preparation  Protein solution containing SDS and DTT were loaded onto filtering columns and washed exhaustively with 8M urea in HEPES buffer [2]. Proteins were reduced with DTT and alkylated with IAA. Protein digestion was performed by overnight digestion with trypsin sequencing grade (Promega).  Nano-LC-MSMS analysis  Peptide samples (1 µg) were analysed in duplicates by nano-LC-MSMS (Dionex RSLCnano 3000) coupled to a Q-Exactive Orbitrap mass spectrometer (Thermo Scientific) applying MS and LC settings as virtually as previously described (PMID: 33330473), or by coupling the same LC system to an orbitrap Exploris 480 mass spectrometer (Thermo Scientific) applying following LC and MS settings; The samples (5 μl) were loaded onto a custom made fused capillary pre-column (2 cm length, 360 μm OD, 75 μm ID packed with ReproSil Pur C18 3 μm resin (Dr Maish, GmbH)) with a flow of 5 μl/min for 7 min. Trapped peptides were separated on a custom made fused capillary column (25 cm length, 360 μm OD, 75 μm ID, packed with ReporSil Pur C13 1.9 μm resin) using a linear gradient ranging from 89 % solution A (0.1% formic acid) to 32% B (80% acetonitrile in 0.1% formic acid) over 56 min followed by a linear gradient to 50% B over 8 min at a flow rate of 250 nL per minute. Mass spectra were acquired in positive ion mode applying automatic data–dependent switch between an Orbitrap survey MS scan in the mass range of 350–1200 m/z followed by peptide fragmentation applying a normalized collisional energy of 30% in a 2 second duty cycle. Target value in the Orbitrap for MS scan was 1,000,000 ions at a resolution of 60,000 at m/z 200 and 200,000 ions at a resolution of 15,000 at m/z 200 for MS/MS scans. Ion selection threshold was set to 20,000 counts. Selected sequenced ions were dynamically excluded for 30 s.  References: 1. Maia J, Batista S, Couto N, et al. Employing Flow Cytometry to Extracellular Vesicles Sample Microvolume Analysis and Quality Control   . Front. Cell Dev. Biol.  . 2020;8:1165.   2) Wisniewski, J. R., Zougman, A., Nagaraj, N. & Mann, M. Universal sample preparation method for proteome analysis. Nature methods 6, 359–362, doi: 10.1038/nmeth.1322 (2009).

### Data Protocol
Database search. The obtained data from the 282 LC-MS runs were searched using VEMS[1,2]. A standard human proteome database from UniProt (3AUP000005640) for Permutated protein sequences, where Arg and Lys were not permutated, were included in the database. Trypsin cleavage allowing a maximum of 4 missed cleavages was used. Carbamidomethyl cysteine was included as fixed modification. Methionine oxidation and N-terminal protein acetylation was included as variable modifications. 10 ppm mass accuracy was specified for precursor ions and 0.01 m/z for fragment ions. The false discovery rate (FDR) for protein identification was set to 1% for peptide and protein identifications. No restriction was applied for minimal peptide length for VEMS search. Identified proteins were divided into evidence groups as defined by Matthiesen et al. [2]. Quantitative analysis. Quantitative data obtained from VEMS results were analyzed in R statistical programming language. Quantitative data was extracted for the 85 patients   (170 LCMS runs) selected in this current study. Intensity based absolute quantitation (IBAQ) [3] were preprocessed by removing common MS contaminants. Quantitative values from duplicate runs were averaged followed by log2(x + 1) transformation and quantile normalization.   1) Carvalho, A. S. et al. Global mass spectrometry and transcriptomics array based drug profiling provides novel insight into glucosamine induced endoplasmic reticulum stress. Molecular & cellular proteomics: MCP 13, 3294–3307, doi: 10.1074/mcp. M113.034363 (2014).  2)Matthiesen, R. et al. SIR: Deterministic protein inference from peptides assigned to MS data. Journal of proteomics 75, 4176–4183, doi: 10.1016/j.jprot.2012.05.010 (2012).  3) Schwanhäusser, B. et al. Nature 473, 337–342 (2011).

### Publication Abstract
None

### Keywords
Human, Extracellular vesicles, Myeloma, Hematological malignancy, Cancer

### Affiliations
Computational and Experimental Biology Group NOVA MEDICAL SCHOOL / FACULDADE DE CIÊNCIAS MÉDICASUNIVERSIDADE NOVA DE LISBOARua Câmara Pestana, 6-6A | 1150-082 Lisboa Portugal E-mail: rune.matthiesen@nms.unl.pt |  www.nms.unl.pt
Computational and Experimental Biology Group, CEDOC

NOVA MEDICAL SCHOOL / FACULDADE DE CIÊNCIAS MÉDICAS

UNIVERSIDADE NOVA DE LISBOA

Rua Câmara Pestana, 6-6A | 1150-082 Lisboa

Portugal

### Submitter
Rune Matthiesen

### Lab Head
Dr Rune Matthiesen
Computational and Experimental Biology Group NOVA MEDICAL SCHOOL / FACULDADE DE CIÊNCIAS MÉDICASUNIVERSIDADE NOVA DE LISBOARua Câmara Pestana, 6-6A | 1150-082 Lisboa Portugal E-mail: rune.matthiesen@nms.unl.pt |  www.nms.unl.pt


