### Accession
PXD035898

### Title
Parallel comparison of mutant proteomes with altered carbon metabolism reveals Hik8 regulates PII phosphorylation and the basal level glycogen accumulation in a cyanobacterium

### Description
To understand the specificity and the cross-talk of such regulations, we simultaneously and quantitatively compared the proteomes of the gene knockout mutants for the regulators. A number of proteins showing differential expression in one or more mutants were identified, including four proteins that are unanimously upregulated or downregulated in all the five mutants. These represent the important nodes of the intricate and elegant regulatory network for carbon metabolism.

### Sample Protocol
Cell pellets were lysed in a buffer containing 0.4 M sucrose, 50 mM MOPS, 10 mM NaCl, 5 mM EDTA (pH 7.0) and 0.5 mM PMSF with a bead beater, and the insoluble debris was removed by centrifugation for 30 min at 5,000 g at 4°C. After precipitation with ice-cold 10% trichloroacetic acid in acetone at -20°C, total proteins were washed with acetone and resolubilized with 4% sodium dodecyl sulfate (SDS) in 0.1 M Tris-HCl, pH 7.6. A BCA protein assay kit (Thermo Scientific, Rockford, IL) was used to determine the protein concentration.

### Data Protocol
The PRM experiment was performed as previously described (Peterson et al., 2012). The full scan mass range was 300-1500 m/z and the orbitrap resolution was 120,000 at 200 m/z. The automatic gain control (AGC) target and maximum injection time were set at 1x106 and 30 ms, respectively. A target list of precursor ions with charge state ≥2 was isolated and fragmented using HCD fragmentation with 32% collision energy and detected with orbitrap at a mass resolution of 30,000 at 200 m/z. The AGC target and maximum injection time were set at 5x104 and 54 ms, respectively.

### Publication Abstract
Carbon metabolism is central to photosynthetic organisms and involves the coordinated operation and regulation of numerous proteins. In cyanobacteria, proteins involved in carbon metabolism are regulated by multiple regulators including the RNA polymerase sigma factor SigE, the histidine kinases Hik8, Hik31 and its plasmid-borne paralog Slr6041, and the response regulator Rre37. To understand the specificity and the cross-talk of such regulations, we simultaneously and quantitatively compared the proteomes of the gene knockout mutants for the regulators. A number of proteins showing differential expression in one or more mutants were identified, including four proteins that are unanimously upregulated or downregulated in all five mutants. These represent the important nodes of the intricate and elegant regulatory network for carbon metabolism. Moreover, serine phosphorylation of P<sub>II</sub>, a key signaling protein sensing and regulating in&#xa0;vivo carbon/nitrogen (C/N) homeostasis through reversible phosphorylation, is massively increased with a concomitant significant decrease in glycogen content only in the hik8-knockout mutant, which also displays impaired dark viability. An unphosphorylatable P<sub>II</sub> S49A substitution restored the glycogen content and rescued the dark viability of the mutant. Together, our study not only establishes the quantitative relationship between the targets and the corresponding regulators and elucidated their specificity and cross-talk but also unveils that Hik8 regulates glycogen accumulation through negative regulation of P<sub>II</sub> phosphorylation, providing the first line of evidence that links the two-component system with P<sub>II</sub>-mediated signal transduction and implicates them in the regulation of carbon metabolism.

### Keywords
Glycogen, Cyanobacteria, Proteomics, Carbon metabolism, Synechocystis

### Affiliations
State Key Laboratory of Molecular Developmental Biology, Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, No.1 West Beichen Rd., Beijing 100101, China
University of Chinese Academy of Sciences

### Submitter
Chengcheng Huang

### Lab Head
Dr Yingchun Wang
State Key Laboratory of Molecular Developmental Biology, Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, No.1 West Beichen Rd., Beijing 100101, China


