### Accession
PXD021560

### Title
Trypanosoma brucei rhodensience suramin resistance shotgun proteomics

### Description
Human African Trypanosomiasis (HAT) is a disease of major economic importance in Sub-Saharan Africa. In eastern and southern Africa. Here we analysed clinical isolates of T brucei rhodensiense, resistant to suramin by shotgun proteomics . And identified parasite proteins whose expression is associated with resistance to suramin.

### Sample Protocol
We utilized parasites stabilates previously isolated from HAT patient with suramin  treatment failure (Tbr EATRO 734) or not (Tbr EATRO 232) in Busoga Uganda in 1964 and 1959 respectively. Both parasite isolates were initially expanded in Swiss white mice and subsequently cryopreserved at BioRI-KALRO cryo-bank. Total proteins were extracted and processed for mass spectrometry analysis

### Data Protocol
We analyzed the raw mass spectrometer files by MaxQuant software version 1.6.0.1 (49) by searching the peptides against the TriTrypDB-9.0_TbruceiTREU927_ Annotated Proteins FASTA database (downloaded 1st August 2017) using the Andromeda search engine. In this process, we set cysteine carbamidomethylation and TMT-10plex labeled N-terminus and lysine as a fixed modifications and N-terminal acetylation and methionine oxidations as variable modifications, and used both types of modifications for protein quantification. We set False Discovery Rate (FDR) to 0.01 for both proteins and peptide-spectrum matches. We determined the FDR by searching a FASTA protein database-comprising target and reversed target sequences (decoy) derived from Tbb, by switching the amino-carboxyl orientation of a proteinâ€™s amino acids to generate random sequences. We set enzyme specificity as C-terminal to arginine and lysine with trypsin as the protease. We allowed a maximum of two missed cleavages in the database search and performed peptide identification with initial precursor mass deviation of up to 7 ppm and fragment mass deviation of up to 20 ppm. We only accepted peptides that contained least seven amino acids long of at most 4600 Da. We specified TmT10plex under isobaric labels for reporter ion MS^2 and set reporter mass tolerance at 0.01Da. We extracted the 10plex reporter ion intensity matrix from the Maxquant protein group matrix file and used the information for downstream comparative proteome analyses

### Publication Abstract
Human African Trypanosomiasis (HAT) is a disease of major economic importance in Sub-Saharan Africa. The HAT is caused by Trypanosoma brucei rhodesiense (Tbr) parasite in eastern and southern Africa, with suramin as drug of choice for treatment of early stage of the disease. Suramin treatment failures has been observed among HAT patients in Tbr foci in Uganda. In this study, we assessed Tbr parasite strains isolated from HAT patients responsive (Tbr EATRO-232) and non-responsive (Tbr EATRO-734) to suramin treatment in Busoga, Uganda for 1) putative role of suramin resistance in the treatment failure 2) correlation of suramin resistance with Tbr pathogenicity and 3) proteomic pathways underpinning the potential suramin resistance phenotype in vivo. We first assessed suramin response in each isolate by infecting male Swiss white mice followed by treatment using a series of suramin doses. We then assessed relative pathogenicity of the two Tbr isolates by assessing changes pathogenicity indices (prepatent period, survival and mortality). We finally isolated proteins from mice infected by the isolates, and assessed their proteomic profiles using mass spectrometry. We established putative resistance to 2.5&#xa0;mg/kg suramin in the parasite Tbr EATRO-734. We established that Tbr EATRO-734 proliferated slower and has significantly enriched pathways associated with detoxification and metabolism of energy and drugs relative to Tbr EATRO-232. The Tbr EATRO-734 also has more abundantly expressed mitochondrion proteins and enzymes than Tbr EATRO-232. The suramin treatment failure may be linked to the relatively higher resistance to suramin in Tbr EATRO-734 than Tbr EATRO-232, among other host and parasite specific factors. However, the Tbr EATRO-734 appears to be less pathogenic than Tbr EATRO-232, as evidenced by its lower rate of parasitaemia. The Tbr EATRO-734 putatively surmount suramin challenges through induction of energy metabolism pathways. These cellular and molecular processes may be involved in suramin resistance in Tbr.

### Keywords
Proteomics, T brucei rhodesience, Suramin resistance

### Affiliations
KEMRI
KEMRI Wellcome Trust

### Submitter
Martin Rono

### Lab Head
Dr Martin rono
KEMRI Wellcome Trust


