### Accession
PXD005788

### Title
TMT analysis of TBX20:CASZ1 mutant mice with dilated cardiomyopathy

### Description
Using an unbiased systems-based screen, we demonstrate the cardiac transcription factor TBX20 physically and genetically interacts with the essential transcription factor CASZ1. This interaction is required for survival as mice heterozygous for both Tbx20 and Casz1 die post-natally as a result of dilated cardiomyopathy (DCM).We have used quantitative proteomic approaches to define the molecular pathways mis-regulated in Tbx20 and Casz1 mice and thus, are associated with DCM. Using an unbiased systems-based screen, we demonstrate the cardiac transcription factor TBX20 physically and genetically interacts with the essential transcription factor CASZ1. This interaction is required for survival as mice heterozygous for both Tbx20 and Casz1 die post-natally as a result of dilated cardiomyopathy (DCM).We have used quantitative proteomic approaches to define the molecular pathways mis-regulated in Tbx20 and Casz1 mice and thus, are associated with DCM.

### Sample Protocol
Hearts from mutant and Nkx2.5Cre (control) mice were perfused, dissected, flash frozen in liquid nitrogen, and stored at -80ï‚°C. In parallel, control and mutant hearts (N = 2 each) were ground in a chilled mortar and pestle, and lysed in 4 mLs lysis buffer (150mM NaCl, 0.5% TritonX-100, 20mM K-HEPES, 110mM KOAc, 2mM MgCl2, 0.1% Tween20, 1uM ZnCl2, 1uM CaCl2, 1X protease inhibitors, 1X phosphatase inhibitors). Once resuspended, lysates were homogenized using the Kinematica PT1200E tissue homogenizer and cleared of cellular debris via centrifugation at 3K rpm at 4ï‚°C for 10 min. Cleared lysates were treated with 10U/mL RNAse A and DNAse for 30 min at room temperature. Following nuclease treatment, proteins were acetone-precipitated from tissue lysates and reconstituted in 7M urea. Heart protein lysates (75 ug) were simultaneously reduced and alkylated in 20 mM TCEP and 20 mM chloroacetamide, respectively, for 45 min at 37C. Proteins were then digested to peptides in-solution using a modified Filter-aided Sample Preparation (FASP) approach. Briefly, Amicon-500 ultrafiltration devices were used to buffer exchange reduced and alkylated lysates by sequential washing (1 x 0.4 mL, 2 x 0.3 mL) in 0.1M TrisHCl, pH 8.0, containing 8M urea, interleaved with centrifugation at 14,000 x g for 10 min at 20C. After the final centrifugation, 20 uL of LC-MS grade water containing 1.5 ug of LysC/trypsin (Promega) was added to the filter and incubated for 2 hrs at 37ï‚°C. Samples were diluted 1:4 with 20 mM TrisHCl, pH 8.0 and incubated for 4 hrs at 37C. The filters were centrifuged as above, then washed 2 x 50 uL with water. The filtrate containing tryptic peptides was recovered, desalted over StageTips (18), concentrated to dryness, and dissolved in 0.1M HEPES, pH 8.2, containing 20% ACN. The peptide concentration was determined by the colorimetric peptide BCA assay kit (Thermo Scientific Pierce).  For each sample, equivalent peptide amounts (30 ug) were labeled with 0.16 mg of TMT6plex tagging reagent. (TMT label 128 and 129 for control duplicates and 130 and 131 for mutant duplicates) for 1 hr at RT. The labeling was quenched with hydroxylamine (0.33%) for 10 min at RT. The TMT-labeled peptides were combined and separated by 2D StageTip fractionation. First, the peptides were separated into four fractions over C18-SCX StageTips with sequential elution (90 uL) in 0.05 M ammonium formate/20% acetonitrile (ACN), 0.05 M ammonium acetate/20% ACN, 0.05 M ammonium bicarbonate/20% ACN, and 0.1% ammonium hydroxide/20% ACN. Then, each fraction was diluted to 10% ACN, adjusted to 0.5% TFA, and further separated into three fractions over SDB-RPS StageTips with sequential elution (40 uL) in 0.2 M ammonium formate/0.5% formic acid (FA)/60% ACN, 0.2 M ammonium acetate/0.5% FA/60% ACN, and 5% ammonium hydroxide/80% ACN. The resulting 12 fractions were evaporated to near dryness and dissolved in 5 uL of 1% FA/5% ACN.Peptides (2 uL) were analyzed by nanoliquid chromatography-tandem mass spectrometry on a Dionex Ultimate 3000 UPLC coupled to an EASYSpray ion source and Q Exactive HF quadrupole orbitrap mass spectrometer. Briefly, TMT-labeled peptides were separated over a 2.5 hr linear gradient (5 to 40% solvent B; solvent A: 0.1% FA, solvent B: 0.1% FA/97% ACN) by reverse phase chromatography (EASYSpray C18 column, 75 um x 50 cm). As peptides eluted from the column, a data-dependent MS acquisition method was employed, which consisted of an analytical MS scan, measuring intact peptide ions, followed by up to 15 tandem MS fragmentation events (resolution: 30,000 at m/z = 200) on the most intense ions.

### Data Protocol
Tandem MS spectra collected across the 12 LC-MS/MS runs were analyzed by Proteome Discoverer (ver 2.1 SP1). In summary, a Processing workflow containing the SEQUEST algorithm was used to search tandem MS spectra against a UniProt-SwissProt mouse database and common contaminants (2016-07, 16,810 sequences), generating peptide spectrum matches. Percolator was used to calculate spectrum q-values based on reverse sequence database searches. For TMT-based quantitative analysis, the reporter ion quantifier module extracted TMT signal-to-noise values from MS/MS spectra. A Consensus workflow was employed which defined peptide and protein filters to limit the false discovery rate to ï‚£ 1%, then assembled peptides into protein groups with strict parsimony, and retained quantitative values from spectra that (1) were unique to the protein group, (2) had a co-isolation threshold of < 25%, and (3) had an average S/N of > 10. TMT S/N values were normalized between TMT reporter channels based on the total sum of the S/N values within each channel over all identified peptides. Protein abundances were calculated as the sum of normalized spectrum S/N values for each channel within each protein group. Protein groups and associated TMT abundances for each sample were exported to Excel and only those containing a minimum of 2 quantified unique peptides were considered for further analysis. The TMT abundance ratio for the experimental comparison (mutant versus control) was calculated as the abundance of each mutant replicate abundance versus the average abundance in the control (N = 2). The fold-change threshold for differential protein abundance was selected empirically based on 5-95% confidence interval limits of the TMT ratio distribution for biological replicates (e.g. mutant 1 vs. mutant 2), which provides an estimate of the biological and technical variability. Protein groups meeting this threshold (Â±1.3-fold) for both mutant replicate conditions were considered significant and analyzed by GO and pathway over-representation analysis.

### Publication Abstract
By the age of 40, one in five adults without symptoms of cardiovascular disease are at risk for developing congestive heart failure. Within this population, dilated cardiomyopathy (DCM) remains one of the leading causes of disease and death, with nearly half of cases genetically determined. Though genetic and high throughput sequencing-based approaches have identified sporadic and inherited mutations in a multitude of genes implicated in cardiomyopathy, how combinations of asymptomatic mutations lead to cardiac failure remains a mystery. Since a number of studies have implicated mutations of the transcription factor TBX20 in congenital heart diseases, we investigated the underlying mechanisms, using an unbiased systems-based screen to identify novel, cardiac-specific binding partners. We demonstrated that TBX20 physically and genetically interacts with the essential transcription factor CASZ1. This interaction is required for survival, as mice heterozygous for both Tbx20 and Casz1 die post-natally as a result of DCM. A Tbx20 mutation associated with human familial DCM sterically interferes with the TBX20-CASZ1 interaction and provides a physical basis for how this human mutation disrupts normal cardiac function. Finally, we employed quantitative proteomic analyses to define the molecular pathways mis-regulated upon disruption of this novel complex. Collectively, our proteomic, biochemical, genetic, and structural studies suggest that the physical interaction between TBX20 and CASZ1 is required for cardiac homeostasis, and further, that reduction or loss of this critical interaction leads to DCM. This work provides strong evidence that DCM can be inherited through a digenic mechanism.

### Keywords
Heart, Mouse, Dcm

### Affiliations
Molecular Biology, Princeton University
Department of Molecular Biology, Princeton University, USA

### Submitter
Todd Greco

### Lab Head
Dr Ileana Cristea
Department of Molecular Biology, Princeton University, USA


