### Accession
PXD017519

### Title
Palaeoproteomics gives new insight into southern african early pastoralism

### Description
The advent of domestication is a major step that transformed the subsistence strategies of past human societies. In Africa, domestic caprines (sheep and goat) were introduced in the north-eastern part of the continent from the Near East more than 9000 years ago. However, their diffusion southwards was slow. They were thought to have made their first appearance in the southern part of the continent ca. 2000 years ago, at a few Later Stone Age sites, including Leopard Cave (Erongo region, Namibia), which provided the oldest directly dated remains assigned to sheep or goat on the basis of morphology of bones and teeth. However, similarities in morphology between these two domesticated caprine species and between them and small wild antelopes, as well as the high fragmentation of the sites osteological remains which alter their initial morphology, raised questions about the species attribution. In this paper, we report species identification of the Leopard Cave remains using palaeoproteomics, a method that uses protein markers in bone and tooth collagen to achieve taxonomic identification of archaeological remains. We also report new direct radiocarbon dates. Wild antelope remains from museum collections were used to enrich the available protein record and propose de novo type I collagen sequences. Our results demonstrate that the remains morphologically described as domesticates actually belong to a wild antelope species and that domestic caprines first appeared at Leopard Cave 1500 years later than previously thought. This study illustrates that the use of palaeoproteomics coupled to direct radiocarbon dates is particularly suited to complement classic zooarchaeological studies, in this case concerning the arrival of the first herding practices in arid environments.

### Sample Protocol
All archaeological remains were sampled using a Dremel drill with a diamond bur that was rinsed with 96% EtOH between each sampling to avoid cross-contamination. A mean of 50 mg of bone or tooth powder was collected and chemically prepared following a modified version of the protocol previously published by our team for archaeological remains from African contexts49. In brief, samples (both modern and archaeological) were decalcified 815 days in a Tris-EDTA buffer (0.05 M and 0.5 M, final pH 7.4, Sigma Aldrich, St. Louis, Missouri, United States). Solutions were replaced every day and kept at 4C until further preparation (hereafter, decalcification solution, D). Once they were totally decalcified, pellets were rinsed with Milli-Q water until they were pH neutral. The remaining proteins were then resuspended in ammonium bicarbonate (ABC, 50 mM pH 8, Sigma Aldrich) for 3 h at 65C. After centrifugation, supernatants were collected (hereafter extract solutions, E) and placed into clean microtubes. D underwent buffer exchange from Tris-EDTA to ABC using 5 mL Vivaspin (3 kDa, Amicon Ultra, Merck Millipore, Burlington, Massachussetts, United States). Both D and E recovered proteins were reduced with dithiothreitol (DTT, final concentration of 10 mM, 20 min, 56C, 350 rpm), alkyled with iodoacetamide (10 mM, 30 min, in the dark, at room temperature) and subjected to trypsin hydrolysis (0.1 g/L, Trypsin Gold, Promega, Madison, Wisconsin, United States). Finally, D peptides were recovered, and solutions were desalted by solid phase extraction (SPE) using C8 cartridges (Sep-Pak C8 Plus Short 400 mg Sorbent, Waters, Milford, Massachussetts, United States). Protein concentrations were measured on a NanoVue instrument (GE HealthCare, Chicago, Illinois, United States), yielding amounts ranging between 0.54 and 1.48 g/L for archaeological samples and between 0.62 and 1.95 g/L for modern samples. Peptides in solutions were kept at 20C until further analysis.

### Data Protocol
The digested samples were analysed by ultra-high-performance-liquid-chromatography coupled to tandem mass spectrometry (UHPLC-MS/MS) following ref. 49. We used a mobile phase gradient composed of A: H2O + FA 0.1% and B: ACN + FA 0.08%, with a flow rate of 300 L/min. The Electrospray-Quadrupole-Time of Flight (ESI-Q-TOF) mass spectrometer (Maxis II ETD, Bruker Daltonics, Billerica, Massachusetts, United States) was used in positive mode on the m/z range 2002200 in data-dependent auto-MS/MS mode. MS/MS spectra were generated using collision-induced dissociation, with ion selection set to m/z 3002200 and charge states set to 1+ to 5+. Data were automatically calibrated with sodium formate injected at the beginning of each analysis. An ABC blank was performed every five analyses to avoid cross-samples contamination. The LC-MS/MS data were converted to .mgf format using DataAnalysis software (version 4.4, Bruker Daltonics). Sequences of the two wild antelopes species were reconstructed using the database-assisted de novo tool of PEAKS Studio software65(version 7.5, BioInformatics Solutions, Waterloo, Canada). The entire PEAKS workflow was set to trypsin enzymatic digestion, allowing 2 missed cleavages and including the following modifications: fixed carbamidomethylation (C) and variable oxidation (M), deamidation (NQ), hydroxylation (P) and phosphorylation (ST). Error tolerance was set to 10 ppm for precursors and 0.02 Da for fragment ions. Both SPIDER and PEAKS DB tools searches were used to propose new COL1A1 and COL1A2 sequences. Each amino acid substitution proposed by the software was verified manually by looking at the MS/MS spectra, using the reference sequence that allowed the best coverage percentage. The de novo scores were set to 5% threshold, FDR 5%, score of 50% and protein hit  20. Once confidently identified, the COL1A1 and COL1A2 were aligned using Geneious 66 software (version 11.0.3, Auckland, New Zealand; substitution matrix Blosum62, gap open penalty 12) to corresponding protein sequences of B. taurus (NM_001034039.2 and NM_174520.2), B. indicus (XM_019981159.1 and XM_019959690.1), B. mutus (XM_005890325.2 and XM_005909695), C. hircus (XM_018064893.1, XM_018064894.1 and XM_0056778936.3), O. aries (XM_015098715.1 and XM_004007726.3), O. aries musimon (XM_012127623.2, XM_012127624.2 and XM_012146567.2) and P. hodgosonii (XM_005964647.1 and XM_005985683.1). Archaeological sample digest data were searched against a restricted protein database using Mascot software (version 2.4.1, www.matrixscience.com, London, UK67). The database was assembled using National Center for Biotechnology Information (NCBI) collagens sequences (last updated December 9, 2018, 88530 sequences for 78817228 residues, all entries). The same set of parameters used for the PEAKS search was used for the Mascot search. Since the results did not allow us to propose clear species identification, they were analysed using ProteinScape software (Bruker, Mannheim, Germany) by implementing the new COL1A1 and COL1A2 sequences of springbok and impala previously de novo reconstructed to the bovid species type I collagen sequence databases.

### Publication Abstract
The advent of domestication is a major step that transformed the subsistence strategies of past human societies. In Africa, domestic caprines (sheep and goat) were introduced in the north-eastern part of the continent from the Near East more than 9000&#xa0;years ago. However, their diffusion southwards was slow. They are thought to have made their first appearance in the southern part of the continent ca. 2000&#xa0;years ago, at a few Later Stone Age sites, including Leopard Cave (Erongo region, Namibia), which provided the oldest directly dated remains assigned to sheep or goat on the basis of morphology of bones and teeth. However, similarities in morphology, not only between these two domesticated caprine species, but also between them and the small wild antelopes, raised questions about the morphological species attribution of these remains. Additionally, the high fragmentation of the site's osteological remains makes it difficult to achieve species-level taxonomic identification by comparative anatomy. In this paper, we report molecular species identification of the Leopard Cave remains using palaeoproteomics, a method that uses protein markers in bone and tooth collagen to achieve taxonomic identification&#xa0;of archaeological remains. We also report new direct radiocarbon dates. Wild antelope remains from museum collections were used to enrich the available protein record and propose de novo type I collagen sequences. Our results demonstrate that the remains morphologically described as domesticates actually belong to a wild antelope species and that domestic caprines first appeared at Leopard Cave 1500&#xa0;years later than previously thought. This study illustrates that the use of palaeoproteomics coupled with direct radiocarbon dates is particularly suited to complement classic zooarchaeological studies, in this case concerning the arrival of the first herding practices in arid environments.

### Keywords
Ancient proteins, Caprines, Radiocarbon dating, Wild antelopes, Namibia, Domestication, Leopard cave

### Affiliations
Mus√©um national d'Histoire naturelle
Museum national d'Histoire naturelle UMR 7245 MCAM 43 rue Cuvier CP 54 F-75005 Paris FRANCE

### Submitter
Louise Le Meillour

### Lab Head
Dr Severine Zirah
Museum national d'Histoire naturelle UMR 7245 MCAM 43 rue Cuvier CP 54 F-75005 Paris FRANCE


