### Accession
PXD014211

### Title
ClpP participates in stress response, biofilm formation, antimicrobials tolerance, and virulence of Enterococcus faecalis and its global substrate proteins were identified by mass spectrometer with Tandem Mass Tags (TMT) labeling

### Description
ClpP is important for bacterial growth and plays an indispensable role in cellular protein quality control systems by refolding or degrading damaged proteins, but the role of ClpP on Enterococcus faecalis is still obscure. A clpP deletion mutant (△clpP) was constructed in E. faecalis strain OG1RF in order to elucidate a more comprehensive picture of the effect of ClpP on E. faecalis. Then the global RNA level of genes was detected by RNA sequencing and the global abundance of proteins was determined by mass spectrometer with Tandem Mass Tags (TMT) labeling. We found the ΔclpP mutant strain showed impaired growth at 20℃ or 45℃, at 5% NaCl or 2 mM H2O2. The surviving bacteria of the ΔclpP mutant strain was decreased after exposure to the high concentration of linezolid or minocycline (100 x MIC) for 96 h compared with that of the parent strain. The ΔclpP mutant strain also demonstrated decreased biofilm formation, but increased virulence in a Galleria mellonella model. The abundances of heat shock proteins, regulatory protein Spx (spxA), heat-inducible transcription repressor HrcA, transcriptional regulator CtsR, ATPase/chaperone ClpC and chaperonin GroEL increased in the ΔclpP mutant strain. The FsrA response regulator and its regulating protein gelatinase GelE, acetyl esterase/lipase, adapter protein MecA and fructose-1,6-bisphosphatase (fbp) also up regulated in the ΔclpP mutant strain. However, the abundances of ribosomal protein L4/L1 family protein (rplD) , ribosomal protein L7/L12 (rplL2), 50S ribosomal protein L13 (rplM), L18 (rplR), L20 (rplT), 30S ribosomal protein S14 (rpsN2) and S18 (rpsR) all reduced in the ΔclpP mutant strain. The dihydroorotase (pyrC), orotate phosphoribosyltransferase (pyrE) and orotidine-5'-phosphate decarboxylase (pyrF) also down regulated in the ΔclpP mutant strain. In conclusion, these results suggest that ClpP participates in stress tolerance, biofilm formation, antimicrobials tolerance, and virulence of E. faecalis.

### Sample Protocol
The E. faecalis strains OG1RF and the ΔclpP mutant were inoculated into TSB and cultured at 37°C for 4 h to logarithmic phase or for 12 h to stationary phase. The cells were harvested at 4°C centrifugation and minced individually with liquid nitrogen and lysed in lysis buffer, followed by 5 min of ultrasonication on ice. Protein concentration was determined again by Bradford protein assay. The supernatant from each sample, containing precisely 0.1 mg of protein, was digested with Trypsin Gold (Promega) at 1:50 enzyme-to-substrate ratio. After 16 h of digestion at 37°C, peptides were desalted with C18 cartridge to remove the high urea, and desalted peptides were dried by vacuum centrifugation. Desalted peptides were labeled with TMT6/10-plex reagents (TMT6/10plex™ Isobaric Label Reagent Set, Thermo Fisher), following the manufacturer’s instructions. For 0.1 mg of peptide, 1 unit of labeling reagent was used. Peptides were dissolved in 100 μL of 0.1 M TEAB, and the labeling reagent was dissolved in 41 μL of acetonitrile. After incubation for 1 h, the reaction was stopped with ammonium hydroxide. Differently labeled peptides were mixed equally and then desalted by peptide desalting spin columns (Thermo Fisher, 89852). TMT-labeled peptide mix was fractionated using a C18 column (Waters BEH C18 4.6×250 mm, 5 μm) on a Rigol L3000 HPLC operating at 1 mL/min, the column oven was set as 50°C. Shotgun proteomics analyses were performed using an EASY-nLCTM 1200 UHPLC system (Thermo Fisher) coupled with an Orbitrap Q Exactive HF-X mass spectrometer (Thermo Fisher) operated in the data-dependent acquisition (DDA) mode. Q Exactive HF-X mass spectrometer was operated in positive polarity mode with spray voltage of 2.3 kV and capillary temperature of 320°C. Two independent experiments were performed.

### Data Protocol
The resulting spectra from each fraction were searched separately against NCBI E. faecalis strains OG1RF (CP002621.1) database https://www.ncbi.nlm.nih.gov/nuccore/CP002621.1) by the search engines: Proteome Discoverer 2.2 (PD 2.2, Thermo). The searched parameters as follows: A mass tolerance of 10 ppm for precursor ion scans and a mass tolerance of 0.02 Da for the product ion scans were used. Carbamidomethyl was specified in PD 2.2 as fixed modifications. Oxidation of methionine, acetylation of the N-terminus and TMT of lysine were specified in PD 2.2 as variable modifications. A maximum of 2 miscleavage sites were allowed. For protein identification, protein with at least 1 unique peptide was identified at FDR less than 1.0% on peptide and protein level, respectively. Proteins containing similar peptides and could not be distinguished based on MS/MS analysis were grouped separately as protein groups. Reporter Quantification (TMT) was used for TMT quantification. The protein quantitation results were statistically analyzed by Mann-Whitney Test, the significant ratios, defined as p < 0.05 and ratio > 1.2 or ratio < 0.83 [fold change, FC], were used to screen the differential abundance proteins (DAPs). Gene Ontology (GO) and InterPro (IPR) analysis were conducted using the interproscan-5 program against the non-redundant protein database (including Pfam, PRINTS, ProDom, SMART, ProSiteProfiles, PANTHER), and the databases COG (Clusters of Orthologous Groups) and KEGG (Kyoto Encyclopedia of Genes and Genomes) were used to analyze the protein family and pathway. The probable interacting partners were predicted using the STRING-db server (http://string.embl.de/) based on the related species. STRING is a database of both known and predicted protein-protein interactions. The enrichment pipeline was used to perform the enrichment analysis of GO, IPR and KEGG, respectively.

### Publication Abstract
None

### Keywords
Enterococcus faecalis; clpp; stress tolerance; biofilm formation; virulence

### Affiliations
Key Laboratory of Medical Molecular Virology of Ministries of Education and Health, School of Basic Medical Science and Institutes of Biomedical Sciences, Shanghai Medical College of Fudan University
Department of Infectious Diseases and the Key Lab of Endogenous Infection, Shenzhen Nanshan People's Hospital and The 6th Affiliated Hospital of Shenzhen University Health Science Center, Shenzhen 518052, China

### Submitter
Jin-xin Zheng

### Lab Head
Dr Zhi-jian Yu
Department of Infectious Diseases and the Key Lab of Endogenous Infection, Shenzhen Nanshan People's Hospital and The 6th Affiliated Hospital of Shenzhen University Health Science Center, Shenzhen 518052, China


