### Accession
PXD011559

### Title
DDA experiments Emiliania huxleyi exposure to 3 doses of HHQ

### Description
Emiliania huxleyi: Cellular cascades induced by bacterial algicides Interactions between phytoplankton and bacteria play a central role in mediating oceanic biogeochemical cycling and microbial trophic structure in the ocean. The intricate relationships between these two domains of life are mediated via excreted molecules that facilitate communication and determine competitive outcomes. Yet, despite their predicted importance, identifying these secreted compounds and understanding their ecological significance has remained a challenge. Research in the Whalen Lab endeavors to (i) identify those bacterially-derived chemical signaling compounds (i.e. infochemicals) that mediate phytoplankton population dynamics, and (ii) determine the underlying physiological processes that contribute to phytoplankton tolerance or susceptibility to these compounds. Recently, the Whalen lab isolated an alkylquinolone-signaling molecule with known quorum sensing function from the globally distributed marine Î³-proteobacteria, Pseudoalteromonas sp. capable of inducing species-specific phytoplankton mortality. This research was the first to suggest quorum sensing compounds have expanded and previously unrecognized ecological roles in regulating primary production and phytoplankton bloom dynamics. We are now investigating in how this alkylquinolone induces phytoplankton mortality via transcriptomic profiling and diagnostic biochemical analysis. Complementary to this transcriptomic examination, we will complete whole-cell proteomic approach to identify those phytoplankton proteins crucial in competitive interactions with bacterial infochemicals, but whose functions may not yet be known. With this proteomic approach in parallel to our transcriptomic investigation, we can establish a better understanding of the eukaryotic macromolecular targets and cellular cascades induced in response to bacterial algicides like alkylquinolones. With the knowledge gained from both approaches we can begin to address how these ?keystone molecules? influence population dynamics and community composition of phytoplankton and bacteria in field-based experiments with the goal of defining a new mechanistic framework for how bacterially derived signaling molecules influence biogeochemical cycles. D= DMSO - control treatment L= low 1 nm HHG additions M= medium 10 nm HHG additions H= high 100 nm HHG additions Each treatment had 4 biological replicates A-D

### Sample Protocol
Ehux cells were grown to 72 hours in each treatment and 4 biologial replicates (from individual culture bottles were isolated for proteomics.  Cells were pelleted using centrifugation and lysed with a sonicating probe.  Proteins were reduced, alkylated and digested with trypsin after being soubilized in 6M urea (see Nunn eta l. 2016). All samples were desalted using NEST group desalting columns. Mass spectrometry was completed on Thermo Lumos with NanoAquity inline.  Column was 75 micorn ID, packed to 30cm with Dr. Maisch C18 packing material All DDA analyses were completed in single runs on the Lumos. 130 minute total run time, gradient 2-35% ACN in 90 minutes. OT resolution 120,000, scan range 375-1575, Rf lens 35%, AGC Target 7e5 MS2: charge state 2-5, Top 20 MS2, AGC targert 5e3, dynamic exclusion 30 s

### Data Protocol
The database for Emiliania Hyxleyi was downloaded 10.10.2018 from Uniprot and concatenated with contaminants. Data was searched using Comet and protein inferences were completed with the TPP to generate pep.xml and prot.xml files.

### Publication Abstract
None

### Keywords
Quorum sensing plankton and bacteria communication, Dda phytoplankton

### Affiliations
University of Washington Department of Genome Sciences
University of Washington

### Submitter
Brook Nunn

### Lab Head
Dr Brook L. Nunn
University of Washington Department of Genome Sciences


