### Accession
PXD033958

### Title
Life-time study of premodified histone for Identification of chromatinized proteome defined by site-specific histone acylations in living cells

### Description
C13, N15 isotope labelled heavy lysine bearing the modification group, such as cr and bhb, were site-sepcifically incorporated into histone H3K56 in living cells for measurement of the modification life of H3K56cr and H3K56bhb without iinterference caused by endogenous histone modification. Utilizaing mass spectrometry, we compared with H3K56cr/bhb peptide with the H3K79 peptide as internal standard at different times. The quantitation analysis of H3K56cr and H3K56bhb at different time points can be comparative.

### Sample Protocol
Plasmids pcDNA4-H3-K56TAG (C-terminal myc-His-tag) was transfected into the relative PylRS stable cell line and supplemented with 0.2 mM Khcr/Khbhb. After growing for 6 h, the transfected HEK293T cells were changed into DMEM with 1mM Khcr/Khbhb. The cells were cultured for additional 24, 36, or 48 hours to express the corresponding H3 protein (H3Kh56cr/H3Kh56bhb). The medium was discarded and Dulbeccoâ€™s phosphate buffered saline (DPBS) was added. After ultraviolet irradiation of the cells on ice for 15 min, cells were harvested. The histones were extracted by TCA precipitation and enriched by his-tag isolation. The eluents were combined, concentrated, and separated by SDS-PAGE. The expressed H3Kh56cr/H3Kh56bhb bands were cut for subjected to HPLC-MS/MS proteomic study.

### Data Protocol
Quantification was analyzed by pFind. The raw files were searched against the expressed histone H3.1 database (1 sequences) by pFind (version 2.8.8, download from http://pfind.org/software/pFind/index.html), with the following parameters: The carbamidomethyl (Cys) was set as the fixed modification, while oxidation (Met), acetyl (protein N-terminal), Kac, K/Rme, Khcr/Khbhb was set as variable modification. +/- 10 ppm was set for peptide pass tolerance and +/- 0.6 Da was set for fragment mass tolerance. A maximum of two missed cleavages was allowed for trypsin digestion. FDR was determined by searching a reverse database and was set to 0.01 for proteins and peptides. Proteins with spectral counts no less than 2 were recognized as possible candidates after  pFind searching. The final ratio of the H3K56hcr/bhb peptide at different time points was calculated based on the weighted average of the peak area compared with that of H3K79 peptide at the same time point.

### Publication Abstract
A generalizable strategy with programmable site specificity for in situ profiling of histone modifications on unperturbed chromatin remains highly desirable but challenging. We herein developed a single-site-resolved multi-omics (SiTomics) strategy for systematic mapping of dynamic modifications and subsequent profiling of chromatinized proteome and genome defined by specific chromatin acylations in living cells. By leveraging the genetic code expansion strategy, our SiTomics toolkit revealed distinct crotonylation (e.g., H3K56cr) and &#x3b2;-hydroxybutyrylation (e.g., H3K56bhb) upon short chain fatty acids stimulation and established linkages for chromatin acylation mark-defined proteome, genome, and functions. This led to the identification of GLYR1 as a distinct interacting protein in modulating H3K56cr's gene body localization as well as the discovery of an elevated super-enhancer repertoire underlying bhb-mediated chromatin modulations. SiTomics offers a platform technology for elucidating the "metabolites-modification-regulation" axis, which is widely applicable for multi-omics profiling and functional dissection of modifications beyond acylations and proteins beyond histones.

### Keywords
Heavy lysine, Life time, Bhb, H3k56, Cr

### Affiliations
Peking University
Professor and Chair of Chemical Biology, Peking-Tsinghua Center for Life Sciences Peking University, Beijing, China. Executive Editor, ACS Chemical Biology

### Submitter
Fangfei Qin

### Lab Head
Dr Peng R. Chen
Professor and Chair of Chemical Biology, Peking-Tsinghua Center for Life Sciences Peking University, Beijing, China. Executive Editor, ACS Chemical Biology


