### Accession
PXD001910

### Title
Subcellular organization of single-cell C4 photosynthesis in Bienertia sinuspersici determined by large scale proteomics

### Description
Bienertia sinuspersici performs C4 photosynthesis without Kranz anatomy through subcellular compartmentalization of carbon fixation within individual cells. In this species, central compartment chloroplasts (C) and peripheral chloroplasts (P) collaborate with cytosolic and mitochondrial components in a NAD-ME type C4 cycle. How the two functionally different chloroplast types can develop within individual cells and the mechanism of import of nuclear encoded, plastid targeted proteins are currently unknown. We used 454 sequencing in combination with large scale label-free proteomics to determine the distribution of photosynthesis-related proteins. Subcellular localization of 169 protein was determined through comparison of protein abundance in four different subcellular fractions. 39 out of the 120 chloroplastic proteins showed differential accumulation between the two chloroplast types. Rubisco, RPP regenerative phase and PSII related proteins accumulated in C chloroplasts whereas C4 related proteins and the NDH complex were more abundant in P chloroplasts. Comparison of transit peptides of differential accumulating proteins indicated no obvious sequence homology or similarities in physico-chemical properties between members of the same group. Protein composition analysis of the central compartment indicated that mitochondria and peroxisomes are the only major components besides chloroplasts in this compartment. The combined information from subcellular and developmental protein profiling was used to generate a first draft of the protein machinery involved in single-cell C4 photosynthesis.

### Sample Protocol
For the developmental samples, 20 leaves of each developmental stage (youngest 0.3-1 mm in size, young 1-3 mm in size and mature approx. 2 cm in size) were harvested for each replicate from mature Bienertia plants. The frozen plant material was resuspended in extraction buffer (2% (w/v) SDS, 10% (w/v) glycerol, 62.5 mM Tris-HCl, pH 6.8, and 0.715 M 2-mercaptoethanol) and immediately boiled for 5 min. 50 Âµg of total protein extract from the 4 different localization samples and three different developmental samples were separated on Novex Tricine 10-20% PAA gels (Invitrogen) in two independent biological replicates. Individual gel lanes were cut into 10 (localization series) or 8 (developmental series) slices and proteins were digested with trypsin. Extracted peptides were analyzed by nano LC-LTQ-Orbitrap MS using data-dependent acquisition and dynamic exclusion.

### Data Protocol
Peak lists (.mgf format) were generated using DTA supercharge (version 1.19) software (http://msquant.sourceforge.net/) and searched with Mascot version 2.2 (Matrix Science) against the generated Bienertia databases. For off-line calibration, first a preliminary search was conducted with the precursor tolerance window set at 30 ppm. Peptides with ion scores above 40 were chosen as benchmarks to determine the offset for each LC-MS/MS run. This offset was then applied to adjust precursor masses in the peak lists of the respective .mgf file for recalibration using a Perl script. The recalibrated peak lists were searched against the Bienertia_Prandom rand DB.  Each of the peak lists were searched using Mascot version 2.2 (maximum P of 0.01) for full tryptic peptides using a precursor ion tolerance window set at 6 ppm, variable Met oxidation, fixed Cys carbamidomethylation, 1 miss cleavage allowed, and for semi-tryptic peptides using a precursor ion tolerance window set at 3 ppm, variable Met oxidation, fixed Cys carbamidomethylation, Glutamine to pyro-glutamic acid (N-term Q) modification,1 miss cleavage allowed and a minimal mascot ion score threshold of 30. This yielded a peptide false discovery rate below 1%, with peptide false positive rate calculated as 2 (decoy_hits/total_hits). The false protein identification rate of protein identified with two or more peptides was zero. To reduce the false protein identification rate of proteins identified by one peptide, the Mascot search results were further filtered as follows: ion score threshold was increased to 40 for one peptide identification and mass accuracy on the precursor ion was required to be within 3 ppm. Precursor ion masses below 700 D were discarded. All filtered results were uploaded into PPDB (http:// ppdb.tc.cornell.edu/; Sun et al., 2009). Detailed information of all identified peptides and protein sequences are accessible through PPDB (http:// ppdb.tc.cornell.edu) by using the unigene identifiers from Supplemental Table 1 as query in the "Gene Name & Annotations" section.  Calculation of adjSPC, NadjSPC, relative abundance and protein ratios and grouping of proteins with a high percentage of shared matched spectra. For protein quantification, spectral counts (SPC) were adjusted for the number of unique peptides vs. shared peptides between similar proteins resulting in an adjusted spectral count (adjSPC) as described in detail in (Friso et al., 2010). Proteins that shared more than approximately 80% of their matched adjusted peptides with other proteins were grouped using a similarity matrix. Normalized adjusted spectral counts (NadjSPC) used for the calculation of C/P as well as CC/TP ratios were calculated from adjSPC through division by the sum of all adjSPC in the corresponding sample.   Quantitative comparison of spectral count data was performed using the beta-binomial test (Pham et al., 2010) in the statistical software package "R" (Team, 2013) with the script provided by the authors. Tests were performed on adjSPC values since the beta-binomial test naturally normalized for the total number of spectral counts in each sample. All derived p-values were further corrected for multiple hypothesis testing using a graphically sharpened method. Adjusted p-values (a.k.a. q-values) smaller than 0.1 were interpreted with indication for enrichment or were considered to be significantly enriched (<0.05). For comparison of transit peptide features, Student's t-test was performed either on the direct data or in the case of index data after transformation to the arcsine.

### Publication Abstract
Kranz C4 species strictly depend on separation of primary and secondary carbon fixation reactions in different cell types. In contrast, the single-cell C4 (SCC4) species Bienertia sinuspersici utilizes intracellular compartmentation including two physiologically and biochemically different chloroplast types; however, information on identity, localization, and induction of proteins required for this SCC4 system is currently very limited. In this study, we determined the distribution of photosynthesis-related proteins and the induction of the C4 system during development by label-free proteomics of subcellular fractions and leaves of different developmental stages. This was enabled by inferring a protein sequence database from 454 sequencing of Bienertia cDNAs. Large-scale proteome rearrangements were observed as C4 photosynthesis developed during leaf maturation. The proteomes of the two chloroplasts are different with differential accumulation of linear and cyclic electron transport components, primary and secondary carbon fixation reactions, and a triose-phosphate shuttle that is shared between the two chloroplast types. This differential protein distribution pattern suggests the presence of a mRNA or protein-sorting mechanism for nuclear-encoded, chloroplast-targeted proteins in SCC4 species. The combined information was used to provide a comprehensive model for NAD-ME type carbon fixation in SCC4 species.

### Keywords
Single-cell c4 photosynthesis, Bienertia sinuspersici, Subcellular organization, Shotgun proteomics

### Affiliations
Plant Biology
Cornell University

### Submitter
Giulia Friso

### Lab Head
Dr Giulia Friso c/o Klaas van Wijk's Lab
Cornell University


