### Accession
PXD016273

### Title
Comparative evaluation of different sample preparation workflows for reproducible, quantitative, and in-depth analysis of urine proteomics

### Description
The growing field of urinary proteomics has provided a promising opportunity to identify biomarkers useful for diagnosis and prognostication of a number of diseases. Urine is abundant and readily collected in a non-invasive manner which makes it eminently available for testing, however sample preparation for proteomics analysis remains one of the biggest challenges in this field.  As newer technologies to analyze tandem mass spectrometry (MS) data develop, utility of urinary proteomics would be enhanced by better sample processing and separation workflows to generate fast, reproducible, and more in-depth proteomics data. In this study, we have evaluated the performance of four sample preparation methods: MStern, PreOmics In-StageTip (iST), Suspension-trapping (S-Trap), and conventional urea In-Solution trypsin hydrolysis for non-depleted urine samples. Data Dependent Acquisition (DDA) mode on QE-HF was used for single-shot label-free data acquisition. Our results demonstrate a high degree of reproducibility within each workflow. PreOmics iST yields the best digestion efficiency with lowest percentage of missed-cleavage peptides. S-trap workflow gave the greatest number of peptide and protein identifications.  Using S-trap method, with 0.5mL urine sample as starting material, we identify ~1500 protein groups and ~17700 peptides from DDA analysis with a single injection. The continued refinement  of sample preparation (presented here), LC separation methods and  mass spectrometry data acquisition can allow the integration of information rich urine proteomic records across large cohorts with other ‘big data’ initiatives becoming more popular in medical research.

### Sample Protocol
Urine Sample Preparation  Pediatric Urine samples from 5 anonymous healthy donors were provided by the Urology Department of the Children’s Hospital of Philadelphia (CHOP). The urine samples were thawed quickly at 37°C and centrifuged at 1000 g for 5 min. Equal volumes of urine from the individual donors were mixed together and stored at -80°C in 0.5 mL aliquots.  SWAG assays were used to determine each aliquot contained ~100 ug protein.  Four different methods including MStern, In-Solution, S-trap and iST were applied for this study. Each method was applied to 4 technical replicates. The 0.5 mL urine aliquots were thawed quickly and 2 mL cold acetone was added to precipitate proteins overnight @ -20C. Subsequently, the samples were centrifuged for 20,000 g for 10min and urine protein pellets were separated.   In-Solution  The urine pellet was dissolved in 100uL 8M urea/200mM Tris-HCL pH 8.5. The dissolved samples were reduced with 5mM DTT for 30min and alkylated with 20mM IAA in dark for 30min. The 100uL urine in 8M urea/200mM Tris-HCl were diluted with 400 uL water before tryspin/LysC digestion (enzyme: protein ratio is 1:50) for overnight at 27°C. The tryptic peptides were acidified with formic acid at a final concentration of 1%. The peptides were centrifuged at 18,000 g and the supernatant was kept for desalting.  MStern  The Mstern method was based on a published method with minor modifications. The urine pellet was dissolved, reduced and alkylated as described above. Samples were then diluted with water to reduce the urea concentration to 2M. The 96-well plate PVDF membranes were dampened with 70% ETOH under vacuum and equilibrated with 2M urea/50mM Tris-HCl pH 8.5.  1/3 of the diluted urine sample was transferred to the PVDF membrane. The membrane was washed with 50mM ammonium carbonate (AB).  The digestion buffer (1ug trypsin and LysC in 5% acetonitrile and 50mM AB) were added to the membrane. The 96 well plate were incubated at 27°C for 2 hrs in a humidified chamber. The tryptic peptides from the membrane were eluted with 40% acetonitrile/0.1% TFA. The eluted peptides were vacuum dried and dissolved in water/0.1% TFA before desalting. Preomics iST  Urine samples were processed using all reagents and instructions supplied by the manufacturer. In brief, the pellet was solubilized in 150uL of ‘Lyse’ buffer (containing Tris-HCl, SDC, 0.1% SDS, TCEP and 2-chloroacetamide) and heated at 95C for 10 minutes.  50uL of resulting denatured, reduced and alkylated solution was transferred to the reaction vessel.  Enzyme (LysC and trypsin) was added and samples were hydrolyzed at 37C for two hours.  The resulting peptide mixture was washed and eluted as per the manufacturer’s instructions. The eluted peptides were vacuum dried and dissolved in water/0.1% TFA before a second desalting. (5)  S-Trap Protein digestion in the S-trap filter was performed following the manufacturer’s instructions with slight modification (Protifi, USA).  Briefly, the protein pellet was dissolved in 100uL 5% SDS, reduced with 5mM DDT, and held for 5 min at 95 °C in a thermomixer. The sample was cooled to room temperature and centrifuged at 18,000 g for  5 min, followed by alkylation with 20 mM IAA for 20min at 37 °C in dark. The lysate was acidified with 12% phosphoric acid at 1:10 (acid: protein volume ratio). 700 uL of S-trap membrane binding and washing buffer (SBW) (90% methanol; 100 mM Triethylammonium bicarbonate, TEAB) was added to the solution and mixed well via pipetting. The resulting protein suspension was transferred to the S-trap filter via centrifugation at 2800 g for 1min.  The filter was washed 4 times with 600 uL SBW at 2500 g for 1min. Trypsin and LysC (1:25, enzyme: protein) in 150uL 50mM TEAB was added to the surface of the filter, and incubated for 3 hrs at 37°C.  The tryptic peptides were eluted via centrifugation for 1 min at 1000 g sequentially with 80uL TEAB, 80 uL water/0.2%TFA, 80 uL 50% acetonitrile/0.2% TFA. The eluted peptides were vacuum dried and dissolved in water/0.1% TFA before desalting. (6). Peptide Desalting To normalize prepared sample conditions across all workflows before LC MS analysis, a 10mg 96 well Oasis HLB plate (Waters, catalog no. 186000128) was used for desalting the tryptic peptides.  The plate was conditioned with 200 uL acetonitrile, equilibrated with 2 x 200 uL water/0.1% TFA. The acidified tryptic peptides were loaded to the plate, and washed three times with 200 uL water/0.1% TFA. The peptides were eluted with 3 x 80 uL 80% acetonitrile/0.1% TFA.  The eluates were dried overnight, then dissolved in water/0.1 % TFA including iRT peptides (Biognosys, catalog no. Ki-3003-2). The peptide input amount crossing all methods for LCMS analysis were normalized corresponding to the urine input amount. Peptides were aliquoted in the auto sampler vials, sonicated for 5 min, and briefly spun before MS analysis.

### Data Protocol
Raw MS files were processed using MaxQuant, version 1.6.1.0 for identification/quantification of peptides and proteins (14). The peptide MS/MS spectra were searched against the UniProtKB/Swiss-Prot Human Reference Proteome database. Full tryptic specific digest was selected with a maximum of two missed tryptic cleavage sites. Oxidation of methionine and acetylation of the protein N-terminus were used as variable modifications while Carbamidomethylation of cysteine was set as a fixed modification. The minimal length required for a peptide was seven amino acids. Fragment ion tolerance was set to 0.5 Da. The MS/MS tolerance was set at 20 ppm. A maximum FDR of 1% at both the peptide and the protein level was allowed. The rest of the parameters were kept as default.  The generated results by MaxQuant was further processed by PTXQC for visualization and quality control analysis. Perseus was used for statistical and bioinformatics analysis.

### Publication Abstract
The growing field of urinary proteomics shows promise to expand the number of biomarkers for the diagnosis and prognosis of a number of human diseases. With the rapid developments in mass spectrometry methods for proteome quantification, there exists an opportunity for improved sample processing and separation workflows to make important contributions to urine proteomic analyses. Here we evaluate the performance of four sample preparation methods: MStern, PreOmics in-StageTip (iST), suspension-trapping (S-Trap), and conventional urea In-Solution trypsin hydrolysis for nondepleted urine samples. Data-dependent acquisition (DDA) mode on a QExactive HF mass spectrometer was used for single-shot label-free data acquisition. Our results demonstrate a high degree of reproducibility within each workflow. PreOmics iST yields the best digestion efficiency, whereas the S-Trap workflow gives the greatest number of peptide and protein identifications. Using the S-Trap method and starting with &#x223c;0.5 mL, we identify &#x223c;1500 protein groups and &#x223c;17&#x202f;700 peptides from DDA analysis with a single injection on the mass spectrometer.

### Keywords
Preomics ist, Sample preparation, Urine proteomics, Reproducibility, S-trap, Label-free, Mass spectrometry, Mstren

### Affiliations
Director of CHOP Proteomics Core Facility
Children's Hospital of Philadelphia

### Submitter
Hossein Fazelinia

### Lab Head
Dr Steven H. Seeholzer
Director of CHOP Proteomics Core Facility


