### Accession
PXD018581

### Title
Proteomics of SARS-CoV and SARS-CoV-2 infected cells

### Description
The pandemic of the novel Coronavirus SARS-CoV-2, cause of Covid-19, has caused tens of thousands of deaths since its emergence in late 2019. An in-depth knowledge of the molecular biology of the virus infection is key for a better understanding of the virus and the progression of the disease. We here provide analysis of changes in proteome and phosphoproteome of H1299 cells untreated or treated with SARS-CoV or SARS-CoV-2 on a time course (4, 12, 24, 36 hrs).

### Sample Protocol
Treated and untreated cells were Lysed in 8M Urea buffer, reduced with 5mM DTT, alkylated with iodoacetamide and digested with LysC and Trypsin. After C18 clean-up (via STAGE-tip protocol), samples were labelled using TMTpro kit: the first 13 channels were dedicated to the samples while the last channel was used as booster channel, consisting of a mixture of the samples and of untreated cells. The samples were then mixed in equal amounts, combined with the booster channel and offline fractionated using a high pH fractionation (5mM ammonium formate) from 2 to 55% acetonitrile on a 96 minute gradient (Agilent). The fractions were combined into 29 fractions: first 12 fractions together (referred to as frA), other fractions were combined with the 29th, 57th and 85th following fraction. Approximately 500ng for each fraction was injected and online fractionated on a Proxeon nLC1200 (Thermo) on a 130 minutes gradient from 2 to 55% acetonitrile with a final clean-up step to 81% ACN and injected into a Q-exactive HF-x (Thermo) mass spectrometer. The MS was operated with the following parameters: MS1 resolution 60000, MS2 resolution 45000, normalized collision energy 30, isolation window 0.7 m/z. The offline fractionated samples were further combined into 5 fractions and subjected to IMAC enrichment via the AssayMAP Bravo system (Agilent). The resulting flow-through from the first IMAC enrichment was further combined into 3 fractions and subjected to another IMAC enrichment with the same system. IMAC enriched samples were online fractionated and injected into a mass spectrometer with a gradient as described above.

### Data Protocol
RAW files were analysed using MaxQuant version 1.6.10.43 with custom settings for allowing quantification with TMTpro. TMTpro correction factors were provided to MQ for automatic correction of reported ion spillage. Peptides were searched against the human uniprot database (Jan 2020), the uniprot SARS-CoV database (Mar 2020) and a proteogenomic version of the SARS-CoV-2 uniprot database modified according to RNA-seq data of the same samples; the same databases were used to build a semi-reverse database for FDR estimation. The search inclueded as fixed modifications carbamidomethylation of Cys and as variable modifications oxidation of Met, Acetylation of N-term and deamidation of Asn and Gln.  The resulting proteingroups.txt was filtered for protein contaminants, proteins only identified in the reverse database, proteins identified only by modified site, proteins identified with less that 2 peptides of which 1 unique and proteins with no missing values. Resulting corrected reporter ion intensities were log transformed and median mad normalized and converted to ratios against controls. All comparisons between samples were performed using two samples Student t-tests using as filter a Benjamini-Hochberg corrected p-value lower that 0.1.

### Publication Abstract
None

### Keywords
Sars-cov-2, Phosphoproteomics, Covid19, Tmt, Sars-cov

### Affiliations
Max Delbr√ºck Center for Molecular Medicine - MDC
Proteome Dynamics, MDC, Germany

### Submitter
Tommaso Mari

### Lab Head
Dr Matthias Selbach
Proteome Dynamics, MDC, Germany


### SDRF
- organism: Homo sapiens
- organism part: lung
- disease: non-small cell lung cancer
- cell type: not available
- label: TMT126, TMT127N, TMT127C, TMT128N, TMT128C, TMT129N, TMT129C, TMT130N, TMT130C, TMT131N, TMT131C, TMT132N, TMT132C
- instrument: Q Exactive HF-X
- modification parameters: Carbamidomethyl
- modification parameters: Oxidation
- modification parameters: Acetyl
- modification parameters: Deamidated

