### Accession
PXD032744

### Title
Independent human mesenchymal stromal cell-derived extracellular vesicle preparations differentially attenuate symptoms in an advanced murine Graft-versus-Host-Disease model

### Description
Extracellular vesicles (EVs) harvested from conditioned media of human mesenchymal stromal cells (MSCs) suppress acute inflammation in various disease models and promote regeneration of damaged tissues. Following successful treatment of an acute steroid-refractory Graft-versus-Host disease (GvHD) patient with EVs prepared from conditioned media of human bone marrow-derived MSCs, we focus on improving the MSC-EV production for the clinical application. Independent MSC-EV preparations all produced according to a standardized procedure, reveal broad immunomodulatory differences. Only a proportion of our MSC-EV products effectively modulate immune responses in a multi-donor mixed lymphocyte reaction (mdMLR) assay. To explore the relevance of such differences, we have established an optimized mouse GvHD model. The functional testing of selected MSC-EV preparations demonstrate that MSC-EV preparations revealing immunomodulatory capabilities in the mdMLR assay also effectively suppress GvHD symptoms in this model. In contrast, MSC-EV preparations, lacking such in vitro activities, also fail to modulate GvHD symptoms in vivo. Searching for differences of the active and inactive MSC-EV preparations, we failed to identify concrete proteins or miRNAs that could serve as surrogate markers. Thus, standardized MSC-EV production strategies may not be sufficient to warrant manufacturing of MSC-EV products with reproducible qualities. Consequently, given this functional heterogeneity, every individual MSC-EV preparation considered for the clinical application should be evaluated for its therapeutic potency prior to administration to patients. Here, we qualified the mdMLR assay for such analyses.

### Sample Protocol
For proteomic profiling MSC-EV sample volumes containing 5 Âµg of protein were used and subjected to in-solution tryptic digestions using a modified version of the Single-Pot Solid-Phase-enhanced Sample Preparation (SP3) protocol (Hughes et al., 2014; Moggridge et al., 2018). 1% SDS-containing lysates were added to Sera-Mag Beads (Thermo Fisher Scientific) in 10 ÂµL 15% formic acid and 30 ÂµL of ethanol. Binding of proteins was achieved by shaking for 15 min at room temperature. SDS was removed by four subsequent washes with 200 ÂµL of 70% ethanol. Proteins were digested overnight at room temperature with 0.4 Âµg of sequencing grade modified trypsin (Promega, Walldorf, Germany) in 40 ÂµL HEPES/NaOH, pH 8.4 in the presence of 1.25 mM TCEP and 5 mM chloroacetamide (Sigma-Aldrich). Beads were separated, washed with 10 ÂµL of an aqueous solution of 2% DMSO, and the combined eluates were dried down. Peptides were reconstituted in 10 ÂµL of H2O and reacted for 1 h at room temperature with TMT16pro labelling reagent (Thermo Fisher Scientific). To this end, 50 Âµg of TMT16pro label reagent were dissolved in 4 ÂµL of acetonitrile and added to the peptides. Excess TMT reagent was quenched by the addition of 4 ÂµL of an aqueous 5% hydroxylamine solution (Sigma-Aldrich). Peptides were reconstituted in 0.1 % formic acid and equal volumes were mixed. Mixed peptides were purified by a reverse phase clean-up step (OASIS HLB 96-well ÂµElution Plate, Waters GmbH, Eschborn, Germany). Peptides were subjected to an off-line fractionation under high pH conditions (Hughes et al., 2014). The resulting 12 fractions were then analysed by LC-MS/MS on an Q Exactive Plus mass spectrometer (Thermo Fisher Scientific). Briefly, peptides were separated using an UltiMate 3000 RSLC (Thermo Fisher Scientific) equipped with a trapping cartridge (Precolumn; C18 PepMap 100, 5 lm, 300 lm i.d. Ã— 5 mm, 100 AÂ°) and an analytical column (Waters nanoEase HSS C18 T3, 75 lm Ã— 25 cm, 1.8 lm, 100 AÂ°). Solvent A: aqueous 0.1% formic acid and 3% DMSO; Solvent B: 0.1% formic acid and 3% DMSO in acetonitrile (all solvents were of LC-MS grade). Peptides were loaded on the trapping cartridge using solvent A for 4 min with a flow of 30 ÂµL/min. Peptides were separated on the analytical column with a constant flow of 0.3 ÂµL/min applying a 1 h gradient of 2-80% of solvent B in A. Peptides were directly analysed in positive ion mode applying with a spray voltage of 2.2 kV and a capillary temperature of 275Â°C using a Nanospray-Flex ion source and a Pico-Tip Emitter 360 lm OD Ã— 20 lm ID; 10 lm tip (New Objective). MS spectra with a mass range of 375-1.200 m/z were acquired in profile mode using a resolution of 70.000 [maximum fill time of 250 ms or a maximum of 3e5 ions (automatic gain control, AGC)]. Fragmentation was triggered for the top 10 peaks with charge 2-4 on the MS scan (data-dependent acquisition) with a 30 s dynamic exclusion window (normalized collision energy was 30). Precursors were isolated with a 0.7 m/z window, and MS/MS spectra were acquired in profile mode with a resolution of 35,000 (maximum fill time of 120 ms or an AGC target of 2e5 ions).

### Data Protocol
For data analysis, protein output file of IsobarQuant (protein.txt â€“ files) was processed using the R programming language (ISBN 3-900051-07-0). Only proteins that were quantified with at least two unique peptides were considered for the analysis. The raw TMT reporter ion intensities (signal_sum columns) were first cleaned for batch effects using limma (Ritchie et al., 2015), and further normalized using variance stabilization normalization (Huber et al., 2002). Proteins were tested for differential expression using the limma package. The replicate information was added as a factor in the design matrix given as an argument to the â€˜lmFitâ€™ function of limma. A protein was annotated as a hit with a false discovery rate (fdr) smaller 5% and a fold-change of at least 100% (ratio of 2) and as a candidate with a fdr below 20% and a fold-change of at least 50 % (ratio of 1.5) (Metsalu and Vilo, 2015).

### Publication Abstract
None

### Keywords
Graft-versus-host disease, Heterogeneity, Extracellular vesicles, Mesenchymal stromal cells, Exosomes, Mesenchymal stem cells

### Affiliations
EMBL
University Hospital Essen (AÃ¶R) Institute for Transfusionmedicine Virchowstr. 179 D-45147 Essen

### Submitter
Frank Stein

### Lab Head
Dr Prof. Dr. Bernd Giebel
University Hospital Essen (AÃ¶R) Institute for Transfusionmedicine Virchowstr. 179 D-45147 Essen


