### Accession
PXD000631

### Title
Comparative Exoprotein Profiling of Different Staphylococcus epidermidis strains Reveals Potential Link between Non-classical Protein Export and Virulence

### Description
This study reports global exoproteome profiling of bovine mastitis strain PM221 and two human strains, commensal-type ATCC12228 and sepsis-associated RP62A to screen for factors underlying adaptation and virulence.

### Sample Protocol
For identification, the separated proteins were in-gel digested into peptides and the resulting peptides were analysed by nanoLC-MS/MS as described by Savijoki et al. 27. Briefly, each gel lane was sliced into seven fractions; the proteins were reduced, alkylated and in-gel digested with trypsin (Sequencing Grade Modified Trypsin V5111, Promega). The resulting tryptic peptides were extracted from the gel, and the recovered peptides were in a final volume of ~130-150 µl. The samples were dried and dissolved in 0.1% trifluoroacetic acid (TFA). The tryptic peptides were analyzed with nanoLC-MS/MS using Ultimate 3000 nano-liquid chromatograph (Dionex) and QSTAR Elite hybrid quadrupole time-of-flight mass spectrometer (Applied Biosystems/MDS Sciex) with nano-ESI ionization. MS data were acquired automatically with Analyst QS 2.0 Software (Applied Biosystems, United States) using SMART IDA.

### Data Protocol
Databases of predicted protein sequences encoded by the PM221 (2530 entries), ATCC12228 (2485 entries, AE015929.1 - AE015935.1) and RP62A (2526 entries, CP000029.1 - CP000028.1) genomes and plasmids were compiled to obtain in-house databases for protein identifications. After LC-MS/MS, MS/MS data obtained from three biological replica samples were first searched separately, and then combined and searched against protein databases using Mascot (Matrix Science, version 2.4.0) and ParagonTM (version 4.0.0.0) search algorithms through the ProteinPilot interface (version 4.0, Applied Biosystems). The search criteria for Mascot searches were: trypsin digestion with one allowed miss-cleavage; carbamidomethyl modification of cysteine as a fixed modification; oxidation of methionine as a variable modification; peptide mass tolerance of 50 ppm; MS/MS fragment tolerance of 0.2 Da; and peptide charges of 1+, 2+, and 3+. Parameters for Paragon searches using the Rapid search mode included the previously mentioned methionine and cysteine modifications.

### Publication Abstract
Staphylococcus epidermidis (SE) includes commensal and pathogenic strains capable of infecting humans and animals. This study reports global exoproteome profiling of bovine mastitis strain PM221 and two human strains, commensal-type ATCC12228 and sepsis-associated RP62A. We identified 451, 395, and 518 proteins from culture supernatants of PM221, ATCC12228, and RP62A, respectively. Comparison of the identified exoproteomes revealed several strain-specific differences related to secreted antigens and adhesins, higher virulence capability for RP62A, and similarities between the PM221 and RP62A exoproteomes. The majority of the identified proteins (&#x223c;80%) were predicted to be cytoplasmic, including proteins known to be associated in membrane vesicles (MVs) in Staphylococcus aureus and immunogenic/adhesive moonlighting proteins. Enrichment of MV fractions from culture supernatants and analysis of their protein composition indicated that this nonclassical protein secretion pathway was being exploited under the conditions used and that there are strain-specific differences in nonclassical protein export. In addition, several predicted cell-surface proteins were identified in the culture media. In summary, the present study is the first in-depth exoproteome analysis of SE highlighting strain-specific factors able to contribute to virulence and adaptation.

### Keywords
Gelc-ms/ms, Staphylococcus epidermidis, Virulence, Exoproteome, Adaptation

### Affiliations
Institute of Biotechnology
Proteomics Unit, Institute of Biotechnology, University of Helsinki

### Submitter
Pia SiljamÃ¤ki

### Lab Head
Dr Tuula A Nyman
Proteomics Unit, Institute of Biotechnology, University of Helsinki


