### Accession
PXD039414

### Title
Robust and High-Throughput Analytical Flow Proteomics Analysis of Cynomolgus Monkey and Human Matrices with Zeno SWATH Data Independent Acquisition

### Description
Modern mass spectrometers routinely allow deep proteome coverage in a single experiment. These methods are typically operated at nano and micro flow regimes, but they often lack throughput and chromatographic robustness, which is critical for large-scale studies. In this context, we have developed, optimized and benchmarked LC-MS methods combining the robustness and throughput of analytical flow chromatography with the added sensitivity provided by the Zeno trap across a wide range of cynomolgus monkey and human matrices of interest for toxicological studies and clinical biomarker discovery. SWATH data independent acquisition (DIA) experiments with Zeno trap activated (Zeno SWATH DIA) provided a clear advantage over conventional SWATH DIA in all sample types tested with improved sensitivity, quantitative robustness and signal linearity as well as increased protein coverage by up to 9-fold. Using a 10-min gradient chromatography, up to 3,300 proteins were identified in tissues at 2 µg peptide load. Importantly, the performance gains with Zeno SWATH translated into better biological pathway representation and improved the ability to identify dysregulated proteins and pathways associated with two metabolic diseases in human plasma. Finally, we demonstrate that this method is highly stable over time with the acquisition of reliable data over the injection of 1,000+ samples (14.2 days of uninterrupted acquisition) without the need for human intervention or normalization. Altogether, Zeno SWATH DIA methodology allows fast, sensitive and robust proteomic workflows using analytical flow and is amenable to large-scale studies. This work provides detailed method performance assessment on a variety of relevant biological matrices and serves as a valuable resource for the proteomics community.

### Sample Protocol
Sample Preparation. Approximately 25 mg of frozen tissues were transferred to 2 mL Eppendorf Protein Lobind tubes containing one 5 mm stainless steel bead (cat# 69989, Qiagen) and 500 µL of lysis buffer consisting of 5% SDS in 50 mM TEAB with protease inhibitors cocktail (cat# A32953, Thermo Fisher Scientific). Tissue pieces were disrupted with TissueLyser LT (Qiagen) at 50 Hz (1 min ‘on’, 1 min ‘off’ on ice) until fully homogenized. Four microliters of plasma was added to 46 µL of lysis buffer and sputum (200 µL), BAL (300 µL), mucus (300 µL) and media (150 µL) samples were first dried to completion under a stream of nitrogen (TurboVap 96, Biotage) and resolubilized in 50 µL of lysis buffer. Homogenates were sonicated on ice for 5 min at 30% amplitude (1 min ‘on’, 15s ‘off’) with a Qsonica sonicator (Q700MPX) and centrifuged for 10 min at 14,000g at 18°C. The supernatants were then collected, and protein concentrations were determined using Pierce BCA Protein Assay Kit (cat# 23225, Thermo Fisher Scientific) following the manufacturer’s instructions. Reduction and alkylation of proteins were performed after incubation of the extracts at 95°C for 10 min by adding 5 µL of 100 mM TCEP and 5 µL of 400 mM chloroacetamide each followed by an incubation at 95°C for 5 min.  On-column digestion was performed using S-Trap sample processing technology in 96-well plate format (100 – 300 μg per well, cat# NC1508276, Protifi LLC) following the manufacturer’s instructions. First, proteins were further denatured by adding 5 µL of phosphoric acid (12%) followed by 300 µL of S-Trap binding buffer (90% MeOH, 100 mM final TEAB, pH 7.1) to protein extracts (15-150 µg of proteins). Proteins were then loaded and captured on the S-Trap columns following centrifugation at 1,500g for 2 min at room temperature. Proteins were then washed 3 times with 200 μL S-Trap binding buffer (90% MeOH, 100 mM final TEAB, pH 7.1) and centrifuged at 1,500g for 2 min. Protein digestion was performed overnight at 37°C with trypsin/Lys-C protease mix (cat# A40009, Thermo Fisher Scientific) in 125 µL 50 mM TEAB (1:30 enzyme:protein ratio). Stepwise elution of peptides was performed with 80 µL of 50 mM TEAB, 80 µL of 0.2% formic acid in water, and 80 µL of 0.2% formic acid in 50% ACN each followed by centrifugation at 1,500g for 2 min. Flowthroughs were then pooled and dried under a stream of nitrogen (TurboVap 96, Biotage) and peptides were resuspended in 100 µL of 0.1% (v/v) formic acid in 5% (v/v) ACN. Peptide concentrations were determined using Pierce Quantitative Colorimetric Peptide Assay (cat# 23275, Thermo Fisher Scientific), and samples were frozen at -80°C until LC-MS data acquisition.  Untargeted Proteomics by LC-MS. Peptide extracts were analyzed using an Exion LC system coupled with a 7600 ZenoTOF mass spectrometer (SCIEX). Peptides were separated on an Acquity BEH C18 column 2.1 × 150 mm, 1.7 μm (cat# 186003687, Waters) and mobile phase solvents consisted in 0.1% formic acid in water (A) and 0.1% formic acid in ACN (B). Peptides were eluted from the column using a 10-minute linear gradient (5-40% B) at a flow rate of 0.2 mL/min and the oven temperature was set at 40°C. Autosampler temperature was set at 10°C. The 7600 ZenoTOF was equipped with an OptiFlow Turbo V ion source and operated in SWATH mode with or without Zeno trap activated. The source conditions were as follows: ionization voltage: 5500 V, positive polarity, temperature: 500°C, ion source gas 1: 60 psi, ion source gas 2: 80 psi, curtain gas: 40 psi. The acquisition settings were as follows: number of Q1 windows: 50, MS1 accumulation time: 100 ms, MS1 m/z range: 400–900 Da, MS2 accumulation time: 18 ms (plasma and media) or 15 ms (rest of the samples), MS2 m/z range: 100–900 Da. Q1 window widths were determined for each sample type from DDA runs using SWATH Variable Window Calculator_V1.2 provided by SCIEX. All tissues were run with the same method optimized on liver sample. External calibration was performed every 3 to 8 runs using the automatic calibration function and ESI positive calibration solution for SCIEX X500B system (cat# 5049910, SCIEX). Ten-point dilution series (3.9 ng – 2000 ng) were prepared the day prior to the experiment, stored at 4°C overnight and injected from low to high concentrations in triplicates in both SWATH DIA and Zeno SWATH DIA modes. Healthy and diseased plasma samples were injected in singlicate in a random order in SWATH mode first followed by Zeno SWATH. All samples were injected no more than 1.7 days after being loaded in the autosampler. Signal stability over time and method robustness were monitored by repetitive injection of 800 ng of HeLa protein digest (cat# 88329, Thermo Fisher Scientific) in Zeno SWATH mode. HeLa protein digest sample was left in the autosampler and the detector parameters were left unchanged during the whole duration of the study.

### Data Protocol
Data Processing. All raw data were processed with DIA-NN (v1.8.1) with a fragment ion m/z range of 200–1800, automatic settings of mass accuracy at the MS2 and MS1 level and scan window, protein inference of ‘Genes’ and quantification strategy of “robust LC (high precision)". For dilution series, triplicate samples from each dilution level and acquisition mode were processed separately with cross-run normalization disabled and analyzed with match-between-runs (MBR) enabled. Clinical plasma samples from each case study and each acquisition mode were processed separately with cross-run normalization enabled (‘RT-dependent’) and analyzed with match-between-runs (MBR) enabled. All database searches were performed using in silico predicted libraries from Human UniProt canonical sequence database (UP000005640) (human biofluids and in vitro samples) and UniProt canonical sequence database (UP000233100) (cynomolgus monkey tissues). Predicted libraries were generated using the following parameters: Trypsin/P, one missed cleavage allowed, N-term M excision, C carbamidomethylation, peptide length range: 7-30 amino acids, precursor charge range: 1-4. Details on library of decoy precursors generation and data matching to spectral libraries can be found in the original DIA-NN article [11]. HeLa digests were processed using an experimental library generated from HeLa and K562 cell extracts provided by SCIEX.

### Publication Abstract
Modern mass spectrometers routinely allow deep proteome coverage in a single experiment. These methods are typically operated at nanoflow and microflow regimes, but they often lack throughput and chromatographic robustness, which is critical for large-scale studies. In this context, we have developed, optimized, and benchmarked LC-MS methods combining the robustness and throughput of analytical flow chromatography with the added sensitivity provided by the Zeno trap across a wide range of cynomolgus monkey and human matrices of interest for toxicological studies and clinical biomarker discovery. Sequential Window Acquisition of All Theoretical Fragment Ion Mass Spectra (SWATH) data-independent acquisition (DIA) experiments with Zeno trap activated (Zeno SWATH DIA) provided a clear advantage over conventional SWATH DIA in all sample types tested with improved sensitivity, quantitative robustness, and signal linearity as well as increased protein coverage by up to 9-fold. Using a 10-min gradient chromatography, up to 3300 proteins were identified in tissues at 2 &#x3bc;g peptide load. Importantly, the performance gains with Zeno SWATH translated into better biological pathway representation and improved the ability to identify dysregulated proteins and pathways associated with two metabolic diseases in human plasma. Finally, we demonstrate that this method is highly stable over time with the acquisition of reliable data over the injection of 1000+ samples (14.2 days of uninterrupted acquisition) without the need for human intervention or normalization. Altogether, Zeno SWATH DIA methodology allows fast, sensitive, and robust proteomic workflows using analytical flow and is amenable to large-scale studies.

### Keywords
Zeno swath, 7600 zenotof, Biofluids, Tissues, Dia

### Affiliations
Group Director OMICS, Integrated Bioanalysis AstraZeneca R&D / Clinical Pharmacology & Safety Sciences  121 Oyster Point Blvd. South San Francisco, CA 94080, USA
AstraZeneca

### Submitter
Kevin Contrepois

### Lab Head
Dr Kevin Contrepois
Group Director OMICS, Integrated Bioanalysis AstraZeneca R&D / Clinical Pharmacology & Safety Sciences  121 Oyster Point Blvd. South San Francisco, CA 94080, USA


