### Accession
PXD004911

### Title
Mouse caecum vs faeces -  Metaproteogenomics Reveals Taxonomic and Functional Changes between Cecal and Fecal Microbiota in Mouse

### Description
We compared the microbiota of paired mouse caecal contents and faeces by applying a multi-omic approach, including 16S rDNA sequencing, shotgun metagenomics, and shotgun metaproteomics. The aim of the study was to verify whether faecal samples are a reliable proxy for the mouse colonic luminal microbiota, as well as to identify changes in taxonomy and functional activity between caecal and faecal microbial communities, which have to be carefully considered when using stool as sample for mouse gut microbiota investigations.

### Sample Protocol
Faecal and caecal samples were resuspended by vortexing in SDS-based extraction buffer and then heated and subjected to a combination of bead-beating and freeze-thawing steps as detailed elsewhere (Tanca et al, Microbiome, 2014). Protein extracts were subjected to on-filter reduction, alkylation, and trypsin digestion according to the filter-aided sample preparation (FASP) protocol, with slight modifications detailed elsewhere (Tanca et al., Proteomics 2013). LC-MS/MS analysis was carried out using an LTQ-Orbitrap Velos mass spectrometer (Thermo Scientific, San Jose, CA, USA) interfaced with an UltiMate 3000 RSLCnano LC system (Thermo Scientific). The single-run 1D LC peptide separation was performed as previously described27, loading 4 Î¼g of peptide mixture per each sample and applying a 485 min separation gradient. The mass spectrometer was set up in a data dependent MS/MS mode, with Higher Energy Collision Dissociation as the fragmentation method.

### Data Protocol
Peptide identification was performed using the Proteome Discoverer informatic platform (version 1.4; Thermo Scientific), with Sequest-HT as search engine and Percolator for peptide validation (FDR < 1%). Search parameters were set as described previously (Tanca et al., Proteomics 2015). Parallel searches were performed using two different sequence databases, and results from the two searches for each sample were merged. The first database was composed of the metagenomic sequences obtained in this study, both as raw reads and assembled contigs (2,158,809 sequences). Paired reads were merged as described above for 16S rDNA analysis. The output sequences were filtered (with a fastq_truncqual option = 15) and clustered at 100% using USEARCH v.5.2.23620. Read assembly into contigs was carried out using Velvet v.1.2.1031, by setting 61 as k-mer length, 200 as insert length, and 300 as minimum contig length. Open reading frames were found from both reads and contigs using FragGeneScan v.1.19, with the training for Illumina sequencing reads with about 0.5% error rate32. The second database was a selection of all bacterial, archaeal, fungal and gut microbiota sequences (79,203,800 sequences in total) from the 2015_02 release of the UniProtKB database. The Normalized Spectral Abundance Factor (NSAF) was calculated in order to estimate protein abundance. Specifically, a spectral abundance factor (SAF) was obtained by dividing the number of spectral counts of a protein by its number of amino acids; then, NSAF values were obtained by dividing the SAF values by the SAF sum for all proteins identified in each sample. NSAF values were finally multiplied by a scale factor corresponding to the average number of spectral counts identified per sample (in order to deal with integers).

### Publication Abstract
Previous studies on mouse models report that cecal and fecal microbial communities may differ in the taxonomic structure, but little is known about their respective functional activities. Here, we employed a metaproteogenomic approach, including 16S rRNA gene sequencing, shotgun metagenomics and shotgun metaproteomics, to analyze the microbiota of paired mouse cecal contents (CCs) and feces, with the aim of identifying changes in taxon-specific functions. As a result, Gram-positive anaerobes were observed as considerably higher in CCs, while several key enzymes, involved in oxalate degradation, glutamate/glutamine metabolism, and redox homeostasis, and most actively expressed by Bacteroidetes, were clearly more represented in feces. On the whole, taxon and function abundance appeared to vary consistently with environmental changes expected to occur throughout the transit from the cecum to outside the intestine, especially when considering metaproteomic data. The results of this study indicate that functional and metabolic differences exist between CC and stool samples, paving the way to further metaproteogenomic investigations aimed at elucidating the functional dynamics of the intestinal microbiota.

### Keywords
Gut microbiota, Mouse, Metaproteomics, Faeces, Caecum

### Affiliations
Proteomics Laboratory, Porto Conte Ricerche, Alghero, Italy
Porto Conte Ricerche, Alghero, Sardinia, Italy

### Submitter
Alessandro Tanca

### Lab Head
Dr Sergio Uzzau
Porto Conte Ricerche, Alghero, Sardinia, Italy


