### Accession
PXD020600

### Title
Multi cell line comparison of lysosome enriched fractions reveals cell type specific features and novel lysosomal proteins

### Description
Lysosomes are the main degradative organelles of mammalian cells and are responsible for the breakdown of the majority of cellular macromolecules and the recycling of their building blocks. To accomplish this task, lysosomes contain ~60 hydrolases. Malfunction of these proteins, as well as other proteins responsible for the transport of small molecules or proteins, lead to the accumulation of their respective substrates. This accumulation of substrates results in heavy impairment of lysosomal function leading to ~70 lysosomal storage disorders (LSDs). A better understanding of lysosomal composition is of high importance not only for a better understanding of LSDs, but also for cellular function. Analysis of lysosomes by mass spectrometry-based proteomics presents an attractive approach to allow for a better characterization of lysosomes. In the current study, we investigated the lysosomal proteome from six different cell lines (HEK293, HeLa, HuH-7, SH-SY5Y, MEF and NIH3T3) by lysosome enrichment using SPIONs (superparamagnetic iron oxide nanoparticles) combined with mass spectrometry-based proteomics. Comparison of the individual proteomes revealed cell type specific characteristics in lysosomal protein expression. To allow for the discrimination of lysosomal from unspecific enriched proteins, we included a differentially SILAC (stable isotope labelling by amino acids in cell culture) labelled control sample. Afterwards, lysosomal proteins were identified using a bimodal distribution model. By combination of data from several cell lines, we were able to generate a high confidence list of putative novel lysosomal proteins of which several were confirmed by immunostaining.

### Sample Protocol
Lysosomes were enriched from six cell lines (HEK293, HeLa, HuH-7, SH-SY5Y, MEF and NIH3T3) using SPION. Moreover, a label switching approach was used, where for two replicates SILAC heavy labelled cells received SPIONs and light labelled cells did not and vice versa for another two replicates. Additionally, whole cell lysate samples were generated by culturing the cells without SILAC amino acid to directly obtain a comparison between the datasets. For mass spectrometry sample preparation, 100 µg of protein was precipitated using chloroform/methanol. Protein pellets were resuspended in 8 M urea, 0.1 M TEAB, and incubated at RT, 800 rpm for 45 min, followed by reduction with 5 mM DTT at 56 °C, 800 rpm for 25 min, and alkylation with 20 mM acrylamide at RT, 800 rpm for 30 min. The reaction was quenched by addition of 5 mM DTT, samples diluted to 4 M urea with 0.1 M TEAB, rLys-C added at an enzyme to protein ratio of 1 to 100, and digestion was performed at 37 °C overnight. The following day, samples were diluted with 0.1 M TEAB to 1.6 M urea, trypsin was added at an enzyme to protein ratio of 1 to 100, and the samples were incubated at 37 °C for 10 h. Finally, the samples were acidified using acetic acid (0.1% final concentration) and 10 µg of peptides desalted by C18 STAGE tips. Eluted peptides were dried using a vacuum centrifuge and resuspended in 5% ACN, 5% FA. Mass spectrometry analyses were performed using a Dionex Ultimate 3000 system coupled to an Orbitrap Fusion Lumos mass spectrometer (both Thermo Scientific, Bremen, Germany). Columns were manufactured in-house as follows: 50 cm spray tips were generated from 360 μm outer diameter/100 μm inner diameter fused silica capillaries using a P-2000 laser puller (Sutter Instruments, Novato, CA) and packed with 1.9 μm particles for the lysosome enriched samples and 3 µm Reprosil AQ C18 particles for the whole cell lysate (Dr. Maisch, Ammerbuch-Entringen, Germany). The samples were resuspended in 5% ACN, 5% FA and loaded on the analytical column at a flow rate of 600 nL/min, 100% solvent A (0.1% FA in water). Regarding the SILAC analysis of the lysosome-enriched samples, the peptides were separated at a flow rate of 300 nL/min with a 240 min linear gradient from 5–35% solvent B (95% ACN, 0.1% FA). Survey spectra were acquired in the Orbitrap mass analyser with a mass range of m/z 375–1,575 at a resolution of 60,000. MS/MS fragmentation was performed in data dependent acquisition mode for charge states between 2–4 by HCD with 30% collision energy and data were acquired in the Orbitrap at a resolution of 30,000. The cycle time was set to 5 s and the precursor isolation width to 1.6 m/z using the quadrupole. For MS1 and MS2 scans, the automatic gain control (AGC) was set to 4 × 105 and 5 × 105, respectively. The LFQ analysis of the whole cell lysate samples, was obtained by separating the peptides at a flow rate of 300 nL/min with a 180 min linear gradient (for the data-independent acquisition mode) and 240 min linear gradient (for the data-dependent acquisition mode) from 5–35% solvent B (95% ACN, 0.1% FA). Survey spectra were acquired in the Orbitrap mass analyser with a mass range of m/z 350 - 1,200 at a resolution of 120,000 and a maximum inject time of 50 ms. MS/MS fragmentation was performed in data independent acquisition mode using 24.1 m/z quadrupole isolation windows (mass range: 350 – 1200 m/z) and a HCD collision energy of 30 %. Data were acquired in the Orbitrap at a resolution of 30,000, a maximum inject time of 54 ms, and a mass range of 200 – 2000 m/z. The cycle time was set to 3 s. For MS1 and MS2 scans, the automatic gain control (AGC) was set to 5 × 105 and 1 × 106, respectively.

### Data Protocol
Thermo *.raw files were analysed with Proteome Discoverer 2.2 (Thermo Fisher Scientific, Bremen, Germany) in combination with Mascot 1.6.1 (www.matrixscience.com). For database searching, UniProt Homo sapiens (release 2019_05, 73,920 entries) and UniProt Mus musculus (release 2019_05, 54,425 entries) in combination with the cRAP database (ftp://ftp.thegpm.org/fasta/cRAP/crap.fasta) including common contaminants were used. The following parameters were defined: Quantification method: SILAC 2plex (13C615N4 or Arg10, 13C615N2 or Lys8); variable modifications: oxidation of methionine, acetylation of protein N-termini; fixed modification: propionamide at cysteine; mass tolerance: 10 ppm for precursor ions, 50 mmu for fragment ions; enzyme: trypsin except proline was the next amino acid; missed cleavage sites: 2. Data were filtered with a false discovery rate (FDR) of 1% at the peptide level using Percolator and proteins were exported with an FDR of 1%. For quantification, SPIONs/control ratios were determined for each individual raw file and unique + razor peptides were used. Furthermore, data were analysed with MaxQuant 1.6.14.0 (https://www.maxquant.org/) in combination with the UniProt reference proteomes for Homo sapiens (release 2020_04, 96,808 entries) and Mus musculus (release 2020_04, 63666 entries) using standard settings. For fixed and variable modifications, the same search parameters as for Proteome Discoverer were used, except that SILAC ratios were not used for protein quantification. For cells receiving SPIONs in the heavy channel, heavy labelled arginine (13C615N4 or Arg10) and lysine (13C615N2 or Lys8) were set as fixed modification and for cells receiving SPIONs in the light channel no isotope labelling was considered. The LFQ analysis of the DIA-measured whole cell lysate samples was performed in Spectronaut (16.2.220903.53000) where both DDA and DIA raw files were used to generate two hybrid spectral libraries using as database UniProt SwissProt Homo sapiens (release 2021_01, 20,394 entries) and UniProt SwissProt Mus musculus (release 2021_01, 17,056 entries), respectively. Note that in the library generation step a common contaminant database was included. For each peptide, the 3 - 6 most abundant b/y ions were selected for library generation, dependent on their signal intensity. For fixed and variable modifications, the same search parameters as for Proteome Discoverer and Max Quant were used. Data were filtered with the following QValue cutoffs: precursor =  0.01, Protein (Experiment) = 0.01, Protein (Run)= 0.05. Dynamic retention time alignment was performed based on the high-precision indexed retention time (iRT) concept (Bruderer et al., 2016). Quantification data was extracted on MS2-level, median normalised globally and missing values filled by run-wise imputation.

### Publication Abstract
Lysosomes, the main degradative organelles of mammalian cells, play a key role in the regulation of metabolism. It is becoming more and more apparent that they are highly active, diverse, and involved in a large variety of processes. The essential role of lysosomes is exemplified by the detrimental consequences of their malfunction, which can result in lysosomal storage disorders, neurodegenerative diseases, and cancer. Using lysosome enrichment and mass spectrometry, we investigated the lysosomal proteomes of HEK293, HeLa, HuH-7, SH-SY5Y, MEF, and NIH3T3 cells. We provide evidence on a large scale for cell type-specific differences of lysosomes, showing that levels of distinct lysosomal proteins are highly variable within one cell type, while expression of others is highly conserved across several cell lines. Using differentially stable isotope-labeled cells and bimodal distribution analysis, we furthermore identify a high confidence population of lysosomal proteins for each cell line. Multi-cell line correlation of these data reveals potential novel lysosomal proteins, and we confirm lysosomal localization for six candidates. All data are available via ProteomeXchange with identifier PXD020600.

### Keywords
Human, Mouse, Lfq, Cell line comparison, Lysosome, Isolated lysosomes, Silac, Ibaq, Dia

### Affiliations
Institute for Biochemistry and Molecular Biology, University of Bonn, Bonn, Germany
Postdoctoral Researcher

### Submitter
Fatema Akter

### Lab Head
Dr Dominic Winter
Institute for Biochemistry and Molecular Biology, University of Bonn, Bonn, Germany


