### Accession
PXD016970

### Title
Compositional and Immunogenic Evaluation of Fractionated Wheat Beers using Mass Spectrometry

### Description
The safety and regulatory status of fermented products derived from gluten-containing grains for patients with celiac disease has remained controversial. Bottom-up mass spectrometry (MS) has complemented immunoassays for the compositional and immunogenic analyses of wheat beers. However, uncharacterized proteolysis during brewing followed by the secondary digestion for MS analysis has made the analysis and data interpretation complicated. In this study, the composition and immunogenic potential of seven commercially available wheat beers were evaluated using bottom-up MS with the aid of fractionation and a multi-step peptide search strategy to capture possible cleavage combinations. Gluten-derived peptides accounted for approximately 50% and 20% of the total number of wheat-derived and barley-derived peptides in investigated beers, respectively. Although relatively large polypeptides cannot be thoroughly characterized using traditional bottom-up proteomics, up to 50% of the identified peptides still contained celiac-immunogenic motifs, and consumption of wheat beers would be risky and not recommended for celiac patients.

### Sample Protocol
Beer samples were fractionated with 30 kDa centrifugal filters (Millipore Sigma, MA, USA) prior to denaturation, reduction, and alkylation, and the resulting fractions were subsequently processed and analyzed separately with qualitative discovery LC-MS/MS. In detail, for each type of beer, proteins and peptides from 500 ���������L of beer were fractionated with 30 kDa centrifugal filters at 14,000 x g for 15 min. The flow-through (fraction below 30 kDa), which contained endogenous beer peptides, was reduced with 10 mM dithiothreitol at 95 ������C for 5 min and alkylated with 10 mM iodoacetamide at room temperature in the dark for 30 min. For the retentate (fraction above 30 kDa), 50 ���������L was collected and diluted 5-fold with 50 mM HEPES buffer (pH 7.8), denatured with 4 M GuHCl at 65 ������C for 15 min, and reduced and alkylated as described previously. This fraction was further diluted 4-fold with 50 mM HEPES buffer (pH 7.8) for chymotrypsin (Sequencing grade, Promega, WI, USA) digestion (1:100 w/w, protease:substrate) at 25������C overnight. The efficiency of digestion was evaluated by SDS-PAGE analysis (Figure S-1). All samples (150 ���������L of each) were desalted using Pierce��������� C18 Centrifugal Columns (Thermo Fisher Scientific, MA, USA) following the manufacturer���������s instructions. The desalted peptide mixtures were dried in a centrifugal evaporator (Jouan, NJ, USA) and reconstituted with 0.1% formic acid (FA) in H2O with 3% acetonitrile (ACN) for LC-MS/MS analysis with duplicate sample preparations and duplicate instrument injections (n=4). Both fractions were analyzed separately using the same data-dependent acquisition method. For each sample, 2 ���������L of reconstituted peptide mixture was chromatographically separated on a Thermo Scientific��������� Ultimate 3000 RSLC ultrahigh performance liquid chromatography (UHPLC) system using a Thermo Scientific��������� Hypersil GOLD��������� C18 Selectivity LC column (100 mm ������ 1 mm i.d., 1.9 ���������m particle size) at a flow rate of 0.06 mL/min using a linear gradient of 2-40% solvent B over 70 min and retaining at 40% solvent B for another 6 min (mobile phase A: 0.1% FA in H2O; mobile phase B: 0.1% FA in ACN). A Thermo Scientific��������� Q Exactive��������� Plus Hybrid Quadrupole-Orbitrap��������� mass spectrometer was used to perform a data-dependent MS/MS analysis. From each full scan (70,000 resolution, 400���������1400 m/z scan range, 3e6 automatic gain control (AGC) target, 100 ms maximum injection times (IT), and 1+ to 5+ charge state), the top ten most intense parent ions were selected for MS2 analysis (70,000 resolution, 200���������2000 m/z scan range, 1e5 AGC target, and 60 ms maximum IT). The minimum AGC target was set at 1.50e3 for an intensity threshold of 2.5e4. The isolation window was set at 2.0 m/z and normalized collision energy at 27.

### Data Protocol
Proteome Discoverer 2.1 (PD) (Thermo Fisher Scientific, MA, USA) was employed to perform a SEQUEST HT search against a database containing all protein sequences of Triticum aestivum (136,866 sequences, Aug11st, 2017), Humulus lupulus (459 sequences, Mar19th, 2018), Saccharomyces cerevisiae (23,784sequences, Mar19th, 2018), and Hordeum vulgare (212,719 sequences, Mar19th, 2018) downloaded from UniProt, combined with the common Repository of Adventitious Proteins (cRAP) sequences database (115 sequences, version 2012.01.01, downloaded from The Global Proteome Machine). Fixed modifications included cysteine carbamidomethylation, and up to three dynamic modifications (methionine oxidation and asparagine/glutamine deamidation) were allowed. The precursor and product ion mass tolerances were ������10 ppm and ������0.02 Da, respectively. A precursor mass range from 350-5000 Da with a maximum of two missed cleavages was permitted. Both fractions (above and below 30 kDa) were analyzed independently with different enzyme cleavage settings. For the retentate, the SEQUEST HT search was performed separately with each of three enzymatic cleavage site settings (chymotrypsin full (FWY/not P), chymotrypsin semi (only one termini resulting from the cleavage of FWY/not P), and no-enzyme (unspecific)) to account for possible cleavages at different degrees of hydrolysis. Note that only high-affinity cleavage sites for chymotrypsin were included in the search but peptides produced by possible low affinity-chymotrypsin cleavage (L and M) can also found by the semi and no-enzyme search. Peptide-spectrum matches were validated using a 1% target-decoy-based False Discovery Rate (FDR). Peptides identified by more than one setting were considered redundant, and only one identification was kept by priority: full cleavage > semi cleavage > unspecific. For the flow-through fraction (< 30 kDa), only a no-enzyme (unspecific) search setting was employed to identify endogenous peptides present in beer, and only high confidence peptides (FDR < 1%) were accepted. For both fractions, peptides with dynamic modifications and multiple charges were considered redundant and counted only once. Results from different replicates for each beer were combined for subsequent analysis.  At the protein level, due to incomplete database annotation, peptides derived from uncharacterized protein sequences were searched by BLAST against UniProt to identify homologous, annotated proteins in UniProt. Protein isoform-level of information was not investigated in this study due to the numerous shared peptides between highly homologous proteins. Therefore, proteins were manually curated to groups by keyword: avenin, gliadin (wheat)/hordein (barley), high molecular weight glutenin (HMW), low molecular weight glutenin (LMW), ns-LTP, serpin, and trypsin/amylase inhibitor. Proteins not containing any of the prior keywords (e.g. endochitinase, hordoindoline, etc.) were grouped as minor proteins.

### Publication Abstract
The safety and regulatory status of fermented products derived from gluten-containing grains for patients with celiac disease remains controversial. Bottom-up mass spectrometry (MS) has complemented immunoassays for the compositional and immunogenic analyses of wheat beers. However, uncharacterized proteolysis during brewing followed by the secondary digestion for MS has made the analysis and data interpretation complicated. In this study, the composition and immunogenic potential of seven commercially available wheat beers were evaluated using bottom-up MS with the aid of fractionation and a multi-step peptide search strategy to identify peptides generated by various types of proteolysis. Gluten-derived peptides accounted for approximately 50% and 20% of the total number of wheat-derived and barley-derived peptides, respectively, in the investigated beers. Although relatively large polypeptides cannot be thoroughly characterized using traditional bottom-up proteomics, up to 50% of peptides identified contained celiac-immunogenic motifs, and consumption of wheat beers would pose risks for celiac patients.

### Keywords
Celiac disease, Proteomics, Wheat beers, Fermentation, Gluten, Mass spectrometry

### Affiliations
UNL
FARRP, UNL  1901 North 21st Street, University of Nebraska-Lincoln Lincoln, NE 68588-6205

### Submitter
Wanying Cao

### Lab Head
Dr Melanie L. Downs
FARRP, UNL  1901 North 21st Street, University of Nebraska-Lincoln Lincoln, NE 68588-6205


