### Accession
PXD001814

### Title
Bienertia s. chloroplast compartments replica 2 CCCP band1to10

### Description
Bienertia sinuspersici performs C4 photosynthesis without Kranz anatomy through subcellular compartmentalization of carbon fixation within individual cells. In this species, central compartment chloroplasts (C) and peripheral chloroplasts (P) collaborate with cytosolic and mitochondrial components in a NAD-ME type C4 cycle. How the two functionally different chloroplast types can develop within individual cells and the mechanism of import of nuclear encoded, plastid targeted proteins are currently unknown. We used 454 sequencing in combination with large scale label-free proteomics to determine the distribution of photosynthesis-related proteins. Subcellular localization of 169 protein was determined through comparison of protein abundance in four different subcellular fractions. 39 out of the 120 chloroplastic proteins showed differential accumulation between the two chloroplast types. Rubisco, RPP regenerative phase and PSII related proteins accumulated in C chloroplasts whereas C4 related proteins and the NDH complex were more abundant in P chloroplasts. Comparison of transit peptides of differential accumulating proteins indicated no obvious sequence homology or similarities in physico-chemical properties between members of the same group. Protein composition analysis of the central compartment indicated that mitochondria and peroxisomes are the only major components besides chloroplasts in this compartment. The combined information from subcellular and developmental protein profiling was used to generate a first draft of the protein machinery involved in single-cell C4 photosynthesis.

### Sample Protocol
Isolation of Bienertia s.Chloroplast compartment, 1D SDS-PAGE, nano LC MSMS, shotgun proteomics

### Data Protocol
For generation of the Bienertia transcriptome, total RNA was isolated from 40 randomly pooled mature and young leaves using the TRIzol method (Invitrogen). Purified RNA was mixed in the ratio 1:3 (young:mature) and precipitated before shipping. mRNA isolation from total RNA and preparation of normalized cDNA was conducted by commercial service from Bio S&T (Montreal, Canada). The normalized cDNA was sequenced using a full run on a Roche 454 (FLX/Titanium) instrument. Sequencing adaptors were removed using gsAssembler (Roche Life Sciences) followed by cleaning by LUCY (Chou and Holmes, 2001) and SeqClean (http://sourceforge.net/projects/seqclean/) software, respectively. The resulting 149835 reads with more than 100 bp were assembled using iAssembler (Zheng et al., 2011)  resulting in 74126 ESTs. A protein sequence database (AADB) was created to be used for searching the mass spectral data. Bienertia ESTs were blasted against all predicted Arabidopsis thaliana proteins (from here on Arabidopsis) to obtain a training set to determine the codon usage matrix model. Most likely reading frames were determined using this codon usage model with the help of ESTscan (Iseli et al., 1999). These sequences were blasted against the predicted proteomes of Arabidopsis, maize and rice and the best hits were used for annotation of the corresponding proteins. This resulted in 40018 protein sequences. A contaminant set (Keratin, BSA etc.) and a decoy database with the randomized protein sequences was added, resulting in a database named Bienertia_Prandom rand (80411 sequences; 19978532 residues).For the developmental samples, 20 leaves of each developmental stage (youngest 0.3-1 mm in size, young 1-3 mm in size and mature approx. 2 cm in size) were harvested for each replicate from mature Bienertia plants. Leaves of the same developmental stage were pooled (separately for each biological replicate) and ground to a fine powder in liquid nitrogen. The frozen plant material was resuspended in extraction buffer (2% (w/v) SDS, 10% (w/v) glycerol, 62.5 mM Tris-HCl, pH 6.8, and 0.715 M 2-mercaptoethanol) and immediately boiled for 5 min. Insoluble material was removed by centrifugation (10 min, 16,000 x g) in a benchtop centrifuge. Protein concentration was determined with RCDC protein quantification kit (Bio-Rad), which tolerates detergents and reducing agents. For the localization samples, protein extracts from the two chloroplast types, the central cytoplasmic compartment and total protoplasts was used that were extensively characterized in a previous study (Offermann et al., 2011b). 50 Âµg of total protein extract from the 4 different localization samples and three different developmental samples were separated on Novex Tricine 10-20% PAA gels (Invitrogen) in two independent biological replicates. Individual gel lanes were cut into 10 (localization series) or 8 (developmental series) slices and proteins were digested with trypsin. Extracted peptides were analyzed by nano LC-LTQ-Orbitrap MS using data-dependent acquisition and dynamic exclusion, essentially as described previously (Majeran et al., 2008).  Processing of the MS Data, Database Searches, and upload into PPDB. Peak lists (.mgf format) were generated using DTA supercharge (version 1.19) software (http://msquant.sourceforge.net/) and searched with Mascot version 2.2 (Matrix Science) against the generated Bienertia databases. For off-line calibration, first a preliminary search was conducted with the precursor tolerance window set at 30 ppm. Peptides with ion scores above 40 were chosen as benchmarks to determine the offset for each LC-MS/MS run. This offset was then applied to adjust precursor masses in the peak lists of the respective .mgf file for recalibration using a Perl script (B. Zybailov, unpublished data). The recalibrated peak lists were searched against the Bienertia_Prandom rand DB.  Each of the peak lists were searched using Mascot version 2.2 (maximum P of 0.01) for full tryptic peptides using a precursor ion tolerance window set at 6 ppm, variable Met oxidation, fixed Cys carbamidomethylation, 1 miss cleavage allowed, and for semi-tryptic peptides using a precursor ion tolerance window set at 3 ppm, variable Met oxidation, fixed Cys carbamidomethylation, Glutamine to pyro-glutamic acid (N-term Q) modification,1 miss cleavage allowed and a minimal mascot ion score threshold of 30. This yielded a peptide false discovery rate below 1%, with peptide false positive rate calculated as 2 (decoy_hits/total_hits).

### Publication Abstract
Kranz C4 species strictly depend on separation of primary and secondary carbon fixation reactions in different cell types. In contrast, the single-cell C4 (SCC4) species Bienertia sinuspersici utilizes intracellular compartmentation including two physiologically and biochemically different chloroplast types; however, information on identity, localization, and induction of proteins required for this SCC4 system is currently very limited. In this study, we determined the distribution of photosynthesis-related proteins and the induction of the C4 system during development by label-free proteomics of subcellular fractions and leaves of different developmental stages. This was enabled by inferring a protein sequence database from 454 sequencing of Bienertia cDNAs. Large-scale proteome rearrangements were observed as C4 photosynthesis developed during leaf maturation. The proteomes of the two chloroplasts are different with differential accumulation of linear and cyclic electron transport components, primary and secondary carbon fixation reactions, and a triose-phosphate shuttle that is shared between the two chloroplast types. This differential protein distribution pattern suggests the presence of a mRNA or protein-sorting mechanism for nuclear-encoded, chloroplast-targeted proteins in SCC4 species. The combined information was used to provide a comprehensive model for NAD-ME type carbon fixation in SCC4 species.

### Keywords
Shorgun proteomics, Chloroplast compartment, Development, Bienertia sinuspersicis

### Affiliations
Plant Biology
Cornell University, Ithaca NY (USA)

### Submitter
Giulia Friso

### Lab Head
Dr Giulia Friso c/o Klaas van Wijk Lab
Cornell University, Ithaca NY (USA)


