### Accession
PXD032043

### Title
Proteomic profiling of black coral (Antipatharia) skeleton reveals hundreds of skeleton-associated proteins across two taxa

### Description
Black corals, ecologically important cnidarians found from shallow to deep ocean depths, form a strong yet flexible skeleton of sclerotized chitin and other biomolecules including proteins. The structure and mechanical properties of the chitin component of the skeleton have been well-characterized. However, the protein component has remained a mystery. Here we used liquid chromatography-tandem mass spectrometry to sequence proteins extracted from two species of common Red Sea black corals following either one or two cleaning steps. We detected hundreds of proteins between the two corals, nearly 70 of which are each othersâ€™ reciprocal best BLAST hit. Unlike stony corals, only a few of the detected proteins were moderately acidic (biased toward aspartic and/or glutamic acid residues) suggesting less of a role for these types of proteins in black coral skeleton formation as compared to stony corals. No distinct chitin binding domains were found in the proteins, but proteins annotated as having a role in protein and chitin modifications were detected. Our results support the integral role of proteins in black coral skeleton formation, structure, and function.

### Sample Protocol
Proteins extracted from ground black coral skeleton cleaned only pre-grinding as well as skeleton cleaned again post-grinding were sequenced at the De Botton Protein Profiling Institute at the Weizmann Institute of Science. Skeletal protein samples were dissolved in 5% SDS and digested with trypsin using the S-trap method overnight at room temperature. The resulting peptides were analyzed using a nanoflow ultra-performance liquid chromatograph (nanoAcquity) coupled to a high resolution, high mass accuracy mass spectrometer (Fusion Lumos) fitted with a Symmetry C18 0.18*20mm trap column (Waters, Inc) for peptide trapping and then  a HSS T3 0.075*250 mm column (Waters, Inc.) for peptide separation using a gradient of 4-28% (80% acetonitrile, 0.1% Formic acid) for 150 minutes. Spray voltage was set to +2kV. The data were acquired in a Fusion Lumos by a Top Speed Data-Dependent Acquisition method with a cycle time of 3 s. An MS1 scan was performed in the Orbitrap at 120,000 resolution with a maximum injection time of 60 ms and the data were scanned between 300-1800 m/z. MS2 was selected using a monoisotopic precursor selection set to peptides, peptide charge states set to +2-+8 and dynamic exclusion set to 30 s. MS2 was performed using HCD fragmentation scanning in the Orbitrap, with the first mass set to 130 m/z at a resolution of 15,000. Maximum injection time was set to 60 ms with automatic gain control of 5x10-4 ions as a fill target.

### Data Protocol
The resulting data were searched against the protein databases derived from our de novo black coral transcriptomes described above using the Byonic search engine (Protein Metrics Inc.). A first search was carried out without any false discovery rate (FDR) filtering, to generate a focused database for a second search. The second search was set to 1% FDR, allowing fixed carbamidomethylation on C and variable oxidation on MW, deamidation on NQ and protein N-terminal acetylation.

### Publication Abstract
None

### Keywords
Deep sea, Coral, Bottom up, Skeleton

### Affiliations
Weizmann institute
Department of Marine Biology, Leon H. Charney School of Marine Sciences, University of Haifa, Mount Carmel, Haifa, 3498838, Israel

### Submitter
David Morgenstern

### Lab Head
Dr Tali Mass
Department of Marine Biology, Leon H. Charney School of Marine Sciences, University of Haifa, Mount Carmel, Haifa, 3498838, Israel


