### Accession
PXD009633

### Title
Proteome Profiling of PBMC-originated, iPSC-derived Lentoid Bodies at differentiation day 35

### Description
Differentiation of pluripotent stem cells into lentoid bodies is important for the understanding of the lens development and investigating the processes critical for lens morphogenesis. This Study was initiated to investigate a comprehensive proteome profiling of the peripheral blood mononuclear cell (PBMC)-originated, induced pluripotent stem cell (iPSC)-derived lentoid bodies through mass spectrometry-based protein sequencing. Briefly, a small aliquot of blood sample was ascertained to collect PBMCs that were reprogrammed to iPSCs using the Sendai-virus delivery system. The PBMC-originated, iPSCs were differentiated into lentoid bodies employing the “fried egg” method using feeder-free conditions. The quantitative real-time PCR (qRT-PCR) confirmed the expression of lens-associated markers, which exhibited at least an order magnitude increased expression in lentoid bodies at differentiation day 35. Subsequently, the total cellular protein was extracted from lentoid bodies at day 35, digested with trypsin, fractionated into 96 fractions and subjected to an mass spectrometry-based label-free quantitative proteomics. mass spectrometry-based proteome profiling revealed 9,717 proteins in iPSC-derived lentoid bodies at differentiation day 35. In here, we report a comprehensive proteome of PBMC-originated, iPSC-derived lentoid bodies at day 35, which will help in better understanding processes critical for the development of the ocular lens.

### Sample Protocol
Sample processing: Briefly, 10 ml of human peripheral blood was used to isolate PBMCs. Subsequently, the cryopreserved PBMCs were reprogramed into iPSCs using Sendai-virus delivery system Cytotune 2.0 kit (Life Technologies, Carlsbad, CA). Finally, the lentoid bodies were generated from human PBMC-originated iPSCs using “fried egg” method with little modifications. The differentiated lentoid bodies were harvested on day 35 for mass spectrometry-based proteome profiling. Protein extraction, and digestion: PBMC-originated, iPSC-derived lentoid bodies at differentiation day 35 were used for an mass spectrometry-based label-free quantitative proteomics. Thirty-six lentoid bodies at day 35 were pooled together and subjected to a mass spectrometry-based label-free proteomics. The frozen samples were homogenized using pestle and mortar and further lysed by sonicating in 8 M urea and 50 mM triethylammonium bicarbonate (TEAB). Protein lysate was centrifuged at 16,000g at 4 °C to exclude cell debris, and protein concentration was estimated using a bicinchoninic acid (BCA) assay. Total protein from the lentoid bodies at day 35 (3,000 µg) was reduced with 10 mM dithiothreitol at room temperature for one hour and alkylated with 30 mM iodoacetamide for 20 minutes in the dark. The protein sample was digested with LysC (1:100) by incubating at room temperature for three hours followed by overnight digestion at 37 °C using sequencing-grade trypsin (1:50) and finally desalted with C18 Sep-Pak. The peptides from the lentoid bodies at day 35 were fractionated by basic pH reversed-phase liquid chromatography into 96 fractions and subjected to a mass spectrometry-based label-free proteomics.  Mass spectrometry analysis: The fractionated peptides were analyzed on an Orbitrap Fusion Lumos Tribrid Mass Spectrometer coupled with the Easy-nLC 1200 nano-flow liquid chromatography system (Thermo Fisher Scientific). The peptides from each fraction were reconstituted in 15 μl 0.1% formic acid and all the 15 μl were loaded on an Acclaim PepMap100 Nano-Trap Column (100 μm × 2 cm, Thermo Fisher Scientific) packed with 5 μm C18 particles at a flow rate of 5 μl per minute. Peptides were resolved at 250 nl per minute flow rate using a linear gradient of 10% to 35% solvent B (0.1% formic acid in 95% acetonitrile) over 95 minutes on an EASY-Spray column (50 cm x 75 µm ID, Thermo Fisher Scientific) packed with 2 µm C18 particles, which was fitted with an EASY-Spray ion source that was operated at a voltage of 2.0 kV.  Mass spectrometry analysis was completed in a data-dependent manner with a full scan in the mass-to-charge ratio (m/z) range of 300 to 1800 in the “Top Speed” setting, three seconds per cycle. MS1 and MS2 were acquired for the precursor ions and peptide fragmentation ions and measured at a resolution of 120,000 at an m/z of 200. MS2 scan was acquired by fragmenting precursor ions using higher-energy collisional dissociation (HCD) method and detected at a mass resolution of 30,000, at an m/z of 200. Automatic gain control for MS1 was set to one million ions and for MS2 was set to 0.05 million ions. A maximum ion injection time was set to 50 milliseconds for MS1 and 100 milliseconds for MS2 (HCD was set to 32 with a stepped collision energy of 5% for MS2). Precursor isolation window was set to 1.6 m/z with 0.4 m/z offset. Dynamic exclusion was set to 30 seconds, and singly-charged ions were rejected. Internal calibration was carried out using the lock mass option (m/z 445.1200025) from ambient air.

### Data Protocol
Data analysis: MaxQuant (v1.5.3.8.) software was used for quantitation and identification. During MS/MS preprocessing, the top 12 peaks in each window of 100 m/z were selected for database search. The mass spectrometry data were then searched using Andromeda search algorithms against a UniProt (Swiss-Prot only) database (released on Apr 10, 2017) with common contaminant proteins. Search parameters included, a) trypsin as a proteolytic enzyme with up to two missed cleavages; b) first search peptide mass error tolerance of 20 ppm and the main search peptide mass error tolerance of 4 ppm; c) fragment mass error tolerance of 20 ppm;  d) carbamidomethylation of cysteine (+57.02146 Da) as a fixed modification: e) oxidation of methionine (+15.99492 Da) and protein acetyl (+ 42.01056 Da) on N-terminus as a dynamic modifications. The minimum peptide length was set to seven amino acids and the minimum number of peptide sequence per protein was set to two peptides. Peptides and proteins were filtered at 1 % false-discovery rate while the label-free quantification, the iBAQ, and the second peptides parameters were enabled. A minimum ratio count was set to two and both unique and razor peptides were used for quantification. After protein identification and quantification was completed, the protein table was imported into Perseus 1.5.2.6 software for the copy number calculation using Proteomic Ruler.

### Publication Abstract
Here, we report proteome profiling of peripheral blood mononuclear cell (PBMC)-originated, induced pluripotent stem cell (iPSC)-derived, lens-like organoids termed lentoid bodies at two differentiation time points. A small aliquot of the blood sample was ascertained to collect PBMCs that were reprogrammed to iPSCs. The PBMC-originated, iPSCs were differentiated to lentoid bodies employing the "fried egg" method. Quantitative real-time PCR (qRT-PCR) analysis revealed increased expression levels of lens-associated markers in lentoid bodies while transmission electron microscopy identified closely packed lens epithelial- and differentiating fiber-like cells in lentoid bodies. Total cellular protein was extracted from lentoid bodies at differentiation day 25 and mass spectrometry identified a total of 9,473 proteins. The low counts of crystallin proteins at differentiation day 25 prompted us to re-examine the proteome at differentiation day 35 as we reasoned that 10 additional days of differentiation will increase the crystallin count. However, we did not detect any substantial increase in crystallin protein counts at differentiation day 35. In conclusion, we report generation and proteome profiles of PBMC-originated, iPSC-derived lentoid bodies at multiple differentiation time points.

### Keywords
Proteome, Mass-spectrometry, Pbmc, Ipsc-derived lentoid bodies

### Affiliations
The Wilmer Eye Institute, Johns Hopkins University School of Medicine, Baltimore, Maryland 21287 USA
Johns Hopkins Wilmer Eye Institute

### Submitter
OM Genetics

### Lab Head
Dr S. Amer Riazuddin
The Wilmer Eye Institute, Johns Hopkins University School of Medicine, Baltimore, Maryland 21287 USA


