### Accession
PXD005011

### Title
Utx interactome in mouse myeloid progenitor cells

### Description
Immunoprecipitation of Utx followed by mass spectrometry analysis to identify Utx binding partners in mouse myeloid progenitor cells.

### Sample Protocol
Cells were lysed in lysis buffer (50 mM Tris-HCl pH=8, 150 mM NaCl, 0.1% NP-40, 1 mM EDTA), supplemented with 1 mM DTT, protease inhibitors (Sigma), and phosphatase inhibitors (Sigma). Anti-UTX antibody (GeneTex, GTX121246) was bound to 100ul of Dynabeads Protein G (Thermo Fisher Scientific) beads and incubated freshly extracted protein for 1.5 h at 4Â°C with rotation. After washing, samples were reduced with 5 mM TCEP (Sigma) in 1x NuPAGE LDS Sample Buffer (Thermo Fisher Scientific) and alkylated with 10 mM iodoacetamide (Sigma). The affinity purified material was electrophoresed in 4-12% Bis-Tris NuPAGE gels (Life Technologies). Gels were fixed and stained with Coomassie. Gel lanes were excised into five sections and IgG bands were discarded. Gel pieces were digested with trypsin and peptides extracted with 0.25% formic acid-50% acetonitrile and dried. Just before analysis the peptides were re-dissolved in 0.5% formic acid and analysed with on-line nano liquid chromatography (Ultimate 3000 RSLCnano System) tandem mass spectrometry on an LTQ Orbitrap Velos mass spectrometer.

### Data Protocol
Raw MS files were processed with Proteome Discoverer v 1.4 (ThermoFisher Scientific). Database searches were performed using Mascot (v 2.5, Matrix Science) with the mouse Swiss-Prot database (December 2015, 16942 sequences) supplemented with an in-house contaminant database. The search parameters were as follow: trypsin digestion, 2 missed cleavages, 10 ppm mass tolerance for precursor ions, 0.5 Da mass tolerance for fragment ions, variable modifications of carbamidomethyl (C), N-acetylation (protein), formyl (N-term), oxidation (M), deamidated (NQ), and pyro-glu (N-term Q). Database search results were further processed with Percolator  within Proteome Discoverer. Protein identification required at least one high-confidence peptide (FDR < 1% based on q-value) and a minimum Mascot protein score of 20.

### Publication Abstract
The histone H3 Lys27-specific demethylase UTX (or KDM6A) is targeted by loss-of-function mutations in multiple cancers. Here, we demonstrate that UTX suppresses myeloid leukemogenesis through noncatalytic functions, a property shared with its catalytically inactive Y-chromosome paralog, UTY (or KDM6C). In keeping with this, we demonstrate concomitant loss/mutation of KDM6A (UTX) and UTY in multiple human cancers. Mechanistically, global genomic profiling showed only minor changes in H3K27me3 but significant and bidirectional alterations in H3K27ac and chromatin accessibility; a predominant loss of H3K4me1 modifications; alterations in ETS and GATA-factor binding; and altered gene expression after Utx loss. By integrating proteomic and genomic analyses, we link these changes to UTX regulation of ATP-dependent chromatin remodeling, coordination of the COMPASS complex and enhanced pioneering activity of ETS factors during evolution to AML. Collectively, our findings identify a dual role for UTX in suppressing acute myeloid leukemia via repression of oncogenic ETS and upregulation of tumor-suppressive GATA programs.

### Keywords
Immunoprecipitation, Interactome, Utx, Protein interactions, Affinity purification

### Affiliations
Wellcome Trust Sanger Institute
Proteomic Mass Spectrometry Wellcome Trust Sanger Institute United Kingdom

### Submitter
Hendrik Weisser

### Lab Head
Dr Jyoti Sharma Choudhary
Proteomic Mass Spectrometry Wellcome Trust Sanger Institute United Kingdom


