### Accession
PXD025386

### Title
Characterization of PARP-6 function in knock out mice and patients with developmental delay

### Description
PARP-6, a member of a family of enzymes (17 in humans) known as poly-ADP- ribose polymerases (PARPs), is a neuronally enriched PARP. While previous studies from our group show that PARP-6 is a regulator of dendrite morphogenesis in hippocampal neurons, its function in the nervous system in vivo is poorly understood. Here, we describe the generation of a PARP-6 loss-of-function mouse model for examining the function of PARP-6 during neurodevelopment in vivo.  Using CRISPR-Cas9 mutagenesis, we generated a mouse line that expresses a PARP-6 truncated variant (PARP-6TR) in place of PARP-6WT. Unlike PARP-6WT, PARP-6TR is devoid of catalytic activity. Homozygous PARP-6TR do not exhibit obvious neuromorphological defects during development, but nevertheless die perinatally. This suggests that PARP-6 catalytic activity is important for postnatal survival. We also report PARP-6 mutations in six patients with several neurodevelopmental disorders, including microencephaly, intellectual disabilities, and epilepsy. The most severe mutation in PARP-6 (C563R) results in the loss of catalytic activity. Expression of the PARP-6C563R mutant in hippocampal neurons decreases dendrite morphogeneis. Taken together, these results suggest that PARP-6 is an essential gene in mice, and the loss of PARP-6 catalytic activity has detrimental effects on neuronal function in humans.

### Sample Protocol
We determined the PARP-6 interactome in neurons using BioID proximity labeling. We generated a PARP-6 variant in which a promiscuous biotin ligase (Myc-BirA*) was fused to the N-terminus of PARP-6WT. When cells are treated with media containing exogenous biotin, BirA* will biotinylate nearby proteins in a diffusion-limited process allowing for identification of proximal proteins using biotin affinity capture. We expressed Myc-BirA*-P2A-GFP (control) and Myc-BirA*âˆ’PARP-6WT-P2A-GFP in cortical neurons using lentiviruses, treated with biotin, and observed an increase in biotinylated proteins. To identify potential PARP-6 interactors, we captured biotinylated proteins using Neutravidin agarose, and performed on-bead trypsin digestion, and subjected resultant peptides to LC-MS/MS (in duplicate) analysis on an Orbitrap Fusion mass spectrometer.

### Data Protocol
Comet (v. 2016.01, rev. 2) was used to search MS2 Spectra against an June 2016 version of a uniprot FASTA protein database containing Rattus norvegicus sequences, concatenated sequence-reversed entries to estimate error thresholds and 179 common contaminant sequences and their reversed forms as well as the Mus musculus PARP6 sequence. The database processing was performed with python scripts available at https://github.com/pwilmart/fasta_utilities.git and Comet results processing used the PAW pipeline from https://github.com/pwilmart/PAW_pipeline.git.  Comet searches for all samples were performed with trypsin enzyme specificity. Monoisotopic parent ion mass tolerance was 1.50 Da. Monoisotopic fragment ion mass tolerance was 1.0005 Da. A static modification of +57.0215 Da was added to all cysteine residues and a variable modification of +15.9949 Da was added on Methionine residues. We used a linear discriminant transformation to improve the identification sensitivity from the Comet analysis. Comet scores combined into linear discriminant function scores, and discriminant score histograms created separately for each peptide charge state (2+, 3+, and 4+). Separate histograms were created for matches to forward sequences and for matches to reversed sequences for all peptides of 7 amino acids or longer. The score histograms for reversed matches were used to estimate peptide false discovery rates (FDR) and set score thresholds for each peptide class. The overall peptide FDR for the analysis was 1.87%, and the overall protein FDR was less than 1% (125 unique proteins with no reversed sequence matches).

### Publication Abstract
PARP6, a member of a family of enzymes (17 in humans) known as poly-ADP-ribose polymerases (PARPs), is a neuronally enriched PARP. While previous studies from our group show that Parp6 is a regulator of dendrite morphogenesis in rat hippocampal neurons, its function in the nervous system in vivo is poorly understood. Here, we describe the generation of a <i>Parp6</i> loss-of-function mouse model for examining the function of Parp6 during neurodevelopment in vivo. Using CRISPR-Cas9 mutagenesis, we generated a mouse line that expressed a Parp6 truncated variant (Parp6<sup>TR</sup>) in place of Parp6<sup>WT</sup>. Unlike Parp6<sup>WT</sup>, Parp6<sup>TR</sup> is devoid of catalytic activity. Homozygous Parp6<sup>TR</sup> do not exhibit obvious neuromorphological defects during development, but nevertheless die perinatally. This suggests that Parp6 catalytic activity is important for postnatal survival. We also report PARP6 mutations in six patients with several neurodevelopmental disorders, including microencephaly, intellectual disabilities, and epilepsy. The most severe mutation in <i>PARP6</i> (C563R) results in the loss of catalytic activity. Expression of Parp6<sup>C563R</sup> in hippocampal neurons decreases dendrite morphogenesis. To gain further insight into <i>PARP6</i> function in neurons we also performed a BioID proximity labeling experiment in hippocampal neurons and identified several microtubule-binding proteins (e.g., MAP-2) using proteomics. Taken together, our results suggest that PARP6 is an essential microtubule-regulatory gene in mice, and that the loss of PARP6 catalytic activity has detrimental effects on neuronal function in humans.

### Keywords
Bioid, Adp-ribosylation, Marylation, Parp-6; nervous system, Spectral counts, Dendritic branching, Orbitrap fusion

### Affiliations
Department of Physiology and Pharmacology Oregon Health & Science University Portland, Oregon 97210, USA
OHSU

### Submitter
Phillip Wilmarth

### Lab Head
Dr Michael S. Cohen
Department of Physiology and Pharmacology Oregon Health & Science University Portland, Oregon 97210, USA


