### Accession
PXD033254

### Title
PhosphoProtein-Peptide Turnover Profiling, exploratory

### Description
We combine pulsed stable isotope labeling of amino acids (pSILAC) with enrichment of phosphorylated peptides in a method we term PhosphoProtein-Peptide Turnover Profiling (PhosphoPPToP). We apply the method to HeLa cells to find peptides whose clearance rates differ from the protein median. Using theoretical modelling, we show that readouts from PPToP are primarily defined by the wiring of the PTM-modification network and not by effects of the PTM on a protein's proteolytic stability. Thus PPToP can be used to identify PTMs occurring, e.g., preferentially early or late in a protein's lifetime. This enables identification of, e.g., protein maturation or protein complex assembly intermediates.

### Sample Protocol
Sample preparation and LC-MS/MS. Cells were washed twice with ice-cold PBS and scraped on ice into lysis buffer (6M Urea, 100 mM Hepes (pH8.5), 5 mM Tris(2-carboxyethyl)phosphine, 30 mM chloroacetamide, 4 mM MgCl2, 2 mM NaVO4, 2 mM NaF, 2 mM Na-pyrophosphate, 1% sodium deoxycholate, 1% Triton X-100, 1x Complete protease inhibitor cocktail (Roche), 1x PhosStop phosphatase inhibitor cocktail (Roche), 0.5% Benzonase (Merck, 70746)), diluted 1:1 with dilution buffer (6M Urea, 100 mM Hepes (pH8.5), 4 mM MgCl2), and sonicated at +4Ã‚Â°C in a Bioruptor Plus, 45 cycles, 30 s on / 30 s off (Diagenode). Lysates were cleared by centrifugation, 16 000 g, at 4Ã‚Â°C for 60 min and supernatants frozen to -80 Ã‚Â°C before continuing. Cleared supernatants were thawed at room temperature (RT) and nucleic acids digested by adding 0.5% fresh Benzonase and incubating 1 h at RT, after which EDTA was raised to 25 mM and SDS to 2%. Protein was precipitated in a 4:4:1 lysate:methanol:chloroform mixture, centrifuged 10 min at 4000 g, RT, and the resulting protein precipitate extracted and washed twice with 70% EtOH in a sonicator water bath. Protein was resuspended to 2.5 mg/ml into digestion buffer (100 mM Hepes (pH8.5), 5 mM Tris(2-carboxyethyl)phosphine, 30 mM chloroacetamide, 1% sodium deoxycholate), TPCK-Trypsin (Thermo, 20233) added to 100 ?g/ml and incubated on an end-over-end shaker overnight at RT. Digested peptides were desalted on SepPak columns using gravity flow (Waters, WAT054945), washed twice with 0.1% trifluoroacetic acid (TFA), eluted with 40% acetonitrile (ACN), and resulting peptides lyophilized. Phosphopeptides were enriched essentially as previously described (Potel et al., 2018, 2021) using a ProPac IMAC-10 column (Thermo, 063276) loaded with Fe3+ on a Dionex Ultimate 3000 HPLC system. The phosphopeptide-containing eluate as well as the flow-through were collected and lyophilized. High-pH peptide prefractionation: 5% of the flow-through ("total") was taken up in 20 mM ammonium formate (pH 10) and prefractionated first into 29 fractions on a 1200 Infinity HPLC (Agilent) using high-pH reversed-phase chromatography (running buffer A: 20 mM ammonium formate pH 10; elution buffer B: ACN) on an X-bridge column (2.1 x 10 mm, C18, 3.5 Ã‚Âµm, Waters). Fractions were then pooled across to generate 12 fractions and vacuum dried. Phosphopeptides were fractionated manually using in-house packed C18 microcolumns as previously described (Potel et al., 2021) into seven fractions. For LC-MS/MS analysis, peptides were reconstituted in 0.1% FA, 4% ACN and analyzed by nanoLC-MS/MS on an Ultimate 3000 RSLC (Thermo Fisher Scientific) connected to a Fusion Lumos Tribrid (Thermo Fisher Scientific) mass spectrometer, using an Acclaim C18 PepMap 100 trapping cartridge (5Ã‚Âµm, 300 Ã‚Âµm i.d. x 5 mm, 100 Ãƒâ€¦) (Thermo Fisher Scientific) and a nanoEase M/Z HSS C18 T3 (100A, 1.8 Ã‚Âµm, 75 Ã‚Âµm x 250 mm) analytical column (Waters). Solvent A: aqueous 0.1% FA; Solvent B: 0.1% FA in ACN (all solvents LC-MS grade from Fisher Scientific). LC-MS/MS analysis parameters for total proteome and phosphofraction were as follows: Peptides were loaded on the trapping cartridge using solvent A for 4 min (3 min for phospho) with a flow of 30 Ã‚Âµl/min. Peptides were separated on the analytical column at 40 Ã‚Â°C with a constant flow of 0.3 Ã‚Âµl/min applying a 100 min gradient of 42Ã¢â‚¬â€œ25% of solvent B in A, followed by a 5 min gradient (25Ã¢â‚¬â€œ40%), and a 4 min washing step at 85% solvent B (both total and phospho). Peptides were directly analyzed in positive ion mode with a spray voltage of 2.4 kV and an ion transfer tube temperature of 275 Ã‚Â°C (both total and phospho). Full scan MS spectra with a mass range of 300Ã¢â‚¬â€œ1500 m/z (375Ã¢â‚¬â€œ1500 m/z for phospho) were acquired on the orbitrap using a resolution of 120 000 (60 000 for phospho) with a maximum injection time of 50 ms (20 ms for phospho) and Normalized AGC Target of 50% (Standard for phospho). Data-dependent acquisition was used with a cycle time of 2 s (3 s for phospho). Precursors were isolated on the quadrupole with an intensity threshold of 1e3 (2e5 for phospho), charge state filter of 2Ã¢â‚¬â€œ7, and an isolation window of 1.2 m/z (1.4 m/z for phospho). Precursors were fragmented using HCD at 30% (32% for phospho) collision energy. For the total proteome: MS/MS spectra were acquired on the ion trap, with a maximum injection time of 50 ms, and a dynamic exclusion window of 45 s. For the phosphoenriched fraction, MS/MS spectra were acquired on the orbitrap, at 30 000 resolution, a maximum injection time of 75 ms, and a dynamic exclusion window of 20 s.

### Data Protocol
Analysis of MS raw files: For PPToP MS raw files were processed using MaxQuant (version 1.6.4.0) (Cox and Mann, 2008) using a reference human proteome (uniprot Proteome ID: UP000005640, downloaded 9.6.2020). Data was processed separately for total and phosphoenriched samples, but for each all time points, fractions, and replicates were run together. Default search parameters were used, except as follows: multiplicity: 2; Heavy channel: Arg10, Lys8; variable modifications: Acetyl (Protein N-term), Oxidation (M), and only for the phosphofraction: Phospho (STY); fixed modifications: Carbamidomethyl (C); maximum number of modifications per peptide: 5; maximum missed cleavage sites: 2 (3 for phospho); LFQ: none; re-quantify: unchecked; match between run: checked.

### Publication Abstract
Post-translational modifications (PTMs) regulate various aspects of protein function, including degradation. Mass spectrometric methods relying on pulsed metabolic labeling are popular to quantify turnover rates on a proteome-wide scale. Such data have traditionally been interpreted in the context of protein proteolytic stability. Here, we combine theoretical kinetic modeling with experimental pulsed stable isotope labeling of amino acids in cell culture (pSILAC) for the study of protein phosphorylation. We demonstrate that metabolic labeling combined with PTM-specific enrichment does not measure effects of PTMs on protein stability. Rather, it reveals the relative order of PTM addition and removal along a protein's lifetime-a fundamentally different metric. This is due to interconversion of the measured proteoform species. Using this framework, we identify temporal phosphorylation sites on cell cycle-specific factors and protein complex assembly intermediates. Our results thus allow tying PTMs to the age of the modified proteins.

### Keywords
Post-translational modification, Phosphorylation, Mass spectrometry, Protein turnover, Protein maturation

### Affiliations
Genome Biology Unit and Structural and Computational Biology Unit, European Molecular Biology Laboratory, 69117 Heidelberg, Germany
Genome Biology Unit and Proteomics Core Facility, European Molecular Biology Laboratory, 69117 Heidelberg, Germany

### Submitter
Henrik Hammarén

### Lab Head
Dr Mikhail Savitski
Genome Biology Unit and Proteomics Core Facility, European Molecular Biology Laboratory, 69117 Heidelberg, Germany


