### Accession
PXD033351

### Title
Pan-cancer tumor-specific antigens and membrane targets from transposable elements

### Description
Cryptic promoters within transposable elements (TEs) are transcriptionally reactivated in tumors to create novel TE-gene chimeric transcripts, which can produce immunogenic antigens. We performed the most comprehensive screen to date for these TE exaptation events in 33 TCGA tumor types, 675 cancer cell lines, and 11,686 GTEx adult tissue transcriptomes and identified 1,068 TE-exapted candidates with the potential to generate shared tumor-specific TE-chimeric antigens (TS-TEAs). Resultant whole lysate and HLA-pulldown mass spectrometry data confirmed that TS-TEAs are presented on cancer cell surfaces. In addition, we highlight the tumor-specific membrane proteins transcribed from TE promoters that can expose novel epitopes on the extracellular surface of cancer cells. Here, we showcase the high pan-cancer prevalence of TS-TEAs and atypical membrane proteins that can be therapeutically exploited through immunotherapy approaches.

### Sample Protocol
We adopted a published protocol by Bassani-Sternberg et al. (Methods in Molecular Biology 2018 and Curr. Opin. Immunol 2016) to generate HLA-I antigen pull-down samples with couple exceptions mentioned below. As per published protocol, anti-HLA-I antibodies were collected from W6/32 (ATCC HB-95) growth medium and were crosslinked to Protein A-sepharose 4B beads (ThermoFisher Scientific, 101041) with dimethylpimelimidate. We harvested and froze down cell line samples until time of lysis. Roughly 500-1,000 million cells were lysed with ice-cold modified lysis buffer (0.3% sodium deoxycholate, 0.75% IGEPAL CA-630, 0.2mM iodoacetamide, 1mM EDTA, 1:200 Protease Inhibitors Cocktail, 1mM Phenyl-methylsulfonyl fluoride, 1% octyl- β-D glucopyranoside in PBS) on ice for 1 hour. The samples were slightly vortexed every 10 minutes to maximize lysis efficiency. The samples were centrifuged at 21,000x g at 4ºC for 1 hour to pellet large cell debris and cell nuclei. The lysate supernatant was transferred to Poly-Prep chromatography columns (Bio-Rad, 7311550) with 500ul Protein A-sepharose 4B beads to remove endogenous antibodies. The endogenous antibody-depleted lysate was then transferred to Poly-Prep chromatography columns containing 1ml of crosslinked W6/32-proteinA beads. The flow-through was collected and loaded to the same column for a total of three flow throughs. Then the HLA-antigen bound beads were washed and HLA-I antigens were collected by acetic acid elution. Then the antigen samples were completely dried using a vacuum centrifuge (ThermoFisher Scientific, SPD1030PI) and stored in -20C until subsequent LC-MS/MS analysis. LC-MS/MS analysis was carried out on an Orbitrap Fusion Lumos (Thermo Fisher Scientific, San Jose, CA) mass spectrometer coupled with a Dionex Ultimate 3000 RSLCnano HPLC (Thermo Fisher Scientific, San Jose, CA). The peptide separation was carried out on a C18 column (Fritted Glass Column, 25 cm × 75 μm, Reprosil-Pur 120 C18-AQ, 1.9 μm, purchased from ESI source solution) at a flow rate of 0.3 μl/min and the following gradient: Time = 0–4 min, 2% B isocratic; 4–8 min, 2–10% B; 8–83 min, 10–25% B; 83–97 min, 25–50% B; 97–105 min, 50–98%. Mobile phase consisted of A, 0.1% formic acid; mobile phase B, 0.1% formic acid in acetonitrile. The instrument was operated in the data-dependent acquisition mode in which each MS1 scan was followed by Higher-energy collisional dissociation (HCD) MS/MS scan of as many precursor ions in 2 second cycle (Top Speed method). The mass range for MS1 was set to 300 to 1800 m/z with a resolution of 120,000 (200 m/z) and the automatic gain control (AGC) target set to 1,000,000 ions with a maximum fill time of 50 ms. For precursor selection, ions with charge state of 1 to 4 were selected. For MS/MS, the selected precursors were fragmented in the Orbitrap using an isolation window of 1.6 m/z, a resolution of 30,000 (200 m/z), and a maximum fill time of 54 ms. Fragmentation energy in HCD MS/MS for charge state of 1 was set at higher level (32%) as opposed to 2 to 4 (27%) for more complete fragmentation. Dynamic exclusion was performed with a repeat count of 1, exclusion duration of 15 s, and a minimum MS ion count for triggering MS/MS set to 10000 counts.

### Data Protocol
Raw files from our HLA-pulldown samples were analyzed using MaxQuant Version 1.6.3.4. The parameters used in the proteomics search different from the default were the following: unspecific enzyme digestion, no protein-level FDR since we were interested in peptides, peptide FDR of 5%, peptide length limit between 8 and 15 AA, and maximum peptide mass of 1500 Da. In addition, a custom proteome database was used consisting of the Uniprot reference database, the sequences of potential antigenic peptides from our analysis, and the potential neoantigen sequences from 4 cell lines we analyzed from the TRON Cell Line Portal. Decoy sequences and contaminants were removed before performing subsequent identification analysis.

### Publication Abstract
None

### Keywords
Human, Antigen, Lc-ms/ms, Mhc pulldown, Transposable element, Cancer

### Affiliations
Department of Genetics, Washington University School of Medicine, St Louis, MO, USA The Edison Family Center for Genome Sciences and Systems Biology, Washington University School of Medicine, St. Louis, MO, USA McDonnell Genome Institute, Washington University School of Medicine, St. Louis, MO, USA
Washington University in St. Louis

### Submitter
Ju Heon Maeng

### Lab Head
Dr Ting Wang
Department of Genetics, Washington University School of Medicine, St Louis, MO, USA The Edison Family Center for Genome Sciences and Systems Biology, Washington University School of Medicine, St. Louis, MO, USA McDonnell Genome Institute, Washington University School of Medicine, St. Louis, MO, USA


