### Accession
PXD033143

### Title
Microproteomics workflows for quantitative proteome analysis of single Caenorhabditis elegans nematodes

### Description
Miniaturization of sample preparation including omissible manual sample handling steps is key for reproducible nanoproteomics as material is often restricted to only hundreds of cells or single model organisms. Here, we demonstrate a highly sensitive digital microfluidics (DMF)-based sample preparation workflow making use of single-pot solid-phase enhanced sample preparation (SP3) in combination with high-field asymmetric-waveform ion mobility spectrometry (FAIMS), and fast and sensitive ion trap detection on an Orbitrap tribrid MS system. Compared to a manual in-tube SP3-supported sample preparation, the number of peptides identified, the achievable standard deviations between replicates, and the number of proteins identified as differentially abundant were markedly increased. We identified up to 5,000 proteins from single nematodes. Moreover, label-free quantification of protein changes in single Caenorhabditis elegans treated with a heat stimulus yielded 45 differentially abundant proteins when compared to the untreated control, demonstrating the potential of this technology for low-input proteomics studies.

### Sample Protocol
• Synchronized Caenorhabditis elegans (C. elegans) at L4 stage of development • Incubation at 33 °C (heat treatment) or 20 °C (control) for one hour • Isolation of single specimens in 1 µL M9 buffer and flash-freezing in liquid nitrogen • Sample preparation via digital microfluidics (DMF) or in-tube, using highly similar protocols • 50 µL 0.08% w/v Pluronic F127 added to each nematode • Aspiration volume of 1 µL for DMF and 3 µL for the in-tube method • Lysis and reduction for 50 min. at room temperature using 6 mg/mL dithiothreitol, 5 mol/L urea, 0.1% w/v Tetronic 90R4 and 1% w/v NP-40 • Alkylation for 50 min. at room temperature in the dark using 25 mmol/L iodoacetamide, 0.1% w/v Tetronic 90R4 and 50 mmol/L borate buffer • Volumes of lysis and alkylation agent: 1 µL for DMF, 3 µL for the in-tube method • Harsh reagents removed via single-pot solid phase-enhanced sample preparation (SP3) • SP3 with 2 µL bead suspension per sample (10 µg/µL beads, to equal parts hydrophilic and hydrophobic, in 0.08% w/v Pluronic F127) • Beads pelleted and immobilized using a magnetic lens (DMF) or a rod magnet (in-tube) to remove supernatant • Protein binding to beads for 30 min. using 3x the sample volume of neat acetonitrile (ACN), supernatant then removed • Consecutive washes with 70% ethanol (4.5 µL for DMF, 100 µL for in-tube) and neat ACN (DMF: 4.5 µL, in-tube: 50 µL) • Overnight digestion using trypsin/Lys. C mix at 4 ng/µL, 0.01% w/w 3-Dodecyloxypropyl-1-β-D-maltopyranoside and 25 mmol/L borate buffer (DMF: 2 µL, in-tube: 6 µL) • Transfer to 100 µL glass LC inserts containing 1 µL (DMF) or 3 µL (in-tube) of 5% formic acid (FA) • Only DMF: Tip flushed with 1.5 µL of 0.05% FA to maximize sample yield • Injection volumes: 5 µL for DMF, 9 µL for the in-tube method • Injection onto Ultimate 3000 RSLCnano UHPLC (Thermo Fisher Scientific, Dreieich, Germany) system at a constant flow rate of 200 nL/min. • Starting eluent composition of 97% eluent A and 3% eluent B (A: 0.05% FA; B: 80% ACN, 0.04% FA), maintained for 2 min. • 90-minute linear gradient from 3-35% eluent B, then 30-minute linear gradient from 35-35% eluent B • Eluent B proportion increased to 90% within one min. and maintained for another 12 min. • Equilibration for 15 min. to restore starting concentrations • Separation on Aurora Series emitter-integrated column (25 cm length, 75 µm inner diameter, 1.6 µm particle diameter, 100 Å pore size; IonOpticks, Fitzroy, Victoria, Australia) • Use of FAIMS Pro interface operated at two compensation voltages (CV) of -50 and -70 V • MS/MS on Orbitrap Fusion Lumos mass spectrometer (Thermo Fisher Scientific, Bremen, Germany) operated in positive polarity with a maximum injection time of 50 ms and a resolution of 60,000 • Precursor ions selected in orbitrap (OT), fragments detected in linear ion trap (IT); operation in OTIT mode

### Data Protocol
• Raw data processed using Proteome Discoverer v.2.2.0.388 • Search against FASTA databases for C. elegans, E. coli strain B/BL21-DE3, as well as the cRAP list of common laboratory contaminants, all downloaded from UniProt on February 17th, 2017 • Sequest HT algorithm used with Minora Feature Detector and Percolator node • Sequest HT parameters: Enzyme: Trypsin [full], Precursor Mass Tolerance: 10 ppm, Fragment Mass Tolerance: 0.6 Da, Fixed modification: Carbamidomethyl, Dynamic modification: Oxidation [O] • Percolator parameters: Maximum Delta Cn: 0.05, Maximum Rank: 0, Target FDR (strict): 0.01, Target FDR (relaxed): 0.05, Validation based on: q-value • Spectrum Files RC parameters: Dynamic Modification: Oxidation [M], Fixed modification: Carbamidomethyl • Feature Mapper parameters: Parameter tuning: Fine, RT alignment: True, Mass tolerance: 10 ppm, Maximum RT shift: 20 min. • Precursor Ions Quantifier: Peptides To Use: Unique, Precursor Abundance Based On: Intensity, Min # Replicate Features: 0, Normalization Mode: Total Peptide Amount, Ratio Calculation: Pairwise Ratio Based • Proteins filtered to include only entries for C. elegans • Proteins required to be represented by at least two peptides, of which one was unique

### Publication Abstract
Miniaturization of sample preparation, including omissible manual sample handling steps, is key for reproducible nanoproteomics, as material is often restricted to only hundreds of cells or single model organisms. Here, we demonstrate a highly sensitive digital microfluidics (DMF)-based sample preparation workflow making use of single-pot solid-phase enhanced sample preparation (SP3) in combination with high-field asymmetric-waveform ion mobility spectrometry (FAIMS), and fast and sensitive ion trap detection on an Orbitrap tribrid MS system. Compared to a manual in-tube SP3-supported sample preparation, the numbers of identified peptides and proteins were markedly increased, while lower standard deviations between replicates were observed. We repeatedly identified up to 5000 proteins from single nematodes. Moreover, label-free quantification of protein changes in single <i>Caenorhabditis elegans</i> treated with a heat stimulus yielded 45 differentially abundant proteins when compared to the untreated control, highlighting the potential of this technology for low-input proteomics studies. LC-MS data have been deposited to the ProteomeXchange Consortium with the data set identifier PXD033143.

### Keywords
Low-input proteomics, Caenorhabditis elegans, Nanoproteomics, Microproteomics, Ion trap, Faims

### Affiliations
Institute for Experimental Medicine, Kiel University, Kiel, Germany
Systematic Proteome Research & Bioanalytics, University of Kiel

### Submitter
Andreas Tholey

### Lab Head
Dr Andreas Tholey
Institute for Experimental Medicine, Kiel University, Kiel, Germany


