### Accession
PXD009225

### Title
Phosphoproteome of Francicella Tularensis

### Description
The pathogenic bacterium Francisella tularensis possesses a non-canonical T6SS encoded on the Francisella pathogenicity island (FPI) that is essential for efficient phagosomal escape and access to the cytosolic compartment of infected host cells. Using a global and site-specific phosphoproteomic analysis of F. novicida, we identified a phosphorylated form of IglB, which constitutes with iglA the outer sheath of the T6SS. We show here that substitution of the unique phosphorylated tyrosine (Y139) of IglB to alanine or to the non-phosphorylatable analogue phenylalanine prevents the formation of the sheath-like structures and impairs normal bacterial phagosomal escape. We propose that IglB phosphorylation is involved in the dynamics of assembly-disassembly of the sheath.

### Sample Protocol
For proteomic analysis, Francisella tularensis subsp. novicida and Francisella tularensis subsp. holartica LVS were analysed in three independent biological replicates. Protein concentration was determined by DC assay (Biorad, CA, USA) according to manufacturer’s instructions. Estimated 1.2 mg of proteins for each biological replicates were digested following a FASP protocol 70 slightly modified. Briefly, proteins were reduced using 100 mM dithiothreitol in 50mM ammonium bicarbonate for 1h at 60°C. Proteins were then split into four samples of 300 µg and applied on 30 kDa MWCO centrifugal filter units (Microcon, Millipore, Germany, Cat No MRCF0R030). Samples were mixed with 200 µL of 8M urea in 50 mM ammonium bicarbonate (UA buffer) and centrifuged for 20 min at 15,000 x g. Filters were washed with 200 µL of UA buffer. Proteins were alkylated for 30min by incubation in the dark at room temperature with 100 µL of 50 mM iodoacetamide in UA buffer. Filters were then washed twice with 100 µL of UA buffer (15,000 x g for 15 min) followed by two washes with 100 µL of 50 mM ammonium bicarbonate (15,000 x g for 10 min). Finally, sequencing grade modified trypsin (Promega, WI, USA) was added to digest the proteins in 1:50 ratio for 16 h at 37°C. Peptides were collected by centrifugation at 15,000 x g for 10min followed by one wash with 50mM ammonium bicarbonate and vacuum dried.  Phosphopeptides enrichment by titanium dioxide (TiO2) and phosphopeptides purification by graphite carbon (GC). Phosphopeptide enrichment was carried out using a Titansphere TiO2 Spin tips (3 mg/200 μL, Titansphere PHOS-TiO, GL Sciences Inc, Japan) on estimated 1,2 mg of digested proteins for each biological replicate. Briefly, the TiO2 Spin tips were conditioned with 20 µL of solution A (80% acetonitrile, 0,1% TFA), centrifuged at 3,000 x g for 2min and equilibrated with 20µL of solution B (75% acetonitrile, 0,075% TFA, 25% lactic acid) followed by centrifugation at 3,000 x g for 2 min. Peptides were dissolved in 10 µL of 2% TFA, mixed with 100 µL of solution B and centrifuged at 1,000 x g for 10min. Sample was applied back to the TiO2 Spin tips two more times in order to increase the adsorption of the phosphopeptides to the TiO2. Spin tips were washed with, sequentially, 20 µL of solution B and two times with 20 µL of solution A. Phosphopeptides were eluted by the sequential addition of 50 µL of 5% NH4OH and 50 µL of 5% pyrrolidine. Centrifugation was carried out at 1,000 x g for 5 min. Phosphopeptides were further purified using GC Spin tips (GL-Tip, Titansphere, GL Sciences Inc, Japan). Briefly, the GC Spin tips were conditioned with 20 µL of solution A, centrifuged at 3,000 x g for 2 min and equilibrated with 20 µL of solution C (0,1% TFA in HPLC-grade water) followed by centrifugation at 3,000 x g for 2 min. Eluted phosphopeptides from the TiO2 Spin tips were added to the GC Spin tips and centrifuged at 1,000 x g for 5 min. GC Spin tips were washed with 20 µL of solution C. Phosphopeptides were eluted with 70 µL of solution A (1,000 x g for 3 min) and vacuum dried.

### Data Protocol
The MS files were processed with the MaxQuant software version 1.5.3.30 and searched with Andromeda search engine against the UniProtKB/Swiss-Prot F. tularensis subsp. novicida database (release 28-04-2014, 1719 entries) and the UniProtKB/Swiss-Prot F. tularensis subsp. holartica database (release 25-06-2015, 2382 entries). To search parent mass and fragment ions, we set an initial mass deviation of 4.5 ppm and 0.5 Da respectively. The minimum peptide length was set to 7 aminoacids and strict specificity for trypsin cleavage was required, allowing up to two missed cleavage sites. Carbamidomethylation (Cys) was set as fixed modification, whereas oxidation (Met), N-term acetylation and phosphorylation (Ser, Thr, Tyr) were set as variable modifications. The match between runs option was enabled with a match time window of 0.7 min and an alignment time window of 20 min. The false discovery rates (FDRs) at the protein and peptide level were set to 1%. Scores were calculated in MaxQuant as described previously (1). The reverse and common contaminants hits were removed from MaxQuant output.  Statistical and bioinformatic analysis, including heatmaps and clustering, were performed with Perseus software (version 1.4.1.3) freely available at www.perseus-framework.org (2).  The phosphopeptides output table and the corresponding logarithmised intensities were used for phosphopeptide analysis. The phosphopeptide table was expanded to separate individual phosphosites, and we kept all sites identified at least once of the three independent replicates of the analysis of F. novicida.  All bioinformatics analysis of phosphosites was performed in Perseus 1.6.0.7. For functional class annotation, we also integrated COG and EggNog databases in Perseus 1.6.0.7.

### Publication Abstract
None

### Keywords
Francicella tularensis, Phospho

### Affiliations
Necker proteomics, INSERM
Proteomics Laboratory  Structure Federative Recherche Necker, INserm US24Paris , France

### Submitter
Chiara guerrera

### Lab Head
Dr GUerrera I C
Proteomics Laboratory  Structure Federative Recherche Necker, INserm US24Paris , France


