### Accession
PXD013496

### Title
SILAC-based quantitative proteomics of endoplasmic reticulum stress

### Description
ER stress and the unfolded protein response (UPR) involve extensive proteome remodeling in many cell compartments. However, a comprehensive analysis has been lagging due to technological limitations. Here we employ SILAC-based proteomics to quantify over 6,200 proteins at increasing concentrations of tunicamycin in HeLa cells. We further compare the effects of tunicamycin to those of thapsigargin and DTT, both activating the UPR through different mechanisms. The systematic description of the proteome-wide expression changes following proteostatic stress is a resource for the scientific community, which enables to discover novel players involved the pathophysiology of disorders linked to proteostasis. Here, we identified 38 proteins, not previously linked to the UPR, whose expression increases, of which 15 would likely remediate ER stress, and the remainder may contribute to pathological outcomes. Unexpectedly, we see few strongly downregulated proteins, despite expression of the pro-apoptotic transcription factor CHOP, suggesting that IRE1-dependent mRNA decay (RIDD) has a limited contribution to ER-stress mediated cell death in our system.

### Sample Protocol
Spike-in SILAC mix, FASP. 200 Î¼g of protein was loaded on the filter, and SDS was replaced by washing two to three times with buffer containing 8 m urea. The proteins were subsequently alkylated using iodoacetamide, and the excess reagent was removed by filtration. The reduced and alkylated proteins were digested using trypsin, eptides  were desalted using a C18 membrane in StageTips.

### Data Protocol
The MaxQuant software (version 1.5.3.2) was used for the analysis of raw files. Peak lists were searched against the human UniProt FASTA database (88,976 entries) and a common contaminants database (247 entries) using the Andromeda search engine [49]. False discovery rate was set to 1% for peptides (minimum length of 7 amino acids) and proteins and was determined by searching a reverse database. A maximum of two missed cleavages were allowed in the database search. Peptide identification was performed with an allowed initial precursor mass deviation up to 7 ppm and an allowed fragment mass deviation 20 ppm. The Match between runs option in MaxQuant was activated

### Publication Abstract
The unfolded protein response (UPR) involves extensive proteome remodeling in many cellular compartments. To date, a comprehensive analysis of the UPR has not been possible because of technological limitations. Here, we employ stable isotope labeling with amino acids in cell culture (SILAC)-based proteomics to quantify the response of over 6200 proteins to increasing concentrations of tunicamycin in HeLa cells. We further compare the effects of tunicamycin (5&#x2005;&#xb5;g/ml) to those of thapsigargin (1&#x2005;&#xb5;M) and DTT (2&#x2005;mM), both activating the UPR through different mechanisms. This systematic quantification of the proteome-wide expression changes that follow proteostatic stress is a resource for the scientific community, enabling the discovery of novel players involved in the pathophysiology of the broad range of disorders linked to proteostasis. We identified increased expression in 38 proteins not previously linked to the UPR, of which 15 likely remediate ER stress, and the remainder may contribute to pathological outcomes. Unexpectedly, there are few strongly downregulated proteins, despite expression of the pro-apoptotic transcription factor CHOP, suggesting that IRE1-dependent mRNA decay (RIDD) has a limited contribution to ER stress-mediated cell death in our system.

### Keywords
Proteomics, Tunicamycin, Silac, Unfolded protein response, Endoplasmic reticulum stress

### Affiliations
MPI Biochemistry
Max-Planck-Institute of Biochemistry, Martinsried, Germany. Department of Biomedical Sciences, University of Padova, Italy

### Submitter
Marta Murgia

### Lab Head
Dr Marta Murgia
Max-Planck-Institute of Biochemistry, Martinsried, Germany. Department of Biomedical Sciences, University of Padova, Italy


