### Accession
PXD006542

### Title
Enterococcus faecalis biofilm proteome

### Description
The biofilm proteome profile of endodontic and systemic pathogen E. faecalis has been analysed in this study to identify markers associated with its biofilm formation. Strong and weak biofilm forming E. faecalis clinical isolates were compared with two standard ATCC strains of E. faecalis in order to elucidate the biological pathways associated with the biofilm formation capability of E. faecalis.

### Sample Protocol
100 Âµg of biofilm proteins extracted from the four E. faecalis strains in duplicates were subjected to trypsin digestion and labelled with iTRAQ Reagent 8-plex kit (AB SCIEX). The samples were then acidified and subjected to strong cation-exchange chromatography to remove all interfering substances. The  eluted fractions eluted were desalted using Sep-Pak C18 cartridges (Water) followed by 2D LC-MS/MS analysis using 1290 Infinity LC system coupled with 1260 Fraction Collector.

### Data Protocol
Peptide identification and quantification was carried out on the ProteinPilot 5.0 software Revision 4769 (AB SCIEX) using the ParagonTM database search algorithm (5.0.0.0.4767) and the integrated false discovery rate (FDR) analysis function. The data were searched against a protein sequence database downloaded from UniProtKB for E. faecalis 29212. A strict unused confidence score of >1.3 corresponding to a peptide confidence level of 95% was used as the selection criteria for proteins.

### Publication Abstract
<i>Enterococcus faecalis</i> is a bacterial pathogen associated with both endodontic and systemic infections. The biofilm formation ability of <i>E. faecalis</i> plays a key role in its virulence and drug resistance attributes. The formation of <i>E. faecalis</i> biofilms on implanted medical devices often results in treatment failure. In the present study, we report protein markers associated with the biofilm formation ability of <i>E. faecalis</i> using iTRAQ-based quantitative proteomics approach. In order to elucidate the biofilm-associated protein markers, we investigated the proteome of strong and weak biofilm-forming <i>E. faecalis</i> clinical isolates in comparison with standard American Type Culture Collection (ATCC) control strains. Comparison of <i>E. faecalis</i> strong and weak biofilm-forming clinical isolates with ATCC control strains showed that proteins associated with shikimate kinase pathway and sulfate transport were up-regulated in the strong biofilm former, while proteins associated with secondary metabolites, cofactor biosynthesis, and tetrahydrofolate biosynthesis were down-regulated. In the weak biofilm former, proteins associated with nucleoside and nucleotide biosynthesis were up-regulated, whereas proteins associated with sulfate and sugar transport were down-regulated. Further pathway and gene ontology analyses revealed that the major differences in biofilm formation arise from differences in metabolic activity levels of the strong and weak biofilm formers, with higher levels of metabolic activity observed in the weak biofilm former. The differences in metabolic activity could therefore be a major determinant of the biofilm ability of <i>E. faecalis</i> The new markers identified from this study can be further characterized in order to understand their exact role in <i>E. faecalis</i> biofilm formation ability. This, in turn, can lead to numerous therapeutic benefits in the treatment of this oral and systemic pathogen. The data has been deposited to the ProteomeXchange with identifier PXD006542.

### Keywords
Enterococcus faecalis, Biofilm, Proteome, Itraq

### Affiliations
National University of Singapore
Dentistry Dean's Office, National University of Singapore, Singapore

### Submitter
Tanujaa Suriyanarayanan

### Lab Head
Dr C J Seneviratne
Dentistry Dean's Office, National University of Singapore, Singapore


### SDRF
- organism: Homo sapiens
- organism part: liver
- cell type: not available
- disease: Hepatocellular carcinoma
- label: label free sample
- instrument: Orbitrap Fusion
- modification parameters: Carbamidomethyl
- modification parameters: Oxidation
- modification parameters: Acetyl

