### Accession
PXD008579

### Title
Protein and peptide turnover measurements using pulsed SILAC-TMT approach

### Description
We evaluated the feasibility of a workflow combining dynamic SILAC experiments with tandem mass tag (TMT)-labeling of ten pulse time-points. Replicate analysis established that the same reproducibility of turnover rates can be obtained for peptides as for proteins facilitating proteoform resolved investigation of protein stability.

### Sample Protocol
Four HeLa cell culture replicates and a MS measurement duplicate of SILAC pulse experiments were performed. Two cell batches were switched from light (K0R0) to heavy (K8R10) label and two replicates were switched from heavy to light label and lysed after different time-points (0, 1, 3, 6, 10, 16, 24, 34, 48, inf.h). Peptides were labeled using TMT, combined, fractionated, and analyzed employing a MS2 or MS3 based method for TMT quantification. In addition, for one replicate, fractional SILAC labeling was directly analyzed on MS1 level omitting the TMT labeling step.

### Data Protocol
Peptide and protein identification and quantification for DDA type of experiments was performed using MaxQuant (v1.5.5.1). Orbitrap raw files were searched against the Swissprot database. Carbamidomethylated cysteine was set as fixed modification and oxidation of methionine, and N-terminal protein acetylation as variable modifications. For pulsed SILAC samples without TMT label, Lys0/Arg0 and Lys8/Arg10 were specified as metabolic labels, whereas for pulsed SILAC-TMT samples, TMT10 was specified as label within a reporter ion MS3 experiment type and K8 and R10 were set as additional variable modifications. Trypsin/P was specified as the proteolytic enzyme, with up to two missed cleavage sites allowed. Precursor tolerance was set to �5 ppm, and fragment ion tolerance to �20 ppm. Results were adjusted to 1 % peptide spectrum match (PSM) and 1 % protein false discovery rate (FDR).

### Publication Abstract
The coordination of protein synthesis and degradation regulating protein abundance is a fundamental process in cellular homeostasis. Today, mass spectrometry-based technologies allow determination of endogenous protein turnover on a proteome-wide scale. However, standard dynamic SILAC (Stable Isotope Labeling in Cell Culture) approaches can suffer from missing data across pulse time-points limiting the accuracy of such analysis. This issue is of particular relevance when studying protein stability at the level of proteoforms because often only single peptides distinguish between different protein products of the same gene. To address this shortcoming, we evaluated the merits of combining dynamic SILAC and tandem mass tag (TMT)-labeling of ten pulse time-points in a single experiment. Although the comparison to the standard dynamic SILAC method showed a high concordance of protein turnover rates, the pulsed SILAC-TMT approach yielded more comprehensive data (6000 proteins on average) without missing values. Replicate analysis further established that the same reproducibility of turnover rate determination can be obtained for peptides and proteins facilitating proteoform resolved investigation of protein stability. We provide several examples of differentially turned over splice variants and show that post-translational modifications can affect cellular protein half-lives. For example, N-terminally processed peptides exhibited both faster and slower turnover behavior compared with other peptides of the same protein. In addition, the suspected proteolytic processing of the fusion protein FAU was substantiated by measuring vastly different stabilities of the cleavage products. Furthermore, differential peptide turnover suggested a previously unknown mechanism of activity regulation by post-translational destabilization of cathepsin D as well as the DNA helicase BLM. Finally, our comprehensive data set facilitated a detailed evaluation of the impact of protein properties and functions on protein stability in steady-state cells and uncovered that the high turnover of respiratory chain complex I proteins might be explained by oxidative stress.

### Keywords
Posttranslational modification, Posttranslational cleavage, Degradation rates, Tmt, Pulsed silac, Synthesis rates, Protein turnover, Splice variants, Proteoforms

### Affiliations
Chair of Proteomics and Bioanalytics, Technical University of Munich (TUM), Freising, Germany
Technical University of Munich, Freising, Germany

### Submitter
Jana Zecha

### Lab Head
Dr Bernhard Kuster
Chair of Proteomics and Bioanalytics, Technical University of Munich (TUM), Freising, Germany


