### Accession
PXD017233

### Title
Second messenger control of mRNA translation by dynamic ribosome modification

### Description
Pseudomonas species employ complex, dynamic signalling networks to fine-tune responses to changing environments, with regulation taking place at the transcriptional, post-transcriptional and post-translational levels. Control of mRNA translation and hence protein abundance is a crucial element of this regulatory network.  Previously, we identified the ribosomal modification protein RimK, which influences the transition between active and sessile bacterial lifestyles. RimK is an ATP-dependent glutamyl ligase that adds glutamate residues to the C-terminus of ribosomal protein RpsF. This in-turn induces specific changes in ribosome function and translational output. RimK activity is itself under complex, multifactorial control: by the bacterial second messenger cyclic-di-GMP; a phosphodiesterase trigger enzyme (RimA); and a polyglutamate-specific protease (RimB). Deletion of the rim operon affects phenotypes including attachment, motility and cytotoxicity, severely compromising rhizosphere colonisation by the soil bacterium Pseudomonas fluorescens.  Using a combination of protein biochemistry, quantitative proteomics and ribosomal profiling experiments, we examined the relationship between ribosomal modification and downstream changes in the P. fluorescens proteome. RimK activity leads to active proteome remodelling by two main routes; indirectly, through changes in the abundance of the global translational regulator Hfq and directly, with translation of surface attachment factors, amino acid transporters and key secreted molecules linked specifically to RpsF modification. Our findings suggest post-translational ribosomal modification as a rapid-response mechanism to tune global gene translation and thereby protein abundance in response to environmental signals.

### Sample Protocol
Cells were extracted with SDS and proteins precipitated with methanol/chloroform. The experiment was performed with 2 biological replicates. In each replicate the samples were labelled with an iTRAQ 4 plex kit (Sciex) as follows: 114=wt LB 28, 115=DK LB 28, 116=wt RS 8, 117=DK RS 8. The labelled samples were combined and fractionated using the Pierceâ„¢ High pH Reversed-Phase Peptide Fractionation Kit producing 7-8 fractions. After analysis on an Orbitrap Fusion (Thermo) using MS3 synchronous precursor selection, the raw data from both replicates were combined and processed in Proteome Discoverer 2.4 (Thermo).

### Data Protocol
The raw data from both replicates were combined and processed in Proteome Discoverer 2.4 (Thermo) with the following main parameters: protein sequence database: P. fluorescens SBW25 (Uniprot, Feb/2016, 6388 entries); variable modifications: oxidation (M), deamidation (N,Q); Percolator strict FDR target: 0.01, reporter ion quantifier: most confident centroid, 20 ppm, HCD, MS3; consensus workflow for statistical analysis: replicates with nested design; use unique peptides for protein groups only, co-isolation threshold 50%, imputation: low abundance resampling , ratio based on pairwise ratios, hypothesis test: t-test (background based) generating adjusted p-values according to Benjamini-Hochberg. The protein results table was exported and used to generate the final protein expression tables and plots in The R Project for Statistical Computing.

### Publication Abstract
Control of mRNA translation is a crucial regulatory mechanism used by bacteria to respond to their environment. In the soil bacterium Pseudomonas fluorescens, RimK modifies the C-terminus of ribosomal protein RpsF to influence important aspects of rhizosphere colonisation through proteome remodelling. In this study, we show that RimK activity is itself under complex, multifactorial control by the co-transcribed phosphodiesterase trigger enzyme (RimA) and a polyglutamate-specific protease (RimB). Furthermore, biochemical experimentation and mathematical modelling reveal a role for the nucleotide second messenger cyclic-di-GMP in coordinating these activities. Active ribosome regulation by RimK occurs by two main routes: indirectly, through changes in the abundance of the global translational regulator Hfq and directly, with translation of surface attachment factors, amino acid transporters and key secreted molecules linked specifically to RpsF modification. Our findings show that post-translational ribosomal modification functions as a rapid-response mechanism that tunes global gene translation in response to environmental signals.

### Keywords
Ribosomal proteins, Cyclic-di-gmp, Signalling, Pseudomonas, Itraq proteome

### Affiliations
Biological Chemistry
John Innes Centre / University of East Anglia

### Submitter
Gerhard Saalbach

### Lab Head
Dr Jacob George Malone
John Innes Centre / University of East Anglia


