### Accession
PXD017860

### Title
HEK293T cell translatome analysis upon 6-MP treatment. Related to Supplementary Table 6 from An et al (2020)

### Description
HEK293T cells were treated with 6-MP and harvested at various time point (6, 9 and 18h). Translatome was measure by Azidohomoalanine (AHA) incorporation, Biotin-click, and streptavidin enrichment.

### Sample Protocol
Samples were resuspended in 100 mM EPPS, pH 8.5 containing 0.1% RapiGest and digested at 37°C for 6 h with Trypsin protease at a 100:1 protein-to-protease ratio. Tandem mass tag labeling of each sample was performed by adding indicated amount of the 20 ng/μL stock of TMT 11plex reagent along with acetonitrile to achieve a final acetonitrile concentration of approximately 30% (v/v). Typically, 5 l for AHA-derived synthesis proteomics. Following incubation at room temperature for 1 h, labeling efficiency of a small aliquot was tested, and the reaction was then quenched with hydroxylamine to a final concentration of 0.5% (v/v) for 15 min.  The TMT-labeled samples were pooled together at a 1:1 ratio. The sample was vacuum centrifuged to near dryness, resuspended in 5% formic acid for 15 min, centrifuged at 10000×g for 5 minutes at room temperature and subjected to C18 solid-phase extraction (SPE) (Sep-Pak, Waters). Mass spectrometry data were collected using an Orbitrap Fusion Lumos mass spectrometer (Thermo Fisher Scientific, San Jose, CA) coupled to a Proxeon EASY-nLC1200 liquid chromatography (LC) pump (Thermo Fisher Scientific). Peptides were separated on a 100 μm inner diameter microcapillary column packed in house with ~35 cm of Accucore150 resin (2.6 μm, 150 Å, Thermo Fisher Scientific, San Jose, CA) with a gradient consisting of 5%–21% (0-170 min) (ACN, 0.1% FA) over a total 180 min run at ~500 nL/min. For analysis, we loaded ~1 ug onto the column. Each analysis used the Multi-Notch MS3-based TMT method, to reduce ion interference compared to MS2 quantification, combined with newly implemented Real Time Search analysis software. The scan sequence began with an MS1 spectrum (Orbitrap analysis; resolution 120,000 at 200 Th; mass range 400−1400 m/z; automatic gain control (AGC) target 5×105; maximum injection time 50 ms). Precursors for MS2 analysis were selected using a 3 seconds TopSpeed method. MS2 analysis consisted of collision-induced dissociation (quadrupole ion trap analysis; Turbo scan rate; AGC 2.0×104; isolation window 0.7 Th; normalized collision energy (NCE) 35; maximum injection time 35 ms). Monoisotopic peak assignment was used, and previously interrogated precursors were excluded using a dynamic window (150 s ± 10 ppm). Following acquisition of each MS2 spectrum, a synchronous-precursor-selection (SPS) API-MS3 scan was collected on the top 10 most intense ions b or y-ions matched by the online search algorithm in the associated MS2 spectrum. MS3 precursors were fragmented by high energy collision-induced dissociation (HCD) and analyzed using the Orbitrap (NCE 65; AGC 2.5×105; maximum injection time 200 ms, resolution was 50,000 at 200 Th).

### Data Protocol
Mass spectra were processed using a Comet-based (v2018.01 rev.2) software. Spectra were converted to mzXML using a modified version of ReAdW.exe. Database searching included all canonical entries from the human UniProt database (SwissProt – 2019-01). This database was concatenated with one composed of all protein sequences in the reversed order. Searches were performed using a 20 ppm precursor ion tolerance for total protein level analysis. The recommended product ion parameters for ion trap ms/ms were used (1.0005 tolerance, 0.4 offset (mono masses), theoretical_fragment_ions = 1). TMT tags on lysine residues and peptide N termini (+229.163 Da for Amino-TMT) and carbamidomethylation of cysteine residues (+57.021 Da) were set as static modifications, while oxidation of methionine residues (+15.995 Da) was set as a variable modification. Peptide-spectrum matches (PSMs) were adjusted to a 1% false discovery rate (FDR) and PSM filtering was performed using a linear discriminant analysis, while considering the following parameters: Comet Log Expect and Diff Seq. Delta Log Expect, missed cleavages, peptide length, charge state, and precursor mass accuracy. For TMT-based reporter ion quantitation, we extracted the summed signal-to-noise (S:N) ratio for each TMT channel and found the closest matching centroid to the expected mass of the TMT reporter ion (integration tolerance of 0.003 Da). For protein-level comparisons, PSMs were identified, quantified, and collapsed to a 1% peptide false discovery rate (FDR) and then collapsed further to a final protein-level FDR of 1%. Moreover, protein assembly was guided by principles of parsimony to produce the smallest set of proteins necessary to account for all observed peptides. Proteins were quantified by summing reporter ion counts across all matching PSMs. PSMs with poor quality, MS3 spectra with more than 3 TMT reporter ion channels missing, or isolation specificity less than 0.7 (or 0.2 when using Real Time Search method), or with TMT reporter summed signal-to-noise ratio that were less than 150, or had no MS3 spectra were excluded from quantification.

### Publication Abstract
Mammalian cells reorganize their proteomes in response to nutrient stress through translational suppression and degradative mechanisms using the proteasome and autophagy systems<sup>1,2</sup>. Ribosomes are central targets of this response, as they are responsible for translation and subject to lysosomal turnover during nutrient stress<sup>3-5</sup>. The abundance of ribosomal (r)-proteins (around 6% of the proteome; 10<sup>7</sup> copies per cell)<sup>6,7</sup> and their high arginine and lysine content has led to the hypothesis that they are selectively used as a source of basic amino acids during nutrient stress through autophagy<sup>4,7</sup>. However, the relative contributions of translational and degradative mechanisms to the control of r-protein abundance during acute stress responses is poorly understood, as is the extent to which r-proteins are used to generate amino acids when specific building blocks are limited<sup>7</sup>. Here, we integrate quantitative global translatome and degradome proteomics<sup>8</sup> with genetically encoded Ribo-Keima<sup>5</sup> and Ribo-Halo reporters to interrogate r-protein homeostasis with and without active autophagy. In conditions of acute nutrient stress, cells strongly suppress the translation of r-proteins, but, notably, r-protein degradation occurs largely through non-autophagic pathways. Simultaneously, the decrease in r-protein abundance is compensated for by a reduced dilution of pre-existing ribosomes and a reduction in cell volume, thereby maintaining the density of ribosomes within single cells. Withdrawal of basic or hydrophobic amino acids induces translational repression without differential induction of ribophagy, indicating that ribophagy is not used to selectively produce basic amino acids during acute nutrient stress. We present a quantitative framework that describes the contributions of biosynthetic and degradative mechanisms to r-protein abundance and proteome remodelling in conditions of nutrient stress.

### Keywords
Human, Lc-msms, Azidohomoalanine (aha) incorporation

### Affiliations
Cell Biology Program, Sloan Kettering Institute, New York
Department of Cell Biology, Harvard Medical School, Boston MA 02115 USA

### Submitter
Alban Ordureau

### Lab Head
Dr Wade Harper
Department of Cell Biology, Harvard Medical School, Boston MA 02115 USA


