### Accession
PXD031421

### Title
Structure and molecular assembly of a MuvB transcription factor complex with the nucleosome

### Description
The core cell cycle machinery genes are transcriptionally regulated by the MuvB family of protein complexes in a cell cycle specific manner. During cell cycle exit in quiescence or senescence, the DREAM complex, which is the repressive form of MuvB, directs transcriptional repression of cell cycle genes; conversely during cell proliferation, the complex of MuvB with the transcription factors (TFs) B-MYB and FOXM1 activate mitotic genes during the G2 phase of the cell cycle. The mechanisms of transcriptional regulation of these complexes are still poorly characterised.  Here we combine biochemical analysis and in vitro reconstitution, with structural analysis by cryo-electron microscopy (cryo-EM) and cross-linking mass spectrometry (XL-MS), to functionally examine these complexes. Our data suggests that MuvB is a chromatin regulator whereby a core region binds the nucleosome and remodels it, thereby exposing nucleosomal DNA. This remodelling activity is supported by B-MYB which directly binds the remodelled DNA. Given the remodelling activity on the nucleosome, we propose that the MuvB complex with B-MYB (MMB) function as a pioneer transcription factor complex. Our data rationalises prior biochemical and cellular studies and provides a molecular framework of interactions on a protein complex, which is key for cell cycle regulation.

### Sample Protocol
After the crosslinking reaction, triethylammonium bicarbonate buffer (TEAB) was added to the sample at a final concentration of 100 mM. Proteins were reduced and alkylated with 5 mM tris-2-carboxyethyl phosphine (TCEP) and 10 mM iodoacetamide (IAA) simultaneously for 60 min in dark and were digested overnight with trypsin at final concentration 50 ng/μL (Pierce). Sample was dried and peptides were fractionated with high-pH Reversed-Phase (RP) chromatography using the XBridge C18 column (2.1 x 150 mm, 3.5 μm, Waters) on a Dionex UltiMate 3000 HPLC system. Mobile phase A was 0.1% v/v ammonium hydroxide and mobile phase B was acetonitrile, 0.1% v/v ammonium hydroxide. The peptides were fractionated at 0.2 mL/min with the following gradient: 5 minutes at 5% B, up to 12% B in 3 min, for 32 min gradient to 35% B, gradient to 80% B in 5 min, isocratic for 5 minutes and re-equilibration to 5% B. Fractions were collected every 42 sec, SpeedVac dried and orthogonally pooled into 12 samples for MS analysis.  LC-MS analysis was performed on the Dionex UltiMate 3000 UHPLC system coupled with the Orbitrap Lumos Mass Spectrometer (Thermo Scientific). Each peptide fraction was reconstituted in 30 μL 0.1% formic acid and 15 μL were loaded to the Acclaim PepMap 100, 100 μm × 2 cm C18, 5 μm trapping column at 10 μL/min flow rate of 0.1% formic acid loading buffer. Peptides were then subjected to a gradient elution on the Acclaim PepMap (75 μm × 50 cm, 2 μm, 100 Å) C18 capillary column connected to a stainless steel emitter with integrated liquid junction (cat# PSSELJ, MSWIL) on the EASY-Spray source at 45 °C. Mobile phase A was 0.1% formic acid and mobile phase B was 80% acetonitrile, 0.1% formic acid. The gradient separation method at flow rate 300 nL/min was as follows: for 95 min gradient from 5%-38% B, for 5 min up to 95% B, for 5 min isocratic at 95% B, re-equilibration to 5% B in 5 min, for 10 min isocratic at 5% B. Precursors between 375-1,600 m/z and charge equal or higher than +3 were selected at 120,000 resolution in the top speed mode in 5 sec and were isolated for CID fragmentation (collision energy 25%) with quadrupole isolation width 1.6 Th and Orbitrap detection with 30,000 resolution. Fragments with targeted mass difference of 31.9721 (DSSO crosslinker) were further subjected to CID fragmentation at the MS3 level with collision energy 35%, iontrap detection, max IT 50 ms, AGC 2×104 and MS2 isolation window 2 Th. Two precursor groups were selected with both ions in the pair. Targeted MS precursors were dynamically excluded for further isolation and activation for 30 seconds with 10 ppm mass tolerance.

### Data Protocol
Identification of crosslinked peptides was performed in Proteome Discoverer 2.4 (Thermo) with the Xlinkx search engine in the MS2_MS3 mode. Precursor, FTMS and ITMS mass tolerances were 20ppm, 30ppm and 0.5 Da respectively with maximum 2 trypsin missed cleavages allowed. Carbamidomethyl at C and Oxidation at M were selected as static and dynamic modifications respectively. Spectra were searched against a FASTA file containing the sequences of the proteins in the complex as well as a random selection of ~1000 Escherichia coli UniProt entries as an additional negative control of the crosslink matches. Crosslinked peptides were filtered at FDR<0.01 using the Percolator node and decoy database search.

### Publication Abstract
None

### Keywords
Muvb, Crosslinking

### Affiliations
The Institute of Cancer Research

### Submitter
James Wright

### Lab Head
Dr Jyoti Choudhary
The Institute of Cancer Research


