### Accession
PXD016322

### Title
Human bone marrow comprise atypical adipocytes with specific lipid metabolism

### Description
Under caloric restriction, bone marrow adipocytes (BM-Ad) do not decrease in size conversely to white adipocytes, suggesting their unique metabolic properties. We compared human primary BM-Ad with paired subcutaneous adipocytes (SC-Ad) using proteomic and lipidomic approaches. We found that while SC-Ad and BM-Ad share similar morphological features, they possess distinct lipid metabolism. BM-Ad shows enrichment in proteins involved in cholesterol metabolism correlating with increased free cholesterol content while proteins involved in lipolysis were downregulated. In particular, monoacylglycerol lipase expression was strongly reduced in BM-Ad, leading to accumulation of monoacylglycerol. Consequently, basal and induced lipolytic responses were absent in BM-Ad, affirming their differences in metabolic fitness upon caloric restriction. These specific metabolic features are not recapitulated in vitro using common protocols to differentiate bone marrow mesenchymal stem cells. Thus, contrary to classical SC-Ad, BM-Ad display a specific lipid metabolism, as they are devoid of lipolytic activity and exhibit a cholesterol-orientated metabolism.

### Sample Protocol
For proteomic and lipidomic studies, 4 samples were used (2 men and 2 women, mean age: 67± 7. 4 years; mean BMI: 26.5 ± 3.1 kg/m2). After 3 washes with PBS, isolated adipocytes (400 µl) were mixed 1.5 mL methanol in glass tubes. Sample mixtures were then incubated with 5 mL of methyl tert-butyl ether (MTBE; Sigma-Aldrich) for 1h at RT under gently shaking. After adding 1.2 mL of water, samples were centrifuged for 10 min at 1000 g. The lipid containing upper phase was transferred to a new glass tube. The lower phase was re-extracted with 2 volumes of MTBE:methanol:water (10:3:2.5) and samples were centrifuged for 10 min at 1000 g before proteomic analyses. The upper phase was collected, combined with the one collected after the first extraction and kept at -80°C for lipidomic analysis (see below). The lower phase was centrifuged for 10 min at 5000 g at RT and the pellet (containing proteins) was washed 2 times with PBS to remove solvents. Pellets were then resuspended with PBS 2% SDS, sonicated for 20 seconds and protein concentration was determined with the DC™ Protein Assay kit (Bio-Rad).  For each sample, 15µg of protein was reduced with modified Laemmli buffer (40 mM Tris pH 6.8, 2% SDS, 10% glycerol, 25mM DTT and 0.01% bromophenol blue) for 15 min at 65°C and alkylated by the addition of 90mM iodoacetamide for 30 min at RT in the dark. Protein samples were loaded on a 1D SDS-PAGE gel (0.15 x 8 cm) and the electrophoretic migration was stopped as soon as the proteins entered the separating gel, in order to isolate all proteins in a single gel band (stained with Coomassie blue).  The corresponding gel slices were excised and washed with 100 mM ammonium bicarbonate buffer. Proteins were in-gel digested using 0.6 µg of modified sequencing grade trypsin (Promega) in 50 mM ammonium bicarbonate overnight at 37°C. The resulting peptides were extracted in 50 mM ammonium bicarbonate followed by 10% formic acid/acetonitrile (1/1 v/v). The peptidic fractions were dried under speed-vacuum and resuspended with 5% acetonitrile and 0.05% trifluoroacetic acid (TFA) for MS analysis.  Peptides were analyzed by nanoLC-MS/MS using an UltiMate 3000 RSLCnano system coupled to a Q-Exactive Plus mass spectrometer (Thermo Fisher Scientific). Five µL of each sample was loaded onto a C-18 precolumn (300 µm ID x 5 mm, Thermo Fisher) in a solvent consisting of 5% acetonitrile and 0.05% TFA with a flow rate of 20 µL/min. After 5 min of desalting, the precolumn was switched online with the analytical C-18 column (75 µm ID x 15 cm, Reprosil C18) equilibrated in 95% solvent A (5% acetonitrile, 0.2% formic acid) and 5% solvent B (80% acetonitrile, 0.2% formic acid). Peptides were eluted using a 5 to 50% gradient of solvent B over 300 min at a flow rate of 300 nL/min. The Q-Exactive Plus was operated in a data-dependent acquisition mode with the XCalibur software. MS survey scans were acquired in the Orbitrap on the 350-1500 m/z range with the resolution set to a value of 70000. The 10 most intense ions per survey scan were selected for HCD fragmentation and the resulting fragments were analyzed in the Orbitrap with the resolution set to a value of 17500. Dynamic exclusion was employed within 30 seconds to prevent repetitive selection of the same peptide. Duplicate technical LC-MS measurements were performed for each sample.

### Data Protocol
Raw mass spectrometry files were processed and quantitated with the MaxQuant software (version 1.6.3.4) with database search using the Andromeda search engine. Data were searched against human entries from the Swissprot protein database (UniProtKB/Swiss-Prot Knowledgebase release 2018_02). Carbamidomethylation of cysteines was set as a fixed modification whereas oxidation of methionine and protein N-terminal acetylation were set as variable modifications. Specificity of trypsin digestion was set for cleavage after K or R, and up to two missed trypsin cleavage sites were allowed. The precursor mass tolerance was set to 20 ppm for the first search and 4.5 ppm for the main Andromeda database search. The mass tolerance MS/MS mode was set to 0.5 Da.  Minimum peptide length was set to seven amino acids, and minimum number of unique peptides was set to one. Andromeda results were validated by the target-decoy approach using a reverse database with both peptide and protein FDR at 1%. For label-free relative quantification of the samples, the “match between runs” option of MaxQuant was enabled with a time window of 0.7min, to allow cross-assignment of MS features detected in the different runs. The “label free quantification” (LFQ) metric from the MaxQuant “protein group.txt” output was used to quantify proteins. Missing protein intensity values were replaced by a constant noise value determined independently for each sample as the lowest value of the total protein population. Only proteins identified in at least three samples in the same location (i.e. SC-Ad or BM-Ad) were considered to be robustly detected and used for statistical and bioinformatic analyses.

### Publication Abstract
Under caloric restriction, bone marrow adipocytes (BM-Ads) do not decrease in size compared to white adipocytes, suggesting they harbor unique metabolic properties. We compare human primary BM-Ads with paired subcutaneous adipocytes (SC-Ads) using proteomic and lipidomic approaches. We find that, although SC-Ads and BM-Ads share similar morphological features, they possess distinct lipid metabolism. Although BM-Ad shows enrichment in proteins involved in cholesterol metabolism, correlating with increased free cholesterol content, proteins involved in lipolysis were downregulated. In particular, monoacylglycerol lipase expression is strongly reduced in BM-Ads, leading to monoacylglycerol accumulation. Consequently, basal and induced lipolytic responses are absent in BM-Ads, affirming their differences in metabolic fitness upon caloric restriction. These specific metabolic features are not recapitulated in&#xa0;vitro using common protocols to differentiate bone marrow mesenchymal stem cells. Thus, contrary to classical SC-Ads, BM-Ads display a specific lipid metabolism, as they are devoid of lipolytic activity and exhibit a cholesterol-orientated metabolism.

### Keywords
Human, Lc-msms, Label free quantificatio, Bone marrow adipocytes

### Affiliations
CNRS
Team: Microenvironnement, Cancer and Adipocytes Institute of Pharmacology and Structural Biology BP64182 205 route de Narbonne 31077 Toulouse Cedex04 France

### Submitter
Manuelle Ducoux-Petit

### Lab Head
Dr Catherine Muller
Team: Microenvironnement, Cancer and Adipocytes Institute of Pharmacology and Structural Biology BP64182 205 route de Narbonne 31077 Toulouse Cedex04 France


