### Accession
PXD000942

### Title
Differences in proteasome targeting in Arabidopsis leaf mesophyll, epidermal and vascular cells

### Description
For the detailed and sensitive detection of the responses to a specific treatment it is important to perform tissue or cell type specific analyses, but this is not easily achievable for the different leaf tissues. Here we developed a method termed MeSelect to effectively separate leaf epidermis, vascular and mesophyll cells basically without contamination from other tissues. The high yield of the MeSelect method allowed for tissue specific high-throughput proteome analyses after inhibition of the proteasome with Syringolin A and the affinity enrichment of polyubiquitylated proteins in epidermis, mesophyll and vasculature.

### Sample Protocol
Frozen plant material was ground with a mortar and pestle. Mesophyll cells were derived from 3 plants, vasculature cells from 9 plants and epidermal cells from 4 plants, and 8-10 leaves were harvested per plant. Proteins from mesophyll cells were extracted with a sequential extraction using native and urea buffer. Proteins from vasculature or epidermis cells were extracted only with urea buffer.  For the pre-clearing step 500 �g protein of mesophyll and vasculature in a total volume of 700 - 1200 �l and 400 �g protein of epidermis in a total volume of 1800 �l were each washed with 100 �l sepharose CL4B. Afterwards the sepharose was pelleted and half of the extract was added to 50 �l washed p62-derived UBA domain agarose conjugate, while the other half was added to 50 �l sepharose CL4B as a control. After incubation for 2 h under agitation the beads were pelleted, the supernatant was removed and the beads were washed 3 times. Proteins were eluted and subjected directly to SDS PAGE on 10% gels. After electrophoretic separation of the proteins, each lane was cut into 5 sections. Proteins were reduced with DTT followed by carbamidomethylation with iodacetamide. The proteins were then digested in-gel with trypsin solution for 16-19 h at 30�C. After digestion, the peptides were extracted and acetonitrile was vaporised before the peptides were desalted using Sep-Pak reverse-phase cartridges. Before subjection to mass spectrometric measurements, each sample was dissolved in 3% acetonitrile, 0.1% trifluoroacetic acid. Mass spectrometry measurements were performed on an LTQ OrbiTrap XL MS (Thermo Fisher) coupled to a NanoLC-AS1 (Eksigent) using electrospray ionisation. For LC separation a capillary column packed with 8 cm C18 beads with a diameter of 3 �m and a pore size of 100 � was used. Peptides were loaded on the column with a flow rate of 500 nl/min for 16 min and eluted by an increasing acetonitrile gradient from 3% acetonitrile to 50% acetonitrile for 60 min with a flow rate of 200 nl/min. Full scans were obtained using the FT Orbitrap at a range of 300-2000 mass/charge followed by MS/MS scans of the five highest parent ions. Dynamic exclusion was enabled at a duration of 120 s.

### Data Protocol
The acquired raw spectra were transformed to mgf data format and searched against the TAIR10 database (32, download on January 17th, 2011) with concatenated decoy database and supplemented with common contaminants (71,032 entries) using the Mascot algorithm (version 2.3.02) (Mascot Science). The search parameters used were: mass = monoisotopic, requirement for tryptic ends, 2 missed cleavages allowed, precursor ion tolerance = +/- 10 ppm, fragment ion tolerance = +/- 0.8 Da, variable modifications of methionine (M, PSI-MOD name: oxidation, mono ? = 15.995) and static modifications of cysteine (C, PSI-MOD name: iodoacetamide derivatized residue, mono ? = 57.0215). Peptide spectrum assignments with ionscore > 30 and expect value < 0.015, except those of known contaminants, were filtered for ambiguity. Peptides matching to several proteins were excluded from further analyses. This does not apply to different splice variants of the same protein or to different loci sharing exactly the same amino acid sequence. All remaining spectrum assignments were inserted into the pep2pro database.

### Publication Abstract
Protein and transcript levels are partly decoupled as a function of translation efficiency and protein degradation. Selective protein degradation via the Ubiquitin-26S proteasome system (UPS) ensures protein homeostasis and facilitates adjustment of protein abundance during changing environmental conditions. Since individual leaf tissues have specialized functions, their protein composition is different and hence also protein level regulation is expected to differ. To understand UPS function in a tissue-specific context we developed a method termed Meselect to effectively and rapidly separate Arabidopsis thaliana leaf epidermal, vascular and mesophyll tissues. Epidermal and vascular tissue cells are separated mechanically, while mesophyll cells are obtained after rapid protoplasting. The high yield of proteins was sufficient for tissue-specific proteome analyses after inhibition of the proteasome with the specific inhibitor Syringolin A (SylA) and affinity enrichment of ubiquitylated proteins. SylA treatment of leaves resulted in the accumulation of 225 proteins and identification of 519 ubiquitylated proteins. Proteins that were exclusively identified in the three different tissue types are consistent with specific cellular functions. Mesophyll cell proteins were enriched for plastid membrane translocation complexes as targets of the UPS. Epidermis enzymes of the TCA cycle and cell wall biosynthesis specifically accumulated after proteasome inhibition, and in the vascular tissue several enzymes involved in glucosinolate biosynthesis were found to be ubiquitylated. Our results demonstrate that protein level changes and UPS protein targets are characteristic of the individual leaf tissues and that the proteasome is relevant for tissue-specific functions.

### Keywords
Mesophyll, Leaf tissues, Quantitative proteomics, Arabidopsis, Ubiquitylation, Ubiquitin proteasome system, Affinity enrichment, Epidermis, Syringolin a, Vasculature

### Affiliations
ETH Zurich
Institute of  Plant Science

### Submitter
Katja Baerenfaller

### Lab Head
Dr Katja Baerenfaller
ETH Zurich


