### Accession
PXD024112

### Title
Retroviral transduction of NOTCH1 into primary human Chronic Lymphocytic Leukemia cells unravels immune escape-mechanisms

### Description
Functional studies to investigate gene mutations recurrent in B cell lymphoma have been hampered by the inability to genetically manipulate primary cells, attributed to low transduction efficacy and procedure-associated toxicity. Alternative approaches utilize cell lines and mouse models, which often only poorly represent the genomic complexity and biology of the primary malignancy. To overcome these limitations, we have developed a method to retrovirally transfer genes into primary malignant B cells with high transduction efficacy and minimal toxicity. Using this method, we investigated the functions of NOTCH1, the most commonly mutated gene in CLL, by generating isogenic primary tumor cells from patients with Chronic Lymphocytic Leukemia (CLL) and Mantle Cell Lymphoma (MCL), differing only in their expression of NOTCH1. Our data demonstrate that NOTCH1 facilitates immune escape of malignant B cells by up-regulating PD-L1, partly dependent on autocrine interferon-g signaling. In addition, NOTCH1 causes silencing of the entire HLA-class II locus via suppression of the transcriptional co-activator CIITA. These NOTCH1-mediated immune escape mechanisms are associated with the expansion of CD4+ T cells in vivo, further contributing to the poor clinical outcome of NOTCH1-mutated CLL and MCL

### Sample Protocol
Sample dissolution, TMT labelling and Reverse-Phase fractionation  The cell pellets were resuspended in lysis buffer containing 100mM Triethylammonium bicarbonate (TEAB, Sigma), 0.1% SDS followed by heating at 90°C for 5 min and probe sonication. Protein concentration was estimated using Bradford assay according to manufacturer’s instructions (BIO-RAD-Quick start).  30ug of total protein were reduced with 2ul of 50mM tris-2-caraboxymethyl phosphine (TCEP, Sigma) for 1 hour at 60°C followed by alkylation with 1ul of 200mM methyl methanethiosulfonate (MMTS, Sigma) for 10min at room temperature (RT). Then protein samples were digested overnight at 37°C using trypsin solution at ratio protein/trypsin ~ 1:30. The next day, protein digest was labelled with the TMT-6plex reagents (Thermo Scientific) for 1 hour. The reaction was quenched with 8 μL of 5% hydroxylamine (Thermo Scientific) for 15 min at room temperature (RT). All the samples were mixed and dried with speed vac concentrator. The dry TMT mix was fractionated on a Dionex Ultimate 3000 system at high pH using the X-Bridge C18 column (3.5 μm, 2.1x150mm, Waters) with 90min linear gradient from 5% to 95% acetonitrile contained 20mM ammonium hydroxide at a flow rate of 0.2 ml/min. Peptides fractions were collected between 20-55 minutes and were dried with speed vac concentrator. Each fraction was reconstituted in 0.1% formic acid for liquid chromatography tandem mass spectrometry (LC–MS/MS) analysis.  LC-MS/MS Peptide fractions were analysed on a Dionex Ultimate 3000 system coupled with the nano-ESI source Fusion Lumos Orbitrap Mass Spectrometer (Thermo Scientific). Peptides were trapped on a 100μm ID X 2 cm microcapillary C18 column (5µm, 100A) followed by 2h elution using 75μm ID X 25 cm C18 RP column (3µm, 100A) at 300nl/min flow rate. In each data collection cycle, one full MS scan (380–1,500 m/z) was acquired in the Orbitrap (120K resolution, automatic gain control (AGC) setting of 3×105 and Maximum Injection Time (MIT) of 100 ms). The subsequent MS2 was conducted with a top speed approach using a 3-s duration. The most abundant ions were selected for fragmentation by collision induced dissociation (CID). CID was performed with a collision energy of 35%, an AGC setting of 1×104, an isolation window of 0.7 Da, a MIT of 50ms. Previously analysed precursor ions were dynamically excluded for 45s. During the MS3 analyses for TMT quantification, precursor ion selection was based on the previous MS2 scan and isolated using a 2.0Da m/z window. MS2–MS3 was conducted using sequential precursor selection (SPS) methodology with the top10 settings. For MS3, HCD was used and performed using 65% collision energy and reporter ions were detected using the Orbitrap (50K resolution, an AGC setting of 1×105 and MIT of 105 ms).

### Data Protocol
Data processing The Proteome Discoverer 2.4. (Thermo Scientific) was used for the processing of CID tandem mass spectra. The SequestHT search engine was used and all the spectra searched against the Uniprot Homo sapiens FASTA database (taxon ID 9606 - Version February 2020). All searches were performed using as a static modification TMT6plex (+229.163 Da) at any N-terminus and lysines and Methylthio at Cysteines (+45.988Da). Methionine oxidation (+15.9949Da) and Deamidation on Asparagine and Glutamine (+0.984) were included as dynamic modifications. Mass spectra were searched using precursor ion tolerance 20 ppm and fragment ion tolerance 0.5 Da. For peptide confidence, 1% FDR was applied and peptides uniquely matched to a protein were used for quantification. Peptide intensities were normalised using median scaling and protein level quantification was obtained by the summation of the normalized peptide intensities. A statistical analysis of differentially-regulated proteins was carried out using the limma R-package from Bioconductor (PMID: 25605792).

### Publication Abstract
None

### Keywords
Hla-class ii locus, Co-activator ciita, Notch1, Mantle cell lymphoma (mcl), Mutated gene, Chronic lymphocytic leukemia (cll)

### Affiliations
Research Assistant
Head of the Department, Proteomics Core

### Submitter
Valar Nila Roamio Franklin

### Lab Head
Dr Clive D'Santos
Head of the Department, Proteomics Core


