### Accession
PXD003025

### Title
Quantitative proteomics of the infectious and replicative forms of Chlamydia trachomatis L2

### Description
The obligate intracellular developmental cycle of Chlamydia trachomatis presents significant challenges in defining its proteome.   In this study we have applied quantitative proteomics to both the intracellular reticulate body (RB) and the extracellular elementary body (EB) from C. trachomatis.  We used C. trachomatis L2 which is a model chlamydial isolate for such a study since it has a high infectivity: particle ratio and there is an excellent quality genome sequence.  EBs and RBs (>99% pure) were quantified by chromosomal and plasmid copy number using PCR to determine the concentrations of chlamydial proteins per bacterial cell.  RBs harvested at 15h post infection (PI) were purified by three successive rounds of gradient centrifugation.  This is the earliest possible time to obtain purified RBs, free from host cell components in quantity, within the constraints of the technology, EBs were purified at 48h PI.  We then used two-dimensional reverse phase UPLC to fractionate RB or EB peptides before mass spectroscopic analysis, providing absolute amount estimates of chlamydial proteins.

### Sample Protocol
C. trachomatis (L2/434/Bu (VR902B) was grown in Buffalo green monkey kidney cells (BGMK) in DMEM. Elementary bodies and Reticulate bodies were purified using density gradient centrifugation. The number bacterial cells for each preparation were determined using genome equivalents. Each sample preparation was reduced using tris-(2-carboxyethyl) phosphine, alkylated with methane-thiosulfonate and digested using trypsin. 100 ug of peptide lysate were lyophilised and reconsituted in buffer A (20 mM ammonium formate) contaning a digest of alcohol dehydrogenase at 20 fmol/uL.   Two dimensional separations were performed using a nanoAcquity 2D UPLC system. For the first dimension, 4.5 uL of the prepared protein lysates were injected onto an Xbridge BEH130 C18, 300 um ID x 50 mm column equilibrated in 20 mM ammonium formate pH 10. The first dimension separation was achieved by increasing the concentration of acetonitrile (buffer B) in 11 steps consisting of 8.2, 11.7, 13, 14.5, 15.9, 17.4, 18.9, 20.8, 23.6, 45, 65%.  At each step the percentrage composition was held for 1 min at a flow rate of 2ul/min and the eluent diluted by buffer C (Water containing 0.1% formic acid) onto a Symmetry C18, 180um x 20 mm trapping cartridge (Waters). After desalting, peptides were separated using an in-line second dimension analytical separation performed on a 75 um ID x 200 mm, 1.7 um BEH130, C18 column (Waters) using a linear gradient of 5 to 40% B over 90 min. All mass spectrometry was performed using a Synapt Q-Tof mass spectrometer using data independent acquisition (MSe). Data were acquired from 50 m/z to 1990 m/z using alternate low and high collision energy (Low = 5V, high = 12-35V ramp)

### Data Protocol
ProteinLynx Global Server 2.3 was used to process each raw data file, to generate reduced charge state and deisotoped precursor and associated product ion mass lists internally withi the software. Each processed file was searched against a protein translation of the C. trachomatis L2/434/BU genome sequence (March 2010). Search parameters were as follows: precursor and product ion tolerance were 10 ppm and 15 ppm respectively, trypsin digestion,  fixed modification for methyl-methane-thiosulfonate modification of Cysteine,  variable modification for the oxidation of methionine and 2 missed cleavages. Quantitation was performed within ProteinLYnx Global Server using the Hi3 approach and using alcohol dehydrogenase as the internal standard.

### Publication Abstract
The obligate intracellular developmental cycle of Chlamydia trachomatis presents significant challenges in defining its proteome. In this study we have applied quantitative proteomics to both the intracellular reticulate body (RB) and the extracellular elementary body (EB) from C. trachomatis. We used C. trachomatis L2 as a model chlamydial isolate for our study since it has a high infectivity:particle ratio and there is an excellent quality genome sequence. EBs and RBs (&gt;99% pure) were quantified by chromosomal and plasmid copy number using PCR, from which the concentrations of chlamydial proteins per bacterial cell/genome were determined. RBs harvested at 15h post infection (PI) were purified by three successive rounds of gradient centrifugation. This is the earliest possible time to obtain purified RBs, free from host cell components in quantity, within the constraints of the technology. EBs were purified at 48h PI. We then used two-dimensional reverse phase UPLC to fractionate RB or EB peptides before mass spectroscopic analysis, providing absolute amount estimates of chlamydial proteins. The ability to express the data as molecules per cell gave ranking in both abundance and energy requirements for synthesis, allowing meaningful identification of rate-limiting components. The study assigned 562 proteins with high confidence and provided absolute estimates of protein concentration for 489 proteins. Interestingly, the data showed an increase in TTS capacity at 15h PI. Most of the enzymes involved in peptidoglycan biosynthesis were detected along with high levels of muramidase (in EBs) suggesting breakdown of peptidoglycan occurs in the non-dividing form of the microorganism. All the genome-encoded enzymes for glycolysis, pentose phosphate pathway and tricarboxylic acid cycle were identified and quantified; these data supported the observation that the EB is metabolically active. The availability of detailed, accurate quantitative proteomic data will be invaluable for investigations into gene regulation and function.

### Keywords
Reticulate body, L2, Chlamydia trachomatis, Elementary body

### Affiliations
Centre for Proteomic Research, University of Southampton

### Submitter
Paul Skipp

### Lab Head
Dr Paul James Skipp
Centre for Proteomic Research, University of Southampton


