### Accession
PXD011353

### Title
PROTEOFORMER 2.0: MS validation

### Description
PROTEOFORMER is a pipeline for generating a search space of candidate translation products based on ribosome profiling data. The pipeline was published in 2014, but since then, several features have been added. For this project, the PROTEOFORMER pipeline 2.0 was tested on human HCT116 and Jurkat cell ribosome profiling data with all its new features and then compared to raw MS data of both cell lines, validating its use as a proteogenomic tool. In this study, results of 3 proteoform calling methods were combined (classical PROTEOFORMER proteoform calling, PRICE and SPECtre). Afterwards, results were eventually combined with the canonical or splicing-included version of human UniProt. The different described combinations were exported as a FASTA file and used as search space for searching matching MS data.

### Sample Protocol
Both human HCT116 and Jurkat cells were grown and prepared for MS/MS analysis, resulting in 4 replicates per cell line. Samples were analysed on an Orbitrap Q exactive instrument..

### Data Protocol
The raw files were searched with the different combination databases of PROTEOFORMER and UniProt, described earlier. For this search, MaxQuant (version 1.6.1.0) was used. It uses the Andromeda search engine to directly parse and query the raw data files. The 4 replicates of each sample were analyzed and combined during one combined MaxQuant run. Sequence database were obtained out of combinations of different analysis IDâ€™s of the PROTEOFORMER pipeline, eventually merged with UniProt, as earlier described. Trypsine was selected as tryptic enzyme and cleavage was allowed even if a proline is following arginine or lysine. Methionine oxidation was selected as a fixed modification, N-terminal acetylation as a variable modification. 2 missed cleavages were allowed. Precursor ion mass tolerance was set to 20 ppm and fragment ion mass tolerance to 4,5 ppm. Contaminants from the built-in MaxQuant contaminant database were omitted. Minimum Andromeda score for an MS/MS identification to pass, was 0 for unmodified and 40 for modified peptides. The minimum Andromeda delta score to let an MS/MS identification pass was 0 for unmodified and 6 for modified peptides. Identifications were allowed to be transferred and matched to non-sequenced and non-identified MS features in other LC-MS runs. The LFQ algorithm was allowed for label-free quantification and the iBAQ method was allowed for calculating protein abundances. Only unique and no razor peptides were selected for protein quantification. Both the PSM, XPSM and protein FDR threshold were set at 0,01.

### Publication Abstract
PROTEOFORMER is a pipeline that enables the automated processing of data derived from ribosome profiling (RIBO-seq, <i>i.e.</i> the sequencing of ribosome-protected mRNA fragments). As such, genome-wide ribosome occupancies lead to the delineation of data-specific translation product candidates and these can improve the mass spectrometry-based identification. Since its first publication, different upgrades, new features and extensions have been added to the PROTEOFORMER pipeline. Some of the most important upgrades include P-site offset calculation during mapping, comprehensive data pre-exploration, the introduction of two alternative proteoform calling strategies and extended pipeline output features. These novelties are illustrated by analyzing ribosome profiling data of human HCT116 and Jurkat data. The different proteoform calling strategies are used alongside one another and in the end combined together with reference sequences from UniProt. Matching mass spectrometry data are searched against this extended search space with MaxQuant. Overall, besides annotated proteoforms, this pipeline leads to the identification and validation of different categories of new proteoforms, including translation products of up- and downstream open reading frames, 5' and 3' extended and truncated proteoforms, single amino acid variants, splice variants and translation products of so-called noncoding regions. Further, proof-of-concept is reported for the improvement of spectrum matching by including Prosit, a deep neural network strategy that adds extra fragmentation spectrum intensity features to the analysis. In the light of ribosome profiling-driven proteogenomics, it is shown that this allows validating the spectrum matches of newly identified proteoforms with elevated stringency. These updates and novel conclusions provide new insights and lessons for the ribosome profiling-based proteogenomic research field. More practical information on the pipeline, raw code, the user manual (README) and explanations on the different modes of availability can be found at the GitHub repository of PROTEOFORMER: https://github.com/Biobix/proteoformer.

### Keywords
Hct116, Msms, Jurkat, Proteoformer

### Affiliations
BioBix
BioBix, Faculty of Bioscience Engineering, Ghent University, Coupure Links 653, Ghent, Belgium

### Submitter
Steven Verbruggen

### Lab Head
Dr Gerben Menschaert
BioBix, Faculty of Bioscience Engineering, Ghent University, Coupure Links 653, Ghent, Belgium


