### Accession
PXD002096

### Title
The influence of simulated microgravity on the proteome of Daphnia magna

### Description
Background/Objectives: The waterflea Daphnia is an interesting candidate for biore- generative life support systems (BLSS). These animals are particularly promising be- cause of their central role in the limnic food web and its mode of reproduction. How- ever, the response of Daphnia to altered gravity conditions has to be investigated, especially on the molecular level, to evaluate the suitability of Daphnia for BLSS in space. Methods: In this study, we applied a proteomic approach to identify key proteins and pathways involved in the response of Daphnia to simulated microgravity gener- ated by a 2D-clinostat. We analysed 5 biological replicates using 2D-DIGE proteomic analysis. Results: We identified 109 protein spots differing in intensity (p < 0.05). Substan- tial fractions of these proteins are involved in actin microfilament organisation, in- dicating the disruption of cytoskeletal structures during clinorotation. Furthermore, proteins involved in protein folding were identified, suggesting altered gravity in- duced break-down of protein structures in general. In addition, simulated micro- gravity increased the abundance of energy metabolism related proteins, indicating an enhanced energy demand of Daphnia. Conclusion: The affected biological processes were also described in other studies using different organisms and systems either aiming to simulate microgravity con- ditions or providing real microgravity conditions. Moreover, most of the Daphnia protein sequences are well conserved throughout taxa, indicating that the response to altered gravity conditions in Daphnia follows a general concept.

### Sample Protocol
Sample preparation To generate samples for proteomic analysis, the frozen biological samples were pul- verised in a mortar containing liquid nitrogen to prevent thawing. The resulting powder was solubilised in 330 μL lysis buffer (2 mol L−1 Thiourea, 6 mol L−1 Urea, 4 % CHAPS, 1 cOmplete ULTRA Tablets Mini (Roche) per 5 mL buffer). Afterwards, each sample was centrifuged using a QIA Shredder Mini Spin Column (Qiagen) for 3 min at 13,200 rpm to get rid of debris. Proteins were precipitated using 30 % trichloroacetic acid for 20 min on ice to inhibit proteolytic activity.19 Subsequently, samples were centrifuged for 10 min at 13,200 rpm, the supernatant was discarded and the protein pellet was washed three times with cold acetone. The pellet was dried and resolved in lysis buffer. The pH of the solution was adjusted to 8 by adding 50 mM NaOH. Pro- tein concentration was determined by Bradford Assay (Coomassie Plus (Bradford) Assay Reagent, Thermo Scientific) according to the manufacturer’s instructions. To reach sufficient protein concentrations for 2D-DIGE, 2 clinorotation runs per group were pooled leading to 5 biological replicates. 2D-DIGE Prior to first dimension isoelectric focusing (IEF), 50 μg protein per biological replicate were labelled with 2D-DIGE Cy3 Dye for control and Cy5 Dye for the clinorotated group following the protocol of the manufacturer (GE Healthcare Life Sciences). An internal pooled standard (IPS) was prepared by pooling 25 μg protein of all biological replicates and labelling of 300 μg with 2D-DIGE Cy2 Dye. 24 cm gel strips (DryStrips pH 4 – 7, GE Healthcare) for IEF were rehydrated with 450 μL rehydration buffer (2 mol L−1 Thiourea, 6 mol L−1 Urea, 4 % CHAPS, 13 mmol L−1 DTT, 2 % pharmalyte pH 3-10, bromphenol blue) for at least 10 h prior to the start of the IEF. For each 2D-DIGE gel 50 μg of one Cy3-labelled control replicate, 50 μg of one clinorotated replicate and 50 μg of Cy2-labelled IPS were combined and 65 mmol L−1 DTT and 2 % pharmalyte pH 3-10 were added. This mixed sample was applied via anodic cup loading on one gel strip. IEF was performed using an IPGPhore (Pharmacia Biotech) with a total of 60 kV h per strip. All five strips for all five biological replicates were processed in parallel. Before second dimension gel electrophoresis, gel strips were equilibrated for 15 min in 15 mL equilibration buffer (50 mmol L−1 Tris-HCl pH 6.8, 6 mol L−1 urea, 30 % gly- cerin, 2 % SDS) containing 1 % DTT on a shaker (40 min−1 , Certomat U, Sartorius). Af- terwards, a second 15 min equilibration step in 15 mL equilibration buffer with 2.5 % iodoacetamide and 200 μL saturated bromphenol blue solution was performed. For second dimension electrophoresis, lab-cast 210 x 260 x 1 mm polyacrylamide gels (Stacking gel: 0.125 mol L−1 6.8 pH Tris-HCl, 4 % acrylamide/bisacrylamide (37.5:1), 0.1 % SDS, 0.05 % APS, 0.1 % TEMED; Separating gel: 1.5 mol L−1 Tris-HCl pH 8.8, 12.5 % acrylamide/bisacrylamide (37.5:1), 0.1 % SDS, 0.05 % APS, 0.05 % TEMED) and an ETTANDaltsix electrophoresis unit (GE Healthcare Life Sciences) were used. Equi- librated gel strips were fixed on top of the gels with the help of 0.5 % agarose solved in SDS running buffer (1x Buffer: 25 mmol L−1 Tris, 192 mmol L−1 glycine, 0.2 % SDS). Electrophoresis was conducted at 10 ◦C for one hour at 5 W per gel and afterwards at 17 W per gel until the dye front reached the end of the gel. Here, 1x running buf- fer was used for anodic and 2x running buffer for the cathodic compartment of the electrophoresis unit. maging and Quantitative Analysis Immediately after electrophoresis, gels were scanned using a Typhoon 9400 fluores- cence scanner (GE Healthcare Life Sciences) with parameter settings recommended by the manufacturers for 2D-DIGE experiments. Image analysis and relative quan- tification were performed with DeCyder TM 2D Software version v7.0 (GE Healthcare Life Sciences). Coordinates of corresponding spots differing significantly in their in- tensity (p ≤ 0.05 after FDR correction, ratio ≥ |2|) were transferred to a pick list for further processing.

### Data Protocol
The MS/MS data were searched with Mascot Version: 2.3.00 (Matrix Science) us- ing the following parameters: i) Enzyme: Trypsin; ii) Fixed Modification: Carbam- idomethyl (C); iii) Variable modifications: Oxidation (M); iv) Peptide tol. 2 Da; v) MS/MS tol. 0.8 Da; vi) Peptide charge 1+, 2+ and 3+; vii) Instrument ESI-TRAP and viii) Allow up to 1 missed cleavages. As database, pre-released gene-predictions of D. magna (V2.4 effective 05/2012) were used. These sequence data were produced by The Center for Genomics and Bioinformatics at Indiana University and distrib- uted via wFleaBase in collaboration with the Daphnia Genomics Consortium (http: //daphnia.cgb.indiana.edu). For protein identification, redundant entries of 90 % similarity or more were detected through the software cd-hit20 and removed. In addition, a common contaminants database (Max Planck Institute of Biochem- istry, Martinsried, Germany: http://maxquant.org/contaminants.zip) was added. Mascot data were further processed with Scaffold 3 (Proteome Software), with “Protein Probability” and “Peptide Probability” set to 95 %. At least 2 unique peptides were used for protein identification. Protein spots having multiple protein identifications were not included in the final data-set.

### Publication Abstract
None

### Keywords
Clinorotation, Functional weightlessness, Bioregenerative life support system, Proteomics, Daphnia

### Affiliations
A
Gene Centre, LMU Munich

### Submitter
Kathrin Otte

### Lab Head
Dr Georg Arnold
Gene Centre, LMU Munich


