### Accession
PXD030315

### Title
Proteomic profile of human CHME3 microglia in response to the miR-124-3p modulation in Swedish neuronal cells

### Description
We aim to evaluate how the levels of miR-124 in APPSwedish neuronal change the signature of interferon-gamma-stimulated human microglia, in a co-culture system. We want to fing possible pathways involved in cell-to-cell signaling and traficking, as well as inflammation-associated and miRNA targets in microglia, with potential interest to be further explored in Alzheimer's disease field.

### Sample Protocol
Cellular lysates of three experimental conditions were collected, in triplicate, and analysed for proteomic analysis: (i) CHME-3 microglia cocultured with non-modulated (mock control) SWE neuroblastoma cells; (ii) CHME-3 microglia cocultured with SWE neuroblastoma cells downregulated for miR-124 (subjected to miR-124 inhibitor); and (iii) CHME-3 microglia cocultured with SWE upregulated for miR-124 (subjected to miR-124 mimic). The cellular lysates were collected in TRIzolÂ® reagent (LifeTechnologies), and the protein fraction (denser) was isolated separately and immediately stored at -80ÂºC. Total protein precipitation was performed by adding 10% trichloroacetic acid (TCA) in acetone, followed by 3 washing cycles with acetone containing 20 mM dithiothreitol (DTT) and centrifuged at 15 000 g for 10 min. Protein pellet was dissolved in a buffer containing 8M urea, 1% SDS (1:1) and protease inhibitor (1:25) and resuspended by sonication, followed by centrifugation at 3 200g for 10 min to remove insoluble particles. Thereafter, protein samples were sent for analysis at EMBL Proteomics Core Facility in Heidelberg, Germany. LC-MS/MS analysis. Samples were subjected to an in-solution tryptic digest using a modified version of the Single-Pot Solid-Phase-enhanced Sample Preparation (SP3) protocol (PMID: 25358341, PMID: 29565595). To this end, samples were added to Sera-Mag Beads (Thermo Scientific, #4515-2105-050250, 6515-2105-050250) in 10 Ã‚Âµl 15% formic acid and 30 Ã‚Âµl of ethanol. Binding of proteins was achieved by shaking for 15 min at room temperature. SDS was removed by 4 subsequent washes with 200 Ã‚Âµl of 70% ethanol. Proteins were digested overnight at room temperature with 0.4 Ã‚Âµg of sequencing grade modified trypsin (Promega, #V5111) in 40 Ã‚Âµl Hepes/NaOH, pH 8.4 in the presence of 1.25 mM TCEP and 5 mM chloroacetamide (Sigma-Aldrich, #C0267). Beads were separated, washed with 10 Ã‚Âµl of an aqueous solution of 2% DMSO and the combined eluates were dried down. Peptides were reconstituted in 10 Ã‚Âµl of H2O and reacted for 1 h at room temperature with 80 Ã‚Âµg of TMT10plex (Thermo Scientific, #90111) (PMID: 24579773) label reagent dissolved in 4 Ã‚Âµl of acetonitrile. Excess TMT reagent was quenched by the addition of 4 Ã‚Âµl of an aqueous 5% hydroxylamine solution (Sigma, 438227). Peptides were reconstituted in 0.1 % formic acid, mixed to achieve a 1:1 ratio across all TMT-channels and purified by a reverse phase clean-up step (OASIS HLB 96-well Ã‚ÂµElution Plate, Waters #186001828BA). Peptides were subjected to an off-line fractionation under high pH conditions (PMID: 25358341). The resulting 12 fractions were then analyzed by LC-MS/MS on an Orbitrap Fusion Lumos mass spectrometer (Thermo Scentific) as previously described (PMID:30858367). To this end, peptides were separated using an Ultimate 3000 nano RSLC system (Dionex) equipped with a trapping cartridge (Precolumn C18 PepMap100, 5 mm, 300 ÃŽÂ¼m i.d., 5 ÃŽÂ¼m, 100 Ãƒâ€¦) and an analytical column (Acclaim PepMap 100. 75 Ãƒâ€” 50 cm C18, 3 mm, 100 Ãƒâ€¦) connected to a nanospray-Flex ion source. The peptides were loaded onto the trap column at 30 Ã‚Âµl per min using solvent A (0.1% formic acid) and eluted using a gradient from 2 to 40% Solvent B (0.1% formic acid in acetonitrile) over 2 h at 0.3 Ã‚Âµl per min (all solvents were of LC-MS grade). The Orbitrap Fusion Lumos was operated in positive ion mode with a spray voltage of 2.4 kV and capillary temperature of 275 Ã‚Â°C. Full scan MS spectra with a mass range of 375Ã¢â‚¬â€œ1500 m/z were acquired in profile mode using a resolution of 120,000 (maximum fill time of 50 ms or a maximum of 4e5 ions (AGC) and a RF lens setting of 30%. Fragmentation was triggered for 3 s cycle time for peptide like features with charge states of 2Ã¢â‚¬â€œ7 on the MS scan (data-dependent acquisition). Precursors were isolated using the quadrupole with a window of 0.7 m/z and fragmented with a normalized collision energy of 38. Fragment mass spectra were acquired in profile mode and a resolution of 30,000 in profile mode. Maximum fill time was set to 64 ms or an AGC target of 1e5 ions). The dynamic exclusion was set to 45 s. Acquired data were analyzed using IsobarQuant (PMID: 26379230) and Mascot V2.4 (Matrix Science) using a reverse UniProt FASTA Homo sapiens (UP000005640) or Mus musculus (UP000000589) database including common contaminants. The following modifications were taken into account: Carbamidomethyl (C, fixed), TMT10plex (K, fixed), Acetyl (N-term, variable), Oxidation (M, variable) and TMT10plex (N-term, variable). The mass error tolerance for full scan MS spectra was set to 10 ppm and for MS/MS spectra to 0.02 Da. A maximum of 2 missed cleavages were allowed. A minimum of 2 unique peptides with a peptide length of at least seven amino acids and a false discovery rate below 0.01 were required on the peptide and protein level (PMID: 25987413).

### Data Protocol
Acquired data were analyzed using IsobarQuant (Franken et al., 2015), Mascot V2.4 (Matrix Science) and a reverse UniProt FASTA Mus musculus (UP000000589) database, including common contaminants. The following modifications were considered: Carbamidomethyl (C, fixed), TMT10plex (K, fixed), Acetyl (N-term, variable), Oxidation (M, variable) and TMT10plex (N-term, variable). The mass error tolerance for full scan MS spectra was set to 10 ppm and for MS/MS spectra to 0.02 Da. A maximum of 2 missed cleavages were allowed. A minimum of 2 unique peptides with a peptide length of at least seven amino acids and a false discovery rate below 0.01 was required on the peptide and protein level (Savitski et al., 2015). 9083 proteins were identified, from which 6748 proteins were quantified. The raw output files of IsobarQuant were processed using the R programming language (Team, 2020). As a quality filter, we only considered proteins that were quantified with at least two unique peptides. Raw TMT reporter ion intensities [Average expression (signal_sum)] were normalized using variance stabilization normalization (Huber et al., 2002). Differential expression was evaluated by computing the respective ratio of normalized TMT signals.

### Publication Abstract
None

### Keywords
Neuron-microglia crosstalk; mirna intercellular signaling; neuroimmune function; vesicular traficking

### Affiliations
EMBL
Neuroinflammation, Signaling and Neuroregeneration Laboratory, Research Institute for Medicines (iMed.ULisboa), Faculty of Pharmacy, Universidade de Lisboa, 1649-003 Lisboa, Portugal

### Submitter
Frank Stein

### Lab Head
Dr Dora Brites
Neuroinflammation, Signaling and Neuroregeneration Laboratory, Research Institute for Medicines (iMed.ULisboa), Faculty of Pharmacy, Universidade de Lisboa, 1649-003 Lisboa, Portugal


