### Accession
PXD015331

### Title
Transcriptional regulation of ER-phagy by MiT/TFE factors controls chondrocyte differentiation and bone formation

### Description
Lysosomal-autophagic degradation of Endoplasmic Reticulum via autophagy (ER-phagy) is emerging as critical regulator of ER homeostasis and function. However, the molecular mechanisms governing ER-phagy are still unknown. Working in chondrocytes, we found that ER-phagy and lysosome biogenesis are co-activated by FGF signaling during hypertrophic differentiation, a mandatory step for bone formation. FGF induced ER-phagy trough IRS1-dependent inhibition of the insulin signaling and activation of MiT/TFE transcription factors, master regulators of lysosome biogenesis. MiT/TFE promoted ER-phagy through the induction of the ER-phagy receptor FAM134B. Notably, the activation of ER-phagy promotes chondrocytes differentiation and secretion of factors required for cartilage replacement by bone. Consistently, medaka fish knock-down for FAM134B have impaired ossification of cranial bones. Thus,  ER-phagy is a transcriptionally regulated process that participates to cell differentiation during development.

### Sample Protocol
Cells were lysed in SDC lysis buffer containing 4% (w/v) SDC, 100 mM Tris -HCl (pH 8.5). Proteome preparation was done using the in StageTip (iST) method (Kulak et al., 2014). Phosphopeptides were enriched using the EasyPhos  workflow (Humphrey et al.2015). Peptides were desalted on STAGE tips and separated on a reverse phase column (50 cm, packed in-house with 1.9-μm C18- Reprosil-AQ Pur reversed-phase beads) (Dr Maisch GmbH) over 120 min single-run gradients at a flowrate of 350nl on an EASY-nLC 1200 system (Thermo Fisher Scientific) and analyzed by electrospray tandem mass spectrometry on a QExactive HFX (Thermo Fischer Scientific) using HCD based fragmentation. MS data was acquired using a data dependent top-15 method with maximum injection time of 20ms, a scan range of 300–1650Th, and an AGC target of 3e6. Survey scans were acquired at a resolution of 60,000. Resolution for HCD spectra was set to 15,000 with maximum ion injection time of 28ms and an underfill ratio of either 30%. Dynamic exclusion was set to 20s. Phosphopeptides were eluted with a 140min gradient. The maximum injection time was 20ms, at a scan range of 300–1650Th and an AGC target of 3e6. Sequencing was performed via higher energy collisional dissociation fragmentation with a target value of 1e5, and a window of 1.6Th. Survey scans were acquired at a resolution of 60,000. Resolution for HCD spectra was set to 15,000 with a maximum ion injection time of 120ms, and an underfill ratio of 40%. Dynamic exclusion was set to 40s, and apex trigger (4 to 7s) was enabled.

### Data Protocol
Raw mass spectrometry data were processed with MaxQuant version 1.5.5.2 using default settings (FDR 0.01, oxidized methionine (M) and acetylation (protein N-term) as variable modifications, and carbamidomethyl (C) as fixed modification). Label free quantitation (LFQ) and “Match between runs” were enabled. For phosphoproteomic analysis phospho STY was used as variable modification. Bioinformatics analysis was performed with Perseus 1.5.4.2. Annotations were extracted from UniProtKB, Gene Ontology (GO), and the Kyoto Encyclopedia of Genes and Genomes (KEGG).

### Publication Abstract
Lysosomal degradation of the endoplasmic reticulum (ER) via autophagy (ER-phagy) is emerging as a critical regulator of cell homeostasis and function. The recent identification of ER-phagy receptors has shed light on the molecular mechanisms underlining this process. However, the signaling pathways regulating ER-phagy in response to cellular needs are still largely unknown. We found that the nutrient responsive transcription factors TFEB and TFE3-master regulators of lysosomal biogenesis and autophagy-control ER-phagy by inducing the expression of the ER-phagy receptor FAM134B. The TFEB/TFE3-FAM134B axis promotes ER-phagy activation upon prolonged starvation. In addition, this pathway is activated in chondrocytes by FGF signaling, a critical regulator of skeletal growth. FGF signaling induces JNK-dependent proteasomal degradation of the insulin receptor substrate 1 (IRS1), which in turn inhibits the PI3K-PKB/Akt-mTORC1 pathway and promotes TFEB/TFE3 nuclear translocation and enhances FAM134B transcription. Notably, FAM134B is required for protein secretion in chondrocytes, and cartilage growth and bone mineralization in medaka fish. This study identifies a new signaling pathway that allows ER-phagy to respond to both metabolic and developmental cues.

### Keywords
Er-phagy, Phosphoprotemics, Fgf signaling

### Affiliations
Max Planck of Biochemistry
Telethon Institute of Genetics and Medicine (TIGEM), Pozzuoli, Italy. Department of Medical and Translational Science, University of Naples "Federico II", Italy.

### Submitter
Natalie Krahmer

### Lab Head
Dr Carmine Settembre
Telethon Institute of Genetics and Medicine (TIGEM), Pozzuoli, Italy. Department of Medical and Translational Science, University of Naples "Federico II", Italy.


