### Accession
PXD035661

### Title
Regulation of Rad52-dependent replication fork recovery through serine ADP-ribosylation of PolD3

### Description
Although Poly(ADP-ribose)-polymerases (PARPs) are key regulators of genome stability, how site-specific ADP-ribosylation regulates DNA repair is unclear. Here, we describe a novel role for PARP1 and PARP2 in regulating Rad52-dependent replication fork repair to maintain cell viability when HR is dysfunctional, suppress replication-associated DNA damage, and maintain genome stability. Mechanistically, Mre11 is required for induction of PARP activity in response to replication stress that in turn promotes break-induced replication (BIR) through assembly of Rad52 at stalled/damaged replication forks. Further, by mapping ADP-ribosylation sites induced upon replication stress, we identify that PolD3 is a target for PARP1/PARP2 and importantly, that its site-specific ADP-ribosylation is required for BIR activity, replication fork recovery and genome stability. Overall, these data identify a critical role for Mre11-dependent PARP activation and site-specific ADP-ribosylation in regulating BIR to maintain genome integrity during DNA synthesis.

### Sample Protocol
2.5x108 U2OS cells were left untreated or exposed to 2mM HU. Cells were washed twice in ice-cold PBS, harvested by gentle scraping, and collected by centrifugation (600g, 4Â°C, 5min). Pellets were suspended in 10 pellet volumes of lysis buffer (6M Gnd-HCl, 50mM Tris-HCl, pH 8.5) with alternating cycles of disruption by shaking and vortexing for 30s. Samples were processed as described previously (Larsen et al., 2018). Briefly, lysates were homogenised by sonication then pre-digested using Lys-C followed by trypsin digestion overnight prior to peptide purification and lyophilisation.  Potential PAR chains were reduced through PARG incubation overnight before affinity purification using a GST-tagged Af1521 macrodomain. TFA-eluted peptides were Stage-Tipped prior to MS analysis. In brief, all samples were analysed using an 80 min gradient on a nanoscale EASY-nLC 1200 system connected to a Orbitrap Fusion Lumos mass spectrometer (ThermoFisher). Full scans were performed at a resolution of 120,000 and an injection time of 250 ms, with MS/MS spectra measured in the Orbitrap at a resolution of 60,000 and an injection time of 1,000 ms.

### Data Protocol
Analysis of the mass spectrometry raw data was performed using MaxQuant software (version 1.5.3.30). MaxQuant default settings were used, with exceptions outlined below. N-terminal acetylation, methionine oxidation, cysteine carbamidomethylation, and ADP-ribosylation on all amino acid residues known to potentially be modified (C, D, E, H, K, R, S, T, and Y), were included as variable modifications. Up to 6 missed cleavages were allowed, and a maximum allowance of 4 variable modifications per peptide was used. Second peptide search was enabled (default), and matching between runs was enabled with a match time window of 0.7 minutes and an alignment time window of 20 minutes. Modified peptides were filtered to have an Andromeda score of >40 (default), and a delta score of >20. Data was automatically filtered by posterior error probability to achieve a false discovery rate of <1% (default), at the peptide-spectrum match, the protein assignment, and the site-specific levels.

### Publication Abstract
Although Poly(ADP-ribose)-polymerases (PARPs) are key regulators of genome stability, how site-specific ADP-ribosylation regulates DNA repair is unclear. Here, we describe a novel role for PARP1 and PARP2 in regulating Rad52-dependent replication fork repair to maintain cell viability when homologous recombination is dysfunctional, suppress replication-associated DNA damage, and maintain genome stability. Mechanistically, Mre11 and ATM are required for induction of PARP activity in response to replication stress that in turn promotes break-induced replication (BIR) through assembly of Rad52 at stalled/damaged replication forks. Further, by mapping ADP-ribosylation sites induced upon replication stress, we identify that PolD3 is a target for PARP1/PARP2 and that its site-specific ADP-ribosylation is required for BIR activity, replication fork recovery and genome stability. Overall, these data identify a critical role for Mre11-dependent PARP activation and site-specific ADP-ribosylation in regulating BIR to maintain genome integrity during DNA synthesis.

### Keywords
Adp-ribosylation, Parps, Adp-ribosyltransferases, Proteomics, Af1521, Dna replication, Dna repair, Adpr, Lumos

### Affiliations
Proteomics Program, Novo Nordisk Foundation Center for Protein Research, University of Copenhagen
NNF Center for Protein Research

### Submitter
Sara Larsen

### Lab Head
Dr Michael Lund Nielsen
Proteomics Program, Novo Nordisk Foundation Center for Protein Research, University of Copenhagen


