### Accession
PXD041002

### Title
TMT data from ovarian cancer cells

### Description
In this project we want to compare ovarian cell cancer treated or non treated with cis-platinum. As acquisition strategy we have used a Real Time Search MS3 method in an Orbitrap Eclipse. The acquisition cycle began with an MS1 scanwhere the most intense ions were selected for fragmentation in the ion trap using CID. MS2 spectra were searched in real time with data acquisition using the sp-human database. MS2 spectra with an Xcorr greater than or equal to 1 and less than 10 ppm precursor mas error, triggered the submission of an MS3 spectrum to the instrument. MS3 spectrum, were collected using the multinotch MS3-based TMT method, in a way were ten MS2 fragment ions were captured in the MS3 precursor population using isolation waveforms with multiple frequency notches

### Sample Protocol
TMT mixes were fractionated using basic pH reversed-phase fractionation, 12 fractions were collected and analyzed in using an Orbitrap Eclipse mass spectrometer (Thermo Fisher Scientific, San Jose, CA, USA) coupled to an EASY-nLC 1000 (Thermo Fisher Scientific (Proxeon), Odense, Denmark) with a 90min gradient

### Data Protocol
Acquired spectra were analyzed using the Proteome Discoverer software suite (v2.4, Thermo Fisher Scientific) and the Mascot search engine (v2.6, Matrix Science ). Data was searched against a customized database including the Swiss-prot Human database plus a list  of common contaminants and all the corresponding decoy entries. For peptide identification a precursor ion mass tolerance of 7 ppm was used for MS1 level, trypsin was chosen as enzyme, and up to three missed cleavages were allowed. The fragment ion mass tolerance was set to 0.5 Da for MS2 spectra. Oxidation of methionine and N-terminal protein acetylation were used as variable modifications whereas carbamidomethylation on cysteines, TMT6plex in Lysines and TMT6plex in peptide N-terminal were set as a fixed modification. False discovery rate (FDR) in peptide identification was set to a maximum of 5%. Peptides were quantified using the reporter ions intensities in MS3. Reporter ion intensities were adjusted to correct for the isotopic impurities of the different TMT reagents according to manufacturer specifications.

### Publication Abstract
Interest in the use of machine learning for peptide fragmentation spectrum prediction has been strongly on the rise over the past years, especially for applications in challenging proteomics identification workflows such as immunopeptidomics and the full-proteome identification of data independent acquisition spectra. Since its inception, the MS&#xb2;PIP peptide spectrum predictor has been widely used for various downstream applications, mostly thanks to its accuracy, ease-of-use, and broad applicability. We here present a thoroughly updated version of the MS&#xb2;PIP web server, which includes new and more performant prediction models for both tryptic- and non-tryptic peptides, for immunopeptides, and for CID-fragmented TMT-labeled peptides. Additionally, we have also added new functionality to greatly facilitate the generation of proteome-wide predicted spectral libraries, requiring only a FASTA protein file as input. These libraries also include retention time predictions from DeepLC. Moreover, we now provide pre-built and ready-to-download spectral libraries for various model organisms in multiple DIA-compatible spectral library formats. Besides upgrading the back-end models, the user experience on the MS&#xb2;PIP web server is thus also greatly enhanced, extending its applicability to new domains, including immunopeptidomics and MS3-based TMT quantification experiments. MS&#xb2;PIP is freely available at https://iomics.ugent.be/ms2pip/.

### Keywords
Tmt cancer

### Affiliations
University Pompeu Fabra
1. Proteomics Unit, Universitat Pompeu Fabra, 08003 Barcelona, Spain 2. Proteomics Unit, Centre for Genomic Regulation, Barcelona Institute of Science and Technology (BIST), 08003 Barcelona, Spain

### Submitter
Cristina Chiva

### Lab Head
Dr Eduard Sabid√≥
1. Proteomics Unit, Universitat Pompeu Fabra, 08003 Barcelona, Spain 2. Proteomics Unit, Centre for Genomic Regulation, Barcelona Institute of Science and Technology (BIST), 08003 Barcelona, Spain


