### Accession
PXD038013

### Title
Quantitative proteome analysis to map the targets of small molecule UM171

### Description
Earlier we identified the protein degradation properties of small molecule UM171 (Subramaniam et al., 2020, Blood). Here we employed quantitative proteomics approach to map the global targets of UM171. Our study revealed multiple targets including the members of CoREST complex such as RCOR1 and LSD1.

### Sample Protocol
MM1s cells were treated with DMSO or 0.1 µM UM171 for 3 h. Cells were harvested by centrifugation and washed with phosphate buffered saline (PBS) before snap freezing in liquid nitrogen.   Cells were lysed by addition of lysis buffer (8 M Urea, 50 mM NaCl, 50 mM 4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid (EPPS) pH 8.5, Protease and Phosphatase inhibitors) and homogenization by bead beating (BioSpec) for three repeats of 30 seconds at 2400. Bradford assay was used to determine the final protein concentration in the clarified cell lysate. 50 µg of protein for each sample was reduced, alkylated and precipitated using methanol/chloroform as previously described (Donovan et al., 2018) and the resulting washed precipitated protein was allowed to air dry. Precipitated protein was resuspended in 4 M Urea, 50 mM HEPES pH 7.4, followed by dilution to 1 M urea with the addition of 200 mM EPPS, pH 8. Proteins were first digested with LysC (1:50; enzyme:protein) for 12 h at RT. The LysC digestion was diluted to 0.5 M Urea with 200 mM EPPS pH 8 followed by digestion with trypsin (1:50; enzyme:protein) for 6 h at 37 °C. Sample digests were acidified with formic acid to a pH of 2-3 prior to desalting using C18 solid phase extraction plates (SOLA, Thermo Fisher Scientific). Desalted peptides were dried in a vacuum-centrifuged and reconstituted in 0.1% formic acid for LC-MS analysis. Data were collected using a TimsTOF Pro2 (Bruker Daltonics, Bremen, Germany) coupled to a nanoElute LC pump (Bruker Daltonics, Bremen, Germany) via a CaptiveSpray nano-electrospray source. Peptides were separated on a reversed-phase C18 column (25 cm x 75 µm ID, 1.6 µM, IonOpticks, Australia) containing an integrated captive spray emitter. Peptides were separated using a 50 min gradient of 2 - 30% buffer B (acetonitrile in 0.1% formic acid) with a flow rate of 250 nL/min and column temperature maintained at 50 ºC. DDA was performed in Parallel Accumulation-Serial Fragmentation (PASEF) mode to determine effective ion mobility windows for downstream diaPASEF data collection (Meier et al., 2020). The ddaPASEF parameters included: 100% duty cycle using accumulation and ramp times of 50 ms each, 1 TIMS-MS scan and 10 PASEF ramps per acquisition cycle. The TIMS-MS survey scan was acquired between 100 – 1700 m/z and 1/k0 of 0.7 - 1.3 V.s/cm2. Precursors with 1 – 5 charges were selected and those that reached an intensity threshold of 20,000 arbitrary units were actively excluded for 0.4 min. The quadrupole isolation width was set to 2 m/z for m/z <700 and 3 m/z for m/z >800, with the m/z between 700-800 m/z being interpolated linearly. The TIMS elution voltages were calibrated linearly with three points (Agilent ESI-L Tuning Mix Ions; 622, 922, 1,222 m/z) to determine the reduced ion mobility coefficients (1/K0). To perform diaPASEF, the precursor distribution in the DDA m/z-ion mobility plane was used to design an acquisition scheme for DIA data collection which included two windows in each 50 ms diaPASEF scan. Data was acquired using sixteen of these 25 Da precursor double window scans (creating 32 windows) which covered the diagonal scan line for doubly and triply charged precursors, with singly charged precursors able to be excluded by their position in the m/z-ion mobility plane. These precursor isolation windows were defined between 400 - 1200 m/z and 1/k0 of 0.7 - 1.3 V.s/cm2.

### Data Protocol
The diaPASEF raw file processing and controlling peptide and protein level false discovery rates, assembling proteins from peptides, and protein quantification from peptides was performed using library free analysis in DIA-NN 1.8 (Demichev et al., 2020). Library free mode performs an in silico digestion of a given protein sequence database alongside deep learning-based predictions to extract the DIA precursor data into a collection of MS2 spectra. The search results are then used to generate a spectral library which is then employed for the targeted analysis of the DIA data searched against a Swissprot human database (January 2021). Database search criteria largely followed the default settings for directDIA including: tryptic with two missed cleavages, carbomidomethylation of cysteine, and oxidation of methionine and precursor Q-value (FDR) cut-off of 0.01. Precursor quantification strategy was set to Robust LC (high accuracy) with RT-dependent cross run normalization. Proteins with missing values in any of the treatments and with poor quality data were excluded from further analysis (summed abundance across channels of <100 and mean number of precursors used for quantification <2). Protein abundances were scaled using in-house scripts in the R framework (R Development Core Team, 2014) and statistical analysis was carried out using the limma package within the R framework (Ritchie et al., 2015)

### Publication Abstract
None

### Keywords
Protein degradation., Um171, Corest

### Affiliations
Dana-Farber Cancer Institute
Department of Cancer Biology, Dana-Farber Cancer Institute, Boston, MA 02215, USA., Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, MA 02115, USA

### Submitter
Eric Fischer

### Lab Head
Dr Eric Fischer
Department of Cancer Biology, Dana-Farber Cancer Institute, Boston, MA 02215, USA., Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, MA 02115, USA


