### Accession
PXD023178

### Title
Protein-protein interaction of Phytophthora infestans RxLR effector AvrCap1 by immunoafinity enrichment in planta

### Description
In plants, NLR (nucleotide-binding domain and leucine-rich repeat-containing) proteins can form receptor networks to confer hypersensitive cell death and innate immunity. One class of NLRs, known as NRCs (NLR required for cell death), are central nodes in a complex network that protects against multiple pathogens and comprises up to half of the NLRome of solanaceous plants. Given the prevalence of this NLR network, we hypothesized that pathogens convergently evolved to secrete effectors that target NRC activities. To test this, we screened a library of 167 bacterial, oomycete, nematode and aphid effectors for their capacity to suppress the cell death response triggered by the NRC-dependent disease resistance proteins Prf and Rpi-blb2. Among five of the identified suppressors, one cyst nematode protein and one oomycete protein suppress the activity of autoimmune mutants of NRC2 and NRC3, but not NRC4, indicating that they specifically counteract a subset of NRC proteins independently of their sensor NLR partners. Whereas the cyst nematode effector SPRYSEC15 binds the nucleotide-binding domain of NRC2 and NRC3, the oomycete effector AVRcap1b suppresses the response of these NRCs via the membrane trafficking-associated protein NbTOL9a (Target of Myb 1-like protein 9a). The interaction was studied by co-immunoprecipitation in plantacoupled with tandem mass spectrometry (IP-MS).

### Sample Protocol
Total proteins were extracted from N. benthamiana leaves three days after agroinfiltration of GFP::AVRcap1b or GFP::PexRD54 and subjected to immunoprecipitation using GFP_Trap_A beads (Chromotek, Munich, Germany), as described previously. PexRD54 was included as a control, as it is also a large P. infestans RxLR effector and extensive studies suggests that its role is likely independent of the NRC network [PXD020751]. Immunoprecipitated samples were separated by SDS-PAGE (4%â€“20% gradient gel, Biorad) and stained with Coomassie brilliant Blue G-250 (SimplyBlue Safe stain, Invitrogen). Enriched protein samples were cut out of the gel and digested with trypsin. Extracted peptides were analyzed by liquid chromatography-tandem mass spectrometry (LC-MS/MS) with the Orbitrap Fusion mass spectrometer and nanoflow-HPLC system U3000 (Thermo Fisher Scientific, UK). A total of three biological replicates for each protein was submitted.

### Data Protocol
Peak lists in the format of Mascot generic files (mgf files) were prepared from raw data using MS Convert (Proteowizard project) and sent to a peptide search on Mascot server 2.4.1 using Mascot Daemon (Matrix Science, Ltd.).  Peak lists were searched against protein databases including typical proteomics contaminants such as keratins, etc. Tryptic peptides with up to 2 possible mis-cleavages and charge states +2, +3, +4 were allowed in the search. The following peptide modifications were included in the search: carbamidomethylated Cysteine (static) and oxidized Methionine (variable). Data were searched with a monoisotopic precursor and fragment ion mass tolerance 10ppm and 0.6Da respectively. A decoy database was used to validate peptide sequence matches. Mascot results were combined in Scaffold 4.4.0 (Proteome Software Inc.) and exported to Excel (Microsoft) for further processing and comparisons. In Scaffold, the peptide and protein identifications were accepted if probability of peptide sequence match and protein inferred exceeded 95.0% and 99% respectively and at least 2 unique peptides identified each protein. Protein probabilities were calculated in Scaffold by the Protein Prophet algorithm; proteins that contained similar peptides and could not be differentiated based on MS/MS analysis alone were grouped to satisfy the principles of parsimony. Searle, B. C. (2010), Scaffold: A bioinformatic tool for validating MS/MS-based proteomic studies. PROTEOMICS, 10: 1265-1269. doi: 10.1002/pmic.200900437 PMID:20077414

### Publication Abstract
None

### Keywords
Rxlr effector, Rd54, Phytophthora infestans, Avrcap1

### Affiliations
The Sainsbury Laboratory, Norwich, UK
Proteomics group, The Sainsbury Laboratory, Colney lane, Norwich, NR4 7UH, United Kingdom

### Submitter
Jan Sklenar

### Lab Head
Dr Jan Sklenar
Proteomics group, The Sainsbury Laboratory, Colney lane, Norwich, NR4 7UH, United Kingdom


