### Accession
PXD039207

### Title
Identification of the novel progression factors in renal cell carcinoma  through global proteomics

### Description
Renal cell carcinoma (RCC) is among the top 8 most diagnosed cancer in United States, which 79,000 patients newly were diagnosed and 13,920 deaths [1]. The clear-cell RCC (ccRCC) is the most common histopathological subtype representing approximately 85~90% in all diagnosed case [2]. In renal cancer treatment, metastasis is considered the most serious problem. Metastatic RCC (mRCC) is a fetal disease. It is known that about 25~30% of patients diagnosed with early kidney cancer have metastasized [3, 4]. The 5-year survival rate of patients with mRCC is 11.7%, and the prognosis is very poor [5]. Approximately 30% of patients have recurrence despite removal of the primary tumor, including 10-15% of patients found in T1 and clinically removed [4]. The mRCC patients show poor response to chemotherapy and radiotherapy, the rate of disappointing for treatment is increased from 15% to 25% [6, 7]. The common sites of RCC metastasis is known to the lung (40%), bone (30%), lymph node (22%) and liver (20%) [8].

### Sample Protocol
Total protein was isolated from three experimental samples and prepared in proteomic analysis through the use of a RIPA buffer (ThermoFisher Scientific, Waltham, MA, USA) containing inhibitors. The cell lysate was treated with 15mM dithiothreitol (DTT) at 56 ℃ for 30 min and alkylated with 60mM iodoacetamide (IAA) at room temperature in the dark for 1 h. before enzymatic digestion, protein purification by TCA solution were conducted for each sample. Then, Trypsin (Thermo Fisher Scientific, Rockford, IL, USA) was added to each and digestion was conducted at 37 ℃ for 18 h, the protein-to-enzyme ratio was 50:1. After digestion, 1% TFA (v/v) was added to each sample to quenching the reaction

### Data Protocol
Gene ontology (GO) analysis of identified Kaa proteins was performed using DAVID software (https://david.ncifcrf.gov/), signal pathways were imported into the Kyoto Encyclopaedia of Genes and Genomes (KEGG) database for analysis. The functional analysis of the identified Kaa proteins was explored using protein-protein interaction (PPI) explored from the STRING database (http://www.string-db.org/) and Cluego in cytoscape. Sequence preference motif analysis for Kaa was conducted using pLOGO (https://plogo.uconn.edu/) using the human proteome as the background.Gene ontology (GO) analysis of identified Kaa proteins was performed using DAVID software (https://david.ncifcrf.gov/), signal pathways were imported into the Kyoto Encyclopaedia of Genes and Genomes (KEGG) database for analysis. The functional analysis of the identified Kaa proteins was explored using protein-protein interaction (PPI) explored from the STRING database (http://www.string-db.org/) and Cluego in cytoscape. Sequence preference motif analysis for Kaa was conducted using pLOGO (https://plogo.uconn.edu/) using the human proteome as the background.

### Publication Abstract
None

### Keywords
Human, Renal carcinoma cell

### Affiliations
Kyungpook national university
kyungpook national university

### Submitter
Ann-Yae Na

### Lab Head
Dr Sangkyu Lee
Kyungpook national university


