### Accession
PXD007040

### Title
Identification of FOXG1 interaction partners

### Description
Rett syndrome is a complex neurodevelopmental disorder that is mainly caused by mutations in MECP2. However, mutations in FOXG1 cause a less frequent non-congenital form called atypical Rett syndrome. FOXG1 is a key transcription factor implicated in forebrain development, where it maintains the balance between progenitor proliferation and neuronal differentiation. Using SILAC based quantitative proteomics and genome-wide small RNA sequencing, we identified that FOXG1 interacts with the ATP-dependent RNA helicase, DDX5/p68 and controls the biogenesis of miRNAs. Both, FOXG1 and DDX5 bind to the miR200b/a/429 primary transcript and associate with the microprocessor complex, whereby DDX5 recruits FOXG1 to DROSHA. In vivo and in vitro experiments show that both FOXG1 and DDX5 are necessary for effective maturation of miR200b/a/429. RNAseq analyses of Foxg1-heterozygote hippocampi and miR200b/a/429 overexpressing Neuro-2a cells revealed that the cAMP-dependent protein kinase type II-beta regulatory subunit (PRKAR2B) is a target of miR200 in neural cells. Since it is known that PRKAR2B inhibits postsynaptic functions by attenuating protein kinase A (PKA) activity, increased PRKAR2B levels may contribute to neuronal dysfunctions in FOXG1 Rett syndrome.

### Sample Protocol
SILAC labeled samples were used for IP. Resulting proteins were processed by in-gel digest and tryptic digested. Samples analyzed by MS were measured on LTQ Orbitrap XL mass spectrometer coupled to an Agilent 1200 nanoflow-HPLC.

### Data Protocol
Software Xcalibur (Thermo Scientific) was were used for data acquisition. The MS raw data files were uploaded into the MaxQuant software version 1.4.1.2. A full length mouse database containing common contaminants was employed, carbamidomethylcysteine was set as fixed modification, and methionine oxidation and protein amino-terminal acetylation were set as variable modifications. Double SILAC was chosen as quantitation mode. Three miss cleavages were allowed, enzyme specificity was trypsin/P, and the MS/MS tolerance was set to 0.5 Da. The average mass precision of identified peptides was in general less than 1 ppm after recalibration. Peptide lists were further used by MaxQuant to identify and relatively quantify proteins using the following parameters: peptide, and protein false discovery rates (FDR) were set to 0.01, maximum peptide posterior error probability (PEP) was set to 0.1, minimum peptide length was set to 6, minimum number peptides for identification and quantitation of proteins was set to two of which one must be unique, and identified proteins have been re-quantified.

### Publication Abstract
Rett syndrome is a complex neurodevelopmental disorder that is mainly caused by mutations in MECP2. However, mutations in FOXG1 cause a less frequent form of atypical Rett syndrome, called FOXG1 syndrome. FOXG1 is a key transcription factor crucial for forebrain development, where it maintains the balance between progenitor proliferation and neuronal differentiation. Using genome-wide small RNA sequencing and quantitative proteomics, we identified that FOXG1 affects the biogenesis of miR200b/a/429 and interacts with the ATP-dependent RNA helicase, DDX5/p68. Both FOXG1 and DDX5 associate with the microprocessor complex, whereby DDX5 recruits FOXG1 to DROSHA. RNA-Seq analyses of Foxg1<sup>cre/+</sup> hippocampi and N2a cells overexpressing miR200 family members identified cAMP-dependent protein kinase type II-beta regulatory subunit (PRKAR2B) as a target of miR200 in neural cells. PRKAR2B inhibits postsynaptic functions by attenuating protein kinase A (PKA) activity; thus, increased PRKAR2B levels may contribute to neuronal dysfunctions in FOXG1 syndrome. Our data suggest that FOXG1 regulates PRKAR2B expression both on transcriptional and posttranscriptional levels.

### Keywords
Mouse, Rett syndrome, Neural development

### Affiliations
Core Facility Proteomics - ZBSA - Freiburg University

### Submitter
Veronica Dumit

### Lab Head
Dr Veronica Dumit
Core Facility Proteomics - ZBSA - Freiburg University


