### Accession
PXD026353

### Title
SCFFbxw5 targets kinesin-13 proteins to facilitate ciliogenesis

### Description
Using substrate screening on protein microarrays we identified the kinesin-13 Kif2c/MCAK as a novel target of the ubiquitin E3 ligase SCF-Fbxw5. In cells, SCF-Fbxw5 ubiquitylates MCAK for proteasomal degradation during G2/M thereby facilitating ciliogenesis in the following G0 phase. Here, we performed mass spectrometry analysis to identify lysine residues of recombinant MCAK that are specifically ubiquitylated in vitro by SCF-Fbxw5 in concert with the E2 enzyme Cdc34 based on diGly remnants after trypsin digestion.

### Sample Protocol
MCAK was expressed and purified from Sf21 cells. Briefly, cells were harvested after 2 day infection with the according baculoviral P2 stock, lysed by sonication in Buffer A (25 mM Hepes pH 7.4, 300 mM KCl, 2 mM MgCl2, 5 mM β-mercaptoethanol, 30 mM imidazole, 50 μM ATP, 1 μg/ml aprotinin/pepstatin/leupeptin and 1 mM PefaBloc (Roche)) and purified over Ni-NTA, following TEV cleavage of the His-Tag overnight. Afterwards, excessive TEV and His-Tag were removed with Ni-NTA and MCAK was further purified over a Superdex 200 10/300 GL column in Buffer B (25 mM Hepes, pH 7.4, 300 mM KCl, 2 mM MgCl2, 1 mM DTT, 1 μg/ml aprotinin/pepstatin/leupeptin). For ubiquitylation, SCF complexes were formed by incubating 2 µM of HisFbxw5/Skp1 subcomplexes with 2 µM neddylated Cul1/Rbx1 subcomplexes for 10 min at 4°C. Next, 4 µM of MCAK was incubated with 75 µM ubiquitin, 0.7 µM Uba1-His (E1), 1 µM Cdc34 (E2), 0.2 µM SCF-Fbxw5  (E3) and 10 mM ATP for 15 min at 30°C. Reaction was stopped by adding 4x sample buffer (50 mM Tris pH 6.8, 1% (w/v) SDS, 0.1% (w/v) bromophenol blue, 10% glycerol, 100 mM DTT) followed by boiling at 95°C for 5 min. Sample was run on a 8-17% gradient polyacrylamide gel and stained with Coomassie Brilliant Blue R-250 dye. In-gel digestion Protein bands of interest were manually excised from gels. The gel pieces were washed once with 60 µL of 1:1 (v/v) 50 mM triethylammonium bicarbonate buffer (TEAB; Sigma-Aldrich, Taufkirchen, Germany) and acetonitrile (ACN; Roth, Karlsruhe, Germany), pH 8.5 for 10 min and shrunk three times for 10 min each in 60 µL ACN and washed in 60 µL 50 mM TEAB, pH 8.5. Gel pieces were dehydrated with 60 µL 100% ACN. A total of 70 µL of 8 ng/µL in 50 mM TEAB trypsin solution (sequencing grade, Thermo-Fisher, Rockford, USA) was added to the dry gel pieces and incubated 4 h at 37°C. The reaction was quenched by addition of 20 µL of 0.1% trifluoroacetic acid (TFA; Biosolve, Valkenswaard, The Netherlands). The resulting peptides were extracted once for 15 min with 50 µL 1:1 (v/v) 50 mM TEAB and ACN, pH 8.5 and once for 15 min in 70 µL ACN. The supernatant from each extraction step was collected and dried in a vacuum concentrator diluted in 15 ul 0.1% TFA. 5 ul were injected into LC.  LC-MS measurements Nanoflow LC-MS2 analysis was performed with an Ultimate 3000 liquid chromatography system coupled to an Orbitrap Q Exactive mass spectrometer (Thermo-Fischer, Bremen, Germany). Samples were delivered to an in-house packed analytical column (inner diameter 75 µm x 20 cm; CS – Chromatographie Service GmbH, Langerwehe, Germany) filled with 1.9 µm ReprosilPur-AQ 120 C18 material (Dr. Maisch, Ammerbuch-Entringen, Germany). Solvent A was 0.1% formic acid (FA; ProteoChem, Denver, CO, USA) in H2O (Bisolve) and solvent B was composed of 0.1% FA (ProteoChem), 10% H2O (Biosolve) and 89.9% ACN (Biosolve). Sample was loaded to the analytical column for 20 min with 3% B at 550 nL/min flow rate. Peptides were separated with 25 min linear gradient (3-40% B) with flow rate of 300 nL/min. The Q Exactive mass spectrometer was operated in data-dependent acquisition mode, automatically switching between MS, acquired at 60,000 (m/z 400) resolution, and MS2 spectra, generated for up to 15 precursors with normalized collision energy of 27% in the HCD cell and measured in the Orbitrap at 15,000 resolution. The MS2 AGC target value was set to 105 with a maximum IT of 50 ms.

### Data Protocol
Raw files were analyzed using Proteome Discoverer with the Sequest (Thermo Fisher Scientific, San Jose, USA; version 2.5). Sequest was set up to search against Uniprot human database (retrieved in November, 2019), common contaminants and sequence of MCAK protein with trypsin as the digestion enzyme. A fragment ion mass tolerance was set to 0.02 Da and a parent ion mass tolerance to 10 ppm. The number of maximal allowed missed cleavages was set to 3. Carbamidomethylation of cysteine was specified as a fixed modification, deamidation of asparagine and glutamine, oxidation of methionine, GG-modification of lysin and acetylation, loss of methione and loss of methionine plus acetylation of the protein N-terminus were specified as variable modifications. Abundances were calculated as intensities.

### Publication Abstract
Microtubule depolymerases of the kinesin-13 family play important roles in various cellular processes and are frequently overexpressed in different cancer types. Despite the importance of their correct abundance, remarkably little is known about how their levels are regulated in cells. Using comprehensive screening on protein microarrays, we identified 161 candidate substrates of the multi-subunit ubiquitin E3 ligase SCF<sup>Fbxw5</sup> , including the kinesin-13 member Kif2c/MCAK. In vitro reconstitution assays demonstrate that MCAK and its closely related orthologs Kif2a and Kif2b become efficiently polyubiquitylated by neddylated SCF<sup>Fbxw5</sup> and Cdc34, without requiring preceding modifications. In cells, SCF<sup>Fbxw5</sup> &#xa0;targets MCAK for proteasomal degradation predominantly during G<sub>2</sub> . While this seems largely dispensable for mitotic progression, loss of Fbxw5 leads to increased MCAK levels at basal bodies and impairs ciliogenesis in the following G<sub>1</sub> /G<sub>0</sub> , which can be rescued by concomitant knockdown of MCAK, Kif2a or Kif2b. We thus propose a novel regulatory event of ciliogenesis that begins already within the G<sub>2</sub> phase of the preceding cell cycle.

### Keywords
Cullin-ring ligase / ubiquitin / fbxw5 / mcak / cilia

### Affiliations
ZMBH, Im Neuenheimer Feld 282, 69122 Heidelberg
ZMBH University Heidelberg Im Neuenheimer Feld 282 69120 Heidelberg Germany

### Submitter
Thomas Ruppert

### Lab Head
Dr Thomas Ruppert
ZMBH University Heidelberg Im Neuenheimer Feld 282 69120 Heidelberg Germany


