### Accession
PXD022993

### Title
Quantitative proteomic analysis of the cell envelope and cytosolic fraction of drug-resistant Neisseria gonorrhoeae

### Description
Neisseria gonorrhoeae (NG) exhibits high genome plasticity caused by an unusually high density and diversity of transposable elements, and easily performs various mechanisms of drug resistance. Here we investigated the i19.05 clinical isolate with reduced susceptibility to penicillin (MIC=0.5 mg/L), tetracycline (MIC=0.5 mg/L), and azithromycin (MIC=1.0 mg/L), which carried no known genetic resistance determinants except of penA, which cannot explain the expression of the resistant phenotype. In addition, it attracted our attention to the presence of a new and unique mutation of Asn105Ser in SurA and several mutations in Omp85 (BamA). The goal of our study was to search for new molecular mechanisms of drug resistance.  The pan susceptible n01.08 NG clinical isolate was involved as a control to compare, as well as a recipient in transformation procedure. The fragments of i19.05 genome contained mutant surA, omp85, and penA genes were amplified and used in spot-transformation of the n01.08 recipient isolate as described (Ilina, 2013). Finally, a resistant transformant NG05 (PenAmut, Ompmut, SurAmut) was obtained. For comprehensive proteomic analysis via LC-MS/MS, the proteins from the all tested N. gonorrhoeae strains were fractionated on cell envelope (CE) (including outer membrane, periplasmic, inner membrane) and cytosol (C). A total of 1125 proteins in the CE fraction, of which 894 were common in all strains were identified. Proteomics of the C fraction in the same experiment yielded a total of 928 proteins, of which 676 were shared among all strains. Proteome coverage for both fractions ranged from 52.72% (1111 proteins) in n01.08 to 54.53% (1149 proteins) in i19.05.

### Sample Protocol
Protein extraction and trypsin digestion. For proteomic analysis, the bacterial cells were harvested from Chocolate agar (BIOMERIEUX, France) into a 10ml ice-cold TE buffer (pH 8.1), storing this bacterial suspension for at least 30 min at Р В Р вЂ Р Р†РІР‚С™Р’В¬Р Р†Р вЂљРІвЂћСћ80Р В РІР‚в„ўР вЂ™Р’В°C. Cells were disrupted by 5-min ultrasonic disintegration in a Branson sonifier 250 (Branson Ultrasonics Corporation, USA) (duty cycle: 40%, output control: 7), unbroken cells were removed by centrifugation (12,000 g, 15 min, 4Р В РІР‚в„ўР вЂ™Р’В°C). Cell fractions were separated by ultracentrifugation for 8 min at 170,000 g. The soluble fraction was collected, the resulting pellet with membrane was dissolved in 1 ml of 2 mM Tris-HCl, pH 7.6 and used for further analysis. Protein concentration was measured by the Bradford method [Bradford, M.M. A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding. Anal. Biochem. 1976, 72, 248Р В Р вЂ Р В РІР‚С™Р Р†Р вЂљРЎС™254.] using the Bradford Protein Assay Kit (Bio-Rad, Hercules, USA). Proteolytic in-gel digestion was performed in three biological replicates as described previously [Bespyatykh, J.; Shitikov, E.; Butenko, I.; Altukhov, I.; Alexeev, D.; Mokrousov, I.; Dogonadze, M.; Zhuravlev, V.; Yablonsky, P.; Ilina, E.; et al. Proteome analysis of the Mycobacterium tuberculosis Beijing B0/W148 cluster. Sci. Rep. 2016, 6].  LC-MS/MS analysis. Analysis was performed on a Q-Exactive HF hybrid quadrupole-Orbitrap mass spectrometer (Thermo Fisher Scientific, Germany) with a nanoelectrospray source Nanospray Flex (Thermo Fisher Scientific, Germany) coupled to Ultimate 3000 RSLCnano (Dionex, USA) chromatography system. Liquid chromatographic separation was performed on a reverse phase column (15 cm Р В РІР‚СљР Р†Р вЂљРІР‚Сњ 75 Р В РЎвЂєР РЋРїС—Р…m i.d., Agilent Technologies, USA) packed with Zorbax 300SB-C18 resin (particle size Р В Р вЂ Р В РІР‚С™Р Р†Р вЂљРЎС™ 3 um, pore diameter Р В Р вЂ Р В РІР‚С™Р Р†Р вЂљРЎС™ 100 A). The HPLC system was configured in a trap-elute mode. Samples were loaded on a trap column a 2 cm Р В РІР‚СљР Р†Р вЂљРІР‚Сњ 75 Р В РЎвЂєР РЋРїС—Р…m i.d. Acclaimed PepMap column by Dionex, USA, with C18 resin with 3 um-particles with 100 A pores with 2 uL/min flow of solvent A (0.1% v/v formic acid) for 5 minutes. Peptides were eluted with a gradient of 5 to 40 % (v/v) of solvent B (0.1% v/v formic acid, 79.9% v/v acetonitrile) across 120 minutes at flow rate of 300 nl/min. After each elution system and columns were washed with 99% of solvent B for 10 minutes and regenerated with 5% of solvent B for 10 minutes. The mass-spectrometer was operated in positive mode in a data-dependent experiment with survey scans acquired at a resolution of 700 at m/z 400 within m/z range of 400-1500 with automatic gain control set for 106 and maximum injection time of 50 ms. As many as 20 of the most abundant precursor ions with a charge +2 and above from the survey scan were selected for HCD fragmentation. The normalized collision energy was 30. MS2 spectra were acquired at resolution of 17500 at m/z 400, automatic gain control was set for 105 and maximum injection time for 100 ms. After fragmentation ions were dynamically excluded from consideration for 10 s with 5 ppm window.

### Data Protocol
Raw data was captured from the mass spectrometer and converted to MGF (Mascot Generic Format) files using ProteoWizard with the following parameters: peakPicking true 2, msLevel 2, zeroSamples removeExtra. For thorough protein identification and quantification raw LC-MS/MS data were analyzed with MaxQuant (version 1.6.10.43) against UniProt knowledgebase, taxon Neisseria gonorrhoeae (strain ATCC 700825 / FA 1090) [242231]. For this procedure, we use the following parameters: Orbitrap instrument type, tryptic digestion with two possible missed cleavages, fixed modification for carbamidomethyl (C), variable modifications for oxidation (M) and acetyl (protein N-term), LFQ label-free quantification. A 1% FDR threshold was applied to search results from individual datasets. Frequently observed contaminants, such as trypsin, bovine proteins and human keratins, were removed from the results.

### Publication Abstract
There is growing concern about the emergence and spread of multidrug-resistant <i>Neisseria gonorrhoeae.</i> To effectively control antibiotic-resistant bacterial pathogens, it is necessary to develop new antimicrobials and to understand the resistance mechanisms to existing antibiotics. In this study, we discovered the unexpected onset of drug resistance in <i>N. gonorrhoeae</i> caused by amino acid substitutions in the periplasmic chaperone SurA and the &#x3b2;-barrel assembly machinery component BamA. Here, we investigated the i19.05 clinical isolate with mutations in corresponding genes along with reduced susceptibility to penicillin, tetracycline, and azithromycin. The mutant strain NG05 (<i>surA<sup>mut</sup> bamA<sup>mut</sup></i>, and <i>penA<sup>mut</sup></i>) was obtained using the pan-susceptible n01.08 clinical isolate as a recipient in the transformation procedure. Comparative proteomic analysis of NG05 and n01.08 strains revealed significantly increased levels of other chaperones, Skp and FkpA, and some transport proteins. Efflux pump inhibition experiments demonstrated that the reduction in sensitivity was achieved due to the activity of efflux pumps. We hypothesize that the described mutations in the <i>surA</i> and <i>bamA</i> genes cause the qualitative and quantitative changes of periplasmic chaperones, which in turn alters the function of synthesized cell envelope proteins.

### Keywords
Resistance, Neisseria gonorrhoeae, Periplasmic chaperones, Neisseria, Gonorrhea, Shotgun proteomics

### Affiliations
Scientific Research Institute of Physical-Chemical Medicine, FMBA
Federal Research and Clinical Center of Physical-Chemical Medicine of Federal Medical Biological Agency

### Submitter
Georgij Arapidi

### Lab Head
Dr Georgij Arapidi
Scientific Research Institute of Physical-Chemical Medicine, FMBA


