### Accession
PXD041524

### Title
A rapid and efficient method for the extraction of histone proteins

### Description
Current protocols used to extract and purify histones are notoriously tedious, especially when using yeast cells. Here, we describe the use of a simple filter-aided sample preparation approach enabling histone extraction from yeast and mammalian cells using acidified ethanol, a condition that not only improves extraction but also inactivates histone-modifying enzymes. We show that our improved method preserves the integrity of histones, and prevents N-terminal clipping of H3, an artefact frequently observed in yeast cells using standard histone extraction protocols.  Our method is scalable and provides efficient recovery of histones when extracts are prepared from as little as two million yeast cells. We further demonstrate the application of this approach for the analysis of histone modifications including fungal clinical isolates available in limited quantity. Altogether, this method enables the study of histones and their modifications in a faster, simpler, and more robust manner.

### Sample Protocol
Samples were loaded on a 1.5 uL pre-column (Optimize Technologies, Oregon City, OR). Peptides were separated on an home-made reversed-phase column (150-Î¼m i.d. x 200 mm) with a 56-min gradient from 10 to 30% acetonitrile (ACN)-0.2% formic acid and a 600 nL/min flow rate on a Easy nLC-1000 connected to a Q-Exactive HF (Thermo Fisher Scientific, San Jose, CA). For data-dependent acquisition, each full MS spectrum acquired at a resolution of 60,000 was followed by tandem-MS (MS/MS) spectra acquisition on the 15 most abundant multiply charged precursor ions. MS/MS experiments were performed using higher energy collision dissociation (HCD) at a collision energy of 27%.

### Data Protocol
All raw files were searched with PEAKS engine (Bioinformatics Solutions, Inc., Version 10) against the Uniprot human database (Jan 2021). We used thresholds of 1% FDR and at least 2 unique peptides per protein to generate a list of proteins/peptides and their relative intensities. Maximal tolerances for precursor and fragments were 10 ppm and 0.01 Da, respectively. Variable protein modification included: acetylation [K, N-term], propionylation [K], methylation (mono+propionylation, di, tri) [K], phosphorylation [S, T, Y], oxidation [M], deamidation [N, Q].

### Publication Abstract
Current protocols used to extract and purify histones are notoriously tedious, especially when using yeast cells. Here, we describe the use of a simple filter-aided sample preparation approach enabling histone extraction from yeast and mammalian cells using acidified ethanol, which not only improves extraction but also inactivates histone-modifying enzymes. We show that our improved method prevents N-terminal clipping of H3, an artifact frequently observed in yeast cells using standard histone extraction protocols. Our method is scalable and provides efficient recovery of histones when extracts are prepared from as few as two million yeast cells. We further demonstrate the application of this approach for the analysis of histone modifications in fungal clinical isolates available in a limited quantity. Compared with standard protocols, our method enables the study of histones and their modifications in a faster, simpler, and more robust manner.

### Keywords
Histone extratction, Fasp, N-terminal clipping

### Affiliations
IRIC-Universite de Montreal
Proteomic Platform

### Submitter
Eric Bonneil

### Lab Head
Dr Pierre Thibault
IRIC-Universite de Montreal


