### Accession
PXD044814

### Title
MinD-RNase E interplay controls localization of polar mRNAs in E. coli

### Description
We have recently shown that the polar transcriptome of E. coli is unique compared to the membrane and cytosolic ones. Several factors were suggested to mediate mRNA localization to the membrane, but the mechanism of localizing mRNAs to the poles is not known. We combined a candidate system approach with high-throughput proteomics to identify factors that mediate polar mRNAs localization. We identified MinD as an essential factor in this process, whose binding to RNAs is indirect. We demonstrate that RNase E, previously shown to interact with MinD, is also required for proper localization of polar mRNAs. Using in silico modeling followed by experimental validation, the membrane-binding site in RNase E was found to mediate binding to MinD. Intriguingly, not only does MinD affect RNase E interaction with the membrane, but it also affects its mode of action and dynamics. Polar accumulation of RNase E in Î”minCDE cells resulted in destabilization and depletion of mRNAs from the poles. Finally, we show that mislocalization of polar mRNAs may prevent polar localization of their protein products. Taken together, MinD and RNase E are assigned novel roles, whereas the interplay between them determines the composition of the polar transcriptome, thus affecting the polar proteome.

### Sample Protocol
MS2-bound proteins were subjected to on-bead digestion with trypsin. The complexes were reduced with 1 mM Dithiothreitol for 30 minutes, followed by alkylation with 5 mM Iodoacetamide for 30 min in the dark. Proteins were digested overnight with trypsin (Promega) and digestion was terminated by adding 0.1% trifluoroacetic acid (TFA). The peptides were desalted and concentrated on C18   stage tips. Prior to MS analysis, peptides were eluted from the stage tips using 80% acetonitrile, vacuum-concentrated and diluted in loading buffer (2% acetonitrile and 0.1% trifluoroacetic acid) and subjected for MS measurements. LC-MS/MS analysis was performed using nano-ultra high-performance liquid chromatography (nano-UPLC; Easy-nLC1000; Thermo Scientific) coupled on-line to a Q-Exactive Plus mass spectrometer (Thermo Scientific). Peptides were eluted using 240-minutes gradient with acetonitrile and water.

### Data Protocol
Raw files were analyzed with MaxQuant in-built with Andromeda search engine. MS/MS spectra were searched with reference to human UNIPROT database. False discovery rates were set for 0.01 for protein and peptide identification. All the statistical analyses of the MaxQuant output tables were performed with the Perseus software. Significant interactors of MS2-binding RNA sequence were examined by performing T-test for test transcripts against control transcripts with a p-value 0.01.

### Publication Abstract
None

### Keywords
Rna localization, E coli, Proteomics, Lc-ms/ms

### Affiliations
Department of Molecular Cell Biology, Weizmann Institute of Science, Rehovot, 76100001, Israel
Weizmann Institute of Science

### Submitter
Tamar Geiger

### Lab Head
Dr Tamar Geiger
Department of Molecular Cell Biology, Weizmann Institute of Science, Rehovot, 76100001, Israel


