### Accession
PXD000521

### Title
Protein abundance changes and ubiquitylation targets identified upon inhibition of the proteasome with Syringolin A

### Description
The numerous roles of the ubiquitin proteasome system (UPS) in cellular control have triggered interest in the identification of these ubiquitylation targets and their sites of ubiquitylation. For the identification of ubiquitylated proteins we employed affinity enrichment using an ubiquitin binding domains (UBAs). A change in the experimental set-up allowed for the identification of proteins that so far have been refractory to identification. We also investigated the changes in protein abundance upon interfering with the UPS by inhibition of the proteasome with the specific inhibitor Syringolin A and using a mutant overexpressing ubiquitin that cannot form K48-linked polyubiquitin chains.

### Sample Protocol
Frozen plant material was ground with a mortar and pestle. To 0.75 g fresh weight plant material, 3 ml of cold native protein extraction buffer was added. Proteins were extracted on ice for 20 min and the extracts were cleared afterwards by centrifugation at 4�C for 15 min at 16 200x g. To the pellet 1.5 ml of urea buffer was added at RT and the pellet was resuspended by vortexing. Proteins were extracted at RT for another 20 min, followed by centrifugation at RT for 10 min at 16 200x g. The supernatant was collected in dialysis bags and the urea concentration was reduced in a two-step dialysis to about 10 mM urea with a two-step dialysis. First, 500 �g protein in a total volume of 800 �l was pre-cleared with 100 �l washed sepharose CL4B under constant agitation for 2 h. Afterwards the sepharose was pelleted and half of the extract was added to 50 �l washed p62-derived UBA domain agarose conjugate, while the other half was added to 50 �l sepharose CL4B as a control. After incubation for 2 h under agitation the beads were pelleted, the supernatant was removed and the beads were washed 3 times. Proteins were eluted and subjected directly to SDS PAGE on 10% gels. After electrophoretic separation of the proteins, each lane was cut into 5 sections. Proteins were reduced with DTT followed  by carbamidomethylation with iodacetamide.  The proteins were then digested in-gel with trypsin solution for 16-19 h at 30�C. After digestion, the peptides were extracted and acetonitrile was vaporised before the peptides were desalted using Sep-Pak reverse-phase cartridges. Before subjection to mass spectrometric measurements, each sample was dissolved in 3% acetonitrile, 0.1% trifluoroacetic acid. Mass spectrometry measurements were performed on an LTQ OrbiTrap XL MS (Thermo Fisher) coupled to a NanoLC-AS1 (Eksigent) using electrospray ionisation. For LC separation a capillary column packed with 8 cm C18 beads with a diameter of 3 �m and a pore size of 100 � was used. Peptides were loaded on the column with a flow rate of 500 nl/min for 16 min and eluted by an increasing acetonitrile gradient from 3% acetonitrile to 50% acetonitrile for 60 min with a flow rate of 200 nl/min. Full scans were obtained using the FT Orbitrap at a range of 300-2000 mass/charge followed by MS/MS scans of the five highest parent ions. Dynamic exclusion was enabled at a duration of 120 s.

### Data Protocol
The acquired raw spectra were transformed to mgf data format and searched against the TAIR10 database (32, download on January 17th, 2011) with concatenated decoy database and supplemented with common contaminants (71,032 entries) using the Mascot algorithm (version 2.3.02) (Mascot Science). The search parameters used were: mass = monoisotopic, requirement for tryptic ends, 2 missed cleavages allowed, precursor ion tolerance = +/- 10 ppm, fragment ion tolerance = +/- 0.8 Da, variable modifications of methionine (M, PSI-MOD name: oxidation, mono ? = 15.995) and static modifications of cysteine (C, PSI-MOD name: iodoacetamide derivatized residue, mono ? = 57.0215). Peptide spectrum assignments with ionscore > 30 and expect value < 0.015, except those of known contaminants, were filtered for ambiguity. Peptides matching to several proteins were excluded from further analyses. This does not apply to different splice variants of the same protein or to different loci sharing exactly the same amino acid sequence. All remaining spectrum assignments were inserted into the pep2pro database.

### Publication Abstract
As proteins are the main effectors inside cells, their levels need to be tightly regulated. This is partly achieved by specific protein degradation via the Ubiquitin-26S proteasome system (UPS). In plants, an exceptionally high number of proteins are involved in Ubiquitin-26S proteasome system-mediated protein degradation and it is known to regulate most, if not all, important cellular processes. Here, we investigated the response to the inhibition of the proteasome at the protein level treating leaves with the specific inhibitor Syringolin A (SylA) in a daytime specific manner and found 109 accumulated and 140 decreased proteins. The patterns of protein level changes indicate that the accumulating proteins cause proteotoxic stress that triggers various responses. Comparing protein level changes in SylA treated with those in a transgenic line over-expressing a mutated ubiquitin unable to form polyubiquitylated proteins produced little overlap pointing to different response pathways. To distinguish between direct and indirect targets of the UPS we also enriched and identified ubiquitylated proteins after inhibition of the proteasome, revealing a total of 1791 ubiquitylated proteins in leaves and roots and 1209 that were uniquely identified in our study. The comparison of the ubiquitylated proteins with those changing in abundance after SylA-mediated inhibition of the proteasome confirmed the complexity of the response and revealed that some proteins are regulated both at transcriptional and post-transcriptional level. For the ubiquitylated proteins that accumulate in the cytoplasm but are targeted to the plastid or the mitochondrion, we often found peptides in their target sequences, demonstrating that the UPS is involved in controlling organellar protein levels. Attempts to identify the sites of ubiquitylation revealed that the specific properties of this post-translational modification can lead to incorrect peptide spectrum assignments in complex peptide mixtures in which only a small fraction of peptides is expected to carry the ubiquitin footprint. This was confirmed with measurements of synthetically produced peptides and calculating the similarities between the different spectra.

### Keywords
Quantitative proteomics, Ubiquitylation, Ubiquitin proteasome system, Affinity enrichment, Syringolin a

### Affiliations
ETH Zurich
Institute of  Plant Science

### Submitter
Katja Baerenfaller

### Lab Head
Dr Katja Baerenfaller
ETH Zurich


