### Accession
PXD016699

### Title
Integrated of proteomics data refines key adaptation and regulatory processes during nitrogen starvation of Nannochloropsis oceanicaIntegrated of proteomics data refines key adaptation and regulatory processes during nitrogen starvation of Nannochloropsis oceanica

### Description
Under nitrogen deficiency situation, Nannochloropsis spp. accumulate large amounts of lipid in the form of triacylglycerides (TAG). Several researches have studied the mechanism of this process from the perspective of transcriptome and metabolome, yet proteome analysis on this process is still sparse and lacking the analysis of the dynamic adaption to nitrogen deficiency. Here, proteomes for 03h, 06h, 12h, 24h, 48h and 10th day of nitrogen deplete and replete conditions were compared. New proteome results were integrated with existing transcriptome and other data.

### Sample Protocol
Protein extraction, preparation  Nannochloropsis cells were collected by centrifugation at 4℃ (Beckman Coulter Allegra X-12R, SX4750 Rotor, 2000g,10min), then the cell pellets were freeze dried, weighted and ground seven times with liquid nitrogen. A tablet phosphatase inhibitor including protease inhibitor cocktails (Phos STOP, Roche) is dissolved in 1 mL of ddH2O. The inhibitor solution is mixed in a ratio of 1:9 with an SDS lysis buffer (4% SDS in 0.1 M Tris / HCl pH 7-8). The fragmented cells were mixed in the ratio 1:20 (w/v) with SDS lysis buffer upon thorough mixing and subsequent  incubation at 70 °C for 4 min , the sample was centrifuged at 4℃ (Eppendorf 5415R, rotor: F54-24-11, 12,000 rpm, 20 min). The supernatant contained the extracted proteins. and was stored on ice. The concentration of the protein was determined by the BioRad® DC assay kit with a BSA calibration line consisting of increasing concentrations of BSA in SDS lysis buffer.  One-dimensional 12.5% (v/v) polyacrylamide gel electrophoresis served for purification and solubilization of the protein. The separating gel was poured and covered with isopropanol for a straight gel front. The separating gel was polymerized for about 2 hours. Isopropanol was removed and the stacking gel was poured over the solid separating gel. After a comb for the gel pockets was inserted, the stacking gel was polymerized after another 2 hours. 50 µg protein sample was increased by using SDS loading buffer (3% w/v) loaded into one of each gel pockets, then the gel was run at room temperature, 300 V and 30 mA until bottom. Gel staining and discoloration Coomassie “Silver Blue stain” was used for protein visualization (Candiano, Bruschi et al., 2004). The gel was washed twice with distilled water for 10min to remove the SDS. Then the gel was dyed with Silver Blue stain solution overnight and washed twice with ddH2O after staining. Protein bands were excised from the stained gel, cut into small cubes (ca. 1 x 1 mm3) and distained in 150μl distaining solution (10% ethanol and 2% phosphoric acid) for 30min at 37°C and 550 rpm in Thermomixer (Comfort, Eppendorf). This procedure was repeated two to three times until the gel pieces and de-istaining solution no longer displayed a blue color. Thereupon, de-istaining solution was removed and the gel pieces were dried at 30 °C in a Speed Vac.  In gel digestion The dried gel pieces were completely immersed in digestion solution (~200 µl). The digestion solution consisted of sequencing grade modified porcine trypsin (Promega, Madison, USA), which was diluted in 40 mM ammonium bicarbonate (pH 8.6) to a concentration of 12.5 ng µl-1. The protein digestion was performed over night at 37 °C with tempered shaker (HLC MHR20, 550 rpm). After protein digestion, the peptides were extracted from the gel pieces as supernatant and were transferred to autosampler vials (12 x 32 mm2 glass screw neck vial, Waters, USA). The extracted peptides were dried using a Speed Vac at room temperature and stored at RT. LC-ESI-MS / MS analysis For MS analysis of the trypsin-digested proteins, dried peptides were resuspended in buffer A (0.1% formic acid (FA) in HPLC class water (Fischer Scientific, GmbH (Germany)) and sonicated for 10 min in an ultrasonic bath (RK-100 H, Heidolph). The LC-ESI-MS / MS system consisted of a nanoACQUITY gradient UPLC pump (Waters Corporation, USA) interfaced to an LTQ Orbitrap Elite mass spectrometer (Thermo Fisher Scientific, USA). For LC, an ACQUITY UPLC 2D VM M-Class Symmetry C18 trap column (100Å, 5μm, 180μm x 20 mm) (Waters Corporation, USA) was coupled to an HSS T3 ACQUITY UPLC M-Class separation column (75μm x 150mm) (Waters Corporation, USA). The nanospray source was a PicoTip Emitter Silica Tip (10μm tip ± 1μm) (New Objective, USA). Xcalibur (Version 2.2 SP1) was used for the software-based instrument control of the mass spectrometer. For the UPLC method, the flow rate was 0.4 μL/min. A 105min gradient was used with 0-5 min: 2% buffer B (0.1% formic acid in acetonitrile, UPLC/MS, Fischer Scientific, GmbH (Germany)); 5-10 min: 2-5% buffer B; 10-71 min: 5-30% buffer B; 72-77 min: 85% buffer B; 77-105 min: 2% buffer B. The analytical column oven was set to 55°C and the heated desolvation capillary was set to 275 °C. For the MS analysis, a full scan was first performed in the Orbitrap in the range of 150-2000 m/z, then the 20 most intensive precursor ions from the full scan were fragmented using the CID method (activation time 10ms and 35% collision energy). The resulting fragments were detected in the ion trap. All precursors of unknown charge or charge ≠ 2 or 3 were rejected for MS / MS analysis.

### Data Protocol
Protein identification and label-free quantification Max Quant (Version 1551) with Andromeda search engine was used for protein identification and label-free quantification (LFQ)[30, 76, 77]. The protein identification was against the complete proteome database of N. oceanica IMET1[18]. The mass tolerance for calibrated precursor ions was set to 4.5 ppm; the mass tolerance for fragment ions was set to 0.6 Da. Only tryptic peptides with up to two missed cleavages were accepted. The oxidation of methionine, acetylation on protein N-terminus and propionamide on cysteine were admitted as a variable peptide modification. The false discovery rate (FDR) was 0.01 for protein. For protein quantification, the “Label free quantification (Max LFQ)” function in MaxQuant was used[77]. This method is based on the “delayed normalization” strategy, which after the identification of peptides determines the normalization coefficients for each LC-MS/MS run.  For the comparison between N- and N+ proteomes, the LFQ-normalized intensities of samples, which were generated by MaxQuant, were transformed to log2 with the software Perseus [78]. For sample comparison, proteins not quantified in at least half of the time points for N- and N+ were removed. For the remaining 1795 proteins missing values were imputed using imputation from normal distribution in Perseus (width 0.3, down shift 1.8). The fold changes between N- and N+ samples were compared (i.e. log2 LFQ (N-/N+)) with two sample t-test (FDR=0.05). All the other analyses were carried out in the MATLAB® (2017, The MathWorks) and R® (3.6.1, R Core Team) environment for statistical computing and graphics.

### Publication Abstract
None

### Keywords
Lc ms/ms, Triacylglycerol, Lipid biosynthesis, Oleaginous algae, Photosynthesis

### Affiliations
Ruhr Uni Bochum
AG Biochemie der Pflanzen

### Submitter
Wuxin You

### Lab Head
Dr Ansgar Poetsch
AG Biochemie der Pflanzen


