### Accession
PXD022688

### Title
SP3 protocol for proteomic plant sample preparation prior LC-MS/MS

### Description
Quantitative protein extraction from biological samples, as well as contaminants removal before LC-MS/MS, is fundamental for the successful bottom-up proteomic analysis. Four sample preparation methods, including the ﬁlter-aided sample preparation (FASP), two single-pot solid-phase-enhanced sample preparations (SP3) on carboxylated or HILIC paramagnetic beads, and protein suspension trapping method (S-Trap) were evaluated for SDS removal from Arabidopsis thaliana (AT) lysate. Finally, the optimized carboxylated SP3 workflow was benchmarked closely against the routine FASP. Ultimately, LC-MS/MS analyses revealed that regarding the number of identifications, number of missed cleavages, proteome coverage, repeatability, reduction of handling time, and cost per assay, the SP3 on carboxylated magnetic particles proved to be the best alternative for SDS and other contaminants removal from plant sample lysate. A robust and efficient two-hour SP3 protocol for a wide range of protein input is presented, benefiting from no need to adjust amount of beads, binding and rinsing conditions, or digestion parameters.

### Sample Protocol
The initial comparison of sample preparation techniques for bottom-up proteomics utilizing 60 ug of AT total protein load was based on the previously described methods: SP3 on carboxylated beads (Hughes et al., 2014; Moggridge et al., 2018), SP3 on HILIC beads (Protocol for cleanup of reduced and alkylated proteins with on-bead digestion, retrieved from http://www.resynbio.com/hilic), S-Trap (Zougman et al., 2014), and FASP (Wiśniewski et al., 2009). Ultimate 3000 RSLCnano system on-line connected to Orbitrap Elite hybrid spectrometer (Thermo Fisher Scientific, Waltham, MA, USA) was utilized to method assessment analysis and for optimizing of SP3 Carboxy protocol. Prior to LC separation, tryptic digests were on-line concentrated and desalted using trapping column (100 μm × 30 mm, 40°C) filled with 3.5-μm X-Bridge BEH 130 C18 sorbent (Waters). After washing of trapping column with 0.1% formic acid (FA), the peptides were eluted from the trapping column onto an analytical column (Acclaim Pepmap100 C18, 3 µm particles, 75 μm × 500 mm, 40°C; Thermo Fisher Scientific) and separated by 120 min nonlinear gradient program (mobile phase A: 0.1% FA in water; mobile phase B: 0.1% FA in 80% Acetonitrile). Equilibration of the trapping column and the column was done prior to sample injection to sample loop. The analytical column outlet was directly connected to the Digital PicoView 550 (New Objective) ion source with sheath gas option and SilicaTip emitter (New Objective; FS360-20-15-N-20-C12) utilization. ABIRD (Active Background Ion Reduction Device, ESI Source Solutions) was installed. MS data were acquired in a data-dependent strategy selecting up to top 10 precursors based on precursor abundance in the survey scan. To measure the optimized SP3 Carboxy and FASP protocols (benchmark test) following material and methods were used. Mass spectrometry analysis of the peptide mixture was done using RSLCnano system (Thermo Fisher Scientific, Waltham, MA, USA) on-line connected to Orbitrap Q-Exactive HF-X system (Thermo Fisher Scientific, MA, USA). Prior to LC separation, tryptic digests were concentrated and desalted on-line using a cartridge trapping column (300 μm × 5 mm) filled with 5-μm particles C18 PepMap100 sorbent (Thermo Fisher Scientific, Waltham, MA, USA). The peptides were eluted from the trapping column onto an Acclaim Pepmap100 C18 analytical column (3-µm particles, 75 μm × 500 mm; ThermoFisher Scientific) and separated by the following gradient program (mobile phase A: 0.1% FA in water; mobile phase B: 0.1% FA in 80% acetonitrile, flow rate 300 nL/min). The gradient elution started at 2 % (0 – 5th min) of mobile phase B, increased from 2 % to 35 % (5th – 107th min), then increased linearly to 80% (107th – 115th min) of mobile phase B and remained at this state for the next 5 min. Equilibration of the trapping column and the column was done prior to sample injection to sample loop. For a quality control purpose, Biognosys iRT peptides were added during each analysis. The analytical column outlet was directly linked to the Digital PicoView 550 (New Objective) ion source with sheath gas option and SilicaTip emitter (New Objective; FS360-20-15-N-20-C12) utilization. ABIRD (Active Background Ion Reduction Device, ESI Source Solutions) was installed. MS data were acquired in a data-dependent strategy selecting up to top 20 precursors based on precursor abundance in the survey scan.

### Data Protocol
Processing of RAW data files for benchmark test (optimized SP3 Carboxy and FASP protocols) was performed using Maxquant (Cox and Mann, 2008) (MQ) proteomics platform (v. 1.6.10.43) with a built-in Andromeda search engine (Cox et al., 2011). Data were searched against Uniprot Arabidopsis thaliana proteome (version from 20190703) and MQ contaminant database. Potential contaminants, reverse sequences, and proteins identified by only peptide modification site were filtered out from the final protein group list. The first search peptide tolerance was set to 20 ppm. For the main search, 4.5 and 20 ppm were set for precursor and fragments ions as limited values. Individual peptide mass tolerance was applied. Carbamidomethylation of cysteine was set as a fixed modification, while oxidation (M) and deamidation (N, Q) were set as a variable. Trypsin was used as the protein-cleaving enzyme with one and two missed cleavages. Only peptides and proteins with false discovery rates (FDRs; q-values) < 1 % were considered for final data evaluation. Quality control analysis was done via PTXQC tool (Bielow et al., 2016) (v.1.0.1). The protein classification and mapping according to cellular components (GOCC, gene ontology cellular component) were done via Perseus software (v. 1.6.10.50). Arabidopsis thaliana protein annotations were downloaded from https://datashare.biochem.mpg.de/s/qe1IqcKbz2j2Ruf. For annotations, protein group identity was summarized for each repetition at the protein level. The GOCC frequency for each expression was counted and a relative content in the samples were calculated. Data were sorted in descending order.

### Publication Abstract
Quantitative protein extraction from biological samples, as well as contaminants removal before LC-MS/MS, is fundamental for the successful bottom-up proteomic analysis. Four sample preparation methods, including the filter-aided sample preparation (FASP), two single-pot solid-phase-enhanced sample preparations (SP3) on carboxylated or HILIC paramagnetic beads, and protein suspension trapping method (S-Trap) were evaluated for SDS removal and protein digestion from <i>Arabidopsis thaliana</i> (AT) lysate. Finally, the optimized carboxylated SP3 workflow was benchmarked closely against the routine FASP. Ultimately, LC-MS/MS analyses revealed that regarding the number of identifications, number of missed cleavages, proteome coverage, repeatability, reduction of handling time, and cost per assay, the SP3 on carboxylated magnetic particles proved to be the best alternative for SDS and other contaminants removal from plant sample lysate. A robust and efficient 2-h SP3 protocol for a wide range of protein input is presented, benefiting from no need to adjust the amount of beads, binding and rinsing conditions, or digestion parameters.

### Keywords
Arabidopsis thaliana, Sds removal, Carboxylated magnetic beads, Bottom-up, Sp3, Fasp, Protein cleanup, Mass spectrometry

### Affiliations
Mendel Centre for Plant Genomics and Proteomics, Central European Institute of Technology, Masaryk University, Kamenice 5, 625 00 Brno, Czech Republic National Centre for Biomolecular Research, Faculty of Science, Masaryk University, Kamenice 5, 625 00 Brno, Czech Republic
Proteomics Core Facility and Research Group Proteomics CEITEC-MU Masaryk University Brno, Czech Republic

### Submitter
Kamil Mikulasek

### Lab Head
Dr Zbyněk Zdráhal
Mendel Centre for Plant Genomics and Proteomics, Central European Institute of Technology, Masaryk University, Kamenice 5, 625 00 Brno, Czech Republic National Centre for Biomolecular Research, Faculty of Science, Masaryk University, Kamenice 5, 625 00 Brno, Czech Republic


