### Accession
PXD000609

### Title
Fractionation of proteins on non-aqueous gradient

### Description
Tissue powder was separated on a non-aqueous gradient to fractionate proteins and metabolites for subcellular locations. In total, twelve fractions (01-12) were collected from three independent gradients (A, B, C)

### Sample Protocol
Dried material from each fraction was solubilized in 6 M urea, 2 M thiourea, 10 mM Tris/HCl (pH 8) and protein concentration was determined by Bradford assay. Equal amounts of protein (20 µg) were reduced by DTT, and disulphide bonds were alkylated using iodoacetamide. Proteins were predigested for 3 h with endoproteinase Lys-C (0.5 µg µL-1; Wako Chemicals, Neuss) at room temperature. After 4-fold dilution with 10 mM Tris/HCl (pH 8), samples were digested further with 4 µL Sequencing Grade Modified trypsin (0.5 µg µL-1; Promega) overnight at room temperature. Subsequently, digested samples were acidified by addition of TFA to a final concentration of 1 % and desalted using C18 tips.

### Data Protocol
Tryptic peptide mixtures were analysed by LC-MS/MS using a nanoflow Easy-nLC (Thermo Scientific) and an Orbitrap hybrid mass spectrometer (LTQ-Orbitrap, Thermo Scientific) as a mass analyser. Peptides were eluted from a 75-μm analytical column (Reprosil C18, Dr. Maisch GmbH) with a linear gradient of 4 to 64 % acetonitrile over 120 min and sprayed directly into the mass spectrometer. Proteins were identified by MS/MS through information-dependent acquisition of fragmentation spectra of multiple charged peptides. Up to five data-dependent MS/MS spectra were acquired in the linear ion trap for each full-scan spectrum acquired at 60,000 full-width half-maximum resolution in the Orbitrap. Overall cycle time was approximately one second.Protein identification and ion intensity quantitation were carried out using MaxQuant version 1.3.0.5 (39). Spectra were matched against the Arabidopsis proteome (TAIR10, 35386 entries) as well as typical contaminants (e.g. human keratin, trypsin) using Andromeda. For peptide identification, carbamidomethylation of cysteine was set as a fixed modification, oxidation of methionine was set as a variable modification, and up to two missed cleavages were allowed. Mass tolerance for the database search was set to 20 ppm on full scans and 0.5 Da for fragment ions. Multiplicity was set to 1. For label-free quantitation, retention time matching between runs was chosen within a time window of two minutes. Peptide false discovery rate (FDR) and protein FDR were set to 0.01. Hits to contaminants (e.g. keratins) and reverse hits identified by MaxQuant were excluded from further analysis.

### Publication Abstract
Non-aqueous fractionation is a technique for the enrichment of different subcellular compartments derived from lyophilized material. It was developed to study the subcellular distribution of metabolites. Here we analyzed the distribution of about 1,000 proteins and 70 metabolites, including 22 phosphorylated intermediates in wild-type Arabidopsis rosette leaves, using non-aqueous gradients divided into 12 fractions. Good separation of plastidial, cytosolic, and vacuolar metabolites and proteins was achieved, but cytosolic, mitochondrial, and peroxisomal proteins clustered together. There was considerable heterogeneity in the fractional distribution of transcription factors, ribosomal proteins, and subunits of the vacuolar-ATPase, indicating diverse compartmental location. Within the plastid, sub-organellar separation of thylakoids and stromal proteins was observed. Metabolites from the Calvin-Benson cycle, photorespiration, starch and sucrose synthesis, glycolysis, and the tricarboxylic acid cycle grouped with their associated proteins of the respective compartment. Non-aqueous fractionation thus proved to be a powerful method for the study of the organellar, and in some cases sub-organellar, distribution of proteins and their association with metabolites. It remains the technique of choice for the assignment of subcellular location to metabolites in intact plant tissues, and thus the technique of choice for doing combined metabolite-protein analysis on a single tissue sample.

### Keywords
Non-aqueous fractionation, Subcellular location, Metabolite-protein interactions

### Affiliations
Metabolic Networks
Plant Systems Biology, Universitaet Hohenheim, Garbenstrasse 30

### Submitter
Waltraud Schulze

### Lab Head
Dr Waltraud Schulze
Plant Systems Biology, Universitaet Hohenheim, Garbenstrasse 30


