### Accession
PXD031793

### Title
Haptenation of Macrophage Migration Inhibitory Factor-Dataset 2

### Description
Firstly, the adductome of he protein macrophage migration inhibitory factor (MIF) with different haptens encompassing different reaction mechanisms and potencies was investigated. After incubation of MIF with the different haptens at 5 times molar excess of hapten, an untargeted Full MS/dd-MS2 experimental approach was used with a QExactive Quadrupole-Orbitrap mass spectrometer. The data were processed with Proteome Discoverer and the suggested sites of adductions were confirmed in parallel reaction monitoring mode (PRM) on the same instruments.

### Sample Protocol
MIF protein (0.32 nmol) was incubated separately with DNCB, PGE, CA, and CMIT/MIT at 5-fold molar ratio of hapten to protein for 24h. After that the excess of hapten was removed using cut off filters, the sample was washed with buffer, and the protein was reduced, alkylated and digested over night with Trypsin. After digestion samples were cleaned up using CC18 SPE catriges, evaporated to dryeness with reconstituted to appropriate solvent. Analysis was performed on a Dionex UltiMate 3000 UHPLC system coupled to a QExactive Quadrupole-Orbitrap mass spectrometer from Thermo Fisher Scientific. Peptides were separated on an AcclaimTM RSLC 120 C18 column (2.2 ÂµM, 150 mm x 2.1) from Thermo Scientific . The mobile phase consisted of (A) 0.1% FA in Milli-Q water containing 5% ACN and (B) 0.1% FA in ACN containing 5% Milli-Q water. Unargeted analysis was performed in positive ion mode in Full MS/dd-MS2 with an exclusion list based on the control untreated sample. Expected chromatographic peak width was 15 seconds, full MS was conducted at a resolution of 120,000, an AGC target of 1e6, a maximum IT of 30 ms and a scan range of 200 to 3000 m/z. dd- MS2 was performed at a resolution of 30,000, an AGC target of 1e5, a maximum IT of 50 ms, an isolation window of 2 m/z and a normalized collision energy of 25. all targeted analysis were performed in PRM mode at a resolution of 30,000, an AGC target of 1x105, a maximum IT of 50 ms, an isolation window of 2 m/z and a normalized collision energy of 30.

### Data Protocol
The only software tool used was Proteome Discoverer (version 2.3) in order to process the raw mass spectrometry data from the untargeted analysis (Full MS/dd-MS2) of the mouse MIF incubated with the different haptens. In the modified PWF Basic Sequest HT processing step, the raw data were searched against the UniProtKb/Swiss-Prot mouse Alb, Hb and MIF with variable modifications of oxidation at methionine, carboxymethylation of cysteine and the added masses of the haptens (1-Chloro-2,4 dinitrobenzene (DNCB), added mass 166.0015 Da), (1,2-Epoxy-3-phenoxypropane (PGE), added mass 150.0681 Da), (Cinnamic Aldehyde (CA), added masses 132.0575/114.0469 Da), (5-chloro-2-methyl-2h-isothiazolin-3-one (CMIT)/ 2-methyl-2h-isothiazol-3-one (MIT), added masses 115.0092/112.9935/99.03203 Da). Possible modification sites of haptens at the amino acid residues of Cys, His, Lys, Arg, Ser, Thr, Tyr, Pro and N-terminus of peptides. In the CWF basic consensus step, the generated peptide spectral matches were grouped, validated and filtered before turning into a peptide and protein report. The validation of the peptide spectral matches was set at a strict target false discovery rate of 0.01 and a relaxed target false discovery rate of 0.05. Peptides below the high confidence level were filtered out. Data from the quantification PRM experiments were handled manually using XCalibur from Thermo Fisher Scientific.

### Publication Abstract
None

### Keywords
Cmit/mit, Biomarker, Dncb, Mif, Ca, Pge

### Affiliations
Stockholm University

### Submitter
Lorena Ndreu

### Lab Head
Dr Isabella Karlsson
Stockholm University


