### Accession
PXD025279

### Title
The molecular appearance of native TRPM7 channel complexes identified by high-resolution proteomics

### Description
The transient receptor potential cation channel, subfamily M, member 7 (TRPM7) is a ubiquitously expressed membrane protein consisting of an ion channel domain and an intracellularly located protein kinase domain. TRPM7 plays a fundamental role in the cellular uptake of divalent cations such as Zn2+, Mg2+ and Ca2+, and thus shapes cellular excitability, plasticity and metabolic activity. The molecular appearance and operation of TRPM7 channel complexes in native tissues have remained unresolved. Here, we investigated the subunit composition of endogenous TRPM7 channels in rodent brain by multi-epitope affinity purification and high-resolution quantitative MS analysis. We found that native TRPM7 channels are high molecular-weight multi-protein complexes that contain the putative metal transporter proteins CNNM1-4 and a small G-protein ARL15. Heterologous reconstitution experiments confirmed the formation of TRPM7/CNNM/ARL15 ternary complexes and indicated that ARL15 effectively and specifically impacts TRPM7 channel activity. These results open up new avenues towards a mechanistic understanding of the cellular regulation and function of TRPM7 channels.

### Sample Protocol
Biochemistry Transient transfection of cultured cells: WT HEK293T cells were transfected with polyethylenimine using a DNA to polyethylenimine ratio of 1:2.5.  For transfection of TRPM7-/- HEK293T cells, plasmid DNA was diluted to 30 µg/ml in Hank’s balanced salt solution, precipitated by addition of 113 mM CaCl2 (final concentration) and added to the cells in culture medium lacking Blasticidin S, Puromycin and 10 mM MgCl2. For transfection, TRPM7, ARL15 and CNNM plasmid DNAs were mixed at a ratio of 3:1:1. Preparation of plasma membrane-enriched protein fractions: Freshly excised brains from adult rats or mice were homogenised in homogenisation buffer (320 mM sucrose, 10 mM Tris/HCl pH 7.4, 1.5 mM MgCl2, 1 mM EGTA and protease inhibitors (Leupeptin, Pepstatin A, Aprotinin (1 µg/ml each), 1 mM Phenylmethylsulfonyl fluoride, 1 mM Iodoacetamide), particulates removed by centrifugation at 1,080xg and homogenised material collected for 10 min at 200,000xg. After hypotonic lysis in 5 mM Tris/HCl pH 7.4 with protease inhibitors for 35 min on ice, the lysate was layered on top of a 0.5 and 1.3 M sucrose step-gradient in 10 mM Tris/HCl pH 7.4, 1 mM EDTA/EGTA, and the plasma membrane-enriched fraction collected after centrifugation (45 min, 123,000xg) at the interface. Membranes were diluted in 20 mM Tris/HCl pH 7.4, collected by centrifugation (20 min, 200,000xg), and resuspended in 20 mM Tris/HCl pH7.4. Cultured cells were harvested in phosphate buffer saline with protease inhibitors, collected by centrifugation (10 min, 500xg) and resuspended in homogenisation buffer. After sonication (2x 5 pulses, duty 50, output 2 (Branson Sonifier 250)), membranes were pelleted for 20 min at 125,000xg and resuspended in 20 mM Tris/HCl pH 7.4.  Immunoprecipitation: Membranes were resuspended in ComplexioLyte CL-47 or CL-91 solubilisation buffer (Logopharm) with added 1 mM EDTA/EGTA and protease inhibitors at a protein to a detergent ratio of 1:8 and incubated for 30 min on ice. Solubilised protein was cleared by centrifugation (10 min, 125000xg, 4°C) and incubated with antibodies cross-linked to Dynabeads (Invitrogen) by overhead rotation for 2 h on ice. After two short washing steps with CL-47, the captured protein was eluted in Laemmli buffer with dithiothreitol added after elution. Eluted proteins were separated by SDS-PAGE. For MS/MS analysis silver-stained protein lanes were cut-out, split at 50 kDa and pieces individually subjected to standard in-gel tryptic digestion. For chemiluminescent detection, proteins were western blotted onto PVDF membranes and probed with the following antibodies: anti-HA (11867423001, Roche), anti-Flag (F3165, Sigma), anti-ßActin (bs-0061R, Bioss Inc.). Blue-native Polyacrylamide gel electrophoresis: Two-dimensional BN-PAGE/SDS-PAGE protein analysis was performed. Membrane protein fractions were solubilised in ComplexioLyte CL-47 as described above, salts exchanged for aminocaproic acid by centrifugation through a sucrose gradient and samples loaded on non-denaturing 1-13% linear polyacrylamide gradient gels (anode buffer: 50 mM Bis-Tris, cathode buffer: 50 mM Tricine, 15 mM Bis-Tris, 0.02% Coomassie Blue G-250). For separation in the second dimension, individual gel lanes were isolated, equilibrated in 2x Laemmli buffer (10 min, 37°C), placed on top of SDS-PAGE gels and Western-probed using anti-TRPM7 (AB15562, Millipore).   Complexome profiling [see publication]  Mass spectrometry Tryptic digests (dried peptides) were dissolved in 0.5% (v/v) trifluoroacetic acid and loaded onto a C18 PepMap100 precolumn (300 µm i.d. × 5 mm; particle size 5 µm) with 0.05% (v/v) trifluoroacetic acid (5 min 20 µL/min) using split-free UltiMate 3000 RSLCnano HPLCs. Bound peptides were then eluted with an aqueous-organic gradient (eluent A: 0.5% (v/v) acetic acid; eluent B: 0.5% (v/v) acetic acid in 80% (v/v) acetonitrile; times referring to AP-MS/csBN-MS): 5 min 3% B, 60/120 min from 3% B to 30% B, 15 min from 30% B to 99% B or 20 min from 30% B to 50% B and 10 min from 50% B to 99% B, respectively, 5 min 99% B, 5 min from 99% B to 3% B, 15/10 min 3% B (flow rate 300 nL/min). Eluted peptides were separated in a SilicaTip™ manually packed 11 cm (AP-MS) or 23 cm (csBN-MS) with ReproSil-Pur 120 ODS-3 (C18; particle size 3 µm; Dr. Maisch HPLC, Germany) and electrosprayed (2.3 kV; transfer capillary temperature 250/300°C) in positive ion mode into an Orbitrap Elite (AP-MS) or a QExactive HF-X (csBN-MS) mass spectrometer. Instrument settings: maximum MS/MS injection time = 200/400 ms; dynamic exclusion duration = 30/60 s; minimum signal/intensity threshold = 2,000/40,000 (counts), top 10/15 precursors fragmented; isolation width = 1.0/1.4 m/z.

### Data Protocol
Peak lists were extracted from fragment ion spectra using ’msconvert.exe’ tool (part of ProteoWizard; http://proteowizard.sourceforge.net/; v3.0.6906 for Orbitrap Elite and v3.0.11098 for Q Exactive HF-X; Mascot generic format with filter options ’peakPicking true 1’- and ’threshold count 500 most-intense’). Precursor m/z values were preliminarily searched with 50 ppm peptide mass tolerance, their mass offset corrected by the median m/z offset of all peptides assigned, and afterwards searched with 5 ppm mass tolerance against all mouse, rat, and human (mouse/rat brain samples) or only human (HEK293 cell samples) entries of the UniProtKB/Swiss-Prot database. Acetyl (Protein N-term), Carbamidomethyl (C), Gln->pyro-Glu (N-term Q), Glu->pyro-Glu (N-term E), Oxidation (M), Phospho (S, T, Y), and Propionamide (C) were chosen as variable modifications, and fragment mass tolerance was set to ±0.8 Da (Orbitrap Elite data) or ± 20 mmu (Q Exactive HF-X data). One missed tryptic cleavage was allowed. The expect value cut-off for peptide assignment was set to 0.5. Related identified proteins (subset or species homologs) were grouped using the name of the predominant member. Proteins either representing exogenous contaminations (e.g., keratins, trypsin, IgG chains) or identified by only one specific peptide were not considered. Label-free quantification of proteins was carried out as described the manuscript. Peptide signal intensities (peak volumes, PVs) from FT full scans were determined, and offline mass calibrated using MaxQuant v1.6.3 (http://www.maxquant.org). Then, peptide PV elution times were pairwise aligned using LOESS regression (reference times dynamically calculated from the median peptide elution times overall aligned datasets). Finally, PVs were assigned to peptides based on their m/z and elution time (±1 min / 2-3 ppm, as obtained directly or indirectly from MS/MS-based identification) using in-house developed software. PV tables were then used to calculate protein abundance ratios in AP versus control, the abundance norm value as an estimate for molecular abundance, and csBN-MS abundance profiles. The latter were smoothed by sliding average (window of 5). Slice numbers were converted to apparent complex molecular weights by the sigmoidal fitting of (log(MW) versus slice number of the observed profile peak maximum of mitochondrial marker protein complexes.

### Publication Abstract
None

### Keywords
Lc-ms, Csbn-ms, Ap-ms, Calcium, Arl15, Trpm7, Cnnm, Magnesium, Zinc

### Affiliations
Institute of Physiology II  Universität Freiburg  Germany
University of Freiburg, Germany

### Submitter
Alexander Haupt

### Lab Head
Dr Prof. Dr. Bernd Fakler
Institute of Physiology II  Universität Freiburg  Germany


