### Accession
PXD009874

### Title
Toward the quantitative characterization of arginine phosphorylations in Staphylococcus aureus

### Description
The Gram positive bacterium Staphylococcus aureus plays an important role as an opportunistic pathogen and causative agent of nosocomial infections. As research gained insight into host specific adaptation and a broad range of virulence mechanisms, S. aureus evolved as a model organism for human pathogens. Hence, the investigation of staphylococcal proteome expression and regulation supports the understanding of the pathogenicity and relevant physiology of this organism. This study focused on the analysis of protein regulation by reversible protein phosphorylation, in particular on arginine residues. Therefore, both proteome and phosphoproteome of S. aureus COL wild type were compared with the arginine phosphatase deletion mutant S. aureus COL ΔptpB under control and stress conditions in a quantitative manner. A gel-free approach, adapted to the special challenges of arginine phosphorylation, was applied to analyze the phosphoproteome of exponential growing cells after oxidative stress caused by sublethal concentrations of H2O2. Together with phenotypic characterization of S. aureus COL ΔptpB, this study disclosed first insights into the physiological role of arginine phosphorylations in Gram positive pathogens. The spectral library based quantification of phosphopeptides finally allowed to link arginine phosphorylation to staphylococcal oxidative stress response, amino acid metabolism and virulence.

### Sample Protocol
For phenotypic analysis of bacterial growth, S. aureus COL wild type and ΔptpB were grown in CDM, the mutant cultures supplemented with 200 µg/ml spectinomycin. To establish reproducible growth during nutrient limitation in CDM, cultures were supplemented with either 10%, 5%, 2% or 1% LB; 1% BSA; 1%, 0.5% or 0.1% casein or decreasing concentrations of CAS from 10% to 0.0001% in decimal steps. For proteome quantification, cells were cultivated in RPMI-1640 and reliable growth was achieved by adding ingredients of the above used CDM in experimentally adapted amounts. For phenotypic characterization under stress conditions set by external stressors, cells were grown in three replicates in LB or BE medium. For cfu determination, aliquots of the culture were harvested 0, 60, 120 and 180 minutes after stress exposure to inoculate LB-agar plates in triplicates. For phosphoproteome analysis, oxidative stress was induced in four replicates of ΔptpB by adding 10 mmol/l hydrogen peroxide and cells were harvested after growing for 60 minutes. For proteome quantification, triplicates of S. aureus COL wild type and ΔptpB were cultivated in unlabeled or heavy RPMI 1640. Stress was induced using 10 mmol/l hydrogen peroxide and cells were harvested after 60 minutes. These heavy samples of wild type and mutant under control and stress conditions were mixed in a stoichiometric ratio of 1:1:1:1. This mixture defined the standard, which was subsequently added to the respective sample in a ratio of 1:1. For enrichment and quantification of phosphorylations, unlabeled samples of wild type and mutant were mixed in a ratio of 1:1 to form the standard, which was then mixed with the respective sample in a ratio of 1:1. For proteome quantification, washed cell pellets were disrupted by bead beating. After removal of cell debris and protein determination, samples were subjected to SDS-PAGE and proteins were in-gel digested with trypsin. Subsequently, samples were analyzed by mass spectrometry. Technical replicates were obtained by dividing the harvested cell culture into two aliquots. The preparation of phosphoproteome samples was done as followed: Harvested cells were washed, incubated in lysis buffer, disrupted by beat beating and treated for 10 min with 1 x Nuclease Mix and 1% (w/v) Octyl β-D-glucopyranoside. The protein extract underwent acetone precipitation before samples were reduced, alkylated and digested. SCX was performed using arginine phosphorylation compatible buffers, followed by titanium dioxide enrichment. To reduce the titanium contamination of the mass spectrometric systems, all samples were examined by light microscopy for remaining bead fragments thereby discarding contaminated samples.

### Data Protocol
All proteomic samples were measured online by LC-ESI-mass spectrometry. Briefly, an Easy-nLCII or Easy-nLC1000 was equipped with self-packed analytical columns. The LC was coupled to a LTQ Orbitrap Velos, Orbitrap Velos Pro or Orbitrap Elite. Using a flow rate of 300 nl/min, an 85 min gradient was applied, injecting 0.1% (v/v) acetic acid as buffer A and 99.9% (v/v) ACN with 0.1% (v/v) acetic acid as buffer B. CID experiments were recorded with a resolution of 60,000 (Full scan, Orbitrap analyzer) or 120,000 (Orbitrap Elite) with an MS1 scan range of m/z 300 to 2,000 and the top 20 precursor ions were selected in windows of 2 Da, omitting singly charged ions as well as unassigned charge states. Selected ions were fragmented with 35% NCE and analyzed in data-dependent MS/MS in the LTQ. Neutral loss option was enabled for fragment spectra, combined with multistage activation and wideband activation in case of the LTQ Orbitrap Velos. HCD experiments were set with a resolution of 30,000 (60,000 for Orbitrap Elite) for full scan mode and the 10 most intense precursor ions were fragmented using 35-40% NCE depending on the instrument. Hereby, 7,500 was used as MS/MS Scan resolution (15,000 for Orbitrap Elite, analyzed in Orbitrap). After a second fragmentation event, already fragmented precursor ions were excluded for 20 s (CID) or 60 s (HCD), respectively. For all measurements, lockmass option was enabled. Spectra of oxidative stress experiment and proteome quantification were searched with MaxQuant against an Uniprot database of S. aureus COL. Maximum mass deviation was 6 ppm at Full scan level and 0.5 Da for CID spectra or 20 ppm for HCD spectra at MS/MS level, considering charge states of +2 to +7. A false discovery rate was applied as well as a minimum number of two unique peptides per protein, seven amino acids per peptide and full tryptic specificity with a maximum of two missed cleavage sites. Oxidation on methionine, carbamidomethylation on cysteine and phosphorylation on serine, threonine, tyrosine or arginine were set as variable modifications during database search and results were filtered for phosphosite localization probability ≥ 0.75. Phosphorylated peptides were additionally subjected to manual validation and all phosphospectra passing the manual validation were added to the spectral library to form a new spectral library, which was used for data evaluation during this study. For the analysis of the phosphoproteome after oxidative stress as well as for quantification of arginine phosphorylations, the raw files were converted to mzXML and subjected to SpectraST library search provided by the Trans-Proteomic Pipeline. Search results were combined and filtered for Δppm < 0.005 Da and dot threshold (CID) or probability > 0.9 (HCD). For phenotypic analysis, reproducibility of bacterial growth was analyzed by NSAF comparison between biological and technical replicates. Proteome quantification was carried out using MaxQuant based integration of precursor peak areas and ratio calculation between sample and standard. The phosphoproteome was quantified combining spectral library search results with Census based quantification. Census based calculation was performed with strict filter parameters to generate precursor based peptide ratios between sample and standard. For all identification and quantification experiments, only unique peptides were considered. In addition, all data and on/off proteins were validated manually to consider only sufficiently matching isotopologue peak pairs. MaxQuant based results identified in at least two out of three biological replicates were subjected to visualization and statistical validation by Perseus and Multiple Experiment Viewer. ANOVA was used to compare all four conditions. Protein identifications passing ANOVA with p-value 0.01 were considered as significant. For proteins quantified in < 4 conditions and for Census based data, results had to pass t-test with p-value 0.01 to be considered as significant. The cut-off for biological relevance of significantly regulated proteins was set by analyzing the distribution of normalized ratios resulting in a cut-off value of at least twofold change for differentially expressed proteins.

### Publication Abstract
The Gram-positive bacterium Staphylococcus aureus plays an important role as an opportunistic pathogen and causative agent of nosocomial infections. As pathophysiological research gained insights into host-specific adaptation and a broad range of virulence mechanisms, S. aureus evolved as a model organism for human pathogens. Hence the investigation of staphylococcal proteome expression and regulation supports the understanding of the pathogenicity and relevant physiology of this organism. This study focused on the analysis of protein regulation by reversible protein phosphorylation, in particular, on arginine residues. Therefore, both proteome and phosphoproteome of S. aureus COL wild type were compared with the arginine phosphatase deletion mutant S. aureus COL &#x394;ptpB under control and stress conditions in a quantitative manner. A gel-free approach, adapted to the special challenges of arginine phosphorylations, was applied to analyze the phosphoproteome of exponential growing cells after oxidative stress caused by sublethal concentrations of H<sub>2</sub>O<sub>2</sub>. Together with phenotypic characterization of S. aureus COL &#x394;ptpB, this study disclosed first insights into the physiological role of arginine phosphorylations in Gram-positive pathogens. A spectral library based quantification of phosphopeptides finally allowed us to link arginine phosphorylation to staphylococcal oxidative stress response, amino acid metabolism, and virulence.

### Keywords
Oxidative stress, Silac quantification, Hydrogen peroxide, Spectral library, Quantification, Virulence, Arginine phosphorylation, Amino acid metabolism, Staphylococcus aureus, Phosphopeptide enrichment

### Affiliations
Institute for Microbiology, Department of Microbial Proteomics, University of Greifswald, Greifswald, Germany
Institute for Microbiology

### Submitter
Sabryna Junker

### Lab Head
Dr Dörte Becher
Institute for Microbiology, Department of Microbial Proteomics, University of Greifswald, Greifswald, Germany


