### Accession
PXD040715

### Title
Repertoire and abundance of secreted virulence factors shape the pathogenic capacity of Pseudomonas syringae pv. aptata

### Description
Pseudomonas syringae pv. aptata is a member of the sugar beet pathobiome and the causative agent of leaf spot disease. Like many pathogenic bacteria, P. syringae relies on the secretion of toxins, which manipulate host-pathogen interactions, to establish and maintain an infection. This study analyzes the secretome of six pathogenic P. syringae pv. aptata strains with different defined virulence capacities in order to identify common and strain-specific features, and correlate the secretome with disease outcome. All strains show a high type III secretion system (T3SS) and type VI secretion system (T6SS) activity under apoplast-like conditions mimicking the infection. Surprisingly, we found that low pathogenic strains show a higher secretion of most T3SS substrates (19 of the 23 detected effectors and accessory harpin proteins), whereas a distinct subgroup of four effectors is exclusively secreted in medium and high pathogenic strains. Similarly, we detected two T6SS secretion patterns: while one set of proteins was highly secreted in all strains, another subset consisting of known T6SS substrates and previously uncharacterized proteins with a highly similar secretion pattern was exclusively secreted in medium and high virulence strains. Taken together, our data shows that P. syringae pathogenicity is correlated with the repertoire and fine-tuning of effector secretion and indicates distinct strategies for establishing virulence of P. syringae pv. aptata in plants.

### Sample Protocol
5 ml overnight cultures (one for each biological replicate) were grown in KB at 28°C with shaking (180 rpm). 60 ml of day cultures were inoculated to OD 0.15 in Hrp-inducing medium and incubated with shaking (180rpm) at 20°C for 6 h. 50 ml of culture were collected by centrifugation at 4500 rcf for 10 min, and supernatants were transferred into a new falcon tube and filter sterilized twice. Collected proteins for the secretome analysis were precipitated by addition of 10% (v/v) trichloroacetic acid (TCA) and incubation at 4°C overnight. Precipitated proteins were collected by centrifugation for 30 min at 4000g and 4°C; the protein pellet was washed two times with 5 mL ice-cold acetone. Dried protein pellets were solubilized in 500 μl reconstitution buffer (1 M Urea in 100 mM ammonium bicarbonate (ABC) + 5 mM Tris(2-carboxyethyl) phosphine (TCEP)), and sonicated for 20 s (Hielscher Ultrasound Technology). Protein reduction by the added TCEP was carried out for 30 min at 37°C, followed by alkylation using iodoacetamide (10 mM) for 30 min at 25°C in the dark. For protein digestion, 1 μg trypsin (Sequencing grade Modified Trypsin, Promega) was added to the sample, which was then incubated at 30°C overnight. After the digest, samples were acidified using 1.5% (v/v) trifluoroacetic acid (TFA) and C18 purified using Chromabond Spin Columns (Macherey-Nagel). Acidified peptides were loaded on spin column that were equilibrated with 400 µl acetonitrile and then 400 µl 0.15 % TFA.  After peptide load, a washing step with 0.15 % TFA was performed, followed by elution using 400 µl 50% acetonitrile. Eluted peptides were then dried and reconstituted in 0.15 % TFA. LC-MS analysis was performed on a Q-Exactive Plus and a Exploris 480 mass spectrometer, both connected to an Ultimate 3000 rapid-separation liquid chromatography (RSLC) nano instrument and a nanospray flex ion source (all Thermo Scientific). Peptide separation on was carried out on a reverse-phase high-performance liquid chromatography (HPLC) column (75 μm x 42 cm) packed in-house with C18 resin (2.4 μm; Dr. Maisch GmbH). First, peptides were loaded on a C18 precolumn (preconcentration set-up), For total proteome analysis, peptide elutionElution was performed in backflush mode with a separating gradient from 94 % solvent A (0.15 % formic acid) and 6 % solvent B (99.85 % ACN, 0.15 % formic acid) to 35 % solvent B, followed by 4 min with 50 % at a flow rate of 300 nl/min.  The operating Q-Exactive Plus parameters were: 70,000 (at m/z 200) resolution, automatic gain control (AGC) setting set to 3e6 ions and a maximum of 50 ms injection time for MS1 survey scans, and 17,500 (at m/z 200) resolution for 50 ms injection time and 1e5 ions for MS/MS. MS/MS spectra were recorded from the 10 most intense ions. For peptide analysis on the Exploris 480 the parameters were set to: 60,000 (at m/z 200) resolution, the ion accumulation time was set to 25 ms for MS and AUTO for MS/MS. The automatic gain control (AGC) was set to 300% for MS survey scans and 200% for MS/MS scans.

### Data Protocol
The database of T3SS effectors was created by using effector protein sequences for P. syringae available in the public protein database (UniProt) (Supplementary Material Table). In particular, we used initial data of T3SS effectors from proteomes in all five P. syringae pv. aptata proteome data available at UniProt database and subsequently add effectors from other P. syringae pathovars to our T3SS effector database used in analysis. The final T3SS effector database generated for identification and comparison with LC-MS/MS analysis of P16 and P21 contained 63 effectors in total, among which 23 effectors were accumulated from proteomes of P. syringae pv. aptata and 40 effectors were accumulated from proteomes of  P. syringae pv. syringae, P. syringae pv. tomato, P. syringae pv. theae, P. syringae pv. phaseilicola, P. syringae pv. maculicola,  P. syringae pv. tabaci, P. syringae pv. spinaceae, and P. syringae pv. cilantro pathovars. Analysis of raw data for label-free quantification analysis was performed using MaxQuant (https://pubmed.ncbi.nlm.nih.gov/19029910/) in standard settings using the assembled protein database.  Carbamidomethylation (C) was set as fixed, oxidation (M) and deamidation (N, Q) as variable modifications.  For spectral based assessment searches were carried out using Mascot (v2.5, Matrix Science) with 10ppm MS1 and 0.02 Da fragment ion tolerance with Carbamidomethylation (C) as fixed, oxidation (M) and deamidation (N, Q) as variable modifications. Search results were evaluated in Scaffold  4 (Proteome Software).

### Publication Abstract
None

### Keywords
Pseudomonas syringae, Secretome

### Affiliations
Max Planck Institute for Terrestrial Microbiology Karl-von-Frisch Str. 10 35043 Marburg Germany
MPI Marburg

### Submitter
Timo Glatter

### Lab Head
Dr Timo Glatter
Max Planck Institute for Terrestrial Microbiology Karl-von-Frisch Str. 10 35043 Marburg Germany


