### Accession
PXD006383

### Title
Human colon aspirate proteins analyzed by LC-MSMS

### Description
Colonic aspirates were collected at diagnostic colonoscopy from inflammatory bowel disease (IBD) and control, treatment-naive children.  The colonic mucosal-luminal interface (MLI) proteomes were analyzed for 18 control and 42 IBD patients by liquid-chromatography mass spectrometry.

### Sample Protocol
Samples were transfered to the lab following collection. A protease inhibitor tablet was added, debris depleted by centrifugation at 700g for 5 minutes, followed by bacteria depletion by centrifugation at 14,000g for 20 minutes. The supernatant was filtered (0.2um) and stored at -80C. The filtrate was thawed on ice and precipitated overnight with trichloroacetic acid (20% v/v). Protein pellets were washed three times with acetone (100%) and dried before resuspension and sonication in lysis buffer (4% SDS, 50 mM Tris, pH 8.0, protease inhibitor cocktail). Protein concentration was quantified by DC protein assay.  An equal amount of colonic MLI protein was mixed with heavy reference proteins (Jurkat, HEK-293, HCT 116, THP-1, HuH-7) to permit Super-SILAC (stable-isotope labeling by amino acids in cell culture). The mixture was digested with trypsin by filter aided sample preparation method (FASP) fractionated into 5 fractions (pH 4, 6, 8, 10 and 12) using SCX resin, and desalted with a 10 Âµm AQUA-C18 resin. High-performance liquid chromatography/electrospray ionization tandem mass spectrometry (HPLC-ESI-MS/MS) was performed using an Ekspert nanoLC 400 coupled to an LTQ Velos Pro Orbitrap Elite MS.

### Data Protocol
Peptides were assigned and quantified using MaxQuant version 1.5.3.30  in a single run against the human Uniprot database (downloaded 2012/07/11). The following parameters were used: a multiplicity of two with Arg10 and Lys8 selected as the heavy labels; a specific digestion mode was implemented with trypsin selected as the enzyme with a maximum of two missed cleavages;  cysteine carbamidomethylation as a fixed modification; methionine oxidation and acetylation (protein N-termini) as variable modifications; the re-quantify and match between runs parameters were enabled; minimum peptide length of seven amino acids; ion mass tolerance of 0.5 Da; protein and peptide false discovery rate (FDR) of 1%.  The MaxQuant output (Ratio H/L normalized) was uploaded into Perseus v1.5.2.6 for data processing. MetaboAnalyst 3.0 GraphPad Prism 7 were utilized for biomarker panel generation.

### Publication Abstract
None

### Keywords
Human, Colon aspirates, Lc-msms

### Affiliations
Ottawa Institute of Systems Biology and Department of Biochemistry, Microbiology and Immunology, University of Ottawa, Ottawa, Canada
University of Ottawa

### Submitter
Shelley Deeke

### Lab Head
Dr Daniel Figeys
Ottawa Institute of Systems Biology and Department of Biochemistry, Microbiology and Immunology, University of Ottawa, Ottawa, Canada


