### Accession
PXD001646

### Title
E. coli source tracking -  A peptide identification-free, genome sequence-independent shotgun proteomics workflow for strain-level bacterial differentiation

### Description
We report a novel, peptide identification-free shotgun proteomics workflow as a bacterial fingerprinting method. This method uses a similarity-clustering algorithm to segregate the mass spectra that are presumably derived from individual peptide speciess and merge them into discrete units of consensus spectra that comprise the proteomic fingerprints of the bacterial isolates being investigated. The novel method was compared to a traditional peptide identification-based shotgun proteomics workflow and a commonly used PCR-DNA fingerprinting technique for performance benchmarking in differentiating 73 isolates of E. coli by their animal sources (human, cow, dog, and pig). The fingerprints generated using by the novel method were richer in information, more discriminative in separating the E. coli isolates by animal sources, and more accurate in classifying query isolates to the correct animal sources . Our data suggest that, by taking a snapshot of the system-wide expression of bacterial cells and circumventing peptide identification, the novel method generated fingerprints that not only reflected the adaptation of E. coli to different animal hosts more precisely than PCR-DNA fingerprinting but also constituted a fuller representation of the bacterial cells' proteomes than traditional shotgun proteomics.

### Sample Protocol
E. coli isolates were cultured and washed to obtain cell pellets for DNA and protein extraction. Each cell pellet was lysed and had protein digested with trypsin. Tryptic digest eluted in 0.1 % formic acid (FA) in water was injected to an LTQ Velos dual-pressure ion trap mass spectrometer (Thermo Fisher Scientific) that was interfaced to a nano-electrospray ion source (spray voltage 1.6kV, capillary temperature 250 Â°C), a Thermo Accela 600 pump, and an Accela autosampler LC. A BioBasic-18 column was used (dimension: 150.0 x 0.1 mm, particle size: 5  m; Thermo Scientific). Solvent A and B in the mobile phase (flow rate: 150  l min-1) were 0.1 % FA in water and 0.1 % FA in acetonitrile, respectively.

### Data Protocol
A protein database was constructed by combining all reviewed E. coli proteins  in the UniProt-SwissProt database, and the common contaminant proteins listed in the common Repository of Adventitious Proteins. Then, the protein database was appended to an equal-size decoy database generated by random shuffling of amino acids between tryptic sites (Elias and Gygi, 2007). Two traditional sequence database searching engines, OMSSA (ver 2.1.8) (Geer et al., 2004) and X!Tandem with K-score plugin (ver 2011.12.01.1) (Craig and Beavis, 2004), were used. Refinement search was enabled in X!Tandem in an error tolerant mode, which allows the identification of single nucleotide polymorphism.

### Publication Abstract
Shotgun proteomics is an emerging tool for bacterial identification and differentiation. However, the identification of the mass spectra of peptides to genome-derived peptide sequences remains a key issue that limits the use of shotgun proteomics to bacteria with genome sequences available. In this proof-of-concept study, we report a novel bacterial fingerprinting method that enjoys the resolving power and accuracy of mass spectrometry without the burden of peptide identification (i.e. genome sequence-independent). This method uses a similarity-clustering algorithm to search for mass spectra that are derived from the same peptide and merge them into a unique consensus spectrum as the basis to generate proteomic fingerprints of bacterial isolates. In comparison to a traditional peptide identification-based shotgun proteomics workflow and a PCR-based DNA fingerprinting method targeting the repetitive extragenic palindromes elements in bacterial genomes, the novel method generated fingerprints that were richer in information and more discriminative in differentiating E. coli isolates by their animal sources. The novel method is readily deployable to any cultivable bacteria, and may be used for several fields of study such as environmental microbiology, applied microbiology, and clinical microbiology.

### Keywords
73 e. coli strains from different animal hosts(cow; pig; dog and human); environmental samples; ltq

### Affiliations
ETH Zurich
Hong Kong University of Science and Technology

### Submitter
Wenguang Shao

### Lab Head
Dr Henry Lam
Hong Kong University of Science and Technology


