### Accession
PXD027454

### Title
A Novel Spectral Annotation Strategy Streamlines Reporting of mono-ADP-ribosylated Peptides Derived from Mouse Liver and Spleen in Response to IFN-gamma

### Description
Mass spectrometry-enabled ADP-ribosylation workflows are developing rapidly, providing researchers a variety of ADP-ribosylome enrichment strategies and mass spectrometric acquisition options. Despite the growth spurt in upstream technologies, systematic ADP-ribosyl (ADPr) peptide mass spectral annotation methods are lacking. HCD-dependent ADP-ribosylome studies are common but the resulting MS2 spectra are complex, owing to a mixture of b/y-ions and the m/p-ion peaks representing one or more dissociation events of the ADPr moiety (m-ion) and peptide (p-ion). In particular, p-ions can dominate HCD spectra but are not recognized by standard spectral annotation workflows. As a result, annotation strategies that are solely reliant upon the b/y-ions result in lower spectral scores that in turn reduce the number of reportable ADPr peptides. To improve the confidence of spectral assignments we implemented an ADPr peptide annotation and scoring strategy. All MS2 spectra are scored for the ADPr m-ions, but once spectra are assigned as an ADPr peptide they are further annotated and scored for the p-ions. We implemented this novel workflow to ADPr peptides enriched from the liver and spleen isolated from mice post 4-hour exposure to systemic IFN-gamma. HCD collision energy experiments were first performed on the Obitrap Fusion Lumos and the Q Exactive, with notable ADPr peptide dissociation properties verified with CID (Lumos). The m-ion and p-ion series score distributions revealed that ADPr peptide dissociation properties vary markedly between instruments and within instrument collision energy settings, with consequences on ADPr peptide reporting and amino acid localization.

### Sample Protocol
ADPr (MARylated) peptides from control, saline, and IFN-gamma elicited mouse tissues (spleen and liver) were analyzed using the Orbitrap Fusion Lumos fronted with an EASY-Spray Source, coupled to an Easy-nLC1000 HPLC pump (Thermo Fisher Scientific), and the Q Exactive Orbitrap (+ Easy-nLC1000) fronted with a Nanospray FLEX ion source (Thermo Fisher Scientific). Lumos collision energy experiments – A pool of mouse liver ADPr peptides from the pilot study were subjected to a dual column set-up: an Acclaim PepMap RSLC C18 trap column, 75 µm X 20 mm; and an EASY-Spray LC Column, 75 µm X 250 mm. The analytical gradient for the ADPr peptide pool was run at 300 nl/min from 5 to 21 % Solvent B (acetonitrile/0.1% formic acid) for 50 minutes, followed by ten minutes of 21 to 30% Solvent B, and another 10 minutes of a jigsaw wash (alternating between 5 and 95% Solvent B). Solvent A was water/0.1% formic acid. The instrument was set to 120 K resolution and the top N precursor ions (within a scan range of m/z 400-1500) in 3 seconds cycle time were subjected to MS/MS. Dynamic exclusion was enabled (30 seconds), the isolation width was m/z 1.2, and the resolution was 120 K (automatic gain control, AGC, 1.0e4). HCD collision energies were set to, 20%, 24%, 26%, 28%, 30%, 32% or 34%. The CID collision energy settings were 20%, 24%, 26%, 28%, 30%, 32%, 34%, 36% and 40%. Q Exactive collision energy experiments – A pool of mouse liver ADPr peptides from the pilot study were subjected to a dual column set-up: an Acclaim PepMap RSLC C18 trap column, 75 µm X 20 mm; and an Acclaim PepMap 100 C18 HPLC column, 75 µm X 250 mm. The analytical gradient for the ADPr peptide pool was run at 300 nl/min from 5 to 21 % Solvent B (acetonitrile/0.1% formic acid) for 50 minutes, followed by ten minutes of 21 to 30% Solvent B, and another 10 minutes of a jigsaw wash. The instrument was set to 70 K resolution (AGC target, 3e6), and the top 10 precursor ions (within a scan range of m/z 400-1500) were subjected to HCD isolation width m/z 1.6, dynamic exclusion enabled (60 seconds), and resolution set to 140 K (AGC target, 5e4). The HCD collision energies were set to, 20%, 24%, 26%, 28%, 30%, 32% or 34%.  Lumos-dependent analysis of IFN-gamma-elicited mouse liver and spleen ADPr peptides - The chromatographic conditions were the same as the Lumos collision energy experiments. The m-ion product scan employed data-dependent HCD acquisition (collision energy 30% +/- 2.5%, isolation width m/z 1.2, scan range m/z 120-440 to capture only the m-ions, and resolution set to 30 K). When two or more ADP-ribose fragment ions (m/z 136.0623, 250.0940, 348.0709, and 428.0372) were detected, alternating HCD (CE 27.5% +/- 2.5%; resolution 120 K) and EThcD (calibrated charge dependent ETD parameters enabled, supplemental activation collision energy 22.5%, and resolution 120 K) scans were triggered. Each ADPr peptide sample was injected five times (n=6 for no treatment, saline and IFN-gamma liver; n= 3 for no treatment, saline and IFN-gamma spleen). eAf1521 input peptides (input proteome) were analyzed using the data-dependent HCD acquisition (resolution 30 K for MS/MS) but without triggered data acquisitions. A summary of all acquisition strategies is provided in the supporting information. Scheduled targeted MS of the PARP14 ADPr peptide - A pool of mouse spleen ADPr peptides were analyzed with the same gradient as the Lumos collision energy experiments above. The PARP14 ADPr peptide (HISGLAQALSK + ADP-ribose, m/z 555.9058, z=3) was analyzed using HCD (CE20 or CE22% / CE24 or CE26%; resolution 120 K), HCD (CE 28% +/- 3%; resolution 120 K) and EThcD (calibrated charge dependent ETD parameters enabled, supplemental activation collision energy 22.5%, and resolution 120 K).

### Data Protocol
ADPr samples’ mass spectra were analyzed using a customized ADPr annotation and scoring module developed internally as an enhancement for Proteome Discoverer (PD version 2.4, Thermo Fisher Scientific)]. Spectral processing steps that were in common to most analyses were as follows: The spectra were queried against the Uniprot mouse (n=63,703 entries) or human (n=96,816 entries) fasta database (both downloaded September 09, 2020) using the SEQUEST-HT algorithm. Trypsin (full) was set as the digestion enzyme, allowing up to 4 missed cleavages and a minimum peptide length of 6 amino acids. ADPr (+541.061 Da) of Asp, Glu, Lys, Arg, Ser, Thr, Tyr and His; oxidation (+15.995 Da) of methionine; and acetylation (+42.011 Da) of the N-terminus, were set as variable modifications. Carbamidomethylation (+57.021 Da) of cysteine was set as a static modification. To note, a sampled analysis of ADPr data using carbamidomethylation and ADPr of Cys as variable modifications did not yield any ADPr-cysteine modified peptides; thus, the Cys acceptor site was not considered for the remainder of the study. Spectral search tolerances were 10 ppm for the precursor mass and 20 mmu (all HCD, EThcD and CID products were measured in the Orbitrap). The peptide false discovery rate (FDR) was calculated using Percolator (target/decoy method, separate databases) and spectra were filtered based on a 1.0% or 5.0% FDR, as indicated. The ‘p-series score’ was calculated in order to validate candidate spectra identified by SEQUEST. Subsequent XCorr thresholds for p-series scores-supported spectra were based on recommended medium confidence cut-offs in the ‘Fixed PSM Scorer’ Node in PD (z=2, 0.8; z=3, 1.0; z≥4, 1.2).  Peptide-spectrum match (PSM) ranks pertains to the SEQUEST search engine rank. The proteome samples’ (input peptides into ADPr workflow that were set aside) mass spectra were analyzed as above with the notable exceptions: ADPr modification was not considered, the spectra were filtered based on a 1.0% FDR cut-off, and proteins with 2 or more unique peptides were considered. Relative quantification was performed by the Feature Mapper and Precursor Ions Quantifier nodes. The maximum retention time shift for chromatographic alignments was set to 10 minutes and the mass tolerance was set to 10 ppm. Feature linking and mapping retention time tolerance was 0, and mass tolerance was 0 ppm with a signal-to-noise threshold of 5. EThcD scans of the m/z 400-1500 acquisitions (n=6 per treatment group in liver, for a total of 18 .raw files; n=3 per treatment group in spleen, 9 .raw files) were the input files the amino acid acceptor site profiling using the ‘IMP-ptmRS’ node in PD2.4. High confidence search engine rank 1 ADPr peptides (Protein Group = 1) were considered. The highest probability acceptor site is reported per peptide, with a minimum probability of 95%, is reported. ADPr samples’ MS/MS spectra yielding ‘0 PSMs’ (unidentified spectra) using the standard search parameters above were exported from Proteome Discoverer as .mzML files. For the HCD collision energy experiments we considered the possibility that some of these spectra may be atypical tryptic peptides thus we used semi-trypsin as a search parameter and decreased the number of amino acids to four. When exporting unidentified HCD spectra, only those with m-series scores ≥30 were included.

### Publication Abstract
Mass-spectrometry-enabled ADP-ribosylation workflows are developing rapidly, providing researchers a variety of ADP-ribosylome enrichment strategies and mass spectrometric acquisition options. Despite the growth spurt in upstream technologies, systematic ADP-ribosyl (ADPr) peptide mass spectral annotation methods are lacking. HCD-dependent ADP-ribosylome studies are common, but the resulting MS2 spectra are complex, owing to a mixture of b/y-ions and the m/p-ion peaks representing one or more dissociation events of the ADPr moiety (m-ion) and peptide (p-ion). In particular, p-ions that dissociate further into one or more fragment ions can dominate HCD spectra but are not recognized by standard spectral annotation workflows. As a result, annotation strategies that are solely reliant upon the b/y-ions result in lower spectral scores that in turn reduce the number of reportable ADPr peptides. To improve the confidence of spectral assignments, we implemented an ADPr peptide annotation and scoring strategy. All MS2 spectra are scored for the ADPr m-ions, but once spectra are assigned as an ADPr peptide, they are further annotated and scored for the p-ions. We implemented this novel workflow to ADPr peptides enriched from the liver and spleen isolated from mice post 4&#xa0;h exposure to systemic IFN-&#x3b3;. HCD collision energy experiments were first performed on the Orbitrap Fusion Lumos and the Q Exactive, with notable ADPr peptide dissociation properties verified with CID (Lumos). The m-ion and p-ion series score distributions revealed that ADPr peptide dissociation properties vary markedly between instruments and within instrument collision energy settings, with consequences on ADPr peptide reporting and amino acid localization. Consequentially, we increased the number of reportable ADPr peptides by 25% (liver) and 17% (spleen) by validation and the inclusion of lower confidence ADPr peptide spectra. This systematic annotation strategy will streamline future reporting of ADPr peptides that have been sequenced using any HCD/CID-based method.

### Keywords
Spleen, Adp-ribosylation, Liver, Ifn-gamma, Lc-ms/ms, Software

### Affiliations
Brigham and Women's Hospital, Harvard Medical School
Center for Interdisciplinary Cardiovascular Sciences, Brigham and Women’s Hospital, US

### Submitter
Sasha  Singh

### Lab Head
Dr Masanori Aikawa
Center for Interdisciplinary Cardiovascular Sciences, Brigham and Women’s Hospital, US


