### Accession
PXD036809

### Title
The histone chaperone NASP maintains H3-H4 reservoirs in the early Drosophila embryo

### Description
Histones are essential for chromatin packaging and histone supply must be tightly regulated as excess histones are toxic. To drive the rapid cell cycles of the early embryo, however, excess histones are maternally deposited. Therefore, soluble histones must be buffered by histone chaperones but the chaperone necessary to stabilize soluble H3-H4 pools in the Drosophila embryo has yet to be identified. Here, we show that CG8223, the Drosophila ortholog of NASP, is a H3-H4-specific chaperone in the early embryo. NASP specifically binds to H3-H4 in the early embryo. We demonstrate that, while NASP is non-essential in Drosophila, NASP is maternal effect lethal gene. Embryos laid by NASP mutant mothers have a reduce rate of hatching and show defects in early embryogenesis. Critically, soluble H3-H4 pools are degraded in embryos laid by NASP mutant mothers. Our work identifies NASP as the critical H3-H4 histone chaperone in the Drosophila embryo.

### Sample Protocol
For NASP-immunoprecipitation (IP), samples were prepared as described previously (Immunoprecipitation and Western blotting). For total protein levels in stage 14 egg chambers and embryos, 20 embryos or stage 14 egg chambers were collected for each replicate, flash frozen, and stored at –80°C until use. Once all samples for four biological replicates were collected, samples were thawed on ice and a 100 µL of NP40 lysis buffer was added. Samples were then homogenized ten times with a B-type pestle, transferred into a 1.5mL Eppendorf tube and centrifuged for 30s at 10,000RCF at 4°C. 100 µL of supernatant was transferred to a new 1.5mL Eppendorf tube for protein precipitation.  Both lysate and IP samples were precipitated using mass spectrometry grade methanol:chloroform:water (3:1:3) and washed three times with methanol. Each wash was followed by a two minute spin at 10,000xg at room temperature. Protein pellets were air dried and resuspended in 5µL of 1% Rapigest SF (Waters). Resuspended proteins were diluted with 32.5 µL mass spectrometry grade water and 10 µL 0.5 M HEPES (pH 8.0), then reduced with 0.5 µL of 0.5 M TCEP (freshly made) for 30 minutes at room temperature. Free sulfhydryl groups were acetylated with 1 µL of fresh 0.5 M Iodoacetamide for 30 minutes at room temperature in the dark and digested with 0.5 µg trypsin/Lys-C (Thermo Fisher) overnight at 37°C shaking. Digested peptides were diluted to 60 µL with water and labeled for 1 hour at room temperature using 16plex TMTpro (Thermo Scientific) or 10plex TMT (Thermo Scientific) for lysate and IP samples, respectively. Labeling was quenched with the addition of fresh ammonium bicarbonate (0.4% v/v final) for one hour at room temperature. Samples were pooled, acidified to pH < 2.0 using formic acid, concentrated to 1/6th original volume via Speed-vac, and diluted back to the original volume with buffer A (95% water, 5% acetonitrile, 0.1% formic acid). Cleaved Rapigest products were removed by centrifugation at 17,000xg for 30 minutes and supernatant transferred to fresh tubes for storage at -80°C until mass spectrometry analysis.  MudPIT liquid chromatography-tandem mass spectrometry Triphasic MudPIT columns were prepared as previously described using alternating layers of 1.5cm C18 resin, 1.5cm SCX resin, and 1.5cm C18 resin(Fonslow et al. 2012). Pooled TMT samples (roughly one-third of pooled IP samples and roughly 20 µg of peptide from lysate samples) were loaded onto the microcapillaries using a high-pressure chamber, followed by a 30 minute wash in buffer A (95% water, 5% acetonitrile, 0.1% formic acid). Peptides were fractionated online by liquid chromatography using an Ultimate 3000 nanoLC system and subsequently analyzed using an Exploris480 mass spectrometer (Thermo Fisher). The MudPIT columns were installed on the LC column switching valve and followed by a 20cm fused silica microcapillary column filled with Aqua C18, 3µm, C18 resin (Phenomenex) ending in a laser-pulled tip. Prior to use, columns were washed in the same way as the MudPIT capillaries. MudPIT runs were carried out by 10µL sequential injections of 0, 10, 20, 40, 60, 80, 100 % buffer C (500mM ammonium acetate, 94.9% water, 5% acetonitrile, 0.1% formic acid) for IP samples and 0, 10, 20, 30, 40, 50, 60, 70, 80, 90, 100% buffer C for global lysate samples, followed by a final injection of 90% C, 10% buffer B (99.9% acetonitrile, 0.1% formic acid v/v). Each injection was followed by a 130 min gradient using a flow rate of 500nL/min (0-6 min: 2% buffer B, 8 min: 5% B, 100 min: 35% B, 105min: 65% B, 106-113 min: 85% B, 113-130 min: 2% B). ESI was performed directly from the tip of the microcapillary column using a spray voltage of 2.2 kV, an ion transfer tube temperature of 275°C and a RF Lens of 40%. MS1 spectra were collected using a scan range of 400-1600 m/z, 120k resolution, AGC target of 300%, and automatic injection times. Data-dependent MS2 spectra were obtained using a monoisotopic peak selection mode: peptide, including charge state 2-7, TopSpeed method (3s cycle time), isolation window 0.4 m/z, HCD fragmentation using a normalized collision energy of 36% (TMTpro) or 32% (TMT 10plex), resolution 45k, AGC target of 200%, automatic (lysate) or 150 ms (IP) maximum injection times, and a dynamic exclusion (20 ppm window) set to 60s.

### Data Protocol
Identification and quantification of peptides were performed in Proteome Discoverer 2.4 (Thermo Fisher) using a UniProt Drosophila melanogaster proteome database (downloaded February 6th, 2019) containing 21,114 protein entries. The database was adjusted to remove splice-isoforms and redundant proteins and supplemented with common MS contaminants. Searches were conducted with Sequest HT using the following parameters: trypsin cleavage (maximum 2 missed cleavages), minimum peptide length 6 AAs, precursor mass tolerance 20ppm, fragment mass tolerance 0.02 Da, dynamic modifications of Met oxidation (+15.995 Da), protein N-terminal Met loss (-131.040 Da), and protein N-terminal acetylation (+42.011 Da), static modifications of TMTpro (+304.207 Da) or TMT 10plex (+229.163 Da) at Lys and N-termini and Cys carbamidomethylation (+57.021 Da). Peptide IDs were filtered using Percolator with an FDR target of 0.01. Proteins were filtered based on a 0.01 FDR, and protein groups were created according to a strict parsimony principle. TMT reporter ions were quantified considering unique and razor peptides, excluding peptides with co-isolation interference greater that 25%. Peptide abundances were normalized based on total peptide amounts in each channel, assuming similar levels of background in the IPs. Protein quantification used all quantified peptides. Post-search filtering was done to include only proteins with two identified peptides. Unique peptides for each canonical and variant histone was manually identified, summed, and statistically analyzed on Graphpad Prism. For IP samples, multiple t-test was performed with no correction (p<0.05). For lysate samples, multiple t-test with Holm-Sidak correction was performed (p<0.05).

### Publication Abstract
None

### Keywords
H3-h4, Histone chaperone, Nasp, Ip-ms, Histone

### Affiliations
Department of Biological Sciences, Vanderbilt University
Vanderbilt University

### Submitter
Jonathan Davies

### Lab Head
Dr Jared T. Nordman
Department of Biological Sciences, Vanderbilt University


