### Accession
PXD000304

### Title
DNMT DKO proteomic analyses -  A proteogenomics approach integrating proteomics and ribosome profiling increases the efficiency of protein identification and enables the discovery of alternative translation start sites

### Description
This study aimed to investigate the link between DNA methylation and protein expression on a genome wide level, using a combination of MBD-sequencing and shotgun and positional proteomics on two human colon cancer cell lines: a wild type HCT116 cell line and a double knock out HCT116 cell line (DNMT1 -/- and DNMT3b -/-).  The protein content of the HCT116 WT and DKO cell lines was not only analyzed using a standard shotgun proteomics approach, but also with the N-terminal COFRADIC technique (Staes et al., 2011), in order to determine the (alternative) translation start site of the expressed proteins.

### Sample Protocol
Not available

### Data Protocol
Not available

### Publication Abstract
Next-generation transcriptome sequencing is increasingly integrated with MS to enhance MS-based protein and peptide identification. Recently, a breakthrough in transcriptome analysis was achieved with the development of ribosome profiling (ribo-seq). This technology is based on the deep sequencing of ribosome-protected mRNA fragments, thereby enabling the direct observation of in vivo protein synthesis at the transcript level. In order to explore the impact of a ribo-seq-derived protein sequence search space on MS/MS spectrum identification, we performed a comprehensive proteome study on a human cancer cell line, using both shotgun and N-terminal proteomics, next to ribosome profiling, which was used to delineate (alternative) translational reading frames. By including protein-level evidence of sample-specific genetic variation and alternative translation, this strategy improved the identification score of 69 proteins and identified 22 new proteins in the shotgun experiment. Furthermore, we discovered 18 new alternative translation start sites in the N-terminal proteomics data and observed a correlation between the quantitative measures of ribo-seq and shotgun proteomics with a Pearson correlation coefficient ranging from 0.483 to 0.664. Overall, this study demonstrated the benefits of ribosome profiling for MS-based protein and peptide identification and we believe this approach could develop into a common practice for next-generation proteomics.

### Keywords
Dnmt1, Hct116, Dnmt3b, Shotgun proteomics, N-terminal cofradic

### Affiliations
VIB - Ghent University

### Submitter
Jolien Hollebeke

### Lab Head


