### Accession
PXD025490

### Title
Isonicotinylation is Isonicotinylation is a histone mark induced by the anti-tuberculosis first-line drug isoniazida histone mark induced by the anti-tuberculosis first-line drug isoniazid

### Description
Isoniazid (INH) is the first-line anti-tuberculosis drug used for nearly seventy years. Metabolites of INH showed hepatotoxicity in human and tumorigenicity in rodents. However, mechanism underlying the side effects of INH is elusive. Histone acylation is known to be modulated by intracellular metabolites. Here, we report INH and its metabolites induces a novel post-translational modification (PTM) on histones, the lysine isonicotinylation (Kinic), also called 4-picolinylation, in cells and mice. INH functions as a donor to promote biosynthesis of isonicotinyl-CoA that is used for a co-factor of intracellular isonicotinylation reaction. Twenty-six isonicotinylation sites were identified on histones in HepG2 cells by mass spectrometry. Acetyltransferases CREB-binding protein (CBP) and P300 were found to catalyze histone Kinic, whilst histone deacetylase HDAC3 functions as a deisonicotinylase. MNase sensitivity assay and RNA-seq analysis showed that histone Kinic relaxes chromatin structure and promotes gene transcription. Importantly, INH-mediated histone Kinic upregulates PIK3R1 gene expression and activates PI3K/Akt/mTOR signaling pathway in liver cancer cells, linking INH to the tumorigenicity in liver. Further, Kinic was found increased in liver cancer patients with concomitant raised PIK3R1 protein level. In addition, histone Kinic also affects TNF and GABAergic synapse signaling pathways that are closely related to INH-caused side effects. Taken together, we demonstrated that lysine isonicotinylation represents the first histone acylation mark with the pyridine ring that may arise broad biological effects. Therefore, INH-induced isonicotinylation is likely a mechanism accounting for side effects in patients who taking long-term INH for anti-tuberculosis therapy and this modification may also increase cancer risk in human.

### Sample Protocol
HepG2 cells were treated with 10 mM of INH for 24 h, then the histones were purified and subjected to trypsin digestion. Peptides were enriched by the pan anti-Kinic antibody and analyzed by HPLC-MS/MS analysis, followed by a protein sequence database search.

### Data Protocol
The tryptic peptides were dissolved in 0.1% formic acid (solvent A), directly loaded onto a home-made reversed-phase analytical column (15-cm length, 75 Î¼m i.d.). The gradient was comprised of an increase from 6 % to 23 % solvent B (0.1 % formic acid in 98 % acetonitrile) over 26 min, 23 % to 35 % in 8 min and climbing to 80 % in 3 min then holding at 80 % for the last 3 min, all at a constant flow rate of 400 nL/min on an EASY-nLC 1000 UPLC system. The peptides were subjected to NSI source followed by tandem mass spectrometry (MS/MS) in Q ExactiveTM Plus (Thermo) coupled online to the UPLC. The electrospray voltage applied was 2.0 kV. The m/z scan range was 350 to 1800 for full scan, and intact peptides were detected in the Orbitrap at a resolution of 70,000. Peptides were then selected for MS/MS using NCE setting as 28 and the fragments were detected in the Orbitrap at a resolution of 17,500. A data-dependent procedure that alternated between one MS scan followed by 20 MS/MS scans with 15.0s dynamic exclusion. Automatic gain control (AGC) was set at 5E4. Fixed first mass was set as 100 m/z. The resulting MS/MS data were processed using Maxquant search engine (v.1.5.2.8). Tandem mass spectra were searched against SwissProt Human database concatenated with reverse decoy database. Trypsin/P was specified as cleavage enzyme allowing up to 4 missing cleavages. The mass tolerance for precursor ions was set as 20 ppm in First search and 5 ppm in Main search, and the mass tolerance for fragment ions was set as 0.02 Da.

### Publication Abstract
None

### Keywords
Histone deisonicotinylases, Histone isonicotinylation, Chromatin structure, Histone isonicotinyltransferases, Gene transcription

### Affiliations
Department of Microbiology & Infectious Disease Center, School of Basic Medical Sciences, Peking University Health Science Center, Beijing 100191, P.R. China
Program for Cancer and Cell Biology, Department of Human Anatomy, Histology and Embryology, and PKU International Cancer Institute, Peking University Health Science Center, #38 Xue Yuan Road, Beijing 100191, China.

### Submitter
Yuhan Jiang

### Lab Head
Dr Hongquan Zhang
Program for Cancer and Cell Biology, Department of Human Anatomy, Histology and Embryology, and PKU International Cancer Institute, Peking University Health Science Center, #38 Xue Yuan Road, Beijing 100191, China.


