### Accession
PXD039878

### Title
KINETIC AND COMPOSITIONAL ANALYSIS OF RNA POLYMERASE II IN TRYPANOSOMA BRUCEI

### Description
Genome-wide analysis of nascent RNA synthesis gives kinetic information on the transcriptional status of RNA polymerase II (Pol II).  In parallel, immunoprecipitation of the largest subunit of Pol II (RPB1) provides the protein composition of the enzyme complex in the presence or absence of the transcriptional inhibitor Actinomycin D.

### Sample Protocol
Procyclic forms were harvested at a density of 5 x 106/ml and incubated with 0.1% paraformaldehyde (w/v) in PBS for 8 min at room temperature to cross-link protein complexes. The reaction was stopped by adding glycine to a final concentration of 125mM and incubating for 5 min. Cells were washed in PBS and resuspended in lysis buffer (20 mM Tris pH7.5, 140 mM KCl, 1.8 mM MgCl2, 0.1% NP40, 10% glycerol) containing protease inhibitor cocktail (EDTA-free, Roche). Cells were lysed by sonication using a Branson Digital Sonifier three times 10s, 30s interval on ice at 10% power. RBP1 immunoprecipitations were performed as before (Das & Bellofatto, 2009; doi: 10.1371/journal.pone.0006959) but using pre-bound anti-RBP1 beads, where antibody was attached to Protein A magnetic beads (Thermo Fisher) by a 2-hour incubation with antibody diluted in lysis buffer. Eluates were loaded on an SDS-polyacrylamide gel and run until they had just entered the 12% resolving gel. Proteins were then in-gel digested with trypsin as described elsewhere (Gunasekera K, Wuethrich D, Braga-Lagache S, Heller M, Ochsenreiter T, 2012; doi:10.1186/1471-2164-13-556). The digests were analyzed by liquid chromatography (LC)-MS/MS (PROXEON coupled to a QExactive HF mass spectrometer, ThermoFisher Scientific) with one injection of 5 ?l digests. Peptides were trapped on a µPrecolumn C18 PepMap100 (5?m, 100 , 300 ?m?5mm, ThermoFisher Scientific, Reinach, Switzerland) and separated by backflush on a C18 column (5 ?m, 100 , 75 ?m?15 cm, C18) by applying a 40-minute gradient of 5% acetonitrile to 40% in water, 0.1% formic acid, at a flow rate of 350 nl/min. The Full Scan method was set with resolution at 60,000 with an automatic gain control (AGC) target of 1E06 and maximum ion injection time of 50 ms. The data-dependent method for precursor ion fragmentation was applied with the following settings: resolution 15,000, AGC of 1E05, maximum ion time of 110 milliseconds, mass window 1.6Êm/z, collision energy 27, under fill ratio 1%, charge exclusion of unassigned and 1+ ions, and peptide match preferred, respectively. Alternatively, protein pull-downs were processed directly on the beads in 8M Urea / 50mMÊTris-HCl pH8, with reduction for 30min at 37¡C with DTT 0.1M / 100mM Tris-HCl pH8, alkylation for 30min at 37¡C in the dark with IAA 0.5M / 100mM Tris-HCl pH8, followed by dilution with 4 volumes of 20mM Tris-HCl pH8 / 2mM CaCl2 prior to overnight digestion at room temperature with 100?ng sequencing grade trypsin (Promega). Digests were analyzed on the same instrumental setup as above applying a 60min acetonitrile gradient.

### Data Protocol
Spectra interpretation was performed with Easyprot on a local server run under Ubuntu against a forward + reverse T. brucei sequence database extended with common contaminants like keratins and trypsin. Static modification of carboamidomethylated on Cys, and variable modifications of oxidation on Met, acetylation on protein N-Term, and deamidation of Gln/Asn were set. Parent and fragment mass tolerances were set to 10ppm. Matches on the reversed sequence database were used to set a Z-score threshold, where 1% false discoveries on the peptide spectrum match level had to be expected. Protein identifications were only accepted, when two unique peptides fulfilling the 1% FDR criterium were identified.

### Publication Abstract
TbMex67 is the major mRNA export factor known to date in trypanosomes, forming part of the docking platform within the nuclear pore. To explore its role in co-transcriptional mRNA export, recently reported in Trypanosoma brucei, pulse labelling of nascent RNAs with 5-ethynyl uridine (5-EU) was performed with cells depleted of TbMex67 and complemented with a dominant-negative mutant (TbMex67-DN). RNA polymerase (Pol) II transcription was unaffected, but the procyclin loci, which encode mRNAs transcribed by Pol I from internal sites on chromosomes 6 and 10, showed increased levels of 5-EU incorporation. This was due to Pol I readthrough transcription, which proceeded beyond the procyclin and procyclin-associated genes up to the Pol II transcription start site on the opposite strand. Complementation by TbMex67-DN also increased Pol I-dependent formation of R-loops and &#x3b3;-histone 2A foci. The DN mutant exhibited reduced nuclear localisation and binding to chromatin compared to wild-type TbMex67. Together with its interaction with chromatin remodelling factor TbRRM1 and Pol II, and transcription-dependent association of Pol II with nucleoporins, our findings support a role for TbMex67 in connecting transcription and export in T. brucei. In addition, TbMex67 stalls readthrough by Pol I in specific contexts, thereby limiting R-loop formation and replication stress.

### Keywords
Rna polymerase ii; transcription; rna export; nuclear pore

### Affiliations
Institute of Cell Biology University of Berne Baltzerstrasse 4 CH-3012 Berne, Switzerland
Proteomics and Mass Spectrometry Core Facility, Department for BioMedical Research (DBMR), University of Bern, Bern, Switzerland

### Submitter
Manfred Heller

### Lab Head
Dr Isabel Roditi
Institute of Cell Biology University of Berne Baltzerstrasse 4 CH-3012 Berne, Switzerland


