### Accession
PXD039766

### Title
The secretome features associated with the development of re-sistance to vemurafenib in BRAFV600E mutated colon cancer cells

### Description
Patients with metastatic colorectal cancer (mCRC) carrying BRAFV600E mutation have worse response to chemotherapy and poor prognosis. The aim of this study was to conduct for the first time comparative proteomics profiling of the secretome from vemurafenib-sensitive vs. resistant colon cancer cells harboring BRAFV600E mutation in order to identify specific secretory features potentially associated with the development of resistance to vemurafenib. Towards this aim, we employed a label-free quantitative LC-MS/MS analysis. Obtained results pointed to aberrant regulation of DNA replication and the endoplasmic reticulum stress as the major secretome features associated with chemoresistant phenotype. Further studies are required to investigate diagnostic values of these proteins in the clinical setting and to examine their roles in acquired resistance to BRAF inhibition not only in colon cancer but also in other BRAF mutated cancers.

### Sample Protocol
For each sample the volume corresponding to 20 µg of protein was taken and processed using the single‐pot solid‐phase enhanced sample preparation (SP3) In short, protein purification, digest and peptide clean-up were performed using a KingFisher Flex System and Carboxylate-Modified Magnetic Particles. Samples were diluted with 100% ethanol to a final concentration of 50% ethanol. Protein digestion was carried out overnight at 37°C with a trypsin:protein ratio of 1:50 in 50 mM Triethylammoniumbicarbonat and peptide elution from the magnetic beads using MilliQ water. The digest solution and water elution were combined and dried to completeness and re-solubilized in MS sample buffer. Mass spectrometry analysis was performed on a Q Exactive HF mass spectrometer equipped with a Digital PicoView source and coupled to a M-Class UPLC. Solvent composition at the two channels was 0.1% formic acid for channel A and 0.1% formic acid, 99.9% acetonitrile for channel B. Column temperature was 50°C. For each sample 2 μL of peptides were loaded on a commercial ACQUITY UPLC M-Class Symmetry C18 Trap Column (100Å, 5 µm, 180 µm x 20 mm) followed by ACQUITY UPLC M-Class HSS T3 Column (100Å, 1.8 µm, 75 µm X 250 mm). The peptides were eluted at a flow rate of 300 nL/min. After a 3 min initial hold at 5% B, a gradient from 5 to 24 % B in 80 min and 24 to 36 % B in an additional 10 min was applied. The column was cleaned after the run by increasing to 95 % B and holding 95 % B for 10 min prior to re-establishing loading condition. The mass spectrometer was operated in data-dependent mode on the top12 most abundant ions using Xcalibur. Full-scan MS spectra (350−1’500 m/z) were acquired at a resolution of 120’000 at 200 m/z after accumulation to an automated gain control (AGC) target value of 100’000 or for a maximum injection time of 50 ms. Precursors with an intensity above 4’500 were selected for MS/MS. Ions were isolated using a quadrupole mass filter with 1.2 m/z isolation window and fragmented by higher-energy collisional dissociation using a normalized collision energy of 28 %. MS2 spectra were recorded at a resolution of 35’000 and a maximum injection time of 54 ms. Charge state screening was enabled, and singly, unassigned charge states and charge states higher than seven were excluded. Precursor masses previously selected for MS/MS measurement were excluded from further selection for 30 s, applying a mass tolerance of 10 ppm.

### Data Protocol
The acquired raw MS data were processed by MaxQuant (version 1.6.2.3), followed by protein identification using the integrated Andromeda search engine. Spectra were searched against the Uniprot Homo sapiens reference proteome (taxonomy 9606, canonical version from 2019-07-09), concatenated to its reversed decoyed fasta database and common protein contaminants. Carbamidomethylation of cysteine was set as fixed modification, while methionine oxidation and N-terminal protein acetylation were set as variable. Enzyme specificity was set to trypsin/P allowing a minimal peptide length of 7 amino acids and a maximum of two missed cleavages. MaxQuant Orbitrap default search settings were used. The maximum false discovery rate (FDR) was set to 0.01 for peptides and 0.05 for proteins. Label free quantification was enabled and a 2 minutes window for match between runs was applied. In the MaxQuant experimental design template, each file is kept separate in the experimental design to obtain individual quantitative values. Protein fold changes were computed based on Intensity values reported in the proteinGroups.txt file.

### Publication Abstract
Patients with metastatic colorectal cancer (mCRC) carrying BRAFV600E mutation have worse response to chemotherapy and poor prognosis. The BRAFV600E inhibitor vemurafenib has shown modest efficacy as monotherapy in BRAF-mutated mCRC due to the development of resistance. The aim of this study was to conduct a comparative proteomics profiling of the secretome from vemurafenib-sensitive vs. -resistant colon cancer cells harboring BRAFV600E mutation in order to identify specific secretory features potentially associated with changes in the resistant cells' phenotype. Towards this aim, we employed two complementary proteomics approaches including two-dimensional gel electrophoresis coupled with MALDI-TOF/TOF mass spectrometry and label-free quantitative LC-MS/MS analysis. Obtained results pointed to aberrant regulation of DNA replication and endoplasmic reticulum stress as the major secretome features associated with chemoresistant phenotype. Accordingly, two proteins implicated in these processes including RPA1 and HSPA5/GRP78 were discussed in more details in the context of biological networks and their importance as potential secretome targets for further functional and clinical evaluation. Expression patterns of RPA1 and HSPA5/GRP78 in tumor tissues from colon cancer patients were also found in additional in silico analyses to be associated with BRAFV600E mutation status, which opens the possibility to extrapolate our findings and their clinical implication to other solid tumors harboring BRAFV600E mutation, such as melanoma.

### Keywords
Vemurafenib, Colon cancer, Er stress, Chemoresistance, Rpa1, Dna replication, Lc-ms/ms, Secretome, Brafv600e, Braf inhibition, Hspa5/ grp78

### Affiliations
ETH
Centre for Applied Bioanthropology, Institute for Anthropological Research, Zagreb, Croatia

### Submitter
Antje Dittmann

### Lab Head
Dr Mirela Sedic
Centre for Applied Bioanthropology, Institute for Anthropological Research, Zagreb, Croatia


