### Accession
PXD039739

### Title
Functional characterization of the RNA interference pathway in A. fumigatus reveals its potential for antifungal therapy

### Description
The RNA interference (RNAi) pathway has evolved numerous functionalities in eukaryotes, with many on display in Kingdom Fungi. RNAi can regulate gene expression, facilitate drug resistance, or even be altogether lost to improve growth potential in some fungal pathogens. In the WHO fungal priority pathogen, Aspergillus fumigatus, the RNAi system is known to be intact and functional. To extend our limited understanding of A. fumigatus RNAi, we first investigated the genetic variation in RNAi-associated genes in a large collection of environmental and clinical genomes, where we found that RNAi is evolutionarily conserved even in clinical strains. Using endogenously expressed inverted-repeat transgenes complementary to both a conditionally essential gene and a nonessential gene, we determined that a subset of the RNAi componentry is active in inverted-repeat transgene silencing in conidia and mycelium. A multi-condition mRNA-seq analysis linked the A. fumigatus dicer-like enzymes and RNA-dependent RNA polymerases to regulation of conidial ribosome biogenesis genes; however, surprisingly few endogenous small RNAs were identified in conidia that could explain this broad change. Although RNAi was not clearly linked to growth or stress response defects in the RNAi knockouts, serial passaging of RNAi knockout strains for six generations resulted in lineages with diminished spore production over time, indicating that loss of RNAi can exert a fitness cost on the fungus. Cumulatively, A. fumigatus RNAi appears to play an active role in defense against double-stranded RNA species alongside a previously unappreciated housekeeping function in regulation of conidial ribosomal biogenesis genes.

### Sample Protocol
Cells were lysed liquid nitrogen with mortar and pestle and proteins were solubilized in 1% SDS, 150 mM NaCl, and 100 mM triethylammonium bicarbonate (TEAB) with the addition of each 1 tablet cOmplete ultra protease inhibitor cocktail and PhosStop (both Roche) per 10 ml lysis buffer. Subsequently, cysteine thiols were reduced and carbamidomethylated in one step for 30 min at 70°C by addition of each 2 µL of 500 mM TCEP (tris(2-carboxyethyl)phosphine) and 625 mM 2-chloroacetamide (CAA) per 100 µg of total protein in 100 µl. Proteins were precipitated by MeOH/Chloroform/H2O  (Wessel and Flügge, 1984, https://doi.org/10.1016/0003-2697(84)90782-6) and resolubilized in 100 mM TEAB in 5:95 trifluoroethanol/H2O (v/v). Proteins were digested for 18 h at 37°C after addition of Trypsin/Lys-C mix at a protease-to-protein ratio of 1:25. Tryptic peptides were dried in vacuum concentrator and resolubilized in 30 µL of 0.05% TFA in H2O/ACN 98/2 (v/v) filtered through 10 kDa MWCO PES membrane spin filters (VWR). The filtrate was transferred to HPLC vials and injected into the LC-MS/MS instrument. Each sample was measured in triplicate (3 analytical replicates of 4 biological replicates). LC-MS/MS analysis was performed on an Ultimate 3000 nano RSLC system connected to a Orbitrap Exploris 480 mass spectrometer (both Thermo Fisher Scientific, Waltham, MA, USA) with FAIMS. Peptide trapping for 5 min on an Acclaim Pep Map 100 column (2 cm x 75 µm, 3 µm) at 5 µL/min was followed by separation on an analytical Acclaim Pep Map RSLC nano column (50 cm x 75 µm, 2µm). Mobile phase gradient elution of eluent A (0.1% (v/v) formic acid in water) mixed with eluent B (0.1% (v/v) formic acid in 90/10 acetonitrile/water) was performed using the following gradient: 0 min at 4% B, 20 min at 6% B, 45 min at 10% B, 75 min at 16% B, 105 min at 25% B, 135 min at 45% B, 150 min at 65% B, 160-165 min at 96% B, 165.1-180 min at 4% B. Positively charged ions were generated at spray voltage of 2.2 kV using a stainless steel emitter attached to the Nanospray Flex Ion Source (Thermo Fisher Scientific). The quadrupole/orbitrap instrument was operated in Full MS / data-dependent MS2 mode. Precursor ions were monitored at m/z 300-1200 at a resolution of 120,000 FWHM (full width at half maximum) using a maximum injection time (ITmax) of 50 ms and 300% normalized AGC (automatic gain control) target. Precursor ions with a charge state of z=2-5 were filtered at an isolation width of m/z 4.0 amu for further fragmentation at 28% HCD collision energy. MS2 ions were scanned at 15,000 FWHM (ITmax=40 ms, AGC= 200%). 4 different compensation voltages were applied (-48V, -63V, -78V).

### Data Protocol
Tandem mass spectra were searched against the UniProt pan proteome databases (2022/12/21 (YYYY/MM/DD) of Neosartorya fumigata / A. fumigatus (https://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/pan_proteomes/UP000002530.fasta.gz) using Proteome Discoverer (PD) 3.0 (Thermo) and the database search algorithms Mascot 2.8, Comet, MS Amanda 2.0, Sequest HT with and without INFERYS Rescoring, and CHIMERYS. Two missed cleavages were allowed for the tryptic digestion. The precursor mass tolerance was set to 10 ppm and the fragment mass tolerance was set to 0.02 Da. Modifications were defined as dynamic Met oxidation, phosphorylation of Ser, Thr, and Tyr, protein N-term acetylation with and without Met-loss as well as static Cys carbamidomethylation. A strict false discovery rate (FDR) < 1% (peptide and protein level) was required for positive protein hits . Furthermore, search engine scores were filtered for either Mascot (>30), Comet (>3), MS Amanda 2.0 (>300), Sequest HT (>3), or CHIMERYS (>2). The Percolator node of PD3.0 and a reverse decoy database was used for qvalue validation of spectral matches. Only rank 1 proteins and peptides of the top scored proteins were counted. Label-free protein quantification was based on the Minora algorithm of PD3.0 using the precursor abundance based on intensity and a signal-to-noise ratio >5. Normalization was performed by using the total peptide amount method. Imputation of missing quan values was applied by using abundance values of 75% of the lowest abundance identified. Differential protein abundance was defined as a fold change of >2, pvalue/ABS(log4ratio) <0.05 and at least identified in 3 of 4 replicates of the sample group with the highest abundance.

### Publication Abstract
None

### Keywords
Rna-based therapeutics, Filamentous fungi, Lc-ms/ms, Rna interference, Aspergillus fumigatus

### Affiliations
Department of Molecular and Applied Microbiology Leibniz Institute for Natural Product Research and Infection Biology - Hans KnÃ¶ll Institute Jena Leibniz-HKI Adolf-Reichwein-Str. 23 Jena, Germany
Leibniz Institute for Natural Product Research and Infection Biology - Hans Knöll Institute

### Submitter
Thomas Krüger

### Lab Head
Dr Axel A. Brakhage
Department of Molecular and Applied Microbiology Leibniz Institute for Natural Product Research and Infection Biology - Hans KnÃ¶ll Institute Jena Leibniz-HKI Adolf-Reichwein-Str. 23 Jena, Germany


