### Accession
PXD038361

### Title
Proteomic anlaysis of Fabry iPSC derived cardiomyocytes treated with GLA modRNA

### Description
Recent studies in non-human model systems have shown therapeutic potential of modified mRNA (modRNA) treatments for lysosomal storage diseases. Here, we assessed the efficacy of a modRNA treatment to restore the expression of the ÃŽÂ±-galactosidase (GLA) gene in a human cardiac model generated from induced-pluripotent stem cell-derived from two patients with Fabry disease. In line with the clinical phenotype, cardiomyocytes from Fabry patientÃ¢â‚¬â„¢s induced pluripotent stem cells show accumulation of the glycosphinolipid Globotriaosylceramide (GB3), which is an ÃŽÂ±-galactosidase substrate. Further, the patient-specific cardiomyocytes have significant upregulation of lysosomal associated proteins. Upon modRNA treatment, a subset of lysosomal proteins were partially restored to wildtype levels, implying the rescue of the molecular phenotype associated with the Fabry genotype. Importantly, a significant reduction of GB3 levels was observed in GLA modRNA treated cardiomyocytes demonstrating that ÃŽÂ±-galactosidase enzymatic activity was restored. Together, our results validate the utility of patient IPSC-derived cardiomyocytes as a model to study disease processes in Fabry disease and the therapeutic potential of GLA modRNA treatment to reduce GB3 accumulation in the heart.

### Sample Protocol
Samples were lysed in 4% sodium deoxycholate in 100 mM Tris, pH 8.5 and heated at 95Ã‚Â°C for 5 min followed by 10 sec tip-probe sonication. The lysate was centrifuged at 16,000 x g for 10 min and supernatant quantified by BCA. An aliquot was removed and normalised to 10 Ã‚Âµg / 9 Ã‚Âµl, and the sample reduced and alkylated by the addition of 1 Ã‚Âµl solution of 100 mM tris(2-carboxyethyl)phosphine, 400 mM 2-choloracetamide in 100 mM Tris, pH 8.5 and heated at 45Ã‚Â°C for 5 min. The sample was cooled on ice briefly and digested overnight at 37Ã‚Â°C with 0.2 Ã‚Âµg of sequencing-grade trypsin (Promega) and 0.2 Ã‚Âµg of sequencing-grade LysC (Wako). The digests were adjusted to final concentration of 50% isopropanol and then acidified to a final concentration of 1% trifluoracetic acid (TFA). Peptides were loaded directly onto in-house packed styrenedivinylbenzene- reverse phase sulfonate (SDB-RPS) micro-columns, washed with 100 Ã‚Âµl of 99% isopropanol, 1% TFA followed by 100 Ã‚Âµl  of 99% ethyl acetate, 1% TFA followed by 100 Ã‚Âµl of 5% acetonitrile, 1% TFA. Peptides were eluted with 60 Ã‚Âµl of 80% acetonitrile, 1% ammonium hydroxide and drive by vacuum centrifugation. Peptides were resuspended in 2% acetonitrile, 0.1% TFA and store at -80Ã‚Â°C until analysis.

### Data Protocol
Peptides were analyzed on a Dionex 3500 nanoHPLC, coupled to an Orbitrap Exploris mass spectrometer (ThermoFischer Scientific) via electrospray ionization in positive mode with 1.9 kV at 275 Ã‚Â°C and RF set to 30%. Separation was achieved on a 50 cm Ãƒâ€” 75 Ã‚Âµm column packed with C18AQ (1.9 Ã‚Âµm; Dr Maisch, Ammerbuch, Germany) (PepSep, Marslev, Denmark) over 75 min at a flow rate of 300 nL/min. The peptides were eluted over a linear gradient of 3Ã¢â‚¬â€œ40% Buffer B (Buffer A: 0.1% formic acid; Buffer B: 80% v/v acetonitrile, 0.1% v/v FA) and the column was maintained at 50 Ã‚Â°C. Electrospray ionization was performed at 1.9 kV with 50% RF lens and positively charged peptides detected via a full scan MS (360-1033 m/z, 250% normalised AGC, 120K resolution, 50 ms injection time) followed data-independent MS/MS analysis performed with HCD of 30% normalized collision energy (NCE) (13.7 m/z isolation, 38 windows with 1 m/z overlap, 2000% normalised AGC, 30K resolution, 55ms injection time). Data were processed with Spectronaut Spectronaut 16.0.220606.53000 searched against the Human UniProt database (June 2021) using directDIA. The minimum peptide length set to 7 amino acids with specific trypsin cleavage and search criteria included oxidation of methionine and protein N-terminal acetylation set as variable modifications, and carbamidomethylation set as a fixed modification. Data were filtered to 1% FDR at the peptide and protein level (Qvalue cut-off < 0.01). Quantification was performed at MS2 level using 3-6 fragment ions which included automated interference fragment ion removal as previously described [PMID: 25724911]. Dynamic mass MS1 and MS2 mass tolerance was enabled, and local (non-linear) regression was performed for retention time calibration. A dynamic extracted ion chromatogram window size was performed, and protein quantification performed with weighted peptide average values. Statistical analysis was performed in Perseus [PMID: 27348712]. Data was log-transformed, median normalised and statistical analysis performed with unpaired student t-tests including multiple hypothesis testing correction using Benajamini-Hochberg.

### Publication Abstract
None

### Keywords
Ipsc, Fabry, Cardiomyocytes

### Affiliations
The University of Melbourne

### Submitter
Benjamin Parker

### Lab Head
Dr Benjamin Parker
The University of Melbourne


