### Accession
PXD024462

### Title
GID E3 ligase supramolecular chelate assembly configures multipronged ubiquitin targeting of an oligomeric metabolic enzyme

### Description
How are E3 ubiquitin ligases configured to match substrate quaternary structures? Here, by studying the yeast GID complex, mutation of which is Glucose-Induced Degradation deficient, we discover supramolecular chelate assembly as an E3 ligase strategy for targeting an oligomeric substrate. Cryo EM structures show that to bind the tetrameric substrate fructose-1,6-bisphosphatase (Fbp1), two otherwise functional GID E3s assemble into a 20-protein Chelator-GIDSR4, which resembles an organometallic supramolecular chelate. The Chelator-GIDSR4 assembly avidly binds multiple Fbp1 degrons and positions Fbp1 so that its protomers are simultaneously ubiquitylated at lysines near its allosteric and substrate binding sites. Significantly, key structural and biochemical features - including capacity for supramolecular assembly - are preserved in the human ortholog, the CTLH E3. Based on our integrative structural, biochemical and cell biological data, we propose that higher-order E3 ligase assembly generally underlies multipronged targeting, capable of simultaneously incapacitating multiple protomers and functionalities of oligomeric substrates.

### Sample Protocol
To determine the preferentially targeted lysines in Fbp1, it was first ubiquitylated with Chelator-GIDSR4 and subjected to proteomic analysis. To capture the initial ubiquitylation events, the assay was performed in a single-turnover pulse-chase format, wherein the concentration of the substrate was significantly exceeding that of E2~Ub. In the pulse, 10 ÂµM Ubc8 was loaded with 30 ÂµM lysine-less ubiquitin mutant (all K>R) and 0.5 ÂµM Uba1 for 15 minutes at room temperature and quenched with 50 mM EDTA. To start the chase, the pulse reaction was mixed with an equal volume of the chase-initiating mixture containing 1 ÂµM GIDAnt, 2 ÂµM Gid7, 2 ÂµM Gid4 and 4 ÂµM Fbp1-6xHis and incubated at room temperature. After 1 minute, the reaction was quenched by adding 10 mM DTT, which was then removed by desalting before proteomic analysis. Proteins were digested and prepared for LC-MS/MS measurements as previously described (Qiao et al., 2020). Briefly, samples were diluted in digestion buffer (1 M urea in 50 mM ammonium bicarbonate, pH 8.0), followed by addition of TCEP and CAA to a final concentration of 10 mM and 40 mM, respectively. After reduction and alkylation for 5 minutes at 45Â°C, samples were enzymatically digested using either trypsin (1:20 w/w, Sigma-Aldrich) alone, trypsin (1:40 w/w)/GluC (1:40 w/w, BioLab) or trypsin (1:40 w/w)/AspN (1:40 w/w, Promega) at 37Â°C overnight. Thereafter, protease activity was quenched and peptides were loaded and cleaned on SDP-RPS StageTips. Peptides were subsequently eluted with 1.25% ammonium hydroxide/80% ACN, dried using a SpeedVac centrifuge (Eppendorf, Concentrator plus) and resuspended in buffer A (2% ACN/0.1% TFA) for LC/MS-MS analysis. Peptide concentration was estimated by UV spectrometry and approximately 200 ng were loaded on a 50 cm reversed phase column (75 Î¼m inner diameter, packed in-house with ReproSil-Pur C18-AQ 1.9 Î¼m resin (Dr. Maisch GmbH)). Column temperature was maintained at 60Â°C using a homemade column oven. Peptides were separated with a binary buffer system of buffer A (0.1% formic acid (FA)) and buffer B (80% acetonitrile plus 0.1% FA), at a flow rate of 300 nl/min. We used an EASY-nLC 1200 system (Thermo Fisher Scientific), which was directly coupled online with the mass spectrometer (Q Excative HF-X, Thermo Fisher Scientific) via a nano-electrospray source. Peptides were eluted with a gradient starting at 3% buffer B and stepwise increased to 8% in 8 min, 36% in 32 min, 45% in 4 minutes and 95% in 4 min. The mass spectrometer was operated in Top12 data-dependent mode (DDA) with a full scan range of 250-1350 m/z at 60,000 resolution with an automatic gain control (AGC) target of 3e6 and a maximum fill time of 20 ms. Precursor ions were isolated with a width of 1.4 m/z and fragmented by higher-energy collisional dissociation (HCD) with a normalized collision energy (NCE) of 28%. Fragment scans were performed at a resolution of 30,000, an AGC of 1e5 and a maximum injection time of 110 ms. Dynamic exclusion was enabled and set to 15 s.

### Data Protocol
Analysis of global proteome of WT vs. Î”Gid7 yeast (Karayel et al., 2020) Spectronaut version 13 (Biognosys) was used to analyze DIA raw files using the yeast FASTA file (Swissprot, 2018) and the proteome library previously published (Karayel et al., 2020) with default settings and enabled cross run normalization. The Perseus software package version 1.6.0.7 was used for the data analysis (Tyanova et al., 2016). Protein intensities were log2-transformed and filtered to make sure that identified proteins showed expression in all biological triplicates of at least one condition. The missing values were subsequently replaced by random numbers that were drawn from a normal distribution (width=0.3 and down shift=1.8). For volcano plots, we used permutation-based FDR, which was set to 0.05 in conjunction with an S0-parameter of 0.1 to determine the significance. Determination of preferentially targeted lysines in Fbp1 by LC-MS/MS (Figure S5) Raw MS data were searched against UniProt Yeast FASTA using MaxQuant (version 1.6.2.10) with a 1% FDR at peptide and protein level. Cysteine carbamidomethylation was set as fixed, protein N-terminal acetylation, methionine oxidation and lysine diGly as variable modifications. The minimum peptide length was set to 7 amino acids, enzyme specificity was set to trypsin and two missed cleavages were allowed, permitting a maximum of 5 modifications per peptide. MS/MS spectra identifying ubiquitylated peptides of interest were obtained and exported using MaxQuant Viewer.

### Publication Abstract
None

### Keywords
Ubiquitin, E3 ligase, Gid, Cryo em, Metabolism, Supramolecular assembly, Ctlh, Gluconeogenesis

### Affiliations
Department of Proteomics and Signal Transduction Max Planck Institute of Biochemistry
Proteomics

### Submitter
Mario Oroshi

### Lab Head
Dr Matthias Mann
Department of Proteomics and Signal Transduction Max Planck Institute of Biochemistry


