### Accession
PXD035952

### Title
Uncovering the protein alterations in anther tissues of three contrasting rice types in response to short-term and long-term heat stress conditions

### Description
Heat stress drastically affects anther tissues resulting in poor plant fertility, necessitating an urgent need to determine the key proteins associated in the response of anthers to heat stress. Employing quantitative proteome analysis, we identified several cultivar-specific protein alterations in anthers of Moroberekan (Japonica, heat sensitive), IR64 (Indica, moderately heat tolerant), and Nagina 22 (Aus, heat tolerant) rice types following short-term (ST_HS; one cycle of 42°C, 4 hours before anthesis) and long-term (LT_HS; 6 cycles of 38°C, 6 hours before anthesis) heat treatments. The protein numbers were higher in Nagina 22 than IR 64 and Moroberekan anthers under unstressed and heat stress. The proteins upregulated in long-term heat stress in Nagina 22 were enriched in biological processes related to unfolded protein binding and carboxylic acid metabolism, including amino acid metabolism. In short-term heat stress, Nagina 22 anthers were enriched in proteins associated with vitamin E biosynthesis while proteins downregulated were related to ribosomal proteins. The expression of Hsp20, DnaJ, and Hsp101 was predominantly observed in Nagina 22. Overall, the heat response in Nagina 22 was associated with its capacity for adequate metabolic control and cellular homeostasis which may be critical for its higher reproductive thermotolerance

### Sample Protocol
Proteins were extracted and quantified using shotgun proteomics (Chaturvedi, Ischebeck, Egelhofer, Lichtscheidl & Weckwerth 2013; Chaturvedi et al. 2015; Valledor & Weckwerth 2014). Two biological replicates of each sample were used in this study. Anther samples were freeze-dried and ground for 2 min in a shaking mill using steel balls (2 mm diameter). The homogenized samples were resuspended in 200 μL of protein extraction buffer (100 mM Tris- HCl, pH 8.0; 5% SDS, 10% glycerol; 10 mM DTT; 1% plant protease inhibitor cocktail (Sigma P9599) and incubated at room temperature for 5 min followed by incubation for 2.5 min at 95°C and centrifugation at 21000 × g for 5 min at room temperature. The supernatant was carefully transferred to a fresh tube. Two hundred microliter of 1.4 M sucrose was added to the supernatant. Proteins were extracted twice with 200 μL TE buffer-equilibrated phenol followed by counter extraction with 400 μL of 0.7 M sucrose. Phenol phases were combined and subsequently mixed with 2.5 volumes of 0.1 M ammonium acetate in methanol for precipitation of proteins. After 16 h of incubation at -20°C, samples were centrifuged for 5 min at 5000 × g. The pellet was washed twice with 0.1 M ammonium acetate, once with acetone and air-dried at room temperature. The pellet was redissolved in 6 M urea and 5% SDS and protein concentration was determined using the bicinchoninic acid assay. Proteins were pre-fractionated by SDS-polyacrylamide gel electrophoresis. Forty micrograms of total protein were loaded onto a gel and run for 1.5 cm. Gels were fixed and stained with methanol: acetic acid: water: Coomassie Brilliant Blue R-250 (40:10:50:0.001). Gels were destained in methanol: water (40:60) and then each lane was divided into two fractions (Ghatak et al. 2021).

### Data Protocol
For protein identification, raw data were searched with the SEQUEST algorithm present in Proteome Discoverer version 1.3 (Thermo, Germany) as described in (Valledor & Weckwerth 2014) using protein FASTA. We have used the following settings in Proteome Discoverer for data analysis which include: Peptide confidence: High, which is equivalent to 1% FDR, and Xcorr of 2, 3, 4, 5, 6 for peptides of charge 2, 3, 4, 5, 6. The variable modifications were set to acetylation of N-terminus and oxidation of methionine, with a mass tolerance of 10 ppm for the parent ion and 0.8 Da for the fragment ion. The number of missed and/or non-specific cleavages permitted was 2. There were no fixed modifications, as dynamic modifications were used. The identified proteins were quantitated based on total ion count followed by a  NSAF normalization strategy (Paoletti et al. 2006). (𝑁𝑆𝐴𝐹)𝜅=(𝑃𝑆𝑀/𝐿)𝜅/∑𝑖=1𝑁(𝑃𝑆𝑀/𝐿)𝑖 The total number spectra count for the matching peptides from protein k (PSM) were divided by the protein length (L), then divided by the sum of PSM/L for all N proteins.

### Publication Abstract
Heat stress drastically affects anther tissues resulting in poor plant fertility, necessitating an urgent need to determine the key proteome regulation associated with mature anther in response to heat stress. We identified several genotype - specific protein alterations in rice anthers of Moroberekan (Japonica, heat sensitive), IR64 (Indica, moderately heat tolerant), and Nagina22 (Aus, heat tolerant) in the short-term (ST_HS; one cycle of 42&#xb0;C, 4 hours before anthesis) and long-term (LT_HS; 6 cycles of 38&#xb0;C, 6 hours before anthesis) heat stress. The proteins upregulated in long-term heat stress in Nagina22 were enriched in biological processes related to unfolded protein binding and carboxylic acid metabolism, including amino acid metabolism. In short-term heat stress, Nagina22 anthers were enriched in proteins associated with vitamin E biosynthesis and GTPase activator activity. In contrast, downregulated proteins were related to ribosomal proteins. The expression of different Hsp20 and DnaJ was genotype specific. Overall, the heat response in Nagina22 was associated with its capacity for adequate metabolic control and cellular homeostasis, which may be critical for its higher reproductive thermotolerance. This study improves our understanding of thermotolerance mechanisms in rice anthers during anthesis and lays a foundation for breeding thermotolerant varieties <i>via</i> molecular breeding.

### Keywords
Nagina 22 rice, Proteomics, Anthers, Lc-ms/ms, Moroberekan rice, Heat stress, Reproductive thermotolerance, Ir64 rice

### Affiliations
University of Vienna

### Submitter
Palak Chaturvedi

### Lab Head
Dr Palak Chaturvedi
University of Vienna


