### Accession
PXD040349

### Title
Integrative multi-omics investigation of microbes responsible for methne emission from cows fed either an ad lib or restricted diet

### Description
Multi-omics integration analysis of rumen microorganisms isolated from cows fed either an ad lib or restricted diet, and comparing this with methane emission rates for the cows.

### Sample Protocol
The rumen samples were transferred to FastPrep-compatible cryo tubes containing approximately 500 µl glass bead mixture with two bead sizes (two thirds of 106 µm (Sigma G4649)plus one third of 0.5 mm (Sigma Z250465)) and the solution was adjusted to 100 mM Tris, pH8, 5% SDS and 10 mM DTT. To lyse the cells, the samples were run in a FastPrep 24 bead-beating instrument for 3 × 45 seconds at a speed of 6.5 m/s. The sample tubes were centrifuged for 15 minutes at 20.000 × g and supernatants were kept at -20 ⁰C until further processing. To reduce the presence of contaminants, the protein extracts were cleaned by one round of Wessel-Flügge precipitation [1] and the pellets were dissolved in 5% SDS, 100 mM Tris-Cl, pH8, 10 mM DTT. The purified extracts were kept at -20 ⁰C until further processing. Protein digestion we performed using suspension trapping (STrap) as described in [2]. The peptide-containing eluates STrap were dried in a SpeedVac (Eppendorf Concentrator Plus) and re-dissolved in 15 µl MS-loading solution (0.05 % trifluoroacetic acid, 2% acetonitrile). Prior to LC-MS/MS-analysis peptide concentrations were estimated using the A205-protocol on a Nanodrop One instrument. The samples were analyzed using an Ultimate3000 RSLCnano UHPLC coupled to a QExactive hybrid quadrupole-orbitrap mass spectrometer (Thermo Fisher, Bremen, Germany). Roughly 800 ng peptides  were loaded onto a trap column (Acclaim PepMap100, C18, 5 µm, 100 Å, 300 µm i.d. × 5 mm) and then backflushed at 300 nL/min onto a 50 cm × 75 µm analytical column (Acclaim PepMap RSLC C18, 2 µm, 100 Å, 75 µm i.d. × 50 cm, nanoViper). Gradient starting conditions were 3.2 % B (99.9 % ACN, 0.1% formic acid), and the 120-minute method consisted of a 2-minute ramp to 9.6 % B, followed by a 93-minute ramp to 36 % B, a 5-minute ramp to 48 % B, and a rapid 0.1-minute ramp to 72% B for washing. The Q-Exactive mass spectrometer was set up to run a Top12-method: a full scan (300-1600 m/z) at R=70.000 followed by (up to) 12 MS/MS scans at R=17.500, using an NCE setting of 28. Singly charged precursors were excluded for MS/MS, as were precursors with >5 charges. Dynamic exclusion was set to 20 seconds.

### Data Protocol
Mass spectrometry raw data were analysed with Fragpipe version 19 and searched against the sample-specific protein sequence database (2.533.362 protein sequences) with MSFragger. The database was supplemented with contaminant protein entries, such as human keratin, trypsin, and bovine serum albumin, in addition to reversed sequences of all protein entries for estimation of false discovery rates (FDR). Oxidation of methionine and protein N-terminal acetylation were used as variable modifications, while carbomidomethylation of cysteine residues were used as fixed modification. Trypsin was chosen as digestive enzyme, maximum missed cleavages allowed was one and matching tolerance levels for both MS and MS/MS were 20 ppm. The results were filtered to 1% FDR and quantification was done using Ionquant including normalization between samples and the feature ‘match between runs’ to maximize protein identifications. Perseus [7] version 1.6.2.3 was used for further analysis. A protein group was considered valid if it was quantified in at least 50% of the replicates in at least one condition (7 restricted and 8 Ad lib).  Protein groups identified as potential contaminants were removed. Calculations of MAG/SAG abundances were done by summing LFQ values for all proteins belonging to each MAG/SAG, and differential abundance between diets were detected by a two-sided Student’s t-test (p<0.05).

### Publication Abstract
Protozoa comprise a major fraction of the microbial biomass in the rumen microbiome, of which the entodiniomorphs (order: Entodiniomorphida) and holotrichs (order: Vestibuliferida) are consistently observed to be dominant across a diverse genetic and geographical range of ruminant hosts. Despite the apparent core role that protozoal species exert, their major biological and metabolic contributions to rumen function remain largely undescribed in vivo. Here, we have leveraged (meta)genome-centric metaproteomes from rumen fluid samples originating from both cattle and goats fed diets with varying inclusion levels of lipids and starch, to detail the specific metabolic niches that protozoa occupy in the context of their microbial co-habitants. Initial proteome estimations via total protein counts and label-free quantification highlight that entodiniomorph species Entodinium and Epidinium as well as the holotrichs Dasytricha and Isotricha comprise an extensive fraction of the total rumen metaproteome. Proteomic detection of protozoal metabolism such as hydrogenases (Dasytricha, Isotricha, Epidinium, Enoploplastron), carbohydrate-active enzymes (Epidinium, Diplodinium, Enoploplastron, Polyplastron), microbial predation (Entodinium) and volatile fatty acid production (Entodinium and Epidinium) was observed at increased levels in high methane-emitting animals. Despite certain protozoal species having well-established reputations for digesting starch, they were unexpectedly less detectable in low methane emitting-animals fed high starch diets, which were instead dominated by propionate/succinate-producing bacterial populations suspected of being resistant to predation irrespective of host. Finally, we reaffirmed our abovementioned observations in geographically independent datasets, thus illuminating the substantial metabolic influence that under-explored eukaryotic populations have in the rumen, with greater implications for both digestion and methane metabolism.

### Keywords
Metagenomics, Metaproteomics, Teagasc

### Affiliations
Norwegian University of Life Sciences
Faculty of Chemistry, Biotechnology and Food Science Norwegian University of Life Sciences Post Office Box 5003 1432, Ås Norway

### Submitter
Magnus Arntzen

### Lab Head
Dr Phil Byron Pope
Faculty of Chemistry, Biotechnology and Food Science Norwegian University of Life Sciences Post Office Box 5003 1432, Ås Norway


