### Accession
PXD003683

### Title
Olfactory bulb proteome characterization in APP/PS1 mice

### Description
Here we used double-transgenic amyloid precursor protein/presenilin 1 (APPswe/PS1dE9) mice and label-free quantitative proteomics to analyze potential early pathological effects on the olfactory bulb (OB) during AD progression.

### Sample Protocol
Animals- The generation of mice expressing a chimeric mouse/human APP (Mo/HuAPPswe), and a human exon-9-deleted variant of PS1 (PS1-dE9) has been previously described (Borchelt DR, Eckman CB et al. Neuron 1996). In our colony, APP/PS1 transgenic mice developed a few cortical Aβ plaques at the age of three months and began to have deficits in learning and memory at the age of 6 months (Aso E Brain Pathol 2012). The animals were kept under controlled temperature, humidity and light conditions with food and water provided ad libitum. Animal care procedures were conducted in accordance with the European Community Council Directive 86/609/EEC and approved by the local ethics committee. Twenty animals, divided into four groups, were used for the proteomic study, with at least 5 wild-type and 5 APP/PS1 transgenic mice per stage (3 and 6-month-old). For histological analysis, three different OB sections from 5 wild-type and 5 APP/PS1 transgenic mice per stage (3 and 6-month-old) were used.  Label free LC-MS/MS – Protein extracts were precipitated with methanol/choloroform, and pellets dissolved in 6M Urea, Tris 100mM pH 7.8.  Protein quantitation was performed with the Bradford assay kit (Bio-Rad). Protein enzymatic cleavage (10ug) was carried out with trypsin (Promega; 1:20, w/w) at 37°C for 16 h as previously described (Andrej Shevchenko, Henrik Tomas, Jan Havlis et al Nature Protocols 2007). Peptides mixtures were separated by reverse phase chromatography using an Eksigent nanoLC ultra 2D pump fitted with a 75 μm ID column (Eksigent 0.075 x 150). Samples were first loaded for desalting and concentration into a 0.5 cm length 300 μm ID precolumn packed with the same chemistry as the separating column. Mobile phases were 100% water 0.1% formic acid (FA) (buffer A) and 100% Acetonitrile 0.1% FA (buffer B). Column gradient was developed in a 240 min two step gradient from 5% B to 25% B in 210 min and 25%B to 40% B in 30 min. Column was equilibrated in 95% B for 9 min and 5% B for 14 min. During all process, precolumn was in line with column and flow maintained all along the gradient at 300 nl/min. Eluting peptides from the column were analyzed using an Sciex 5600 Triple-TOF system. Information data acquisition was acquired upon a survey scan performed in a mass range from 350 m/z up to 1250 m/z in a scan time of 250 ms. Top 30 peaks were selected for fragmentation. Minimum accumulation time for MS/MS was set to 100 ms giving a total cycle time of 3.8 s. Product ions were scanned in a mass range from 230 m/z up to 1500 m/z and excluded for further fragmentation during 15 s.

### Data Protocol
Peptide Identification and Quantification – MS/MS data acquisition was performed using Analyst 1.5.2 (AB Sciex) and spectra ﬁles were processed through Protein Pilot TM Software (v.4.0.8085-ABSciex) using ParagonTM Algorithm (v.4.0.0.0) (Shilov IV, Seymour SL et al Mol Cell Proteomics 2007) for database search, Progroup™ for data grouping, and searched against the concatenated target-decoy UniProt mouse database. False discovery rate was performed using a non lineal fitting method (Tang WH, Shilov IV, Seymour SL JPR 2008) and displayed results were those reporting a 1% Global false discovery rate or better. The peptide quantification was performed using the Progenesis LC−MS software (ver. 2.0.5556.29015, Nonlinear Dynamics). Using the accurate mass measurements from full survey scans in the TOF detector and the observed retention times, runs were aligned to compensate for between-run variations in our nanoLC separation system. To this end, all runs were aligned to a reference run automatically chosen by the software, and a master list of features considering m/z values and retention times was generated. The quality of these alignments was manually supervised with the help of quality scores provided by the software. The peptide identifications were exported from Protein Pilot and imported in Progenesis LC− MS where they were matched to the respective features. Output data files were managed using R scripts for subsequent statistical analyses and representation. Proteins identified by site (identification based only on a modification), reverse proteins (identified by decoy database) and potential contaminants were filtered out. Proteins quantified with at least one unique peptide, an ANOVA  p-value lower than 0.05, and an absolute fold changes of <0.77 (down-regulation) or >1.3 (up-regulation) in linear scale were considered to be significantly differentially expressed and used for further evaluation.

### Publication Abstract
None

### Keywords
Proteomics, Olfactory bulb

### Affiliations
Proteomics unit
Clinical Neuroproteomics Group, Navarrabiomed (Spain)

### Submitter
Enrique SantamarÃ­a

### Lab Head
Dr Enrique Santamaria
Clinical Neuroproteomics Group, Navarrabiomed (Spain)


