### Accession
PXD002170

### Title
Comparative analysis of label-free and 8-plex iTRAQ approach for quantitative tissue proteomic analysis

### Description
MS-based approaches have been successfully utilized for the comprehensive characterization of the cell proteome. However, in the case of quantitative proteomics the open question still remains, which quantification method is more suitable for identification of proteins with the altered abundance, particularly for the highly complex clinical samples. In this study, a thorough comparison of label-free method (intensity-based) and 8-plex iTRAQ was conducted as applied to the analysis of tumor samples from non-muscle invasive and muscle-invasive bladder cancer. In the case of the label-based approach, the results obtained for the unfractionated and fractionated sample were included. To obtain representative results from each approach, both experiments were conducted according to routinely applicable protocols. Additionally, the sampling procedure was standardized to reduce the effect of biological and analytical variability on the quantification results.

### Sample Protocol
Bladder cancer tissue specimens were collected from patients undergoing transurethral resection of bladder cancer. Approximately 20 mg of bladder cancer tissue (n=4 pTa stage, n=4 pT2+ stage) was homogenized in 150 uL of lysis buffer (4% SDS, 0.1M DTE, 0.1M Tris-HCl pH 7.6) using blade homogenizer followed by sonication. 200 ug and 100 ug of protein extracts were processed using the FASP (Filter Aided Sample Preparation), separately for label-free and iTRAQ experiments, respectively. Tryptic digest for the label-free quantification were lyophilized and analyzed by LC-MS/MS.  In the case of label-based method, tryptic digest peptides were labelled using 8-plex iTRAQ Reagent kit (AB Sciex) according to manufacturer instructions. Samples from non-invasive tumor tissue (pTa stage) were labelled using 113-116 tags, whereas for the invasive tumors (pT2+) 117-119 and 121 tags were used. Subsequently, eight individual samples were mixed and lyophilized to dryness and subsequently 80 ug were purified using Pierce C18 Tips, 100uL bed (Thermo Scientific).  As an alternative approach, the high pH reverse phase chromatography on the Dionex P680 HPLC system was applied to purify and pre-fractionate the remaining peptide mixture. Labelled peptides were lyophilized to dryness and redissolved in 250 uL of high pH buffer. After loading of 200 uL onto XBridge 4.6 x 150 mm C18 column (BEH Technology) at flow rate 0.04 mL/min in 0.05% NH4OH, sample was eluted with a gradient of solvent A: 0.05% NH4OH in water verses solvent B: 0.05% NH4OH in 100% acetonitrile starting at 5% B for 15 min, then to 35%B at 25 min then to 80%B at 30 min followed by 5 min rinsing at 80%B.  In total, five 1.2 mL fractions were collected starting from 21 min up to 35 min of the gradient. Prior to the LC-MS/MS analysis, three fractions with the lowest peptide content (1, 4, 5) were pooled.  10 ug of protein digest were loaded onto a Dionex Ultimate 3000 RSLS nano flow system (Dionex, Camberly UK). After loading onto a Dionex 0.1x20 mm 5 um C18 nano trap column at a flow rate of 5 ul/min in 98% 0.1% formic acid and 2% acetonitrile, samples were applied onto an Acclaim PepMap C18 nano column 75 um x 50 cm, 2 um 100 A at a flow rate of 0.3 ul/min. The trap and nano flow column were maintained at 35Â°C. The samples were eluted with a gradient of solvent A: 0.1% formic acid verses solvent B: 80% acetonitrile starting at 1%B for 5 min rising to 5%B at 10 min then to 25%B at 360 min then 65%B at 480 min. The eluent was ionized using a Proxeon nano spray ESI source operating in positive ion mode into an Orbitrap Velos FTMS (Thermo Finnigan, Bremen, Germany). Ionization voltage was 2.6 kV and the capillary temperature was 275oC.  The mass spectrometer was operated in MS/MS mode scanning from 380 to 2000 amu. The top 20 multiply charged ions were selected from each scan for MS/MS analysis using CID at 40% collision energy. The resolution of ions in MS1 was 60,000 and 7,500 at m/z 400 for CID MS2. For the iTRAQ samples, the top 20 multiply charged ions were selected from each scan for MS/MS analysis using HCD at 45% collision energy. AGC settings were 1,000,000 for full scan in the FTMS and 200,000 for MSn. Resolution in MS2 at m/z 115 was 16,300. Dynamic exclusion was enabled with a repeat count of 1, exclusion duration of 30 seconds.

### Data Protocol
The processing of the individual raw MS data files was conducted using commercially available software i.e. Proteome Discoverer v. 1.4.0.288 (Thermo Scientific). The RT and m/z feature detection was followed by the manufacturer guidelines by using the below settings. Database search was carried out against Human Swiss-Prot Database (30/10/2013) containing only the canonical sequences with 20 277 entries using Sequest search engine implemented in Proteome Discoverer. The following search parameters were applied: a) precursor mass tolerance 10 ppm, b) fragment mass tolerance: 0.8 Da and 0.05 Da for label-free and iTRAQ experiments, respectively, c) fixed modification: carbamidomethylation of cysteine (C) and additionally for the labeling experiment an iTRAQ modification of N-terminus and lysine residues were added, d) variable modification: oxidation of methionine (M) and for label-based experiment iTRAQ modification on tyrosine (Y) was added, e) allowing one missed cleavage site. The false discovery rate evaluation was performed by the search against the reverse database and by using Percolator node (Proteome Discoverer 1.4). Identified peptides were initially filtered accounting for the mass deviation below 5 ppm between experimental and theoretical mass, false discovery rate below 1% (assigned in Proteome Discoverer as high confidence peptides). The peak area-based quantification uses a precursor ions to assess the relative abundance of identified proteins in label-free approach. For each precursor ion, peak area (i.e. area under the curve) is calculated from the extracted ion chromatogram during the data processing in Proteome Discoverer by using Precursor Ions Area Detector node; whereas the the 8-plex iTRAQ quantification was performed based on the reporter ion intensities detected by the Reporter Ions Quantifier Node in Proteome Discoverer.

### Publication Abstract
High resolution proteomics approaches have been successfully utilized for the comprehensive characterization of the cell proteome. However, in the case of quantitative proteomics an open question still remains, which quantification strategy is best suited for identification of biologically relevant changes, especially in clinical specimens. In this study, a thorough comparison of a label-free approach (intensity-based) and 8-plex iTRAQ was conducted as applied to the analysis of tumor tissue samples from non-muscle invasive and muscle-invasive bladder cancer. For the latter, two acquisition strategies were tested including analysis of unfractionated and fractioned iTRAQ-labeled peptides. To reduce variability, aliquots of the same protein extract were used as starting material, whereas to obtain representative results per method further sample processing and MS analysis were conducted according to routinely applied protocols. Considering only multiple-peptide identifications, LC-MS/MS analysis resulted in the identification of 910, 1092 and 332 proteins by label-free, fractionated and unfractionated iTRAQ, respectively. The label-free strategy provided higher protein sequence coverage compared to both iTRAQ experiments. Even though pre-fraction of the iTRAQ labeled peptides allowed for a higher number of identifications, this was not accompanied by a respective increase in the number of differentially expressed changes detected. Validity of the proteomics output related to protein identification and differential expression was determined by comparison to existing data in the field (Protein Atlas and published data on the disease). All methods predicted changes which to a large extent agreed with published data, with label-free providing a higher number of significant changes than iTRAQ. Conclusively, both label-free and iTRAQ (when combined to peptide fractionation) provide high proteome coverage and apparently valid predictions in terms of differential expression, nevertheless label-free provides higher sequence coverage and ultimately detects a higher number of differentially expressed proteins. The risk for receiving false associations still exists, particularly when analyzing highly heterogeneous biological samples, raising the need for the analysis of higher sample numbers and/or application of adjustment for multiple testing.

### Keywords
Tissue proteomics, Lc-ms/ms, Label-free, Itraq, Bladder cancer

### Affiliations
Biomedical Research Foundation, Academy of Athens, Greece
Biotechnology Division, Centre of Basic Research,  Biomedical Research Foundation of the Academy of Athens (BRFAA) Athens, Greece

### Submitter
Agnieszka Latosinska

### Lab Head
Dr Dr Antonia Vlahou
Biotechnology Division, Centre of Basic Research,  Biomedical Research Foundation of the Academy of Athens (BRFAA) Athens, Greece


