### Accession
PXD021821

### Title
Enhanced epigenetic profiling of classical human monocytes reveals a specific signature of healthy aging in the DNA methylome

### Description
The impact of healthy aging on molecular programming of immune cells is poorly understood. Here, we report comprehensive characterization of healthy aging in human classical monocytes, with a focus on epigenomic, transcriptomic, and proteomic alterations, as well as the corresponding proteomic and metabolomic data for plasma, using healthy cohorts of 20 young and 20 older males (~27 and ~64 years old on average). For each individual, we performed eRRBS-based DNA methylation profiling, which allowed us to identify a set of age-associated differentially methylated regions (DMRs) â€“ a novel, cell-type specific signature of aging in DNA methylome. Hypermethylation events were associated with H3K27me3 in the CpG islands near promoters of lowly-expressed genes, while hypomethylated DMRs were enriched in H3K4me1 marked regions and associated with age-related increase of expression of the corresponding genes, providing a link between DNA methylation and age-associated transcriptional changes in primary human cells.

### Sample Protocol
For each donor, snap frozen monocytes (1x10^6) were shipped on dry ice to Biognosys for protein extraction and LC-MS analysis. Details of Biognosys Discovery protein profiling pipeline (HRM ID+ mass spectrometry) can be found on their website (https://biognosys.com/technology/#discovery-proteomics). On average, 5,580 proteins were quantified per sample. In total, 5,804 proteins represented by 74,348 peptides were quantified across all samples.

### Data Protocol
Statistical assessment was performed by Biognosys (https://biognosys.com/technology/#discovery-proteomics). In brief, for each protein the fold change of each peptide ion variant was estimated as average abundance of peptide ion variant across biological replicates in old group / average abundance of peptide ion variant across biological replicates in young group. The values then were log-transformed and fold changes of all peptides belonging to the same proteins were compared to zero using two-sided paired t-test. Multiple testing correction was performed as described in Storey et al.64 We removed MHC proteins from the analysis due to high similarity between the variants that could not be accurately resolved using this data.

### Publication Abstract
The impact of healthy aging on molecular programming of immune cells is poorly understood. Here, we report comprehensive characterization of healthy aging in human classical monocytes, with a focus on epigenomic, transcriptomic, and proteomic alterations, as well as the corresponding proteomic and metabolomic data for plasma, using healthy cohorts of 20 young and 20 older males (~27 and ~64 years old on average). For each individual, we performed eRRBS-based DNA methylation profiling, which allowed us to identify a set of age-associated differentially methylated regions (DMRs) - a novel, cell-type specific signature of aging in DNA methylome. Hypermethylation events were associated with H3K27me3 in the CpG islands near promoters of lowly-expressed genes, while hypomethylated DMRs were enriched in H3K4me1 marked regions and associated with age-related increase of expression of the corresponding genes, providing a link between DNA methylation and age-associated transcriptional changes in primary human cells.

### Keywords
Human; monocyte; aging;

### Affiliations
Department of Pathology and Immunology Washington University School of Medicine
Washington University in St. Louis

### Submitter
Irina Shchukina

### Lab Head
Dr Maxim Artyomov
Department of Pathology and Immunology Washington University School of Medicine


