### Accession
PXD016379

### Title
The longer the worse: a combined proteomic and targeted study of the long-term versus short-term effects of silver nanoparticles on macrophages

### Description
Despite considerable research effort devoted to the study of the effects of silver nanoparticles on mammalian cells in recent years, data on the potential long terms effects of this nanomaterial remain scarce, and centered on epithelial cells. The aim of this study was to explore the effects of silver nanoparticles on macrophages. To this end, RAW 264.7 murine macrophages were exposed to either 1 µg/ml silver nanoparticles for 20 days, i.e. a chronic exposure scheme, or to 20 µg/ml silver nanoparticles for 24 hours, i.e. an acute exposure scheme. A proteomic study was then conducted to study and compare the cellular responses to both exposure schemes. They proved to be essentially different, and stronger for the chronic exposure scheme. Targeted validation studies showed effects of chronic exposure to silver nanoparticles on detoxifying enzymes such as biliverdin reductase B, which was increased, and on central metabolism enzymes such as triose phosphate isomerase, which activity decreased under chronic exposure to silver nanoparticles.  Chronic exposure to silver nanoparticles also induced a decrease of reduced glutathione content, a decreased phagocytic activity and reduced macrophages responses to lipopolysaccharide, as exemplified by nitric oxide and interleukin 6 production. Overall, chronic exposure to silver nanoparticles induced stronger effects than acute exposure on macrophages in the metabolic (glutathione level, mitochondrial potential) and functional (phagocytosis, cytokine production) parameters tested.

### Sample Protocol
Total cellular proteins were extracted by lysis of the cells in 7M urea, 2M thiourea, 4% (w/v) CHAPS, 5mM TCEP, 30mM spermine base and 60 mM HCl. AFter 30 minutes at room temperature, the cells extracts were clarified by centrifugation (15000g, 15 minutes), the supernatants collected and the protein concentrations determined. 250 micrograms of proteins were separated by two-dimensional electrophoresis (linear 4-8 pH gradient in the first dimension, 10% polyacrylamide gels in the second dimension). The proteins were detected by silver staining, the spots of interest excised, and destained with Farmer's reducer (ferricyanide and thiosulfate). After 5 rinses in water, one rinse in 100mM ammonium bicarbonate and another 5 rinses in water, the spots were dehydrated in 20% ethanol. They can be stored frozen at -20°C at this step. In gel digestion was performed with an automated protein digestion system, MassPrep Station (Waters, Milford, USA). The gel plugs were washed twice with 50 µL of 25 mM ammonium hydrogen carbonate (NH4HCO3) and 50 µL of acetonitrile. The cysteine residues were reduced by 50 µL of 10 mM dithiothreitol at 57°C and alkylated by 50 µL of 55 mM iodoacetamide. After dehydration with acetonitrile, the proteins were cleaved in gel with 10 µL of 7 ng/µL of modified porcine trypsin (Promega, Madison, WI, USA) in 25 mM NH4HCO3. The digestion was performed overnight at room temperature. The generated peptides were extracted with 40 µL of 60% acetonitrile in 0.1% formic acid. Acetonitrile was evaporated under vacuum before nanoLC-MS/MS analysis. NanoLC-MS/MS analysis was performed using a nanoACQUITY Ultra-High-Performance-LC (Waters Corporation, Milford, USA) coupled to the TripleTOF 5600 (Sciex, Ontario, Canada) . The nanoLC system was composed of ACQUITY UPLC® CSH130 C18 column (250 mm x 75 µm with a 1.7 µm particle size, Waters Corporation, Milford, USA) and a Symmetry C18 precolumn (20 mm × 180 µm with a 5 µm particle size, Waters Corporation, Milford, USA). The solvent system consisted of 0.1% formic acid in water (solvent A) and 0.1% formic acid in acetonitrile (solvent B). 4 µL of sample were loaded into the enrichment column during 3 min at 5 µL/min with 99% of solvent A and 1% of solvent B. Elution of the peptides was performed at a flow rate of 300 nL/min with a 8-35% linear gradient of solvent B in 9 minutes. The TripleTOF 5600 (Sciex, Ontario, Canada) was operated in positive mode, with the following settings: ion spray voltage floating (ISVF) 2300 V, curtain gas (CUR) 10, interface heater temperature (IHT) 150, ion source gas 1 (GS1) 2, declustering potential (DP) 80 V. Data-dependent acquisition was performed using the Information-dependent acquisition (IDA) mode with Top 10 MS/MS scans. The MS scan had an accumulation time of 250 ms on m/z [400;1250] range and the MS/MS scans 100 ms on m/z [150;1800] range in high sensitivity mode. Switching criteria were set to ions with charge state of 2+ - 4+ and an abundance threshold of more than 500 counts, exclusion time was set at 4 s. IDA rolling collision energy script was used for automatically adapting the CE. Mass calibration of the analyser was achieved using peptides from digested BSA. The complete system was fully controlled by AnalystTF 1.7 (Sciex). Raw data collected were processed and converted with MSDataConverter in .mgf peak list format.

### Data Protocol
For protein identification, the MS/MS data were interpreted using a local Mascot server with MASCOT 2.5.1 algorithm (Matrix Science, London, UK) against UniProtKB/SwissProt (2017_09 (555594 protein entries). The research was carried out without taxonomy restrictions. Spectra were searched with a mass tolerance of 15 ppm for MS and 0.05 Da for MS/MS data, allowing a maximum of one trypsin missed cleavage. Carbamidomethylation of cysteine residues, oxidation of methionine residues, acetylation of protein n-terminus and phosphorylation of serine, threonine and tyrosine residues were specified as variable modifications. Protein identifications were validated with at least two peptides with Mascot ion score above 30.

### Publication Abstract
None

### Keywords
Nanoparticles, 2d gel, Mouse, Lc-ms/ms, Macrophage

### Affiliations
LSMBO
Laboratoire de Spectrométrie de Masse BioOrganique (LSMBO), Université de Strasbourg, CNRS, IPHC UMR 7178, 67087 Strasbourg Cedex 2, France

### Submitter
Hélène Diemer

### Lab Head
Dr Helene DIEMER
Laboratoire de Spectrométrie de Masse BioOrganique (LSMBO), Université de Strasbourg, CNRS, IPHC UMR 7178, 67087 Strasbourg Cedex 2, France


