### Accession
PXD028082

### Title
Bioinformatics and mass spectrometry-based proteomics reveal alcohol dehydrogenase 1B as a blood biomarker candidate to monitor acetaminophen-induced liver injury

### Description
Acute liver injury (ALI) is a severe disorder resulting from excessive hepatocyte cell death and frequently caused by acetaminophen intoxication. Clinical management of ALI progression is hampered by the dearth of blood biomarkers available. In this study, a bioinformatics workflow was developed to screen Omics databases and identify potential biomarkers for hepatocyte cell death. Then, discovery proteomics was harnessed to select among these candidates those that were specifically detected into the blood of acetaminophen-induced ALI patients. Among these candidates, the isoenzyme alcohol dehydrogenase 1B (ADH1B) was massively leaked into the blood. To evaluate ADH1B, we developed a targeted proteomics assay and quantified ADH1B in serum samples collected at different times from 17 patients admitted for acetaminophen-induced ALI. Serum ADH1B concentrations increased markedly during the acute phase of the disease and dropped to undetectable levels during recovery. In contrast to alanine aminotransferase activity, the rapid drop in circulating ADH1B concentrations was followed by an improvement of the in-ternational normalized ratio (INR) within 10 to 48h and was associated with a favorable outcome. In conclusion, the combination of Omics data exploration and proteomics revealed ADH1B as a new blood biomarker which could be useful to improve the monitoring of acetaminophen-induced ALI.

### Sample Protocol
Blood samples were collected from patients admitted for acetaminophen-induced ALI/ALF. Serum was isolated by centrifugation at 1000 g for 15 min in the 30 min following collection. Aliquots of serum supernatants were immediately frozen at -80 °C be-fore shipping to the proteomics laboratory in dry ice. After receipt in the proteomics laboratory, samples were thawed and aliquoted (50 µL) before storage at -80 °C. The French Blood Service (EFS) provided anonymous serum samples from healthy donors with no history of acetaminophen-induced hepatotoxicity. Serum samples were digested using multiple enzyme digestion – filter aided sample preparation (MED-FASP). Briefly, each serum sample (3 µl) was loaded on a 10-kDa cutoff ultrafiltration device (Amicon). Proteins were denatured and reduced on the device in 4 M urea, 25 mM NH4HCO3 and 20 mM TCEP. The sample was washed with 4 M urea, 25 mM NH4HCO3 before performing alkylation in 4 M urea, 25 mM NH4HCO3 and 55 mM iodoacetamide. After two additional washing steps, the sample volume was reduced to 50 μL and proteins were digested for 2 h at 37 °C using trypsin/LysC mix (Promega) at a protein/enzyme ratio of 1:20 (w/w). The urea concentration was reduced to 1 M and digestion was allowed to proceed 5 h at 37 °C. Proteolytic peptides were recovered by adding 100 µL of NH4HCO3 25 mM to the filter and centrifuging for 15 min at 12 000 g at 4 °C. The peptide digest was purified on a C18 macro SpinColumns (Harvard Apparatus) and 5 µg aliquotes of each sample were dried by vacuum centrifugation. One aliquote of each patients was solubilised in 20 µL of 5% acetonitrile, 0.1% trifluoroacetic acid containing 0.5 UI/µL HRM iRT. 2 μL (equivalent to around 500 ng protein) of this solution was analysed by nanoLC- MS/MS (Ultimate 3000 nano RSLC and Q Exactive HF equipped with Nanospray Flex Ion Source, Thermo Fisher Scien-tific). Peptides were sampled on an Acclaim™ PepMap™ 100 C18 300 µm x 5 mm pre-column (Thermo Fisher Scientific) and separated on a Reprosil-Pur 120 C18-AQ, 1.9 μm, 75 µm x 25 cm column (Dr Maisch). The nanoLC method consisted in a 60-min non-linear gradient ranging from 4% to 50% solvent B (80% acetonitrile, 20% water, 0.1% formic acid) at a flow rate of 300 nL/min, with the column oven at 35 °C. MS and MS/MS data were acquired using Xcalibur (Thermo Fisher Scientific). A data-dependent top-20 MS acquisition method was launched with the Q Exactive HF in positive mode. The nanospray voltage was set to 2 kV, the ion transfer capillary temperature at 270 °C; and the S-Lens RF amplitude at 55%. Survey full-scan MS spectra (m/z = 400–1600) were acquired in the Orbitrap at a resolution of 60 000 after the accumulation of 10^6 ions (maximum filling time: 200 ms). The 20 most intense ions (excluding z = 1 and unassigned charge states) from the survey scan were fragmented using HCD with a normalized collision energy of 30, a resolution of 15,000 and an AGC target of 10^5 ions (max injection time: 50 ms). Dynamic exclusion was set to 30s, intensity threshold to trigger MS/MS at 2x10^4 and default charge state to 2.

### Data Protocol
Peptides and proteins were identified using Mascot (version 2.7.0.1) through concomitant searches against the Uniprot database (Homo sapiens taxonomy, downloaded in May 2021) and a homemade database of frequently observed non-human contaminants. Trypsin/P was chosen as the enzyme and two missed cleavages were allowed. Precursor and frag-ment mass error tolerances were set at respectively at 10 and 20 ppm. Peptide modifica-tions allowed during the search were: Carbamidomethyl (C, fixed), Acetyl (Protein N-term, variable) and Oxidation (M, variable). The Proline software [38] was then used to filter the results: conservation of rank 1 peptides, peptide length ≥ 6 amino acids, peptide score ≥ 25, allowing to reach a false discovery rate of peptide-spectrum-match identifications < 1% as calculated on peptide-spectrum-match scores by employing the reverse database strategy, and minimum of one specific peptide per identified protein group. Proline was then used to perform a compilation, grouping and MS1 label-free quantification of the identified pro-tein groups based on razor and specific peptides. Statistical analyses were performed using ProStaR [39]. Proteins identified in the reverse and contaminant databases, proteins identified by MS/MS in less than two replicates of one condition, and proteins for which fewer than 3 abundance values were available in a single condition, were removed from the list. After log2 transformation, abundance values were normalised by vsn (variance stabilizing normalization) before imputing missing values (slsa algorithm for partially observed values in the condition and DetQuantile algo-rithm for totally absent values in the condition); statistical testing was performed using a Limma moderated t-test. Differentially recovered proteins were sorted out using a log2(Fold Change) cut-off of 1 and a p-value threshold (on the remaining proteins) that guarantees a Benjamini-Hochberg FDR < 1%.

### Publication Abstract
None

### Keywords
Proteomics; mass spectrometry; liver; blood; biomarker; acetaminophen

### Affiliations
Univ. Grenoble Alpes, CEA, Inserm, IRIG, BioSanté U1292, Grenoble, France
Laboratoire d'Etude de la Dynamique des ProtÃ©omes

### Submitter
Hesse Anne-Marie

### Lab Head
Dr Virginie Brun
Univ. Grenoble Alpes, CEA, Inserm, IRIG, BioSanté U1292, Grenoble, France


