### Accession
PXD000587

### Title
Large-scale proteome characterization of the cassava root

### Description
Cassava is the most important root crop in the tropics but rapid post-harvest physiological root deterioration (PPD) is a major constraint to commercial cassava production. We used label-free quantitative proteomics to generate an extensive cassava root and PPD proteome. Over 2400 unique proteins were identified in the cassava root and nearly 300 proteins showed significant abundance regulation during PPD. A candidate gene for reducing PPD was identified from the regulated proteins with enzymatic assays and afterwards verified with a transgene approach. This demonstrates the relevance of proteomics approach for crop improvements.

### Sample Protocol
Stem-propagated 10 month-old plants of cassava cultivar 60444 were grown in greenhouse (16 h light, 60% humidity, day-night temperatures: 26�C - 17�C). Roots were harvested at 10 am and roots were sliced into 5 mm thick slices. The root slices were layered on Petri dishes containing filter papers pre-soaked with 2 ml of distilled water to prevent drying. The sliced roots were then randomized and stored at a constant temperature of 28�C in the dark. For each time point, three replicates consisting of two root slices each were sampled. Slices were taken and frozen in liquid nitrogen after incubation for 0, 6, 12, and 24 h.  Root slices were ground in liquid nitrogen. Total proteins were extracted from root powder with an SDS extraction buffer in a 1:2 ratio (w/v) and the protein extracts were ultracentrifuged at 100000 g for 45 min. Equal amounts of proteins (180 �g) per sample were subjected to 1-D SDS-PAGE. After electrophoresis the gel was stained with Coomassie. Each lane, corresponding to one replicate, was cut into 5 equal gel slices longitudinally and each gel slice was diced into small pieces. The gel pieces were destained completely and washed prior to in-gel tryptic digest with incubation overnight at 30�C. Peptides were eluted and purified using Sep-Pak reverse-phase cartridges. Samples were analysed on a LTQ-Orbitrap mass spectrometer coupled to an Eksigent-Nano-HPLC system. Solvent composition at the two channels was 0.2% formic acid, 1% acetonitrile for channel A and 0.2% formic acid, 80% acetonitrile for channel B. Peptides were loaded on a self-made tip column packed with reverse phase C18 material and eluted with a flow rate of 200 nl per min by a gradient from 3 to 15% of B in 5 min, 40% B in 50 min, 97% B in 56 min. Full-scan MS spectra (300?2000 m/z) were acquired with a resolution of 60000 at 400 m/z after accumulation to a target value of 500000. Collision induced dissociation (CID) MS/MS spectra were recorded in data dependent manner in the ion trap from the five most intense signals above a threshold of 500, using a normalized collision energy of 28% and an activation time of 30 ms. Charge state screening was enabled and singly charge states were rejected. Precursor masses already selected for MS/MS were excluded for further selection for 60 s and the exclusion window was set to 20 ppm. The size of the exclusion list was set to a maximum of 500 entries.

### Data Protocol
Raw data of gel slices with equal molecular weight were loaded together into the commercial software package Progenesis LCMS Version. 4.0 (Nonlinear Dynamics), a software tool developed for label-free quantification of LC-MS data. For data loading, the option "High Mass Accuracy Instrument" was selected. LC-MS data were normalized and aligned according to manufacturer specifications. In the aligning step we manually seeded "three to five" vectors along the retention time gradient to give the automatic aligning a good starting point.  For the annotation of the mastermap, Mascot generic files (.mgf file format) generated with Progenesis LCMS (using up to five tandem mass spectra for each sequenced feature with the top 200 fragment ion peaks and denoising and deisotopting option from Progenesis LCMS) were searched against the cassava 4.1 protein database with known mass spectrometry contaminants attached using the Mascot 2.3 search engine. Parameters for precursor tolerance and fragment ion tolerance were set to � 10 ppm and � 0.6 Da, respectively. The Mascot results were exported and filtered. Only rank one assignments, and assignments matching the term bold red (significant and assigned to the best protein match) were kept. Mascot ion score cut-off was set at 25, which resulted in a peptide FDR < 1%. The three biological replicates were grouped for quantification analysis. The Progenesis experiments were first performed individually for each molecular weight fraction and subsequently combined within the software using the "combine fractions" approach. Peptide features confidently assigned to proteins are stacked for quantification. The statistical analysis was then performed for each protein on the stacks of normalized feature volumes. ANOVA statistical test was applied to the selected PPD time points using data from the three biological replicates and proteins with significant regulation were identified (p<0.05). We also applied a minimum 1.5-fold change cut-off to the list of regulated proteins to identify proteins with differential regulation.  Cassava proteins were ranked with the T3PQ method, which is based on the linearity between the average of the three most intense MS signals of different tryptic peptides of a given protein and its abundance.

### Publication Abstract
Cassava (Manihot esculenta) is the most important root crop in the tropics, but rapid postharvest physiological deterioration (PPD) of the root is a major constraint to commercial cassava production. We established a reliable method for image-based PPD symptom quantification and used label-free quantitative proteomics to generate an extensive cassava root and PPD proteome. Over 2600 unique proteins were identified in the cassava root, and nearly 300 proteins showed significant abundance regulation during PPD. We identified protein abundance modulation in pathways associated with oxidative stress, phenylpropanoid biosynthesis (including scopoletin), the glutathione cycle, fatty acid &#x3b1;-oxidation, folate transformation, and the sulfate reduction II pathway. Increasing protein abundances and enzymatic activities of glutathione-associated enzymes, including glutathione reductases, glutaredoxins, and glutathione S-transferases, indicated a key role for ascorbate/glutathione cycles. Based on combined proteomics data, enzymatic activities, and lipid peroxidation assays, we identified glutathione peroxidase as a candidate for reducing PPD. Transgenic cassava overexpressing a cytosolic glutathione peroxidase in storage roots showed delayed PPD and reduced lipid peroxidation as well as decreased H<sub>2</sub>O<sub>2</sub> accumulation. Quantitative proteomics data from ethene and phenylpropanoid pathways indicate additional gene candidates to further delay PPD. Cassava root proteomics data are available at www.pep2pro.ethz.ch for easy access and comparison with other proteomics data.

### Keywords
Root, Proteomics, Post-harvest deterioration, Cassava

### Affiliations
ETH Zurich
Institute of  Plant Science

### Submitter
Katja Baerenfaller

### Lab Head
Dr Herve Vanderschuren
ETH Zurich


