### Accession
PXD031887

### Title
Whole cell proteomics analysis of Trypanosoma brucei 75kDa invariant surface glycoprotein knockout cells

### Description
A superfamily of invariant surface glycoproteins (ISGs) populates the African trypanosome surface, one of which, ISG75, is implicated in uptake of the century-old drug suramin. We disrupted the ISG75 gene locus encoding an array of six ISG75 paralogs on chromosome V by CRISPR/Cas9 knockout. Here we characterise this knockout strain by quantitative whole cell proteomics. Additionally, we analyse the way ISG75 knockout cells respond to suramin on the proteome level.

### Sample Protocol
Cells were collected by centrifugation (1000 x g), washed twice with PBS containing Complete Mini Protease Inhibitor Cocktail (Roche Applied Science) and resuspended in 3 x NuPage LDS buffer, 2.5 x Sample Reducing Agent (Thermo Fisher Scientific). Aliquots containing 5 x 106 cells were sonicated and run 1 cm into NuPAGE bis-tris 4â€“12% gradient polyacrylamide gels (Thermo Fisher Scientific) in MOPS-SDS buffer. Following IntstantBlue (Expedeon) staining each sample was excised in full, destained and subjected to tryptic digest and reductive alkylation. Samples were analyzed by liquid chromatography-tandem mass spectrometry (LC-MS2) performed by the Proteomic Facility at the University of Dundee on a Ultimate3000 nano rapid separation LC system (Dionex) coupled to a Q Exactive HF or Q Exactive Plus (suramin experiment) Hybrid Quadrupole-Orbitrap mass spectrometer (Thermo-Fisher Scientific). To assess parasite response to suramin, WT and ISG75Crc4 cells were challenged with 2 x EC50 concentrations of suramin for 48 hours (essentially as described in PMID: 32354742).

### Data Protocol
All sample mass spectra were analyzed using MaxQuant version 1.6 (PMID:19029910; PMID: 24942700) searching the T. brucei brucei 927 annotated protein database, release 39.0, supplemented with ISG75, ISG65, ISG64/65L, ISGLA and ISGLB sequences from the pre-release Lister strain 427 2018 assembly version  (PMID:19843604). Minimum peptide length was set at six amino acids, and false discovery rates of 0.01 were calculated at the levels of peptides, proteins, and modification sites based on the number of hits against the reversed sequence database. When the identified peptide sequence set of one protein contained the peptide set of another protein, these two proteins were assigned to the same protein group.  Sample layout: (result file ISG75.txt.rar) (1) Analysis of ISG75 CRISPR/Cas9 knockout clones (clone4 and clone4.6) (1.1) parental strain (three fractions (a,b,c) each): WT_1(a,b,c), WT_2 (a,b,c), WT_3 (a,b,c), WT _4 (a,b,c), WT_5 (a,b,c); (experiment 1, 2, 3, 4, 5) (1.2) ISG75 knockout clone 4 (three fractions (a,b,c) each) : ISG75Crc4_1(a,b,c), ISG75Crc4_2(a,b,c), ISG75Crc4_3(a,b,c), ISG75Crc4_4(a,b,c), ISG75Crc4_5(a,b,c); (experiment 6, 7, 8, 9, 10) (1.3) ISG75 knockout clone 4.6 (three fractions (a,b,c) each) : ISG75Crc4.6_1(a,b,c), ISG75Crc4.6_2(a,b,c), ISG75Crc4.6_3(a,b,c); (experiment 11, 12, 13) (2) Suramin treatment ISG75 knockout clone4 (result file suramin.txt.rar) (2.1) ISG75 knockout untreated: ISG75KO_NT1, ISG75KO_NT2, ISG75KO_NT3; (experiment 31, 32, 33) (2.2) ISG75 knockout suramin treated: ISG75KO_T1, ISG75KO_T2, ISG75KO_T3; (experiment 41, 42, 43) (2.3) WT untreated: WT_NT1, WT_NT2, WT_NT3, WT_NT4; (experiment 11, 12, 13, 14) (2.4) WT suramin treated: WT_T1, WT_T2, WT_T3; (experiment 21, 22, 23)

### Publication Abstract
The surface proteins of parasitic protozoa mediate functions essential to survival within a host, including nutrient accumulation, environmental sensing and immune evasion. Several receptors involved in nutrient uptake and defence from the innate immune response have been described in African trypanosomes and, together with antigenic variation, contribute towards persistence within vertebrate hosts. Significantly, a superfamily of invariant surface glycoproteins (ISGs) populates the trypanosome surface, one of which, ISG75, is implicated in uptake of the century-old drug suramin. By CRISPR/Cas9 knockout and biophysical analysis, we show here that ISG75 directly binds suramin and mediates uptake of additional naphthol-related compounds, making ISG75 a conduit for entry of at least one structural class of trypanocidal compounds. However, ISG75 null cells present only modest attenuation of suramin sensitivity, have unaltered viability i<i>n vivo</i> and <i>in vitro</i> and no alteration to suramin-invoked proteome responses. While ISG75 is demonstrated as a valid suramin cell entry pathway, we suggest the presence of additional mechanisms for suramin accumulation, further demonstrating the complexity of trypanosomatid drug interactions and potential for evolution of resistance.

### Keywords
Invariant surface glycoprotein, Suramin, Crispr/cas9, Drug metabolism, Xenobiotics, Drug accumulation, Trypanosome

### Affiliations
Division of Biological Chemistry & Drug Discovery
School of Life Sciences
University of Dundee
Dundee DD1 5EH
Biological Chemistry and Drug Discovery Wellcome Centre for Anti-Infectives Research (WCAIR) School of Life Sciences University of Dundee DUNDEE DD15EH UK

### Submitter
Martin Zoltner

### Lab Head
Dr Mark C. Field
Biological Chemistry and Drug Discovery Wellcome Centre for Anti-Infectives Research (WCAIR) School of Life Sciences University of Dundee DUNDEE DD15EH UK


