### Accession
PXD027636

### Title
Identification of human mRNA processing complexes bound to histone H2A.2 3’ untranslated region

### Description
Processing of 3’ untranslated region (3’-UTR) of precursor messenger RNAs (pre-mRNA) is a fundamental step in mRNA maturation, where the mRNA transcript is cleaved at a specific site, located downstream the open reading frame of the encoded gene. The position of the cleavage site is determined by cis-acting RNA sequence elements, which are recognized with high specificity by large 3’ end processing multiprotein complexes. Two main categories of metazoan 3’ pre-mRNA processing have been identified so far, describing elaboration of either canonical mRNAs – occurring during all phases of the cell cycle and containing a characteristic Poly-Adenylation Signal (PAS) element – or histone mRNAs – prevalently occurring during the S phase and containing a Histone Downstream Element (HDE). In this work, we isolate both human endogenous canonical and histone 3’ pre-mRNA processing complexes from human nuclear extracts by using a pre-mRNA containing the 3’UTR sequence of replication-dependent histone H2A.2. This H2A.2 pre-mRNA (H2A_4m), containing both an HDE sequence and a PAS element in a native overlaid position, was modified with a 5’ photocleavable biotin tag, and with modification of few bases to allow complex assembly and purification by affinity chromatography on streptavidin-agarose beads, elution by UV irradiation and subsequent mass spectrometric identification of bound 3’ mRNA processing factors.

### Sample Protocol
Nuclear extracts prepared from suspension cultures of human HEK293-F cells were used to prepare samples for 3’ pre-mRNA processing complex assembly and purification. All stock solutions of the synthetic RNA sequences used here were annealed in binding buffer (15 mM HEPES-KOH, pH 8.0, 15% glycerol, 75 mM KCl, 10 mM EDTA) by incubation at 90°C for 5 min and cooling on ice at 5°C. For complex isolation experiments, to each nuclear extract sample 80 mM EDTA pH 8 was added to a final concentration of 20 mM. For negative control samples, stem loop RNA and antisense U7 snRNA were pre-incubated with nuclear extracts, at a concentration of 10 µg/ml, for 20 minutes on ice. The synthetic RNA oligonucleotide H2A_4m was then added to nuclear extracts to a concentration of 40 nM. In case of experiments with synthetic U7 snRNA, a pre-annealed equimolar mixture of H2A_4m and U7 was used and added to nuclear extracts to a concentration of 40 nM. The RNA-nuclear extract mixtures were next incubated at 12°C for 10 minutes, cooled down on ice and centrifuged to remove precipitated material (10000 g, 10 minutes, 4°C). Streptavidin-agarose beads, previously equilibrated with binding buffer, were added and the resulting suspensions were placed on a rotating wheel for 1 hour at 4°C, in order to allow binding of the complexes onto the beads. These were subsequently recovered by low speed centrifugation (20g, 3 minutes, 4°C), washed 3 times with binding buffer and resuspended in 3 bead volumes of the same buffer. Bead suspensions were then transferred to new 0.5 ml tubes, placed on a Petri dish filled with ice, and the RNA-protein complexes were eluted by irradiation for a total of 30 minutes with a 100w UV lamp (Cole-Parmer) at 366 nm. After every five minutes of UV exposure, the tubes were vortexed and placed on a new ice bed to ensure uniform exposure and prevents overheating. Eluted pre-mRNA processing complexes were finally recovered by centrifugation (20g, 3 minutes, 4°C) and supernatant collection, and either used for SDS-PAGE and subsquent Western Blot analysis or MS analysis, or flash-frozen in liquid nitrogen and stored at -80°C for later use. Three biological replicates of complex purification using different RNA baits (H2A.2 RNA and H2A.2 + U7/Stem Loop RNAs for first experiment, and H2A.2 RNA, H2A.2 + antisense U7/Stem Loop RNAs and 38G/40A H2A.2 RNA for second experiment) were performed. The eluted proteins were solubilized in Laemmli buffer and stacked in the top of a 4-12% NuPAGE gel (Invitrogen). After staining with R-250 Coomassie Blue (Biorad), proteins were digested in-gel using trypsin (modified, sequencing purity, Promega). The resulting peptides were analyzed by online nanoliquid chromatography coupled to MS/MS (Ultimate 3000 RSLCnano and Q-Exactive HF, Thermo Fisher Scientific) using a 120 min gradient. For this purpose, the peptides were sampled on a precolumn (300 μm x 5 mm PepMap C18, Thermo Scientific) and separated in a 75 μm x 250 mm C18 column (Reprosil-Pur 120 C18-AQ, 1.9 μm, Dr. Maisch). The MS and MS/MS data were acquired by Xcalibur (Thermo Fisher Scientific).

### Data Protocol
Peptides and proteins were identified by Mascot (version 2.7.0, Matrix Science) through concomitant searches against the Uniprot database (Homo sapiens taxonomy, May 2021 download), a homemade classical database containing the sequences of classical contaminant proteins found in proteomic analyses (human keratins, trypsin, etc.), and the corresponding reversed databases. Trypsin/P was chosen as the enzyme and two missed cleavages were allowed. Precursor and fragment mass error tolerances were set at respectively at 10 and 20 ppm. Peptide modifications allowed during the search were: Carbamidomethyl (C, fixed), Acetyl (Protein N-term, variable) and Oxidation (M, variable). The Proline software was used for the compilation, grouping, and filtering of the results (conservation of rank 1 peptides, peptide length ≥ 6 amino acids, peptide score ≥ 25, allowing to reach a false discovery rate of peptide-spectrum-match identifications < 1% as calculated on peptide-spectrum-match scores by employing the reverse database strategy, and minimum of one specific peptide per identified protein group). Proline was then used to perform a compilation, grouping and MS1 label-free quantification of the identified protein groups based on razor and specific peptides.

### Publication Abstract
Human pre-mRNA processing relies on multi-subunit macromolecular&#xa0;complexes, which recognize specific RNA sequence elements essential for assembly and activity. Canonical pre-mRNA processing proceeds via the recognition of a polyadenylation signal (PAS) and a downstream sequence element (DSE), and produces polyadenylated mature mRNAs, while replication-dependent (RD)&#xa0;histone pre-mRNA processing&#xa0;requires association with a stem-loop (SL) motif and a histone downstream element (HDE), and produces cleaved but non-polyadenylated mature mRNAs. H2AC18 mRNA, a specific H2A RD histone pre-mRNA, can be processed to give either a non-polyadenylated mRNA, ending at the histone SL, or a polyadenylated mRNA. Here, we reveal how H2AC18 captures the two human pre-mRNA processing complexes in a mutually exclusive mode by overlapping a canonical PAS (AAUAAA) sequence element with a HDE. Disruption of the PAS sequence on H2AC18 pre-mRNA prevents recruitment of the canonical complex in vitro, without affecting the histone machinery. This shows how the relative position of cis-acting elements in histone pre-mRNAs allows the selective recruitment of distinct human pre-mRNA complexes, thereby expanding the capability to regulate 3' processing and polyadenylation.

### Keywords
3’-utr, Nanolc-ms/ms, Mrnas, 3’ pre-mrna processing complexes

### Affiliations
EDyP

### Submitter
Yohann Couté

### Lab Head
Dr Yohann Couté
EDyP


