### Accession
PXD004962

### Title
Quantitative Shotgun Proteomics Unveils Candidate Novel Oesophageal Adenocarcinoma-Specific Proteins

### Description
Selective markers are needed for earlier diagnosis, optimal staging and treatment of oesophageal adenocarcinoma.  This quantitative shotgun proteomic study used patient-matched fresh frozen tissue samples of OAC, normal oesophagus and normal stomach.

### Sample Protocol
Fresh frozen tissue samples were disrupted using a bead mill and then lysed using a modified FASP method.  Tryptic peptides were TMT-labelled and then samples from one patient pooled and 24 fractions generated by OFFGEL electrophoresis.  Fractions were then subjected to LC-MS/MS on an LTQ orbitrap.

### Data Protocol
Protein identifications were made using the Easyprot pipeline as follows. Thermo RAW files were converted to peak lists using ReAdW (ThermoFinnigan) and CID and HCD spectra were merged for simultaneous identification and quantification (25). Peaklist files were searched against the Uniprot human reference proteome (release 09/01/2013, containing 87,613 entries) using PhenyxÂ® (26) with a precursor ion tolerance of 10 parts per million. Variable peptide modifications included TMT-modified N-termini and lysines (additional 229.1629 Da) and oxidized methionines, with carbamidomethylation of cysteines set as a fixed modification. Trypsin was selected as the digestion enzyme, with one potential missed cleavage and a minimum of a single-tryptic terminus, a peptide length of 6 amino acids and a z-score of 4 were required. All datasets were searched separately, once using a forward and using a reversed peptide database. A peptide z-score threshold was then calculated to maintain the false positive peptide ratio below 1%. A single unique peptide was accepted for protein identification.  For protein quantitation, raw TMT reporter ion intensities were extracted from each peptide ID and then used to generate a mean protein expression ratio between tissue types.

### Publication Abstract
Esophageal cancer is the eighth most common cancer worldwide and the majority of patients have systemic disease at presentation. Esophageal adenocarcinoma (OAC), the predominant subtype in western countries, is largely resistant to current chemotherapy regimens. Selective markers are needed to enhance clinical staging and to allow targeted therapies yet there are minimal proteomic data on this cancer type. After histological review, lysates from OAC and matched normal esophageal and gastric samples from seven patients were subjected to LC MS/MS after tandem mass tag labeling and OFFGEL fractionation. Patient matched samples of OAC, normal esophagus, normal stomach, lymph node metastases and uninvolved lymph nodes were used from an additional 115 patients for verification of expression by immunohistochemistry (IHC).Over six thousand proteins were identified and quantified across samples. Quantitative reproducibility was excellent between technical replicates and a moderate correlation was seen across samples with the same histology. The quantitative accuracy was verified across the dynamic range for seven proteins by immunohistochemistry (IHC) on the originating tissues. Multiple novel tumor-specific candidates are proposed and EPCAM was verified by IHC.This shotgun proteomic study of OAC used a comparative quantitative approach to reveal proteins highly expressed in specific tissue types. Novel tumor-specific proteins are proposed and EPCAM was demonstrated to be specifically overexpressed in primary tumors and lymph node metastases compared with surrounding normal tissues. This candidate and others proposed in this study could be developed as tumor-specific targets for novel clinical staging and therapeutic approaches.

### Keywords
Tmt, Shotgun proteomics, Oesophageal adenocarcinoma

### Affiliations
University of Edinburgh
University of Geneva

### Submitter
Robert O'Neill

### Lab Head
Dr Alexander Scherl
University of Geneva


