### Accession
PXD007864

### Title
AP-MS analysis of CobB interactions in E. coli

### Description
Sirtuins are an essential family of nicotinamide adenine dinucleotide (NAD)-dependent enzymes, conserved from bacteria to humans.  Here we study the bacterial sirtuin CobB in E. coli.  To demonstrate its highly conserved enzymatic activities and substrates, we used proteomics to define CobB protein interactions in E. coli.

### Sample Protocol
The IP elutions were digested in-solution using a modified filter-assisted sample preparation (FASP) method. Digested peptides were desalted on Stage Tips using Empore SDB-RPS extraction discs. Desalted peptides (~2 ÂµL) were analyzed by nLC-MS/MS using a Dionex Ultimate 3000 nRSLC coupled directly to a Q Exactive mass spectrometer equipped with an EASY-Spray ion source (ThermoFisher Scientific). Peptides were separated by reverse phase chromatography over a linear 60 min acetonitrile (ACN) gradient from 4 to 35% mobile phase B at a flow rate of 250 nl/min on an EASY-Spray column (PepMap, 2 Âµm, 75 Âµm x 50 cm) with an integrated fused silica emitter. The mass spectrometer was operated in data-dependent acquisition mode with a single acquisition cycle comprising one full-scan mass spectrum in the orbitrap, followed by HCD fragmentation of the top 20 most intense precursor ions.

### Data Protocol
MS/MS spectra were extracted and searched using Proteome Discoverer (v.2.1)/SEQUEST (v.1.3) against an E. coli protein sequence database (UniProt-SwissProt, 2013_08) appended with common contaminants (4,385 total sequence entries), with the following parameters: full enzyme specificity; 2 missed cleavages; precursor mass tolerance: 10 ppm; fragment mass tolerance: 0.02 Da; static modification of carbamidomethylation of cysteine, and dynamic modifications of methionine oxidation, and phosphorylation of serine, threonine, and tyrosine residues. For estimation of false discovery rates, the UniProt protein database sequences were reversed and an independent database search was performed. Both forward and reverse SEQUEST peptide spectrum matches were analyzed by Scaffold (v4.4.8; Proteome Software) with a refinement search using the X!Tandem algorithm. Probabilities for peptide spectral matches were calculated using the LFDR algorithm in Scaffold. Protein identifications were filtered in Scaffold at a maximum 1% protein and peptide false discovery rate and a minimum of two unique peptides.

### Publication Abstract
Lipoic acid is an essential metabolic cofactor added as a posttranslational modification on several multimeric enzyme complexes. These protein complexes, evolutionarily conserved from bacteria to humans, are core regulators of cellular metabolism. While the multistep enzymatic process of adding lipoyl modifications has been well characterized in <i>Escherichia&#xa0;coli</i>, the enzyme required for the removal of these lipoyl moieties (i.e., a lipoamidase or delipoylase) has not yet been identified. Here, we describe our discovery of sirtuins as lipoamidases in bacteria and establish their conserved substrates. Specifically, by using a series of knockout, overexpression, biochemical, <i>in vitro</i>, proteomic, and functional assays, we determined the substrates of sirtuin CobB in <i>E.&#xa0;coli</i> as components of the pyruvate dehydrogenase (PDH), &#x3b1;-ketoglutarate dehydrogenase (KDH), and glycine cleavage (GCV) complexes. <i>In vitro</i> assays provided direct evidence for this specific CobB activity and its NAD<sup>+</sup> dependence, a signature of all sirtuins. By designing a targeted quantitative mass spectrometry method, we further measured sirtuin-dependent, site-specific lipoylation on these substrates. The biological significance of CobB-modulated lipoylation was next established by its inhibition of both PDH and KDH activities. By restricting the carbon sources available to <i>E.&#xa0;coli</i>, we demonstrated that CobB regulates PDH and KDH under several growth conditions. Additionally, we found that SrtN, the sirtuin homolog in Gram-positive <i>Bacillus&#xa0;subtilis</i>, can also act as a lipoamidase. By demonstrating the evolutionary conservation of lipoamidase activity across sirtuin homologs, along with the conservation of common substrates, this work emphasizes the significance of protein lipoylation in regulating central metabolic processes.<b>IMPORTANCE</b> Here, we demonstrate that sirtuin lipoamidase activity exists in both Gram-positive and Gram-negative bacteria and establishing its conservation from bacteria to humans. Specifically, we discovered that CobB and SrtN act as lipoamidases in <i>E.&#xa0;coli</i> and <i>B.&#xa0;subtilis</i>, respectively. Intriguingly, not only is this sirtuin enzymatic activity conserved, but also the lipoylated substrates and functions are conserved, as bacterial sirtuins negatively regulate the lipoylation levels and activities of PDH and KDH. Considering that PDH and KDH regulate two carbon entry points into the tricarboxylic acid cycle, our finding highlights lipoylation as a conserved molecular toggle that regulates central metabolic pathways. Indeed, our findings from tests in which we limited nutrient availability support this. Furthermore, this study illustrates how the integration of technologies from different disciplines provides avenues to uncover enzymatic activities at the core of cellular metabolism regulation.

### Keywords
Orbitrap, Protein interactions, Sirtuin

### Affiliations
Princeton University
Princeton University, Dept. of Molecular Biology, Princeton, NJ USA

### Submitter
Elizabeth Rowland

### Lab Head
Dr Ileana M. Cristea
Princeton University, Dept. of Molecular Biology, Princeton, NJ USA


