### Accession
PXD022007

### Title
PLCP activity profiling in the ryegrass apoplast

### Description
This dataset identified active PLCPs in the ryegrass proteome in uninfected plants.

### Sample Protocol
2.35 ml of apoplastic fluid was incubated for 4 h at RT in 50 mM sodium acetate pH 6, 10 mM DTT and 2 mM DCG-04 (Greenbaum et al., 2000). As a negative control, one set of samples was incubated with an equivalent amount of DMSO instead of DCG-04. The pulldown experiment was subsequently performed with tryptic on-bead digestion as described in Schulze Hüynck et al., 2019. LC-MS/MS analysis was performed with an UltiMate 3000 RSCL nano-HPLC system (Thermo) online coupled to an Impact II Q-TOF mass spectrometer (Bruker) via a CaptiveSpray ion source boosted with acetonitrile-saturated nitrogen gas stream. Peptides were loaded on a Acclaim PepMap100 C18 trap column (3 µm, 100 Å, 75 µm i.d. x 2 cm, Thermo) and separated on a Acclaim PepMap RSLC C18 column (2 µm, 100 Å, 75 µm i.d. x 50 cm, Thermo) with a 2h elution protocol that included an 80min separation gradient from 5% to 35% solvent B (solvent A: H2O, 0.1% FA, solvent B: ACN,0.1% FA) at a flow of 300 nL/minute at 60 °C. Line-mode MS spectra were acquired in mass range 200 – 1400 m/z with a Top14 method at 4 Hz sampling rate for MS1 spectra and an intensity-dependent acquisition rate of 5 to 20 Hz for MS2 spectra. The capillary voltage for the CaptiveSpray ion source was 1600V. Collision energies of 7 eV and 9 eV were applied in two equal steps with the ion transfer time set to 61 and 100 µs, respectively, during acquisition of each MS2 spectrum.

### Data Protocol
Peptides were identified by matching spectra against a combination of a custom Epichloë festucae database  (“EfFl1_Proteins_Annotated_2020-05.fasta” containing 7077 sequences, Aug. 2018  ),   a Lolium perenne database (lope_proteins.V1.0.fasta, 40068 entries, downloaded 18/03/2019, Byrne et al., 2015) and the sequences of maize and Arabidopsis PLCPs (“PLCPs_Ath+Maize.fasta”, 52 entries) using the Andromeda search engine integrated into the MaxQuant software package (version 1.6.0.16) with standard settings (Tyanova et al., 2016). Carbamidomethylation of cysteine (+ 56.0214 Da) was set as a fixed peptide modification. Oxidation of methionine (+ 15.9949 Da) and acetylation of protein N-termini (+ 42.0106 Da) were set as variable modifications. For the apoplast proteome sample, triplex dimethyl isotope labelling with light ((CH3)2, +28.0313 Da), medium ((CD2H)2,+32.0564 Da) and heavy (13CD3)2), (+36.0756 Da) dimethyl label at Lys residues and peptide N-termini was additionally considered. The “requantify” option was enabled and false discovery rates (FDR) for peptide sequence matches and protein identifications were set to < 0.01. Only proteins quantified in at least 2 of the 3 biological replicates were used for pairwise comparisons of each of the three conditions. Protein ratios were median-normalized within each replicate before assessing differential expression with a moderated t-test using the “limma” package for R (Ritchie et al., 2015). Proteins changing at least 50% in abundance (log2 fold change <-0.58 or > 0.58) supported by a moderated t-test p-value < 0.05 and were considered significantly changed in abundance.

### Publication Abstract
None

### Keywords
Epichloe, Ryegrass, Protease, Plcp, Abpp

### Affiliations
Forschungszentrum Juelich GmbH ZEA-3 Analytics Wilhelm-Johnen-Str. D-52425 Juelich Germany
Forschungszentrum Juelich
ZEA-3 Analytics
Wilhelm-Johnen-Str
D-52425 Juelich

### Submitter
Fatih Demir

### Lab Head
Dr Pitter Florian Huesgen
Forschungszentrum Juelich GmbH ZEA-3 Analytics Wilhelm-Johnen-Str. D-52425 Juelich Germany


