### Accession
PXD020589

### Title
An HPF1/PARP1-based chemical biology strategy for exploring ADP-ribosylation

### Description
Strategies for installing authentic ADP-ribosylation (ADPr) at desired positions are fundamental for creating the tools needed to explore this elusive PTM in essential cellular processes. Here we describe a phospho-guided chemoenzymatic approach based on the Ser-ADPr writer complex for rapid, scalable preparation of a panel of pure, precisely-modified peptides. Integrating this streamlined methodology with phage display technology, we have developed the first site-specific as well as broad-specificity antibodies to mono-ADPr. These recombinant antibodies have been selected and characterized using multiple ADP-ribosylated peptides and tested by immunoblotting and immunofluorescence for their ability to detect physiological ADPr events. By enabling mono-ADPr proteomics and poly-to-mono comparisons at the modification site level, we have revealed the prevalence of mono-ADPr upon DNA damage and illustrated its dependence on PARG and ARH3. These and future tools created on our versatile chemical biology/recombinant antibody platform have broad potential to elucidate ADPr signaling pathways in health and disease.

### Sample Protocol
For SILAC experiments on histones, purified histones were treated or not with 3 Î¼M recombinant PARG for 3 hr at 37 ÂºC. Reactions were stopped with the addition of 8 M Urea in 0.1 M Tris (pH 8.0). Light histones (K0) were mixed with the same amounts of heavy histones (K8) and transferred onto 10-kDa-cutoff Vivacon 500 flat filters and partially-digested as described in Leideker et al., 2016. Immunoprecipitation of endogenous PARP-1 was performed according to the manifactureâs protocol , with slight modifications: After the last washing step, PARP-1 nanotrap beads were eluted twice in 6 M guanidinium-hydrochloride, 200 mM TEAB (pH 9). Pooled elutions were diluted to 2 M guanidinium-hydrochloride, 200 mM TEAB (pH 9) and incubated with 2 ug of Lys-C overnight at room temperature. Digested peptides were desalted on C18 stagetips according to Rappsilber et al., with slight modifications: the stagetips were conditioned once with 100% methanol, equilibrated with 40% ACN 0.1% FA, washed twice with 100 mM ABC (pH 9) before sample loading, washed twice with 100 mM ABC (pH 9) after sample loading, then eluted with 40% ACN 0.1% FA. For PARG treatment, dried peptides were resuspended in PARP-1 buffer (50 mM Tris-HCl, pH 7.5, 50 mM NaCl, 1 mM MgCl2) supplemented with 3 Î¼M of recombinant PARG and incubated at 37 C for 3 hours. Untreated peptides were resuspended in PARP-1 buffer without recombinant PARG and incubated at 37 Â°C for 3 hours. After PARG treatment, light (R0K0) and heavy (R10K8) samples were mixed 1 to 1 and desalted with HpH stagetips. From the eluate, a small aliquote of 1/10 was taken, dried, and injected for mass spectrometry analysis. The remaining 9/10 was resuspended in 2 mM Mg Cl2, 100 mM TEAB (pH 8.5) and processed with boronic acid agarose beads. The boronate elution was dried overnight in a speedvac (Eppendorf), then processed by stagetipping before injection for MS analysis. For IP experiments with anti-mono-ADPr, AbD33205 was coupled to Dynabeads M-270 epoxy, then stored at 4 Â°C until use. Cellular lysates were diluted to a final concentration of 0.5% IGEPAL, 0.05% sodium deoxycholate, then incubated with dynabeads-coupled mono-ADPr antibody (AbD33205) on a rotator for 2 hours at 4 Â°C. Following incubation, the beads were washed seven times in PBS, and elution was carried out twice with 6M guanidinium-hydrochloride, 2 mM MgCl2, 200 mM TEAB (pH 8.5). The eluates were pooled and diluted to a final concentration of 2M guanidinium-hydrochloride, 2 mM MgCl2, 200 mM TEAB (pH 8.5), then digested for 4 hours at room temperature with Lys-C (Pierce) at ~ 1:100 enzyme-to-substrate ratio. Boronic acid agarose beads (Sigma-Aldrich) were washed once with 1% formic acid (FA), once with 1M ABC (adjusted to pH 9 with ammonium hydroxide), once with 100 mM ABC (adjusted to pH 9 with ammonium hydroxide), then incubated with digested peptides for 1h at room temperature on a roller. The suspension was transferred on top of a gravity flow column (Biorad), washed five times with 1 M ABC (pH 9), two times with 100 mM ABC (pH 9), then eluted three times with 1% FA. The elutates were dried to completion overnight on a speedvac. The dried peptides were desalted on C18 stagetips and the eluates were dried to completion on a speedvac, then resuspended with 0.1% FA and briefly stored until MS analysis. For mono-ADPr enrichment by AbD33204, total and nuclear proteins of wild-type U2OS cells were lysed in 6 M guanidinium-hydrochloride, 2.5 mM TCEP, 10 mM CAA and 100 mM Tris-HCl. The samples were boiled for 10 minutes at 95Â°C, sonicated for 10 cycles of 30 seconds on/off on a bioruptor (Diagenode) at 4 Â°C, then centrifuged at 20â000 g for 20 minutes. The samples were quantified with BCA assay (Pierce). Lysates were diluted with 20 mM Tris-HCl to a final guanidinium-hydrochloride concentration lower than 0.6 M. Afterwards proteins were digested with trypsin 1:200 enzyme-to-substrate ratio at 37Â°C overnight. Digested cell lysates were acidified, desalted on C18 Sep Pak Vac 3cc (500mg) cartridges and lyophilized. Peptides were resuspended in PTMScanÂ® IAP Buffer. The peptide solution was cleared by centrifugation and incubated with AbD3204 for 2 hrs on an end-to-end shaker. Afterwards Protein A Agarose Beads were added and incubated again for 1 h on an end-to-end shaker. The solutions were centrifuged at 2â000 g for 30 seconds. First beads were washed twice with PTMScanÂ® IAP Buffer and then three time with chilled HPLC water. Elution of peptides was carried out twice with 0.15 % TFA. The eluates were desalted by performing a C18 StageTip twice. The eluates were dried to completion on a speedvac, then resuspended with 0.1% FA for MS analysis.

### Data Protocol
For IP experiments with anti-mono-ADPr antibodies, raw files were analyzed with MaxQuant proteomics suite of algorithms (version 1.5.3.17), integrated with the search engine Andromeda. The data were searched against the human proteome database (downloaded 09.10.2015 from UniProt) with the following parameters. The maximum allowed mass deviation was set to 4.5 ppmfor precursor ions and 20 ppm for fragment ions; the minimum peptide length was set to 6 amino acids and the maximum number of missed cleavages was set to 5 with the maximum charge state 7. Variable modifications included oxidation (M), acetylation (Protein N-term and K), Amidation (C-term), ADP-ribosylation (S). The variable modification ADP-ribosylation allowed for neutral losses of adenine (m/z 136.0618); adenosine with loss of water (m/z 250.0935); AMP (m/z 348.0704); ADP (m/z 428.0367) and ADP-ribose (m/z 542.0684) (Hengel and Goodlett, 2012), with AMP listed first. FTMS top peaks per 100 Da were set to 20. For confident identification of ADP-ribosylation sites, we considered only those sites that have been previously reported in the most extensive proteome-wide study of ADPr (Hendriks et al., 2019). For SILAC experiments on histones, HCD data were collected and searched against a human histone database (generated from a human proteome database downloaded on 09.10.2015 from UniProt) with the parameters given above with the following changes: Multiplicity was set to 2, with Lys8 as the Heavy Label. Maximum labeled AAs were set to 7. Maximum missed cleavages were set to 6, and maximum charge was 7. The minimum peptide length was set to 6 amino acids. Variable modifications included acetylation (Protein N-term and K), methylation (KR) and ADP-ribosylation (S). For SILAC experiments of PARP-1 immunoprecipitation, HCD data were collected and searched against the human proteome database (downloaded 09.10.2015 from UniProt) with the parameters given above for histones SILAC experiments with the following changes: Lys8 and Arg10 as the Heavy Label.

### Publication Abstract
Strategies for installing authentic ADP-ribosylation (ADPr) at desired positions are fundamental for creating the tools needed to explore this elusive post-translational modification (PTM) in essential cellular processes. Here, we describe a phospho-guided chemoenzymatic approach based on the Ser-ADPr writer complex for rapid, scalable preparation of a panel of pure, precisely modified peptides. Integrating this methodology with phage display technology, we have developed site-specific as well as broad-specificity antibodies to mono-ADPr. These recombinant antibodies have been selected and characterized using multiple ADP-ribosylated peptides and tested by immunoblotting and immunofluorescence for their ability to detect physiological ADPr events. Mono-ADPr proteomics and poly-to-mono comparisons at the modification site level have revealed the prevalence of mono-ADPr upon DNA damage and illustrated its dependence on PARG and ARH3. These and future tools created on our versatile chemical biology-recombinant antibody platform have broad potential to elucidate ADPr signaling pathways in health and disease.

### Keywords
Lc-ms, Adp-ribosylation, Parp1, Fusion

### Affiliations
Max Planck Insitute for Biology of Ageing
Max-Planck Insitute for Biology of Ageing

### Submitter
Edoardo José Longarini

### Lab Head
Dr Ivan Matic
Max Planck Insitute for Biology of Ageing


