### Accession
PXD020970

### Title
Cellular proteins interacting with curved HIV-1 capsid lattices

### Description
Early events of HIV-1 lifecycle, such as post-entry virus trafficking, uncoating and nuclear import, remain poorly understood due to limited information about virus-host interactions. We used a multidisciplinary approach. First, a mass spectrometry (MS)-based proteomics methodology enabled us to identify cellular proteins that specifically bind to 3-dimensional curved hexameric capsid lattices closely mimicking the assembled HIV-1 core. We then employed virology, live cell imaging, CRISPR gene knockout, x-ray crystallography, and biochemistry to demonstrate that Sec24C (a component of the COPII complex) is the first example of a cytoplasmic protein that utilizes an FG containing motif to directly and specifically bind the curved hexameric HIV-1 capsid lattices.

### Sample Protocol
THP1 cell lysates were incubated with preassembled wild type HIV-1 capsid tubes, which closely mimic curved capsid hexameric lattices present in assembled HIV-1 cores. The bound proteins were subjected to SDS-PAGE separation followed by in-gel proteolysis by trypsin. The peptides were separated on a self-made C18 analytical column packed with 2.7 Âµm Cortecs particles and analyzed on an Orbitrap Fusion mass spectrometer (Thermo Fisher Scientific) coupled to an Easy-nLC 1200 system (Thermo Fisher Scientific) through a nanoelectrospray ion source.

### Data Protocol
MS/MS spectra were extracted from raw data files and converted into mgf files using a Proteome Discoverer Software (ver. 2.1.0.62). These .mgf files were then independently searched against human database using an in-house Mascot server (Version 2.6, Matrix Science). Mass tolerances were +/-10 ppm for MS peaks, and +/-25 ppm for MS/MS fragment ions. Trypsin specificity was used allowing for 1 missed cleavage. Methionine oxidation, protein N-terminal acetylation, and peptide N-terminal pyroglutamic acid formation were allowed as variable modifications. Scaffold (version 4.8, Proteome Software, Portland, OR, USA) was used to validate MS/MS based peptide and protein identifications. Peptide identifications were accepted if they could be established at greater than 95.0 % probability as specified by the Peptide Prophet algorithm. Protein identifications were accepted if they could be established at greater than 99.0 % probability and contained at least two identified unique peptides.

### Publication Abstract
None

### Keywords
Hiv-1, Human proteins, Proteomics, Capsid

### Affiliations
Mamuka Kvaratskhelia, University of Colorado School of Medicine, Division of Infectious Diseases, 12700 E. 19th Avenue, Research Complex 2, Aurora, CO 80045, USA
University of Colorado

### Submitter
Nikoloz Shkriabai

### Lab Head
Dr Mamuka Kvaratskhelia
Mamuka Kvaratskhelia, University of Colorado School of Medicine, Division of Infectious Diseases, 12700 E. 19th Avenue, Research Complex 2, Aurora, CO 80045, USA


