### Accession
PXD007830

### Title
Candida albicans proteomics_Biofilm without treatment

### Description
The LC-MS/MS approach was utilized to identify qualitative changes in protein expressed by planktonic and biofilm cells of C. albicans and with Aureobasidin treatment.

### Sample Protocol
The planktonic (cells dispersed from biofilm) and biofilm cells were stained with Trypan blue (Sigma, USA) at 1:1 ratio and counted using a haemacytometer. Approximately 2.7x108 (planktonic) and 1.7 x109 (biofilm) were used for extraction. Protein extraction was performed using the PROTEOSPIN detergent-free total protein isolation kit (Norgen, Canada). Briefly, the harvested cells were resuspended in 50Âµl lysis solution (provided in the kit), vortexed vigorously for 2 mins and centrifuged at 14, 000 x g for 2 mins. The samples were filtered through a column provided and protein eluted was treated with 10mM dithiothreitol (DTT; GE healthcare, UK) at 37 oC for 1 h and alkylated with 55 mM iodoacetamide (IAA; GE healthcare, UK, UK). The proteins in the sample were digested with 1:50 (trypsin: protein) of MS grade porcine trypsin (Calbiochem, Germany) at 37 oC overnight. The samples were desalted using a Pierce C-18 spin column (Thermo Scientific) or Zip-Tip C-18 Cartridge column and completely dried in a refrigerated CentriVap centrifugal vacuum concentrator (Labconco, USA) before mass spectrometry analysis.

### Data Protocol
De novo sequencing was conducted with PEAKS Studio 7.5 (Bioinformatics Solutions Inc., Canada) with default parameters, except that: i) parent mass error tolerance was 1.5 Da, ii) fragment mass error tolerance was 0.5 Da, iii) trypsin as digestion enzyme, iv) carbamidomethylation (+57.02 Da, C) as fixed modification, v) oxidation (+15.99 Da, M) as variable modification, vi) maximum variable post-translation modification allowed per peptide was three. Sequence homology search tool (SPIDER) of the PEAKS (v 7.5) software was used to detect all proteins in Candida biofilm and planktonic forms. The list of proteins derived from SPIDER software stand higher chances of detection. SPIDER will search the database for homologous peptides, and attempt to merge these into protein hits as well.

### Publication Abstract
Candidiasis is the most common fungal infection associated with high morbidity and mortality among immunocompromised patients. The ability to form biofilm is essential for Candida albicans pathogenesis and drug resistance. In this study, the planktonic cell and biofilm proteomes of C. albicans SC5314 strain analyzed using Liquid Chromatography-Mass Spectrometry (LC-MS) were compared. In total, 280 and 449 proteins are annotated from the planktonic cell and biofilm proteomes, respectively. The biofilm proteome demonstrated significantly higher proportion of proteins associated with the endomembrane system, mitochondrion and cytoplasm than planktonic proteome. Among proteins detected, 143 and 207 biological processes are annotated, of which, 38 and 102 are specific to the planktonic cell and biofilm proteomes, respectively, while 105 are common biological processes. The specific biological processes of C. albicans planktonic cell proteome are associated with cell polarity, energy metabolism and nucleotide (purine) metabolism, oxido-reduction coenzyme metabolic process, monosaccharide and amino acid (methionine) biosynthesis, regulation of anatomical structure morphogenesis and cell cycling, and single organism reproduction. Meanwhile, regulation of cellular macromolecule biosynthesis and metabolism, transcription and gene expression are major biological processes specifically associated with C. albicans biofilm proteome. Biosynthesis of leucine, isoleucine, and thiocysteine are highlighted as planktonic-related pathways, whereas folate metabolism, fatty acid metabolism and biosynthesis of amino acids (lysine, serine and glycine) are highlighted as biofilm-related pathways. In summary, LC-MS-based proteomic analysis reveals different adaptative strategies of C. albicans via specific biological and metabolic processes for planktonic cell and biofilm lifestyles. The mass spectrometry data are available via ProteomeXchange with identifiers PXD007830 (for biofilm proteome) and PXD007831 (for planktonic cell proteome).

### Keywords
Candida albicans, Planktonic culture, Biofilm, Aureobasidin

### Affiliations
National University of Singapore
Medical Microbiology, Faculty of Medicine, University of Malaya

### Submitter
Mun Fai Loke

### Lab Head
Dr Loke Mun Fai
Medical Microbiology, Faculty of Medicine, University of Malaya


