### Accession
PXD016963

### Title
Fast and highly efficient affinity enrichment of Azide-A-DSBSO cross-linked peptides.

### Description
Cross-linking mass spectrometry is an increasingly used, powerful technique to study protein-protein interactions or to provide structural information. Due to sub-stochiometric reaction efficiencies, cross-linked peptides are usually low abundant. This results in challenging data evaluation and the need for an effective enrichment. Here we describe an improved, easy to implement, one-step method to enrich azide-tagged, acid-cleavable disuccinimidyl bis-sulfoxide (DSBSO) cross-linked peptides using dibenzocyclooctyne (DBCO) coupled Sepharose������ beads. We probed this method using recombinant Cas9 and E. coli ribosome. For Cas9, the number of detectable cross-links was increased from ~100 before enrichment to 580 cross-links after enrichment. To mimic a cellular lysate, E. coli ribosome was spiked into a tryptic HEK background at a ratio of 1:2 ��������� 1:100. The number of detectable unique cross-links maintained high at ~100. The estimated enrichment efficiency was improved by factor 4 -5 (based on XL numbers) compared to enrichment via biotin and streptavidin. We were still able to detect cross-links from 0.25 ������g cross-linked E. coli ribosome in a background of 100 ������g tryptic HEK peptides, indicating a high enrichment sensitivity. In contrast to conventional enrichment techniques, like SEC, the time needed for preparation and MS measurement is significantly reduced. This robust, fast and selective enrichment method for azide-tagged linkers will contribute to map protein-protein interactions, investigate protein architectures in more depth and help to understand complex biological processes.

### Sample Protocol
Cross-linking of Cas9 or E. coli ribosome using DSBSO (1 mM, 1 h, room temperature). For ribosome samples, an additional denaturation step (Na-DOC, 1.5 %) and digestion of nucleotides using benzonase was performed. Reduction of proteins was perforemd using DTT (10  mM, 30 min, 56 ������C) and alkylation was done using IAA (20 mM, 30 min, room temperature, in the dark). Digestion was done sequentially using LysC (2 h, 37 ������C) and trypsin (16 h, 37 ������C) in a 1:50 (w/w) ratio each. Enrichment of resulting cross-linked peptides using DBCO or BARAC biotin method as described in the mansucript, folllowed by LC - MS/MS measurement

### Data Protocol
MS data were analyzed with the help of Thermo Proteome Discoverer (2.3.0.523). Peptide identification was performed by MS Amanda (2.3.0.12368). The peptide mass tolerance was set to ������5 ppm and the fragment mass tolerance to ������0.02 Da. Carbamidomethyl (+57.021 Da) at cysteine was set as static modification. Oxidation (+15.995 Da) at methionine was set as dynamic modification. The result was filtered to 1% FDR (false discovery rate) on peptide level using the Target Decoy PSM Validator integrated in Thermo Proteome Discoverer, PSM hits were additionally filtered for a minimum score of 150. Cross-links were identified either using XlinkX 2.225 as node within Proteome Discoverer (2.3.0.523) or using MeroX (2.0.0.6) as indicated. For XlinkX the cross-link modification DSBSO was defined as C11H16O6S2 with the following crosslink-fragments: Alkene C3H2O, thiol C8H12O4S2, sulfenic acid C8H14O5S2 and linker specificity towards lysine and N-terminal amino residues was set. Fixed carbamidomethylation of cysteine and variable oxidation of methionine residues were set as modifications. Standard settings were used, with a minimal XlinkX score of 40 and a minimal delta score of 4 and results were filtered at 5 % FDR at peptide level using the XlinkX validator node. To analyze data with MeroX, raw files were first converted to mgf format using MSConvertGUI (3.0.19085-a306312d7) without using any filter. Data analysis was performed using the following settings: C-terminal cleavage sites lysine and arginine with 3 missed cleavages, allowed peptide length: 5 to 30, as static modification acetamidation of cysteine and as variable modification oxidation of methionine was set. The cross-linker DSBSO was defined as follows: DSBSO with specificity towards lysine and N-termini, fragments at site 1 and 2: Alkene and Thiol, as given for XlinkX above. Additionally, the following diagnostic ions were set: C8H12NO, C8H15N2O, C8H12NOS, C8H15N2OS. Dead ends were allowed to react with H2O. Further settings: precursor precision 4 ppm, fragment ion precision 8 ppm, S/N ratio 1.5, precursor masses were corrected (max 3 isotope shifts). Prescore intensity 10 %, FDR cutoff 5 %, score cutoff -1, for analysis with large databases, the proteome wide mode with a minimum peptide score of 10 was used, otherwise RISEUP mode was used. Quantification of cross linked peptides was performed label free using apQuant (3.1.1.27312)30 within Proteome Discoverer 2.3. As database for Cas9 samples the sequence of S. Pyogenes Cas9 with fused HaloTag������ plus the full human Swiss-Prot (as of 13th March 2018, 20271 proteins) was used. For ribosome samples a shotgun database containing 171 proteins, generated earlier9, was used and in experiments spiked with tryptic HEK peptides the full human Swiss-Prot (as of 13th March 2018, 20271 proteins) was added

### Publication Abstract
Cross-linking mass spectrometry is an increasingly used, powerful technique to study protein-protein interactions or to provide structural information. Due to substochiometric reaction efficiencies, cross-linked peptides are usually low abundance. This results in challenging data evaluation and the need for an effective enrichment. Here we describe an improved, easy to implement, one-step method to enrich azide-tagged, acid-cleavable disuccinimidyl bis-sulfoxide (DSBSO) cross-linked peptides using dibenzocyclooctyne (DBCO) coupled Sepharose beads. We probed this method using recombinant Cas9 and <i>E.&#xa0;coli</i> ribosome. For Cas9, the number of detectable cross-links was increased from &#x223c;100 before enrichment to 580 cross-links after enrichment. To mimic a cellular lysate, <i>E.&#xa0;coli</i> ribosome was spiked into a tryptic HEK background at a ratio of 1:2-1:100. The number of detectable unique cross-links was maintained high at &#x223c;100. The estimated enrichment efficiency was improved by a factor of 4-5 (based on XL numbers) compared to enrichment via biotin and streptavidin. We were still able to detect cross-links from 0.25 &#x3bc;g cross-linked <i>E.&#xa0;coli</i> ribosomes in a background of 100 &#x3bc;g tryptic HEK peptides, indicating a high enrichment sensitivity. In contrast to conventional enrichment techniques, like SEC, the time needed for preparation and MS measurement is significantly reduced. This robust, fast, and selective enrichment method for azide-tagged linkers will contribute to mapping protein-protein interactions, investigating protein architectures in more depth, and helping to understand complex biological processes.

### Keywords
Cross-linking, Affinity enrichment, Click reaction, Dbco, Dsbso, Mass spectrometry

### Affiliations
IMP / University of Vienna
IMP, Campus Vienna Biocenter 1, 1030 Vienna, Austria

### Submitter
Manuel Matzinger

### Lab Head
Dr Karl Mechtler
IMP, Campus Vienna Biocenter 1, 1030 Vienna, Austria


