### Accession
PXD023646

### Title
Proteomics of biphenyl labelling study

### Description
protein based stable isotope probing was performed for identifying microorganisms actually responsible for biphenyl biodegradation in soil environment.

### Sample Protocol
Extraction and quantification of proteins from soil microcosms (3 ml) were carried out using NoviPure Soil Protein Kit (QIAGEN) and BCA Protein Assay Kit (QIAGEN), respectively. The extracted proteins were separated by sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE). For each sample, the corresponding SDS-PAGE gel lane was cut into two pieces. Protein in the gel pieces were reduced with dithionite, carbamylated with iodoacetamide and digested with trypsin as described by Jehmlich et al (2008). Peptides were purified using ZipTip C18-columns (Millipore) and analyzed by nano liquid chromatography tandem mass spectrometry (nLC-MS/MS) on a nanoAquity UPLC (Waters) coupled to an Orbitrap Fusion

### Data Protocol
The obtained MS/MS spectra were searched against the specific metagenome database (bulk assembly) by XTandem {Craig, 2004 #504} in OpenMS {RÃ¶st, 2016 #505} pipeline. Precursor and fragment mass tolerance were set to 5 ppm and 0.02 Da, respectively. Two tryptic miss cleavages were allowed; carbamidomethylation of cysteines were treated as fixed modification and oxidation of methionine as dynamic modification. Proteins were considered to be identified when at least two unique peptides were recovered with medium confidence (False discovery rate, FDR < 0.05).   Incorporation of 13C was determined for peptides at high confidence level (FDR < 0.01) using MetaProSIP {Sachsenberg, 2015 #506} with a mass window of 10 ppm. Centroid MS1 data, the metagenome database and feature data annotated with identification (generated by IDMapper in OpenMS) were fed as input for MetaProSIP

### Publication Abstract
Most microorganisms in the biosphere remain uncultured and poorly characterized. Although the surge in genome sequences has enabled insights into the genetic and metabolic properties of uncultured microorganisms, their physiology and ecological roles cannot be determined without direct probing of their activities in natural habitats. Here we employed an experimental framework coupling genome reconstruction and activity assays to characterize the largely uncultured microorganisms responsible for aerobic biodegradation of biphenyl as a proxy for a large class of environmental pollutants, polychlorinated biphenyls. We used <sup>13</sup>C-labeled biphenyl in contaminated soils and traced the flow of pollutant-derived carbon into active cells using single-cell analyses and protein-stable isotope probing. The detection of <sup>13</sup>C-enriched proteins linked biphenyl biodegradation to the uncultured Alphaproteobacteria clade UBA11222, which we found to host a distinctive biphenyl dioxygenase gene widely retrieved from contaminated environments. The same approach indicated the capacity of Azoarcus species to oxidize biphenyl and suggested similar metabolic abilities for species of Rugosibacter. Biphenyl oxidation would thus represent formerly unrecognized ecological functions of both genera. The quantitative role of these microorganisms in pollutant degradation was resolved using single-cell-based uptake measurements. Our strategy advances our understanding of microbially mediated biodegradation processes and has general application potential for elucidating the ecological roles of uncultured microorganisms in their natural habitats.

### Keywords
Soil, 13c-labelled peptides

### Affiliations
Helmholtz Centre for Environmental Research - UFZ

### Submitter
Songcan Chen

### Lab Head
Dr Florin Musat
Helmholtz Centre for Environmental Research - UFZ


