### Accession
PXD007183

### Title
Proteomics of Metronidazole Resistance in three seminal lines of Giardia duodenalis

### Description
Metronidazole (Mtz) is the frontline chemotherapeutic for a variety of anaerobic pathogens, including the gastroenterinal parasitic protist, Giardia duodenalis. However, treatment failure and clinical resistance is common and increasing. In Giardia, in vitro drug-resistant lines allow controlled experimental interrogation of resistance mechanisms in in vitro, isogenic cultures. These mechanisms centrally involve regulation of glycolytic and antioxidant oxidoreducatases, however this is inconsistent across isolate lines, likely incomplete, and further complicated by phenotypic plasticity in Giardia, which regains Mtz susceptibility after cessation of drug selection, or via passage through the life cycle. Global, quantitative approaches are required to reconstruct complex Mtz resistance phenotypes, resolve isolate variability, and explore unresolved hypotheses regarding post-translational modifications as the regulators of this phenotype. Our study addresses these gaps by comprehensively characterising proteomic changes in three seminal Mtz resistant lines compared to their isogenic, Mtz-susceptible, parental lines. Using quantitative proteomics, we have calculated differentially expressed proteins (DEPs) in MtzR lines, and probed via Western blot changes in protein acetylation, methylation, ubiquitination and phosphorylation post-translational modifications (PTMs). Finally, we have also performed the first controlled, longitudinal study

### Sample Protocol
Trophozoites were grown to confluence according to growth rate in t25 flasks (Falcon), followed by decanting of media and unattached and non-viable trophozoitesAdherent trophozoites were then harvested by chilling on wet ice before collection by centrifugation. Total protein was extracted from pelleted trophozoites using the TriPure reagent (Roche) according to the manufacturer’s instructions for protein extraction.  Protein pellets were solubilised in 2% SDS in 50mM Tris (pH 8.8) (Sigma Aldrich) before reduction in 5mM dithiothreitol followed by alkylation in 10mM iodoacetamide (with alkylation quenched with 5mM dithiothreitol). Proteins were precipitated via Methanol/Chloroform [29] and transferred to 8M Urea in 50mM Tris (pH 8.8) and protein concentration quantitated by BCA assay (Pierce). A two-stage digestion was performed with firstly Lys-C (Wako) overnight at 30°C (1µg enzyme to 100µg protein), followed with Trypsin (Promega) digestion 37°C (1µg enzyme for 100µg protein) for 6 hours. Samples were acidified to 1% trifluoroacetic acid and then desalted using solid phase extraction (SPE) with in-house tips packed with styrene divinyl benzene (3M Empore). Peptide extracts were dried by vacuum centrifuge and quantified via Micro BCA (Pierce).  For TMT labelling, a total of 35mg of peptides per channel were reconstituted in 200mM HEPES (pH 8) per label reaction. Samples were labelled across three TMT 10plex reactions (Thermo, San Jose, CA) using 0.14mg of each reagent. Samples were incubated with labels for 1 hour at room temperature, and then quenched with 5% hydroxylamine (Sigma Aldrich). Each of the 10 labelled samples for each of the three TMT tenplex were combined, dried by vacuum centrifuge, reconstituted in 1% formic acid, and desalted on a 200mg C18 SepPak (Waters, Massachusetts) prior to SCX fractionation as described previously [30]. A total of 10 pooled SCX fractions were desalted using SPE as before, dried down using a vacuum centrifuge and reconstituted in 1% formic acid for nanoflow liquid chromatography tandem mass spectrometry (NanoLC-MS/MS).

### Data Protocol
Raw data files produced in Xcalibur (Thermo Scientific) were processed in Proteome Discoverer V1.3 (Thermo Scientific) and searched using Mascot against the WB C6 (ATCC 50803) V5.0 genome release obtained from GiardiaDB.org [31]. Parameters and modifications were as follows: MS tolerance was set to ±10 ppm, MS/MS tolerance to 0.1 Da, one missed cleavage was allowed; static modifications were set for carbamidomethylation of cysteines, while variable modifications were set to TMT 10plex modification of peptide N-termini and lysine residues, methionine oxidation, and deamidation of asparagine and glutamine. Search results only included peptides below the Mascot significance threshold filter of p = 0.05. Protein grouping for homologous peptide identification was enabled such that protein identifications based of peptides with amino acid sequences equal to, or contained within the sequence of more than one protein, the two proteins were grouped together in a single protein group. The mass spectrometry raw data files, database search results and TMT labelling protein

### Publication Abstract
None

### Keywords
Proteomics, Tmt labelling, Giardia

### Affiliations
Population Health and Immunity, Walter and Eliza Hall Institute of Medical Research

### Submitter
Samantha Emery

### Lab Head
Dr Aaron Jex
Population Health and Immunity, Walter and Eliza Hall Institute of Medical Research


