### Accession
PXD016254

### Title
RBS-ID: Comprehensive identification of RNA-binding sites at single amino acid-resolution

### Description
We developed RBS-ID, which greatly simplifies the RNA moiety by chemical cleavage, reducing the complexity of MS/MS search space to accurately identify and localize RBS in peptides. RBS-ID comprehensively and robustly identifies RNA-binding sites at both proteome and single protein level.

### Sample Protocol
In RBS-ID, RBP and RNA are first in vivo UV-crosslinked. Upon tryptic digestion of RBPs, peptide-RNA conjugates are purified in an RNA-dependent manner such as silica bead based RNA extraction. Then, chemical cleavage is applied for complete digestion of RNAs crosslinked on peptides into homogenous monomers, greatly simplifying the RNA moiety. The resulting peptides are subjected to tandem mass analysis. Finally, the tandem mass spectra are searched against a highly simplified spectral searching space, and the RBS within peptides are directly localized with the definitive mass shift.

### Data Protocol
Peak list generation and file conversion was performed using RawConverter, as data dependent acquisition of charge states 2-7. MS-GF+ search was performed with mzXML files using default parameters except the following: tda 1, inst 3, addFeatures 1. Fixed modification considered was C2H3N1O1 (carbamidomethlyation) on Cys at any residue. Variable modifications considered were C7H9N1O5 (Uridine minus carbamidomethlyation) on Cys and C9H12N2O6 (Uridine) on remaining 19 amino acids at any residue. The Swiss-prot human database (May 2019) or the Uniprot E.coli BL21-DE3 database supplemented with spCas9 (Uniprot accession: Q99ZW2) sequence was used, and the decoy database was generated by MS-GF+. The output mzid file was converted to tsv file for post-processing. For each experiment, PSMs within Â±5 ppm window from the mean precursor isotopic error of PSMs with MS-GF+ Q-Value = 0 were collected. Then, RBS-containing PSMs below peptide-level FDR = 0.01 were taken. Them. PSMs where RBS was not specifically localized were discarded (including those localized to the C-terminus of peptides). Next, for each peptide, we collected PSMs of RBS with exclusive maximum spectral counts. RBS site locations within proteins were mapped to the proteome databases used above.

### Publication Abstract
RNA-binding sites (RBSs) can be identified by liquid chromatography and tandem mass spectrometry analyses of the protein-RNA conjugates created by crosslinking, but RBS mapping remains highly challenging due to the complexity of the formed RNA adducts. Here, we introduce RBS-ID, a method that uses hydrofluoride to fully cleave RNA into mono-nucleosides, thereby minimizing the search space to drastically enhance coverage and to reach single amino acid resolution. Moreover, the simple mono-nucleoside adducts offer a confident and quantitative measure of direct RNA-protein interaction. Using RBS-ID, we profiled ~2,000 human RBSs and probed Streptococcus pyogenes Cas9 to discover residues important for genome editing.

### Keywords
Human, Rna, Lc-ms/ms, Rbp. rbs

### Affiliations
School of Biological Sciences, Seoul National University, Republic of Korea  Center for RNA Research, Institute for Basic Science, Republic of Korea
Seoul National University

### Submitter
Jong Woo Bae

### Lab Head
Dr Jong-Seo Kim
School of Biological Sciences, Seoul National University, Republic of Korea  Center for RNA Research, Institute for Basic Science, Republic of Korea


