### Accession
PXD040621

### Title
The broccoli-derived antioxidant sulforaphane changes the growth of gastrointestinal microbiota, allowing for the production of anti-inflammatory metabolites.

### Description
Sulforaphane is a naturally occurring, potent antioxidant and anti-inflammatory compound, found in cruciferous plants such as broccoli. Recently there have been a large number of clinical trials assessing broccoli sprout extracts as sulforaphane-based therapies for conditions including fibrosis, cancer and preeclampsia. As sulforaphane is orally administered, there is also the potential for impact on the gut microbiome. Here, we have determined the effect of sulforaphane on the growth of 43 common human gastrointestinal bacterial commensals and pathogens, which represented the four main phyla found in the human gastrointestinal microbiome. The pathogenic Escherichia coli strain ECE2348/69 showed the most significant increases in growth in the presence of sulforaphane compared to control conditions. Proteomic analysis of this isolate showed that sulforaphane increased anaerobic respiration, whilst metabolomic profiling identified differentially produced metabolites involved in amino acid biosynthesis and known to decrease inflammation in human cells. Therefore, sulforaphane can increase growth of specific gastrointestinal bacterial isolates, correlating with increased production of anti-inflammatory metabolites, that may provide a novel mechanism for modulating inflammatory states in patients.

### Sample Protocol
For proteome extraction, 1 mL of an OD600=0.6 or 105 CFU/mL culture was pelleted by centrifugation at 4000 x g for 5 min. Pellets were washed in ice-cold 1x phosphate buffered saline (PBS) and snap frozen. Samples were resuspended in 100 ul of 4% sodium dodecyl sulphate (SDS) in 100mM Tris pH 8.5 by boiling for 10 min at 95Â°C. Samples were then quantified by bicinchoninic acid (BCA) assays and 50Î¼g of samples reduced/alkylated with 20mM tris(2-carboxyethyl)phosphine and 60mM iodoacetamide for 1 hour in the dark. Reduced/alkylated samples were then cleaned up using S-traps according to the manufacturerâ€™s instructions. Samples were digested overnight with 3Î¼g of trypsin (~1:16  protease/protein ratio) before digests were collected by centrifugation with washes of 100mM Tetraethylammonium bromide, followed by 0.2% formic acid followed by 0.2% formic acid / 50% acetonitrile . Samples were dried down and then further cleaned up using home-made C18 Stage tips (ref)  before being dried and then stored at âˆ’20Â°C.

### Data Protocol
Proteome digests were re-suspended in Buffer A* (2% acetonitrile, 0.01% trifluoroacetic acid) and 2Î¼g of each sample separated on a Dionex 3000 UPLC equipped with a PepMap100 C18 20 mm x 75Î¼m trap and a PepMap C18 500 mm x 75Î¼m analytical column (Thermo Fisher Scientific) coupled to a Orbitrap Q-Exactive plus mass spectrometer (Thermo Fisher Scientific). Samples were loaded onto the trap column for 5 minutes at 5 Î¼L/min using Buffer A (0.1% formic acid, 2% DMSO) and then separated over a 120 minute analytical gradient by altering the concentration of Buffer B (0.1% formic acid, 77.9% acetonitrile, 2% DMSO) from 2% to 23% B over 95 minutes, then from 23% B to 40% B over 10 minutes, from 40% to 80% over 5 minutes and then equilibrated with 2% Buffer B for 10 minutes. The Orbitrap Q-Exactive plus mass spectrometer was operated in a data-dependent manner automatically switching between the acquisition of a single Orbitrap MS scan (375-1800 m/z, maximal injection time of 50 ms, an Automated Gain Control (AGC) set to a maximum of 3*106 and a resolution of 70k) and up to 20 MS/MS HCD scans of precursors (Stepped NCE of 28;32;38%, a maximal injection time of 60 ms, a AGC of 2*105 and a resolution of 17.5k). Data files were searched using MaxQuant (v1.6.17.0) against the E. coli ECE2348/69 proteome with oxidation on Methionine as a variable modification. Searches were performed with the Match between Runs option end label free quantitation (LFQ) options enabled. The resulting outputs were processed within the Perseus (v1.6.0.7) analysis environment to remove reverse matches and common protein contaminates prior to further analysis. Protein sequences were matched to the E. coli MG1655-377GD proteome with PATRIC proteome comparison tool to enable the match of protein names to a reference strain. For label-free quantification (LFQ) comparisons biological replicates were grouped, proteins identified in less than three biological replicates of at least one condition removed and missing values imputed based on the observed total peptide intensities with a range of 0.3 Std dev and a downshift of 1.8 Std dev using Perseus. Differentially produced proteins were determined as those with log2 fold-change > 1 or <-1, and p<0.05.

### Publication Abstract
None

### Keywords
Sulforaphane, Escherichia coli strain ece2348/69

### Affiliations
University of Melbourne
Centre of Innate Immunity and Infectious Disease, Hudson Institute of Medical Research, Clayton, Victoria, 3168, Australia. Department of Molecular and Translational Sciences, Monash University, Clayton, Victoria, 3800, Australia.

### Submitter
Nichollas Scott

### Lab Head
Dr Emily Gulliver
Centre of Innate Immunity and Infectious Disease, Hudson Institute of Medical Research, Clayton, Victoria, 3168, Australia. Department of Molecular and Translational Sciences, Monash University, Clayton, Victoria, 3800, Australia.


