### Accession
PXD031803

### Title
Reduction of Phosphoribulokinase Amount and Re-Routing Metabolism in Chlamydomonas reinhardtii CP12 Mutants

### Description
The chloroplast protein CP12 is involved in the dark/light regulation of the Calvin-Benson-Bassham cycle, and in particular in the dark inhibition of two enzymes: glyceraldehyde-3-phosphate dehydrogenase (GAPDH) and phosphoribulokinase (PRK), but other functions related to stress have been proposed. We knocked out the unique CP12 gene to prevent its expression in Chlamydomonas reinhardtii (ΔCP12). The growth rates of both wild-type and ΔCP12 cells were nearly identical, as was the GAPDH protein abundance and activity in both cell lines. On the contrary, the abundance of PRK and its specific activity were significantly reduced in ΔCP12, as revealed by relative quantitative proteomics. Isolated PRK lost irreversibly its activity over-time in vitro, that was prevented in the presence of recombinant CP12 in a redox-independent manner. We have identified amino acid residues in the CP12 protein that are required for this new function preserving PRK activity. Numerous proteins involved in redox homeostasis and stress responses were more abundant and the expressions of various metabolic pathways were also increased or decreased in the absence of CP12. These results highlight CP12 as a moonlighting protein with additional functions beyond its well-known regulatory role in carbon metabolism.

### Sample Protocol
C. reinhardtii CC4349 (purchased from Chlamydomonas Resource Center, University of Minnesota, USA) was maintained in Tris-Acetic acid-Phosphate (TAP) medium supplemented with agar (15 g.L-1) and Hygromycin (30 µg.mL-1) for the transformant lines. These cells are cell-wall less widely used in C. reinhardtii field. On this strain that CRISPR-Cas9 method has been developed according to Chang et al (Enhanced Lipid Productivity in AGP Knockout Marine Microalga Tetraselmis Sp. Using a DNA-Free CRISPR-Cas9 RNP Method. 2020 Bioresour Technol, 303, 122932,doi:10.1016/j.biortech.2020.122932) and Song et al (The Generation of Metabolic Changes for the Production of High-Purity Zeaxanthin Mediated by CRISPR-Cas9 in Chlamydomonas reinhardtii. Microb Cell Fact 2020, 19, 220, doi:10.1186/s12934-020-01480-4). To generate knock-out ΔCP12-Cr mutants, CRISPR‑Cas9 RNP based genome editing method was used as described previously with a few modifications described in this paper. WT-Cr and ΔCP12-Cr cells were grown in 20 mL TAP medium (supplemented with 30 µg.mL-1 hygromycin for ΔCP12-Cr) under 50 µmol photon.m2.s-1 8.5 h dark / 15.5 h light at 23°C and 110 rpm agitation. A few mL was added in hermetically closed bottle of 500 mL with 200 mL MOPS culture medium to reach an OD at 680nm of 0.05. The MOPS culture medium was : 20 mM MOPS, 1 mM KPi, Beijering Salts (7.5 mM NH4Cl, 0.4 mM MgSO4 and 0.34 mM CaCl2) and Hutner’s trace elements at pH 7.4. Cultures were grown under continuous light, 120 µmol photon.m2.s-1, 110 rpm at 25°C and 2% CO2 were bubbled during growth via El-Flow Bronkhurst manometres. Concerning growth monitoring under dark/light conditions, a few mL were added from continuous light culture to reach an OD at 680 nm of 0,1. WT-Cr and ΔCP12-Cr were grown phototrophically under 50 µmol of photons.m2.s-1, 8.5 h dark / 15.5 h light, at 23°C, 110 rpm and 2% CO2. The culture density was monitored by absorbance at 680 nm, and the number of cells was counted using a Neubauer cell on a microscope Motic BA310E.   109 cells in the stationary phase of the growth were collected in biological triplicates, for the two conditions, the one where CP12 was not expressed (∆CP12-Cr cells) versus the control where it was (WT-Cr cells), then pelleted and kept in ice. Following the manufacturer Protifi’s protocol for digestion on S-Trap™ Micro Spin columns, we extracted the proteins, reduced them by DTT, alkylated them by iodoacetamide and digested them by trypsin/Lys C protease including 0.025% ProteaseMax Surfactant Trypsin Enhancer (v/v).  Tryptic peptide aliquots of 1 µg were used for LC-MSMS analysis on an ESI-Q-Exactive Plus mass spectrometer (Thermo Scientific) coupled to a nano liquid chromatography Ultimate 3000 Separation by LC was performed on an analytical C18 reversed phase EASY-Spray column (PepMaPTM RSLC, C18, 2µm, 100 Å, 75µm ID x 50 cm), using a three step-linear gradient from 2% to 30% of mobile phase B (0.1% v/v formic acid (FA)/ 80% v/v acetonitrile) in mobile phase A (0.1% v/v FA) for 90 min, a second step from 30% to 50% of B in A for 17 min. Mass spectral data were acquired using a Top 10 Data Dependent Acquisition mode with a dynamic exclusion of 30 s, in a 350-1900 m/z range for the full MS (resolution 70 000 at m/z 200), and MS/MS acquisition at resolution 17,500 at m/z 200. For peptide ionization in the nanosource EASY-Spray, voltage was set at 1.9 kV and the capillary temperature at 250 °C.

### Data Protocol
For quantitative proteomics (Figure 3 in this paper, Table S1 &S2 and  Figure S2) : Raw data for the biological triplicates, for the two conditions, the one where CP12 was not expressed (∆CP12-Cr cells) : file names 2021118_15__029_BMG_GdtL_dCP12_2 / 2021118_18__029_BMG_GdtL_dCP12_3/ 2021122_12__029_BMG_GdtL_dCP12_1_bis) versus the control where it was (WT-Cr cells) (file names : 2021118_03_029_BMG_GdtL_Wt1/ 2021118_06__029_BMG_GdtL_Wt2/ 2021118_09__029_BMG_GdtL_Wt3) were processed by the MaxQuant computational proteomics platform (version 1.6.5.0) used for protein identification and quantification (Cox et al. 2011, J. Proteome Res., 10 (4), 1794-1805) and the Perseus statistical platform (version 1.6.5.0) (Tyanova et al. 2016, Nature Methods13, 731–740) The database of C. reinhardtii v5.6 ex-tracted from Phytozome v13 (19526 entries) was used for the search, supplemented with a set of 245 frequently observed contaminants. The following criteria were used for peptide search: (i) trypsin cleavage authorized before proline with two missed cleavages allowed; (ii) monoisotopic precursor tolerance of 20 ppm in the first search used for recalibration, followed by 4.5 ppm for the main search and 0.5 Da for fragment ions from MS/MS; (iii) cysteine carbamidomethylation (+57.025) as a fixed modification and methionine oxida-tion (+15.999), serine, threonine and tyrosine phosphorylation (+79.996) and N-terminal acetylation (+42.011) as variable modifications; (iv) a maximum of five modifications per peptide allowed and (v) minimum peptide length was 7 amino acids and a maximum mass of 4,600 Da. Spectral alignment was performed and the “Match between runs”option has been enabled to allow the transfer of identifications between LC-MS/MS, based on the m/z and the retention time by using the default settings. The False-Discovery Rate (FDR) on the identification at the peptide and protein levels were set to 1% and de-termined by searching a reverse database. For protein quantification, unique and razor peptides were used, and considered irrespectively of their modification state mentionned above. The statistical analysis was carried out with the Perseus program. The LFQ normalized intensities were transformed by a base logarithm of 2 to obtain a normal distribution. A minimum of one peptide without any modification was considered to identify and quantify a protein. Quantifiable proteins (valid values) were defined as those detected in 66% of the total group (WT and ∆CP12): 2524 proteins were kept for the statistical test without replacing missing values. To determine whether a given detected protein was specifically differential, a two-sample t-test was applied using a permutation based FDR-controlled at 0.01 or at 0.05 (250 permutations). The results were illustrated in Volcano plot. When p value was adjusted using a scaling factor s0 to a value of 0.1, one hundred and thirty (130) proteins were highlighted with a relative abundance difference  statistically significant.

### Publication Abstract
The chloroplast protein CP12 is involved in the dark/light regulation of the Calvin-Benson-Bassham cycle, in particular, in the dark inhibition of two enzymes: glyceraldehyde-3-phosphate dehydrogenase (GAPDH) and phosphoribulokinase (PRK), but other functions related to stress have been proposed. We knocked out the unique <i>CP12</i> gene to prevent its expression in <i>Chlamydomonas reinhardtii</i> (&#x394;CP12). The growth rates of both wild-type and &#x394;CP12 cells were nearly identical, as was the GAPDH protein abundance and activity in both cell lines. On the contrary, the abundance of PRK and its specific activity were significantly reduced in &#x394;CP12, as revealed by relative quantitative proteomics. Isolated PRK lost irreversibly its activity over-time in vitro, which was prevented in the presence of recombinant CP12 in a redox-independent manner. We have identified amino acid residues in the CP12 protein that are required for this new function preserving PRK activity. Numerous proteins involved in redox homeostasis and stress responses were more abundant and the expressions of various metabolic pathways were also increased or decreased in the absence of CP12. These results highlight CP12 as a moonlighting protein with additional functions beyond its well-known regulatory role in carbon metabolism.

### Keywords
Quantitative proteomics, Enzymatic activity, Carbon allocation, Nitrogen uptake, Intrinsically disordered protein, Photosynthesis, Calvin benson bassham, Glyoxylate pathway, Cp12

### Affiliations
CNRS
Proteomic Platform of Institut de Microbiologie de la Méditerranée CNRS 3479

### Submitter
Regine Lebrun

### Lab Head
Dr Lebrun Regine
Proteomic Platform of Institut de Microbiologie de la Méditerranée CNRS 3479


