### Accession
PXD034997

### Title
The regulatory landscape of the yeast phosphoproteome – quantitative proteomics DIA data

### Description
We systematically exposed the yeast Saccharomyces cerevisiae to a wide array of 101 perturbations and generated a quantitative mass spectrometry-based atlas of early phosphoproteomic responses. Perturbations covered broad classes of environmental, chemical, and drug perturbations and systematically targeted diverse aspects of yeast cell biology. Importantly, most perturbations focused on capturing early signaling responses (5 min treatments), with the reasoning that functional phosphorylation of specific substrates occurs rapidly following stimulation, but promiscuous phosphorylation and confounding changes in transcript and protein abundances takes place more slowly. Large-scale proteomics and phosphoproteomics was enabled by our recently developed R2-P2 (Rapid-Robotic Phosphoproteomics) sample preparation workflow in combination with DIA measurements. In parallel to phosphoproteomic measurements we performed total proteome measurements on the same samples for 30 selected perturbations. This submission includes 126 quantitative DIA-MS raw files that were used for the total proteome analysis.

### Sample Protocol
Frozen cell pellets were resuspended in a lysis buffer composed of 8 M urea, 75 mM NaCl, and 50 mM HEPES pH 8. Cells were lysed by 4 cycles of bead beating (30-s beating, 1-min rest on ice) with zirconia/silica beads followed by clarification with centrifugation. Protein concentration of every lysate was measured by BCA assay and lysates adjusted to 1 mg protein per ml lysis buffer. Proteins were reduced with 5 mM dithiothreitol (DTT) for 30 min at 55°C and alkylated with 15 mM iodoacetamide in the dark for 15 min at room temperature. The alkylation reaction was quenched by incubating with additional 10 mM DTT for 15 min at room temperature. Lysates were stored at -80°C until further processing.  Lysates were scrambled across 96-deep well plates and biological replicates were blocked from being on the same 96-well plate. Each plate contained 4 samples containing the same pooled lysate to assess sample preparation reproducibility between 96-well plates. To purify proteins and perform tryptic digestions, 96-well plates were processed using the R2-P2 (Rapid-Robotic proteomics) protocol implemented on a KingFisher™ Flex (Thermo Fisher Scientific) magnetic particle processing robot as established by our group before (Leutert et al., Mol Sys Bio, 2019). Lyophilized peptide samples were dissolved in 4% formic acid, 3% acetonitrile and analyzed by nLC-MS/MS. Peptides were loaded onto a 100 μm ID × 3 cm precolumn packed with Reprosil C18 3 μm beads (Dr. Maisch GmbH), and separated by reverse-phase chromatography on a 100 μm ID × 35 cm analytical column packed with Reprosil C18 1.9 μm beads (Dr. Maisch GmbH) and housed into a column heater set at 50°C. DIA-MS measurements for total proteomic samples were performed on an Orbitrap Exploris 480 Mass Spectrometer (Thermo Fisher Scientific) equipped with an Easy1200 nanoLC system (Thermo Fisher Scientific). Peptides were separated by a 60-min effective gradient ranging from 6 to 30% acetonitrile in 0.125% formic acid. For proteomic measurements we acquired 30 × 24 m/z (covering 363-1095m/z) precursor isolation window MS/MS DIA spectra (30,000 resolution, AGC target 1e6, auto inject time, 27 NCE) using a staggered window pattern and optimized window placements. Precursor spectra (60,000 resolution, standard AGC target, auto inject time) were interspersed every 30 MS/MS spectra.

### Data Protocol
The  S. cerevisiae S288C reference protein fasta database containing the translations of all 6713 systematically named ORFs, except "Dubious" ORFs and pseudogenes created on 05/11/2015 by SGD (https://www.yeastgenome.org/) was used for all searches. For spectral library generation and spectral library searches of DIA data, Spectronaut v.15 (Biognosys) was used. A peptide spectral library was generated by searching DDA data covering a deep yeast phosphoproteome together with DIA data covering the quantitative proteomic measurements. Standard search parameters were used, including fixed modification of cysteine carbamidomethylation and variable modification of methionine oxidation. A PSM and peptide FDR cutoff of <0.01 was chosen. Standard spectral library search setting with following adjustment were chosen: decoy limit strategy was set to dynamic with a library size fraction of 0.1, but not less than 5000, a precursor FDR cutoff of <0.01 was enforced by choosing the data filtering setting “Qvalue”, no imputation or cross run normalization was performed.  Sample outliers due to obvious LC, MS or sample preparation problems as judged by LC, MS or sample quality control parameters were excluded.

### Publication Abstract
None

### Keywords
Signaling, Orbitrap exploris 480, Budding yeast, R2-p2, Stress response, Dia, Saccharomyces cerevisiae

### Affiliations
Department of Genome Sciences, University of Washington, Seattle, WA 98195, USA

### Submitter
Mario Leutert

### Lab Head
Dr Judit Villén
Department of Genome Sciences, University of Washington, Seattle, WA 98195, USA


