### Accession
PXD039434

### Title
Analysis of CDPK1 targets identifies a trafficking adaptor complex that regulates microneme exocytosis in Toxoplasma - FTS TurboID

### Description
Apicomplexan parasites use Ca2+-regulated exocytosis to secrete essential virulence factors from specialized organelles called micronemes. Ca2+-dependent protein kinases (CDPKs) are required for microneme exocytosis; however, the molecular events that regulate trafficking and fusion of micronemes with the plasma membrane remain unresolved. Here, we combine sub-minute resolution phosphoproteomics and bio-orthogonal labeling of kinase substrates in Toxoplasma gondii to identify 163 proteins phosphorylated in a CDPK1-dependent manner. In addition to known regulators of secretion, we identify uncharacterized targets with predicted functions across signaling, gene expression, trafficking, metabolism, and ion homeostasis. One of the CDPK1 targets is a putative HOOK activating adaptor. In other eukaryotes, HOOK homologs form the FHF complex with FTS and FHIP to activate dynein-mediated trafficking of endosomes along microtubules. We show the FHF complex is partially conserved in T. gondii, consisting of HOOK, an FTS homolog, and two parasite-specific proteins (TGGT1_306920 and TGGT1_316650). CDPK1 kinase activity and HOOK are required for the rapid apical trafficking of micronemes as parasites initiate motility. Moreover, parasites lacking HOOK or FTS display impaired microneme protein secretion, leading to a block in the invasion of host cells. Taken together, our work provides a comprehensive catalog of CDPK1 targets and reveals how vesicular trafficking has been tuned to support a parasitic lifestyle.

### Sample Protocol
FTS-TurboID and mNG-TurboID (cytosolic control) parasites were infected onto confluent HFFs in 15 cm dishes. After the parasites lysed the HFF monolayer (approximately 40 hrs post-infection), extracellular parasites were passed through 5 µm filters and washed twice with DMEM by pelleting at 1000 x g for 10 min. At least 1 x 108 Parasites were resuspended in 200 µL of DMEM. 200 µL of biotin (final concentration 500 µM) or vehicle DMSO in DMEM was mixed with parasites. Tubes were incubated in a water bath at 37°C for 5 min. Parasites were pelleted at 12,000 x g for 1 min at 4°C for a total of three 1 mL PBS washes. Parasites were resuspended in a RIPA NP-40 lysis buffer (10 mM Tris-HCl pH 7.5, 140 mM NaCl, 1% NP-40 IGEPAL CA 630, 0.1% sodium deoxycholate, 0.1% SDS, 1x HALT protease inhibitor, and 125 units/mL benzonase) at a parasite concentration of 5 x 108 parasites/mL. Parasites were lysed at room temperature for 15 min and stored at -20°C. Results are representative of three independent experiments. Streptavidin magnetic beads were washed three times with a RIPA NP-40 lysis buffer. Thawed lysates were spun at 16,000 x g for 5 min at 4°C and the supernatant was used as the pulldown input. 20 µL of streptavidin magnetic beads was used for 200 µL of lysates (1 x 108 parasites). Beads were incubated with lysate rotating for 1 hr. Beads received a series of 1 mL washes: twice with the RIPA NP-40 lysis buffer, once with 1 M KCl, once with 0.1M Na2CO3 (pH 11), once with 2 M urea in 10 mM Tris pH 8.0), and twice with RIPA NP-40 lysis buffer. Proteins were eluted by incubating beads with 2.5 mM biotin in S-trap sample buffer (5% SDS, 50 mM TEAB, pH 7.5) for 10 min at 95°C. Proteins were prepared for mass spectrometry using a modified version of the S-trap protocol (Protifi). Proteins in lysates were reduced with 5 mM TCEP for 10 min at 55°C and alkylated with 15 mM MMTS for 10 min at room temperature. The lysates were acidified to a final concentration of 1.2% v/v phosphoric acid. A 6X volume of S-trap binding buffer (90% methanol, 100 mM TEAB, pH 7.55) was mixed to each sample to precipitate proteins. The solution was loaded onto S-trap micro columns (Protifi) and spun at 4,000 x g for 1 min until all the solution had passed through the column. The columns were washed four times with 150 µL of S-trap binding buffer and centrifuged at 4,000 x g for 1 min between each wash. Proteins were digested on-column with 0.75 µg of trypsin (Promega) in 50 mM TEAB pH 8.5 overnight at 37°C in a humidified incubator. Digested peptides were eluted in three steps at 4,000 x g for 1 min: 40 µL of 50 mM TEAB, 40 µL of 0.2% formic acid, and 35 µL of 50% acetonitrile/0.2% formic acid. The peptide concentrations of eluted peptides were quantified using the Pierce Fluorometric Peptide Assay (Thermo Fisher Scientific) according to manufacturer’s instructions. The remaining samples were frozen in liquid nitrogen and lyophilized.

### Data Protocol
Lyophilized peptides were resuspended in 25 µL of 0.1% formic acid and were analyzed on an Exploris 480 Orbitrap mass spectrometer equipped with a FAIMS Pro source (Bekker-Jensen et al., 2020) connected to an EASY-nLC chromatography system using 0.1% formic acid as Buffer A and 80% acetonitrile/0.1% formic acid as Buffer B. Peptides were separated at 300 nL/min on a gradient of 2% B for 1 min, 2–25% B for 41 min, 25–40% B for 6 min, 40–100% B for 12 min, 100–2% B for 3 min, 2% B for 3 min, 2–98% B for 3 min, and 98% B for 3 min. The orbitrap and FAIMS were operated in positive ion mode with a positive ion voltage of 1800V; with an ion transfer tube temperature of 270°C; using a standard FAIMS resolution and compensation voltage of -50 and -65V, an inner and outer electrode temperature of 100°C with 4.5 mL/min carrier gas. Full scan spectra were acquired in profile mode at a resolution of 60,000, with a scan range of 350–1400 m/z, 300% AGC target, 25 msec maximum injection time, intensity threshold of 5 x 103, 2–6 charge state, dynamic exclusion of 20 sec, 15 data dependent scans (DDA Top 15), and mass tolerance of 10 ppm. MS2 spectra were generated with a HCD collision energy of 30 at a resolution of 15,000, first mass at 110 m/z, with an isolation window of 1.3 m/z, and a normalized AGC target of 200% with an automatically determined maximum injection time. Raw files were analyzed in Proteome Discoverer 2.4 (Thermo Fisher Scientific) to generate peak lists and protein and peptides identifications using Sequest HT (Thermo Fisher Scientific) and the ToxoDB release 49 GT1 protein database. The maximum missed cleavage sites for trypsin was limited to 2. The following modifications were included in the search: dynamic oxidation (+15.995 Da; M), dynamic phosphorylation (+79.966 Da; S,T,Y), dynamic biotinylation (+226.078 Da; K, N-terminus), dynamic acetylation (+42.011 Da; N-terminus), and static methylthio (+45.988 Da; C). Label free quantification of proteins was performed using summed abundances from unique peptides. Abundances were normalized on the total peptide amount. Pairwise ratios were calculated for protein abundances comparing strains and conditions. Significance values were derived from t-tests across three replicates and adjusted with Benjamini-Hochberg correction.

### Publication Abstract
None

### Keywords
T. gondii, Signaling, Lc-msms, Vesicular trafficking

### Affiliations
Associate Professor, Department of Biology, Massachusetts Institute of Technology and Whitehead Institute of Biomedical Research
Whitehead Institute for Biomedical Research
Department of Biology, Massachusetts Insitute of Technology

### Submitter
Alex Chan

### Lab Head
Dr Dr. Sebastian Lourido
Associate Professor, Department of Biology, Massachusetts Institute of Technology and Whitehead Institute of Biomedical Research


