### Accession
PXD040866

### Title
"Dissection of paracrine/autocrine interplay in lung tumor microenvironment mimicking cancer cell-monocyte co-culture models reveals proteins that promote inflammation and metastasis"

### Description
Tumor cell-monocyte interactions play crucial roles in shaping up the pro-tumorigenic phenotype and functional output of tumor-associated macrophages. Within the tumor microenvironment, such heterotypic cell‚Äìcell interactions are known to occur via secretory proteins. Secretory proteins establish a diabolic liaison between tumor cells and monocytes, leading to their recruitment, subsequent polarization and consequent tumor progression. We co-cultured model lung adenocarcinoma cell line A549 with model monocytes, THP-1 to delineate the interactions between them. The levels of prototypical pro-inflammatory cytokines like TNF-ùõº, IL-6 and anti-inflammatory cytokines like IL-10 were measured by ELISA. Migration, invasion and attachment independence of lung cancer cells was assessed by wound healing, transwell invasion and colony formation assays respectively. The status of EMT was evaluated by immunofluorescence. Identification of secretory proteins differentially expressed in monocultures and co-culture was carried out using SILAC LC‚ÄìMS/MS. Various insilico tools like Cytoscape, Reacfoam, CHAT and Kaplan‚ÄìMeier plotter were utilized for association studies, pathway analysis, functional classification, cancer hallmark relevance and predicting the prognostic potential of the candidate secretory proteins respectively. Co-culture of A549 and THP-1 cells in 1:10 ratio showed early release of prototypical pro-inflammatory cytokines TNF-ùõº and IL-6, however anti-inflammatory cytokine, IL-10 was observed to be released at the highest time point. The conditioned medium obtained from this co-culture ratio promoted the migration, invasion and colony formation as well as the EMT of A549 cells. Co-culturing of A549 with THP-1 cells modulated the secretion of proteins involved in cell proliferation, migration, invasion, EMT, inflammation, angiogenesis and inhibition of apoptosis. Among these proteins Versican, Tetranectin, IGFBP2, TUBB4B, C2 and IFI30 were found to correlate with the inflammatory and pro-metastatic milieu observed in our experimental setup.  Furthermore, dysregulated expression of these proteins was found to be associated with poor prognosis and negative disease outcomes in lung adenocarcinoma compared to other cancer types. Pharmacological interventions targeting these proteins may serve as useful therapeutic approaches in lung adenocarcinoma. In this study, we have demonstrated that the lung cancer cell-monocyte cross-talk modulates the secretion of IFI30, RNH1, CLEC3B, VCAN, IGFBP2, C2 and TUBB4B favoring tumor growth and metastasis.

### Sample Protocol
Before MS analysis, each sample was subjected to in-solution digestion. An equal proportion of protein was lyophilized per sample and resuspended in 100mM TEABC (pH = 8.0). After being reduced using 5mM dithiothreitol at 60‚Å∞C for 30 mins, the samples were alkylated at room temperature with 20mM iodoacetamide. Proteins were digested at 37¬∞C for 16 hours with an enzyme ‚Äì protein ratio of 1:20 (w/w) utilizing sequencing grade trypsin (modified sequencing grade; Promega, Madison, WI). Peptides were dried in a vacufuge concentrator after trypsin digestion, desalted with C18 Stage Tips, and kept at -80¬∞C till LC-MS/MS analysis could be performed.

### Data Protocol
Analysis of LC-MS/MS data Raw data generated from LC-MS/MS was investigated using proteome discoverer [PD] (Version 2.1.1) software (Thermo Fisher Scientific, Bremen, Germany) employing the Mascot and Sequest search engine algorithms when compared to the Human refseq-81 database. Carbamidomethylation at the cysteine residue was used as a static modification, whereas oxidation of methionine and SILAC (C6-arginine/lysine) were used as variable modifications in the database search variables. For both the precursor¬†and the fragment ions, a mass error of 10 ppm and 0.05 Da, respectively, was permitted. Trypsin, a¬†designated protease, and¬†a single missed cleavage was considered acceptable. For quantification, exclusively unique peptides were taken into account. Peptide Spectrum Match (PSM) entries with associated peak regions were exported from PD. For the co-culture setting, the PSM entry¬†was divided up into two independent¬†files: heavy and light, which corresponded to heavy-labelled cancer cells and light-labelled¬†monocytes, respectively. Peptide and protein lists were created using an in-house tool. For further analysis those proteins were further taken into consideration¬†that were present in at least two of the three replicates. Normalization of¬†proteins was achieved individually¬†between mono-culture and their respective co-culture (i.e., A549 monoculture and A549 co-culture; THP-1 monoculture and THP-1 co-culture) established on¬†the cumulative peak area of all the proteins. The ratio of total peak area in mono- and co-culture was used to determine the normalizing factor. Proteins ascertained from each cell line in a monoculture scenario were compared with those ascertained in a co-culture scenario (i.e., A549 monoculture versus A549 co-culture; THP-1 monoculture versus THP-1 co-culture). Using these two comparative¬†sets of data, three protein lists were¬†generated: i) Proteins that were exclusively found in A549 and THP-1 mono-cultures, respectively, ii) Proteins that were exclusively found in A549 and THP-1 co-cultures, respectively and iii) Proteins that were found in both mono- and co-culture scenarios. Fold change was computed for the third group of proteins by evaluating the area of each proteins in mono-¬†as well as¬†co-culture scenarios. Identified proteins that have¬†at least two unique peptides and four PSMs with a %CV of ‚â§30¬†are documented.

### Publication Abstract
None

### Keywords
Thp-1, Lung cancer, A549, Silac

### Affiliations
Head, Department of Biotechnology, University of Kashmir
Department of Biotechnology, School of Biological Sciences, University of Kashmir 

### Submitter
Aabid Koul

### Lab Head
Dr Prof. Raies Ahmad Qadri
Head, Department of Biotechnology, University of Kashmir


