### Accession
PXD000464

### Title
LC-MS data of peptides from p53 isolated by custom immunoaffinity chromatography

### Description
We exploited a primary human fibroblast system in which the p53 tumor suppressor protein accumulates to high concentrations due to infection by an adenovirus type 5 mutant that lacks the E1B 55kDa E3 ubiquitin ligase, which targets p53 for degradation, to examine select post-translational modifications (PTMs) present on this transcriptionally inactivated endogenous human p53, as well as on p53 activated in response to etoposide treatment. These forms of p53 were isolated from whole cell lysates by immunoaffinity chromatography using a cocktail of monoclonal antibodies, followed by SDS-PAGE, and peptides derived from these species were subjected to nano-ultra-high performance liquid chromatography-MS and MS/MS analyses on high resolution Orbitrap platforms. Proteomics informatics: Raw LC-MS/MS data were processed into mgf-format peaklist files and searched using Mascot (v. 2.2.7) against the human p53 sequence determined by direct sequencing of the p53 gene from cDNA derived from the specific batch of HFFs used in this study (identical to the consensus sequence for UniProt P04637), employing a window of 8 ppm for the precursor and 1.2 Da for the fragment ion species and allowing for <= 3 missed trypsin and semi-trypsin cleavages, carbamidomethylation of cysteines as a fixed modification, with methionine oxidation, N-terminal protein acetylation, phosphorylation of Ser and Thr, acetylation of Lys, methylation, dimethylation, and trimethylation of Lys and Arg, diglycine modification of Lys (ubiquitinylation), and deamidation of Asn and Gln, iteratively as variable modifications. Aggregate search results for each sample were collated and consolidated using Scaffold (vs. 4.0.7) according to the PeptideProphet and ProteinProphet parsimony algorithms implemented therein, and filtered to the 90% peptide confidence level and to a precursor mass accuracy within 2-3 ppm.

### Sample Protocol
Not available

### Data Protocol
Not available

### Publication Abstract
The p53 tumor suppressor protein accumulates to very high concentrations in normal human fibroblasts infected by adenovirus type 5 mutants that cannot direct assembly of the viral E1B 55-kDa protein-containing E3 ubiquitin ligase that targets p53 for degradation. Despite high concentrations of nuclear p53, the p53 transcriptional program is not induced in these infected cells. We exploited this system to examine select post-translational modifications (PTMs) present on a transcriptionally inert population of endogenous human p53, as well as on p53 activated in response to etoposide treatment of normal human fibroblasts. These forms of p53 were purified from whole cell lysates by means of immunoaffinity chromatography and SDS-PAGE, and peptides derived from them were subjected to nano-ultra-high-performance LC-MS and MS/MS analyses on a high-resolution accurate-mass MS platform (data available via ProteomeXchange, PXD000464). We identified an unexpectedly large number of PTMs, comprising phosphorylation of Ser and Thr residues, methylation of Arg residues, and acetylation, ubiquitinylation, and methylation of Lys residues-for example, some 150 previously undescribed modifications of p53 isolated from infected cells. These modifications were distributed across all functional domains of both forms of the endogenous human p53 protein, as well as those of an orthologous population of p53 isolated from COS-1 cells. Despite the differences in activity, including greater in vitro sequence-specific DNA binding activity exhibited by p53 isolated from etoposide-treated cells, few differences were observed in the location, nature, or relative frequencies of PTMs on the two populations of human p53. Indeed, the wealth of PTMs that we have identified is consistent with a far greater degree of complex, combinatorial regulation of p53 by PTM than previously anticipated.

### Keywords
P53 post-translational modification

### Affiliations
Molecular Biology

### Submitter
David Perlman

### Lab Head


