### Accession
PXD006204

### Title
Determining qualitative and quantitative differences in the protein, mRNA and miRNA content of myeloid-derived suppressor cells and their released exosomes

### Description
Myeloid-derived suppressor cells (MDSCs) are immature myeloid cells that accumulate in the tumor microenvironment of most cancer patients. There MDSCs suppress both adaptive and innate immune responses, hindering immunotherapies. Moreover, many cancers are accompanied by inflammation, a processes that further intensifies MDSC suppressive activity, causing aggressive tumor progression and metastasis. MDSCs collected from tumor-bearing mice profusely release nano-scale membrane-bound extracellular vesicles, called exosomes, which carry biologically active proteins between cells and contribute directly to the immune suppressive functions of MDSC. Many studies on other cell types have shown that exosomes may also carry microRNAs (miRNAs) and messenger RNAs (mRNAs) which can also be transferred to surrounding and distant cells. However, to the best of our knowledge, the miRNA and mRNA cargo of MDSC-derived exosomes has not yet been interrogated. This study aims to identify and quantify the cargo of MDSC and their immunosuppressive exosomes to gather knowledge that can offer insights on the mechanisms by which MDSCs contribute to immune suppression, focusing on the role of exosomes as intercellular communication mediators in the tumor microenvironment. In order to achieve our objective a well-established mouse model based on a conventional mammary carcinoma (4T1 cells) and heightened inflammation (4T1 transduced to express the cytokine interleukin-1b) was used. We provide evidence that MDSC-derived exosomes carry proteins, mRNAs and miRNAs. Relative quantitation demonstrated quantitative differences between the exosome cargo and the cargo of their parental cells, supporting the hypothesis that selective loading into the exosomes is possible. Additionally, quantitative and functional analyses of the exosome cargo generated under conventional and heightened inflammation conditions are consistent with clinical observations that inflammation is linked to cancer development.

### Sample Protocol
"BALB/c mice were injected with 7000 wild-type syngeneic 4T1 mammary carcinoma cells or 4T1 cells transduced to constitutively express the cytokine IL-1? (4?1/IL?1?).(DOI: 10.1074/mcp.M110.002980) MDSCs were harvested from blood of 1 to 3 mice and MDSC populations confirmed to be > 90% Gr1+ CD11b+ by flow cytometry (1×107- 4×108 cells) were used for further analyses. An aliquot equivalent to 9×106- 3.6×108 cells were used for exosome release, isolation and characterization as previously reported by Burke et al.(DOI:10.1021/pr400879c) MDSC and MDSC-derived exosomes from 4T1 and 4T1/IL-1? tumor-bearing mice are termed ?conventional? and ?inflammatory?, respectively. MDSCs were thoroughly washed by centrifugation at 900 × g for 10 min at 4°C using 10 mL of cold phosphate buffered saline solution from Sigma-Aldrich. MDSC pellets were lysed by incubation in 8M urea in 50 mM ammonium bicarbonate buffer with protease inhibitor cocktail for 30 min at room temperature. In order to ensure that cells are completely lysed and the lysate is homogeneous, MDSCs were further lysed mechanically using a set of syringes with needles of sequentially smaller gauge size (18, 20, 21.5) obtained from Becton Dickinson & Co. and performing 5-10 strokes each time. Cell debris were removed by centrifugation at 14,000 × g for 10 min, supernatants transferred to 3 KDa filters and buffer exchanged to reach an 8 mM urea concentration. The protein content of MDSC was measured using the Pierce BCA Protein Assay Kit (Thermo Fisher Scientific). Exosomes lysates were prepared following Burke et al. For both MDSC and MDSC-derived exosome lysates, 25 ?g aliquots of lysate were reduced, alkylated and tryptic digested as reported in Burke et al. All digested lysates were lyophilized and reconstituted in solvent A (97.5:2.5 H2O-acetonitrile in 0.1% formic acid) prior to LC-MS/MS. Lysates were analysed using an Ultimate 3000 RSLCnano system (Dionex) coupled to an orbitrap Fusion Lumos Tribrid (Thermo Fisher Scientific). Chromatographic separation was obtained using a C18 PepMap trap (0.3 × 5 mm, 5 ?m particle size, 100 Å) and C18 Acclaim PepMap RSLC column (0.075 × 250 mm, 2?m particle size, 100 Å), both obtained from Thermo Fisher Scientific. Samples were loaded into the trap, desalted and concentrated using Solvent A at a flow rate of 5 ?l/min for 10 min, and then eluted from the trap and further separated in the analytical column by linearly increasing Solvent B (acetonitrile-H2O, (75:25) in 0.1% formic acid) from 5 to 55%, during the span of 150 min under a flow rate of 0.3 ?L/min.  Precursor ion mass resolution of 120,000 (at m/z 200) and product ion unit mass resolution were used. Ions were collected based on a target automatic gain control of 4×105 and 4×103 for precursor and product ions, respectively. The maximum injection times for precursor and product ions were set to 50 ms and 100 ms, respectively. Data dependent acquisition was carried out in a fixed 3 second duty cycle, in which the top n most abundant precursor ions (intensity > 1×104) carrying charges from +2 to +7 were isolated by the quadrupole within a 1.6 m/z isolation window. Dynamic exclusion was set to exclude precursor ions for 60s after being selected once in 30s. The isolated precursor ions were fragmented using CID with helium set to 35% normalized collision energy. "

### Data Protocol
A total of 60 data files were converted into peak lists and formated as mzXML files using msconvert from ProteoWizard (DOI: 10.1093/bioinformatics/btn323) and searched against the Uniprot Mus musculus reference protein sequence database including isoforms (XX entries, January 2015) using three search engines (MSGF+, X!Tandem and OMSSA) in PepArML.(DOI: 10.1002/0471250953.bi1323s44) Search parameters allowed for up to 1 missed cleavage and, a precursor (monoisotopic or first 13C peak) and product ion mass tolerance of 0.05 and 0.5 Da, respectively. Carbamidomethylation was considered as a fixed modification and, oxidation of methionine, deamination of N-terminus glutamine, dehydration of N-terminus glutamic acid and pyro-carbamidomethylation of N-terminus cysteine as variable modifications. Spectral FDR were estimated in PepArML following Elias and Gygi.(DOI: 10.1007/978-1-60761-444-9_5) Identified peptides were filtered by spectral FDR ? 0.31% and global parsimony was applied. Proteins were inferred based on the presence of at least two unshared peptides in at least one of the four sample types. The resulting protein FDR was estimated to be ? 1% in all cases using MAYU.(DOI: 10.1074/mcp.M900317-MCP200) Label-free relative quantitation was performed by spectral counting. An in-house software was used to provide the number of peptide spectrum matches (PSM) or spectral counts for each inferred protein based on PepArML search results, after spectral FDR filtering. Differences in abundance between conditions were estimated following Old et al.(DOI: 10.1074/mcp.M500084-MCP200) Ratios from spectral counts (Rsc) were estimated for each identified protein and statistically significant differences in abundance were determined using the Fisher?s Exact test with Benjamini-Hochberg adjustment for multiple testing.

### Publication Abstract
Myeloid-derived suppressor cells (MDSC) are immature myeloid cells that accumulate in the circulation and the tumor microenvironment of most cancer patients. There, MDSC suppress both adaptive and innate immunity, hindering immunotherapies. The inflammatory milieu often present in cancers facilitates MDSC suppressive activity, causing aggressive tumor progression and metastasis. MDSC from tumor-bearing mice release exosomes, which carry biologically active proteins and mediate some of the immunosuppressive functions characteristic of MDSC. Studies on other cell types have shown that exosomes may also carry RNAs which can be transferred to local and distant cells, yet the mRNA and microRNA cargo of MDSC-derived exosomes has not been studied to date. Here, the cargo of MDSC and their exosomes was interrogated with the goal of identifying and characterizing molecules that may facilitate MDSC suppressive potency. Because inflammation is an established driving force for MDSC suppressive activity, we used the well-established 4T1 mouse mammary carcinoma system, which includes "conventional" as well as "inflammatory" MDSC. We provide evidence that MDSC-derived exosomes carry proteins, mRNAs, and microRNAs with different quantitative profiles than those of their parental cells. Several of these molecules have known or predicted functions consistent with MDSC suppressive activity, suggesting a potential mechanistic redundancy.

### Keywords
Myeloid-derived suppressor cells, Spectral counting, Lc-ms/ms, Exosomes

### Affiliations
University of Maryland, College Park
Georgetown University

### Submitter
Nathan Edwards

### Lab Head
Dr Catherine Fenselau, Ph.D.
University of Maryland, College Park


