### Accession
PXD008714

### Title
Proteomics of CLIC4 overexpressing human pulmonary artery endothelial cells

### Description
To identify CLIC4 effectors  by studying proteins expressiosn altered by CLIC4 overexpression in human pulmonary artery endothelial cells.

### Sample Protocol
Preparation of samples for LC-MS/MS analysis. Cell homogenates or fraction containing purified proteins were dissolved in 300µL of 9M urea and reduced by addition of 50mmol/L-dithiothreitol prior to heating at 100°C for 2 min and then carbamidomethylated by addition of mmol/L iodoacetamide in the dark for 30 min. SDS-PAGE was performed on 10 µg of each sample using 10% NuPAGE Novex bis-tris gels and reagents (Invitrogen). Gels were stained with InstantBlue® and each sample-containing lane were cut into a series of regions based on the position of molecular weight (MW) markers (SeaBlue Marker®, Invitrogen) and the distribution of proteins observed in the gel. Gel pieces were then washed with 1 mL water and dehydrated in 1 mL acetonitrile  for 30 min. Proteins contained within each gel slice were digested in 0.3 mL 50mmol/L ammonium bicarbonate containing 1 ng/mL trypsin (sequencing grade, Promega) at 37C for 18 hrs. On the following day, tryptic peptides were extracted by addition of 0.3 mL 0.1% formic acid in 2% acetonitrile and mixed for 30 min, 0.42 mL was removed, and dried. LC-MS/MS analysis method. The dried samples were reconstituted in 30µL of 0.1% TFA and 8µL of the solution was injected onto a C18 trap-column (ProteCol, 0.3 x 10mm, 300Å; SGE Analytical Science) followed by reverse phase separation on a C18 column (PicoFrit, 75μm ID × 10 cm ProteoPrep column, New Objective) using Agilent 1200 LC series (Agilent Technologies UK Ltd.). The mobile phases were 0.1% formic acid in water (buffer A) and 0.1% formic acid in acetonitrile (buffer B). A linear gradient of 0-54% buffer B was introduced at a flow rate of 300nL/min for 48 min.  The peptides eluting from the reversed phase column were analysed on-line using a Thermo LTQ linear ion trap MS equipped with a dynamic nano-spray interface. The MS method consisted of a cycle combining one full MS scan (m/z 400-1600) with three data dependent MS/MS events (35% collision energy at each event) for 70 minutes. Dynamic activation time was set at 30 s with scans at every 2 s.

### Data Protocol
MS data analysis. MS data was assessed on the basis of ion intensity of peptide ions with coincident LC retention time and m/z values. These data were displayed as an intensity map by Progenesis LC-MS software (Nonlinear Dynamics). The software aligns data based on the LC retention time of each analysis to create a single aggregate run containing all of the MS data with those representing peptide ions indicated. Feature outline maps were then generated and these outlines were used to facilitate detection and quantification of peptide ions from individual analyses. The peptide quantitation algorithm in the software reports peptide abundance as the sum of the peak areas (pixel intensities) within its isotope boundaries. Each abundance value is then transformed to a ‘normalised’ abundance by applying a global scaling factor calculated from distribution factors of all peptide ratios in the analysis. This corrects for experimental/technical variation and ensures that up-regulation and down-regulation has the same weight. The distribution is calculated using log(ratio) values and as log(1) = 0.0 and can be modelled as follows: y'i = αkyi . Protein abundances for individual sample are then calculated as the sum of all component peptide abundances.  The identity of peptides was determined by exporting the MS/MS output files from the matched analysis into the Turbo SEQUEST search engine (BioWorks Browser 3.3; Thermo Electron Corporation) and interrogating it against the NCBI human RefSeq database release 51. The search parameters were as follows: peptide tolerance was set to 2 atomic mass unit, MS/MS tolerance was set at 1 atomic mass unit, charge states allowed is from +1 to +3 and up to two missed cleavages were allowed. Modifications were set to allow for the detection S-carbamidomethylated cysteine (+57 Da). Each MS2 spectra was matched to only a single peptide with the highest scoring SEQUEST peptide hit. Cross-correlating (XCorr) MS data, with values >1.5, >2.0 and >2.5 for singly, doubly and triply charged ions respectively, were used to select peptides of high quality and reduced the possibility of including redundant peptides in subsequent data analysis.  All keratins were removed from the protein identification list. The current status of each protein, its name and molecular weight was updated by batch processing it on the NCBI protein database website. Where protein names contained the word ‘predicted’ or ‘like protein’, the accuracy of the assigned names were examined and corrected, where necessary. The issue of redundancy in identified proteins was addressed by including all possible assignments in the results table to make the data as transparent as possible. In those cases where an identified protein shares its sequence with other entries in the database then all entries are listed. The uniqueness (or otherwise) of the assignment of peptides to the sequence of one or more proteins was used to determine whether assignment could be made to a specific protein or to a group of related proteins (isoforms). Quantification was based on at least 2 peptides that were all uniquely assigned to one protein or to a group of related forms.   Protein abundance values were calculated on the basis of the ion intensities of the component peptides.  For detection of differentially expressed proteins, only proteins identified on the basis of at least two unique peptides were considered. Differential proteins were identified on the basis of 1.5-fold difference in abundance in AdCLIC4 versus AdControl and P-value <0.05. Ingenuity Pathway Analysis was used for mapping the identified proteins to known biological pathways.

### Publication Abstract
None

### Keywords
Human, Clic4, Proteomics, Endothelial cells

### Affiliations
Centre for Pharmacology & Therapeutics
Imperial College London

### Submitter
Vahitha Abdul Salam

### Lab Head
Dr Vahitha Abdul Salam
Imperial College London


