### Accession
PXD005319

### Title
Olfactory bulb proteomics in human Alzheimer's disease

### Description
Olfactory dysfunction is among the earliest features of Alzheimer´s disease (AD). Although neuropathological abnormalities have been detected in the olfactory bulb (OB), little is known about its dynamic biology. Here, OB- proteome analysis was performed across different AD stages using a label-free approach.

### Sample Protocol
OB specimens derived from control and AD cases were homogenized in lysis buffer containing 7 M urea, 2 M thiourea, 4% (w/v) CHAPS, 50 mM DTT. The homogenates were spinned down at 100.000 x g for 1 h at 15°C. Prior to proteomic analysis, protein extracts were precipitated with methanol/choloroform, and pellets dissolved in 6M Urea, Tris 100mM pH 7.8. Protein quantitation was performed with the Bradford assay kit (Bio-Rad). Label free LC-MS/MS – The protein extract for each sample was diluted in Laemmli sample buffer and loaded into a 0.75 mm thick polyacrylamide gel with a 4% stacking gel casted over a 12.5% resolving gel. The run was stopped as soon as the front entered 3 mm into the resolving gel so that the whole proteome became concentrated in the stacking/resolving gel interface. Bands were stained with Coomassie Brilliant Blue and excised from the gel. Protein enzymatic cleavage (10ug) was carried out with trypsin (Promega; 1:20, w/w) at 37°C for 16 h as previously described 75. Peptides mixtures were separated by reverse phase chromatography using an Eksigent nanoLC ultra 2D pump fitted with a 75 μm ID column (Eksigent 0.075 x 250). Samples were first loaded for desalting and concentration into a 0.5 cm length 100 μm ID precolumn packed with the same chemistry as the separating column. Mobile phases were 100% water 0.1% formic acid (FA) (buffer A) and 100% Acetonitrile 0.1% FA (buffer B). Column gradient was developed in a 240 min two step gradient from 5% B to 25% B in 210 min and 25%B to 40% B in 30 min. Column was equilibrated in 95% B for 9 min and 5% B for 14 min. During all process, precolumn was in line with column and flow maintained all along the gradient at 300 nl/min. Eluting peptides from the column were analyzed using an Sciex 5600 Triple-TOF system. Information data acquisition was acquired upon a survey scan performed in a mass range from 350 m/z up to 1250 m/z in a scan time of 250 ms. Top 35 peaks were selected for fragmentation. Minimum accumulation time for MS/MS was set to 100 ms giving a total cycle time of 3.8 s. Product ions were scanned in a mass range from 230 m/z up to 1500 m/z and excluded for further fragmentation during 15 s.

### Data Protocol
MS/MS data acquisition was performed using Analyst 1.7.1 (AB Sciex) and spectra ﬁles were processed through Protein Pilot Software (v.5.0-Sciex) using Paragon™ algorithm (v.5.0) for database search, Progroup™ for data grouping, and searched against the concatenated target-decoy UniProt proteome reference Human database (Proteome ID: UP000005640, 70902 proteins, December 2015). False discovery rate was performed using a non lineal fitting method and displayed results were those reporting a 1% Global false discovery rate or better. The peptide quantification was performed using the Progenesis LC−MS software (ver. 2.0.5556.29015, Nonlinear Dynamics). Using the accurate mass measurements from full survey scans in the TOF detector and the observed retention times, runs were aligned to compensate for between-run variations in our nanoLC separation system. To this end, all runs were aligned to a reference run automatically chosen by the software, and a master list of features considering m/z values and retention times was generated. The quality of these alignments was manually supervised with the help of quality scores provided by the software. The peptide identifications were exported from Protein Pilot software and imported in Progenesis LC− MS software where they were matched to the respective features. Output data files were managed using R scripts for subsequent statistical analyses and representation. Proteins identified by site (identification based only on a modification), reverse proteins (identified by decoy database) and potential contaminants were filtered out. Proteins quantified with at least two unique peptides, an ANOVA p-value lower than 0.05, and an absolute fold change of <0.77 (down-regulation) or >1.3 (up-regulation) in linear scale were considered to be significantly differentially expressed.

### Publication Abstract
Olfactory dysfunction is among the earliest features of Alzheimer's disease (AD). Although neuropathological abnormalities have been detected in the olfactory bulb (OB), little is known about its dynamic biology. Here, OB- proteome analysis showed a stage-dependent synaptic proteostasis impairment during AD evolution. In addition to progressive modulation of tau and amyloid precursor protein (APP) interactomes, network-driven proteomics revealed an early disruption of upstream and downstream p38 MAPK pathway and a subsequent impairment of Phosphoinositide-dependent protein kinase 1 (PDK1)/Protein kinase C (PKC) signaling axis in the OB from AD subjects. Moreover, a mitochondrial imbalance was evidenced by a depletion of Prohibitin-2 (Phb2) levels and a specific decrease in the phosphorylated isoforms of Phb1 in intermediate and advanced AD stages. Interestingly, olfactory Phb subunits were also deregulated across different types of dementia. Phb2 showed a specific up-regulation in mixed dementia, while Phb1 isoforms were down-regulated in frontotemporal lobar degeneration (FTLD). However, no differences were observed in the olfactory expression of Phb subunits in progressive supranuclear palsy (PSP). To sum up, our data reflect, in part, the missing links in the biochemical understanding of olfactory dysfunction in AD, unveiling Phb complex as a differential driver of neurodegeneration at olfactory level.

### Keywords
Proteomics, Olfactory bulb

### Affiliations
Proteomics unit
Clinical Neuroproteomics Group, Navarrabiomed, Departamento de Salud, UPNA, IdISNA, 31008 Pamplona, Spain

### Submitter
Enrique SantamarÃ­a

### Lab Head
Dr ENRIQUE SANTAMARIA
Clinical Neuroproteomics Group, Navarrabiomed, Departamento de Salud, UPNA, IdISNA, 31008 Pamplona, Spain


