### Accession
PXD004223

### Title
The proteome of normal human sclera

### Description
The sclera provides an opaque protective coat for five-sixths of the surface of the human eye. A detailed investigation of the proteome of the normal human sclera may provide insights into the biology and pathophysiology of the sclera. We conducted an in-depth proteomic analysis of sclera from five adults. Proteins were fractionated using SDS-PAGE. After in-gel digestion, peptides were analyzed using LC-MS/MS on an Orbitrap Elite mass spectrometer. We identified 1952 non-redundant proteins in the human sclera, including 18 different collagen protein chains (types 1, II, III, IV, VI, VIII, X, XII, XIV, XV, XVI, XVIII), proteins of the small leucine-rich proteoglycan family (decorin, biglycan, lumican, keratocan, prolargin, fibromodulin, and mimecan), non-collagenous glycoproteins (fibronectin, vitronectin, and laminin), extracellular matrix proteins (thrombospondins 1, 2, 3, 4, dystroglycan, and transgelins 1, 2, 3), matrix metalloproteinases 3 and 14, matrix metalloproteinase inhibitors 1, 2, and 3, and integrins alpha-V, alpha-1, alpha-2, beta-1, beta-2, and beta-5.

### Sample Protocol
Five human donor eyes were obtained from the Lions Eye Institute, Tampa, FL. Right eyes were selected from five adults (4 males, 1 female), age 51-76 y, with no history of eye disease or previous eye surgery. The causes of death were myocardial infarction, end-stage renal disease (2), and cancer, not on chemotherapy (2). Time of death to enucleation was 3-7 h for all subjects. Globes were placed in eye bank vials and refrigerated until received at the Wilmer Eye Institute. Time from enucleation to tissue freezing was 24-36 h for all subjects. A corneal surgeon (S.F.) dissected the globes following a standardized protocol for isolation of human eye tissues [6]. Sclera was dissected free from cornea, trabecular meshwork, ciliary body, choroid, and optic nerve head. All rectus muscles were cut at their insertion into the sclera. Sclera samples were immediately snap frozen and stored continuously at –80°C until processing. Sclera was thawed, cut into 2-3 mm pieces, and ground into a fine powder using a cryogenic grinder (Mixer Mill 400, Retsch, Haan, Germany). Sclera samples were suspended in 1 mL of lysis buffer (10 mM HEPES, 42 mM KCl, ¬¬0.1 mM EDTA, 0.1 mM EGTA, 1mM dithiothreitol [DTT], 1 phosphatase inhibitor [Pierce, PI78420], 1 protease inhibitor [Sigma, P8340]), and homogenized using a Retsch Mixer Mill 400 in liquid nitrogen cooled canisters at 30 Hz for 1.5 minutes. Samples were then sonicated three times for ten seconds, storing on ice to prevent overheating between sonications. Sodium dodecyl sulfate was added for a final concentration of 2% (w/v). Samples were incubated at room temperature for 10 min to lyse the cells and extract the proteins. The samples were then spun for 45 min, 14,000  g, at room temperature. Protein concentration of the supernatant was determined using the Pierce microBCA kit.  Proteins were separated using SDS-PAGE gels. Double distilled H2O was added to the samples to achieve 114 g of protein in 45 L volume in a low retention Eppendorf tube. Four mL of 4 LDS SE + 100 mM DDT was added to each tube. Tubes were vortexed, spun briefly, and placed on a 106C heat block for 10 min to denature the proteins. Tubes were vortexed and spun twice. One hundred and fourteen g of sample was run on each column on a 1D SDS gel using NuPAGE 4-12% BT 1.5, 200V for 35 min. Gels were stained with Coomassie blue (250 mL methanol, 50 mL acetic acid, 200 mL dH2O, 0.5 g Coomassie brilliant blue) on a rotating table at room temperature for 30 min. Gels were destained (50 mL methanol, 75 mL acetic acid, 875 mL dH2O) overnight. Each gel was cut into 12 bands. Gel pieces were placed in separate vials and disrupted with forceps. Gels were destained, dehydrated with acetonitrile, and dried down with a Speedvac concentrator. Samples were reduced and alkylated with 10 mM DTT in 25 mM ammonium bicarbonate, washed with ultrapure water, dehydrated with 100% acetonitrile, and dried. Samples were digested with trypsin/LysC (Promega, V5073) at 37C overnight with shaking. Supernatant with peptides was removed, incubated with 100% acetonitrile, and then dried. Peptides were resuspended in 0.1% formic acid in water, placed in a desalting column (Pierce Spin-Tip), spun, eluted with 0.1% formic acid in 80% acetonitrile, and then dried. Samples were resuspended in 0.1% formic acid in water and spike with 5 fmol/L calibration peptides (Pierce Peptide Retention Time Calibration Mixture).   Samples were run LC-MS/MS using a one-hour gradient of 2-30% acetonitrile (Fisher Scientific) with 0.1% formic acid (Suprapur) using an EASY-Spray source (Thermo) coupled with an Orbitrap Elite (Thermo Scientific) mass spectrometer. EASY-Spray source was run at 35°C using a 25 cm x 50 m integrated spray time column. Peptides were trapped at 980 bar on a 2 cm x 75 m trapping column. The trap was a 3 m particle, and the column was 2 m Acclaim PepMap C18. All individual samples were run with two technical replicates.

### Data Protocol
MS raw data were batch processed using PASS Suite (Integrated Analysis, Bethesda, Maryland), X!Tandem and OMSSA search engines, and the UniProt Sequence database. UniProt database included all human sequences, and parameters allowed for 2 missed tryptic cleavages, a fragment ion mass error of 1 Da, and potential protein modifications for carbamidomethylation, oxidation, phosphorylation, deamidation, and carbamylation.  Peptide identifications were accepted if they were established at >95.0% probability by the Peptide Prophet algorithm [7] with Scaffold delta-mass correction. Protein identifications were accepted if they were established at >99.0% probability and contained at least 1 identified peptide. Protein Prophet algorithm was used to assign protein probabilities [8] using a false discovery rate of 0.12% for peptides and 0.80% for proteins. The peptide threshold was 95% minimum and protein threshold was 99.0% minimum with a one-peptide minimum. Proteins that contained similar peptides and could not be differentiated based on MS/MS analysis alone were grouped to satisfy the principles of parsimony. Proteins sharing significant peptide evidence were grouped into clusters. Proteins were annotated with Gene Ontology terms from National Center for Biotechnology Information (downloaded March 8, 2014).

### Publication Abstract
The optic nerve is a white matter tract that conveys visual information to the brain. The sclera comprises the white, protective outer layer of the eye. A characterization of the proteome of normal human retrobulbar optic nerve and sclera may facilitate studies of the eye. We conducted a proteomic analysis of optic nerve and sclera from five adults. Proteins were fractionated using SDS-PAGE. After in-gel digestion, peptides were analyzed using LC-MS/MS on an Orbitrap Elite mass spectrometer. We identified 2711 non-redundant proteins in retrobulbar optic nerve and 1945 non-redundant proteins in sclera. Optic nerve proteins included proteins expressed by oligodendrocytes (laminin, proteolipid protein, fibronectin), myelin proteins (myelin basic protein, myelin-associated glycoprotein), and paranodal structural proteins (ankyrin &#x3b2;, spectrin). Sclera included 18 collagen protein chains, small leucine-rich proteoglycans (decorin, biglycan, lumican, keratocan, prolargin, fibromodulin, mimecan), non-collagenous glycoproteins (fibronectin, vitronectin, laminin), extracellular matrix proteins (thrombospondins 1-4, dystroglycan, transgelins 1-3), and integrins alpha-V, alpha-1 and 2, beta-1, -2, and -5. Twenty-one unambiguous alternative splicing protein isoforms were identified in optic nerve and ten unambiguous alternative splicing protein isoforms were identified in sclera. The mass spectrometry proteomics data have been deposited to the ProteomeXchange Consortium via the PRIDE partner repository with the dataset identifier PXD001581.

### Keywords
Collagen, Myopia, Proteome, Eye / human, Sclera

### Affiliations
Wilmer Eye Institute, Smith Building, M015, 400 N. Broadway, Baltimore, MD 21287
Johns Hopkins University

### Submitter
Pingbo Zhang

### Lab Head
Dr Richard D. Semba, M.D., M.P.H.,
Wilmer Eye Institute, Smith Building, M015, 400 N. Broadway, Baltimore, MD 21287


