### Accession
PXD001657

### Title
A Cell-based Systems Biology Assessment of Human Blood to Monitor Immune Responses After Influenza Vaccination

### Description
Systems biology is an approach to comprehensively study complex interactions within a biological system. Most published systems vaccinology studies have utilized whole blood or peripheral blood mononuclear cells (PBMC) to monitor the immune response after vaccination.  Because human blood is comprised of multiple hematopoietic cell types, the potential for masking responses of under-represented cell populations is increased when analyzing whole blood or PBMC. To investigate the contribution of individual cell types to the immune response after vaccination, we established a rapid and efficient method to purify human T and B cells, natural killer (NK) cells, myeloid dendritic cells (mDC), monocytes, and neutrophils from fresh venous blood. Purified cells were fractionated and processed in a single day. RNA-Seq and quantitative shotgun proteomics were performed to determine expression profiles for each cell type prior to and after inactivated seasonal influenza vaccination. Our results show that transcriptomic and proteomic profiles generated from purified immune cells differ significantly from PBMC. Differential expression analysis for each immune cell type also shows unique transcriptomic and proteomic expression profiles as well as changing biological networks at early time points after vaccination. This cell type-specific information provides a more comprehensive approach to monitor vaccine responses.

### Sample Protocol
Protein extracts from PBMC and sorted immune cells (1x106 cells) from the two vaccinated subjects were prepared as previously described using a modified lysis buffer (50% Trifluoroethanol 50 mM HEPES) and quantified by BCA assay.  An immune cell common standard (ICCS) control sample composed of protein extracts from PBMC and CD15+ cells (80% and 20%, respectively, by protein weight) was included in all iTRAQ experiments.   Ten ug of reduced, alkylated, and trypsinized protein extracts were labeled with iTRAQ tags (AB Sciex), pooled, and analyzed by MudPIT using an Eksigent 2-D nanoLC pump coupled to a nanoESI-LTQ-OrbitrapXL mass spectrometer (Thermo Scientific).  The precursor ions were analyzed in the Orbitrap followed by 4 CID fragment ion scans in the ion trap to identify peptides. The precursor ions were then fragmented by HCD to measure reporter ion intensities in the Orbitrap.

### Data Protocol
For each precursor ion, the CID and HCD spectra were merged using Proteome Discoverer v1.3 (Thermo Scientific). The merged fragmentation spectra were searched against a forward and reverse concatenated human Ensembl protein and common contaminants database (gene model 74) using the Sequest database search engine running under Proteome Discoverer. Precursor mass tolerance was set to 20 ppm and fragment mass tolerance was set to 0.8 Da. iTRAQ modification of N-terminus and Îµ-amine of lysines and Beta-methylthiolation of cysteines were used as static/constant modifications of the peptides. Oxidation of methionine and tryptophan and deamidation of asparagine and glutamine were used as dynamic/variable modifications of the peptides. Protein assembly, reporter ion quantitation and statistical analysis were performed with a 5% peptide and protein FDR using ProteoIQ v2.61 (Premier Biosoft).  A slope of the regression line >0.8 between the technical replicates of the common control (ICCS) based upon pseudospectral counts was required as a quality control threshold

### Publication Abstract
Systems biology is an approach to comprehensively study complex interactions within a biological system. Most published systems vaccinology studies have utilized whole blood or peripheral blood mononuclear cells (PBMC) to monitor the immune response after vaccination. Because human blood is comprised of multiple hematopoietic cell types, the potential for masking responses of under-represented cell populations is increased when analyzing whole blood or PBMC. To investigate the contribution of individual cell types to the immune response after vaccination, we established a rapid and efficient method to purify human T and B cells, natural killer (NK) cells, myeloid dendritic cells (mDC), monocytes, and neutrophils from fresh venous blood. Purified cells were fractionated and processed in a single day. RNA-Seq and quantitative shotgun proteomics were performed to determine expression profiles for each cell type prior to and after inactivated seasonal influenza vaccination. Our results show that transcriptomic and proteomic profiles generated from purified immune cells differ significantly from PBMC. Differential expression analysis for each immune cell type also shows unique transcriptomic and proteomic expression profiles as well as changing biological networks at early time points after vaccination. This cell type-specific information provides a more comprehensive approach to monitor vaccine responses.

### Keywords
Human, Tiv, Immune response, Quantitative proteomics, Vaccine response, Immune cells, Itraq, Systems vaccinology, Blood

### Affiliations
Department of Pathology, Microbiology, and Immunology Vanderbilt University School of Medicine Nashville, TN 37232 USA
Vanderbilt University School of Medicine

### Submitter
Andrew Link

### Lab Head
Dr Andrew Link
Department of Pathology, Microbiology, and Immunology Vanderbilt University School of Medicine Nashville, TN 37232 USA


