### Accession
PXD004476

### Title
Human Bile LC-MS/MS -  One Sample, One Shot - Evaluation of sample preparation protocols for the mass spectrometric proteome analysis of human bile fluid without extensive fractionation

### Description
The proteome analysis of bile fluid represents a promising strategy to identify biomarker candidates for various diseases of the hepatobiliary system. However, to obtain substantive results in biomarker discovery studies large patient cohorts necessarily need to be analyzed. Consequently, this would lead to an unmanageable number samples to be measured if sample preparation protocols with extensive fractionation methods are applied. Hence, the performance of simple workflows allowing for "one sample, one shot" experiments have been evaluated in this study. In detail, sixteen different protocols implying modifications at the stages of desalting, delipidation, deglycosylation and tryptic digestion have been tested. Each method has been individually evaluated regarding various performance criteria and comparative analyses have been conducted to uncover possible complementarities. Here, the best performance in terms of proteome coverage has been assessed for a combination of acetone precipitation with in-gel digestion. Finally, a mapping of all obtained protein identifications with putative biomarkers for hepatocellular carcinoma (HCC) and cholangiocellular carcinoma (CCC) revealed several proteins easily detectable in bile fluid. These results can build the basis for future studies with large and well-defined patient cohorts in a more disease-related context, instead of a technical one as reported here.

### Sample Protocol
16 different sample preparation protocols of human bile fluid have been evaluated. Varialbe sample preparation steps included: acetone precipitation, buffer exchange, in-gel digestion, in-solution digestion, deglycosylation, delipidation.  Mass spectrometric analyses were performed on an Ultimate 3000 RSLCnano system online coupled to an Orbitrap Elite mass spectrometer (both Thermo Scientific). 350 ng of tryptic peptides in 15 l 0.1 % TFA were analyzed per run. After injection, samples were pre-concentrated on a trap column (Acclaim PepMap 100, 300 m x 5 mm, C18, 5 m, 100 ) for 7 min with 0.1 % TFA at a flow rate of 30 l/min. Reversed-phase separation was performed on an analytical column (Acclaim PepMap RSLC, 75 m x 50 cm, nano Viper, C18, 2 m, 100 ) with a 60 min gradient from 5 % to 40 % solvent B (solvent A: 0.1 % formic acid; solvent B: 0.1 % formic acid, 84 % acetonitrile) at a flow rate of 400 nl/min and a temperature of 60 C. The mass spectrometer was operated in a data-dependent mode. Full-scan MS spectra were acquired at a resolution of 60,000 in the Orbitrap analyzer. MS/MS spectra of the 20 highest abundant precursor masses of each MS spectrum were measured in the linear ion trap. For fragmentation, collision-induced dissociation was used.

### Data Protocol
For peptide and protein identification, raw LC-MS/MS data were analyzed using Proteome Discoverer software (ver. 1.4, Thermo Fisher Scientific Inc.). Mascot (ver. 2.5.1, Matrix Science Ltd., London, UK) was used to search against human Uniprot/Swiss-Prot database (release 2015_05, number of sequences: 548,454). Mass tolerances were set to 5 ppm and 0.4 Da for precursor and fragment ion masses, respectively. Trypsin was set as protease and one miscleavage was allowed. Taxonomy was restricted to homo sapiens. Except the oxidation of methionine, that was set as variable modification in all searches, different chemical modifications of amino acids have been considered with respect to the applied workflow. For in-gel digestion procedures, propionamide modification of cysteine was considered as a variable modification. For in-solution digestion, carbamidomethyl modification of cysteine was set as a static one. If deglycosylation was applied, deamination of glutamine and asparagine were additionally set as dynamic modifications. Confidence of peptide identifications was estimated using percolator node, which is implemented in Proteome Discoverer. The function of this particular search node has been described earlier in more detail[12]. Peptide identifications with false discovery rates > 1 % (q-value > 0.01) were discarded. Protein grouping option of Proteome Discoverer was enabled in all analyses.

### Publication Abstract
None

### Keywords
Cholangiocellular carcinoma, Proteomics, Method comparison, Biomarkers, Hepatocellular carcinoma, Bile fluid

### Affiliations
Medizinisches Proteom-Center (MPC), Ruhr-Universität Bochum, Germany
Medizinisches Proteom-Center (MPC) Ruhr-Universität Bochum Clinical Proteomics - Infection Research Universitaetsstrasse 150, D-44801 Bochum

### Submitter
Dominik Andre Meger

### Lab Head
Dr Dominik Andre Megger
Medizinisches Proteom-Center (MPC) Ruhr-Universität Bochum Clinical Proteomics - Infection Research Universitaetsstrasse 150, D-44801 Bochum


