### Accession
PXD036701

### Title
Proteomic study of a breast cancer cell line following hemin treatment

### Description
Heme, an iron-containing prosthetic group found in many proteins, carries out diverse biological functions such as electron transfer, oxygen storage and enzymatic reactions. Hemin, the oxidised form of heme, is used to treat porphyria and also to activate heme-oxygenase (HO) which catalyses the rate-limiting step in heme degradation. Our group has previously demonstrated that hemin displays antitumor activity in breast cancer (BC). The aim of this work has been to study the effect of hemin on protein expression modifications in a BC cell line to gain insight into the molecular mechanisms of hemin antitumor activity. For this purpose, we carried out proteome analysis by Mass Spectrometry (MS) which showed that 1309 proteins were significantly increased in hemin-treated cells, including HO-1 and the proteases that regulate HO-1 function, and 921 proteins were significantly decreased. Furthermore, the MS-data analysis showed that hemin regulates the expression of heme- and iron- related proteins, adhesion and cytoskeletal proteins, cancer signal transduction proteins and enzymes involved in lipid metabolism. By biochemical and cellular studies, we further corroborated the most relevant in-silico results. Altogether, these results show the multiple physiological effects that hemin treatment displays in BC and demonstrate its potential as anticancer agent.

### Sample Protocol
LM3 cells were treated with hemin (80 ÂµM) and its respective control (vehicle: 0.4mL of NaOH 0.5N and 0.5mL Tris-HCl 0.5N) for 24h. This concentration was chosen based on our previous results in BC16. Cells were lysed with radio- immunoprecipitation assay (RIPA) lysis buffer (25mM Trisâ€“HCl, pH 7.5; 150mM NaCl; 1% TX100; 0.2% SDS; 0.5% sodium deoxycholic acid; protease and phosphatases inhibitors) on ice, following Bradford (BioRad) protein quantification. The same amount of protein was incubated with Laemmli buffer for five minutes at 95Â°C. Proteins were separated by SDSâ€“polyacrylamide gel electrophoresis on a 4â€“15% gradient gel. The gel was stained with Coomassie using the GelCode Blue Safe Protein Stain reagent (Thermo) and used for mass analysis. Two independent experiments were performed (seven biological replicates were processed for control and seven for hemin condition. Sample preparation â€“ Gel lanes of interest were excised, chopped and washed twice with 150 ÂµL of de-staining buffer (25 mM ammonium bicarbonate, 50% ethanol). Gel pieces were dehydrated twice in 150 ÂµL of 100% ethanol and the gel pieces were dried by vacuum centrifugation. Then, 50 ÂµL of digestion buffer (25mM Tris HCl, 10% acetonitrile, 10ng/ÂµL of trypsin) was added. After incubation for 20 min on ice, 50 ÂµL of ammonium bicarbonate buffer (25mM) was added and the gel pieces were incubated at 37Â°C overnight. Peptides in the supernatant were collected and more peptides were extracted from the gel pieces by repeated incubation of the gel pieces at 25Â°C in 100 ÂµL of extraction buffer (3% TFA, 30% acetonitrile), with a subsequent centrifugation and collection of the supernatants. Finally, the gel pieces were dehydrated by incubation at 25Â°C in 100 ÂµL of 100% acetonitrile and the supernatant was unified with the supernatants from previous extraction steps. Acetonitrile was removed by vacuum-centrifugation and 70 ÂµL of 2 M Tris HCl as well as 10 mM tris (2-carboxyethyl) phosphine and 40 mM 2-chloroacetamide was added. After incubation for 30 min at 37Â°C, peptides were acidified to 1% TFA and desalted using Stage Tips.

### Data Protocol
Liquid Chromatography with tandem mass spectrometry (LC-MS/MS) data acquisition - Purified and desalted peptides were loaded onto a 30 cm column (inner diameter: 75 microns; packed in-house with ReproSil-Pur C18-AQ 1.9-micron beads, Dr. Maisch GmbH) via the autosampler of the Thermo Easy-nLC 1000 (Thermo Fisher Scientific) at 60Â°C. Using the nanoelectrospray interface, eluting peptides were directly sprayed onto the benchtop Orbitrap mass spectrometer Q Exactive HF (Thermo Fisher Scientific). Peptides were loaded in buffer A (0.1% formic acid) at 250 nl/min and percentage of buffer B (80% acetonitrile, 0.1% formic acid) increased from 2% to 30% over 120 min, followed by an increase to 60% over 10 min and then 95% over the next 5 min. Percentage of buffer B was maintained at 95% for another 5 min. The mass spectrometer was operated in a data-dependent mode with survey scans from 300 to 1650 m/z (resolution of 60000 at m/z =200), and up to 10 of the top precursors were selected and fragmented using higher energy collisional dissociation (HCD with a normalised collision energy of value of 28). The MS2 spectra were recorded at a resolution of 15000 (at m/z = 200). AGC target for MS and MS2 scans were set to 3E6 and 1E5 respectively within a maximum injection time of 100 and 60 min for MS and MS2 scans respectively. Dynamic exclusion was set to 30 m. Data Analysis â€“ Raw data were processed using the MaxQuant computational platform (version 1.6.5.0) (https://www.nature.com/articles/nbt.1511) with standard settings applied. Shortly, the peak list was searched against the Uniprot database of Mus musculus (55.466 entries) with an allowed precursor mass deviation of 4.5 ppm and an allowed fragment mass deviation of 20 ppm. MaxQuant, by default, enables individual peptide mass tolerances, which was used in the search. Cysteine carbamidomethylation, methionine oxidation and N-terminal acetylation were set as variable modifications. Proteins were quantified across samples using the label-free quantification algorithm in MaxQuant as label-free quantification (LFQ) intensities. Bioinformatic analysis and heatmaps generation were made with the free access program Perseus, developed at Max Planck Biochemistry institute by Dr. Cox (https://maxquant.net/perseus/)

### Publication Abstract
Heme, an iron-containing prosthetic group found in many proteins, carries out diverse biological functions such as electron transfer, oxygen storage and enzymatic reactions. Hemin, the oxidised form of heme, is used to treat porphyria and also to activate heme-oxygenase (HO) which catalyses the rate-limiting step in heme degradation. Our group has previously demonstrated that hemin displays antitumor activity in breast cancer (BC). The aim of this work has been to study the effect of hemin on protein expression modifications in a BC cell line to gain insight into the molecular mechanisms of hemin antitumor activity. For this purpose, we carried out proteome analysis by Mass Spectrometry (MS) which showed that 1309 proteins were significantly increased in hemin-treated cells, including HO-1 and the proteases that regulate HO-1 function, and 921 proteins were significantly decreased. Furthermore, the MS-data analysis showed that hemin regulates the expression of heme- and iron-related proteins, adhesion and cytoskeletal proteins, cancer signal transduction proteins and enzymes involved in lipid metabolism. By biochemical and cellular studies, we further corroborated the most relevant in-silico results. Altogether, these results show the multiple physiological effects that hemin treatment displays in BC and demonstrate its potential as anticancer agent.

### Keywords
Hemin, Proteomic, Heme oxygenase-1, Breast cancer, Mass spectrometry

### Affiliations
Cancer Biology laboratory, INIBIBB_UNS_CONICET
CONICET

### Submitter
GEORGINA COLO

### Lab Head
Dr Alejandro Curino
Cancer Biology laboratory, INIBIBB_UNS_CONICET


