### Accession
PXD005828

### Title
Proteomic of Hydra vulgaris

### Description
This study aimed to identify the total proteome of Hydra vulgaris, a freshwater polyp, belongs to the cnidarians family with feature of not having a visual eye, while being sensitive to light.  Proteins were identified using a combination of gel electrophoresis and data-independent nanoflow liquid chromatography mass spectrometry resulting in the identification of more than 5,200 proteins from all cellular components

### Sample Protocol
Mass spectrometry analysis of Hydra vulgaris was performed using 5 different sample preparation protocols (use of different buffer to resuspended the Hydra vulgaris pellet, SDS PAGE 12% and 15%, use the the FOCUS Membrane Proteins Kit from Agro bio-G-Biosciences). The 5 protocol are described in detail in the Dataset Brief. Gels bands were cut, reduced, alkylated and enzymatically digested overnight. Resulting peptides were extracted and analyzed by nanoLC-MSMS. NanoLC-MS/MS analyses were performed on two systems: i/  a nanoACQUITY Ultra-Performance-LC system (UPLC) coupled to a Q-Exactive Plus Orbitrap (Thermo Fisher Scientific) mass spectrometer : ii/  a nanoACQUITY Ultra-Performance-LC system (UPLC) coupled to a TripleTOF 5600 (AB Sciex).

### Data Protocol
Mass data collected were searched using a local Mascot server (version 2. 2. 1, MatrixScience, London, UK) against an in-house generated protein database composed of protein sequences of Hydra vulgaris (downloaded from NCBI (22.785 sequences) and UniprotKB (6.810 sequences), taxonomy: 6087, release 2016-12-20). AprÃ¨s suppression des sequences redondantes et des sequences strictement incluse dans une sÃ©quence, il ne reste plus que 24007 protÃ©ines.  A list of 118 contaminants such as human keratins and trypsin frequently observed in proteomics analyses were add. These sequences were combined with reverse sequences for all entries using an in-house database generation toolbox available at https://msda.unistra.fr (total 48.250 entries).  The following search parameters were applied: carbamidomethylation of cysteines was and oxidation of methionines were set as variable modifications. Specificity of trypsin digestion was set for cleavage after K or R, and one missed trypsin cleavage site was allowed. The mass tolerances for protein identification on MS and MS/MS peaks were 10 ppm and 0.07 Da respectively.  The FDR was calculated by performing the search in the concatenated target and decoy databases in Mascot. Peptides identified were validated by applying the target-decoy approach, using Proline software (http://proline.profiproteomics.fr/), by adjusting the FDR to 1%, at PSM and protein levels. At peptide level, only the PSM with the best Mascot score was retained for each peptide sequence. Spectra identifying peptides in both target and decoy database searches were first assembled to allow competition between target and decoy peptides for each MS/MS query. Finally, the total number of validated hits was computed as Ntarget+Ndecoy, the number of false-positive hits was estimated as 2Ã—Ndecoy, and the FDR was then computed as 2Ã—Ndecoy/(Ntarget + Ndecoy). Proline software automatically determined a threshold Mascot Adjusted e-value to filter peptides and computed the FDR as described so as to automatically adjust it to 1%. At protein level, a composite score was computed for each protein group based on the MudPIT scoring method implemented in Mascot: For each nonduplicate peptide identifying a protein group, the difference between its Mascot score and its homology threshold was computed, and these â€œscore offsetsâ€� were then summed before adding them to the average homology (or identity) thresholds for the peptide. Therefore, less significant peptide matches contributed less to the total protein score. Protein groups were filtered by applying a threshold to this MudPIT protein score to obtain a final protein-level FDR of 1%. To optimize discrimination between true-positive and true-negative protein hits, the software applies a selection scheme approach by adjusting the FDR separately for the subset of proteins identified by more than one validated peptide and then for the single-peptide hits.

### Publication Abstract
None

### Keywords
Hydra vulgaris sds-gel-page nanolcmsms

### Affiliations
CNRS
CNRS, UMR7178, LSMBO, 67087 Strasbourg, France  UniversitÃ© de Strasbourg, IPHC, 25 rue Becquerel, 67087 Strasbourg, France

### Submitter
Francois Delalande

### Lab Head
Dr Sarah CIANFERANI
CNRS, UMR7178, LSMBO, 67087 Strasbourg, France  UniversitÃ© de Strasbourg, IPHC, 25 rue Becquerel, 67087 Strasbourg, France


