### Accession
PXD002857

### Title
A systematic analysis of methylpeptide false discovery rates

### Description
All large-scale LC-MS/MS post-translational methylation site discovery experiments require methylpeptide spectrum matches (methyl-PSMs) to be identified at acceptably low false discovery rates (FDRs). To meet estimated methyl-PSM FDRs, methyl-PSM filtering criteria are often determined using the target-decoy approach. The efficacy of this methyl-PSM filtering approach has, however, yet to be thoroughly evaluated. Here we conduct a systematic analysis of methyl-PSM FDRs across a range of sample preparation workflows (each differing in their exposure to the alcohols methanol and isopropanol) and mass spectrometric instrument platforms (each employing a different mode of MS/MS dissociation). Through 13CD3-methionine labeling (heavy-methyl SILAC) of S. cerevisiae cells and in-depth manual data inspection, accurate lists of true positive methyl-PSMs were determined, allowing methyl-PSM FDRs to be compared to target-decoy approach-derived methyl-PSM FDR estimates. These results show that global FDR estimates produce extremely unreliable methyl-PSM filtering criteria; we demonstrate that this is an unavoidable consequence of the high number of amino acid combinations capable of producing peptide sequences that are isobaric to methylated peptides of a different sequence. Separate methyl-PSM FDR estimates were also found to be unreliable due to prevalent sources of false positive methyl-PSMs that produce high peptide identity score distributions. Incorrect methylation site localizations, peptides containing cysteinyl-S-Î²-propionamide, and methylated glutamic or aspartic acid residues can partially, but not wholly, account for these false positive methyl-PSMs. Together these results indicate that the target-decoy approach is an unreliable means of estimating methyl-PSM FDRs and methyl-PSM filtering criteria. We suggest that orthogonal methylpeptide validation (e.g. heavy-methyl SILAC or its offshoots) should  be considered a prerequisite for obtaining high confidence methyl-PSMs in large-scale LC-MS/MS methylation site discovery experiments, and make recommendations on how to reduce methyl-PSM FDRs in samples not amenable to heavy isotope labeling.

### Sample Protocol
We investigate data obtained from whole cell lysates from a model organism, S. cerevisiae, grown in media containing either unlabeled or 13CD3-labeled methionine; lysates were mixed and prepared for LC-MS/MS analysis using three sample preparation workflows (prefractionation using: HILIC; SDS-PAGE and Coomassie staining; SDS-PAGE (no staining). Samples were subjected to LC-MS/MS analysis using the following three mass spectrometric instrument platforms, each employing a different MS/MS dissociation method: LTQ Orbitrap Velos Pro (collision-induced dissociation; CID), LTQ Orbitrap Velos Pro ETD (electron-transfer dissociation; ETD), and Q Exactive Plus (higher-energy collision dissociation; HCD).

### Data Protocol
By making use of the isotopic labeling of enzyme-mediated methylation, in-depth automated and manual inspections of LC-MS/MS data were performed to accurately determine true positive methyl-PSMs following sequence database searches. Specifically, the following data processing and validation steps were undertaken: 1) sequence database searching using Mascot (via Proteome Discoverer); 2) automated and manual identification of heavy labeled partner peptides (MaxQuant; Perl; XCalibur); 3) manual interrogation of MS/MS spectra; 4) manual techinal and sample replicate analysis; 5) synthetic peptide validation.

### Publication Abstract
All large scale LC-MS/MS post-translational methylation site discovery experiments require methylpeptide spectrum matches (methyl-PSMs) to be identified at acceptably low false discovery rates (FDRs). To meet estimated methyl-PSM FDRs, methyl-PSM filtering criteria are often determined using the target-decoy approach. The efficacy of this methyl-PSM filtering approach has, however, yet to be thoroughly evaluated. Here, we conduct a systematic analysis of methyl-PSM FDRs across a range of sample preparation workflows (each differing in their exposure to the alcohols methanol and isopropyl alcohol) and mass spectrometric instrument platforms (each employing a different mode of MS/MS dissociation). Through (13)CD3-methionine labeling (heavy-methyl SILAC) of Saccharomyces cerevisiae cells and in-depth manual data inspection, accurate lists of true positive methyl-PSMs were determined, allowing methyl-PSM FDRs to be compared with target-decoy approach-derived methyl-PSM FDR estimates. These results show that global FDR estimates produce extremely unreliable methyl-PSM filtering criteria; we demonstrate that this is an unavoidable consequence of the high number of amino acid combinations capable of producing peptide sequences that are isobaric to methylated peptides of a different sequence. Separate methyl-PSM FDR estimates were also found to be unreliable due to prevalent sources of false positive methyl-PSMs that produce high peptide identity score distributions. Incorrect methylation site localizations, peptides containing cysteinyl-S-&#x3b2;-propionamide, and methylated glutamic or aspartic acid residues can partially, but not wholly, account for these false positive methyl-PSMs. Together, these results indicate that the target-decoy approach is an unreliable means of estimating methyl-PSM FDRs and methyl-PSM filtering criteria. We suggest that orthogonal methylpeptide validation (e.g. heavy-methyl SILAC or its offshoots) should be considered a prerequisite for obtaining high confidence methyl-PSMs in large scale LC-MS/MS methylation site discovery experiments and make recommendations on how to reduce methyl-PSM FDRs in samples not amenable to heavy isotope labeling. Data are available via ProteomeXchange with the data identifier PXD002857.

### Keywords
False discovery rates, Lysine methylation, Target-decoy approach, Heavy-methyl silac, Arginine methylation

### Affiliations
UNSW
School of Biotechnology and Biomolecular Sciences, Systems Biology Initiative, UNSW, Australia

### Submitter
Gene Hart-Smith

### Lab Head
Dr Marc Ronald Wilkins
School of Biotechnology and Biomolecular Sciences, Systems Biology Initiative, UNSW, Australia


