### Accession
PXD028502

### Title
Correlative proteomics identify the key roles of stress tolerance strategies in Acinetobacter baumannii in response to polymyxin and macrophages

### Description
Opportunistic pathogen Acinetobacter baumannii possesses stress tolerance strategies against host innate immunity and antibiotic killing. However, how the host-pathogen-antibiotic interaction affects the overall molecular regulation of pro- and anti-pathogenic machineries remains unexplored. Here, we simultaneously investigate proteomic changes in A. baumannii and macrophages following infection in the absence or presence of the last-line polymyxins. We discover that macrophages and polymyxins exhibit complementary effects to disarm several A. baumannii stress tolerance and survival strategies, including oxidative stress resistance, copper tolerance, bacterial iron acquisition and stringent response regulation systems. Using bacterial mutants with impaired stringent response associated (p)ppGpp synthetase/hydrolase spoT we demonstrate that spot mutants exhibit significantly enhanced susceptibility to polymyxin killing and reduced in vivo survivability compared to the wild-type strain. Together, our findings highlight that improved understanding of host-pathogen-antibiotic interplay is critical for optimisation of antibiotic use in patients and discovery of new antibiotics to tackle multidrug-resistant bacterial infections.

### Sample Protocol
To extract proteins from infected THP-1-dMs, cell monolayers were washed twice with ice cold DPBS (1×) to remove residual media prior to treatment with lysis buffer 1% sodium deoxycholate (SDC; Sigma, D6750-100G) in 100 mM HEPES (pH 8.5). For bacterial protein extraction, the previously isolated bacterial pellet was resuspended with 1% SDC in 100 mM HEPES (pH 8.5). SDC-treated samples were heated at 95°C for 10 min prior to probe sonication at output strength 3 and 50% duty cycle. Sample protein was then quantified using the Pierce BCA Protein Assay Kit (Thermo Fisher Scientific; Illinois, USA). The total protein was normalised to 150 μg followed by protein denaturation and alkylation with 10 mM bond-breaker tris-(2-carboxyethyl) phosphine (TCEP) and 40 mM chloroacetamide (CAA). Samples were then subjected to overnight trypsin digestion at 37°C. SDC was subsequently removed by 1% formic acid precipitation and the sample peptides purified using the Eppendorf PerfectPure C18 ZipTip protocol. Acetonitrile in eluted samples was removed by a vacuum concentrator. Peptide samples were then reconstituted in 0.1% formic acid and sonicated in a water bath prior to liquid chromatography-tandem mass spectrometry (LC-MS/MS) analyses.

### Data Protocol
Mass spectrometric analyses were performed by the Monash Proteomics & Metabolomics Facility. Briefly, a Dionex UltiMate 3000 RSLCnano system was employed with an Acclaim PepMap RSLC analytical column (75 µm × 50 cm, nanoViper, C18, 2 µm, 100Å; Thermo Scientific) and an Acclaim PepMap 100 trap column (100 µm × 2 cm, nanoViper, C18, 5 µm, 100Å; Thermo Scientific). Tryptic peptides of bacterial cells were analysed on an Orbitrap Fusion Tribrid Mass Spectrometer (Thermo Fisher Scientific) operated in data-dependent acquisition (DDA). For AB5075, raw files were analysed with MaxQuant v1.6.5.0 and statistical analysis of the label-free quantification (LFQ) intensities was performed in Perseus v1.6.2.3. The AB5075 proteome was originally annotated using the Prokaryotic Genome Analysis Tool (PGAT) AB5075-UW reference genome, and the annotation was updated using NCBI Assembly Database (https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/963/815/GCF_000963815.1_ASM96381v1/). For THP-1-dMs, the tryptic peptides were analysed on a QExactive Plus Mass Spectrometer (Thermo Scientific) operated in data-independent acquisition (DIA) mode.  The raw files of THP-1-dMs data were analysed with Spectronaut v13 Laika (Biognosys) to obtain relative LFQ values using in-house standard parameters. The expression dataset of each experimental groups was visualised with a principal component analysis (PCA) score plot generated using MetaboAnalyst and Venn diagram generated using InteractiVenn. Differences in protein expression levels were evaluated through comparison of the mean LFQ intensities among all experimental groups and expressed as the log2 fold change (log2 FC). Significance was determined using a two-sided, two-sample t-test with a false discovery rate (FDR) adjusted p-value. Differentially expressed proteins (DEPs) with a log2 FC >1 or <-1 and FDR <0.05 were considered statistically significant. Statistically significant bacterial DEPs were subjected to pathway analyses according to Clusters of Orthologous Groups (COGs) and predicted by eggNOG and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathways using R. Statistical significance of enrichment analysis was examined using Fisher’s Exact Test (FDR <0.2). THP 1-dMs DEPs were functionally analysed on KEGG and Reactome pathways using WebGestalt with the pathway enrichment significance set at FDR <0.05 using the Benjamini-Hochberg method.

### Publication Abstract
None

### Keywords
Acinetobacter baumannii, Stringent response, Proteomic, Polymyxin, Macrophage

### Affiliations
Monash University
Biomedicine Discovery Institute, Infection Program, Department of Microbiology, Monash University, Clayton, Victoria, Australia

### Submitter
Ralf Schittenhelm

### Lab Head
Dr Jian Li
Biomedicine Discovery Institute, Infection Program, Department of Microbiology, Monash University, Clayton, Victoria, Australia


