### Accession
PXD003762

### Title
Probing the HIV-Host Interactome in Infected Cells

### Description
Although genetically compact, HIV-1 commandeers vast arrays of cellular machinery that sustain and protect it during cycles of viral outgrowth. Transposon-mediated saturation linker scanning mutagenesis was used to isolate fully replication-competent viruses harboring a potent foreign epitope tag. Using these viral isolates, we performed differential isotopic labeling and affinity-capture mass spectrometric analyses on samples obtained from cultures of human lymphocytes to classify the vicinal interactomes of the viral Env and Vif proteins as they occur during natural infection. Importantly, interacting proteins were recovered without bias, regardless of their potential for positive, negative or neutral impact on viral replication. Specific host associations made with trimerized Env during its biosynthesis, at virological synapses, with innate immune effectors and with certain cellular signaling pathways (ex. Notch1) were identified. We also defined Vif associations with host proteins involved in the control of nuclear transcription and nucleoside biosynthesis as well as those that interact both stably and transiently with the cytoplasmic protein degradation apparatus. Our approach is broadly applicable to elucidating pathogen-host interactomes, providing high certainty identification of interactors by their direct access during cycling infection. Understanding the pathophysiological consequences of these associations is likely to provide strategic targets for antiviral intervention.

### Sample Protocol
Proteins that were co-isolated following immunoaffinity purification of Vif-3XFLAG and Env-3XFLAG were heated under reducing conditions (100 mM DTT) at 70Â°C for 10 min with agitation, then alkylated with 100 mM iodoacetamide at room temperature in the dark for 30 min. Reduced and alkylated proteins were resolved by 1D gel electrophoresis on a 4â€“12% NuPAGE Novex Bis-Tris gel and stained with SimplyBlue SafeStain (Invitrogen). In-gel digestion with trypsin was performed by dividing and pooling each lane into 5, 9, or 10 gel fractions for the Vif exchange, Vif-3XFLAG I-DIRT, and Env-3XFLAG, I-DIRT experiments, respectively. Gel fractions were destained in 50% acetonitrile (ACN) in 50 mM ammonium bicarbonate (ABC), dehydrated in 100% acetonitrile, and incubated overnight at 37Â°C with 12.5 ng/ml trypsin in 50 mM ABC. The reaction was quenched and peptides were extracted with 0.5% formic acid (FA) for 4 hrs, followed by a second extraction for 2 hrs with 50% ACN in 0.5% FA. Extracted peptides were desalted by C18- StageTips, concentrated by vacuum centrifugation to ~1 Âµl and suspended in 10 Âµl of 1% FA. Desalted peptides (4 Âµl) were analyzed by direct injection on to a C18 analytical column (Acclaim PepMap RSLC, 1.8 Âµm; 75 Âµm Ã— 25 cm) using a Dionex Ultimate 3000 nanoLC coupled online to an LTQ Orbitrap Velos mass spectrometer (Thermofisher Scientific, San Jose, CA). Peptide separation and MS detection was performed over 90 min at 250 nl/min using a discontinuous ACN gradient from 4% to 16% mobile phase B over 60 min, followed by 16% to 40 % mobile phase B over 30 min (mobile phase A, 0.1% FA; mobile phase B, 97% ACN in 0.1% FA). The mass spectrometer was operated in data-dependent acquisition mode with dynamic exclusion enabled and preview scan mode disabled. One acquisition cycle comprised a single full-scan mass spectrum (m/z = 350-1700) in the orbitrap (resolution = 60,000 at m/z = 400), followed by collision-induced dissociation (CID) fragmentation in the linear ion trap of the top 15 most intense precursor ions. CID fragmentation was performed with isolation width of 2.0 Da, normalized collision energy of 30, and activation time of 10 ms.

### Data Protocol
MS/MS spectra were extracted from Thermo RAW files that were derived from a single gel lane, and searched by Proteome Discoverer/SEQUEST (version 1.3, Thermo Fisher Scientific) against the human subset of the UniProt SwissProt protein sequence database (release 2010-11) appended with common contaminants and HIV-1 polyprotein and mature protein sequences (20,349 sequences). SEQUEST search parameters were as follows: full trypsin specificity with 2 missed cleavages, precursor and fragment tolerances of 10 ppm and 0.5 Da, fixed modification of carbamidomethylation of cysteine, and variable modifications for methionine oxidation, 13C6- lysine and arginine, deamidation of asparagine, and phosphorylation of serine, threonine, and tyrosine. Putative SEQUEST peptide-spectrum matches (PSMs) were scored by the Percolator algorithm to assign individual (PEP) and calculate global error probabilities (q-values) by a semi-supervised approach using reverse database searches. PSMs were filtered to achieve an estimated 1% peptide false discovery rate (q-value < 0.01) and then PSMs were assembled into protein groups requiring a minimum of two unique peptides and using the â€œstrict maximum parsimony principleâ€� option. Isotope-based relative quantification was performed using the Proteome Discoverer precursor ions quantification workflow. Briefly, the â€œevent detectorâ€� module detected and filter precursor (MS1) isotopic envelopes (mass precision = 2 ppm, signalto-noise ratio = 5, RT tolerance = 0.2 min, and single-peak/missing channels = 1). Each profile was m/z and RT-matched with its corresponding PSM, and the cognate light and heavy isotope pairs were subjected to intensity-based summation. The following precursor ion quant parameters were used: Replace missing quan values with minimum intensity, True; Use single-peak quan channels, True; Reject all quan values if not all quan channels are present, False; Maximum allowed fold change, 50; Use ratios above maximum allowed fold change, True. After automated calculation of light and heavy peptide abundances, an additional quantitative peptide-level filter was applied. Quantified peptides with light (forward) or heavy (reverse) quantitative values that were less than Â½ (Vif I-DIRT) or Â¼ (Env I-DIRT) of the median value (across all fractions) were excluded. Isotope ratios were calculated as light/total for forward I-DIRT experiments or heavy/total for reverse I-DIRT, Vif isotope exchange, and Env SILAC experiments. Peptides that corresponded to unique peptide sequences, i.e. not shared in other protein groups, were used to calculate the median protein isotope ratio (minimum of three quantified PSMs) and coefficient of variation. Protein groups, accessions numbers, and quantitative values were exported to Excel for downstream statistical and functional analysis.

### Publication Abstract
Although genetically compact, HIV-1 commandeers vast arrays of cellular machinery to sustain and protect it during cycles of viral outgrowth. Transposon-mediated saturation linker scanning mutagenesis was used to isolate fully replication-competent viruses harbouring a potent foreign epitope tag. Using these viral isolates, we performed differential isotopic labelling and affinity-capture mass spectrometric analyses on samples obtained from cultures of human lymphocytes to classify the vicinal interactomes of the viral Env and Vif proteins as they occur during natural infection. Importantly, interacting proteins were recovered without bias, regardless of their potential for positive, negative or neutral impact on viral replication. We identified specific host associations made with trimerized Env during its biosynthesis, at virological synapses, with innate immune effectors (such as HLA-E) and with certain cellular signalling pathways (for example, Notch1). We also defined Vif associations with host proteins involved in the control of nuclear transcription and nucleoside biosynthesis as well as those interacting stably or transiently with the cytoplasmic protein degradation apparatus. Our approach is broadly applicable to elucidating pathogen-host interactomes, providing high-certainty identification of interactors by their direct access during cycling infection. Understanding the pathophysiological consequences of these associations is likely to provide strategic targets for antiviral intervention.

### Keywords
I-dirt, Vif, Ap-ms, Env, Protein interactions

### Affiliations
Dept. of Molecular Biology, Princeton, NJ USA 08544
Molecular Biology, Princeton University

### Submitter
Todd Greco

### Lab Head
Dr Ileana Cristea
Dept. of Molecular Biology, Princeton, NJ USA 08544


