### Accession
PXD010117

### Title
Dissecting the malaria interactome

### Description
To identify malaria protein interactions systematically we combined blue native polyacrylamide electrophoresis fractionation with quantitative mass spectrometry and machine learning. Our integrative approach identified over twenty thousand protein interactions in Plasmodium schizonts, organized into 600 complexes, the majority of which are novel, and generated a high-confidence Plasmodium protein interaction network.

### Sample Protocol
Purified Plasmodium schizonts (wild type or expressing an HA-tagged protein) were lysed in 50 mM Tris pH 8, 150 mM NaCl, 1 mM EDTA, containing Halt protease and phosphatase inhibitor cocktail (Thermo Scientific), supplemented with 0.1 or 1% NP-40, as previously described [Pardo et al, 2010]. Anti-HA antibody (Roche) was coupled to Protein G Dynabeads (Life Technologies). Cleared lysates were incubated with anti-HA coupled Dynabeads for 2 hours at 4 degreesC. After removing the supernatant beads were washed four times with IPP150 buffer (Pardo, 2010) and then 3 times with 50 mM ammonium bicarbonate. Beads were then resuspended in 50 mM ammonium bicarbonate and 1 ug of trypsin (sequencing grade, Roche) was added. Digestion was carried out at 37 degrees C overnight with constant shaking. Peptide solutions were recovered from the beads, supplemented with acetonitrile to 60% final concentration and filtered through a 0.65 μm pore membrane plate (Multiscreen HTS DV, Millipore). Peptides were then dried, reduced with 40 mM TCEP for 15 minutes at room temperature, acidified with formic acid at 0.5% final concentration and frozen until further analysis.  LC-MS/MS analysis  Peptides from immunoprecipitation experiments were analysed by online nanoLC-MS/MS on an Orbitrap Fusion Tribrid mass spectrometer coupled with an Ultimate 3000 RSLCnano System. Samples were first loaded and desalted on a nanotrap (100 µm id x 2 cm) (PepMap C18, 5 µ) at 10 µL/min with 0.1% formic acid for 10 min and then separated on an analytical column (75 µm id x 25 cm) (PepMap C18, 2µ) over a 120 min linear gradient of 5 – 40% B (B = 80% CH3CN/0.1% formic acid) at 300 nL/min, and the total cycle time was 150 min. The Orbitrap Fusion was operated in the Top Speed mode at 3 s per cycle. The survey scans (m/z 375-1500) were acquired in the Orbitrap at a resolution of 120,000 at m/z 200 (AGC 4x105 and maximum injection time 50 ms). The multiply charged ions (2-7) with a minimal intensity of 1x104 counts were subject to MS/MS in HCD with a collision energy at 30% and an isolation width of 1.6 Th then detected in the linear ion trap (AGC 1x104 and maximum injection time 35 ms). Dynamic exclusion width was set at ± 10 ppm for 30 s.

### Data Protocol
Raw data files from HA immunoprecipitation experiments were analysed using MaxQuant (version 1.6.1.0). Trypsin was set as digestion mode with a maximum of two missed cleavages allowed. Main search peptide tolerance was set to 20 ppm, and MS/MS match tolerance set to 0.5 Da. Acetylation at the N-terminus, oxidation of methionine, and deamidation of asparagine or glutamine were set as variable modifications. Peptide and protein identifications were set at 1% FDR. Unique and razor peptides were used for quantification. Database searches were conducted against protein sequence databases from GeneDB (Logan-Klumpler et al., 2012) (P. berghei – 5019 sequences)   and additionally with mouse protein sequences from UniProt (2017)(Mus musculus - 16950 sequences). Proteins identified as reverse hits or mouse proteins were removed for further analysis. Proteins identified only in the bait sample and not in control or other baits samples were deemed to be specific interactors.

### Publication Abstract
Malaria represents a major global health issue, and the identification of new intervention targets remains an urgent priority. This search is hampered by more than one-third of the genes of malaria-causing Plasmodium parasites being uncharacterized. We report a large-scale protein interaction network in Plasmodium schizonts, generated by combining blue native-polyacrylamide electrophoresis with quantitative mass spectrometry and machine learning. This integrative approach, spanning 3 species, identifies &gt;20,000 putative protein interactions, organized into 600 protein clusters. We validate selected interactions, assigning functions in chromatin regulation to previously unannotated proteins and suggesting a role for an EELM2 domain-containing protein and a putative microrchidia protein as mechanistic links between AP2-domain transcription factors and epigenetic regulation. Our interactome represents a high-confidence map of the native organization of core cellular processes in Plasmodium parasites. The network reveals putative functions for uncharacterized proteins, provides mechanistic and structural insight, and uncovers potential alternative therapeutic targets.

### Keywords
Malaria, Berghei

### Affiliations
Wellcome Trust Sanger Institute
Functional Proteomics, Institute Cancer Research
&
Proteomic Mass Spectrometry, Wellcome Trust Sanger Institute

### Submitter
James Wright

### Lab Head
Dr Jyoti Choudhary
Wellcome Trust Sanger Institute


