### Accession
PXD011080

### Title
Building comprehensive MS-friendly databases for proteomic analysis of bacterial species of unknown genetic background

### Description
Computational approach to build non-redundant protein databases for strains with unknown genomic background

### Sample Protocol
MS-based proteomics using SDS-PAGE as fractionation method and LTQ-Orbitrap MS.

### Data Protocol
Sample from 8 strains were analyzed by MaxQuant using different structured databases for comparison

### Publication Abstract
In proteomics, peptide information within mass spectrometry (MS) data from a specific organism sample is routinely matched against a protein sequence database that best represent such organism. However, if the species/strain in the sample is unknown or genetically poorly characterized, it becomes challenging to determine a database which can represent such sample. Building customized protein sequence databases merging multiple strains for a given species has become a strategy to overcome such restrictions. However, as more genetic information is publicly available and interesting genetic features such as the existence of pan- and core genes within a species are revealed, we questioned how efficient such merging strategies are to report relevant information. To test this assumption, we constructed databases containing conserved and unique sequences for 10 different species. Features that are relevant for probabilistic-based protein identification by proteomics were then monitored. As expected, increase in database complexity correlates with pangenomic complexity. However, <i>Mycobacterium tuberculosis</i> and <i>Bordetella pertussis</i> generated very complex databases even having low pangenomic complexity. We further tested database performance by using MS data from eight clinical strains from <i>M. tuberculosis</i>, and from two published datasets from <i>Staphylococcus aureus</i>. We show that by using an approach where database size is controlled by removing repeated identical tryptic sequences across strains/species, computational time can be reduced drastically as database complexity increases.

### Keywords
Databases, Proteogenomics, Bacterial strains, Pangenomics

### Affiliations
Dept. of Biochemistry, Universidade Fereal do Rio Grande do Norte
Dept. of Biochemistry, Federal University of Rio Grande do Norte

### Submitter
Gustavo De Souza

### Lab Head
Dr Gustavo de Souza
Dept. of Biochemistry, Universidade Fereal do Rio Grande do Norte


