### Accession
PXD035931

### Title
A seed-like proteome in oil-rich tubers

### Description
Fresh tubers were harvested from purple nutsedge and dried.

### Sample Protocol
Preparation of extracts Total extracts of purple nutsedge were obtained from deep frozen tubers ground in a precooled mortar with 1-2 g sea-sand and 5 ml grinding buffer (50 mM Tris pH 7.4, 10 mM KCl, 6 M urea, 200 µM phenylmethylsulfonyl fluoride). Afterwards, 1.5 ml of the suspension was transferred to a 2 ml reaction cup and solid SDS was added to reach a concentration of 5% (w/v) SDS (sodium dodecyl sulfate). Subsequently, the sample was incubated 1 h at 4°C under constant agitation, followed by a centrifugation for 15 min at 19,000 g at 4°C. A 200 µL fraction of the supernatant was recovered. For protein precipitation, 50 µL 50 % (v/v) TCA were added and the smples incubated for 30 min on ice. Then, 1 ml Acetone was added to the suspension and the fraction was stored for at least 24 h at -20°C for protein precipitation.  Peptide sample preparation The protein pellets were dissolved in 6 M urea and 5 % (w/v) SDS. Protein concentrations were determined with a Pierce BCA protein assay kit (Thermo Fisher Scientific, Waltham, MA, USA‎). 10 µg of protein for yellow nutsedge or 40 µg for purple nutsedge was run on a SDS-PAGE gel until they entered the separation gel. A single gel piece per sample containing all proteins was excised, tryptic digested and derivatised (Shevchenko et al., 2006). Peptides were desalted over an EmporeTM Octadecyl C18 47 mm extraction disks 2215 (Supelco, St. Paul, MN, USA) as described (Rappsilber et al., 2007).   LC-MS/MS analysis Dried peptide samples were reconstituted in 20 µl LC-MS sample buffer (2% acetonitrile, 0.1% formic acid). 3 µl of each sample were subjected to reverse phase liquid chromatography for peptide separation using an RSLCnano Ultimate 3000 system (Thermo Fisher Scientific). Therefore, peptides were loaded on an Acclaim PepMap 100 pre-column (100 µm x 2 cm, C18, 5 µm, 100 Å; Thermo Fisher Scientific) with 0.07% trifluoroacetic acid at a flow rate of 20 µL/min for 3 min. Analytical separation of peptides was done on an Acclaim PepMap RSLC column (75 µm x 50 cm, C18, 2 µm, 100 Å; Thermo Fisher Scientific) at a flow rate of 300 nL/min. The solvent composition was gradually changed within 94 min from 96 % solvent A (0.1 % formic acid) and 4 % solvent B (80 % acetonitrile, 0.1 % formic acid) to 10 % solvent B within 2 minutes, to 30 % solvent B within the next 58 min, to 45% solvent B within the following 22 min, and to 90 % solvent B within the last 12 min of the gradient. All solvents and acids had Optima grade for LC-MS (Thermo Fisher Scientific). Eluting peptides were on-line ionized by nano-electrospray (nESI) using the Nanospray Flex Ion Source (Thermo Fisher Scientific) at 1.5 kV (liquid junction) and transferred into a Q Exactive HF mass spectrometer (Thermo Fisher Scientific). Full scans in a mass range of 300 to 1650 m/z were recorded at a resolution of 30,000 followed by data-dependent top 10 HCD fragmentation at a resolution of 15,000 (dynamic exclusion enabled). LC-MS method programming and data acquisition was performed with the XCalibur 4.0 software (Thermo Fisher Scientific).

### Data Protocol
Calculation of relative intensity-based absolute quantification (riBAQ) values.  MS data was processed with Max Quant software version 1.6.2.10 or 2.0.3.1 (Cox and Mann, 2008, Cox et al., 2014) with standard settings except for: „Match between runs“ was turned on; „iBAQ“ was selected for label free quantification; FTMS recalibration was turned on. The three above mentioned nutsedge protein libraries were used. The libraries, the meta data file, raw data files, MaxQuant search files as well as ProteinGroup and Peptide search results created by MaxQuant are available on ProteomeXchange/PRIDE (Vizcaíno et al., 2014) under the identifiers PXD021894 and PXD031123. iBAQ values were determined with Max Quant 1.6.2.10. The data was further processed with Perseus software version 1.6.2.2 (Tyanova et al., 2016). Reverse hits, contaminants and proteins only identified by a modified peptide were removed from the data matrix. Then, all iBAQ values were divided by the total iBAQ value per sample and multiplied by 1000 yielding riBAQ in per mill. These values were the basis for all further data analysis.

### Publication Abstract
There are numerous examples of plant organs or developmental stages that are desiccation-tolerant and can withstand extended periods of severe water loss. One prime example are seeds and pollen of many spermatophytes. However, in some plants, also vegetative organs can be desiccation-tolerant. One example are the tubers of yellow nutsedge (Cyperus esculentus), which also store large amounts of lipids similar to seeds. Interestingly, the closest known relative, purple nutsedge (Cyperus rotundus), generates tubers that do not accumulate oil and are not desiccation-tolerant. We generated nanoLC-MS/MS-based proteomes of yellow nutsedge in five replicates of four stages of tuber development and compared them to the proteomes of roots and leaves, yielding 2257 distinct protein groups. Our data reveal a striking upregulation of hallmark proteins of seeds in the tubers. A deeper comparison to the tuber proteome of the close relative purple nutsedge (C. rotundus) and a previously published proteome of Arabidopsis seeds and seedlings indicates that indeed a seed-like proteome was found in yellow but not purple nutsedge. This was further supported by an analysis of the proteome of a lipid droplet-enriched fraction of yellow nutsedge, which also displayed seed-like characteristics. One reason for the differences between the two nutsedge species might be the expression of certain transcription factors homologous to ABSCISIC ACID INSENSITIVE3, WRINKLED1, and LEAFY COTYLEDON1 that drive gene expression in Arabidopsis seed embryos.

### Keywords
Tubers, Purple nutsedge

### Affiliations
University of Münster
Westphalian Wilhelms-University Münster Institute for Biology and Biotechnology of Plants Green Biotechnology Schlossplatz 8 48143 Münster

### Submitter
Till Ischebeck

### Lab Head
Dr Till Ischebeck
Westphalian Wilhelms-University Münster Institute for Biology and Biotechnology of Plants Green Biotechnology Schlossplatz 8 48143 Münster


