### Accession
PXD009464

### Title
Barley seed proteome analysis -  Genes for seed longevity in barley identified by genomic analysis on near isogenic lines

### Description
Genes controlling differences in seed longevity between two barley (Hordeum vulgare) accessions were identified by combining a quantitative genetics approach with ˈomicsˈ technologies in Near Isogenic Lines (NILs). The NILs were derived from crosses between the spring barley landraces L94 from Ethiopia and Cebada Capa from Argentina, which produce short-lived and long-lived seeds, respectively. The NILs harbor introgressions from Cebada Capa in four QTLs for seed longevity on 1H and 2H in the background of L94. A label-free proteome analysis was performed on mature, non aged seeds of the two parental lines and the L94 NILs.

### Sample Protocol
Samples of approximately 100 mature seeds per genotype were ground with a coffee grinder (Siemens, MC23200GB). 200 400 mg of ground material were transferred to 2 mL Safe Lock Eppendorf® tubes and whole proteome samples (50 µg) were processed using the FASP method (Wisniewski et al., 2009), as described in detail in Hartl et al. (2015). Cysteines were alkylated by incubation with 55 mM chloroacetamide in urea (UA) buffer for 30 min at room temperature in the dark, followed by three washing steps with UA buffer. Incubation with LysC in UA buffer (1:50 enzyme to protein ratio) was performed overnight at room temperature. The samples were diluted with ABC buffer (50 mM ammonium bicarbonate) to a final urea concentration of 2 M. Trypsin digestion (1:100 enzyme-to-protein ratio) was performed for four hours at room temperature. The samples were passed through the filter by centrifugation at 14.000 g for 10 min and residual peptides were eluted with 50 µl ABC buffer. Formic acid was added to a final concentration of 0.5%. Peptides were desalted and pre fractionated prior LC-MS/MS analysis into three fractions using the Empore Styrene Divenylbenzene Reversed Phase Sulfonate material (SDB-RPS, 3M), as described in detail by Kulak et al. (2014).Dried peptides were re dissolved in 2% acetonitrile (ACN), 0.1% trifluoroacetic acid (TFA) for analysis, and adjusted to a final concentration of 0.1 µg/µl. Samples were analyzed using an EASY-nLC 1000 (Thermo Fisher) coupled to a Q Exactive TM Plus mass spectrometer (Thermo Fisher). Peptides were separated on 16 cm frit-less silica emitters (New Objective, 0.75 µm inner diameter), packed in-house with reversed-phase ReproSil-Pur C18 AQ 1.9 µm resin (Dr. Maisch). Peptides (0.5 µg) were loaded on the column and eluted for 120 min using a segmented linear gradient of 0% to 95% solvent B (solvent A: 5% ACN, 0.5% formic acid (FA); solvent B: 100% ACN, 0.5% FA) at a flow rate of 250 nL/min. Mass spectra were acquired in data dependent acquisition mode using a topN (N=15) method. MS spectra were acquired in the Orbitrap analyzer with a mass range of 300 1750 m/z at a FWHM (full width at half maximum) resolution of 70.000 and an AGC target value of 3x106 ions. Precursors were selected with an isolation window of 1.3 m/z. HCD fragmentation was performed at a normalized collision energy of 25. MS/MS spectra were acquired applying an AGC target value of 105 ions at an FWHM resolution of 17.500 and a fixed first mass of m/z 100. Singly positively charged peptides, peptides with a charge state greater than 6, or with unassignable charge state were excluded from fragmentation for MS/MS, dynamic exclusion for 30 s prevented repeated selection of precursors.

### Data Protocol
Raw data were processed using the MaxQuant software (version 1.5.2.8, http://www.maxquant.org/) (Cox and Mann, 2008) with label-free quantification (LFQ) and iBAQ enabled (Cox et al., 2014). MS/MS spectra were searched by the Andromeda search engine against the PGSB barley genome database (http://pgsb.helmholtz-muenchen.de/plant/barley/). Sequences of 248 common contaminant proteins and decoy sequences were automatically added during the search. Trypsin was chosen as proteolytic enzyme used for protein digestion and a maximum of two missed cleavages was allowed. The minimal peptide length was set to seven amino acids. Carbamidomethylation of cysteine residues was set as fixed, oxidation of methionine and protein N-terminal acetylation as variable modifications. Peptide-spectrum-matches and proteins were retained, if they were below a false discovery rate of 1%. Subsequent quantitative statistical analyses were performed in Perseus (version 1.5.2.6, http://www.maxquant.org/; Cox and Mann, 2012).

### Publication Abstract
Genes controlling differences in seed longevity between 2 barley (Hordeum vulgare) accessions were identified by combining quantitative genetics "omics" technologies in near isogenic lines (NILs). The NILs were derived from crosses between the spring barley landraces L94 from Ethiopia and Cebada Capa from Argentina. A combined transcriptome and proteome analysis on mature, nonaged seeds of the 2 parental lines and the L94 NILs by RNA-sequencing and total seed proteomic profiling identified the UDP-glycosyltransferase MLOC_11661.1 as candidate gene for the quantitative trait loci on 2H, and the NADP-dependent malic enzyme (NADP-ME) MLOC_35785.1 as possible downstream target gene. To validate these candidates, they were expressed in Arabidopsis under the control of constitutive promoters to attempt complementing the T-DNA knockout line nadp-me1. Both the NADP-ME MLOC_35785.1 and the UDP-glycosyltransferase MLOC_11661.1 were able to rescue the nadp-me1 seed longevity phenotype. In the case of the UDP-glycosyltransferase, with high accumulation in NILs, only the coding sequence of Cebada Capa had a rescue effect.

### Keywords
Barley, Seed, Longevity, Label-free quantification

### Affiliations
Institute of Plant Biology and Biotechnology University of Muenster Germany
University of Muenster

### Submitter
Iris Finkemeier

### Lab Head
Dr Prof. Iris Finkemeier
Institute of Plant Biology and Biotechnology University of Muenster Germany


