### Accession
PXD035007

### Title
Endoxifen Downregulates AKT Phosphorylation Through Protein Kinase C Beta 1 in ER+/HER2- Breast Cancer

### Description
In phase I/II clinical trials, Z-endoxifen demonstrated substantial oral bioavailability and promising antitumor activity in endocrine-refractory estrogen-receptor positive breast cancer (ER+ BC) and other solid tumors, with plasma concentrations reportedly as high as 5 ï�­M. Therefore, we explored the potential mechanisms of Z-endoxifen antitumor activity that extends beyond ERÎ± inhibition. In estrogen unstimulated aromatase-expressing ER+/human epidermal growth factor 2 receptor negative (HER2-) MCF7AC1 BC cells, Z-endoxifen at 5 ï�­M, but not ERï�¡-targeting 0.01 and 0.1 ï�­M concentrations inhibited growth and induced apoptosis, suggesting an ERÎ±-independent effect. Utilizing an unbiased mass spectrometry approach, we explored Z-endoxifen effects on other signaling pathways. Z-endoxifen at 5 ÂµM profoundly altered the phosphoproteome with minimal impact on total proteome. Computational analysis revealed Protein kinase C beta (PKCï�¢) and AKT1 as the prevalent upstream kinases for Z-endoxifen-downregulated protein phosphorylations. Notably, in ER+/HER2- BC models, Z-endoxifen at 5 ï�­M attenuated AKTSer473 and AKT substrates in vitro in the presence of insulin and PKC agonist PMA and in vivo.  Further, Z-endoxifen inhibited PKCï�¢1 kinase activity compared to other PKC isoforms in vitro and bound to PKCÎ²1. While PMA stimulated PKCï�¢1Ser661 phosphorylation correlated with AKTSer473 and AKT substrate phosphorylation, Z-endoxifen at 5 ï�­M uniquely blocked these effects and induced PKCÎ²1 protein degradation. siRNA-mediated PKCï�¢1 knockdown attenuated AKTSer473 phosphorylation, suggesting PKCÎ²1-mediated suppression of AKT signaling by Z-endoxifen. Further, Z-endoxifen at 5 ï�­M replicates the pan-AKT inhibitor MK-2206 effects on apoptosis. These findings implicate PKCÎ²1 as a novel Z-endoxifen substrate responsible for suppressing AKT signaling and inducing apoptosis in breast cancer.

### Sample Protocol
- Cell lysis and in-solution trypsin digestion  Following treatment of MCF7AC1 cells with vehicle control or 0.01, 0.1 and 5 ÂµM Z-endoxifen for 24 hours in CSS media, cells were harvested and lysed in 8 M urea buffer (8 M urea, 20 mM HEPES pH 8.0, 1 mM sodium orthovanadate, 2.5 mM sodium pyrophosphate, 1 mM Î²-glycerophosphate, and 5 mM sodium fluoride), followed by sonication, and centrifugation at 15,000 x g at 4Â°C for 20 min. 2 mg of protein lysates from each treatment condition was used for digestion with trypsin. Briefly, the protein lysates were reduced with 5 mM dithiothreitol at 37Â°C for 1 hour and alkylated with 10 mM iodoacetamide at room temperature in dark for 30 min. The protein lysates were then diluted in 20 mM HEPES pH 8.0 to a final concentration < 2 M urea and digested with TPCK-treated trypsin (Worthington Biochemical Corp. Lakewood, NJ) overnight at room temperature. Digested peptides were acidified with 20%trifluoroacetic acid (TFA) to a final concentration of 1% TFA. The tryptic peptides were desalted using SepPak C18 cartridge (Waters Corporation, Milford, MA).  - Tandem Mass Tag (TMT) labeling of peptides and basic reversed-phase liquid chromatography (bRPLC) fractionation The lyophilized tryptic peptides were reconstituted in 150 Âµl 100 mM triethylammonium bicarbonate (TEABC) and measured with peptide BCA assay (Thermo Scientific). 1 mg peptides from each sample in the final volume of 100 ul 1mM TEABC were mixed with 1 mg Âµg TMTpro reagent that was dissolved in 20 Âµl anhydrous acetonitrile. After 1 hour incubation at RT, 10 Î¼L of 5% hydroxylamine was added and incubated for 15 min at RT to quench the labeling reaction. Peptides labeled by different TMT reagents were then mixed and dried with Speed-Vac, reconstituted in 20 mM ammonium formate, and fractionated by high-pH reversed-phase liquid chromatography on Dionex Ultimate 3000 (Thermo Scientific). Peptides (12 mg) were separated on a 4.6 mm Ã— 50 cm Ã— 3.5 Î¼m Xbridge column (Waters) with a 2-hour gradient from 2 to 40% mobile phase B (MPB). Mobile phase A was composed of 20 mM ammonium formate in water, and MPB was composed of 20 mM ammonium formate in 80% acetonitrile. A total of 96 fractions were collected and concatenated into 24 fractions. A 20-Î¼g equivalent of each fraction was set aside for global proteome analysis, and the rest of each sample was concentrated into 12 fractions and dried before phosphopeptide enrichment. - Phosphopeptide enrichment Each fraction was reconstituted in 1 ml of 80% acetonitrile in 0.1% TFA. Phosphopeptides were enriched using the immobilized metal affinity chromatography (IMAC) approach. In brief, nickel-nitrilotriacetic (Ni-NTA) superflow agarose beads were stripped of nickel with 100 mM EDTA, incubated with 10 mM FeCl3 solution and equilibrated in 80% ACN/0.1%TFA. 10 Î¼l IMAC beads were mixed with each fractionated peptide in 80% acetonitrile/0.1% TFA and rotated for 30 min at RT. Subsequently, incubated IMAC beads were washed with 500 Î¼l 80% ACN/0.1%TFA four times and 500 Î¼L 0.1% FA one time. Phosphopeptides were eluted from IMAC beads with 200 Î¼L of 500 mM dibasic sodium phosphate (pH 7.0) for three times. The eluted phosphopeptides were desalted with C18 Stage Tips and Speed-Vac dried.  - LC-MS/MS analysis  The peptide fractions were loaded on a 2 cm trap column (Acclaim PepMap 100, C18, 5 Âµm particle size, 100 Âµm i.d. 100 Ã… pore size, Thermo Scientific, San Jose, CA) using 0.1% formic acid with a flow rate 20 Âµl/min for 4 minutes. The peptides were separated on a 50 cm analytical column (Acclaim PepMap 100, C18, 2 Âµm particle size, 75 Âµm i.d. 100 Ã… pore size, Thermo Scientific, San Jose, CA) with a 135 min gradient from 3% to 40% acetonitrile in 0.1% formic acid at a flow rate of 0.3 Âµl/min. The spray voltage was set to 2.3 kV while capillary temperature was set to 275Â°C. The samples were analyzed on an Orbitrap Fusio Lumos mass spectrometer (Thermo Scientific, Bremen, Germany). The MS instrument was operated in data-dependent acquisition mode. A survey full scan MS (from 350â€“1,500 m/z) was acquired in the Orbitrap with resolution 120,000 at m/z 200 with a maximum AGC target value of 800,000 ions. The data-dependent MS/MS was carried out using Top Speed method with a duty cycle of 2 seconds. Singly charge precursor ions were excluded while precursor ions with charge states 2-7 were sequentially isolated and fragmented in the higher-energy collisional dissociation (HCD) cell using 34% normalized collision energy (NCE). The maximum ion injection time for MS and MS/MS were set to 50 ms. Fragment ion spectra were detected in Orbitrap mass analyzer with a resolution 30,000 at m/z 200. Dynamic exclusion was enabled one event of fragmentation followed by exclusion of the precursor for next 45 seconds within 7 ppm of the selected m/z. For all measurements with the Orbitrap detector, a lock-mass ion from ambient air (m/z 445.120025) was used for internal calibration.

### Data Protocol
Proteome Discoverer software suite (v 2.5; Thermo Fisher Scientific, San Jose, CA) was used for quantitation and database searches. The MS/MS data were searched using the SEQUEST search algorithm against a Human Uniport protein database supplemented with frequently observed contaminants. Search parameters included trypsin as a protease with full specificity and a maximum of two allowed missed cleavages; carbamidomethylation of cysteine and TMTpro tag (+304.207â€‰Da) on lysine residues or peptide N-terminus as a fixed modification; oxidation at methionine and phosphorylation at serine/threonine/tyrosine as variable modifications. The precursor tolerance was set at 10 ppm, while the fragment match tolerance was set to 0.02 Da. The PSMs, peptides and proteins were filtered at 1% false discovery rate cut-off calculated using target-decoy database searches. The probability of an identified phosphorylation of specific Ser/Thr/Tyr residue on each identified phosphopeptide was determined from the PhosphoRS algorithm.

### Publication Abstract
Endoxifen, a secondary tamoxifen metabolite, is a potent antiestrogen exhibiting estrogen receptor alpha (ER&#x3b1;) binding at nanomolar concentrations. Phase I/II clinical trials identified clinical activity of Z-endoxifen (ENDX), in endocrine-refractory metastatic breast cancer as well as ER&#x3b1;+ solid tumors, raising the possibility that ENDX may have a second, ER&#x3b1;-independent, mechanism of action. An unbiased mass spectrometry approach revealed that ENDX concentrations achieved clinically with direct ENDX administration (5&#x2009;&#xb5;M), but not low concentrations observed during tamoxifen treatment (&lt;0.1&#x2009;&#xb5;M), profoundly altered the phosphoproteome of the aromatase expressing MCF7AC1 cells with limited impact on the total proteome. Computational analysis revealed protein kinase C beta (PKC&#x3b2;) and protein kinase B alpha or AKT1 as potential kinases responsible for mediating ENDX effects on protein phosphorylation. ENDX more potently inhibited PKC&#x3b2;1 kinase activity compared to other PKC isoforms, and ENDX binding to PKC&#x3b2;1 was confirmed using Surface Plasma Resonance. Under conditions that activated PKC/AKT signaling, ENDX induced PKC&#x3b2;1 degradation, attenuated PKC&#x3b2;1-activated AKT<sup>Ser473</sup> phosphorylation, diminished AKT substrate phosphorylation, and induced apoptosis. ENDX's effects on AKT were phenocopied by siRNA-mediated PKC&#x3b2;1 knockdown or treatment with the pan-AKT inhibitor, MK-2206, while overexpression of constitutively active AKT diminished ENDX-induced apoptosis. These findings, which identify PKC&#x3b2;1 as an ENDX target, indicate that PKC&#x3b2;1/ENDX interactions suppress AKT signaling and induce apoptosis in breast cancer.

### Keywords
Endoxifen, Pkc-beta-1, Breast cancer, Phosphoproteome, Akt, Mass spectrometry, Apoptosis

### Affiliations
Department of Laboratory Medicine and Pathology, Mayo Clinic, Rochester, MN 55905
Laboratory Medicine and Pathology Mayo Clinic USA

### Submitter
Akhilesh Pandey

### Lab Head
Dr Akhilesh Pandey
Laboratory Medicine and Pathology Mayo Clinic USA


