### Accession
PXD034053

### Title
Consequences of loss of spinophilin on striatal mGluR5 phosphorylation and interactome

### Description
Constitutive knockout of the obsessive-compulsive disorder-associated protein, SAPAP3, results in repetitive motor dysfunction, such as excessive grooming, caused by increased mGluR5 activity in striatal medium spiny neurons (MSNs). However, signaling mechanisms that mediate mGluR5-dependent grooming dysfunction are not fully understood. Here, we investigate the function of the striatal signaling hub protein, spinophilin, in regulating mGluR5 phosphorylation and protein interactions, which may predict spinophilin’s role in regulating mGluR5-dependent grooming dysfunction.

### Sample Protocol
Spinophilin. Grooming behavior was measured in Noldus Phenotyper Cages (see below) for 2.5 hours in 4-6 month old SAPAP3 WT or KO mice. After behavior recording, whole striata were dissected, flash frozen, and stored at -80ºC. Lysates were prepared by weighing and homogenizing (as described above) in low-ionic lysis buffer (45 L/mg of tissue) and 500 L of striatal lysate (1.5 – 2.0 mg total protein) was added to microfuge tube and 2 g of sheep-spinophilin primary antibody (Thermo; PA5-48102) was spiked into the lysate and tubes incubated overnight at 4oC with rotation. The following morning, 15 L of Protein G magnetic beads was spiked into the sample and incubated two-hours at 4oC with rotation. Beads were washed as described above and resuspended in 40 L of 2X Laemmli sample buffer.  mGluR5. Dissected whole striatum was immediately homogenized (as described above) in 1 mL of low-ionic lysis buffer and 750 L of striatal lysate was added to a microfuge tube and 1 g of rabbit-mGluR5 primary antibody (Millipore; AB5675) was spiked into the lysate and tubes incubated for 2.5 hours at 4oC with rotation, then 15 L of Protein G magnetic beads was added, and samples were incubated for an additional 2.5 hours at 4oC with rotation. Samples were placed on magnetic tube rack and the lysate was transferred into a new microfuge tube where an aliquot of lysate was taken to measure immunodepletion and an additional 1g of rabbit-mGluR5 primary antibody was spiked into the lysate and samples were incubated overnight at 4oC with rotation. The next day, another aliquot of lysates was taken to measure immunodepletion. For each round of IP, Protein G magnetic beads were washed three times with immunoprecipitation wash buffer (150 mM NaCl, 50 mM Tris-HCl pH 7.5, 0.5% (v/v) Triton X-100) and washed beads were resuspended in 30 L of 1X PBS. Beads from the first and second IP were combined after confirming the sequential IPs sufficiently depleted mGluR5 from lysate via immunoblot (see below) analysis (Figure S9).TMT-LC/MS  Sample Preparation. After IP and wash steps, beads (3 WT and 3 KO in the first batch and 2 WT and 2 KO in the second) were submitted to the Center for Proteome analysis. 30 µL of 8 M Urea in 100 mM Tris pH 8.5 was added and proteins were reduced with 5 mM tris(2-carboxyethyl)phosphine hydrochloride (TCEP, Sigma-Aldrich Cat No:  C4706) for 30 minutes at room temperature. The resulting free cysteine thiols were alkylated with 10 mM chloroacetamide (CAA, Sigma Aldrich Cat No: C0267) for 30 min at room temperature in the dark. Samples were diluted with 50 mM Tris.HCl, pH 8.5 to a final urea concentration of 2 M and treated with 1 µL PNGaseF (New England Biolabs, Cat No P0705L) for 2 hrs at 35 ºC.  Samples were then digested overnight at 35 ºC with 0.5 µg Trypsin/Lys-C (Mass Spectrometry grade, Promega Corporation, Cat No: V5072).   Peptide Purification and Labeling. Digestions were acidified with trifluoracetic acid (TFA, 0.5% v/v) and desalted on SPE columns  (Pierce Cat no 89870) with a wash of 300 µL 0.1% TFA followed by elution in 70% acetonitrile 0.1% formic acid (FA).   TMT labeling. Peptides were dried by speed vacuum and resuspended in 24 µL of 50 mM triethylammonium bicarbonate pH 8.0 (TEAB) and labeled for two hours at room temperature with 0.2 mg of Tandem Mass Tag (TMT) reagent which was resuspended in 24 µL acetonitrile (Thermo Fisher Scientific, TMT™ Isobaric Label Reagent Set; Cat No 90111, lot no. WC306775 for 6-plex and WD320959 for 4-plex) [10]. Labelling reactions were quenched by adding 0.2% hydroxylamine (final v/v) to the reaction mixtures at room temperature for 15 minutes. Labeled peptides were then combined, mixed, and dried by speed vacuum. After drying, samples were resuspended in 0.1% TFA and desalted using Waters SepPak cartridges (Waters™ WAT054955) with a wash of 1 mL 0.1% TFA followed by elution in 70% acetonitrile 0.1% formic acid (FA). The specific TMT label corresponding to each WT and KO sample are designated in Figure S9.

### Data Protocol
Mass spectrometry data analysis. Resulting RAW files were analyzed in Proteome Discover™ 2.5 (Thermo Fisher Scientific, RRID: SCR_014477) with a Mus musculus UniProt FASTA (both reviewed and unreviewed sequences) plus common contaminants (49922 total sequences). Quantification methods utilized isotopic impurity levels available from Thermo Fisher Scientific. SEQUEST HT searches were conducted with a maximum number of 3 missed cleavages; precursor mass tolerance of 10 ppm; and a fragment mass tolerance of 0.02 Da. Static modifications used for the search were carbamidomethylation on cysteine (C) residues. Dynamic modifications used for the search were oxidation of methionines, TMT label on the N-termini of peptides, TMT label on lysine (K) residues, phosphorylation (S, T, Y) and deamidation of N (max 3 dynamic mods). Dynamic Protein terminus modifications allowed were: acetylation (N-terminus), Met-loss or Met-loss plus acetylation (N-terminus). Percolator False Discovery Rate was set to a strict setting of 0.01 and a relaxed setting of 0.05. IMP-ptm-RS node was used for all modification site localization scores. In the consensus workflows, data were normalized using a fasta file of all 3 GRM5 isoforms. Co-isolation thresholds of 50% and average reporter ion S/N cutoffs of 10 were used for quantification. Lot specific isotopic impurity levels were corrected for. Resulting normalized abundance values for each sample type, abundance ratio and log2(abundance ratio) values; and respective p-values (t-test) from Proteome Discover™ were exported to Microsoft Excel.

### Publication Abstract
None

### Keywords
Striatum, Mouse, Phosphatase targeting proteins, Tmt-lc/ms

### Affiliations
Indiana University School of Medicine
Center for Proteome Analysis, Indiana University School of Medicine, Indianapolis, IN

### Submitter
Emma Doud

### Lab Head
Dr Amber L. Mosley
Center for Proteome Analysis, Indiana University School of Medicine, Indianapolis, IN


