### Accession
PXD034920

### Title
Actinomycin D treated cytotoxic T lymphocytes

### Description
Effector cytotoxic T lymphocytes (CTLs) are critical for ridding the body of infected or cancerous cells. Recognition of an antigenic ligand by the CTL’s T cell receptor (TCR) triggers a signalling cascade that ultimately results in the targeted release of cytolytic granules and/or secretion of cytokines and chemokines to alert and recruit additional immune cells. These signalling cascades are capable of initiating transcription, translation, and cytoskeletal rearrangements. While previous work has demonstrated how translation and intracellular reorganisation contribute to CTL effector responses, the role of transcription is less well studied. To address this, we examined the impact of blocking transcription on the CTL proteome during TCR stimulation. These data demonstrated a strong transcriptional requirement for expression of cytokines and chemokines but not cytolytic molecules. Together with functional studies, these data reveal differential molecular control of the cell-cell communication and cytolytic functions of effector CTLs. CTLs exhibit complete and persistent priming for cytolytic activity prior to target cell encounter, but they require de novo transcription to recruit additional immune cells that amplify the response.

### Sample Protocol
To generate effector CTLs in vitro, splenocytes from OT-I TCR transgenic Rag1-deficient mice were cultured with 10 nM SIINFEKL peptide (Cambridge Bioscience) for 3 days. Media was then changed and cells expanded daily with IL2. On day 7, CTLs were cultured on plates uncoated or coated with 1 μg/mL anti-CD3e (clone 145-2C11, BD Biosciences) in the presence of 1:200 DMSO control or 5 μg/mL actinomycin D for 4 hours. Three million CTLs per condition were washed twice in ice-cold PBS and flash-frozen. Dataset includes 4 biological replicates  Cell pellets were lysed in 5% SDS, 50 mM TEAB pH 8.5, 10 mM TCEP at room temperature under agitation (5 min, 1200 rpm on shaker), boiled (5 min, 500 rpm on shaker), then sonicated with a BioRuptor (30 sec on, 30 sec off x30 cycles). Protein concentration was determined using EZQ protein quantitation kit (Invitrogen) as per manufacturer instructions. Lysates were alkylated with 20 mM iodoacetamide for 1 hour (room temperature in the dark), before protein clean up by the S-Trap method (Zougman et al., 2014) as per manufacturer protocol (Protifi). Briefly, phosphoric acid (to final concentration 1.2%) then S-Trap binding buffer (90% methanol, 100 mM TEAB, pH 7.1) at a ratio 1:7 v:v lysate:binding buffer was added to sample lysates containing 100-300 µg total protein.  Acidified lysates were loaded to S-Trap mini columns and spun (4,000 g, 30 s) until all SDS lysate/S-Trap buffer had passed through the column. Proteins were digested on column in digest buffer (50 mM sodium bicarbonate) containing Trypsin at a 1:20 enzyme to protein ratio (2 hrs, 47oC without shaking). Peptides were eluted by spinning in digest buffer (1000g, 1 min), then 0.2% aqueous formic acid (1000g, 1 min), then 50% acetonitrile 0.2% formic acid (4000g, 1 min). Peptide concentration was quantified using CBQCA protein quantitation kit (Invitrogen) as per manufacturer protocol. Peptides were dried in vacuo then dissolved in 5% Formic Acid for analysis by LC-ES-MS/MS.  2 µg peptide was analysed per sample. Samples were injected onto a nanoscale C18 reverse-phase chromatography system (UltiMate 3000 RSLC nano, Thermo Scientific) then electrosprayed into an Q Exactive HF-X Mass Spectrometer (Thermo Scientific). For liquid chromatography buffers were as follows:  buffer A (0.1% formic acid in Milli-Q water (v/v)) and buffer B (80% acetonitrile and 0.1% formic acid in Milli-Q water (v/v). Sample were loaded at 15 μL/min onto a trap column (100 μm × 2 cm, PepMap nanoViper C18 column, 5 μm, 100 Å, Thermo Scientific) equilibrated in 0.1% trifluoroacetic acid (TFA). The trap column was washed for 3 min at the same flow rate with 0.1% TFA then switched in-line with a Thermo Scientific, resolving C18 column (75 μm × 50 cm, PepMap RSLC C18 column, 2 μm, 100 Å). The peptides were eluted from the column at a constant flow rate of 300 nl/min with a linear gradient from 3% buffer B to 6% buffer B in 5 min, then from 6% buffer B to 35% buffer B in 115 min, and finally to 80% buffer B within 7 min. The column was then washed with 80% buffer B for 4 min and re-equilibrated in 3% buffer B for 15 min. two blanks were run between each sample to reduce carry-over. The column was kept at a constant temperature of 50oC.   The data was acquired using an easy spray source operated in positive mode with spray voltage at 1.9 kV, the capillary temperature at 250oC and the funnel RF at 60.  The MS was operated in DIA mode as reported earlier (Muntel et al., 2019) with some modifications. A scan cycle comprised a full MS scan (m/z range from 350-1650, with a maximum ion injection time of 20 ms, a resolution of 120 000 and automatic gain control (AGC) value of 5x106).  MS survey scan was followed by MS/MS DIA scan events using the following parameters: default charge state of 3, resolution 30.000, maximum ion injection time 55 ms, AGC 3x106, stepped normalized collision energy 25.5, 27 and 30, fixed first mass 200 m/z. The inclusion list (DIA windows) and windows widths are shown in table X.   Data for both MS and MS/MS scans were acquired in profile mode. Mass accuracy was checked before the start of samples analysis.  References:  Muntel, J., Kirkpatrick, J., Bruderer, R., Huang, T., Vitek, O., Ori, A., and Reiter, L. (2019). Comparison of Protein Quantification in a Complex Background by DIA and TMT Workflows with Fixed Instrument Time. J Proteome Res 18, 1340-1351.  Zougman, A., Selby, P.J., and Banks, R.E. (2014). Suspension trapping (STrap) sample preparation method for bottom-up proteomics analysis. Proteomics 14, 1006-1000.

### Data Protocol
The data were processed, searched and quantified with Spectronaut (Bruderer et al., 2015) 14 using the directDIA option. Cleavage Rules were set to Trypsin/P, Peptide maximum length was set to 52 amino acids, Peptide minimum length was set to 7 amino acids and Missed Cleavages set to 2. Calibration Mode was set to Automatic. Search criteria included carbamidomethylation of cysteine as a fixed modification, as well as oxidation of methionine, deamidation of asparagine and glutamine and acetylation (protein N-terminus) as variable modifications.  The FDR threshold was set to 1% Q-value at both the Precursor and Protein level. The single hit definition was to Stripped sequence.  The directDIA data were searched against a hybrid database from databases in July 2019 Uniprot release. This consisted of all manually annotated mouse SwissProt entries, combined with mouse TrEMBL entries with protein level evidence available and a manually annotated homologue within the human SwissProt database.  The Major Group Quantity was set to the Sum of peptide quantity and the Minor Group Quantity was set to the Sum of the precursor quantity, Cross Run Normalization was disabled.  References:  Bruderer, R., Bernhardt, O.M., Gandhi, T., Miladinovic, S.M., Cheng, L.Y., Messner, S., Ehrenberger, T., Zanotelli, V., Butscheid, Y., Escher, C., et al. (2015). Extending the limits of quantitative proteome profiling with data-independent acquisition and application to acetaminophen-treated three-dimensional liver microtissues. Mol Cell Proteomics 14, 1400-1410.

### Publication Abstract
None

### Keywords
Actinomycin d, Transcription, Mouse, T cell, Dia

### Affiliations
Cambridge Institute for Medical Research, University of Cambridge, Cambridge Biomedical Campus, CB2 0XY, UK
School of Life Sciences, University of Dundee

### Submitter
Julia Marchingo

### Lab Head
Dr Gillian M. Griffiths
Cambridge Institute for Medical Research, University of Cambridge, Cambridge Biomedical Campus, CB2 0XY, UK


