### Accession
PXD016174

### Title
Sgt1 interactome analysis under proteostatic stress and steady state conditions

### Description
Buildup of misfolded proteins are prevented by chaperone-assisted refolding and/or proteasome dependent degradation. The coordination of these two protein quality control (PQC) systems is not fully understood. We identified the essential Hsp90 co-chaperone Sgt1 as a new member of a general PQC linking folding and degradation. Upon proteostatic stress, e.g. heat shock, Sgt1 accumulates in an Hsp90- and proteasome dependent manner at an until now unknown spatial PQC compartment in both yeast and human cells. In order to find further components of this new PQC compartment we performed pull down experiments and a protein interaction analysis of cells expressing either GFP tagged Sgt1 (or GFP alone, as a negative control) that were either grown at 30Â°C or under heat shock conditions (42Â°C, 30 minutes).

### Sample Protocol
Cells were grown in synthetic media to an OD600 of 0.7-0.8. For induction of GFP-Sgt1 foci cells were shifted to 42Â°C for 30 minutes. Fifty OD600 of exponentially growing cells were harvested, washed twice in ice-cold water and once in ice-cold IP buffer (115 mM KCl, 5 mM NaCl, 2 mM MgCl2, 1 mM EDTA/NaOH pH 8.0, 20 mM HEPES/KOH pH 7.45). Cell were snap frozen and kept at -80Â°C. A volume of 200 Âµl glass beads, 1 mM DTT, 0.5 Âµg/ml Pefablock, 1x cOmplete Mini (Roche) were added (calculated for 1 ml final volume) and cells were opened by 10 times vortexing for 45 seconds with 1 min chilling on ice between the cycles. Lysates were transferred to a new tube and buffer was added to 1 ml final volume. Lysates were incubated with 0.5 % Nonidet P-40 (Roche) for 20 min on ice, followed by centrifugation at 16.000xg for 20 min at 4Â°C. 800 Âµl of pre-cleared cell lysate was incubated with 20 Âµl bed volume GFP-TrapA beads (Chromotek) to pull down GFP tagged proteins for 1 hour at 4Â°C on an over-head rotator. Beads were pelleted by centrifugation at 8.000 rpm and washed 4 times with IP buffer containing 0.5% Nonidet P-40. Beads were subjected for a final wash in TEAB buffer [50 mM] followed by 2 sequential rounds of elution in formic acid [1%]. Elutions were pooled and cysteines were reduced with dithiothreitol (Biomol) at 56Â°C for 30 min (10 mM in 50 mM HEPES (Biomol), pH 8.5), and further alkylated with iodoacetamide (Merck) at room temperature, in the dark for 30 min (20 mM in 50 mM HEPES, pH 8.5). For sample clean up and digestion, the SP3 protocol (1) was used and trypsin (sequencing grade, Promega) was added (enzyme to protein ratio 1:20) for overnight digestion at 37Â°C. Peptides were extracted and labelled with TMT10plex (2) Isobaric Label Reagent (ThermoFisher) according the manufacturerâ€™s instructions. For further sample clean up an OASISÂ® HLB ÂµElution Plate (Waters) was used. Offline high pH reverse phase fractionation was carried out on an Agilent 1200 Infinity high-performance liquid chromatography system, equipped with a Gemini C18 column (3 Î¼m, 110 Ã…, 100 x 1.0 mm, Phenomenex) (3).  An UltiMate 3000 RSLC nano LC system (Dionex) fitted with a trapping cartridge (Âµ-Precolumn C18 PepMap 100, 5 Âµm, 300 Âµm i.d. x 5 mm, 100 Ã…, Thermo Fisher) and an analytical column (Acclaim PepMapâ„¢ 100, 75 Âµm x 50 cm, naoViper column, Thermo Fisher). Trapping was carried out with a constant flow of solvent A (0.1% formic acid in water) at 30 ÂµL/min onto the trapping column for 6 minutes. Subsequently, peptides were eluted via the analytical column with a constant flow of 0.3 ÂµL/min with increasing percentage of solvent B (0.1% formic acid in acetonitrile) from 2% to 4% in 4 min, from 4% to 8% in 2 min, then 8% to 28% for a further 96 min, and finally from 28% to 40% in another 10 min. The outlet of the analytical column was coupled directly to a QExactive plus (Thermo Fisher) mass spectrometer using the proxeon nanoflow source in positive ion mode. The peptides were introduced into the QExactive plus via a Pico-Tip Emitter 360 Âµm OD x 20 Âµm ID; 10 Âµm tip (New Objective) and an applied spray voltage of 2.3 kV. The capillary temperature was set at 320Â°C. Full mass scan was acquired with mass range 375-1200 m/z in profile mode in the FT with resolution of 70000. The filling time was set at maximum of 250 ms with a limitation of 3x106 ions. Data dependent acquisition (DDA) was performed with the resolution of the Orbitrap set to 35000, with a fill time of 120 ms and a limitation of 2x105 ions. A normalized collision energy of 32 was applied. A loop count of 10 with count 1 was used and a minimum AGC trigger of 2e2 was set. Dynamic exclusion time of 30 s was used. The peptide match algorithm was set to â€˜preferredâ€™ and charge exclusion â€˜unassignedâ€™, charge states 1, 5 - 8 were excluded. MS2 data was acquired in profile mode.  1.  C. S. Hughes, S. Foehr, D. A. Garfield, E. E. Furlong, L. M. Steinmetz, J. Krijgsveld, Ultrasensitive proteome analysis using paramagnetic bead technology. Mol. Syst. Biol. 10, 757 (2014). 2.  T. Werner, G. Sweetman, M. F. Savitski, T. Mathieson, M. Bantscheff, M. M. Savitski, Ion coalescence of neutron encoded TMT 10-plex reporter ions. Anal. Chem. 86, 3594â€“3601 (2014). 3.  M. Reichel, Y. Liao, M. Rettel, C. Ragan, M. Evers, A. M. Alleaume, R. Horos, M. W. Hentze, T. Preiss, A. A. Millar, In planta determination of the mRNA-binding proteome of arabidopsis etiolated seedlings. Plant Cell. 28, 2435â€“2452 (2016).

### Data Protocol
IsobarQuant (1) and Mascot (v2.2.07) were used to process the acquired data, which was searched against Uniprot Saccharomyces cerevisiae proteome database (UP000002311). Carbamidomethyl (C) and TMT (K) (fixed modifications), Acetyl (N-term), Oxidation (M) and TMT10 (N-term) (variable modifications) were included into the search parameters. For the full scan (MS1) a mass error tolerance of 10 ppm and for MS/MS (MS2) spectra of 0.02 Da was set. Further parameters: Trypsin as protease, with maximum two missed cleavages, minimum peptide length of seven amino acids; at least two unique peptides for protein identification. False discovery rate on peptide and protein level was set to 0.01. The protein output files of IsobarQuant were processed using R (2). Raw TMT reporter ion signals (signal_sum columns) were batch-cleaned using the removeBatchEffect function from the limma package (3) and further normalized using the vsn package (variance stabilization normalization) (4). Proteins were tested for differential expression and called hits with a false discovery smaller 1% and a fold-change cut-off of 100%. 1.  H. Franken, T. Mathieson, D. Childs, G. M. A. Sweetman, T. Werner, I. TÃ¶gel, C. Doce, S. Gade, M. Bantscheff, G. Drewes, F. B. M. Reinhard, W. Huber, M. M. Savitski, Thermal proteome profiling for unbiased identification of direct and indirect drug targets using multiplexed quantitative mass spectrometry. Nat. Protoc. 10, 1567â€“1593 (2015). 2.  R. R Development Core Team, R: A Language and Environment for Statistical Computing (2011). 3.  M. E. Ritchie, B. Phipson, D. Wu, Y. Hu, C. W. Law, W. Shi, G. K. Smyth, limma powers differential expression analyses for RNA-sequencing and microarray studies. Nucleic Acids Res. 43, e47â€“e47 (2015). 4.  W. Huber, A. von Heydebreck, H. SÃ¼ltmann, A. Poustka, M. Vingron, Variance stabilization applied to microarray data calibration and to the quantification of differential expression. Bioinformatics. 18 Suppl 1, S96-104 (2002).

### Publication Abstract
In this paper, we show that the essential Hsp90 co-chaperone Sgt1 is a member of a general protein quality control network that links folding and degradation through its participation in the degradation of misfolded proteins both in the cytosol and the endoplasmic reticulum (ER). Sgt1-dependent protein degradation acts in a parallel pathway to the ubiquitin ligase (E3) and ubiquitin chain elongase (E4), Hul5, and overproduction of Hul5 partly suppresses defects in cells with reduced Sgt1 activity. Upon proteostatic stress, Sgt1 accumulates transiently, in an Hsp90- and proteasome-dependent manner, with quality control sites (Q-bodies) of both yeast and human cells that co-localize with Vps13, a protein that creates organelle contact sites. Misfolding disease proteins, such as synphilin-1 involved in Parkinson's disease, are also sequestered to these compartments and require Sgt1 for their clearance.

### Keywords
Hapi, Sgt1, Protein quality control, Saccharomyces cerevisiae, Heat shock

### Affiliations
EMBL
Department of Immunology and Microbiology Institute of Biomedicine Sahlgrenska Academy University of Gothenburg

### Submitter
Frank Stein

### Lab Head
Dr Thomas Nystroem
Department of Immunology and Microbiology Institute of Biomedicine Sahlgrenska Academy University of Gothenburg


