### Accession
PXD006142

### Title
Phosphoproteomic analysis of T cell receptor stimulation-induced changes in human primary CD4+CD25- T cells in the absence or presence of regulatory T cells.

### Description
This is an addition to dataset PXD004291. This data were only used for confirmation of certain phosphopetides. The dataset was not included in inital analysis due to suboptimal sample preparation.

### Sample Protocol
8 to 12 Million cells per sample were washed with PBS and frozen in liquid nitrogen. After thawing, cell pellets were lysed in lysis buffer (8 M urea, 1mM sodium orthovanadate, 1 x phosSTOP, 1x protease inhibitor tablets (both Roche, Basel, Switzerland) in 100 mM Tris, pH 8.5). Cells were homogenized by vortexing for 10 min. Lysates were cleared by centrifugation for 10 min at 20,000 g and 4°C. Supernatant was transferred into a new tube and the protein amount was determined with the Bio-Rad Protein assay (Biorad, The Netherlands). Next, proteins were digested utilizing the filter assisted sample preparation (FASP) protocol (Wiśniewski, J. R.; Zougman, A.; Nagaraj, N.; Mann, M., Universal sample preparation method for proteome analysis. Nat Methods 2009, 6, (5), 359-62). The reduced and alkylated sample was first treated with Lys-C (Sigma-Aldrich) in an enzyme/substrate ratio of 1∶50 (w/w) for 4 h, after this trypsin (Promega, Madison, USA) was used o/n at 37°C in an enzyme/substrate ratio of 1:100 (w/w). Resulting peptide mixtures were desalted and chemically labeled using stable isotope dimethyl labelling as described previously (Boersema PJ, Raijmakers R, Lemeer S, Mohammed S, Heck AJ. Nat Protoc. 2009;4(4):484-94.). The unstimulated sample was labeled “light”, the stimulated conventional T cells were labeled “medium” and the “heavy” label was used for the stimulated suppressed T cell sample. The samples were mixed in a 1:1:1 (L:M:H) ratio based on peptide intensities. Ti4+-IMAC material was prepared and used as described earlier (Zhou H, Ye M, Dong J, Corradini E, Cristobal A, Heck AJ, Zou H, Mohammed S. Nat Protoc. 2013 Mar;8(3):461-80.). Briefly, 500 µg Ti4+-IMAC beads were loaded onto a GELloader tip (Eppendorf) with a C8 plug. After conditioning the columns with loading buffer (80% acetonitrile (ACN), 6% TFA), samples reconstituted in loading buffer were loaded onto the columns and centrifuged at 100 g for 30 min. In total four washing steps ensure the removal of unbound peptides from the columns. The bound peptides were eluted into a new tube containing 30 µl 10% formic acid (FA) with 20 µl 10% ammonia. A final elution was performed with 2 µl 80% ACN/2% FA. The eluate was further acidified by adding 3 µl of 100% FA and directly used for LC-MS analysis. The phospho-enriched samples were directly analysed on an Orbitrap Elite mass spectrometer (Thermo Fisher Scientific) coupled to an Easy UHPLC system The peptides were eluted from the reverse phase column during a 3 h gradient and directly sprayed into the mass spectrometer with in house-made gold-coated silica emitters. The columns were made in-house from either Reprosil C18 (3 μm, Dr. Maisch, Germany; 20 mm × 100 μm inner diameter) for the trap column or Poroshell 120 EC-C18 (2.7 μm, Agilent; 40 cm x 50 μm inner diameter) for the analytical column. during a 2h gradient (7-30% ACN in 91 min, 30-100% ACN 3 min, 100% ACN 5min, 100-7% 1 min, 7% ACN 20min, flow rate: 100nl/min). The Elite mass spectrometer was operated in data-dependent acquisition mode using the following settings: ESI voltage, 1.5 kV; inlet capillary temperature, 320 °C; full scan automatic gain control (AGC) target, 1e6 ions at 60000 resolution; scan range, 350-1500 m/z; Orbitrap full scan maximum injection time, 250 ms; data-dependent decision tree (HCD/ETD) (Frese CK, Altelaar AF, Hennrich ML, Nolting D, Zeller M, Griep-Raming J, Heck AJ, Mohammed S. J Proteome Res. 2011 May 6;10(5):2377-88.); normalized collision energy, 32; dynamic exclusion time, 30s ; isolation window, 1.5 m/z; 20 MS2 scans per full scan.

### Data Protocol
The raw data obtained, were initially processed with proteome discoverer 1.4 (Thermo Fisher). The created peak lists were searched with Mascot (Matrix Science, Version 2.3) against a concatenated forward-reverse Uniprot database (taxonomy homo sapiens, containing 41008 entries) and the following parameters: 50 p.p.m. precursor mass tolerance and 0.05 Da fragment ion tolerance for OT spectra and 0.6 Da fragment ion tolerance for IT spectra. Up to two missed cleavages were accepted, oxidation of methionine, phosphorylation of STY and the dimethyl label on lysines and the N-terminus was set up as variable modification whereas cysteine carbamidomethylation as fixed modifications. For the phospho-enriched samples the site occupation probabilities were calculates using the phosphoRS node in PD. Afterwards all phosphosites were filtered for 75% localization probability. Triplex dimethyl labelling was chosen as quantification method. All peptides are filtered for a minimal Mascot score of 20 and 1% FDR.

### Publication Abstract
None

### Keywords
T-cells, Phosphoproteomics

### Affiliations
Scientific Director Netherlands Proteomics Centre and  Scientific Director Utrecht Institute for Pharmaceutical Sciences Biomolecular Mass Spectrometry and Proteomics Bijvoet Center for Biomolecular Research and Utrecht Institute for Pharmaceutical Sciences Utrecht University
Faculty of Science

### Submitter
N. Binai

### Lab Head
Dr Albert Heck
Scientific Director Netherlands Proteomics Centre and  Scientific Director Utrecht Institute for Pharmaceutical Sciences Biomolecular Mass Spectrometry and Proteomics Bijvoet Center for Biomolecular Research and Utrecht Institute for Pharmaceutical Sciences Utrecht University


