### Accession
PXD027497

### Title
Venomics of the desert black cobra Walterinnesia morgani from Çörten village, Turkey

### Description
Combined transcriptomics-assisted bottom-up and top-down MS characterization of the venom proteome of the desert black cobra Walterinnesia morgani from Çörten village, Turkey

### Sample Protocol
For bottom-up MS, venom was decomplexed through RP-HPLC and SDS-PAGE. Protein bands of interest were excised from Coomassie Brilliant Blue-stained SDS-PAGE gels and subjected to in-gel reduction and alkylation, followed by overnight sequencing-grade trypsin digestion. tryptic peptides were separated by nano-Acquity UltraPerformanceLC® (UPLC®) using BEH130C18 (100μm × 100 mm, 1.7μm particle size) column in-line with a Waters SYNAPT G2 High Definition Mass Spectrometry System.  For top-down MS, reduced and non-reduced samples were analyzed each by LC (RP-HPLC)-high resolution (HR) electrospray ionization (ESI)-MS/MS, using (a) an LTQ Orbitrap XL mass spectrometer (Thermo, Bremen, Germany) was coupled to an Agilent 1200 HPLC system (Agilent,Waldbronn, Germany), using a Supelco Discovery 300Å C18 (2.1 × 150 mm, particle size, 3 mm) column; or (b) a Vanquish ultra-high-performance liquid chromatography (UHPLC) system coupled to a Q-Exactive quadrupole orbital ion trap (Thermo Fisher Scientific, Bremen, Germany). LC separation was performed on a Supelco Discovery Biowide C18 column (300Å pore size, 2 x 150 mm column size, 3 µm particle size).

### Data Protocol
Fragmentation spectra were interpreted using the on-line form of the MASCOT Server (version 2.6) at http://www.matrixscience.com against the last update of the NCBI non-redundant database, including the W. aegyptia venom gland transcriptomic data deposited with the SRA and TSA databases of NCBI (BioProject accession number PRJA506018) publicly available in the MassIVE repository under accession number MSV000081885 (ftp://massive.ucsd.edu/MSV000081885/) and in proteomXchange with accession number PXD008597. All MS/MS data were manually checked and good quality unmatched fragmentation spectra were submitted to manual (de novo) sequencing. Peptide sequences assigned by de novo MS/MS were matched to homologous snake venom proteins available in the NCBI non-redundant protein sequences database using the default parameters of the BLASTP program

### Publication Abstract
We report a novel hybrid, molecular and elemental mass spectrometry (MS) setup for the absolute quantification of snake venom proteomes shown here for two desert black cobra species within the genus <i>Walterinnesia</i>, <i>Walterinnesia aegyptia</i> and <i>Walterinnesia morgani</i>. The experimental design includes the decomplexation of the venom samples by reverse-phase chromatography independently coupled to four mass spectrometry systems: the combined bottom-up and top-down molecular MS for protein identification and a parallel reverse-phase microbore high-performance liquid chromatograph (RP-&#x3bc;HPLC) on-line to inductively coupled plasma (ICP-MS/MS) elemental mass spectrometry and electrospray ionization quadrupole time-of-flight mass spectrometry (ESI-QToF MS). This allows to continuously record the absolute sulfur concentration throughout the chromatogram and assign it to the parent venom proteins separated in the RP-&#x3bc;HPLC-ESI-QToF parallel run via mass profiling. The results provide a locus-resolved and quantitative insight into the three desert black cobra venom proteome samples. They also validate the units of measure of our snake venomics strategy for the relative quantification of snake venom proteomes as % of total venom peptide bonds as a proxy for the % by weight of the venom toxins/toxin families. In a more general context, our work may pave the way for broader applications of hybrid elemental/molecular MS setups in diverse areas of proteomics.

### Keywords
Snake venom, Venomics, Walterinnesia morgani

### Affiliations
CSIC 
Evolutionary and translational venomics Lab, Instituto de Biomedicina de Valencia, CSIC

### Submitter
Davinia Pla 

### Lab Head
Dr Juan J Calvete
Evolutionary and translational venomics Lab, Instituto de Biomedicina de Valencia, CSIC


