### Accession
PXD015306

### Title
Absolute quantification of proteins and histone modifications in Drosophila embryos

### Description
Mass spectrometry analysis for absolute and relative quantification of the nuclear proteome and of methyl-marks on selected lysine residues in histone H3 during two stages of Drosophila embryogenesis: 2-4 hr and 14-16 hr old.

### Sample Protocol
Histones were acid-extracted from 2-4 hr and 14-16 hr old embryos as previously described (Scheuermann et al., 2010). After the precipitation of acid-extracted proteins with acetone, pellets were dissolved in appropriate amounts of Lämmli Buffer to reach 1.37e6 nuclei/μl in each sample. 15 μL were loaded on an 8-16% gradient SDS-PAGE gel (SERVA Lot V140115-1) and stained with Coomassie Blue to visualize the histone bands. Histone bands were excised and propionylated as described before (Villar-Garea et al., 2012). Isotope heavy-labelled peptides (synthetized by JPT) 500 fmol each were added in the samples before in-gel trypsin digestion. Digested peptides were sequentially desalted using C18 Stagetips (3M Empore) and porous carbon material (TipTop Carbon, Glygen) as described elsewhere (Rappsilber et al., 2007) and resuspended in 15 μl of 0.1% FA. For LC-MS/MS purposes, 5 μl were injected in a RSLCnano system (Thermo) and then separated in a 15-cm reverse phase column (75 μm ID homepacked with ReproSil-Pur C18-AQ 2.4 μm from Dr. Maisch) with a 50-min gradient from 5 to 60% acetonitrile in 0.1% formic acid. The effluent from the HPLC was directly electrosprayed into a Q Exactive HF instrument.

### Data Protocol
Data analysis was performed with the Skyline (version 3.7) (MacLean et al., 2010) by using doubly and triply charged peptide masses for extracted ion chromatograms (XICs). Selection of respective peaks was identified based on the retention time and fragmentation spectra of the spiked in heavy-labeled peptides. Integrated peak values (Total Area MS1) were exported as csv. file for further calculations. Total area MS1 from endogenous peptides was normalized to the respective area of heavy-labeled peptides. The sum of all normalized total area MS1 values of the same isotopical and different structural modified peptide in one sample resembled the amount of total peptide. The relative abundance of an observed modified peptide was calculated as percentage of the sum of all modification states for this peptide. For absolute quantifications, MaxQuant search was used.

### Publication Abstract
Gene transcription in eukaryotes is regulated through dynamic interactions of a variety of different proteins with DNA in the context of chromatin. Here, we used mass spectrometry for absolute quantification of the nuclear proteome and methyl marks on selected lysine residues in histone H3 during two stages of Drosophila embryogenesis. These analyses provide comprehensive information about the absolute copy number of several thousand proteins and reveal unexpected relationships between the abundance of histone-modifying and -binding proteins and the chromatin landscape that they generate and interact with. For some histone modifications, the levels in Drosophila embryos are substantially different from those previously reported in tissue culture cells. Genome-wide profiling of H3K27 methylation during developmental progression and in animals with reduced PRC2 levels illustrates how mass spectrometry can be used for quantitatively describing and comparing chromatin states. Together, these data provide a foundation toward a quantitative understanding of gene regulation in Drosophila.

### Keywords
Drosophila, Histone modification, Qexactive-hf

### Affiliations
CECAD, University of Cologne, AG Krüger
Protein Analysis Unit, BioMedical Center, Faculty of Medicine, Ludwig-Maximilians-University Munich, Germany

### Submitter
Andreas Schmidt

### Lab Head
Dr Axel Imhof
Protein Analysis Unit, BioMedical Center, Faculty of Medicine, Ludwig-Maximilians-University Munich, Germany


