### Accession
PXD017875

### Title
Identification of X-linked genes that drive sex differences in murine embryonic stem cells

### Description
Dosage imbalance of X-chromosomal genes contributes to sex differences, in particular during early development, when both X chromosomes are active in females. X-encoded inhibitors of the differentiation-promoting MAP kinase (MAPK) signalling pathway slow down development, increase levels of naive pluripotency factors and decrease MAPK target gene expression. Through a hierarchical CRISPR screening approach in murine embryonic stem cells (mESC) we have comprehensively identified X-linked genes that modulate MAPK signalling, pluripotency factor expression and differentiation. We show that multiple genes act in concert to drive sex differences in mESC. Dusp9, a known negative regulator of the MAPK pathway alters phosphorylation of MAPK pathway intermediates in female cells, while Klhl13 underlies differences in pluripotency factor expression and differentiation by targeting differentiation-promoting factors for degradation. Klhl13 is a member of the Cullin 3 E3 ubiquitin ligase complex, where it acts as a substrate adaptor to target proteins for proteasomal degradation (Dhanoa et al., 2013). We reasoned that the Klhl13-mediated sex differences we have identified might be due to reduced protein abundance of Klhl13 substrate proteins in female compared to male cells, which affect pluripotency factors, differentiation and MAPK target gene expression. To identify Klhl13 substrates in mESCs, we comprehensively profiled Klhl13 interaction partners and then selected those with increased protein levels in K13-HOM mutant cells. To identify interaction partners, we ectopically expressed either full-length Klhl13 or the substrate-binding Kelch domain, tagged with a green fluorescent protein (GFP), and identified binding partners by Immunoprecipitation-Mass Spectrometry (IP-MS) using a GFP-specific antibody. These experiments were performed in female mESCs with a homozygous Klhl13 deletion. By using this approach, we identify Scml2 and Alg13 as Klhl13 substrates.

### Sample Protocol
GFP Immunoprecipitation The GFP Immunoprecipitation protocol was performed as described previously with minor modifications  (Hubner et al., 2010). Briefly, cells were treated with 15µM of MG132 for 3h prior to harvesting. Cells were pelleted and resuspended in 1 ml of Lysis Buffer containing 150 mM NaCl, 50 mM Tris, pH 7.5, 5% glycerol, 1% IGEPAL-CA-630, 1 mM MgCl2, 200 U benzonase (Merck), and EDTA-free complete protease inhibitor cocktail (Roche). Cells were incubated on ice for 30 min to allow cell lysis. Lysates were centrifuged at 4,000xg and 4°C for 15 min and the supernatant was incubated with 50 µl magnetic beads coupled to monoclonal mouse anti-GFP antibody (Miltenyi Biotec) for 20 min on ice. Magnetic columns were equilibrated by washing first with 250µl of 100% EtOH followed by two washes with the same volume of lysis buffer. After the 20 min incubation, the lysates were applied to the column followed by three washes with 800µl of ice-cold wash buffer I (150 mM NaCl, 50 mM Tris, pH 7.5, 5% glycerol, and 0.05% IGEPAL-CA-630) and two washes with 500 µl of wash buffer II (150 mM NaCl, 50 mM Tris, pH 7.5, and 5% glycerol). Column-bound proteins were subsequently pre-digested with 25 µl 2 M urea in 50 mM Tris, pH 7.5, 1 mM DTT, and 150 ng trypsin (Roche) for 30 min at room temperature. Proteins were eluted by adding two times 50µl elution buffer (2 M urea in 50 mM Tris, pH 7.5, and 5 mM chloroacetamide). Proteins were further digested overnight at room temperature. The tryptic digest was stopped by adding formic acid to a final concentration of 2%.  Sample Preparation for proteomics with Label-Free Quantification (LFQ) Proteomics sample preparation was done according to a published protocol with minor modifications (Kulak et al., 2014). Approximately 2*107 cells were lysed under denaturing conditions in a buffer containing 3 M guanidinium chloride (GdmCl), 5 mM tris(2-carboxyethyl)phosphine, 20 mM chloroacetamide and 50 mM Tris-HCl pH 8.5. Lysates were denatured at 95°C for 10 min shaking at 1000 rpm in a thermal shaker and sonicated in a water bath for 10 min. A small aliquot of cell lysate was used for the bicinchoninic acid (BCA) assay to quantify the protein concentration. 50 µg protein of each lysate was diluted with a dilution buffer containing 10% acetonitrile and 25 mM Tris-HCl, pH 8.0, to reach a 1 M GdmCl concentration. Then, proteins were digested with LysC (Roche, Basel, Switzerland; enzyme to protein ratio 1:50, MS-grade) shaking at 700 rpm at 37°C for 2h. The digestion mixture was diluted again with the same dilution buffer to reach 0.5 M GdmCl, followed by a tryptic digestion (Roche, enzyme to protein ratio 1:50, MS-grade) and incubation at 37°C overnight in a thermal shaker at 700 rpm. LC-MS/MS Instrument Settings for Shotgun Proteome Profiling Peptide desalting was performed according to the manufacturer’s instructions (Pierce C18 Tips, Thermo Scientific, Waltham, MA). Desalted peptides were reconstituted in 0.1% formic acid in water and further separated into four fractions by strong cation exchange chromatography (SCX, 3M Purification, Meriden, CT). Eluates were first dried in a SpeedVac, then dissolved in 5% acetonitrile and 2% formic acid in water, briefly vortexed, and sonicated in a water bath for 30 seconds prior injection to nano-LC-MS. LC-MS/MS was carried out by nanoflow reverse phase liquid chromatography (Dionex Ultimate 3000, Thermo Scientific) coupled online to a Q-Exactive HF Orbitrap mass spectrometer (Thermo Scientific), as reported previously (Gielisch and Meierhofer, 2015). Briefly, the LC separation was performed using a PicoFrit analytical column (75 μm ID × 50 cm long, 15 µm Tip ID; New Objectives, Woburn, MA) in-house packed with 3-µm C18 resin (Reprosil-AQ Pur, Dr. Maisch, Ammerbuch, Germany). Peptides were eluted using a gradient from 3.8 to 38% solvent B in solvent A over 120 min at 266 nL per minute flow rate. Solvent A was 0.1 % formic acid and solvent B was 79.9% acetonitrile, 20% H2O, 0.1% formic acid. For the IP samples, a one hour gradient was used. Nanoelectrospray was generated by applying 3.5 kV. A cycle of one full Fourier transformation scan mass spectrum (300−1750 m/z, resolution of 60,000 at m/z 200, automatic gain control (AGC) target 1*106) was followed by 12 data-dependent MS/MS scans (resolution of 30,000, AGC target 5*105) with a normalized collision energy of 25 eV. In order to avoid repeated sequencing of the same peptides, a dynamic exclusion window of 30 sec was used. In addition, only peptide charge states between two to eight were sequenced.

### Data Protocol
Data Analysis Raw MS data were processed with MaxQuant software (v1.6.0.1) and searched against the mouse proteome database UniProtKB with 22,286 entries, released in December 2018. Parameters of MaxQuant database searching were a false discovery rate (FDR) of 0.01 for proteins and peptides, a minimum peptide length of seven amino acids, a first search mass tolerance for peptides of 20 ppm and a main search tolerance of 4.5 ppm, and using the function “match between runs”. A maximum of two missed cleavages was allowed for the tryptic digest. Cysteine carbamidomethylation was set as fixed modification, while N-terminal acetylation and methionine oxidation were set as variable modifications. Contaminants, as well as proteins identified by site modification and proteins derived from the reversed part of the decoy database, were strictly excluded from further analysis. The calculation of significantly differentially expressed proteins for both the proteomics (K13 HOM vs XX wildtype) as well as the IP datasets (GFP-Kelch/D-GFP-Klhl13 vs GFP/D-GFP-Klhl13) was done with Perseus (v1.6.1.3). LFQ intensities, originating from at least two different peptides per protein group were transformed by log2. Only groups with valid values in at least one group were used, missing values were replaced by values from the normal distribution. Statistical analysis for differential expression was done by a two-sample t-test with Benjamini-Hochberg (BH, FDR of 0.05) correction for multiple testing. The processed output files can be found in Supp. Table S3. For the identification of Klhl13 interaction partners, cut-offs were set from the data displayed in the volcano plots using a previously published method (Keilhauer et al., 2015). Briefly, a graphical formula as a smooth combination of the following parameters was implemented: -log10(p) ≥ c/ |x| - xo x: enrichment factor of a protein p: p-value of the t-test, calculated from replicates xo: fixed minimum enrichment c: curvature parameter We optimized parameters c and xo such as to have 10% FDR (left-sided outliers) while maximizing the number of right-sided outliers. In the case of the GFP-Kelch IP, c = 0.32 and xo = 0.02. For the GFP-Klhl13 IP, c = 0.28 and xo = 0.04.

### Publication Abstract
None

### Keywords
Klhl13, Sex differences, Mapk, Differentiation, Pluripotency, Mouse embryonic stem cells

### Affiliations
Mass Spectrometry Facility MPIMG
Max Planck Institute for Molecular Genetics

### Submitter
David Meierhofer

### Lab Head
Dr Edda G. Schulz
Max Planck Institute for Molecular Genetics


