### Accession
PXD035283

### Title
Ubiquitin E3 ligase FBXO21 regulates cytokine mediated signaling pathways, but is dispensable for steady-state hematopoiesis

### Description
Hematopoietic cell fate decisions such as self-renewal and differentiation are highly regulated through multiple molecular pathways. One pathway, the ubiquitin proteasome system (UPS), controls protein levels by tagging them with polyubiquitin chains and promoting their degradation through the proteasome. Ubiquitin E3 ligases serve as the substrate-recognition component of the UPS. Through investigating the FBOX family of E3 ligases, we discovered that Fbxo21 was highly expressed in the hematopoietic stem and progenitor cell (HSPC) population, and showed low to no expression in mature myeloid populations. To determine the role of FBXO21 on HSPC maintenance, self-renewal, and differentiation, we generated shRNAs against FBXO21 and a hematopoietic specific Fbxo21 conditional knockout (cKO) mouse model. We found that silencing FBXO21 in HSPCs led to a loss in colony formation and an increase in cell differentiation in vitro. Additionally, stressing the HSPC populations in our Fbxo21 cKO mouse with 5-FU injections resulted in a decrease in survival, despite these populations showing minimal alterations during steady-state hematopoiesis. Although FBXO21 has previously been proposed to regulate cytokine signaling via ASK and p38, our results show that depletion of FBXO21 led to altered ERK signaling in vitro. Together, these findings suggest ubiquitin E3 ligase FBXO21 regulates HSPCs through cytokine mediated pathways.

### Sample Protocol
Mouse Lin-, Sca1+, cKit+(LSK) cells were isolated from wildtype micebone marrow and infected with a RenillaControl or Fbxo21shRNA retrovirus containing a GFP or mCherry tag. On day 4 post infection, cells were then sorted for GFP+or mCherry+cells. Samples were prepared and TMT-labeled per manufacturer’s protocol (ThermoFisher TMT10plex kit), cleaned with C18 tips, combined in equal concentration (Pierce colorimetric peptide assay kit), and fractionated using high pH reverse phase fractionation kit (ThermoFisher). Samples were speedvac to dry, resuspended in 0.1% formic acid before analysis.Samples were loaded onto trap column Acclaim PepMap 100 75μm x 2 cm C18 LC Columns (Thermo ScientificTM) at flow rate of 5μl/min then separated with a Thermo RSLC Ultimate 3000 (Thermo ScientificTM) from 5-20% solvent B (0.1% FA in 80% ACN) from 10-98 minutes at 300nL/min and 50ºC with a 120 minutes total run time for fractions one and two. For fractions three to six, solvent B was used at 5-45% for the same duration. Eluted peptides were analyzed by a Thermo Orbitrap Fusion LumosTribrid (Thermo ScientificTM) mass spectrometer in a data dependent acquisition mode using synchronous precursor selection method. A survey full scan MS (from m/z 375-1500) was acquired in the Orbitrap with a resolution of 120,000. The AGC target for MS2 in iontrap was set as 1x104 and ion filling time set as 150ms and fragmented using CID fragmentation with 35% normalized collision energy. The AGC target for MS3 in orbitrap was set as 1x105 and ion filling time set as 200ms with a scan range of 100-500 and fragmented using HCD with 65% normalized collision energy.

### Data Protocol
Protein identification was performed using proteome discoverer software version 2.4(Thermo Fisher Scientific) by searching MS/MS data against the UniProtmouse protein databaseon March 23rd, 2022. The search was set up for full tryptic peptides with a maximum of 2 missed cleavage sites. Oxidation, TMT6plex of the amino terminus, GG and GGQ ubiquitination,  phosphorylation, and acetylation were included as variable modifications and carbamidomethylation and TMT6plex of the amino terminus were set as fixed modifications. The precursor mass tolerance threshold was set at 10ppm for a maximum fragment mass error of 0.6 Da with a minimum peptide length of 6 and amaximum peptide length of 144. The significance threshold of the ion score was calculated based on a false discovery rate calculated using the percolator node. Protein accessions were put into Ingenuity Pathway Analysis (QIAGEN Inc.) to identify gene symbols and localizations. Gene ontology pathway analysis was performed using DAVID Bioinformatics Database 6.8 using the functional annotation tool.

### Publication Abstract
Hematopoietic cell fate decisions such as self-renewal and differentiation are highly regulated through multiple molecular pathways. One pathway, the ubiquitin proteasome system (UPS), controls protein levels by tagging them with polyubiquitin chains and promoting their degradation through the proteasome. Ubiquitin E3 ligases serve as the substrate-recognition component of the UPS. By investigating the FBOX family of E3 ligases, we discovered that Fbxo21 was highly expressed in the hematopoietic stem and progenitor cell (HSPC) population, and exhibited low to no expression in mature myeloid populations. To determine the role of FBXO21 on HSPC maintenance, self-renewal, and differentiation, we generated shRNAs against FBXO21 and a hematopoiesis-specific Fbxo21 conditional knockout (cKO) mouse model. We found that silencing FBXO21 in HSPCs led to a loss in colony formation and an increase in cell differentiation in vitro. Additionally, stressing the HSPC populations in our Fbxo21 cKO mouse with 5-fluorouracil injections resulted in a decrease in survival, despite these populations exhibiting minimal alterations during steady-state hematopoiesis. Although FBXO21 has previously been proposed to regulate cytokine signaling via ASK and p38, our results indicate that depletion of FBXO21 led to altered ERK signaling in vitro. Together, these findings suggest ubiquitin E3 ligase FBXO21 regulates HSPCs through cytokine-mediated pathways.

### Keywords
Fbox, Hematopoietic stem cell, Ubiquitin e3 ligase

### Affiliations
University of Nebraska Medical Center
University of Nebraska Medical Center School of Medicine Department of Genetics, Cell Biology, & Anatomy Buckley Lab Omaha, Nebraska United States of America

### Submitter
Samantha Swenson

### Lab Head
Dr Shannon Mychel Buckley
University of Nebraska Medical Center School of Medicine Department of Genetics, Cell Biology, & Anatomy Buckley Lab Omaha, Nebraska United States of America


