### Accession
PXD029574

### Title
Identification of METTL5 interaction partners

### Description
METTL5 is an RNA methyltransferase whose stability was noted to be poor when expressed in both mammalian and bacterial expression systems. To determine if METTL5 has an endogenous stabilizing binding partner, FLAG-tagged METTL5 was expressed in Freestyle 293 cells and pulled down with anti-FLAG-M2 antibody-coupled magnetic beads. Bound proteins were fractionated by SDS-PAGE, and bands of interest were cut out for proteomics analysis.

### Sample Protocol
Sample processing was performed by MSBioWorks LLC.   Trypsin digestion was performed using a robot (ProGest, DigiLab) as follows: 1) washed with 25mM ammonium bicarbonate followed by acetonitrile, 2) reduced with 10mM dithiothreitol at 60C followed by alkylation with 50mM iodoacetamide at RT, 3) digested with trypsin (Promega) at 37C for 4 hours, 4) quenched with formic acid and the supernatant was analyzed directly without further processing.  The gel digest was analyzed by nano LC/MS/MS with a Waters NanoAcquity HPLC system interfaced to a ThermoFisher Q Exactive. Peptides were loaded on a trapping column and eluted over a 75Î¼m analytical column at 350nL/min; both columns were packed with Luna C18 resin (Phenomenex). The mass spectrometer was operated in data-dependent mode, with MS and MS/MS performed in the Orbitrap at 70,000 FWHM resolution and 17,500 FWHM resolution, respectively. The fifteen most abundant ions were selected for MS/MS.

### Data Protocol
Data were searched using a local copy of Mascot with the following parameters:  Enzyme: Trypsin Database: Swissprot Human (forward and reverse appended with common contaminants) Fixed modification: Carbamidomethyl (C) Variable modifications: Oxidation (M), Acetyl (Protein N-term), Pyro-Glu (N-term Q), Deamidation (NQ) Mass values: Monoisotopic Peptide Mass Tolerance: 10 ppm Fragment Mass Tolerance: 0.02 Da Max Missed Cleavages: 2 Mascot DAT files were parsed into the Scaffold software for validation, filtering and to create a nonredundant list per sample. Data were filtered using a minimum protein value of 90%, a minimum peptide value of 50% (Prophet scores) and requiring at least two unique peptides per protein.

### Publication Abstract
Ribosomal RNAs (rRNAs) have long been known to carry chemical modifications, including 2'O-methylation, pseudouridylation, N<sup>6</sup>-methyladenosine (m<sup>6</sup>A), and N<sup>6,6-</sup>dimethyladenosine. While the functions of many of these modifications are unclear, some are highly conserved and occur in regions of the ribosome critical for mRNA decoding. Both 28S rRNA and 18S rRNA carry single m<sup>6</sup>A sites, and while the methyltransferase ZCCHC4 has been identified as the enzyme responsible for the 28S rRNA m<sup>6</sup>A modification, the methyltransferase responsible for the 18S rRNA m<sup>6</sup>A modification has remained unclear. Here, we show that the METTL5-TRMT112 methyltransferase complex installs the m<sup>6</sup>A modification at position 1832 of human 18S rRNA. Our work supports findings that TRMT112 is required for METTL5 stability and reveals that human METTL5 mutations associated with microcephaly and intellectual disability disrupt this interaction. We show that loss of METTL5 in human cancer cell lines and in mice regulates gene expression at the translational level; additionally, Mettl5 knockout mice display reduced body size and evidence of metabolic defects. While recent work has focused heavily on m<sup>6</sup>A modifications in mRNA and their roles in mRNA processing and translation, we demonstrate here that deorphanizing putative methyltransferase enzymes can reveal previously unappreciated regulatory roles for m<sup>6</sup>A in noncoding RNAs.

### Keywords
Mettl5

### Affiliations
Yale University
Department of Molecular, Cellular, and Developmental Biology Yale University

### Submitter
Sigrid Nachtergaele

### Lab Head
Dr Sigrid Nachtergaele
Department of Molecular, Cellular, and Developmental Biology Yale University


