### Accession
PXD034568

### Title
Growth temperature is the principal driver of chromatinization in archaea

### Description
Across the tree of life, DNA in living cells is associated with proteins that coat chromosomes, constrain their structure and influence DNA-templated processes such as transcription and replication. In bacteria and eukaryotes, HU and histones, respectively, are the principal constituents of chromatin, with few exceptions. Archaea, in contrast, have more diverse repertoires of nucleoid-associated proteins (NAPs). The evolutionary and ecological drivers behind this diversity are poorly understood. Here, we combine a systematic phylogenomic survey of known and predicted NAPs with quantitative protein abundance data to shed light on the forces governing the evolution of archaeal chromatin. Our survey identifies the Diaforarchaea as a hotbed of NAP gain and loss and we validate novel candidate NAPs in two members of this clade, Thermoplasma volcanium and Methanomassiliicoccus luminyensis, using sucrose gradient-based nucleoid enrichment coupled to quantitative mass spectrometry. Comparative analysis across a panel of 19 archaea revealed that investment in NAP production varies over two orders of magnitude, from <0.03% to >5% of total protein. Integrating genomic and ecological data, we demonstrate that growth temperature is an excellent predictor of relative NAP investment across archaea. Our results suggest that high levels of chromatinization have evolved as a mechanism toprevent uncontrolled helix opening and runaway denaturation – rather than, for example, to globally orchestrate gene expression – with implications for the origin of chromatin in both archaea and eukaryotes.

### Sample Protocol
Sample processing: 50 mL aliquots of 10-day (3-day) cultures of M. luminyensis (M. alvus) were pelleted under anaerobic conditions and stored at -80°C. Protein pellets were prepared following PreOmics iST kit guidelines. 10mg of frozen pellet were resuspended in lysis buffer and volumes adjusted after the heating step to load 100µg for WCE and from 30-100µg as measured by nanodrop absorbance at 205nm. As indicated by the manufacturer, samples were heated for 10 min at 95°C in the lysis buffer. The heating step was extended to 1 hour for H. volcanii nucleoid enrichment samples to reverse cross-linking (see below). Following the manufacturer’s instruction, a DNA sonication step was included (Diagenode Bioruptor, 10 cycles; 30 sec ON/OFF, High intensity) followed by digestion (using the PreOmics iST kit Trypsin/LysC mix) for 1.5 hrs at 37°C with shaking. The whole procedure was carried out without interruption and pellets stored at -80°C in MS-LOAD buffer before being processed by mass spectrometry (two biological replicates with technical replicates for each, except for M. luminyensis nucleoid enrichment which was done in biological triplicates). LC-MS:Samples were injected in technical duplicates in either nano-flow or micro-flow modes.  For nano-flow analysis, chromatographic separation was performed using an Ultimate 3000 RSLC nano liquid chromatography system (Thermo Scientific) coupled to an Orbitrap Q-Exactive mass spectrometer (Thermo Scientific) via an EASY-Spray source. Peptide solutions were injected and loaded onto a trapping column (Acclaim PepMap 100 C18, 100μm × 2cm) for desalting and concentration at 8μL/min in 2% acetonitrile, 0.1% TFA. Peptides were then eluted on-line to an analytical column (EASY-Spray PepMap RSLC C18, 75μm × 75cm) at a flow rate of 200nL/min. Peptides were separated using a 120 minute gradient, 4-25% of buffer B for 90 minutes followed by 25-45% buffer B for another 30 minutes (composition of buffer B – 80% acetonitrile, 0.1% FA) and subsequent column conditioning and equilibration. Eluted peptides were analysed by the mass spectrometer in positive polarity using a data-dependent acquisition mode. Ions for fragmentation were determined from an initial MS1 survey scan at 70,000 resolution, followed by HCD (Higher-energy Collision Induced Dissociation) of the top 12 most abundant ions at 17,500 resolution. MS1 and MS2 scan AGC targets were set to 3e6 and 5e4 for maximum injection times of 50ms and 50ms respectively. A survey scan m/z range of 400–1800 was used, normalised collision energy set to 27 and charge exclusion enabled for unassigned and +1 ions. Dynamic exclusion was set to 45 seconds. For samples run in micro-flow, chromatographic separation was performed using an Ultimate 3000 RSLC nano liquid chromatography system (Thermo Scientific) coupled to an Orbitrap Q-Exactive mass spectrometer (Thermo Scientific) via anEASY-Spray source. Peptide solutions were injected directly onto the analytical column (Waters nanoEase M/Z Peptide CSH C18, 300μm × 15cm) at working flow rate of 5μL/min for 4 minutes. Peptides were then separated using a 121 minute stepped gradient: 0-4% of buffer B for 11minutes, 4-47.5% of buffer B for 114 minutes (composition of buffer A – 95/5%: H2O/DMSO + 0.1% FA, buffer B – 75/20/5% MeCN/H2O/DMSO + 0.1% FA), followed by column conditioning and equilibration. Eluted peptides were analysed by the mass spectrometer in positive polarity using a data-dependent acquisition mode. Ions for fragmentation were determined from an initial MS1 survey scan at 70,000 resolution, followed by HCD(Higher-energy Collision Induced Dissociation) of the top 10 most abundant ions at 17,500 resolution. MS1 and MS2 scan AGC targets were set to 3e6 and 1e5 for maximum injection times of 50ms and 100ms respectively. A survey scan m/z range of 400–1800 was used, normalised collision energy set to 27 and charge exclusion enabled for unassigned and +1 ions. Dynamic exclusion was set to 45 seconds.

### Data Protocol
Data were processed using the MaxQuant software platform (v1.6.10.43) , with database searches carried out by the in-built Andromeda search engine against various GenBank organism specific databases from NCBI. A reverse decoy database approach was used at a 1% false discovery rate (FDR) for peptide spectrum matches. Search parameters were as follows: maximum missed cleavages set to 2, fixed modification of cysteine carbamidomethylation and variable modifications of methionine oxidation, protein N-terminal acetylation, asparagine deamidation and cyclisation of glutamine to pyro-glutamate. Label-free quantification was enabled with a LFQ minimum ratio count of 1. ‘Match between runs’ function was used with match and alignment time limits of 0.7 and 20 minutes respectively.

### Publication Abstract
None

### Keywords
Chromatin, Nucleoid, Thermophile, Archaea, Histones

### Affiliations
Head of Proteomics & Metabolomics Facility
Medical Research Council - London Institute of Medical Sciences

### Submitter
Alex Montoya

### Lab Head
Dr Dr Pavel Shliaha
Head of Proteomics & Metabolomics Facility


