### Accession
PXD006848

### Title
Photoperiodic control of the Arabidopsis proteome

### Description
Plants respond to seasonal cues such as the photoperiod, to adapt to current conditions and to prepare for environmental changes in the season to come. To assess photoperiodic responses at the protein level, we quantified the proteome of the model plant Arabidopsis thaliana by mass spectrometry across four photoperiods. This revealed coordinated changes of abundance in the proteins of photosynthesis, primary metabolism and secondary metabolic processes such as pigment biosynthesis, consistent with higher metabolic activity in long photoperiods.  Higher translation rates during the day than during the night likely contribute to these changes, but rhythmic RNA profiles will alter their effects. Photoperiodic control of protein levels might be greatest if high translation rates only coincide with high transcript levels in some photoperiods. We term this mechanism ‘translational coincidence’, mathematically model its components, and demonstrate its effect  on the Arabidopsis proteome. Datasets from a green alga and a cyanobacterium suggest that this mechanism is general, contributing to the seasonal control of the proteome in many phototrophic organisms, and favouring RNA rhythms even for stable proteins.

### Sample Protocol
Plant Material and Growth Conditions The same plant material used for transcriptome analysis in (Flis et al., 2016) was the basis of this proteome study. Briefly, Arabidopsis thaliana Col-0 plants were grown on GS 90 soil mixed in a ratio 2:1 (v/v) with vermiculite. Plants were grown for 1 week in a 16 h light (250 μmol m−2 s−1, 20 °C)/8 h dark (6 °C) regime followed by an 8 h light (160 μmol m−2 s−1, 20 °C)/16 h dark (16 °C) regime for one week. Plants were then replanted with five seedlings per pot, transferred for 1 week to growth cabinets with an 8 h photoperiod (160 μmol m−2s−1, 20 °C throughout the day/night cycle) and then distributed into small growth cabinets with a 18, 12, 8 or 6 h photoperiod (160 μmol m−2s−1 and 20/18 °C in the day/night). This growth protocol was used to decrease differences in size between plants at the time of harvest, and to prevent an early transition to flowering that would otherwise occur of plants were grown from germination in long photoperiods. Plant material was harvested 9 days after transfer at the end of the night and at the end of the day. Plant material was homogenized using a Ball-Mill (Retch, Germany). Approximately 50 mg of material per sample was aliquoted into 2 mL Eppendorf tubes whilst frozen and distributed for analysis to consortium partners in three biological and two technical replicates.   Protein Extraction and Digestion  Frozen plant material was suspended in 100 µL SDS extraction medium (4% w/v SDS, 40 mM Tris, 60 μL ml-1 protease inhibitor cocktail (Roche)) and mixed vigorously. The extract was cleared by centrifuged for 10 min at 16,000 g followed by ultracentrifugation at 100,000 g for 45 min. The resulting supernatant was diluted 4:1 (v/v) in Laemmli sample buffer and incubated at 65°C for 5 min. For each sample, 400 μg protein was subjected to electrophoresis overnight on a 10% SDS-polyacrylamide gel at 60 V. Samples were loaded randomized on the gels to minimize positional effects. Gels were stained in Coomassie Blue solution (20% v/v methanol, 10% v/v acetic acid, 0.1% m/v Coomassie Brilliant Blue R) for 45 min then de-stained twice in 10% v/v methanol, 5% v/v acetic acid for 1 h at room temperature. Each lane of the gel was cut into 7 fractions and transferred to a 96 deep well plate. Gel pieces were fully destained by three rounds of 50% v/v methanol, 100 mM ammonium bicarbonate, incubating each time for 1 h at 37 °C. In-gel digestion of proteins using trypsin was performed as previously reported (Shevchenko et al., 1996). Volumes of solutions were adjusted to ensure that the gel pieces were fully covered during the reduction, alkylation and washing steps. Following in-gel tryptic digestion peptides were purified by reversed-phase chromatography on Finisterre C18 SPE columns (Teknokroma, Barcelona, Spain) and dried in a vacuum centrifuge at 45°C.   Mass Spectrometry Analysis Peptides were re-suspended in 40 μL 3% v/v acetonitrile, 0.1% v/v formic acid. Measurements were performed on a LTQ-Orbitrap Velos (Thermo Scientific) coupled with a NanoLC 1D HPLC (Eksigent). Samples were loaded onto a laboratory-made capillary column (9 cm long, 75 μm inner diameter), packed with Magic C18 AQ beads (3 μm, 100 Å, Microm) and eluted with a 5% to 40% v/v acetonitrile concentration gradient over 70 min, followed by 80% v/v acetonitrile for 10 min, at a flow rate of 0.25 μL min-1. Peptide ions were detected in a full MS1 scan for mass-to-charge ratios between 300 and 2000. MS2 scans were performed for the ten peptides with the highest MS signal (minimal signal strength 500 hits, isolation width mass-to-charge ratio 3 m/z, relative collision energy 35%). Peptide masses for which MS/MS spectra had been recorded were excluded from further MS/MS scans for 30 seconds.

### Data Protocol
Peak Area Based Protein Quantification and Statistical Analysis Quantitative analysis of MS/MS measurements was performed with Progenesis LCMS software (Nonlinear Dynamics). One run was selected as a reference and for each run 15 vectors were placed manually on prominent peaks before applying the automatic alignment and peak picking functions of Progenesis. Normalization factors across all samples ranged between 0.7 and 1.4.The best eight spectra for each MS1 signal peak were exported to Mascot. Mascot search parameters were set as follows: Arabidopsis TAIR10 genome annotation, requirement for tryptic ends, one missed cleavage allowed, fixed modification: carbamidomethylation (cysteine), variable modification: oxidation (methionine), peptide mass tolerance = ± 10 ppm, MS/MS tolerance = ± 0.6 Da, allowed peptide charges of +2 and +3. Spectra were also searched against a decoy database of the Arabidopsis proteome and results were filtered to ensure a FDR below 1 % on the protein level. Additionally, peptide identifications with a Mascot score below 25 were excluded. Mascot results were imported into Progenesis, quantitative peak area information extracted and the results exported for data plotting and statistical analysis.  Statistical analysis to identify significantly changing proteins was performed in R (version 3.2.3; R Core Team, 2015) using log2-transformed relative abundance values. First, analysis of variance (ANOVA) was performed across photoperiods. The resulting p-values were corrected for multiple testing with the Benjamini–Hochberg method to control the global FDR. Next, significant changes between photoperiods were computed by pairwise-comparison using the Tukey Honest Significant Differences (TukeyHSD) post-hoc test followed by correction with the Benjamini–Hochberg method. The results of this analysis for all proteins are presented in Table EV3. Overrepresentation analysis of functional categories was performed using KEGG pathway annotations (Kanehisa et al., 2016) and gene ontology (GO) terms (Ashburner et al., 2000). Arabidopsis GO annotations were obtained from the Gene Ontology Consortium database (http://www.geneontology.org). Over-representation of GO terms and KEGG pathways was assessed using Fisher's exact test.

### Publication Abstract
Plants respond to seasonal cues such as the photoperiod, to adapt to current conditions and to prepare for environmental changes in the season to come. To assess photoperiodic responses at the protein level, we quantified the proteome of the model plant <i>Arabidopsis thaliana</i> by mass spectrometry across four photoperiods. This revealed coordinated changes of abundance in proteins of photosynthesis, primary and secondary metabolism, including pigment biosynthesis, consistent with higher metabolic activity in long photoperiods. Higher translation rates in the day than the night likely contribute to these changes, via an interaction with rhythmic changes in RNA abundance. Photoperiodic control of protein levels might be greatest only if high translation rates coincide with high transcript levels in some photoperiods. We term this proposed mechanism "translational coincidence", mathematically model its components, and demonstrate its effect on the <i>Arabidopsis</i> proteome. Datasets from a green alga and a cyanobacterium suggest that translational coincidence contributes to seasonal control of the proteome in many phototrophic organisms. This may explain why many transcripts but not their cognate proteins exhibit diurnal rhythms.

### Keywords
Arabidopsis thaliana, Sds page, Photoperiod, Shotgun proteomics, Size fractionation, End of day

### Affiliations
Max Planck Institute of Molekular Plant Physiology
Plant Proteomics Group, Max Planck Institute of Molecular Plant Physiology, Am Muehlenberg 1, 14476 Potsdam-Golm, Germany

### Submitter
Alexander Graf

### Lab Head
Dr Alexander Graf
Plant Proteomics Group, Max Planck Institute of Molecular Plant Physiology, Am Muehlenberg 1, 14476 Potsdam-Golm, Germany


