### Accession
PXD016503

### Title
Interrogating the higher order structures of snake venom proteins

### Description
Snake  venoms contain complex  mixtures  of  proteins  that  play  vital  roles  in  the  survival  of venomous   snakes. In   line with   their   diverse   pharmacological   activities, the   protein compositions of snake venoms can be highly variable, and efforts to characterise the primary structures  of  such  proteins  are  extensive  and  ongoing.  In  addition, a  significant  knowledge gap  exists  in terms  of higher-order interactionsbetweenproteinsproposedto  modulate venom potency, which poses a challenge for treatment of envenomation and development of successful therapeutic applications.Here we use a multifaceted mass spectrometrybased approach  to  characteriseproteinsfrom the medically  significant  venomsof Collettâ€™s snake Pseudechis collettiand the puff adder Bitis arietans.Following chromatographic fractionation and bottom-up proteomics identification, native mass spectrometry identified, among other components, a 117 kDa non-covalent L-amino acid oxidase dimer in the P. collettivenom and a 60 kDa C-type lectin tetramer in the B. arietansvenom. Furthermore, a 27.7 kDa covalently-linked phospholipase A2(PLA2) dimer was identified in P. collettivenom, and thePLA2species were shownto  adopt  a  highly  compact  geometry  based  on  ion  mobility  measurements. Exploration of the catalytic efficiencies of the monomeric and dimeric forms of PLA2revealed that the dimeric  PLA2 possessed  greater  bioactivity  than the monomeric  PLA2s.This  work contributes  to  ongoing  efforts  to  catalogue  the  protein  components  of  snake  venoms,  and notably,itemphasises the importance of understanding higher-order protein interactions in venomsand the utility of a combined mass spectrometric approachfor this task.

### Sample Protocol
FASP insloution digest, LC-MS/MS, native MS using Ion mobility

### Data Protocol
Raw data from the LTQ XL Orbitrap was coverted into mgf files using Proteome Discover. Protein were identified and matched using Mascot. Protein identification results were exported to microsoft excel and analysed there. Spectra from the Waters SYNAPT mass spec was analysed using Masslynx. Data from Agilent HRMS was analysed using Mass Hunter.

### Publication Abstract
Snake venoms contain complex mixtures of proteins vital for the survival of venomous snakes. Aligned with their diverse pharmacological activities, the protein compositions of snake venoms are highly variable, and efforts to characterise the primary structures of such proteins are ongoing. Additionally, a significant knowledge gap exists in terms of the higher-order protein structures which modulate venom potency, posing a challenge for successful therapeutic applications. Here we use a multifaceted mass spectrometry approach to characterise proteins from venoms of Collett's snake Pseudechis colletti and the puff adder Bitis arietans. Following chromatographic fractionation and bottom-up proteomics analysis, native mass spectrometry identified, among other components, a non-covalent l-amino acid oxidase dimer in the P. colletti venom and a C-type lectin tetramer in the B. arietans venom. Furthermore, a covalently-linked phospholipase A<sub>2</sub> (PLA<sub>2</sub>) dimer was identified in P. colletti venom, from which the PLA<sub>2</sub> species were shown to adopt compact geometries using ion mobility measurements. Interestingly, we show that the dimeric PLA<sub>2</sub> possesses greater bioactivity than the monomeric PLA<sub>2</sub>s. This work contributes to ongoing efforts cataloguing components of snake venoms, and notably, emphasises the importance of understanding higher-order venom protein interactions and the utility of a combined mass spectrometric approach for this task. SIGNIFICANCE: The protein constituents of snake venoms represent a sophisticated cocktail of biologically active molecules ideally suited for further exploration in drug design and development. Despite ongoing efforts to characterise the diverse protein components of such venoms there is still much work required in this area, particularly in moving from simply describing the protein primary sequence to providing an understanding of quaternary structure. The combined proteomic and native mass spectrometry workflow utilised here gives new insights into higher order protein structures in selected snake venoms, and can underpin further investigation into the protein interactions which govern snake venom specificity and potency.

### Keywords
Snake venom, Lc-ms/ms

### Affiliations
Services & Facilities Officer, Adelaide Proteomic Centre
Department of Chemistry, School of Physical SciencesUniversity of Adelaide, Adelaide Australia, 5005

### Submitter
Parul Mittal

### Lab Head
Dr Tara L. Pukala
Department of Chemistry, School of Physical SciencesUniversity of Adelaide, Adelaide Australia, 5005


