### Accession
PXD008223

### Title
CX-MS of TFIIH and Def1 complex

### Description
The DNA damage response is an essential process for the survival of living cells. In a subset of stress-responsive genes in humans,Elongin controls transcription in response to multiple stimuli, such as DNA damage, oxidative stress, and heat shock. Yeast Elongin (Ela1-Elc1), along with Def1, is known to facilitate ubiquitylation and degradation of RNA polymerase II (pol II) in response tomultiple stimuli, yet transcription activity has not been examined. We have found that Def1 copurifies from yeast whole-cell extract with TFIIH,the largest general transcription factor required for transcription initiation and nucleotide excision repair. The addition of recombinant Def1 and Ela1-Elc1 enhanced transcription initiation in an in vitro reconstituted system including pol II, the general transcription factors, and TFIIS. Def1 also enhanced transcription restart from TFIIS-induced cleavage in a pol II transcribing complex. In the Î”def1 strain, heat shock genes were misregulated, indicating that Def1 is required for induction of some stress-responsive genes in yeast. Taken together, our results extend the understanding of the molecular mechanism of transcription regulation on cellular stress and reveal functional similarities to the mammalian system.

### Sample Protocol
The Def1-TFIIH fraction of the HiTrapQ was reacted with BS3 and digested to peptides. Peptides and crosslinked peptides were desalted on a Pierce C18 Spin Column (Thermo Fisher Scientific) and eluted by 75%acetonitrile. One-fifth of the eluted peptides were dried in a SpeedVac, reconstituted in 0.1%formic acid, and measured immediately in the mass spectrometer. The rest of the sample was enriched for crosslinked peptides by either size exclusion chromatography (Superdex Peptide 3.2/300; GE Healthcare) or strong cation exchange chromatography (Mini S PC 3.2/3; GE Healthcare). Fractions containing enriched cross-linked peptides were retained for MS analysis. The peptides were analyzed by a 2-h gradient LC-MS on a Q-Exactive Plus mass spectrometer (Thermo Fisher Scientific).

### Data Protocol
The resulting data files were analyzed by Proteome Discoverer (Thermo Fisher Scientific) for protein identification. The cross-links were identified using the FindXL software package. For each protein sample, the data from the nonenriched and enriched fractions were pulled together into one (nonredundant) list of crosslinks. The false-positive rate for the cross-link in the final list was determined to be 1.5% by decoy analyses with erroneous cross-linker masses.

### Publication Abstract
The DNA damage response is an essential process for the survival of living cells. In a subset of stress-responsive genes in humans, Elongin controls transcription in response to multiple stimuli, such as DNA damage, oxidative stress, and heat shock. Yeast Elongin (Ela1-Elc1), along with Def1, is known to facilitate ubiquitylation and degradation of RNA polymerase II (pol II) in response to multiple stimuli, yet transcription activity has not been examined. We have found that Def1 copurifies from yeast whole-cell extract with TFIIH, the largest general transcription factor required for transcription initiation and nucleotide excision repair. The addition of recombinant Def1 and Ela1-Elc1 enhanced transcription initiation in an in vitro reconstituted system including pol II, the general transcription factors, and TFIIS. Def1 also enhanced transcription restart from TFIIS-induced cleavage in a pol II transcribing complex. In the <i>&#x394;def1</i> strain, heat shock genes were misregulated, indicating that Def1 is required for induction of some stress-responsive genes in yeast. Taken together, our results extend the understanding of the molecular mechanism of transcription regulation on cellular stress and reveal functional similarities to the mammalian system.

### Keywords
Transcription, Rna polymerase ii, Tfiih, Stress response, Def1

### Affiliations
Dept. of Biological Chemistry, The Hebrew University of Jerusalem

### Submitter
Nir Kalisman

### Lab Head
Dr Nir Kalisman
Dept. of Biological Chemistry, The Hebrew University of Jerusalem


