### Accession
PXD030544

### Title
Role of H/ACA snoRNA in translation

### Description
Trypanosomes are unicellular parasites that cycle between the insect and mammalian host and is the causative agent of African sleeping sickness and the wasting disease, nagana, in cattle. Previous studies from our lab suggested that individual rRNA modification such as pseudouridine (Y) and 2â€™-O-methylation are developmentally regulated during the complex life cycle of these parasites. In this study we identified that the Y composition of translating ribosomes are different from rRNA derived from total RNA. Ablation of single snoRNA guiding Y in one of the inter-subunit bridges of the ribosome affects its affinity to translate specific mRNA. Mutational analysis in the pseudouridylation pocket of this snoRNA supports for the notion that the guided modification and not the chaperonic activity, is essential for the observed phenotype. Structural probing of rRNA using DMS-seq suggested that a single modification can affect the rRNA structure locally as wells as globally. Quantitative profiling of the composition of mutant ribosomes suggested the absence of eS12 ribosomal protein in the mature ribosomes. Addback experiments validated the role of eS12 during the loss of a single Y in these mutant cells. Orthologous experiments in related leishmania spp. also verify  importance of a single Y in the function of ribosome. Ongoing experiments intend to visualize such ribosomes lacking a single RNA modification and decipher their significance towards ribosome structure.

### Sample Protocol
Samples in Ripa were precipitated in 80% ethanol.The proteins pellets were dissolved in 8.5M Urea, 10mM DTT, 400 mM ammonium bicarbonate (ABC), sonicated twice (90%, 10-10 cycles, 5'), and centrifuged. The proteins in the solution were reduced (60ÂºC for 30 min), modified with 35.2mM iodoacetamide in 100mM ammonium bicarbonate (in the dark, room temperature for 30 min) and digested in 1.5M Urea, 70mM ABC with modified trypsin (Promega) at a 1:50 enzyme-to-substrate ratio, overnight at 37oC. A second digestion was done for 4 hours.  The resulting peptides were desalted using C18 Top tip columns (Glygen) dried and re-suspended in 50mM Hepes (pH 7.9). Labeling by Dimethylation was done in the presence of 100mM NaCBH3 (Sterogene cat#9704 1M), by adding Light Formaldehyde (35% Frutarom cat#5551810, 12.3M ) to one of the samples, and Heavy Formaldehyde (20% w/w, Cambridge Isotope laboratories cat#CDLM-4599-1,6.5M) to the other sample to a final concentration of 200mM. The sample was incubated 1 hour. Neutralization was done with 1M ABC for 30min and equal amounts of the light and heavy peptides were mixed, cleaned on C18 Top tip columns and re-suspended in 0.1% formic acid.   The peptides were resolved by reverse-phase chromatography on 0.075 X 180-mm fused silica capillaries (J&W) packed with Reprosil reversed phase material (Dr Maisch GmbH, Germany). The peptides were eluted with linear 180 minutes gradient of 5 to 28% 15 minutes gradient of 28 to 95% and 25 minutes at 95% acetonitrile with 0.1% formic acid in water at flow rates of 0.15 Î¼l/min. Mass spectrometry was performed by Q-Exactive HFX mass spectrometer (Thermo) in a positive mode using repetitively full MS scan followed by collision induces dissociation (HCD) of the 30 most dominant ions selected from the first MS scan.

### Data Protocol
The mass spectrometry data was analyzed using the Proteome Discoverer 1.4 (Thermo) software searching against the T.brucei from Trytrip v.35 database. Results were filtered with rank 1 peptides and 1% false discovery rate. The ratios were normalized according to the proteins median ratio. Perseus (Mathias Mann's lab) software was used for statistical analysis of the data.

### Publication Abstract
Trypanosomes are protozoan parasites that cycle between insect and mammalian hosts and are the causative agent of sleeping sickness. Here, we describe the changes of pseudouridine (&#x3a8;) modification on rRNA in the two life stages of the parasite using four different genome-wide approaches. CRISPR-Cas9 knock-outs of all four snoRNAs guiding &#x3a8; on helix 69 (H69) of the large rRNA subunit were lethal. A single knock-out of a snoRNA guiding &#x3a8;530 on H69 altered the composition of the 80S monosome. These changes specifically affected the translation of only a subset of proteins. This study correlates a single site &#x3a8; modification with changes in ribosomal protein stoichiometry, supported by a high-resolution cryo-EM structure. We propose that alteration in rRNA modifications could generate ribosomes preferentially translating state-beneficial proteins.

### Keywords
Quantitative proteomics, Snorna, Pseudouridine, Rna modification, Trypanosome

### Affiliations
Bar-Ilan University

### Submitter
Tirza Doniger

### Lab Head
Dr Prof. Shulamit Michaeli
Bar-Ilan University


