### Accession
PXD028846

### Title
Shared and specific functions of Arfs 1-5 at the golgi revealed by systematic knockouts

### Description
DP-ribosylation factors (Arfs) are small GTPases regulating membrane  traffic in the secretory pathway. They are closely related and appear to  have overlapping functions, regulators, and effectors. The functional  specificity of individual Arfs and the extent of redundancy are still  largely unknown. We addressed these questions by CRISPR/Cas9-mediated  genomic deletion of the human class I (Arf1/3) and class II (Arf4/5)  Arfs, either individually or in combination. Most knockout cell lines  were viable with slight growth defects only when lacking Arf1 or Arf4.  However, Arf1+4, and Arf4+5 could not be deleted simultaneously. Class I  Arfs are non-essential and Arf4 alone is sufficient for viability. Upon  Arf1 deletion, the Golgi was enlarged and recruitment of vesicle coats  decreased, confirming a major role of Arf1 in vesicle formation at the  Golgi. Knockout of Arf4 caused secretion of ER-resident proteins,  indicating specific defects in coatomer-dependent ER protein retrieval  by KDEL receptors. The knockout cell lines will be useful tools to study  other Arf-dependent processes.

### Sample Protocol
The mass spectrometer was operated in DDA mode with a total cycle time of ~1 s. Each MS1 scan was followed by high-collision-dissociation of the 10 most abundant precursor ions with dynamic exclusion set to 45 s. For MS1, 3Â·106 ions were accumulated in the Orbitrap over a maximum time of 100 ms and scanned at a resolution of 70,000 FWHM (at 200 m/z). MS2 scans were acquired at a target setting of 10e5 ions, maximum accumulation time of 100 ms and a resolution of 35,000 FWHM (at 200 m/z). Singly charged ions and ions with unassigned charge state were excluded from triggering MS2 events. The normalized collision energy was set to 27%, the mass isolation window to 1.4 m/z, and one microscan was acquired for each spectrum

### Data Protocol
Acquired raw-files were imported into Progenesis QI software (v2.0, Nonlinear Dynamics Limited) to extract peptide precursor ion intensities across all samples applying the default parameters. The generated mgf file was searched using MASCOT against a human database (consisting of 41484 forward and reverse protein sequences downloaded from Uniprot on 20200417) and 392 commonly observed contaminants using the following search criteria: full tryptic specificity was required (cleavage after lysine or arginine residues, unless followed by proline); 3 missed cleavages were allowed; carbamidomethylation was set as fixed modification; oxidation and protein N-terminal acetylation were applied as variable modifications; mass tolerance of 10 ppm (precursor) and 0.02 Da (fragments). The database search results were filtered using the ion score to set the false discovery rate to 1% on the peptide and protein level, respectively, based on the number of reverse protein sequence hits in the dataset. Results from label-free quantitation were processed using the SafeQuant R package v.2.3.2. to obtain peptide relative abundances.

### Publication Abstract
None

### Keywords
Gga, Ap1, Arf gtpases, Copi, Golgi, Secretory pathway

### Affiliations
Biozentrum, University of Basel
Proteomics Core Facility, Biozentrum, University of Basel, Switzerland

### Submitter
Katarzyna Buczak

### Lab Head
Dr Alexander Schmidt
Proteomics Core Facility, Biozentrum, University of Basel, Switzerland


