### Accession
PXD014236

### Title
Rpn4- and Proteasome- Mediated Yeast Response to Ethanol Stress: a View from a Proteome

### Description
Distilled spirits production using S. cerevisiae requires understanding the mechanisms of yeast cell response to the alcohol stress. Reportedly, specific mutations in genes of the ubiquitin-proteasome system, e.g. RPN4, may result in strains exhibiting either hyper-resistance or increased sensitivity to different alcohols. In this work, we studied Rpn4-dependent response of different yeast strains to short-term ethanol exposure. Three S. cerevisiae strains were used: wild-type (WT), mutant strain with RPN4 gene deletion (rpn4-delta), and mutant strain with decreased proteasome activity due to PRE1 deregulation (YPL). The resistance tests demonstrated an increased sensitivity of mutant strains to ethanol compared with WT. Comparative proteomics analysis revealed significant differences in molecular responses to ethanol between different strains. GO analysis of proteins upregulated in WT showed enrichments represented by oxidative and heat responses, protein folding/unfolding and protein degradation. Enrichment of at least one of these responses was not observed in mutant strains. At the same time, trehalose synthesis and accumulation of stress granules were enhanced in the mutant strains. We suggest that these pathways could partially compensate for the failure of ubiquitin-proteasome system to cope with the ethanol stress. Also, we suggest impaired compensatory activation of autophagic degradation system in rpn4-delta strain and propose that Rpn4 can be a regulator for autophagy upon ethanol stress. These findings can be a basis for creating genetically modified yeast strains resistant to high levels of alcohol, being further used for fermentation in ethanol production.

### Sample Protocol
Description of yeast strains. Wild-type yeast strain BY4741 and its derivative rpn4-delta were obtained from Euroscarf (Oberursel, Germany). Rpn4-delta has deletion of RPN4 gene encoding a transcriptional regulator for the proteasomal genes. This strain exhibits decreased level of proteasomes, low protein proteolysis rates, and displays sensitivity to various stress conditions. YPL strain was obtained in-house using BY4742 strain and has pre1-8 mutation in the promoter of PRE1 encoding the essential structural subunit of the 20S proteolytic proteasome complex. This mutation is the substitution of Rpn4p binding site (PACE) to the site of PstI restriction endonuclease. Such mutation leads to PRE1 down-regulation followed by a decrease in proteasome activity down to the level of rpn4-delta strain. Accordingly, the activity of Rpn4p in YPL strain increases due to Rpn4 stabilization. Note also, that YPL strain is sensitive to proteotoxic stress and resistant to genotoxic stress. Growth conditions and ethanol stress induction. Yeast cells subjected to LC-MS/MS analysis were prepared as follows. Overnight yeast cultures grown on YPD (yeast extract, peptone, dextrose) media were diluted to OD600 = 0.02 in the fresh YPD media and then grown to log phase (OD600 ~ 1) at 30C. Ethanol was added to the cell media to a final concentration of 7% (v/v) and incubation was continued for 1.5 hours at 30C. Cells were divided into aliquots, each containing 1e8 cells, then, washed in PBS buffer and pelleted by centrifugation. The supernatant was quantitatively removed and cell pellets were stored at -80C until further use. Sample preparation for LC-MS/MS. Aliquots were resuspended in 100 uL of lysis buffer containing 0.1 % w/v ProteaseMAX Surfactant (Promega, USA) in 50 mM ammonium bicarbonate, and 10 % v/v ACN. Each cell lysate was stirred for 60 min at 550 rpm at room temperature. Cells were lysed using homogenizer Bandelin Sonopuls HD2070 (Bandelin Electronic, Berlin, Germany) by sonication for 2 minutes at each 30, 60, 80 % amplitudes on ice. The supernatant was collected after centrifugation at 13 000 rpm for 10 min at room temperature (Centrifuge 5415R; Eppendorf, Hamburg, Germany). Total protein concentration was measured using BCA assay. Protein extracts were reduced in 10 mM DTT at 56C for 20 min and alkylated in 10 mM IAA at room temperature for 30 min in the dark. Then, samples were digested overnight at 37C using trypsin protease (Sequencing Grade Modified Trypsin, Promega, Madison, WI, USA) added at a ratio of 1:50 w/w. Enzymatic digestion was terminated by addition of acetic acid (5 % w/v). After the reaction was stopped, the samples were shaken (550 rpm) for 25 min at room temperature followed by centrifugation at 13 000 rpm for 10 min at 20C (Centrifuge 5415R; Eppendorf, Germany). Supernatant was dried in SpeedVac at 45C. Peptides were stored at -80C until the LC-MS/MS analysis.LC-MS/MS analysis. Samples were desalted prior to LC-MS/MS analysis using Oasis cartridges for solid phase extraction (Oasis HLB, 1 cc, 10 mg, 30 um particle size, Waters). Then, the peptide concentration for each sample was measured using BCA assay. Loaded sample quantity was 3 ug per injection. LC-MS/MS analysis was performed using Orbitrap Q Exactive HF mass spectrometer (Thermo Fisher Scientific, San Jose, CA, USA) coupled with UltiMate 3000 nanoflow LC system (Thermo Fisher Scientific, Germering, Germany). Mass spectrometry measurements were performed using data-dependent acquisition (DDA) top 15 mode. Trap column u-Precolumn C18 PepMap100 (5 um, 300 um i.d. 5 mm, 100 A) (Thermo Fisher Scientific, USA) and analytical column EASY-Spray PepMap RSLC C18 (2 um, 75 um, i.d. 500 mm, 100 A) (Thermo Fisher Scientific, USA) were employed for separations. The column temperature was set to 50C. Mobile phases were as follows: (A) 0.1 % FA in water; (B) 95 % ACN, 0.1 % FA in water. Samples were eluted using the gradient from 5 % B to 45 % B for 120 min at 270 nL/min flow rate.

### Data Protocol
Protein identification and quantitation. Raw files were converted to MGF using MSConvert utility from ProteoWizard software. X!Tandem (version Cyclone 2012.10.01.1) searches were run against the above-described customized proteomic database using the following parameters: precursor mass tolerance of Â±10 ppm, fragment mass tolerance of Â±0.01 Da, one allowed missed cleavage site for trypsin and fixed cysteine carboxyamidomethylation. Open search profiling was used to optimize a set of potential modifications for the subsequent database close search. Because the open search profiles do not return the abundant mass shifts, variable modifications were switched off. Pepxmltk was used to convert X!Tandem output files (t.xml) into pep.xml format. Search results were then filtered to 1 % false discovery rate (FDR) at peptide-spectrum match (PSM) level using MP score (version 1.0). After post-search validation, more than 15,000 PSMs, 11,000 peptide identifications, and 2000 protein groups were identified per each sample replicate. NSAF quantitation method was used for label-free quantitation. Protein abundances were transformed into log scale and normalized to eliminate technical biases. LFQ values for proteins undetected in some sample replicates were imputed by (10^-3 * minimal protein abundance) from the respective LC-MS/MS replicate.

### Publication Abstract
Distilled spirits production using Saccharomyces cerevisiae requires understanding of the mechanisms of yeast cell response to alcohol stress. Reportedly, specific mutations in genes of the ubiquitin-proteasome system, e.g., RPN4, may result in strains exhibiting hyper-resistance to different alcohols. To study the Rpn4-dependent yeast response to short-term ethanol exposure, we performed a comparative analysis of the wild-type (WT) strain, strain with RPN4 gene deletion (rpn4-&#x394;), and a mutant strain with decreased proteasome activity and consequent Rpn4 accumulation due to PRE1 deregulation (YPL). The stress resistance tests demonstrated an increased sensitivity of mutant strains to ethanol compared with WT. Comparative proteomics analysis revealed significant differences in molecular responses to ethanol between these strains. GO analysis of proteins upregulated in WT showed enrichments represented by oxidative and heat responses, protein folding/unfolding, and protein degradation. Enrichment of at least one of these responses was not observed in the mutant strains. Moreover, activity of autophagy was not increased in the RPN4 deletion strain upon ethanol stress which agrees with changes in mRNA levels of ATG7 and PRB1 genes of the autophagy system. Activity of the autophagic system was clearly induced and accompanied with PRB1 overexpression in the YPL strain upon ethanol stress. We demonstrated that Rpn4 stabilization contributes to the PRB1 upregulation. CRISPR-Cas9-mediated repression of PACE-core Rpn4 binding sites in the PRB1 promoter inhibits PRB1 induction in the YPL strain upon ethanol treatment and results in YPL hypersensitivity to ethanol. Our data suggest that Rpn4 affects the autophagic system activity upon ethanol stress through the PRB1 regulation. These findings can be a basis for creating genetically modified yeast strains resistant to high levels of alcohol, being further used for fermentation in ethanol production.

### Keywords
Rpn4, Ethanol, Shotgun proteomics, Cell stress, S. cerevisiae

### Affiliations
V.L. Talrose Institute for Energy Problems of Chemical Physics
V.L. Talrose Institute for Energy Problems of Chemical Physics, N.N. Semenov Federal Research Center of Chemical Physics, Russian Academy of Sciences, 119334 Moscow, Russia

### Submitter
julia Bubis

### Lab Head
Dr Mikhail Gorshkov
V.L. Talrose Institute for Energy Problems of Chemical Physics, N.N. Semenov Federal Research Center of Chemical Physics, Russian Academy of Sciences, 119334 Moscow, Russia


