### Accession
PXD013730

### Title
Trrap-dependent lysine acetylation prevents neurodegeneration by modulating Sp1 transcriptional activity

### Description
The acetylation levels of histones and other proteins change during aging and have been linked to neurodegeneration. Here we show that deletion of the histone acetyltransferase (HAT) co-factor Trrap specifically impairs the function of the transcription factor Sp1 and the recruitment of HATs to Sp1 target genes. Modulation of Sp1 function by Trrap acts as a hub regulating multiple processes involved in neuron and neural stem cells function and maintenance including microtubule dynamics and the Wnt signaling pathway. Consistently, Trrap conditional mutants exhibit all hallmarks of neurodegeneration including dendrite retraction and axonal swellings, neuron death, astrogliosis, microglia activation, demyelination and decreased adult neurogenesis. Our results uncovered a novel functional network, essential to prevent neurodegeneration, and involving the specific regulation of Sp1 transcription factor by Trrap-HAT.

### Sample Protocol
First, homogenates of the cortex tissues were prepared using the bead-beating device (24 tissue homogenizer) from Precellys.  Frozen tissue was transferred on ice to bead-beating tubes (Precellys CKMix, 0.5 mL) containing ice-cold PBS with Protease and Phosphatase Inhibitor cocktail (Roche) and beaten for 2 cycles of 20s @6000rpm with a 30 s break at 4 °C. Homogenates were prepared at an estimated protein concentration of 10 µg/µL; based on 5% protein content of fresh brain tissues by weight.  A volume of homogenate corresponding to approximately 500 µg protein was transferred to 1.5 mL Eppendorf tubes and was taken further for lysis.  Lysis was carried out by resuspension of the homogenate in lysis buffer (final concentration 4 % SDS, 0.1 M HEPES (pH8), 50 mM DTT) to a final protein concentration of 1 µg/µL, followed by followed by sonication in a Bioruptor (Diagenode) (10 cycles,  1 min ON/ 30 s OFF, 20 °C). Samples were heated (95 °C, 10 min), and sonication steps repeated. The lysates were clarified by brief centrifugation, incubated with iodacetamide, 15 mM) at RT in the dark. Each sample was treated with 4 volumes ice cold acetone to precipitate the proteins (overnight, -20 °C). The samples were centrifuged at 20800 xg (30 min, 4 °C). Supernatant was removed and the pellets washed twice with 400 µl of ice cold 80% acetone/20% water.  Pellets were air-dried before dissolving in digestion buffer (3 M urea in 0.1 M HEPES, pH 8) at 1 µg/µl. A 1:100 w/w amount of LysC (Wako sequencing grade) was added to each sample before incubation (4 h, 37 °C, 1000 rpm). Samples were diluted 1:1 with milliQ water and incubated with a 1:100 w/w amount of trypsin (Promega sequencing grade) (overnight, 37 °C, 650 rpm). Digests were acidified with 10 % trifluoroacetic acid and desalted with Waters Oasis HLB µElution Plate 30µm in the presence of a slow vacuum according to manufacturer’s instructions. The eluates were dried down with the speed vacuum centrifuge. Peptide labelling with TMT and subsequent high pH fractionation and LC-MS were carried out as described in detail in Buczak et al, 2018.  Briefly, peptide samples, obtained from the digestion were labelled with TMT-10plex isobaric mass tags (Thermo) according to the manufacturer’s instructions.  Equal amounts of the labelled peptides from the 10 samples (5 replicates each condition) were mixed, desalted and pre-fractionated into 16 fractions using high pH reverse phase fractionation on an Agilent Infinity 1260 HPLC, before each fraction was measured individually by nano-LC-MS on an Orbitrap Fusion Lumos employing SPS-MS3 data acquisition.

### Data Protocol
The fraction data were subsequently searched together in Mascot 2.5.1 (Matrix Science) using Proteome Discoverer 2.0 (Thermo) against the Swissprot Mus musculus database (2016, 16756 entries) and a list common contaminants.  Reporter ion intensity values for the PSMs were exported and processed using in-house written R scripts to remove common contaminants and decoy hits. Additionally only PSMs having reporter ion intensities above 1 × 103 in all the relevant TMT channels were retained for quantitative analysis described in detail in Buczak et al, 2018.  Briefly, reporter ion (TMT) intensities were log2-transformed and normalized. Peptide-level data were summarized into their respective protein groups by taking the median value. For differential protein expression, the five replicates of the two conditions respectively within the TMT10-plex were taken together. Protein ratios were calculated for all the protein groups quantified with at least 2 peptides.

### Publication Abstract
Brain homeostasis is regulated by the viability and functionality of neurons. HAT (histone acetyltransferase) and HDAC (histone deacetylase) inhibitors have been applied to treat neurological deficits in humans; yet, the epigenetic regulation in neurodegeneration remains elusive. Mutations of HAT cofactor TRRAP (transformation/transcription domain-associated protein) cause human neuropathies, including psychosis, intellectual disability, autism, and epilepsy, with unknown mechanism. Here we show that Trrap deletion in Purkinje neurons results in neurodegeneration of old mice. Integrated transcriptomics, epigenomics, and proteomics reveal that TRRAP via SP1 conducts a conserved transcriptomic program. TRRAP is required for SP1 binding at the promoter proximity of target genes, especially microtubule dynamics. The ectopic expression of Stathmin3/4 ameliorates defects of TRRAP-deficient neurons, indicating that the microtubule dynamics is particularly vulnerable to the action of SP1 activity. This study unravels a network linking three well-known, but up-to-date unconnected, signaling pathways, namely TRRAP, HAT, and SP1 with microtubule dynamics, in neuroprotection.

### Keywords
Wnt signalling, Neurodegeneration, Trrap, Lysine acetylation, Sp transcription factors

### Affiliations
The Francis Crick Institute
Leibniz Institute on Aging Fritz Lipmann Institute (FLI)  Beutenbergstrasse 11  07745 Jena Germany

### Submitter
Joanna Kirkpatrick

### Lab Head
Dr Zhao-Qi Wang
Leibniz Institute on Aging Fritz Lipmann Institute (FLI)  Beutenbergstrasse 11  07745 Jena Germany


