### Accession
PXD023730

### Title
RNAseq and quantitative proteomic analysis of Dictyostelium knock-out cells lacking the core autophagy proteins ATG9 and/or ATG16

### Description
Autophagy is an evolutionary ancient mechanism that sequesters substrates for degradation within autolysosomes. The process is driven by many autophagy-related (ATG) proteins, including the core members ATG9 and ATG16. Here, we applied RNAseq and tandem mass tag (TMT) proteomic approaches to identify differentially expressed genes (DEGs) and proteins (DEPs) in ATG9‾, ATG16‾ and ATG9‾/16‾ strains in comparison to AX2 wild-type cells. In total, we identified 332 (279 up and 53 down), 639 (487 up and 152 down) and 260 (114 up and 146 down) DEGs and 124 (83 up and 41 down), 431 (238 up and 193 down) and 677 (347 up and 330 down) DEPs in ATG9‾, ATG16‾ and ATG9‾/16‾ strains, respectively. Thus, in the single knock-out strains, the number of DEGs was higher than the number of DEPs while in the double knock-out strain the number of DEPs was higher. Comparison of RNAseq and proteomic data further revealed, that only a small proportion of the transcriptional changes were reflected on the protein level. This suggests that most changes in protein abundance were not caused by transcriptional changes but were rather due to changes in protein homeostasis. Gene ontology (GO) analysis revealed an enrichment of DEPs involved in lipid metabolism and oxidative phosphorylation. Furthermore, we found increased expression of the anti-oxidant enzymes glutathione reductase (gsr) and catalase A (catA) in ATG16‾ and ATG9‾/16‾ cells, respectively, indicating adaptation to excess reactive oxygen species (ROS). Our study provides the first combined transcriptome and proteome analysis of ATG9‾, ATG16‾ and ATG9‾/16‾ cells and adds to our knowledge of functions of these proteins.

### Sample Protocol
Dictyostelium cell pellets were lysed with SDT buffer (4 % SDS, 1 mM DTT, 100 mM Tris-HCl, pH 7.6) and then boiled for 15 min at 100 °C. The cell suspension was centrifuged at 14,000 xg for 40 min and the supernatant passed through 0.22 μm filters. The protein concentration of the filtered supernatant was determined using the BCA Protein Assay Kit (Bio-Rad, USA), and purity was determined by SDS-PAGE. 200 µg protein of each sample was digested with 4 μg trypsin (V5280, Promega) at 37 °C for 16 h. The resulting tryptic peptides were labeled using the TMT Reagent Kit (Thermo Fisher Scientific, USA) according to the manufacturer’s instructions.

### Data Protocol
The data files produced by 15 fractions MS/MS were processed by Proteome Discoverer 1.4 and searched by Mascot 2.2 (Matrix Science, MA) against 12,746 D. discoideum protein-coding sequences deposited in the UniProt database. Each of the identified proteins contained at least two unique peptides. Protein quantification was accomplished by correlating the relative intensities of reporter ions extracted from tandem mass spectra to that of the peptides selected for MS/MS fragmentation. To determine the DEPs between different strains, a fold change > 1.2 and < 0.83, and a p-value < 0.05 were considered to represent up- or down-regulation in two comparable groups, respectively.

### Publication Abstract
None

### Keywords
Tandem mass tag (tmt), Dictyostelium discoideum, Rnaseq, Atg16, Parallel reaction monitoring (prm), Proteome, Atg9, Autophagy

### Affiliations
Shanxi University, Wucheng Road 92, Taiyuan 030006, China
Shanxi University


### Submitter
Qiuhong Xiong

### Lab Head
Dr Qiuhong Xiong
Shanxi University, Wucheng Road 92, Taiyuan 030006, China


