### Accession
PXD010941

### Title
Enhanced RNA Interactome capture (eRIC) and RNA Interactome capture (RIC) of Jurkat cells (DSMZ, ACC-282).

### Description
1.0 â€“ 1.3 x 10e8 proliferating Jurkat cells at a density of about 1-1.5 x 10e6 cells/mL were employed per sample. Cells were irradiated or not with 254 nm UV light and subjected to eRIC or RIC to determine the RNA-bound proteome. Data correspond to two biologically independent experiments.

### Sample Protocol
Capture of RNP complexes using eRIC and RIC: Cells were lysed in lysis buffer (20 mM Tris-HCl (pH 7.5), 500 mM LiCl, 1 mM EDTA, 5 mM DTT, 0.5% (w/v) LiDS),  homogenized using a syringe and snap frozen. After thawing, lysates were incubated for 15 min at 60 Â°C (only eRIC), clarified, and incubated 1 h with rotation with 300 ï�­L of equilibrated LNA2.T-coupled beads at 37-40 Â°C (eRIC) or oligo(dT)25 magnetic beads (NEB) at 4 Â°C (RIC). RNP complexes were washed at 40 Â°C (eRIC) or 4 Â°C  (RIC) once with lysis buffer and twice with each of the buffers 1 (20 mM Tris-HCl (pH 7.5), 500 mM LiCl, 1 mM EDTA, 5 mM DTT, 0.1% (w/v) LiDS), 2 (20 mM Tris-HCl (pH 7.5), 500 mM LiCl, 1 mM EDTA, 5 mM DTT, 0.02% (v/v) NP40), and 3 (20 mM Tris-HCl (pH 7.5), 200 mM LiCl, 1 mM EDTA, 5 mM DTT, 0.02% (v/v) NP40). eRIC: bead suspension was incubated in water for 5 min at 40 Â°C and 800 rpm (pre-elution), and proteins were then eluted at 37 Â°C 800 rpm for 30-60 min with RNases A/T1 in 10 mM Tris-HCl(pH 7.5), 0.15 mM NaCl, 5mM DTT, 0.01% NP40. RIC: RNP complexes were eluted in 165 ï�­L of elution buffer (20 mM Tris-HCl (pH 7.5), 1 mM EDTA) for 5 min at 55 Â°C and 800 rpm and RNA was digested as for eRIC. Sample Preparation for Mass Spectrometry and TMT labeling: Captured proteins were reduced in 10 mM DTT in 50 mM HEPES pH 8.5 at 56 Â°C for 30 min, and alkylated with 20 mM 2-chloroacetamide in 50 mM HEPES pH 8.5 for 30 min at RT. Samples were prepared using the SP3 protocol. Proteins were digested by trypsin (Promega) at 37 Â°C overnight using an enzyme to protein ratio of 1:50. Peptides were labelled with TMT10plex Isobaric Label Reagent (Thermo Fisher Scientific) according to the manufacturerâ€™s instructions. For further sample clean up an OASIS HLB ÂµElution Plate (Waters) was used. Offline high pH reverse phase fractionation was carried out on an Agilent 1200 Infinity high-performance liquid chromatography system, equipped with a Gemini C18 column (3 Î¼m, 110 Ã…, 100 x 1.0 mm, Phenomenex). Mass spectrometry data acquisition: An UltiMate 3000 RSLC nano LC system (Dionex) fitted with a trapping cartridge (Âµ-Precolumn C18 PepMap 100, 5Âµm, 300 Âµm i.d. x 5 mm, 100 Ã…) and an analytical column (nanoEaseâ„¢ M/Z HSS T3 column 75 Âµm x 250 mm C18, 1.8 Âµm, 100 Ã…, Waters) was employed. Trapping was carried out with a constant flow of solvent A (0.1% formic acid in water) at 30 ÂµL/min onto the trapping column for 6 minutes. Subsequently, peptides were eluted via the analytical column with a constant flow of 0.3 ÂµL/min with increasing percentage of solvent B (0.1% formic acid in acetonitrile) from 2% to 4% in 4 min, from 4% to 8% in 2 min, from 8% to 28% in 96 min, and finally from 28% to 40% in 10 min. The outlet of the analytical column was coupled directly to a QExactive plus mass spectrometer (Thermo Fisher Scientific) using the proxeon nanoflow source in positive ion mode. The peptides were introduced into the QExactive plus via a Pico-Tip Emitter 360 Âµm OD x 20 Âµm ID; 10 Âµm tip (New Objective) applying a spray voltage of 2.3 kV. The capillary temperature was set at 320 Â°C. Full mass scan was acquired with mass range 350-1400 m/z in profile mode in the FT with resolution of 70000. The filling time was set at maximum of 100 ms with a limitation of 3x10e6 ions. Data dependent acquisition was performed with the resolution of the Orbitrap set to 35000, with a fill time of 120 ms and a limitation of 2x10e5 ions. A normalized collision energy of 32 was applied. The instrument was set to alternate between MS and data-dependent MS/MS based acquisition with up to a maximum of 10 MS/MS events per cycle. A minimum AGC trigger of 2e2 and a dynamic exclusion time of 30 s were used. The peptide match algorithm was set to â€˜preferredâ€™ and charge exclusion to â€˜unassignedâ€™, charge states +1 and +5 to +8 were excluded. MS/MS data was acquired in profile mode.

### Data Protocol
IsobarQuant and Mascot (v2.2.07) were used to process the acquired data, which was searched against the Uniprot Homo sapiens proteome database UP000005640, which contains common contaminants and reversed sequences. The following modifications were included into the search parameters: Carbamidomethyl (C) and TMT10 (K) (fixed modifications), Acetyl (N-term), Oxidation (M) and TMT10 (N-term) (variable modifications). A mass error tolerance of 10 ppm and 0.02 Da was set for the full scan (MS1) and the MS/MS spectra, respectively. A maximum of two missed cleavages were allowed and the minimal peptide length was set to seven amino acids.  At least two unique peptides were required for protein identification. The false discovery rate on peptide and protein level was set to 0.01. The R programming language (ISBN 3-900051-07-0) was used to analyze the raw output data of IsobarQuant. Potential batch effects were removed using the limma package. A variance stabilization normalization was applied to the raw data using the vsn package. Individual normalization coefficients were estimated for crosslinked and non-crosslinked conditions. Normalized data were tested for differential expression using the limma package. The replicate factor was included into the linear model. For comparisons between crosslinked vs non-crosslinked samples, hits were defined as those proteins with a false discovery rate of less than 5 % and a fold change greater than 2. The R package fdrtool was employed to calculate false discovery rates using the t values from the limma output.

### Publication Abstract
Following the realization that eukaryotic RNA-binding proteomes are substantially larger than anticipated, we must now understand their detailed composition and dynamics. Methods such as RNA interactome capture (RIC) have begun to address this need. However, limitations of RIC have been reported. Here we describe enhanced RNA interactome capture (eRIC), a method based on the use of an LNA-modified capture probe, which yields numerous advantages including greater specificity and increased signal-to-noise ratios compared to existing methods. In Jurkat cells, eRIC reduces the rRNA and DNA contamination by &gt;10-fold compared to RIC and increases the detection of RNA-binding proteins. Due to its low background, eRIC also empowers comparative analyses of changes of RNA-bound proteomes missed by RIC. For example, in cells treated with dimethyloxalylglycine, which inhibits RNA demethylases, eRIC identifies m6A-responsive RNA-binding proteins that escape RIC. eRIC will facilitate the unbiased characterization of RBP dynamics in response to biological and pharmacological cues.

### Keywords
Rbpome, Rna-binding proteins, Rbps, Eric, Ric, Rna interactome

### Affiliations
EMBL
European Molecular Biology Laboratory, Meyerhofstrasse 1, 69117 Heidelberg, Germany.

### Submitter
Frank Stein

### Lab Head
Dr Matthias W. Hentze
European Molecular Biology Laboratory, Meyerhofstrasse 1, 69117 Heidelberg, Germany.


