### Accession
PXD032899

### Title
Development and validation of a secretome-specific prognostic and predictive gene expression signature for colorectal cancer

### Description
Relapse and metastatic progression is a frequent event in colorectal cancer patients detected at early stages. The risk of recurrence requires the development of new biomarkers to correctly predict biological behavior of early stage II and stage III patients and their response to adjuvant chemotherapy. Here, we combined the proteomic quantification of secreted proteins involved in metastasis with a transcriptional analysis to develop a risk score algorithm based on the expression of six genes (SEC6). The SEC6 signature was predictive of survival and recurrence for stage II and III patients in four different datasets including a total of 1534 patients and was also associated with deficient mismatch repair, CpG-island methylator positive status and BRAF mutation. SEC6 was also predictive of beneficial or detrimental effects from 5-Fluorouracil-containing regimes and the improved response to more aggressive chemotherapies based on FOLFOX and FOLFIRI.  In summary, the SEC6 risk-score algorithm may constitute a new tool for decision-making in colorectal cancer management.

### Sample Protocol
Secretome preparation from SW620, KM12SM and KM12L4 cell lines To obtain the secretome, cells were seeded at a density of 6 x 106 cells per plate. Twenty-four hours later, cells were washed with PBS to remove serum and cell debris and cultured with serum-free DMEM for 48 h. Then, medium was collected, centrifuged to remove cell debris and concentrated using Vivaspin devices (Sartorius), at 6000 g for 1 h at 4Â° C. Label-free quantification of secreted proteins  Concentrated supernatants containing 50 Âµg of protein were precipitated with 5 vol of acetone overnight at -20 Â° C.  Next day, the samples were resuspended and cleaned with OMIXs C18.  Samples were reconstituted in 5% glycerol and 2% IPG, pH 3.0-10 (GE Healthcare) and loaded onto 12-well isoelectric focusing strips (13 cm, ImmobilineDryStrip, pH 3.0-10) (GE Healthcare) for OFFGEL fractionation. A total of 6 fractions were obtained from each sample and cleaned again with OMIXs before being reconstituted in 5 Î¼L of a 0.1% FA, 2% ACN solution. The peptides were run on a C18-A1 ASY column (Thermo Fisher Scientific) and then eluted on a C18 Biosphere column (10 cm in length, 75 Âµm in diameter, and 3 Âµm in particle size). After that, they were separated in a 180 min gradient of 0-35% buffer B in buffer A (buffer A: 0.1% FA / 2% ACN; buffer B: 0.1% FA in ACN) at a flow of 300 nL/min on a nanoEasy HPLC with nanoelectrospray (Proxeon). Mass spectra corresponding to the full scan spectrum (m/z 400-1200) were obtained with a resolution of 60,000 and the 15 most intense ions were selected for fragmentation by collision-induced dissociation (CID) in the ionic trap, with collision energy normalized to 35%. A dynamic exclusion window of 30 s was applied. The tolerance for the parent ion was set at 10 ppm and 0.5 Da for the product ions.

### Data Protocol
Label-free quantification of secreted proteins Mass spectrometry data acquired on the LTQ-Orbitrap-Velos were analyzed with Maxquant 1.6.15.0 using LFQ quantification 41. The raw data of the mass spectra (* .raw) corresponding to the cell line experiment were interrogated against the Uniprot human reference proteome UP000005640 (79,038 proteins) database. Statistical and quality control analysis were performed using Perseus 1.6.14 42.  Microarray analysis of differential gene expression Raw data were processed with RMA algorithm included in Transcriptome Analysis Console (Applied Biosystems) for normalization and gene level analysis. For each experimental condition, three independent RNA replicates were processed and analyzed. Fold-changes between experimental conditions were calculated as a ratio between the mean of the gene expression signals. Statistical analysis was performed with e-bayes limma included in Transcriptome Analysis Console (Applied Biosystems). Gene expression results were compared with the KM12SM and SW620 expression values from GSE59857 dataset, which contains the gene expression analysis for 155 CRC cell lines.  Bioinformatics tools The identified and quantified proteins in the proteomic studies were analyzed by systems biology in order to obtain the prediction of the enriched functions. Gene ontology (GO) analysis was performed using g:Profiler web site 43. Kyoto Encyclopedia of Genes and Genomes (KEGG) pathways were analyzed by Gene Set Enrichment Analysis (GSEA) using the WebGestAlt web site 44. Venn diagrams were illustrated using InteractiVenn 45.  Unsupervised hierarchical clustering was performed by Euclidean distance method using Perseus 1.6.14.  Xena platform was used for visualizing and interpreting cancer genomics data

### Publication Abstract
The necessity to accurately predict recurrence and clinical outcome in early stage colorectal cancer (CRC) is critical to identify those patients who may benefit from adjuvant chemotherapy. Here, we developed and validated a gene-based risk-score algorithm for patient stratification and personalised treatment in early stage disease based on alterations in the secretion of metastasis-related proteins. A quantitative label-free proteomic analysis of the secretome of highly and poorly metastatic CRC cell lines with different genetic backgrounds revealed 153 differentially secreted proteins (fold-change &gt;5). These changes in the secretome were validated at the transcriptomic level. Starting from 119 up-regulated proteins, a six-gene/protein-based prognostic signature composed of IGFBP3, CD109, LTBP1, PSAP, BMP1, and NPC2 was identified after sequential discovery, training, and validation in four different cohorts. This signature was used to develop a risk-score algorithm, named SEC6, for patient stratification. SEC6 risk-score components showed higher expression in the poor prognosis CRC subtypes: consensus molecular subtype 4 (CMS4), CRIS-B, and stem-like. High expression of the signature was also associated with patients showing dMMR, CIMP<sup>+</sup> status, and BRAF mutations. In addition, the SEC6 signature was associated with lower overall survival, progression-free interval, and disease-specific survival in stage II and III patients. SEC6-based risk stratification indicated that 5-FU treatment was beneficial for low-risk patients, whereas only aggressive treatments (FOLFOX and FOLFIRI) provided benefits to high-risk patients in stages II and III. In summary, this novel risk-score demonstrates the value of the secretome compartment as a reliable source for the retrieval of biomarkers with high prognostic and chemotherapy-predictive capacity, providing a potential new tool for tailoring decision-making in patient care.

### Keywords
Colorectal cancer, Prognostic signature, Stage stratification, Metastasis, Secreted proteins

### Affiliations
CIB.CSIC
Department of Molecular Biomedicine. Centro de Investigaciones BiolÃ³gicas Margarita Salas. CSIC.

### Submitter
Vivian de los Rios

### Lab Head
Dr Jose Ignacio Casal
Department of Molecular Biomedicine. Centro de Investigaciones BiolÃ³gicas Margarita Salas. CSIC.


