### Accession
PXD016815

### Title
Reactive oxygen species (ROS) production triggers unconventional secretion of antioxidant enzymes

### Description
Nutrient deprivation triggers the release of signal-sequence-lacking antioxidant superoxide dismutase 1 (SOD1). We now report that secreted SOD1 is functionally active and accompanied by export of other antioxidant enzymes such as thioredoxins, (Trx1 and Trx2) and peroxiredoxin Ahp1. Our data reveal that starvation increases mitochondrial activity, which generates higher, but non-toxic, levels of reactive oxygen species (ROS). Inhibiting mitochondrial activity or ROS prevents antioxidants secretion. We suggest that in response to ROS production that can permeate the plasma membrane, the cells respond by secreting antioxidants to prevent ROS-mediated damage in the extracellular space. These findings provide an understanding of why cells secrete signal sequence lacking antioxidant enzymes in response to starvation.

### Sample Protocol
Preparation of samples for total proteome SILAC analysis  Yeast strain BY4742, which is an auxotroph for lysine, was grown for 2 days in 5 ml of synthetic complete media containing either 1 mM L-lysine (light) or 1 mM L-lysine [13C615N2] (heavy) until stationary phase. Cultures were then diluted to OD600 ���0.005 in 50 ml fresh medium of the same composition until reaching a density of 0.2-0.6. The heavy labelled cells were then washed twice in water and starved in 2% potassium acetate for 2.5 hours. It was not possible to mix the cultures prior to lysis, as the starved cells would sense the nutrients from the growing culture. Therefore, 40 OD units each of growing and starved cells were first pelleted, resuspended in ice-cold lysis buffer (6 M urea, 50 mM Tris���Cl pH 8.0, 0.5% SDS, 0.5% NP-40, 10 mM DDT), and then mixed 1:1. Total cell extracts were prepared in triplicates by glass-bead disruption in lysis buffer. After removal of cell debris by centrifugation at 2000��g, proteins were precipitated by the addition of TCA to a final concentration of 20%. The protein pellet was washed with ice-cold acetone which was removed by drying the pellet at 50��C. Proteins were resuspended in a small amount of buffer (6 M urea, 50 mM Tris���Cl pH 7.5) and quantified by the BCA quantification method (Bio-Rad, Hercules, CA). Urea concentration was reduced to 4 M and the pH was adjusted to 8.8 with 25 mM Tris���HCl. Samples were digested with Lys-C in a 1:10 enzyme���protein ratio and desalted using an Oasis Plus HLB cartridge (Waters, Milford, MA).

### Data Protocol
The peptide mixes were analyzed using an Orbitrap Fusion Lumos mass spectrometer (Thermo Fisher Scientific, San Jose, CA, USA) coupled to an EasyLC 1000 (Thermo Fisher Scientific).   Peptides were loaded directly onto the analytical column and were separated by reversed-phase chromatography using a 50-cm column with an inner diameter of 75 ��m, packed with 2 ��m C18 particles spectrometer (Thermo Scientific, San Jose, CA, USA). Chromatographic gradients started at 95% buffer A and 5% buffer B with a flow rate of 300 nl/min for 5 minutes and and gradually increased to 22% buffer B and 78% A in 79 min and then to 35% buffer B and 65% A in 11 min. After each analysis, the column was washed for 10 min with 10% buffer A and 90% buffer B. Buffer A: 0.1% formic acid in water. Buffer B: 0.1% formic acid in acetonitrile.  The mass spectrometer was operated in positive ionization mode with nanospray voltage set at 2.4 kV and source temperature at 275��C. Ultramark 1621 for the was used for external calibration of the FT mass analyzer prior the analyses, and an internal calibration was performed using the background polysiloxane ion signal at m/z 445.1200. The acquisition was performed in data-dependent adquisition (DDA) mode and full MS scans with 1 micro scans at resolution of 120,000 were used over a mass range of m/z 350-1500 with detection in the Orbitrap mass analyzer. Auto gain control (AGC) was set to 1E5 and charge state filtering disqualifying singly charged peptides was activated. In each cycle of data-dependent acquisition analysis, following each survey scan, the most intense ions above a threshold ion count of 10000 were selected for fragmentation. The number of selected precursor ions for fragmentation was determined by the ���Top Speed��� acquisition algorithm and a dynamic exclusion of 60 seconds. Fragment ion spectra were produced via high-energy collision dissociation (HCD) at normalized collision energy of 28% and they were acquired in the ion trap mass analyzer. AGC was set to 1E4, and an isolation window of 1.6 m/z and a maximum injection time of 200 ms were used. All data were acquired with Xcalibur software v4.1.31.9.  Acquired data was analysed with MaxQuant software suite (v1.5.4.3) and the Andromeda search engine was used for peptide identification. Samples were searched against a S. cerevisiae database (version February 2014) containing a list of common contaminants and all the corresponding decoy entries. Endopeptidase LysC was chosen as enzyme and carbamidomethylation (C) was set as a fixed modification, whereas oxidation (M), acetylation (N-terminal), and 13C6,15N2-Lys (+8) were used as variable modifications. Searches were performed using a peptide tolerance of 4.5 ppm, a product ion tolerance of 0.5 Da and the resulting data files were filtered for FDR < 5 %.

### Publication Abstract
Nutrient deprivation triggers the release of signal-sequence-lacking Acb1 and the antioxidant superoxide dismutase 1 (SOD1). We now report that secreted SOD1 is functionally active and accompanied by export of other antioxidant enzymes such as thioredoxins (Trx1 and Trx2) and peroxiredoxin Ahp1 in a Grh1-dependent manner. Our data reveal that starvation leads to production of nontoxic levels of reactive oxygen species (ROS). Treatment of cells with N-acetylcysteine (NAC), which sequesters ROS, prevents antioxidants and Acb1 secretion. Starved cells lacking Grh1 are metabolically active, but defective in their ability to regrow upon return to growth conditions. Treatment with NAC restored the Grh1-dependent effect of starvation on cell growth. In sum, starvation triggers ROS production and cells respond by secreting antioxidants and the lipogenic signaling protein Acb1. We suggest that starvation-specific unconventional secretion of antioxidants and Acb1-like activities maintain cells in a form necessary for growth upon their eventual return to normal conditions.

### Keywords
Yeast, Proteomics, Ros

### Affiliations
Centre for Genomic Regulation, The Barcelona Institute of Science and Technology, 08003 Barcelona, Spain
Centre de Regulació Genòmica

### Submitter
Eduard Sabidó

### Lab Head
Dr Amy Curwin
Centre for Genomic Regulation, The Barcelona Institute of Science and Technology, 08003 Barcelona, Spain


