### Accession
PXD029432

### Title
Integrative functional proteomics identifies PARP1 protein complex differences and PARP inhibitor-mediated DDR and AKT-mTOR signaling in ovarian carcinoma: Phosphoproteomics

### Description
BRCA1/2-deficient ovarian carcinoma (OC) has been shown to be particularly sensitive to PARP inhibitors (PARPis) and BRCA1/2 mutation status is currently used as a predictive biomarker for PARPi therapy. Despite eliciting major clinical benefit for the majority of patients, a significant proportion of BRCA1/2-deficient OC tumors do not respond to PARPis for reasons that are incompletely understood. Using an integrated chemical, phospho- and ADP-ribosylation proteomics approach we sought to develop additional mechanism-based biomarker candidates for PARPi therapy in OC and identify new targets for combination therapy to overcome primary resistance. Chemical proteomics with PARPi baits in a BRCA1-isogenic OC cell line pair, as well as patient-derived BRCA1-profiecient and deficient tumor samples, and subsequent validation by co-immunoprecipitation showed differential PARP1 and PARP2 protein complex composition with Ku70 and Ku80 in PARPi-sensitive, BRCA1-deficient UWB1.289 (UWB) cells compared to PARPi-insensitive, BRCA1-reconstituted UWB1.289 (UWB+B) cells. Global phosphoproteomics and ADP-ribosylation proteomics furthermore revealed that rucaparib induced the cell cycle pathway and NHEJ pathway in UWB cells, but down-regulated ErbB signaling in UWB+B cells. In addition, we observed AKT PARylation and pro-survival AKT-mTOR signaling in UWB+B cells after PARPi treatment. Consistently, synergy of PARPis with DNAPK or AKT inhibitors was more pronounced in UWB+B cells identifying these pathways as actionable vulnerabilities. In conclusion, the combination of chemical proteomics, phosphoproteomics and ADP-ribosylation proteomics can identify differential PARP1/2 complexes and diverse, but actionable drug compensatory signaling in OC.

### Sample Protocol
UWB and UWB+B cells were treated with rucaparib (1 micromolar) for 24 hours. Cells were harvested, lysed with urea lysis buffer, sonicated using a microtip sonicator and digested. Peptides were extracted using the Cell Signaling protocol (#8803 Cell Signaling Technology). TMT labeling, IMAC enrichment, MS analysis and data search were performed as described earlier[Sumi, N.J., et al., Divergent Polypharmacology-Driven Cellular Activity of Structurally Similar Multi-Kinase Inhibitors through Cumulative Effects on Individual Targets. Cell Chemical Biology, 2019. 26(9): p. 1240]. Briefly, samples were labeled using TMT reagents as described by the manufacturer (Thermo Fisher TMTsixplexâ„¢ Isobaric Mass Tagging Kit, #90064). Labeling efficiency was confirmed by LC-MS/MS. After sample combination and lyophilization, peptides were redissolved in 250 ÂµL of 20 mM ammonium formate buffer (pH 10.0). Basic pH reversed-phase liquid chromatography separation was performed on an XBridge column (Waters). Twelve concatenated peptide fractions were dried by vacuum. pSTY peptides were further enriched using IMAC magnetic beads (CST) with a KingFisher robot (ThermoFisher). All samples were spiked with PRTC standard peptides to confirm consistent performance of the LC-MS/MS analyses.
TMT 6plex experiments with 12 bRPLC phosphopeptide fractions were analyzed in duplicate with LC-MS/MS.  UWB1.289 cells (UWB) and UWB1.289 cells plus BRCA1 (BRAC1) were analyzed in triplicate for DMSO controls (channels 126, 127, 128) and 1 micromolar rucaparib treatment for 24 hours (channels 129, 130, 131).

### Data Protocol
The acquired LC-MS/MS data were searched with MaxQuant (pSTY) with the human UniProt database using the embedded search engine Andromeda [Cox, J. and M. Mann, MaxQuant enables high peptide identification rates, individualized p.p.b.-range mass accuracies and proteome-wide protein quantification. Nat Biotechnol, 2008. 26(12): p. 1367-72]. Carbamidomethylated cysteines were set as fixed modification and oxidation of methionine, N-terminal protein acetylation, and phosphorylation of serine, threonine, and tyrosine as variable modifications. Further, the precursor and fragment ion tolerance were set to 20 ppm and 0.05 Da, respectively. The data were then filtered for 2% protein FDR and 5% peptide and site FDR, plus common contaminants (non-human, etc). For the analysis and comparison of TMT 6-plex global pSTY data, the reporter ion intensity was used for the relative quantification of each peptide. IRON normalization of MaxQuant data was performed as described before[Welsh, E.A., et al., Iterative rank-order normalization of gene expression microarray data. BMC Bioinformatics, 2013. 14: p. 153]. The proteins with phosphorylations were filtered for absolute value of log2 fold change (>2 the average standard deviation) and p value (<0.05). The experiment was performed with three biological replicates. Database searches were repeated in Mascot to support this upload.

### Publication Abstract
BRCA1/2-deficient ovarian carcinoma (OC) has been shown to be particularly sensitive to poly (ADP-ribose) polymerase inhibitors (PARPis). Furthermore, BRCA1/2 mutation status is currently used as a predictive biomarker for PARPi therapy. Despite providing a major clinical benefit to the majority of patients, a significant proportion of BRCA1/2-deficient OC tumors do not respond to PARPis for reasons that are incompletely understood. Using an integrated chemical, phospho- and ADP-ribosylation proteomics approach, we sought here to develop additional mechanism-based biomarker candidates for PARPi therapy in OC and identify new targets for combination therapy to overcome primary resistance. Using chemical proteomics with PARPi baits in a BRCA1-isogenic OC cell line pair, as well as patient-derived BRCA1-proficient and BRCA1-deficient tumor samples, and subsequent validation by coimmunoprecipitation, we showed differential PARP1 and PARP2 protein complex composition in PARPi-sensitive, BRCA1-deficient UWB1.289 (UWB) cells compared to PARPi-insensitive, BRCA1-reconstituted UWB1.289+BRCA1 (UWB+B) cells. In addition, global phosphoproteomics and ADP-ribosylation proteomics furthermore revealed that the PARPi rucaparib induced the cell cycle pathway and nonhomologous end joining (NHEJ) pathway in UWB cells but downregulated ErbB signaling in UWB+B cells. In addition, we observed AKT PARylation and prosurvival AKT-mTOR signaling in UWB+B cells after PARPi treatment. Consistently, we found the synergy of PARPis with DNAPK or AKT inhibitors was more pronounced in UWB+B cells, highlighting these pathways as actionable vulnerabilities. In conclusion, we demonstrate the combination of chemical proteomics, phosphoproteomics, and ADP-ribosylation proteomics can identify differential PARP1/2 complexes and diverse, but actionable, drug compensatory signaling in OC.

### Keywords
Brca, Phosphoproteomics, Parp1, Ovarian cancer

### Affiliations
Moffitt Cancer Center
Moffitt Cancer Center Tampa, FL, USA

### Submitter
John Koomen

### Lab Head
Dr Uwe Rix
Moffitt Cancer Center Tampa, FL, USA


