### Accession
PXD045705

### Title
Insights into enhancing Centella asiatica organ cell biofactories via hairy root protein profiling

### Description
The objectives of this study encompassed a thorough exploration of the potential implications of protein profiling in hairy roots, specifically focusing on optimizing and enhancing C. asiatica organ cell biofactories. In this pursuit, we categorized established C. asiatica hairy root lines according to their capacity for centelloside production, classifying them into HIGH, MID, or LOW categories. For comparative analysis, wild adventitious (Adv) roots were extracted from in vitro C. asiatica seedlings and cultivated in solid MS medium at 25°C in complete darkness, serving as control specimens. This meticulous, label-free proteomic analysis enabled the successful identification of several proteins. Our research substantially builds upon and extends the findings presented by Alcalde et al. (2022) (DOI: 10.3389/fpls.2022.1001023). In their study, distinctive morphological and metabolic variations were noted among different C. asiatica hairy root lines. Such differences are presumably attributable to the random incorporation of a selective set of genes from the T-DNA, with particular emphasis on the rol and aux genes.

### Sample Protocol
Four samples, each with an initial fresh weight of one gram, were cultivated from HIGH, MID, LOW hairy root lines, and adventitious roots. These samples were grown on solid MS medium at 25ºC in darkness and were subcultured biweekly. Sampling intended for protein extraction was executed two weeks post the final subculture. The extraction of Total Soluble Protein (TSP) was carried out utilizing the methodology delineated by Wang et al. (2006) (DOI: doi.org/10.1002/elps.200500722), albeit with minor modifications. To briefly describe, 0.5 gram of the hairy root were pulverized in liquid nitrogen and 0.25 g were transferred to a 2 mL tube. This tube was then filled with 10% trichloroacetic acid/acetone and was centrifuged at 10,000g for 5 minutes at 4 °C. Following the removal of the supernatant, the resultant was rinsed with 0.1 M ammonium acetate in 80% methanol and subjected to another round of centrifugation. The sediment was subsequently washed with 80% acetone and air-dried at ambient temperature for approximately 10 minutes. The tube containing the sediment was then filled with phenol/SDS buffer (referenced from Wang et al., 2006), rigorously mixed, and allowed to cool on ice for 5 minutes. Post-centrifugation, the upper phase was isolated into a new tube and combined with 0.1 M ammonium acetate in 80% methanol, left to incubate overnight at -20 °C, and centrifuged at 10,000g for 10 minutes at 4 °C. To conclude, the pellet underwent a singular wash with 100% methanol and another with 80% acetone before being air-dried. The resuspended protein was ultimately quantified using the RC DC Protein Assay Kit II (Bio-Rad, CA, USA), after resuspension in 6 M urea. Protein identification was performed at the PROTEORED© proteomic facility of the University of Alicante (Spain) as detailed in Belchí-Navarro et al. (2019) with some modifications. In brief, protein extract (30 µg) was digested with trypsin, and later peptides were passed through a C18 column (Pierce® C18 Spin Columns, Thermo Scientific) and resuspended in formic acid (0.1%). Peptides were analysed with an Agilent 6550 iFunnel Q-TOF mass spectrometer (Agilent Technologies) coupled to an Agilent 1290 UHPLC chromatograph using an Agilent AdvanceBio Peptide mapping column (2.1 mm x 250 mm, 2.7 µm particle size, operated at 50 °C). Analytes were eluted with a linear gradient of 3-40% ACN in 0.1% formic acid and with a constant flow of 0.4 ml/min.

### Data Protocol
The LC-MS output files were loaded in Progenesis QI for Proteomics (Nonlinear Dynamics) v4.0 label-free analysis software, and protein quantification based on the MS peak intensity was performed. Peptide/protein identification was performed using SpectrumMIll Proteomics Workbench searching the fragmentation spectra against Apiaceae and Rhizobium/Agrobacterium subset of NCBInr database with the following settings: trypsin, up to 2 missed cleavages, carbamidomethylation of Cys as fixed modifications, oxidation of Met as variable modification and mass tolerance of 20ppm for precursor and 50ppm for product ions. Peptide hits were filtered for score ≥ 6 and percent scored peak intensity (%SPI)≥60.

### Publication Abstract
Recent advancements in plant biotechnology have highlighted the potential of hairy roots as a biotechnological platform, primarily due to their rapid growth and ability to produce specialized metabolites. This study aimed to delve deeper into hairy root development in <i>C. asiatica</i> and explore the optimization of genetic transformation for enhanced bioactive compound production. Previously established hairy root lines of <i>C. asiatica</i> were categorized based on their centelloside production capacity into HIGH, MID, or LOW groups. These lines were then subjected to a meticulous label-free proteomic analysis to identify and quantify proteins. Subsequent multivariate and protein network analyses were conducted to discern proteome differences and commonalities. Additionally, the quantification of rol gene copy numbers was undertaken using qPCR, followed by gene expression measurements. From the proteomic analysis, 213 proteins were identified. Distinct proteome differences, especially between the LOW line and other lines, were observed. Key proteins related to essential processes like photosynthesis and specialized metabolism were identified. Notably, potential biomarkers, such as the Tr-type G domain-containing protein and alcohol dehydrogenase, were found in the HIGH group. The presence of ornithine cyclodeaminase in the hairy roots emerged as a significant biomarker linked with centelloside production capacity lines, indicating successful <i>Rhizobium</i>-mediated genetic transformation. However, qPCR results showed an inconsistency with rol gene expression levels, with the HIGH line displaying notably higher expression, particularly of the <i>rolD</i> gene. The study unveiled the importance of ornithine cyclodeaminase as a traceable biomarker for centelloside production capacity. The strong correlation between this biomarker and the <i>rolD</i> gene emphasizes its potential role in optimizing genetic transformation processes in <i>C. asiatica</i>.

### Keywords
Molecular farming, Centella asiatica, Biomarkers, Centelloside production, Rhizobium rhizogenes, Plant biotechnology

### Affiliations
Dpt. Biologia, Sanidad y Medio ambiente Fac. Farmacia y Ciencias de la Alimentacion Univ. Barcelona Av.Joan XIII, 27-31 08028 Barcelona SPAIN
University of Barcelona

### Submitter
Diego Alberto Hidalgo Martinez

### Lab Head
Dr Diego A. Hidalgo
Dpt. Biologia, Sanidad y Medio ambiente Fac. Farmacia y Ciencias de la Alimentacion Univ. Barcelona Av.Joan XIII, 27-31 08028 Barcelona SPAIN


