### Accession
PXD006833

### Title
Human bladder,colon,kidney,liver cancer LC MS/MS

### Description
We propose that enrichment of low-abundance proteins benefits MPs finding. ProteoMinerTM is an equalizing technique by reducing high-abundance proteins and enriching low-abundance proteins in biological liquids. With triton X-100/TBS buffer extraction, ProteoMinerTM enrichment and peptide fractionation, 25 MPs with 73 unique peptides were identified from four human tissues, including 10 membrane proteins and 6 nucleus proteins. Then 18 MPs were confirmed with at least 2 unique peptide identification and 6 MPs were confirmed with 1 unique peptide identification by PRM assay. Hence these results demonstrated ProteoMinerTM as a powerful means in discovery of MPs.

### Sample Protocol
Tissues were ground in a tissue lyser and further lysed by probe sonication in the extraction buffer of PBS (pH 7.5), HEPES (20 mM, pH 8.0), NH4HCO3 (50 mM, pH 8.0) or triton buffer (1% TritonX-100, 20 mM Tris, 150 mM NaCl, pH 7.4) with protease inhibitor cocktail at  0-4 �? Bradford assay was adopted to measure the concentrations of extracted proteins with bovine serum albumin as a standard. Lower abundance proteins were enriched with ProteoMinerTM Protein Enrichment Kits according to manufacturer’s protocol. In brief, the columns were conditioned by the extraction buffer and then the extracted tissue proteins were loaded onto them under gravity speed. The nonspecific binding proteins were washed out by the same extraction buffer, and then the specific binding proteins were eluted by the elution buffer provided in the kits. The amount of input proteins and the volume of ProteoMinerTM beads used were calculated according to the kit manual suggestions. The effects of higher abundance protein depletion were evaluated by protein recovery ratio and the images of SDS-PAGE gels to show the protein pattern changes with ProteoMinerTM treatment.

### Data Protocol
Acquired MS data were converted to MGF files by Proteome Discoverer 1.4 (Thermo Scientific, Waltham, MA) and the exported MGF files were searched using Mascot version2.3.02 against the Swiss-prot human database ( released on 2017_04 with 20183 protein sequences). The false discovery rate (FDR) was set to less than 1% at both PSM and protein level. Trypsin was selected as the specific enzyme with a maximum of two missed cleavages permitted per peptide. Parameters included fixed modification: Carbamidomethylation (C); variable modification: Oxidation (M), Deamidatioin (N, Q). Data were searched with a peptide mass tolerance of 20ppm and a fragment mass tolerance of 0.05Da.

### Publication Abstract
Human Proteome Project (HPP) aims at mapping entire human proteins with a systematic effort upon all the emerging techniques, which would enhance understanding of human biology and lay a foundation for development of medical applications. Until now, 2563 missing proteins (MPs, PE2-4) are still undetected even using the most sensitive approach of protein detection. Herein, we propose that enrichment of low-abundance proteins benefits MPs finding. ProteoMiner is an equalizing technique by reducing high-abundance proteins and enriching low-abundance proteins in biological liquids. With triton X-100/TBS buffer extraction, ProteoMiner enrichment, and peptide fractionation, 20 MPs (at least two non-nested unique peptides with more than eight a.a. length) with 60 unique peptides were identified from four human tissues including eight membrane/secreted proteins and five nucleus proteins. Then 15 of them were confirmed with two non-nested unique peptides (&#x2265;9 a.a.) identified by matching well with their chemically synthetic peptides in PRM assay. Hence, these results demonstrated ProteoMiner as a powerful means in discovery of MPs.

### Keywords
Lc ms/ms", Liver, "bladder, Colon, Kidney

### Affiliations
BGI
Professor and Director Center of Proteomic Analysis BGI-Shenzhen

### Submitter
He Yanbin

### Lab Head
Dr Siqi Liu
Professor and Director Center of Proteomic Analysis BGI-Shenzhen


### SDRF
- organism: Homo sapiens
- organism part: liver, colon, kidney, bladder
- disease: not available
- cell type: not applicable
- label: label free sample
- instrument: Q Exactive HF
- modification parameters: Carbamidomethyl
- modification parameters: Oxidation
- modification parameters: Deamidated

