### Accession
PXD025744

### Title
Proteomic-based research strategy identified laminin subunit alpha 2 as a potential urinary-specific biomarker for the medullary sponge kidney disease

### Description
Medullary sponge kidney (MSK) disease, a rare kidney malformation featuring recurrent renal stones and nephrocalcinosis, continues to be diagnosed using expensive and time-consuming clinical/instrumental tests (mainly urography). Currently, no molecular diagnostic biomarkers are available. To identify such we employed a proteomic-based research strategy utilizing urine from 22 patients with MSK and 22 patients affected by idiopathic calcium nephrolithiasis (ICN) as controls. Notably, two patients with ICN presented cysts. In the discovery phase, the urine of 11 MSK and 10 controls, were randomly selected, processed, and analyzed by mass spectrometry. Subsequently, several statistical algorithms were undertaken to select the most discriminative proteins between the two study groups. ELISA, performed on the entire patientsâ€™ cohort, was used to validate the proteomic results. After an initial statistical analysis, 249 and 396 proteins were identified exclusive for ICN and MSK, respectively. A Volcano plot and ROC analysis, performed to restrict the number of MSK-associated proteins, indicated that 328 and 44 proteins, respectively, were specific for MSK. Interestingly, 119 proteins were found to differentiate patients with cysts (all patients with MSK and the two ICN with renal cysts) from ICN without cysts. Eventually, 16 proteins were found to be common to three statistical methods with laminin subunit alpha 2 (LAMA-2) reaching the higher rank by a Support Vector Machine, a binary classification/prediction scheme. ELISA for LAMA-2 validated proteomic results. Thus, using high-throughput technology, our study identified a candidate MSK biomarker possibly employable in future for the early diagnosis of this disease.

### Sample Protocol
Urine samples from 10 MSK disease and 11 ICN randomly selected patients (including the 2 with kidney cysts as an additional anatomic feature) were used for MS analysis. Samples were solubilized in 0.1 ml of 4% wt/vol sodium dodecylsulfate, 50 mM dithiothreitol, and 0.1 M Tris/HCl, pH 7.6, at 90C for 5 minutes and sonicated and processed by the Filter Aided SamplePreparation procedure using 30k Vivacom filtration devices. Briefly, 50-ml aliquots were mixed with 0.2 ml of 8 M urea in 0.1 M Tris/HCl, pH 8.5 (UT solution), loaded into the filtration devices and centrifuged at 14,000g for 15 minutes. Concentrates were diluted in the devices with 0.2 ml of UT solution and centrifuged again. After centrifugation, concentrates were mixed with 0.1 ml of 50 mM iodoacetamide in UT solution and incubated in darkness at room temperature for 30 minutes followed by centrifugation for 15 minutes. Then the concentrates were diluted with 0.2 ml of 8 M urea in 0.1 M Tris/HCl, pH 8.5, and concentrated again. This step was repeated twice. The samples were then diluted with 120 ml of 40 mM NaHCO3 containing 1 mg of trypsin. After an overnight digestion, the peptides were collected by centrifugation of the filter units for 20 minutes, and the filter device was rinsed with 50 ml 0.5 M NaCl and centrifuged to suppress hydrophobic interactions.

### Data Protocol
Raw MS files were processed with Thermo Scientific Proteome Discoverer software version 1.3 (Thermo Scientific, Waltham, MA). Peak list files were searched by the MASCOT and SEQUEST search engine against the reference human proteome downloaded from Uniprot (http://www.uniprot.org/). The resulting peptide hits were filtered for a maximum 1% false discovery rate using Percolator, the peptide mass deviation was set to 10 ppm and a minimum of 6 amino acids per identified peptide were required. The database search parameters were mass tolerance precursor 20 ppm, mass tolerance fragment Collision-induced dissociation 0.8 Da with a dynamic modification of deamidation, oxidation, and static modification of alkylation with iodoacetamide. For all searches, the option trypsin with 2 missed cleavages was selected. Proteins were grouped by applying the maximum parsimony rule. The match between the runs option was also enabled with a match time window of 0.5 minutes and an alignment time window of 20 minutes. Relative, label-free quantification of proteins was done using MaxQuant software.

### Publication Abstract
None

### Keywords
Biomarker; laminin subunit alpha 2; msk; proteomic

### Affiliations
IRRCS G. Gaslini
IRCCS Gaslini

### Submitter
Martina Bartolucci

### Lab Head
Dr Andrea Petretto
IRRCS G. Gaslini


