### Accession
PXD031532

### Title
CRISPR activation screen identifies BCL-2 proteins and B3GNT2 as drivers of cancer resistance to T cell-mediated cytotoxicity

### Description
The cellular processes that govern tumor resistance to immunotherapy remain poorly understood. To gain insight into these processes, we perform a genome-scale CRISPR activation screen for genes that enable human melanoma cells to evade cytotoxic T cell killing. Overexpression of four top candidate genes (CD274 (PD-L1), MCL1, JUNB, and B3GNT2) confer resistance in diverse cancer cell types and mouse xenografts. By investigating the resistance mechanisms, we find that MCL1 and JUNB modulate the mitochondrial apoptosis pathway. JUNB encodes a transcription factor that downregulates FasL and TRAIL receptors, upregulates the MCL1 relative BCL2A1, and activates the NF-B pathway. B3GNT2 encodes a poly-N-acetyllactosamine synthase that targets >10 ligands and receptors to disrupt interactions between tumor and T cells and reduce T cell activation. Inhibition of candidate genes sensitize tumor models to T cell cytotoxicity. Our results demonstrate that systematic gain-of-function screening can elucidate resistance pathways and identify potential targets for cancer immunotherapy.

### Sample Protocol
Magnetic beads were resuspended in 100 mM Tris pH 7.8, reduced, alkylated and digested with trypsin at 37 ºC overnight.  This solution was subjected to solid phase extraction to concentrate the peptides and remove unwanted reagents followed by injection onto a Shimadzu HPLC with fraction collector. Eight fractions were collected and after concentration were injected on a Waters NanoAcquity HPLC equipped with a self-packed Aeris 3 µm C18 analytical column 0.075 mm by 20 cm, (Phenomenex).  Peptides were eluted using standard reverse-phase gradients.  The effluent from the column was analyzed using a Thermo Orbitrap Elite mass spectrometer (nanospray configuration) operated in a data dependent manner for 54 minutes.

### Data Protocol
The resulting fragmentation spectra were correlated against the known database using Proteome Discover 1.4 (Thermo Fisher Scientific).  Scaffold Q+S (Proteome Software) was used to provide consensus reports for the identified proteins.

### Publication Abstract
None

### Keywords
Melanoma, B3gnt2, Shotgun proteomics, Co-ip

### Affiliations
MIT
Broad Institute

### Submitter
Julia Joung

### Lab Head
Dr Feng Zhang
Broad Institute


