### Accession
PXD027706

### Title
Proteomic anlysis of dental calculus samples from Xiaohe

### Description
This analysis is a part of a larger study of the genomic origins of the Tarim Basin mummies. The dental calculus of seven individuals from Xiaohe contained ruminant dairy proteins, confirming milk consumption by the earliest inhabitants of Xiaohe.

### Sample Protocol
Total protein extractions were performed on dental calculus obtained from seven Xiaohe individuals excavated from layers 4 and 5. Only individuals with calculus deposits >5 mg were analyzed, and 5-10 mg of dental calculus was processed for each sample. Samples were extracted and digested using filter-aided sample preparation (FASP), following decalcification in 0.5M EDTA. Extracted peptides were analyzed by LC-MS/MS using a Q-Exactive mass spectrometer (Thermo Scientific, Bremen, Germany) coupled to an ACQUITY UPLC M-Class system (Waters AG, Baden-DÃ¤ttwil, Switzerland) according to previously described protocols. Potential contamination and sample carryover were monitored through the use of extraction blanks as well as injection blanks between each sample.

### Data Protocol
Tandem mass spectra were converted to Mascot generic files by MSConvert version 3.0.11781 using the 100 most intense peaks in each spectra. All MS/MS samples were analyzed using Mascot (Matrix Science, London, UK; v2.6.0). Mascot was set up to search the SwissProt Release 2019_08 database (560,823 entries) assuming the digestion enzyme trypsin. Mascot was searched with a fragment ion mass tolerance of 0.050 Da and a parent ion tolerance of 10.0 PPM. Carbamidomethyl of cysteine was specified in Mascot as a fixed modification. Deamidated of asparagine and glutamine and oxidation of methionine and proline were specified in Mascot as variable modifications. A subset of samples were analyzed in duplicate, and the results were combined using multidimensional protein identification technology (MUDPIT) prior to analysis.  MS/MS based protein and peptide identifications were validated using Scaffold (version Scaffold_4.9.0, Proteome Software Inc., Portland, OR). Peptide identifications were accepted if they could be established at greater than 86.0% probability to achieve an FDR less than 1.0% by the Peptide Prophet algorithm with Scaffold delta-mass correction. Protein identifications were accepted if they could be established at an FDR of less than 5.0% and contained at least 2 unique peptides. Final protein and peptide FDR was 1.8% and 0.99%. Protein probabilities were assigned by the Protein Prophet algorithm. After establishing the presence of the milk proteins beta-lactoglobulin and alpha-S1-casein using these criteria, we expanded our analysis to accept further milk proteins identified on the basis of single peptides for high scoring PSMs (>60), which resulted in the additional identification of alpha-lactalbumin. Proteins that contained similar peptides and could not be differentiated based on MS/MS analysis alone were grouped to satisfy the principles of parsimony. All samples yielded proteomes typical of a dental calculus oral microbiome, and damage-associated modifications (N,Q deamidation) characteristic of ancient proteins were observed.

### Publication Abstract
None

### Keywords
Lc-ms, Dental calculus

### Affiliations
Max Planck Institute for the Science of Human History Department of Archaeogenetics
Max Planck Institute for the Science of Human History

### Submitter
Ashley Scott

### Lab Head
Dr Christina Warinner
Max Planck Institute for the Science of Human History Department of Archaeogenetics


