### Accession
PXD043096

### Title
Better late than never: optimising the proteomic analysis of field-collected octopus

### Description
Proteomics, the temporal study of proteins expressed by an organism, is a powerful technique that can reveal how organisms respond to biological perturbations, such as disease and environmental stress. Yet, the use of proteomics for addressing ecological questions has been limited, partly due to inadequate protocols for the sampling and preparation of animal tissues from the field. Although RNAlater is an ideal alternative to freezing for tissue preservation in transcriptomics studies, its suitability for the field could be more broadly examined. Moreover, existing protocols require samples to be preserved immediately to maintain protein integrity, yet the effects of delays in preservation on proteomic analyses have not been thoroughly tested. Hence, we optimised a proteomic workflow for wild-caught samples. First, we conducted a preliminary in-lab test using SDS-PAGE analysis on aquaria-reared Octopus berrima confirming that RNAlater can effectively preserve proteins up to 6 h after incubation, supporting its use in the field. Subsequently, we collected arm tips from wild-caught Octopus berrima and preserved them in homemade RNAlater immediately, 3 h, and 6 h after euthanasia. Processed tissue samples were analysed by liquid chromatography tandem mass spectrometry to ascertain protein differences between time delay in tissue preservation, as well as the influence of sex, tissue type, and tissue homogenisation methods. Over 3500 proteins were identified from all tissues, with bioinformatic analysis revealing protein abundances were largely consistent regardless of sample treatment. However, nearly 10 % additional proteins were detected from tissues homogenised with metal beads compared to liquid nitrogen methods, indicating the beads were more efficient at extracting proteins. Our optimised workflow demonstrates that sampling non-model organisms from remote field sites is achievable and can facilitate extensive proteomic coverage without compromising protein integrity.

### Sample Protocol
Tissues were prepared according to the four proposed comparisons - sex, skin, time delay and tissue homogenisation method. Tissues (approximately 5 mg) were homogenised by either immersion in LN2 and crushing into a powder using a mortar and pestle, or by combining metal beads (7X tissue weight) and 375 ÃƒÆ’Ã¢â‚¬Å¡Ãƒâ€šÃ‚Âµl homemade RIPA buffer (Table S1) in a tube using the Bullet Blender Storm 24 (Next Advance, Troy, USA) for tough muscular tissues such as octopus arms. The latter consisted of 1 min bead-beating and 30 seconds of cooling to prevent excess heat generation, which was repeated three more times before centrifugation for 30 minutes at 4 ÃƒÆ’Ã¢â‚¬Å¡Ãƒâ€šÃ‚Â°C, 20000 ÃƒÆ’Ã†â€™ÃƒÂ¢Ã¢â€šÂ¬Ã¢â‚¬ï¿½g. The supernatant was then transferred to cold acetone (6X volume of tissue weight for LN2; 4X for beads; Chem-Supply, Port Adelaide, Australia) and stored overnight at -20 ÃƒÆ’Ã¢â‚¬Å¡Ãƒâ€šÃ‚Â°C. Upon centrifugation (4 ÃƒÆ’Ã¢â‚¬Å¡Ãƒâ€šÃ‚Â°C, 20000 ÃƒÆ’Ã†â€™ÃƒÂ¢Ã¢â€šÂ¬Ã¢â‚¬ï¿½g), the pellets were air-dried and dissolved in 8 M urea (Merck, Darmstadt, Germany) with 50 mM ammonium hydrogen carbonate (Merck) and sonicated twice for a minute. Protein was quantified using an EZQ Protein Quantitation kit (Thermo Fisher Scientific). The solution was combined with 10 mM dithiothreitol (Roche, Basel, Switzerland) for 45 minutes at room temperature followed by the addition of 20 mM iodoacetamide (Honeywell, Charlotte, USA) for a 30 minute in-the-dark incubation at room temperature. Urea was diluted to 1 M using 50 mM ammonium hydrogen carbonate, with 1 ÃƒÆ’Ã¢â‚¬Å¡Ãƒâ€šÃ‚Âµg trypsin (Promega, Madison, USA) added to 50 ÃƒÆ’Ã¢â‚¬Å¡Ãƒâ€šÃ‚Âµg protein for overnight incubation at 37 ÃƒÆ’Ã¢â‚¬Å¡Ãƒâ€šÃ‚Â°C. The digestion was halted by formic acid (Sigma Aldrich, Burlington, USA). Upon peptide clean-up using ZipTips (Merck, Co Wicklow, Ireland), 1 ÃƒÆ’Ã¢â‚¬Å¡Ãƒâ€šÃ‚Âµg of peptide sample was analysed by LC-MS/MS on an Ultimate 3000 RSLCnano or EASY-nLC 1200 system connected to an Orbitrap Exploris 480 mass spectrometer (Thermo Fisher Scientific). Peptides were resuspended in 0.1% formic acid and loaded onto a 25 cm fused silica column (75 ÃƒÆ’Ã¢â‚¬Å¡Ãƒâ€šÃ‚Âµm inner diameter, 360 um outer diameter) heated to 50ÃƒÆ’Ã¢â‚¬Å¡Ãƒâ€šÃ‚Â°C. The column was packed in-house with 1.9 um ReproSil-Pur 120 C18-AQ particles (Dr. Maisch, Ammerbuch, Germany). Peptide separation was conducted over a 70-minute linear gradient (3 to 20% acetonitrile in 0.1% formic acid) at a flow rate of 300 nl/min. Compensation voltages (-50 and -70 V) were applied from a FAIMS Pro interface (Thermo Fisher Scientific) to filter the entry of ionised peptides into the mass spectrometer. For each compensation voltage, the cycle time was limited to 1.5 seconds and the dynamic exclusion period set to 40 seconds. MS scans (m/z 300 to 1500) were measured at resolution 60000 (m/z 200) in positive ion mode. MS/MS scans were acquired at resolution 15000 in a data-dependent manner (minimum threshold of 1 x 104 precursor ions), with peptide fragmentation performed with 27.5% higher-energy collision dissociation.

### Data Protocol
To ensure statistical independence, bioinformatics analyses were conducted one comparison at a time using separate datasets differing by only one variable (e.g. sex). Raw files were processed with Proteome Discoverer 2.5 software (Thermo Fisher Scientific), using the Sequest HT search engine and a UniProt unreviewed ÃƒÆ’Ã‚Â¢ÃƒÂ¢Ã¢â‚¬Å¡Ã‚Â¬Ãƒâ€¦Ã¢â‚¬Å“CephalopodaÃƒÆ’Ã‚Â¢ÃƒÂ¢Ã¢â‚¬Å¡Ã‚Â¬ÃƒÂ¯Ã‚Â¿Ã‚Â½ database (version 2021_03; 73232 protein entries). Carbamidomethylation of cysteine was set as a fixed modification, while variable modifications consisted of methionine oxidation, N-terminus acetylation, N-terminus loss of methionine and N-terminus loss of both methionine and acetylation. Other parameters in this workflow were at default settings i.e. maximum two missed cleavages, with a 10 ppm precursor mass tolerance and 0.1 Da fragment mass tolerance. Protein ratio calculations were based on pairwise ratios using ANOVA, with other parameters at default settings i.e. protein and peptide false discovery rates were set to 0.01, peptide confidence was set to at least ÃƒÆ’Ã‚Â¢ÃƒÂ¢Ã¢â‚¬Å¡Ã‚Â¬Ãƒâ€¹Ã…â€œHighÃƒÆ’Ã‚Â¢ÃƒÂ¢Ã¢â‚¬Å¡Ã‚Â¬ÃƒÂ¢Ã¢â‚¬Å¾Ã‚Â¢ and at least one peptide sequence was required to identify a protein.   This analysis was repeated for other tissue comparisons, with each protein list filtered for Master proteins possessing a ÃƒÆ’Ã‚Â¢ÃƒÂ¢Ã¢â‚¬Å¡Ã‚Â¬Ãƒâ€¹Ã…â€œHighÃƒÆ’Ã‚Â¢ÃƒÂ¢Ã¢â‚¬Å¡Ã‚Â¬ÃƒÂ¢Ã¢â‚¬Å¾Ã‚Â¢ protein false discovery rate confidence to obtain totals of unique proteins for every comparison. To determine if experimental conditions in each comparison were significantly different, the protein lists were further filtered with an additional criterion (p-value ÃƒÆ’Ã‚Â¢ÃƒÂ¢Ã¢â€šÂ¬Ã‚Â°Ãƒâ€šÃ‚Â¤ 0.05 for adjusted abundance ratio) before heat maps (using normalized protein abundances) and volcano plots were generated.

### Publication Abstract
Proteomics, the temporal study of proteins expressed by an organism, is a powerful technique that can reveal how organisms respond to biological perturbations, such as disease and environmental stress. Yet, the use of proteomics for addressing ecological questions has been limited, partly due to inadequate protocols for the sampling and preparation of animal tissues from the field. Although RNAlater is an ideal alternative to freezing for tissue preservation in transcriptomics studies, its suitability for the field could be more broadly examined. Moreover, existing protocols require samples to be preserved immediately to maintain protein integrity, yet the effects of delays in preservation on proteomic analyses have not been thoroughly tested. Hence, we optimised a proteomic workflow for wild-caught samples. First, we conducted a preliminary in-lab test using SDS-PAGE analysis on aquaria-reared Octopus berrima confirming that RNAlater can effectively preserve proteins up to 6 h after incubation, supporting its use in the field. Subsequently, we collected arm tips from wild-caught Octopus berrima and preserved them in homemade RNAlater immediately, 3 h, and 6 h after euthanasia. Processed tissue samples were analysed by liquid chromatography tandem mass spectrometry to ascertain protein differences between time delay in tissue preservation, as well as the influence of sex, tissue type, and tissue homogenisation methods. Over 3500 proteins were identified from all tissues, with bioinformatic analysis revealing protein abundances were largely consistent regardless of sample treatment. However, nearly 10% additional proteins were detected from tissues homogenised with metal beads compared to liquid nitrogen methods, indicating the beads were more efficient at extracting proteins. Our optimised workflow demonstrates that sampling non-model organisms from remote field sites is achievable and can facilitate extensive proteomic coverage without compromising protein integrity.

### Keywords
Wild specimen, Differential protein abundance, Ecological proteomics, Proteomics methods, Liquid chromatography, Cephalopod, Octopus, Mass spectrometry

### Affiliations
University of Adelaide
Environment Institute, University of Adelaide, Adelaide, South Australia, Australia

### Submitter
Qiaz Hua

### Lab Head
Dr Qiaz Hua
Environment Institute, University of Adelaide, Adelaide, South Australia, Australia


