### Accession
PXD001876

### Title
Identification of myristoylated proteins in zebrafish (Danio rerio)

### Description
Multi-functional capture reagents reported here enable robust identification of metabolically-tagged myristoylated proteomes with unprecedented confidence resulting from the combination of chemical probe-based enrichment, and release and direct detection of lipid-modified peptides by MS. Whilst capture reagents containing enzymatically cleavable linkers have been previously reported they typically require an extra proteolytic step and their capacity to enable detection of lipidated peptides has not been demonstrated. Herein, we report the largest database (85 counts) of experimentally validated human proteins that are myristoylated at an endogenous level in living cells. Furthermore, we demonstrate the first profile of myristoylation in a living multicellular organism and the confident identification of over 50 novel targets. Importantly, this is also the first example of analysis of any protein lipidation event during development. Our methodology is novel in analytical/chemical approach, and provides quantitative and dynamic information. This submission concerns myristoylated proteomes of zebrafish origin.

### Sample Protocol
Zebrafish were treated with YnMyr (clickable protein myristoylation probe) or myristic acid control to metabolically tag intracellular proteins via activation by cellular acyl-CoA synthase followed by transfer by N-myristoyl transferases. Zebrafish were harvested, metabolically tagged proteins reacted with a capture reagent via copper-catalyzed alkyne azide cycloaddition, and affinity-enriched. The enriched proteomes were analyzed through on-bead proteolytic digestion, LC-MS/MS-based analysis, and software-aided peptide sequencing and protein identification. Please see DOI: 10.1002/anie.201500342 for details.

### Data Protocol
Selected data sets were also processed with PEAKS7, which as a default performs de novo peptide sequencing prior to database searches, in order to improve the accuracy of the results. The software also searches for common PTMs (PEAKS PTM) and point mutations (SPIDER). Samples originating from zebrafish experiments were searched against the same reference Uniprot database that was used in MaxQuant analyses. Trypsin (specific, up to three missed cleavages allowed) was selected for database searches, and no enzyme was chosen in de novo searches (up to 5 candidates per spectrum reported). The maximal mass error was set to 5 ppm for precursor ions and 0.01 Da for product ions. Carbamidomethylation was selected as a fixed modification, and methionine oxidation as well as the lipid-derived adduct (+463.2907 Da) to any amino acid at peptide N-terminus were set as variable modifications. The maximal number of modifications per peptide was set as five. The false discovery rate was set to 0.01 for peptides and minimum of 1 unique peptide per protein was required. For N-terminally modified peptides, b1 ions were required.  Please see DOI: 10.1002/anie.201500342 for details.

### Publication Abstract
Novel multifunctional reagents were applied in combination with a lipid probe for affinity enrichment of myristoylated proteins and direct detection of lipid-modified tryptic peptides by mass spectrometry. This method enables high-confidence identification of the myristoylated proteome on an unprecedented scale in cell culture, and allowed the first quantitative analysis of dynamic changes in protein lipidation during vertebrate embryonic development.

### Keywords
Lc-ms/ms, Zebrafish, Myristoylated peptides, Protein myristoylation, Chemical probes

### Affiliations
Department of Chemistry, Imperial College London, Exhibition Road, London SW7 2AZ, UK
Imperial College London

### Submitter
Remigiusz Serwa

### Lab Head
Dr Edward W. Tate
Department of Chemistry, Imperial College London, Exhibition Road, London SW7 2AZ, UK


