### Accession
PXD025634

### Title
Cell Data-Indenpendent acquisition LC-MS/MS

### Description
In this work, a microwell-chip was prepared and modified. The microwell-chip was used for extraction of metabolites and subsequent protein digestion. Next, direct electrospray ionization mass spectrometry (ESI-MS) was adopted for metabolome identification and a data independent acquisition (DIA)-MS approach was established for simultaneous proteome profiling and phosphoproteome analysis. In particular, application of this strategy provides a multi-omics view of cellular changes.

### Sample Protocol
The microwell chips were fabricated using microelectromechanical (MEMS) techniques of photoetching, silicon deep reactive ion etching and laser microdissection. The inner surface of the microwells was covalently modified by highly hydrophilic silane derivatives. The picked single-cell was immediately transferred into the microwell on the chip and extracted the metabolites from the cells following 6h trypsin digestion.

### Data Protocol
The raw files were loaded to PEAKS Studio X+. The precursor mass and fragment ion were 15 ppm and 0.02 Da and trypsin was set as the digestion enzyme. N-terminal protein acetylation and methionine oxidation were set as the variable modifications. Phosphorylation (STY) was added in the variable modifications only for phosphoproteome studies. The maximum allowed missed cleavages per peptide and maximum allowed variable PTM per peptide were both set at three. For DIA-based proteins and peptides identification by PEAKS Studio X+, the same searching parameters were applied as that used in building the spectral libraries. 1% FDR at the peptide level and protein Score (-10lgP) â‰¥ 20 was applied. Possible metabolites were searched in the metabolome database (HMDB) (http://www.hmdb.ca/), METLIN metabolite database (https://metlin.scripps.edu/) and Lipidmaps (https://www.lipidmaps.org/) based on the accurate mass, isotopic profiles and MS/MS fragmentation patterns.

### Publication Abstract
Single-cell-based genomics and transcriptomics analysis have revealed substantial cellular heterogeneity among seemingly identical cells. Knowledge of the cellular heterogeneity at multiomics levels is vital for a better understanding of tumor metastasis and drug resistance, stem cell differentiation, and embryonic development. However, unlike genomics and transcriptomics studies, single-cell characterization of metabolites, proteins, and post-translational modifications at the omics level remains challenging due to the lack of amplification methods and the wide diversity of these biomolecules. Therefore, new tools that are capable of investigating these unamplifiable "omes" from the same single cells are in high demand. In this work, a microwell chip was prepared and the internal surface was modified for hydrophilic interaction liquid chromatography-based tandem extraction of metabolites and proteins and subsequent protein digestion. Next, direct electrospray ionization mass spectrometry was adopted for single-cell metabolome identification, and a data-independent acquisition-mass spectrometry approach was established for simultaneous proteome profiling and phosphoproteome analysis without phosphopeptide enrichment. This integrated strategy resulted in 132 putatively annotated compounds, more than 1200 proteins, and the first large-scale phosphorylation data set from single-cell analysis. Application of this strategy in chemical perturbation studies provides a multiomics view of cellular changes, demonstrating its capability for more comprehensive investigation of cellular heterogeneity.

### Keywords
Lc-ms/ms, Dia, Cell

### Affiliations
National center for protein sciences(Beijing)
National Center for Protein Sciences Beijing, State Key Laboratory of Proteomics, Beijing Proteome Research Center, Beijing Institute of Lifeomics, Beijing

### Submitter
Yuanyuan Li

### Lab Head
Dr Weijie Qin
National Center for Protein Sciences Beijing, State Key Laboratory of Proteomics, Beijing Proteome Research Center, Beijing Institute of Lifeomics, Beijing


