### Accession
PXD001949

### Title
AP-MS of GFP-MCM2 in the cellular response to etoposide induced DNA damage.

### Description
SILAC quantitative proteomics using immunoprecipitation of GFP-tagged fusion proteins to identify proteins interacting with the MCM complex, and quantify changes in interactions in response to DNA damage. U2OS cells expressing GFP-MCM2 were used to purify the MCM complex following treatment or not with etoposide, a topoisomerase II inhibitor causing DNA damage.

### Sample Protocol
Trypsin digested peptides were separated using a Dionex Ultimate 3000 nanoHPLC system. 10 ul of sample (a total of 2 ug) in 1% (vol/vol) formic acid was loaded with a constant flow of 4 ul/min onto an Acclaim PepMap100 C18 column (0.3 mm id x 5 mm, Dionex Corporation). After trap enrichment peptides were eluted off onto a PepMap C18 nano column (75 um x 50 cm, Dionex Corporation) with a linear gradient of 5-35% solvent B (90% acetonitrile with 0.1% formic acid) over 240 minutes with a constant flow of 200 nl/min. The HPLC system was coupled to an OrbiTrap QExactive mass spectrometer (Thermo Fisher Scientific Inc) via an EasySpray source. The spray voltage was set to 2.0 kV and the temperature of the column was set to 40 oC. Full scan MS survey spectra (m/z 350-1600) in profile mode were acquired in the Orbitrap with a resolution of 70,000 after accumulation of 1,000,000 ions. The ten most intense peptide ions from the preview scan in the Orbitrap were fragmented by collision induced dissociation (normalised collision energy 35% and resolution of 17,500) after the accumulation of 50,000 ions. Maximal filling times were 250 ms for the full scans and 60 ms for the MS/MS scans. Precursor ion charge state screening was enabled and all unassigned charge states as well as singly, 7 and 8 charged species were rejected. The dynamic exclusion list was restricted to a maximum of 500 entries with a maximum retention period of 40 seconds and a relative mass window of 10 ppm. The lock mass option was enabled for survey scans to improve mass accuracy. Data were acquired using the Xcalibur software.

### Data Protocol
Data were processed, searched and quantified using the MaxQuant software package version 1.4.1.2 employing the Human Uniprot database (16/07/2013, 88,354 entries). The settings used for the MaxQuant analysis were: 2 miscleavages were allowed; fixed modification was carbamidomethylation on cysteine; enzymes were Trypsin (K/R not before P); variable modifications included in the analysis were methionine oxidation and protein N-terminal acetylation. A mass tolerance of 7 ppm was used for precursor ions and a tolerance of 20 ppm was used for fragment ions. The re-quantify option was selected to calculate the ratio for isotopic patterns not assembled in SILAC pairs as often observed during pulldown experiments. To achieve reliable identifications, all proteins were accepted based on the criteria that the number of forward hits in the database was at least 100-fold higher than the number of reverse database hits, thus resulting in a false discovery rate (FDR) of less than 1%. A minimum of 2 peptides were quantified for each protein. Protein isoforms and proteins that cannot be distinguished based on the peptides identified are grouped and displayed on a single line with multiple accession numbers.

### Publication Abstract
The minichromosome maintenance complex (MCM) proteins are required for processive DNA replication and are a target of S-phase checkpoints. The eukaryotic MCM complex consists of six proteins (MCM2-7) that form a heterohexameric ring with DNA helicase activity, which is loaded on chromatin to form the pre-replication complex. Upon entry in S phase, the helicase is activated and opens the DNA duplex to recruit DNA polymerases at the replication fork. The MCM complex thus plays a crucial role during DNA replication, but recent work suggests that MCM proteins could also be involved in DNA repair. Here, we employed a combination of stable isotope labeling with amino acids in cell culture (SILAC)-based quantitative proteomics with immunoprecipitation of green fluorescent protein-tagged fusion proteins to identify proteins interacting with the MCM complex, and quantify changes in interactions in response to DNA damage. Interestingly, the MCM complex showed very dynamic changes in interaction with proteins such as Importin7, the histone chaperone ASF1, and the Chromodomain helicase DNA binding protein 3 (CHD3) following DNA damage. These changes in interactions were accompanied by an increase in phosphorylation and ubiquitination on specific sites on the MCM proteins and an increase in the co-localization of the MCM complex with &#x3b3;-H2AX, confirming the recruitment of these proteins to sites of DNA damage. In summary, our data indicate that the MCM proteins is involved in chromatin remodeling in response to DNA damage.

### Keywords
Etoposide, Dna repair, Mcm2

### Affiliations
Anatomy and Cell Biology, University of Sherbrooke, Canada
Universit√© de Sherbrooke

### Submitter
Francois-Michel Boisvert

### Lab Head
Dr Francois-Michel Boisvert
Anatomy and Cell Biology, University of Sherbrooke, Canada


