### Accession
PXD016946

### Title
Ant-oxidative effect of Lingzhi Protein Hydrolysate on Lipopolysaccharide-Stimulated A549 Cells

### Description
The oxidation stress suppression of enzymatic digestion reactions from Linzhi on LPS-stimulated A549 was not reported yet. In addition, the molecular mechanisms which directly related to physiological actions in our body is largely unknown. So, explorer the molecular mechanism of the protein hydrolysate in the reduction of oxidative stress on human cell line may lead more widespread realized in Lingzhi as valuable raw material for functional food additive

### Sample Protocol
Cell lysate was prepared by sonication at 20 Hz for 5 seconds. Protein was precipitated by cold-acetone precipitation (1:5 (v/v)) and solubilized by 0.1% RapidGest SF in 20 mM ammonium bicarbonate.The amount of 3 ������g protein was reduced (4 mM dithiothreitol; DTT, 72 ������C, 30 min) and alkylated (15 mM Iodoacetamide; IAA, RT in the dark, 30 min) prior to the digestion using trypsin at a 1:50 protein/trypsin concentration ratio (37 ������C, 4 hours). The mixture was evaporated and reconstituted in 0.1% formaic acid/LC-water. All samples (were analyzed in bioreplication(n=2) /quadruplicate (n=4) using a Synapt HDMS���������. The LC-runs was loaded onto a NanoAcquity system. Symmetry C18 5 ������m, analytical reversed phase column was used. Linear elution will be performed with 300 nl/min, using 2 eluents. The first eluent is LC-MS Water with 0.1% Formic acid and the second eluent was 100% Acetonitrile with 0.1% Formic acid. The peptide was eluted using a linear gradient for 50 minutes. For all LC-runs, LC-MS/MS operated in the V-mode of analysis with a typical resolving power of at least 10000 full-width half-maximum (FWHM). Data acquisition in LC-MS runs were operated in the Data-independent acquisition mode (DIA) using MSE-mode to obtain all spectral peptide ion including low abundant proteins (Blackburn, Cheng, Williamson, & Goshe, 2010). Mass spectral information was collected +2, +3 and +4 charge state, m/z range from 300 to 1800, scan time 1 second, collision energy was ramped between 4 to 15V

### Data Protocol
Protein was quantified by Progenesis������ QI for proteomics. Raw MS spectra were imported into the software using lock mass calibration (m/z= 784.8625), and identified against Uniprot/Swissprot human databases.  Parameters were set to: peptide tolerance, 60 ppm; fragment tolerance, 30 ppm; minimum fragment ion matches per peptide, 3; minimum fragment ion matches per protein, 3; minimum peptide ion matches per protein, 1; primary digest reagent, trypsin; cysteine carbamidomethylation as fixed modification and methionine oxidation, asparagine and glutamine deamination as variable modifications. The normalization of relative protein abundances was performed by NormalyzerDE (ver 1.1.13) ((Willforss, Chawade, & Levander, 2019)), in which Quantile-normalization be applied to relative expression data analysis. Differentially expressed proteins among treatment conditions were identified using one-way analysis of variance (ANOVA) at the confidence level of 95%.

### Publication Abstract
None

### Keywords
Lingzhi, Protein hydrolysate, Antioxidative peptide

### Affiliations
Mahidol University
National Omics Center (NOC), NSTDA, THAILAND

### Submitter
Sucheewin Krobthong

### Lab Head
Dr Sucheewin Krobthong
National Omics Center (NOC), NSTDA, THAILAND


