### Accession
PXD020397

### Title
Periodontal pathogens do not exacerbate macroscopic arthritis but alter the synovial proteome

### Description
Rheumatoid arthritis (RA) and periodontitis (PD) are chronic inflammatory diseases that appear to occur in tandem. Autoantibodies against citrullinated peptide antigens linked pathogeneses with PD preceding RA. However, the mutual impact PD exerts on RA and vice versa has not yet been defined. To address this issue, we set up an animal model and analyzed how the prime inducers of citrullination - Porphyromonas gingivalis and Aggregatibacter actinomycetemcomitans ÃƒÂ¢Ã¢â€šÂ¬Ã¢â‚¬Å“ differ in their pathogenic potential. Our experimental setup included collagen induced arthritis (CIA) in the mouse, oral inoculation with P. gingivalis or A. actinomycetemcomitans to induce alveolar bone loss and the combination of both diseases in inverted orders of events. Label-free quantitative proteome analysis revealed that P. gingivalis and A. actinomycetemcomitans led to differential expression patterns in the synovial membranes that were reminiscent of cellular and humoral immune responses, respectively. The P. gingivalis and A. actinomycetemcomitans specific signatures in the synovial proteomes suggest a role for oral pathogens in shaping disease subtypes and setting the stage for subsequent therapy response.

### Sample Protocol
Male F1 (DBA/1 x B10.Q) mice were inoculated with P. gingivalis or A. actinomycetemcomitans, respectively, to induce periodontal disease and immunized with bovine type II collagen to induce collagen induced arthritis. Proteins were extracted from synovial membranes of mice in 20 Ãƒâ€šÃ‚Âµl of lysis buffer containing 2 % sodium dodecyl sulfate, 50 mM dithiothreitol, and 50 mM Tris/HCl (pH 7.6). In-solution digestion of proteins with trypsin was done by filter-aided sample preparation (FASP). Peptide content was quantified using the Qubit Protein Assay Kit ( Thermo Fisher Scientific, Waltham, MA, USA). Separation and mass spectrometric analysis of peptides were performed on a nanoAcquity UPLC system (Waters, Manchester, UK) coupled to a Waters Synapt G2-S mass spectrometer.  160 ng of sample peptides supplemented with 40 fmol of Hi3 ClpB_ECOLI standard for protein absolute quantification (Waters, Manchester, UK) were trapped and desalted using a pre-column (ACQUITY UPLC Symmetry C18, 5 Ãƒâ€šÃ‚Âµm, 180 Ãƒâ€šÃ‚Âµm x 20 mm, Waters, Manchester, UK) at a flow rate of 10 Ãƒâ€šÃ‚Âµl/min for 4 min with 99.9 % mobile phase A (0.1 % formic acid in water) Subsequently, peptides were separated on an analytical column (ACQUITY UPLC HSS T3, 1.8 Ãƒâ€šÃ‚Âµm, 75 Ãƒâ€šÃ‚Âµm x 250 mm, Waters, Manchester, UK) using a gradient from 3 % to 32 % mobile phase B (0.1 % formic acid in acetonitrile) over 120 min at a flow rate of 300 nL/min and 55 Ãƒâ€šÃ‚Â°C. Data acquisition for peptide ions after ion-mobility separation was conducted with data-independent alternations of scans at low and elevated collision energy (CE) to obtain precursor and fragment ions for 0.6 s respectively (referred to as HDMSE).

### Data Protocol
Progenesis QI for Proteomics version 4.1 (Nonlinear Dynamics, Newcastle upon Tyne, UK) was used for raw data processing, protein identification and quantification. Alignment compensation of the HDMSE data was conducted to account for variations of LC separation in-between runs. Peptide and protein identifications were obtained by searching against a Mus musculus database containing 17,006 reviewed protein sequences (UniProt release 2019_02) appended with the sequences of ClpB_ECOLI (P63284) and porcine trypsin. Two missing cleavage sites were allowed, oxidation of methionine residues was considered as variable modification, and carbamidomethylation of cysteines as fixed modification. The false discovery rate based on the simultaneous search of a decoy database was set to 1%. Peptides were required to be identified by at least three fragment ions and proteins by at least six fragment ions from minimally two peptides. Subsequently peptide ion data were filtered to retain only peptide ions that met the following criteria: (i) identified at least two times within the dataset, (ii) ion score greater 5.5, (iii) mass error below 13.0 ppm, (iiii) at least 6 amino acid residues in length. Finally, only proteins with at least two unique peptides were included into the quantitative analysis. For label-free quantification, the Hi3 method implemented into the Progenesis QI for Proteomics workflow was applied using the Hi3 ClpB_ECOLI standard (Waters, Manchester, UK) as a reference. Downstream multivariate and single variate analyses were performed on protein abundances that were significantly different between all samples.

### Publication Abstract
None

### Keywords
Mus musculus, Synovial membrane, Periodontitis, Collagen-induced arthritis, Hdmse, Rheumatoid arthritis

### Affiliations
Core Facility for Cell Sorting & Cell Analysis, Laboratory for Clinical Immunology, University Medical Center Rostock, 18057 Rostock, Schillingallee 70, Germany
Core Facility Proteome Analysis, University Medicine Rostock, Schillingallee 69, 18057 Rostock

### Submitter
Stefan Mikkat

### Lab Head
Dr Brigitte Mueller-Hilke
Core Facility for Cell Sorting & Cell Analysis, Laboratory for Clinical Immunology, University Medical Center Rostock, 18057 Rostock, Schillingallee 70, Germany


