### Accession
PXD001103

### Title
proteomic analysis of B16-MDSCs

### Description
In order to analyse the B16-MDSCs proteome in a large-scale format, we used a multi-dimensional fractionation approach which combines peptide chromatographic fractionation strategies coupled to mass spectrometry.

### Sample Protocol
Sample preparation for proteomic analysis. B16-MDSCs and DC cellular pellets were resuspended in lysis buffer containing 7 M urea, 2 M thiourea, 4% (v/v) CHAPS, 50 mM DTT. Homogenates were spinned down at 14,000 x rpm for 1 h at 15ºC. Protein concentration was measured in the supernatants with the Bradford assay kit (Bio-rad). Proteomic analysis using iTRAQ approach. A shotgun comparative proteomic analysis of total cell extracts using iTRAQ (isobaric Tags for Relative and Absolute Quantitation) was performed. A Global experiment was carried out with three biological replicates in each experimental condition. Peptide labeling. Protein extracts (160 μg) were precipitated with methanol/choloroform, and pellets dissolved in 7M urea, 2 M thiourea, 4% (v/v) CHAPS. Protein quantitation was performed with the Bradford assay kit (Bio-Rad). iTRAQ labeling of each sample was performed according to the manufacturer's protocol (ABSciex). Briefly, a total of 80 μg of protein from each B16 and DC cell sample was reduced with 50 mM tris (2-carboxyethyl) phosphine (TCEP) at 60 °C for 1 h, and cysteine residues were alkylated with 200 mM methylmethanethiosulfonate (MMTS) at room temperature for 15 min. Protein enzymatic cleavage was carried out with trypsin (Promega; 1:20, w/w) at 37 °C for 16 h. Each tryptic digest was labelled according to the manufacturer's instructions with one isobaric amine-reactive tags as follows: Tag113, DC-1; Tag114, DC-2; Tag115, DC-3; Tag116, B16-1; Tag117, B16-2; Tag118, B16-3. After 1h incubation, each set of labelled samples were independently pooled and evaporated until < 40 μl in a vacuum centrifuge.  Peptide fractionation. To increase proteome coverage, the peptide pool was injected to an Ettan LC system with a X-Terra RP18 precolumn (2.1 x 20mm) and a high pH stable X-Terra RP18 column (C18; 2.1 mm x 150mm; 3.5μm) (Waters) at a flow rate of 40 μl/min. Peptides were eluted with a mobile phase B of 5–65% linear gradient over 35 min (A, 5 mM ammonium bicarbonate in water at pH 9.8; B, 5 mM ammonium bicarbonate in acetonitrile at pH 9.8). 8 fractions were collected, evaporated under vacuum and reconstituted into 20 μl of 2% acetonitrile, 0.1% formic acid, 98% MilliQ-H20 prior to mass spectrometric analysis. Triple-TOF 5600 Mass Spectrometry Analysis. Peptides mixtures were separated by reverse phase chromatography using an Eksigent nanoLC ultra 2D pump fitted with a 75 μm ID column (Eksigent 0.075 x 150). Samples were first loaded for desalting and concentration into a 0.5 cm length 300 μm ID precolumn packed with the same chemistry as the separating column. Mobile phases were 100% water 0.1% formic acid (FA) (buffer A) and 100% Acetonitrile 0.1% FA (buffer B). Column gradient was developed in a 70 min two step gradient from 2% B to 30% B in 60 min and 30%B to 40% B in 10 min. Column was equilibrated in 95% B for 5 min and 2% B for 15 min. During all process, precolumn was in line with column and flow maintained all along the gradient at 300 nl/min. Eluting peptides from the column were analyzed using an AB Sciex 5600 TripleTOFTM system. Information data acquisition was acquired upon a survey scan performed in a mass range from 350 m/z up to 1250 m/z in a scan time of 250 ms. Top 25 peaks were selected for fragmentation. Minimum accumulation time for MS/MS was set to 75 ms giving a total cycle time of 2.1 s. Product ions were scanned in a mass range from 100 m/z up to 1700 m/z and excluded for further fragmentation during 15 s. After MS/MS analysis, data files were processed using ProteinPilotTM 4.5 software from AB Sciex which uses the algorithm ParagonTM (v.4.0.0.0) for database search and ProgroupTM for data grouping and searched against Uniprot mouse database. False discovery rate was performed using a non-lineal fitting method and displayed results were those reporting a 1% Global False Discovery Rate (FDR) or better.

### Data Protocol
Relative quantification and protein identification were performed with the ProteinPilotTM software (version 4.5; ABSciex) using the ParagonTM algorithm as the search engine. Each MS/MS spectrum was searched against a database of murine protein sequences (Uniprot complete mouse proteome). The search parameters allowed for cysteine modification by MMTS and biological modifications programd in the algorithm (i.e. phosphorylations, amidations, semitryptic fragments, etc.). Reporter ion intensities were bias corrected for the overlapping isotope contributions from the iTRAQ tags according to the certiﬁcate of analysis provided by the reagent manufacturer (ABsciex). The peptide and protein selection criteria for relative quantitation were performed as follows. Only peptides unique for a given protein were considered for relative quantitation, excluding those common to other isoforms or proteins of the same family. Proteins were identified on the basis of having at least one peptide with an ion score above 99% confidence. Among the identified peptides, some of them were excluded from the quantitative analysis for one of the following reasons: (i) The peaks corresponding to the iTRAQ labels were not detected; (ii) the peptides were identified with low identification confidence (<1.0%); (iii) the sum of the signal-to-noise ratio for all of the peak pairs was <6 for the peptide ratios. The protein sequence coverage (95% conf.) was estimated for specific proteins by the percentage of matching amino acids from the identified peptides having confidence greater than or equal to 95% divided by the total number of amino acids in the sequence. Several quantitative estimates provided for each protein by ProteinPilot were utilized: the fold change ratios of differential expression between labelled protein extracts; the p-value, representing the probability that the observed ratio is different than 1 by chance. A decoy database search strategy was also used to estimate the false discovery rate (FDR), defined as the percentage of decoy proteins identified against the total protein identification. The FDR was calculated by searching the spectra against the decoy database generated from the target database. The results were then exported into Excel for manual data interpretation. Although relative quantification and statistical analysis were provided by the ProteinPilot software, an additional 1.3-fold change cutoff for all iTRAQ ratios (ratio <0.77 or >1.3) was selected to classify proteins as up- or down-regulated. Proteins with iTRAQ ratios below the low range (0.77) were considered to be underexpressed, whereas those above the high range (1.3) were considered to be overexpressed.

### Publication Abstract
Myeloid-derived suppressor cells (MDSCs) exhibit potent immunosuppressive activities in cancer. MDSCs infiltrate tumors and strongly inhibit cancer-specific cytotoxic T cells. Their mechanism of differentiation and identification of MDSC-specific therapeutic targets are major areas of interest. We have devised a highly efficient and rapid method to produce very large numbers of melanoma-infiltrating MDSCs ex vivo without inducing tumors in mice. These MDSCs were used to study their differentiation, immunosuppressive activities and were compared to non-neoplastic counterparts and conventional dendritic cells using unbiased systems biology approaches. Differentially activated/deactivated pathways caused by cell type differences and by the melanoma tumor environment were identified. MDSCs increased the expression of trafficking receptors to sites of inflammation, endocytosis, changed lipid metabolism, and up-regulated detoxification pathways such as the expression of P450 reductase. These studies uncovered more than 60 potential novel therapeutic targets. As a proof of principle, we demonstrate that P450 reductase is the target of pro-drugs such as Paclitaxel, which depletes MDSCs following chemotherapy in animal models of melanoma and in human patients. Conversely, P450 reductase protects MDSCs against the cytotoxic actions of other chemotherapy drugs such as Irinotecan, which is ineffective for the treatment of melanoma.

### Keywords
Mdscs

### Affiliations
Proteomics unit
Proteomics Unit. navarrabiomed-miguel Servet Foundation (Pamplona-Spain)

### Submitter
Enrique SantamarÃ­a

### Lab Head
Dr Enrique Santamaria
Proteomics Unit. navarrabiomed-miguel Servet Foundation (Pamplona-Spain)


