### Accession
PXD025970

### Title
Protein and Signaling Pathway Responses to rhIL-6 Intervention before Lobaplatin Treatment in Osteosarcoma Cells using Quantitative Phosphoproteome Analysis

### Description
IL-6 plays a pivotal role in the process of drug resistance to kinds of chemotherapeutics including lobaplatin. However, the underlying changes of phosphoproteins and related signaling pathways in the posttranslational modification level is still uncover. In our study, a quantitative phosphoproteome analysis of the responses to rhIL-6 & lobaplatin treatment was performed in osteosarcoma cells. Cells were divided into three goups: the control group, the lobaplatin group, and the rhIL-6 & lobaplatin group. Three biological replicates were performed in each group. A total of 3373 proteins and 12183 peptides was identified and quantified, of which 3232 phosphorylated proteins and 11358 phosphorylated peptides were obtained. Between the rhIL-6 & lobaplatin group and the lobaplatin group, twenty-three statistically differentially expressed phosphorylated peptides were identified. The characteristics of differentially expressed phophoproteins were analyzed by GO and KEGG enrichment, kinases of the phosphoproteins were predicted and protein-protein interaction was illustrated/demonstrated using STRING. Conservative motifs that surround the phosphorylated serine, threonine and tyrosine residues were analyzed using the Motif-x algorithm. Western blotting were performed to verify the differentially expressed phosphorylated FLNC and the signaling pathway it involved in. Immunohistochemistry staining showed that p-FLNC and its kinase AKT1 as well as ERK1/2 are positively expressed in the platinum-resistant specimens, while these proteins are negative or weak in those of platinum-sensitive specimens. In conclusion, p-FLNC and the MAPK signaling pathway plays a crucial role in the process of IL-6 induced chemoresistance in osteosarcoma. It is the first phosphoproteomic study which reveals the signature of IL-6 intervention before lobaplatin treatment in human osteosarcoma cells.

### Sample Protocol
Sample Preparation SDT buffer was added to the sample. The lysate was sonicated (this step can be skipped for protein solution) and then boiled for 15 min. After centrifuged at 14000g for 40 min, the supernatant was quantified with the BCA Protein Assay Kit (P0012, Beyotime). The sample was stored at -80 °C. SDS-PAGE Separation 20 µg of proteins for each sample were mixed with 5X loading buffer respectively and boiled for 5 min. The proteins were separated on 12.5% SDS-PAGE gel. Protein bands were visualized by Coomassie Blue R-250 staining. Filter-aided sample preparation (FASP Digestion) 200 μg of proteins for each sample were incorporated into 30 μl SDT buffer (4% SDS, 100 mM DTT, 150 mM Tris-HCl pH 8.0). The detergent, DTT and other low-molecular-weight components were removed using UA buffer (8 M Urea, 150 mM Tris-HCl pH 8.5) by repeated ultrafiltration (Sartorius, 30 kD). Then 100 μl iodoacetamide (100 mM IAA in UA buffer) was added to block reduced cysteine residues and the samples were incubated for 30 min in darkness. The filters were washed with 100 μl UA buffer three times and then 100 μl 0.1M TEAB buffer twice. Finally, the protein suspensions were digested with 4 μg trypsin (Promega) in 40 μl 0.1M TEAB buffer overnight at 37 °C, and the resulting peptides were collected as a filtrate. The peptide content was estimated by UV light spectral density at 280 nm using an extinctions coefficient of 1.1 of 0.1% (g/l) solution that was calculated on the basis of the frequency of tryptophan and tyrosine in vertebrate proteins. TMT Labeling 100 μg peptide mixture of each sample was labeled using TMT reagent according to the manufacturer’s instructions (Thermo Fisher Scientific). Enrichment of phosphorylated peptides The labeled peptides were mixed ,desalted and dissolved in 200ul loading buffer (30%ACN, 20mg/ml, 2%TFA). Then the peptides solution was transferred to tubes with TiO2, and rotated for  2hr. The mixture was then transferred to stagetips , washed twice by 200ul wash buffer 1 (70% ACN, 0.1% TFA) and once in order by Elute buffer 1 (20% ACN, 0.6% ammonia) , Elute buffer 2 (50% ACN, 1% ammonia) , Elute buffer 3 (40% ACN, 4.5% ammonia). At last,  the enriched phosphopeptides were freeze dried and dissolved in 0.1%FA. Peptide Fractionation with reversed phase (RP) Chromatography TMT labeled peptides were fractionated by RP chromatography using the Agilent 1260 infinity II HPLC. The peptide mixture was diluted with buffer A (10mM HCOONH4, 5% ACN, pH 10.0) and loaded onto a XBridge Peptide BEH C18 Column, 130Å, 5 µm, 4.6 mm X 100 mm column. The peptides were eluted at a flow rate of 1 ml/min with a gradient of 0%–7% buffer B (10mM HCOONH4, 85% ACN, pH 10.0) for 5 min, 7–40% buffer B during 5-40 min, 40%–100% buffer B during 45-50 min, 100% buffer B during 50-65 min. The elution was monitored by absorbance at 214 nm, and fractions were collected every 1 min during 5-50min. The collected fractions were dried down via vacuum centrifugation at 45 °C.  Each fraction was injected for nanoLC-MS/MS analysis. The peptide mixture was loaded onto the C18-reversed phase analytical column (Thermo Fisher Scientific，Acclaim PepMap RSLC 50um X 15cm, nano viper,P/N164943) in buffer A (0.1% Formic acid) and separated with a linear gradient of buffer B (80% acetonitrile and 0.1% Formic acid) at a flow rate of 300 nl/min. The linear gradient was determined by the project proposal: 1.5 hours gradient：6% buffer B for 5 min, 6-28% buffer B for 63 min, 28-38% buffer B for 10 min,38-100% buffer B for 7 min , hold in 100% buffer B for 5 min. LC-MS/MS Analysis LC-MS/MS analysis was performed on a Q Exactive Plus mass spectrometer (Thermo Fisher Scientific) that was coupled to Easy nLC (Thermo Fisher Scientific) for 90/120 min (determined by project proposal). The mass spectrometer was operated in positive ion mode. MS data was acquired using a data-dependent top10 method dynamically choosing the most abundant precursor ions from the survey scan (350–1800 m/z) for HCD fragmentation. Automatic gain control (AGC) target was set to 3e6, and maximum inject time to 45 ms. Survey scans were acquired at a resolution of 70,000 at m/z 200 and resolution for HCD spectra was set to 17,500 at m/z 200, and isolation width was 2 m/z. Normalized collision energy was 30 eV.

### Data Protocol
Data Analysis 1. MS/MS spectra were searched using MASCOT engine (Matrix Science, London, UK; version 2.6) embedded into Proteome Discoverer 2.1. The following parameters were set. Proteins which Fold change>1.2 and p value (Student’s t test) were considered to be a differentially expressed proteins. Item Value  Protein Database  See the project report  Enzyme  Trypsin  Max Missed Cleavages  2  Instrument  ESI-TRAP  Precursor Mass Tolerance  ±20 ppm  Fragment Mass Tolerance  0.1Da  Use Average Precursor Mass  False  Modification Groups From Quan  Method  TMT 6plex  Dynamic modifications  Oxidation(M),Acetyl(Protein N-term),  Deamidated(NQ)  Static  modifications  Carbamidomethyl (C),  Database pattern  decoy  Peptide FDR  ≤0.01 2. Gene Ontology (GO) Annotation At first, all protein sequences were aligned to Homo sapiens(see project report) database downloaded from NCBI (ncbi-blast-2.2.28+-win32.exe), only the sequences in top 10 and E-value<=1e-3 were kept. Secondly, select the GO term (database version: go_201504.obo) of the sequence with top Bit-Score by Blast2GO. Then, completed the annotation from GO terms to proteins by Blast2GO Command Line. After the elementary annotation, InterProScan were used to search EBI database by motif and then add the functional information of motif to proteins to improve annotation. Then further improvement of annotation and connection between GO terms were carried out by ANNEX. Fisher’s Exact Test were used to enrich GO terms by comparing the number of differentialy expressed proteins and total proteins correlated to GO terms 3. KEGG Pathway Annotation Pathway analysis was performed using KEGG database. Fisher’s Exact Test were used to indentify the significantly enriched pathways by comparing the number of differentialy expressed proteins and total proteins correlated to pathways.

### Publication Abstract
None

### Keywords
Human, Lobaplatin, Osteosarcoma, Phosphoproteome analysis, Il-6

### Affiliations
Orthopedic Oncology Institute, Department of Orthopedic Surgery, Tangdu Hospital, Fourth Military Medical University
Orthoped OncoloOrthopedic Oncology Institute, Department of Orthopedic Surgery, Tangdu Hospital, Fourth Military Medical UniversityInstitute, Department of Orthopedic Surgery, Tangdu HospitalOrthopec

### Submitter
Qiong Ma

### Lab Head
Dr Qiong Ma
Orthoped OncoloOrthopedic Oncology Institute, Department of Orthopedic Surgery, Tangdu Hospital, Fourth Military Medical UniversityInstitute, Department of Orthopedic Surgery, Tangdu HospitalOrthopec


