### Accession
PXD018387

### Title
Allosteric HSP70 inhibitors perturb mitochondrial proteostasis and overcome proteasome inhibitor resistance in multiple myeloma

### Description
Proteasome inhibitor (PI) resistance remains a central challenge in multiple myeloma. To identify pathways mediating this resistance, we mapped genetic co-dependencies associated with the proteasome. These studies identified cytosolic heat shock protein 70 (HSP70) chaperones as a potential target, mirroring recent studies that have shown mechanism of overcoming PI-induced stress. Here, we first underscore this relationship by mapping genetic co-dependencies in cancer proteostasis. These results lead us to explore HSP70 inhibitors as potential therapeutics. We show these compounds exhibit increased efficacy against both acquired and intrinsic PI-resistant myeloma models, unlike HSP90 inhibition. Surprisingly, shotgun and pulsed-SILAC proteomics reveal that JGâ€™s overcome PI resistance not via the expected mechanism of inhibiting cytosolic HSP70s, but instead through mitochondrial-localized HSP70, HSPA9, destabilizing the 55S mitoribosome. Analysis of myeloma patient data further supports strong effects of global proteostasis capacity, and particularly HSPA9 expression, on response to PI. Our results characterize dynamics of myeloma proteostasis networks under therapeutic pressure while further motivating investigation of HSPA9 as a specific target in PI-resistant disease.   This dataset corresponds to figure 5, experiment outlined in figure 5a.

### Sample Protocol
For pulsed-SILAC experiments, MM1.S cells were grown in Light SILAC media for at least six passages to allow complete labeling and adaptation to dialyzed FBS. At time = 0, cells were exchanged to Heavy SILAC media with 350nM JG342 or DMSO. Cells were collected at 16, 21, and 26 time points, washed with PBS, snap frozen, and stored at -80C. Cell pellets were lysed in 6M GdnHCL, 0.1M Tris pH 8.5, with 5mM TCEP and 10mM 2-chloro acetamide. Lysates were sonicated for 45 seconds at 1 Hz cycles on ice and cleared by centrifugation at 16000g for 10 minutes at 4C. Protein concentration was measured with 660 assay (Pierce 22660) and 100 ug protein was digested with 2 ug Trypsin (Pierce 90057) for 18-22 hours with digestion halted by addition of TFA to 1% vol/vol. Acidified samples were centrifuged at 17,200g for 5 minutes at RT to remove precipitate. Cleared samples were desalted on SOLA columns (Thermo 60109001) according to manufacturer instructions and eluted in 50% Acetonitrile with 0.1% FA prior and vacuum dried prior to storage at -80C. Miniaturized TMT labeling was performed based on modified protocol from Zecha et al (2019). Briefly, peptide concentrations were measured using the Pierce Peptide Colormetric Assay (Pierce 23275). 20 ug peptides were resuspended in 17.5 uL 50 mM HEPES pH 8.5 and labeled with 50 ug TMT reagent dissolved in 2.5 uL Acetonitrile for 1 hour at 25C and 500 rpm. Reactions were quenched by adding hydroxylamine to final concentration of 0.4% and incubation for 15 min at 500 rpm. TMT labeled samples were combined, acidified by addition of 0.1% TFA, vacuum dried, and stored at -80C. For 7-plex pulsed SILAC-TMT LC-MS/MS, 500 ng of peptides were injected into Easy-Spray reversed phase column (Thermo ES800) on a nanoACQUITY UPLC (Waters) coupled to a Fusion Lumos Mass Spectrometer (Thermo) with the following non-linear gradient in which A is 0.1% Formic Acid and B is Acetonitrile plus 0.1% Formic Acid: 8% B to 30% B for 110 minutes, 30% B to 50% B for 20 minutes, 50% B to 70% B for 5 minutes, 70% B to 80% B for 1 minute, and 8% B for 7 minutes to re-equilibrate. For MS1 data acquisition, scan range was set to 375-1500 m/z, AGC target was set to 4e5, and maximum injection time (IT) was set to 50ms. For MS2 data acquisition, isolation window was set to 0.7 m/z, with HCD energy set to 38 percent, orbitrap resolution was set to 50000, and AGC target was set to 1.0e5.

### Data Protocol
Mass spectrometry data was processed in Maxquant (Tvanova et al 2016) version 1.6.2.1 with the following settings: PSM/Protein FDR were set to 0.01, Carbidomethylation was set as fixed modification and methionine oxidation and N-terminal acetylation were set as variable modifications,  minimum peptide length = 7, matching time window set to 0.7 min, alignment time window set to 20 min, and match between runs was used, along with other default settings.  Data was searched against the Uniprot Swiss-Prot human proteome (ID:9606, downloaded from Uniprot in 2018). For TMT-pSILAC multiplexing analysis, separate parameter groups were used for heavy and light analysis run in the same MS2 experiment using the isobaric labels function in Maxquant. For the heavy parameter group, heavy arginine was set as fixed modification and heavy lysine modifications were added into TMT-tag masses without altering the diagnostic peaks, which denote the TMT cleaved label masses. Data analysis was performed in Perseus (Tvanova et al 2016) and R. Subcellular compartment gene lists were downloaded from Uniprot and restricted to reviewed entries.

### Publication Abstract
None

### Keywords
Hsp70, Multiple myeloma, Tmt, Pulsed-silac

### Affiliations
Stanford University
Department of Laboratory Medicine, Wiita Lab, University of California, San Francisco, USA

### Submitter
Ian Ferguson

### Lab Head
Dr Arun Wiita
Department of Laboratory Medicine, Wiita Lab, University of California, San Francisco, USA


