### Accession
PXD020545

### Title
i. Bioplatforms Australia: Antibiotic Resistant Sepsis Pathogens Framework Initiative – Escherichia coli

### Description
The Antibiotic Resistant Sepsis Pathogens Framework Initiative aims to develop a framework dataset of 5 sepsis pathogens (Escherichia coli, Klebsiella pneumoniae complex, Staphylococcus aureus, Streptococcus pneumoniae and Streptococcus pyogenes, 5 strains each) using an integrated application of genomic, transcriptomic, metabolomic and proteomic technologies.  This submission contains the results from five Escherichia coli strains (B36, MS14384, MS14385, MS14386, MS14387) grown in either RPMI or pooled human sera.  Six replicates of each condition were subjected to shotgun proteomics and label-free MS1-based quantitation.

### Sample Protocol
Complete information about the bacterial cultures is available from the Bioplatforms Australia Antibiotic Resistant Sepsis Pathogens Initiative website. https://data.bioplatforms.com/organization/pages/bpa-sepsis/methods  10ml of bacterial culture (each replicate) were repeatedly washed with PBS, and snap frozen in liquid nitrogen. Freeze-dried bacteria were resuspended in 600µL of bead-beating buffer (50mM Tris-HCl, 145mM NaCl) containing 400µL of 0.1mm acid-washed glass beads. These were then subjected to bead beating (4 cycles of 1 min, 5m/s) with 1 minute rest on ice between cycles. Cellular debris was removed by centrifugation at 20,000 x g for 30 m at 4oC. 250 μL of protein sample was mixed with ice cold water : methanol : chloroform in a ratio of 3 : 4 : 1 to precipitate proteins. Precipitate was resuspended in 6 M urea / 2 M thiourea, and protein quantitation was performed using a QubitTM kit (Invitrogen, Carlsbad CA). The solutions were reduced for 1 h with 10 mM dithiothreitol (DTT) followed by alkylation using 20 mM iodoacetamide for 1 h in the dark. Samples were diluted to 1mL in 100mM TEAB, pH adjusted to 7.5 and digested with 1/25 (w/w) of porcine sequencing grade trypsin (Promega, Madison WI) for 24 h at 37oC.  Formic acid was then added to a final concentration of 2% (v/v) to precipitate lipids, and centrifuged at 14, 000 RPM at 25oC for 15 m to pellet lipids.  The supernatant was removed and subjected to purification using Oasis® HLB columns (Waters Corporation, Milford MA). Cartridges were prepared by washing in succession: 1 mL 100% methanol, 1 mL 100% acetonitrile (MeCN), 1 mL 70% (v/v) MeCN / 0.1% (v/v) trifluoroacetic acid (TFA). Samples were washed through cartridges 5 times and finally washed with 5 mL 0.1% TFA. Peptides were eluted with 50% MeCN / 0.1% TFA, and lyophilised by vacuum centrifugation.  Lyophilised peptides were resuspended at 2 μg/μL in 2% acetonitrile, 0.1% TFA with spiked iRT peptides using a PCV-3000 (5 sec spin @ 1500rpm, 10 sec hard vortex, 30 cycles).  Samples were centrifuged at 16100 rcf for 5 min (Centrifuge 5415D, Eppendorf) and supernatant vialled for MS analysis.  One μL was analysed on a Thermo LC-MS system comprised of an Ultimate 3000 RSLC system coupled to an Orbitrap QExactive Plus mass spectrometer.  The samples were loaded onto a trap column (Acclaim PepMap 100, 100 μm x 2 cm, Thermo) at 15 μL/min followed by development on a Acclaim PepMap RSLC column (75 μm x 50cm, Thermo) at 250 μL/min using a 2 hr gradient from 6% to 30% acetonitrile in 0.1% formic acid. MS analysis consisted of a Top12 method (MS scans: 375-1575m/z, 70k res; MSMS scans: 17.5k res, 1.8m/z isolation window, 27 NCE, charge state 2-5, peptide match preferred, 15 sec dynamic exclusion).  Strain-based samples were acquired in a randomised order with interleaved wash cycles.

### Data Protocol
Mass spectrometer RAW files were searched using MaxQuant (v1.6.5.0) against strain specific databases based on the sequenced genomes, as well as the human proteome (Uniprot, UP000005640, downloaded 8 March 2019) and common contaminant proteins (from MaxQuant).   The genomic sequences were submitted to ENA database as study PRJEB29930, and the protein databases are available from NCBI GenBank through the following accessions: Strain B36: NZ_LR130545.1; NZ_LR130546.1; NZ_LR130547.1 Strain MS14384: NZ_LR130562.1; NZ_LR130563.1 Strain MS14385: NZ_LR130555.1; NZ_LR130556.1; NZ_LR130557.1; NZ_LR130558.1; NZ_LR1305559.1; NZ_LR130560.1; NZ_LR130561.1 Strain MS14386: NZ_LR130552.1; NZ_LR130553.1; NZ_LR130554.1 Strain MS14387: NZ_LR130564.1; NZ_LR130565.1; NZ_LR130566.1  The combined fasta databases are included in the submission.    MaxQuant search parameters consisted of: Trypsin/P with 3 missed cleavages, fixed modification: carbamidomethylation (C), up to 5 variable modifications: Oxidation (M), Deamidation (NQ), Acetyl (Protein N-term), default instrument parameters: Orbitrap with main search tolerance of 4.5ppm. LFQ was performed with min ratio count of 2, stabilised large ratios and MSMS requirement for LFQ comparisons.

### Publication Abstract
None

### Keywords
Dda, Escherichia coli, Lfq, Sera, Proteomics, Antibiotic resistance, Sepsis, Orbitrap, Rpmi

### Affiliations
Bioplatforms Australia (project manager)
Monash Proteomics and Metabolomics Facility, Monash University

### Submitter
Robert Goode

### Lab Head
Dr Mabel Lum
Bioplatforms Australia (project manager)


