### Accession
PXD008033

### Title
Chromatin protein profiling during the mammalian cell cycle reveals the extent of mitotic preservation of the regulatory landscape

### Description
While regulation of transcription, replication and cell division relies on dynamic protein binding to DNA and chromatin, it is unclear which regulatory components remain bound to compacted mitotic chromosomes. To comprehensively quantify the chromatin-associated proteome in different phases of the cell cycle, we utilized the buoyant density of DNA-protein complexes after crosslinking. This revealed variable associations for thousands of proteins to DNA including their frequent and variable phosphorylation. While epigenetic modifiers that promote transcription are lost from mitotic chromatin, repressive modifiers generally remain associated. Interestingly, while proteins involved in transcriptional elongation are evicted, most identified transcription factors are retained on mitotic chromatin to varying degrees, including core promoter binding proteins. This predicts conservation of the regulatory landscape on mitotic chromosomes, which genome-wide measurements of chromatin accessibility confirm. This work establishes a novel approach to study chromatin, provides a comprehensive catalogue of chromatin changes during the cell cycle, and reveals the degree with which the genomic regulatory landscape is maintained through mitosis.

### Sample Protocol
Samples containing 25-50ug of peptides were labeled with TMT 10-plex reagents (Thermo Fisher Scientific) as previously described70. TMT labeled peptides were offline fractionated at high pH on a YMC Triart C18 0.5 x 250 mm column (YMC Europe GmbH) using the Agilent 1100 system (Agilent Technologies). 72 fractions were collected for each experiment and concatenated into 12 or 24 fractions as previously described (Wang et al., 2011 PMID: 21500348). For each LC-MS run, approximately 1 ug of peptides were loaded onto a PepMap 100 C18 2 cm trap (Thermo Fisher) using the Proxeon EASY NanoLC-1000 system (Thermo Fisher). On-line peptide separation was performed on the 15 cm EASY-Spray C18 column (ES801, Thermo Fisher) by applying a linear gradient of increasing ACN concentration at a flow rate of 150 nL/min. An Orbitrap Fusion Tribrid (Thermo Fisher) mass spectrometer was operated in a data-dependent mode and TMT reporter ions were quantified using a synchronous precursor selection (SPS)-based MS3 technology, as previously described (McAlister et al., 2014 PMID: 24927332). In brief, the top 20 most intense precursor ions from the Orbitrap survey scan were selected for collision-induced dissociation (CID) fragmentation. The ion-trap analyzer was used to generate the MS2 CID spectrum from which the notches for the MS3 scan were selected. The MS3 spectrum was recorded using the Orbitrap analyzer at a resolution of 60000.

### Data Protocol
Thermo MS raw data were searched against the human uniprot database (downloaded January 29 2015) and a database of common contaminants with either Thermo Proteome Discoverer PD 2.1 (Thermo Scientific) and the Sequest HT search engine or MaxQuant (version 1.5.3.8) and the Andromeda search engine.  The search parameters in PD were fully tryptic digestion, maximum of two missed cleavages, minimum peptide length of six amino acids, fixed carbamidomethyl modifications of cysteine as well as TMT6plex [+229 Da] of lysines and peptide N-termini; and oxidation of methionine as well as acetylation of the protein terminus as variable modifications. The maximum allowed mass tolerance for precursor ions measured in the Orbitrap was set to 10 ppm. The ion trap fragment ion mass tolerance was set to 0.6 Da. Search parameters in MaxQuant71 were essentially the same as in PD, except that the mass tolerance for precursor ions was set to 20 ppm in a first-pass search prior to mass recalibration, followed by 4.5 ppm in the main search of the re-calibrated data, and then the fragment ion mass tolerance was set to 0.5 Da. Protein and peptide identifications were filtered to an false-discovery rate of 0.01 based on the target-decoy search strategy72. For TMT MS3 reporter ion quantification the mass tolerance was set to 20 ppm in PD (0.1 Da in MaxQuant), only scans with average reporter ion s/n above 10 were used. Protein abundances were calculated based on the summed abundances of all unique and razor peptide reporter ion signals attributed to a protein.  For bioinformatic analyses the PD Proteins tables were exported to txt format, and the MaxQuant proteinGroups.txt table was used.  Proteome Discoverer results were first filtered for contaminants, proteins with at least 2 peptides quantified, and signal in all 3 replicates of at least one cell cycle stage. Proteins with more than one Entrez Gene ID or no ID were removed. In addition, master protein candidates were retained if peptides used in their quantification were not used in a master protein. In the case of multiple candidates for the same protein, proteins were selected by: 1) Non-isoform entry and in the case of ties, 2) Lowest coverage (longest protein).  Subsequently the reporter ion intensities for each channel were scaled down to the lowest signal reporter.  MaxQuant search results were first filtered for reverse, contaminant, and proteins with no gene name. Proteins were then filtered to have quantitative information for at least all 3 replicates of one cell cycle stage.  Intensities were then normalized using the VSN package in R as has been described for isobaric labeled peptides73. Modified peptides were first filtered on an Andromeda score >= 40 and delta score >= 8.  Subsequently, peptide reporter ion intensities were scaled using VSN as above. Clustering of proteins To identify patterns for differential chromatin binding across the measured cell cycle stages, we used linear modeling with the limma package74 to determine which proteins are significantly changing between all possible cell cycle contrasts.  Reporter ion intensities from these significantly changing proteins were clustered based on their distance to the mean for the 9 replicates measured for each peptide.  Clustering was carried out using affinity propagation clustering33 implemented in the R package apcluster34.

### Publication Abstract
Regulation of transcription, replication, and cell division relies on differential protein binding to DNA and chromatin, yet it is unclear which regulatory components remain bound to compacted mitotic chromosomes. By utilizing the buoyant density of DNA-protein complexes after cross-linking, we here develop a mass spectrometry-based approach to quantify the chromatin-associated proteome at separate stages of the cell cycle. While epigenetic modifiers that promote transcription are lost from mitotic chromatin, repressive modifiers generally remain associated. Furthermore, while proteins involved in transcriptional elongation are evicted, most identified transcription factors are retained on mitotic chromatin to varying degrees, including core promoter binding proteins. This predicts conservation of the regulatory landscape on mitotic chromosomes, which we confirm by genome-wide measurements of chromatin accessibility. In summary, this work establishes an approach to study chromatin, provides a comprehensive catalog of chromatin changes during the cell cycle, and reveals the degree to which the genomic regulatory landscape is maintained through mitosis.

### Keywords
Chromatin, Mitotic bookmarking, Cell cycle

### Affiliations
FMI
Friedrich Miescher Institute for Biomedical Research Epigenetics Department Lab 1.22

### Submitter
Paul Ginno

### Lab Head
Dr Dirk Sch√ºbeler
Friedrich Miescher Institute for Biomedical Research Epigenetics Department Lab 1.22


