### Accession
PXD000470

### Title
Proteome analysis of Brachypodium distachyon leaf under salt stress and recovery

### Description
Brachypodium distachyon is a new model plant for wheat, barley and several potential biofuel grasses. The Bd21 leaf was initially treated with salt concentrations ranging from 80 to 320 mM followed by a recovery process prior to proteome analysis. Leaf total protein was extracted according to the method of Wang et al. with minor modifications, followed by 2D DIGE separation as recommended by the manufacturer (GE Healthcare). The differentially expressed proteins spots were identified using ImageMaster 2D Platinum Software Version 5.0 (Amersham Biosciences). Protein identification was carried out by the ABI 4800 Proteomics Analyzer MALDI-TOF/TOF (Applied Biosystems, Foster City) operating in a result-dependent acquisition mode. Peptide mass maps were acquired in positive ion reflector mode (20 kV accelerating voltage) with 1000 laser shots per spectrum. Monoisotopic peak masses were automatically determined within the mass range 800-4000 Da with a signal to noise ratio minimum set to 10 and a local noise window width of m/z 250. Up to five of the most intense ions with minimum signal to noise ratio 50 were selected as precursors for MS/MS acquisition, excluding common trypsin autolysis peaks and matrix ion signals. In MS/MS-positive ion mode, spectra were averaged, collision energy was 2 kV, and default calibration was set. Monoisotopic peak masses were automatically determined with a signal to noise ratio minimum set to 5 and a local noise window width of m/z 250. The MS together with MS/MS spectra were searched against NCBI Brachypodium protein database (26035 entries in total, downloaded on July 16, 2011) using the software MASCOT version 2.1 (Matrix Science) with the following parameter settings: trypsin cleavage, one missed cleavage allowed, carbamidomethylation set as fixed modification, oxidation of methionines and phosphorylation of serine, threonine, and tyrosine allowed as variable modification, peptide mass tolerance set to 100 ppm and fragment tolerance set to Â± 0.3 Da. All searches were evaluated based on the significant scores obtained from MASCOT. The results were creditable with Protein Score C. I. % and Total Ion Score C. I. % both above 95% and significance threshold p < 0.05 for the MS/MS.

### Sample Protocol
Not available

### Data Protocol
Not available

### Publication Abstract
Salinity is a major abiotic stress affecting plant growth and development. Understanding the molecular mechanisms of salt response and defense in plants will help in efforts to improve the salt tolerance of crops. Brachypodium distachyon is a new model plant for wheat, barley, and several potential biofuel grasses. In the current study, proteome and phosphoproteome changes induced by salt stress were the focus. The Bd21 leaves were initially treated with salt in concentrations ranging from 80 to 320 mm and then underwent a recovery process prior to proteome analysis. A total of 80 differentially expressed protein spots corresponding to 60 unique proteins were identified. The sample treated with a median salt level of 240 mm and the control were selected for phosphopeptide purification using TiO2 microcolumns and LC-MS/MS for phosphoproteome analysis to identify the phosphorylation sites and phosphoproteins. A total of 1509 phosphoproteins and 2839 phosphorylation sites were identified. Among them, 468 phosphoproteins containing 496 phosphorylation sites demonstrated significant changes at the phosphorylation level. Nine phosphorylation motifs were extracted from the 496 phosphorylation sites. Of the 60 unique differentially expressed proteins, 14 were also identified as phosphoproteins. Many proteins and phosphoproteins, as well as potential signal pathways associated with salt response and defense, were found, including three 14-3-3s (GF14A, GF14B, and 14-3-3A) for signal transduction and several ABA signal-associated proteins such as ABF2, TRAB1, and SAPK8. Finally, a schematic salt response and defense mechanism in B. distachyon was proposed.

### Keywords
Salt, Brachypodium distachyon, Proteome, Leaf, Recovery

### Affiliations
College of Life Sciences

### Submitter
Dong-Wen Lv

### Lab Head


