### Accession
PXD005223

### Title
Transcriptomic and proteomic analysis of alternative splicing in Physcomitrella patens moss

### Description
Alternative splicing (AS) has a significant potential to impact on the eukaryotic cell transcriptome and proteome. However, the extent of this influence as well as mechanisms, providing the tissue-specific pattern of  AS are poorly studied. Here, we analyzed the AS of two life forms, protonema and gametophores, as well as the protoplasts, of a model organism, the Physcomitrella patens moss, using the data of transcriptome and proteome profiling. Altogether, 12,043 genes subjected to alternative splicing were identified. The alternative splicing patterns of 302 genes involved in stress response, growth, and development are changed considerably in the process of the gametophore development, whereas AS of 276 genes involved in transcription regulation, development, cell interactions, and cell response to biotic and abiotic stresses altered upon protoplastitation. Using mass spectrometry analysis, we studied the extent to which AS contributes to proteome diversity and identified 117 isoform-specific peptides for 36 genes with AS events. Our results indicate that AS event have a little impact on the proteome diversity and mostly result in the addition of extra amino acids rather than editing of existing proteins sequences. Among  differentially expressed and spliced genes we found serine/arginine-rich (SR) genes, which are known to regulate AS in cells. We identified new isoforms of these genes and data evidencing their translation were obtained at transcriptome and proteome level. We also found that treatment with abscisic and jasmonic acids led to the isoform-specific response in protonema. Besides, we revealed the potential lncRNAmRNA and lncRNApre-mRNA interactions that can influence both the expression and alternative splicing of genes. It can point at an additional level of gene expression and AS regulation by non-coding transcripts in the Physcomitrella patens moss.

### Sample Protocol
Protonema and Gametophore Growth and Protoplast Isolation Conditions Protonema and gametophores were grown as previously described (Fesenko et. al ). To isolate RNA and protein, five-day protonema and 8-weak gametophore were used. Protoplasts were obtained by a modified procedure of Liu and Vidali (2011) as previously described (Fesenko et. al); protoplast sediment was frozen and used for further isolation of proteins and RNA. Protein Extraction and Trypsinolysis Proteins were isolated by phenol extraction method as described previously (Fesenko et al, 2015). Briefly, three volumes of cold buffer (500 mM Tris-HCl, pH 8.0, 50 mM EDTA, 700 mM sucrose, 100 mM ÊCl, 1 mM PMSF, 2% 2-mercaptoethanol, 1% Triton X-100) were added to the grinded powder of protonema, gametophore, and protoplasts and the mixture was incubated for 10 min at 4°C. After the incubation, equal amount of phenol equilibrated in Tris-HCl (pH 8.0) was added to the solution; it was centrifuged for 10 min at 5500g at 4°C. Phenolic phase was collected and re-extracted. The protein was precipitated with cold 0.1 M ammonium acetate solution in methanol upon an overnight incubation at 20°. The precipitate was dissolved in the sample buffer (8 Ì urea, 2 Ì thiourea, 10 mM Tris-HCl, pH 8.0). Protein concentration in samples was measured according to Bradford. Solution of DTT (2 M) was added to the protein precipitate dissolved in sample buffer to a final concentration of 5 mM; the mixture was incubated under stirring at 50°C for 30 min. Then, it was cooled to 24°C, 1 M Iodoacetamide (IAA) solution was added, and the mixture was further incubated in darkness under stirring for 20 min at 24°C. Prior to addition of trypsin, the sample was diluted four-fold with 50 mM ammonium bicarbonate. The trypsin solution was added to the sample at the amount of 1/100 to protein concentration; the mixture was incubated at 37°C overnight. Trypsinolysis was terminated with the addition of formic acid to a final concentration of 5% in the solution and the sample was desalted on a C18 cartridge. LC-MS/MS Analysis Tryptic peptides were analyzed on a TripleTOF 5600+ (ABSciex, Canada) quadrupole time-of-flight mass spectrometer equipped with a NanoSpray III ion source combined with a NanoLC Ultra 2D+ nano-HPLC system (Eksigent) chromatography system. Samples were applied onto a ChromXP C18, 3 µm, 120 A, 350 µm ? 0.5 mm (Eksigent, Dublin, CA) trap column in sample loading buffer or buffer A (98.9% water, 1% methanol, and 0.1% formic acid (v/v)) at 3.5 µL/min during 10 min and eluted with further separation on a 3C18-CL-120 (3 µm 120 A) 0.075 mm ? 150 mm (Eksigent, Dublin, CA) column at 300 nL/min. Chromatographic separation was performed from 5 to 40%gradient of buffer B (99.9% acetonitrile and 0.1% formic acid (v/v)) in 120 min. Between the runs, the trap and separation columns were washed with 95% buffer B during 7 min and equilibrated in 5% buffer B for 25 min. Based on the first overview spectrum MS1 (measurement and ion selection for further MS2 analysis in the mass range 3001250 m/z, signal accumulation time 250 ms), 50 parent ions with the maximum intensity in the current spectrum were selected for further MS/MS analysis of fragments (UNIT quadrupole resolution 0.7 amu, mass measurement range 2001800 m/z, ion beam focused for higher sensitivity, signal accumulation time 50 ms for each selected parent ion). For collision-induced dissociation, nitrogen and fixed average energy of collisions of 40 V, independently of the parameters of ion selected, were used. Collision energy changed linearly from 25 to 55 V during the period of signal accumulation (50 ms). Parent ions analyzed under the MS2 mode (their m/z values) were excluded for the second round of selection for 15 s.

### Data Protocol
To identify proteins, the data obtained were analyzed with the ProteinPilot 4.5 revision 1656 (ABSciex, US) and search algorithm Paragon 4.5.0.0 revision 1654 (ABSciex, US) using standard set of identification seting over the protein database uniref100, P.patens taxon, UniProtKB database for Physcomitrella patens, and cosmoss org 1.6 database. The following search parameters were used : cysteines alkylation; enzyme specificity; Triple TOF5600 equipment; organism species unspecified since the database  contains only the P. patens proteins; deep search with further additional statistical analysis of identification reliability. Peptides were identified using a Protein Pilot built-in algorithm ProGroup. Statistical analysis of identification reliability (and determination of the threshold value of unused score) was performed using another Protein Pilot built-in algorithm, ProteomicS Performance Evaluation Pipeline Software (PSPEP).

### Publication Abstract
Alternative splicing (AS) can significantly impact the transcriptome and proteome of a eukaryotic cell. Here, using transcriptome and proteome profiling data, we analyzed AS in two life forms of the model moss Physcomitrella patens, namely protonemata and gametophores, as well as in protoplasts. We identified 12 043 genes subject to alternative splicing and analyzed the extent to which AS contributes to proteome diversity. We could distinguish a few examples that unambiguously indicated the presence of two or more splice isoforms from the same locus at the proteomic level. Our results indicate that alternative isoforms have a small effect on proteome diversity. We also revealed that mRNAs and pre-mRNAs have thousands of complementary binding sites for long non-coding RNAs (lncRNAs) that may lead to potential interactions in transcriptome. This finding points to an additional level of gene expression and AS regulation by non-coding transcripts in Physcomitrella patens. Among the differentially expressed and spliced genes we found serine/arginine-rich (SR) genes, which are known to regulate AS in cells. We found that treatment with abscisic (ABA) and methyl jasmonic acids (MeJA) led to an isoform-specific response and suggested that ABA in gametophores and MeJA in protoplasts regulate AS and the transcription of SR genes.

### Keywords
Alternative splicing, Long non-coding rnas, Proteome, Physcomitrella patens moss

### Affiliations
Federal Research and Clinical Center of Physical-Chemical Medicine of Federal Medical Biological Agency
Shemyakin-Ovchinnikov Institute of Bioorganic Chemistry of the Russian Academy of Sciences

### Submitter
Georgij Arapidi

### Lab Head
Dr Georgij Arapidi
Shemyakin-Ovchinnikov Institute of Bioorganic Chemistry of the Russian Academy of Sciences


