### Accession
PXD038638

### Title
MS-based phosphoproteomic of cytarabine-treated FLT3-ITD cells

### Description
ITD mutations in the FLT3 gene occur in the 30% of acute myeloid leukemia patients. The integration of ITD in the tyrosine kinase domain (TKD-ITD) of the FLT3 receptor has been shown to confer resistance to standard chemotherapy treatment. We applied state-of-the-art, high-sensitive, mass spectrometry (MS)-based (phospho)proteomics to investigate the molecular mechanisms underlying the sensitivity to cytarabine therapy in FLT3-ITD cells.

### Sample Protocol
Cell lysis was performed by adding SDC lysis buffer containing 4% (w/v) SDC, 100 mM Tris -HCl (pH 8.5). Samples were immediately boiled at 95Â° for 5 minutes and sonicated in bioruptor for 10 cycles at high intensity 30s on/30s off. Next, protein concentration was quantified by BCA assay. For the proteome preparation, we used the inStageTip (iST) method for the proteome preparation. Phosphoproteome preparation was performed by the EasyPhos workflow.

### Data Protocol
Raw mass spectrometry data were analyzed in the MaxQuant environment version 1.5.1.6, employing the Andromeda engine for database search. Proteome and phosphoproteome samples were analysed together by specifying two separate groups and setting group specific parameters for each sample type. MS/MS spectra were matched against the Mus musculus UniProtKB FASTA database. Enzyme specificity was set to trypsin. MaxQuants Label free Quantification method and a minimum ration count of two was used for the total proteome samples. For proteome and phosphoproteome analysis, where possible, the identity of peptides present but not sequenced in a given run was obtained by transferring identifications across liquid chromatography (LC)-MS runs. For phosphopeptide identification, an Andromeda minimum score and minimum delta score threshold of 40 and 17 were used, respectively. Peptides had to be fully tryptic in both proteome or phosphoproteme samples and up to two or four missed cleavages were allowed for protease digestion, respectively.

### Publication Abstract
None

### Keywords
Aml resistance, Chemotherapy, Ms-based phosphoproteomic

### Affiliations
University of Rome Tor Vergata
Department of Biology, University of Rome Tor Vergata, Via delle Ricerca Scientifica 1, 00133 Rome, Italy

### Submitter
Francesca Sacco

### Lab Head
Dr Francesca Sacco
Department of Biology, University of Rome Tor Vergata, Via delle Ricerca Scientifica 1, 00133 Rome, Italy


