### Accession
PXD029546

### Title
Crizotinib, Tyrosine Kinase Inhibitors, Create an Antibacterial Active against Gram-positive Bacteria by Targeting the CTP Synthetase PyrG

### Description
Abstract   The infection of drug resistant bacteria seriously threats to public health, which makes it more urgent to find novel antibacterial compounds. Compared with traditional development approaches, drug repurposing provides a faster and more effective strategy to find new antimicrobial agents. Here, we screened an FDA-approved small-molecule library upon S. aureus, and identified crizotinib as an antimicrobial agent. We confirmed the antibacterial activities of crizotinib in vitro and in vivo by MIC, growth curve, SEM and mice experiment. At the same time, crizotinib also showed low tendency to develop resistance. Mechanistically, quantitative proteomics, bioinformatics analyses, qRT-PCR and flow cytometry biochemical validation experience confirmed that crizotinib exerted its antibacterial effects by interfering pyrimidine metabolism and disrupting DNA synthesis eventually. Furthermore, our data from drug affinity responsive target stability, bio-layer interferometry and a series of functional assays demonstrated that crizotinib could target PyrG directly in pyrimidine metabolism pathway. Taken together, our results indicated that crizotinib could be a potential antimicrobial agent to treat Gram-positive bacterial infections in the future.

### Sample Protocol
S. aureus Newman grew at an OD600 ≈ 0.2 in TSB at 37 °C, treated with 1/2 × MIC crizotinib for 1 h or 2 h with their corresponding control group, and then defined as 1 h group and 2 h group, respectively. Bacteria were collected using centrifugation at 10000 ×g for 10 min at 4 °C, washed three times with pre-cold PBS, and resuspended with sodium chloride-tris-EDTA buffer (STE buffer, 10mM Tris-Cl, 0.1M NaCl, 1mM EDTA, pH 8.0) with the final concentration of 0.1 mg/ml lysostaphin at 37 ℃ for 30 min to disrupt cell walls. The cells were lysed by intermittent sonication in 1% SDS lysis buffer (Beyotime: China) on ice. The proteins were quantified using BCA Protein Assay Kit (Thermo, USA). 500 μg total protein from each group were reduced with 50 mM dithiothreitol（DTT）at 37 °C for 1 h, alkylated with 100 mM 2-iodoacetamide (IAA) at room temperature for 30 min, subsequently digested with trypsin (Hua Li Shi, China) at 37 ℃ for 12 h, and then desalted and lyophilized (27).

### Data Protocol
The proteins were considered for further analysis following three criteria: protein level FDR ≤ 1%; unique peptides ≥1; peptide length ≥ 7 aa. The differentially expressed proteins (DEPs) were considered follow as: a p-value < 0.05 and 1.5-fold-change of > 1.5. Three biological repeats were performed for the proteomics study.  Kyoto Encyclopedia of Genes and Genomes (KEGG) pathways enrichment analysis was performed on the DEPs to identify over-represented biological pathways using the ClueGO (version 2.3.3) program and visualized with Cytoscape (version 3.7.1). The p-value of the pathway enrichment was calculated based on the right-hypergeometric test and corrected with Bonferroni. Herein, the p-value was set as two-sided hypergeometric tests and adjusted using Bonferroni correction, and p < 0.05 was considered statistically significant

### Publication Abstract
None

### Keywords
Infection, Keywords: crizotinib, Gram-positive, Pyrg

### Affiliations
MOE Key Laboratory of Tumor Molecular Biology and Key Laboratory of Functional Protein Research of Guangdong Higher Education Institutes, Institute of Life and Health Engineering, Jinan University, Guangzhou, China.
MOE Key Laboratory of Tumor Molecular Biology and Key Laboratory of Functional Protein Research of Guangdong Higher Education Institutes, Institute of Life and Health Engineering, Jinan University, Guangzhou 510632, China

### Submitter
Yun-Dan Zheng

### Lab Head
Dr Xuesong Sun
MOE Key Laboratory of Tumor Molecular Biology and Key Laboratory of Functional Protein Research of Guangdong Higher Education Institutes, Institute of Life and Health Engineering, Jinan University, Guangzhou 510632, China


