### Accession
PXD000838

### Title
Mitochondrial proteomics investigation of a cellular model of impaired dopamine homeostasis, an early step in Parkinson's disease pathogenesis

### Description
Impaired dopamine homeostasis is an early event in the pathogenesis of Parkinson's disease. Generation of intracellular reactive oxygen species consequent to dopamine oxidation leads to mitochondrial dysfunction and eventually cell death. Alterations in the mitochondrial proteome due to dopamine exposure were investigated in the SH-SY5Y human neuroblastoma cell line. The combination of two orthogonal proteomic approaches, two-dimensional electrophoresis and shotgun proteomics, was used to highlight the specific pathways perturbed by the increase of intracellular dopamine, in comparison with those perturbed by a specific mitochondrial toxin (4-methyphenylpyridinium, MPP+), a neurotoxin causing Parkinsonism-like symptoms in animal models. Proteins altered by MPP+ did not completely overlap with those affected by dopamine treatment. In particular, the MPP+ target complex I component NADH dehydrogenase [ubiquinone] iron-sulfur protein 3 was not affected by dopamine together with 26 other proteins. The comparison of proteomics approaches highlighted the fragmentation of some mitochondrial proteins, suggesting an alteration of the mitochondrial protease activity. Pathway and disease association analysis of the proteins affected by dopamine revealed the overrepresentation of the Parkinson's disease and the parkin-ubiquitin proteasomal system pathways and of gene ontologies associated to generation of precursor metabolites and energy, response to topologically incorrect proteins and programmed cell death. These alterations may be globally interpreted in part as the result of a direct effect of dopamine on mitochondria (e.g. alteration of the mitochondrial protease activity) and in part as the effect on mitochondria of a general activation of cellular processes (e.g. regulation of programmed cell death).

### Sample Protocol
Mitochondrial fractions were lysed in 50 µl of 0.1% Rapid Gest SF Surfactant (Waters, Milford, MA, http://www.waters.com) diluted in 50 mM (NH4)2CO3, pH 8.0, according to the manufacturer instructions, and the protein amount was determined with the Bradford method using  the Bio-Rad Protein Assay Dye Reagent Concentrate (Bio-Rad). Tryptic digestion was performed in RapiGest SF as previously described.47 Prior to proteolysis, mitochondrial fractions were subjected to reduction with 10 mM TCEP (30 min at 55°C ) and alkylation with 20 mM iodoacetamide (IAA; 30 min. at RT). Peptide digestion was conducted using 1.5 μg sequence-grade trypsin (Promega, Madison, WI, USA) at 37 °C overnight. The reaction was stopped by acidification with 0.1% formic acid (FA) at 37°C for 30 minutes. To get rid of the acid-labile surfactant RapiGest SF, sample was centrifuged for 10 min at 16200 g and the supernatant saved for the LC MSE analysis.  Samples were diluted with an aqueous solution of 0.1% FA, 3% CH3CN (at a final peptide concentration of 0.4 μg/μL) and loaded on a 5 μm Symmetry C18 trapping column 180 μm × 20 mm (Waters) and separated by a 170 min reversed phase gradient at 250 nL/min (3–40% CH3CN over 145 min) on a nano ACQUITY UPLC System (Waters), using a 1.7 μm BEH 130 C18 Nano Ease 75 μm × 25 cm nano scale LC column (Waters). 150 fmol/μL of MassPrepYeast Enolase digestion standard (Waters), prepared by digesting Yeast Enolase (UniProtKB/Swiss-Prot AC: P00924) with sequencing grade trypsin, were added to each sample as internal standard. The lock mass ([Glu1]-Fibrinopeptide B, Sigma, 500 fmol/μL) was delivered from the auxiliary pump of the instrument with a constant flow rate of 600 nL/min. Separated peptides were mass analyzed by a hybrid quadrupole orthogonal acceleration time-of-flight mass spectrometer (Q-Tof Premier, Waters) directly coupled to the chromatographic system and programmed to step between low (4 eV) and high (15-40 eV) collision energies on the gas cell, using a scan time of 1.5 s per function over 50-1990 m/z.   Mass spectrometry data were acquired in Expression mode (MSE), a data-independent parallel parent and fragment ion analysis without the selection of a fixed ion transmission window on the first mass analyzer prior to collision induced dissociation.48 Continuum LC-MS data from four replicates experiments for each samples were processed for qualitative and quantitative analysis using the software ProteinLynx Global Server v. 2.4 (PLGS, Waters Corp.).

### Data Protocol
Qualitative identification of proteins was obtained with the embedded ion accounting algorithm of the software PLGS 2.4 (Waters Corp.), searching in the human database UniProt KB/Swiss-Prot Protein Knowledgebase (release 2013_08, 24-July-13; 540732 sequence entries, comprising 192091492 amino acids abstracted from 221115 references; taxonomical restrictions: Human, 20266 sequences) to which data from S. cerevisiae Enolase were appended (UniProtKB/Swiss-Prot AC: P00924).25, 49 The search parameters included: automatic tolerance for precursor ions and for product ions, minimum of 3 fragment ions matched per peptide, minimum of 7 fragment ions matched per protein, minimum of 2 peptides matched per protein, 1 missed cleavage, carbamidomethylation of cysteine as fixed modification and oxidation of methionine as variable modification, false positive rate (FPR) fixed below 4% for protein identification and 150 fmol of the Yeast Enolase internal standard set as calibration protein concentration.

### Publication Abstract
Impaired dopamine homeostasis is an early event in the pathogenesis of Parkinson's disease. Generation of intracellular reactive oxygen species consequent to dopamine oxidation leads to mitochondrial dysfunction and eventually cell death. Alterations in the mitochondrial proteome due to dopamine exposure were investigated in the SH-SY5Y human neuroblastoma cell line. The combination of two orthogonal proteomic approaches, two-dimensional electrophoresis and shotgun proteomics (proteomeXchange dataset PXD000838), was used to highlight the specific pathways perturbed by the increase of intracellular dopamine, in comparison with those perturbed by a specific mitochondrial toxin (4-methylphenylpyridinium, MPP(+)), a neurotoxin causing Parkinsonism-like symptoms in animal models. Proteins altered by MPP(+) did not completely overlap with those affected by dopamine treatment. In particular, the MPP(+) target complex I component NADH dehydrogenase [ubiquinone] iron-sulfur protein 3 was not affected by dopamine together with 26 other proteins. The comparison of proteomics approaches highlighted the fragmentation of some mitochondrial proteins, suggesting an alteration of the mitochondrial protease activity. Pathway and disease association analysis of the proteins affected by dopamine revealed the overrepresentation of the Parkinson's disease and the parkin-ubiquitin proteasomal system pathways and of gene ontologies associated with generation of precursor metabolites and energy, response to topologically incorrect proteins and programmed cell death. These alterations may be globally interpreted in part as the result of a direct effect of dopamine on mitochondria (e.g. alteration of the mitochondrial protease activity) and in part as the effect on mitochondria of a general activation of cellular processes (e.g. regulation of programmed cell death).

### Keywords
Dopamine; human proteome project; mitochondria; mpp+; parkinson's disease; respiratory chain; shotgun proteomics; mse

### Affiliations
Proteomics and Metabonomics Laboratory,
CERC- S.Lucia Foundation Rome ,
AND

Italy
and
Dep. of Surgery and Experimental Medicine,
Faculty of Surgery and Medicine,
University of Rome "Tor Vergata"
Rome, Italy
Department of Experimental Medicine and Surgery, University of Rome "Tor Vergata", Rome, and Santa Lucia IRCCS Foundation, Rome, Italy

### Submitter
Luisa Pieroni

### Lab Head
Dr Andrea Urbani
Department of Experimental Medicine and Surgery, University of Rome "Tor Vergata", Rome, and Santa Lucia IRCCS Foundation, Rome, Italy


