### Accession
PXD044171

### Title
Quantitative Phosphoproteomics of whole-body sponge collapse

### Description
Sponges (Porifera) are early-branching Metazoa who do not possess muscles or neurons, however are able to undergo a whole-body movement that involves the closure of their canal system and collapse of an epithelial tent. In this study we profile the proteomic responses of the freshwater sponge Spongilla lacustris during nitric oxide (NO) and agitation induced movements to elucidate the early evolution of coordination in animals. Specifically, we used tandem mass tag (TMT) labeling-based quantification of enriched phosphopeptides to systematically measure quantitative differences in protein phosphorylation. We identified and quantified 12165 unique phosphopeptides in the sponge. NO treatment resulted in quantitative changes of phosphorylation levels on 390 unique phosphopeptides mapping to 270 unique proteins. In turn, agitation led to quantitative changes of phosphorylation levels on 303 unique phosphopeptides (229 proteins).

### Sample Protocol
9 x 50 Spongilla lacustris gemmules were plated in 20 mL filtered lake water (in 55 mm culture dishes) and kept at 18Â°C in the dark. Due to the fact that phosphorylation cascades often occure rapidly, treatment times for agitation and NOC-12 were shortened compared to the TPP experiment. Three plates were agitated (2 min, 500 rpm) and three were treated with 45.5 uM NOC-12 for 2 min. The induction of movements was monitored via brightfield microscopy. For all plates (including 3 plates as control), the medium was discarded and sponges were immediately flash-frozen by dipping the bottom of the culture dishes into liquid nitrogen. This ensured immediate stop of cellular function and ensured that measured phosphorylation events are solely attributed to the treatments. The plates were thawed at 16Â°C for about 30 seconds before addition of 300 uL of lysis buffer (4 M guanidinium isothiocyanate, 50 mM 2-[4-(2-hydroxyethyl)piperazin-1-yl]ethanesulfonic acid (HEPES), 10 mM magnesium chloride, 10 mM tris(2-carboxyethyl)phosphine (TCEP), 1% N-lauroylsarcosine, 5% isoamyl alcohol and 40% acetonitrile adjusted to pH 8.5 with 10 M sodium hydroxide). Sponges were scraped off the culture dish surface using a cell scraper and transferred to 1.5 mL reaction cups with the lysis buffer. Samples were centrifuged at 16,000 x g for 10 minutes at room temperature to remove cell debris and nucleic acid precipitates. Three volumes of ice-cold isopropanol were then added to one volume of supernatant to induce protein precipitation, and the protein precipitates were washed 4 times with 80% isopropanol. Next, proteins were digested overnight by adding trypsin (TPCK-Trypsin, Pierce) at a ratio of 1:25 w/w trypsin:protein in 100â€‰Î¼L of digestion buffer (30â€‰mM chloroacetamide, 5â€‰mM TCEP, 100â€‰mM HEPES, pH 8.5). After digestion, peptides were acidified with trifluoroacetic acid (TFA, final concentration of 1%) and desalted with solidâ€�phase extraction by loading the samples onto a Waters tC18 Sep-Pak 50-mg column. Samples were washed twice with 1â€‰ml 0.1% TFA and eluted with 400â€‰Î¼l 50% acetonitrile acidified with 0.1% TFA before lyophilization.  For phosphopeptides enrichment, lyophilized peptides were resuspended in buffer A (70% ACN, 0.07% TFA), centrifuged at 16,000 x g, and the supernatant was added to Fe-NTA agarose beads (PureCube) on a multiscreenHTS-HV 0.45 Âµm 96 well filter plate (Merck Millipore). After loading, beads were washed 6 times with 200â€‰Î¼L buffer A, each time followed by centrifugation at 250 x g for 1 minute at room temperature. Phosphopeptides were eluted by addition of 0.2% diethylamine in 80% ACN (2 x 50â€‰Î¼L), before lyophilization. For TMT labeling, dried phosphopeptides were dissolved in 10â€‰Î¼L 100 mM HEPES pH 8.5 before addition of 4 Î¼L of TMT reagent at a concentration of 20â€‰Î¼g/â€‰Î¼L in acetonitrile. After 1h at room temperature, the labeling reaction was quenched for 15 minutes by addition of 5â€‰Î¼L of 5% hydroxylamine. Labeled peptides were then pooled and subsequently lyophilized.   Labeled phosphopeptides were dissolved into 50 ÂµL of buffer A (20 mM ammonium formate at pH 10) and loaded onto a C18 microcolumn made in-house and packed with 1 mg of C18 bulk material (ReproSil-Pur 120 C18-AQ 5 Âµm, Dr. Maisch) on top of the C18 resin disk (AttractSPE disks bio - C18, Affinisep). The microcolumn was first washed twice with 20 ÂµL buffer A before phosphopeptides fractionation. The different fractions were collected after addition and elution of 10 ÂµL of the following solutions: 3%, 5%, 7%, 9%, 11%, 13%, 15%, 17%, 19%, 21%, 23%, 25%, 27%, 29%, 31%, 33%, 35% and 40% ACN in buffer A. The fractionation was carried on in a bench top centrifuge, at room temperature and with centrifugation speed matching flow rates of approximately 10 ÂµL/min. Each n fraction was pooled with the n + 6 and n + 12 fractions, resulting in 6 fractions. The different fractions were finally lyophilized before LC-MS/MS analysis. MS equipment: Orbitrap Fusion Lumos (Thermo Fisher Scientific) using a Nanospray-Flex ion source. MS settings (Spray voltage: 2.4 kV, Capillary temperature: 275 ËšC, Full scan mass range 375-1400 m/z, MS1 Orbitrap resolution: 120000, MS1 Maximum IT: 50 ms, MS1 AGC target: 4e5, MS2 resolution: 30000, MS2 AGC target: 1e5, MS2 maximum IT: 110 ms, MS2 isolation window: 0.7, MS2 fixed first mass: 110, MS2 nCE: 36, MS2 dynamic exclusion: 20 s.

### Data Protocol
Mass spectrometry raw files were converted to mzmL format using MSConvert (Proteowizard) using peak picking from the vendor algorithm and keeping the 200 most intense peaks. Files were then searched using MSFragger v3.4 in Fragpipe v17 against the Spongilla lacustris proteome (proteins with minimal length of 70 aa identified from transcriptome with TransDecoder (version 3.0.1)) including known contaminants and the reversed protein sequences. The search parameters were as followed: tryptic digestion with a maximum of 2 missed cleavages, peptide length = 7-50, peptide tolerance = 20â€‰ppm; MS/MS tolerance = 10 ppm; topN peaks = 200; fixed modifications = carbamidomethyl on cysteine and TMT10plex on lysine; variable modifications = acetylation on protein N termini, oxidation of methionine and TMT10plex on peptide N termini, as well as variable phosphorylation of serine, threonine and tyrosine residues with a maximum of 4 phosphorylations per peptides. Peptide Spectrum Matches (PSMs) validation was performed by philosopher version 4.1.0 and false discovery rate was fixed at 1% at the PSMs, peptides and proteins level. TMT quantification was performed by TMT-Integrator using the default settings.  The raw output files of FragPipe (psm.tsv â€“ files)  were processed using the R programming language. Only peptide spectral matches (PSMs) with a phosphorylation probability greater 0.75 were considered for the analysis. Phosphorylated amino acids were marked with a (P) in the amino acid sequences behind the phosphorylated amino acid and concatenated with the protein ID in order to create a unique ID for each phospho peptide. Raw TMT reporter ion intensities were summed for all PSMs with the same phosphopeptide ID. Log2 transformed summed TMT reporter ion intensities were first cleaned for batch effects using the â€˜removeBatchEffectsâ€™ function of the limma package and further normalized using the vsn package (variance stabilization normalization). Proteins were tested for differential expression using the limma package. The replicate information was added as a factor in the design matrix given as an argument to the â€˜lmFitâ€™ function of limma. A phosphopeptide was annotated as a hit with a false discovery rate (fdr) smaller 5 % and a fold-change of at least 100 % and as a candidate with a fdr below 20 % and a fold-change of at least 50 %.

### Publication Abstract
A hallmark of animals is the coordination of whole-body movement. Neurons and muscles are central to this, yet coordinated movements also exist in sponges that lack these cell types. Sponges are sessile animals with a complex canal system for filter-feeding. They undergo whole-body movements resembling "contractions" that lead to canal closure and water expulsion. Here, we combine live 3D optical coherence microscopy, pharmacology, and functional proteomics to elucidate the sequence and detail of shape changes, the tissues and molecular physiology involved, and the control of these movements. Morphometric analysis and targeted perturbation suggest that the movement is driven by the relaxation of actomyosin stress fibers in epithelial canal cells, which leads to whole-body deflation via collapse of the incurrent and expansion of the excurrent canal system. Thermal proteome profiling and quantitative phosphoproteomics confirm the control of cellular relaxation by an Akt/NO/PKG/PKA pathway. Agitation-induced deflation leads to differential phosphorylation of proteins forming epithelial cell junctions, implying their mechanosensitive role. Unexpectedly, untargeted metabolomics detect a concomitant decrease in antioxidant molecules during deflation, reflecting an increase in reactive oxygen species. Together with the secretion of proteinases, cytokines, and granulin, this indicates an inflammation-like state of the deflating sponge reminiscent of vascular endothelial cells experiencing oscillatory shear stress. These results suggest the conservation of an ancient relaxant-inflammatory response of perturbed fluid-carrying systems in animals and offer a possible mechanism for whole-body coordination through diffusible paracrine signals and mechanotransduction.

### Keywords
Phosphoproteomics, Actomyosin, Evolution, Spongilla

### Affiliations
EMBL
Developmental Biology Unit, European Molecular Biology Laboratory, 69117 Heidelberg, Germany.

### Submitter
Frank Stein

### Lab Head
Dr Detlev Arendt
Developmental Biology Unit, European Molecular Biology Laboratory, 69117 Heidelberg, Germany.


