### Accession
PXD016563

### Title
Laser-capture proteomics of human FFPE substantia nigra

### Description
Laser-capture microdissection (LCM) allows the visualization and isolation of morphologically distinct subpopulations of cells from heterogeneous tissue specimens. In combination with formalin-fixed and paraffin-embedded (FFPE) tissue it provides a powerful tool for retrospective and clinically relevant studies of tissue proteins in a healthy and diseased context.  In this study, we have developed an optimized protocol to facilitate efficient LCM analysis of FFPE tissue specimens. First, we optimized protein extraction from FFPE tissue by comparing different extraction buffers and investigating the influence of immunohistochemical and haematoxylin & eosin staining on proteins. SDS present in the protein extracts was removed with the SP3 digest method, which was modified to improve protein and peptide recoveries. Using a label-free approach protein expression of microdissected samples was compared to intact tissue sections from substantia nigra to evaluate the efficiency of LCM for the purification of small cell populations. The optimized protocol was used to analyse samples containing as few as ~3,000 cells isolated from the substantia nigra, using FFPE tissue. Replicate samples of 15 healthy donors were analysed in five separate TMT10plex batches, resulting in the quantification of >5,600 protein groups.

### Sample Protocol
Deparaffinized FFPE sections were resuspended in 100 µL SDS (2% SDS in 300 mM Tris-HCl pH 8.0), sodium deoxycholate (SDC; 1% SDC in 300 mM Tris-HCl pH 8.5) or Rapigest (0.2% Rapigest in 50 mM ammonium bicarbonate (ABC)) extraction buffer. Samples were boiled (25 min, 99 °C, 350 rpm) and sonicated for 20 cycles (Bioruptor Plus bath sonicator, Diagenode; 30 s on, 30 s off) followed by heating for 2 h at 80 °C (500 rpm) and another sonication step (20 cycles). After centrifugation (10 min, 21,300 g) the supernatant was transferred into a new tube. Proteins were reduced with 10 mM DTT (45 min, 50 °C) followed by alkylation with 20 mM IAA (45 min, 22 °C, dark). Rapigest and SDC extracts were in-solution digested overnight (1:50 trypsin:protein ratio, 37 ºC). The next day another portion of trypsin was added (4 h). After acidification (f.c. 1% formic acid (FA)) and centrifugation (10 min, 21,300 g) samples were desalted by SPE (Waters Sep-Pak tC18 cartridges, 50 mg). Laser-capture microdissected tissue samples were resuspended in 100 µL SDS buffer followed by the preparation steps described above. Protein extracts in SDS buffer were cleaned and digested with the SP3 method as described previously with modifications. Briefly, 10 µL of a 20 µg/µL SP3 bead stock (Sera-Mag SpeedBead carboxylate-modified magnetic particles; GE Healthcare Life Sciences) for LCM samples (or a 1:10 protein:bead ratio for sections) and 500 µL acetonitrile (ACN; f.c. of 70%) were added to 200 µL of protein extract and incubated for 10 min (1000 rpm). Tubes were mounted on a magnetic rack, beads were washed (2x 70% ethanol, 1x ACN; 1 mL each) and resuspended in 80 µL 50 mM ABC followed by digestion overnight with trypsin (1:50 trypsin:protein ratio for sections, 0.5 µg for LCM samples; 37 °C, 1,000 rpm). The next day another portion of trypsin was added (4 h). Peptides were cleaned by addition of 9 µL 10% FA (f.c. 1%) and 1750 µL ACN (f.c. 95%) followed by incubation for 10 min. After spinning down (1,000 g) beads were washed with 1.5 mL ACN. Peptides were eluted from the beads with 50 µL 2% DMSO and acidified with 2.6 µL 20 % FA (f.c. 1%) followed by centrifugation (15,000 g). Dried peptide samples were resuspended in 50 µL 100 mM HEPES pH 8.5. TMT10plex tags, dissolved in anhydrous ACN, were added in a 1:10 peptide:TMT ratio. Additional anhydrous ACN was added to a final volume of 22 µL (2 h, 22 °C, 750 rpm). Unreacted TMT was quenched by incubation with 5 µL 5% hydroxylamine for 30 min. Samples belonging to the same batch were combined and dried in vacuo. TMT labelled samples were fractionated using off-line high pH reversed phase chromatography (Dionex Ultimate 3000; 4.6 x 250 mm XBridge BEH130 C18 column, 3.5 µm, 130 Å; Waters). Peptides were separated into 48 fractions, which were concatenated into 24 fractions and subsequently dried in vacuo. Label-free peptides were analysed on a Q-Exactive Plus mass spectrometer coupled to a Dionex RSLCnano HPLC (Thermo Scientific). Samples were loaded onto a 100 µm × 2 cm Acclaim PepMap-C18 trap column (5 µm, 100 Å). Peptides were separated on a 75 µm × 50 cm EASY-Spray C18 column (2 µm, 100 Å) at 40 ºC over a linear gradient from 10% to 35% B in 110 min (200 nL/min). Solvents used were 0.1% FA (A) and 80% ACN/0.1% FA (B). The spray was initiated by applying 2.5 kV to the EASY-Spray emitter. The ion transfer capillary temperature was set to 250 °C and the RF of the S-lens to 50%. Data were acquired in DDA top 15 mode. Full scan MS spectra were acquired with a resolution of 70,000 (m/z 350-1,400, AGC 1E6, max. IT 20 ms). Precursor ions (charge 2-6) were isolated in the quadrupole with a window of m/z 1.5 and fragmented in the HCD cell with NCE 27. Spectra of fragment ions were acquired with a resolution of 17,500 (AGC 1E5, max. IT 60 ms, dynamic exclusion 40 ms). TMT labelled samples were analysed on an Orbitrap Fusion Tribrid mass spectrometer. LC and spray parameters used were the same as described above. Peptides were separated over a linear gradient from 10% to 40% B in 210 min (200 nL/min). The full scan was acquired in the orbitrap (m/z 350-1,400, resolution 120,000, AGC 2E5, max. IT 50 ms). Precursor ions (charge 2-7) were isolated with a window of m/z 1.6 and fragmented in the ion trap with 35% CID collision energy. The ion trap scan rate was set to “rapid” (AGC 4E3, max. IT 180 ms, dynamic exclusion 80 s). During the MS3 analysis 5 fragment ions were co-isolated using synchronous precursor selection in a window of m/z 2 and further fragmented with a HCD collision energy of 55%. The fragments were analysed in the orbitrap with a resolution of 50,000 (AGC 5E4, max. IT 115 ms). The cycle duration was 2 s. TMT batch 4 was measured with the following changes of parameters: linear gradient from 10% to 40% B in 150 min (200 nL/min), full scan: AGC 2E5, isolation window m/z 1.2, ion trap scan rate “turbo” (AGC 1E4, max. IT 50 ms, dynamic exclusion 60 s), MS3: max. IT 86 ms, cycle time 3 s.

### Data Protocol
Acquired tandem mass spectra were searched against the Uniprot human database (downloaded on 14th of June 2018, 20,349 proteins) using MaxQuant (version 1.6.2.3), allowing up to two missed cleavages and a mass tolerance of 4.5 ppm for precursor ions and 0.5 Da (Ion trap, used for SPS-MS3 scans) or 20 ppm (Orbitrap) for MS2 fragment ions. Carbamidomethylation of cysteine was used as a fixed modification, with oxidation of methionine, deamidation of asparagine and glutamine and acetylation of the protein N-terminus used as variable modifications. For TMT data the type of search was set to reporter ion MS3 using TMT10plex. The fractions of all five TMT batches were searched together. For label-free quantification the Ã¢â‚¬Å“match between runsÃ¢â‚¬ï¿½ option in MaxQuant was only applied to the search for the comparison of microdissected samples with intact tissue sections. All other searches of label-free data, regarding the optimization of protein extraction, were performed without this option. A false discovery rate of 1% was applied to peptide and protein identifications.

### Publication Abstract
Laser-capture microdissection (LCM) allows the visualization and isolation of morphologically distinct subpopulations of cells from heterogeneous tissue specimens. In combination with formalin-fixed and paraffin-embedded (FFPE) tissue it provides a powerful tool for retrospective and clinically relevant studies of tissue proteins in a healthy and diseased context. We first optimized the protocol for efficient LCM analysis of FFPE tissue specimens. The use of SDS containing extraction buffer in combination with the single-pot solid-phase-enhanced sample preparation (SP3) digest method gave the best results regarding protein yield and protein/peptide identifications. Microdissected FFPE human substantia nigra tissue samples (&#x223c;3,000 cells) were then analyzed, using tandem mass tag (TMT) labeling and LC-MS/MS, resulting in the quantification of &gt;5,600 protein groups. Nigral proteins were classified and analyzed by abundance, showing an enrichment of extracellular exosome and neuron-specific gene ontology (GO) terms among the higher abundance proteins. Comparison of microdissected samples with intact tissue sections, using a label-free shotgun approach, revealed an enrichment of neuronal cell type markers, such as tyrosine hydroxylase and alpha-synuclein, as well as proteins annotated with neuron-specific GO terms. Overall, this study provides a detailed protocol for laser-capture proteomics using FFPE tissue and demonstrates the efficiency of LCM analysis of distinct cell subpopulations for proteomic analysis using low sample amounts.

### Keywords
Ffpe, Laser-capture microdissection, Substantia nigra, Tmt

### Affiliations
Centre for Gene Regulation & Expression, School of Life Sciences, University of Dundee
Drug Discoverey Sciences Department, Boehringer Ingelheim Pharma GmbH & Co. KG, Biberach an der Riss, Germany

### Submitter
Eva Griesser

### Lab Head
Dr Angus I. Lamond
Centre for Gene Regulation & Expression, School of Life Sciences, University of Dundee


