### Accession
PXD009838

### Title
Stable isotope dynamic labelling of secretomes: Strataclean Linearity C4PR_LIV

### Description
We improve on currently-available resources by describing a mass spectrometry (MS)-based strategy using stable isotope dynamic labelling of secretomes (SIDLS) that discriminates between authentic secretory proteins and intracellular proteins within the secretome of cultured cells. By monitoring the rate of incorporation of labelled amino acids into newly synthesised proteins as they appear in the media, we can differentiate those proteins that have been destined for secretion, and exhibit rapid labelling, from those with low rates of labelling or low turnover relative to the growth rate of the cells which is a feature of intracellular proteins. Part of the wet lab protocol for our analysis of secretomes, is the use of the resin, Strataclean (Agilent). In this Supplementary dataset to our main paper (also in PRIDE/ProteomeXchange), we characterise the linearity of capture of secreted proteins using Stracalean resin, by label-free quantitative mass spectrometry. We mix cell-coditioned culture media with 'virgin' media in different ratios and analayse the capture capacity of equal quantitites of StrataClean reagent.

### Sample Protocol
Cell-conditioned media samples were concentrated using StrataClean prior to proteolytic digestion with trypsin and LC-MSMS analysis on a Nano-Acquity RP-HPLC system, coupled in-line to an LTQ Orbitrap Q-Exactive mass spectrometer. Digests (2 microL) from each secretome taken at each time point, were loaded onto a trap column (Acclaim PepMap 100, 2 cm x 75 microM inner diameter, C18, 3 microM, 100 Å) at 5 microL per min with an aqueous solution containing 0.1%(v/v) TFA and 2%(v/v) acetonitrile. After 3 min, the trap column was set in-line with an analytical column (Easy-Spray PepMap® RSLC 15 cm x 50cm inner diameter, C18, 2 microlM, 100 Å) (Dionex). Peptides were loaded in 0.1%(v/v) formic acid and eluted with a linear gradient of 3.8–40% buffer B (HPLC grade acetonitrile 80%(v/v) with 0.1%(v/v) formic acid) over 95 min at 300 nl per min, followed by a washing step (5 min at 99% solvent B) and an equilibration step (15 min at 3.8% solvent). All peptide separations were carried out using an Ultimate 3000 nano system (Dionex/Thermo Fisher Scientific). The column was operated at a constant temperature of 35 °C and the LC system coupled to a Q-Exactive mass spectrometer. The Q-Exactive was operated in data-dependent mode with survey scans acquired at a resolution of 70,000 at m/z 200. Up to the top 10 most abundant isotope patterns with charge states +2, +3, and/or +4 from the survey scan were selected with an isolation window of 2.0 Th for fragmentation by higher energy collisional dissociation with normalized collision energies of 30. The maximum ion injection times for the survey scan and the MS/MS scans were 250 and 100 ms, respectively, and the ion target value was set to 1E6 for survey scans and 1E5 for the MS/MS scans. Repetitive sequencing of peptides was minimized through dynamic exclusion of the sequenced peptides for 20 s.

### Data Protocol
Acquired MS data were searched and analysed using Andromeda and MaxQuant 1.5.8.3 against a reviewed human UniProt protein database (date: 07/12/2015), using the default settings; briefly: the minimum required peptide length was seven amino acids long and a single missed cleavage was allowed. Cysteine carbamidomethylation was set as a fixed modification and methionine oxidation was allowed as a variable modification. The initial precursor and fragment ion maximum mass deviations were set to 20 ppm and 0.5 Da, respectively. Peptide and protein false discovery rates (FDRs) were set to 1%, the ‘requant’ function activated and ‘match between runs’ enabled with the default parameters. To assess changes in the label-free abundance of proteins identified from the 1% FDR Andromeda searches, we used the intensity values reported by MaxQuant (in proteinGroups.txt) on a per protein basis in the cell-conditioned:virgin culture medium mixtures.

### Publication Abstract
Analysis of secretomes critically underpins the capacity to understand the mechanisms determining interactions between cells and between cells and their environment. In the context of cancer cell micro-environments, the relevant interactions are recognized to be an important determinant of tumor progression. Global proteomic analyses of secretomes are often performed at a single time point and frequently identify both classical secreted proteins (possessing an N-terminal signal sequence), as well as many intracellular proteins, the release of which is of uncertain biological significance. Here, we describe a mass spectrometry-based method for stable isotope dynamic labeling of secretomes (SIDLS) that, by dynamic SILAC, discriminates the secretion kinetics of classical secretory proteins and intracellular proteins released from cancer and stromal cells in culture. SIDLS is a robust classifier of the different cellular origins of proteins within the secretome and should be broadly applicable to nonproliferating cells and cells grown in short term culture.

### Keywords
Human, Oesophageal cancer, Secretome

### Affiliations
University of Liverpool
Dept. of Cellular & Molecular Physiology Institute of Translational Medicine University of Liverpool Crown Street Liverpool L69 3BX

### Submitter
Dean Hammond

### Lab Head
Dr Dean Hammond
Dept. of Cellular & Molecular Physiology Institute of Translational Medicine University of Liverpool Crown Street Liverpool L69 3BX


