### Accession
PXD023399

### Title
Palmer et al:  Dental Method Comparison

### Description
The dental calculus proteome from human remains is a considered a valuable archaeological resource for studying the diet, oral microbiome, health/disease of individuals and occupation activities individuals within past societies/civilisations. The combination of advancements in LC/MS instrument sensitivity and sample prepartion methods enables the analysis of these well preserved proteomes otherwise invisible to other biomolecular approaches. Maximizing proteome recovery from this minute and finite resource is paramount to our understanding of these past societies and civilisations in terms of diet and disease. Here we compare the more traditional ultrafiltration-based and acetone precipitation approaches with the newer paramagnetic bead approach (SP3) in a step towards achieving this aim. We specifically evaluate different acids (EDTA and HCl) and proteome extraction methodologies (Ultrafiltration, Acetone Precipitation and SP3)to test the influence of demineralization acid and method of extraction on the recovered proteome complexity (including numbers of peptides recovered) and sequence coverageand observed for these ancient dental calculus specimens.

### Sample Protocol
Dental calculus specimens were extracted using either a modified version of the Cleland SP3 protocol for bone or the Procopio and Buckley ultrafilitration protocol for bone as stated in the paper by Palmer et al entitled: Comparing the use of magnetic beads with ultrafiltration for ancient dental calculus proteomics. Dental calculus was pre-washed using either HCl or EDTA and then demineralised in either HCl or EDTA respectively. The resulting supernatants were aliquoted, stored temporarily at -20 oC for 24 h and the remaining pellets were treated with GuHCl buffer and incubated at 4 oC for 18 h. On completion of the GuHCl treatment, the supernatant was removed and processed along it's equivalent soluble fraction. All the fractions for each sample were processed separately and the buffers exchanged from EDTA, HCl and GuHCl to 50 mM ABC for the fractions to be processed by either 3kDa MWCO ultrafiltration columns or acetone (acetone precipitation). The fractions for extraction by SP3 were left in either EDTA or GuHCl following demineralisation and extraction of the soluble and insoluble protein fractions respectively. Acetone precipitation was used as an alternative to SP3 for the HCl soluble protein fractions only. All fractions were reduced and alkylated with DTT and IAM prior to tryptic digestion and desalting using CMIX18 zip-tips prior to LC-MS/MS.

### Data Protocol
Raw spectra were converted to Mascot generic files (created assuming 0~13Cs with peak list generation using ExtractMSN) and initially searched against SwissProt (556,568 sequences; 199,530,821 residues) and HOMD (4,026,163 sequences; 1,246,432,637 residues) using the Mascot Search Engine  (version 2.4.1; Matrix Science, London, UK) against as decoy database. Each search included: the fixed carbamidomethyl modification of cysteine (+57.02 Da), the variable modifications of deamidation (+0.98 Da) of asparagine/glutamine, and oxidation (+15.99 Da) of proline and methionine residues. Enzyme specificity was set to trypsin with up to 2 missed cleavages allowed; mass tolerance were set to 10 ppm for the precursor ions and 0.5 Da for the fragment ions, with all spectra considered as having 2+ or 3+ precursors for the data generated by the LC-Orbitrap Elite system. All parameters were the same with the following exception for the QE-HF; the fragment ion specificity was reduced to 0.02 Da. Scaffold software version 4.11.0 (Proteome Software Inc., Portland, OR) was used to analyse the Mascot data generated. X!Tandem was selected within Scaffold to validate the peptides and proteins identified previously by Mascot. Peptide identifications were accepted if they exceeded the 95% probability threshold for the Peptide prophet algorithm with Scaffold delta mass correction. Protein identities were accepted if contained 2 unique peptides and established at greater than 99% probability. The samples were considered individually and as distinct groups for each experimental condition. The false-discovery rate (FDR) for the peptide identity threshold and peptide identity or homology threshold were generated using Mascotâ€™s reverse peptide decoy databases.

### Publication Abstract
Over the past two decades, proteomic analysis has greatly developed in application to the field of biomolecular archaeology, coinciding with advancements in LC-MS/MS instrumentation sensitivity and improvements in sample preparation methods. Recently, human dental calculus has received much attention for its well-preserved proteomes locked in mineralized dental plaque which stores information on human diets and the oral microbiome otherwise invisible to other biomolecular approaches. Maximizing proteome recovery in ancient dental calculus, available only in minute quantities and irreplaceable after destructive analysis, is of paramount importance. Here, we compare the more traditional ultrafiltration-based and acetone precipitation approaches with the newer paramagnetic bead approach in order to test the influence of demineralization acid on recovered proteome complexity obtained from specimens as well as the sequence coverages matched for significant proteins. We found that a protocol utilizing EDTA combined with paramagnetic beads increased proteome complexity, in some cases doubling the number of unique peptides and number of proteins matched, compared to protocols involving the use of HCl and either acetone precipitation or ultrafiltration. Although the increase in the number of proteins was almost exclusively of bacterial origin, a development that has implications for the study of diseases within these ancient populations, an increase in the peptide number for the dairy proteins &#x3b2;-lactoglobulin and casein was also observed reflecting an increase in sequence coverage for these dietary proteins of interest. We also consider structural explanations for the discrepancies observed between these two key dietary proteins preserved in archaeological dental calculus.

### Keywords
Archaeologica dairy proteinl, Acetone precipitation, Deamidation, Edta, Hcl, Beta-lactoglobulin, Ultrafiltration, Orbi-trap, Qe, Casein, Dental calculus proteome, Sp3, Mass spectrometry

### Affiliations
University of Manchester
School of Natural Sciences, Manchester Institute of Biotechnology, The University of Manchester, 131 Princess Street, Manchester, M1 7DN, U.K.

### Submitter
Karren Palmer

### Lab Head
Dr Michael Buckley
School of Natural Sciences, Manchester Institute of Biotechnology, The University of Manchester, 131 Princess Street, Manchester, M1 7DN, U.K.


