### Accession
PXD004218

### Title
Direct in-gel profiling of protease specificity (DIPPS)

### Description
Determination of protease specificity is of crucial importance for understanding protease function. We have developed the first gel-based label-free proteomic approach (DIPPS) that enables quick and reliable determination of protease cleavage specificities under large variety of experimental conditions. The methodology is based on in-gel digestion of the gel-separated proteome with the studied protease, enrichment of cleaved peptides by gel extraction and subsequent mass spectrometry analysis combined with a length-limited unspecific database search. We applied the methodology to profile ten proteases ranging from highly specific (trypsin, endoproteinase GluC, caspase-7 and legumain) to broadly specific (matrix-metalloproteinase-3, thermolysin and cathepsins K, L, S and V). Using DIPPS, we were able to perform specificity profiling of thermolysin at its optimal temperature of 75C, which confirmed the applicability of the method to extreme experimental conditions. Moreover, DIPPS enabled the first global specificity profiling of legumain at pH as low as 4.0, which revealed a pH-dependent change in the specificity of this protease, further supporting its broad applicability.

### Sample Protocol
The proteins were separated on 12.5 % SDS-PAGE Precast gels (Lonza) with 150 µg of total proteins loaded per lane. SDS-PAGE was stained for 1 hour with Coomassie Brilliant blue stain followed by overnight destaining in 30 % ethanol/10% acetic acid. The whole protein lanes were cut using a gel grid cutter (The gel company) to produce 30 even sized bands. Each sample was combined from 5 adjacent bands to produce a total of 6 samples per condition. The gel bands were further sliced into approximately 1-mm3 pieces and destained twice with 150 µl 50 % ACN (acetonitrile) in 25 mM ABC (ammonium bicarbonate) for 30 min at 25 ˚C and 1200 rpm. The gel pieces were dehydrated with 100 % ACN and vacuum dried. The cysteine residues were reduced with 150 µl 10 mM DTT (dithiothreitol) in 25 mM ABC for 30 min at 56 ˚C. Afterwards alkylation was performed with 150 µl 55 mM IAM (iodoacetamide) in 25 mM ABC in the dark for 30 min at 25 ˚C. The excess IAM was quenched with 150 µl 20 mM DTT in 25 mM ABC for 30 min at 37 ˚C, and the gel pieces were further washed with 150 µl MS grade water followed by two additions of 150 µl 100 % ACN until complete dehydration. The samples were vacuum dried and stored at – 20 ˚C until digestion.  In-gel digestion For the in-gel digestion, 80 µl of 1 µM protease solution in the appropriate protease buffer were added to the samples and incubated on ice for 30 min to allow the gel pieces to absorb the buffer with the protease. Prolegumain was first activated in activation buffer (50 mM citric buffer, pH 4.0, 5 mM DTT) for 30 min at 37 ˚C. Activated 16.7 µM legumain was diluted in 0.1 M citric buffer pH 4.0, pH 5.0 or pH 6.0 with 5 mM DTT for in-gel digestion. Prior to in-gel digestion legumain was pretreated with 10 µM E-64 in order to inhibit trace amounts of cathepsin L like protease from Spodoptera frugiperda (Johnson & Jiang, 2005).The cathepsins with different active concentrations (Cathepsin B = 47 µM, Cathepsin K = 33 µM, Cathepsin L = 45 µM, Cathepsin S = 20 µM, Cathepsin V = 98 µM) were diluted in 0.1 M citric buffer pH 6.0 with 5 mM DTT. Matrix metalloproteinase-3 was diluted in 50 mM Tris.HCl, 150 mM NaCl, 10 mM CaCl2 buffer pH 7.5. Caspase-7 with active concentration 50 µM was diluted in 0.1 M HEPES pH 7.4 with 5 mM DTT and incubated for 18 hours at 37 ˚C. Thermolysin (Promega) was profiled at 1 µM in 50 mM Tris.HCl, 0.5 mM CaCl2, pH 8.0 for 2 hours at 75 °C. The sequencing grade modified porcine trypsin (Promega) was used at 1 µM in 0.1 M ABC buffer pH 7.8. Glu-C (Promega) from S. aureus V8 was diluted in 100 mM phosphate buffer pH 7.8 to 1 µM final concentration. To prevent the gel pieces from drying out during digestion 100 µl of protease assay buffer were added to each sample and incubated for 2 hours at 37 ˚C unless otherwise noted. Peptide extraction and desalting The peptides generated by the investigated protease were extracted using 150 µl of 50 % ACN in 5 % FA for 30 min at 25 ˚C. The extracted peptide solution was transferred into a new tube, and the extraction step was repeated twice. The extracted peptides were vacuum concentrated to approximately 100 µl to remove the excess acetonitrile. The peptides were desalted using 4 stacks of C18 discs (Empore) in a 200-µl pipet tip. The C18 tips were first activated with 100 % methanol and washed with 80 % ACN/3% acetic acid and subsequently equilibrated with 0.1 % FA in water and loaded with 100 µl of peptide solution. The bound peptides were washed with 0.1 % FA followed by the peptide elution with 60 % ACN/0.1 % FA in water. Acetonitrile was removed by vacuum concentrating to a final volume of 12 μl.  LC-MS/MS analysis  The samples were analyzed by LC-MS/MS using an EASY-nanoLC II HPLC unit (Thermo Fischer Scientific) coupled to an Orbitrap LTQ Velos mass spectrometer (Thermo Fischer Scientific) automatically operated via Xcalibur software (Thermo Fischer Scientific). The samples containing 0.1 % FA were loaded onto a C18 trapping column (Proxeon Easy-column, Thermo Fischer Scientific) and separated on a C18 PicoFrit Aquasil analytical column (New Objective). The peptides were eluted using a 5 to 40 % (v/v) 50 min linear gradient of acetonitrile with 0.1 % FA at a flow rate of 300 nl/min. The full MS mass spectra were acquired with the Orbitrap mass analyzer in the mass range of 300 to 2,000 m/z at resolution of 30,000 in the profile mode. The MS/MS spectra were obtained by HCD fragmentation of the nine most intense MS precursor ions and recorded at resolution of 7,500 in the centroid mode. Only the precursor ions with assigned charge state (>1) were chosen for MS/MS fragmentation. The dynamic exclusion was set to repeat count of 1, repeat duration of 30 s and exclusion duration of 20 s.

### Data Protocol
Database searches were performed using the MaxQuant software package version 1.5.6.0 imbedded with the Andromeda search engine (Cox & Mann, 2008; Cox et al, 2011). The searches were performed against the UniProt derived human reference proteome (UniProtKB, Homo sapiens, canonical database containing 20,114 entries, released in November 2016) and Uniprot derived Aeropyrum pernix canonical database (1700 entries, released in April 2017). The protease-generated peptides were searched using the unspecific cleavage settings with peptide length limitation from 8 to 40 amino acid residues and a maximum molecular weight of 4,600 Da. The search engine parameters were set to consider carbamidomethylation of cysteine residues (+57.0215 Da) as fixed modification and oxidation of methionine (+15.9949 Da) and acetylation of protein N-termini (+42.0106 Da) as variable modifications. The precursor ion and fragment ion mass tolerances were set as 4.5 ppm and 0.5 Da, respectively. A reversed database search was performed with 1 % false discovery rate for protein and peptide identification. The identified peptide list (peptides.txt) was filtered to remove the reverse and contaminant peptides and peptides with scores less than 40 and PEP values larger than 0.05. Additionally, the identified peptide N- or C-termini corresponding to N- or C-protein termini and duplicated cleavage sites were removed. The protease cleavage window was used for the protease specificity determination using the iceLogo tool (Colaert et al, 2009). The reference set used for the calculation of the chance of the amino acid occurrence (p-value) at a certain position was the human reference proteome (also used for the database searches). The background proteome was sampled using the random sampling method. The amino acid occurrences (enrichment and fold of change) obtained by iceLogo were plotted as heat maps using Gnuplot (www.gnuplot.info) and the heat map script as shown previously (Eckhard et al, 2016).

### Publication Abstract
Determination of protease specificity is of crucial importance for understanding protease function. We have developed the first gel-based label-free proteomic approach (DIPPS-direct in-gel profiling of protease specificity) that enables quick and reliable determination of protease cleavage specificities under large variety of experimental conditions. The methodology is based on in-gel digestion of the gel-separated proteome with the studied protease, enrichment of cleaved peptides by gel extraction, and subsequent mass spectrometry analysis combined with a length-limited unspecific database search. We applied the methodology to profile ten proteases ranging from highly specific (trypsin, endoproteinase GluC, caspase-7, and legumain) to broadly specific (matrix-metalloproteinase-3, thermolysin, and cathepsins K, L, S, and V). Using DIPPS, we were able to perform specificity profiling of thermolysin at its optimal temperature of 75&#xb0;C, which confirmed the applicability of the method to extreme experimental conditions. Moreover, DIPPS enabled the first global specificity profiling of legumain at pH as low as 4.0, which revealed a pH-dependent change in the specificity of this protease, further supporting its broad applicability.

### Keywords
Cathepsin, Degradomics, Protease, Legumain, Specificity profiling

### Affiliations
Department of Biochemistry and Molecular and Structural Bilogy, Jozef Stefan Institute, Ljubljana, Slovenia
Jozef Stefan Institute

### Submitter
Marko Fonovic

### Lab Head
Dr prof. dr. Marko Fonovic
Department of Biochemistry and Molecular and Structural Bilogy, Jozef Stefan Institute, Ljubljana, Slovenia


