### Accession
PXD031048

### Title
An efficient urine peptidomics workflow identifies chemically defined dietary gluten peptides from patients with celiac disease

### Description
This study sought to identify proteins derived from dietary wheat, rye, and barley in human urine samples. The urinary peptidomes of celiac disease patients and healthy controls were compared.

### Sample Protocol
Endogenous human peptides and dietary peptides from wheat, rye, and barley were enriched from human urine using centrifugal filters for protein depletion, followed by mixed cation exchange solid phase extraction for removal of salts and urinary metabolites. Samples wereanalyzed by data-dependent LC-MS/MS.

### Data Protocol
MS data were searched against FASTA databases containing human and wheat proteins using PEAKS software (Bioinformatics Solutions, Inc.). Carbamidomethylation (on Cys) was set as a fixed modification and deamidation (on Asn and Gln), oxidation (on Met), and N-terminal pyroglutamination (on Gln and Glu) were allowed as variable modifications, with a maximum of3 variable modifications per peptide. The cleavage type was set to "nonspecific" since endogenous peptides were analyzed. The allowed mass tolerances were 10 ppm for precursor ions and 0.5 Da or 0.02 Da for product ions.

### Publication Abstract
None

### Keywords
Urine, Celiac disease, Peptidomics, Wheat

### Affiliations
Stanford University, Brigham and Women's Hospital, Harvard Medical School
Chan Zuckerberg Biohub

### Submitter
Brad Palanski

### Lab Head
Dr Joshua Elias
Chan Zuckerberg Biohub


