### Accession
PXD003122

### Title
A molecular model of sea urchin reversible adhesion based on quantitative proteomics

### Description
Marine bioadhesives have unmatched performances especially in wet environments, being valuable sources of inspiration for industrial and biomedical applications. In sea urchins specialized adhesive organs, called tube feet, mediate adhesion. These are composed by a disc, which produces adhesive and de-adhesive secretions for strong reversible attachment, and a motile stem. After detachment, the secreted adhesive remains bound to the substratum as a footprint. Previous studies showed that sea urchin adhesive is composed of proteins and sugars, but so far only one protein, Nectin, was shown to be over-expressed as a transcript in tube feet discs, suggesting its involvement in sea urchin adhesion. Here we use high-resolution quantitative mass-spectrometry technologies to profile Paracentrotus lividus tube feet differential proteome, comparing protein expression levels in its adhesive part (disc) versus the non-adhesive part (stem). This allowed us to identify 163 highly over-expressed disc proteins and propose the first molecular model of sea urchin reversible adhesion. The secreted adhesive proteome was also analyzed, whereby we found that 70% of its components fall within five protein groups, involved in adhesive exocytosis and protection against microbes. Our data also provides evidence that Nectin is not only highly expressed in tube feet discs but is a component of the adhesive itself, thus constituting the first report of a sea urchin tube foot adhesive protein. These results give us an unprecedented insight on the molecular mechanics underlying sea urchins reversible adhesion, opening new doors to develop new, wet-reliable, reversible, efficient, ecological biomimetic adhesives.

### Sample Protocol
Protein extraction and digestion: Tube feet disc and stem proteins from Paracentrotus lividus (Lamark 1816) were extracted by automated frozen disruption methodology. in a Mikro-Dismembrator (Retsch) and set to 3000 rpm for 5 min. To avoid sample loss, the resulting powder was resuspended for 5 min with vigorous shaking in solubilization buffer (6M urea 50mM ammonium bicarbonate and 1:10000 protease inhibitors). The resulting homogenate was then centrifuged at 11500 rpm for 10 min and the obtained supernatant was divided in 3 technical replicates for the subsequent fractionation steps and stored at -20 ºC. The tube feet disc and stem protein extracts were submitted to reduction with DTT at a final concentration of 4 mM at 56 °C for 25 min; subsequently samples were alkylated with iodoacetamide at a final concentration of 8 mM at room temperature for 30 min in the dark. Proteins were then digested with Lys-c in 6 M urea and incubated for 4 h at 37 °C. The solution was then diluted to a final urea concentration of 2 M with 50 mM ammonium bicarbonate, and the solution was incubated at 37 °C overnight with trypsin. The digestion was quenched by acidification to 5% formic acid and peptides were desalted using Sep-Pak C18 cartridges, dried in vacuo, and stored at −80 °C for further use. The adhesive was washed five times with 1 ml milliQ H20 and between washes centrifuged 20 min at 13400 rpm at 4 °C. Then, it was suspended in 1.5 mL of a protein precipitating solution to eliminate the non-protein components (10% trichloroacetic acid, 0.07% β-mercaptoethanol (w/v)) for 1h30 at 4°C, and washed three times with 1.5 mL chilled 0.07% β-mercaptoethanol in acetone (v/v), centrifuging 20 min at 13400 rpm at 4 °C between washes. The obtained pellet was vacuum dried and solubilized in 50 μL of 2% sodium dodecyl sulfate, 0.5 M dithiothreitol in 63.2 mM Tris–HCl pH 6.8. The homogenized suspension was heated for 3 h at 60°C and 800rpm. After cooling, it was centrifuged at 13400 rpm for 20 min at RT. Adhesive protein extract was then loaded on a 12.5% SDS-PAGE gel and run for 10 min to purify the proteins from detergents and buffer components, followed by Coomassie blue R-250 staining. The obtained concentrated protein bands were in-gel digested with Lys-C and Trypsin according to standard procedure.  Protein separation by chromatography and analysis by mass spectrometry: The digests from tube feet discs and stems were fractioned using a strong cation exchange (SCX). The SCX system consisted of an Agilent 1200 HPLC system (Agilent Technologies, Waldbronn, Germany) with one C18 Opti-Lynx (Optimized Technologies, OR) trapping cartridges and a Zorbax BioSCX-Series II column (0.8-mm inner diameter, 50-mm length, 3.5 microms particle size). The peptides were dissolved in 10% formic acid (FA) and loaded onto the trap columns at 100 μl min−1 and subsequently eluted onto the SCX column with 80% acetonitrile (ACN) and 0.05% FA. SCX solvent A consists of 0.05% formic acid in 20% ACN, while solvent B was 0.05% formic acid, 0.5 M NaCl in 20% ACN. The SCX salt gradient is as follows: 0–0.01 min (0–2% B); 0.01–8.01 min (2–3% B); 8.01–14.01 min (3–8% B); 14.01–28 min (8–20% B); 28–38 min (20–40% B); 38–48 min (40–90% B); 48–54 min (90% B); 54–60 min (0% B). A total of 50 SCX fractions (1 min each, that is, 40 μl elution volume) were collected and the 25 most peptide-rich fractions were used for subsequent LC-MS/MS analysis.  Nanoscale liquid chromatography coupled to tandem mass spectrometry (LC-MS/MS) was performed for each SCX fractions on a reversed-phase easy nano-LC 1000 (Thermo Fisher Scientific, Odense, Denmark) coupled to an Orbitrap Q-exactive mass spectrometer (Thermo Scientific, Bremen, Germany) using higher-energy collisional dissociation (HCD) fragmentation. Briefly, peptides were loaded on a double-fritted trap column (100 µm inner diameter x 2 cm, packed with 5 μm C18 resin, ReproSil-Pur AQ; Dr. Maisch, Ammerbuch, Germany) at a flow rate of 5 µl/min in 100% buffer A (0.1% formic acid in HPLC grade water). Peptides were transferred to an analytical column (50 µm inner diameter x 50 cm, packed with 2.7 µm C18 particles, Poroshell 120 EC-C18; Agilent Technologies, Waldbronn, Germany) and separated using a 180 minutes gradient from 7 to 30% buffer B (0.1% formic acid in 100% acetonitrile) at a flow rate of 100 nl/min. For mass spectrometric analysis, the survey scans were acquired at 35,000 resolution to a scan range from 350 to 1500 m/z. The 20 most intense precursors were submitted to HCD fragmentation using an MS/MS resolution set to 17,500, a precursor automatic gain control (AGC) target set to 5 × 104, a precursor isolation width set to 1.5 Da, and a maximum injection time set to 120 ms.

### Data Protocol
Data processing: RAW output files were submitted to the Mascot (version 2.5.1) via Proteome Discoverer (version 1.3, Thermo Fisher Scientific) and searched against UniProt/SwissProt database for SEA URCHIN (Taxonomic Identifier: 7656; 10/2014 database containing 32238 entries: 285 reviewed (swissprot) + 32554 unreviewed (trembl)). The accession KT351732 was manually added to the previous database because although already deposited on NCBI it will be kept confidential until the publication of the manuscript referenced as (8). Enzyme specificity was set to trypsin, maximum missed cleavages = 2. Carbamidomethylcysteine was set as fixed and methionine oxidation were set as variable modifications.  The following peptide filters were set: minimal mascot ion score 20; Peptide length between 6 and 45 amino acids; Peptide confidence high; Search engine rank 1; Peptide rank 1 and a peptide false discovery rate (FDR) was set to 1%. Concerning the adhesive, in-gel digested proteins were analyzed using only the RP-LC-MS/MS. files.

### Publication Abstract
None

### Keywords
Reversible adhesion, Tube foot adhesive protein, Paracentrotus lividus, Quantitative proteomics, Echinoderm, Sea urchin

### Affiliations
Biomolecular Mass Spectrometry and Proteomics  Bijvoet Center for Biomolecular Reseach and Utrecht Institute of Pharmaceutical Sciences Utrecht University Padualaan 8, 3584 CH Utrecht, The Netherland
Utrecht University

### Submitter
Nicolas Lebesgue

### Lab Head
Dr Albert J.R Heck
Biomolecular Mass Spectrometry and Proteomics  Bijvoet Center for Biomolecular Reseach and Utrecht Institute of Pharmaceutical Sciences Utrecht University Padualaan 8, 3584 CH Utrecht, The Netherland


