### Accession
PXD029395

### Title
Jaltomata herrerae nectar proteins

### Description
The purpose of this project was to identify nectar proteins in Jaltomata herrerae

### Sample Protocol
Eighteen microliters of Jaltomata herrerae nectar was electrophoresed on a 4-20% Tricine gel under denaturing conditions and stained with PAGE-Blue (Thermo-Scientific Cat#24620). Protein bands were excised and submitted for identification at the University of Minnesotaâ€™s Center for Mass Spectrometry and Proteomics Facility. In-gel trypsin digestion and STAGE Tip peptide cleanup was performed as previously described (1) except that iodoacetamide was used as the alkylating reagent during the digest protocol.  The peptides were analyzed by capillary LC-MS/MS on an Orbitrap Velos system with data dependent acquisition (Orbi-Orbi mode with HCD activation) as described previously (2).  1. Thu YM, et al. (2016) Slx5/Slx8 promotes replication stress tolerance by facilitating mitotic progression. Cell reports 15(6):1254-1265. 2. Lin-Moshier Y, et al. (2013) Re-evaluation of the role of calcium homeostasis endoplasmic reticulum protein (CHERP) in cellular calcium signaling. J Biol Chem 288(1):355-367.

### Data Protocol
The PEAKS Studio 8.5 (3) (Bioinformatics Solutions, Waterloo, ON, CA) was used for database searching.  We formatted a custom protein sequence database from theoretical translations being derived from existing RNA-seq data (NCBI SRA accession numbers: SRX2676120, SRX2676119, and SRX2676134) and common lab contaminant proteins (from https://www.thegpm.org/crap/). Search parameters were: de novo precursor tolerance 20 ppm and fragment ion tolerance 0.1 Da; database search precursor mass tolerance 50.0 ppm; fragment mass error tolerance 0.1 Da; precursor mass search type monoisotopic; trypsin enzyme specificity; fixed modification cysteine carbamidomethylation; variable modifications methionine oxidation, pyroglutamic acid, deamidation of asparagine and glutamine, protein N-terminal acetylation; maximum variable modifications per peptide 3; false discovery rate calculation On; spectra merge off; charge state correction; spectral filter quality >0.65. We employed the SPIDER module for consideration of amino acid substitutions in the target protein sequence database.   3. Ma B, et al. (2003) PEAKS: powerful software for peptide de novo sequencing by tandem mass spectrometry. Rapid communications in mass spectrometry : RCM 17(20):2337-2342.

### Publication Abstract
None

### Keywords
Nectar, Jaltomata, Nectarin

### Affiliations
Department of Plant & Microbial Biology, University of Minnesota
University of Minnesota

### Submitter
Clay Carter

### Lab Head
Dr Clay J. Carter
Department of Plant & Microbial Biology, University of Minnesota


