### Accession
PXD000683

### Title
Genomics and Proteomics Provide New Insight into the Commensal and Pathogenic Lifestyles of Bovine and Human-associated Staphylococcus epidermidis

### Description
The goal of the present study is to understand mechanisms by which SE can infect the bovine host and persist in the mammary gland. For this purpose we determined the whole genome and four plasmid sequences of the SE strain PM221 originating from spontaneous persistent bovine IMI. We compared the genomic data with those of ATCC12228 and RP62A to screen for conserved and acquired gene pools. To confirm and complement these findings two-dimensional difference gel electrophoresis (2D DIGE) involving each three strains, surfome shaving and experimental infections in a bovine model was applied to indicate potential differences.

### Sample Protocol
Trypsin-shaving – LC-MS/MS analysis. PM221 and ATCC12228 were cultured under aerobic and microaerophilic conditions at 37°C. Bacteria were harvested by centrifugation. Enzymatic digestion was conducted by adding trypsin at 37°C for 15 min. Released peptides and trypsin was recovered by filtration and the recovered trypsin-digestions were incubated for 16 h at 37°C. Trypsin-digested peptides were purified using ZipTips. Peptides were identified using an Ultimate 3000 nano-LC (Dionex) and QSTAR Elite hybrid quadrupole TOF mass spectrometer (Applied Biosystems / MDS Sciex) with nano-ESI ionization. The samples were first concentrated and desalted on a C18 trap column (10 mm × 150 μm, 3 μm, 120 Å; PROTECOLTM, SGE Analytical Science, Griesheim, Germany) followed by peptide separation on a PepMap100 C18 analytical column (15 cm × 75 μm, 5 μm, 100 Å; LC Packings, Sunnyvale, CA) at 200 nl/min. The tryptic peptides were eluted in a linear gradient of acetonitrile from 0%  to 40% in 120 min (in 0.1% formic acid) at a flow rate of 200 nL• min-1. MS data were acquired automatically with Analyst QS 2.0 Software (Applied Biosystems, United States) using SMART IDA. The information-dependent acquisition method consisted of a 0.5-s TOF-MS survey scan of m/z 400–1400. From every survey scan, the two most abundant ions with charge states 2+ to 4+ were selected for product ion scans. Once an ion was selected for MS/MS fragmentation, it was put on an exclusion list for 60 s.

### Data Protocol
The MS/MS data were searched against the PM221 and ATCC12228 databases using Mascot and Paragon search engines through the ProteinPilotTM interface. The search criteria for Mascot searches were: trypsin digestion with one missed cleavage allowed, carbamidomethyl modification of cysteine as a fixed modification, and oxidation of methionine as variable modification. For the LC-MS/MS spectra, both the maximum precursor ion mass tolerance and MS/MS fragment ion mass tolerance were 0.2 Da, and a peptide charge state of +1, +2, +3 was used. The parameters for Paragon searches included the Rapid search mode and the carbamidomethyl modification of cysteine as a fixed and oxidation of methionine as a variable modification. The Compid tool was used to parse significant hits from both data searches. Protein identifications that had probability-based Mascot Mowse scores ≥ 50 and p < 0.05 and/or Paragon Unused ProtScores ≥ 1.3 and p < 0.05 were considered reliable high-confidence identifications. To estimate the false discovery rates (FDRs), all Mascot and Paragon searches were repeated using identical search parameters and validation criteria against PM221 and ATCC12228 decoy databases containing all protein sequences in both forward and reverse orientations. Sequences were reversed using the Compid tool and the FDR percentages were calculated using the formula 2xnreverse/(nreverse + nforward).

### Publication Abstract
The present study reports comparative genomics and proteomics of Staphylococcus epidermidis (SE) strains isolated from bovine intramammary infection (PM221) and human hosts (ATCC12228 and RP62A). Genome-level profiling and protein expression analyses revealed that the bovine strain and the mildly infectious ATCC12228 strain are highly similar. Their genomes share high sequence identity and synteny, and both were predicted to encode the commensal-associated fdr marker gene. In contrast, PM221 was judged to differ from the sepsis-associated virulent human RP62A strain on the basis of distinct protein expression patterns and overall lack of genome synteny. The 2D DIGE and phenotypic analyses suggest that PM221 and ATCC12228 coordinate the TCA cycle activity and the formation of small colony variants in a way that could result in increased viability. Pilot experimental infection studies indicated that although ATCC12228 was able to infect a bovine host, the PM221 strain caused more severe clinical signs. Further investigation revealed strain- and condition-specific differences among surface bound proteins with likely roles in adhesion, biofilm formation, and immunomodulatory functions. Thus, our findings revealed a close link between the bovine and commensal-type human strains and suggest that humans could act as a reservoir of bovine mastitis-causing SE strains.

### Keywords
Pathogenicity, Staphylococcus epidermidis, Imi, Proteome, Genome

### Affiliations
Institute of Biotechnology
Institute of Biotechnology, Proteomics Unit, University of Helsinki, Finland

### Submitter
Pia SiljamÃ¤ki

### Lab Head
Dr Tuula A Nyman
Institute of Biotechnology, Proteomics Unit, University of Helsinki, Finland


