### Accession
PXD016106

### Title
15-141 The endoplasmic reticulum-associated mRNA-binding protein ERBP1

### Description
Kinetoplastids rely heavily on post-transcriptional mechanisms for control of gene expression, and on RNA-binding proteins that regulate mRNA splicing, translation and decay. Trypanosoma brucei ERBP1 (Tb927.10.14150) and ERBP2 (Tb927.9.9550) were previously identified as mRNA binding proteins that lack canonical RNA-binding domains. We here show that ERBP1 is associated with the endoplasmic reticulum, like ERBP2, and that the two proteins interact in vivo. Loss of ERBP1 from bloodstream-form T. brucei initially resulted in a growth defect but proliferation was restored after more prolonged cultivation. Results from a pull-down of tagged ERBP1 suggest that it preferentially binds to ribosomal protein mRNAs. The ERBP1 sequence resembles that of Saccharomyces cerevisiae Bfr1, which also localises to the endoplasmic reticulum and binds to ribosomal protein mRNAs. However, unlike Bfr1, ERBP1 does not bind to mRNAs encoding secreted proteins, and it is also not recruited to stress granules after starvation.

### Sample Protocol
To purify TAP-ERBP1 for mass spectrometry, the protein was subjected to two steps of affinity purification. Briefly, the cleared lysate was incubated with IgG sepharose beads, washed, and then bound proteins were released using TEV protease. The resulting preparation was then allowed to adhere to a calmodulin affinity column, and proteins were eluted with EGTA. Co-purifying proteins from three independent experiments were analyzed by LC/MS by the ZMBH Mass Spectrometry facility. Cell lines expressing TAP-GFP served as control.

### Data Protocol
DATABASE SEARCHING-- Tandem mass spectra were extracted by [unknown] version [unknown]. Charge state deconvolution and deisotoping were not performed. All MS/MS samples were analyzed using Mascot (Matrix Science, London, UK; version 2.4.1). Mascot was set up to search the TriTrypDB-81TREU927_TriTrypDB-9.0_TbruceiTREU927_04022015 database (unknown version, 11567 entries) assuming the digestion enzyme trypsin. Mascot was searched with a fragment ion mass tolerance of 0,50 Da and a parent ion tolerance of 100 PPM. Carbamidomethyl of cysteine was specified in Mascot as a fixed modification. Deamidated of asparagine and glutamine and oxidation of methionine were specified in Mascot as variable modifications.   CRITERIA FOR PROTEIN IDENTIFICATION-- Scaffold (version Scaffold_4.8.4, Proteome Software Inc., Portland, OR) was used to validate MS/MS based peptide and protein identifications. Peptide identifications were accepted if they could be established at greater than 95,0% probability by the Peptide Prophet algorithm (Keller, A et al Anal. Chem. 2002;74(20):5383-92) with Scaffold delta-mass correction. Protein identifications were accepted if they could be established at greater than 95,0% probability and contained at least 2 identified peptides.  Protein probabilities were assigned by the Protein Prophet algorithm (Nesvizhskii, Al et al Anal. Chem. 2003;75(17):4646-58). Proteins that contained similar peptides and could not be differentiated based on MS/MS analysis alone were grouped to satisfy the principles of parsimony.

### Publication Abstract
None

### Keywords
15-141 rna-binding proteins; trypanosoma brucei; post-transcripcional control

### Affiliations
ZMBH, Im Neuenheimer Feld 282, 69122 Heidelberg
Centre for Molecular Biology at the university Heidelberg (ZMBH), Im Neuenheimer Feld 282; 69120 Heidelberg, Germany

### Submitter
Thomas Ruppert

### Lab Head
Dr Thomas Ruppert
Centre for Molecular Biology at the university Heidelberg (ZMBH), Im Neuenheimer Feld 282; 69120 Heidelberg, Germany


