### Accession
PXD017160

### Title
Plasma membrane proteomics

### Description
Aluminum (Al) toxicity in acid soil is a worldwide agricultural problem that inhibits crop growth and productivity. However, the signal pathways associated with Al tolerance in plants still remain largely unclear. In this study, tandem mass tag (TMT)-based quantitative proteomic methods was used to identify the expression changes of plasma membrane (PM) proteins in Tamba black soybean (TBS) root tips under Al stress. The results showed that a total of 907 PM proteins were identified, and 90 differentially expressed proteins (DEPs) were disclosed between Al-treated and untreated plants, with 46 up-regulated and 44 down-regulated (fold change > 1.3 or < 0.77, P < 0.05). Functional enrichment based on GO, KEGG and protein domain revealed that the DEPs were associated with membrane trafficking and transporters, modifying cell wall composition, defense response and signal transduction.

### Sample Protocol
Seeds of TBS were rinsed soaked in double distilled H2O for 30 min and then transferred to moist towel in the dark at 25 °C. After germination, uniform seedlings were transferred onto a floating mesh in opaque beaker with 1/2 strength hoagland nutrient solution. The nutrient solution was renewed every day. Plants were cultured in a tissue culture room at 27/22 °C day/night temperatures with 14 h of light (200 µmol photons m-2·s-1). When the true leaf was fully expanded, the seedlings were pre-treated with 0.5 mM CaCl2 (pH 4.3) for 24 h. Two-week-old seedlings were transferred into 0.5 mM CaCl2 solution containing AlCl3 at 0 (CK), 50 µM (Al treatment) at pH 4.3 for 3 d. After treatment, root apices (0–2 cm) were excised, immediately frozen in liquid nitrogen. The frozen root tissues were taken out from liquid nitrogen and quickly ground into fine powder. After adding ice-cold buffer H (330mM sucrose, 50 mM Na4P2O7, 25 mM NaF, 5% glycerol, 0.5% PVP, 10 mM EDTA, 1 mM Na2MoO4, 0.1% protease inhibitor cocktail, 3 mM DTT) (1 mL per g fresh weight of tissue), the samples were centrifuged for 10 min (10 000 × g at 4°C). Next, we collected the supernatant and centrifuged for 30 min (100 000  g at 4 °C). The obtained crude microsomal pellets were washed with buffer H without DTT, and incubated in buffer H with 0.02% Brij-58 but without DTT (2 μL per μg of crude microsomal protein on ice) for 45 min to dissolve. The PM-enriched protein fraction was obtained after centrifuged for 30 min (100 000 × g at 4°C). The protein concentration was measured by BCA kit (Beyotime Biotechnology, Beijing, China) according to the manufacturer’s instructions. The protein solution was reduced with 5 mM dithiothreitol for 30 min at 56 °C and alkylated with 11 mM iodoacetamide for 15 min at room temperature away from light. The protein sample was then diluted to urea concentration less than 2M. Finally, trypsin was added at 1:50 trypsin-to-protein mass ratio for the digestion overnight and 1:100 trypsin-to-protein mass ratio for a 4 h-digestion. After trypsin digestion, peptide was desalted by Strata X C18 (Phenomenex) and vacuum-dried. Peptide was reconstituted in 0.5 M TEAB and processed according to the manufacturer’s protocol for TMT kit. Briefly, TMT reagent were thawed and reconstituted in acetonitrile. The peptide mixtures were then incubated for 2h at room temperature, desalted and vacuum-dried. The tryptic peptides were fractionated into fractions by high pH reverse-phase HPLC using Agilent 300Extend C18 column. Briefly, peptides were separated with a gradient of 8% to 32% acetonitrile (pH 9.0) over 60 min into 60 fractions. Then, the peptides were combined into 18 fractions and vacuum-dried. The tryptic peptides were dissolved in 0.1% formic acid (solvent A), directly loaded onto a home-made reversed-phase analytical column (15-cm length, 75 μm i.d.). The gradient was comprised of an increase from 6% to 23% solvent B (0.1% formic acid in 98% acetonitrile) over 26 min, 23% to 35% in 8 min and climbing to 80% in 3 min then holding at 80% for the last 3 min, all at a constant flow rate of 400 nL/min on an EASY-nLC 1000 UPLC system. The peptides were subjected to NSI source followed by tandem mass spectrometry (MS/MS) in Q Exactive TM Plus (Thermo) coupled online to the UPLC. The electrospray voltage applied was 2.0 kV. The m/z scan range was 350 to 1800 for full scan, and intact peptides were detected in the Orbitrap at a resolution of 70,000. Peptides were then selected for MS/MS using NCE setting as 28 and the fragments were detected in the Orbitrap at a resolution of 17,500. A data-dependent procedure that alternated between one MS scan followed by 20 MS/MS scans with 15.0s dynamic exclusion. Automatic gain control (AGC) was set at 5E4. Fixed first mass was set as 100 m/z.

### Data Protocol
The resulting MS/MS data were processed using Maxquant search engine (v.1.5.2.8). Spectra search was performed against the Soybean genome sequence databases downloaded from the phytozome database (https://phytozome.jgi.doe.gov/pz/portal.html, containing 88,647 unigenes) concatenated with reverse decoy database. FDR of protein identification and PSM identification was set to 1%. To be considered diferentially expressed, proteins were required to exhibit a P value ≤ 0.05 calculated by the software. For protein abundance ratios measured using TMT, we considered a 1.3-fold change and a P value < 0.05 as the thresholds for identifying significant changes. Gene Ontology (GO) annotation proteome was derived from the UniProt-GOA database (www. http://www.ebi.ac.uk/GOA/). Firstly, Converting identified protein ID to UniProt ID and then mapping to GO IDs by protein ID. If some identified proteins were not annotated by UniProt-GOA database, the InterProScan soft would be used to annotated protein’s GO functional based on protein sequence alignment method. Then proteins were classified by Gene Ontology annotationbased on three categories: biological process, cellular component and molecular function. Identified proteins domain functional description were annotated by InterProScan (a sequence analysis application) based on protein sequence alignment method, and the InterPro domain database was used. InterPro (http://www.ebi.ac.uk/interpro/) is a database that integrates diverse information about protein families, domains and functional sites, and makes it freely available to the public via Web-based interfaces and services. Central to the database are diagnostic models, known as signatures, against which protein sequences can be searched to determine their potential function. InterPro has utility in the large-scale analysis of whole genomes and meta-genomes, as well as in characterizing individual protein sequences. Kyoto Encyclopedia of Genes and Genomes (KEGG)database was used to annotate protein pathway. Firstly, using KEGG online service tools KAAS to annotated protein’s KEGG database description. Then mapping the annotation result on the KEGG pathway database using KEGG online service tools KEGG mapper. We used wolfpsort a subcellular localization predication soft to predict subcellular localization. Wolfpsort an updated version of PSORT/PSORT II for the prediction of eukaryotic sequences. Proteins were classified by GO annotation into three categories: biological process, cellular compartment and molecular function. For each category, a two-tailed Fisher’s exact test was employed to test the enrichment of the differentially expressed protein against all identified proteins. The GO with a corrected p-value < 0.05 is considered significant. Encyclopedia of Genes and Genomes (KEGG) database was used to identify enriched pathways by a two-tailed Fisher’s exact test to test the enrichment of the differentially expressed protein against all identified proteins. The pathway with a corrected p-value < 0.05 was considered significant. These pathways were classified into hierarchical categories according to the KEGG website. For each category proteins, InterPro (a resource that provides functional analysis of protein sequences by classifying them into families and predicting the presence of domains and important sites) database was researched and a two-tailed Fisher’s exact test was employed to test the enrichment of the differentially expressed protein against all identified proteins.  Protein domains with a p-value < 0.05 were considered significant. For further hierarchical clustering based on different protein functional classification (such as: GO, Domain, Pathway, Complex). We first collated all the categories obtained after enrichment along with their P values, and then filtered for those categories which were at least enriched in one of the clusters with P value <0.05. This filtered P value matrix was transformed by the function x = −log10 (P value). Finally these x values were z-transformed for each functional category. These z scores were then clustered by one-way hierarchical clustering (Euclidean distance, average linkage clustering) in Genesis. Cluster membership were visualized by a heat map using the “heatmap.2” function from the “gplots” R-package.

### Publication Abstract
Aluminum (Al) toxicity in acid soil is a worldwide agricultural problem that inhibits crop growth and productivity. However, the signal pathways associated with Al tolerance in plants remain largely unclear. In this study, tandem mass tag (TMT)-based quantitative proteomic methods were used to identify the differentially expressed plasma membrane (PM) proteins in Tamba black soybean (TBS) root tips under Al stress. Data are available via ProteomeXchange with identifier PXD017160. In addition, parallel reaction monitoring (PRM) was used to verify the protein quantitative data. The results showed that 907 PM proteins were identified in Al-treated plants. Among them, compared to untreated plants, 90 proteins were differentially expressed (DEPs) with 46 up-regulated and 44 down-regulated (fold change &gt; 1.3 or &lt; 0.77, <i>p</i> &lt; 0.05). Functional enrichment based on GO, KEGG and protein domain revealed that the DEPs were associated with membrane trafficking and transporters, modifying cell wall composition, defense response and signal transduction. In conclusion, our results highlight the involvement of GmMATE13, GmMATE75, GmMATE87 and H<sup>+</sup>-ATPase in Al-induced citrate secretion in PM of TBS roots, and ABC transporters and Ca<sup>2+</sup> have been implicated in internal detoxification and signaling of Al, respectively. Importantly, our data provides six receptor-like protein kinases (RLKs) as candidate proteins for further investigating Al signal transmembrane mechanisms.

### Keywords
Aluminum resistance, Tbs, Plasma membrane, Proteomics, Tmt, Prm

### Affiliations
Southwest University

### Submitter
yunmin wei

### Lab Head
Dr Yongxiong Yu
Southwest University


