### Accession
PXD003887

### Title
Black pepper proteome -  Transcriptome- Assisted Label-Free Quantitative Proteomics Analysis Reveals Novel Insights into Piper nigrum-Phytophthora capsici Phytopathosystem

### Description
In this study, we have evaluated the proteomic changes that occur in Piper nigrum L.(black pepper) after infection by the pathogen Phytophthora capsici. We report novel leaf proteins from black pepper identified by an integrated transcriptome-assisted label-free quantitative proteomics pipeline. Several previously described methods were used to create this data set. Detached leaves were inoculated with either mock treatment, or the oomycete pathogen and small tissue samples only around the site of inoculation were collected for protein sample preparations. In order to quantify protein abundance in the samples being compared, we used a label free method of spiking samples with a known ratio of pre-digested peptide samples to normalize endogenous protein abundance in the MS detection. Our study attempts to explain the basal immune components of black pepper when challenged with P. capsici.

### Sample Protocol
Total protin isolated from leaf samples using TCA/Acetone precipitation, solubilized using Urea/Thiourea followed by a sequential buffer exchange using 3KDa spin filtration units using Ammonium bicarbonate buffer. For  trypsin digestion, an aliquot of 100ug of proteni were used, spiked with MassPREP peptide standards and were further analyzed using nanoUPLC-MS acquisition.

### Data Protocol
The enhanced MS spectra were analyzed using PLGS 2.5.3 for protein identification and label-free relative protein quantification. A custom generated transcriptome database was used for idenfication of proteins with a false positive rate set to 4%. Label-free quantification were carried using PLGS auto normalization function were proteins were identified with a confidence of >95%. Yeast alcohol dehydrogenase was used as internal standard for normalization during relative quantification. All the protein sequences corresponding to protein hits identified by MS/MS analysis were manually retrieved in FASTA format from the custom black pepper protein database, annotated using BLAST2GO pipeline and were further analyzed manually.

### Publication Abstract
Black pepper (Piper nigrum L.), a tropical spice crop of global acclaim, is susceptible to Phytophthora capsici, an oomycete pathogen which causes the highly destructive foot rot disease. A systematic understanding of this phytopathosystem has not been possible owing to lack of genome or proteome information. In this study, we explain an integrated transcriptome-assisted label-free quantitative proteomics pipeline to study the basal immune components of black pepper when challenged with P. capsici. We report a global identification of 532 novel leaf proteins from black pepper, of which 518 proteins were functionally annotated using BLAST2GO tool. A label-free quantitation of the protein datasets revealed 194 proteins common to diseased and control protein datasets of which 22 proteins showed significant up-regulation and 134 showed significant down-regulation. Ninety-three proteins were identified exclusively on P. capsici infected leaf tissues and 245 were expressed only in mock (control) infected samples. In-depth analysis of our data gives novel insights into the regulatory pathways of black pepper which are compromised during the infection. Differential down-regulation was observed in a number of critical pathways like carbon fixation in photosynthetic organism, cyano-amino acid metabolism, fructose, and mannose metabolism, glutathione metabolism, and phenylpropanoid biosynthesis. The proteomics results were validated with real-time qRT-PCR analysis. We were also able to identify the complete coding sequences for all the proteins of which few selected genes were cloned and sequence characterized for further confirmation. Our study is the first report of a quantitative proteomics dataset in black pepper which provides convincing evidence on the effectiveness of a transcriptome-based label-free proteomics approach for elucidating the host response to biotic stress in a non-model spice crop like P. nigrum, for which genome information is unavailable. Our dataset will serve as a useful resource for future studies in this plant. Data are available via ProteomeXchange with identifier PXD003887.

### Keywords
Label-free quantitative proteomics, Piper nigrum, Plgs

### Affiliations
University of Oxford
Rajiv Gandhi Center for Biotechnology, Thiruvanthapuram, India

### Submitter
Chidambareswaren Mahadevan

### Lab Head
Dr Manjula Sakuntala
Rajiv Gandhi Center for Biotechnology, Thiruvanthapuram, India


