### Accession
PXD020357

### Title
Kinome Profiling of Primary Endometrial Tumors Using Multiplexed Inhibitor Beads and Mass Spectrometry Identifies SRPK1 As A Candidate Therapeutic Target

### Description
Protein kinases (collectively termed the kinome) represent one of the most tractable drug targets in the pursuit of new and effective cancer treatments. However, less than 20% of the kinome is currently being explored as primary targets for cancer therapy, leaving the majority of the kinome untargeted for drug therapy.  Chemical proteomics approaches such as Multiplexed Inhibitor Beads and Mass Spectrometry (MIB-MS) have been developed that measure the abundance of a significant proportion of the kinome, providing a strategy to interrogate kinome landscapes and dynamics.  Kinome profiling of cancer cell lines using MIB-MS has been extensively characterized, however, the application of this method to measure tissue kinome(s) has not been thoroughly explored.  Here, we present a quantitative proteomics workflow specifically designed for kinome profiling of tissues that pairs MIB-MS with a newly designed s-SILAC kinome standard.  Using this workflow, we mapped the kinome landscape of endometrial carcinoma (EC) tumors and normal endometrial (NE) tissues and identified several kinases overexpressed in EC tumors, including Serine/Arginine-Rich Splicing Factor kinase, (SRPK1).  Immunohistochemical (IHC) analysis of EC tumor TMAs confirmed MIB-MS findings and showed SRPK1 protein levels were highly expressed in endometrioid and uterine serous cancer (USC) histological subtypes.  Querying large-scale genomics studies of EC tumors revealed high expression of SRPK1 correlated with poor survival.  Inhibition of SRPK1 in USC cells altered mRNA splicing, downregulating several oncogenes including MYC and Survivin resulting in apoptosis. Taken together, we present a SILAC-based MIB-MS kinome profiling platform for measuring kinase abundance in tumor tissues, and demonstrate its application to identify SRPK1 as a plausible kinase drug target for the treatment of EC.

### Sample Protocol
MIB-MS Cells or tumors were lysed on ice in buffer containing 50 mM HEPES (pH 7.5), 0.5% Triton X-100, 150 mM NaCl, 1 mM EDTA, 1 mM EGTA, 10 mM sodium fluoride, 2.5 mM sodium orthovanadate, 1X protease inhibitor cocktail (Roche), and 1% each of phosphatase inhibitor cocktails 2 and 3 (Sigma). Particulate was removed by centrifugation of lysates at 13,000 rpm for 10 minutes at 4Â°C and filtration through 0.45 Âµm syringe filters. Protein concentrations were determined by BCA analysis.  An equal amount of the s-SILAC reference (5 mg) lysate was added to our non-labeled (5 mg) sample (cell, or tumor tissue) and analyzed on MIB-beads. For SPEC2 cell Label Free Quantitation experiments, 5 mg triplicates of control vs treatment were used. Endogenous kinases were isolated by flowing lysates over kinase inhibitor-conjugated Sepharose beads (purvalanol B, VI16832, PP58 and CTx-0294885 beads) in 10 ml gravity-flow columns. After 2x10 ml column washes in high-salt buffer and 1x10 ml wash in low-salt buffer (containing 50 mM HEPES (pH 7.5), 0.5% Triton X-100, 1 mM EDTA, 1 mM EGTA, and 10 mM sodium fluoride, and 1M NaCl or 150 mM NaCl, respectively), retained kinases were eluted from the column by boiling in 2x500 Âµl of 0.5% SDS, 0.1 M TrisHCl (pH 6.8), and 1% 2-mercaptoethanol. Eluted peptides were reduced by incubation with 5 mM DTT at 65Â°C for 25 minutes, alkylated with 20 mM iodoacetamide at room temperature for 30 minutes in the dark, and alkylation was quenched with DTT for 10 minutes. Samples were concentrated to approximately 100 Âµl with Millipore 10kD cutoff spin concentrators. Detergent was removed by chloroform/methanol extraction, and the protein pellet was resuspended in 50 mM ammonium bicarbonate and digested with sequencing-grade modified trypsin (Promega) overnight at 37Â°C. Peptides were cleaned with PepClean C18 spin columns (Thermo), dried in a speed-vac, resuspended in 50 Î¼l 0.1% formic acid, and extracted  with ethyl acetate (10:1 ethyl acetate:H2O). Briefly, 1 mL ethyl acetate was added to each sample, vortexed and centrifuged at max speed for 5 minutes, then removed. This process is repeated 4 more times. After removal of ethyl acetate following the 5th centrifugation, samples were placed at 60Â°C for 10 minutes to evaporate residual ethyl acetate. The peptides were dried in a speed vac, and subsequent LC-/MS/MS analysis was performed. Proteolytic peptides were resuspended in 0.1% formic acid and separated with a Thermo RSLC Ultimate 3000 on a Thermo Easy-Spray C18 PepMap 75Âµm x 50cm C-18 2 Âµm column with a 240 min gradient of 4-25% acetonitrile with 0.1% formic acid at 300 nL/min at 50Â°C. Eluted peptides were analyzed by a Thermo Q Exactive mass spectrometer utilizing a top 15 methodology in which the 15 most intense peptide precursor ions were subjected to fragmentation. The AGC for MS1 was set to 3x106 with a max injection time of 120 ms, the AGC for MS2 ions was set to 1x105 with a max injection time of 150 ms, and the dynamic exclusion was set to 90 s. Total Proteome SPEC2 cell lysates for total proteome analysis were processed as described in Coscia et al, 2016 (PMC5007461). Cells were harvested in a buffer containing 4% SDS and 10mM HEPES (pH 8), placed at 99Â°C for 10 min then homogenized. Lysates (100Âµg protein) were reduced with 10 mMDTT for 30 minutes at 65Â°C, then alkylated with 55 mM iodoacetamide at room temperature for 30 minutes in the dark. Following reduction and alkylation, proteins were precipitated with 80% acetone overnight at -20Â°C. Samples were centrifuged and the subsequent protein pellet was washed with 80% acetone then resuspended in 100 Î¼l 6M urea+2M thiourea in 10mM HEPES (pH 8) and digested with LysC (1Î¼g) for 3h at room temperature. Following LysC digestion, 400 Î¼l ammonium bicarbonate (50 mM) was added and samples were digested overnight with trypsin (1Î¼g) at 37Â°C. The following day, peptides were acidified with 0.1% formic acid and cleaned by subsequent elution from C-18 (Phenominex) and Hypercarb/Hypersep PGC columns (ThermoFisher Scientific) and dried via speed-vac. Dried peptides were resuspended in  50 Î¼l 0.1% formic acid, and extracted  with ethyl acetate as described in the MIB processing section. Peptides were resuspended in 0.1% formic acid and separated with a Thermo RSLC Ultimate 3000 on a Thermo Easy-Spray C18 PepMap 75Âµm x 50cm C-18 2 Âµm column with a 305 min gradient of 2-20% (180 min) 20%-28% (45 min) 28%-48% (20 min) acetonitrile with 0.1% formic acid at 300 nL/min at 50Â°C. Eluted peptides were analyzed by a Thermo Q Exactive mass spectrometer utilizing a top 15 methodology in which the 15 most intense peptide precursor ions were subjected to fragmentation. The AGC for MS1 was set to 3x106 with a max injection time of 120 ms, the AGC for MS2 ions was set to 1x105 with a max injection time of 150 ms, and the dynamic exclusion was set to 90 s.

### Data Protocol
MIB MS Raw data analysis of SILAC and LFQ experiments was performed using MaxQuant software 1.6.1.0 and searched using andromeda 1.5.6.0 against the swiss-prot human protein database (downloaded on April 24th, 2019, 20402 entries). The search was set up for full tryptic peptides with a maximum of two missed cleavage sites. All settings were default and searched using acetylation of protein N-terminus and oxidized methionine as variable modifications. Carbamidomethylation of cysteine was set as fixed modification. The precursor mass tolerance threshold was set at 10 ppm and maximum fragment mass error was 0.02 Da. SILAC quantification was performed using MaxQuant by choosing multiplicity as 2 in group-specific parameters and Arg10 and Lys8 as heavy labels. Parameters for protein quantitation were as follows: Label min ratio count=1, peptides used for quantition= unique, only use modified proteins was selected, Discard unmodified counterpart was selected and normalized average ratio estimation selected. The MQ output file was filtered for human protein kinases (n=537) in excel and kinase ratio normalized H/L intensities were imported into Perseus (1.6.2.3) for analysis. Normalized H/L s-SILAC ratios were 1/(x) transformed and filtered in the following manner:  kinases identified by site only were removed, reverse, or potential contaminant were removed and filtered for kinases identified by >1 unique peptide. Kinase s-SILAC ratios were log2 transformed and filtered for mininum valid number of 3. Filtered s-SILAC ratios were annotated and subjected to a Student's T-test using Permutation-based FDR <0.05. LFQ quantification was performed as follows: LFQ min ratio count was set to 2, Fast LFQ was selected, LFQ min number of neighbors was set to 3, LFQ average number of neighbors was set to 6, Skip normalization was not selected and match-between runs was selected. The MQ output protein LFQ intensities were imported into Perseus (1.6.2.3) for analysis. Protein LFQ values were filtered in the following manner: proteins identified by site only were removed, reverse, or potential contaminant were removed and filtered for proteins identified by >1 unique peptide. LFQ data was log2 transformed, filtered for min valid number of 3. Filtered LFQ kinase values were annotated and subjected to a Student's T-test using Permutation-based FDR <0.05. Total Proteome  Raw data analysis of SILAC experiments was performed using Maxquant software 1.6.1.0 and searched using andromeda 1.5.6.0 against the swiss-prot human protein database (downloaded on 2019, 20402 entries). The search was set up for full tryptic peptides with a maximum of two missed cleavage sites. All settings were default and searched using acetylation of protein N-terminus, oxidized methionine, and phosphorylation of serine, threonine, and tyrosine as variable modifications with a maximum number of modifications per peptide being 5. Carbamidomethylation of cysteine was set as fixed modification. The precursor mass tolerance threshold was set at 10 ppm and maximum fragment mass error was 0.02 Da. The significance threshold of the ion score was calculated based on a false discovery rate of < 1%. LFQ min ratio count was set to 2, Fast LFQ was selected, LFQ min number of neighbors was set to 3, LFQ average number of neighbors was set to 6, Skip normalization was not selected and match-between runs was selected. The MQ output protein LFQ intensities were imported into Perseus (1.6.2.3) for analysis. Protein LFQ values were filtered in the following manner:  proteins identified by site only were removed, reverse, or potential contaminant were removed and filtered for proteins identified by >1 unique peptide. LFQ data was log2 transformed, filtered for min valid number of 3, and normalized using Z-scores, annotated and subjected to a Student's T-test with comparing SPHINX31 vs DMSO in serum-competent SPEC-2 cells. Parameters for the Student's T-test were the following: S0=0.1, side both using Permutation-based FDR <0.05.

### Publication Abstract
Endometrial carcinoma (EC) is the most common gynecologic malignancy in the United States, with limited effective targeted therapies. Endometrial tumors exhibit frequent alterations in protein kinases, yet only a small fraction of the kinome has been therapeutically explored. To identify kinase therapeutic avenues for EC, we profiled the kinome of endometrial tumors and normal endometrial tissues using Multiplexed Inhibitor Beads and Mass Spectrometry (MIB-MS). Our proteomics analysis identified a network of kinases overexpressed in tumors, including Serine/Arginine-Rich Splicing Factor Kinase 1 (SRPK1). Immunohistochemical (IHC) analysis of endometrial tumors confirmed MIB-MS findings and showed SRPK1 protein levels were highly expressed in endometrioid and uterine serous cancer (USC) histological subtypes. Moreover, querying large-scale genomics studies of EC tumors revealed high expression of SRPK1 correlated with poor survival. Loss-of-function studies targeting SRPK1 in an established USC cell line demonstrated SRPK1 was integral for RNA splicing, as well as cell cycle progression and survival under nutrient deficient conditions. Profiling of USC cells identified a compensatory response to SRPK1 inhibition that involved EGFR and the up-regulation of IGF1R and downstream AKT signaling. Co-targeting SRPK1 and EGFR or IGF1R synergistically enhanced growth inhibition in serous and endometrioid cell lines, representing a promising combination therapy for EC.

### Keywords
Srpk1, Endometrial

### Affiliations
Cancer Biology Department, Fox Chase Cancer Center, Philadelphia, PA, USA
Fox Chase Cancer Center

### Submitter
James Duncan

### Lab Head
Dr James Stuart Duncan
Cancer Biology Department, Fox Chase Cancer Center, Philadelphia, PA, USA


