### Accession
PXD028595

### Title
Discovery of kinetochore and spindle assembly checkpoint proteins in apicomplexan parasites

### Description
Kinetochores are multiprotein assemblies that bind chromosomes to the spindle during mitosis. Despite conserved roles, repertoires of kinetochore proteins vary greatly across eukaryotes. In particular, most known components are not clearly detected in a group of parasites known as the Apicomplexa. Furthermore, flexibility in scale of amplification and an apparent incapability to delay cell cycle progression in response to spindle integrity has coined an idea these parasites divide in the absence of spindle checkpoints. In this study, we reunite divergent apicomplexan kinetochore components to a common eukaryotic set and additionally identify 9 kinetochore proteins that share little homology to known proteins. AKiTs are essential for chromosome segregation and show modes of division parallel to eukaryotic metaphase to anaphase transition and spindle assembly checkpoint signaling. These findings suggest conserved spindle assembly checkpoint signaling maintains fidelity during chromosome segregation in apicomplexan parasites.

### Sample Protocol
Sample preparation: Beads were re-suspended in 100 μl of 6M urea in 50 mM ammonium bicarbonate (AB). 2 μl of Dithioerythritol (DTE) 50 mM in distilled water were added and the reduction was carried out at 37°C for 1h. Alkylation was performed by adding 2 μl of iodoacetamide (400 mM in distilled water) during 1 hour at room temperature in the dark. Urea concentration was reduced to 1M by addition of 500 μl of AB and overnight digestion was performed at 37 °C with 5 μL of freshly prepared trypsin (Promega) at 0.1 μg/μl in AB. Supernatant was collected and completely dried under speed-vacuum. Samples were then desalted with a C18 microspin column (Harvard Apparatus, Holliston, MA, USA) according to manufacturer’s instructions, completely dried under speed-vacuum and stored at -20°C. ESI-LC-MSMS: Samples were diluted in 20 μl of loading buffer (5% CH3CN, 0.1% FA) and 2 μl was injected on column. LC-ESI-MS/MS was performed on a Q-Exactive HF Hybrid Quadrupole-Orbitrap Mass Spectrometer (Thermo Scientific) equipped with an Easy nLC 1000 liquid chromatography system (Thermo Scientific). Peptides were trapped on a Acclaim pepmap100, C18, 3μm, 75μm x 20mm nano trap-column (Thermo Scientific) and separated on a 75 μm x 250 mm, C18, 2μm, 100 Å Easy-Spray column (Thermo Scientific). The analytical separation was run for 90 min using a gradient of H2O/FA 99.9%/0.1% (solvent A) and CH3CN/FA 99.9%/0.1% (solvent B). The gradient was run as follows: 0-5 min 95 % A and 5 % B, then to 65 % A and 35 % B for 60 min, then to 10 % A and 90 % B for 10 min and finally stay 15 min at 10 % A and 90 % B. Flow rate was of 250 nL/min for a total run time of 90 min. Full scan resolution was set to 60’000 at m/z 200 with an AGC target of 3 x 106 and a maximum injection time of 60 ms. Mass range was set to 400-2000 m/z. For data dependant analysis, up to twenty precursor ions were isolated and fragmented by higher-energy collisional dissociation (HCD) at 27% NCE. Resolution for MS2 scans was set to 15’000 at m/z 200 with an AGC target of 1 x 105 and a maximum injection time of 60 ms. Isolation width was set at 1.6 m/z. Full MS scans were acquired in profile mode whereas MS2 scans were acquired in centroid mode. Dynamic exclusion was set to 20s.

### Data Protocol
Database search: Peaklists (MGF file format) were generated from raw data using the MS Convert conversion tool from ProteoWizard. The peaklist files were searched against the Plasmodium bergheii ANKA database (PlasmoDB.org, release 48, 5076 entries) combined with an in-house database of common contaminant using Mascot (Matrix Science, London, UK; version 2.5.1). Trypsin was selected as the enzyme, with one potential missed cleavage. Precursor ion tolerance was set to 10 ppm and fragment ion tolerance to 0.02 Da. Variable amino acid modifications were oxidized methionine and deaminated (NQ) as well as phosphorylated serine, threonine and tyrosine. Fixed amino acid modification was carbamidomethyl cysteine. The Mascot search was validated using Scaffold 4.11.1 (Proteome Software).

### Publication Abstract
Kinetochores are multiprotein assemblies directing mitotic spindle attachment and chromosome segregation. In apicomplexan parasites, most known kinetochore components and associated regulators are apparently missing, suggesting a minimal structure with limited control over chromosome segregation. In this study, we use interactomics combined with deep homology searches to identify 13 previously unknown components of kinetochores in Apicomplexa. Apicomplexan kinetochores are highly divergent in sequence and composition from animal and fungal models. The nanoscale organization includes at least four discrete compartments, each displaying different biochemical interactions, subkinetochore localizations and evolutionary rates across the phylum. We reveal alignment of kinetochores at the metaphase plate in both Plasmodium berghei and Toxoplasma gondii, suggestive of a conserved "hold signal" that prevents precocious entry into anaphase. Finally, we show unexpected plasticity in kinetochore composition and segregation between apicomplexan lifecycle stages, suggestive of diverse requirements to maintain fidelity of chromosome segregation across parasite modes of division.

### Keywords
Immunoprecipitation, Lc-msms, Apicomplexan parasite, Spindle assembly

### Affiliations
Department of Microbiology and Molecular Medicine, Faculty of Medicine, University of Geneva, CH-1211 Geneva, Switzerland.
University of Geneva

### Submitter
alexandre Hainard

### Lab Head
Dr Mathieu Brochet
Department of Microbiology and Molecular Medicine, Faculty of Medicine, University of Geneva, CH-1211 Geneva, Switzerland.


