### Accession
PXD040720

### Title
Testing TMT-SPS-MS3 and label free approach in recalcitrant Mago (Mangifera indica L.) peel tissues

### Description
Significant progress has been observed in recent years in proteomics tools. However, minimal application of the newest proteomic tools has been massively applied in plant recalcitrant tissues in non-model fruits. This constant occurrence constrains the progress in understanding essential molecular processes during postharvest shelf life (PSL) of fleshy fruits. In fact, comparative proteomic studies in fruit like mango, guava, papaya, and avocado are minimal. Therefore, we set up a label free quantitation (LFQ) and TMT-SPS-MS3 pipeline for comparative study of mango peels during PSL. Our protocol included different protein extraction and peptide fractionation before mass spectrometry analysis. In doing so we were able to confidently identify 4,400 proteins with high confidence, among which 1087 were differentially accumulated. The combination of phenolic protein extraction, high-pH reverse-phase peptide fractionation and LFQ exhibited the best approach for relative quantification, but other protein extraction, peptide fractionation and data acquisition showed significant complementation. LFQ and Byonic also allowed the determination of intact N-glyco peptides on proteins such as the desiccation-related protein, which molecular dynamics analysis support the occurrence of N-glycans providing stability of the protein. Our study represents the first massive proteomics comparison of mango peels during PSL.

### Sample Protocol
Phenol-based protein extraction was based on the protocol described by  Monribot-Villanueva et al., [11]. Three g of pooled tissue from peel from five fruits were ground to a fine powder with a mortar and pestle in liquid nitrogen adding 1:10 w/w of polyvinylpolypyrrolidone (PVPP): fresh weight (FW) ratio. The powder was suspended in three volumes of ice-cold extraction buffer (w/v) [(150 mM Tris-HCl, pH 8.0, 100 mM KCl, 1.4 M sucrose, 1% Triton X-100, 1% (w/v) dithiothreitol (DTT), 7.5 Î¼l Î²-Mercaptoethanol per 15 mL of buffer, 150 Î¼l 100 mM phenylmethylsulfonyl fluoride (PMSF) per 15 mL of buffer], and 100 Î¼L g-1 FW of protease inhibitor cocktail (Sigma-aldrichÂ®, Cat No P9599). The samples were mixed, and an equal volume of phenol solution was added and then be mixed. The samples were shaken in ice (4 Â°C) for 30 min. and then centrifuged at 10,000 x g for 30 min at 4 Â°C. The upper phenolic phase was collected and precipitated overnight at -20 Â°C with cold acetone (with 0.07% Î²-Mercaptoethanol) until a total 10 mL volume. Following centrifugation at 3000 x g for 30 min at 4 â—¦C, the protein pellets were washed twice with fresh cold acetone. After drying under a laboratory fume hood for 5 min, the pellets were dissolved in 300 ÂµL 50 mM phosphate buffer saline (PBS, pH 7.4) with 1% of SDS. The supernatant was recovered to be centrifuged at 15,000 x g for 10 min at 24 Â° C, the protein sample was recovered and stored at -80 Â°C. The protein concentration was determined using a BCA protein assay kit (Pierce, Thermo Scientific, Cat No. 23225). Electrophoresis under denaturing conditions (SDS-PAGE) was performed according to the manufacturerâ€™s instructions (Bio-Rad, Cat No) and stained with Sypro Ruby solution Chloroform/methanol -based protein extraction Chloroform/methanol -based protein extraction was based on the protocol described by Vertommen et al., 2010 [16]. The samples were taken at the same stages as in the phenol extraction. The powder was suspended in 6 mL of buffer PBS 1X, SDS 0.1%(w/v). The samples were mixed and then centrifuged at 3,000 xg for 30 min at 4 Â°C and the liquid phase transferred into 2 mL centrifuge tubes, 750 ul each one. it was added 600 ul of Chloroform and 600 ul of methanol to each tube. Following centrifugation at 16000 xg for 5 min at 25 â—¦C, three phases were formed. The upper and down phase formed after centrifugation were removed and the interphase was dried under a laboratory fume hood. The dried interphases were resuspended in a total 400 Î¼L of phosphate-buffered saline (PBS) 1x (Sigma) with SDS (1%). These were vortex and then centrifuged at 10,000 x g for 15 min at 24 â—¦C and the supernatant was transferred to a new tube. The protein samples were stored at -80 Â° C. The protein concentration was determined using a BCA protein assay kit (Pierce, Thermo Scientific). Electrophoresis under denaturing conditions (SDS-PAGE) was performed according to the manufacturerâ€™s instructions (Bio-Rad) and stained with Sypro Ruby solution. Tandem Mass tagging  After protein digestion one half of the samples were used for peptide tagging where Tandem Mass Tag (TMT) 6-plex reagents (Thermo Scientificâ„¢, Cat No. 90064B) were used according to the manufacturerâ€™s instructions in the following order: 126, 127 N, and 128 C for tissues connected with samples after harvesting (T0) and 129 N, 130C, and 131 for samples after six days of PSL (T6). Reduction of technical variation in sample processing for label free quantification  We minimized technical variation during sample preparation prior to label-free quantitative analysis according to Scheerlinck, et al., [17]. In doing so, a second half of digests samples relate to T0 and T6 were suspended in 100 Î¼L of solution A containing 50 mM triethylammonium bicarbonate (TEAB) and 1% Sodium deoxycholate (SDC). Thereafter, the mixed was sonicated and then precipitated with 100 Î¼L of 2% Trifluoroacetic acid (TFA) by incubating for 30 min in ice and centrifugated at 10,000 xg for 15 min at 4 â—¦C.  Then, the supernatant was transferred to a new tube and precipitated three times with a lower concentration of TFA (0.5%). The final supernatant was recovered and dried off with CentriVap vacuum concentrator (Labconco) for further peptide fractionation. Prefractionation and desalting Were fractionation by Strong Cation Exchange (Cartridges HyperSepâ„¢ SCX Cat No. 60108-421) or high-pH reversed-phase liquid chromatography spin columns (Pierceâ„¢ High pH Reversed-Phase Cat No. 84868, HPRP).  For SCX we obtained five final fractions associated with different concentration of eluding buffer (75, 100, 250 and 500 Î¼M KCl) while for HPRP we obtained four final fractions associated with different concentration of eluding acetonitrile buffer (12. 5, 15, 17.5 and 20 %). All fractions were desalted with ZipTip-Î¼C18 tips (Merck Millipore, Darmstadt, Germany) and dried using a CentriVap vacuum concentrator (Labconco).

### Data Protocol
For TMT The spectra generated with decision tree-driven MS/MS and (SPS)- MS3 were processed with Proteome Discoverer 2.4.1 (PD, Thermo Fisher Scientific) using the search engines Mascot version 2.4.1, SEQUEST HT [18] and MS Amanda [19]. Our proteomics strategy included isobaric labeling with the tandem mass tag system (TMT) and the SPS-MS3, where MS2(CID) was used for peptide identification and simultaneously, MS3 (HCD) was used to acquire TMT reporter ions used for quantification. The searches were directed against predicted proteins from a previously reported Mangifera indica transcriptome and genome [14]. Search parameters included: full-tryptic protease specificity, allowing two missed cleavages, static modifications covered carbamidomethylation of cysteine (+57.021 Da), and TMT 6-plex N-terminal/lysine residues (+229.163 Da). Dynamic modifications included methionine oxidation (+15.995 Da) and deamidation of asparagine/glutamine (+0.984 Da). The resulting peptide hits were filtered for a maximum of 1% FDR using the Percolator algorithm [20]. For label-free, the spectra generated with decision tree-driven MS/MS were processed with Proteome Discoverer 2.4.1 (PD, Thermo Fisher Scientific) the same search engines, dynamic and static modifications mention above and FDR. For comparative analysis we used the Minora node with the following parameters: In the peak feature detection was set a minimum trace length of five and maximum âˆ†RT of isotope pattern multiplets (min) of 0.2, the PSM confidence was set at least High for the feature to ID linking, precursor abundances was based on intensity, and the normalization and scaling was according to the total peptide amount.   For accurate protein identification and quantification, we consider in all cases at least two peptides, a protein coverage equal or more than 10 %, and coefficient of variation more than 30 %.  Intact N-glycopeptides determination   Raw data connected to LFQ and TMT-SPS-MS3 were subjected to ByonicTM platform v3.6.0 (Protein Metrix Inc). To scrutinized LFQ we included full-tryptic protease specificity static modifications to covered carbamidomethylation of cysteine (+57.021 Da) and methionine oxidation (+15.995 Da) as dynamic modification. Precursor mass tolerance was set at 40 ppm, CID and HCD fragment mass tolerance set at 0.4 Daltons and 20 pmm, respectively. For intact N-glycopeptides determination we selected the 52 most common N-glycans in plants as common modification 1. For TMT-SPS-MS3 we used the same parameters as well as the TMT 6-plex N-terminal/lysine residues (+229.163 Da) as static modification.

### Publication Abstract
Despite substantial advances in the use of proteomic technologies, their widespread application in fruit tissues of non-model and recalcitrant species remains limited. This hampers the understanding of critical molecular events during the postharvest period of fleshy tropical fruits. Therefore, we evaluated label-free quantitation (LFQ) and TMT-SPS-MS3 (TMT) approaches to analyse changes in the protein profile of mango peels during postharvest period. We compared two extraction methods (phenol and chloroform/methanol) and two peptide fractionation schemes (SCX and HPRP). We accurately identified 3065 proteins, of which, 1492 were differentially accumulated over at 6 days after harvesting (DAH). Both LFQ and TMT approaches share 210 differential proteins including cell wall proteins associated with fruit softening, as well as aroma and flavour-related proteins, which were increased during postharvest period. The phenolic protein extraction and the high-pH reverse-phase peptide fractionation was the most effective pipeline for relative quantification. Nevertheless, the information provided by the other tested strategies was significantly complementary. Besides, LFQ spectra allowed us to track down intact N-glycopeptides corroborating N-glycosylations on the surface of a desiccation-related protein. This work represents the largest proteomic comparison of mango peels during postharvest period made so far, shedding light on the molecular foundation of edible fruit during ripening.

### Keywords
Mango peel, Proteomics, Protein extraction, Postharvest shelf life

### Affiliations
Instituto de Ecologia A.C.
Instituto de Ecolog'ia A.C.

### Submitter
Miguel Elizalde

### Lab Head
Dr Dr. Eliel Ruiz May
Instituto de Ecologia A.C.


