### Accession
PXD035448

### Title
Tropodithietic acid, a multifunctional antimicrobial, facilitates adaption and  colonization of the producer, Phaeobacter piscinae S26

### Description
In the marine environment, surface-associated bacteria often produce an array of antimicrobial secondary metabolites (MSMs), which have predominantly been perceived as competition molecules. However, they may also affect other hallmarks of surface-associated living, such as motility and biofilm formation. Here, we investigate the ecological significance of an antibiotic secondary metabolite, tropodithietic acid (TDA), in the producing bacterium, Phaeobacter piscinae S26. We constructed a markerless in-frame deletion mutant deficient in TDA biosynthesis wherein TDA production was abolished. Molecular networking demonstrated that other chemical sulphur-containing features, likely related to TDA, were also altered in the secondary metabolome. We found dramatic changes in the physiology of the TDA-deficient mutant, S26ÃƒÅ½Ã¢â‚¬ï¿½tdaB, compared to the wild type S26. Growth of the two strains were similar; however, S26ÃƒÅ½Ã¢â‚¬ï¿½tdaB cells were shorter and more motile. Transcriptome and proteome profiling revealed an increase in expression of genes and relative abundance of proteins related to a type IV secretion system, a prophage, and a gene transfer agent (GTA) in S26ÃƒÅ½Ã¢â‚¬ï¿½tdaB. All these systems may contribute to horizontal gene transfer (HGT), which may facilitate fast adaptation to novel niches. We speculate that, once a TDA-producing population has been established in a new niche, the accumulation of TDA acts as a signal of successful colonization, prompting a switch to a sessile lifestyle. This would lead to a decrease in motility and the rate of HGT, whilst filamentous cells could form the base of a biofilm. In addition, the antibiotic properties of TDA may inhibit invading competing microorganisms.

### Sample Protocol
Proteome sample preparation. 5 mL sample was harvested at 5,000 x g for 5 min, and the supernatant was transferred to a 15 mL falcon tube. The cell fraction was washed twice with 1 mL ice-cold PBS buffer. From here on, Eppendorf Protein LoBind tubes were used. Between washing steps, cells were harvested at 7,000 x g for 1 min and PBS buffer was carefully removed. Lysis buffer (6M Guanidinium Hydrochloride, 10mM tris-(2-carboxy-ethyl)-phosphine, 40mM 2-Chloroacetamide (CAA), 50mM HEPES, pH8.5) was added and samples were incubated at 95Ãƒâ€šÃ‚Â°C for 5 min and subsequently cooled on ice. Ice-cold acetone was added to the supernatant to a final percentage of 80% and stored overnight at -20Ãƒâ€šÃ‚Â°C to precipitate protein. Proteins were precipitated at 2,000 x g for 20 min, acetone carefully removed and the pellet dried in a fumehood. The protein pellet was resuspended in lysis buffer and proteins from supernatant and cell samples were treated identically using a modified version of the protocol described by Schoof et al., 2016 (80) (Supplementary Methods).   Proteome (Mass Spectrometry) Analysis. For each sample, 500 ng of peptide was loaded onto a 2cm C18 trap column (ThermoFisher Scientific), connected in-line to a 50cm C18 reverse-phase analytical column (Thermo EasySpray ES803) using 100% Buffer A (0.1% Formic acid in water) at 750bar, using the Thermo EasyLC 1000 HPLC system, and the column oven operating at 45Ãƒâ€šÃ‚Â°C. Peptides were eluted over a 140 minute gradient ranging from 6 to 60% of 80% acetonitrile, 0.1% formic acid at 250 nl/min, and the Q-Exactive instrument (ThermoFisher Scientific) was run in a DD-MS2 top10 method. Full MS spectra were collected at a resolution of 70,000, with an AGC target of 3ÃƒÆ’Ã¢â‚¬â€�106 or maximum injection time of 20 ms and a scan range of 300ÃƒÂ¢Ã¢â€šÂ¬Ã¢â‚¬Å“1750 m/z. The MS2 spectra were obtained at a resolution of 17,500, with an AGC target value of 1ÃƒÆ’Ã¢â‚¬â€�106 or maximum injection time of 60 ms, a normalised collision energy of 25 and an intensity threshold of 1.7e4. Dynamic exclusion was set to 60 s, and ions with a charge state <2 or unknown were excluded. MS performance was verified for consistency by running complex cell lysate quality control standards, and chromatography was monitored to check for reproducibility.

### Data Protocol
The raw files were analysed using Proteome Discoverer 2.4. Label-free quantitation (LFQ) was enabled by the Minora Feature Detector algorithm in the processing step and Precursor Ions Quantifier node in the consensus step, and spectra were matched against the assembled and annotated genome. Dynamic modifications were set as Oxidation (M), Deamidation (N,Q) and Acetyl on protein N-termini. Cysteine carbamidomethyl was set as a static modification. All results were filtered to a 1% FDR. The data was processed using the Perseus software (v. 1.6.15, https://maxquant.net/perseus/). Proteins were filtered out if present in less than three of the five replicates of any of the sample. Values were log2-transformed and missing values were imputed from a downshifted normal distribution (width: 0.3, downshift: 1.8). A linear model was applied to the data using LIMMA (81) in Rstudio (v. 3.6.3) and comparisons were made between S26ÃƒÅ½Ã¢â‚¬ï¿½tdaB and S26 for each sample fraction (cell or supernatant). To test for significant differences between these comparisons, empirical Bayes statistics were applied to estimate the false discovery rate (FDR, q) with a significance threshold of q=0.01.

### Publication Abstract
Microbial populations swiftly adapt to changing environments through horizontal gene transfer. While the mechanisms of gene transfer are well known, the impact of environmental conditions on the selection of transferred gene functions remains less clear. We investigated ICEs, specifically the ICE<i>clc</i>-type, in <i>Pseudomonas aeruginosa</i> clinical isolates. Our findings revealed co-evolution between ICEs and their hosts, with ICE transfers occurring within strains. Gene functions carried by ICEs are positively selected, including potential virulence factors and heavy metal resistance. Comparison to publicly available <i>P. aeruginosa</i> genomes unveiled widespread antibiotic-resistance determinants within ICE<i>clc</i> clades. Thus, the ubiquitous ICE<i>clc</i> family significantly contributes to <i>P. aeruginosa</i>'s adaptation and fitness in diverse environments.

### Keywords
Secondary metabolites, Prophage, Motility, Phaeobacter, Biofilm, Niche colonization, Tropodithietic acid, Gene transfer agent, Horizontal gene transfer

### Affiliations
DTU Bioengineering
Department of Bioengineering, Technical University of Denmark, DK-2800 Kgs. Lyngby, Denmark

### Submitter
Shengda Zhang

### Lab Head
Dr Lone Gram
Department of Bioengineering, Technical University of Denmark, DK-2800 Kgs. Lyngby, Denmark


