### Accession
PXD035889

### Title
Deficiency of the frontotemporal dementia gene GRN results in gangliosidosis

### Description
Haploinsufficiency of GRN causes frontotemporal dementia (FTD). The GRN locus produces progranulin (PGRN), which is cleaved to lysosomal granulin polypeptides. The function of lysosomal granulins and why their absence causes neurodegeneration are unclear. Here we discover that PGRN-deficient human cells and murine brains, as well as human frontal lobes from GRN-mutation FTD patients have increased levels of gangliosides, glycosphingolipids that contain sialic acid. In these cells and tissues, levels of lysosomal enzymes that catabolize gangliosides were normal, but levels of bis(monoacylglycero)phosphates (BMP), lipids required for ganglioside catabolism, were reduced with PGRN deficiency. Our findings indicate that granulins are required to maintain BMP levels to support ganglioside catabolism, and that PGRN deficiency in lysosomes leads to gangliosidosis. Lysosomal ganglioside accumulation may contribute to neuroinflammation and neurodegeneration susceptibility observed in FTD due to PGRN deficiency and other neurodegenerative diseases.

### Sample Protocol
50 g of protein extracts from cell pellets or mouse tissue were subjected to disulfide bond reduction with 5 mM TCEP (room temperature, 10 min) and alkylation with 25 mM chloroacetamide (room temperature, 20 min), followed by methanol–chloroform precipitation. For lysosomal samples, disulfide bond reduction and alkylation were performed using 20 g of extracts, followed by trichloroacetic acid precipitation. Samples were resuspended in 50 L of 200 mM EPPS, pH 8.5, and digested at 37°C for 2 h with LysC protease at a 200:1 protein-to-protease ratio. Trypsin was then added at a 100:1 protein-to-protease ratio, and the reaction was incubated for 6 h at 37°C. Tandem mass tag labeling of each sample was performed by adding 10 L each of the 20 ng/μL stock of TMTpro reagent along with acetonitrile to achieve a final acetonitrile concentration of approximately 30% (v/v). After incubation at room temperature for 1 h, the labeling efficiency of a small aliquot was tested, and the reaction was then quenched with hydroxylamine to a final concentration of 0.5% (v/v) for 15 min. The TMTpro-labeled samples were pooled together at a 1:1 ratio. The sample was vacuum centrifuged to near dryness, resuspended in 5% formic acid and subjected to C18 solid-phase extraction (SPE) (Sep-Pak, Waters).

### Data Protocol
Raw mass spectra obtained were processed using Sequest. Mass spectra were converted to mzXML using a version of ReAdW.exe. Database searching included all entries from the Human Reference Proteome. Searches were performed with the following settings: 1) 20 ppm precursor ion tolerance for total protein level analysis, 2) Product ion tolerance was set at 0.9 Da, 3) TMTpro on lysine residues or N-termini at +304.207 Da, and 4) Carbamidomethylation of cysteine residues (+57.021 Da) as a static modification and oxidation of methionine residues (+15.995 Da) as a variable modification. Peptide-spectrum matches (PSMs) were adjusted to a 1% false discovery rate50 PSM filtering was performed using a linear discriminant analysis. To quantify the TMTpro-based reporter ions in the datasets, the summed signal-to-noise (S:N) ratio for each TMTpro channel was obtained and found the closest matching centroid to the expected mass of the TMTpro reporter ion (integration tolerance of 0.003 Da). Proteins were quantified by summing reporter ion counts across all matching PSMs, as described51 PSMs with poor quality, or isolation specificity less than 0.7, or with TMTpro reporter summed signal-to-noise ratio that were less than 100 or had no MS3 spectra were excluded from quantification.

### Publication Abstract
None

### Keywords
Bmp, Pgrn, Lysosome, Frontotemporal dementia, Gangliosides, Haploinsufficiency

### Affiliations
Harvard T.H. Chan School of Public Health
Harvard Medical School
Department of Cell Biology Harvard Medical School Boston, MA USA

### Submitter
Sebastian Boland

### Lab Head
Dr J. Wade Harper
Department of Cell Biology Harvard Medical School Boston, MA USA


