### Accession
PXD006238

### Title
Tropheryma whipplei proteome - Analyses of new genomic, transcriptomic or proteomic data commonly result in trashing many unidentified data escaping the ‘canonical’ DNA-RNA-protein scheme

### Description
Analyses of new genomic, transcriptomic or proteomic data commonly result in trashing many unidentified data escaping the ‘canonical’ DNA-RNA-protein scheme. Testing systematic exchanges of nucleotides over long stretches produces inversed RNA pieces (here named “swinger” RNA) differing from their template DNA. These may explain some trashed data. Here analyses of genomic, transcriptomic and proteomic data of the pathogenic Tropheryma whipplei according to canonical genomic, transcriptomic and translational 'rules' resulted in trashing 58.9% of DNA, 37.7% RNA and about 85% of mass spectra (corresponding to peptides). In the trash, we found numerous DNA/RNA fragments compatible with “swinger” polymerization. Genomic sequences covered by «swinger» DNA and RNA are 3X more frequent than expected by chance and explained 12.4 and 20.8% of the rejected DNA and RNA sequences, respectively. As for peptides, several match with “swinger” RNAs, including some chimera, translated from both regular, and «swinger» transcripts, notably for ribosomal RNAs. Congruence of DNA, RNA and peptides resulting from the same swinging process suggest that systematic nucleotide exchanges increase coding potential, and may add to evolutionary diversification of bacterial populations.

### Sample Protocol
Whole cell Twist protein extract (300 µg) was separated on 11% SDS-PAGE. The proteins were visualized by CBB staining and the gel was excised in duplicate into 63 equally sized pieces. After several washes, the proteins in the gel were reduced, alkylated, and digested with trypsin. Tryptic peptides were extracted with acetonitrile and the collected supernatant dried. Each sample was reconstituted in 4% acetonitrile 0.1% trifluoroacetic acid then  injected twice 10 µL on the nanoliquid chromatography system (Ultimate 3000, Thermo Fisher Scientific, Illkirch). Tryptic peptides were first loaded on a C18 μ-precolumn (PepMap100, 300 μm i.d. x 5 mm, 5 μm, 100 Å, Thermo Fisher Scientific, Illkirch) with mobile phase 4% acetonitrile 0.1% trifluoroacetic acid at 20 µL/min. Peptides were then eluted on a reverse phase C18 column (Acclaim PepMap100, 75 μm i.d. x 15 cm, 3 μm, 100 Å, Thermo Fisher Scientific, Illkirch) with a 2 hours gradient increasing from 2% to 40% acetonitrile 0.5% formic acid at 300 nL/min. Peptides were detected using a linear ion trap mass spectrometer (LTQ XL, Thermo Fisher Scientific, Illkirch) equipped with a nano electrospray ion source. A data dependent acquisition method was used to fragment the top 10 most intense peptides (2+, 3+ and 4+ charges states) by collision induced dissociation determined from an initial MS survey from 350 to 1600 m/z. MS/MS scans were performed with an isolation width of 2 m/z, collision energy of 35%. Wide band activation and dynamic exclusion (30 s) options were activated during the experiments.

### Data Protocol
All six frames (three for each strand) of all 23 swinger versions of the reference T. whipplei genome are translated according to the regular bacterial genetic code. Each of these six hypothetical sequences are trypsinized in silico, meaning that after each arginine and each lysine, the long hypothetical peptides are cut. The resulting short peptides are used by Thermos Protein Discoverer to predict a theoretical mass spectrometry distribution, which is matched with observed mass spectrometry data. Only peptides longer than five residues are considered. In case a peptide is coded by a sequence that includes stop codon(s), stop(s) are translated as ‘X’. Each peptide including stop(s) is duplicated 18 times, and X is replaced by one of 20 amino acid species, excluding leucine and isoleucine, which have equal masses and are not distinguishable by mass spectrometry. By default, Thermos Protein Discoverer considers ‘X’ as leucine/isoleucine. Hence predicted peptides including stops are replicated 19 times, once coding stops by X corresponding to leucine/isoleucine, and 18 other replicates where X is replaced in each case by another letter symbolizing an amino acid (besides leucine/isoleucine). Hence predicted peptides include the possibility that any amino acid could be inserted for stop(s). When a peptide includes more than one stop, each of its 19 variants inserts the same letter at stops. The raw files were computed using the Proteome Discoverer 1.3 software (Thermo Fisher Scientific, Illkirch). Scans were searched against the predicted tryptic peptide banks described above, translated from each of the 23 swinger versions of Tropheryma’s genome. Consensus searches were handled with the Sequest (Thermo Fisher Scientific, Illkirch) algorithm with the following mass tolerances: Parent = 1 Da and Fragment = 0.5 Da (monoisotopic masses). Fixed carbamidomethyl (C) and variable Oxydation (M) modifications were activated. One missed trypsin cleavage was allowed. False discovery rate was estimated against a reverse decoy database using the Percolator algorithm. Only peptides matching at false discovery rate q <0.05 with predicted peptides were retained. We use q, the statistical probability accounting for false discovery rates: q = 0.05 means that in a population of positive results, 1 among 20 positive results could be a false positive. This differs from regular statistical significance p values, which adjust type I error rates (false positives) on the complete population, and not specifically on the population tests yielding positive results. This procedure is more conservative than considering the whole population of negative and positive results. An additional criterion was used to select matches between predicted and observed mass spectrometry distributions. While q indicates whether the match is better than expected by chance (according to the decoy peptide population), it does not reflect the quality of the match, the same way as a correlation may be statistically significant, yet the correlation coefficient and its slope low, meaning that the correlation explains a small fraction of the variation. This property is expressed here by Xcorr, a measure of goodness of fit between expected and observed mass spectra: peptides with q <0.05 were considered only when Xcorr>2. These peptides are then compared with those predicted from the in silico trypsinization of the ‘regular’ proteome of Tropheryma, to detect whether a peptide matching the predicted swinger-translated peptides could be also peptides resulting from the regular proteome of Tropheryma. Such cases were rare and represent about 1% of the swinger peptides detected. This redundancy phenomenon between swinger and regular proteomes is minor and hence can be at this point ignored.

### Publication Abstract
None

### Keywords
Noncanonical sequences, Tropheryma whipplei

### Affiliations
URMITE
URMITE, IHU -  Méditerranée Infection 19-21 Boulevard Jean Moulin 13005 Marseille, France Phone: (+33) 4 Fax : (+33) 4 13 73 24 02

### Submitter
Herve Seligmann

### Lab Head
Dr Herve Seligmann
URMITE, IHU -  Méditerranée Infection 19-21 Boulevard Jean Moulin 13005 Marseille, France Phone: (+33) 4 Fax : (+33) 4 13 73 24 02


