### Accession
PXD008036

### Title
Olfactory proteomics in Parkinson disease

### Description
Olfactory dysfunction is one of the earliest features in Lewy-type alpha-synucleinopathies (LTS) such as Parkinson´s disease (PD). However, the underlying molecular mechanisms associated to smell impairment is poorly understood. To reveal the missing links in the biochemical understanding of olfactory dysfunction in PD, we have applied mass spectrometry-based proteomics in postmortem olfactory bulbs (OBs) dissected from parkinsonian subjects with different LTS staging respect to elderly controls (n= 24, mean age 79 years).

### Sample Protocol
Sample preparation for proteomic analysis - OB specimens derived from control and PD cases were homogenized in lysis buffer containing 7 M urea, 2 M thiourea, 4% (w/v) CHAPS, 50 mM DTT. The homogenates were spinned down at 100.000 x g for 1 h at 15°C. Prior to proteomic analysis, protein extracts were precipitated with methanol/choloroform, and pellets dissolved in 6M Urea, Tris 100mM pH 7.8. Protein quantitation was performed with the Bradford assay kit (Bio-Rad). Label free LC-MS/MS – The protein extract for each sample was diluted in Laemmli sample buffer and loaded into a 0.75 mm thick polyacrylamide gel with a 4% stacking gel casted over a 12.5% resolving gel. The run was stopped as soon as the front entered 3 mm into the resolving gel so that the whole proteome became concentrated in the stacking/resolving gel interface. Bands were stained with Coomassie Brilliant Blue and excised from the gel. Protein enzymatic cleavage (10ug) was carried out with trypsin (Promega; 1:20, w/w) at 37°C for 16 h as previously described (Shevchenko A, Tomas H, Havlis J et al. Nature Protocols 2006). Peptides mixtures were separated by reverse phase chromatography using an Eksigent nanoLC ultra 2D pump fitted with a 75 μm ID column (Eksigent 0.075 x 250). Samples were first loaded for desalting and concentration into a 0.5 cm length 100 μm ID precolumn packed with the same chemistry as the separating column. Mobile phases were 100% water 0.1% formic acid (FA) (buffer A) and 100% Acetonitrile 0.1% FA (buffer B). Column gradient was developed in a 240 min two step gradient from 5% B to 25% B in 210 min and 25%B to 40% B in 30 min. Column was equilibrated in 95% B for 9 min and 5% B for 14 min. During all process, precolumn was in line with column and flow maintained all along the gradient at 300 nl/min. Eluting peptides from the column were analyzed using an Sciex 5600 Triple-TOF system. Information data acquisition was acquired upon a survey scan performed in a mass range from 350 m/z up to 1250 m/z in a scan time of 250 ms. Top 35 peaks were selected for fragmentation. Minimum accumulation time for MS/MS was set to 100 ms giving a total cycle time of 3.8 s. Product ions were scanned in a mass range from 230 m/z up to 1500 m/z and excluded for further fragmentation during 15 s.

### Data Protocol
Peptide Identification and Quantification – MS/MS data acquisition was performed using Analyst 1.5.2 (Sciex) and spectra ﬁles were processed through Protein Pilot Software (v.4.0.8085-Sciex) using ParagonTM Algorithm (v.4.0.0.0) (Shilov IV, Seymour SL, Patel AA Mol Cell Prot 2007) for database search, Progroup™ for data grouping, and searched against the concatenated target-decoy UniProt proteome reference Human database (Proteome ID: December 2015). False discovery rate was performed using a non lineal fitting method (Tang WH, Shilov IV, Seymour SL JPR 2008) and displayed results were those reporting a 1% Global false discovery rate or better. The peptide quantification was performed using the Progenesis LC−MS software (ver. 3.0, Nonlinear Dynamics). Using the accurate mass measurements from full survey scans in the TOF detector and the observed retention times, runs were aligned to compensate for between-run variations in our nanoLC separation system. To this end, all runs were aligned to a reference run automatically chosen by the software, and a master list of features considering m/z values and retention times was generated. The quality of these alignments was manually supervised with the help of quality scores provided by the software. The peptide identifications were exported from Protein Pilot and imported in Progenesis LC− MS software where they were matched to the respective features. Output data files were managed using R scripts for subsequent statistical analyses and representation. Proteins identified by site (identification based only on a modification), reverse proteins (identified by decoy database) and potential contaminants were filtered out. Proteins quantified with at least one unique peptide, an ANOVA  p-value lower than 0.05, and an absolute fold changes of <0.77 (down-regulation) or >1.3 (up-regulation) in linear scale were considered to be significantly differentially expressed.

### Publication Abstract
Olfactory dysfunction is one of the earliest features in Lewy-type alpha-synucleinopathies (LTSs) such as Parkinson's disease (PD). However, the underlying molecular mechanisms associated to smell impairment are poorly understood. Applying mass spectrometry-based quantitative proteomics in postmortem olfactory bulbs across limbic, early-neocortical, and neocortical LTS stages of parkinsonian patients, a proteostasis impairment, was observed, identifying 268 differentially expressed proteins between controls and PD phenotypes. In addition, network-driven proteomics revealed a modulation in ERK1/2, MKK3/6, and PDK1/PKC signaling axes. Moreover, a cross-disease study of selected olfactory molecules in sporadic Alzheimer's disease (AD) cases revealed different protein derangements in the modulation of secretagogin (SCGN), calcyclin-binding protein (CACYBP), and glucosamine 6 phosphate isomerase 2 (GNPDA2) between PD and AD. An inverse correlation between GNPDA2 and &#x3b1;-synuclein protein levels was also reflected in PD cerebrospinal fluid. Interestingly, PD patients exhibited significantly lower serum GNPDA2 levels than controls (n&#xa0;= 82/group). Our study provides important avenues for understanding the olfactory bulb proteostasis imbalance in PD, deciphering mechanistic clues to the equivalent smell deficits observed in AD and PD pathologies.

### Keywords
Proteomics, Olfactory bulb, Parkinson

### Affiliations
Proteomics unit
Clinical Neuroproteomics Lab Navarrabiomed Proteored Pamplona, Spain

### Submitter
Enrique SantamarÃ­a

### Lab Head
Dr Enrique Santamaria
Clinical Neuroproteomics Lab Navarrabiomed Proteored Pamplona, Spain


