### Accession
PXD007529

### Title
Enrichment and quantitation of newly secreted proteins under serum condition by combining BONCAT and pulsed SILAC

### Description
Despite the increasing interest of secretome involved in paracrine/autocrine mechanisms, the majority of the mass spectrometric cell secretome studies have been performed using serum-free medium (SFM), due to low-abundance of secreted proteins and high-abundance of contaminating fetal bovine serum (FBS) proteins in serum-containing medium (SCM). In this study, through the combination of bioorthogonal non-canonical amino acid tagging (BONCAT) and pulsed-SILAC (pSILAC), the low-abundant secreted proteins were captured selectively and quantified. We set up the combined BONCAT-pSILAC method by optimizing capturing condition and by using a composite human-FBS database, and applied it to the analysis of differentially secreted proteins between SFM and SCM. The amount of secreted proteins for 24 h in SCM was much bigger than that from SFM for U-87MG glioblastoma cells (1.28-fold) and mesenchymal stem cells from human umbilical cord blood (hUCB-MSCs; 2.01-fold). The proteins secreted more in SCM were predominantly predicted to be truly secretory. Therefore, this study suggests the analysis of the secretome should be processed in serum-containing medium that promotes cell proliferation and secretion.

### Sample Protocol
1. Sample Preparation Newly synthesized and secreted proteins were enriched from concentrated media using the Click-iT® Protein Enrichment Kit. Typically 100 µl of agarose resin slurry was used for concentrated SCM. To determine the appropriate volume of SCM for the CuAAC reaction, the enrichment experiments were performed at five different volume conditions (3, 10, 20, 30 and 60 mL) of SCM. The SCM was concentrated up to ~250 µl through ultrafiltration using ‘Amicon Ultra-15’ centrifugal filter devices (Millipore, MA), and was exchanged into a denaturation buffer containing 8 M urea and 100 mM Tris (pH 8.2) by repeating dilution-ultrafiltration twice. CuAAC reaction was carried out overnight at RT after mixing the sample with appropriate resin and solutions supplied by the vendor. The whole mixture adjusted to 0.5 ml with water was then transferred onto a 0.22-µm centrifugal filter unit. All the water soluble materials were removed by spinning the filter unit, leaving only the resin. After then, the resin with proteins attached was treated with 20 mM DTT in 0.5 ml of 1% SDS at 70°C for 15 min, and then with 40mM iodoacetamide in 0.5 ml of 1% SDS at RT for 30 min in the dark, and washed with 0.5 ml of 1% SDS, 0.5 ml of 8 M urea/100 mM Tris (pH 8.2) and 0.5 ml of 20% acetonitrile. Each washing step was repeated at least five times. The washed resin was resuspended in a buffer containing 100 mM Tris (pH 8.2), 2 mM CaCl2, and 10% acetonitrile, mixed with 0.5 µg trypsin, and incubated for 16 h at 37°C. The peptides of trypsin cleavage product were collected by centrifugation and the resin was washed with 0.5 mL of water. The two solutions were combined, acidified with 0.5% TFA, and analyzed by LC-MS/MS.  2. Liquid chromatography and tandem mass spectrometry (LC-MS/MS) LTQ-XL mass spectrometer was used during BONCAT optimization experiments. Peptide samples were reconstituted in 0.4% acetic acid and one fifth of the sample was injected into a reversed-phase Magic C18aq column on an Agilent 1200 HPLC system. The column was pre-equilibrated with 95% solvent A (0.1% formic acid in water) and 5% solvent B (0.1% formic acid in acetonitrile). The peptides were eluted at a flow rate of 0.4 μL/min with a linear gradient of 5–40% solvent B over 120 min. The ESI voltage was set to 1.9 kV, the capillary voltage to 30 V, and the temperature of the heated capillary to 250°C. The MS survey was scanned from 300 to 2,000 m/z, followed by three data-dependent MS/MS scans with the following options: isolation width, 1.5 m/z; normalized collision energy, 25%; dynamic exclusion duration, 180 s. The optimization experiment was performed in duplicate, triplicate or quadruplicate.  Q Exactive mass spectrometer was used in the BONCAT-pSILAC experiments.  One microgram of sample reconstituted in 0.4% acetic acid was injected into a reversed-phase C18 column on an Eksigent nanoLC-ultra 1D plus system at 95% solvent A and 5% solvent B. The peptides were eluted with a linear gradient from 5% to 40% solvent B over 200 min followed by 80% solvent B wash and 95% solvent A re-equilibration at a flow rate of 300 nL/min with a total run time of 230 min. Survey full-scan MS spectra (m/z 350–1800) were acquired at a resolution of 70000. Source ionization parameters were as follows: spray voltage, 2.5 kV; capillary temperature, 300°C; and s-lens level, 44.0. The MS peak width at half height was <30 s. The MS/MS spectra of the 12 most intense ions from the MS1 scan with a charge state ≥2 were acquired with the following options: resolution, 17500; isolation width, 2.0 m/z; normalized collision energy, 27%; ion selection threshold, 4.00E±03 counts; and peptide match, ‘preferred’. The experiment was performed in triplicate.

### Data Protocol
Raw data of LC-MS/MS were processed using SEQUEST in Proteome Discoverer 2.1.1.21. Human UniProtKB database (released in January 2016) combined with experimentally validated FBS database (HFDB) was used, unless otherwise indicated. Search parameters were two missed trypsin cleavage sites, cysteine carbamidomethylation as fixed modification, methionine oxidation and N-terminal protein acetylation as variable modifications. Peptide identification was performed with an allowed initial precursor mass tolerance up to 15 ppm and an allowed fragment mass deviation 0.05 Da. Peptide and protein results were filtered to 1% FDR. In case of pSILAC data, three SequestHT nodes were used. First node was set to medium- and heavy-isotope of SILAC as variable modifications, and second and third nodes were set to medium-isotope and heavy-isotope as fixed modifications, respectively. For protein quantification, the H/M ratio of proteins was calculated by dividing the intensities of the medium- and heavy-isotope and transformed to log2 values.

### Publication Abstract
Despite the increased interest in secretomes associated with paracrine/autocrine mechanisms, the majority of mass spectrometric cell secretome studies have been performed using serum-free medium (SFM). On the other hand, serum-containing medium (SCM) is not recommended very much because the secretome obtained with SCM is easily contaminated with fetal bovine serum (FBS) proteins. In this study, through the combination of bioorthogonal non-canonical amino acid tagging (BONCAT) and pulsed-SILAC (pSILAC), we analyzed differentially secreted proteins between SFM and SCM in a cancer-derived human cell, U87MG, and a mesenchymal stem cell derived from human Wharton's jelly (hWJ-MSCs). In most cases, the bioinformatic tools predicted a protein to be truly secretory when the secretion level of the protein was more in SCM than in SFM. In the case of hWJ-MSCs, the amount of proteins secreted in SCM for 24&#x2009;hours was larger than that of SFM (log<sub>2</sub> fold change&#x2009;=&#x2009;0.96), even considering different cell proliferation rates. hWJ-MSCs proteins secreted more in SCM included several positive markers of MSC paracrine factors implicated in angiogenesis, neurogenesis and osteogenesis, and upstream regulators of cell proliferation. Our study suggests the analysis of the secretome should be processed in SCM that promotes cell proliferation and secretion.

### Keywords
Fbs, Lc-ms/ms, Serum, Boncat, Secretome, Pulsed silac

### Affiliations
Yokohama City University
Center for Theragnosis, Korea Institute of Science and Technology, Seoul, Korea

### Submitter
Jihye Shin

### Lab Head
Dr Cheolju Lee
Center for Theragnosis, Korea Institute of Science and Technology, Seoul, Korea


