### Accession
PXD011369

### Title
Comparative analysis of WT CBL and CBL 70Z interactomes.

### Description
Cbl (Casitas B-lineage lymphoma) is a prominent post-translational regulator of protein tyrosine kinases (PTKs) that targets activated receptors for ubiquitination-mediated degradation. However, mutations within its E3 ubiquitin ligase activity rendering RING domain are often not sufficient to induce transformation. Rather, a conserved residue (Y371) within the linker helix region (LHR) of Cbl has been a mutational hotspot in myeloproliferative diseases. Interestingly, phosphorylation of Y371 is a prerequisite for Cblâ€™s ability to ubiquitinate PTKs. We have previously shown that Y371 mutations affect the conformation of Cbl that can be related to its transformation potential. We wanted to investigate for any additional advantage that the Cbl-Y371 mutants could gain over their wild-type counterpart so as to explain their oncogenicity. In our current study, we utilized CBL-70Z, a well-characterized oncogenic CBL mutant lacking residues 366-382 that behave similar to the prevalent MDS/MPN CBL mutants, as an example to explore differences in interactomes between Wild type CBl and CBL 70Z. Any divergent interacting abilities between the two can be used to decipher the mechanism by which the mutants elicit oncogenic transformation.

### Sample Protocol
Myc-tagged empty vector (EV), full-length WT-CBL and CBL-70Z were expressed in HEK293 cells and immunoprecipitated with Myc trap in triplicate experiments. The beads were washed twice with IP lysis buffer and once each with IP wash buffer and HPLC grade water. Proteins from the beads were eluted and digested following established protocol [Hubner NC, Bird AW, Cox J, Splettstoesser B, Bandilla P, Poser I, Hyman A, Mann M. Quantitative proteomics combined with BAC TransgeneOmics reveals in vivo protein interactions. J Cell Biol. 2010 May 17;189(4):739-54]. Digestion of samples was performed by subsequent use of both endoLysC (Alpha Laboratories) and trypsin (Promega).  MS analysis Tryptic peptides were separated by nanoscale C18 reverse-phase liquid chromatography using an EASY-nLC 1200 (Thermo Fisher Scientific) coupled online to an LTQ Orbitrap Velos mass spectrometer (Thermo Fisher Scientific) via nanoelectrospray ion source (Thermo Fisher scientific). Peptides were separated on a 20 cm fused silica emitter (New Objective) packed in house with reverse phase Reprosil Pur Basic 1.9 Âµm (Dr. Maisch GmbH). For the full scan a resolution of 60,000 at 250 Th was used. The top ten most intense ions in the full MS were isolated for fragmentation with a target of 50,000 ions at a resolution of 15,000 at 250 Th. MS data were acquired using the XCalibur software (Thermo Fisher Scientific).

### Data Protocol
The MS Raw files were processed with MaxQuant software [Cox, J. and M. Mann, MaxQuant enables high peptide identification rates, individualized p.p.b.-range mass accuracies and proteome-wide protein quantification. Nat Biotech, 2008. 26(12): p. 1367-1372] version 1.5.5.1 and searched with Andromeda search engine [Cox, J., et al., Andromeda: A Peptide Search Engine Integrated into the MaxQuant Environment. J Proteome Res, 2011. 10(4): p. 1794-1805.], querying UniProt [The UniProt Consortium, The Universal Protein Resource (UniProt) in 2010. Nucleic Acids Res, 2010. 38(Database issue): p. D142-D148] Homo sapiens (09/07/2016; 92939 entries). Database was searched requiring specificity for trypsin cleavage and allowing maximum two missed cleavages. Methionine oxidation, N-terminal acetylation were specified as variable modifications, and Cysteine carbamidomethylation as fixed modification. The peptide, protein and site false discovery rate (FDR) was set to 1 %. The common reverse and contaminant hits (as defined in MaxQuant output) were removed. Only protein groups identified with at least one uniquely assigned peptide were used for the analysis. For label-free quantification, proteins quantified in all 3 replicates in at least one group, were measured according to the label-free quantification algorithm available in MaxQuant [Cox, J., Hein, M. Y., Luber, C. A., Paron, I., Nagaraj, N. and Mann, M. (2014). Accurate proteome-wide label-free quantification by delayed normalization and maximal peptide ratio extraction, termed MaxLFQ. Mol. Cell. Proteomics 13, 2513-2526]. Significantly enriched proteins were selected using a t-test analysis with a 5% FDR.

### Publication Abstract
Casitas B-lineage lymphoma (CBL) is a ubiquitin ligase (E3) that becomes activated upon Tyr371-phosphorylation and targets receptor protein tyrosine kinases for ubiquitin-mediated degradation. Deregulation of CBL and its E3 activity is observed in myeloproliferative neoplasms and other cancers, including breast, colon, and prostate cancer. Here, we explore the oncogenic mechanism of E3-inactive CBL mutants identified in myeloproliferative neoplasms. We show that these mutants bind strongly to CIN85 under normal growth conditions and alter the CBL interactome. Lack of E3 activity deregulates CIN85 endosomal trafficking, leading to an altered transcriptome that amplifies signaling events to promote oncogenesis. Disruption of CBL mutant interactions with EGFR or CIN85 reduces oncogenic transformation. Given the importance of the CBL-CIN85 interaction in breast cancers, we examined the expression levels of CIN85, CBL, and the status of Tyr371-phosphorylated CBL (pCBL) in human breast cancer tissue microarrays. Interestingly, pCBL shows an inverse correlation with both CIN85 and CBL, suggesting that high expression of inactivated CBL could coordinate with CIN85 for breast cancer progression. Inhibition of the CBL-CIN85 interaction with a proline-rich peptide of CBL that binds CIN85 reduced the proliferation of MDA-MB-231 cells. Together, these results provide a rationale for exploring the potential of targeting the EGFR-CBL-CIN85 axis in CBL-inactivated mutant cancers.

### Keywords
Ubiquitination, Endosomal trafficking, Label-free quantitation, Cbl, Rtks, On-beads digestion.

### Affiliations
CRUK Beatson Institute, Switchback Road, Glasgow G61 1BD, UK.
Proteomics

### Submitter
Sergio Lilla

### Lab Head
Dr Sara Rossana Zanivan
CRUK Beatson Institute, Switchback Road, Glasgow G61 1BD, UK.


