### Accession
PXD012877

### Title
Modeling protein destiny in developing tomato fruit

### Description
Proteomics and transcriptomics data of tomato fruits at 9 developmental stages were used to calculate with a mathematical model the rate constants of synthesis and degradation for over 1,000 proteins.

### Sample Protocol
Samples were taken on tomato fruit pericarps.Total tomato proteins were extracted as in Faurobert et al. (Methods Mol. Biol. 2007, 355:9-14). LC-MS/MS analyses were performed with a NanoLC-Ultra System (nano2DUltra, Eksigent, Les Ulis, France) coupled with aQ-Exactive mass spectrometer (Thermo Electron, Waltham, MA, USA) as in Havé et al. (J. Exp. Bot. 2017, 69:1369-1385). For each sample, 800 ng (4μl from a 0.200 ng/μl solution) of protein digest were loaded onto a Biosphere C18 pre-column (0.1 × 20 mm, 100 Å, 5 μm; Nanoseparation) at 7.5 μl min −1 and desalted with 0.1% formic acid and 2% ACN. After 3 min, the pre-column was connected to a Biosphere C18 nanocolumn (0.075 × 300 mm, 100 Å, 3723 μm; Nanoseparation).

### Data Protocol
Protein identification was performed using the protein sequence database of S. lycopersicum Heinz assembly v2.40 (ITAG2.4) downloaded from https://solgenomics.net/ (34725 entries). A contaminant database, which contains the sequences of standard contaminants, was also interrogated. Database search was performed with X!Tandem (version 2015.04.01.1; http://www.thegpm.org/TANDEM/). Identified proteins were filtered and sorted by using X!TandemPipeline version 3.3.4 (Langella et al., 2017, J. Proteome Res. 16:494-503). Criteria used for protein identification were (1) at least two different peptides identified with an E-value smaller than 0.01, and (2) a protein E-value (product of unique peptide E-values) smaller than 10 −5 . Peptide ions were quantified based on extracted ion chromatograms (XIC) using MassChroQ software (Valot et al., 2011, Proteomics, 11:3572-3577) version 2.2. Peptide intensities of each sample were normalized by using a method taking retention time into account, as described in Millan-Oropeza et al (2017, J. Proteome Res. 16:2597-2613). Peptides shared by several proteins were removed. In addition, peptides showing an unstable retention time across samples were considered as unreliable and removed.

### Publication Abstract
Protein synthesis and degradation are essential processes that regulate cell status. Because labeling in bulky organs, such as fruits, is difficult, we developed a modeling approach to study protein turnover at the global scale in developing tomato (<i>Solanum lycopersicum</i>) fruit. Quantitative data were collected for transcripts and proteins during fruit development. Clustering analysis showed smaller changes in protein abundance compared to mRNA abundance. Furthermore, protein and transcript abundance were poorly correlated, and the coefficient of correlation decreased during fruit development and ripening, with transcript levels decreasing more than protein levels. A mathematical model with one ordinary differential equation was used to estimate translation (<i>k<sub>t</sub></i> ) and degradation (<i>k<sub>d</sub></i> ) rate constants for almost 2,400 detected transcript-protein pairs and was satisfactorily fitted for &gt;1,000 pairs. The model predicted median values of &#x223c;2 min for the translation of a protein, and a protein lifetime of &#x223c;11 d. The constants were validated and inspected for biological relevance. Proteins involved in protein synthesis had higher <i>k<sub>t</sub></i> and <i>k<sub>d</sub></i> values, indicating that the protein machinery is particularly flexible. Our model also predicts that protein concentration is more strongly affected by the rate of translation than that of degradation.

### Keywords
Systems biology, Transcriptome, Lc-ms/ms, Protein turnover

### Affiliations
Génétique Quantitative et Évolution – Le Moulon
INRA – Université Paris-Sud – CNRS – AgroParisTech
Ferme du Moulon
91190 Gif-sur-Yvette
France
UMR 1332 BFP, INRA, Univ Bordeaux, F33883 Villenave d’Ornon, France

### Submitter
Mélisande Blein-Nicolas

### Lab Head
Dr Sophie Colombié
UMR 1332 BFP, INRA, Univ Bordeaux, F33883 Villenave d’Ornon, France


