### Accession
PXD016915

### Title
Impact of miRNAs on the USSC proteome after XXL-differentiation

### Description
The contribution of microRNA-mediated posttranscriptional regulation on the final proteome in differentiating cells remains elusive. Here, we evaluated the impact of microRNAs (miRNAs) on the proteome of human umbilical cord blood-derived unrestricted somatic stem cells (USSC) during retinoic acid (RA) differentiation by a systemic approach using next generation sequencing analysing mRNA and miRNA expression and quantitative mass spectrometry-based proteome analyses. Interestingly, regulation of mRNAs and their dedicated proteins highly correlated during RA-incubation. Additionally, RA-induced USSC demonstrated a clear separation from native USSC thereby shifting from a proliferating to a metabolic phenotype. Bioinformatic integration of up- and downregulated miRNAs and proteins initially implied a strong impact of the miRNome on the XXL-USSC proteome. However, quantitative proteome analysis of the miRNA contribution on the final proteome after ectopic overexpression of downregulated miR-27a-5p and miR-221-5p or inhibition of upregulated miR-34a-5p, respectively, followed by RA-induction revealed only minor proportions of differentially abundant proteins. In addition, only small overlaps of these regulated proteins with inversely abundant proteins in non-transfected RA-treated USSC were observed. Hence, mRNA transcription rather than miRNA-mediated regulation is the driving force for protein regulation upon RA-incubation, strongly suggesting that miRNAs are fine-tuning regulators rather than active primary switches during RA-induction of USSC.

### Sample Protocol
For analyses of proteome, USSC lines 4/101, 4/146, 5/03, 5/73, 7/18, and 8/77 were cultured in parallel. USSC were seeded on poly-D-lysine (PDL, 0.5 mg/ml) and laminin (13 ������g/ml) pre-coated glass coverslips in 6-well plates with a density of 110.000-140.000 USSC per well. 24h later, USSC were incubated with differentiation medium (XXL) including DMEM GlutaMAXTM (GibcoTM), 15% FBS, penicillin/streptomycin, 50 ng/ml beta-NGF, 20 ng/ml basic FGF (both Peprotech), 1 mM dibutyryl-cAMP, 0.5 mM 3-isobutyl-methylxanthine and 10 ������M all-trans-retinoic acid (all Sigma-Aldrich, Steinheim, Germany). Medium was changed every 2-3 days. USSC were used in passage 7-9 for all experiments. Native and XXL-USSC were washed three times with PBS to remove FBS and subsequently removed by scraping the cells with a cell scraper in 1 ml ice-cold PBS per well. Cell suspension was centrifuged at 800 x g for 5 min at 4������C, the supernatant was removed, and cells were lysed by adding buffer containing 30mM Tris Base, 7M urea, 2M thiourea, 4% CHAPS (pH 8.5). The lysate was sonicated 6 x 10 sec and centrifuged at 16.000 x g for 15 min. Protein concentration of the supernatant was determined by Pierce��������� 660nm Protein Assay (Thermo Fisher Scientific). For in-gel digestion, 5 ������g protein per sample was loaded on an SDS-PAGE and run shortly for ~10 min (5 mm running distance). Afterwards, proteins were visualized by silver staining according to Nesterenko et al. The resulting lane was cut out, decolorized with a 1:1 mix of 30 mM sodium thiosulfate (Fluka) and 100 mM potassium hexacyanoferrate (III) (Merck), reduced with 10 mM dithiothreitol (Serva) and alkylated with 55 mM iodacetamide (Sigma Aldrich). Overnight trypsin digestion (2 ������g, Serva) was performed at 37������C and peptides were extracted with 50% acetonitrile (Sigma Aldrich) and 0.05% trifluoroacetic acid. For transfection experiments, USSC line 5/03 was used. USSC were seeded on PDL and laminin pre-coated glass coverslips with a density of 150,000-170,000 USSC / well on 6 well plates and each well was transfected with 3 pMol of the respective miRIDIAN miRNA mimic or inhibitor (Dharmacon) using Dharmafect (Dharmacon) according to the manufacturer's protocol. USSC were incubated with XXL-medium 24h after transfection and proteomes were analysed 3 days upon XXL-induction as described above. Extracted peptides were separated by a Ultimate 3000 RSCLnano System (Thermo Fisher Scientific) with a Acclaim PepMap100 trap column (3 ������m C18 particle size, 100 ������ pore size, 75 ������m inner diameter, 2 cm length, Thermo Fisher Scientific) as a precolumn using 0.1% TFA as a mobile phase and a Acclaim PepMapRSLC (2 ������m C18 particle size, 100 ������ pore size, 75 ������m inner diameter, 25 cm length, Thermo Fisher Scientific) as analytical column. The flow rate was constant with 300 nl/min using a 2h gradient of 0.1%FA (Fluka) to 0.1% FA/60% acetonitrile. Separated peptides were eluted via nano electrospray ionization into the mass spectrometer (QExactive hybrid quadrupole-orbitrap mass spectrometer, Thermo Fisher Scientific). Mass spectra were recorded in positive ion mode with a mass range of 300���������2,000 m/z and a resolution of 70,000. Up to ten precursors (+2, +3 charge states) were isolated within a 2 m/z isolation window and fragmented via higher-energy collisional dissociation. MS/MS spectra were recorded in centroid mode with a maximal ion time of 60 ms and a target value for the automatic gain control set to 100,000. The resolution was 17,500 at a scan range of 200 to 2,000 m/z. Already fragmented precursors were excluded from further isolation for the next 100 s.

### Data Protocol
Proteome Discoverer (Version 1.4, Thermo Fisher Scientific http://www.thermoscientific.com/en/product/proteome-discoverer-software.html) and Mascot search engine were used for protein identification. MS/MS spectra were searched against the UniProtKB Database (version: 2016/02). Following search parameters were used: Mass tolerance of 10 ppm (MSmode) and 0.4 Da (MS/MS mode), enzyme specificity was trypsin, two missed cleavage sites were considered during the search against human database. Mass range setting was 350���������5,000 Da. Carbamidomethylation of cysteine was set as fixed modification. Oxidation of methionine was accepted as variable modification. For positive identification, we considered a false discovery rate (FDR) of < 1% on peptide level (high peptide confidence, default p < 0.01). For FDR calculation, a decoy approach based on reversed protein sequences using Proteome Discoverer was applied. Label-free relative quantification was performed with Progenesis QI for proteomics 2.0 (Nonlinear Dynamics, Newcastle upon Tyne, http://www.nonlinear.com/progenesis). Only non-conflicting peptides were taken into account. Automatic alignment of runs to reference run was at least 84.3%. For filtering, peak picking limits were set to automatic and the maximum charge was set to 3. Normalization factor (NF) was ��������� 2 except one sample (NF 2.68). Peptides with a score < 20 and a mass error >10 ppm were excluded from quantification. A minimum of 2 unique peptides per protein was required for quantification. R was used for statistical analysis calculating differences between samples by ANOVA (FDR corrected p-value or q-value < 0.05) and Tukey������s post-hoc test (p < 0.01).

### Publication Abstract
The contribution of microRNA-mediated posttranscriptional regulation on the final proteome in differentiating cells remains elusive. Here, we evaluated the impact of microRNAs (miRNAs) on the proteome of human umbilical cord blood-derived unrestricted somatic stem cells (USSC) during retinoic acid (RA) differentiation by a systemic approach using next generation sequencing analysing mRNA and miRNA expression and quantitative mass spectrometry-based proteome analyses. Interestingly, regulation of mRNAs and their dedicated proteins highly correlated during RA-incubation. Additionally, RA-induced USSC demonstrated a clear separation from native USSC thereby shifting from a proliferating to a metabolic phenotype. Bioinformatic integration of up- and downregulated miRNAs and proteins initially implied a strong impact of the miRNome on the XXL-USSC proteome. However, quantitative proteome analysis of the miRNA contribution on the final proteome after ectopic overexpression of downregulated miR-27a-5p and miR-221-5p or inhibition of upregulated miR-34a-5p, respectively, followed by RA-induction revealed only minor proportions of differentially abundant proteins. In addition, only small overlaps of these regulated proteins with inversely abundant proteins in non-transfected RA-treated USSC were observed. Hence, mRNA transcription rather than miRNA-mediated regulation is the driving force for protein regulation upon RA-incubation, strongly suggesting that miRNAs are fine-tuning regulators rather than active primary switches during RA-induction of USSC.

### Keywords
Human, Microrna, Umbilical cord blood stem cells, Neuronal differentiation, Retinoic acid, Unrestricted somatic stem cells

### Affiliations
Heinrich Heine University Duesseldorf
Institute for Molecular Medicine I Medical Faculty Heinrich-Heine-University Duesseldorf Universitaetsstrasse 1 40225 Duesseldorf Germany

### Submitter
Jessica Schira-Heinen

### Lab Head
Dr Kai Stuehler
Institute for Molecular Medicine I Medical Faculty Heinrich-Heine-University Duesseldorf Universitaetsstrasse 1 40225 Duesseldorf Germany


