### Accession
PXD028558

### Title
APIR: a universal FDR-control framework for boosting peptide identification power by aggregating multiple proteomics database search algorithms

### Description
Advances in mass spectrometry (MS) have enabled high-throughput analysis of proteomes in biological systems. The state-of-the-art MS data analysis relies on database search algorithms to quantify proteins by identifying peptide-spectrum matches (PSMs), which convert mass spectra to peptide sequences.  Different database search algorithms use distinct search strategies and thus may identify unique PSMs.  However,  no existing approaches can aggregate all user-specified database search algorithms with guaranteed control on the false discovery rate (FDR) and guaranteed increase in the identified peptides. To fill in this gap, we propose a statistical framework, Aggregation of Peptide Identification Results (APIR), that is universally compatible with all database search algorithms.  Notably, under a target FDR threshold, APIR is guaranteed to identify at least as many, if not more, peptides as individual database search algorithms do.  Evaluation of APIR on a complex protein standard shows that APIR outpowers individual database search algorithms and guarantees the FDR control. Realdata studies show that APIR can identify disease-related proteins and post-translational modifications missed by some individual database search algorithms.  Note that the APIR framework is easily extendable to aggregating discoveries made by multiple algorithms in other high-throughput biomedical data analysis, e.g., differential gene expression analysis on RNA sequencing data.

### Sample Protocol
The  complex  proteomics  standard  (CPS)  (part  number  400510)  was  purchased  by  Agilent  (Agilent,Santa Clara, CA, USA). CPS contains soluble proteins extracted from the archaeon Pyrococcus furiosus (Pfu). All other chemicals were purchased from Sigma Aldrich (Sigma Aldrich, St. Louis, MO, USA). CPS (500ug total protein) was dissolved in 100uL of 0.5 M tri-etrhylammonium bicarbonate (TEAB) and 0.05% sodium dodecyl sulfate (SDS) solution. Proteins were reduced using tris(2-carboxyethyl)phosphine hydrochloride (TCEP) (4 uLof 50mM solution added in the protein mixture and sample incubated at 60 0 C for 1hour) and alkylated using methyl methyl methanethiosulfonate (MMTS) (2 uL of 50mM solution added in the protein mixture and sample incubated at room temperature for 15 minutes). To enzymatically digest the proteins, 20ug trypsin dissolved 1:1 in ultrapure water was added in the sample and this was incubated overnight (16 hours) in dark at 37 0 C. The tryptic peptides were cleaned with C-18 tips (part number 87784) from Thermo Fisher Scientific (Thermo Fisher Scientific, Waltham, MA, USA) following the manufacturerâ€™s instructions.  Peptides were LC-MS analysed using the Ultimate 3000 uPLC system (EASY-Spray column, part number ES803A, Thermo Fisher Scientific) hyphenated with the Orbitrap Fusion Lumos mass spectrometry instrument (Thermo Fisher Scientific).  Peptides were fragmented using low energy CID and detected with the linear ion trap detector.

### Data Protocol
We benchmarked the five database search algorithms---SEQUEST, Mascot, MaxQuant, Byonic, and MS-GF+. We first generated a reference database by concatenating the Uniprot Pyrococcus furiosus database, the Uniprot Human database, and two contaminant databases: the CRAPome, and the contaminant databases from MaxQuant. During the process, we performed in silico digestion of Pfu proteins and removed human proteins that contained Pfu peptides from the reference database. We then input the Pfu mass spectra and the resulting database into a database search algorithm.   Byonic, SEQUEST, and Mascot were each run in Proteome Discoverer 2.3.0.523 (ThermoScientific). The following settings were used for all 5 database search algorithms: 10ppm precursor tolerance; 0.6 Da fragment tolerance; static modifications:  methylthio (C); dynamic modifications: deamination (NQ), oxidation (M). Percolator was used in conjunction with both SEQUEST and Mascot, and the target decoy mode was set to separate. To acquire the total list of identified PSMs, peptides, and proteins, internal FDRs for all database search algorithms were set to 100%.  MaxQuant was implemented with the following settings:10 ppm match tolerance; 0.6 Da fragment tolerance; static modifications: methylthio (C); dynamic modifications: deamination (NQ), oxidation (M); second peptide search: True.  To acquire the total list of identified PSMs, peptides, and proteins, the internal FDR was set to 100%.  MaxQuant outputs a posterior error probability (PEP) for each target PSM and decoy PSM. MS-GF+ was implemented with the following settings: 10ppm match tolerance; static modifications: methylthio (C); dynamic modifications: deamination (NQ), oxidation (M). To acquire the total lists of identified PSMs, peptides, and proteins, the internal FDR was set to 100%.

### Publication Abstract
None

### Keywords
Complex proteomic standard, Proteomics, Pyrococcus furiosus (pfu)

### Affiliations
Departments of Pediatrics and Immuno-Oncology, Beckman Research Institute, City of Hope National Medical Center, Duarte CA 91010
University of California, Los Angeles

### Submitter
Yiling Chen

### Lab Head
Dr Leo David Wang
Departments of Pediatrics and Immuno-Oncology, Beckman Research Institute, City of Hope National Medical Center, Duarte CA 91010


