### Accession
PXD025587

### Title
Quantitative proteomics identifies Ola1p as a positive regulator of heat shock response in Saccharomyces cerevisiae

### Description
Ola1 is implicated in various stress response pathways, as well as in cancer and tumor progression. However, Ola1p functions are divergent between species and still remain poorly understood. Here, we studied the role of Ola1p in the heat shock response in Saccharomyces cerevisiae. We show that deletion of OLA1 results in higher levels of proteins with protective functions for proteome stability under heat stress. We found that Ola1p enriches in detergent-resistant protein aggregates during heat stress. In heat-stressed yeast cells, it rapidly forms assemblies that localize to stress granules. In vitro, Ola1p assembly is protein autonomous and coincides with structural rearrangements that occur at similar temperatures and pH values seen in cells. We further observed that deletion of OLA1 leads to increased ubiquitination of protein aggregates formed during heat stress. Upon stress relief, central factors for protein refolding and the clearance of protein aggregates are synthesized in higher amounts in ola1 cells, whereas global translation reinitiation is decreased in these cells. Our data suggest that Ola1p is a positive factor for stabilizing the misfolded proteome that is sequestered in cytoplasmic stress granules during heat stress and, thereby, enables cells to efficiently restore a functional proteome after stress relief.

### Sample Protocol
: For the 2nSILAC-based loss-of-function experiment (performed in 4 independent replicates), BY4741 wildtype and Δola1 cells were differentially labeled with light, medium-heavy, or heavy arginine (Arg0, Arg6, Arg10) and lysine (Lys0, Lys4, Lys8), including a label switch. Cells were differentially treated (-/+ heat shock; 42°C, 60 min) and mixed in equal amounts based on cell numbers. Mixed cells were washed with ddH2O, resuspended in 500 µl of urea buffer (8 M urea, 75 mM NaCl, 50 mM Tris-HCl pH 8.0), and disrupted by bead beating using a Minilys homogenizer three times for 2 min at 4,000 rpm with at least 2 min cooling on ice between the cycles. Cell debris was removed by centrifugation for 5 min at 15,000 x g and 4 °C.  10 µg of protein were used for proteolytic in-solution digestion. Cysteine residues were reduced with tris(2-carboxyethyl)phosphine, alkylated with iodoacetamid, and proteins were digested using LysC and trypsin as previously described (Dannenmaier et al., 2018, Anal. Chem. 90). Peptides were desalted using StageTips as described before with minor modifications (Rappsilber et al., 2003, Anal. Chem. 75). In brief, peptide mixtures were loaded onto StageTips conditioned with methanol, followed by 80% (v/v) acetonitrile (ACN)/0.5% (v/v) acetic acid and 0.5% (v/v) acetic acid. Peptides were washed twice using 0.5% (v/v) acetic acid and eluted using 80% (v/v) ACN/0.5% (v/v) acetic acid. Solvents were evaporated and peptides were resuspended in 0.1% (v/v) trifluoroacetic acid. LC/MS analysis: Nano-HPLC-ESI-MS/MS analyses were performed using an Ultimate 3000 RSLCnano system (Thermo Scientific) online coupled to a Q Exactive Plus (Thermo Scientific). The RSLC system was equipped with PepMap C18 precolumns (Thermo Scientific; length: 5 mm; inner diameter: 0.3 mm; loading flow rate: 30 µl/min) and an Acclaim PepMap analytical column (Thermo Scientific; length: 500 mm; inner diameter: 75 µm; particle size: 2 µm; packing density: 10 nm; flow rate: 0.25 µl/min). Peptides were washed and concentrated on precolumns and separated on the analytical column using a binary solvent consisting of solvent A (0.1% [v/v] formic acid [FA]) and solvent B (86% [v/v] ACN/0.1% [v/v] FA). Peptides were eluted using a gradient starting with 1% solvent B for 3 min, followed by 1-2.5% B in 12 sec, 2.5-22% B in 142 min, 22-39% B in 60 min, 39-54% B in 10 min, 54-95% B in 3 min, and 5 min at 95% B. The Q Exactive Plus was equipped with a Nanospray Flex ion source (Thermo Scientific) and fused silica emitter (PicoTip, New Objective) and was operated in data-dependent acquisition mode. Full MS scans were acquired with a mass range of m/z 375-1700, a resolution of 70,000 at m/z 200, a maximum automatic gain control (AGC) of 3 x 10E6 ions, and a maximum injection time of 60 ms. The 15 most intensive peptide ions (z ≥ 2) were selected for fragmentation by higher-energy collisional dissociation. MS/MS scans were acquired with a normalized collision energy of 28%, an AGC target of 10E5 ions, a maximum injection time of 120 ms, an isolation window of 3.0 m/z, a resolution of 35,000, and a dynamic exclusion time of 45 sec.

### Data Protocol
MaxQuant/Andromeda was used for protein identification and quantification (version 1.5.5.1) (Cox and Mann, 2008, Nat. Biotechnol. 26; Cox et al., 2011, J. Proteome Res. 4). MS/MS data were searched against a yeast-specific database obtained from the Saccharomyces Genome Database (SGD; http://www.yeastgenome.org/) and a list of common contaminants included in the MaxQuant software package. Database searches were carried out using trypsin and LysC specificity, a maximum number of two missed cleavages, and a mass tolerance of 4.5 ppm for precursor and 20 ppm for fragment ions. Oxidation of methionine and acetylation of protein N-termini were considered as variable and carbamidomethylation of cysteine residues as fixed modification. Multiplicity was set to three, and Arg10/Lys8 were set as ‘heavy’, Arg6/Lys4 as ‘medium-heavy’ and Arg0/Lys0 as ‘light’ label. The options ‘re-quantify’ and ‘match between runs’ were enabled. Protein identification was based on peptides with a minimum length of seven amino acids and at least one unique peptide. The false discovery rate for peptide and protein identification was 0.01. For 2nSILAC-based relative protein quantification, a minimum ratio count of one was required.   Mean log2 ratios of normalized SILAC ratios were calculated, and a two-sided Student's t-test was performed to determine p-values. For outlier analysis, the Perseus software (version 1.6.14) (Tyanova et al., 2016, Nat. Methods 13) was used to determine Significance B p-values (Cox and Mann, 2008, Nat. Biotechnol. 26). Proteins with a t-test p-value and a Significance B p-value of ≤ 0.05 were considered as significantly changed in abundance.

### Publication Abstract
The universally conserved P-loop ATPase Ola1 is implicated in various cellular stress response pathways, as well as in cancer and tumor progression. However, Ola1p functions are divergent between species, and the involved mechanisms are only poorly understood. Here, we studied the role of Ola1p in the heat shock response of the yeast Saccharomyces cerevisiae using a combination of quantitative and pulse labeling-based proteomics approaches, in&#xa0;vitro studies, and cell-based assays. Our data show that when heat stress is applied to cells lacking Ola1p, the expression of stress-protective proteins is enhanced. During heat stress Ola1p associates with detergent-resistant protein aggregates and rapidly forms assemblies that localize to stress granules. The assembly of Ola1p was also observed in&#xa0;vitro using purified protein and conditions, which resembled those in living cells. We show that loss of Ola1p results in increased protein ubiquitination of detergent-insoluble aggregates recovered from heat-shocked cells. When cells lacking Ola1p were subsequently relieved from heat stress, reinitiation of translation was delayed, whereas, at the same time, de novo synthesis of central factors required for protein refolding and the clearance of aggregates was enhanced when compared with wild-type cells. The combined data suggest that upon acute heat stress, Ola1p is involved in the stabilization of misfolded proteins, which become sequestered in cytoplasmic stress granules. This function of Ola1p enables cells to resume translation in a timely manner as soon as heat stress is relieved.

### Keywords
Depletome, Proteomics, Heat shock response, Saccharomyces cerevisiae, Native silac

### Affiliations
Department of Biochemistry and Functional Proteomics, Institute of Biology II, Faculty of Biology and Signalling Research Centres BIOSS and CIBSS, University of Freiburg, 79104 Freiburg im Breisgau, Germany
AG Warscheid
Biologie II
Albert-Ludwigs-Universität Freiburg
Schänzlestr. 1
79104 Freiburg
Germany


### Submitter
Friedel Drepper

### Lab Head
Dr Prof. Dr. Bettina Warscheid
Department of Biochemistry and Functional Proteomics, Institute of Biology II, Faculty of Biology and Signalling Research Centres BIOSS and CIBSS, University of Freiburg, 79104 Freiburg im Breisgau, Germany


