### Accession
PXD026569

### Title
A quantitative metabolic analysis reveals Acetobacterium woodii as a flexible and robust host for formate-based bioproduction

### Description
Cheap and renewable feedstocks such as the one carbon substrate formate are emerging for sustainable production in a growing chemical industry. By quantitatively analyzing physiology, transcriptome, proteome in chemostat cultivations in combination with computational analyses, we investigated the acetogen Acetobacterium woodii as a potential host for bioproduction from formate alone and together with autotrophic and heterotrophic co-substrates. Continuous cultivations with a specific growth rate of 0.05 h-1 on formate showed high specific substrate uptake rates (47 mmol g‑1 h‑1). Co-utilization of formate with H2, CO, CO2 or fructose was achieved without catabolite repression and with acetate as the sole metabolic product. A transcriptomic comparison of all growth conditions revealed a distinct adaptation of A. woodii to growth on formate as 570 genes were changed in their transcription. Transcriptome and proteome showed higher expression of the Wood-Ljungdahl pathway during growth on formate and gaseous substrates, underlining its function during utilization of one carbon substrates. Flux balance analysis showed varying flux levels for the WLP (0.7-16.4 mmol/g/h) and major differences in redox and energy metabolism. Growth on formate, H2/CO2, and formate+H2/CO2 resulted in low energy availability (0.20-0.22 ATP/acetate) which was increased during co-utilization with CO or fructose (0.31 ATP/acetate for formate+H2/CO/CO2, 0.75 ATP/acetate for formate+fructose). Unitrophic and mixotrophic conversion of all substrates was further characterized by high energetic efficiencies. In silico analysis of bioproduction of ethanol and lactate from formate and autotrophic and heterotrophic co-substrates showed promising energetic efficiencies (70-92%). Collectively, our findings reveal A. woodii as a promising host for flexible and simultaneous bioconversion of multiple substrates, underline the potential of substrate co-utilization to improve the energy availability of acetogens and encourage metabolic engineering of acetogenic bacteria for the efficient synthesis of bulk chemicals and fuels from sustainable one carbon substrates.

### Sample Protocol
Samples were lysed in 100 µl of the lysis buffer (100 mM Tris pH=8.6, 1 % sodium dodecyl-sulphate (SDS), 40 mM chloroacetamide and 10 mM (tris(2-carboxyethyl)phosphine) (TCEP)) followed by three cycles of sonication (15 s per cycle, 20 % amplitude). Lysates were then spun down for 5 min at 14,000 g and 100 Âµg of protein (after protein estimation) were precipitated overnight using acetone. The following day, protein pellets were re-solubilized in 50 µl of 25 % trifluoroethanol (TFE) in 100 mM Tris (pH=8.6), after which solution was diluted to 10 % TFE with 100 mM ammonium-bicarbonate and subjected to overnight digestion with trypsin (1:67 ratio protein to trypsin). Resulting peptide mixture was offline desalted, then chromatographically separated using an Ultimate 3000 RCS Nano Dionex system equipped with an Ionopticks Aurora Series UHPLC C18 column (250 mm x 75 Âµm, 1.6 Âµm) (Ionopticks, Australia). Solvent A was 0.1 % formic acid in water and solvent B acetonitrile containing 0.1 % formic acid. Total run per sample was 136.5 min with the following gradient: 0-5.5 min: 2 % B; 5.5-65.5 min: 2-17 % B; 65.5-95.5 min: 25-37 % B, 105.5-115.5 min: 37-95 % B, 115.5-125.5 min: 95 % B; 125.5-126.5 min: 95-2% B; 126.5-136.5 min: 2% B at a flow rate of 400 nl/min and 50 °C. Peptides were measured on the timsTOF Pro mass spectrometer (Bruker Daltonics, Germany) that was operated in positive mode with enabled trapped Ion Mobility Spectrometry (TIMS) at 100 % duty cycle (100 ms cycle time). Scan mode was set to parallel accumulationâ€“serial fragmentation (PASEF) for the scan range of 100 µl 1700 m/z. Source capillary voltage was set to 1500 V and dry gas flow to 3 L/min at 180 °C.

### Data Protocol
Raw data processing was carried out using MaxQuant (v1.6.17.0). Database matching was performed against a genome predicted publicly available A. woodii protein database (GCF_000247605.1_ASM24760v1; downloaded on 24.02.2021, 3546 entries). For peptide as well as protein matching, false discovery rate was set to 1 %, minimum peptide length was set to six and up to two mis-cleavages were allowed. Oxidation of methionine was set as variable and carbamidomethylation as fixed modification. Match between run feature was enabled for the match window of 1 min and alignment window of 20 min.  Resulting table of protein â€œIntensitiesâ€ was then imported to Perseus (v 1.6.14.0)3, where data was transformed, normalized (mean subtraction per column) and grouped. Matrix was then filtered to keep only those proteins with reported values in at least three replicates in at least one of the groups. Missing values were consequently imputed from normal distribution (downshift 1.8, width 3) and pairwise Student's t-tests were carried out between the groups, with or without multi-testing correction (permutation-based FDR < 5%) as indicated.

### Publication Abstract
Cheap and renewable feedstocks such as the one-carbon substrate formate are emerging for sustainable production in a growing chemical industry. We investigated the acetogen Acetobacterium woodii as a potential host for bioproduction from formate alone and together with autotrophic and heterotrophic co-substrates by quantitatively analyzing physiology, transcriptome, and proteome in chemostat cultivations in combination with computational analyses. Continuous cultivations with a specific growth rate of 0.05 h<sup>-1</sup> on formate showed high specific substrate uptake rates (47&#xa0;mmol&#xa0;g<sup>-1</sup> h<sup>-1</sup>). Co-utilization of formate with H<sub>2</sub>, CO, CO<sub>2</sub> or fructose was achieved without catabolite repression and with acetate as the sole metabolic product. A transcriptomic comparison of all growth conditions revealed a distinct adaptation of A. woodii to growth on formate as 570 genes were changed in their transcript level. Transcriptome and proteome showed higher expression of the Wood-Ljungdahl pathway during growth on formate and gaseous substrates, underlining its function during utilization of one-carbon substrates. Flux balance analysis showed varying flux levels for the WLP (0.7-16.4&#xa0;mmol&#xa0;g<sup>-1</sup>&#xa0;h<sup>-1</sup>) and major differences in redox and energy metabolism. Growth on formate, H<sub>2</sub>/CO<sub>2</sub>, and formate&#xa0;+&#xa0;H<sub>2</sub>/CO<sub>2</sub> resulted in low energy availability (0.20-0.22 ATP/acetate) which was increased during co-utilization with CO or fructose (0.31 ATP/acetate for formate&#xa0;+&#xa0;H<sub>2</sub>/CO/CO<sub>2</sub>, 0.75 ATP/acetate for formate&#xa0;+&#xa0;fructose). Unitrophic and mixotrophic conversion of all substrates was further characterized by high energetic efficiencies. In silico analysis of bioproduction of ethanol and lactate from formate and autotrophic and heterotrophic co-substrates showed promising energetic efficiencies (70-92%). Collectively, our findings reveal A. woodii as a promising host for flexible and simultaneous bioconversion of multiple substrates, underline the potential of substrate co-utilization to improve the energy availability of acetogens and encourage metabolic engineering of acetogenic bacteria for the efficient synthesis of bulk chemicals and fuels from sustainable one carbon substrates.

### Keywords
Continuous fermentation, Omics analysis, Mixed gas fermentation, Substrate co-utilization, Metabolic modelling, Acetogens, Formate bioeconomy

### Affiliations
Senior Scientist at Institute of Chemical Technologies and Analytics
Faculty of Technical Chemistry, TU Wien
Getreidemarkt 9/164, 1060 Vienna

Full Professor for Analytical Chemistry   Institute of Chemical Technologies and Analytics TU Wien Getreidemarkt 9/E164   A-1060 Vienna Austria

### Submitter
Tamara Tomin

### Lab Head
Dr Ruth Birner-Gruenberger
Full Professor for Analytical Chemistry   Institute of Chemical Technologies and Analytics TU Wien Getreidemarkt 9/E164   A-1060 Vienna Austria


