### Accession
PXD010664

### Title
Unbiased proteomic analyses of plasma and brain tissue from TBI and AD mouse models II

### Description
The relationship between repetitive mild traumatic brain injury (r-mTBI) and Alzheimer’s disease (AD) is well-recognized. However, the precise nature of how r-mTBI leads to or precipitates AD pathogenesis is currently not understood.  Part A: Plasma biomarkers potentially provide non-invasive tools for detecting neurological changes in the brain, and can reveal overlaps between long-term consequences of r-mTBI and AD. In this study we address this by generating time-dependent molecular profiles of response to r-mTBI and AD pathogenesis in mouse models using unbiased proteomic analyses. To model AD, we used the well-validated hTau and PSAPP(APP/PS1) mouse models that develop age-related tau and amyloid pathological features respectively, and our well-established model of r-mTBI in C57BL/6 mice. Plasma were collected at different ages (3, 9, and 15 months-old for hTau and PSAPP mice), encompassing pre-, peri- and post-“onset” of the cognitive and neuropathological phenotypes, or at different timepoints after r-mTBI (24hrs, 3, 6, 9 and 12 months post-injury). Liquid chromatography/mass spectrometry (LC-MS) approaches coupled with Tandem Mass Tag labeling technology were applied to develop molecular profiles of protein species that were significantly differentially expressed as a consequence of mTBI or AD. Mixed model ANOVA after Benjamini-Hochberg correction, and a stringent cut-off identified 31 proteins significantly changing in r-mTBI groups over time and, when compared with changes over time in sham mice, 13 of these were unique to the injured mice.  The canonical pathways predicted to be modulated by these changes were LXR/RXR activation, production of nitric oxide and reactive oxygen species and complement systems. We identified 18 proteins significantly changing in PSAPP mice and 19 proteins in hTau mice compared to their wildtype littermates with ageing.  Six proteins were found to be significantly regulated in all three models i.e. r-mTBI, hTau and PSAPP mice compared to their controls.  The top canonical pathways coincidently changing in all three models were LXR/RXR activation, and production of nitric oxide and reactive oxygen species. This work suggests potential biomarkers for TBI and AD pathogenesis and for the overlap between these two, and warrant targeted investigation in human populations.  Part B: In this part of the study we address the above problem by utilizing our unbiased proteomic approach to generate detailed time-dependent brain molecular profiles of response to repetitive mTBI and AD pathogenesis in established mouse models. The same animal models described above were used herein. A LC/MS approach coupled with TMT labeling was also employed. Results: Mixed model ANOVA after Benjamin Hochberg correction identified 30 and 47 proteins that were specifically unique and changing in the hippocampus and cortex, respectively, within the r-mTBI group alone when compared with changes overtime in sham mice. PI3K/AKT signaling, Protein Kinase A signaling and PPAR/RXR activation in the hippocampus, and Protein Kinase A signaling, GNRH signaling and B cell receptor signaling in the cortex were the top canonical systems significantly altered in injury groups compared to sham mice. Mixed model AONVA identified 19 proteins significantly changing in the cortex of PSAPP mice and 7 proteins in hTau mice compared to their relative wildtype littermates respectively. In addition to the heterogeneous changes observed in the TBI and AD mouse models, there was a notable convergence and coincidental change in 6 unique proteins identified in the repetitive mTBI model and the hTau and PSAPP model. These proteins ostensibly indicate significant common pathobiological responses involving alterations in mitochondrial bioenergetics and energy metabolism, aberrant cytoskeletal reorganization and alterations in intracellular signaling transduction cascades. Conclusion: We believe that this work could help identify the common molecular substrates responsible for the precipitation of AD pathogenesis following repetitive mTBI, and also help to identify novel biological targets for therapeutic modulation in mTBI and AD.

### Sample Protocol
(i) Plasma protein were extracted using an improvised selective re-suspension procedure in Hexafluoroisopropyl Alcohol (HFIPA). Briefly, 50ul plasma samples were brought up in 350l of methanol and centrifuged at 21,000rcf at room temperature for 1 minute. The pellet was re-suspended in 500l of hexane and the process of centrifugation and re-suspension was repeated with hexane, and subsequently with 100l of 58% HFIPA. The final supernatant following resuspension and centrifugation in HFIPA was collected and stored at -80oC.  (ii) Subcellular protein extraction from brain homogenates: Snap frozen brain tissue were weighed and homogenized in liquid chromatography and mass spectrometry (LC/MS) grade chilled water at a volume of 2.5fold (cortice) and 4.5fold (hippocampi) wet weight respectively. Fifty microliters of homogenates were added to a 50ul aqueous buffer solution containing 2X phosphate buffered saline (PBS), 2X sodium chloride and proteinase inhibitor. Samples in aqueous buffer were further homogenized with a probe sonicator and underwent centrifugation at 20,000g for 15 mins at 4oC. Supernatant (non-membrane fraction) was collected in a different Eppendorf tube to obtain cytosolic and nuclear protein fractions. Pelleted samples were re-suspended in an aqueous solution containing 1X volume of ice-cold methanol and centrifuged at 20,000g for 15mins at 4oC. The precipitant was further re-suspended in a 1X volume of ice-cold isopropanol and hexane at a 2:1 dilution, followed by centrifugation at 20,000g for 15mins, and membrane protein pellet fractions were re-suspended in 25mM TEAB and 0.5% SDC. Original supernatants (non-membrane fractions) from the first centrifugation step were brought up in 0.5fold volume of ice-cold methanol, centrifuged at 20,000g for 15mins, and the nuclear protein pellet fraction were re-suspended in 25mM TEAB and 0.5% SDC. Supernatant fractions from the last step was brought up in 7fold volume of ice-cold methanol, centrifuged at 20,000g for 15mins, and cytosolic protein pellets were re-suspended in 25mM TEAB and 0.5% SDC. Membrane and cytosolic proteins were used in the entire study.  (iii) Following BCA analysis and validation of protein separation using sypro-red and comassie staining for total protein after polyacrylamide gel electrophoresis, 50ug of hippocampal tissue fractions and 100ug of cortical tissue fractions underwent trypsin digestion at a 1:100 enzymatic concentration.    (iv) We employed a multiplexed isobaric labeling strategy to allow for simultaneous identification and quantification of proteins from multiple biological samples.   All three time points  (3, 9, 15 months) in the AD mouse models (disease vs. control) were included in the six-plex TMT kit. While all five time points (24hrs, 3, 6, 9, 12 months post-injury) in the TBI mouse model (r-sham vs r-TBI) were included in the 10 plex TMT kit.   (v) Traces of SDC and TEAB were removed from the samples by resuspension in ethyl acetate to separate the different phases. Final lower phase was taken to dryness in the speed vacuum, and re-suspended in 100ul of 0.1% formic acid for purification and concentration of peptides by Zip-tiping. Samples were dried and re-suspended in 20ul of 0.1% formic acid and transferred into an auto-sampler vial, and analyzed by nano-UPLC MS on a Q-Exactive Orbitrap instrument (Thermo).  (vi) DDA settings for the experiments were as follows: full-scan MS resolution = 140 000 full width at half maximum at 200 m/z, full-scan range = 380–1250 m/z, isolation width = 1.2 m/z, higher energy C-trap dissociation relative collision energy = 29, a minimum m/z setting of 100 m/z was used for all MS2 spectra, MS2 resolution = 17 500, dynamic exclusion = 180 s, and a Top 15 high/low duty cycle was used for precursor ion selection. These settings, essentially the narrow isolation window and the ultra-long gradient, were used to minimize the deleterious effects on quantitative accuracy that result from co-isolation of isobaric precursors without resorting to MS3-based methods.

### Data Protocol
(i) Data Processing and Statistical Analysis of proteomics data: The animals were assigned to multiplexes and the labels randomly and analyzed blindly to prevent any unwanted experimental bias. Samples were analyzed by LC-MS/MS (Q-Exactive). PMi Preview software was used to survey the data files and if necessary add other modifications to the search criteria. Also, Preview results were used to choose the precursor and fragment ion mass tolerances (4 ppm, 0.02 Da, respectively) as well as dynamic modifications. The following settings were used to search the data using SEQUEST and MSAMANDA as the search algorithms; dynamic modifications; Oxidation / +15.995 Da (M), Deamidated / +0.984 Da (N, Q), static modifications of TMT6plex / +229.163 Da (N-Terminus, K), Carbamidomethyl +57.021 (C).  Only unique peptides were considered for quantification purposes. Percolator feature of Proteome Discoverer was used to set a false discovery rate (FDR) of 0.01 and peptides passing this cutoff value were exported to JMP (SAS) 8.0.2 for data cleaning and statistical analysis. Proteins identified in at least half of the total number of plexes were used for the quantitative analysis. After ln transformation of the raw ion counts, each channel was normalized by central tendency normalization where medians were used. The ratios were formed by dividing each timepoint (3m, 6m, 9m, or 12m)  with 24h timepoint within the TBI and Sham group separately per  plex. Peptides that were not quantified in at least 8 runs were omitted from the analysis. Mixed model ANOVA was used to analyze the data. The following linear model (1) was used per injury/genotype in animal models. yijklmn=μ+ ai+ bj + ck(j)+dl+pm+eijklm (1) where: μ is the population mean; aj is the random variable mouse pair ~NID(0,σ2b); bl is the random variable protein incomplete block ~NID(0, σ2b); cm(l) is the random variable peptide incomplete block ~NID(0, σ2c); dl is the fixed variable time type~NID; pm is the random variable TMT plex no. (0,σ2p);, eijklm is the random variable error within the experiment ~NID(0, σ2e). The FDR multiple testing correction as per Benjamini & Hochberg (B-H) was applied for most stringent analyses to identify a “top tier” of significant proteins and prevent identification of false positives.   (ii) The analytical approach: The data set was analyzed using two different approaches. First approach was to divide values from each of the TBI animal with a randomly matched Sham animal at a given timepoint to measure the response to TBI at each time point. However, since each TBI animal was being directly compared to a random Sham animal instead of all TBI or Sham animals being compared to a single reference animal, statistical noise was introduced to data. Moreover this type of ratio (TBI/Sham), can only be analyzed using a method that tests if the ratio is significantly different from 0 (since the data is ln transformed). There was either none or very small number of proteins that were changing due to TBI using this statistical analysis approach.  In order to reduce statistical noise and increase the power, 24H time point was chosen as reference and TBI and Sham animals were analyzed separately using mixed model ANOVA method which is more powerful than one sample t-test. Finding proteins that are changing over time separately in TBI and Sham groups, enabled upload of these lists on IPA where molecules and pathways that are unique to one groups can be identified. We confirmed the validity of this approach (using a reference sample) in other proteomic experiments were transgenic mouse models were used and significant protein lists included the transgene product.

### Publication Abstract
The relationship between repetitive mild traumatic brain injury (r-mTBI) and Alzheimer's disease (AD) is well-recognized. However, the precise nature of how r-mTBI leads to or precipitates AD pathogenesis is currently not understood. Plasma biomarkers potentially provide non-invasive tools for detecting neurological changes in the brain, and can reveal overlaps between long-term consequences of r-mTBI and AD. In this study we address this by generating time-dependent molecular profiles of response to r-mTBI and AD pathogenesis in mouse models using unbiased proteomic analyses. To model AD, we used the well-validated hTau and PSAPP(APP/PS1) mouse models that develop age-related tau and amyloid pathological features, respectively, and our well-established model of r-mTBI in C57BL/6 mice. Plasma were collected at different ages (3, 9, and 15 months-old for hTau and PSAPP mice), encompassing pre-, peri- and post-"onset" of the cognitive and neuropathological phenotypes, or at different timepoints after r-mTBI (24 h, 3, 6, 9, and 12 months post-injury). Liquid chromatography/mass spectrometry (LC-MS) approaches coupled with Tandem Mass Tag labeling technology were applied to develop molecular profiles of protein species that were significantly differentially expressed as a consequence of mTBI or AD. Mixed model ANOVA after Benjamini-Hochberg correction, and a stringent cut-off identified 31 proteins significantly changing in r-mTBI groups over time and, when compared with changes over time in sham mice, 13 of these were unique to the injured mice. The canonical pathways predicted to be modulated by these changes were LXR/RXR activation, production of nitric oxide and reactive oxygen species and complement systems. We identified 18 proteins significantly changing in PSAPP mice and 19 proteins in hTau mice compared to their wild-type littermates with aging. Six proteins were found to be significantly regulated in all three models, i.e., r-mTBI, hTau, and PSAPP mice compared to their controls. The top canonical pathways coincidently changing in all three models were LXR/RXR activation, and production of nitric oxide and reactive oxygen species. This work suggests potential biomarkers for TBI and AD pathogenesis and for the overlap between these two, and warrant targeted investigation in human populations. Data are available via ProteomeXchange with identifier PXD010664.

### Keywords
Htau, Alzheimer’s disease, Plasma biomarker, Proteomics, Brain profiles, Ps1/app, Mild-tbi

### Affiliations
Roskamp Institute
Traumatic Brain Injury Research, Roskamp Institute, Sarasota, FL, USA

### Submitter
Joseph Ojo

### Lab Head
Dr Joseph Ojo
Traumatic Brain Injury Research, Roskamp Institute, Sarasota, FL, USA


