### Accession
PXD013677

### Title
A safe alternative to H37Rv: the attenuated double auxotroph Mycobacterium tuberculosis ΔleuDΔpanCD

### Description
Currently available model organisms such as Mycobacterium smegmatis and Mycobacterium bovis Bacillus Calmette-Guérin (BCG) have significantly contributed to our understanding of tuberculosis (TB) biology, these models have limitations such as differences in genome size, growth rates and virulence. Attenuated Mycobacterium tuberculosis strains may provide more representative, safer models to study M. tuberculosis biology. For example, the M. tuberculosis ΔleuDΔpanCD double auxotroph, has undergone rigorous in vitro and in vivo safety testing. Like other auxotrophic strains, this has subsequently been approved for use in biosafety level (BSL) 2 facilities. Auxotrophic strains have been assessed as models for drug-resistant M. tuberculosis and for studying latent TB. These offer the potential as safe and useful models, but it is important to understand how well these recapitulate salient features of non-attenuated M. tuberculosis. We therefore performed a comprehensive comparison of M. tuberculosis H37Rv and M. tuberculosis ΔleuDΔpanCD. These strains demonstrated similar in vitro and intra-macrophage replication rates, similar responses to anti-TB agents and whole genome sequence conservation. Shotgun proteomics analysis suggested that M. tuberculosis ΔleuDΔpanCD has an increased propensity to enter a dormant state during acid stress, which has been verified using a dual-fluorescent replication reporter assay. Importantly, infection of human peripheral blood mononuclear cells with the 2 strains elicited comparable cytokine production, demonstrating the suitability of M. tuberculosis ΔleuDΔpanCD for immunological assays. We provide comprehensive evidence to support the judicious use of M. tuberculosis ΔleuDΔpanCD as a safe and suitable model organism for M. tuberculosis research, without the need for a BSL3 facility.

### Sample Protocol
Protein extraction Mycobacterial cells from 25 ml culture were suspended in 300 µl lysis buffer consisting of 8 M urea, 50 mM triethylammonium bicarbonate (TEAB), 5 mM tris (2-carboxyethyl)phosphine) (TCEP) and containing EDTA-free protease inhibitor cocktail (Roche). Cells were disrupted by mechanical bead-beating, using glass beads (425-600 μm, Sigma-Aldrich) and a Ribolyser (Bio101 SAVANT, Vista, CA). Lysis steps consisted of 8 cycles of 20 s each, at a speed of 4.0 m.s−1. Samples were cooled on ice for 1 minute between each lysis step. Whole-cell lysates were centrifuged (12 000 g, 10 minutes), the supernatants transferred to a new microfuge tube and centrifuged once more (12 000 g, 10 minutes) to remove all cellular debris. Whole-cell lysates were filter-sterilized using a 0.22 µm PVDF membrane before removal from the BSL3 facility. Protein concentration was determined using a Bradford method (Bio-Rad).  Filter-aided sample preparation and trypsin digestion  Proteomic samples were processed using a modified version of the filter-aided sample preparation (FASP)(34). Briefly, 40 µg protein from each replicate was reduced with 5 mM TCEP at room temperature for 1 hour and subsequently alkylated with 5.5 mM iodoacetamide for 1 hour in the dark. Reduced and alkylated protein samples were transferred to a 0.5 ml Amicon ultra 30 kDa cut-off spin filter (Millipore) and centrifuged at 12 000 g for 15 minutes. Four hundred microlitres UB buffer (8 M urea in 50 mM TEAB) was added to each filter, and the samples were centrifuged (12 000 g, 15 minutes). This process was repeated twice for a total of three UB washes. Four hundred microlitres 50 mM TEAB was added to each filter, and the samples were centrifuged (12 000 g, 15 minutes). This was repeated twice for a total of three TEAB washes. Proteins were on-filter digested for 24 h at 37 °C, using a 1:50 ratio of trypsin to protein. The filter unit was placed in a new collection tube after digestion, and the peptides were obtained in the flow-through by centrifugation (12 000 g, 15 minutes). Peptides were eluted once more from the filter unit by the addition of 250 μl 100mM NaCl and further centrifugation (12 000 g, 15 minutes). Peptide samples were desalted using in-house packed STAGE-tips, prepared with Empore™ C18 SPE disks (Sigma-Aldrich), and dried before storage at -20 °C.  Mass spectrometry  A total of 1 μg peptide mixture from each sample was analysed, independently, on an Orbitrap Fusion Tribrid mass spectrometer (Thermo Fisher Scientific), connected to a Thermo Scientific UltiMate 3000 RSLCnano System (Thermo Fisher Scientific). Peptide separation was performed on a PepMap 300 C18 LC pre-column (300 μm ID x 5 mm, 5 μm, 300 Å), followed by separation on an analytical column (75 μm ID x 350 mm) packed with C18 Aeris Peptide 3.6 μm beads (Phenomenex 04A-4507), at a flow rate of 300 nL/min. Solvent A was 2% acetonitrile in 0.1% formic acid, and solvent B was 100% acetonitrile in 0.1% formic acid. The gradient used was as follows: solvent B was maintained at 2% for 10 minutes, followed by an increase from 2 to 10% B in 5 minutes, 10-35% B in 70 minutes, 35-50% B in 10 minutes, 50-80% B in 0.1 minutes, and maintained at 80% B for 10 minutes. The Orbitrap Fusion was operated in positive ion data-dependent mode for Orbitrap-MS and Orbitrap-MS2 data acquisition. Data were acquired using the Xcalibur software package. The precursor ion scan (full scan) was performed in the Orbitrap in the range of 350-1500 m/z with a nominal resolution of 120 000. The number of accumulated ions was set to 4 x 105. Ion filtering for Orbitrap-MS2 data acquisition was performed using the quadrupole with a transmission window of 1.5 m/z. The most intense ions above an intensity threshold of 5 × 103 were selected for high-energy collisional dissociation (HCD). An HCD normalized collision energy of 32.5% was applied to the most intense ions, and fragment ions were analysed in the Orbitrap at a resolution of 30 000. The number of Orbitrap-MS2 events between full scans was determined on-the-fly to maintain a 3s fixed duty cycle. Dynamic exclusion of ions within a ±10 p.p.m. m/z window was implemented using a 30 s exclusion duration. An electrospray voltage of 2.0 kV and capillary temperature of 280 °C, with no sheath and auxiliary gas flow, was used. The automatic gain control (AGC) settings were 4 × 105 ions with a maximum ion accumulation time of 50 ms for Oribtrap-MS, and 5 × 104 ions with a maximum ion accumulation time of 50 ms for Orbitrap-MS2 scans, respectively. Ions with <2+ or undetermined charge state were excluded from MS2 selection.

### Data Protocol
All tandem mass spectra were analysed using MaxQuant 1.5.5.1, and searched against a customized M. tuberculosis proteome database. Custom database construction was performed as previously described (Heunis et al. 2017, PMID: 28820946). Briefly, proteins containing single amino acid variants, derived from WGS analysis, were concatenated to the M. tuberculosis reference proteome (downloaded from Uniprot on 26 September 2017, containing 3993 entries). The custom M. tuberculosis database contained 4012 entries. Peak list generation was performed within MaxQuant and searches were performed using default parameters in MaxQuant. An initial search precursor mass tolerance of 20 ppm and a fragment mass tolerance of 0.5 Da was employed. The enzyme specificity was set to consider fully tryptic peptides, and two missed cleavages were allowed. Carbamidomethylation of cysteines was set as fixed modification, and oxidation of methionine and N-terminal acetylation was allowed as variable modifications. Proteins were considered confidently identified when they contained at least two unique tryptic peptides, employing a protein and peptide false discovery rate (FDR) of less than 1% in MaxQuant. Proteins that contained similar peptides and that could not be differentiated based on tandem mass spectrometry analysis alone were grouped to satisfy the principles of parsimony. Reverse hits, contaminants and proteins only identified by site were removed before downstream statistical and bioinformatics analysis. A label-free quantification strategy was employed using the MaxLFQ algorithm within MaxQuant. Assigned LFQ values were used for statistical analysis in Perseus 1.6.0.7. Data were transformed (log 2) and filtered to contain at least three valid LFQ values in one group for pair-wise comparisons. Missing values were imputed using random numbers drawn from a normal distribution that simulates signals from low abundance proteins. Welch’s t-tests were performed, and p-values were corrected for multiple hypothesis testing using the Benjamini-Hochberg FDR method.

### Publication Abstract
Although currently available model organisms such as <i>Mycobacterium smegmatis</i> and <i>Mycobacterium bovis</i> Bacillus Calmette-Gu&#xe9;rin (BCG) have significantly contributed to our understanding of tuberculosis (TB) biology, these models have limitations such as differences in genome size, growth rates and virulence. However, attenuated <i>Mycobacterium tuberculosis</i> strains may provide more representative, safer models to study <i>M. tuberculosis</i> biology. For example, the <i>M. tuberculosis</i> &#x394;<i>leuD</i>&#x394;<i>panCD</i> double auxotroph, has undergone rigorous <i>in vitro</i> and <i>in vivo</i> safety testing. Like other auxotrophic strains, this has subsequently been approved for use in biosafety level (BSL) 2 facilities. Auxotrophic strains have been assessed as models for drug-resistant <i>M. tuberculosis</i> and for studying latent TB. These offer the potential as safe and useful models, but it is important to understand how well these recapitulate salient features of non-attenuated <i>M. tuberculosis.</i> We therefore performed a comprehensive comparison of <i>M. tuberculosis</i> H37Rv and <i>M. tuberculosis</i>&#x394;<i>leuD</i>&#x394;<i>panCD</i>. These strains demonstrated similar <i>in vitro</i> and intra-macrophage replication rates, similar responses to anti-TB agents and whole genome sequence conservation. Shotgun proteomics analysis suggested that <i>M. tuberculosis</i>&#x394;<i>leuD</i>&#x394;<i>panCD</i> has a heightened stress response that leads to reduced bacterial replication during exposure to acid stress, which has been verified using a dual-fluorescent replication reporter assay. Importantly, infection of human peripheral blood mononuclear cells with the 2 strains elicited comparable cytokine production, demonstrating the suitability of <i>M. tuberculosis</i>&#x394;<i>leuD</i>&#x394;<i>panCD</i> for immunological assays. We provide comprehensive evidence to support the judicious use of <i>M. tuberculosis</i>&#x394;<i>leuD</i>&#x394;<i>panCD</i> as a safe and suitable model organism for <i>M. tuberculosis</i> research, without the need for a BSL3 facility.

### Keywords
Mycobacterium tuberculosis, Biosafety level 2, Attenuated auxotroph, Model organism

### Affiliations
DST/NRF Centre of Excellence for Biomedical Tuberculosis Research/South African Medical Research Council Centre for Tuberculosis Research, Division of Molecular Biology and Human Genetics, Faculty of Medicine and Health Sciences, Stellenbosch University, Cape Town, South Africa
Division of Molecular Biology and Human Genetics, Faculty of Medicine and Health Sciences, Stellenbosch University, Cape Town

### Submitter
Samantha Sampson

### Lab Head
Dr Samantha L. Sampson
DST/NRF Centre of Excellence for Biomedical Tuberculosis Research/South African Medical Research Council Centre for Tuberculosis Research, Division of Molecular Biology and Human Genetics, Faculty of Medicine and Health Sciences, Stellenbosch University, Cape Town, South Africa


