### Accession
PXD013039

### Title
Circadian analysis of the mouse cerebellum

### Description
The cerebellum contains a circadian clock, generating internal temporal signals. The daily oscillations of cerebellar proteins were investigated here in mice using large-scale two-dimensional differences in gel electrophoresis (2D-DIGE). Analysis of 2D-DIGE gels highlighted the rhythmic variation in the intensity of 27/588 protein spots (5%) over 24-h based on cosinor regression. Notably, rhythmic expression of most abundant cerebellar proteins was clustered in two main phases (i.e., midday and midnight), leading to bimodal distribution. Only 6 proteins identified here to be rhythmic in the cerebellum are also known to oscillate in the suprachiasmatic nuclei, including 2 proteins involved in synapse activity (Synapsin 2 (Syn2) and vesicle-fusing ATPase (Nsf)), 2 others participating in carbohydrate metabolism (triosephosphate isomerase (Tpi1) and alpha-enolase (Eno1)), and Glutamine synthetase (Glul), as well as Tubulin alpha (Tuba4a). Most oscillating cerebellar proteins were not previously identified in circadian proteomic analyses of any tissue. Strikingly, daily accumulation of mitochondrial proteins was clustered to the mid-resting phase, as previously observed for distinct mitochondrial proteins in the liver. This study highlights extensive rhythmic aspects of the cerebellar proteome.

### Sample Protocol
The cerebellum of mice was collected at 5AM, 11AM, 17PM, and 23PM, immediately snap-frozen in liquid nitrogen, then stored at -80°C until analysis. Half of each frozen cerebellum was sliced thinly on ice and incubated for 1h30 under agitation in a denaturing buffer composed of 8 M urea, 2 M thiourea, 4% Chaps, 1% DTT, 0,5% Triton X-100, 0,005% TLCK, and protease inhibitors at 0.02 to 2 mM.  After sonication on ice (10-30 sec., 135 Watts), followed by centrifugation (5 min, 12000g, 4°C), proteins were TCA-precipitated, then vacuum-dried using a Speedvac (ThermoFisher Scientific; Rockford, IL, USA), and dissolved in a solution composed of 7 M Urea, 2 M Thiourea, 30 mM Tris (pH 8.5) and 4% Chaps. Homogenization was completed by sonication on ice (10 sec, 135 watts). Protein concentrations were determined using the Bio-Rad Protein Assay (BioRad, Hercules, CA, USA). Protein samples were labelled using a CyDye DIGE Fluor Minimal Dye Labeling Kit (GE HealthCare, Uppsala, Sweden) using 400 pmol of Cy3 and Cy5 were used to randomly label 50 μg of protein samples from the different groups, and 7.2 nmol of Cy2 were used to label a mixture of all the samples (equal amounts from each protein extracts = internal standard). After incubation in the dark for 30 min on ice, protein labelling was quenched by addition of 10 mM lysine and incubation in the dark for 10 min on ice. Samples were then randomly multiplexed and diluted with 7 M urea, 2 M thiourea, 2% Chaps, 2% DTT, 2% ampholytes (SERVALYT Carrier Ampholytes 3-10, Serva, Heidelberg, Germany), and a trace of bromophenol blue to a total volume of 400 μL. After isoelectric focusing (24 cm pH3-10 non-linear IPG Ready strip, Biorad, Hercules, CA, USA), and SDS-Page (precast 2D HPE large gel NF 12.5% kits; HPE Flap Top Tower; Gel company, San Francisco, CA, USA), gel images were acquired using an Ettan DIGE Imager (GE HealthCare) at 100 μm resolution and analysed using the Progenesis Samespots analysis software (v4.5; Nonlinear Dynamics). After excision of gel spots, in-gel reduction, alkylation, and trypsin digestion, the extracted peptides were analyzed on a nanoUPLC (NanoAcquityUPLC, Waters, US) coupled to a hybrid mass spectrometer, either the MaXis Q-Tof (Bruker Daltonics) or Synapt HDMS G1 Q-Tof (Waters, Milford, MA) equipped with a Z-spray ion source and a lock mass system. The chromatographic solvent system consisted of 0.1% HCOOH in water (solvent A) and 0.1% formic acid in acetonitrile (solvent B). Three µl of each sample  were concentrated/desalted on a trap column (C18, 180μm x 20mm, 5 µm; Waters) at 1% of B at a flow rate of 5 µl/min for 3 min. Afterwards, peptides were eluted from the separation column (BEH130 C18, 75 μm x 250 mm, 1.7 μm; Waters) maintained at 60°C using a 9 min gradient from 1-35% of B at a flow rate of 450 nL/min. Mass spectrometers were operated in positive mode, with automatic switching between MS and MS/MS scans. For the Maxis, external mass calibration of the Tof (MaXis) was achieved before each set of analyses using Tuning Mix (Agilent Technologies, Paolo Alto, USA) in the mass range of 322-2722m/z, and mass correction was achieved by recalibration of acquired spectra to the applied lock mass using hexakis (2,2,3,3,-tetrafluoropropoxy)phosphazine ([M+H]+ 922.0098m/z). Up to 5 most intense multiply charged precursors per MS scan were isolated, using an isolation window adapted to the isolated m/z (2-5m/z), then fragmented using energy collisional dissociation. This system was fully controlled by compass HyStar v3.2 and OtofControl v3.2 (Bruker Daltonics, GE). For the Synapt G1, the TOF was calibrated using Glu-fibrino-peptide B on the 50−2000 m/z range. Online correction of this calibration was performed with Glu-fibrinopeptide B as the lock-mass: the ion (M+2H)2+ at m/z 785.8426 was used to calibrate MS data and the ion (M+H)+ at m/z 684.3469 was used to calibrate MS/MS data. This was done via a lock spray interface, with a lock spray frequency set at 30 s. The three most abundant peptides (intensity threshold: 40 count·sec−1), preferably doubly and triply charged ions, were selected on each MS spectrum for further isolation. The system was fully controlled by the MassLynx software (v.4.1., Waters).

### Data Protocol
MS/MS data were analysed using the MascotTM search engine (v2.5.1, Matrix Science, London, UK) installed on a local server. Spectra were searched against a target-decoy version of the Mus Musculus protein database downloaded from Swissprot (December 2017, 16872 target + decoy entries) to which common contaminants (e.g. trypsin and keratins) were automatically added using the MSDA software suite [31]. Mascot search parameters included MS and MS/MS tolerances of respectively 10 ppm and 0.05 Da (MaXis) or 25 ppm and 0.07 Da (Synapt). A maximum of one missed cleavage was allowed, and carbamidomethylation of cysteine residues was set as fixed modification, and oxidation of methionine residues as variable modification.   Stringent filtering criteria were applied using Proline software (v1.5; http://proline.profiproteomics.fr/) to obtain high confidence identifications (Mascot peptide ion score > 25; peptide FDR < 1% based on e-values, protein FDR < 1% base on PSM scores, minimum length of seven amino acids). Afterwards, single peptide-based identifications and the identification of common contaminants such as keratin and trypsin, were not considered.

### Publication Abstract
The cerebellum contains a circadian clock, generating internal temporal signals. The daily oscillations of cerebellar proteins were investigated in mice using a large-scale two-dimensional difference in gel electrophoresis (2D-DIGE). Analysis of 2D-DIGE gels highlighted the rhythmic variation in the intensity of 27/588 protein spots (5%) over 24 h based on cosinor regression. Notably, the rhythmic expression of most abundant cerebellar proteins was clustered in two main phases (i.e., midday and midnight), leading to bimodal distribution. Only six proteins identified here to be rhythmic in the cerebellum are also known to oscillate in the suprachiasmatic nuclei, including two proteins involved in the synapse activity (Synapsin 2 [SYN2] and vesicle-fusing ATPase [NSF]), two others participating in carbohydrate metabolism (triosephosphate isomerase (TPI1] and alpha-enolase [ENO1]), Glutamine synthetase (GLUL), as well as Tubulin alpha (TUBA4A). Most oscillating cerebellar proteins were not previously identified in circadian proteomic analyses of any tissue. Strikingly, the daily accumulation of mitochondrial proteins was clustered to the mid-resting phase, as previously observed for distinct mitochondrial proteins in the liver. Moreover, a number of rhythmic proteins, such as SYN2, NSF and TPI1, were associated with non-rhythmic mRNAs, indicating widespread post-transcriptional control in cerebellar oscillations. Thus, this study highlights extensive rhythmic aspects of the cerebellar proteome.

### Keywords
Cerebellum, Lc-msms, Mouse, Circadian rhythm

### Affiliations
CNRS
Laboratoire de Spectrométrie de Masse BioOrganique, Analytical Science department, IPHC, CNRS, Strasbourg University, UMR7178, Strasbourg, france

### Submitter
Fabrice BERTILE

### Lab Head
Dr Sarah Cianférani
Laboratoire de Spectrométrie de Masse BioOrganique, Analytical Science department, IPHC, CNRS, Strasbourg University, UMR7178, Strasbourg, france


