### Accession
PXD011723

### Title
MS-rescue: A Computational Pipeline to Increase the Quality and Yield of Immunopeptidomics Experiments

### Description
We propose a new pipeline for the refinement of spectrum-peptide assignment, designed specifically for MHC ligand identification. By modeling the peptidome as a collection of a limited number of specificities, corresponding to the MHC alleles of the cell line, our method achieves increased sequencing depth, while at the same time removing potential experimental outliers and contaminants.

### Sample Protocol
Jurkat cells were resuspended in 1ml 2x lysis buffer per 108 cells. Lysates were cleared by subsequent centrifugation steps at 500 g for 10 min and then 20,000 g for 60 min. 1 mg per sample of anti-HLA class I antibody (W6/32, ATCC HB-95) was bound and cross-linked to 1 ml Protein A beads (GE healthcare) and used for immunoprecipitation of HLA complexes. In brief, lysates were incubated with the antibody beads over night at 4ºC in 1x lysis buffer and washed subsequently with 10 bed volumes of 50 mM Tris, pH 8.0 containing either 150 mM, 450 mM and finally no salt. Peptides were eluted with 5 ml of 10% acetic acid. Dried peptides were suspended and injected onto a 4.6 x 50 mm ProSwift RP-1S column (Thermo Fisher Scientific). Peptides were separated from larger complex components by elution using a 500 μl/min flow rate over 10 min from 2-34% ACN in 0.1% TFA. Alternate fractions that did not contain the beta-2-microglobulin were pooled and two final fractions were analyzed by nUPLC-MS2.

### Data Protocol
Analysis of raw data was performed using Peaks 8.5 software (Bioinformatics Solutions). Sequence interpretation of MS2 spectra was carried out using a database comprising all SwissProt Homo sapiens entries downloaded on  18/5/2017; 42,221 entries. Peptides with a length of less than 7 amino acids were excluded from the analysis results and peptides with a Peaks score of below 5 were ignored.

### Publication Abstract
LC-MS/MS has become the standard platform for the characterization of immunopeptidomes, the collection of peptides naturally presented by major histocompatibility complex molecules to the cell surface. The protocols and algorithms used for immunopeptidomics data analysis are based on tools developed for traditional bottom-up proteomics that address the identification of peptides generated by tryptic digestion. Such algorithms are generally not tailored to the specific requirements of MHC ligand identification and, as a consequence, immunopeptidomics datasets suffer from dismissal of informative spectral information and high false discovery rates. Here, a new pipeline for the refinement of peptide-spectrum matches (PSM) is proposed, based on the assumption that immunopeptidomes contain a limited number of recurring peptide motifs, corresponding to MHC specificities. Sequence motifs are learned directly from the individual peptidome by training a prediction model on high-confidence PSMs. The model is then applied to PSM candidates with lower confidence, and sequences that score significantly higher than random peptides are rescued as likely true ligands. The pipeline is applied to MHC class I immunopeptidomes from three different species, and it is shown that it can increase the number of identified ligands by up to 20-30%, while effectively removing false positives and products of co-precipitation. Spectral validation using synthetic peptides confirms the identity of a large proportion of rescued ligands in the experimental peptidome.

### Keywords
Immunopeptidome; hla; lc-ms/ms; netmhcpan; binding prediction

### Affiliations
University of Oxford
The Jenner Institute Nuffield Department of Medicine University of Oxford

### Submitter
Nicola Ternette

### Lab Head
Dr Nicola Ternette
The Jenner Institute Nuffield Department of Medicine University of Oxford


