### Accession
PXD017838

### Title
A DOT1B/RNaseH2 protein complex is involved in R-loop processing, genomic integrity and antigenic variation in Trypanosoma brucei

### Description
The parasite Trypanosoma brucei periodically changes the expression of its protective variant surface glycoproteins (VSGs) to evade the hostâ€™s immune system in a process known as antigenic variation. One route to change VSG expression is by transcriptional activation of a previously silent VSG expression site (ES), a subtelomeric region containing the VSG genes. Homologous recombination of a different VSG from a large reservoir into the active ES represents another route. The conserved histone methyltransferase DOT1B is involved in transcriptional silencing of inactive ES and influences ES switching kinetics. The molecular machinery that enables DOT1B to execute these regulatory functions remains elusive, however. To better understand DOT1B-mediated regulatory processes, we purified DOT1B-associated proteins using complementary biochemical approaches. We identified several novel DOT1B interactors. One of these was the trimeric Ribonuclease H2 complex, previously shown to resolve RNA-DNA hybrids, maintain genome integrity, and play a role in antigenic variation. Our study revealed that DOT1B depletion results in an increase in RNA-DNA hybrids, accumulation of DNA damage and recombination-based ES switching events. Surprisingly, the observed VSG deregulation was similar to Ribonuclease H2 mutants. We here propose that  the epigenetic regulator DOT1B forms a complex with RNase H2 that acts in resolving R-loops to ensure genome integrity and contribute to the tightly regulated process of antigenic variation.

### Sample Protocol
Samples were run on a Novex Bis-Tris 4-12% gradient gel (Thermo Fisher) with MOPS buffer (Thermo Fisher) for 10 min at 180V. The gel was stained with commassie blue G250 dye (Biozym). Each lane was cut into pieces, minced and destained in water/50% EtOH. The gel pieces were dehydrated with pure ACN, reduced with 10 mM DTT (Sigma Aldrich) and subsequently alkylated with 55 mM iodoacetamide in the dark. The again dried gel pieces were digested with 1 umol trypsin at 37 degC overnight. The digested peptides were desalted and stored on StageTips (Rappsilber et al., 2008) until measurement.  Peptides were separated along a 240 min gradient (90 min for VSG analysis) on a EasyLC 1000 uHPLC system using a C18 reverse phase column, packed in-house with Reprosil C18 (Dr. Maisch GmbH).  The column was enclosed into a column oven (Sonation) and the peptides were sprayed into a Q Exactive Plus mass spectrometer (Thermo Fisher). The mass spectrometer was operated in a data-dependent acquisition mode using a top10 method. Spray voltage was set at ca. 2.4 kV.

### Data Protocol
The aquired raw files were analyzed with MaxQuant (version 1.5.8.2) (Cox and Mann, 2008) using the Trypanosoma brucei protein database downloaded from tritrypDB. Prior to bioinformatics analysis, contaminants, reverse hits, proteingroups only identified by site and proteingroups with less than 2 peptides (one of them unique) were removed.

### Publication Abstract
None

### Keywords
Lc ms-ms, Trypanosomes

### Affiliations
Quantitative Proteomics
Institute of Molecular Biology (IMB)
Institute of Molecular Biology Ackermannweg 4 55128 Mainz

### Submitter
F Butter

### Lab Head
Dr Falk Butter
Institute of Molecular Biology Ackermannweg 4 55128 Mainz


