### Accession
PXD016825

### Title
NlpI scaffolds hydrolases within peptidoglycan multi-enzyme complexes in Escherichia coli

### Description
We performed enrichment of nlpI to study its function in Escherichia coli.

### Sample Protocol
Affinity chromatography This protocol was adapted from a previously published method (Vollmer et al., 1999). Sepharose beads were activated following the instructions of the manufacturer (GE).  Coupling of 2 mg of protein to 0.13 g of activated sepharose beads was carried out overnight at 6Â°C with gentle agitation in protein buffer. After washing the beads with protein buffer, the remaining coupling sites were blocked by incubation in AC blocking buffer (200 mM Tris/HCl, 10 mM MgCl2, 500 mM NaCl, 10% glycerol and 0.25% Triton X-100, pH 7.4) with gentle agitation over-night at 6Â°C. The beads were washed alternatingly with AC blocking buffer and AC acetate buffer (100 mM sodium acetate, 10 mM MgCl2, 500 mM NaCl, 10% glycerol and 0.25% Triton X-100, pH 4.8), and finally re-suspended in AC buffer I (10 mM Tris/maleate, 10 mM MgCl2, 50 mM NaCl, 1% Triton X-100, pH 6.8). As control (Tris-Sepharose) one batch of activated Sepharose beads was treated identically with the exception that no protein was added. Affinity chromatography was performed at 6Â°C. E. coli membrane fraction extracted out of 2 L per sample (see above) containing 50 mM NaCl (or 400 mM NaCl for high salt chromatography) was incubated with gentle agitation over-night. The column was washed with 10 ml of AC wash buffer (10 mM Tris/maleate, 10 mM MgCl2, 50 mM NaCl and 0.05% Triton X-100, pH 6.8). Retained proteins were eluted with 20 ml of AC elution buffer I (10 mM Tris/maleate, 10 mM MgCl2, 150 mM NaCl, 0.05% Triton X-100, pH 6.8) followed by a second elution step with 1 ml of AC elution buffer II (10 mM Tris/maleate, 10 mM MgCl2, 1 M NaCl, 0.05% Triton X-100, pH 6.8). Both elution fractions were stored at -20Â°C. For the high salt affinity chromatography, the AC high salt wash buffer (10 mM Tris/ maleate, 10 mM MgCl2, 400 mM NaCl and 0.05% Triton X-100, pH 6.8) and the AC high salt elution buffer (10 mM Tris/maleate, 10 mM MgCl2, 2 M NaCl, 0.05% Triton X-100, pH 6.8) were used. Elutions were analyzed by liquid chromatography (LC)-MS/MS. Mass spectrometry to identify NlpI affinity chromatography hits For liquid chromatography (LC)-MS/MS, tryptic peptides were desalted (Oasis HLB Î¼Elution Plate, Waters), dried in vacuum and reconstituted in 20 Î¼l of 4% acetonitrile, 0.1% formic acid. In total 1 Î¼g of peptide was separated with a nanoACQUITY UPLC system (Waters) fitted with a trapping column (nanoAcquity Symmetry C18; 5 Î¼m [average particle diameter]; 180 Î¼m [inner diameter] Ã— 20 mm [length]) and an analytical column (nanoAcquity BEH C18; 1.7 Î¼m [average particle diameter]; 75 Î¼m [inner diameter] Ã— 200 mm [length]). Peptides were separated on a 240 min gradient and were analyzed by electrospray ionizationâ€“tandem mass spectrometry on an Orbitrap Velos Pro (Thermo Fisher Scientific). Full-scan spectra from a mass/charge ratio of 300 to one of 1,700 at a resolution of 30,000 full width at half maximum were acquired in the Orbitrap mass spectrometer. From each full-scan spectrum, the 15 ions with the highest intensity were selected for fragmentation in the ion trap. A lock-mass correction with a background ion (mass/charge ratio, 445.12003) was applied. The raw mass spectrometry data was processed with MaxQuant (v1.5.2.8) (Cox and Mann, 2008) and searched against an Uniprot E.coli K12 proteome database. The search parameters were as following: Carbamidomethyl (C) (fixed), Acetyl (N-term) and Oxidation (M) (variable) were used as modifications.  For the full scan MS spectra (MS1) the mass error tolerance was set to 20 ppm and for the MS/MS spectra (MS2) to 0.5 Da. Trypsin was selected as protease with a maximum of two missed cleavages. For protein identification a minimum of one unique peptide with a peptide length of at least seven amino acids and a false discovery rate below 0.01 were required on the peptide and protein level. The match between runs function was enabled, a time window of one minute was set. Label free quantification was selected using iBAQ (calculated as the sum of the intensities of the identified peptides and divided by the number of observable peptides of a protein) (Schwanhausser et al., 2011), with the log fit function enabled.

### Data Protocol
The proteinGroups.txt file, an output of MaxQuant, was loaded into R (ISBN 3-900051-07-0) for further analysis. The iBAQ-values of the MaxQuant output were first batch-corrected using the limma package (Ritchie et al., 2015) and then normalized with the vsn package (Huber et al., 2002). Individual normalization coefficients were estimated for each biological condition separately. Limma was used again to test the normalized data for differential expression. Proteins were classified as a â€˜hitâ€™ with a log2 fold change higher than 4 and a â€˜candidateâ€™ with a log2 fold change higher than 2.

### Publication Abstract
The peptidoglycan (PG) sacculus provides bacteria with the mechanical strength to maintain cell shape and resist osmotic stress. Enlargement of the mesh-like sacculus requires the combined activity of peptidoglycan synthases and hydrolases. In Escherichia coli, the activity of two PG synthases is driven by lipoproteins anchored in the outer membrane (OM). However, the regulation of PG hydrolases is less well understood, with only regulators for PG amidases having been described. Here, we identify the OM lipoprotein NlpI as a general adaptor protein for PG hydrolases. NlpI binds to different classes of hydrolases and can specifically form complexes with various PG endopeptidases. In addition, NlpI seems to contribute both to PG elongation and division biosynthetic complexes based on its localization and genetic interactions. Consistent with such a role, we reconstitute PG multi-enzyme complexes containing NlpI, the PG synthesis regulator LpoA, its cognate bifunctional synthase, PBP1A, and different endopeptidases. Our results indicate that peptidoglycan regulators and adaptors are part of PG biosynthetic multi-enzyme complexes, regulating and potentially coordinating the spatiotemporal action of PG synthases and hydrolases.

### Keywords
Affinity chromatography, Escherichia coli

### Affiliations
EMBL
European Molecular Biology Laboratory

### Submitter
Frank Stein

### Lab Head
Dr Athanasios Typas
European Molecular Biology Laboratory


