### Accession
PXD041831

### Title
Cross-linking mass spectrometry uncovers interactions between high-density lipoproteins and the SARS-CoV-2 spike glycoprotein

### Description
Multiple studies to date have shown that high-density lipoprotein (HDL) levels are reduced in patients with coronavirus disease 2019 (COVID-19), and that the extent of this reduction is associated with poor clinical outcomes. While lipoproteins are known to play a key role during the lifecycle of hepatitis C virus, the direct influence they have upon coronavirus (CoV) infections remains poorly understood. In this study we utilise cross-linking mass spectrometry (XL-MS) to determine circulating protein interactors of the severe acute respiratory syndrome (SARS)-CoV-2 spike glycoprotein. Spike was revealed to interact with HDL, through direct interactions with apolipoprotein (Apo)-A1, ApoC3 and ApoD, highlighting multiple modalities of how SARS-CoV-2 and HDL may interact. XL-MS of plasma isolated from COVID-19 patients uncovered HDL protein interaction networks, dominated by acute phase serum amyloid proteins (SAA), whereby serum amyloid A2 (SAA2) was shown to bind to ApoD. ApoD was confirmed to also interact with core apolipoproteins of both LDL and VLDL, suggesting that SARS-CoV-2 may bind multiple lipoprotein species. The interaction between ApoD and spike was further validated in cells using immunoprecipitation-MS, which uncovered a novel interaction between both ApoD and spike with membrane-associated progesterone receptor component 1 (PGRMC1). Mechanistically, XL-MS coupled with data-driven structural modelling determined that ApoD may interact within the receptor-binding domain (RBD) of spike, posing the hypothesis that ApoD may interfere with binding to the angiotensin-converting enzyme 2 (ACE2) receptor. However, utilising multiple cell-based assays we determined ApoD to have no effect upon viral replication or infectivity.

### Sample Protocol
Protein-protein interactions within HDL, plasma and recombinant proteins were interrogated using the MS-cleavable cross-linker disuccinimidyl sulfoxide (DSSO, Thermo Fisher) and DMTMM (Merck), following recently published protocols(18, 19). Recombinant SARS-CoV and SARS-CoV-2 spike protein was mixed with HDL isolated by potassium bromide (kBr) density ultracentrifugation (Merck, LP3-5MG) diluted in PBS (pH 7.5) at a concentration of 1 µg/µL and allowed to incubate overnight at 4oC. DSSO was prepared as a 50 mM stock solution in DMSO and added to protein mixtures at a final concentration of 1 mM for 30 minutes at room temperature. DMTMM was prepared in PBS and used at a final concentration of 10 mM. Crosslinking reactions were quenched through the addition of Tris-HCL (pH 8.5) at a final concentration of 50 mM for 30 minutes at room temperature. Crosslinked protein samples were then digested and purified as described below prior to fractionation using a strong-cation exchange (SCX) Dionex UltiMate 3000 RSLC system (Luna 5 µm, 100 Å, 50 x 2 mm. Phenomenex). Resulting peptide fractions were dried using a speed vac (Thermo Fisher, Savant SPD131DDA) and desalted robotically using C18 cartridges (Bravo AssayMAP, Agilent Technologies), peptides were then resuspended in 5% DMSO / 10% FA / 85% H2O prior to MS-analysis. Experiments using recombinants proteins (ApoD, a.a 21-189, Abcam ab184598, Full-length Spike a.a 11-1208, Thermo Fisher RP-87680 and Spike-RBD, a.a 319-541, R&D 10500-CV) only, involved mixing at an equimolar ratio prior to overnight incubation at 4oC and following the same protocol as described above. Experiments involving depleted plasma, LDL (Merck, LP2-2MG) and VLDL (Merck, LP1) followed the same XL-MS protocol above.

### Data Protocol
Thermo Scientific Proteome Discoverer software (version 2.2.0.388) was used to search non-crosslinked raw data files against the human database (UniProtKB/Swiss-Prot version 2018_02, 20,400 protein entries) using Mascot (version 2.6.0, Matrix Science). The mass tolerance was set at 10 ppm for precursor ions and 0.8Da for fragment ions. Trypsin was used as the digestion enzyme with up to two missed cleavages being allowed. Carbamidomethylation of cysteines and oxidation of methionine residues were chosen as fixed and variable modifications, respectively. Label-free quantification was conducted through the in-built Minora feature detection node, alongside retention time alignment using the feature mapper node, with a maximum retention time shift of 10 minutes being allowed. Unique and Razor peptide precursor areas were used for quantification and cross-sample normalisation was achieved using the total peptide amount. MS/MS-based peptide and protein identifications were validated with the following filters, a peptide probability of greater than 95.0% (as specified by the peptide prophet algorithm), a protein probability of greater than 99.0%, and at least two unique peptides per protein. DSSO XLMS acquired data was searched in Thermo Scientific Proteome Discoverer software (version 2.5) using the third party XlinkX node developed by the Heck lab(20). The inbuilt MS-cleavable MS2/MS3 DSSO search strategy was used. XLMS raw files for plasma analyses were searched against a curated human database of previous in-house DDA analyses (1048 protein entries), while HDL XLMS raw files were searched against an in-house HDL proteome database containing sequence information for 348 proteins including the SARS-CoV and SARS-CoV-2 spike protein. Sequest HT was used with Trypsin as the digestion enzyme with up to two missed cleavages being allowed. Methionine oxidation, DSSO modifications of lysine residues and N-terminal acetylation were set as dynamic modifications, and carbamidomethylation of cysteines was set as a static. MS2_MS3 strategy was set and the DSSO / +158.004 Da (K) crosslink modification was used while keeping all other settings default. DMTMM XLMS data was searched using pLink2(21), peptide mass accuracy was ±10 ppm, fragment mass accuracy was ±20 ppm, and a 5% FDR filter was applied at the spectral level. XLMS data visualisation was conducted using xiview.org(22).

### Publication Abstract
High-density lipoprotein (HDL) levels are reduced in patients with coronavirus disease 2019 (COVID-19), and the extent of this reduction is associated with poor clinical outcomes. While lipoproteins are known to play a key role during the life cycle of the hepatitis C virus, their influence on coronavirus (CoV) infections is poorly understood. In this study, we utilize cross-linking mass spectrometry (XL-MS) to determine circulating protein interactors of the severe acute respiratory syndrome (SARS)-CoV-2 spike glycoprotein. XL-MS of plasma isolated from patients with COVID-19 uncovered HDL protein interaction networks, dominated by acute-phase serum amyloid proteins, whereby serum amyloid A2 was shown to bind to apolipoprotein (Apo) D. XL-MS on isolated HDL confirmed ApoD to interact with SARS-CoV-2 spike but not SARS-CoV-1 spike. Other direct interactions of SARS-CoV-2 spike upon HDL included ApoA1 and ApoC3. The interaction between ApoD and spike was further validated in cells using immunoprecipitation-MS, which uncovered a novel interaction between both ApoD and spike with membrane-associated progesterone receptor component 1. Mechanistically, XL-MS coupled with data-driven structural modeling determined that ApoD may interact within the receptor-binding domain of the spike. However, ApoD overexpression in multiple cell-based assays had no effect upon viral replication or infectivity. Thus, SARS-CoV-2 spike can bind to apolipoproteins on HDL, but these interactions do not appear to alter infectivity.

### Keywords
Plasma, Covid-19, Xl-ms

### Affiliations
Kavli Institute for Nanoscience Discovery, University of Oxford, Oxford, UK
King's College London

### Submitter
Sean Burnap

### Lab Head
Dr Manuel Mayr
King's College London


