### Accession
PXD008726

### Title
Mouse WT vs. A20 KO BMDM differential proteomics

### Description
A20 is a negative regulator of NF-κB signaling, crucial to control inflammatory responses and ensure tissue homeostasis. A20 is thought to restrict NF-κB activation both by its ubiquitin-editing activity as by non-enzymatic activities. Besides its role in NF-κB signaling, A20 also acts as a protective factor inhibiting apoptosis and necroptosis. Because of the ability of A20 to both ubiquitinate and deubiquitinate substrates and its involvement in many cellular processes, we hypothesized that deletion of A20 might generally impact on protein levels, thereby disrupting cellular processes. We performed a differential proteomics study of bone marrow derived macrophages (BMDMs) from control and myeloid-specific A20 knockout mice, both in untreated conditions and after LPS and TNF treatment, and demonstrate proteome-wide changes in protein expression upon A20 deletion. Several inflammatory proteins are up-regulated in the absence of A20, even without an inflammatory stimulus. Depending on the treatment and the time, more proteins are regulated. Together these changes may affect multiple signaling pathways disturbing tissue homeostasis and inducing (autoimmune) inflammation, as suggested by genetic studies in patients.

### Sample Protocol
Bone marrow cells were isolated from tibia and femurs from A20myel-KO mice and control wild-type (WT) littermate mice, and flushed with RPMI medium 1640. Bone marrow derived macrophages (BMDMs) were generated by culturing bone marrow cells in RPMI medium supplemented with 10% fetal bovine serum, 25 units/ml penicillin, 25 µg/ml streptomycin, L-glutamine and 40 ng/ml mouse macrophage colony stimulating factor (MCSF). Every 48 h the medium was refreshed and on day 7, 3 x 106 BMDMs were seeded. The BMDMs were stimulated or not with 10 µg/ml lipopolysaccharide (LPS) or 1000 U/ml mouse tumor necrosis factor (TNF) for 1 or 6 h. Cells were washed twice with ice-cold Dulbecco’s phosphate buffered saline  and scraped in 500 µl urea lysis buffer containing 20 mM HEPES pH 8.0, 9 M urea, 1 mM sodium orthovanadate, 2.5 mM sodium pyrophosphate and 1 mM β-glycerophosphate. Lysates were sonicated at 15 W output with 3 bursts of 15 s each and cleared by centrifugation for 15 min at 20,000 x g at room temperature, followed by reduction and alkylation of cysteines with respectively 5 mM dithiothreitol (DTT) for 30 min at 55°C and 10 mM iodoacetamide (IAA) for 15 min at room temperature in the dark. Proteins were digested with mass spectrometry grade Lysyl Endopeptidase (enzyme/substrate ratio of 1/250 w/w, Wako) for 3 h at 37°C. Samples were diluted with 20 mM HEPES pH 8.0 to a final concentration of 2 M urea and the proteins were further digested with sequencing-grade, modified trypsin (enzyme/substrate ratio of 1/100 w/w) overnight at 37°C. Digestion was terminated by adding trifluoroacetic acid (TFA) to a final concentration of 0.5%, peptide mixtures were desalted using Pierce C18 tips, dried and re-dissolved in 20 µl water/acetonitrile (ACN, 98/2 v/v) containing 0.1% TFA for subsequent LC-MS/MS analysis.

### Data Protocol
The obtained peptide mixtures were introduced into an LC-MS/MS system, being an Ultimate 3000 RSLC nano LC in-line connected to a Q Exactive HF mass spectrometer (Thermo Fisher Scientific). The MS/MS spectra were searched with the MaxQuant software (version 1.5.4.1) against murine (Mus musculus) protein sequences in the SwissProt database (release of November 2015 containing 16,601 sequences), with a precursor mass tolerance set at 20 ppm for the first search and 4.5 ppm for the main search. Acetylation of protein N-termini and methionine oxidation were set as variable modifications and carbamidomethylation of cysteine as a fixed modification. Trypsin was selected as protease and cleavage was allowed when arginine or lysine was followed by proline. Two missed cleavages were allowed. The ‘matching between runs’ feature was activated. The false discovery rate (FDR) at the peptide spectrum match (PSM) and the protein level was set to 1% and the minimum peptide length was set to 7. For analysis of phosphorylation and ubiquitination sites, a separate search was performed with phosphorylation on serine, threonine and tyrosine included as variable modification.  Data analysis was performed with the MSqRob software (version 0.6.3) that uses a peptide-based model to estimate the protein fold changes directly from peptide intensities. In total, 34,376 peptides were identified. The peptide intensities were log2 transformed and quantile normalized. Reversed sequences and potential contaminants were removed from the data. Peptides belonging to protein groups that contained one or more proteins that were also present in a smaller protein group, were discarded. Relative protein quantification was performed on 30,559 remaining peptides. The treatment effect was specified as fixed and contains all combinations of genotypes (WT and A20 KO) and stimulants (untreated, LPS and TNF). We accounted for the correlation structure in the peptide-level data by incorporating a random mouse effect (correlation due to the longitudinal design) and a random sample effect (peptides intensities from the same protein pool are more alike). We also corrected for peptide-specific effects by including a random effect for peptide sequence. A random peptide sequence effect enables a different shrinkage (regularization) of the sequence effect and the treatment effect. For each protein, we tested for differential abundance between genotypes (A20 KO vs WT BMDMs) at baseline, after 1 h of LPS treatment, after 6 h of LPS treatment, after 1 h of TNF treatment and after 6 h of TNF treatment.

### Publication Abstract
A20 is a negative regulator of NF-&#x3ba;B signaling; it controls inflammatory responses and ensures tissue homeostasis. A20 is thought to restrict NF-&#x3ba;B activation both by its ubiquitin-editing activity as well as by its nonenzymatic activities. Besides its role in NF-&#x3ba;B signaling, A20 also acts as a protective factor inhibiting apoptosis and necroptosis. Because of the ability of A20 to both ubiquitinate and deubiquitinate substrates, and its involvement in many cellular processes, we hypothesized that deletion of A20 might generally impact on protein levels, thereby disrupting cellular signaling. We performed a differential proteomics study on bone marrow-derived macrophages (BMDMs) from control and myeloid-specific A20 knockout mice, both in untreated conditions and after LPS or TNF treatment, and demonstrated A20-dependent changes in protein expression. Several inflammatory proteins were found up-regulated in the absence of A20, even without an inflammatory stimulus, but, depending on the treatment and the treatment time, more proteins were found regulated. Together these protein changes may affect normal signaling events, which may disturb tissue homeostasis and induce (autoimmune) inflammation, in agreement with A20s proposed identity as a susceptibility gene for inflammatory disease. We further verify that immune-responsive gene 1 (IRG1) is up-regulated in the absence of A20 and that its levels are transcriptionally regulated.

### Keywords
Lc-msms, Bmdm, A20

### Affiliations
Ghent University
VIB-UGent Center for Medical Biotechnology Department of Biochemistry, Ghent University Ghent, Belgium

### Submitter
Emmy Van Quickelberghe

### Lab Head
Dr Kris Gevaert
VIB-UGent Center for Medical Biotechnology Department of Biochemistry, Ghent University Ghent, Belgium


