### Accession
PXD026378

### Title
Variation in upstream open reading frames contributes to allelic diversity in protein abundance

### Description
The 5' untranslated region (UTR) sequence of eukaryotic mRNAs may contain upstream open reading frames (uORFs), which can regulate translation of the main open reading frame (mORF). The current model of translational regulation by uORFs posits that when a ribosome scans an mRNA and encounters a uORF, translation of that uORF can prevent ribosomes from reaching the mORF and cause decreased mORF translation. In this study, we first observed that rare variants in the 5' UTR dysregulate protein abundance. Upon further investigation, we found that rare variants near the start codon of uORFs can repress or derepress mORF translation, causing allelic changes in protein abundance. This finding holds for common variants as well, and common variants that modify uORF start codons also contribute disproportionately to metabolic and whole-plant phenotypes, suggesting that translational regulation by uORFs serves an adaptive function. These results provide evidence for the mechanisms by which natural sequence variation modulates gene expression, and ultimately, phenotype.

### Sample Protocol
The plants were grown using 3:1 Metromix 900 (SunGro)/Turface MVP (Profile Products) in the greenhouse under a 16-hr light/8-hr dark photoperiod and 27 ºC/ 22 ºC day/ night temperatures. Third and fourth leaves of two weeks old plants were collected separately, flash frozen and stored at -80 ºC. Proteins were extracted at 4 °C from pulverized leaves into 50 mM HEPES (pH 7.5), 5 mM Na2EDTA, 2 mM dithiotreitol, and 1X plant protease inhibitor cocktail (P9599; Sigma-Aldrich). After clarification by centrifugation at 16,000 Xg, protein concentration was measured by Bradford assay following manufacturer’s instructions. 60 µg of protein was precipitated in 4:1:3 (v/v) methanol/chloroform/water and collected by centrifugation. The resulting pellet was washed once with the same mix, lyophilized to dryness, resuspended into 100 µl of 8 M urea, and reduced for 1 hr at 22 °C with 10 mM dithiothreitol, followed by alkylation with 20 mM iodoacetamide for 1 hr at dark. The reaction was quenched with 20 mM dithiotreitol and diluted with 900 µl of 25 mM ammonium bicarbonate to reduce the urea concentration below 1.5 M, and digested overnight at 37 °C with 0.5 µg of sequencing-grade modified porcine trypsin (Promega). The resulting peptides were lyophilized to a final volume of ~250 µl, acidified with 10% trifluoroacetic acid until the pH was below 3.0, and then desalted and concentrated with a 100 µl Bond Elut OMIX C18 pipette tip (Agilent Technologies) according to the manufacturer’s instructions. The peptides were eluted in 50 µl of 75% acetonitrile and 0.1% acetic acid, lyophilized, and resuspended in 18 µl 5% acetonitrile and 0.1% formic acid. 4 µl sample was used for each mass spectrometry analysis. Nano-scale LC separation of the tryptic peptides was performed using a Dionex Ultimate™ 3000 Rapid Separation system equipped with a 75 µm x 25 cm Acclaim PepMap RSLC C18 column (Thermo Fisher Scientific), in combination with a 2-hr linear 4% to 36% acetonitrile gradient in 0.1% formic acid and a flow rate of 250 nl/min. Eluted peptides were analyzed online by a Q Exactive Plus spectrometer (Thermo Fisher Scientific) in the positive ESI mode. Data-dependent acquisition of full MS scans (mass range of 380-1500 m/z) at a resolution of 70,000 was collected, with the automatic gain control (AGC) target set to 3 x 106, and the maximum fill time set to 200 msec. High-energy collision-induced dissociation fragmentation of the 15 strongest peaks was performed with an intensity threshold of 4 x 104 counts and an isolation window of 3.0 m/z, and excluded precursors that had unassigned, +1, +7, +8, or >+8 charge states. MS/MS scans were conducted at a resolution of 17,500, with an AGC target of 2 x 105 and a maximum fill time of 100 msec. Dynamic exclusion was performed with a repeat count of 2 and an exclusion duration of 30 sec, while the minimum MS ion count for triggering MS/MS was set to 4 x 103 counts. Each genotype was analyzed with five biological and three technical replicates. A digest of cytochrome C (Thermo Scientific) was analyzed to ensure optimal peak shape.

### Data Protocol
The resulting MS/MS datasets were queried by Proteome Discoverer (version 2.0.0.802; Thermo Fisher Scientific) against the Z. mays B73 RefGen_v5 [released 2020] (https://download.maizegdb.org/Zm-B73-REFERENCE-NAM-5.0/). Peptides were assigned by SEQUEST HT, allowing a maximum of 1 missed tryptic cleavages, a minimum peptide length of 6, a precursor mass tolerance of 10 ppm, and fragment mass tolerances of 0.02 Da. Carbamidomethylation of cysteines and oxidation of methionine were specified as static and dynamic modifications, respectively. Label-free quantification based on MS1 precursor ion intensity was performed in Proteome Discoverer with a minimum Quan value threshold set to 0.0001 for unique peptides; the “3 Top N” peptides were used for area calculation. Only peptides with FDRs of 0.01 (high confidence) were used for data analysis.

### Publication Abstract
None

### Keywords
Lc-ms, Utr, Snp, Proteomics, Rare alleles, Maize

### Affiliations
Biology department, Washington University in Saint Louis, Saint Louis, MO, USA
Washington University in St. Louis

### Submitter
Sujina Mali

### Lab Head
Dr Richard D Vierstra
Biology department, Washington University in Saint Louis, Saint Louis, MO, USA


