### Accession
PXD042310

### Title
Integrative Analysis of Transcriptome and Proteome for a new Rumen-derived Bacillus Licheniform in regulating microenvrionment Treating with Different Carbon Sources

### Description
Bacillus licheniformis (B. licheniformis) is a microorganism with a wide range of probiotic properties and applications. Isolation and identification of novel strains is a major aspect of microbial research. Besides, the role of different carbon sources affects B. licheniformis in regulating micro-environment and the mechanisms need to be further investigated. In this study, we first isolated and identified a new strain of B. licheniformis from bovine rumen fluid. Microcrystalline cellulose (MC) and cellobiose (CB) as the certain carbon sources to treat strain. Further, a combination of transcriptome and proteome analyses was used to different carbon sources effects. The results showed that B. licheniformis ABC transporter proteins, antibiotic synthesis, flagellar assembly, cellulase-related pathways and proteins were significantly up-regulated in the MC treatment compared to the CB treatment, and lactate metabolism was inhibited. In addition, MC was used as a certain carbon source to improve bacterial inhibition of B. licheniformis, its own disease resistance and to regulate the rumen micro-environment. In conclusion, our research provides a potential new probiotic for feed research and a theoretical basis for investigating the mechanisms by which bacteria respond to different carbon sources.

### Sample Protocol
The total protein of B.licheniformis was extracted by used of SDT sample buffer (4%SDS, 100 mM Dithiothreitol, 100 mM Tris-HCl pH 8.0) and analyzed by Easy nLC 1200 (Thermo fish) and combined with Q-Exactive HF-X MS.     Max Quant version 1.6.0.16 was used for processing the LC-MS/MS data. The B.licheniformis protein database was obtained from NCBI to the identification of MS results. The false discovery rate (FDR) for peptide and protein identification was set at less than 1%. Use the label-free quantification with a minimum of two ratio counts to normalized the protein levels for comparison between samples. To analyze the LFQ number of proteins in each sample and use Max Quant system weighting and normalizing data. Fold change â‰¥1.5 and p <0.05 set as significance.

### Data Protocol
Protein Pilot software (version 5.0) (AB SCIEX) was used to match the mass spectrometry raw data to the contents of a B. licheniformis database with the Paragon algorithm (Shilov et al., 2007). The differentially expressed protein were identified based on a fold change > 1.2 or < 0.8 and p < 0.05. A comparative analysis between proteome with transcriptome was carried on the online platform of Majorbio Cloud Platform (https://cloud.majorbio.com), and the heat map was generated using Heatmapper (http://www.heatmapper. ca/). The Kyoto Encyclopedia of Genes and Genomes (KEGG) pathways enriched among the differentially expressed protein and differentially expressed genes were also identified.

### Publication Abstract
None

### Keywords
Cows rumen fluid, Bacillus licheniformis, Different carbon sources, Transcriptome, Proteome

### Affiliations
College of Animal Science and Technology, Ningxia University, 489 Helanshan West Road, Xixia District, Yinchuan, Ningxia
Docter

### Submitter
Yuchen Cheng

### Lab Head
Dr Xiaofeng Xu
College of Animal Science and Technology, Ningxia University, 489 Helanshan West Road, Xixia District, Yinchuan, Ningxia


