### Accession
PXD002080

### Title
CPTAC, TCGA Cancer Proteome Study of Colorectal Tissue: Proteome, VU, part 1

### Description
The goal of the CPTAC, TCGA Cancer Proteome Study of Colorectal Tissue is to analyze the proteomes of TCGA tumor samples that have been comprehensively characterized by molecular methods. Ninety-five TCGA tumor samples were used in this study.

### Sample Protocol
Analytical Sample Protocol - Alkylation: Iodoacetamide, Fractionation: Peptide IEF, Fractions: 15, Proteolysis: Trypsin, Starting Amount: 200 ug; Chromatography Protocol - Column Length: 11 cm, Column Type: C18, Gradient Length: 95 min, Injected: 1 ug, Inside Diameter: 100 um, Particle Size: 5 um; Mass Spectrometry Protocol - Dissociation: CID, Instrument: Thermo LTQ Orbitrap Velos, MS1 Resolution: 60000, Precursors: Top 8. See associated protocols and methods documents at https://cptac-data-portal.georgetown.edu.

### Data Protocol
Processed using the CPTAC Common Data Analysis Pipeline at NIST. Peptide identifications from MSGF+. Filtered at 1% FDR. See associated protocols and methods documents at https://cptac-data-portal.georgetown.edu.

### Publication Abstract
Extensive genomic characterization of human cancers presents the problem of inference from genomic abnormalities to cancer phenotypes. To address this problem, we analysed proteomes of colon and rectal tumours characterized previously by The Cancer Genome Atlas (TCGA) and perform integrated proteogenomic analyses. Somatic variants displayed reduced protein abundance compared to germline variants. Messenger RNA transcript abundance did not reliably predict protein abundance differences between tumours. Proteomics identified five proteomic subtypes in the TCGA cohort, two of which overlapped with the TCGA 'microsatellite instability/CpG island methylation phenotype' transcriptomic subtype, but had distinct mutation, methylation and protein expression patterns associated with different clinical outcomes. Although copy number alterations showed strong cis- and trans-effects on mRNA abundance, relatively few of these extend to the protein level. Thus, proteomics data enabled prioritization of candidate driver genes. The chromosome 20q amplicon was associated with the largest global changes at both mRNA and protein levels; proteomics data highlighted potential 20q candidates, including HNF4A (hepatocyte nuclear factor 4, alpha), TOMM34 (translocase of outer mitochondrial membrane 34) and SRC (SRC proto-oncogene, non-receptor tyrosine kinase). Integrated proteogenomic analysis provides functional context to interpret genomic abnormalities and affords a new paradigm for understanding cancer biology.

### Keywords
Cptac cptac-tcga_colorectal_cancer cptac-vu

### Affiliations
Georgetown University
Vanderbilt University

### Submitter
Nathan Edwards

### Lab Head
Dr Daniel C. Liebler, Ph.D.
Vanderbilt University


### SDRF
- Characteristics[organism: Homo sapiens
- Characteristics[organism part: Sigmoid Colon, Cecum, Hepatic Flexure, Ascending Colon
- Characteristics[disease: Colon adenocarcinoma
- cell type: not available
- label: AC=MS:1002038;NT=label free sample
- instrument: AC=MS:1001742;NT=LTQ Orbitrap Velos
- modification parameters: NT=Oxidation;MT=Variable;TA=M;AC=UNIMOD:35
- modification parameters.1: NT=Carbamidomethyl;AC=UNIMOD:4;TA=C;MT=Fixed

