### Accession
PXD020442

### Title
FAIMS analysis of bacterial glycopeptides

### Description
Hydrophilic Interaction Liquid Chromatography (HILIC) glycopeptide enrichment is an indispensable tool for the high-throughput characterisation of glycoproteomes. Despite its utility HILIC enrichment is associated with a number of short comings such as requiring large amounts of starting material, potentially introducing chemical artefacts such as formylation, and biasing/under-sampling specific classes of glycopeptides. Here we investigate HILIC enrichment independent approaches for the study of bacterial glycoproteomes. Using three Burkholderia species (B. cenocepacia; B. dolosa and B. ubonensis) we demonstrate that short aliphatic O-linked glycopeptides are typically absent from HILIC enrichments yet are readily identified in whole proteome samples. Using FAIMS fractionation we show at low compensation voltages (CVs) short aliphatic glycopeptides can be enriched from complex samples providing an alternative means to identify glycopeptides recalcitrant to hydrophilic based enrichment. Combining whole proteome and FAIMS analysis we show the observable glycoproteome of these Burkholderia species is at least 30% larger than initially thought. Excitingly the ability to enrich glycopeptides using FAIMS appears generally applicable with N-linked glycopeptides of Campylobacter fetus subsp. fetus also enrichable at low FAIMS CVs. Combined these results demonstrate that FAIMS provides an alternative means to access glycopeptides and is a valuable tool for glycoproteomic analysis.

### Sample Protocol
Generation of bacterial lysates for glycoproteome analysis: Bacterial strains were grown to confluency before flooding plates with 5 mL of pre-chilled sterile phosphate-buffered saline (PBS) and bacterial cells collected by scraping. Cells were washed 3 times in PBS to remove media contaminates, then collected by centrifugation at 10,000 x g at 4˚C for 10 min and then snap frozen. Snap frozen cells were resuspended in 4% SDS, 100mM Tris pH 8.0, 20mM Dithiothreitol (DTT) and boiled at 95˚C with shaking at 2000rpm for 10 min. Samples were clarified by centrifugation at 17,000 x g for 10 min, the supernatants were then collected, and protein concentration determined by a bicinchoninic acid assay (Thermo Fisher Scientific, Waltham, MA, USA). 1mg of protein from each sample was acetone precipitated by mixing one volume of sample with 4 volumes of ice-cold acetone. Samples were precipitated overnight at -20˚C and then spun down at 16,000 x g for 10 min at 0˚C. The precipitated protein pellets were resuspended in 80% ice-cold acetone and precipitated for an additional 4 hours at -20˚C. Samples were centrifuged at 17,000 x g for 10 min at 0˚C, the supernatant discarded, and excess acetone driven off at 65˚C for 5 min.   Digestion of protein samples: Protein digestion was undertaken as previously described with minor alterations [20]. Briefly, dried protein pellets were resuspended in 6 M urea, 2 M thiourea in 40 mM NH4HCO3 then reduced for 1 hour with 20mM DTT followed by alkylation with 40mM chloroacetamide for 1 hour in the dark. Samples were then digested with Lys-C (1/200 w/w) for 3 hours before being diluted with 5 volumes of 40 mM NH4HCO3 and digested overnight with trypsin (1/50 w/w). Digested samples were acidified to a final concentration of 0.5% formic acid and desalted with 50 mg tC18 Sep-Pak columns (Waters corporation, Milford, MA, USA) according to the manufacturer’s instructions. tC18 Sep-Pak columns were conditioned with 10 bed volumes of Buffer B (0.1% formic acid, 80% acetonitrile), then equilibrated with 10 bed volumes of Buffer A* (0.1% TFA, 2% acetonitrile) before use. Samples were loaded on to equilibrated columns then columns washed with at least 10 bed volumes of Buffer A* before bound peptides were eluted with Buffer B. Eluted peptides were aliquoted into samples for ZIC-HILIC enrichment or total proteome analysis then dried by vacuum centrifugation and stored at -20 ˚C.

### Data Protocol
Proteome Databases: B.a cenocepacia LMG 16656 / J2315 - Uniprot: UP000001035 B. dolosa AU0158 - Uniprot: UP000032886 B. ubonensis MSMB22 - Burkholderia Genome Database Strain number: 3404 C. fetus subsp. fetus NCTC 10842 - Uniprot: UP000001035  Data Analysis - Glycopeptide identification: Raw data files were batched processed using Byonic v3.5.3 (Protein Metrics Inc. [70]) with the proteome databases denoted within Table 1. Data was searched on a desktop with two 3.00GHz Intel Xeon Gold 6148 processors, a 2TB SDD and 128 GB of RAM using a maximum of 16 cores for a given search. For all searches, a semi-tryptic N-ragged specificity was set and a maximum of two missed cleavage events allowed. Carbamidomethyl was set as a fixed modification of cystine while oxidation of methionine was included as a variable modification. A maximum mass precursor tolerance of 5 ppm was allowed while a mass tolerance of up to 10 ppm was set for HCD fragments and 20 ppm for EThcD fragments. Burkholderia species searches were conducted allowing two O-linked glycans; Hex-HexNAc-HexNAc (elemental composition: C22O15H36N2, mass: 568.2115) and Suc-Hex-HexNAc-HexNAc (elemental composition: C26O18H40N2, mass: 668.2276) while C. fetus fetus searches were conducted allowing two N-linked glycans; Hex-HexNAc4-diNAcBac (elemental composition: C48O29H78N6, mass: 1202.4813) and HexNAc5-diNAcBac (elemental composition: C50O29H81N7, mass: 1243.5078) where diNAcBac is the bacterial specific sugar 2,4-diacetamido-2,4,6 trideoxyglucopyranose [64]. To ensure high data quality technical replicates were combined using R (https://www.r-project.org/) and only glycopeptides with a Byonic score >300 were used for further analysis. This score cut-off is in line with previous reports highlighting that score thresholds greater than at least 150 are required for robust glycopeptide assignments with Byonic [29, 71]. It should be noted that a score threshold of above 300 resulted in false discovery rates of less than 1% for all combined datasets.   Data Analysis - iBAQ analysis: To assess the relative abundance of proteins iBAQ analysis was undertaken. Unenriched proteome datasets were processed using MaxQuant (v1.6.3.4 [72]) searching against the reference B. cenocepacia strain J2315. Carbamidomethylation of cysteine was set as a fixed modification while oxidation of methionine and acetylation of protein N-termini were allowed as variable modifications. An enzyme specificity of “trypsin/P” was set with a maximum of 2 missed cleavage events. The resulting outputs were processed in the Perseus (v1.5.0.9) [73] analysis environment to remove reverse matches and common protein contaminates prior to further analysis.

### Publication Abstract
None

### Keywords
Hilic, Glycosylation, Burkholderia, Faims

### Affiliations
University of Melbourne
Department of Microbiology and Immunology, University of Melbourne at the Peter Doherty Institute for Infection and Immunity, Melbourne 3000, Australia

### Submitter
Nichollas Scott

### Lab Head
Dr Nichollas E. Scott
Department of Microbiology and Immunology, University of Melbourne at the Peter Doherty Institute for Infection and Immunity, Melbourne 3000, Australia


