### Accession
PXD030335

### Title
Identification of TBBPA-degraders in anaerobic bioreactor at environmentally relevant concentrations by combined protein-based stable isotope probing and metagenomics

### Description
The aim of this study was to identify TBBPA-degrading organisms in a complex microbial community by a metagenome-based functional metaproteomic approach, using protein-based stable isotope probing (protein-SIP). Firstly, the degradation kinetics were evaluated in order to simulate the decrease of residual mass of the labelled compound based on experimental data. In sequence, a metagenome was generated, and biomass was collected in different time-points for protein-SIP in incubations with 13C-TBBPA. This approach allowed for the identification organisms assimilating labelled carbon from the cometabolic degradation of a micropollutant.

### Sample Protocol
Protein extraction was performed using liquid-liquid extraction using TEAB (0.05 M TEAB buffer stock, 1.0 mg/L NaDOC, pH â‰¤ 8) as resuspension buffer. Extracted proteins were digested using in-gel digestion protocol, following desalting and analysis of tryptic peptides by automated liquid chromatograph-electrospray ionization tandem mass spectrometry (LC-ESI-MS/MS).

### Data Protocol
To identify the peptides/proteins a search using OMSSA in an OpenMS pipeline, with the precursor mass tolerance set to 5 ppm, the fragment mass tolerance to 0.01 Da, and false discovery rate (FDR) to 1%. RIA and the labelling ratio (LR) were determined using OpenMS and the MetaProSIP tool with a minimum correlation of 0.7 and a mass window of 10 ppm.

### Publication Abstract
Tetrabromobisphenol A (TBBPA) is the most used flame retardant worldwide and has become a threat to aquatic ecosystems. Previous research into the degradation of this micropollutant in anaerobic bioreactors has suggested several identities of putative TBBPA degraders. However, the organisms actively degrading TBBPA under in situ conditions have so far not been identified. Protein-stable isotope probing (protein-SIP) has become a cutting-edge technique in microbial ecology for enabling the link between identity and function under in situ conditions. Therefore, it was hypothesized that combining protein-based stable isotope probing with metagenomics could be used to identify and provide genomic insight into the TBBPA-degrading organisms. The identified <sup>13</sup>C-labelled peptides were found to belong to organisms affiliated to Phytobacter, Clostridium, Sporolactobacillus, and Klebsilla genera. The functional classification of identified labelled peptides revealed that TBBPA is not only transformed by cometabolic reactions, but also assimilated into the biomass. By application of the proteogenomics with labelled micropollutants (protein-SIP) and metagenome-assembled genomes, it was possible to extend the current perspective of the diversity of TBBPA degraders in wastewater and predict putative TBBPA degradation pathways. The study provides a link to the active TBBPA degraders and which organisms to favor for optimized biodegradation.

### Keywords
Stable isotope probing, Metaproteomics, Tetrabromobisphenol a, Anaerobic digestion

### Affiliations
aDepartment of Chemistry and Bioscience, Aalborg University, Aalborg East, Denmark
Aalborg University

### Submitter
Jan Struckmann Poulsen

### Lab Head
Dr Jeppe Lund Nielsen
aDepartment of Chemistry and Bioscience, Aalborg University, Aalborg East, Denmark


