### Accession
PXD000478

### Title
Transcriptome and proteome quantification of a tumor model provides novel insights into post-transcriptional gene regulation

### Description
Genome-wide transcriptome analyses have allowed for systems- level insights into gene regulatory networks. Due to the limited depth of quantitative proteomics, however, our understanding of post-transcriptional gene regulation and its effects on protein complex stoichiometry are lagging behind. Here, we employ deep sequencing and iTRAQ technology to determine transcript and protein expression changes of a Drosophila brain tumour model at near genome-wide resolution. In total, we quantify more than 6,200 tissue-specific proteins, corresponding to about 70% of all transcribed protein-coding genes. Using our integrated data set, we demonstrate that post-transcriptional gene regulation varies considerably with biological function and is surprisingly high for genes regulating transcription. We combine our quantitative data with protein-protein interaction data and show that post-transcriptional mechanisms significantly enhance co-regulation of protein complex subunits beyond transcriptional co-regulation. Interestingly, our results suggest that only about 11% of the annotated Drosophila protein complexes are co-regulated in the brain. Finally, we refine the composition of some of these core protein complexes by analysing the co-regulation of potential subunits. Our comprehensive transcriptome and proteome data provide a rich resource for quantitative biology and offer novel insights into understanding post- transcriptional gene regulation in a tumour model.        Data generated by LC-MS/MS analysis were searched against a database containing the translation of all open reading frames (ORF) in FlyBase (r5.25) and common contaminants concatenated to a reversed decoy database that allowed for estimating the false discovery rate (FDR) using the target-decoy strategy. Proteome Discoverer (version 1.3.0.211, Thermo Fisher Scientific) was used as a search engine interface for Mascot, Sequest, X!-Tandem, and ZCore. Oxidation of methionine was set as dynamic modification, methylthio (C) and iTRAQ4plex label (K, N-terminus) as static modifications. The minimal peptide length was set to seven amino acids, and a maximum of two missed cleavages was allowed for trypsin and LysC digested samples. To allow for an integrative analysis of transcriptome and proteome data, protein level changes were determined using only peptides that mapped unambiguously to one gene. Peptides that could be derived from proteins encoded by different gene models ("shared peptides") were excluded. Furthermore, only peptides that showed less than two-fold difference between duplicate iTRAQ channels were included in the analysis. Peptide identifications from different search engines were combined using a modified version of the combined FDR score. Reporter ion intensities were corrected for isotope impurities in the iTRAQ labels. To account for the error structure and stabilise the variance of the reporter ion intensities a variance stabilising transformation (vsn) was applied. Protein ratios were calculated as the 20% trimmed mean from the median-centred peptide ratios. Proteins were filtered for a maximum FDR of 5%.        Additional transcriptomics data will be accessible under the GEO Submission record GSE51412.

### Sample Protocol
Not available

### Data Protocol
Not available

### Publication Abstract
None

### Keywords
Brain tumor, Drosophila, Itraq

### Affiliations
IMBA

### Submitter
Christoph JÃ¼schke

### Lab Head


