### Accession
PXD028864

### Title
Quantitative structural proteomics unveils the conformational changes of proteins under ER stress

### Description
Using cysteine footprinting to globally profile protein structure changes. The method was further applied to study the unfolding effect of tunicamycin.

### Sample Protocol
After harvested, cells were lyzed by freeze-thaw and small molecules were removed by centrifugal filter. Proteins were then labeled by cysteine-targeting probe. Labeled proteins were digested, and labeled peptides were enriched by biotin-avidin interaction. LC-MS/MS was used to analyse the enriched peptides.

### Data Protocol
The resulting raw files with MS/MS spectra were converted into mzXML files and then searched against the bakerâ€™s yeast (Saccharomyces cerevisiae) or human (Homo sapiens) protein database (downloaded from Uniprot) using the SEQUEST algorithm (version 28).1 The following parameters were used during the search: 10 ppm precursor mass tolerance; 0.025 Da product ion mass tolerance; two missed cleavages; variable modifications: oxidation of methionine (+15.9949 Da) and modification on the cysteine residue (+158.0691); static modifications: TMT (+229.1629) on the lysine residue and the peptide N-terminus. For the newly synthesized protein experiment, cysteine modification was set as +387.2321,   and heavy lysine (+8.0142) and heavy arginine (+6.0201) were added to differential modifications. The false discovery rates (FDR) of cysteine-containing peptide and protein identifications were evaluated and controlled by the target-decoy method.2 Each protein sequence was listed in both forward and reversed orders. Linear discriminant analysis (LDA) was employed to control the quality of peptide identifications using the multiple parameters, including XCorr, mass accuracy (ppm), peptide length, and charge state.3 Peptides with shorter than seven amino acid residues in length were discarded. Furthermore, peptide spectral matches were filtered to < 1%. The dataset was restricted to cysteine-containing peptides when determining FDRs.

### Publication Abstract
Protein structures are decisive for their activities and interactions with other molecules. Global analysis of protein structures and conformational changes cannot be achieved by commonly used abundance-based proteomics. Here, we integrated cysteine covalent labeling, selective enrichment, and quantitative proteomics to study protein structures and structural changes on a large scale. This method was applied to globally investigate protein structures in HEK293T cells and protein structural changes in the cells with the tunicamycin (Tm)-induced endoplasmic reticulum (ER) stress. We quantified several thousand cysteine residues, which contain unprecedented and valuable information of protein structures. Combining this method with pulsed stable isotope labeling by amino acids in cell culture, we further analyzed the folding state differences between pre-existing and newly synthesized proteins in cells under the Tm treatment. Besides newly synthesized proteins, unexpectedly, many pre-existing proteins were found to become unfolded upon ER stress, especially those related to gene transcription and protein translation. Furthermore, the current results reveal that N-glycosylation plays a more important role in the folding process of the tertiary and quaternary structures than the secondary structures for newly synthesized proteins. Considering the importance of cysteine in protein structures, this method can be extensively applied in the biological and biomedical research fields.

### Keywords
Human hek293t lc-ms/ms

### Affiliations
School of Chemistry and Biochemistry Georgia Institute of Technology
Georgia Institute of Technology

### Submitter
Kejun Yin

### Lab Head
Dr Ronghu Wu
School of Chemistry and Biochemistry Georgia Institute of Technology


