### Accession
PXD003202

### Title
Symbiosis induces widespread changes in the proteome of the model cnidarian Aiptasia

### Description
Coral reef ecosystems are metabolically founded on the mutualism between corals and photosynthetic dinoflagellates of the genus Symbiodinium. The glass anemone Aiptasia sp. has become a tractable model for this symbiosis. We utilized label-free liquid chromatography electrospray-ionization tandem mass spectrometry to analyze the effects of symbiosis on the proteomes of symbiotic and aposymbiotic Aiptasia. We quantified more than 3,300 proteins in 1,578 protein clusters, with 81 protein clusters showing significantly different expression. Symbiotic anemones showed higher expression of proteins involved in lipid storage and transport, nitrogen transport and cycling, intracellular trafficking, endocytosis and inorganic carbon transport. These changes reflect shifts in host metabolism and energy reserves due to increased organic and inorganic nutritional exchange with the symbionts. Aposymbiotic anemones exhibited increased expression of multiple systems responsible for mediating reactive oxygen stress, suggesting that the host derives direct or indirect protection from oxidative stress while in symbiosis. Aposymbiotic anemones also increased their expression of an array of proteases and chitinases, indicating a metabolic shift from autotrophy to heterotrophy. These results provide a comprehensive Aiptasia proteome with more direct relative quantification of protein abundance than transcriptomic methods, allowing more powerful studies of coral physiology and ecosystem function.

### Sample Protocol
LC-ESI MS/MS with label-free quantification of whole animal tissue, symbionts removed. Animal samples dissolved with sodium deoxycholate buffer modified from Leon et al 2013.

### Data Protocol
The LC-ESI-MS/MS spectra were searched against a custom protein database constructed from an Aiptasia genome concatenated with all cnidarian sequences from Uniprot (28/4/14 release, Swiss-Prot and TrEMBL), all open reading frames (min. 100 base pairs) from the Aiptasia CC7 line transcriptome generated by Lehnert et al. (2012) and a common contaminant database (CRAPome v1.0). The database was processed with MScDB software to reduce peptide-level redundancy. The combined database contains 320,798 amino acid sequences (Table 1). Searches were conducted using the SEQUEST HT algorithm and in Proteome Discoverer 1.4.1.14 (Thermo Scientific) with Percolator scoring. SEQUEST HT searches were conducted assuming trypsin digestion with a maximum of two missed cleavages, a minimum peptide length of six amino acids, one minimum matching peptide per protein, maximum delta Cn of 0, a fragment ion mass tolerance of 0.60 Da and a parent ion tolerance of 10.0 parts per million. Carbamidomethylation of cysteine was specified as a fixed modification, and oxidation of methionine and carbamylation of the N-terminus were specified as variable modifications.   The Proteome Discoverer search result files (.msf) were exported to Scaffold (4.3.4, Proteome Software Inc.) to further validate identifications and for quantification via total ion count (TIC). All LC-ESI-MS/MS samples were analyzed using SEQUEST and additional X!Tandem (The GPM, thegpm.org; version Cyclone 2010.12.01.1) assuming trypsin digestion with a maximum of two missed cleavages. Carbamidomethylation of cysteine was specified as a fixed modification, oxidation of methionine and carbamylation of the N-terminus were specified as variable modifications. Protein probabilities were assigned by the Protein Prophet algorithm. The false discovery rate (FDR) was calculated empirically using a decoy database with the methods of Kalli and Hess  (2012). The FDR threshold for reported peptides and proteins was set to 0.1% and 1%, respectively, with a minimum of two peptides per protein. Proteins that shared significant peptide evidence were grouped into clusters to satisfy the principles of parsimony using Scaffoldâ€™s Protein Cluster Analysis algorithm. Specifically, for two or more proteins to be clustered, the sum of the probabilities of shared peptides had to be greater than 95%, while sharing at least 50% total identification evidence with at least one other member of the cluster.

### Publication Abstract
Coral reef ecosystems are metabolically founded on the mutualism between corals and photosynthetic dinoflagellates of the genus Symbiodinium. The glass anemone Aiptasia sp. has become a tractable model for this symbiosis, and recent advances in genetic information have enabled the use of mass spectrometry-based proteomics in this model. We utilized label-free liquid chromatography electrospray-ionization tandem mass spectrometry to analyze the effects of symbiosis on the proteomes of symbiotic and aposymbiotic Aiptasia. We identified and obtained relative quantification of more than 3,300 proteins in 1,578 protein clusters, with 81 protein clusters showing significantly different expression between symbiotic states. Symbiotic anemones showed significantly higher expression of proteins involved in lipid storage and transport, nitrogen transport and cycling, intracellular trafficking, endocytosis and inorganic carbon transport. These changes reflect shifts in host metabolism and nutrient reserves due to increased nutritional exchange with the symbionts, as well as mechanisms for supplying inorganic nutrients to the algae. Aposymbiotic anemones exhibited increased expression of multiple systems responsible for mediating reactive oxygen stress, suggesting that the host derives direct or indirect protection from oxidative stress while in symbiosis. Aposymbiotic anemones also increased their expression of an array of proteases and chitinases, indicating a metabolic shift from autotrophy to heterotrophy. These results provide a comprehensive Aiptasia proteome with more direct relative quantification of protein abundance than transcriptomic methods. The extension of "omics" techniques to this model system will allow more powerful studies of coral physiology, ecosystem function, and the effects of biotic and abiotic stress on the coral-dinoflagellate mutualism.

### Keywords
Symbiosis, Coral, Cnidarian, Label-free, Model organism, Aiptasia

### Affiliations
Victoria University of Wellington
School of Biological Sciences, Victoria University of Wellington, Wellington, New Zealand

### Submitter
Clinton Oakley

### Lab Head
Dr Simon Davy
School of Biological Sciences, Victoria University of Wellington, Wellington, New Zealand


