### Accession
PXD001119

### Title
Targeted Proteomics of NERD and FLS2 on LTQ-Orbitrap Velos Mass Spectrometer

### Description
Targeted proteomics has become increasingly popular recently because of its ability to precisely quantify selected proteins in complex cellular backgrounds. We demonstrate the utility of an LTQ-Orbitrap Velos Pro mass spectrometer in targeted parallel reaction monitoring (PRM) despite its unconventional dual ion trap configuration. We evaluated absolute specificity (>99%) and sensitivity (100 amol on column in 1Âµg total cellular extract) using full and mass range scans as survey scans together with data dependent (DDA) and targeted MS/MS acquisition. The instrument duty cycle was a critical parameter limiting sensitivity necessitating peptide retention time scheduling. We assessed synthetic peptide and recombinant peptide standards to predict or experimentally determine target peptide retention times. We applied optimized PRM to protein degradation in signaling regulation, an area that is receiving increased attention in plant physiology. We quantified relative abundance of selected proteins in plants mutant for enzymatic components of the N-end rule degradation (NERD) pathway such as the two tRNA-arginyl-transferases ATE1 and ATE2 and the two E3 ubiquitin ligases PROTEOLYSIS1 and 6 searching for upregulated proteins. We also targeted FLAGELLIN SENSITIVE2 (FLS2), a pattern recognition receptors responsible pathogen sensing, in ubiquitin ligase mutants to assay the attenuation of plant immunity by degradation of the receptor.

### Sample Protocol
Total proteins were extracted form Arabidopsis thaliana wild type and mutant plants using a phenol based extraction protocol. Total proteins were in-solution digested with trypsin or separated using SDS-PAGE followed by in gel-digestion of selected gel bands. Peptides were separated using C18 reverse phase liquid chromatography and electrosprayed on-line into an LTQ-Orbitrap Velos Pro mass spectrometer. Various targeted PRM based proteomics strategies were evaluated using both standard peptides and peptides from selected proteins of interest. Proteins possibly affected by the NERD protein degradation pathway and the pattern recognition receptor FLS2 were quantified using the validated and optimized PRM based strategy in treated and untreated wild type and the mutant genotypes.

### Data Protocol
Raw data from the mass spectrometer was analysed using the Proteome Discoverer version 1.4. software suite controlling the Mascot search engine version 2.4.0 on a local server to identify peptides and proteins. For MS/MS peak list generation minimum S/N was set to 1.5. For database search the enzyme was set to trypsin tolerating two missed cleavages. The parent mass tolerance was set to 7 ppm, the MS/MS fragment mass tolerance to 0.8 Da. Cysteine carbamidomethylation was set as a fixed modification, oxidation of methionine was tolerated as a variable modification. Global false discovery rate (FDR) and spectrum posterior error probabilities (PEP) were calculated with the reversed decoy database model using the target decoy PSM validator module in Proteome Discoverer. Q-values and PEPs were reported automatically for each PSM by Proteome Discoverer. The parameters were set to the commonly accepted default values of 0.05 and 0.01. Peptide precursor and product ion chromatograms were extracted and ion signal peaks were integrated for combined or separate precursor and PRM quantification using the Skyline software. The following transition settings were used: (i) filter settings: precursor charges 2, 3; ion charges 1, 2; and ion types b,y and p (precursor) as well as 6 product ions from m/z to precursor. (ii) library settings: ion match tolerance 0.5 Da and pick 3 product ions. (iii) full scan MS1 filtering: isotope peaks included count; precursor mass analyser Orbitrap; peaks 1 and resolving power 30,000 at m/z 400.  (iv) MS/MS filtering: acquisition method targeted; product mass analyser QIT; EIC peak width 0.5 Da and (v) retention time filtering: use only scans within 1 minute of MS/MS IDs.

### Publication Abstract
Targeted proteomics has become increasingly popular recently because of its ability to precisely quantify selected proteins in complex cellular backgrounds. Here, we demonstrated the utility of an LTQ-Orbitrap Velos Pro mass spectrometer in targeted parallel reaction monitoring (PRM) despite its unconventional dual ion trap configuration. We evaluated absolute specificity (&gt;99%) and sensitivity (100 amol on column in 1 &#x3bc;g of total cellular extract) using full and mass range scans as survey scans together with data-dependent (DDA) and targeted MS/MS acquisition. The instrument duty cycle was a critical parameter limiting sensitivity, necessitating peptide retention time scheduling. We assessed synthetic peptide and recombinant peptide standards to predict or experimentally determine target peptide retention times. We applied optimized PRM to protein degradation in signaling regulation, an area that is receiving increased attention in plant physiology. We quantified relative abundance of selected proteins in plants that are mutant for enzymatic components of the N-end rule degradation (NERD) pathway such as the two tRNA-arginyl-transferases ATE1 and ATE2 and the two E3 ubiquitin ligases PROTEOLYSIS1 and 6. We found a number of upregulated proteins, which might represent degradation targets. We also targeted FLAGELLIN SENSITIVE2 (FLS2), a pattern recognition receptor responsible for pathogen sensing, in ubiquitin ligase mutants to assay the attenuation of plant immunity by degradation of the receptor.

### Keywords
Fls2, Targeted proteomics, Ltq-orbitrap, Prm, Nerd

### Affiliations
Leibniz Institute of Plant Biochemistry

### Submitter
Wolfgang Hoehenwarter

### Lab Head
Dr Wolfgang Hoehenwarter
Leibniz Institute of Plant Biochemistry


