### Accession
PXD030139

### Title
Octamer sequences with HEXplorer score amplitude of -10.35, –0.15 or +10.32

### Description
Correct pre-mRNA processing in higher eukaryotes vastly depends on splice site recognition. Beyond conserved 5’ss and 3’ss motifs, splicing regulatory elements (SREs) play a pivotal role in this recognition process. Here, we present in silico designed sequences with arbitrary a priori prescribed splicing regulatory HEXplorer properties that can be concatenated to arbitrary length without changing their regulatory properties. We performed pulldown analysis to identify RNA-binding proteins, that bind either sequence segments with a HEXplorer score amplitude of +10.32, –0.15 or -10.35.

### Sample Protocol
Samples were prepared by in-gel digestion with trypsin essentially as describedd (PMID: 29322779). Resulting peptides were separated over 2h on C18 material using an U3000 Rapid Separation Liquid Chromatography system and anlalysed on an Orbitrap Elite essentially as described in PMID  32202429

### Data Protocol
Mass spectrometric data was analysed with MaxQuant version 1.5.7.0 (PMID: 27809316) for protein identification and quantification essentially using standard parameters. 'Match between runs' was enabled as well as LFQ quantification. Searches were carried out in Homo sapiens proteome (UP000005640) sequences downloaded from UniProt KB on 16th August 2016 with a sequence entry of MBP-MS2 added.

### Publication Abstract
Correct pre-mRNA processing in higher eukaryotes vastly depends on splice site recognition. Beyond conserved 5'ss and 3'ss motifs, splicing regulatory elements (SREs) play a pivotal role in this recognition process. Here, we present in silico designed sequences with arbitrary a priori prescribed splicing regulatory HEXplorer properties that can be concatenated to arbitrary length without changing their regulatory properties. We experimentally validated in silico predictions in a massively parallel splicing reporter assay on more than 3000 sequences and exemplarily identified some SRE binding proteins. Aiming at a unified 'functional splice site strength' encompassing both U1 snRNA complementarity and impact from neighboring SREs, we developed a novel RNA-seq based 5'ss usage landscape, mapping the competition of pairs of high confidence 5'ss and neighboring exonic GT sites along HBond and HEXplorer score coordinate axes on human fibroblast and endothelium transcriptome datasets. These RNA-seq data served as basis for a logistic 5'ss usage prediction model, which greatly improved discrimination between strong but unused exonic GT sites and annotated highly used 5'ss. Our 5'ss usage landscape offers a unified view on 5'ss and SRE neighborhood impact on splice site recognition, and may contribute to improved mutation assessment in human genetics.

### Keywords
Slice site, Hexplorer, Splicing, Splicing regulatory element

### Affiliations
Molecular Proteomics Laboratory BMFZ Heinrich Heine University Düsseldorf Germany
Molecular Proteomics Laboratory

### Submitter
Gereon Poschmann

### Lab Head
Dr Gereon Poschmann
Molecular Proteomics Laboratory BMFZ Heinrich Heine University Düsseldorf Germany


