### Accession
PXD014754

### Title
C1R.A*01:01, C1R.A*02:01, C1R.A*24:02 MHC class I immunopeptidome LC-MSMS

### Description
Characterisation of peptide ligands of Major histocompatibility class (MHC) I isolated by immunoaffinity purification from the C1R (Class I reduced) B-lymphoblastoid cell line, transfected with the MHC class I allele HLA-A*01:01, or HLA-A*02:01, HLA-A*24:02. In addition, public mass spectrometry (MS) datasets of HLA-I and HLA-II immunopeptidome derived from patientsâ€™ samples, PBMC or cell lines, and shotgun proteomics from trypsin/elastase digestion were analysed.

### Sample Protocol
Snap frozen pellets of 5 x 10^9 C1R transfected cells were lysed by a mixture of detergent (NP-40) and mechanical lysis. The MHC complexes were isolated from the lysate using solid phase bound W6/32 monoclonal antibody (BB7.2 for C1R.A*02:01) and eluted with 10% acetic acid. MHC/peptide mixtures were fractioned by RP-HPLC and peptide containing fractions vacuum concentrated prior to reconstitution in 0.1% Formic acid. Peptides were analysed in positive mode by an AB SCIEX 6600 TripleTOF mass spectrometer. Data were collected from 2 pellets.

### Data Protocol
Peptide spectra were assigned using PEAKS Studio 8.5, searching against the reviewed human proteome (swiss-prot accessed November 2018) with the following parameters: Parent Mass Error Tolerance: 50 ppm; Fragment Mass Error Tolerance; 0.1 Da, Enzyme: None; Instrument: TripleTOF 6600; Max Variable PTM Per Peptide: 3; Variable Modifications: Deamidation(NQ), Citrullination, Phosphorylation (STY), Oxidation (M). For public MS datasets, the parameters were set as based on the default settings for the MS instrument used. In addition, a PEAKS PTM search was performed on in-house generated datasets as well as public MS datasets of HLA-I immunopeptidome.

### Publication Abstract
The presentation of post-translationally modified (PTM) peptides by cell surface HLA molecules has the potential to increase the diversity of targets for surveilling T cells. Although immunopeptidomics studies routinely identify thousands of HLA-bound peptides from cell lines and tissue samples, in-depth analyses of the proportion and nature of peptides bearing one or more PTMs remains challenging. Here we have analyzed HLA-bound peptides from a variety of allotypes and assessed the distribution of mass spectrometry-detected PTMs, finding deamidation of asparagine or glutamine to be highly prevalent. Given that asparagine deamidation may arise either spontaneously or through enzymatic reaction, we assessed allele-specific and global motifs flanking the modified residues. Notably, we found that the <i>N</i>-linked glycosylation motif NX(S/T) was highly abundant across asparagine-deamidated HLA-bound peptides. This finding, demonstrated previously for a handful of deamidated T cell epitopes, implicates a more global role for the retrograde transport of nascently N-glycosylated polypeptides from the ER and their subsequent degradation within the cytosol to form HLA-ligand precursors. Chemical inhibition of Peptide:N-Glycanase (PNGase), the endoglycosidase responsible for the removal of glycans from misfolded and retrotranslocated glycoproteins, greatly reduced presentation of this subset of deamidated HLA-bound peptides. Importantly, there was no impact of PNGase inhibition on peptides not containing a consensus NX(S/T) motif. This indicates that a large proportion of HLA-I bound asparagine deamidated peptides are generated from formerly glycosylated proteins that have undergone deglycosylation via the ER-associated protein degradation (ERAD) pathway. The information herein will help train deamidation prediction models for HLA-peptide repertoires and aid in the design of novel T cell therapeutic targets derived from glycoprotein antigens.

### Keywords
Erad, Mass spectromtry, Deamidation, Hla

### Affiliations
Infection and Immunity Program and Department of Biochemistry and Molecular Biology, Monash Biomedicine Discovery Institute, Monash University, Clayton, Victoria 3800, Australia (lab head) (lab head)
Monash University

### Submitter
Shutao Mei

### Lab Head
Dr Anthony W. Purcell
Infection and Immunity Program and Department of Biochemistry and Molecular Biology, Monash Biomedicine Discovery Institute, Monash University, Clayton, Victoria 3800, Australia (lab head) (lab head)


