### Accession
PXD009498

### Title
SILAC APMS analysis of CCDC103 interactome

### Description
To get insights into the function of CCD103, we characterized its partners by performing a proteomic analysis in human cells. We fused this domain to GFP and stably expressed it in HeLa cells using site-specific integration with the Flp-In system. Following differential labeling of GFP-CCDC103 and control cells with isotopically labelled amino-acids (SILAC), whole cell extracts were immuno-precipitated (IP) with anti-GFP antibodies and immunoprecipitates were subjected to quantitative mass-spectrometry analysis.  SILAC labels: K0R0 control Hela H9 vs K4R6 x-FLAG- GFP-CCDC103

### Sample Protocol
HeLa cells were grown for 15 days in each isotopically labeled media (CIL/Eurisotop), to ensure complete incorporation of isotopically labeled arginine and lysine (light label (K0R0, L) or heavy label L-Lysine-2HCl (2H4, 96–98%)/L-Arginine-HCl (13C6, 99%) (K4R6, M) (percentages represent the isotopic purity of the labeled amino acids). Six 15-cm diameter plates were used per SILAC condition. Cells were rinsed with PBS, trypsinized and cryogrinded in lysis buffer (0.5% triton X-100, 20 mM HEPES, pH 7.4, 150 mM NaCl, protease inhibitor cocktail). Extracts were incubated 20 min at 4°C and clarified by centrifugation for 10 min at 20,000 g. For all IP experiments, extracts were pre-cleared by incubation with Protein G Sepharose beads (GE healthcare) for 1 h at 4°C. The control was extracted from the SILAC light condition prepared from H9 HeLa cells that did not express the GFP fusion. Each extract was then incubated with 50 µl of GFP-Trap beads (ChromoTek) for 75 min at 4°C, washed five times with lysis buffer, and beads from the different isotopic conditions were finally pooled. Bound proteins were eluted by adding 1% SDS to the beads and boiling for 10 min. The eluate was then reduced with DTT (BDH 443553B, 10 mM) at 95°C for 2 min and alkylated using iodoacetamide (Sigma I1149, 50 mM) for 30 min in the dark. Proteins were separated on SDS/PAGE, the lanes of interest were cut in 10 slices, and proteins were in gel-digested with trypsin in 20 mM NH4HCO3 (Trypsin Gold, Promega V5280). Peptides were extracted from gel pieces and resuspended in 0.1% formic acid/2% acetonitrile solution before being analyzed by mass spectrometry. Peptides were analyzed by nano-flow liquid chromatography coupled to Fourier transform tandem mass spectrometry (nanoLC-FT-MS/MS) using a LTQ Velos Pro Orbitrap Elite mass spectrometer coupled to an Ultimate 3000 (Thermo Fisher Scientific). Desalting and pre-concentration of samples were performed on-line on a Pepmap precolumn (0.3 mm 10 mm, Thermo Fisher Scientific) in buffer A (2% acetonitrile, 0.1% formic acid). A gradient consisting of 2–40% buffer B (B = 99.9% acetonitrile with 0.1% formic acid; 3–33 min), 40–80% B (33–34 min), was used to separate peptides at 300 nL/min from a Pepmap capillary reversed-phase column (0.075 mm x 150 mm, Thermo Fisher Scientific). Mass spectra were acquired using a top-20 collision-induced dissociation (CID) data-dependent acquisition (DDA) method. The Orbitrap was programmed to perform a FT 400–1,400 Th mass range  scan (60,000 resolution) with the top 20  ions in intensity selected for collision-induced dissociation (CID) data-dependent acquisition (DDA) MS/MS in the LTQ. FT spectra were internally calibrated using a single lock mass (445.1200 Th). Target ion numbers were 500,000 for FT full scan on the Orbitrap and 10,000 MSn on the LTQ.

### Data Protocol
Data were acquired using the Xcalibur software v2.2. Protein identification and quantitation were performed using the program MaxQuant (version 1.5.2.8; http://www.maxquant.org/). Few parameters were not default: database: human reference proteome set (canonical isoforms downloaded from Expasy on May 29th 2017); enzyme specificity trypsin/P; variable modifications: methionine oxidation and protein N-Acetylation; Fixed modifications: Cysteine carbamidomethylation; MS/MS tolerance: 0.5 Da; False Discovery Rate (FDR): 1%. In addition to the FDR, proteins were considered to be identified if they had at least two peptides including one unique/Razor peptide and they were considered quantified if they had at least one quantified SILAC pairs. Proteins labeled as REV (non-real proteins from the reverse database) and CONT (contaminants) were automatically discarded, as well as proteins that did not show any SILAC M/L, H/L and H/M ratio. B Significance calculation were done with the software Perseus v1.4.2 as previously described 59 to highlight statistically significant protein ratios (pvalue<0.05).

### Publication Abstract
None

### Keywords
Human, Hela cells, Ltq-orbitrap, Apms, Silac

### Affiliations
IGMM, CNRS, Université de Montpellier, Montpellier, France
IGF, CNRS, INSERM, University of Montpellier, Montpellier, France

### Submitter
Franck Vandermoere

### Lab Head
Dr Edouard Bertrand
IGMM, CNRS, Université de Montpellier, Montpellier, France


