### Accession
PXD014909

### Title
Human HEK293T nuclear extract LC-MSMS

### Description
Five tandem-domain reader proteins (BPTF, CBP, TAF1, TRIM24 and TRIM33) were designed and prepared as the enriched probes, and a group of binding proteins from HEK293T were enriched by domain probes and identified by HPLC-MS/MS.

### Sample Protocol
All proteins were digested in gel by trypsin first, then each sample of peptides was reconstituted in 7 µl HPLC buffer A (0.1% (v/v) formic acid in water) and 5 µl was injected into a Nano-LC system (EASY-nLC 1000, Thermo Fisher Scientific, Waltham, MA). We used C18 column (50 μm inner-diameter × 15 cm, 2 μm C18) to separate each sample with a 85-minute HPLC-gradient (linear gradient from 2 to 35% HPLC buffer B 0.1% formic acid in acetonitrile in 75 minutes and then to 90% buffer B in 10 minutes). HPLC elution was electro-sprayed to an Orbit rap Q-Exactive mass spectrometer (Thermo Fisher Scientific, Waltham, MA). The source was operated at 1.8 kV. We carried out the mass spectrometric analysis in data-dependent mode with an automatic switch between a full MS scan and an MS/MS scan in the orbit rap. The automatic gain control (AGC) target was 3e6; scan range was from 400 to 1350, with a resolution of 70,000 in full MS survey scan. We selected the 10 most intense peaks with charge state 2 and above as fragmentation by higher-energy collision dissociation (HCD) with normalized collision energy of 27%. MS2 spectra were acquired with 17,500 resolutions. Finally we searched the MS/MS data results against UniProt database using MaxQuant software (v1.5.2.8) with a less than 1% overall false discovery rate (FDR) for peptides. And the peptide sequences were searched using trypsin specificity, with a maximum of two missed cleavages.  The nuclear proteins enriched by tandem-domain probes (BPTF-PB, CBP-BP, TRIM24-PB, and TAF1-BB) were set as experiment group, while the nuclear proteins enriched only by Glutatahione Sepharose 4B beads (GE Healthcare) were set as control group. After SDS-PAGE, the proteins of experimental group and control group were digested in gel and identified by HPLC-MS/MS. Every identified protein has a LFQ intensity after searching by MaxQuant software (v1.5.2.8). The ratio was obtained by dividing the control LFQ intensity by the experimental LFQ intensity of each identified proteins. The ratio greater than 2.0-fold (ratio >2.0) was defined as "upregulated" proteins and ratio lower than 0.5-fold (ratio <0.5) was defined as "downregulated" proteins. Then the bioinformatics analysis of upregulated proteins was carried out by DAVID and KEGG data base furtherly.

### Data Protocol
Categorical annotation was supplied in the form of Gene Ontology (GO). Database for Annotation Visualization and Integrated Discovery (DAVID) was used to analyze the biological process (BP), molecular function (MF) and cellular component (CC) of the proteins. Distribution of different proteins in metabolic pathways was showed by Kyoto Encyclopedia of Genes and Genomes (KEGG) pathways.

### Publication Abstract
Histone posttranslational modifications (HPTMs) play important roles in eukaryotic transcriptional regulation. Recently, it has been suggested that combinatorial modification codes that comprise two or more HPTMs can recruit readers of HPTMs, performing complex regulation of gene expression. However, the characterization of the multiplex interactions remains challenging, especially for the molecular network of histone PTMs, readers and binding complexes. Here, we developed an integrated method that combines a peptide library, affinity enrichment, mass spectrometry (MS) and bioinformatics analysis for the identification of the interaction between HPTMs and their binding proteins. Five tandem-domain-reader proteins (BPTF, CBP, TAF1, TRIM24 and TRIM33) were designed and prepared as the enriched probes, and a group of histone peptides with multiple PTMs were synthesized as the target peptide library. First, the domain probes were used to pull down the PTM peptides from the library, and then the resulting product was characterized by MS. The binding interactions between PTM peptides and domains were further validated and measured by isothermal titration calorimetry analysis (ITC). Meanwhile, the binding proteins were enriched by domain probes and identified by HPLC-MS/MS. The interaction network of histone PTMs-readers-binding complexes was finally analyzed via informatics tools. Our results showed that the integrated approach combining MS analysis with ITC assay enables us to understand the interaction between the combinatorial HPTMs and reading domains. The identified network of "HPTMs-reader proteins-binding complexes" provided potential clues to reveal HPTM functions and their regulatory mechanisms.

### Keywords
Nuclear extract, Hek293t, Lc-msms

### Affiliations
Tianjin Medical University
2011 Collaborative Innovation Center of Tianjin for Medical Epigenetics, Tianjin Key Laboratory of Medical Epigenetics, Key Laboratory of Immune Microenvironment and Disease (Ministry of Education), Department of Biochemistry and Molecular Biology, Tianjin Medical University, Tianjin 300070, China

### Submitter
Chen Pu

### Lab Head
Dr Zhang Kai
2011 Collaborative Innovation Center of Tianjin for Medical Epigenetics, Tianjin Key Laboratory of Medical Epigenetics, Key Laboratory of Immune Microenvironment and Disease (Ministry of Education), Department of Biochemistry and Molecular Biology, Tianjin Medical University, Tianjin 300070, China


