### Accession
PXD005349

### Title
Combining Affinity Enrichment, Cross-Linking with Photo-Amino Acids, and Mass Spectrometry for Probing Protein Kinase D2 Interactions

### Description
We present an approach that relies on the affinity enrichment of a target protein from a complex mixture, followed by a UV-induced activation of incorporated diazirine photo-reactive amino acids (photo-methionine and photo-leucine) and a covalent fixation of interaction partners. The captured protein complexes are enzymatically digested and interacting proteins are identified and quantified by label-free LC/MS analysis. Using HeLa cell lysates with photo-methionine and photo-leucine-labeled proteins, we were able to capture and preserve protein interactions that are otherwise elusive in conventional pull-down experiments. Our approach is exemplified for mapping the protein interaction network of protein kinase D2, but can be applied to any protein system.

### Sample Protocol
The affinity enrichment and crosslinking experiments were carried out at 4 °C using spin columns (Pierce Screw Cap, Thermo Scientific). After equilibration of 20 µg glutathione sepharose matrix (4B, GE Healthcare) by washing three times with 400 µl of binding buffer (50 mM HEPES, 150 mM NaCl, 2 mM MgCl2, 1 mM TCEP, 10% (v/v) glycerol, pH 7.5), 5 µg of GST-PKD2 in 200 µl of binding buffer were bound by incubation for 1 h. Additionally, GST and matrix control samples were prepared using equimolar amounts of GST (Pierce, Thermo Fisher Scientific) or binding buffer. Affinity-captured GSTPKD2 and the control samples were incubated with 200 µl of soluble photoaffinity-labeled protein fractions for 2 h. Photo-crosslinking was carried out by UV-A irradiation with a dose of 8000 mJ cm-2 in a custom-built irradiation chamber. After sample irradiation, the sepharose matrix was washed three times with 400 µl of binding buffer with an increased salt concentration (500 mM NaCl). Subsequently, each sample was resuspended in 40 µl of 8 M urea/400 mM ammonium bicarbonate, incubated at 37 °C (15 min, 400 rpm) and subjected to in-solution digestion with Glu-C and trypsin. Peptide solutions were analyzed by LC-MS/MS on an Ultimate 3000 RSLC nano-HPLC system (Thermo Fisher Scientific) coupled to an Orbitrap Fusion mass spectrometer (Thermo Fisher Scientific) via a Nanospray Flex nano-ESI ion source (Thermo Fisher Scientific). Samples were loaded onto a RP C8 precolumn (Acclaim PepMap, 300 μm × 5 mm, 5 μm, 100 Å, Thermo Fisher Scientific) at a flow rate of 30 µl/min. After washing on the precolumn with 0.1% (v/v) TFA for 15 min at 30 µl/min, peptides were eluted and separated on a C18 separation column (Acclaim PepMap, 75 μM × 250 mm, 2 μm, 100 Å, Thermo Fisher Scientific) that had been equilibrated with solvent A (0.1% (v/v) FA). Elution of peptides from the separation column was performed with a gradient from 0-35% solvent B (ACN, 0.08% (v/v) FA) within 180 min at 300 nl/min. Peptides were directly transferred to the nano-ESI ion source and analyzed in data-dependent MS/MS acquisition mode: High-resolution precursor ion scans (m/z 300-1500, R = 120,000 at m/z 200, AGC target value = 4.0e5, max. injection time = 50 ms) in the Orbitrap analyzer were acquired every 5 s. Within these 5 sec the most abundant signals (charge states 2-8, intensity threshold = 3000 counts) were selected for MS/MS experiments using HCD with a normalized collision energy of 27% and an isolation window of 2 Th. Fragment ion spectra were acquired in the Orbitrap (R = 15,000 at m/z 200, AGC target value = 5.0e4, max. injection time = 200 ms). Fragmented precursor ions were dynamically excluded from HCD for 60 s if targeted two times within 30 s (mass tolerance window 2 ppm). Data acquisition and inspection of raw data were conducted via the Xcalibur software (version 3.0.63, Thermo Fisher Scientific).

### Data Protocol
The affinity enrichment photo-crosslinking experiments were conducted as biological replicates, i.e., independent cell culture and photo-reactive amino acid incorporation, with two technical replicates, i.e., affinity enrichment, photo-crosslinking, and MS analysis each. Raw files from LC-MS/MS measurements were analyzed using the MaxQuant software (version 1.5.2.8). Protein identification was performed with the Andromeda search engine using the human reference proteome downloaded from the UniProt database (http://www.uniprot.org, FASTA file; download date: 2015-01-20; 89,734 entries) and an integrated database of common contaminants (245 entries). Database searches were conducted with standard settings for Orbitrap precursor and fragment ion analysis allowing maximum mass differences of 20 ppm (first search) / 4.5 ppm (main search) and 20 ppm. Trypsin and Glu-C were set as specific proteolytic enzymes, allowing up to three missed cleavage sites. Carbamidomethylation of cysteine was set as fixed modification and oxidation of methionine was set as variable modification. Protein identifications were filtered using a target decoy database (reversed sequence order) allowing a maximum false discovery rate of 1%. Furthermore, the “match between runs” feature was enabled and label-free quantification of proteins (LFQ) was conducted using MaxLFQ (minimum ratio count = 1). The “proteinGroups” output file was used for subsequent data analysis. Analysis of data processed with MaxQuant was carried out with the Perseus software (version 1.5.1.6) as described previously. LFQ intensities of identified proteins stored in the “proteinGroups” file were loaded as expression values for a pairwise comparison of each sample to the matrix control sample. Proteins identified only with modified peptides or in the decoy database and potential contaminants were discarded. Afterward, LFQ intensities were logarithmized and replicates were grouped for each sample. The sample groups were filtered for proteins with at least six valid values for LFQ intensities in at least one group. Missing values were imputed from the total matrix based on a normal distribution using default software parameters. Statistical analysis was conducted to compare LFQ intensities of each sample replicate group with the matrix control replicate group using a student’s t-test with a significance threshold of p-value ≤ 0.01. The logarithmized ratios of the means of both groups were plotted against the negative logarithmized p-values as “volcano” plots to visualize significant differences of LFQ intensities. Enrichment factors of proteins were calculated as the ratios of mean LFQ intensities in each sample and mean LFQ intensities in the matrix controls. Overlap of proteins with significant LFQ intensity differences were assigned and visualized with a Venn diagram generator.

### Publication Abstract
We present a novel approach that relies on the affinity capture of protein interaction partners from a complex mixture, followed by their covalent fixation via UV-induced activation of incorporated diazirine photoreactive amino acids (photo-methionine and photo-leucine). The captured protein complexes are enzymatically digested and interacting proteins are identified and quantified by label-free LC/MS analysis. Using HeLa cell lysates with photo-methionine and photo-leucine-labeled proteins, we were able to capture and preserve protein interactions that are otherwise elusive in conventional pull-down experiments. Our approach is exemplified for mapping the protein interaction network of protein kinase D2, but has the potential to be applied to any protein system. Data are available via ProteomeXchange with identifiers PXD005346 (photo amino acid incorporation) and PXD005349 (enrichment experiments).

### Keywords
Photo-cross-linking, Photo-methionine, Photo-leucine, Affinity enrichment, Pkd2, Protein interaction, Orbitrap fusion

### Affiliations
Department of Pharmaceutical Chemistry and BioanalyticsInstitute of Pharmacy Martin Luther University Halle-Wittenberg Wolfgang-Langenbeck-St. 4 D-06120 Halle (Saale), Germany
Martin Luther University Halle-Wittenberg

### Submitter
Björn Häupl

### Lab Head
Dr Andrea Sinz
Department of Pharmaceutical Chemistry and BioanalyticsInstitute of Pharmacy Martin Luther University Halle-Wittenberg Wolfgang-Langenbeck-St. 4 D-06120 Halle (Saale), Germany


