### Accession
PXD009671

### Title
The proteomics analyses of different tissues(the posterior silk gland of wild silkworm)

### Description
We collected a total of 9.8 million mass spectra data generated in the laboratory from the proteomics analyses of different silkworm tissues, including the posterior silk gland (29), middle silk gland (30), ovary and testis (31), head (32), brain, prothoracic glands, subesophageal ganglion (33), hemolymph (34), fat body (35) and embryo (36,37) of domestic silkworm, and the posterior silk gland of wild silkworm (38).

### Sample Protocol
Briefly, the sample proteins were extracted and separated by SDS-PAGE, followed by in-gel digestion. The digested peptide mixtures were resuspended with 0.1% methanoic acid, followed by shotgun LC-MS/MS analysis. Additionally, two samples of domestic silkworm posterior silk gland were used for iTRAQ-based proteomics analysis according to a previous study (39). The raw data from LC-MS/MS were converted into mgf format.

### Data Protocol
The B. mori protein database was downloaded from the Silkworm Genome Database web site (http://silkworm.swu.edu.cn/silkdb, containing 14,623 entries, named SilkPDB) and NCBI (http://www.ncbi.nlm.nih.gov/protein/?term=txid7091, release date: Nov 22, 2012, containing 7,859 entries, named as NCBI BmPDB). The core of proteogenomics analysis is the identification of peptides that are not present in the annotated proteome. To identify all potential novel peptides, a database of the six-frame translation of the B. mori genome was constructed (containing 7,786,569 entries, named SilkGDB). The B. mori genome was downloaded from the Silkworm Genome Database (http://silkworm.swu.edu.cn/silkdb). To identify exonâ€“exon junction peptides, which indicate hypothetical splice variants, we constructed an exon-skipping database containing all possible combinations of two exons from the annotated genes in SilkPDB (containing 675,409 entries, named SilkASDB). To confirm the known gene models, we searched all mass spectra data against two protein databases, SilkPDB and NCBI BmPDB. A protein was identified with at least one uniquely mapped peptide. We also searched against two mixed databases, one comprising SilkPDB and SilkGDB and the other comprising SilkASDB and SilkPDB. The following search parameters were used for the Mascot (version 2.3.02) search engine: (1) trypsin was used to cleave the peptides (one missed cleavage was allowed); (2) the peptide mass error tolerance was 20 ppm; and (3) fragment mass tolerance was 0.02 Da for iTRAQ data and 0.6 Da for the others. (4) The carbamidomethyl of cysteine was considered a fixed modification, and the acetylation of the protein N terminus, oxidation of methionine, and deamidation of glutamine and asparagine were considered variable modifications. We filtered peptides using a 1% false discovery rate (FDR) cutoff estimated by searching against a decoy database.

### Publication Abstract
The silkworm genome has been deeply sequenced and assembled, but accurate genome annotation, which is important for modern biological research, remains far from complete. To improve silkworm genome annotation, we carried out a proteogenomics analysis using 9.8 million mass spectra collected from different tissues and developmental stages of the silkworm. The results confirmed the translational products of 4307 existing gene models and identified 1701 novel genome search-specific peptides (GSSPs). Using these GSSPs, 74 novel gene-coding sequences were identified, and 121 existing gene models were corrected. We also identified 1182 novel junction peptides based on an exon-skipping database that resulted in the identification of 973 alternative splicing sites. Furthermore, we performed RNA-seq analysis to improve silkworm genome annotation at the transcriptional level. A total of 1704 new transcripts and 1136 new exons were identified, 2581 untranslated regions (UTRs) were revised, and 1301 alternative splicing (AS) genes were identified. The transcriptomics results were integrated with the proteomics data to further complement and verify the new annotations. In addition, 14 incorrect genes and 10 skipped exons were verified using the two analysis methods. Altogether, we identified 1838 new transcripts and 1593 AS genes and revised 5074 existing genes using proteogenomics and transcriptome analyses. Data are available via ProteomeXchange with identifier PXD009672. The large-scale proteogenomics and transcriptome analyses in this study will greatly improve silkworm genome annotation and contribute to future studies.

### Keywords
Wild silkworm, Proteomics, Posterior silk gland

### Affiliations
College of Animal Sciences, Zhejiang University, Hangzhou, P. R. China
Zhejiang university,china

### Submitter
Boxiong Zhong

### Lab Head
Dr Zhong Bo-Xiong
College of Animal Sciences, Zhejiang University, Hangzhou, P. R. China


