### Accession
PXD006607

### Title
Proteomic analysis of human Medulloblastoma reveals distinct activated pathways between subgroups

### Description
Deregulations in fundamental signaling pathways are key events in pathogenesis of cancer. One intriguing illustration that still holds blind spots is the pediatric brain tumor arising from the developing cerebellum: medulloblastoma (MB). Extensive high-throughput sequencing led to the characterization of four MB subgroups (WNT, SHH, Group 3 and Group 4) delineated with distinct molecular signatures and clinical outcomes. However, up-to-date these analyses have not attained the global comprehension of their dynamic network complexity. Wishing to uncover a comprehensive view of all MB subgroups we employed a proteomic analysis to integrate accurate protein expression and activity that should ultimately give rise to a realistic picture of how MB cancer cells are regulated. In this study we present the first analysis regrouping methylation status, whole-transcriptome sequencing and quantitative proteomics (proteome and phosphoproteome) using a super SILAC / spiked in strategy across 38 flash frozen primary human MBs (5 WNT, 10 SHH, 10 Group 3 and 13 Group 4). First, our data pinpointed that proteomic analysis could reveal MB subgroup identity. Second, analysis of proteome and phosphoproteome highlighted disregulated signaling pathways that have not been predicted by transcriptomic analysis. Altogether, combined multi-scale analyses of MB have allowed us to identify unrevealed pathways involved in human MB genesis and progression.

### Sample Protocol
Tumor samples and SILAC reference were lysed in 1 ml of a buffer containing 8M Urea, 50mM ammonium bicarbonate (ABC), protease inhibitor (Roche #11836170001) and phosphatase inhibitor (Roche #04906837001) using the Precellys 24 system. Lysates were sonicated to further reduce viscosity and were centrifuged at 20,000 g for 10 minutes. Protein concentration was measured using BCA assay (Sigma). Equal amount of tumors and SILAC reference lysate were then mixed (1mg of each) and reduced by adding 5mM of dithiothreitol (Sigma #D0632) for 45 min at room temperature. Samples were subsequently alkylated using iodoacetamide (GE Healthcare #RPN6302) at a final concentration of 10 mM for 30 min in the dark. Samples were then diluted 10 fold with 50 mM ABC to reach a final concentration < 1M of Urea prior digestion with Trypsin-LysC (Promega #V5072) overnight at 37°C. Digested samples were acidified with formic acid (FA; REF) to a final concentration of 2.5% (pH<3). Digested lysates were then centrifugated at 3,000 g for 10 min to remove precipitate. Peptides were desalted using 500mg C18 cartridge (Waters #WAT036945) and dried using SpeedVac apparatus. Desalted tumor/SILAC peptide mix were reconstituted in 300µl 5mM Potassium phosphate monobasic (Sigma #P8709), 0.1% FA and 30% acetonitrile (Sigma #14261) (Buffer A), loaded on a Ressource S column (GE healthcare, Cat # 17-1178-01) using an FPLC instrument (GE healthcare Akta Purifier) at a flow rate of 0.1ml/min and fraction size of 0.3ml. Elution was performed using a nonlinear increasing concentration of buffer B (5mM Potassium phosphate monobasic, 300mM potassium chloride, 0.1% FA and 30% acetonitrile). FPLC run consisted first of 4 column volumes (CV) of 100% buffer A after injection of the sample followed by linear increase to 8% of buffer B over 1 CV; from 8 to 29% in 7 CV; from 29 to 35% in 0.6 CV; from 35 to 100% in 1 CV and hold 100% buffer B for 3 CV. All 57 fractions were collected in 96 x 2ml well plates and were pooled to obtain 12 fractions of comparable peptide quantity. Fractions were desalted using 50mg C18 cartridge (Waters # WAT054955). After elution from C18, eluate were divided in 2 (5% for proteome and 95% for phosphoproteome) and dried using SpeedVac apparatus. For phosphopeptide enrichment, we used prepacked TiO2 tips (Titansphere GL science #5010-21312). Peptides were re-suspended in 100µl of TiO2 buffer A (0.4% Trifluoroacetic acid, 80% acetonitrile) complemented with 40µl of TiO2 buffer B (25% Lactic acid, 75% TiO2 buffer A). Tips were washed with 30µl of TiO2 buffer A followed by 30µl of TiO2 buffer B. Peptide were then loaded onto the tips and centrifuged at a 1000g for 5min. Samples were reapplied to the tips and spun again. Tips were then washed twice with 30µl of TiO2 buffer B and three times with 30µl of TiO2 buffer A. Tips were placed onto collection tubes containing 30% of 20% FA and eluted with 20µl of 10% Ammonium hydroxide solution (Sigma #338818). Samples were then dried to near completion using SpeedVac and analyzed immediately by MS.

### Data Protocol
Data were acquired using the Xcalibur software (v 3.0) and the resulting spectra were interrogated by Sequest HT through Thermo Scientific Proteome Discoverer (v 1.4) with the SwissProt Homo Sapiens database (032015). The mass tolerances in MS and MS/MS were set to 10 ppm and 0.6 Da, respectively. We set carbamidomethyl cysteine, oxidation of methionine, N-terminal acetylation, heavy 13C615N2-Lysine (Lys8) and 13C615N4-Arginine (Arg10) as variable modifications. We set specificity of trypsin digestion and allowed 2 missed cleavage sites. The resulting files were further processed using myProMS. The Sequest HT target and decoy search result were validated at 1% false discovery rate (FDR) with Percolator. For SILAC-based protein quantification, peptides XICs (Extracted Ion Chromatograms) were retrieved from Thermo Scientific Proteome Discoverer. Only proteins containing at least 1 proteotypic peptide were quantified with all available peptide data. Global median absolute deviation was applied as normalization procedure to compensate for mixing errors of the different SILAC cultures. Protein ratios were computed as the geometrical mean of related peptides. To estimate ratio significance, a t test was performed with a Benjamini–Hochberg FDR control threshold set to 0.05.

### Publication Abstract
The current consensus recognizes four main medulloblastoma subgroups (wingless, Sonic hedgehog, group 3 and group 4). While medulloblastoma subgroups have been characterized extensively at the (epi-)genomic and transcriptomic levels, the proteome and phosphoproteome landscape remain to be comprehensively elucidated. Using quantitative (phospho)-proteomics in primary human medulloblastomas, we unravel distinct posttranscriptional regulation leading to highly divergent oncogenic signaling and kinase activity profiles in groups 3 and 4 medulloblastomas. Specifically, proteomic and phosphoproteomic analyses identify aberrant ERBB4-SRC signaling in group 4. Hence, enforced expression of an activated SRC combined with p53 inactivation induces murine tumors that resemble group 4 medulloblastoma. Therefore, our integrative proteogenomics approach unveils an oncogenic pathway and potential therapeutic vulnerability in the most common medulloblastoma subgroup.

### Keywords
Rnaseq, Whole genome sequencing, Medulloblastoma, Proteomic, Multiomics, Methylome, Phosphoproteomic

### Affiliations
Mass Spectrometry and Proteomics facility (LSMP) Institut Curie PSL Research University FRANCE
Institut Curie

### Submitter
Guillaume Arras

### Lab Head
Dr Damarys Loew
Mass Spectrometry and Proteomics facility (LSMP) Institut Curie PSL Research University FRANCE


### SDRF
- organism: Homo sapiens
- organism part: primary human medulloblastomas
- cell type: not available
- disease: primary human medulloblastomas
- label: SILAC heavy
- instrument: Orbitrap Fusion
- modification parameters: Oxidation
- modification parameters: Acetyl
- modification parameters: Carbamidomethyl
- modification parameters: Label:13C(6)15N(2) 
- modification parameters: Label:13C(6)15N(4)
- modification parameters: not available, Phospho, Phospho, Phospho, Phospho, Phospho, Phospho, Phospho, Phospho, Phospho, Phospho, Phospho, Phospho, Phospho, Phospho, Phospho, Phospho, Phospho, Phospho, Phospho, Phospho, Phospho, Phospho, Phospho, Phospho

