### Accession
PXD014561

### Title
Identification of Syntaxin 7 and 12 as the first selective substrates for SGK3

### Description
Deregulation of the PI3K/Akt pathway is a common feature in many types of cancer. Recent data has demonstrated that prolonged inhibition of breast cancer cell lines with PI3K and Akt inhibitors leads to up-regulation and increased activity of the AGC kinase Serum and glucocorticoid activated kinase 3, SGK3, which in turn substitutes for Akt and leads to activation of mTORC1. SGK3 and Akt exhibit similar substrate specificity for Ser/Thr residues in the RxRxxS/T motif. However, SGK3 can be regulated by both PI3K class I and class III and possesses a unique PX domain, enabling it to bind to PI(3)P lipids in endosomes. Still, the known SGK3 substrates are either shared or have not been tested whether Akt can phosphorylate them. In the present study, we undertook genetic and pharmacologic phosphoproteomic screens to search for selective SGK3 substrates. We identified a significant number of potential in vivo SGK3 selective substrates, including the endosomal proteins STX7, STX12, RFIP4 and WDR44. In vitro phosphorylation studies demonstrated that these substrates were directly phosphorylated at the sites identified in vivo by SGK3 but, unlike all other SGK3 substrates reported, not by Akt. We provide evidence that endogenous STX7 and STX12 are physiological SGK3 substrates whose phosphorylation is potently stimulated by IGF1. Our data pinpoint that STX7 and STX12 are the first selective SGK3 substrates in vivo and this phosphorylation enhances their interaction with the respective SNARE complexes. Finally, we suggest that STX7 and STX12 phosphorylation can be utilised as in vivo biomarkers for SGK3 activity.

### Sample Protocol
Wild type (WT) and SGK3 KO cells and Wild type cells treated with MDSO, 14H and MK3 inhibitor cells were harvested with 2% SDS lysis buffer (2% SDS (w/v), 250mM NaCl, 50mM HEPES pH 8.5, 1mM Benzamidine, 2mM PMSF, 2mM Sodium orthovanadate, 10mM Na B glycerophosphate, 5mM Na pyrophosphate,50mM NaF, supplemented with protease inhibitor cocktail tablets (Roche) and PhosSTOP phosphatase inhibitors (Roche). Following protein quantification, proteins were reduced, alkylated and subjected to Lys-c and tryptic digestion. Post digestion, peptides were acidified and desalted and the vacuum dried samples are subjected to TiO2-based Phosphopeptide enrichment. An aliquot was saved and processed separately for the total proteomic analysis. Enriched phospho and total proteome samples are labelled with Tandem mass tags and after label check efficiency samples were pooled and subjected to basic reverse-phased liquid chromatography. The collected fractions were pooled into either 12 or 13 factions of each sample and analyzed on Orbitrap Fusion Tribrid Mass spectrometer that is interfaced with Dionex nano liquid chromatography system.   Orbitrap Fusion Parameters: Phosphoproteomic and total proteomics data acquisition was carried out on Orbitrap Fusion Tribrid mass spectrometer coupled to Dionex Ultimate 3000 RSLCnano system (Thermo Scientific). Peptides were loaded on a pre-column (C18, 100 Ao, 100 Î¼, 2 cm PN: 164564) and resolved on 50 cm analytical column. Peptides were separated with a binary buffer system of buffer A (0.1% formic acid (FA), 3%DMSO) and buffer B (0.1% (v/v), 3%DMSO) FA, 90% Acetonitrile (v/v)) by applying a non-linear gradient of 5% to 25% for 180 minutes and then increased to 35% for 30 minutes with a total run of 240 minutes. Mass spectrometry data was acquired in a data dependent mode operated in top speed mode for 2.5 sec for each duty cycle. The full scans were acquired with 120,000 resolution at 200m/z and measured with the Orbitrap mass analyser. The product ions are fragmented using higher-energy collisional dissociation (HCD) with a normalized collision energy of 37.5% and measured using Orbitrap mass analyser with 30,000 resolution at 200m/z. The precursor peptides from the survey scan for MS2 fragmentation using HCD. Full scans were acquired in the Orbitrap mass analyser with 120,000 resolution at 200 m/z and the MS/MS was acquired with Orbitrap with 60,000 resolution at 200 m/z. The Automatic-gain control ion targets were set at 2E5 and 5E4 for MS and MS/MS respectively. The ion filling times are kept at 50 ms and 250 ms for MS and MS/MS respectively. Dynamic exclusion was set at 35 sec and monoisotopic precursor selection was enabled.  Data Processing The raw mass spectrometry data was processed and searched against the Uniprot human protein database (March 2017) using the MaxQuant pipeline (MaxQuant version 1.5.8.3 for screen1 and 1.6.0.13 for screen2) (Cox and Mann, 2008). A separate total and phosphoproteome search was performed for the screen1 and a combined phosphoproteome and total proteome search was performed by specifying two different parameters (Phosphoproteome Group = 0 and Total proteome Group = 1). Peak list files were generated using MaxQuant and the data was searched with integrated Andromeda and the PTM score algorithm was implemented for phosphosite localization. The following parameters were specified; Group specific parameters: TMT 10 plex with 0.003 Da reporter mass tolerance to extract the individual TMT reporter channels; precursor intensity fraction of 0.75 was selected to filter out all of the peptide precursors below this value, which are potentially interfered by co-isolation, to achieve a reliable quantification; Oxidation of Met, Deamidation of Asn and Gln, Protein N-ter acetylation and Phosphorylation of (STY), (this modification was not specified in Group = 1 (total proteomic analysis) as variable modifications and Carbamidomethylation of Cys as fixed modification); minimum peptide length of 7 amino acids (main search peptide tolerance of 4.5 ppm was specified); FDR <0.01 at PSM for peptide and protein levels was applied with a minimum score of 40, specified for modified peptides. Phosphorylated and non-phosphorylated form of a given peptide was excluded for protein level quantification. The MaxQuant output tables, pSTY site table and protein groups tables were independently processed using Perseus software suite.

### Data Protocol
Data Processing The raw mass spectrometry data was processed and searched against the Uniprot human protein database (March 2017) using the MaxQuant pipeline (MaxQuant version 1.5.8.3 for screen1 and 1.6.0.13 for screen2) (Cox and Mann, 2008). A separate total and phosphoproteome search was performed for the screen1 and a combined phosphoproteome and total proteome search was performed by specifying two different parameters (Phosphoproteome Group = 0 and Total proteome Group = 1). Peak list files were generated using MaxQuant and the data was searched with integrated Andromeda and the PTM score algorithm was implemented for phosphosite localization. The following parameters were specified; Group specific parameters: TMT 10 plex with 0.003 Da reporter mass tolerance to extract the individual TMT reporter channels; precursor intensity fraction of 0.75 was selected to filter out all of the peptide precursors below this value, which are potentially interfered by co-isolation, to achieve a reliable quantification; Oxidation of Met, Deamidation of Asn and Gln, Protein N-ter acetylation and Phosphorylation of (STY), (this modification was not specified in Group = 1 (total proteomic analysis) as variable modifications and Carbamidomethylation of Cys as fixed modification); minimum peptide length of 7 amino acids (main search peptide tolerance of 4.5 ppm was specified); FDR <0.01 at PSM for peptide and protein levels was applied with a minimum score of 40, specified for modified peptides. Phosphorylated and non-phosphorylated form of a given peptide was excluded for protein level quantification. The MaxQuant output tables, pSTY site table and protein groups tables were independently processed using Perseus software suite.

### Publication Abstract
The serum- and glucocorticoid-regulated kinase (SGK) isoforms contribute resistance to cancer therapies targeting the PI3K pathway. SGKs are homologous to Akt and these kinases display overlapping specificity and phosphorylate several substrates at the same residues, such as TSC2 to promote tumor growth by switching on the mTORC1 pathway. The SGK3 isoform is up-regulated in breast cancer cells treated with PI3K or Akt inhibitors and recruited and activated at endosomes, through its phox homology domain binding to PtdIns(3)P. We undertook genetic and pharmacological phosphoproteomic screens to uncover novel SGK3 substrates. We identified 40 potential novel SGK3 substrates, including four endosomal proteins STX7 (Ser126) and STX12 (Ser139), RFIP4 (Ser527) and WDR44 (Ser346) that were efficiently phosphorylated in vitro by SGK3 at the sites identified in vivo, but poorly by Akt. We demonstrate that these substrates are inefficiently phosphorylated by Akt as they possess an n&#x2009;+&#x2009;1 residue from the phosphorylation site that is unfavorable for Akt phosphorylation. Phos-tag analysis revealed that stimulation of HEK293 cells with IGF1 to activate SGK3, promoted phosphorylation of a significant fraction of endogenous STX7 and STX12, in a manner that was blocked by knock-out of SGK3 or treatment with a pan SGK inhibitor (14H). SGK3 phosphorylation of STX12 enhanced interaction with the VAMP4/VTI1A/STX6 containing the SNARE complex and promoted plasma membrane localization. Our data reveal novel substrates for SGK3 and suggest a mechanism by which STX7 and STX12 SNARE complexes are regulated by SGK3. They reveal new biomarkers for monitoring SGK3 pathway activity.

### Keywords
Phosphoprpteomis, Sgk3, Stx7 and stx12

### Affiliations
Faculty of Medical Sciences, Institute for Cell and Molecular Biosciences, Newcastle upon Tyne, UK
MRC Protein Phosphorylation Unit, university of Dundee

### Submitter
Raja Sekhar Nirujogi

### Lab Head
Dr Prof Matthias Trost
Faculty of Medical Sciences, Institute for Cell and Molecular Biosciences, Newcastle upon Tyne, UK


