### Accession
PXD012698

### Title
Label-free quantitative proteomic analysis of DNA methyltransferase mutants of the cyanobacterium Synechocystis sp. PCC 6803 – Analysis of the membrane-enriched fraction

### Description
Total protein and membrane-enriched fractions of Synechocystis sp. PCC 6803 wild type (WT) and mutant strains affected with inactivated DNA methyltransferase genes were analyzed by LC-HDMSE. The mutation of the DNA methyltransferase gene sll0729 encoding M.Ssp6803II initially led to a strong phenotype, including a lowered chlorophyll/phycocyanin ratio, impaired growth, and alterations in gene expression. Prolonged cultivation revealed instability of the initially obtained phenotype. Colonies showing normal pigmentation and WT-like growth appeared regularly and in high frequencies on agar plates. These colonies represent suppressor mutants, since the sll0729 gene is still completely inactivated and methylation of the M.Ssp6803II target GGCC sites does not occur. The proteomic analysis comprises the WT, two strains of suppressor mutants sll0729_1 and sll0729_15 as well as mutants of the DNA methyltransferase genes slr6095 and sll8009. For each of these five strains we investigated three biological replicates. Comparisons between the WT and these two suppressor mutant strains, but also between them and the slr6095 and sll8009 mutants enabled the detection of expression differences, which are specifically linked to the absence of M.Ssp6803II-related DNA methylation.

### Sample Protocol
Cells were disrupted using Precellys 24 homogenizer (peqLab Biotechnologie GmbH, Erlangen, Germany) in a buffer containing 10 mM Tris/HCl, pH 7.4, 138 mM NaCl; 2.7 mM KCl; 1 mM MgCl2. An aliquot of the resulting total protein extract was centrifuged at 22000 g for 100 min to obtain the membrane-enriched fraction. Samples containing 75 µg of protein in 100 µl extraction buffer (50 mM ammonium bicarbonate (ABC), 1.5% sodium deoxycholate (SDC), 10 mM dithiothreitol) were incubated at 95 °C for 5 min and subsequently sonicated for 10 min using a bath sonicator. Alkylation was performed with 15 mM iodoacetamide for 20 min at room temperature. Thereafter 50 mM ABC and sequencing grade trypsin (Promega) were added to obtain an enzyme/protein ratio of 1:50 in a final volume of 320 µl. Digestion was performed at 37 °C for about 16 h. SDC was removed from the digest solutions using the phase transfer surfactant method (Masuda et al. 2008). 320 μl of ethylacetate and 6.5 µl of 25% trifluoroacetic acid were added, the mixture was rigorously shaken for 3 min and subsequently centrifuged at 12000 g for 10 min to obtain aqueous and organic phases. 200 µl of the aqueous phase was collected using a gel loading tip. Finally the peptide solutions were desalted with OASIS HLB 1cc Vac Cartridges (Waters, Manchester, UK). LC-HDMSE analyses were carried out using a nanoAcquity UPLC system (Waters) coupled to a Waters Synapt G2-S mass spectrometer via a NanoLockSpray ion source. Mobile phase A contained 0.1% formic acid in water, and mobile phase B contained 0.1% formic acid in acetonitrile. Peptide samples corresponding to approximately 360 ng of digested protein from the membrane-enriched fraction supplemented with 40 fmol of Hi3 Phos B standard for protein absolute quantification (Waters) were trapped and desalted using a precolumn (nanoAcquity UPLC Symmetry C18, 5 µm, 180 µm x 20 mm, Waters) at a flow rate of 10 µl/min for 4 min with 99.9% A. Peptides were separated on an analytical column (ACQUITY UPLC HSS T3, 1.8 µm, 75 µm x 250 mm, Waters) at a flow rate of 300 nl/min using a gradient from 3% to 32% B over 150 min. The column temperature was maintained at 35 °C. The SYNAPT G2-S instrument was operated in data-independent mode with ion-mobility separation as an additional dimension of separation (referred to as HDMSE). By executing alternate scans at low and elevated collision energy (CE) of each 0.6 sec, information on precursor and fragment ions, respectively, was acquired. In low-energy MS mode acquisitions were performed at constant CE of 4 eV whereas drift time-dependent CE settings (Distler et al. 2014) were applied in elevated-energy MS mode. As a reference compound, 100 fmol/μl [Glu1]-fibrinopeptide B was delivered at 500 nl/min to the reference sprayer of the NanoLockSpray source. Lock spray was acquired once every 30 s for a 1 s period. Samples were measured once without technical replication.

### Data Protocol
Progenesis QI for Proteomics version 4.1 (Nonlinear Dynamics, Newcastle upon Tyne, UK) was used for raw data processing, protein identification and label free quantification. For Apex3D processing, the thresholds for low and high energy scan ions were set to 135 and 30 counts, respectively. Alignment was performed to compensate for between-run variation in the LC separation. Peak picking parameters included (i) sensitivity set automatic, (ii) maximum ion charge of +4. For the database search a database containing 3507 protein sequences from Synechocystis sp. (strain PCC 6803 / Kazusa) (UniProt release 2019_01) appended with the sequences of rabbit phosphorylase B (P00489) and porcine trypsin was compiled. Precursor and fragment ion mass tolerances were automatically determined. Two missing cleavage sites were allowed, oxidation of methionine residues was considered as variable modification, and carbamidomethylation of cysteines as fixed modification. The false discovery rate was set to 4%. Peptides were required to be identified by at least three fragment ions and proteins by at least six fragment ions. Subsequently peptide ion data were filtered to retain only peptide ions that met the following criteria: (i) identified at least two times within the dataset, (ii) ion score greater or equal 5.5, (iii) mass error below 10.0 ppm or, exceptionally, below 13.0 ppm if the peptide was identified at least 8 times, and (iiii) at least 6 amino acid residues in length. Moreover, identification results based on charge state deconvolution were removed. Proteins were quantified by the absolute quantification Hi3 method using Hi3 Phos B Standard (Waters) as reference. Protein abundance changes by a factor of at least two, accompanied by ANOVA p-values < 0.01 were regarded as significant. Only proteins identified by at least two unique peptides were included in the quantitative analysis. To estimate the final rate of false peptide identifications, the search was repeated using a shuffled target-decoy database, applying identical peptide and protein filtering criteria. Comparing the number of decoy peptides to those identified with the target sequences resulted in a false positive rate of 0.08%.

### Publication Abstract
DNA methylation plays a crucial role for gene regulation among eukaryotes, but its regulatory function is less documented in bacteria. In the cyanobacterium <i>Synechocystis</i> sp. PCC 6803 five DNA methyltransferases have been identified. Among them, M.Ssp6803II is responsible for the specific methylation of the first cytosine in the frequently occurring motif GGCC, leading to N4-methylcytosine (GG<sup>m4</sup>CC). The mutation of the corresponding gene <i>sll0729</i> led to lowered chlorophyll/phycocyanin ratio and slower growth. Transcriptomics only showed altered expression of <i>sll0470</i> and <i>sll1526</i>, two genes encoding hypothetical proteins. Moreover, prolonged cultivation revealed instability of the initially obtained phenotype. Colonies with normal pigmentation and wild-type-like growth regularly appeared on agar plates. These colonies represent suppressor mutants, because the <i>sll0729</i> gene was still completely inactivated and the GGCC sites remained unmethylated. The suppressor strains showed smaller cell size, lowered DNA content per cell, and decreased tolerance against UV compared to wild type. Promoter assays revealed that the transcription of the <i>sll0470</i> gene was still stimulated in the suppressor clones. Proteomics identified decreased levels of DNA topoisomerase 4 subunit A in suppressor cells. Collectively, these results indicate that GG<sup>m4</sup>CC methylation is involved in the regulation of gene expression, in the fine-tuning of DNA replication, and DNA repair mechanisms.

### Keywords
Dna methyltransferase, Suppressor mutants, Synechocystis sp. pcc 6803, Hdmse, Label-free quantification

### Affiliations
Department of Plant Physiology, University of Rostock, Germany
Core Facility Proteome Analysis, University Medicine Rostock, Schillingallee 69, 18057 Rostock

### Submitter
Stefan Mikkat

### Lab Head
Dr Martin Hagemann
Department of Plant Physiology, University of Rostock, Germany


