### Accession
PXD023872

### Title
Identification of the MuRF1 skeletal muscle ubiquitylome through quantitative proteomics

### Description
MuRF1 is a muscle-specific E3 ubiquitin ligase and component of the ubiquitin proteasome system.  MuRF1 is transcriptionally upregulated under conditions that cause muscle loss, in both rodents and humans, and is a recognized marker of muscle atrophy.  In this study, we used in vivo electroporation to determine if MuRF1 overexpression alone can cause muscle atrophy and, in combination with ubiquitin proteomics, identify the endogenous MuRF1 substrates in skeletal muscle.  Tibialis anterior (TA) muscles were transfected with an untagged MuRF1 plasmid or control plasmid for 14 days.  A total of 963 ubiquitination sites, corresponding to 250 proteins, were quantified from the TA muscle. Statistical analysis revealed that the overexpression of MuRF1 resulted in significant upregulation of 153 ubiquitination sites on 45 proteins and significant downregulation of 16 sites on 11 proteins. Substrates of MuRF1 include contractile and metabolic proteins, deubiquitinases, p62, and VCP.  Moreover, MuRF1-mediated ubiquitination leads to destabilization and breakdown of the sarcomere and reveals a role for MuRF1 in the regulation of additional proteolytic pathways in skeletal muscle.

### Sample Protocol
4 C57BL/6 mice were electroporated with the untagged MuRF1 plasmid in one TA muscle and an empty vector (control) plasmid in the contralateral TA muscle.  After 14 days, the mice were anesthetized with isoflurane, and the TA muscles were excised, weighed, and frozen in liquid nitrogen.  Muscle tissues were homogenized in 5 ml urea lysis buffer (9 M urea, 20 mM HEPES pH 8.0). The samples were sonicated with 3 pulses of 15 s at an amplitude of 20% using a 3 mm probe, with incubation on ice for 1 min between pulses. After centrifugation for 15 min at 20,000xg at room temperature (RT), proteins were reduced with 5 mM DTT and incubation for 30 min at 55˚C, then alkylated with 10 mM CAA and incubation for 15 min at RT in the dark. The protein concentration was measured using a Bradford assay (Bio-rad) and from each sample 13.8 mg protein was used to continue the protocol. Samples were diluted with 20 mM HEPES pH 8.0 to 4 M urea and proteins were digested with 69 µg LysC (Wako) (1/200, w/w) for 4 hours at 37°C. Samples were diluted to 2 M urea and digested with 69 µg trypsin (Promega) (1/200, w/w) overnight at 37˚C. The resulting peptide mixture was acidified with trifluoroacetic acid (TFA) and after 15 min incubation on ice, samples were centrifuged for 15 min at 1,780xg at RT. Immunocapture of GlyGly-modified peptides was then performed using the PTMScan® Ubiquitin Remnant Motif (K-ε-GG) Kit (Cell Signaling Technology) according to the manufacturer’s instructions. Before enrichment, aliquots corresponding to 200 µg of digested protein were taken for shotgun (SG) proteomics analysis. Purified di-glycine (GG) modified peptides were dried under vacuum in HPLC inserts, and stored at −20 °C until LC-MS/MS analysis. Purified peptides for SG analysis were re-dissolved in 20 µl solvent A (0.1% TFA in water/ACN (98:2, v/v) and 2 µg of each sample was injected for LC-MS/MS analysis on an Ultimate 3000 RSLCnano system in-line connected to an Orbitrap Fusion Lumos MS (Thermo) equipped with a pneu-Nimbus dual ion source (Phoenix S&T). Trapping was performed at 10 μl/min for 4 min in solvent A on a 20 mm trapping column (made in-house, 100 μm internal diameter (I.D.), 5 μm beads, C18 Reprosil-HD, Dr. Maisch, Germany) and the sample was loaded on a 200 cm long micro pillar array column (PharmaFluidics) with C18-endcapped functionality mounted in the Ultimate 3000’s column oven at 50°C. For proper ionization, a fused silica PicoTip emitter (10 µm I.D., New Objective) was connected to the µPAC™ outlet union and a grounded connection was provided to this union. Peptides were eluted by a nonlinear increase from 1 to 55% MS solvent B (0.1% FA in water/ACN (2:8, v/v)) over 145 min, first at a flow rate of 750 nl/min, then at 300 nl/min, followed by a 10-min wash reaching 99% MS solvent B and re-equilibration with MS solvent A (0.1% FA in water). The MS was operated in data-dependent mode, automatically switching between MS and MS/MS acquisition. Full-scan MS spectra (300-1500 m/z) were acquired in 3 s acquisition cycles at a resolution of 1.2E5 in the Orbitrap analyzer after accumulation to a target AGC value of 2E5 with a maximum injection time of 30 ms. The precursor ions were filtered for charge states (2-7 required), dynamic range (60 s; +/- 10 ppm window) and intensity (minimal intensity of 3E4). The precursor ions were selected in the multipole with an isolation window of 1.2 Da and accumulated to an AGC target of 5E3 or a maximum injection time of 40 ms and activated using HCD fragmentation (34% NCE). The fragments were analyzed in the Ion Trap Analyzer at normal scan rate. Purified GG modified peptides were re-dissolved in 20 µl solvent A of which 15 µl was injected for LC-MS/MS analysis on an Ultimate 3000 RSLCnano system in-line connected to a Q Exactive HF MS (Thermo). Trapping and loading the sample was performed as described for the SG analysis. Peptides were eluted by a nonlinear increase from 1 to 55% MS solvent B over 116 min, first at a flow rate of 750 nl/min, then at 300 nl/min, followed by a 14-min wash reaching 99% MS solvent B and re-equilibration with MS solvent A. The MS was operated in data-dependent mode, automatically switching between MS and MS/MS acquisition for the 8 most abundant ion peaks per MS spectrum. Full-scan MS spectra (375-1500 m/z) were acquired at a resolution of 6E4 in the orbitrap analyser after accumulation to a target value of 3E6. The 8 most intense ions above a threshold value of 8.3E3 were isolated (window of 1.5 Th) for fragmentation at a normalized collision energy of 28% after filling the trap at a target value of 1E5 for maximum 120 ms. MS/MS spectra (200-2000 m/z) were acquired at a resolution of 1.5E4 in the orbitrap analyser. The S-lens RF level was set at 50 and we excluded precursor ions with single, unassigned and >7 charge states from fragmentation selection. QCloud was used to control instrument longitudinal performance during the project.

### Data Protocol
Data analysis was performed with MaxQuant (version 1.6.3.4) using the Andromeda search engine with default search settings including a false discovery rate set at 1% on the peptide and protein level. Two different searches were performed to analyze the spectra from the di-glycine-enriched samples and the SG samples. In both searches, spectra were interrogated against the mouse proteins in the Swiss-Prot Reference Proteome database (database release version of June 2019 containing 22,282 mouse protein sequences, (http://www.uniprot.org)). The mass tolerance for precursor and fragment ions was set to 4.5 and 20 ppm, respectively, during the main search. Enzyme specificity was set as C-terminal to arginine and lysine, also allowing cleavage at proline bonds with a maximum of three missed cleavages. Variable modifications were set to oxidation of methionine residues, acetylation of protein N-termini and GlyGly modification of lysine residues, while carbamidomethylation of cysteine residues was set as fixed modification. Matching between runs was enabled with a matching time window of 0.7 min and an alignment time window of 20 min. Only proteins with at least one unique or razor peptide were retained leading to the identification of 1,412 proteins and 13,329 GlyGly modified sites. Proteins were quantified by the MaxLFQ algorithm integrated in the MaxQuant software. A minimum ratio count of two unique or razor peptides was required for quantification.  Further data analysis of the SG results was performed with the Perseus software (version 1.6.2.1) after loading the proteingroups file from MaxQuant. Reverse database hits were removed, LFQ intensities were log2 transformed and replicate samples were grouped. Proteins with less than three valid values in at least one group were removed and missing values were imputed from a normal distribution around the detection limit leading to a list of 844 quantified proteins that was used for further data analysis. Then, a t-test was performed (FDR=0.05 and s0=1) to compare control and MuRF1 samples and a volcano plot was generated. 24 proteins were found to be significantly regulated. For the analysis of the di-glycine modified peptide data, the GlyGly(K)Sites file was loaded in the Perseus software (version 1.6.2.1). Reverse hits were removed, the site table was expanded, the intensity values were log2 transformed and the median was subtracted. Replicate samples were grouped, GlyGly(K)sites with less than three valid values in at least one group were removed and missing values were imputed from a normal distribution around the detection limit leading to a list of 963 quantified GlyGly peptides that was used for further data analysis. Then, a t-test was performed (FDR=0.05 and s0=1) to compare control and MuRF1 samples and a volcano plot was generated. 169 GlyGly peptides were significantly regulated and plotted in a heatmap after non-supervised hierarchical clustering. GO term enrichment analyses were performed using DAVID with a 0.1 EASE score cutoff.

### Publication Abstract
MuRF1 (TRIM63) is a muscle-specific E3 ubiquitin ligase and component of the ubiquitin proteasome system. MuRF1 is transcriptionally upregulated under conditions that cause muscle loss, in both rodents and humans, and is a recognized marker of muscle atrophy. In this study, we used in vivo electroporation to determine whether MuRF1 overexpression alone can cause muscle atrophy and, in combination with ubiquitin proteomics, identify the endogenous MuRF1 substrates in skeletal muscle. Overexpression of MuRF1 in adult mice increases ubiquitination of myofibrillar and sarcoplasmic proteins, increases expression of genes associated with neuromuscular junction instability, and causes muscle atrophy. A total of 169 ubiquitination sites on 56 proteins were found to be regulated by MuRF1. MuRF1-mediated ubiquitination targeted both thick and thin filament contractile proteins, as well as, glycolytic enzymes, deubiquitinases, p62, and VCP. These data reveal a potential role for MuRF1 in not only the breakdown of the sarcomere but also the regulation of metabolism and other proteolytic pathways in skeletal muscle.

### Keywords
Muscle atrophy, Protein degradation, Murf1, Electroporation, Ubiquitin proteomics

### Affiliations
Department of Internal Medicine, Division of Endocrinology and Metabolism, Carver College of Medicine, University of Iowa, Iowa City, Iowa, 52242 USA
VIB Proteomics Core

### Submitter
Delphi Van Haver

### Lab Head
Dr Sue Bodine
Department of Internal Medicine, Division of Endocrinology and Metabolism, Carver College of Medicine, University of Iowa, Iowa City, Iowa, 52242 USA


