### Accession
PXD001732

### Title
The role of PER1/2 and feeding rhythms in daily oscillations of mitochondrial nutrient utilization and  rate-limiting enzymes

### Description
Mitochondria are major suppliers of cellular energy through nutrients oxidation. Little is known about the mechanisms that enable mitochondria to cope with changes in nutrient supply and energy demand that naturally occur throughout the day. To address this question, we applied mass-spectrometry based quantitative proteomics on isolated mitochondria from mice sacrificed throughout the day and identified extensive oscillations in the mitochondrial proteome. We found that rate-limiting mitochondrial enzymes that process lipids and carbohydrates accumulate in a diurnal manner and dependent on the clock proteins PER1/2. In this conjuncture, we uncovered daily oscillations in mitochondrial respiration that peak during different times of the day in response to different nutrients. Notably, the diurnal regulation of mitochondrial respiration was blunted in mice lacking PER1/2 or upon nutritional challenge such as high fat diet. We propose that the clock proteins PER1/2 optimize mitochondrial metabolism to daily changes in energy supply/demand and thereby serve as a rheostat for mitochondrial nutrient utilization.

### Sample Protocol
Isolated mitochondrion pellets were lysed with a buffer containing 100 mM Tris-HCl pH8, 4% SDS and 50 mM DTT. Samples were incubated at 95Â°C for 5 minutes followed by sonication in bioruptor (Diagenode) for 15 min. Proteins were alkylated with Iodoacetamide (IAA) (55 mM final concentration) for 30 minutes at 4Â°C in the dark and after that samples were cleared by centrifugation at 14,000 rpm for 10 minutes at room temperature. Proteins were precipitated at -20Â°C overnight with 5 volumes of ethanol/acetate. Protein pellets were washed twice with ice-cold ethanol/acetate and finally resuspended with sonication in a buffer containing 6 M guanidinium hypochloride. For protein digestion samples were diluted two fold with 6 M urea/2 M thiourea in 10 mM Hepes and subsequently 1:100 LysC endopeptidase (Wako) was added and incubated for 3 hours at 37Â°C. Samples were then diluted to 0.6 M Guanidinium hypochloride with 10% acetonitrile prior to digestion with 1:100 Trypsin (Sigma-Aldrich) overnight at 37 C. The digestion was stopped by addition of 1 ul of TFA (trifluoroacetic acid). The acidified peptides were desalted and purified by loading on Stage Tips (self-made pipette tips containing 3 layers of SDB-RPS (Sigma-Aldrich) as previously described (Kulak et al., 2014).

### Data Protocol
Raw MS files were processed with MaxQuant (version 1.5.03.82), a freely available software suite. Peak list files were searched by the ANDROMEDA a search engine, incorporated into the MaxQuant framework (Cox et al., 2011), against the Mus musculus Uniprot FASTA database version of 29 May 2013 and a file containing common observed contaminants. The search included variable modifications for oxidation of methionine, protein N-terminal acetylation and carbamidomethylation as fixed modification. Peptides with at least seven amino acids were considered for identification specifying as enzyme Trypsin with cleavage C-terminal after K or R allowing 2 missed cleavages. The false discovery rate, determined by searching a decoy database, was set at 0.01 false discovery rate (FDR) for both peptides and proteins. Identification across different replicates and adjacent fractions was achieved by enabling matching between runs option in MaxQuant within a time window of 0.7 minutes and an alignment time window of 20 minutes. . During the main search, parent masses were allowed an initial mass deviation of 4.5ppm and fragment ions were allowed a mass deviation of 0.5 Da. Label-free quantification (LFQ) of proteins was done using the MaxLFQ algorithm (Cox et al., 2014) integrated into MaxQuant. The parameters were as follows: Minimum ratio count was set to 2, the FastLFQ option was enabled, LFQ minimum number of neighbors was set to 3 and the LFQ average number of neighbors to 6, as per default.

### Publication Abstract
Mitochondria are major suppliers of cellular energy through nutrients oxidation. Little is known about the mechanisms that enable mitochondria to cope with changes in nutrient supply and energy demand that naturally occur throughout the day. To address this question, we applied MS-based quantitative proteomics on isolated mitochondria from mice killed throughout the day and identified extensive oscillations in the mitochondrial proteome. Remarkably, the majority of cycling mitochondrial proteins peaked during the early light phase. We found that rate-limiting mitochondrial enzymes that process lipids and carbohydrates accumulate in a diurnal manner and are dependent on the clock proteins PER1/2. In this conjuncture, we uncovered daily oscillations in mitochondrial respiration that peak during different times of the day in response to different nutrients. Notably, the diurnal regulation of mitochondrial respiration was blunted in mice lacking PER1/2 or on a high-fat diet. We propose that PERIOD proteins optimize mitochondrial metabolism to daily changes in energy supply/demand and thereby, serve as a rheostat for mitochondrial nutrient utilization.

### Keywords
Per1/2, Circadian clocks; mitochondria; proteomics; respiration

### Affiliations
Proteomics
Department of Proteomics and Signal Transduction Max Planck Institute of Biochemistry Am Klopferspitz 18 D-82152 Martinsried Germany

### Submitter
Mario Oroshi

### Lab Head
Dr Matthias Mann
Department of Proteomics and Signal Transduction Max Planck Institute of Biochemistry Am Klopferspitz 18 D-82152 Martinsried Germany


