### Accession
PXD001641

### Title
Single muscle fiber proteomics reveals unexpected mitochondrial specialization

### Description
Mammalian skeletal muscles are composed of multinucleated cells termed slow or fast fibers according to their contractile and metabolic properties. Here we developed a high sensitivity workflow to characterize the proteome of single fibers. Analysis of segments of the same fiber by traditional and unbiased proteomics methods yielded the same subtype assignment. We discovered novel subtype specific features, most prominently mitochondrial specialization of fiber types in substrate utilization. The fiber type resolved proteomes can be applied to a variety of physiological and pathological conditions and illustrate the utility of single cell type analysis for dissecting proteomic heterogeneity.

### Sample Protocol
Peptides were separated on 50 cm columns of ReproSil-Pur C18-AQ 1.9 Î¼m resin (Dr. Maisch GmbH) packed in-house. The columns were kept at 50 Â°C using a custom-made column oven controlled by the SprayQC software [40]. Liquid chromatography performed on an EASY-nLC 1000 ultra-high pressure system was coupled through a nanoelectrospray source to a Q Exactive mass spectrometer (all from Thermo Fisher Scientific). Peptides were loaded in buffer A (0.1% (v/v) formic acid) applying a nonlinear 270-min gradient of 2â€“60% buffer B (0.1% (v/v) formic acid, 80% (v/v) acetonitrile) at a flow rate of 250 nl/min. Data acquisition switched between a full scan and 5 to 10 data-dependent MS/MS scans. Multiple sequencing of peptides was minimized by excluding the selected peptide candidates for 45 sec.

### Data Protocol
Two types of raw files are present. (1) 82 from single mouse muscle fibers. Animals were CD1 strain, 3 months old, male. Fibers were derived from >10 different animals. (2) Fractions of whole muscle library, 26 files with name library_musclemix. Single fibers can be subdivided into three groups that should not be compared to each other due to several technical differences. File names allow distinction, (a) PrESTs, (b) half fibers (c) all other files. The MaxQuant software (version 1.4.3.19) was used for the analysis of raw files and peak lists were searched against the mouse Uniprot FASTA database version of 25 February 2012 (81213 entries) and a common contaminants database (247 entries) by the Andromeda search engine [22] [41]. False discovery rate was set to 1% for peptides (minimum length of 7 amino acids) and proteins and was determined by searching a reverse database. A maximum of three missed cleavages were allowed in the database search. Peptide identification was performed with an allowed initial precursor mass deviation up to 7 ppm and an allowed fragment mass deviation 20 ppm.

### Publication Abstract
Mammalian skeletal muscles are composed of multinucleated cells termed slow or fast fibers according to their contractile and metabolic properties. Here, we developed a high-sensitivity workflow to characterize the proteome of single fibers. Analysis of segments of the same fiber by traditional and unbiased proteomics methods yielded the same subtype assignment. We discovered novel subtype-specific features, most prominently mitochondrial specialization of fiber types in substrate utilization. The fiber type-resolved proteomes can be applied to a variety of physiological and pathological conditions and illustrate the utility of single cell type analysis for dissecting proteomic heterogeneity.

### Keywords
Mitochondria, Exercise, Single cell, Muscle fibers, Metabolism

### Affiliations
Proteomics
Max Planck Institute of Biochemistry Department of Proteomics and Signal Transduction Germany

### Submitter
Mario Oroshi

### Lab Head
Dr Matthias Mann
Max Planck Institute of Biochemistry Department of Proteomics and Signal Transduction Germany


