### Accession
PXD000848

### Title
iTRAQ-based quantitative proteomic analysis of membrane and nuclear fractions of glioma (WHO grade II, III and IV) tissues to map on Chromosome 12 and identification of mass spectrometry evidence for 'missing proteins'

### Description
Glioma tumors arise from normal glial cells (astrocytes). Gliomas are morphologically and clinically classified into four malignancy grades as Grade I, II, III and IV. Grade I and II comprise low grade gliomas that grow slowly, are well differentiated and have a better prognosis and survival compared to grades III and IV. In contrast, grade III-IV lack endothelial proliferation and are highly vascular with a tendency to infiltrate and have areas of extensive necrosis and hypoxia. In the present study, we used iTRAQ (isobaric tags for relative and absolute quantitation) - based quantitative proteomic approach followed by liquid chromatography and high resolution tandem mass spectrometry (LC-MS/MS) to identify differential membrane and nuclear proteins mapping to Chromosome 12 from three different grades of astrocytoma tumors (WHO Grade II, III and IV) compared to control.

### Sample Protocol
Glioma tissues (WHO Grade II, III and IV) samples were collected at the time of surgery with informed consent from patients and approval of the Institutional Ethics Committee, Nizamâ€™s Institute of Medical Sciences (NIMS), Hyderabad. Six of the age and sex matched tumor samples and control samples were pooled separately from each grade and subcellular fractionation was done. Proteins were extracted from membrane and nuclear fraction. The equal amounts of protein from each grade of tumor and control tissues were then subjected to tryptic digestion. Tryptic peptides were labeled using 4-plex iTRAQ labels and pooled. The pooled peptides from membrane fractions were pre fractionated using SCX while excess labels from pooled peptides of nuclear fraction were removed using SCX cartridge. All the peptide samples were then cleaned using C18 tips and subjected to LC-MS analysis. High resolution mass spectrometry data was acquired using LTQ Orbitrap Velos ETD mass spectrometer. MS data was acquired with FT analyzer at the resolving power of 60,000. MS/MS was carried out in HCD mode with resolving power of 15,000 at 400 m/z.

### Data Protocol
Sequest search engine was used to search membrane and nuclear iTRAQ data and searches were submitted via Proteome Discoverer software (version 1.2 (grade IV membrane), 1.3 (grade III membrane and all grades nuclear) and 1.4 (grade II membrane)). The search parameters used for database dependent searches were as follows: a) trypsin as a proteolytic enzyme (with up to one missed cleavage) b) peptide mass error tolerance of 20 ppm; c) fragment mass error tolerance of 0.1 Da; d) oxidation of methionine as variable modification and N-terminal 4plex-iTRAQ and carbamidomethylation of cysteine as fixed modification. Sequest peptide data was extracted with 1% false discovery rate (FDR) threshold.

### Publication Abstract
In line with the aims of the Chromosome-centric Human Proteome Project (C-HPP) to completely annotate proteins of each chromosome and biology/disease driven HPP (B/D-HPP) to decipher their relation to diseases, we have generated a nonredundant catalogue of protein-coding genes for Chromosome 12 (Chr. 12) and further annotated proteins associated with major neurological disorders. Integrating high level proteomic evidence from four major databases (neXtProt, Global Proteome Machine (GPMdb), PeptideAtlas, and Human Protein Atlas (HPA)) along with Ensembl data resource resulted in the identification of 1066 protein coding genes, of which 171 were defined as "missing proteins" based on the weak or complete absence of experimental evidence. With functional annotations using DAVID and GAD, about 40% of the proteins could be grouped as brain related with implications in cancer or neurological disorders. We used published and unpublished high confidence mass spectrometry data from our group and other literature consisting of more than 5000 proteins derived from clinical specimens from patients with human gliomas, Alzheimer's disease, and Parkinson's disease and mapped it onto Chr. 12. We observed a total of 202 proteins mapping to human Chr. 12, 136 of which were differentially expressed in these disease conditions as compared to the normal. Functional grouping indicated their association with cell cycle, cell-to-cell signaling, and other important processes and networks, whereas their disease association analysis confirmed neurological diseases and cancer as the major group along with psycological disorders, with several overexpressed genes/proteins mapping to 12q13-15 amplicon region. Using multiple strategies and bioinformatics tools, we identified 103 differentially expressed proteins to have secretory potential, 17 of which have already been reported in direct analysis of the plasma or cerebrospinal fluid (CSF) from the patients and 21 of them mapped to cancer associated protein (CAPs) database that are amenable to selective reaction monitoring (SRM) assays for targeted proteomic analysis. Our analysis also reveals, for the first time, mass spectrometric evidence for two "missing proteins" from Chr. 12, namely, synaptic vesicle 2-related protein (SVOP) and IQ motif containing D (IQCD). The analysis provides a snapshot of Chr. 12 encoded proteins associated with gliomas and major neurological conditions and their secretability which can be used to drive efforts for clinical applications.

### Keywords
Nuclear proteins, Glioma, Synaptic vesicle 2-related protein, Sub cellular fractionation, Glioblastoma, Chromosome 12, Iq motif containing d, Diffused astrocytoma, Itraq, Anaplastic astrocytoma, Membrane proteins

### Affiliations
Institute of Bioinformatics, Bangalore - 560066, India
Institute of Bioinformatics

### Submitter
Ravi Sirdeshmukh

### Lab Head
Dr Dr. Ravi Sirdeshmukh
Institute of Bioinformatics, Bangalore - 560066, India


