### Accession
PXD035773

### Title
Proteogenomics reveals two distinct biological pilocytic astrocytoma subgroups

### Description
Pilocytic astrocytoma (PA) is the most common pediatric brain tumor and driven by aberrant MAPK signaling, typically mediated by BRAF alterations. While five-year overall survival rates exceed 95%, tumor recurrence constitutes a major clinical challenge in incompletely resected tumors despite chemotherapeutic or radiation based therapies. Therefore, we used proteogenomics to discern the biological heterogeneity of PA to improve classification of this tumor entity and identify novel therapeutic targets.   Our proteogenomics approach integrates RNA sequencing and LC/MS-based proteomic profiling data from a cohort of 58 confirmed, primary PA samples. An integrative genomics approach was conducted to discern the biological heterogeneity of PA and to identify aberrant pathway activation in these biological subgroups.   In summary, Pilocytic astrocytomas segregate into two groups where younger patients are significantly associated with Group 1. Importantly, we validate the two distinct biological subgroups in two non-overlapping cohorts. The biological heterogeneity seen here may improve biological classification and reveal novel therapeutic targets specifically useful for non-resectable tumors with high risk of recurrent or progressive disease.

### Sample Protocol
Proteins were extracted from frozen tissue sections as described elsewhere (Poschmann et al. [1]). Briefly, cells were homogenised in urea buffer with a TissueLyser (Qiagen, Hilden, Germany) and subsequent sonication. After centrifugation for 15 min at 14,000 xg and 4°C, supernatants were collected and proteins contained were precipitated overnight at -20°C with acetone at a 1:4 (v/v) ratio. Protein concentration was determined via Pierce 660 nm Protein Assay (Fischer Scientific, Schwerte, Germany) and 10 µg protein per sample were desalted through electrophoretic migration at 50 V for 10 min on a 4 -12 % Bis-Tris polyacrylamide gel (Novex NuPAGE, Thermo Scientific, Darmstadt, Germany). After silver staining, protein bands were cut out reduced, alkylated and digested with trypsin before peptide extraction via sonication. Peptides were dissolved and diluted with 0.1 % TFA (v/v). For mass spectrometric analysis, 15 µL peptide solution per sample were analysed on a nano-high-performance liquid chromatography electrospray ionisation mass spectrometer. The analytical system was composed of a RSLCnano U3000 HPLC coupled to a QExactive plus mass spectrometer via a nano-electrospray ion source (Thermo Fischer Scientific, Bremen, Germany). Injected peptides were concentrated and desalted at a flow rate of 6 µL/min on a trapping column (Acclaim PepMao C18, 2 cm x 100 µm x 3 µm particle size, 100 Å pore size, Thermo Fischer Scientific, Bremen, Germany) with 0.1 % TFA (v/v) for 10 min. Subsequently, peptides were separated at a constant flowrate of 300 nL/min over a 120 min gradient on an analytical column (Acclaim PepMap RSLC C18, 25 cm x 75 µm x 2 µm particle size, 100 Å pore size, Thermo Fischer Scientific, Bremen, Germany) at 60°C. Separation was achieved through a gradient from 4 to 40% solvent B (solvent A: 0.1% (v/v) formic acid in water, solvent B: 0.1% (v/v) formic acid, 84% (v/v) acetonitrile in water). Afterwards, peptides were ionised at a voltage of 1,400 V and introduced into the mass spectrometer operated in positive mode. MS scans were recorded in profile mode in a range from 350-2000 m/z at a resolution of 70,000 while tandem mass spectra were recorded at a resolution of 17,500. Tandem mass spectra were recorded with a data dependent Top10 method and 30% normalised collision energy. Dynamic exclusion was activated with a repeat count of 1 for 100 ms.

### Data Protocol
Proteome Discoverer (version 2.1.0.81, Thermo Fisher Scientific, Bremen, Germany) was applied for peptide/protein identification with Mascot (version 2.4.1, Matrix Science, London, UK) as search engine employing the UniProt database (human; including isoforms; date 2017-12-20). A false discovery rate of 1% (p ≤ 0.01) on peptide level was set as identification threshold. Proteins were quantified with Progenesis QI for Proteomics (Version 2.0, Nonlinear Dynamics, Waters Corporation, Newcastle upon Tyne, UK).

### Publication Abstract
Pilocytic astrocytoma (PA), the most common pediatric brain tumor, is driven by aberrant mitogen-activated protein kinase signaling most commonly caused by BRAF gene fusions or activating mutations. While 5-year overall survival rates exceed 95%, tumor recurrence or progression constitutes a major clinical challenge in incompletely resected tumors. Here, we used similarity network fusion (SNF) analysis in an integrative multi-omics approach employing RNA transcriptomic and mass spectrometry-based proteomic profiling to molecularly characterize PA tissue samples from 62 patients. Thereby, we uncovered that PAs segregated into two molecularly distinct groups, namely, Group 1 and Group 2, which were validated in three&#xa0;non-overlapping cohorts. Patients with Group 1 tumors were significantly younger and showed worse progression-free survival compared to patients with group 2 tumors. Ingenuity pathways analysis (IPA) and gene set enrichment analysis (GSEA) revealed that Group 1 tumors were enriched for immune response pathways, such as interferon signaling, while Group 2 tumors showed enrichment for action potential and neurotransmitter signaling pathways. Analysis of immune cell-related gene signatures showed an enrichment of infiltrating T Cells in Group 1 versus Group 2 tumors. Taken together, integrative multi-omics of PA identified biologically distinct and prognostically relevant tumor groups that may improve risk stratification of this single pathway driven tumor type.

### Keywords
Brain tumour, Lc-msms, Pilocytic astrocytoma

### Affiliations
Department of Pediatric Oncology, Hematology and Clinical Immunology, Medical Faculty, University Hospital Düsseldorf, Düsseldorf, Germany
Medicines Discovery Catapult

### Submitter
Maike Langini

### Lab Head
Dr Marc Remke
Department of Pediatric Oncology, Hematology and Clinical Immunology, Medical Faculty, University Hospital Düsseldorf, Düsseldorf, Germany


