### Accession
PXD013361

### Title
CD4_TMT

### Description
A Study of the Global Proteomics by Differentiation of cd4+Tcell

### Sample Protocol
Lysis, Tryptic digestion, TMT 6 plex, Fractionation, LC-MS

### Data Protocol
Mass spectrometry Data is processed using Maxquant,

### Publication Abstract
In this article we use high-throughput epigenomics, transcriptomics, and proteomics data to construct fine-graded models of the "protein-coding units" gathering all transcript isoforms and chromatin accessibility peaks associated with more than 4000 genes in humans. Each protein-coding unit has the structure of a directed acyclic graph (DAG) and can be represented as a Bayesian network. The factorization of the joint probability distribution induced by the DAGs imposes a number of conditional independence relationships among the variables forming a protein-coding unit, corresponding to the missing edges in the DAGs. We show that a large fraction of these conditional independencies are indeed verified by the data. Factors driving this verification appear to be the structural and functional annotation of the transcript isoforms, as well as a notion of structural balance (or frustration-free) of the corresponding sample correlation graph, which naturally leads to reduction of correlation (and hence to independence) upon conditioning.

### Keywords
Human, T cell

### Affiliations
Kyung hee university
Department of New Biology;Daegu Gyeongbuk Institute of Science and Technology (DGIST);Republic of Korea

### Submitter
Min Jung Kim

### Lab Head
Dr Min-Sik Kim
Department of New Biology;Daegu Gyeongbuk Institute of Science and Technology (DGIST);Republic of Korea


