### Accession
PXD013611

### Title
PRMT5 methylome profiling uncovers a direct link to splicing regulation in human acute myeloid leukemia

### Description
Protein arginine methyltransferase 5 (PRMT5) belongs to the class II arginine methyltransferases and catalyzes monomethylation and symmetrical dimethylation of arginines on proteins. It has recently emerged as a promising cancer drug target, and two PRMT5 inhibitors are currently in clinical trials for a range malignancies. Despite the recognized therapeutic potential, it is unclear which PRMT5 functions underlie its oncogenic activity. In this study, we aimed to further understand the role of PRMT5 in acute myeloid leukemia (AML). Using an enzymatic dead version of PRMT5 as well as a PRMT5-specific inhibitor, we demonstrated the requirement of the catalytic activity of PRMT5 for the survival of AML cells. By using cutting-edge proteomics techniques we identified PRMT5 substrates and investigated their role in the survival of AML cells. We found that the function of the splicing regulator SRSF1 relies on its methylation by PRMT5. Consistent with this, we found that loss of PRMT5 led to changes in alternative splicing. This revealed multiple affected essential genes, linking PRMT5 activity to its loss of function phenotype. Our results show that PRMT5 directly regulates SRSF1 activity in leukemia, and they provide potential biomarkers for the treatment response to PRMT5 inhibitors.

### Sample Protocol
5 million cells were lysed in 330 µL of ice-cold RIPA buffer with 3 U of benzonase and protease and phosphatase inhibitors (cOmplete and PhosStop, Roche) using a sonication probe with 5 cycles of 50% amplitude with 5 seconds intervals. Proteins were precipitated by addition of 1.2 ml of ice-cold acetone for 2 hours at -20C and then pelleted by centrifugation at 20.000 g. The samples were resuspended in 150 µL of the digestion buffer, containing 8M urea, 0.1M HEPES, pH 8.0 using a sonication probe with 5 cycles of 50% amplitude with 5 seconds intervals. Protein concentration was estimated with Pierce™ BCA Protein Assay Kit.  An aliquot containing 300 µg of protein material was adjusted to 200 µL with the digestion buffer. Proteins were reduced and alkylated by addition of 10 mM TCEP and 40 mM chloroacetamide and digested with lysC (1:50 protease: protein ratio) at 37C for 4 hours. The sample was then diluted 8-fold with 0.1 M HEPES, pH 8 buffer and digested with trypsin (1:25 protease:protein ratio) at 37C overnight. The obtained peptides were desalted on a 30 mg Oasis HLB cartridge (WAT094225) according to the manufacturer recommendations, vacuum-dried to 2-3 µL volumespeedvaced with 2 µL DMSO to prevent complete evaporation and  and resuspended in TMT labelling buffer (0.1 M HEPES, pH 8). Peptide concentration was measured by Pierce BCA Protein Assay Kit. 60 µg of peptide material for each sample was labelled with 0.5 mg of the TMT labelling reagent according to the manufacturer recommendations. 10 samples labelled by complementary 10plex TMT reagents  were pooled for the use in downstream procedures.

### Data Protocol
Data were analyzed in Proteome Discoverer 2.3 software. All the files from all acquisitions (hyperfractionations and enrichment) were searched together. A database search was performed with Mascot 2.3.2 using Homo Sapiens UniProt database containing only reviewed entries and canonical isoforms (retrieved on 06/11/2017). Oxidation (M), Deamidated (NQ), Dimethyl (R), Methyl (R) were set as variable modifications, while Carbamidomethyl (C), TMT6plex (K), and TMT6plex (N-term) were specified as fixed modifications. A maximum of four missed cleavages were permitted (such a high number was chosen due to lower trypsin efficiency at methylated Arg sites). The precursor and fragment mass tolerances were 10 ppm and 0.6 Da, respectively. Peptides were validated by Mascot percolator with a 0.01 posterior error probability (PEP) threshold. ptmRS algorithm was used to validate Arg methylation position using standard settings for CID data.

### Publication Abstract
Protein arginine methyltransferase 5 (PRMT5) has emerged as a promising cancer drug target, and three PRMT5 inhibitors are currently in clinical trials for multiple malignancies. In this study, we investigated the role of PRMT5 in human acute myeloid leukemia (AML). Using an enzymatic dead version of PRMT5 and a PRMT5-specific inhibitor, we demonstrated the requirement of the catalytic activity of PRMT5 for the survival of AML cells. We then identified PRMT5 substrates using multiplexed quantitative proteomics and investigated their role in the survival of AML cells. We found that the function of the splicing regulator SRSF1 relies on its methylation by PRMT5 and that loss of PRMT5 leads to changes in alternative splicing of multiple essential genes. Our study proposes a mechanism for the requirement of PRMT5 for leukemia cell survival and provides potential biomarkers for the treatment response to PRMT5 inhibitors.

### Keywords
Prmt5, Human, Ptm, Tmt, Arginine methylation

### Affiliations
Syddansk University
Department of Biochemistry and Molecular Biology, VILLUM Center for Bioanalytical Sciences, University of Southern Denmark, Campusvej 55, DK - 5230 Odense M, Denmark

### Submitter
Pavel Shliaha

### Lab Head
Dr Ole N Jensen
Department of Biochemistry and Molecular Biology, VILLUM Center for Bioanalytical Sciences, University of Southern Denmark, Campusvej 55, DK - 5230 Odense M, Denmark


