### Accession
PXD032910

### Title
Lactiplantibacillus plantarum, LC-MSMS

### Description
This project uses TMT labeling quantitative proteomics technology to carry out research, and a total of 898 proteins have been identified. Differentially expressed proteins were screened according to the criteria of expression fold change of more than 1.5-fold (up-regulation more than 1.5-fold or down-regulation less than 0.67) and P value<0.05. Among them, taking the comparison group Control VS H2O2 as an example, there were 31 up-regulated differentially expressed proteins and 81 down-regulated differentially expressed proteins. Through GO enrichment and KEGG pathway analysis, it was found that these differentially expressed proteins are mainly involved in important biological processes such as single-organism metabolic process, small molecule metabolic process, organophosphate metabolic process, organophosphate biosynthetic process and carbohydrate derivative biosynthetic process, and are mainly involved in the regulation of Metabolic pathways, Fructose and mannose metabolism, Oxidative phosphorylation, Tyrosine and Degradation of aromatic compounds and other important KEGG metabolic pathways.

### Sample Protocol
L. plantarum KM1 was cultured for 18 h, the sample group (5 mM H2O2) and control group (0 mM H2O2) were obtained respectively. Each group made three biological replicates.

### Data Protocol
The target protein set used Omicsbean software for Gene Ontology (GO) function annotation and Kyoto Encyclopedia of Genes and Genomes (KEGG) signal pathway. GO and KEGG pathway enrichment analysis of the differentially expressed proteins (DEPs) was performed by Fisherâ€™s exact test method. The detailed information of GO function annotation was obtained by consulting the related database (http://www.geneontology.org/). Information about the KEGG pathway could be accessed through the KEGG database (http://www.kegg.jp/). The STRING database (http://string-db.org) was used to analyze the protein-protein interactions (PPI) of L. plantarum KM1 including direct physical interactions and functional correlations.

### Publication Abstract
<i>Lactiplantibacillus plantarum</i> KM1 was screened from natural fermented products, which had probiotic properties and antioxidant function. The survival rate of <i>L. plantarum</i> KM1 was 78.26% at 5 mM H<sub>2</sub>O<sub>2</sub>. In this study, the antioxidant mechanism of <i>L. plantarum</i> KM1 was deeply analyzed by using the proteomics method. The results demonstrated that a total of 112 differentially expressed proteins (DEPs) were screened, of which, 31 DEPs were upregulated and 81 were downregulated. The Kyoto Encyclopedia of Genes and Genomes (KEGG) enrichment analysis indicated that DEPs participated in various metabolic pathways such as pyruvate metabolism, carbon metabolism, trichloroacetic acid cycle, amino acid metabolism, and microbial metabolism in diverse environments. These metabolic pathways were related to oxidative stress caused by H<sub>2</sub>O<sub>2</sub> in <i>L. plantarum</i> KM1. Therefore, the antioxidant mechanism of <i>L. plantarum</i> KM1 under H<sub>2</sub>O<sub>2</sub> stress provided a theoretical basis for its use as a potential natural antioxidant.

### Keywords
Lc-msms, Lactiplantibacillus plantarum

### Affiliations
Jilin Agricultural University
Jilin Agricultural University-Jilin Province Food Biomanufacturing Technology Innovation Center

### Submitter
Yuan Tian

### Lab Head
Dr Yuhua Wang
Jilin Agricultural University-Jilin Province Food Biomanufacturing Technology Innovation Center


