### Accession
PXD024328

### Title
Quantification of the translatome in wild type Arabidopsis and amiNAA10 plants

### Description
Wild type Arabidopsis Col-0 plants and mutants depleted of the catalytic subunit of the ribosome-associated N-acetyltransferase A (NatA) complex (amiNAA10) were grown on soil for six weeks under short-day conditions (8h light, light intensity: 100 µE, 21 °C/18 °C temperature (day/night), 50 % humidity). The translatome of both genotypes after 2 hours of light (10 am) was determined by feeding of the trackable Met-analogue azidohomoalanine for 4 hours to leaf disks floated on ½ Hoagland medium containing 50 µM azidohomoalanine in the light.

### Sample Protocol
Samples were harvested by snap freezing in liquid nitrogen. Soluble proteins were extracted for 15 min at 4 °C from grinded material (80 mg) with 0.850 ml extraction buffer (8 M urea, 200 mM Tris pH 8, 4% CHAPS, 1 M NaCl). The cleared protein extract was subjected to enrichment of azidohomoalanine-containing protein species with the Click-iT® Protein Enrichment kit (C10416, Invitrogen) according to manufacturer’s manual. The enriched proteins were digested on the resin with trypsin. After over-night trypsin digestion the supernatant was acidified, desalted using stage tipps (Rappsilber et al. Nat. Protoc. 2007) and dried in a vacuum concentrator before nanoLC-MS analysis with an Ultimate 3000 liquid chromatography system coupled to an QExactive HF mass spectrometer (Thermo-Fischer, Bremen, Germany). Samples were dissolved in 0.1% TFA, injected to a self-packed analytical column (75um x 200mm; ReproSil Pur 120 C18-AQ; Dr Maisch GmbH) and eluted with a flow rate of 300 nl / min in an acetonitrile-gradient (3% - 40%). The mass spectrometer was operated in data-dependent acquisition mode, automatically switching between MS and MS2. Collision induced dissociation MS2 spectra were generated for up to 20 precursors with normalized collision energy of 29%.

### Data Protocol
Raw files were processed using MaxQuant version 1.5.3.30 (J. Cox, M. Mann, Nat Biotechnol 2008, 26, 1367) for peptide identification and quantification. MS2 spectra were searched against the Uniprot arabidopsis thaliana proteome database and the contaminants database by Andromeda search engine with the following parameters: Carbamidomethylation of cysteine residues and Acetyl (Protein N-term), Oxidation (M) as variable modifications, trypsin/P as the proteolytic enzyme with up to 2 missed cleavages was allowed. The maximum false discovery rate for proteins and peptides was 0.01 and a minimum peptide length of 7 amino acids was required. All other parameters were default parameters of MaxQuant. LFQ values were were calculated by MaxQuant and used for quantitative data analysis.

### Publication Abstract
N-terminal protein acetylation (NTA) is a prevalent protein modification essential for viability in animals and plants. The dominant executor of NTA is the ribosome tethered N<sup>&#x3b1;</sup>-acetyltransferase A (NatA) complex. However, the impact of NatA on protein fate is still enigmatic. Here, we demonstrate that depletion of NatA activity leads to a 4-fold increase in global protein turnover via the ubiquitin-proteasome system in Arabidopsis. Surprisingly, a concomitant increase in translation, actioned via enhanced Target-of-Rapamycin activity, is also observed, implying that defective NTA triggers feedback mechanisms to maintain steady-state protein abundance. Quantitative analysis of the proteome, the translatome, and the ubiquitome reveals that NatA substrates account for the bulk of this enhanced turnover. A targeted analysis of NatA substrate stability uncovers that NTA absence triggers protein destabilization via a previously undescribed and widely conserved nonAc/N-degron in plants. Hence, the imprinting of the proteome with acetylation marks is essential for coordinating proteome stability.

### Keywords
Arabidopsis thaliana, Co-translational protein modification, Translation, N-acetyltransferase a, N-terminal protein acetylation

### Affiliations
Core facility for Mass Spectrometry, ZMBH, University of Heidelberg, Im Neuenheimer Feld 282, 69120 Heidelberg, Germany
ZMBH, Im Neuenheimer Feld 282, 69122 Heidelberg

### Submitter
Thomas Ruppert

### Lab Head
Dr Thomas Ruppert
Core facility for Mass Spectrometry, ZMBH, University of Heidelberg, Im Neuenheimer Feld 282, 69120 Heidelberg, Germany


