### Accession
PXD016607

### Title
Arabidopsis SME interactome reveals spliceosome components

### Description
Alternative splicing (AS) can produce multiple transcripts with different exon-intron structures from a single pre-mRNA. Pre-mRNA splicing is catalyzed by a dynamic macromolecular ribonucleoprotein (RNP) complex termed the spliceosome. The spliceosome consists of several small nuclear ribonucleoproteins (snRNPs) that bind uridine-rich small nuclear RNA (snRNA). In U1, U2, U4 and U5 snRNPs, snRNA interacts with the conserved Smith antigen (Sm) proteins via a bipartite Sm sequence motif. In eukaryotes, seven Sm proteins (B, D1/2/3, E, F and G) form a heptameric ring-shaped complex surrounding the snRNA.  Here, we performed a tandem affinity purification using SMEB as a bait in Arabidopsis cell suspension cultures. At least 45 known/hypothesized and potential novel spliceosome components were identified in Arabidopsis.

### Sample Protocol
Wild-type (Col-0) Arabidopsis cell suspension cultures grown in a 16 h photoperiod were transformed with N-terminal GSrhino-tagged SMEB constructs by Agrobacterium tumefaciens co-cultivation. In total, two individual TAP experiments were performed; in each of them SMEB-GSrhino recombinant protein and its interacting partners were purified. Protein extract preparation, tandem affinity purification, proteolysis and mass spectrometry analysis were performed as described by Van Leene et al. (2015) Nat Protocol.

### Data Protocol
Resulting mass spectrometry data files (.RAW) were processed by MaxQuant (52; version 1.5.5.1). Data were searched against the Arabidopsis proteome concatenated with protein contaminants of TAP or general proteomic contaminants (available at http://psb.ugent.be/tap-fasta-datasets/,27). Carbamidomethyl was set as fixed modification for Cys, whereas Met oxidation, protein N-terminal acetylation and methylation of Asp and Glu were variable modifications. Matching between the two replicates was enabled, with an alignment window of 20 min and 0.7 matching time window. The resulting protein identification table (“proteinGroups.txt”) was filtered for contaminant and decoy protein hits. In addition, aspecific background proteins frequently identified in previous Arabidopsis TAP experiments were omitted (27).

### Publication Abstract
None

### Keywords
Pre-mrna splicing, Spliceosome, Alternative splicing, Splicing, Arabidopsis, Sme, Sm ring

### Affiliations
Department of Plant Biotechnology and Bioinformatics, Oxidative Stress Signaling Lab, Ghent University
PSB-VIB

### Submitter
Patrick Willems

### Lab Head
Dr Frank Van Breusegem
Department of Plant Biotechnology and Bioinformatics, Oxidative Stress Signaling Lab, Ghent University


