### Accession
PXD029505

### Title
Site-Link for identification of chromatinized proteome defined by site-specific histone acylations in living cells

### Description
By coupling the mass spectrometry, we developed the Site-Link strategy based on the GCE method for analyzing the interacting proteome of histone lysine acylations in living cells.   As K* could mimic the natural lysine to be recognized by natural lysine synthase, histones bearing the site-specifically incorporated K*acyl, should also be also integrated into nucleosomes and further compacted into chromatin similar as Kacyl  with cell passaging. The subsequent 365-nm light irradiation could activate the diazirine and covalently capture potential effector proteins on native chromatin in living cells. The effector proteins could be identified after gel-based proteomics and MS/MS analysis of the crosslinked histone-effector complexes.

### Sample Protocol
Plasmids pcDNA4-H3-K56TAG (C-terminal myc-His-tag) or pcDNA4-H3-K79TAG (C-terminal myc-His-tag) was transfected into the relative PylRS stable cell line and supplemented with 0.2 mM K*cr/K*bhb. After growing for 6 h, the transfected HEK293T cells were changed into DMEM with 1mM K*cr/K*bhb. The cells were cultured for additional 36 hours to express the corresponding H3 protein (H3K*56cr/H3K*56bhb/H3K*79cr). The medium was discarded and Dulbeccoâ€™s phosphate buffered saline (DPBS) was added. After ultraviolet irradiation of the cells on ice for 15 min, cells were harvested. The histones were extracted by TCA precipitation and enriched by his-tag isolation. The eluents were combined, concentrated, and separated by SDS-PAGE. The crosslinked bands and the relative bands without UV irradiation were cut for subjected to HPLC-MS/MS proteomic study.

### Data Protocol
The raw files were searched against Human fasta database (20,361 sequences) (downloaded from Uniprot on July 1st, 2018) by MaxQuant (version 1.6.1.0). with the following parameters: The carbamidomethyl (Cys) was set as the fixed modification, while oxidation (Met) was set as variable modification. +/- 10 ppm was set for peptide pass tolerance and +/- 0.6 Da was set for fragment mass tolerance. A maximum of two missed cleavages was allowed for trypsin digestion.   FDR was determined by searching a reverse database and was set to 0.01 for proteins and peptides. Proteins with spectral counts no less than 2 were recognized as possible candidates after Maxquant searching.

### Publication Abstract
A generalizable strategy with programmable site specificity for in situ profiling of histone modifications on unperturbed chromatin remains highly desirable but challenging. We herein developed a single-site-resolved multi-omics (SiTomics) strategy for systematic mapping of dynamic modifications and subsequent profiling of chromatinized proteome and genome defined by specific chromatin acylations in living cells. By leveraging the genetic code expansion strategy, our SiTomics toolkit revealed distinct crotonylation (e.g., H3K56cr) and &#x3b2;-hydroxybutyrylation (e.g., H3K56bhb) upon short chain fatty acids stimulation and established linkages for chromatin acylation mark-defined proteome, genome, and functions. This led to the identification of GLYR1 as a distinct interacting protein in modulating H3K56cr's gene body localization as well as the discovery of an elevated super-enhancer repertoire underlying bhb-mediated chromatin modulations. SiTomics offers a platform technology for elucidating the "metabolites-modification-regulation" axis, which is widely applicable for multi-omics profiling and functional dissection of modifications beyond acylations and proteins beyond histones.

### Keywords
Site-specific, Histone acylation, Hek293t, Lc-ms/ms, Proteome

### Affiliations
Professor and Chair of Chemical Biology,  Peking-Tsinghua Center for Life Sciences Peking University, Beijing, China. Associate Editor, ACS Chemical Biology
Peking University

### Submitter
Fangfei Qin

### Lab Head
Dr Peng R. Chen
Professor and Chair of Chemical Biology,  Peking-Tsinghua Center for Life Sciences Peking University, Beijing, China. Associate Editor, ACS Chemical Biology


