### Accession
PXD017363

### Title
Proteomic analysis of mucopolysaccharidosis IIIB mouse brain

### Description
Mucopolysaccharidosis IIIB (MPS IIIB) is an inherited metabolic disease due to deficiency of ÃƒÅ½Ã‚Â±-N-Acetylglucosaminidase (NAGLU) enzyme with subsequent storage of undegradedheparan sulfate (HS). The main clinical manifestations of the disease are profound intellectual disability and neurodegeneration. To identify potential biomarkers and novel neuropathological mechanisms of MPS IIIB, a label-free quantitative proteomic approach was applied to compare the proteome profile of brains from MPS IIIB and control mice. Proteins were identified through a bottom up analysis and 130 were significantly under-represented and 74 over-represented in MPS IIIB mouse brains compared to wild type (WT). Multiple bioinformatic analyses of the differentially abundant proteins allowed to define three major clusters: proteins involved in cytoskeletal regulation, synaptic vesicle trafficking, and energy metabolism. The results highlight the involvement of these clustered proteins in the neuropathology of MPS IIIB disease. The proteins identified in this study would provide potential targets for diagnostic and therapeutic studies of MPS IIIB.

### Sample Protocol
Whole mouse brains were collected from five MPS IIIB and five WT male mice (8-month old). Brain tissues were lysed in ice-cold lysis buffer containing 7 M urea, 2 M thiourea, 4% CHAPS, 30 mMTris-HCl pH=7.8 supplemented with protease inhibitor cocktail. Mechanical disruption of tissues was performed on ice using 2 mL Dounce homogenizer applying ten strokes per brain sample. Lysed tissues were centrifuged for 20 min at 13000 rpm; the supernatants were collected and protein concentration was quantified by Bradford assay using Bio-Rad Protein Assay Dye Reagent Concentrate. Protein extract aliquots (50 ÃƒÅ½Ã‚Â¼g) from the five biological replicates for each condition (NAGLU-/- and WT) were fractionated on 10% SDS-PAGE. The resolved proteins were stained using Gel Code Blue Stain Reagent and each gel lane was divided into 5 pieces and hydrolyzed by in situ trypsin digestion. Peptide mixtures were dissolved in 0.2% HCOOH and analyzed by LC-MS/MS using a Q-ExactiveTM mass spectrometer (Thermo Scientific, Bremen, Germany) coupled with an UltiMate 3000 RSLC nanoLC system (Thermo Scientific). The peptide mixture was desalted by a pre-column(Acclaim PepMap C18, 300ÃƒÅ½Ã‚Â¼m ÃƒÆ’Ã¢â‚¬â€� 5 mm nanoViper, 5 ÃƒÅ½Ã‚Â¼m, 100 ÃƒÆ’Ã¢â‚¬Â¦, Thermo Scientific), in 0.05% formic acid and 2% acetonitrile. Finally, the peptide mixture was fractionated on reverse phase capillary column (Acclaim Easy Spray PepMap RSLC C18, 75 ÃƒÅ½Ã‚Â¼m ÃƒÆ’Ã¢â‚¬â€� 15cm nanoViper, 3ÃƒÅ½Ã‚Â¼m, 100 ÃƒÆ’Ã¢â‚¬Â¦, Thermo Scientific) and eluted by a nonlinear gradient: 4% B for 5 min, from 4 to 40% B in 45 min and from 40 to 90 % B in 1 min at flow rate of 300nL/min. (Eluent A, 0.1% formic acid; Eluent B, 80% acetonitrile, 0, 08% formic acid). MS analysis was setup as follows: data dependent Full MS/ddMS2; ten most intense precursor ions fragmentation; 30 sec dynamic exclusion; MS Resolution 70000; MS/MS Resolution 17500.

### Data Protocol
Protein identification was performed using Thermo Proteome DiscovererÃƒÂ¢Ã¢â‚¬Å¾Ã‚Â¢ platform (version 1.3.0.339, Thermo Scientific, Bremen, Germany), interfaced to SEQUEST HT Search Engine server (University of Washington, USA). The peak lists were processed according to the follow parameters: (I) Spectrum Selector. Min Precursor Mass: 350 Da, Max. Precursor Mass: 5000 Da, Minimum Peak Count: 1; (II) SEQUEST HT: 1. Input Data. Protein Database: Swiss-prot, Enzyme: Trypsin, Maximum Missed Cleavage Sites: 2, Instrument: ESI-FTICR, Taxonomy: Mus Musculus. 2. Tolerances. Precursor Mass Tolerance: 5 ppm, Fragment Mass Tolerance: 0.8 Da. 3. Dynamic Modification. Methionine Oxidation, N-terminal Glutamine cyclization to Pyroglutamic Acid, N-terminal protein Acetylation. 4. Static modification: Cysteine Carboamidomethylation; (III) Target Decoy/PSM validator: 1. Maximum Delta Cn: 0.05; 2. Target FDR (strict): 0.01; 3. Target FDR (Relaxed): 0.05. Proteins identified by a minimum of two peptides were accepted.

### Publication Abstract
Mucopolysaccharidosis IIIB (MPS IIIB) is an inherited metabolic disease due to deficiency of &#x3b1;-N-Acetylglucosaminidase (NAGLU) enzyme with subsequent storage of undegraded heparan sulfate (HS). The main clinical manifestations of the disease are profound intellectual disability and neurodegeneration. A label-free quantitative proteomic approach was applied to compare the proteome profile of brains from MPS IIIB and control mice to identify altered neuropathological pathways of MPS IIIB. Proteins were identified through a bottom up analysis and 130 were significantly under-represented and 74 over-represented in MPS IIIB mouse brains compared to wild type (WT). Multiple bioinformatic analyses allowed to identify three major clusters of the differentially abundant proteins: proteins involved in cytoskeletal regulation, synaptic vesicle trafficking, and energy metabolism. The proteome profile of NAGLU<sup>-/-</sup> mouse brain could pave the way for further studies aimed at identifying novel therapeutic targets for the MPS IIIB. Data are available via ProteomeXchange with the identifier PXD017363.

### Keywords
Mucopolysaccharidosis iiib; quantitative proteomics; naglu; lysosomes

### Affiliations
 Department of Molecular Medicine and Medical Biotechnology - University of Naples Federico II
Department of  Molecular Medicine and Medical Biotechnology - University of Naples "Federico II"

### Submitter
Michele  Costanzo

### Lab Head
Dr Michele Costanzo
Department of  Molecular Medicine and Medical Biotechnology - University of Naples "Federico II"


