### Accession
PXD008075

### Title
Cell wall enrichment unveils proteomic changes in the cell wall during treatment of Mycobacterium smegmatis with sub-lethal concentrations of Rifampicin

### Description
In this study, we successfully developed two gel-free approaches to specifically look at cell wall proteins in Mycobacterium smegmatis. The cell wall was subjected to differential centrifugation, differential detergent solubilisation and phase separation to yield the genuine cell wall proteome. The protein extracts were digested by filter-assisted sample preparation for LC-MS/MS analysis on a Q-Exactive mass spectrometer and identified proteins subjected to a strict bioinformatics pipeline. This resulted in the unprecedented coverage of 96 lipoproteins, 475 membrane proteins containing at least one transmembrane helix and 73 secreted proteins. We used this gel-free approach to study the changes in the cell wall proteome during exposure of M. smegmatis to sub-lethal concentration of Rifampicin in a quantitative manner. This enabled us to characterise in detail the dysregulation of transporters such as TatB and ABC transporters, virulence factors such as MCE proteins and PknG, as well as proteins involved in cell wall and lipid synthesis.

### Sample Protocol
Growth conditions and harvesting M. smegmatis strain mc2155 was grown 7H9 broth with OADC. Samples for the rifampicin experiment, on reaching an optical density (OD600) of 1.2 were treated with either 5 µg/mL rifampicin - corresponding to ½ MIC - in DMSO (treated) or with DMSO only (control). The bacterial cultures were harvested at 255 minutes post-treatment by centrifugation. Pellets were washed twice by re-suspension in PBS, snap frozen in liquid nitrogen and stored at 80°C before lysis. This was repeated, yielding a total of eight cultures, four in each treatment group.  Cell wall protein preparation Washed cell pellets were resuspended in lysis buffer and lysed by sonication for six times 30 seconds with 1-minute cooling on ice in between. Cell debris were pelleted by centrifugation at 8000 x g for 10 min and the lysate was filtered through a 0.2 µm filter. Equal amount of lysate per biological replicate was split into 4 tubes and the cell wall fraction was pelleted by centrifugation at 22,000 x g for 30 minutes at 4°C. The supernatant containing the cytosolic fraction was kept for further analysis. The cell wall pellet was washed twice with PBS, the resulting cell wall pellet was resuspended in (i) 2% Triton X114 (Sigma), (ii) 0.1% Triton X100, 0.05% Tween-20 and 0.2% CHAPS, (iii) 1% DDM or (iv) 0.15% deoxycholate and 0.1% SDS and further incubated for 2 h shaking at 4°C. Insoluble aggregates were removed by centrifugation at 22,000 x g for 10 min. Triton X114 samples were further subjected to phase separation by incubation in a water bath at 37°C for 15 minutes. The aqueous phase was discarded and the detergent phase backwashed twice with PBS. The total proteome of each sample was then precipitated with chloroform-methanol and the protein pellet was solubilised in UA buffer (8 M urea, 0.1 M Tris, pH 8.5).  Cell debris preparation included The cell debris were washed once with PBS and resuspended in 0.1 M Tris, pH 8.5, 2% SDS and 100 mM DTT and boilt at 95 °C for 30 minutes. The samples were cooled down to room temperature on ice, brought up to 6 M Urea and shaken at room temperature for 30 minutes.    Filter-aided sample preparation Samples were washed and digested according to the fasp protocol. Briefly, samples were loaded on to Ultracel 30,000 MWCO centrifugal units (Amicon Ultra, Merck) and three times buffer exchanged into UA buffer. Proteins were reduced by addition of 100 mM DTT for 30 minutes. Reduced cysteines were alkylated by addition of 0.05 M Iodoacetamide for 20 minutes in the dark at room temperature. Alkylating reagent was removed by three round of buffer exchange with UA buffer followed by three round of buffer exchange into ABC buffer (0.05 M ammonium bicarbonate pH 8.0, 20 mM CaCl2). Sequence-grade Trypsin (NEB) was added at an enzyme to protein ratio of 1:50 (second experiment) and digestion was allowed for 18 hrs at 37°C in a wet chamber. Tryptic peptides were eluted and desalted using homemade C18 STAGE tips. Peptides were then eluted into glass capillary tubes. Eluted peptides were dried in a vacuum and resuspended with 2% ACN at a concentration of 200 ng/µl.  LC-MS/MS analysis Peptides were seperated by HPLC using a homemade 2cm precolumn connected to a 75 μM × 20 cm analytical column packed with C18 Luna beads (5 μm diameter, 100 Å pore size) connected to an Ultimate 3500 RS nano UPLC system (Dionex). 200 ng of peptides per sample was loaded onto the column with a starting mobile phase of 2% ACN and separated by the two following gradients: 10 min at 2% ACN, increase to 6% ACN for 2 min, to 30% ACN over 60 min, to 80% ACN over 5 min, followed by a column wash of 80% for 13 min for the first (cell wall preparation) experiment; 10 min at 2% ACN, increase to 6% ACN for 2 min, to 60% ACN over 60 min, to 80% ACN over 5 min, followed by a column wash of 80% for 13 min for the second (rifampicin treatment) experiment. Mass spectra were collected on an Orbitrap Q Exactive mass spectrometer (Thermo) using a top-10 method. MS spectra were acquired at a resolution of 70 000 with a target value of 3E6 ions or a maximum integration time of 250 ms. The scan range was restricted between 300 and 1750 m/z. Peptides were fragmented HCD with the energy set at 25 NCE. Intensity threshold for ions selection was fixed at 1.7E4 with charge exclusion of z = 1 and z > 5. The MS/MS spectra were acquired at a resolution of 17 500, with a target value of 2E5 ions or a maximum integration time of 120 ms and the isolation window was set at 4.0 m/z.

### Data Protocol
RAW data was processed with Maxquant version 1.5.3.12 for protein and peptide identification using the Andromeda search engine and the Uniprot proteome for M. smegmatis (Proteome ID: UP000000757, 6601 proteins, 14/05/2016). Default parameter settings were used for the MS/MS database search, and carbamidomethylation of cysteine residues and acetylation of the protein N-terminus were selected as fixed modification. Trypsin/P was selected as protease, LFQ quantitation and the match between runs feature was enabled. The proteinGroups.txt output of MaxQuant was loaded in R and contaminants and reverse hits were removed. iBAQ quantitation values were median normalized. Significance of altered abundance was determined by means of the student’s t-test with p<0.05 for all proteins with at least three non-zero values in each treatment group.

### Publication Abstract
Understanding the cell wall of mycobacteria is crucial for improving drug design or identifying new antigens suitable to vaccination. Yet this remains problematic due to the complexity of the cell wall composition. In this study, we successfully developed gel-free approaches to study cell wall proteins in Mycobacterium smegmatis. The cell wall was subjected to differential centrifugation, differential detergent solubilisation and phase separation to yield the genuine cell wall proteome. Next, protein extracts were digested by filter-assisted sample preparation for LC-MS/MS analysis on a Q Exactive mass spectrometer, and identified proteins filtered through a stringent bioinformatics pipeline. This yielded the unprecedented coverage of 96 lipoproteins, 475 membrane proteins and 73 secreted proteins. Employing this approach, we next quantified changes in the cell wall proteome during exposure of M. smegmatis to sub-lethal concentration of rifampicin. This facilitated detailed characterisation of the dysregulation of ABC transporters, virulence factors such as Mce proteins and PknG, and proteins involved in cell wall and lipid synthesis. Crucially, these cell wall proteins are under-represented in previous proteome analysis of M. smegmatis. This approach enables further quantitative proteomic studies of the role of the cell wall proteome of mycobacteria in virulence or during drug exposure. SIGNIFICANCE: We developed novel gel-free sample preparation workflows for the cell wall fraction of mycobacteria that significantly increase the coverage of the cell wall proteome compared to previous studies. We then provide a data analysis workflow that enables the removal of likely cytosolic contaminants in the cell wall fraction post-measurement. Combined, these approaches increase the coverage of the cell wall proteome while ensuring that the identified proteins are true cell wall proteins and not carry-over of high-abundance contaminants from the cytosol. We have applied these approaches to quantify the dysregulation of cell wall proteins during exposure of M. smegmatis to rifampicin, which has shed new light on the coordinated down-regulation of ABC transporters as well as virulence factors present in the cell wall proteome.

### Keywords
Cell wall proteomics, Rifampicin, Mycobacteria, Tb drugs, Sub lethal, Mass spectrometry, Tb

### Affiliations
Division of Chemical and Systems Biology, Department of Integrative Biomedical Science, Institute for Infectious Disease and Molecular Medicine, Faculty of Health Science, University of Cape Town, South Africa.
University of Cape Town

### Submitter
Alexander Giddey

### Lab Head
Dr Jonathan M Blackburn
Division of Chemical and Systems Biology, Department of Integrative Biomedical Science, Institute for Infectious Disease and Molecular Medicine, Faculty of Health Science, University of Cape Town, South Africa.


