### Accession
PXD007099

### Title
ZBTB2 as a novel reader for unmethylated DNA

### Description
In this study, we investigated the dynamics during differentiation of the in vivo binding sites of ZBTB2, a putative reader for unmethylated DNA. We performed DNA pull-downs followed by mass spectrometry, using a genomic sequence containing either unmethylated or methylated CpGs, to study the influence of DNA methylation on ZBTB2 binding. Additionally, we performed interaction proteomics to identify ZBTB2 interaction partners. We found that ZBTB2 recruits a zinc finger module of three proteins to unmethylated DNA.

### Sample Protocol
Proteins were on-bead digested with trypsin, after which tryptic peptides were purified on C18-StageTips (Rappsilber et al. 2003). Samples from DNA pull-downs were labelled by on-StageTip dimethyl labelling, in which StageTips were loaded with 300 Î¼l labelling buffer (2% of either CH2O (light) or CD2O (medium) in 10mM NaH2PO4, 35mM Na2HPO4, 0.6M NaBH3CN) and centrifuged.  Tryptic peptides were eluted from Stage-Tips. For dimethyl labelled samples, the respective light and medium labelled samples were combined into a forward and a reverse reaction. Peptides were then separated on an Easy-nLC 1000 (Thermo) connected online to an LTQ-Orbitrap Fusion Tribrid mass spectrometer (Thermo) or an LTQ-Orbitrap Q-Exactive mass spectrometer (Thermo), using a gradient of acetonitrile (7%-32% or 5%-30%, respectively), followed by washes at 50% then 90% acetonitrile, for 140 min or 120 min of total data collection. For samples measured on the Fusion, scans were collected in data-dependent top-speed mode of a 3 second cycle with dynamic exclusion set at 60 sec. For samples measured on the Q-Exactive, the top ten most intense precursor ions were selected for fragmentation.

### Data Protocol
Peptides were searched against the UniProt mouse proteome (release 2015_12) with MaxQuant v1.5.1.0 (Cox and Mann 2008), using default settings, the appropriate dimethyl labels, and re-quantify and/or match between runs enabled. Data were analysed with Perseus version 1.4.0.0 and in-house R scripts.

### Publication Abstract
Proteins that bind to DNA depending on its methylation status play an important role in methylation-mediated regulation of gene expression. Using a variety of genomics and proteomics approaches, we identify zinc finger and BTB domain-containing protein 2 (ZBTB2) as a reader of unmethylated DNA in mouse embryonic stem cells. ZBTB2 preferentially binds to CpG island promoters, where it acts as a transcriptional activator. The binding of ZBTB2 to its targets is direct and independent of two other zinc finger proteins, ZBTB25 and ZNF639, which we show to interact with ZBTB2. Our data suggest an anticorrelation between ZBTB2 DNA binding and DNA methylation, indicating that ZBTB2-binding dynamics <i>in&#xa0;vivo</i> are sensitive to differential DNA methylation. ZBTB2 is intricately interwoven with DNA methylation, as we find not only that its binding to DNA is methylation sensitive, but also that ZBTB2 regulates the turnover of methylated DNA In ZBTB2 knockout cells, several pluripotency factors are upregulated, inducing a delay in differentiation. We propose that ZBTB2 is a novel DNA methylation-sensitive transcription factor that regulates cellular differentiation.

### Keywords
Npc, Mouse, Esc, Zbtb2

### Affiliations
Department of Molecular Biology, Faculty of Science, Radboud Institute for Molecular Life Sciences, Radboud University Nijmegen, Nijmegen, the Netherlands
University of Zurich

### Submitter
Ino Karemaker

### Lab Head
Dr Michiel Vermeulen
Department of Molecular Biology, Faculty of Science, Radboud Institute for Molecular Life Sciences, Radboud University Nijmegen, Nijmegen, the Netherlands


