### Accession
PXD036182

### Title
In vivo proximity labeling identifies the interactome of the endoribonuclease Zucchini in Drosophila ovary.

### Description
PIWI-interacting RNAs (piRNAs) are small RNAs that play a conserved role in genome defense by silencing transposable elements. The piRNA processing pathway is dependent on the sequestration of RNA precursors and protein factors in specific subcellular compartments. Therefore, a highly resolved spatial proteomics approach can help identify the local interactions and thereby elucidate the unknown aspects of piRNA biogenesis. Herein, we performed TurboID proximity labeling to investigate the interactome of Zucchini (Zuc), a key factor of piRNA biogenesis in germline cells and somatic follicle cells of the Drosophila ovary. Quantitative mass spectrometry analysis of biotinylated proteins defined the Zuc-proximal proteome, including the well-known partners of Zuc in piRNA biogenesis. Many of these were enriched in the cellular compartment of mitochondria or the outer mitochondrial membrane (OMM), where Zuc was specifically localized. A unique subset of proteins in the Zuc proximal proteome was characterized in comparison with the Tom20 proximal proteome, despite the overlapping subcellular localization of Zuc and Tom20 on OMM, indicating that the proximal proteomes in this analysis have target protein specificity beyond the subcellular localization dependency. Interestingly, the data indicated that chaperone function-related and endomembrane system/vesicle transport proteins are novel interacting partners of Zuc. The functional relevance of several candidates in piRNA biogenesis was validated by derepression of transposable elements after knockdown. Our results not only present potential Zuc-interacting proteins, but also suggest unrecognized biological processes, providing new insights into the physiological functions and molecular mechanisms of Zuc.

### Sample Protocol
Sample preparation for proteomics Ovary tissues were lysed with lysis buffer (2% SDS in 1X TBS, 1X protease inhibitor cocktail) and sonicated (Covaris M220 ultrasonicator). Cold acetone was added to the lysate for protein precipitation overnight at -20°C. The pellet was then washed with 90% cold acetone and 10% TBS. After air-drying, the pellet was solubilized with 8 M urea in 50 mM ammonium bicarbonate (ABC). A total of 3 mg of protein in 500 µl of 8M urea, and 50 mM ABC were used for subsequent trypsin digestion. Proteins were denatured for 1 h at 37°C with shaking at 450 rpm and reduced with 10 mM DTT for 1 h at 37°C with shaking at 450 rpm. Proteins were alkylated with 40 mM iodoacetamide at 37°C for 1 h with shaking at 450 rpm. The samples were diluted eight times with 50 mM ABC. CaCl2 was then added at a final concentration of 1 mM. The protein samples were digested with trypsin (Thermo Scientific, #20233) for 16 h at 37°C at a 1:50 (w/w) trypsin-to-protein ratio. Streptavidin (SA) beads (Invitrogen, #65001) were then added to the samples after several washes with 2 M urea in 1X TBS and incubated for 1 h at RT. The beads were washed twice with 2 M urea in 50 mM ABC, and then with pure water. Biotinylated peptides were eluted with 100 µl of 80% ACN, 0.2% TFA, and 0.1% formic acid at 60°C for five times. The combined elution fractions were dried using a Speed Vac and used for mass spectrometry analysis.

### Data Protocol
MS data analysis All MS/MS data were searched using MaxQuant (version 1.6.2.3) with the Andromeda search engine at a 10 ppm precursor ion mass tolerance against the Uniprot Drosophila melanogaster proteome database (42,824 entries, UniProt (http://www.uniprot.org/)). Label-free quantification (LFQ) and matching between runs were performed with the following search parameters: trypic digestion, fixed carbaminomethylation on cysteine, dynamic oxidation of methionine, dynamic protein N-terminal acetylation and dynamic biotin labels of lysine residues. A false discovery rate (FDR) of less than 1% was obtained for uniquely labeled peptides and proteins.  Bioinformatics For data processing and visualization of mass spectrometry data, LFQ intensity values from MaxQuant were analyzed using the R package DEP which provides tools for filtering, variance normalization, imputation of missing values and statistical testing for differentially expressed proteins. For the analysis, we first filtered out proteins that contained missing values in any replicate for each condition. Background correction and normalization were performed using a variance-stabilizing transformation (vsn). Condition-specific missing values were imputed using a small deterministic value (R package Msnbase:MinDET method). The missing values were replaced with a minimal value observed in that sample, which is estimated as being the q-th quantile (default ‘q = 0.01’ was used). Differential enrichment analysis (DEA) was performed by applying protein-wise linear models combined with empirical Bayes statistics between conditions (R package limma). The q-value was generated from raw p-values using R package qvalue for further differential enrichment analysis. Protein-protein interactions (PPIs) were derived from the Search Tool for the Retrieval of Interacting Genes/Proteins (STRING, stringApp v1.7 in Cytoscape v3.9.0-BETA1) database (STRING, confidence score cut-off value:0.4, maximum additional interactors:0). Clustering of the PPI networks was achieved using the Markov cluster algorithm (MCL, granular parameter:4). Gene Ontology (GO) biological process (BP) and cellular component (CC) enrichment analyses were conducted using the ClueGO software (ClueGO v2.5.8 in Cytoscape v3.9.0-BETA1). Overview of total proteins with the data-centered per protein was visualized using a heatmap.

### Publication Abstract
PIWI-interacting RNAs (piRNAs) are small RNAs that play a conserved role in genome defense. The piRNA processing pathway is dependent on the sequestration of RNA precursors and protein factors in specific subcellular compartments. Therefore, a highly resolved spatial proteomics approach can help identify the local interactions and elucidate the unknown aspects of piRNA biogenesis. Herein, we performed TurboID proximity labeling to investigate the interactome of Zucchini (Zuc), a key factor of piRNA biogenesis in germline cells and somatic follicle cells of the Drosophila ovary. Quantitative mass spectrometry analysis of biotinylated proteins defined the Zuc-proximal proteome, including the well-known partners of Zuc. Many of these were enriched in the outer mitochondrial membrane (OMM), where Zuc was specifically localized. The proximal proteome of Zuc showed a distinct set of proteins compared with that of Tom20, a representative OMM protein, indicating that chaperone function-related and endomembrane system/vesicle transport proteins are previously unreported interacting partners of Zuc. The functional relevance of several candidates in piRNA biogenesis was validated by derepression of transposable elements after knockdown. Our results present potential Zuc-interacting proteins, suggesting unrecognized biological processes.

### Keywords
Drosophila, Lc-ms/ms, Turboid, Proximity labeling

### Affiliations
School of Biological Sciences, Seoul National University
Assistant professor, School of Biological Sciences, Seoul National University.

### Submitter
Jong-Seo Kim

### Lab Head
Dr Jong-Seo Kim
Assistant professor, School of Biological Sciences, Seoul National University.


