### Accession
PXD029934

### Title
Proteomics of the venom from Apis mellifera

### Description
Venoms have convergently evolved in all major animal lineages and are ideal candidates to unravel the underlying genomic processes of convergent trait evolution. However, few animal groups have been studied in detail, and large-scale comparative genomic analyses to address toxin gene evolution are rare. Hyper-diverse hymenopterans are the most speciose group of venomous animals, but the origin of their toxin genes have been largely overlooked. We combined proteo-transcriptomics with comparative genomics compiling an up-to-date list of core bee venom proteins to investigate the origin of 11 venom genes in 30 hymenopteran genomes including two new stingless bees.

### Sample Protocol
The samples were reduced / alkylated and digested with trypsin (2 μg, liquid digestion) and desalted on OMIC C18. The peptide samples were taken up in 20 μl of buffer A (0.1% formic acid). 0.5 µL / sample were injected for on-line analysis using nano-flow HPLC (RSLC U3000, Thermo Fisher Scientific) coupled to a mass spectrometer equipped with a nanoelectrospray source (Q Exactive HFX , Thermo Fisher Scientific). Peptides were separated on a capillary column (0.075 mm × 250 mm, Acclaim Pepmap 100, reverse phase C18, NanoViper, Thermo Fisher Scientific) at a gradient of 6-40% buffer B over 123 min (A = 0.1 % formic acid B = 0.1% formic acid, 80% acetonitrile) at a flow rate of 300 nl / min. The spectra were recorded using Xcalibur 4.1 software (Thermo Fisher Scientific.

### Data Protocol
The spectra were recorded using Xcalibur 4.1 software (Thermo Fisher Scientific). PEAKS Studio 8.5 software (Bioinformatics solutions, Waterloo, ON, Canada) was used to match MS/MS spectra obtained from proteomic analysis each venom. MS spectra were elucidated based on a personalized database resulting from assembled contigs translated into their six reading frames. Carbamidomethylation was set as fixed modification, while oxidation (M) was set as variable modifications, with maximum missed cleavages at 3 for trypsin digestion. Parent mass and fragment mass error tolerance were set at 5 ppm and 0.015 Da respectively. False discovery rate (FDR) of 1% and unique peptide ≥2 were used for filtering out inaccurate proteins. A −10lgP > 120 was used to estimate whether the detected proteins was identified by enough reliable peptides MS/MS spectra. In order to identify more relevant sequences, the Spider algorithm from PEAKS Studio software was used to find additional mutations or to correct the sequences. This algorithm corrects the sequences stored in transcriptomic database with de novo sequences based on MS/MS spectra, which allowed to identify post-translational modifications (PTMs) and mutations. Minimum ion intensity for mutation and PTMs was set to 5%, and ALC score ≥ 90 for de novo sequences leading to low precursor mass error in order to identify reliable PTM’s and potential mutations.

### Publication Abstract
None

### Keywords
Venom

### Affiliations
IBMM, Univ Montpellier, CNRS, ENSCM, 34095 Montpellier, France
Protein Analytics,  Institute of Biiochemistry, Faculty of Medicine, University Giessen, Friedrichstrasse 24, 35392 Giessen

### Submitter
Günter Lochnit

### Lab Head
Dr Sebastien Dutertre
IBMM, Univ Montpellier, CNRS, ENSCM, 34095 Montpellier, France


