### Accession
PXD033960

### Title
Confirmation of direct interaction between H3K56cr and GLYR1 by H3K*56cr Site-Link in living cells

### Description
As we have found that H3K56cr can crosslink with GLYR1 in living cells, we now perform H3K*cr Site-Link in GLYR1 overexpressed cells. The subsequent 365-nm light irradiation could activate the diazirine and covalently capture potential effector proteins on native chromatin in living cells. The crosslinked band could be identified after gel-based proteomics and pLink assisted MS/MS analysis of the crosslinked peptide.

### Sample Protocol
Plasmids pcDNA4-H3-K56TAG (C-terminal myc-His-tag) was transfected into the relative GLYR1 overexpressed Kcr-PylRS stable cell line containing and supplemented with 0.2 mM K*cr. After growing for 6 h, the transfected HEK293T cells were changed into DMEM with 1mM K*cr. The cells were cultured for additional 12/24/36/48 hours to express the corresponding H3 protein H3K*56cr. The medium was discarded and Dulbeccoâ€™s phosphate buffered saline (DPBS) was added. After ultraviolet irradiation of the cells on ice for 15 min, cells were harvested. The histones were extracted by TCA precipitation and enriched by his-tag isolation. The eluents were combined, concentrated, and separated by SDS-PAGE. The relative crosslinked bands were cut for subjected to HPLC-MS/MS proteomic study.

### Data Protocol
The raw files were searched against our expressed histone H3 and GLYR1 fasta (downloaded from Uniprot on March 13th, 2022) by pLink (version 2.3.11). with the following parameters: The carbamidomethyl (Cys) was set as the fixed modification, while oxidation (Met) was set as variable modification. K*cr-N2 was set as the Xlinking part. +/- 10 ppm was set for peptide pass tolerance and +/- 0.6 Da was set for fragment mass tolerance. A maximum of two missed cleavages was allowed for trypsin digestion. FDR was determined by searching a reverse database and was set to 0.01 for proteins and peptides.

### Publication Abstract
A generalizable strategy with programmable site specificity for in situ profiling of histone modifications on unperturbed chromatin remains highly desirable but challenging. We herein developed a single-site-resolved multi-omics (SiTomics) strategy for systematic mapping of dynamic modifications and subsequent profiling of chromatinized proteome and genome defined by specific chromatin acylations in living cells. By leveraging the genetic code expansion strategy, our SiTomics toolkit revealed distinct crotonylation (e.g., H3K56cr) and &#x3b2;-hydroxybutyrylation (e.g., H3K56bhb) upon short chain fatty acids stimulation and established linkages for chromatin acylation mark-defined proteome, genome, and functions. This led to the identification of GLYR1 as a distinct interacting protein in modulating H3K56cr's gene body localization as well as the discovery of an elevated super-enhancer repertoire underlying bhb-mediated chromatin modulations. SiTomics offers a platform technology for elucidating the "metabolites-modification-regulation" axis, which is widely applicable for multi-omics profiling and functional dissection of modifications beyond acylations and proteins beyond histones.

### Keywords
Plink, H3k56cr, Glyr1

### Affiliations
Peking University
Professor and Chair of Chemical Biology, Peking-Tsinghua Center for Life Sciences Peking University, Beijing, China. Executive Editor, ACS Chemical Biology

### Submitter
Fangfei Qin

### Lab Head
Dr Peng R. Chen
Professor and Chair of Chemical Biology, Peking-Tsinghua Center for Life Sciences Peking University, Beijing, China. Executive Editor, ACS Chemical Biology


