### Accession
PXD016125

### Title
The heat shock response relies on orthogonal mechanisms to maintain protein homeostasis in yeast

### Description
We globally analyzed the heat stress response (HSR) in yeast on the transcriptomic, tranlatomic and the proteomic level. The cells were stressed at 37 °C, 42 °C and 46 °C and the fractionated proteomes measured on a Fusion mass spectrometer. With this approach we were able to reveal a modular character of the HSR dependent on the severity of the stress. We showed that at temperatures up to 42 °C besides upregulation of stress protective factors refilling the pool of degraded and aggregating proteins is an important branch of the HSR. Whereas, at sublethal temperatures (46 °C) lots of aggregation and phase transition processes were observed. This is part two of the pride submission and only contains the proteomic data under inhibition of the proteasome with bortezomib.

### Sample Protocol
LFQ 50 ml cultures were inoculated to an OD595 = 0.2 and grown at 25 °C until an OD of 0.8 was reached. The cells were treated with 50 µM bortezomib and preincubated at 25 °C for 1 h before they were heat stressed for 30 min at 37 °C, 42 °C and 46 °C in a water bath, respectively. For measurement of recovery, the cells were shifted back to 25 °C for 60 min. The cells were harvested for 1 min at 4200 x g and the samples prepared on the basis of the protocol published by (Wallace et al., 2015)(total fraction). The concentration of the samples was determined via BCA assay (Pierce) and the lysates flash frozen in liquid nitrogen. For MS analysis 250 µg protein were precipitated by the methanol-chloroform-water method as described elsewhere (Wessel and Flugge, 1984). The pellets were dissolved in 198 µl 8 M Urea, 0.1 M Tris (pH 8.0) and 2 µl 5 M DTT were added. The samples were prepared according to the filter aided sample preparation (FASP) method (Wiśniewski et al., 2009) and digested with Trypsin (Promegea) overnight at 37 °C. After stop of digest with TFA, the samples were desalted with 50 mg SEP PAK (tC18) columns (Waters). After equilibrating the column with 1 ml acetonitrile (ACN), 500 µl elution buffer (80 % (v/v) ACN, 0.5 % (v/v) formic acid (FA) and three times with 1 ml 0.1 % (v/v) TFA, the samples were loaded and the column was washed three times with 1 ml 0.1 % (v/v) TFA and one time with 250 µl 0.5 % (v/v) FA. The samples were eluted with two times 250 µl elution buffer by gravity flow and then with 250 µl elution buffer and vacuum applied. The samples were dried in a speed vacuum centrifuge (Eppendorf).The samples were dissolved in 20 µl 1 % FA and the solutions filtered through 0.22 µm centrifugal filters (Merck; 2 min at 16200 x g). MS/MS measurement MS analysis was performed on an Orbitrap Fusion instrument coupled to an Ultimate3000 Nano-HPLC via an electrospray easy source (all Thermo Fisher Scientific). Samples were loaded on a 2 cm PepMap RSLC C18 trap column (particles 3 µm, 100A, inner diameter 75 µm, Thermo Fisher Scientific) with 0.1% TFA and separated on a 50 cm PepMap RSLC C18 column (particles 2 µm, 100A, inner diameter 75 µm, Thermo Fisher Scientific) constantly heated at 50 °C. The gradient was run from 5-32% acetonitrile, 0.1% formic acid during a 152 min method (7 min 5%, 105 min to 22%, 10 min to 32%, 10 min to 90%, 10 min wash at 90%, 10 min equilibration at 5%) at a flow rate of 300 nl/min. For measurements on the fusion instrument survey scans (m/z 300-1500) were acquired in the orbitrap with a resolution of 120000 at m/z 200 and the maximum injection time set to 50 ms (target value 2e5). Most intense ions of charge states 2-7 were selected for fragmentation with high-energy collisional dissociation at a collision energy of 30 %. The instrument was operated in top speed mode and spectra acquired in the ion trap with the maximum injection time set to 50 ms (target value 1e4). The option to inject ions for all available parallelizable time was enabled. Dynamic exclusion of sequenced peptides was set to 60 s. Real-time mass calibration was based on internally generated fluoranthene ions. Data were acquired using Xcalibur software version 3.0sp2 (Thermo Fisher Scientific). Data were acquired using Xcalibur software version 3.1sp3 (Thermo Fisher Scientific).

### Data Protocol
Quantitative analysis of the MS data was performed with MaxQuant (version 1.6.2.6) (Cox et al., 2014, Cox and Mann, 2008). The raw files were searched against the S. cerevisiae proteome database downloaded from UniprotDB. All files were assigned to the same fraction, only tryptic peptides were searched, cleavage sites before proline were included. Up to two missed cleavage sites were allowed as well as a peptide tolerance of 4.5 ppm. As variable modifications N-terminal acetylation and methionine oxidation were selected, as fixed modification carbamidomethylation with maximal 5 modifications per peptide. Label free quantification (LFQ) was performed with a min. ratio count of 1. Otherwise, the orbitrap instrument settings of MaxQuant were applied. Minimal peptide length was set to 7 amino acids, the maximal length was 25 amino acids. To enhance protein recovery match between runs was applied (match time window 0.7 min; align window 20 min). The identification parameters were left preset with a protein FDR of 1 %. Label min. ratio count was set to 1 and unique + razor peptides were taken into account. For the statistic validation of the results, the peptides were also searched against a reverse decoy database calculated by MaxQuant. The processed raw data were further evaluated by Perseus (version 1.6.2.1) (Tyanova et al., 2016). At first, the protein groups file was filtered by potential contaminant hits, hits from the reverse database and hits only identified by site. Subsequently, the LFQ intensities were log2 transformed, grouped into replicates and the rows filtered on 3 valid values in each replicate group, respectively. Missing values were calculated from the normal Gaussian distribution. Concluding volcano plots with a both sided t-test, a FDR of 0.05 and a S0 of 0.1 were plotted (Benjamini and Hochberg, 1995). The bortezomib LFQ data were additionally histone normalized to correct for the internal normalization used by MaxQuant, which assumes that the overall protein concentration is constant in all samples.

### Publication Abstract
Life is resilient because living systems are able to respond to elevated temperatures with an ancient gene expression program called the heat shock response (HSR). In yeast, the transcription of hundreds of genes is upregulated at stress temperatures. Besides stress protection conferred by chaperones, the function of the majority of the upregulated genes under stress has remained enigmatic. We show that those genes are required to directly counterbalance increased protein turnover at stress temperatures and to maintain the metabolism. This anaplerotic reaction together with molecular chaperones allows yeast to efficiently buffer proteotoxic stress. When the capacity of this system is exhausted at extreme temperatures, aggregation processes stop translation and growth pauses. The emerging concept is that the HSR is modular with distinct programs dependent on the severity of the stress.

### Keywords
Aggregation, Heat shock response, Proteome, Chaperones

### Affiliations
Dept. Chemie
Lichtenbergstraße 4
85748 Garching

Technische Universität München, Dept. Chemistry, Chair of Biotechnology

### Submitter
Moritz Mühlhofer

### Lab Head
Dr Johannes Buchner
Technische Universität München, Dept. Chemistry, Chair of Biotechnology


