### Accession
PXD016924

### Title
Definition and characterization of the mitochondrial proteome of human cells (MitoCoP)

### Description
Aim of this project was the definition and characterization of the mitochondrial proteome of human cells, referred to as MitoCoP. To this end, we combined the complementary strengths of subtractive proteomics, ImportOmics and subcellular protein profiling in a global multidimensional classification approach. This multifaceted human mitochondria-centered analysis was expanded to a functional characterization of the mitochondrial proteome. The association of proteins with mitochondrial membranes was assessed by quantitative proteomic and single-protein studies. Protein copy numbers per mitochondrial unit were determined followed by a thorough compilation and functional classification of a high-quality human mitochondrial proteome. We further examined the organization of mitochondrial proteins in large functional assemblies by Blue Native (BN) complexome profiling and characterized functional protein networks of so far not described mitochondrial constituents by quantitative affinity purification mass spectrometry (q-AP-MS) and biochemical analysis. And finally, we systematically studied mitochondrial proteome dynamics and established a comprehensive subunit-resolved turnover map of large membrane protein machineries and central biosynthesis pathways of human mitochondria.

### Sample Protocol
LC/MS sample preparation: In this part of the project, six comprehensive quantitative MS-based experiments were performed that contributed to the accurate definition and characterization of the MitoCoP. (1) In subtractive proteomics (pure versus crude mitochondrial fractions; pM-vs-cM) experiments, crude and gradient-purified mitochondrial fractions were prepared from differentially SILAC-labeled HEK cells and mixed in equal amounts. Proteins were digested in solution using multiple proteases (LysC/trypsin, chymotrypsin, AspN or GluC) followed by offline high pH reversed-phase liquid chromatography (RP-LC) for peptide fractionation (32 fractions; n = 4 biological replicates including label-switch). (2) In importomics experiments, equal amounts of differentially SILAC-labeled TOMM40-shRNA HeLa cells treated with doxycycline (DOX) and the corresponding mock-treated control cells were mixed. Gradient-purified mitochondria were prepared, proteins were digested in solution with trypsin, and peptides were fractionated offline by high pH RP-LC (32 fractions; n = 4 biological replicates including label-switch). (3) In spatial proteomics experiments, total cell lysate and nuclear, cytosolic, microsomal, crude and pure mitochondrial fractions were prepared from HEK cells (n = 3; label-free). Proteins were digested in solution using LysC/trypsin, and peptides were fractionated offline by high pH RP-LC (16 fractions). (4) To determine half-lives of MitoCoP proteins, pulsed SILAC (pSILAC) experiments were performed using HeLa cells (n = 3 with label-switch). Equal numbers of differentially SILAC-labeled pulsed and control cells were mixed. Crude mitochondrial fractions were prepared and proteins were digested in solution using LysC/trypsin. Peptides were fractionated offline by high pH RP-LC (8 fractions). (5) Complexome profiling: gradient-purified mitochondria were prepared from HEK cells and analyzed by blue native polyacrylamide gel electrophoresis (n = 1, label-free). The gel lane was cut into 116 slices and proteins were in-gel digested with trypsin. (6) Carbonate/Sonication assays: gradient-purified mitochondria, prepared from SILAC-labeled HEK cells, were either treated with carbonate or sonified (n = 4; with label-switch). Samples were separated into soluble and insoluble fractions. Proteins were digested in-solution with trypsin and peptides were labeled by stable isotope dimethyl labeling. Differentially labeled peptides from equal volumes of soluble and insoluble fractions of each replicate were mixed and fractionated by high pH RP fractionation using StageTips (8 fractions). LC/MS sample preparation further included reduction of cysteine residues with 10 mM dithiothreitol (gel slices) or 10 mM Tris(2-carboxy-ethyl)phosphine (in solution digests) and subsequent alkylation of free thiol groups with 55 mM iodoacetamide.   LC/MS analysis: Nano-HPLC-ESI-MS/MS analyses were performed at an Orbitrap Elite mass spectrometer connected to an UltiMate 3000 RSLCnano HPLC system (Thermo Scientific). Peptides were washed and preconcentrated using PepMap C18 precolumns (length, 5 mm; inner diameter, 0.3 mm) at a flow rate of 30 ������l/min. Peptide separation was performed using a C18 reversed-phase nano LC column (Acclaim PepMap; length, 500 mm; inner diameter, 75 ������m; particle size, 2 ������m; packing density, 100 ������) at 43������C and a flowrate of 250 nl/min. Peptides were separated using a binary solvent system consisting of 4% (v/v) dimethyl sulfoxide (DMSO)/0.1% (v/v) FA (solvent A) and 30% (v/v) ACN/48% (v/v) methanol/4% (v/v) DMSO/0.1% (v/v) FA (solvent B). Peptides were separated by increasing concentrations of solvent B. LC gradients differed in length and slope between experiments depending on the complexity of individual samples (details are available upon request).  Peptides eluting from the LC column were transferred to a stainless steel emitter (Thermo Scientific) for electrospray ionization using a Nanospray Flex ion source with DirectJunctionTM adaptor (Thermo Scientific) and applying a spray voltage of 1.8 kV and a capillary temperature of 200������C. Mass spectrometric data were acquired in data-dependent mode with a mass range of m/z 370 to 1,700, a resolution of 120,000 at m/z 400, a target value of 1 x 10E06, and a maximum injection time of 200 ms for MS survey scans. The most intense precursor ions with a charge ��������� +2 (up to 15 for the complexome profiling experiment; up to 20 for pM-vs-cM experiments; up to 25 for all other experiments) were selected for low energy collision-induced dissociation in the linear ion trap with the following parameters: normalized collision energy, 35%; activation q, 0.25; activation time, 10 ms; target value, 5,000; maximum injection time, 150 ms; dynamic exclusion time, 45 s.

### Data Protocol
MS data analysis: Mass spectrometric raw data of all experiments (subtractive proteomics, i.e. pM-vs-cM experiments; TOMM40 importomics experiments; spatial proteomics experiments; pSILAC experiments; complexome profiling; carbonate/sonication assays) were jointly processed using MaxQuant v.1.6.0.1 (Cox und Mann, 2008). For peptide identification, mass spectra were correlated with the Uniprot human proteome set including isoforms (retrieved 08/2018; 95,106 entries) and a list of common contaminants provided by MaxQuant using Andromeda (Cox et al., 2011). Database searches were performed with a mass tolerance of 4.5 ppm for precursor ions and 0.5 Da for fragment ions, carbamidomethylation of cysteine as fixed and oxidation of methionine and N-terminal acetylation as variable modifications. To account for differences in the workflows of the individual experiments, different parameter groups with group-specific parameters were defined. For group 0 (containing data of pM-vs-cM and importomics experiments), Arg10 and Lys8 were selected as heavy labels, multiplicity was set to 2, Trypsin/P was selected as enzymatic specificity, a maximum of 3 missed cleavages was allowed, and the option 'requantify' was enabled. For groups 1 - 3 (pM-vs-cM data), specific parameters were the same as for group 0 except that enzymatic specificity and maximum missed cleavage sites were chymotrypsin+/4 missed cleavages (group 1), AspN/2 missed cleavages (group 2), and GluC_DE (cleavage after D and E)/4 missed cleavages (group 3). Data in group 4 (subcellular and complexome profiling experiments) were searched with Trypsin/P and a maximum of 3 missed cleavage sites. For group 5 (pSILAC data), specific parameters were Arg10/Lys8 and Arg6/Lys4 as heavy and medium-heavy labels, multiplicity of 3, Trypsin/P, maximum of missed cleavages of 3, and option 'requantify' enabled. Group-specific parameters for group 7 (carbonate/sonication assays) were DimethLys0/DimethNter0 as light and DimethLys6/DimethNter6 as heavy labels, multiplicity of 2, Trypsin/P, maximum of 3 missed cleavages, and 'requantify' enabled. Proteins were identified based on ��������� 1 unique peptide with a length of ��������� 6 amino acids across all experiments. The options 'match between runs' (defined in a way that allowed for matching within parameter groups only) and 'iBAQ' were enabled. A peptide spectrum match false discovery rate (FDR) of 1% was applied using the decoy mode 'Revert'. An FDR of 1% was further applied to the list of proteins identified. Relative protein quantification in SILAC and peptide stable isotope dimethyl-labeling experiments was based on unique peptides and a minimum ratio count of 1. Information about downstream bioinformatics data analysis is available upon request.

### Publication Abstract
None

### Keywords
Importomics, Mitochondria, Complexome profiling, Spatial proteomics, Proteomics, Rnai-mediated protein knockdown, Silac, Pulsed silac, Subtractive proteomics

### Affiliations
Department of Biochemistry and Functional Proteomics, Institute of Biology II, Faculty of Biology, University of Freiburg, 79104 Freiburg im Breisgau, Germany
AG Warscheid
Biologie II
Albert-Ludwigs-Universität Freiburg
Schänzlestr. 1
79104 Freiburg
Germany


### Submitter
Friedel Drepper

### Lab Head
Dr Bettina Warscheid
Department of Biochemistry and Functional Proteomics, Institute of Biology II, Faculty of Biology, University of Freiburg, 79104 Freiburg im Breisgau, Germany


