### Accession
PXD013590

### Title
SILAC quantification of proteins in the control and UFBP1-expressing MGC803 cells

### Description
In this project, we used SILAC quantitative proteomics to identify proteins that are differentially regulated by UFM1-binding and PCI domain-containing protein 1 (UFBP1, also called DDRGK domain-containing protein 1 or DDRGK1) in a gastric cancer cell line. The aim of this study is to discover the proteins that are responsible for the biological function of UFBP1.

### Sample Protocol
Sample processing protocol Construction of stable cell lines: Lentiviral particles expressing pLVX-IRES-zsGreen1 and pLVX-UFBP1-FLAG-IRES-zsGreen1 were used to infect MGC803 cells. The cells were sorted out by flow cytometry to obtain the control or UFBP1-expressing cells.  SILAC labeling: The control cells (MGC803-pLVX) were grown in light medium (containing light lysine K0: 12C6, 14N2 and light arginine R0: 12C6, 14N4) and the UFBP1-expressing cells (MGC803-UFBP1) were grown in heavy medium (containing heavy lysine K8: 13C6, 15N2 and heavy arginine R10: 13C6, 15N4). Cells were cultured for more than six passages to ensure the complete incorporation of heavy arginine and lysine.  In-solution digestion and fractionation: Same number of cells from control and experimental samples were mixed, lysed, and digested with Lys-C and trypsin after reduction and alkylation. The resulting peptide samples were fractionated with a C18 column and each fraction was concentrated in a vacuum centrifugal concentrator (CentriVap, Labconco, USA), desalted with C18 ZipTip (Millipore, USA), concentrated again, and dissolved in 0.1% formic acid.  LC-MS/MS analysis: LC-MS/MS analysis was performed on an LTQ-Orbitrap Elite hybrid mass spectrometer. Purified peptides in buffer A (0.1% formic acid) were loaded to the enrichment column and then separated in an analytical column with a 150 min linear gradient of buffer B (80% acetonitrile and 0.1% formic acid) at a flow rate of 200 nL/min. Mass spectra were acquired in a positive-ion mode with an automatic Xcalibur data system. The experiment was performed for three biological replicates.

### Data Protocol
Raw MS data were searched using MaxQuant (Version 1.6.1.1) against a human protein UniProt database (released in July 2017). The precursor and fragment ion mass tolerances were set to 10 ppm and 0.6 Da, respectively. Cysteine carbamidomethylation was set as fixed modification while methionine oxidation and heavy arginine and lysine (Arg10 and Lys 8) were set as variable modification. The maximum of two trypsin missed cleavages were allowed and only peptides with â‰¥ 7 amino acids were included for the identification. The false discovery rate (FDR) for peptides was set to 1%, which was determined by searching a reversed decoy database. Relative protein abundance H/L ratio was obtained from the intensities of heavy peptides to light peptides from the experimental and control samples.

### Publication Abstract
The key component in the UFM1 conjugation system, UFM1-binding and PCI domain-containing protein 1 (UFBP1), regulates many biological processes. Recently it has been shown that low UFBP1 protein level is associated with the worse outcome of gastric cancer patients. However, how it responses to the sensitivity of gastric cancer to chemotherapy drugs and the underlying molecular mechanism remain elusive. Here, we discovered that high UFBP1 expression increases the progression-free survival of advanced gastric cancer patients treated with platinum-based chemotherapy. Cell-line based studies unveiled that UFBP1 expression enhances while UFBP1 knockdown attenuates the sensitivity of gastric cancer cells to cisplatin. High-throughput SILAC-based quantitative proteomic analysis revealed that the protein level of aldo-keto reductase 1Cs (AKR1Cs) is significantly downregulated by UFBP1. Flow cytometry analysis showed that UFBP1 expression increases while UFBP1 knockdown reduces reactive oxygen species upon cisplatin treatment. We further disclosed that UFBP1 attenuates the gene expression of AKR1Cs and the transcription activity of the master oxidative stress-response transcription factor Nrf2 (nuclear factor erythroid-2-related factor 2). Detailed mechanistic studies manifested that UFBP1 promotes the formation of K48-linked polyubiquitin chains on Nrf2 and thus augments its proteasome-mediated degradation. Experiments using genetic depletion and pharmacological activation in vitro and in vivo demonstrated that UFBP1 enhances the sensitivity of gastric cancer cells to cisplatin through the Nrf2/AKR1C axis. Overall, this work discovered a novel prognostic biomarker for gastric cancer patients treated with platinum-based chemotherapy and elucidated the underlying molecular mechanism, which may benefit to future personalized chemotherapy.

### Keywords
Human, Silac quantification, Orbitrap elite, Lc-ms/ms, Mgc803, Ufbp1

### Affiliations
College of Pharmaceutical Sciences,Soochow University,China
Soochow University

### Submitter
Dan Li

### Lab Head
Dr Guoqiang Xu
College of Pharmaceutical Sciences,Soochow University,China


