### Accession
PXD042441

### Title
Molecular and clinical analyses of PHF6 mutant myeloid neoplasia provide clues as to their pathogenesis and therapeutic targeting

### Description
PHF6 mutations (PHF6MT) are identified in various myeloid neoplasms (MN). However, little is known about the precise function and consequences of PHF6 in MN. Our comprehensive genomic analyses focused on three main findings. Firstly, we revealed a different pattern of genes correlating with PHF6MT in male and female cases. When analyzing male and female cases separately, in only male cases, RUNX1 and U2AF1 were co-mutated with PHF6. In contrast, female cases revealed co-occurrence of ASXL1 mutations and X-chromosome deletions with PHF6MT. Next, proteomics analysis revealed a direct interaction between PHF6 and the pioneer transcription factor RUNX1. Both proteins co-localize in active enhancer regions that define the context of lineage differentiation. Finally, we demonstrated a negative prognostic role of PHF6MT, especially in association with RUNX1. The negative effects on survival were additive as PHF6MT cases with RUNX1 mutations had worse outcomes when compared to cases carrying single mutation or wild-type.

### Sample Protocol
Anti-PHF6 and isotype antibody immunoprecipitation products were subjected to SDS-polyacrylamide gel electrophoresis and stained with colloidal Coomassie Blue (Gel Code Blue, Pierce Chemical). 8 Gel slices were excised from the top to the bottom of the lane for each sample; proteins were reduced with dithiothreitol (Sigma-Aldrich, D0632, 10 mM), alkylated with iodoacetamide (Sigma-Aldrich, I1149, 55 mM), and digested in situ with trypsin. Peptides were extracted from gel pieces 3 times using 60% acetonitrile and 0.1% formic acid/water. The dried tryptic peptide mixture was redissolved in 20 Î¼L of 1% formic acid for mass spectrometric analysis. Tryptic peptide mixtures were analyzed by on-line LC-coupled tandem mass spectrometry (LC-MS/MS) on an Orbitrap mass spectrometer (Theomo fisher).

### Data Protocol
Initial protein identifications from MS/MS data used Proteome Discovery and Uniprot human protein database containing 85,299 protein sequences. The Swiss Protein database search parameters included 2 missed tryptic cleavage sites allowed, precursor ion mass tolerance of 10 ppm, fragment ion mass tolerance of 0.6 Da, protein modifications for Met oxidation, and Cys carbamidomethylation. Decoy database search was included for false positive estimation. A minimum Xcorr value of 2 and peptide rank 1 were used for automatically accepting peptide MS/MS spectra. Proteins with only one unique peptide sequence (single hits) were also eliminated.

### Publication Abstract
None

### Keywords
Human, Aml, Lc-msms, Phf6

### Affiliations
Cleveland Clinic
Department of Translational Hematology and Oncology Research, Taussig Cancer Institute, Cleveland Clinic, Cleveland, OH, USA

### Submitter
Xiaorong Gu

### Lab Head
Dr Jaroslaw P. Maciejewski
Department of Translational Hematology and Oncology Research, Taussig Cancer Institute, Cleveland Clinic, Cleveland, OH, USA


