### Accession
PXD012513

### Title
Proteomic analysis of Lactobacillus nagelii in the presence of Saccharomyces cerevisiae isolated from water kefir and comparison with Lactobacilus hordei

### Description
Water kefir is a slightly alcoholic and traditionally fermented beverage, which is prepared from sucrose, water, kefir grains, and dried or fresh fruits (e.g. figs). Lactobacillus (L.) nagelii, L. hordei and Saccharomyces (S.) cerevisiae are predominant and stable lactic acid bacteria and yeasts, respectively, isolated from water kefir consortia. The growth of L. nagelii and L. hordei are improved in the presence of S. cerevisiae. In this work we demonstrate that quantitative comparative proteomics enables the investigation of interactions between LAB and yeast to predict real-time metabolic exchange in water kefir. It revealed 73 differentially expressed (DE) in L. nagelii TMW 1.1827 in the presence of S. cerevisiae. The presence of the yeast induced changes in the changes in the carbohydrate metabolism of L. nagelii and affected reactions involved in NAD+/NADH homeostasis. Furthermore, the DE enzymes involved in amino acid biosynthesis or catabolism predict that S. cerevisiae releases glutamine, histidine, methionine and arginine, which are subsequently used by L. nagelii to ensure its survival in the water kefir consortium. In co-culture with S. cerevisiae, L. nagelii profits from riboflavin, most likely secreted by the yeast. The reaction of L. nagelii to the presence of S. cerevisiae differs from that one of the previously studied L. hordei, which displays 233 differentially expressed proteins, changes in citrate metabolism and an antidromic strategy for NAD+/NADH homeostasis. So far, aggregation promotion factors, i.e. formation of a specific glucan and bifunctional enzymes were only detected in L. hordei.

### Sample Protocol
Co-cultivated L. nagelii and S. cerevisiae, as well as single-cultured L. nagelii and S. cerevisiae were incubated anaerobically in WKM at 30 C for 10 h in triplicate and prepared for proteomic analysis, as previously described (Xu et al., 2019a). First of all, these samples were treated with trichloroacetic acid (TCA, 6. 25 % w/v), centrifuged (5000 rpm, 5 min) at 4°C, washed with acetone and reconstituted in lysis buffer (8 M urea, 5 mM EDTA di-sodium salt, 100 mM (NH)4HCO3, 1 mM dithiothreitol (DDT)).Subsequently, the cells were mechanically disrupted with acid-washed glass beads (G8772, 425-600 um, Sigma, Germany). Proteins were reduced with 10 mM DTT at 30 °C for 30 min, and subsequently carbamidomethylated with 55 mM chloroacetamide in the dark for 60 min. Finally, proteins were digested by trypsin and desalted by C18 solid phase extraction using Sep-Pak columns (Waters, WAT054960). Purified peptide samples were dried in a SpeedVac concentrator (Acid-Resistant CentriVap Vacuum Concentrator, Labconco) and resuspended in an aqueous solution containing 2 % acetonitrile and 0.1 % formic acid to a final concentration of 0.25 µg/µl.  Generated peptides were analyzed on a Dionex Ultimate 3000 nano LC system, coupled to a Q-Exactive HF mass spectrometer (Thermo Scientific, Bremen, Germany), as described previously (Xu et al., 2019b). Peptides were delivered to a trap column (75 µm × 2 cm, self-packed with Reprosil-Pur C18 ODS-3 5 µm resin, Dr. Maisch, Ammerbuch, Germany) at a flow rate of 5 µl/min in solvent A0 (0.1 % formic acid in water). Peptides were separated on an analytical column (75 µm × 40 cm, self-packed with Reprosil-Gold C18, 3 µm resin, Dr. Maisch, Ammerbuch, Germany), using a 120 min linear gradient from 4-32 % solvent B (0.1% formic acid, 5% DMSO in acetonitrile) and solvent A1 (0.1 % formic acid, 5 % DMSO in water) at a flow rate of 300 nl/min. The mass spectrometer was operated in data dependent mode, automatically switching between MS1 and MS2 spectra. MS1 spectra were acquired over a mass-to-charge (m/z) range of 360-1300 m/z at a resolution of 60,000 (at m/z 200) using a maximum injection time of 50 ms and an AGC target value of 3e6. Up to 20 peptide precursors were isolated (isolation window 1.7 m/z, maximum injection time 25 ms, AGC value 1e5), fragmented by higher-energy collisional dissociation (HCD), using 25 % normalized collision energy (Letort et al.) and analyzed at a resolution of 15,000 with a scan range from 200 to 2000 m/z.

### Data Protocol
To enable differentiation of L. nagelii and S. cerevisiae proteins and their identification, peptide and protein identification plus quantification were performed with MaxQuant (version 1.5.7.4) by searching the MS2 data against all protein sequences obtained from UniProt - reference proteome S. cerevisiae S288c (6,724 entries, downloaded 13.03.2017) and all protein sequences from L. nagelii TMW 1.1827 (cf. section 3.1, GenBank CP018176 - CP018179), using the embedded search engine Andromeda (Cox et al., 2011), as previously described (Xu et al., 2019a). Carbamidomethylated cysteine was a fixed modification. Oxidation of methionine, and N-terminal protein acetylation were variable modifications. Precursor and fragment ion tolerances were 10 ppm and 20 ppm, respectively. Label-free quantification and data matching between consecutive analyses were enabled within MaxQuant. Search results were filtered for a minimum peptide length of 7 amino acids, 1 % peptide and protein false discovery rate (FDR) plus common contaminants and reverse identifications. MaxQuant output files were further analyzed using Perseus (version 1.5.6.0) (Tyanova et al., 2016). iBAQ intensities were log2-transformed for further statistical analysis. NCBI annotation, PSORTb subcellular localization, SEED category (subcategory and subsystem) as previously annotated (cf. section 2.1) were added to the matrix through identifier matching. For the comparison between two groups, t-tests were performed. Log2 fold change ≥ 2 or ≤ -2 and -Log10 P-value ≥ 2 (p value ≤ 0.05) were considered to be significantly differentially expressed proteins of L. nagelii TMW 1.1827 in the presence of S. cerevisiae TMW 3.221.

### Publication Abstract
Water kefir is a slightly alcoholic and traditionally fermented beverage, which is prepared from sucrose, water, kefir grains, and dried or fresh fruits (e.g., figs). <i>Lactobacillus</i> (<i>L.</i>) <i>nagelii, L. hordei</i>, and <i>Saccharomyces</i> (<i>S.</i>) <i>cerevisiae</i> are predominant and stable lactic acid bacteria and yeasts, respectively, isolated from water kefir consortia. The growth of <i>L. nagelii</i> and <i>L. hordei</i> are improved in the presence of <i>S. cerevisiae</i>. In this work we demonstrate that quantitative comparative proteomics enables the investigation of interactions between LAB and yeast to predict real-time metabolic exchange in water kefir. It revealed 73 differentially expressed (DE) in <i>L. nagelii</i> TMW 1.1827 in the presence of <i>S. cerevisiae</i>. The presence of the yeast induced changes in the changes in the carbohydrate metabolism of <i>L. nagelii</i> and affected reactions involved in NAD<sup>+</sup>/NADH homeostasis. Furthermore, the DE enzymes involved in amino acid biosynthesis or catabolism predict that <i>S. cerevisiae</i> releases glutamine, histidine, methionine, and arginine, which are subsequently used by <i>L. nagelii</i> to ensure its survival in the water kefir consortium. In co-culture with <i>S. cerevisiae, L. nagelii</i> profits from riboflavin, most likely secreted by the yeast. The reaction of <i>L. nagelii</i> to the presence of <i>S. cerevisiae</i> differs from that one of the previously studied <i>L. hordei</i>, which displays 233 differentially expressed proteins, changes in citrate metabolism and an antidromic strategy for NAD<sup>+</sup>/NADH homeostasis. So far, aggregation promotion factors, i.e., formation of a specific glucan and bifunctional enzymes were only detected in <i>L. hordei</i>.

### Keywords
Functional genome prediction, Lactobacillus hordei, Proteomic analysis, Lactobacillus nagelii, Metabolism

### Affiliations
Bavarian Center for Biomolecular Mass Spectrometry (BayBioMS) Technische Universität München (TUM) Gregor-Mendel-Straße 4 85354 Freising DEUTSCHLAND / GERMANY Tel. / Phone: +49 8161 71-6130 E-Mail: tina.ludwig@tum.de www.baybioms.tum.de
TU Munich

### Submitter
Christina Ludwig

### Lab Head
Dr Christina Ludwig
Bavarian Center for Biomolecular Mass Spectrometry (BayBioMS) Technische Universität München (TUM) Gregor-Mendel-Straße 4 85354 Freising DEUTSCHLAND / GERMANY Tel. / Phone: +49 8161 71-6130 E-Mail: tina.ludwig@tum.de www.baybioms.tum.de


