### Accession
PXD043659

### Title
Exoproteome of a furC-overexpressing strain: FurC tunes peptidoglycan remodeling and intercell transfer

### Description
To study the effect of FurC overexpression in the exoproteome composition of Anabaena, sp. PCC120, SWATH-MS analyses have been carried out with the isolated exoproteome of the FurC overexpressing strain, EB2770FurC (named in the present dataset "EB") comparing to the one of the wild-type strain (WT). This data togheter combined with physiological and functional studies of FurC protein revealed a potential role of this regulator in the modulation of general cell-wall biogenesis and recycling processes, as well as and in intercellular transfer.

### Sample Protocol
Cultures of two biological replicates of wild type (WT) Anabaena and the furC-overexpressing strain were started from plate in 40 mL of BG11C. When cultures reached the late exponential phase of growth, cells were harvested by gentle centrifugation at room temperature. These cells were used to set up 80 mL cultures of each replicate at an OD750 of 0.15 in BG11C. Cultures were grown until they reached late-log phase (12 days, OD750=1-1.5), after which cultures were sequentially filtered through 0.45 and 0.22 Ã‚Âµm with MCE filters (Filter-LabÃ‚Â®). Samples were immediately frozen at -80 Ã‹Å¡C and lyophilized. The lyophilized material was resuspended in 2.5 mL of buffer A (50 mM Tris-HCl pH 7.5 and CompleteÃ¢â€žÂ¢ EDTA free protease inhibitor cocktail) and dialyzed against 3 L of buffer A, performing 2 O/N steps. After dialysis, the sample was transferred to 15 mL tubes, frozen at -80 Ã‹Å¡C and lyophilized. Samples were resuspended in 800 Ã‚ÂµL of buffer containing 50 mM Tris-HCl pH 7.5; 1 mM PMSF, 1 mM EDTA, 2 % SDS and a cocktail of protease inhibitors and were centrifuged at 20000 g for 2 min to eliminate particles. The supernatants were subjected to precipitation with trichloroacetic acid (TCA)/acetone. Subsequently, sample digestion was performed as follows; the precipitated extracts were resuspended in 20 Ã‚ÂµL of 50 mM ammonium bicarbonate with 0.2 % Rapidgest (Waters) and the amount of proteins was quantified using Qubit system. 10 Ã‚Âµg of total protein was incubated with 4.5 mM DTT for 30 min at 60 Ã‹Å¡C and then with 10 mM chloroacetamide, 30 min in darkness at room temperature. Trypsin treatment was performed at 37 Ã‹Å¡C O/N in a ratio 1:40 (Trypsin:Protein). Digestion was stopped by addition of formic acid. A mixture of SCIEX synthetic peptides was also added to each sample to a final concentration of 50 femtomole/Ã‚ÂµL, for posterior normalization of chromatograms necessary for SWATH processing.

### Data Protocol
SWATH-MS (Sequential Window Acquisition of all Theoretical Mass Spectra)  Mass spectrometry analyses were carried out at the Proteomics Service of the Instituto de BioquÃƒÂ­mica Vegetal y FotosÃƒÂ­ntesis (Seville, Spain). The analyses were performed in a triple quadrupole TOF hybrid mass spectrometer (5600 plus, Sciex), equipped with a nano electrospray source coupled to an Eksigent model 425 nanoHPLC. Analyst TF 1.7 was the software used to control the equipment, as well as for the acquisition and processing of data. A volume equivalent to 1 Ã‚Âµg of protein was injected in LC-MS/MS. Peptides were first loaded onto a trap column (Acclaim PepMap 100 C18, 5 Ã‚Âµm, 100 Ãƒâ€¦, 100 Ã‚Âµm id Ãƒâ€” 20 mm, Thermo Fisher Scientific) isocratically in 0.1 % formic acid/2 % acetonitrile (v/v) at a flow rate of 3 ÃŽÂ¼L/min for 10 min. Subsequently, they were eluted on a reverse phase analytical column, Acclaim PepMap 100 C18, 3 Ã‚Âµm, 100 Ãƒâ€¦, 75 Ã‚Âµm id Ãƒâ€” 150 mm, Thermo Fisher Scientific, coupled to a PicoTip emitter (F360-20-10-N- 20_C12 from New Objective). Peptides were eluted with a linear gradient of 2-35 % (v/v) of solvent B in 60 min at a flow rate of 300 nL/min. As solvents A and B, formic acid 0.1% (v/v) and acetonitrile with formic acid 0.1 % (v/v) were used, respectively. The voltage source was set at 2600 V and the heater temperature remained at 100 Ã‹Å¡C. Gas 1 was selected at 15 psi, gas 2 to zero, and the curtain to 25 psi. For the protein identification necessary for the creation of spectral ion libraries for SWATH-MS quantification, the acquisition was carried out with a DDA method (data dependent acquisition), consisting of a TOF-MS Scan between 400-1250 m/z, accumulation time of 250 ms, followed by 50 MS/MS (230 -1500 m/z), accumulation time of 65 ms and with a total 3,54 s cycle time. Protein identification was performed using the ProteinPilotÃ¢â€žÂ¢ software (version 5.0.1, Sciex) with the ParagonÃ¢â€žÂ¢ Algorithm. The search was conducted against the UniProt proteome Nostoc sp. PCC7120 FASTA (15/07/2021) combined with the SCIEX contaminants database.  Automatically generated reports in ProteinPilot were manually inspected for false discovery rate (FDR); cut-off proteins with only proteins identified at an FDR Ã¢â€°Â¤1% were considered for subsequent analyses. SWATH experiments were performed using three technical replicates of each biological replicate (1 Ã‚Âµg protein) using a DIA method (data-independent acquisition). The method consisted of repeated acquisition cycles of time-of-flight mass spectrometry TOF MS/MS scans (230Ã¢â‚¬â€œ1,500Ã¢â‚¬â€°m/z, 60Ã¢â‚¬â€°ms acquisition time) of 60 overlapping sequential precursor isolation windows of variable width (1Ã¢â‚¬â€°m/z overlap) covering the 400Ã¢â‚¬â€œ1,250Ã¢â‚¬â€°m/z mass range from a previous TOF MS scan (400Ã¢â‚¬â€œ1,250Ã¢â‚¬â€°m/z, 50Ã¢â‚¬â€°ms acquisition time) for each cycle with 3.68 s total cycle time. Autocalibration of the equipment and chromatographic conditions were controlled by an injection of a standard Pepcalmix (Sciex), between the replicates. SWATH data processing and statistical analysis SWATH-MS spectral alignment was performed using the PeakView 2.2 software (Sciex) with the MicroApp SWATH 2.0 using the spectral library generated through DDA method (1% FDR). Normalization of protein intensity was performed with MarkerView software (version 1.2.1.1, AB Sciex). The R package limma v3.54.1 (67) was used to assess the significant differences in abundance of exoproteins between EB2770FurC and Anabaena sp. PCC7120. Briefly, normalized data were log2-transformed, median centered and the three technical replicates averaged before fitting the data to a linear model (lmFit, robust method), and using moderated t-statistics with Limma's empirical Bayes method to analyze differential expression. The p-values were adjusted for multiple hypotheses testing using Benjamini-Hochberg FDR correction. Proteins were classified as differentially abundant proteins (DAPs) if the adjusted p-value (q-value) was less than 0.001.

### Publication Abstract
Microbial extracellular proteins and metabolites provide valuable information concerning how microbes adapt to changing environments. In cyanobacteria, dynamic acclimation strategies involve a variety of regulatory mechanisms, being ferric uptake regulator proteins as key players in this process. In the nitrogen-fixing cyanobacterium <i>Anabaena</i> sp. strain PCC 7120, FurC (PerR) is a global regulator that modulates the peroxide response and several genes involved in photosynthesis and nitrogen metabolism. To investigate the possible role of FurC in shaping the extracellular environment of <i>Anabaena</i>, the analysis of the extracellular metabolites and proteins of a <i>furC</i>-overexpressing variant was compared to that of the wild-type strain. There were 96 differentially abundant proteins, 78 of which were found for the first time in the extracellular fraction of <i>Anabaena</i>. While these proteins belong to different functional categories, most of them are predicted to be secreted or have a peripheral location. Several stress-related proteins, including PrxA, flavodoxin, and the Dps homolog All1173, accumulated in the exoproteome of <i>furC</i>-overexpressing cells, while decreased levels of FurA and a subset of membrane proteins, including several export proteins and <i>amiC</i> gene products, responsible for nanopore formation, were detected. Direct repression by FurC of some of those genes, including <i>amiC1</i> and <i>amiC2,</i> could account for odd septal nanopore formation and impaired intercellular molecular transfer observed in the <i>furC</i>-overexpressing variant. Assessment of the exometabolome from both strains revealed the release of two peptidoglycan fragments in <i>furC</i>-overexpressing cells, namely 1,6-anhydro-N-acetyl-&#x3b2;-D-muramic acid (anhydroMurNAc) and its associated disaccharide (&#x3b2;-D-GlcNAc-(1-4)-anhydroMurNAc), suggesting alterations in peptidoglycan breakdown and recycling.IMPORTANCECyanobacteria are ubiquitous photosynthetic prokaryotes that can adapt to environmental stresses by modulating their extracellular contents. Measurements of the organization and composition of the extracellular milieu provide useful information about cyanobacterial adaptive processes, which can potentially lead to biomimetic approaches to stabilizing biological systems to adverse conditions. <i>Anabaena</i> sp. strain PCC 7120 is a multicellular, nitrogen-fixing cyanobacterium whose intercellular molecular exchange is mediated by septal junctions that traverse the septal peptidoglycan through nanopores. FurC (PerR) is an essential transcriptional regulator in <i>Anabaena</i>, which modulates the response to several stresses. Here, we show that <i>furC</i>-overexpressing cells result in a modified exoproteome and the release of peptidoglycan fragments. Phenotypically, important alterations in nanopore formation and cell-to-cell communication were observed. Our results expand the roles of FurC to the modulation of cell-wall biogenesis and recycling, as well as in intercellular molecular transfer.

### Keywords
Perr, Furc, Peptidoglycan, Swath, Transfer

### Affiliations
Department of Biochemistry and Molecular and Cell Biology. University of Zaragoza and BIFI
University of Zaragoza

### Submitter
Cristina Sarasa Buisán

### Lab Head
Dr María F. Fillat
Department of Biochemistry and Molecular and Cell Biology. University of Zaragoza and BIFI


