### Accession
PXD007795

### Title
olfactory proteomics in aged Tg2576 mice

### Description
Olfaction is often deregulated in Alzheimer´s disease (AD) patients, being also impaired in transgenic Tg2576 AD mouse model, which overexpress the Swedish mutated form of human amyloid precursor protein (APP). However, little is known about the molecular mechanisms that accompany the neurodegeneration of olfactory structures in Tg2576 mice. For that, we have applied proteome- and transcriptome-wide approaches to probe molecular disturbances in the olfactory bulb (OB) dissected from aged Tg2576 mice (18 months of age) respect to age matched wild-type (WT) littermates

### Sample Protocol
Sample preparation for proteomic analysis – Murine OB specimens were homogenized in lysis buffer containing 7 M urea, 2 M thiourea, 50 mM DTT. The homogenates were spinned down at 100.000 x g for 1 h at 15ºC. Protein concentration was measured in the supernatants with the Bradford assay kit (Bio-rad). Protein Digestion and Peptide iTRAQ Labeling. A shotgun comparative proteomic analysis of OB proteomes using iTRAQ (isobaric Tags for Relative and Absolute Quantitation) was performed. Protein extracts were precipitated with Clean-up kit (Bio-Rad), and pellets dissolved in 7M urea, 2 M thiourea. Protein quantitation was performed with the Bradford assay kit (Bio-Rad). iTRAQ labeling of each sample was performed according to the manufacturer's protocol (Sciex). Briefly, equal amounts of OB proteins (50 μg) from each sample were reduced with 50 mM tris (2-carboxyethyl) phosphine (TCEP) at 60 °C for 1 h. Cysteine residues were alkylated with 200 mM methylmethanethiosulfonate (MMTS) at room temperature for 15 min. Protein enzymatic cleavage was carried out with trypsin (Promega; 1:20, w/w) at 37 °C for 16 h. Each tryptic digest was labelled according to the manufacturer's instructions with one isobaric amine-reactive tags as follows: Tag113, WT-1; Tag114, WT-2; Tag115, WT-3; Tag116, Tg2576-1; Tag117, Tg2576-2; Tag118, tg2576-3. After 2h incubation, the set of labelled samples were pooled and evaporated in a vacuum centrifuge.  Liquid Chromatography tandem mass Spectrometry and data Analysis. To increase the proteome coverage, the peptide pool was submitted to cation exchange chromatography using spin Columns (Pierce). 12 fractions were collected (from 10mM to 150 mM KCl), evaporated under vacuum and reconstituted into10 μl of 2% acetonitrile, 0.1% formic acid, 98% MilliQ-H20 prior to mass spectrometric analysis. Peptide mixtures were separated by reverse phase chromatography using an Eksigent nanoLC ultra 2D pump fitted with a 75 μm ID column (Eksigent 0.075 x 250). Samples were first loaded for desalting and concentration into a 0.5 cm length 100 μm ID precolumn packed with the same chemistry as the separating column. Mobile phases were 100% water 0.1% formic acid (FA) (buffer A) and 100% Acetonitrile 0.1% FA (buffer B). Column gradient was developed in a 146 min two step gradient from 5% B to 25% B in 120 min and 25%B to 70% B in 15 min. Column was equilibrated in 95% B for 9 min and 5% B for 14 min. During all process, precolumn was in line with column and flow maintained all along the gradient at 300 nl/min. Eluting peptides from the column were analyzed using an Sciex 5600 Triple-TOF system. Information data acquisition was acquired upon a survey scan performed in a mass range from 350 m/z up to 1250 m/z in a scan time of 250 ms. Top 25 peaks were selected for fragmentation. Minimum accumulation time for MS/MS was set to 100 ms giving a total cycle time of 2.75 s. Product ions were scanned in a mass range from 100 m/z up to 1500 m/z and excluded for further fragmentation during 15 s.

### Data Protocol
Data analysis. The raw MS/MS spectra search were processed using the freely available MaxQuant software (v.1.5.8.3) [20]. Initial maximum precursor and fragment mass deviations were set to 25 ppm and 0.5 Da, respectively. Variable modification (methionine oxidation and N-terminal acetylation) and fixed modification (MMTS) were set for the search and trypsin with a maximum of 1 missed cleavages was chosen for searching. The minimum peptide length was set to 7 amino acids and the false discovery rate (FDR) for PSM and protein identification was set to 1%. Matching against the protein sequence database was carried out after improving the precursor ion mass accuracy using the mass-dependent recalibration option of the software. The false-discovery rate was controlled at various levels by using a target-decoy search strategy, which integrates multiple peptide parameters such as length, charge, number of modifications and the identification score into a single quality that acts as the statistical evidence on the quality of each single peptide spectrum match. The frequently observed laboratory contaminants were removed and the protein identification was considered valid only when at least one unique or “razor” peptide was present. Following protein identification, the intensity for each identified protein was calculated using ion reporter intensities. The protein quantification was calculated using at least 2 razor + unique peptides, and calculation of statistical significance was carried out using two-way Student-t test (p < 0.05). A 1.3-fold change cutoff for all iTRAQ ratios (ratio ≤0.77 or ≥1.3) was selected to classify proteins as up- or down-regulated. Proteins with iTRAQ ratios below the low range (0.77) were considered to be underexpressed, whereas those above the high range (1.3) were considered to be overexpressed. The freely available software Perseus (version 1.5.6.0) [21] was used for statistical analysis and data visualization

### Publication Abstract
Olfaction is often deregulated in Alzheimer's disease (AD) patients, and is also impaired in transgenic Tg2576 AD mice, which overexpress the Swedish mutated form of human amyloid precursor protein (APP). However, little is known about the molecular mechanisms that accompany the neurodegeneration of olfactory structures in aged Tg2576 mice. For that, we have applied proteome- and transcriptome-wide approaches to probe molecular disturbances in the olfactory bulb (OB) dissected from aged Tg2576 mice (18 months of age) as compared to those of age matched wild-type (WT) littermates. Some over-represented biological functions were directly relevant to neuronal homeostasis and processes of learning, cognition, and behavior. In addition to the modulation of CAMP responsive element binding protein 1 (CREB1) and APP interactomes, an imbalance in the functionality of the I&#x3ba;B&#x3b1;-NF&#x3ba;B p65 complex was observed during the aging process in the OB of Tg2576 mice. At two months of age, the phosphorylated isoforms of olfactory I&#x3ba;B&#x3b1; and NF&#x3ba;B p65 were inversely regulated in transgenic mice. However, both phosphorylated proteins were increased at 6 months of age, while a specific drop in I&#x3ba;B&#x3b1; levels was detected in 18-month-old Tg2576 mice, suggesting a transient activation of NF&#x3ba;B in the OB of Tg2576 mice. Taken together, our data provide a metabolic map of olfactory alterations in aged Tg2576 mice, reflecting the progressive effect of APP overproduction and &#x3b2;-amyloid (A&#x3b2;) accumulation on the OB homeostasis in aged stages.

### Keywords
Neurodegeneration, Proteomics, Olfactory bulb, Alzheimer

### Affiliations
Proteomics unit
Proteomics Unit, Navarrabiomed-proteored (Spain)

### Submitter
Enrique SantamarÃ­a

### Lab Head
Dr Enrique Santamaria
Proteomics Unit, Navarrabiomed-proteored (Spain)


