### Accession
PXD011446

### Title
proteome analysis of PSP and FTLD subjects

### Description
A mild olfactory dysfunction has been observed in frontotemporal dementias (FTD). However, the underlying molecular mechanisms associated to this deficit are poorly understood. Here, we applied mass spectrometry-based quantitative proteomics to analyse pathological effects on the olfactory bulb (OB) from progressive supranuclear palsy (PSP) and frontotemporal lobar degeneration TDP43 proteinopathy (FTLD-TDP) subjects

### Sample Protocol
Sample preparation for proteomic analysis - OB specimens derived from control, FTLD-TDP and PSP cases were homogenized using ‘’mini potters’’ in lysis buffer containing 8 M urea, 0,1 M TEAB, 1mM DTT, 10% ACN, pH=8. The homogenates were spanned down at 14.000 x g for 15min at 4°C. Protein quantitation was performed with the Bradford assay kit (Bio-Rad). Reduction, alkylation, digestion and TMT labeling – For the proteomic analysis, 25 µg of protein from each sample were used. Protein were first reduced with 10mM TCEP for 1h at 30ºC.  Then, alkylation was performed for 30 min at room temperature using 40mM IAA. Samples were diluted 5 times to reduce the Urea concentration <1M and protein enzymatic cleavage was carried out overnight with trypsin (Promega; 1:50, w/w) at 37°C. Subsequently, each of the samples was labeled with one of the TMT reagents according to the manufacturer’s instructions (Thermo Scientific). Two TMT experiments were performed, one for each disease analyzed. Thereby, control cases were labeled with 127N, 127C, 128N, and 128C and disease cases were labeled with 129N, 129C, 130N and 130C. Labels 126 and 131 were used to label two pools of all samples as intern quality control of the procedure. After leaving the labeling reaction for 60min, all samples were pooled and dried in a speed-vacuum.  Off-gel electrophoresis – samples previously purified with C18 Macro Spin Columns (Harvard Apparatus) were fractionated using Off-gel electrophoresis (OGE) separation using Agilent 3100 off-gel fractionator, according to the manufacturer’s instructions. Dried samples were reconstituted using OGE solution and focused using an immovilized pH gradient dry strip (13cm, pH 3-10). Samples were then desalted and purified using MicroSpins C18 columns, dried in a speed-vacuum and stored at -20ºC until analysis.  LC-MS/MS – A LTQ Orbitrap Q-exactive Plus mass spectrometer (Thermo Fisher) coupled with nanoflow high-pressure liquid chromatography (HPLC) was used to analyzed the OGE fractions, as previously described [25]. Peptides were reconstituted using 5%ACN 0,1%FA were trapped on 2 cm x 75 µm ID, 3 µm pre-column and separated on an Easy-spray column, 50 cm x 75 µm ID, PepMap C18, 2 µm (Thermo Scientific). The analytical separation was run for 60 min using a gradient of H2O/FA 99.9%/0.1% (solvent A) and CH3CN/FA 99.9%/0.1% (solvent B) at a flow rate of 300 nL/min. For MS survey scans, the OT resolution was set to 140’000 and the ion population was set to 3 × 106 with an m/z window from 350 to 2000. Twenty precursor ions were selected for higher-energy collisional dissociation (HCD) with a resolution of 35,000, an ion population set to 1 × 105 (isolation window of 0.5 m/z) and a normalized collision energy set to 30%.

### Data Protocol
Protein identification – MS data were processed using EasyProt. Peak list were obtained using the 12 OGE fractions and the combination of HCD-CID raw data peak list were generated. These data were submitted to the EasyProt software platform (version 2.3, build 718) that uses Phenyx software (GeneBio, Geneva, Switzerland) for the protein identification. The search was performed using Uniprot/Swiss-Prot database (2014-10, 66903) (ref EasyProt--an easy-to-use graphical platform for proteomics data analysis). The following criteria were used: Homo Sapiens taxonomy, oxidized methionine as variable modification, and cysteine carbamethylation, TMT10 lysine and TMT10 amino-terminus as fixed modifications. One missed cleavage was selected and parent-ion tolerance was set to 10ppm and the accuracy of fragment ions to 0.6Da. Only proteins with a less than 1% false discovery rate (FDR) and at least two different unique peptides were selected for further analysis. A minimum peptide length of 6 amino acids was used. Protein quantification- Isobaric quantification was performed using the Isobar R package (General statistical modeling of data from protein relative expression isobaric tags.) The manufacturer’s isotopic distribution data was used to correct the isotopic impurities of TMT10 reporter-ion intensities. The equal median intensity method was used to normalize the reporter intensities. Peptides which did not present reporter intensities were not quantified. To test the ratio’s accuracy and biological significance, technical and biological variability were calculated for each protein ratio. A ratio p value and sample p value were calculated for each variable. Furthermore, only proteins with a cut-off threshold value higher than 1.33 or lower than 0.77 were considered.

### Publication Abstract
Olfactory dysfunction is considered an early prodromal marker of many neurodegenerative diseases. Neuropathological changes and aberrant protein aggregates occur in the olfactory bulb (OB), triggering a tangled cascade of molecular events that is not completely understood across neurological disorders. This study aims to analyze commonalities and differences in the olfactory protein homeostasis across neurological backgrounds with different spectrums of smell dysfunction. For that, an integrative analysis was performed using OB proteomics datasets derived from subjects with Alzheimer's disease (AD), Parkinson's disease (PD), mixed dementia (mixD), dementia with Lewy bodies (DLB), frontotemporal lobar degeneration (FTLD-TDP43), progressive supranuclear palsy (PSP) and amyotrophic lateral sclerosis (ALS) with respect to OB proteome data from neurologically intact controls. A total of 80% of the differential expressed protein products were potentially disease-specific whereas the remaining 20% were commonly altered across two, three or four neurological phenotypes. A multi-level bioinformatic characterization revealed a subset of potential disease-specific transcription factors responsible for the downstream effects detected at the proteome level as well as specific densely connected protein complexes targeted by several neurological phenotypes. Interestingly, common or unique pathways and biofunctions were also identified, providing novel mechanistic clues about each neurological disease at olfactory level. The analysis of olfactory epithelium, olfactory tract and primary olfactory cortical proteotypes in a multi-disease format will functionally complement the OB dyshomeostasis, increasing our knowledge about the neurodegenerative process across the olfactory axis.

### Keywords
Ftld, Proteomics, Psp, Dementia

### Affiliations
Proteomics unit
Proteomics Unit, Navarrabiomed-IDISNA Proteored-ISCIII Pamplona, Spain

### Submitter
Enrique SantamarÃ­a

### Lab Head
Dr Enrique Santamaria
Proteomics Unit, Navarrabiomed-IDISNA Proteored-ISCIII Pamplona, Spain


