### Accession
PXD015783

### Title
Identification of MRCKA as an integral component of ovarian cancer signaling and a therapeutic target

### Description
High-grade serous ovarian carcinoma (HGSOC) is the most lethal gynecological cancer with few effective targeted therapies. HGSOC tumors exhibit genomic instability with frequent alterations in the protein kinome; however, only a small fraction of the kinome has been therapeutically targeted in HGSOC. Using multiplexed inhibitor beads and mass spectrometry (MIB-MS), we mapped the kinome landscape of HGSOC patient tumors and tumors isolated from HGSOC patient-derived xenograft (PDX) models.  Kinome profiling of HGSOC tumors uncovered a prevalent MIB-MS kinome signature consisting of established HGSOC driver kinases, as well as several kinases previously unexplored in HGSOC.  Loss-of-function analysis targeting the tumor kinome signature in HGSOC cells nominated CDC42BPA (also known as MRCKA) as a putative therapeutic target.  Characterization of MRCKA knockdown in established HGSOC cell lines demonstrated MRCKA was integral to signaling that regulated the cell cycle checkpoint and focal adhesion/ actin remodeling, and depletion of MRCKA impaired cell migration, proliferation and survival.  Moreover, small molecule inhibition of MRCKA using BDP9066 inhibited cell growth and induced apoptosis in HGSOC cells, as well as blocked spheroid formation, supporting MRCKA as a novel therapeutic target for the treatment of HGSOC.

### Sample Protocol
Cells were lysed on ice in buffer containing 50 mM HEPES (pH 7.5), 0.5% Triton X-100, 150 mM NaCl, 1 mM EDTA, 1 mM EGTA, 10 mM sodium fluoride, 2.5 mM sodium orthovanadate, 1X protease inhibitor cocktail (Roche), and 1% each of phosphatase inhibitor cocktails 2 and 3 (Sigma). Particulate was removed by centrifugation of lysates at 13,000 rpm for 10 minutes at 4°C and filtration through 0. 45 µm syringe filters. Protein concentrations were determined by BCA analysis. To compare kinome signatures within and across large sample sizes, we designed a super-SILAC method consisting of a cocktail of cancer cell lines that encompasses the activity of the “cancer kinome” to be used as a control for individual samples. Five cancer cell lines were selected from the NCI-60 panel (UACC257, MOLT4, COLO205, ACHN and PC3) that differed in their origin, gene expression patterns and mutation status. The cancer cell lines were labeled using SILAC and mixed equally providing a diverse SILAC ([13C6, 15N4] arginine (Arg 10) and [ 13C6 ] lysine (Lys 8 )) heavy reference standard (s-SILAC) to be used repeatedly in kinome profiling assays. For super-SILAC experiments, an equal amount of the s-SILAC reference (5 mg) lysate was added to our non-labeled (5 mg) sample (OVCAR4 cells transfected with siNT2 or siCDC42BPA) and analyzed on MIB-beads. Endogenous kinases were isolated by flowing lysates over kinase inhibitor-conjugated Sepharose beads (purvalanol B, VI16832, PP58 and CTx-0294885) in 10 ml gravity-flow columns. After 2x10 ml column washes in high-salt buffer and 1x10 ml wash in low-salt buffer (containing 50 mM HEPES (pH 7.5), 0.5% Triton X-100, 1 mM EDTA, 1 mM EGTA, and 10 mM sodium fluoride, and 1M NaCl or 150 mM NaCl, respectively), retained kinases were eluted from the column by boiling in 2x500 µl of 0.5% SDS, 0.1 M TrisHCl (pH 6.8), and 1% 2-mercaptoethanol. Eluted peptides were reduced by incubation with 5 mM DTT at 65°C for 25 minutes, alkylated with 20 mM iodoacetamide at room temperature for 30 minutes in the dark, and alkylation was quenched with DTT for 10 minutes. Samples were concentrated to approximately 100 µl with Millipore 10kD cutoff spin concentrators. Detergent was removed by chloroform/methanol extraction, and the protein pellet was resuspended in 50 mM ammonium bicarbonate and digested with sequencing-grade modified trypsin (Promega) overnight at 37°C. Peptides were cleaned with PepClean C18 spin columns (Thermo), dried in a speed-vac, resuspended in 50 μl 0.1% formic acid, and extracted  with ethyl acetate (10:1 ethyl acetate:H2O). Briefly, 1 mL ethyl acetate was added to each sample, vortexed and centrifuged at max speed for 5 minutes, then removed. This process is repeated 4 more times. After removal of ethyl acetate following the 5th centrifugation, samples were placed at 60°C for 10 minutes to evaporate residual ethyl acetate. The peptides were dried in a speed vac, and subsequent LC-/MS/MS analysis was performed. Proteolytic peptides were resuspended in 0.1% formic acid and separated with a Thermo RSLC Ultimate 3000 on a Thermo Easy-Spray C18 PepMap 75µm x 50cm C-18 2 µm column with a 240 min gradient of 4-25% acetonitrile with 0.1% formic acid at 300 nL/min at 50°C. Eluted peptides were analyzed by a Thermo Q Exactive plus mass spectrometer utilizing a top 15 methodology in which the 15 most intense peptide precursor ions were subjected to fragmentation. The AGC for MS1 was set to 3x106 with a max injection time of 120 ms, the AGC for MS2 ions was set to 1x105 with a max injection time of 150 ms, and the dynamic exclusion was set to 90 s.

### Data Protocol
Raw data analysis of SILAC experiments was performed using Maxquant software 1.6.1.0 and searched using andromeda 1.5.6.0 against the swiss-prot human protein database (downloaded on July 26, 2018). The search was set up for full tryptic peptides with a maximum of two missed cleavage sites. All settings were default and searched using acetylation of protein N-terminus and oxidized methionine as variable modifications. Carbamidomethylation of cysteine was set as fixed modification. The precursor mass tolerance threshold was set at 10 ppm and maximum fragment mass error was 0.02 Da. SILAC quantification was performed using MaxQuant by choosing multiplicity as 2 in group-specific parameters and Arg10 and Lys8 as heavy labels. Match between runs was employed and the significance threshold of the ion score was calculated based on a false discovery rate of < 1%. MaxQuant normalized ratios were imported into Perseus software (1.6.2.3) for quantitation. MIB-MS profiles were processed in the following manner: normalized MIB-MS s-SILAC ratios were transformed 1/(x) to generate light / heavy ratios, followed by log2/(x) transformation. Rows were filtered for kinases present in 100% of samples leaving 168 kinases. No imputation of missing values was performed. Differences in s-SILAC ratios amongst cells treated with control siRNAs or MRCKA siRNAs were determined using a two-sample student’s T-test with the following parameters, (S0 0.1, and Side, Both) using Benjamini-Hochberg FDR, FDR < 0.05. A Volcano plot was then generated in RStudio 1.1.423 using ggplot2 and ggrepel libraries to visualize differences in MIB-binding amongst treatments.

### Publication Abstract
None

### Keywords
Human, Cdc42bpa, High-grade serous ovarian carcinoma, Mrcka

### Affiliations
Cancer Biology Program, Fox Chase Cancer Center, Philadelphia, PA, USA
Fox Chase Cancer Center

### Submitter
James Duncan

### Lab Head
Dr James Stuart Duncan
Cancer Biology Program, Fox Chase Cancer Center, Philadelphia, PA, USA


