### Accession
PXD006153

### Title
Metabolite Perception Regulates Colonization and Virulence during Polymicrobial Infection

### Description
Many human infections are polymicrobial in origin, and interactions among community inhabitants shape colonization patterns and pathogenic potential1.  However, few interspecies interactions have been functionally dissected at the molecular level or characterized on a systems level.  Periodontitis, which is the sixth most prevalent infectious disease worldwide2, ensues from the action of dysbiotic polymicrobial communities3.  The keystone pathogen Porphyromonas gingivalis and the accessory pathogen Streptococcus gordonii interact to form communities in vitro and exhibit increased fitness in vivo3, 4.  The mechanistic basis of this polymicrobial synergy, however, has not been fully elucidated.  Here we show that streptococcal 4 aminobenzoate/para-amino benzoic acid (pABA) is required for maximal accumulation of P. gingivalis in dual species communities.  Metabolomic and proteomic data showed that exogenous pABA is utilized for folate biosynthesis, and leads to decreased stress and elevated expression of fimbrial interspecies adhesins.  Moreover, pABA increased the colonization and survival of P. gingivalis in a murine oral infection model.  However, pABA also caused a reduction in virulence in vivo and suppressed extracellular polysaccharide production by P. gingivalis.  Collectively, these data reveal a multidimensional aspect to P. gingivalis-S. gordonii interactions and establish pABA as a critical cue produced by a partner species that enhances fitness of P. gingivalis while diminishing virulence.

### Sample Protocol
Bacteria were cultured to mid-log phase and reacted for 18 h with pABA (1 mg/mL) or vehicle control.  Two technical replicates were measured for each of four independent biological replicates. Bacterial pellets were resuspended in 50 mM Tris-HCl pH 8.0, 10 mM DTT.  2,2,2-trifluoroethanol (Sigma) was added to 50% (v/v) final concentration and samples were incubated at 56Â°C for 45 min.  Following incubation, iodoacetamide was added to a concentration of 25 mM and samples were incubated at room temperature in the dark for 30 min. Samples were diluted 10-fold with 2 mM CaCl2, 50 mM Tris-HCl, pH 8.0. Samples were digested with trypsin (Pierce) at 37Â°C for 5 h. Digestion was quenched by adding formic acid to 1% (v/v).  Tryptic peptides were filtered through Amicon Ultra 30 kD spin filtration columns and bound, washed, and eluted from HyperSep C18 SpinTips (Thermo Scientific). Eluted peptides were dried by speed-vac and resuspended in Buffer C (5% acetonitrile, 0.1% formic acid) for analysis by LC-MS/MS. For LC-MS/MS analysis, peptides were separated on an Acclaim C18 PepMap RSLC column (Dionex; Thermo Scientific, Waltham, MA) and reverse phase chromatography on a Dionex Ultimate 3000 RSLCnano UHPLC system (Thermo Scientific). Peptides were eluted using a 5-40% acetonitrile gradient over 90 min at 300 nl/min flow rate. Eluted peptides were directly injected into an Orbitrap Fusion mass spectrometer (Thermo Scientific) by nano-electrospray and subject to data-dependent tandem mass spectrometry, with full precursor ion scans (MS1) collected at 120,0000 resolution. Monoisotopic precursor selection and charge-state screening were enabled, with ions of charge >+1 selected for collision-induced dissociation (CID).  Fragmentation scans (MS2) were collected per MS1 using the topspeed mode with precursor priority set to most intense. Dynamic exclusion was active with 60 s exclusion for ions selected once within a 35 s window.

### Data Protocol
Spectra were searched against an P. gingivalis 33277 (NCBI NC_010729.1) protein sequence database and common contaminant proteins (ms-blender [Kwon T, Choi H, Vogel C, Nesvizhskii AI, Marcotte EM, MSblender: a probabilistic approach for integrating peptide identifications from multiple database search engines, Journal of Proteome Research, 10(7):2949-58 (20110] using tandem, comet, and MSGF+ search engines). Fully-tryptic peptides were considered, with up to two missed cleavages. Tolerances of 10 ppm (MS1) and 0.5 Da (MS2), carbamidomethylation of cysteine as static modification, and oxidized methionine as dynamic modification were used. High-confidence peptide-spectral matches (PSMs) were filtered at <1% false discovery rate.

### Publication Abstract
Many human infections are polymicrobial in origin, and interactions among community inhabitants shape colonization patterns and pathogenic potential <sup>1</sup> . Periodontitis, which is the sixth most prevalent infectious disease worldwide <sup>2</sup> , ensues from the action of dysbiotic polymicrobial communities <sup>3</sup> . The keystone pathogen Porphyromonas gingivalis and the accessory pathogen Streptococcus gordonii interact to form communities in vitro and exhibit increased fitness in vivo <sup>3,4</sup> . The mechanistic basis of this polymicrobial synergy, however, has not been fully elucidated. Here we show that streptococcal 4-aminobenzoate/para-amino benzoic acid (pABA) is required for maximal accumulation of P. gingivalis in dual-species communities. Metabolomic and proteomic data showed that exogenous pABA is used for folate biosynthesis, and leads to decreased stress and elevated expression of fimbrial adhesins. Moreover, pABA increased the colonization and survival of P. gingivalis in a murine oral infection model. However, pABA also caused a reduction in virulence in vivo and suppressed extracellular polysaccharide production by P. gingivalis. Collectively, these data reveal a multidimensional aspect to P. gingivalis-S. gordonii interactions and establish pABA as a critical cue produced by a partner species that enhances the fitness of P. gingivalis while diminishing its virulence.

### Keywords
Porphyromonas gingivalis, P. gingivalis, Periodontitis, Streptococcal 4 aminobenzoate/para-amino benzoic acid (paba), Polymicrobial

### Affiliations
Institute for Cellular and Molecular Biology, and Center for Systems and Synthetic Biology, The University of Texas at Austin, Austin, Texas, United States of America
University of Texas

### Submitter
John Houser

### Lab Head
Dr Edward Marcotte
Institute for Cellular and Molecular Biology, and Center for Systems and Synthetic Biology, The University of Texas at Austin, Austin, Texas, United States of America


