### Accession
PXD001592

### Title
The impact II, a very high resolution quadrupole time-of-flight instrument for deep shotgun proteomics

### Description
Hybrid quadrupole time-of-flight (QTOF) is one of the two major mass spectrometric technologies used in proteomics. Although based on simple fundamental principles, it has over the last decades greatly evolved in terms of achievable resolution, mass accuracy and dynamic range. The Bruker impact platform of QTOF instruments takes advantage of these developments and here we develop and evaluate the impact II for shotgun proteomics applications. Adaption of our heated LC system achieved very narrow peptide elution peaks. The impact II is equipped with a new collision cell with both axial and radial ion ejection, more than doubling ion extraction at high MS/MS frequencies. The new reflectron and detector improve resolving power compared to the previous model up to 80%, i.e. to 40,000 at m/z 1222. We analyzed the ion current from the inlet capillary and found very high transmission (>80%) up to the collision cell. Simulation and measurement indicated 60% transfer into the flight tube. We adapted MaxQuant for QTOF data, improving absolute average mass deviations to better than 1.45 ppm. More than 4,800 proteins can be identified in a single run of HeLa digest in a 90 min gradient. The workflow achieved high technical reproducibility (R2>0.99) and accurate fold change determination in spike-in experiments over three orders of magnitude in complex mixtures. Using label-free quantification we rapidly quantified haploid against diploid yeast and characterized overall proteome differences in mouse cell lines originated from different tissues. Finally, after high pH reversed-phase fractionation we identified 9,515 proteins in a triplicate measurement of HeLa peptide mixture and 11,257 proteins in cerebellum â€“ the highest proteome coverage measured with a QTOF instrument so far.

### Sample Protocol
Preparation of HeLa lysates - HeLa cells (ATCC, S3 subclone) were cultured in Dulbecco`s modified Eagle`s medium (DMEM) containing 10% fetal bovine serum, 20 mM glutamine and 1% penicillin-streptomycin (all from PAA Laboratories). Cells were collected by centrifugation at 200 g for 10 min, washed once with cold phosphate buffered saline (PBS) and centrifuged again. Supernatant was carefully discarded and the cell pellet shock frozen in liquid nitrogen and stored at -80 Â°C until further use. A pellet containing 5x107 cells was resuspended in 1.5 ml of ice cold Milli-Q water, then an equal volume of trifluoroethanol (Sigma-Aldrich) was added. The cell suspension was kept on ice for 10 min, vortexed for 1 min and sonicated for 2 min at 20% duty cycle and output control 3 (Branson Ultrasonics sonifier; model 250). After the addition of 200 Âµl Tris (pH 8.5, final concentration: 100 mM), 400 Âµl TCEP (final concentration: 10 mM) and 400 Âµl 2-chloroacetamide (CAA) (final concentration: 40 mM) the lysate was incubated for 10 min at 95 Â°C. Then the sample was diluted to 15 ml with 50 mM ammonium bicarbonate. The mixture was digested by adding LysC (Wako Chemicals GmbH, ratio 1 Âµg LysC:100 Âµg sample protein) for 2 h at 37 Â°C, followed by adding trypsin (ratio 1 Âµg trypsin:75 Âµg sample protein, Promega GmbH) at 37 Â°C overnight. After a further digestion with trypsin (ratio 1:125) for 5 h at 37 Â°C, the digested peptides with an estimated concentration of 1 Âµg/Âµl were diluted 1:4 with water and acidified by adding formic acid (FA) (final concentration: 0.2%) and purified on Sep-Pak tC18 cartridges (Waters) according to manufacturer`s instructions. Peptide concentration was determined using a NanoDrop spectrophotometer (Thermo Scientific).

### Data Protocol
Analysis of proteomic data - All data were analyzed with the MaxQuant software (version 1.5.2.8 or version 1.5.0.1) (38, 39) with the Andromeda search engine (38) with the adaptions and developments described above. The false discovery rate (FDR) was set to 1% for both proteins and peptides and we specified a minimum length of 7 amino acids. MaxQuant scored peptides for identification based on a search with an initial allowed mass deviation of the precursor ion of up to 0.07 Da after time-dependent recalibration of the precursor masses. The allowed fragment mass deviation was 40 ppm. The Andromeda search engine was used for the MS/MS spectra search against the Uniprot human database (downloaded on 06-21-2014, containing 88,976 entries and 247 contaminants), the Uniprot Saccharomyces cerevisiae database (downloaded on 06-21-2014, containing 6,643 entries), the Uniprot mouse database (downloaded on 06-21-2014, containing 51,573 entries) and UPS fasta file provided by Sigma-Aldrich (http://www.sigmaaldrich.com/life-science/proteomics/mass-spectrometry/ups1-and-ups2-proteomic.html) for quantitative study.  Enzyme specificity was set as C-terminal to Arg and Lys, also allowing cleavage at proline bonds and a maximum of two missed cleavages. Carbamidomethylation of cysteine was selected as fixed modification and N-terminal protein acetylation and methionine oxidation as variable modifications.  The â€˜match between runsâ€™ feature of MaxQuant was used to transfer identifications to other LC-MS/MS runs based on their masses and retention time (maximum deviation 0.7 min) and this was also used in quantification experiments. All quantifications were performed with the label-free algorithms described recently (39). Further analysis of data was performed in the MaxQuant Viewer, in the Perseus post data acquisition package that is part of MaxQuant (all freely available at www.maxquant.org) and in the R statistical computing environment.  Potential contaminants as well as proteins identified only by site modification were strictly excluded from further analysis. For the quantitative analysis of the UPS standards in yeast, entries were only accepted with valid values in all 12 replicates. Results were then filtered for Welch-significance regulation of UPS-2 proteins.  Hierarchical clustering were based on LFQ intensities of the samples. After filtering (3 valid values in at least one group), remaining missing values were imputed from a normal distribution (width: 0.3; down shift: 1.8). Two sample t-test was performed with a FDR of 0.05. Hierarchical clustering of significantly different proteins was performed after z-score normalisation.  For the quantitative analysis of the different cell lines, we required a minimum peptide ratio count of 2 and at least one â€˜razor peptideâ€™ for quantification. For the principal component analysis (PCA), we furthermore limited the dataset to entries with a minimum of 4 valid values in at least one group of 6 replicates. Remaining missing values were imputed from a normal distribution (width: 0.3; down shift: 1.8).

### Publication Abstract
Hybrid quadrupole time-of-flight (QTOF) mass spectrometry is one of the two major principles used in proteomics. Although based on simple fundamentals, it has over the last decades greatly evolved in terms of achievable resolution, mass accuracy, and dynamic range. The Bruker impact platform of QTOF instruments takes advantage of these developments and here we develop and evaluate the impact II for shotgun proteomics applications. Adaption of our heated liquid chromatography system achieved very narrow peptide elution peaks. The impact II is equipped with a new collision cell with both axial and radial ion ejection, more than doubling ion extraction at high tandem MS frequencies. The new reflectron and detector improve resolving power compared with the previous model up to 80%, i.e. to 40,000 at m/z 1222. We analyzed the ion current from the inlet capillary and found very high transmission (&gt;80%) up to the collision cell. Simulation and measurement indicated 60% transfer into the flight tube. We adapted MaxQuant for QTOF data, improving absolute average mass deviations to better than 1.45 ppm. More than 4800 proteins can be identified in a single run of HeLa digest in a 90 min gradient. The workflow achieved high technical reproducibility (R2 &gt; 0.99) and accurate fold change determination in spike-in experiments in complex mixtures. Using label-free quantification we rapidly quantified haploid against diploid yeast and characterized overall proteome differences in mouse cell lines originating from different tissues. Finally, after high pH reversed-phase fractionation we identified 9515 proteins in a triplicate measurement of HeLa peptide mixture and 11,257 proteins in single measurements of cerebellum-the highest proteome coverage reported with a QTOF instrument so far.

### Keywords
Lc_ms/ms, Proteomics, Qtof

### Affiliations
Proteomics and Signal Transduction, Max-Planck-Institute of Biochemistry, Am Klopferspitz 18, 82152 Martinsried, Germany
Proteomics

### Submitter
Mario Oroshi

### Lab Head
Dr Matthias Mann
Proteomics and Signal Transduction, Max-Planck-Institute of Biochemistry, Am Klopferspitz 18, 82152 Martinsried, Germany


