### Accession
PXD020673

### Title
Effects of AKT inhibition in the pancreatic cancer cell proteome

### Description
The AKT pathway is a major regulator of human pancreatic adenocarcinoma progression and a key pharmacological target. The mechanisms of adaptation to long-term silencing of AKT isoforms of human and mouse pancreatic adenocarcinoma cancer cells were studied.

### Sample Protocol
Protein extracts derived from adapted cells to silencing of AKT isoforms were homogenized in lysis buffer containing 7 M urea, 2 M thiourea, 4% (w/v) CHAPS, 50 mM DTT. The homogenates were spinned down at 100.000 x g for 1 h at 15ºC. Protein concentration was measured in the supernatants with the Bradford assay kit (Biorad).  A shotgun comparative proteomic analysis of cellular proteomes using iTRAQ (isobaric Tags for Relative and Absolute Quantitation) was performed [48]. Protein extracts were precipitated with methanol/choloroform, and pellets dissolved in TEAB 0.5M and Urea 6M. Protein quantitation was performed with the Bradford assay kit (Bio-Rad). iTRAQ labeling of each biological condition (in duplicates) was performed according to the manufacturer's protocol (Sciex). Briefly, equal amounts of cellular proteins (50 μg) were reduced with 50 mM tris (2-carboxyethyl) phosphine (TCEP) at 60 °C for 1 h. Cysteine residues were alkylated with 200 mM methylmethanethiosulfonate (MMTS) at room temperature for 15 min. Protein enzymatic cleavage was carried out with trypsin (Promega; 1:50, w/w) at 37 °C for 16 h. Each tryptic digest was labelled according to the manufacturer's instructions with one isobaric amine-reactive tags as follows: Tag113, control cells A; Tag114, control cells B; Tag115, shAKT1-A; Tag116, shAKT1-B; Tag117, shAKT2-A; Tag118, shAKT2-B; Tag119, shAKT3-A; Tag121, shAKT3-B. After 1h incubation, labelled samples were pooled and evaporated until < 40 μl in a vacuum centrifuge.  To increase proteome coverage, the peptide pool was fractionated by Pierce Strong Ion Exchange Spin Columns (ThermoFisher). Briefly, the pool mixture was resuspended in 10 mM KH2PO4 20% ACN pH 3 and the column was activated using 10 mM KH2PO4 20% ACN pH 3 0.5 M KCl. After washing steps with 10 mM KH2PO4 20% ACN pH 3, the pool mixture was added to the column and incubated for 5 minutes at room temperature. Different elution solutions were applied (from 10mM to 300mM KCl), collecting 12 peptide fractions. After drying-down the eluted samples, C18 Zip Tip Solid Phase Extraction (Millipore) were perform to purify and concentrated the peptide mixtures. All fractions were reconstituted into 20 μl of 2% acetonitrile, 0.1% formic acid, 98% MilliQ-H20 prior to mass spectrometric analysis.

### Data Protocol
Peptides mixtures were separated by reverse phase chromatography using an Eksigent nanoLC ultra 2D pump fitted with a 75 μm ID column (Eksigent 0.075 x 250). Samples were first loaded for desalting and concentration into a 0.5 cm length 300 μm ID precolumn packed with the same chemistry as the separating column. Mobile phases were 100% water 0.1% formic acid (FA) (buffer A) and 100% Acetonitrile 0.1% FA (buffer B). Column gradient was developed in a 70 min two step gradient from 2% B to 30% B in 60 min and 30%B to 40% B in 10 min. Column was equilibrated in 95% B for 5 min and 2% B for 15 min. During all process, precolumn was in line with column and flow maintained all along the gradient at 300 nl/min. Eluting peptides from the column were analyzed using an Sciex 5600 Triple-TOF system. Information data acquisition was acquired upon a survey scan performed in a mass range from 350 m/z up to 1250 m/z in a scan time of 250 ms. Top 25 peaks were selected for fragmentation. Minimum accumulation time for MS/MS was set to 75 ms giving a total cycle time of 2.1 s. Product ions were scanned in a mass range from 100 m/z up to 1500 m/z and excluded for further fragmentation during 15 s.  Data Analysis. After MS/MS analysis, data files were processed using ProteinPilot™ 5.0 software from Sciex which uses the algorithm Paragon™ (v.4.0.0.0) [49] for database search and Progroup™ for data grouping and searched against Uniprot Human database (September 2016, 70550 entries). The search parameters allowed for cysteine modification by MMTS and biological modifications programmed in the algorithm. Reporter ion intensities were bias corrected for the overlapping isotope contributions from the iTRAQ tags according to the certiﬁcate of analysis provided by the reagent manufacturer (Sciex). The peptide and protein selection criteria for relative quantitation were performed as previously described [50]. Several quantitative estimates provided for each protein by ProteinPilot were utilized: the fold change ratios of differential expression between labelled protein extracts; the p-value, representing the probability that the observed ratio is different than 1 by chance. A decoy database search strategy was also used to estimate the false discovery rate (FDR), defined as the percentage of decoy proteins identified against the total protein identification. The FDR was calculated by searching the spectra against the decoy database generated from the target database using a non-lineal fitting method [51] and displayed results were those reporting a 1% Global FDR or better.  Results were exported for data interpretation. Relative quantification and statistical analysis were provided by the ProteinPilot software, with an additional 1.3-fold change cutoff for all iTRAQ ratios (ratio ≤0.77 or ≥1.3) to classify proteins as up- or down-regulated. Proteins with iTRAQ ratios below the low range (0.77) were considered to be under-expressed, whereas those above the high range (1.3) were considered to be overexpressed.

### Publication Abstract
Cancer cells acquire resistance to cytotoxic therapies targeting major survival pathways by adapting their metabolism. The AKT pathway is a major regulator of human pancreatic adenocarcinoma progression and a key pharmacological target. The mechanisms of adaptation to long-term silencing of AKT isoforms of human and mouse pancreatic adenocarcinoma cancer cells were studied. Following silencing, cancer cells remained quiescent for long periods of time, after which they recovered proliferative capacities. Adaptation caused profound proteomic changes largely affecting mitochondrial biogenesis, energy metabolism and acquisition of a number of distinct cancer stem cell (CSC) characteristics depending on the AKT isoform that was silenced. The adaptation to AKT1 silencing drove most de-differentiation and acquisition of stemness through C-MYC down-modulation and NANOG upregulation, which were required for survival of adapted CSCs. The changes associated to adaptation sensitized cancer cells to inhibitors targeting regulators of oxidative respiration and mitochondrial biogenesis. In vivo pharmacological co-inhibition of AKT and mitochondrial metabolism effectively controlled pancreatic adenocarcinoma growth in pre-clinical models.

### Keywords
Adenocarcinoma, Proteomics, Akt, Pancreas, Cancer

### Affiliations
Proteomics unit
Proteored-isciii Proteomics Platform-Navarrabiomed (Pamplona-Spain)

### Submitter
Enrique SantamarÃ­a

### Lab Head
Dr Enrique Santamaria
Proteored-isciii Proteomics Platform-Navarrabiomed (Pamplona-Spain)


