### Accession
PXD036978

### Title
Dynamic Reconfiguration of Switchgrass Proteomes in Response to Rust (Puccinia novopanici) Infection

### Description
Switchgrass (Panicum virgatum) can be infected by the rust pathogen (Puccinia novopanici) resulting in lowering biomass yields and quality. Here, two cultivars with divergent rust resistance were used to evaluate the dynamic changes in proteomes following rust inoculation.  Label-free quantitative proteomics was conducted on leaf extracts harvested from a susceptible cultivar Summer at 7, 11, and 18 DAI, and leaves collected at 18 DAI from controls and infected plants of the more resistant cultivar Kanlow.

### Sample Protocol
Leaf materials were extracted and analyzed as described previously. (Zogli, Alvarez, Naldrett, Palmer et al. 2020). Briefly, proteins were extracted using Tris-buffered phenol (pH 8.8). Proteins precipitated from phenol using several washes with acetone and methanol, were denatured using urea/thiourea, and 800 Âµg protein was reduced and alkylated with iodoacetamide. Samples were diluted 10-fold and digested with trypsin first for 24h, and then for a further 3h using a 1:50 enzyme:substrate ratio. Lyophilized peptide digests were resolubilized in 2% acetonitrile containing 0.1% formic acid and analyzed by LC-MS/MS on an RSLCnano system (ThermoFisher Scientific) coupled to a Q-Exactive HF mass spectrometer (ThermoFisher Scientific).

### Data Protocol
Obtained mass spectra were analyzed in Proteome Discoverer 2.2 software (ThermoFisher Scientific), and searched against the common contaminants database cRAP (116 entries, www.theGPM.org), the version 5.1 of the switchgrass genome  (79335 entries, www.phytozome-next.jgi.doe.org) and the Uniprot database for Puccinia using Mascot 2.6.2. Protein N-terminal acetylation, asparagine and glutamine deamidation, and methionine oxidation were set as variable modifications, whilst Cys carbamidomethylation was specified as a fixed modification. A maximum of two trypsin missed cleavages were permitted and the precursor and fragment mass tolerances were set to 10 ppm and 0.02 Da, respectively. Peptides were validated by Percolator with a 0.01 posterior error probability (PEP) threshold. The data were searched using a decoy database to set the false discovery rate to 1% (high confidence).

### Publication Abstract
Switchgrass (<i>Panicum virgatum</i> L.) can be infected by the rust pathogen (<i>Puccinia novopanici</i>) and results in lowering biomass yields and quality. Label-free quantitative proteomics was conducted on leaf extracts harvested from non-infected and infected plants from a susceptible cultivar (Summer) at 7, 11, and 18 days after inoculation (DAI) to follow the progression of disease and evaluate any plant compensatory mechanisms to infection. Some pustules were evident at 7 DAI, and their numbers increased with time. However, fungal DNA loads did not appreciably change over the course of this experiment in the infected plants. In total, 3830 proteins were identified at 1% false discovery rate, with 3632 mapped to the switchgrass proteome and 198 proteins mapped to different <i>Puccinia</i> proteomes. Across all comparisons, 1825 differentially accumulated switchgrass proteins were identified and subjected to a STRING analysis using Arabidopsis (<i>A. thaliana</i> L.) orthologs to deduce switchgrass cellular pathways impacted by rust infection. Proteins associated with plastid functions and primary metabolism were diminished in infected Summer plants at all harvest dates, whereas proteins associated with immunity, chaperone functions, and phenylpropanoid biosynthesis were significantly enriched. At 18 DAI, 1105 and 151 proteins were significantly enriched or diminished, respectively. Many of the enriched proteins were associated with mitigation of cellular stress and defense.

### Keywords
Rust, Lfq, Switchgrass, Proteome

### Affiliations
University of Nebraska - Lincoln

### Submitter
Sophe Alvarez

### Lab Head
Dr Sophie Alvarez
University of Nebraska - Lincoln


