### Accession
PXD028237

### Title
Identification of the Tau aggresome in progressive supranuclear palsy by in situ biotinylation by antibody recognition

### Description
Here, we used an unbiased proteomic approach, which combines antibody-mediated proximity-detection of co-aggregated proteins together with mass spectrometry. Biotinylation by antibody recognition (BAR) is a recently developed method, by which a primary antibody recognises the target of interest in fixed samples. A secondary antibody conjugated to horseradish peroxidase (HRP) recognises the primary antibody, and with the addition of biotin phenol and hydrogen peroxide, facilitates biotinylation of proteins in close proximity. Biotinylated proteins are then biochemically isolated and identified using mass spectrometry (MS).

### Sample Protocol
Vibratome tissue sections were washed with 1xTBS before endogenous peroxidases were quenched by incubating in 1% H2O2 solution in TBS for 1 hr at room temperature (RT). Sections were then washed and blocked for 2hrs at RT in 10% (v/v) goat serum (GS) + 0.3M glycine in TBS. Primary antibody recognising p-Tau (AT8) was diluted in 1% GS in TBS, while negative control sections with primary antibodies omitted were incubated on sections for 2 nights at 4oC. Sections were washed before Goat conjugated Superboost™ HRP was added to 1% GS and incubated overnight at 4oC. The next day, sections were washed then incubated with biotin-XX-tyramide reagent (Thermo) for 45 min at RT. The biotinylation reaction was conducted for 95 seconds with 0.03% H2O2 before the solution was removed and stop solution was added before being washed. In order to isolate biotinylated proteins, biotinylated tissue was first homogenised using a hand-held douncer in BAR lysis buffer (1% SDS, 1% sodium deoxycholate). After homogenisation, tissue was probe-sonicated 10 times using a Sonic Ruptor 250 at 50% power and pulser settings set to 30%. To reverse cross-link the fixed tissue, samples were heated at 95°C for 1 hour, then further heated to 60°C for two hours. After reverse cross-linking, samples were cooled to 4°C before appropriate amounts of protease and phosphatase inhibitor cocktail (Roche) were added to samples. Resulting lysates were centrifuged at 1,000 ×g for 5 minutes at 4°C.Proteins within the clarified lysate were quantified using the Pierce BCA protein assay kit (Thermo). Equal amounts of protein were aliquoted before 50 µL of pre-washed streptavidin-coated magnetic beads (GE) were added and samples were left to rotate overnight at 4°C. A magnetic rack was then used to isolate magnetic beads and associated biotinylated proteins. To reduce non-specific binding, magnetic beads were washed twice with BAR lysis buffer, then twice with RIPA lysis buffer (50 mM Tris-HCl, pH 7.5, 150 mM NaCl, 1% NP-40, 1mM EDTA, 1mM EGTA, 0.1% SDS, 0.5% Sodium deoxycholate, pH 7.4), once with 1M KCl, 1M Na2CO3, 2M urea (in 10 mM Tris-HCl, pH 8.0) and finally twice with RIPA buffer and BAR lysis buffer.Before trypsin digests were conducted, excess detergent was washed off from the beads by resuspending the beads five times in 20 mM ammonium bicarbonate (AmBic). Washed beads were incubated with 10mM Dithiothreitol (DTT) in 20 mM AmBic for 1 hour at 60°C. Samples were then alkylated using 15 mM of iodoacetamide (IAA) in 20 mM AmBic for 1 hour at room temperature in the dark. For trypsin digestion, proteins were incubated with 5µg/mL of sequencing grade trypsin (Promega) overnight at 37°C whilst shaking. Resulting peptides were lyophilized using a Speedvac. For desalting, peptides were resuspended in 0.1% formic acid (FA) and desalted using pre-washed and equilibrated C18 OMIX tips (Agilent). Once desalted, samples were again lyophilised using a Speedvac before being stored at -80°C until MS analysis.

### Data Protocol
Resulting raw files were searched using Proteome Discoverer 2.4 software (Thermo), which made use of the Sequest search algorithm. Here, the Homo sapiens Uniprot FASTA database 9606 was used. For peptide identifications, the settings accounted for a 20-ppm precursor ion FT-MS tolerance as well as a 0.1 Da MS/MS fragment ion tolerance for HCD fragmentation. For static modifications, the program was set to search for carbamidomethylation on cysteine. For dynamic modifications on the protein terminus, the program was set to Acetyl (N-Terminal). For dynamic modifications, the program was set to Oxidation on Methionine and Phosphorylation on Serine, Threonine, and Tyrosine. Trypsin was assigned as the enzyme, allowing for up to three missed cleavages.    The data were further processed using a Label-Free Quantitationworkflow featuring the Minora Feature node. This made use of the percolator node, enabling an estimation of false discovery rates (FDR) at the protein level and peptide spectrum match (PSM) level. After obtaining FDR, results were adjusted to <1% total FDR. In addition, a q-value of <0.01 was required.

### Publication Abstract
Progressive supranuclear palsy (PSP) is a late-onset neurodegenerative disease defined pathologically by the presence of insoluble phosphorylated-Tau (p-Tau) in neurons and glia. Identifying co-aggregating proteins within p-Tau inclusions may reveal important insights into processes affected by the aggregation of Tau. We used a proteomic approach, which combines antibody-mediated biotinylation and mass spectrometry (MS) to identify proteins proximal to p-Tau in PSP. Using this proof-of-concept workflow for identifying interacting proteins of interest, we characterized proteins proximal to p-Tau in PSP cases, identifying &gt;84% of previously identified interaction partners of Tau and known modifiers of Tau aggregation, while 19 novel proteins not previously found associated with Tau were identified. Furthermore, our data also identified confidently assigned phosphorylation sites that have been previously reported on p-Tau. Additionally, using ingenuity pathway analysis (IPA) and human RNA-seq datasets, we identified proteins previously associated with neurological disorders and pathways involved in protein degradation, stress responses, cytoskeletal dynamics, metabolism, and neurotransmission. Together, our study demonstrates the utility of biotinylation by antibody recognition (BAR) approach to answer a fundamental question to rapidly identify proteins in proximity to p-Tau from post-mortem tissue. The application of this workflow opens up the opportunity to identify novel protein targets to give us insight into the biological process at the onset and progression of tauopathies.

### Keywords
Tauopathy; aggresome; biotinylation

### Affiliations
Macquarie University

### Submitter
Albert Lee

### Lab Head
Dr Albert Lee
Macquarie University


