### Accession
PXD031948

### Title
Orphan precursor proteins and ROS activate the mitochondrial UPR_Interaction proteomics

### Description
The mitochondrial unfolded protein response (UPRmt) safeguards mitochondria from proteotoxic damage by activating a dedicated transcriptional response in the nucleus to restore proteostasis. How the mitochondrial protein misfolding information is signalled to the nucleus as part of the human UPRmt remains unclear. Here, we showed that UPRmt signalling is carried out by the co-occurrence of two individual signals – ROS production in mitochondria and accumulation of orphan mitochondrial proteins in the cytosol. Combining proteomics and genetic approaches, we identified that mitochondrial protein misfolding caused the release of ROS into the intermembrane space (IMS), which was essential for UPRmt signalling. ROS released from mitochondria oxidized the cytosolic HSP40 protein DNAJA1 to enhance recruitment of cytosolic HSP70 to orphan mitochondrial proteins that accumulated in parallel in the cytosol due to mitochondrial import defect. This recruitment leads to the release of HSF1 from HSP70 and its subsequent translocation to the nucleus to activate transcription of UPRmt related genes. Strikingly, we found that combining ROS and the accumulation of orphan mitochondrial proteins in the cytosol was sufficient and necessary to activate the UPRmt. Our findings reveal that monitoring of ROS and orphan mitochondrial proteins in cytosol allows an elegant surveillance to control the UPRmt via HSF1 activation in human cells. These observations reveal a novel link between mitochondrial and cytosolic proteostasis and provide molecular insight into UPRmt signalling.

### Sample Protocol
Crosslinking was performed by incubating cells in PBS containing 0.8 mg/mL Dithiobis[succinimidyl propionate] (DSP, Proteochem) for 30 min at RT. Crosslinking reactions were quenched with PBS containing 200 µM glycine for 15 min at RT. Cells were lysed in cell lysis buffer (50 mM Tris pH 8.0, 150 mM NaCl, 1 % (v/v) NP40) containing protease inhibitor and allowed to incubate for 30 min at 4 oC. Lysates containing 2 mg of total proteins were used to perform IP with 10 µl Dynabeads protein A (Thermo Fischer Scientific) containing 1 µg of appropriate antibodies or 10 µl Anti-FLAG M2 magnetic beads (Sigma) for 2 h at 4 oC. Immunoprecipitated proteins were eluted from beads for immunoblotting or digested for interaction proteomics. After immunoprecipitation steps 25 µL of SDC (2% SDC, 1mM TCEP, 4mM chloroacetamide, 50 mM Tris pH 8.5) buffer was added to the beads. The mixtures were heated up to 95 oC and the supernatants were collected. For digestion, 25 µL of 50 mM Tris, pH 8.5 containing 1:50 (w/w) LysC (Wako Chemicals) and 1:100 (w/w) trypsin (Promega) was added and allowed to incubate for overnight at 37 oC. Digestion was stop by adding 150 µL of isopropanol containing 1 % (v/v) TFA. Peptide purification was performed with SDB-RPS disc (Sigma) and then dried.

### Data Protocol
MS raw data processing was performed with MaxQuant (v 1.6.17.0) and its in-build label-free quantification algorithm MaxLFQ applying default parameters. Acquired spectra were searched against the human reference proteome (Taxonomy ID 9606) downloaded from UniProt (12-03-2020; “One sequence per gene”, 20531 sequences) and a collection of common contaminants (244 entries) using the Andromeda search engine integrated in MaxQuant. Identifications were filtered to obtain false discovery rates (FDR) below 1% for both peptide spectrum matches (PSM; minimum length of 7 amino acids) and proteins using a target-decoy strategy. Results were then exported to Excel files for further processing. Abundance of interactors was normalised to the abundance of DNAJA1 from each sample. Fold changes were calculated from normalised data. GO enrichment analysis of DNAJA1 interactome was performed by using DAVID. GO enrichments were visualized with EnrichmentMap (v.3.3.2) plugin in Cytoscape (v.3.7.1). Subcellular locations of increased interactors upon GTPP treatment was manually curated from UniProt.

### Publication Abstract
The mitochondrial unfolded protein response (UPR<sup>mt</sup>) is essential to safeguard mitochondria from proteotoxic damage by activating a dedicated transcriptional response in the nucleus to restore proteostasis<sup>1,2</sup>. Yet, it remains unclear how the information on mitochondria misfolding stress (MMS) is signalled to the nucleus as part of the human UPR<sup>mt</sup> (refs. <sup>3,4</sup>). Here, we show that UPR<sup>mt</sup> signalling is driven by the release of two individual signals in the cytosol-mitochondrial reactive oxygen species (mtROS) and accumulation of mitochondrial protein precursors in the cytosol (c-mtProt). Combining proteomics and genetic approaches, we identified that MMS causes the release of mtROS into the cytosol. In parallel, MMS leads to mitochondrial protein import defects causing c-mtProt accumulation. Both signals integrate to activate the UPR<sup>mt</sup>; released mtROS oxidize the cytosolic HSP40 protein DNAJA1, which leads to enhanced recruitment of cytosolic HSP70 to c-mtProt. Consequently, HSP70 releases HSF1, which translocates to the nucleus and activates transcription of UPR<sup>mt</sup> genes. Together, we identify a highly controlled cytosolic surveillance mechanism that integrates independent mitochondrial stress signals to initiate the UPR<sup>mt</sup>. These observations reveal a link between mitochondrial and cytosolic proteostasis and provide molecular insight into UPR<sup>mt</sup> signalling in human cells.

### Keywords
Proteostasis, Mitochndria, Interactome, Dnaja1, Uprmt

### Affiliations
Institute of Biochemistry II, Frankfurt
Institute of Biochemistry II Goethe University Frankfurt – Medical School University Hospital  Germany

### Submitter
Reymond Sutandy

### Lab Head
Dr Christian Münch
Institute of Biochemistry II Goethe University Frankfurt – Medical School University Hospital  Germany


