### Accession
PXD027859

### Title
B27 inmunopeptidome in vaccinica infected cells

### Description
Identification of natural human leukocyte antigen (HLA) ligandome is a key element to understand the cellular immune responses. Advanced high throughput mass spectrometry analyses identify a relevant, but not complete, fraction of the many tens of thousands of self-peptides generated by the antigen processing in live cells. In infected cells, in addition to this complex HLA ligandome, a minority of peptides from degradation of the few proteins encoded by the viral genome are also bound to HLA class I molecules. In this study, the standard immunoproteomics strategy was modified to include the classical acid stripping treatment after virus infection to enrich the HLA ligandome in virus ligands. Complexes of HLA-B*27:05-bound peptide pools were isolated from vaccinia virus (VACV)-infected cells treated with acid stripping after virus infection. The HLA class I ligandome was identified using high throughput mass spectrometry analyses, yielding 42 and 52 natural peptides processed and presented in untreated and after acid stripping treatment VACV-infected human cells, respectively. Most of these virus ligands were identified in both conditions, but a relevant fraction of VACV ligands detected by mass spectrometry was dependent of acid stripping treatment with almost twice more exclusive viral ligands that the untreated VACV-infected condition. Theoretical binding affinity prediction of the VACV HLA-B*27:05 ligands and acute antiviral T cell response characterization in the HLA transgenic mice model showed no differences between HLA ligands identified under the two conditions: untreated and acid stripping condition. These findings indicated that acid stripping treatment could be useful to identify HLA class I ligands from virus-infected cells.

### Sample Protocol
B27-C1R is an HLA-B*27:05 transfectant of the human lymphoid cell line HMy2.C1R.  Cells were cultured in RPMI 1640 with 10% fetal bovine serum. The monoclonal antibody (mAb) used in this study was W6/32 (specific for a monomorphic HLA class I). Uninfected or VACV-infected cells after 2 h of infection (to allow entry of the virus) at moi 10 were washed with RPMI in the absence of serum and incubated for 90 s with ice-cool acid-stripping medium (0.3 M glycine-HCl and 1% BSA in water, pH 2.4) as previously reported. Culture medium was added to neutralize the pH. Cells were washed three times, resuspended in culture medium, and incubated at 37ÂºC overnight to allow normal viral cycle and HLA class I re-expression. HLA-bound peptides were isolated from three independent biological replicates of 1x109 cells from uninfected or VACV-infected cells, treated or not with acid stripping after virus infection as previously described. Cells were lysed in 1% Igepal CA-630 (Merck KGaA, Darmstadt, Germany), 20 mM Tris/HCl buffer, and 150 mM NaCl, pH 7.5 in the presence of a Protease Inhibitor Cocktail (Merck KGaA, Darmstadt, Germany). After centrifugation, the supernatants were passed first through a control precolumn containing CNBr-activated Sepharose 4B (GE Healthcare, Buckinghamshire, UK) to remove non-specific peptides. The HLA-B*27:05/peptide complexes were isolated via affinity chromatography from the soluble cell extract fraction with the W6/32 mAb. The HLA-bound peptides were eluted with 1% aqueous trifluoroacetic acid (TFA), separated from the HLA molecules, and concentrated by ultra-filtration with a Vivaspin 2 filter, 5,000 MWCO HY (Sartorius Stedim Biotech, Goettingen, Germany), as previously described.

### Data Protocol
Raw MS/MS data were converted to mgf files with PeakView 1.2 (Sicex) and searched with Peaks Studio 7.0 (Bioinformatics Solutions). The database (42651 entries) contained the Human Reference Proteome and the Vaccinia (Western Reserve) Proteome downloaded from Uniprot in July 2019. The following parameters were used: no enzyme, MS tolerance of 25 ppm, MS/MS tolerance of 0.05 and oxidation of M and pyroglutamic acid formation from Q and E as variable modifications. Identifiactions were foiltered at a FDR > 1% at the peptide level.

### Publication Abstract
Identification of a natural human leukocyte antigen (HLA) ligandome is a key element to understand the cellular immune response. Advanced high throughput mass spectrometry analyses identify a relevant, but not complete, fraction of the many tens of thousands of self-peptides generated by antigen processing in live cells. In infected cells, in addition to this complex HLA ligandome, a minority of peptides from degradation of the few proteins encoded by the viral genome are also bound to HLA class I molecules. In this study, the standard immunopeptidomics strategy was modified to include the classical acid stripping treatment after virus infection to enrich the HLA ligandome in virus ligands. Complexes of HLA-B*27:05-bound peptide pools were isolated from vaccinia virus (VACV)-infected cells treated with acid stripping after virus infection. The HLA class I ligandome was identified using high throughput mass spectrometry analyses, yielding 37 and 51 natural peptides processed and presented untreated and after acid stripping treatment VACV-infected human cells, respectively. Most of these virus ligands were identified in both conditions, but exclusive VACV ligands detected by mass spectrometry detected on acid stripping treatment doubled the number of those identified in the untreated VACV-infected condition. Theoretical binding affinity prediction of the VACV HLA-B*27:05 ligands and acute antiviral T cell response characterization in the HLA transgenic mice model showed no differences between HLA ligands identified under the two conditions: untreated and under acid stripping condition. These findings indicated that acid stripping treatment could be useful to identify HLA class I ligands from virus-infected cells.

### Keywords
Immunopeptidomics, Vaccinica, Hla, Hla-b*27

### Affiliations
Proteomics Unit - Dpt. of Macromolecular Structures - Spanish National Biotechnolgy Centre (CNB-CSIC)
Proteomics Unit and Functional Proteomics Group. Spanish National Biotechnology Centre (CNB-CSIC).

### Submitter
Miguel Marcilla

### Lab Head
Dr Fernando Corrales
Proteomics Unit and Functional Proteomics Group. Spanish National Biotechnology Centre (CNB-CSIC).


