### Accession
PXD008960

### Title
Improvements to the rice genome annotation through large-scale analysis of RNA-Seq and proteomics data sets

### Description
We have performed a Proteogenomics meta-analysis of data sets deposited in ProteomeXchange: PXD000265, PXD000313, PXD000923, PXD001030, PXD001058, PXD002291, PXD002739, PXD002740 and PXD003156 and using 29 RNA-Seq data sets on rice (Oryza sativa). We created a search database comprising translated reads that had been mapped onto the rice genome, as well as officially annotated rice proteins sequences. The RNA Seq database was pre-processed to identify “novel transcripts” for those not mapping fully to an existing exon, and “novel junctions” for those reads mapped with a gap, implying a potential novel splice site that was not annotated in the official gene set. Confidentially identified “novel peptides” i.e. those mapping to a novel junction or novel transcript were post-processed to ensure that there were no other better explanations for the corresponding spectra e.g. peptide from a canonical gene with a modification or amino acid substitution. Data were exported from the pipeline in PSI mzIdentML 1.2 format, containing chromosomal coordinates, and further converted to PSI proBed format for genome visualisation. Novel peptides were searched against other plant databases using BLAST to see if they had predicted in genes from other species. A total of 1584 novel peptides were identified, mapping to ~700 genomic loci in which either new genes have been predicted (~100) or updates to existing gene models have been predicted (~600).

### Sample Protocol
See details in PXD000265, PXD000313, PXD000923, PXD001030, PXD001058, PXD002291, PXD002739, PXD002740 and PXD003156 and in RNA Seq data sets from The European Nucleotide Archive (ENA, http://www.ebi.ac.uk/ena) database - SRR1288363, SRR1288364, SRR1288365, SRR1288366, SRR1288367, SRR1288368, SRR1288369, SRR1618548, SRR1944689, SRR1944690, SRR1944691, SRR1944692, SRR1944693, SRR1944694, SRR1944695, SRR1944696, SRR1944697, SRR1948083, SRR2040568, SRR2040569, SRR2040570, SRR2405440, SRR3090733, SRR3090734, SRR3090735, SRR3090736, SRR3090737, SRR3090738, SRR3090739.

### Data Protocol
MS datasets were downloaded from PXD000265, PXD000313, PXD000923, PXD001030, PXD001058, PXD002291, PXD002739, PXD002740 and PXD003156 totalling 2,051,418 spectra. The raw RNA Sequencing data for this study was collected from The European Nucleotide Archive (ENA, http://www.ebi.ac.uk/ena) database - SRR1288363, SRR1288364, SRR1288365, SRR1288366, SRR1288367, SRR1288368, SRR1288369, SRR1618548, SRR1944689, SRR1944690, SRR1944691, SRR1944692, SRR1944693, SRR1944694, SRR1944695, SRR1944696, SRR1944697, SRR1948083, SRR2040568, SRR2040569, SRR2040570, SRR2405440, SRR3090733, SRR3090734, SRR3090735, SRR3090736, SRR3090737, SRR3090738, SRR3090739. The reads from each run were aligned using TopHat (v2.0.12) against the Oryza sativa genome (IRGSP-1.0.30) respectively. Accepted matches in Bam format and junction information in Bed format was produced by TopHat. The accepted matches were passed to Cufflinks (v2.2.1) to assemble transcripts. All junction events were first de-duplicated and then aligned against Oryza sativa annotation (IRGSP-1.0.30) to filter out known junctions by home-made scripts, only predictions of potential novel splice events were kept for further custom database construction. All the assembled transcripts in GTF format were annotated by cuffmerge from Cufflinks, and only those novel transcripts (including partial novel ones and entire novel ones) were kept for further custom database construction. These nucleotide sequences were translated into 6 frames, during which those sequences with length less than 15 nucleotides (5 amino acids) were discarded. To reduce the custom database size, for each nucleotide sequence, we only kept the frame with longest resulting translated sequences. After taking all the translated novel events and marking them as “novel events”, we added the known Oryza sativa protein from RefSeq (IRGSP-1.0.30), marked as known proteins, and contamination protein sequences (http://www.thegpm.org/crap/, 245 sequences) into the customized database.  All collected MS/MS data were converted into MGF format file by ProteoWizard (v3.0.4238). The MS/MS data were then searched by IPeak, IPeak is a Java-based open source software tool, which uses the Percolator to re-score PSMs from MS-GF+ (v9733), MyriMatch(v2.2.8634) and X!Tandem (v2009.10.01.1) and then uses the FDRScore algorithm to combine the results from these search engines. Different datasets were searched using the parameters matching the original publications. The original PSMs in mzIdentML format were first filtered by FDRScore, PSMs with FDRScore over 0.01 were discarded, equating to keeping PSMs with q-value (global FDR) < 0.01. The novel peptides passed the FDRScore filtration were further post-processed for validation. For details of post-processing see the Methods section of the corresponding manuscript.

### Publication Abstract
Rice (<i>Oryza sativa</i>) is one of the most important worldwide crops. The genome has been available for over 10 years and has undergone several rounds of annotation. We created a comprehensive database of transcripts from 29 public RNA sequencing data sets, officially predicted genes from Ensembl plants, and common contaminants in which to search for protein-level evidence. We re-analyzed nine publicly accessible rice proteomics data sets. In total, we identified 420K peptide spectrum matches from 47K peptides and 8,187 protein groups. 4168 peptides were initially classed as putative novel peptides (not matching official genes). Following a strict filtration scheme to rule out other possible explanations, we discovered 1,584 high confidence novel peptides. The novel peptides were clustered into 692 genomic loci where our results suggest annotation improvements. 80% of the novel peptides had an ortholog match in the curated protein sequence set from at least one other plant species. For the peptides clustering in intergenic regions (and thus potentially new genes), 101 loci were identified, for which 43 had a high-confidence hit for a protein domain. Our results can be displayed as tracks on the Ensembl genome or other browsers supporting Track Hubs, to support re-annotation of the rice genome.

### Keywords
Big data analysis, Proteogenomics, Rna-seq, Proteomics, Rice

### Affiliations
BGI-Shenzhen
Institute of Integrative Biology, University of Liverpool

### Submitter
Da Qi

### Lab Head
Dr Andrew R Jones
Institute of Integrative Biology, University of Liverpool


