### Accession
PXD016918

### Title
Middle-Down Proteomic Analyses with Ion Mobility Separations of Endogenous Isomeric Proteoforms

### Description
We integrated FAIMS into the WCX-HILIC ETD MS/MS system to improve identification of histonetail proteforms

### Sample Protocol
The mouse embryonic stem cells (~500 M) were grown under 2i-LIF conditions. Crude histones (8 mg per the BCA assay) were extracted per Sidoli et al., [PMID: 27286567] with TCA precipitation replaced by desalting using PD-10 columns. The H3 histone family was purified in 2 mg injections on the 250 ������ 4.6 mm Jupiter C18 5 ������M 300 ������ column employing Solvent A (0.1% TFA) at 34 - 60% and Solvent B (90% ACN, 0.1% TFA). Fractions containing the H3.1, H3.2, and (low-abundant) H3.3 histones were pooled (the H3.1 and H3.2 tails are identical). To sever the tails (ARTKQ TARKS TGGKA PRKQL ATKAA RKSAP ATGGV KKPHR YRPGT VALRE for H3.1/H3.2), we added GluC at 1:10 enzyme/histones ratio, resuspended at 0.75 g/L in 100 mM NH4HCO3, and digested over 8 hours.

### Data Protocol
All ETD spectra were charge-deconvoluted by the X!Tract node of Proteome Discoverer and searched using the Mascot 2.6.0 utility against the MS_HistoneDB database. Proteoforms were quantified by the python implementation of HistoneCoder and isoScale software to deduce the relative abundances of proteoforms within an acquisition using the following steps (see [PMID: 25073878] supporting information for a detailed explanation of the algorithms). 1. for every spectrum the intensity of all fragment ions was summed to obtain TIC 2. the proteoforms identified by Mascot were validated by finding site-determining fragment ions that unambigiously localize each PTM. Spectra with no validated proteoforms were discarded 2. if more than one proteoform was validated the TIC was split between the two validated proteoforms with the highest Mascot score based on the ratio of site-determining ions observed for both proteoforms. 3. abundance of a given proteoform is calculated by summing TIC of spectra where it was identified 4. the relative abundance of a proteoform is calculated by dividing its abundance by the sum of abundances of all proteoforms.

### Publication Abstract
Biological functions of many proteins are governed by post-translational modifications (PTMs). In particular, the rich PTM complement in histones controls the gene expression and chromatin structure with major health implications via a combinatoric language. Deciphering that "histone code" is the great challenge for proteomics given an astounding number of possible proteoforms, including isomers with different PTM positions. These must be disentangled on the top- or middle-down level to preserve the key PTM connectivity, which condensed-phase separations failed to achieve. We reported the capability of ion mobility spectrometry (IMS) methods to resolve such isomers for model histone tails. Here, we advance to biological samples, showing middle-down analyses of histones from mouse embryonic stem cells via online chromatography to fractionate proteoforms with distinct PTM sets, differential or field asymmetric waveform IMS (FAIMS) to resolve the isomers, and Orbitrap mass spectrometry with electron transfer dissociation to identify the resolved species.

### Keywords
Etd, Histones, Middle down, Faims

### Affiliations
Syddansk University
University of Southern Denmark, Department of Biochemistryand Molecular Biology, Protein Research Group (lab head)

### Submitter
Pavel Shliaha

### Lab Head
Dr Ole N Jensen
University of Southern Denmark, Department of Biochemistryand Molecular Biology, Protein Research Group (lab head)


