### Accession
PXD010133

### Title
Proteomic profiling of diffuse gliomas defines clinically and histopathologically relevant disease subtypes

### Description
Background: Molecular profiling of diffuse gliomas has provided significant insights into the pathogenesis, classification and prognostication of these malignancies. However, previous molecular studies of glioma have largely focused on genomic readouts and targeted proteomic profiling technologies. Consequently, proteomic and downstream functional landscape of gliomas in general, and molecular subgroups in particular, remains largely unexplored. Here, we utilize liquid chromatography tandem mass spectrometry to profile genomically-defined cohorts of gliomas spanning the full range of World Health Organization (WHO) grades.  Methods: Bulk frozen tissue and purified micro-dissected regions from formalin-fixed paraffin-embedded (FFPE) tissues were assembled and utilized to define robust proteomic signatures of both low grade, infiltrative and high-grade tumors. As a final analysis, primary tumor tissue was compared with both IDH-mutated and IDH-wildtype glioblastoma stem cell (GSC) lines to further overcome tissue heterogeneity and pinpoint proteins differences likely arising in the relevant glial cellular drivers of tumor development.  Results: In aggregate, 5,496 unique proteins over 3 glioma cohorts were identified, and span common molecular subclasses based on IDH and 1p19q co-deletion status and all four WHO grades. Supervised clustering highlights substantial proteome and systems-level pathway differences between different genetically defined glioma subtypes and WHO grades. By using bulk tumor statistical analysis, 833 proteins distinguish different WHO grade tumors, while FFPE tumor dissection reveals 287 proteins in GBMs with abundance changes according to IDH mutation status.  Using our integrative approach, calcium signaling, proteins of the endoplasmic reticulum and extracellular integrin proteins are most conserved proteomic markers that distinguish aggressive, IDH-wt, from IDH-mut GBM tumors in primary and tissue culture models gliomagenesis.  Conclusions: This proteomic survey provides the largest and most diverse unbiased protein-based brain tumor resource to date. Current treatments for glial tumors are largely non-specific and overlap between genomic subtypes and WHO grades. Our analysis provides early insight into the vast downstream and epigenetic protein-level differences within this molecular framework. Given the central position proteins occupy in driving biology and phenotype, further characterization of the substantial proteomic diversity that exist between the molecular subtypes and grades of gliomas, proteomics may help define more personalized prognostic and predictive biomarkers for precision care.   

### Sample Protocol
LC-MS/MS analysis was performed on Thermo Scientific Q Exactive Plus high-resolution mass spectrometer as previously described 19. Briefly, for FFPE tissues we prepared 10 m thick sections on charged glass slides and stained with hematoxylin and eosin (H&E) to highlight tumor-rich regions from surrounding brain, hemorrhage and tissue necrosis. Areas enriched in lesional tissue were carefully macro-dissected off adjacent deeper slides with a scalpel and proteins were prepared in MS-compatible cell lysis reagent (0.2% RapigestTM, Waters Corp.) by sonicating on ice for 5 cycles of 10 seconds each. Protein crosslinks were reversed by boiling (at 95ºC) in presence of 5µM DTT for 60 minutes followed by 80ºC for 90 minutes. Proteins were quantitated using Coomasie protein quantification assay (Pierce) and trypsin (Sigma) digestion was performed overnight at 37ºC. 15 μg of trypsin-digested peptides were cleaned in C18 tips for MS analysis. Fresh frozen bulk tumors and tissue culture samples were processed identically but in 8M urea lysis reagent without the 95ºC and 80ºC crosslink reversal step.   Tissue lysates from each sample were processed in trypsin and unfractionated digests were analyzed by LC-MS/MS to generate proteomic profiles. For all MS/MS experiments, liquid chromatography and nanoelectrospray pump was used in a 120 min (fresh frozen tumor and GSC samples) and 90 min (FFPE tumor samples) data-dependent acquisition (DDA) program and raw data files were searched using MaxQuant Andromeda search engine (see Biostatistical and informatics analysis section).  Each sample was concentrated using Omix C18MB (Agilent Technologies) tips and eluted with 3 µL of buffer A (0.1% formic acid, 65% acetonitrile).  To each sample, 57µL of buffer B (0.1% formic acid) was added, of which 10µL (1.5µg of peptides) was loaded from a 96-well microplate autosampler onto a C18 trap column using the EASY-nLC1000 system (Thermo Fisher Scientific, San Jose, California) and running Buffer C (0.1% formic acid).   The trap column consisted of IntegraFrit capillary (inner diameter 75µm, New Objective) cut to 3.2cm in length and packed in-house with 5µm Pursuit C¬18 (Agilent Technologies.).  Peptides were eluted from the trap column at 300nl/min with an increasing concentration of Buffer D (0.1% formic acid in acetonitrile) over a 120 min gradient onto a resolving 15cm long PicoTip Emitter (75µm inner diameter, 8µm tip, New Objective) packed in-house with 3µm Pursuit C¬18 (Agilent Technologies.).  The liquid chromatography setup was coupled online to Q Exactive Plus (Thermo Fisher Scientific, San Jose, California) mass spectrometer using a nanoelectrospray ionization source (Thermo Fisher Scientific, San Jose, California) with capillary temperature set to 275°C and spray voltage of 2kV.  A 120 or 90 min data-dependent acquisition (DDA) method was setup on the Q Exactive Plus.  The full MS1 scan from 400-1500 m/z was acquired in the Orbitrap at a resolution of 70,000 in profile mode with subsequent fragmentation of top 12 parent ions using the HCD cell and detection of fragment ions in the Orbitrap using centroid mode at a resolution of 17,500.  The following MS method parameters were used: MS1 Automatic Gain Control (AGC) target was set to 3e6 with maximum injection time (IT) of 100ms, MS2 AGC was set to 5e4 with maximum IT of 50 ms, isolation window was 1.6Da, underfill ratio 2%, intensity threshold 2e4, normalized collision energy (NCE) was set to 27, charge exclusion was set to fragment only 2+,3+ and 4+ charge state ions, peptide match set to preferred and dynamic exclusion set to 42 (for 90 min method) or 48 seconds (for 120 min method).

### Data Protocol
Raw data files were searched using MaxQuant Andromeda search engine (www.coxdocs.org) against the Human Swissprot protein database (July, 2017 version) using match between runs algorithm. Analysis of proteomic datasets was performed using biostatistics software platforms Perseus (www.coxdocs.org), R (www.r-project.org), and Cytoscape (www.cytoscape.org).  LFQ intensity values were used to perform statistical analysis by filtering out protein IDs with less then 60% of samples containing a measurement and imputing the remaining missing values (width 0.42, downshift 1.75).  Cellular components and biological processes were assessed by uploading the Gene names to DAVID functional annotation tool (https://david.ncifcrf.gov).  Pearson correlations were performed by omitting imputed values and using protein measurements found in 100% of the analyzed samples.  For hierarchical clustering protein measurements were Z-score transformed and plotted using complete Euclidean hierarchical clustering method with K-means preprocessing using either whole datasets or only statistically significant protein changes.  Protein level changes between glioma subgroups were determined by either modified Welch’s student t-test or multi-sample ANOVA using FDR adjustment of 0.1.  GO term enrichment analysis was performed using the Fisher exact test in Perseus with FDR settings of 0.02.  Gene set enrichment analysis (GSEA) was used to define driver pathways within subgroups using the gene_set permutation type with a weighted tTest enrichment statistic of 0.1.  Heatmaps and networks of the proteins in GSEA were visualized using Cytoscape.

### Publication Abstract
None

### Keywords
Gliomas, Classification, Biomarkers, Neuro-oncology, Mass spectrometry, Proteome-wide analysis

### Affiliations
UHN
Mt. Sinai Hospital

### Submitter
Ugljesa Djuric

### Lab Head
Dr Phedias Diamandis
UHN


