### Accession
PXD004025

### Title
Assessment of Label-Free Quantification in Discovery Proteomics and Impact of Technological Factors and Natural Variability of Protein Abundance

### Description
Proteome wide protein quantification has become facile in the last years supported by label-free discovery proteomics approaches and powerful, easy to use software. We set out to independently evaluate this highly peptide centric technology focusing on the performance of some of the currently most popular quantitative proteomics software, Proteome Discoverer, Scaffold, MaxQuant and Progenesis QIP. The sample to sample variability introduced into protein abundance estimates by the technology itself, thereby affecting the validity of changes in protein abundance reported to be biological in nature, was determined. Using the same experimental design we were interested in the potential of shotgun proteomics to uncover biological contributions to changes in protein abundance in essentially isogenic plants grown under the same environmental conditions. The accuracy, precision, limit of quantification, intra-analysis repeatability and other performance metrics of protein quantification were determined with the softwares’ ready-to-run, default parameters and some modified settings. MaxQuant and Progenesis QIP each had decided advantages and exhibited direct proportionality between measured and actual protein abundance. The inherent variability in protein quantification in measurements of Arabidopsis thaliana proteomes was substantially higher than the biological sample to sample variability. Strikingly, correlation between Arabidopsis proteins quantified with MaxQuant and Progenesis QIP was poor due to differential protein inference. Both of these points warrant caution in the interpretation of shotgun proteomics studies. The abundance of around 99% of proteins was constant in repeated sampling of the Arabdiopsis thaliana Col-0 inbred accession; however a handful showed substantial quantitative variability. The function stress/stimulus response was highly significantly overrepresented for these proteins. Heritable information beyond genetic polymorphism that affects gene expression levels has recently been documented for these functions. We speculate on the potential of discovery proteomics to measure gene expression levels as a conduit of the epigenotype shaping quantitative traits.

### Sample Protocol
Standard proteins were dissolved according to the manufacturer’s instruction or in ddH2O. SDS-PAGE of standard proteins was performed according to Laemmli. Disulfide bonds were reduced with Dithiothreitol (DTT) and alkylated with an excess of Iodoacetamide (IAA). Proteins were digested with trypsin at an enzyme to protein ratio of 1:50 (w/w) at 37°C over night. Tryptic peptides were desalted with in-house made STAGE Tips containing 6 layers of 3M Empore C18 solid phase extraction matrix (3M, Minneapolis, USA) in a 100 µl pipette tip as described (Majovsky et al., 2014). Six isobaric mixtures of the ten digested standard proteins were prepared to give a six point dilution series (10 fmol/µl, 30 fmol/µl, 100 fmol/µl, 300 fmol/µl, 600 fmol/µl, 1000 fmol/µl) of all of them. The molar composition in fmol/µl and the weight amount of total protein in µg/µl of each of the mixtures is given in Supplemental Table 1. One µl of each mixture was injected into an LC-MS system to measure the dilution series under naked conditions. The six mixtures were spiked into one µg of trypsin digested Arabidopsis thaliana total protein extract and injected to give a measurement of the dilution series under matrix conditions. The ratio of standard to matrix protein weight amount was approximately 11%. Frozen leaf rosettes were ground to a fine, light green powder under liquid nitrogen with a mortar and pestle. Proteins were extracted from the plant tissue with a phenol based procedure described in detail previously (Majovsky et al., 2014).  Peptides were injected into an EASY-nLC 1000 nano liquid chromatography system (Thermo Fisher Scientific). Peptides were separated using C18 reverse phase chemistry using an Acclaim PepMap 100 pre-column (length 2cm, inner diameter 75 µm, particle diameter 3 µm) in-line with an EASY-Spray ES803 column (length 50 cm, inner diameter 75 µm, particle diameter 2 µm) (both from Thermo Fisher Scientific). Peptides were eluted into an EASY-Spray ion source (Thermo Fisher Scientific) with a 180 min gradient increasing from 5% to 40% ACN in ddH2O (60 min gradient for QC measurements of the standard proteins individually) and a flow rate of 300 nl/min and electrosprayed into an Orbitrap Velos Pro mass spectrometer (Thermo Fisher Scientific). The source voltage was set to 2 kV, the S-Lens RF level to 50%. The delta multipole offset was -7.00. The instrument method consisted of one survey (full) scan of the entire ion population in the Orbitrap mass analyzer followed by up to 20 data dependent CID product ion scans of selected precursor ions in the linear quadrupole ion trap (LTQ). A single micro scan per mass spectrum was acquired in both mass analyzers. The AGC target value was set to 1e06 and the maximum injection time (max IT) to 500 ms in the Orbitrap. The parameters were set to 1e04 and 100 ms in the LTQ with an isolation width of 2 Da and normalized collision energy of 35 for precursor isolation and MS/MS scanning. Dynamic exclusion was enabled with a repeat count of 1, a repeat duration of 30 s an exclusion duration of 60s and a relative exclusion width of 10 ppm. Full scan mass spectra were internally calibrated on the fly using the lock mass option with the m/z 445.120024. Four blank injections were run following every sample injection to reduce carryover of the most abundant proteins to around 5%.

### Data Protocol
Thermo .raw files were imported into Proteome Discoverer v1.4. (PD). Peak lists generated with a precursor signal to noise ratio of 1.5 and default settings were used to search a custom made database containing the sequences of the protein standards taken from NCBI (STD, 10 sequences, 3040 residues) with the Mascot algorithm v.2.5.1 on an in-house Mascot server. The enzyme specificity was set to trypsin and two missed cleavages were tolerated. Carbamidomethylation of cysteine was set as a fixed modification and oxidation of methionine as a variable modification. The precursor tolerance was set to 7 ppm and the product ion mass tolerance was set to 0.8 Da. A decoy database search was performed to determine the peptide false discovery rate (FDR) with the Target Decoy PSM Validator module. A 1% peptide FDR threshold was applied. Mascot .mgf files were imported into the Skyline software to produce a spectral library of the standard proteins. The STD standard proteins database in .FASTA format was imported into Skyline to populate a spectral tree. The .raw files were imported into Skyline and XICs of the standard peptides in the respective measurements were produced using the settings described previously (Majovsky et al., 2014). Thermo. Raw files were imported into all tested software without any file conversion. The software versions, settings and employed PQIs are summarized in Table 1 of the paper. All database searches were performed using a concatenated database that combined TAIR10 amended with common contaminates and STD (TAIR10STD, 35,414 sequences, 14,493,054 residues). For PD peptides and proteins were identified with the Mascot software as described above. A protein quantitation matrix containing the #PSMs for each identified protein in each measurement was produced by opening all of the .msf files of the measurements in a single report. The NSAF PQI was calculated manually. Additionally, peptide precursor ion intensities were extracted from the .raw files using the Precursor Ions Area Detector module. The .msf files were imported into the Scaffold Q+ (Scaffold) software. A 1% peptide and 1% protein FDR threshold was applied. Two protein quantitation matrices were produced, one containing the NSAF, the other the Top3 PQI by selecting the respective PQIs in the Quantitative Analysis setup. MaxQuant used its integrated Andromeda search engine to identify peptides and proteins. Carbamidomethylation of cysteine was set as a fixed modification and oxidation of methionine and acetylation of protein N-termini as variable modifications. For all other settings and parameters deviating from software defaults see Table 1 of the paper. Progenesis QIP settings were software defaults with the exceptions listed in Table 1 of the paper. Mascot .mgf files for database search with Mascot were created in QIP excluding MS/MS spectra ranking greater than 5. Peptides and proteins were identified as described above for PD. The peptide FDR threshold was adjusted to 1% and the Mascot search result was imported into Progenesis as a .xml file. Peak picking and alignment of Ovalbumin peptide ion signal peaks were manually adjusted using the Review peak Picking Window and the Select and Edit Buttons in the Run tab.

### Publication Abstract
We evaluated the state of label-free discovery proteomics focusing especially on technological contributions and contributions of naturally occurring differences in protein abundance to the intersample variability in protein abundance estimates in this highly peptide-centric technology. First, the performance of popular quantitative proteomics software, Proteome Discoverer, Scaffold, MaxQuant, and Progenesis QIP, was benchmarked using their default parameters and some modified settings. Beyond this, the intersample variability in protein abundance estimates was decomposed into variability introduced by the entire technology itself and variable protein amounts inherent to individual plants of the Arabidopsis thaliana Col-0 accession. The technical component was considerably higher than the biological intersample variability, suggesting an effect on the degree and validity of reported biological changes in protein abundance. Surprisingly, the biological variability, protein abundance estimates, and protein fold changes were recorded differently by the software used to quantify the proteins, warranting caution in the comparison of discovery proteomics results. As expected, &#x223c;99% of the proteome was invariant in the isogenic plants in the absence of environmental factors; however, few proteins showed substantial quantitative variability. This naturally occurring variation between individual organisms can have an impact on the causality of reported protein fold changes.

### Keywords
Maxquant, Arabidopsis, Protein inference, Epigenetics, Progenesis qip, Shotgun proteomics, Scaffold, Label-free

### Affiliations
Leibniz Institute of Plant Biochemistry

### Submitter
Wolfgang Hoehenwarter

### Lab Head
Dr Wolfgang Hoehenwarter
Leibniz Institute of Plant Biochemistry


