### Accession
PXD026374

### Title
PLCG1 is required for AML1-ETO leukemia stem cell self-renewal

### Description
In an effort to identify novel drugs targeting fusion-oncogene induced acute myeloid leukemia (AML), we  performed high-resolution proteomic analysis. In AML1-ETO (AE) driven AML we uncovered a deregulation of phospholipase C (PLC) signaling. We identified PLCgamma 1 (PLCG1) as a specific target of the AE fusion protein which is induced after AE binding to intergenic regulatory DNA elements. Genetic  inactivation of PLCG1 in murine and human AML inhibited AML1-ETO dependent self-renewal programs,  leukemic proliferation, and leukemia maintenance in vivo. In contrast, PLCG1 was dispensable for normal  hematopoietic stem- and progenitor cell function. These findings are extended to and confirmed by  pharmacologic perturbation of Ca++-signaling in AML1-ETO AML cells, indicating that the PLCG1  pathway poses an important therapeutic target for AML1-ETO positive leukemic stem cells.

### Sample Protocol
A nanoflow HPLC (EASY-nLC1000, Thermo Fisher Scientific) coupled online to an Orbitrap Exploris 480 Mass Spectrometer (Thermo Fischer Scientific) via a nans electrospray ion source was utilized for the sample analysis. Approximately 300 ng of peptides were loaded onto a 50-cm column with 75-µM diameter, packed in house with 1.9µM C18 ReproSil particles (Dr. Maisch GmbH). The column temperature was maintained using a homemade column oven at 60°C. The peptides were separated with the binary buffer system of 0.1% formic acid (buffer A) and 60% ACN plus 0.1%formic acid (buffer B), at a flow rate of 300nl/min. Peptides were eluted over a duration of 120 minutes with a gradient of 30% buffer B over 95 minutes and increased to 60% over for 5minutes. This was followed by a quick ramp up to 95% over 5 minutes and declined back to 5% over 5mins to re-equilibrate the column. Mass spectra was acquired in a data dependent mode. Briefly, with one full scan at a target of 3e6 ions (300-1650 m/z, R=60,000 ), followed by Top15 MS/MS scans with HCD (high energy collisional dissociation) (target 1e5 ions, maximum filling time 28ms, Isolation window 1.4 m/z, and normalized collision energy 27), detected in the Orbitrap at a resolution of 15,000. Dynamic exclusion 30s and charge inclusion (2-5) were enabled.

### Data Protocol
MS raw files were processed using Maxquant (Cox J, Nat Biotech 2008) version 1.5.5.2 supported by Andromeda search engine. The data was searched for proteins and peptides using a target-decoy approach with a reverse database against Uniprot Human (version 2016) fasta file with a false discovery rate of less than 1% at the levels of protein and peptide. Default settings were enabled such as oxidized methionine (M), acetylation (protein N-term), and carbamidomethyl (C) as fixed modification and Trypsin as enzyme specificity. A maximum of 2 missed cleavages was allowed and a minimum peptide length of seven amino acids set. The proteins were assigned to the same protein groups if two proteins could not be discriminated by unique peptides. The label-free quantification was performed using the MaxLFQ algorithm (Cox J et al, 2014) and match between run feature was enabled for identification of peptide across runs based on mass accuracy and normalized retention times. For label free protein quantification minimum ratio count was set to 2. The Maxquant output table was analyzed in Perseus (Tyanova et al, Nat Method 2016), prior to the analysis contaminants marked as reverse hits, contaminants and only identified by site-modification were filtered out.

### Publication Abstract
None

### Keywords
Aml, Aml1-eto, Leukemic stem cells, Plcg1, Proteome, Fusion oncogene

### Affiliations
Department of Proteomics and Signal Transduction MaxPlanck Institute of Biochemistry
Dept Proteomics and Signal Transduction, MaxPlanck Institute of Biochemistry

### Submitter
Ashokkumar Jayavelu

### Lab Head
Dr Prof.Dr.Matthias Mann
Department of Proteomics and Signal Transduction MaxPlanck Institute of Biochemistry


