### Accession
PXD044393

### Title
Enhancing proteome coverage by using strong anion exchange in tandem with basic-pH reversed-phase chromatography for sample multiplexing-based proteomics

### Description
Sample multiplexing-based proteomic strategies rely on fractionation to improve proteome coverage. Tandem mass tag (TMT) experiments, for example, can currently accommodate up to 18 samples with proteins spanning several orders of magnitude, thus necessitating fractionation to achieve reasonable proteome coverage. Here, we present a simple yet effective peptide fractionation strategy that partitions a pooled TMT sample with a two-step elution using a strong anion exchange (SAX) spin column prior to gradient-based basic pH reversed-phase (BPRP) fractionation. We highlight our strategy with a TMTpro18-plex experiment using nine diverse human cell lines in biological duplicate. We collected three datasets, one using only BPRP fractionation, and two others of each SAX-partition followed by BPRP. The three datasets quantified a similar number of proteins and peptides, and the data highlight noticeable differences in the distribution of peptide charge and isoelectric point between the SAX partitions. The combined SAX partition dataset contributed 10% more proteins and 20% more unique peptides that were not quantified by BPRP fractionation alone. In addition to this improved fractionation strategy, we provide an online resource of relative abundance profiles for over 11,000 proteins across the nine human cell lines investigated herein.

### Sample Protocol
Cells were homogenized by 12 passes through a 21-gauge (1.25 inches long) needle. The homogenate was sedimented by centrifugation at 21,000 x g for 5 min and the supernatant was transferred to a new tube. Protein concentrations were determined using the bicinchoninic acid (BCA) assay (ThermoFisher Scientific). Proteins were subjected to disulfide bond reduction with 5 mM tris (2-carboxyethyl) phosphine (room temperature, 15 min) and alkylation with 10 mM iodoacetamide (room temperature, 20 min in the dark). Excess iodoacetamide was quenched with 10 mM dithiothreitol (room temperature, 15 min in the dark). Methanol-chloroform precipitation was performed prior to protease digestion. Samples were resuspended in 200 mM EPPS, pH 8.5 and digested at room temperature for 14 h with LysC protease at a 100:1 protein-to-protease ratio. Trypsin was then added at a 100:1 protein-to-protease ratio and the reaction was incubated for 6 h at 37 Â°C.

### Data Protocol
Database searching included all entries from the human UniProt Database (downloaded: August 2022). The database was concatenated with one composed of all protein sequences for that database in reversed order. Searches were performed using a 50-ppm precursor ion tolerance for total protein level profiling. The product ion tolerance was set to 0.9 Da. These wide mass tolerance windows were selected to maximize sensitivity in conjunction with Comet searches and linear discriminant analysis (LDA). TMTpro labels on lysine residues and peptide N-termini (+304.207 Da), as well as carbamidomethylation of cysteine residues (+57.021 Da), were set as static modifications, while oxidation of methionine residues (+15.995 Da) was set as a variable modification. Peptide-spectrum matches (PSMs) were adjusted to a 1% false discovery rate (FDR). PSM filtering was performed using LDA and then assembled further to a final protein-level FDR of 1%. Proteins were quantified by summing reporter ion counts across all matching PSMs, also as described previously. Reporter ion intensities were adjusted to correct for the isotopic impurities of the different TMTpro reagents according to manufacturer specifications. The signal-to-noise (S/N) measurements of peptides assigned to each protein were summed and these values were normalized so that the sum of the signal for all proteins in each channel was equivalent to account for equal protein loading.

### Publication Abstract
Sample multiplexing-based proteomic strategies rely on fractionation to improve proteome coverage. Tandem mass tag (TMT) experiments, for example, can currently accommodate up to 18 samples with proteins spanning several orders of magnitude, thus necessitating fractionation to achieve reasonable proteome coverage. Here, we present a simple yet effective peptide fractionation strategy that partitions a pooled TMT sample with a two-step elution using a strong anion-exchange (SAX) spin column prior to gradient-based basic pH reversed-phase (BPRP) fractionation. We highlight our strategy with a TMTpro18-plex experiment using nine diverse human cell lines in biological duplicate. We collected three data sets, one using only BPRP fractionation and two others of each SAX-partition followed by BPRP. The three data sets quantified a similar number of proteins and peptides, and the data highlight noticeable differences in the distribution of peptide charge and isoelectric point between the SAX partitions. The combined SAX partition data set contributed 10% more proteins and 20% more unique peptides that were not quantified by BPRP fractionation alone. In addition to this improved fractionation strategy, we provide an online resource of relative abundance profiles for over 11,000 proteins across the nine human cell lines, as well as two additional experiments using ovarian and pancreatic cancer cell lines.

### Keywords
Rts, Eclipse, Sps-ms3, Sax, Faims, Tmtpro

### Affiliations
Harvard Medical School
Harvard Medical School Department of Cell Biology Harvard Medical School Boston, MA, USA

### Submitter
Joao Paulo

### Lab Head
Dr Joao A. Paulo
Harvard Medical School Department of Cell Biology Harvard Medical School Boston, MA, USA


