### Accession
PXD013647

### Title
The tumor suppressor SCRIB is a negative modulator of the Wnt/-catenin signaling (part 1)

### Description
Epithelial tissues are highly organized structures that rely on cell polarity pathways driven by membrane receptors and scaffolding proteins. SCRIBBLE is a cytoplasmic scaffold present at cell-cell junctions in polarized cells that contains LRR and PDZ domains as its paralogue LANO, a member of the LAP protein family. The family is composed of DENSIN-180, ERBIN, SCRIB and LRRC1. Based on phylogenic tree, LAP proteins can be split in two branches with the first branch is made up with SCRIB and LRCC1 and the second one of ERBIN and DENSIN-180.  The current interactome of SCRIB and LRRC1 consists of 67 and 30 protein interaction partners respectively with only 9 (10%) common partners (BioGRID, version 3.5). Our study uncovers the proteome associated to SCRIB and LRRC1 and described for the first time protein associated to each paralogue.

### Sample Protocol
Tissue culture and transfection HEK293T cells were grown in Dulbecco’s modified Eagle medium (DMEM; Sigma) supplemented with 10% fetal bovine serum (FBS; Invitrogen) and penicillin/streptomycin (Bioshop) in a 37°C humidified incubator with 5% CO2.Stable cell lines were generated either by calcium phosphate transfection or polyethylenimine(PEI) transfection.  Immunopurification 5x107 HEK293T cells expressing either FLAG-SCRIB or FLAG-LRRC1 were used for affinity purification procedure. Briefly, cells were lysed and solubilized in TAP lysis buffer (0.1% Igepal CA 630, 10% glycerol, 50mM Hepes-NaOH; pH 8.0, 150mM NaCl, 2mM EDTA, 2mM DTT, 10mM NaF, 0.25mM NaOVO3, 50mM β-glycerophosphate, and protease inhibitor cocktail  (Calbiochem). After 30 min centrifugation at 40,000xg (18,000 rpm in a Beckman JA20 rotor), the soluble fraction was incubated overnight at 4°C with anti-FLAG M2 agarose beads (Sigma). Beads were washed with 100 volumes of TAP lysis buffer. Beads elution was performed using Laemmli buffer.  Mass spectrometry analysis.  Protein extract were loaded on NuPAGE 4-12% Bis-Tris acrylamide gels (Life Technologies) to stack proteins in a single band that was stained with Imperial Blue (Pierce, Rockford, IL) and cut from the gel. Gels pieces were submitted to an in-gel trypsin digestion. Briefly, gel pieces were washed and destained using 100 mM NH4HCO3/acetonitrile (50/50). Destained gel pieces were shrunk with acetonitrile and were re-swollen in the presence of 100 mM ammonium bicarbonate in 50% acetonitrile and dried at room temperature. Protein bands were then rehydrated and cysteines were reduced using 10 mM DTT in 100 mM ammonium bicarbonate pH 8.0 for 45 min at 56 C before alkylation in the presence of 55 mM iodoacetamide in 100 mM ammonium bicarbonate pH 8.0 for 30 min at room temperature in the dark. Proteins were then washed twice in 100 mM ammonium bicarbonate and finally shrunk by incubation for 5 min with 100 mM ammonium bicarbonate in 50% acetonitrile. The resulting alkylated gel pieces were dried at room temperature. The dried gel pieces were re-swollen by incubation in 100 mM ammonium bicarbonate pH 8.0 supplemented with trypsin (12.5 ng/µL; Promega) for 1 h at 4 C and then incubated overnight at 37°C. Peptides were harvested by collecting the initial digestion solution and carrying out two extractions; first in 5% formic acid and then in 5% formic acid in 60% acetonitrile. Pooled extracts were dried down in a centrifugal vacuum system. Samples were reconstituted with 0.1% trifluoroacetic acid in 4% acetonitrile and analyzed by liquid chromatography (LC)-tandem mass spectrometry (MS/MS) using an Orbitrap Fusion Lumos Tribrid Mass Spectrometer (Thermo Electron, Bremen, Germany) both online with a nanoRSLC Ultimate 3000 chromatography system (Dionex, Sunnyvale, CA). Peptides were separated on a Dionex Acclaim PepMap RSLC C18 column.   For peptide ionization in the EASY-Spray nanosource in front of the Orbitrap Fusion Lumos Tribrid Mass Spectrometer, spray voltage was set at 2.2 kV and the capillary temperature at 275 °C. The Orbitrap Lumos was used in data dependent mode to switch consistently between MS and MS/MS. Time between Masters Scans was set to 3 seconds. MS spectra were acquired with the Orbitrap in the range of m/z 400-1600 at a FWHM resolution of 120 000 measured at 400 m/z. AGC target was set at 4.0e5 with a 50 ms Maximum Injection Time. For internal mass calibration the 445.120025 ions was used as lock mass. The more abundant precursor ions were selected and collision-induced dissociation fragmentation was performed in the ion trap to have maximum sensitivity and yield a maximum amount of MS/MS data. Number of precursor ions was automatically defined along run in 3s windows using the “Inject Ions for All Available parallelizable time option” with a maximum injection time of 300 ms. The signal threshold for an MS/MS event was set to 5000 counts. Charge state screening was enabled to exclude precursors with 0 and 1 charge states. Dynamic exclusion was enabled with a repeat count of 1 and a duration of 60 s.

### Data Protocol
The iBAQ intensity were processed using the freely available MaxQuant computational proteomics platform, version 1.6.2.1. The acquired raw LC Orbitrap MS data were first processed using the integrated Andromeda search engine. Spectra were searched against the human database subset of the Swiss-Prot database (date 180919, 20394 entries). This database was supplemented with a set of 245 frequently observed contaminants. The following parameters were used for searches: (i) trypsin allowing cleavage before proline; (ii) two missed cleavages were allowed; (ii) monoisotopic precursor tolerance of 20 ppm in the first search used for recalibration, followed by 4.5 ppm for the main search and 0.5 Da for fragment ions from MS/MS ; (iii) cysteine carbamidomethylation (+57.02146) as a fixed modification and methionine oxidation (+15.99491) and N-terminal acetylation (+42.0106) as variable modifications; (iv) a maximum of five modifications per peptide allowed; and (v) minimum peptide length was 7 amino acids and a maximum mass of 4,600 Da. The match between runs option was enabled to transfer identifications across different LC-MS/MS replicates based on their masses and retention time within a match time window of 0.7 min and using an alignment time window of 20 min. The quantification was performed using a minimum ratio count of 1 (unique+razor) and the second peptide option to allow identification of two co-fragmented co-eluting peptides with similar masses. The false discovery rate (FDR) at the peptide and protein levels were set to 1% and determined by searching a reverse database. For protein grouping, all proteins that cannot be distinguished based on their identified peptides were assembled into a single entry according to the MaxQuant rules. The statistical analysis was done with Perseus program (version 1.6.1.3) from the MaxQuant environment (www.maxquant.org). The  iBAQ intensity were uploaded from the ProteinGroups.txt file. The iBAQ intensities are roughly proportional to the molar quantities of the proteins. Proteins marked as contaminant, reverse hits, and “only identified by site” were discarded. Quantifiable proteins were defined as those detected in at least 100% of samples in at least one condition. Protein intensities were base 2 logarithmized to obtain a normal distribution. Missing values were replaced using data imputation by randomly selecting from a normal distribution centred on the lower edge of the intensity values that simulates signals of low abundant proteins using default parameters (a downshift of 1.8 standard deviation and a width of 0.3 of the original distribution). To determine whether a given detected protein was specifically differential a two-sample t-test were done using permutation based FDR-controlled at 0.01 and employing 250 permutations. The differential proteomics analysis was carried out on identified proteins after removal of proteins only identified with modified peptides, peptides shared with other proteins, proteins from contaminant database and protein.

### Publication Abstract
SCRIB is a scaffold protein containing leucine-rich repeats (LRR) and PSD-95/Dlg-A/ZO-1 domains (PDZ) that localizes at the basolateral membranes of polarized epithelial cells. Deregulation of its expression or localization leads to epithelial defects and tumorigenesis in part as a consequence of its repressive role on several signaling pathways including AKT, ERK, and HIPPO. In the present work, a proteomic approach is used to characterize the protein complexes associated to SCRIB and its paralogue LANO. Common and specific sets of proteins associated to SCRIB and LANO by MS are identified and an extensive landscape of their associated networks and the first comparative analysis of their respective interactomes are provided. Under proteasome inhibition, it is further found that SCRIB is associated to the &#x3b2;-catenin destruction complex that is central in Wnt/&#x3b2;-catenin signaling, a conserved pathway regulating embryonic development and cancer progression. It is shown that the SCRIB/&#x3b2;-catenin interaction is potentiated upon Wnt3a stimulation and that SCRIB plays a repressing role on Wnt signaling. The data thus provide evidence for the importance of SCRIB in the regulation of the Wnt/&#x3b2;-catenin pathway.

### Keywords
Scrib; lrrc1; proteomics analysis

### Affiliations
Life Sciences
Inserm, CBRS, Ais Marseille University, CRCM, Marseille Proteomics

### Submitter
Luc Camoin

### Lab Head
Dr Luc Camoin
Inserm, CBRS, Ais Marseille University, CRCM, Marseille Proteomics


