### Accession
PXD008089

### Title
Metaproteomics and direct Protein-SIF of the deep-sea mussel Bathymodiolus childressi and its bacterial symbionts

### Description
Bathymodiolus childressi is a species of deep-sea mussels found predominantly in the Gulf of Mexico. It colonizes cold seeps such as brine pool and oil seeps. The success of these animals in such environment is thought to be due to the symbiotic association of the mussel host with several species of bacteria. The aim of this study is to understand the role of the different partners involved in the symbiotic system using various “-omics” approaches. In addition to protein identification we used the mass spectrometry data generated and submitted with this project to derive the stable carbon isotope ratios for the different members of the symbiosis using the direct Protein-SIF method. The respective isotope pattern file and SIF computation files are included with this submission.

### Sample Protocol
Lysis buffer (4% (w/v) SDS, 100 mM Tris-HCl (pH 7.6), 0.1 M DTT) was added in a 10:1 ratio to gill samples. Cell disruption was done by beat-beating in lysing matrix D tubes (MP Biomedicals, Cat. No. 116913100) for 45 s at 6.0 m/s with a OMNI bead ruptor bead mill (Omni International, https://www.omni-inc.com/) and subsequent incubation at 95° C for 10 minutes. Cell debris was pelleted for 5 min at 21,000 x g. Protein preparation and digestion was done following the filter-aided sample preparation (FASP) protocol of Wisniewski et al.  (2009, Nat. Meth. 6(5)) with modifications. Briefly, 30 µl of the cleared lysate were mixed with 200 µl of UA solution (8 M urea in 0.1 M Tris/HCl pH 8.5) in a 10 kDa MWCO 500 µl centrifugal filter unit (VWR International) and centrifuged at 14,000 x g for 40 min. Subsequently, 200 µl of UA solution were added and filters spun at 14,000 x g for 40 min. After adding 100 µl of IAA solution (0.05 M iodoacetamide in UA solution) filter units were incubated at 22° C for 20 min. IAA solution was removed by centrifugation. Filter units were washed three times with 100 µl of UA solution.  After this, the filters were washed three times with 100 µl of 50 mM ammonium bicarbonate buffer (ABC). 0.8 µg of MS grade trypsin (Thermo Scientific Pierce, Rockford, IL, USA) in 40 µl ABC were added to the filter units and filters incubated overnight in a wet chamber at 37° C. Peptides were eluted via centrifugation at 14,000 x g for 20 min, followed by addition of 50 µl of 0.5 M NaCl and centrifugation. Per gill, the eluate of three filter units was combined and desalted using Sep-Pak C18 Plus Light Cartridges (Waters, Milford, MA, USA) according to manufacturer’s instructions, with using 0.2 % FA in uHPLC water for buffer exchange. Approximate peptide concentrations were determined using the Pierce Micro BCA assay (Thermo Scientific Pierce, Rockford, IL, USA) following the manufacturer’s instructions. Samples were analyzed via 1D- and 2D-LC-MS/MS. Carryover was reduced by two wash and one blank run between samples. For 1D-LC-MS/MS runs, a UltiMateTM 3000 RSLCnano Liquid Chromatograph (Thermo Fisher Scientific) was used to load 1.5-3 μg of peptide onto a 5 mm, 300 µm ID C18 Acclaim® PepMap100 pre-column (Thermo Fisher Scientific). After loading, peptides were eluted onto the analytical column (50 cm x 75 µm analytical EASY-Spray column packed with PepMap RSLC C18, 2 µm material (Thermo Fisher Scientific), column was heated to 45 °C). An Easy-Spray source connected the analytical column to a Q Exactive Plus hybrid quadrupole-Orbitrap mass spectrometer (Thermo Fisher Scientific). Peptide separation with 460 min gradients and mass spectra acquisition were achieved as described by Kleiner et al. (2017, doi: http://dx.doi.org/10.1101/130575). For 2D-LC-MS/MS, 4.5 μg of peptide were loaded onto a 10 cm, 300 µm ID Poros 10 S SCX column (Thermo Fisher Scientific) using the same LC as for 1D analysis. Peptides were eluted from the SCX column onto the C18 pre-column with 20 μl injections of pH steps (P/N CTIBiphase buffers, Column Technology Inc.) from the autosampler with increasing pH values (pH 2.5-pH 8). After each of the 10 pH step injections, the pre-column was switched in line with the 50 cm x 75 µm analytical EASY-Spray column. Peptides were separated using a 120 min gradient and mass spectra acquired as in Kleiner et al. (2017, doi: http://dx.doi.org/10.1101/130575).

### Data Protocol
For protein identification, a database containing protein sequences predicted from metatranscriptomic and -genomic data of the B. childressi symbiosis was used. The cRAP protein sequence database (http://www.thegpm.org/crap/), which contains sequences of common lab contaminants, was added to the database. The final database contained 38,418 protein sequences. For protein identification, MS/MS spectra were searched against this database using the Sequest HT node in Proteome Discoverer version 2.0.0.802 (Thermo Fisher Scientific) as described in Petersen et al. 2016, Nat. Microb. 2(16195).  An overview of the files submitted has been uploaded as B_childressi_DataOverview.txt

### Publication Abstract
Most autotrophs use the Calvin-Benson-Bassham (CBB) cycle for carbon fixation. In contrast, all currently described autotrophs from the Campylobacterota (previously Epsilonproteobacteria) use the reductive tricarboxylic acid cycle (rTCA) instead. We discovered campylobacterotal epibionts ("Candidatus Thiobarba") of deep-sea mussels that have acquired a complete CBB cycle and may have lost most key genes of the rTCA cycle. Intriguingly, the phylogenies of campylobacterotal CBB cycle&#xa0;genes suggest they were acquired in multiple transfers from Gammaproteobacteria closely related to sulfur-oxidizing endosymbionts associated with the mussels, as well as from Betaproteobacteria. We hypothesize that "Ca. Thiobarba" switched from the rTCA cycle to a fully functional CBB cycle during its evolution, by acquiring genes from multiple sources, including co-occurring symbionts. We also found key CBB cycle genes in free-living Campylobacterota, suggesting that the CBB cycle may be more widespread in this phylum than previously known. Metatranscriptomics and metaproteomics confirmed high expression of CBB cycle genes in mussel-associated "Ca. Thiobarba". Direct stable isotope fingerprinting showed that "Ca. Thiobarba" has typical CBB signatures, suggesting that it uses this cycle for carbon fixation. Our discovery calls into question current assumptions about the distribution of carbon fixation pathways in microbial lineages, and the interpretation of stable isotope measurements in the environment.

### Keywords
Methanotrophic symbionts, Stable carbon isotopes, Bacteria-animal symbiosis, Chemosynthetic symbiosis, 1d-lc-ms/ms, Epsilonproteobacteria, Stable isotope fingerprinting, Q exactive plus, Cold seeps, 2d-lc-ms/ms, Calvin cycle

### Affiliations
University of Greifswald, Greifswald Intitute of Marine Biotechnology
Department of Geoscience, University of Calgary, Calgary, AB, Canada; Department of Plant and Microbial Biology, North Carolina State University, Raleigh, North Carolina, USA

### Submitter
Tjorven Hinzke

### Lab Head
Dr Manuel Kleiner
Department of Geoscience, University of Calgary, Calgary, AB, Canada; Department of Plant and Microbial Biology, North Carolina State University, Raleigh, North Carolina, USA


