### Accession
PXD027945

### Title
Proteomics of HS-5 and NKTert stroma cells

### Description
Comparison of the proteome of the human bone marrow stroma cell lines NKTert and HS-5.

### Sample Protocol
HS-5 or NKTert cells were trypsinized, washed twice with PBS and snap frozen in liquid nitrogen. Cell pellets were thawed and resuspended in 50 Âµl PBS. 50 Âµl of lysis buffer were added (100 mM Hepes/NaOH pH 8.5, 1% SDS and EDTA-free protease inhibitor). Samples were heated to 95Â°C for 5 min. DNA and RNA were degraded by the addition of benzonase at 4Â°C following incubation for 1 h at 37Â°C. Protein concentrations of lysates were determined by BCA protein determination. 10 Âµg of each lysate were subjected to an in-solution tryptic digest using a modified version of the Single-Pot Solid-Phase-enhanced Sample Preparation (SP3) protocol (PMID: 25358341, PMID: 29565595). To this end, lysates were added to Sera-Mag Beads (Thermo Scientific, #4515-2105-050250, 6515-2105-050250) in 10 Âµl 15% formic acid and 30 Âµl of ethanol. Binding of proteins was achieved by shaking for 15 min at room temperature. SDS was removed by 4 subsequent washes with 200 Âµl of 70% ethanol. Proteins were digested overnight at room temperature with 0.4 Âµg of sequencing grade modified trypsin (Promega, #V5111) in 40 Âµl Hepes/NaOH, pH 8.4 in the presence of 1.25 mM TCEP and 5 mM chloroacetamide (Sigma-Aldrich, #C0267). Beads were separated, washed with 10 Âµl of an aqueous solution of 2% DMSO and the combined eluates were dried down. Peptides were reconstituted in 10 Âµl of H2O and reacted for 1 h at room temperature with 80 Âµg of TMT10plex (Thermo Scientific, #90111) (PMID: 24579773) label reagent dissolved in 4 Âµl of acetonitrile. Excess TMT reagent was quenched by the addition of 4 Âµl of an aqueous 5% hydroxylamine solution (Sigma, 438227). Peptides were reconstituted in 0.1 % formic acid, mixed to achieve a 1:1 ratio across all TMT-channels and purified by a reverse phase clean-up step (OASIS HLB 96-well ÂµElution Plate, Waters #186001828BA). Peptides were subjected to an offline fractionation under high pH conditions (PMID: 25358341). The resulting 12 fractions were then analyzed by LC-MS/MS on an Orbitrap Fusion Lumos mass spectrometer (Thermo Scentific) as previously described (PMID:30858367). To this end, peptides were separated using an Ultimate 3000 nano RSLC system (Dionex) equipped with a trapping cartridge (Precolumn C18 PepMap100, 5 mm, 300 Î¼m i.d., 5 Î¼m, 100 Ã…) and an analytical column (Acclaim PepMap 100. 75 Ã— 50 cm C18, 3 mm, 100 Ã…) connected to a nanospray-Flex ion source. The peptides were loaded onto the trap column at 30 Âµl per min using solvent A (0.1% formic acid) and eluted using a gradient from 2 to 40% Solvent B (0.1% formic acid in acetonitrile) over 2 h at 0.3 Âµl per min (all solvents were of LC-MS grade). The Orbitrap Fusion Lumos was operated in positive ion mode with a spray voltage of 2.4 kV and capillary temperature of 275 Â°C. Full scan MS spectra with a mass range of 375â€“1500 m/z were acquired in profile mode using a resolution of 120,000 (maximum fill time of 50 ms or a maximum of 4e5 ions (AGC) and a RF lens setting of 30%. Fragmentation was triggered for 3 s cycle time for peptide like features with charge states of 2â€“7 on the MS scan (data-dependent acquisition). Precursors were isolated using the quadrupole with a window of 0.7 m/z and fragmented with a normalized collision energy of 38. Fragment mass spectra were acquired in profile mode and a resolution of 30,000 in profile mode. Maximum fill time was set to 64 ms or an AGC target of 1e5 ions). The dynamic exclusion was set to 45 s. Acquired data were analyzed using IsobarQuant (PMID: 26379230) and Mascot V2.4 (Matrix Science) using a reverse UniProt FASTA Homo sapiens database (UP000005640 from May 2016) including common contaminants. The following modifications were taken into account: Carbamidomethyl (C, fixed), TMT10plex (K, fixed), Acetyl (N-term, variable), Oxidation (M, variable) and TMT10plex (N-term, variable). The mass error tolerance for full scan MS spectra was set to 10 ppm and for MS/MS spectra to 0.02 Da. A maximum of 2 missed cleavages were allowed. A minimum of 2 unique peptides with a peptide length of at least seven amino acids and a false discovery rate below 0.01 were required on the peptide and protein level (PMID: 25987413).

### Data Protocol
Data were processed in the R programming language (ISBN 3-900051-07-0). The raw output files of IsobarQuant (protein.txt â€“ files) were loaded. For the analysis only proteins quantified with at least two unique peptides were considered, leading to 6657 proteins passing the quality control filters. First, raw signal-sums (signal_sum columns) were cleaned for batch effects using limma (PMID: 25605792). Further normalizations was performed with vsn (variance stabilization normalization - PMID: 12169536). Differentially abundant proteins were analyzed using limma (Ritchie et al., 2015). Information about replicates was added as a factor in the design matrix. Gene set enrichment analysis for the KEGG pathways (Kanehisa and Goto, 2000) was performed using GSEA (Subramanian et al., 2005). A heatmap of the protein abundance of proteins in the lysosomal pathway was visualised using R.

### Publication Abstract
Signals provided by the microenvironment can modify and circumvent pathway activities that are therapeutically targeted by drugs. Bone marrow stromal cell coculture models are frequently used to study the influence of the bone marrow niche on <i>ex&#xa0;vivo</i> drug response. Here, we show that mesenchymal stromal cells from selected donors and NKTert, a stromal cell line, which is commonly used for coculture studies with primary leukemia cells, extensively phagocytose apoptotic cells. This could lead to misinterpretation of results, especially if viability readouts of the target cells (e.g. leukemic cells) in such coculture models are based on the relative proportions of dead and alive cells. Future coculture studies which aim to investigate the impact of bone marrow stromal cells on drug response should take into account that stromal cells have the capacity to phagocytose apoptotic cells.

### Keywords
Stroma hs-5 nktert proteomics

### Affiliations
Department of Medicine V, Hematology, Oncology and Rheumatology, University of Heidelberg, Heidelberg, Germany
EMBL

### Submitter
Frank Stein

### Lab Head
Dr Sascha Dietrich
Department of Medicine V, Hematology, Oncology and Rheumatology, University of Heidelberg, Heidelberg, Germany


