### Accession
PXD033016

### Title
Coisolation of peptide pairs for peptide identification and MS/MS-based quantification

### Description
SILAC-based metabolic labeling is a widely adopted proteomics approach that enables quantitative comparisons among a variety of experimental conditions. Despite its quantitative capacity, SILAC experiments analyzed with data dependent acquisition (DDA) do not fully leverage peptide pair information for identification and suffer from undersampling compared to label-free proteomic experiments. Herein, we developed a data dependent acquisition strategy that coisolates and fragments SILAC peptide pairs and uses y-ions for their relative quantification. To facilitate the analysis of this type of data, we adapted the Comet sequence database search engine to make use of SILAC peptide paired fragments and developed a tool to annotate and quantify MS/MS spectra of coisolated SILAC pairs. In an initial feasibility experiment, this peptide pair coisolation approach generally improved expectation scores compared to the traditional DDA approach. Fragment ion quantification performed similarly well to precursor quantification in the MS1 and achieved more quantifications. Lastly, our method enables reliable MS/MS quantification of SILAC proteome mixtures with overlapping isotopic distributions, which are difficult to deconvolute in MS1-based quantification. This study demonstrates the initial feasibility of the coisolation approach. Coupling this approach with intelligent acquisition strategies has the potential to improve SILAC peptide sampling and quantification.

### Sample Protocol
Saccharomyces cerevisiae (DBY10144 or BY4742) SILAC-labeled proteomes grown at >>5 doublings with respective SILAC labels. Lysates were reduced with DTT, alkylated with IAA, and quenched with DTT. Proteomes were digested with 1:100 (w:w) Lys-C. Following C18 desalting, SILAC-labeled peptides were mixed either Lys0:Lys8 or Lys6:Lys8 at the following ratios (light:heavy): 10:1, 4:1, 2:1, 1:1, 1:2, 1:4, and 1:10. SILAC peptide mixtures were subjected to LC-MS/MS with an Easy nLC 1200 in tandem with an Orbitrap Eclipse Tribrid MS. Peptides were seperated over a 30 cm x 100um analytical column packed with 1.9um C18 beads using a 90 minute acetonitrile gradient (120 minute run). MS data was acquired via our coisolation SILAC acquistion strategy or by DDA or both. Coisolation SILAC MS includes either a 6.5 m/z (Lys0:Lys8 mixtures) or 5 m/z (Lys6:Lys8) mass offset isolation windows centered between SILAC pairs based on precursor charge state.Both left and right offset coisolation scans for a target precursor were conducted to ensure SILAC pair coisolation when the peptide contained one lysine. DDA scans were acquired with 1.6 m/z windows. All scan parameters are in accompanying mansucript.

### Data Protocol
A specieal version of Comet was built for analysis of the coisolated SILAC peptide pair MS/MS data. Details for the Comet binary are in the accompanying manuscript. The search parameter to enable coisolation search is silac_pair_fragments =1 or 2 (2 is preferred for best search results). Static modification of +57 on cysteine and variable modifications of +16 (Oxidation) on methionine, +42 (Protein N-terminal Acetylation), and heavy lysine (either +8.0142 for Lys0:Lys8 or +1.9941 for Lys6:Lys8 which also requires static adjustment of +6 for the lysine mass). For analysis of MS/MS data, raw files were converted via msConvert to mzML and searched with Comet against the SGD yeast fasta database. Then our coiso_silac Python package read in Comet pin files, mzML, and Dinosaur MS1 features in order to filter PSMs at 1% FDR (via mokapot), read in MS/MS data with Pyteomics, and custom code to annotate MS/MS spectra, quantify SILAC ratios for MS1 mapped Dinosaur features and MS/MS, and generate outputs. Documentation for coiso_silac Python package is at https://gitlab.com/public_villenlab/coiso_silac and all data analysis and figures can be accessed on GitLab at https://gitlab.com/public_villenlab/coiso_silac_analysis.

### Publication Abstract
Stable-isotope labeling with amino acids in cell culture (SILAC)-based metabolic labeling is a widely adopted proteomics approach that enables quantitative comparisons among a variety of experimental conditions. Despite its quantitative capacity, SILAC experiments analyzed with data-dependent acquisition (DDA) do not fully leverage peptide pair information for identification and suffer from undersampling compared to label-free proteomic experiments. Herein, we developed a DDA strategy that coisolates and fragments SILAC peptide pairs and uses y-ions for their relative quantification. To facilitate the analysis of this type of data, we adapted the Comet sequence database search engine to make use of SILAC peptide paired fragments and developed a tool to annotate and quantify MS/MS spectra of coisolated SILAC pairs. This peptide pair coisolation approach generally improved expectation scores compared to the traditional DDA approach. Fragment ion quantification performed similarly well to precursor quantification in the MS1 and achieved more quantifications. Lastly, our method enables reliable MS/MS quantification of SILAC proteome mixtures with overlapping isotopic distributions. This study shows the feasibility of the coisolation approach. Coupling this approach with intelligent acquisition strategies has the potential to improve SILAC peptide sampling and quantification.

### Keywords
Yeast, Quantification, Comet, Peptide pairs, Silac, Ms/ms

### Affiliations
University of Washington
Department of Genome Sciences University of Washington Seattle, WA  USA

### Submitter
Ian Smith

### Lab Head
Dr Judit Vill√©n
Department of Genome Sciences University of Washington Seattle, WA  USA


