### Accession
PXD037241

### Title
Bacteriocin producing Enterococcus faecium

### Description
Proteomic Characterization of Bacteriocin-Producing Enterococcus faecium Strains from foodstuffs

### Sample Protocol
Proteins were obtained using an optimised phenol extraction protocol and digested with trypsin using filter-aided sample preparation (FASP).

### Data Protocol
The purified peptide mixtures were analysed by reversed-phase liquid chromatography coupled to a timsTOFâ„¢ Pro tandem mass spectrometer (Bruker Daltonik GmbH, Bremen), using a 120 min gradient. Proteome Discoverer Software (Thermo Fisher Scientific, Bremen, Germany, v1.4.1.14) converted the raw data into mascot generic files (MGFs).

### Publication Abstract
Enterococcus belongs to a group of microorganisms known as lactic acid bacteria (LAB), which constitute a broad heterogeneous group of generally food-grade microorganisms historically used in food preservation. Enterococci live as commensals of the gastrointestinal tract of warm-blooded animals, although they also are present in food of animal origin (milk, cheese, fermented sausages), vegetables, and plant materials because of their ability to survive heat treatments and adverse environmental conditions. The biotechnological traits of enterococci can be applied in the food industry; however, the emergence of enterococci as a cause of nosocomial infections makes their food status uncertain. Recent advances in high-throughput sequencing allow the subtyping of bacterial pathogens, but it cannot reflect the temporal dynamics and functional activities of microbiomes or bacterial isolates. Moreover, genetic analysis is based on sequence homologies, inferring functions from databases. Here, we used an end-to-end proteomic workflow to rapidly characterize two bacteriocin-producing Enterococcus faecium (Efm) strains. The proteome analysis was performed with liquid chromatography coupled to a trapped ion mobility spectrometry-time-of-flight mass spectrometry instrument (TimsTOF) for high-throughput and high-resolution characterization of bacterial proteins. Thus, we identified almost half of the proteins predicted in the bacterial genomes (&gt;1100 unique proteins per isolate), including quantifying proteins conferring resistance to antibiotics, heavy metals, virulence factors, and bacteriocins. The obtained proteomes were annotated according to function, resulting in 22 complete KEGG metabolic pathway modules for both strains. The workflow used here successfully characterized these bacterial isolates and showed great promise for determining and optimizing the bioengineering and biotechnology properties of other LAB strains in the food industry.

### Keywords
Enterococcus, Bacteriocins

### Affiliations
University of Santiago de Compostela
Laboratory of Food Hygiene and Control. Department of Analytical Chemistry, Nutrition and Food Science, School of Veterinary Sciences, University of Santiago de Compostela, Lugo, Spain.

### Submitter
Marcos Quintela-Baluja

### Lab Head
Dr Marcos Quintela-Baluja
Laboratory of Food Hygiene and Control. Department of Analytical Chemistry, Nutrition and Food Science, School of Veterinary Sciences, University of Santiago de Compostela, Lugo, Spain.


