### Accession
PXD019504

### Title
Global proteome analysis of paired colorectal cancer sample using LC-MS/MS

### Description
To find differentially expressed protein in colorectal cancer compared with normal colon tissue, we performed global proteome analysis using LC-MS.

### Sample Protocol
Proteomes were extracted from frozen tissue using urea-based lysis buffer. After reducing, alkylation, and digestion of samples, peptide samples were desalted by C18 StageTip.Desalted peptides were dried up and re-dissolved in 3% acetonitrile/0.1% formic acid. Desalted peptides were injected into a 50-cm C18 capillary column mounted to an Easy-nLC 1200 system coupled to an Orbitrap Fusion Lumos mass spectrometer (Thermo Scientific). MS data were acquired with an automatic switch between a full scan and 10 data-dependent MS/MS scans. The target value for full-scan MS spectra was 1Ã—10E6 charges in the 375-1500 m/z range with a maximum injection time of 50 ms and a resolution of 60,000 at 200 m/z in profile mode. Isolation of precursors was performed with a window of 1.4 m/z. Precursors were fragmented by higher-energy C-trap dissociation with a normalized collision energy of 30 eV. MS/MS scans were acquired at a resolution of 15,000 at 200 m/z with an ion target value of 5x10E4, maximum injection time of 100 ms, and dynamic exclusion for 15 s in centroid mode.

### Data Protocol
Label-free protein quantification was carried out with MaxQuant (version 1.6.4.0) and the Andromeda search engine. The first and the main maximum precursor mass tolerances were set to 20 and 6 ppm, respectively. The reference human proteome database was downloaded from UniProt (with updates up to Sept. 2018). The search assumed trypsin and Lys-C digestions with up to 2 missed cleavages. A minimum of 1 peptide was required for protein identification, but 2 peptides were required to calculate a protein level ratio. The modifications used as variable modifications for protein identification and quantification included oxidation of methionine, acetylation of the protein N-terminus, and deamidation of glutamine and asparagine. Significantly up-regulated and down-regulated proteins were identified with Perseus software. Note: Processed data(txt) include 2 unpaired-sample results.

### Publication Abstract
Colorectal cancer (CRC) is one of the most prevalent and lethal malignancies. Especially for early stage CRC, prognostic molecular markers are needed to guide therapy. In this study, we first extracted total proteomes from matched pairs of fresh cancer and benign mucosal tissues from 22 CRC patients. Global proteomic profiling with Fourier transform liquid chromatography-mass spectrometry sequencing and label free quantitation uncovered that P4HA1 (prolyl 4-hydroxylase alpha 1) was overexpressed in CRC relative to benign colonic mucosa. We then investigated expression by immunohistochemical staining with P4HA1-specific antibodies using CRC tissue microarrays. Independent validation cohorts of 599 cases of early stage CRC and 91 cases of late stage CRC were examined. Multivariate and univariate survival analyses revealed that high expression of P4HA1 protein was an independent poor prognostic marker for patients with early stage CRC, especially of the microsatellite stable subtype. Our study provides strong support for P4HA1 as a predictive protein marker for precision diagnostics, therapeutic decision-making, and drug development for early stage colorectal cancer and demonstrates the utility of proteomic profiling to identify novel protein biomarkers.

### Keywords
Human, Label-free, Cancer

### Affiliations
Department of pathology, Memorial Sloan Kettering Cancer Center
MSKCC

### Submitter
Atsushi Tanaka

### Lab Head
Dr Michael Roehrl
Department of pathology, Memorial Sloan Kettering Cancer Center


