### Accession
PXD026647

### Title
The unphosphorylated form of the PAQosome core subunit RPAP3 binds ribosomal preassembly complexes to modulate ribosome biogenesis.

### Description
The PAQosome (Particle for Arrangement of Quaternary structure) is a twelve-subunit HSP90 co-chaperone involved in the biogenesis of several human protein complexes. Two mechanisms of client selection have previously been identified, namely the selective recruitment of specific adaptors and the differential use of homologous core subunits. Here, we describe a third client selection mechanism by showing that RPAP3, one of the core PAQosome subunits, is phosphorylated at several Ser residues in HEK293 cells. Affinity purification coupled with mass spectrometry (AP-MS) using expression of tagged RPAP3 with single phospho-null mutations at Ser116, Ser119 or Ser121 reveals binding of the unphosphorylated form to several proteins involved in ribosome biogenesis. In vitro phosphorylation assays indicate that the kinase CK2 phosphorylates these RPAP3 residues. This finding is supported by data showing that pharmacological inhibition of CK2 enhances binding of RPAP3 to ribosome preassembly factors in AP-MS experiments. Moreover, silencing of PAQosome subunits interferes with ribosomal assembly factorsâ€™ interactome. Altogether, these results indicate that RPAP3 phosphate group addition/removal at specific residues modulates binding to subunits of preribosomal complexes and allows speculating that PAQosome posttranslational modifications is a mechanism of client selection.

### Sample Protocol
For FLAG-tagged affinity purification: PIH1D1-WT and RPAP3-WT or the various RPAP3 mutant (S116A, S116D, S119A, S119D, S121A and S121D) (cloned into the expression vector p3xFLAG-CMV-14) were transiently transfected in two wells of a 6-well plate containing HEK 293T cells at 60% confluency using jetPRIME according to manufacturerâ€™s instructions. The empty vector was used as a negative control. Cells were incubated in complete DMEM media (supplemented with 10% fetal bovine serum, 2â€‰mM glutamine and 100 U/mL penicillin and 100 Âµg/mL streptomycin) and harvested 48 h later.  PIH1D1-WT transfected cells were incubated with starved conditions (DMEM containing 1g/L glucose supplemented with 2â€‰mM glutamine and 100 U/mL penicillin and 100 Âµg/mL streptomycin). In indicated experiments, 10ÂµM of CX-4945 or 100ÂµM of TBB was added 32 h after RPAP3-WT transfection and cells were harvested at 48 h post transfection.  DMSO-treated cells were used as negative controls.  In the case of siRNA treated purifications, cells were grown to a confluence of 20% and transfected with SMARTpool siGenome siRNA targeting human RPAP3, human WDR92 or a non-targeting siRNA control at a concentration of 100â€‰nM. The following day, cells were cotransfected again with siRNAs and either the 3xFLAG-RRP1B or 3xFLAG-PDCD11 expression vector and harvested 48 h later. In every conditions, cell membranes were disrupted by two liquid nitrogen freeze-thaw cycles in 1 mL lysis buffer (100 mM KCl, 50 mM HEPES-KOH pH 8.0, 2 mM EDTA, 0.1% NP-40, 10% glycerol, 1 mM DTT and cOmplete EDTA-free Protease Inhibitor Cocktail). The protein extracts were cleared of insoluble material by centrifugation (12,000â€‰g, 30â€‰min, 4â€‰Â°C). 20ÂµL of anti-FLAG M2 magnetic bead slurry were washed twice in 1 mL of lysis buffer, then incubated for three hours with 1,5 to 2 mg of total proteins at 4â€‰Â°C on a tube rotator. The beads were then washed three times in 1 mL lysis buffer and three more times in 1 mL wash buffer (75 mM KCl, 50â€‰mM ammonium bicarbonate pH 8.0). The bound proteins were eluted by three successive 30 min incubations in 300 ÂµL of ammonium hydroxide solution pH 11-12 (roughly 10%) in HPLC-grade water. The combined fractions were dried in a speed-vac and then resuspended in 10 ÂµL of 6M urea. Reduction buffer was added at a volume of 2.5 ÂµL (45 mM DTT, 100 mM ammonium bicarbonate) and the samples were incubated for 30â€‰min at 37â€‰Â°C. An additional 2.5 ÂµL of alkylation buffer (100 mM iodoacetamide, 100 mM ammonium bicarbonate) were included to the mix followed by 20â€‰min incubation at 24â€‰Â°C and in absence on light. 20 ÂµL of HPLC-grade water were added to reduce urea concentration and trypsin digestion was performed using a 1:20 (enzyme:protein) ratio of sequencing grade modified trypsin (Promega, V5111) and 18 h incubation at 37â€‰Â°C on a ThermoMixer. Following a quick centrifugation (500 g, 1 min) the peptides were collected. For most samples, trifluoroacetic acid was added to the samples and residual salts and detergents were removed using an Oasis MCX 96-well Î¼Elution Plate loaded onto a Positive Pressure-96 Processor according to the manufacturerâ€™s instructions.  3XFLAG-PIH1D1 samples were re-solubilized under agitation for 15 min in 10 ÂµL of 0.2% formic acid. Desalting/cleanup of the digests was performed by using C18 ZipTip pipette tips. Eluates were dried down in vacuum centrifuge and then re-solubilized under agitation for 15 min in 12 ÂµL of 2% acetonitrile, 1% formic acid, of which 5 ÂµL were used for LC-MS/MS.

### Data Protocol
Mass spectrometry and data analysis: High performance liquid chromatography (HPLC) was performed using C18 resin (5 Âµm particles, 300 Ã… pores) extracted from a Jupiter LC Column (00B-4053-E0) and packed in a PicoFrit Column (NEW OBJECTIVE, PF360-75-15-N-5), the latter of which was loaded into a Easy-nLC II instrument (ThermoFisher). Employed buffers were: 0.2% formic acid (buffer A) and 100% acetonitrile/0.2% formic acid (buffer B). Peptide elution was performed with a two-slope gradient at a flowrate of 250â€‰nL/min. Solvent B gradually increased from 2 to 37% over the course of 90â€‰min and then from 37 to 80% B in 10â€‰min. The HPLC system was coupled to an Orbitrap Fusion Tribrid mass spectrometer (Thermo Scientific, IQLAAEGAAPFADBMBCX) through a nano-ESI source (ThermoFisher, ES071). Nanospray and S-lens voltages were set to 1.3â€“1.8â€‰kV and 50â€‰V, respectively. Capillary temperature was set to 225â€‰Â°C. Full scan MS survey spectra (m/z 360-1560) in profile mode were acquired in the Orbitrap with a resolution of 120,000 with a target value at 1e6. The 25 most intense peptide ions were fragmented in the HCD collision cell and analyzed in the linear ion trap with a target value at 2e4 and normalized collision energy at 28. Target ions selected for fragmentation were dynamically excluded for 25 sec. The peak list files were generated with Proteome Discoverer v2.3 (ThermoFisher, OPTON-30812) using the following parameters: minimum mass set to 500â€‰Da, maximum mass set to 6â€‰kDa, no grouping of MS/MS spectra, precursor charge set to auto, and minimum number of fragment ions set to 5. Mass spectrometry experimental condition used for 3X-FLAG-PIH1D1-digested sample were previously described in Cloutier et al, 2017.  Label-Free Quantification (LFQ) and normalization was performed using MaxQuant v1.6.2.10 and the characterized human UniProtKB database (release on June 3th 2018). Trypsin was chosen as digestion enzyme allowing a maximum of 2 miscleavages. Methionine oxidation and N-terminal acetylation where selected as variable modifications while cysteine carbamidomethylation was set as fixed modification. Fast LFQ was enabled and results were refined through the re-quantify option. Orbitrap was selected as instrument type with default parameters. Peptide identification was transferred across LC-MS runs based on similar m/z ratios and retention time using the â€œMatch between runsâ€� feature. Further statistical analysis was carried out with Perseus (version 1.6.1.3). All purifications were done in triplicate and proteins detected in two out of three experiments were kept for further analysis. Missing values were replaced by randomly generated intensities normally distributed with a width of 0.3 times and a downshift of 1.8 times the standard deviation of non-zero intensities. Significant differences between protein intensities from bait purifications and corresponding controls groups were then determined using a two-tailed T-test subsequently adjusted for multiple hypothesis testing with a permutation-based false discovery rate (FDR) of 0.05 and a fudge factor (s0) of 0.1 with 10,000 iterations. In the case of mutant comparaison with their WT counterpart (RPAP3 S116A/WT, RPAP3 S116D/WT, RPAP3 S119A/WT , RPAP3 S119D/WT, RPAP3 S121A/WT and RPAP3 S121D/WT), drug-treated or siRNA-treated samples purifications, proteins that did not show enrichment in either conditions compared to the empty vector control (i.e. p-values over 0.05 and/or protein intensity ratio under 2) were discarded. Differential co-purification of the resulting subset of proteins in either condition was then carried out as detailed for bait versus control experiments. FLAG-tagged PIH1D1 purified protein search was performed with Mascot 2.5 (Matrix Science) against the Refseq Human protein database (Txid 9606) and against the R2TP sequence. The peak list files were generated with Proteome Discoverer (version 2.3) using the following parameters: minimum mass set to 500 Da, maximum mass set to 6000 Da, no grouping of MS/MS spectra, precursor charge set to auto, and minimum number of fragment ions set to 5. The mass tolerances for precursor and fragment ions were set to 10 ppm and 0.6 Da, respectively. Trypsin was used as the enzyme allowing for up to 2 missed cleavage. Cysteine carbamidomethylation was specified as a fixed modification, and methionine oxidation, serine, threonine and tyrosine phosphorylation modifications as variable modifications. Data analysis was performed using Scaffold (version 4.7) to select peptide containing phosphorylated residue.  Spectrum was retrieve by using Tune software (Version 2.0).  Reference Cloutier P, Poitras C, Durand M, Hekmat O, Fiola-Masson Ã‰, Bouchard A, Faubert D, Chabot B, Coulombe B. R2TP/Prefoldin-like component RUVBL1/RUVBL2 directly interacts with ZNHIT2 to regulate assembly of U5 small nuclear ribonucleoprotein. Nat Commun. 2017 May 31;8:15615. doi: 10.1038/ncomms15615. PMID: 28561026.

### Publication Abstract
The PAQosome (particle for arrangement of quaternary structure) is a 12-subunit HSP90 co-chaperone involved in the biogenesis of several human protein complexes. Two mechanisms of client selection have previously been identified, namely, the selective recruitment of specific adaptors and the differential use of homologous core subunits. Here, we describe a third client selection mechanism by showing that RPAP3, one of the core PAQosome subunits, is phosphorylated at several Ser residues in HEK293 cells. Affinity purification coupled with mass spectrometry (AP-MS) using the expression of tagged RPAP3 with single phospho-null mutations at Ser116, Ser119, or Ser121 reveals binding of the unphosphorylated form to several proteins involved in ribosome biogenesis. <i>In vitro</i> phosphorylation assays indicate that the kinase CK2 phosphorylates these RPAP3 residues. This finding is supported by data showing that pharmacological inhibition of CK2 enhances the binding of RPAP3 to ribosome preassembly factors in AP-MS experiments. Moreover, the silencing of PAQosome subunits interferes with ribosomal assembly factors' interactome. Altogether, these results indicate that RPAP3 phosphate group addition/removal at specific residues modulates binding to subunits of preribosomal complexes and allows speculating that PAQosome posttranslational modification is a mechanism of client selection.

### Keywords
Ribosome preassembly factors, Rpap3, Ap-ms, Ribosome biogenesis, Paqosome, Ck2 phosphorylation, Chaperone

### Affiliations
Laboratory of Gene Transcription and Proteomics Discovery Platform, Institut de recherches cliniques de Montréal, 110 avenue des Pins Ouest, Montréal, QC H2W 1R7, Canada
[1] Translational Proteomics Laboratory, Institut de recherches cliniques de MontrÃ©al (IRCM), 110 avenue des Pins ouest, MontrÃ©al, QuÃ©bec H2W 1R7, Canada. [2] Department of Biochemistry, UniversitÃ© de MontrÃ©al, Pavillon Roger-Gaudry, CP 6128, Succ Centre-Ville, Montreal, QuÃ©bec H3C 3J7, Canada.

### Submitter
Christian Poitras

### Lab Head
Dr Benoit Coulombe
[1] Translational Proteomics Laboratory, Institut de recherches cliniques de MontrÃ©al (IRCM), 110 avenue des Pins ouest, MontrÃ©al, QuÃ©bec H2W 1R7, Canada. [2] Department of Biochemistry, UniversitÃ© de MontrÃ©al, Pavillon Roger-Gaudry, CP 6128, Succ Centre-Ville, Montreal, QuÃ©bec H3C 3J7, Canada.


