### Accession
PXD035426

### Title
Mapping diversity in African trypanosomes using high resolution spatial proteomics

### Description
African trypanosomes are dixenous eukaryotic parasites that impose a significant human and veterinary disease burden on sub-Saharan Africa. Diversity between species and life-cycle stages is concomitant with distinct host and tissue tropisms within this group. Here, the spatial proteomes of two African trypanosome species, Trypanosoma brucei and Trypanosoma congolense, have been mapped, each in mammalian and insect life-stages represented by bloodstream form (BSF) and procyclic form (PCF) respectively. Using the hyperLOPIT (hyperplexed localisation of organelle proteins by isotope tagging) methodology, this work has provided four highly comprehensive spatial proteomes.

### Sample Protocol
T. brucei and T. congolense BSF and PCF cells were harvested and lysed by nitrogen cavitation to release subcellular compartments. Soluble proteins and crude membranes were separated on an iodixanol cushion followed by pelleting of the membranous fraction, which was then resolved by equilibrium density gradient centrifugation using a pre-formed iodixanol gradient. Fractions were collected and pooled to 10 per experiment, which, in addition to the cytosolic-enriched soluble fraction, formed the basis for 11-plex TMT labelling. TMT labelled pools were combined and subjected to high pH UPLC fractionation prior to LC-SPS-MS3 analysis on an Orbitrap Fusion Lumos Tribrid. Three biologically independent experimental iterations were carried out per cell-type.

### Data Protocol
Raw files from LC-MS analysis were analysed with ProteomeDiscoverer (v2.4, Thermo Fisher Scientific) configured with the Mascot (v2.7.0, Matrix Science) search engine using the sequence database selected according to the species. The T. brucei sequence database was formed by downloading protein sequences from TriTrypDB (v50) for T. brucei TREU927 and supplementing it with the sequences for BES40 genes from T. brucei Lister 427 from TriTrypDB (v50). The T. congolense sequence database was formed by downloading sequences from TriTrypDB (v50) for T. congolense IL3000 2019. Both databases were supplemented with (i) common contaminants from an expanded list of the GPM (Global Proteome Machine) cRAP (common Repository of Adventitious Proteins) database downloaded from CamProtR (January 2021 version â€“ downloaded 29/03/21, cambridgecentreforproteomics.github.io/camprotR/articles/crap.html) with the sequence for Benzonase (P13717, UniProt) manually appended, and (ii) common serum proteins from the cRFP (common Repository of FBS Proteins) database (Shin et al., 2019, Journal of Proteome Research). For all analyses a precursor error tolerance of 10 ppm and mass fragment error tolerance 0.6 Da were selected, reporter ions were quantified using the most confident centroid method, and trypsin was set as the enzyme of choice with a maximum of 2 missed cleavages permitted. Static modifications included: Carbamidomethyl (C), TMT6plex (N-term), and TMT6plex (K). Dynamic modifications included: Deamidated (NQ), Oxidation (M), TMT6plex (S), and TMT6plex (T). Percolator was used to assess the FDR (false discovery rate) and only high-confidence peptides were retained.

### Publication Abstract
African trypanosomes are dixenous eukaryotic parasites that impose a significant human and veterinary disease burden on sub-Saharan Africa. Diversity between species and life-cycle stages is concomitant with distinct host and tissue tropisms within this group. Here, the spatial proteomes of two African trypanosome species, Trypanosoma brucei and Trypanosoma congolense, are mapped across two life-stages. The four resulting datasets provide evidence of expression of approximately 5500 proteins per cell-type. Over 2500 proteins per cell-type are classified to specific subcellular compartments, providing four comprehensive spatial proteomes. Comparative analysis reveals key routes of parasitic adaptation to different biological niches and provides insight into the molecular basis for diversity within and between these pathogen species.

### Keywords
Protein localisation, Spatial proteome, Trypanosoma congolense, African trypanosomes, Hyperlopit, Trypanosoma brucei, Parasite diversification, Speciation

### Affiliations
School of Biological Sciences, University of Bristol
University of Cambridge

### Submitter
Nicola Moloney

### Lab Head
Dr Paula MacGregor
School of Biological Sciences, University of Bristol


