### Accession
PXD020611

### Title
S. cerevisiae engineered chassis for dicarboxylic acid production

### Description
Chassis strain suitable for producing multiple compounds is a central concept in synthetic biology. Design of a chassis using computational, first-principle, models is particularly attractive due to the predictability and control it offers, including against phenotype reversal due to adaptive mutations. Yet, the theory of model-based chassis design has not been put to experimental test. Here, we report two Saccharomyces cerevisiae chassis strains for dicarboxylic acid production based on genome-scale metabolic modelling. The chassis strain, harboring gene knockouts in serine biosynthesis and in pentose-phosphate pathway, is geared for higher flux towards three target products - succinate, fumarate and malate - but does not appreciably secrete any. Introducing modular product-specific mutations resulted in improved secretion of the corresponding acid as predicted by the model. Adaptive laboratory evolution of the chassis-derived producer cells further improved production for succinate and fumarate attesting to the evolutionary robustness of the underlying growth-product coupling. In the case of malate, which exhibited decreased production during evolution, the multi-omics analysis revealed flux bypass at peroxisomal malate dehydrogenase not accounted in the model. Transcriptomics, proteomics and metabolomics analysis showed overall concordance with the flux re-routing predicted by the model. Together, our results provide experimental evidence for model-based design of microbial chassis and have implications for computer-aided design of microbial cell factories.

### Sample Protocol
Samples were collected at mid-exponential growth phase, 10 mL of fermentation broth was transferred into ice-cold 15 mL FalconÂ® and immediately centrifuged at 10000xg for 2 min at 0Â°C. After centrifugation supernatant was discarded and cell pellet was washed once with ice-cold PBS buffer. After centrifugation, both cell pellet were snap-frozen in liquid nitrogen and kept at -80Â°C until extraction. Frozen cell pellets were lysed using 0.1% RapiGest in 100 mM ammonium bicarbonate. Three cycles of sonication (1 cycle: 15 seconds sonication, 15 seconds on ice) (Cell disruptor, Sonifier, Branson) were applied to the lysate, followed by 15 minutes bead beating using Precellys Lysing Kit (KT0361-1-004.2). Cell lysate was transferred into a new tube after centrifugation (5 minutes, 5000 x g) and incubated at 80Â°C for 15 minutes. Benzonase (25U, Merck) was added to the lysate and incubated for 30 minutes at 37Â°C. Cysteines were reduced using 10mM of dithiothreitol (56Â°C, 30 minutes). The sample was cooled to 24 Â°C and alkylated with 10mM of iodacetamide (room temperature, in the dark, 30 minutes). Proteins were precipitated with TCA, pellet was washed by acetone and dried. The proteins were digested in 50 mM HEPES (pH 8.5) using LysC (Wako) with an enzyme to protein ration 1:50 at 37Â°C for 4 hours, followed by trypsin (Promega) with an enzyme to protein ratio 1:50 at 37Â°C overnight.  TMT10plexâ„¢ Isobaric Label Reagent (ThermoFisher) was added to the samples according the manufacturers instructions. Labeled peptides were cleaned up using OASISÂ® HLB ÂµElution Plate (Waters). Offline high pH reverse phase fractionation was performed using an Agilent 1200 Infinity high-performance liquid chromatography (HPLC) system, equipped with a Gemini C18 column (3 Î¼m, 110 Ã…, 100 x 1.0 mm, Phenomenex).

### Data Protocol
Proteomics data analysis was performed in R using default limma package conditions. Only proteins that were quantified with two unique peptides matches were accounted. Under these conditions, a total of 3305 proteins were quantified and used to calculate differential protein abundances between tested strains.

### Publication Abstract
None

### Keywords
Chassis cell, Yeast, Adaptive laboratory evolution

### Affiliations
EMBL
Structural and Computational Biology Unit, European Molecular Biology Laboratory, Heidelberg, Germany; The Medical Research Council Toxicology Unit, University of Cambridge, Cambridge, UK

### Submitter
Frank Stein

### Lab Head
Dr Kiran R. Patil
Structural and Computational Biology Unit, European Molecular Biology Laboratory, Heidelberg, Germany; The Medical Research Council Toxicology Unit, University of Cambridge, Cambridge, UK


