### Accession
PXD017300

### Title
LSD1 Prevents Aberrant Heterochromatin Formation in Neurospora crassa

### Description
Heterochromatin is a specialized form of chromatin that restricts access to DNA and inhibits genetic processes, including transcription and recombination. In Neurospora crassa, constitutive heterochromatin is characterized by trimethylation of lysine 9 on histone H3, hypoacetylation of histones, and DNA methylation. Here we explore whether the conserved histone demethylase, lysine-specific demethylase 1 (LSD1), regulates heterochromatin in Neurospora, and if so, how. Though LSD1 is implicated in heterochromatin regulation, its function is inconsistent across different systems; orthologs of LSD1 have been shown to either promote or antagonize heterochromatin expansion by removing H3K4me or H3K9me respectively. We identify three members of the Neurospora LSD complex (LSDC): LSD1, PHF1, and BDP-1, and strains deficient for any exhibit variable spreading of heterochromatin and establishment of new heterochromatin domains dispersed across the genome. Heterochromatin establishment outside of canonical domains in Neurospora share the unusual characteristic of DNA methylation-dependent H3K9me3; typically, H3K9me3 establishment is independent of DNA methylation. Consistent with this, the hyper-H3K9me3 phenotype of LSD1 knock-out strains is dependent on the presence of DNA methylation, as well as HCHC-mediated histone deacetylation, suggesting spreading is dependent on some feedback mechanism. Altogether, our results suggest LSD1 works in opposition to HCHC to maintain proper heterochromatin boundaries.

### Sample Protocol
Neurospora crassa culturing, protein extraction, incubations and enrichments with bait proteins have been previously described: Honda, S. and Selker, E.U., 2009. Tools for fungal proteomics: multifunctional neurospora vectors for gene replacement, protein expression and protein purification. Genetics, 182(1), pp.11-23. Baits and sample key: Lsd1 bait: HON1294_6, Bdf1 bait: HON1374_Bdf1_1Dband13, and Phf1 bait: HON1374_Phf1_1Dband12.  Protein samples were run for 5 minutes into a 10% Bis-Tris Gel to remove impurities. Gel bands were excised after Coomassie-staining, washed twice for 30 min in 50 mM ammonium bicarbonate and 50% (v/v) acetonitrile, reduced in 10 mM DTT/100 mM ammonium bicarbonate, and alkylated in 55mM iodoacetamide and 100mM ammonium bicarbonate. Dehydrated gels were reswelled in digestion buffer containing 50 mM ammonium bicarbonate, 5 mM CaCl2, and 12.5 ng/mL sequencing grade modified trypsin (Promega). Following an overnight incubation at 37C, extracted peptides were identified by tandem mass spectrometry using LTQ Velos linear ion trap mass spectrometers (ThermoFisher). Peptides were separated using an Agilent 1100 series capillary LC system (Agilent Technologies) and 0.5 3 250mm Zorbax SB-C18 column (Agilent Technologies). Data-dependent collection of MS/MS spectra used the dynamic exclusion feature of the instrumentâ€™s control software (repeat count equal to 1, exclusion list size of 100, exclusion duration of 30 sec, and exclusion mass width of -1 to +4) to obtain MS/MS spectra of the five most abundant parent ions (minimum signal of 10,000) following each survey scan from m/z 350-2000. There was no averaging of microscans, a maximum MS1 inject time of 200 msec, a maximum MS2 inject time of 100 msec, and automatic gain control targets of 30,000 in MS1 mode and 10,000 in MS2 mode.

### Data Protocol
For data analysis, databases of Neurospora crassa sequences was downloaded from www.uniprot.org in June 2012 (for HON1294) or April 2013 (HON1374) and appended to 179 common contaminants. We used an appended sequence-reversed decoy database to estimate error thresholds, giving a total of 26,996 entries (HON1294) or 27,010 entries (HON1374). The database processing was performed with python scripts available at https://github.com/pwilmart/fasta_utilities. SEQUEST (version 28, revision 12, Thermo Fisher) searches for all samples were performed with trypsin specificity. Average parent ion mass tolerance was 2.5 Da. Average fragment ion mass tolerance was 1.0 Da. The ion series used in scoring were b and y. A static modification of +57 Da was added to all cysteine residues. A variable modification of +16 Da on methionine residues was also allowed with a maximum of 3 modifications per peptide. SEQUEST scores were combined into linear discriminant function scores to improve identification sensitivity, and discriminant score histograms created separately for each peptide charge state (1+, 2+, and 3+). Separate histograms were created for matches to forward sequences and for matches to reversed sequences for all peptides of 7 amino acids or longer. False discovery rates (estimated from decoy counts) were less than 5% for both peptide spectrum matches and proteins. Python scripts used for processing can be found at https://github.com/pwilmart/PAW_pipeline.

### Publication Abstract
Heterochromatin is a specialized form of chromatin that restricts access to DNA and inhibits genetic processes, including transcription and recombination. In Neurospora crassa, constitutive heterochromatin is characterized by trimethylation of lysine 9 on histone H3, hypoacetylation of histones, and DNA methylation. We explored whether the conserved histone demethylase, lysine-specific demethylase 1 (LSD1), regulates heterochromatin in Neurospora, and if so, how. Though LSD1 is implicated in heterochromatin regulation, its function is inconsistent across different systems; orthologs of LSD1 have been shown to either promote or antagonize heterochromatin expansion by removing H3K4me or H3K9me respectively. We identify three members of the Neurospora LSD complex (LSDC): LSD1, PHF1, and BDP-1. Strains deficient for any of these proteins exhibit variable spreading of heterochromatin and establishment of new heterochromatin domains throughout the genome. Although establishment of H3K9me3 is typically independent of DNA methylation in Neurospora, instances of DNA methylation-dependent H3K9me3 have been found outside regions of canonical heterochromatin. Consistent with this, the hyper-H3K9me3 phenotype of &#x394;lsd1 strains is dependent on the presence of DNA methylation, as well as HCHC-mediated histone deacetylation, suggesting that spreading is dependent on some feedback mechanism. Altogether, our results suggest LSD1 works in opposition to HCHC to maintain proper heterochromatin boundaries.

### Keywords
Lysine-specific demethylase, Heterochromatin, Neurospora crassa, Immunoprecipitation mass spectrometry

### Affiliations
Dr. Eric U. Selker Institute of Molecular Biology University of Oregon Eugene, Oregon 97403, USA
OHSU

### Submitter
Phillip Wilmarth

### Lab Head
Dr Eric U. Selker
Dr. Eric U. Selker Institute of Molecular Biology University of Oregon Eugene, Oregon 97403, USA


