### Accession
PXD015828

### Title
Total proteome analysis of WT vs TRAP1 KO in HCT-116 and HEK-293T cells grown on different carbon sources.

### Description
The molecular chaperone TRAP1, the mitochondrial isoform of cytosolic HSP90, remains poorly understood with respect to its pivotal role in the regulation of mitochondrial metabolism. To characterize the impact of TRAP1 in maintaining the cellular proteome under different energy sources we carried out a label free quantitation (LFQ) MS analysis of the total cellular proteome with WT and KO HEK293T and HCT116 cells cultured with the three different cocktails of carbon sources (Glc + Pyr + Gln, Gal + Pyr only, Gln only). Linked manuscript : Abhinav Joshi, Li Dai, Yanxin Liu, Jungsoon Lee, Nastaran Mohammadi Ghahhari, Gregory Segala, Kristin Beebe, Francis T.F. Tsai, David A. Agard, Len Neckers, and Didier Picard* (*corresponding author).

### Sample Protocol
TRAP1 KO HEK293T and HCT116 cells were generated using CRISPR/Cas9 genome editing as described in the article ( Additional file 1: Figure S1A). Cellular foci were then individually picked, subcultured and finally analyzed by immunoblotting with antibodies against TRAP1 to identify clones that were devoid of the protein. Before any single carbon source OCR experiment, the cells were grown overnight in medium with the carbon source to be tested in order to acclimatize and to stabilize them metabolically. The carbon sources were added to DMEM lacking Glc, Pyr and Gln with 10% FBS, 100 u/ml penicillin and 100 µg/ml streptomycin as follows: (i) Glc only: 4.5 g/l glucose; (ii) Gln only: 2 mM glutamine; (iii) Pyr only: 1 mM sodium pyruvate; (iv) Gal and Pyr: 10 mM galactose, 1 mM sodium pyruvate. Three biological replicates of 9 x 106 WT and KO HEK293T and HCT116 cells grown in different carbon source cocktails (Glc + Pyr + Gln, Gal+ Pyr and Gln only) were pooled together and lysed in FASP lysis buffer (100 mM Tris-HCl pH 7.5, 4% SDS, 10 mM TCEP) at 95 ºC for 5 min followed by centrifugation at 14,000 g for 10 min. 100 µg of each clarified sample were digested by the FASP method (Wiśniewski et al 2008). 50 µg of the resulting peptide mixtures were desalted on Waters SEP-PAK C18 micro elution plates and eluted with 100 μl of 40% acetonitrile, 0.1% formic acid. 6 μl of the eluate were used for LC-MS/MS analysis. Samples were analyzed on a Fusion orbitrap trihybrid mass spectrometer  interfaced via a nanospray source to a Dionex RSLC 3000 nanoHPLC system (Thermo Fisher Scientific, Bremen, Germany). Peptides were separated on a custom packed reversed-phase C18 nanocolumn (75 μm ID × 40 cm, 1.8 μm particles, Reprosil Pur, Dr. Maisch) with a gradient from 5 to 55% acetonitrile in 0.1% formic acid in 120 min. Full MS survey scans were performed at 120,000 resolution. All survey scans were internally calibrated using the 445.1200 background ion mass. In data-dependent acquisition controlled by Xcalibur 2.1 software (Thermo Fisher), a maximum number of multi-charged precursor ions was selected for tandem MS analysis within a maximum cycle time of 3.0s. Precursor ions selected with an isolation window of 1.6 m/z were fragmented by HCD fragmentation and analysed in the linear trap. Selected ions were then dynamically excluded from further selection during 120s.

### Data Protocol
Tandem MS data were processed by the MaxQuant software (1.5.3.30) (Cox and Mann, 2008) incorporating the Andromeda search engine (Cox et al, 2011). The UniProt human proteome database of January  2017 was used (UP000005640_9606.fasta , 20’987 sequences), supplemented with sequences of common contaminants. Trypsin (cleavage at K,R) was used as the enzyme definition, allowing 2 missed cleavages. Carbamidomethylation of cysteine was specified as a fixed modification. N-terminal acetylation of protein and oxidation of methionine were specified as variable modifications. Mass tolerances applied were at 4 ppm for precursors (after recalibration by MaxQuant) and 0.5 Da for ion trap MS2 fragments. All identifications were filtered at 1% FDR at both the peptide and protein levels with default MaxQuant parameters (see parameters.txt and mqpar.xml files). For protein quantitation the LFQ label-free values determined using the “match between runs” function (Cox et al, 2014)  were used. MaxQuant data were further processed with Perseus software (Tyanova et al, 2016) for the filtering, log2-transformation and normalization of values and the statistical analyses and GO annotations.

### Publication Abstract
None

### Keywords
Mitochondrion, Hsp90, Trap1, Oxidative phosphorylation, Molecular chaperone

### Affiliations
University of Lausanne
Département de Biologie Cellulaire, Université de Genève

### Submitter
Manfredo Quadroni

### Lab Head
Dr Didier Picard
Département de Biologie Cellulaire, Université de Genève


