### Accession
PXD014799

### Title
PRMT1 controls the Senescence Associated Secretory Phenotype in Replication Stress Response

### Description
The protein arginine methyltransferase PRMT1 is overexpressed in various human cancers and linked to poor response to therapy. Although various inhibitors targeting its activity are under development, we still do not fully understand how PRMT1 is involved in the cellular processes underpinning tumorigenesis and chemoresistance. Mass spectrometryÃ¢â‚¬â€œbased proteomics revealed PRMT1 as a regulator of global arginine methylation changes in response to replicative stress in cancer cells. We show that, upon cisplatin, DNA-dependent protein kinase binds to- and phosphorylates PRMT1, inducing its chromatin recruitment and redirecting its enzymatic activity from soluble protein targets towards the histone substrate Arg3 of histone H4 (H4R3). On chromatin, DNA-PK/PRMT1 axis induces the Senescence-Associated Secretory Phenotype through the deposition of H4R3me2a at the pro-inflammatory gene promoters and by sustaining p65 binding to chromatin. Finally, PRMT1 inhibition reduced the clonogenic outgrowth of ovarian cancer cells exposed to low doses of CDDP and sensitized them to apoptosis. While unravelling a novel role of PRMT1 in replication stress response, our findings suggest the opportunity of targeting PRMT1 to sensitize cancer cells to genotoxic chemotherapeutics.

### Sample Protocol
Arginine methyl-peptides enrichment: For the standard SILAC, equal numbers of Light and Heavy-labelled SK-OV-3 cells were mixed in a 1:1 ratio and washed twice with PBS. For the triple SILAC, equal numbers of Light, Medium and Heavy-labelled SK-OV-3 cells were mixed in a 1:1:1 ratio and washed twice with PBS. Cell pellets were lysed in urea lysis buffer (9 M urea, 20 mM HEPES ph 8.0) supplemented with 1X Roche proteases and phosphatases inhibitors, sonicated and cleared by ultracentrifugation (20.000 x g for 15 min at 15Â°C). For in-solution digestion, 50 mg of proteins were reduced by adding 4.5 mM DTT (Sigma-Aldrich) for 30 min at 55Â°C, alkylated with 5.5 mM iodoacetamide (IAA: 10% v/v for 15 min at room temperature in the dark, Sigma Aldrich) and digested overnight with sequencing grade trypsin (1:100 w/w, Promega) after a 4x dilution in 25mM ammonium bicarbonate solution. Protease digestion was terminated by addition of trifluoroacetic acid to pH < 3. Precipitated material was removed by centrifugation for 15 min at 1780g at room temperature. In the triple SILAC, peptides were purified using reversed-phase Sep-Pak C18 cartridges (Waters, Milford, MA) and eluted off the Sep-Pak with 40% ACN with a subsequent step of removal of acetonitrile by 48 hours lyophilisation. Lyophilised peptides were dissolved in 25 mM NH4OH and subsequently off-line fractionated by High-pH fractionation using a Phenomenex JupiterÂ® C12 4 Âµm Proteo 90, LC 250 x 4.6 mm, Ea column on an AKTA-FPLC system operating at 1 ml/min. Buffer A was 25 mM NH4OH and Buffer B was 25 mM NH4OH in 90% ACN. Fractions were collected at 1-min intervals. Samples were initially loaded onto the column at 1 ml/min for 3 min, after which the fractionation gradient was as follows: 5% B to 30% B in 60 min, 30% B to 60% in 2 min and ramped to 70% B for 3 min. The total number of fractions concatenated was set to 14 throughout all experiments. After lyophilisation, each fraction was dissolved in 250ul of 1x immuno-Affinity Purification Buffer (IAP buffer, #9993, Cell Signaling Technologies) and subjected to two consecutive steps of methyl-R peptides enrichment using the ADMA antibody-conjugated beads [adme-R] Kit #13522, Cell Signaling Technologies) and MMA antibody-conjugated beads (PTMScan Mono-Methyl Arginine Motif [mme-RG] Kit #12235, Cell Signaling Technologies) following the manufacturerÃ¢â‚¬â„¢s instruction. After peptides incubation with the antibody conjugated beads for 2 hours at 4 Ã‚Â°C, the immuno-precipitates were washed twice in ice-cold IAP buffer followed by three washes in water; then, bound methyl-peptides were eluted with 2 Ãƒâ€” 50 ÃŽÂ¼l 0.15% TFA. Peptide eluates were desalted on reversed phase C18 StageTips and subjected to a second round of trypsin digestion prior to LC-MS/MS analysis. PRMT1 Protein Immunoprecipitation: 3 biological replicates of PRMT1 co-IP were performed, each using 2 mg of SK-OV-3 nuclear lysate as input, prepared as follows. Briefly, 100x106 SK-OV-3 cells were harvested, washed twice with cold PBS and re-suspended in 2 volumes of Lysis Buffer A (10 mM Hepes KOH pH 7.9, 1.5 mM MgCl2, 10 mM KCl, 0.2% NP40, 1X Roche Protease Inhibitors). After 20 strokes with a dounce homogenizer, cells were centrifuged 15 min at 3750 rpm. The supernatant was collected and the pellet was washed twice with PBS and re-suspended into 2 volumes of Buffer C (420 mM NaCl, 0.1% NP40, 0.5 mM DTT, 20 mM Hepes KOH pH 7.9, 2 mM MgCl2, 0.2 mM EDTA and 20% glycerol, 1X Roche Protease Inhibitors). The suspension was rotated 1h at 4Â°C and then ultra-centrifuged at 33000 rpm for 1h. After dilution of buffer C to 150 mM NaCl (IP Buffer), the lysate was rotated at 4Â° overnight with 5 ug of anti-PRMT1 antibody (ab73246), in presence and absence of 50 fold molar excess of the specific competing peptide (ab73687). G-protein-coupled magnetic beads (Dynabeads, Invitrogen, 100.04D) were then added and incubated for 2 hours; the captured immuno-complexes were washed 4 times with the IP Buffer and incubated with LSD sample buffer supplemented with 100 mM DTT in order to elute the immuno-precipitated proteins. Proteins were separated on 4-12% Bis-Tris acrylamide SDS-PAGE pre-cast gels (Novex Tris-Glycine Gels, Life technologies) and visualized using Colloidal Coomassie staining (Instant Blue, Expedeon), prior to protein digestion and LC-MS/MS analysis.

### Data Protocol
Nano-LC-MS/MS Analysis: Peptide mixtures were analysed by online nano-flow liquid chromatography tandem mass spectrometry using an EASY-nLCÃ¢â€žÂ¢ 1000 (Thermo Fisher Scientific, Odense, Denmark) connected to a Q-Exactive instrument (Thermo Fisher Scientific) through a nano-electrospray ion source. The nano-LC system was operated in one column set-up with a 50 cm analytical column (75-ÃŽÂ¼m inner diameter, 350-ÃŽÂ¼m outer diameter) packed with C18 resin (EasySpray PEPMAP RSLC C18 2M 50 cm x 75 M, Thermo Fisher Scientific) configuration. Solvent A was 0.1% formic acid (FA) and solvent B was 0.1% FA in 80% ACN. Samples were injected in an aqueous 0.1% TFA solution at a flow rate of 500 nL/min. SILAC and hmSILAC immuno-enriched methyl-peptides were separated with a gradient of 5Ã¢â‚¬â€œ40% solvent B over 90 min followed by a gradient of 40Ã¢â‚¬â€œ60% for 10 min and 60Ã¢â‚¬â€œ80% over 5 min at a flow rate of 250 nL/min in the EASY-nLC 1000 system. Label-free peptides from PRMT1 immunoprecipitation were analysed in technical triplicates using a top-15 method with a linear 90 min gradient from 5 to 30% of solvent B, a MS1 resolution of 60,000 at m/z 400 and an accumulation time of 80msec for MS/MS. The Q-Exactive was operated in the data-dependent mode (DDA) to automatically switch between full scan MS and MSMS acquisition. Survey full scan MS spectra (from m/z 300-1150) were analysed in the Orbitrap detector with resolution R=35,000 at m/z 400. The ten most intense peptide ions with charge states Ã¢â€°Â¥2 were sequentially isolated to a target value of 3e6 and fragmented by Higher Energy Collision Dissociation (HCD) with a normalized collision energy setting of 25%. The maximum allowed ion accumulation times were 20ms for full scans and 50ms for MSMS and the target value for MSMS was set to 1e6. The dynamic exclusion time was set to 20s.  Acquired raw data were analysed using the integrated MaxQuant software v1.6.2.10, using the Andromeda search engine. In MaxQuant, the estimated false discovery rate (FDR) of all peptide identifications was set to a maximum of 1%. The main search was performed with a mass tolerance of 6 ppm. Enzyme specificity was set to Trypsin/P. A maximum of 3 missed cleavages was permitted, and the minimum peptide length was fixed at 6 amino acids. Carbamidomethylation of Cysteine was set as a fixed modification. The September 2018 (2018_07) version of the Uniprot sequence database was used for peptide identification. To assign and quantify SILAC methyl-peptides, N-terminal acetylation, Methionine oxidation, mono-methyl-K/R, di-methyl-K/R were set as variable modifications.

### Publication Abstract
None

### Keywords
Prmt1, Replication stress response, Sasp, Arginine methylation

### Affiliations
Department of Experimental Oncology, IEO, European Institute of Oncology IRCCS, Milan 20139, Italy
European Institute of Oncology IRCSS, Department of Experimental Oncology, Milan, Italy

### Submitter
Enrico Massignani

### Lab Head
Dr Tiziana Bonaldi
Department of Experimental Oncology, IEO, European Institute of Oncology IRCCS, Milan 20139, Italy


