### Accession
PXD020992

### Title
A structural model of the human mSWI/SNF (BAF) complex bound to the nucleosome informs disease mechanisms

### Description
Mammalian SWI/SNF complexes are multi-component ATP-dependent chromatin-remodeling complexes that regulate genomic architecture. Here we present the first structural model of the human canonical BAF chromatin-remodeling complex bound to a nucleosome generated using cryo-EM, cross-linking mass spectrometry, and homology modeling. Endogenously-purified BAF tethers to the nucleosome H2A/H2B acidic patch regions bilaterally through the SMARCB1 C-terminal  helix and a SMARCA4 C-terminal post-SnAc region, each of which are mutated in disease and disrupt nucleosome remodeling. We describe the structural organization of the BAF core module, a connecting ARP module and the ATPase module, scaffolded by the SMARCA4/2 ATPase subunits. Importantly, we assign and model disease-associated mutations throughout the entire BAF complex, identifying mutations that break identified subunit–nucleosome contacts and subunit–subunit interfaces of BAF in the nucleosome-bound conformation. Taken together, this comprehensive structural model of the human BAF complex provides the first insights into the functional impact of mutations that cause cancer and other diseases.

### Sample Protocol
Eluted BAF complex was dialyzed overnight against a 25 mM HEPES pH 7.5, 100 mM KCl, 1 mM MgCl2, 0.1 mM EDTA, 5% Sucrose, 1 mM DTT) and mixed with a 3-fold molar excess of the mononucleosomes. The mixture was supplemented with 1 mM AMP-PNP (Sigma), incubated for 2 hours at 4°C. Sample was fixed with 3mM BS3 for 3 hours and loaded on top of a linear, 11-ml 10-40% sucrose gradient in 20 mM HEPES, pH 7.5, 100 mM KCl, 1 mM MgCl2, 0.1 mM EDTA, 1 mM DTT. Centrifugation was carried out in a SW41 rotor at 37,000 rpm for 16 hours at 4°C. 550-μL fractions were manually collected from the top of the gradient and quenched with Tris-HCl, pH 7.5 at a final concentration of 50 mM. Fractions containing BAF–nucleosome complex were concentrated in an Amicon Ultra concentrator (30-kDa cutoff; Millipore).   Sample was prepared according to previously described preparation protocol using SP3 beads (Hughes et al., 2014): 10uL of SP3 beads (10ug/uL ) and an equal volume of acetonitrile were added to the crosslinked samples and incubated at 60º C for 30 minutes with shaking. Due to high concentration of sucrose, the acetonitrile and sucrose solutions were separated into two phases. The beads were concentrated with a magnet and washed with100% acetonitrile. The beads were then suspended in 100 uL 8M Urea in 1 M ammonium bicarbonate and treated with TECP/CAA for 1 hr at 37º C in the dark. The solution for SP3 enrichment was evaporated and suspended in 8M Urea in 1 M ammonium bicarbonate and treated with TECP/CAA for 1 hr at 37º C in the dark as well. Then both samples were diluted 10 times with water and digested by addition of trypsin (20:1, protein:trypsin) overnight at 37º C.   The digested peptides were desalted by passage over C18 cartridges (The Nest group, Southborough, MA),  and dried by Speed-Vac. The peptides from both digests were resuspended in 50 uL Buffer A (25 mM ammonium formate, 20% acetonitrile, 0.1% formic acid, pH 2.8) and combined. 1 ug of sample was reserved for direct MS analysis and the remaining sample was fractionated using an in-house prepared microcapillary strong cation exchange column (200 mm X 20 cm; Proteomix SCX 3um, Sepax Technologies). We used a binary HPLC pump with split flow with microcapillary flowrate at 2-3 uL/min.  Peptides were loaded onto the microcapillary column equilibrated in Buffer A and washed with Buffer A. Bound peptides were eluted with 20 μl of Buffer A containing 30%, 50%, 70%, and 100% Buffer B (800 mM ammonium formate, 20% acetonitrile, pH 2.8), followed by 50 μl elutions with Buffer D (0.5 M ammonium acetate, 30% acetonitrile). All fractions were dried in a Speed-vac, and resuspended in 0.1% trifluoroacetic acid (TFA), 2% acetonitrile.  Peptides were analyzed by electrospray ionization microcapillary reverse phase HPLC on a Thermo Scientific Fusion with HCD fragmentation and serial MS events that included one FTMS1 event at 30,000 resolution followed by FTMS2 events at 15,000 resolution. Other instrument settings included: MS1 scan range (m/z): 400-1500; cycle time 3 sec; Charge states 4-10; Filters MIPS on, relax restriction = true; Dynamic exclusion enabled: repeat count 1, exclusion duration 30s; Filter IntensityThreshold, signal intensity 50000; Isolation mode, quadrupole; Isolation window 2Da; HCD normalized collision energy 28%, isolation width 2 Da; AGC target 500,000, Max injection time 200ms. A 90 min gradient from 5% ACN to 40% ACN was used.

### Data Protocol
The RAW files were converted to mzXML files by Rawconverter (Winter et al., 2015). For crosslinked peptide searches, we used two different crosslink database searching algorithms: pLink2 (Yang et al., 2012) and an in-house designed Nexus. Crosslinking data were analyzed using pLink2 (Yang et al., 2012) against a database containing only BAF and histones sequences with fixed modification Carbamidomethyl[C] and variable modifications:  Oxidation[M], Gln->pyro-Glu[AnyN-term], Glu->pyro-Glu[AnyN-term], Xlink_DSS[K] and Xlink_DSS[ProteinN-term].        .   For Nexus searches, the same databases were used with the following parameter settings: (a) up to three miscleavages; (b) static modification on Cysteines (+57.0215 Da); (c) differential oxidation modification on Methionines (+15.9949 Da); (d) differential modification on the peptide N-terminal Glutamic acid residues (-18.0106 Da) or N-terminal Glutamine residues (-17.0265 Da); (e) differential mono-BS3 modification on Lysine residue (+156.0806 Da). A 5% of FDR cutoff was used for both pLink and Nexus. After performing the pLink2 and Nexus analyses, the search results were combined and each spectrum was manually evaluated for the quality of the match to each peptide using the COMET/Lorikeet Spectrum Viewer (TPP). Crosslinked peptides are considered confidently identified if at least 4 consecutive b or y ions for each peptide are observed and the majority of the observed ions are accounted for. Search results that did not meet these criteria were removed. Intralinks involving a crosslink between identical residues were only kept if the spectral evidence strongly supported the identification; that is, the major fragment ions correspond to the intralinked peptide sequence and no/few other fragment ions were observed. The 4.9% of interlinked and 5.4% of intralinked spectra were removed after manual examination. .Spectra and crosslink maps can be viewed at https://www.yeastrc.org/proxl_public/viewProject.do?project_id=258.

### Publication Abstract
Mammalian SWI/SNF complexes are ATP-dependent chromatin remodeling complexes that regulate genomic architecture. Here, we present a structural model of the endogenously purified human canonical BAF complex bound to the nucleosome, generated using cryoelectron microscopy (cryo-EM), cross-linking mass spectrometry, and homology modeling. BAF complexes bilaterally engage the nucleosome H2A/H2B acidic patch regions through the SMARCB1 C-terminal &#x3b1;-helix and the SMARCA4/2 C-terminal SnAc/post-SnAc regions, with disease-associated mutations in either causing attenuated chromatin remodeling activities. Further, we define changes in BAF complex architecture upon nucleosome engagement and compare the structural model of endogenous BAF to those of related SWI/SNF-family complexes. Finally, we assign and experimentally interrogate cancer-associated hot-spot mutations localizing within the endogenous human BAF complex, identifying those that disrupt BAF subunit-subunit and subunit-nucleosome interfaces in the nucleosome-bound conformation. Taken together, this integrative structural approach provides important biophysical foundations for understanding the mechanisms of BAF complex function in normal and disease states.

### Keywords
Nucleosome acidic patch, Mammalian swi/snf, Mutations, Human disease, Cryo-em, Homology model, Canonical baf complex, Yeast swi/snf, Atp-dependent chromatin remodeling

### Affiliations
Institute For Systems Biology
Institute for Systems Biology

### Submitter
Jie Luo

### Lab Head
Dr Jeff Ranish
Institute For Systems Biology


