### Accession
PXD046941

### Title
Landscape of protein-protein-interactions during hepatitis C virus assembly and release

### Description
Assembly of infectious hepatitis C virus (HCV) particles requires multiple cellular proteins including for instance apolipoprotein E (ApoE). To describe these protein-protein-interactions, we performed an affinity purification mass spectrometry screen (AP-MS) of HCV infected cells. We used functional viral constructs with epitope-tagged envelope protein 2 (E2), protein (p)7 or nonstructural protein 4B (NS4B), as well as cells expressing a tagged variant of ApoE. We also evaluated assembly stage-dependent remodeling of protein complexes by using viral mutants carrying point mutations abrogating particle production at distinct steps of the HCV particle production cascade. Detected were five ApoE binding proteins, 12 p7 binders, seven primary E2-interactors and 24 proteins interacting with NS4B. Cell-derived PREB, STT3B, SPCS2 as well as viral NS2 interacted with both p7 and E2. Only GTF3C3 interacted with E2 and NS4B, highlighting that HCV assembly and replication complexes exhibit largely distinct interactomes. An HCV core protein mutation, preventing core protein decoration of lipid droplets, profoundly altered the E2 interactome. In cells replicating this mutant, E2 interactions with HSPA5, STT3A/B, RAD23A/B and ZNF860 were significantly enhanced, suggesting that E2 protein interactions partly depend on core protein functions. Bioinformatic and functional studies including STRING network analyses, RNA interference, and ectopic expression support a role of Rad23A and Rad23B in facilitating HCV infectious virus production. Both Rad23A and Rad23B are involved in the endoplasmic reticulum (ER)-associated protein degradation (ERAD). Collectively, our results provide a map of host proteins interacting with HCV assembly proteins and they give evidence for the involvement of ER protein folding machineries and the ERAD pathway in the late stages of the HCV replication cycle.

### Sample Protocol
Protein eluates from co-immunoprecipitation were denatured with 6 M urea, reduced with 10 mM DTT, alkylated with 55 mM iodoacetamide, digested using LysC, diluted with H2O and digested with trypsin. On the following day, the peptides were acidified with 0.1% TFA, desalted via reverse phased C18 tips and separated via high performance liquid chromatography (HPLC). Subsequently, a nano electrospray ion source coupled to an orbitrap mass spectrometer (Q Exactive HF for the first dataset, Q Exactive HF-X for the second dataset, Thermo Scientific) was used to ionize peptides. Peptides were recorded via the Xcalibur software (Thermo Scientific).

### Data Protocol
Mass spectrometric data were analyzed via the MaxQuant software (Max-Planck Institute of biochemistry, Martinsried, Germany) additionally using the human Uniprot FASTA database (Version July 2015, 91647 entries), Andromeda search engine for a common contaminants databases (247 entries) and a full-length HCV Jc1 GT2a proteome. False discovery rate (FDR) was set to 1% on peptide and protein level. For a successful protein identification, at least one unique razor peptide with a length of minimum six amino acids per protein group was demanded. Quantifications were performed using the MaxLFQ label free algorithms.  Proteome data sets obtained from mass spectrometric analyses were analyzed utilizing Perseus software (Version 1.5.3.3, 1.5.5.5 and 1.6.1.1, Max-Planck Institute of biochemistry, Martinsried, Germany). Protein hits were filtered for common contaminants, proteins only identified by one peptide or reverse hits. Then, LFQ intensities were logarithmized (log2) while missing values were imputed with values from a Gaussian distribution (width=0.3, downshift=1.7). Three to four biological replicates derived from hepatoma cells transfected with wt virus were grouped epitope-tagged viruses were compared to the untagged Jc1 wt control. IBAQ values were calculated by the MaxQuant software and indicate the relative amounts of specific proteins. We used the MaxQuant software to calculate intensity based absolute quantification (IBAQ) values as a measure of protein abundance. The IBAQ value was determined by dividing the total peptide precursor ion intensities by the number of theoretically observable peptides. Thereby, the IBAQ value allows an estimation of protein amounts. Statistical significance between each tested pair was tested via the non-parametric two-sample Welchâ€™s t-test (FDRâ‰¤0.05-0.1; S0=0-1, at least three valid values identified in the interactomes of the bait proteins; for p7 one valid value was accepted). For volcano plots, the logarithmic probability (p) was plotted against the LFQ intensity difference (log2). In the second data set, viral mutants were included, which arrest HCV assembly at certain steps. Filtering, logarithmization and imputation were performed as described before. Two-sided students t test (FDR â‰¤ 0.05, S0=1, three valid values in the first group) was utilized to detect significant enriched hits for each viral variant. For detection of significant changes of protein-protein interactions of stage-dependent alterations in assembly, refined analyses were necessary. Total protein abundances of each sample were normalized to the respective bait protein. Groups of all independent biological replicates of each virus were formed. Multi-parametric two-sided t-test (FDRâ‰¤0.05; S0=0; three valid values for the respective mutant) was applied together with the second ANOVA (pâ‰¤0.05) conducted for hits, which were found in the screen for factors showing stage-dependent regulation during assembly. Significant pairs were detected by applying post-hoc tests (FDRâ‰¤0.05) and depicted by hierarchical clustering of Z-scores.

### Publication Abstract
None

### Keywords
Ap-ms, Virus assembly, Hepatitis c virus, Hcv, Virus host interaction

### Affiliations
Proteomics
Institute of Innate Immunity University Hospital Bonn

### Submitter
Mario Oroshi

### Lab Head
Dr Felix Meissner
Institute of Innate Immunity University Hospital Bonn


