### Accession
PXD015025

### Title
Absolute Quantification of Yeast Proteins Using Synthetic Peptide Standards and TMT labeling

### Description
The absolute protein amounts for the selected yeast ribosomal proteins (RPs) were determined using the standard peptides and TMT labelling. The range of the standard tryptic peptides were selected and synthesized based on the preliminary data-dependent experiments. The synthetic peptides were mixed and four aliquots containing different amounts of the peptides were labelled each with a different TMT labelling reagent. The yeast cell lysate was digested and labelled in parallel. The labelled cellular and standard samples were combined, and the absolute peptide amounts in the lysate were calculated by comparison to the known amounts in the standard TMT channels.

### Sample Protocol
SpikeTides TQ synthetic peptide standards were purchase from JPT Peptide Technologies (Berlin, Germany). The dried aliquots of each peptide were dissolved in two solvents for each peptide: in 10% acetonitrile, 25 mM triethylammonium bicarbonate (TEAB), and in 0.5% aqueous sodium deoxycholate (SDC), 25 mM TEAB solution. Aliquots containing 100 pmol of each peptide were and each dissolution were combined into two standard peptide mixtures, according to the solvent used for solubilization. The peptide mixtures were digested with 2.5 ug of  trypsin (Pierce Trypsin Protease, MS Grade, Thermo Fisher Scientific) for 2 h at 37°C, then another 2.5 ug of  trypsin was added and the digestion was continued for 2 h at 37°C. The digested peptide samples were diluted to 800 µl with the solvent, 50 mM TEAB/10% acetonitrile for dissolution 1 and 50 mM TEAB, 0.5% SDC, and the equal aliquots form each mixture were labelled with two of the TMT 10plexTM isobaric reagents according to the manufacturer’s instructions (Thermo Scientific). The labelled samples were combined into one TMT set, concentrated using vacuum centrifugation and SDC was removed by acidification with 10% TFA and subsequent centrifugation at 13 000 rpm for 10 min. The peptides in the supernatant were purified using the Pierce C18 Spin Columns (Thermo Scientific) according to the manufacturer’s instructions. The purified peptide solution eluted from the C18 spin column was dried on Speedvac and reconstituted in 15μl of 3% acetonitrile, 0.2% formic acid for the data-dependent LC-MS/MS analysis. The dissolution in 0.5% SDC/25 mM TEAB was found to yield higher relative abundance for some of the standards, and thus it was chosen for the subsequent experiments. Yeast cell pellets were homogenized using a FastPrep®-24 instrument (Matrix D (green-capped tubes); MP Biomedicals, OH, USA) for 5 repeated 40 seconds cycles at 6.5 m/s, with 30-60 seconds pause between. The lysate was transferred to new tubes, diluted ten times with the lysis buffer and protein concentration was determined using Pierce™ BCA Protein Assay (Thermo Scientific) and the Benchmark Plus microplate reader (BIO-RAD) with BSA solutions as standards. The pooled Reference sample was prepared by aliquoting the equal protein amount from each lysate sample. An aliquot of the pooled Reference containing 30 ug of protein was reduced by addition of 2M DL-Dithiothreitol (DTT) to a final concentration of 100 mM and incubated at 56°C for 30 min. The sample was processed according to the filter-aided sample preparation (FASP) method modified from (Wisniewski JR et. al. Nat Methods. 2009 May;6(5):359-62). In short, reduced sample was diluted to 300 ul by addition of 8M urea were applied on Nanosep 30k Omega filters (Pall Life Sciences) and 200 ul 8M urea were used to repeatedly wash away the SDS. Alkylation was performed with 10 mM methyl methanethiosulfonate (MMTS) diluted in digestion buffer (0.5% SDC, 50 mM TEAB) for 30 min at room temperature and the filters were then repeatedly washed with digestion buffer. Trypsin  in digestion buffer was added (1.5 ug, mass ratio 1:20) and the sample was incubated in 37°C for 2 h, then another 1.5 ug portion of trypsin was added and incubated for 4 hours. Digested peptides were collected by centrifugation at 10 000 rpm for 15 min, followed by a wash with 20 ul of the digestion buffer, the sample was split into the aliquots containing 5.0 ug and 25 ug of total peptides, the volume of the smaller aliquot was topped up with the digestion buffer to match the volume of the bigger aliquot, and the aliquots were labelled with two of the TMT 10plexTM isobaric reagents according to the manufacturer’s instructions. The digested mixture of the standard peptides from the 0.5% SDC/25 mM TEAB dissolutions was aliquoted into four samples that contained 125 fmol, 500 fmol, 2.5 pmol and 12.5 pmol of each standard peptide, respectively. The volumes of the smaller aliquots were equalized to 100 ul using the digestion buffer, and the aliquots were labelled with four of the TMT 10plexTM isobaric reagents. The labelled samples were combined into one TMT set, concentrated using vacuum centrifugation and SDC was removed by acidification with 10% TFA and subsequent centrifugation at 13 000 rpm for 10 min. The peptides in the supernatant were purified using the Pierce C18 Spin Columns (Thermo Scientific). The purified peptide solution eluted from the C18 spin column was dried on Speedvac and reconstituted in 15μl of 3% acetonitrile, 0.2% formic acid for the LC-MS/MS analysis.

### Data Protocol
Identification and quantification were performed using Proteome Discoverer version 2.2 (Thermo Fisher Scientific). The sequences of the synthetic peptides were supplemented with porcine trypsin sequence and combined into a fasta file (52 sequences). Mascot 2.5.1 (Matrix Science, London, United Kingdom) was used as a database search engine. Precursor mass tolerance was set to 10 ppm, fragment mass tolerance to 0.6 Da; Trypsin/P with 1 missed cleavage was used as an enzyme rule; methionine oxidation was set as variable modification, TMT-6-plex on peptide N-terminus and on lysine were set as fixed modifications. Fixed Value PSM node was used for the validation of the search results. The m/z values and the retention time values for the standard-only sample were exported as a mass list and used as the targeted mass inclusion list for the yeast cellular TMT set. Quantification was performed on the MS3 spectra in Proteome Discoverer 2.2. The reporter ion S/N values were extracted using the ‘’ Most Confident Centroid’’ setting with reporter mass tolerance 10 mmu (standard-only sample) or 3 mmu (yeast cellular sample). Due to the fact that the dynamic exclusion was not used in the experimental method, each identified peptide was assigned to multiple quantification spectra. The quantification spectra for each of the synthetic peptide were then inspected manually, and the ‘’best quality’’ quantitative result was selected for each peptide, based on two factors: 1) the average reporter S/N should be as high as possible and 2) there should be no abundant unassigned signals in the MS2 spectrum, which corresponds to the given MS3 quantitative spectrum. The multinotch synchronous precursor selection was set at the 5 most abundant MS2 fragment ions, thus the 5 most abundant fragments were inspected in each MS2 spectrum of interest. The quantitative result for the ‘’best quality’’ MS3 spectrum was assigned as a quantitative result for the corresponding peptide. Then, the S/N abundances and the known amounts in the TMT channels that corresponded to the synthetic standards were used to calculate the Linear regression slope (intercept was set to 0) for each peptide (LINEST function in Microsoft Office Excel 2010); the linear regression slope and the S/N abundance from the cellular yeast channel from the SAME ‘’best quality’’ quantitative spectrum was used to calculate the amount of the corresponding peptide in the cellular yeast sample.

### Publication Abstract
Cells maintain reserves in their metabolic and translational capacities as a strategy to quickly respond to changing environments. Here we quantify these reserves by stepwise&#xa0;reducing nitrogen availability in yeast steady-state chemostat cultures, imposing severe restrictions on total cellular protein and transcript content. Combining multi-omics analysis with metabolic modeling, we find that seven metabolic superpathways maintain &gt;50% metabolic capacity in reserve, with glucose metabolism maintaining &gt;80% reserve capacity. Cells maintain &gt;50% reserve in translational capacity for 2490 out of 3361 expressed genes (74%), with a disproportionately large reserve dedicated to translating metabolic proteins. Finally, ribosome reserves contain up to 30% sub-stoichiometric ribosomal proteins, with activation of reserve translational capacity associated with selective upregulation of 17 ribosomal proteins. Together, our dataset provides a quantitative link between yeast physiology and cellular economics, which could be leveraged in future cell engineering through targeted proteome streamlining.

### Keywords
Yeast, Tandem mass tag, Absolute quantification, Tmt, Synthetic peptides, Saccharomyces cerevisiae, Standard peptides, Orbitrap fusion

### Affiliations
Professor, Department of Biology and Biological Engineering, Chalmers University of Technology, SE-41296 Gothenburg, Sweden; Wallenberg Center for Protein Research, Chalmers University of Technology, SE-41296 Gothenburg, Sweden; Novo Nordisk Foundation Center for Biosustainability, Chalmers University of Technology, SE-41296 Gothenburg, Sweden; Novo Nordisk Foundation Center for Biosustainability, Technical University of Denmark, DK-2800 Lyngby, Denmark
Proteomics Core Facility, Sahlgrenska Academy, University of Gothenburg, Gothenburg, Sweden

### Submitter
Egor Vorontsov

### Lab Head
Dr Jens Nielsen
Professor, Department of Biology and Biological Engineering, Chalmers University of Technology, SE-41296 Gothenburg, Sweden; Wallenberg Center for Protein Research, Chalmers University of Technology, SE-41296 Gothenburg, Sweden; Novo Nordisk Foundation Center for Biosustainability, Chalmers University of Technology, SE-41296 Gothenburg, Sweden; Novo Nordisk Foundation Center for Biosustainability, Technical University of Denmark, DK-2800 Lyngby, Denmark


