### Accession
PXD035505

### Title
Human Cardiovascular Tissue Extracellular Vesicles – Iodixanol Mass Exclusion

### Description
Fewer than 50% of patients develop vascular and valvular calcification, implying differential pathogenesis. Tissue-entrapped extracellular vesicles (EVs) are found in mineralization but their contents and functions are unstudied. Tissue EVs were isolated from normal and diseased human carotid arteries and aortic valves by enzymatic digestion, serial (ultra)centrifugation and density-gradient separation. Mass spectrometry characterized the density-gradient separation and ability to exclude iodixanol from the density gradient.

### Sample Protocol
Peptide samples were analyzed on an Orbitrap Fusion Lumos mass spectrometer fronted with an EASY-Spray Source (heated at 45°C), and coupled to an Easy-nLC1000 HPLC pump (Thermo Scientific). The peptides were subjected to a dual column set-up: an Acclaim PepMap RSLC C18 trap analytical column, 75 µm X 20 mm (pre-column), and an EASY-Spray LC column, 75 µm X 250 mm (Thermo Fisher Scientific). The analytical gradient was run at 300 nl/min, with Solvent A composed of water/0.1% formic acid and Solvent B composed of acetonitrile/0.1% formic acid). The acetonitrile and water were LC-MS-grade. For 90-minute gradients, the analytical gradient was run from 5-21% Solvent B for 75 minutes and 21-30% Solvent B for 15 minutes. The Orbitrap analyzer was set to 120 K resolution, and the top N precursor ions in 3 seconds cycle time within a scan range of 375-1500 m/z (60 seconds dynamic exclusion enabled) were subjected to collision induced dissociation (CID; collision energy, 30%; isolation window, 1.6 m/z; AGC target, 1.0 e4). The ion trap analyzer was set to a rapid scan rate for peptide sequencing (tandem mass spectrometry; MS/MS). When targeted mass exclusion of iodixanol was performed, it was enacted at an m/z of 775.8645 (z=2) with an exclusion mass width of 10 ppm. The retention time window was determined by pilot injections for each sample type, with a 7-minute window for the 90-minute gradient.

### Data Protocol
Tissue-derived EV MS/MS data were queried against the Human (UP000005640, downloaded November 21, 2018), Hathewaya histolytica (May 16, 2018), and Hathewaya proteolytica (UP000183952, July 9, 2018) UniProt databases simultaneously, using the HT-SEQUEST search algorithm, via the Proteome Discoverer (PD) Package (version 2.2, Thermo Scientific). Trypsin was set as the digestion enzyme while allowing up to four miss-cleavages, using 10 ppm precursor tolerance window and 0.6 Da fragment tolerance window. Oxidation of methionine and N-terminus acetylation were set as variable modifications, and carbamidomethylation of cysteine was set as a fixed modification. The peptide false discovery rate (FDR) was calculated using Percolator provided by PD and peptides were filtered based on a 1.0% FDR. Quantification utilized unique peptides (those assigned to a given Master protein group and not present in any other protein group) and razor peptides (peptides shared among multiple protein groups). Razor peptides were used to quantify only the protein with the most identified peptides and not for the other proteins they are contained in. In order to quantify peptide precursors detected in the MS1 but not sequenced from sample to sample, we enabled the ‘Feature Mapper’ node. Chromatographic alignment was done with a maximum retention time (RT) shift of 10 minutes and a mass tolerance of 10 ppm. Feature linking and mapping settings were: RT tolerance minimum of 0 minutes, mass tolerance of 10 ppm and signal-to-noise minimum of 5. Precursor peptide abundance quantification was based chromatographic intensities. Total peptide amount was used for normalization.

### Publication Abstract
None

### Keywords
Human, Carotid endarterectomy, Lc-msms, Exosome, Atherosclerotic plaque, Calcification, Extracellular vesicle, Aortic valve

### Affiliations
Brigham and Women's Hospital, Harvard Medical School

### Submitter
Mark Blaser

### Lab Head
Dr Elena Aikawa
Brigham and Women's Hospital, Harvard Medical School


