### Accession
PXD004352

### Title
Social Network Architecture of Human Immune Cells Unveiled by Quantitative Proteomics

### Description
The immune system is unique in its dynamic interplay between numerous cell types. However, a system-wide view of how immune cells communicate to protect against disease has not been established. Here, we applied high-resolution mass spectrometry-based proteomics to generate a publicly accessible protein atlas of 28 primary human immune cell populations in steady and activated states at a depth of > 10,000 proteins. Cell-type-specific protein copy numbers reveal that immune cells are most specialized at the level of ligands and receptors, thereby connecting distinct functions of the immune system. By integrating total and secreted proteomes, we deduce paracrine immune dynamics upon microbial encounter and discover fundamental intercellular communication structures as well as novel connections between cellular senders and receivers of biological information. Our comprehensive cell-type-resolved proteomic resource of human immune cells provides a framework for the orchestration of cellular interplay and a reference for altered communication associated with pathology.

### Sample Protocol
Cell pellets and supernatants were lysed in 8 M Urea and 2 M Urea, respectively, 10 mM HEPES (pH 8), 10 mM DTT and sonicated at 4°C for 15 min (level 5, Bioruptor, Diagenode). Alkylation of reduced cysteines was performed in the dark for 30 min with 55 mM iodacetamide (IAA) followed by a two-step proteolytic digestion. Samples were digested at room temperature (RT) with LysC (1:50, w/w, Wako) for 3h. Cell lysates were adjusted to 2M Urea with 50 mM ammoniumbicarbonate and then both cell lysates and supernatants were digested with trypsin (1:50, w/w, Promega) at room temperature overnight. The resulting peptide mixtures were acidified and loaded on C18 StageTips (EmporeTM, IVA-Analysentechnik). Peptides were eluted with 80% acetonitrile (ACN), dried using a SpeedVac centrifuge, and resuspended in 2% ACN, 0.1% trifluoroacetic acid (TFA), and 0.5% acetic acid. For MACS enriched cell pellets lysis was performed in 4% SDS, 10 mM HEPES (pH 8, Biomol), 10 mM DTT. Cells were heat treated at 95°C for 10 min and sonicated at 4°C for 15 min (level 5, Bioruptor, Diagenode). Proteins were precipitated with acetone at -20°C overnight and resuspended the next day in 8 M Urea, 10 mM Hepes (pH 8). Proteolytic digest steps were carried out as described above. Chemicals were purchased from Sigma-Aldrich unless stated otherwise.

### Data Protocol
MaxQuant software (version 1.5.3.2) was used to analyze MS raw files (Cox and Mann, 2008). MS/MS spectra were searched against the human Uniprot FASTA database (Version May 2013, 90,507 entries) and a common contaminants database (247 entries) by the Andromeda search engine (Cox et al., 2011). Cysteine carbamidomethylation was applied as fixed and N-terminal acetylation, deamidation at NQ, and methionine oxidation as variable modification. Enzyme specificity was set to trypsin with a maximum of 2 missed cleavages and a minimum peptide length of 7 amino acids. A false discovery rate (FDR) of 1% was applied at the peptide and protein level. Peptide identification was performed with an allowed initial precursor mass deviation of up to 7 ppm and an allowed fragment mass deviation of 20 ppm. Nonlinear retention time alignment of all measured samples was performed in MaxQuant. Peptide identifications were matched across different replicates within a time window of 1 min of the aligned retention times. A library for ‘match between runs’ in MaxQuant was built from duplicate and additional single shot MS runs from MACS enriched cell types. Protein identification required at least 1 ‘razor peptide’ in MaxQuant. A minimum ratio count of 1 was required for valid quantification events via MaxQuant’s Label Free Quantification algorithm (MaxLFQ) (Cox et al., 2014). We enabled FastLFQ with a minimum of 3 and an average of 6 neighbors. Data were filtered for common contaminants and peptides only identified by side modification were excluded from further analysis

### Publication Abstract
The immune system is unique in its dynamic interplay between numerous cell types. However, a system-wide view of how immune cells communicate to protect against disease has not yet been established. We applied high-resolution mass-spectrometry-based proteomics to characterize 28 primary human hematopoietic cell populations in steady and activated states at a depth of &gt;10,000 proteins in total. Protein copy numbers revealed a specialization of immune cells for ligand and receptor expression, thereby connecting distinct immune functions. By integrating total and secreted proteomes, we discovered fundamental intercellular communication structures and previously unknown connections between cell types. Our publicly accessible (http://www.immprot.org/) proteomic resource provides a framework for the orchestration of cellular interplay and a reference for altered communication associated with pathology.

### Keywords
Human immune cells, Proteomics, Systems immunology, Mass spectrometry, Intercellular communication

### Affiliations
Max Planck Institute of Biochemistry Department of Proteomics and Signaltransduction Am Klopferspitz 18 D-82152 Martinsried/Munich
Proteomics

### Submitter
Mario Oroshi

### Lab Head
Dr Matthias Mann
Max Planck Institute of Biochemistry Department of Proteomics and Signaltransduction Am Klopferspitz 18 D-82152 Martinsried/Munich


