### Accession
PXD022608

### Title
Operation of a TCA cycle subnetwork in the mammalian nucleus

### Description
Nucleic acid and histone modifications critically depend on central metabolism for substrates and co-factors. Although a few enzymes related to the formation of these metabolites have been reported in the nucleus, the corresponding metabolic pathways are considered to function elsewhere in the cell. Here we show that a substantial part of the mitochondrial tricarboxylic acid (TCA) cycle, the biosynthetic hub of epigenetic modification factors, is operational also in the nucleus. Using 13C-tracer analysis, we identified activity of glutamine-to-fumarate, citrate-to-succinate, and glutamine-to-aspartate routes in the nuclei of HeLa cells. Proximity labeling mass-spectrometry revealed a spatial vicinity of the involved enzymes with core nuclear proteins, supporting their nuclear location. We further show nuclear localization of aconitase 2 and 2-oxoglutarate dehydrogenase in mouse embryonic stem cells. Together, our results demonstrate operation of an extended metabolic pathway in the nucleus warranting a revision of the canonical view on metabolic compartmentalization and gene expression regulation.

### Sample Protocol
Affinity Purification of the biotinylated proteins. The frozen cell pellets were re-suspended in 8 mL of ice-cold lysis buffer with the following composition: 50 mM Tris pH 7.5, 150 mM NaCl, 1% Triton X-100, 0.1% SDS, 1 mM EDTA, 1mM EGTA, 1 mM PMFS (Merck, P7626), 1 mg/mL aprotinin (Carl-Roth, A162.1), 0.5 mg/mL leupeptin (Carl-Roth, CN33.3), 250 Units HS-Nuclease (MoBiTec, GE-NUC10700-01). Following re-suspension, the samples were incubated for one hour at 4oC under constant mild rotation (30 rpm). Subsequently, the cell lysates were sonicated at 4oC for 30 seconds, 5 times, with 30 seconds rest in-between, and were further centrifuged at 17000g for 30 minutes at 4oC to remove any insoluble material. 80 Î¼L of Streptavidin Sepharose High Performance beads (GE Healthcare, 17-5113-01) were equilibrated in 1 mL of lysis buffer for 30 minutes at 4oC under constant mild rotation (30 rpm). The equilibrated beads were centrifuged at 2000g for 5 minutes at 4oC, then transferred to the lysed supernatants and incubated for 3 hours at 4oC under constant mild rotation (30 rpm). After the incubation, the beads were centrifuged at 2000g for 5 minutes at 4oC, and finally 7.5 mL of the supernatant were discarded. The remaining beads-lysates were transferred to a Spin Column (PierceTM, Thermo Fisher Scientific, 69705), washed once with 800 Î¼L of lysis buffer, and then five times with 700 Î¼L of 50 mM ammonium bicarbonate pH 8.3. After the washing, the column was plugged and the beads were transferred to a fresh eppendorf tube with 300 Î¼L of 50 mM ammonium bicarbonate pH 8.3. The same procedure was repeated two more times to ensure all beads were collected. One Î¼g of Sequencing Modified Trypsin (Promega, V5117) was added and the samples were incubated at 37 Â°C for 16 hours under constant shaking (500 rpm). The following day, 0.5 Î¼g of trypsin were added and the beads were incubated for 2 additional hours under the same conditions. Following incubation, the beads were transferred to a Spin Column and the digested peptides were eluted with 150 Î¼L of 50 mM ammonium bicarbonate pH8. This step was performed one additional time. The eluted peptides were dried in a speed-vac and stored at -80oC until further processing. All steps were performed using low retention tips (TipOneÂ® RPT Tips, Starlab) and low protein binding collection tubes (Thermo Fisher Scientific, 90411). Dried samples were dissolved in 1% formic acid with 4% acetonitrile and subjected to OASISÂ® HLB Î¼Elution Plate (Waters) for desalting according to manufacturerâ€™s instructions. Desalted peptides were reconstituted in 50 mM HEPES (pH 8.5) and labelled with TMT10plex158 Isobaric Label Reagent (Thermo Fisher Scientific) according the manufacturerâ€™s instructions. For further sample clean up an OASISÂ® HLB Î¼Elution Plate (Waters) was used. Offline high pH reverse phase fractionation was carried out on an Agilent 1200 Infinity high-performance liquid chromatography system, equipped with a Gemini C18 column (3 Î¼m, 110 Ã…, 100 x 1.0 mm, Phenomenex). An UltiMate 3000 RSLC nano LC system (Dionex) fitted with a trapping cartridge (Î¼-Precolumn C18 PepMap 100, 5Î¼m, 300 Î¼m i.d. x 5 mm, 100 Ã…) and an analytical column (nanoEaseâ„¢ M/Z HSS T3 column 75 Î¼m x 250 mm C18, 1.8 Î¼m, 100 Ã…, Waters). Trapping was carried out with a constant flow of solvent A (0.1% formic acid in water) at 30 Î¼L/minute onto the trapping column for 6 minutes. Subsequently, peptides were eluted via the analytical column with a constant flow of 0.3 Î¼L/minute with increasing percentage of solvent B (0.1% formic acid in acetonitrile) from 2% to 4% in 4 minutes, from 4% to 8% in 2 minutes, then 8% to 28% for a further 96 minutes, and finally from 28% to 40% in another 10 minutes. The outlet of the analytical column was coupled directly to a QExactive plus (Thermo Fisher Scientific) mass spectrometer using the proxeon nanoflow source in positive ion mode. The peptides were introduced into the QExactive plus via a Pico-Tip Emitter 360 Î¼m OD x 20 Î¼m ID; 10 Î¼m tip (New Objective) and an applied spray voltage of 2.3 kV. The capillary temperature was set at 320Â°C. Full mass scan was acquired with mass range 350-1400 m/z in profile mode in the FT with resolution of 70000. The filling time was set at maximum of 100 ms with a limitation of 3x106 ions. Data dependent acquisition (DDA) was performed with the resolution of the Orbitrap set to 35000, with a fill time of 120 ms and a limitation of 2x105 ions. A normalized collision energy of 32 was applied. A loop count of 10 with count 1 was used and a minimum AGC trigger of 2e2 was set. Dynamic exclusion time of 30 s was used. The peptide match algorithm was set to â€˜preferredâ€™ and charge exclusion â€˜unassignedâ€™, charge states 1, 5 - 8 were excluded. MS2 data were acquired in profile mode.

### Data Protocol
IsobarQuant and Mascot (v2.2.07) were used to process the acquired data, which was searched against a Uniprot Homo sapiens proteome database (UP000005640) containing common contaminants and reversed sequences. The following modifications were included into the search parameters: Carbamidomethyl (C) and TMT10 (K) (fixed modification), Acetyl (N-term), Oxidation (M) and TMT10 (N-term) (variable modifications). For the full scan (MS1) a mass error tolerance of 10 ppm and for MS/MS (MS2) spectra of 0.02 Da was set. Further parameters were set: Trypsin as protease with an allowance of maximum two missed cleavages: a minimum peptide length of seven amino acids; at least two unique peptides were required for a protein identification. The false discovery rate on peptide and protein level was set to 0.01. Raw data of IsobarQuant were loaded into R (ISBN 3-900051-07-0). As a quality criteria, only proteins which were quantified with at least two different unique peptides were used for downstream analysis. The â€œsignal_sumâ€� columns of the â€œproteinsâ€�-output sheet from IsobarQuant were cleaned for potential batch-effects with limma and subsequently normalized with vsn (variance stabilization). Missing values were imputed with the impute function (method = â€œknnâ€�) from the MSNBase package.

### Publication Abstract
Nucleic acid and histone modifications critically depend on the tricarboxylic acid (TCA) cycle for substrates and cofactors. Although a few TCA cycle enzymes have been reported in the nucleus, the corresponding pathways are considered to operate in mitochondria. Here, we show that a part of the TCA cycle is operational also in the nucleus. Using <sup>13</sup>C-tracer analysis, we identified activity of glutamine-to-fumarate, citrate-to-succinate, and glutamine-to-aspartate routes in the nuclei of <i>HeLa</i> cells. Proximity labeling mass spectrometry revealed a spatial vicinity of the involved enzymes with core nuclear proteins. We further show nuclear localization of aconitase 2 and 2-oxoglutarate dehydrogenase in mouse embryonic stem cells. Nuclear localization of the latter enzyme, which produces succinyl-CoA, changed from pluripotency to a differentiated state with accompanying changes in the nuclear protein succinylation. Together, our results demonstrate operation of an extended metabolic pathway in the nucleus, warranting a revision of the canonical view on metabolic compartmentalization.

### Keywords
Nucleus, Proximity-biotinylation mass spectrometry, Metabolic pathways, Stable isotope metabolomics study

### Affiliations
EMBL
MRC Toxicology Unit, University of Cambridge

### Submitter
Frank Stein

### Lab Head
Dr Kiran Raosaheb Patil
MRC Toxicology Unit, University of Cambridge


