### Accession
PXD002054

### Title
Bank Vole Protein Turnover Study C4PR_LIV

### Description
We have used dietary administration of stable isotope labelled lysine to assess protein turnover rates for proteins from four tissues in the bank vole, Myodes glareolus. The annotated genome for this species is not available, so protein identification was attained through cross-species matching to the mouse. For proteins for which confident identifications were derived, the pattern of lysine incorporation over 40d was used to define the rate of synthesis of individual proteins in the four tissues. The data were heavily filtered to retain a very high quality data-set of turnover rates for 1088 proteins. Comparative analysis of the four tissues revealed different median rates of degradation (kidney: 0.099 per day; liver 0.136 per day; heart, 0.054 per day and skeletal muscle, 0.035 per day). These data were compared with protein degradation rates from other studies on intact animals or from cells in culture.

### Sample Protocol
10 Bank Voles were heavy isotopically-labelled to a maximum RIA of 0.5 in a 'dynamic-SILAC' experimental configuration, by dietary administration of stable isotope labelled lysine (+6Da). 2 animals were sacrificed after 1 day of labelling, 2 after 5 days, 2 after 12 days, 2 after 25 days and, finally, 2 after 40 days labelling. Whole tissue preparations from each animal were homogenised and protein extracted and solubilised using Rapigest, prior to being digested in solution using Trypsin. Tryptic peptides were analysed by LC-MSMS on a q-Exactive instrument after separation over a 95 min linear gradient of 3.8 â€“ 40 % buffer B at 300 nl per min flow rate, on an Easy-Spray PepMap, 15 cm C18 column. The MS instrument was operated in data-dependent mode with survey scans acquired at a resolution of 70,000 at m/z 200. Up to the top 10 most abundant isotope patterns with charge states +2, +3 and/or +4 from the survey scan were selected for fragmentation by HCD.

### Data Protocol
Thermo raw MS data files from each tissue and duplicate animal over the labelling trajectory were analyzed using the Progenesis-QI software package from Nonlinear Dynamics/WAters. Aligned, detected peptide features in each raw data file were quantified and each peak list, on a tissue by tissue basis (n = 10 in each data-set) per experiment, were merged and searched using Mascot (version 2.4.1) against a UniProt Mouse database. The search parameters allowed for a single trypsin missed cleavage and carbamidomethylation of cysteine residues as a fixed modification. Oxidation of methionine and [13C6] lysine were allowed as variable modifications. A peptide tolerance of 10ppm was set, with an MS/MS tolerance of 0.01Da. Protein identifications were based on both arginine- and lysine-terminated tryptic peptide matches, but only lysine-terminated peptides carried the dynamic stable isotope labeling pattern for turnover measurement. To improve the quality of the search results and allow determination of false discovery rate (FDR), automatic decoy database searches were performed in Mascot. An FDR cut-off of 1% at the peptide level was applied to the database searches; only those lysine-containing peptide matches that satisfied these criteria were imported into Progenesis-QI to match feature-derived peptide quantitation(s) with identification. All other (lower quality) peptide data were ignored. To enable the feature-level quantitation performed by Progenesis-QI to be used in turnover calculations, we used Proteolabels, a development version of the Progenesis post-processor (PPP). Proteolabels searches the Progenesis-QI feature table to recover cognate pairs of heavy and light features that co-exist in the retention time dimension of aligned 2D LC-MS/MS maps, separated by the mass shift in m/z space due to incorporation of one or more [13C6]lysine residues. In house generated scripts were created for post-processing of Proteolabels output files. The scripts, written to separate mono- and multi-labelled peptides, parse peptide feature data and aggregate to protein-level, were written in Perl. To obtain protein-level turnover data, RIAt values for peptides belonging to the same protein were grouped together and fitted using the nls() package in R. The resultant kinetic profiles were plotted with 95% confidence intervals, using ggplot2.

### Publication Abstract
Understanding the role of protein turnover in the maintenance of proteostasis requires accurate measurements of the rates of replacement of proteins in complex systems, such as intact animals. Moreover, any investigation of allometric scaling of protein turnover is likely to include species for which fully annotated proteomes are not available. We have used dietary administration of stable isotope labeled lysine to assess protein turnover rates for proteins from four tissues in the bank vole,Myodes glareolus The annotated genome for this species is not available, so protein identification was attained through cross-species matching to the mouse. For proteins for which confident identifications were derived, the pattern of lysine incorporation over 40 days was used to define the rate of synthesis of individual proteins in the four tissues. The data were heavily filtered to retain a very high quality dataset of turnover rates for 1088 proteins. Comparative analysis of the four tissues revealed different median rates of degradation (kidney: 0.099 days(-1); liver 0.136 days(-1); heart, 0.054 days(-1), and skeletal muscle, 0.035 days(-1)). These data were compared with protein degradation rates from other studies on intact animals or from cells in culture and indicate that both cell type and analytical methodology may contribute to variance in turnover data between different studies. These differences were not only due to tissue-specific proteins but were reflected in gene products common to all tissues. All data are available via ProteomeXchange with identifier PXD002054.

### Keywords
Vole protein turnover

### Affiliations
University of Liverpool

### Submitter
Dean Hammond

### Lab Head
Dr Robert John Beynon
University of Liverpool


