### Accession
PXD015768

### Title
Proteomics analysis in GGT subjects

### Description
Proteomic analysis of brain structures derived from GGT subjects

### Sample Protocol
FC and WM specimens derived from control and GGT cases were homogenized in lysis buffer containing 7 M urea, 2 M thiourea, 4% (w/v), 50 mM DTT supplemented with protease and phosphatase inhibitors. The homogenates were spinned down at 100.000 x g for 1 h at 15ºC. After protein precipitation, protein concentration was measured in the supernatants with the Bradford assay kit (Biorad).  Protein Digestion and Peptide iTRAQ Labeling An iTRAQ (isobaric Tags for Relative and Absolute Quantitation)-based quantitative proteomic analysis was performed for each brain area (FC & WM) as previously described (Zelaya MV, et al. Oncotarget 2015; Lachén-Montes M et al. Methods in Molecular biology, 2019). iTRAQ labeling of each sample was performed according to the manufacturer's protocol (Sciex). Briefly, equal amounts of protein (200 μg) from each sample were reduced with 50 mM tris (2-carboxyethyl) phosphine (TCEP) at 60 °C for 1 h. Cysteine residues were alkylated with 200 mM methylmethanethiosulfonate (MMTS) at room temperature for 15 min. Protein enzymatic cleavage was carried out with trypsin (Promega; 1:20, w/w) at 37 °C for 16 h. For both iTRAQ experiments, each tryptic digest was labelled with one isobaric amine-reactive tags as follows: Tag113, control-1; Tag114, contro-2; Tag115, control-3; Tag116, control-4; Tag117, GGT-1; Tag118, GGT-2; Tag119, GGT-3; Tag121, GGT-4. After 2h incubation, the sets of FC and WM labelled samples were independently pooled and evaporated in a vacuum centrifuge.  Phosphopeptide enrichment and LC-MS/MS The enrichment of phosphorylated peptides was performed applying the SIMAC protocol as previously described (Thingholm TE, Jensen ON et al. Mol Cell Proteomics 2008). Unbound peptide pools (nonmodified peptides) were dried in a vacuum centrifuge and reconstituted with 40 ul of 5 mM ammonium bicarbonate (ABC) pH 9.8, and injected to an ÄKTA pure 25 system (GE Healthcare Life Sciences) with a high pH stable X-Terra RP18 column (C18; 2.1 mm × 150 mm; 3.5 μm) (Waters). Mobile phases were 5mM ammonium formate in 90% ACN at pH 9.8 (buffer B) and 5mM ammonium formate in water at pH 9.8 (buffer A). Column gradient was developed in a 80 min three step gradient from 5% B to 30% B in 5 min, 30% B to 60% B in 40 min, 15 min in 60% B and 60% B to 90% B in 20 min. Column was equilibrated in 95% B for 30 min and 2% B for 10 min. Thirteen fractions were collected and evaporated under vacuum. Peptide fractions were reconstituted into a final concentration of 0.5µg/µL of 2% ACN, 0.5% FA, 97.5% MilliQ-water prior to mass spectrometric analysis. Then, peptides mixtures were separated by reverse phase chromatography using an Eksigent nanoLC ultra 2D pump fitted with a 75 μm ID column (Eksigent 0.075 x 250). Samples were first loaded for desalting and concentration into a 2 cm length 100 μm ID precolumn packed with the same chemistry as the separating column. Mobile phases were 100% water 0.1% formic acid (FA) (buffer A) and 100% Acetonitrile 0.1% FA (buffer B). Non-modified peptide fractions were analyzed following the next conditions. Column gradient was developed in a 135 min three step gradient from 2% B to 30% B in 90 min, from 30%B to 40% B in 10 min and from 40% to 80% in 10 min. Column was equilibrated in 97% B for 3 min and 2% B for 23 min. During all process, precolumn was in line with column and flow maintained all along the gradient at 300 nl/min. Eluting peptides from the column were analyzed using a 5600 Triple-TOF system (Sciex). Information data acquisition was acquired upon a survey scan performed in a mass range from 350 m/z up to 1250 m/z in a scan time of 250 ms. Top 35 peaks were selected for fragmentation. Minimum accumulation time for MS/MS was set to 11 ms giving a total cycle time of 3.8s. Product ions were scanned in a mass range from 100 m/z up to 1500 m/z and excluded for further fragmentation during 15 s. In the case of fractions that contained the phosphorylated peptides derived from FC and WM, column gradient was developed in 140 min two step gradient from 2% to 35% B in 100 min and from 35% to 70% in 20 min. Column was equilibrated in 95% B for 5 min and 2% B for 15 min. Precolumn was in line with column and flow maintained all along the gradient at 300 nl/min. Eluting peptides from the column were analyzed using a 5600 Triple-TOF system (Sciex) following the same conditions as the non-modified peptides.

### Data Protocol
The raw MS/MS spectra search were processed using the MaxQuant software (v.1.5.8.3) (Tyanova S, Temu T, Cox J. Nature protocols 2016) and searched against the Uniprot proteome reference for Homo Sapiens (Proteome ID: UP000005640_9606, February 2019). The parameters used were as follows: Initial maximum precursor (25ppm), fragment mass deviations (40ppm); variable modification (methionine oxidation and N-terminal acetylation) and fixed modification (MMTS); enzyme (trypsin) with a maximum of 1 missed cleavages; minimum peptide length (7 amino acids); false discovery rate (FDR) for PSM and protein identification (1%). Frequently observed laboratory contaminants were removed. Protein identification was considered valid with at least one unique or “razor” peptide. The protein quantification was calculated using at least 2 razor + unique peptides, and statistical significance was calculated by a two-way Student-t test (p < 0.05). A 1.3-fold change cut-off was used. Proteins with iTRAQ ratios below the low range (0.77) were considered to be down-regulated, whereas those above the high range (1.3) were considered to be upregulated. Additionally, in the analysis of the FC and WM phosphorylated fractions, phosphoserine, phosphothreonine, and phosphotyrosine were chosen as variable modifications for database searching. The Perseus software (version 1.5.6.0) (Tyanova S, Temu T, Sinitcyn P et al. Nature Methods 2016) was used for statistical analysis and data visualization. Search results files and MS raw data were deposited to the ProteomeXchange Consortium (http://proteomecentral.proteomexchange.org) via the PRIDE partner repository with the dataset identifiers PXDXXXXX. The identification of significantly dysregulated regulatory/metabolic pathways in FC and WM proteomic datasets was performed using Metascape (Zhou Y, Zhou B, Pache L, Chang M et al. Nature Communications 2019). The interactome of Tau was analyzed using FpClass (http://dcv.uhnres.utoronto.ca/FPCLASS/) (Kotlyar M, Pastrello C et al Nature methods 2015), a database of predicted protein-protein interactions (PPIs), the curated Biological General Repository for Interaction Datasets (BioGRID: https://thebiogrid.org) (Oughtred R, Stark C et al Nucleic Acid Res 2019), and STRING (Szklarczyk D, Gable AL et al Nucleic Acid research 2019).

### Publication Abstract
Globular glial tauopathy (GGT) is a progressive neurodegenerative disease involving the grey matter and white matter (WM) and characterized by neuronal deposition of hyper-phosphorylated, abnormally conformed, truncated, oligomeric 4Rtau in neurons and in glial cells forming typical globular astrocyte and oligodendrocyte inclusions (GAIs and GOIs, respectively) and coiled bodies. Present studies centre on four genetic GGT cases from two unrelated families bearing the P301T mutation in MAPT and one case of sporadic GGT (sGGT) and one case of GGT linked to MAPT K317M mutation, for comparative purposes. Clinical and neuropathological manifestations and biochemical profiles of phospho-tau are subjected to individual variations in patients carrying the same mutation, even in carriers of the same family, independently of the age of onset, gender, and duration of the disease. Immunohistochemistry, western blotting, transcriptomic, proteomics and phosphoproteomics, and intra-cerebral inoculation of brain homogenates to wild-type (WT) mice were the methods employed. In GGT cases linked to MAPT P301T mutation, astrocyte markers GFAP, ALDH1L1, YKL40 mRNA and protein, GJA1 mRNA, and AQ4 protein are significantly increased; glutamate transporter GLT1 (EAAT2) and glucose transporter (SLC2A1) decreased; mitochondrial pyruvate carrier 1 (MPC1) increased, and mitochondrial uncoupling protein 5 (UCP5) almost absent in GAIs in frontal cortex (FC). Expression of oligodendrocyte markers OLIG1 and OLIG2mRNA, and myelin-related genes MBP, PLP1, CNP, MAG, MAL, MOG, and MOBP are significantly decreased in WM; CNPase, PLP1, and MBP antibodies reveal reduction and disruption of myelinated fibres; and SMI31 antibodies mark axonal damage in the WM. Altered expression of AQ4, GLUC-t, and GLT-1 is also observed in sGGT and in GGT linked to MAPT K317M mutation. These alterations point to primary astrogliopathy and oligodendrogliopathy in GGT. In addition, GGT linked to MAPT P301T mutation proteotypes unveil a proteostatic imbalance due to widespread (phospho)proteomic dearrangement in the FC and WM, triggering a disruption of neuron projection morphogenesis and synaptic transmission. Identification of hyper-phosphorylation of variegated proteins calls into question the concept of phospho-tau-only alteration in the pathogenesis of GGT. Finally, unilateral inoculation of sarkosyl-insoluble fractions of GGT homogenates from GGT linked to MAPT P301T, sGGT, and GGT linked to MAPT K317M mutation in the hippocampus, corpus callosum, or caudate/putamen in wild-type mice produces seeding, and time- and region-dependent spreading of phosphorylated, non-oligomeric, and non-truncated 4Rtau and 3Rtau, without GAIs and GOIs but only of coiled bodies. These experiments prove that host tau strains are important in the modulation of cellular vulnerability and phenotypes of phospho-tau aggregates.

### Keywords
Phosphoproteomics, Ggt, Proteomics

### Affiliations
Proteomics unit
Clinical Neuroproteomics Unit Navarrabiomed Proteored-ISCIII Pamplona-Spain

### Submitter
Enrique SantamarÃ­a

### Lab Head
Dr ENRIQUE SANTAMARÍA
Clinical Neuroproteomics Unit Navarrabiomed Proteored-ISCIII Pamplona-Spain


