### Accession
PXD033192

### Title
Proteomic analysis of pigmented and amelanotic melanoma cells

### Description
Mass spectrometry-based proteomics comparison between amelanotic and pigmented melanoma cells for new biomarkers discovery.

### Sample Protocol
Extracted proteins were reduced using 10 mM dithiothreitol (DTT) and alkylated with 55mM iodoacetamide. Proteins were sequentially digested using Endo-LysC and trypsin and the resulting peptides were desalted and further analyzed using an Easy-nanoLC II coupled online to an LTQ-Orbitrap Velos Pro. A data dependent method was used in which the top 15 most abundant ions from an Orbitrap full scan were selected for fragmentation and detection in the linear trap. For peptide separation a 240 min gradient of 2-30% B (0.01% Formic Acid - FA, 80% Acetonitrile - ACN) was used at 300 nl/min. The precolumn and analytical column were pre-equilibrated in A (0.1% FA, 2% ACN). For most of the samples multiple injections were performed and exclusion lists were used to avoid redundant fragmentation.

### Data Protocol
The raw data were analyzed using the Andromeda algorithm integrated in MaxQuant, with Trypsin/P as the selected protease using maximum 2 missed cleaveages. The mass deviations used were 20 ppm maximum allowed mass deviation for the first searchand and 4.5 ppm for the main search for ions detected in the Orbitrap. For fragment ions detected in the linear trap 0.5 Da was considered. The modifcations used during the search were, Cys carbamidomethylation as a static modification and Met oxidation and protein N-terminus acetylation as variable modifications. The results were filtered at 1% FDR (PSM and protein level). For all searches the human version of the UniProtKB database was considered.

### Publication Abstract
Melanoma is a form of skin cancer that can rapidly invade distant organs. A distinctive feature of melanomas is their pigmentation status, as melanin is present in most skin melanomas, whilst many metastatic tumors could become amelanotic. Besides the obvious malfunction of the key genes of the melanin pathway, the amelanotic tumors could bear a characteristic molecular signature accounting for their aggressivity. Using mass spectrometry-based proteomics we report here a distinctive panel of biomarkers for amelanotic aggressive melanoma that differ from the less invasive pigmented cells. The developed method allows the label-free quantification of proteins identified by LC-MS/MS analysis. We found a set of proteins comprising AHNAK, MYOF, ANXA1, CAPN2, ASPH, EPHA2, THBS1, TGM2, ACTN4 along with proteins involved in cell adhesion/migration (integrins, PLEC, FSCN1, FN1) that are highly expressed in amelanotic melanoma. Accompanying the down regulation of pigmentation specific proteins such as tyrosinase and TYRP1, these biomarkers are highly specific for a type of highly invasive melanoma. Interestingly, the LC-MS/MS proteomics analysis in hypoxia revealed that the abundance of this specific set of proteins found in normoxia was rather unaltered in these conditions. These biomarkers could therefore predict a metastatic behaviour for the amelanotic cells in the early stages of the tumor development and thus serve in melanoma prognostic. Applying this algorithm to related databases including melanoma samples published by independent laboratories/public databases we confirm the specificity of the newly found signatures. Overall, we begin to unravel the molecular alterations in the amelanotic melanoma and how basic proteomics offers insights into how to assess the clinical, pathological and misdiagnosis differences between the main subtypes of melanoma.

### Keywords
Proteomics, Amelanotic melanoma, Melanoma biomarkers

### Affiliations
Institute of Biochemistry of the Romanian Academy (IBRA)
Dept. of Molecular Cell Biology

### Submitter
Cristian Munteanu

### Lab Head
Dr È˜tefana M. Petrescu
Dept. of Molecular Cell Biology


