### Accession
PXD042987

### Title
Nuclear Hsp104 safeguards the dormant translation machinery during quiescence

### Description
The resilience of cellular proteostasis declines with age, which drives protein aggregation and compromises viability. The nucleus has emerged as a key quality control compartment that unloads the cytosolic protein biosynthesis system from misfolded proteins. Here, we find that age-associated metabolic cues target the yeast protein disaggregase Hsp104 to the nucleus to maintain a functional nuclear proteome during quiescence. The switch to respiratory metabolism and the accompanying decrease in translation rates direct cytosolic Hsp104 to the nucleus to interact with latent translation initiation factor eIF2 subcomplexes and to suppress protein aggregation. Hindering Hsp104 from entering the nucleus in quiescent cells results in delayed re-entry into the cell cycle due to compromised resumption of protein synthesis. In sum, we report that cytosolic-nuclear partitioning of the Hsp104 disaggregase is a critical mechanism to protect the latent protein synthesis machinery during quiescence, ensuring the rapid restart of translation once nutrients are replenished.

### Sample Protocol
Cells harboring endogenously BirA*-tagged Hsp104 variants were inoculated to OD600 0.1 and biotin (50 µM final concentration, solubilized in DMSO) was added 9 h before harvesting. 48 h after inoculation, samples were collected by centrifugation (5 min, 4 000 g) and washed once in water with 10 mM sodium azide. Cell pellets were resuspended in 500 µl ice-cold RIPA buffer without detergents (50 mM Tris pH 7.5, 150 mM NaCl, 1.5 mM MgCl2, 1 mM EGTA, 1 mM DTT, 1 mM PMSF, Roche cOmplete EDTA-free protease inhibitor cocktail). Appr. 500 µl of glass beads (500 µm diameter) were added and mechanical lysis was performed in three cycles of 1 min. Following cell lysis, 0.4% SDS and 1% (v/v) IGEPAL CA-630 (Sigma-Aldrich) were admixed and samples were sonicated for 3 cycles of 10 sec (60% amplitude). After incubation for 20 min at room temperature, cell debris and insoluble material were removed by centrifugation (24 000 g, 15 min, 4°C). The cleared lysates were transferred to fresh tubes containing 30 µl pre-equilibrated Pierce™ Streptavidin Magnetic Beads (60 µl slurry, Thermo Scientific). After 3 hours of incubation at 4°C tumbling, the beads were washed extensively with RIPA buffer (with 0.1 % SDS), followed by 3x washes with 50 mM ammonium bicarbonate buffer. Finally, 200 µl of ammonium bicarbonate buffer and sequencing-grade trypsin (1 µg, Promega) were added to the beads. After overnight digestion at 37°C, the generated peptides were lyophilized and analyzed by mass spectrometry. For LC-MS/MS analysis, desalted peptides were introduced into an Ultimate 3000 RSLCnano system (Thermo). These peptides were then separated in a 15-cm analytical column (75 μm ID, packed with ReproSil-Pur C18-AQ 2.4 μm from Dr. Maisch). The separation involved a 50-minute gradient ranging from 5% to 60% acetonitrile in 0.1% formic acid. The HPLC effluent was directly electrosprayed into a Qexactive HF mass spectrometer (Thermo) operating in data-dependent mode, allowing for automatic switching between full scan MS and MS/MS acquisition. Survey full scan MS spectra were acquired within a range of m/z 375-1600, with a resolution (R) of 60,000 at m/z 400 (using an AGC target of 3x10^6). The ten most intense peptide ions, with charge states ranging from 2 to 5, were sequentially isolated up to a target value of 1x10^5 and fragmented using 27% normalized collision energy. The mass spectrometric conditions included a spray voltage of 1.5 kV, no sheath and auxiliary gas flow, a heated capillary temperature of 250°C, and an ion selection threshold of 33,000 count

### Data Protocol
For protein identification and quantification, MaxQuant 1.5.2.8 software was utilized. The following parameters were employed: database - UP000002311_559292_Scerevisiae_20171017, MS tolerance - 10 ppm, MS/MS tolerance - 0.5 Da, peptide false discovery rate (FDR) - 0.1, protein FDR - 0.01, minimum peptide length - 5, variable modification - oxidation (M), fixed modification - carbamidomethyl (C), peptides for protein quantitation - razor and unique, minimum peptides - 1, and minimum ratio count - 2. The codes to perform analyses on the BioID LC-MS data are available at GitHub repository (https://github.com/wasimaftab/LIMMA-pipeline-proteomics/tree/master).

### Publication Abstract
The resilience of cellular proteostasis declines with age, which drives protein aggregation and compromises viability. The nucleus has emerged as a key quality control compartment that handles misfolded proteins produced by the cytosolic protein biosynthesis system. Here, we find that age-associated metabolic cues target the yeast protein disaggregase Hsp104 to the nucleus to maintain a functional nuclear proteome during quiescence. The switch to respiratory metabolism and the accompanying decrease in translation rates direct cytosolic Hsp104 to the nucleus to interact with latent translation initiation factor eIF2 and to suppress protein aggregation. Hindering Hsp104 from entering the nucleus in quiescent cells results in delayed re-entry into the cell cycle due to compromised resumption of protein synthesis. In sum, we report that cytosolic-nuclear partitioning of the Hsp104 disaggregase is a critical mechanism to protect the latent protein synthesis machinery during quiescence in yeast, ensuring the rapid restart of translation once nutrients are replenished.

### Keywords
Eif2, Bioid, Protein aggregation, Translation, Hsp104, Ribosome reactivation, Aging, Disaggregation, Nuclear protein quality control, Quiescence, Proteasome

### Affiliations
Biomedical Center-LMU
Department of Molecular Biosciences,  The Wenner-Gren Institute,  Stockholm University,  10691 Stockholm, Sweden

### Submitter
Ignasi Forne

### Lab Head
Dr Sabrina Büttner
Department of Molecular Biosciences,  The Wenner-Gren Institute,  Stockholm University,  10691 Stockholm, Sweden


