### Accession
PXD022888

### Title
Dietary restriction transforms the mammalian protein sulfhydrome in a tissue-specific and cystathionine γ-lyase-dependent manner

### Description
Hydrogen sulfide (H2S) is a cytoprotective redox-active metabolite that signals through protein persulfidation (R-SSnH). Despite the known importance of persulfidation on relatively few identified proteins, tissue-specific sulfhydrome profiles and their associated functions are not well characterized, specifically under conditions known to modulate H2S production. We hypothesized that dietary restriction (DR), which increases lifespan and can boost H2S production, expands tissue-specific sulfhydromes. Here, we found protein persulfidation was enriched in liver, kidney, muscle, and brain but decreased in heart of young and aged male mice under two forms of DR, with DR promoting persulfidation in numerous metabolic and aging-related pathways. Mice lacking H2S producing enzyme cystathionine γ-lyase (CGL) had overall decreased tissue protein persulfidation and failed to functionally augment sulfhydromes in response to DR. Overall, we defined tissue- and CGL-dependent sulfhydromes and how diet transforms their makeup, underscoring the breadth for DR and H2S to impact biological processes and organismal health.

### Sample Protocol
7 mg of proteins were extracted and alkylated with 343 µM Maleimide-PEG2- biotin as described above in the BTA assay. Alkylated proteins were precipitated with ice cold acetone, resuspended in denaturation buffer (8 M urea, 1 mM MgSO4 and 30 mM Tris–HCl, pH 7.5). After denaturation for 10 minutes at 90°C proteins were diluted with 7 volumes of dilution buffer (1 mM CaCl2, 100 mM NaCl and 30 mM HEPES-NaOH pH 7.5), followed by incubating with trypsin (Promega, #V5111) (ratio of 1:50 W/W) overnight at 37°C. After overnight digestion, trypsin was heat inactivated at 95°C for 10min. Prior to adding the digested peptide to the streptavidin-agarose resin containing spin columns, the columns (Pierce, Catalog No. 69705) were first washed with PBS two times, followed by adding 0.400 mL streptavidin-agarose resin (Thermo Scientific, #20347) to the column and washing the resin 1x with PBS. The resin was then equilibrated with a 1:7 denaturation buffer: dilution buffer wash four times prior to plugging the outlet of the column and incubated at 4°C with continuous rotation for better mixing for 18 hours. After the overnight incubation, the protein resin mix in the columns was washed six times in 0.85 mL of wash buffer 1 (30 mM Hepes, 1 mM EDTA, 150 mM NaCl, 0.5 % Triton X-100, 0.1 % SDS, pH 7.5) followed by another six washes with 0.85 mL of wash buffer 2 (30 mM Hepes, 1 mM EDTA, 600 mM NaCl, 0.5 % Triton X-100, 0.1 % SDS, pH 7.5) and finally another three washes with 0.85 mL of wash buffer 3 (30 mM Hepes, 1 mM EDTA, 100 mM NaCl, pH 7.5). All washes were done using the Vac-Man Laboratory Vacuum Manifold (Promega A7231). Resin with bound peptide was first incubated with freshly prepared 500 µL elution buffer (30 mM Hepes, 1 mM EDTA, 100 mM NaCl, 10 mM TCEP, 10mM TEAB, pH 7.5) for 30 min at 25˚C followed by centrifugation of the spin tube inside of a 2 mL collection tube at 2000 g for 2 minutes to collect the eluate. A C18 column (Thermo #89870) was used to remove the TCEP followed by eluting peptide with 70% ACN. Eluted peptides were dried and suspended with 100 µl of 50 mM TEAB buffer. Thiol groups were alkylated and labeled with 2.95 mM iodoTMT (Thermo Scientific #90101) for 1h in the dark. Reactions were then quenched by vortexing and incubating for 15 minutes at 37oC in 20 mM DTT in the dark. Each of the 6 samples labeled with a unique IodoTMT tag were then mixed into a single tube and dried.  Finally, the combined labeled peptides were reconstituted with 1% Acetic Acid for further analyses via LC/MS.

### Data Protocol
MS raw data files of iodoTMT labeled samples were analyzed by searched Proteome discoverer version 2.4.1.15 (Thermo Fisher Scientific) with the following settings. We used Mus musculus in SwissProt version 2019 (Swiss Institute of Bioinformatics, Switzerland) FASTA file. For quantification method we used iodoTMT6plex with the TMT reporter ion isotope distribution (lot number VH308730). For processing setup, we used integration tolerance as 20 ppm and integration method were set to most confident centroid. We used FTMS mass analyzer. We selected MS1 precursor for precursor selection section. For the enzyme name we used trypsin (full) with a maximum missed cleavage site of 2. Precursor mass tolerance were 10 ppm and fragment mass tolerance were set to 0.2 Da. For dynamic modification we allowed methionine oxidation (+15.995 Da). We selected static modification as iodoTMT6plex/+329.227 Da of cysteine. For consensus setup, we used intensity as reporter abundance. Validation mode were set to automatic (control peptide level error rate if possible). Target FDR (strict) for PSMs and target FDR (strict) for peptide were set at 0.01, on the other hand Target FDR (relaxed) for PSMs and target FDR (relaxed) for peptide were set at 0.05. Peptide confidence were set to high with minimum peptide length allowed to 6.  For confidence threshold we used target FDR (strict) and target FDR (relaxed) at 0.01 and 0.05, respectively. Minimum PSM confidence level was set to high.

### Publication Abstract
Hydrogen sulfide (H<sub>2</sub>S) is a cytoprotective redox-active metabolite that signals through protein persulfidation (R-SS<sub>n</sub>H). Despite the known importance of persulfidation, tissue-specific persulfidome profiles and their associated functions are not well characterized, specifically under conditions and interventions known to modulate H<sub>2</sub>S production. We hypothesize that dietary restriction (DR), which increases lifespan and can boost H<sub>2</sub>S production, expands tissue-specific persulfidomes. Here, we find protein persulfidation enriched in liver, kidney, muscle, and brain but decreased in heart of young and aged male mice under two forms of DR, with DR promoting persulfidation in numerous metabolic and aging-related pathways. Mice lacking cystathionine &#x3b3;-lyase (CGL) have overall decreased tissue protein persulfidation numbers and fail to functionally augment persulfidomes in response to DR, predominantly in kidney, muscle, and brain. Here, we define tissue- and CGL-dependent persulfidomes and how diet transforms their makeup, underscoring the breadth for DR and H<sub>2</sub>S to impact biological processes and organismal health.

### Keywords
Persulfidation, Cysteine, Redox metabolism, Cystathionine γ-lyase (cgl, Sulfhydrome, Longevity, Sulfhydration, Cth), Cse, Every other day (eod) fasting, Hydrogen sulfide (h2s), Dietary restriction (dr), Biotin thiol assay (bta)

### Affiliations
Cleveland Clinic Lerner Research Institute Cardiovascular and Metabolic Sciences
Cleveland Clinic Lerner Research Institute 

### Submitter
Nazmin Bithi

### Lab Head
Dr Christopher Hine
Cleveland Clinic Lerner Research Institute Cardiovascular and Metabolic Sciences


