### Accession
PXD009367

### Title
Application of SPOT to Zn-treated HL-60 cells for multi-omics mechanism discovery, phosphoproteomics

### Description
Application of SPOT to Zn-treated HL-60 cells for multi-omics mechanism discovery. A time-resolved, multi-omics experiment using an automated, unified sample preparation platform. These datasets represent additional phosphoproteomics datasets(not prepared using SPOT) that were used along with the SPOT datasets for mechanism discovery

### Sample Protocol
To supplement the label-free data, results from the analysis of 1 h and 24 h Zn-treated SILAC labeled and phosphopeptide-enriched HL60 cells were incorporated into the computational network analysis. These samples were prepared in a similar manner as previously described for A549 cells (Norris et al, 2017). However, samples were analyzed on a Q Exactive HF mass spectrometer (Thermo Scientific) in line with a Dionex Ultimate 3000 NanoLC and autosampler equipped with a nanoelectrospray ionization source. Phosphopeptides were loaded on a MudPIT column as described previously (Norris et al, 2017). An 8-step salt pulse gradient (25, 50, 75, 100, 150, 250, 500, and 1000 mM ammonium acetate) was performed. Following each salt pulse, peptides were gradient-eluted from the reverse phase analytical column at a flow rate of 350 nL/minute, and the mobile phase solvents consisted of 0.1% FA in water (solvent A) and 0.1% FA in ACN (solvent B). A 95-min reverse phase gradient was used that consisted of 2â€“50% solvent B in 83 min followed by a 12 min equilibration at 2% solvent B for the peptides from the first seven SCX fractions. For the last fraction, the peptides were eluted from the reverse phase analytical column using a gradient of 2-98% solvent B in 83 minutes. The instrument method consisted of MS1 using an MS automatic gain control target value of 3e6 followed by up to 15 MS/MS scans of the most abundant ions detected in the preceding MS scan. A maximum MS/MS ion time of 40 ms was used with an MS2 automatic gain control target of 1e5. Dynamic exclusion was set to 30 s, high energy collision dissociation was set to 27% of the normalized collision energy, and peptide match and isotope exclusion were enabled.

### Data Protocol
All instrument output files (transcriptomics, metabolomics, and proteomics) were uploaded into a central in-house database and arranged into an exportable file with fold change represented as treated/control and with the UniProt gene symbol and protein function appended to each data type specific report (e.g., metabolomics, proteomics, or transcriptomics). Metabolites were tagged with UniProt gene symbols based on the Human Metabolome Database associations. Additionally, the database generates integrated exports, listing all detected and significantly changed analytes. The database provides responses (representative of all collected data) to queries based on protein function, gene symbol, metabolite, and fold change. File exports from this central database were used to import the data into two custom data analysis and visualization tools. The first tool provided enrichment analysis and data driven network construction, with empirical data overlaid onto interacting partners based on literature annotation, for each time point over the course of the experiment. The details of these tools will be published elsewhere. The second tool, provided visualization (through the use of Cytoscape (Shannon et al, 2003) of the constructed data networks.

### Publication Abstract
Proteomics, metabolomics, and transcriptomics generate comprehensive data sets, and current biocomputational capabilities allow their efficient integration for systems biology analysis. Published multiomics studies cover methodological advances as well as applications to biological questions. However, few studies have focused on the development of a high-throughput, unified sample preparation approach to complement high-throughput omic analytics. This report details the automation, benchmarking, and application of a strategy for transcriptomic, proteomic, and metabolomic analyses from a common sample. The approach, sample preparation for multi-omics technologies (SPOT), provides equivalent performance to typical individual omic preparation methods but greatly enhances throughput and minimizes the resources required for multiomic experiments. SPOT was applied to a multiomics time course experiment for zinc-treated HL-60 cells. The data reveal Zn effects on NRF2 antioxidant and NFkappaB signaling. High-throughput approaches such as these are critical for the acquisition of temporally resolved, multicondition, large multiomic data sets such as those necessary to assess complex clinical and biological concerns. Ultimately, this type of approach will provide an expanded understanding of challenging scientific questions across many fields.

### Keywords
Phosphoproteomics, Spot, Multi-omics

### Affiliations
Department of Biochemistry, Mass Spectrometry Research Center, Vanderbilt University
Vanderbilt University

### Submitter
Danielle Gutierrez

### Lab Head
Dr Richard M. Caprioli
Department of Biochemistry, Mass Spectrometry Research Center, Vanderbilt University


