### Accession
PXD009189

### Title
A network of chaperones prevents and detects failures in membrane protein lipid bilayer integration

### Description
A fundamental step in membrane protein biogenesis is their integration into the lipid bilayer with a defined topology. Despite this, it remains unclear how cells detect and handle failures in membrane integration. Our data show that single point mutations in the membrane protein connexin 32 (Cx32), which cause Charcot-Marie-Tooth disease, can cause failures in membrane integration. This leads to Cx32 transport defects and its rapid degradation. We found multiple chaperones to detect and remedy this aberrant behavior: the ER membrane complex (EMC) aids in membrane integration of low-hydrophobicity transmembrane segments. If they fail to integrate, these are recognized by the ER lumenal chaperone BiP. Ultimately, the E3 ligase gp78 ubiquitinates Cx32 proteins targeting them for degradation. Thus, cells use a coordinated system of chaperones for the complex task of membrane protein biogenesis, which can be compromised by single point mutations, causing human disease.

### Sample Protocol
HEK293T cells were seeded in P100 plates and transfected with 10 ug of FLAG- tagged Cx32. Immunoprecipitations were performed in digitonin buffer as described above, but using a rabbit monoclonal anti-FLAG (Sigma-Aldrich, F7425) and a rabbit IgG isotype control (Thermo Scientific, 10500C), in a total of three replicates each. Two additional washing steps were performed without digitonin. Proteins were digested, eluted, desalted and purified as previously described (Keilhauer et al., 2015). Nanoflow LC-MS/MS analyses were performed with an UltiMate 3000 Nano HPLC system (Thermo Scientific) coupled to an Orbitrap Fusion mass spectrometer (Thermo Scientific). Peptides were loaded on a trap column (Acclaim C18 PepMap100 75 um ID x 2 cm) with 0.1% TFA, then transferred to an analytical column (Acclaim C18 PepMap RSLC, 75 um ID x 50 cm, 0.1% FA) heated at 50 Â°C and separated using a 105 min gradient from 5% to 22% followed by a 10 min gradient from 22% to 32% acetonitrile in 0.1% FA at a flow rate of 300 nl/min. Peptides were ionized using an EASY-ETD/IC source. Orbitrap Fusion was operated in a top speed data dependent mode with a cycle time of 3 s. Full scan (MS1) acquisition (scan range of 300 - 1500 m/z) was performed in the orbitrap at a resolution of 120000 and with an automatic gain control (AGC) ion target value of 2e5. Dynamic exclusion of 60 s as well as EASY-IC internal calibration was enabled. Most intense precursors with charge states of 2-7 and a minimum intensity of 5e3 were selected for fragmentation. Isolation was performed in the quadrupole using a window of 1.6 m/z. Fragments were generated using higher- energy collisional dissociation (HCD, collision energy: 30%). The MS2 AGC target was set to 1e4 and a maximum injection time for the ion trap of 50 ms was used ("inject ions for all available parallelizable time" enabled).

### Data Protocol
MS raw files were analyzed with MaxQuant software (version 1.5.3.8) (Cox and Mann, 2008) with most default settings and a protein database containing human sequences (downloaded May 2017 from Uniprot, taxonomy ID: 9606). Note: Although the target protein Connexin 32 was FLAG-tagged and occasionally mutated, the original protein sequence was not modified. The following parameter settings were used: PSM and protein FDR 1%; enzyme specificity trypsin/P; minimal peptide length: 7; variable modifications: methionine oxidation, N-terminal acetylation; fixed modification carbamidomethylation. For label-free protein quantification, the MaxLFQ algorithm (Cox et al., 2014) was used as part of the MaxQuant environment: (LFQ) minimum ratio count: 1; peptides for quantification: unique and razor. The "match between runs" option was enabled (match time window: 0.7 min, alignment time window: 20 min). Statistical analysis was performed in Perseus (version 1.5.3.2) (Tyanova et al., 2016). Proteins identified only by site, reverse hits or potential contaminants were removed. LFQ intensities were log2 transformed and the matrix was separated to submatrices. Data were then filtered for at least two valid values in at least one replicate group (antibody or isotype control). Then, missing values were imputed from normal distribution (width: 0.3, down shift: 1.8 standard deviations, mode: total matrix). The three replicates of antibody and isotype control were compared via a two-sided, two- sample Student's t-test (S0=0, permutation-based FDR method with FDR=0.05 and 250 randomizations). Enrichment values and corresponding -log10 P values were plotted.

### Publication Abstract
A fundamental step in membrane protein biogenesis is their integration into the lipid bilayer with a defined orientation of each transmembrane segment. Despite this, it remains unclear how cells detect and handle failures in this process. Here we show that single point mutations in the membrane protein connexin 32 (Cx32), which cause Charcot-Marie-Tooth disease, can cause failures in membrane integration. This leads to Cx32 transport defects and rapid degradation. Our data show that multiple chaperones detect and remedy this aberrant behavior: the ER-membrane complex (EMC) aids in membrane integration of low-hydrophobicity transmembrane segments. If they fail to integrate, these are recognized by the ER-lumenal chaperone BiP. Ultimately, the E3 ligase gp78 ubiquitinates Cx32 proteins, targeting them for degradation. Thus, cells use a coordinated system of chaperones for the complex task of membrane protein biogenesis, which can be compromised by single point mutations, causing human disease.

### Keywords
Connexin, Membrane protein, Protein degradation, Protein folding, Chaperone, Er membrane complex, Protein quality control, Topology, Endoplasmic reticulum

### Affiliations
Karolinska Institutet
Lehrstuhl fuer Organische Chemie II Department Chemie TU Muenchen

### Submitter
Matthias Stahl

### Lab Head
Dr Stephan A. Sieber
Lehrstuhl fuer Organische Chemie II Department Chemie TU Muenchen


