### Accession
PXD030318

### Title
Rabbit liver tissue infected with rabbit hemorrhagic disease virus (RHDV) LC-MS

### Description
The rabbit hemorrhagic disease virus (RHDV) represents the causative agent of a highly contagious disease in rabbits that is often associated with high mortality. Because of the lack of a suitable cell culture system for RHDV, the pathogenic mechanism and replication of RHDV remains unclear. In order to analyze the pathogenic mechanism of RHDV to rabbits, we used New Zealand white rabbits infected with RHDV, collected liver tissues 32 hours after infection, and used TMT labeling for LC-MS analysis. Subsequently, it was compared and analyzed with the protein data of the liver tissue of the uninfected rabbits. Perform bioinformatics analysis on significantly different proteins. Finally, comprehensively analyze the influence of RHDV on host protein and pathway expression levels. This study provides clues to clarify the pathogenic mechanism of RHDV in rabbits.

### Sample Protocol
(1) Protein Extraction. The sample was grinded by liquid nitrogen into cell powder and then transferred to a 5-mL centrifuge tube. After that, four volumes of lysis buffer (8 M urea, 1% Protease Inhibitor Cocktail) was added to the cell powder, followed by sonication three times on ice using a high intensity ultrasonic processor (Scientz). The remaining debris was removed by centrifugation at 12,000 g at 4 °C for 10 min. Finally, the supernatant was collected and the protein concentration was determined with BCA kit according to the manufacturer’s instructions. (2) Trypsin Digestion. For digestion, the protein solution was reduced with 5 mM dithiothreitol for 30 min at 56 °C and alkylated with 11 mM iodoacetamide for 15 min at room temperature in darkness. The protein sample was then diluted by adding 100 mM TEAB to urea concentration less than 2M. Finally, trypsin was added at 1:50 trypsin-to-protein mass ratio for the first digestion overnight and 1:100 trypsin-to-protein mass ratio for a second 4 h-digestion. (3) TMT Labeling. After trypsin digestion, peptide was desalted by Strata X C18 SPE column (Phenomenex) and vacuum-dried. Peptide was reconstituted in 0.5 M TEAB and processed according to the manufacturer’s protocol for TMT kit. Briefly, one unit of TMT reagent were thawed and reconstituted in acetonitrile. The peptide mixtures were then incubated for 2 h at room temperature and pooled, desalted and dried by vacuum centrifugation.  (4) HPLC Fractionation. The tryptic peptides were fractionated into fractions by high pH reverse-phase HPLC using Agilent 300Extend C18 column (5 μm particles, 4.6 mm ID, 250 mm length). Briefly, peptides were first separated with a gradient of 8% to 32% acetonitrile (pH 9.0) over 60 min into 60 fractions. Then, the peptides were combined into 18 fractions and dried by vacuum centrifuging. (5) LC-MS/MS Analysis. The tryptic peptides were dissolved in 0.1% formic acid (solvent A), directly loaded onto a home-made reversed-phase analytical column (15-cm length, 75 μm i.d.). The gradient was comprised of an increase from 6% to 23% solvent B (0.1% formic acid in 98% acetonitrile) over 26 min, 23% to 35% in 8 min and climbing to 80% in 3 min then holding at 80% for the last 3 min, all at a constant flow rate of 400 nL/min on an EASY-nLC 1000 UPLC system. The peptides were subjected to NSI source followed by tandem mass spectrometry (MS/MS) in Orbitrap FusionTM TribridTM (Thermo) coupled online to the UPLC. Intact peptides were detected in the Orbitrap at a resolution of 60,000. Peptides were selected for MS/MS using NCE setting as 35; ion fragments were detected in the Orbitrap at a resolution of 30,000. A data-dependent procedure that alternated between one MS scan followed by 10 MS/MS scans was applied for the top 10 precursor ions above a threshold intensity greater than 5E3 in the MS survey scan with 30.0 s dynamic exclusion. The electrospray voltage applied was 2.0 kV. Automatic gain control (AGC) was used to prevent overfilling of the orbitrap; 5E4 ions were accumulated for generation of MS/MS spectra. For MS scans, the m/z scan range was 350 to 1550. Fixed first mass was set as 100 m/z.

### Data Protocol
1. Database Search. The resulting MS/MS data were processed using Maxquant search engine (v.1.5.2.8). Tandem mass spectra were searched against OryCun2.0 (GCA_000003625.1) database concatenated with reverse decoy database. Trypsin/P was specified as cleavage enzyme allowing up to 2 missing cleavages. The mass tolerance for precursor ions was set as 20 ppm in First search and 5 ppm in Main search, and the mass tolerance for fragment ions was set as 0.02 Da. Carbamidomethyl on Cys was specified as fixed modification and oxidation on Met was specified as variable modifications. FDR was adjusted to < 1% and minimum score for peptides was set > 40. 2. GO Annotation. Gene Ontology (GO) annotation proteome was derived from the UniProt-GOA database (www.http://www.ebi.ac.uk/GOA/). Firstly, converting identified protein ID to UniProt ID and then mapping to GO IDs by protein ID. If some identified proteins were not annotated by UniProt-GOA database, the InterProScan soft would be used to annotated protein’s GO functional based on protein sequence alignment method. Then proteins were classified by Gene Ontology annotation based on three categories: biological process, cellular component and molecular function. 3. Domain Annotation. Identified proteins domain functional description were annotated by InterProScan (a sequence analysis application) based on protein sequence alignment method, and the InterPro domain database was used. InterPro (http://www.ebi.ac.uk/interpro/) is a database that integrates diverse information about protein families, domains and functional sites, and makes it freely available to the public via Web-based interfaces and services. Central to the database are diagnostic models, known as signatures, against which protein sequences can be searched to determine their potential function. InterPro has utility in the large-scale analysis of whole genomes and meta-genomes, as well as in characterizing individual protein sequences. 4. KEGG Pathway Annotation. Kyoto Encyclopedia of Genes and Genomes (KEGG) database(https://www.genome.jp/kegg/) was used to annotate protein pathway. Firstly, using KEGG online service tools KAAS to annotated protein’s KEGG database description. Then mapping the annotation result on the KEGG pathway database using KEGG online service tools KEGG mapper. 5. Subcellular Localization. We used wolfpsort a subcellular localization predication soft to predict subcellular localization. Wolfpsort an updated version of PSORT/PSORT II for the prediction of eukaryotic sequences. 6. Enrichment of Gene Ontology analysis. Proteins were classified by GO annotation into three categories: biological process, cellular compartment and molecular function. For each category, a two-tailed Fisher’s exact test was employed to test the enrichment of the differentially expressed protein against all identified proteins. The GO with a corrected p-value < 0.05 is considered significant. 7. Enrichment of pathway analysis. Encyclopedia of Genes and Genomes (KEGG) database was used to identify enriched pathways by a two-tailed Fisher’s exact test to test the enrichment of the differentially expressed protein against all identified proteins. The pathway with a corrected p-value < 0.05 was considered significant. These pathways were classified into hierarchical categories according to the KEGG website. 8. Enrichment of protein domain analysis. For each category proteins, InterPro (a resource that provides functional analysis of protein sequences by classifying them into families and predicting the presence of domains and important sites) database was researched and a two-tailed Fisher’s exact test was employed to test the enrichment of the differentially expressed protein against all identified proteins.  Protein domains with a p-value < 0.05 were considered significant. 9. Enrichment-based Clustering. For further hierarchical clustering based on different protein functional classification (such as: GO, Domain, Pathway, Complex). We first collated all the categories obtained after enrichment along with their P values, and then filtered for those categories which were at least enriched in one of the clusters with P value <0.05. This filtered P value matrix was transformed by the function x = −log10 (P value). Finally these x values were z-transformed for each functional category. These z scores were then clustered by one-way hierarchical clustering (Euclidean distance, average linkage clustering) in Genesis. Cluster membership were visualized by a heat map using the “heatmap.2” function from the “gplots” R-package.

### Publication Abstract
None

### Keywords
Lc-ms, Liver, Rabbit, Rhdv

### Affiliations
 Shanghai Veterinary Research Institute, Chinese Academy of Agricultural Sciences
Shanghai Veterinary Research Institute,Chinese Academy of Agricultural Sciences

### Submitter
Jie Zhu

### Lab Head
Dr Guangqing Liu
Shanghai Veterinary Research Institute,Chinese Academy of Agricultural Sciences


