### Accession
PXD042136

### Title
Proteomic profiling of chemotherapy responses in FOLFOX-resistant colorectal cancer cells

### Description
we aim to compare the difference of chemotherapy responses between FOLFOX-resistant and wild type colorectal cancer cells for suggesting novel treatment targets.

### Sample Protocol
The peptides fractioning and comparison were conducted on Orbitrap Fusion Lumos Mass Spectrometer (Thermo Scientific) in University Research Facility in Chemical and Environmental Analysis, The Hong Kong Polytechnic University. For the fractioning, Di-onex Ultimate 3000 RSLCnano System decorated with Acclaim PepMap RSLC analytical columns (NanoViper, C18) (Thermo Scientific) and Trap Column Cartridges Holders with nanoViper Fittings (Thermo Scientific) were used. The proteomics buffer concentration for all samples were maintained by 98% water with 2% acetonitrile and 0.1% formic acid. 1 ÂµL of samples were injected for fractioning and collected peptides data from described experiments were analysed by Orbitrap Fusion Lumos Mass Spectrometer. The acquisi-tion parameters of the instrument were: scan number: 1-10, mass range: m/z 50-6000, res-olution: 500,000 FWHM at m/z 200, and mass accuracy: less than 1ppm using internal and less than 3 ppm using external calibration.

### Data Protocol
MS and MS/ MS peptides spectra were analysed by Progenesis QI for proteomics (Version 4.2, Nonlinear Dynamics, Newcastle, UK) software compared with Swiss-Prot dataset (Sequence of Homo sapiens, Release date: June 1, 2018). The parameters for analy-sis were as follows: tolerance level: 10 ppm, maximum mass cleavage: 1, peptides charge: (2-4+), modification: both fixed and variable, Oxidation (M). The 1% False Discovery Rate (FDR) was applied to identify the peptides from complex mixtures. A fold change (FC) more than 2, p < 0.05, and replicated proteins in all samples were counted as differentially expressed proteins (DEPs) between the chemoresistant and wild type cells to complete the analysis.

### Publication Abstract
Chemoresistance mechanisms of colorectal cancer remain largely elusive. We aim to compare the difference of chemotherapy responses between FOLFOX-resistant and wild-type colorectal cancer cells by proteomic profiling to suggest novel treatment targets. FOLFOX-resistant colorectal cancer cells DLD1-R and HCT116-R were developed by chronic exposure to progressive FOLFOX doses. Proteomic profiling of FOLFOX-resistant and wild-type cells under FOLFOX exposure were conducted by mass-spectrometry-based protein-analysis technology. Verification of selected KEGG pathways was conducted by Western blot. DLD1-R had significantly higher FOLFOX-chemoresistance (10.81 times) than its wild-type counterpart. A total of 309 and 90 differentially expressed proteins were identified in DLD1-R and HCT116-R, respectively. In terms of gene ontology molecular function, RNA binding and cadherin binding ranked first for DLD1 and HCT116 groups, respectively. For gene set enrichment analysis, ribosome pathway and DNA replication were significantly up-regulated and down-regulated in DLD1-R, respectively. The most significantly up-regulated pathway in HCT116-R was regulation of the actin cytoskeleton. Up-regulations in the ribosome pathway (DLD1-R) and actin cytoskeleton (HCT116-R) were verified by Western blot. There were several significantly altered signaling pathways in FOLFOX-resistant colorectal cancer cells under FOLFOX with notable up-regulations in the ribosomal process and actin cytoskeleton.

### Keywords
Colorectal cancer, Chemoresistance, Folfox

### Affiliations
Department of Health Technology and Informatics, The Hong Kong Polytechnic University
HTI, Hong Kong Polytechnic University, Hong Kong

### Submitter
Shing Yau Tam

### Lab Head
Dr Shing Yau Tam
HTI, Hong Kong Polytechnic University, Hong Kong


