### Accession
PXD000120

### Title
Proteogenomic Analysis of Human Colon Carcinoma Cell Lines LIM1215, LIM1899 and LIM2405

### Description
In this study, we have integrated shotgun proteomics approach to compare the protein expression profiles of 3 human colon cancer cell lines (LIM1215, LIM1899 and LIM2405) representing alternative forms of colorectal cancer. We have performed detailed analysis of our proteomics data (coupled with RNA-Seq data, database identifiers not available), which has identified several cancer-associated proteins with differential expression patterns. We have also identified some protein networks which appear dysregulated in these cell lines. Protein Identification and Data Analysis: Spectra files were converted to mzXML format and processed through the global proteome machine software (GPM) (version 2.1.1), an open source protein identification system that uses the X!Tandem algorithm. For each experiment, 16 fractions were processed sequentially with output files for each individual fraction and a merged, non-redundant output file generated for protein identifications with Log (e) values less than –1. Peptide identification was determined using a parent ion mass error of +3 Da and –0.5 Da and fragment ion tolerance of 0.4 Da.30 Mass spectra were searched based on peptide trypticity with up to three missed cleavages. Carbamidomethyl was considered as a complete modification and partial modifications were also considered, including oxidation of methionine and threonine, and deamidation of asparagine and glutamine. MS/MS spectra were searched against the Homo sapiens database (Database derived from SwissProt, Ensemble, and NCBI) and reverse database searching was used to estimate false discovery rates (calculated at less than 1%).31 Data were analyzed using various bioinformatics tools, including DAVID Functional Annotation Tool (http://david.abcc.ncifcrf.gov/), Pathway Commons (http://www.pathwaycommons.org) and GeneCards (www.genecards.org).

### Sample Protocol
Not available

### Data Protocol
Not available

### Publication Abstract
As part of the genome-wide and chromosome-centric human proteomic project (C-HPP), we have integrated shotgun proteomics approach and a genome-wide transcriptomic approach (RNA-Seq) of a set of human colon cancer cell lines (LIM1215, LIM1899 and LIM2405) that were selected to represent a wide range of pathological states of colorectal cancer. The combination of a standard proteomics approach (1D-gel electrophoresis coupled to LC/ion trap mass spectrometry) and RNA-Seq allowed us to exploit the greater depth of the transcriptomics measurement (&#x223c; 9800 transcripts per cell line) versus the protein observations (&#x223c; 1900 protein identifications per cell line). Conversely, the proteomics data were helpful in identifying both cancer associated proteins with differential expression patterns as well as protein networks and pathways which appear to be deregulated in these cell lines. Examples of potential markers include mortalin, nucleophosmin, ezrin, LASP1, alpha and beta forms of spectrin, exportin, the carcinoembryonic antigen family, EGFR and MET. Interaction analyses identified the large intermediate filament family, the protein folding network and adapter proteins in focal adhesion networks, which included the CDC42 and RHOA signaling pathways that may have potential for identifying phenotypic states representing poorly and moderately differentiated states of CRC, with or without metastases.

### Keywords
Colon cancer

### Affiliations
Biomedical Sciences

### Submitter
Susan Fanayan

### Lab Head


