### Accession
PXD001831

### Title
Phosphoproteome of the cyanobacterium Synechocystis sp. PCC 6803 and its dynamics during nitrogen starvation

### Description
Cyanobacteria have shaped the earthâ€™s biosphere as the first oxygenic photoautotrophs and still play an important role in many ecosystems. The ability to adapt to changing environmental conditions is an essential characteristic in order to ensure survival. To this end, numerous studies have shown that bacteria use protein post-translational modifications such as Ser/Thr/Tyr phosphorylation in cell signalling, adaptation and regulation. Nevertheless, our knowledge of cyanobacterial phosphoproteomes and their dynamic response to environmental stimuli is relatively limited. In this study, we applied gel-free methods and high accuracy mass spectrometry towards the unbiased detection of Ser/Thr/Tyr phosphorylation events in the model cyanobacterium Synechocystis sp. PCC 6803. We could identify over 300 phosphorylation events in cultures grown on nitrate as exclusive nitrogen source. Chemical dimethylation labelling was applied to investigate proteome and phosphoproteome dynamics during nitrogen starvation. Our dataset describes the most comprehensive (phospho)proteome of Synechocystis to date, identifying 2,382 proteins and 183 phosphorylation events and quantifying 2,111 proteins and 148 phosphorylation events during nitrogen starvation. Global protein phosphorylation levels were increased in response to nitrogen depletion after 24 hours. Among the proteins with increased phosphorylation, the PII signalling protein showed the highest fold-change, serving as positive control. Other proteins with increased phosphorylation levels comprised functions in photosynthesis and in carbon and nitrogen metabolism. This study reveals dynamics of Synechocystis phosphoproteome in response to environmental stimuli and suggests an important role of protein Ser/Thr/Tyr phosphorylation in fundamental mechanisms of homeostatic control in cyanobacteria.

### Sample Protocol
BActerial proteins were extracted with SDS-buffer, precipitated by acetone-methanol mix and digested over night with trypsin. Peptides were on-column dimethylation labeled in light, medium-heavy or heavy isoform. Phosphopeptide enrichment was performed by titaniumoxide chromatography and proteome analysis was performed by isoelectric focusing based fractionation. All samples were subjected to nanoLC-MS/MS analyses on a LTQ Orbitrap Elite mass spectrometer. Peptide fragmentation was performed by higher energy collision dissociation (HCD).

### Data Protocol
All raw MS spectra were processed with MaxQuant software suite (version 1.5.1.0) (Cox et al., 2009) and default settings. Identified peaks were searched against the target-decoy databases of Synechocystis sp. PCC 6803 from Cyanobase (http://genome.microbedb.jp/cyanobase) and Uniprot (http://www.uniprot.org), containing 3672 and 3507 protein sequences and 248 common contaminants, with the following database search criteria: Trypsin was defined as cleaving enzyme and up to two missed cleavages were allowed. Carbamido-methylation of cysteines was set as a fixed and methionine oxidation, protein N-termini acetylation and S/T/Y phosphorylation were set as variable modifications. Light-, medium-heavy- and heavy- dimethylation labeling on peptide N-termini and lysine residues was defined for samples from the quantitative experiments. The initial mass tolerance of precursor ions was limited to 6 ppm and 0.5 ppm for fragment ions. False discovery rates (FDRs) of peptides and proteins were set to 1%, respectively. Quantification of dimethylation labeled peptides required at least two ratio counts. Peptides were only allowed with a posterior error probability (PEP) < 1% at the peptide- and < 5% at the phosphopeptide level. Phosphopeptide MS/MS spectra were further manually filtered and validated with stringent acceptance criteria

### Publication Abstract
Cyanobacteria have shaped the earth's biosphere as the first oxygenic photoautotrophs and still play an important role in many ecosystems. The ability to adapt to changing environmental conditions is an essential characteristic in order to ensure survival. To this end, numerous studies have shown that bacteria use protein post-translational modifications such as Ser/Thr/Tyr phosphorylation in cell signaling, adaptation, and regulation. Nevertheless, our knowledge of cyanobacterial phosphoproteomes and their dynamic response to environmental stimuli is relatively limited. In this study, we applied gel-free methods and high accuracy mass spectrometry toward the detection of Ser/Thr/Tyr phosphorylation events in the model cyanobacterium Synechocystis sp. PCC 6803. We could identify over 300 phosphorylation events in cultures grown on nitrate as exclusive nitrogen source. Chemical dimethylation labeling was applied to investigate proteome and phosphoproteome dynamics during nitrogen starvation. Our dataset describes the most comprehensive (phospho)proteome of Synechocystis to date, identifying 2382 proteins and 183 phosphorylation events and quantifying 2111 proteins and 148 phosphorylation events during nitrogen starvation. Global protein phosphorylation levels were increased in response to nitrogen depletion after 24 h. Among the proteins with increased phosphorylation, the PII signaling protein showed the highest fold-change, serving as positive control. Other proteins with increased phosphorylation levels comprised functions in photosynthesis and in carbon and nitrogen metabolism. This study reveals dynamics of Synechocystis phosphoproteome in response to environmental stimuli and suggests an important role of protein Ser/Thr/Tyr phosphorylation in fundamental mechanisms of homeostatic control in cyanobacteria.

### Keywords
Synechocystis sp. pcc 6803, Peptide dimethylation labeling, Cyanobacteria, Ltq-orbitrap, Phosphoproteome, Nitrogen starvation

### Affiliations
University Tuebingen Department of Quantitative Proteomics
Prof. Dr. Boris Macek Proteome Center Tuebingen Interfaculty Institute for Cell Biology University of Tuebingen Auf der Morgenstelle 15 72076 Tuebingen Germany Phone: +49/(0)7071/29-70556 Fax: +49/(0)7071/29-5779 Web: http://www.pct.uni-tuebingen.de/

### Submitter
Philipp Spaet

### Lab Head
Dr Boris Macek
Prof. Dr. Boris Macek Proteome Center Tuebingen Interfaculty Institute for Cell Biology University of Tuebingen Auf der Morgenstelle 15 72076 Tuebingen Germany Phone: +49/(0)7071/29-70556 Fax: +49/(0)7071/29-5779 Web: http://www.pct.uni-tuebingen.de/


