### Accession
PXD009848

### Title
Aluminum and salt induced protein changes  in tomato plants transformed with bar gene

### Description
In this project, seedlings of tomato (Solanum lycopersicum) ‘Money Maker’ plants transformed with a bar gene were treated under 20 µM Al3+ and salt (200 mM NaCl) for two weeks.  The treatments were conducted in hydroponic solutions in a greenhouse at 24±2C without supplemental light. Roots harvested were immediately frozen in liquid nitrogen and stored at -80 oC.  The treatment experiment was conducted at Agricultural Research Station, Tennessee State University, Nashville, TN 37209

### Sample Protocol
Root tissues were ground into a fine powder in liquid nitrogen. The tissue powder mixed in 10% TCA in acetone, 80% methanol/0.1 M ammonium acetate and 80% acetone; then between washes powdered tissue samples were centrifuged at 16000 x g for 20 min at 4 oC. Proteins were extracted from pellets using the dense SDS/phenol extraction method. Proteins were precipitated in 0.1M ammonium acetate in methanol (1:6; v/v). Following washes in cold methanol and acetone, the air-dried pellets were solubilized in a buffer containing 500 mM triethylammonium bicarbonate (TEAB), 0.1% SDS, 8 M urea, and 1X protease inhibitors (Sigma, St. Louis, MO, USA) (Zhou et al., 2016). Urea concentration in protein extracts was reduced to 1 M, TEAB to 100 mM concentration with 50 mM TEAB buffer. Protein concentration was determined by using a Bradford Assay Kit (Bio-Rad, Hercules, CA, USA).   The nine  labeled samples were combined and subjected to reverse-phase (RP) solid-phase extraction (SPE) procedure to remove SDS and excess tags using cartridges (Sep-Pak C18 cartridge, 1-cm3, 50-mg) (Waters; Milford, MA, USA). Peptides were eluted in 500 µL of 50% (v/v) acetonitrile with 0.1% trifluoroacetic acid (TFA) and dried using a CentiVac Concentrator (LabConco, Kansas City, MO, USA).   Peptides were reconstituted   and   then separated on a high pH reverse phase (hpRP) chromatography using a Dionex UltiMate 3000 HPLC system with micro fraction collection and UV detection (Sunnyvale, CA) reported previously by Yang and colleagues. During this process TMT 10-plex tagged tryptic peptides were reconstituted in buffer A (20 mM ammonium formate pH 9.5 in water), and loaded onto an XTerra MS C18 column (3.5 µm, 2.1x150 mm) from Waters, (Milford, MA) with 20 mM ammonium formate (NH4FA), pH 9.5 as buffer A and 80% acetonitrile (ACN)/20% 20 mM NH4FA as buffer B. The LC was performed using a gradient from 10-45% of buffer B in 30 minutes at a flow rate 200 µL/min. Forty-eight fractions were collected at 1 minute intervals and pooled into a total of 12 fractions based on the UV absorbance at 214 nm and with multiple fraction concatenation strategy. All of the fractions were dried and reconstituted in 100 µL of 2% ACN/0.5% formate (FA) for nanoLC-MS/MS analysis. NanoLC-MS/MS analysis was carried out at the Functional & Comparative Proteomics Center, USDA-ARS, Cornell University, Ithaca, NY, using an Orbitrap Fusion (Thermo-Fisher Scientific, San Jose, CA) mass spectrometer equipped with nano ion source using high energy collision dissociation (HCD) similar to previous reports. The Orbitrap is coupled with the UltiMate3000 RSLCnano (Dionex, Sunnyvale, CA).  Each reconstituted fraction (8 L) was injected onto a PepMap C-18 RP nano trap column (3 µm, 75 µm  20 mm, Dionex) with nanoViper Fittings at 20 L/min flow rate for on-line desalting and then separated on a PepMap C-18 RP nano column (3 µm, 75µm x 15cm), and eluted in a 120 min gradient of 5% to 38% ACN in 0.1% formic acid at 300 nL/min., followed by a 7-min ramping to 95% ACN-0.1% FA and a 7-min hold at 95% ACN-0.1% FA. The column was re-equilibrated with 2% ACN-0.1% FA for 20 min prior to the next run. The Orbitrap Fusion is operated in positive ion mode with nano spray voltage set at 1.6 kV and source temperature at 275 °C. External calibration for FT, IT and quadrupole mass analyzers was performed. An internal calibration was performed using the background polysiloxane ion signal at m/z 445.120025 as the celebrant. The instrument was operated in data-dependent acquisition (DDA) mode using FT mass analyzer for one survey MS scan for selecting precursor ions followed by Top 3 second data-dependent HCD-MS/MS scans for precursor peptides with 2-7 charged ions above a threshold ion count of 10,000 with normalized collision energy of 37.5%.  MS survey scans at a resolving power of 120,000 (fwhm at m/z 200), for the mass range of m/z 400-1600 with AGC =3e5 and Max IT = 50 ms, and MS/MS scans at 50,000 resolution with AGC=1e5, Max IT = 120ms and with Q isolation window (m/z) at 1.6 for the mass range m/z 105-2000. Dynamic exclusion parameters were set at 1 within 50s exclusion duration with ±10 ppm exclusion mass width. All data was acquired under Xcalibur 3.0 operation software and Orbitrap Fusion Tune 2.0 (Thermo-Fisher Scientific).

### Data Protocol
All MS and MS/MS raw spectra from each set of TMT10-plex experiments were processed and database searched using Sequest HT software within Proteome Discoverer 1.4 (PD 1.4, Thermo) against    tomato protein database version ITAG3.20.  The default search settings used for 10-plex TMT quantitative processing and protein identification in PD 2.1 searching software were: two mis-cleavages for full trypsin with fixed Carbamidomethyl of cysteine, fixed 10-plex TMT modifications on lysine and N-terminal amines and variable modifications of methionine oxidation and deamidation on asparagine and glutamine residues. Identified peptides were filtered for a maximum 1% FDR using the Percolator algorithm in PD 1.4. Peptide confidence was set to high. The TMT10-plex quantification method within PD 1.4 was used to calculate the reporter ratios with a mass tolerance ±10 ppm without applying isotopic correction factors. Only peptide spectra containing all reporter ions were designated as “quantifiable spectra” and used for peptide/protein quantitation.

### Publication Abstract
Glyoxalase I (Gly I) is the first enzyme in the glutathionine-dependent glyoxalase pathway for detoxification of methylglyoxal (MG) under stress conditions. Transgenic tomato 'Money Maker' plants overexpressing tomato <i>SlGlyI</i> gene (tomato unigene accession SGN-U582631/Solyc09g082120.3.1) were generated and homozygous lines were obtained after four generations of self-pollination. In this study, <i>SlGlyI-</i>overepxressing line (GlyI), wild type (WT, negative control) and plants transformed with empty vector (ECtr, positive control), were subjected to Al-treatment by growing in Magnavaca's nutrient solution (pH 4.5) supplemented with 20&#x2009;&#xb5;M Al<sup>3+</sup> ion activity. After 30 days of treatments, the fresh and dry weight of shoots and roots of plants from Al-treated conditions decreased significantly compared to the non-treated conditions for all the three lines. When compared across the three lines, root fresh and dry weight of GlyI was significant higher than WT and ECtr, whereas there was no difference in shoot tissues. The basal 5&#x2009;mm root-tips of GlyI plants expressed a significantly higher level of glyoxalase activity under both non-Al-treated and Al-treated conditions compared to the two control lines. Under Al-treated condition, there was a significant increase in MG content in ECtr and WT lines, but not in GlyI line. Quantitative proteomics analysis using tandem mass tags mass spectrometry identified 4080 quantifiable proteins and 201 Al-induced differentially expressed proteins (DEPs) in root-tip tissues from GlyI, and 4273 proteins and 230 DEPs from ECtr. The Al-down-regulated DEPs were classified into molecular pathways of gene transcription, RNA splicing and protein biosynthesis in both GlyI and ECtr lines. The Al-induced DEPs in GlyI associated with tolerance to Al<sup>3+</sup> and MG toxicity are involved in callose degradation, cell wall components (xylan acetylation and pectin degradation), oxidative stress (antioxidants) and turnover of Al-damaged epidermal cells, repair of damaged DNA, epigenetics, gene transcription, and protein translation. A protein-protein association network was constructed to aid the selection of proteins in the same pathway but differentially regulated in GlyI or ECtr lines. Proteomics data are available via ProteomeXchange with identifiers PXD009456 under project title '25Dec2017_Suping_XSexp2_ITAG3.2' for <i>SlGlyI-</i>overexpressing tomato plants and PXD009848 under project title '25Dec2017_Suping_XSexp3_ITAG3.2' for positive control ECtr line transformed with empty vector.

### Keywords
Salt, Root, Aluminum

### Affiliations
USDA-ARS

### Submitter
Ted Thannhauser

### Lab Head
Dr Theodore Thannhauser
USDA-ARS


