### Accession
PXD016069

### Title
Olfactory proteomics in Dementia with Lewy bodies

### Description
Proteomic analysis were performed in olfactory bulbs derived from controls and individuals with dementia with lewy bodies

### Sample Protocol
Sample preparation for proteomic analysis - OB specimens derived from DLB cases and non-neurological subjects were homogenized using ‘’mini potters’’ in lysis buffer containing 8 M urea, 0,1 M TEAB as previously described 31. The homogenates were spanned down at 14.000 x g for 15min at 4°C. Protein quantitation was performed with the Bradford assay kit (Bio-Rad). Reduction, alkylation, digestion and TMT labeling – For the proteomic analysis, 25 µg of protein from each sample were used. Proteins were first reduced with 10mM TCEP for 1h at 30ºC. Then, alkylation was performed for 30 min at room temperature using 40mM IAA. Samples were diluted 5 times to reduce the Urea concentration <1M and protein enzymatic cleavage was carried out overnight with trypsin (Promega; 1:50, w/w) at 37°C. Subsequently, each of the samples was labeled with one of the TMT reagents according to the manufacturer’s instructions (Thermo Scientific). Thereby, control cases were labeled with 127N, 127C, 128N, and 128C and disease cases were labeled with 129N, 129C, 130N and 130C. Labels 126 and 131 were used to label two pools of all samples as intern quality control of the procedure. After leaving the labeling reaction for 60min, all samples were pooled and dried in a speed-vacuum.  Off-gel electrophoresis – samples previously purified with C18 Macro Spin Columns (Harvard Apparatus) were fractionated using Off-gel electrophoresis (OGE) separation using Agilent 3100 off-gel fractionator, according to the manufacturer’s instructions. Dried samples were reconstituted using OGE solution and focused using an immobilized pH gradient dry strip (13cm, pH 3-10). Samples were then desalted and purified using MicroSpins C18 columns, dried in a speed-vacuum and stored at -20ºC until analysis.  LC-MS/MS – A LTQ Orbitrap Q-exactive Plus mass spectrometer (Thermo Fisher) coupled with nanoflow high-pressure liquid chromatography (HPLC) was used to analyzed the OGE fractions, as previously described 32. Peptides were reconstituted using 5% ACN 0,1%FA were trapped on 2 cm x 75 µm ID, 3 µm pre-column and separated on an Easy-spray column, 50 cm x 75 µm ID, PepMap C18, 2 µm (Thermo Scientific). The analytical separation was run for 60 min using a gradient of H2O/FA 99.9%/0.1% (solvent A) and CH3CN/FA 99.9%/0.1% (solvent B) at a flow rate of 300 nL/min. For MS survey scans, the OT resolution was set to 140’000 and the ion population was set to 3 × 106 with an m/z window from 350 to 2000. Twenty precursor ions were selected for higher-energy collisional dissociation (HCD) with a resolution of 35’000, an ion population set to 1 × 105 (isolation window of 0.5 m/z) and a normalized collision energy set to 30%.

### Data Protocol
Protein identification – MS data were processed using EasyProt. Peak list were obtained using the 12 OGE fractions and the combination of HCD-CID raw data peak list were generated. These data were submitted to the EasyProt software platform (version 2.3, build 718) that uses Phenyx software (GeneBio, Geneva, Switzerland) for the protein identification 33. The search was performed using Uniprot/Swiss-Prot database (2014-10, 66903). The following criteria were used: Homo Sapiens taxonomy, oxidized methionine as variable modification, and cysteine carbamethylation, TMT10 lysine and TMT10 amino-terminus as fixed modifications. One missed cleavage was selected and parent-ion tolerance was set to 10 ppm and the accuracy of fragment ions to 0.6 Da. Only proteins with a less than 1% false discovery rate (FDR) and at least two different unique peptides were selected for further analysis. A minimum peptide length of 6 amino acids was used. Protein quantification- Isobaric quantification was performed using the Isobar R package 34. The manufacturer’s isotopic distribution data was used to correct the isotopic impurities of TMT10 reporter-ion intensities. The equal median intensity method was used to normalize the reporter intensities. Peptides which did not present reporter intensities were not quantified. To test the ratio’s accuracy and biological significance, technical and biological variability were calculated for each protein ratio. A ratio p value and sample p value were calculated for each variable.

### Publication Abstract
Olfactory dysfunction is one of the prodromal symptoms in dementia with Lewy bodies (DLB). However, the molecular pathogenesis associated with decreased smell function remains largely undeciphered. We generated quantitative proteome maps to detect molecular alterations in olfactory bulbs (OB) derived from DLB subjects compared to neurologically intact controls. A total of 3214 olfactory proteins were quantified, and 99 proteins showed significant alterations in DLB cases. Protein interaction networks disrupted in DLB indicated an imbalance in translation and the synaptic vesicle cycle. These alterations were accompanied by alterations in AKT/MAPK/SEK1/p38 MAPK signaling pathways that showed a distinct expression profile across the OB-olfactory tract (OT) axis. Taken together, our data partially reflect the missing links in the biochemical understanding of olfactory dysfunction in DLB.

### Keywords
Brain, Proteomics, Olfactory bulb

### Affiliations
Proteomics unit
Clinical Neuroproteomics unit Navarrabiomed-Proteored-ISCIII (Spain)

### Submitter
Enrique SantamarÃ­a

### Lab Head
Dr ENRIQUE SANTAMARIA
Clinical Neuroproteomics unit Navarrabiomed-Proteored-ISCIII (Spain)


