### Accession
PXD038815

### Title
Loss of phospholipase PLAAT3 causes a mixed lipodystrophic and neurological syndrome due to impaired PPAR𝛾 signaling

### Description
PLAAT3 is a phospholipid modifying enzyme predominantly expressed in neural and white adipose tissue (WAT). It is a potential drug target for metabolic syndrome as Plaat3 deficiency in mice protects against diet-induced obesity. We identified seven patients from four unrelated consanguineous families, with homozygous loss-of-function variants in PLAAT3, presenting a severe lipodystrophy syndrome (LS) associated with metabolic complications, as well as neurological features including demyelinating neuropathy and intellectual disability. Multi-omics analysis of mouse Plaat3-/- and patient-derived WAT showed enrichment of arachidonic acid-containing membrane phospholipids and a strong decrease in the signaling of PPAR𝛾, the master regulator of adipocyte differentiation. The present dataset describes the proteomics shotgun analysis of WAT samples from Plaat3-/- and Plaat3-/+ mice. We found that proteins involved in fatty acid metabolic and lipid biosynthetic processes were less abundant in Plaat3-/- WAT, whereas proteins involved in autophagy and catabolism were more abundant. In line with transcriptomics data obtained from mice and human WAT, PPAR𝛾 and its coactivators were identified as strong transcriptional regulators within the set of downregulated proteins. Along with validation experiments in PLAAT3-deficient human adipose stem cells, these findings establish PLAAT3 deficiency as a novel hereditary LS with neurological manifestations, caused by a PPAR𝛾-dependent defect in WAT differentiation and function.

### Sample Protocol
Analysis was performed on inguinal WAT of 9 Plaat3-/- (4 male, 5 female) and 9 Plaat3+/- (4 male, 5 female) mice. Inguinal white adipose fat tissue in 1.5ml tubes was disrupted by 2 cycles of grinding with a disposable micropestle and freeze-thawing. Then, 250µl lysis buffer containing 5%SDS and 50mM TEAB, pH8.5 was added to the tissue and grinded. The tubes were spun down, transferred to PIXUL plate, sonicated for 30’and centrifugated for 5’ at 2204xg at RT. The bottom (non-lipid) layer was transferred and protein concentration was measured by BCA assay. 100µg of protein was reduced with 15mM DTT (incubation for 30’ at 55˚C) and alkylated with 30mM IAA (incubation for 15’ at RT in the dark). Phosphoric acid was added (f.c. 1.2%) and samples were diluted 7-fold with binding buffer containing 90%MeOH in 100mM TEAB, pH7.55. The samples were loaded on a S-Trap plate and centrifuged for 2’ at 1,500xg at RT. After protein binding, the S-trapTM plate was washed three times with 200µl binding buffer followed by 2’ centrifugation at 1,500xg at RT. 50mM TEAB containing trypsin (1/100, w/w) was added for digestion ON at 37°C. Peptides were eluted in three times, first with 80µl 50mM TEAB, then with 80µl 0.2% formic acid (FA) in water and finally with 80µl 0.2% FA in water/acetonitrile (ACN) (50/50, v/v) by centrifugation for 2’ at 1,500xg. Eluted peptides were dried completely by vacuum centrifugation.  TMTproTM 18-plex labels (0.5mg) were equilibrated to RT immediately before use and dissolved in 20µl anhydrous ACN. The peptides were re-suspended in 80µl 100mM TEAB (pH8.5), peptide amount was adjusted to 40µg for each sample. Peptides were labeled for 1 hour at RT using 0.5 mg of TMTProTM label (135N;132C;131N;132N;134N;133C;133N;134C;131C: Plaat3+/- replicates; 130C;128N;126C;127C;129C;129N;128C;130N;127N: Plaat3-/- replicates). The reaction was quenched for 15’ at RT by addition of 4.2µl 5% H3NO. The 18 labeled samples were combined, 100µg labeled peptides was isolated, dried by vacuum centrifugation, re-dissolved in 100µl 0.1% TFA and desalted on a C18 OMIX tip. The tip was washed 3x with 100µl pre-wash buffer (0.1% TFA in water/ ACN (20:80, v/v)) and pre-equilibrated 5x with 100µl of wash buffer (0.1% TFA in water) before the sample was loaded on the tip. After peptide binding and 3 washes with 100µl of wash buffer, peptides were eluted 2x with 100µl elution buffer (0.1% TFA in water/ACN (40:60, v/v)). Elutions were combined, dried and re-dissolved in 100µl loading solvent A (0.1% TFA in water/ACN (98:2, v/v)). 95µl was injected for fractionation by RP-HPLC (Agilent series 1200) connected to a Probot fractionator (LC Packings). Peptides were trapped on a 4cm pre-column (in-house, 250µm ID, 5µm C18 beads) for 10’ at 25µl/min and then separated on a 15cm analytical column (in-house, 250µm ID, 3µm C18 beads). Elution was done by a linear gradient from 100% RP-HPLC solvent A (10mM ammonium acetate (pH 5.5) in water/ACN (98:2, v/v)) to 100% RP-HPLC solvent B (70% ACN, 10mM ammonium acetate (pH 5.5)) in 100’ at a constant flow rate of 3 µL/min. 1 minute fractions were collected between 20’ and 92’ and pooled every 24’. All 24 fractions were dried. Each fraction was re-dissolved in 20µl loading solvent A (0.1% TFA in water:ACN (98:2, v:v)). 15µl of the sample was injected for LC-MS/MS analysis an Ultimate 3000 RSLCnano system in-line connected to an Orbitrap Fusion Lumos mass spectrometer.  Trapping was performed at 20 μl/min for 2’ in loading solvent A on a 20mM trapping column (300μm internal diameter (I.D.), 5μm beads). The peptides were separated on a 110cm prototype µPAC™ column. It was kept at a constant temperature of 50°C. Peptides were eluted by a linear gradient reaching 26.4% MS solvent B (0.1% FA in water/ACN (2:8, v/v)) after 45’, 44% MS solvent B at 55 min, 56% MS solvent B at 60’, followed by a 10’ wash at 56% MS solvent B and re-equilibration with MS solvent A (0.1% FA in water). The first 15’ the flow rate was set to 600nl/min after which it was kept constant at 300nl/min. The mass spectrometer was operated in data-dependent mode. Full-scan MS spectra (375-1500m/z) were acquired at a resolution of 120,000 in the Orbitrap analyzer after accumulation to a target AGC value of 400,000 with a maximum injection time of 50ms. The precursor ions were filtered for charge states (2-7 required), dynamic exclusion (60s; +/- 10ppm window) and intensity (minimal intensity of 5E4). The precursor ions were selected in the quadrupole with an isolation window of 0.7Da and accumulated to an AGC target of 1E4 or a maximum injection time of 50ms and activated using CID fragmentation (35% NCE). The fragments were analyzed in the Ion Trap Analyzer at turbo scan rate. 10 most intense MS2 fragments were selected in the ion trap using MS3 multi-notch isolation windows of 2m/z. An orbitrap resolution of 60k was used with an AGC target of 1.0e5 or a maximum injection time of 118ms and activated using HCD fragmentation (65% NCE).

### Data Protocol
LC-MS/MS runs of all 18 samples were searched together using the MaxQuant algorithm (version 2.1.3.0) with mainly default search settings, including a false discovery rate set at 1% on peptide and protein level. Spectra were searched against the mouse protein sequences in the Swiss-Prot database (database release version of January 2022), containing 21,986 sequences (www.uniprot.org). The mass tolerance for precursor and fragment ions was set to 4.5 and 20 ppm, respectively, during the main search. Enzyme specificity was set to C-terminal of arginine and lysine, also allowing cleavage at Arg/Lys-Pro bonds with a maximum of two missed cleavages. Variable modifications were set to oxidation of methionine residues and acetylation of protein N-termini whereas carbamidomethylation of cysteine residues was set as fixed modification. Only proteins with at least one unique or razor peptide were retained leading to the identification of 4,036 proteins. MS3-based quantification using TMTpro labels was chosen as quantification method and a minimum ratio count of 2 unique peptides was required for quantification. Further data analysis of the shotgun results was performed with an in-house R script, using the proteinGroups output table from MaxQuant. Reverse database hits were removed, reporter intensities were log2 transformed, median normalized and replicate samples were grouped. Proteins with less than three valid values in at least one group were removed and missing values were imputed from a normal distribution centered around the detection limit (package DEP), leading to a list of 4,033 quantified proteins in the experiment, used for further data analysis. To compare protein abundance between pairs of sample groups, statistical testing for differences between two group means was performed, using the package limma. Statistical significance for differential regulation was set by a raw p<0.05 and a fold change |log2FC|≥0.5 leading to 41 proteins that were significantly downregulated and 8 proteins that were significantly upregulated in Plaat3-/- WAT.

### Publication Abstract
None

### Keywords
Loss of phospholipase plaat3 causes a mixed lipodystrophic and neurological syndrome due to impaired ppar𝛾 signaling

### Affiliations
VIB-UGent
1Center for Medical Genetics, Ghent University Hospital, Ghent, 9000, Belgium 2Department of Biomolecular Medicine, Faculty of Medicine and Health Sciences, Ghent

### Submitter
Sara Dufour

### Lab Head
Dr Bart Dermaut
1Center for Medical Genetics, Ghent University Hospital, Ghent, 9000, Belgium 2Department of Biomolecular Medicine, Faculty of Medicine and Health Sciences, Ghent


