### Accession
PXD030369

### Title
Omics-based analysis of honey bee (Apis mellifera) response to Varroa sp. interaction reveals changes impairing the winter bee generation

### Description
Extensive annual losses of honey bees (Apis mellifera L.) represent a global problem for agriculture and biodiversity. The parasitic mite Varroa destructor in association with viral co-infections plays a key role in this phenomenon; however, the precise mechanisms are still unclear. We employed a unique combination of transcriptomic, proteomic, metabolomic, and functional analyses to elucidate the effects of Varroa parasitisation. We focused on complex differences between parasitised and unparasitised ten-days old honey bee workers collected from identical colonies before overwintering. Honey bees exposed to mite parasitation during their development revealed alterations in transcriptome and proteome related to immunity, oxidative stress, olfactory recognition, metabolism of sphingolipids and RNA regulatory mechanisms. Specifically, immune reactions and sphingolipids metabolism were strongly up-regulated in parasitised honey bees; whereas olfactory recognition and oxidative stress pathways were down-regulated compared to unparasitised bees. Additionally, the metabolomic analysis confirmed the depletion of nutrients, decreased energy stores and generally disrupted metabolism of parasitised workers, as previously reported. By virtue of comprehensive omics-based analysis, we define the key changes in the honey bee facing Varroa parasitism and suggest possible mechanisms underlying its detrimental effects. This study provides a theoretical basis for future efforts in efficient control strategies against Varroa mites.

### Sample Protocol
Control and varroa positive honeybees were obtained from colonies in Kyvalka (GPS location: 49.18867474N, 16.4513211E)and (GPS location: 49.2245775N, 16.4625528E) respectively. Samples were collected and stored at -80 until ready for protein extraction. Heads were removed from the body and the honey stomach and intestines were removed from the abdomens. Samples were homogenised using a hand held pestle and proteins extracted with 300ul (head) and 400ul (abdomen) respectively. Samples were centrifuged at 10,000 g for 5 min. All supernatant was collected and acetone precipitated overnight. Samples were resuspended in resuspension buffer 120ul (head) and 350ul (abdomen) respectively. Samples were digested using ammonium bicarbonate, 0.5M dithiothreitol, 0.5M iodoacetamide and Sequence Grade Trypsin (0.5ug/ml). Digested samples were purified using C18 spin columns and dried in a SpeedyVac concentrator for 2 hours. 1ug of head proteins and .75ug of abdomen protein samples were loaded onto the mass spec for 2 and 1 hours respectively.

### Data Protocol
The following search parameters were used: first search peptide tolerance of 20 ppm, second search peptide tolerance 4.5 ppm, carbamidomethylation of cysteine of N-terminals were set as variable modifications and a maximum of 2 missed cleavage sites allowed. FDR were set to 1 % for both peptides and proteins, and the FDR was estimated following searches against a target-decoy database. Peptides with a minimum length of seven amino acids were considered for identification and proteins were only considered identified when more than one unique peptide for each protein was observed. MaxQuant data was analysed in Perseus v. 1.6.6.0. label-free quantification intensities were log2 transformed and proteins that had non-existent values (indicative of absence or very low abundance in a sample) were used in statistical analysis following the imputation of representative numbers based on the lowest value for each data set, which was calculated as a 1.8 downshift from the mean value allowing for 0.3 width in the downshift for the standard deviation. ANOVA significance was calculated for the experimental groups using a p-value of 0.05, only ANOVA significant proteins were used for further analysis.

### Publication Abstract
The extensive annual loss of honey bees (Apis mellifera L.) represents a global problem affecting agriculture and biodiversity. The parasitic mite Varroa destructor, associated with viral co-infections, plays a key role in this loss. Despite years of intensive research, the complex mechanisms of Varroa - honey bee interaction are still not fully defined. Therefore, this study employed a unique combination of transcriptomic, proteomic, metabolomic, and functional analyses to reveal new details about the effect of Varroa mites and naturally associated factors, including viruses, on honey bees. We focused on the differences between Varroa parasitised and unparasitised ten-day-old worker bees collected before overwintering from the same set of colonies reared without anti-mite treatment. Supplementary comparison to honey bees collected from colonies with standard anti-Varroa treatment can provide further insights into the effect of a pyrethroid flumethrin. Analysis of the honey bees exposed to mite parasitisation revealed alterations in the transcriptome and proteome related to immunity, oxidative stress, olfactory recognition, metabolism of sphingolipids, and RNA regulatory mechanisms. The immune response and sphingolipid metabolism were strongly activated, whereas olfactory recognition and oxidative stress pathways were inhibited in Varroa parasitised honey bees compared to unparasitised ones. Moreover, metabolomic analysis confirmed the depletion of nutrients and energy stores, resulting in a generally disrupted metabolism in the parasitised workers. The combined omics-based analysis conducted on strictly parasitised bees revealed the key molecular components and mechanisms underlying the detrimental effects of Varroa sp. and its associated pathogens. This study provides the theoretical basis and interlinked datasets for further research on honey bee response to biological threats and the development of efficient control strategies against Varroa mites.

### Keywords
Apis mellifera, Disease, Proteomics, Varroa destructor

### Affiliations
Maynooth Univeristy
Lab Head, Medical Mycology Lab

### Submitter
Rachel Ward

### Lab Head
Dr Kevin Kavanagh
Lab Head, Medical Mycology Lab


