### Accession
PXD040856

### Title
Comparison of gel and urea methods of digestion for proteomic analysis of organ perfusion solutions

### Description
The aim of this study was to develop an optimized Shotgun proteomics method for perusate by comparing and identifying the most effective and reliable techniques for protein estimation and digestion.

### Sample Protocol
Perfusate samples were obtained from the COPE COMPARE kidney trial and the liver defatting study. For gel digestion, sample proteins were separated by SDS-PAGE and Coomassie stained, followed by tryptic digestion and extraction of peptides. For urea digestion, reducing, alkylating and digesting agents were added directly to the protein sample without prior gel separation. Both sets of digests underwent label-free LC-MS/MS analysis on an Orbitrap Fusion instrument.

### Data Protocol
MS raw data files were processed using Proteome Discoverer version 2.2 (Thermo Fisher), and searched against the Homo sapiens UniProt database version 2021 containing 564,918 protein sequences.The MS1 mass tolerance was set to 10ppm, while fragment mass tolerance for MS/MS spectra was set to 0.6 Da. oxidation of methionine (M) and deamidation of asparagine (N) and glutamine (Q), were specified as dynamic modifications while carbamidomethylation of cysteine (C) residues was chosen as a static modification.

### Publication Abstract
None

### Keywords
Liver, Proteomic, Organ transplantation, Kidney, Perfusate

### Affiliations
Nuffield department of Surgical Sciences, University ofOxford UK
Nuffield Department of Surgical Sciences, University of Oxford, Oxford, United Kingdom

### Submitter
Sadr Shaheed

### Lab Head
Dr Dr Sadr ul Shaheed
Nuffield department of Surgical Sciences, University ofOxford UK


