### Accession
PXD002247

### Title
The proteome of native adult Müller cells from the murine retina

### Description
We set out to first establish and validate a novel rapid method of enriching adult Müller cells, which yields sufficient cell numbers for proteomic profiling while keeping cells viable and morphologically, as well as physiologically fully intact. From these preparations we collected a comprehensive data set of the proteome of native murine Müller cells. Since many Müller cell functions depend on cell-cell interactions occurring at the plasma membrane level, we additionally included a subcellular fractionation approach that enabled to discriminate proteins primarily residing in the cytosol or the nucleus from those allocated in the plasma membrane and thus generate a first time Müller cell surfaceome study. We consider these data a first important step to better understanding the complex crosstalk between various retinal cell types and Müller cells as key players in health and ultimately also under disease conditions.

### Sample Protocol
Müller cell isolation and MACS sorting Subcellular fractionation of isolated retinal cell types iST sample preparation qRT-PCR Immunofluorescent labelings of retinal slices Mass Spectrometry LC-MS/MS analysis was performed on an LTQ OrbitrapXL (Thermo Fisher Scientific Inc., Waltham, MA, U.S.A.). Approximately 0.5 μg of sample was automatically loaded to the HPLC system. A nano trap column was used (300 μm inner diameter × 5 mm, packed with Acclaim PepMap100 C18. 5 μm, 100 Å; LC Packings, Sunnyvale, CA) before separation by reversed phase chromatography (PepMap, 25 cm, 75 µm ID, 2µm/100 Å pore size, LC Packings) operated on a RSLC (Ultimate 3000, Dionex, Sunnyvale, CA). A 170 min run constitutes from the following gradients of ACN in 0.1% formic acid (FA): 135 minutes of 6 % to 31 % and 10 minutes from 31 % to 72 % ACN. Between each gradient the ACN in 0.1% FA concentration was set back to starting conditions for 20 minutes. From the high resolution MS prescan, the 10 most abundant peptide ions were selected for fragmentation in the linear ion trap if they were at least doubly charged and if they exceeded an intensity of at least 200 counts, with a dynamic exclusionof 60 seconds. During fragment analysis, a high-resolution (60,000 full width at half-maximum) MS spectrum was acquired with a mass range from 200 to 1500 Da.

### Data Protocol
Label-free analyses (includes database search and protein identification The acquired spectra of the different samples were loaded and analyzed using Progenesis QI software for proteomics (Version 2.0, Nonlinear Dynamics, Waters, Newcastle upon Tyne, U.K.) for label-free quantification as previously described (22). The profile data of the MS scans were transformed into peak lists with respective m/z values, intensities, abundances, and m/z width. MS/MS spectra were transformed similarly and then stored in peak lists comprising m/z and abundance. Using one sample as reference, the retention times of the other samples were aligned by automatic alignment to a maximal overlay of the 2D features. Features with one or more than seven charges were masked at this point and excluded from further analyses. After alignment and feature exclusion, samples were allocated to their respective experimental groups (Müller cell-enriched versus –depleted fractions), and raw abundances of all features were normalized. Normalization corrects for factors resulting from experimental variation and was automatically calculated over all features in all samples to correct for technical variation. All MS/MS spectra were exported from the Progenesis QI software as Mascot generic files (mgf) and used for peptide identification with Mascot (version 2.5) using the Ensembl Mouse protein database (mus musculus; release 75, containing 51772 sequences). Search parameters used were 10 ppm peptide mass tolerance, 0.6 Da fragment mass tolerance, one missed cleavage allowed, carbamidomethylation was set as fixed modification, and methionine oxidation and deamidation of asparagine and glutamine as variable modifications. Mascot integrated decoy database search was set to a false discovery rate (FDR) of 1% when searching was performed on the concatenated mgf files with a percolator ion score cut-off of 13 and an appropriate significance threshold p. Identifications were re-imported into Progenesis QI. For quantification, all unique peptides of an identified protein were included and the total cumulative normalized abundance was calculated by summing up the abundances of all unique peptides allocated to the respective protein. No minimal thresholds were set neither for the method of peak picking nor selection of data to use for quantification. Statistical analysis was performed using transformed (“log-like” arcsinh(.)function) normalized abundances for one-way analysis of variance (ANOVA) calculations of all detected features. ANOVA values of p < 0.05 and additionally regulation of ≥2-fold or ≤ 0.5-fold were regarded as significant for all further results.  Heat maps Heat maps of significantly altered proteins were generated for the two different sample preparation methods separately, using the heat map function in the Excel (Microsoft) addin XLStat including the OMICS module (Addinsoft). Normalized protein abundances were used for independent clustering of samples and proteins, applying hierarchical clustering based on Euclidian distances. The corresponding colored matrices are displayed in figure 3, with high abundant proteins in the respective sample displayed in green and low abundant proteins in red.  Classification of identified proteins In order to assess subcellular localization of identified proteins, FASTA files were generated and submitted to Phobius, a combined transmembrane topology and signal peptide predictor (http://phobius.sbc.su.se/index.html, (23)). Proteins were accordingly classified into three categories depending on the presence of an exclusive secretory signal peptide (cell surface proteins), a secretory signal peptide in combination with at least one transmembrane domain (plasma membrane proteins) or no signal peptide with or without (a) transmembrane domain(s) (other proteins).   Pathway enrichment analyses and Network generation Pathway enrichment analysis was performed with Genomatix Generanker (http://www.genomatix.de, Genomatix). Gene names of significantly different proteins were uploaded and corresponding overrepresented pathways and processes are given in table 2.  For protein network generation the Genomatix Pathway System (GePS, http://www.genomatix.de, Genomatix) was used. Corresponding gene names and fold changes of significantly different proteins were uploaded and networks were created using Mus musculus as the organism with literature mining conducted at the function word level. Statistics Data are expressed as mean ± standard error (SEM). Statistical analysis was made using Prism (Graphpad Software, San Diego, CA, USA); significance was determined by the non-parametric Mann-Whitney U test.

### Publication Abstract
To date, the proteomic profiling of M&#xfc;ller cells, the dominant macroglia of the retina, has been hampered because of the absence of suitable enrichment methods. We established a novel protocol to isolate native, intact M&#xfc;ller cells from adult murine retinae at excellent purity which retain in situ morphology and are well suited for proteomic analyses. Two different strategies of sample preparation - an in StageTips (iST) and a subcellular fractionation approach including cell surface protein profiling were used for quantitative liquid chromatography-mass spectrometry (LC-MSMS) comparing M&#xfc;ller cell-enriched to depleted neuronal fractions. Pathway enrichment analyses on both data sets enabled us to identify M&#xfc;ller cell-specific functions which included focal adhesion kinase signaling, signal transduction mediated by calcium as second messenger, transmembrane neurotransmitter transport and antioxidant activity. Pathways associated with RNA processing, cellular respiration and phototransduction were enriched in the neuronal subpopulation. Proteomic results were validated for selected M&#xfc;ller cell genes by quantitative real time PCR, confirming the high expression levels of numerous members of the angiogenic and anti-inflammatory annexins and antioxidant enzymes (e.g. paraoxonase 2, peroxiredoxin 1, 4 and 6). Finally, the significant enrichment of antioxidant proteins in M&#xfc;ller cells was confirmed by measurements on vital retinal cells using the oxidative stress indicator CM-H2DCFDA. In contrast to photoreceptors or bipolar cells, M&#xfc;ller cells were most efficiently protected against H2O2-induced reactive oxygen species formation, which is in line with the protein repertoire identified in the proteomic profiling. Our novel approach to isolate intact glial cells from adult retina in combination with proteomic profiling enabled the identification of novel M&#xfc;ller glia specific proteins, which were validated as markers and for their functional impact in glial physiology. This provides the basis to allow the discovery of novel glial specializations and will enable us to elucidate the role of M&#xfc;ller cells in retinal pathologies - a topic still controversially discussed.

### Keywords
Macs, Muller glia cell, Lc-msms, Retina, Fasp

### Affiliations
Research Unit Protein Science Helmholtz Zentrum Muenchen German Research Center for Environmental Health (GmbH) D-85764 Neuherberg Germany Ph.: +49-89-3187-3941 Fax: +49-89-3187-4426
Helmholtz Zentrum München

### Submitter
Christine von Toerne

### Lab Head
Dr Stefanie Hauck
Research Unit Protein Science Helmholtz Zentrum Muenchen German Research Center for Environmental Health (GmbH) D-85764 Neuherberg Germany Ph.: +49-89-3187-3941 Fax: +49-89-3187-4426


