### Accession
PXD025709

### Title
Human colorectal cancer LC-MS/MS

### Description
The global crotonylome of human colorectal cancer and paired paracancerous tissues using quantitative proteomics.

### Sample Protocol
Briefly, proteins were extracted from these four pairs of tissue samples followed by trypsin digestion, enrichment of crotonylation modified peptides, and analysis using a trapped ion mobility quadrupole time-of-flight mass spectrometer (timsTOF Pro)

### Data Protocol
Database search MaxQuant search engine (V.1.6.6.0, http://www.maxquant.org/) software was used to analyze the MS/MS raw data. Tandem mass spectra were searched against the Human_SwissProt concatenated with reverse decoy database. Trypsin/P was specified as cleavage enzyme allowing up to 4 missing cleavages. The mass tolerance for precursor ions was set as 40 ppm in First search and 40 ppm in Main search, and the mass tolerance for fragment ions was set as 0.04 Da. Carbamidomethyl on cysteine was specified as fixed modification, and acetylation on protein N-terminal, oxidation on methionine, and crotonylation on lysine were specified as variable modifications. False discovery rate (FDR) was adjusted to < 1%.  Bioinformatics analysis Gene Ontology (GO) annotation proteome was derived from the UniProt-GOA database (V.5.14-53.0, http://www.ebi.ac.uk/interpro/). Kyoto Encyclopedia of Genes and Genomes (KEGG) database was used to annotate protein pathway. KEGG online service tool KAAS (V.2.0, http://www.genome.jp/kaas-bin/kaas_main) was used to annotate the submitted protein, and then annotated protein matched into the corresponding pathway in the database through KEGG mapper V2.5, (http://www.kegg.jp/kegg/mapper.html). Wolfpsort (V.0.2, http://www. genscript.com/psort/wolf_psort.html), a subcellular localization predication soft was used to predict subcellular localization. Soft MoMo (V5.0.2, http://meme-suite.org/tools/momo) was used to analysis the model of sequences constituted with amino acids in specific positions of modify-21-mers (10 amino acids upstream and downstream of the site) in all protein sequences. GO term and KEGG pathway enrichment were performed using t test with P < 0.05. R Package pheatmap (V.2.0.3, https://cran.r-project.org/web/packages/cluster/) was used for enrichment-based clustering. All differentially expressed modified protein database accession or sequence were searched against the STRING database version 10.1 for protein-protein interactions and R package networkD3 (V.0.4, https://cran.r-project.org/web/packages /networkD3/) was used to display the network.

### Publication Abstract
None

### Keywords
Human, Colorectal cancer, Lc-ms/ms

### Affiliations
Shanxi Medical University

### Submitter
Jiayi Hou

### Lab Head
Dr Jiayi Hou
Shanxi Medical University


