### Accession
PXD004264

### Title
Paradigm of biased PAR1 activation and inhibition in endothelial cells dissected by phosphoproteomics

### Description
Objective: Thrombin is the key serine protease of the coagulation cascade and mediates cellular responses by activation of protease-activated receptors (PARs). The predominant thrombin receptor is PAR1 and in endothelial cells (ECs) thrombin dynamically regulates a plethora of phosphorylation events. However, it has remained unclear if thrombin signaling is exclusively mediated through PAR1. Furthermore, mechanistic insight into activation and inhibition of PAR1-mediated EC signaling is lacking. In addition, signaling networks of biased PAR1 activation after differential cleavage of the PAR1 N-terminus have remained an unresolved issue.
Approach and Results: Here, we used a quantitative phosphoproteomics approach to show that ‘classical’ and ‘peptide’ activation of PAR1 induce highly similar signaling, that low thrombin concentrations initiate only limited phosphoregulation, and that the PAR1 inhibitors vorapaxar and parmodulin-2 demonstrate distinct antagonistic properties. Subsequent analysis of the thrombin-regulated phosphosites in presence of PAR1 inhibitors revealed that biased activation of PAR1 is not solely linked to a specific G-protein downstream of PAR1. In addition, we showed that only the canonical thrombin PAR1 tethered ligand induces extensive early phosphoregulation in ECs.
Conclusions: Our study provides detailed insight in the signaling mechanisms downstream of PAR1. Our data demonstrates that thrombin-induced EC phosphoregulation is mediated exclusively through PAR1, that thrombin and thrombin-TL peptide induce similar phosphoregulation and that only canonical PAR1 cleavage by thrombin generates a tethered ligand that potently induces early signaling. Furthermore, platelet PAR1 inhibitors directly affect EC signaling, indicating it will be a challenge to design a PAR1 antagonist that will target only those pathways responsible for tissue pathology.

### Sample Protocol
Proteome analysis and phosphopeptide isolation were performed as described previously with minor modifications (6). Briefly, light, medium and heavy SILAC-labeled cell-lysates were mixed together in a 1:1:1 ratio. Mixed proteins were reduced, alkylated and digested with trypsin using the FASP method (44). For proteome analysis, digested peptides were acidified to pH 2.5 with Trifluoroacetic acid (TFA), loaded onto Empore-C18 StageTips (45), eluted with 80% ACN, 0.5% acetic acid, and stored at -80°C until MS analysis. Phosphopeptides were enriched using Phos-TiO beads (GL Sciences) in the presence of 80% ACN, 6% TFA (46), eluted with 15% ammonium hydroxide, 40% ACN, loaded onto Empore-C18 StageTips (45), eluted with 80% ACN, 0.5% acetic acid, and stored at -80°C until MS analysis. The Phos-TiO flow through was collected and processed as described above performing a second round of Phos-TiO enrichment, which was run separately at the MS. Digested peptides were separated by nanoscale C18 reverse chromatography coupled on line to an Orbitrap Fusion Tribrid mass spectrometer (Thermo Scientific) via a nanoelectrospray ion source (Nanospray Flex Ion Source, Thermo Scientific). Peptides were loaded on a 20 cm 75–360 µm inner-outer diameter fused silica emitter (New Objective) packed in-house with ReproSil-Pur C18-AQ, 1.9 μm resin (Dr Maisch GmbH). The column was installed on a Dionex Ultimate3000 RSLC nanoSystem (Thermo Scientific) using a MicroTee union formatted for 360 μm outer diameter columns (IDEX) and a liquid junction. The spray voltage was set to 2.15 kV. Buffer A was composed of 0.5 % acetic acid and buffer B of 0.5 % acetic acid, 80% acetonitrile. Peptides were loaded for 17 min at 300 nl/min at 5% buffer B, equilibrated for 5 minutes at 5%  buffer B (17-22 min) and eluted by increasing buffer B from 5-15% (22-87 min) and 15-38% (87-147 min), followed by a 10 minute wash to 90% and a 5 min regeneration to 5%. For proteome: survey scans of peptide precursors from 400 to 1500 m/z were performed at 240K resolution (at 200 m/z) with a 1.5 × 105 ion count target and maximum injection time of 50 ms. Tandem mass spectrometry was performed by isolation with the quadrupole with isolation window 1.6 m/z, HCD fragmentation with normalized collision energy of 30, and rapid scan mass spectrometry analysis in the ion trap. The MS2 ion count target was set to 1 × 104 and maximum injection time of 35 ms. The instrument was run in top speed mode with 3 s cycles. For phosphoproteome: survey scans of peptide precursors from 400 to 1500 m/z were acquired in the Orbitrap analyzer at 240K resolution (at 200 m/z) with 2 × 105 ion count target and maximum injection time of 50 ms. Tandem mass spectrometry was performed on the 10 most intense ions by isolation with the quadrupole with isolation window 1.6 m/z, HCD fragmentation with normalized collision energy of 30, and analysis in the Orbitrap with a resolution of 30K (at 200 m/z). The MS2 ion count target was set to 5 × 104 and maximum injection time of 60 ms. Only those precursors with charge state 2–7 were sampled for MS2. The dynamic exclusion duration was set to 60 s with a 10 ppm tolerance around the selected precursor and its isotopes. Monoisotopic precursor selection was turned on. All data were acquired with Xcalibur software.

### Data Protocol
For the thrombin/PAR1 inhibitor dataset: The RAW MS files were processed with MaxQuant computational platform (version 1.5.2.8). Proteins and peptides were identified using the Andromeda search engine by querying the human Uniprot (release 2015-02). For the tethered ligand peptides dataset: The RAW MS files were processed with MaxQuant computational platform (version 1.5.3.30). Proteins and peptides were identified using the Andromeda search engine by querying the human Uniprot (release 2017-03). Precursor and fragment ions were selected using standard settings. Trypsin with full enzyme specificity and only peptides with a minimum length of 7 amino acids were selected. A maximum of two missed cleavages was allowed. Carbamidomethylation (Cys) was set as fixed modification, while Oxidation (Met), N-acetylation and phospho(STY) as variable modifications. For protein, peptide and phosphorylation site identification, we required a maximum false discovery rate (FDR) of 1%. The relative quantification based on SILAC was performed by MaxQuant enabling the ‘re-quantify’ and ‘match between runs’ options. MS data processing was performed using Perseus version 1.5.5.1 as described before.(3) Briefly, first, reverse and contaminant hits were removed from the MaxQuant output files. Next, only class I phosphorylation sites, which refer to phosphorylation sites where the phosphorylation could be assigned with a localization probability of at least 0.75 and a score difference ≥ 5(9), were used for the analysis. For each experiment, the normalized SILAC ratios were transformed using the binary logarithm (log2). For each phosphorylation site, the SILAC ratios were calculated separately for singly, doubly or multiply phosphorylated peptides. Phosphosites were compared with the PhosphoSitePlus database(10) (release 20 March 2017) to determine not previously reported phosphosites. For the thrombin/PAR1 inhibitor dataset: MS data was combined from 3 (10 nM thrombin), 2 (thrombin-TL peptide, vorapaxar + 10 nM thrombin and parmodulin-2 + 10 nM thrombin) or 1 (50 pM thrombin) biological replicate(s), resulting in the following replicates per time point: 7 (10 nM thrombin), 4 (thrombin-TL peptide and parmodulin-2 + 10 nM thrombin), 6 (vorapaxar + 10 nM thrombin), or 3 (50 pM thrombin). Quantified phosphosites were defined as phosphosites where a SILAC ratio was accurately measured in the following number of replicates at each time point: 5 out of 7 (10 nM thrombin), 3 out of 4 (thrombin-TL peptide and parmodulin-2 + 10 nM thrombin ), 4 out of 6 (vorapaxar + 10 nM thrombin) or 3 out of 3 (50 pM thrombin). Regulated phosphosites were defined as phosphosites where the SILAC ratio (log2) was higher than 0.6 in the following number of replicates per time point: 5 out of 7 (10 nM thrombin), 3 out of 4 (thrombin-TL peptide and parmodulin-2 + 10 nM thrombin), 4 out of 6 (vorapaxar + 10 nM thrombin) or 3 out of 3 (50 pM thrombin). For the tethered ligand peptides dataset: MS data was combined from 2 biological replicates, resulting in two replicates per time point per stimulation condition. Quantified phosphosites were defined as phosphosites where a SILAC ratio was accurately measured in both replicates at each time point for each stimulation condition. Regulated phosphosites were defined as phosphosites where the SILAC ratio (log2) was higher than 0.7 in both replicates of a time point for one of the five stimulation conditions. Interaction network of the regulated phosphosites was made using the Phosphopath plug-in from Cytoscape.

### Publication Abstract
None

### Keywords
Phosphoproteomics, Par1, Par1-inhibitors, Thrombin, Silac, Endothelial cells

### Affiliations
Sanquin Research
Dept. Plasma Proteins, Sanquin Research, Amsterdam, The Netherlands

### Submitter
Bart van den Eshof

### Lab Head
Dr Maartje van den Biggelaar
Dept. Plasma Proteins, Sanquin Research, Amsterdam, The Netherlands


