### Accession
PXD021164

### Title
RBPome of primary human T helper cells

### Description
Post-transcriptional gene regulation is complex, dynamic and ensures proper T cell function. The targeted transcripts can simultaneously respond to various factors as evident for Icos, an mRNA regulated by several RNA binding proteins (RBPs), including Roquin. However, fundamental information about the entire RBPome involved in post-transcriptional gene regulation in T cells is lacking. Here, we applied global RNA interactome capture (RNA-IC) and orthogonal organic phase separation (OOPS) to human and mouse primary T cells and identified the core T cell RBPome. This defined 798 mouse and 801 human proteins as RBPs, unexpectedly containing signaling proteins like Stat1, Stat4 and Vav1. The data represent an atlas of RNA-binding proteins in human and mouse T helper cells as a resource for studying higher order post-transcriptional gene regulation.

### Sample Protocol
Eluates from the RBPome capture were incubated with 10ug/ml Rnase A in 100mM Tris, 50mM NaCl, 1mM EDTA at 37°C at 37°C for 30 min. Rnase-treated eluates were acetone precipitated and resuspended in denaturation buffer (6 M urea, 2 M thiourea, 10 mM Hepes pH 8), reduced with 1mM DTT and alkylated with 5.5 mM IAA. Samples were diluted 1:5 with 62.5 mM Tris pH 8.1 and proteins digested with 0.5 ug Lys-C and 0.5 ug Trypsin at room temperature over night. The resulting peptides were desalted using stage-tips containing C18 material (3M, St. Paul). Peptides were separated on a reverse phase column (50 cm length, 75 μm inner diameter) packed in-house with ReproSil-Pur C18-AQ 1.9 µm resin (Dr. Maisch GmbH). Reverse-phase chromatography was performed with an EASY-nLC 1000 ultra-high pressure system, coupled to a Q-Exactive HF-X Mass Spectrometer (Thermo Scientific). Peptides were loaded with buffer A (0.1% (v/v) formic acid) and eluted with a nonlinear 100-min gradient of 5–60% buffer B (0.1% (v/v) formic acid, 80% (v/v) acetonitrile) at a flow rate of 300 nl/min. After each gradient, the column was washed with 95% buffer B and re-equilibrated with buffer A. Column temperature was kept at 60° C by an in-house designed oven with a Peltier element and operational parameters were monitored in real time by the SprayQc software. MS data were acquired using a data-dependent top 12 method in positive mode. Target value for the full scan MS spectra was 3 × 106 charges in the 300−1,650 m/z range with a maximum injection time of 20 ms and a resolution of 60,000. The precursor isolation windows was set to 1.4 m/z and capillary temperature was 250°C. Precursors were fragmented by higher-energy collisional dissociation (HCD) with a normalized collision energy (NCE) of 27. MS/MS scans were acquired at a resolution of 15,000 with an ion target value of 1 × 105, a maximum injection time of 60 ms. Repeated sequencing of peptides was minimized by a dynamic exclusion time of 30 s.

### Data Protocol
MS raw files were analyzed by the MaxQuant software (version 1.5.1.6) and peak lists were searched against the human Uniprot FASTA database, and a common contaminants database (247 entries) by the Andromeda search engine. Cysteine carbamidomethylation was set as fixed modification, methionine oxidation and N-terminal protein acetylation as variable modifications. False discovery rate was 1% for proteins and peptides (minimum length of 7 amino acids) and was determined by searching a reverse database. Peptides specificity was set to trypsin requiring C-terminal arginine or lysine with a maximum of two missed cleavages. Maximal allowed precursor mass deviation for peptide identification was 4.5 ppm after time-dependent mass calibration and maximal fragment mass deviation was 20ppm. “Match between runs” was activated with a retention time alignment window of 20 min, a match time window of 0.5 min, and a minimum ratio setting of 2.

### Publication Abstract
None

### Keywords
Human, Rna interaction capture, Rna binding proteins

### Affiliations
Department of Proteomics and Signal Transduction, Max-Planck-Institute of Biochemistry, Munich, Germany
Department of Proteomics and Signal Transduction, Max-Planck Institute of Biochemistry, Martinsried, Germany

### Submitter
Michael Wierer

### Lab Head
Dr Michael Wierer
Department of Proteomics and Signal Transduction, Max-Planck-Institute of Biochemistry, Munich, Germany


