### Accession
PXD033468

### Title
Discovery and characterization of a new class of O-linking oligosaccharyltransferase from bacteria in the Moraxellaceae family

### Description
Bacterial protein glycosylation can be mediated by oligosaccharyltransferases (OTases) that transfer a preassembled lipid-linked oligosaccharide or polysaccharide en bloc to acceptor proteins. O-linking OTases transfer O-antigen or capsular polysaccharides to the side chains of serine or threonine residues. Three major families of bacterial O-linking OTases have been described so far: PglL, PglS, and TfpO. TfpO enzymes are limited to transferring only short glycans whereas there are no clear upper limits for the other two families. Herein, we describe the discovery of a novel family of bacterial O-linking OTases from Moraxellacea bacteria that are similar in size and sequence to TfpO enzymes but can transfer long-chain bacterial glycans to acceptor proteins. Bioinformatic analyses show that these enzymes cluster in different clades than known bacterial OTases. Using a representative enzyme from Moraxella osloensis termed TfpMMo, we determine that the enzyme glycosylates the C-terminal amino acid side chain of a pilin protein and find that pilin fragments as short as three amino acids are substrates for the OTase. The ability of TfpMMo to transfer long-chain polysaccharide shows that this ability is not limited to the PglS and PglL families. TfpMMo is also shown to have broad substrate specificity and can transfer diverse glycans including those with glucose, galactose, or 2-N-acetyl sugars at the reducing end. The glycan substrate promiscuity of TfpMMo could allow this enzyme to be used to produce bacterial glycoconjugate vaccines. The discovery of a new class of O-linking OTase furthers our understanding of the mechanisms that underly glycan specificity by these and other O-linking OTases and enables more comparative studies of this important enzyme family.

### Sample Protocol
In gel sample: Lys-C digestion of recombinant M. osloensis PilAMo In-gel digestion was accomplished according to the protocol of Shevchenko et al (44) with minor modifications. Gel-separated glycosylated PilMo was excised and destained with destaining solution (50 mM NH4HCO3, 50% ethanol) twice for 10 min. at room temperature with shaking at 750 RPM. The destained band was then dehydrated with 100% ethanol for 10 min. and dried by vacuum-centrifugation for 10 min. before being rehydrated in 10 mM DTT in 50 mM NH4HCO3. Reduction was carried out for 60 min. at 56°C after which the gel band was dehydrated twice with 100% ethanol for 10 mins to remove the remaining reduction buffer. The reduced sample was then sequentially alkylated with 55 mM iodoacetamide in 50 mM NH4HCO3 for 45 min. at RT in the dark. The alkylated sample was then washed four times for 10 min. with 50 mM NH4HCO3 followed by 100% ethanol, followed by 50 mM NH4HCO3 followed by 100% ethanol before being dried by vacuum-centrifugation. The dried alkylated sample was then rehydrated with 20 ng/µl Lys-C endoprotease (Wako Chemicals) in 40 mM NH4HCO3 at 4°C for 1hr. Excess Lys-C was removed, gel pieces were covered in 40mM NH4HCO3 and incubated overnight at 37°C. Peptides were concentrated and desalted using C18 stage tips (45,46) then eluted in buffer B (0.5% acetic acid, 80% acetonitrile (ACN)) before being dried and stored at -20°C prior to LC-MS analysis.  In Solution: M. osloensis CCUG350 glycoproteomic sample preparation Acetone precipitated M. osloensis CCUG350 / ATCC19976 samples were processed using the in-StageTip preparation approach as previously described (55). Cells were resuspended in 4% sodium deoxycholate (SDC), 100 mM Tris pH 8.0 and boiled at 95˚C with shaking for 10 minutes to solubilise the proteome. Samples were allowed to cool for 10 minutes and then boiled for a further 10 minutes before the protein concentration was determined by bicinchoninic acid assays (Thermo Fisher Scientific). Samples were reduced / alkylated with the addition of Tris-2-carboxyethyl phosphine hydrochloride and Chloroacetamide (final concentration 10 mM and 40mM respectively), and samples incubated in the dark for 1 hour at 45˚C. Following reduction / alkylation samples were digested overnight with Lys-C (1/50 w/w Wako chemicals) at 37˚C with shaking at 1000rpm. Digested samples were then mixed with 1.25 volumes of isopropanol to prevent the precipitation of SDC (56) and samples acidified with 10% TFA to a final concentration of 1% TFA. Home-made SDB-RPS StageTips were prepared according to previously described protocols (46,55,57) with six discs of SDB-RPS excised using a blunt 16-gauge Hamilton needle and logged in a 200 μL tip. SDB-RPS StageTips were placed in a Spin96 tip holder (57) to enable batch-based spinning of samples and tips conditioned with 100% acetonitrile; followed by 30% methanol, 1% TFA followed by 90% isopropanol, 1% TFA with each wash spun through the column at 1000 x g for 3 minutes. Acidified isopropanol / peptide mixtures were loaded onto the SDB-RPS columns and spun through the tips washed with 90% isopropanol, 1% TFA followed by 1% TFA in Milli-Q water. Peptides were eluted with 80% acetonitrile, 5% ammonium hydroxide and dried by vacuum centrifugation at room temperature before being stored at -20˚C.

### Data Protocol
Open searching of PilMo and the annotation of HexHAxA modified FLPANCRGT The identification of PilMo glycosylation events was accomplished using open database searching as previously described (49). Briefly, datafiles were processed with MSfragger 3.4 (50,51) in FragPipe (version 17.1) searching against the Moraxella osloensis PilMo sequence (NCBI Accession: WP_156627541.1). Searches were undertaken using “Lys-C” enzyme specificity with carbamidomethylation of cysteine as a fixed modification and Oxidation of methionine as a variable modification with a maximum of 2 missed cleavages allowed. To enable identification of potential glycosylation events a mass tolerance, referred to as a delta mass, of 0 to 2000 Da was allowed. Delta masses observed on the C-terminal peptide 762FLPANCRGT770 were manually inspected to identify potential glycosylation events. Parallel reaction monitoring results corresponding to the HexHAxA-modified glycopeptide 762FLPANCRGT770 was manually extracted using the Freestyle Viewer (1.7 SP1, Thermo Fisher Scientific) the MS/MS data annotated using the Interactive Peptide Spectral Annotator (52) (http://www.interactivepeptidespectralannotator.com/PeptideAnnotator.html). Annotation of spectra allowed for the modification of the terminal T residue with HexHexA (338.0849 Da) as well as Hex (162.0528 Da).

### Publication Abstract
None

### Keywords
M. osloensis, Glycosylation

### Affiliations
University of Melbourne
Omniose/VaxNewMo, St. Louis, MO 63110

### Submitter
Nichollas Scott

### Lab Head
Dr Christian M. Harding
Omniose/VaxNewMo, St. Louis, MO 63110


