### Accession
PXD019898

### Title
Preparation of the RecJ3/4-RNase J complex from Haloferax volcanii

### Description
Purification of RecJ3/4-RNase J complex from Haloferax.volcanii

### Sample Protocol
RecJ3/4-RNase J complex purified from H. volcanii. A) SDS-PAGE of cell lysate (lanes 1-2) and proteins purified by Ni2+ affinity chromatography (HisTrap, lanes 3-4) from H. volcanii H26 ΔrecJ3 nph:pitA strain expressing His6- RecJ3 (HJ02-pJAM4251, lanes 1 and 3) or an empty vector control (HJ02- pJAM202c, lanes 2 and 4, lane4=MCRJ3 in data files). Mr, molecular mass markers. *, non-specific proteins detected in empty vector control. Red bars, gel regions of lane 3 (3-1, 3-2, and 3- 3) and lane 4 (4C) excised and analyzed by LC-MS/MS from gels stained with Bio- Safe Coomassie Blue. B) Summary of proteins identified by LC-MS/MS analysis to be specific to the RecJ3 expression strain. Yellow highlight, proteins detected at a normalized total spectra – control (4C) > 150, protein threshold FDR ≤ 1.0%, 2 peptide minimum, peptide threshold FDR ≤ 0.1%, and percent coverage > 30%. Theor. Mr, theoretical molecular mass based on genome sequence. n.d., not detected above background. Normalized total spectra of sample minus control is indicated. 90KDRJ3=unique band that copurifies with RecJ3 and runs around 90kDa, RecJ3=unique band at the RecJ3 region,lowerRJ3= unique bands lower to the RecJ3 region

### Data Protocol
Protein samples from the Ubl, RecJ3/4, RNase J and Cdc48a pull down assays were separated by 10% SDS-PAGE, visualized by staining with Bio-Safe Coomassie (Bio-Rad) and destained in double deionized water. SDS-PAGE gels were under reducing conditions with exception of the Ubl pull down samples, which were analyzed under non-reducing conditions to examine non-covalent interactions. Unique protein bands were excised and analyzed by LC-MS/MS. Equivalent regions of the gel were similarly analyzed for the empty vector control. Proteins were treated in the gel slices with 45 mM dithiothreitol (DTT) and 100 mM 2-chloroacteamide (CAA). To minimize CAA carryover prior to trypsin digest, liquid was removed from the treated gel pieces and samples were washed with 25 mM ammonium bicarbonate buffer (pH&gt;7.9), dehyrated by treatment with acetonitrile, and treated by centrifugal evaporation (SpeedVac) to complete dryness. Samples were treated with trypsin (1 µg to 50 µg protein) at 37°C for 15 h. Tryptic peptides were injected onto a capillary trap (Thermo Scientific PepMap) and desalted for 5 min with 0.1% vol/vol formic acid at a flow rate of 3 µl∙min -1 prior to loading onto an Thermo Scientific C18 Pep Map nanoflow high performance liquid chromatography (HPLC) column. The elution gradient of the HPLC column started at 3% solvent A (0.1% vol/vol formic acid, 3% vol/vol acetonitrile, and 96.9% v/v H 2 O), 97% solvent B (0.1% vol/vol formic acid, 96.9% vol/vol acetonitrile, and 3% vol/vol H 2 O) and finished at 60% solvent A, 40% solvent B using a flow rate of 300 µl∙min -1 for 30 min. LC-MS/MS analysis of the eluting fractions was carried out on an LTQ Orbitrap XL mass spectrometer (Thermo Scientific, San Jose, CA). The instrument under Xcalibur 2.07 with LTQ Orbitrap Tune Plus 2.55 software was operated in the data dependent mode to automatically switch between MS and MS/MS acquisition. Full MS scans were acquired with a resolution of 60,000 in the Orbitrap from m/z 300–2000. Ten most intense ions were fragmented by collision induced dissociation (CID) at a target value of 5,000 or maximum ion time of 150 ms. Dynamic exclusion was set to 60 seconds. Typical mass spectrometric conditions include a spray voltage of 2.2 kV, no sheath and auxiliary gas flow, a heated capillary temperature of 200 o C, a capillary voltage of 44 V, a tube lens voltage of 165 V, an ion isolation width of 1.0 m/z, a normalized CID collision energy of 35% for MS2 in linear ion trap. The ion selection threshold was 500 counts for MS2. An activation q = 0.25 and activation time of 30 ms were set. Raw data were analyzed using Mascot (Matrix Science, London, UK; version 2.2.2) against H. volcanii and target decoy databases with the latter including a set of reversed sequences generated by Mascot. Mascot was searched with a fragment ion mass tolerance of 0.8 Da and a parent ion tolerance of 15 ppm. Carbamidomethylation of Cys was indicated as a fixed modification while deamidation of Asn and Gln, oxidation of Met, and isopeptide linkage to Gly-Gly- were specified as variable modifications. Scaffold (Proteome Software Inc., Portland, OR) was used to validate MS/MS based peptide and protein identifications, where protein probabilities were assigned by the Protein Prophet algorithm and peptide probabilities were assigned by the Peptide Prophet algorithm (28,29) . Protein identities were based on a threshold of 99.9% probability and &lt; 0.1% False Discovery Rate (FDR) and are summarized in Tables S3-S8. Normalized total spectral counts were calculated using the quantitative package associated with Proteome Software, Inc. (Portland, OR, USA) in Scaffold (30) .

### Publication Abstract
None

### Keywords
Recj4, Rnasej, Recj3

### Affiliations
Professor, Microbiology and Cell Science, University Of Florida, USA.
Memorial Sloan Kettering Cancer CE

### Submitter
Swathi Dantuluri

### Lab Head
Dr Julie Maupin-Furlow
Professor, Microbiology and Cell Science, University Of Florida, USA.


