### Accession
PXD034851

### Title
Viral biomarker detection and validation using MALDI Mass Spectrometry Imaging (MSI)

### Description
INTRODUCTION: Mass Spectrometry Imaging (MSI) is a hybrid mass spectrometry technique that integrates aspects of traditional microscopy and mass spectrometry-based omics analysis. The traditional MALDI TOF/TOF instrument still remains the dominant platform for this type of anal-ysis. However, with reduced mass resolution compared to other platforms it is insufficient to rely on mass resolution alone for peptide identification. Here we propose a hybrid method of data analysis that integrates both image-based analysis and a parallel protein identification workflow using peptide mass fingerprinting, and its successful application to the detection and validation of viral biomarkers. METHODS: FFPE samples were imaged as described previously in an UltrafleXtreme MALDI TOF/TOF. Total mass spectra were exported and searched against the mouse FASTA da-tabase, while companion images were exported and processed with image J. RESULTS: Peptide mass fingerprinting (PMF) revealed 14 target peptides that were successfully assigned to viral proteins while a pixel based correlational analysis revealed a very high R2 correlation (>0.81) be-tween those same peptides assigned to the NS1 and VP1 viral proteins. CONCLUSIONS: We successfully identified and validated the presence of viral biomarkers to a high degree of confidence using MALDI MSI.

### Sample Protocol
Tissue samples were prepared and analysed according to a well-established protocol from Oâ€™Rourke et al (2018. O'Rourke, M.B.; Padula, M.P.; Smith, C.; Youssef, P.; Cordwell, S.; Witting, P.; Sutherland, G.; Crossett, B. Optimal Preparation of Formalin Fixed Samples for Peptide Based Matrix Assisted Laser Desorption/Ionization Mass Spectrometry Imaging Workflows. J Vis Exp 2018, e56778, doi:10.3791/56778.)

### Data Protocol
2.6 Data Analysis MSI After acquisition, the data was processed as described previously[12] using Scills Lab 2014 b, with the added step of importing all experimental data and then normalizing to the total ion count of the global spectrum. Each segmented image was then validat-ed manually by inspecting the associated peak in the global mass spectra to ensure that the entirety of the peak width had been integrated. Images were also inspected to en-sure that the detected peak was genuine and not an artifact or background noise, as described previously[19,20]. The resulting validated images were then exported as grey scale .TIF files and then imported into Image J[21] where a pixel-by-pixel correlation was performed based on intensity. Since each image only contained a single m/z value, there was no interference from other signals.  2.7 Data analysis peptide mass fingerprinting  Once image analysis was complete, the m/z values for each validated image were ex-ported and then compared to the m/z values of a theoretical digest of the target pro-teins: NS1 and VP1. The suspected viral peptide m/z values were then searched with MASCOT[22] against the Mus musculus proteome spiked with amino acid sequences of both VP1 and NS1. A matched decoy list was also included as a negative control. The resulting searches were then performed with the following settings: Enzyme: trypsin, missed cleavages: 2, Variable modifications: Oxidation (M), Peptide tolerance 1.2 Da, monoisotopic masses only, mass value [M+H]+.

### Publication Abstract
(1) Background: MALDI imaging is a technique that still largely depends on time of flight (TOF)-based instrument such as the Bruker UltrafleXtreme. While capable of performing targeted MS/MS, these instruments are unable to perform fragmentation while imaging a tissue section necessitating the reliance of MS1 values for peptide level identifications. With this premise in mind, we have developed a hybrid bioinformatic/image-based method for the identification and validation of viral biomarkers. (2) Methods: Formalin-Fixed Paraffin-Embedded (FFPE) mouse samples were sectioned, mounted and prepared for mass spectrometry imaging using our well-established methods. Peptide identification was achieved by first extracting confident images corresponding to theoretical viral peptides. Next, those masses were used to perform a Peptide Mmass Fingerprint (PMF) searched against known viral FASTA sequences against a background mouse FASTA database. Finally, a correlational analysis was performed with imaging data to confirm pixel-by-pixel colocalization and intensity of viral peptides. (3) Results: 14 viral peptides were successfully identified with significant PMF Scores and a correlational result of &gt;0.79 confirming the presence of the virus and distinguishing it from the background mouse proteins. (4) Conclusions: this novel approach leverages the power of mass spectrometry imaging and provides confident identifications for viral proteins without requiring MS/MS using simple MALDI Time Of Flight/Time Of Flight (TOF/TOF) instrumentation.

### Keywords
Peptide identification, Mass spectrometry imaging, Maldi

### Affiliations
Director, Centre for inflammation, Centenary Institute
University of Technology Sydney

### Submitter
Matthew O'Rourke

### Lab Head
Dr Phil P Hansbro
Director, Centre for inflammation, Centenary Institute


