### Accession
PXD036004

### Title
Multiomic analysis of Cohesin reveals that ZBTB transcription factors contribute to chromatin interactions

### Description
One bottleneck in understanding the principles of 3D chromatin structures is caused by the paucity of known regulators. Cohesin is essential for 3D chromatin organization, and its interacting partners are candidate regulators. Here, we performed proteomic profiling of the Cohesin in chromatin and identified transcription factors, RNA-binding proteins, and chromatin regulators associated with Cohesin. Acute protein degradation followed by time-series genomic binding quantitation and BAT Hi-C analysis were conducted, and the results showed that the transcription factor ZBTB21 contributes to Cohesin chromatin binding, 3D chromatin interactions and transcriptional repression. Strikingly, multiomic analyses revealed that the other four ZBTB factors interacted with Cohesin, and double degradation of ZBTB21 and ZBTB7B led to a further decrease in Cohesin chromatin occupancy. We propose that multiple ZBTB transcription factors orchestrate the chromatin binding of Cohesin to regulate chromatin interactions, and we provide a catalog of many additional proteins associated with Cohesin that warrant further investigation.

### Sample Protocol
The ChIP experiment was performed based on a previous publication. Briefly, 10 million cells were fixed with formaldehyde. Then the fixed cells were precleared with NP-40 lysis buffer, nuclear lysis buffer containing glycerol, and urea to obtain the chromatin fractions. The chromatin fractions were fragmented with MNase and sonication, and the soluble chromatin fractions were used for chromatin immunoprecipitation. Different antibodies were added to the chromatin extracts and incubated at 4Â°C overnight. Incubate antibody-chromatin preparations with protein G Dynabeads. After extensive wash, the ChIP products were eluted.Eluted protein preparations were analyzed by mass spectrometry.

### Data Protocol
Mass spectrometry raw data were mapped to mouse or human proteins in Uniprot database by MS analyzing platform in the National Center for Protein Sciences at Peking University, Beijing. Then, the peptide counts of all isoforms of a protein were combined to generate a single PSM value for each protein. The total peptide counts of candidate proteins were at least two times more than the control Anti-GFP,Anti-SMC1 or input samples were selected. In addition, ribosomal, mitochondrial, secreted, plasma membrane proteins were removed from the protein candidate lists. Next, the selected proteins were searched and analyzed by the Pubmed, Uniprot, and g:Profiler databases. In addition, the Gene Ontology of candidate proteins was carried out by using R package Cluster Profiler.The experiments were runned by Orbitrap Fusion Lumos.

### Publication Abstract
One bottleneck in understanding the principles of 3D chromatin structures is caused by the paucity of known regulators. Cohesin is essential for 3D chromatin organization, and its interacting partners are candidate regulators. Here, we performed proteomic profiling of the cohesin in chromatin and identified transcription factors, RNA-binding proteins and chromatin regulators associated with cohesin. Acute protein degradation followed by time-series genomic binding quantitation and BAT Hi-C analysis were conducted, and the results showed that the transcription factor ZBTB21 contributes to cohesin chromatin binding, 3D chromatin interactions and transcriptional repression. Strikingly, multiomic analyses revealed that the other four ZBTB factors interacted with cohesin, and double degradation of ZBTB21 and ZBTB7B led to a further decrease in cohesin chromatin occupancy. We propose that multiple ZBTB transcription factors orchestrate the chromatin binding of cohesin to regulate chromatin interactions, and we provide a catalog of many additional proteins associated with cohesin that warrant further investigation.

### Keywords
Human, Chip-ms, Mouse, Ch12f3, Hek293ts, B16, Mescs, Label-free

### Affiliations
School of life science, Peking University, Beijing
PEKING University

### Submitter
RUI WANG

### Lab Head
Dr Xiong Ji
School of life science, Peking University, Beijing


