### Accession
PXD007682

### Title
Francisella tularensis DsbA mutant MS/SILAC analysis

### Description
Comparison of enriched membrane fractions of Francisella tularensis subsp. holarctica strain FSC200 and its DsbA mutant by SILAC analysis.

### Sample Protocol
Bacteria were cultivated in Chamberlains chemically defined medium, wild type was isotopicaly labeled with R10 and K8. Bacteria were disrupted by french press, light and heavy samples were mixed 1:1 (protein ratio) and membrane fractions were enriched by high speed centrifugation. Samples were dissolved by sodium deoxycholate, reduced, alkylated, digested by trypsin and desalted. LC-MS analysis was performed on Ultimate 3000 RSLCnano System coupled on-line through Nanospray Flex ion source with Q-Exactive mass spectrometer.

### Data Protocol
Database search and relative protein quantification was performed using Proteome Discoverer 1.3 (v. 1.3.0.339) and Mascot search algorithm. The reference proteome set of F. tularensis FSC200 was downloaded from UniProt/KB. Search parameters: digestion with trypsin, max. 2 missed cleavages, peptide mass tolerance of 10 ppm, fragment mass tolerance of 0.02 Da, fixed carbamidomethylation of cysteine, variable oxidation of methionine and SILAC labels Arg10 and Lys8. The strict target value of FDR for a decoy database search of 0.01 was applied (high confidence). Only unique peptides were considered for quantification and the light to heavy ratios were normalized on protein median for each replicate. For relative protein quantification only proteins with minimum of 1 quantified peptide per at least six measurements out of nine were considered. Light to heavy ratios were converted to log2 and median was determined from the three measurements for each biological replicate. Average, standard deviation and the T-statistic p-value were then evaluated from the three biological replicates. For further study proteins showing the absolute change larger than 1.5 and p-value â‰¤ 0.05 were considered.

### Publication Abstract
The DsbA homolog of <i>Francisella tularensis</i> was previously demonstrated to be required for intracellular replication and animal death. Disruption of the <i>dsbA</i> gene leads to a pleiotropic phenotype that could indirectly affect a number of different cellular pathways. To reveal the broad effects of DsbA, we compared fractions enriched in membrane proteins of the wild-type FSC200 strain with the <i>dsbA</i> deletion strain using a SILAC-based quantitative proteomic analysis. This analysis enabled identification of 63 proteins with significantly altered amounts in the <i>dsbA</i> mutant strain compared to the wild-type strain. These proteins comprise a quite heterogeneous group including hypothetical proteins, proteins associated with membrane structures, and potential secreted proteins. Many of them are known to be associated with <i>F. tularensis</i> virulence. Several proteins were selected for further studies focused on their potential role in tularemia's pathogenesis. Of them, only the gene encoding glyceraldehyde-3-phosphate dehydrogenase, an enzyme of glycolytic pathway, was found to be important for full virulence manifestations both <i>in vivo</i> and <i>in vitro</i>. We next created a viable mutant strain with deleted <i>gapA</i> gene and analyzed its phenotype. The <i>gapA</i> mutant is characterized by reduced virulence in mice, defective replication inside macrophages, and its ability to induce a protective immune response against systemic challenge with parental wild-type strain. We also demonstrate the multiple localization sites of this protein: In addition to within the cytosol, it was found on the cell surface, outside the cells, and in the culture medium. Recombinant GapA was successfully obtained, and it was shown that it binds host extracellular serum proteins like plasminogen, fibrinogen, and fibronectin.

### Keywords
Q-exactive, Silac, Francisella tularensis

### Affiliations
Faculty of Military Health Sciences, University of Defense, Hradec Kralove, Czech Republic
Dept. of Molecular Pathology and Biology, Faculty of Military Health Sciences, University of Defense, Czech Republic

### Submitter
Jana Klimentova

### Lab Head
Dr Jiri Stulik
Dept. of Molecular Pathology and Biology, Faculty of Military Health Sciences, University of Defense, Czech Republic


