### Accession
PXD036708

### Title
Structure of nucleosome-bound human PBAF complex

### Description
BAF and PBAF are mammalian SWI/SNF family chromatin remodeling complexes that possess multiple histone/DNA-binding subunits and create nucleosome-depleted/free regions for transcription activation. Despite previous structural studies and recent advance of SWI/SNF family complexes, it remains incompletely understood how PBAF-nucleosome complex is organized. Here we determined structure of 13-subunit human PBAF in complex with acetylated nucleosome in ADP-BeF3-bound state. Four PBAF-specific subunits work together with nine BAF/PBAF-shared subunits to generate PBAF-specific modular organization, distinct from that of BAF at various regions. PBAF-nucleosome structure reveals six histone-binding domains and four DNA-binding domains/modules, the majority of which directly bind histone/DNA. This multivalent nucleosome-binding pattern, not observed in previous studies, suggests that PBAF may integrate comprehensive chromatin information to target genomic loci for function. Our study reveals molecular organization of subunits and histone/DNA-binding domains/modules in PBAF-nucleosome complex and provides structural insights into PBAF-mediated nucleosome association complimentary to the recently reported PBAF-nucleosome structure.

### Sample Protocol
The Cross-linking Mass Spectrometry (XL-MS) analysis was performed as previously described. The purified PBAF complex (0.45 μM) was incubated with nucleosome at a ratio of 1:1 in the presence of ADP-BeF3 followed by cross-linking MS analyses. The PBAF-NCP complex was incubated with DSS (250 μM) at 25℃ with shaking at 500 rpm (ThermoMixer) for 1 h. Reaction was terminated by adding 20 mM ammonium bicarbonate (Sigma). The cross-linked sample was precipitated with cooled acetone and dried in a speed vac. The pellet was dissolved in 8 M Urea, 100 mM Tris-HCl pH 8.5, followed by TCEP reduction, iodoacetamide (Sigma) alkylation, and trypsin (Promega) digestion overnight at 37℃ using a protein/enzyme ratio of 50:1 (w/w). Tryptic peptides were desalted with Pierce C18 spin column (GL Sciences) and separated in a proxeon EASY-nLC liquid chromatography system by applying a step-wise gradient of 0-85% acetonitrile (ACN) in 0.1% foricacid. Peptides eluted from the liquid chromatography were directly electrosprayed into the mass spectrometer with a distal 2 kV spray voltage. Using mass spectrometry for analysis of the degree of nucleosome acetylation, we prepared 2 μg acetylated nucleosomes as described above. The samples were loaded onto the SDS PAGE gel (6%-16%) and then the protein slices were excised from the CCB (Coomassie brilliant blue)-stained gel. Excised slices were firstly destained twice with 200 μl of 50 mM NH4HCO3 and 50% acetonitrile and then dried twice with 200μl of acetonitrile. The dried pieces of gels were incubated in ice-cold digestion solution (trypsin 12.5 ng/μl and 20mM NH4HCO3) for 20 min and then transferred into a 37°C incubator for digestion overnight. Finally peptides in the supernatant were collected after extraction twice with 200μl extract solution (5% formic acid in 50% acetonitrile). The peptide solution described above was dried under the protection of N2. The tryptic peptides were dissolved in 0.1% formic acid (solvent A), directly loaded onto a reversed-phase analytical column (Acclaim PepMap C18, 75μm x 25cm). The gradient was comprised of an increase from 2% to 30% solvent B (0.1% formic acid in 98% acetonitrile) over 50min, 30% to 50% in 5 min and climbing to 80% in 1min then holding at 80% for the last 4 min, all at a constant flow rate of 300 nl/min on an EASY-nLC 1200 UPLC system. The peptides were subjected to NSI source followed by tandem mass spectrometry (MS/MS) in Orbitrap Exploris 480 MS coupled online to the UPLC. Spray voltage were set to 2.3 kV, funnel RF level at 50, and heated capillary temperature at 320 °C. For DDA experiments full MS resolutions were set to 60,000 at m/z 200 and full MS AGC target was 300% with an IT of 25 ms. Mass range was set to 350–1600. AGC target value for fragment spectra was set at 200% with a resolution of 15,000 and injection times of 50 ms and Top12. Intensity threshold was kept at 2E5. Isolation width was set at 1.6 m/z. Normalized collision energy was set at 30%. The tandem mass spectrometry analyses of the acetylated nucleosome samples were technically repeated by 3 times.

### Data Protocol
Data-dependent tandem mass spectrometry (MS/MS) analyses were performed on Thermo Q-Exactive instrument in a 60-minute gradient. The acquired raw data files were processed with pLink2 software 60 and the results were visualized using the xiNET online server. The MS raw files of acetylated nucleosome were processed using PEAKS Online X ( build 1.6.2022-01-07_161036, Bioinformatics Solutions Inc.). The data was searched against the swissprot human database consisting of reviewed canonical sequences (total entry 20332). Parent mass tolerance was set to 10ppm, with fragment mass tolerance of 0.02 Da. Specific tryptic cleavage was selected with allows for a maximum of 2 missed cleavages. For identification of all proteins, fixed modifications of carbamidomethylation (57.02 Da) on cysteine residues were specified. Variable modifications of oxidation (15.99 Da) on methionine and methylation on lysine were specified. The minimum peptide is 6 in length. The minimum number of unique peptides for protein identification is 2. Lable free quantification was also performed using the PEAKS Online X quantification module, allowing a mass tolerance of 20 ppm and quantifying all peptides that pass a 1% FDR threshold.

### Publication Abstract
BAF and PBAF are mammalian SWI/SNF family chromatin remodeling complexes that possess multiple histone/DNA-binding subunits and create nucleosome-depleted/free regions for transcription activation. Despite previous structural studies and recent advance of SWI/SNF family complexes, it remains incompletely understood how PBAF-nucleosome complex is organized. Here we determined structure of 13-subunit human PBAF in complex with acetylated nucleosome in ADP-BeF<sub>3</sub>-bound state. Four PBAF-specific subunits work together with nine BAF/PBAF-shared subunits to generate PBAF-specific modular organization, distinct from that of BAF at various regions. PBAF-nucleosome structure reveals six histone-binding domains and four DNA-binding domains/modules, the majority of which directly bind histone/DNA. This multivalent nucleosome-binding pattern, not observed in previous studies, suggests that PBAF may integrate comprehensive chromatin information to target genomic loci for function. Our study reveals molecular organization of subunits and histone/DNA-binding domains/modules in PBAF-nucleosome complex and provides structural insights into PBAF-mediated nucleosome association complimentary to the recently reported PBAF-nucleosome structure.

### Keywords
Lc-ms, Pbaf, Mass spectrometry analysis of acetylated nucleosome, Ncp

### Affiliations
Fudan University
Fudan University, Shanghai, China

### Submitter
Li Wang

### Lab Head
Dr Yanhui Xu
Fudan University, Shanghai, China


