### Accession
PXD012940

### Title
Comparative proteomics of clinically relevant proteins in modern and traditional bread wheat cultivars

### Description
Bread wheat (Triticum aestivum L.) is one of the most valuable cereal crops for the human consumption. Its grain storage proteins greatly impact bread quality, though may cause food intolerances or allergies in susceptible individuals. Consequently, the investigation of a proteome polymorphism among wheat varieties is important to spot the genotypes, which would be promising donors for the breeding of hypoallergenic cereals. Herein, we discovered diversity of grain proteins in three Ukrainian wheat cultivars: ‘Sotnytsia’, ‘Panna’ (both modern selection) and ‘Ukrainka’ (old landrace). Firstly, proteins were isolated with a SDS-containing buffer that allowed extraction of various groups of storage proteins (glutenins, gliadins, globulins and albumins). Secondly, the proteome was profiled by the two-dimensional gel electrophoresis, revealing 810 clearly-separated gel spots. Software-assisted analysis of gel images, showed 66 differentially abundant proteins. Using multi-enzymatic digestion, followed by the tandem mass spectrometry, we identified 49 differentially accumulated proteins. Parallel ultrahigh-performance liquid chromatography profiling and direct mass spectrometry quantification complemented the results. With this approach we quantified 127 proteins, 12 being differentially abundant. Principal component analysis confirmed genotype as a major source of variation in both cases. Non-gluten fraction was the most diverse among investigated bread wheat cultivars. Information from public databases of clinically relevant plant proteins highlighted variable groups of wheat allergens/toxins. Data suggested that one of the modern cultivars contained less health affecting proteins in grain. Finally, we proposed set of genetic landmarks for the development of DNA marker system, which will enable fast and efficient assessment of medical safety of multiple wheat genotypes to facilitate breeding programs.

### Sample Protocol
Grain proteins were extracted according to the developed protocol. Following grinding of 1 g of seeds to a fine flour with liquid nitrogen in a mortar, 10 mL of extraction buffer (2% SDS, 10% glycerol, 50 mM dithiothreitol and 50 mM Tris pH 6.8) was added. The mixture was incubated at room temperature with vigorous shaking for 1 h. To remove insoluble components, samples were centrifuged at 10,000 g for 15 min. Proteins were precipitated with 4 volumes of cold acetone and kept overnight at –20°C. Precipitate was collected by centrifugation at 4,000 g, 4°C for 15 min. The same volume of cold acetone was poured onto the pellet and discarded after centrifugation at 4,000 g, 4°C for 15 min. Protein extracts from different cultivars were resuspended in the solubilization buffer (8 M urea, 2 M thiourea, 2% CHAPS and 2% Triton X-100). Then, concentration was measured by Pierce detergent compatible Bradford assay (ThermoFisher Scientific). Protein aliquots 500 μg were adjusted to 340 μL with isoelectric focusing buffer (8 M urea, 2 M thiourea, 2% CHAPS, 1% amidosulfobetaine-14, 1% Triton X-100, 2% ampholytes and 1% DeStreak). Upon centrifugation, supernatant was transferred to 18 cm immobilized pH gradient strips 3-10 non-linear (GE Healthcare). First dimension separation according to the isoelectric points was done in Ettan IPGphor 3 unit (GE Healthcare). After isoelectric focusing, strips were placed on top of 12% polyacrylamide gels and sealed by 0.5% agarose with 0.002% bromophenol blue in running buffer. Second dimension separation according to molecular weights was performed in Protean II xi Cell (Bio-Rad). Gels were stained by sensitive colloidal Coomassie G-250. Images were digitalized with resolution 300 dpi and 16-bit grayscale pixel depth on Umax ImageScanner (GE Healthcare).  In-gel digestion and filter-assisted sample preparation. For the in-gel digestion we followed the standard protocol. Protein spots were digested with 20 µL of 10 ng/µL enzyme (Promega) in 10 mM ammonium bicarbonate and 10% acetonitrile at 25°C (chymotrypsin) or 37°C (trypsin), in 50 mM Tris pH 8.0 and 0.5 mM CaCl2 at 60°C (thermolysin). Total protein extracts were digested according to FASP protocol. Following activation of centrifugal filter units Microcon Ultracel YM-10 (Merck Millipore) by 200 µL of 1% formic acid and centrifugation at 10,000 g for 40 min, 100 µg protein aliquots, adjusted to 200 μL with urea buffer (8 M urea, 100 mM Tris pH 8.5), were loaded on the filter and centrifuged. Subsequently, proteins were washed with 200 µL of urea buffer. Then, collection tubes were exchanged and 75 µL of chymotrypsin 100 ng/μL in 50 mM ammonium bicarbonate was added to the filter. Digestion was carried out at 25°C overnight. Afterwards, peptides were collected by centrifugation at 10,000 g for 20 min. For peptide purification after FASP protocol we used Sep-Pak Light C18 cartridges (Waters). Concentration of the peptides was measured by NanoDrop 2000 spectrophotometer (ThermoFisher Scientific) assuming that 0.1% solution of proteins has at 280 nm an extinction of 1.0 absorbance units. Tandem mass spectrometry for protein identification. The peptides were analyzed by liquid chromatography-tandem mass spectrometry (LC-MS/MS), using nanoAcquity UHPLC (Waters) and Q-TOF Premier (Waters). The data were recorded in the MSE mode (parallel high and low energy traces without precursor ion selection). Direct mass spectrometry relative quantification. Aliquots of purified complex peptide mixtures (300 ng) were separated in biological triplicate, using Acquity M-Class UHPLC (Waters). Samples were loaded onto the nanoEase Symmetry C18 trap column (20 mm length, 180 μm diameter, 5 μm particles size). Afterwards, peptides were introduced to the nanoEase HSS T3 C18 analytical column (100 mm length, 75 μm diameter, 1.8 μm particle size). For the thorough separation, a 90 min gradient of 5-35% acetonitrile with 0.1% formic acid was applied at a flow rate of 300 nL/min. The column outlet was connected to the PicoTip emitter and samples were nanosprayed (3.1 kV capillary voltage) to the quadrupole time-of-flight mass spectrometer Synapt G2-Si with ion mobility option (Waters). Spectra were recorded in a data-independent manner in high definition MSE mode. Ions with 50-2000 m/z were detected in both channels, with a 1 s spectral acquisition scan rate.

### Data Protocol
Quantitative software-assisted gel analysis was done in SameSpots 5.1 (TotalLab) to reveal protein spots differentially accumulated between varieties. Firstly, software aligned images, then dust/stain particles, obvious streaks and damaged gel areas were filtered. Relative volumes were normalized to median distribution of normalization reference gel to compensate for minor differences in sample loading. All spots were reviewed and manually edited if necessary. Differentially accumulated proteins were chosen based on ANOVA p-value ≤ 0.01 and ratio ≥ 2.5. Additionally, we applied post-hoc Tukey's honestly significant difference test to assess changes between specific varieties. For protein identification from gel spots the data were processed using ProteinLynx Global Server 3.0 (Waters). Spectra were searched against wheat proteome sequences downloaded from UniProt in April 2018 (136,892 entries, uniprot.org). Search parameters were as specified in the following chapter for chymotrypsin, but one allowed miscleavage for trypsin and thermolysin. Thermolysin was defined as cutting on N-terminus after alanine, phenylalanine, isoleucine, leucine, methionine and valine, but not before proline. Identities, were accepted if two or more different peptides with score higher than 95% reliability threshold were matched. Reliability scores were adjusted based on distribution of target/decoy queries. For direct mass spectrometry relative quantification spectra were preprocessed with Compression and Archival Tool 1.0 (Waters) to reduce noise, removing ion counts below 15. Data processing was done in Progenesis QI 4.0 (Waters). For peak picking the following threshold parameters were applied: low energy 320 counts and high energy 40 counts. Precursors and fragment ions were coupled, using correlations of chromatographic elution profiles in low/high energy traces. Then, peaks retention times were aligned across all chromatograms. Peak intensities were normalized to median distribution of all ions, assuming the majority of signals are unaffected by experimental conditions. The label-free quantification relied on measurement of peak areas of three most intense precursor peptides, preferentially unique. For the protein identification, Ion Accounting 4.0 (Waters) search algorithm was applied. The reference sequence file was as mentioned above. Workflow parameters for the protein identification searches were: maximum two possible chymotrypsin miscleavages, a fixed carbamidomethyl cysteine, variable oxidized methionine and deamidated glutamine. Chymotrypsin was defined as cutting on C-terminus after tyrosine, phenylalanine, tryptophan, leucine and methionine, but not before proline. The precursor and peptide fragment mass tolerance was automatically determined by the software. Peptide matching was limited to less than 4% false discovery rate against the randomized database. Identifications were accepted if at least two distinct reliable peptides (score ≥ 5.5, mass accuracy ≤ 15 ppm) matched the protein sequence. Protein grouping feature was then applied to show only hits with unique peptides. We considered as differentially abundant proteins which satisfied the same statistic and effect size criteria as in two-dimensional gel electrophoresis section. To assess clinical relevance of identified proteins, found to be differentially accumulated in grain from studied varieties, we used Allergome (allergome.org), ProPepper (propepper.net), and GluPro.

### Publication Abstract
Bread wheat (<i>Triticum aestivum</i> L.) is one of the most valuable cereal crops for human consumption. Its grain storage proteins define bread quality, though they may cause food intolerances or allergies in susceptible individuals. Herein, we discovered a diversity of grain proteins in three Ukrainian wheat cultivars: Sotnytsia, Panna (both modern selection), and Ukrainka (landrace). Firstly, proteins were isolated with a detergent-containing buffer that allowed extraction of various groups of storage proteins (glutenins, gliadins, globulins, and albumins); secondly, the proteome was profiled by the two-dimensional gel electrophoresis. Using multi-enzymatic digestion, we identified 49 differentially accumulated proteins. Parallel ultrahigh-performance liquid chromatography separation followed by direct mass spectrometry quantification complemented the results. Principal component analysis confirmed that differences among genotypes were a major source of variation. Non-gluten fraction better discriminated bread wheat cultivars. Various accumulation of clinically relevant plant proteins highlighted one of the modern genotypes as a promising donor for the breeding of hypoallergenic cereals.

### Keywords
Grain, Two-dimensional gel electrophoresis, Cereal allergens, Food quality, Mass spectrometry, Triticum aestivum

### Affiliations
Institute of Neuroimmunology, Slovak Academy of Sciences
Plant Science and Biodiversity Center SAS

### Submitter
Maksym Danchenko

### Lab Head
Dr Andrej Kovac
Institute of Neuroimmunology, Slovak Academy of Sciences


