### Accession
PXD031376

### Title
Differential metabolic adaptations of brain cells to feeding a ketogenic diet in mice

### Description
It is largely unknown how cells in the central nervous system (CNS) adapt their energy metabolism to nutritional challenges or disease. To unravel the molecular basis of these adaptations, we generated a comprehensive proteome dataset of all major CNS cell types. Magnetic bead sorting was used to acutely isolate neurons, astrocytes, oligodendrocytes, microglia, and endothelial cells from the cortex of individual young and adult mice challenged by feeding ketogenic diet or by neuroinflammatory disease. Label-free quantification of cell type-specific proteomes was performed by using an ion mobility-enhanced data-independent acquisition (DIA) workflow with alternating low and elevated energy (referred to as UDMSE). The combination of tissue metabolomic and expression analyses with cellular proteomic and transcriptomic analyses revealed that the strategies to manage the altered availability of energy metabolites differed between CNS cell types. Our data provide insights into key mechanisms of cell type-specific metabolic remodeling and a framework for the crosstalk between CNS cells in response to CNS challenges, which may have implications for the management of neurological diseases.

### Sample Protocol
Preparation of CNS cell types from mouse cortices by magnetic cell isolation technology (MACS). Solubilization of cell fractions in a CHAPS/urea lysis buffer, followed by automated in-solution protein digestion according to the filter-aided sample preparation (FASP) protocol and analysis by LC-MS. 1D LC separation of peptides using a Waters nanoAcquity nanoUPLC system. Mass spectrometric analysis performed by data-independent acquisition (DIA) on a Waters Synapt G2-S quadrupole time-of-flight (QTOF) mass spectrometer equipped with ion mobility option.

### Data Protocol
Peptide and protein identification with Waters ProteinLynx Global SERVER (PLGS). Retention time alignment, exact mass and retention time (EMRT) and ion mobility clustering, data normalization, isoform/homology filtering and calculation of absolute in-sample amounts for each detected protein according to the TOP3 quantification approach with ISOQuant. Statistical analysis and visualization of data with an RStudio pipeline employing the Bioconductor packages "limma" and "q-value".

### Publication Abstract
None

### Keywords
Inflammatory demyelinating disease, Metabolomics, Brain metabolism, Transcriptomics, Mass spectrometry, Ketogenic diet, Label-free proteomics

### Affiliations
Neuroproteomics Group, Max Planck Institute for Multidisciplinary Sciences, GÃ¶ttingen
Neuroproteomics Group Max Planck Institute for Multidisciplinary Sciences, City Campus Hermann-Rein-Str. 3 37075 Goettingen, Germany Tel: +49 551 20131-313 Fax: +49 551 20131-323 E-mail: jahn@mpinat.mpg.de

### Submitter
Olaf Jahn

### Lab Head
Dr Olaf Jahn
Neuroproteomics Group Max Planck Institute for Multidisciplinary Sciences, City Campus Hermann-Rein-Str. 3 37075 Goettingen, Germany Tel: +49 551 20131-313 Fax: +49 551 20131-323 E-mail: jahn@mpinat.mpg.de


