### Accession
PXD000255

### Title
Label-free analysis of EAE vs Sham Spinal cord and PBMCs

### Description
Here we present the data obtained from a label-free quantitative proteomics analysis of soluble spinal cord extract derived from a mouse model of multiple sclerosis (EAE) and sham-induced mice.  Samples were prepared offline using the FASP approach and then submitted for nano-LC-MS/MS analysis on an Orbitrap Velos instrument. After statistical evaluation of the data, 431 differentially expressed proteins (KS-test, p < 0.05) out of a total of ~1400 unique proteins were identified in the comparative spinal cord analysis (peptide FDR=0.55%).Database search and protein identification: Tandem mass spectra were extracted from .RAW files and searched using the SEQUEST-PVM v.27 (rev.9) (Eng et al., 1994) database program  against a Mouse protein database downloaded as FASTA-formatted sequences from EBI-IPI (database version 3.72) which contains 56957 entries (with priority given to UniProt identifiers) as well as reverse decoy sequences to empirically assess the false identification rate. This search program was executed on a cluster computer to match the MS/MS spectra to the corresponding most highly correlated peptide sequences  Mass tolerances for precursor (MS) and product ions (MS/MS) were set to 3 and 0 m/z, respectively. Searches were performed with the enzyme selectivity set to trypsin with one missed cleavage allowed and protein modifications included fixed carbamidomethylation of cysteines (57 Da). Match likelihoods were assigned a statistical confidence score using the STATQUEST probabilistic model (Kislinger et al., 2003) and candidate peptide identifications were filtered using an estimated peptide confidence score of â‰¥95%. A 10 ppm high accuracy mass filter accounting for isotopic shifts in the spectra was applied post-SEQUEST analysis thus improving the fidelity of protein identifications. Protein quantitation: To estimate relative protein levels, spectral counts were transformed into normalised spectral abundance factors (NSAF) as previously described (Mosley et al., 2009). Briefly, this involves dividing the spectral count (SC) of a protein by its length (Mw) and finally normalises this value to the sum of all SC/Mw.

### Sample Protocol
Not available

### Data Protocol
Not available

### Publication Abstract
Multiple sclerosis is a chronic demyelinating disorder characterized by the infiltration of auto-reactive immune cells from the periphery into the central nervous system resulting in axonal injury and neuronal cell death. Experimental autoimmune encephalomyelitis represents the best characterized animal model as common clinical, histological, and immunological features are recapitulated. A label-free mass spectrometric proteomics approach was used to detect differences in protein abundance within specific fractions of disease-affected tissues including the soluble lysate derived from the spinal cord and membrane protein-enriched peripheral blood mononuclear cells. Tissues were harvested from actively induced experimental autoimmune encephalomyelitis mice and sham-induced ("vehicle" control) counterparts at the disease peak followed by subsequent analysis by nanoflow liquid chromatography tandem mass spectrometry. Relative protein quantitation was performed using both intensity- and fragmentation-based approaches. After statistical evaluation of the data, over 500 and 250 differentially abundant proteins were identified in the spinal cord and peripheral blood mononuclear cell data sets, respectively. More than half of these observations have not previously been linked to the disease. The biological significance of all candidate disease markers has been elucidated through rigorous literature searches, pathway analysis, and validation studies. Results from comprehensive targeted mass spectrometry analyses have confirmed the differential abundance of &#x223c; 200 candidate markers (&#x2265; twofold dysregulated expression) at a 70% success rate. This study is, to our knowledge, the first to examine the cell-surface proteome of peripheral blood mononuclear cells in experimental autoimmune encephalomyelitis. These data provide a unique mechanistic insight into the dynamics of peripheral immune cell infiltration into CNS-privileged sites at a molecular level and has identified several candidate markers, which represent promising targets for future multiple sclerosis therapies. The mass spectrometry proteomics data associated with this manuscript have been deposited to the ProteomeXchange Consortium with the data set identifier PXD000255.

### Keywords
Spinal cord, Mouse, Label-free lc ms/ms, Eae, Orbitrap

### Affiliations
Department of Biochemistry and Molecular Biology

### Submitter
L Dagley

### Lab Head


