### Accession
PXD026631

### Title
Repetition of comparison of sample preservation methods for field collection of animal-microbe symbioses for metaproteomic analyses

### Description
When studying gene expression in microbe-animals symbioses collected in the field it is essential to quickly and efficiently preserve in situ symbiont and host protein abundance patterns. One of the most commonly used sample preservation methods for samples targeted for proteomic analyses is flash freezing, however, liquid nitrogen or dry ice needed for flash freezing are often not available at remote field sites. We replicated our experiment from PXD014591 to test if RNAlater allows preserving proteins in animal-microbe symbioses as efficiently as flash freezing and without introducing issues with downstream processing. We used the marine gutless oligochaete Olavius algarvensis as a model for testing. Olavius algarvensis lives in shallow water sediments off the coast of Elba, Italy. It has no digestive and excretory system and harbors five bacterial symbionts that fulfill its nutritional and waste recycling needs (Kleiner et al., 2012, PNAS 109(19):1173-82). We compared six RNAlater preserved and eight flash frozen samples in terms of the number of identified proteins, abundances of individual proteins and potential biases against specific protein or taxonomic groups. Six worms were incubated in RNAlater for 24 hours. After incubation, RNAlater was removed and samples were stored at -80°C. Eight worms were directly flash frozen in liquid nitrogen and stored at -80 °C immediately after preservation.

### Sample Protocol
We conducted a tryptic digest following the filter-aided sample preparation protocol, adapted from Wisniewski et al. (2009, Nat. Meth. 6(5)), for all samples. Briefly, 60 µl of SDT-lysis buffer (4% (w/v) SDS, 100 mM Tris-HCl pH 7.6, 0.1 M DTT) were added to the samples and samples were heated to 95 °C for 10 min. To minimize sample loss, we skipped the centrifugation step described in the original FASP protocol. 60 µl of each lysate were mixed with 400 µL of UA solution (8 M urea in 0.1 M Tris/HCl pH 8.5) in a 10 kDa MWCO 500 µL centrifugal filter unit (VWR International, Cat. No. 82031-350) and we centrifuged the mixture at 14,000 x g for 40 min. Subsequently, we added 200 µl of UA solution and centrifuged  again at 14,000 x g for 40 min. 100 µl of IAA solution (0.05 M iodoacetamide in UA solution) were added and samples were incubated at 22 °C for 20 min in the dark. We removed the IAA solution by centrifugation followed by three washing steps with 100 µL of UA solution. Next, we washed the filter three times with 100 µL of ABC buffer (50 mM ammonium bicarbonate). 0.6 μg of Pierce MS grade trypsin (Thermo Scientific, Cat. No. 90059) in 40 µL of ABC buffer were added to the filter and filters were incubated overnight in a wet chamber at 37 °C. The next day, we eluted the peptides by centrifugation at 14,000 x g for 20 min followed by addition of 50 µL of 0.5 M NaCl and another centrifugation step. Peptides were quantified using the Pierce MicroBCA Kit (Thermo Scientific, Cat. No. 23235) following the instructions of the manufacturer.  Samples were analyzed by one-dimensional liquid chromatographytandem mass spectrometry (1D-LC-MS/MS) using an alternating block-randomization (Oberg und Vitek, 2009 J. Proteome Res. 8 (5) 21442156). For each run, 600 ng of peptide were loaded onto a 2 cm, 75 μm ID C18 Acclaim PepMap 100 pre-column (Thermo Fisher Scientific) using an UltiMate 3000 RSLCnano Liquid Chromatograph (Thermo Fisher Scientific). The pre-column was connected to a 50 cm × 75 μm analytical EASY-Spray column packed with PepMap RSLC C18, 2 μm material (Thermo Fisher Scientific), which was heated to 60 °C via the integrated column heater. The analytical column was connected via an Easy Spray source to a Q Exactive™ HF Hybrid Quadrupole-Orbitrap™ Mass Spectrometer (Thermo Fisher Scientific). Peptides were separated on the analytical column at a flow rate of 300 nl min−1 using a 140 min gradient going from buffer A (0.2% formic acid, 5% acetonitrile) to 31% buffer B (0.2% formic acid in acetonitrile) in 102 min, then from 31% to 50% buffer B in 18 min and ending with 20 min at 99% buffer B. Eluting peptides were ionized with electrospray ionization and analyzed in a Q Exactive™ HF Hybrid Quadrupole-Orbitrap™ Mass Spectrometer (Thermo Fisher). Full scans were acquired in the Orbitrap mass spectrometer at 60,000 resolution. MS/MS scans of the 15 most abundant precursor ions were acquired in the Orbitrap mass spectrometer at 15,000 resolution. The mass (m/z) 445.12003 was used as lock mass as described in Olsen et al., (2005 Molecular & Cellular Proteomics 4 (12) 2010-2021) with the modification that lock mass was detected in the full scan rather than by separate selected ion monitoring scan injection. Lock mass use was set to ‘best’. Single charged ions were excluded from MS/MS analysis. Dynamic exclusion was set to 25 s. An average of 62,352 MS/MS spectra were acquired per sample run.

### Data Protocol
For protein identification an existing protein sequence database from project PXD007510 was used. The final database contained 1,439,794 protein sequences including host and symbiont protein sequences, as well as the cRAP protein sequence database (http://www.thegpm.org/crap/) containing protein sequences of common laboratory contaminants. MS/MS spectra were searched against the database using the Sequest HT node in Proteome Discoverer version 2.2.0.388 (Thermo Fisher Scientific) with the following parameters: trypsin (full), maximum two missed cleavages, 10 ppm precursor mass tolerance, 0.1 Da fragment mass tolerance and maximum of three equal dynamic modifications per peptide, namely: oxidation on M (+ 15.995 Da), carbamidomethyl on C (+ 57.021 Da) and acetyl on protein N terminus (+ 42.011 Da). False discovery rates (FDRs) for peptide spectral matches (PSMs) were calculated and filtered using the Percolator Node in Proteome Discoverer. The Percolator algorithm uses semi-supervised learning and a decoy database search strategy to learn to distinguish between correct and incorrect PSMs’. Percolator was run with a maximum delta Cn 0.05, a strict target FDR of 0.01, a relaxed target FDR of 0.05 and validation based on q-value.The Protein FDR Validator Node in Proteome Discoverer was used to calculate q-values for infered proteins based on the results from a search against a target-decoy database. Proteins with a q-value of <0.01 were categorized as high-confidence identifications and proteins with a q-value of 0.010.05 were categorized as medium-confidence identifications  Search results for all samples were combined into a multiconsensus report with Proteome Discoverer and additional filtering of 5% FDR confidence was applied on the protein level. The multiconsensus file was then normalized using the normalized spectral abundance factor (NSAF) (Zybailov et al., 2006 J. Proteome Res. 5 (9) 2339-2347).

### Publication Abstract
Field studies are central to environmental microbiology and microbial ecology, because they enable studies of natural microbial communities. Metaproteomics, the study of protein abundances in microbial communities, allows investigators to study these communities "<i>in situ</i>," which requires protein preservation directly in the field because protein abundance patterns can change rapidly after sampling. Ideally, a protein preservative for field deployment works rapidly and preserves the whole proteome, is stable in long-term storage, is nonhazardous and easy to transport, and is available at low cost. Although these requirements might be met by several protein preservatives, an assessment of their suitability under field conditions when targeted for metaproteomic analyses is currently lacking. Here, we compared the protein preservation performance of flash freezing and the preservation solution RNA<i>later</i> using the marine gutless oligochaete Olavius algarvensis and its symbiotic microbes as a test case. In addition, we evaluated long-term RNA<i>later</i> storage after 1&#x2009;day, 1&#x2009;week, and 4&#x2009;weeks at room temperature (22&#xb0;C to 23&#xb0;C). We evaluated protein preservation using one-dimensional liquid chromatography-tandem mass spectrometry. We found that RNA<i>later</i> and flash freezing preserved proteins equally well in terms of total numbers of identified proteins and relative abundances of individual proteins, and none of the test time points was altered, compared to time zero. Moreover, we did not find biases against specific taxonomic groups or proteins with particular biochemical properties. Based on our metaproteomic data and the logistical requirements for field deployment, we recommend RNA<i>later</i> for protein preservation of field-collected samples targeted for metaproteomic analyses. <b>IMPORTANCE</b> Metaproteomics, the large-scale identification and quantification of proteins from microbial communities, provide direct insights into the phenotypes of microorganisms on the molecular level. To ensure the integrity of the metaproteomic data, samples need to be preserved immediately after sampling to avoid changes in protein abundance patterns. In laboratory setups, samples for proteomic analyses are most commonly preserved by flash freezing; however, liquid nitrogen or dry ice is often unavailable at remote field locations, due to their hazardous nature and transport restrictions. Our study shows that RNA<i>later</i> can serve as a low-hazard, easy-to-transport alternative to flash freezing for field preservation of samples for metaproteomic analyses. We show that RNA<i>later</i> preserves the metaproteome equally well, compared to flash freezing, and protein abundance patterns remain stable during long-term storage for at least 4&#x2009;weeks at room temperature.

### Keywords
1d-lc, Proteobacteria, Rnalater, Metaproteomics, Microbiom, Chemosynthetic symbiosis, Olavius algarvensis, Metaproteome, Sample preservation, Thiotrophic endosymbionts, Field samples, Gutless oligochaetes

### Affiliations
North Carolina State University

### Submitter
Marlene Jensen

### Lab Head
Dr Manuel Kleiner
North Carolina State University


