### Accession
PXD003168

### Title
Host-microbiota interactions in the pathogenesis of antibiotic-associated diseases: Clindamycin

### Description
Improved understanding of the interplay between host and microbes stands to illuminate new avenues for disease diagnosis, treatment and prevention. Here, we provide a high-resolution view of the dynamics between host and gut microbiota during antibiotic induced intestinal microbiota depletion, opportunistic Salmonella typhimurium and Clostridium difficile pathogenesis, and recovery from these perturbed states in a mouse model. Host-centric proteome and microbial community profiles provide an unprecedented longitudinal view revealing the interdependence between host and microbiota in evolving dysbioses. Time- and condition-specific molecular and microbial signatures are evident and clearly distinguished from pathogen-independent inflammatory fingerprints. Our data reveal that mice recovering from antibiotic treatment or C. difficile infection retain lingering signatures of inflammation despite compositional normalization of the microbiota, and host responses could be rapidly and durably relieved through fecal transplant. These experiments define a novel platform for combining orthogonal, untargeted approaches to shed new light on the gastrointestinal ecosystem.

### Sample Protocol
Fecal proteomics was conducted as previously described (Lichtman et al., 2013) with the following notable changes. Instead of analyzing all 4 fractions (20%, 40%, 60% and 100% acetonitrile) from the C4 solid-phase extraction chromatography, only the 40% and 60% fractions were digested and analyzed. Each sample was only analyzed with no technical replicates on a LTQ-Orbitrap Velos mass spectrometer (Thermo Scientific, San Jose).

### Data Protocol
the msConvert program (v3.0.45) was used to generate peak lists from the original data and spectra were assigned to peptides using the SEQUEST (v28.12) (Eng et al., 1994) algorithm. Spectra were queried against a composite “target-decoy” sequence database consisting of the mouse proteome (Uniprot, downloaded October 30, 2012) and reversed decoy versions of the same sequences(Elias and Gygi, 2007). SEQUEST search parameters included using a semi-specific enzyme specificity, 50 ppm precursor mass tolerance, 1 Da fragment ion mass tolerance, static carbidomethylation of cysteines and differential methionine oxidation and glutamine/asparagine deamidation. Each mass spectrometry run was filtered to a 1% peptide FDR and the experiment-wide FDR was filtered to 5% using a linear discriminant analysis(Huttlin et al., 2010).

### Publication Abstract
Improved understanding of the interplay between host and microbes stands to illuminate new avenues for disease diagnosis, treatment, and prevention. Here, we provide a high-resolution view of the dynamics between host and gut microbiota during antibiotic-induced intestinal microbiota depletion, opportunistic Salmonella typhimurium and Clostridium difficile pathogenesis, and recovery from these perturbed states in a mouse model. Host-centric proteome and microbial community profiles provide a nuanced longitudinal view, revealing the interdependence between host and microbiota in evolving dysbioses. Time- and condition-specific molecular and microbial signatures are evident and clearly distinguished from pathogen-independent inflammatory fingerprints. Our data reveal that mice recovering from antibiotic treatment or C. difficile infection retain lingering signatures of inflammation, despite compositional normalization of the microbiota, and host responses could be rapidly and durably relieved through fecal transplant. These experiments demonstrate insights that emerge from the combination of these orthogonal, untargeted approaches to the gastrointestinal ecosystem.

### Keywords
Antibiotic, Mouse, Fecal transplant, C difficile, Gut microbiome

### Affiliations
Chan Zuckerberg Biohub
Department of Chemical & Systems Biology, Stanford University

### Submitter
Joshua Elias

### Lab Head
Dr Joshua E Elias
Department of Chemical & Systems Biology, Stanford University


