### Accession
PXD008406

### Title
Expression of Siglec-E alters the proteome of LPS-activated macrophages but does not affect LPS-driven cytokine production or TLR4 endocytosis Expression of Siglec-E alters the proteome of LPS-activated macrophages but does not affect LPS-driven cytokine production or TLR4 endocytosis

### Description
Siglec-E is a murine CD33-related siglec that functions as an inhibitory receptor and is expressed mainly on neutrophils and macrophage populations.  Recent studies have suggested that siglec-E is an important negative regulator of LPS-TLR4 signalling and one report (Wu et al 2016) claimed that siglec-E is required for TLR4 endocytosis following uptake of Escherichia coli by macrophages and dendritic cells (DCs). Our attempts to reproduce these observations using cells from wildtype (WT) and siglec E deficient mice were unsuccessful. We used a variety of assays to determine if siglec-E expressed by different macrophage populations can regulate TLR-4 signalling in response to LPS, but found no consistent differences in cytokine secretion in vitro and in vivo, comparing 3 different strains of siglec-E-deficient mice with matched WT controls. No evidence was found that the siglec-E deficiency was compensated by expression of siglecs-F and –G, the other murine inhibitory CD33-related siglecs.  Quantitative proteomics was used as an unbiased approach and provided additional evidence that siglec-E does not suppress inflammatory TLR4 signaling.  Interestingly, proteomics revealed a siglec E dependent alteration in macrophage phenotype that could be relevant to functional responses in host defence. In support of this, siglec-E-deficient mice exhibited enhanced growth of Salmonella enterica serovar Typhimurium in the liver following intravenous infection, but macrophages lacking siglec-E did not show altered uptake or killing of bacteria in vitro. Using various cell types including bone marrow derived DCs (BMDC), splenic DCs and macrophages from WT and siglec-E-deficient mice, we showed that siglec-E is not required for TLR4 endocytosis following E.coli uptake or LPS challenge.  We failed to see expression of siglec-E by BMDC even after LPS-induced maturation, but confirmed previous studies that splenic DCs express low levels of siglec-E.  Taken together, our findings do not support a major role of siglec-E in regulation of TLR4 signalling functions or TLR4 endocytosis in macrophages or DCs. Instead, they reveal that induction of siglec-E by LPS can modulate the phenotype of macrophages, the functional significance of which is currently unclear.

### Sample Protocol
WT and siglec-E-deficient mice on C57BL/6J and Balb/c genetic backgrounds were generated as described previously (McMillan et al., 2013;McMillan et al., 2014). Mice were bred and maintained under specific pathogen–free conditions within our own institutional colonies. Animal experimentation was approved by the University of Dundee Animal Ethics Committee and carried out under UK Home Office Project License PPL60/3856. Bone marrow cells were cultured in bacteriological plastic petri dishes with DMEM-F12 media supplemented with penicillin and streptomycin, glutamine, 10% FBS and either 20% L929 conditioned medium or M-CSF (25ng/ml) for 7 days. BMDM were primed with 1 ng/ml LPS for 3 days and stimulated with 100 ng/ml LPS for 7 hours. Monensin-containing GolgiStop (BD Biosciences) was added for the last 6 hours of culture. After washing, cells were lysed in lysis buffer (4% SDS, 50 mM TEAB pH 8.5, 10 mM TCEP), boiled and sonicated with a BioRuptor (30 cycles: 30 seconds on, 30 seconds off) before alkylation with 20 mM iodoacetamide for 1 hour at room temperature in the dark. Four biological replicates from 4 independent biological samples were processed for proteomics. Lysates were subjected to the SP3 protein clean-up procedure (Hughes et al., 2014), eluted into digestion buffer (0.1% SDS, 50 mM TEAB pH 8.5, 1 mM CaCl2) and digested with trypsin at a 1:50 (enzyme:protein) ratio. TMT 10plex labelling and peptide clean-up were performed according to the SP3 protocol. Samples were eluted into 2% DMSO, combined and dried under vacuum. TMT samples were fractionated using offline high pH reverse-phase chromatography. Samples were dissolved in 200 μL ammonium formate (10 mM, pH 10) and peptides were fractionated using RP-High pH HPLC.   A  C18 Column from Waters (XBridge peptide BEH, 130Å, 3.5 µm 2.1 X 150 mm, Ireland) with a guard column (XBridge, C18, 3.5 µm, 2.1X10mm, Waters) was used on an Ultimate 3000 HPLC (Thermo-Scientific). Buffers A and B used for fractionation consist,  respectively of 10 mM ammonium formate in milliQ water and 10 mM ammonium formate with 90% acetonitrile, both buffers were adjusted to pH 10 with ammonia.  Fractions were collected using a WPS-3000FC auto-sampler (Thermo-Scientific) at 1 min intervals. Column and guard column were equilibrated with 2% buffer B for 20 min at a constant flow rate of 0.2 ml/min. Samples (100 µl) were loaded onto the column at 0.2 ml/min, and separation gradient started 1 min after sample loading. Peptides were eluted from the column with a gradient of 2% buffer B to 5% B in 6 min then from 5% B to 60% B in 55 min. The column was washed for 16 min at 100% buffer B and equilibrated at 2% buffer B for 20 min as mentioned above.   The fraction collection started 1 min after injection and stopped after 80 min (total of 80 fractions, 200 µl each). The total number of fractions was concatenated to 22 and the content of the fractions was dried and re-suspended in 50 µl 1% formic acid prior to analysis with LC-MS. Analysis of peptides was performed on a Q-Exactive-HF (Thermo Scientific) mass spectrometer coupled with a Dionex Ultimate 3000 RS Nano-LC system (Thermo Scientific). LC buffers were as follows:  buffer A (2% acetonitrile and 0.1% formic acid in Milli-Q water (v/v)) and buffer B (80% acetonitrile and 0.08% formic acid in Milli-Q water (v/v). Aliquots of each fraction were loaded at 5 μL/min onto a trap column (100 μm × 2 cm, PepMap nanoViper C18 column, 5 μm, 100 Å, Thermo Scientific) equilibrated in loading buffer (0.1% formic acid in Milli-Q water (v/v)). The trap column was washed for 5 min at the same flow rate and then the trap column was switched in-line with a Thermo Scientific, resolving Easy-Spray C18 column (75 μm × 50 cm, PepMap RSLC C18 column, 2 μm, 100 Å). The spray voltage was routinely set to 1.9 kV, and the temperature of the heated capillary was set to 250 °C. The peptides were eluted from the column at a constant flow rate of 300 nl/min with a linear gradient from 5% buffer B to 40% buffer B over 116 mins, and then to 98% buffer B over 10 mins. The column was then washed with 98% buffer B for 15 min and re-equilibrated in 95% buffer A (along with the trap column) for 17 mins. The Q-Exactive HF was used in data dependent mode. A scan cycle comprised MS1 scan (m/z range from 335-1800, with a maximum ion injection time of 50 ms, a resolution of 120,000 and automatic gain control (AGC) value of 3x106, followed by 15 sequential dependant MS2 scans (with an isolation window set to 0.4 m/z, resolution at 60,000, maximum ion injection time at 200 ms and AGC 1x105¬). To ensure mass accuracy, the mass spectrometer was calibrated on the first day that the runs are performed.

### Data Protocol
The raw mass spectrometric data was loaded into MaxQuant (version 1.5.3.30) (Cox and Mann, 2008), using the Andromeda search engine software(Cox et al., 2011). Enzyme specificity was set to that of trypsin/P, allowing for cleavage of N-terminal to proline residues and between aspartic acid and proline residues. Other parameters used were: (i) variable modifications- methionine oxidation, protein N-acetylation; (ii) fixed modifications, cysteine carbamidomethylation; (iii) database: Uniprot - mouse (downloaded 130501, 50800 sequences); (iv) labels: 10-plex TMT (v) MS/MS tolerance: FTMS- 50ppm , ITMS- 0.5 Da; (vi) minimum peptide length, 7; (vii) maximum missed cleavages, 2; (viii) and (ix) PSM & Protein false discovery rate, 1%. For bioinformatic analysis, Reporter ion intensities (corrected) results from MaxQuant were imported into Perseus software (version 1.5.1.6). The normalised corrected reporter ion intensities for each label were used to calculate ratios .and all ‘Contaminant’, ‘Reverse’ & ‘Only identified by site’ proteins were removed from the data. For normalization procedures, the grand total reporter ion intensity for each channel was multiplied by a global scaling factor to adjust its total intensity to the average total intensity across the 10 channels. This corrects for small sample loading and labeling reaction efficiency differences.  All bioinformatics analyses were performed with the Perseus software of the MaxQuant computational platform (Cox and Mann, 2008;Cox et al., 2011;Tyanova et al., 2016). Proteins above 2-fold change [log2(2) =1], proteins with nominal p-value less than 0.05 [-log10(0.05)=1.301] were considered as differentially expressed proteins. GO over representation enrichment analysis was done using WEB-based Gene SeT AnaLysis Toolket and geneontology database (Wang et al., 2017).

### Publication Abstract
Siglec-E is a murine CD33-related siglec that functions as an inhibitory receptor and is expressed mainly on neutrophils and macrophage populations. Recent studies have suggested that siglec-E is an important negative regulator of lipopolysaccharide (LPS)-toll-like receptor 4 (TLR4) signaling and one report (1) claimed that siglec-E is required for TLR4 endocytosis following uptake of <i>Escherichia coli</i> by macrophages and dendritic cells (DCs). Our attempts to reproduce these observations using cells from wild-type (WT) and siglec-E-deficient mice were unsuccessful. We used a variety of assays to determine if siglec-E expressed by different macrophage populations can regulate TLR4 signaling in response to LPS, but found no consistent differences in cytokine secretion <i>in vitro</i> and <i>in vivo</i>, comparing three different strains of siglec-E-deficient mice with matched WT controls. No evidence was found that the siglec-E deficiency was compensated by expression of siglecs-F and -G, the other murine inhibitory CD33-related siglecs. Quantitative proteomics was used as an unbiased approach and provided additional evidence that siglec-E does not suppress inflammatory TLR4 signaling. Interestingly, proteomics revealed a siglec-E-dependent alteration in macrophage protein composition that could be relevant to functional responses in host defense. In support of this, siglec-E-deficient mice exhibited enhanced growth of <i>Salmonella enterica</i> serovar Typhimurium in the liver following intravenous infection, but macrophages lacking siglec-E did not show altered uptake or killing of bacteria <i>in vitro</i>. Using various cell types including bone marrow-derived DCs (BMDCs), splenic DCs, and macrophages from WT and siglec-E-deficient mice, we showed that siglec-E is not required for TLR4 endocytosis following <i>E. coli</i> uptake or LPS challenge. We failed to see expression of siglec-E by BMDC even after LPS-induced maturation, but confirmed previous studies that splenic DCs express low levels of siglec-E. Taken together, our findings do not support a major role of siglec-E in regulation of TLR4 signaling functions or TLR4 endocytosis in macrophages or DCs. Instead, they reveal that induction of siglec-E by LPS can modulate the phenotype of macrophages, the functional significance of which is currently unclear.

### Keywords
Mouse bmdm lc-msms

### Affiliations
Research Associate
Paul R crocker: Professor of Glycoimmunology and Head of the Division of Cell Signalling and Immunology

### Submitter
Manjula Nagala

### Lab Head
Dr Paul R crocker, Manjula Nagala
Paul R crocker: Professor of Glycoimmunology and Head of the Division of Cell Signalling and Immunology


