### Accession
PXD016213

### Title
No evidence for major spacial differences within the microbial communities  in anaerobic digestion plants under different feeding and stirring regimes

### Description
In anaerobic digestion plants (ADP) homogenization of the feed, the fermenter content and the microbial communities represents a precondition for effective and robust biogas production,but also a major energy consumer. For a 850 m3 agricultural AD-P equipped with eight sampling ports we investigated whether different feeding and stirring regimes enable a sufficient homogenization of the microbial communities using metaproteome and TRFLP analysis. Systematic comparison of the samples by scatter plots and students t-test revealed only a limited number of slightly changed metaproteins, taxonomies and biological processes, indicating no systematic differences between the microbial communities in center and rim as well as between top and bottom. However, comparison of the amount of shared identified metaproteins between the sample ports showed minor variation, which might be correlated with the applied stirring strategy. In sum, the applied stirring and feeding conditionswere sufficient to homogenize the microbial communities in AD-Ps largely.

### Sample Protocol
Metaproteome analyzes was carried as previously published(Heyer et al., 2019a). In brief, 2 g sample, 5 g silica beads, 2 mL 2 M sucrose solution and 3.5 mL phenol (10 g phenol dissolved in 1 mL aqua dest.) were shaked in a ball mill for 5 min at 1,800 rpm for cell lysis (FastPrep 96, MP Biomedicals, Eschwege, Germany). After centrifugation (10,000 G, RT, 10 min) the upper phenol phase was transferred into a new 50 mL reaction tube and was washed with the same volume 1 M succrose solution in the ball mill for 5 min at 1,800 rpm. After a further centrifugation step (10,000 G, RT, 10 min) the phenol phase was transferred again into a new 50 mL reaction tube and the proteins were precipitated for 20 min by addition of the fourfold volume of ice-cold 100 mM ammonium acetate in methanol into a freezer at -20 C.  Subsequently, the supernatant was remove after a centrifugation step (10,000 G, 4 °C, 10 min), the pellet was dried and dissolved in 1 mL 7 M urea, 2 M thiourea as well as 0.01 g/mL dithiothreitol. For a better solubilization of the pellet, a few silica beads were added and the reaction tube wasshaked for 3 min in the ball mill. Finally, the sample was centrifuged (10,000 G, 4 °C, 10 min), only the supernatant was transferred into a new reation tube and stored at -20 °C until further usage. For protein quantification the amido black assay was carried out (Popov et al., 1975; Racusen, 1973). Therefore, 50 µL sample was precipitated with 300 µL amido black staining solution (0.26 mg/L amido black, 90% methanol, 10% acetic acid), the sample was centrifuged (16,400 G, RT, 5 min) and the supernatant was decanted. In order to removed unbound dye the pellet was washed twice with 10% acetic acid in methanol and the supernant was decanted each time after a further centrifugation step (16,400 G, RT, 5 min). Subsequently, the protein pellet was dissolved in 1 mL 100 mM sodium hydroxide solution, the absorption was measured in a phometer(Spectrophotometer Genesys 10S UV-Vis, Thermo Scientific, Waltham, USA) at wavelength 615 nm and the values were compared against a calibration curve created with defined amountsof bovine serum albumin. For tryptic digestion a slighthy modified version of the FASP protocol (Wisniewski et al., 2009) was applied. At first 100 µg protein extract were loaded on a FASP filter (Pall Nanosep, MWCO 10kDa Omega, VWR, Dresden) and the urea buffer was removed similar to all following buffers by centrifugation (10,000G, RT, 10 min). Proteins were washed then with 200 µL 8 M urea. Reduction of disulfide bonds was carried out by addition of 100 µL 400 mMdithiothreitol for 20 min at 56 °C on a thermomixer at 300 rpm and alkylation by addition of 100 µL 550 mMiodacetamid for 20 min in the dark. Subsequently, one washing step with 100 µL 8 M urea and three washing steps with 100 µL 50 mM ammonium bicarbonate were carried out and the entire flow-through was discarded. For trypsination 200 µL 50 mM ammonium bicarbonatecontaining 1 µg trypsin were added and the samples were shakedover night at 37 °C on a thermomixer. After a further centrifugation step (10,000G, RT, 5 min), remaining peptides on the filter were rinsed by addition of 50 µL extraction buffer (50 mM ammonium bicarbonate, 5% acetonitrile) and 50 µl LC-MS grade water. Finally, the filters were removed, the samples were dried and stored at -20 °C. Before LC-MS/MS measurement peptides weredissolved in 50 µL 98% LC–MS Water, 2% ACN and 0.05% TFA, centrifuged (13,000G, RT, 10 min) and transferred into an vial. Afterwards, the vials were analysed with an LC–MS/MS system comprising an UltiMate 3000 RSLCnano splitless liquid chromatography system, coupled online to an Orbitrap Elite™ Hybrid Ion Trap-Orbitrap MS (both from Thermo Fisher Scientific, Bremen, Germany).

### Data Protocol
For the further result evaluation MS raw data were converted to the MASCOT generic format (.mgf) using the Proteome Discoverer Software 1.4 (Thermo Fisher Scientific, Bremen, Germany). Protein database search was carried out with MASCOT 2.5 software (Matrix Science, London, England) (Perkins et al., 1999),Xtandem(Craig & Beavis, 2004) and OMSSA (Geer et al., 2004) against aggregated metagenomes and UniProtKB /Swiss-Prot (version: novemer 2017) similar as used in previous publications (Heyer et al., 2019b). The following search parameters were applied: trypsin, one missed cleavage, monoisotopic mass, carbamidomethylation (cysteine) as fixed modification and oxidation (methionine) as variable modifications, ±10 ppm precursor and ±0.5 Da MS/MS fragment tolerance, 113C and +2/+ 3 charged peptide ions, 1% FDR. All files are made available to the public by an upload to PRIDE (Vizcaino et al., 2016), which could be accessed with the accession number XXXXX . For the taxonomic and functional data interpretation mascot results were exported as MASCOT result files (.dat-files) and uploaded into the MPA 2 Software (Heyer et al., 2019a)(http://www.mpa.ovgu.de/, version 2.12). Taxonomic and functional metainformation, in particular the NCBI taxonomy(Federhen, 2012), the UniProtKB keywords, the Enzyme Commission numbers and KEGG Orthology, were retrieved from the UniProtKBhomepage using the JAPI (Patient et al., 2008). Identified proteins assigned to sequences from the metagenome carrying no annotation were linked to annotated UniProtKB entries using BLAST search an e value smaller than 10exp−4. Redundant protein identifications were grouped when they shared at least one peptide and named then metaprotein. The taxonomy of each metaprotein was redefined to the common ancestor taxonomy of all peptides grouped to this metaprotein. Finally the matrix containing all identified metaproteins were exported as comma separated file.

### Publication Abstract
In anaerobic digestion plants (ADs), homogenization of the feed, fermenter content and microbial communities is crucial for efficient and robust biogas production. However, mixing also requires a significant amount of energy. For an 850&#xa0;m<sup>3</sup> agricultural AD equipped with eight sampling ports, we investigated whether different feeding and stirring regimes enable a sufficient homogenization of the microbial community using metaproteomics and terminal restriction fragment length polymorphism (TRFLP) analysis. Systematic comparison of samples taken at the top and the bottom as well as at the rim and the center of the AD using scatter plots and students t-test revealed only a small number of differences in metaproteins, taxonomies and biological processes. Obviously, the applied stirring and feeding conditions were sufficient to largely homogenize the content of the AD.

### Keywords
Metaproteomics, Stirring, Trflp, Microbial communities, Anaerobic digestion; biogas

### Affiliations
OvGU
Otto-von-Guericke Universität Magdeburg

### Submitter
Robert Heyer

### Lab Head
Dr Dirk Benndorf
Otto-von-Guericke Universität Magdeburg


