### Accession
PXD005526

### Title
Proteome analysis of huntingtin knock-in allelic series - Cerebellum

### Description
The objective of this project is to study the proteome in different tissues of huntingtin knock-in allelic series in mice. In the current part of the study, cerebellum samples from the complete remaining allelic series collected at 2, 6, and 10 months were analyzed. To this end, comprehensive quantitative, label-free proteome analysis was performed using Evotec‚Äôs quantitative Deep Proteome Profiling technology.

### Sample Protocol
Depending on tissue weight, tissues were lysed in 500-900ml lysis buffer (8 M urea, 50 mM Tris-HCl pH 8.2, 75 mM NaCl, 5 mM EDTA, 5 mM EGTA, 10 mM sodium pyrophosphate, protease inhibitor cocktail Complete Mini (Roche, Mannheim, Germany)). Tissues were homogenized using an Ultra Turrax and homogenates sonicated using an ultrasonication probe attached to the power supply unit SONOPLUS HD2070 (Bandelin Electronics, Germany). All samples were sonicated for 3x30 s on ice with the power output of the SONOPLUS HD2070 set to 25% and the pulsing set to 3. Samples are centrifuged at >14.000g, 4C for 20 min to remove tissue/cell debris. Protein concentration of the supernatant is determined by Bradford Assay.  200mg of protein per sample were reduced with 10 mM dithiothreitol for 45 min and alkylated with 55 mM iodoacetamide for 30 min in the dark. Subsequently, endoproteinase Lys-C (Wako) was added at an enzyme-to-substrate ratio of 1:50 and incubated for 4 h at room temperature. Samples were thereafter diluted 1:4 with 20 mM Tris-HCl pH 8.2 before adding trypsin (Promega) at an enzyme-to-substrate ratio of 1:50 followed by overnight incubation. The resulting peptide mixtures were acidified by addition of TFA to a final concentration of 0.5% and subsequently desalted using C18 Sep-Pak columns (100 mg sorbent weight, Waters). Peptides were eluted with 50% ACN, 0.5% acetic acid, samples split into two aliquots (100mg each), snap-frozen in liquid nitrogen and lyophilized. 200mg of all peptide samples were subsequently fractionated by high pH reversed phase chromatography (Wang et al., 2011). Briefly, lyophilisates were reconstituted in 20 mM ammonium formate (pH 10, buffer ‚ÄùA‚Äù), loaded on-to an XBridge C18 200 x 4.6 mm analytical column (Waters) operated with the A¬® kta Explorer system (GE Healthcare) and separated by applying a gradient of increasing buffer ‚ÄùB‚Äù concentration (20 mM ammonium formate, 80% ACN, pH 10). The collected 18 fractions of eluting peptides were combined in a concatenated way to generate 6 fractions for each individual sample. Samples were frozen in liquid nitrogen and lyophilized. After desalting via C18 Sep-Pak columns (100 mg sorbent weight, Waters), sample fractions were snap-frozen, lyophilized, and reconstituted in 0.1% FA for MS analysis.  Samples were loaded onto a reverse phase analytical column (packed in-house with C18 beads), resolved by an acetonitrile gradient using a UPLC system (Thermo Fisher Scientific) and directly electrosprayed via a nanoelectrospray ion source into a Q Exactive mass spectrometer (Thermo Fisher Scientific). The Q Exactive mass spectrometer was operated in the data-dependent acquisition mode acquiring full scans at a resolution of 70.000 and fragmentation spectra (MS/MS mode) of the ten most abundant peptide ions at a resolution of 17.500 in the Orbitrap mass analyzer.

### Data Protocol
All acquired raw files were processed together with the MaxQuant software suite (version 1.5.3.31) using the Andromeda search engine for peptide and protein identification and quantification (Cox and Mann, 2008; Cox et al., 2011). The experiments were collectively searched against a Uniprot mouse database (version 09/2015). Carbamidomethylation of cysteine was set as a fixed modification and oxidation of methionine and Nterminal acetylation were set as variable modifications. The minimum required peptide length was seven amino acids and up to two missed cleavages were allowed. A false discovery rate (FDR) of 1% was selected for both protein and peptide identifications and a posterior error probability (PEP) less or equal to 1% for each peptide-to-spectral match was required. The match between runs option was enabled for a time window of 0.7 min.

### Publication Abstract
None

### Keywords
Cerebellum, Proteomics, Huntington's disease

### Affiliations
Christoph Schaab
Evotec

### Submitter
Manuela Machatti

### Lab Head
Dr Christoph Schaab
Christoph Schaab


