### Accession
PXD015196

### Title
The cytotoxic natural product vioprolide A targets nucleolar protein 14 essential for ribosome biogenesis

### Description
The cytotoxic and antifungal peptolides vioprolide A-D were discovered in 1996, however their mode-of-action has so far been unsolved. Anti-cancer effect of the most potent cytotoxic derivative vioprolide A (VioA) were evaluated, showing a prominent potency against human acute lymphoblastic leukemia (ALL) cells. In order to decipher the molecular mode-of-action, a VioA-derived photoprobe (VioA-P) was synthesized and utilized for MS-based AfBPP experiments. In addition thermal protein profiling (TPP) experiments with unmodified VioA were conducted for target discovery. TPP experiments revealed a strong thermal stabilization of two proteins after filtering. NOP14, the protein showing the strongest stabilization effect could further be verified as target with continuative biochemical experiments. As various protein-protein interactions with NOP14 are essential in ribosome biogenesis, e.g. with NOC4L and EMG1 for proper maturation of 18S rRNA and 40S assembly and export, the interactome of NOP14 was revealed by MS-based co-Immunoprecipitation. Our findings indicate, that VioA treatment leads to a loss of interaction between NOP14 and EMG1, whereas the NOP14-NOC4L interface remained unperturbed. These findings could be verified independently by western blot, also showing that overall protein expression of NOP14, EMG1 and NOC4L remained unaltered upon VioA treatment. Delocalization of EMG1 from the nucleus into the cytoplasm as consequence of missing interaction with NOP14 could be verified by immunostaining.

### Sample Protocol
Label-free AfBPP: 6x106 Jurkat cells were treated with VioA-P (0.5 uM, 1 uM, 2 uM) or DMSO. Samples were subjected to UV irradiation, pelleted (3 min, 800 g) and washed with PBS. Cells were lysed with 1 mL lysis buffer (1% (v/v) NP40 and 1% (w/v) sodium deoxycholate) at 4°C for 15 min. Samples were centrifuged and protein concentration in resulting supernatant determined by BCA assay. Equal protein amounts (360 ug) were adjusted to final volume of 280 uL and mixed with trifunctional linker (100 uM, 10 mM stock solution in DMSO), THPTA ligand (500 uM, 10 mM stock solution in DMSO, Sigma-Aldrich) and CuSO4 (1 mM, 50 mM stock solution in H2O) for CuAAC reaction. Click reaction was performed at rt for 1h and proteins were precipitated in 4-fold volume access cold acetone (-20°C), washed with MeOH and enriched on 50 uL avidin-agarose beads in PBS supplemented with 0.2% SDS at rt for 1 h. Beads were washed with 0.2% SDS in PBS, 6 M urea, PBS and supplemented with 200 uL denaturation buffer (7 M urea, 2 M thiourea in 20 mM Hepes, pH 8.0). Reduction and alkylation were performed at rt with 1 mM DTT for 45 min, 5.5 mM IAA for 30 min and 4 mM DTT for 30 min. Proteins were digested with 1 uL Lys-C (0.5 mg/mL, 25°C, 2 h), diluted in 600 uL 50 mM TEAB buffer and digested with 2 uL Trypsin (0.5 mg/mL, 37°C, o.n.). Digestion was stopped with formic acid (FA) and peptides were desalted with Sep-Pak C18 1 cc Vac cartridges (Waters Corp.). Thermal proteome profiling: Jurkat cells were supplemented with VioA or DMSO respectively. After incubation (37°C, 5% CO2, 1 h) cells were washed with PBS and resulting cell pellets taken up in PBS. Cell suspension was distributed into PCR tubes, VioA and DMSO treated samples heated in parallel for 3 min. to 9 different respective temperatures and cells lysed by four freeze-thaw cycles. Next, PBS (50 uL) was added and samples subjected to centrifugation (100,000 g, 20 min, 4°C). For protein amount normalization in the supernatant, protein concentration of the two lowest temperature point samples were determined via BCA assay. Proteins were precipitated with acetone (400 uL) overnight (-20Â°C), pelleted and resuspended by sonication in methanol (500 uL). Subsequently, proteins were resuspended in X-buffer (300 uL, 7 M urea, 2 M thiourea in 20 mM Hepes buffer pH 7.5) by sonication. Reduction and alkylation were performed at rt with 1 mM DTT for 45 min, 5.5 mM IAA for 30 min and 4 mM DTT for 30 min. Proteins were digested with 1 uL Lys-C (0.5 mg/mL, 25°C, 2 h), diluted in 50 mM TEAB buffer and digested with 4 uL Trypsin (0.5 mg/mL, 37°C, o.n.). Digestion was stopped with FA and peptides desalted with Sep-Pak C18 1 cc Vac cartridges (Waters Corp.), dried and taken up in elution buffer and a volume corresponding to 25 ug protein in the two lowest temperature points further used. Samples were dried, taken up in buffer 1 (10 uL, 10% MeCN 90% 200 mM TEAB buffer) and incubated (25°C, 15 min, 400 rpm). TMT labels (TMT10plexTM isobaric labels, Thermo Fisher Scientific) were taken up in MeCN (41 uL). 10 uL of each label solution was added to a sample according to applied temperature and incubated (25°C, 1 h, 400 rpm). Reaction was stopped with 5% (v/v) hydroxylamine solution. Labeled peptides were combined into a single sample per experiment, buffer 2 (100 uL, 60% (v/v) 200 mM TEAB buffer 40% (v/v) MeCN) added and samples dried. Peptides were reconstituted in 0.5% FA (200 uL), desalted using C18 StageTips (Empore disk-C18; 47 mm; Agilent Technologies) and dried. Reconstituted samples were fractionated by HILIC-separation and fractions collected. Co-IP: Cells were incubated with VioA or DMSO. Crosslinking reaction was performed with 2 mM DSSO in 1 mL PBS at 37°C and 250 rpm for 1 h. Reaction was quenched with 50 mM Tris-HCl pH 8.0 and cells were lysed in 50 mM Tris-HCl, 150 mM NaCl, 10% glycerol, 1% NP40, pH 8.0. Unsoluble fraction was pelleted. 30 uL of Protein AG beads (Pierce) were equilibrated with 1 mL wash buffer (50 mM Tris-HCl, 150 mM NaCl, 10% glycerol, 0.05% NP40, pH 8.0) and were incubated with 500 ug protein lysate and 10 uL polyclonal rabbit anti-NOP14 antibody (0.1 mg/mL, Sigma-Aldrich) or 0.4 uL of rabbit IgG isotype control (2.5 mg/mL, Cell signaling) o.n. at 4°C. Proteins were washed with wash buffer and with basic buffer (50 mM Tris-HCl, 150 mM NaCl, 10% glycerol pH 8.0). Beads were treated with 25 uL of elution buffer I (50 mM Tris-HCl, 2 M urea, 1 mM DTT, 0.5 ng/uL trypsin) for 30 min at rt and with100 uL elution buffer II (50 mM Tris-HCl, 2 M urea, 5 mM IAA) overnight at 37°C. Peptides were desalted using self-made stage tips (2xC18). LC-MS/MS: Samples were analyzed by LC-MS/MS using an UltiMate 3000 nano HPLC system (Dionex) equipped with an Acclaim C18 PepMap100 75 um ID x 2 cm trap and an Acclaim Pepmap RSLC C18 separation column (75 um ID x 50 cm) in an EASY-spray setting coupled to an Orbitrap Fusion or Q-Exactive Plus (Thermo Fisher) using a 120 or 132 min. gradient.

### Data Protocol
AfBPP data and co-IP data: MS raw data was analyzed using MaxQuant software (version 1.6.2.10) with Andromeda search engine (searches were carried out against Uniprot database for Homo sapiens (taxon identifier: 9606, downloaded on 15.10.2018, canonical). Carbamidomethlyation of cysteines (C) was set as fixed modifications and oxidation of methionines (M) and and acetylation of N-termini as variable modifications. Trypsin (without N-terminal cleavage to proline) was set as proteolytic enzyme with a maximum of two allowed missed cleavages. For main search, precursor mass tolerance was set to 4.5 ppm and fragment mass tolerance to 0.5 Da. Label free quantification (LFQ) mode was activated and with a LFQ minimum ratio count of 1. Second peptide identification was enabled, and false discovery rate (FDR) determination carried out by applying a decoy database and thresholds were set to 1% FDR at peptide-spectrum match and at protein levels. Normalized LFQ intensities extracted from the MaxQuant result table proteinGroups.txt were further analyzed with Perseus software (version 1.6.3.2). Prior to analysis, putative contaminants, reverse hits and only identified by site hits were removed. Normalized LFQ intensities were log2 transformed and proteins with two valid values for AfBPP data and three valid values for co-IP data in at least one group were used for missing value imputation from normal distribution (width 0.3, downshift 1.8, total matrix). Two-sample StudentsÃ¢â‚¬â„¢s t-test including Benjamini-Hochberg multiple testing correction (FDR = 0.05) was performed. Proteins with an enrichment factor of 2 (log2(x) = 1) and -log10 t-test p-value of 1.3 for AfBPP data and an enrichment factor of 4 (log2(x) = 2) for co-IP data and -log10 t-test p-value of 1.3 were considered as significantly enriched proteins. Thermal proteome profiling: Recorded raw files were processed using MaxQuant software (version 1.6.2.10) and MS/MS spectra were searched against the Uniprot FASTA database for homo sapiens (taxon identifier: 9606, downloaded on 15.10.2018, canonical). Trypsin (without proline) was set as proteolytic enzyme with a maximum of two missed cleavages allowed. Fractions were set according to HILIC-fractionation. Group-specific parameters were set to Ã¢â‚¬Å“Reporter ion MS2Ã¢â‚¬Â with 10plex TMT isobaric labels for N-terminal and lysine residue modification selected. Reporter mass tolerance was set to 0.003 Da and Filter by PIF activated (min. reporter PIF set to 0.75). Ion isotopic distribution was set for each TMT label (N-terminal and lysine) as stated in Table S5 for the use of isotope correction. Cysteine carbamidomethylation was set as fixed modification and methionine oxidation as well as N-terminal protein acetylation as variable modifications with a max. number of modifications per peptide set to 5. The second peptide identification and Ã¢â‚¬Å“match between runsÃ¢â‚¬Â (0.7 min match time window, 20 min alignment time window) was enabled. False discovery rate determination was carried out using a decoy database and thresholds were set to 1% FDR at peptide-spectrum match and at protein levels. Protein quantification was carried out with label minimum ratio count set to 2 and peptides for quantification set to Ã¢â‚¬Å“unique and razorÃ¢â‚¬Â. Remaining parameters were utilized as given in the default settings. Downstream data analysis of corrected reporter intensities calculated by MaxQuant in the protein groups table were further used for the determination of melting curves and Tm shifts which was performed with R (version 3.5.1 Ã¢â‚¬Å“Feather SprayÃ¢â‚¬Â) and the TPP package (version 3.0.3, Franken et al.). Corrected intensities were normalized to the lowest applied temperature TMT channel. Data analysis was carried out as stated by the authors with minor changes (fcColumn = c(6,8,9); thresholdlower = c(0.4,0,0)). Prior to visualization of obtained output files, data was filtered using the following published (Franken et al., Becher et al.) criteria: R2 > 0.8 for fitted curves for DMSO and VioA treatment; plateau of < 0.3 for DMSO curves; steepest slope of protein melting curves in paired set of DMSO and VioA treated conditions < -0.06; melting point difference for each protein between both DMSO replicates < 1.5°C.Following additional criteria were applied for hit identification: One of the p values for the two replicate experiments is < 0.1 and the other is < 0.05; melting point shifts for both paired replicates

### Publication Abstract
Novel targets are needed for treatment of devastating diseases such as cancer. For decades, natural products have guided innovative therapies by addressing diverse pathways. Inspired by the potent cytotoxic bioactivity of myxobacterial vioprolides&#x2005;A-D, we performed in-depth studies on their mode of action. Based on its prominent potency against human acute lymphoblastic leukemia (ALL) cells, we conducted thermal proteome profiling (TPP) and deciphered the target proteins of the most active derivative vioprolide&#x2005;A (VioA) in Jurkat cells. Nucleolar protein 14 (NOP14), which is essential in ribosome biogenesis, was confirmed as a specific target of VioA by a suite of proteomic and biological follow-up experiments. Given its activity against ALL cells compared to healthy lymphocytes, VioA exhibits unique therapeutic potential for anticancer therapy through a novel mode of action.

### Keywords
Lc-ms, Thermal proteome profiling, Target identification, Natural product, Co-ip, Afbpp

### Affiliations
Technical University of Munich 
Stephan A. Sieber, Center for Integrated Protein Science at the Department of Chemistry, Technische Universitä¤t München, Garching b. München 85747, Germany; Chair of Organic Chemistry II, Technische Universität München, Garching b. München 85747, Germany

### Submitter
Volker Kirsch

### Lab Head
Dr Stephan A. Sieber
Stephan A. Sieber, Center for Integrated Protein Science at the Department of Chemistry, Technische Universitä¤t München, Garching b. München 85747, Germany; Chair of Organic Chemistry II, Technische Universität München, Garching b. München 85747, Germany


