### Accession
PXD043254

### Title
Non - canonical functions of UHRF1 maintain DNA methylation homeostasis in cancer cell

### Description
We have generated and validated degron alleles of UHRF1 and/or DNMT1 in several human colorectal cancer cell lines. We then used genomics and bioinformatics to precisely describe he DNA demethylation dynamics in these cells, leading to the conclusion that UHRF1 maintains DNA methylation in cancer cells not only by stimulating DNMT1. Proteomics and genetics lead us to conclude that UHRF1 regulates DNMT3A, DNMT3B and TET2 activity in addition to regulating DNMT1. The tools we have developed will be valuable for future research efforts, and our results advance our understanding of cancer epigenetics, with potentially important therapeutic applications.

### Sample Protocol
Total proteome sample preparation Previously flash-frozen samples were quickly placed on ice and pellets were solubilized in 200 µL lysis buffer (6 M guanidinium Chloride, 100 mM Tris–HCl pH 8.5, 2 mM DTT) and heated for 10 min at 99°C at 1,400 rpm. Subsequently, samples were sonicated at 4°C in 30 s on/off intervals for 15 cycles. If the viscosity of the samples was sufficiently reduced, protein concentrations were estimated. For concentration measurements, the Pierce™ BCA Protein Assay Kit (23225, Thermo Fisher Scientific) was employed following the manufacturer's instructions. After at least 20 min of incubation with 40 mM chloroacetamide, 30 µg of each proteome sample was diluted in a 30 µL lysis buffer supplemented with CAA and DTT. Samples were diluted in 270 µL digestion buffer (10% acetonitrile, 25 mM Tris–HCl pH 8.5, 0.6 μg Trypsin/sample (Pierce™ Trypsin Protease, 90058, Thermo Fisher Scientific) and 0.6 μg/sample LysC (Pierce™ LysC Protease, 90051, Thermo Fisher Scientific) and proteins digested for 16 h at 37°C with constant shaking at 1,100 rpm. To stop protease activity 1% (v/v) trifluoroacetic acid (TFA) was added the next day and samples were loaded on self-made StageTips consisting of three layers of SDB-RPS matrix (Empore) (Rappsilber, Mann and Ishihama, 2007). After loading, two washing steps with 0.1% (v/v) TFA were scheduled and peptides were eluted by 80% acetonitrile and 2% ammonium hydroxide. Upon evaporation of the eluates in a SpeedVac centrifuge, samples were resuspended in 20 µL 0.1% TFA and 2% acetonitrile. After complete solubilization of peptides by constant shaking for 10 min at 2,000 rpm, peptide concentrations were estimated at 280 nm. Chromatin aggregation capture Previously flash-frozen samples were quickly placed on ice and pellets were solubilized in 1 mL ChAC lysis buffer (20 mM HEPES pH 7.4, 10 mM NaCl, 3 mM MgCl2, 0.1% NP40, freshly added 1× cOmplete™ EDTA-free Protease Inhibitor Cocktail (04693132001, Roche)) and incubated for 10 min on ice. Nuclei were pelleted by centrifugation (2,300 g, 5 min, 4°C) and the supernatant was discarded. In the differential fraction analysis (Figure 2A), the supernatant was saved as the cytosolic fraction. Upon a second wash of the nuclei pellet with the ChAC lysis buffer, the nuclei were taken into 3 mL crosslinking buffer (PBS pH 7.4 (806552, Sigma), 1x cOmplete™ EDTA-free Protease Inhibitor Cocktail). Formaldehyde (28906, Thermo Fisher Scientific) was added to a final concentration of 1% and samples were incubated for 10 min on an orbital shaker at room temperature. Excess formaldehyde was then quenched by 125 mM Glycine for 5 min and crosslinked cells were washed twice with ice-cold PBS. Nuclei were lysed in 300 µL SDS buffer (50 mM HEPES pH 7.4, 10 mM EDTA pH 8.0, 4% UltraPure™ SDS Solution (24730020, Invitrogen), freshly added 1× cOmplete™ EDTA-free Protease Inhibitor Cocktail) by gentle pipetting. After 10 min incubation at room temperature, 900 µL freshly prepared Urea buffer (10 mM HEPES pH 7.4, 1 mM EDTA pH 8.0, 8 M Urea (U4883, Sigma)) was added. Tubes were carefully inverted 7 times and centrifuged at 20,000 g and room temperature for 30 min. The supernatant was discarded without perturbing the pellet. The pellet was resuspended in 300 µL Sonication buffer (10 mM HEPES pH 7.4, 2mM MgCl2, freshly added 1× cOmplete™ EDTA-free Protease Inhibitor Cocktail). Before sonication, two additional wash steps can be scheduled (one SDS and Urea wash and one SDS only wash) (Kustatscher, Wills, et al., 2014), but to our hands, this did not notably improve the chromatin enrichment efficiency. The chromatin samples were sonicated using a Bioruptor ® Plus at 4°C for 15 cycles (30 s on, 60 s off). The protein concentration was estimated by the Pierce™ BCA Protein Assay Kit. Next, protein aggregation capture (PAC) was performed. Here 1,000 µg of undiluted Sera-MagTM beads (1 mg, GE24152105050250, Sigma) per 100 µg chromatin solution were washed three times by 70% acetonitrile. 300 µL of the chromatin solution corresponding to 100 µg was added after the last wash to the beads and chromatome-bead mixtures were vortexed. After 10 min incubation on the bench, the samples were again vortexed and rested on the bench. A first wash followed this with 700 µL 100% acetonitrile, a second wash with 1 mL 95% acetonitrile, and a third wash with 1 mL 70% ethanol. The remaining ethanol was allowed to evaporate and beads were resuspended in 400 µL 50 mM HEPES pH 8.5 supplemented with fresh 5 mM TCEP and 5.5 mM CAA. Samples were incubated for 30 min at room temperature upon which LysC (1:200) and Trypsin (1:100) were added. Proteins were digested overnight at 37°C. From this step on, samples were treated exactly like the total proteome samples.

### Data Protocol
Chromatomes and proteomes with DIA-NN DIA-NN-based analysis of raw MS data acquired in DIA mode was performed by using version 1.8 in “high accuracy” mode. Instead of a previously measured precursor library, spectra and RTs were predicted by a deep learning-based algorithm and spectral libraries were generated from FASTA files. Cross-run normalization was established in an RT-dependent manner. Missed cleavages were set to 1. N-terminal methionine excision was activated and cysteine carbamidomethylation was set as a fixed modification. Proteins were grouped with the additional command “--relaxed-prot-inf”. Match-between runs was enabled and the precursor FDR was set to 1%.

### Publication Abstract
None

### Keywords
Dnmt1, Dna methylation, Uhrf1, Epigenetics, Chromatin proteome

### Affiliations
Faculty of Biology and Center for Molecular Biosystems (BioSysM), Human Biology and BioImaging, LMU Munich & Department of Proteomics and Signal Transduction, Max-Planck Institute of Biochemistry
Université Paris Cité, CNRS, Epigenetics and Cell Fate, F-75013 Paris, France

### Submitter
Enes Ugur

### Lab Head
Dr Pierre-Antoine Defossez
Université Paris Cité, CNRS, Epigenetics and Cell Fate, F-75013 Paris, France


