### Accession
PXD035669

### Title
A proteomic investigation of sporophyte and gametophyte in the red seaweed Asparagopsis

### Description
The red seaweed, Asparagopsis taxiformis, is a promising ruminant feed-additive with significant anti-methanogenic properties, which could provide a global solution to climate-change. Emerging genomics in this species has provided a strong foundation for in-depth molecular investigations, including proteomics. Here, we investigated the A. taxiformis (L6) proteome of sporophyte and gametophyte (male and female), using soluble and insoluble extraction methods followed by mass spectrometry. A combined total of 741 and 2007 unique non-redundant proteins were identified using genome-derived and transcriptome-derived databases, respectively. Genome-derived proteins were used for further analysis including a relatively large proportion of proteins that were annotated as ion binding proteins (i.e. iron, zinc, manganese, potassium and copper), which may play a role in heavy metal bioremediation. In addition, we identified 44 photosynthesis-related proteins (e.g. phycobilisomes, photosystem I, photosystem II and ATPase) and 58 stress-related proteins (e.g. heat shock proteins and vanadium-dependent haloperoxidases). Forty proteins were predicted to be secreted, supporting a role for them in extracellular activities such as mucilage development and defence. Two Gal-2,6-Sulfurylases II proteins are likely necessary for carrageenan biosynthesis, as well as development of reproductive-associated structures. These findings provide a comprehensive overview of expressed proteins in A. taxiformis (L6) at two different life stages, highlighting the potential of proteomic approaches and warrant further studies for understanding protein functions.

### Sample Protocol
Sample collection A. taxiformis (L6) were sampled randomly as sporophytes and gametophytes (male and female) from the sub-tidal from ~0.5-3m depth by snorkelling from the Moffat Beach located North-East of Caloundra CBD, Sunshine Coast (Queensland, Australia). Within 1 h post-collection, samples were transported with sufficient seawater to the laboratory of the University of the Sunshine Coast. In the laboratory, the samples were cleaned with sterile seawater to remove visible mud, epiphytes and epibionts. The samples were then frozen (in 1.5 ml tubes) in liquid nitrogen before storage at −80◦C until use. Protein extraction and proteomic analysis Frozen A. taxiformis (L6) gametophyte (male and female) tips and whole sporophytes were manually powdered using a sterile mortar and pestle in liquid nitrogen. Proteins were extracted using two different solutions: (1) Tris-HCl, pH 7.4 buffer, and (2) 6 M urea, chaotropic agent, with and without acetone precipitation. For a single extraction, approximately 100 mg of powdered sample was placed into a 50 ml tube. To obtain the water-soluble proteins, the powdered sample was homogenised with 20 mM Tris-HCl pH 7.8 buffer containing protease inhibitor (100x, Merck), briefly vortexed and centrifuged at 4,000xg for 25 min at 4°C, followed by separate extraction for supernatant and pellet. The initial supernatant was subjected to extraction by non-acetone and acetone procedures. For without acetone precipitation, the supernatant was collected and then centrifuged through an Amicon Ultra-4 centrifugal filter unit (3 kDa; Millipore Corporation, Billerica, MA, USA) at 4,000xg for 40 min (4°C) to purify proteins followed by lyophilisation in a vacuum concentrator (Express SpeedVac Concentrator SC250, Thermo Fisher Scientific, USA). This extraction resulted in two protein samples one over 3 kDa and one below 3 kDa, labelled as TU (Tris Upper) and TB (Tris Bottom) respectively. However, for acetone precipitation, the supernatant was mixed with a pre-chilled (-20°C) solution of 100% acetone (volume= 4X that of the protein samples to be precipitated) and kept overnight at -20°C for precipitation. On the next morning, the supernatant was centrifuged at 4,000xg for 25 min at 4°C, decanted, and the protein pellets were air dried to evaporate the acetone from the uncapped tube which was labelled as TA (Tris Acetone) sample.  For water-insoluble proteins, the remaining pellets were resuspended in lysis buffer (6 M urea) vortexed and centrifuged at 4,000xg for 25 min (4°C). As above, the supernatant was collected, centrifuged through an Amicon Ultra-4 centrifugal filter unit and lyophilized in a vacuum concentrator. This lysis buffer extraction resulted in two protein samples one over 3 kDa and one below 3 kDa, labelled as UU (Urea Upper) and UB (Urea Bottom) respectively. Similarly, acetone precipitation of the supernatant was carried out as above, designated as UA (Urea-acetone) sample. Lyophilised protein samples were separated by solid-phase extraction (SPE) using a C18 column, 500 mg sorbent per cartridge, 55-105 µm -particle size (Waters) except for the TA, UA and TU samples after dissolved in 0.1% trifluoroacetic acid (TFA) in Milli-Q water. In brief, the columns were conditioned and equilibrated with 100% Acetonitrile (ACN) and 0.1% trifluoroacetic acid (TFA) in Milli-Q water respectively. Then samples were loaded followed by washing two times with 0.1% TFA in Milli-Q water and finally, eluted by 0.5% acetic acid in 70% can in MilliQ. Afterwards, the SPE eluate was centrifuged at 4,000xg for 8 min (4°C), and evaporated in a vacuum concentrator (Express SpeedVac Concentrator SC250, Thermo Fisher Scientific, USA) and stored at −20 ◦C prior to further analysis.

### Data Protocol
Proteomic identification and annotation Proteins were analysed using PEAKS v7.0 (BSI, Canada) against the protein database built from the reference genome database (10,863 proteins) (Zhao et al, 2022), as well as from the A. taxiformis (L6) de novo transcriptome database (2,320,110 proteins). De novo sequencing of proteins, database searches and characterisation of specific post-translational modifications were used to analyse the raw data; false discovery rate was set to ≤ 1%, and [-10*log(p)] was calculated accordingly, where p is the probability that an observed match is a random event. The PEAKS used the following parameters: (i) precursor ion mass tolerance, 15 ppm; (ii) fragment ion mass tolerance, 0.1 Da (the error tolerance); (iii) tryptic enzyme specificity with three missed cleavages allowed; (iv) monoisotopic precursor mass and fragment ion mass; (v) a fixed modification of cysteine carbamidomethylation; and (vi) variable modifications including lysine acetylation, deamidation on asparagine and glutamine, oxidation of methionine and conversion of glutamic acid and glutamine to pyroglutamate. MS raw data was deposit into the PRIDE database under accession number XXXXX.  OmicsBox software v1.4.11 (BioBam Bioinformatics SL, Valencia, Spain) was used for protein functional annotation. Photosynthesis- and stress-related proteins were identified based on Chondrus crispus gene annotations [11] while A. taxiformis (L6) gene annotations were used to identify predicted secreted proteins (Zhao et al, 2022).

### Publication Abstract
The red seaweed <i>Asparagopsis taxiformis</i> is a promising ruminant feed additive with anti-methanogenic properties that could contribute to global climate change solutions. Genomics has provided a strong foundation for in-depth molecular investigations, including proteomics. Here, we investigated the proteome of <i>A. taxiformis</i> (Lineage 6) in both sporophyte and gametophyte stages, using soluble and insoluble extraction methods. We identified 741 unique non-redundant proteins using a genome-derived database and 2007 using a transcriptome-derived database, which included numerous proteins predicted to be of fungal origin. We further investigated the genome-derived proteins to focus on seaweed-specific proteins. Ontology analysis indicated a relatively large proportion of ion-binding proteins (i.e., iron, zinc, manganese, potassium and copper), which may play a role in seaweed heavy metal tolerance. In addition, we identified 58 stress-related proteins (e.g., heat shock and vanadium-dependent haloperoxidases) and 44 photosynthesis-related proteins (e.g., phycobilisomes, photosystem I, photosystem II and ATPase), which were in general more abundantly identified from female gametophytes. Forty proteins were predicted to be secreted, including ten rhodophyte collagen-alpha-like proteins (RCAPs), which displayed overall high gene expression levels. These findings provide a comprehensive overview of expressed proteins in <i>A. taxiformis</i>, highlighting the potential for targeted protein extraction and functional characterisation for future biodiscovery.

### Keywords
Asparagopsis taxiformis (l6), Gametophyte, Proteomics, Lc-ms/ms, Sporophyte

### Affiliations
University of the Sunshine Coast
Centre for Bioinnovation, University of the Sunshine Coast, Maroochydore, Queensland 4558, Australia.

### Submitter
Tianfang Wang

### Lab Head
Dr Tianfang Wang
Centre for Bioinnovation, University of the Sunshine Coast, Maroochydore, Queensland 4558, Australia.


