### Accession
PXD000901

### Title
Phospho-networks in Ovarian Cancer Cell Lines

### Description
Cancer cells acquire pathological phenotypes through accumulation of mutations that perturb signaling processes. While thousands of mutations have been identified, mostly by genome-wide sequencing, systematic interpretation of their role in cancer and impact on cellular information processing is presently missing. Here, we propose a computational approach (ReKINect) to identify mutations attacking signaling networks. We demonstrate six types of network-attacking mutations (NAMs) including changes in kinase modulation, network rewiring as well as the genesis and extinction of specific phosphorylation sites. Through global, quantitative analysis of the exomes and (phospho-)proteomes of five ovarian cancer cell lines we identify and validate numerous NAMs. Finally, we explore the entire cancer genome repertoire and predict hundreds of NAMs affecting kinase and SH2 driven signaling. Our approach is scalable with the complexity of cancer genomes and cell signaling, and can be readily applied in personalized precision medicine.

### Sample Protocol
Es2, Ovas, Ovise, Tov-21 and Koc-7c cells were labeled with medium Stable Isotope Labeling by Amino acids in cell Culture (SILAC) and grown to ~80% confluency in sufficient 15cm dishes to provide 12mg of protein as starting material in triplicate. Each medium-labeled sample would be subsequently mixed 1:1 with a Spike-in SILAC{Geiger et al., 2011, Nat Protoc, 6, 147-57} sample labeled with heavy SILAC containing a mix of peptides from the different ovarian cell lines, so that such internal standard would allow inter-sample comparisons. After synchronization, cells were lysed using ice-cold modified RIPA buffer supplemented with Roche complete protease inhibitor cocktail tablets and ÃŸ-glycerophosphate (5mM), NaF (5mM), Na-orthovanadate (1mM, activated). The samples were processed further as detailed in the Extended Experimental Procedures section and analyzed by liquid chromatography-mass spectrometry.

### Data Protocol
MaxQuant 1.3 analysis, default settings. Genome specific database used for searching, with cell-line specific FASTA file being generated.

### Publication Abstract
Cancer cells acquire pathological phenotypes through accumulation of mutations that perturb signaling networks. However, global analysis of these events is currently limited. Here, we identify six types of network-attacking mutations (NAMs), including changes in kinase and SH2 modulation, network rewiring, and the genesis and extinction of phosphorylation sites. We developed a computational platform (ReKINect) to identify NAMs and systematically interpreted the exomes and quantitative (phospho-)proteomes of five ovarian cancer cell lines and the global cancer genome repository. We identified and experimentally validated several NAMs, including PKC&#x3b3; M501I and PKD1 D665N, which encode specificity switches analogous to the appearance of kinases de novo within the kinome. We discover mutant molecular logic gates, a drift toward phospho-threonine signaling, weakening of phosphorylation motifs, and kinase-inactivating hotspots in cancer. Our method pinpoints functional NAMs, scales with the complexity of cancer genomes and cell signaling, and may enhance our capability to therapeutically target tumor-specific networks.

### Keywords
Phospho, Proteome, Ovarian, Silac, Tio2, Cancer

### Affiliations
Technical University of Denmark (DTU)
Technical University of Denmark

### Submitter
Pau Creixell

### Lab Head
Dr Rune Linding
Technical University of Denmark


