### Accession
PXD027949

### Title
ProMetheus: a database of hmSILAC-validated protein methylation sites

### Description
Protein arginine (R) methylation is a post-translational modification that has been shown to play a role in various biological processes, such as RNA splicing, DNA repair, immune response, signal transduction and tumor development. Here, we present a dataset of high-quality methylations obtained from several different heavy methyl SILAC (hmSILAC) experiments analyzed with a machine learning model that was trained to recognize hmSILAC doublets and show that this model allows for improved high-confidence identification of methyl-peptides. The results of our analysis of the interactions between R-methylated proteins further support the idea that this modification plays a role in modulating protein:protein interactions and suggest a potential new role of R methylation in immunity and macrophage metabolism. Moreover, we intersect the methyl-site dataset with a phosphosite dataset to investigate the cross-talk between R methylation and phosphorylation. Finally, we explore the application of hmSILAC to identify unconventional methylated residues on both histone and non-histone proteins.

### Sample Protocol
The annotation of the human methyl-proteome was obtained by combining datasets derived from two distinct proteomics strategies. The majority of the methylations were annotated from hmSILAC experiments that allow this PTM to be orthogonally validated (as described in Ong et al. 2004). This initial list was then integrated with another list of regulated methyl-peptides derived from MS-proteomics experiments where the biochemical protocol to separate and affinity-enrich methyl-peptides was coupled to metabolic labelling with standard SILAC amino acids (i.e., K and R) and stimulation of cells with different perturbations (e.g. from pharmacological inhibition of genetic modulation of PRMTs, to treatment with genotoxic drugs). Each SILAC experiment was done in duplicates (forward and reverse) and only peptides whose regulation was reproducible in at least one pair of experiments was considered significantly regulated. The attached .xlsx table includes sheets that briefly summarize the protocols adopted to prepare each sample. Peptide mixtures were analysed by online nano-flow liquid chromatography tandem mass spectrometry using an EASY-nLC 1000 (Thermo Fisher Scientific) connected to a Q Exactive instrument (Thermo Fisher Scientific) through a nano-electrospray ion source. The nano-LC system was operated in one column set-up with a 50cm analytical column (75 um inner diameter) packed with C18 resin (easySpray PEPMAP RSLC C18 2M 50cm x 75 M, Fisher Scientific) configuration. Solvent A was 0.1% formic acid (FA) and solvent B was 0.1% FA in 80% ACN. Samples were injected in an aqueous 0.1% TFA solution at a flow rate of 500 nL/min. Peptides were separated with a gradient of 5-40% solvent B over 90 min followed by a gradient of 40-60% for 10 min and 60-80% over 5 min at a flow rate of 250 nL/min in the EASY-nLC 1000 system. The Q-Exactive was operated in the data-dependent mode (DDA) to automatically switch between full scan MS and MS/MS acquisition. Survey full scan MS spectra (from m/z 300-1150) were analysed in the Orbitrap detector with resolution R=35,000 at m/z 400. The ten most intense peptide ions with charge states 2+ were sequentially isolated to a target value of 3e+6 and fragmented by Higher Energy Collision Dissociation (HCD) with a normalized collision energy setting of 25%. The maximum allowed ion accumulation times were 20 ms for full scans and 50 ms for MS/MS and the target value for MS/MS was set to 1e+6. The dynamic exclusion time was set to 20s.

### Data Protocol
Acquired MS raw data were analyzed using the integrated MaxQuant software v1.3.0.5 or v1.5.2.8 (for SILAC) or v1.6.2.10 (for hmSILAC), using the Andromeda search engine (Cox and Mann, 2008; Cox, et al., 2011). In all MaxQuant searches, the estimated FDR of all peptide identifications was set to a maximum of 1%. The main search was performed with a mass tolerance of 4.5 ppm. A maximum of 3 missed cleavages was permitted, and the minimum peptide length was fixed at 6 amino acids. The June 2020 version of the Uniprot reference proteome (Proteome ID: UP000005640) was used for peptide identification. Other parameters were different, according to the experiment and the sample to be analysed. 1) For the hmSILAC experiments, each MS raw data was analyzed twice to identify Heavy and Light methyl-peptides separately. In the ÃƒÂ¢Ã¢â€šÂ¬Ã…â€œLightÃƒÂ¢Ã¢â€šÂ¬Ã¯Â¿Â½ analysis, mono-methylation of K/R (+14.016 Da), di-methylation of K/R (+28.031 Da), tri-methylation of K (+42.047 Da) and oxidation of M (+15.995 Da) were specified as variable modifications; Carbamidomethylation of Cys (+57.021 Da) was indicated as fixed modification. In the ÃƒÂ¢Ã¢â€šÂ¬Ã…â€œHeavyÃƒÂ¢Ã¢â€šÂ¬Ã¯Â¿Â½ analysis, heavy mono-methylation of K/R (+18.038 Da), heavy di-methylation of K/R (+36.076 Da), heavy tri-methylation of K (+54.114 Da) and oxidation of M were specified as variable modifications; Carbamydomethylation of Cysteine (C) and isotope-labelled Methionine (MS) (+4.022 Da) were indicated as fixed modifications. Enzyme specificity was set to either Trypsin/P or LysargiNase. 2) For the standard SILAC methyl-proteomics experiments, we indicated K8+R10 and/or K4+R6 as SILAC labels. N-terminal acetylation (+42.010 Da), M oxidation, mono-methyl-K/R and di-methyl-K/R were set as variable modifications; Carbamidomethylation of C was set as fixed modification. Enzyme specificity was set to Trypsin/P. 3) For histone hmSILAC experiments, MS raw data were also analysed twice. Mono-methylation (light or heavy) was allowed on K, R, D, E, H, Q, N, S and T and we included K acetylation (+42.011 Da) as variable modification to address the fact that this modification is very abundant and likely coexists with methylation on histones. Enzyme specificity was set to Trypsin/P, ArgC, LysC or LysargiNase. Finally, to reduce search complexity, a database containing only human histone sequences was used. Identification of light and heavy methyl-peptides doublets was carried out with hmSEEKER.

### Publication Abstract
Protein arginine methyltransferase 1 (PRMT1) is overexpressed in various human cancers and linked to poor response to chemotherapy. Various PRMT1 inhibitors are currently under development; yet, we do not fully understand the mechanisms underpinning PRMT1 involvement in tumorigenesis and chemoresistance. Using mass spectrometry-based proteomics, we identified PRMT1 as regulator of arginine methylation in ovarian cancer cells&#xa0;treated with cisplatin. We showed that DNA-dependent protein kinase (DNA-PK) binds to and phosphorylates PRMT1 in response to cisplatin, inducing its chromatin recruitment and redirecting its enzymatic activity toward Arg3 of histone H4 (H4R3). On chromatin, the DNA-PK/PRMT1 axis induces senescence-associated secretory phenotype through H4R3me2a deposition at pro-inflammatory gene promoters. Finally, PRMT1 inhibition reduces the clonogenic growth of cancer cells exposed to low doses of cisplatin, sensitizing them to apoptosis. While unravelling the role of PRMT1 in response to genotoxic agents, our findings indicate the possibility of targeting PRMT1 to overcome chemoresistance in cancer.

### Keywords
Silac, Heavy methyl silac, Methylation

### Affiliations
European Institute of Oncology IRCCS, 20139 Milan, Italy
European Institute of Oncology IRCSS, Department of Experimental Oncology, Milan, Italy

### Submitter
Enrico Massignani

### Lab Head
Dr Tiziana Bonaldi
European Institute of Oncology IRCCS, 20139 Milan, Italy


