### Accession
PXD022469

### Title
Proteome-wide evaluation of plasma depletion methods vis-à-vis fractionation techniques

### Description
Human plasma is one of the most widely used tissues in clinical analysis, and plasma-based biomarkers are used for monitoring patient health status and/or response to medical treatment to avoid unnecessary invasive biopsy. Data driven plasma proteomics has suffered from a lack of throughput and detection sensitivity, largely due to the complexity of the plasma proteome, and in particular the enormous quantitative dynamic range, estimated to be between 9-13 orders of magnitude between the lowest and highest abundance protein. A major challenge is to identify workflows which can achieve depth of plasma proteome coverage, while minimizing the complexity of sample workup and maximizing sample throughput. In this study, we have performed intensive depletion of high abundant plasma proteins or enrichment of low abundant proteins using Hu6, Hu14 and ProteoMiner followed by SDS PAGE and C18 prefractionation techniques. We compared the performance of each of these fractionation approaches to identify the method which satisfies requirements for analysis of clinical samples, and to include good plasma proteome coverage in combination with reasonable sample output. In this study, we have reported that gel-based prefractionation approaches can replace expensive and time-consuming chromatographic separation, thereby significantly accelerating analysis time. In addition, we showed that a variety of methodologies can achieve similarly high proteome coverage, allowing flexibility depending on project specific needs. These considerations are important in the effort to accelerate plasma proteomics research so as to provide efficient, reliable and accurate diagnoses for the population as a whole.

### Sample Protocol
Depletion of high abundant proteins using immunoaffinity-based Columns: Human 6 (HU6) and Human 14 (HU14) columns Depletion of 300 µl of crude plasma using antibody-based immunoaffinity columns HU6 (4.6 x 50 mm, Agilent, CA) and HU14 (4.6 x 100 mm, Agilent CA,). The HU6 column eliminates approximately 85-90% of total plasma protein, by binding the six high abundance plasma proteins (albumin, IgG, IgA, transferrin, haptoglobin, and antitrypsin) whereas the Hu14 column eliminates approximately 94% of total plasma protein and contains antibodies to the following fourteen high abundance plasma proteins (albumin, IgG, antitrypsin, IgA, transferrin, haptoglobin, fibrinogen, alpha2-macroglobulin, alpha1-acid glycoprotein, IgM, apolipoprotein AI, apolipoprotein AII, complement C3 and transthyretin). Plasma was diluted and filtered using spin filters (Corning Costar Spin-X centrifuge tube filter, 0.45 µm Cellulose Acetate, MERCK, Germany) before chromatography to remove any particulates. Chromatography and fraction collection were performed on an Agilent 1290 UHPLC system (Agilent, Santa Clara, CA). Binding buffer (A) and elution buffer (B), were commercial proprietary products and the LC methods used were based on manufacturer’s instructions (Agilent, Santa Clara, CA).  Highly abundant proteins were retained on the columns while buffer A was in use, and the flow-through proteins represented the low abundance protein fraction. The high abundance proteins were then eluted with buffer B. Only the low abundance protein fractions were processed for further fractionation and LC-MS/MS analysis.  Enrichment of low abundant proteins using bead-based depletion We sought to compare high abundance protein depletion against a low abundance protein enrichment approach, using beads bound to complex mixtures of combinatorial peptide ligands (ProteoMiner Beads; BIO-RAD, Berkeley, CA). This approach reduces the dynamic range of plasma proteins, since low and high abundance proteins have a similar chance of being bound to their respective ligands, which are distributed in similar proportions. Excess high abundance proteins are washed away, once they have reached their ligand saturation point, whereas low abundance proteins may never saturate their ligand, and are enriched within the sample. To apply this method, 300 µl of plasma was used to enrich low abundance proteins following the manufacturer’s instructions from the ProteoMiner protein-enrichment kit (BIO-RAD, Berkeley, CA). Both bound and unbound (flow-through) fractions were processed for further fractionation and LC-MS/MS analysis.  Depleted plasma proteins from the above approaches were concentrated, and buffer exchanged to 50 mM ammonium bicarbonate back to the original volume (300 µl) using Amicon 3kDa concentrators (Millipore, Billerica, MA). Protein concentrations of depleted plasma were determined by use of a nanolitre scale spectrophotometer (DeNovix DS-11+, POCD, Sydney, Australia) used in microvolume mode with 2 µl of each 3kDa concentrated sample, 280 nm wavelength, and using the extinction coefficient of BSA (E1% (g/100mL) = 6.67). Following absorbance reading, the samples were retrieved from the DeNovix sample surface to minimize total protein loss. The step was repeated three times and mean values used as the final sample concentration. To compare the protein profile across all methods, 50 µg of concentrated plasma from all methods were used as the starting point for further fractionation approaches.

### Data Protocol
Computational Analysis Initially, the data processing of the raw files was performed on MaxQuant (version 1.6.10.43) (20) using the Andromeda search engine (21). For profiling and quantitation, we used the filter of ≥2 unique peptides per protein. The UniProt Homo sapiens database (Proteome ID UP000005640) was used, in combination with reverse decoy protein sequences (used for protein identification false discovery rate estimation) and frequent contaminant sequences. The default value of precursor mass tolerance (±0.07 Da) for the first search and (±0.006 Da) for the main search were applied. Mass tolerance for matching peaks to theoretical MSMS ion series was five ppm. Protein identification false discovery rate (FDR) was set to <1%, to ensure only high-confidence identifications for the peptide to spectra match, protein, and site decoy fractions. Enzyme specificity was set to trypsin, with a maximum of two missed cleavages. Searches included variable modifications of protein N-terminal acetylation, methionine oxidation, pyroGlu and pyroGln and fixed modification of carbamidomethylation of cysteines. The maximum number of modifications per peptide was set at six, minimum peptide length was six amino acids. To validate and transfer identifications among adjacent fractions and across different replicates, the ‘match between runs’ (MBR) option in MaxQuant was enabled, with a retention time window of 0.7 min and an alignment time window of 20 min.  All the commonly identified proteins in all methods were quantified for peak area abundance within the MaxQuant environment using the MaxLFQ (label-free quantification) algorithm (22). The MaxLFQ algorithm combines the measured intensities from all matched peptides combined into proteins for final abundance data (23). Finally, the MaxLFQ values obtained from MaxQuant for each protein were statically analyzed in the Perseus environment. This workflow of Andromeda →MBR →MaxLFQ → Perseus is frequently used in studies using data-dependent quantitative proteomics (24).   All the raw files were also processed through the Trans-Proteomic Pipeline (TPP) version 5.1.0 released on 2017-11-03. This software takes into account additional factors, including the number of sibling experiments identifying the same peptide ions, the number of replicate ion identifications, sibling ions, and sibling modification states. TPP uses the iProphet algorithm to determine the false discovery rate (FDR) for protein identification. To perform the TPP analysis, the LC-MS/MS raw files were first converted to open format mzML files using the MSconvertGUI with the default parameters. The Uniprot database and sequence input were similar to the MaxQuant search. The 384 LC-MS/MS raw files were processed using three search engines; X! Tandem (with the k-score plug-in) (25), Mascot and Comet, with all parameters kept similar. The PeptideProphet and ProteinProphet algorithms were employed to compute the probability scores for peptide and protein identification. The accurate mass model in PeptideProphet was used for high confidence peptide identifications (26) as well as a requirement that each protein be identified with a minimum of two peptides, each with a peptide probability score >99%. All three search engine results were merged and validated using iProphet (27). An iProphet probability of >0.99 was used as the cutoff for final protein identification (28). The total sum mapping resulted in the identification of 10 million spectra, but we used only the common peptides representing only the highest confidence data, and without a search engine bias.

### Publication Abstract
Human plasma is one of the most widely used tissues in clinical analysis, and plasma-based biomarkers are used for monitoring patient health status and/or response to medical treatment to avoid unnecessary invasive biopsy. Data-driven plasma proteomics has suffered from a lack of throughput and detection sensitivity, largely due to the complexity of the plasma proteome and in particular the enormous quantitative dynamic range, estimated to be between 9 and 13 orders of magnitude between the lowest and the highest abundance protein. A major challenge is to identify workflows that can achieve depth of plasma proteome coverage while minimizing the complexity of the sample workup and maximizing the sample throughput. In this study, we have performed intensive depletion of high-abundant plasma proteins or enrichment of low-abundant proteins using the Agilent multiple affinity removal liquid chromatography (LC) column-Human 6 (Hu6), the Agilent multiple affinity removal LC column-Human 14 (Hu14), and ProteoMiner followed by sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS PAGE) and C18 prefractionation techniques. We compared the performance of each of these fractionation approaches to identify the method that satisfies requirements for analysis of clinical samples and to include good plasma proteome coverage in combination with reasonable sample output. In this study, we report that one-dimensional (1D) gel-based prefractionation allows parallel sample processing and no loss of proteome coverage, compared with serial chromatographic separation, and significantly accelerates analysis time, particularly important for large clinical projects. Furthermore, we show that a variety of methodologies can achieve similarly high plasma proteome coverage, allowing flexibility in method selection based on project-specific needs. These considerations are important in the effort to accelerate plasma proteomics research so as to provide efficient, reliable, and accurate diagnoses, population-based health screening, clinical research studies, and other clinical work.

### Keywords
Plasma, Prefractionation techniques, Biomarkers, Electrophoresis, Proteome, Label-free quantitation, Mass spectrometry, Chromatography

### Affiliations
Centre for Healthy Brain Ageing, School of Psychiatry, University of New South Wales, Sydney, NSW, 2052, Australia
University of New South Wales

### Submitter
Gurjeet Kaur

### Lab Head
Dr Gurjeet Kaur
Centre for Healthy Brain Ageing, School of Psychiatry, University of New South Wales, Sydney, NSW, 2052, Australia


