### Accession
PXD033305

### Title
Regulatory dynamics of gene expression in the developing male gametophyte of Arabidopsis

### Description
Sexual reproduction in angiosperms requires the production and delivery of two male gametes by a three-celled haploid male gametophyte. This demands synchronised gene expression in a short developmental window to ensure double fertilization and seed set. While transcriptomic changes in developing pollen are known for Arabidopsis, no studies have integrated RNA and proteomic data in this model. Further, the role of alternative splicing has not been fully addressed, yet post-transcriptional and post-translational regulation may have a key role in gene expression dynamics during microgametogenesis. We have refined and substantially updated global transcriptomic and proteomic changes in developing pollen for two Arabidopsis accessions. Despite the superiority of RNA-seq over microarray-based platforms, we demonstrate high reproducibility and comparability. We identify thousands of long non-coding RNAs as potential regulators of pollen development, hundreds of changes in alternative splicing and provide insight into mRNA translation rate and storage in developing pollen. Our analysis delivers an integrated perspective of gene expression dynamics in developing Arabidopsis pollen and a foundation for studying the role of alternative splicing in this model.

### Sample Protocol
Total proteins were isolated from UNM, BCP, TCP and MPG using the TRI Reagent solution (Sigma-Aldrich, product No. T9424) following the manufacturer’s instructions. Individual protein samples were processed by filter-aided sample preparation (FASP) method with some modifications. The samples were mixed with 8M UA buffer (8M urea in 100 mM Tris-HCl, pH 8.5), loaded onto the Microcon device with MWCO 30 kDa (Merck Millipore) and centrifuged at 7,000× g for 30 min at 20°C. The retained proteins were washed (all centrifugation steps after sample loading done at 14,000× g) with 200 μL UA buffer. The final protein concentrates kept in the Microcon device were mixed with 100 μL of UA buffer containing 50 mM iodoacetamide and incubated in the dark for 20 min. After the next centrifugation step, the samples were washed three times with 100 μL UA buffer and three times with 100 μL of 50 mM NaHCO3. Trypsin (sequencing grade, Promega) was added onto the filter and the mixture was incubated for 18 h at 37°C (enzyme:protein ratio 1:100). The tryptic peptides were finally eluted by centrifugation followed by two additional elutions with 50 μL of 50mM NaHCO3. Directly after FASP peptides were extracted into LC-MS vials by 2.5% formic acid (FA) in 50% acetonitrile (can) and 100% ACN with addition of polyethylene glycol (20,000; final concentration 0.001%) and concentrated in a SpeedVac concentrator (Thermo Fisher Scientific) prior LC-MS analyses. LC-MS/MS analyses of all peptide mixtures were done using RSLCnano system (SRD-3400, NCS-3500RS CAP, WPS-3000 TPL RS) connected to Orbitrap Elite hybrid spectrometer (Thermo Fisher Scientific). Prior to LC separation, tryptic digests were online concentrated and desalted using trapping column (100 μm × 30 mm) filled with 3.5-μm X-Bridge BEH 130 C18 sorbent (Waters). After washing of trapping column with 0.1% FA, the peptides were eluted (flow 300 nl/min) from the trapping column onto an analytical column (Acclaim Pepmap100 C18, 3 µm particles, 75 μm × 500 mm; Thermo Fisher Scientific) by 100 min nonlinear gradient program (1-56% of mobile phase B; mobile phase A: 0.1% FA in water; mobile phase B: 0.1% FA in 80% ACN). Equilibration of the trapping column and the column was done prior to sample injection to sample loop. The analytical column outlet was directly connected to the Digital PicoView 550 (New Objective) ion source with sheath gas option and SilicaTip emitter (New Objective; FS360-20-15-N-20-C12) utilization. ABIRD (Active Background Ion Reduction Device, ESI Source Solutions) was installed. MS data were acquired in a data-dependent strategy selecting up to top 10 precursors based on precursor abundance in the survey scan (350-2000 m/z). The resolution of the survey scan was 60 000 (400 m/z) with a target value of 1×106 ions, one microscan and maximum injection time of 200 ms. HCD MS/MS (32% relative fragmentation energy) spectra were acquired with a target value of 50 000 and resolution of 15 000 (400 m/z). The maximum injection time for MS/MS was 500 ms. Dynamic exclusion was enabled for 45 s after one MS/MS spectra acquisition and early expiration was disabled. The isolation window for MS/MS fragmentation was set to 2 m/z.

### Data Protocol
The analysis of the mass spectrometric RAW data files was carried out using the MaxQuant software (version 1.6.0.16). MS/MS ion searches were done against modified cRAP database (106) containing protein contaminants like keratin, trypsin etc., and UniProtKB protein database for Arabidopsis thaliana (ftp://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase /reference_proteomes/Eukaryota/UP000006548/UP000006548_3702.fasta.gz; downloaded 22.11.2017, number of protein sequences 27,567). Default precursor and fragment masses tolerances were used with software MS data recalibration enabled. Oxidation of methionine and proline, deamidation (N, Q) and acetylation (protein N-terminus) as optional modification, carbamidomethylati©(C) as fixed modification and one enzyme miss cleavage were set. Peptides and proteins with FDR threshold <0.01 and proteins having at least one unique or razor peptide were considered only. Match between runs among all analyses and second peptide identification features of MaxQuant were enabled.

### Publication Abstract
None

### Keywords
Lc-ms, Ler-0, Microgametogenesis, Arabidopsis, Male gametophyte, Rna-seq, Col-0, Proteome

### Affiliations
Proteomics Core Facility and Research Group Proteomics
CEITEC-MU
Masaryk University
Brno, Czech Republic
Proteomics Core Facility and Research Group Proteomics, CEITEC-MU, Masaryk University, Brno, Czech Republic

### Submitter
David Potesil

### Lab Head
Dr Zbynek Zdrahal
Proteomics Core Facility and Research Group Proteomics, CEITEC-MU, Masaryk University, Brno, Czech Republic


