### Accession
PXD012954

### Title
Targeting the scaffolding role of LSD1 (KDM1A) poises acute myeloid leukemia cells for Retinoic Acid induced differentiation

### Description
The histone demethylase LSD1 is deregulated in several tumors, including leukemias, providing the rationale for the clinical use of LSD1 inhibitors. In acute promyelocytic leukemia (APL), pharmacological doses of retinoic acid (RA) induce differentiation of APL cells through degradation of the PML-RAR oncogene. APL cells are resistant to LSD1 inhibition or knock-out, but LSD1 inhibition sensitizes them to physiological doses of RA without altering the stability of PML-RAR, and extends survival of leukemic mice upon RA treatment. Non-enzymatic activities of LSD1 are essential to block differentiation of leukemic cells, while the combination of LSD1 inhibitors (or LSD1 knock-out) with low doses of RA releases a differentiation-associated gene expression program, not strictly dependent on changes in histone H3K4 methylation (known substrate of LSD1). An integrated proteomic/epigenomic/mutational analysis showed that LSD1 inhibitors alter the recruitment of LSD1-containing complexes to chromatin through inhibition of the interaction between LSD1 and GFI1, a relevant transcription factor in hematopoiesis.

### Sample Protocol
LSD1 co-IP for mass-spectrometry analysis of protein-protein interactions:  NB4 nucleosolic fraction was quantified and diluted in the IP buffer (10 mM Tris HCl pH 7.6, 150 mM NaCl, 0.2% NP-40) supplemented with 1X Roche Protease Inhibitors and 0.5 mM PMSF, in order to obtain a concentration of 2 mg/ml. Preclearing of the lysate was carried out by 1hr incubation with protein G magnetic beads. The lysate was  re-quantified and diluted with the IP buffer supplemented with protease inhibitors up to a concentration of 1.3 mg/ml. An aliquot of the Input was collected before the addition of the antibody. The samples were incubated with the antibody overnight on a rotating wheel at 4Â°C. For the acquisition of the basal LSD1 interactome, 120 fold molar excess of LSD1 blocking peptide was incubated together with the antibody as negative control. The peptide was added to the light channel in the forward experiment and to the heavy channel in the reverse one. The following day, 50-100 Âµl of Dynabeads-protein G pre-equilibrated in PBS and supplemented with 0.5% BSA were added to the extracts and incubated for 3 hours on a rotating wheel at 4Â°C. Beads were then washed 3 times with IP buffer and once with Washing Buffer IP (10 mM Tris HCl pH 7.6, 250 mM NaCl, 0.2% NP-40). Then, bound material was eluted by incubation with LSD Sample Buffer (NuPAGE), supplemented with 100 mM DTT, at 95Â°C for 5 minutes. Eluted samples were loaded on SDS-PAGE for protein separation and subsequent analysis. In gel digestion of gel-separated proteins was carried out with Trypsin. After digestion and extraction from the gel pieces, the digested peptides were desalted and concentrated by reversed-phase chromatography onto C18 Stage Tip micro-columns (Rappsilber, J. et al., 2007). Peptides were then eluted from the stage tips with buffer B (80% ACN, 0.5% acetic acid), lyophilized, re-suspended in 0.1% Formic Acid and subjected to LC-MS/MS analysis.   MS-based histone PTM profiling:  Cells were homogenized in lysis buffer (10% sucrose, 0.5 mM EGTA, 60 mM KCl, 15 mM NaCl, 15 mM HEPES, 0.5 mM PMSF, 5 Âµg/ml Aprotonin, 5 Âµg/ml Leupeptin, 1 mM DTT, 5 mM NaButirrate, 5 mM NaF, 30 Âµg/ml Spermine, 30 Âµg/ml Spermidine and 0.5% Triton X-100) and nuclei were separated from cytoplasm by centrifugation on sucrose cushions for 30 minutes at 3750 rpm. Then, histones were extracted through 0.4 N hydrochloric acid for 5 hours at 4Â°C. Extracted histones were lyophilized, resuspended in milliQ water and quantified by Bradford assay. 5 Âµg of histones were in-solution digested prior to LC-MS/MS analysis through the hybrid chemical labelling â€œPro-PICâ€� method. This approach is based on an initial conversion of free lysines to their propionylated forms under mild aqueous conditions followed by trypsin digestion and labelling of new peptide N-termini with phenyl isocyanate (PIC). The digested peptides were desalted and concentrated by reversed-phase chromatography onto micro-column C18 Stage Tips. Peptides were then eluted from the stage tips with Elution Buffer (60% ACN, 0.1% trifluoroacetic acid), lyophilized, re-suspended in 1% trifluoroacetic acid and subjected to LC-MS/MS analysis. The samples were analysed onto a Hybrid Quadrupole-Orbitrap Q Exactive Mass Spectrometer, upon separation along a 100-minutes gradient from 0% to 40% Acetonitrile (ACN) using a Reversed Phase-UHPLC system. MaxQuant software v.1.6.0.1 is used for the analysis of MS data, for both protein and peptides identification including as variable modifications mono-, di- and tri-methyl lysine, mono- and dimethyl arginine and lysine acetylation. The Uniprot HUMAN histones 1502 database was used for histone peptide identification. Enzyme specificity was set to ArgC, since the propionylation of lysine residues allows the trypsin cutting only at the C-terminal of arginine residues. A maximum of 3 missed cleavages were permitted, and the minimum peptide length was fixed at 6 amino acids. PIC at the N-terminal of each peptide was set as a fixed modification. Histone modifications were quantified by label free approach, in particular by calculating the area under the curve (AUC) of each modified histone peptide and, then extrapolating the percentage relative abundance (%RA), which corresponds to the ratio of the AUC of each individual modified peptide over the sum of the AUC of all modified isoforms of the same peptide. To estimate the possible changes upon LSD1 inhibition/depletion, we also calculated the ratio of the relative abundance (%RA) of the treated and KO samples over the same value in the untreated and WT samples, respectively.

### Data Protocol
Acquired Raw MS data were analyzed with the integrated MaxQuant software v.1.6.0.1, using the Andromeda search engine. The February 2018 version (UniProt Release 2018_02) of the Uniprot sequence was used for peptide identification. Enzyme specificity was set to Trypsin/P, meaning that trypsin cleavage occurs also in the presence of proline, after lysine or arginine residues. In MaxQuant, the estimated false discovery rate (FDR) of all peptide identifications was set to a maximum of 1%. A maximum of 3 missed cleavages was permitted, and the minimum peptide length was fixed at 7 amino acids. Carbamidomethylation of cysteine was set as a fixed modification. â€˜Requantifyâ€™ and â€˜Match between runsâ€™ were enabled. The uploaded data includes the input of the dynamic IP, which was excluded from following steps in the analysis, and data from another Knock-out sample (KO1), which was also not cited in the publication.

### Publication Abstract
The histone demethylase LSD1 is deregulated in several tumors, including leukemias, providing the rationale for the clinical use of LSD1 inhibitors. In acute promyelocytic leukemia (APL), pharmacological doses of retinoic acid (RA) induce differentiation of APL cells, triggering degradation of the PML-RAR oncogene. APL cells are resistant to LSD1 inhibition or knockout, but targeting LSD1 sensitizes them to physiological doses of RA without altering of PML-RAR levels, and extends survival of leukemic mice upon RA treatment. The combination of RA with LSD1 inhibition (or knockout) is also effective in other non-APL, acute myeloid leukemia (AML) cells. Nonenzymatic activities of LSD1 are essential to block differentiation, while RA with targeting of LSD1 releases a differentiation gene expression program, not strictly dependent on changes in histone H3K4 methylation. Integration of proteomic/epigenomic/mutational studies showed that LSD1 inhibitors alter the recruitment of LSD1-containing complexes to chromatin, inhibiting the interaction between LSD1 and the transcription factor GFI1.

### Keywords
Lsd1, Protein-protein interaction, Lsd1 inhibitors, Acute myeloid leukemia

### Affiliations
European Institute of Oncology IRCSS, Department of Experimental Oncology, Milan, Italy
Department of Experimental Oncology, IEO, European Institute of Oncology IRCCS, Milan, Italy

### Submitter
Enrico Massignani

### Lab Head
Dr Tiziana Bonaldi
Department of Experimental Oncology, IEO, European Institute of Oncology IRCCS, Milan, Italy


