### Accession
PXD004654

### Title
Temperature-dependent regulation of the Ochrobactrum anthropi proteome

### Description
Ochrobactrum anthropi is a gram-negative rod belonging to the Brucellaceae family, able to colonize a variety of environments, and actually reported as a human opportunistic pathogen. Despite its low virulence, the bacterium causes a growing number of hospital-acquired infections mainly, but not exclusively, in immunocompromised patients. The aim of this study was to obtain an overview of the global proteome changes occurring in O. anthropi in response to different growth temperatures, in order to achieve a major understanding of the mechanisms by which the bacterium adapts to different habitats and to identify some potential virulence factors. Combined quantitative mass spectrometry-based proteomics and bioinformatics approaches were carried out on two O. anthropi strains grown at temperatures miming soil/plants habitat (25 C) and human host environment (37 C), respectively. Proteomic analysis led to the identification of over 150 differentially expressed proteins in both strains, out of over 1200 total protein identifications. Among them, proteins responsible for heat shock response (DnaK, GrpE), motility (FliC, FlgG, FlgE), and putative virulence factors (TolB) were identified. The study represents the first quantitative proteomic analysis of O. anthropi performed by high-resolution quantitative mass spectrometry.

### Sample Protocol
O. anthropi strains were cultured onto Columbia blood agar plates at 37 C or at 25 C in an atmosphere enriched with 5% CO2. After 48 h, bacteria were precipitated and subsequently resuspended and lysed. Proteins were reduced, alkylated and digested by trypsin. The resulting tryptic peptides were purified by SPE C18 cartridges. Concentrated eluates from each condition were reconstituted and labelled by either regular formaldehyde (CH2O; “light form”, “L”) or deuterated formaldehyde (CD2H; “heavy form”, “H”) in presence of NaBH3CN. A technical replicate using reverse labelling was used for each biological replicate. Thus, a total of 12 H:L mixtures (3 biological replicates,  two strains, two technical replicates per condition) was subjected to mass spectrometric analysis. Before nanoLC-MS/MS analysis, each peptide mixture was fractionated (n=6) by SCX StageTips.

### Data Protocol
Mass spectrometry analysis was performed on a Q-Exactive Hybrid Quadrupole-Orbitrap Mass Spectrometer coupled online to an Easy nano-LC1000 system (Thermo Fisher Scientific, Germany) using a 55 min linear gradient and a TOP12 data-dependent method. Raw data were processed by Proteome Discoverer 1.4 (Thermo Fisher Scientific, Germany) using Sequest as search engine. Data were searched against the protein sequence database of O. anthropi downloaded from the Swiss-Prot database (January 2015, 27935 sequences). A FDR of 5% at the peptide level was estimated using the target-decoy strategy. A minimum of two peptide identifications per protein was required. Besides, only protein hits identified in at least 2 biological replicates were considered for quantitative analysis. For protein quantification, technical replicates were combined in pairs after transforming the H:L peptide ratios of the reverse experiment into their reciprocal.

### Publication Abstract
Ochrobactrum anthropi is a Gram-negative rod belonging to the Brucellaceae family, able to colonize a variety of environments, and actually reported as a human opportunistic pathogen. Despite its low virulence, the bacterium causes a growing number of hospital-acquired infections mainly, but not exclusively, in immunocompromised patients. The aim of this study was to obtain an overview of the global proteome changes occurring in O. anthropi in response to different growth temperatures, in order to achieve a major understanding of the mechanisms by which the bacterium adapts to different habitats and to identify some potential virulence factors. Combined quantitative mass spectrometry-based proteomics and bioinformatics approaches were carried out on two O. anthropi strains grown at temperatures miming soil/plants habitat (25&#xb0;C) and human host environment (37&#xb0;C), respectively. Proteomic analysis led to the identification of over 150 differentially expressed proteins in both strains, out of over 1200 total protein identifications. Among them, proteins responsible for heat shock response (DnaK, GrpE), motility (FliC, FlgG, FlgE), and putative virulence factors (TolB) were identified. The study represents the first quantitative proteomic analysis of O. anthropi performed by high-resolution quantitative mass spectrometry.

### Keywords
Bacteria / heat adaptation / ochrobactrum anthropi / quantitative proteomics / stable isotope dimethyl labeling

### Affiliations
University of Catanzaro
Department of Experimental & Clinical Medicine Building G, Level 3 “Magna Græcia” University of Catanzaro Campus “S. Venuta”, Viale Europa, Loc. Germaneto 88100 Catanzaro- Italy

### Submitter
Mariaconcetta Varano

### Lab Head
Dr Marco Gaspari
Department of Experimental & Clinical Medicine Building G, Level 3 “Magna Græcia” University of Catanzaro Campus “S. Venuta”, Viale Europa, Loc. Germaneto 88100 Catanzaro- Italy


