### Accession
PXD022477

### Title
Proteomic study identified several proteins involved in cellular processes in recurrent otitis media

### Description
Recurrent otitis (ROM) is a pathological conditions in which a children have at least three episodes of acute otitis media over a course of 6 months. In this study we focused in proteins dysregulated between ROM and adenotonsillar hypertrophy (ATH). Tissue sample of seven ROM and seven ATH were obtained and submitted to two-dimensional gel electrophoresis (2-DE) and mass spectrometry (MS) for protein identification and western-blotting for data validation. We identified 23 significantly (p<0.05) protein spots. The expression of two proteins were validated by western-blotting. The 2-DE coupled with MS approach identified several proteins involved in cellular and metabolic processes. Further study are needed to better understanding the role of this proteins in ROM.

### Sample Protocol
A total of 14 patients were examined (median age  8 years; 8 males and 6 females),  7 affected by adenotonsillar hypertrophy (ATH) and 7 presenting recurrent otitis media (ROM). ROM patients were defined as having at least three episodes of acute otitis media (AOM) in a period of 6 months or four AOM in 12 months,while ATH patients show an abnormal growth of pharyngeal or palatine tonsils.Patients undergoes either adenoidal or adenotonsillar surgery for improving breathing or reducing recurrent AOM, but only adenoidal tissue was used for proteomic analysis.  Samples washed from blood of ROM and ATH specimens (300 mg each) were manually homogenized in 1.5 ml of dissolution TUC buffer [7 M urea, 2 M thiourea, 4% CHAPS, 40 mM Tris, 65 mM DTT and 0.24% Bio-Lyte (3–10)] with a protease inhibitor mix (2 mM Phenylmethylsulfonyl fluoride (PMSF), 1 mM benzamidine, 1 mM Ethylenediaminetetraacetic acid (EDTA), 1 mM Sodium fluoride (NaF)). After vortex, the solutions were centrifuged at 10,000 × g at 4°C for 30 min and the protein content of the supernatant was determined using the Bradford assay.. For 2-DE analysis, 300 μg of proteins from each sample were used. ReadyStrip™ 3-10 NL 17-cm immobilized pH gradient (IPG) strips were rehydrated in a dissolution buffer at 50 V for 12 h at 20 °C, and isoelectric focusing (IEF) was performed in a PROTEAN IEF Cell (Bio-Rad Laboratories, Inc., Hercules, CA, USA). After the IEF, serial incubations were performed: first, the IPG strips were equilibrated for 10 min in an equilibration buffer (6 M urea, 2% SDS, 50 mM Tris-HCl (pH 8.8), 30% glycerol) and then in another equilibration buffer containing 4% iodoacetamide. For the second dimension, the equilibrated IPG strips were transferred to a 12% polyacrylamide gel. After electrophoresis, gels were fixed in 40% methanol and 10% acetic acid overnight, and then stained for 6h with Flamingo stain. These gels, after Flamingo destaining, were stained for 48h with colloidal Coomassie Brilliant Blue. 2-DE gels were scanned with a Molecular Imager PharosFX System. Double experimental replicates were performed per sample. For all gels, molecular weights were determined by comparison with Precision Plus Protein Pre-stained Standards (Bio-Rad Laboratories, Inc., Hercules, CA, USA), covering a range from 10 to 250 kDa and analyzed using the Proteomweaver 4.0 software (both from Bio-Rad Laboratories, Inc., Hercules, CA, USA). Spots obtain after excision from 2-DE gels, were washed four times with 50 mM NH4HCO3 and acetonitrile (ACN; Sigma-Aldrich, St. Louis, MO, USA) alternatively, and dried under vacuum in a SpeedVac system. For gel spot digestion, three microliters of 12.5 ng/µL sequencing grade modified trypsin (Promega, Madison, WI, USA) in 50 mM NH4HCO3 were added, and samples were digested overnight at 37 °C. Finally, peptides extraction was made with three changes extraction by 50% ACN/0.1% formic acid (FA; Fluka, Ammerbuch, Germany), peptide mixtures were dried under vacuum and stored at −20 °C, until mass spectrometry (MS) analysis was performed. Samples were dissolved in 12 µL of 3% ACN/0.1% FA and 4 microliters of each sample were analyzed by LC-MS/MS on a LTQ-Orbitrap XL mass spectrometer (Thermo Fisher Scientific, Waltham, MA, USA) coupled to nano-HPLC Ultimate 3000 (Dionex – Thermo Fisher Scientific). Once the obtained peptides were separated in a 10 cm pico-frit column (75 μm ID, 15 μm Tip;  New Objective) packed in-house with C18 material  (Aeris Peptide 3.6 µm XB-C18, Phenomenex). H2O/FA 0.1% and ACN/FA 0.1% were used as eluents A and B, respectively and peptides were analyzed at a flow rate of 0.25 μL/min using a linear gradient of eluent B from 3% to 40% in 20 min. A Data Dependent Acqusition (DDA) was used: a full scan between 300 and 1700 Da was performed at high resolution (60000) on the Orbitrap. The ten most intense ions were then selected for CID fragmentation and acquisition of MS/MS data in low resolution in the linear ion trap.

### Data Protocol
Quantification of Spot Levels 2-DE image analysis was performed using the Proteomweaver 4.0 software. The analysis process was carried out by matching all gels from seven ROM and seven ATH. The Proteomweaver 4.0 algorithm matched all of the gels to find quantitative differences. Differences were considered significant when the ratio of the mean percentage relative volume (%V) (%V = V(single spot)/V(total spot)) was ±1.5-fold and satisfied the non-parametric Wilcoxon test (P < 0.05). Fold change was calculated as the ratio between the mean %V of ROM  and ATH.  MS data analysis Raw data files were analyzed with the software package Proteome Discoverer 1.4 (Thermo Fisher Scientific) and searched with Mascot Search Engine (version 2.2.4, Matrix Science, London, UK). Spectra were searched against the human section of the Uniprot database (version July 2018) using the following parameters: enzyme specificity was set to trypsin with 1 missed cleavage allowed, precursor and fragment ions tolerance were 10 ppm and 0.6 Da, respectively. Carbamidomethylcysteine and oxidation of methionine were set as fixed modification and variable modification, respectively. Proteins were considered as positive hits if for each protein at least 3 unique peptides were identified with high confidence (FDR < 1%).  PANTHER Analysis Proteins identified by MS were analysed by PANTHER 11.0 (Protein Analysis Through Evolutionary Relationships; http://www.pantherdb.org) and Gene Ontology (http://amigo.geneontology.org/rte). Proteins were then classified according to their involvement in biological processes, molecular function, protein class, and pathways. Since the majority of the proteins identified participated in multiple processes, only the most relevant ones were reported.  Statistical Analysis Statistical analyses were carried out with the non-parametric Wilcoxon signed-rank test for matched samples for both 2-DE and Western blot data. P < 0.05 was considered to indicate a statistically significant difference. All analyses were conducted with Graph Pad Prism 5 for Windows.

### Publication Abstract
None

### Keywords
Recurrent otitis adenotonsillar hypertrophy, Lc-ms/ms, 2-de

### Affiliations
Department of Biomedical Sciences, University of Padova, Italy
University of Padova

### Submitter
Giorgio  Arrigoni

### Lab Head
Dr Giorgio Arrigoni
Department of Biomedical Sciences, University of Padova, Italy


