### Accession
PXD000225

### Title
Phosphoproteomics study on protein kinase C delta induced cell death

### Description
The proteolytic activation of protein kinase C delta generates a catalytic fragment called PKC delta-CF, which induces cell death. However, the mechanisms underlying PKC delta-CF-mediated cell death are largely unknown. On the basis of an engineering leukemic cell line with inducible expression of PKC delta-CF, here we employ SILAC-based quantitative phosphoproteomics to systematically and dynamically investigate the overall phosphorylation events during cell death triggered by PKC delta-CF expression. Totally, 3000 phosphorylation sites were analyzed. We combined SILAC for quantitation, strong-cation exchange chromatography and titanium dioxide chromatography for phosphopeptide enrichment, and high-accuracy mulistage MS. All MS/MS ion spectra were analyzed using Sequest version v.27, rev. 1146 (Thermo Fisher Scientific, San Jose, CA) which were incorporated into the Sorcerer engine version 4.0.4 build (Sage-N Research). Sequest were set up to search the ipi.HUMAN.v3.65 database (86379 entries) (ftp.ebi.ac.uk/pub/data bases/IPI/current) with its target-decoy database as a reversed complement assuming the semi-enzyme tryptic digestion allowed for three missed tryptic cleavages with full mass from 600 to 4600. The precursor ion tolerance was set to Â±10 p.p.m and MS2 ions tolerance was set to 1 Da. Searches parameters for non-phosphopeptides were allowed for a static modification of Carbamidomethyl cysteine (C, +57.021465) and dynamic modifications of oxidized methionine (M, +15.99492) on, SILAC-labeled lysine (K, 6C2N, +8.014199) on and SILAC-labeled arginine (R, 6C4N, +10.008269) with up to 4 total dynamic modifications and up to 3 of any particular dynamic modification. In addition, searches for phosphopeptides were allowed for dynamic modifications of phosphorylation serine (S), threonine (T), and tyrosine (Y) (+79.966331) with up to 6 total dynamic modifications and up to 4 of any particular dynamic modification. The ASCORE algorithm was selected to assess the possibility of phosphorylation sites. DATSelect 2.0.39 was used to filter and group the data files derived from SORCERER-SEQUEST. The filter thresholds were set to XCorr 2+, 3+, 4+: > 2.5, > 3.0, > 3.5 and DeltaCN > 0.08. For protein identification, two peptides were required with estimated false-discovery rates (FDR) < 1%. For phosphopeptide identification, only unique peptide was required with FDR <2%, whereas DeltaCN should be adjusted to >0.125 if the FDR for phosphopeptide assignments was >2%. Furthermore, additional filtering was used to remove those indubitably false phosphopeptides, including sequences that contained both heavy and light isotopic variants of arginine and lysine. Finally, the FDR fall within 1%.

### Sample Protocol
Not available

### Data Protocol
Not available

### Publication Abstract
The proteolytic activation of protein kinase C&#x3b4; (PKC&#x3b4;) generates a catalytic fragment called PKC&#x3b4;-CF, which induces cell death. However, the mechanisms underlying PKC&#x3b4;-CF-mediated cell death are largely unknown. On the basis of an engineering leukemic cell line with inducible expression of PKC&#x3b4;-CF, here we employ SILAC-based quantitative phosphoproteomics to systematically and dynamically investigate the overall phosphorylation events during cell death triggered by PKC&#x3b4;-CF expression. Totally, 3000 phosphorylation sites were analyzed. Considering the fact that early responses to PKC&#x3b4;-CF expression initiate cell death, we sought to identify pathways possibly related directly with PKC&#x3b4; by further analyzing the data set of phosphorylation events that occur in the initiation stage of cell death. Interacting analysis of this data set indicates that PKC&#x3b4;-CF triggers complicated networks to initiate cell death, and motif analysis and biochemistry verification reveal that several kinases in the downstream of PKC&#x3b4; conduct these networks. By analysis of the specific sequence motif of kinase-substrate, we also find 59 candidate substrates of PKC&#x3b4; from the up-regulated phosphopeptides, of which 12 were randomly selected for in vitro kinase assay and 9 were consequently verified as substrates of PKC&#x3b4;. To our greatest understanding, this study provides the most systematic analysis of phosphorylation events initiated by the cleaved activated PKC&#x3b4;, which would vastly extend the profound understanding of PKC&#x3b4;-directed signal pathways in cell death. The MS data have been deposited to the ProteomeXchange with identifier PXD000225.

### Keywords
Pkcdelta phosphoproteomics apoptosis

### Affiliations
Pathophysiology 

### Submitter
Li Xia

### Lab Head


