### Accession
PXD012174

### Title
The human phosphoproteome map based on PRIDE data

### Description
This project contains raw data, intermediate files and results used to create the PRIDE human phosphoproteome map. The map is based on joint reanalysis of 110 publicly available human datasets. All relevant datasets were retrieved from the PRIDE database, and after manual curation, only assays that employed dedicated phospho-enrichment sample preparation strategies (e. g. metal oxide affinity chromatography, anti-P-Tyr antibodies, etc.) were included. Raw files were jointly processed with MaxQuant computational platform using standard settings (see Data Processing Protocol). In total, the joint analysis allowed identification of 252,189 phosphosites at 1% peptide spectrum match false discovery rate (PSM FDR) (MQ search results available in ‘txt-100PTM’ folder), of which 121,896  passed the additional 1% site localization FDR threshold (MQ search results available in ‘txt-001PTM’ folder).

### Sample Protocol
Authors of the original studies deposited in the PRIDE database generated all the samples and mass spectrometry assays used in this analysis. Across all the studies, various techniques have been employed to prepare protein extracts and enrich the phosphopeptides. Detailed experimental protocols should be accessed under the relevant PXD identifier.  The following datasets were used in this analysis:  PXD004373 PXD002425 PXD001060 PXD000681 PXD004817 PXD003627 PXD002765 PXD001962 PXD003198 PXD006475 PXD001180 PXD004357 PXD004981 PXD001550 PXD000696 PXD000089 PXD000217 PXD003492 PXD004181 PXD004340 PXD000901 PXD002850 PXD000523 PXD002614 PXD006055 PXD001101 PXD003108 PXD000496 PXD002255 PXD003908 PXD002172 PXD001546 PXD001565 PXD000462 PXD000680 PXD002646 PXD002127 PXD000474 PXD001374 PXD004940 PXD000612 PXD003523 PXD001333 PXD001559 PXD001305 PXD004452 PXD005181 PXD001196 PXD005366 PXD000293 PXD002704 PXD001603 PXD001381 PXD004252 PXD001186 PXD001094 PXD002871 PXD000341 PXD001428 PXD004447 PXD002055 PXD004415 PXD002023 PXD000185 PXD001129 PXD002839 PXD002990 PXD000985 PXD000836 PXD002394 PXD000599 PXD003660 PXD001812 PXD000275 PXD002072 PXD000472 PXD001114 PXD002735 PXD001121 PXD001222 PXD001739 PXD000750 PXD004442 PXD001543 PXD000451 PXD000242 PXD001189 PXD001170 PXD003822 PXD000597 PXD005214 PXD003115 PXD004945 PXD000970 PXD001253 PXD000497 PXD003531 PXD002286 PXD000238 PXD001115 PXD002635 PXD003708 PXD002496 PXD000222 PXD003712 PXD002135 PXD003529 PXD000225 PXD001563 PXD001560

### Data Protocol
Raw files were processed with MaxQuant (MQ) computational platform using default settings. Specifically, files were assigned to 17 parameter groups depending on the experimental design, instrument type, and methods used in individual studies (details of all parameter groups can be found in the mqpar.xml file). All of the data (6,801 raw files) was searched together using the MQ integrated Andromeda search engine, against the UniProt Human Reference Proteome, containing canonical and isoform sequences (71,567 sequences, accessed May 2017). These were also supplemented with common laboratory contaminants provided by MQ.  Cysteine carbamidomethylation was set as a fixed modification, while oxidation of methionine (M), protein N-terminal acetylation, and phosphorylation of serine (S), threonine (T) and tyrosine (Y) as variable modifications. Minimum peptide length was set to 7 amino acids, and peptides were allowed to have a maximum of two missed-cleavages. Majority of the MS assays used here were acquired using various Orbitrap instruments (THERMO raw files). For these raw files, the precursor mass tolerance was set to the default value of 20 ppm for the first search, and 4.5 for the main search. Fragment mass tolerances were set to 20 ppm and 0.5 Da for FT and IT detectors respectively. All other mass tolerance setting were kept at default values (see mqpar.xml file for more details).  We performed two searches on the data. In the first search, the false-discovery rate (FDR) was set to 0.01 at peptide spectrum match (PSM), protein, and site decoy fraction (PTM site FDR) levels. Minimum score for modified peptides was 40, and minimum delta score for modified peptides was 6. MQ results for this search are available in ‘txt-001PTM’ folder. In order to better characterize the false discovery correction at the site level, a second search was performed using the exact same parameters, with the exception of site decoy fraction FDR (PTM site FDR) which was relaxed to 1. MQ results for this search are available in ‘txt-100PTM’ folder.

### Publication Abstract
Protein phosphorylation is a key post-translational modification regulating protein function in almost all cellular processes. Although tens of thousands of phosphorylation sites have been identified in human cells, approaches to determine the functional importance of each phosphosite are lacking. Here, we manually curated 112 datasets of phospho-enriched proteins, generated from 104 different human cell types or tissues. We re-analyzed the 6,801 proteomics experiments that passed our quality control criteria, creating a reference phosphoproteome containing 119,809 human phosphosites. To prioritize functional sites, we used machine learning to identify 59 features indicative of proteomic, structural, regulatory or evolutionary relevance and integrate them into a single functional score. Our approach identifies regulatory phosphosites across different molecular mechanisms, processes and diseases, and reveals genetic susceptibilities at a genomic scale. Several regulatory phosphosites were experimentally validated, including identifying a role in neuronal differentiation for phosphosites in SMARCC2, a member of the SWI/SNF chromatin-remodeling complex.

### Keywords
Phosphorylation, Ptm, Reanalysis, Post translational modification, Public data reuse

### Affiliations
European Molecular Biology Laboratory, European Bioinformatics Institute (EMBL-EBI), Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SD, UK
EBI

### Submitter
Andrew Jarnuczak

### Lab Head
Dr Juan Antonio Vizcaino
European Molecular Biology Laboratory, European Bioinformatics Institute (EMBL-EBI), Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SD, UK


