### Accession
PXD018527

### Title
Analysis of the interactome of CD6 in CRISPR/CAS9 modified mouse CD4+ T cells

### Description
Using a CRISPR/Cas9-based approach we engineer primary CD4+ T cells in which the CD6 protein was tagged with an affinity Twin-Strep-tag (OST) with the purpose of determining by quantitative mass spectrometry the composition and dynamics of the signalosome assembling around the tagged protein prior to and following T cell activation.  Affinity purification of the OST tagged protein was performed using Streptactin beads, from T cells that were non-stimulated, or stimulated for 30s or 120s with anti-CD3 and anti-CD4 antibodies, or 300s with pervanadate. Each AP-MS purification is associated with a corresponding control (purification from non edited WT CD4+ T cells, cultured and stimulated in the same conditions). The number of replicate biological experiments was n=5 (cells non-stimulated or stimulated for 30s with anti-CD3/CD4), n=2 (cells stimulated for120s with antiCD3/CD4), or n=3 (cells stimulated 300s with pervanadate), and each sample was analyzed in duplicate or triplicate by LC-MS, resulting in 78 raw files.

### Sample Protocol
CD4+ T cells were purified from pooled lymph nodes and spleen, and cultured to perform the CRISPR/CAS9 edition and introduce the OST sequence. Edited CD4+ T cells (OST) and non edited control cells were left unstimulated or stimulated with antibodies (incubation with anti-CD3 and anti-CD4, followed by one round of washing and then cross-linkage with a purified Rabbit anti-Rat for 30s or 120s at 37°C) or with pervanadate (300s). Stimulation was stopped by the addition of twice concentrated lysis buffer and postnuclear lysates were then used for affinity purification on Strep-Tactin Sepharose beads. Proteins were eluted from the beads with 2.5 mM D-biotin, followed by SDS-PAGE short migration (one band) and in-gel trypsin digestion. Tryptic peptides were resuspended in 17 µl of 2% acetonitrile and 0.05% trifluoroacetic acid and analyzed by nano-liquid chromatography (LC) coupled to tandem MS, using an UltiMate 3000 system (NCS-3500RS Nano/Cap System; Thermo Fisher Scientific) coupled to an Orbitrap Q Exactive HFX mass spectrometer (Thermo Fisher Scientific). Five microliters of each sample were loaded on a C18 precolumn (300 µm inner diameter × 5 mm, Thermo Fisher Scientific) in a solvent made of 2% acetonitrile and 0.05% trifluoroacetic acid, at a flow rate of 20 µl/min. After 5 min of desalting, the precolumn was switched online with the analytical C18 column (75 µm inner diameter × 50 cm, Acclaim PepMap C18, 2 µM, Thermo Fisher Scientific) equilibrated in 95% solvent A (5% acetonitrile, 0.2% formic acid) and 5% solvent B (80% acetonitrile, 0.2% formic acid). Peptides were eluted using a 10%-45% gradient of solvent B over 60min at a flow rate of 350 nl/min. The mass spectrometer was operated in data-dependent acquisition mode with the Xcalibur software.  MS survey scans were acquired with a resolution of 60,000 and an AGC target of 3e6. The 12 most intense ions were selected for fragmentation by high-energy collision induced dissociation, and the resulting fragments were analyzed at a resolution of 15000, using an AGC target of 1e5 and a maximum fill time of 22ms. Dynamic exclusion was used within 30 s to prevent repetitive selection of the same peptide.

### Data Protocol
Raw MS files were processed with MaxQuant software (version 1.5.2.8) for database search with the Andromeda search engine and quantitative analysis of the the each interactome. Data were searched against Mus musculus entries in the UniProt protein database (release UniProtKB/Swiss-Prot+TrEMBL 2017_01, 89297 entries including isoforms), plus the One-Strep-tag peptide sequence, and the set of common contaminants provided by MaxQuant. Carbamidomethylation of cysteines was set as a fixed modification, whereas oxidation of methionine, protein N-terminal acetylation, and phosphorylation of serine, threonine, and tyrosine were set as variable modifications. Specificity of trypsin digestion was set for cleavage after K or R, and two missed trypsin cleavage sites were allowed. The precursor mass tolerance was set to 20 ppm for the first search and 4.5 ppm for the main Andromeda database search. The mass tolerance in tandem MS mode was set to 20 ppm. Minimum peptide length was set to 7 amino acids, and minimum number of unique or razor peptides was set to one. The I=L option of MaxQuant was enabled to avoid erroneous assignation of undistinguishable peptides belonging to very homologous proteins. Andromeda results were validated by the target decoy approach using a reverse database at both a peptide and protein false-discovery rate of 1% at both PSM and protein level. For label-free relative quantification of the proteins across biological replicates and stimulation time points, the match between runs option of MaxQuant was enabled with a match time window of 1 min, to allow cross-assignment of MS features detected in the different runs, after alignment of the runs with a time window of 20min. Protein quantification was based on unique and razor peptides. The minimum ratio count was set to 1 for LFQ calculation, and computation of the iBAQ metric was also enabled.

### Publication Abstract
None

### Keywords
Tcr, Ap-ms, Interactome, Ost tag, Crispr/cas9, Primary t cell

### Affiliations
Institute of Pharmacology and Structural Biology, CNRS, Toulouse
CNRS/IPBS

### Submitter
Anne Gonzalez de Peredo

### Lab Head
Dr Odile Burlet-Schiltz
Institute of Pharmacology and Structural Biology, CNRS, Toulouse


