### Accession
PXD022259

### Title
Proteomic analysis of rest and activated Tregs

### Description
The aim of the project is to do proteomic analysis of total cell lysate from rest and activated mouse Tregs. The proteomic data were mapped to the Gene Ontology Cellular Component (GOCC)database to obtain the list of membrane proteins on activated Tregs. The set of membrane proteins identified were compared with the potential Siglec-1 counter-receptors on activated Tregs.

### Sample Protocol
Resting and activated regulatory T cells were induced from freshly prepared mouse CD4 T cells from spleen and lymph nodes. The whole cell lysate were processed using paramagnetic bead (SP3 bead) technology, labelled using TMT10plexâ„¢ Isobaric label reagent, prefractionated and analyzed by mass spectrometry. Three biological replicates of resting and activated Tregs were prepared. In order to fit the 10 isobaric labels, two technique repeats of the third bioligical repeats were done for the labeling experiments. The activated Tregs correspond to the first 5 reporter ions, while the rest Tregs correspond to the 6-10 reporter ions.

### Data Protocol
The proteomic raw data were imported to MaxQuant Version 1.6.17.0 to search protein FASTA files of mouse in Uniprot database.

### Publication Abstract
None

### Keywords
Proteomics, Treg

### Affiliations
Division of Cell Signalling and Immunology, School of Life Sciences, University of Dundee, Dundee, DD1 5EH, Scotland, United Kingdom
Imperial College London

### Submitter
Gang Wu

### Lab Head
Dr Paul R. Crocker
Division of Cell Signalling and Immunology, School of Life Sciences, University of Dundee, Dundee, DD1 5EH, Scotland, United Kingdom


