### Accession
PXD024237

### Title
BioID with P. falciparum protein PhiL1

### Description
The aim of the project was to identify proteins proximal to Plasmodium falciparum PhiL1

### Sample Protocol
100 mL of highly synchronous PfPhIL1-BirA-GFP or 3D7 parasites at a parasitemia of 7 to 10% were grown until 40 hours post invasion. Then one half of the cultures was grown in culture media supplemented with 150 ÂµM biotin (Sigma-Aldrich) and compound 2 was added for 8 hours. Erythrocytes were lysed with 0.03% saponin and the isolated parasites were washed twice with PBS, taken up in lysis buffer (50 mM Tris-HCl pH 7.5, 500 mM NaCl, 1 mM DTT, 1 mM PMSF, 2x protease inhibitor mix, 1% Triton) and frozen at -80Â°C. Three freeze-thaw cycles were performed; the samples were centrifuged at 25,000 g for 60 minutes at 4Â°C and the supernatant was stored at -80Â°C. For purification of biotinylated proteins, 50 ÂµL streptavidin sepharose (GE Healthcare) was added to the lysate and incubated overnight with end-over-end rotation at 4Â°C. The beads were washed twice in lysis buffer, once in dH2O, twice in Tris-HCl (pH 7.5) and three times in 100 mM triethylammonium bicarbonate buffer (TEAB) pH 7.5 (Sigma-Aldrich). The washed beads were resuspended in 200 ÂµL 50 mM ammonium bicarbonate (pH 8.3) and on-bead trypsin digest (rolling with 1 Âµg of trypsin (Roche) for 16 h at 37Â°C followed by a second trypsin digest with 0.5 Âµg trypsin for 2 hours) was performed. Next the samples were centrifuged at 2,000 g for 5 minutes, resuspended in 2 x 150 ÂµL 50 mM ammonium bicarbonate (pH 8.3, â€œAmbicâ€�), transferred to and collected in a spin column (Pierce Spin Columns with Snap Cap, Thermo Scientific) placed in a low binding tube (Low Protein Binding Microcentrifuge tubes, Thermo Scientific). Subsequently, the left-over biotinylated peptides were eluted from the beads by 2 x 150 ÂµL 80% acetonitrile and 20% trifluoroacetic acid (â€œACNâ€�). Then, the samples were dried using SpeedVac (Thermo Fisher Scientific) and stored at -20Â°C. Dried peptides were sent to the Proteomics Core Facility at EMBL Heidelberg. Peptides were dissolved in 1% formic acid/4% acetonitrile, sonicated in the ultrasonic bath for 5 minutes and desalted using an OASISÂ® HLB ÂµElution Plate (Waters). Cleaned peptides were dissolved in 50 mM HEPES pH 8.5 and labelled with TMT6plex Isobaric Label Reagent (ThermoFisher) according to the manufacturerâ€™s instructions. After labelling, samples were pooled and purified from unreacted TMT label using OASISÂ® HLB ÂµElution Plate (Waters). An UltiMate 3000 RSLC nano LC system (Dionex) fitted with a trapping cartridge (Âµ-Precolumn C18 PepMap 100, 5 Âµm, 300 Âµm i.d. x 5 mm, 100 Ã…) and an analytical column (nanoEaseâ„¢ M/Z HSS T3 column 75 Âµm x 250 mm C18, 1.8 Âµm, 100 Ã…, Waters). Trapping was carried out with a constant flow of solvent A (0.1% formic acid in water) at 30 ÂµL/min onto the trapping column for 6 minutes. Subsequently, peptides were eluted via the analytical column with a constant flow of 0.3 ÂµL/min with increasing percentage of solvent B (0.1% formic acid in acetonitrile) from 2% to 4% in 4 min, from 4% to 8% in 2 min, then 8% to 28% for a further 37 min, and finally from 28% to 40% in another 9 min. The outlet of the analytical column was coupled directly to a QExactive plus (Thermo) mass spectrometer using the proxeon nanoflow source in positive ion mode. The peptides were introduced into the QExactive plus via a Pico-Tip Emitter 360 Âµm OD x 20 Âµm ID; 10 Âµm tip (New Objective) and an applied spray voltage of 2.1 kV. The capillary temperature was set at 275Â°C. Full mass scan was acquired with mass range 375-1200 m/z in profile mode with resolution of 70,000. The filling time was set at maximum of 10 ms with a limitation of 3x106 ions. Data-dependent acquisition (DDA) was performed with the resolution of the Orbitrap set to 17500, with a fill time of 50 ms and a limitation of 2x105 ions. A normalized collision energy of 32 was applied. Dynamic exclusion time of 30 s was used. The peptide match algorithm was set to â€˜preferredâ€™ and charge exclusion â€˜unassignedâ€™, charge states 1, 5 - 8 were excluded. MS2 data was acquired in profile mode.

### Data Protocol
IsobarQuant and Mascot (v2.2.07) were used to process the acquired data, which was searched against a Uniprot Plasmodium falciparum (UP000001450) proteome database containing common contaminants and reversed sequences. The following modifications were included into the search parameters: Carbamidomethyl (C) and TMT10 (K) (fixed modification), Acetyl (Protein N-term), Oxidation (M) and TMT10 (N-term) (variable modifications). For the full scan (MS1) a mass error tolerance of 10 ppm and for MS/MS (MS2) spectra of 0.02 Da was set. Further parameters were set: trypsin as protease with an allowance of maximum two missed cleavages: a minimum peptide length of seven amino acids; at least two unique peptides were required for a protein identification. The false discovery rate on peptide and protein level was set to 0.01.

### Publication Abstract
The inner membrane complex (IMC) is a defining feature of apicomplexan parasites, which confers stability and shape to the cell, functions as a scaffolding compartment during the formation of daughter cells and plays an important role in motility and invasion during different life cycle stages of these single-celled organisms. To explore the IMC proteome of the malaria parasite Plasmodium falciparum we applied a proximity-dependent biotin identification (BioID)-based proteomics approach, using the established IMC marker protein Photosensitized INA-Labelled protein 1 (PhIL1) as bait in asexual blood-stage parasites. Subsequent mass spectrometry-based peptide identification revealed enrichment of 12 known IMC proteins and several uncharacterized candidate proteins. We validated nine of these previously uncharacterized proteins by endogenous GFP-tagging. Six of these represent new IMC proteins, while three proteins have a distinct apical localization that most likely represents structures described as apical annuli in Toxoplasma gondii. Additionally, various Kelch13 interacting candidates were identified, suggesting an association of the Kelch13 compartment and the IMC in schizont and merozoite stages. This work extends the number of validated IMC proteins in the malaria parasite and reveals for the first time the existence of apical annuli proteins in P. falciparum. Additionally, it provides evidence for a spatial association between the Kelch13 compartment and the IMC in late blood-stage parasites.

### Keywords
Plasmodium falciparum, Lc-msms, Phil1

### Affiliations
Centre for Structural Systems Biology, 22607 Hamburg, Germany; Bernhard Nocht Institute for Tropical Medicine, 20359 Hamburg, Germany; University of Hamburg, 20146 Hamburg, Germany
EMBL

### Submitter
Frank Stein

### Lab Head
Dr Tim W. Gilberger
Centre for Structural Systems Biology, 22607 Hamburg, Germany; Bernhard Nocht Institute for Tropical Medicine, 20359 Hamburg, Germany; University of Hamburg, 20146 Hamburg, Germany


