### Accession
PXD012923

### Title
Proteomic analysis of meningiomas reveals clinically-distinct molecular patterns

### Description
Meningiomas represent one of the most common and clinically heterogeneous brain tumor types that only modestly correlate with histopathologic features. While emerging molecular profiling efforts have linked specific genomic drivers to distinct clinical patterns, the proteomic landscape of meningiomas remains largely unexplored.  We utilize mass spectrometry to profile a clinically well-annotated cohort (n=69) of meningiomas stratified to span all three World Health Organization (WHO) grades and various degrees of clinical aggressiveness.  In total, we quantify 3042 unique proteins and compare the patterns across different clinical parameters. Unsupervised clustering analysis highlighted distinct proteomic (n=106 proteins, Welch’s t-test, P<0.01) and pathway-level (e.g. Notch and PI3K/AKT/mTOR) differences between convexity and skull base meningiomas. Supervised comparative analyses of different pathological grades revealed distinct patterns between benign (WHO Grade I) and atypical/malignant (WHO Grade II and III) meningiomas with classic oncogenes often enriched in higher grade lesions. Independent of WHO grade, clinically aggressive meningiomas, that rapidly recurred, also had distinctive protein patterns that converged on mRNA processing and impaired activation of the extracellular matrix naba matrisome complex. Larger sized meningiomas, and those with previous radiation exposure, also had distinct protein profiles.  Collectively, we highlight distinct clinically-dependent proteomic patterns of meningiomas that may help better predict outcome and guide the development of more personalized and directed therapies.

### Sample Protocol
Meningioma tissue was collected from our hospital archives as FFPE curls, deparaffinized in xylene and subsequently washed in ethanol gradients. The samples were then kept in 50% ethanol for 60 minutes and then 200 μL of tissue lysis buffer (0.25% Rapigest (Waters) in 200 mM Tris, 50 mM ammonium bicarbonate, 5mM DTT) was added to each sample. The samples were kept on ice for 30 minutes and then sonicated for 5 pulses, 10 seconds each, at 3.5 amplitude setting (QSonica XL-2000 Sonicator). Lysates were placed on a heat block at 95°C for 45 minutes followed by 80°C for 90 minutes with gentle mixing every 10 minutes to remove protein crosslinks. Lysates were centrifuged at 14,000 x g, and supernatants were transferred to a clean centrifuge tube followed by protein quantification using Pierce Coomassie protein assay kit (ThermoFisher Scientific). Depending on protein amounts, 10 μg or the whole sample was alkylated in 15mM iodoacetamide, 50mM ammonium bicarbonate (Sigma) for 30 minutes at room temperature in the dark. Samples were digested with HPLC-grade trypsin (Sigma) in a 1:30 enzyme to substrate ratio overnight in a 37°C incubator with constant shaking. The digestion reaction was halted with 1% trifluoroacetic acid, which also precipitates the detergent from Rapigest. Samples were again centrifuged at 14,000 x g and the supernatant was transferred to a clean tube. For short term storage, samples were kept at 4°C until desalting and then moved to -80°C for longer periods.

### Data Protocol
All mass spectrometry raw data files were searched using the Maxquant Andromeda search engine (www.coxdocs.org) version 1.6.2.3 against the Human Swissprot protein database (July, 2017 version) using the match between runs algorithm across all samples. The parameters used for the search were: trypsin/P enzyme with up to 2 missed cleavages allowed; carbamidomethylation of cysteine residues as a fixed modification; oxidation of methionine, phosphorylation of serine, threonine and tyrosine, N-terminal protein acetylation and ubiquitination of lysine as variable modifications; first search peptide tolerance set to 20 ppm against a small “human-first” search database for mass recalibration and main search peptide tolerance set to 4.5 ppm; contaminants were included in the search; database was reversed for calculating peptide and protein-level false discovery rates (FDR) at 1%; match between runs was set at a 0.7 minute window; LFQ minimum ratio count was set to 1.

### Publication Abstract
None

### Keywords
Neuro-oncology, Proteome, Stratification, Mass spectrometry, Meningiomas

### Affiliations
Princess Margaret Cancer Centre
Neuropathologist, Department of Pathology, University Health Network Assistant Professor, Department of Laboratory Medicine and Pathobiology, University of Toronto Scientist, Princess Margaret Cancer Centre

### Submitter
MICHAIL PAPAIOANNOU

### Lab Head
Dr Phedias Diamandis
Neuropathologist, Department of Pathology, University Health Network Assistant Professor, Department of Laboratory Medicine and Pathobiology, University of Toronto Scientist, Princess Margaret Cancer Centre


### SDRF
- organism: Homo sapiens
- organism part: brain
- cell type: not applicable
- disease: Meningioma
- fractionation method: no fractionation
- label: AC=MS:1002038;NT=label free sample
- instrument: AC=MS:1002634; NT=Q Exactive Plus
- separation: AC=PRIDE:0000563;NT=Reversed-phase chromatography
- modification parameters: NT=Carbamidomethyl;AC=UNIMOD:4;TA=C;MT=Fixed
- modification parameters.1: NT=Oxidation;AC=UNIMOD:35;MT=Variable;TA=M
- modification parameters.2: NT=Acetyl;AC=UNIMOD:1;PP=Protein N-term;MT=variable
- modification parameters.3: NT=Phospho;AC=UNIMOD:21;MT=Variable;TA=S
- modification parameters.4: NT=Phospho;AC=UNIMOD:21;MT=Variable;TA=T
- modification parameters.5: NT=Phospho;AC=UNIMOD:21;MT=Variable;TA=Y
- modification parameters.6: NT=Ubiquitination;AC=UNIMOD:853;MT=Variable,TA=K

