### Accession
PXD014648

### Title
Homologous recombination repair and cholesterol-mediated drug efflux induce dose-dependent chemoresistance in nutrient-deprived neuroblastoma cells

### Description
Neuroblastoma is a solid pediatric tumor with heterogeneous clinical behaviors. Chemotherapy is widely used to treat neuroblastoma. However, dose-dependent responses and chemoresistance mechanisms of neuroblastoma cells to anticancer drugs remain challenges in various conditions. Here, we investigated the dose-dependent effects of topotecan on human neuroblastoma cells (SK-N-SH, SH-SY5Y, and SK-N-BE) under various nutrient supply conditions. Serum-starved human neuroblastoma cells showed reduced toxicity, and their survival rate increased upon treatment with a high concentration (1 μM) of topotecan. Quantitative profiling of global and phosphoproteome identified 12,959 proteins and 48,812 phosphosites, respectively, from SK-N-SH cells. Network analysis revealed that topotecan up-regulated DNA repair and cholesterol-mediated topotecan efflux, resulting in topotecan resistance. Results of DNA damage assay, cell cycle, and quantitative analyses of membrane cholesterol supported the validity of these resistance factors and their applicability to all neuroblastoma cells investigated in this study. Our results provide a model for high dose-dependent chemoresistance in neuroblastoma cells that could enable a patient-dependent chemotherapy screening strategy.

### Sample Protocol
SK-N-SH neuroblastoma cells were purchased from the Korea Cell Line Bank (Seoul, Korea) and were provided from the Samsung Medical Center (Seoul, Korea). Cells were cultured in a Dulbecco’s Modified Eagle’s Medium/F-12 Nutrient Mixture Ham (DMEM/F-12) 1:1 mixture (purchased from WelGENE, South Korea) with 10 % FBS and 1 % antibiotics. The cultured cells were incubated at 37 °C in a humidified 5 % CO2 atmosphere. The cells were lysed using a probe sonicator with lysis buffer (4% SDS, 0.1 M Tris-HCl pH 7.6), repeating 30 seconds on ice for 5 times. The homogenate was centrifuged at 16,000 g and 20ºC for 10 min and the supernatant was transferred to a new tube. Protein concentration was measured using the BCA protein assay. The lysates obtained from the 12 different cell suspensions were individually digested using the FASP protocol, as described previously. (Wisniewski et al., 2009) The proteins were reduced in SDT buffer (4 % SDS in 0.1M Tris-HCl, pH 7.6, and 0.1 M DTT) at 37 ºC for 45 min with shaking at 300 rpm and boiled for 10 min at 95 ºC on a thermomixer. The protein sample was then transferred to a membrane filter (YM-30, Millipore Corporation) and mixed with 200 μL of 8 M urea (in 0.1 M Tris-HCl, pH 8.5). The protein sample on the membrane filter was centrifuged three times at 14,000  g and 20 ºC for 60 min to remove SDS. Subsequently, 100 μL of 0.05 M iodoacetamide in 8 M urea was added for 25 min at room temperature in the dark to alkylate free thiol groups in the proteins. The protein samples on the membrane filters were diluted with 200 μL of 8 M urea and concentrated four times. Finally, 100 μL of 50 mM ammonium bicarbonate (pH 8.0) was added to the filter, followed by two rounds of centrifugation at 14,000 g and 20 ºC for 30 min. Trypsin was added to the filter unit at an enzyme to protein ratio of 1:50 (w/w) and the proteins were digested at 37 ºC overnight. A second digestion was carried out with trypsin (1:100 ratio) at 37 ºC for 6 h. The resulting peptides were eluted by centrifugation at 14,000 g and 20 ºC for 30 min. The filter was rinsed with 60 μL of 50 mM ammonium bicarbonate and centrifuged at 14,000 g and 20 ºC for 20 min. The peptides in the eluents were combined, dried, and kept at −80 ºC until subsequent TMT labeling. Each set of six peptide samples from the 0.1 and 1 μM of topotecan treatments were labeled with a 6-plex TMT reagent. Peptide samples (100 μg each) were each labeled with 126, 127, 128, 129, 130, or 131 TMT reagents, according to the manufacturer's instructions. Each set TMT-labeled peptide samples was pooled and dried using vacuum centrifugation. The peptide pool was first subjected to IMAC phosphopeptide enrichment and the whole phosphopeptide sample was fractionated into 12 online non-contiguously fractionating and concatenating (NCFC) fractions using the DO-NCFC-RP/RP-MS/MS system to generate 12 phosphopeptide LC-MS/MS data set while the flow through sample (i.e. non-phosphopeptide) was subsequently fractionated into 24 online NCFC fractions to generate 24 global LC-MS/MS data set. Both global proteome and phosphoproteome analysis were performed using quadrupole-orbitrap mass spectrometry (Exploris 480, Thermo Fisher Scientific) with high-field asymmetric wave-form ion mobility spectrometry (FAIMS). MS analysis was performed in the positive electrospray ionization (ESI) mode with an electric potential of 3 kV. The temperature of the desolvation capillary was set at 250 ºC and the temperature of both inner and outer electrode of FAIMS were set to 100 ºC. Nitrogen was used for FAIMS carrier gas which was turned on for 1 minute at the end of each experi-ment. -35V, -50V and -65V were considered as multiple compensation voltage (CV) of FAMIS with cycle time of 1 second. For each CV, full MS scans ranged from 350 to 1400 Th were ac-quired at a resolution of 60,000 and a maximum ion injection time of 20 ms. Monoisotopic pre-cursor selection (MIPS) mode was activated for peptide and minimum intensity of 100000 were filtered. The isolation window of 1.2 Th were fragmented using a data-dependent acquisition method with a normalized collision energy (NCE) of 28 for higher-energy collisional dissocia-tion (HCD). The MS/MS scans were obtained at a resolution of 15,000, with a fixed first mass of 120 Th and a maximum ion injection time of 32 ms. The exclusion duration was set to 25 s, and charge states of 2-5 and undetermined ions were included. The normalized automatic gain control (AGC) target value of full MS and MS/MS scans was set to 500 and 1000 respectively.

### Data Protocol
The precursor masses of the MS/MS scans were refined by Multiplexed Post-Experiment Monoisotopic Mass Refinement (mPE-MMR) prior to conducting a database search. The MS/MS data (e.g., mgf files) were searched against a protein database consisting of the UniProt human reference database (released June 2018, including 94,744 entries) and common contaminants (179 entries) using the MS-GF+ search engine (v9949, http://proteomics.ucsd.edu/software-tools/ms-gf/). For global proteome datasets, the database searches were performed with the following parameters: semi-tryptic cleavage and a precursor mass tolerance of 10 ppm. For static modifications, TMT (+229.162932 Da) on lysine and N-terminus and carbamidomethylation (+57.0214 Da) on cysteine were considered. Methionine oxidation (+15.994915 Da), along with asparagine and glutamine deamidation, were set as variable modifications. For phosphoproteome datasets, phosphorylation of serine, threonine, and tyrosine was used as an additional variable modification. The search results were filtered using a false discovery rate of 1% peptide spectrum match (PSM)-level, which was estimated by the target decoy search strategy.

### Publication Abstract
None

### Keywords
Human, Phosphoproteomics, Proteomics, Lc-ms/ms, Topotecan, Neuroblastoma

### Affiliations
Korea University
1. Department of Chemistry, Korea University, Seoul 02841, Republic of Korea 2. Center for Proteogenome Research, Korea University, Seoul 02841, Republic of Korea

### Submitter
Dowoon Nam

### Lab Head
Dr Sang-Won Lee
1. Department of Chemistry, Korea University, Seoul 02841, Republic of Korea 2. Center for Proteogenome Research, Korea University, Seoul 02841, Republic of Korea


