### Accession
PXD045447

### Title
Antiangiogenic Potential of an Olive Oil Extract: Insights from a Proteomic Study

### Description
In this comprehensive study, we delve into the examination of the phenolic fraction found in Extra Virgin Olive Oils (EVOOs) and its potential impact on endothelial cells, with a particular focus on its role in angiogenesis. To accomplish this, we employed a phenolic extract derived from EVOO sourced from the Picual variety in Ja√©n, Spain. Our investigation of an extract offers a compelling advantage, enabling us to obtain larger quantities of bioactive molecules compared to conventional dietary consumption, provided that the concentrations of these diverse compounds within the mixture are deemed safe. Furthermore, this approach permits us to meticulously isolate and assess the effects of these molecules independently from the other components of EVOO, such as fatty acids and more.

### Sample Protocol
We carried out a gel-assisted proteolysis, entrapping the protein solution in a polyacrylamide gel matrix. Samples (45 ŒºL) were thoroughly mixed with 14 ŒºL of 40% acrylamide monomer solution, and 2.5 ŒºL of 10% ammonium persulfate and 1 ŒºL of TEMED were quickly added. The mixture was allowed to completely polymerize for 20 min before performing an in-gel digestion. Using scalpel, gel plugs were cut into 1‚Äì2 mm cubes and treated with 50% acetonitrile/25 mM ammonium bicarbonate. Samples were dehydrated and desiccated with acetonitrile (ACN) before reduction with 10mM dithiothreitol (DTT) in 50 mM ammonium bicarbonate for 30 min at 56 ¬∫C. Excess DTT was removed, and cysteine residues were carbamidomethylated with 55 mM iodoacetamide in 50 mM ammonium bicarbonate for 20 min at room temperature in the dark. After carbamidomethylation, the gel pieces were dehydrated again; proteins were digested by rehydrating the gel pieces in trypsin solution at 10 ng/ŒºL (Pierce trypsin protease, MS grade; Thermo Fisher Scientific, Waltham, MA, USA) and thereafter incubated at 30 ¬∫C overnight. Peptides were extracted from the gel pieces with ACN/0.1% formic acid (FA) for 30 min at room temperature. The samples were dried in a SpeedVac‚Ñ¢Ô∏è (Thermo Fisher Scientific, Waltham, MA, USA) vacuum concentrator to remove ACN and residual ammonium bicarbonate, redissolved in 50 ŒºL of 0.1% FA, sonicated for 3 min and centrifuged at 14,000x g for 5 min. Finally, the samples were quantified again in a NanoDrop‚Ñ¢Ô∏è (Thermo Fisher Scientific, Waltham, MA, USA); 0.1% FA was added to equalize all samples at an identical protein concentration before being transferred to the injection vial.  Liquid Chromatography High-Resolution Mass Spectrometry (HPLC-MS): Samples were injected onto an Easy nLC 1200 UHPLC system coupled to a hybrid linear trap quadrupole Orbitrap Q-Exactive HF-X mass spectrometer (Thermo Fisher Scientific, Waltham, MA, USA). Software versions used for the data acquisition and operation were Tune 2.9 and Xcalibur 4.1.31.9. HPLC solvents were as follows: solvent A consisted of 0.1% formic acid in water, and solvent B consisted of 0.1% formic acid in 80% acetonitrile. From a thermostated autosampler, 2 ¬µL that correspond to 100 ng of the peptide mixture were automatically loaded onto a trap column (Acclaim PepMap 100, 75 ¬µm x 2 cm, C18, 3 ¬µm, 100 A, Thermo Fisher Scientific, Waltham, MA, USA) at a flow rate of 20 ¬µL/min and eluted onto a 50 cm analytical (PepMap RSLC C18, 2 ¬µm, 100 A, 75 ¬µm x 50 cm, Thermo Fisher Scientific, Waltham, MA, USA). The peptides were eluted from the analytical column with a 180 min gradient ranging from 2% to 20% solvent B, followed by a 30 min gradient from 20% to 35% solvent B and finally, to 95% solvent B for 15 min before re-equilibration to 2% solvent B at a constant flow rate of 300 nL/min. The LTQ Velos ESI Positive Ion Calibration Solution (Thermo Fisher Scientific, Waltham, MA, USA) was used to externally calibrate the instrument prior to sample analysis, and an internal calibration was performed on the polysiloxane ion signal at m/z 445.120024 from ambient air. MS1 scans were performed from m/z 300‚Äì1750 at a resolution of 120,000. Using a data-dependent acquisition mode, the 20 most intense precursor ions of all precursor ions with +2 to +5 charge were isolated within a 1.2 m/z window and fragmented to obtain the corresponding MS/MS spectra. The fragment ions were generated in a higher energy collisional dissociation (HCD) cell with a fixed first mass at 110 m/z and detected in an Orbitrap mass analyzer at a resolution of 30,000. The dynamic exclusion for the selected ions was 30 s. Maximal ion accumulation time allowed in MS and MS2 mode was 50 ms. Automatic gain control was used to prevent overfilling of the ion trap and was set to 3 √ó 106 ions and 105 ions for a full MS and MS2 scan, respectively.

### Data Protocol
Data Analysis for Protein Identification: MS/MS spectra were searched against SwissProt Homo sapiens protein database canonical version 2021.09.30 (20,315 sequences). The UniProt Olea europaea database (79,094 sequences, version 2023-06-26) and the UniProt Arabidopsis Thaliana database (18,588 sequences, version 2023-06-28) were used as plant contaminant databases. The acquired raw data were analyzed in Proteome DiscovererTM 2.5 (Thermo Fisher Scientific,Waltham,MA, USA) platform with the SEQUEST¬Æ HT engine using mass tolerances of 10 ppm and 0.02 Da for precursor and fragment ions, respectively. Two missed tryptic cleavage sites were allowed. Oxidation of methionine and N-terminal acetylation were set as variable modifications, whilst carbamidomethylation of cysteine residues, was set as fixed modification. Peptide spectral matches (PSM) and consecutive protein assignments were validated using the Percolator¬Æ algorithm (K√§ll et al., 2007), based on a target-decoy approach using a reversed protein database as the decoy by imposing a strict cut-off of 1% false discovery rate (FDR). Peptide identifications were grouped into proteins according to the law of parsimony, and results were filtered to contain only proteins with at least two unique peptide sequences.   Label-Free Relative Quantification for Differential Expression Analysis: Label-free quantitation was implemented using the Minora feature of Proteome DiscovererTM 2.5 (Thermo Fisher Scientific, Waltham, MA, USA), setting the following parameters: maximum retention time alignment of 10 min with minimum S/N of 5 for feature linking mapping. Abundances were based on precursor intensities. Normalization was performed based on total peptide amount, and samples were scaled on all averages (for every protein and peptide, the average of all samples is 100). The normalized and scaled relative abundance of every protein was expressed as mean ¬± standard deviation (SD) of three biological replicates. Protein abundance ratios were directly calculated from the grouped protein abundances. Abundance ratio ùò±-values were calculated by ANOVA based on the abundances of individual proteins or peptides.

### Publication Abstract
Dimethyl fumarate is an ester from the Krebs cycle intermediate fumarate. This drug is approved and currently used for the treatment of psoriasis and multiple sclerosis, and its anti-angiogenic activity was reported some years ago. Due to the current clinical relevance of this compound and the recently manifested importance of endothelial cell metabolism on the angiogenic switch, we wanted to elucidate whether dimethyl fumarate has an effect on energetic metabolism of endothelial cells. Different experimental approximations were performed in endothelial cells, including proteomics, isotope tracing and metabolomics experimental approaches, in this work we studied the possible role of dimethyl fumarate in endothelial cell energetic metabolism. We demonstrate for the first time that dimethyl fumarate promotes glycolysis and diminishes cell respiration in endothelial cells, which could be a consequence of a down-regulation of serine and glycine synthesis through inhibition of PHGDH activity in these cells. Dimethyl fumarate alters the energetic metabolism of endothelial cells in vitro and in vivo through an unknown mechanism, which could be the cause or the consequence of its pharmacological activity. This new discovery on the targets of this compound could open a new field of study regarding the mechanism of action of dimethyl fumarate.

### Keywords
Evoo, Angiogenesis, Proteomics, Polyphenols, Olive oil

### Affiliations
Department of Molecular Biology and Biochemistry, Faculty of Sciences, University of Malaga, Spain.
University of Malaga

### Submitter
Casimiro Cardenas

### Lab Head
Dr Miguel Angel Medina
Department of Molecular Biology and Biochemistry, Faculty of Sciences, University of Malaga, Spain.


