### Accession
PXD001346

### Title
Exosomal proteome from mouse CSF

### Description
Exosomes were isolated from the mouse CSF using total exosome isolation kit (Life Technologies) from 0h and 6h LPS treated mice. These exosomes were separated on the RP-HPLC in to 20 fractions and analysed on the Qexactive.

### Sample Protocol
EVs were isolated from the 50ul of the CSF using total exosome isolation kit (Life technologies; 4478360). Isolated EVs were suspended in the 20 µl of PBS and mixed with 5µl of SDS loading dye with one µl of the XT reducing agent (Biorad). Samples were boiled for 5 minutes at 95°C. These samples were briefly separated by SDS-PAGE loaded in 4-12% precast gradient gel (Criterion XT, Biorad) and stained with coomassie blue (SimplyBlue™, Life technologies). Stained protein bands were carefully excised under sterile conditions and carefully washed with 100 µl of the sterile water was added, briefly vortexed and incubated at room temperature for 15 minutes, supernatant was discarded after centrifugation. Then 100 µl of 50% acetonitrile was added and briefly vortexed, supernatant was discarded after 15 minutes of incubation. Next, bands were washed with 100 µl of the 100% acetonitrile, briefly vortex and incubated for 15 minutes at room temperature and supernatant was discarded after centrifugation. These bands were completely dried in speedvac. In-gel digestion was done by using trypsin enzyme (Promega; V5280; Trypsin Gold, Mass Spectrometry grade) and incubated at 37°C overnight. Next day samples were briefly centrifuged and supernatant was transferred to new tube, pH was adjusted to two using formic acid to inactivate the trypsin. These digested peptide materials were pre-fractionated using RP-HPLC (C18-HD; 3µM, 12 cm column with 250µM diameter), 100 fractions of one minute each were collected that were further pooled to make 20 final fractions for LC-MS/MS analysis.

### Data Protocol
The mass spectrometer was operated in data-dependent, positive ionization mode, automatically switching between MS and MS/MS acquisition for the 10 most abundant peaks in a given MS spectrum. The source voltage was 3.4 kV and the capillary temperature was 275°C. One MS1 scan (m/z 400−2000, AGC target 3 × 106 ions, maximum ion injection time 80 ms) acquired at a resolution of 70,000 (at 200 m/z) was followed by up to 10 tandem MS scans (resolution 17,500 at 200 m/z) of the most intense ions fulfilling predefined selection criteria (AGC target 5 × 104 ions, maximum ion injection time 60 ms, isolation window 2 Da, fixed first mass 140 m/z, spectrum data type: centroid, underfill ratio 2%, intensity threshold 1.7xE4, exclusion of unassigned, 1, 5-8, >8 charged precursors, peptide match preferred, exclude isotopes on, dynamic exclusion time 20 s). The HCD collision energy was set to 25% Normalized Collision Energy and the polydimethylcyclosiloxane background ion at 445.12002 Da was used for internal calibration (lock mass). From the MS/MS data in each LC-MS/MS run, Mascot Generic Files were created using the Mascot Distiller software (version 2.4.3.3, Matrix Science, www.matrixscience.com/Distiller). While generating these peak lists, grouping of spectra was allowed in Mascot Distiller with a maximal intermediate retention time of 30 s, and a maximum intermediate scan count of 5 was used where possible. Grouping was done with 0.005 Da precursor tolerance. A peak list was only generated when the MS/MS spectrum contained more than 10 peaks. There was no de-isotoping and the relative signal to noise limit was set at 2. These peak lists were then searched using the Mascot search engine (MatrixScience, www.matrixscience.com) using the Mascot Daemon interface (version 2.4, Matrix Science). Spectra were searched against the SwissProt restricted to Mus musculus taxonomy. Variable modifications were set to methionine oxidation, pyro-glutamate formation of amino terminal glutamine, and acetylation of the protein N-terminus. Mass tolerance on precursor ions was set to 10 ppm (with Mascot’s C13 option set to 1), and on fragment ions to 20 mmu. The instrument setting was put on ESI-QUAD. Enzyme was set to trypsin, allowing for 1 missed cleavage. Only peptides that were ranked first and scored above the Mascot significance score threshold set at 99% confidence were withheld. The 95% threshold corresponds to a chance of 1 in 100 for an identified peptide to be a false positive (approximately corresponding to 1% FDR). The peptides were further mapped back to the original protein database using an in-house Perl script to filter out and keep only those peptides matching to one protein. Among all isoforms for a given protein, the representative isoform, being the one with the longest sequence, was considered.

### Publication Abstract
Here, we identified release of extracellular vesicles (EVs) by the choroid plexus epithelium (CPE) as a new mechanism of blood-brain communication. Systemic inflammation induced an increase in EVs and associated pro-inflammatory miRNAs, including miR-146a and miR-155, in the CSF Interestingly, this was associated with an increase in amount of multivesicular bodies (MVBs) and exosomes per MVB in the CPE cells. Additionally, we could mimic this using LPS-stimulated primary CPE cells and choroid plexus explants. These choroid plexus-derived EVs can enter the brain parenchyma and are taken up by astrocytes and microglia, inducing miRNA target repression and inflammatory gene up-regulation. Interestingly, this could be blocked in&#xa0;vivo by intracerebroventricular (icv) injection of an inhibitor of exosome production. Our data show that CPE cells sense and transmit information about the peripheral inflammatory status to the central nervous system (CNS) via the release of EVs into the CSF, which transfer this pro-inflammatory message to recipient brain cells. Additionally, we revealed that blockage of EV secretion decreases brain inflammation, which opens up new avenues to treat systemic inflammatory diseases such as sepsis.

### Keywords
Mouse, Csf, Extracellular vesicles, Exosomes

### Affiliations
VIB Department of Medical Protein Research, UGent Proteome Analysis and Bioinformatics Unit A. Baertsoenkaai 3 B9000 Gent Belgium
VIB-Medical protein research

### Submitter
Sriram Balusu

### Lab Head
Dr Kris Gevaert
VIB Department of Medical Protein Research, UGent Proteome Analysis and Bioinformatics Unit A. Baertsoenkaai 3 B9000 Gent Belgium


