### Accession
PXD008555

### Title
Integrated multi-omic analysis of M. tuberculosis H37Ra redefines virulence attributes

### Description
M. tuberculosis H37Ra, an avirulent tubercle bacillus, is a commonly used model to investigate virulence attenuation in M. tuberculosis. Comparative high-throughput studies have earlier correlated its avirulence to the presence of specific mutations or absence of certain proteins. However, a recent sequencing study of H37Ra has disproved several genomic differences earlier reported to be associated with virulence. This warrants further investigations on the H37Ra proteome as well. In this study, we carried out an integrated analysis of the genome, transcriptome and proteome of H37Ra. In addition to confirming single nucleotide variations and insertion-deletions that were reported earlier, our study provides novel insights into the mutation spectrum in the promoter regions of 7 genes. In all, we provide protein coding evidence for 3,199 proteins representing ~79% of the total predicted gene count. Transcriptome analysis revealed the expression of 3,945 high-confidence transcripts including several transcripts mapping to the genome that were previously thought to be non-coding. We identified 9 genes whose coding potential was hitherto reported to be absent in H37Ra . These include 2 putative virulence factors belonging to ESAT-6 like family of proteins. Furthermore, proteogenomic analysis enabled us to identify 63 novel proteins coding genes and correct 25 existing gene models in H37Ra genome. A majority of these were found to be conserved in the virulent strain H37Rv as well as in other mycobacterial sp. suggesting that that the differences in the virulent and avirulent strains of M. tuberculosis are not entirely dependent on the expression of certain proteins or their absence but may possibly be ascertained to functional changes.

### Sample Protocol
M. tuberculosis H37Ra strain (MTCC: 300) was cultured in Middlebrook 7H9 culture medium (BD) supplemented with OADC (Oleic acid, dextrose and catalase).The culture was incubated at 37Â°C in a roller bottle shaker and was harvested at logarithmic phase. Bacterial pellets were obtained by centrifugation which was further washed and lysed using 4% SDS buffer. The bacterial lysate was reduced, alkylated and subjected to in-solution as well as in-gel trypsin digestion. 28 fractions (12 bRPLC and 16 SDS-PAGE) were collected, dried and desalted using  desalted using C18 StageTips. LC-MS/MS analysis was carried out using Orbitrap Fusion Tribrid mass spectrometer (Thermo Scientific, Bremen, Germany) coupled to Easy-nLC1200 nano-flow UHPLC (Thermo Scientific, Bremen, Germany).

### Data Protocol
Mass spectrometry data obtained from all LC-MS/MS analysis were searched against M. tuberculosis H37Ra protein database downloaded from NCBI (updated May, 2016), combined with common contaminants (4,143 total sequences) using SequestHT and Mascot (version 2.2) search algorithms through Proteome Discoverer 2.1 (Thermo Scientific, Bremen, Germany). Enzyme specificity was set as trypsin with maximum two missed cleavage allowed. The minimum peptide length was specified to be 7 amino acids. Carbamidomethylation of cysteine was specified as fixed modification and oxidation of methionine and protein N-terminal acetylation were included as variable modifications. The precursor and fragment ion mass tolerance were set as 10 ppm and 0.05 Da repectively. The data were also searched against a decoy database and MS/MS identifications of <1% false discovery rate (FDR) score threshold was considered for further analysis. The unannotated spectra were searched against the 6 frame translated M. tuberculosis H37Ra genome, 3 frame translated RNA-Seq transcripts, alternative start codon database and hypothetical N-terminal peptide database with the above mentioned parameters.

### Publication Abstract
H37Ra is a virulence attenuated strain of <i>Mycobacterium tuberculosis</i> widely employed as a model to investigate virulence mechanisms. Comparative high-throughput studies have earlier correlated its avirulence to the presence of specific mutations or absence of certain proteins. However, a recent sequencing study of H37Ra, has disproved several genomic differences earlier reported to be associated with virulence. This warrants further investigations on the H37Ra proteome as well. In this study, we carried out an integrated analysis of the genome, transcriptome, and proteome of H37Ra. In addition to confirming single nucleotide variations (SNVs) and insertion-deletions that were reported earlier, our study provides novel insights into the mutation spectrum in the promoter regions of 7 genes. We also provide transcriptional and proteomic evidence for 3,900 genes representing ~80% of the total predicted gene count including 408 proteins that have not been identified previously. We identified 9 genes whose coding potential was hitherto reported to be absent in H37Ra. These include 2 putative virulence factors belonging to ESAT-6 like family of proteins. Furthermore, proteogenomic analysis enabled us to identify 63 novel proteins coding genes and correct 25 existing gene models in H37Ra genome. A majority of these were found to be conserved in the virulent strain H37Rv as well as in other mycobacterial species suggesting that the differences in the virulent and avirulent strains of <i>M. tuberculosis</i> are not entirely dependent on the expression of certain proteins or their absence but may possibly be ascertained to functional changes.

### Keywords
Tuberculosis, Next-generation sequencing, Genome annotation, Virulence attenuation, Orbitrap

### Affiliations
Institute of Bioinformatics, Bangalore, Karnataka, India.
Institute of Bioinformatics, 7th floor, Discover Building, International Tech Park, Bangalore, India

### Submitter
Keshava Prasad T. S.

### Lab Head
Dr T. S. Keshava Prasad
Institute of Bioinformatics, 7th floor, Discover Building, International Tech Park, Bangalore, India


