### Accession
PXD024034

### Title
Transcriptomic, proteomic, and physiological comparative analyses of flooding mitigation of the damage induced by low-temperature stress in direct seeded early indica rice at the seedling stage

### Description
Background: Low temperature (LT) often occurs at the seedling stage in the early rice-growing season, especially for direct seeded early-season indica rice, and using flooding irrigation can mitigate LT damage in rice seedlings. The molecular mechanism by which flooding mitigates the damage induced by LT stress has not been fully elucidated. Thus, LT stress at 8C, LT accompanied by flooding (LTF) and CK (control) treatments were established for three days to determine the transcriptomic, proteomic and physiological response in direct seeded rice seedlings at the seedling stage. Results: LT damaged chloroplasts, and thylakoid lamellae, and increased osmiophilic bodies and starch grains compared to CK, but LTF alleviated the damage to chloroplast structure caused by LT. The physiological characteristics of treated plants showed that compared with LT, LTF significantly increased the contents of rubisco, chlorophyll, PEPCK, ATP and GA3 but significantly decreased soluble protein, MDA and ABA contents. 4D-label-free quantitative proteomic profiling showed that photosynthesis-responsive proteins, such as phytochrome, as well as chlorophyll and the tricarboxylic acid cycle were significantly downregulated in LT/CK and LTF/CK comparison groups. However, compared with LT, phytochrome, chlorophyllide oxygenase activity and the glucan branching enzyme in LTF were significantly upregulated in rice leaves. Transcriptomic and proteomic studies identified 72818 transcripts and 5639 proteins, and 4983 genes that were identified at both the transcriptome and proteome levels. Differentially expressed genes (DEGs) and differentially expressed proteins (DEPs) were significantly enriched in glycine, serine and threonine metabolism, biosynthesis of secondary metabolites, glycolysis/gluconeogenesis and metabolic pathways.  Conclusion: Through transcriptomic, proteomic and physiological analyses, we determined that a variety of metabolic pathway changes were induced by LT and LTF. GO and KEGG enrichment analyses demonstrated that DEGs and DEPs were associated with photosynthesis pathways, antioxidant enzymes and energy metabolism pathway-related proteins. Our study provided new insights for efforts to reduce the damage to direct seeded rice caused by low-temperature stress and provided a breeding target for low temperature flooding-resistant cultivars. Further analysis of translational regulation and metabolites may help to elucidate the molecular mechanisms by which flooding mitigates low-temperature stress in direct seeded early indica rice at the seedling stage.

### Sample Protocol
The sample was first ground in liquid nitrogen, and then the powder was transferred to a 5-mL centrifuge tube. After that, four volumes of lysis buffer (8 M urea, 1% Protease Inhibitor Cocktail) were added to the cell powder, followed by sonication three times on ice using a high-intensity ultrasonic processor (Scientz, China). The samples were centrifuged at 12,000×g at 4°C for 10 min and the debris was removed. Finally, the protein concentration was determined with a BCA kit (Beyotime, China) according to the manufacturer’s instructions.  For digestion, the protein solution was reduced with 5 mM dithiothreitol for 30 min at 56°C and alkylated with 11 mM iodoacetamide for 15 min at room temperature in darkness. The protein sample was then diluted by adding 100 mM TEAB to urea concentrations less than 2 M. Finally, trypsin was added at a 1:50 trypsin-to-protein mass ratio for the first digestion overnight and a 1:100 trypsin-to-protein mass ratio for a second 4 h-digestion.  After trypsin digestion, the peptide was desalted by a Strata X C18 SPE column (Phenomenex, USA) and vacuum-dried. The peptide was reconstituted in 0.5 M TEAB and processed according to the manufacturer's protocol for the TMT kit. The peptide mixtures were then incubated for 2 h at room temperature and pooled, desalted and dried by vacuum centrifugation.

### Data Protocol
The tryptic peptides were dissolved in 0.1% formic acid (solvent A) and directly loaded onto a homemade reversed-phase analytical column (15 cm length, 75 μm i.d.). The gradient consisted of an increase from 6% to 23% solvent B (0.1% formic acid in 98% acetonitrile) over 26 min, 23%-35% in 8 min and climbing to 80% in 3 min then holding at 80% for the last 3 min, all at a constant flow rate of 400 nL/min on an EASY-nLC 1000 UPLC system (Thermo, Waltham, USA).  The peptides were subjected to an NSI source followed by tandem mass spectrometry (MS/MS) in a Q ExactiveTM Plus (Thermo) coupled online to UPLC. The electrospray voltage applied was 2.0 kV. The m/z scan range was 350 to 1.800 for full scan, and intact peptides were detected in the Orbitrap at a resolution of 70,000. Peptides were then selected for MS/MS using the NCE setting of 28, and the fragments were detected in the Orbitrap at a resolution of 17,500. A data-dependent procedure was employed that alternated between one MS scan followed by 20 MS/MS scans with 15.0 s dynamic exclusion. Automatic gain control (AGC) was set at 5E4. Fixed first mass was set as 100 m/z.  The resulting MS/MS data were processed using the MaxQuant search engine (v.1.5.2.8). Tandem mass spectra were searched against the human UniProt database concatenated with the reverse decoy database. Trypsin/P was specified as a cleavage enzyme allowing up to 4 missing cleavages. The mass tolerance for precursor ions was set as 20 ppm in the first search and 5 ppm in the main search, and the mass tolerance for fragment ions was set as 0.02 Da. Carbamidomethyl on Cys was specified as a fixed modification and acetylation modification and oxidation on Met were specified as variable modifications. The FDR was adjusted to < 1%, and the minimum score for modified peptides was set as > 40.

### Publication Abstract
None

### Keywords
Rice; low temperature; flooding; proteome; transcriptome; physiological traits.

### Affiliations
Jiangxi Agricultural University

### Submitter
wenxia Wang

### Lab Head
Dr Wenxia wang
Jiangxi Agricultural University


