### Accession
PXD033025

### Title
Cellular proteome and secretome analysis of electrically pulse-stimulated human myotubes

### Description
In this study, we aimed to investigate alterations in secreted proteins as well as expression profile changes in cellular proteome in human skeletal muscle cells that are triggered by a well-established in vitro model of exercise using electrical pulse stimulation (EPS). Here, skeletal muscle cells isolated from muscle biopsies from seven donors were used. The cells from each donor were then exposed to prolonged EPS (i.e. 10 V, 2 ms, 0.1 Hz for 24 h), and non-stimulated cells from the same donors were used as a control.

### Sample Protocol
Proteins in the conditioned media and cell proteins were analyzed from seven EPS-stimulated and unstimulated control myotubes using a global proteomic approach. In brief, the media samples were first concentrated using 10 kDa AmiconÂ® Ultra centrifugal filters (Merck) followed by protein reduction, alkylation, and trypsin (Promega) digestion. The resulting peptide mixtures were purified by the STAGE-TIP method using a C18 resin disk (3 M Empore) before the samples were analyzed by a nanoLC-MS/MS using nEASY-LC coupled to QExacitvePlus (Thermo Electron, Bremen, Germany) using a 60 min LC separation gradient.  Cultured myotubes were harvested in Laemmli buffer (0.5 M Tris-HCl, 10% SDS and 20% glycerol). All samples were analyzed prepared using a Protein Aggregation Capture on Microparticles method, described previously (PMID: 30833379) followed by protein reduction, alkylation, and trypsin digestion. The resulting tryptic peptides were analyzed using EvoSepOne coupled to timsTOF Flex (Bruker) using 88 min separation gradient.

### Data Protocol
The resulting LC-MS/MS files from secreted proteins and cell proteome analysis were processed with MaxQuant version 1.6.1.0 and 1.6.17.0, respectively, for protein identification and lablel-free quantification (LFQ) using default settings. The searches were done against the human Uniprot Reference Proteome database (September 2018 and December 2020).

### Publication Abstract
Electrical pulse stimulation (EPS) has proven to be a useful tool to interrogate cell-specific responses to muscle contraction. In the present study, we aimed to uncover networks of signaling pathways and regulatory molecules responsible for the metabolic effects of exercise in human skeletal muscle cells exposed to chronic EPS. Differentiated myotubes from young male subjects were exposed to EPS protocol 1 (i.e. 2&#xa0;ms, 10&#xa0;V, and 0.1&#xa0;Hz for 24&#xa0;h), whereas myotubes from middle-aged women and men were exposed to protocol 2 (i.e. 2&#xa0;ms, 30&#xa0;V, and 1&#xa0;Hz for 48&#xa0;h). Fuel handling as well as the transcriptome, cellular proteome, and secreted proteins of EPS-treated myotubes from young male subjects were analyzed using a combination of high-throughput RNA sequencing, high-resolution liquid chromatography-tandem mass spectrometry, oxidation assay, and immunoblotting. The data showed that oxidative metabolism was enhanced in EPS-exposed myotubes from young male subjects. Moreover, a total of 81 differentially regulated proteins and 952 differentially expressed genes (DEGs) were observed in these cells after EPS protocol 1. We also found 61 overlapping genes while comparing the DEGs to mRNA expression in myotubes from the middle-aged group exposed to protocol 2, assessed by microarray. Gene ontology (GO) analysis indicated that significantly regulated proteins and genes were enriched in biological processes related to glycolytic pathways, positive regulation of fatty acid oxidation, and oxidative phosphorylation, as well as muscle contraction, autophagy/mitophagy, and oxidative stress. Additionally, proteomic identification of secreted proteins revealed extracellular levels of 137 proteins were changed in myotubes from young male subjects exposed to EPS protocol 1. Selected putative myokines were measured using ELISA or multiplex assay to validate the results. Collectively, our data provides new insight into the transcriptome, proteome and secreted proteins alterations following <i>in vitro</i> exercise and is a valuable resource for understanding the molecular mechanisms and regulatory molecules mediating the beneficial metabolic effects of exercise.

### Keywords
Electrical pulse stimulation, Exercise, Oxidative metabolism, Glycolysis, Skeletal muscle

### Affiliations
University of Oslo
Head of Proteomics

### Submitter
Tuula Nyman

### Lab Head
Dr Tuula Nyman
Head of Proteomics


