### Accession
PXD014640

### Title
ER-shaping Atlastin proteins differentially contribute to flavivirus replication and assembly

### Description
Flaviviruses, including Dengue virus (DV) and Zika virus, extensively remodel the cellular endomembrane network to generate replication organelles that promote viral genome replication and virus production. However, it remains unclear how these membranes and associated cellular proteins act during the virus cycle. Here, we show that atlastins (ATLs), a subset of ER resident proteins involved in neurodegenerative diseases, have dichotomous effects on flaviviruses with ATL2 depletion leading to replication organelle defects and ATL3 depletion to changes in virus production pathways. We characterized non-conserved functional domains in ATL paralogues and show that the ATL interactome is profoundly reprogrammed upon DV infection. Screen analysis confirmed non-redundant ATL functions and identified a specific role for ATL3, and its interactor ARF4, in vesicle trafficking and virion maturation. Our data identify ATLs as central hubs targeted by flaviviruses to establish their replication organelle and to achieve efficient virion maturation and secretion.

### Sample Protocol
For global proteomic analysis of stable cell lines used for affinity-purification, 4 independent samples were prepared for each cell line. Cells were lysed in 4% SDS-TrisCl (pH=8), homogenized by two cycles of heating (98 °C, 5’) and sonication (15’), and clarified and normalized cell lysates were acetone precipitated twice). Proteins were denatured by incubation in 40 µl U/A buffer (8 M Urea, 100 mM Tris-HCl pH 8.5), and reduction and alkylation carried out with 10 mM DTT and 55 mM iodoacetamide in 50 mM ABC buffer (50 mM NH4HCO3 in water pH 8.0), respectively. After digestion with 1 µg LysC (WAKO Chemicals USA) at room temperature for 3 h, the suspension was diluted in ABC buffer, and the protein solution was digested with trypsin (Promega) overnight at room temperature. Peptides were purified on stage tips with 3 C18 Empore filter discs (3M) and analyzed by liquid chromatography coupled to mass spectrometry on an Orbitrap XL instrument (Thermo Fisher Scientific) as described previously.

### Data Protocol
Raw mass-spectrometry data were processed with MaxQuant software versions 1.5.6.2 using the built-in Andromeda search engine to search against the human proteome (UniprotKB release 2015_08 including isoforms and unreviewed sequences) containing forward and reverse sequences plus the DV polyprotein (DV-2 strain 16681, UniprotID P29990 with the individual viral cleavage products manually annotated), and the label-free quantitation algorithm as described previously. Additionally, the intensity-based absolute quantification (iBAQ) algorithm and Match Between Runs option were used. In MaxQuant, carbamidomethylation was set as fixed and methionine oxidation and N-acetylation as variable modifications, using an initial mass tolerance of 6 ppm for the precursor ion and 0.5 Da for the fragment ions. Search results were filtered with a false discovery rate (FDR) of 0.01 for peptide and protein identifications.  Perseus software version 1.6.2.2 was used to further process the proteomic datasets. Protein tables were filtered to eliminate the identifications from the reverse database and common contaminants. In analyzing mass spectrometry data, only proteins identified on the basis of at least one peptide and a minimum of 3 quantitation events in at least one experimental group were considered. For the proteomic dataset, label-free quantification (LFQ) protein intensity values were log-transformed and missing values filled by imputation with random numbers drawn from a normal distribution calculated for each sample. Significantly enriched or depleted proteins were determined by Welch’s t-tests with permutation-based false discovery rate statistics. We performed 250 permutations and the FDR threshold was set at 0.05. The parameter S0 was set at 1 to separate background from specifically enriched interactors.

### Publication Abstract
Flaviviruses, including dengue virus and Zika virus, extensively remodel the cellular endomembrane network to generate replication organelles that promote viral genome replication and virus production. However, it remains unclear how these membranes and associated cellular proteins act during the virus cycle. Here, we show that atlastins (ATLs), a subset of ER resident proteins involved in neurodegenerative diseases, have dichotomous effects on flaviviruses-with ATL2 depletion leading to replication organelle defects, and ATL3 depletion to changes in virus production pathways. We characterized non-conserved functional domains in ATL paralogues and show that the ATL interactome is profoundly reprogrammed following dengue virus infection. Screen analysis confirmed non-redundant ATL functions and identified a specific role for ATL3, and its interactor ARF4, in vesicle trafficking and virion maturation. Our data identify ATLs as central hubs targeted by flaviviruses to establish their replication organelle and to achieve efficient virion maturation and secretion.

### Keywords
Atl-2, Atl-3, Lfq, A549, Lc-ms/ms, Proteome, Calnexin, Label-free

### Affiliations
Systems Arbovirology, Leibniz Institute of Virology
Department of Infectious Diseases, Molecular Virology, Heidelberg University, Im Neuenheimer Feld 344, Heidelberg, Germany

### Submitter
Pietro Scaturro

### Lab Head
Dr Ralf Bartenschlager
Department of Infectious Diseases, Molecular Virology, Heidelberg University, Im Neuenheimer Feld 344, Heidelberg, Germany


