### Accession
PXD044936

### Title
Integrated small RNA, mRNA and protein omics reveal a miRNA network orchestrating metabolic maturation of the developing human heart

### Description
As the fetal heart develops, cardiomyocyte proliferation potential decreases while fatty acid oxidative capacity increases, a highly regulated transition known as cardiac maturation. Small noncoding RNAs, such as microRNAs (miRNAs), contribute to the establishment and control of tissue-specific transcriptional programs. However, small RNA expression dynamics and genome wide miRNA regulatory networks controlling maturation of the human fetal heart remain poorly understood. Transcriptome profiling of small RNAs revealed the temporal expression patterns of miRNA, piRNA, circRNA, snoRNA, snRNA and tRNA in the developing human heart between 8 and 19 weeks of gestation. Our analysis revealed that miRNAs were the most dynamically expressed small RNA species throughout mid-gestation. Cross-referencing differentially expressed miRNAs and mRNAs predicted 6,200 mRNA targets, 2134 of which were upregulated and 4066 downregulated as gestation progresses. Moreover, we found that downregulated targets of upregulated miRNAs predominantly control cell cycle progression, while upregulated targets of downregulated miRNAs are linked to energy sensing and oxidative metabolism. Furthermore, integration of miRNA and mRNA profiles with proteomes and reporter metabolites revealed that proteins encoded in mRNA targets, and their associated metabolites, mediate fatty acid oxidation and are enriched as the heart develops.This study revealed the small RNAome of the maturing human fetal heart. Furthermore, our findings suggest that coordinated activation and repression of miRNA expression throughout mid-gestation is essential to establish a dynamic miRNA-mRNA-protein network that decreases  cardiomyocyte proliferation potential while increasing the oxidative capacity of the maturing human fetal heart.

### Sample Protocol
We used 6 whole frozen ventricles from 10- and 18- weeks fetal hearts (n=3; n=3). To achieve efficient lysis and protein extraction from the tissue samples, ventricles were minced, washed with PBS, and homogenized in RIPA lysis and extraction buffer (Thermo Fisher Scientific) followed by mechanical lysis through sonication for a total of 3 minutes and then centrifugation at 25,000g for 1 hour at 4°C to remove DNA, RNA, and other cell debris. Protein supernatants were retained, and proteins were digested following an in solution tryptic digestion protocol.The samples were sent for Liquid chromatography coupled with tandem mass spectrometry (MS/MS) analysis and were analyzed using a nanoLC Q-Exactive Mass Spectrometer with C18 column (Thermo Scientific, Waltham, MA, USA). Water and acetonitrile containing 0.1% formic acid were used as eluents. Each sample was analyzed independently for 120 minutes.

### Data Protocol
Raw peaks were pre-processed (raw signals exacting, data baselines filtering, peak identification, and integration) using GPMDB (Global Proteome Machine and Database) with x!Tandem algorithm.Common proteins identified across all samples were retained and P value < 0.05 was considered significant. Differentially abundant proteins were compared against differentially expressed mRNAs identified as miRNA targets. mRNA-protein pairs were considered significant if differential expression directionality was the same and P value < 0.05.

### Publication Abstract
None

### Keywords
Lc-msms, Human fetal heart

### Affiliations
The Hospital for Sick Children
Translational Medicine, The Hospital for Sick Children, 686 Bay Street, Toronto, Ontario, Canada, M5G0A4

### Submitter
Paul Delgado Olguin

### Lab Head
Dr Paul Delgado-Olguín
Translational Medicine, The Hospital for Sick Children, 686 Bay Street, Toronto, Ontario, Canada, M5G0A4


