### Accession
PXD015947

### Title
A Non-interventional Clinical Trial Assessing Immune Responses After Radiofrequency Ablation of Liver Metastases From Colorectal Cancer

### Description
HLA ligandome analysis of colorectal adenocarcinoma tissues obtained from liver metastases (mCRC) as well as adjacent non-malignant liver tissues was performed to characterize the natural HLA class I and class II presented ligands on both mCRC and adjacent liver tissue.

### Sample Protocol
Immunopreciptiation of HLA class I and II molecules was performed by described previously standard methods (Kowalewski et al. Methods Mol Biol. 2013; 960:145-157.). For HLA class I immunoprecipitation, pan-HLA class I-specific mAb W6/32 (Barnstable et al. Cell. 1978; 14(1):9-20.) was employed and for HLA class II pan-HLA class II mAb Tü39 (Ziegler et al. Immunobiology. 1986; 171(1-2):77-92.) as well as HLA-DR-specific mAb L243 (Goldman et al. Br J Haematol. 1982; 52(3):411-20.) were used and subsequently eluted in each case, using 0.2% trifluoroacetic acid. Peptide extracts were separated by nanoflow uHPLC using a 50μm × 25 cm column (PepMap RSLC, ThermoFisher) and an acetonitrile gradient ranging from 2.4 to 32.0% over the course of 90 min. Samples were analyzed in up to six technical replicates of each sample, as previously described (Löffler et al. Cancer Res. 2018; 78(16):4627-4641. doi: 10.1158/0008-5472.CAN-17-1745). Eluting peptides were analyzed in an online coupled linear trap quadrupole (LTQ) Orbitrap XL mass spectrometer (ThermoFisher), equipped with a nanoelectron spray ion source, employing a top 5 collision-induced dissociation (CID) fragmentation method. Sample weight of tissues employed for lysis and HLA ligand isolation is provided in Table 1 in (Löffler et al. Front. Immunol. 2019; 10:2526. doi: 10.3389/fimmu.2019.02526).

### Data Protocol
The Mascot search engine (Mascot 2.2.04, Matrix Science, Boston, MA) was used to search the human proteome contained in the Swiss-Prot database (20,279 reviewed protein sequences, as of September 2013) without any enzymatic restriction (required Mascot ion score ≥20; search engine rank: 1). As a dynamic modification oxidized methionine was allowed. The false discovery rate was estimated with the Percolator algorithm (Käll et al. Nat Methods. 2007; 4(11):923-5. https://doi.org/10.1038/nmeth1113) and set to 5%. Peptide lengths for HLA class I-eluted peptides were limited from 8 to 12 amino acids (required charge state: 2–3) and for HLA class II-eluted peptides from 9 to 25 amino acids (required charge state: 2–5). Protein inference was disabled, allowing for multiple protein annotations of peptides. Files are labelled according to identifiers (IRISS) given in Table 1 in (Löffler et al. Front. Immunol. 2019; 10:2526. doi: 10.3389/fimmu.2019.02526). Files are categorized according to assessed sample materials as Tumor (metastatic colorectal adenocarcinoma; bulk tissue; malignant), NMT (liver; bulk tissue; non-malignant). Technical replicates are labelled as Rep#.

### Publication Abstract
<b>Background:</b> Radiofrequency ablation (RFA) is an established treatment option for malignancies located in the liver. RFA-induced irreversible coagulation necrosis leads to the release of danger signals and cellular content. Hence, RFA may constitute an endogenous <i>in situ</i> tumor vaccination, stimulating innate and adaptive immune responses, including tumor-antigen specific T cells. This may explain a phenomenon termed abscopal effect, namely tumor regression in untreated lesions evidenced after distant thermal ablation or irradiation. In this study, we therefore assessed systemic and local immune responses in individual patients treated with RFA. <b>Methods:</b> For this prospective clinical trial, patients with liver metastasis from colorectal carcinoma (mCRC) receiving RFA and undergoing metachronous liver surgery for another lesion were recruited (<i>n</i> = 9) during a 5-year period. Tumor and non-malignant liver tissue samples from six patients were investigated by whole transcriptome sequencing and tandem-mass spectrometry, characterizing naturally presented HLA ligands. Tumor antigen-derived HLA-restricted peptides were selected by different predefined approaches. Further, candidate HLA ligands were manually curated. Peripheral blood mononuclear cells were stimulated <i>in vitro</i> with epitope candidate peptides, and functional T cell responses were assessed by intracellular cytokine staining. Immunohistochemical markers were additionally investigated in surgically resected mCRC from patients treated with (<i>n</i> = 9) or without RFA (<i>n</i> = 7). <b>Results:</b> In all six investigated patients, either induced immune responses and/or pre-existing T cell immunity against the selected targets were observed. Multi-cytokine responses were <i>inter alia</i> directed against known tumor antigens such as cyclin D1 but also against a (predicted) mutation contained in ERBB3. Immunohistochemistry did not show a relevant influx of immune cells into distant malignant lesions after RFA treatment (<i>n</i> = 9) as compared to the surgery only mCRC group (<i>n</i> = 7). <b>Conclusions:</b> Using an individualized approach for target selection, RFA induced and/or boosted T cell responses specific for individual tumor antigens were more frequently detectable as compared to previously published observations with well-characterized tumor antigens. However, the witnessed modest RFA-induced immunological effects alone may not be sufficient for the rejection of established tumors. Therefore, these findings warrant further clinical investigation including the assessment of RFA combination therapies e.g., with immune stimulatory agents, cancer vaccination, and/or immune checkpoint inhibitors.

### Keywords
Hla-ligandomics, Colon cancer, Colorectal adenocarcinoma, Hla-restricted peptide, Hla-ligand, Colon, Metastasis, Neoplasia, Hla, Peptide, Cancer

### Affiliations
1 Department of Immunology, Interfaculty Institute for Cell Biology, Tübingen, Germany
2 German Cancer Consortium (DKTK), DKFZ Partner Site Tübingen, Tübingen, Germany
University of Tübingen, Interfaculty Institute for Cell Biology, Department of Immunology, Auf der Morgenstelle 15, 72076 Tuebingen, Germany

### Submitter
Lena Mühlenbruch

### Lab Head
Dr Hans-Georg Rammensee
University of Tübingen, Interfaculty Institute for Cell Biology, Department of Immunology, Auf der Morgenstelle 15, 72076 Tuebingen, Germany


