### Accession
PXD021366

### Title
Phosphoproteome  profiling uncovers a key role for CDKs in TNF signaling

### Description
Tumor  necrosis factor (TNF) is one of the few cytokines successfully  targeted by therapies against inflammatory diseases. However, blocking  this well studied and pleiotropic ligand can cause dramatic  side-effects. We reasoned that a systems-level proteomic analysis of TNF  signaling could dissect its diverse functions and offer  a base for developing more targeted therapies. Combining  phosphoproteomics time course experiments with spatial and kinase  inhibitor analysis clusters phosphorylations into functional modules.  The majority of regulated phosphorylations could be assigned to  an upstream kinase by inhibiting master kinases and spatial proteomics  revealed phosphorylation-dependent translocations of hundreds of  proteins upon TNF stimulation. Phosphoproteome analysis of TNF-induced  apoptosis and necroptosis revealed a key role for  transcriptional cyclin-dependent kinase (CDK) activity to promote  cytokine production and prevent excessive cell death downstream of the  TNF signaling receptor. Our comprehensive interrogation of TNF induced pathways and sites can be explored at http://tnfviewer.biochem.mpg.de/.

### Sample Protocol
We applied the Easy Phos protocol (Humphrey et al., 2015, Humphrey et al., 2018) to enrich for phosphopeptides. In short, 10 x 106 U937 cells or one full 15 cm dish of BMDMs were stimulated washed with ice cold TBS and lysed in 2% sodium deoxycholate (SDC) and 100 mM Tris-HCl pH 8.5 and boiled immediately. After sonication protein amounts were adjusted to 1 mg before reduction (10 mM tris(2-carboxy(ethyl)phosphine (TCEP)), alkylation (40 mM 2-chloroacetamide (CAA)) and digestion with Trypsin and LysC (1:100, enzyme/protein, w/w) overnight. For proteome measurements 20 µg of peptide was taken and desalted using SDB-RPS stage tips. 500 ng of desalted peptides were resolubilized in 5 µl 2% ACN and 0.3% TFA and injected into the mass spectrometer. For the phosphoenrichment isopropanol (final conc. 50%), trifluoroacetic acid (TFA, final conc. 6 %) and monopotassium phosphate (KH2PO4, final conc. 1mM) were added to the rest of the digested lysate. Lysates were shaken then spun for 3 min at 2000 x g and supernatants were incubated with TiO2 beads for 5 minutes at 40 °C (1:10, protein/beads, w/w). Beads were washed 5 times with isopropanol and 5% TFA and phosphopeptides were eluted off the beads with 40% acetonitrile (ACN) and 15% of ammoniumhydroxide (25% NH4OH) on C8 stage tips. After 20 minutes of SpeedVac at 45 °C phosphopeptides were desalted on SDB-RPS stage tips and resolubilized in 5 µl 2% ACN and 0.3% TFA and injected in the mass spectrometer.

### Data Protocol
For the experiment measured in DDA mode MS raw files were processed by the MaxQuant version 1.5.38 (Cox and Mann, 2008) and fragments lists were searched against the human uniprot FASTA database (21,039 entries, 2015) (Cox et al., 2011) with cysteine carbamidomethylation as a fixed modification and N-terminal acetylation and methionine oxidations as variable modifications. We set the false discovery rate (FDR) to less than 1% at the peptide and protein levels and specified a minimum length of 7 amino acids for peptides. Enzyme specificity was set as C-terminal to Arginine and Lysine as expected using Trypsin and LysC as proteases and a maximum of two missed cleavages.  For experiments measured in DIA mode MS raw files were processed by the Spectronaut software version 13 (Biognosys, (Bruderer et al., 2015)). First, hybrid libraries were generated in Spectronaut Pulsar by combining the DDA runs of fractionated samples of either proteome or phosphoproteome (for the phosphoproteome samples were treated with TNF and the phosphatase inhibitors sodium orthovanadate (1 mM) and calyculin A (50 ng/ml)) with the DIA runs of the respective experiments. Thereby we used the human (21,039 entries, 2015) or the mouse uniport FASTA databases (22,216 entries, 2015) as forward databases. To generate phosphoproteome libraries Serine/Threonine/Tyrosine phosphorylation was added as variable modification to the default settings which include cysteine carbamidomethylation as fixed modification and N-terminal acetylation and methionine oxidations as variable modifications. Maximum number of fragment ions per peptide was increased from 6 to 15. The false discovery rate (FDR) was set to less than 1% at the peptide and protein levels and a minimum length of 7 amino acids for peptides was specified. Enzyme specificity was set as C-terminal to Arginine and Lysine as expected using Trypsin and LysC as proteases and a maximum of two missed cleavages. To generate proteome libraries default settings were used. The experimental DIA runs were then analysed against the hybrid library by using default settings for the analysis of the proteome and for the analysis of the phosphoproteome samples the localization cutoff was set to 0.

### Publication Abstract
None

### Keywords
Transcription, Phosphoproteomics, Necroptosis, Spatial proteomics, Cell death, Cyclin-dependent kinases, Kinases, Tnf, Apoptosis

### Affiliations
Department of Proteomics and Signal Transduction Max Planck Institute of Biochemistry Germany
Proteomics

### Submitter
Mario Oroshi

### Lab Head
Dr Matthias Mann
Department of Proteomics and Signal Transduction Max Planck Institute of Biochemistry Germany


