### Accession
PXD036785

### Title
Multi-Omics analysis revealed a significant alteration of critical metabolic pathways due to sorafenib-resistance in Hep3B cell lines

### Description
Hepatocellular carcinoma (HCC) is the second prominent cause of cancer-associated death worldwide. Usually, HCC is diagnosed in advanced stages, where sorafenib, a multiple target ty-rosine kinase inhibitor, is used as the first line of treatment. Unfortunately, resistance to sorafenib is usually encountered within six months of the treatment. Therefore, there is a critical need to identify the underlying reasons for drug resistance. In the present study, we investigated the proteomic and metabolomics alterations accompanying to sorafenib resistance in hepatocellular carcinoma Hep3B cells by employing ultra-high-performance liquid chromatography quadrupole time of flight mass spectrometry (UHPLC-QTOF-MS). The Bruker Human Metabolome Database (HMDB) library was used to identify the differentially abundant metabolites through MetaboScape 4.0 software (Bruker). For protein annotation and identification, the Uniprot proteome for Homo sapiens (Human) da-tabase was utilised through MaxQuant. The results revealed that 27 metabolites and 18 proteins were significantly dysregulated due to sorafenib resistance in Hep3B cells compared to the parental phenotype. D-alanine, L-proline, o-tyrosine, succinic acid and phosphatidylcholine (PC, 16:0/16:0) were among the significantly altered metabolites. Ubiquitin carboxyl-terminal hydrolase isozyme L1, mitochondrial superoxide dismutase, UDP-glucose-6-dehydrogenase, sorbitol dehydrogenase and calpain small subunit 1 were among the significantly altered proteins. The findings revealed that resistant Hep3B cells demonstrated significant alterations in amino acid and nucleotide met-abolic pathways, energy production pathways and other pathways related to cancer aggressive-ness, migration, proliferation, and drug-resistance. Joint pathway enrichment analysis unveiled unique pathways, including the antifolate resistance pathway and other important pathways that maintain cancer cells' survival, growth, and proliferation. Collectively, the results identified po-tential biomarkers for sorafenib-resistant HCC and gave insights into their role in chemotherapeutic drug resistance, cancer initiation, progression, and aggressiveness, which may contribute to better prognosis and chemotherapeutic outcomes.

### Sample Protocol
Metabolite extraction: For metabolomics, three biological replicates were used. An equal number of cells were utilized for each sample. The cells were centrifuged separately at 15000 rpm for 10 min to separate the cells and cell-free supernatants. The supernatants were dis-carded, and to each cell pellet (~3x 106), 1 mL of 0.1% formic acid in methanol was added, followed by vertexing for 2 min x 4, interrupted by 15 min incubation on ice. This is followed by sonication using a COPLEY probe-sonicator (QSONICA SONI-CATOR, USA) for 30 seconds with 30% amplifier in an ice bath. Subsequently, the cel-lular debris of each sample was collected by centrifugation at 15000 rpm for 10 min at 4Â°C. The supernatants were separated and dried at 37 Â± 1 Â°C using EZ-2 Plus (Gene-Vac-Ipswich, UK). Finally, 200 ÂµL of 0.1% formic acid in water was used to resuspend the dried samples, which were then vortexed for 2 min, filtered by a 0.45Âµm hydro-philic Nylon Syringe Filter and transferred to 200 Î¼L (micro-inserts) in LC vials. The quality control (QC) sample was prepared by pooling the same volume (10 Î¼L) of each sample, and all samples were placed in the autosampler at a temperature set at 4â„ƒ and analyzed with UHPLC-QTOF-MS.

### Data Protocol
To identify the proteins and peptides, the raw data were processed by way of MaxQuant 1.6.17.0 19 using the Uniprot proteome for Homo sapiens (Proteome ID: UP000005640, 79,052 entries, 14/04/2022) and the Andromeda search engine. Default parameters were applied in the MS/MS database search, including methionine oxidation and acetylation of protein N-termini as variable modifications and carbamidomethylation of cysteine residues as a fixed modification. Peptide spectral matching (PSMs) was filtered with a 20-ppm precursor mass tolerance and 1% false discovery rate (FDR). For label-free quantification (LFQ), the MaxLFQ algorithm was used. In the in-silico digestion, the default trypsin/P enzymatic cleavage rule was utilised. To ensure a reliable label-free quantification, the protein groups were identified with at least two peptides and were selected for further downstream analysis.

### Publication Abstract
Hepatocellular carcinoma (HCC) is the second prominent cause of cancer-associated death worldwide. Usually, HCC is diagnosed in advanced stages, wherein sorafenib, a multiple target tyrosine kinase inhibitor, is used as the first line of treatment. Unfortunately, resistance to sorafenib is usually encountered within six months of treatment. Therefore, there is a critical need to identify the underlying reasons for drug resistance. In the present study, we investigated the proteomic and metabolomics alterations accompanying sorafenib resistance in hepatocellular carcinoma Hep3B cells by employing ultra-high-performance liquid chromatography quadrupole time of flight mass spectrometry (UHPLC-QTOF-MS). The Bruker Human Metabolome Database (HMDB) library was used to identify the differentially abundant metabolites through MetaboScape 4.0 software (Bruker). For protein annotation and identification, the Uniprot proteome for Homo sapiens (Human) database was utilized through MaxQuant. The results revealed that 27 metabolites and 18 proteins were significantly dysregulated due to sorafenib resistance in Hep3B cells compared to the parental phenotype. D-alanine, L-proline, o-tyrosine, succinic acid and phosphatidylcholine (PC, 16:0/16:0) were among the significantly altered metabolites. Ubiquitin carboxyl-terminal hydrolase isozyme L1, mitochondrial superoxide dismutase, UDP-glucose-6-dehydrogenase, sorbitol dehydrogenase and calpain small subunit 1 were among the significantly altered proteins. The findings revealed that resistant Hep3B cells demonstrated significant alterations in amino acid and nucleotide metabolic pathways, energy production pathways and other pathways related to cancer aggressiveness, such as migration, proliferation and drug-resistance. Joint pathway enrichment analysis unveiled unique pathways, including the antifolate resistance pathway and other important pathways that maintain cancer cells' survival, growth, and proliferation. Collectively, the results identified potential biomarkers for sorafenib-resistant HCC and gave insights into their role in chemotherapeutic drug resistance, cancer initiation, progression and aggressiveness, which may contribute to better prognosis and chemotherapeutic outcomes.

### Keywords
Human, Lc-msms, Drug-resistance, Timstof, Cells

### Affiliations
Sharjah Institute for Medical Research, University of Sharjah
1 - College of Pharmacy, University of Sharjah, Sharjah 2 - Sharjah Institute for Medical Research, University of Sharjah, Sharjah

### Submitter
Core Facility

### Lab Head
Dr Mohammad H Semreen
1 - College of Pharmacy, University of Sharjah, Sharjah 2 - Sharjah Institute for Medical Research, University of Sharjah, Sharjah


