### Accession
PXD018617

### Title
Allosteric HSP70 inhibitors perturb mitochondrial proteostasis and overcome proteasome inhibitor resistance in multiple myeloma, dataset 2

### Description
Proteasome inhibitor (PI) resistance remains a central challenge in multiple myeloma. To identify pathways mediating this resistance, we mapped genetic co-dependencies associated with the proteasome. These studies identified cytosolic heat shock protein 70 (HSP70) chaperones as a potential target, mirroring recent studies that have shown mechanism of overcoming PI-induced stress. Here, we first underscore this relationship by mapping genetic co-dependencies in cancer proteostasis. These results lead us to explore HSP70 inhibitors as potential therapeutics. We show these compounds exhibit increased efficacy against both acquired and intrinsic PI-resistant myeloma models, unlike HSP90 inhibition. Surprisingly, shotgun and pulsed-SILAC proteomics reveal that JGâ€™s overcome PI resistance not via the expected mechanism of inhibiting cytosolic HSP70s, but instead through mitochondrial-localized HSP70, HSPA9, destabilizing the 55S mitoribosome. Analysis of myeloma patient data further supports strong effects of global proteostasis capacity, and particularly HSPA9 expression, on response to PI. Our results characterize dynamics of myeloma proteostasis networks under therapeutic pressure while further motivating investigation of HSPA9 as a specific target in PI-resistant disease.  This dataset corresponds to experiment outlined in figure 4a.

### Sample Protocol
For multiplexed TMT drug experiments (Fig 4a), 5e6 cells were seeded in 5 mLs in 6 well plates and treated with compounds as follows: MM1.S cells were treated with 1.75uM JG98, 2.5nM bortezomib, 1uM CB-5083, 150nM DMAG, or DMSO. RPMI-8226 cells were treated with 1.5uM JG98, 7.5nM bortezomib, 1uM CB-5083, 200nM DMAG, or DMSO. KMS34 cells were treated with 2uM JG98, 7.5nM Bortezomib, 1uM CB-5083, 500nM DMAG, or DMSO. Cells were collected after 22 hours of drug treatment, washed with PBS, snap frozen in liquid nitrogen, and stored at -80C. Cell pellets were lysed in 6M GdnHCL, 0.1M Tris pH 8.5, with 5mM TCEP and 10mM 2-Chloro Acetamide. Lysates were sonicated for 45 seconds at 1 Hz cycles on ice and cleared by centrifugation at 16000g for 10 minutes at 4C. Protein concentration was measured with 660 assay (Pierce 22660) and 100 ug protein was digested with 2 ug Trypsin (Pierce 90057) for 18-22 hours with digestion halted by addition of TFA to 1% vol/vol. Acidified samples were centrifuged at 17,200g for 5 minutes at RT to remove precipitate. Cleared samples were desalted on SOLA columns as per manufacturer instructions (Thermo 60109001), eluted in 50% Acetonitrile with 0.1% FA prior and vacuum dried prior to storage at -80C. Miniaturized TMT labeling was performed based on modified protocol from Zecha et al (PMC6601210). Briefly, peptide concentrations were measured using the Pierce Peptide Colormetric Assay (Pierce 23275). 20 ug peptides were resuspended in 17.5 uL 50 mM HEPES pH 8.5 and labeled with 50 ug TMT reagent dissolved in 2.5 uL Acetonitrile for 1 hour at 25C and 500 rpm. Reactions were quenched by adding hydroxylamine to final concentration of 0.4% and incubation for 15 min at 500 rpm. TMT labeled samples were combined, acidified by addition of 0.1% TFA, vacuum dried, and stored at -80C. For 10-plex TMT shotgun experiments, peptides were fractionated using a High pH Reversed-Phase Fractionation kit (Pierce, 84868), as previously described in Welle et al (27765818). 1 ug of peptides were injected into a Dionex Ultimate 3000 NanoRSLC instrument with 15-cm Acclaim PEPMAP C18 column (Thermo, 164534) reverse phase column coupled to a Thermo Q Exactive Plus. HPLC non-linear gradient was as follows with buffer A 0.1% FA and buffer B 0.1% FA in 80% Acetonitrile: 3-8% B for 11 minutes, 8-34% B for 80 minutes, 34-50% B for 15 minutes, 50-70% B for 5 minutes with hold at 70% for 3 minutes, and 99% B column wash for 4 minutes prior to re-equilibration for 13 minutes. For MS1 acquisition, spectra were collected in data dependent top 15 method with full MS scan resolution of 70,000, AGC target was set to 3e6, and maximum IT set to 50ms. For MS2 acquisition, resolution was set to 35,000, AGC set to 1e5, and maximum IT to 100ms with Normalized Collison energy of 32.

### Data Protocol
Mass spectrometry data was processed in Maxquant version 1.6.2.1 with the following settings: PSM/Protein FDR were set to 0.01, Carbidomethylation was set as fixed modification and methionine oxidation and N-terminal acetylation were set as variable modifications,  minimum peptide length = 7, matching time window set to 0.7 min, alignment time window set to 20 min, and match between runs was used, along with other default settings.  Data was searched against the Uniprot Swiss-Prot human proteome (ID:9606, downloaded from Uniprot in 2018).

### Publication Abstract
None

### Keywords
Hsp70, Multiple myeloma, Tmt

### Affiliations
Stanford University
Department of Laboratory Medicine, Wiita Lab, University of California, San Francisco, USA

### Submitter
Ian Ferguson

### Lab Head
Dr Arun Wiita
Department of Laboratory Medicine, Wiita Lab, University of California, San Francisco, USA


