### Accession
PXD003179

### Title
Untargeted data independent acquisition proteomics analysis using Orbitrap mass spectrometers and DIA-Umpire

### Description
Untargeted data independent acquisition proteomics analysis using Orbitrap mass spectrometers and DIA-Umpire

### Sample Protocol
Q Exactive datasets The raw files of two series Q Exactive DIA and DDA datasets were downloaded from PeptideAtlas (http://www.peptideatlas.org, No. PASS00589). One is HEK-293 cells sample and the other is the human liver micro-tissue samples treated by series concentrations of Aacetaminophen (APAP). Each series of sample contains both DDA and DIA experiments. In   this paper we processed DDA and DIA experiments independently, so we define there are four Q Exactive datasets: HEK-293 DDA, HEK-293 DIA, Microtissue DDA, and Microtissue DIA datasets.  Orbitrap Fusion datasets The MS system, Orbitrap Fusion Tribrid mass spectrometer (Thermo Scientific, San Jose, CA), was coupled with an Ultimate 3000 RSLCnano system (Thermo Fisher Scientific). 1ug tryptically predigested HeLa cell (Thermo Scientific, San Jose, CA) were directly loaded onto self-packed column. The 3 um ReproSil-Pur C18-AQ particles (Dr Maisch, Ammerbuch, Germany) were packed into a 30 cm self-pulled column with a 100 um inner diameter and 7 um opening to prepare an analytical column using �tone-arch��frit. The mobile phases consisted of (A) 0.1% formic acid and (B) 0.1% formic acid and acetonitrile. Peptides were separated through a gradient of up to 85% buffer B over 135 minutes at flow rate of 500 nL/min. The MS instrument was operated in the positive ion mode, with an electrospray through a heated ion transfer tube (250 簞C), followed by a stacked ring ion guide (RF-lens) evacuated by a rotary vane pump to ~2 Torr. Full scan MS spectra were acquired in orbitrap (m/z range: 400��250) with the resolution set to 60,000. Full scan target was 3e5 with a maximum fill time of 50 ms. All data were acquired in profile mode using positive polarity. MS/MS spectra of both DDA and DIA data were acquired in orbitrap as well with a resolution of 15,000 and higher-collisional dissociation (HCD) MS/MS fragmentation.   For DDA data, up to top 15 most intense ions were selected for MS/MS for each scan cycle.  Target value for fragment scans was set at 1e5 with a maximum fill time of 35 ms and intensity threshold was kept at 2e4. Isolation width was set at 1.4 Th. There were two independent DDA datasets conducted. DIA experiments were run by different isolation window settings. A total of five DIA datasets including 25, 20, 15, 10, and 5 Da SWATH-type of fix isolation window sizes were conducted, which resulted in 2.7, 3.3, 3.9, 6.2, and 13 seconds cycle time, respectively. For each DIA experiment, the target value for fragment scans was set at 1e5 with a maximum fill time of 50 ms. Each DIA or DDA dataset was run in triplicates.

### Data Protocol
DIA-Umpire pseudo MS/MS extraction All .raw files were converted into mzXML format using msconvert.exe with vendor (Xcalibur) peak picking option to generate centroid spectra. The DIA mzXML files were first processed by the signal extraction (SE) module of DIA-Umpire v2.0 to generate pseudo MS/MS spectra in MGF format. For detection of precursor ion signal, the following parameters were used: 10 p.p.m mass tolerance for Orbitrap Fusion datasets and 15 p.p.m for Q Exactive datasets, charge state range from 1+ to 5+ for MS1 precursor ions, 2+ to 5+ for MS2 unfragmented precursor ions. For detection of fragment ions, 20 p.p.m mass tolerance for Orbitrap Fusion datasets and 25 p.p.m for Q Exactive datasets. Signal-to-noise ratio for both precursor and fragment signals was set to 1.1. The maximum retention time range was set to two minutes and maximum of two consecutive gaps is allowed. Note that the signal quality of the centroid spectra generated from Xcalibur library via msconvert.exe was high therefore background detection for noise removal was not used. The MS2 spectra   in the resulting mzXML files contained isolation window ranges already, therefore there was no need to specify isolation setting in the parameter of DIA-Umpire_SE module.  DDA and DIA (pseudo) MS/MS database search The DDA and DIA pseudo MS/MS spectra were searched using X! Tandem, Comet and MSGF+ search engines using the following parameters - allow tryptic peptides only, up to one missed cleavage, methionine oxidation as variable modification and cysteine carbamidomethylation as static modification. The precursor-ion mass tolerance and the fragment-ion mass tolerance were set, respectively, to 10 p.p.m. and 20 p.p.m. for Orbitrap Fusion and to 15 p.p.m. and 25 p.p.m. for Q Exactive dataset. We used the nonredundant human protein sequence FASTA file from the UniProt/Swiss-Prot database (release date: June 19, 2015) appended with corresponding reversed sequences which were considered as decoys for target-decoy analysis. The output files from the search engines were further analyzed by PeptideProphet and combined by iProphet.  False discovery rates estimation for database peptide ion identifications False discovery rate (FDR) of peptide ion identifications was estimated using target-decoy approach based on maximum iProphet probabilities of each peptide ion (peptide sequence, charge state, modification and modification site) in each dataset. If the maximum iProphet probability of a peptide ion passed the desired FDR threshold, then all identifications of same peptide ion across all files within the same dataset were accepted. This strategy generally reduces missing identifications between replicates while maintaining desired FDR in dataset level.

### Publication Abstract
We describe an improved version of the data-independent acquisition (DIA) computational analysis tool DIA-Umpire, and show that it enables highly sensitive, untargeted, and direct (spectral library-free) analysis of DIA data obtained using the Orbitrap family of mass spectrometers. DIA-Umpire v2 implements an improved feature detection algorithm with two additional filters based on the isotope pattern and fractional peptide mass analysis. The targeted re-extraction step of DIA-Umpire is updated with an improved scoring function and a more robust, semiparametric mixture modeling of the resulting scores for computing posterior probabilities of correct peptide identification in a targeted setting. Using two publicly available Q Exactive DIA datasets generated using HEK-293 cells and human liver microtissues, we demonstrate that DIA-Umpire can identify similar number of peptide ions, but with better identification reproducibility between replicates and samples, as with conventional data-dependent acquisition. We further demonstrate the utility of DIA-Umpire using a series of Orbitrap Fusion DIA experiments with HeLa cell lysates profiled using conventional data-dependent acquisition and using DIA with different isolation window widths.

### Keywords
Human, Dia-umpire, Dia

### Affiliations
University of Michigan
University of Michigan, Ann Arbor, MI, US

### Submitter
Chih-Chiang Tsou

### Lab Head
Dr Alexey I. Nesvizhskii
University of Michigan, Ann Arbor, MI, US


