### Accession
PXD019708

### Title
Definition of aberrant MYB CBP/P300 complex in AML

### Description
Dysregulated gene expression is one of the most prevalent features in human cancers. Here, we show that most subtypes of acute myeloid leukemia (AML) depend on the aberrant assembly of the MYB transcriptional co-activator complex. By rapid and selective peptidomimetic interference with the binding of CBP/P300 to MYB, but not CREB or MLL1, we find that the leukemic functions of MYB are mediated by CBP/P300-mediated co-activation of a distinct set of transcriptional factor complexes that are aberrantly assembled with MYB in AML cells. This therapeutic remodeling is accompanied by dynamic redistribution of CBP/P300 complexes to genes that control cellular differentiation and growth. Thus, aberrantly organized transcription factor complexes control convergent gene expression programs in AML cells. These findings establish a compelling strategy for pharmacologic reprogramming of oncogenic gene expression that supports its targeting for leukemias and other human cancers caused by dysregulated gene control.

### Sample Protocol
MV411 cells were collected by centrifugation and nuclei were enriched by sucrose density gradient centrifugation. Nuclear suspensions were homogenized, cleared, diluted and immediately used for MYB, CBP/P300 or IgG control immunoprecipitations using 7.5 ug of antibodies conjugated to 1 mg Epoxy-coated magnetic beads. Eluates were resolved by SDS-PAGE. Proteins were reduced with DTT and alkylated using iodoacetamide, prior to tandem proteolysis with LysC (1:25 protease:sample, assuming 1 microgram of protein per immunoprecipitation. 37C,4h) and trypsin (1:25 w/w, 37C, 16h). Peptides were eluted in 1% formic acid in 70% acetonitrile (30 min, 25C, 1400 rpm), and lyophilized. Peptide pellets were reconstituted in 8 ul of 0.1% formic acid and 3 ul of each sample were used for mass spectrometry analysis. The LC system consisted of a vented trap-elute setup (EasynLC1000, Thermo Fisher scientific) coupled to a Orbitrap Lumos mass spectrometer.

### Data Protocol
Mass spectra were analyzed using MaxQuant (version1.6.0.16, Cox J; Mann M MaxQuant enables high peptide identification rates, individualized p.p.b.-range mass accuracies and proteome-wide protein quantification. Nat. Biotechnol 2008, 26 (12), 1367â€“72)). For identification, spectra were matched against the human UniProt database (as of October 2017), supplemented with contaminant proteins from the cRAP database (Mellacheruvu D; Wright Z; et al. The CRAPome: a contaminant repository for affinity purification-mass spectrometry data. Nat. Methods 2013) with FDR <0.01. After m/z recalibration, mass tolerance was set at 4.5 and 20 ppm for precursor and fragment ions, respectively. Cysteine carbamidomethylation was set as fixed chemical modification, while methionine oxidation and protein N-terminus acetylation were set as variable. Protease specificity was set to trypsin/P, with up to two missed cleavages allowed. The match between runs feature was enabled (0.7 min tolerance, 20 min alignment). Quantification was performed using the LFQ algorithm. Contaminating peptides such as keratin and non-human proteins were excluded and label-free quantification (LFQ) intensity values for protein groups were used for analysis. IgG, MYB and CBP/P300 immunoprecipitation samples were analyzed together (3 replicates per condition; 6 samples total). Proteins in the following categories were excluded from analysis: (i) Zero LFQ intensity across all replicates, (ii) Zero LFQ intensity in 5 of 6 replicates, (iii) Proteins with LFQ intensity recorded in only 1 replicate in both sets of samples. Enrichment scores were defined as the log2 ratio of MYB:IgG and CBP/P300:IgG.

### Publication Abstract
None

### Keywords
Human, Lc-ms, Aml, Myb, Proteomics, Cbp/p300

### Affiliations
Molecular Pharmacology Program, Sloan Kettering Institute
Memorial Sloan Kettering Cancer Center

### Submitter
Alex Kentsis

### Lab Head
Dr Alex Kentsis
Molecular Pharmacology Program, Sloan Kettering Institute


