### Accession
PXD015161

### Title
Proximity labeling of FAMA transcription factor complexes in seedlings of Arabidopsis thaliana using TurboID

### Description
Identifying specific protein interactors and spatially or temporally restricted local proteomes contributes significantly to our understanding of cellular processes in virtually all aspects of life. Obtaining such data is challenging, especially when the protein, cell type or event of interest is rare. In recent years, different proximity labeling techniques have been developed that have greatly improved our ability to tackle these questions. However, while effective in mammalian systems, use in plants has been extremely limited due to technical challenges. Recent technological improvements in the form of two highly active versions of the biotin ligase BirA* (TurboID and miniTurboID), prompted us to test this new system on two challenging but widely asked questions in plants: what are interaction partners of low-abundant proteins and what are organellar proteomes in rare and transient cell types. To address the first question, we used the transcription factor FAMA as a test case. FAMA is a master regulator of guard cell development and promotes terminal differentiation of the guard cell precursor by both activating and repressing hundreds of genes. FAMA-expressing young guard cells are rare and restricted to the epidermis of developing aerial tissues, which makes them a good model system to test TurboID applicability under material-limiting conditions. For this experiment, young Arabidopsis seedlings expressing FAMA-TurboID were treated with biotin to label FAMA complexes. Col-0 wild type and seedlings expressing nuclear TurboID under the FAMA promoter were used as controls for unspecific binding of proteins to the beads and stochastic labeling of nuclear proteins, respectively. Biotinylated proteins were isolated by affinity purification with streptavidin-coupled beads and identified by LC-MS/MS. Analysis of proteins labeled by FAMA-TurboID fusions revealed known interactors of this late stomatal lineage specific transcription factor, as well as novel proteins that could explain its dual function as an activator and repressor. Comparison with proteins obtained from classical co-immunoprecipitation approaches with FAMA showed that proximity labeling is superior for identification of meaningful interaction partners of low-abundant proteins.

### Sample Protocol
Five day old Arabidopsis seedlings expressing FAMA-TurboID-Venus under the FAMA promoter or TurboID-YFP-NLS under the FAMA or UBQ10 promoter, as well as wild-type seedlings, were treated with 50 µM biotin for 0, 0.5 or 3 hours to label proteins in proximity of FAMA (FAMA-TurboID-Venus plants) and in nuclei (TurboID-YFP-NLS plants). Proteins were extracted by resuspending the ground plant material in extraction buffer (50 mM Tris pH 7.5, 150 mM NaCl, 0.1 % SDS, 1 % Triton-X-100, 0.5 % Na-deoxycholate, 1 mM EGTA, 1 mM DTT, 1x complete, 1 mM PMSF), adding 1 μl Lysonase and incubating for 15 minutes at 4°C, sonicating four times for 30 seconds and centrifuging at 15,000 g at 4°C for 15 minutes. The supernatants were buffer exchanged with PD-10 desalting columns to remove excess biotin from the extract and eluted proteins were incubated with Dynabeads MyOne Streptavidin C1 (Invitrogen) overnight at 4°C for affinity purification of biotinylated proteins. The beads were then washed with the following buffers: 2x with cold extraction buffer, 1x with cold 1 M KCl, 1x with cold 100 mM Na2CO3, 1x with 2M Urea in 10 mM Tris pH 8 at room temperature and 2x with cold extraction buffer without complete and PMSF. For on-beads tryptic digest, the streptavidin beads were washed twice each with 50 mM Tris pH 7.5 and 2 M Urea in 50 mM Tris pH 7.5, followed by three hours incubation in Trypsin buffer (50 mM Tris pH 7.5, 1 M Urea, 1m M DTT, 0.4 µg Trypsin) at 25°C. The supernatant from tryptic digest and two washes with 60 µl 1 M Urea in 50 mM Tris pH 7.5 were reduced and alkylated and digested overnight with 0.5 µg Trypsin. The digest was acidified by adding formic acid to a final concentration of ~ 1 % and desalted using OMIX C18 pipette tips. LC-MS/MS was done on a Q-Exactive HF hybrid quadrupole-Orbitrap mass spectrometer (Thermo Fisher), equipped with an Easy LC 1200 UPLC liquid chromatography system (Thermo Fisher). Peptides were separated using analytical column ES803 (Thermo Fisher). The flow rate was 300 nl/min and a 120-min gradient was used. Peptides were eluted by a gradient from 3 to 28 % solvent B (80 % acetonitrile, 0.1 % formic acid) over 100 minutes and from 28 to 44 % solvent B over 20 minutes, followed by short wash at 90 % solvent B. Precursor scan was from mass-to-charge ratio (m/z) 375 to 1,600 and top 20 most intense multiply charged precursors were selection for fragmentation. Peptides were fragmented with higher-energy collision dissociation (HCD) with normalized collision energy (NCE) 27.

### Data Protocol
Protein identification, normalization and label-free quantification (LFQ) were done in MaxQuant (version 1.6.2.6). Settings were default settings with minor modifications. Methionine oxidation and N-terminal acetylation were set as variable modifications and Carbamidomethylcysteine as fixed modification. Maximum number of modifications per peptide was five. Trypsin/P with a maximum of two missed cleavages was set as digest. Peptides were searched against the latest TAIR10 protein database containing a total of 35,386 entries (TAIR10_pep_20101214, updated 2011-08-23, www.arabidopsis.org) plus a list of likely contaminants containing Trypsin, human Keratin, streptavidin, YFP, TID-mVenus and TID-YFP and against the contaminants list of MaxQuant. Minimum allowed peptide length was seven. FTMS and TOF MS/MS tolerance were 20 ppm and 40 ppm, respectively, and the peptide FDR and protein FDR were 0.01. Unique and razor peptides were used for protein quantification. LFQ minimum ratio count was set to two and fast LFQ was active with a minimum and average number of neighbors of three and six, respectively. Match between runs and second peptides were checked. Subsequent filtering and statistical analysis to identify enriched proteins was done in Perseus (version 1.6.2.3). The MaxQuant output files (‘txt’ folder), as well as the data analysis in Perseus (version 1.6.2.3), are provided as extra files.

### Publication Abstract
Defining specific protein interactions and spatially or temporally restricted local proteomes improves our understanding of all cellular processes, but obtaining such data is challenging, especially for rare proteins, cell types, or events. Proximity labeling enables discovery of protein neighborhoods defining functional complexes and/or organellar protein compositions. Recent technological improvements, namely two highly active biotin ligase variants (TurboID and miniTurbo), allowed us to address two challenging questions in plants: (1) what are in vivo partners of a low abundant key developmental transcription factor and (2) what is the nuclear proteome of a rare cell type? Proteins identified with FAMA-TurboID include known interactors of this stomatal transcription factor and novel proteins that could facilitate its activator and repressor functions. Directing TurboID to stomatal nuclei enabled purification of cell type- and subcellular compartment-specific proteins. Broad tests of TurboID and miniTurbo in <i>Arabidopsis</i> and <i>Nicotiana benthamiana</i> and versatile vectors enable customization by plant researchers.

### Keywords
Arabidopsis thaliana, Protein complex, Plant, Seedling, Streptavidin affinity purification, Lc-ms/ms, Bhlh transcription factor, Stomatal guard cell, Turboid, Fama, Proximity labeling

### Affiliations
Carnegie Institution at Stanford

### Submitter
Shouling Xu

### Lab Head
Dr Shouling Xu
Carnegie Institution at Stanford


