### Accession
PXD021792

### Title
Proteomics and phosphoproteomics on NB1 and IMR32 neuroblastoma cells stimulated with ALAKAL2

### Description
High-risk neuroblastoma (NB) is notoriously difficult to treat and is responsible for a disproportionate number of childhood deaths due to cancer. One long accepted indicator of high-risk NB and poor prognosis is amplification of the neural MYC (MYCN) oncogene, which is currently therapeutically intractable in this patient population. The identification of Anaplastic Lymphoma Kinase (ALK) as an oncogene in NB raised the possibility of using ALK tyrosine kinase inhibitors (TKIs) in the treatment of NB patients who harbor activating ALK mutations. The number of ALK-positive NB patients on primary diagnosis is in the range of 8-10%, a figure that increases substantially in the relapsed patient population. ALK signaling is activated by the ALKAL2 ligand located on the distal portion of chromosome 2, along with the ALK receptor and MYCN loci, in the ‘2p gain’ region that has been associated with NB. The question of whether dysregulation of ALK ligand may also play a role in NB has not been addressed, although notably, one of the first oncogenes described was the v-sis oncogene that shares more than 90% homology with the PDGF ligand. Therefore, we tested whether ALKAL ligand was able to potentiate NB progression in the absence of ALK mutation. We show here that overexpression of ALKAL2 is sufficient to drive rapid onset and penetrant Th-MYCN driven NB in the absence of ALK mutation, and that these tumours are sensitive to ALK TKI therapy. These results suggest that additional NB patients, such as those exhibiting 2p gain, may also benefit from ALK TKI based therapeutic intervention.

### Sample Protocol
The samples were digested with trypsin using the filter-aided sample preparation (FASP) method (Wisniewski et al., 2009). Briefly, 30 ug from each sample was reduced with 100 mM dithiothreitol at 60°C for 30 min, transferred to 30 kDa MWCO Pall Nanosep centrifugation filters (Sigma-Aldrich), washed several times with 8 M urea and once with digestion buffer prior to alkylation with 10 mM methyl methanethiosulfonate in digestion buffer for 30 min. Digestion was performed in 50 mM TEAB, 1% sodium deoxycholate (SDC) buffer at 37°C by addition of 0.3 µg Pierce MS grade Trypsin (Thermo Fisher Scientific) and incubated overnight. An additional portion of trypsin was added and incubated for another three hours. Peptides were collected by centrifugation, and labelled using TMT 11-plex isobaric mass tagging reagents (Thermo Scientific) according to the manufacturer instructions. After pooling of the TMT set, SDC was removed by acidification with 10% TFA. The proteins were pre-fractionated into 40 fractions by basic reversed-phase chromatography (bRP-LC) using a Dionex Ultimate 3000 UPLC system (Thermo Fisher Scientific). Peptide separation was performed using a reversed-phase XBridge BEH C18 column (3.5 μm, 3.0x150 mm, Waters Corporation) and a linear gradient from 3% to 40% acetonitrile in 10 mM ammonium formate buffer at pH 10.00 over 17 min, followed by an increase to 90% acetonitrile over 5 min. The fractions were concatenated into 20 fractions, dried and reconstituted in 3% acetonitrile, 0.2% formic acid.

### Data Protocol
The data files for the TMT set were merged for identification and relative quantification using Proteome Discoverer version 2.2 (Thermo Fisher Scientific). The search was performed by matching against the Homo sapiens Swissprot Database (version November 2017, Swiss Institute of Bioinformatics, Switzerland) using Mascot 2.5 (Matrix Science) with a precursor mass tolerance of 5 ppm and fragment mass tolerance of 0.6 Da. Tryptic peptides were accepted with zero missed cleavage, variable modifications of methionine oxidation and fixed cysteine alkylation; TMT-label modifications of N-terminal and lysine were selected. Percolator was used for the validation of identified proteins and the quantified proteins were filtered at 1% FDR and grouped by sharing the same sequences to minimise redundancy. TMT reporter ions were identified in the MS3 HCD spectra with 3 mmu mass tolerance, and the TMT reporter intensity values for each sample were normalized on the total peptide amount. Only peptides unique for a given protein were considered for quantification.

### Publication Abstract
None

### Keywords
Alk, Phosphorylation, Nb, Tandem mass tag, Alkal2, Tmt, Neuroblastoma, Anaplastic lymphoma kinase, Cancer

### Affiliations
SAMBIO Core Facilities, Sahgrenska Academy, University of Gothenburg
Professor, Institute of Biomedicine, University of Gothenburg, Gothenburg, Sweden

### Submitter
Proteomics Core Facility

### Lab Head
Dr Ruth H. Palmer
Professor, Institute of Biomedicine, University of Gothenburg, Gothenburg, Sweden


