### Accession
PXD020764

### Title
Multiomic analysis implicates OXPHOS/fatty acid oxidation as a potential druggable target in platinum-resistant high grade serous ovarian cancer

### Description
In the United States, high grade serous ovarian cancer (HGSOC) is the most lethal gynecologic malignancy and tumor response to platinum-based chemotherapy is a major determinant of clinical outcome. Although recent efforts have dramatically improved the median survival of advanced ovarian cancer, the initial treatment of the disease has remained the same. The initial therapy includes surgery and chemotherapy and the response rate is very high (85%). Unfortunately, 15% of patients do not respond to this therapy and have platinum-refractory disease. These patients have a very short survival and there is an urgent need to identify novel pharmaceutically targetable pathways to treat these patients. We generated extensive proteomic (global, phospho, ubiquitin, acetylation, pTyr) and RNASeq-based dynamic molecular profiles (+/-carboplatin at 8 and 24 hours) from HGSOC intra-patient cell line pairs (PEA1/PEA2, PEO1/PEO4, PEO14/PEO23) derived from 3 patients before and after acquiring platinum resistance. The molecular profiles revealed a multi-faceted response to carboplatin (e.g., induction of a DNA damage response, ubiquitination of ribosomal proteins, and metabolic changes), as well as novel carboplatin-induced post-translational modifications. Higher oxidative phosphorylation (OXPHOS) and fatty acid beta-oxidation (FAO) pathway expression was observed in resistant compared with sensitive cells. These expression findings were validated via metabolite profiling of cell lines and proteomic profiling of platinum sensitive and refractory HGSOC patient derived xenograft (PDX) models. Both pharmacologic inhibition and CRISPR knockout of CPT1A, which represents the rate limiting step of FAO, sensitizes HGSOC cells to platinum. The metabolic signature identified in the cell line and PDX models is correlated with survival in a previously reported proteomic analysis of HGSOC, and thus FAO is a candidate druggable pathway to overcome platinum resistance.

### Sample Protocol
Pairs of cell lines were derived from three patients before and after platinum treatment and 10 patient-derived xenograft (PDX) models were derived from HGSOC patients (5 platinum-sensitive, 5 refractory). Cells and PDX were treated with 80 uM Carboplatin and harvested after 8 or 24 hours of treatment. A control sample was treated with vehicle and harvested at 24 hours. The experiment was repeated on three different days. Cell lines underwent extensive proteomic profiling (global proteome, global phosphoproteome, tyrosine phosphoproteome, acetylome, ubiquitylome) and PDX samples underwent global proteome and phosphoproteome profiling on protein lysates that underwent proteolytic digestion and TMT-labeled prior to shotgun mass spectrometry (LC-MS/MS) analysis. Analyses of the cell lines were done in biological triplicate to ensure reproducibility. For the cell lines, the 54 individual samples were distributed across six TMT-10plex groups, with a bridge channel consisting of equal amount of protein digest from each of the 54 samples included in each group. For PDX samples, the 60 individual samples were distributed across six TMT-11plex groups, with a bridge channel consisting of equal amount of protein digest from each of the 60 samples included in each group. All proteomic profiling (global proteome, global phosphoproteome, tyrosine phosphoproteome, acetylome, ubiquitylome) workflows began with proteolytic digestion. Following digestion, subproteome samples were enriched by IMAC (for the phosphoproteome) or by Ab. Global and enriched samples were then TMT-labeled. Global and phosphoproteome samples were subjected to basic reverse-phased fractionation. All samples then underwent LC-MS/MS analysis. For full proteome analysis of the PDX samples, since proteins in a PDX model can be either mouse- or human-origin and a PDX is inherently more complex than human or mouse alone, a real-time search-based data acquisition method was utilized to only perform scans on precursors that were matched uniquely to a peptide of human origin.

### Data Protocol
Mass spectra were processed using a Sequest-based in-house software pipeline. Spectra were searched against a database including all entries from the human UniProt database (February 04, 2014). This database was concatenated with one composed of all protein sequences in the reversed order as well as known common protein contaminants. Sequest searches were performed using a 50 ppm precursor ion tolerance, requiring trypsin protease specificity, while allowing up to two missed cleavages. The product ion tolerance was set to 0.9 Da. TMT tags on peptide N termini/lysine residues (+229.162932 Da) and carbamidomethylation of cysteine residues (+57.02146 Da) were set as static modifications while methionine oxidation (+15.99492 Da) was set as variable modifications. For each PTM analysis, phosphorylation on serine, threonine and tyrosine (+79.966 Da), lysine ubiquitylation (+114.04293 Da) or lysine acetylation (-187.15237 Da) was included as variable modification. Peptide-spectrum matches (PSMs) were adjusted to a 1% false discovery rate (FDR) (17327847). and then collapsed further to a final protein-level FDR of 1%. To confidently localize PTM sites only the ones with Ascore values > 13 (p â‰¤ 0.05) were considered. Proteins and PTM sites were quantified by summing reporter ion counts across all matching PSMs. MS3 spectra with TMT reporter summed signal-to-noise ratio less than 100, or a MS/MS isolation specificity less than 0.5 were excluded from quantification.

### Publication Abstract
None

### Keywords
Human, Fractionation, Phosphorylation, Cell line, Pdx, Tmt, Shotgun proteomics, Acetylation, Global, Ubiquitinylation, Ovarian cancer

### Affiliations
Clinical Research Division, Fred Hutchinson Cancer Research Center
Fred Hutchinson Cancer Research Center

### Submitter
Jacob Kennedy

### Lab Head
Dr Amanda G Paulovich
Clinical Research Division, Fred Hutchinson Cancer Research Center


