### Accession
PXD006191

### Title
Protein landscape for human B-cell differentiation

### Description
The present project reports the quantitative proteome of different B-cell populations corresponding to the last steps of B-cell differentiation (i.e., naive (N), centroblast (CB), centrocytes (CC), memory cells (M), plasmablasts (PC).

### Sample Protocol
Tonsilar samples from 5 individuals (3 child and 2 adults) were proccessed to obtain the cellular suspensions which were separated by FACSAria (5 B-cell populations were separated: naive, centroblast, centrocyte, and plasmablast). Then, cell samples were processed to obtain the proteins by using a protein extraction protocol based on the usage of urea. Equal amount of protein samples were run in a 4-20% SDS-PAGE gel for naive, centroblast, centrocyte and memory-cell samples and, afterthat, proteins were digested in-gel (5 gel bands per sample). Proteins from plasmablasts (PC1-5) were digested in -solution due to the limits during the isolation process (the number of separated plasmablast was not enough to be processed in gel). Peptides were analyzed in a LTQ Orbitrap XL mass spectrometer over 140 min for in-gel digested proteins and 170 min for in-solution digested proteins.

### Data Protocol
Progenesis QI was used for quantification of the multiplexed LC-MS data. In-gel digested proteins (from N1-5, CB1-5, CC1-5, and M1-5) were quantitatively compared accross the 5 biological samples. PC results were evaluated in a cualitative manner. Search parameters were set as follows: carbamidomethylation of cysteine as a fixed modification, oxidation of methionine and acetylation of the protein n-terminus as variable ones, precursor and fragment mass tolerance were set to 10 ppm and 0.6 Da, respectively, and fully tryptic digestion with up to two missed cleavages. False discovery rate (FDR) was set at 1%. The results generated were considered per band from the gel (thus, BAND1 contains N1_b1+N2_b1+N3_b1+N4_b1+N5_b1+CB1_b1+...+M5_b1, and so on).

### Publication Abstract
Human B-cell differentiation has been extensively investigated on genomic and transcriptomic grounds; however, no studies have accomplished so far detailed analysis of antigen-dependent maturation-associated human B-cell populations from a proteomic perspective. Here, we investigate for the first time the quantitative proteomic profiles of B-cells undergoing antigen-dependent maturation using a label-free LC-MS/MS approach applied on 5 purified B-cell subpopulations (naive, centroblasts, centrocytes, memory and plasma B-cells) from human tonsils (data are available <i>via</i> ProteomeXchange with identifier PXD006191). Our results revealed that the actual differences among these B-cell subpopulations are a combination of expression of a few maturation stage-specific proteins within each B-cell subset and maturation-associated changes in relative protein expression levels, which are related with metabolic regulation. The considerable overlap of the proteome of the 5 studied B-cell subsets strengthens the key role of the regulation of the stoichiometry of molecules associated with metabolic regulation and programming, among other signaling cascades (such as antigen recognition and presentation and cell survival) crucial for the transition between each B-cell maturation stage.

### Keywords
B-cell differentiation, Memory, Plasmablast, Centrocyte, Naive, Protein quantification, Centroblast

### Affiliations
Cancer Research Institute-USAL
L11. Department of Medicine. Cancer Research Center - USAL. 37007 Salamanca, Spain.

### Submitter
Paula DÃ­ez

### Lab Head
Dr Manuel Fuentes, Ph.D.
L11. Department of Medicine. Cancer Research Center - USAL. 37007 Salamanca, Spain.


