### Accession
PXD032113

### Title
Enhanced RNA Interactome capture (eRIC) and total proteome of mouse brain, kidney and liver

### Description
Male mice on a homogenous C57BL6/J genetic background were sacrificed at 11 to 13 weeks of age by cervical dislocation. Brain, kidneys and liver were immediately harvested, flash frozen in liquid nitrogen and stored at -80Ã‚Â°C until use. Animal handling was in accordance with guidelines approved by the European Molecular Biology Laboratory (EMBL). Organ sections (thickness 30 Ã‚Âµm) were prepared in a Cryostat (Leica Biosystems, Leica CM3050 S) set to -20 Ã‚Â°C and deposited onto SuperFrost glass slides (Carl Roth, H880); the glass slides were pre-cooled to -20Ã‚Â°C in the cryostat. ~10 sections were placed on a glass slide, and a total of around 150 sections were prepared for each sample per mouse. Tissue sections on glass slides were then transferred onto metal plates placed on dry ice. The tissue sections were exposed or not to 1 J/cm2 of 254 nm UV light in a XL 1500 UV Spectrolinker (Spectronics Corporation) and subjected to eRIC to determine the RNA-bound proteome. eRIC eluates obtained from the respective organs of two mice were combined per independent experiment. The analyses were conducted in duplicate for each organ studied. For the no crosslink controls, organ sections from four mice were pooled to generate one unique no crosslink eRIC sample per organ studied.

### Sample Protocol
Capture of ribonucleoprotein complexes using eRIC: After UV exposure, organ sections were recovered by scraping directly into 15 mL ice-cold lysis buffer (20 mM Tris-HCl (pH 7.5), 500 mM LiCl, 1 mM EDTA, 5 mM DTT, 0.5% (w/v) LiDS) supplemented with protease inhibitors and RNase inhibitor. For the non-crosslink control, every other section was lysed directly without exposure to UV. The lysates were passed once through a 25-Gauge needle (BD Microlance, 300400) and 7 times through a 27-Gauge needle (BD Microlance, 302200). The homogenates were snap frozen in liquid nitrogen and stored at −80 °C until use. Lysates were thawed at 37 °C, incubated for 15 min at 45 °C, cooled down on ice, and the debris was pelleted 5 min at 16.000 x g at 4 °C. The supernatant was transferred to fresh tubes and complemented with 5 mM extra of DTT. 200 µL of each sample were taken as input and used for the total proteome analysis. The rest of the supernatant was incubated 1 h with rotation with 15 mg of magnetic beads (Perkin Elmer, M-PVA C11) coupled to LNA2.T probe (composed of a primary amine at the 5’ end, a flexible C6 linker, and 20 thymidine nucleotides in which every other base is a LNA: /5AmMC6/+TT+TT+TT+TT+TT+TT+TT+TT+TT+TT (+T: LNA, T: DNA). RNP complexes were washed at 40 °C once with lysis buffer and twice with each of the buffers 1 (20 mM Tris-HCl (pH 7.5), 500 mM LiCl, 1 mM EDTA, 5 mM DTT, 0.1% (w/v) LiDS), 2 (20 mM Tris-HCl (pH 7.5), 500 mM LiCl, 1 mM EDTA, 5 mM DTT, 0.02% (v/v) NP40), and 3 (20 mM Tris-HCl (pH 7.5), 200 mM LiCl, 1 mM EDTA, 5 mM DTT, 0.02% (v/v) NP40). Each wash was performed with 5 mL of the corresponding buffer for 5 min at 37°C with gently rotation. Bead suspension was incubated in 220 µl of nuclease-free water for 10 min at 40 °C and 800 rpm (pre-elution). The bead suspension was then divided into two aliquots: 200 μL were used for RNase-mediated elution of RNA-bound proteins. The beads were collected on a magnet and the supernatant discarded. Proteins were eluted at 37 °C 800 rpm for 60 min with ∼200 U RNase T1 (Sigma-Aldrich, R1003–100KU) and ∼200 U RNase A (Sigma-Aldrich, R5503) in 10 mM Tris-HCl (pH 7.5), 0.15 mM NaCl, 5 mM DTT, 0.01% NP40. Sample Preparation for Mass Spectrometry and TMT labeling: eRIC samples were concentrated using a SpeedVac, and treated with 10 mM DTT in HEPES buffer (50 mM HEPES, pH 8.5) for 30 min at 56 ºC to reduce disulphide bridges in proteins. Reduced cysteines were then alkylated for 30 min at room temperature with 20 mM 2-chloroacetamide in HEPES buffer (protected from light). Samples were prepared using the SP3 protocol and trypsin (sequencing grade, Promega) was added at an enzyme to protein ratio of 1:50. Following an overnight incubation at 37 °C, the digested peptides were recovered in HEPES buffer applying two successive elution steps. The peptides were subsequently labelled with TMT10plex Isobaric Label Reagent (ThermoFisher) according to the manufacturer’s instructions. The samples were combined and cleaned using an OASIS HLB µElution Plate (Waters). Offline high pH reverse phase fractionation was carried out on an Agilent 1200 Infinity high-performance liquid chromatography system, equipped with a Gemini C18 column (3 μm, 110 Å, 100 x 1.0 mm, Phenomenex). Mass spectrometry data acquisition: An UltiMate 3000 RSLC nano LC system (Dionex) fitted with a trapping cartridge (µ-Precolumn C18 PepMap 100, 5 µm, 300 µm i.d. x 5 mm, 100 Å) and an analytical column (nanoEase™ M/Z HSS T3 column 75 µm x 250 mm C18, 1.8 µm, 100 Å, Waters) was used. Trapping was carried out with a constant flow of trapping solution (0.05% trifluoroacetic acid in water) at 30 µL/min onto the trapping column for 6 minutes. Subsequently, peptides were eluted via the analytical column running solvent A (0.1% formic acid in water, 3% DMSO) with a constant flow of 0.3 µL/min, with increasing percentage of solvent B (0.1% formic acid in acetonitrile, 3% DMSO) from 2% to 8% in 4 min, from 8% to 28% for a further 104 min, from 28% to 40% in another 4 min, and finally from 40% to 80% for 4 min, followed by re-equilibration back to 2% B in 4 min. The outlet of the analytical column was coupled directly to an Orbitrap Fusion Lumos Tribrid Mass Spectrometer (Thermo Scientific) using the Nanospray Flex ion source in positive ion mode. The peptides were introduced into the Fusion Lumos using a Pico-Tip Emitter 360 µm OD x 20 µm ID; 10 µm tip (New Objective) and an applied spray voltage of 2.4 kV. The capillary temperature was set to 275°C. A full mass scan was acquired with a mass range of 375-1500 m/z in profile mode in the orbitrap with a resolution of 120000. The filling time was set to a maximum of 50 ms with a limitation of 4x105 ions. Data dependent acquisition (DDA) was performed with the resolution of the Orbitrap set to 30000, with a fill time of 94 ms and a limitation of 1x105 ions. A normalized collision energy of 38 was applied. MS data was acquired in profile mode.

### Data Protocol
IsobarQuant and Mascot (v2.2.07) were used to process the acquired data, which was searched against the Mus musculus (UP000000589) Uniprot proteome database containing common contaminants and reversed sequences. The following modifications were included into the search parameters: Carbamidomethyl (C) and TMT10 (K) (fixed modification), Acetyl (Protein N-term), Oxidation (M) and TMT10 (N-term) (variable modifications). A mass error tolerance of 10 ppm and 0.02 Da was set, respectively, for the full scan (MS1) and MS/MS (MS2) spectra. Further parameters were: Trypsin digestion with a maximum two missed cleavages tolerated; a minimum peptide length of seven amino acids; at least two unique peptides required for protein identification. The false discovery rate was set to 0.01 on both the peptide and protein level. The protein output files of IsobarQuant were processed in R (ISBN 3-900051-07-0). Raw TMT reporter ion intensities (Ã¢â‚¬Ëœsignal_sumÃ¢â‚¬â„¢) were first cleaned for batch effects using the Ã¢â‚¬ËœremoveBatchEffectsÃ¢â‚¬â„¢ function of the limma package and further normalized using the vsn (variance stabilization normalization) package. Individual normalization coefficients were estimated for each tissue and condition (+UV, -UV) of the eRIC samples. Proteins were tested for differential expression using the limma package. The replicate information was added as a factor in the design matrix given as an argument to the Ã¢â‚¬ËœlmFitÃ¢â‚¬â„¢ function of the limma package. eRIC hits were defined as those proteins with a false discovery rate (FDR) of less than 5 % and a fold change (FC) greater than 2 in eRIC eluates from UV-irradiated samples compared to the corresponding no-UV controls.

### Publication Abstract
None

### Keywords
Rna interactome capture, Rbp, Rbpome, Eric, Ric, Rna binding protein

### Affiliations
EMBL
European Molecular Biology Laboratory, Meyerhofstrasse 1, 69117 Heidelberg, Germany

### Submitter
Frank Stein

### Lab Head
Dr Matthias W. Hentze
European Molecular Biology Laboratory, Meyerhofstrasse 1, 69117 Heidelberg, Germany


