### Accession
PXD024816

### Title
Mutation of Arabidopsis SME1 improves oxidative stress resilience

### Description
Alternative splicing is an important posttranscriptional gene regulatory process, acting in diverse adaptive and basal plant processes. Splicing of precursor-messenger RNA (pre-mRNA) is catalyzed by a dynamic ribonucleoprotein complex, termed the spliceosome. In a suppressor screen, we identified a nonsense mutation in the Sm protein SME1 to alleviate photorespiratory H2O2-dependent cell death in catalase-deficient plants. Furthermore, sme1-2 mutants showed increased tolerance to the superoxide-generating herbicide methyl viologen. mRNA-seq and proteomic analyses indicated a constitutive stress response and pre-mRNA splicing alterations in sme1-2 mutants under control conditions. With a pulldown experiment, using SME1 as a bait, we provide experimental evidence for proteins associated with Arabidopsis thaliana spliceosome complexes, by identifying almost 50 proteins homologous to mammalian spliceosome-associated proteins and propose four novel proteins previously not-reported or hypothesized to be part of plant spliceosome complexes. Mutation of one of these interactors,  the Sm core assembly protein ICLN also resulted in a decreased sensitivity to methyl viologen. Taken together, we show that SME1 mutation and impaired snRNP assembly result in enhanced resilience to oxidative stress.

### Sample Protocol
Three-week-old seedling shoots grown on full strength MS medium were collected and frozen in liquid nitrogen. After the plant material had been ground to a fine powder with mortar and pestle, the frozen tissue was suspended in 1 mL protein extraction buffer containing 1% (w/v) 3-[(3-cholamidopropyl)-dimethylammonio]-1-propanesulfonate (CHAPS), 0.5% (w/v) deoxycholate, 5 mM EDTA, 5 mM EGTA, protease inhibitor cocktail (Roche Applied Science, 1 tablet/10 mL), and 10% (v/v) glycerol in 50 mM HEPES, pH 7.5. The sample was vortexed vigorously and subsequently centrifuged at 16,000g for 10 min at 4°C to precipitate sample debris. The cleared supernatant was then recovered and, after measurement of the protein concentrations with the Bradford assay (DV BioRad Kit), guanidinium hydrochloride was added to a final concentration of 4 M. Cysteine residues were reduced with 10 mM dithiothreitol (Sigma-Aldrich) for 30 min at 55°C and subsequently alkylated with 20 mM iodoacetamide (Sigma-Aldrich) for 15 min at room temperature in the dark. Proteins were then precipitated by addition of amphipols (Apols A8-35; Anatrace) to a final concentration of 1 mg/mL and acidified to pH of 2-3 by means of 20% trifluoroacetic acid (TFA). After a 10-min incubation at room temperature, the samples were centrifuged at 20,000g for 10 min at room temperature and the supernatants were discarded. The protein-containing pellets were washed once with 500 µL 0.1% TFA and centrifuged again at 20,000g for 10 min at room temperature before discarding the supernatants again. The washed pellets were solubilized in 400 µL 100 mM HEPES (pH 8.0) and vortexed vigorously before predigesting the proteins with EndoLysC (enzyme/substrate ratio of 1/250 w/w; Wako) for 2 h at 27°C followed by a complete digestion with trypsin (enzyme/substrate ratio of 1/100, w/w; Promega) at 37°C at 550 rpm rotation. The next day, the digestion was halted by adding TFA (Sigma-Aldrich) to a final 1% concentration. The peptide mixtures were desalted with macro spin columns (Harvard apparatus) and eluted with an elution buffer containing 60% acetonitrile (ACN) and 0.1% TFA. The eluates were vacuum dried completely and solubilized in 2% can and 0.1% TFA in MilliQ. The peptide concentrations were determined on the Lunatic device (Unchained Labs). Proteome samples of sme1-1 and wild-type seedlings were fractionated prior to LC-MS/MS analysis to increase the proteome coverage for a more in-depth analysis. The samples were resuspended in loading solvent A (2% ACN in 0.1% TFA) and vortexed vigorously before introducing approximately 50 µg into an Agilent 1100 series capillary column operating in micro flow. First, the sample was loaded on the trapping column (35 × 0.3 mm I.D., 5.0 µm beads C18) for 9 min and back-flushed toward the analytical column (150 × 0.3 mm I.D., 3.0 µm beads C18). Peptides were separated with a linear gradient of 100 min from 100% buffer A (10 mM ammonium bicarbonate, pH 5.5) to 100% buffer B (10 mM ammonium bicarbonate, 70% ACN, pH 5.5), followed by a 10-min wash at 100% buffer B and reequilibration at 100% buffer A, at a flow rate of 3.0 µL/min. Fraction collection started after 20 min for 70 min resulting in a total of 70 fractions that were pooled to a final 10 fractions per sample in total. An UV cell monitored both the 214 nm (amine bond) and 280 nm (aromatic residues) throughout the whole run. The samples were vacuum dried completely and resuspended in 20 µL loading solvent A only moments before analysis. LC-MS/MS analysis For each sample, 3 µg of the peptide mixtures were introduced for LC-MS/MS analysis into an Ultimate 3000 RSLC nano LC Pro-Flow system (Thermo Fisher Scientific) in-line connected to a Q Exactive HF mass spectrometer (Thermo Fisher Scientific). For more details on Lc-MS/MS settings see publication.

### Data Protocol
In total, 48,359 spliced protein isoforms from Araport11 were appended to 16,371 protein sequences in silico translated from Scallop-assembled transcripts (described above). In addition, the Common Repository of Adventitious Proteins database (ftp://ftp.thegpm.org/fasta/cRAP, 116 proteins) was added to identify peptides matching common contaminants. The resulting protein database was used to search RAW files with MaxQuant (version 1.6.7.0) (Cox and Mann, 2008). Variable modifications included protein N-terminal acetylation and methionine oxidation, whereas cysteine carbamidomethylation was set as fixed modification. Matching-between-runs and the LFQ algorithm were enabled with default settings. The resulting ‘peptides.txt’ table was used for differential protein testing by means of MSqRob (Goeminne et al. 2016). For quantification of proteins, we aggregated peptides according to their respective gene locus, e.g. peptides matching AT2G18740.1 and AT2G18740.2 are assigned to AT2G18740. In addition, peptides matching reverse/contaminant proteins or matching gene loci for which no other peptides were identified, were filtered, leaving 31,604 out of 31,799 peptides (99.4%) for reliable protein quantification. These peptide intensities were loaded and pre-processed by MSqRob. Afterward, by means of the preprocessed peptides intensities, a model was fitted with genotype as fixed effect, and MS/MS run, replicate, and peptide sequence as random effects. Afterward, sme1-1 was contrasted against the wild-type and significant proteins were filtered (Q value ≤ 0.05). For peptide identification of offline HPLC-fractionated samples, we used MaxQuant (Cox and Mann, 2008) with similar search settings as described above.

### Publication Abstract
None

### Keywords
Arabidopsis thaliana, Sm protein, Spliceosome, Oxidative stress, Catalase2

### Affiliations
Department of Plant Biotechnology and Bioinformatics, Ghent University, 9052 Ghent, Belgium; Center for Plant Systems Biology, VIB, 9052 Ghent, Belgium
PSB-VIB

### Submitter
Patrick Willems

### Lab Head
Dr Frank Van Breusegem
Department of Plant Biotechnology and Bioinformatics, Ghent University, 9052 Ghent, Belgium; Center for Plant Systems Biology, VIB, 9052 Ghent, Belgium


