### Accession
PXD008355

### Title
A systematic phosphoproteome screen to identify targets of the Target of Rapamycin (TOR) kinase in Arabidopsis thaliana

### Description
The target of rapamycin (TOR) kinase is a central regulatory hub that translates environmental and nutritional information into permissive or restrictive growth decisions. Although the TOR pathway is conserved among eukaryotes, plants developed unique adaptations to this pathway to cope with their autotrophic and sessile nature. Here, we captured for the first time a proteome-wide view of the plant TOR phosphorylation and interaction landscape. After sampling four biological repeats, proteins were extracted, digested and phosphopeptides were enriched. In total, 8302 phosphopeptides on 2558 proteins were identified from all samples. After label free quantitative analysis, a filtered dataset of 5500 phosphopeptides on 2056 proteins was retained. To identify TOR-dependent sites, the control-, AZD8055- and rapamycin-treated samples were statistically analyzed using a linear mixed model to evaluate the effect of the treatment, time and the interaction between both. For T10, T20 and T40, the sucrose control samples were compared to the AZD8055 or rapamycin samples. In addition, the AZD8055 samples were compared to the rapamycin samples, and the sucrose control samples (T10, T20, T40) were compared to T0. Overall, the strongest effect occurred with AZD8055. Therefore, an additional filter was applied retrieving only phosphopeptides that changed at least two-fold upon AZD8055 treatment. In total, 96 unambiguous TOR-dependent phosphosites were detected on 66 proteins, linking TOR to a plethora of biological processes.

### Sample Protocol
For the pilot experiment, 35S:S6K-HA PSB-D cell culture was sucrose-starved and cells were harvested 30 min after sucrose addition. For the systematic phosphoproteomics experiment, cells from each condition were generated independently, starting from one pooled 3-day-old wild-type PSB-D cell culture per replicate experiment, which was split into 10 subcultures. Four biological repeats were done, starting each time from a new independent wild-type culture. Sucrose, AZD8055 and rapamycin concentrations were applied as described before. Cells were grinded, mixed with homogenization buffer (290 mM sucrose, 250 mM Tris–HCl (pH 8.0), 25 mM EDTA, 25 mM sodium fluoride, 50 mM sodium pyrophosphate, 1 mM ammonium molybdate, 0.5% w/v polyvinyl pyrrolidone, 0.07% beta-mercaptoethanol, and 1 X Complete Ultra EDTA-free Protease Inhibitor Cocktail Tablet, 1 X Phosphatase Inhibitor Cocktail Table PhosSTOP) and sonicated for 10 min. Samples were centrifuged for 15 min at 12000 g (4C). Supernatant was transferred to two 50 mL Falcon tubes (5 mL/tube). To each tube, 15 mL methanol, 5 mL chloroform and 15 mL H2O was added, and mixture was centrifuged at 1500 g (4°C). Aqueous phase was removed and 20 mL methanol was added on top of the interface containing the white protein pellet without mixing. After centrifugation for 10 min at 12000 g (4°C), supernatant was discarded and pellets placed on ice, washed with 5 mL acetone, vortexed and centrifuged for 10 min at 12000 g. Pellets were transferred to 2 mL tubes, washed with acetone and dried in a Speedvac. 1 mL lysis buffer (7 M urea, 2 M thiourea) was added, vortexed and incubated for 10 min at room temperature. 0.5 mL lysis buffer was added and vortexing and incubation was repeated. Sample was centrifuged for 15 min at 21000g. Supernatant was reduced with 5 mM DTT for 45 min at 50°C and alkylated with Iodoacetamide (15 mM final) for 45 min in the dark at 26°C. Sample was cleaned up using a NAP-25 gel filtration column and centrifuged for 10 min at maximal speed (4°C). 300 µg protein was diluted with 50 mM NH4HCO3 to a volume of 180 µL and digested with 3 µg Lys-C/Trypsin for 16 h at 37C. Digest was centrifuged at maximal speed for 10 min, supernatant was isolated, acidified with 20 L 10% Formic Acid (FA) and desalted with MonoSpin C18 columns (100 L/column). Eluates were pooled, evaporated in a Speedvac. Peptide pellet was dissolved in 50 µL 0.1% trifluoroacetic acid (TFA), 80% AcN solution and loaded onto an equilibrated Titansphere Phos-TiO spin tip, together with 100 µL buffer B, and mixed by pipetting. Spin tip was centrifuged at 1000 g for 10 min, flow-through was reloaded onto the spin tip and centrifugation was repeated. For washing, 20 µL buffer B was loaded onto the tip, followed by 2 X 20 L buffer A and centrifugation at 3000 g for 2 min. Phosphopeptides were eluted with 50 µL 5% ammonium hydroxide, followed by elution with 50 µL 50% pyrrolidine. The eluate was acidified with 100 µL 20% TFA. A MonoSpin C18 column was equilibrated with 20 µL solution Y (0.1% TFA, 80% AcN), and 20 µL solution X (0.1% TFA, 5% AcN) and centrifuged at 2000 g for 2 min. The phosphopeptide sample was loaded together with 100 µL solution X onto the column. After centrifugation, the column was washed with 20 µL solution X and centrifugation was repeated. Phosphopeptides were eluted with 100 µL solution Y and dried in a Speedvac. Peptides were dissolved in 30 µL loading solvent A (0.1% TFA in H2O/AcN (98:2, v/v)) and 10 µl was injected for LC-MS/MS analysis on an Ultimate 3000 RSLCnano system (Thermo) in-line connected to a Q Exactive mass spectrometer (Thermo). Trapping was performed at 10 μL/min for 4 min in loading solvent A on a 20 mm trapping column (made in-house, 100 μm internal diameter (I.D.), 5 μm beads, C18 Reprosil-HD, Dr. Maisch, Germany) followed by a 150 mm analytical column (made in-house, 75 µm I.D., 3 µm beads C18 Reprosil-HD, Dr. Maisch). Peptides were eluted by a linear increase from 2 to 55% MS solvent B (0.1% FA in H2O/AcN (2:8, v/v)) over 120 min at a constant flow rate of 300 nL/min, followed by a 5-min wash reaching 99% MS solvent B and re-equilibration with MS solvent A (0.1% FA in H2O/AcN (2:8, v/v)). The Q Exactive was operated in data-dependent mode, automatically switching between MS and MS/MS acquisition for the 10 most abundant ion peaks per MS spectrum. Full-scan MS spectra (400-2000 m/z) were acquired at a resolution of 70,000 in the orbitrap analyzer after accumulation to a target value of 3,000,000. The 10 most intense ions above a threshold value of 17,000 were isolated (window of 2.0 Th) for fragmentation at a normalized collision energy of 25 after filling the trap at a target value of 50,000 for maximum 60 ms. MS/MS spectra (200-2000 m/z) were acquired at a resolution of 17,500 in the orbitrap analyzer. The S-lens RF level was set at 50 and we excluded precursor ions with unassigned, single and charge states above 5 from fragmentation selection.

### Data Protocol
To analyze the phosphoproteome data, raw MS data were analyzed using MaxQuant (http://www.biochem.mpg.de/5111795/maxquant). For the systematic phosphoproteome analysis, a label free quantitative analysis was performed using the MaxQuant LFQ peptide intensities in Perseus (http://www.biochem.mpg.de/5111810/perseus). First, data was transformed with the ‘expand site table’ option and LFQ values were Log2 transformed. Phosphopeptides were filtered for reverse hits and potential contaminants. Replicates were grouped and the dataset was filtered for phosphopeptides present in at least three biological repeats of at least one experiment. Missing values were imputed from a normal distribution around the detection limit per sample (width = 0.3 and down shift = 1.8). For statistical analysis of differentially regulated phosphopeptides, a linear mixed model was fitted to the Log2 LFQ data of each peptide separately. Fixed effects were treatment and time and their interaction effect. A random effect for repeat was included in the model to account for the correlation between observations originating from the same culture. Fisher scoring was used in the first iteration to speed up calculations. The Kenward-Roger approximation for computing the denominator degrees of freedom for the tests of fixed effects was used. The following contrasts were estimated with Wald tests: T10_AZD8055 vs T10_Ctrl, T20_AZD8055 vs T20_Ctrl, T40_AZD8055 vs T40_Ctrl, T10_rapamycin vs T10_Ctrl, T20_rapamycin vs T20_Ctrl, T40_rapamycin vs T40_Ctrl, T10_AZD8055 vs T10_rapamycin, T20_AZD8055 vs T20_rapamycin, T40_AZD8055 vs T40_rapamycin, T10_Ctrl vs T0_Ctrl, T20_Ctrl vs T0_Ctrl, T40_Ctrl vs T0_Ctrl (with Ctrl being the sucrose-treated experiments where no TOR inhibitor was applied). q-values were calculated to control the false discovery rate (FDR) (Storey, 2002). A peptide was considered differentially phosphorylated between two conditions when the FDR < 0.05. The analysis was performed with the mixed procedure of SAS version 9.4 (www.sas.com). Multiple testing correction was done with the MULTTEST procedure. Residual diagnostics were carefully examined. Because in general the strongest effect was observed with AZD8055, an additional filter was applied for phosphopeptides that were up- or downregulated at least two-fold upon TOR inhibition with AZD8055. Finally, phosphopeptides with a phosphorylation localization probability below 0.7 were removed. For cluster analysis, the average estimated Log2 LFQ intensities of each condition*time level were normalized per row by median subtraction and loaded into the multiple array viewer (http://mev.tm4.org). Hierarchical clustering was performed on both the phosphosite and sample tree with both leaf orders optimized using Euclidean distance and average linkage clustering.

### Publication Abstract
The target of rapamycin (TOR) kinase is a conserved regulatory hub that translates environmental and nutritional information into permissive or restrictive growth decisions. Despite the increased appreciation of the essential role of the TOR complex in plants, no large-scale phosphoproteomics or interactomics studies have been performed to map TOR signalling events in plants. To fill this gap, we combined a systematic phosphoproteomics screen with a targeted protein complex analysis in the model plant Arabidopsis thaliana. Integration of the phosphoproteome and protein complex data on the one hand shows that both methods reveal complementary subspaces of the plant TOR signalling network, enabling proteome-wide discovery of both upstream and downstream network components. On the other hand, the overlap between both data sets reveals a set of candidate direct TOR substrates. The integrated network embeds both evolutionarily-conserved and plant-specific TOR signalling components, uncovering an intriguing complex interplay with protein synthesis. Overall, the network provides a rich data set to start addressing fundamental questions about how TOR controls key processes in plants, such as autophagy, auxin signalling, chloroplast development, lipid metabolism, nucleotide biosynthesis, protein translation or senescence.

### Keywords
Cell culture, Label-free quantitative ms, Sucrose, Phosphoproteomics, Arabidopsis, Target of rapamycin (tor), Rapamycin, Azd8055

### Affiliations
Department of Plant Biotechnology and Bioinformatics, Ghent University, 9052 Ghent, Belgium VIB Center for Plant Systems Biology, 9052 Ghent, Belgium
VIB-UGHENT

### Submitter
Jelle Van Leene

### Lab Head
Dr Geert De Jaeger
Department of Plant Biotechnology and Bioinformatics, Ghent University, 9052 Ghent, Belgium VIB Center for Plant Systems Biology, 9052 Ghent, Belgium


### SDRF
- organism: Arabidopsis thaliana
- organism part: Whole Organism
- cell type: not available
- disease: not available
- label: label free sample
- instrument: Q Exactive
- modification parameters: Carbamidomethyl
- modification parameters: Oxidation
- modification parameters: Phospho
- modification parameters: Acetyl

