### Accession
PXD022995

### Title
Ross Sea Surface Water Metaproteomics

### Description
We sampled the microbial community at the sea ice edge in McMurdo Sound, Ross Sea at the same location (-77.62S, 165.41E) for four weeks (as described in Wu et al 2019, Nat. Comms.). We had four sampling dates corresponding to weeks 1 to 4: December 28 2014, January 6, 15, and 22 2015. Large volumes of water (150--250 L) were filtered from 1 m depth at the sea ice edge, and passed through three filters sequentially (3.0, 0.8, and 0.1 um, each 293 mm Supor filters). Filters with collected biomass were then placed in tubes with a sucrose-based preservative buffer (20 mM EDTA, 400 mM NaCl, 0.75 M sucrose, 50 mM Tris-HCl, pH 8.0) and stored at -80 C until sample processing. We extracted proteins after buffer exchange into a 3\% SDS solution as previously described Wu et al 2019, Nat. Comms.

### Sample Protocol
To prepare samples for LC-MS/MS, the precipitated protein was resuspended in 100 uL 8 M urea, and then we ran a Pierce BCA Protein Assay Kit (ThermoFisher Scientific) to quantify the protein concentration in each sample. We then reduced the protein sample using 10 uL of 0.5 M dithiothreitol, and incubated the sample for 30 minutes at 60 C. Samples were then alkylated using 20 uL 0.7 M iodacetimide in the dark for 30 minutes, diluted with 50 mM ammonium bicarbonate, and digested with trypsin using a 1:50 trypsin:protein ratio. We then acidified (1.5 uL trifluoroacetic acid (TFA) and 5 uL formic acid added) and desalted samples. We desalted the samples by first conditioning the solid-phase columns with methanol (1 mL), then 50% acetonitrile (ACN) and 0.1% TFA, and then 2x 1 mL of 0.1% TFA. Samples were loaded onto columns that were subsequently washed 5x with 1 mL 0.1% TFA each time. Finally, peptides were eluted from the columns with 2x 0.6 mL 50% ACN 0.1% TFA, and 1x of 0.6 mL 70% ACN and 0.1% TFA.  We used a one-dimensional liquid chromatography tandem mass spectrometry to characterize the metaproteome. We used a one-dimensional liquid chromatography tandem mass spectrometry to characterize the metaproteome. For the largest filter size (3.0 um) we used three injections per sample, and two injections per sample for the 0.8 and 0.1 um filters. We ensured that the protein concentration in each urea-resuspended sample was equivalent across sampling weeks and within each filter size. We used a LC gradient from 0 to 10.5 minutes with 0.3 uL per minute flow of 5% solution B, from 10.5 minutes to 60 minutes the flow was 0.25 uL per minute and solution B increased to 25.0%, from 60--90 minutes %B increased to 60%, from 90--97 minutes %B increased to 95%, from 97--102 minutes %B remained at 95%, from 102--105 the flow rate increased to 0.3 uL per minute and %B decreased to 5%, which for 20 minutes. Solution A is 0.1% formic acid in water, and solution B is 0.1% formic acid in ACN. Peptides were injected onto a 75 um Ã— 30 cm column (New Objective, Woburn, MA) self-packed with 4 um, 90 A, Proteo C18 material (Phenomenex, Torrance, CA), and then online LC was performed using a Dionex Ultimate 3000 UHPLC (Thermo Scientific, San Jose, CA).  We used a data-dependent acquisition approach with a VelosPRO Orbitrap mass spectrometer (MS; Thermo Scientific, San Jose, CA) to characterize the metaproteome for each sample. We used an MS method with the following parameters: dynamic exclusion enabled, with an exclusion list of 500 and an exclusion duration of 25 seconds; a m/z precursor mass range from 300--2000 m/z; and a resolution of 60000. MS2 scans were collected with a TopN method (N = 10), using Collision-Induced Dissociation with a normalized collision energy of 35.0, an isolation width of 2.0 m/z, a minimum signal of 30000 required, and a default charge state of 2. Ions with charge states less than 2 were rejected, and those above 2 were not rejected. Lastly, we used polysiloxane as a lock mass.

### Data Protocol
For a database of potential proteins present, we used a metatranscriptome obtained from a nutrient incubation experiment conducted using water collected during week 2 of protein sampling (Jabre et al 2020, bioRxiv). Prior to database searching we removed all redundant protein sequences (P. Wilmarth, fasta-utilities), and appended the cRAP database of common laboratory contaminants. We then applied a Savitsky-Golay noise filter, a baseline filter, and applied a high-resolution peak picking approach to centroid the MS data (Weisser et al 2013, JPR). To identify peptides, we conducted a database search with MSGF+ (Kim and Pevzner 2014, Nat. Comms.). We used a 1% False Discovery Rate at the peptide-spectrum match level. Once we had identified peptides within each MS injection, we quantified these peptides at the MS1 level using the `FeatureFinderIdentification' approach (Weisser and Choudhary 2017, JPR), where peptides identified in one injection can aid identifying peptides in a different injection without a corresponding MS2 spectra.

### Publication Abstract
Micronutrients control phytoplankton growth in the ocean, influencing carbon export and fisheries. It is currently unclear how micronutrient scarcity affects cellular processes and how interdependence across micronutrients arises. We show that proximate causes of micronutrient growth limitation and interdependence are governed by cumulative cellular costs of acquiring and using micronutrients. Using a mechanistic proteomic allocation model of a polar diatom focused on iron and manganese, we demonstrate how cellular processes fundamentally underpin micronutrient limitation, and how they interact and compensate for each other to shape cellular elemental stoichiometry and resource interdependence. We coupled our model with metaproteomic and environmental data, yielding an approach for estimating biogeochemical metrics, including taxon-specific growth rates. Our results show that cumulative cellular costs govern how environmental conditions modify phytoplankton growth.

### Keywords
Phytoplankton

### Affiliations
Department of Biology, Dalhousie University, Halifax, Nova Scotia, Canada
Dalhousie University

### Submitter
Scott McCain

### Lab Head
Dr Erin M. Bertrand
Department of Biology, Dalhousie University, Halifax, Nova Scotia, Canada


