### Accession
PXD027843

### Title
Degradation of diatom protein in seawater: a peptide-level view

### Description
Peptides were identified during degradation of the diatom Thalassiosira weissflogii with and without the use of the enzyme trypsin.  This allows comparison of protein degradation occurring within the experiment (no trypsin added; peptides already present due to in situ degradation) and the protein still available for future degradation (peptides released from protein when trypsin is added during analysis).  Over the 12-day degradation experiment 31% of the particulate organic carbon was depleted and there was no preferential degradation of the overall protein pool.  However, there was distinct differentiation in the cellular location, secondary structure and modifications of the peptides that were either degraded or remained. During the initial period of rapid algal decay and bacterial growth, intracellular components from the cytoplasm were selectively consumed, resulting in the accumulation of membrane-associated proteins and peptides in the detrital pool.  Accompanying this was an increase in the importance of membrane-bound …ë-helix motifs.  At the end of the 12 day experiment, the natural bacterial assemblage was focusing on degradation of membrane-associated proteins, which was the dominant source of peptides in both the residual pool (released by trypsin) and the actively degraded pool (no trypsin added). Methylated arginine, a post-translationally modified amino acid that is produced within the diatom prior to senescence, is found in high amounts within the detrital peptide pool, suggesting a link between in-cell modification and resistance to immediate degradation.  Another modification - asparagine deamidation - appears to occur during degradation and deamidated peptides also accumulate within the detritus.  The bacterial community decomposing the algal material was rich in proteobacteria, and employed a growth approach focusing on accumulation of solubilized material across their membranes and on DNA replication. At this early stage of diagenesis, no changes in bulk amino acids (THAA) were observed, yet a peptidomic approach allowed us to observe the differential changes in diatom protein preservation by discriminating between intracellular location, secondary structure, and modifications status.

### Sample Protocol
Algal degradation A culture of the centric diatom Thalassiosira weissflogii was grown to approximately 1 X 106 cells/mL, concentrated by centrifugation, and frozen at -80 ‚Å∞C to render the algal culture non-viable. An aliquot was thawed and resuspended to 2 g/L by dry weight using 1 ¬µm filtered and UV sterilized seawater from the Damariscotta Estuary, Gulf of Maine.  Unsterilized Damariscotta estuary seawater filtered to 1 ¬µm was used as an inoculum to induce bacterial decomposition of algal material, with 1 mL added to 15 L of algal suspension. At each timepoint (1, 2,5 & 12 days), 100 mL of the degradation slurry was vacuum filtered onto 25 mm diameter 0.3 um pore size glass fiber membranes (Sterlitech GF75) at 4 ¬∞C and stored frozen prior to proteomic analysis. Further experimental details can be found in Adams et al 2019.    Total hydrolyzable amino acids  Approximately 30 mg of material scraped from a filter was hydrolyzed in 6 N HCl as described in Cowie & Hedges (1992). Amino acids were derivatized as in Gray et al. (2017) using the AccQ Tag Ultra derivatization kit from Waters (Milford, Massachusetts). Amino acids were separated and quantified via LC-MS using a full scan method in positive ion mode, with a scan range of 100 to 600 m/z and a resolution of 60,000, on a Thermo Scientific Q Exactive HF Orbitrap.  Glutamine and asparagine are deaminated during hydrolysis and converted into glutamic and aspartic acids, respectively.    Peptide extraction Peptides were extracted with and without the use of the enzyme trypsin.  Approximately 100 mg of material was scraped from a filter into a detergent-free ammonium bicarbonate buffer following a protocol adapted from Bridoux et al. (2015) and Duffy et al (submitted). The chilled suspension was lysed via three cycles each of mechanical disruption with silica beads (50% 100 ¬µm diameter and 50% 400 ¬µm), freeze-thawing, and 30 seconds in a high-power water bath sonicator. Both the trypsin digested component and the no-digest fraction were treated identically except for the addition of trypsin, and were subjected to reduction of disulfides and carbamidomethylation.  Resultant peptides were desalted using a NestGroup macro-spin C18 column (Southborough, Massachusetts) and resuspended in 5% acetonitrile with 0.1% formic acid and Waters Hi3 E. coli peptide standard mixture (100 fmol/L).  Reverse-phase liquid chromatography-high resolution mass spectrometry (LC-HRMS) analysis was performed in duplicate with a Thermo Science (Waltham, Massachusetts) EASY-nanoLC system coupled to a Thermo Orbitrap Fusion Tribrid HRMS equipped with an NSI source. Peptides were separated on a home-packed analytical column consisting of a 37 cm long, 75-¬µm i.d. fused-silica capillary column packed with C18 particles (Magic C18AQ, 100ÀöA, 5mm; Michrom) coupled to a 4 cm long, 100 m i.d. precolumn (Magic C18AQ, 200ÀöA, 5mm; Michrom). Solvents of 100% LC/MS grade water with 0.1% formic acid (A) and 100% LC/MS grade acetonitrile with 0.1% formic acid (B) were used to elute peptides over a 120-minute gradient from 5-35% solvent B. All analyses were carried out in positive mode at an NSI spray voltage of 2 kV, and data-dependent acquisition (DDA) on the top 10 ions using first higher energy dissociation (HCD) and then electron transfer dissociation (ETD) fragmentation methods for duplicate injections.

### Data Protocol
Peptidomics Mass spectra were evaluated for presence and identification of peptides using a combination database-driven and de novo approach (Muth & Renard, 2018; O‚ÄôBryon et al., 2020).  The first step, identifying peptide spectral matches (PSMs), is database-independent and identifies the mass spectra that look peptide-like.  The PSMs are then converted to peptides via a combination of database searching and de novo sequencing.  This combining of database and de novo is termed de novo-directed peptidomics, and was performed using PeaksDB within Peaks Studio (v8.5; Bioinformatics Solutions, Waterloo, Canada; J. Zhang et al. (2012)). The de novo-directed approach has been shown to significantly improve sensitivity and accuracy in comparison to existing database search techniques (J. Zhang et al., 2012).  We used a FASTA protein database composed of 84,000 sequence entries predicted from transcriptomes of 8 T. weissflogii strains contained in the Marine Microbial Metatranscriptome Sequencing Project (NCBI BioProject PRJNA248394, Keeling et al., 2014). The database also included Gulf of Maine bacterial sequences (Li et al., 2011) as an aid in identifying heterotrophs. Search parameters included tryptic enzymatic constraint, 2 max. missed cleavages, 8 max. modifications per peptide, 15 ppm peptide mass tolerance, and 0.5 Da fragment mass tolerance. Results from technical replicates were combined.   Gene Ontology Terms and Secondary Structures To identify gene ontology (GO) term annotations, the peptide sequences were aligned to the UniProt protein database using UniPept‚Äôs metaproteomic functional analysis tool (Gurdeep Singh et al., 2019), which is almost built upon a lowest common ancestor peptide search. The GO categories were condensed from the broader set in order to eliminate redundancy using the REViGO tool available at http://revigo.irb.hr/; see Table SX for the GO exhaustive listing. Protein secondary structures for all samples were estimated using the Proteus2 algorithm (Montgomerie et al., 2008) for the combination of proteins identified by PeaksDB database searching and de novo sequencing with database mapping with PepExplorer. Output is the highest likelihood of individual amino acids being part of the following structure classes: coils (unformed), ùõÇ-helices, ùú∑-strands, membrane ùõÇ-helices, and membrane ùú∑-strands.

### Publication Abstract
None

### Keywords
Marine metaproteomics, Early diagenesis, Phytoplankton bloom, Protein cycling, Amino acids, Particulate organic matter, Marine organic matter, Peptidomics, Peptide degradation, Environmental peptidomics

### Affiliations
University of Washington
Oceanography, University of Washington, USA

### Submitter
Richard Keil

### Lab Head
Dr Richard Keil
Oceanography, University of Washington, USA


