### Accession
PXD017163

### Title
Organelle remodeling across infection time: Cytomegalovirus-host protein interactions viewed through the lens of Inter-ViSTA

### Description
Nearly all biological processes rely on the finely-tuned coordination of protein interactions across cellular space and time. Accordingly, generating protein interactome datasets has become routine in biological studies, yet effectively interpreting these datasets remains computationally challenging. Here, we describe the development of Inter-ViSTA, a computational web platform for protein Interaction Visualization in Space and Time Analysis. Inter-ViSTA enables users to quickly build animated interaction networks by automatically synthesizing information on protein abundances, functional annotations, multiprotein complexes, and subcellular localizations. We then leverage Inter-ViSTA, in conjunction with quantitative mass spectrometry and molecular virology, to define the virus-host protein interactions of the human cytomegalovirus (HCMV) anti-apoptotic protein, pUL37x1 (vMIA). We find that spatially and temporally controlled protein interactions underly pUL37x1 functions during infection, facilitating the pro-viral remodeling of both mitochondria and peroxisomes. Reciprocal isolations, microscopy, and CRISPR-based genetic tools further characterize these associations, revealing new mechanisms at the core of the pUL37x1 manipulation of mitochondrial integrity, such as interactions with the MICOS and SAMM50 complexes. Finally, we show that pUL37x1 activates the peroxisomal protein PEX11ÃŸ to regulate peroxisome fission during infection, a key metabolic aspect of virus assembly and spread.

### Sample Protocol
IP samples were digested for mass spectrometry using Filter Aided Sample Prep (FASP) and desalted via StageTips. Peptides were eluted as a single fraction in 5% ammonium hydroxide, 80% acetonitrile and analyzed by LC-MS/MS on an LTQ-Orbitrap Velos.

### Data Protocol
The MS/MS data were analyzed by Proteome Discoverer 2.2 using a FASTA file containing human and herpesvirus protein sequences and common contaminants (22,269 sequences, download 4/2016 from Uniprot). The Spectrum Files RC node and Minora Feature Detector nodes were used to perform offline mass recalibration and label-free MS1 quantitation respectively. MS/MS spectra were analyzed using Sequest HT for forward and reverse searches to determine FDR. Sequest was run with settings for a fully tryptic search with a maximum of two missed cleavages, precursor mass tolerance of 4 ppm, fragment mass tolerance of 0.3 Da, static carbamidomethylation of cysteine, dynamic phosphorylation of serine, threonine, and tyrosine, dynamic deamidation of asparagine, dynamic oxidation of methionine, and dynamic loss of methione plus acetylation of the protein N-terminus. The matched spectra were validated by the Percolator node and then PTM site localization was determined in the ptmRS node. A consensus file was then created in Proteome Discoverer 2.2 for each dataset using the Feature Mapper and Precursor Ions Quantifier nodes. Label-free MS1 signals were mapped across samples to pick up missing values from peptides not selected for sequencing in every sample and then were quantified by taking the max peak intensity for each peptide. Label-free quant data was not normalized or scaled in Proteome Discoverer and protein and peptide FDR was required to be 1%. Two unique peptide sequences were required for protein inference and only unique or razor peptides were used for label-free quantitation.

### Publication Abstract
Nearly all biological processes rely on the finely tuned coordination of protein interactions across cellular space and time. Accordingly, generating protein interactomes has become routine in biological studies, yet interpreting these datasets remains computationally challenging. Here, we introduce Inter-ViSTA (Interaction Visualization in Space and Time Analysis), a web-based platform that quickly builds animated protein interaction networks and automatically synthesizes information on protein abundances, functions, complexes, and subcellular localizations. Using Inter-ViSTA with proteomics and molecular virology, we define virus-host interactions for the human cytomegalovirus (HCMV) anti-apoptotic protein, pUL37x1. We find that spatiotemporal controlled interactions underlie pUL37x1 functions, facilitating the pro-viral remodeling of mitochondria and peroxisomes during infection. Reciprocal isolations, microscopy, and genetic manipulations further characterize these associations, revealing the interplay between pUL37x1 and the MIB complex, which is critical for mitochondrial integrity. At the peroxisome, we show that pUL37x1 activates PEX11&#x3b2; to regulate fission, a key aspect of virus assembly and spread.

### Keywords
Ap-ms, Ul37, Hcmv, Pul37x1, Ppi

### Affiliations
Princeton University
Department of Molecular Biology, Princeton University, USA

### Submitter
Joel Federspiel

### Lab Head
Dr Ileana Cristea
Department of Molecular Biology, Princeton University, USA


