### Accession
PXD033046

### Title
Pseudolycoriella hygida saliva proteome

### Description
This experiment aimed at indentifying secreted proteins/enzymes from saliva of the fly Pseudolycoriella hygida. To do this, twelve-day-old Psl. hygida larvae were selected and the last segment of the larvae was tied with elastane fishing line with the help of a stereomicroscope. This was performed to avoid contamination of saliva samples with feaces. Groups of 20 tied larvae were transferred to 1.5-mL microtubes and a few holes were made in the microtube cap. Then, the microtubes were incubated at 22 °C for 3 h. After removing the larvae, the microtubes were briefly centrifuged at 15,000 g for 30 s, and the secreted saliva, usually 2-3 ul per tube, was stored at -20 °C until gel electrophoresis.

### Sample Protocol
Thirty micrograms of protein were run on a one-dimensional reducing 12.5% SDS-PAGE for 2 h at 200 V and the resulting gel was stained for 16 h with 0.08% colloidal Coomassie Brilliant Blue G-250. For shotgun protein identification, the gel lane containing all proteins was cut in two and each piece was analysed separately. Proteins were in-gel digested with trypsin (Sequencing Grade Modified Trypsin) and the resulting peptides were run on an UltiMate 3000 RSLCnano System (Thermo Fisher Scientific) coupled to a tribrid linear ion trap-quadrupole-Orbitrap mass spectrometer (Orbitrap Fusion™ Lumos™ Tribrid™, Thermo Fisher Scientific) with a 120-min total analysis time.

### Data Protocol
Proteins were identified with MaxQuant against a database of the Psl. hygida proteome generated via in silico translation of the genome. Genome sequencing was also performed as part of this project.  Identification was performed with the following paramters: Version 1.6.3.4 User name Krijgveld lab Machine name PC-KRIJGSVELD35 Date of writing 05/19/2021 18:40:48 Include contaminants True PSM FDR 0.01 PSM FDR Crosslink 0.01 Protein FDR 0.01 Site FDR 0.01 Use Normalized Ratios For Occupancy True Min. peptide Length 7 Min. score for unmodified peptides 0 Min. score for modified peptides 40 Min. delta score for unmodified peptides 0 Min. delta score for modified peptides 6 Min. unique peptides 0 Min. razor peptides 1 Min. peptides 1 Use only unmodified peptides and True Modifications included in protein quantification Oxidation (M);Acetyl (Protein N-term) Peptides used for protein quantification Razor Discard unmodified counterpart peptides True Label min. ratio count 2 Use delta score False iBAQ True iBAQ log fit True Match between runs True Matching time window [min] 1 Alignment time window [min] 20 Find dependent peptides False Fasta file Z:\05_Projects\External_GuilhermeBrancini\B_hygida_proteome_1.fasta Decoy mode revert Include contaminants True Advanced ratios True Fixed andromeda index folder  Temporary folder  Combined folder location  Second peptides True Stabilize large LFQ ratios True Separate LFQ in parameter groups False Require MS/MS for LFQ comparisons True Calculate peak properties False Main search max. combinations 200 Advanced site intensities True Write msScans table False Write msmsScans table True Write ms3Scans table False Write allPeptides table True Write mzRange table True Write pasefMsmsScans table True Write accumulatedPasefMsmsScans table True Max. peptide mass [Da] 4600 Min. peptide length for unspecific search 8 Max. peptide length for unspecific search 25 Razor protein FDR True Disable MD5 False Max mods in site table 3 Match unidentified features False Epsilon score for mutations  Evaluate variant peptides separately True Variation mode None MS/MS tol. (FTMS) 20 ppm Top MS/MS peaks per Da interval. (FTMS) 12 Da interval. (FTMS) 100 MS/MS deisotoping (FTMS) True MS/MS deisotoping tolerance (FTMS) 7 MS/MS deisotoping tolerance unit (FTMS) ppm MS/MS higher charges (FTMS) True MS/MS water loss (FTMS) True MS/MS ammonia loss (FTMS) True MS/MS dependent losses (FTMS) True MS/MS recalibration (FTMS) False MS/MS tol. (ITMS) 0.5 Da Top MS/MS peaks per Da interval. (ITMS) 8 Da interval. (ITMS) 100 MS/MS deisotoping (ITMS) False MS/MS deisotoping tolerance (ITMS) 0.15 MS/MS deisotoping tolerance unit (ITMS) Da MS/MS higher charges (ITMS) True MS/MS water loss (ITMS) True MS/MS ammonia loss (ITMS) True MS/MS dependent losses (ITMS) True MS/MS recalibration (ITMS) False MS/MS tol. (TOF) 40 ppm Top MS/MS peaks per Da interval. (TOF) 10 Da interval. (TOF) 100 MS/MS deisotoping (TOF) True MS/MS deisotoping tolerance (TOF) 0.01 MS/MS deisotoping tolerance unit (TOF) Da MS/MS higher charges (TOF) True MS/MS water loss (TOF) True MS/MS ammonia loss (TOF) True MS/MS dependent losses (TOF) True MS/MS recalibration (TOF) False MS/MS tol. (Unknown) 0.5 Da Top MS/MS peaks per Da interval. (Unknown) 8 Da interval. (Unknown) 100 MS/MS deisotoping (Unknown) False MS/MS deisotoping tolerance (Unknown) 0.15 MS/MS deisotoping tolerance unit (Unknown) Da MS/MS higher charges (Unknown) True MS/MS water loss (Unknown) True MS/MS ammonia loss (Unknown) True MS/MS dependent losses (Unknown) True MS/MS recalibration (Unknown) False

### Publication Abstract
Soil biota has a crucial impact on soil ecology, global climate changes, and effective crop management and studying the diverse ecological roles of dipteran larvae deepens the understanding of soil food webs. A multi-omics study of <i>Pseudolycoriella hygida</i> comb. nov. (Diptera: Sciaroidea: Sciaridae) aimed to characterize carbohydrate-active enzymes (CAZymes) for litter degradation in this species. Manual curation of 17,881 predicted proteins in the <i>Psl</i>. <i>hygida</i> genome identified 137 secreted CAZymes, of which 33 are present in the saliva proteome, and broadly confirmed by saliva CAZyme catalytic profiling against plant cell wall&#xa0;polysaccharides and <i>p</i>NP-glycosyl substrates. Comparisons with two other sciarid species and the outgroup <i>Lucilia cuprina</i> (Diptera: Calliphoridae) identified 42 CAZyme families defining a sciarid CAZyme profile. The litter-degrading potential of sciarids corroborates their significant role as decomposers, yields insights to the evolution of insect feeding habits, and highlights the importance of insects as a source of biotechnologically relevant enzymes.

### Keywords
Bradysia; diptera; proteomics; saliva; glycosyl hydrolases; biomass

### Affiliations
Departamento de Química, Faculdade de Filosofia, Ciências e Letras de Ribeirão Preto, Universidade de São Paulo, Ribeirão Preto – SP, Brazil
Universidade de São Paulo

### Submitter
Richard Ward

### Lab Head
Dr Richard John Ward
Departamento de Química, Faculdade de Filosofia, Ciências e Letras de Ribeirão Preto, Universidade de São Paulo, Ribeirão Preto – SP, Brazil


