### Accession
PXD012576

### Title
Proteomics of different types of immunity in potato

### Description
Plants have a wide variety of ways to defend against pathogens. A commonly used model of the plant immune system divides it into a general response triggered by pathogen associated molecular patterns (PAMPs) and a specific response triggered by effectors. The first type of response is known as PAMP triggered immunity (PTI) and the second as effector-triggered immunity (ETI). We have performed a proteomical analysis of one PTI and two ETI models in potato and compared their effect on protein expression.

### Sample Protocol
The proteins were separated on an SDS-PAGE gel. The whole lane was removed and washed, and the proteins were digested using trypsin. The digests were desalted by C18-based spin columns and analysed with HPLC-MS/MS analysis with the application of an Eksigent nanoLC2D HPLC system with an online to an LTQ Orbitrap XL ETD.

### Data Protocol
The raw data was converted to Mascot generic files (mgf) with ProteoWizard. A protein database consisting of Solanum proteins from UniProt (www.uniprot.org), downloaded 24 August 2011; protein sequences from the Potato Genome Project and the Agrobacterium proteins from UniProt, downloaded 10 March 2015, concatenated with an equal size decoy database was generated. The mgf files were used for searches against this database with Mascot version 2.3.01 in the Proteios software environment. Search tolerances were 7 ppm for precursors and 0.5 Da for MS/MS fragments. For the searches used for quantitative analyses, one missed cleavage was allowed and carbamidomethylation of cysteine residues was used as fixed modification and oxidation of methionines as variable modification. For the identification of methylated proteins, carbamidomethylation of cysteine was used as a fixed modification while oxidation of methionines, mono-, di-, and trimethylation of lysine and mono-, di-, and trimethylation of arginine were used as variable modifications. The results were imported to Proteios where q values were calculated using the target-decoy method described by KÃ¤ll et al. The search results were then filtered at a peptide-spectrum match q-value of 0.01 to obtain a false discovery rate of 1% in the filtered list. For quantitative analysis, a label-free approach based on precursor ion intensities was used with all data processing steps performed within Proteios. MS1 peptide feature detection was performed using Dinosaur, while the other data processing steps were performed in Proteios. The resulting peptide data was normalized using Loess-G normalization in the Normalyzer software. The normalized data was analyzed using DanteR.

### Publication Abstract
None

### Keywords
Potato, Pti, Effector triggered immunity, Eti, Immunity, Pamp-triggered immunity

### Affiliations
Department of Plant Protection Biology
Swedish University of Agricultural Sciences, Department of Plant Protection Biology

### Submitter
Svante Resjo

### Lab Head
Dr Erik Andreasson
Swedish University of Agricultural Sciences, Department of Plant Protection Biology


