### Accession
PXD011984

### Title
UHPLC-MS/MS analysis of the proteomic profiles of cocoa beans from different genotypes

### Description
Amongst other compounds, cocoa flavour is dependent on peptides that are formed during fermentation of cocoa beans from proteins such as albumin and vicilin. In this study the proteomic profiles of cocoa beans from four genotypes (ICS 1, ICS 39, IMC 67 and SCA 6) with different genetic background and flavour profiles have been analysed by employing a bottom-up label free LC-MS/MS approach. From a total of 430 identified proteins, 250 proteins were found significantly differentially expressed among the four cocoa genotypes analysed. Of these, 61 proteins with a fold change of 2 or more were further investigated, showing that the majority is involved in stress response. Furthermore, several of these 61 proteins could also be linked to oxidation-reduction processes. PCA analysis allowed a clear separation of the genotypes based on their proteomic profile, with an aminohydrolase and a sulphite oxidase greatly contributing to the separation. Aspartyl protease was more abundant in the genotypes ICS 1 and ICS compared to IMC 67, while a serine carboxypeptidase was significantly more expressed in the genotype ICS 39 in comparison with the other genotypes. Both these enzymes catalyse the degradation of storage proteins during fermentation. A Beta-amylase, an enzyme which catalyses the release of maltose was detected at a significantly higher level in the genotype SCA 6 compared to ICS 1 and IMC 67.  Two Amine oxidases were significantly more abundant in the genotype SCA 6 compared to ICS 39, and in the genotype ICS 1 versus ICS 39, while two alcohol dehydrogenase were higher expressed in the genotype SCA 6 compared to IMC 67. These enzyme catalyse oxidation of amines and alcohols with release of aldehydes and ketones. The data shows that UHPLC-MS/MS can be employed to differentiate cocoa beans from various varieties, and thus in theory be linked to differences in their flavour profile.

### Sample Protocol
The cocoa seeds were freeze-dried and defatted using petroleum ether. Polyphenols were then extracted from the defatted cocoa seeds to avoid complexation with protein. Proteins were extracted from the cocoa powder with a solution containing chaotropic agents. An aliquot of each sample solution containing a total amount of approximately 160 Âµg of proteins based on the Bradford assay, was reduced and alkylated, and subsequently digested with trypsin (1:50 ratio). The trypsin digests were desalted and analyzed on a UHPLC-MS/MS system consisting of an Orbitrap Qexactive (Thermo Scientific) mass spectrometer coupled to a Dionex Ultimate 3000 (Thermo Scientific) UHPLC system.

### Data Protocol
All MS/MS spectra were processed using Mascot Distiller software (Matrix Science Ltd, London, UK; Version 2.7.1.0) in order to convert the raw LC-MS/MS data into peak lists suitable for database searching using the Mascot search routine. For the evaluation of the effect of harvest time and tree, a Mascot server Version 2.4.1 was used, while the analysis of the different cocoa genotypes was carried out employing a Mascot Server Version 2.6 (Matrix Science Ltd). Mascot searches were carried out against the Cacao Matina 1-6 Genome v1.1 Theobroma cacao database (http://www.cacaogenomedb.org/Tcacao_genome_v1.1#tripal_analysis-downloads-box; accessed on 31st May 2015; 59,577 sequences; 23,720,084 residues), and a custom made contaminants database (70 sequences; 31,845 residues). Functional annotation was carried out by matching the proteins accession codes from the Cacao Matina 1-6 Genome v1.1 Theobroma cacao to the GoMapMan database (http://protein.gomapman.org). Label-free quantitation was carried out using replicate protocol with Mascot Distiller software.

### Publication Abstract
In this study the proteomic profiles of cocoa beans from four genotypes with different flavour profiles were analysed by bottom-up label-free UHPLC-MS/MS. From a total of 430 identified proteins, 61 proteins were found significantly differentially expressed among the four cocoa genotypes analysed with a fold change of &#x2265;2. PCA analysis allowed clear separation of the genotypes based on their proteomic profiles. Genotype-specific abundances were recorded for proteases involved in the degradation of storage proteins and release of flavour precursors. Different genotype-specific levels of other enzymes, which generate volatiles compounds that could potentially lead to flavour-inducing compounds, were also detected. Overall, this study shows that UHPLC-MS/MS data can differentiate cocoa bean varieties.

### Keywords
Cocoa beans proteome, Cocoa beans, Plant proteomics, Label free quantitation, Theobroma cacao

### Affiliations
University of Reading

### Submitter
Emanuele Scollo

### Lab Head
Dr Professor Rainer Cramer
University of Reading


