### Accession
PXD030423

### Title
A super-enhancer regulated RNA-binding protein cascade drives pancreatic cancer

### Description
Pancreatic ductal adenocarcinoma (PDAC) is a lethal malignancy in need of new therapeutic options. Using an unbiased analysis of super-enhancers (SEs) as sentinels of core genes involved in cell-specific function, we uncover a druggable SE-mediated RNA-binding protein (RBP) cascade that supports PDAC growth through enhanced mRNA translation. This cascade is driven by a SE associated with the RBP heterogeneous nuclear ribonucleoprotein F, which stabilizes protein arginine methyltransferase 1 (PRMT1) to, in turn, control the translational mediator ubiquitin-associated protein 2-like. All three of these genes and the regulatory SE are essential for PDAC growth and coordinately regulated by the Myc oncogene. In line with this, modulation of the RBP network by PRMT1 inhibition reveals a unique vulnerability in Myc-high PDAC patient organoids and markedly reduces tumor growth in vivo. Our study highlights a functional link between epigenetic regulation and mRNA translation and identifies components that comprise unexpected therapeutic targets for PDAC.

### Sample Protocol
The PTMScan Asymmetric Di-Methyl Arginine Motif [adme-R] Kit (Cell Signaling Cat# 13474) was used to immunoprecipitate arginine-asymmetrically dimethylated peptides from 11 x 15 cm tissue culture plates at 85% confluence, according to the manufacturer's instruction. The enriched samples were labeled with dimethyl 30.0439 as the light channel (WT) and 36.0757 as the heavy channel (KO). The labeled samples were pooled and analyzed by LCMS on a Fusion Lumos mass spectrometer (Thermo). The digest was injected directly onto a 30 cm, 75 ùúám ID column packed with BEH 1.7 ùúám C18 resin (Waters). Samples were separated at a flow rate of 400 nl/min on a nLC 1200 (Thermo). Solutions A and B were 0.1% formic acid in water and 0.1% formic acid in 90% acetonitrile, respectively. A gradient of 1-30% B over 100 min, an increase to 50% B over 20 min, an increase to 90% B over 10 min and held at 100% B for a final 10 min was used for 140 min total run time. The column was re-equilibrated with 20 ùúál of A prior to the injection of sample. Peptides were eluted directly from the tip of the column and nanosprayed directly into the mass spectrometer by application of 2.5 kV at the back of the column. The Lumos was operated in a data dependent mode. Full MS scans were collected in the Orbitrap at 120K resolution with a mass range of 400 to 1500 m/z and an AGC target of 4e5. The cycle time was set to 3 sec, and within these 3 secs, the most abundant ions per scan were selected for HCD MS/MS with an AGC target of 4e5 and 15K resolution. Maximum fill times were set to 50 ms and 100 ms for MS and MS/MS scans, respectively. Quadrupole isolation at 1.6 m/z was used, monoisotopic precursor selection was enabled, and dynamic exclusion was used with exclusion duration of 5 sec.

### Data Protocol
Protein and peptide identification were done with Integrated Proteomics Pipeline ‚Äì IP2 (Integrated Proteomics Applications). Tandem mass spectra were extracted from raw files using RawConverter and searched with ProLuCID against Uniprot mouse database. The search space included all fully-tryptic and half-tryptic peptide candidates, carbamidomethylation on cysteine was considered as a static modification, methylation and dimethylation were considered variable modifications on arginine. Heavy and light dimethyl labeling were considered as modifications on N-terminus and lysine. Data was searched with 50 ppm precursor ion tolerance and 600 ppm fragment ion tolerance. Identified proteins were filtered to 10 ppm precursor ion tolerance using DTASelect and utilizing a target-decoy database search strategy to control the false discovery rate to 1% at the peptide level. The modified peptides of UBAP2L were manually validated and dimethyl quantitation was done with Skyline.

### Publication Abstract
None

### Keywords
Prmt1, Tumor growth, Pancreatic cancer, Rna-binding protein, Super-enhancers

### Affiliations
Gene Expression Laboratory, Salk Institute for Biological Studies, La Jolla, CA 92037, USA
Salk

### Submitter
Jolene Diedrich

### Lab Head
Dr Ronald M. Evans
Gene Expression Laboratory, Salk Institute for Biological Studies, La Jolla, CA 92037, USA


