### Accession
PXD012162

### Title
Proteomic profiling of breast cancer-derived extracellular vesicles

### Description
In-depth (phospho)proteome characterization of breast cancer derived extracellular vesicles (EVs).

### Sample Protocol
Extracellular vesicles were purified using differential ultracentrifugation. Purified EV pellets were lysed in Urea lysis buffer (8M urea buffer in 50 mM ammonium bicarbonate pH 8.5, Complete mini EDTA-free protease inhibitor cocktail (Roche), PhosSTOP phosphatase inhibitor cocktail (Roche)) and further sonicated using a BioruptorÂ® Plus sonication device (Diagenode) for 15 cycles (30 sec on, 30 sec off). Extracted proteins were then reduced (4 mM DTT) and alkylated (8 mM iodoacetamide) before being digested with LysC (Wako, Richmond, VA, USA) for 4 h (enzyme/substrate ratio 1:75) at 37Â°C. Samples were diluted four times and digested further by Trypsin (Promega, Madison, WI, USA) (enzyme/substrate ratio 1:100) at 37Â°C overnight. Finally, the digestion was quenched with 5% formic acid and the resulting peptides were cleaned-up on SepPak C18 cartridges (Waters Corporation, Milford, MA) and further used directly for single run proteome analysis or submitted to phosphorylation enrichment.

### Data Protocol
All raw MS files were analyzed with the MaxQuant software (version 1.5.8.3). MS/MS spectra were searched by Andromeda against a reviewed homo sapiens Uniprot database using the following parameters: trypsin digestion; maximum of 2 missed cleavages; cysteine carbamidomethylation as fixed modification; oxidized methionine, protein N-terminal acetylation, and serine/threonine/tyrosine phosphorylation (for the phosphoproteome data analysis only) as variable modifications. Mass tolerance was set to 4.5 ppm and 20 ppm for the MS1 and MS2 respective. The protein and PSM False Discovery Rate (FDR) were set to 1%. Peptide identifications by MS/MS were transferred between runs to replace missing values for quantification, with a 0.7-min window after retention time alignment.

### Publication Abstract
Extracellular vesicles (EVs) are a potential source of disease-associated biomarkers for diagnosis. In breast cancer, comprehensive analyses of EVs could yield robust and reliable subtype-specific biomarkers that are still critically needed to improve diagnostic routines and clinical outcome. Here, we show that proteome profiles of EVs secreted by different breast cancer cell lines are highly indicative of their respective molecular subtypes, even more so than the proteome changes within the cancer cells. Moreover, we detected molecular evidence for subtype-specific biological processes and molecular pathways, hyperphosphorylated receptors and kinases in connection with the disease, and compiled a set of protein signatures that closely reflect the associated clinical pathophysiology. These unique features revealed in our work, replicated in clinical material, collectively demonstrate the potential of secreted EVs to differentiate between breast cancer subtypes and show the prospect of their use as non-invasive liquid biopsies for diagnosis and management of breast cancer patients.

### Keywords
Extracellular vesicles, Breast cancer

### Affiliations
Utrecht University
Biomolecular Mass Spectrometry & Proteomics Group  Utrecht University

### Submitter
Stamatia Rontogianni

### Lab Head
Dr A.F.M. (Maarten) Altelaar
Biomolecular Mass Spectrometry & Proteomics Group  Utrecht University


