### Accession
PXD011211

### Title
Chaperonin CCT Checkpoint Function in Basal Transcription Factor TFIID Assembly

### Description
TFIID is a cornerstone of eukaryotic gene regulation. Distinct TFIID complexes with unique subunit composition exist and several TFIID subunits are shared with other complexes, conveying intricate cellular decision making to control subunit allocation and functional assembly of this essential transcription factor. However, the underlying molecular mechanisms remain poorly understood. Here, we used quantitative proteomics to examine TFIID submodules and assembly mechanisms in human cells. Structural and mutational analysis of the cytoplasmic TAF5/TAF6/TAF9 submodule identified novel interactions crucial for TFIID integrity, and for allocating TAF9 to TFIID or the SAGA co-activator complex. We discover a key checkpoint function for the chaperonin CCT, which specifically associates with nascent TAF5 for subsequent handover to TAF6/TAF9 and ultimate holo-TFIID formation. Our findings illustrate at the molecular level how multisubunit complexes are crafted in the cell, involving checkpoint decisions facilitated by a chaperone machine.

### Sample Protocol
Protein lysates were incubated in binding buffer (20 mM Hepes-KOH pH 7.9, 300 mM NaCl, 20% glycerol, 2 mM MgCl2, 0.2 mM EDTA, 0.1% NP-40, 0.5 mM DTT and 1x Roche protease inhibitor cocktail) on a rotating wheel for 1 h at 4o C in triplicates with GBP-coated agarose beads (Chromotek) or control agarose beads (Chromotek). The beads were washed two times with binding buffer containing 0.5% NP-40, two times with PBS containing 0.5% NP-40, and two times with PBS. On-bead digestion of bound proteins was performed overnight in elution buffer (100 mM Tris-HCl pH 7.5, 2 M urea, 10 mM DTT) with 0.1 Âµg/ml of trypsin at RT and eluted tryptic peptides were bound to C18 stagetips prior to mass spectrometry analysis. Tryptic peptides were eluted from the C18 stagetips in H2O:acetonitril (50:50) with 0.1% formic acid and dried prior to resuspension in 10% formic acid. A third of this elution was injected into Q-Exactive (Thermo Fischer) in the MS/MS mode with 90 min total analysis time. Blank samples consisting of 10% formic acid were run for 45 min between GFP and non-GFP samples, to avoid carry-over between runs.

### Data Protocol
The raw data files were analyzed with MaxQuant software (version 1.5.3.30) using Uniprot human FASTA database. Label-free quantification values (LFQ) and match between run options were selected. Intensity based absolute quantification (iBAQ) algorithm was also activated for subsequent relative protein abundance estimation. The obtained protein files were analyzed by Perseus software (MQ package, version 1.5.4.0), in which contaminants and reverse hits were filtered out. Protein identification based on non-unique peptides as well as proteins identified by only one peptide in the different triplicates were excluded to increase protein prediction accuracy. For identification of the bait interactors LFQ intensity-based values were transformed on the logarithmic scale (log2) to generate Gaussian distribution of the data. This allows for imputation of missing values based on the normal distribution of the overall data (in Perseus, width = 0.3; shift = 1.8). The normalized LFQ intensities were compared between grouped GFP triplicates and non-GFP triplicates, using 1% permutation-based false discovery rate (FDR) in a two-tailed t-test. The threshold for significance (S0), based on the FDR and the ratio between GFP and non-GFP, samples was kept at the constant value of 1 for comparison purposes. Relative abundance plots were obtained by comparison of the iBAQ values of GFP interactors. The values of the non-GFP iBAQ values were subtracted from the corresponding proteins in the GFP pull-down and were next normalized on a chosen co-purifying protein for scaling and data representation purposes.

### Publication Abstract
TFIID is a cornerstone of eukaryotic gene regulation. Distinct TFIID complexes with unique subunit compositions exist and several TFIID subunits are shared with other complexes, thereby conveying precise cellular control of subunit allocation and functional assembly of this essential transcription factor. However, the molecular mechanisms that underlie the regulation of TFIID remain poorly understood. Here we use quantitative proteomics to examine TFIID submodules and assembly mechanisms in human cells. Structural and mutational analysis of the cytoplasmic TAF5-TAF6-TAF9 submodule identified novel interactions that are crucial for TFIID integrity and for allocation of TAF9 to TFIID or the Spt-Ada-Gcn5 acetyltransferase (SAGA) co-activator complex. We discover a key checkpoint function for the chaperonin CCT, which specifically associates with nascent TAF5 for subsequent handover to TAF6-TAF9 and ultimate holo-TFIID formation. Our findings illustrate at the molecular level how multisubunit complexes are generated within the cell via mechanisms that involve checkpoint decisions facilitated by a chaperone.

### Keywords
Proteomics, Chaperonin cct, Tfiid, Crystal structure

### Affiliations
RMCU, University Medical Center, Utrecht 
Professor of Medical Epigenetics German Cancer Consortium (DKTK), partner site Freiburg German Cancer Research Center (DKFZ) Foundation under Public Law c/o Zentrale Klinische Forschung (ZKF) University Medical Center Freiburg Breisacher Str. 66, 79106 Freiburg, Germany

### Submitter
Simona Antonova

### Lab Head
Dr Marc Timmers
Professor of Medical Epigenetics German Cancer Consortium (DKTK), partner site Freiburg German Cancer Research Center (DKFZ) Foundation under Public Law c/o Zentrale Klinische Forschung (ZKF) University Medical Center Freiburg Breisacher Str. 66, 79106 Freiburg, Germany


