### Accession
PXD002828

### Title
Anopheles culicifacies species A and species B Midgut Proteome

### Description
Background: Mosquito midgut is an important target for the host parasite interaction studies as it plays a major role in parasite growth and maturation and vector susceptibility. Proteomic approaches coupled with bioinformatics analysis have been used to study expression of functional proteins/enzymes of An. culicifacies susceptible and refractory species midgut in order to understand the mechanism of refractoriness that may help in contributing to unravel the host pathogen interactions.  Methodology/Principle findings: In the present study proteomics approaches namely in solution and in gel digestion strategies followed by LC/MS/MS analysis were used. Further bioinformatics analysis were carried out to find out the functional annotated proteins, biological process, molecular function and their sub-cellular location using Gene ontology, SMART analysis, CELLO etc. In solution and in gel approach coupled with LC-MS/MS identified a total of 91 proteins in susceptible species and 69 proteins in refractory species. Comparative analysis between susceptible and refractory species of An. culicifacies indicated that mainly proteins involved in proteolysis mechanism, catalytic activity, peptidases activity and immune related proteins were found to be dominating in refractory as compared to susceptible species. Conclusion/Significance: Based on the present data a significant increase in number of proteins in midgut of refractory An. culicifacies species B were found that may conclude that these proteins may be responsible for the inhibiting parasite growth and linked to the melanization of oocysts or parasite lysis mechanisms in natural populations of refractory mosquito. Hence the progress of our studies at protein level suggests that these identified annotated putative proteins/enzymes may help to explore natural vector-parasite systems and reveal valuable insights into the mechanism of refractoriness which in turn further useful for bringing of novel strategies for control of malaria.

### Sample Protocol
Midgut dissection and extract preparation  100 mosquitoes each of sugar fed An. culicifacies species A and species B were collected for midgut dissection. Firstly, mosquitoes were chilled on ice for 15-20 minutes. Midgut tissue were isolated by a longitudinal section during mosquito dissection by pulling and pooled in a 100 ¬µl lysis buffer (50mM Tris, 150mM NaCl, 1% NP-40, 1 tablet of protease inhibitor). Pooled midgut (100) was sonicated for 3 pulses for 15-20 sec on ice. Homogenized midgut tissue was centrifuged at 5000 rpm for 5 mins and supernatant was collected and stored at ‚àí800C in a fresh tube for in solution and in gel digestion approach. Total protein concentration was quantified by Bradford method (Sigma) using bovine serum albumin (BSA) as a standard.    In solution digestion The extracted midgut lysates after quantification were in-solution digested with trypsin. Lysates were reduced and alkylated and trypsin digested. Briefly, lystaes were denatured with urea first and then reduced with DTT (10mM) in ammonium biocarbonate solution for 1 hour at 56 0C. Next step is alkylation of reduced protein with IAA (25mM) in ammonium biocarbonate for 30 min at 250C. To alkylated protein mixture ammonium bicarbonate (100mM) was added in order to dilute urea to a final concentration (0.5M). Next step is digestion in which trypsin was added to a protein mixture with concentration about 50 times lower than protein mixture and incubated the mixture overnight at 37 0C. These peptide mixtures were used further for mass spectrometry analysis.  SDS PAGE  Midgut tissue samples were separated on gel electrophoresis for in gel digestion strategy.  Briefly, midgut sample (30 ùúág) was mixed in sample buffer (0.625M Tris HCl, 10% SDS, glycerol, and distilled water) containing ùõΩ-mercaptoethanol (10% vol/vol). After mixing, sample were heated at 950C in a water bath for 5min and loaded onto a 12% acrylamide gel. This was subjected to electrophoresis on a Bio-Rad apparatus (Bio-Rad, USA). Protein molecular weight markers (Genei protein range marker, Bangalore Genei) were used in the gel. After complete run silver staining was applied according to the manufacturer‚Äôs protocol (G-Biosciences). Stained gel was visualized in gel documentation (UVitech). This gel was then excised into different bands and stored in separate tubes. After proper excision, these gel bands were followed by digestion with proteomics grade trypsin (Roche diagnostics).  In-Gel Protein Digestion  Silver stained gel bands containing mosquito midgut proteins were in-gel digested with trypsin. Separate gel bands were destained, reduced and alkylated and trypsin digested. Briefly, the excised gel slices were subjected to reduction and were dried in a vacuum centrifuge. Sample was reduced with DTT (10mM) in ammonium bicarbonate (100mM), alkylated with IAA (55mM), dehydrated with acetonitrile and digestion with trypsin (12.5 ng/ùúáL) in an ice cold bath. The detailed methodology was applied same as in our earlier publications (Vijay et al, 2014).    Mass spectrometry conditions   After trypsinization of the midgut protein lysates, peptides were desalted using C18 packed ziptips. Desalted peptides were dried down using a speed vac and reconstituted in 2% acetonitrile with 0.1% Formic acid. In case of in gel digested sample, after trypsinization of the different gel bands, peptides were extracted using 50 % acetonitrile containing 5% formic acid.  Extracted peptides were dried down using a speed vac and reconstituted in 2% Acetonitrile with 0.1% Formic acid. Both in solution and in gel digested samples were loaded onto a PicoFrit C18 nanospray column (New Objective) using a Thermo Scientific Surveyor Autosampler operated in the no waste injection mode. Peptides from the in-solution digests were eluted from the column using a linear acetonitrile gradient from 2 to 35% acetonitrile over 210 minutes and peptides from the in-gel digests were eluted from the column using a linear acetonitrile gradient from 2 to 32% acetonitrile over 85 minutes followed by high and low organic washes for another 5 minutes into an LTQ XL mass spectrometer (Thermo Scientific) via a nanospray source with the spray voltage set to 1.8kV and the ion transfer capillary set at 180 0C. A data-dependent top 7 method was used where a full MS scan from m/z 350-1500 was followed by MS/MS scans on the seven most abundant ions.

### Data Protocol
LC/MS/MS derived data were searched and analyzed using Proteome Discoverer 1.4 (Thermo Scientific). Raw data files were converted into respective format. For MS/MS search SEQUEST algorithm was used against the most recent species-specific mosquito database (Anopehles sp, Aedes sp, Cules sp etc) downloaded from NCBI. Search parameters were: trypsin was the selected enzyme allowing for up to two missed cleavages per peptide, Carbamidomethyl Cysteine was used as a static modification and Oxidation of Methionine as a variable modification. Proteins were identified when two or more unique peptides had X-correlation scores greater than 1.5, 2.0, and 2.5 for respective charge states of +1, +2, and +3.  Annotations of the proteins were identified using bioinformatics analysis. Biological functions and Biological process were found using gene ontology (GO) standards. Cellular localization was identified using CELLO (http://cello.life.nctu.edu.tw/) (Yu et al, 2006). Conserve domains were identified using SMART programme (http://smart.embl-heidelberg.de/) (18), Interproscan analysis. Along with that signal peptides were also found out using SignalP 4.1 (http://www.cbs.dtu.dk/services/SignalP/) of identified proteins (19). Network pathway analysis as an evidence view was identified using String 9.0 database (Search Tool for the Retrieval of Interacting Genes/Proteins) (http://string-db.org/).

### Publication Abstract
None

### Keywords
Refractory, Anopheles, Midgut, Lc/ms/ms

### Affiliations
National Institute of Malaria Research
Dr. Arun Sharma Protein Biochemistry and Structural Biology Division National Institute of Malaria Research (ICMR) Sector-8, Dwarka, New Delhi- 110077 E-mail: arushar@gmail.com

### Submitter
sonam vijay

### Lab Head
Dr Dr Arun Sharma
Dr. Arun Sharma Protein Biochemistry and Structural Biology Division National Institute of Malaria Research (ICMR) Sector-8, Dwarka, New Delhi- 110077 E-mail: arushar@gmail.com


