### Accession
PXD002436

### Title
Monitoring cellular phosphorylation signaling pathways into chromatin and down to the gene level

### Description
Protein phosphorylation, one of the most common and important modifications of acute and reversible regulation of protein function, plays a dominant role in almost all cellular processes. These signaling events regulate cellular responses including proliferation, differentiation, metabolism, survival and apoptosis. Several studies have been successfully used to identify phosphorylated proteins and dynamic changes in phosphorylation status after stimulation. Nevertheless, it is still rather difficult to elucidate precise complex phosphorylation signaling pathways. In particular, how signal transduction pathways directly communicate from the outer cell surface through cytoplasmic space and then directly into chromatin networks to change the transcriptional and epigenetic landscape remains poorly understood. Here we describe the optimization and comparison of methods based on thiophosphorylation affinity enrichment, which can be utilized to monitor phosphorylation signaling into chromatin by isolation of phosphoprotein containing nucleosomes, a method we term phosphorylation-specific chromatin affinity purification (PS-ChAP). We utilized this PS-ChAP approach in combination with quantitative proteomics to identify changes in the phosphorylation status of chromatin bound proteins on nucleosomes following perturbation of transcriptional processes. We also demonstrate that this method can be employed to map phosphoprotein signaling into chromatin containing nucleosomes through identifying the genes those phosphorylated proteins are found on via thiophosphate PS-ChAP-qPCR. Thus, our results showed that PS-ChAP offers a new strategy for studying cellular signaling and chromatin biology, allowing us to directly and comprehensively investigate phosphorylation signaling into chromatin to investigate if these pathways are involved in altering gene expression.

### Sample Protocol
UltraLink Iodoacetyl Resin (iodoacetyl-beads) were washed twice using 50mM Tris buffer before being mixed with the peptides/proteins at a ratio of 200Âµl beads to 1 mg peptide/proteins and rotated at room temperature overnight (protected from light). For the peptide enrichment, the beads were washed by water, 5M NaCl, water, 50% ACN and 5% formic acid then eluted by oxone. While for the protein enrichment, the beads were washed by 50mM Tris buffer and HEPES 100 (20mM HEPES, 5mM MgCl2, 100mM NaCl) then followed by trypsin digestion. The beads were also eluted using oxone

### Data Protocol
pFind Studio 2.8 was used for data analysis, including pFind for database search (1, 2) and pBuild for result validation. The data were searched using pFind against the human IPI database (version 3.87, 91,464 sequences), using a mass tolerance of 20 ppm for precursor ions and 0.5 Da for fragment ions. Serine, threonine and tyrosine phosphorylation, methionine oxidation, lysine and arginine SILAC label were set as variable modifications and up to two missed cleavages were allowed for trypsin digestion. The target and reversed database was used for searching, and pBuild used the target-decoy approach to filter search results with false discovery rate less than 1% at the MS/MS level (3), and then obtained MS/MS based peptide and protein identifications.

### Publication Abstract
Protein phosphorylation, one of the most common and important modifications of acute and reversible regulation of protein function, plays a dominant role in almost all cellular processes. These signaling events regulate cellular responses, including proliferation, differentiation, metabolism, survival, and apoptosis. Several studies have been successfully used to identify phosphorylated proteins and dynamic changes in phosphorylation status after stimulation. Nevertheless, it is still rather difficult to elucidate precise complex phosphorylation signaling pathways. In particular, how signal transduction pathways directly communicate from the outer cell surface through cytoplasmic space and then directly into chromatin networks to change the transcriptional and epigenetic landscape remains poorly understood. Here, we describe the optimization and comparison of methods based on thiophosphorylation affinity enrichment, which can be utilized to monitor phosphorylation signaling into chromatin by isolation of phosphoprotein containing nucleosomes, a method we term phosphorylation-specific chromatin affinity purification (PS-ChAP). We utilized this PS-ChAP(1) approach in combination with quantitative proteomics to identify changes in the phosphorylation status of chromatin-bound proteins on nucleosomes following perturbation of transcriptional processes. We also demonstrate that this method can be employed to map phosphoprotein signaling into chromatin containing nucleosomes through identifying the genes those phosphorylated proteins are found on via thiophosphate PS-ChAP-qPCR. Thus, our results showed that PS-ChAP offers a new strategy for studying cellular signaling and chromatin biology, allowing us to directly and comprehensively investigate phosphorylation signaling into chromatin to investigate if these pathways are involved in altering gene expression. The mass spectrometry proteomics data have been deposited to the ProteomeXchange Consortium with the data set identifier PXD002436.

### Keywords
Proteomics thiophosphorylation signaling

### Affiliations
Epigenetics Program Department of Biochemistry and Biophysics Smilow Center for Translational Research Perelman School of Medicine University of Pennsylvania
upenn

### Submitter
yumiao han

### Lab Head
Dr Benjamin A. Garcia
Epigenetics Program Department of Biochemistry and Biophysics Smilow Center for Translational Research Perelman School of Medicine University of Pennsylvania


