### Accession
PXD014576

### Title
Comprehensive proteomic analysis of mouse embryonic fibroblast lysosomes by mass spectrometry

### Description
The lysosome is the main degradative organelle in eukaryotic cells. It is involved in diverse cellular functions from intracellular digestion to metabolic signaling. Despite various efforts to enrich for lysosomes and analyze their proteome with mass spectrometry based proteomics approaches, no systematic evaluation of sample preparation parameters for the analysis of lysosome enriched fractions has been performed. Using samples enriched for lysosomes with paramagnetic nanoparticles, we conducted a systematic evaluation of the protocol for the analysis of the lysosomal proteome. We compared centrifugation and protein precipitation for the concentration of lysosome enriched fractions and tested four different digestion methods in combination with each of these approaches. This included filter aided sample preparation (FASP), in-gel digestion, and in solution digestion using either RapiGest or Urea. Furthermore, we evaluated three formats for sample desalting, four gradient lengths for LC-MSMS analysis of unfractionated samples, as well as fractionation by strong anion exchange tip columns into three or six fractions. Using the combined data, we generated a draft map of lysosomal proteome for mouse embryonic fibroblasts as well as a spectral library for the analysis of lysosomes by data independent acquisition (DIA) and evaluated different LC gradient lengths for DIA.

### Sample Protocol
Lysosomes enriched fractions were obtained from mouse embryonic fibroblasts (MEFs) using superparamagnetic iron oxide nanoparticles (SPIONs). For sample preparation, isolated lysosomes were centrifuged at 4Â°C, 20,000 g for 30 min to have a pellet while precipitation of lysosomal proteins was performed by chloroform/methanol extraction method. The samples were tryptically digested in-gel, in solution using urea and RapiGest, as well as by FASP. The digested samples were then desalted by C18 STAGE tips. Urea digested peptides were fractionated by SAX tips in 6 or 3 fractions and subsequently desalted by C18 STAGE tips. The non-fractionated samples were desalted by Oasis HLB cartridges. For desalting comparison, the urea digested samples were desalted using Oasis/Sep-Pak cartridges or C18 STAGE tips. Analyses were performed using a Dionex Ultimate 3000 system coupled to an Orbitrap Fusion Lumos mass spectrometer (both Thermo Scientific, Bremen, Germany). The separation of peptides on analytical column was performed at a flow rate of 300 nl/min with 60, 120, 180, and 240 min linear gradients from 5% - 35% solvent B (95% ACN/0.1% FA). Survey spectra were acquired in the Orbitrap mass analyzer with a mass range of m/z 375-1575 at a resolution of 60,000. MS/MS fragmentation for the data dependent acquisition mode was performed with charge states between 2-8 using HCD followed by data acquisition in the Orbitrap at a resolution of 30,000. For data-independent acquisition (DIA) analysis of the samples, the following method was applied: one MS1 scan at 120k resolution with an AGC target of 5e5 and max injection time of 20 ms in the range of 350 to 1,200 m/z followed by 18 to 58 DIA scans depending on the gradient length (0.5h: 18, 1h: 24, 2h: 36, 3h: 47, 4h: 58). The isolation window widths were adjusted for each gradient length to cover the same mass range as the MS1 scan including a 0.5 m/z overlap (0.5h: 47.7 m/z, 1h: 35.9 m/z, 2h: 24.1 m/z, 3h: 18.6 m/z, 4h: 15.2 m/z). The DIA scan resolution was set to 30k with an AGC target of 1e6 and max injection time of 60 ms. The HCD collision energy was set to 27%. Resulting cycle times based on the widow designs for the methods: 0.5h: 1.89s, 1h: 2.34s, 2h: 3.44s, 3h: 4.46s and 4h: 5.45s.

### Data Protocol
DDA data were analyzed with Proteome Discoverer 2.2TM (Thermo Fisher Scientific, Bremen, Germany) in combination with Mascot (www.matrixscience.com). For database searching, Uniprot Mus musculus (release 2019_04, 54425 entries) was used with the following parameters: variable modifications: oxidation of methionine, acetylation of protein N-termini; fixed modification: propionamide at cysteine; mass tolerance: 10 ppm for precursor ions, 50 mmu for fragment ions; enzyme: trypsin except proline was the next amino acid; missed cleavage sites: 2. Data were filtered with a false discovery rate (FDR) of 1% at the peptide level using Percolator and proteins were exported with a FDR of 5%. Label free quantification was performed using the Minora feature detector node included in Proteome Discoverer.  DIA data were analyzed in Spectronaut Pulsar39 (Version: 13.2.19, Biognosys, Schlieren, Switzerland) with a library generated using the Pulsar search engine of Spectronaut.The libraray was generated based on the same parameters as defined for the analysis of the DDA raw files with Proteome Discoverer 2.2. except that that the mass tolerances were defined dynamically by the Pulsar algorithm. The DIA data were analyzed using this library and applying the  default settings in Spectronaut. For retention time alignment the high precision iRT concept was applied. A minimum requirement of 3 fragment ions were set to detect a peak, whereby the precursor information was only used to enhance peak detection. Data were filtered by a FDR of 1% on precursor and protein level applying a Q-value of <0.01. Data were exported to MS Excel and the data comparison was performed in GraphPad Prism 6.07 and by the Venn diagrams tool. The numbers of lysosomal proteins were calculated from the Protein files by filtering through annotated lysosomal list. The annotated lysosomal list is the combination of our manually curated bonafide list and panther gene ontology database for mouse.

### Publication Abstract
None

### Keywords
Lc-msms, Lysosome, Dia

### Affiliations
Institute for Biochemistry and Molecular Biology, University of Bonn,Germany.
University of Bonn

### Submitter
Jasjot Singh

### Lab Head
Dr Dominic Winter
Institute for Biochemistry and Molecular Biology, University of Bonn,Germany.


