### Accession
PXD004695

### Title
Saprolegnia parasitica proteome

### Description
In this project, we made a comprehensive quantitative proteomic analysis of four defined developmental stages of S. parasitica (mycelium, primary cysts, secondary cysts and germinated cysts) to gain greater insight into the types of proteins specifically linked to each stage.  A total of 2423 unique proteins were identified using qualitative (gel-based) and quantitative (iTRAQ, isobaric tags for relative and absolute quantitation) approaches. Of these, 358 proteins, associated with various biological processes, were found to be significantly enriched between different life cycle stages of S. parasitica. The transcript abundance of several cyst and mycelium enriched proteins was also checked by quantitative real-time PCR. This is the first large scale proteomic analysis on Saprolegnia and the data from this study will enhance our current knowledge about this pathogen by identifying biological processes key to each developmental stage.

### Sample Protocol
The frozen sample of all four developmental stages were freeze dried, weighed (~10mg) and disrupted in the MM400 Mixer Mill (Retsch, Germany) for 10 min at 30 beats/sec. The resulting powder was resuspended in extraction buffer (3% SDC, 50mM TEAB, 1mM EDTA and protease inhibitor) and the total protein concentration was determined using the Bradford assay. Proteins were reduced, alkylated and hydrolyzed in the presence of trypsin (Srivastava et al., 2013).

### Data Protocol
Our in-house Automated Proteomics Pipeline, which automates the processing of proteomics tasks such as peptide identification, validation and quantitation from LC-MS/MS data and allows easy integration of many separate proteomic tools (Malm et al., 2014), was used to analyze the MS data. The raw MS data file was first analyzed using the Mascot Distiller software (version 2.4.3.2, Matrix Science, London, UK) and the resulting mgf files were converted into the mzML file format using msconvert (Kessner et al., 2008). The Saprolegnia protein database (20,088 entries) was then searched using several search engines in parallel, i.e., MS-GF+ (Kim et al., 2010) v1.0 (v8299), MyriMatch (Tabb et al., 2007) (version 2.1.120), Comet (Eng et al., 2013) (version 2013.01 rev.0) and X!Tandem (Craig and Beavis, 2004) (version 2011.12.01.1, LabKey, Insilicos, ISB, Seattle, WA). The following settings were used for the searches: trypsin specific digestion with two missed cleavages allowed; peptide tolerance of 200 ppm; fragment tolerance of 0.5 Da; methylthio on Cys and iTRAQ 4-plex for peptide N-t and Lys used as fixed modifications; oxidized Met and Tyr for iTRAQ 4-plex analysis in variable mode. The results from all search engines were validated by PeptideProphet (Keller et al., 2002). Protein quantitation was performed from the intensities of the iTRAQ reporter ions, which were extracted using the TPP tool Libra (Li et al., 2003) (TPP v4.6 OCCUPY rev 3) after the isotopic correction factors provided by the manufacturer of the iTRAQ reagent had been applied. The iTRAQ channels were normalized using the sum of all the reporter ion intensities from each iTRAQ channel and equalizing each channelâ€™s contribution by dividing the individual reporter ion intensities by the corresponding channel-specific correction factor. The pep.xml files obtained from PeptideProphet were combined using iProphet (Shteynberg et al., 2011) and the protein lists were assembled using ProteinProphet (Nesvizhskii et al., 2003). The final protein ratios were calculated using LibraProteinRatioParser (Li et al., 2003) and a concatenated target-decoy database-search strategy was used to check the false positives rate (<1 %) for all searches.

### Publication Abstract
Several water mold species from the <i>Saprolegnia</i> genus infect fish, amphibians, and crustaceans in natural ecosystems and aquaculture farms. <i>Saprolegnia parasitica</i> is one of the most severe fish pathogens. It is responsible for millions of dollars of losses to the aquaculture industry worldwide. Here, we have performed a proteomic analysis, using gel-based and solution (iTRAQ) approaches, of four defined developmental stages of <i>S. parasitica</i> grown <i>in vitro</i>, i.e., the mycelium, primary cysts, secondary cysts and germinated cysts, to gain greater insight into the types of proteins linked to the different stages. A relatively high number of kinases as well as virulence proteins, including the ricin B lectin, disintegrins, and proteases were identified in the <i>S. parasitica</i> proteome. Many proteins associated with various biological processes were significantly enriched in different life cycle stages of <i>S. parasitica</i>. Compared to the mycelium, most of the proteins in the different cyst stages showed similar enrichment patterns and were mainly related to energy metabolism, signal transduction, protein synthesis, and post-translational modifications. The proteins most enriched in the mycelium compared to the cyst stages were associated with amino acid metabolism, carbohydrate metabolism, and mitochondrial energy production. The data presented expand our knowledge of metabolic pathways specifically linked to each developmental stage of this pathogen.

### Keywords
Germinated cyst, Primary cysts, Quantitative proteomics, Oomycete, Secondary cysts, Saprolegnia parasitica, Mycelium, Mass spectrometry

### Affiliations
Biotechnology
Division of Glycoscience, Royal Institute of Technology (KTH), AlbaNova University Centre, Stockholm, SE-10691, Sweden;

### Submitter
Vaibhav Srivastava

### Lab Head
Dr Vincent Bulone / Vaibhav Srivastava
Division of Glycoscience, Royal Institute of Technology (KTH), AlbaNova University Centre, Stockholm, SE-10691, Sweden;


