### Accession
PXD012520

### Title
Evosep One Enables Robust Quantitative Deep Proteome Coverage using Tandem Mass Tags while Significantly Reducing Machine Time

### Description
The balance between comprehensively analyzing the proteome and using valuable mass spectrometry time is a genuine challenge in the field of proteomics. Multidimensional fractionation strategies have significantly increased proteome coverage, but often at the cost of increased mass analysis time, despite advances in mass spectrometer acquisition rates. Recently, the Evosep One liquid chromatography system was shown to analyze peptide samples in a high throughput manner without sacrificing in depth proteomics coverage. We demonstrate incorporation of Evosep One technology into our multiplexing workflow for quantitative analysis of tandem mass tag (TMT)-labeled non-small cell lung carcinoma (NSCLC) patient-derived xenografts (PDXs). Using a 30 samples per day Evosep workflow, >12,000 proteins were identified in 48 hours of mass spectrometry time, which is comparable to the number of proteins identified by our conventional concatenated EASY-nLC workflow in 67.5 hours. Shorter Evosep gradient lengths reduced the number of protein identifications by 10%, while decreasing mass analysis time by 50%. Thus, our Evosep workflow enables quantitative analysis of multiplexed samples in less time without conceding depth of proteome coverage.

### Sample Protocol
Samples were reduced, alkylated, digested, and TMT labelled using the TMT 10plex labelling strategy.  Per 10plex were 8 samples and 2 mixed controls.  Samples were separated for fractionating using a high pH linear gradient on a C18 reversed phase column.  Fractions were collected and either loaded onto the Evosep LC via Evotips or concatenated and and loaded into the atosampler of the Easy-nLC1200.  For all workflows, peptides were introduced by nano-electrospray into the Thermo Fisher Scentific Orbitrap Fusion Lumos mass spectrometer using an Easy-Spray source, and data were acquired using MultiNotch syncronous precursor selection.

### Data Protocol
MS Raw files were analyzed using Proteome Discoverer 2.2 and fragment lists searched against the human and mouse UniProt reference databases by Sequest HT and MS Amanda 2.0.  Complete tryptic peptides with a maximum of two missed cleavages were allowed.  Oxidation of methionine, and asparagine and glutamine deamidation were included as variable modifications while cysteine carbamidomethylation and TMT labelling of peptide N-termini and lysine residues were considered fixed modifications.  Search results were filtered through Percolator allowing a strict q-value of 0.01 and relaxed q-value of 0.05.  TMT reporter ions were quantified using Proteome Discoverer 2.2 on the MS3 level.

### Publication Abstract
The balance between comprehensively analyzing the proteome and using valuable mass spectrometry time is a genuine challenge in the field of proteomics. Multidimensional fractionation strategies have significantly increased proteome coverage, but often at the cost of increased mass analysis time, despite advances in mass spectrometer acquisition rates. Recently, the Evosep One liquid chromatography system was shown to analyze peptide samples in a high-throughput manner without sacrificing in-depth proteomics coverage. We demonstrate the incorporation of Evosep One technology into our multiplexing workflow for analysis of tandem mass tag (TMT)-labeled nonsmall cell lung carcinoma (NSCLC) patient-derived xenografts (PDXs). By the use of a 30 samples per day Evosep workflow, &gt;12&#x202f;000 proteins were identified in 48 h of mass spectrometry time, which is comparable to the number of proteins identified by our conventional concatenated EASY-nLC workflow in 60 h. Shorter Evosep gradient lengths reduced the number of protein identifications by 10% while decreasing the mass analysis time by 50%. This Evosep workflow will enable quantitative analysis of multiplexed samples in less time without conceding depth of proteome coverage.

### Keywords
Nonsmall cell lung carcinoma (nsclc) patient-derived xenografts (pdx), Evosep, Mass analysis time, Lc-ms/ms, Tmt

### Affiliations
Bioinformatics Solutions Inc
SPARC Biocentre, The Hospital for Sick Children, Program in Cell Biology, The Hospital for Sick Children, Department of Medical Biophysics, University of Toronto.

### Submitter
Jonathan Krieger

### Lab Head
Dr Michal F. Moran
SPARC Biocentre, The Hospital for Sick Children, Program in Cell Biology, The Hospital for Sick Children, Department of Medical Biophysics, University of Toronto.


