### Accession
PXD002875

### Title
Proteome-wide quantitative multiplexed profiling of protein expression: Carbon source dependency in S. cerevisiae

### Description
The global proteomic alterations in the budding yeast Saccharomyces cerevisiae due to differences in carbon sources can be comprehensively examined using mass spectrometry-based multiplexing strategies. Here we investigate changes in the S. cerevisiae proteome resulting from cultures grown in minimal media using galactose, glucose, or raffinose as the carbon source. We used a TMT 9-plex strategy to determine alterations in relative protein abundance due to a particular carbon source, in triplicate, thereby permitting subsequent statistical analyses. We quantified over 4700 proteins across all 9 samples, of which 1003 demonstrated statistically significant differences in abundance in at least one condition. The majority of altered proteins were classified as functioning in metabolic processes and as having cellular origins of plasma membrane and mitochondria. In contrast, proteins remaining relatively unchanged in abundance included those having nucleic acid-related processes, such as transcription and RNA processing. In addition, the comprehensiveness of the dataset enabled the analysis of subsets of functionally-related proteins, such as phosphatases, kinases, and transcription factors. Moreover, alterations in protein abundance levels of full sets of proteins that comprise distinct biological pathways, such as galactose metabolism and the TCA cycle, can be examined collectively. As a resource, these data can be mined further in efforts to understand better the roles of carbon source fermentation in yeast metabolic pathways and potentially utilize observed alterations for industrial applications, such as biofuel feedstock production.

### Sample Protocol
TMT9-plex analysis of S. cerevisiae grown on three carbon sources. The procedure was as follows: 1) Three starter cultures of minimal media with raffinose as the carbon source were each inoculated with a single colony. Cultures were grown overnight in raffinose media. Cultures were centrifuged, washed in deionized water and diluted to OD600 of 0.1 in galactose-, glucose-, or raffinose-containing growth media. At OD600 of 0.6, cultures were harvested, cells were lysed, and proteins were extracted via mechanical lysis and chloroform-methanol precipitation. 2) Proteins were digested with LysC and trypsin and labeled with TMT reagents. 3) The pooled samples were separated using basic pH reversed-phase (BPRP) chromatography. 4) Desalted peptides were subjected to high-pressure liquid chromatography (HPLC) and TMT-MS3-based mass spectrometry.

### Data Protocol
Data analysis. Mass spectra were processed using a SEQUEST-based in-house software pipeline (Huttlin et al., 2010). Spectra were converted to mzXML using a modified version of ReAdW.exe. Database searching included all entries from the yeast SGD (Saccharomyces Genome Database) (March 11, 2014). This database was concatenated with a database composed of all protein sequences in the reversed order. Searches were performed using a 50 ppm precursor ion tolerance for total protein level analysis. The product ion tolerance was set to 0.9 Da. These wide mass tolerance windows were chosen to maximize sensitivity in conjunction with Sequest searches and linear discriminant analysis (Beausoleil et al., 2006; Huttlin et al., 2010). TMT tags on lysine residues and peptide N termini (+229.163 Da) and carbamidomethylation of cysteine residues (+57.021 Da) were set as static modifications, while oxidation of methionine residues (+15.995 Da) was set as a variable modification. Peptide-spectrum matches (PSMs) were adjusted to a 1% false discovery rate (FDR) (Elias and Gygi, 2007, 2010). PSM filtering was performed using a linear discriminant analysis, as described previously (Huttlin et al., 2010), while considering the following parameters: XCorr, Î”Cn, missed cleavages, peptide length, charge state, and precursor mass accuracy. For TMT-based reporter ion quantitation, we extracted the signal-to-noise (S/N) ratio for each TMT channel and found the closest matching centroid to the expected mass of the TMT reporter ion. PSMs were identified, quantified, and collapsed to a 1% peptide false discovery rate (FDR) and then collapsed further to a final protein-level FDR of 1%. Moreover, protein assembly was guided by principles of parsimony to produce the smallest set of proteins necessary to account for all observed peptides.

### Publication Abstract
The global proteomic alterations in the budding yeast Saccharomyces cerevisiae due to differences in carbon sources can be comprehensively examined using mass spectrometry-based multiplexing strategies. In this study, we investigate changes in the S. cerevisiae proteome resulting from cultures grown in minimal media using galactose, glucose, or raffinose as the carbon source. We used a tandem mass tag 9-plex strategy to determine alterations in relative protein abundance due to a particular carbon source, in triplicate, thereby permitting subsequent statistical analyses. We quantified more than 4700 proteins across all nine samples; 1003 proteins demonstrated statistically significant differences in abundance in at least one condition. The majority of altered proteins were classified as functioning in metabolic processes and as having cellular origins of plasma membrane and mitochondria. In contrast, proteins remaining relatively unchanged in abundance included those having nucleic acid-related processes, such as transcription and RNA processing. In addition, the comprehensiveness of the data set enabled the analysis of subsets of functionally related proteins, such as phosphatases, kinases, and transcription factors. As a resource, these data can be mined further in efforts to understand better the roles of carbon source fermentation in yeast metabolic pathways and the alterations observed therein, potentially for industrial applications, such as biofuel feedstock production.

### Keywords
Tmt, Glucose, Galactose, Fusion, : yeast, Raffinose

### Affiliations
Harvard Medical School
Harvard Medical School, Boston, MA, USA

### Submitter
Joao Paulo

### Lab Head
Dr Joao A. Paulo
Harvard Medical School, Boston, MA, USA


