### Accession
PXD003016

### Title
Comparative Proteome Analysis in Schizosaccharomyces pombe identifies metabolic targets to improve protein production and secretion

### Description
This dataset contains additional experimental results for the determination of the effect of Fluconazole and amino acid supplementation to the NW10 strain of S. pombe.  Protein secretion in yeast is a complex process and its efficiency depends on a variety of parameters. We performed a comparative proteome analysis of a set of Schizosaccharomyces pombe strains producing the alpha-glucosidase maltase in increasing amounts to investigate the overall proteomic response of the cell to the burden of protein production along the various steps of protein production and secretion. Proteome analysis of these strains, utilizing an isobaric labeling /two dimensional LC-MALDI MS approach, revealed complex changes, from chaperones and secretory transport machinery to proteins controlling transcription and translation. We also found an unexpectedly high amount of changes in enzyme levels of the central carbon metabolism and a significant upregulation of several amino acid biosyntheses. These amino acids were partially underrepresented in the cellular protein compared to the composition of the model protein. Additional feeding of these amino acids resulted in a 1.5-fold increase in protein secretion. Membrane fluidity was identified as a second bottleneck for high-level protein secretion and addition of fluconazole to the culture caused a significant decrease in ergosterol levels, while protein secretion could be further increased by a factor of 2.1. In summary, we show that high level protein secretion causes global changes of protein expression levels in the cell and that precursor availability and membrane composition limit protein secretion in this yeast.  In this respect, comparative proteome analysis is a powerful tool to identify targets for an efficient increase of protein production and secretion in S. pombe. Additional supernatant analysis was performed using LC-ESI MS.

### Sample Protocol
An isobaric labeling strategy encompassing four-plex iTRAQ was utilized to compare the protein abundance ratios between the three different S. pombe strains.  In addition, a global internal standard (GIS) approach was applied, in which an equal amount (1/4) from each of three tryptic peptide samples was pooled to create a reference sample (Mix). A two-dimensional LC separation scheme encompassing high-pH (pH 10) reversed-phase HPLC in the first and low-pH (pH 2) ion-pairing reversed-phase HPLC in the second dimension (RP x IP-RP-HPLC) was applied. Second dimension separations was coupled offline to MALDI TOF/TOF MS. Supernatant analysis was performed by LC-ESI QExactive MS.

### Data Protocol
Protein identification and quantification were performed with the ProteinPilot v4.0 software (AB Sciex) using the Paragon algorithms. MS/MS data from 36 HPLC runs (i.e. 3 biological replicates × 4 pooled fractions from first dimension separation × 3 technical replicates) were searched separately against the PomBase database (released on 09th of December 2010, 5,038 entries) downloaded from EMBL-EBI (ftp://ftp.sanger.ac.uk/pub/yeast/pombe/Protein_data/OLD/Pompep/). The following search parameters were selected: iTRAQ 4-plex peptide label, cysteine carbamidomethylation, trypsin specificity, one missed cleavage site allowed ID focused on biological modification and amino acid substitution, and processing including quantification and “Thorough ID”, in which the precursor ion mass accuracy and fragment ion mass accuracy are MALDI 5800 built-in function of ProteinPilot software. All reported proteins were identified with 99% confidence as determined by ProteinPilot Unused ProtScores (2.0) with the corresponding local false positive discovery rate (FDR) below 5%. The identified proteins were grouped by the ProGroup algorithm (ABSciex) to minimize redundancy.  Only proteins with two or more unique peptides were considered for relative quantification.

### Publication Abstract
Protein secretion in yeast is a complex process and its efficiency depends on a variety of parameters. We performed a comparative proteome analysis of a set of Schizosaccharomyces pombe strains producing the &#x3b1;-glucosidase maltase in increasing amounts to investigate the overall proteomic response of the cell to the burden of protein production along the various steps of protein production and secretion. Proteome analysis of these strains, utilizing an isobaric labeling/two dimensional LC-MALDI MS approach, revealed complex changes, from chaperones and secretory transport machinery to proteins controlling transcription and translation. We also found an unexpectedly high amount of changes in enzyme levels of the central carbon metabolism and a significant up-regulation of several amino acid biosyntheses. These amino acids were partially underrepresented in the cellular protein compared with the composition of the model protein. Additional feeding of these amino acids resulted in a 1.5-fold increase in protein secretion. Membrane fluidity was identified as a second bottleneck for high-level protein secretion and addition of fluconazole to the culture caused a significant decrease in ergosterol levels, whereas protein secretion could be further increased by a factor of 2.1. In summary, we show that high level protein secretion causes global changes of protein expression levels in the cell and that precursor availability and membrane composition limit protein secretion in this yeast. In this respect, comparative proteome analysis is a powerful tool to identify targets for an efficient increase of protein production and secretion in S. pombe Data are available via ProteomeXchange with identifiers PXD002693 and PXD003016.

### Keywords
2d-lc maldi ms, 5800 proteome analyzer, Isobaric labeling, Itraq, High/low ph lc

### Affiliations
Systematic Proteome Research & Bioanalytics, University of Kiel
Systematic Proteome Research & Bioanalytics, Christian-Albrechts-Univeritaet Kiel, 24105 Kiel, Germany

### Submitter
Andreas Tholey

### Lab Head
Dr Tholey, Andreas
Systematic Proteome Research & Bioanalytics, Christian-Albrechts-Univeritaet Kiel, 24105 Kiel, Germany


