### Accession
PXD033322

### Title
Macrophage membrane proteomics upon depolarization

### Description
After depolarization treatment, a series of membrane protein were changed significantly. We collected the total membrane protein of the mouse peritoneal macrophages as previously (Karmakar et al., 2020), and then analyzed by Mass Spectrometry

### Sample Protocol
Plasma membrane protein extracted as described (Karmakar et al., 2020). Briefly, cells were washed with ice-cold PBS and collected after centrifugation at 3600 rpm for 5 min. The supernatant was discarded, 450 μl Buffer A (in mM: 250 sucrose, 20 HEPES, 10 KCl, 1.5 MgCl2, 1 EDTA, 1 EGTA, protease cocktail) was added to re-suspend the cell pellets, and the cells were lysed 20 times with a 29G needle. Buffer A was added to a volume of 1 ml and centrifuged at 700×g for 5 min. The supernatant was transferred to a new tube and centrifuged at 10000×g for 10 min. The supernatant was transferred to a new tube and centrifuged at 100,000×g for 1 h. The supernatant was discarded and the membrane proteins were subjected to MS analysis.

### Data Protocol
The original data from the mass spectrometer was imported into the library search software, and the corresponding analysis parameters were set according to the experimental plan. The secondary mass spectrometry data of this experiment were retrieved using Proteome Discoverer (v2.4.1.15). Search parameter settings: the database is Mus_musculus_10090_SP_20201214.fasta (17063 sequences), an anti-library is added to calculate the false discovery rate (FDR) caused by random matching, and a common contamination library is added to the database to eliminate contamination in identification results protein effect. The digestion method is set to Trypsin (Full); the number of missed cleavage sites is set to 2; the minimum length of the peptide is set to 6 amino acid residues; the maximum number of peptide modifications is set to 3. The mass error tolerance of primary precursor ions was set to 10 ppm, and the mass error tolerance of secondary fragment ions was set to 0.02 Da. Carbamidomethyl (C), TMTpro (peptide N-Terminus), TMTpro (K) were set as fixed modifications, and Acetyl (protein N-Terminus), Oxidation (M) were set as variable modifications. The quantitative method is set to TMTpro-16plex, and the FDR identified by protein, peptide, and PSM is set to 1%. Based on the Raw file detected by mass spectrometry, we firstly construct a sample-specific protein database according to the source of the sample, and then use the analysis software to search the database. Then perform quality control analysis of peptide fragments and protein levels based on the database search results. Last, annotating the identified proteins with common functional databases, including GO, KEGG, InterPro, COG, and STRING databases.

### Publication Abstract
None

### Keywords
Depolarization, Membrane protein

### Affiliations
Zhejiang Universigy School of Medicine
Zhejiang University

### Submitter
Weiwei Yu

### Lab Head
Dr Di Wang
Zhejiang Universigy School of Medicine


