### Accession
PXD022881

### Title
Combined Protein Signature of ALDH1A1, LGALS3, LGALS3BP Predicts Cancer Recurrence Post-Liver Transplantation Beyond Milan Hepatocellular Carcinoma

### Description
Background: Liver transplantation (LT) for Hepatocellular carcinoma (HCC) can be offered to patients beyond Milan criteria. However, there are currently no molecular markers that can be used on HCC explant histology to predict recurrence, which arises in up to 20% of LT recipients. The goal of our study was to identify proteins on HCC explant predictive of recurrence post-transplant, thereby guiding surveillance strategies and identifying patients beyond Milan criteria who would fare well following LT.  Methods: LT recipients who had been transplanted at the University Health Network for HCC beyond Milan criteria in the context of Hepatitis B cirrhosis were identified. Snap-frozen samples from the dominant tumors of recurrent (N=7) and non-recurrent (N=4) patients were analyzed using LC-MS/MS on a Q-Exactive Plus mass spectrometer to delineate a distinctive proteomic signature. These tumors were also profiled by a Human Gene 2.0 ST microarray platform to identify a transcriptomic signature predictive of recurrence and analyzed with R packages. STRING database was used to characterize the implicated pathways. Kaplan-Meier estimator was used to generate a combined proteomic/transcriptomic signature predictive of HCC recurrence in patients with HCC beyond Milan criteria at time of LT. Significantly predictive proteins were verified and internally validated by immunoblotting or immunohistochemistry.  Results: A total of 79 proteins and 636 genes were significantly differentially expressed in recurrent HCC, compared to non-recurrent (p<0.05). Among these, LGALS3, LGALS3BP, HAL, THBS1, and BLMH, were significantly increased in recurrent HCC at the protein and gene expression level. In turn, ALDH1A1 protein and gene expression were significantly decreased in recurrent HCC.  Univariate survival analysis depicted ALDH1A1 (HR=0.084, 95%CI 0.01-0.68, p=0.02), LGALS3BP (HR=7.14, 95%CI 1.20-42.96, p=0.03), and LGALS3 (HR=2.89, 95%CI 1.01-8.3, p=0.049) as the key dysregulated proteins and genes  in the patients with HCC recurrence versus those with non-recurrence by both proteomic and transcriptomic analysis. Decreased ALDH1A1 and significantly increased LGALS3 protein expression in recurrent HCC was verified by immunoblotting. Increased LGALS3BP protein expression in recurrent HCC was orthogonally verified and validated by immunohistochemistry in 30 independent HCC samples.  Conclusion: Protein and gene expression of the cancer stem cell marker ALDH1A1 was protective against cancer recurrence in patients transplanted for HCC beyond Milan criteria. Conversely, increased expression of LGALS3 and LGALS3BP on explant was significantly predictive of post-transplant recurrence. These findings were internally validated, suggesting potential utility in identifying patients with HCC beyond Milan who would clearly benefit from transplant with limited recurrence risk and guiding post-transplant surveillance.

### Sample Protocol
Frozen HCC tissue samples obtained from the UHN biobank were processed and analysed in blinded fashion. Rapigest detergent (0.2%) was added to each pieces of tissue ~100mg in weight. Tissue was then homogenized in a tissue dissociator. Ammonium bicarbonate was added and the samples were subjected to sonication (10seconds, 3 times) on ice. Samples were then centrifuged at 15,000g at 4◦C for 20 minutes. Supernatant was collected and vortexed. Total protein concentration was performed using the Coomassie assay. Each sample was then normalized to 250µg of total protein. Samples were denatured at 80◦C for 15 minutes, followed by reduction in 10mM dithiothreitol (final concentration) for 15 minutes at 65◦C and alkylation in 20mM (final concentration) iodoacetamide for 40 minutes in the dark at room temperature. Finally, trypsin (Promega) was added at 1:50 w/w and incubated overnight at 37◦C. Digested samples were acidified with trifluoroacetic acid (1% v/v) and vortexed for 1 minute, then left at room temperature for 5 minutes. Samples were centrifuged at 15,000g for 10 minutes to remove Rapigest, then transferred into the new tubes and frozen at -20◦C until further analysis.  Following digestion, strong cation exchange (SCX) chromatography and fractionation were performed on an HPLC system (Agilent 1100) using a 60-min two-step gradient. The resulting fractions corresponding to chromatographic peaks of eluting peptides were pooled into 10 fractions). Peptides from each fraction were extracted and desalted with 10μL OMIX C18 MB tips (Agilent), eluted in 3μL of 65% v/v acetonitrile, and diluted to 41μL with 0.1% v/v formic acid in MS-grade pure water. Samples were randomized and subjected to mass spectrometry (MS) on a Thermo Scientific EASY-nLC1000 system, coupled to a Q-Exactive Plus hybrid quadrupole-orbitrap mass spectrometer using a nano-electrospray ionization source (Thermo Scientific). For each sample, 18μL of eluted peptides were injected onto a 3.3cm C18 pre-analytical column (IntegraFrit capillary, New Objective; inner diameter: 75μm; bead size: 5μm; Agilent Technologies) followed by a C18 resolving analytical column (PicoTip emitter, inner diameter: 15cm x 75μm; tip: 8μm tip; bead size: 3μm; Agilent Technologies). Samples were run on a 60-minute gradient of increasing concentrations of Buffer B (100% acetonitrile) in 0.1% formic acid/99.9% MS grade water (Thermo Scientific). The method started at 1% Buffer B, and the concentration was increased to 5% at 2 minutes, with increases to 35% (49 minutes), 65% (52 minutes) and 100% (53 minutes). The spectra were obtained under data-dependent acquisition mode, consisting of full MS1 scans (m/z range: 400-1500; resolution: 70,000) followed by MS2 scans of the top 15 parent ions (resolution: 17,500).

### Data Protocol
For protein identification, the RAW files of each MS run were generated by XCalibur software v3.0.63 (Thermo Scientific). Raw data were analyzed by MaxQuant software (version 1.5.3.28) and searched against the human Uniprot FASTA database (HUMAN5640_sProt–072016, update of July 20, 2016). Proteins and peptides were identified with a false discovery rate of 1%. A minimum length of 6 amino acids was selected. The false positive rate was determined using reversed mode. Trypsin/P was selected as digestion enzyme, and a maximum of 2 missed cleavages was enabled. While cysteine carbamidomethylation was selected as a fixed modification, methionine oxidation and N-terminal acetylation were set as variable modifications. The initial peptide tolerance against a ‘human-first-search’ database was set to 20ppm. The main search peptide mass tolerance was 40ppm, and the fragment mass MS/MS tolerance was 0.5Da. Matching between runs was selected. Normalized label-free quantification (LFQ) of proteins was derived from extracted ion current information from razor and unique peptides with a minimum ratio count of 1.   Proteomics data were analyzed using Perseus software (version 1.5.2.6). Reverse hits and contaminants were manually checked and removed. Distribution of log2-transformed LFQ intensity values of all proteins quantified was examined for each sample. Following filtering of proteins that were identified in <50% of the samples, we subjected the zero-value intensities to imputation (assuming that low abundance values were missing), keeping a normal distribution, with a downshift of 1.8 standard deviations, and a width of 0.5 for each sample. After imputation, we determined the differentially expressed proteins between recurrent and non-recurrent HCC samples by comparing their mean log2-transformed LFQ intensities using the two-tailed independent t-test (P<0.05), followed by Benjamini-Hochberg adjustment. Principal component analysis was performed in Perseus. Two components that explained the most variability in samples were selected.

### Publication Abstract
None

### Keywords
Galectin, Recurrence, Liver, Proteomics, Hepatocellular carcinoma

### Affiliations
Toronto General Hospital Research Institute, UniversityHealth Network, Toronto, Canada Department of Medicine, Division of Nephrology, University Health Network, Toronto, Canada
University Health Network

### Submitter
Sergi Clotet Freixas

### Lab Head
Dr Ana Konvalinka
Toronto General Hospital Research Institute, UniversityHealth Network, Toronto, Canada Department of Medicine, Division of Nephrology, University Health Network, Toronto, Canada


