### Accession
PXD018276

### Title
Coactosin phosphorylation controls Entamoeba histolytica 1 cell membrane protusions and cell motility

### Description
Invasion of the colon wall by Entamoeba histolytica during amoebic dysentery entails migration of trophozoites through tissue layers that are rich in extracellular matrix (ECM). Transcriptional silencing of the E. histolytica matrix surface metalloprotease EhMSP-1 produces hyper-adherent, less motile trophozoites that are deficient in forming invadosomes. Reversible protein phosphorylation is often implicated in regulation of cell motility and invadosome formation. To identify such intermediaries of the EhMSP-1 silenced phenotype, here we compared the phosphoproteome of EhMSP-1 silenced and vector control trophozoites using quantitative mass spectrometry based proteomics. Six proteins were found to be differentially phosphorylated in EhMSP-1 silenced and control cells, including EhCoactosin, a member of the ADF/Cofilin family of actin binding proteins, which was hyperphosphorylated at serine 147 (S147). Regulated over-expression of wild type, phosphomimetic (S147D), and non phosphorylatable (S147A) EhCoactosin variants was used to test if S147 phosphorylation functions in control of E. histolytica actin dynamics. Each of the over-expressed proteins co37 localized with F-actin during E. histolytica phagocytosis. Nonetheless, trophozoites overexpressing EhCoactosin S147D formed more and poorly coordinated small membrane protrusions compared to control or S147A expressing cells, while trophozoites overexpressing EhCoactosin S147A were significantly more motile within a model of mammalian ECM. Therefore, although EhCoactosinâ€™s actin-binding ability appeared unaffected by S147 phosphorylation, EhCoactosin phosphorylation helps to regulate amoebic motility. These data help to understand the mechanisms underlying altered adherence and motility in EhMSP-1 silenced trophozoites, and lay the groundwork for identifying kinases and phosphatases critical for control of amoebic invasiveness.

### Sample Protocol
Total protein was extracted from the EhMSP-silenced and wild type Entamoeba histolytica and in-solution digested by trypsin.  Tryptic peptides were subjected to stable isotope based dimethyl labeling, and pooled for phosphopeptide enrichment by IMAC and TiO2, followed by mass spectrometry on the LTQ-Orbitrap Discovery mass spectrometer for differential phosphopeptide profiling using data-dependent acquisition.   An aliquot of the labeled pooled peptides were fractionated by strong cation exchange (SCX) cartridge (step elution: 0, 20, 25, 30, 35, 40, 45, 50, 55, 60, 65, 70, 75, 80, 85, 90, 95, 100, 200, and 350 mM KCl) and each fraction was zip-tipped and analyzed on the LTQ-Orbitrap Discovery mass spectrometer for differential expression profiling using data-dependent acquisition.

### Data Protocol
Instrument files (.raw) were searched using the SEQUEST search engine in the Proteome Discoverer 2.2 (Thermo Fisher Scientific) against curated E. histolytica databases (AmoebaDB-34_EhistolyticaHM1IMSS_AnnotatedProteins.fasta, AmoebaDB-34_HM1IMSS-A_AnnotedProteins.fasta, AmoebaDB-34_HM1IMSS-B_AnnotedProteins.fasta; downloaded November 2017).  Variable modifications included oxidation (M), and phosphorylation (S/T/Y). Static modifications included carbamidomethylation (C), and isomeric dimethyl modifications [WT: C(2)H(4), EhMSP-: C(2)D(6)H(-2), on K or N-terminus].  Percolator was included in the workflow to limit the false positive rates to less than 1% in the data set. The relative abundances of peptides EhMSP -/WT were quantified by the Precursor Ion Quantifier node and statistically analyzed using the Consensus workflow in the Proteome Discoverer.

### Publication Abstract
None

### Keywords
Lc-msms, Phosphopeptides, Entamoeba histolytica

### Affiliations
Infectious Diseases Division Departments of Medicine, and Microbiology and Molecular Genetics University of Vermont College of Medicine
Vermont Genetics Network Proteomics Facility
Department of Biology
University of Vermont


### Submitter
Ying Wai Lam

### Lab Head
Dr Christopher Huston
Infectious Diseases Division Departments of Medicine, and Microbiology and Molecular Genetics University of Vermont College of Medicine


