### Accession
PXD015221

### Title
MOA:Multiomics analysis of potato response to Potato virus Y (PVY) infection

### Description
Time series response of potato cv. DÃ©sirÃ©e, which is tolerant to PVY infection, was analysed in both inoculated as well as upper non-inoculated leaves. Additionally, transgenic plants deficient in accumulation of salicylic acid (NahG- DÃ©sirÃ©e) were studied in the same setting.

### Sample Protocol
Potato plants of cv. DÃ©sirÃ©e and its SA-deficient counterpart (NahG-DÃ©sirÃ©e) [14,24,25] were inoculated with PVYNTN as reported in Stare et al. [11]. Healthy potato plants were grown in stem node tissue culture. Two weeks after node segmentation, they were transferred to soil in a growth chamber and kept at 21  2 C in the light and 18  2 C in the dark, at a relative humidity of 75  2% with 70â€“90 mmol/m2/s2 radiation (L36W/77 lamp, Osram, Germany) and a 16 h photoperiod. After four weeks of growth in soil, the potato plants were inoculated with PVYNTN (isolate NIB-, GENBANK accession number AJ585342) or mock-inoculated as described in [26]. Three bottom leaves were dusted with carborundum and then inoculated by applying a buffered suspension of sap of either PVYNTN-infected (for PVYNTN inoculation) or healthy (for mock-inoculation) potato plants. Samples of inoculated leaves from both groups of plants were collected at 4 dpi, flash frozen in liquid nitrogen, and stored at ô€€€80 C for transcriptome or proteome analysis. Three inoculated leaves of both genotypes at 4 days following PVY/mock treatment were used for one-dimensional nano-flow LC analysis (UltiMate 3000, Thermo Scientific,Waltham, MA, USA) coupled to an Orbitrap LTQ XL mass spectrometer.

### Data Protocol
The proteins were identified using the SEQUEST algorithm and Proteome Discoverer (v 1.3, Thermo Scientific). In-silico peptide lists were generated with the following settings: trypsin as the digestion enzyme and a maximum of three missed cleavages. Mass tolerance was set to 5 ppm for precursor ions and 0.8 Da for fragment ions. Additionally, a decoy database containing reversed sequences was used to estimate the false discovery rate (FDR). Only high confidence (FDR  0.01%) peptide identifications with a minimum XCorr of 2.0, and proteins with at least two distinct peptides, were considered. The datamatrix of the ProteomeDiscoverer, which contained spectral count information, was used for quantitative analysis. For the analysis purpose, missing values (proteins not identified in the sample) were replaced with 0.5 of the minimum protein expression value [Larrainzar, E.;Wienkoop, S.;Weckwerth,W.; Ladrera, R.; Arrese-Igor, C.; Gonzalez, E.M. Medicago truncatula Root Nodule Proteome Analysis Reveals Differential Plant and Bacteroid Responses to Drought Stress. Plant Physiol. 2007, 144, 1495â€“1507]. Additionally, proteins were identified and quantified using the label-free quantification (LFQ) data matrix of MaxQuant (v1.5.3.8) software [Cox, J.; Mann, M. MaxQuant enables high peptide identification rates, individualized p.p.b.-range mass accuracies and proteome-wide protein quantification. Nat. Biotechnol. 2008, 26, 1367â€“1372]. To search the MS data against a FASTA file, we created from downloads the complete set of available potato sequences, as described [Ramsak, Å½.; Baebler, Å .; Rotter, A.; Korbar, M.; Mozetic, I.; Usadel, B.; Gruden, K. GoMapMan: Integration, consolidation and visualization of plant gene annotations within the MapMan ontology. Nucleic Acids Res. 2014, 42, D1167â€“D1175]. From the results log2FC and p-value was calculated and only the differences p < 0.05 were considered as significant. From the results log2FC and p-value was calculated and only the differences p < 0.05 were considered as significant.

### Publication Abstract
None

### Keywords
Solanum tuberosum l., Lc-ms/ms, Potato virus y (pvy)

### Affiliations
Molecular Systems Biology
department of ecogenomics and systemsbiology University of Vienna

### Submitter
Stefane Wienkoop

### Lab Head
Dr Stefanie Wienkoop
department of ecogenomics and systemsbiology University of Vienna


