### Accession
PXD047847

### Title
Specialized replication mechanisms maintain genome stability at human centromeres

### Description
Reveal a specific set of proteins important to maintain centromere integrity, through quantitative PICh (Proteomics of Isolated Chromatin). Centromeric chromatin was pulled down through RNA probes annealing specifically to a 300bp-long conserved centromeric sequence.

### Sample Protocol
PICh was performed on the indicated cell lines as previously described in triplicate. Briefly, ca. 10^9 cells/conditions were fixed in 3.7% formaldehyde for 45 minutes at RT; chromatin was extracted and sonicated with a microtip, pre-cleared with anti-streptavidin agarose beads (Invitrogen) and hybridized with an equimolar mix of six ~ 50 bp each 2'F-RNA biotinylated oligos (synthetized at W.M. Keck Biotechnology Resource Lab at Yale University). These probes, covering a 321 bp region, match the consensus sequence of 21 Higher Order Repeats families, distributed across all human centromeres bearing CENP-A binding. Sequences were published in Giunta, Herve et al (2021)11. Chromatin was recovered by MyOne Streptavidin dynabeads (Invitrogen), subjected to stringent washes and eluted from beads by competition with D-Biotin (Invitrogen). Proteins from eluates were precipitated with Trichloroacetic Acid (TCA, Sigma-Aldrich) and decrosslinked at 99Ã‚Â°C for 5 mins.

### Data Protocol
Disulphide bridges in cysteine containing proteins were reduced with dithiothreitol (56Ã‚Â°C, 30 min, 10 mM in 50 mM HEPES, pH 8.5), which were then alkylated with 2-chloroacetamide (room temperature, in the dark, 30 min, 20 mM in 50 mM HEPES, pH 8.5). Samples were purified by SP3 protocol112,113 and trypsinized (sequencing grade, Promega) in an enzyme:protein ratio 1:50 overnight at 37Ã‚Â°C. Peptides were recovered in two sequential elution in HEPES buffer from supernatant upon magnetic beads separation.  Peptides were labelled with TMT6plex114 Isobaric Label Reagent (ThermoFisher) according to the manufacturerÃ¢â‚¬â„¢s instructions. Samples were combined for the TMT4plex and for further sample clean up an OASISÃ‚Â® HLB Ã‚ÂµElution Plate (Waters) was used. Offline high pH reverse phase fractionation was carried out on an Agilent 1200 Infinity high-performance liquid chromatography system, equipped with a Gemini C18 column (3 ÃŽÂ¼m, 110 Ãƒâ€¦, 100 x 1.0 mm, Phenomenex). Liquid Chromatography was performed on an UltiMate 3000 RSLC nano LC system (Dionex) fitted with a trapping cartridge (Ã‚Âµ-Precolumn C18 PepMap 100, 5Ã‚Âµm, 300 Ã‚Âµm i.d. x 5 mm, 100 Ãƒâ€¦) and an analytical column (nanoEaseÃ¢â€žÂ¢ M/Z HSS T3 column 75 Ã‚Âµm x 250 mm C18, 1.8 Ã‚Âµm, 100 Ãƒâ€¦, Waters). Trapping was carried out with a constant flow of 0.05% trifluoroacetic acid at 30 Ã‚ÂµL/min onto the trapping column for 6 minutes. Peptides were eluted from the analytical column through a constant flow of solvent A (0.1% formic acid in water) at 0.3 Ã‚ÂµL/min with increasing percentage of solvent B (0.1% formic acid in acetonitrile). The outlet of the analytical column was coupled directly to a Fusion Lumos (Thermo) mass spectrometer using the Nanospray FlexÃ¢â€žÂ¢ ion source in positive ion mode.  The peptides were introduced into the Fusion Lumos via a Pico-Tip Emitter 360 Ã‚Âµm OD x 20 Ã‚Âµm ID; 10 Ã‚Âµm tip (New Objective) and an applied spray voltage of 2.4 kV. The capillary temperature was set at 275Ã‚Â°C. Full mass scan (MS1) was acquired with mass range 375-1500 m/z in profile mode in the orbitrap with resolution of 60000. The filling time was set at maximum of 50 ms. Data dependent acquisition (DDA) was performed with the resolution of the Orbitrap set to 15000, with a fill time of 54 ms and a limitation of 1x105 ions. A normalized collision energy of 36 was applied. MS2 data was acquired in profile mode.

### Publication Abstract
The high incidence of whole-arm chromosome aneuploidy and translocations in tumors suggests instability of centromeres, unique loci built on repetitive sequences and essential for chromosome separation. The causes behind this fragility and the mechanisms preserving centromere integrity remain elusive. We show that replication stress, hallmark of pre-cancerous lesions, promotes centromeric breakage in mitosis, due to spindle forces and endonuclease activities. Mechanistically, we unveil unique dynamics of the centromeric replisome distinct from the rest of the genome. Locus-specific proteomics identifies specialized DNA replication and repair proteins at centromeres, highlighting them as difficult-to-replicate regions. The translesion synthesis pathway, along with other factors, acts to sustain centromere replication and integrity. Prolonged stress causes centromeric alterations like ruptures and translocations, as observed in ovarian cancer models experiencing replication stress. This study provides unprecedented insights into centromere replication and integrity, proposing mechanistic insights into the origins of centromere alterations leading to abnormal cancerous karyotypes.

### Keywords
Centromere, Replication stress, Proteomics, Genome instability, Dna replication, Dna damage

### Affiliations
EMBL
Institut Curie, CNRS, UMR 144, 26 rue dÃ¢â‚¬â„¢Ulm, 75005 Paris, France

### Submitter
Frank Stein

### Lab Head
Dr Daniele Fachinetti
Institut Curie, CNRS, UMR 144, 26 rue dÃ¢â‚¬â„¢Ulm, 75005 Paris, France


