### Accession
PXD033196

### Title
Proteins bound to BTBD7_hsa_circ_0000563 in coronary artery disease

### Description
We aimed to clarify the possible functional role of hsa_circ_0000563 in coronary artery disease. Therefore, the ChIRP-MS was conducted to explore the interaction between BTBD7_hsa_circ_0000563 and proteins on a genomic scale in human peripheral blood mononuclear cell (PBMC). This project is the raw files of the proteins bound to hsa_circ_0000563 found by ChIRP-MS in PBMC.

### Sample Protocol
18 CAD patients and 20 controls were recruited from the First Affiliated Hospital of Nanjing Medical University. Nine ml of overnight fasting blood samples or ten ml of artery blood samples were drawn from subjects used for ChIRP by venipuncture upon admission or artery puncture before coronary angiography respectively. With density gradient centrifugation, PBMCs were isolated from the middle white monolayer using Lymphocyte Separation Medium (TBD, Tianjin, China). And then, after density gradient centrifugation again, the PBMC clumps were preserved at -80 掳C until use.  The ChIRP assay was conducted to explore the interaction between BTBD7_hsa_circ_0000563 and proteins on a genomic scale. First, four biotinylated tiling DNA oligonucleotide probes targeting the splice-junction sequence of BTBD7_hsa_circ_0000563 were designed and synthesized by Ribobio Co. Ltd (Ribobio, China) (Probe_1 CATTTCTTGTGCAAGCATGT-/3bio/; Probe_2 TCATCCATTTCTTGTGCAAG-/3bio/; Probe_3 CCATTTCTTGTGCAAGCATG-/3bio/; Probe_4 ATCCATTTCTTGTGCAAGCA-/3bio/). A split-probe strategy was conducted in order to solve the problem that the precipitation of non-specific DNA fragments by the oligonucleotide probes may be potential source of noise in CHIRP. In order to ensure the specificity and accuracy of BTBD7_hsa_circ_0000563 probes, LacZ probes were used as the negative control and GAPDH groups were set. And qPCR was performed to measure the recovery of target RNA in each group above. After crosslinking with 1% formaldehyde at room temperature for 10鈥塵in, PBMCs were lysed. Subsequently, chromatin from lysed PBMCs was collected and than hybridized with different tiling probes pools of BTBD7_hsa_circ_0000563 at 37鈥壜癈 for 4鈥塰 with shaking. Afterwards, complexes were incubated with beads conjugated with Streptavidin at 37鈥壜癈 for 30 min. Finally, the chromatin bound to beads was eluted for LC-MS/MS assays to determine the profile of proteins bound to BTBD7_hsa_circ_0000563. The lyophilized peptide fractions were resuspended in ddH2O with 0.1% formic acid. Two ul aliquots of the dissolved sample above were loaded into a nanoViper C18 (Acclaim PepMap 100, 75渭m脳2 cm) trap column, after which the trapping and desalting procedure were conducted using 20 渭L 100% solvent A (0.1% formic acid). The Easy nLC 1200 system (ThermoFisher) was used to carry out the online Chromatography separation. Afterwards, an elution gradient of 5-38% solvent B (80% acetonitrile, 0.1% formic acid) in 60 min was used on an analytical column (Acclaim PepMap RSLC, 75渭m脳25 cm C18-2 渭m 100 脜). The tandem MS data were acquired on a ThermoFisher Q Exactive mass spectrometer (ThermoFisher, USA) fitted with a Nano Flex ion source, of which the ion spray voltage was 1.9 kV and the interface heater temperature was 275鈩�. The data dependent acquisition (DDA) mass spectrum techniques were used for MS scan and tandem MS data acquiring. For the MS1 scan, the scan ranged from 350 to 2,000 m/z at a resolution of 70,000 and a maximum injection time of 100 ms. For the MS2 scan, only spectra with a charge state of 2-5 were selected for fragmentation by higher-energy collision dissociation with a normalized collision energy of 28. The MS2 spectra were acquired in the ion trap in rapid mode with an AGC target of 8,000 and a maximum injection time of 50 ms. Dynamic exclusion was set for 25 s.

### Data Protocol
The MS/MS data were analyzed for identi铿乧ation and quanti铿乧ation of proteins bound to BTBD7_hsa_circ_0000563 using PEAKS Studio 10.6. The false discovery rate (FDR) was set < 0.7% after searching against target database with a maximum of two missed cleavages. The settings of Oxidation (M), Acetylation (Protein N-term), Deamidation (NQ), Pyro-glu from E, Pyro-glu from Q for variable modifications as well as fixed Carbamidomethylation of cysteine were selected. Precursor and fragment mass tolerance were set to 10 ppm and 0.05 Da respectively.

### Publication Abstract
None

### Keywords
Proteins, Chirp-ms, Btbd7_hsa_circ_0000563, Coronary artery disease

### Affiliations
The First Affiliated Hospital of Nanjing Medical University

### Submitter
Enzhi Jia

### Lab Head
Dr Enzhi Jia
The First Affiliated Hospital of Nanjing Medical University


