### Accession
PXD029548

### Title
High-Resolution Complexome-Profiling of Mitochondria Reveals Quality Control Pathways of Protein Import

### Description
Mitochondria contain ~1,000 different proteins that may assemble into larger entities such as respiratory (super)complexes and preprotein translocases. The molecular/structural organization of major parts of the mitochondrial proteome, however, has not yet been resolved. We report a quantitative mapping of mitochondrial protein assemblies by high-resolution complexome-profiling of >90% of the yeast mitochondrial proteome, termed MitCOM. Analysis of MitCOM unraveled an unexpected complexity of protein assemblies with each protein found in an average of at least six assemblies. Organization of proteins was distinct between the major classes of mitochondrial function, but independent of their biochemical properties. We detected interactions of the mitochondrial receptor for cytosolic ribosomes, of prohibitin scaffolds and respiratory complexes. Identification of quality control factors operating at the mitochondrial protein entry gate uncovered pathways for preprotein ubiquitylation, deubiquitylation and degradation. MitCOM, made accessible through an interactive viewer on the CEDAR database, may serve as a comprehensive resource for analyzing functional organization and interaction of mitochondrial protein machineries and pathways.

### Sample Protocol
Construction of yeast strains and growth conditions: The yeast strains used in this study are derived from Saccharomyces cerevisiae BY4741 and YPH499. Yeast strain were cultured following standard protocols at temperatures between 24°C to 37°C in complete YP-media or selective minimal media. The cell cultures were grown until the early logarithmic growth phase, based on the optical density at a wavelength of 600 nm. Please refer to the manuscript for further details.  Isolation of mitochondria: Purification of mitochondria was performed by differential centrifugation. Yeast cells were harvested at an early logarithmic growth phase. Cells were washed with distilled H2O and resuspended in DTT buffer at a concentration of 2 ml per g wet weight of the cell pellet, followed by incubation for 30-45 min at growth temperature under constant shaking. Cells were then washed in zymolyase buffer and resuspended in zymolase buffer at a concentration of 7 ml per g of cells. Subsequently, cells were incubated with 4 mg zymolyase per g of cells under constant shaking for 30-45 min at growth temperature to digest the cell wall. Afterwards, cells were pelleted and washed once with zymolyase buffer. The obtained spheroblasts were resuspended in ice-cold homogenization buffer using 6.5 ml of buffer per g of cells. Cells were homogenized using a glass potter with 15 strokes up and down. Subsequently, cell debris and large organelles like the nucleus were removed. The supernatant was subjected to a second centrifugation step to isolate mitochondria. The mitochondrial pellet was resuspended in SEM buffer and washed again in SEM buffer. The isolated mitochondria were resuspended in SEM buffer. The protein concentration was determined using the Bradford assay and mitochondria were aliquoted at a protein concentration of 10 mg/ml. Mitochondria were frozen in liquid nitrogen and stored at -80°C. Cryo-slicing blue native electrophoresis: For high resolution complexome profiling, a blue native gradient gel was used. Mitochondria corresponding to 1 mg protein amount were pelleted and solubilized in lysis buffer containing 1% [w/v] digitonin for 30 min on ice. Subsequently, the sample was loaded on a sucrose gradient, consisting of 50% [w/v] sucrose and 20 % [w/v] sucrose. After centrifugation, the upper phase was removed and the remaining supernatant was mixed with loading dye. The sample was applied to a loading zone of 5 cm width. Gel running was performed at 15 mA in the presence of BN cathode buffer and anode buffer. After 1h, the BN cathode buffer was replaced by a cathode buffer lacking Coomassie G and the electrophoresis was continued for 2.5 h at 15 mA. A 2.5 cm lane was then excised, fixed in 30% ethanol / 15% acetic acid, embedded in tissue embedding media and subjected to cryo-slicing as detailed in (Müller et al., 2019). Using a step size of 0.3 mm along the gel lane, 245 slices were obtained, extensively washed and separately digested with trypsin as described. Mass spectrometry: Dried peptides were dissolved in 0.5% (v/v) trifluoroacetic acid and loaded onto a C18 PepMap100 precolumn with 0.05% (v/v) trifluoroacetic acid using a split-free UltiMate 3000 RSLCnano HPLC with autosampler. Peptides were then eluted with an aqueous-organic gradient. Gradient-eluted peptides were resolved on a SilicaTip(TM) column emitter manually packed 23 cm with ReproSil-Pur 120 ODS-3 and electrosprayed in positive ion mode into a QExactive HF-X mass spectrometer.

### Data Protocol
Peak lists were extracted from fragment ion spectra using ProteoWizard’s “msconvert”. Raw files were processed with MaxQuant. Precursor m/z values were calibrated using the respective MaxQuant values. The m/z-corrected peak lists were searched with 2.5 ppm mass tolerance against the UniprotKB/SwissProt yeast and a general contaminations database (cRAP) using Mascot. Acetyl (Protein N-term), Carbamidomethyl (C), Formyl (N-term, S, T), Gln->pyro-Glu (N-term Q), Glu->pyro-Glu (N-term E) and Oxidation (M) were chosen as variable modifications, fragment mass tolerance was set to ± 20 mmu. One missed tryptic cleavage was allowed. Results were exported with PSM (Peptide-Spectrum-Match) FDR set to 3% and a minimum ion score of 0.5. Related identified proteins (subset or species homologs) were grouped using the name of the predominant member. Proteins either representing exogenous contaminations (e.g., keratins, trypsin, IgG chains) or identified by only one specific peptide in less than three samples were not further considered in this work. Label-free quantification of proteins was carried out using an improved method with some optimizations for BN-MS data. Briefly, peptide signal intensities (peak volumes, PVs) from FT full scans were extracted and offline mass calibrated using MaxQuant. After time alignment, peptide intensities were assigned to peptide identifications based on matching m/z and elution times as obtained directly or indirectly from MS/MS-based identification. Run-to-run intensity variations were corrected for by normalizing each run to the local median of relative intensity differences. In a second step, missed, inconsistent or falsely assigned peptide intensities were effectively excluded from the determination of protein abundance profiles. Protein-specific peptide intensity value matrices (peptides vs. runs) were checked for their internal consistency (between runs and with regard to the relative peptide ionization efficiencies. Expected intensity values were calculated to derive a measure of (in)accuracy for peptide intensity values. A time- and run-dependent detectability threshold was estimated for each of the matrix cells (peptide/run pairs). Intensity values from qualified runs (columns in each protein matrix) where then aggregated to protein-specific intensity value vectors, termed protein reference ridges. The latter basically reflected the maximum protein coverage of MS/MS-identified and quantified peptides with their relative ionisation efficiencies, and was used to determine molecular abundances. Protein quantification was finally achieved by weighted fitting of its measured peptide intensities / threshold values in 5 consecutive slices (similar to a „sliding average) to its reference ridge. Each of the obtained protein profile was decomposed into individual components (“peaks”) using a custom program (Python/lmfit). After initial peak detection, an additive Gaussian composite model was iteratively expanded and fitted to the profile, adding one potential peak to the model at each iteration, located at the point of largest difference between the evaluated model and the signal. Sensible limits and stop conditions were applied to avoid overfitting.

### Publication Abstract
Mitochondria have crucial roles in cellular energetics, metabolism, signalling and quality control<sup>1-4</sup>. They contain around 1,000 different proteins that often assemble into complexes and supercomplexes such as respiratory complexes and preprotein translocases<sup>1,3-7</sup>. The composition of the mitochondrial proteome has been characterized<sup>1,3,5,6</sup>; however, the organization of mitochondrial proteins into stable and dynamic assemblies is poorly understood for major parts of the proteome<sup>1,4,7</sup>. Here we report quantitative mapping of mitochondrial protein assemblies using high-resolution complexome profiling of more than 90% of the yeast mitochondrial proteome, termed MitCOM. An analysis of the MitCOM dataset resolves &gt;5,200 protein peaks with an average of six peaks per protein and demonstrates a notable complexity of mitochondrial protein assemblies with distinct appearance for respiration, metabolism, biogenesis, dynamics, regulation and redox processes. We detect interactors of the mitochondrial receptor for cytosolic ribosomes, of prohibitin scaffolds and of respiratory complexes. The identification of quality-control factors operating at the mitochondrial protein entry gate reveals pathways for preprotein ubiquitylation, deubiquitylation and degradation. Interactions between the peptidyl-tRNA hydrolase Pth2 and the entry gate led to the elucidation of a constitutive pathway for the removal of preproteins. The MitCOM dataset-which is accessible through an interactive profile viewer-is a comprehensive resource for the identification, organization and interaction of mitochondrial machineries and pathways.

### Keywords
Lc-ms, Mitochondria, Bn-ms, Yeast, Complexome, Cryo-slicing, Bn-page

### Affiliations
Institute of Physiology II Universität Freiburg Germany
University of Freiburg, Germany

### Submitter
Alexander Haupt

### Lab Head
Dr Prof. Dr. Bernd Fakler
Institute of Physiology II Universität Freiburg Germany


