### Accession
PXD000054

### Title
Proteogenomic analysis of Helicobacter pylori strain 26695

### Description
Dear Sir or Madam,  we report an in-depth proteogenomics study of Helicobacter pylori strain 26695 and provide the supporting MS data via ProteomExchange. The study includes 2 biological replicates with 6 different datasets:  G1: in-gel digestion with trypsin, replicate 1 G2: in-gel digestion with trypsin, replicate 2 T1: SEC fractionation of low molecular weight (LMW) proteins and subsequent trypsin digestion, replicate 1 T2: SEC fractionation of LMW proteins and subsequent trypsin digestion, replicate 2 A1: SEC fractionation of LMW proteins and subsequent AspN digestion, replicate 1 A2: SEC fractionation of LMW proteins and subsequent AspN digestion, replicate 2 L1: SEC fractionation of LMW proteins and subsequent LysC digestion, replicate 1 L2: SEC fractionation of LMW proteins and subsequent LysC digestion, replicate 2  In our proteogenomics approach, we could identify four previously missing protein annotations and were able to correct sequences of six protein coding regions. Furthermore we identified signal peptidase cleavage sites for 72 different proteins. MGFs were generated by Maxquant 1.1 [1] using recalibration of peptide parent masses. For PRIDE (http://www.ebi.ac.uk/pride) submission, we made an additional database search with Mascot and X!Tandem using the SearchGUI [2]. Therefore we searched against a NCBI database of H. pylori strain 26695 complemented with the sequence corrections, signal peptide cleavage sites and missing annotations with the same configurations as described in materials and methods. For pride xml export we used the software PeptideShaker (http://code.google.com/p/peptide-shaker/). The complemented database has entries which will be submitted to the UniProtKB via SPIN. The entries have the according SPIN number as accession number. The NCBI accession numbers for the shortened sequences due to signal peptide cleavage are extended with “_1”. The fasta database is added to the submission. For additional information, please contact me: stephan.mueller@ufz.de  Yours sincerely, Stephan Mueller  References: [1] Cox J, Neuhauser N, Michalski A, Scheltema RA, Olsen JV, Mann M. Andromeda: a peptide search engine integrated into the MaxQuant environment. Journal of proteome research. 2011;10:1794-805. [2] Vaudel M, Barsnes H, Berven FS, Sickmann A, Martens L. SearchGUI: An open-source graphical user interface for simultaneous OMSSA and X!Tandem searches. Proteomics. 2011;11:996-9.

### Sample Protocol
Not available

### Data Protocol
Not available

### Publication Abstract
None

### Keywords
Helicobacter pylori strain, Proteogenomics, Coding sequence, Ltq orbitrap, Proteomics, Signal peptide

### Affiliations
Proteomics

### Submitter
Stephan MÃ¼ller

### Lab Head


