### Accession
PXD018185

### Title
Impact of whole-body versus nose-only exposure systems on systemic, respiratory, and cardiovascular endpoints in a 2-month cigarette smoke exposure study in the ApoE-/- mouse model [Lung data]

### Description
The objective of this study was to compare the biological and toxicological response of apolipoprotein E-deficient (Apoe-/-) mice to 3R4F mainstream smoke exposure for 2 months in whole-body exposure chambers (WBEC) and nose-only exposure chambers (NOEC). Female ApoE-/- mice were randomized into four groups: two Sham groups, exposed to filtered air, and two 3R4F groups, exposed to CS from the 3R4F reference cigarette (550 Âµg TPM/L). Half the number of mice in the Sham- and CS-exposed groups were exposed in WBECs and the other half in NOECs. The exposure phase lasted 9 weeks and included 1 week of adaption, during which exposure in both chamber types was escalated in dose and duration to a maximum of 4 h per day. The TPM concentration and exposure duration in WBECs were matched to those in NOECs on the basis of the regimen that the mice tolerated, as determined by in-life findings on acute signs of nicotine toxicity. Fresh air breaks were introduced during the exposure period to maintain carboxyhemoglobin (COHb) concentrations at acceptable levels. More frequent and longer fresh air breaks were required for exposure in the WBEC than in the NOEC because of the greater internal volumeâ€”and, consequently, the longer durationâ€”required to clear smoke from the WBEC than from the NOEC. For animals in NOECs, a 30-min fresh air break was introduced after 2 and 3 h of exposure. For animals in WBECs, a 30-min fresh air break was introduced after 1 and 2 h of exposure and a 60-min fresh air break after the third hour of exposure. The general condition and health of the mice following exposure were monitored throughout the study. Full necropsy was performed 16-20 h after the last exposure without prior fasting, in accordance with previously described methods (Vanscheeuwijck et al., 2002). Differences between WBEC and NOEC expsoure were analyzed with regard to aerosol uptake, disease endpoints (adaptive changes in nasal epithelia, changes in lung function and inflammatory parameters, plasma cholesterol/triglyceride levels in lipoprotein fractions, and atherosclerosis plaque development), and systems biology endpoints (changes in the lung proteome and liver, nasal epithelial, and heart transcriptomes). All procedures involving animals were performed in a facility accredited by the Association for Assessment and Accreditation of Laboratory Animal Care International and licensed by the Agri-Food & Veterinary Authority of Singapore, with approval from an Institutional Animal Care and Use Committee and in compliance with the National Advisory Committee for Laboratory Animal Research Guidelines on the Care and Use of Animals for Scientific Purposes (NACLAR, 2004). Here, the protein expression data for lung tissue assessed by iTRAQ®-based quantitative proteomics will be described.

### Sample Protocol
Whole-body perfusion with cold saline was performed prior to organ removal. Proteins were extracted from one of the right lung lobes as previously described (Lee et al. 2018; Phillips et al. 2015). Briefly, samples were processed and analyzed in randomized order. Frozen right lung tissue samples were homogenized by using a bead-assisted procedure in a Tissue Lyser II (Qiagen, Hilden, Germany) in tissue lysis buffer (BioRad Laboratories, Hercules, CA, USA) before acetone precipitation. Protein precipitates were resuspended in 0.5 M triethylammonium bicarbonate (Sigma-Aldrich, St. Louis, MO, USA), 1 M urea (Sigma-Aldrich), and 0.1% sodium dodecyl sulfate (Sigma-Aldrich).  The protein suspensions (50ug) were processed by using the iTRAQ® 8-plex labeling procedure in accordance with the manufacturer's instructions (AB Sciex, Framingham, MA, USA). The samples were analyzed in random order by using an Easy nanoLC 1000 instrument (Thermo Fisher Scientific, Waltham, MA, USA) connected online to a Q Exactive(TM)mass analyzer (Thermo Fisher Scientific). Each sample was injected twice, with two different analytical methods (one fast and another sensitive) on the same column, as previously described  (Kelstrup et al. 2014). The outputs of both mass spectrometry runs were combined as merged mass lists and interrogated against the mouse reference proteome set (UniProt, version July 2014, canonical isoforms only) by using Proteome Discoverer, version 1.4 (Thermo Fisher Scientific). SequestHT implemented in Proteome Discoverer was used as the search tool, and iTRAQ® reporter-ion intensities were determined from Proteome Discoverer. The Percolator node of Proteome Discoverer was used to estimate peptide-level false discovery rate (FDR)-adjusted p-values (q-values).

### Data Protocol
iTRAQ peptide-level quantification data were exported and further processed in the R statistical environment (R Development Core Team, 2007). The quantification data were filtered for a q-value < 0.01 and for 'unique' quantification results as defined in Proteome Discoverer. Global variance stabilizing normalization was performed with the corresponding Bioconductor package in R (Huber et al. 2002; Hultin-Rosenberg et al. 2013). Each iTRAQ reporter-ion set was normalized to its median, and protein expression values were calculated as the median of these normalized peptide-level quantification values (Herbrich et al. 2013). For detecting differentially abundant proteins, a linear model was fitted for each group comparison, and p-values were calculated from moderated t-statistics with the empirical Bayes approach (Gentleman et al. 2004). The Benjamini-Hochberg (BH) FDR method was then used to correct for multiple testing effects. Proteins with adjusted p-values <0.05 were considered as being differentially abundant.    References   Gentleman RC, Carey VJ, Bates DM, et al. (2004) Bioconductor: open software development for computational biology and bioinformatics. Genome biology 5(10):R80   Herbrich SM, Cole RN, West Jr KP, et al. (2013) Statistical inference from multiple iTRAQ experiments without using common reference standards. Journal of proteome research 12(2):594-604   Huber W, von Heydebreck A, SÃ¼ltmann H, Poustka A, Vingron M (2002) Variance stabilization applied to microarray data calibration and to the quantification of differential expression. Bioinformatics 18(suppl_1):S96-S104 doi:10.1093/bioinformatics/18.suppl_1.S96  Hultin-Rosenberg L, Forshed J, Branca RM, LehtiÃ¶ J, Johansson HJ (2013) Defining, comparing, and improving iTRAQ quantification in mass spectrometry proteomics data. Molecular & Cellular Proteomics 12(7):2021-2031   Kelstrup CD, Jersie-Christensen RR, Batth TS, et al. (2014) Rapid and deep proteomes by faster sequencing on a benchtop quadrupole ultra-high-field Orbitrap mass spectrometer. Journal of proteome research 13(12):6187-6195   Lee KM, Hoeng J, Harbo S, et al. (2018) Biological changes in C57BL/6 mice following 3 weeks of inhalation exposure to cigarette smoke or e-vapor aerosols. Inhal Toxicol 30(13-14):553-567 doi:10.1080/08958378.2019.1576807  Phillips B, Esposito M, Verbeeck J, et al. (2015) Toxicity of aerosols of nicotine and pyruvic acid (separate and combined) in Sprague-Dawley rats in a 28-day OECD 412 inhalation study and assessment of systems toxicology. Inhal Toxicol

### Publication Abstract
None

### Keywords
Lc-msms, Mouse, Lung, Itraq, Systems toxicology

### Affiliations
Philip Morris International R&D, Philip Morris Products S.A., Quai Jeanrenaud 5, 2000 Neuchâtel, Switzerland
Philip Morris International R&D, Philip Morris Products S.A., Quai Jeanrenaud 5, 2000 NeuchÃ¢tel, Switzerland

### Submitter
Bjoern Titz

### Lab Head
Dr Julia Hoeng
Philip Morris International R&D, Philip Morris Products S.A., Quai Jeanrenaud 5, 2000 NeuchÃ¢tel, Switzerland


