### Accession
PXD012151

### Title
Proteomic analysis of focal cerebral ischemia/reperfusion injury in rats

### Description
Stroke is a leading cause of mortality and long-term disability and ischemic stroke accounts for 87% of all strokes. Though timely recanalization of the occluded vessel is essential in the treatment of ischemic stroke, it is well known to cause ischemia-reperfusion (I/R) injury which result in neuronal cell death, brain tissue loss and severe neurological deficits. In this work, we employed a global proteomic approach to examine the changes of cerebral cortex proteins in rats undergoing acute and long-term I/R injury. In vivo middle cerebral artery occlusion (MCAO) model of focal cerebral I/R injury in rats was established. The animals were divided into three model groups with 2 h-MCAO followed with different reperfusion time, 1 day, 7 days and 14 days, respectively. For each model group a sham group was correspondingly set. Each group included four animals. For proteomic analysis, cerebral cortex proteins were extracted and analyzed by SDS-PAGE, whole-lane slicing, in-gel digestion and label-free quantitative LC-MS/MS. A total of 5621 proteins were identified and their quantities between the surgery and corresponding sham groups and across the three reperfusion time points were compared for mechanism investigation. This dataset includes all the raw files of the 840 LC-MS runs (6 groups x 4 animals x 35 gel squares/sample), as well as their identification and quantitation results at the levels of peptide fragments, peptides and proteins, respectively.

### Sample Protocol
(1) Sample preparation.  Rats which were suffered MCAO/R were transcardially perfused with 60 mL pre-cold saline and brains were obtained immediately. The right cerebral cortices of damaged region were separated, snap-frozen in liquid nitrogen and stored at -80C until use. Protein extraction procedures was referenced as previous article (Wisniewski, J., et al., Nature Methods, 2009, 6, 359-362). The 50-mg pieces of frozen samples were homogenized in 400 uL of 0.1 M Tris-HCl, pH 7.6 using high throughput grinder at 40 Hz shake frequency for 100 s. Then 50-uL aliquots of 20 % w/v SDS and 1 M DTT were added to the homogenates and the mixtures were incubated for 3 min at 95C. The lysates were sonicated by ultrasonic cell crusher (VCX-150 Sonics & Materials, Inc.) to reduce viscosity for 80 s (10-s pulse at 30% amplitude with 20-s interval, eight cycles) and then centrifuged at 16000 ï‚´ g for 15 min at 4C. The supernatants were collected and stored in aliquots at -80C. The protein concentrations were determined using adsorptions at 230, 280, 320 nm. (2) SDS-PAGE.  Slab SDS-PAGE gels (38 wide x 43 high x 1 mm thick, 4-17.85% T linear pore-size gradient, 5% C, with six 4-mm wide wells) were prepared with an automatic gradient gel former. The samples were loaded at 20 ug per lane and run at 10 mA/gel constant current in a 0.05 M Tris-0.1 M glycine-0.1% w/v SDS buffer (pH 9.0) until the 10 kDa pre-stained marker band in the marker lanes reached to the gel bottom. A total of 24 samples (6 groups x 4 animals) were loaded randomly on five SDS-PAGE gels. The gels were stained with CBB R-250. The apparent molecular masses of the pre-stained protein markers were labeled at the left of the gel pattern (see *Gel pattern -WML 181218.pptx*, page 1). (3) Gel slicing and square numbering.  A set of laboratory-made cutters with multiple razor blades were used to i) vertically cut two 1.1 mm-wide gel lanes from each sample lane, and then ii) horizontally cut each lane into 35 square pieces. Note in each step the cutting was performed simultaneously on all lanes in one gel. (see *Gel pattern -WML 181218.pptx*, page 2). For each sample one set of the 35 gel squares, numbered from spot 1 to spot 35 from the gel top, was subjected to proteomic analysis.   (4) In-gel digestion.  All the 840 gel pieces were subjected to standardized procedures of detaining, reduction, alkylation and in-gel trypsin digestion as previously described [Jin, Y., et al., Electrophoresis 2007, 28, 843-863] except two modifications, namely, i) 10 uL 5 ng/uL trypsin was used for each gel, and ii) for spots 1-3 of each lane, the sonication step to extract tryptic peptides was replaced by shaking for 20 min to avoid the extraction of low-molecular-mass polyacrylamide from the soft gel top. The same numbered gel pieces of the 24 sample, e.g. spot 1 of the 24 samples, were treated in one batch for in-gel digestion. Totally 35 batches, spots 1 to 35, were treated sequentially. The tryptic peptide solutions were dried by vacuum centrifuge and reconstituted with 12 uL 1% v/v formic acid/5% v/v acetonitrile. (5) Nano-UPLC-MS/MS.  The peptide separations were performed on an Ultimate 3000 RSLCnano HPLC (Dionex). Mobile phase for sample-loading was 0.1% v/v trifluoroacetic acid in water. Mobile phases A and B for elution on analytical column were 0.1% v/v formic acid in water and 0.1% v/v formic acid in acetonitrile, respectively. For each analysis, 3 uL of the peptide solution, injected in micro-pickup mode, was initially loaded to the trap column (Acclaim PepMap100 C18, 5 um, 100 A, 100 um i.d. x 2 cm, Dionex) at 15 uL/min for 5 min. Then the peptides were eluted from the trap column and separated in the analytical column (Acclaim PepMap RSLC C18, 2 um, 100 A, 75 um i.d. x 15 cm, Dionex) at 400 nL/min with a 32 min-long gradient as follows; 1% B for 5 min, ramp to 40% B over 20 min, ramp from 40 to 85% B over 1 min and hold for 5 min, then ramp back to 1% B over 1 min. The column and sampler chamber was maintained at 35C and 8C, respectively. To ensure mass accuracy, a calibration segment (1 mM sodium formate, NaFA) was inserted at the beginning of the gradient. The nano-LC system was coupled to a maXis II ETD Q-TOF MS (Bruker) through a Captive Spray nanoBooster source. The source parameters setting were as follow: capillary voltage at 1600 V, nanoBooster gas pressure at 0.2 bars and Dry gas flow and temperature set to 3 L/min and 150C, respectively. The nanoBooster reservoir was filled with acetonitrile. Tandem mass spectra from collision induced dissociation (CID) were acquired at an intensity-dependent rate of 4-16 Hz between precursor scans with dynamic exclusion enabled and for precursor ions with charges between +2 to +5.

### Data Protocol
LC-MS/MS data were re-calibrated using Data Analysis 4.4 (Bruker) with the signals of NaFA that was infused at the first 5 min of the LC-gradient. Progenesis QI for proteomics (QIP) ver. 4.1 (Nonlinear Dynamic, Waters) was used for peak list generation (for identification by Mascot) and then quantitation. Briefly, the raw LC-MS/MS files from the 840 gel pieces (6 groups x 4 animals x 35 gel pieces) were imported into QIP in 35 *experiments* and each experiment contained the data from the same numbered squares in the 24 sample lanes, e.g. all spots 1 in the 24 sample lanes were grouped in one *experiment* and all spots 2 in another. For each experiment (spot), the data was processed with QIP for primary signal filtering, retention time alignment, peak picking, normalization and generation of the peak list (.mgf). The peak list was then analyzed by Mascot (ver. 2.6.02) with parameters set as follows; database, UniProtKB Rattus norvegicus (2017-12-21, 36081 sequences); mass tolerance for precursor ions, 10 ppm; mass tolerance for fragment ions, 0.05 Da; digest reagent, trypsin; maximum missed cleavage, 1; fixed modification, carbamidomethylation at Cys; variable modifications, oxidation at Met and acetylation at N-terminal amines; false positive rate (FPR) at peptide level, 1%. The identification result output from the Mascot search (.XML) was then imported back to the QIP experiment for further quantitation using the High-N (N=3) setting. After the QIP processing of all the 35 experiments were done, the *multiple-fraction experiment* workflow of QIP was used to combine the 35 experiments into one experiment.

### Publication Abstract
Stroke is a leading cause of mortality and disability, and ischemic stroke accounts for more than 80% of the disease occurrence. Timely reperfusion is essential in the treatment of ischemic stroke, but it is known to cause ischemia-reperfusion (I/R) injury and the relevant studies have mostly focused on the acute phase. Here we reported on a global proteomic analysis to investigate the development of cerebral I/R injury in the subacute and long-term phases. A rat model was used, with 2 h-middle cerebral artery occlusion (MCAO) followed with 1, 7, and 14 days of reperfusion. The proteins of cerebral cortex were analyzed by SDS-PAGE, whole-gel slicing, and quantitative LC-MS/MS. Totally 5621 proteins were identified, among which 568, 755, and 492 proteins were detected to have significant dys-regulation in the model groups with 1, 7, and 14 days of reperfusion, respectively, when compared with the corresponding sham groups (<i>n</i> = 4, fold change &#x2265;1.5 or &#x2264;0.67 and <i>p</i> &#x2264; 0.05). Bioinformatic analysis on the functions and reperfusion time-dependent dys-regulation profiles of the proteins exhibited changes of structures and biological processes in cytoskeleton, synaptic plasticity, energy metabolism, inflammation, and lysosome from subacute to long-term phases of cerebral I/R injury. Disruption of cytoskeleton and synaptic structures, impairment of energy metabolism processes, and acute inflammation responses were the most significant features in the subacute phase. With the elongation of reperfusion time to the long-term phase, a tendency of recovery was detected on cytoskeleton, while inflammation pathways different from the subacute phase were activated. Also, lysosomal structures and functions might be restored. This is the first work reporting the proteome changes that occurred at different time points from the subacute to long-term phases of cerebral I/R injury and we expect it would provide useful information to improve the understanding of the mechanisms involved in the development of cerebral I/R injury and suggest candidates for treatment.

### Keywords
Time course, Ischemia-reperfusion injury, Rat, Middle cerebral artery occlusion (mcao), Brain stroke, Proteomics

### Affiliations
Institute of Biomedical and Pharmaceutical Sciences, Guangdong University of Technology

### Submitter
Ya Jin

### Lab Head
Dr Ya Jin
Institute of Biomedical and Pharmaceutical Sciences, Guangdong University of Technology


