### Accession
PXD022282

### Title
Multi-level proteomics reveals host-perturbation strategies of SARS-CoV-2 and SARS-CoV

### Description
The  sudden  global  emergence  of  SARS-CoV-2  urgently  requires  an  in-depth  understanding  of  molecular  functions  of  viral  proteins  and  their  interactions  with  the  host  proteome.  Several  omics  studies  have  extended  our  knowledge  of  COVID-19  pathophysiology,  including  some  focused  on  proteomic aspects1â€“3. To understand how SARS-CoV-2 and related coronaviruses manipulate the host we here characterized interactome, proteome and signaling processes in a systems-wide manner. This identified  connections  between  the  corresponding  cellular  events,  revealed  functional  effects  of  the  individual  viral  proteins  and  put  these  findings  into  the  context  of  host  signaling  pathways.  We  investigated  the  closely  related  SARS-CoV-2  and  SARS-CoV  viruses  as  well  as  the  influence  of  SARS-CoV-2  on  transcriptome,  proteome,  ubiquitinome  and  phosphoproteome  of  a  lung-derived  human  cell  line.  Projecting  these  data  onto  the  global  network  of  cellular  interactions  revealed  relationships  between  the  perturbations  taking  place  upon  SARS-CoV-2  infection  at  different  layers  and  identified  unique  and  common  molecular  mechanisms  of  SARS  coronaviruses.  The  results  highlight  the  functionality  of  individual  proteins  as  well  as  vulnerability  hotspots  of  SARS-CoV-2,  which  we  targeted  with  clinically  approved  drugs.  We exemplify  this  by  identification  of  kinase  inhibitors as well as MMPase inhibitors with significant antiviral effects against SARS-CoV-2.

### Sample Protocol
DIA Measurements Samples were loaded onto a 50 cm reversed phase column (75 Î¼m inner diameter, packed in house with ReproSil-Pur C18-AQ 1.9 Î¼m resin [Dr. Maisch GmbH]). The column temperature was maintained at 60Â°C using a homemade column oven. A binary buffer system, consisting of buffer a (0.1% formic acid (FA)) and buffer B (80% ACN plus 0.1% FA) was used for peptides separation, at a flow rate of 300 nl/min. An EASY-nLC 1200 system (Thermo Fisher Scientific), directly coupled online with the mass spectrometer (Orbitrap Exploris 480, Thermo Fisher Scientific) via a nano-electrospray source, was employed for nano-flow liquid chromatography. The FAIMS device was placed between the nanoelectrospray source and the mass spectrometer and was used for measurements of the proteome and the PTM-library samples. Spray voltage were set to 2650 V, RF level to 40 and heated capillary temperature to 275Â°C. For proteome measurements we use a 100 min gradient starting at 5% buffer B followed by a stepwise increase to 30% in 80 min, 60% in 4 min and 95% in 4 min. The buffer B concentration stayed at 95% for 4 min, decreased to 5% in 4 min and stayed there for 4 min. The mass spectrometer was operated in data-independent mode (DIA) with a full scan range of 350-1650 m/z at 120,000 resolution at 200 m/z, normalized automatic gain control (AGC) target of 300% and a maximum fill time of 28 ms. One full scan was followed by 22 windows with a resolution of 15,000, normalized automatic gain control (AGC) target of 1000% and a maximum fill time of 25 ms in profile mode using positive polarity. Precursor ions were fragmented by higher-energy collisional dissociation (HCD) (NCE 30%). Each of the selected CVs (-40, -55 and -70) was applied to sequential survey scans and MS/MS scans; the MS/MS CV was always paired with the appropriate CV from the corresponding survey scan.  For phosphopeptide samples, 5Âµl were loaded and eluted with a 70 min gradient starting at 3% buffer B followed by a stepwise increase to 19% in 40 min, 41% in 20 min, 90% in 5 min and 95% in 5 min. The mass spectrometer was operated in data-independent mode (DIA) with a full scan range of 300-1400 m/z at 120,000 resolution at 200 m/z and a maximum fill time of 60 ms. One full scan was followed by 32 windows with a resolution of 30,000. Normalized automatic gain control (AGC) target and maximum fill time were set to 1000% and 54 ms, respectively, in profile mode using positive polarity. Precursor ions were fragmented by higher-energy collisional dissociation (HCD) (NCE stepped 25-27.5-30%). For the library generation, we enriched A549 cell lysates for phosphopeptides and measured them with 7 different CV settings (-30, -40, -50, -60, -70, -80 or -90 V) using the same DIA method. The noted CVs were applied to the FAIMS electrodes throughout the analysis. For the analysis of K-É›-GG peptide samples, half of the samples were loaded. We used 120 min gradient starting at 3% buffer B followed by a stepwise increase to 7% in 6 min, 20% in 49 min, 36% in 39 min, 45% in 10 min and 95% in 4 min. The buffer B concentration stayed at 95% for 4 min, decreased to 5% in 4 min and stayed there for 4 min. The mass spectrometer was operated in data-independent mode (DIA) with a full scan range of 300-1350 m/z at 120,000 resolution at m/z 200, normalized automatic gain control (AGC) target of 300% and a maximum fill time of 20 ms. One full scan was followed by 46 windows with a resolution of 30,000. Normalized automatic gain control (AGC) target and maximum fill time were set to 1000% and 54 ms, respectively, in profile mode using positive polarity. Precursor ions were fragmented by higher-energy collisional dissociation (HCD) (NCE 28%). For K-É›-GG peptide library, we mixed the first replicate of each sample and measured them with eight different CV setting (-35, -40, -45, -50, -55, -60, -70 or -80 V) using the same DIA method.

### Data Protocol
Spectronaut version 14 (Biognosys)  was used to generate the libraries and analyze all DIA files using the human fasta file (2019) containing viral proteins . For the generation of the PTM-specific libraries, the DIA single CV runs were combined with the actual DIA runs and either phosphorylation at Serine/Threonine/Tyrosine or GlyGly at Lysine was added as variable modification to default settings. Maximum number of fragment ions per peptide was increased to 25. The proteome DIA files were analyzed using direct DIA approach with default settings and disabled cross run normalization . All PTM DIA files were analyzed using their respective hybrid library and either phosphorylation at Serine/Threonine/Tyrosine or GlyGly at Lysine was added as an additional variable modification to default settings with disabled PTM localization filter. For PTM site analysis, Spectronaut normal report output tables were aggregated to PTM sites using the peptide collapse plug-in tool for Perseus . PTM sites were aggregated using the linear model based approach.

### Publication Abstract
None

### Keywords
Sars-cov-2, Ubiquitination, Phosphorylation, Covid19, Proteomics, Ptm, Lc-ms/ms, Dia, Mass spectrometry

### Affiliations
Proteomics
Department of Proteomics and Signal Transduction, Max Planck Institute of Biochemistry, 82152 Martinsried, Germany

### Submitter
Mario Oroshi

### Lab Head
Dr Matthias Mann
Department of Proteomics and Signal Transduction, Max Planck Institute of Biochemistry, 82152 Martinsried, Germany


