### Accession
PXD017054

### Title
Chromatin accessibility promotes hematopoietic and leukemia stem cell activity

### Description
Chromatin organization is a highly orchestrated process that influences gene expression, in part by modulating access of regulatory factors to DNA and nucleosomes. We found that the chromatin accessibility regulator HMGN1, a target of recurrent DNA copy gains in leukemia, controls myeloid differentiation. HMGN1 amplification was associated with increased accessibility, expression, and histone H3K27 acetylation of loci important for hematopoietic stem cell (HSC) function and AML, such as HoxA cluster genes. In vivo, HMGN1 overexpression was linked to decreased quiescence and increased HSC activity in bone marrow transplantation. HMGN1 overexpression also cooperated with the AML-ETO9a fusion oncoprotein to impair myeloid differentiation and enhance leukemia stem cell (LSC) activity. Inhibition of histone acetyltransferases CBP/p300 relieved the HMGN1-associated differentiation block. These data nominate factors that modulate chromatin accessibility as regulators of HSCs and LSCs and suggest that targeting HMGN1 or its downstream effects on histone acetylation could be therapeutically active in AML.

### Sample Protocol
Myeloid progenitor cells were washed twice with ice-cold PBS and lysed in urea buffer (8 M urea, 75 mM NaCl, 50 mM EPPS pH 8.0, complete protease inhibitors (Roche) and PhosSTOP phosphatase inhibitors (Roche). Protein concentration was determined by BCA (Thermo Fisher Scientific). Next, 100 ��g of protein were reduced with 5 mM TCEP and alkylated with 14 mM iodoacetamide (IAA). All incubations were performed at room temperature for 30 min. Proteins were precipitated by methanol/chloroform, resuspend in 200 mM EPPS and digested with LysC for 12 h at room temperature (1:100, LysC:Protein). Then, trypsin was added to the peptide mixture and further digested for 5 h at 37 ��C (1:75, Trypsin:Protein). After digestion, peptide concentration was calculated using the quantitative colorimetric peptide assay (Pierce). A total of 25 ��g of peptides were labeled with TMT-10plex. After labeling, all samples were combined in equal amounts and fractionated in a basic pH reverse phase chromatography. All 96 fractions collected were combined into 24, 12 of which were desalted via STAGE-TiP and dried in a vacuum centrifuge. Finally, peptides were resuspended in 3% ACN, 1% formic acid and analyzed by LC-SPS-MS3 in an Orbitrap Fusion (Thermo Fisher Scientific) coupled to a Proxeon EASY-nLC II LC pump (Thermo Fisher Scientific).

### Data Protocol
Mass spectra were processed using a SEQUEST-based in-house software pipeline. Spectra were converted to mzXML using a modified version of ReAdW.exe. Database searching was performed against the mouse Uniprot database (February 2014) and the most common contaminants. This database was concatenated with a database composed of all protein sequences in the reversed order. Searches were performed using a 20-ppm precursor ion tolerance and a product ion tolerance of 0.9 Da. TMT tags on lysine residues and peptide N termini (+229.163 Da) and carbamidomethylation of cysteine residues (+57.021 Da) were set as static modifications, whereas oxidation of methionine residues (+15.995 Da) was set as a variable modification. For TMT-based reporter ion quantitation, we extracted the signal-to-noise ratio for each TMT channel and found the closest matching centroid to the expected mass of the TMT reporter ion. Peptide-spectrum matches were identified, quantified, and collapsed to a 1% peptide false discovery rate (FDR) and then collapsed to a final protein-level FDR of 1%. Protein assembly was guided by principles of parsimony to produce the smallest set of proteins necessary to account for all observed peptides. Proteins were quantified by summing reporter ion counts across all matching peptide-spectrum matches using in-house software. Peptide-spectrum matches with poor quality, MS3 spectra with TMT reporter-summed signal to noise ratio that is <100, MS2 isolation specificity <0.7, or no MS3 spectra were excluded from quantitation. Finally, quantifications were normalized assuming equal protein loading for all ten samples in the TMT.

### Publication Abstract
Chromatin organization is a highly orchestrated process that influences gene expression, in part by modulating access of regulatory factors to DNA and nucleosomes. Here, we report that the chromatin accessibility regulator HMGN1, a target of recurrent DNA copy gains in leukemia, controls myeloid differentiation. HMGN1 amplification is associated with increased accessibility, expression, and histone H3K27 acetylation of loci important for hematopoietic stem cells (HSCs) and leukemia, such as HoxA cluster genes. In vivo, HMGN1 overexpression is linked to decreased quiescence and increased HSC activity in bone marrow transplantation. HMGN1 overexpression also cooperates with the AML-ETO9a fusion oncoprotein to impair myeloid differentiation and enhance leukemia stem cell (LSC) activity. Inhibition of histone acetyltransferases CBP/p300 relieves the HMGN1-associated differentiation block. These data nominate factors that modulate chromatin accessibility as regulators of HSCs and LSCs, and suggest that targeting HMGN1 or its downstream effects on histone acetylation could be therapeutically active in AML.

### Keywords
Aml, Chromatin, Leukemia, Hmgn1

### Affiliations
Harvard Medical School
Finley and Gygi lab Cell Biology Department Harvard Medical School US

### Submitter
Miguel Prado

### Lab Head
Dr Daniel Finley
Finley and Gygi lab Cell Biology Department Harvard Medical School US


