### Accession
PXD003232

### Title
Identification of N-terminally arginylated proteins from the moss Physcomitrella patens

### Description
Protein arginylation is a post-translational modification at the N-terminus of proteins and is crucial for viability and physiology in higher eukaryotes. The lack of arginylation causes severe developmental defects in plants and embryo lethality in Drosophila and in mice. Although several studies investigated impact and function of the responsible enzyme, the arginyl-tRNA protein transferase (ATE) in plants, identification of arginylated proteins by mass spectrometry was not hitherto achieved. In the present study, we report the identification of targets and interaction partners of ATE in the model plant Physcomitrella patens by mass spectrometry employing two different immuno-affinity strategies and a recently established transgenic ATE:GUS reporter line (Schuessele et al., 2015 New Phytol., DOI: 10.1111/nph.13656). Here we use a commercially available antibody against the fused reporter protein (GUS) to pull down ATE and its interacting proteins and validate the in vivo interaction with a class I small heatshock protein via Förster resonance energy transfer (FRET). Additionally, we apply and modify a method that already successfully identified arginylated proteins from mouse proteomes by using custom-made antibodies specific for N-terminal arginine. As a result, we identify four arginylated proteins from Physcomitrella patens with high confidence.

### Sample Protocol
Immunoprecipitation (IP) of N-terminally arginylated proteins  Antibodies against N-terminal arginine were ordered from GenScript USA Inc. (Piscataway, NJ) custom-made using synthetic peptides (REHKHANQHMSVC, RDHKHANQHMSVC, sequences were chosen according to (DOI: 10.1371/journal.pbio.0050258 )). The obtained antibodies were subjected to negative selection against “false positive” synthetic peptides (HKERDANQHMSVC, HKRREANQHMSVC, HKHANQHMSVC, sequences were chosen according to (DOI: 10.1371/journal.pbio.0050258 )). For this purpose a mixture containing equal amounts of the false positive peptides was coupled to Sulfolink® Coupling Resin (Thermo Scientific) according to the manufacturer’s instructions. A mixture of equal amounts of both antibodies was applied and fractions of the flow-through, the washing steps as well as elution fractions were collected. The specificity of the antibodies from all fractions was monitored via dot blot analysis using the peptides containing N-terminal arginine as well as the false positive peptides. Antibodies from the washing and the flow-through fractions were used for the immunoprecipitation of arginylated proteins. For this, the Dynabeeds M270 Epoxy Co-Immunoprecipitation Kit with the same buffer as before was used. For each experiment of the glucose and MG132 treated samples 8-10 g fresh weight gametophores from liquid culture were used. Immunoprecipitation of arginylated proteins after cultivation in darkness was performed with 5 g (fresh weight) gametophores harvested from hydroponic ring cultures. Eluted proteins were precipitated using acetone as described before. The dimethylation reaction was carried out according to (doi:10.1038/nbt.1611) with modifications. Protein pellets were dissolved in 100 mM HEPES-NaOH, pH 7.5, 0.2% SDS and reduced at 95°C for 10 min using Reducing Agent (Life Technologies). Alkylation of cysteines was performed at a final concentration of 100 mM IAA for 20 min at RT. Dimethylation was performed by adding 2 µl per 100 µl sample of 4% isotope-labeled formaldehyde solution (13C,d2, Sigma Aldrich) and 2 µl of a 500 mM NaCNBH3 solution. The reaction was carried out at 37°C for 4 h. Then the same amounts of formaldehyde and NaCNBH3 were added and the reaction continued overnight. The reaction was stopped by adding 2 µl of 4% NH4OH solution per 100 µl sample for 1 h at 37°C. Finally, the dimethylated samples were again precipitated using acetone without DTT as described before. Dry protein pellets were dissolved in 50 mM Tris-HCl pH 7.6, 2% SDS and mixed with Laemmli buffer prior to SDS page. MS/MS sample preparation After Coomassie staining (PageBlue, Thermo Scientific), gel slices were excised, chopped to small pieces and destained with 30% ACN (Promochem, Teddington, UK) in 100 mM NH4HCO3 (Sigma-Aldrich) for 10 min. The step was repeated until the gel was completely destained. The supernatant was discarded and the gel was equilibrated in 100 µl 100mM NH4HCO3. The supernatant was discarded again and the gel was shrunk with 100 µl ACN for 5 min with gentle shaking. Again, the supernatant was discarded and the gel was vacuum-dried. Digests with trypsin (Promega) or elastase (Promega) were performed overnight at 37°C in 50mM NH4HCO3 (pH 8). About 0.1 µg of protease was used for one gel band. Peptides were extracted from the gel slices with 5 % formic acid.   Mass spectrometry (LC-MS/MS) Nano LC-MS/MS analyses were performed using an LTQ-Orbitrap Velos Pro (Thermo Scientific) equipped with an EASY-Spray Ion Source and coupled to an EASY-nLC 1000 (Thermo Scientific). Peptides were applied on a trapping column (2 cm x 75 µm ID, PepMap C18, 3 µm particles, 100 Å pore size) and separated on an EASY-Spray column (25 cm x 75 µm ID, PepMap C18, 2 µm particles, 100 Å pore size) with a 30 minute linear gradient from 3% to 30% acetonitrile and 0.1% formic acid, and 200 nl/min flow rate. MS scans were acquired in the Orbitrap analyzer with a resolution of 30 000 at m/z 400. A TOP5 data-dependent MS/MS method was used and MS/MS scans were acquired in the Orbitrap analyzer with a resolution of 7500 at m/z 400 using HCD fragmentation with 30% normalized collision energy. Dynamic exclusion was applied with a repeat count of 1 and an exclusion duration of 30 seconds; unassigned and singly charged precursors were excluded from the selection. Minimum signal threshold for precursor selection was set to 50 000. Predictive AGC was used with AGC target value of 5e5 for MS scans and 5e4 for MS/MS scans. Lock mass option was applied for internal calibration using background ions from protonated decamethylcyclopentasiloxane (m/z 371.10124).

### Data Protocol
MS data analysis Raw data processing and database search for identification of arginylated proteins Raw data processing was performed using Mascot Distiller V2.5.1.0 (Matrix Science, Boston, Massachusetts, USA). Database searches on the processed raw data was performed using Mascot Daemon V2.4 (Matrix Science) against the Physcomitrella patens database containing all version 1.6 protein models (56), as well as their reversed sequences used as decoys, and simultaneously against an in-house database containing all sequences of known typical contaminants (e. g. human Keratins, trypsin, 267 entries total, available on request). The fixed modifications were carbamidomethyl (C) +57.021464 Da and 13C,d2 dimethyl (K) +34.063117 Da. Variable modifications used for the database search were Gln >pyro Glu (N term Q) -17.026549 Da, Oxidation (M) +15.994915 Da, Acetyl (N-term) +42.010565 Da, 13C,d2 dimethyl (N-term) +34.063117 Da and phospho (ST) +79.966331 Da. Additionally, the mass shifts of an N-terminally dimethylated arginine (+190.164228), as well as the mass shift of a dimethylated arginine at N-terminal glutamine residues that were converted into glutamate residues (+191.148243) were included. For all searches the peptide mass tolerance was ± 8 ppm and the fragment mass tolerance was set to ± 0.02 Da. The enzyme specificity was set to semitryptic with a total of 3 missed cleavage sites in the case of the trypsin digests. For samples digested with elastase specificity was set to none. All Mascot searches were loaded into Scaffold 4 (Version 4.3.4, Proteome Software Inc., Portland, Oregon, USA) and an additional database search against the Physcomitrella patens database using the same settings was performed using X!Tandem  implemented in the Scaffold 4 software. All loaded data was analyzed using the legacy PeptideProphet scoring (high mass accuracy) with independent sample protein grouping. Results were filtered using the Protein- and PeptideProphetTM  implemented in Scaffold 4 software.

### Publication Abstract
Protein arginylation is a posttranslational modification of both N-terminal amino acids of proteins and sidechain carboxylates and can be crucial for viability and physiology in higher eukaryotes. The lack of arginylation causes severe developmental defects in moss, affects the low oxygen response in Arabidopsis thaliana and is embryo lethal in Drosophila and in mice. Although several studies investigated impact and function of the responsible enzyme, the arginyl-tRNA protein transferase (ATE) in plants, identification of arginylated proteins by mass spectrometry was not hitherto achieved. In the present study, we report the identification of targets and interaction partners of ATE in the model plant Physcomitrella patens by mass spectrometry, employing two different immuno-affinity strategies and a recently established transgenic ATE:GUS reporter line (Schuessele et al., 2016 New Phytol. , DOI: 10.1111/nph.13656). Here we use a commercially available antibody against the fused reporter protein (&#x3b2;-glucuronidase) to pull down ATE and its interacting proteins and validate its in vivo interaction with a class I small heatshock protein via F&#xf6;rster resonance energy transfer (FRET). Additionally, we apply and modify a method that already successfully identified arginylated proteins from mouse proteomes by using custom-made antibodies specific for N-terminal arginine. As a result, we identify four arginylated proteins from Physcomitrella patens with high confidence.Data are available via ProteomeXchange with identifier PXD003228 and PXD003232.

### Keywords
N-end rule pathway, Physcomitrella patens, Arginylation

### Affiliations
Head, Chair Plant Biotechnology Faculty of Biology University of Freiburg Schaenzlestrasse 1 D-79104 Freiburg Germany
Faculty of Biology, University of Freiburg (Chair Plant Biotechnology), Schaenzlestr. 1, D-79104 Freiburg

### Submitter
Ralf Reski

### Lab Head
Dr Ralf Reski
Head, Chair Plant Biotechnology Faculty of Biology University of Freiburg Schaenzlestrasse 1 D-79104 Freiburg Germany


