### Accession
PXD015566

### Title
The crucial role of CPT2 in the asthmatic bronchial smooth muscle remodeling

### Description
Asthma is a very frequent airway disease that affects 6 to 20% of the population. Severe asthma, represents 3 to 5% of all asthmatic patients and is histologically characterized by an increased bronchial smooth muscle (BSM) mass and clinically by viral exacerbations. Functionally, BSM remodeling had a poor prognostic value in asthma, since higher BSM mass was associated with lower lung function and increased exacerbation rate. However, the role of BSM as a potential actor of asthma exacerbation has only been sparsely suggested. Thus, we hypothesis that asthmatic BSM cell metabolism is modified compare to that of non-asthmatic and that could be a potential target to reduce asthmatic BSM cell proliferation and remodeling in asthma.

### Sample Protocol
Extracted proteins were loaded and short separated on a 10% acrylamide SDS-PAGE gel. Migration was stopped when samples were entered the resolving gel and proteins were visualized by colloidal blue staining. Each Protein lanes were cut and then individually cut into small pieces. Proteins were reduced, alkylated and digested in peptides by trypsin. NanoLC-MS/MS analysis were performed using an Ultimate 3000 RSLC Nano-UPHLC system (Thermo Scientific, USA) coupled to a nanospray Orbitrap Fusion™ Lumos™ Tribrid™ Mass Spectrometer (Thermo Fisher Scientific, California, USA). Each peptide extracts were loaded on a 300 µm ID x 5 mm PepMap C18 precolumn (Thermo Scientific, USA) at a flow rate of 10 µL/min. After a 3 min desalting step, peptides were separated on a 50 cm EasySpray column (75 µm ID, 2 µm C18 beads, 100 Å pore size, ES803, Thermo Fischer Scientific) with a 4-40% linear gradient of solvent B (0.1% formic acid in 80% ACN) in 48 min. The separation flow rate was set at 300 nL/min. The mass spectrometer operated in positive ion mode at a 2.0 kV needle voltage. Data was acquired using Xcalibur 4.1 software in a data-dependent mode. MS scans (m/z 375-1500) were recorded at a resolution of R = 120000 (@ m/z 200) and an AGC target of 4×105 ions collected within 50 ms, followed by a top speed duty cycle of up to 3 seconds for MS/MS acquisition. Precursor ions (2 to 7 charge states) were isolated in the quadrupole with a mass window of 1.6 Th and fragmented with HCD@30% normalized collision energy. MS/MS data was acquired in the ion trap with rapid scan mode, AGC target of 3x103 ions and a maximum injection time of 300 ms. Selected precursors were excluded for 60 seconds.

### Data Protocol
For protein identification, Sequest HT and Mascot 2.4 algorithms through Proteome Discoverer 1.4 Software (Thermo Fisher Scientific Inc.) were used for protein identification in batch mode by searching against a Uniprot Homo sapiens database (71 536 entries, release march 2018). Two missed enzyme cleavages were allowed. Mass tolerances in MS and MS/MS were set to 10 ppm and 0.6 Da. Oxidation of methionine, deamidation of asparagine and glutamine, acetylation of lysine and N-terminal acetylation of the protein were searched as dynamic modifications. Carbamidomethylation on cysteine was searched as static modification. Peptide validation was performed using Percolator algorithm [2] and only “high confidence” peptides were retained corresponding to a 1% False Positive Rate at peptide level. Raw LC-MS/MS data were imported in Progenesis LC-MS QI 2.0 (Nonlinear Dynamics Ltd, U.K) for feature detection, alignment, and quantification. All sample features were aligned according to retention times by manually inserting up to one hundred landmarks followed by automatic alignment to maximally overlay all the two-dimensional (m/z and retention time) feature maps. Singly charged ions and ions with higher charge states than six were excluded from the analysis. All remaining features were used to calculate a normalization factor for each sample to correct for experimental variation. Peptide identifications (with p<0.01, see above) were imported into Progenesis. For quantification, all unique peptides of an identified protein were included, and the total cumulative abundance was calculated by summing the abundances of all peptides allocated to the respective protein. ANOVA as performed at the protein level and filtered to identify features with p <0.05. Quantitative data were considered for proteins quantified based on a minimum of 2 unique peptides.

### Publication Abstract
None

### Keywords
Lc-ms/ms, Smooth muscle cells, Asthma

### Affiliations
U1045, CRCTB, Bâtiment PTIB, Avenue de Haut Lévèque, 33000 Bordeaux, France
Plateforme Proteome Bordeaux

### Submitter
Dupuy Jean-William

### Lab Head
Dr Thomas Trian
U1045, CRCTB, Bâtiment PTIB, Avenue de Haut Lévèque, 33000 Bordeaux, France


