### Accession
PXD003635

### Title
SCN circadian proteomes of miR-132/212 KO and WT mice by super-SILAC-based quantitative mass spectrometry

### Description
To identify potential mechanisms underlying the phenotype of miR-132/212 KO mice under short photoperiods, we used quantitative mass spectrometry to analyze the SCN proteomes of miR-132/212 KO and WT mice under a 8:16 LD schedule at 4 time points, spaced 6h apart, across a 24h cycle (n=4 per time point per genotype). Quantification was achieved using a spike-in reference of three murine neural cell lines, including Neuro2A, and the adult mouse hypothalamic cell lines mHypoA-2/28 (CLU188) and mHypoA-SCN mix (CLU497) that had been labeled by SILAC (stable isotope labeling by amino acids in cell culture).

### Sample Protocol
SCN tissues of individual mice were lysed in 80 uL lysis buffer (4% SDS, 50 mM Tris-HCl (pH 8.0) with proteinase and phosphatase inhibitor cocktail (Roche) and homogenized with a Pellet pestle. The lysates were sonicated 3 times with 10 s each with at least 30 sec on ice between each pulse. Protein concentrations were determined using the Bio-Rad DC Protein Assay. The proteins were processed using the Filter Aided Sample Preparation method (FASP) as previously described with some modifications. Briefly, SCN lysates (45 ug of proteins) and heavy SILAC-labeled cell lysates (15 ug from each Neuro2A, CLU188 and CLU497 cells) were mixed at a 1:1 weight ratio prior to transfer onto the filter. After centrifugation at 16,000xg for 10 min, followed by two washes of 200 uL 8 M urea, 50 mM Tris-HCl, pH 8.0 (UA buffer), samples were reduced by incubating samples in the presence of 20 mM dithiothreitol in UA buffer. After centrifugation, samples were subjected alkylated by the addition of 200 uL of 20 mM iodoacetamide in UA buffer (30 min at room temperature in darkness). Excess SDS molecules were removed by two more washes using 200 uL of UA buffer, followed by two more washes of 200 uL 50mM Tris-HCl (pH 8.0) to dilute the urea. Samples were incubated in 200 uL of 50mM Tris-HCl (pH 8.0), supplemented with 3 ug of Trypsin (TPCK Treated, Worthington) at 37 Â°C overnight. 200 uL of 50mM Tris-HCl (pH 8.0) was added to elute the peptides by centrifugation (twice). Peptides were fractionated, using an in-house constructed SCX column with five pH fractions (pH 4.0, 6.0, 8.0, 10.0 and 12.0, Buffer composition: 20 mM boric acid, 20 mM phosphoric acid, and 20 mM acetic acid, with the pH adjusted by using 1 M NaOH.) Finally, the fractionated samples were desalted using in-house C18 desalting cartridges and dried in a speed-vac prior to LC-MS analysis. To save MS processing time, peptides acquired from pH 4.0 and pH 12.0 in each SCN sample were combined and dried, respectively. All resulting peptide mixtures were analyzed by high-performance liquid chromatography/electrospray ionization tandem mass spectrometry (HPLC-ESIMS/MS). The HPLC-ESI-MS/MS consisted of an automated  nanoLC 400 system (Eksigent, Dublin, CA, USA) coupled with an LTQ Velos Pro Orbitrap Elite mass spectrometer (ThermoFisher Scientific, San Jose, CA) equipped with a nanoelectrospray interface operated in positive ion mode.

### Data Protocol
All raw files were processed and analyzed by MaxQuant (Version 1.3.0.5) against the decoy Uniprot-Mouse database release (2013_05), including commonly observed contaminants. The following parameters were used: cysteine carbamidomethylation was selected as a fixed modification, with methionine oxidation, protein N-terminal acetylation and heavy proline set as variable modifications. Enzyme specificity was set to trypsin. Up to two missing cleavages of trypsin were allowed. SILAC double labelling (light: K0R0; heavy: K8R10) was set as the search parameter in order to assess the conversion efficiency. The precursor ion mass tolerances were 7 ppm and the fragment ion mass tolerance was 0.5 Da for MS/MS spectra. The false discovery rate (FDR) for peptides and proteins was set at 1% and a minimum length of six amino acids was used for peptide identification. The protein-group file was imported into Perseus (version 1.5.2.4) for data statistical analysis. First, MS raw dataset (4569 proteins) was filtered to generate a stringently quantified dataset of 2335 proteins, which only includes proteins with a minimum peptide ratio count of 2 and with quantification values in at least 16 of 32 MS measurements (16 independent SCN samples). One-way ANOVA was used to analyze this stringent dataset for identifying proteins exhibited statistically significant (p<0.05) alterations in abundance from miR-132/212 KO and WT proteome data across the 24-h cycle (n=4 per CT; 4 CT in total), respectively.

### Publication Abstract
The central circadian pacemaker, the suprachiasmatic nucleus (SCN), encodes day length information by mechanisms that are not well understood. Here, we report that genetic ablation of miR-132/212 alters entrainment to different day lengths and non-24&#xa0;hr day-night cycles, as well as photoperiodic regulation of Period2 expression in the SCN. SCN neurons from&#xa0;miR-132/212-deficient mice have significantly reduced dendritic spine density, along with altered methyl CpG-binding protein (MeCP2) rhythms. In Syrian hamsters, a model seasonal rodent, day length regulates spine density on SCN neurons in a melatonin-independent manner, as well as expression of miR-132, miR-212, and their direct target, MeCP2. Genetic disruption of Mecp2 fully restores the level of dendritic spines of miR-132/212-deficient SCN neurons. Our results reveal that, by regulating the dendritic structure of SCN neurons through a MeCP2-dependent mechanism, miR-132/212 affects the capacity of the SCN to encode seasonal time.

### Keywords
Mouse, Lc-ms/ms, Scn

### Affiliations
university of ottawa
Ottawa Institute of Systems Biology and Department of Biochemistry, Microbiology and Immunology, Faculty of Medicine, University of Ottawa, ON, Canada

### Submitter
kerwin chiang

### Lab Head
Dr Daniel Figeys
Ottawa Institute of Systems Biology and Department of Biochemistry, Microbiology and Immunology, Faculty of Medicine, University of Ottawa, ON, Canada


